<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/8/10/3321" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/8/10/3321.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/8/10/3321.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/8/10/3321.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/8/10/3321/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/8/10/3321.atom" />
<meta name="HW.pisa" content="ggg;8/10/3321" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Comparative Genomics Approaches Accurately Predict Deleterious Variants in Plants" />
<meta name="DC.Identifier" content="10.1534/g3.118.200563" />
<meta name="DC.Date" content="2018-10-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2018 Kono et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Recent advances in genome resequencing have led to increased interest in prediction of the functional consequences of genetic variants. Variants at phylogenetically conserved sites are of particular interest, because they are more likely than variants at phylogenetically variable sites to have deleterious effects on fitness and contribute to phenotypic variation. Numerous comparative genomic approaches have been developed to predict deleterious variants, but the approaches are nearly always assessed based on their ability to identify known disease-causing mutations in humans. Determining the accuracy of deleterious variant predictions in nonhuman species is important to understanding evolution, domestication, and potentially to improving crop quality and yield. To examine our ability to predict deleterious variants in plants we generated a curated database of 2,910 Arabidopsis thaliana mutants with known phenotypes. We evaluated seven approaches and found that while all performed well, their relative ranking differed from prior benchmarks in humans. We conclude that deleterious mutations can be reliably predicted in A. thaliana and likely other plant species, but that the relative performance of various approaches does not necessarily translate from one species to another." />
<meta name="DC.Contributor" content="Thomas J.Y. Kono" />
<meta name="DC.Contributor" content="Li Lei" />
<meta name="DC.Contributor" content="Ching-Hua Shih" />
<meta name="DC.Contributor" content="Paul J. Hoffman" />
<meta name="DC.Contributor" content="Peter L. Morrell" />
<meta name="DC.Contributor" content="Justin C. Fay" />
<meta name="article:published_time" content="2018-10-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Comparative Genomics Approaches Accurately Predict Deleterious Variants in Plants" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Recent advances in genome resequencing have led to increased interest in prediction of the functional consequences of genetic variants. Variants at phylogenetically conserved sites are of particular interest, because they are more likely than variants at phylogenetically variable sites to have deleterious effects on fitness and contribute to phenotypic variation. Numerous comparative genomic approaches have been developed to predict deleterious variants, but the approaches are nearly always assessed based on their ability to identify known disease-causing mutations in humans. Determining the accuracy of deleterious variant predictions in nonhuman species is important to understanding evolution, domestication, and potentially to improving crop quality and yield. To examine our ability to predict deleterious variants in plants we generated a curated database of 2,910 &lt;i&gt;Arabidopsis thaliana&lt;/i&gt; mutants with known phenotypes. We evaluated seven approaches and found that while all performed well, their relative ranking differed from prior benchmarks in humans. We conclude that deleterious mutations can be reliably predicted in &lt;i&gt;A. thaliana&lt;/i&gt; and likely other plant species, but that the relative performance of various approaches does not necessarily translate from one species to another.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2018/10/01" />
<meta name="citation_mjid" content="ggg;8/10/3321" />
<meta name="citation_id" content="8/10/3321" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/8/10/3321" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/8/10/3321.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/8/10/3321.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/8/10/3321.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.118.200563" />
<meta name="citation_pmid" content="30139765" />
<meta name="citation_volume" content="8" />
<meta name="citation_issue" content="10" />
<meta name="citation_num_pages" content="9" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="3321" />
<meta name="citation_lastpage" content="3329" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Thomas J.Y. Kono" />
<meta name="citation_author_institution" content="Department of Agronomy &amp; Plant Genetics, University of Minnesota, St. Paul, MN 551085" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-1388-3194" />
<meta name="citation_author" content="Li Lei" />
<meta name="citation_author_institution" content="Department of Agronomy &amp; Plant Genetics, University of Minnesota, St. Paul, MN 551085" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-5708-0118" />
<meta name="citation_author" content="Ching-Hua Shih" />
<meta name="citation_author_institution" content="Department of Genetics, Washington University, St. Louis, MO 63110" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-6652-1197" />
<meta name="citation_author" content="Paul J. Hoffman" />
<meta name="citation_author_institution" content="Department of Agronomy &amp; Plant Genetics, University of Minnesota, St. Paul, MN 551085" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-7693-8957" />
<meta name="citation_author" content="Peter L. Morrell" />
<meta name="citation_author_institution" content="Department of Agronomy &amp; Plant Genetics, University of Minnesota, St. Paul, MN 551085" />
<meta name="citation_author_email" content="justin.fay@rochester.edu" />
<meta name="citation_author_email" content="pmorrell@umn.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-6282-1582" />
<meta name="citation_author" content="Justin C. Fay" />
<meta name="citation_author_institution" content="Department of Genetics, Washington University, St. Louis, MO 63110" />
<meta name="citation_author_email" content="justin.fay@rochester.edu" />
<meta name="citation_author_email" content="pmorrell@umn.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-1893-877X" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=Abecasis. G. R.;citation_author=A. Auton;citation_author=L. D. Brooks;citation_author=M. A. DePristo;citation_title=An integrated map of genetic variation from 1,092 human genomes.;citation_pages=56-65;citation_volume=491;citation_year=2012;citation_issue=7422;citation_pmid=23128226;citation_doi=10.1038/nature11632" />
<meta name="citation_reference" content="citation_author=I. Adzhubei;citation_author=D. M. Jordan;citation_author=S. R. Sunyaev;citation_pages=7201;citation_volume=76;citation_year=2013;citation_pmid=23315938;citation_doi=10.1002/0471142905.hg0720s76" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=I. A. Adzhubei;citation_author=S. Schmidt;citation_author=L. Peshkin;citation_author=V. E. Ramensky;citation_author=A. Gerasimova;citation_title=A method and server for predicting damaging missense mutations.;citation_pages=248-249;citation_volume=7;citation_year=2010;citation_issue=4;citation_pmid=20354512;citation_doi=10.1038/nmeth0410-248" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=N. Ahituv;citation_author=N. Kavaslar;citation_author=W. Schackwitz;citation_author=A. Ustaszewska;citation_author=J. Martin;citation_title=Medical sequencing at the extremes of human body mass.;citation_pages=779-791;citation_volume=80;citation_year=2007;citation_issue=4;citation_pmid=17357083;citation_doi=10.1086/513471" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=S. F. Altschul;citation_author=W. Gish;citation_author=W. Miller;citation_author=E. W. Myers;citation_author=D. J. Lipman;citation_title=Basic local alignment search tool.;citation_pages=403-410;citation_volume=215;citation_year=1990;citation_issue=3;citation_pmid=2231712;citation_doi=10.1006/jmbi.1990.9999" />
<meta name="citation_reference" content="citation_title=Plant Bioinformatics;citation_title=UniProtKB/Swiss-Prot, the manually annotated section of the UniProt KnowledgeBase: how to use the entry view;citation_author=E. Boutet;citation_author=D. Lieberherr;citation_author=M. Tognolli;citation_author=M. Schneider;citation_author=P. Bansal;citation_pages=23-54;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=A. R. Boyko;citation_author=S. H. Williamson;citation_author=A. R. Indap;citation_author=J. D. Degenhardt;citation_author=R. D. Hernandez;citation_title=Assessing the evolutionary impact of amino acid mutations in the human genome.;citation_pages=e1000083-e1000083;citation_volume=4;citation_year=2008;citation_issue=5;citation_pmid=18516229;citation_doi=10.1371/journal.pgen.1000083" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=M. S. Breen;citation_author=C. Kemena;citation_author=P. K. Vlasov;citation_author=C. Notredame;citation_author=F. A. Kondrashov;citation_title=Epistasis as the primary factor in molecular evolution.;citation_pages=535-538;citation_volume=490;citation_year=2012;citation_issue=7421;citation_pmid=23064225;citation_doi=10.1038/nature11510" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=J. Cao;citation_author=K. Schneeberger;citation_author=S. Ossowski;citation_author=T. Günther;citation_author=S. Bender;citation_title=Whole-genome sequencing of multiple Arabidopsis thaliana populations.;citation_pages=956-963;citation_volume=43;citation_year=2011;citation_issue=10;citation_pmid=21874002;citation_doi=10.1038/ng.911" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=B. Charlesworth;citation_title=The role of background selection in shaping patterns of molecular evolution and variation: evidence from variability on the Drosophila X chromosome.;citation_volume=191;citation_year=2012;citation_pmid=22377629;citation_doi=10.1534/genetics.111.138073" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=D. Charlesworth;citation_author=J. H. Willis;citation_title=The genetics of inbreeding depression.;citation_pages=783-796;citation_volume=10;citation_year=2009;citation_issue=11;citation_pmid=19834483;citation_doi=10.1038/nrg2664" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=Y. Choi;citation_author=G. E. Sims;citation_author=S. Murphy;citation_author=J. R. Miller;citation_author=A. P. Chan;citation_title=Predicting the functional effect of amino Acid substitutions and indels.;citation_pages=e46688-e46688;citation_volume=7;citation_year=2012;citation_issue=10;citation_pmid=23056405;citation_doi=10.1371/journal.pone.0046688" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=S. Chun;citation_author=J. C. Fay;citation_title=Identification of deleterious mutations within three human genomes.;citation_volume=19;citation_year=2009;citation_pmid=19602639;citation_doi=10.1101/gr.092619.109" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=S. Chun;citation_author=J. C. Fay;citation_title=Evidence for hitchhiking of deleterious mutations within the human genome.;citation_pages=e1002240-e1002240;citation_volume=7;citation_year=2011;citation_issue=8;citation_pmid=21901107;citation_doi=10.1371/journal.pgen.1002240" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=L. Comai;citation_title=The advantages and disadvantages of being polyploid.;citation_pages=836-846;citation_volume=6;citation_year=2005;citation_issue=11;citation_pmid=16304599;citation_doi=10.1038/nrg1711" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=G. M. Cooper;citation_author=J. Shendure;citation_title=Needles in stacks of needles: finding disease-causal variants in a wealth of genomic data.;citation_pages=628-640;citation_volume=12;citation_year=2011;citation_issue=9;citation_pmid=21850043;citation_doi=10.1038/nrg3046" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=F. Cruz;citation_author=C. Vilà;citation_author=M. T. Webster;citation_title=The legacy of domestication: accumulation of deleterious mutations in the dog genome.;citation_volume=25;citation_year=2008;citation_pmid=18689870;citation_doi=10.1093/molbev/msn177" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Comput Biol;citation_author=E. V. Davydov;citation_author=D. L. Goode;citation_author=M. Sirota;citation_author=G. M. Cooper;citation_author=A. Sidow;citation_title=Identifying a high fraction of the human genome to be under selective constraint using GERP++.;citation_pages=e1001025-e1001025;citation_volume=6;citation_year=2010;citation_issue=12;citation_pmid=21152010;citation_doi=10.1371/journal.pcbi.1001025" />
<meta name="citation_reference" content="citation_journal_title=Hum. Mol. Genet.;citation_author=C. Dong;citation_author=P. Wei;citation_author=X. Jian;citation_author=R. Gibbs;citation_author=E. Boerwinkle;citation_title=Comparison and integration of deleteriousness prediction methods for nonsynonymous SNVs in whole exome sequencing studies.;citation_volume=24;citation_year=2015;citation_pmid=25552646;citation_doi=10.1093/hmg/ddu733" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=S. W. Doniger;citation_author=H. S. Kim;citation_author=D. Swain;citation_author=D. Corcuera;citation_author=M. Williams;citation_title=A catalog of neutral and deleterious polymorphism in yeast.;citation_pages=e1000183-e1000183;citation_volume=4;citation_year=2008;citation_issue=8;citation_pmid=18769710;citation_doi=10.1371/journal.pgen.1000183" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=J. T. Dudley;citation_author=R. Chen;citation_author=M. Sanderford;citation_author=A. J. Butte;citation_author=S. Kumar;citation_title=Evolutionary meta-analysis of association studies reveals ancient constraints affecting disease marker discovery.;citation_volume=29;citation_year=2012;citation_pmid=22389448;citation_doi=10.1093/molbev/mss079" />
<meta name="citation_reference" content="citation_title=Mathematical population genetics.;citation_author=W. J. Ewens;citation_year=2004;citation_doi=10.1007/978-0-387-21822-9" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. C. Fay;citation_author=G. J. Wyckoff;citation_author=C.-I. Wu;citation_title=Positive and Negative Selection on the Human Genome;citation_pages=1227-1234;citation_volume=158;citation_year=2001;citation_issue=3;citation_pmid=11454770" />
<meta name="citation_reference" content="citation_journal_title=Curr. Opin. Genet. Dev.;citation_author=J. C. Fay;citation_title=The molecular basis of phenotypic variation in yeast.;citation_pages=672-677;citation_volume=23;citation_year=2013;citation_pmid=24269094;citation_doi=10.1016/j.gde.2013.10.005" />
<meta name="citation_reference" content="citation_journal_title=Trends in Ecology &amp; Evolution;citation_journal_abbrev=Trends in Ecology &amp; Evolution;citation_author=C. Finlayson;citation_title=Biogeography and evolution of the genus Homo.;citation_pages=457-463;citation_volume=20;citation_year=2005;citation_issue=8;citation_pmid=16701417;citation_doi=10.1016/j.tree.2005.05.019" />
<meta name="citation_reference" content="citation_journal_abbrev=J Stat Softw;citation_author=J. Friedman;citation_author=T. Hastie;citation_author=R. Tibshirani;citation_title=Regularization Paths for Generalized Linear Models via Coordinate Descent.;citation_pages=1-22;citation_volume=33;citation_year=2010;citation_issue=1;citation_pmid=20808728;citation_doi=10.1111/j.1467-9868.2005.00503.x" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=A. González-Pérez;citation_author=N. López-Bigas;citation_title=Improving the assessment of the outcome of nonsynonymous SNVs with a consensus deleteriousness score, Condel.;citation_pages=440-449;citation_volume=88;citation_year=2011;citation_issue=4;citation_pmid=21457909;citation_doi=10.1016/j.ajhg.2011.03.004" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=D. M. Goodstein;citation_author=S. Shu;citation_author=R. Howson;citation_author=R. Neupane;citation_author=R. D. Hayes;citation_title=Phytozome: a comparative platform for green plant genomics.;citation_volume=40;citation_year=2012;citation_pmid=22110026;citation_doi=10.1093/nar/gkr944" />
<meta name="citation_reference" content="citation_journal_title=Hum. Mutat.;citation_author=D. G. Grimm;citation_author=C.-A. Azencott;citation_author=F. Aicheler;citation_author=U. Gieraths;citation_author=D. G. MacArthur;citation_title=The evaluation of tools used to predict the impact of missense variants is hindered by two types of circularity.;citation_pages=513-523;citation_volume=36;citation_year=2015;citation_pmid=25684150;citation_doi=10.1002/humu.22768" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=T. Günther;citation_author=K. J. Schmid;citation_title=Deleterious amino acid polymorphisms in Arabidopsis thaliana and rice.;citation_pages=157-168;citation_volume=121;citation_year=2010;citation_issue=1;citation_pmid=20198468;citation_doi=10.1007/s00122-010-1299-4" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=A. M. Hancock;citation_author=B. Brachi;citation_author=N. Faure;citation_author=M. W. Horton;citation_author=L. B. Jarymowycz;citation_title=Adaptation to Climate Across the Arabidopsis thaliana Genome;citation_pages=83-86;citation_volume=334;citation_year=2011;citation_issue=6052;citation_pmid=21980108;citation_doi=10.1126/science.1209244" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=B. M. Henn;citation_author=L. R. Botigué;citation_author=S. Peischl;citation_author=I. Dupanloup;citation_author=M. Lipatov;citation_title=Distance from sub-Saharan Africa predicts mutational load in diverse human genomes.;citation_volume=113;citation_year=2016;citation_pmid=26712023;citation_doi=10.1073/pnas.1510805112" />
<meta name="citation_reference" content="citation_journal_title=Human mutation;citation_journal_abbrev=Hum Mutat;citation_author=S. Hicks;citation_author=D. A. Wheeler;citation_author=S. E. Plon;citation_author=M. Kimmel;citation_title=Prediction of missense mutation functionality depends on both the algorithm and sequence alignment employed.;citation_pages=661-668;citation_volume=32;citation_year=2011;citation_issue=6;citation_pmid=21480434;citation_doi=10.1002/humu.21490" />
<meta name="citation_reference" content="citation_journal_title=J. Biogeogr.;citation_author=M. H. Hoffmann;citation_title=Biogeography of Arabidopsis thaliana L. Heynh. (Brassicaceae).;citation_pages=125-134;citation_volume=29;citation_year=2002;citation_doi=10.1046/j.1365-2699.2002.00647.x" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=D. M. Jordan;citation_author=S. G. Frangakis;citation_author=C. Golzio;citation_author=C. A. Cassa;citation_author=J. Kurtzberg;citation_title=Identification of cis-suppression of human disease mutations by comparative genomics.;citation_pages=225-229;citation_volume=524;citation_year=2015;citation_pmid=26123021;citation_doi=10.1038/nature14497" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=M. Kircher;citation_author=D. M. Witten;citation_author=P. Jain;citation_author=B. J. O’Roak;citation_author=G. M. Cooper;citation_title=A general framework for estimating the relative pathogenicity of human genetic variants.;citation_pages=310-315;citation_volume=46;citation_year=2014;citation_pmid=24487276;citation_doi=10.1038/ng.2892" />
<meta name="citation_reference" content="citation_author=F. A. Kondrashov;citation_author=I. B. Rogozin;citation_author=Y. I. Wolf;citation_author=E. V. Koonin;citation_year=2002" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=T. J. Y. Kono;citation_author=F. Fu;citation_author=M. Mohammadi;citation_author=P. J. Hoffman;citation_author=C. Liu;citation_title=The role of deleterious substitutions in crop genomes.;citation_volume=33;citation_year=2016;citation_pmid=27301592;citation_doi=10.1093/molbev/msw102" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=C. Kosiol;citation_author=T. Vinař;citation_author=R. R. da Fonseca;citation_author=M. J. Hubisz;citation_author=C. D. Bustamante;citation_title=Patterns of positive selection in six Mammalian genomes.;citation_pages=e1000144-e1000144;citation_volume=4;citation_year=2008;citation_issue=8;citation_pmid=18670650;citation_doi=10.1371/journal.pgen.1000144" />
<meta name="citation_reference" content="citation_journal_title=PLoS biology;citation_journal_abbrev=PLoS Biol;citation_author=E. M. Leffler;citation_author=K. Bullaughey;citation_author=D. R. Matute;citation_author=W. K. Meyer;citation_author=L. Ségurel;citation_title=Revisiting an old riddle: what determines genetic diversity levels within species?;citation_pages=e1001388-e1001388;citation_volume=10;citation_year=2012;citation_issue=9;citation_pmid=22984349;citation_doi=10.1371/journal.pbio.1001388" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=B. Li;citation_author=V. G. Krishnan;citation_author=M. E. Mort;citation_author=F. Xin;citation_author=K. K. Kamati;citation_title=Automated inference of molecular mechanisms of disease from amino acid substitutions.;citation_volume=25;citation_year=2009;citation_pmid=19734154;citation_doi=10.1093/bioinformatics/btp528" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=Q. Liu;citation_author=Y. Zhou;citation_author=P. L. Morrell;citation_author=B. S. Gaut;citation_title=Deleterious variants in Asian rice and the potential cost of domestication.;citation_volume=34;citation_year=2017;citation_pmid=28087781;citation_doi=10.1093/molbev/msw296" />
<meta name="citation_reference" content="citation_journal_title=Trends in genetics :  TIG;citation_journal_abbrev=Trends Genet;citation_author=S. Lockton;citation_author=B. S. Gaut;citation_title=Plant conserved non-coding sequences and paralogue evolution.;citation_pages=60-65;citation_volume=21;citation_year=2005;citation_issue=1;citation_pmid=15680516;citation_doi=10.1016/j.tig.2004.11.013" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=K. E. Lohmueller;citation_author=A. R. Indap;citation_author=S. Schmidt;citation_author=A. R. Boyko;citation_author=R. D. Hernandez;citation_title=Proportionally more deleterious genetic variation in European than in African populations.;citation_pages=994-997;citation_volume=451;citation_year=2008;citation_issue=7181;citation_pmid=18288194;citation_doi=10.1038/nature06611" />
<meta name="citation_reference" content="citation_journal_title=Trends in genetics :  TIG;citation_journal_abbrev=Trends Genet;citation_author=J. Lu;citation_author=T. Tang;citation_author=H. Tang;citation_author=J. Huang;citation_author=S. Shi;citation_title=The accumulation of deleterious mutations in rice genomes: a hypothesis on the cost of domestication.;citation_pages=126-131;citation_volume=22;citation_year=2006;citation_issue=3;citation_pmid=16443304;citation_doi=10.1016/j.tig.2006.01.004" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=M. Lynch;citation_author=J. S. Conery;citation_title=The Evolutionary Fate and Consequences of Duplicate Genes;citation_pages=1151-1155;citation_volume=290;citation_year=2000;citation_issue=5494;citation_pmid=11073452;citation_doi=10.1126/science.290.5494.1151" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=T. A. Manolio;citation_author=F. S. Collins;citation_author=N. J. Cox;citation_author=D. B. Goldstein;citation_author=L. A. Hindorff;citation_title=Finding the missing heritability of complex diseases.;citation_pages=747-753;citation_volume=461;citation_year=2009;citation_issue=7265;citation_pmid=19812666;citation_doi=10.1038/nature08494" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=N. J. Marini;citation_author=P. D. Thomas;citation_author=J. Rine;citation_title=The use of orthologous sequences to predict the impact of amino acid substitutions on protein function.;citation_pages=e1000968-e1000968;citation_volume=6;citation_year=2010;citation_issue=5;citation_pmid=20523748;citation_doi=10.1371/journal.pgen.1000968" />
<meta name="citation_reference" content="citation_author=S. Mezmouk;citation_author=J. Ross-Ibarra;citation_year=2014;citation_pmid=24281428;citation_doi=10.1534/g3.113.008870" />
<meta name="citation_reference" content="citation_journal_title=Human Molecular Genetics;citation_journal_abbrev=Hum Mol Genet;citation_author=M. P. Miller;citation_author=S. Kumar;citation_title=Understanding human disease mutations through the use of interspecific genetic variation;citation_pages=2319-2328;citation_volume=10;citation_year=2001;citation_issue=21;citation_pmid=11689479;citation_doi=10.1093/hmg/10.21.2319" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=L. A. Miosge;citation_author=M. A. Field;citation_author=Y. Sontani;citation_author=V. Cho;citation_author=S. Johnson;citation_title=Comparison of predicted and actual consequences of missense mutations.;citation_volume=112;citation_year=2015;citation_pmid=26269570;citation_doi=10.1073/pnas.1511585112" />
<meta name="citation_reference" content="citation_journal_title=J. Comput. Biol.;citation_author=S. Mirarab;citation_author=N. Nguyen;citation_author=S. Guo;citation_author=L.-S. Wang;citation_author=J. Kim;citation_title=PASTA: ultra-large multiple sequence alignment for nucleotide and amino-acid sequences.;citation_pages=377-386;citation_volume=22;citation_year=2015;citation_pmid=25549288;citation_doi=10.1089/cmb.2014.0156" />
<meta name="citation_reference" content="citation_journal_title=Nat. Rev. Genet.;citation_author=P. L. Morrell;citation_author=E. S. Buckler;citation_author=J. Ross-Ibarra;citation_title=Crop genomics: advances and applications.;citation_pages=85-96;citation_volume=13;citation_year=2012;citation_doi=10.1038/nrg3097" />
<meta name="citation_reference" content="citation_journal_title=J. Hered.;citation_author=B. T. Moyers;citation_author=P. L. Morrell;citation_author=J. K. McKay;citation_title=Genetic costs of domestication and improvement.;citation_pages=103-116;citation_volume=109;citation_year=2018;citation_doi=10.1093/jhered/esx069" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=P. C. Ng;citation_author=S. Henikoff;citation_title=SIFT: predicting amino acid changes that affect protein function;citation_pages=3812-3814;citation_volume=31;citation_year=2003;citation_issue=13;citation_pmid=12824425;citation_doi=10.1093/nar/gkg509" />
<meta name="citation_reference" content="citation_journal_title=Annual review of genomics and human genetics;citation_journal_abbrev=Annu Rev Genomics Hum Genet;citation_author=P. C. Ng;citation_author=S. Henikoff;citation_title=Predicting the effects of amino acid substitutions on protein function.;citation_pages=61-80;citation_volume=7;citation_year=2006;citation_pmid=16824020;citation_doi=10.1146/annurev.genom.7.080505.115630" />
<meta name="citation_reference" content="citation_title=Evolution by gene duplication;citation_author=S. Ohno;citation_year=1970;citation_doi=10.1007/978-3-642-86659-3" />
<meta name="citation_reference" content="citation_journal_title=Human mutation;citation_journal_abbrev=Hum Mutat;citation_author=A. Olatubosun;citation_author=J. Väliaho;citation_author=J. Härkönen;citation_author=J. Thusberg;citation_author=M. Vihinen;citation_title=PON-P: integrated predictor for pathogenicity of missense variants.;citation_pages=1166-1174;citation_volume=33;citation_year=2012;citation_issue=8;citation_pmid=22505138;citation_doi=10.1002/humu.22102" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=K. S. Pollard;citation_author=M. J. Hubisz;citation_author=K. R. Rosenbloom;citation_author=A. Siepel;citation_title=Detection of nonneutral substitution rates on mammalian phylogenies.;citation_volume=20;citation_year=2010;citation_pmid=19858363;citation_doi=10.1101/gr.097857.109" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=S. L. K. Pond;citation_author=S. D. W. Frost;citation_author=S. V. Muse;citation_title=HyPhy: hypothesis testing using phylogenies.;citation_volume=21;citation_year=2005;citation_pmid=15509596;citation_doi=10.1093/bioinformatics/bti079" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. Poon;citation_author=B. H. Davis;citation_author=L. Chao;citation_title=The coupon collector and the suppressor mutation: estimating the number of compensatory mutations by maximum likelihood.;citation_volume=170;citation_year=2005;citation_pmid=15879511;citation_doi=10.1534/genetics.104.037259" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=S. Renaut;citation_author=L. H. Rieseberg;citation_title=The accumulation of deleterious mutations as a consequence of domestication and improvement in sunflowers and other compositae crops.;citation_volume=32;citation_year=2015;citation_pmid=25939650;citation_doi=10.1093/molbev/msv106" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=X. Robin;citation_author=N. Turck;citation_author=A. Hainard;citation_author=N. Tiberti;citation_author=F. Lisacek;citation_title=pROC: an open-source package for R and S+ to analyze and compare ROC curves.;citation_pages=77-77;citation_volume=12;citation_year=2011;citation_pmid=21414208;citation_doi=10.1186/1471-2105-12-77" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=E. Rodgers-Melnick;citation_author=P. J. Bradbury;citation_author=R. J. Elshire;citation_author=J. C. Glaubitz;citation_author=C. B. Acharya;citation_title=Recombination in diverse maize is stable, predictable, and associated with genetic load.;citation_volume=112;citation_year=2015;citation_pmid=25775595;citation_doi=10.1073/pnas.1413864112" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=J. M. Schwarz;citation_author=C. Rödelsperger;citation_author=M. Schuelke;citation_author=D. Seelow;citation_title=MutationTaster evaluates disease-causing potential of sequence alterations.;citation_pages=575-576;citation_volume=7;citation_year=2010;citation_issue=8;citation_pmid=20676075;citation_doi=10.1038/nmeth0810-575" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=Y. B. Simons;citation_author=M. C. Turchin;citation_author=J. K. Pritchard;citation_author=G. Sella;citation_title=The deleterious mutation load is insensitive to recent population history.;citation_pages=220-224;citation_volume=46;citation_year=2014;citation_pmid=24509481;citation_doi=10.1038/ng.2896" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=T. Slotte;citation_author=K. M. Hazzouri;citation_author=J. A. Ågren;citation_author=D. Koenig;citation_author=F. Maumus;citation_title=The Capsella rubella genome and the genomic consequences of rapid mating system evolution.;citation_pages=831-835;citation_volume=45;citation_year=2013;citation_pmid=23749190;citation_doi=10.1038/ng.2669" />
<meta name="citation_reference" content="citation_journal_title=Hum. Genet.;citation_author=P. D. Stenson;citation_author=M. Mort;citation_author=E. V. Ball;citation_author=K. Shaw;citation_author=A. D. Phillips;citation_title=The human gene mutation database: building a comprehensive mutation repository for clinical and molecular genetics, diagnostic testing and personalized genomic medicine.;citation_pages=1-9;citation_volume=133;citation_year=2014;citation_pmid=24077912;citation_doi=10.1007/s00439-013-1358-4" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=E. A. Stone;citation_author=A. Sidow;citation_title=Physicochemical constraint violation by missense substitutions mediates impairment of protein function and disease severity.;citation_volume=15;citation_year=2005;citation_pmid=15965030;citation_doi=10.1101/gr.3804205" />
<meta name="citation_reference" content="citation_journal_title=Protein Engineering Design and Selection;citation_journal_abbrev=Protein Eng Des Sel;citation_author=S. R. Sunyaev;citation_author=F. Eisenhaber;citation_author=I. V. Rodchenkov;citation_author=B. Eisenhaber;citation_author=V. G. Tumanyan;citation_title=PSIC: profile extraction from sequence alignments with position-specific counts of independent observations;citation_pages=387-394;citation_volume=12;citation_year=1999;citation_issue=5;citation_pmid=10360979;citation_doi=10.1093/protein/12.5.387" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_title=Analysis of the genome sequence of the flowering plant Arabidopsis thaliana.;citation_pages=796-815;citation_volume=408;citation_year=2000;citation_issue=6814;citation_pmid=11130711;citation_doi=10.1038/35048692" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=K. R. Thornton;citation_author=A. J. Foran;citation_author=A. D. Long;citation_title=Properties and modeling of GWAS when complex disease risk is due to non-complementing, deleterious mutations in genes of large effect.;citation_pages=e1003258;citation_volume=9;citation_year=2013;citation_pmid=23437004;citation_doi=10.1371/journal.pgen.1003258" />
<meta name="citation_reference" content="citation_journal_title=Human mutation;citation_journal_abbrev=Hum Mutat;citation_author=J. Thusberg;citation_author=A. Olatubosun;citation_author=M. Vihinen;citation_title=Performance of mutation pathogenicity prediction methods on missense variants.;citation_pages=358-368;citation_volume=32;citation_year=2011;citation_issue=4;citation_pmid=21412949;citation_doi=10.1002/humu.21445" />
<meta name="citation_reference" content="citation_journal_title=Nat. Protoc.;citation_author=R. Vaser;citation_author=S. Adusumalli;citation_author=S. N. Leng;citation_author=M. Sikic;citation_author=P. C. Ng;citation_title=SIFT missense predictions for genomes.;citation_pages=1-9;citation_volume=11;citation_year=2016;citation_pmid=26633127;citation_doi=10.1038/nprot.2015.123" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=J. Yang;citation_author=S. Mezmouk;citation_author=A. Baumgarten;citation_author=E. S. Buckler;citation_author=K. E. Guill;citation_title=Incomplete dominance of deleterious alleles contributes substantially to trait variation and heterosis in maize.;citation_pages=e1007019;citation_volume=13;citation_year=2017;citation_doi=10.1371/journal.pgen.1007019" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=M. Zhang;citation_author=L. Zhou;citation_author=R. Bawa;citation_author=H. Suren;citation_author=J. A. Holliday;citation_title=Recombination rate variation, hitchhiking, and demographic history shape deleterious load in poplar.;citation_volume=33;citation_year=2016;citation_pmid=27512114;citation_doi=10.1093/molbev/msw169" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Comparative Genomics Approaches Accurately Predict Deleterious Variants in Plants" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10.cover-source.jpg" />
<meta name="twitter:description" content="Recent advances in genome resequencing have led to increased interest in prediction of the functional consequences of genetic variants. Variants at phylogenetically conserved sites are of particular interest, because they are more likely than variants at phylogenetically variable sites to have deleterious effects on fitness and contribute to phenotypic variation. Numerous comparative genomic approaches have been developed to predict deleterious variants, but the approaches are nearly always assessed based on their ability to identify known disease-causing mutations in humans. Determining the accuracy of deleterious variant predictions in nonhuman species is important to understanding evolution, domestication, and potentially to improving crop quality and yield. To examine our ability to predict deleterious variants in plants we generated a curated database of 2,910 Arabidopsis thaliana mutants with known phenotypes. We evaluated seven approaches and found that while all performed well, their relative ranking differed from prior benchmarks in humans. We conclude that deleterious mutations can be reliably predicted in A. thaliana and likely other plant species, but that the relative performance of various approaches does not necessarily translate from one species to another." />
<meta name="og-title" property="og:title" content="Comparative Genomics Approaches Accurately Predict Deleterious Variants in Plants" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/8/10/3321" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Recent advances in genome resequencing have led to increased interest in prediction of the functional consequences of genetic variants. Variants at phylogenetically conserved sites are of particular interest, because they are more likely than variants at phylogenetically variable sites to have deleterious effects on fitness and contribute to phenotypic variation. Numerous comparative genomic approaches have been developed to predict deleterious variants, but the approaches are nearly always assessed based on their ability to identify known disease-causing mutations in humans. Determining the accuracy of deleterious variant predictions in nonhuman species is important to understanding evolution, domestication, and potentially to improving crop quality and yield. To examine our ability to predict deleterious variants in plants we generated a curated database of 2,910 Arabidopsis thaliana mutants with known phenotypes. We evaluated seven approaches and found that while all performed well, their relative ranking differed from prior benchmarks in humans. We conclude that deleterious mutations can be reliably predicted in A. thaliana and likely other plant species, but that the relative performance of various approaches does not necessarily translate from one species to another." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10.cover-source.jpg" />
    <title>Comparative Genomics Approaches Accurately Predict Deleterious Variants in Plants | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"b8EnLeBxtzkIlFcOwJ93984gRrCS0anJkZ0vpPNtnbo","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"490979","apath":"\/ggg\/8\/10\/3321.atom","pisa":"ggg;8\/10\/3321","ac":{"\/ggg\/8\/10\/3321.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/8\/10\/3321.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;8\/10\/3321"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;8\/10\/3321"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/8\/10\/3321.abstract":true,"\/content\/8\/10\/3321":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00222l5BxyJkcKMO64JJ8bHph0Vmf0kOOsavGH\\\\\\\/HLDTLPWhmeXdddVaEoFKiE1udYhC12aapWvDdsOwYhcLosvHyFzy9VvCWm1NKK7cNPgCqwSnXFID55TI8weA3pC9UPDHlvoL6UovEPw8w\\\\\\\/oisc9whCtX1PQMb4OziT9i7NkTQ6MaZA7GvC4EQe2XTAyScrnjtC9Cz7E42ZQrek\\\\\\\/ljVOolXRltXJewT1wSgPG8EWGvkmLK33\\\\\\\/sSfdZq5swO7Qmt3nJzZWuaH7atGxH6AdlN28DZLPypiw8jjYjvwKxKwlK9\\\\\\\/ccYonAUcpManJDbk7iBmGGhkDmAaLi+O\\\\\\\/qNnScGGMF4I6zKDLL2JkyqXM06Mmo+V2rQLaOfPGVvo\\\\\\\/vaoSJvOhpGQcMItlbpicuJkhwMu6S62Kil1YaMNn+NpvcGJyt09qShLn+PQfPS7qWB5Oqf8KFFFdET+pakm4bMo7bliDXDM8KvXuz7iyGSoOboLVgUyLg+J2iOxjSzSsJ2QvRTQD40AuROK76GdGYqj81XqMvmqCwi1veitJnNUKjG\\\\\\\/5j6Yzj53i35e21stK6h2cw0vcccxkkWqm7Wy6YKnEbzIO77Ew2C7ogBzl5SYDqbtLEMX4Lo7Uixf5h1zt+3krRFcDW6qMyO3CBfs8i1BTaDAt5O4ao8QM3YHapUioJasVxTi2hXiemheNs4aqD3m4su8nW61biTonexLEUn92rKna5DFsnOpOvHuPEaW+IYg+ye\\\\\\\/42JckFEXDuLmy5C+5cL2eXhwd2TuR0h1kbXNHYeesOqnPwpfiAjUK5bVLqy+kI\\\\\\\/IMd7VL+rE8QpWy0NAW5DmD7yhxIbSAg\\\\\\\/iAftNgmz8AgvkJXYyE1\\\\\\\/\\\\\\\/VZRiVXByr9qx9J4ZMbf1wcFwVZn2VLK0oIFWnE4Cjtm4fyY7hd1H81RJlpnt3a9g7MeV2FHuoYb5vYE2J9F4nRJlzMEjmpo8G2rOgL43m1EDSCWgaBOj0NlTDi2A20N0HFJYN5ZP0GLRAGyBhEBZjjYColeVBSw607i+J10VsHzKxJyEVtAtB\\\\\\\/Un0BPQtVM7IS1ITLV2YRKDCCXj6A3CvmvDA8vns14WpPdRPXbCqBu7AirVEDUt7IgL9cM4UnEC6fuXfk7jb31ap\\\\\\\/UjxjEeMZwuJ6Lgaf5M55LJ9ei1FaOx5DVt0c7SUQPdMJrtSFGM2H0Qc+hTM3shQGP21fmJFb+7ufecpolg\\\\\\\/8qqjabjSaJevNMYdyIzyACJlNEKcNzqi4P1Hd\\\\\\\/LQ5ZWCbxMYjZf25PJ8s3li3ip6z\\\u0022,\\\u0022iv\\\u0022:\\\u0022QYB6t8bT5KED67zCI57TLg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00222193ad14f07eabe3ab138c8dce722c2c4304e818664e295dec3f468dff22edb3\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022oC3SDmVi0Lal7iGKQTVlgwUlSYwr5oMhDHybJsHDZMIW61zsgHG9RBMWznMfYYnLZws3av2n6+r81XDdG1KCGhyzxILXKtwQ\\\\\\\/JzMrW7oapQklZXsMINCGBgJscadEJGJB4UvuTfFEJC22gT3KWxM945rNcmRbBXHFn7JvyZRfen7zZjet1Jo\\\\\\\/GsPqcrJvwDE\\\\\\\/o6yWrplkrhi382J+jzqrW2BCE2qOY4jj4Q\\\\\\\/Y2opzsEI23sPk5dW+x6DA\\\\\\\/+8sztiogJkf7VJrie+jX3YT53cC9o9FgoBIv5WCS3wjAHNkHErcIrte1bWR4m2e+z\\\\\\\/Urz2KoQA17tAP\\\\\\\/sV7nxfIU4JFDV26luxPZH4nliw2SpSX6tdOsCMQ6Q6FPix81CedRBlYeWid\\\\\\\/2eu9OQTM68tGDnlN3PEPXnjPI0dnYAidUG9hFbXm9ATEVYF55aB+8VamnNAHIZ0WdofX1Zxit47jkqT0vjSavZo246NtPZp4JmeWztTKIykKniyXwvaXWHGDebb7NCyCaQvvkQpQHSVA97oHDHfskNIM1z+lKwwpFVY82k+addcHt57rYTm09aoJ5AnzXIySjEZOc+TaW1SI7EVnfT1QZJTSpWNP3lT96OZOJpK6yNstPU0C8kM+fLrmzpo9fCKJ6dVgyFVSchSWfaDIpkxhzk704ZCOvXLqaErOziHhZEzg5ZPrhRrHEXw84reOEUIiuNxDjlhZnrx+Qyt\\\\\\\/En\\\\\\\/FVKvLfsx9cyppANapLAl7inRIWj\\\\\\\/89O2hmQnct4CB4iT455+ayqfe\\\\\\\/vC0JU7cZ7YWwtJU893lq7O8QuloGjnUqWq0GBDiz10xTMezX+sTYwuzpR9YgTBQsYjF\\\\\\\/bs7wD1zAOBt3u0opJ4hGk+y+HMa42ec+n1S7QvUEemIUsLQGey21ySAjipTtiEPu\\\\\\\/xwrJk2h08Js6BF88R6B\\\\\\\/2anE3kIJqr9YDaCvEd2thJpd7b8w+DyLmTPmO50fd+MZvzYWFGjike\\\\\\\/HIQ+MuFsFYKeQrPaMs0p2UzjnGVBpz+yZW1FX8FG1LYMVLTW9v6QNBVt2SgNXNWqD2kIyOXNezs8FXQNyHSwdo6Wnvh4EnNKOA4VK3s06vSPoEWeDTnq0i2W5tdIoX1PHtoePZl1FHLIADipBI+7zJEFGbk7weGewkVRHkXBlBpcAxYo06HTnA\\\\\\\/ygrYhRvbc9ILQ2cHN+ArLHdg1c\\\\\\\/clVuCnae0Z3CSBH8rhKId1amP5MsfBFp6hO8skJ87l\\\\\\\/qEjfdzu9jYBXuHXfcSEKiC7JLCgcAWS6CtOSM5pOq\\\\\\\/wg7ESu+XzUpzc+d7keOdKid+wXgGCmqCv5F+VSULUBuCjetA4YwZ8LbPsLSFI3e1SWdmClq+\\\\\\\/FOaFnP2Ls5d1GtHVb+SiS9tG9+\\\\\\\/wGYoXoZIWo7zRqsF5YNleaiIsOtbiY31f20ZpxvDgCwRXWXLjtv9YG3fqC\\\\\\\/c5QbCXTnk2k2b7\\\\\\\/G7cMVzdJQy86dqQzAmBXUUNKcKZVR6iLruzx1DCn6JhcCqhbWv2BWy6z3hhelXUzLbMUZpJ8tafD1nwiqT4VlEzDcpTsuPIJtGuBsk2N8FTXzjkwM\\\\\\\/1n6fKtHIYRx2qGdQg\\\\\\\/oglGwQXcxxAjMZIOr44ydMup\\\\\\\/Ilf9RXevORGcbkYbZHrkv++2DRxBmpiBMqn0719Qn28wahtSp8eO7kvEoJeGrQlBD3oRx6mInihrFBKm3\\\\\\\/5S23duTv5mQF+rBWJXPnm5HsvcN7l+trk1qJHGU2vhzKzzn2hZIl10iveXCYy29manIA2ezvH\\\\\\\/7M1Xovs6YOd6cRkUQkRgxHx0zYEX1VCmU31amnw6G36n0nW+PmBZuFqI9mPIb06Me7aUzU64GN25Ke+5LUiYCYgtGjOdrhpb9P0WLrAH5h9nS+Fvx5JuJG0tzKWRrrzt63vH0+0coZblffKdC6jBSrRspDCeyrTveozENHY7+DPTa21udViYttaJBshcTijpqVa4wXfa9iDgIa460IKWk3h08LdITuw8ZcOjdFrylDfle\\\\\\\/j1647VW96xEL5IDs5cwcPqWNwdjaECJYLrpLVGIl+DnArP0P1DW5rye6HfygRW+VdfEDM8mvciHhAa51f9+7aWsMt0sAtC0OmvYApZbDqDn8MPGxe0e1FYpYPa88mlCup01ObKA9WAgB7vr+LOHULCLj3rO1104xp7AVNi\\\\\\\/zIEzp02Hpf+jM1gixarqatAX3eLd5szjHErsI92q\\\\\\\/v6QjZstIq73Ie3mYbUyP6n0vTcUCnfA\\\\\\\/Kfv5Gpshea9iZVtHKZ5OT2q7fOYiFwUtcv8A9K8VixjSnmEbM1QjZHYDVNhu1Dm19YF3YYYYqqa\\\\\\\/kD7A754W22ebmkgBrgd5whPAa4Qu\\\\\\\/K08EfCxTOJ8b2O30eFlYdCsm5ALOqAkyTzUtmFuDktYbVYSWJssWjgG98inj1XR\\\\\\\/NtS0KwKyQKESdQ01krCpPoAnUNivWJ78ibrHvRBP\\\\\\\/\\\\\\\/LTd8WTj6mFz16FnFJZvnbJzWBITL647+9oFLiNqHQdMUUj15nw9vSydBUdiO3RK47RzF6C\\\\\\\/K+EfAICPUkJYdIMCm88eooy10ky93HunCJ9QMkhhDjBMnloZJhl1tFpn3lQw5fYWQTi\\\\\\\/nIrNUdykPB3d3gL9Kd+cP\\\\\\\/kKVROzi856GE4EEbsv0BrXNevJzEHCz8xtvxlep7O7Gd+pbXuQKkm2v5aXdN4YIP6UCQddtqzn8CIQFwADFQG\\\\\\\/oBu9\\\\\\\/Mz92EGDlMabcblTP8k5fkfaFH\\\\\\\/6KnAFDdKEjbmOe8FhvKdEz810uPAKkRi6Fo7obXP0DZRjCqAqisdKMYHE0UkNE\\\\\\\/uS\\\\\\\/auvhBLmAtSLULJyVEOYFJgJRoGRNdlBrkif\\\\\\\/AhcWwZO5lK+ooLTvdJ18xGgVjBTuuLiMgQQ1YubvbrR0hkUWI3gxn1Qlo3JI75Jx+eXqBHU2gkgBVU4Y06D1DzZFXSBgRDAOvhtdFjUy\\\\\\\/4UylNeZg+\\\\\\\/rTMi8a3ib3WzfJFswpEjW3qFLdeU4cHlLzO+g2NxtvdA0L1ntcrWh4icdmuvrtEDy\\\\\\\/y2F7xd4FqL6cYO3nbfdjxHNJ9E3RfDrAhUkG6x\\\\\\\/72ptOiK51C4yHThxIilxHkXtrcHVDYfxuw+RgoR6rypn4Wl6lNZ70aIdzkZlA8yu4bZjpxJvV+4swzN2+2UNwcoM2e4mHGG5NDJ8W2yUuqgXa06KCONaNst0smbXnGfiRbFIUffW0qd4ibgQ12ECvafSh9sf\\\\\\\/K0InHK8ndILHkGdDKRvykSZYvKQs\\\\\\\/mj0AxdV2pOXzbhjD97DuAJqDEIfFI88rk+0MSELGOLSia7Q6RQsTRJNuEswQkGwMEg5iYumvA04MKZU\\\\\\\/fcjagKgweBbjNK1GmsVpBql0wP86nD0cmg2D0I0vu3nKQaqzOXVvQsvd0osHR+LYiJDj03TJw4j9T\\\\\\\/Gf4kWJpR3uq7wEZTgkTv4I1umyNZrlgmv+7QV99O9M4iyrYJvmL0IqFMX4CWuLPOePyGLaPXy3+nyPJlQgFlA39isOCLzK3yJh7U0\\\\\\\/a7wY2CpXFClC\\\\\\\/l7\\\\\\\/ouAogyZo6Z7QaUL07Sl9lgaK545DMUKS6+Zmxo0AZCRuU6P+Xew1t18D+wTQd\\\\\\\/jKvMWxNEeibvblKSdeiZhvHanbgf5NbBMHJLLQqS1w0rIfTCtIvoXhOvOvpSQGK+u1ErlKl5Q48fO\\\\\\\/sejK58wKkZoZKsfYb+hB1ua5Zxj3mk9ATxjPalYvQl0gzJjgy5KwN8\\\\\\\/7Q\\\\\\\/f3X5rcZLVlwF8IoYnDBsxFnC2pJYQ\\\\\\\/VjENeIM66+1woGIg4H+H6vE7wmCIaOf4lqG3i\\\\\\\/vKaTYhw+gaQmOUX6irIx1H\\\\\\\/JSoLzX2Bb3snpvccIksWm3gbmoJLxbMdSkCtcddiVLQ5XkYCFyxm0m5MuvdKn3E0xlwmwqP+0\\\\\\\/8i1xvUNHkz1kv09FHdhvV4gbv\\\\\\\/mS0MJD19rqhAFRGkAyLJRfLZtKPzkAsaF4QbxJDnvOCbQuRCoPUzXKiu\\\\\\\/8QghPhIWJEpwQCmDG\\\\\\\/jwOJSuhOrUA\\\\\\\/acrVke3xASRgUy4IZmcPLxkmNtTs9LGmNQ1sfTrVBsvxUJf9grqeX0U++ePAYuCp19PEBaWjsZfB41vL2Ef1pDUa6K9l3YFC7YRHBZDx8NJacvRQC2PXr5aL7HDtAEs1PynOw0HvxTeE5mffUiKzZ6RCkmJ8V+vZcjKxAQltKb9QdqA8rZeXvfWK6vhbbj3r2OnkRpzoicl14jwj7dLWAH6yXyR6kfeD6lF+QRq8+hnBcrZXUP+whP6OFsTAfy0X2cUiev2TByMo8JO2t9sAuC49kI52CIYw07oRWaqyR4DFD4gil6OlI7gZaDBakzkMI+s0Xgdc2ZU6oIWBlK1SB59n2rcOIbIYz5hxjh2oS6YhWAsPOZRB\\\\\\\/LwqvyZuIp1pcOJHOKhQoUTmoZBPtLVYCCt2wyDjgtoGXCEsK0d7UX+ber4H5DKyHccW7op9LTgzRKs1iHtx\\\\\\\/44j7PfeY3yXR7z+RNgeH66rV1nEF+1UCe0WzldfbhTSg1CncLSK6ljaxQGC6CRnu0N5aGH1y\\\\\\\/QAqINexjN0akUDk2izOcdEDVnVoR63CU68hLuX0\\\\\\\/2ulNcnH+xdB1LZwEaGH0yRues8esqEZSeAGsJzYSkLFQ7OfuYYXI4ZxLHRz9lR4abEp0Ujs4m+NrX3hR5DOFhInixcr01eXBWbKVSulbtK\\\\\\\/dK8IXf8LHh479KLYruAFgoyR564y3a92LCakWWW7TERaV1pWcaTbevWfz\\\\\\\/UoOs4M1zV1XNVd7leWTCSIUQzONgBV9yZzUmvdDtHiS4PcZdbnMk56PzSFZ6NxnzCa6havm\\\\\\\/so6RJM3lnSvr19+q7eIo6yVw5B3kGWQDa4V4lMwmSgSVuSFXMyRprqRgb6Pr1ogtYcGLx7utM3CX0HCqOrOO5gQ031OsngT6avcESBzI2d6SVwT81g6v6oqvaNddH9O4me6PdBFX+EQD4o5pAAKTXlVTkAdTVpxdamV4j7eWDmbvk+JA9ASsQpTGDmKTzJvwOFssBnTI1g+T3\\\\\\\/cdYpkQks1WFsvwqYVXJhAAzNixq9zLBkNqlLp4h\\\\\\\/M3ezQhfmS9peTrgJv3M3hTg0DSvA\\\\\\\/fzJBUc+ovo2FNcxgg9Ky+F5DLfAQcJz\\\\\\\/NT+Yqzm79oBIvoQPjvJ2KpSBCkj+ogfDW5onDQ4xVkGONrYDQjpLjBSS2hKrdl7qWd10\\\\\\\/\\\\\\\/WyQ5VfGyrSYo4cXGZlivD6eOG9QddG1TmEhRAX3ZYaPaE9PoTkFY8tMJMo3xluhudDCRFS1sOz4Y3zFGJ9Je9H484Q1iX0lH\\\\\\\/xwA2fLZWlcpd2W2mO\\\\\\\/OLlRdAUTtbGOOqxJY1e2Uce0CYP4aU2ZWHJHaebRtU8PQOYlQcG1E6GNBsCKNeBUbOwRNb7+iqMCE2ljUk1bn\\\\\\\/F7HSavFyLMruAe3VCx38TtzDh43pdh+shWRgCBURmzvjLeui3ZcG5D7cOXs8FiV6D+M5kY6154lozErnBUM3HpDIcu9hcDmszZmU52ZTpGzNMotPto2vUBvmnW0esS+UPYumwuMhdqmy45Lckmvt82wQZLxWgarRHx+3HV5USojvl2rp+ektyJpJdZchvfkGR5IcY\\\\\\\/cZSCEWVrO7g3bViRERBVVZqxx8tM15AoicCnx9jf\\\\\\\/sSKjNiHi\\\\\\\/2bXPELp2kjTCiXOfKNiythC1zqFNALuRhLIP0re4KDVeO0m4AFNpEL1q6IYvN2sTe+nFE4I++FcT6nnoB44o4Ce4d8+avabE24HUJtsFZQIhJugmr8X3zqf1iNRA2VmCs6zL5mlIIgq8XciPS16rAD6+WprqdDyNW8tl3stJTlet2XBqiVKSDAmRBtiSHOd7HQ4XYu8eWzNlr6DLTxT5Dr6BpML917cUz5fxsXnbsXpJx\\\\\\\/p28npmBUqgdcnY7ct3JoH\\\\\\\/u1dMfaGKiKlKnFO+nQCk06amvlaSunFBiK3AwZv7rr6iHq4ssivYuBcHJ\\\\\\\/MiDoxD8CTNh\\\\\\\/G7n1sKMLhC\\\\\\\/7T8z\\\\\\\/Xaz4qeC\\\\\\\/ISg5H+\\\\\\\/lIdxrVrgKWH1+tOj+xe8cW\\\\\\\/JRGvCukxGfFZOKPGAGRgGVTU1ah7duws\\\\\\\/l03J+aAk7xmrFrAmylr1zI2HHMxd8QjC8X62ZAB32SWr20rc3ufOPaNXjxeHZ2\\\\\\\/xDR7tK7lE4DDfJ2oo2mmKVNFpJVDjD05kBtBI3QEcuZVVDqKpJrQlO7NX0UuNf8Jv12sgCJ1\\\\\\\/JwSnW8pu8XasgSJWeNcsyE9A73cI34r7zc+qRM8coAHOtPw+W5vmNY5chMsWalMAWqUFdWDjD3un43yIinDa1\\\\\\\/itHagg8nG91cW64f\\\\\\\/vcv9LMV\\\\\\\/n39wUE\\\\\\\/VBY0cZx\\\\\\\/QaCk3anSxZgvH15hnev0ODsIH5xfZwcpkATQ6uYlUP\\\\\\\/9gZRtVqFXURdd+dFZI29LVjD5B\\\\\\\/Mlho6amZ6PGIqJF0FAlBeYEFyMDWaCRTc2M+gs7UyybdOUfBhsCGJIj7Y1HTxaZ5WdYwO8x8zyNNf1s+vvAU0YF8rZMpPh2jf5P1uKjSEwmYtKsvN1HZpPRdy1mp4piPs+2Sa2dz\\\\\\\/tCM1BA7GYpHIl4CaecDm8\\\\\\\/ZTxkp1wez8pvcYFJtQ8yS9o1eCVgaQ9RLYw5pcaRsWZazwjtmHHfj3L25nllNHhD0np89W5pEASQqcQHxoUUtldDsy5PYaT90vEFytFx+CfnxGy9iGaO1682s+9LL7msnqFK6\\\\\\\/9SQySNxFzFdEAF\\\\\\\/gWEipDS4O9WBuFDVlwxb88ES7bmUeFGEJ7biVT3k3dz0YVNaeGxR79DWkT9Og1eoCpvHPQPiA1owS7+2slTqgx1gNGXjBzxvfiI7xCXK9NhuawwUaI9tRTmibFkWYSIwZrYS692IZbDTDzvNY+P6DNgoIRKhPYtdFdwbQ1cJKpK7qPQuOAeY7B\\\\\\\/3srdevECx6BECYqb5H1kFAbhP4Z610pRhtzKop0\\\\\\\/O74e4mqdX4T\\\\\\\/tUDb1lrXgAmMFmlKwoHy6oGUOGH5O32HRNOfo2LagZ4HH0ExT48RfVWWgkmWUiVMpeL+VgNjCTzAMNMjZ2tOItXBPxFa8Ls+pqIiW+21A\\\\\\\/qi49LCqRSsl4VU6uPCOg7EYoslKjvIeyaheww\\\\\\\/rNNT3tK1Q2T7kFmCzswDsw5GsbsRUA5jfEWtws+Ba8TgyOvU7dBppQg8Xl2u6aw\\\\\\\/m8pvM8bSgLcKksDH88JV82UCT0QdK\\\\\\\/6bp5KE2N6dFryaQ3UvlAEu\\\\\\\/lDGKB1MRKNqHPWM8S\\\\\\\/b5S\\\\\\\/JyoAts1XAoSsrkaozMUiDcuyHw7Bv0+5ldp0nbDiSkBML5p3kERz5LDegSzbHS7huXO289m5y9f3y\\\\\\\/2c1FnyyX1aT\\\\\\\/UQI9B3CwtmyJWfRxtoL4pdK5UbVlXDNH2Z8W713LjGDwsmTgsb4ManePpfZhUw0CVHEh0DypCVWQ0QIZuaU4RONLFeSQyNvViRMY1L088lJTuKgXqZxQfmu0Li6X92qfwThYA5YCsf4bfI4LpimQE88FXSj15HiUHrxog1UuyboW1uuXeX\\\\\\\/vRQ\\\\\\\/yw6mBMLtc7GoqmTmVw5N3UOJ1xEaCPFEurxM\\\\\\\/OpU29+eyF4b4ZI9V4S6RAC5YOQTFAY32sq+5oVBfGLwKrOBUBfnuIT4TWPnM9jkcVMYf0b6j2tbZnVkaVZ8afZrIHdtGRbaCU8MIQfS99ovQasMPwiEnv6JwSWOJfe87GWTIaEfHliYZOYJFBf0ndd3\\\\\\\/i7vrQXyO8eHpdls7KxIaUdvwIvDe9Q2AiGiKdHIr4HkxaS44rQrgARkJv92u6kL9lh3r38KOR\\\\\\\/NeFSSiFW5P0MyaYxiI5aweQjoTUDP6Gc73eks+eO0qpR28ZvSHgJ60Hewzyyhu3Rv5860AMgY6TdTuyexygGwHd\\\\\\\/gWF\\\\\\\/i9zz0+ds+\\\\\\\/ce\\\\\\\/vHG4vieqKsHGSW1JPeSSqzHJT5ZOafiSdwiFwiR4hH3lZhwv6SS2EJ93JSbRCDyMg4rucuTt7Je6MGPgzsTzml89pcdyAuC09rKrwphfvn5e3HeeIfICz+HOv06HNbyLzvW73MRzqx9JYl\\\\\\\/jydR7dvkxWuqhzSokUGCaNeeUB5GgdcW54Wu6WxAI2kH6HPzwGF3Qjeg2p4e203p+n9SEJAlI8M4HoBjmesjnfG3YpFMJQ\\\\\\\/+5tV5qgO+\\\\\\\/K9FA8Ta8ZegcZQi75K2pT6foI1WgDo++Pwql3DIHtFEuPiOdegy3j6SwhSbHeu6aON3F1sANW3z9L3ruZZdgU\\\\\\\/VYXXeeofanfZ58MSZL1qWKInG3dB8XVnxFf+oLN6M85LEG46mt33USpotEEtdkhYKaReSTuf4G+ZlNdT430iX9vQZR0k3pJAFdVEOgxfxjURjueAdf52ias7TAIc8YOv07fP\\\\\\\/BNUg4fURBbAQR2Sr3KzD\\\\\\\/HvmmNeVM8lbSM9Te4KJeVuAlgkN3t4SKnLLfOMMC5+DX83qZqvn84Lh9401bLAXXNh5ULMGacM9oK\\\\\\\/GzN1heDnJgUSTvi7qCBcT\\\\\\\/In7nj9KyIiGJE6M8prEWaBe3ClAcLFQhC9czKsQZ0q1eKcZsEMjfeR5EZ8JNYPzmNAvsB2u905HOsdUAAJkJZZBDAiwBzHD0WWV\\\\\\\/6qHI9IW0ix2COf6EcguElsb9s3xBCD+\\\\\\\/DVpklX6RN7NgUbhxdWIUKMB2O0n95sGwBlJg+zNQr5azNSdHCoxK6zwkx9rl0GeBPIfkeVBWBZG\\\\\\\/2cWUP+ILxnVkH6LmZq6uSHme5TH\\\\\\\/xyU8mil2ZX8qFJxXHgYb74hgp2FfL9gLgMSjzcy1TEDmpCztKyTDxQRsvSd8Jl2bk5lhVgIFUyWr3nvSk4b8YqzMgVJR1anFzuCrlAMWE\\\\\\\/qkTBdz2Pc10WazUR\\\\\\\/2lywlyJ\\\\\\\/Bt\\\\\\\/MyBlqtavrMna+VGaD1yQjqULTrwf4d0gzzLcXeNW1yBuUtIkBJkbw\\\\\\\/YduzUsh\\\\\\\/NlRAVTGl76ibwO07Zx1wjTUuvbsNV0InlnWBKB1k9eIbgE49sfVr3fDuODtO49TRuW7B0mbxgY2adI\\\\\\\/Es1xYgsX+mJX4oYRY+91BpZAvh\\\\\\\/3jfhUbXdD+LHE\\\\\\\/Qu0zIMv05Lj05qbWSu5Y7Cn5ApinWIDX8xnvsHW2PkuWE97YT7E5DZVTmZSRRwVJANRJJta8LOR854X+HGRVHkExf1QXAKXBto1dTzS\\\\\\\/vlvDJLdvbmaTs63muzvB\\\\\\\/pqkVgPBJdpqWoluaDHilThK9LJjyyGvekouHqdyMGceYl4\\\\\\\/CJlavcVdXWGfDYSB7hDLi6P2XyTL3x26HDDLxysWLnjkYWnDzYSmsxxGbIYw\\\\\\\/A0vdIxu6kHeaycoD\\\\\\\/cYF5Aj5ZTVEq2BExF4Om8ygX8NoYIJjK0Nu8tsKNhf32tib9WYJqquNA9mBP9wtWJ3t2vq1JXhs8d16p3HUwAOMQFGi0R0djI9MdQ\\\\\\\/b86epN1NavwQEoY12tq15KS5TKOrz2wwWfo7n5EhSwPBWaKZ9S9TYNeR2wpMTEOqUB5uKv+PKXrC5\\\\\\\/kB3uit3uBpvf+LVBidmAwIZh9i8YwnTTNLyLtId+e91oJZ36+zqwx9L0kqx21oYkI+e5ASg3u1ArIkX4REAQQbDJep+bNI9o5YNsTesGpr\\\\\\\/gTVX3thMIoiYLC48uc9smv+Zi1Ijs\\\\\\\/YNMRz554rjzrkY3DugLk6Zk9NxL6wca196ALFQTAL8MhKT4cdFYIQh\\\\\\\/HyJXb5UW0GdrfTj4yjgvpas4asbUF27ETsldWZcGYLC5k7WYOnDE5Zoim\\\\\\\/m3gFbz9T+gfktxHReXa2XAznCXSSwojLDWI28vcLdvuJitVfFJ+nuwtfPGcbEUkJ4bLDxtvbn77SSpM1ScmCaD6FLtdYYBCDZqZs\\\\\\\/86aBlgVPWPVRQtvIVcgEuoU\\\\\\\/M31SXF1E2urj\\\\\\\/mH\\\\\\\/a8UqKdHwz6xCa7m2t\\\\\\\/gOY81wOaGwpVpGQjdnzjP9CXkRinfD9CZs2a3mo+\\\\\\\/wsxTtx4On6lYI8jy8d83sGBVTEZFobPDoF1OY4NamupLRHbjcd12J2xa9YibF3F9sWeYK15vq6MTozyQ6VlSDbKnjII6dSqt7lZJO9vXtMePCojKCSKna6w9Aj+e6zVmSvAS0nr7LPqSrOHvQ80o023jyw2JW3DXHkfUxirfe+\\\\\\\/NpOzF8ddLt+jntAh3QpiLtXCrT4d+ps7\\\\\\\/cuyPEKNaylb8U7Y1E7Go6buSHgLAUZtVXEu4SqUAARLeVZ7A17ViPdkqnMWFt0PWfZqJwcTyjw0m5qdLuAAf5QNR\\\\\\\/0oXZmOrhMTQJhMFwbvZT+ADNjODqf6xIPCzIT1SShQH1XKe53U8bIr1F1r3QwMQSlseNtPExoizGClEr9yU1r+xwrO75D3hb88LSui7lh1F217zSijQPvXLrnMNRbM9Gddtax4YEvPaAeIKbuWkESbFTV0PqbVt+mBVCZBsK40IGibymYAyOgA4b7qV0mXoosFreFjIxCKEB6Yny1BYiyR6r8egk\\\\\\\/7D1bEdUDhECCCw9pEzgWhPIud0sQQG3qtJaj\\\\\\\/IPbmAuINMqkiWPV2KExatcAY4Fue+JcK+XyfZVEYMDrQwJHNPswr\\\\\\\/AeJuKy12HfGsoRsNzWBVruVBnch\\\\\\\/EF9QPGjeK3ja7op6lEmrVv4KMorzcAxMjApMctzrtMyHSLKm+ei+JRWdOHToQLlbd6V7sIu1j4asVYmzqzXaZ4kMoM1oVdDbtXv5ZNzPt6jRHXM\\\\\\\/Ce3QtDdIyMxBUHiSf1WWc6y0KqCqFjCP52ZtFMpBbuwlCqxCCGiOvy6bikZnL0l\\\\\\\/sJCiIPxlhsXWq+C+Wy1kW5kGlA\\\\\\\/QJUwUOwz+TX1PiLsPO6Yfyrubyciick4Fv7ijHWoqZK0BnXuaiDjK7jGHqfFbribYwA4wZHUeVWrQkyaMrljtB9CCjpjpK32uUuwRbIiXVA4rg0b0Sv04X3z1tOGsH5gNse\\\\\\\/\\\\\\\/zF69rSOf0CI3DzErt+ctfQqKUELTLixDAF1\\\\\\\/EWF3uZuK5Dxma1GHz56Fb7RIAW8l\\\\\\\/EhekRumINptLVpgfzg2aYkCJBvjR+Mq5r8QFNGWxcS29Khhds3jwzG16RQiWFVOOkO14wG6HLbqeUsQHDRFa4kVHIxCwFlhFcQR8ozFzPA1CWluTn4bhHOIU7zVxpGZUkA5lLg2dRA8abdWZs7u31E2KBs+f36A7HagjYIcF2YghvVpujIXbWddEAwW\\\\\\\/hmv8463Hiw9U7wZJRHaf59932AtQ\\\\\\\/mxo+7gGLhNxSQ1JJhHLJhJkpsmuaawrgXXvHYjdvV7U3os9UXb8W66ZTqlU2lL4S1ASlI+VXBDCwRhg4Xr56pHeH432NlFVFTaBE+iLaUBQdTulFAox9wnMHMbGWGrGz1IEbkmrHlJ5bJMSInkqUPX04Y4sDl9peOudrv40tdtY3EwyWtB\\\\\\\/YH09p+u1wGKGZHe7stY6a+eIvFtpB4QEyMuhBRU2Q4QNT78ELTxggJK5p\\\\\\\/iw2dGq\\\\\\\/aM+daCjxSdBSUYy+gMCLtW7Nxe5k12dcVWFmug\\\\\\\/cHB2FTfqDsz4ZHHBGOKgJrNcVpW5gESknDtBOQN\\\\\\\/bkwAR8mWMzf8sjFjX+msb7oHhRq0KGDDw70ROQ7j18mFcAV3I6Rx1S7ozUQBMAeKDMJ9NmfIO\\\\\\\/l8Vq44k1YBSjLKFQ4Cc6wWgH7qJGODhQxQonUCCC7znM9DMY0axgb85dI0N2XHtwjrhXzWB71Xm3DPSNr+3GSiUcAsiWBYx8\\\\\\\/nZ3NcWUVgYzImY49T+Ib6\\\\\\\/nXQtz0RsfNnp90o25hNt1v3uZAiyZ1O3Z9ft8C7VDz74bpXbCJEOl+p8SpLN6zp6Hvp08J+gdWjByZx\\\\\\\/qzNPMcZtfutXIvPj4GPa2aGhpqwO00LpijIeUii3FV+13tMIcwgDCUY0NbyRSBGlEmHBQ+yPsGKoa7HpSZI2i3Pn6CHxdcCARmV1Wl8G+OBfSdArVllFHeO2bJ+fdp3SyQBhStfxMzoOc1mKq2A4TI5Jqgco50MXB3i9sLqogI8lt8tTmbracJM2lLq\\\\\\\/q39kbjOFetxi1bNG9bnHtCmnGozJqoBSJ6BY9IjHELa3Vn8ReJbnsicPFKGKtuaVCD3vRTQMWP9onss34m7TOtEde\\\\\\\/ukBPzZ1vwgnq+H4rIBgXQP3BMRuJcP2Z3j9Fg3Ws+KJrKhhM2zMiVcGd6XHTvlLh2T0h9zYxTrnhK+Wmo1unlQgTFSZzTsSrr4Ugb3ch4BCmFH98Nxga9fOB4Xm92AQz4Z0bR+kDgGF28ZQDMgHmHYkLSTcI6vK\\\\\\\/MUR05+RkVpq6e6RBgNoPFdafz27VGEbLCRwwAAK1fN\\\\\\\/CpEJKftFHGfV1H63ZzlIIetRBLODIkcuvaH+gwvqYaM+oAmy01\\\\\\\/Dr5QvixbPFVmQM1+vHSgfH6rcPwmS98hxO1ynYj1JWmUqDOXM9+6y4EJbhNgykE61WxXNlia1oEL4ghVRkLYXYOHc3E5mXmO\\\\\\\/KjiEbFxZTpcCIvKWNA22ohlKNUR4JsCnL04KlLgXqxiA+meg2WaYbz72p+UYRo5F03qLSB4OVxV4Zx9P0tWzPpFCkj+9ClW0qKyhHbYIZG9BHmqmbIBha6yqVKvkWZ8\\\\\\\/70sDvxLvZNZH+o5IqIIZjK2m1dIn8a65WXy2qg2wd4sVJ2nLXWjvFyrzljMpemwNGCyWX6K+EyD5PS44YHcp+xoIo0SV1jRIChJtHOGeF1gM6IjWkoxtP0hw58czswafg1Azfh4z9kMS9AbB6LmU6g6q6cu\\\\\\\/O2+VWW3+NkiuL9oGQi4Smde+ANS39BdMtC0OtmmEfM3x9x5\\\\\\\/yNuhH7ziGEQbclZHRsIhUb4ZrDacZxpv0ml0rZmR1Uk2MISKr7PP1irAIEIZh+KRibFQ\\\\\\\/InYhwd7c71sK2fJfiNTUUR8f+jmJYYS5BeFNd3rYv43eNu63m64RN6c3bqYmJcjlrko3R2KwhiiWJHoiwLAMJQ1s7nup3OKia2SIw3nGqyfnFOJsFndXUHH8pA4e2QZg\\\\\\\/\\\\\\\/c3chjUpI3cqIu5j9sgqf8aCKf015Mp4+p\\\\\\\/mkSOGgcLTPQ3L4iYkNQxSrKvHfdP27bhJP0a1Ltq3\\\\\\\/4QAPOvbjyQiJbeJb55WJy5YxBza17uH1K7uozrDIPyKSvUeKaefmvLmjiw\\\\\\\/ELIV+dxnrRlATfvfkk15Bf6I3wV+uGu4dDD7bPEJBsXe\\\\\\\/uAqSTeEiZV345hfJMMUA+SNv7TzbF\\\\\\\/GbSSDUnDHqvpI\\\\\\\/wASm0tQSqkYuffdV4ZX0\\\\\\\/Zz2aZkKgdKI8DGhzoTT555ENt+X\\\\\\\/PhBs5USmCqwEj0YboXw1EF18cqHVqT+Lw1uPfc6t0yzUH+Ky+uDF9KXAYSRvV37cdK56A6aN\\\\\\\/aX8mTqQTZ3WbaWn2MTsZs8eKwQl3mVObWuHvjj1OC8BvlwqAuycN5XxC18Ath3evzk+2T99jo4zfZ+2sEjpya1yeyPuHCLbx\\\\\\\/v0z7AcWpf6ZeUZ2xbgylDv+9FsEyhUMz204VrQTOF\\\\\\\/+N9I3a4LViGHs6cFsHLEoq44DLpj0ND4JU1r\\\\\\\/MA\\\\\\\/Nslvyi5APXWaO7ezAj7HUaQipAdEyQyET0fPZgDE0JCAES0oputJRam9e3ph6m+1V4pMnJhJUZmzZqxdRLqPdurbFwrb+Cbpfj\\\\\\\/atTNQ3pSj\\\\\\\/XbxG2Vb4nt6psJc+nv6JOapNA7YiSDIrApr7TcgzDtqT09IOIkKK+jHVQXgXA3x6869ADJN3Jq2Td7S6sTqM\\\\\\\/vCqgerBOuLseh2RIm7BUwus0U1WuVUojHkfU3\\\\\\\/sHhK14i7uld9KqE0InqLeeOaGuUAMMTgkXZNEiZwLroLqpJMDOSDPMZXNaiOAmDGEtFufyA1ctB3Jnfenp3tk2OlScwK\\\\\\\/I9HZTgNIT1gYcELKqDnI4o+sB3mB\\\\\\\/21+WxOc6LN9iPBCuKwdX7YCcIZvgDuBnrT+YTu3b0HCu04IkcxvKwb2IxoMkqaMcVnhNI3Zidh6YYa1CmeC7Xdx\\\\\\\/vx3E7sTQP2NmS\\\\\\\/qYkDYzk6FeRtSeIO1ZpF2n4ngBHuu+rF1ShUrk0PBVCeiXuYU5sqwEaNZZIHit8Lvn91XVeek9CIdWlJORXBF1sQAKST9BUNGGYr3stBiGWaYSjUsPoO8QK8ZLqOkpYGmNs3i\\\\\\\/b6fyffmDuGkvjcSoqQJoTXdZALNrj6eS6Q93Z8HS2mW2ebnB1Z4hllGSdmW+R9BQ+l23GFaPa0SUwlRUxUS7y5NhYlGOM1DF8infLiwlfpWER+HSik9nQUh3ehFv\\\\\\\/5bFImIOgdmXgMaIiNig0LSCJTDWZC3Te+\\\\\\\/p9\\\\\\\/OBJW3nOzjgeiBrv2WylolIOGabisoHer2Sf7av9iSvqldf0LTQ1OBNRiZ6VpjlZaknYvG1Tf7LpMM5dtaBqZDgxbjxRpZzcrdVyFQeo3dEK7diHIbxW3JAEdr6Tm1tUHElNc46gePk9UxaRh+n3tZyRowsA1L6DoOqZjgSNiZwrIbnLndf7+70StLRlaWZEPtrRQTLG7LP1Dg\\\\\\\/t906Sf+PQBeolS2AeztoMbReKTUiYhpMeaUMFAOo43AXN58od3PZrXRUXBJGqKekYhpKdz+z0sVws2NmYeU7y1IBRdlw0DCJVKiq14DXZWB4yCXeBzMriJrxYor7eSFwLqDkHvHw6z9qOZjTvLKwKMX6swBSM1DRvc41IypXbwk6RbRo3XQrDh4c1MWfPakl1SHnzWbkvsxX07giBhC9p0uQ0pw8VVnLLRKkRBNLpkoMmj9mv++FhzYQxVrWCj4yP3HEgsEi4xPtWJY8IlrUNjiLcbBBE5iYdM9RidPg+AWaAt\\\\\\\/xvbjni7It8Ybn7ApgR8nNZlwy+8a5UPeoffW\\\\\\\/EoXlyFMoZSq1I9HXCGwETiJsPPW5Q0cSpOPxK7yyPCx2RJ+4viT\\\\\\\/R1Q79Va8q4hvWdwyv223tFHiMzHzRATvoWyLOYOvwJqB1D8Ed9XQmO+CS1MpeP7W9bUw0Jznbr89WkcSYqtvAzX1e5daOxdYS5rcgU++9EitJLeAz76z5\\\\\\\/zQuRGZ\\\\\\\/GQwMnAFmDbkqzNNC9dDlUbDD4RrR6Zn\\\\\\\/sEUQXAKkydmsCFB2+x0ovVabDMildNeQJ8r2u3cnpjmfbX0oBwbZZ8PXZ\\\\\\\/RRg9z7BRI+Ho\\\\\\\/WnmFhLKvd0pHYwtNQBcl1DjCFMXtGDTQ721xcHF0Zfo7Ix0T6zFqSoqbfEqWQS7NCiGBNzD30\\\\\\\/D2GldW6mO7Wip2fPPNrSgSv4H5cfY4IEdb35\\\\\\\/v\\\\\\\/XBfOgbERargk3jpoXum9V1o1Sf\\\\\\\/VfXTPVm1mxHfdR2oCAxwj2IYEXdZ+o69vDWVzqwAEt9QHk1+lZwLavwushbsZCkidYoCX03+tmL0SkrIbnHZrOPxZJ8rlRJZ7iEy8RTxVCH8Btr0MaMOM\\\\\\\/WYKLUBmbcnOLPvB4N\\\\\\\/hLbV1VZlh3YE0IX4hO3Knw4jhkVOrsgTto4sry\\\\\\\/dU6cd2Alq3tCaGyVKVn6\\\\\\\/rGvsVt0\\\\\\\/Reygs0wjQ+5wbLnVxF2BPUkbGjWVijz1rafeIB1mzidPnlryLeem\\\\\\\/nrmLWWf98N71X2PzpDakMmuEsE+MyRjZ\\\\\\\/3g3rXhN8qKUNcw2sC3wYzCSeeIsWflr56xFPkcSo\\\\\\\/D1myn5cziPAWHhh2BoMY10rATyqcUBf6IX6yLC7zSJG00MW0JHCs\\\\\\\/eL9ygJdfysdimLKJ\\\\\\\/SKw8KvM7zF7d5LFBfP76Qs6E045QoYmfHuYaibVr8IbUhNZV20WauiNUc4olBQNDCwsti2nqP6sqPOKaxIZYnOo+zhU33wTTbo++lIKsMp30h0h3xDqjYkOCdR\\\\\\\/VoCEvhnsIQzTSuXGMpFVa1kXuDMok36PVyKTrKScPCNK3gnKmp4cAT1agEhdRODdF+Y1xDoK5fy5bqEdTx9AfkIOd4bvPKjYr\\\\\\\/owYztB3A85DvLmYcVFloxZeBhelFvRbWsqt55RYujCMQwcEhIvm46YbpKRtZdHc989Hejbc4tcyU05tHeFrXRvdS7zKBA+WBzW3HFABVUDwG9ORwPGTCR8AKbtti5MQXGPm0AI14UkllnL8Jtjhat0SzZ49EU5+J2ULsaTl4gsimAUeT3sgsZ66g5N+xoDxDEzCJT2Na3jlML28C3nMM1PsduVf8pH3IVwugyowhKJ0SkqOyLfPj9owONFrnCmmUl9UP9CtP3DISMmkxjU+yptvJ\\\\\\\/UshW16P9COqg0MnFpT9+QV1mdR+t+3XCLR88DCxKC7CeS89Tb3E0s7Jz3tceIVC\\\\\\\/LE71Tz5HAtpbSt5KV+ApL72Ndd3Atuukr6LPI4rFKFmtAnfIwfJ9olXhAba4FAHZJfcBLWj\\\\\\\/FLRmAGWmPu96bmt7LAGSedSJIkuOfzojN+xWuB5mcKkWot8L1BJ90RBfCWspS7\\\\\\\/6KygKMebe2MJFV2dNlIA6wiyLDTYqkCtD0m9jBzL9YPpIGWIBpc8Nh70Z3AX\\\\\\\/Rxyvi4Z2DoNEsEnAfdR\\\\\\\/ODLd+W1f+p3ttsfSzy+x+ITnQkTgY9uHIK6njKV74j\\\\\\\/yuZQReJycH04N4HXVhuyMwkpV2tKGNDw5Ix5CGxkJrV+nm6F6vestbQMPgt23vDUDKYeqiMUTERhM8Shwyt3ibbA1oy7QQdtgBsDICGkQGrkmgok8mg4fF3ZoOwdDgWFAL4W5yD8jiGIm5LgYMdpuGAzLOpbeiZhoKFFLX5YFrF29CixasgNLfLi43R73g1Drf3U8TObBCyRB22a0yf+7PqwXBis5mnQYW7xP0ibEc0rarsQ0PCQTHSUF+QZiJb7jiR\\\\\\\/MY9zznwuBWTNr2Ymw4cxV1FGXbWsNg\\\\\\\/NZl3JZbjNJLMbF22mX4TbU3Phmd+1JI5Iy+NV9PchiXhVQ4lBB2mga9Ey+vHOPFVVOb\\\\\\\/0wamaaKA4asgeD8d3NREu+5Mlv7fPVcnfSuvKuQA\\\\\\\/qjx48nqt1pP7Udp00lW9X2qV4PYIRfHOdX3UqUGIQATb2uYZHPob1LxGhsCocMon1C0qn+IVRa3UqJOlBqtP3fffs9tIgDx5zxVHhSjz+uZMVABW\\\\\\\/pGyGfaVBW49xxk4KHv3OAbhg0mIN3YT9wLRp6yxsDv4EeGuju+5H+lyIZ69Bd5Kf\\\\\\\/nrLS\\\\\\\/FrwBQEA\\\\\\\/KLdzpkmejg80xeOUcuD\\\\\\\/0pRXC+GcxlhhuRf2FqZjZ6HC1B2gKkKj\\\\\\\/07n+6ews2MOYcwnfZS\\\\\\\/lZfFhJOl0P8fbKCN4Lr0YNiNS25paUDETWF+HNM9VZdcB+3+RGQP1l6RSK5N2e\\\\\\\/TfSiHURHykCqMQARL6IGCvz3zvRhMOUROdoyJHDxY0uTMGxgqlW1\\\\\\\/3RAocrEn7oqdJ\\\\\\\/QRMZcvbqiwVBQCxVq0ZyogGz9obTv5dOEX9rXzDwjJz3g\\\\\\\/nxx60yE7es2kS+iFtrsiwanxmQzDx+ZOv6Ytw2Dq21g\\\\\\\/kjMx6iBPn1ZeGwkyoHCRcygZv7ZkybB9MBUpmclo8LXSzIeeynPaNRN1cBw8cAh5XU2KVTmSCRBkE9rQaYEIGSkXkQvIHd0V4eRIlILz7c88rTPmpHVvUGjavi4IBrJqEghTFD1SGLhd\\\\\\\/uTqQluMauQw7GU9KIFWsa+yJ6pv+eP0hQHOQY+M4zKdbaUxnVYKkB3e43xTjmOozoW2EiHUz7J93uAdUmxH0pNi7uOQKI5Jm\\\\\\\/roF7IKlxEHZQf8AJCNf8sjN2xOSQL88S3tLpJFvt3yjF5BPJDz5nU02Rio5UXfjPv4AgPO9mCXqjUhHO44xuaqX5cgV3YDrY\\\\\\\/Rf7yq2tcZo1S\\\\\\\/\\\\\\\/pm0tV5xBHmwLB19XlXHbf5zCDXZesD3MBMlzFL+nBU4yy6uf9kyTwZSoYdcHlurcn4FWUOeHqKBrzIMp8xRTcfJL9lR296yfQtohcuPKEB9wQW1YpQWfTZ79Nm4zoy6bKrCnVRNyoNsiKWHIQlsdlrSYU7oGVyjV4kbhZUzB45OIzw893zmdGQQJ8Kuw0RDhqWxQ470mjv23yWBM8APOjuOJzlGMxi0fdE7ZUF1hKbkoQXTo6+fbn3xcIB7o4br2RbyuqEBVnXdY36via9Mxr5hDjO6zIhbJ+D2M49VJbvav44naqTPvbmZE6Uvx\\\\\\\/AUwgotTm\\\\\\\/oGk1oyAoADs7cv1vJPTLx7TORl4l85xN\\\\\\\/aopjYn8sJ9Z9UYua+0oIDDABnWWG+UZyEX4\\\\\\\/+NE\\\\\\\/9g4D79H\\\\\\\/xn3hNedaMJTgGlQ3qRHyW3xQrYVUG9bRsHjOOQzb1TqMztDphTmG5bkdQSVQDn+bWUfAc2F2PmpTWwlRIqUBy2sWEEeiUQJfN8Ash7yFl+oSuUVF2SkoLK7\\\\\\\/60tWo1i5f2pcr2L4tfG1QXPZJE2jsFUuko2LRbsXo3d5t5sU+WXLsku\\\\\\\/QDdEbNa40D6GYpa6UbWGQOUsXJLr9VMiaVvLXlh8AZYvnGYokkerwuctXL2b0aOUQ9O2cLps1399ci7ngKjv1Gcgr5Yl1qdZQLcIAHiMDyoza2ttPJjuchZa8KZIwWJRcZNiWRFF9Tq0jqmPZ7sAIvbZJvK974C3QMDpiiB5zBr7K5kFsCrxsKXpSutgBDv0RxNFKpMkHLBOs7ecOsu\\\\\\\/1FJCN\\\\\\\/K\\\\\\\/1avg4+VFo+Wo9b9KKbPTBd8XxP6phVmhmUcAvz\\\\\\\/SFRyQrzbSZEk89VUvOLwqtfAsf23Eb38bM4ftS\\\\\\\/wRxhTWeZn7oADjfdHv9HNboWv6qIpulMA0pOvhMz4S19U3I\\\\\\\/wmrCSdIW7LLDjU5NL5PIphSuJZCw7\\\\\\\/jv9OLg+BgIK0s79YKkasiftTz1STiRw8RtLxAQ85LvVqX5QEe6frSOzqmX2u7SjBlh6mgBaBJ6VpdtalnMp08GghmsC7vbTN\\\\\\\/VX2KYxTtdaLAuPyM7QWrkLyNOyLtOHXYaEsGjPMYgzzitHed+JQlI\\\\\\\/m3wvyZv2sxiI7038PSimwQ+1yJC6Jh9hy1L37ZaSnhrlsgvkH7kM71iDsQ4Ef40JyC90JU6CT87FCH8HDITUs7+EOYTGvxmoP0BYVqggRSuaSxeVSarOeD25YjXN3vgV8vn2CqhqgoZ9oB5wa7Ru1OifzcH117Psry6ysHE5zXdHdgTzF3PSgHw3ej9W49xUP3Ksi\\\\\\\/1PR5XPBYjYF6Tm9HOIQXsAGWyMeSSi1fH9vGGMoX+mAVdu5bzpIJvfSnBpOy2C2JNJahW9NusoQ9TBNlwK+ULZQvcqrMe4v1tTC7wdOy9YhC+8wRaHwxSSYaVU0eBcidbNB+P1qoaikUmmN4vWyWHR4Psp4pnTtBOlZzfcY1hRVY+sXrw4ibnSMhQC+nPICnqzlOUKFHY42nhsOuugKLXmZph2eSTyWn7HDL02Oq+vL6R4lkqNk+gwOVqRbl+s\\\\\\\/nPcTMUV9OS5f8OHvCdb9ASSYkcpJIFsBZLiGGP62uQpinOFtd2RVvxbni8rTBgXT5EaVYZtgFarQNF4dO2XdpuXteu9BPMW9iOO5QggxgefRi67\\\\\\\/Dy9dUSsHyAlua+ciF0weGxngmIxrA2Fbx1YigVjMxrx+GahBhT5shB9gIPblytHFfvIwCeyOnfQa8Zq7SlPapwMtkRtBV0SF0BEr+ve8PgQwbWwjlInbG2c\\\\\\\/jR+EJNhWcBj9yjboH5FN4yL4xDlqCHh1VhYRIcMULM8K0QZh3iwN4\\\\\\\/19Ns6QDL3XIZtQTBsMPMNtlCWfy\\\\\\\/eeB8bFcq2lOaZxzq9KTvt++C6VogogsyYhsPTN9+cEKfDxt3DpkykjiT+wtpHvSM7vCnu4fyKIjh2DOalwhL6d7fcbn23JMP3dXiqLPueVQaJySbsM5\\\\\\\/p\\\\\\\/Gqf40KzifRW1Vdali3M7LhR4Sm6on2oqQ6h4yFzDEobasfnTneZjRyzvlroswj6URU18Do2h+Dv59CuWyO5AhDhiIvhWU\\\\\\\/JE4ZzKop82Yp1KpnI1ZJ7N0TD5NGn5OjylGm12iwgcIpELM4nw9MVMEDPBmbsRU69rsqMxTOp1zqYpTEi8cGCYpEZladAPcaeduYXSR1uaiVWDnXYhe3XBcbjG\\\\\\\/vZUc9eUA4fvoxFKtykwjQbu5cwRXO2BHWpZVD2BDfq4LktGthublCfgQHTo07asznF2Ztag\\\\\\\/2bFGvRj+WWActDwjBWuoBcZzL4IrolvMgLPjQdLhx8gIvS60S8Km71q5SOjr+UCKCJzRBd4phieGnMmVcBbvzdFlapBqYGj9+eRk41FeB6Qb7cVM06QYZCYRGi+CdTuhBOMqkIA1cghhgu6UIirTBBx+6EGLV1rS+laHwxGMuMX43HBaYxu6wad9bUDVFN5lL64ExP9vmcOQrXwcIxXfghR0zzfPIrCb0qVorGf3yoSkqjCrxPUE1p0SX8369HhoSouXm7mH4FGqHyXP2mxWgL0IYH2jo0bGGxidgfqQ4dWCv1yJ9TNMLF9fXEkFqrSzDT\\\\\\\/PjxR4Vc12nqfdSaMCIGrjdcrQ\\\\\\\/JPeX\\\\\\\/YfqwQv18FKjyKcwakeXSqEXho8JKHi45ZbcGZYm\\\\\\\/+kyOGFUf3H37jRFWj9IN4UwdVufsLnCz58EgEw7gBMy3qfEB\\\\\\\/bqwi868gh\\\\\\\/itt6nCmf61hJa3+WbH\\\\\\\/CnO2p0aiGyz7lSwJIigN8tXqRJ3lotFa7cKk4NvhxecsgMfi3+o5zK1wimYbqn+0kuE1MRrkrp3vim238VX2yTXqWuHD1t3x3qDJVdry1TJ2qqL2gtdHCmidnySp7Ujl9jZ5uSuKiX6yyxez7wvOgUmcgnan\\\\\\\/tN\\\\\\\/ZqSXXvCDiYQdiaVrXtILwfU6pptzVG1zdkAxn3+OMKKYRgQkp7vRa82LzkExAIUfoMggGm7JEwpET4sZODTzqGgDN0NE7Av6j1s8M2Ums653aS9H9Ujqi8ejoN7ew\\\\\\\/hy0zANWrgtqMrdcy1YwiGtNJF1T7+v3yuKL0yncFM9n3XAtFPa\\\\\\\/XwyeYrY1jHWYFCyU0CYnbvRoEjm\\\\\\\/UzLyFzlUjgx12i7unJqnfGWMllddlUb3ol65bljZ6MNnEE\\\\\\\/JGW5J9M3ANJGuAoKWizqdKARXIvYm6VSuJo95Fu\\\\\\\/fMrCuJVwFAuLeUIgOUjcgorB8Pk1e8+hjOxSCcRnm4NgFoZM1IQ+KL7R8a8yiyQoXLXAELgM7P6mvaEftrNyt966DAG1x3JqMq3\\\\\\\/ek1sElgsvVsinjwTjMY72+19tVzZinYnysFS0vulgWWW1946NpVA90sAezzLJUtTxHJdmNMSk6oelU69qQZXlt9apfNC06qKzwISYEL8YW5o2FibitqcUKkgA0EYoJwkKXCpdqrIyZHCPeRJ\\\\\\\/puxFuptp\\\\\\\/Hh4AAKcensPzWksBGd6N6w\\\\\\\/OdZkTZREnXnXhiUnLIYDeM9xRX4Ks9+k73pkReQjFHDVceMD4wcixAHGE6jVsZ6zrvrYIl531aH3Z4QUw0pcsx3mCbnOdb4I3kJY1oY\\\\\\\/OXr35dRtLDfqkSWqIyQSXXcAYPBD4oXjhCZFhjdZRSaemDoRE48x2fc9+qlgStZmAPheGt7wPxnOgdFIX40CjykPlwJBZz2Q\\\\\\\/\\\\\\\/fOhM82zO1BYi9MJ2Sk7Rcl5LVLlPmEIcKiPKwYHcop+BiC0i\\\\\\\/BcCoZdmx\\\\\\\/4rVe2RdXX9vdtyXxYPFfI3ALK0aRzkgyRn3tgbg611ExViOnfga2LJy+wQHMcdQhErZda6lM7d3sHIr2Hp2yb4\\\\\\\/5JPzkv3HBNaLB0qovD60baGWBZYwDVzVcGA6WAEilYnGCyLqrsHbRSTIm8c1in0zvLGUo69eLUNJFo5NixZCQt\\\\\\\/vvc3IQVNFoke4syDRtn85OwN+ovApkZEhJYTslVCxaPpBOESeCX+320y+A72uERA3EAD4L0AeYo48r1CDBDM7Jkvt2PBPrA\\\\\\\/lQy8nAdQzMsFbaonaFhyHUQD\\\\\\\/PHAootngedG0603UFFKJe31GVhTmWWmccJYHAMcqDhrRDOMtfd3TbyKhcgS9TQbmI4MQVLMnfEEXXJEickuXx0z\\\\\\\/db2EgTGGF\\\\\\\/kNuKeSvpe1YaqpSNk6+Ql5idftXxa\\\\\\\/HKBQj1fLjclsYiC1ZSoPbynqnmSNAqqDJOg4YOilYPi36UV0WhPadIAuBbiuymfFvGdry+EmACp\\\\\\\/0vstPyGdg4pyrdYD+7jndcfp17SAbu0lzxD89l7ctrK2TePrLkfn2nwDtoVVfKQM9jrorTYE7WZS7R3D0wr3sw35z\\\\\\\/xXbZx6Idpc2qJ9MBYDz\\\\\\\/vN+3HdY90U4vTV4yR064i3Dnecenk7yYZKWSDHe\\\\\\\/j+7nymdBcD\\\\\\\/0uEb93jV8tuLoJHU9to7shISOtyhFY2T1DlcaIJatA9E55VCf7e5BCZBRnUU4UFUjiPuzZHnKSLBsE35cOyJAtcy3eoeq6bYZE5EESTnXwwV25vwsGFgKptgxze4cC9Xjm2zp5eHh3jeshbuLAgXfA2pClNBFdkExaAf4z9+RdkwexdWUu5ACnGb8Ir32n2P2gwhxzddL92Mg0odVwUF6wH7bYOlXuBP64m7lUPD4nLQh5qwlvmJFB6ExTQUMWdbI98XMYN69xKV3waCtUR37mLiCCQsAoAr\\\\\\\/hk6\\\\\\\/1BuHhRHn7B\\\\\\\/2deLWpXyo7eN\\\\\\\/v5qdKrBJ0ze\\\\\\\/erb5ZWA5sBZ\\\\\\\/5GWdu95y0oSbYZVJt0vGMSgVP71TheX4cgYtj+ppI2vJv2bM4df0c7nFQ6ITOW1bb10ol3LVx56LDVQAGYXRhMPiUSzxuLAO94vlYDA6LmvZqzz4Wgq4wrl9C0YmNqgEbWds8gNNpZFv3XMd7\\\\\\\/Z98SQhovgKwGuVBypQ3jGh\\\\\\\/\\\\\\\/le+siifUvy0btwv4LsyAnxmTaFtXXL35ePJS3Sbha3Aso7MI60UiPpeBUvSTqY\\\\\\\/6VC9Oodcp+Ir\\\\\\\/ujgIhFJscGMP6hXR3W1M3frgdzP2EhFfDpxqEEWQAHnw\\\\\\\/3DK9vQrsKy9x5BdMLnWBE5OziJ+X9QNzkROmd6IvkOkY1BOOVZvdLfZF2rvsL\\\\\\\/hcWooxbEiGoheBoQdxTQyEmU3MGmzd8sGhsHPZBoSzk4cKbbVZy0wRvmHJWwYSnM3j+lK\\\\\\\/bKbWcJNUi1+YBIIo6rdPdrfjD51gSw1Ao+bG\\\\\\\/qTWUrJ417kDorRda7XntZP9mTogn2kwCrE6HQAavmZRiljiZ1did4Ex027es5c3EgzFxtv0uTq90vrLSM87kh8n3M3eS5g40vS7Vab611I9WltlgSvZeRkmkZQa0rQp7rBB2ymso3e6iLGwrBq2S6lOc4hLJonhby4FF5mewPc+3N41ollRm5xKWaZgKkZSdL0E0OKHYVpnKFBqk7ZknDaBX3SLhTu5V8M6+AiFo8jB6V3ca+6xvdr2Opt2pHZFDL4AVLmxWzFloabzVBOjySaS3xU3KDWBo\\\\\\\/z6UKQc2p5qyPjx1Toe74IHVzfnZ8QRajNDBmerg5hPRKWMpG\\\\\\\/Na7phT0P\\\\\\\/4FnUWER2OAh9d\\\\\\\/TYqBAoOvCFNBzAR06MObs9nT05oXMhnoHqZ2ZeG9PZsHbfRKYglGswdOMT7kKz60OFBStE3D6taMI30wwiXHnWtQPYIIR8ORgT4qEPiVmXjUN7GSnOz4B0Ny+k80z3bt7XhpZBH\\\\\\\/+dnBgH9urmCTfHd69dTFoBS80uxMa9qfvdzodX\\\\\\\/TfDoUWwVBsj35QaVSxPuv1itZfjQrBJM+eh9ZDBiB0JWUL0+JMyR4yRJDWx8kFQ2\\\\\\\/Iz+LFoiu1Dvyg+QNIw25UnzwcQ7YNLRIrPyCWh5f28rnUrmaW7znC7nnXrDHSnZkb7Yt+dbNKXy4erd0Gp9u9ly6w+W0JtiOTzANVn\\\\\\\/7\\\\\\\/YZHhVOtwd28WC92Az3LYvMeSWwQ35iHPUKYdpF31\\\\\\\/dQV0RbUOvswgDqu5V0aPGed\\\\\\\/D81pJLODdfNaAdwJ4DOoudbJZSTdz4j2zuy1SzKSyfKw\\\\\\\/BfRrFxuVZt\\\\\\\/T2mewOwiUoPVE7uoOM5tOtGGdFaaQmAEMYX67hWmaIRmIC2eQIbuyjlZAq3dBG4vNU8klKbpj4yRvoxWh4SYo\\\\\\\/eOmKL6GiCoKBm8Z2xax3NlNyCAUhHoy59C9GB6fu3GH\\\\\\\/cOYZLIVrjphpft+ePMRcrljxR\\\\\\\/rr37DB47VEYSl6DHFfy0noOj0WoGzN4PKQ8q\\\\\\\/VIpZkX\\\\\\\/GBFwTORtXb0TzuZIEdyzRYFO+LlsGgx1u43WVw0Qssma7vJU+gJ6sI1sVx3xen\\\\\\\/u6bXlTG\\\\\\\/Gsle2hpliAYwXfSObBoCaZkYb2Aw+okht4PeyGqHLjqSM+brie+4ZGuL3hwyXXKPncF+NSvpekriUKX0Wpd\\\\\\\/hbar9MfWB+pdvYqpJG4968v\\\\\\\/xwi9ZrHRBexy3eiG7wKPSNNE9v7\\\\\\\/m6Dd6hHN8TiRvXSD\\\\\\\/MvWIEFOkGzeMnY0bJNZHXTnUO7Mhk9QUE5d\\\\\\\/vJnPSLerMmgzXQoMUCnf5nZJQf\\\\\\\/IhifwzdAtmmY0Lg9LpJRSZinfeROXsx9xQlT01fz01J+O+G5QUg\\\\\\\/JamWdHOw\\\\\\\/sbusniiuf+DnT6A7s1GXJZv+RdaoI0gZoVQ+fQuXHKi+9+6Z+92RRta4E8TQhlIQIJ7srUOlh7vAtZWeabIhXvnN\\\\\\\/UVPB9G0k+C4EdZlig35uvyflYW80sVX\\\\\\\/hWtxnYngNb0poBAiKsWId1J2SfB70u2kQ0LM6ycEkCQ1FYIUtYfpFuowvGVVpGb4+dbpVtrVZv2JI9IcPEE7JuohtaqTT0SeaahcH28yMMJu1hic3G2\\\\\\\/ZzBYlYR1viaOFgTmVoUmACsQe66uYmW3s6Oa0OMo4\\\\\\\/zF7Y4Dp0uHwVQDCAaci5OhKHxAtTzoVE\\\\\\\/Sgu5bAHqjkdc4xdmCIki0SUIrKEF2mmKCOkWMiyq\\\\\\\/UQCiQk7+lOSPsQOro9ltWtIaGS0hiKFHmb\\\\\\\/vke7siRApRFTZtBcNqanUZaoejg385OCcUfBjVb8k41Ea+N4N7j9ZOJ5+qAzdIQY8LnCMrXkm+rj4EdYlqNfEVGYf\\\\\\\/nzJDsxjifniW6kisPcEX4c349e3zpeUSDb1jAgqACaA9nl2VP63QEslk1bhbg0RD8n90kbJCBg2luiKFi1VD1RPtoMMVDjSkb3YMpCuX6pYrXND8ixlipwGl\\\\\\\/dIFrQTdHphVu3z4686mZFKKSPZhpZ+baLWRSfO9BvRoMiHV2lZmBW0QxnllKmQdV+tk0Bl9LOVdU5s7UCMExYVGe37s3f4Z8IUrLRIqhJLmMfgwyyn1ySRl8kCAYwHrG8T2B\\\\\\\/rrOdMagKMWp3pzwzjJiQNH9X2C603nWjFD9iSKrilxeE7UBR9+X7jPbpsLeDVbckMvKNvOSugd0ne8m4AKBiLd0HtzxIA0BWiyh\\\\\\\/UECqZjrqyL34CiMAEZC+ucqvdGzUPDnszMnYK+lPbClVTDNbyWeA28De+rTdvbP7olPUPfJAlW4R86MHaN72TY+ddMlUaUIKaZo4gMtu59wpl6oK+pJ8szrgruAA8Lcy0Q5OLkAmg1xIQ0jvUZ9nlvpNc1Ahv+zZsqsi670zZFDRwX4+BP\\\\\\\/PDbvzmKDO24rqMxcDPImv29kDTJvJNmJqddsBRMXIyFligQtdR07XakjgG\\\\\\\/HdjDj4W8AjkWgIGLmIoYMlnyLRmj6S04UYoBtMTf3c6b1BwcbIqB0R0Eg2GpvgN24weUJRpUdTotXsrf6ylUYoown7PugCZjvnYXu\\\\\\\/Q82M5xyOZQSemYkLkqe6CQUlryLi3jvieocZJmncdzObnge8l7ud84+azICT4GTdr7F0Q2tbn+AGOjMILrWcuDMt56sue6v954o+NbzobiHwNrw0nC+Fs781QgadFMneMI\\\\\\\/r1YUUzaQCWxtUK0wAkAMiWYjU3VJKMbMX\\\\\\\/HxPv1CGeyEbsM0UjWtHT9oU4VdhGMhpih37nnVpHFqznrV\\\\\\\/tMc3QhUuHfSbv1Q81Ou3gBSxCaF9J66hhAV0cmGfePMROPd5KCCKkShm4LCJ+SqzKTrItxcY9i8QSF6Ctqh9viyif\\\\\\\/IJdeZpAzTtPbHaFsvqeDZmJPxDMtEZXkXH1t5PVZ4YPV7kf5zN6MvvcEp0M8A5fTjjJfHLGAQmYG3WS1sLAWL5Z2XvMo2MNpijS+M6x1qTEOAEeDFiOQgiBgiITuOlCDCVGISdQ3+K44J8WcWvA8PYj4S3pYsEP5lL9oTQjlum\\\\\\\/WJXHQRskw5mmRXZyBfFf0nYPOVMHoKRA5\\\\\\\/YrGf1iy4AsfisuoCk7ISVd120z4cTHB6H6FXAEQd88QbHEamoLiyfBiqbJKrLx9KONezYQw3hOvUbAeUrIKqOmooXT+zfNm4Utx+z\\\\\\\/e807X0KGppTajbnXrkhzTn0TrL8sELPOggYponTvXLG9ufVwEx8IgoEq2VV1izkExzl\\\\\\\/GpY4Dh0Xco09zmCMhguHIt5NzeaKIuj90J\\\\\\\/NRONVgJruNFnrDqC\\\\\\\/KG69PIuF96IWwJcdZKR4f\\\\\\\/HDVY1cdHUVWjICGH6\\\\\\\/4+z9qux7p4oH0\\\\\\\/b5Su2GFhwKsqQad7V7PcGJWFVJ1O0GqvqvXltBqO5HbboMek1RCNTxrLRY7dhc7y5W3TZYInWVmzdm1Cc4YmLoWfaPLFituT6n8Wivj6X7Sz7lZbd9InV9lvXR3qVvSjhYuK+d\\\\\\\/LGvIz6YPLL2mjOkMLY8kY8FXYbnnMvgeEdXL2zn46kZwKDRdSSuG2j8kw2y71ez0BXbf+UsEz8uKYXdypsampHlIInteHhTKr04RfLzfLUc7ywLuFlpnR0O8ZZqVCiwPi+Ct9zVUn9d\\\\\\\/hJkJbz2brFld5bRFYZ+z+YfsKzJUB37CrGZuEPjiC3mS6GahZQhlOYnjC3mxjGnqWx\\\\\\\/et6jghf92UW1wFUJoqNyddOVU9\\\\\\\/8yJmkrynnHqmjiShHECD1HOIvza\\\\\\\/x9MrxdS9iR555Ao5B9OGjxpkVDyXO3Kpn8CTWLfSNT7Z1513B1+bXb8nls5Wqrby\\\\\\\/tFSxYZFNXcQBuBRbpx6wFSaz9uRNoE0fGdJEnjZ83Is3zZsdmUzqsdx+LLZiyLM66MFXLeyHNK9x3XmfxeMd8WO3l5cf0JSlSb4irOwH\\\\\\\/Wh\\\\\\\/Nb5\\\\\\\/YRgjm02mO6yuKRO4IofIyIdUYYIRaaUfb74wL88C1siPH4g+DjAtXgvvYdj8dZ6uY373hZuRZv4LkPHGMYMdfbM60NKP7YbE0YE\\\\\\\/9d+ajci3xnpOh6+LNx4Ugf8ruyakuPbRstflhIFNpL5WMbY4OqIxH5KHwbXX4tyt60xZpSvSC\\\\\\\/0dyTT02oCEgg6BOjHuDTbDQmmhDdJdU+yfEYTaSWbdEjdD2ZG6g6rEM7B3bhcUwnqzqv4f\\\\\\\/95fdsd7YjLKA4XNIAq9IGLbpcvLL3bmLhjJcoWE5JlOc1jJEn0ikhArKqf2SkYRskNnpQfzJt8WfacPpcCSiAO+HArjbGzHkXXfiPZGBCAiW\\\\\\\/8\\\\\\\/u2g1qcfsIV7yzBUkJEnqC\\\\\\\/1y\\\\\\\/x3OwdNUeuE8TbFjkRuGMuGseSh9NZu1hAYSeCjeTSIXJEhxD6\\\\\\\/fiKYQ\\\\\\\/\\\\\\\/GioKrKubK6KraLHS922wbkbwvcIjoKbrR+ITWtuKKDraez2T3fPy5H0EBW2YPLLtDDO4k8JHCtfRMvkaWuZi32au0qbqCgJShyaMqqhTZb0G8VN1mWM8xVpxF+M0XIQOH\\\\\\\/\\\\\\\/1rhZ2Wq\\\\\\\/\\\\\\\/l+JBDDccOgBmedS2dHieKoc16mfq+brE8I8d3kUzcnDnCyFG1ZgPgcV2ANXTB1qcdH0Lvhr\\\\\\\/aZasSGOOn3Av9\\\\\\\/nzehGyThrbj0OuOi\\\\\\\/PUQk6eB2iQehgtfd0+I8cpF6+F3YjHhkrefYwcFOwvCnUo+0bTalhZOzmEbKdEsE4L9eK1Fjpt7BZ9qtM3LVq+\\\\\\\/Q469r2yZlYrUYIwsH88xN17LeOWD1k7\\\\\\\/uZPmZnOvUmsSWCM+NLLzpF+Sx35pd80Xtmfouzh2ff7vWOxsuvffeUBry+KP\\\\\\\/Ep857YiJY9Czm9sOx+lQfwtkezm2XlcfZjcHZaHENPWfibJsbcPem5q\\\\\\\/4F15uKXEy7kTGxoMQge81H0jO2EWviyNNR+M\\\\\\\/yxEhZwPhzy5qQk+tZd4Xyk7mnZx76t\\\\\\\/o9OoSm+PZRljl\\\\\\\/EFiyjz\\\\\\\/HXxBqojSR8IUkywSl6E85TcQSd6zgpQb21rMemUR749FNpkGTmtfFG7KItrMp8djUio+L1RUptrH4zXbufFYXxKYPAvfrcQDyXpTeKgIqy3yI047BZ7PpiPovUIzEFl0\\\\\\\/4dCxovgXiXEEdbaC8A8X+cwDf4pe04Sk5gpmVzQT2vTkzZ1lutoe6FZO62H+FRhEKk\\\\\\\/CZU2hwCh1LR4cvE3\\\u0022,\\\u0022iv\\\u0022:\\\u0022BbDB6rBJQMQHJmJu7LJuFg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00220791f6482b7791b6978abb74cabf4a7d2073d03cbd81ad482a594d6e2114be64\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022rSgUhuNvoVzZZcxR3hNzaek0o+fAw0nOUmKd3kps+dGdzQvdgcM0DVIsy4CmqgixGko7hkq5Pk\\\\\\\/k4CthMW55EBGdzc9JH\\\\\\\/BIcUl8qWXLl6OXyZ7\\\\\\\/X2DjvPGrYIeD+Mhl5T2OmZTyLqgHvglr1F7KNvhqh36GxXDcI93w82OEs\\\\\\\/Zy09aTAj4HVht2mAvkMM0Wge1stTwfE+XRYWmekqBQ8vYlM3S7D1SAXkZEHmw\\\\\\\/XnfPvqgQjqeuAs3EFzLioMy+P2a9L5AYC2jOytsLvWVBobhlQArt7GVvU5ff6ZXxD2ueLiGmpiNzO5\\\\\\\/aWPP3L472HZZh5JH+Q7xi8qAovGKWatcq36ypxAcHWQiYQ2qTSgzs58wgMmIBFLI24LHNWrhKvzHoqDGnomSZpaMFYq\\\\\\\/CT+mnF+UNcmvu6IMCr\\\\\\\/x4MRqaPzZDHf+bX0UvUss8R9cOiMFGQ2h0sPpQjHJ65Qv+Ix6mBC+Axck8BSetmihLDuzhZfH2CTOHmLkKKi2faBFKWbgRwRfvw3uhln2GtDgz4DKC0xUo0LRFIWkfjSuxAkVYM8ajVNARYg8UU6Skpx\\\\\\\/3ZTBwixNZq0jZJA2LoAmt1huco7EQX7uYSBkSJdBUFtNPUmmQZVFT+I0eO2l3Nkawkjk\\\\\\\/zTFRzOKe04BPjcyJgKPCzwdhV12bFI\\\\\\\/uG5fy5g\\\\\\\/DgGYxY90K\\\\\\\/HojPG5J+1cb4w3beW8Dc9QuobrKHD0E6CE6MtEnwnI0onKwP8aQYyBP5MtBP6IuS7UzOWPBYRMYUv6zq3ta275mD3o85oyxvSpOrrIPkg\\\\\\\/3B\\\\\\\/ZZ84UTaskxr2Invd8jSMzTX3AlnB+IbAbjXOI+Jl9uB9GaSVW2tFXmvVKN3TVrpv\\\\\\\/wuWTrQmBAa6nqG\\\\\\\/dRxGn3RGd+85D02\\\\\\\/27Jj8Pa7ZctJh68shgSB6jwxP95JFbBDPuOsH5ZUyC379TOdMbtHG0BohOjydNjUVEqFsn2vCpsokRup+HZszyArOr2q7+MbzjOlMlB+qzVaUBzQxGUR9stVfOsGEkMTQU+Zhk3T5XxDWim8iw2lOqUZ5skz6c0FsHdFYuhhKkvg6CeTqDi267V8f59c9RqH\\\\\\\/3w4RJxR5X3uTs63G6bVAr2hAcBWkZArhPBNSTQ0sW7dIs3RYQ2dbEhgyQLTcwCgrLWtSKGEGxPGdRWTHVgYLlcq443b+igJO228LmDAp7vfasJWbcC\\\\\\\/7FZHrWEM9\\\\\\\/iW9Ip1\\\\\\\/KPbZAvYIjBRJmnc46UkNUMxK3\\\\\\\/CLYiqoIrLiFvnBqOu65EuNdQdY1+0SOyAL+0lcmqxVuThWyK2vGJsBShoMEU\\\\\\\/o9AQe3ONzMWxwwYfnnCsXzCZOG0BfvfVOigkGlJNv2zYN1jyF8onuZD6eQt8AKxTlZl9Kej1XufTD03lNT6creushtAsJ7A9M5JgColPT6Lhp9x+bZWGinmEtzYKmSed9miIRMPz2hqzavDUUUWC4zU67maAX6ESOS+1UiUKPKbcbZLwKTbEb4s0BTIh6ynKwSAg3uyp56pOl3Ff\\\\\\\/Ve5r+bY1KqevUn7i1\\\\\\\/C7RnHvgCRcyZj2o973YK3ZVFwhwouABNY81z2YuBb\\\\\\\/YEGEsDbpgEGb2\\\\\\\/PYjwZdGvcBzFCJuIglupm5D5RYy3ZArDDt\\\\\\\/AGqRuEZqxSW5oVXgL0E9uyApKTeia7Pxf82i1OMjyCJIJd5966y61Q5WU01dFX0hiaGKHdoNGlEHrfuFpZoBXaPpXBkGzFYG3EHL0xBMzRFhHFeJgR+\\\\\\\/bwH1xSkhABGVR+bZMQx8wfkLL4Tg6Q6ZfRUufHsuld+5pPfy4SdxZJFOyzsTKFOcnlORCO76as8Hcvz7LChgOzX+EJFmNn01SFnaJNriD5s\\\\\\\/lQOVpeVSjgVGslIp3C8uY5mgwqegu6Ji1GlolNcMlNjc7Rfuwun9DlyvHXafv3BnDzlaZKXo6ZzeYfXUum3Bxc9JPi37UMDNUEzzhPj90+\\\\\\\/CJzwNp3tnZWbxwsoCvjAprBakMPemNfC\\\\\\\/EAoF86iFcZjmjpkqzbOqGbpXq1tU2x4rR5U=\\\u0022,\\\u0022iv\\\u0022:\\\u0022xTI2bZtZti1EUrZ912fLfQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00229ed429bd40f22d23b0317e50e5f622e341e03895fc8f04954f5d49fbd3392061\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-490979 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/10/3321" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-g1pCnVE3iXGbPEiVd8qi56LK1OOazf3PtGKYvEJBG7s" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/10/3321" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-TphpvCrt_LmKQKIcL1CrWhw7YYP9A5Xdz6x-tSctweM" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/10/3311" title="Early Diverging Insect-Pathogenic Fungi of the Order Entomophthorales Possess Diverse and Unique Subtilisin-Like Serine Proteases" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/10/3331" title="A Novel Mutation in Brain Tumor Causes Both Neural Over-Proliferation and Neurodegeneration in Adult Drosophila" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490979" id="node490979" data-pisa="ggg;8/10/3321" data-pisa-master="ggg;g3.118.200563" data-apath="/ggg/8/10/3321.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Comparative Genomics Approaches Accurately Predict Deleterious Variants in Plants</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-1388-3194" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Thomas J.Y. Kono</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-5708-0118" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Li Lei</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-6652-1197" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Ching-Hua Shih</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-7693-8957" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Paul J. Hoffman</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0001-6282-1582" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Peter L. Morrell</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0003-1893-877X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Justin C. Fay</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3321-3329;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200563 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node490979" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Thomas J.Y. Kono </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy & Plant Genetics, University of Minnesota, St. Paul, MN 551085</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Thomas%2BJ.Y.%2BKono%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kono%20TJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AThomas%2BJ.Y.%2BKono%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-1388-3194" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Thomas J.Y. Kono</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Li Lei </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy & Plant Genetics, University of Minnesota, St. Paul, MN 551085</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Li%2BLei%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lei%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALi%2BLei%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-5708-0118" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Li Lei</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Ching-Hua Shih </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, Washington University, St. Louis, MO 63110</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ching-Hua%2BShih%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Shih%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AChing-Hua%2BShih%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-6652-1197" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Ching-Hua Shih</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Paul J. Hoffman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy & Plant Genetics, University of Minnesota, St. Paul, MN 551085</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Paul%2BJ.%2BHoffman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hoffman%20PJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APaul%2BJ.%2BHoffman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-7693-8957" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Paul J. Hoffman</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Peter L. Morrell </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy & Plant Genetics, University of Minnesota, St. Paul, MN 551085</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Peter%2BL.%2BMorrell%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Morrell%20PL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APeter%2BL.%2BMorrell%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-6282-1582" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Peter L. Morrell</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:justin.fay@rochester.edu" class="" data-icon-position="" data-hide-link-title="0">justin.fay@rochester.edu</a>
<a href="mailto:pmorrell@umn.edu" class="" data-icon-position="" data-hide-link-title="0">pmorrell@umn.edu</a></span></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Justin C. Fay </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, Washington University, St. Louis, MO 63110</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Justin%2BC.%2BFay%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fay%20JC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJustin%2BC.%2BFay%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-1893-877X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Justin C. Fay</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:justin.fay@rochester.edu" class="" data-icon-position="" data-hide-link-title="0">justin.fay@rochester.edu</a>
<a href="mailto:pmorrell@umn.edu" class="" data-icon-position="" data-hide-link-title="0">pmorrell@umn.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/8/10/3321" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:490979" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:490979/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/10/3321.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:490979" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:490979/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/10/3321.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:490979" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:490979/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/10/3321.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:490979" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:490979/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/8/10/3321.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:490979" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:490979/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">Recent advances in genome resequencing have led to increased interest in prediction of the functional consequences of genetic variants. Variants at phylogenetically conserved sites are of particular interest, because they are more likely than variants at phylogenetically variable sites to have deleterious effects on fitness and contribute to phenotypic variation. Numerous comparative genomic approaches have been developed to predict deleterious variants, but the approaches are nearly always assessed based on their ability to identify known disease-causing mutations in humans. Determining the accuracy of deleterious variant predictions in nonhuman species is important to understanding evolution, domestication, and potentially to improving crop quality and yield. To examine our ability to predict deleterious variants in plants we generated a curated database of 2,910 <em>Arabidopsis thaliana</em> mutants with known phenotypes. We evaluated seven approaches and found that while all performed well, their relative ranking differed from prior benchmarks in humans. We conclude that deleterious mutations can be reliably predicted in <em>A. thaliana</em> and likely other plant species, but that the relative performance of various approaches does not necessarily translate from one species to another.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/deleterious-mutations" class="hw-term hw-article-keyword hw-article-keyword-deleterious-mutations" rel="nofollow">deleterious mutations</a></li><li class="kwd"><a href="/keyword/phenotypes" class="hw-term hw-article-keyword hw-article-keyword-phenotypes" rel="nofollow">phenotypes</a></li><li class="kwd"><a href="/keyword/genome" class="hw-term hw-article-keyword hw-article-keyword-genome" rel="nofollow">genome</a></li><li class="kwd"><a href="/keyword/training-set" class="hw-term hw-article-keyword hw-article-keyword-training-set" rel="nofollow">training set</a></li></ul><p id="p-4">Dramatically increased numbers of reference genomes and whole genome resequencing data sets have facilitated the discovery of sequence variants and increased interest in the annotation of functional variants in many organisms. Functional annotation can yield insight into the genetic basis of phenotypic variation and is often a critical step in the identification of genes and variants underlying human disease (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Ahituv <em>et al.</em> 2007</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Cooper and Shendure 2011</a>). In particular, interest in identifying putatively deleterious variants has increased, because these variants may contribute substantially to phenotypic variation (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Manolio <em>et al.</em> 2009</a>; <a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">Thornton <em>et al.</em> 2013</a>). Because deleterious variants are more likely to disrupt phylogenetically conserved sites, the availability of comparative genomics data has made it possible to develop computational approaches to identifying deleterious variants genome-wide (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Ng and Henikoff 2006</a>). Although a number of approaches have been developed to identify deleterious variants within noncoding sequences (<em>e.g.</em>, <a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Pollard <em>et al.</em> 2010</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Kircher <em>et al.</em> 2014</a>), most have focused on variants that alter the amino acid sequence of proteins (<a id="xref-ref-56-2" class="xref-bibr" href="#ref-56">Ng and Henikoff 2006</a>). This focus on amino acid substitutions in protein coding sequences is in part driven by the observation that amino acid-altering single nucleotide polymorphisms (SNPs) are more often associated with phenotypic variation than other classes of variants, but also because they are the most readily identifiable class of variants that are likely to have a biological impact (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1000 Genomes Project Consortium <em>et al.</em> 2012</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Fay 2013</a>; <a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Stenson <em>et al.</em> 2014</a>).</p><p id="p-5">While identification of disease-causing and potentially “actionable” genetic variants is fundamental to personalized medicine, identifying deleterious variants is also broadly relevant to understanding the genetic basis of phenotypic variation. In humans, annotation of deleterious variants improves prediction accuracy of complex traits (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Dudley <em>et al.</em> 2012</a>). For domesticated organisms, complementation of recessive deleterious variants between haplotypes is thought to be one of the primary mechanisms underlying heterosis (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Charlesworth and Willis 2009</a>). This suggests that identification of deleterious alleles may be applied to hybrid breeding strategies (<a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">Yang <em>et al.</em> 2017</a>). Elevated proportions of deleterious relative to neutral variants in domesticated species suggest a cost of domestication (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Moyers <em>et al.</em> 2018</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Lu <em>et al.</em> 2006</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Cruz <em>et al.</em> 2008</a>; <a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Rodgers-Melnick <em>et al.</em> 2015</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Liu <em>et al.</em> 2017</a>). Studies of the genomic distribution and genetic contribution of deleterious variants can contribute both to understanding the origin and domestication of crop species and to advancing breeding and crop improvement strategies (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Morrell <em>et al.</em> 2012</a>).</p><p id="p-6">Accurate prediction of deleterious variants is a key component of assessing their contribution to phenotypic variation. Numerous approaches for predicting deleterious variants have been developed. The performance of an approach is typically assessed using the proportion of known, disease-causing human variants that are accurately classified as deleterious. Benchmarking of various approaches using standardized test sets has shown substantial variability among approaches, and improved performance is often achieved through combining results from multiple tools (<a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">Thusberg <em>et al.</em> 2011</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">González-Pérez and López-Bigas 2011</a>; <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Olatubosun <em>et al.</em> 2012</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Grimm <em>et al.</em> 2015</a>). However, the causes of performance differences across approaches are not well understood. While all approaches rely on sequence conservation at the phylogenetic level to identify deleterious variants, some approaches also incorporate protein structure, physical or biochemical properties of amino acid changes, or other attributes of protein sequence when they are available. The earliest conservation metrics used heuristic measures, sometimes including filtering or weighting to account for phylogenetic distance (<a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Sunyaev <em>et al.</em> 1999</a>; <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Miller and Kumar 2001</a>; <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Ng and Henikoff 2003</a>). More recent approaches have incorporated evolutionary models that account for phylogenetic distance based on putatively neutrally evolving nucleotide sites (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Chun and Fay 2009</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Davydov <em>et al.</em> 2010</a>). Reference bias and the alignments used to calculate conservation metrics are not often emphasized, but are important for making accurate predictions and may account for some of the variability among predictions (<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Chun and Fay 2009</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Hicks <em>et al.</em> 2011</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Adzhubei <em>et al.</em> 2013</a>). Additionally, different predictions have been found using human-based or mouse-based queries of the same substitution (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Miosge <em>et al.</em> 2015</a>). The accuracy of predictions is particularly dependent on the availability of annotated genomes among related species and the potential to generate sequence alignments.</p><p id="p-7">Despite most approaches being developed for and applied to humans, there has been growing interest in identifying deleterious variants in non-human species in order to understand genomic patterns of variation and their contribution to complex traits, especially in plants. Patterns of deleterious variation have been examined in <em>Arabidopsis thaliana</em> (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Cao <em>et al.</em> 2011</a>), rice (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Günther and Schmid 2010</a>; <a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Liu <em>et al.</em> 2017</a>), maize (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Mezmouk and Ross-Ibarra 2014</a>; <a id="xref-ref-64-2" class="xref-bibr" href="#ref-64">Rodgers-Melnick <em>et al.</em> 2015</a>), sunflower (<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Renaut and Rieseberg 2015</a>), poplar (<a id="xref-ref-76-1" class="xref-bibr" href="#ref-76">Zhang <em>et al.</em> 2016</a>), barley, and soybean (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Kono <em>et al.</em> 2016</a>). However, the accuracy of predictions in plants has only been examined for a small number of known variants (<a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Günther and Schmid 2010</a>) and only in the past few years have a diverse set of plant genomes and protein homologs become available (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Goodstein <em>et al.</em> 2012</a>). Furthermore, plants are known to have a larger number of multi-gene families and a higher frequency of polyploidy than occurs in mammals (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Lockton and Gaut 2005</a>). These genome-specific factors influence whether a sequence variant is truly deleterious in a given species (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Comai 2005</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Charlesworth 2012</a>).</p><p id="p-8">The goal of this study was to evaluate the ability of various approaches to predict deleterious variants in plants. The model system <em>A. thaliana</em> is a particularly attractive plant species for evaluating approaches that predict deleterious variants because decades of basic research in development, physiology, cell biology, and plant-pathogen interactions have identified large numbers of amino acid-altering mutations with phenotypic consequences. We identified seven approaches that can predict deleterious variants outside of humans (Table S1). Among these approaches, SIFT (<a id="xref-ref-55-2" class="xref-bibr" href="#ref-55">Ng and Henikoff 2003</a>), PolyPhen2 (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Adzhubei <em>et al.</em> 2013</a>) and PROVEAN (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Choi <em>et al.</em> 2012</a>) generate their own alignments using hits from non-redundant protein databases, whereas MAPP (<a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Stone and Sidow 2005</a>), GERP++ (<a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Davydov <em>et al.</em> 2010</a>), and two versions of a likelihood ratio test (<a id="xref-ref-13-3" class="xref-bibr" href="#ref-13">Chun and Fay 2009</a>) make predictions using pre-specified alignments as input (Table S1). Because new genome sequences are continually becoming available, the BAD_Mutations pipeline was developed to flexibly identify homologs and generate alignments for any protein of interest (<a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Kono <em>et al.</em> 2016</a>). BAD_Mutations uses TBLASTX (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Altschul <em>et al.</em> 1990</a>) to identify the best match (homolog) from each specified genome and aligns them with PASTA (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Mirarab <em>et al.</em> 2015</a>). For the four approaches that require alignments, we used the BAD_Mutations pipeline applied to 42 plant genomes. BAD_Mutations was also used to implement two approaches based on a likelihood ratio test (<a id="xref-ref-13-4" class="xref-bibr" href="#ref-13">Chun and Fay 2009</a>; <a id="xref-ref-38-3" class="xref-bibr" href="#ref-38">Kono <em>et al.</em> 2016</a>).</p><p id="p-9">To evaluate predictions of deleterious variants in plants, we generated a curated database of 2,910 <em>A. thaliana</em> mutants with known phenotypic alterations. We evaluated the ability of seven approaches to identify these deleterious variants and found that while performance was better than similar assessments in humans, the relative ranking and the highest performing approach differed from previously reported comparisons using human data. Our results demonstrate that reliable prediction of deleterious variants can be achieved in <em>A. thaliana</em>, and likely other plant species, expanding the potential value of using deleterious variants to understand naturally occurring variation and to improve crop breeding strategies.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Generation of a curated set of <em>Arabidopsis thaliana</em> mutations</h3><p id="p-10">We curated a set of amino acid-altering mutations with phenotypic impacts. Both morphological and biochemical phenotypes were represented, and mutations were in both single-copy and duplicated genes. These mutations were obtained from two sources. We generated a manually curated set of 542 amino acid-altering mutations in 155 genes with phenotypic effects that are described in the literature. These mutations were found by searching the <em>Arabidopsis</em> Information Resource (<a href="http://www.arabidopsis.org">http://www.arabidopsis.org</a>) for genes with either dominant or recessive alleles that differ by nucleotide substitutions. We also identified mutations using a literature search in Google Scholar (<a href="http://scholar.google.com">http://scholar.google.com</a>). For each variant, we recorded the amino acid substitution, position, and link to the published paper (Table S2). We excluded nonsense mutations because they frequently completely eliminate gene function. We identified a second set of 2,617 amino acid-altering mutations in 960 genes from the manually curated UniProt/Swiss-Prot database (<a href="http://www.uniprot.org/">http://www.uniprot.org/</a>) (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Boutet <em>et al.</em> 2016</a>). The two sets were independently generated and had an overlap of 249 mutants. Using mutants with named alleles as a proxy for those with morphological <em>vs.</em> biochemical phenotypes, 65% of our manually curated set and 33% of the Swiss-Prot set had macroscopic phenotypes. Duplicated genes were defined by those proteins with a significant BLASTP hit (E-value &lt; 0.05) to another <em>A. thaliana</em> protein with &gt; 60% identity. By this criterion 466 of 995 proteins were classified as duplicated.</p><p id="p-11">Single nucleotide polymorphisms (SNPs) without any known phenotype were obtained from a set of 80 sequenced <em>A. thaliana</em> strains (Ensembl, version 81, “Cao_SNPs”, (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Cao <em>et al.</em> 2011</a>)). At the time of download, these were the only SNP set available for unrestricted use. After filtering out sites with heterozygous or missing genotype calls, there were 10,797 biallelic amino acid-altering SNPs in the 995 proteins. We used a subset of 1,583 common SNPs (&gt;10%) as those least likely to have phenotypic effects. Our rationale is that on average, strongly deleterious alleles are less likely to reach high frequency in a population, owing to the effects of purifying selection (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Fay <em>et al.</em> 2001</a>). We also assessed performance by measuring the enrichment of deleterious variants predicted for rare compared to common polymorphisms (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Boyko <em>et al.</em> 2008</a>). A second set of common amino acid-altering SNPs were identified in an independent set of genes. Excluding the original set of 995 genes, we randomly selected 1,000 proteins from 35,386 peptides in the <em>A. thaliana</em> database. We removed 21 that carried no amino acid polymorphism in the 1,001 genomes project (<a href="http://www.1001genomes.org">http://www.1001genomes.org</a>). In the remaining 979 genes, we identified 40,736 biallelic amino acid altering SNPs in the 1,001 genomes project, of which 3,717 were common (&gt;10%).</p></div><div id="sec-3" class="subsection"><h3>Performance evaluations of seven approaches</h3><p id="p-12">We assessed amino acid substitutions using seven approaches: LRT (<a id="xref-ref-13-5" class="xref-bibr" href="#ref-13">Chun and Fay 2009</a>), LRT-masked (33), PolyPhen2 (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Adzhubei <em>et al.</em> 2010</a>), SIFT 4G (<a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">Vaser <em>et al.</em> 2016</a>), Provean (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Choi <em>et al.</em> 2012</a>), MAPP (<a id="xref-ref-69-2" class="xref-bibr" href="#ref-69">Stone and Sidow 2005</a>) and GERP++ (<a id="xref-ref-18-3" class="xref-bibr" href="#ref-18">Davydov <em>et al.</em> 2010</a>). PolyPhen2 predictions were generated using the standalone software (v2.2.2) with the PolyPhen2 bundled non-redundant database (uniref100-release 2011_12) and the probabilistic variant classifier using the default HumDiv model. Precomputed SIFT 4G predictions were obtained for <em>A. thaliana</em> (TAIR10.23) (<a href="http://sift.bii.a-star.edu.sg">http://sift.bii.a-star.edu.sg</a>) and are based on the UniRef90 database (2011). SIFT 4G predictions were not available for 855 substitutions, predominantly because the amino acid change involved more than one nucleotide change within a codon. Provean predictions (v1.1.5) were generated for all mutations using NCBI’s non-redundant database (04/02/2016). MAPP and GERP++ predictions were generated using BAD_Mutations alignments and trees (see below). GERP++ generates predictions for single nucleotide positions rather than codons, based on a deficit of observed substitutions compared to that expected given a neutral substitution rate. To assess GERP++ performance we used the GERP++ score at the first, second or third position of the codon if the amino acid substitution could occur by a single change at one of those positions and the average of the GERP++ scores at the first and second positions for all other types of changes. In addition, because GERP++ did not initially perform well on the <em>A. thaliana</em> data using neutral substitution rates estimated from each alignment (default) we used a uniform neutral rate of 10 substitutions per site across all genes.</p></div><div id="sec-4" class="subsection"><h3>Implementation of BAD_Mutations pipeline</h3><p id="p-13">Predictions using a likelihood ratio test (LRT) were performed with the BAD_Mutations pipeline (<a id="xref-ref-38-4" class="xref-bibr" href="#ref-38">Kono <em>et al.</em> 2016</a>). The pipeline is comprised of Python and Bourne Again Shell (BASH) scripts and incorporates several open-source tools, including the alignment tool PASTA (<a id="xref-ref-52-2" class="xref-bibr" href="#ref-52">Mirarab <em>et al.</em> 2015</a>) and maximum likelihood methods implemented in HyPhy (<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Pond <em>et al.</em> 2005</a>). The processing step of BAD_Mutations consists of five major subcommands: (1) setup; (2) fetch; (3) align; (4) predict; and (5) compile (Figure S1). The <strong>setup</strong> subcommand generates the configuration files. The <strong>fetch</strong> subcommand downloads gzipped CDS FASTA files from both Phytozome (<a href="https://phytozome.jgi.doe.gov/pz/portal.html">https://phytozome.jgi.doe.gov/pz/portal.html</a>) and Ensembl Plants (<a href="http://plants.ensembl.org/index.html">http://plants.ensembl.org/index.html</a>), and then creates BLAST databases for identifying homologs. The <strong>align</strong> subcommand uses BLAST to identify homologs of any query protein and generates a protein alignment and phylogenetic tree using PASTA (<a id="xref-ref-52-3" class="xref-bibr" href="#ref-52">Mirarab <em>et al.</em> 2015</a>). The <strong>predict</strong> subcommand generates predictions for a list of codons of interest by sending a custom batch command to implement a likelihood ratio test using HyPhy. The likelihood ratio test compares the log likelihood of evolution at a single codon under a neutral model (<em>dN = dS</em>) to a model allowing for constraint (<em>dN = ωdS</em>), where <em>dN</em> and <em>dS</em> are the synonymous and nonsymous substitution rates and <em>ω</em> is a free parameter for selective constraint (<a id="xref-ref-13-6" class="xref-bibr" href="#ref-13">Chun and Fay 2009</a>). The <strong>compile</strong> subcommand is to generate the report and p-values. The user manual, including a brief tutorial, is available at <a href="https://github.com/MorrellLAB/BAD_Mutations/blob/master/Manual/Manual_v1.0.md">https://github.com/MorrellLAB/BAD_Mutations/blob/master/Manual/Manual_v1.0.md</a>.</p><p id="p-14">The BAD_Mutations pipeline makes use of sequenced and annotated genomes. We used BLAST searches of the <em>A. thaliana</em> gene sequences against 42 Angiosperm genomes, retaining the top hit from each species with a BLAST E-value threshold of 0.05. The homolog searches were restricted to Angiosperm genomes to avoid extensive saturation of synonymous sites. Protein alignments were generated with PASTA (<a id="xref-ref-52-4" class="xref-bibr" href="#ref-52">Mirarab <em>et al.</em> 2015</a>), and a likelihood ratio test (LRT) for constraint on each codon of interest was calculated using HyPhy (<a id="xref-ref-60-2" class="xref-bibr" href="#ref-60">Pond <em>et al.</em> 2005</a>). Sequences with ‘N’s or other ambiguous nucleotides were discarded prior to the likelihood ratio test. The LRT differs compared to its original formulation (<a id="xref-ref-13-7" class="xref-bibr" href="#ref-13">Chun and Fay 2009</a>) in that: i) <em>dS</em> was estimated using all codons for each gene separately, ii) query sequences were optionally masked (the entire sequence changed to N = missing) in the likelihood calculation to avoid any reference bias and iii) branches with <em>dS</em> greater than 3 were set to 3 to avoid spuriously high estimates of <em>dS</em>. Additionally, the original LRT used heuristics to eliminate sites with <em>dN</em> &gt; <em>dS</em>, the derived allele present in another species, or sites with fewer than 10 species in the alignment. Rather than eliminating sites, we used logistic regression to provide a single probability of being deleterious based on the LRT test and these additional pieces of information.</p><p id="p-15">Logistic regression was applied using both the masked and unmasked LRT <em>p</em>-values, where the masked <em>p</em>-values were generated from alignments without the <em>A. thaliana</em> reference allele. For the unmasked logistic regression, we used the terms log10(LRT <em>p</em>-value), <em>dN/dS</em>, <em>Rn</em>, and <em>An</em>, where <em>Rn</em> and <em>An</em> are the number of <em>A. thaliana</em> reference and alternative (<em>i.e.</em>, mutant) amino acids observed in the alignment, respectively. For the masked model, we replaced <em>An</em> and <em>Rn</em> with the absolute value of <em>Rn – An</em> and the maximum of <em>Rn</em> and <em>An</em>, respectively. For both models <em>p</em>-values &lt; 1e-16 were set to 1e-16 and constraint values &gt; 10 were set to 10. Ten-fold cross-validation was used to assess the fit of the logistic regression. The average area under the ROC (receiver operating characteristic) curve based on cross-validation was 0.9575 (unmasked) and 0.9471 (masked). Because these values were nearly identical to the performance of the model fit to the entire dataset, 0.9581 (unmasked) and 0.9471 (masked), we used the logistic regression coefficients from the full dataset:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3321/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3321/embed/mml-math-1.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3321/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3321/embed/mml-math-2.gif"/></noscript></span></span>Sensitivity, specificity, and area under the curve (AUC) were calculated for each approach using the pROC package in R (<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Robin <em>et al.</em> 2011</a>). We define sensitivity as the proportion of phenotype-altering variants that are predicted to be deleterious, and specificity as the proportion of variants without known phenotypic effects that are predicted to be neutral. Confidence intervals for each were calculated by 2,000 replicates of stratified bootstrapping, where each replicate contains the same number of positives and negatives as in the original sample.</p><p id="p-16">Combined predictions were generated based on the combined scores of six approaches: LRT, LRT-masked, PolyPhen2, Provean, GERP++, and MAPP. SIFT 4G was not included in the combined predictions because it had missing predictions for a large number (855) variants. Sites with missing predictions from one or more of the remaining approaches (<em>n</em> = 215) were removed. Combined predictions were generated using: 1) logistic regression with each approach’s score as a predictive variable, 2) support vector machine, 3) random forest, 4) linear discriminant analysis and 5) generalized linear model with lasso penalized maximum likelihood implemented by the glmnet package in R (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Friedman <em>et al.</em> 2010</a>). The formula used for the ensemble methods was T∼LRT+LRTm+PPH+PROVEAN+SIFT+GERP+MAPP, where T is a vector of 0/1 for true negative and true positive, and the explanatory terms are the raw scores from each of the singular prediction approaches. The performance of each model was assessed by AUC values obtained from 10-fold cross-validation. The R script is available at <a href="https://github.com/MorrellLAB/BAD_Mutations/blob/master/Manuscript_Scripts/script/ensemble.R">https://github.com/MorrellLAB/BAD_Mutations/blob/master/Manuscript_Scripts/script/ensemble.R</a></p></div><div id="sec-5" class="subsection"><h3>Availability of data and materials</h3><p id="p-17">LRT predictions were implemented in the Python package BAD_Mutations which is freely available from <a href="http://github.com/MorrellLAB/BAD_Mutations.git">http://github.com/MorrellLAB/BAD_Mutations.git</a>. All the scripts used for data analysis in this manuscript are available at <a href="https://github.com/MorrellLAB/BAD_Mutations/tree/master/Manuscript_Scripts">https://github.com/MorrellLAB/BAD_Mutations/tree/master/Manuscript_Scripts</a>. Alignments of CDS from 42 plant species and Table S2 are available at Data Repository of the University of Minnesota: <a href="https://doi.org/10.13020/D6N69S">https://doi.org/10.13020/D6N69S</a>. Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.6998387">https://doi.org/10.25387/g3.6998387</a>.</p></div></div><div class="section results" id="sec-6"><h2 class="">Results</h2><div id="sec-7" class="subsection"><h3>Curation of a test set of <em>Arabidopsis thaliana</em> mutants</h3><p id="p-18">To evaluate approaches that predict deleterious variants, we generated a database of <em>A. thaliana</em> amino acid substitutions from mutants with described phenotypic alterations and common amino acid polymorphisms unlikely to affect fitness. Out of 2,910 mutants in 995 genes, 81% were from manually curated entries in UniProtKB/Swiss-Prot (<em>n</em> = 2,368), 10% were from our own literature curation (<em>n</em> = 293) and 8.6% were independently identified in both sets (<em>n</em> = 249) (Table S2). Within the same 995 genes, 1,583 common amino acid polymorphisms were identified in 80 accessions (<a id="xref-ref-9-3" class="xref-bibr" href="#ref-9">Cao <em>et al.</em> 2011</a>). For our analyses, we assume mutations that cause a deviation from the wildtype phenotype are likely deleterious.</p></div><div id="sec-8" class="subsection"><h3>Performance of approaches designed to identify deleterious variants</h3><p id="p-19">Using the database of <em>A. thaliana</em> mutations, we assessed seven approaches for their ability to distinguish deleterious and neutral changes. The approaches were selected because they can generate predictions in non-human organisms. Comparison of sensitivity to specificity showed that each approach could reliably distinguish deleterious and neutral substitutions (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). A likelihood ratio test (LRT) implemented using the BAD_Mutations pipeline showed significantly higher performance than all other approaches as measured by the area under the curve (AUC) of sensitivity <em>vs.</em> specificity (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>, Table S3). A reference masked version of LRT (LRTm), designed to eliminate reference bias (<a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Simons <em>et al.</em> 2014</a>), was the approach with the second highest performance. PROVEAN and PolyPhen2 showed similar performance as measured by AUC, significantly higher than SIFT, GERP++ and MAPP. The relative ranking by AUC was identical when 1,050 mutations with missing predictions for at least one approach were removed (Table S3). We also found very similar measures of performance when we used common SNPs in a set of independent, randomly selected genes rather than common SNPs within the 995 genes with known phenotype altering mutations (Table S3).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/10/3321/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparison of approaches that distinguish deleterious and neutral amino acid substitutions. The fraction of true positives (sensitivity) vs. the fraction of true negatives (specificity) is shown for seven approaches (LRTm is a masked version of LRT, PPH2 is PolyPhen2). The curves are based on 2,910 deleterious variants and 1,583 neutral variants. Vertical and horizontal dashed lines show the cutoff at 95% specificity and 95% sensitivity, respectively." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1648953636" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Comparison of approaches that distinguish deleterious and neutral amino acid substitutions. The fraction of true positives (sensitivity) &lt;em&gt;vs.&lt;/em&gt; the fraction of true negatives (specificity) is shown for seven approaches (LRTm is a masked version of LRT, PPH2 is PolyPhen2). The curves are based on 2,910 deleterious variants and 1,583 neutral variants. Vertical and horizontal dashed lines show the cutoff at 95% specificity and 95% sensitivity, respectively.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/10/3321/F1.medium.gif" width="440" height="433"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/8/10/3321/F1.medium.gif" width="440" height="433"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/10/3321/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/10/3321/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491085" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-20" class="first-child">Comparison of approaches that distinguish deleterious and neutral amino acid substitutions. The fraction of true positives (sensitivity) <em>vs.</em> the fraction of true negatives (specificity) is shown for seven approaches (LRTm is a masked version of LRT, PPH2 is PolyPhen2). The curves are based on 2,910 deleterious variants and 1,583 neutral variants. Vertical and horizontal dashed lines show the cutoff at 95% specificity and 95% sensitivity, respectively.</p><div class="sb-div caption-clear"></div></div></div><p id="p-21">A second means of assessing performance is through comparing predictions of rare <em>vs.</em> common variants. Common variants are likely neutral or nearly neutral, whereas deleterious alleles are expected to be kept at low frequency (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Ewens 2004</a>). Using SNPs identified in a set of 80 <em>A. thaliana</em> strains, we found each approach identified more deleterious SNPs at low compared to common frequencies (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). At minor allele frequencies between 2/80 (2.5%) and 8/80 (10%), the LRTm and SIFT predicted a lower proportion of deleterious SNPs compared to the other approaches, indicating that they are less sensitive to detecting alleles under weak selection. At the lowest frequency 1/80 (1.25%), which is expected to include many rare and potentially strongly deleterious variants, LRT called the largest proportion of SNPs deleterious.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/10/3321/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The proportion of SNPs called deleterious across frequency classes. The fraction of SNPs called deleterious by each approach (legend) at its 95% specificity threshold across five frequency classes, labeled by the number of minor alleles present (n = 80). The minor allele is defined as the allele that is less frequent in the sample. Sample sizes for the five classes are 5,303 (1), 1,646 (2), 1,250 (3-4), 1,015 (5-8) and 1,583 (&gt;8)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1648953636" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The proportion of SNPs called deleterious across frequency classes. The fraction of SNPs called deleterious by each approach (legend) at its 95% specificity threshold across five frequency classes, labeled by the number of minor alleles present (&lt;em&gt;n&lt;/em&gt; = 80). The minor allele is defined as the allele that is less frequent in the sample. Sample sizes for the five classes are 5,303 (1), 1,646 (2), 1,250 (3-4), 1,015 (5-8) and 1,583 (&gt;8).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/10/3321/F2.medium.gif" width="440" height="376"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/8/10/3321/F2.medium.gif" width="440" height="376"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/10/3321/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/10/3321/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491093" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-22" class="first-child">The proportion of SNPs called deleterious across frequency classes. The fraction of SNPs called deleterious by each approach (legend) at its 95% specificity threshold across five frequency classes, labeled by the number of minor alleles present (<em>n</em> = 80). The minor allele is defined as the allele that is less frequent in the sample. Sample sizes for the five classes are 5,303 (1), 1,646 (2), 1,250 (3-4), 1,015 (5-8) and 1,583 (&gt;8).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-9" class="subsection"><h3>Performance across phenotypic and duplicate gene categories</h3><p id="p-23">To further characterize differences in performance we compared class of variants, including those identified by genome-wide mutant screens or by directly targeting individual proteins. Mutants identified from screens have gross morphological or easily observable phenotypic effects and are often assigned allele names, whereas directed mutants are not often given allele names and tend to have biochemical phenotypes. To compare these two groups, we split the data into those with allele names (1,910), as a proxy for those with gross phenotypes, and those without allele names (1,000), as a proxy for biochemical phenotypes. As measured by AUC, some of the approaches performed better than others and performance was more similar for the gross phenotypic class compared to the biochemical class (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3a</a>). Both SIFT and PolyPhen2 demonstrated the largest difference in performance for predicting mutations with gross phenotypic alterations <em>vs.</em> biochemical phenotypes. For this type of mutation, the performance of PolyPhen2 was comparable to the LRT.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/10/3321/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Performance of approaches across different classes of sites. Performance is measured by the area under the curve (AUC) of the approach&#x2019;s sensitivity vs. specificity. A &#x2013; comparison of mutants with biochemical (n = 1,000) vs. gross phenotypes (n = 1,910). B &#x2013; comparison of performance for substitutions in duplicated (n = 2,098) vs. single copy genes (n = 2,395). Confidence intervals were determined by 2,000 bootstrapping iterations." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1648953636" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Performance of approaches across different classes of sites. Performance is measured by the area under the curve (AUC) of the approach&#x2019;s sensitivity &lt;em&gt;vs.&lt;/em&gt; specificity. A &#x2013; comparison of mutants with biochemical (n = 1,000) &lt;em&gt;vs.&lt;/em&gt; gross phenotypes (n = 1,910). B &#x2013; comparison of performance for substitutions in duplicated (n = 2,098) &lt;em&gt;vs.&lt;/em&gt; single copy genes (n = 2,395). Confidence intervals were determined by 2,000 bootstrapping iterations.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/10/3321/F3.medium.gif" width="440" height="224"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/8/10/3321/F3.medium.gif" width="440" height="224"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/10/3321/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/10/3321/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491102" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-24" class="first-child">Performance of approaches across different classes of sites. Performance is measured by the area under the curve (AUC) of the approach’s sensitivity <em>vs.</em> specificity. A – comparison of mutants with biochemical (n = 1,000) <em>vs.</em> gross phenotypes (n = 1,910). B – comparison of performance for substitutions in duplicated (n = 2,098) <em>vs.</em> single copy genes (n = 2,395). Confidence intervals were determined by 2,000 bootstrapping iterations.</p><div class="sb-div caption-clear"></div></div></div><p id="p-25">Gene duplication may reduce prior selective constraints on a protein, enabling variants to occur at previously conserved sites (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Kondrashov <em>et al.</em> 2002</a>). Thus, duplicated genes may pose challenges to predicting deleterious alleles, and none of the approaches explicitly distinguish orthologs and paralogs. We identified 466 of the 995 genes as duplicated in <em>A. thaliana</em> based on BLASTP hits with 60% or more identity. We compared the performance of these genes to the remaining single copy genes. Each approach showed equal or better performance for duplicated <em>vs.</em> single copy genes. SIFT had the largest increase in performance (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3b</a>).</p></div><div id="sec-10" class="subsection"><h3>Approach dissimilarity and composite predictions</h3><p id="p-26">As reported previously (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Doniger <em>et al.</em> 2008</a>; <a id="xref-ref-13-8" class="xref-bibr" href="#ref-13">Chun and Fay 2009</a>; <a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">González-Pérez and López-Bigas 2011</a>; <a id="xref-ref-58-2" class="xref-bibr" href="#ref-58">Olatubosun <em>et al.</em> 2012</a>), we found substantial disagreement in predictions among the approaches. At a 95% specificity threshold, 93.6% of mutants were predicted deleterious by one or more approach but only 51.3% were predicted deleterious by at least six of the seven approaches (Table S2). Similarly, only 0.25% of common SNPs were predicted deleterious by all approaches but 16.6% were predicted deleterious by at least one approach (LRT and LRTm were considered separately). Comparing the disagreement between approaches, we found LRT and LRTm to produce very similar predictions, but to be distinct from most of the other approaches (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). We used five models that combined the predictions of all approaches except for SIFT, which had a higher proportion of missing calls. Only two of these ensemble models, a linear discriminant analysis and a generalized linear model with penalized maximum likelihood, performed significantly higher than LRT based on an AUC (Table S4).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/10/3321/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Dissimilarities among approaches. Dissimilarities were computed by the pairwise number of disagreements between each approach applied to mutants and common SNPs (n = 4,493). Dissimilarities are represented by a tree based on hierarchical clustering and values below nodes are bootstrap support based on 2,000 iterations." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1648953636" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Dissimilarities among approaches. Dissimilarities were computed by the pairwise number of disagreements between each approach applied to mutants and common SNPs (n = 4,493). Dissimilarities are represented by a tree based on hierarchical clustering and values below nodes are bootstrap support based on 2,000 iterations.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/10/3321/F4.medium.gif" width="440" height="402"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/8/10/3321/F4.medium.gif" width="440" height="402"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/10/3321/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/10/3321/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491116" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-27" class="first-child">Dissimilarities among approaches. Dissimilarities were computed by the pairwise number of disagreements between each approach applied to mutants and common SNPs (n = 4,493). Dissimilarities are represented by a tree based on hierarchical clustering and values below nodes are bootstrap support based on 2,000 iterations.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-11"><h2 class="">Discussion</h2><p id="p-28">In this study, we benchmarked the potential for several widely-used approaches to distinguish putatively deleterious and neutral amino acid substitutions in <em>A. thaliana</em>. Prior evaluations of performance focused on large sets of mutants for single proteins or known human disease variants (<a id="xref-ref-55-3" class="xref-bibr" href="#ref-55">Ng and Henikoff 2003</a>; <a id="xref-ref-2-3" class="xref-bibr" href="#ref-2">Adzhubei <em>et al.</em> 2013</a>). Overall, we find high performance across approaches in their ability to distinguish neutral and deleterious variants, validating their use in plants. The highest performance is achieved by a likelihood ratio test (LRT) implemented using the BAD_Mutations pipeline, in this case using alignments from 42 plant genomes. However, the relative performance depended on the test set and, as discussed below, differs from previous benchmarking studies in humans. Thus, we recommend caution in interpreting slight differences in performance and advocate the use of multiple methods to achieve the highest confidence.</p><p id="p-29">Below, we discuss our results along with characteristics of the approaches and test data that may contribute to differences in predictions and performance when applied to non-human species. One important consideration is the distinction between deleterious variants and those that impact protein function and have phenotypic consequences. While these two groups are overlapping, they are not identical. Because conservation between species is directly related to fitness, we have used the term “deleterious” when referring to the prediction approaches. However, the test sets used to evaluate approaches are composed of variants known to affect protein function or phenotype. Thus, regardless of the nomenclature, any evaluation of approach performance necessarily assumes a large overlap between conserved amino acid positions and those that affect protein function as measured by phenotype. Equally relevant, we use common variants as “neutral” controls even though some common variants are likely to affect protein function due to local adaptation (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Hancock <em>et al.</em> 2011</a>) or hitchhiking (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Chun and Fay 2011</a>). Despite potential contamination, common variants provide the only large set of negative controls that can be used for training and estimating rates of false positives (<a id="xref-ref-56-3" class="xref-bibr" href="#ref-56">Ng and Henikoff 2006</a>). Both common and rare variants may also have compensatory effects on deleterious variants (<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Poon <em>et al.</em> 2005</a>). These potential interactions between variants further complicates the identification of truly deleterious variants in any species.</p><div id="sec-12" class="subsection"><h3>Phylogenetic power, alignments, and reference databases</h3><p id="p-30">Phylogenetic power is critical to all comparative genomic approaches that predict deleterious variants. When homologs are too closely related, not enough time has passed for neutral sites to accumulate amino acid substitutions. When homologs are too distantly related, functional sites may not be conserved due to compensatory changes or divergence in homolog function (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Marini <em>et al.</em> 2010</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Breen <em>et al.</em> 2012</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Jordan <em>et al.</em> 2015</a>). The LRT differs from the other approaches examined in that it uses synonymous sites as an internal control to account for the expected amount of protein divergence under a neutral model. As such, even homologs that are nearly identical in their amino acid sequences are informative, given that they have accumulated changes at synonymous sites. However, distantly related homologs are uninformative if divergence at synonymous sites is saturated, thus the LRT should only be applied to organisms where a sufficient number of related genomes are available. In this study, the majority of total dS values for the gene alignments was between 10 and 50, which provides sufficient divergence to test the likelihoods of constraint and relaxation (<a id="xref-ref-13-9" class="xref-bibr" href="#ref-13">Chun and Fay 2009</a>). GERP++ is similar to the LRT in that it uses a neutral substitution rate to make its predictions but differs in that the neutral rate must be specified rather than being estimated from synonymous sites within the alignment. GERP++ also does not make use of the genetic code to distinguish synonymous and nonsynonymous changes. In this regard, GERP++ was not appropriately applied since we used a fixed neutral rate for all genes rather than an alignment specific neutral rate.</p><p id="p-31">Out of the approaches compared, phylogenetic power cannot explain the differences between the LRT, MAPP, and GERP++ because they used the same alignments. However, we did notice substantial differences in performance based on the number of ungapped sequences present in the BAD_Mutations alignment at the position being queried (Figure S2). Both LRT and LRTm performed better than the other approaches when there were 10 or fewer sequences at the position of interest. We did not see this pattern when we used the number of sequences present at any position in the alignment, which was typically close to 42. We also did not see this pattern when we examined performance based on the number of sequences used by Provean or PolyPhen2, typically over 100 per gene.</p><p id="p-32">All approaches studied here use alignments to make their predictions, making the protein database and choice of homologs to be included in the alignment a critical step. For MAPP, GERP++, and LRT we used alignments generated using the BAD_Mutations pipeline which queries proteins from a set of annotated reference genomes, in this case from 42 Angiosperm species. SIFT and PolyPhen2 use the UniRef database (2011), whereas PROVEAN uses the most recent non-redundant protein database from NCBI. Both PROVEAN and PolyPhen2 are known to be sensitive to the choice of the reference database and criteria for inclusion of homologs (<a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">Choi <em>et al.</em> 2012</a>; <a id="xref-ref-2-4" class="xref-bibr" href="#ref-2">Adzhubei <em>et al.</em> 2013</a>). Despite the choice of homologs being an important step in predicting deleterious substitutions, the use of a plant-specific or entire non-redundant database does not appear to contribute to performance differences: the target database used for prediction does not determine the ranking of approaches in terms of their AUC (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>).</p><p id="p-33">Despite faster runtime of the ensemble approaches with respect to the LRT-based approach, there are circumstances where the LRT-based approach would have higher accuracy. The LRT-based approaches have higher performance in cases where there is shallow alignment depth across the phylogeny, for example, in newly formed genes or rare isoforms of a transcript. The LRT-based approach is able to estimate substitution rates and predict the impact of a variant while the heuristic approaches or the ensemble approaches would likely not make a prediction, and return a missing value.</p></div><div id="sec-13" class="subsection"><h3>Training and test sets</h3><p id="p-34">Performance of an individual approach depends on both the training and test sets used to measure it. Because performance is typically measured using common SNPs and known disease variants in humans, there has been some concern over the lack of independence between training and test sets (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Dong <em>et al.</em> 2015</a>; <a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Grimm <em>et al.</em> 2015</a>). However, another consideration that has not yet been examined is whether performance in one species translates to other distantly related species, which may not have the same availability of homologs from sequenced genomes spanning a range of phylogenetic relatedness. The performance of individual approaches could depend on the study system in that some approaches may expect homologs at certain phylogenetic distances, low rates of compensatory change, or low rates of gene duplication.</p><p id="p-35">Previous studies of the accuracy of prediction approaches made use of five human test datasets (<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Dong <em>et al.</em> 2015</a>; <a id="xref-ref-29-3" class="xref-bibr" href="#ref-29">Grimm <em>et al.</em> 2015</a>). We find better performance across approaches in our <em>A. thaliana</em> dataset than that reported for humans (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). It is unclear why the approaches uniformly perform better in <em>A. thaliana</em>. One possibility is that the neutral and deleterious variants in <em>A. thaliana</em> are more distinct from one another than in humans. The very large proportion of phenotype changing variants in our <em>A. thaliana</em> test set that are identified as deleterious means that this test data set is less useful for approach comparison due to the small number of cases that are difficult to predict correctly.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/491049/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/491049/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Performance measured by AUC of approaches based on different test sets</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-14" class="subsection"><h3>Population and gene-specific performance</h3><p id="p-38">Because nearly all measures of performance use either common polymorphism or recently fixed amino acid substitutions as a proxy for neutral SNPs, population and gene-specific factors that influence neutral polymorphism are expected to influence measures of performance. Humans have a small effective population size relative to other mammals (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Leffler <em>et al.</em> 2012</a>) and consequently a high ratio of nonsynonymous to synonymous diversity (<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Fay <em>et al.</em> 2001</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Kosiol <em>et al.</em> 2008</a>). Thus, distinguishing neutral and deleterious variants may be more difficult in humans than other species, and approaches trained using human polymorphism may be more conservative with respect to weakly deleterious variants. In comparison, predicting deleterious variants in <em>A. thaliana</em> may be facilitated by the fact that <em>A. thaliana</em> has slightly larger effective population size (<a id="xref-ref-9-4" class="xref-bibr" href="#ref-9">Cao <em>et al.</em> 2011</a>).</p><p id="p-39">It should be noted that both demographic history and the process of local adaptation could play important roles in the distribution of deleterious variants. In populations that are colonizing or expanding into novel environments, the selective coefficients against individual variants may change (<a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Slotte <em>et al.</em> 2013</a>), and locally adaptive variants may become appreciably enriched. Both humans and <em>A. thaliana</em> are known to have undergone demographic expansion in their recent evolutionary histories (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Hoffmann 2002</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Finlayson 2005</a>). While the relative extent of local adaptation in these two species is difficult to quantify, both exhibit an excess of low-frequency amino acid polymorphism characteristic of deleterious variants (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Lohmueller <em>et al.</em> 2008</a>; <a id="xref-ref-9-5" class="xref-bibr" href="#ref-9">Cao <em>et al.</em> 2011</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Henn <em>et al.</em> 2016</a>).</p><p id="p-40">Another potentially important factor in predicting deleterious variants is gene duplication. <em>A. thaliana</em> carries remnants of a whole genome duplication along with numerous tandem duplications (<a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">The Arabidopsis Genome Initiative 2000</a>) more than are present in the human genome (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Lynch and Conery 2000</a>). Gene duplication can lead to relaxed selection during subfunctionalization or pseudogenization (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Ohno 1970</a>), enabling amino acid variants to accumulate in recently duplicated genes. However, we found very similar performance between duplicate and single copy genes, consistent with a similar finding in humans using PolyPhen2 (<a id="xref-ref-2-5" class="xref-bibr" href="#ref-2">Adzhubei <em>et al.</em> 2013</a>). Because we only included genes with known mutant phenotypes, the sample of recently duplicated genes is limited.</p></div><div id="sec-15" class="subsection"><h3>Conclusions and future directions</h3><p id="p-41">Most approaches developed to predict deleterious mutations were trained using human data and in many cases, can only be used for human proteins (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Li <em>et al.</em> 2009</a>; <a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Schwarz <em>et al.</em> 2010</a>; <a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Kircher <em>et al.</em> 2014</a>). This study demonstrates that several generalized approaches perform exceptionally well in <em>A. thaliana</em>, implying that they should also work well for other plant species. Because of the similarly high performance, other considerations such as ease of implementation and compute time may be considered when choosing an approach to identify deleterious mutations in plants. Notably, LRT requires longer run times than any of the other approaches, typically 5.2 hr of computing time per gene compared to 14.5 and 9.4 min per gene for PolyPhen2 and Provean, respectively. One way the BAD_Mutations pipeline could be sped up while retaining the flexibility of querying customizable plant genomes is by using heuristic measures of site-specific conservation rather than the LRT. Provocatively, we found similar performance (AUC = 0.9551) for a logistic model that only used the number of reference and alternative alleles in the alignment (<em>Rn</em> and <em>An</em>). However, such heuristic measures may not be robust to a change in the reference species and its distance to other genomes in the database. A second approach would be to use predictions from the combined output of multiple prediction approaches, as this has been shown to be highly effective in humans (<em>e.g.</em>, <a id="xref-ref-27-3" class="xref-bibr" href="#ref-27">González-Pérez and López-Bigas 2011</a>). Although we did not find an ensemble predictor that greatly improved performance, removing LRT predictions did not reduce the performance of the ensemble predictions.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-42">We thank members of the Morrell Lab for discussion and software testing. We also would like to thank Drs. Danelle Seymour and Karl Schmid for helpful comments on an earlier version of the manuscript. Hardware and software support were provided by the University of Minnesota Supercomputing Institute. This work was supported by the US National Science Foundation Plant Genome Program grant (DBI-1339393 to JCF and PLM), the US Department of Agriculture Biotechnology Risk Assessment Research Grants Program (BRAG) (USDA BRAG 2015-06504 to PLM), and a University of Minnesota Doctoral Dissertation Fellowship (to TJYK).</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-4"><p id="p-43">Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.6998387">https://doi.org/10.25387/g3.6998387</a>.</p></li><li class="fn" id="fn-5"><p id="p-44"><em>Communicating editor: Y. Kim</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2018-07-08"><span class="received-label">Received </span>July 8, 2018.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2018-08-10"><span class="accepted-label">Accepted </span>August 10, 2018.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2018 Kono <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3321.1" data-doi="10.1038/nature11632"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">1000 Genomes Project Consortium,</span>, </li><li><span class="cit-auth"><span class="cit-name-surname">G. R.</span>  <span class="cit-name-given-names">Abecasis</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Auton</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brooks</span>  <span class="cit-name-given-names">L. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">DePristo</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">An integrated map of genetic variation from 1,092 human genomes.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">491</span>: <span class="cit-fpage">56</span>–<span class="cit-lpage">65</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature11632</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DAbecasis%26rft.auinit1%253DG.%2BR.%26rft.volume%253D491%26rft.issue%253D7422%26rft.spage%253D56%26rft.epage%253D65%26rft.atitle%253DAn%2Bintegrated%2Bmap%2Bof%2Bgenetic%2Bvariation%2Bfrom%2B1%252C092%2Bhuman%2Bgenomes.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature11632%26rft_id%253Dinfo%253Apmid%252F23128226%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature11632&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23128226&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3321.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000310434500030&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-other" id="cit-8.10.3321.2" data-doi="10.1002/0471142905.hg0720s76"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Adzhubei</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jordan</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sunyaev</span>  <span class="cit-name-given-names">S. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-comment">Predicting functional effect of human missense mutations using PolyPhen-2. Curr. Protoc. Hum. Genet. Editor. Board Jonathan Haines Al 0 7: Unit7.20</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/0471142905.hg0720s76</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.volume%253D76%26rft.spage%253D7201%26rft_id%253Dinfo%253Adoi%252F10.1002%252F0471142905.hg0720s76%26rft_id%253Dinfo%253Apmid%252F23315938%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/0471142905.hg0720s76&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23315938&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3321.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3321.3" data-doi="10.1038/nmeth0410-248"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Adzhubei</span>  <span class="cit-name-given-names">I. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schmidt</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peshkin</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ramensky</span>  <span class="cit-name-given-names">V. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gerasimova</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">A method and server for predicting damaging missense mutations.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">248</span>–<span class="cit-lpage">249</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nmeth0410-248</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DAdzhubei%26rft.auinit1%253DI.%2BA.%26rft.volume%253D7%26rft.issue%253D4%26rft.spage%253D248%26rft.epage%253D249%26rft.atitle%253DA%2Bmethod%2Band%2Bserver%2Bfor%2Bpredicting%2Bdamaging%2Bmissense%2Bmutations.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth0410-248%26rft_id%253Dinfo%253Apmid%252F20354512%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth0410-248&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20354512&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3321.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000276150600004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3321.4" data-doi="10.1086/513471"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ahituv</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kavaslar</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schackwitz</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ustaszewska</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Medical sequencing at the extremes of human body mass.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">80</span>: <span class="cit-fpage">779</span>–<span class="cit-lpage">791</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1086/513471</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DAhituv%26rft.auinit1%253DN.%26rft.volume%253D80%26rft.issue%253D4%26rft.spage%253D779%26rft.epage%253D791%26rft.atitle%253DMedical%2Bsequencing%2Bat%2Bthe%2Bextremes%2Bof%2Bhuman%2Bbody%2Bmass.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F513471%26rft_id%253Dinfo%253Apmid%252F17357083%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/513471&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17357083&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3321.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000245465200018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3321.5" data-doi="10.1006/jmbi.1990.9999"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Altschul</span>  <span class="cit-name-given-names">S. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gish</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Myers</span>  <span class="cit-name-given-names">E. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lipman</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">Basic local alignment search tool.</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">215</span>: <span class="cit-fpage">403</span>–<span class="cit-lpage">410</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/S0022-2836(05)80360-2</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DAltschul%26rft.auinit1%253DS.%2BF.%26rft.volume%253D215%26rft.issue%253D3%26rft.spage%253D403%26rft.epage%253D410%26rft.atitle%253DBasic%2Blocal%2Balignment%2Bsearch%2Btool.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fjmbi.1990.9999%26rft_id%253Dinfo%253Apmid%252F2231712%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/jmbi.1990.9999&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2231712&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3321.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1990ED16700008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-book" id="cit-8.10.3321.6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Boutet</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lieberherr</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tognolli</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schneider</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bansal</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">UniProtKB/Swiss-Prot, the manually annotated section of the UniProt KnowledgeBase: how to use the entry view</span>, pp. <span class="cit-fpage">23</span>–<span class="cit-lpage">54</span> in <span class="cit-source">Plant Bioinformatics</span>, edited by <span class="cit-ed"><span class="cit-name-surname">D.</span>  <span class="cit-name-given-names">Edwards</span></span>. <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">New York, NY</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3321.7" data-doi="10.1371/journal.pgen.1000083"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Boyko</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Williamson</span>  <span class="cit-name-given-names">S. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Indap</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Degenhardt</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hernandez</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Assessing the evolutionary impact of amino acid mutations in the human genome.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e1000083</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1000083</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DBoyko%26rft.auinit1%253DA.%2BR.%26rft.volume%253D4%26rft.issue%253D5%26rft.spage%253De1000083%26rft.epage%253De1000083%26rft.atitle%253DAssessing%2Bthe%2Bevolutionary%2Bimpact%2Bof%2Bamino%2Bacid%2Bmutations%2Bin%2Bthe%2Bhuman%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000083%26rft_id%253Dinfo%253Apmid%252F18516229%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000083&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18516229&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3321.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3321.8" data-doi="10.1038/nature11510"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Breen</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kemena</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vlasov</span>  <span class="cit-name-given-names">P. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Notredame</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kondrashov</span>  <span class="cit-name-given-names">F. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Epistasis as the primary factor in molecular evolution.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">490</span>: <span class="cit-fpage">535</span>–<span class="cit-lpage">538</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature11510</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DBreen%26rft.auinit1%253DM.%2BS.%26rft.volume%253D490%26rft.issue%253D7421%26rft.spage%253D535%26rft.epage%253D538%26rft.atitle%253DEpistasis%2Bas%2Bthe%2Bprimary%2Bfactor%2Bin%2Bmolecular%2Bevolution.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature11510%26rft_id%253Dinfo%253Apmid%252F23064225%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature11510&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23064225&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3321.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000310196200041&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3321.9" data-doi="10.1038/ng.911"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cao</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schneeberger</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ossowski</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Günther</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bender</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Whole-genome sequencing of multiple <em>Arabidopsis thaliana</em> populations.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">956</span>–<span class="cit-lpage">963</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.911</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DCao%26rft.auinit1%253DJ.%26rft.volume%253D43%26rft.issue%253D10%26rft.spage%253D956%26rft.epage%253D963%26rft.atitle%253DWhole-genome%2Bsequencing%2Bof%2Bmultiple%2BArabidopsis%2Bthaliana%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.911%26rft_id%253Dinfo%253Apmid%252F21874002%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.911&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21874002&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3321.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3321.10" data-doi="10.1534/genetics.111.138073"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The role of background selection in shaping patterns of molecular evolution and variation: evidence from variability on the <em>Drosophila</em> X chromosome.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">191</span>: <span class="cit-fpage">233</span>–<span class="cit-lpage">246</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.111.138073</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.138073%26rft_id%253Dinfo%253Apmid%252F22377629%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkxLzEvMjMzIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMC8zMzIxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3321.11" data-doi="10.1038/nrg2664"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Willis</span>  <span class="cit-name-given-names">J. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The genetics of inbreeding depression.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">783</span>–<span class="cit-lpage">796</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg2664</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DCharlesworth%26rft.auinit1%253DD.%26rft.volume%253D10%26rft.issue%253D11%26rft.spage%253D783%26rft.epage%253D796%26rft.atitle%253DThe%2Bgenetics%2Bof%2Binbreeding%2Bdepression.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2664%26rft_id%253Dinfo%253Apmid%252F19834483%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2664&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19834483&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3321.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000270915400014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3321.12" data-doi="10.1371/journal.pone.0046688"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Choi</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sims</span>  <span class="cit-name-given-names">G. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Murphy</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chan</span>  <span class="cit-name-given-names">A. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Predicting the functional effect of amino acid substitutions and indels.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e46688</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0046688</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DChoi%26rft.auinit1%253DY.%26rft.volume%253D7%26rft.issue%253D10%26rft.spage%253De46688%26rft.epage%253De46688%26rft.atitle%253DPredicting%2Bthe%2Bfunctional%2Beffect%2Bof%2Bamino%2BAcid%2Bsubstitutions%2Band%2Bindels.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0046688%26rft_id%253Dinfo%253Apmid%252F23056405%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0046688&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23056405&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3321.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3321.13" data-doi="10.1101/gr.092619.109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chun</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fay</span>  <span class="cit-name-given-names">J. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Identification of deleterious mutations within three human genomes.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">1553</span>–<span class="cit-lpage">1561</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.092619.109</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.092619.109%26rft_id%253Dinfo%253Apmid%252F19602639%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjE5LzkvMTU1MyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzgvMTAvMzMyMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3321.14" data-doi="10.1371/journal.pgen.1002240"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chun</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fay</span>  <span class="cit-name-given-names">J. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Evidence for hitchhiking of deleterious mutations within the human genome.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e1002240</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1002240</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DChun%26rft.auinit1%253DS.%26rft.volume%253D7%26rft.issue%253D8%26rft.spage%253De1002240%26rft.epage%253De1002240%26rft.atitle%253DEvidence%2Bfor%2Bhitchhiking%2Bof%2Bdeleterious%2Bmutations%2Bwithin%2Bthe%2Bhuman%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002240%26rft_id%253Dinfo%253Apmid%252F21901107%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002240&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21901107&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3321.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3321.15" data-doi="10.1038/nrg1711"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Comai</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">The advantages and disadvantages of being polyploid.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">836</span>–<span class="cit-lpage">846</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg1711</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DComai%26rft.auinit1%253DL.%26rft.volume%253D6%26rft.issue%253D11%26rft.spage%253D836%26rft.epage%253D846%26rft.atitle%253DThe%2Badvantages%2Band%2Bdisadvantages%2Bof%2Bbeing%2Bpolyploid.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg1711%26rft_id%253Dinfo%253Apmid%252F16304599%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg1711&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16304599&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3321.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000232990900014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3321.16" data-doi="10.1038/nrg3046"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cooper</span>  <span class="cit-name-given-names">G. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shendure</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Needles in stacks of needles: finding disease-causal variants in a wealth of genomic data.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">628</span>–<span class="cit-lpage">640</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg3046</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DCooper%26rft.auinit1%253DG.%2BM.%26rft.volume%253D12%26rft.issue%253D9%26rft.spage%253D628%26rft.epage%253D640%26rft.atitle%253DNeedles%2Bin%2Bstacks%2Bof%2Bneedles%253A%2Bfinding%2Bdisease-causal%2Bvariants%2Bin%2Ba%2Bwealth%2Bof%2Bgenomic%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3046%26rft_id%253Dinfo%253Apmid%252F21850043%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3046&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21850043&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3321.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3321.17" data-doi="10.1093/molbev/msn177"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cruz</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vilà</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Webster</span>  <span class="cit-name-given-names">M. T.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">The legacy of domestication: accumulation of deleterious mutations in the dog genome.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">2331</span>–<span class="cit-lpage">2336</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msn177</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsn177%26rft_id%253Dinfo%253Apmid%252F18689870%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msn177&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18689870&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3321.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000260152700010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3321.18" data-doi="10.1371/journal.pcbi.1001025"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Davydov</span>  <span class="cit-name-given-names">E. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goode</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sirota</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cooper</span>  <span class="cit-name-given-names">G. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sidow</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Identifying a high fraction of the human genome to be under selective constraint using GERP++.</span> <abbr class="cit-jnl-abbrev">PLOS Comput. Biol.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e1001025</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pcbi.1001025</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BComput%2BBiol%26rft.aulast%253DDavydov%26rft.auinit1%253DE.%2BV.%26rft.volume%253D6%26rft.issue%253D12%26rft.spage%253De1001025%26rft.epage%253De1001025%26rft.atitle%253DIdentifying%2Ba%2Bhigh%2Bfraction%2Bof%2Bthe%2Bhuman%2Bgenome%2Bto%2Bbe%2Bunder%2Bselective%2Bconstraint%2Busing%2BGERP%252B%252B.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.1001025%26rft_id%253Dinfo%253Apmid%252F21152010%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.1001025&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21152010&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3321.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3321.19" data-doi="10.1093/hmg/ddu733"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dong</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wei</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jian</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gibbs</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boerwinkle</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Comparison and integration of deleteriousness prediction methods for nonsynonymous SNVs in whole exome sequencing studies.</span> <abbr class="cit-jnl-abbrev">Hum. Mol. Genet.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">2125</span>–<span class="cit-lpage">2137</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/hmg/ddu733</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHum.%2BMol.%2BGenet.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fhmg%252Fddu733%26rft_id%253Dinfo%253Apmid%252F25552646%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/hmg/ddu733&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25552646&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3321.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3321.20" data-doi="10.1371/journal.pgen.1000183"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Doniger</span>  <span class="cit-name-given-names">S. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">H. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Swain</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Corcuera</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Williams</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">A catalog of neutral and deleterious polymorphism in yeast.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e1000183</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1000183</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DDoniger%26rft.auinit1%253DS.%2BW.%26rft.volume%253D4%26rft.issue%253D8%26rft.spage%253De1000183%26rft.epage%253De1000183%26rft.atitle%253DA%2Bcatalog%2Bof%2Bneutral%2Band%2Bdeleterious%2Bpolymorphism%2Bin%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000183%26rft_id%253Dinfo%253Apmid%252F18769710%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000183&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18769710&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3321.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3321.21" data-doi="10.1093/molbev/mss079"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dudley</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sanderford</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Butte</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Evolutionary meta-analysis of association studies reveals ancient constraints affecting disease marker discovery.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">2087</span>–<span class="cit-lpage">2094</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/mss079</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmss079%26rft_id%253Dinfo%253Apmid%252F22389448%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/mss079&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22389448&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3321.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000308851600001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-book" id="cit-8.10.3321.22" data-doi="10.1007/978-0-387-21822-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ewens</span>  <span class="cit-name-given-names">W. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-source"><em>Mathematical population genetics</em>.</span> <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">New York, NY</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/978-0-387-21822-9</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMathematical%2Bpopulation%2Bgenetics.%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-0-387-21822-9%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-0-387-21822-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3321.23"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fay</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wyckoff</span>  <span class="cit-name-given-names">G. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">C.-I.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Positive and negative selection on the human genome.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">158</span>: <span class="cit-fpage">1227</span>–<span class="cit-lpage">1234</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DFay%26rft.auinit1%253DJ.%2BC.%26rft.volume%253D158%26rft.issue%253D3%26rft.spage%253D1227%26rft.epage%253D1234%26rft.atitle%253DPositive%2Band%2BNegative%2BSelection%2Bon%2Bthe%2BHuman%2BGenome%26rft_id%253Dinfo%253Apmid%252F11454770%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1OC8zLzEyMjciO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzEwLzMzMjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3321.24" data-doi="10.1016/j.gde.2013.10.005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fay</span>  <span class="cit-name-given-names">J. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The molecular basis of phenotypic variation in yeast.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Genet. Dev.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">672</span>–<span class="cit-lpage">677</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.gde.2013.10.005</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BOpin.%2BGenet.%2BDev.%26rft.volume%253D23%26rft.spage%253D672%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.gde.2013.10.005%26rft_id%253Dinfo%253Apmid%252F24269094%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.gde.2013.10.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24269094&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3321.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3321.25" data-doi="10.1016/j.tree.2005.05.019"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Finlayson</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Biogeography and evolution of the genus <em>Homo</em>.</span> <abbr class="cit-jnl-abbrev">Trends Ecol. Evol.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">457</span>–<span class="cit-lpage">463</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.tree.2005.05.019</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2BEcology%2B%2526%2BEvolution%26rft.stitle%253DTrends%2Bin%2BEcology%2B%2526%2BEvolution%26rft.aulast%253DFinlayson%26rft.auinit1%253DC.%26rft.volume%253D20%26rft.issue%253D8%26rft.spage%253D457%26rft.epage%253D463%26rft.atitle%253DBiogeography%2Band%2Bevolution%2Bof%2Bthe%2Bgenus%2BHomo.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tree.2005.05.019%26rft_id%253Dinfo%253Apmid%252F16701417%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tree.2005.05.019&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16701417&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3321.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000231232900014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3321.26" data-doi="10.1111/j.1467-9868.2005.00503.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Friedman</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hastie</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tibshirani</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Regularization paths for generalized linear models via coordinate descent.</span> <abbr class="cit-jnl-abbrev">J. Stat. Softw.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">22</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.18637/jss.v033.i01</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DJ%2BStat%2BSoftw%26rft.aulast%253DFriedman%26rft.auinit1%253DJ.%26rft.volume%253D33%26rft.issue%253D1%26rft.spage%253D1%26rft.epage%253D22%26rft.atitle%253DRegularization%2BPaths%2Bfor%2BGeneralized%2BLinear%2BModels%2Bvia%2BCoordinate%2BDescent.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1467-9868.2005.00503.x%26rft_id%253Dinfo%253Apmid%252F20808728%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1467-9868.2005.00503.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20808728&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3321.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000275203200001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3321.27" data-doi="10.1016/j.ajhg.2011.03.004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">González-Pérez</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">López-Bigas</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Improving the assessment of the outcome of nonsynonymous SNVs with a consensus deleteriousness score, condel.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">88</span>: <span class="cit-fpage">440</span>–<span class="cit-lpage">449</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.ajhg.2011.03.004</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DGonzalez-Perez%26rft.auinit1%253DA.%26rft.volume%253D88%26rft.issue%253D4%26rft.spage%253D440%26rft.epage%253D449%26rft.atitle%253DImproving%2Bthe%2Bassessment%2Bof%2Bthe%2Boutcome%2Bof%2Bnonsynonymous%2BSNVs%2Bwith%2Ba%2Bconsensus%2Bdeleteriousness%2Bscore%252C%2BCondel.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2011.03.004%26rft_id%253Dinfo%253Apmid%252F21457909%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2011.03.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21457909&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3321.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3321.28" data-doi="10.1093/nar/gkr944"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goodstein</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Howson</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neupane</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Phytozome: a comparative platform for green plant genomics.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">D1178</span>–<span class="cit-lpage">D1186</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkr944</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkr944%26rft_id%253Dinfo%253Apmid%252F22110026%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkr944&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22110026&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3321.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000298601300176&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3321.29" data-doi="10.1002/humu.22768"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Grimm</span>  <span class="cit-name-given-names">D. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Azencott</span>  <span class="cit-name-given-names">C.-A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aicheler</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gieraths</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">MacArthur</span>  <span class="cit-name-given-names">D. G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">The evaluation of tools used to predict the impact of missense variants is hindered by two types of circularity.</span> <abbr class="cit-jnl-abbrev">Hum. Mutat.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">513</span>–<span class="cit-lpage">523</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/humu.22768</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHum.%2BMutat.%26rft.volume%253D36%26rft.spage%253D513%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fhumu.22768%26rft_id%253Dinfo%253Apmid%252F25684150%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/humu.22768&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25684150&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3321.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3321.30" data-doi="10.1007/s00122-010-1299-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Günther</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schmid</span>  <span class="cit-name-given-names">K. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Deleterious amino acid polymorphisms in <em>Arabidopsis thaliana</em> and rice.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">121</span>: <span class="cit-fpage">157</span>–<span class="cit-lpage">168</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-010-1299-4</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DGunther%26rft.auinit1%253DT.%26rft.volume%253D121%26rft.issue%253D1%26rft.spage%253D157%26rft.epage%253D168%26rft.atitle%253DDeleterious%2Bamino%2Bacid%2Bpolymorphisms%2Bin%2BArabidopsis%2Bthaliana%2Band%2Brice.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-010-1299-4%26rft_id%253Dinfo%253Apmid%252F20198468%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-010-1299-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20198468&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3321.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3321.31" data-doi="10.1126/science.1209244"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hancock</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brachi</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Faure</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Horton</span>  <span class="cit-name-given-names">M. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jarymowycz</span>  <span class="cit-name-given-names">L. B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Adaptation to climate across the <em>Arabidopsis thaliana</em> genome.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">334</span>: <span class="cit-fpage">83</span>–<span class="cit-lpage">86</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.1209244</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DHancock%26rft.auinit1%253DA.%2BM.%26rft.volume%253D334%26rft.issue%253D6052%26rft.spage%253D83%26rft.epage%253D86%26rft.atitle%253DAdaptation%2Bto%2BClimate%2BAcross%2Bthe%2BArabidopsis%2Bthaliana%2BGenome%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1209244%26rft_id%253Dinfo%253Apmid%252F21980108%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjExOiIzMzQvNjA1Mi84MyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzgvMTAvMzMyMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3321.32" data-doi="10.1073/pnas.1510805112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Henn</span>  <span class="cit-name-given-names">B. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Botigué</span>  <span class="cit-name-given-names">L. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peischl</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dupanloup</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lipatov</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Distance from sub-Saharan Africa predicts mutational load in diverse human genomes.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">113</span>: <span class="cit-fpage">E440</span>–<span class="cit-lpage">E449</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1510805112</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1510805112%26rft_id%253Dinfo%253Apmid%252F26712023%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTEzLzQvRTQ0MCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzgvMTAvMzMyMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3321.33" data-doi="10.1002/humu.21490"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hicks</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wheeler</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Plon</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kimmel</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Prediction of missense mutation functionality depends on both the algorithm and sequence alignment employed.</span> <abbr class="cit-jnl-abbrev">Hum. Mutat.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">661</span>–<span class="cit-lpage">668</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/humu.21490</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHuman%2Bmutation%26rft.stitle%253DHum%2BMutat%26rft.aulast%253DHicks%26rft.auinit1%253DS.%26rft.volume%253D32%26rft.issue%253D6%26rft.spage%253D661%26rft.epage%253D668%26rft.atitle%253DPrediction%2Bof%2Bmissense%2Bmutation%2Bfunctionality%2Bdepends%2Bon%2Bboth%2Bthe%2Balgorithm%2Band%2Bsequence%2Balignment%2Bemployed.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fhumu.21490%26rft_id%253Dinfo%253Apmid%252F21480434%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/humu.21490&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21480434&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3321.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3321.34" data-doi="10.1046/j.1365-2699.2002.00647.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hoffmann</span>  <span class="cit-name-given-names">M. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Biogeography of <em>Arabidopsis thaliana</em> L. Heynh. (Brassicaceae).</span> <abbr class="cit-jnl-abbrev">J. Biogeogr.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">125</span>–<span class="cit-lpage">134</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1046/j.1365-2699.2002.00647.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiogeogr.%26rft.volume%253D29%26rft.spage%253D125%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1365-2699.2002.00647.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1365-2699.2002.00647.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000173919700010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3321.35" data-doi="10.1038/nature14497"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jordan</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Frangakis</span>  <span class="cit-name-given-names">S. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Golzio</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cassa</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kurtzberg</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Identification of cis-suppression of human disease mutations by comparative genomics.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">524</span>: <span class="cit-fpage">225</span>–<span class="cit-lpage">229</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature14497</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D524%26rft.spage%253D225%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature14497%26rft_id%253Dinfo%253Apmid%252F26123021%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature14497&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26123021&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3321.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3321.36" data-doi="10.1038/ng.2892"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kircher</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Witten</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jain</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">O’Roak</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cooper</span>  <span class="cit-name-given-names">G. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A general framework for estimating the relative pathogenicity of human genetic variants.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">310</span>–<span class="cit-lpage">315</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.2892</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D46%26rft.spage%253D310%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2892%26rft_id%253Dinfo%253Apmid%252F24487276%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2892&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24487276&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3321.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-other" id="cit-8.10.3321.37"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kondrashov</span>  <span class="cit-name-given-names">F. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rogozin</span>  <span class="cit-name-given-names">I. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wolf</span>  <span class="cit-name-given-names">Y. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koonin</span>  <span class="cit-name-given-names">E. V.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> Selection in the evolution of gene duplications. Genome Biol. 3: research0008.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3321.38" data-doi="10.1093/molbev/msw102"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kono</span>  <span class="cit-name-given-names">T. J. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mohammadi</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoffman</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">The role of deleterious substitutions in crop genomes.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">2307</span>–<span class="cit-lpage">2317</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msw102</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsw102%26rft_id%253Dinfo%253Apmid%252F27301592%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msw102&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27301592&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3321.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3321.39" data-doi="10.1371/journal.pgen.1000144"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kosiol</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vinař</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">da Fonseca</span>  <span class="cit-name-given-names">R. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hubisz</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bustamante</span>  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Patterns of positive selection in six mammalian genomes.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e1000144</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1000144</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DKosiol%26rft.auinit1%253DC.%26rft.volume%253D4%26rft.issue%253D8%26rft.spage%253De1000144%26rft.epage%253De1000144%26rft.atitle%253DPatterns%2Bof%2Bpositive%2Bselection%2Bin%2Bsix%2BMammalian%2Bgenomes.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000144%26rft_id%253Dinfo%253Apmid%252F18670650%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000144&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18670650&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3321.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3321.40" data-doi="10.1371/journal.pbio.1001388"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Leffler</span>  <span class="cit-name-given-names">E. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bullaughey</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Matute</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meyer</span>  <span class="cit-name-given-names">W. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ségurel</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Revisiting an old riddle: what determines genetic diversity levels within species?</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">e1001388</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pbio.1001388</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DLeffler%26rft.auinit1%253DE.%2BM.%26rft.volume%253D10%26rft.issue%253D9%26rft.spage%253De1001388%26rft.epage%253De1001388%26rft.atitle%253DRevisiting%2Ban%2Bold%2Briddle%253A%2Bwhat%2Bdetermines%2Bgenetic%2Bdiversity%2Blevels%2Bwithin%2Bspecies%253F%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.1001388%26rft_id%253Dinfo%253Apmid%252F22984349%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.1001388&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22984349&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3321.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3321.41" data-doi="10.1093/bioinformatics/btp528"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krishnan</span>  <span class="cit-name-given-names">V. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mort</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xin</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kamati</span>  <span class="cit-name-given-names">K. K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Automated inference of molecular mechanisms of disease from amino acid substitutions.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">2744</span>–<span class="cit-lpage">2750</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btp528</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp528%26rft_id%253Dinfo%253Apmid%252F19734154%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp528&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19734154&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3321.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000271106800002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3321.42" data-doi="10.1093/molbev/msw296"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morrell</span>  <span class="cit-name-given-names">P. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gaut</span>  <span class="cit-name-given-names">B. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Deleterious variants in Asian rice and the potential cost of domestication.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">908</span>–<span class="cit-lpage">924</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msw296</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsw296%26rft_id%253Dinfo%253Apmid%252F28087781%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msw296&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28087781&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3321.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3321.43" data-doi="10.1016/j.tig.2004.11.013"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lockton</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gaut</span>  <span class="cit-name-given-names">B. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Plant conserved non-coding sequences and paralogue evolution.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">60</span>–<span class="cit-lpage">65</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.tig.2004.11.013</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DLockton%26rft.auinit1%253DS.%26rft.volume%253D21%26rft.issue%253D1%26rft.spage%253D60%26rft.epage%253D65%26rft.atitle%253DPlant%2Bconserved%2Bnon-coding%2Bsequences%2Band%2Bparalogue%2Bevolution.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2004.11.013%26rft_id%253Dinfo%253Apmid%252F15680516%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tig.2004.11.013&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15680516&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3321.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000226485800013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3321.44" data-doi="10.1038/nature06611"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lohmueller</span>  <span class="cit-name-given-names">K. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Indap</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schmidt</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boyko</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hernandez</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Proportionally more deleterious genetic variation in European than in African populations.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">451</span>: <span class="cit-fpage">994</span>–<span class="cit-lpage">997</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature06611</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DLohmueller%26rft.auinit1%253DK.%2BE.%26rft.volume%253D451%26rft.issue%253D7181%26rft.spage%253D994%26rft.epage%253D997%26rft.atitle%253DProportionally%2Bmore%2Bdeleterious%2Bgenetic%2Bvariation%2Bin%2BEuropean%2Bthan%2Bin%2BAfrican%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature06611%26rft_id%253Dinfo%253Apmid%252F18288194%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature06611&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18288194&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3321.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000253313100049&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3321.45" data-doi="10.1016/j.tig.2006.01.004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tang</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shi</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">The accumulation of deleterious mutations in rice genomes: a hypothesis on the cost of domestication.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">126</span>–<span class="cit-lpage">131</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.tig.2006.01.004</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DLu%26rft.auinit1%253DJ.%26rft.volume%253D22%26rft.issue%253D3%26rft.spage%253D126%26rft.epage%253D131%26rft.atitle%253DThe%2Baccumulation%2Bof%2Bdeleterious%2Bmutations%2Bin%2Brice%2Bgenomes%253A%2Ba%2Bhypothesis%2Bon%2Bthe%2Bcost%2Bof%2Bdomestication.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2006.01.004%26rft_id%253Dinfo%253Apmid%252F16443304%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tig.2006.01.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16443304&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3321.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000236534100003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3321.46" data-doi="10.1126/science.290.5494.1151"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Conery</span>  <span class="cit-name-given-names">J. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">The evolutionary fate and consequences of duplicate genes.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">290</span>: <span class="cit-fpage">1151</span>–<span class="cit-lpage">1155</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.290.5494.1151</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DLynch%26rft.auinit1%253DM.%26rft.volume%253D290%26rft.issue%253D5494%26rft.spage%253D1151%26rft.epage%253D1155%26rft.atitle%253DThe%2BEvolutionary%2BFate%2Band%2BConsequences%2Bof%2BDuplicate%2BGenes%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.290.5494.1151%26rft_id%253Dinfo%253Apmid%252F11073452%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyOTAvNTQ5NC8xMTUxIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMC8zMzIxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3321.47" data-doi="10.1038/nature08494"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Manolio</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Collins</span>  <span class="cit-name-given-names">F. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cox</span>  <span class="cit-name-given-names">N. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goldstein</span>  <span class="cit-name-given-names">D. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hindorff</span>  <span class="cit-name-given-names">L. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Finding the missing heritability of complex diseases.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">461</span>: <span class="cit-fpage">747</span>–<span class="cit-lpage">753</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature08494</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DManolio%26rft.auinit1%253DT.%2BA.%26rft.volume%253D461%26rft.issue%253D7265%26rft.spage%253D747%26rft.epage%253D753%26rft.atitle%253DFinding%2Bthe%2Bmissing%2Bheritability%2Bof%2Bcomplex%2Bdiseases.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature08494%26rft_id%253Dinfo%253Apmid%252F19812666%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature08494&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19812666&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3321.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000270547500027&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3321.48" data-doi="10.1371/journal.pgen.1000968"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Marini</span>  <span class="cit-name-given-names">N. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thomas</span>  <span class="cit-name-given-names">P. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rine</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The use of orthologous sequences to predict the impact of amino acid substitutions on protein function.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e1000968</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1000968</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DMarini%26rft.auinit1%253DN.%2BJ.%26rft.volume%253D6%26rft.issue%253D5%26rft.spage%253De1000968%26rft.epage%253De1000968%26rft.atitle%253DThe%2Buse%2Bof%2Borthologous%2Bsequences%2Bto%2Bpredict%2Bthe%2Bimpact%2Bof%2Bamino%2Bacid%2Bsubstitutions%2Bon%2Bprotein%2Bfunction.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000968%26rft_id%253Dinfo%253Apmid%252F20523748%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000968&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20523748&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3321.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3321.49" data-doi="10.1534/g3.113.008870"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mezmouk</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ross-Ibarra</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> The pattern and distribution of deleterious mutations in maize. G3 (Bethesda)4: <span class="cit-fpage">163</span>–<span class="cit-lpage">171</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.113.008870</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.113.008870%26rft_id%253Dinfo%253Apmid%252F24281428%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjc6IjQvMS8xNjMiO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzEwLzMzMjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3321.50" data-doi="10.1093/hmg/10.21.2319"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Understanding human disease mutations through the use of interspecific genetic variation.</span> <abbr class="cit-jnl-abbrev">Hum. Mol. Genet.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">2319</span>–<span class="cit-lpage">2328</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/hmg/10.21.2319</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHuman%2BMolecular%2BGenetics%26rft.stitle%253DHum%2BMol%2BGenet%26rft.aulast%253DMiller%26rft.auinit1%253DM.%2BP.%26rft.volume%253D10%26rft.issue%253D21%26rft.spage%253D2319%26rft.epage%253D2328%26rft.atitle%253DUnderstanding%2Bhuman%2Bdisease%2Bmutations%2Bthrough%2Bthe%2Buse%2Bof%2Binterspecific%2Bgenetic%2Bvariation%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fhmg%252F10.21.2319%26rft_id%253Dinfo%253Apmid%252F11689479%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/hmg/10.21.2319&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11689479&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3321.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000171946100001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3321.51" data-doi="10.1073/pnas.1511585112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Miosge</span>  <span class="cit-name-given-names">L. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Field</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sontani</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cho</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Comparison of predicted and actual consequences of missense mutations.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">112</span>: <span class="cit-fpage">E5189</span>–<span class="cit-lpage">E5198</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1511585112</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1511585112%26rft_id%253Dinfo%253Apmid%252F26269570%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTEyLzM3L0U1MTg5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMC8zMzIxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3321.52" data-doi="10.1089/cmb.2014.0156"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mirarab</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nguyen</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">L.-S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">PASTA: ultra-large multiple sequence alignment for nucleotide and amino-acid sequences.</span> <abbr class="cit-jnl-abbrev">J. Comput. Biol.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">377</span>–<span class="cit-lpage">386</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1089/cmb.2014.0156</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BComput.%2BBiol.%26rft.volume%253D22%26rft.spage%253D377%26rft_id%253Dinfo%253Adoi%252F10.1089%252Fcmb.2014.0156%26rft_id%253Dinfo%253Apmid%252F25549288%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1089/cmb.2014.0156&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25549288&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3321.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3321.53" data-doi="10.1038/nrg3097"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Morrell</span>  <span class="cit-name-given-names">P. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buckler</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ross-Ibarra</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Crop genomics: advances and applications.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">85</span>–<span class="cit-lpage">96</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg3097</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BGenet.%26rft.volume%253D13%26rft.spage%253D85%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3097%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3097&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3321.54" data-doi="10.1093/jhered/esx069"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Moyers</span>  <span class="cit-name-given-names">B. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morrell</span>  <span class="cit-name-given-names">P. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McKay</span>  <span class="cit-name-given-names">J. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Genetic costs of domestication and improvement.</span> <abbr class="cit-jnl-abbrev">J. Hered.</abbr> <span class="cit-vol">109</span>: <span class="cit-fpage">103</span>–<span class="cit-lpage">116</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/jhered/esx069</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BHered.%26rft.volume%253D109%26rft.spage%253D103%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjhered%252Fesx069%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/jhered/esx069&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3321.55" data-doi="10.1093/nar/gkg509"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ng</span>  <span class="cit-name-given-names">P. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Henikoff</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">SIFT: predicting amino acid changes that affect protein function.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">3812</span>–<span class="cit-lpage">3814</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkg509</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DNg%26rft.auinit1%253DP.%2BC.%26rft.volume%253D31%26rft.issue%253D13%26rft.spage%253D3812%26rft.epage%253D3814%26rft.atitle%253DSIFT%253A%2Bpredicting%2Bamino%2Bacid%2Bchanges%2Bthat%2Baffect%2Bprotein%2Bfunction%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkg509%26rft_id%253Dinfo%253Apmid%252F12824425%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkg509&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12824425&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3321.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000183832900117&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3321.56" data-doi="10.1146/annurev.genom.7.080505.115630"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ng</span>  <span class="cit-name-given-names">P. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Henikoff</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Predicting the effects of amino acid substitutions on protein function.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Genomics Hum. Genet.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">61</span>–<span class="cit-lpage">80</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1146/annurev.genom.7.080505.115630</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bgenomics%2Band%2Bhuman%2Bgenetics%26rft.stitle%253DAnnu%2BRev%2BGenomics%2BHum%2BGenet%26rft.aulast%253DNg%26rft.auinit1%253DP.%2BC.%26rft.volume%253D7%26rft.spage%253D61%26rft.epage%253D80%26rft.atitle%253DPredicting%2Bthe%2Beffects%2Bof%2Bamino%2Bacid%2Bsubstitutions%2Bon%2Bprotein%2Bfunction.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.genom.7.080505.115630%26rft_id%253Dinfo%253Apmid%252F16824020%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.genom.7.080505.115630&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16824020&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3321.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000242032200003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-book" id="cit-8.10.3321.57" data-doi="10.1007/978-3-642-86659-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ohno</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1970</span> <span class="cit-source">Evolution by gene duplication</span>. <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">Berlin</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/978-3-642-86659-3</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%2Bby%2Bgene%2Bduplication%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-3-642-86659-3%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-3-642-86659-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3321.58" data-doi="10.1002/humu.22102"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Olatubosun</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Väliaho</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Härkönen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thusberg</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vihinen</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">PON-P: integrated predictor for pathogenicity of missense variants.</span> <abbr class="cit-jnl-abbrev">Hum. Mutat.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">1166</span>–<span class="cit-lpage">1174</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/humu.22102</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHuman%2Bmutation%26rft.stitle%253DHum%2BMutat%26rft.aulast%253DOlatubosun%26rft.auinit1%253DA.%26rft.volume%253D33%26rft.issue%253D8%26rft.spage%253D1166%26rft.epage%253D1174%26rft.atitle%253DPON-P%253A%2Bintegrated%2Bpredictor%2Bfor%2Bpathogenicity%2Bof%2Bmissense%2Bvariants.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fhumu.22102%26rft_id%253Dinfo%253Apmid%252F22505138%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/humu.22102&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22505138&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3321.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3321.59" data-doi="10.1101/gr.097857.109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pollard</span>  <span class="cit-name-given-names">K. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hubisz</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosenbloom</span>  <span class="cit-name-given-names">K. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Siepel</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Detection of nonneutral substitution rates on mammalian phylogenies.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">110</span>–<span class="cit-lpage">121</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.097857.109</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.097857.109%26rft_id%253Dinfo%253Apmid%252F19858363%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjIwLzEvMTEwIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMC8zMzIxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3321.60" data-doi="10.1093/bioinformatics/bti079"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pond</span>  <span class="cit-name-given-names">S. L. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Frost</span>  <span class="cit-name-given-names">S. D. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Muse</span>  <span class="cit-name-given-names">S. V.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">HyPhy: hypothesis testing using phylogenies.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">676</span>–<span class="cit-lpage">679</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/bti079</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbti079%26rft_id%253Dinfo%253Apmid%252F15509596%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bti079&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15509596&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3321.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000227241200017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3321.61" data-doi="10.1534/genetics.104.037259"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Poon</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">B. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chao</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">The coupon collector and the suppressor mutation: estimating the number of compensatory mutations by maximum likelihood.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">170</span>: <span class="cit-fpage">1323</span>–<span class="cit-lpage">1332</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.104.037259</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.104.037259%26rft_id%253Dinfo%253Apmid%252F15879511%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3MC8zLzEzMjMiO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzEwLzMzMjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3321.62" data-doi="10.1093/molbev/msv106"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Renaut</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rieseberg</span>  <span class="cit-name-given-names">L. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">The accumulation of deleterious mutations as a consequence of domestication and improvement in sunflowers and other compositae crops.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">2273</span>–<span class="cit-lpage">2283</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msv106</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsv106%26rft_id%253Dinfo%253Apmid%252F25939650%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msv106&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25939650&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3321.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3321.63" data-doi="10.1186/1471-2105-12-77"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Robin</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Turck</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hainard</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tiberti</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lisacek</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">pROC: an open-source package for R and S+ to analyze and compare ROC curves.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">77</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2105-12-77</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DRobin%26rft.auinit1%253DX.%26rft.volume%253D12%26rft.spage%253D77%26rft.epage%253D77%26rft.atitle%253DpROC%253A%2Ban%2Bopen-source%2Bpackage%2Bfor%2BR%2Band%2BS%252B%2Bto%2Banalyze%2Band%2Bcompare%2BROC%2Bcurves.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-12-77%26rft_id%253Dinfo%253Apmid%252F21414208%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-12-77&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21414208&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3321.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3321.64" data-doi="10.1073/pnas.1413864112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rodgers-Melnick</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bradbury</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Elshire</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glaubitz</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Acharya</span>  <span class="cit-name-given-names">C. B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Recombination in diverse maize is stable, predictable, and associated with genetic load.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">112</span>: <span class="cit-fpage">3823</span>–<span class="cit-lpage">3828</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1413864112</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1413864112%26rft_id%253Dinfo%253Apmid%252F25775595%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTEyLzEyLzM4MjMiO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzEwLzMzMjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3321.65" data-doi="10.1038/nmeth0810-575"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schwarz</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rödelsperger</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schuelke</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seelow</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">MutationTaster evaluates disease-causing potential of sequence alterations.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">575</span>–<span class="cit-lpage">576</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nmeth0810-575</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DSchwarz%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D7%26rft.issue%253D8%26rft.spage%253D575%26rft.epage%253D576%26rft.atitle%253DMutationTaster%2Bevaluates%2Bdisease-causing%2Bpotential%2Bof%2Bsequence%2Balterations.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth0810-575%26rft_id%253Dinfo%253Apmid%252F20676075%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth0810-575&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20676075&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3321.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000280500000014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3321.66" data-doi="10.1038/ng.2896"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Simons</span>  <span class="cit-name-given-names">Y. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Turchin</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pritchard</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sella</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The deleterious mutation load is insensitive to recent population history.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">220</span>–<span class="cit-lpage">224</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.2896</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D46%26rft.spage%253D220%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2896%26rft_id%253Dinfo%253Apmid%252F24509481%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2896&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24509481&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3321.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3321.67" data-doi="10.1038/ng.2669"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Slotte</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hazzouri</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ågren</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koenig</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maumus</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The <em>Capsella rubella</em> genome and the genomic consequences of rapid mating system evolution.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">831</span>–<span class="cit-lpage">835</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.2669</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D45%26rft.spage%253D831%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2669%26rft_id%253Dinfo%253Apmid%252F23749190%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2669&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23749190&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3321.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3321.68" data-doi="10.1007/s00439-013-1358-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stenson</span>  <span class="cit-name-given-names">P. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mort</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ball</span>  <span class="cit-name-given-names">E. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shaw</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Phillips</span>  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The human gene mutation database: building a comprehensive mutation repository for clinical and molecular genetics, diagnostic testing and personalized genomic medicine.</span> <abbr class="cit-jnl-abbrev">Hum. Genet.</abbr> <span class="cit-vol">133</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">9</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00439-013-1358-4</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHum.%2BGenet.%26rft.volume%253D133%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00439-013-1358-4%26rft_id%253Dinfo%253Apmid%252F24077912%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00439-013-1358-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24077912&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3321.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3321.69" data-doi="10.1101/gr.3804205"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stone</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sidow</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Physicochemical constraint violation by missense substitutions mediates impairment of protein function and disease severity.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">978</span>–<span class="cit-lpage">986</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.3804205</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.3804205%26rft_id%253Dinfo%253Apmid%252F15965030%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE1LzcvOTc4IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMC8zMzIxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3321.70" data-doi="10.1093/protein/12.5.387"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sunyaev</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eisenhaber</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rodchenkov</span>  <span class="cit-name-given-names">I. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eisenhaber</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tumanyan</span>  <span class="cit-name-given-names">V. G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">PSIC: profile extraction from sequence alignments with position-specific counts of independent observations.</span> <abbr class="cit-jnl-abbrev">Protein Eng.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">387</span>–<span class="cit-lpage">394</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/protein/12.5.387</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProtein%2BEngineering%2BDesign%2Band%2BSelection%26rft.stitle%253DProtein%2BEng%2BDes%2BSel%26rft.aulast%253DSunyaev%26rft.auinit1%253DS.%2BR.%26rft.volume%253D12%26rft.issue%253D5%26rft.spage%253D387%26rft.epage%253D394%26rft.atitle%253DPSIC%253A%2Bprofile%2Bextraction%2Bfrom%2Bsequence%2Balignments%2Bwith%2Bposition-specific%2Bcounts%2Bof%2Bindependent%2Bobservations%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fprotein%252F12.5.387%26rft_id%253Dinfo%253Apmid%252F10360979%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/protein/12.5.387&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10360979&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3321.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000080964300004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3321.71" data-doi="10.1038/35048692"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">The Arabidopsis Genome Initiative</span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Analysis of the genome sequence of the flowering plant <em>Arabidopsis thaliana</em>.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">408</span>: <span class="cit-fpage">796</span>–<span class="cit-lpage">815</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/35048692</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.volume%253D408%26rft.issue%253D6814%26rft.spage%253D796%26rft.epage%253D815%26rft.atitle%253DAnalysis%2Bof%2Bthe%2Bgenome%2Bsequence%2Bof%2Bthe%2Bflowering%2Bplant%2BArabidopsis%2Bthaliana.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F35048692%26rft_id%253Dinfo%253Apmid%252F11130711%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/35048692&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11130711&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3321.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000165831300037&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference  in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3321.72" data-doi="10.1371/journal.pgen.1003258"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thornton</span>  <span class="cit-name-given-names">K. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Foran</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Long</span>  <span class="cit-name-given-names">A. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Properties and modeling of GWAS when complex disease risk is due to non-complementing, deleterious mutations in genes of large effect.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e1003258</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1003258</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D9%26rft.spage%253De1003258%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003258%26rft_id%253Dinfo%253Apmid%252F23437004%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003258&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23437004&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3321.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference  in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3321.73" data-doi="10.1002/humu.21445"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thusberg</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Olatubosun</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vihinen</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Performance of mutation pathogenicity prediction methods on missense variants.</span> <abbr class="cit-jnl-abbrev">Hum. Mutat.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">358</span>–<span class="cit-lpage">368</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/humu.21445</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHuman%2Bmutation%26rft.stitle%253DHum%2BMutat%26rft.aulast%253DThusberg%26rft.auinit1%253DJ.%26rft.volume%253D32%26rft.issue%253D4%26rft.spage%253D358%26rft.epage%253D368%26rft.atitle%253DPerformance%2Bof%2Bmutation%2Bpathogenicity%2Bprediction%2Bmethods%2Bon%2Bmissense%2Bvariants.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fhumu.21445%26rft_id%253Dinfo%253Apmid%252F21412949%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/humu.21445&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21412949&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3321.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000288464100008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference  in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3321.74" data-doi="10.1038/nprot.2015.123"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vaser</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Adusumalli</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leng</span>  <span class="cit-name-given-names">S. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sikic</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ng</span>  <span class="cit-name-given-names">P. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">SIFT missense predictions for genomes.</span> <abbr class="cit-jnl-abbrev">Nat. Protoc.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">9</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nprot.2015.123</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BProtoc.%26rft.volume%253D11%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnprot.2015.123%26rft_id%253Dinfo%253Apmid%252F26633127%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nprot.2015.123&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26633127&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3321.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference  in text" id="ref-75">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3321.75" data-doi="10.1371/journal.pgen.1007019"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mezmouk</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baumgarten</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buckler</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guill</span>  <span class="cit-name-given-names">K. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Incomplete dominance of deleterious alleles contributes substantially to trait variation and heterosis in maize.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">e1007019</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1007019</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D13%26rft.spage%253De1007019%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1007019%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1007019&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-76-1" title="View reference  in text" id="ref-76">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3321.76" data-doi="10.1093/molbev/msw169"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bawa</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Suren</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holliday</span>  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Recombination rate variation, hitchhiking, and demographic history shape deleterious load in poplar.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">2899</span>–<span class="cit-lpage">2910</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsw169%26rft_id%253Dinfo%253Apmid%252F27512114%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msw169&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27512114&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3321.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/8/10/3321.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/10/3311" title="Early Diverging Insect-Pathogenic Fungi of the Order Entomophthorales Possess Diverse and Unique Subtilisin-Like Serine Proteases" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/10/3331" title="A Novel Mutation in Brain Tumor Causes Both Neural Over-Proliferation and Neurodegeneration in Adult Drosophila" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/8/10"><span>Volume 8 Issue 10, October 2018</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/8/10.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/8/10.cover-source.jpg?itok=p8rfSNcT" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 8 (10)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/10/3321.abstract" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-CGXm8C7UW9PqnziI2Szv2TVHKc---kGnyuniSabllzk" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/8/10/3321" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/8/10/3321" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/8/10/3321" class="active" data-icon-position="" data-hide-link-title="0">Comparative Genomics Approaches Accurately Predict Deleterious Variants in Plants</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/490979" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-2fjfEaSoNuWzzlgT6d76hx1Z9A5M7B3LWqCgTvhwLfA" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/10/3321.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/8/10/3321" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-sn1LeEhT1aQYYCHjYBFcOfuXpl95rwdaig_vIvf4Pmg" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/8/10/3321" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/10/3321.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490979" id="node490979--2" data-pisa="ggg;8/10/3321" data-pisa-master="ggg;g3.118.200563" data-apath="/ggg/8/10/3321.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Comparative Genomics Approaches Accurately Predict Deleterious Variants in Plants</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-1388-3194" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Thomas J.Y. Kono</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-5708-0118" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Li Lei</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-6652-1197" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Ching-Hua Shih</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-7693-8957" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Paul J. Hoffman</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0001-6282-1582" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Peter L. Morrell</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0003-1893-877X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Justin C. Fay</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3321-3329;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200563 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node490979--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Thomas J.Y. Kono </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy & Plant Genetics, University of Minnesota, St. Paul, MN 551085</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Thomas%2BJ.Y.%2BKono%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kono%20TJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AThomas%2BJ.Y.%2BKono%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-1388-3194" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Thomas J.Y. Kono</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Li Lei </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy & Plant Genetics, University of Minnesota, St. Paul, MN 551085</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Li%2BLei%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lei%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALi%2BLei%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-5708-0118" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Li Lei</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Ching-Hua Shih </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, Washington University, St. Louis, MO 63110</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ching-Hua%2BShih%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Shih%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AChing-Hua%2BShih%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-6652-1197" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Ching-Hua Shih</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Paul J. Hoffman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy & Plant Genetics, University of Minnesota, St. Paul, MN 551085</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Paul%2BJ.%2BHoffman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hoffman%20PJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APaul%2BJ.%2BHoffman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-7693-8957" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Paul J. Hoffman</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Peter L. Morrell </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy & Plant Genetics, University of Minnesota, St. Paul, MN 551085</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Peter%2BL.%2BMorrell%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Morrell%20PL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APeter%2BL.%2BMorrell%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-6282-1582" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Peter L. Morrell</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:justin.fay@rochester.edu" class="" data-icon-position="" data-hide-link-title="0">justin.fay@rochester.edu</a>
<a href="mailto:pmorrell@umn.edu" class="" data-icon-position="" data-hide-link-title="0">pmorrell@umn.edu</a></span></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Justin C. Fay </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, Washington University, St. Louis, MO 63110</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Justin%2BC.%2BFay%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fay%20JC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJustin%2BC.%2BFay%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-1893-877X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Justin C. Fay</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:justin.fay@rochester.edu" class="" data-icon-position="" data-hide-link-title="0">justin.fay@rochester.edu</a>
<a href="mailto:pmorrell@umn.edu" class="" data-icon-position="" data-hide-link-title="0">pmorrell@umn.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/8/10/3321" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/8/10/3321&amp;title=Comparative%20Genomics%20Approaches%20Accurately%20Predict%20Deleterious%20Variants%20in%20Plants" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/8/10/3321&amp;title=Comparative%20Genomics%20Approaches%20Accurately%20Predict%20Deleterious%20Variants%20in%20Plants" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/8/10/3321&amp;title=Comparative%20Genomics%20Approaches%20Accurately%20Predict%20Deleterious%20Variants%20in%20Plants" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/8/10/3321&amp;text=Comparative%20Genomics%20Approaches%20Accurately%20Predict%20Deleterious%20Variants%20in%20Plants" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/8/10/3321&amp;title=Comparative%20Genomics%20Approaches%20Accurately%20Predict%20Deleterious%20Variants%20in%20Plants" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/8/10/3321&amp;t=Comparative%20Genomics%20Approaches%20Accurately%20Predict%20Deleterious%20Variants%20in%20Plants" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/8/10/3321&amp;title=Comparative%20Genomics%20Approaches%20Accurately%20Predict%20Deleterious%20Variants%20in%20Plants" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/8/10/3321&amp;title=Comparative%20Genomics%20Approaches%20Accurately%20Predict%20Deleterious%20Variants%20in%20Plants" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/490979/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490979" id="node490979--3" data-pisa="ggg;8/10/3321" data-pisa-master="ggg;g3.118.200563" data-apath="/ggg/8/10/3321.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Comparative Genomics Approaches Accurately Predict Deleterious Variants in Plants</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-1388-3194" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Thomas J.Y. Kono</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-5708-0118" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Li Lei</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-6652-1197" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Ching-Hua Shih</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-7693-8957" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Paul J. Hoffman</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0001-6282-1582" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Peter L. Morrell</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0003-1893-877X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Justin C. Fay</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3321-3329;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200563 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node490979--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Thomas J.Y. Kono </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy & Plant Genetics, University of Minnesota, St. Paul, MN 551085</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Thomas%2BJ.Y.%2BKono%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kono%20TJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AThomas%2BJ.Y.%2BKono%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-1388-3194" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Thomas J.Y. Kono</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Li Lei </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy & Plant Genetics, University of Minnesota, St. Paul, MN 551085</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Li%2BLei%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lei%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALi%2BLei%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-5708-0118" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Li Lei</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Ching-Hua Shih </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, Washington University, St. Louis, MO 63110</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ching-Hua%2BShih%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Shih%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AChing-Hua%2BShih%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-6652-1197" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Ching-Hua Shih</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Paul J. Hoffman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy & Plant Genetics, University of Minnesota, St. Paul, MN 551085</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Paul%2BJ.%2BHoffman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hoffman%20PJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APaul%2BJ.%2BHoffman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-7693-8957" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Paul J. Hoffman</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Peter L. Morrell </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy & Plant Genetics, University of Minnesota, St. Paul, MN 551085</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Peter%2BL.%2BMorrell%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Morrell%20PL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APeter%2BL.%2BMorrell%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-6282-1582" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Peter L. Morrell</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:justin.fay@rochester.edu" class="" data-icon-position="" data-hide-link-title="0">justin.fay@rochester.edu</a>
<a href="mailto:pmorrell@umn.edu" class="" data-icon-position="" data-hide-link-title="0">pmorrell@umn.edu</a></span></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Justin C. Fay </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, Washington University, St. Louis, MO 63110</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Justin%2BC.%2BFay%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fay%20JC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJustin%2BC.%2BFay%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-1893-877X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Justin C. Fay</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:justin.fay@rochester.edu" class="" data-icon-position="" data-hide-link-title="0">justin.fay@rochester.edu</a>
<a href="mailto:pmorrell@umn.edu" class="" data-icon-position="" data-hide-link-title="0">pmorrell@umn.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=3321&amp;rft.epage=3329&amp;rft.atitle=Comparative%20Genomics%20Approaches%20Accurately%20Predict%20Deleterious%20Variants%20in%20Plants&amp;rft.volume=8&amp;rft.issue=10&amp;rft.date=2018-10-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Kono%2C+Thomas+J.Y.&amp;rft.au=Lei%2C+Li&amp;rft.au=Shih%2C+Ching-Hua&amp;rft.au=Hoffman%2C+Paul+J.&amp;rft.au=Morrell%2C+Peter+L.&amp;rft.au=Fay%2C+Justin+C."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/490979/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/490979/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/490979/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/490979/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/490979/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/490979/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/490979/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/490979/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/490979/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/490979/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/490979/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/490979/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/8/10/3321" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-6" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-11" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/8/10/3321.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/8/10/3321.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/8/10/3321.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/8/10/3321.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
