<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/8/10/3231" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/8/10/3231.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/8/10/3231.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/8/10/3231.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/8/10/3231/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/8/10/3231.atom" />
<meta name="HW.pisa" content="ggg;8/10/3231" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Recursive Algorithms for Modeling Genomic Ancestral Origins in a Fixed Pedigree" />
<meta name="DC.Identifier" content="10.1534/g3.118.200340" />
<meta name="DC.Date" content="2018-10-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2018 Zheng et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="The study of gene flow in pedigrees is of strong interest for the development of quantitative trait loci (QTL) mapping methods in multiparental populations. We developed a Markovian framework for modeling ancestral origins along two homologous chromosomes within individuals in fixed pedigrees. A highly beneficial property of our method is that the size of state space depends linearly or quadratically on the number of pedigree founders, whereas this increases exponentially with pedigree size in alternative methods. To calculate the parameter values of the Markov process, we describe two novel recursive algorithms that differ with respect to the pedigree founders being assumed to be exchangeable or not. Our algorithms apply equally to autosomes and sex chromosomes, another desirable feature of our approach. We tested the accuracy of the algorithms by a million simulations on a pedigree. We demonstrated two applications of the recursive algorithms in multiparental populations: design a breeding scheme for maximizing the overall density of recombination breakpoints and thus the QTL mapping resolution, and incorporate pedigree information into hidden Markov models in ancestral inference from genotypic data; the conditional probabilities and the recombination breakpoint data resulting from ancestral inference can facilitate follow-up QTL mapping. The results show that the generality of the recursive algorithms can greatly increase the application range of genetic analysis such as ancestral inference in multiparental populations." />
<meta name="DC.Contributor" content="Chaozhi Zheng" />
<meta name="DC.Contributor" content="Martin P. Boer" />
<meta name="DC.Contributor" content="Fred A. van Eeuwijk" />
<meta name="article:published_time" content="2018-10-01" />
<meta name="article:section" content="Multiparental Populations" />
<meta name="citation_title" content="Recursive Algorithms for Modeling Genomic Ancestral Origins in a Fixed Pedigree" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;The study of gene flow in pedigrees is of strong interest for the development of quantitative trait loci (QTL) mapping methods in multiparental populations. We developed a Markovian framework for modeling ancestral origins along two homologous chromosomes within individuals in fixed pedigrees. A highly beneficial property of our method is that the size of state space depends linearly or quadratically on the number of pedigree founders, whereas this increases exponentially with pedigree size in alternative methods. To calculate the parameter values of the Markov process, we describe two novel recursive algorithms that differ with respect to the pedigree founders being assumed to be exchangeable or not. Our algorithms apply equally to autosomes and sex chromosomes, another desirable feature of our approach. We tested the accuracy of the algorithms by a million simulations on a pedigree. We demonstrated two applications of the recursive algorithms in multiparental populations: design a breeding scheme for maximizing the overall density of recombination breakpoints and thus the QTL mapping resolution, and incorporate pedigree information into hidden Markov models in ancestral inference from genotypic data; the conditional probabilities and the recombination breakpoint data resulting from ancestral inference can facilitate follow-up QTL mapping. The results show that the generality of the recursive algorithms can greatly increase the application range of genetic analysis such as ancestral inference in multiparental populations.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2018/10/01" />
<meta name="citation_mjid" content="ggg;8/10/3231" />
<meta name="citation_id" content="8/10/3231" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/8/10/3231" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/8/10/3231.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/8/10/3231.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/8/10/3231.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.118.200340" />
<meta name="citation_pmid" content="30068523" />
<meta name="citation_volume" content="8" />
<meta name="citation_issue" content="10" />
<meta name="citation_num_pages" content="15" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Multiparental Populations" />
<meta name="citation_firstpage" content="3231" />
<meta name="citation_lastpage" content="3245" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Chaozhi Zheng" />
<meta name="citation_author_institution" content="Biometris, Wageningen University and Research, Wageningen, The Netherlands" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-6030-3933" />
<meta name="citation_author" content="Martin P. Boer" />
<meta name="citation_author_institution" content="Biometris, Wageningen University and Research, Wageningen, The Netherlands" />
<meta name="citation_author" content="Fred A. van Eeuwijk" />
<meta name="citation_author_institution" content="Biometris, Wageningen University and Research, Wageningen, The Netherlands" />
<meta name="citation_reference" content="citation_journal_title=Theoretical population biology;citation_journal_abbrev=Theor Popul Biol;citation_author=S. Baird;citation_author=N. Barton;citation_author=A. Etheridge;citation_title=The distribution of surviving blocks of an ancestral genome.;citation_pages=451-471;citation_volume=64;citation_year=2003;citation_issue=4;citation_pmid=14630482;citation_doi=10.1016/S0040-5809(03)00098-4" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=L. E. Bauman;citation_author=J. S. Sinsheimer;citation_author=E. M. Sobel;citation_author=K. Lange;citation_title=Mixed effects models for quantitative trait loci mapping with inbred strains.;citation_volume=180;citation_year=2008;citation_pmid=18791243;citation_doi=10.1534/genetics.108.091058" />
<meta name="citation_reference" content="citation_journal_title=Genetica;citation_journal_abbrev=Genetica;citation_author=J. H. Bennett;citation_title=Junctions in inbreeding.;citation_pages=392-406;citation_volume=26;citation_year=1953;citation_issue=5-6;citation_pmid=13142314;citation_doi=10.1007/BF01690623" />
<meta name="citation_reference" content="citation_journal_title=J. R. Stat. Soc. Series B Stat. Methodol.;citation_author=J. H. Bennett;citation_title=The distribution of heterogeneity upon inbreeding.;citation_pages=88-99;citation_volume=16;citation_year=1954" />
<meta name="citation_reference" content="citation_journal_title=Theoretical population biology;citation_journal_abbrev=Theor Popul Biol;citation_author=H. Bickeböller;citation_author=E. A. Thompson;citation_title=Distribution of genome shared IBD by half-sibs: approximation by the Poisson clumping heuristic.;citation_pages=66-90;citation_volume=50;citation_year=1996;citation_issue=1;citation_pmid=8776838;citation_doi=10.1006/tpbi.1996.0023" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=K. W. Broman;citation_title=Genotype probabilities at intermediate generations in the construction of recombinant inbred lines.;citation_volume=190;citation_year=2012;citation_pmid=22345609;citation_doi=10.1534/genetics.111.132647" />
<meta name="citation_reference" content="citation_title=Génétique et Populations;citation_title=Un isolat d’Amerique centrale: les indiens Jicaques du Honduras;citation_author=A. Chapman;citation_author=A. Jacquard;citation_year=1971" />
<meta name="citation_reference" content="citation_journal_title=Theoretical population biology;citation_journal_abbrev=Theor Popul Biol;citation_author=N. Chapman;citation_author=E. Thompson;citation_title=A model for the length of tracts of identity by descent in finite random mating populations.;citation_pages=141-150;citation_volume=64;citation_year=2003;citation_issue=2;citation_pmid=12948676;citation_doi=10.1016/S0040-5809(03)00071-6" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=G. A. Churchill;citation_author=D. C. Airey;citation_author=H. Allayee;citation_author=J. M. Angel;citation_author=A. D. Attie;citation_title=The Collaborative Cross, a community resource for the genetic analysis of complex traits.;citation_pages=1133-1137;citation_volume=36;citation_year=2004;citation_issue=11;citation_pmid=15514660;citation_doi=10.1038/ng1104-1133" />
<meta name="citation_reference" content="citation_journal_title=Theoretical population biology;citation_journal_abbrev=Theor Popul Biol;citation_author=C. C. Cockerham;citation_author=B. S. Weir;citation_title=Descent measures for two loci with some applications.;citation_pages=300-330;citation_volume=4;citation_year=1973;citation_issue=3;citation_pmid=4747657;citation_doi=10.1016/0040-5809(73)90013-0" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_title=The genome architecture of the collaborative cross mouse genetic reference population.;citation_volume=190;citation_year=2012;citation_pmid=22345608;citation_doi=10.1534/genetics.111.132639" />
<meta name="citation_reference" content="citation_journal_title=Theoretical population biology;citation_journal_abbrev=Theor Popul Biol;citation_author=C. Denniston;citation_title=An extension of the probability approach to genetic relationships: one locus.;citation_pages=58-75;citation_volume=6;citation_year=1974;citation_issue=1;citation_pmid=4425405;citation_doi=10.1016/0040-5809(74)90031-8" />
<meta name="citation_reference" content="citation_journal_title=Theoretical population biology;citation_journal_abbrev=Theor Popul Biol;citation_author=K. P. Donnelly;citation_title=The probability that related individuals share some section of genome identical by descent.;citation_pages=34-63;citation_volume=23;citation_year=1983;citation_issue=1;citation_pmid=6857549;citation_doi=10.1016/0040-5809(83)90004-7" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=C. Durrant;citation_author=H. Tayem;citation_author=B. Yalcin;citation_author=J. Cleak;citation_author=L. Goodstadt;citation_title=Collaborative cross mice and their power to map host susceptibility to aspergillus fumigatus infection.;citation_volume=21;citation_year=2011;citation_pmid=21493779;citation_doi=10.1101/gr.118786.110" />
<meta name="citation_reference" content="citation_title=The Theory of Inbreeding;citation_author=R. Fisher;citation_year=1949" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=R. Fisher;citation_title=A fuller theory of junctions in inbreeding.;citation_pages=187-197;citation_volume=8;citation_year=1954;citation_doi=10.1038/hdy.1954.17" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=L. A. Garcia-Cortes;citation_title=A novel recursive algorithm for the calculation of the detailed identity coefficients.;citation_pages=33;citation_volume=47;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. Haldane;citation_author=C. Waddington;citation_title=INBREEDING AND LINKAGE;citation_pages=357-374;citation_volume=16;citation_year=1931;citation_issue=4;citation_pmid=17246626" />
<meta name="citation_reference" content="citation_journal_title=Theoretical population biology;citation_journal_abbrev=Theor Popul Biol;citation_author=W. G. Hill;citation_author=B. S. Weir;citation_title=Prediction of multi-locus inbreeding coefficients and relation to linkage disequilibrium in random mating populations.;citation_pages=179-185;citation_volume=72;citation_year=2007;citation_issue=2;citation_pmid=17575994;citation_doi=10.1016/j.tpb.2006.05.006" />
<meta name="citation_reference" content="citation_journal_abbrev=Plant Biotechnol J;citation_author=B. E. Huang;citation_author=A. W. George;citation_author=K. L. Forrest;citation_author=A. Kilian;citation_author=M. J. Hayden;citation_title=A multiparent advanced generation inter-cross population for genetic analysis in wheat.;citation_pages=826-839;citation_volume=10;citation_year=2012;citation_issue=7;citation_pmid=22594629;citation_doi=10.1111/j.1467-7652.2012.00702.x" />
<meta name="citation_reference" content="citation_title=The Genetic Structure of Populations;citation_author=A. Jacquard;citation_year=1974;citation_doi=10.1007/978-3-642-88415-3" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=F. Johannes;citation_author=M. Colome-Tatche;citation_title=Quantitative epigenetics through epigenomic perturbation of isogenic lines.;citation_volume=188;citation_year=2011;citation_pmid=21385727;citation_doi=10.1534/genetics.111.127118" />
<meta name="citation_reference" content="citation_journal_title=Annals of human genetics;citation_journal_abbrev=Ann Hum Genet;citation_author=G. Karigl;citation_title=A recursive algorithm for the calculation of identity coefficients.;citation_pages=299-305;citation_volume=45;citation_year=1981;citation_issue=Pt 3;citation_pmid=7305283;citation_doi=10.1111/j.1469-1809.1981.tb00341.x" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=P. X. Kover;citation_author=W. Valdar;citation_author=J. Trakalo;citation_author=N. Scarcelli;citation_author=I. M. Ehrenreich;citation_title=A Multiparent Advanced Generation Inter-Cross to fine-map quantitative traits in Arabidopsis thaliana.;citation_pages=e1000551-e1000551;citation_volume=5;citation_year=2009;citation_issue=7;citation_pmid=19593375;citation_doi=10.1371/journal.pgen.1000551" />
<meta name="citation_reference" content="citation_journal_title=BMC Biol.;citation_author=C. H. Li;citation_author=Y. X. Li;citation_author=P. J. Bradbury;citation_author=X. Wu;citation_author=Y. S. Shi;citation_title=Construction of high-quality recombination maps with low-coverage genomic sequencing for joint linkage analysis in maize.;citation_pages=78;citation_volume=13;citation_year=2015;citation_doi=10.1186/s12915-015-0187-4" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=C. B. Liller;citation_author=A. Walla;citation_author=M. P. Boer;citation_author=P. Hedley;citation_author=M. Macaulay;citation_title=Fine mapping of a major QTL for awn length in barley using a multiparent mapping population.;citation_pages=269-281;citation_volume=130;citation_year=2017;citation_doi=10.1007/s00122-016-2807-y" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=E. Y. Liu;citation_author=Q. Zhang;citation_author=L. McMillan;citation_author=F. Pardo-Manuel de Villena;citation_author=W. Wang;citation_title=Efficient genome ancestry inference in complex pedigrees with inbreeding.;citation_volume=26;citation_year=2010;citation_pmid=20529906;citation_doi=10.1093/bioinformatics/btq187" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=I. J. Mackay;citation_author=P. Bansept-Basler;citation_author=T. Barber;citation_author=A. R. Bentley;citation_author=J. Cockram;citation_title=An eight-parent multiparent advanced generation inter-cross population for winter-sown wheat: creation, properties, and validation.;citation_pages=1603-1610;citation_volume=4;citation_year=2014;citation_doi=10.1534/g3.114.012963" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=A. K. MacLeod;citation_author=C. S. Haley;citation_author=J. A. Woolliams;citation_title=Marker densities and the mapping of ancestral junctions.;citation_pages=69-79;citation_volume=85;citation_year=2005;citation_issue=1;citation_pmid=16089037;citation_doi=10.1017/S0016672305007329" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=O. C. Martin;citation_author=F. Hospital;citation_title=Distribution of parental genome blocks in recombinant inbred lines.;citation_volume=189;citation_year=2011;citation_pmid=21840856;citation_doi=10.1534/genetics.111.129700" />
<meta name="citation_reference" content="citation_journal_title=Biometrics;citation_author=R. Nadot;citation_author=G. Vayssiex;citation_title=Algorithme du calcul des coefficients d’identite.;citation_pages=347-359;citation_volume=29;citation_year=1973;citation_doi=10.2307/2529397" />
<meta name="citation_reference" content="citation_journal_title=Plant Biotechnol. J.;citation_author=L. Pascual;citation_author=N. Desplat;citation_author=B. E. Huang;citation_author=A. Desgroux;citation_author=L. Bruguier;citation_title=Potential of a tomato magic population to decipher the genetic control of quantitative traits and detect causal variants in the resequencing era.;citation_pages=565-577;citation_volume=13;citation_year=2015;citation_pmid=25382275;citation_doi=10.1111/pbi.12282" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. V. Rockman;citation_author=L. Kruglyak;citation_title=Breeding designs for recombinant inbred advanced intercross lines.;citation_volume=179;citation_year=2008;citation_pmid=18505881;citation_doi=10.1534/genetics.107.083873" />
<meta name="citation_reference" content="citation_journal_title=Annals of human genetics;citation_journal_abbrev=Ann Hum Genet;citation_author=J. Rostron;citation_title=On the computation of inbreeding coefficients.;citation_pages=469-475;citation_volume=41;citation_year=1978;citation_issue=4;citation_pmid=655636;citation_doi=10.1111/j.1469-1809.1978.tb00918.x" />
<meta name="citation_reference" content="citation_journal_title=Mol. Breed.;citation_author=W. Sannemann;citation_author=B. E. Huang;citation_author=B. Mathew;citation_author=J. Leon;citation_title=Multi-parent advanced generation inter-cross in barley: high-resolution quantitative trait locus mapping for flowering time as a proof of concept.;citation_pages=86;citation_volume=35;citation_year=2015;citation_doi=10.1007/s11032-015-0284-7" />
<meta name="citation_reference" content="citation_journal_title=Genet. Res.;citation_author=P. Stam;citation_title=The distribution of the fraction of the genome identical by descent in finite random mating populations.;citation_pages=131-155;citation_volume=35;citation_year=1980;citation_doi=10.1017/S0016672300014002" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=V. Stefanov;citation_title=Distribution of Genome Shared Identical by Descent by Two Individuals in Grandparent-Type Relationship;citation_pages=1403-1410;citation_volume=156;citation_year=2000;citation_issue=3;citation_pmid=11063711" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=F. Teuscher;citation_author=K. W. Broman;citation_title=Haplotype probabilities for multiple-strain recombinant inbred lines.;citation_volume=175;citation_year=2007;citation_pmid=17151250;citation_doi=10.1534/genetics.106.064063" />
<meta name="citation_reference" content="citation_journal_title=Annals of human genetics;citation_journal_abbrev=Ann Hum Genet;citation_author=E. A. Thompson;citation_title=A recursive algorithm for inferring gene origins.;citation_pages=143-152;citation_volume=47;citation_year=1983;citation_issue=Pt 2;citation_pmid=6881911;citation_doi=10.1111/j.1469-1809.1983.tb00981.x" />
<meta name="citation_reference" content="citation_journal_title=Mathematical Medicine and Biology;citation_journal_abbrev=Math Med Biol;citation_author=E. A. Thompson;citation_title=Two-locus and Three-locus Gene Identity by Descent in Pedigrees;citation_pages=261-279;citation_volume=5;citation_year=1988;citation_issue=4;citation_pmid=3241098;citation_doi=10.1093/imammb/5.4.261" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=B. S. Weir;citation_author=C. C. Cockerham;citation_title=PEDIGREE MATING WITH TWO LINKED LOCI;citation_pages=923-940;citation_volume=61;citation_year=1969;citation_issue=4;citation_pmid=5364970" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=C. E. Welsh;citation_author=L. McMillan;citation_title=Accelerating the inbreeding of multi-parental recombinant inbred lines generated by sibling matings.;citation_pages=191-198;citation_volume=2;citation_year=2012;citation_doi=10.1534/g3.111.001784" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. Z. Xu;citation_title=Genetic mapping and genomic selection using recombination breakpoint data.;citation_volume=195;citation_year=2013;citation_pmid=23979575;citation_doi=10.1534/genetics.113.155309" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=C. Zheng;citation_title=Modeling X-linked-linked ancestral origins in multiparental populations.;citation_volume=5;citation_year=2015;citation_pmid=25740936;citation_doi=10.1534/g3.114.016154" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=C. Zheng;citation_author=M. P. Boer;citation_author=F. A. van Eeuwijk;citation_title=A general modeling framework for genome ancestral origins in multiparental populations.;citation_volume=198;citation_year=2014;citation_pmid=25236451;citation_doi=10.1534/genetics.114.163006" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=C. Zheng;citation_author=M. P. Boer;citation_author=F. A. van Eeuwijk;citation_title=Accurate genotype imputation in multiparental populations from low-coverage sequence.;citation_year=2018;citation_pmid=30045858;citation_doi=10.1534/genetics.118.300885" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. J. Zhou;citation_author=A. Ghazalpour;citation_author=E. M. Sobel;citation_author=J. S. Sinsheimer;citation_author=K. Lange;citation_title=Quantitative trait loci association mapping by imputation of strain origins in multifounder crosses.;citation_volume=190;citation_year=2012;citation_pmid=22143921;citation_doi=10.1534/genetics.111.135095" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Recursive Algorithms for Modeling Genomic Ancestral Origins in a Fixed Pedigree" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10.cover-source.jpg" />
<meta name="twitter:description" content="The study of gene flow in pedigrees is of strong interest for the development of quantitative trait loci (QTL) mapping methods in multiparental populations. We developed a Markovian framework for modeling ancestral origins along two homologous chromosomes within individuals in fixed pedigrees. A highly beneficial property of our method is that the size of state space depends linearly or quadratically on the number of pedigree founders, whereas this increases exponentially with pedigree size in alternative methods. To calculate the parameter values of the Markov process, we describe two novel recursive algorithms that differ with respect to the pedigree founders being assumed to be exchangeable or not. Our algorithms apply equally to autosomes and sex chromosomes, another desirable feature of our approach. We tested the accuracy of the algorithms by a million simulations on a pedigree. We demonstrated two applications of the recursive algorithms in multiparental populations: design a breeding scheme for maximizing the overall density of recombination breakpoints and thus the QTL mapping resolution, and incorporate pedigree information into hidden Markov models in ancestral inference from genotypic data; the conditional probabilities and the recombination breakpoint data resulting from ancestral inference can facilitate follow-up QTL mapping. The results show that the generality of the recursive algorithms can greatly increase the application range of genetic analysis such as ancestral inference in multiparental populations." />
<meta name="og-title" property="og:title" content="Recursive Algorithms for Modeling Genomic Ancestral Origins in a Fixed Pedigree" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/8/10/3231" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="The study of gene flow in pedigrees is of strong interest for the development of quantitative trait loci (QTL) mapping methods in multiparental populations. We developed a Markovian framework for modeling ancestral origins along two homologous chromosomes within individuals in fixed pedigrees. A highly beneficial property of our method is that the size of state space depends linearly or quadratically on the number of pedigree founders, whereas this increases exponentially with pedigree size in alternative methods. To calculate the parameter values of the Markov process, we describe two novel recursive algorithms that differ with respect to the pedigree founders being assumed to be exchangeable or not. Our algorithms apply equally to autosomes and sex chromosomes, another desirable feature of our approach. We tested the accuracy of the algorithms by a million simulations on a pedigree. We demonstrated two applications of the recursive algorithms in multiparental populations: design a breeding scheme for maximizing the overall density of recombination breakpoints and thus the QTL mapping resolution, and incorporate pedigree information into hidden Markov models in ancestral inference from genotypic data; the conditional probabilities and the recombination breakpoint data resulting from ancestral inference can facilitate follow-up QTL mapping. The results show that the generality of the recursive algorithms can greatly increase the application range of genetic analysis such as ancestral inference in multiparental populations." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10.cover-source.jpg" />
    <title>Recursive Algorithms for Modeling Genomic Ancestral Origins in a Fixed Pedigree | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"nd9OuBMrtn5KZ9xPePpEobC7aMlUc9cA7wBX6jUsvcY","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"490964","apath":"\/ggg\/8\/10\/3231.atom","pisa":"ggg;8\/10\/3231","ac":{"\/ggg\/8\/10\/3231.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/8\/10\/3231.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;8\/10\/3231"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;8\/10\/3231"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/8\/10\/3231":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022bEAXQP+o3x2s7rdvlPfCrSU82dDXSFy5p4BlJxHjBAUz0U2YA+J0plQlmBzm\\\\\\\/GZ96VFxDpEOxkZDpae8\\\\\\\/IeVYy8maH+FyI8cDTqOSVJGwLJFp04drKk3QOlD250wmZxfzfFQ7wDGqo1geoN1y+T6YOUSO4u\\\\\\\/QuffoFFWSe2DwT9bsdOBNGkiEXZcqXrAuRCGw7mepI3JTMa8zLbiX2+nwd5NONjr83H4cjvq1jAfOH70H9pD38XoJHUwqZ8GWVSvnJptWtoRjb2NlpOhI5CvjhFXq0GVXcTgGkI9hTNzxdDvkSF6DabaZpUKYGhEGH017SRnXkJGcUVo7aM3GFMv+BSHZH8fZVkHtAYwK6VgqQK8hcQYinLERKoE4yQvj\\\\\\\/gL18CqZbIZcsBtsIqSAbxyCTped8rVEhoXOVcnkxFvQBKKvMaQSLMWxh8S3vkObpGlPXBqDVZOr\\\\\\\/8mbuDPYp\\\\\\\/BP\\\\\\\/hBWnon1FKuhpiVq4b\\\\\\\/dO03PW53uKcANx\\\\\\\/LkmpmnWDZtQVWVMW9TfrpmQYANLCpxqhIgqXh8E+E9zFXs5g+lmwugeG3e4TG\\\\\\\/jocxF16O12ZWtEoaZs6WnyYx6n0tztPC2lLD8\\\\\\\/QpkhHIOhEbQ5RH28kBEEmTpTAwPkbm+pt+5Wxw\\\\\\\/5sueudkjFmwyZavfBxGIjNQpVZRXfgdmueRIbqO+4Ed4epz9cBbzwPr63dtf7aGE7Z8gFDPOY5kUk2+C5Ld9B3aeBFyFhr\\\\\\\/TtRU2zYzXfHDq\\\\\\\/j4i46fSraTKEF8frdSDVd+46SyEzETGpe3boKODaGXfOCX+qnajK89FiBYQOc1Wq\\\\\\\/2b0z\\\\\\\/iEvYpYSHwG851h5\\\\\\\/9wjgKfuBo3m3LXrRd4YhmK1VNq98HEl0FaRAyyZQuWsqdd3\\\\\\\/dAY4h54hDLtgxDLqmPp9xq8bVwcC\\\\\\\/3t2Mau3vmU1T7NAVqdujSPKlNGunOWuRRXqfPagJsOZQ6zaG+twkkmLZXuNbyP4mDiI7svgYc0yFwKJaAFZ8mPpPLLuOxZUyQ7ttd7SpX5rfEfZS6bah\\\\\\\/JzWDj5rkYjbcOxC4OfV5qlbS\\\\\\\/9HDNLsOSssvFkD\\\\\\\/Um6SOF1PXb5YZFlUyagWY8UPfUe6ebHZ3E6tg5RHIlz89yTeu\\\\\\\/o0vaPBP1HBRMtLw9lrVEQoYpRtFKoA7evmZ8o\\\\\\\/R1blP\\\\\\\/8Zb7m8tLT0B21R22yu2X43Pe5cicNYJsg7+lM2vz42AH2OYlnUsGGmC2zGj1D2+CEEQ7x9TjZ27KOrFoj6x\\\\\\\/M7FGPKHSCPN9kRxn4INujSKCt6JXect\\\u0022,\\\u0022iv\\\u0022:\\\u0022L3GUzR4kTqCpTuahpzYblA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00226af740e78e5760e89647e72f4831277075ba5f682c06b7df4bd00240e95211e6\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022ZsbXoo+f+jkOqbYJrezqn3LGYGjUvQNLiSy+ub4xb1lhAAqZoP70PVEF4oqsNYXD2ifjr95BjT2TrOfzyzbeyX1e+X6rt9go8uycomEecmdRercWgLWQNymptN6lIbg3kZCzxM6c0P6a1PbxQnm+vnjPYwnxuQJq5r\\\\\\\/6xpIfyOrEfJDTmgm\\\\\\\/tf5JZ\\\\\\\/3\\\\\\\/jzw2XG+r9mg9BMRPPa2\\\\\\\/AbiQITEGxJzTceBrOhsJGF+gYsrDPqiLQkcOS0WiiuAk1\\\\\\\/n5JukKwEUDT4QPjwcYbUsqszJnzJ71TvLp0NmuFATwQmK5EV7rJ9XbevD8FT81\\\\\\\/BpNT313IxMF+dpL94p+77zjFVUF3EmP8JsYPEzQ\\\\\\\/uaz5iYcQxLbO9oXBOy16NQY6opcXS0cYCgj3y49bhfICnqANB7B84LS9Hv7jrv\\\\\\\/JBP6xARWXn3jDYp2s3n0rnohI2T5fd5pfPh1ubCU3b8A4QGZx+D5aBT+3hFMUOVbKIG3fPOlauU8ZW6r+1jAD9zuGbHqZIILgenXn9FSZdXq4ZyiW9rTU6umobXKcA+DwXeGzt34bZvHLHuJxlkzSBSY9RvIAxDcYR2AGjRTOuZEgG7M+pBVRbWj\\\\\\\/OYxmR\\\\\\\/J69VM6R2bhDOyapej4vunCrLq2MQjA0lffLVwIayovlKxe+3pP+luDQSqEq\\\\\\\/NP0cTM8Dhye5SopUDscC+q3zIBLZFwVUaDYUVEKynrfX6cAGx1TepAhmkGGbfVKl4SeEsIjyTn\\\\\\\/iqz0upZdXLo92ygxmK8yy7eZQOcKpg9atQlOy\\\\\\\/5YHT7dvqb9QU0JlYboXHzSkEVnUQnPERiU2ViCcDzTmOxcuUXrdBrvSOm0huJK+kruG9QFYlFGqz0YRqg9fJWxgtgDZuMW9mLk2WZxM\\\\\\\/c7L2R+ilSe9h6ea\\\\\\\/oHmfIg+QvkHghNRi+MjiweZ1DLjg72LAoNYlLL1LgeIVYMXu1RqLElmU5i1JbL2zPteGeqgA3gNkUvVr2Jp3PjVx4lD87audOv5UriW062ilM\\\\\\\/a9GkL0EG2x46y7MEblqGobL9duRZGEKZXCWsHozE11w1II0gmjctkvfKzeFHOjBP0UMbx9GfzYjGHEZw\\\\\\\/5s0LsdqbHsvsU+ITZIampNTgIR1aP+GzzIDueiG1wqM0YXJgt53bj9tyA4ljVgbrQ0qNfYS9jStEfxV0b5z50eh01KnYbE5Hu3WqR8AZI3rZPjm0Qq0kz4a+FqjQIywz+Lv+sLG67itJBYq43dBQvfvQ6AMIZqzK7c3NhTkdozDl44X7hpdsQUdwT91W4fBU+mHSHT\\\\\\\/eErWHXCYWoNsQpXvSZVh6mw4rp7DQgIU1o7W6HebbdqDUbyKNjtuy0LdaS4mT0aWJEsGRBCBiYtfWYnfbMmPEDUVhMF3l3BZBOZC\\\\\\\/hy23XTPD8EKaAHAZheTXEggCf0TSGxHDR203yaxHqtcI8LwXcAB9mOLLdNOmTJgwOtFAcwpFsHwPI0aP0YblWA+1H0a9ghujeq+9RIVMtVhr3iLIMuOf30BUWDL9xAaHsBdKybQ9+5iSvcxyG4VcTNQsinDaUoFOr+RyDn7orZkrEXvOKGkCpu3dZt1Aez+VKvypbXhIC5DtoZfuE\\\\\\\/n+K396iVzPjl+LN\\\\\\\/YM90t8vNiqdaAfXl6yjZXZgRmq2uuuS2QiKSEQeZm+vh0pT1Q2+0noVmSjc4vDylKfViV6LxucMfN65bGa2e+DFvL1D7fRMJdWzg3eTRjpl9nIjbyp9mbN1p9rv8HDXtj0z2HLPSX+TB0hZFRTukZdpMqy7ia80ks1u0saMi0PcvS5xcREqrf7sd3nAmKd+Gsp44BW\\\\\\\/TTYT4NcStZxuJKmqvypKDD5pQLt\\\\\\\/TVaWKkhxWDDmIHRr0VtqWnKOtaEIFKlq\\\\\\\/X2R\\\\\\\/fHcMPGgPIAcqVTsuEUddrTtG2vGpk3J1f+XtzH3gvlP2Dyz\\\\\\\/eUrH6P+vhRTMKY7hTvc4+xb5JfZ0pljEIijavwout5gvhX1FksVEggvpaXSeTYtaSxy0CqCqD\\\\\\\/VrdkqM\\\\\\\/q4hUuLOwnqYhwrBFJAuwpOnDRExC1E\\\\\\\/\\\\\\\/HlAoQzzXlNlYK18oXpDsmMx3wydIaiFVVa0wmsZeEc6u0u1mKkhLPt+2GtwFMIYaUyEn+qmZNBDDv7p2nYddxluCllUwdwVJ0wA2Ix0buqgE+XNNKeie5yuTc6sJUBhlRz4qPr\\\\\\\/Mt3hJdLCIGj\\\\\\\/M7EITQt2rx\\\\\\\/i\\\\\\\/+NatGDkQzr05AviZQ8IKVHbwYoiU8AGQxrM9E7SOYwqKISYvwbvWG0i\\\\\\\/GthK\\\\\\\/bGvk2aDuBuGw\\\\\\\/er6lNKxJzOyiQBSy+pSxG4d8TX3WOUSHTaXfyPK50b6BLaZToPPzTePJTb9k4MkUkz0tcUI8F3Bxm0ZRgF89WwFEIONtrgZY0D7re1h78owmMyH8Kx4ONmZPFusTclo\\\\\\\/9UPkgPJ411ypIDjCU4LFkwGD6S95awBMzKgFHuJUkRGcxnC7AUD\\\\\\\/xmZ2sphcBsgFdV8dHuDej94KmKUtne4mNgRzfaFxOG\\\\\\\/jvT\\\\\\\/4j6SbaA8MghsiHmXZGR7xWCS\\\\\\\/YHZNO6ZfFzPNb9B0CNzrJxllLBUg3mR3Hma1F4BVCCaYmCUfAFwzLLV8hvMJrPsy7bCP2kdWdKWMqytq730XQHylziRtxZlEkbSRwADWWSxsPsJ1DFTWjkcavcfzpJp5PHeIJKoGTM6RqcFxVQOxKTFePMwkThuzWMp5BHGv3eJUlpvHAsiTFy80sE3JQK6xTM6bMmx39FyhtjDAY7TQWT2me4hyH5KV806t+qM9s0VGN912WMI1RYycIGdAbgeQNE2QyHJ6+tFQagRbvE2g9QMZQf8+W+H\\\\\\\/yNbuUz4wGlgQSgHbeHnHv3tCmuxhwvKL1MU4Vuk5zlhYJ\\\\\\\/wo9ahBpuWzewnnF9hKUQpmJvvgiFPuDsrRAkZnYjYknFOqTXbYKiO06kOyH2ZCMuaEn\\\\\\\/rrq3GYyxGgbEsLko6POCRf50WqIcUa9yBNZOLykWDYHHozo66ssnOLwDMlWRrxeKSas0A2eSBKXL5FOJnBqxNZtJTE\\\\\\\/Fh8Z+fyopsdNFUWtFiQooEa95RWYABLisVzGlWoqep1QGa6A3e7k6Gri7Elaiw\\\\\\\/zUEAa2hQDg7uaMpX7iAQnbuFJqUxfdGdCgssGOazMZBYQU5WH+Vp1UdGWEP8fypqYAzbjSMsXgBsPTJJ01Bvx6xWT2xrywjH2lxYmWRNgQt7FDbQZMuxgXDcwIvnUwuMCrmO9P2Bx1TNh56N4XEUo2tdokxNdU8se\\\\\\\/keVAC6MeYN6ReLFi\\\\\\\/LcN9JDZ7O8sISZzeI4eDnBiyYANKrQuDqfR09TMmZlMpDmvzVXBm0XBHpXf75YnPOxazrfHqQ72Vi73U7vcZkSg7GlZIsBuGGe\\\\\\\/j4chjVwjthexKhT2\\\\\\\/FxB8wQsSTGGZ6BlQ86Q+rTQvWBYYKbbEWstzYzWN\\\\\\\/kLkSNy1Et5+ufWmbCNgS8WG14aWnachf38mo3X87jyB1y6z97L+TEBshOXUXshW6c9CqYwDLUpf+dFPeKvIei2DnLcSCAcLbjbQ\\\\\\\/41\\\\\\\/MxOOXwR8R3lGPPBxGiqeCqPS+oUUFroQrbDMJyvJ2qLZfubG9w40JwAd7Mgy5SDwc1VFyq0shJHYMiykQGQkEkIFn3zM\\\\\\\/p8wNxu2sXIAKarwbyzeKE+2ijW\\\\\\\/BnDdGWbfbt48DBmV5MsG3GM9MXujlgDJqXMImJKD82pU0G1DIi\\\\\\\/FzIu4fYC\\\\\\\/Mst9sOj2pN7nTsQ4Qi8+smB1geyr6IfYTblsZykkbsQyuOwHcMEPcdasXny\\\\\\\/X9U0zqHIhvGtJOdKs5nxGm3zVmysnHXiaaXUNDwtWe+mSWxOxSc4wLcxW4S+5ziLPtpD9n4LWM1Pgrg2sFBzBWGRdMbHxHsNxfmQ6ah827Rh6PylJnjyrqOnwZRi6q7DdUWMOVRDwy1pfuCywKieJ6YskUsO8U9pOA0E51GJ26ye\\\\\\\/JWuiYvlFYdSCejwRlXnKE30qWIU6h2iHwslNsEhsjAgZRa2+C+8MBbE0GwSTqDLr4bxqutq9yOziDk+PpjCBk8rvY4e8FN\\\\\\\/IHbkdBig9uHmcqkXGuZ5RkeInSZeb2tJeHepKpZpQDtAFn5CLVZCqlE66gDHcm2kFYgbL3vNVm\\\\\\\/J96IcxsCUnRlWiiO4o+FeMhvKLZ6Y8YHB5bYQzdsGoksI\\\\\\\/CIjU8G5ADSJrx4nztYmmZwwKnWwrv5FMNKSRTfJDiQeqRdgVva9zKdGuOqN1PuMDeb2MXL+gcRaQaVGs5PeDsHZy+Rs8HEYi836jChqJPFm\\\\\\\/A\\\\\\\/z6927vCbqd\\\\\\\/LlfirrZkPCdARSv\\\\\\\/vEXsShBTgIUNenyIMYdJUVOcL0mdeqZfLaKZkqgtYCY+lgfwZXWVcCCyApnfEgyQWUCSPqtv6ct2GF2+7B2uCAFkmwvtXkM0cpySXGGBPGGM4MFbKfcQFbRts4M1G4s13tulTF5Xdoe2PJvB74+tqvuLMMyD52b\\\\\\\/StGkedKGygCc3p\\\\\\\/Ku689UpahnXpSvYqubPEwcdRQ84mTFyPwa42KfrfPjRvp55aBxwau0xLfsrZ3dblG2+hdJFCQMeUpkwiIG0mcm94cYMldk4m\\\\\\\/adnw1o8l0F+lIlnqx7m1nJAQnx8jnxa3TtOxRQlGAUlSK3sSnCA1+65er\\\\\\\/CP7j6FK8csehVs6y10Rcw0xKKL8rv3DcV2+SkfO2OlME14QUYahE372\\\\\\\/Fed9iRDN0KHrSA4T4t0vsDcE0CkkuipBOiqkNAUhlpEGAjK4DfU2d2PCr96jKJrbnHRBG3LtkndDxnk6PugJGctu1oDark+O0cNW9z+p91GScuDR\\\\\\\/+P8j+sTdym2RzhJoe2zjCacjetbXG4K4vF2DKLu8dvP2RA1mY8hd\\\\\\\/JMf4akyuQOjw++8ngxgT0gWNGB4\\\\\\\/wcjdQ3Chi4sFMvOeBLmm2u4GQi\\\\\\\/aUeQ5uBqyipDPO9Ud9ZrPvCEVf8+FDjUGjoFWJlKLZQMtj4BgSNvFitFbtz08nWm3IqyhPxs1YHsMrx8Vbd1oLAvn\\\\\\\/ez+YZSF76W7N94+OzHxP06oxoEHhsmD4ViU9Hf0Zvh+UzBHJO9MJNJ7ptuBtDMMkzw6UBzo3E3mygnGHjsiL\\\\\\\/CBeLFNARgJzBfPsNRUtHRaZxX34MVP1Kbtgkip5e73Vo+Rsm4xgAWKEVrgfiN8bRGh3v7eVEpMdKzawNLWbOdHPpgxU2QzbsvlJUnIeNyjvo3ip9qpQdm8iZ9tzBz\\\\\\\/vRtH0TS6dci6UtJoHFwZ0u2tAUwkT\\\\\\\/XUy1OUJN45qUDZuQrZaREbUgeQDEZerZUoTn5ZYZL6avhmNTMfaH74p\\\\\\\/sUOteo\\\\\\\/fgV60km9sPTC2xf8y5zvtNYZittjxlGnLoN2rmNyZtTL1MDZenxBuIz3RhYOimZ7BeuLomj4rsNAjnDmWiJnrc1Vl3DKZvFhYp3fBT3rVH3JZGnfa1ncfWMKBBfBsXv9Ai4QmBLF4Q4KrQ3PTUIO0m7pegZertHbk10iRVQcEz6sHwPrSd+sX1XTFJz2lG+CidQdqrZVAguKO93ggR\\\\\\\/CHb7cNODiwZmizNlvnKO0GkXMJPVupEbzOkcdaKjUmk4HVSFzWw9dTX0lKNpfSdgaraTV920P5dt0kTnyo\\\\\\\/ysQzmIVzoqI3klbktXsyoASU6HYzTK8nhL27\\\\\\\/t8WNptbTTU8G9nRBq1gyqrLmFalxigwds5b+xdzXN9WoUr2EkJaQkSe1kxZJqhOy+w2K367aYMtxmKbjSAXEpN\\\\\\\/4P8TVL+HcSrWNbrk8R102Eb6ins\\\\\\\/584BFucQHgI\\\\\\\/SfMNmPE+lgmPbV6Y\\\\\\\/XZ1xOL0x+7Nbgr4r9LwvbbUP356GYmluEh0iCHB8EOF6SRN+tuWuw8ZY11DRWtByF92pkrlwuDrSBVSJjBWlVTBDPBbioTqeHIh8SWVXGpTiygWgVX2cAYMJJS34oUY+DYZn5bj1AOo\\\\\\\/Af+Pc+UO44PCPvzhet+7h\\\\\\\/VT4zfLHvvoiBlIBofSAwp+HVDpA8jt5Xs2XUuyafBSRYuiv2FS5jEi+sKwA0k5Vo9al5Na9SAi00Ygt4E7hkF2rip1AW980vH1uKQT3OGmgTN8jc8gfUmkrA9qfPea+N9Yk73wPlCN7Z2bMQbKCOLJmKb9WWEifdFjdxoJ+8XeEmcjbN2xalfL5GgnVpPHb\\\\\\\/3s0hTR\\\\\\\/pQ+O5nhQEjQoWZB+EAK8mo+vuCS6vcDYrNW5dcBaXx0ZxQ0o5U3nB2lIU1F6JrCPft+7ssSuR15Y9DdZq6qGsbFBxEvqW0gEQb3j9\\\\\\\/aNH46\\\\\\\/\\\\\\\/IXli2Zy+1HhXGeyJn\\\\\\\/Z+85t3zPOUKKBqXpQS4gD4hGt6sr0z5RWbpYZgDchN3shWjmxLyyV2JoGvuI0L0OKXnrL8E+ro4TzlG9Qmbqnl\\\\\\\/KQO9JkokgRzmQaBZdsnfyu3sqB9+EEnUnY2qZvjbe56jB5xYWJi\\\\\\\/cnD6GRjBhqgOoFgNzwKmvfkpojMyTwbgImZPNABT\\\\\\\/VJOJgu3XxNV+Gi4AMZsycVcimdCPf5WaCI31ROFMMnKNWgvL4Ax60QQl69u6pUL8dp400OSr0iyS+9Fzbats4b2ghoXMQh0ChAxzZhRqrrWB0hvtRb\\\\\\\/4AeZgyHT2cNs\\\\\\\/yV1RpZbljXP+O\\\\\\\/CFqDkXmXlgnRQD4cbCWMO45fbspzctCzA1dVYZxUG+XSF9xwoJ0tjVOafMRoMtikbbC7LNQ5KOEqJHl\\\\\\\/X4v+0GYZs3H+M9aB8YTvrghUUeTWXStQFzaFo1DLkVQgRy8c+JVy5s4euWVnk8i5uohdJlmR2GU3PkWRotu\\\\\\\/BgZYbLX4wZwyqCYzJWoAMzFoyH1uYD0LsX42OCYfwy8\\\\\\\/Y5o5iMQ5MiKUw95Z3mkwr7hKYrY2uZGGTRDsuQzO4GrpQZTHus66ZNWXseWVxsSVvxqTGZ7R7JSgWzXrWlxkY2aDcdZ7PYx4kbzxCE0HJ6GSl3TAnNsj5Nd0fkDBTM\\\\\\\/5LBka2rVZjgwxVwEk14YrYU6+cRZJaG3b8LCdc6FT7Nelq8N\\\\\\\/3fTbYl0pQVE6wXoSMmjDprMm2tB6BiXw9z439yv+7wNZCicVJaJVpiU3z17MR7gW+ChIOOx3dG2ffyoioSoqT9tz0eCJO\\\\\\\/Rf0y40ZE+uPaTQabs9eZUxKCqXkw6NTm4TacM7ogHK2+8Ls0gpN2ooCQ\\\\\\\/wYW6wiLY8AVc19ETe2CpeHryf5ihJE\\\\\\\/WQ0mdbAxqEoQB2tbk5FFJkED6tVtsWt7LNIc5+O8iplJIwbWgtHm5n1aU3BV5g9uv1PA7m1GMgUZnA1dv2QmGSaJijUR8cdz+fbsIAgji5Gc27Z+760hP9VA\\\\\\\/JTJZy6Ob4hUZEuUu+J\\\\\\\/F3EIeGiD5IGAIX67DAxQz2BUU8oc2BXjvLi+3lF3xcJQxBic7ZGEtFgZzYCgXPMXUFg4t2q\\\\\\\/D2jQlheet+WjzD9\\\\\\\/Yx2gsMTU11seBXyCwi71iE+ahq\\\\\\\/3PIOUpxU\\\\\\\/Fpr3P4fvGXfA5vno2owO9QeobW10ATDqWoEhnuKT1p6RTj71VtigRmOEGlt4Rty0a090ruGO+KNWVGPErhlyxCPJYNCwyE0+\\\\\\\/55E4o\\\\\\\/Q8yyDmMWmdkRP1N5E5WA8IGoqymx1wpFMN1lq9p2PpaiuP3POpKgzsEYw6FvfTnokrvoDrRkQhF5A\\\\\\\/lajkiHpnS8s9UgstfSVobsR2Tmi06dD2+bpA6JoMoHsYo+i0PlTbKp+x6D8j3bvZZPiI1Q5zlskEcPgJJAGzCer84BZSnwHGEoa\\\\\\\/QXu9mq2Kei9iDcvhhRGtAO6ECrFMLAh3Npn79jCMZqER\\\\\\\/CgqLFRjwRXqrOB6tLYc0aN5fPEmvA5Eq8l\\\\\\\/U1iwC+ngRDf9b7EJJO4dunkpNPv9IT+L24TT4Ss8wZntNrcoW4ukkN2pwFUKLB2SENu2+SQjj3ikrGgtS8FBoToAl6FZN1KXcB+ZkqLL2k\\\\\\\/CRPf4IZLEhZ2RRi56N2IrKfydTNIzCIEzfJhsyDQSsK4mdPghlzHx8ns3lmtPCCH875QTXaNdbrSCXw\\\\\\\/kNvoPrhNGn2HjlVXrDDq\\\\\\\/QdM2SvFxbfHWr169OZa\\\\\\\/xuN0LcGqsMuITYGZzysue2Kaz7f0JLrp8AEbPY1EzXbeZqIyVIu7zALbDR0JlkBj65Lzzv7cS1QyimVeC5LPLNvjoRARJnmGVPfsfo\\\\\\\/UJylQeaWcsrIcUPuXnjrZ1Ov0REqYO7uSiniM2hGXg1ZxVAnVjQ8GtBcChoAHDoNzZjVqNDD1zx83Wf7vZhK+yaA3pd03o1PCH9aPdhhVWdh3Zefp3Pj2YLGQ2HtrVAloAJ9LPRrDzsQDnSCRRwkGPIyTyj5Hqzdkozchmen6iFvsIosUpWZD6M0o6mp1Ebl90z\\\\\\\/K+0CEEI7XfLZZLe8cCmLf2Uz2tMezpIw0hxFlSDMGIX9XcrMZPDN7Lk8t0N6mZXsHwFc5l3S3DSRPmbTTLXW4M7W2W3mis32sFKJqniLKzh7J\\\\\\\/S+ICo7BQ7fyrh\\\\\\\/1jzzsneyLDYMVVkd23pnBDFLkJ63i8fUzyco7J1lvdKOBY2tnKnmjrdKgBwORGesHaMh4DNGspPvt87yfOCk9ltVjRhGFEqf4ZAE7Zb+1PfowD4JNF644ujSRp4KOh8oKS7qZ\\\\\\\/Y6CVuWnr7WlGiOvM+M8N\\\\\\\/1NiOOfRFSuciC\\\\\\\/FT04oIuSVZuNTfRELSiX7jVqN3g4IwXJO0mI36zvFrPfahPoIGm30kkEPg0u3ry0fKOse5ubts6Cy+9qodvF9VYwJr+HBfsoFazqeGeFHIY00A2obiJFTmosnYHD2x68uR8\\\\\\\/HL+jIQu\\\\\\\/mwHJbG4ZND5u1vmU9VDU8Z47ky++2mbltxlqmybKHdbcGlDzVg1ndqKIPAHtfX46xlByREAt6u6+5geQSO9dIk9j56GYgE04zqZlvqPKGhjmOhZpg1lqrbdBDAYcpNe+S15FIwE50V6y43zaoogc78o3KRwrhBxP+AzaOG85lqaO+6VCx+I4zsfNvP6PDCLgJN9k7ILh5E5QNNWmCNvZXgtaE04VuOS6IbsdXoH+Bmlxpp4LqY4yRbtz3glMHzEYgova6gL3UaK\\\\\\\/oXkpi4pNsxMndLv223R2zqrYZuNpdNiAN5Xf3GoXMwolZLI9hgIrWbkrfDN0ZoxuEok4ERxhi76X+VTyd\\\\\\\/JlzYnlIGvfRzi+cIy8Ywo7RrNCG0LKyVOowJskbBNhMe4H3aRb1rQA9dg+WH7GD+SB0oOjp+sFUqqYBE0qEgojpa42KeccfyASMki1wmVSEukKjP9kxTuf0t4uJPXeHNICI2niJsn7\\\\\\\/WDgj5moRc0iBUC4UAU03lSfNVzQ7N\\\\\\\/Ia1Yvv8ko8O6aCw5CMzPomkWQTzStt3kO30cFCy0T6EPgaWXqSCoJYAoZ3peRJN8LuSxxludt7gSh\\\\\\\/qpWFHTTcFDMWKxx7VBZ\\\\\\\/jxgta\\\\\\\/MS5e+VxWwaaCSEZBtfUH1t75rq3uEwSAPXw4gbqmoRpqb83Mz+gYgkNbcnuEP7V7RAueJ6HW6QW2ITjtfulQ4O1vnVwQQQxI00tZ38Sksihd+PBVhVBeYZwNSQ+GVo7cszVgGKpIi0LZP8LUI11k4GX4U9LThduJAe\\\\\\\/xvTVsaXvGn2ji5OxC5WPo\\\\\\\/GrzSthZE0mUSTqSiBzN6l6bB2Q0ELQ2K2bYivHrpYqJWmvn0X4UK4aLIUX4xrfqX6wZHPM4BIge4hjXOC1n6SHZbZJM3JuQpYDw8JWKSl5Cc66oGaBlBb9UKGbxZxdZGrXdih6DyD6WoljbLVoQlyO7zqxO0XaXxEOfzro+Pm3mYDKE4NXbOXBTXmrehxXRALTx\\\\\\\/C7qnZ0BhANL\\\\\\\/cGoBwEYCsUK55I\\\\\\\/CYQX0Lncf2AdhjHwZASmzH5rtrkhzy0\\\\\\\/hzA7oBZWfc8CYBYN0GY\\\\\\\/uLP7bza\\\\\\\/5Au7zaQ+I1fRGS\\\\\\\/asq7BRN4zy6SHEGGuZ5GLvcTMIVCMEk3fpOVJ1Hl3QKtB6AeaBbRUwfydR0O8EbKfp2+OWnx1Oz\\\\\\\/DHs44mG\\\\\\\/5MI670nM6CYYERfRM90GJZHCefPTWUsyViBqEyKV2e+QReUgIJsTqXWF+427kZJmWhP7+1ULKfPTgdiQofCGZ8x1EJ20S7N5PkasKe\\\\\\\/3sYRVqgrb5LaVT9d03o68gndT1mqi1rfPn4t5nLqsZuTu1FxIbgDnXH1NcGc0GjOoGoVQWvbB6o3Igguft5rHdjCc3ngt+l9vFtepZU8A7iHigyFNOkbr\\\\\\\/EgEKB5Ca\\\\\\\/ZOCgnxd0+EJH6k0aQVYkk5LMxr9rCxvd1zEqtwcI93SJTNdJn4SLMbYMkQjn3vULmQ95azYYlbHOW4izhtxI9TQcDirrms\\\\\\\/36\\\\\\\/eq7df6dY7pl\\\\\\\/Tw4c2WGZ7D0LDkoRA093EIZ5fIzCDR9niamKeq6xGLiDBgwFMWCBrTjeMs2AV27Bt2Pb6EZXB4v74YAxGNiNEhjdEBwx9O0ZIjKOrdsaBx9d5eHZoAASSHedIyVb\\\\\\\/siTKpObFvQLtCvEU+LKCRviIYr6LtAmnc1Rdwz\\\\\\\/Y3jr08kOIMDXSaD6Kb1pDgakBu6FkxPRPjC2iwkKCeqAMe51h5jA4hifiOEiCYpirLAElLtNKLVgSzibjdBI6szLVTVsrVBqXc4Tldd6TOYtKBPrGBleqVPG7JJ+foYc4N7Ex7+X14ZJ0dKtQi8eeVwQF4ZABqVNW\\\\\\\/N830tsIuZf4kg\\\\\\\/BbVAoWRF+6lFuf3AOPA0nnixpuedxu17j+2F0vMxsM6M2bzlTZq4CpkSBj3tsIUZW27csp6zWtqkt0wWUUGV29Qoe\\\\\\\/Rr9FpX2wIP34+a5ghqwsHYPk3aM\\\\\\\/QzXIzsFpNwB1OEQyQtLSt6aL7+rLDuiHM9Wxh9r+htz2bPACi9BfnjEBiMCXFxHszyUnV1Kq9lah60Uk6o0cvtNBdGNxO095A9CEXN4Q27Nu\\\\\\\/zsw3QRUH2FxCs7Rly01OYwkFFOXGiEZkEKARNZLB5Yl7+oi\\\\\\\/oARi6BX81PGXsLr4GK5HfgG17lCV61DyKNLjdH7xRfECNgBcA7IcFgSRHGX4+rhihPNQCy9xGqFraKakJdruzGdW\\\\\\\/rNXxNbvT\\\\\\\/J8RNbgiLYGlNvGrzt+0xo2YpR8oBoRXtdjpbVe2nhpC+RhS1yRvusKMWV5ylAK+A9RxkDP+egB6jZjfFK5CCujsgfuPed1VHVqNAs20sgAIhyY5oDDzyCX\\\\\\\/AZ\\\\\\\/onqXfbFF+6FqQy\\\\\\\/WbjSax+eTugIBrrPpjkDjuR\\\\\\\/pHI\\\\\\\/\\\\\\\/H\\\\\\\/bZGEjcQ\\\\\\\/CoC1DoKdXK7r96FNlEqI81NN84kqs4OLNUEQePvIg3S\\\\\\\/\\\\\\\/7trDdth8T5gvtCUIub\\\\\\\/6fbKcY3N4Q7gNlo7st2HWnHpQKcLU5yUV2tRj720HVi\\\\\\\/QolZBGp3EdvMujML2BcFIoog\\\\\\\/yOSpDiLAffkB\\\\\\\/rZCr8z6+MEPdixTCtjexp18K6vTRMJdnu38uJBR3gm5GrEiE05xF5OlkEPWV9OpeuW\\\\\\\/3TyHD6np3cxuvxWKPr9Qlmh4WGVKz53g4bjnZfyqm26DHmHbis2vckAAy\\\\\\\/pBLWKzEWcs9mxTzHrAmql2hBUL9H8tXAWhzvxMURM5f0FG29p44KPXOHKKFFiMGliM+xR2g+L9cyY4bhfoXFpPZe6c2YYOicaQBJgjFW4MZ2bL9KbUFD1WYZXbhdLw7kQZ61fZ1fBXbF2+qE8KyVUUd1s7cfKAd1B+kwFandx\\\\\\\/vapcVkwXzc2LeAFizio4AGPqRmmWXSKU4mozSaHBR+xG5oeRmP4e8Pj1ZA3dDr+u8+5A7WmGssEFqLQ9DEAFA0d5pSvHfm34qku8UtMOaZA1bQ2E1Ljm1EV6Ba8PnnYMe6xNrGSpcar6bbHEmoGGtXpmXZuG3pzbm2MdgXsTlVdxVy\\\\\\\/uKM49EOOrfUAQRUl6tJgyucEMfStris1nJL0w43\\\\\\\/\\\\\\\/8xi2F+UmhqkLg7BUI48w\\\\\\\/R+NWUZxMbLJ4xlLJKwccCAfcI4js6+hhBKJxqPYAQ1ACi32rybk2op4486K+zokCT5Bf1niQDM8K6t2UuByGRMNWcEDZZblesBL0ujY5BQmm7AtljSDAvr3ugi3McapWYyvJikARe52zLRZxOKy+20SFkD095mGy610dF1+Y1xc3\\\\\\\/YalVAsp3y6gmALhPLQh7IzpmD9o2T5Hhm7P4bb9JRIUWzwrqHaQYGvtWcVz9wyZYzdXIslNbddCgGpWXusb7HuVm\\\\\\\/OjWon\\\\\\\/6ZlsjHlbMpRP4sflpxFyXv\\\\\\\/UZjVf+G4TwNDTgJykGe7Jo8WnaoN6GYNbdTwKLWdMvXlijizRWo3wt0fEjeEuXhTuhDN3J+0UX1RU3b3PDdQ6rYmSMhbsmZHOCI12hMqrIrwWrXlmhDQ5kaZKAiWT2bFVisUY2DeXnehwR05o8Oma0KCbmqvsICdOsFKKUSga8Iew9LgRblN9YBkXtDkwUEst6Sl551EskrNZtrgAKt5qvkSvoaIQdJ9wSq5fmK07yvwskfkVP3B6yaHKvSd8ljyiqlP2Gq3XnEZG10utc6eOD48gvhdAOfpdlKPEqZt33BnhGn07fYSQ+ahslP3AqwLS6k8EvpVi7gdpII6s0UWC+tYRgdXx1iXhrVG2Xpc9oWfLWzLWRZBxIgvStHMGcT0eQ384+xae9Ae5KiIj5PcfnfHvVqbNHkqer9jrlUoD7ua6ZNCWArSQSK1i0o4UtzUKkTQzEzqcQVUSrcwChK2tKUUIZjJjPxl5TLqbmpm7WPVk0uAg3jB0g9kmKQ2saWDGQSNDXa9ksB7\\\\\\\/v4In\\\\\\\/hywK3eeT\\\\\\\/ARcPUuWCoSYBqMFOSnAyROirGNBniEtAnz2vcpHAxKQnMGLY2z+LnzvVhZYrE6tx\\\\\\\/1AR6K3xxJVA7yXkL2gPNglXiJf69OZxHqz0Z+Sl702T1hhMa5Tjkn0cr9ctj7qP5qLDlV5\\\\\\\/6BcF3tOM9rhxcENrXvXdqVGVpuk858qBUIgZ+ooHZxJ85kLSOC75KXqSWD0NFocIyohULUWcr60AqAgf0MO3PSAihpAHxMeZGXUZ93I1J4fia2XMUfJ1wS3Q9usrarbRs+GSJDeoBF7spA4hbC0uZqI++ngdSTq4tXwF3yNTsYrdB9d6dS6VpbwmJdEEq\\\\\\\/87aTja8yeL9yDpM8b5mQO7maRi8IxOV3k8KaTCDk\\\\\\\/IKEKdke0YWpkfA73Ax2q92HGR4GfYQwTijHAhu2D5ed5zoEWUt51m4l0h6Hww5mRzqlwzKKNCY4dvPE+Ly\\\\\\\/nz44CXqne1Rdg4FVA\\\\\\\/XSlkB\\\\\\\/KoBrsm3ZbHg2Kr+pLNzwJHHu8LQDD6hQhQBjxo2i9pRTvqUfUWzhuysrWGwQVDkN90E+NyZoHMYpDiJp6GtSBWbxwKpFqd7gd6l8jL3tw9bDiFXACarYenRo737GjLD6Cnym2uCveSO1jOwvpFiv+hpKvzp4EIKZAFvadi0n0B3Bsa6yVS8bQ\\\\\\\/wT21yOyw7Ir4H82WfZxw1S8rdOEYBot0IKSpNOg8RDcxemHoIGX4UD9TNpHMQup5kDZnqobkO7fiu0feIEc0flbxWHzeLsyYefLlYF6TQN7otjf75vs1Xg2QEebP5iwL56MqwO8b99gK4YHqVleyuVr8OVVbbBzfbnevGlvqGubw5CWUJwZOtYBB6jmwa1fLHhku24+1exTaoNihi7iNOQ\\\\\\\/11s1KgwZEMmkJSdmFemt5nRuKDvIVKnsWWp+Tk8DA3z0sIvMa0jYAtPTRkD\\\\\\\/U\\\\\\\/UI+uxHzXWbGdc9i9xBd\\\\\\\/\\\\\\\/PaUsFTt4kPS1tSTVd9DKEW9Nt06qLe5dr5zhseBTOTbDcf13qd+\\\\\\\/F2kvBbA7d+li\\\\\\\/Qu4jipmseyT7skfuWHtz\\\\\\\/u9qEEHgTh00HTHijne4igv0MX3JXWGsktGUwMqAO\\\\\\\/Y\\\\\\\/F+GKwVahGi2onE2h2JV7t\\\\\\\/ffXnzJy5E9fBdTXfPKxSagby6TRcBzpbgW7prBwrtyJXVzgwC3Y2hDu83SXn6CiQk6H7b5Z4beS4U7UGLcTaXsZKzWzO7KNFavT\\\\\\\/HyaUcYRp8zJ0H3ETbMNkegbAyYaHE41+f65BRmfmOiZR30NJ+GC3Rg0u8PsQvJAKETR\\\\\\\/+lVX7ZDF5RGoNqpXhPf1FPifZMrdCmCTfp3hY9j3UYv9i6LncdgejS46+1BtBcjrvzkrys3VMCvH3uoTLOnhiT6yZzgcjUc36LLTKouNvYS5cDCh8HN+YjJdL1qZ1sQRPJvHPBKeYAHf+p6uBZwxM4+89J7vBp8Hf10xzTbA+Su7eTYcBHkBzoLJWiGV5p4J3EcBwBArEf1+6rtNMjY+TziOkWZq4tb7ubgA65FN89BMpL8P4oFhDet5R0+cdLpLgoTix+9GMN+DXUTyzaC0wAluBDJ0tz\\\\\\\/dmDC8RGkf69gcSqV5\\\\\\\/7rHs04+\\\\\\\/pbFWoEbnNY1q30+uBPqxs3BcdwtUa7fLiy1HLAas3TiMUWurF481OLsvMhhEBGP+lBRbso6ulqOtQhkoPKeSaCVhACtEZvjMCDzkg0bIgPfKnrm+Q93dhIsCpiP8wHDzwtSXZyli+OTZNY1bynivvX5iHvS1MVu\\\\\\\/1TrY4VEpU\\\\\\\/P+3RKi\\\\\\\/DEUPW4WKdlLstxy3apoaT6bIGFqVfFeP4LWpo+xjs+BJfL779zbbuzBpD5erQrLbl4LMRgwfQD4M1nkQ+Iyxa6sdvLrbfUNGQbL282CNoEn\\\\\\\/9G6strgh5v2tOcGxNMdXZgf9lsWXzRexI3pim5XpGwZrGkhdPQVP3\\\\\\\/1rTU1GVRH94F0OHW4gPGay7D46XsflgBUU8jU46vLlT7tK16RGTr+aU9+HKa3TssgPgnwzMNDDhD8JdoTqjSo+cRDnd7vcoPKIx\\\\\\\/fZiv+yKEaH\\\\\\\/2opwht0xQvAH4h1me6m22ZNNkChJkPaEMX4ifztnjIKV8CKg7WBve6kzWmVO15ueI2AUquJPd3LWChR2oxzim+yqIZeznWR9F3d8qvlkecIkJ3kt0NKwSH+A6UU7N25h1jLkyPLM\\\\\\\/lcMh\\\\\\\/Luf8WWu5uXQIdebGOAFGFI\\\\\\\/hkdjEC7bkWzqub6sV0+wAa8qB46g46c02JtH57eLBC\\\\\\\/DMDA8NADIMvsiNj3mY6OzhDSlUMIGgDymwCHbIHnB++QI\\\\\\\/JT97SgB2xXCzQ+QhQ1E41C\\\\\\\/D0IHyB9Znu4FNxo4LP1H7v591WHiS3FHB\\\\\\\/NfSesVUJs3l5QU6CDLBKGGoV4IJzBDXwZyTVMdmTdHb\\\\\\\/Ix3i\\\\\\\/Jhp8PStgGbzDLgjifIW7I38ZReMQLLgMNhVvIeN9pbFnQHaE7Q\\\\\\\/rDQM\\\\\\\/52YrFLaRW39oydDCZpUkFhapgsPlqaoutBE8KX\\\\\\\/lscKcopDgKxU1ZvBT1GVWcHJj5tUuMl7VV3cXDiENGYwk0oiOgOYXVYx+MUbQdNyCO3gBmqYSACsjVH+uqeids0sd84+illw458StoYmw3R75XZx4yfKrPCm78g86yXNXX20OiawiuLingY5iVub6W5Ao3iJBBE7h5x\\\\\\\/9JxwVKmD7KyiXWoUtKR67QaRmp99TgLECbUmMX1Z8HLbSHuYDI\\\\\\\/pP4u7PGbslBT8L7BCxX3oVLReL57eMPFlOWFlBFkJ9VgmpbvYSqsU4a7aVeGTRTE4gGjBKZcHZjV8ZBnfY7PxiciYJDwo+0TPABeeOf8RXCCHERPw+Z4+f+LB0EMs1+8hgy7fbNpOm5x1G6VBfJGu7wHLO1WndIAgy5A4cxFrXz1KRMcjqMAKcPZx4yuj\\\\\\\/I7YpHlhDl4\\\\\\\/ccZXSH8A7dkwK6kBysUgB+lgIMdtNQEKT4fwMo+DAwur1dV3puoWwwNY4iQlDv96JL8b+wjGL+E6nrtZASmRg1jGTNB+8VKLote+DHHc+ph\\\\\\\/CpzoXaIZ1zNP8ZYEdGycktQDvQZWK+eMKU3NjKGdv4f0jJFDAmqUYzCJUKGHCBRTw1Pz3kdIGGlY+7yJCieJnEDc1lnGhJzqdDZDEpQwBkF5YD9X\\\\\\\/xKoOG9BbCEaqqgEbMUKUNL4Be5Y6i0JkU3waOQ10EgPknCqfo+5tw2URCB+14YubjCRMDfQCiq0pM\\\\\\\/FzT6Rw5zkBemVrkisgiEPRcLZqEefOzbFEPrxHx1yTHdIG\\\\\\\/SGsd2PnY8UZDpEH5GENxPciGb3zAAQS1DExO\\\\\\\/mYCARy1mm+WdSwbmIohntXK7JfjDo+k6nqU1hAP39ZlrvqaJwzYCUPE1gFr6ABwqvSBIE9aJrht5g9vMkWp5rqkfMbCV+3gFUEvDJvggQC02QPWEFc++lgYEbu9fLSB3Tv1V03DsEaGyduezBiLE+2xU8fEe\\\\\\\/aJYQOcty1Eye94bodvHINVgk6kUI7W9Y+n5KXdBf0Df\\\\\\\/t079+9MJ+\\\\\\\/afz0umrBUCDNNYlzrHfdKEZMNBmYa6Yy7YzqGCGQa2lYyc3Oe2qLCVFvmuB71+06MtkF25gBRplRLJiipqFQYYAY+WYLd2LK2xZUI5e7TEOGzZUmYc06vnWoUFgsmt4PnjDjKI7\\\\\\\/4QCEwlG6vcMHs1L6tQphnH7Bih\\\\\\\/DqBPcF5EmGdrWQcP+mh0QTkSdG2cV6Z7jhuox7pzuN68WvmpAjM33UNlA7Njw5i44DBgEooJZSQ9xZfrRvGtPJdK5nn1M1q7BXghNmWoHFjIiIeNZInSWiTiHkQQXGOH7+AsVi5rbpgcwYmzWu+vpfG7B7rZWAlKhZZqAYCGBSnJNu00q7xgZV11hyLbLk8N1Ru+gQLH3xp9vkG1JyACqhRdfd9vCWJZSxQ41lZJ9036t1U22kMHHAoaDckCFJrv6Noy+eh6gjMD4xAT3vxclZw8gNkkFP45BJ9WSekmdBYWqt0xER5oxfh6\\\\\\\/bAKoBY5KCKRBPN4BJi4eqhakuS0QM0Td3w+42J4lEsM\\\\\\\/dHe\\\\\\\/CuhrYp+3XHxbkTOjDY7+REP1ZfY2LDsSDBwDImPSt0PHnUpQtI8juVq3YLQkCy0ry+1vKay7PXBkzBsV0NTWDSYg3VZpOBV1wnXc3PTTwJDsL67yo8LwYvMJL6GJfEOPyiFamKy3FX15YWnIKnY+L0emroeF7wjMfPllBOI3T7q20XJRtJ7JTw7zSJBLseyPy5cdSCtsWh6kCRg0T4ONX0ifTZjFzCvwGvbrO6hmebqvjOMt8XhY1nOfODpRpA6PtOQrG\\\\\\\/iiL\\\\\\\/itmXd6S9eZUCn8VhSL6XOvsEeuZ5nhvn6BXttWkZC1LwmM5sEroMPIBbS4VP0FAIluNMCmX2QUHdXG0OGHeH35U113F5vLXeQS3PgjCwqKMya7UFEivu9R6yOw5CUK4ewQdW7nnrpYnfSeITMdoCrqQAlEkbsVG5ZzNVax5lOCaH9DGQFSVo7aqJupzbWMyKaeeS+3enAD3eTTZQUw6jEKMDih7nxgBJBjy1o+MlBF\\\\\\\/ZAzlidF3gv\\\\\\\/SzBungE2SFcK4qDieniZv6lOFHZaGwTWH1zTrueriFTkH\\\\\\\/XmSJiQuf7O\\\\\\\/\\\\\\\/r4WG+UTJHdlJ333KdAwg6TBlUL1jQwoq8u6iQ7OUWy4wLw\\\\\\\/Cp4CZEV6IkJFuGThE97zbi7QwLLC\\\\\\\/KFgd2Tg0KAaj+BwCLYOcCqEUs06Nf+8\\\\\\\/Bqj+ZlM+asngLHjMKk8g4kGhs2Y8zQdwr33mRu9vFRKcqO21b6NFfkYOGBrMfAPhivHVfHFXQgAxxRta8vjU6L1QTlfrWXf5TwAWe7YB543Xo3FiPzZXqertxCtGWUrtqnh93Aio6Kd592+kMaHwY8eWlbyryY8QeHkjUhCZfDVJMrqAIQeQS0tuxIREnxnsJtgAg99X3JDpAsRztnOsTtQxl8FPEESE4AfjmJIoDdQAt9QX5LeFO2xE3ETZH9oHSb4yQlSfQd5pnWtAieUA7gFbDBPibkRSjouizgtd7XIjuyBDnfQSKorOPO6OezIQmZOT\\\\\\\/zca8A6lZzfPjdX89Wws7d+nLWSXWS5cW1jsXtowr7X7Zuz0G8Ej90XCybzZRPaeewZvPo0aL2FR5sBI9WTozDWQZXOMg86nqp1MQsAro3lLXyXjcw5l8WutreWZaTSyu\\\\\\\/Ad50rDLCFa05iAEmfUN3I+Y48Gxf65Yu\\\\\\\/GxZdI6xMI59cpC148Y4D8VhF2crZBSaAiuMLE5q38dDOntKo1crWrzvq7bxOM\\\\\\\/LpGttEXES1RkuawyRn\\\\\\\/aoZw1feZfPbkAOzm77z8+EE4aMoQORKvULhXXBndTpdWNuduOc7o7wg7n2H5NkJHQrlRCIRnbGXnf0+KaceqV4lfoIrY7X1o1XdnpLmLYNmHOT4M29f7a5JDHkxeMFVeXrYemMirsbtkymUxFzuWnTn0tQAG6hXFnw3lJVx2zOQixnXqzlSVi3WuiCE2ubU0+h5Swhxs5mhneal\\\\\\\/x6SUpy\\\\\\\/tIK+lCEVMKZwQ80cGi94DjHtjBCWfmzFetNUmE2frFQ1373MriHJMSd1kQDK\\\\\\\/OyiYmdnM4wc9jBwBMxp5BLZnVa1sgh2pfv\\\\\\\/JhUAgnycCwF8i7v7bmQbNy\\\\\\\/sI66uTWpAMxT3G7jYEj6dOO3JbQZMjTsVeT5im75LvDkrfeYF0hAgsnT1LIxWFs5z5\\\\\\\/OKztZn\\\\\\\/zDgKw69v\\\\\\\/BpKQOgKZ11sSCZGHWFFMWjuKN3eC4lvbi4eirhXVJy33f72f2xW+Uda46m+1GEFpMWB6xnfCGY1\\\\\\\/OvzOY9uegBfko7cv1GMDS13EFAHubxcReCTvxn\\\\\\\/VeZytpCB+RGIrNpQatWWySPPbs2XHktVQ79Zd6XotBwpwrA166Fvj4MnTXFZmzwb9uKnUEhi3\\\\\\\/jCuuLN4KaTaMXxRubNefEgRVPWz2Sh\\\\\\\/hitp13mYxYZCcobobrUq5hi6fDOQGa92gH3iAvL+fLIluEiUe74bLbSV0CzW9dGyFRfg4H9g5OLdiwiL1aiJF3EChjnEhrFiPgA6E+l1NFoLckJIupsArU35HN4Wv0fzDc1TCRHBRXY\\\\\\\/A022SSAWmYLmXHN\\\\\\\/sjml5J6E1jN79Gv3AOF1eshkqcB9prRW0C6RiIfFwmFRfs0dANHxDvZ3Er\\\\\\\/T32+rmSqpB9Jr48MVUSMCMbqk0ilqYPpmF3KAY6H9uUU8ZcdBgX7rLIWWvycmPt+Pe4UMZSW8pwrBhXB7EEjPnYItr6VOG8BWe8T6gYNxgtTSumBEctMGgP\\\\\\\/D6h8ZsuPPEOaCi6\\\\\\\/8CueMBnVHKWPA3x67xaVqLQJaYh1p9\\\\\\\/NZWcozKvRBHjj+MtuIuroLOL12WYP8tlEQ\\\\\\\/wC4w431qbWsC8F\\\\\\\/AXP\\\\\\\/fIEUTG+vdfQH0PLPVJQ+91QATBK8vEdWdG4E\\\\\\\/2qscFDHll+zwL5XwN2YprFd8njxezpSxub3v8qyIuFH\\\\\\\/se8SRWpevw0uml2PCTa\\\\\\\/kSEU9BNvtDHF27hpy84734JnXS8WHvrVTimBwGKcoobbnrvFhvmQAeYeDOvVcMGtZ6c+sefc4RT0uTECmDmYO58sYwJ81AEfNxbmKp98deyp7nA0LpIZk66kKSNotc1nKT\\\\\\\/hsClU1w4K2IwJ28t8lucrRC030+kM5QhL0EVasflQGuxDZKtot9DC4RusY2FPaf1sAV6idNa7I0+wD20GrfWjSc4U92DSN7KZfNzZyAz+Z\\\\\\\/PDUemklZa2nkLbeWTOU4DA2QFzPR4itp7LbIwpvfxkdwCnkQSl+HMxz4iertPBS3C+k6OkFpZPwIcJgwMNaUsyLJxvra9RDGdBRtct2IR77wPF64QTlxo7V6o1hO2QY8uA75KCiFrTjSNisgiZWfnSoAXiYfLunG7QF9S2fcp46SdzLkLryBbHL7jUordUxZCpCoTUWjXMtK4Sca1Ypc98T0qGHEq4dP4WHgnJgLYLJ7hCcwAlZD6eUpU8TmZBkbWvIgNbXf+XgdGxFqZnuiMs\\\\\\\/ipqJdBXzYyoL9AjWdAJLFXu6bgAiTHIX7GEiEk0l05btyJ6bvBs5KEa6UH10OE77aL9Hl2\\\\\\\/jzzCSA4STMBqHnpDbJrXCOvOMi+ynxrwEJM4bbCx4TS11uaWZVNAVxtYrBI33ou3BNc88LUnz2Ir7mIEZnh1EK8aqyucoyqw9EU459foYM6Ziolo2JimSXz15B7prcBrMGrRanoT2ndDZf+\\\\\\\/ZZRkFM5E6VS8S\\\\\\\/G0ZBbZRsPTyu0Qv964wMfHbppyo0uMRq9hKoeKsmapdlzzb2TFnqxquGsqty1CZspzr4A\\\\\\\/LCVgxuZ6N6LiSeqW52wfUCJrcIhWkQ3rU4yDDVjtQXkctHlDeJ6iojh3iY8iwLvd3SJW2b6W0kPlgOirukijnyB0xWmde0hc9k7YfbngXQIJMwwM+UyKagyf7GBKnETFCufv6Pjws6gW0zGMPFfBkk2eFemyqK8hVkfrrLqC71xC1fE5D4M+wOFIHkYXjCbalhlim4QzxdG0EgTdk9h4E00yKAoubd6HC8NB7i9YtgFLE84q5OuuNW2UYu0n7jpHmQAYOAl0IuIPXxin2Ts6bYHlkFaGht9zWnSt9F5xfpx9o7yh7RYG6QL6Es4MY3TfSd\\\\\\\/5M4nf8ILJG+9cLZvLvfbCrluMDcgjdR\\\\\\\/QpHZfd576BEu0ETZrZjoW8g3PzFWVJxvCGnofAzRkRwsegMtMi2vHWMLCP9lxkZ4I\\\\\\\/Y5Eun0iRuCthsZvgLYcDGtoBjR3djyxVk8+rnAe6yD3jlvqlUV5g+DzJOGLA0wD74KhtW6UXgd8Tk\\\\\\\/BnqpXTB\\\\\\\/ZhFZzQWQARr+tyMqc3QyerZkGvachxi8J0siyjUbP9j0kwJc\\\\\\\/MNtd4YzUHOerJkcOQGmo\\\\\\\/ycQnUFfn9F5Wr9GP+D0Ui765ns\\\\\\\/tuCiwy+YN3AhU36i+3l7GXPvwO\\\\\\\/TlkXUmLXjhin1tJjPz6vqJI3UaMv1uv8w7dmng\\\\\\\/kwmOonASixtUmAzHb9i8a2+szwWCls7rxqfeh\\\\\\\/McPcpyMDfiNIRSzD0dahojWP2UCf7BS+\\\\\\\/VDyYbGVxbcvU7Se+aOa4JKmVF8tfq2BGz8Z9aBPUFwNJAsR42ZyiLqu\\\\\\\/LNQv69Spvtm7WBuBpoNs88iLoQazspxxwJp6qvjG57QNioLEYSHTHnmsJMiczZgFJJav932+g6fqRzn0xBOQfUIw76\\\\\\\/au73+dXlVElLJoMf1xQB0+hF\\\\\\\/0AzrYeOJuAZqYzsX7EKhwpLKwDoa4liAsYQzJcfvXnb3NEHozi5yqfWQf7GiEJcFOwtz\\\\\\\/wCtoJKiilxD1zMK52oHEnhnernVmnm7H1wDuzFz6uFRGKcb610Ke1cwoX+RtZwF+UY27PGGTKBg9JAzwFfwKAZHxZUAEaxXcKVD9b2S6xHtc1mNYHJD+VlYbJ\\\\\\\/3r1MiIf1\\\\\\\/oEjJZepBVx3OiQX9fa5mD5PHYg093s9wBkBEq1V2ZnzzHuOEa0KNI168jZLZeD67OuJ5YPTVozV7t02v8hGAIU3Kl7ST3GoDrrkRsd2RnCebRsGSEcjDriVqJc6\\\\\\\/YRLSpaV4Wd6xhgSpQGcIxEYEXqK5lE4zeRFSXqEzHyBojr\\\\\\\/uSwvMXfkvhlc9qBmF8dzFeauJ3l3PvJRjthTH5Zrxg4CImGrZUninGQjjdObgo6+hrZNoovHU8aDl1XK+ziL71YcNUv5YgWJLh0EXprRCF2cbouETKu57wGm5GxixweIjbpel4aFCZIiiQ+cArDQPufYYGmBTZ\\\\\\\/rIiGk+WPK38xTkR\\\\\\\/V\\\\\\\/0xoLDHFLj3nckQWFPVFlz74hjiGpNYno+7C7+w\\\\\\\/nnMTy48dprTsrx2x1DETzWgHCHRYNqUIRH1XOjGQaqz1hmt9n5kPtMbKOY3vV5RwG3e+Tg91Fmf9+J+osJw0wkyOD5iinuYSuqG57QwpwugJMRSd1zl3Lm+wuwnsCk1JHWIC6W7TgJbqULkTwivGsE7zFANUbFX1lysOwo8aJME5DpxJJLbd+BtYmzKruOGdNTkYDaqv2GrWscIcvWDfEdv+q5zDAuz6PfuRKlmydsabkshvrYWYKfSfthPwHXxEQ68NIRreF5qFaFUUtQ\\\\\\\/58zm0p0\\\\\\\/W2Tfi+uHmRVgb4EgxAPxzMgk7STWijFRykpx\\\\\\\/0E\\\\\\\/ndwNKhJ5GYR1kU2uZeMcLMpgr577rItxbCU6YKfZxS96MajnKgsTqCVmqwowXiJHydhUtbRno6V7hKP7q4daWG6fZL5i\\\\\\\/U0L4otORSBunTQYdEqM1OsqKiWtD2zEI6OITsKnrovfvdGdr4J1I0SHC22oTC\\\\\\\/elVAZyurrZiusghWFPCbfxJKmFJ0iiXuAma\\\\\\\/DusAq6iyBE6ZqsnlZ05BfnuPtIRuI9VSWOCy2O9Wyi8cQXUuftA4axMKMVL1+VUYsZqbI9eDL2ueJof+2no28zvKDC78\\\\\\\/3e4zlSkN8dWVW8nkgOzGIDfAF5paMEmsYBMMAdNXeVfyslBfWnjpZh7gNqzNB5khtr5x7jFcVD\\\\\\\/KBKGa6hKdpYq6OWKpDwvPOLWwVpnqOzIyc7lBc7KPr+2lD1DGILfyotLm\\\\\\\/azELE\\\\\\\/pijh7oKQIM4bLrctfsiR\\\\\\\/Q+FaYdF1SrRBItKaDXyQ5cFFH6zwegkL6cB+jBBwxKHtciuRcE4zvO\\\\\\\/lW43NddLc62rsknEj+vkkPnVKKG3X9cKSA5XGTjciRCMp68iaPYSJAng+9k0HtL9i0gIbCQFnfPs1Wi5xupxS2AgCD2qsKUIzX8DCzP0OC\\\\\\\/BSg1eWHvHPUgxA7XHpJfUdDEoLspJPOusjxDqvEQ3i976Xsif9ZORx10SV6W6rWfvvM3yFYQOsJC3wwGwPdhpcWzkk11APiYHuNEVIWQF9XxONJMOM3+bjIF34Tq+cO4ozvFFozilSrebEKqr5YomPJNqswQVt264Z4WE5OLtJ9sqtQnDLlOGjCEcUIzVpCT8wQkPpez6HcAVo83ZInqeHU3nuLRC\\\\\\\/C3g80jBgTS\\\\\\\/jwmThjHpZvGywb7oDMyqRBE3bjHAczdtLEJ0ByT9f+RWp5+fBSou8Uis+ERbDc2R3mrpTDkTK7WeOO5d0ClfhV9Nojg6Vt4HZCVGC\\\\\\\/Rvf2zWh1Rw+1Wk0dbpBkjRX\\\\\\\/gG3O0uiTtOmX1PKGnKxU8ULAhzL\\\\\\\/nBfMr9CjgIP1xziOc+V6W038BelBOWdx7nf1gF3\\\\\\\/oeqdYC14mm6BIO3Ni5aezDcMz2oVqt8n4CeXSAHA+BKFRqKiu7jD\\\\\\\/At8N8fJKgwCz9C9dlIGSQiYwQ7tfS0jOPSPq5B5n1XqDIhASlZN6Ox5mDSBQWv9z7ARVKLAZyip6I\\\\\\\/U7nmzjceBeNwL3wK7u+RMBaeShU2gex8y1FxyW3o6ghAQnNnGcifmsxDYRxFD\\\\\\\/fJuLSErf6Iezsf1NCtJsS6qTfslxvWGaG3nJrW9xHI6dSd6rJNa\\\\\\\/cqzTuf9xjMQ78D9IwTov936wtUU99fVr72U9whVlLMtfscCvYvM5y8YtZlH252bYMCaUSgX8S7yaA0lJMqlgmGM6VKDFDNKhEL7klMdW4BRh1vxDvGdHGcIB7pwMHpGgwM8zPEpTz2y6Wkgg29yrVZf72aQIb8XnvcQsw3bOuHxi7xl54OoQkYV7+LfmDGp2tpAlvVBVuvm9ApPuA4eyvLu\\\\\\\/flVkW\\\\\\\/nl8tKdhR9wWCr31oL7oCWZ0OAMaKTLfkrO\\\\\\\/lRXlyeS0bTdlzr37ONd8sFsBXQaCBzUTqE50JRv59E+X2O+dB7FgXGCrq2u7dN9SztaGsCZM63naAoO4B7p6ZiEtNFgjG2+5T2WpwMFkDBvd8GZ2MHMaX4auSk3gsKa4tDpNBOgdRGTGvuUT+E\\\\\\\/ws1baWCAEakuu6DcG5+wSGg\\\\\\\/Ulngn\\\\\\\/TRP+Rp+VI4TF2PjfyS2EeCfRcBbW2a\\\\\\\/xyfNNXTP73rfMPFuoGVk6KPSFdHq0xAK+wx9vpOg6LGVtxCCU9xa3PD0X6S6QdRmysWE3SN7vr+8ZMyp0T0HyeJPvTH7NoDmVwooT04SIzMp0Y5BAzUDJ95KEq+nxiFGS5UfL58kKd70U64PB1aLglutvH3uvuqyKzTeeve0HBA\\\\\\\/6VFsGUJvfObYXPQ54jhoTOamT2YyzQc3RnCbPb6a0y2\\\\\\\/45CblqO7V4lLvtr6QY\\\\\\\/Xfs6qtFSF82YW+EUjw5sLoVw2\\\\\\\/+GTD3091bYOch+QvlDyeU9EBo5Ii8tgajgK4rnlVxw33wOsuWJGcxuCmNoiDKh8pbn7+KFQKje7ydPvfWWzUwliXyigeXHv0dQkyKu\\\\\\\/Te2dJ77zXUv68jUpcLJhUpX0HXVHkb6gchUG67nwUbAEb4WQjaKV3bd66Fc2tD9+V+UBTgKrLz1OiKOxjqxDKvuFEiYJMeqO90HNkCFA5IcbbiGa7rMZL4TLZg02dYJjRq20+0dHDO649S3Drl+xOG6jIvRnOJOtZX9V5085pUp0UzcaDwo6H5cYruNI5SKVjFWAsAbQSQyjOTE9SYpbcrLvTww\\\\\\\/S4Y7I3oBvIXNIJdFk6hzcAQUGn\\\\\\\/PzbYU3oVIKP4adTq08ipFhyNS4iPtrobocWb8dv3H9dZ1NsGxdr4jjZwpV48cRsd2O0yKJOTLkln3ZyIt7Gp2CSU29XJdHIorYIkmxXRW5reYmzXvhRE1MtustZInjp5AL5ok61SixnqZwh9V5gIlottbZgf13aBgYwX8eTl3BBsIeaji2RVUspD07bbr1hq9IwxmTOiCKj\\\\\\\/mZ7SCBTGTN\\\\\\\/+9Qq7PJdeZIbVoSyNj45WkQOhZxgJVhH1qaC192KY3W8cc\\\\\\\/cmHrvjDRfXR6N7MGRs2uxjIe9vxc1gJADzE1ssgQ6u9nRLwgLhrtLZdX79BtIqNCP\\\\\\\/kfgvCjem420tLnIGMGomiHacGz8LA4qSGSAYmYlpVQLw9GxP\\\\\\\/9K0MU53966usp9pp+NWsNL56B+jAeaf6BjwM++azhEice6t4uR+2WXg7VJ+9N5aW1JneiREC8XbM13zX36NAyV9Y2RXLwd3wS8q\\\\\\\/DRVltGGchaJr2t\\\\\\\/4LR5k9rlvbvcDpS+kJfR9PWE60Vbl+LKeYCJjJ2JGCcysyRDUGICw5EbHgR6CllBsyVpgAumGBtTMR3VOoxLqo1zTH2L3M4BO05v2UjFTpYdhFVoX4CGfpQcPKtx1Hyv3sq\\\\\\\/kQn5xmEJK+VAvuaWjJ0FZ5sDfbHMr4ZO+NMDir8Zmm6i7P6z+wY868kcxZNwZg5yzfeNOD9UaY3jg+k6b6LmPYoTjwetpBecnRzJDw\\\\\\\/V5k17BE1obnO+jFBVSQg4cC1DCBkxGIsnUDdgAQ7Xzvv68AOkllRRK+6HCafF9yNsF3nk6io2jY+kk6pdWMtAvN8j04PL+2AfrPDT6nwlb6gEtc8NxXYTiS\\\\\\\/8UqxzocHv4o0IMQFwBXB3PX4F0sdgtKTKX9HObvVruFvLHz4eXc61Wm\\\\\\\/5jCge+Aj6d10HMitN997deCpMUlQ0RwmwOsdEuRDjiOEIUNifHuLB3uYfd1VRhtgE55YUIhX6zYPSbgsfsUsLigqlAKs600jb+Wwqthv8kKf3lXYRRgb8lDkQs+fDWbw59sJVo6L4AnVkwQuZyebSIdTQbWoF1BDGK96FpsOt\\\\\\\/9YngR3QQ\\\\\\\/DtZe+ebpLfRfe0uv0T+crxLmBMWtnivXG+1QVlV4iqR0vI5OeONXvDQ4GZeIvnbsuMXhUHFit0v3Shs5lTbrlXJhBVlKAYUKHEVPyzZeMKBS\\\\\\\/wuXbLIDSivF+oGHJEBzqUVGQliZEzIz0VBsE8Nm34RFH+6\\\\\\\/8qBU+t3WMZGv8WzVnmajvjMAiebd7lDVEoYXIVy5iEKRibeathuvNGOLgQ6nYEJf4qV3d9A1NrKbcrtcdiSZEAPYDBR17eZqmzyt3qn09HYlZDiIrX4qdyrV\\\\\\\/ZpvuNdKM3b0RI+lkoJLD7N6xUF6vDsKgqYfG8GCVn3IPwZu9ZPI4ikCwO\\\\\\\/2GS\\\\\\\/tu79yAgIA5XnGitYPQ2C1foH2j7PyvRXeuwRNuoVb5glsszAkk9Tz6QogbRBuQO+5AQj5MW1DfSzSwnjc1Q8eKGmp\\\\\\\/+58Qr3BXDnNvnVux2TWl8PhFgNqQM4q\\\\\\\/jaT7NH9E8c9AlVXJ4yl3D52\\\\\\\/EXGhlURvWOg9wYTrPMiFsG0zPYyXAP3BYekgnfCGeLDmzx20o+k5zy2C3cI1xisjGZr7UwysxOu0h+dh9le\\\\\\\/HUnsvak55r05zkW\\\\\\\/zz62FSpfHIcPB7m0ej+WImrRY+Jwm3IUzdPRL62Q9\\\\\\\/eUxW8IgTy624n18l7kmcSWXJgh0RTgCh5irdKAj73+pkx0jZbHB\\\\\\\/CzqALarXVfy1cqWnCQ+KPaZZF2EFYvdIYJTe\\\\\\\/0xg4K1In6jqSlapSPYSq25A\\\\\\\/hhxAP50f4YA5ZFSVCilxZDQraFPu\\\\\\\/iri2jeTnjjUoEpL1Kaw2j1yIuTIhKYt8GxGj4y7NA3BQsoQq+McCy\\\\\\\/A7+NsTffi7QW52wgjINnZ8A77m0S10PQ\\\\\\\/6LOArMKffRF9ncIXQvv7LsbdAmlEZc25TfxDMp3xay7zKSwnAw+HoW9IUG2JQk2swTnda\\\\\\\/yrrnJ0\\\\\\\/tQJT0NYlfharRa+FwQkarzPNd0IxrYAaVf+b\\\\\\\/x4nPEbN\\\\\\\/ZoE1e2+mEZ5KwGrJV8uNLEImdgkQU9XL\\\\\\\/QMuuinX\\\\\\\/4bBHWqklIObS2f0Lj4481TFc4zyRyIPlaru7lTA3QE\\\\\\\/FDCmvF8lt0RVSIiBMatu9ABGPvtL8NcjCWN18oXY+otSzl2gGZ0dFGWxHwUtt7V73cHmxfLBysEJlrIkwrHIV0gm\\\\\\\/GZP3KXfKq86I5vLIJT0XR0EvsJXx8i8rQKq9f1gC5XDgG64N6XKw3mb+mGKBY\\\\\\\/NinqIfE4Ecp0yo4VR+ys4fPSthYQFVDwHnICNU25eBGVxMqGDiT9Z7XJwPfQgMQPVsvqS68exbVQ+Wc8HDgsUw5Mycl2ttiAI+8FQQg99VVbT2XMJrY1b4fp9aJZRTGsuSRuFVObNxPKvYpHFitCJNX+OqLwd4IHNw84KRJGwT0HouVhzfn51eVXUZeqoKJmGt9YHitWIBA\\\\\\\/SfuH7+AyTf01IfQiPwVebTnR1Dv0u6ewTDmL2\\\\\\\/IS3cGNonMhgcBT+2PIxiJcJxiLcu\\\\\\\/GwYVdxEn2jTLVUp9alJ1\\\\\\\/onWqlnHqregfMWweMxfbkO4RSZtxNlpiLttXi2v3g0p\\\\\\\/Hd5MWoWq2XShlBQskkYDe17qjvtAJ6EtaB9HmoiyLT39RKwlt0NCv2bAjdR6bDX5jAhttlEyZ39HJVnQOjCOtdw35I7fvsAm5zE9wXKh\\\\\\\/FDW9wmj2iOsF8tvuJieeZ8+xoUwbcMLfxrB+Gn5JYaDqZpcT5YasPvg06g\\\\\\\/5jBjkT9rBbJAEbNaD4NWIlaClLbrGVoAOZzJkt8UGTgQUzPL15XevlxJdO\\\\\\\/Um\\\\\\\/FwrJCiHCyHjHhFCQwzYstGZG89Somem4BQT4pGxvfxDJxf1J1RaHEWFyVIj\\\\\\\/Hi760UrBB5FOULyim7\\\\\\\/CeUgVc1lA8VxGtE\\\\\\\/UXQyGnOdiYiHwrCeOllAlQoLwf0ZK4mBeX\\\\\\\/S8BaTmm8nqVxMoGWrsoNEDtxVQOlSbt3ZuFAyLaFgUZIGpz0yh2SfQffUMplFW3iEexC7PcD83\\\\\\\/PB9KCQv0\\\\\\\/hs+2JD\\\\\\\/Wtl0b324jcIF3ak9kQKF+lQX6N5NbZceW9uY8rB9eLSeE3G7b+XHnGniufi8QY8krPwnTrvNTrlZACcTxKjnTCotDU7nU8ie9rp42bDFG5nU71uIsfaoXiJBgPOCGPlB\\\\\\\/tgfkN5GYH7NM5ia+HO0wTBaOZT72AI7tekZN6r5QMMYdrDKEu1L7h+jEook5KtWdb4CDUzuZk9IFlIQgpQeTAsql3JX6Nw5W+ZTQ36l2t89Oe8AhECLn48+YB2c3egkG2gYYJr9kHQaW\\\\\\\/qqGuzeZUoFAqXK9sq64kE0wlnIQjepMin3uYUHgZ7fjGBz2nLGLMRuZUTpQ3GTGW2P0qz1solInoLV89XLEap13psn26ehEtxFqB3jzkZOBvPumN9pb1408Ai+mugj55lqgA1X3XNCO9g7Wn+nSmULBG7jK5Z5StMqDASOpUZ3BFEoQt2D1dj5pmI7Cn3Nchentu9TNiRlBPe9sly\\\\\\\/Gm63u+IrvYZOBmyCgWNSdk\\\\\\\/Y0oNnkFxYXDGkUPv07wF7Zns4jV9xYKpsJs5obz1s8upitIscOOQXskHQbw8KhV\\\\\\\/rABXgpo\\\\\\\/hHF1Gl1rxz0zmztbY3HCU6Sb+vyGnpb8rQ\\\\\\\/gqKcz3N\\\\\\\/LD7r2s7yFFfpHuSvrmwXezpkQS2Sj78hcbaZeHU79g\\\\\\\/GEAg1uMbsX\\\\\\\/tktxzJUU0UZJalvXg==\\\u0022,\\\u0022iv\\\u0022:\\\u00221pgbaRkgfp3fyq7nNDu0hA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002268c3152bbc627009aae6add009819d1f509237c1546987043ac4c08dc45e63c6\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00228qMuTUIr46whMLHqyUQyN2zttPiPoystVniGxk4+hKFmNsJyl+IMc903+HxEWRRHSUc9VLunE4FfGGLRE5CWZQmXVoi3T1hNbQ\\\\\\\/gVs2P\\\\\\\/huSqOeYUHzyY8mv9eE\\\\\\\/yQynLJjbGU2yB4rQ8NzxMTaX0PMUIPioSf+q371JGxgPn1NK4zUf2bHcXtP+ZyMg4H1gy93Hxd\\\\\\\/7tLc4xIpEQqYgfxYMWvjBfIKMIlCxaR8QmAmqsdtuo\\\\\\\/TdouMV\\\\\\\/WU5FUXqUkH691h+KzUlvMC7Cxmk\\\\\\\/MR7ZWVjUq88GRyq71nXtDgEdQxFThGNDvvUpsRTVCYHJE5j1y2gqlMvWSygJ8ZyO778kJsnyIpBHUFHJyzFNkcKX52xlkBeaM7LjVOhQFZtZo88irxHX8edJc5W7qGTkP0rYRYYPYhJERe7wkZUmSNsmLv7vtLb9QqloEknlb6gASg4nEBm86vKydgLYeTkYvMbSqzseRmSNTVn0xv0WU8DvIsr\\\\\\\/SQMLNaTdUiJMEts69R83YN9ccwMMBuH9BSjzULABCnkZaF\\\\\\\/GZJynTEFZkLbwD3S3ZPjuVyQToLeHNSuRClDqdDG1uPdc5So4zpfjCWyDec3eUM77fjqDs7pfahtnzSvNr8zqKxJveSmJY+cyCl\\\\\\\/qgZWKxpCi2Y9Zw3M9c8saIkhSDIVDTot+GgcyehL5Ahu+M\\\\\\\/kT3uCq4w+gC+DVOL+XqOMDNQwg35tWuayrgW36aXLaUeU\\\\\\\/9JYeZYzbvzYRKkMeSzqPDyBJDTF8eSFLrKJUxTf5ADDCh0US6BROhFgpAOPpswSkgNyFjniDWFhOMwHYiqWddwM0LhyWb75f0+qexmIrwEEnxLIplVFI32hiVWrRUvyqYG26s\\\\\\\/CZ7wBTjw1MrOSeJmwjqxv\\\\\\\/\\\\\\\/dLco2a5EW4xkVradQmKbMkST4r3eLFZAKeaDeIKKQLxTWQ6juTSGl1VYEhWZ3q8V4POr9AwqPjNyjA59Y6tdX9m9k1g5BgBfkjF3gybH9SuFZA5Si0QK+V6nN\\\\\\\/YPmehQ9ccqXbDIXhxYE2K1slEyFtRZI3+z6upoXdI3AR6RkDd2U6vXGGorIQTQd7LKFz131S4fMMBgQnLz8+WV3spG1M\\\\\\\/UoBBpeaudocnV0EgwdwxlOR2iQu1lP0oB3qcIB89QbKUlAf9thzkEnvyyHxvENB3ZiwGWb5BZVwJbUUoBreNQNVFdVnc6RyOmMMHEEaBg3s60jZpcpmxy7t1DiPsGEumdw3BO1SGhQBceJ1PcIvMN9IlKImSIPnyu4xXH+FYKgUrJt9P\\\\\\\/OMjVwufgMZGFpx0yiXhDwX3SP0sXAGpjoj2R5tPcyhoJlMqFLqn+eMWZ9NuZJ0\\\\\\\/55uZiHAvWki9y5CkT+xVNpKCr8eKul0EbZhgdbcIdInUzegP11CC6BDaJSnA5XprTnef5v0L5PyHGCZXbLj2uussZqhHCFeefsunYtddR1o6lZGZyp7LQxrU2zYk1+s82aROAibBE+cJUpYNkpqgH0SomX3Yd6Oj7k2ZmtBPDLyiXoroAH+rBN2GRSVzRVR6aG\\\\\\\/CXMoPIHLv6YP+TWL2Er5LsicKFV+WjqCZ3SeUGhh24Wnl701DAqsNB2eLS\\\\\\\/ZzvtXFOMH2xwmUzhJ56XN1C9S3KzYEyBOLJm3gL\\\\\\\/yyDKF47ZF20TCf8YyKSxcRV8zP60J\\\\\\\/8L+geIMvP354aCPZyoKSQV5ud6IXpJYeEnH26GmYpZmKv3sM8SeNvUkDdycasBA6fDsAcfh4oWD35JRmXVw56Mjg8tvmSgMBeI1Nhb5+laQRjNxmcmzkxmY2VZsRm20yRNJ4BR2ZHX0Ky54dfp1xdb7T4PYhhymptEsssehZtBdKC6PvwVb9v+3Xwoc1m\\\\\\\/RmNEqn2GS8TFRq7g53OB1UtPN1kpK7W2DWYPn4+zTKzltgcQdUe4rF85yATbg5sbUc6PJagxhbTPvImFbJ6YbZmSuhQ2k7oNTE8A1fWym8lVFYr\\\\\\\/re6Cwn\\\\\\\/Vtgj9tihIerrrfOIUst\\\\\\\/5E4g==\\\u0022,\\\u0022iv\\\u0022:\\\u002243vfSDMtGmyYaF5FZS\\\\\\\/tkA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022dd3006f910726657d5696db20fd75d1caee09789afe1a721767a7356b3d1e1e2\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-490964 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-multiparental-populations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/10/3231" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-AN4sFIc5cXoN6O15pdetVh0VwF0RJcP_J--eR5AgHmY" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/10/3231" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-kVqmgsQazp0wpUyAdePatkDXWAIrNUTzex5WLj-O8yU" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/10/3221" title="Null Mutation of the Fascin2 Gene by TALEN Leading to Progressive Hearing Loss and Retinal Degeneration in C57BL/6J Mice" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/10/3247" title="Genomic Structural Variations Within Five Continental Populations of Drosophila melanogaster" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490964" id="node490964" data-pisa="ggg;8/10/3231" data-pisa-master="ggg;g3.118.200340" data-apath="/ggg/8/10/3231.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Recursive Algorithms for Modeling Genomic Ancestral Origins in a Fixed Pedigree</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-6030-3933" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Chaozhi Zheng</span>, <span class="highwire-citation-author" data-delta="1">Martin P. Boer</span> and <span class="highwire-citation-author" data-delta="2">Fred A. van Eeuwijk</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3231-3245;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200340 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node490964" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Chaozhi Zheng </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Biometris, Wageningen University and Research, Wageningen, The Netherlands</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Chaozhi%2BZheng%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zheng%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AChaozhi%2BZheng%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-6030-3933" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Chaozhi Zheng</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Martin P. Boer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Biometris, Wageningen University and Research, Wageningen, The Netherlands</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Martin%2BP.%2BBoer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Boer%20MP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMartin%2BP.%2BBoer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Fred A. van Eeuwijk </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Biometris, Wageningen University and Research, Wageningen, The Netherlands</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Fred%2BA.%2Bvan%2BEeuwijk%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=van%20Eeuwijk%20FA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFred%2BA.%2Bvan%2BEeuwijk%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/8/10/3231" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:490964" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:490964/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/10/3231.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:490964" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:490964/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/10/3231.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:490964" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:490964/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/8/10/3231.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:490964" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:490964/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">The study of gene flow in pedigrees is of strong interest for the development of quantitative trait loci (QTL) mapping methods in multiparental populations. We developed a Markovian framework for modeling ancestral origins along two homologous chromosomes within individuals in fixed pedigrees. A highly beneficial property of our method is that the size of state space depends linearly or quadratically on the number of pedigree founders, whereas this increases exponentially with pedigree size in alternative methods. To calculate the parameter values of the Markov process, we describe two novel recursive algorithms that differ with respect to the pedigree founders being assumed to be exchangeable or not. Our algorithms apply equally to autosomes and sex chromosomes, another desirable feature of our approach. We tested the accuracy of the algorithms by a million simulations on a pedigree. We demonstrated two applications of the recursive algorithms in multiparental populations: design a breeding scheme for maximizing the overall density of recombination breakpoints and thus the QTL mapping resolution, and incorporate pedigree information into hidden Markov models in ancestral inference from genotypic data; the conditional probabilities and the recombination breakpoint data resulting from ancestral inference can facilitate follow-up QTL mapping. The results show that the generality of the recursive algorithms can greatly increase the application range of genetic analysis such as ancestral inference in multiparental populations.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/junction-theory" class="hw-term hw-article-keyword hw-article-keyword-junction-theory" rel="nofollow">Junction theory</a></li><li class="kwd"><a href="/keyword/identical-descent" class="hw-term hw-article-keyword hw-article-keyword-identical-by-descent" rel="nofollow">Identical by descent</a></li><li class="kwd"><a href="/keyword/ancestral-inference" class="hw-term hw-article-keyword hw-article-keyword-ancestral-inference" rel="nofollow">Ancestral inference</a></li><li class="kwd"><a href="/keyword/qtl-mapping-resolution" class="hw-term hw-article-keyword hw-article-keyword-qtl-mapping-resolution" rel="nofollow">QTL mapping resolution</a></li><li class="kwd"><a href="/keyword/collaborative-cross-cc" class="hw-term hw-article-keyword hw-article-keyword-collaborative-cross-cc" rel="nofollow">Collaborative Cross (CC)</a></li><li class="kwd"><a href="/keyword/recombinant-inbred-line-ril" class="hw-term hw-article-keyword hw-article-keyword-recombinant-inbred-line-ril" rel="nofollow">Recombinant Inbred Line (RIL)</a></li><li class="kwd"><a href="/keyword/multiparent-advanced-generation-intercross-magic" class="hw-term hw-article-keyword hw-article-keyword-multiparent-advanced-generation-intercross-magic" rel="nofollow">Multiparent Advanced Generation InterCross (MAGIC)</a></li><li class="kwd"><a href="/keyword/multiparental-populations" class="hw-term hw-article-keyword hw-article-keyword-multiparental-populations" rel="nofollow">multiparental populations</a></li><li class="kwd"><a href="/keyword/mpp" class="hw-term hw-article-keyword hw-article-keyword-mpp" rel="nofollow">MPP</a></li></ul><p id="p-3">Many complicated experimental crosses have recently been produced for mapping QTL, particularly in plants (<em>e.g.</em>, <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Kover <em>et al.</em> 2009</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Sannemann <em>et al.</em> 2015</a>). In contrast to traditional biparental populations, multiple parents have been used to increase genetic diversity and thus QTL segregation probability, and many generations of intercross mating have been used to increase accumulated recombination breakpoints in sampled offspring in the final generation and thus QTL mapping resolution. The expected density of recombination points is one of key quantities for optimizing experimental designs prior to collecting genotypic and phenotypic data (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Rockman and Kruglyak 2008</a>). Furthermore, the identification of recombination breakpoints from genotypic data provides useful information for increasing detection power and mapping resolution (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Xu 2013</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Li <em>et al.</em> 2015</a>). The primary aims of this paper are to develop the theory of gene flow in a fixed pedigree with arbitrary structure to calculate the prior distribution of recombination points, and to apply the theory for ancestral inference and detection of recombination points from genotypic data in multiparental populations.</p><p id="p-4">The theory of one-locus gene flow in a pedigree has been well developed. The haploid genomes of pedigree founders are designated by distinct labels, called founder genome labels (FGLs). A set of genes are identical by descent (IBD) if they carry the same FGLs. The identity state for two genes is either IBD or non-IBD. The IBD probability for two distinct genes within an individual is the inbreeding coefficient, and it is the kinship coefficient for the two genes randomly sampled from two distinct individuals. <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Rostron (1978)</a> developed a recursive algorithm for calculating the two-gene coefficients of inbreeding and kinship. <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Karigl (1981)</a> developed a recursive algorithm for calculating the coefficients of the fifteen gene identity states for four genes between two individuals. <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Thompson (1983)</a> showed that a similar algorithm applies to the probabilities of joint descent of multiple genes from a specific FGL. <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Bauman <em>et al.</em> (2008)</a> and <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Zhou <em>et al.</em> (2012)</a> developed a recursive algorithm for calculating the ancestral coefficients such as the probability of one gene descending from a given FGL and the probability of two genes descending from two given FGLs (distinct or not).</p><p id="p-5">The theory of gene flow at two linked loci in a pedigree has also been developed. <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Weir and Cockerham (1969)</a> and <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Cockerham and Weir (1973)</a> developed a recursive algorithm for calculating the two-locus inbreeding coefficient of an individual that is defined as a linear function of the probabilities of the fifteen identity states for four genes, two at each of two linked loci. <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Thompson (1988)</a> developed a recursive algorithm for calculating the two-locus kinship between two individuals defined as the probability of IBD at both loci, see <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Hill and Weir (2007)</a> for calculating multi-locus IBD probabilities in random mating populations. In contrast to the identity coefficients, the ancestral coefficient at two loci is defined as the two-locus diplotype probabilities, and there are <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-1.gif"/></noscript></span></span> possible diplotypes for four genes in an individual with respect to <em>L</em> distinct FGLs. Thus, the calculation of the two-locus ancestral inferences has been developed only for simple breeding systems including selfing, brother-sister (or sibling) mating, and parent-offspring mating (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Haldane and Waddington 1931</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Johannes and Colome-Tatche 2011</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Broman 2012</a>).</p><p id="p-6">There has been much interest in developing the theory of gene flow in a pedigree with chromosomes assumed to be a genomic continuum. <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Donnelly (1983)</a> developed a formal mathematical framework, where the crossover processes in a chromosome pedigree follow jointly a continuous time Markov random walk on the vertices of a hypercube, the time parameter being position along homologous chromosomes. Here a vertex represents one of possible gene transmissions from founders to all non-founders, and the number of vertices in a d-dimensional hypercube is <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-2.gif"/></noscript></span></span>, <em>d</em> being the number of non-founders. This framework has been used in many kinds of small pedigree systems (<em>e.g.</em>, <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bickeböller and Thompson 1996</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Stefanov 2000</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Martin and Hospital 2011</a>). However, it is impractical to apply Donnelly’s Markovian framework to a large complex pedigree since the number of states (hypercube vertices) increases exponentially with the pedigree size.</p><p id="p-7">Alternatively, the theory of junctions has been developed to study the gene flow for a genomic continuum (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Fisher 1949</a>, <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">1954</a>). A junction is defined as a boundary point (recombination breakpoint) on a chromosome where two distinct FGLs meet. <a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Fisher (1949</a>, <a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">1954</a>) and <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Bennett (1953</a>, <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">1954</a>) developed the theory of junctions for simple breeding systems including selfing, brother-sister mating, and parent-offspring mating. It has been extended to populations (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Stam 1980</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Baird <em>et al.</em> 2003</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Chapman and Thompson 2003</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">MacLeod <em>et al.</em> 2005</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Zheng <em>et al.</em> 2014</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Zheng 2015</a>). In particular, <a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">Zheng <em>et al.</em> (2014)</a> and <a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Zheng (2015)</a> developed a Markovian framework for modeling ancestral origins within an individual with the state space being the possible pairs of FGLs, and thus the number of states is much smaller than that of Donnelly’s Markovian framework (<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Donnelly 1983</a>). On the other hand, our Markovian framework (<a id="xref-ref-45-3" class="xref-bibr" href="#ref-45">Zheng <em>et al.</em> 2014</a>; <a id="xref-ref-44-3" class="xref-bibr" href="#ref-44">Zheng 2015</a>) is restricted by two assumptions: the mating scheme from one generation to the next is random mating, and the FGLs are assumed to be exchangeable so that the probability distribution of ancestral origins in offspring is invariant to all possible permutations of the FGLs. However, the exchangeability generally does not hold for a mapping population produced via an arbitrary breeding pedigree.</p><p id="p-8">In this paper, we relax the restriction of our previous Markovian framework by extending it to a fixed pedigree. We first describe a recursive algorithm (denoted by EXCH) under the assumption of FGLs being exchangeable, which is applicable for the construction of Markovian framework in simple breeding schemes. Then we develop a recursive algorithm (denoted by NON-EXCH) for modeling ancestral origins to relax the exchangeability assumption. The two recursive algorithms apply to both autosomes and sex chromosomes if they exist. The results of both recursive algorithms are compared with those from extensive simulations on a classical pedigree. We first apply the two algorithms to compare different population designs (or breeding pedigrees) prior to experiments, for example, in terms of the overall density of junctions (recombination breakpoints), an important factor of QTL mapping resolution. In addition, the non-exchangeability can be illustrated under various breeding designs. Then we apply the two algorithms to incorporate pedigree information for ancestral inference from genotypic data in simulated and real collaborative cross (CC) populations, resulting in conditional probabilities that are necessary for downstream QTL mapping.</p><div class="section other1" id="sec-1"><h2 class="">Recursive Algorithm Exch</h2><div id="sec-2" class="subsection"><h3>Notations and overview</h3><p id="p-9">The symbols are briefly explained in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>, and some of them are illustrated in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>. Pedigree members with unspecified mother and father are called the founders of the pedigree, and the other members are non-founders. The recursive algorithm presupposes that pedigree members are ordered under the constraint that parents always precede children. We denote by subscripts <em>a</em>, <em>b</em>, <em>c</em> the members of a pedigree, and denote by <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-3.gif"/></noscript></span></span> individual <em>a</em> comes after individual <em>b</em> <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-4.gif"/></noscript></span></span> We denote by superscripts <em>m</em> the maternally derived genes or chromosomes, and <em>p</em> for the paternally derived. We denote by superscripts <em>o</em>, <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-5.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-6.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-7.gif"/></noscript></span></span> the unspecified parental origins (<em>m</em> or <em>p</em>) of genes or chromosomes. For the sake of brevity, <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-8.gif"/></noscript></span></span> denotes a horizontal piece-wise equation, so that <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-9.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-10.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-11.gif"/></noscript></span></span> for the autosomes of individual <em>a</em>, XX chromosomes of female <em>a</em>, and XY chromosomes of male <em>a</em>, respectively.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/491038/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/491038/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">List of symbols and their brief descriptions</span><div class="sb-div caption-clear"></div></div></div><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/10/3231/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Illustration of some quantities. An identity state may correspond to many ancestral states. Different FGLs are shown by different colors, and the irrelevant chromosomes are shown as gray. (A) Some two- or three-gene ancestral coefficients and their corresponding identity coefficients. (B) Some expected ancestral junction densities and their corresponding expected identity junction densities. See Table 1 for brief explanations of these quantities." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-863117274" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Illustration of some quantities. An identity state may correspond to many ancestral states. Different FGLs are shown by different colors, and the irrelevant chromosomes are shown as gray. (A) Some two- or three-gene ancestral coefficients and their corresponding identity coefficients. (B) Some expected ancestral junction densities and their corresponding expected identity junction densities. See Table 1 for brief explanations of these quantities.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/10/3231/F1.medium.gif" width="440" height="277"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/8/10/3231/F1.medium.gif" width="440" height="277"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/10/3231/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/10/3231/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/490990" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-10" class="first-child">Illustration of some quantities. An identity state may correspond to many ancestral states. Different FGLs are shown by different colors, and the irrelevant chromosomes are shown as gray. (A) Some two- or three-gene ancestral coefficients and their corresponding identity coefficients. (B) Some expected ancestral junction densities and their corresponding expected identity junction densities. See <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a> for brief explanations of these quantities.</p><div class="sb-div caption-clear"></div></div></div><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/10/3231/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Pictorial representation of recurrence equations. (A) The equation for ancestral coefficient  . (B) The equation for ancestral junction density   The FGLs i, j, and k are shown by different colors, and the irrelevant chromosomes are shown as gray. The symbol &#x2640; denotes the mother of individual a, and see Table 1 for other notations." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-863117274" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Pictorial representation of recurrence equations. (A) The equation for ancestral coefficient &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-81&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-82&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;. (B) The equation for ancestral junction density &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-83&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-84&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; The FGLs &lt;em&gt;i&lt;/em&gt;, &lt;em&gt;j&lt;/em&gt;, and &lt;em&gt;k&lt;/em&gt; are shown by different colors, and the irrelevant chromosomes are shown as gray. The symbol &#x2640; denotes the mother of individual &lt;em&gt;a&lt;/em&gt;, and see Table 1 for other notations.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/10/3231/F2.medium.gif" width="405" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/8/10/3231/F2.medium.gif" width="405" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/10/3231/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/10/3231/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491066" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-11" class="first-child">Pictorial representation of recurrence equations. (A) The equation for ancestral coefficient <span class="inline-formula" id="inline-formula-81"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/F2/embed/mml-math-81.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/F2/embed/mml-math-81.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-82"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/F2/embed/mml-math-82.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/F2/embed/mml-math-82.gif"/></noscript></span></span>. (B) The equation for ancestral junction density <span class="inline-formula" id="inline-formula-83"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/F2/embed/mml-math-83.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/F2/embed/mml-math-83.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-84"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/F2/embed/mml-math-84.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/F2/embed/mml-math-84.gif"/></noscript></span></span> The FGLs <em>i</em>, <em>j</em>, and <em>k</em> are shown by different colors, and the irrelevant chromosomes are shown as gray. The symbol ♀ denotes the mother of individual <em>a</em>, and see <a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a> for other notations.</p><div class="sb-div caption-clear"></div></div></div><p id="p-12">In the derivations of recurrence relations of a quantity, we will focus on a particular ordering <span class="inline-formula" id="inline-formula-85"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-85.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-85.gif"/></noscript></span></span> if the quantity concerns genes in two individuals; we will always trace the maternally derived gene or chromosome within non-founder <em>a</em> back to its two parental genes or chromosomes if the maternally derived gene or chromosome concerns the quantity, and otherwise trace the paternally derived gene or chromosome. Throughout this paper, ♂ and ♀ always denote the father and mother of individual <em>a</em>, rather than any other, respectively.</p><p id="p-13">The identity coefficient <span class="inline-formula" id="inline-formula-86"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-86.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-86.gif"/></noscript></span></span> denotes that two genes at a single locus have identity state <span class="inline-formula" id="inline-formula-87"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-87.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-87.gif"/></noscript></span></span> where the first gene is in individual <em>a</em> and has parental origin <span class="inline-formula" id="inline-formula-88"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-88.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-88.gif"/></noscript></span></span> and the second gene is in individual <em>b</em> and has parental origin <span class="inline-formula" id="inline-formula-89"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-89.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-89.gif"/></noscript></span></span> (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1A</a>). There are only two two-gene identity states: IBD (11) or non-IBD (12), and <span class="inline-formula" id="inline-formula-90"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-90.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-90.gif"/></noscript></span></span> Similarly, we denote by <span class="inline-formula" id="inline-formula-91"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-91.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-91.gif"/></noscript></span></span> the three-gene identity coefficient with the identity state being the non-IBD state <span class="inline-formula" id="inline-formula-92"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-92.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-92.gif"/></noscript></span></span> which is required together with two-gene identity coefficients for deriving the recursive relations for junction densities.</p><p id="p-14">An expected identity junction density is defined as the expected number of the specified type of recombination breakpoints per Morgan along two homologous chromosomes. Here expectation concerns the stochasticity of gene flow from founders to descendants on a fixed pedigree. And identity indicates that only the identity patterns (not specific FGLs) on the two sides of junctions matter. In the recursive algorithm NON-EXCH, we will introduce ancestral junction densities where the specific FGLs do matter. For example, <span class="inline-formula" id="inline-formula-93"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-93.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-93.gif"/></noscript></span></span> denotes that the four genes, two at each side of a breakpoint along two chromosomes, have genetic identity state <span class="inline-formula" id="inline-formula-94"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-94.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-94.gif"/></noscript></span></span> (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1B</a>). Here the first and third genes are on the left and right sides of a breakpoint, respectively, and they are in individual <em>a</em> and have parental origin <span class="inline-formula" id="inline-formula-95"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-95.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-95.gif"/></noscript></span></span> And the second and fourth genes are on the left and right sides, respectively, and they are in individual <em>b</em> and have parental origin <span class="inline-formula" id="inline-formula-96"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-96.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-96.gif"/></noscript></span></span> We adopt the notation of <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Nadot and Vayssiex (1973)</a> for an identity state: the genes are labeled by natural integers starting 1, and the same integer is assigned to the gene that is IBD with a previous gene.</p><p id="p-15">Following the previous framework (<a id="xref-ref-45-4" class="xref-bibr" href="#ref-45">Zheng <em>et al.</em> 2014</a>; <a id="xref-ref-44-4" class="xref-bibr" href="#ref-44">Zheng 2015</a>), we model ancestral origins along two homologous chromosomes within an individual by a continuous time Markov process, which can be described by an initial distribution <em>π</em> of ancestral origins and a rate matrix <em>Q</em>. Under the assumption of exchangeability among FGLs, the initial distribution <em>π</em> of individual <em>a</em> is determined by the identity coefficient <span class="inline-formula" id="inline-formula-97"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-97.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-97.gif"/></noscript></span></span> and the rate matrix <em>Q</em> of individual <em>a</em> is determined by the five expected identity junction densities <span class="inline-formula" id="inline-formula-98"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-98.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-98.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-99"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-99.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-99.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-100"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-100.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-100.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-101"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-101.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-101.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-102"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-102.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-102.gif"/></noscript></span></span> see <a id="xref-ref-44-5" class="xref-bibr" href="#ref-44">Zheng (2015)</a> for an illustrative construction of rate matrix <em>Q</em> from the five junction densities.</p><p id="p-16">In the following, we describe a recursive algorithm for calculating the identity coefficients and the expected junction densities for an individual in a given pedigree. In short, let <span class="inline-formula" id="inline-formula-103"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-103.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-103.gif"/></noscript></span></span> be an indicator function that equals 1 if statement <em>S</em> is true and 0 otherwise, and let <span class="inline-formula" id="inline-formula-104"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-104.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-104.gif"/></noscript></span></span> be an indicator function that equals 0 if statements <span class="inline-formula" id="inline-formula-105"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-105.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-105.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-106"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-106.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-106.gif"/></noscript></span></span> are true and 1 otherwise.</p></div><div id="sec-3" class="subsection"><h3>Identity coefficients</h3><p id="p-17">The recurrence relations for the two- and three-gene identity coefficients are similar to the recursive algorithm of <a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Karigl (1981)</a>. However, the calculation proceeds by tracing back genes instead of individuals, see also <em>e.g.</em>, <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Jacquard (1974)</a>, <a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Nadot and Vayssiex (1973)</a>, <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Denniston (1974)</a>, and <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Garcia-Cortes (2015)</a>. Thus, we can account for the asymmetry between maternally and paternally derived chromosomes, and account for autosomes and sex chromosomes simultaneously. Throughout this paper, we assume that there are no crossovers between X and Y sex chromosomes within a male. In addition, we focus on non-IBD state <span class="inline-formula" id="inline-formula-107"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-107.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-107.gif"/></noscript></span></span> instead of IBD state <span class="inline-formula" id="inline-formula-108"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-108.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-108.gif"/></noscript></span></span> to simplify the recurrence relations.</p><p id="p-18">We derive the recurrence relations according to the Mendelian inheritance rules: (1) a maternally (or paternally) derived autosomal gene descends from the two genes within the mother (or father, respectively) with equal probability <span class="inline-formula" id="inline-formula-109"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-109.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-109.gif"/></noscript></span></span> (2) similar to (1) for a maternally derived X-linked gene, and (3) a paternally derived sex-linked gene within a female (or male) must descend from the X-linked (or Y-linked, respectively) gene of the father. The recurrence relations of the two-gene identity coefficient <span class="inline-formula" id="inline-formula-110"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-110.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-110.gif"/></noscript></span></span> for non-founder <em>a</em> are given by<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-111.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-111.gif"/></noscript></span></span>for <span class="inline-formula" id="inline-formula-111"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-112.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-112.gif"/></noscript></span></span> Here the first equation follows from the rules 1 and 2 for the maternally derived gene within individual <em>a</em>, the second equation follows from the rules 1 and 3 for the paternally derived gene within individual <em>a</em>, and the last equation is obtained by reversing the ordering of two genes. The symmetries such as <span class="inline-formula" id="inline-formula-112"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-113.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-113.gif"/></noscript></span></span> apply to three-gene identity coefficients by permuting the three genes, and thus we consider only a particular ordering <span class="inline-formula" id="inline-formula-113"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-114.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-114.gif"/></noscript></span></span> and we have<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-115.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-115.gif"/></noscript></span></span>for <span class="inline-formula" id="inline-formula-114"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-116.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-116.gif"/></noscript></span></span> where the first (second) equation traces the maternally (or paternally, respectively) derived gene within individual <em>a</em>.</p><p id="p-19">The boundary conditions of these recurrence relations are given according to the assignment of FGLs to the founders in a pedigree. If we assign distinct FGLs to each haploid genome of an outbred founder, all multi-gene non-IBD probabilities are 1 if any pair of genes is not from the same haploid genome, and 0 otherwise. Four-gene or higher order identity coefficients may be derived similarly, but they are not required for the derivations of junction densities.</p></div><div id="sec-4" class="subsection"><h3>Expected identity junction densities</h3><p id="p-20">Let <span class="inline-formula" id="inline-formula-115"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-117.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-117.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-116"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-118.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-118.gif"/></noscript></span></span> be the maternal (paternal) map expansion, the expected density of recombination breakpoints on the maternally (paternally) derived chromosome of individual <em>a</em>. The implicit two-gene identity state for the map expansions is <span class="inline-formula" id="inline-formula-117"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-119.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-119.gif"/></noscript></span></span> It holds that<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-120.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-120.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-118"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-121.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-121.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-119"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-122.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-122.gif"/></noscript></span></span> according to the reversibility of chromosome direction (<a id="xref-ref-44-6" class="xref-bibr" href="#ref-44">Zheng 2015</a>). The expected overall junction density for individual <em>a</em> is given by <span class="inline-formula" id="inline-formula-120"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-123.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-123.gif"/></noscript></span></span> In the following, we derive the recurrence relations for the five expected junction densities <span class="inline-formula" id="inline-formula-121"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-124.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-124.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-122"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-125.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-125.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-123"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-126.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-126.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-124"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-127.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-127.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-125"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-128.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-128.gif"/></noscript></span></span> from which <span class="inline-formula" id="inline-formula-126"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-129.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-129.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-127"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-130.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-130.gif"/></noscript></span></span> can be derived according to the above equations of map expansions. The recurrence relations for the two map expansions of non-founder <em>a</em> are relatively simple, and they are given by (<a id="xref-ref-44-7" class="xref-bibr" href="#ref-44">Zheng 2015</a>)<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-131.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-131.gif"/></noscript></span></span>according to the theory of junctions that a new identity junction is formed whenever a recombination event occurs between two homologous chromosomes that are non-IBD at the location of a crossover (<a id="xref-ref-16-3" class="xref-bibr" href="#ref-16">Fisher 1954</a>).</p><p id="p-21">Since chromosomes are modeled as a genomic continuum, the shared junction type (1122) between two chromosomes can be formed only by duplicating the chromosome segment harboring the recombination breakpoint. For the expected junction density <span class="inline-formula" id="inline-formula-128"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-132.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-132.gif"/></noscript></span></span> of non-founder <em>a</em>, we have<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-133.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-133.gif"/></noscript></span></span>for <span class="inline-formula" id="inline-formula-129"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-134.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-134.gif"/></noscript></span></span> where and the last equation is obtained by reversing the ordering of the two haplotypes. In the first equation, <span class="inline-formula" id="inline-formula-130"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-135.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-135.gif"/></noscript></span></span> refers to the same chromosome in individual <em>a</em>, and thus equals <span class="inline-formula" id="inline-formula-131"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-136.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-136.gif"/></noscript></span></span> by definition. We trace the maternally derived haplotype within non-founder <em>a</em> back to its two parental haplotypes if the maternally derived haplotype concerns the junction density, and otherwise track the paternally derived haplotype.</p><p id="p-22">We derive the recurrence relations for <span class="inline-formula" id="inline-formula-132"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-137.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-137.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-133"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-138.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-138.gif"/></noscript></span></span> jointly. The recombination breakpoint for <span class="inline-formula" id="inline-formula-134"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-139.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-139.gif"/></noscript></span></span> occurs on the first chromosome of a homologous pair, and it is on the second chromosome for <span class="inline-formula" id="inline-formula-135"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-140.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-140.gif"/></noscript></span></span> The junction type <span class="inline-formula" id="inline-formula-136"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-141.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-141.gif"/></noscript></span></span> becomes <span class="inline-formula" id="inline-formula-137"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-142.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-142.gif"/></noscript></span></span> when reversing the ordering of two haplotype of junction type <span class="inline-formula" id="inline-formula-138"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-143.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-143.gif"/></noscript></span></span> We have<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-144.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-144.gif"/></noscript></span></span>for <span class="inline-formula" id="inline-formula-139"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-145.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-145.gif"/></noscript></span></span> There are no contributions from the three-gene non-IBD probabilities in the equations for <span class="inline-formula" id="inline-formula-140"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-146.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-146.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-141"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-147.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-147.gif"/></noscript></span></span> because crossovers between the two parental haplotypes of the tracing haplotype within individual <em>a</em> are invisible.</p><p id="p-23">The boundary conditions are given by the assignment of FGLs to the founders. The within- and between-founder identity junction densities are 0 if a single FGL is assigned to the whole haploid genome of each founder.</p></div></div><div class="section other2" id="sec-5"><h2 class="">Recursive Algorithm Non-Exch</h2><div id="sec-6" class="subsection"><h3>Notations and overview</h3><p id="p-24">Let Ω denote the set of distinct FGLs assigned to the founders of a pedigree. We adopt the same notations in the recursive algorithm with FGL exchangeability except for the following changes. As shown in <a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1</a>, we replace identity states by ancestral states: <span class="inline-formula" id="inline-formula-142"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-148.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-148.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-143"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-149.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-149.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-144"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-150.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-150.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-145"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-151.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-151.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-146"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-152.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-152.gif"/></noscript></span></span> for the two-chromosome expected junction densities, and <span class="inline-formula" id="inline-formula-147"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-153.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-153.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-148"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-154.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-154.gif"/></noscript></span></span> explicitly for the map expansions, where <span class="inline-formula" id="inline-formula-149"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-155.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-155.gif"/></noscript></span></span> are different from each other (denoted by <span class="inline-formula" id="inline-formula-150"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-156.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-156.gif"/></noscript></span></span>).</p><p id="p-25">We represent a two-gene ancestral state by <span class="inline-formula" id="inline-formula-151"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-157.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-157.gif"/></noscript></span></span> and a three-gene ancestral state by <span class="inline-formula" id="inline-formula-152"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-158.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-158.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-153"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-159.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-159.gif"/></noscript></span></span> are not necessary different from each other. In addition we introduce one-gene ancestral coefficient <span class="inline-formula" id="inline-formula-154"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-160.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-160.gif"/></noscript></span></span> the probability that the maternally <span class="inline-formula" id="inline-formula-155"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-161.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-161.gif"/></noscript></span></span> or paternally <span class="inline-formula" id="inline-formula-156"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-162.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-162.gif"/></noscript></span></span> derived gene in individual <em>a</em> carries FGL <span class="inline-formula" id="inline-formula-157"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-163.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-163.gif"/></noscript></span></span> The identity coefficients can be obtained by summing the corresponding ancestral coefficients. For example, <span class="inline-formula" id="inline-formula-158"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-164.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-164.gif"/></noscript></span></span> Similar relationships hold between the expected identity junction densities and the expected ancestral junction densities, for example, <span class="inline-formula" id="inline-formula-159"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-165.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-165.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-160"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-166.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-166.gif"/></noscript></span></span></p><p id="p-26">Similarly, the initial distribution <em>π</em> of the Markov chain can be constructed from the two-gene ancestral coefficients, and the rate matrix <em>Q</em> can be constructed from the expected ancestral junction densities, without assuming the exchangeability of FGLs.</p></div><div id="sec-7" class="subsection"><h3>Ancestral coefficients</h3><p id="p-27"><a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Bauman <em>et al.</em> (2008)</a> and <a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">Zhou <em>et al.</em> (2012)</a> derived the recurrence relations of the one- and two-gene ancestral coefficients for autosomes by tracing two distinct genes simultaneously in their parents. We derive equivalent recurrence relations for both autosomes and sex chromosomes by tracing one gene once in its parent, instead of simultaneously tracing two genes within an individual. In addition, we derive the recurrent relations of the three-gene ancestral coefficients that are required in the recurrence relations of the expected ancestral junction densities.</p><p id="p-28">The relations of the ancestral coefficients are very similar to those of the identity coefficients, and they are based on the same rules of Mendelian inheritance. For example for the one-gene ancestral coefficient <span class="inline-formula" id="inline-formula-161"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-167.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-167.gif"/></noscript></span></span> of non-founder <em>a</em>, we have<span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-168.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-168.gif"/></noscript></span></span>See Appendix A for the recurrence relations of the two-gene ancestral coefficient <span class="inline-formula" id="inline-formula-162"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-169.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-169.gif"/></noscript></span></span> and three-gene ancestral coefficient <span class="inline-formula" id="inline-formula-163"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-170.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-170.gif"/></noscript></span></span></p><p id="p-29">The boundary conditions of the recurrence relations of the ancestral coefficients are given by the FGLs assigned to the founders in a pedigree.</p></div><div id="sec-8" class="subsection"><h3>Expected ancestral junction densities</h3><p id="p-30">The ancestral map expansions can be expressed in terms of the expected ancestral junction densities as follows<span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-171.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-171.gif"/></noscript></span></span>Here <span class="inline-formula" id="inline-formula-164"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-172.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-172.gif"/></noscript></span></span> refers to the junctions on the single chromosome of <em>a</em> with parental origin <em>o</em>, while the terms on the right-hand side refer to the junctions on the two homologous chromosomes of <em>a</em>. We have <span class="inline-formula" id="inline-formula-165"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-173.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-173.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-166"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-174.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-174.gif"/></noscript></span></span> where the equal signs are based on the reversibility of chromosome direction, and the unequal signs refer to the non-exchangeability of FGLs <em>i</em> and <em>j</em>.</p><p id="p-31">The recurrence relations for the ancestral map expansions are given by<span class="disp-formula" id="disp-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-175.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-175.gif"/></noscript></span></span>where the contributions of the two-gene ancestral coefficients account for the asymmetry of FGLs <em>i</em> and <em>j</em>, for example, <span class="inline-formula" id="inline-formula-167"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-176.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-176.gif"/></noscript></span></span> A new ancestral junction is formed whenever a recombination event occurs between two homologous chromosomes that have the unordered FGLs <em>i</em> and <em>j</em> at the location of a crossover.</p><p id="p-32">The recurrence relations for <span class="inline-formula" id="inline-formula-168"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-177.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-177.gif"/></noscript></span></span> are the same as those for <span class="inline-formula" id="inline-formula-169"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-178.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-178.gif"/></noscript></span></span> except that identity states are replaced by ancestral states. We have<span class="disp-formula" id="disp-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-179.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-179.gif"/></noscript></span></span>for <span class="inline-formula" id="inline-formula-170"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-180.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-180.gif"/></noscript></span></span> where the last equation is obtained by reversing the ordering of the two haplotypes. See Appendix B for the recurrence relations of <span class="inline-formula" id="inline-formula-171"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-181.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-181.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-172"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-182.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-182.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-173"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-183.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-183.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-174"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-184.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-184.gif"/></noscript></span></span></p><p id="p-33">As before, the within- and between-founder ancestral junction densities are 0 if a single FGL is assigned to the whole haploid genome of each founder.</p></div><div id="sec-9" class="subsection"><h3>Data availability</h3><p id="p-34">Ancestral inference using the two recursive algorithms is implemented in the previous developed RABBIT software, where the function magicReconstruct is extended to incorporate pedigree information. RABBIT is available at <a href="https://github.com/chaozhi/RABBIT.git">https://github.com/chaozhi/RABBIT.git</a>, and it is offered under the GNU Affero general public license, version 3 (AGPL-3.0). Example scripts for using the recursive algorithms, simulating genotypic data, and ancestral inference are included. The real CC data have been described by <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Durrant <em>et al.</em> (2011)</a>.</p></div></div><div class="section other3" id="sec-10"><h2 class="">Application to Multiparental Populations</h2><div id="sec-11" class="subsection"><h3>Simulation evaluation</h3><p id="p-35">Before applying the two recursive algorithms EXCH and NON-EXCH to multiparental populations, we evaluate their accuracy by forward simulations on the classical pedigree of Native Americans (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>) (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Chapman and Jacquard 1971</a>). The pedigree has been previously used for evaluating recursive algorithms (<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Jacquard 1974</a>; <a id="xref-ref-23-3" class="xref-bibr" href="#ref-23">Karigl 1981</a>; <a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Garcia-Cortes 2015</a>). We simulate two linkage groups: one pair of homologous autosomes and one pair of sex chromosomes. We assign FGLs to the haploid or diploid genomes of the founders. Each descendant gamete is specified as a list of FGL segments determined by chromosomal crossovers. The number of crossovers in a linkage group of a gamete follows a Poisson distribution with mean 1. We assume no genetic interference so that the positions of crossovers are independently and randomly distributed across the chromosomes. We obtain simulated results for the pedigree member “M22” (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>) by averaging over <span class="inline-formula" id="inline-formula-175"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-185.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-185.gif"/></noscript></span></span> simulations.</p><p id="p-36"><a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a> shows the comparisons between the numerical results from the recursive algorithm EXCH and the simulated results. A unique FGL is assigned to each haploid genome of each founder, so that in total twelve distinct FGLs are assigned to the fixed founders. The differences between the numerical and simulated results are less than 0.002, which is very likely due to the stochasticity of gene flow from founders to descendants. The identity coefficient <span class="inline-formula" id="inline-formula-176"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-186.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-186.gif"/></noscript></span></span> for autosomes is in agreement with the previous result (<a id="xref-ref-23-4" class="xref-bibr" href="#ref-23">Karigl 1981</a>).</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/491155/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/491155/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">The identity coefficients and the expected identity junction densities for pedigree member ”M22”</span><div class="sb-div caption-clear"></div></div></div><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/10/3231/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The pedigree of Native Americans. It consists of 6 founders and 16 non-founders. Circles denote females, and rectangles for males." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-863117274" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;The pedigree of Native Americans. It consists of 6 founders and 16 non-founders. Circles denote females, and rectangles for males.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/10/3231/F3.medium.gif" width="371" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/8/10/3231/F3.medium.gif" width="371" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/10/3231/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/10/3231/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491131" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-38" class="first-child">The pedigree of Native Americans. It consists of 6 founders and 16 non-founders. Circles denote females, and rectangles for males.</p><div class="sb-div caption-clear"></div></div></div><p id="p-39"><a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a> evaluates the recursive algorithm NON-EXCH by the forward simulations. A unique FGL is assigned to the whole diploid genome of each founder, so that in total six distinct FGLs are assigned to the fixed founders. Here we use a different assignment of FGLs because <a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a> was otherwise too large. The results show that founder “J” does not contribute to the maternally derived autosome and X chromosome in offspring “M22”, which is relatively straightforward from the pedigree structure in <a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3</a>. The differences between the numerical and simulated results, including those for <span class="inline-formula" id="inline-formula-183"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-193.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-193.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-184"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-194.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-194.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-185"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-195.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-195.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-186"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-196.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-196.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-187"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-197.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-197.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-188"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-198.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-198.gif"/></noscript></span></span> are less than 0.001.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/491164/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/491164/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">The ancestral coefficients and the expected ancestral junction densities for pedigree member ”M22”</span><div class="sb-div caption-clear"></div></div></div><p id="p-41">We confirmed that the numerical results from the algorithm NON-EXCH are reduced to those from the algorithm EXCH, by summing over possible FGLs under the given identity pattern. In addition, we confirmed the consistency between the two algorithms in the multiparental populations shown in Figure 4.</p></div><div id="sec-12" class="subsection"><h3>Breeding design</h3><p id="p-42">We apply the recursive algorithm EXCH to multiparental populations, which have become attractive for QTL mapping in many animal and plant species. Specifically, we study the inbreeding process by calculating the IBD probability <span class="inline-formula" id="inline-formula-191"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-201.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-201.gif"/></noscript></span></span> and measure the QTL mapping resolution by calculating the expected overall junction density <span class="inline-formula" id="inline-formula-192"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-202.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-202.gif"/></noscript></span></span> We focus on multi-way funnel breeding schemes (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). In the funnel scheme, the founders of each line are randomly permuted, and each line is produced by a intercross scheme that combines all founder genomes through several generations of pairwise crosses prior to repeated generations of inbreeding by <em>e.g.</em>, sibling mating. The alternating backcross and the father-daughter backcross have been studied by <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Welsh and McMillan (2012)</a> for accelerating inbreeding by a simulation approach.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/10/3231/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Illustration of different 8-way funnel breeding schemes. The alternating backcross scheme alternates between mother-son and father-daughter matings in generation  and the father-daughter backcross alternates between father-daughter and random sibling matings in generation  Circles denote females, and rectangles for males." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-863117274" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Illustration of different 8-way funnel breeding schemes. The alternating backcross scheme alternates between mother-son and father-daughter matings in generation &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-203&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; and the father-daughter backcross alternates between father-daughter and random sibling matings in generation &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-204&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; Circles denote females, and rectangles for males.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/10/3231/F4.medium.gif" width="440" height="308"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/8/10/3231/F4.medium.gif" width="440" height="308"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/10/3231/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/10/3231/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491151" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-43" class="first-child">Illustration of different 8-way funnel breeding schemes. The alternating backcross scheme alternates between mother-son and father-daughter matings in generation <span class="inline-formula" id="inline-formula-193"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/F4/embed/mml-math-203.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/F4/embed/mml-math-203.gif"/></noscript></span></span> and the father-daughter backcross alternates between father-daughter and random sibling matings in generation <span class="inline-formula" id="inline-formula-194"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/F4/embed/mml-math-204.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/F4/embed/mml-math-204.gif"/></noscript></span></span> Circles denote females, and rectangles for males.</p><div class="sb-div caption-clear"></div></div></div><p id="p-44"><a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5A and B</a> show the IBD probability and the expected overall junction density as a function of generation for the recombinant inbred lines (RIL) by selfing and the RIL+ by selfing. These plant breeding schemes have been adopted in many recently produced crop multiparent advanced generation intercross (MAGIC) populations (<em>e.g.</em>, <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Huang <em>et al.</em> 2012</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Mackay <em>et al.</em> 2014</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Pascual <em>et al.</em> 2015</a>). The RIL+ by selfing has one additional generation of intercrossing instead of selfing in the RIL scheme (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Teuscher and Broman 2007</a>). Thus, the RIL+ scheme has smaller IBD probability <span class="inline-formula" id="inline-formula-195"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-205.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-205.gif"/></noscript></span></span> (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5A</a>), and has about 1 per M larger overall junction density <span class="inline-formula" id="inline-formula-196"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-206.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-206.gif"/></noscript></span></span> in a given later generation (<span class="inline-formula" id="inline-formula-197"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-207.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-207.gif"/></noscript></span></span>)(<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5B</a>).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/10/3231/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The identity coefficient  and the expected overall junction density  for the breeding schemes in Figure 4. (A-B) The results for selfing mating schemes 1-2 in Figure 4. (C-D) The results for the autosomes of mating schemes 3-5 with backcross starting from  in Figure 4. (E-F) The results for the sex chromosomes." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-863117274" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;The identity coefficient &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-208&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; and the expected overall junction density &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-209&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; for the breeding schemes in Figure 4. (A-B) The results for selfing mating schemes 1-2 in Figure 4. (C-D) The results for the autosomes of mating schemes 3-5 with backcross starting from &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-210&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; in Figure 4. (E-F) The results for the sex chromosomes.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/10/3231/F5.medium.gif" width="440" height="429"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/8/10/3231/F5.medium.gif" width="440" height="429"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/10/3231/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/10/3231/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491070" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-45" class="first-child">The identity coefficient <span class="inline-formula" id="inline-formula-198"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/F5/embed/mml-math-208.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/F5/embed/mml-math-208.gif"/></noscript></span></span> and the expected overall junction density <span class="inline-formula" id="inline-formula-199"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/F5/embed/mml-math-209.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/F5/embed/mml-math-209.gif"/></noscript></span></span> for the breeding schemes in <a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>. (A-B) The results for selfing mating schemes 1-2 in <a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4</a>. (C-D) The results for the autosomes of mating schemes 3-5 with backcross starting from <span class="inline-formula" id="inline-formula-200"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/F5/embed/mml-math-210.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/F5/embed/mml-math-210.gif"/></noscript></span></span> in <a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4</a>. (E-F) The results for the sex chromosomes.</p><div class="sb-div caption-clear"></div></div></div><p id="p-46"><a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5C and E</a> show that the IBD probabilities <span class="inline-formula" id="inline-formula-201"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-211.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-211.gif"/></noscript></span></span> for the alternating backcross and the father-daughter backcross are the same as those of the RIL by sibling for autosomes and they are smaller than those of the RIL by sibling for sex chromosomes. <a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Welsh and McMillan (2012)</a> measured the inbreeding process by the number of generations to reach complete fixation of genome as homozygous, and showed that the number of generation increases from the alternating backcross to the father-daughter backcross and to the RIL by sibling. The differences in the number of generation can be partially because the differences of the IBD probabilities for sex chromosomes, and partially because the complete fixation refers to one individual for the alternating backcross and two individuals for the father-daughter backcross.</p><p id="p-47"><a id="xref-fig-5-5" class="xref-fig" href="#F5">Figure 5D and F</a> show that the expected overall junction densities <span class="inline-formula" id="inline-formula-202"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-212.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-212.gif"/></noscript></span></span> for the alternating backcross and the father-daughter backcross are lower than those of the RIL by sibling for autosomes and sex chromosomes, and that the junction densities for the alternating backcross are slightly smaller than those of the father-daughter backcross for autosomes. These results are consistent with those of <a id="xref-ref-42-3" class="xref-bibr" href="#ref-42">Welsh and McMillan (2012)</a>: the number of chromosome segments in the final inbred lines increases from the alternating backcross to the father-daughter backcross and to the RIL by sibling.</p></div><div id="sec-13" class="subsection"><h3>FGL exchangeability</h3><p id="p-48">We apply the recursive algorithm NON-EXCH to study the prior FGL exchangeability for the multi-way funnel breeding schemes (<a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4</a>). We assume that all founder parents are fully inbred, and assign FGLs from A to H to the eight inbred parents in order from left to right. To examine the FGL exchangeability, we calculate the ancestral coefficient <span class="inline-formula" id="inline-formula-203"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-213.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-213.gif"/></noscript></span></span> and the expected ancestral junction density <span class="inline-formula" id="inline-formula-204"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-214.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-214.gif"/></noscript></span></span> the latter being the only ancestral junction type for complete inbred individuals.</p><p id="p-49"><a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a> shows the exchangeability in terms of <span class="inline-formula" id="inline-formula-205"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-215.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-215.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-206"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-216.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-216.gif"/></noscript></span></span> for the females in generation <span class="inline-formula" id="inline-formula-207"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-217.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-217.gif"/></noscript></span></span> 11, and 22 for the RIL by sibling. The left panels show the results for autosomes. The FGL non-exchangeability confirms the pedigree inconsistency introduced by <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Liu <em>et al.</em> (2010)</a>: each of the four mating pairs of founder parents is impossible at a single locus in an individual, that is, <span class="inline-formula" id="inline-formula-208"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-218.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-218.gif"/></noscript></span></span> for <span class="inline-formula" id="inline-formula-209"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-219.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-219.gif"/></noscript></span></span> or <span class="inline-formula" id="inline-formula-210"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-220.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-220.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-211"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-221.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-221.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-212"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-222.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-222.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-213"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-223.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-223.gif"/></noscript></span></span> The FGL non-exchangeability shows a different pattern for <span class="inline-formula" id="inline-formula-214"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-224.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-224.gif"/></noscript></span></span> there are three levels of expected junction densities and the four mating pairs of founder parents have the highest values. For both <span class="inline-formula" id="inline-formula-215"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-225.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-225.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-216"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-226.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-226.gif"/></noscript></span></span> the non-exchangeability diminishes in generation 22 with almost complete inbreeding.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/10/3231/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The FGL non-exchangeability patterns for the ancestral coefficient  and the expected ancestral junction density  in the eight-way RIL by sibling. The FGLs of eight founders are A-H from left to right." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-863117274" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;The FGL non-exchangeability patterns for the ancestral coefficient &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-227&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; and the expected ancestral junction density &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-228&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; in the eight-way RIL by sibling. The FGLs of eight founders are A-H from left to right.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/10/3231/F6.medium.gif" width="440" height="354"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/8/10/3231/F6.medium.gif" width="440" height="354"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/10/3231/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/10/3231/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491004" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-50" class="first-child">The FGL non-exchangeability patterns for the ancestral coefficient <span class="inline-formula" id="inline-formula-217"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/F6/embed/mml-math-227.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/F6/embed/mml-math-227.gif"/></noscript></span></span> and the expected ancestral junction density <span class="inline-formula" id="inline-formula-218"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/F6/embed/mml-math-228.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/F6/embed/mml-math-228.gif"/></noscript></span></span> in the eight-way RIL by sibling. The FGLs of eight founders are A-H from left to right.</p><div class="sb-div caption-clear"></div></div></div><p id="p-51">The right panels of <a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6</a> show the exchangeability patterns for sex chromosomes of the RIL by sibling. The founder parents D, G, and H cannot pass their X chromosomes beyond F1, and thus their FGLs are impossible in generation <span class="inline-formula" id="inline-formula-219"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-229.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-229.gif"/></noscript></span></span> Similar to autosomes, <span class="inline-formula" id="inline-formula-220"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-230.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-230.gif"/></noscript></span></span> for <span class="inline-formula" id="inline-formula-221"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-231.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-231.gif"/></noscript></span></span> or <span class="inline-formula" id="inline-formula-222"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-232.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-232.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-223"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-233.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-233.gif"/></noscript></span></span>. For <span class="inline-formula" id="inline-formula-224"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-234.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-234.gif"/></noscript></span></span> or <span class="inline-formula" id="inline-formula-225"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-235.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-235.gif"/></noscript></span></span> the probability of FGL C is around twice as large as that of A, B, E or F in generation <span class="inline-formula" id="inline-formula-226"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-236.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-236.gif"/></noscript></span></span> because the X chromosome carrying FGL C in generation 1 is inherited with probability 1, whereas each of the X chromosomes carrying FGLs A, B, E, and F in generation 1 is inherited with probability 1/2. Similarly, the values of <span class="inline-formula" id="inline-formula-227"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-237.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-237.gif"/></noscript></span></span> involving FGL C increase with inbreeding generation, they are about twice as large as others in generation 22.</p><p id="p-52">We examine the exchangeability patterns for the other multi-way funnel breeding schemes, and focus on autosomes since the FGL exchangeability does not hold in general for sex chromosomes because of the asymmetry between X and Y chromosomes. The non-exchangeability of <span class="inline-formula" id="inline-formula-228"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-238.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-238.gif"/></noscript></span></span> disappears in a completely inbred individual for all 4- 8- and 16-way breeding schemes, because of random chromosomal segregations over many inbreeding generations. However, the FGL exchangeability of <span class="inline-formula" id="inline-formula-229"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-239.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-239.gif"/></noscript></span></span> often does not hold even after for a completely inbred individual, for example, see <a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7</a> for the breeding schemes after 100 generations of selfing inbreeding.</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/10/3231/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The FGL non-exchangeability patterns of the expected ancestral junction density  The results are for autosomes of the multi-way RIL by 100 generations of selfing or sibling. The FGLs for founder parents are letters starting from A up to P from left to right." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-863117274" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;The FGL non-exchangeability patterns of the expected ancestral junction density &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-240&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; The results are for autosomes of the multi-way RIL by 100 generations of selfing or sibling. The FGLs for founder parents are letters starting from A up to P from left to right.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/10/3231/F7.medium.gif" width="430" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/8/10/3231/F7.medium.gif" width="430" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/10/3231/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/10/3231/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491161" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-53" class="first-child">The FGL non-exchangeability patterns of the expected ancestral junction density <span class="inline-formula" id="inline-formula-230"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/F7/embed/mml-math-240.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/F7/embed/mml-math-240.gif"/></noscript></span></span> The results are for autosomes of the multi-way RIL by 100 generations of selfing or sibling. The FGLs for founder parents are letters starting from A up to P from left to right.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-14" class="subsection"><h3>Ancestral inference</h3><p id="p-54">Either one of the two recursive algorithms can be used as a prior to incorporate pedigree information for ancestral inference in multiparental populations from genotypic data, which will be illustrated as follows by simulated and real CC populations.</p><div id="sec-15" class="subsection"><h4>Simulated CC:</h4><p id="p-55">We simulate a CC population (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Churchill <em>et al.</em> 2004</a>) with 100 independent funnels, and take the female at the last generation. The SNP data for the eight founder mouse strains are from <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Collaborative Cross Consortium (2012)</a>, with marker density <span class="inline-formula" id="inline-formula-231"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-241.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-241.gif"/></noscript></span></span> SNPs per cM. The genotypic data of a sample female are obtained by simulating the FGLs forwardly and then combining them with the founder SNP data. Allelic errors are assumed to occur independently, and we simulate genotypic data of founders and sampled individuals with the same allelic error probability 0.005.</p><p id="p-56">We analyze four simulated data sets: F11-AA, F11-XX, F22-AA, and F22-XX, where the first part denotes the generation, and the second part denotes the 19 pairs of autosomes (AA) or the sex chromosomes (XX). We perform two analyses for each data set by applying each of the two recursive algorithms as the prior for modeling ancestral origins along two homologous chromosomes within an individual. See <a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure 6</a> for the prior distributions of <span class="inline-formula" id="inline-formula-232"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-242.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-242.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-233"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-243.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-243.gif"/></noscript></span></span> as a part of results obtained from the algorithm NON-EXCH. The true allelic error probability is used for the genotypic data of founders and sampled individuals.</p><p id="p-57"><a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a> shows the evaluation of the prior FGL exchangeability on ancestral inference in the simulated CC population. The two analyses of the data set F22-AA are almost the same because there is approximately no prior FGL exchangeability for the autosomes in generation <span class="inline-formula" id="inline-formula-234"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-244.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-244.gif"/></noscript></span></span> and the improvement of NON-EXCH over EXCH for F11-AA is <span class="inline-formula" id="inline-formula-235"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-245.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-245.gif"/></noscript></span></span> because of the FGL non-exchangeability such as that of <span class="inline-formula" id="inline-formula-236"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-246.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-246.gif"/></noscript></span></span> in <a id="xref-fig-6-4" class="xref-fig" href="#F6">Figure 6</a>. The improvements for sex chromosomes are larger than those for autosomes because of the more pronounced FGL non-exchangeability (<a id="xref-fig-6-5" class="xref-fig" href="#F6">Figure 6</a>). The improvement for the data set F22-XX is <span class="inline-formula" id="inline-formula-237"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-247.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-247.gif"/></noscript></span></span>, and it increases to <span class="inline-formula" id="inline-formula-238"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-248.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-248.gif"/></noscript></span></span> for F11-XX.</p><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/491159/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/491159/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">Evaluation of the prior FGL exchangeability on ancestral inference in the simulated CC population consisting of 100 funnels</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-16" class="subsection"><h4>Real CC:</h4><p id="p-61">The real CC population consists of 120 lines that are sampled at generation <em>t</em> in the range of <span class="inline-formula" id="inline-formula-239"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-249.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-249.gif"/></noscript></span></span> (<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Durrant <em>et al.</em> 2011</a>). For each line, we estimate the sampling generation by using the algorithm EXCH and taking the generation with the maximum likelihood, where the funnel code is not required. Then we estimate the funnel code by using the algorithm NON-EXCH and an iterative maximum likelihood, where a funnel code is proposed by slightly disturbing the current funnel code and it is accepted if the likelihood is increased. We compare the algorithms EXCH and NON-EXCH with GAIN (<a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Liu <em>et al.</em> 2010</a>), the latter being specifically developed for the CC.</p><p id="p-62">To study the effect of marker densities on ancestral inference, we analyze only the first pair of homologous autosomes and thin the full dataset by taking every second SNP markers, and repeat to obtain nested sub datasets. The data fractions or the relative marker densities are given by <span class="inline-formula" id="inline-formula-240"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-250.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-250.gif"/></noscript></span></span> The absolute maximum marker density is 145 SNPs per cM, <span class="inline-formula" id="inline-formula-241"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-251.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-251.gif"/></noscript></span></span> times higher than that of simulated CC. From the full dataset <span class="inline-formula" id="inline-formula-242"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-252.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-252.gif"/></noscript></span></span> we calculate the marginal posterior probabilities by NON-EXCH, EXCH, and GAIN, and set the pseudo-true values to the most probable ancestral origins if they are the same among the three methods. Overall the pseudo-true ancestral origins for <span class="inline-formula" id="inline-formula-243"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-253.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-253.gif"/></noscript></span></span> observed genotypes are obtained.</p><p id="p-63"><a id="xref-fig-8-1" class="xref-fig" href="#F8">Figure 8</a> shows that the results on ancestral inferences are only slightly different among the three methods. The results of GAIN and NON-EXCH contain no pedigree inconsistencies, whereas NON-EXCH assigns ancestral origins to the impossible mating pairs of founder parents with probability around 0.005. The wrongly assigned probability for GAIN is larger than those of NON-EXCH and EXCH, and the difference increases with the decreasing marker density. The ranking of the wrong called probability from lowest to highest is NON-EXCH, EXCH, and GAIN, and the improvement of NON-EXCH over EXCH is around <span class="inline-formula" id="inline-formula-244"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-254.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-254.gif"/></noscript></span></span> and the improvement of NON-EXCH over GAIN is around <span class="inline-formula" id="inline-formula-245"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-255.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-255.gif"/></noscript></span></span> when relative density <span class="inline-formula" id="inline-formula-246"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-256.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-256.gif"/></noscript></span></span> The similar performances between algorithms NON-EXCH and EXCH indicate that the prior FGL exchangeability is a reasonable approximation for autosomes in the CC.</p></div></div></div><div class="section discussion" id="sec-17"><h2 class="">Discussion</h2><p id="p-64">We have developed two novel recursive algorithms for modeling genomic ancestral origins along two homologous chromosomes in a pedigree with arbitrary but known structure. The algorithms apply to both autosomes and sex chromosomes when they exist, and allow selfing in the pedigree. The extensive simulations on a real example pedigree show that the numerical results from the two recursive algorithms are consistent with the simulated results, apart from the stochasticity of gene flow. The Markov property is assumed for the ancestral origin process along two chromosomes, and thus genetic interference is assumed to be absent. The assumption does not affect the expected junction densities calculated from the two recursive algorithms, although it does affect the distribution of inter-junction distances including the variance of junction densities.</p><p id="p-65">One important application of the recursive algorithms is to design a breeding scheme for accelerating the inbreeding process to obtain immortal lines and for maximizing the resolution of mapping QTL. <a id="xref-ref-42-4" class="xref-bibr" href="#ref-42">Welsh and McMillan (2012)</a> studied inbreeding processes among different types of multiparental crosses by simulations, and it takes approximately 5.5 hr to complete 100,000 simulations of eight-way RILs. In contrast, it takes less than one second for our recursive algorithms. <a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Rockman and Kruglyak (2008)</a> compared different intercross breeding designs in multiparental RILs by simulations, aiming at increasing the density of junctions (recombination breakpoints) and thus fine-mapping resolution. Our previous recursive algorithms (<a id="xref-ref-45-5" class="xref-bibr" href="#ref-45">Zheng <em>et al.</em> 2014</a>; <a id="xref-ref-44-8" class="xref-bibr" href="#ref-44">Zheng 2015</a>) can be used for calculating the junction density in random mating schemes, and the two new algorithms extend the calculation for any breeding schemes with fixed pedigrees. The two recursive algorithms can also be used in random mating schemes by applying them to many pedigrees that are simulated according to specified mating schemes and averaging the results, which would still require less number of replicates and less computational time than simulation studies.</p><p id="p-66">The second important application of the two recursive algorithms is to provide an appropriate way of incorporating pedigree information for analyzing genotypic data in bi- or multiparental populations. Specifically, the two recursive algorithm can be used to calculate the process parameter values of hidden Markov models for genotypic data, that is, the prior probability distribution of ancestral origins (FGLs) at an initial site and the prior transition probability matrix describing how ancestral origins change along two homologous chromosomes within an individual. See <a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1</a> of <a id="xref-ref-44-9" class="xref-bibr" href="#ref-44">Zheng (2015)</a> for an example. The application to the ancestral inference in the CC shows that the new algorithms implemented in RABBIT performs only slightly better than GAIN (<a id="xref-fig-8-2" class="xref-fig" href="#F8">Figure 8</a>). However, GAIN is specifically designed for the CC, and our previous algorithm applies only to breeding schemes with stage-wise random mating. The new recursive algorithms have pronounced advantages of generality and computational efficiency, and they apply to arbitrary breeding pedigrees that are far away from random mating. For example, one of the multiparental barley populations consists of backcrossing, half-diallel crossing, and selfing, where one inbred founder is much stronger represented in the offspring lines (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Liller <em>et al.</em> 2017</a>).</p><div id="F8" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/10/3231/F8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Evaluation of the recursive algorithms that are applied in the ancestral inference for the first pair of autosomes of the 120 real CC lines. Panels (A-C) show the results for the wrongly assigned probability, the wrongly called probability, and the pedigree inconsistency, respectively. The pedigree inconsistency is measured by the sum of the posterior probabilities over the four mating pairs of founder parents." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-863117274" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Evaluation of the recursive algorithms that are applied in the ancestral inference for the first pair of autosomes of the 120 real CC lines. Panels (A-C) show the results for the wrongly assigned probability, the wrongly called probability, and the pedigree inconsistency, respectively. The pedigree inconsistency is measured by the sum of the posterior probabilities over the four mating pairs of founder parents.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 8" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/10/3231/F8.medium.gif" width="215" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 8" src="https://www.g3journal.org/content/ggg/8/10/3231/F8.medium.gif" width="215" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/10/3231/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 8" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/10/3231/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491100" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 8</span> <p id="p-67" class="first-child">Evaluation of the recursive algorithms that are applied in the ancestral inference for the first pair of autosomes of the 120 real CC lines. Panels (A-C) show the results for the wrongly assigned probability, the wrongly called probability, and the pedigree inconsistency, respectively. The pedigree inconsistency is measured by the sum of the posterior probabilities over the four mating pairs of founder parents.</p><div class="sb-div caption-clear"></div></div></div><p id="p-68">Furthermore, we have applied the two recursive algorithms to incorporate pedigree information for genotype imputation in multiparental populations (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Zheng <em>et al.</em> 2018</a>), and the application for genetic linkage map construction in multiparental populations is under development.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-73">The authors thank two anonymous reviewers for their valuable comments. This research was supported by the Stichting Technische Wetenschappen (STW) - Technology Foundation, which is part of the Nederlandse Organisatie voor Wetenschappelijk Onderzoek - Netherlands Organization for Scientific Research, and which is partly funded by the Ministry of Economic Affairs. The specific grant number was STW-Rijk Zwaan project 12425. CZ designed the study, created the model, developed the software and algorithm, and wrote the first draft of the manuscript. MPB and FAE provided critical feedback, helped shape the manuscript, and acquitted financial support. All authors read and approved the final manuscript.</p></div><div class="section app" id="app-1"><h2 class="">APPENDIX A: RECURRENCE RELATIONS OF ANCESTRAL COEFFICIENTS</h2><p id="p-69">We derive the recurrence relations of one-, two- and three-gene ancestral coefficients. For the one-gene ancestral coefficient <span class="inline-formula" id="inline-formula-247"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-257.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-257.gif"/></noscript></span></span> of non-founder <em>a</em>, we have<span class="disp-formula" id="disp-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-258.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-258.gif"/></noscript></span></span>And for two-gene ancestral coefficient <span class="inline-formula" id="inline-formula-248"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-259.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-259.gif"/></noscript></span></span> we have<span class="disp-formula" id="disp-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-260.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-260.gif"/></noscript></span></span>for <span class="inline-formula" id="inline-formula-249"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-261.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-261.gif"/></noscript></span></span> where the last equation is obtained by reversing the ordering of the two genes. See <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a> for a pictorial representation of the equation for <span class="inline-formula" id="inline-formula-250"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-262.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-262.gif"/></noscript></span></span></p><p id="p-70">The symmetries such as <span class="inline-formula" id="inline-formula-251"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-263.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-263.gif"/></noscript></span></span> apply to three-gene ancestral coefficients by permuting the three genes, and thus we consider only a particular ordering <span class="inline-formula" id="inline-formula-252"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-264.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-264.gif"/></noscript></span></span>, and we have<span class="disp-formula" id="disp-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-265.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-265.gif"/></noscript></span></span>for <span class="inline-formula" id="inline-formula-253"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-266.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-266.gif"/></noscript></span></span></p></div><div class="section app" id="app-2"><h2 class="">APPENDIX B: RECURRENCE RELATIONS OF EXPECTED ANCESTRAL JUNCTION DENSITIES</h2><p id="p-71">The joint recurrence relations of <span class="inline-formula" id="inline-formula-254"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-267.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-267.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-255"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-268.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-268.gif"/></noscript></span></span> are the same as those of <span class="inline-formula" id="inline-formula-256"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-269.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-269.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-257"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-270.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-270.gif"/></noscript></span></span> except that identity states are replaced by ancestral states and that the contributions of three-gene ancestral coefficients account for the asymmetry of FGLs. We have<span class="disp-formula" id="disp-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-271.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-271.gif"/></noscript></span></span>for <span class="inline-formula" id="inline-formula-258"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-272.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-272.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-259"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-273.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-273.gif"/></noscript></span></span> in the 3rd equation and <span class="inline-formula" id="inline-formula-260"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-274.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-274.gif"/></noscript></span></span> in the 6th equation can be transformed into the form of <span class="inline-formula" id="inline-formula-261"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-275.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-275.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-262"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-276.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-276.gif"/></noscript></span></span> respectively, by swapping <em>i</em> and <em>j</em>. There are no contributions of the three-gene ancestral coefficients in the equations for <span class="inline-formula" id="inline-formula-263"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-277.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-277.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-264"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-278.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-278.gif"/></noscript></span></span> because crossovers between the two parental haplotypes of the tracing haplotype within individual <em>a</em> are invisible. See <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2B</a> for a pictorial representation of the equation for <span class="inline-formula" id="inline-formula-265"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-279.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-279.gif"/></noscript></span></span></p><p id="p-72">The joint recurrence relations of <span class="inline-formula" id="inline-formula-266"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-280.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-280.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-267"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-281.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-281.gif"/></noscript></span></span> are the same to those of <span class="inline-formula" id="inline-formula-268"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-282.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-282.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-269"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-283.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-283.gif"/></noscript></span></span> except that the contributions of the three-gene ancestral coefficients have slightly different forms. We have<span class="disp-formula" id="disp-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-284.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-284.gif"/></noscript></span></span>for <span class="inline-formula" id="inline-formula-270"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-285.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-285.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-271"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-286.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-286.gif"/></noscript></span></span> in the 3rd equation and <span class="inline-formula" id="inline-formula-272"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-287.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-287.gif"/></noscript></span></span> in the 6th equation can be transformed into the form of <span class="inline-formula" id="inline-formula-273"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-288.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-288.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-274"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-289.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3231/embed/mml-math-289.gif"/></noscript></span></span>, respectively, by swapping <em>i</em> and <em>j</em>.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-6"><p id="p-74"><em>Communicating editor: D. J. de Koning</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2018-04-19"><span class="received-label">Received </span>April 19, 2018.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2018-07-31"><span class="accepted-label">Accepted </span>July 31, 2018.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2018 Zheng <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3231.1" data-doi="10.1016/S0040-5809(03)00098-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Baird</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barton</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Etheridge</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">The distribution of surviving blocks of an ancestral genome.</span> <abbr class="cit-jnl-abbrev">Theor. Popul. Biol.</abbr> <span class="cit-vol">64</span>: <span class="cit-fpage">451</span>–<span class="cit-lpage">471</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/S0040-5809(03)00098-4</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheoretical%2Bpopulation%2Bbiology%26rft.stitle%253DTheor%2BPopul%2BBiol%26rft.aulast%253DBaird%26rft.auinit1%253DS.%2BJ.%26rft.volume%253D64%26rft.issue%253D4%26rft.spage%253D451%26rft.epage%253D471%26rft.atitle%253DThe%2Bdistribution%2Bof%2Bsurviving%2Bblocks%2Bof%2Ban%2Bancestral%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0040-5809%252803%252900098-4%26rft_id%253Dinfo%253Apmid%252F14630482%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0040-5809(03)00098-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14630482&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3231.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3231.2" data-doi="10.1534/genetics.108.091058"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bauman</span>  <span class="cit-name-given-names">L. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sinsheimer</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sobel</span>  <span class="cit-name-given-names">E. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lange</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Mixed effects models for quantitative trait loci mapping with inbred strains.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">180</span>: <span class="cit-fpage">1743</span>–<span class="cit-lpage">1761</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.108.091058</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.108.091058%26rft_id%253Dinfo%253Apmid%252F18791243%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE4MC8zLzE3NDMiO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzEwLzMyMzEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3231.3" data-doi="10.1007/BF01690623"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bennett</span>  <span class="cit-name-given-names">J. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1953</span> <span class="cit-article-title">Junctions in inbreeding.</span> <abbr class="cit-jnl-abbrev">Genetica</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">392</span>–<span class="cit-lpage">406</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/BF01690623</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetica%26rft.stitle%253DGenetica%26rft.aulast%253DBennett%26rft.auinit1%253DJ.%2BH.%26rft.volume%253D26%26rft.issue%253D5-6%26rft.spage%253D392%26rft.epage%253D406%26rft.atitle%253DJunctions%2Bin%2Binbreeding.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF01690623%26rft_id%253Dinfo%253Apmid%252F13142314%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF01690623&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=13142314&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3231.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3231.4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bennett</span>  <span class="cit-name-given-names">J. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1954</span> <span class="cit-article-title">The distribution of heterogeneity upon inbreeding.</span> <abbr class="cit-jnl-abbrev">J. R. Stat. Soc. Series B Stat. Methodol.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">88</span>–<span class="cit-lpage">99</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BR.%2BStat.%2BSoc.%2BSeries%2BB%2BStat.%2BMethodol.%26rft.volume%253D16%26rft.spage%253D88%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3231.5" data-doi="10.1006/tpbi.1996.0023"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bickeböller</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">E. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Distribution of genome shared ibd by half-sibs: approximation by the poisson clumping heuristic.</span> <abbr class="cit-jnl-abbrev">Theor. Popul. Biol.</abbr> <span class="cit-vol">50</span>: <span class="cit-fpage">66</span>–<span class="cit-lpage">90</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1006/tpbi.1996.0023</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheoretical%2Bpopulation%2Bbiology%26rft.stitle%253DTheor%2BPopul%2BBiol%26rft.aulast%253DBickeb%25C3%25B6ller%26rft.auinit1%253DH.%26rft.volume%253D50%26rft.issue%253D1%26rft.spage%253D66%26rft.epage%253D90%26rft.atitle%253DDistribution%2Bof%2Bgenome%2Bshared%2BIBD%2Bby%2Bhalf-sibs%253A%2Bapproximation%2Bby%2Bthe%2BPoisson%2Bclumping%2Bheuristic.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Ftpbi.1996.0023%26rft_id%253Dinfo%253Apmid%252F8776838%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/tpbi.1996.0023&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8776838&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3231.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1996VD25300004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3231.6" data-doi="10.1534/genetics.111.132647"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Broman</span>  <span class="cit-name-given-names">K. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genotype probabilities at intermediate generations in the construction of recombinant inbred lines.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">190</span>: <span class="cit-fpage">403</span>–<span class="cit-lpage">412</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.111.132647</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.132647%26rft_id%253Dinfo%253Apmid%252F22345609%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkwLzIvNDAzIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMC8zMjMxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-book" id="cit-8.10.3231.7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chapman</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jacquard</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1971</span> <span class="cit-article-title">Un isolat d’Amerique centrale: les indiens Jicaques du Honduras</span>, <span class="cit-source">Génétique et Populations</span>, <span class="cit-publ-name">Presses Universitaires de France</span>, <span class="cit-publ-loc">Paris</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3231.8" data-doi="10.1016/S0040-5809(03)00071-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chapman</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">A model for the length of tracts of identity by descent in finite random mating populations.</span> <abbr class="cit-jnl-abbrev">Theor. Popul. Biol.</abbr> <span class="cit-vol">64</span>: <span class="cit-fpage">141</span>–<span class="cit-lpage">150</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/S0040-5809(03)00071-6</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheoretical%2Bpopulation%2Bbiology%26rft.stitle%253DTheor%2BPopul%2BBiol%26rft.aulast%253DChapman%26rft.auinit1%253DN.%2BH.%26rft.volume%253D64%26rft.issue%253D2%26rft.spage%253D141%26rft.epage%253D150%26rft.atitle%253DA%2Bmodel%2Bfor%2Bthe%2Blength%2Bof%2Btracts%2Bof%2Bidentity%2Bby%2Bdescent%2Bin%2Bfinite%2Brandom%2Bmating%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0040-5809%252803%252900071-6%26rft_id%253Dinfo%253Apmid%252F12948676%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0040-5809(03)00071-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12948676&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3231.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3231.9" data-doi="10.1038/ng1104-1133"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Churchill</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Airey</span>  <span class="cit-name-given-names">D. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Allayee</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Angel</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Attie</span>  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">The collaborative cross, a community resource for the genetic analysis of complex traits.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">1133</span>–<span class="cit-lpage">1137</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng1104-1133</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DChurchill%26rft.auinit1%253DG.%2BA.%26rft.volume%253D36%26rft.issue%253D11%26rft.spage%253D1133%26rft.epage%253D1137%26rft.atitle%253DThe%2BCollaborative%2BCross%252C%2Ba%2Bcommunity%2Bresource%2Bfor%2Bthe%2Bgenetic%2Banalysis%2Bof%2Bcomplex%2Btraits.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1104-1133%26rft_id%253Dinfo%253Apmid%252F15514660%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1104-1133&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15514660&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3231.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000224832800005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3231.10" data-doi="10.1016/0040-5809(73)90013-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cockerham</span>  <span class="cit-name-given-names">C. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weir</span>  <span class="cit-name-given-names">B. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1973</span> <span class="cit-article-title">Descent measures for 2 loci with some applications.</span> <abbr class="cit-jnl-abbrev">Theor. Popul. Biol.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">300</span>–<span class="cit-lpage">330</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/0040-5809(73)90013-0</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheoretical%2Bpopulation%2Bbiology%26rft.stitle%253DTheor%2BPopul%2BBiol%26rft.aulast%253DCockerham%26rft.auinit1%253DC.%2BC.%26rft.volume%253D4%26rft.issue%253D3%26rft.spage%253D300%26rft.epage%253D330%26rft.atitle%253DDescent%2Bmeasures%2Bfor%2Btwo%2Bloci%2Bwith%2Bsome%2Bapplications.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0040-5809%252873%252990013-0%26rft_id%253Dinfo%253Apmid%252F4747657%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0040-5809(73)90013-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=4747657&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3231.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3231.11" data-doi="10.1534/genetics.111.132639"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">Collaborative Cross Consortium</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The genome architecture of the collaborative cross mouse genetic reference population.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">190</span>: <span class="cit-fpage">389</span>–<span class="cit-lpage">401</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.111.132639</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.132639%26rft_id%253Dinfo%253Apmid%252F22345608%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkwLzIvMzg5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMC8zMjMxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3231.12" data-doi="10.1016/0040-5809(74)90031-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Denniston</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">1974</span> <span class="cit-article-title">Extension of probability approach to genetic relationship - one locus.</span> <abbr class="cit-jnl-abbrev">Theor. Popul. Biol.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">58</span>–<span class="cit-lpage">75</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/0040-5809(74)90031-8</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheoretical%2Bpopulation%2Bbiology%26rft.stitle%253DTheor%2BPopul%2BBiol%26rft.aulast%253DDenniston%26rft.auinit1%253DC.%26rft.volume%253D6%26rft.issue%253D1%26rft.spage%253D58%26rft.epage%253D75%26rft.atitle%253DAn%2Bextension%2Bof%2Bthe%2Bprobability%2Bapproach%2Bto%2Bgenetic%2Brelationships%253A%2Bone%2Blocus.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0040-5809%252874%252990031-8%26rft_id%253Dinfo%253Apmid%252F4425405%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0040-5809(74)90031-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=4425405&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3231.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3231.13" data-doi="10.1016/0040-5809(83)90004-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Donnelly</span>  <span class="cit-name-given-names">K. P.</span></span></li></ol><cite>, <span class="cit-pub-date">1983</span> <span class="cit-article-title">The probability that related individuals share some section of genome identical by descent.</span> <abbr class="cit-jnl-abbrev">Theor. Popul. Biol.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">34</span>–<span class="cit-lpage">63</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/0040-5809(83)90004-7</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheoretical%2Bpopulation%2Bbiology%26rft.stitle%253DTheor%2BPopul%2BBiol%26rft.aulast%253DDonnelly%26rft.auinit1%253DK.%2BP.%26rft.volume%253D23%26rft.issue%253D1%26rft.spage%253D34%26rft.epage%253D63%26rft.atitle%253DThe%2Bprobability%2Bthat%2Brelated%2Bindividuals%2Bshare%2Bsome%2Bsection%2Bof%2Bgenome%2Bidentical%2Bby%2Bdescent.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0040-5809%252883%252990004-7%26rft_id%253Dinfo%253Apmid%252F6857549%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0040-5809(83)90004-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=6857549&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3231.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1983QK30500003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3231.14" data-doi="10.1101/gr.118786.110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Durrant</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tayem</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yalcin</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cleak</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goodstadt</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Collaborative cross mice and their power to map host susceptibility to aspergillus fumigatus infection.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">1239</span>–<span class="cit-lpage">1248</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.118786.110</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.118786.110%26rft_id%253Dinfo%253Apmid%252F21493779%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjIxLzgvMTIzOSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzgvMTAvMzIzMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-book" id="cit-8.10.3231.15"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fisher</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1949</span> <span class="cit-source">The Theory of Inbreeding</span>, <span class="cit-publ-name">Oliver and Boyd</span>, <span class="cit-publ-loc">London</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3231.16" data-doi="10.1038/hdy.1954.17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fisher</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1954</span> <span class="cit-article-title">A fuller theory of junctions in inbreeding.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">187</span>–<span class="cit-lpage">197</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/hdy.1954.17</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D8%26rft.spage%253D187%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.1954.17%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.1954.17&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1954YC17800002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3231.17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Garcia-Cortes</span>  <span class="cit-name-given-names">L. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">A novel recursive algorithm for the calculation of the detailed identity coefficients.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">33</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D47%26rft.spage%253D33%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3231.18"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Haldane</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Waddington</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">1931</span> <span class="cit-article-title">Inbreeding and linkage.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">357</span>–<span class="cit-lpage">374</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DHaldane%26rft.auinit1%253DJ.%2BB.%2BS.%26rft.volume%253D16%26rft.issue%253D4%26rft.spage%253D357%26rft.epage%253D374%26rft.atitle%253DINBREEDING%2BAND%2BLINKAGE%26rft_id%253Dinfo%253Apmid%252F17246626%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo4OiJnZW5ldGljcyI7czo1OiJyZXNpZCI7czo4OiIxNi80LzM1NyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzgvMTAvMzIzMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3231.19" data-doi="10.1016/j.tpb.2006.05.006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hill</span>  <span class="cit-name-given-names">W. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weir</span>  <span class="cit-name-given-names">B. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Prediction of multi-locus inbreeding coefficients and relation to linkage disequilibrium in random mating populations.</span> <abbr class="cit-jnl-abbrev">Theor. Popul. Biol.</abbr> <span class="cit-vol">72</span>: <span class="cit-fpage">179</span>–<span class="cit-lpage">185</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.tpb.2006.05.006</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheoretical%2Bpopulation%2Bbiology%26rft.stitle%253DTheor%2BPopul%2BBiol%26rft.aulast%253DHill%26rft.auinit1%253DW.%2BG.%26rft.volume%253D72%26rft.issue%253D2%26rft.spage%253D179%26rft.epage%253D185%26rft.atitle%253DPrediction%2Bof%2Bmulti-locus%2Binbreeding%2Bcoefficients%2Band%2Brelation%2Bto%2Blinkage%2Bdisequilibrium%2Bin%2Brandom%2Bmating%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tpb.2006.05.006%26rft_id%253Dinfo%253Apmid%252F17575994%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tpb.2006.05.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17575994&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3231.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3231.20" data-doi="10.1111/j.1467-7652.2012.00702.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">B. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">George</span>  <span class="cit-name-given-names">A. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Forrest</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kilian</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayden</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A multiparent advanced generation inter-cross population for genetic analysis in wheat.</span> <abbr class="cit-jnl-abbrev">Plant Biotechnol. J.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">826</span>–<span class="cit-lpage">839</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1467-7652.2012.00702.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPlant%2BBiotechnol%2BJ%26rft.aulast%253DHuang%26rft.auinit1%253DB.%2BE.%26rft.volume%253D10%26rft.issue%253D7%26rft.spage%253D826%26rft.epage%253D839%26rft.atitle%253DA%2Bmultiparent%2Badvanced%2Bgeneration%2Binter-cross%2Bpopulation%2Bfor%2Bgenetic%2Banalysis%2Bin%2Bwheat.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1467-7652.2012.00702.x%26rft_id%253Dinfo%253Apmid%252F22594629%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1467-7652.2012.00702.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22594629&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3231.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000307051700007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-book" id="cit-8.10.3231.21" data-doi="10.1007/978-3-642-88415-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jacquard</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1974</span> <span class="cit-source">The Genetic Structure of Populations</span>, <span class="cit-publ-name">Springer-Verlag</span>, <span class="cit-publ-loc">New York</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/978-3-642-88415-3</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BGenetic%2BStructure%2Bof%2BPopulations%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-3-642-88415-3%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-3-642-88415-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3231.22" data-doi="10.1534/genetics.111.127118"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Johannes</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Colome-Tatche</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Quantitative epigenetics through epigenomic perturbation of isogenic lines.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">188</span>: <span class="cit-fpage">215</span>–<span class="cit-lpage">227</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.111.127118</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.127118%26rft_id%253Dinfo%253Apmid%252F21385727%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg4LzEvMjE1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMC8zMjMxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3231.23" data-doi="10.1111/j.1469-1809.1981.tb00341.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Karigl</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">1981</span> <span class="cit-article-title">A recursive algorithm for the calculation of identity coefficients.</span> <abbr class="cit-jnl-abbrev">Ann. Hum. Genet.</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">299</span>–<span class="cit-lpage">305</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1469-1809.1981.tb00341.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnals%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAnn%2BHum%2BGenet%26rft.aulast%253DKarigl%26rft.auinit1%253DG.%26rft.volume%253D45%26rft.issue%253DPt%2B3%26rft.spage%253D299%26rft.epage%253D305%26rft.atitle%253DA%2Brecursive%2Balgorithm%2Bfor%2Bthe%2Bcalculation%2Bof%2Bidentity%2Bcoefficients.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1469-1809.1981.tb00341.x%26rft_id%253Dinfo%253Apmid%252F7305283%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1469-1809.1981.tb00341.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7305283&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3231.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1981LY98800009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3231.24" data-doi="10.1371/journal.pgen.1000551"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kover</span>  <span class="cit-name-given-names">P. X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valdar</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trakalo</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scarcelli</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ehrenreich</span>  <span class="cit-name-given-names">I. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A multiparent advanced generation inter-cross to fine-map quantitative traits in <em>Arabidopsis thaliana</em>.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e1000551</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1000551</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DKover%26rft.auinit1%253DP.%2BX.%26rft.volume%253D5%26rft.issue%253D7%26rft.spage%253De1000551%26rft.epage%253De1000551%26rft.atitle%253DA%2BMultiparent%2BAdvanced%2BGeneration%2BInter-Cross%2Bto%2Bfine-map%2Bquantitative%2Btraits%2Bin%2BArabidopsis%2Bthaliana.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000551%26rft_id%253Dinfo%253Apmid%252F19593375%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000551&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19593375&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3231.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3231.25" data-doi="10.1186/s12915-015-0187-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">C. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y. X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bradbury</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shi</span>  <span class="cit-name-given-names">Y. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Construction of high-quality recombination maps with low-coverage genomic sequencing for joint linkage analysis in maize.</span> <abbr class="cit-jnl-abbrev">BMC Biol.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">78</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12915-015-0187-4</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BBiol.%26rft.volume%253D13%26rft.spage%253D78%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12915-015-0187-4%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12915-015-0187-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3231.26" data-doi="10.1007/s00122-016-2807-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liller</span>  <span class="cit-name-given-names">C. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Walla</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boer</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hedley</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Macaulay</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Fine mapping of a major QTL for awn length in barley using a multiparent mapping population.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">130</span>: <span class="cit-fpage">269</span>–<span class="cit-lpage">281</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-016-2807-y</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D130%26rft.spage%253D269%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-016-2807-y%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-016-2807-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3231.27" data-doi="10.1093/bioinformatics/btq187"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">E. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McMillan</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pardo-Manuel de Villena</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Efficient genome ancestry inference in complex pedigrees with inbreeding.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">i199</span>–<span class="cit-lpage">i207</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btq187</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq187%26rft_id%253Dinfo%253Apmid%252F20529906%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btq187&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20529906&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3231.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3231.28" data-doi="10.1534/g3.114.012963"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mackay</span>  <span class="cit-name-given-names">I. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bansept-Basler</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barber</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bentley</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cockram</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">An eight-parent multiparent advanced generation inter-cross population for winter-sown wheat: creation, properties, and validation.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">1603</span>–<span class="cit-lpage">1610</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.114.012963</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft.volume%253D4%26rft.spage%253D1603%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.114.012963%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/g3.114.012963&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3231.29" data-doi="10.1017/S0016672305007329"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">MacLeod</span>  <span class="cit-name-given-names">A. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haley</span>  <span class="cit-name-given-names">C. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Woolliams</span>  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Marker densities and the mapping of ancestral junctions.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">85</span>: <span class="cit-fpage">69</span>–<span class="cit-lpage">79</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1017/S0016672305007329</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DMacLeod%26rft.auinit1%253DA.%2BK.%26rft.volume%253D85%26rft.issue%253D1%26rft.spage%253D69%26rft.epage%253D79%26rft.atitle%253DMarker%2Bdensities%2Band%2Bthe%2Bmapping%2Bof%2Bancestral%2Bjunctions.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672305007329%26rft_id%253Dinfo%253Apmid%252F16089037%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672305007329&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16089037&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3231.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3231.30" data-doi="10.1534/genetics.111.129700"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">O. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hospital</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Distribution of parental genome blocks in recombinant inbred lines.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">189</span>: <span class="cit-fpage">645</span>–<span class="cit-lpage">654</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.111.129700</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.129700%26rft_id%253Dinfo%253Apmid%252F21840856%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg5LzIvNjQ1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMC8zMjMxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3231.31" data-doi="10.2307/2529397"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nadot</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vayssiex</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">1973</span> <span class="cit-article-title">Algorithme du calcul des coefficients d’identite.</span> <abbr class="cit-jnl-abbrev">Biometrics</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">347</span>–<span class="cit-lpage">359</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2307/2529397</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiometrics%26rft.volume%253D29%26rft.spage%253D347%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2529397%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2529397&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3231.32" data-doi="10.1111/pbi.12282"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pascual</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Desplat</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">B. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Desgroux</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bruguier</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Potential of a tomato magic population to decipher the genetic control of quantitative traits and detect causal variants in the resequencing era.</span> <abbr class="cit-jnl-abbrev">Plant Biotechnol. J.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">565</span>–<span class="cit-lpage">577</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/pbi.12282</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BBiotechnol.%2BJ.%26rft.volume%253D13%26rft.spage%253D565%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fpbi.12282%26rft_id%253Dinfo%253Apmid%252F25382275%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/pbi.12282&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25382275&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3231.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3231.33" data-doi="10.1534/genetics.107.083873"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rockman</span>  <span class="cit-name-given-names">M. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kruglyak</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Breeding designs for recombinant inbred advanced intercross lines.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">179</span>: <span class="cit-fpage">1069</span>–<span class="cit-lpage">1078</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.107.083873</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.083873%26rft_id%253Dinfo%253Apmid%252F18505881%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3OS8yLzEwNjkiO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzEwLzMyMzEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3231.34" data-doi="10.1111/j.1469-1809.1978.tb00918.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rostron</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1978</span> <span class="cit-article-title">Computation of inbreeding coefficients.</span> <abbr class="cit-jnl-abbrev">Ann. Hum. Genet.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">469</span>–<span class="cit-lpage">475</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1469-1809.1978.tb00918.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnals%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAnn%2BHum%2BGenet%26rft.aulast%253DRostron%26rft.auinit1%253DJ.%26rft.volume%253D41%26rft.issue%253D4%26rft.spage%253D469%26rft.epage%253D475%26rft.atitle%253DOn%2Bthe%2Bcomputation%2Bof%2Binbreeding%2Bcoefficients.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1469-1809.1978.tb00918.x%26rft_id%253Dinfo%253Apmid%252F655636%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1469-1809.1978.tb00918.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=655636&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3231.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3231.35" data-doi="10.1007/s11032-015-0284-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sannemann</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">B. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mathew</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leon</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Multi-parent advanced generation inter-cross in barley: high-resolution quantitative trait locus mapping for flowering time as a proof of concept.</span> <abbr class="cit-jnl-abbrev">Mol. Breed.</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">86</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s11032-015-0284-7</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBreed.%26rft.volume%253D35%26rft.spage%253D86%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11032-015-0284-7%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11032-015-0284-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3231.36" data-doi="10.1017/S0016672300014002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stam</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">1980</span> <span class="cit-article-title">The distribution of the fraction of the genome identical by descent in finite random mating populations.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">131</span>–<span class="cit-lpage">155</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1017/S0016672300014002</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BRes.%26rft.volume%253D35%26rft.spage%253D131%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672300014002%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672300014002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3231.37"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stefanov</span>  <span class="cit-name-given-names">V.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Distribution of genome shared identical by descent by two individuals in grandparent-type relationship.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">156</span>: <span class="cit-fpage">1403</span>–<span class="cit-lpage">1410</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DStefanov%26rft.auinit1%253DV.%2BT.%26rft.volume%253D156%26rft.issue%253D3%26rft.spage%253D1403%26rft.epage%253D1410%26rft.atitle%253DDistribution%2Bof%2BGenome%2BShared%2BIdentical%2Bby%2BDescent%2Bby%2BTwo%2BIndividuals%2Bin%2BGrandparent-Type%2BRelationship%26rft_id%253Dinfo%253Apmid%252F11063711%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1Ni8zLzE0MDMiO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzEwLzMyMzEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3231.38" data-doi="10.1534/genetics.106.064063"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Teuscher</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Broman</span>  <span class="cit-name-given-names">K. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Haplotype probabilities for multiple-strain recombinant inbred lines.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">175</span>: <span class="cit-fpage">1267</span>–<span class="cit-lpage">1274</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.106.064063</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.106.064063%26rft_id%253Dinfo%253Apmid%252F17151250%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3NS8zLzEyNjciO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzEwLzMyMzEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3231.39" data-doi="10.1111/j.1469-1809.1983.tb00981.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">E. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1983</span> <span class="cit-article-title">A recursive algorithm for inferring gene origins.</span> <abbr class="cit-jnl-abbrev">Ann. Hum. Genet.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">143</span>–<span class="cit-lpage">152</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1469-1809.1983.tb00981.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnals%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAnn%2BHum%2BGenet%26rft.aulast%253DThompson%26rft.auinit1%253DE.%2BA.%26rft.volume%253D47%26rft.issue%253DPt%2B2%26rft.spage%253D143%26rft.epage%253D152%26rft.atitle%253DA%2Brecursive%2Balgorithm%2Bfor%2Binferring%2Bgene%2Borigins.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1469-1809.1983.tb00981.x%26rft_id%253Dinfo%253Apmid%252F6881911%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1469-1809.1983.tb00981.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=6881911&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3231.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3231.40" data-doi="10.1093/imammb/5.4.261"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">E. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1988</span> <span class="cit-article-title">2-locus and 3-locus gene identity by descent in pedigrees.</span> <abbr class="cit-jnl-abbrev">IMA J. Math. Appl. Med. Biol.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">261</span>–<span class="cit-lpage">279</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/imammb/5.4.261</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMathematical%2BMedicine%2Band%2BBiology%26rft.stitle%253DMath%2BMed%2BBiol%26rft.aulast%253DTHOMPSON%26rft.auinit1%253DE.%2BA.%26rft.volume%253D5%26rft.issue%253D4%26rft.spage%253D261%26rft.epage%253D279%26rft.atitle%253DTwo-locus%2Band%2BThree-locus%2BGene%2BIdentity%2Bby%2BDescent%2Bin%2BPedigrees%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fimammb%252F5.4.261%26rft_id%253Dinfo%253Apmid%252F3241098%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/imammb/5.4.261&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=3241098&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3231.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3231.41"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Weir</span>  <span class="cit-name-given-names">B. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cockerham</span>  <span class="cit-name-given-names">C. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1969</span> <span class="cit-article-title">Pedigree mating with 2 linked loci.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">61</span>: <span class="cit-fpage">923</span>–<span class="cit-lpage">940</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DWeir%26rft.auinit1%253DB.%2BS.%26rft.volume%253D61%26rft.issue%253D4%26rft.spage%253D923%26rft.epage%253D940%26rft.atitle%253DPEDIGREE%2BMATING%2BWITH%2BTWO%2BLINKED%2BLOCI%26rft_id%253Dinfo%253Apmid%252F5364970%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo4OiJnZW5ldGljcyI7czo1OiJyZXNpZCI7czo4OiI2MS80LzkyMyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzgvMTAvMzIzMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3231.42" data-doi="10.1534/g3.111.001784"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Welsh</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McMillan</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Accelerating the inbreeding of multi-parental recombinant inbred lines generated by sibling matings.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">191</span>–<span class="cit-lpage">198</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.111.001784</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft.volume%253D2%26rft.spage%253D191%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.111.001784%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/g3.111.001784&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3231.43" data-doi="10.1534/genetics.113.155309"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">S. Z.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genetic mapping and genomic selection using recombination breakpoint data.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">195</span>: <span class="cit-fpage">1103</span>–<span class="cit-lpage">1115</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.113.155309</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.155309%26rft_id%253Dinfo%253Apmid%252F23979575%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5NS8zLzExMDMiO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzEwLzMyMzEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3231.44" data-doi="10.1534/g3.114.016154"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zheng</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Modeling X-linked-linked ancestral origins in multiparental populations.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">777</span>–<span class="cit-lpage">801</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.114.016154</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.114.016154%26rft_id%253Dinfo%253Apmid%252F25740936%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjc6IjUvNS83NzciO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzEwLzMyMzEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3231.45" data-doi="10.1534/genetics.114.163006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zheng</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boer</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Eeuwijk</span>  <span class="cit-name-given-names">F. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A general modeling framework for genome ancestral origins in multiparental populations.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">198</span>: <span class="cit-fpage">87</span>–<span class="cit-lpage">101</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.114.163006</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.163006%26rft_id%253Dinfo%253Apmid%252F25236451%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiMTk4LzEvODciO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzEwLzMyMzEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3231.46" data-doi="10.1534/genetics.118.300885"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zheng</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boer</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Eeuwijk</span>  <span class="cit-name-given-names">F. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Accurate genotype imputation in multiparental populations from low-coverage sequence.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-fpage">genetics.300885.2018</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.118.300885</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.118.300885%26rft_id%253Dinfo%253Apmid%252F30045858%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiMjEwLzEvNzEiO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzEwLzMyMzEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3231.47" data-doi="10.1534/genetics.111.135095"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ghazalpour</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sobel</span>  <span class="cit-name-given-names">E. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sinsheimer</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lange</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Quantitative trait loci association mapping by imputation of strain origins in multifounder crosses.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">190</span>: <span class="cit-fpage">459</span>–<span class="cit-lpage">473</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.111.135095</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.135095%26rft_id%253Dinfo%253Apmid%252F22143921%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkwLzIvNDU5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMC8zMjMxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/8/10/3231.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/10/3221" title="Null Mutation of the Fascin2 Gene by TALEN Leading to Progressive Hearing Loss and Retinal Degeneration in C57BL/6J Mice" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/10/3247" title="Genomic Structural Variations Within Five Continental Populations of Drosophila melanogaster" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/8/10"><span>Volume 8 Issue 10, October 2018</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/8/10.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/8/10.cover-source.jpg?itok=p8rfSNcT" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 8 (10)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/10/3231" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-K_Be5jY2zUIXqJZ6PCAVvSZG7JywlexZckKi4M6A3qg" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/multiparental-populations" class="" data-icon-position="" data-hide-link-title="0">Multiparental Populations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/8/10/3231" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/8/10/3231" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/8/10/3231" class="active" data-icon-position="" data-hide-link-title="0">Recursive Algorithms for Modeling Genomic Ancestral Origins in a Fixed Pedigree</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/490964" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-X20nVepQItNE8dM1gLSX-Lo9OZFBbN_txGYJuqSiS68" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/10/3231.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/8/10/3231" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-4DMb-nuY5OsIdfCmKAwHEuZpGj7wBHDPXDfmGYnZoiQ" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/8/10/3231" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/10/3231.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490964" id="node490964--2" data-pisa="ggg;8/10/3231" data-pisa-master="ggg;g3.118.200340" data-apath="/ggg/8/10/3231.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Recursive Algorithms for Modeling Genomic Ancestral Origins in a Fixed Pedigree</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-6030-3933" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Chaozhi Zheng</span>, <span class="highwire-citation-author" data-delta="1">Martin P. Boer</span> and <span class="highwire-citation-author" data-delta="2">Fred A. van Eeuwijk</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3231-3245;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200340 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node490964--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Chaozhi Zheng </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Biometris, Wageningen University and Research, Wageningen, The Netherlands</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Chaozhi%2BZheng%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zheng%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AChaozhi%2BZheng%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-6030-3933" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Chaozhi Zheng</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Martin P. Boer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Biometris, Wageningen University and Research, Wageningen, The Netherlands</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Martin%2BP.%2BBoer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Boer%20MP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMartin%2BP.%2BBoer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Fred A. van Eeuwijk </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Biometris, Wageningen University and Research, Wageningen, The Netherlands</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Fred%2BA.%2Bvan%2BEeuwijk%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=van%20Eeuwijk%20FA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFred%2BA.%2Bvan%2BEeuwijk%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/8/10/3231" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/8/10/3231&amp;title=Recursive%20Algorithms%20for%20Modeling%20Genomic%20Ancestral%20Origins%20in%20a%20Fixed%20Pedigree" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/8/10/3231&amp;title=Recursive%20Algorithms%20for%20Modeling%20Genomic%20Ancestral%20Origins%20in%20a%20Fixed%20Pedigree" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/8/10/3231&amp;title=Recursive%20Algorithms%20for%20Modeling%20Genomic%20Ancestral%20Origins%20in%20a%20Fixed%20Pedigree" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/8/10/3231&amp;text=Recursive%20Algorithms%20for%20Modeling%20Genomic%20Ancestral%20Origins%20in%20a%20Fixed%20Pedigree" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/8/10/3231&amp;title=Recursive%20Algorithms%20for%20Modeling%20Genomic%20Ancestral%20Origins%20in%20a%20Fixed%20Pedigree" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/8/10/3231&amp;t=Recursive%20Algorithms%20for%20Modeling%20Genomic%20Ancestral%20Origins%20in%20a%20Fixed%20Pedigree" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/8/10/3231&amp;title=Recursive%20Algorithms%20for%20Modeling%20Genomic%20Ancestral%20Origins%20in%20a%20Fixed%20Pedigree" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/8/10/3231&amp;title=Recursive%20Algorithms%20for%20Modeling%20Genomic%20Ancestral%20Origins%20in%20a%20Fixed%20Pedigree" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/490964/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490964" id="node490964--3" data-pisa="ggg;8/10/3231" data-pisa-master="ggg;g3.118.200340" data-apath="/ggg/8/10/3231.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Recursive Algorithms for Modeling Genomic Ancestral Origins in a Fixed Pedigree</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-6030-3933" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Chaozhi Zheng</span>, <span class="highwire-citation-author" data-delta="1">Martin P. Boer</span> and <span class="highwire-citation-author" data-delta="2">Fred A. van Eeuwijk</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3231-3245;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200340 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node490964--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Chaozhi Zheng </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Biometris, Wageningen University and Research, Wageningen, The Netherlands</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Chaozhi%2BZheng%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zheng%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AChaozhi%2BZheng%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-6030-3933" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Chaozhi Zheng</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Martin P. Boer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Biometris, Wageningen University and Research, Wageningen, The Netherlands</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Martin%2BP.%2BBoer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Boer%20MP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMartin%2BP.%2BBoer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Fred A. van Eeuwijk </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Biometris, Wageningen University and Research, Wageningen, The Netherlands</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Fred%2BA.%2Bvan%2BEeuwijk%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=van%20Eeuwijk%20FA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFred%2BA.%2Bvan%2BEeuwijk%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=3231&amp;rft.epage=3245&amp;rft.atitle=Recursive%20Algorithms%20for%20Modeling%20Genomic%20Ancestral%20Origins%20in%20a%20Fixed%20Pedigree&amp;rft.volume=8&amp;rft.issue=10&amp;rft.date=2018-10-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Zheng%2C+Chaozhi&amp;rft.au=Boer%2C+Martin+P.&amp;rft.au=van+Eeuwijk%2C+Fred+A."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/490964/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/490964/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/490964/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/490964/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/490964/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/490964/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/490964/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/490964/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/490964/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/490964/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/490964/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/490964/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495847" id="node495847" data-pisa="ggg;10/3/951" data-pisa-master="ggg;g3.119.400763" data-apath="/ggg/10/3/951.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/951" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Genome-Wide Association Study in Two Cohorts from a Multi-generational Mouse Advanced Intercross Line Highlights the Difficulty of Replication Due to Study-Specific Heterogeneity</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494983" id="node494983" data-pisa="ggg;9/12/4045" data-pisa-master="ggg;g3.119.400353" data-apath="/ggg/9/12/4045.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/9/12/4045" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Genetic Architecture of Chilling Tolerance in Sorghum Dissected with a Nested Association Mapping Population</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="493460" id="node493460" data-pisa="ggg;9/7/2317" data-pisa-master="ggg;g3.119.400098" data-apath="/ggg/9/7/2317.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/9/7/2317" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Testing Pleiotropy <em>vs.</em> Separate QTL in Multiparental Populations</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Multiparental%20Populations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Multiparental Populations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/8/10/3231" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Recursive Algorithm Exch</a></li><li class="odd"><a href="#sec-5" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Recursive Algorithm Non-Exch</a></li><li class="even"><a href="#sec-10" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Application to Multiparental Populations</a></li><li class="odd"><a href="#sec-17" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#app-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">APPENDIX A: RECURRENCE RELATIONS OF ANCESTRAL COEFFICIENTS</a></li><li class="even"><a href="#app-2" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">APPENDIX B: RECURRENCE RELATIONS OF EXPECTED ANCESTRAL JUNCTION DENSITIES</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/8/10/3231.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/8/10/3231.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/8/10/3231.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
