<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/8/10/3185" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/8/10/3185.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/8/10/3185.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/8/10/3185.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/8/10/3185/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/8/10/3185.atom" />
<meta name="HW.pisa" content="ggg;8/10/3185" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Linkage Disequilibrium and Evaluation of Genome-Wide Association Mapping Models in Tetraploid Potato" />
<meta name="DC.Identifier" content="10.1534/g3.118.200377" />
<meta name="DC.Date" content="2018-10-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2018 Sharma et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Relation" content="https://doi.org/10.25387/g3.6716033" />
<meta name="DC.Description" content="Genome-wide association studies (GWAS) have become a powerful tool for analyzing complex traits in crop plants. The current study evaluates the efficacy of various GWAS models and methods for elucidating population structure in potato. The presence of significant population structure can lead to detection of spurious marker-trait associations, as well as mask true ones. While appropriate statistical models are needed to detect true marker-trait associations, in most published potato GWAS, a ‘one model fits all traits’ approach has been adopted. We have examined various GWAS models on a large association panel comprising diverse tetraploid potato cultivars and breeding lines, genotyped with single nucleotide polymorphism (SNP) markers. Phenotypic data were generated for 20 quantitative traits assessed in different environments. Best Linear Unbiased Estimates (BLUEs) for these traits were obtained for use in assessing GWAS models. Goodness of fit of GWAS models, derived using different combinations of kinship and population structure for all traits, was evaluated using Quantile-Quantile (Q-Q) plots and genomic control inflation factors (λGC). Kinship was found to play a major role in correcting population confounding effects and results advocate a ‘trait-specific’ fit of different GWAS models. A survey of genome-wide linkage disequilibrium (LD), one of the critical factors affecting GWAS, is also presented and our findings are compared to other recent studies in potato. The genetic material used here, and the outputs of this study represent a novel resource for genetic analysis in potato." />
<meta name="DC.Contributor" content="Sanjeev Kumar Sharma" />
<meta name="DC.Contributor" content="Katrin MacKenzie" />
<meta name="DC.Contributor" content="Karen McLean" />
<meta name="DC.Contributor" content="Finlay Dale" />
<meta name="DC.Contributor" content="Steve Daniels" />
<meta name="DC.Contributor" content="Glenn J. Bryan" />
<meta name="article:published_time" content="2018-10-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Linkage Disequilibrium and Evaluation of Genome-Wide Association Mapping Models in Tetraploid Potato" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Genome-wide association studies (GWAS) have become a powerful tool for analyzing complex traits in crop plants. The current study evaluates the efficacy of various GWAS models and methods for elucidating population structure in potato. The presence of significant population structure can lead to detection of spurious marker-trait associations, as well as mask true ones. While appropriate statistical models are needed to detect true marker-trait associations, in most published potato GWAS, a ‘one model fits all traits’ approach has been adopted. We have examined various GWAS models on a large association panel comprising diverse tetraploid potato cultivars and breeding lines, genotyped with single nucleotide polymorphism (SNP) markers. Phenotypic data were generated for 20 quantitative traits assessed in different environments. Best Linear Unbiased Estimates (BLUEs) for these traits were obtained for use in assessing GWAS models. Goodness of fit of GWAS models, derived using different combinations of kinship and population structure for all traits, was evaluated using Quantile-Quantile (Q-Q) plots and genomic control inflation factors (λ&lt;sub&gt;GC&lt;/sub&gt;). Kinship was found to play a major role in correcting population confounding effects and results advocate a ‘trait-specific’ fit of different GWAS models. A survey of genome-wide linkage disequilibrium (LD), one of the critical factors affecting GWAS, is also presented and our findings are compared to other recent studies in potato. The genetic material used here, and the outputs of this study represent a novel resource for genetic analysis in potato.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2018/10/01" />
<meta name="citation_mjid" content="ggg;8/10/3185" />
<meta name="citation_id" content="8/10/3185" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/8/10/3185" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/8/10/3185.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/8/10/3185.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/8/10/3185.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.118.200377" />
<meta name="citation_pmid" content="30082329" />
<meta name="citation_volume" content="8" />
<meta name="citation_issue" content="10" />
<meta name="citation_num_pages" content="18" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="3185" />
<meta name="citation_lastpage" content="3202" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Sanjeev Kumar Sharma" />
<meta name="citation_author_institution" content="The James Hutton Institute, Dundee DD2 5DA, UK" />
<meta name="citation_author_email" content="glenn.bryan@hutton.ac.uk" />
<meta name="citation_author_email" content="sanjeev.sharma@hutton.ac.uk" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-8373-5509" />
<meta name="citation_author" content="Katrin MacKenzie" />
<meta name="citation_author_institution" content="Biomathematics and Statistics Scotland (BioSS), Dundee DD2 5DA, UK" />
<meta name="citation_author" content="Karen McLean" />
<meta name="citation_author_institution" content="The James Hutton Institute, Dundee DD2 5DA, UK" />
<meta name="citation_author" content="Finlay Dale" />
<meta name="citation_author_institution" content="The James Hutton Institute, Dundee DD2 5DA, UK" />
<meta name="citation_author" content="Steve Daniels" />
<meta name="citation_author_institution" content="Cygnet PB Ltd, Cambridge, CB21 6AS, UK" />
<meta name="citation_author" content="Glenn J. Bryan" />
<meta name="citation_author_institution" content="The James Hutton Institute, Dundee DD2 5DA, UK" />
<meta name="citation_author_email" content="glenn.bryan@hutton.ac.uk" />
<meta name="citation_author_email" content="sanjeev.sharma@hutton.ac.uk" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-7752-5740" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=U. Achenbach;citation_author=J. Paulo;citation_author=E. Ilarionova;citation_author=J. Lubeck;citation_author=J. Strahwald;citation_title=Using SNP markers to dissect linkage disequilibrium at a major quantitative trait locus for resistance to the potato cyst nematode Globodera pallida on potato chromosome V.;citation_pages=619-629;citation_volume=118;citation_year=2009;citation_issue=3;citation_pmid=19020852;citation_doi=10.1007/s00122-008-0925-x" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=J. Berdugo-Cely;citation_author=R. I. Valbuena;citation_author=E. Sanchez-Betancourt;citation_author=L. S. Barrero;citation_author=R. Yockteng;citation_title=Genetic diversity and association mapping in the Colombian Central Collection of Solanum tuberosum L. Andigenum group using SNPs markers.;citation_pages=e0173039;citation_volume=12;citation_year=2017;citation_doi=10.1371/journal.pone.0173039" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=M. W. Bonierbale;citation_author=R. L. Plaisted;citation_author=S. D. Tanksley;citation_title=RFLP Maps Based on a Common Set of Clones Reveal Modes of Chromosomal Evolution in Potato and Tomato.;citation_pages=1095-1103;citation_volume=120;citation_year=1988;citation_issue=4;citation_pmid=17246486" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=D. Botstein;citation_author=R. L. White;citation_author=M. Skolnick;citation_author=R. W. Davis;citation_title=Construction of a genetic linkage map in man using restriction fragment length polymorphisms.;citation_pages=314-331;citation_volume=32;citation_year=1980;citation_issue=3;citation_pmid=6247908" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=M. Bouaziz;citation_author=C. Ambroise;citation_author=M. Guedj;citation_title=Accounting for population stratification in practice: a comparison of the main strategies dedicated to genome-wide association studies.;citation_pages=e28845-e28845;citation_volume=6;citation_year=2011;citation_issue=12;citation_pmid=22216125;citation_doi=10.1371/journal.pone.0028845" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=J. E. Bradshaw;citation_author=C. A. Hackett;citation_author=B. Pande;citation_author=R. Waugh;citation_author=G. J. Bryan;citation_title=QTL mapping of yield, agronomic and quality traits in tetraploid potato (Solanum tuberosum subsp. tuberosum).;citation_pages=193-211;citation_volume=116;citation_year=2008;citation_issue=2;citation_pmid=17938877;citation_doi=10.1007/s00122-007-0659-1" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=F. Breseghello;citation_author=M. E. Sorrells;citation_title=Association mapping of kernel size and milling quality in wheat (Triticum aestivum L.) cultivars.;citation_volume=172;citation_year=2006;citation_pmid=16079235;citation_doi=10.1534/genetics.105.044586" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=B. B. D’hoop;citation_author=P. L. C. Keizer;citation_author=M. J. Paulo;citation_author=R. G. F. Visser;citation_author=F. A. van Eeuwijk;citation_title=Identification of agronomically important QTL in tetraploid potato cultivars using a marker-trait association analysis.;citation_pages=731-748;citation_volume=127;citation_year=2014;citation_pmid=24408376;citation_doi=10.1007/s00122-013-2254-y" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=B. B. D’hoop;citation_author=M. J. Paulo;citation_author=K. Kowitwanich;citation_author=M. Sengers;citation_author=R. G. F. Visser;citation_title=Population structure and linkage disequilibrium unravelled in tetraploid potato.;citation_pages=1151-1170;citation_volume=121;citation_year=2010;citation_issue=6;citation_pmid=20563789;citation_doi=10.1007/s00122-010-1379-5" />
<meta name="citation_reference" content="citation_journal_title=Euphytica;citation_author=B. B. D’hoop;citation_author=M. J. Paulo;citation_author=R. A. Mank;citation_author=H. J. van Eck;citation_author=F. A. van Eeuwijk;citation_title=Association mapping of quality traits in potato (Solanum tuberosum L.).;citation_pages=47-60;citation_volume=161;citation_year=2008;citation_doi=10.1007/s10681-007-9565-5" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=G. Evanno;citation_author=S. Regnaut;citation_author=J. Goudet;citation_title=Detecting the number of clusters of individuals using the software STRUCTURE: a simulation study.;citation_pages=2611-2620;citation_volume=14;citation_year=2005;citation_issue=8;citation_pmid=15969739;citation_doi=10.1111/j.1365-294X.2005.02553.x" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=K. J. Felcher;citation_author=J. J. Coombs;citation_author=A. N. Massa;citation_author=C. N. Hansey;citation_author=J. P. Hamilton;citation_title=Integration of two diploid potato linkage maps with the potato genome sequence.;citation_pages=e36347-e36347;citation_volume=7;citation_year=2012;citation_issue=4;citation_pmid=22558443;citation_doi=10.1371/journal.pone.0036347" />
<meta name="citation_reference" content="citation_journal_title=BMC Plant Biol.;citation_author=M. Fischer;citation_author=L. Schreiber;citation_author=T. Colby;citation_author=M. Kuckenberg;citation_author=E. Tacke;citation_title=Novel candidate genes influencing natural variation in potato tuber cold sweetening identified by comparative proteomics and association mapping.;citation_pages=113;citation_volume=13;citation_year=2013;citation_pmid=23919263;citation_doi=10.1186/1471-2229-13-113" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. J. Garris;citation_author=T. H. Tai;citation_author=J. Coburn;citation_author=S. Kresovich;citation_author=S. McCouch;citation_title=Genetic structure and diversity in Oryza sativa L.;citation_volume=169;citation_year=2005;citation_pmid=15654106;citation_doi=10.1534/genetics.104.035642" />
<meta name="citation_reference" content="citation_journal_title=Mol. Breed.;citation_author=C. Gebhardt;citation_author=A. Ballvora;citation_author=B. Walkemeier;citation_author=P. Oberhagemann;citation_author=K. Schuler;citation_title=Assessing genetic potential in germplasm collections of crop plants by marker-trait association: a case study for potatoes with quantitative variation of resistance to late blight and maturity type.;citation_pages=93-102;citation_volume=13;citation_year=2004;citation_doi=10.1023/B:MOLB.0000012878.89855.df" />
<meta name="citation_reference" content="GenABEL project developers, 2013 GenABEL: genome-wide SNP association analysis." />
<meta name="citation_reference" content="citation_journal_title=Biometrics;citation_journal_abbrev=Biometrics;citation_author=J. C. Gower;citation_title=A comparison of some methods of cluster analysis.;citation_pages=623-637;citation_volume=23;citation_year=1967;citation_issue=4;citation_pmid=6080200;citation_doi=10.2307/2528417" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=C. A. Hackett;citation_author=J. E. Bradshaw;citation_author=G. J. Bryan;citation_title=QTL mapping in autotetraploids using SNP dosage information.;citation_pages=1885-1904;citation_volume=127;citation_year=2014;citation_pmid=24981609;citation_doi=10.1007/s00122-014-2347-2" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=C. A. Hackett;citation_author=K. McLean;citation_author=G. J. Bryan;citation_title=Linkage Analysis and QTL Mapping Using SNP Dosage Data in a Tetraploid Potato Mapping Population.;citation_pages=e63939;citation_volume=8;citation_year=2013;citation_pmid=23704960;citation_doi=10.1371/journal.pone.0063939" />
<meta name="citation_reference" content="citation_journal_title=Brief. Funct. Genomics;citation_author=D. Hall;citation_author=C. Tegstrom;citation_author=P. K. Ingvarsson;citation_title=Using association mapping to dissect the genetic basis of complex traits in plants.;citation_volume=9;citation_year=2010;citation_pmid=20053815;citation_doi=10.1093/bfgp/elp048" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=J. P. Hamilton;citation_author=C. N. Hansey;citation_author=B. R. Whitty;citation_author=K. Stoffel;citation_author=A. N. Massa;citation_title=Single nucleotide polymorphism discovery in elite North American potato germplasm.;citation_pages=302-302;citation_volume=12;citation_year=2011;citation_pmid=21658273;citation_doi=10.1186/1471-2164-12-302" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=M. A. Hardigan;citation_author=J. Bamberg;citation_author=C. R. Buell;citation_author=D. S. Douches;citation_title=Taxonomy and Genetic Differentiation among Wild and Cultivated Germplasm of Solanum sect. Petota.;citation_pages=1-16;citation_volume=8;citation_year=2015;citation_doi=10.3835/plantgenome2014.06.0025" />
<meta name="citation_reference" content="citation_author=C.N. Hirsch;citation_author=C.D. Hirsch;citation_author=K. Felcher;citation_author=J. Coombs;citation_author=D. Zarka;citation_year=2013;citation_pmid=23589519;citation_doi=10.1534/g3.113.005595" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=J. M. E. Jacobs;citation_author=H. J. Vaneck;citation_author=P. Arens;citation_author=B. Verkerkbakker;citation_author=B. T. L. Hekkert;citation_title=A Genetic-Map of potato (Solanum-Tuberosum) Intergrating Molecular Markers, Icluding Transposons, and Classical Markers.;citation_pages=289-300;citation_volume=91;citation_year=1995;citation_doi=10.1007/BF00220891" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=B. Kloosterman;citation_author=J. A. Abelenda;citation_author=M. D. C. Gomez;citation_author=M. Oortwijn;citation_author=J. M. de Boer;citation_title=Naturally occurring allele diversity allows potato cultivation in northern latitudes.;citation_pages=246-250;citation_volume=495;citation_year=2013;citation_pmid=23467094;citation_doi=10.1038/nature11912" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=B. Kloosterman;citation_author=M. Oortwijn;citation_author=J. Uitdewilligen;citation_author=T. America;citation_author=R. de Vos;citation_title=From QTL to candidate gene: genetical genomics of simple and complex traits in potato using a pooling strategy.;citation_pages=158-158;citation_volume=11;citation_year=2010;citation_pmid=20210995;citation_doi=10.1186/1471-2164-11-158" />
<meta name="citation_reference" content="citation_author=R. Koenker;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Am. J. Potato Res.;citation_author=S. A. Kolech;citation_author=D. Halseth;citation_author=K. Perry;citation_author=D. Wolfe;citation_author=D. S. Douches;citation_title=Genetic Diversity and Relationship of Ethiopian Potato Varieties to Germplasm from North America, Europe and the International Potato Center.;citation_pages=609-619;citation_volume=93;citation_year=2016;citation_doi=10.1007/s12230-016-9543-3" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=L. Li;citation_author=J. Strahwald;citation_author=H. R. Hofferbert;citation_author=J. Lubeck;citation_author=E. Tacke;citation_title=DNA variation at the invertase locus invGE/GF is associated with tuber quality traits in populations of potato breeding clones.;citation_volume=170;citation_year=2005;citation_pmid=15802505;citation_doi=10.1534/genetics.104.040006" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=X. H. Li;citation_author=Y. L. Wei;citation_author=K. J. Moore;citation_author=R. Michaud;citation_author=D. R. Viands;citation_title=Association Mapping of Biomass Yield and Stem Composition in a Tetraploid Alfalfa Breeding Population.;citation_pages=24-35;citation_volume=4;citation_year=2011;citation_doi=10.3835/plantgenome2010.09.0022" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=X. Q. Li;citation_author=H. De Jong;citation_author=D. M. De Jong;citation_author=W. S. De Jong;citation_title=Inheritance and genetic mapping of tuber eye depth in cultivated diploid potatoes.;citation_pages=1068-1073;citation_volume=110;citation_year=2005;citation_issue=6;citation_pmid=15719211;citation_doi=10.1007/s00122-005-1927-6" />
<meta name="citation_reference" content="citation_journal_title=Am. J. Potato Res.;citation_author=G. J. Lightbourn;citation_author=R. E. Veilleux;citation_title=Production and evaluation of somatic hybrids derived from monoploid potato.;citation_pages=425-435;citation_volume=84;citation_year=2007;citation_doi=10.1007/BF02987188" />
<meta name="citation_reference" content="citation_journal_title=Phytopathology;citation_author=H. Lindqvist-Kreuze;citation_author=M. Gastelo;citation_author=W. Perez;citation_author=G. A. Forbes;citation_author=D. de Koeyer;citation_title=Phenotypic Stability and Genome-Wide Association Study of Late Blight Resistance in Potato Genotypes Adapted to the Tropical Highlands.;citation_pages=624-633;citation_volume=104;citation_year=2014;citation_doi=10.1094/PHYTO-10-13-0270-R" />
<meta name="citation_reference" content="citation_journal_title=Am. J. Potato Res.;citation_author=S. L. Love;citation_title=Founding clones, major contributing ancestors, and exotic progenitors of prominent North American potato cultivars.;citation_pages=263-272;citation_volume=76;citation_year=1999;citation_doi=10.1007/BF02853624" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. Malosetti;citation_author=C. G. van der Linden;citation_author=B. Vosman;citation_author=F. A. van Eeuwijk;citation_title=A mixed-model approach to association mapping using pedigree information with an illustration of resistance to Phytophthora infestans in potato.;citation_volume=175;citation_year=2007;citation_pmid=17151263;citation_doi=10.1534/genetics.105.054932" />
<meta name="citation_reference" content="citation_journal_title=Tree Genet. Genomes;citation_author=F. Marroni;citation_author=S. Pinosio;citation_author=G. Zaina;citation_author=F. Fogolari;citation_author=N. Felice;citation_title=Nucleotide diversity and linkage disequilibrium in Populus nigra cinnamyl alcohol dehydrogenase (CAD4) gene.;citation_pages=1011-1023;citation_volume=7;citation_year=2011;citation_doi=10.1007/s11295-011-0391-5" />
<meta name="citation_reference" content="citation_journal_title=Genetic epidemiology;citation_journal_abbrev=Genet Epidemiol;citation_author=K. Miclaus;citation_author=R. Wolfinger;citation_author=W. Czika;citation_title=SNP selection and multidimensional scaling to quantify population structure.;citation_pages=488-496;citation_volume=33;citation_year=2009;citation_issue=6;citation_pmid=19194989;citation_doi=10.1002/gepi.20401" />
<meta name="citation_reference" content="citation_journal_title=J. Classif.;citation_author=F. Murtagh;citation_author=P. Legendre;citation_title=Ward’s Hierarchical Agglomerative Clustering Method: Which Algorithms Implement Ward’s Criterion?;citation_pages=274-295;citation_volume=31;citation_year=2014;citation_doi=10.1007/s00357-014-9161-z" />
<meta name="citation_reference" content="citation_journal_title=Am. Nat.;citation_author=M. Nei;citation_title=GENETIC DISTANCE BETWEEN POPULATIONS.;citation_pages=283-292;citation_volume=106;citation_year=1972;citation_doi=10.1086/282771" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=K. Pajerowska-Mukhtar;citation_author=B. Stich;citation_author=U. Achenbach;citation_author=A. Ballvora;citation_author=J. Luebeck;citation_title=Single Nucleotide Polymorphisms in the Allene Oxide Synthase 2 Gene Are Associated With Field Resistance to Late Blight in Populations of Tetraploid Potato Cultivars.;citation_volume=181;citation_year=2009;citation_pmid=19139145;citation_doi=10.1534/genetics.108.094268" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_title=Genome sequence and analysis of the tuber crop potato.;citation_pages=189-195;citation_volume=475;citation_year=2011;citation_issue=7355;citation_pmid=21743474;citation_doi=10.1038/nature10158" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=A. Prashar;citation_author=C. Hornyik;citation_author=V. Young;citation_author=K. McLean;citation_author=S. K. Sharma;citation_title=Construction of a dense SNP map of a highly heterozygous diploid potato population and QTL analysis of tuber shape and eye depth.;citation_pages=2159-2171;citation_volume=127;citation_year=2014;citation_doi=10.1007/s00122-014-2369-9" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. K. Pritchard;citation_author=M. Stephens;citation_author=P. Donnelly;citation_title=Inference of Population Structure Using Multilocus Genotype Data;citation_pages=945-959;citation_volume=155;citation_year=2000;citation_issue=2;citation_pmid=10835412;citation_doi=10.1111/j.1471-8286.2005.01155.x" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=U. R. Rosyara;citation_author=W. S. De Jong;citation_author=D. S. Douches;citation_author=J. B. Endelman;citation_title=Software for Genome-Wide Association Studies in Autopolyploids and Its Application to Potato.;citation_pages=1-10;citation_volume=9;citation_year=2016;citation_doi=10.3835/plantgenome2016.03.0024" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=C. A. Schmitz Carley;citation_author=J. J. Coombs;citation_author=D. S. Douches;citation_author=P. C. Bethke;citation_author=J. P. Palta;citation_title=Automated tetraploid genotype calling by hierarchical clustering.;citation_pages=717-726;citation_volume=130;citation_year=2017;citation_doi=10.1007/s00122-016-2845-5" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=E. Schönhals;citation_author=F. Ortega;citation_author=L. Barandalla;citation_author=A. Aragones;citation_author=J. I. R. de Galarreta;citation_title=Identification and reproducibility of diagnostic DNA markers for tuber starch and yield optimization in a novel association mapping population of potato (Solanum tuberosum L.).;citation_pages=767-785;citation_volume=129;citation_year=2016;citation_doi=10.1007/s00122-016-2665-7" />
<meta name="citation_reference" content="citation_author=S.K. Sharma;citation_author=D. Bolser;citation_author=J. de Boer;citation_author=M. Sonderkaer;citation_author=W. Amoros;citation_year=2013;citation_pmid=24062527;citation_doi=10.1534/g3.113.007153" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=I. Simko;citation_author=S. Costanzo;citation_author=K. G. Haynes;citation_author=B. J. Christ;citation_author=R. W. Jones;citation_title=Linkage disequilibrium mapping of a Verticillium dahliae resistance quantitative trait locus in tetraploid potato ( Solanum tuberosum) through a candidate gene approach.;citation_pages=217-224;citation_volume=108;citation_year=2004;citation_issue=2;citation_pmid=14523522;citation_doi=10.1007/s00122-003-1431-9" />
<meta name="citation_reference" content="citation_journal_title=Molecular genetics and genomics :  MGG;citation_journal_abbrev=Mol Genet Genomics;citation_author=I. Simko;citation_author=K. G. Haynes;citation_author=E. E. Ewing;citation_author=S. Costanzo;citation_author=B. J. Christ;citation_title=Mapping genes for resistance to Verticillium albo-atrum in tetraploid and diploid potato populations using haplotype association tests and genetic linkage analysis.;citation_pages=522-531;citation_volume=271;citation_year=2004;citation_issue=5;citation_pmid=15107986;citation_doi=10.1007/s00438-004-1010-z" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=I. Simko;citation_author=K. G. Haynes;citation_author=R. W. Jones;citation_title=Assessment of linkage disequilibrium in potato genome with single nucleotide polymorphism markers.;citation_volume=173;citation_year=2006;citation_pmid=16783002;citation_doi=10.1534/genetics.106.060905" />
<meta name="citation_reference" content="citation_journal_title=Plant Breed.;citation_author=J. Śliwka;citation_author=I. Wasilewicz-Flis;citation_author=H. Jakuczun;citation_author=C. Gebhardt;citation_title=Tagging quantitative trait loci for dormancy, tuber shape, regularity of tuber shape, eye depth and flesh colour in diploid potato originated from six Solanum species.;citation_pages=49-55;citation_volume=127;citation_year=2008;citation_doi=10.1111/j.1439-0523.2008.01420.x" />
<meta name="citation_reference" content="citation_journal_title=Plant Breed.;citation_author=B. Stich;citation_author=C. Urbany;citation_author=P. Hoffmann;citation_author=C. Gebhardt;citation_title=Population structure and linkage disequilibrium in diploid and tetraploid potato revealed by genome-wide high-density genotyping using the SolCAP SNP array.;citation_pages=718-724;citation_volume=132;citation_year=2013;citation_doi=10.1111/pbr.12102" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=L. Tommasini;citation_author=T. Schnurbusch;citation_author=D. Fossati;citation_author=F. Mascher;citation_author=B. Keller;citation_title=Association mapping of Stagonospora nodorum blotch resistance in modern European winter wheat varieties.;citation_pages=697-708;citation_volume=115;citation_year=2007;citation_issue=5;citation_pmid=17634916;citation_doi=10.1007/s00122-007-0601-6" />
<meta name="citation_reference" content="citation_journal_title=Plant Pathol.;citation_author=B. R. Trognitz;citation_author=F. C. Trognitz;citation_title=Occurrence of the R1 allele conferring resistance to late blight in potato R-gene differentials and commercial cultivars.;citation_pages=150-155;citation_volume=56;citation_year=2007;citation_doi=10.1111/j.1365-3059.2006.01489.x" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=J. Uitdewilligen;citation_author=A. M. A. Wolters;citation_author=B. B. D’Hoop;citation_author=T. J. A. Borm;citation_author=R. G. F. Visser;citation_title=A Next-Generation Sequencing Method for Genotyping-by-Sequencing of Highly Heterozygous Autotetraploid Potato.;citation_pages=e62355;citation_volume=8;citation_year=2013;citation_pmid=23667470;citation_doi=10.1371/journal.pone.0062355" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=C. Urbany;citation_author=B. Stich;citation_author=L. Schmidt;citation_author=L. Simon;citation_author=H. Berding;citation_title=Association genetics in Solanum tuberosum provides new insights into potato tuber bruising and enzymatic tissue discoloration.;citation_pages=7-7;citation_volume=12;citation_year=2011;citation_pmid=21208436;citation_doi=10.1186/1471-2164-12-7" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=H. J. van Eck;citation_author=P. G. Vos;citation_author=J. P. T. Valkonen;citation_author=J. Uitdewilligen;citation_author=H. Lensing;citation_title=Graphical genotyping as a method to map Ny ((o,n)sto) and Gpa5 using a reference panel of tetraploid potato cultivars.;citation_pages=515-528;citation_volume=130;citation_year=2017;citation_doi=10.1007/s00122-016-2831-y" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=H. J. Vaneck;citation_author=J. M. E. Jacobs;citation_author=P. Stam;citation_author=J. Ton;citation_author=W. J. Stiekema;citation_title=Multiple alleles for tuber shape in diploid potato detected by qualitative and quantitative genetic analysis using RFLPs.;citation_pages=303-309;citation_volume=137;citation_year=1994;citation_issue=1;citation_pmid=7914504" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=R. E. Voorrips;citation_author=G. Gort;citation_author=B. Vosman;citation_title=Genotype calling in tetraploid species from bi-allelic marker data using mixture models.;citation_pages=172-172;citation_volume=12;citation_year=2011;citation_pmid=21595880;citation_doi=10.1186/1471-2105-12-172" />
<meta name="citation_reference" content="citation_title=GWAS in tetraploid potato: Identification and validation of SNP markers associated with glycoalkaloid content;citation_author=P. G. Vos;citation_author=M. J. Paulo;citation_author=P. M. Bourke;citation_author=C. A. Maliepaard;citation_author=R. G. F. Visser;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=P. G. Vos;citation_author=M. J. Paulo;citation_author=R. E. Voorrips;citation_author=R. G. F. Visser;citation_author=H. J. van Eck;citation_title=Evaluation of LD decay and various LD-decay estimators in simulated and SNP-array data of tetraploid potato.;citation_pages=123-135;citation_volume=130;citation_year=2017;citation_doi=10.1007/s00122-016-2798-8" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=P. G. Vos;citation_author=J. Uitdewilligen;citation_author=R. E. Voorrips;citation_author=R. G. F. Visser;citation_author=H. J. van Eck;citation_title=Development and analysis of a 20K SNP array for potato (Solanum tuberosum): an insight into the breeding history.;citation_pages=2387-2401;citation_volume=128;citation_year=2015;citation_doi=10.1007/s00122-015-2593-y" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=J. M. Yu;citation_author=G. Pressoir;citation_author=W. H. Briggs;citation_author=I. V. Bi;citation_author=M. Yamasaki;citation_title=A unified mixed-model method for association mapping that accounts for multiple levels of relatedness.;citation_pages=203-208;citation_volume=38;citation_year=2006;citation_issue=2;citation_pmid=16380716;citation_doi=10.1038/ng1702" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Commun;citation_author=K. Zhao;citation_author=C. W. Tung;citation_author=G. C. Eizenga;citation_author=M. H. Wright;citation_author=M. L. Ali;citation_title=Genome-wide association mapping reveals a rich genetic architecture of complex traits in Oryza sativa.;citation_pages=467-467;citation_volume=2;citation_year=2011;citation_pmid=21915109;citation_doi=10.1038/ncomms1467" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Linkage Disequilibrium and Evaluation of Genome-Wide Association Mapping Models in Tetraploid Potato" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10.cover-source.jpg" />
<meta name="twitter:description" content="Genome-wide association studies (GWAS) have become a powerful tool for analyzing complex traits in crop plants. The current study evaluates the efficacy of various GWAS models and methods for elucidating population structure in potato. The presence of significant population structure can lead to detection of spurious marker-trait associations, as well as mask true ones. While appropriate statistical models are needed to detect true marker-trait associations, in most published potato GWAS, a ‘one model fits all traits’ approach has been adopted. We have examined various GWAS models on a large association panel comprising diverse tetraploid potato cultivars and breeding lines, genotyped with single nucleotide polymorphism (SNP) markers. Phenotypic data were generated for 20 quantitative traits assessed in different environments. Best Linear Unbiased Estimates (BLUEs) for these traits were obtained for use in assessing GWAS models. Goodness of fit of GWAS models, derived using different combinations of kinship and population structure for all traits, was evaluated using Quantile-Quantile (Q-Q) plots and genomic control inflation factors (λGC). Kinship was found to play a major role in correcting population confounding effects and results advocate a ‘trait-specific’ fit of different GWAS models. A survey of genome-wide linkage disequilibrium (LD), one of the critical factors affecting GWAS, is also presented and our findings are compared to other recent studies in potato. The genetic material used here, and the outputs of this study represent a novel resource for genetic analysis in potato." />
<meta name="og-title" property="og:title" content="Linkage Disequilibrium and Evaluation of Genome-Wide Association Mapping Models in Tetraploid Potato" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/8/10/3185" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Genome-wide association studies (GWAS) have become a powerful tool for analyzing complex traits in crop plants. The current study evaluates the efficacy of various GWAS models and methods for elucidating population structure in potato. The presence of significant population structure can lead to detection of spurious marker-trait associations, as well as mask true ones. While appropriate statistical models are needed to detect true marker-trait associations, in most published potato GWAS, a ‘one model fits all traits’ approach has been adopted. We have examined various GWAS models on a large association panel comprising diverse tetraploid potato cultivars and breeding lines, genotyped with single nucleotide polymorphism (SNP) markers. Phenotypic data were generated for 20 quantitative traits assessed in different environments. Best Linear Unbiased Estimates (BLUEs) for these traits were obtained for use in assessing GWAS models. Goodness of fit of GWAS models, derived using different combinations of kinship and population structure for all traits, was evaluated using Quantile-Quantile (Q-Q) plots and genomic control inflation factors (λGC). Kinship was found to play a major role in correcting population confounding effects and results advocate a ‘trait-specific’ fit of different GWAS models. A survey of genome-wide linkage disequilibrium (LD), one of the critical factors affecting GWAS, is also presented and our findings are compared to other recent studies in potato. The genetic material used here, and the outputs of this study represent a novel resource for genetic analysis in potato." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10.cover-source.jpg" />
    <title>Linkage Disequilibrium and Evaluation of Genome-Wide Association Mapping Models in Tetraploid Potato | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"veW9mqL3XevJ8_XJCnplRK-RaWjnaPxFknM3F4J022o","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"490969","apath":"\/ggg\/8\/10\/3185.atom","pisa":"ggg;8\/10\/3185","ac":{"\/ggg\/8\/10\/3185.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/8\/10\/3185.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;8\/10\/3185"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;8\/10\/3185"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/8\/10\/3185.abstract":true,"\/content\/8\/10\/3185":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00225zzIgSpIJteK61LoDVKI\\\\\\\/JUEb9DOZzQ672WAou7YueCFkW3gss7Z6rlBRxKdzoLNSUZchfJRp9kXgoE3GL4WJ5hGtXMd08Vgm+pCwwRDyjN01Tw86eWxwcrOEI21g6yb43+QpL7791lSNV1i3kfVkvs0Uo\\\\\\\/t\\\\\\\/WcHM4FP1rYEqYQzIveoB2yj9gu43OW\\\\\\\/W+X9PSv\\\\\\\/qN3lqfyDpnN5r0+8q7i8NM8GoNhhe6nvwRtO+kyiWfMUBUDAy94mmzDXssjUP4U+LiUV+djV+5BdGyqKq6NOjw\\\\\\\/tD+\\\\\\\/eMmJx7tuH9uVHBFZdX6sIbPzUqr8+s8iES\\\\\\\/4cEG1tnkvEJVw73fS7dZjzgq8ovKOOPTrnLvon4qhIMoOwPDRu4p80Z+NFm8OrJ2WuxvdCubQ9MZcFOTHgetA2Zl6dDWidb8iJSluAPgbCrEAXSFzrFIHnOUwUaKxFYr4wqGbyaUYBbLJ7hkM7E3Gj5xa3V3\\\\\\\/vxeL\\\\\\\/xbpPRDe\\\\\\\/JuockwLzbE4s8X5oYvTphDTFGMC9Cskv7BCjy6oxmV9we3kY4lwV0TwZK00eVY+\\\\\\\/d6YNAy0bn\\\\\\\/4hZLmjsGvMgZM2ukLDCF0mMlk4sbnpXBBGInGBLX35JCRS9c8M+Q2+eIq9FNKR8qFooKQZR4AcM1F4sOdJJUGBEfEL1Dd8Pd0wYFb\\\\\\\/wbjfPiQAWoSMiy+kkr6Y8cOXlYb98DAJo4nA6eQOwwqBEt1PazduJ2W8qtowkWTQLCQxd5\\\\\\\/hN8LhlR6Ou0a1IJjaE6Xyq9\\\\\\\/KQ3Ca1uIH\\\\\\\/TEtQCIVpJD0USGmdp8+A9ajFxPaXBBEZMkNaIZT5JHoCYazATGE6SH\\\\\\\/It8qeKyqSJVx8+cW3VtGcb9A7cbg4ibJx7Q+gr655Xw3Gmv1MxJfZFD7Y4QAWsSqy3uHvG4kxOor3U2mOvM\\\\\\\/kKBIhEOzvbVGV3ZbphXNiaw9Fph8bbC4CIlud+Vs\\\\\\\/8H8WHDE7n83n1Yh\\\\\\\/WAGxytZy2SDgQcIVJmJxqPe4nYwM+3FjvD\\\\\\\/IRdKTRGClTJf8m90EKrRhq1MPZDapgfRXn+tSfa8dDl4ozzulnAh5FBbM3gQaRVSuUe3gzJwXASC0NW7p6u\\\\\\\/J60zHZcHvRtDzRl6r5g3r4Uz8OOrolYpwC74dYAKU1QBH2XVyF1HqX7+lbuqtdCJpmMwh4ulLAMxut5MbJ2qlZhNFCZZH2PeXQJYuo7f9A\\\\\\\/dygXteo2+0IX3X5yY1dWbYQl2Zl24gJLqLF\\\\\\\/97BI2q2FA2ungOlRYU1rd5G5J213JG7IF6lnFeL5goxb93IYo\\\u0022,\\\u0022iv\\\u0022:\\\u0022FbRXtoUPpna\\\\\\\/lRzYw7vunQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002209d2a6a48d98367db2b9b81abffebc079b1f0fd41090e6bc9358c3c39f92059f\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022\\\\\\\/6exbO5p+LWOPUXq61+iU0yZ4NoRrz6P4PqfyuVVJ4SlSauSdWNkT2NUvd5BudiX5+DD7DgTg5e0I7R8ywxcY9aKYubZo4x0\\\\\\\/RjrtUE73cAcnhvzCUZUTP3WZtjFqGPrvomioa0ZbH+eCJ3oebaXPf6E7eYEHVyXhpS10doiAit3bSN3pfpzINYHyVjWNssbzC8AJZQW3lMKB1Uerit0+8RfiVbj5q5zIl3cB7Ur+XPwzWeUN7as6ce5\\\\\\\/+fVffCpxb9T23P9F13fZy9B9XoqKXO0Pmfkjdk3L3DixRrag7ryYbxvOE3HdHw2YuoK\\\\\\\/OrIRgFRyP4eMizYQGov\\\\\\\/QXi+srun91dAbCTi7za8l7GPfhVsZvFIcP9iblUrfqQOyAiCLUQGY3ECKzeEBY\\\\\\\/jvha0xslWuI43eImUr5TTAElKVY\\\\\\\/BmhdHJgLBcMHASGJ2ccZRPQ+OSBf2ZQ7nvo1uIujgvedmRjLFIzdtru0pEifdrhHHZ3sI66Qg4fAaHbDuH6UU+drayN5Zjx8ywJavViH91V9IosOQbFOLcAxUqQ3xJAlJ0hEVFIUu9HnlIH9XsT2tWMzC20Zh95vjd2hTTiJTslSzsp06WAT3LLhlolH5yrwu5km+uA9tV+dfgz9CNREpCWbTQttb3ynOHmbXwqmaVYELPLO4JVZnLvrZdNXWF4zJIP+qgK4uY4HpvLPP2xThFUnK3z6T37\\\\\\\/hMUxblVMhzvshtKLn+CZylnu6V2QZsRiG1hf7AMduJ2C4Zw9iB7T9XQ\\\\\\\/tf97vYmrzvMWTaUlLkRBDO670sDAF7QsM1GxoBh9IDmuiXn5\\\\\\\/pvbSeJktf+4DLhaM3PvF17RVvKz+IMRIogFHIu7ZkozPZGPlHxR6tqmXH+frZzAofuE3xGKzu2sAmAm2L58PhqZO\\\\\\\/TjDRjRbaHc+lJrU\\\\\\\/UITjAMIra9iR2AUWw5PwHcWz2X8X1xiVBF3UeQZvCBEgusGprdYdj+d74KvnC0MqIGyP0seoMHOJdsVbu1q7nuKzWefJhW42FdVPak9Aqd6FJr67YlJ4RUycY649bAUzhUQPDrdvTs2Y2PuaPjWqS1jA+g7Fmg3iczm774gtIQoJ9b3C3IH8yRX6cJw1LZM8+1emu3guek06j0NM4DywLN3QkwnIr9O1vMvvYwTi\\\\\\\/fZ3lUYdQbC5cxesHPkrMl602BkfxFr9nJQ2Ol64zTChAKZlmZ0R2ngcM1NhOf\\\\\\\/EAUDg\\\\\\\/iq+l\\\\\\\/I3RqIiIa3hO1R16nTAZxHr2MLiRCjaqdQJt9S8ozmi6jabs5oZrph1PijWc7OgUnaOZszDiVdTQr\\\\\\\/eGBfDkRcZyHAN1IgUE6Euk7Q7Tt5QMbixQQoABvqXP2Xdnhyq\\\\\\\/hiJuMsYovoZdhw4hrknu8tzxhNIu8FV4MFs+SeqKyWcOqbaEkhOLTT1A\\\\\\\/8usRktCrGmL6F2rGW33mrrVDuTEharh2+Zc+twF4014oiOI8mNLBsatirRpnv4oXNLmpj6sg5bDdx0cpxrxQKdvv1IO8rgcim3LjcLYJTrGNuCjbQP9yeHhXYBdRYqDlocCv+kivKs68qhiFPKaOofCzydqyoJ7zF9PfTdr1H2AuKej1nA77Hs2iphWnQh0N\\\\\\\/zGtu0JM\\\\\\\/GVYDrZi2AIxX2hS1PbxQGGoDL4\\\\\\\/xvVC49W7kFHlmsO1tQw7ccjTIFbcjryOUEa6Pw2rDMl8dXuAC4+bT62UZHiifRWXHS7wAap0aXQZdw8IDBWH+z4Ip3BB798ibtfyk9qMvintyVsweEidkJ\\\\\\\/ZSf5PUf9Q3i1OqyyPrHiuBsmLTFBZKiq4vx4\\\\\\\/1z4UJk2bbvlqloIaqjY5toIh0YbOsEPVcnQfh7aAizJVuGzslNKiFtmEzKO4TMQ4iepIsFy9Vtv+lzDlRTCNhLDERq5iGn67AjUdJxS7IIfUGiaX8mqHMgXoyJitBl2ZYdJ0btutIWWdZcrBA6Lcq\\\\\\\/ksGPBrLYbP5LkluqcrcGQue0HnEeMqpk5si6VUksrgau+WeontbVnxN8URtHvq+3Aq00qMrgCSFIqSSeQzRjt\\\\\\\/Cddan2abPw\\\\\\\/qxeZ0eYDqmFWPfRbH9rlEpivkAWD\\\\\\\/7kZNWTU7b77VPqIZZd9Nm9n84O\\\\\\\/O+1EDsEPjDwCytF359FeDELFI66bBhsSzuM2Mw03KV1XY+GoM0MRRTuO4Yi0UBW8qRHhIzCwltLuWPqdf2kVKPm7vMpBcbXbHwY+36Cke8JT\\\\\\\/0I412B6iA7R9JdImB7Rkadn+s\\\\\\\/mRahe6NsnahQqnlUbKaIxecxsUIPmd7j+9jp+3W+61CRFiuJwmd6ljFyE8tHtTsdft\\\\\\\/Q+ZAikW+q0ixDC3rWKHzq0c68Omk6RDxAr6h\\\\\\\/TJaQm8HFUgeAWbK21Z3jwcTjZU4R2sEEekBG3F+U+\\\\\\\/+EUrCaIo\\\\\\\/Y7lPK5VMiCS7EGBsytJg3uxtsy8lHyoI98TYw67LHkQ1mNOQmCoCdOsSH3S8wPUgCcgOK7YucFdb\\\\\\\/fRGZb9luX0iEId+d1JI1l4pASNd4\\\\\\\/ICrjzr7VAaXiSTv3mxHwN7T6rUGZQBHJbBBxnDVZ5wOe4S4xlwJ3AtYmRLeE1mHuhnfdgSpwrD3qihI6jZydDzJ3LS0xpkli7Kvv8TDlrcDNYsN6yL1IPGpFjIloybyQK1aieNe+gezuap2D11JgGiaRj\\\\\\\/ZQ3nlq1nHtsKeA2xUDA8fU4KQjz2jss33Alo\\\\\\\/ixEm1H45Imj1q\\\\\\\/wxJelDCK+C6TvJ3VwYZ2i6NyOzFxtw6+EaIZCfl\\\\\\\/NVVpQJux7kCkx1gj0nKHNk+yRbCzaZ5LuQNCh4\\\\\\\/mfFzcMUBPuAfqvvRg+kz+D2zodg4qiGHTeyZ9mLXOukbiEU2YfC3dMzJ5LcQVNv4xjox\\\\\\\/9A3M\\\\\\\/OjZW2DDMgnAS5wdetk6txf5KBh8iudJF5eSemn\\\\\\\/jOWQWXjX9MKdvtBShvambDW3B0BFvOibkjL+8Cj9+l0AgwZbkFmlYfHT+VlrQkLyAvEHXqnuN+m\\\\\\\/WSEhqhvMT8LXvA2tdtMWwFNJZgov5qBM8e4dxK+85ifa\\\\\\\/oxLUhT5uUHVHQ0EJZ8md+TDNDlbF+1xW+4JaXjA+oLRNxwRPBCYsopzY4Tfb4yxUK3zhTtMCGCap0ZPjMlCVO6qMvFxnIxk7j5DD\\\\\\\/N7sh7sm5MgalvaBOhPf0QJMF7nNV+vGKRfRBWCV5sv5IFDLj7NcM9a4DPOXtR6b9dm0nh\\\\\\\/EgOMa5exPg6S1WDfdEArn7uw2bAXCRd9UhaSXCmjCk\\\\\\\/QxYPQm0D2V3xmvEUwVSPpyECskQl4hhrBjP98aEuOaJtQ1gG4j\\\\\\\/U8tDkPT\\\\\\\/XAXUh3ea6k6Qz8p2iL8B22g4FT7tRIEpiDdQ\\\\\\\/e\\\\\\\/a6RF\\\\\\\/5c4HeQwDXsyJ\\\\\\\/LvELqCR+At+zr465C7OrEjg4MRfALnACgJ2iUX8k6zJro8VaDxR+cGFsssoOCHtY6GlSmtaHCsV\\\\\\\/Kfwppnq8Jk3DFCYPcgYjcyQK\\\\\\\/T53ANBN8MQpsFL6eYx4kyia8bnGJXMrL2TyE9sv\\\\\\\/4KWY4EdwgZMeP+eHMFmbDDE8fxnWeF2Mp63YEzXNT7vrcUfjHluBTdiozKpSgxpTWIeqzRNh+P\\\\\\\/9XF2n3LSsUgVcdNd9dL9qYvZmdg7v9Z3gS45YAKgHMWQClPjNNpOLjPt8p2WlvMI9IGUyJb3gr8TpnGmRwjs+FmS4vx0MlNuv8+WbDbfzcRRptaXm5TiR0fTNcTCHtiSjP\\\\\\\/06PUi+Rb1VZxKa1VfDUr9DcRjVSVmeAiZYh1FAb\\\\\\\/OBBy+lCKCVxovMcLLRqTenJyUhwOD5Z\\\\\\\/Fg72xNfRb7g0IFmkrxH13nbM6yZn6WatoX+YJx1QDKdlTMslkPSepWn3veUvtSNyClfmY\\\\\\\/B1j7dcmUuf77Mq+NBXXLLJETSheYznnH43Gl7QtXIv4SRYzVRsTeDfi9i\\\\\\\/N44R6tE+5MMDVQSC4YK6ho79Z5tQebfsuK63x6I2o+tQu2hn+iGt8q9Z08QkH7PvmdPLwXwfNPi0z2czfF128ZruxdA1LPDlVJnxqWbJ\\\\\\\/jSoSx\\\\\\\/odeGg0FNr7eJyhn7qVPpQgmaRJgstkt14cUIaywI3u9Tn8owvUkgwmeogzt1RaF+An3F5GFakkATEuzeRcUFXlGyatLgYVHsADBDvAldT16o\\\\\\\/2exdOB5AkWNygUx1HduEsTFPz+tpIiCUlO23djkGOGFtaOrBRA4pr6iESYxeknzOaIg177yQ3IYeJer0oQx1nFvS8oOnu6mD5ZvJQDWbzYs301rYYbZT1UcIqy5nBgdVKuL+eoCnHseXhROrs3gT\\\\\\\/LkE7Tsi3VK8I8uMtzewCzp08X8ehtt6unAm20UFMv8fG+xCnHhtXYKxXTxs\\\\\\\/o7xl4UySS2Tednn1OtRQrRlg1OdgCxOtpU5J9Pcmi+aJBUlKKcKAek1X8kQbWXkoH3v2DUB9ZxkskmQIvgc\\\\\\\/BBD79Zfgx4FR1n2RGIhhhFALSFBQ6zKCoNH8mpzvrJvinsenkFqDO5mFXiZ4jKv06Kou7tfCuLcRXQk+WOAHpl6qJeHEFO0cJvdFAWWyJAfL98lNiuibbz\\\\\\\/GgL7ZPia\\\\\\\/TO0CCgllrWsP0WsIICJi2CW3oXDCokTko1XDe9+eS7u2wUVU3xsozvAZJHpv02HC+Ey2ZCcVdsj8cwopKzfFr4oyjgVadh\\\\\\\/OcDmIZ3BSoT4alDsBvglPjWudmdoeqHiM2dH0y+AIDENAxClM4+APDy7px0WEX4KgDHcSgvBki\\\\\\\/1tCqQ9qMemDsANXoyFfB2+dOdEHxBqCU3j2sEIaLGXg5cnZhMw0ZAG+An+a8Kn8+wRJZ0oGvtj6Aky95N1W3Vr02ezDsnpaOsnWnI3\\\\\\\/+UGId0bhuHTBkpiUhgfoKE\\\\\\\/YqjzLcuaIRD4Blu45\\\\\\\/7IwNAHuCiVlZ\\\\\\\/Bp0l68n2QAmM49FRs\\\\\\\/NiIv\\\\\\\/3IKTKI6MDPP5KkIuMAjF6UAgWqUDLr5hDkPQc+lcL61MMg5jxR3AQEDz+u2pXVn7RnKkldaApE6BUUkoM1i8admc9xT+MoQCElH2BXdPVle+B0iyzqBLg3\\\\\\\/dskevnreGIyb1+F6qyQXlfIw3wbzBoUF1rtjSA88rkidB\\\\\\\/yDAD3+eIYgvpBnRA8nzEhwQIJidIoj+t+ehAfk0v2kx0RXtm9SBelTzieSRSIzUNGzjM97UQrUa\\\\\\\/3C5y4knG4AY73\\\\\\\/HUjpll0fjY7WAd09nQzQEqKM3EzhK38ncW0xTFrcUpwVjMHGnF8gWrMq\\\\\\\/52a0anVGN+7qfSdAX78Y3q4Oyf3uy3ZxOlefqQRFHV65ih5BKeIGV4rMmWLXhbH4WKOk8TBsH+WNQO6LBGhhtMNy+CZiiJNbg96CPTodvWpsiR42+61O0fIU0yI6\\\\\\\/ZhovgtMTAqVoqzo9YnPVIB3ns1ewQcI3NwL2zlER9HHYkU7arELYtmcl5XBBe1OqubOZNRVaFCcaNlzxiMmqBGDKf1jXsRhZpUKjTQf0YSSx+26UljfJAFW2WJrg7UdynlkNtvRDD1ZeNo7hgY0X2ffs71lzX+EQxGjTFr6N2OHBHSmH2m\\\\\\\/ta3R2eOEX8i0ifLc3U9V71tqhZqf1vi2ZFeGC91fk+h3l1KHqAr9NUTFSc6+NhzkHqZKh2PsU6cbl1RvBSoIRgIytDPTaIPdeGnwm\\\\\\\/Ow390TYtcN7ap9q1Y3P8WnXdgdZJhK87HG0Aeoi9VE0V99PR6rx\\\\\\\/SGmqjhWL7sOqYqkV\\\\\\\/jk2Jv+kKZgQD2pedCSPxxGQduP1XFQ5knE27tQH5IANbiRmuS\\\\\\\/W3BZoqTVM15TFXQ0XUn80LPYYLGKhE9eXCulD7a2CkCOCIgmwtTKB3yK3X\\\\\\\/BARui71IAM6bLIS710lMuGjS6Wgg6ZBPGsK1WtvnBF45BrEorZl41RVgZwqvZ1HqcOoWS4A1iGtrR\\\\\\\/O9ykNEBb9Dd6J0484IaV\\\\\\\/QKBSfgVlS860L9hQRDtkYRBCa3ITsA4QX4OYkxoIFsA\\\\\\\/WWEtCm46MzKKFdsK4LyJqeDtccCvP+Q3dU78xqtVIFzoTqQMGVdnbbpml\\\\\\\/xbkrPGmxIlghClg3sigsXunCCZBcGnLM71DKC4sDIUI6+GaQfd\\\\\\\/LWgvOQxqc8SgsmS9mtJd+FWc0+iWH5Vlh3H+SCKgcjf2sRlvVAFIGWHjhjfkzmB880X821DRBOitrb3rKi0eicESg7iQZEpzPwIvK8LmmPMeOEGj8pOhIWAB8Q1bbKXasqfvGwIVntehZtpQXfBOlc8Xxbc22l6qqgSPzc0k7xPsnTTMP+OjQuaClfy8ERHtj593r381ueWDqQP6PKbYXuI8iPx3IdAWau34cUMjZ566SzRfojD+hTwTNA07ziMEs3Kv6jaFYMIjjUbQK+RGV3wzOJMume2wTFeKwch\\\\\\\/LCy6Q0QCmnU1cGo8mRXTAsRRHvBJhmmL470l96TFSIlibkS\\\\\\\/0mVH+Pj8u+asqwELVtuHUFNopY8HdFRu60mVPET+N8zGnwvABKUzJ9UlL5o6TmurNTEXIoFn35uVAb8zDaRDPGG1fRCZjPWGImqugxc4XG2fd\\\\\\\/KqMPNCL+nbHMuJAghfOJ4KLfDzmGpOZJD5\\\\\\\/MydrBC4yLEeVMDJNf7kvPmf2gvniLVhLdKzdPDNFTPDagzN24PYDydYAczwOHf+Z6J8xd2d6DPhfPtauCJXQUni\\\\\\\/UIPRCRGfvut3gItYzpk0C2JCPxRPiHlAuO\\\\\\\/7rZfhZgRuFFMPStDsET8DfVtCBPIVJmp6QA0WxdfdoOfDJ7K4cFVu7VxNbvrsW5dyiKEB6MtYkwlb\\\\\\\/Bc1p\\\\\\\/DrCD2ncyF1a3+UXo5fpFXRT8S0ShUYfvNYzOWJG49g5Wn3nI+EOxztUD8ly6c1t1j4QEeMFGVVJrSfh2HjfFDSYKOeQkCgsjleTyebMft6r0sH9ZKwEbA\\\\\\\/QNCfGo\\\\\\\/4KmRsw9nlxRXJu+K+bVNMrAcqUp0nzZ1dBVn+yXU5m2CSlgWgwAohr1nEkFzZW9hhbGsj5UPO4bvGzvWDdsDYLPtUXppk+8nrac\\\\\\\/Q1oCREEws1Uhh5PPSGrlkNBo8XjeMo8GB7BrZXO\\\\\\\/8s1I3yYvtQznJfY9AS51qNLFK2BFh5ZSJGIXLEX1lVkFEp5\\\\\\\/7ZJzZOEC9WqRGObRJtsgDJvyNxYIFSCEqECULNtbJfz0bDXJ8UgnDZLCn92eEgPVoMr3uhbDFZhO8paFFb9e2M67FLelmvz8WHAdcM0ghML4CHmHcyKimwN1zXcD8EnYOItjG0pbujt2hbamctiyOaeE0X2jIeV80BWSVFEORKlnpeoh\\\\\\\/n5p6d9elzb1HKPq11jN8Z7HIW0hPO2g\\\\\\\/irBH8FLVuS2GOisdqaajn7zwMOcLfBuysgOz8nHoempx4qu2oznNsIUZPF80BeOgO2fces9iAGjvkKEE2fQf6DhE6rQj0eeepQoWAMpSkWwQcrRSYIQBaVzkZ2xSht+AdWzEY7md6P13eZxolBogzcrpL9HYyzH\\\\\\\/WVkVJferxCq0wrW3KSkklK6GoqAWFO0OBkXGfPmnImYjST5klGzrkM3qtwvC1dzWt6FfR8O4U8gUMeQ3oath\\\\\\\/2Byaf33JqaR24KqC0yk\\\\\\\/TOXslUpZp5rMwKQYsOCxrOnYdG7UncleUnBxXOLJcwPsbpY+oD3lKyQrfVefB24fmMST30MB7nUCwVeXd4uN7LMgUFbM7W\\\\\\\/d5jXGLIGLLwWt4z7DqDFZvaW+0WKQaCnCCXFVp7UXFdsQ9klmmRncm6xctnFJS8\\\\\\\/LtRWGCtbP4zgP9mZbavijExtsaEEgqvlpzufOB7bC5ZyL2S60PNElnu7XLg9OR50RHUsPTQlQkAipV59\\\\\\\/QE86w+IRE6JeUcfjeJEcfMBUlUc9PjFwahr+T5nbvFRje1JLMlpLjkZHNC+Y2usXalXlVa5lbmIOovR549Kq0TQB\\\\\\\/F\\\\\\\/Tzl7dsi1udWfbZ5nHUfKSoCVN10E3OMevjRGb1KwoDh0bagTxkkdeK0xgHPbwjUANALLDFQxdfWKiNAnjyUFnCNm2d7pL+O0vYOaQ\\\\\\\/2W8VUV657tyaPAYzN5GVTACEZH8Z3HpIC3QJclIyvLJNGdgOMrcnNpCOqeySc\\\\\\\/5RlOgLdNrHEZ+1CwsHreD7Bu9pfndhEAifYP88RYhxU6T53rLk1VN4Dp7iAjz9IZqna1mD6vfzEyeEtoItcmZKR\\\\\\\/GY0LJh1OCZqu\\\\\\\/TJcgWwWGklDBHJRRNdS3gSpN15QCOnUZYT7D\\\\\\\/gu7ONqoILqgQZCS3gLcBb4err19lbKSECOmSs7D60Q1RcS0Z1eZjhE\\\\\\\/kIPJsXbhN2kuA6TjZy4vg18uwKi\\\\\\\/6tD6zb151Iss96v89\\\\\\\/mAKytqrF0YYXIkf402X3cZV4zH+QCqrTpTEYCVu95ubS\\\\\\\/JKaWHsRNcj6xovIjWoVMrg8X+MVwv4ihqLPrNPPURNxb\\\\\\\/wFss\\\\\\\/AqcIFPNtFFbH6p0Gk+sBNW1dokMTyLKlheIB3BDlyHAoWKAN42YMCy1M3XPbSiV+oFxLV4B9SIBTSxmPn6nUQbAlaBn6xqaMlrUcqdv4IRFWTFNHhU8w2o5mjH36W6y2dDjDW7Y0HxQU6CjnqLwzrXoxsp1gdb0hlIjRjrk7BcZ1MqgNdpTSabkQLLGKEr90vJquWV4sw+GtUSS7qEnCo\\\\\\\/NeILBP81j0VR4q8kLGvhifj8FxvwlG2ZCx4tHpmIP+xpkQHFDyFBcOThGM1tmwN5EeoIy99uz8KBtzOW150Na7ql7g0++IvPoFSq99PJGNoUfU7giZ+05dt32fs8QxYeLDui+CEF3uT5A9d4IZM\\\\\\\/C3GHbTIArdxUdM8sJprPqNTQZg5jcAW5gVSkXnF+JpfRXCWgol9CI\\\\\\\/epq7wcDFm2MJ+9f5OIEGDaWKsplVkioLeU2H04HanZVyPg5YwhkNuX4hhEeCJpY+HbDn\\\\\\\/1p+La6HHt0mzdTiWAdRr15rTPaKBRbCm8Tdn1m1OThl2kcef9lk3Qp9oGv6vJW\\\\\\\/Z5oNDbKiBLRGOmB+mSTfxBzAyQJ3xFs7mcmu+dkj2BvFlzOv4TV\\\\\\\/KbElHBw\\\\\\\/tE1hiNnnqkZYV+x3ONWcrgiMUkUBJ1YD2AtGSRERzdaKdpoJ9+oVleOiXwcCeSXiwTucIIyfBaYpFH2ZIjg0\\\\\\\/eSrvKBZySukrfSglbw0xTkrODU655ewp61buoPularEBGdZ\\\\\\\/GWVxGUeGX5pE6w\\\\\\\/v7CaBKwiW2x4Gjm0KXb4bW8YI9TjqKk07euRYFb94cgHvOF9JbhUwDQtBLBdBlTMqp995woSdxhYpwmoAwnF7GUinpqB17xtAdm9a3DpkHmhN1nf34dh8sE0bvuQeqkAYL+oYDQwHbZ3eZB4qiTtuFjRd63L1WgSKAVjsOA+5FPvLPU5sd+9HsR1g7+J7FPvwbxsqvF8+\\\\\\\/+g42bU2I2i2PpNAVNw4fJfHzCf4fNYmijzc\\\\\\\/U1UzqBDRmHLWXdB3\\\\\\\/lZ43dSYQ+8\\\\\\\/d7iIa6x674lYVGkj1s8kTgZgO2Z7ssrsJyjSY4qyoCivZYwHpjSyHsOcsaOgrNPnbPnfhyFz\\\\\\\/xPo8Cv8QFKQvn6UxB4y5zABYZYGHEV91qMdcI8vT09c1CpknB\\\\\\\/gIIQk9Np26FNiQuKCIfIGqx5nysOHPv13bav9zf7cGsWgje3JTI3Y9h4X58baqrVU19K+zBTKrCUD6DNZo0zRNTELt2LqWa0JlKwidKP2YZQeE37LqLOqPhxxnDuLfmGQqen60JSnr2saFeIb8cZMhXB+T4Lyzx2Si4E9o92AvjW6qEgJAuJ1nWx50OJ15HUdSZngrq9f4U+7EC9rZlYZJSROABnXmhk6vEUHqgMW7TgTJlj+vasPH72RmancKtZ\\\\\\\/eKIjVvrB\\\\\\\/5Y0z\\\\\\\/6plmsPgkK8NhhTQ7w28ztcPdsZlvaRvF19PHstMMEebf2wFIel09WDuyR\\\\\\\/PlfennIriZ12\\\\\\\/lVNskGIn2lCXy58tTBRHPu\\\\\\\/rCP+mfB87m24jrayr6ldBv+hUACr2tjneiBBQN4ZHdiJG5gacfviwhniVrRPUU6gjH4FLKy6eBNLAaiv7Y8zgcJ22ln8yQ2Iq7OI0UlbdB0VyemupUk8J0gAAXdAlNTYg6NQlaHy+o\\\\\\\/ovgXnb4ZsfQ9XVMa0j4Sl850Uq5Y6QBkPcl7hZCkIETd\\\\\\\/Pjxjq1eWvgV+sqln9fwwSSHY2WTWytJgKUc7gdPEP2r5lRdTB3oVr5mccWB8olVw22IDYp4NCceMquGTOlCFiw55YhYUOJ\\\\\\\/Ih\\\\\\\/cSDNRz57BR5w+D9EiBfAR9Rv1Y\\\\\\\/hjfSmlYiUpdyNfLhQ\\\\\\\/gjagwkwztV525cXEJ3Y0KunrWlWGpT7riL3wt999ZGaOPsfUNhxVZJ585cmoKeGD\\\\\\\/ReLk2WGEMw8jGaaUTx+xhwdI8+dgiST5GADLaiMb31J\\\\\\\/drP0JOQYnv6GJpjzg\\\\\\\/3eDO\\\\\\\/Km5uk2rEriKNvjL1lV0jj91zhnX4VaHhhbAPe36rwnf2ZGK8EF0dPJFB80jRPBDMuFPxIG801Kk4OFbPQsLnBa9gWJ6xIUkWuUwi\\\\\\\/cPSoPrU+O4iiwb1gKBv+PUhlzr8scmE5dNIj40ecZGZtMZcLTr4UEvOCR5iY+RcPZsftw0WyMYOimWe6PEC\\\\\\\/PdGncdXquHa5dtlNmbxB3Wuj57a387vAcU8+zjxB+ff3CdplECrLpNwT8ZqbkjAY67vqWDAP1u6CER3zEUxZEndlZ4Jh2piOkgbdrIJaJsaeAhIk4vaakkzOuGeaqiSWUm+SC2J6w7mT9rldQxcrbG1R5N7T7h6DNXM1AyV80wtWKfjjv3NbFF23IVmFO7Mbei0XzMN1+p3ZnyKa2e15lyaETQ89fqWuRvCGXxFldDK09ObO4pOSfpziCFfBhL7j08uc0sLUy6q\\\\\\\/bYdLy9cYWKaVxDzc9b3ogpg7Lst2PwPXMM5ohkkI1qHQWMhAY4PVlMO1M0VE2jq50ReJLKNb6aWwBlqaXQqwzIqoN\\\\\\\/uA8RlA\\\\\\\/8\\\\\\\/6CSyukF\\\\\\\/2o03czTHO54FTLkLSiFQiAqPr8QOMQDhaeA31AzJ5ze6l6hw4Mz0B3MZMnNmx02xxpC0596bi85xSVtSuaXJlNlDkmbfpk6amnnsFhH\\\\\\\/vXWNIoI2E7GWhC+enwIdSh0D5\\\\\\\/5CWDw+RgUiVH0KNsiIH6mHHjee3P6BD5WadcoRC8ak96hfJla6n9yp+gle\\\\\\\/LspnAAYxaE+NztCzq9DzE3IGu3OnY76duyEn0QiQPgY7HWd6nz6hLVycWu7CULDGqODOu2uhDIT4syFwXF030LQ\\\\\\\/LkZ7XGYqeSnVXMai4oDeUk8XWTeCb3pMA82Jwq0DdlUm0ThimMTdFCZBeNykTkh\\\\\\\/yfGko4UGqUgRSQ83bDHPXT51DMf67YFWXxMqq29ZyGq7tT\\\\\\\/FhgxqgmFXV2qHHcYKBep9kRsSHj2a\\\\\\\/N6i6otYbe6liPxa65Z4hIZS672+aqm91rfNJdN8ZjEBNSO8xMep+loVj4WLUUXreIk4TqPWagRi\\\\\\\/dTPtVlmjCNrkWO9HKWh+QZedLMntChda4KGFL4Ltq5mq0lLFn5\\\\\\\/CCY5aWPdyIfrvs3vCy2QVaLip99fhMzeqasEFYQI4HOeyr\\\\\\\/8Vr5Bdt81YKQ483p0aK6cS4EFaH3HxM4J7J\\\\\\\/kcR38trrm6ryjJNch4vQOhNqJ8+M7Itujwx77jz1jVQy\\\\\\\/HH\\\\\\\/chotQMfREEZdbD7xkMxa2ZZPFC5SY35QaAsyWxdtDMAhSBImQY3mYlouAMWK+VHto4uYIY14f86lU5u9SfKbI69pRTiacCPaEIndtkUiijrMwKuGMGUWfq9VeOX9KUJdwBQZt23PqeODYhqOQeEkjzz9YoXsLX2WVc\\\\\\\/yzAQpmdxafLck\\\\\\\/aitQ47H1PzUyGwG6czETXwXLosVzIJEmi2wGRAOFMp2Dda+re4Wx+013ZBBfoYYbMcs08y8\\\\\\\/Y13Z3519Fjr0glZ0B70UBS3SWB9UccC73q7s1Sv9ix7JXabkmXNT0qwvLUu8SzRYwAkkr3s2MVAPXQwa7h+0DvsvIRuPV6HhzNoFjngEnjCqxn3u4pPrd2oQUc7LYE\\\\\\\/Q1FR3exvIgg62ySJpi7YNzTo5Z1vyDtepp+lUxc4ZtTVVehr0JZMi2OnTs7C9v+KblNJmXJhCiHrLn1GNahgHXK9eHdg8QA42ZVqxpokiMVgEqS8eQ4oilJOOIV781\\\\\\\/hUpKkVlJJkltD2QoTtEjDCKXImjqzTp1apDTdQomCNMfvJjO3cMFPXUWUNL7X3Y41Gh2Q+SiZmy7\\\\\\\/vV5i73mfep+cczQxGkYanuHP95oTdYcd+EBeOyNS2XNYRDQriywC4lMI5iZWrAFUomCdlAssl0DaX9DcD0XzMyUbAU1KImYiQ1D+GHZ4nKYsci3new3fqv2TdNKglzVbLFfMKdrFS2Ljvw8bb2i9XoXt4\\\\\\\/oWMa8UVxVVghMbs6KVf0dbkQ9gAqX+asoQhXP+wvBZOtijLToH1zQROwhAk9OsV66eRtB2WvCbL+bvTyAZ8gIIZKqgPhILIfc0hX1RkLIshseyyRlPEzAIs7T2RnhpDbWFGFIkOGM+C9wioeEq6ONpZQvQcEOOKRZ0MReDHKo0LuouG6D5yWiN8DPJJAqO0LagbtuPnT8J1zImPIDYDMRvVJQW0+Hfy4uIjdwFQPwDW7pdiE5aqD5XbN1dJSTRRbaQO0tKxtmaliWc8n3SdWY2yHMZh95S4HzefJ59TfhyaRm7bAhaQmlp8mNVASq6KH\\\\\\\/KO3p54+U+InEw7O+BxOXHRbSkogLyAqalcRoouHD7npC0lewPLsS7VCwgq9oyJedQLOITHIk+p7DF4\\\\\\\/65hpiA3bigBOqCCmA+U5SoUUsHiKwFTYIMK7qdhcktT38rtDi9cCg+E9oxtH0F\\\\\\\/vW46KQJPTWgRkiUQSQit7RmZhit9UTse7VqTmmhKeDfaebTtVEmve4enoFu+SZ7OO0QF4nQGipdNgWbMvjO+ZasHUYYlnUpaX4uxuNqkKARpFImH1pp9kwXkHE3c\\\\\\\/S+qQkODZZARHBlaiYYLNcRYr6xzlqOvmMpjiFj6YoBF70gKzvLUFQzAm7HPX3SS0rh4C4WfrWi6U894suBUmsl1jNkCkG6iL1BbKk7MUoc6+mgNE1LkiD01PrDEMD6cADnYHFZYh172QIctNt\\\\\\\/ErrAfLqT\\\\\\\/BiyRzdsUVuJoFn4TLzA0bAl7LNZuw2fjZL1RsWUVo\\\\\\\/FKBrcwyFzFGF6fZhXzgyYGRg8CYSTKMoopW8eLkjqv9ohUS7gAhE2oO70oFVFWWZTKd74TUW6pFQpNbCCTg8Yfnql0b6Pu0CK1MF+NYukocuh36HfcY3j8UcFIa2XAVVpCwO35A3QPP4NGBRDY5R7farjG31sI\\\\\\\/R61ZnAhjhraCWjAcaL+LBNhkGnpeEbxpSGiY6h3BrfzmTud0U1F6ukaVKkKrPRmDzRbKTFTnB1uMYcnXndJOX9EtzFa5aPGXmGeSt94Sq76xuwOQhpLny5MsSfWBTM4I0tBgPe8C3jaM9DKHcHItkPOCSKfcaC3bzMMl+a8SD16FJqZxCfx0caKk+C01rNTk397OrLNS8VVf9LVV3yHef0C2u+CCAG6CHXW9BgpxjtSTHGMe\\\\\\\/tvwh5EGS8DDWtGZjUHdtxqfVv6mRlfcjuAP\\\\\\\/\\\\\\\/L7t8oaXKPzeQwXHvH2iRK6MqUaU5khIth+9Rh+YHeaxuS1MGoVztTNjg6ZuzWXOBnvgxqz2Ca1Zp5msOllKDHmuvGqYXRFFyy75WBZvYuqMZ73OL79VVs8ErCRl97UpSiuhM3safs6+Lf4QRgYeROheS4VcacbTc5ke3q4Y3QzRahVdvA4+Rg0uppjRiw+3csJ9cchqIJpDZhRSw9dOf4Akp5HbJB50ivpPRnuKx8D\\\\\\\/jl4PZoAnkssH9JiwL+V4yI3T8Hi5VRR97oSAoIQjNoUL+a\\\\\\\/L+phBQHoZ1vxKjGkDUhIkXP\\\\\\\/MY00VFVpgYZhHxmVICe6YSQjCkIhHneX999J68HBvdERdLa9FLWSJ4iybeoLVuCTuzj0\\\\\\\/MUDDA5SjQcXAJ1qwR\\\\\\\/G2iAVkyD6xCR1wNwNhieaSCDGrGbc\\\\\\\/kvSX\\\\\\\/gR2pjkCwcc4o++ZOf8E1uHU7CzjGAbanTcKkKgMRIbnzR1SQ5ZNTvNbyPYmiOzT0L9eXzPjRYHAC1FGHhpd1DNoWhsM14jqVhJft8QEY3cbVv8dKPXRJpYpPuCzawdEvzS\\\\\\\/6ozhzDCWFl3rGVAoTPL2pS6abiKH\\\\\\\/z3CZgQTkiEqr1pnGrTV\\\\\\\/kTYc675p\\\\\\\/Gqna0jIC+FOIDfmVxfa4Ud0KfsjO42PnLHl9I53zrLQHXYxU2JN0sc41byBtUvSw3GhH+dmE+HSTqlwmnNLts3BKtIT\\\\\\\/xE79rZ9DMVDiEciKDchro4Lxrd3gSACoEuMcIOCekw4NcWXT3zi5qILfsE5f5KlgpWGiCX3ur33bhS5X9Fsoj73sI7bW+IOcHZgqIDqIUDrlbrgSvazGbxRMXh4GF6T8JPhdn9fXvSa\\\\\\\/\\\\\\\/tzv3N2zVZ7t\\\\\\\/58sxQgsPQrvLlw3S3bZ0XrdTVgaPBmIyUouoS\\\\\\\/jV78ZD4\\\\\\\/rAdTkgZms8HwOPiBZvjVmtG9FHFnF\\\\\\\/tCoY8nAavLKnpHJu6agsISrPO0EuwhQmejDpWD1t6jG9ahSDuZFNaiHVM9jEvW0aSc6NLP7b76PI3GX3Bg\\\\\\\/wm\\\\\\\/NPr1nwmvH5T3vbfdWhR5t+Dh2exXJkMUso3dKeB2I+jKlY02ZqKme0ddXfl2xNH8t4pTSsoqZ92Nop+7xXPXCkUTP8PkKcnmHkrM2tnT4OPIfRVt4GfnZ4amCWPoa22ZNocaPdCvy6i0RPR7ylzxqW9VcR1fBJLHeRYCPoNXUOBP9FOImFePWel9NbWlYlslpt0HEjSNKvZO68vIXYCd18gsJzMFnih+dNQlxnRaxJz8+KJdMgGYUVwYj\\\\\\\/1spJHEv2yLlyWl+4j2zQYpIPIOcs9KkLYLvU3vKORekDygwRZObESiDEXuGLwBVwB+X7L4xMZXm20Xy1s1Tdymxq3dsA6XHlhEA8boccDcMLturVyrTQvMkQyOSSYOyD1Pceb6aZHMu6y2Jfe2jsBUH5FRSwufacEr14P8xv6sMTxZl65ZsXKCrhEuBFCyaih6PN5BfLQdYnaWEGt9rnd7kJSZyWU\\\\\\\/iLkCpBko6dRv6LaLApnM7W8JCDxbWg1gswwQtfelJm9Z\\\\\\\/7BzArlPN3XCYnxwaMUaAJ16qSBecC64tks4\\\\\\\/vt7bUQGzWdZZdpZbQQ2k\\\\\\\/avt272sZgPvvtM03QF37y96a5dnRpEbUuwL09P4OY0X3bdvdkujM1DV1Y0eZrBe+3\\\\\\\/7tucOHCKW8VTQdu35c1yfTAhsy3S\\\\\\\/bqd0ZsLy3GyhnO0FJf0ypl5UpshnvGl+k69FnKbdNFeXOoCxEHcK5X1J\\\\\\\/fUtmwkmjAn31zK5RdJF861apvyh9Pwou2GylDJnhcCLEGlDLms6w5V\\\\\\\/orEPWZ5zt+ppvQL89UXBib1BT3q6jCTUcwlMZioIuK3FPJwubC3r+OTIUrulgwjIpm\\\\\\\/AbJDOmTX3WuXGvw2KB8mS6SljBUZon2T2aCc02hIUfE8Hxpa\\\\\\\/0HUflug5Z5n+gE0OsJZ2UbZF808qoN5SEZ1RZk1Acm3q4lDJ1rrOf6SR7xyPpS3l6F0q6yH1qFYiQNe\\\\\\\/UNpxYgWcEZBWkfkZQUjrTn9G91aH+C6CryYJGHW4kP\\\\\\\/PURXwY15v53ZIXeUPBXF1o1qZ5pU9T3xzLcXgJptS7Jyg2fZ1evLvlLD17cHeCDsO6\\\\\\\/ElkoJPaFZQKMJ5pvZ5LMwj5xtnKBWkuIQXyMiAY\\\\\\\/V7cf4qJ7BosBcXatIVCxyLKQgGsOcBXffZtzcUaGtHk0OmGGJ3ZUWkHrNE9u5KrvtiYEUrhl5i7631KaGKMP3IeFFkRj9Ikkycz4\\\\\\\/azXrjDanYU\\\\\\\/x5z5UNejQjtos94GYAgOK7tQxdlzW0h0f8rnY\\\\\\\/YNNMRIE708F7N83SBK3VxZCosbn5oDLxOf8KVF1ANor6+xWzQaSTiZGtAAVCcE9x2VaB\\\\\\\/aLWFb2Lki\\\\\\\/kRN5bafVot41VKG4mDpMidzcq5w9tMwAfrstDA5rYBuPAKaqksU4bcYioyrEnBkoC7GXrz\\\\\\\/s06t50rvbSJmIabHkrYu4XRP9yUlnfoskDTZonWbju46WJIbMWH8gG+ZArWhNKyP3Nx0tEh63d9vLkbZc3L9GYG7x9c\\\\\\\/bBMYaf0ojK9cqAMb0m9ypHj1bQFbfk5hOBPXx0yHgTyftuyeYjQR1QgYtVkYBCo9wOW81iVZJ9eIsYVy4yPPQzpkwe3LQwWHLm24fy1scX2irBDLqKFsoCdO7i5+lEf7wD9+EPVzUP1yh0h+FhKOdUTn7PWZVSOP1RCHR5bj35yOhebgyefB3i0TOxSELG5Q8BTsFfnbwhCsE5eXCNfbWvETXF2DP834pwzv6Pp3lxtUtyh968AMUgyZ41g8nnxwhFJZhdO2Id3aV0PjCRusMq9g1h2lfeJVTcm70TR6gQc1HHSa6rqH1m3LtTfQX+gjf1wnV9zCTRWFLs13p7\\\\\\\/iWwmpqAPH\\\\\\\/+CiX773ZD6eKpPPMUDOjeOFDnP3wMtsW8hneXjGPjJoxhHqvussH6CyFlLesjgywJuXoFp88DB07YV+6McJ1jnFumScZcUkTEvyYjbPrqgUENFrTJHVjsM6KkmA\\\\\\\/caFCMUOGAmO1DcxSqlzWjoYhAcVFc2crUSUr\\\\\\\/lBVuhIavhAX15UEjQV5QHxRZENgufj\\\\\\\/VTz2MFNXXeYLL1jPUPTJTJv8t0TfrsTqUfBamkCUqKQGx3EEGUzO3uTfQrNgikITVY7uXv1oz8TchbmC2b8Z7SrPa9blKvfSjmFxPricfdwYs5E54Y+hwzfrAllTaZC2ojJnpw9WwzTY68OCiZOcido74kzPY+ZBBgnPt1bbP7ZxVVAdMHJPI2AOJtbt6byhC2MCv0oFiKpQQvf3iykK\\\\\\\/G0kADlB7FynXmotW5S7v4xSRAORGyl\\\\\\\/48t+vXbcbXwXDy1TP6JLfLg8T+fFtrmnpZ11iQZWnchi\\\\\\\/jbpc\\\\\\\/NVtsI31e\\\\\\\/j1Sy7N3f6M4dphJe1+JnwSsTUdywd6n+yeRnGIeOYLoxpxitC4HAU1kfjnyEtj4K3\\\\\\\/UyQUuVa4p1cXRyLck6lQOVtBUdn2sEfAyFUHW5fb+YqgcA8QnvGm\\\\\\\/hYaN2ME1tn9ycdfgw0Tb\\\\\\\/Pe6a2U5Z\\\\\\\/IlXS1yv0llU1jjYUBqbD02wohvPMJTL3NmHKwJbJKdjP9Pd2bkRki9JxweG4L\\\\\\\/kot04N4MuE6xNjO6FMLY\\\\\\\/XSMm+2YN\\\\\\\/+daW27YzTdi9gugrqKAjrKOBTxfHWnkkkq+VCdEVEuEnfxuCYe9WH3ZZk2bazFOhofw1N3a5+uQYBpjkJTKKzyz9c3FWkMMGKOoGk1vaM3UKL7E6LQ1RzOREiyLyETgk6B2WctByZO7cVG613zUW6QKQ3fxymy5CcD+nC+gBJiUL6K884EnWpaDc5OZRippwMWVt9xD2QdGktOx9ca5ddkUt3NGZN+ZNa1KMRTPxQdkeCHGT4Bvv5\\\\\\\/epYc+9KTPPj6xHt42neadIA5iHrVU9xqFYzq63sLJ1jPoN6EbiTE9nhuyo+8CzOjC\\\\\\\/txBYAzFiRst9\\\\\\\/HshcbcEgWJlZeiRDZannCTYd\\\\\\\/IbTq9WaWBhGnZlaP4xK3pEWF65UdgUem57w\\\\\\\/VICEljkrVU+wxQZt4BijXAVlfZE2XrOhU+rkZi9iWenG3N+Bz1qJKKWd1olob5TMGEaro\\\\\\\/eHBKNdVAsvzK23\\\\\\\/T1E3lMJtO4VlpVxSdUzlaa15OeGUXH95lV05jyuXuOmQ8+Z6rFV5fmI+LSdjqEPjU57mIR4jxbY\\\\\\\/A75fJYCbEoJcJ442w3xSxqr7OeC2ZYMQL91le\\\\\\\/T\\\\\\\/MG5rWoEGiVw2TsLs4qokSqHrwkZXZ2oLP0CnCS8mvsGO5RMAUS2rH\\\\\\\/IZ697IPHv+Rq99tTfNqspafzW2EFJ3CQlp30VuvIzzKuR+pZYu8FwaDFEK1ZTMLvwKLhOv\\\\\\\/XHJeLC6XAu414sF6wZanjfgjFtG7M10ar7+CdMppa2VJZA7OFJunLxtNZCBSfGgSz5uTnJS4yfImHLSiawPNoBavUmzZb7N0CIZtHMs4pvh0\\\\\\\/UqFhMKaLQ2dAIRwoOiEYUat2M073hhUJPx3CAgNljysevxu56a0Nh\\\\\\\/PRdp84U9UhVgUDsPLuc2vPCAxYySvexF5S\\\\\\\/TWsvjbrmlfHbJ2NrS6wKItmCyJ7D+tVOIA51JbZ8krTuQe9+VFd8YHDlWuavx3bPCN8+NG3CJPB\\\\\\\/VsUjQW9OCuqO\\\\\\\/JQLsT1jlNxEHhIeA0PGY3Wd2ZMDL373tZ9H+HS3LIZZhDkEGqKaFdxhHRuILsbp8PcWSHfPhQUZ88gmdibAILrmv6QNk9XjtwitlYupxA+Rlgzl8sUOKOR+LdZYYHxv4jJjrUzDTeVrXApmJePpLxsHJkk+GDMhRviJNpzUYP\\\\\\\/FnSYumFL2sAQ4xdmmh2O7E0gq9XLbpyNPCgQ\\\\\\\/la5IhHf4FlOLqEf4NqOWtdgCJQ92jP3CrzV1Jwdu1lYmymdC9kVaQV4dKp90\\\\\\\/LabnltIp6Dd\\\\\\\/VWBGYMq3cuk1VDYeMM9xKIqf2H8IPIQCFYan8BRUbaIsSlO3LP\\\\\\\/rjxgPy4BjUyVmDofO9iONBxmXoMTUhrRAxvlKwXpd5yuZhNsXYjmDsLJ1VOTsWk8TirolTK9noJV4RrgWG5zpyJLq68OVu\\\\\\\/VfDSsKZ12gsppBrXgdGf+c8nfdmSDCHjbRFs2bLrnArTHsP4a83IH6W\\\\\\\/UWNlEpwXkAFpQAUjjLCqD1VbGWTAQx5ynnCrERRGITNU05yWOPEzEwqsQvVa8RY3lWCaJ72GR9GQOsqGIDt6UcMLkSrO5fEnT8Ad+a7AMau6HsEYJ4JSijYqiZG8IIeBZn5+6ALQOsuF9IcrMi3wNPRkdj69K2VE1ULwz4\\\\\\\/BY2MUS+glXYyUGg1o2dCaVYw1sFd6EKCP3sOjrAottFBLZf1lfM6WyGeSNk6Mj2gvTIceSCynnmb0Dw6VLKFQ9IF4pJevouXCEcysHz+spc8msauwqpF1uAUZoHkzemAW3k5LCQr0JFcjbI3gxDNTUWfwuDr5s+Fi7IXJDhQWJ9+G452GfNp07rIMSAz+qO2QqGLgxoy5kPfKIKcp7WK5GPeoBlQ0\\\\\\\/SCyEHBSwFvCCRYTEBLDwAfhBftWnXtoMr0xnUpOvIvOKKs7mjlIsp5bSQn3g3uA1ojdwVteCWf4S+oGF0wNRxtfWCmyZrBjRgp81qJS++XueiEdkDfBO09kqr0FutYPmeH9qQ4VB0z75EytbuX+aEMJ+q9L6fBg11kuE9K099BTliZPd6ZNwoWvOgRQcD\\\\\\\/YpKskYqh085mkOXosCHNJf0Va\\\\\\\/v4GviK4GDI3L8lu+AUdovaRZ2EvDb+gRHH\\\\\\\/XcCYycerojf9X2P1RueDIvuZph\\\\\\\/OOc5XWSdE+5qcHy8xbVJ2NWNELbE+uo8MPfoQAcIaDg0dY9aFtfjGQdpS6Rbe+toC3TZ7Tfq\\\\\\\/7MJNb9Yf5RS6Yg8axT2i+mKjImoyNFp69e2s3wbxM80PiHm0dmREVIltgMuflA9mR\\\\\\\/JodR\\\\\\\/VGodCtibncVWNqLIOaIoJvbPq6RoR21a54jY0gQPds0JJBBEpcmOJ6Dz4XeKPJ9RyqQF4SeBME8iYug05tS51bE1bvvDNW7biYnFjz2i+rHWxITApxxrPQ7fNyNAt1pL\\\\\\\/xn+V5YnlR0S7MyVAojMQQ0LcqOxHYsZPUZgf\\\\\\\/sLviOP4C+mksjFwDr8G3F4\\\\\\\/B\\\\\\\/bpftBgybxjUNU16N\\\\\\\/QYqiusYXwnW+kNMipckK8SJ10lETDkb2S6yMu1CmM\\\\\\\/GtNFhSK5fis6ttz+XfH1GDf\\\\\\\/jdtcovsLaEJmp40CTDUVr1GsdMZBoPC5+tKqFVxjEmKC0uobs\\\\\\\/HIWcH8\\\\\\\/e0khY2JwVkoYtsioorteT+K5a2uPKRjRc\\\\\\\/xFDf2GngWI4+njE3bJFMsrGYdEF3lVhV0EQ\\\\\\\/cBzZF6lJC9lWkw0gh9zVUuWO5fmQ1aGsfKOrBb7saME21QB0Fn35Q0NrOSU11qsegfpInQ5PverTQiFnF4PmxfL9GGIukrZQgQvz5O0hk7DAVlk\\\\\\\/93Khr0YLrbb+WDwgl8YT5axhbykIuHZEQ\\\\\\\/WA4GCVVILl+vq\\\\\\\/8sU39+qcvQsy15Reg\\\\\\\/y06\\\\\\\/T\\\\\\\/\\\\\\\/DTmYSAQEzT5iX9ors7h4UPBYcr7F3qJ\\\\\\\/U2bq7eYOfKh5OJNDOM2tycKAQSZFP3chH9W9SKEYB5wVtoSu4Z+xD\\\\\\\/OoGfKNf7pKJp5hfU1+H23qK6Pf0Tm9rcgeWBjKXgiKqSjEkPy4ZbKljcZ0paXfncHB+rLVlNkFh5rGb0hiY\\\\\\\/isN2apXEn0ZsLBfPlOrA1Ow4EG9mg1wqUwI4MmDljVTqsLC1zSHfFV45U5LuAeB4YO9DqYq0L8FtSIL5fU5lzBYWLZGUIhnxNkKCyn66Rw5yo2WRSO7NHu1PDQkCw9FvrFXmx9fY+X\\\\\\\/njG66yFTZqKyzdgDBLEONZM2HxJwE1lOwX3K4Er0C90N20R2cqA24ElF8YVX2h30g9tRGIlWGechCI9R4oBCuCC12yMujUfTEIUiBdqIC3f8gcmY6ZnUOBGuvLVDBOkg7EkO5xmv+nggeNGaQCn\\\\\\\/dFNTEMKXLo2nqRoUHwTRn3J5t6WabOdPecmcsVDpnh6mj\\\\\\\/z92WUv8AmhJt9Kw4EEYCUI5JwdBD2u5hTP4ilV48KJ7uszb9LhEGIZC02dFLq9Rb7j5SnQhtrpcfLX0oHKIUi9HFcLu8fUK+uO0aquRvKi5BnHa4JOhHt3wDjLJ2hk5St1LvPB7oCxDBLEpT7gg7hWt1cBvJskQrErxVyGDM8zeDDy20Au7yAOANLlGiR4AnVtrBynulXG9vBc68g1tOmyHdvYG8Ipuhrhy\\\\\\\/VfwrE9p13G\\\\\\\/wMUVabSy0mChABzbj0Fu\\\\\\\/ED36D1IzG0btw70C0JaJVOYhiRBD8VSqb1phSYLT8dmztDnvG1UOr\\\\\\\/2+HAMYP3D+VJsieWeRqSOjaCpJQQsLnbYsfs53e3G1zzbVfmZxC9zHluqpgU1iE\\\\\\\/sYNpKYMwaXPlRpkziqGnM0qSdZ9e6uoYjdXsfROIkC77pEYpz7eXRVEGRD9jZhIvAvh07+nJJfCsiBsdY4QwFL0pkEB\\\\\\\/LTewr9M9sZNus55WqjLKQTmRqiOsae22hL6e7pjUPU0NqTFI4X9j1E7oz4SkWwfqMhhC3TRc6FN2zhGIu2n\\\\\\\/5vMzhHKN88okhATyYRqFnqxkTpg5UJPl8MdVyDqhS+N++vkChVNAt9at\\\\\\\/Ij+\\\\\\\/z9oCRyvm\\\\\\\/nb\\\\\\\/CDp1Ied0l8rnNnSDBxWJjadSkX\\\\\\\/UQri60ydwR\\\\\\\/QajSKJ1UWUFebH+GZSJbxINt0SEHDuSNzoy3G2orza9QfjfUtSFYeBBXYyN2OqbyjY+OIqNt18+3R5L2RLbYzCOZh7XSKLouzYp+zT7qF6izkYklBG7n4bmUd9Ptgn+RtWuU3annqHt0z2GJ2qKwOmgn24qk8yS9sgAbCd7ffPCZD09zxtBxhn4+D2z4GFjgphYbiIkaLcXDKLg9IMvgtLpuEY7afh93r8DkyJL9F7Umj31lw9cPaeKPT4BQVQz0EvV83dCNbylSNzhmVapl6o1rXakj3EKxBJ9NhmFB7rQ8cwW3FaVx8b3ttvJQlJ7ICa7lBUYcMO4p+k25krKoozsp2xhJOlVTWt2qzTsJxMrH4TvyqoVmIBfuOhh0seLBryoEcmv1sfqMmQSw2KMx7nj9VD4YbfkVQM\\\\\\\/SzpHrTfTxpwk\\\\\\\/nrdHhJwumLF+MBaBfM3D64KBaMHaKd7PVJrkwBfn7g5qJcik3uSMVLvw3MYYFPrXyr3G5Js42E2pnL6fpCxwyqjFsJAfTw\\\\\\\/0n7\\\\\\\/ieylWZ+Xe\\\\\\\/DW+xv3P71FyFd6BReMUXERnkodQR42YNAh2cu\\\\\\\/haJbpSGdu910mqY7WIjVrN3jinAf8IAJ54lzrdDPFufJLA3JYkb+iqDiwC1Vr9z2ysa\\\\\\\/IWfpXunwCjzdWtg6ct7kx2m4N+9eDoKMrGj1TDarAYHbF\\\\\\\/4h57F0H+bRQokvDZPut8dwcImkXL\\\\\\\/khRMurvh52lCYDSbDaW+4v96OuvqTrggUkhwgnmKzRyIGBrIG3HtkB8OWmSteU9HFJSLFKnTEmb6mj2WIso0azrDCDDgeButjhNmYKIkiL\\\\\\\/o7ar0mVOGn0mMD39L3q9Lw8DZHSO4NVlOs7lkS7qaNmyFnSbs1NGZ9BTqOhTX8zEVhVlpWQHCMsp+1kM0tijiwoyjSXdg6qTERfHh2CHZivMi8sKolbVad6E5Vf\\\\\\\/YzkAL1MpW4enYUQiCMoTAGHkGu1h3X8DQFqbzdKZFmWOWOTreIWRioOKtjW8OiAnLOur27GBkIL+MxSow60FihGn2O4tYWOAC7HNmIP47WvYQqliiswdd9sSL77KmCyApXigCuILy\\\\\\\/+pRqoOwqZaIHbpNekAsi\\\\\\\/xcWGiWzi8fzNJjyoST7ZYIYSMd8vHjS8BhxhxxfUoOMatalgJvsGBXctjesSUpV48wBrGiFylvO+h9Rkx97OtbbhMkEylr8OB8x9N0KM3TxsZ64RGlQIJPQSwVKcsFOsd2qJ57k4ErnfK0yD5iqlJqMbjbZF3vKmesLTGDmiupQHPp\\\\\\\/XnMBlgIMewv0BEEMzlrEY73EctzSU7phJTcsgVs+QGg5K\\\\\\\/nbDJEpXR9asKNg1qlpa+pIc0az7m9jfFrV6rC5363p9MZhloHMwnNYKky1mvc0OOhDEBSCLAwMuYTAut4A3mP+pzZ3JFJPZw7WlZ6L1lEAWSx0IWgWYPmtn1irnE9bvLrBNXtGjHgmFGEKojb8ARaJrTUiXsuEh3bYUVBL0WRPXaMa8x4MRqqzL00pHQl6gAo\\\\\\\/+SGRvwXC7rML5IHRXIVJIKGTszW7\\\\\\\/lq1MXN8CdF52GEZttVy9fJakr\\\\\\\/+gFdvO87AKWaCH+nS9XRFUGTZZZWLUUKhPHV+NSDhakOm0TTeG65ZxilpM3aR4Dv8RNNV14ritkAJHTrhbynEMwSB3TryJI36sWAXi+uC\\\\\\\/\\\\\\\/X41MWsubtAI9GgCFTD+JfE8rK+Uf1Sm7Lv+jNUpG5S6aaV7ohbEitFmAPYToaN9boO0K4aFxJ+ek5MUv1vOKtgKGO6ArV8NqUFlLmj3arKf2HQ8GvvuJj7DD34Q\\\\\\\/pEOCGawtX6aIw9svrnTq6oTvyxNXHcllDefDeCDdOrRRiSZIEXFl6PuXst17dC4eGjQg5ZyUzQHaJ\\\\\\\/ZbJB46whYFkLwdbcx2m02IR14+SLwqWEj9+Cu0C0o+7HTLpe9iBsj2UuaRvLEnqb8hR35xETvGFamRAWA7Wof5q+57I5STgf3bnUOX4E9OJyPo+CDgRf4YjJBnaWo0iCaoT2DD24YyY2xg3yiCzpFwJdr76fAYUZuTESbGWq+Otlk62YlV3z6GKiKM16YM29\\\\\\\/2gXQiB8CronAms3f1wfBhXKEPEC3ywsT5emR69Uk1NLksC\\\\\\\/6C65C3Pezvuywj+ml7bybuKrRDPMLlpDfXbgfyTVYtPkGmmi7cvUFGPOHMEJe7M6FzwE9e5f1KokorIXBhbFr4K9noBjGfSlTPG+N06bRVn953S1qJuyVzxopHCl8igQXr+IteqUEnMxCNp8+b3c\\\\\\\/vWFI4zcQot5jQBPTCKEUsByZUHPsWE7llK1veq4IbZUEWxXsqhy6nMGkCW2RTe6J0GVIC2GRoUQ6ITPU6zSntLE9G9NUTCvbx7p+zSoBQReGoLD\\\\\\\/K4a+XgoYwxUgxqUAzQ7WWJFXLYpcschzPyZsZPAN+gmCzHWR8RbHdxz4FaG+pJPF3gW7VMlYBarYCUtBozQBWefuYs6dMIA0TtqdvGl09pDDO+n36OrJtJpTNba9KqLTJn01dPb2rQMGYWaCIXFPaA23UqTQf4VdSR0GcdtpIxcCpzObr\\\\\\\/\\\\\\\/Ezv7vGSP4ibrKnDYZxVy82XZkLRwCg2fZsad4s7QJ\\\\\\\/rr25TsZNMtxBujTmpVmRFf7tLQ1IB2I6InZTyKHbJgmnX1OFKnvntv3dMstkN4yWpyyGZtVgZofD\\\\\\\/zSAI\\\\\\\/hAIr6fIfTPgn1UvE4FQjKe2tdaYMxLna9xGdWK5J1UuFTmAPtTFcpujXAN+MXXRgGi2NSWh8Da+vjF8neNMDMTdOAKDeHeXZUawZ5lm3GcZgQ1kv6o\\\\\\\/W1Bwfh7Ob4ozkH2Bsbs6bav1Hj9X217WK9p0WGedNFOWMPN0Uar3IbS81igDqiNfmzyftU5QMhnqCTPSgnFBTVPzneSbTH3FBuQeY+10td0Mse1H2d+6NZhYenFIpqrzjcCOCeNd7rydsTfrC\\\\\\\/RjOpUELXnswpoKU1eReXsMD1bJpH\\\\\\\/\\\\\\\/IG84w4BTBe5uNjA2rn4mYXOFa1y24mH7xSi5Mnl1O8DkNk4lq2g60BCvylHt7caYkvpyaXa6I6prYfpnDsNDTXjfXVd6DmWAlvv+fBvL4SKnNi0kTnqR3yGzOit3Eagi3BdMWzqT9fD2kPlIfHS30jiTMBW08OeWZfMNaI3oZ7eZ1LN8vW99jLZ+qHPtbIdoPlT\\\\\\\/7a38IllE9JFfty+4G9vAD+L2wqPnGGfz95tugh2wTMqmdIHl5g\\\\\\\/23ciu\\\\\\\/6xXDR+LTT4lwLvOaUu38yrsBIinZhHummEnT\\\\\\\/y6pv0rblgz+\\\\\\\/uEyAPpkqrMzC7LU\\\\\\\/w7GFN94WT\\\\\\\/BvfRs1cwRib3+bTg8siGsnn4fdpP2ct2O0ri1SwsH4MAdpZGLBA48FIs3QsyTdIYq0ZinKleHEZSEHDdSAha2LaMyJeA4keAGGzIOxFCQO6eUR8vSt7Tu3SnUItiHn+hlDxzTTic\\\\\\\/avfD6Rik5QgrFXbfP78jBEEkPv+3ZK9uNEbPoytgh\\\\\\\/Ws84etKQjdY+nRTKzb4CesBPCnFySV7K3RlhpgY5FLZdwBE3QhXhFDFuX2eygkwmT9AC8tjTRbzTk7RSQD\\\\\\\/WVWXeKFxT8AFf9qrbYtFrWGb+CY2XrS\\\\\\\/HRyumjFhxmcPjTYUOluAQktr\\\\\\\/KPRd7heZg6Rt+PECg5wIzjEh5WwDQzNQkYnkc34vNPIvEOiEwCFQLm1Whh8WFQYUVDtijV80qgDrhcD94h3NivCw1ZGq0QVdesPcaXdfi5MG20rP36\\\\\\\/Dza6taESXgLCowpsuyt3\\\\\\\/Q2jPam6mitq0Rowg85EaCwd737QmHeRdzg9M+ImVSRXggt8GOGeQl0kP8wkahTL0i9FmsXHEK23zAq9PlRs7VCw53thCph\\\\\\\/nyWXiyWL7qwMTP8Zr\\\\\\\/qZg4RWpPDcYaYxC7M6C3YNk6ARoGYqVGY\\\\\\\/zPjkbzNcGV2EASLXfcZBMCAVUSSq8GTIwhOXR\\\\\\\/\\\\\\\/Yw24L3fVMK+IGZehFGsyD8K+mW+MnxuIE1ruziZihrE3s6uBrIZnx8D9yI4yh0X9fE4ZvlheB6AQlwMyg2rmxBuYtPPAPJAXOGsYBPEKnKifapdeZCgwpab+FJicVkhT+FY3j+mtYpzIcRzUlExdIjOYNMl12kefUOaRrj2APSr9dQhNOMBzxS\\\\\\\/amztttBZOF5Xq6UP9wQTJE3If1cF\\\\\\\/7tqbgnM+4+nviiTUksSOloeB7n5npKjClevyVsFOHaxbkDNLt3fBi2s2ilpxRrhxcHEPd0I\\\\\\\/HSN\\\\\\\/ZF0MYnU0+T2Jx0AsE+vFFMb4znLdlkrfvekNK5xbSebZ8TBiL4yMAC\\\\\\\/T3a2sa3fMuuDXR9VTd\\\\\\\/5hFAj0pJNS\\\\\\\/+FiZw4mbfKNfshfdIfePQ42FS6UJcsBSWuFre3oQgL0XJ0rsdZEHKSL\\\\\\\/6FWbk0A0uxkrg\\\\\\\/RnXqka+RLUypMw5ubSQ4k9l4+NozLI0TpgNdBNjSOUHmWJe6WqfA+K3XBdGXtyhH1KPImH3RopqBP9ZF\\\\\\\/8evZdoG15Xr+D5ZYSR2pCPDs63e9cXlvh6Wo\\\\\\\/pSEP8jjoa5155KKN8yEV68NZWpUOpHS4G4N4WjvmjspsHIWl4lQiClh0wHAobaWYVdjpxIy2luJ4wiLXt1nSCjS\\\\\\\/rkIALQ4oxFszvQpvbxxdg9qrmVlM3pQZXhL3T8M4loDSyNkZY5KXTmejUAeX\\\\\\\/9ZJ3CNNO19HuHwq7lj4DI1j2Vz+BGUuGJ8HDr87Q2AHobg2JgX44uhf1umgJslmAT7bhUGEfFVydHiue59KifTC8CKGbuGZ3hXvEeAs9I8JtIa4v4Z3812hPkA\\\\\\\/bzytMMWZceK\\\\\\\/tgfMsad0ao4DW88uks1QWT4rjigy\\\\\\\/0ZUGE7ujz9fKvnd17SRm8UiyBzSUPC+hFgBkYiDETcNjQ20prZXN5Un9QkV\\\\\\\/3VWce1L9gzQDV\\\\\\\/WDrjXOYrz1DEoDy7MOr3mlmEUsuQtGxLdk55M6QgJlM0vkvwfo9cXfk9kyYfoReISOgTkDtFDBztn1m6n2C8pH7XJxWX9xAW09wYmrZ4KJAoixTZfNZdgzurJHir1Nmvv0juQ+cKvLXpmrqk7cuTc4u0DHGD8dGbsbElVSHXYjrUgMEzezGwE67Pm\\\\\\\/Tgcxq8ZXMLL7OMU9d8n3On99Een1vdBiOsQ3\\\\\\\/hFpZntrFKgekzz3oO6pzrZ28s\\\\\\\/8GSBO0zSh+yrw2firLtXpOveZaaB1EVxco\\\\\\\/jH\\\\\\\/NtLstDq6rorApfTUbikXTUe5rZMXrkws5bedb+ZvruyxkB\\\\\\\/y669xO3sEBWf2\\\\\\\/ncmrO9LhOJ5WLTgwJYlGEjBADVBisBchH7juti3\\\\\\\/w\\\\\\\/I+25aE8TC6mUURSP1af4th4xToWyfDESZPs9H6QPaRSgn5Jbw7WC7MZ7yjYUrx2fpZWYqZebqZVHQM+OMXPz2R0X140c4dHbbdR8wxaIw4GpA4PERCNgwJI2Fm5ldMnmF+EnoKxczwHQpL2y6heivkZ6GdkvKrrkv\\\\\\\/rPk\\\\\\\/Gf6wHfM3SSH72xvke1Z36WTYxizZkR1wHCaQ3GeEe9IG4hdeuBYKO2bPlPvQpkGXXOJQDCY6eUUk81ohUeeSyhqE8CvvFFiIJg9pMGy1gpgGIHnxJhFOCPM1BlpF\\\\\\\/ahOnAGCGXu+Lic\\\\\\\/AF6D6vp+qj1PqX8ZzGYj63x4c+0fY9cXH2FO\\\\\\\/lo2ckxLxPeASVLs1JRvjwXpv9igILOa\\\\\\\/LTiVMvML11ibpK15c67qQcRTGQ9qz9zrZOEcZjun8LdUsdxPoXgcxzX2iHLm6zSNtfFfoBzNdfgnIVr2f7B6Ka0LlO2NlUbijBDDOjilej62rkmeeEl7c0T8mwJKQzPYO36PyU3t\\\\\\\/fYvf5P4kkEQTPjiTE0xcdXwW0Ahg78jjGIjM6GD1lL\\\\\\\/LqZVY6KRNjuzMeGEYuXa448pUGDMVfGPfnz3Pg3v0E2lBTyLBrPpu5HtT51Q5fymZE6+mN71qVYeKe7kD97v74hTuA2dcIFLW55\\\\\\\/TRNUF3b+sh0cJfUco8d4eTgt9fGVCDPOK1eytumHVigHqGD5nhXgpPWxd4pPCNYhVlJMRMI4lfgc\\\\\\\/szddlDA3\\\\\\\/C2zXBXJi3kWDWCdaViaxAHdToravI6WZAukPymixV39Xqv5qakeThCufOfSxL6SaQniZ1LdCykXpUPzla7rf5cE\\\\\\\/GzqX5TmUdCXg7Q7cppLhG3JZxYUbgwCZX1TvRVNL2EYUPMK4vcyL9mN5q+PscYcvyVl2dkFclNBhsX7bzTQBvVF6nFkuPqScutnVnCMm42\\\\\\\/BUKebQASPsRguhFiQArxvbeBSDLOYpVusOip5hbOn9mQUv82qyZwZzBr1L6mU5tHL\\\\\\\/v+HfSW9TWqRfmndpOlfaJ8u3C2rT8dbuRnI0ATXyiJInuYZlyG2ZnFoZGWi1s51vc2zdY+KAtCrIRXJy6haYM2K3ElSJ6nqIx1xSJZGnsr4ORWiyoS0uPTGYAL1iG\\\\\\\/W8UNLyp1EEZwuTQbxbTKPkJJeLIirI1zBoP+aR0E7XAKawfX1uGfXkoenBWhgZBEsnyP4bG07935m1GsjWQ23ysvmIH1hLvHC1hSNjEycyjftQCKUOppb6Plbxfsp+oJmjpktAdl5LzCXG8Fw8F4F1t+5Stli1SNASnI8StvpnlHPxV5jTbFpsf4TkUkTevr32n7uzLMBVFQzSvTg\\\\\\\/S12CQHNzinPX+F6tYF7LGJ1XOgGkGfjquvg+k7O8NVymFAQU4eDDjltReJExoC9ZTg5SHtDGumHFOnOOmRvZWHVBzdZ5Vm\\\u0022,\\\u0022iv\\\u0022:\\\u0022BGoJsWh3cu\\\\\\\/Mj0ynmxlZoA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00226cd823551bcd84d82d00b4de0716bf90b9add604e9f45d7bed30bf8b5715c197\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022+RPaRpquFtnIDaf+l4BpVsRCfRS73Z+tK1Lyc5TUslLKS9RqqQgkm3oOewlpw1TT5e9t2l6nBFlPOItIGF56h3adE56cAqEzRrc06r4UycgLu14yPFyeQp\\\\\\\/vu+Ct+bOSPz5m3mZq8wSbFC4+SoLtKUCb9ali58oyJsL2chApudg57l7uNVttguOEOEZWF11VOmIz8FDEyumeoN6Ae79J0pTH7jGrSDWjQUzZliq4DrCyWU2R95FMM95CDrZUl8a9sHX6trVv4lfYdP\\\\\\\/2h2NO0Rnl7LwlE4Qsnum+8GihN6l9bHVl7hvwCkB3GtPetR+GTBHRZ2ERp+QrQ+FF2zMZKLqonxSeaM8SQdrh5U7xA8Qvzpx2+GcITx6bc+u9N7YbqyVP\\\\\\\/8EN\\\\\\\/SiY1k5eRg5dGBa4nmjZIO\\\\\\\/xUfBbg4Tg7lUsrxp2kRiU+SoCRu+RYdqp+2LxNyeBP+NS8s+PGVVu2AUU8qN3bS2SOxF+h8fxh4FBZX1b0r9FASW+h4mPLt9PdbfSXjLYWZoYOXlI\\\\\\\/6jS9vsLDSfFDlA4b8UXyOu5t+tP8OUNHcBJBUmSAS12VBySFsT1zOUDc5YWqxhxzJa96UBy\\\\\\\/biEFgocRktcnyOs3Y9jcWi++eJfCQZ91mwGYhkwUgXTydDwAn\\\\\\\/WbD8zU7u4BXdZcw70SUWGw9RPoaqwDkvBfdDAp70DjoynsJkekE5Vlzi9o7B35mZzZ7iRjMquD2klYBktNlxiGki5jDHym38EV9M3zE1NL7uWUmEVEL58V4OWCrbGhSbdftEPTr69rPVwocrgc4QGLMc8wIcCmiFC7E+L9rxVwQxIc5Vds+K6fbOvvXAO+VDTBF\\\\\\\/74U59pYPJsME3y7D6fBySgEOmNiqWolOaO2uhAJrp7pDg+UtLb1dIzjoGsSZCyaFFQuftrt6pu1wYQUz30TWED\\\\\\\/gfhhk8GLEWeaQ8IVU230vi9pTrKLumLfQ\\\\\\\/JqUZDwTaQwsaLRXhr+mDbHO7RSDm+qX0qEwI3wSsdl8lcrjc2nsjWPvoO52ZJoNNDa53xyqkvAFrI7YOv++357fwypJhTlGWuDMGGoOX2fbXcbsGGrO6oqXAnPePRYX7A3hLEGXRFLhLJb1jvDWOMJE0uCNBabfMIKlFxuotLo5KhjjydlBt4c4xm7Mm\\\\\\\/pacmvxwH8GwJ+0Iz7XZcV6bSiQeD13cCKLzYf2QiOzlhaO3V+qZQhQxGqpLCmNHvYpPiWZYVzxl8PpPrX0tREj9ZLCQup8W3DDn783L6kuwFd8lRj5KDku9YlbOeAkFe84vgnBo3cO8XJ0\\\\\\\/xijRYadPb7\\\\\\\/BIa2op9h7+Nb+enCG+TmHHqbze9ZxH9XoBuP3FwBQXkZe0LvWzAT8L+EgBmOO+SboZS\\\\\\\/9T7c4t1\\\\\\\/DBF8JE3MfR6OoV\\\\\\\/9nTCM\\\\\\\/Cy6ObF8ZVS23+7OlYqKwer3uFQ9aRW2BCqh9vEsYrDzdSvERZ6Ibo9vVeLCKj73e8uxru0HpUr3lzgKd6\\\\\\\/7426kdOrFRvVFesrEpHpHVqWNTYXE3R6xOpG\\\\\\\/yMTNC22HgAfqx\\\\\\\/sJRatzxdyHsxAraC5nO1wehPKgw+IiZN8\\\\\\\/\\\\\\\/K7MQqSsjYlblXEalsc+mPCb1j1N3+FaIiSWSNizuyDmRt9ctrbBpKe1zx3lCciNQyDg+ZQ3u+UZAZ\\\\\\\/hm8JgMZfjP99+Ekb5pw6lfEACKZtnPDNadjYe7DEOGzwYuAmOm+4zqA2kO2Mr61njKYeprYSMq5YwHOLWb5+yDlfvJ0JkKBaCjUvwnuvodUL5bH5tT86zxkk7tgNj8K4YjjxJq5QL9Na+Lg3OluEBBGsQN6oZvFFJjSuXeXD0TQb\\\\\\\/mqaV4AuUMq4eL1ejBdyIJxs5efBVfoOjIjs99\\\\\\\/RaVQGDCUJwRfaV8xDm9qxEeLegx7QQ7V9++0IQ7y+YkDnx+iaYZiWEGVS1upf4S+dmUOsluvp2SRyj2kBa5\\\\\\\/nmZL4UvgM2iEZOx4CPidXM879wb\\\\\\\/exv0okjr2DwJhEtlxIWdtDv1uwsBb8bOxJlbERMZMYuJAXsXOw=\\\u0022,\\\u0022iv\\\u0022:\\\u0022tYhkeCDjkCoVuXMqMNxqQQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00227ab83b26ca91694d36b6e66aa49b84e9043e16b8c8ac73ef1470df547e157486\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-490969 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/10/3185" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-k58qO3YV0SRneSQ45MQFQyQMgbbSHv-W_-mcL8zt2EU" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/10/3185" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-zhdmK-YDEv37l5QRNU52edzGFWcf-7tqrBd24Bw8cZc" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/10/3173" title="Accounting for Programmed Ribosomal Frameshifting in the Computation of Codon Usage Bias Indices" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/10/3203" title="Estimating Genetic Relatedness in Admixed Populations" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490969" id="node490969" data-pisa="ggg;8/10/3185" data-pisa-master="ggg;g3.118.200377" data-apath="/ggg/8/10/3185.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Linkage Disequilibrium and Evaluation of Genome-Wide Association Mapping Models in Tetraploid Potato</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-8373-5509" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Sanjeev Kumar Sharma</span>, <span class="highwire-citation-author" data-delta="1">Katrin MacKenzie</span>, <span class="highwire-citation-author" data-delta="2">Karen McLean</span>, <span class="highwire-citation-author" data-delta="3">Finlay Dale</span>, <span class="highwire-citation-author" data-delta="4">Steve Daniels</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0002-7752-5740" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Glenn J. Bryan</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3185-3202;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200377 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node490969" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Sanjeev Kumar Sharma </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The James Hutton Institute, Dundee DD2 5DA, UK</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sanjeev%2BKumar%2BSharma%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sharma%20SK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASanjeev%2BKumar%2BSharma%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-8373-5509" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Sanjeev Kumar Sharma</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:glenn.bryan@hutton.ac.uk" class="" data-icon-position="" data-hide-link-title="0">glenn.bryan@hutton.ac.uk</a>
<a href="mailto:sanjeev.sharma@hutton.ac.uk" class="" data-icon-position="" data-hide-link-title="0">sanjeev.sharma@hutton.ac.uk</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Katrin MacKenzie </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Biomathematics and Statistics Scotland (BioSS), Dundee DD2 5DA, UK</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Katrin%2BMacKenzie%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=MacKenzie%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKatrin%2BMacKenzie%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Karen McLean </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The James Hutton Institute, Dundee DD2 5DA, UK</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Karen%2BMcLean%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=McLean%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKaren%2BMcLean%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Finlay Dale </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The James Hutton Institute, Dundee DD2 5DA, UK</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Finlay%2BDale%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dale%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFinlay%2BDale%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Steve Daniels </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Cygnet PB Ltd, Cambridge, CB21 6AS, UK</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Steve%2BDaniels%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Daniels%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASteve%2BDaniels%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Glenn J. Bryan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The James Hutton Institute, Dundee DD2 5DA, UK</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Glenn%2BJ.%2BBryan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bryan%20GJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGlenn%2BJ.%2BBryan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-7752-5740" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Glenn J. Bryan</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:glenn.bryan@hutton.ac.uk" class="" data-icon-position="" data-hide-link-title="0">glenn.bryan@hutton.ac.uk</a>
<a href="mailto:sanjeev.sharma@hutton.ac.uk" class="" data-icon-position="" data-hide-link-title="0">sanjeev.sharma@hutton.ac.uk</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/8/10/3185" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:490969" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:490969/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/10/3185.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:490969" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:490969/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/10/3185.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:490969" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:490969/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/10/3185.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:490969" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:490969/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/8/10/3185.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:490969" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:490969/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Genome-wide association studies (GWAS) have become a powerful tool for analyzing complex traits in crop plants. The current study evaluates the efficacy of various GWAS models and methods for elucidating population structure in potato. The presence of significant population structure can lead to detection of spurious marker-trait associations, as well as mask true ones. While appropriate statistical models are needed to detect true marker-trait associations, in most published potato GWAS, a ‘one model fits all traits’ approach has been adopted. We have examined various GWAS models on a large association panel comprising diverse tetraploid potato cultivars and breeding lines, genotyped with single nucleotide polymorphism (SNP) markers. Phenotypic data were generated for 20 quantitative traits assessed in different environments. Best Linear Unbiased Estimates (BLUEs) for these traits were obtained for use in assessing GWAS models. Goodness of fit of GWAS models, derived using different combinations of kinship and population structure for all traits, was evaluated using Quantile-Quantile (Q-Q) plots and genomic control inflation factors (λ<sub>GC</sub>). Kinship was found to play a major role in correcting population confounding effects and results advocate a ‘trait-specific’ fit of different GWAS models. A survey of genome-wide linkage disequilibrium (LD), one of the critical factors affecting GWAS, is also presented and our findings are compared to other recent studies in potato. The genetic material used here, and the outputs of this study represent a novel resource for genetic analysis in potato.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/linkage-disequilibrium" class="hw-term hw-article-keyword hw-article-keyword-linkage-disequilibrium" rel="nofollow">Linkage disequilibrium</a></li><li class="kwd"><a href="/keyword/kinship" class="hw-term hw-article-keyword hw-article-keyword-kinship" rel="nofollow">Kinship</a></li><li class="kwd"><a href="/keyword/population-structure" class="hw-term hw-article-keyword hw-article-keyword-population-structure" rel="nofollow">Population structure</a></li><li class="kwd"><a href="/keyword/genome-wide-association-studies-gwas" class="hw-term hw-article-keyword hw-article-keyword-genome-wide-association-studies-gwas" rel="nofollow">Genome-wide association studies (GWAS)</a></li><li class="kwd"><a href="/keyword/mixed-models" class="hw-term hw-article-keyword hw-article-keyword-mixed-models" rel="nofollow">Mixed models</a></li><li class="kwd"><a href="/keyword/tetraploid" class="hw-term hw-article-keyword hw-article-keyword-tetraploid" rel="nofollow">Tetraploid</a></li><li class="kwd"><a href="/keyword/potato" class="hw-term hw-article-keyword hw-article-keyword-potato" rel="nofollow">Potato</a></li><li class="kwd"><a href="/keyword/single-nucleotide-polymorphism-snp-0" class="hw-term hw-article-keyword hw-article-keyword-single-nucleotide-polymorphism-snp" rel="nofollow">Single-nucleotide polymorphism (SNP) </a></li></ul><p id="p-3">Cultivated potato, <em>Solanum tuberosum</em> Group <em>tuberosum</em> is a highly heterozygous, autotetraploid crop. Due to the many genetic complexities posed by autopolyploidy, most potato genetic studies have been performed at the diploid level using bi-parental populations. Significant recent progress has been made in the development of improved algorithms and software for linkage and QTL analysis in autopolyploid crop species (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Hackett <em>et al.</em> 2014</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Hackett <em>et al.</em> 2013</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Rosyara <em>et al.</em> 2016</a>). The genes affecting many key agronomic and economically important potato traits remain undiscovered, despite the availability of dense genetic maps as well as the potato genome sequence and associated genomic resources (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Sharma <em>et al.</em> 2013</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Potato Genome Sequencing Consortium 2011</a>). A major contributory factor is that the genetic locations of many trait QTL are imprecisely determined due to the complex genetic architecture of many traits as well as to the use of highly heterozygous parental lines for QTL studies. Moreover, markers found to be tightly linked to useful trait alleles in biparental crosses often have limited transferability to wider germplasm pools. Moreover, biparental populations display recombination limited to a single meiosis, which hampers the resolution of QTL mapping (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Stich <em>et al.</em> 2013</a>). The accurate molecular dissection of several complex traits at the tetraploid level across a wide range of potato germplasm using high-throughput molecular marker platforms is a highly desirable goal.</p><p id="p-4">Association mapping studies have expanded significantly in number in crop plants and are gaining importance for studying the genetic architecture of traits of agricultural and adaptive significance. Association mapping offers considerable advantages over use of bi-parental populations, such as enhanced mapping resolution, a greater number of traits that can be analyzed in a single study, as well as an increased likelihood that marker-trait associations discovered are likely to be robust across a wider germplasm pool. This latter benefit raises the possibility of using association mapping panels as ‘training sets’ for the establishment of genomic selection models. Early association mapping attempts in potato were largely confined to candidate gene approaches (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Achenbach <em>et al.</em> 2009</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Gebhardt <em>et al.</em> 2004</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Li <em>et al.</em> 2005a</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Malosetti <em>et al.</em> 2007</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Pajerowska-Mukhtar <em>et al.</em> 2009</a>; <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Urbany <em>et al.</em> 2011</a>). Though successful in identifying marker-trait associations, the candidate gene approach has some critical issues such as (1) bias caused by the selection criteria used to choose genes on the basis of information available from previous genetic, biochemical and/or physiological studies (<a id="xref-ref-52-2" class="xref-bibr" href="#ref-52">Stich <em>et al.</em> 2013</a>), (2) limitation to traits for which the biochemical and molecular basis are known (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Hall <em>et al.</em> 2010</a>), and (3) it is prone to missing important but unknown genes involved in the target traits (<a id="xref-ref-52-3" class="xref-bibr" href="#ref-52">Stich <em>et al.</em> 2013</a>). Nevertheless, association mapping using candidate genes has been shown to be a powerful approach for complex trait analysis in potato. Potato association mapping studies using genome-wide markers have been performed (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">D’hoop <em>et al.</em> 2014</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">D’hoop <em>et al.</em> 2008</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Fischer <em>et al.</em> 2013</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Lindqvist-Kreuze <em>et al.</em> 2014</a>; <a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Rosyara <em>et al.</em> 2016</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Schönhals <em>et al.</em> 2016</a>; <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Vos <em>et al.</em> 2016</a>). These truly genome-wide association studies (GWAS) have led to significant findings of marker alleles associated with important potato traits, such as tuber shape, flesh color, after cooking darkening and enzymatic browning (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">D’hoop <em>et al.</em> 2014</a>), late blight resistance (<a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Lindqvist-Kreuze <em>et al.</em> 2014</a>), starch content (<a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">Schönhals <em>et al.</em> 2016</a>). Perhaps most notable among these was the important role played by GWAS in identifying the <em>StCDF1</em> gene, a circadian clock gene underlying a major-effect QTL for plant maturity (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Kloosterman <em>et al.</em> 2013</a>).</p><p id="p-5">GWAS requires a detailed understanding and exploration of population structure to minimize false-positive and false-negative associations due to population stratification. In early potato association mapping studies, assessment of population structure and GWAS were performed using ‘conventional’ marker types such as amplified-fragment length polymorphisms (AFLPs) and simple sequence repeats (SSRs) (<a id="xref-ref-8-3" class="xref-bibr" href="#ref-8">D’hoop <em>et al.</em> 2014</a>; <a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">D’hoop <em>et al.</em> 2008</a>) and a limited set of single-nucleotide polymorphisms (SNPs) (<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Achenbach <em>et al.</em> 2009</a>; <a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Pajerowska-Mukhtar <em>et al.</em> 2009</a>). However, for a comprehensive analysis the use of high-throughput and marker-dense genotypic data are preferable. The first such platform in potato containing 8303 SNPs (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Felcher <em>et al.</em> 2012</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Hamilton <em>et al.</em> 2011</a>) has been deployed to characterize diploid and tetraploid germplasm (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Hardigan <em>et al.</em> 2015</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Hirsch <em>et al.</em> 2013</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Kolech <em>et al.</em> 2016</a>; <a id="xref-ref-52-4" class="xref-bibr" href="#ref-52">Stich <em>et al.</em> 2013</a>). A further challenge for performing genetic studies at the tetraploid level has been the lack of suitable automated methods for correctly inferring marker allele dosage. SNPs are co-dominant markers and methods for calling allele dosage in autotetraploids have recently been developed (<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Hackett <em>et al.</em> 2013</a>; <a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Voorrips <em>et al.</em> 2011</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Schmitz Carley <em>et al.</em> 2017</a>). These developments coupled with the potential for performing high-throughput assays have rendered SNPs as the preferred markers for GWAS. Association studies in potato utilizing high-throughput genome-wide markers are increasing (<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Kloosterman <em>et al.</em> 2013</a>; <a id="xref-ref-33-3" class="xref-bibr" href="#ref-33">Lindqvist-Kreuze <em>et al.</em> 2014</a>; <a id="xref-ref-44-3" class="xref-bibr" href="#ref-44">Rosyara <em>et al.</em> 2016</a>; <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">van Eck <em>et al.</em> 2017</a>; <a id="xref-ref-60-2" class="xref-bibr" href="#ref-60">Vos <em>et al.</em> 2016</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Berdugo-Cely <em>et al.</em> 2017</a>; <a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">Schmitz Carley <em>et al.</em> 2017</a>).</p><p id="p-6">We have developed an association mapping panel of 341 tetraploid potato cultivars and breeding lines, largely comprising European founder and cultivated germplasm but including 29 non-European cultivars. Here, we report the genotyping of this panel using an 8k SNP array and the examination of various GWAS models using a set of 20 quantitative traits. We also present an assessment of genetic diversity, individual relatedness, population structure, and a genome-wide survey of linkage disequilibrium (LD) in potato.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Field Trials and Phenotyping</h3><p id="p-7">Field trials were conducted in 2012 and 2013 at two different sites (Cambridge and York, United Kingdom). At each site replicated trials (two replicates) with two nitrogen levels (100 and 200 kg/ha) were conducted according to an alpha design, thereby leading to a total of eight ‘environments’. Twenty quantitative traits were measured, but not all were phenotyped in every environment (Table S1). The traits analyzed were plant height, yield, days to emergence (assessed as days to reach 50% emergence), stolon attachment (1-9, 1 very strong), tuber uniformity (1-9, 9 very uniform), foliage (height to breadth ratio, assessed at ∼80% ground cover but before canopy closure), total number of tubers, tuber dry matter (percentage dry matter based on air-dried and water-immersed weights), eye depth (1-9; 1 deep, 9 shallow), tuber shape (length to breadth ratio), tuber skin brightness (tuber skin texture; 1-9, 9 very smooth), mean number of stems per plant (assessed after full emergence but before the development of secondary stems; main stems only), tubers per stem, tuber flesh color (1-9; 1 white, 9 deep yellow), after cooking blackening (tubers steamed for 20 min and assessed for darkening after 30 min; 1-9, 1 severe), enzymatic browning (darkening of tuber cut surface assessed after 1 hr; 1-9, 1 severe), tuber fry quality (percentage glucose in tubers stored at 6° with CIPC (Chlorpropham) for ∼3 months after harvest), dormancy break (number of tubers with sprouts &gt; 3 mm, storage at 10° without CIPC), ethylene treated tuber sprouting, and maturity. To generate phenotypic values for each trait, genotype was modeled as a fixed effect while all other effects were treated as random, and the Best Linear Unbiased Estimates (BLUEs) for all traits were calculated using REML implemented in Genstat 15<sup>th</sup> edition (VSN International Limited, <a href="http://www.vsni.co.uk">http://www.vsni.co.uk</a>). These trait BLUEs were used in GWAS as well as all other analyses involving phenotypic values.</p><p id="p-8">Field trials were rogued by potato field staff for any discrepancies in the observed phenotypes and misidentification/mislabeling of clones. Genotypes which showed inconsistent phenotypes were sent for DNA fingerprinting at Science and Advice for Scottish Agriculture (SASA, <a href="https://www.sasa.gov.uk/">https://www.sasa.gov.uk/</a>) for further validation against the varietal SSR genotypic database held at SASA. Correspondence of phenotypic data between replicates, sites and years was also assessed for any gross inconsistencies. For all traits, residuals plots from REML analysis were examined for presence of any outliers and adherence to a normal distribution.</p></div><div id="sec-3" class="subsection"><h3>Germplasm and Genotyping</h3><p id="p-9">The germplasm panel comprised 351 diverse tetraploid genotypes including a set of 57 advanced breeding lines. Seed tuber material was obtained from SASA or from stocks held at The James Hutton Institute (Hutton) and multiplied in 2011, the year preceding the first field trial. In addition, we included seven diverse diploid potato genotypes in the genotyping panel to act as ‘outgroups’ and to aid validation of population structure analysis. The diploid material included the sequenced doubled monoploid <em>Solanum tuberosum</em> group Phureja DM1-3 516 R44 clone (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Lightbourn and Veilleux 2007</a>; <a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Potato Genome Sequencing Consortium 2011</a>), hereafter referred to as DM. The details of the germplasm included in the study are presented in Table S2.</p><p id="p-10">Genomic DNA was extracted from young leaf tissue from individual field grown plants using the Qiagen DNeasy Plant Maxi Kit (Qiagen), quantified using the Quant-iTTM PicoGreen dsDNA Assay Kit (Invitrogen, San Diego, CA), and normalized to a concentration of 30 ng/μL. The panel was genotyped at GenProbe (Liverpool, UK) for the Infinium 8k Potato SNP Array (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Felcher <em>et al.</em> 2012</a>; <a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Hamilton <em>et al.</em> 2011</a>) according to the manufacturer’s protocols. The cultivar Adirondack was ‘spiked’ as a control in all 16 Infinium array BeadChip genotyping batches of 24 genotypes each. SNP genotypes were called using R package fitTetra (<a id="xref-ref-59-2" class="xref-bibr" href="#ref-59">Voorrips <em>et al.</em> 2011</a>) which employs a mixture model based on the allele signal ratio and fits distributions to the response signals to classify the individuals into one of the five possible genotypic classes in the tetraploid genotype model, giving dosage information (0, 1, 2, 3 or 4) for each SNP. SNPs with inconsistent genotype calls across control tetraploid samples (cv. Adirondack) and those reported as mapping to more than one location in the potato genome (<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Hirsch <em>et al.</em> 2013</a>) were excluded (Table S3). For performing GWAS, SNPs with 20% (or more) missing data and 5% (or below) minor allele frequency (MAF) were excluded. Physical positions for all SNPs were derived from <a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">Sharma <em>et al.</em> (2013)</a>.</p></div><div id="sec-4" class="subsection"><h3>Kinship and Population Structure Analyses</h3><p id="p-11">SNP dosage scores were used as input data in the SAS/STAT Distance Procedure for the estimation of Gower’s Similarity Metric (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Gower 1967</a>) among all individuals and the estimated matrix was used as a genomic relationship matrix. Population structure was assessed using three different approaches, namely principal component analysis (PCA), non-metric multidimensional scaling (nMDS) and STRUCTURE (STR) analysis. The genotype membership matrices (Q) obtained from the population structure analyses were used as covariates in the GWAS regression models. PCA was performed on the genomic relationship matrix and the first 20 eigenvectors were calculated. For nMDS, the square root of the symmetric relationship matrix obtained via Singular Value Decomposition was used and the analysis was performed with data fitting up to 20 dimensions. PCA and nMDS were performed using JMP Genomics 7.1 software (SAS Institute, Cary NC). Model-based analysis of population structure was carried out using STRUCTURE software (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Pritchard <em>et al.</em> 2000</a>) with ‘admixture’ model assuming allele frequencies are correlated between inferred populations. The ‘burn-in’ and simulation stages were both set at 10,000 iterations. The number of subpopulations was evaluated from 1 to 10 and each analysis was replicated three times. The optimal population structure estimates (Q matrices) from nMDS (dimensions) and PCA (eigenvectors) were chosen by evaluating their respective screeplots displaying ‘badness-of-fit’ and eigenvalues, respectively. In STRUCTURE subpopulations were inferred by the <em>ΔK</em> method of <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Evanno <em>et al.</em> (2005)</a> and a single replicate from the chosen <em>K</em> was used to assign the probability that a genotype belonged to in each subpopulation.</p></div><div id="sec-5" class="subsection"><h3>Genome-wide Association Analysis</h3><p id="p-12">GWAS model procedures were as described by <a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Yu <em>et al.</em> (2006)</a> implemented in GWASpoly R package (<a id="xref-ref-44-4" class="xref-bibr" href="#ref-44">Rosyara <em>et al.</em> 2016</a>) using additive model; and follow the linear mixed model equation as expressed below:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3185/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/10/3185/embed/mml-math-1.gif"/></noscript></span></span>where <strong>y</strong> is a vector of observed phenotypes; <strong>β</strong> is a vector of fixed effects other than the SNP under testing and/or population structure effects; <strong>α</strong> is a vector of SNP effects; <strong>v</strong> is a vector of population effects where <em>Q</em>, modeled as a fixed effect, refers to the incidence matrix for subpopulation covariates relating <strong>y</strong> to <strong>v</strong>; <strong>µ</strong> is a vector of random polygenic background effects with covariance proportional to a kinship matrix <em>K</em> with Var[<strong>µ</strong>] = <em>σ<sub>g</sub><sup>2</sup>K</em>; <strong>ε</strong> is a vector of residual effects with Var[<strong>ε</strong>] = <em>Iσ<sub>e</sub></em><sup>2</sup>; X, S and Z are incidence matrices of 1s and 0s relating y to <strong>β</strong>, <strong>α</strong> and <strong>µ</strong>, respectively; and <em>σ<sub>g</sub><sup>2</sup></em> and <em>σ<sub>e</sub><sup>2</sup></em> denote genetic and residual variance, respectively. This mixed model equation was adapted to perform GWAS using four different statistical models, as detailed below:</p><ol class="list-alphalower " id="list-1"><li id="list-item-1"><p id="p-13">Naïve model, without controlling for population structure ‘Q’ or individual relatedness ‘K’</p></li><li id="list-item-2"><p id="p-14">Kinship model, controlling just for individual relatedness</p></li><li id="list-item-3"><p id="p-15">Population Structure model, controlling for population structure effects</p></li><li id="list-item-4"><p id="p-16">Full model, accounting for kinship as well as population structure confounding effects</p></li></ol><p id="p-17">In addition to the description above, all these four models had SNP fitted as a fixed effect and are hereafter referred to as Naïve, K, Q and QK models, respectively. Q and QK models were successively evaluated using population structure estimates derived from three different methods as described in the preceding section, thereby leading to a total of eight GWAS models being tested. Fitness of different GWAS models for all traits was evaluated using Quantile-Quantile (Q-Q) plots of the observed <em>vs.</em> expected –log10(<em>p</em>) values which should follow a uniform distribution under the null hypothesis. Models were ranked using genomic control inflation factor (λ<sub>GC</sub>) metric calculated using R package ‘GenABEL’ (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">GenABEL project developers 2013</a>). Bonferroni correction (with genome-wide α = 0.05) was used for establishing a p-value detection threshold for statistical significance.</p></div><div id="sec-6" class="subsection"><h3>Linkage Disequilibrium</h3><p id="p-18">Pearson correlation coefficient (r<sup>2</sup>) was used to calculate correlations between marker-pairs using SNP dosage scores (0 to 4). LD was calculated based on marker pairs located within (a) chromosomal short arm, (b) chromosomal long arm, (c) euchromatin (both chromosomal arms combined), (d) heterochromatin, and (e) whole chromosomal region for all 12 chromosomes. Extent of LD decay was estimated by implementing Quantile regression (R package ’quantreg’; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Koenker 2017</a>) on the 90<sup>th</sup> percentile as recommended by <a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Vos <em>et al.</em> (2017)</a>. From the fitted regression two LD estimators were obtained, <em>viz</em>. LD<sub>1/2max,90</sub> and LD<sub>1/10,90</sub>, denoting the distances at which LD equals one-half of its maximum fitted r<sup>2</sup> value (r<sup>2</sup><sub>max,90</sub>) and where r<sup>2</sup> equals 0.1 on the 90<sup>th</sup> percentile, respectively. For whole chromosomal regions, extent of LD decay was also estimated using non-linear regression of r<sup>2</sup> against the physical map distance according to <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Marroni <em>et al.</em> (2011)</a>. Marker positions for the different subsets of the genome were derived from <a id="xref-ref-47-3" class="xref-bibr" href="#ref-47">Sharma <em>et al.</em> (2013)</a>.</p></div><div id="sec-7" class="subsection"><h3>Genetic Diversity</h3><p id="p-19">The measure of polymorphic information content (PIC) for each SNP was calculated according to <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Botstein <em>et al.</em> (1980)</a>. Pairwise genetic distances were calculated using <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Nei’s (1972)</a> distance estimate which were further subjected to hierarchical clustering using ward.d2 method (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Murtagh and Legendre 2014</a>). Trees were constructed using R package dendextend.</p></div><div id="sec-8" class="subsection"><h3>Data availability</h3><p id="p-20">Supplemental Material is available at figshare and brief captions for the uploaded material are provided here, as follows: Table S1 (traits phenotyped); Table S2 (germplasm included in the study); Table S3 (List of SNPs mapping to multiple positions); Table S4 (GenomeStudio and fitTetra genotype call comparisons); Table S5 (Chromosomal region-wise details of SolCAP SNPs); Table S6 (Size of pericentromeric heterochromatin); Table S7 (Pairwise measures of population differentiation); Table S8 (Significant marker-trait associations); Table S9 (Annotation of SolCAP SNPs used in the study); Table S10 (Genotype and phenotype data); Figure S1 (Manual and automated SNP genotype calling example); Figure S2 (PCA performed on the genomic relationship matrix of the association panel clones); Figure S3 (Traits Pairs plot); Figure S4 and S5 (LD in chromosomal short and long arms, respectively); Figure S6 (nMDS Screeplot); Figure S7 (population structure present in the association panel); Figure S8 (clustering of genotypes based on <a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Nei (1972)</a> genetic distance); Figure S9 (PCA Screeplot); Figure S10 (STRUCTURE results); Figures S11 and S12 (nMDS and PCA based population stratification); Figures S13 and 14 (Q-Q plots for different categories of model comparisons). Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.6716033">https://doi.org/10.25387/g3.6716033</a>.</p></div></div><div class="section results-discussion" id="sec-9"><h2 class="">Results and Discussion</h2><div id="sec-10" class="subsection"><h3>Germplasm and Genotyping</h3><p id="p-21">Germplasm included in the study, including geographic origin, year of introduction, and predominant market class, is listed in Table S2. The tetraploid panel and selected diploid material were genotyped using the 8303 Illumina Infinium SNP array (<a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">Felcher <em>et al.</em> 2012</a>). fitTetra analysis resolved genotypic classes for 6059 SNPs. Genotyping accuracy was assessed by comparing genotype scores among 16 control (cv. Adirondack) samples. Genotype calls were largely in agreement with a consistency rate of 98.3%. SNPs displaying inconsistent genotyping calls across control samples as well as those with multiple mapping locations in the potato genome were excluded yielding a set of 5,718 robust SNPs for further analyses. Illumina standard SNP calling software GenomeStudio, at the time of analysis, was not suitable for automatic genotype calling in tetraploids. Genotype score comparisons (for tetraploids) from these platforms largely corroborated each other (data not shown) and a representative example is shown for the marker ‘solcap_snp_c1_1000’ (Figure S1). A comparison of genotype scorings from both approaches for all diploid genotypes is presented in Table S4.</p><p id="p-22">Genotypic data were further verified by the correspondence of the clusters obtained using PCA with known information about the germplasm. The very tight clustering of 16 control ‘Adirondack’ samples further demonstrated genotyping data integrity with negligible or very low level of technical variation across the 16 genotyping batches. The remainder of the tetraploid clones clustered into a cohesive group while the diploid clones formed three distinct groups, the sequenced Group Phureja clone DM clustering with the closely related Group Phureja diploid cultivar ‘Mayan Gold’ as expected (Figure S2).</p><p id="p-23">The germplasm panel displays a broad range of genotypic and phenotypic diversity, reflecting the diverse geographic origins, release dates, and market classes of the selected potato varieties and breeding clones. Distribution of minor allele frequencies (MAF) across all tetraploid genotypes is shown in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>. The genotype panel showed enrichment for common SNPs with higher MAF (&gt;0.1) values, desirable for performing GWAS. Genetic diversity in the panel was assessed using estimates of marker PIC values. The PIC values of the SolCAP SNPs for the association panel ranged from 0.06 to 0.50 with a mean value of 0.34 (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). <a id="xref-ref-52-5" class="xref-bibr" href="#ref-52">Stich <em>et al.</em> (2013)</a> reported a similar average PIC value of 0.35 in a study involving cultivated European tetraploid material genotyped with the same panel. The SolCAP SNP platform is derived using SNPs from only six potato cultivars (<a id="xref-ref-21-3" class="xref-bibr" href="#ref-21">Hamilton <em>et al.</em> 2011</a>), which could possibly cause ascertainment bias and limit its applicability in analyzing the germplasm which is not well represented by these six cultivars. However, in the current analysis the majority of SNPs were highly polymorphic with 63.5% showing PIC values above 0.35 and none falling below 0.05 (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>). The observed PIC values indicate high levels of polymorphism in European potato cultivars further validating the suitability of the deployed SNP platform for genetic studies on cultivated potato. Among the six cultivars used in developing the SNP platform, only one (Bintje) is from Europe, the remainder being North American. Potato is believed to have been introduced into North America from Europe (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Love 1999</a>), which explains the good representation and sharing of SNP alleles between European and North American cultivars. Moreover, <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Uitdewilligen <em>et al.</em> (2013)</a> reported that more than half of the common SNPs identified in a panel of 83 tetraploid potato cultivars were discovered by sequencing any three random cultivars. Given its history the SNP array used here may well display ascertainment bias for a very diverse set of potato germplasm including non-cultivated material however the inclusion of six diverse cultivars in its design makes it eminently suitable for genotyping a wide cultivated potato population as in this study.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/10/3185/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Minor Allele Frequency (MAF) distribution of 5,718 SolCAP SNPs in 341 tetraploid genotypes." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-551705518" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Minor Allele Frequency (MAF) distribution of 5,718 SolCAP SNPs in 341 tetraploid genotypes.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/10/3185/F1.medium.gif" width="440" height="397"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/8/10/3185/F1.medium.gif" width="440" height="397"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/10/3185/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/10/3185/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491059" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-24" class="first-child">Minor Allele Frequency (MAF) distribution of 5,718 SolCAP SNPs in 341 tetraploid genotypes.</p><div class="sb-div caption-clear"></div></div></div><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/10/3185/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Distribution of polymorphic information content (PIC) values calculated for 341 tetraploid genotypes across 5,718 SolCAP SNPs." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-551705518" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Distribution of polymorphic information content (PIC) values calculated for 341 tetraploid genotypes across 5,718 SolCAP SNPs.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/10/3185/F2.medium.gif" width="440" height="389"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/8/10/3185/F2.medium.gif" width="440" height="389"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/10/3185/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/10/3185/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491045" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-25" class="first-child">Distribution of polymorphic information content (PIC) values calculated for 341 tetraploid genotypes across 5,718 SolCAP SNPs.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-11" class="subsection"><h3>Phenotyping</h3><p id="p-26">The association panel was phenotyped for 20 quantitative traits in up to eight environments (Cambridge Nitrogen plus-2012, York Nitrogen plus-2012, Cambridge Nitrogen minus-2012, York Nitrogen minus-2012, Cambridge Nitrogen plus-2013, York Nitrogen plus-2013, Cambridge Nitrogen minus-2013, York Nitrogen minus-2013). A ‘pairs’ plot showing a scatterplot of the relationship between traits, the distribution of individual traits and the correlation between traits is presented in Figure S3. The residuals for all traits were normally distributed (data not shown). <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a> shows a biplot for the first 2 dimensions for the PCA performed on the full set of 20 traits. The first two axes account for 17.4% and 12.8% of the variation in the data. The colors denote the subpopulations identified by nMDS analysis (described in the subsequent sections). <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a> clearly shows the effects of breeding for different traits for varied end-uses over the last more than 100 years, for example, dry matter and yield both have high but quite different correlations with the first two principal components. There is no clear or obvious relationship between any of the traits and the nMDS-derived subpopulations.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/10/3185/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Biplot of the first two significant components obtained by applying PCA to the BLUEs of 20 traits for 290 tetraploid clones. Genotypes are colored according to the nMDS-based subpopulations." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-551705518" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Biplot of the first two significant components obtained by applying PCA to the BLUEs of 20 traits for 290 tetraploid clones. Genotypes are colored according to the nMDS-based subpopulations.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/10/3185/F3.medium.gif" width="440" height="436"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/8/10/3185/F3.medium.gif" width="440" height="436"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/10/3185/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/10/3185/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491110" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-27" class="first-child">Biplot of the first two significant components obtained by applying PCA to the BLUEs of 20 traits for 290 tetraploid clones. Genotypes are colored according to the nMDS-based subpopulations.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-12" class="subsection"><h3>Assessment of Genome-wide Linkage Disequilibrium</h3><p id="p-28">Linkage disequilibrium in the association panel was estimated using Pearson’s r<sup>2</sup> statistic using pairwise combinations of SNPs present across all 12 chromosomes. Previously LD in potato has been assessed using a varying level of structural units ranging from a few fragments per chromosome (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Simko <em>et al.</em> 2006</a>) to genome-wide superscaffolds (<a id="xref-ref-52-6" class="xref-bibr" href="#ref-52">Stich <em>et al.</em> 2013</a>). A more recent study (<a id="xref-ref-61-2" class="xref-bibr" href="#ref-61">Vos <em>et al.</em> 2017</a>) estimated short-range LD separately for pericentromeric heterochromatin and chromosomal arms. Here, we have estimated extent of LD in different regions (short and long arms individually as well as combined, and pericentromeric heterochromatin) of each chromosome as well as the whole chromosome. Marker positions in these subsets of the genome were obtained from <a id="xref-ref-47-4" class="xref-bibr" href="#ref-47">Sharma <em>et al.</em> (2013)</a> and a list of markers included in these specific regions is provided in Table S5.</p><p id="p-29">The r<sup>2</sup><sub>max,90</sub>, LD<sub>1/2max,90</sub> and LD<sub>1/10,90</sub> estimates for the different subsets of the genome are provided in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>. The trends observed among 12 chromosomes displayed a moderate LD decay in potato. Inter-chromosomal differences in LD decay patterns were also visible with each chromosome showing a different physical distance range exceeding which the LD decayed below the most commonly used significant threshold value of 0.1 (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">D’hoop <em>et al.</em> 2010</a>; <a id="xref-ref-52-7" class="xref-bibr" href="#ref-52">Stich <em>et al.</em> 2013</a>). The minimum and maximum LD<sub>1/10,90</sub> limits for short and long chromosomal arms were different and also differed by chromosome. However, the overall LD<sub>1/10,90</sub> for short and long arms and the combined euchromatin were comparable <em>viz</em>. 2.52 Mb, 2.76 Mb and 2.73 Mb, respectively. LD decay plots for short and long chromosomal arms for all 12 chromosomes are shown in Figure S4 and Figure S5, respectively. LD decay in the pericentromeric heterochromatin for all chromosomes was negligible due to the suppression of recombination in this fraction of the genome, therefore, it was not possible to model LD decay for pericentromeric heterochromatin at the set thresholds as also reported by <a id="xref-ref-61-3" class="xref-bibr" href="#ref-61">Vos <em>et al.</em> (2017)</a>. Despite this insignificant LD decay in heterochromatin, the overall LD<sub>1/10,90</sub> calculated at whole chromosome level (3.27 Mb) was within twofold limits of that observed for the euchromatin (2.73 Mb). Chromosome-scale LD blocks and decay patterns for all 12 chromosomes are shown in <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>; and combined LD decay plots over all chromosomes for the analyzed subsets of the genome are presented in <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>. The extended spread of the points, as compared to the typical L-shaped LD decay patterns, in the higher LD range (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>) closely approximates the magnitude of the pericentromeric heterochromatin boundaries reported for all chromosomes (<a id="xref-ref-47-5" class="xref-bibr" href="#ref-47">Sharma <em>et al.</em> 2013</a>; Table S6). All chromosomes exhibited modest LD<sub>1/10,90</sub> at the whole chromosomal-region level, ranging from 2.54 Mb (Chromosome 9) to 4.68 Mb (Chromosome 3), whereas chromosome 8 displayed extremely conserved LD<sub>1/10,90</sub> of 20.04 Mb. <a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Berdugo-Cely <em>et al.</em> (2017)</a> also report slowest LD decay for chromosome 8 (LD<sub>1/10,90</sub> = 8 Mb) in their autotetraploid Andigena population (N = 652). <a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4</a> also shows that large LD blocks on all chromosomes are visible but the one present on chromosome 8 is the most prominent. Inheritance of such large ancestral LD block suggests chromosome 8 has undergone selection pressures which differ from other chromosomes in some as yet unknown way during the last 150 years of breeding. <a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Vos <em>et al.</em> (2015)</a> have reported chromosomal positions of new variants introduced since 1945 by introgression breeding. Although the numerical breakdown per chromosome is not provided, from the illustration provided (Figure 5 in <a id="xref-ref-62-2" class="xref-bibr" href="#ref-62">Vos <em>et al.</em> 2015</a>) it is evident that chromosome 8 has received the least (almost negligible) number of introgressed segments/variants in the last six decades of potato breeding.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/491032/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/491032/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Extent of LD decay in different regions of the 12 chromosomes</span><div class="sb-div caption-clear"></div></div></div><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/10/3185/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Upper panels: Linkage disequilibrium (LD) measure r2 in the association panel plotted vs. the physical map distance (Mb) between pairs of SNPs located on the whole chromosomal region for all 12 chromosomes. The trend line of the nonlinear quantile regression of r2 (90th percentile) vs. the physical map distance between the SNP markers is depicted in red, dashed blue line depicts the standard LD decay threshold (r2 = 0.1). Lower panels: Genome-wide LD scans for all 12 chromosomes using 5,718 SNPs. Red areas show regions with higher LD." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-551705518" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Upper panels: Linkage disequilibrium (LD) measure r&lt;sup&gt;2&lt;/sup&gt; in the association panel plotted &lt;em&gt;vs.&lt;/em&gt; the physical map distance (Mb) between pairs of SNPs located on the whole chromosomal region for all 12 chromosomes. The trend line of the nonlinear quantile regression of r&lt;sup&gt;2&lt;/sup&gt; (90&lt;sup&gt;th&lt;/sup&gt; percentile) &lt;em&gt;vs.&lt;/em&gt; the physical map distance between the SNP markers is depicted in red, dashed blue line depicts the standard LD decay threshold (r&lt;sup&gt;2&lt;/sup&gt; = 0.1). Lower panels: Genome-wide LD scans for all 12 chromosomes using 5,718 SNPs. Red areas show regions with higher LD.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/10/3185/F4.medium.gif" width="437" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/8/10/3185/F4.medium.gif" width="437" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/10/3185/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/10/3185/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/490993" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-35" class="first-child">Upper panels: Linkage disequilibrium (LD) measure r<sup>2</sup> in the association panel plotted <em>vs.</em> the physical map distance (Mb) between pairs of SNPs located on the whole chromosomal region for all 12 chromosomes. The trend line of the nonlinear quantile regression of r<sup>2</sup> (90<sup>th</sup> percentile) <em>vs.</em> the physical map distance between the SNP markers is depicted in red, dashed blue line depicts the standard LD decay threshold (r<sup>2</sup> = 0.1). Lower panels: Genome-wide LD scans for all 12 chromosomes using 5,718 SNPs. Red areas show regions with higher LD.</p><div class="sb-div caption-clear"></div></div></div><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/10/3185/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Linkage disequilibrium (LD) measure r2 plotted vs. the physical map distance (Mb) between pairs of SNPs in the association panel located on (A) chromosomal short arm, (B) chromosomal long arm, (C) both arms combined (euchromatin), and (D) whole chromosome region. The illustrations include combined analyses for all 12 chromosomes except chromosomal short arm for which chromosome 2 is excluded as it is pericentric. The trend line of the nonlinear quantile regression of r2 (90th percentile) vs. the physical map distance between the SNP markers is depicted in red, dashed blue line depicts the standard LD decay threshold (r2 = 0.1)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-551705518" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Linkage disequilibrium (LD) measure r&lt;sup&gt;2&lt;/sup&gt; plotted &lt;em&gt;vs.&lt;/em&gt; the physical map distance (Mb) between pairs of SNPs in the association panel located on (A) chromosomal short arm, (B) chromosomal long arm, (C) both arms combined (euchromatin), and (D) whole chromosome region. The illustrations include combined analyses for all 12 chromosomes except chromosomal short arm for which chromosome 2 is excluded as it is pericentric. The trend line of the nonlinear quantile regression of r&lt;sup&gt;2&lt;/sup&gt; (90&lt;sup&gt;th&lt;/sup&gt; percentile) &lt;em&gt;vs.&lt;/em&gt; the physical map distance between the SNP markers is depicted in red, dashed blue line depicts the standard LD decay threshold (r&lt;sup&gt;2&lt;/sup&gt; = 0.1).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/10/3185/F5.medium.gif" width="440" height="101"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/8/10/3185/F5.medium.gif" width="440" height="101"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/10/3185/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/10/3185/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491165" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-36" class="first-child">Linkage disequilibrium (LD) measure r<sup>2</sup> plotted <em>vs.</em> the physical map distance (Mb) between pairs of SNPs in the association panel located on (A) chromosomal short arm, (B) chromosomal long arm, (C) both arms combined (euchromatin), and (D) whole chromosome region. The illustrations include combined analyses for all 12 chromosomes except chromosomal short arm for which chromosome 2 is excluded as it is pericentric. The trend line of the nonlinear quantile regression of r<sup>2</sup> (90<sup>th</sup> percentile) <em>vs.</em> the physical map distance between the SNP markers is depicted in red, dashed blue line depicts the standard LD decay threshold (r<sup>2</sup> = 0.1).</p><div class="sb-div caption-clear"></div></div></div><p id="p-37">Discrepancies in LD estimates could arise from differences in the type of populations under examination, spacing and type of markers used, genotyping methods, physical/genetic distance units covered and the significance thresholds used for defining the LD decay. For these reasons a direct comparison of LD patterns and estimates observed in previous potato studies is not feasible. However, overall LD trends observed in different studies can still be compared for drawing broad inferences. <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">D’hoop <em>et al.</em> (2010)</a> and <a id="xref-ref-50-2" class="xref-bibr" href="#ref-50">Simko <em>et al.</em> (2006)</a> reported decay of LD to r<sup>2</sup> equalling 0.1 within a distance of 5 cM and 10 cM, respectively. In our study, overall LD<sub>1/10,90</sub> for euchromatin and whole chromosomal regions were 2.73 Mb and 3.27 Mb, respectively. Relating physical size of the assembled potato genome (727 Mb; <a id="xref-ref-41-3" class="xref-bibr" href="#ref-41">Potato Genome Sequencing Consortium 2011</a>) and only the euchromatin (∼398 Mb, Table S6; <a id="xref-ref-47-6" class="xref-bibr" href="#ref-47">Sharma <em>et al.</em> 2013</a>) to a representative genetic map in the order of 754 cM (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Prashar <em>et al.</em> 2014</a>), 1 Mb corresponds to about 1 cM and 2 cM using the respective scales of the genome. Considering these estimates, the extent of LD decay observed in our study is comparable to those reported by <a id="xref-ref-9-3" class="xref-bibr" href="#ref-9">D’hoop <em>et al.</em> (2010)</a> and <a id="xref-ref-50-3" class="xref-bibr" href="#ref-50">Simko <em>et al.</em> (2006)</a>. Recently <a id="xref-ref-61-4" class="xref-bibr" href="#ref-61">Vos <em>et al.</em> (2017)</a> suggested LD<sub>1/2max,90</sub> as an unbiased estimator for comparing extent of LD decay across different studies. The implementation of the same estimator in the current study facilitates direct comparison of LD decay estimates with those obtained by <a id="xref-ref-61-5" class="xref-bibr" href="#ref-61">Vos <em>et al.</em> (2017)</a>. These authors studied LD decay rates for different age (year of release) potato groups and reported a decline in LD over the last century from an LD<sub>1/2max,90</sub> of 1.5 Mb in cultivars released before 1945 to 0.6 Mb in the recent cultivars released after 2005. The share of cultivars for these two age categories in their panel was 8.4% and 47.5% respectively as compared to 10% and 12% for the same categories in our association panel. Considering the minority of the cultivars from the latter category, the extent of LD decay (overall euchromatin LD<sub>1/2max,90</sub> = 0.91 Mb; <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>) observed in our collection of genotypes corroborates well and lies within the range of LD for different age groups as reported by <a id="xref-ref-61-6" class="xref-bibr" href="#ref-61">Vos <em>et al.</em> (2017)</a>. These authors also conclude that in general background levels of LD are reached at a physical distance ranging from 2 to 4 Mb which are also comparable to those obtained here.</p><p id="p-38"><a id="xref-ref-52-8" class="xref-bibr" href="#ref-52">Stich <em>et al.</em> (2013)</a> reported a sharp LD decay to an r<sup>2</sup> value of 0.1 within a distance of 275 bp. This is in steep contrast to the findings observed in the current study as well as to all other previous reports in potato. Potato is propagated vegetatively and cultivars are not separated from their progenitors by many meiotic generations, for example, the maximum number reported by <a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Gebhardt <em>et al.</em> (2004)</a> and <a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">D’hoop <em>et al.</em> (2008)</a> was 5 and 10 respectively. This peculiar feature present in potato also contradicts the very rapid decay in LD observed by <a id="xref-ref-52-9" class="xref-bibr" href="#ref-52">Stich <em>et al.</em> (2013)</a>. <a id="xref-ref-61-7" class="xref-bibr" href="#ref-61">Vos <em>et al.</em> (2017)</a> attributed contrasting LD estimates reported by <a id="xref-ref-52-10" class="xref-bibr" href="#ref-52">Stich <em>et al.</em> (2013)</a> to their use of average r<sup>2</sup> as an LD estimator combined with non-linear regression. In addition to the LD estimators described above, we also estimated LD decay using average r<sup>2</sup> and non-linear regression (data not shown) as implemented by <a id="xref-ref-52-11" class="xref-bibr" href="#ref-52">Stich <em>et al.</em> (2013)</a>. Since LD estimates by these authors were derived from all genomic superscaffolds (euchromatin as well as heterochromatin) using the same SNP array, the chromosome-scale LD estimates obtained here using the alternative method as described above can be compared to the results reported by <a id="xref-ref-52-12" class="xref-bibr" href="#ref-52">Stich <em>et al.</em> (2013)</a>. For our panel overall chromosome-scale LD<sub>1/10</sub> was 1.09 Mb which despite being derived from average r<sup>2</sup> fitted using non-linear regression is still within reasonable LD decay limits expected for potato but much slower than that reported by <a id="xref-ref-52-13" class="xref-bibr" href="#ref-52">Stich <em>et al.</em> (2013)</a>. These observations in combination with <a id="xref-ref-61-8" class="xref-bibr" href="#ref-61">Vos <em>et al.</em> (2017)</a>, derived from two different measures of LD estimation, contradict the claims made by <a id="xref-ref-52-14" class="xref-bibr" href="#ref-52">Stich <em>et al.</em> (2013)</a> and further suggest the discrepancy in the latter study could be due to the use of a highly diverse and much smaller (N = 36) set of tetraploid genotypes as compared to a larger collection of genotypes deployed in the other studies.</p></div><div id="sec-13" class="subsection"><h3>Analysis of Kinship and Population Structure</h3><p id="p-39">Genotyping was carried out for 351 tetraploid and 7 diploid potato genotypes. Initial data quality checks revealed ten pairs of tetraploid genotypes showing more than 99% similarity. These genotypes also contained those clones which during the roguing of field trials had shown morphological features incompatible with those expected from previous knowledge. The identical pairs were DNA fingerprinted with SSRs and only the individuals, one per pair, for which phenotypic assessments and fingerprinting patterns matched with the potato database were retained, thereby, reducing the net number of genotypes in the tetraploid panel from 351 to 341.</p><p id="p-40">The individual relatedness between genotypes in the revised panel was estimated using a genomic relationship matrix. Kinship analysis assigned more closely related lines such as those belonging to individual breeding programs and/or possessing resistance to specific pest or disease (potato cyst nematode, virus etc.) into ten smaller clusters (hereafter referred to as ‘Kinship groups’). Although the categorization of clones into distinct groups is subjective, the clustering of the panel into ten kinship groups is supported by the hierarchical clustering derived dendrogram (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>). Kinship groups 1 and 2 together are most divergent from the rest of the tetraploid germplasm and quite different from each other. Kinship group 1 largely comprises cultivars from the USA and Canada, whereas group 2 is mainly a collection of ‘salad’ and early potato varieties from various European countries. Regarding the other groups which show quite weak differentiation overall, Kinship groups 3-5 are quite similar comprising mainly Dutch, UK and German cultivars. Kinship group 6 is a small collection of mainly very old UK ‘heritage’ cultivars, as well as two ‘russetted’ US cultivars. Kinship groups 7 and 8 predominantly comprise older UK potato varieties, with groups 9 and 10 being mainly more modern UK and Irish cultivars and breeding lines carrying potato cyst nematode and late blight resistance.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/10/3185/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Heatmap displaying relationships among the 341 lines present in the association panel. The red diagonal represents perfect relationship of each line with itself; the symmetric off-diagonal elements represent relationship for pairs of lines. The blocks of warmer colors on the diagonal show clusters of closely related lines. The adjoining dendrogram illustrates Kinship groups identified in the panel. The details of members of these Kinship groups are present in Table S2." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-551705518" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Heatmap displaying relationships among the 341 lines present in the association panel. The red diagonal represents perfect relationship of each line with itself; the symmetric off-diagonal elements represent relationship for pairs of lines. The blocks of warmer colors on the diagonal show clusters of closely related lines. The adjoining dendrogram illustrates Kinship groups identified in the panel. The details of members of these Kinship groups are present in Table S2.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/10/3185/F6.medium.gif" width="440" height="343"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/8/10/3185/F6.medium.gif" width="440" height="343"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/10/3185/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/10/3185/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491015" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-41" class="first-child">Heatmap displaying relationships among the 341 lines present in the association panel. The red diagonal represents perfect relationship of each line with itself; the symmetric off-diagonal elements represent relationship for pairs of lines. The blocks of warmer colors on the diagonal show clusters of closely related lines. The adjoining dendrogram illustrates Kinship groups identified in the panel. The details of members of these Kinship groups are present in Table S2.</p><div class="sb-div caption-clear"></div></div></div><p id="p-42">The level of population stratification was examined using a number of nMDS solutions ranging from 1 to 20 dimensions. The Screeplot in Figure S6 illustrates the badness-of-fit measures for fitting different nMDS solutions. A 5-dimension solution was deemed closest to the point of inflection (elbow junction) with a criterion of representing the full data set in the smallest possible number of dimensions. At the chosen nMDS solution, there was no clear visual separation of clones into distinct subpopulations indicating a lack of strong population structure within the panel (Figure S7). However, nMDS analysis was still able to detect subtle relationships among the clones, possibly resulting from the weak underlying stratification among genotypes, as depicted in the nMDS-based dendrogram (<a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7</a>) showing clustering of clones into five subpopulations (Q). The nMDS-based dendrogram revealed two large clusters where one cluster comprised four subpopulations while the other formed a single subpopulation. These five putative subpopulations also showed low levels of population differentiation (<em>F</em>st, Table S7) which further supports the absence of any strong population structure in the panel used here. These findings are in accordance with the observations made in previous studies (<a id="xref-ref-10-4" class="xref-bibr" href="#ref-10">D’hoop <em>et al.</em> 2008</a>; <a id="xref-ref-15-3" class="xref-bibr" href="#ref-15">Gebhardt <em>et al.</em> 2004</a>; <a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Malosetti <em>et al.</em> 2007</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Simko <em>et al.</em> 2004a</a>; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Simko <em>et al.</em> 2004b</a>; <a id="xref-ref-52-15" class="xref-bibr" href="#ref-52">Stich <em>et al.</em> 2013</a>) and further validate the notion that cultivated potato lacks a distinct population structure.</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/10/3185/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Heatmap and hierarchical clustering of the association panel using a 5-dimension solution from the nMDS (non-metric multidimensional scaling) analysis for adequately representing the panel members (341 clones) in the optimum number of dimensions (subpopulations). The details of members of the identified five subpopulations are present in Table S2." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-551705518" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Heatmap and hierarchical clustering of the association panel using a 5-dimension solution from the nMDS (non-metric multidimensional scaling) analysis for adequately representing the panel members (341 clones) in the optimum number of dimensions (subpopulations). The details of members of the identified five subpopulations are present in Table S2.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/10/3185/F7.medium.gif" width="325" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/8/10/3185/F7.medium.gif" width="325" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/10/3185/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/10/3185/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/491007" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-43" class="first-child">Heatmap and hierarchical clustering of the association panel using a 5-dimension solution from the nMDS (non-metric multidimensional scaling) analysis for adequately representing the panel members (341 clones) in the optimum number of dimensions (subpopulations). The details of members of the identified five subpopulations are present in Table S2.</p><div class="sb-div caption-clear"></div></div></div><p id="p-44">The correspondence of kinship groups to the nMDS-derived subpopulations was very high with six kinship groups nesting into their respective subpopulations as single large blocks. Members of the remaining kinship groups did show their assignment to separate subpopulations but still the allocation was in cohesive blocks of at least three genotypes. This allocation of members of individual kinship groups to more than one subpopulation could be attributed to their shared lineage. The details of individual members belonging to these kinship groups and subpopulations are present in Table S2a. The germplasm was also analyzed using phylogenetic analysis (<a id="xref-ref-39-3" class="xref-bibr" href="#ref-39">Nei 1972</a>) by including all tetraploid and diploid genotypes. The clustering of tetraploid clones corroborated well with nMDS-derived subpopulations as illustrated in Figure S8. Some advanced breeding lines (<em>e.g.</em>, ‘GL’ series) formed distinct clusters and others were interspersed among the cultivar groups. The four diploid wild genotypes clustered into a separate group, which surprisingly included one tetraploid clone, 3053-18, a ‘late blight R-gene differential’ for the resistance gene R5. This result is likely due to close genetic relationship of 3053-18 to the late blight resistant wild species in this cluster, close relatives of which were used to introgress late blight resistance (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Trognitz and Trognitz 2007</a>). The remaining three diploid genotypes (DM, <em>S. tuberosum</em> group <em>Phureja</em> cv. MayanGold and <em>S. stenotomum</em>) were clustered with one of the tetraploid clusters predominantly containing older (mainly salad) cultivars and advanced breeding lines.</p><p id="p-45">Population structure was also assessed using PCA (Figure S9) and STRUCTURE (Figure S10). Population structure obtained using PCA and nMDS was compared to visualize patterns of variation in terms of ‘country of origin’ and ‘year of release’. The assessment using ‘year of release’ failed to reveal any specific patterns for both PCA as well as nMDS (data not shown). This observation contrasts with that of <a id="xref-ref-9-4" class="xref-bibr" href="#ref-9">D’hoop <em>et al.</em> (2010)</a> who reported subpopulations differing in year of market release in a set of 430 tetraploid cultivars. <a id="xref-ref-52-16" class="xref-bibr" href="#ref-52">Stich <em>et al.</em> (2013)</a> also did not detect any correlation between population structure and the cultivar release date. They attributed this difference to the lack of enough statistical power in their analysis as their panel comprised only 36 tetraploid cultivars, a weakness that does not apply to the current study.</p><p id="p-46">Visualization of nMDS-based population structure in terms of ‘country of origin’ of each genotype yielded a more harmonious representation of the germplasm as compared to the PCA. The subpopulations were not clearly defined but a country-specific pattern was apparent (Figure S11). One subpopulation (olive green, bottom-most) comprised modern UK cultivars while another subpopulation (red, right-most) contained older UK varieties. Two interspersed subpopulations at the center (cyan and brown) primarily contained ‘mainland’ European germplasm; and the remaining one (blue, upper-left) included mainly international non-European material. The analysis suggests the majority of the UK potato cultivars represent more diverse genetic pools than rest of the European and non-European cultivated potato germplasm with limited admixture. Similarly in other studies, population stratification within a region has been reported. For example, two subpopulations of European winter wheat from multiple national programs (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Tommasini <em>et al.</em> 2007</a>) and four subpopulations from a collection of US soft winter wheat cultivars with release dates spanning approximately 30 years (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Breseghello and Sorrells 2006</a>) were previously reported. In contrast to nMDS, no region-based grouping patterns were visible in clusters generated using first two PCA eigenvectors (Figure S12). Previous studies have also shown nMDS as a more appropriate method for visualizing population structure as it can capture the underlying population stratification with the first few dimensions that may not be inferred with the same number of eigenvectors using PCA or principal co-ordinates analysis (PCO) (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Miclaus <em>et al.</em> 2009</a>).</p></div><div id="sec-14" class="subsection"><h3>Evaluation of Population Structure Estimation Methods for GWAS</h3><p id="p-47">All GWAS analyses described here as well as in the subsequent sections were performed on 290 tetraploid clones. This reduction in the association panel was due to clone identity issues, as described in the preceding section, as well as loss of some genotypes due to lack of enough seed tubers for successive field trials. Seed tubers for these genotypes were not supplemented from other sources to avoid inducing any bias arising due to variation in seed tuber generation as the seed material for the reported study was generated from a designated seed production site in the preceding year. Regarding markers, a total of 5,157 SNPs were retained for GWAS after filtering for missing data (&gt;= 20%) and minor allele frequency (MAF=&lt;0.05) thresholds. Performance of GWAS models was evaluated using Q-Q plots of the expected <em>vs.</em> observed –log<sub>10</sub>(<em>p</em>) values and genomic control inflation factors (λ<sub>GC</sub>) achieved for each ‘trait x model’ combination. The success (ranking) of GWAS models in controlling the overall genome-wide type I error rate (<em>i.e.</em>, keeping the genomic control inflation factor to the desired level of λ<sub>GC</sub>∼1), summarized over 20 traits, is shown in <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/491080/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/491080/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Ranking of GWAS models derived using Genomic Control Inflation Factor (λ<sub>GC</sub>) metric summarized over 20 traits</span><div class="sb-div caption-clear"></div></div></div><p id="p-50">Various methods have been used for controlling the confounding effects of population structure for GWAS. Here we examined GWAS models with population structure estimates (Q matrices) from three different methods <em>viz</em>. STRUCTURE, principal component analysis and non-metric multidimensional scaling. <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a> displays trait-wise λ<sub>GC</sub> estimates for Q and QK GWAS models involving Q matrices from above three methods and their overall rankings are provided in <a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>. In Q models, STRUCTURE was marginally more effective (8 traits) in controlling <em>p</em>-value inflation than nMDS and PCA while the latter two performed better for 6 traits each. Evaluation of these models as QK models (<em>i.e.</em>, with correction for population structure as well as kinship) controlled the <em>p</em>-value inflation observed in Q models with different methods for population structure estimation and brought λ<sub>GC</sub> close to 1. This further implies that with the inclusion of correction for kinship effects, the method used for population structure estimation is not very critical for performing GWAS in potato. Figure S13 and Figure S14 illustrate Q-Q plots obtained from Q and QK models tested using three different Q-matrices, respectively. Overall the results indicate that no single method was able to capture population confounding effects better than the others for all 20 traits. Nevertheless, when overall model performance was taken into consideration, nMDS seemed to provide slight advantage over PCA and STRUCTURE (<a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2A</a>) and was selected for all subsequent assessment of GWAS models.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/491141/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/491141/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Genomic Control Inflation Factor (λ<sub>GC</sub>) analyses of GWAS models as a function of three different methods<sup><em><sup>a</sup></em></sup> for adjusting population structure</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-15" class="subsection"><h3>Evaluation of Different GWAS Models</h3><p id="p-54">Four principal GWAS models (Naïve, K, Q and QK) were investigated for all 20 traits. The λ<sub>GC</sub> values and rankings for these four models achieved for all 20 traits are presented in <a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>. Combined ranking summary for main GWAS models is provided in <a id="xref-table-wrap-2-4" class="xref-table" href="#T2">Table 2B</a>. The efficiency of K and QK models, in keeping <em>p</em>-values close to the expected (λ<sub>GC</sub>∼1), were comparable where K model was most effective for 7 traits and QK for 13 traits. Furthermore, comparison of the individual components of the QK model revealed that correction for Kinship was more efficient in overcoming the germplasm confounding effects than adjusting for population structure. As compared to K Model, Q Model never ranked first or second for any of the 20 traits and performed close to K and QK models for only one trait (Dormancy break, <a id="xref-table-wrap-4-2" class="xref-table" href="#T4">Table 4</a>). Q model performed better (ranked third) than Naïve model for all but one trait (Stolon attachment) and λ<sub>GC</sub> for both models were always inferior (larger) to the K model. <a id="xref-fig-8-1" class="xref-fig" href="#F8">Figure 8</a> displays Q-Q plots obtained for four principal GWAS models for all 20 traits. These plots further reveal that Q-Q plots for K and QK models were largely indistinguishable for all traits and wherever one model performed better than the other, it only improved the analysis marginally. This further implies that kinship alone is sufficient to control type I error (false positives) for conducting GWAS in cultivated potato. Also, the ineffectiveness of Q model in controlling <em>p</em>-value inflation further confirms lack of significant population structure in potato.</p><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/490984/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/490984/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">Genomic Control Inflation Factor (λ<sub>GC</sub>) analyses of four principal GWAS models</span><div class="sb-div caption-clear"></div></div></div><div id="F8" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/10/3185/F8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Q-Q plots comparing the inflation of p-values for the four principal GWAS models for all 20 traits using the additive marker model. Red circles: Na&#xEF;ve model; Green squares: K model; Blue diamonds: Q model; and Black triangles: QK model. Red line indicates p-values under the expected normal distribution." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-551705518" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Q-Q plots comparing the inflation of &lt;em&gt;p&lt;/em&gt;-values for the four principal GWAS models for all 20 traits using the additive marker model. Red circles: Na&#xEF;ve model; Green squares: K model; Blue diamonds: Q model; and Black triangles: QK model. Red line indicates &lt;em&gt;p&lt;/em&gt;-values under the expected normal distribution.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 8" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/10/3185/F8.medium.gif" width="377" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 8" src="https://www.g3journal.org/content/ggg/8/10/3185/F8.medium.gif" width="377" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/10/3185/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 8" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/10/3185/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/490987" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 8</span> <p id="p-56" class="first-child">Q-Q plots comparing the inflation of <em>p</em>-values for the four principal GWAS models for all 20 traits using the additive marker model. Red circles: Naïve model; Green squares: K model; Blue diamonds: Q model; and Black triangles: QK model. Red line indicates <em>p</em>-values under the expected normal distribution.</p><div class="sb-div caption-clear"></div></div></div><p id="p-57">A careful assessment of kinship and population structure in GWAS studies is required to avoid spurious associations arising from systematic differences in allele frequencies due to the difference in sample ancestries. However, the results described above indicate that this may only be critical when the trait in question is also confounded with the kinship and/or population structure present in the germplasm. Overcompensation of mixed models for population structure and relatedness can lead to false negatives (type II error) as also reported by <a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Zhao <em>et al.</em> (2011)</a> in rice. In a study involving evaluation of GWAS models for 13 traits, <a id="xref-ref-44-5" class="xref-bibr" href="#ref-44">Rosyara <em>et al.</em> (2016)</a> report a preference for the QK model, however, trait specific model fitness details are not divulged. Results here instruct that a single model type is not suitable for all traits and emphasize the need to carefully select an optimal combination of covariates for each trait to avoid false associations as well as overfitting of the GWAS models. Overall, the results endorse greater significance for correcting kinship than population structure in GWAS for cultivated potato.</p></div><div id="sec-16" class="subsection"><h3>Model Validation Using GWAS Results</h3><p id="p-58">We demonstrate here the effectiveness of different models <em>viz</em>. Naïve, K, Q and QK for performing GWAS in potato and validate our findings using previously published reports. <a id="xref-fig-9-1" class="xref-fig" href="#F9">Figure 9</a> and <a id="xref-table-wrap-5-1" class="xref-table" href="#T5">Table 5</a> shows Manhattan plots and significantly associated SNPs identified around well reported QTL regions in potato. Only true marker-trait associations <em>i.e.</em>, (a) from models where <em>p</em>-value inflation was close to the expected normal value (λ<sub>GC</sub>∼1) and (b) which passed the set Bonferroni correction <em>p</em>-value (α = 0.05) threshold are included for reporting GWAS results. Effectiveness of K and QK models over Q and Naïve models in controlling type I error is clearly evident in the Manhattan plots (<a id="xref-fig-9-2" class="xref-fig" href="#F9">Figure 9</a>) and the results mainly follow the same pattern as revealed by Q-Q plots (<a id="xref-fig-8-2" class="xref-fig" href="#F8">Figure 8</a>) and λ<sub>GC</sub> values (<a id="xref-table-wrap-2-5" class="xref-table" href="#T2">Table 2</a>). Trait associations obtained from all four main models are listed in Table S8.</p><div id="F9" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/10/3185/F9.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparison of Manhattan plots for Na&#xEF;ve, K, Q and QK models. Significance threshold (black dashed line) is based on the genome-wide false positive rate (&#x3B1; = 0.05) for the Bonferroni correction method and the marker-trait associations (MTAs) crossing the set threshold are depicted in large red dots." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-551705518" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Comparison of Manhattan plots for Na&#xEF;ve, K, Q and QK models. Significance threshold (black dashed line) is based on the genome-wide false positive rate (&#x3B1; = 0.05) for the Bonferroni correction method and the marker-trait associations (MTAs) crossing the set threshold are depicted in large red dots.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 9" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/10/3185/F9.medium.gif" width="296" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 9" src="https://www.g3journal.org/content/ggg/8/10/3185/F9.medium.gif" width="296" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/10/3185/F9.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 9" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/10/3185/F9.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/490989" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 9</span> <p id="p-59" class="first-child">Comparison of Manhattan plots for Naïve, K, Q and QK models. Significance threshold (black dashed line) is based on the genome-wide false positive rate (α = 0.05) for the Bonferroni correction method and the marker-trait associations (MTAs) crossing the set threshold are depicted in large red dots.</p><div class="sb-div caption-clear"></div></div></div><div id="T5" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/491017/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/491017/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 5</span> <span class="caption-title">Significant marker-trait associations (MTAs) in the potato GWAS panel</span><div class="sb-div caption-clear"></div></div></div><p id="p-62">GWAS for Tuber Shape and Eye Depth co-identified ‘solcap_snp_c1_8019’ as the most significant association located at 48.9 Mb on chromosome 10. <a id="xref-ref-44-6" class="xref-bibr" href="#ref-44">Rosyara <em>et al.</em> (2016)</a> also detected the same SNP for these linked traits in their association panel using the same SNP array. Previous biparental mapping studies (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Li <em>et al.</em> 2005b</a>; <a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Prashar <em>et al.</em> 2014</a>; <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Śliwka <em>et al.</em> 2008</a>; <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Vaneck <em>et al.</em> 1994</a>) have also mapped major QTL for Tuber Shape and Eye Depth to the same locus on chromosome 10. <a id="xref-ref-42-3" class="xref-bibr" href="#ref-42">Prashar <em>et al.</em> (2014)</a> identified ‘solcap_snp_c1_8020’ as the main QTL SNP for this locus which we also report among the most significant associations detected here for Tuber Shape and Eye Depth (Table S8).</p><p id="p-63">Genetic mapping studies have previously reported a major locus affecting plant maturity on chromosome 5 (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Bradshaw <em>et al.</em> 2008</a>). Analysis of a proxy (for maturity) phenotype ‘Stolon Attachment’ detected a significant association (solcap_snp_c2_50302) on chromosome 5 around 0.5 Mb away from the <em>StCDF1</em> gene, identified as the gene underlying the maturity locus by <a id="xref-ref-25-3" class="xref-bibr" href="#ref-25">Kloosterman <em>et al.</em> (2013)</a>. Previous GWAS attempt (<a id="xref-ref-44-7" class="xref-bibr" href="#ref-44">Rosyara <em>et al.</em> 2016</a>) with the same SNP array did not detect any association for maturity, however a recent study (<a id="xref-ref-45-3" class="xref-bibr" href="#ref-45">Schmitz Carley <em>et al.</em> 2017</a>) with SolCAP 8K array does identify ‘solcap_snp_c2_50302’ as the most significant association for the maturity locus. Interestingly, analyses of Foliage and Plant Height also revealed the same SNP (solcap_snp_c2_50302) as the most significant association validating the confounded behavior of these traits with maturity.</p><p id="p-64">The <em>Y</em>-locus controlling the white-to-yellow flesh color in potato maps to chromosome 3 (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Bonierbale <em>et al.</em> 1988</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Jacobs <em>et al.</em> 1995</a>) and is believed to be mainly regulated by the β-carotene hydroxylase (<em>bch</em>) gene (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Kloosterman <em>et al.</em> 2010</a>). Out of the three copies of <em>bch</em> present on chromosome 3 (<a id="xref-ref-41-4" class="xref-bibr" href="#ref-41">Potato Genome Sequencing Consortium 2011</a>), one isoform (PGSC0003DMG400009501) is located at 44.1 Mb. In this study, a strong association for Flesh Color was observed at 49.4 Mb (solcap_snp_c2_20285) on Chromosome 3 with another significant association at 45.6 Mb (solcap_snp_c2_25653, Table S8), approximately 1.5 Mb away from the causal gene.</p><p id="p-65">Among the GWAS models examined here, Q, K and QK were able to reduce <em>p</em>-value inflation relative to the Naïve model, with K and QK further performing significantly better than the Q model. These trends were consistent in λ<sub>GC</sub> comparison, Q-Q plots and Manhattan plots. K and QK models corroborated well with each other, however, for their co-identified marker-trait associations <em>p</em>-values from the K model were generally (4 out of 6) lower and thus more significant than those reported by the QK model (<a id="xref-table-wrap-5-2" class="xref-table" href="#T5">Table 5</a>). Moreover, association for Plant Height was only detected using K model and its absence in the QK model perhaps indicates overfitting of the mixed model. Lack of significant associations as well as appearance of only marginally significant results for some of the traits (Plant Height and Stolon Attachment) could be attributed to low marker density of the SolCAP SNP array as also suggested by <a id="xref-ref-44-8" class="xref-bibr" href="#ref-44">Rosyara <em>et al.</em> (2016)</a>. These authors emphasized the importance of higher marker density and larger population size for future GWAS studies. Here we report that a careful selection of model parameters is equally critical for obtaining true associations and improving success rates of GWAS in potato.</p></div><div id="sec-17" class="subsection"><h3>Implications for GWAS in Potato</h3><p id="p-66">The genotypes evaluated here were targeted to diverse end-uses but the population structure analysis did not detect clear separation of cultivars into distinct subpopulations. This is largely expected in potato as breeders do not take population structure into account while choosing parents of a cross. Such practice, however, is reported to generate diverse levels of individual relatedness (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Garris <em>et al.</em> 2005</a>) as also observed in the examined association panel. The presence of a weak population structure in cultivated potato <em>per se</em> could be attributed to admixed populations and also possibly to introgression breeding for developing disease resistance using common resistant parents. Moreover, autotetraploidy and high levels of heterozygosity have a strong tendency to ‘blur’ differences between genotypes and contribute to weak population structuring. <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bouaziz <em>et al.</em> (2011)</a> compared several methods to counter population stratification for correcting different types of population structure simulated in their study and further opined that addressing populations with a weak population structure requires more careful analysis than with discrete populations. <a id="xref-ref-63-2" class="xref-bibr" href="#ref-63">Yu <em>et al.</em> (2006)</a> have described five categories to which population members can be assigned. Briefly, these include populations with minimal population structure or familial relatedness (type I), family-based samples (type II), samples with familial relationships within structured population (type III), samples with explicit population structure (type IV) and samples with very high levels of population structure coupled with diverse levels of familial relatedness (type V). It is also implied that while suitable methods for accounting for population structure and individual relatedness need to be implemented to control for false positives (type I error), excessive or unnecessary use of these measures could also lead to loss in statistical power. Considering the description above, the genotypes in our association panel are likely to fall somewhere between type II and III categories as a prominent individual relationship among the panel members was visible accompanied by the presence of weak population stratification. These findings are further supported by the evaluation of different GWAS models where K model performs significantly better than the Q model for all traits and the QK model is only marginally more effective (than K model) for 13 (out of 20) traits. Other studies (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Li <em>et al.</em> 2011</a>; <a id="xref-ref-44-9" class="xref-bibr" href="#ref-44">Rosyara <em>et al.</em> 2016</a>; <a id="xref-ref-63-3" class="xref-bibr" href="#ref-63">Yu <em>et al.</em> 2006</a>) have also reported greater improvement with K model relative to Naïve and Q model implying the efficacy of kinship over population structure in controlling confounding effects due to relatedness.</p></div></div><div class="section conclusions" id="sec-18"><h2 class="">Conclusions</h2><p id="p-67">The current study analyses LD in various subsets of the genome and reports a moderate level of LD decay in potato. <a id="xref-ref-52-17" class="xref-bibr" href="#ref-52">Stich <em>et al.</em> (2013)</a>, with their estimates of very rapid decay of LD, predict a low success in GWAS using an 8K SNP array on a panel of 300 genotypes and further suggest a requirement of about 3 million SNPs for enhanced success. LD decay estimates provided here suggest the suitability of the 8k array for performing GWAS in potato, although success will also depend on the trait being examined. GWAS with monogenic traits (<em>e.g.</em>, tuber shape) with large effect QTL will have greater success with 8k SNP array than with the polygenic traits (<em>e.g.</em>, yield) with complex architecture as observed in the current and previous studies (<a id="xref-ref-44-10" class="xref-bibr" href="#ref-44">Rosyara <em>et al.</em> 2016</a>). Moreover, analyzing pathogen specific resistance traits using 8K array will be challenging due to possible ascertainment bias. Nevertheless, <a id="xref-ref-33-4" class="xref-bibr" href="#ref-33">Lindqvist-Kreuze <em>et al.</em> (2014)</a> have suitably exploited this array for performing GWAS for late blight resistance in tetraploid potato. Previous GWAS efforts have identified significant associations by targeting candidate loci for the traits of interest (<a id="xref-ref-15-4" class="xref-bibr" href="#ref-15">Gebhardt <em>et al.</em> 2004</a>; <a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Li <em>et al.</em> 2005a</a>; <a id="xref-ref-35-3" class="xref-bibr" href="#ref-35">Malosetti <em>et al.</em> 2007</a>; <a id="xref-ref-40-3" class="xref-bibr" href="#ref-40">Pajerowska-Mukhtar <em>et al.</em> 2009</a>; <a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">Simko <em>et al.</em> 2004a</a>), by using a varying number of random AFLP markers (<a id="xref-ref-8-4" class="xref-bibr" href="#ref-8">D’hoop <em>et al.</em> 2014</a>; <a id="xref-ref-10-5" class="xref-bibr" href="#ref-10">D’hoop <em>et al.</em> 2008</a>) and more recently by employing a genome-wide SNP array (<a id="xref-ref-44-11" class="xref-bibr" href="#ref-44">Rosyara <em>et al.</em> 2016</a>; <a id="xref-ref-45-4" class="xref-bibr" href="#ref-45">Schmitz Carley <em>et al.</em> 2017</a>; <a id="xref-ref-33-5" class="xref-bibr" href="#ref-33">Lindqvist-Kreuze <em>et al.</em> 2014</a>). These findings also imply that the number of markers required for performing GWAS in potato may not be as high as predicted by <a id="xref-ref-52-18" class="xref-bibr" href="#ref-52">Stich <em>et al.</em> (2013)</a>, further suggesting that LD decay rates observed in their study are not typical for potato. Moreover, recently genotyping platforms with a larger number of SNPs (<a id="xref-ref-62-3" class="xref-bibr" href="#ref-62">Vos <em>et al.</em> 2015</a>; Illumina version 3 SNP array) have been developed. These marker-dense platforms would provide for a more accurate correction for different levels of relatedness in GWAS models and will further aid in identifying genomic regions that have been influenced by targeted breeding history for developing cultivars with enhanced traits.</p><p id="p-68">The primary aim of GWAS is to filter real marker-trait associations that arise from physical linkage of loci from spurious effects that result as a consequence of potential confounders such as kinship and population structure. The germplasm examined here shows multiple levels of relatedness with a prominent kinship and weak population structure, despite which the nMDS analysis was able to reveal the underlying patterns of subpopulations in the panel. These subpopulations showed an apparent structuring according to country of origin and further provide novel insights into the breeding strategies adopted in Europe. Evaluation of various GWAS models for 20 traits further demonstrated the importance of applying trait-specific models and exploiting different population structure methods for performing association mapping in potato. The main finding reveals kinship, rather than population structure, as the major factor controlling the level of spurious associations in cultivated potato.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-69">The authors thank the funding and support provided by the UK Technology Strategy Board (Project No: TP100963) and project consortium members, namely Cygnet Potato Breeders Ltd., McCain Foods (GB) Ltd., James Hutton Ltd. and Agriculture &amp; Horticulture Development Board Potatoes, UK. We gratefully acknowledge Doctors Kelci Miclaus (JMP Genomics), Luciano Silva (JMP Genomics), Roger Payne (Genstat), Jeffrey Endelman (GWASpoly) and Roeland Voorrips (fitTetra) for the advice and technical support received for using their respective software. We also acknowledge funding from the Scottish Government.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-14"><p id="p-70">Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.6716033">https://doi.org/10.25387/g3.6716033</a>.</p></li><li class="fn" id="fn-15"><p id="p-71"><em>Communicating editor: D. J. de Koning</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2018-05-02"><span class="received-label">Received </span>May 2, 2018.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2018-07-30"><span class="accepted-label">Accepted </span>July 30, 2018.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2018 Sharma <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3185.1" data-doi="10.1007/s00122-008-0925-x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Achenbach</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paulo</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ilarionova</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lubeck</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Strahwald</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Using SNP markers to dissect linkage disequilibrium at a major quantitative trait locus for resistance to the potato cyst nematode Globodera pallida on potato chromosome V.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">118</span>: <span class="cit-fpage">619</span>–<span class="cit-lpage">629</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-008-0925-x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DAchenbach%26rft.auinit1%253DU.%26rft.volume%253D118%26rft.issue%253D3%26rft.spage%253D619%26rft.epage%253D629%26rft.atitle%253DUsing%2BSNP%2Bmarkers%2Bto%2Bdissect%2Blinkage%2Bdisequilibrium%2Bat%2Ba%2Bmajor%2Bquantitative%2Btrait%2Blocus%2Bfor%2Bresistance%2Bto%2Bthe%2Bpotato%2Bcyst%2Bnematode%2BGlobodera%2Bpallida%2Bon%2Bpotato%2Bchromosome%2BV.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-008-0925-x%26rft_id%253Dinfo%253Apmid%252F19020852%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-008-0925-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19020852&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3185.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000262577300019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3185.2" data-doi="10.1371/journal.pone.0173039"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Berdugo-Cely</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valbuena</span>  <span class="cit-name-given-names">R. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sanchez-Betancourt</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barrero</span>  <span class="cit-name-given-names">L. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yockteng</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Genetic diversity and association mapping in the Colombian Central Collection of Solanum tuberosum L. Andigenum group using SNPs markers.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">e0173039</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0173039</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D12%26rft.spage%253De0173039%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0173039%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0173039&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3185.3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bonierbale</span>  <span class="cit-name-given-names">M. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Plaisted</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tanksley</span>  <span class="cit-name-given-names">S. D.</span></span></li></ol><cite>, <span class="cit-pub-date">1988</span> <span class="cit-article-title">RFLP Maps Based On A Common Set Of Clones Reveal Modes Of Chromosomal Evolution In Potato and Tomato.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">120</span>: <span class="cit-fpage">1095</span>–<span class="cit-lpage">1103</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DBonierbale%26rft.auinit1%253DM.%2BW.%26rft.volume%253D120%26rft.issue%253D4%26rft.spage%253D1095%26rft.epage%253D1103%26rft.atitle%253DRFLP%2BMaps%2BBased%2Bon%2Ba%2BCommon%2BSet%2Bof%2BClones%2BReveal%2BModes%2Bof%2BChromosomal%2BEvolution%2Bin%2BPotato%2Band%2BTomato.%26rft_id%253Dinfo%253Apmid%252F17246486%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjEyMC80LzEwOTUiO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzEwLzMxODUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3185.4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Botstein</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">White</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Skolnick</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">R. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1980</span> <span class="cit-article-title">Construction of A Genetic-Linkage Map in Man Using Restriction FragmentLength Polymorphisms.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">314</span>–<span class="cit-lpage">331</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DBotstein%26rft.auinit1%253DD.%26rft.volume%253D32%26rft.issue%253D3%26rft.spage%253D314%26rft.epage%253D331%26rft.atitle%253DConstruction%2Bof%2Ba%2Bgenetic%2Blinkage%2Bmap%2Bin%2Bman%2Busing%2Brestriction%2Bfragment%2Blength%2Bpolymorphisms.%26rft_id%253Dinfo%253Apmid%252F6247908%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=6247908&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3185.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1980JX05900002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3185.5" data-doi="10.1371/journal.pone.0028845"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bouaziz</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ambroise</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guedj</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Accounting for Population Stratification in Practice: A Comparison of the Main Strategies Dedicated to Genome-Wide Association Studies.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e28845</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0028845</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DBouaziz%26rft.auinit1%253DM.%26rft.volume%253D6%26rft.issue%253D12%26rft.spage%253De28845%26rft.epage%253De28845%26rft.atitle%253DAccounting%2Bfor%2Bpopulation%2Bstratification%2Bin%2Bpractice%253A%2Ba%2Bcomparison%2Bof%2Bthe%2Bmain%2Bstrategies%2Bdedicated%2Bto%2Bgenome-wide%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0028845%26rft_id%253Dinfo%253Apmid%252F22216125%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0028845&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22216125&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3185.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3185.6" data-doi="10.1007/s00122-007-0659-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bradshaw</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hackett</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pande</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Waugh</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bryan</span>  <span class="cit-name-given-names">G. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">QTL mapping of yield, agronomic and quality traits in tetraploid potato (Solanum tuberosum subsp tuberosum).</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">116</span>: <span class="cit-fpage">193</span>–<span class="cit-lpage">211</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-007-0659-1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DBradshaw%26rft.auinit1%253DJ.%2BE.%26rft.volume%253D116%26rft.issue%253D2%26rft.spage%253D193%26rft.epage%253D211%26rft.atitle%253DQTL%2Bmapping%2Bof%2Byield%252C%2Bagronomic%2Band%2Bquality%2Btraits%2Bin%2Btetraploid%2Bpotato%2B%2528Solanum%2Btuberosum%2Bsubsp.%2Btuberosum%2529.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-007-0659-1%26rft_id%253Dinfo%253Apmid%252F17938877%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-007-0659-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17938877&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3185.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000251797200004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3185.7" data-doi="10.1534/genetics.105.044586"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Breseghello</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Association mapping of kernel size and milling quality in wheat (Triticum aestivum L.) cultivars.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">172</span>: <span class="cit-fpage">1165</span>–<span class="cit-lpage">1177</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.105.044586</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.105.044586%26rft_id%253Dinfo%253Apmid%252F16079235%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3Mi8yLzExNjUiO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzEwLzMxODUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3185.8" data-doi="10.1007/s00122-013-2254-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">D’hoop</span>  <span class="cit-name-given-names">B. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Keizer</span>  <span class="cit-name-given-names">P. L. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paulo</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Visser</span>  <span class="cit-name-given-names">R. G. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Eeuwijk</span>  <span class="cit-name-given-names">F. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Identification of agronomically important QTL in tetraploid potato cultivars using a marker-trait association analysis.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">127</span>: <span class="cit-fpage">731</span>–<span class="cit-lpage">748</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-013-2254-y</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D127%26rft.spage%253D731%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-013-2254-y%26rft_id%253Dinfo%253Apmid%252F24408376%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-013-2254-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24408376&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3185.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3185.9" data-doi="10.1007/s00122-010-1379-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">D’hoop</span>  <span class="cit-name-given-names">B. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paulo</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kowitwanich</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sengers</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Visser</span>  <span class="cit-name-given-names">R. G. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Population structure and linkage disequilibrium unravelled in tetraploid potato.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">121</span>: <span class="cit-fpage">1151</span>–<span class="cit-lpage">1170</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-010-1379-5</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DD%2527hoop%26rft.auinit1%253DB.%2BB.%26rft.volume%253D121%26rft.issue%253D6%26rft.spage%253D1151%26rft.epage%253D1170%26rft.atitle%253DPopulation%2Bstructure%2Band%2Blinkage%2Bdisequilibrium%2Bunravelled%2Bin%2Btetraploid%2Bpotato.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-010-1379-5%26rft_id%253Dinfo%253Apmid%252F20563789%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-010-1379-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20563789&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3185.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3185.10" data-doi="10.1007/s10681-007-9565-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">D’hoop</span>  <span class="cit-name-given-names">B. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paulo</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mank</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Eck</span>  <span class="cit-name-given-names">H. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Eeuwijk</span>  <span class="cit-name-given-names">F. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Association mapping of quality traits in potato (Solanum tuberosum L.).</span> <abbr class="cit-jnl-abbrev">Euphytica</abbr> <span class="cit-vol">161</span>: <span class="cit-fpage">47</span>–<span class="cit-lpage">60</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s10681-007-9565-5</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEuphytica%26rft.volume%253D161%26rft.spage%253D47%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10681-007-9565-5%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10681-007-9565-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000254877800004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3185.11" data-doi="10.1111/j.1365-294X.2005.02553.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Evanno</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Regnaut</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goudet</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Detecting the number of clusters of individuals using the software STRUCTURE: a simulation study.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">2611</span>–<span class="cit-lpage">2620</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1365-294X.2005.02553.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DEvanno%26rft.auinit1%253DG.%26rft.volume%253D14%26rft.issue%253D8%26rft.spage%253D2611%26rft.epage%253D2620%26rft.atitle%253DDetecting%2Bthe%2Bnumber%2Bof%2Bclusters%2Bof%2Bindividuals%2Busing%2Bthe%2Bsoftware%2BSTRUCTURE%253A%2Ba%2Bsimulation%2Bstudy.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2005.02553.x%26rft_id%253Dinfo%253Apmid%252F15969739%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2005.02553.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15969739&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3185.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000229961500029&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3185.12" data-doi="10.1371/journal.pone.0036347"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Felcher</span>  <span class="cit-name-given-names">K. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coombs</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Massa</span>  <span class="cit-name-given-names">A. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hansey</span>  <span class="cit-name-given-names">C. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hamilton</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Integration of Two Diploid Potato Linkage Maps with the Potato Genome Sequence.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e36347</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0036347</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DFelcher%26rft.auinit1%253DK.%2BJ.%26rft.volume%253D7%26rft.issue%253D4%26rft.spage%253De36347%26rft.epage%253De36347%26rft.atitle%253DIntegration%2Bof%2Btwo%2Bdiploid%2Bpotato%2Blinkage%2Bmaps%2Bwith%2Bthe%2Bpotato%2Bgenome%2Bsequence.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0036347%26rft_id%253Dinfo%253Apmid%252F22558443%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0036347&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22558443&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3185.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3185.13" data-doi="10.1186/1471-2229-13-113"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fischer</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schreiber</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Colby</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuckenberg</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tacke</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Novel candidate genes influencing natural variation in potato tuber cold sweetening identified by comparative proteomics and association mapping.</span> <abbr class="cit-jnl-abbrev">BMC Plant Biol.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">113</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2229-13-113</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BPlant%2BBiol.%26rft.volume%253D13%26rft.spage%253D113%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2229-13-113%26rft_id%253Dinfo%253Apmid%252F23919263%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2229-13-113&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23919263&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3185.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3185.14" data-doi="10.1534/genetics.104.035642"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Garris</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tai</span>  <span class="cit-name-given-names">T. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coburn</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kresovich</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCouch</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Genetic structure and diversity in Oryza sativa L.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">169</span>: <span class="cit-fpage">1631</span>–<span class="cit-lpage">1638</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.104.035642</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.104.035642%26rft_id%253Dinfo%253Apmid%252F15654106%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2OS8zLzE2MzEiO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzEwLzMxODUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3185.15" data-doi="10.1023/B:MOLB.0000012878.89855.df"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gebhardt</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ballvora</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Walkemeier</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oberhagemann</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schuler</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Assessing genetic potential in germplasm collections of crop plants by marker-trait association: a case study for potatoes with quantitative variation of resistance to late blight and maturity type.</span> <abbr class="cit-jnl-abbrev">Mol. Breed.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">93</span>–<span class="cit-lpage">102</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1023/B:MOLB.0000012878.89855.df</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBreed.%26rft.volume%253D13%26rft.spage%253D93%26rft_id%253Dinfo%253Adoi%252F10.1023%252FB%253AMOLB.0000012878.89855.df%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1023/B:MOLB.0000012878.89855.df&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-other" id="cit-8.10.3185.16"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">GenABEL project developers</span>, <span class="cit-pub-date">2013</span> GenABEL: genome-wide SNP association analysis.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3185.17" data-doi="10.2307/2528417"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gower</span>  <span class="cit-name-given-names">J. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1967</span> <span class="cit-article-title">A Comparison of Some Methods of Cluster Analysis.</span> <abbr class="cit-jnl-abbrev">Biometrics</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">623</span>–<span class="cit-lpage">637</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiometrics%26rft.stitle%253DBiometrics%26rft.aulast%253DGower%26rft.auinit1%253DJ.%2BC.%26rft.volume%253D23%26rft.issue%253D4%26rft.spage%253D623%26rft.epage%253D637%26rft.atitle%253DA%2Bcomparison%2Bof%2Bsome%2Bmethods%2Bof%2Bcluster%2Banalysis.%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2528417%26rft_id%253Dinfo%253Apmid%252F6080200%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2528417&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=6080200&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3185.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1967A572200001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3185.18" data-doi="10.1007/s00122-014-2347-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hackett</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bradshaw</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bryan</span>  <span class="cit-name-given-names">G. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">QTL mapping in autotetraploids using SNP dosage information.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">127</span>: <span class="cit-fpage">1885</span>–<span class="cit-lpage">1904</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-014-2347-2</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D127%26rft.spage%253D1885%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-014-2347-2%26rft_id%253Dinfo%253Apmid%252F24981609%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-014-2347-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24981609&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3185.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3185.19" data-doi="10.1371/journal.pone.0063939"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hackett</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McLean</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bryan</span>  <span class="cit-name-given-names">G. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Linkage Analysis and QTL Mapping Using SNP Dosage Data in a Tetraploid Potato Mapping Population.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e63939</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0063939</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D8%26rft.spage%253De63939%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0063939%26rft_id%253Dinfo%253Apmid%252F23704960%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0063939&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23704960&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3185.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3185.20" data-doi="10.1093/bfgp/elp048"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hall</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tegstrom</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ingvarsson</span>  <span class="cit-name-given-names">P. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Using association mapping to dissect the genetic basis of complex traits in plants.</span> <abbr class="cit-jnl-abbrev">Brief. Funct. Genomics</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">157</span>–<span class="cit-lpage">165</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bfgp/elp048</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBrief.%2BFunct.%2BGenomics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbfgp%252Felp048%26rft_id%253Dinfo%253Apmid%252F20053815%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bfgp/elp048&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20053815&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3185.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000276191200009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3185.21" data-doi="10.1186/1471-2164-12-302"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hamilton</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hansey</span>  <span class="cit-name-given-names">C. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Whitty</span>  <span class="cit-name-given-names">B. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stoffel</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Massa</span>  <span class="cit-name-given-names">A. N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Single nucleotide polymorphism discovery in elite north american potato germplasm.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">302</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2164-12-302</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DHamilton%26rft.auinit1%253DJ.%2BP.%26rft.volume%253D12%26rft.spage%253D302%26rft.epage%253D302%26rft.atitle%253DSingle%2Bnucleotide%2Bpolymorphism%2Bdiscovery%2Bin%2Belite%2BNorth%2BAmerican%2Bpotato%2Bgermplasm.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-12-302%26rft_id%253Dinfo%253Apmid%252F21658273%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-12-302&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21658273&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3185.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3185.22" data-doi="10.3835/plantgenome2014.06.0025"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hardigan</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bamberg</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buell</span>  <span class="cit-name-given-names">C. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Douches</span>  <span class="cit-name-given-names">D. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Taxonomy and Genetic Differentiation among Wild and Cultivated Germplasm of Solanum sect. Petota.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">16</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2014.06.0025</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D8%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2014.06.0025%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2014.06.0025&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3185.23" data-doi="10.1534/g3.113.005595"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hirsch</span>  <span class="cit-name-given-names">C.N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hirsch</span>  <span class="cit-name-given-names">C.D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Felcher</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coombs</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zarka</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> Retrospective View of North American Potato (Solanum tuberosum L.) Breeding in the 20th and 21st Centuries. G3 (Bethesda<em>)</em> 3:1003–1013.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.113.005595</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.113.005595%26rft_id%253Dinfo%253Apmid%252F23589519%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjg6IjMvNi8xMDAzIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMC8zMTg1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3185.24" data-doi="10.1007/BF00220891"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jacobs</span>  <span class="cit-name-given-names">J. M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vaneck</span>  <span class="cit-name-given-names">H. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arens</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Verkerkbakker</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hekkert</span>  <span class="cit-name-given-names">B. T. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">A Genetic-Map of potato (Solanum-Tuberosum) Intergrating Molecular Markers, Icluding Transposons, and Classical Markers.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">289</span>–<span class="cit-lpage">300</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/BF00220891</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D91%26rft.spage%253D289%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00220891%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00220891&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1995RL40300016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3185.25" data-doi="10.1038/nature11912"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kloosterman</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abelenda</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gomez</span>  <span class="cit-name-given-names">M. D. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oortwijn</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Boer</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Naturally occurring allele diversity allows potato cultivation in northern latitudes.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">495</span>: <span class="cit-fpage">246</span>–<span class="cit-lpage">250</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature11912</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D495%26rft.spage%253D246%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature11912%26rft_id%253Dinfo%253Apmid%252F23467094%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature11912&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23467094&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3185.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000316652300053&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3185.26" data-doi="10.1186/1471-2164-11-158"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kloosterman</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oortwijn</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Uitdewilligen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">America</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Vos</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">From QTL to candidate gene: Genetical genomics of simple and complex traits in potato using a pooling strategy.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">158</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2164-11-158</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DKloosterman%26rft.auinit1%253DB.%26rft.volume%253D11%26rft.spage%253D158%26rft.epage%253D158%26rft.atitle%253DFrom%2BQTL%2Bto%2Bcandidate%2Bgene%253A%2Bgenetical%2Bgenomics%2Bof%2Bsimple%2Band%2Bcomplex%2Btraits%2Bin%2Bpotato%2Busing%2Ba%2Bpooling%2Bstrategy.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-11-158%26rft_id%253Dinfo%253Apmid%252F20210995%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-11-158&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20210995&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3185.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-other" id="cit-8.10.3185.27"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Koenker</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> quantreg: Quantile Regression.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3185.28" data-doi="10.1007/s12230-016-9543-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kolech</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Halseth</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Perry</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wolfe</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Douches</span>  <span class="cit-name-given-names">D. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genetic Diversity and Relationship of Ethiopian Potato Varieties to Germplasm from North America, Europe and the International Potato Center.</span> <abbr class="cit-jnl-abbrev">Am. J. Potato Res.</abbr> <span class="cit-vol">93</span>: <span class="cit-fpage">609</span>–<span class="cit-lpage">619</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s12230-016-9543-3</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BJ.%2BPotato%2BRes.%26rft.volume%253D93%26rft.spage%253D609%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs12230-016-9543-3%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s12230-016-9543-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3185.29" data-doi="10.1534/genetics.104.040006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Strahwald</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hofferbert</span>  <span class="cit-name-given-names">H. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lubeck</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tacke</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span><span class="cit-comment">a</span> <span class="cit-article-title">DNA variation at the invertase locus invGE/GF is associated with tuber quality traits in populations of potato breeding clones.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">170</span>: <span class="cit-fpage">813</span>–<span class="cit-lpage">821</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.104.040006</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.104.040006%26rft_id%253Dinfo%253Apmid%252F15802505%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTcwLzIvODEzIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMC8zMTg1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3185.30" data-doi="10.3835/plantgenome2010.09.0022"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">X. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wei</span>  <span class="cit-name-given-names">Y. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moore</span>  <span class="cit-name-given-names">K. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Michaud</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Viands</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Association Mapping of Biomass Yield and Stem Composition in a Tetraploid Alfalfa Breeding Population.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">24</span>–<span class="cit-lpage">35</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2010.09.0022</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D4%26rft.spage%253D24%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2010.09.0022%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2010.09.0022&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3185.31" data-doi="10.1007/s00122-005-1927-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">X. Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">De Jong</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">De Jong</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">De Jong</span>  <span class="cit-name-given-names">W. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span><span class="cit-comment">b</span> <span class="cit-article-title">Inheritance and genetic mapping of tuber eye depth in cultivated diploid potatoes.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">110</span>: <span class="cit-fpage">1068</span>–<span class="cit-lpage">1073</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-005-1927-6</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DLi%26rft.auinit1%253DX.%2BQ.%26rft.volume%253D110%26rft.issue%253D6%26rft.spage%253D1068%26rft.epage%253D1073%26rft.atitle%253DInheritance%2Band%2Bgenetic%2Bmapping%2Bof%2Btuber%2Beye%2Bdepth%2Bin%2Bcultivated%2Bdiploid%2Bpotatoes.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-005-1927-6%26rft_id%253Dinfo%253Apmid%252F15719211%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-005-1927-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15719211&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3185.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3185.32" data-doi="10.1007/BF02987188"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lightbourn</span>  <span class="cit-name-given-names">G. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Veilleux</span>  <span class="cit-name-given-names">R. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Production and evaluation of somatic hybrids derived from monoploid potato.</span> <abbr class="cit-jnl-abbrev">Am. J. Potato Res.</abbr> <span class="cit-vol">84</span>: <span class="cit-fpage">425</span>–<span class="cit-lpage">435</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/BF02987188</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BJ.%2BPotato%2BRes.%26rft.volume%253D84%26rft.spage%253D425%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF02987188%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF02987188&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3185.33" data-doi="10.1094/PHYTO-10-13-0270-R"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lindqvist-Kreuze</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gastelo</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Perez</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Forbes</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Koeyer</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Phenotypic Stability and Genome-Wide Association Study of Late Blight Resistance in Potato Genotypes Adapted to the Tropical Highlands.</span> <abbr class="cit-jnl-abbrev">Phytopathology</abbr> <span class="cit-vol">104</span>: <span class="cit-fpage">624</span>–<span class="cit-lpage">633</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1094/PHYTO-10-13-0270-R</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhytopathology%26rft.volume%253D104%26rft.spage%253D624%26rft_id%253Dinfo%253Adoi%252F10.1094%252FPHYTO-10-13-0270-R%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1094/PHYTO-10-13-0270-R&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3185.34" data-doi="10.1007/BF02853624"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Love</span>  <span class="cit-name-given-names">S. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Founding clones, major contributing ancestors, and exotic progenitors of prominent North American potato cultivars.</span> <abbr class="cit-jnl-abbrev">Am. J. Potato Res.</abbr> <span class="cit-vol">76</span>: <span class="cit-fpage">263</span>–<span class="cit-lpage">272</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/BF02853624</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BJ.%2BPotato%2BRes.%26rft.volume%253D76%26rft.spage%253D263%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF02853624%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF02853624&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3185.35" data-doi="10.1534/genetics.105.054932"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Malosetti</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van der Linden</span>  <span class="cit-name-given-names">C. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vosman</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Eeuwijk</span>  <span class="cit-name-given-names">F. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">A mixed-model approach to association mapping using pedigree information with an illustration of resistance to Phytophthora infestans in potato.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">175</span>: <span class="cit-fpage">879</span>–<span class="cit-lpage">889</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.105.054932</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.105.054932%26rft_id%253Dinfo%253Apmid%252F17151263%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTc1LzIvODc5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMC8zMTg1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3185.36" data-doi="10.1007/s11295-011-0391-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Marroni</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pinosio</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zaina</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fogolari</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Felice</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Nucleotide diversity and linkage disequilibrium in Populus nigra cinnamyl alcohol dehydrogenase (CAD4) gene.</span> <abbr class="cit-jnl-abbrev">Tree Genet. Genomes</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">1011</span>–<span class="cit-lpage">1023</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s11295-011-0391-5</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTree%2BGenet.%2BGenomes%26rft.volume%253D7%26rft.spage%253D1011%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11295-011-0391-5%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11295-011-0391-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3185.37" data-doi="10.1002/gepi.20401"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Miclaus</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wolfinger</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Czika</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">SNP Selection and Multidimensional Scaling to Quantify Population Structure.</span> <abbr class="cit-jnl-abbrev">Genet. Epidemiol.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">488</span>–<span class="cit-lpage">496</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/gepi.20401</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetic%2Bepidemiology%26rft.stitle%253DGenet%2BEpidemiol%26rft.aulast%253DMiclaus%26rft.auinit1%253DK.%26rft.volume%253D33%26rft.issue%253D6%26rft.spage%253D488%26rft.epage%253D496%26rft.atitle%253DSNP%2Bselection%2Band%2Bmultidimensional%2Bscaling%2Bto%2Bquantify%2Bpopulation%2Bstructure.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fgepi.20401%26rft_id%253Dinfo%253Apmid%252F19194989%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/gepi.20401&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19194989&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3185.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000269432400003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3185.38" data-doi="10.1007/s00357-014-9161-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Murtagh</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Legendre</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Ward’s Hierarchical Agglomerative Clustering Method: Which Algorithms Implement Ward’s Criterion?</span> <abbr class="cit-jnl-abbrev">J. Classif.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">274</span>–<span class="cit-lpage">295</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00357-014-9161-z</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BClassif.%26rft.volume%253D31%26rft.spage%253D274%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00357-014-9161-z%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00357-014-9161-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3185.39" data-doi="10.1086/282771"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nei</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1972</span> <span class="cit-article-title">GENETIC DISTANCE BETWEEN POPULATIONS.</span> <abbr class="cit-jnl-abbrev">Am. Nat.</abbr> <span class="cit-vol">106</span>: <span class="cit-fpage">283</span>–<span class="cit-lpage">292</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1086/282771</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BNat.%26rft.volume%253D106%26rft.spage%253D283%26rft_id%253Dinfo%253Adoi%252F10.1086%252F282771%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/282771&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1972M475000002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3185.40" data-doi="10.1534/genetics.108.094268"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pajerowska-Mukhtar</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stich</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Achenbach</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ballvora</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luebeck</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Single Nucleotide Polymorphisms in the Allene Oxide Synthase 2 Gene Are Associated With Field Resistance to Late Blight in Populations of Tetraploid Potato Cultivars.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">181</span>: <span class="cit-fpage">1115</span>–<span class="cit-lpage">1127</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.108.094268</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.108.094268%26rft_id%253Dinfo%253Apmid%252F19139145%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE4MS8zLzExMTUiO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzEwLzMxODUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3185.41" data-doi="10.1038/nature10158"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">Potato Genome Sequencing Consortium</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genome sequence and analysis of the tuber crop potato.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">475</span>: <span class="cit-fpage">189</span>–<span class="cit-lpage">195</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature10158</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DXu%26rft.auinit1%253DX.%26rft.volume%253D475%26rft.issue%253D7355%26rft.spage%253D189%26rft.epage%253D195%26rft.atitle%253DGenome%2Bsequence%2Band%2Banalysis%2Bof%2Bthe%2Btuber%2Bcrop%2Bpotato.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature10158%26rft_id%253Dinfo%253Apmid%252F21743474%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature10158&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21743474&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3185.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000292690500042&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3185.42" data-doi="10.1007/s00122-014-2369-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Prashar</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hornyik</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Young</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McLean</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sharma</span>  <span class="cit-name-given-names">S. K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Construction of a dense SNP map of a highly heterozygous diploid potato population and QTL analysis of tuber shape and eye depth.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">127</span>: <span class="cit-fpage">2159</span>–<span class="cit-lpage">2171</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-014-2369-9</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D127%26rft.spage%253D2159%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-014-2369-9%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-014-2369-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3185.43" data-doi="10.1111/j.1471-8286.2005.01155.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pritchard</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Donnelly</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Inference of population structure using multilocus genotype data.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">155</span>: <span class="cit-fpage">945</span>–<span class="cit-lpage">959</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DPritchard%26rft.auinit1%253DJ.%2BK.%26rft.volume%253D155%26rft.issue%253D2%26rft.spage%253D945%26rft.epage%253D959%26rft.atitle%253DInference%2Bof%2BPopulation%2BStructure%2BUsing%2BMultilocus%2BGenotype%2BData%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1471-8286.2005.01155.x%26rft_id%253Dinfo%253Apmid%252F10835412%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTU1LzIvOTQ1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMC8zMTg1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3185.44" data-doi="10.3835/plantgenome2016.03.0024"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rosyara</span>  <span class="cit-name-given-names">U. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">De Jong</span>  <span class="cit-name-given-names">W. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Douches</span>  <span class="cit-name-given-names">D. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Endelman</span>  <span class="cit-name-given-names">J. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Software for Genome-Wide Association Studies in Autopolyploids and Its Application to Potato.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">10</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2015.08.0073</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D9%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2016.03.0024%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2016.03.0024&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3185.45" data-doi="10.1007/s00122-016-2845-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schmitz Carley</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coombs</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Douches</span>  <span class="cit-name-given-names">D. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bethke</span>  <span class="cit-name-given-names">P. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Palta</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Automated tetraploid genotype calling by hierarchical clustering.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">130</span>: <span class="cit-fpage">717</span>–<span class="cit-lpage">726</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-016-2845-5</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D130%26rft.spage%253D717%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-016-2845-5%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-016-2845-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3185.46" data-doi="10.1007/s00122-016-2665-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schönhals</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ortega</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barandalla</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aragones</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Galarreta</span>  <span class="cit-name-given-names">J. I. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Identification and reproducibility of diagnostic DNA markers for tuber starch and yield optimization in a novel association mapping population of potato (Solanum tuberosum L.).</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">129</span>: <span class="cit-fpage">767</span>–<span class="cit-lpage">785</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-016-2665-7</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D129%26rft.spage%253D767%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-016-2665-7%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-016-2665-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3185.47" data-doi="10.1534/g3.113.007153"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sharma</span>  <span class="cit-name-given-names">S.K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bolser</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Boer</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sonderkaer</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amoros</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> Construction of Reference Chromosome-Scale Pseudomolecules for Potato: Integrating the Potato Genome with Genetic and Physical Maps. G3 (Bethesda) 3 :2031–2047.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.113.007153</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.113.007153%26rft_id%253Dinfo%253Apmid%252F24062527%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjk6IjMvMTEvMjAzMSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzgvMTAvMzE4NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3185.48" data-doi="10.1007/s00122-003-1431-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Simko</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Costanzo</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haynes</span>  <span class="cit-name-given-names">K. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Christ</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">R. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span><span class="cit-comment">a</span> <span class="cit-article-title">Linkage disequilibrium mapping of a Verticillium dahliae resistance quantitative trait locus in tetraploid potato (Solanum tuberosum) through a candidate gene approach.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">217</span>–<span class="cit-lpage">224</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-004-1637-5</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DSimko%26rft.auinit1%253DI.%26rft.volume%253D108%26rft.issue%253D2%26rft.spage%253D217%26rft.epage%253D224%26rft.atitle%253DLinkage%2Bdisequilibrium%2Bmapping%2Bof%2Ba%2BVerticillium%2Bdahliae%2Bresistance%2Bquantitative%2Btrait%2Blocus%2Bin%2Btetraploid%2Bpotato%2B%2528%2BSolanum%2Btuberosum%2529%2Bthrough%2Ba%2Bcandidate%2Bgene%2Bapproach.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-003-1431-9%26rft_id%253Dinfo%253Apmid%252F14523522%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-003-1431-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14523522&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3185.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000188079400005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3185.49" data-doi="10.1007/s00438-004-1010-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Simko</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haynes</span>  <span class="cit-name-given-names">K. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ewing</span>  <span class="cit-name-given-names">E. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Costanzo</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Christ</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span><span class="cit-comment">b</span> <span class="cit-article-title">Mapping genes for resistance to Verticillium albo-atrum in tetraploid and diploid potato populations using haplotype association tests and genetic linkage analysis.</span> <abbr class="cit-jnl-abbrev">Mol. Genet. Genomics</abbr> <span class="cit-vol">271</span>: <span class="cit-fpage">522</span>–<span class="cit-lpage">531</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00438-004-1010-z</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bgenetics%2Band%2Bgenomics%2B%253A%2B%2BMGG%26rft.stitle%253DMol%2BGenet%2BGenomics%26rft.aulast%253DSimko%26rft.auinit1%253DI.%26rft.volume%253D271%26rft.issue%253D5%26rft.spage%253D522%26rft.epage%253D531%26rft.atitle%253DMapping%2Bgenes%2Bfor%2Bresistance%2Bto%2BVerticillium%2Balbo-atrum%2Bin%2Btetraploid%2Band%2Bdiploid%2Bpotato%2Bpopulations%2Busing%2Bhaplotype%2Bassociation%2Btests%2Band%2Bgenetic%2Blinkage%2Banalysis.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00438-004-1010-z%26rft_id%253Dinfo%253Apmid%252F15107986%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00438-004-1010-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15107986&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3185.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000222752700002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3185.50" data-doi="10.1534/genetics.106.060905"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Simko</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haynes</span>  <span class="cit-name-given-names">K. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">R. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Assessment of linkage disequilibrium in potato genome with single nucleotide polymorphism markers.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">173</span>: <span class="cit-fpage">2237</span>–<span class="cit-lpage">2245</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.106.060905</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.106.060905%26rft_id%253Dinfo%253Apmid%252F16783002%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3My80LzIyMzciO3M6NDoiYXRvbSI7czoxOToiL2dnZy84LzEwLzMxODUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3185.51" data-doi="10.1111/j.1439-0523.2008.01420.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Śliwka</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wasilewicz-Flis</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jakuczun</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gebhardt</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Tagging quantitative trait loci for dormancy, tuber shape, regularity of tuber shape, eye depth and flesh colour in diploid potato originated from six Solanum species.</span> <abbr class="cit-jnl-abbrev">Plant Breed.</abbr> <span class="cit-vol">127</span>: <span class="cit-fpage">49</span>–<span class="cit-lpage">55</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1439-0523.2008.01420.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BBreed.%26rft.volume%253D127%26rft.spage%253D49%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1439-0523.2008.01420.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1439-0523.2008.01420.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3185.52" data-doi="10.1111/pbr.12102"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stich</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Urbany</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoffmann</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gebhardt</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Population structure and linkage disequilibrium in diploid and tetraploid potato revealed by genome-wide high-density genotyping using the SolCAP SNP array.</span> <abbr class="cit-jnl-abbrev">Plant Breed.</abbr> <span class="cit-vol">132</span>: <span class="cit-fpage">718</span>–<span class="cit-lpage">724</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/pbr.12102</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BBreed.%26rft.volume%253D132%26rft.spage%253D718%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fpbr.12102%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/pbr.12102&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3185.53" data-doi="10.1007/s00122-007-0601-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tommasini</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schnurbusch</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fossati</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mascher</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Keller</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Association mapping of Stagonospora nodorum blotch resistance in modern European winter wheat varieties.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">115</span>: <span class="cit-fpage">697</span>–<span class="cit-lpage">708</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-007-0601-6</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DTommasini%26rft.auinit1%253DL.%26rft.volume%253D115%26rft.issue%253D5%26rft.spage%253D697%26rft.epage%253D708%26rft.atitle%253DAssociation%2Bmapping%2Bof%2BStagonospora%2Bnodorum%2Bblotch%2Bresistance%2Bin%2Bmodern%2BEuropean%2Bwinter%2Bwheat%2Bvarieties.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-007-0601-6%26rft_id%253Dinfo%253Apmid%252F17634916%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-007-0601-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17634916&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3185.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000249019700010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3185.54" data-doi="10.1111/j.1365-3059.2006.01489.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Trognitz</span>  <span class="cit-name-given-names">B. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trognitz</span>  <span class="cit-name-given-names">F. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Occurrence of the R1 allele conferring resistance to late blight in potato R-gene differentials and commercial cultivars.</span> <abbr class="cit-jnl-abbrev">Plant Pathol.</abbr> <span class="cit-vol">56</span>: <span class="cit-fpage">150</span>–<span class="cit-lpage">155</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1365-3059.2006.01489.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPathol.%26rft.volume%253D56%26rft.spage%253D150%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-3059.2006.01489.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-3059.2006.01489.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3185.55" data-doi="10.1371/journal.pone.0062355"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Uitdewilligen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wolters</span>  <span class="cit-name-given-names">A. M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">D’Hoop</span>  <span class="cit-name-given-names">B. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Borm</span>  <span class="cit-name-given-names">T. J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Visser</span>  <span class="cit-name-given-names">R. G. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">A Next-Generation Sequencing Method for Genotyping-by-Sequencing of Highly Heterozygous Autotetraploid Potato.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e62355</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0062355</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D8%26rft.spage%253De62355%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0062355%26rft_id%253Dinfo%253Apmid%252F23667470%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0062355&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23667470&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3185.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3185.56" data-doi="10.1186/1471-2164-12-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Urbany</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stich</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schmidt</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Simon</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berding</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Association genetics in Solanum tuberosum provides new insights into potato tuber bruising and enzymatic tissue discoloration.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">7</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2164-12-7</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DUrbany%26rft.auinit1%253DC.%26rft.volume%253D12%26rft.spage%253D7%26rft.epage%253D7%26rft.atitle%253DAssociation%2Bgenetics%2Bin%2BSolanum%2Btuberosum%2Bprovides%2Bnew%2Binsights%2Binto%2Bpotato%2Btuber%2Bbruising%2Band%2Benzymatic%2Btissue%2Bdiscoloration.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-12-7%26rft_id%253Dinfo%253Apmid%252F21208436%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-12-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21208436&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3185.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3185.57" data-doi="10.1007/s00122-016-2831-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">van Eck</span>  <span class="cit-name-given-names">H. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vos</span>  <span class="cit-name-given-names">P. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valkonen</span>  <span class="cit-name-given-names">J. P. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Uitdewilligen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lensing</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Graphical genotyping as a method to map Ny ((o,n)sto) and Gpa5 using a reference panel of tetraploid potato cultivars.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">130</span>: <span class="cit-fpage">515</span>–<span class="cit-lpage">528</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-016-2831-y</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D130%26rft.spage%253D515%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-016-2831-y%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-016-2831-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3185.58"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vaneck</span>  <span class="cit-name-given-names">H. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jacobs</span>  <span class="cit-name-given-names">J. M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stam</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ton</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stiekema</span>  <span class="cit-name-given-names">W. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">MULTIPLE ALLELES FOR TUBER SHAPE IN DIPLOID POTATO DETECTED BY QUALITATIVE AND QUANTITATIVE GENETIC-ANALYSIS USING RFLPS.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">137</span>: <span class="cit-fpage">303</span>–<span class="cit-lpage">309</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DVan%2BEck%26rft.auinit1%253DH.%2BJ.%26rft.volume%253D137%26rft.issue%253D1%26rft.spage%253D303%26rft.epage%253D309%26rft.atitle%253DMultiple%2Balleles%2Bfor%2Btuber%2Bshape%2Bin%2Bdiploid%2Bpotato%2Bdetected%2Bby%2Bqualitative%2Band%2Bquantitative%2Bgenetic%2Banalysis%2Busing%2BRFLPs.%26rft_id%253Dinfo%253Apmid%252F7914504%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTM3LzEvMzAzIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvOC8xMC8zMTg1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3185.59" data-doi="10.1186/1471-2105-12-172"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Voorrips</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gort</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vosman</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genotype calling in tetraploid species from bi-allelic marker data using mixture models.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">172</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2105-12-172</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DVoorrips%26rft.auinit1%253DR.%2BE.%26rft.volume%253D12%26rft.spage%253D172%26rft.epage%253D172%26rft.atitle%253DGenotype%2Bcalling%2Bin%2Btetraploid%2Bspecies%2Bfrom%2Bbi-allelic%2Bmarker%2Bdata%2Busing%2Bmixture%2Bmodels.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-12-172%26rft_id%253Dinfo%253Apmid%252F21595880%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-12-172&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21595880&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3185.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-book" id="cit-8.10.3185.60"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vos</span>  <span class="cit-name-given-names">P. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paulo</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bourke</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maliepaard</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Visser</span>  <span class="cit-name-given-names">R. G. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">GWAS in tetraploid potato: Identification and validation of SNP markers associated with glycoalkaloid content</span>, <span class="cit-comment">pp. 69–95 in <em>Development and application of a 20K SNP array in potato</em>, PhD thesis by Peter Vos, Wageningen University &amp; Research, Wageningen, The Netherlands</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3185.61" data-doi="10.1007/s00122-016-2798-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vos</span>  <span class="cit-name-given-names">P. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paulo</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Voorrips</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Visser</span>  <span class="cit-name-given-names">R. G. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Eck</span>  <span class="cit-name-given-names">H. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Evaluation of LD decay and various LD-decay estimators in simulated and SNP-array data of tetraploid potato.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">130</span>: <span class="cit-fpage">123</span>–<span class="cit-lpage">135</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-016-2798-8</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D130%26rft.spage%253D123%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-016-2798-8%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-016-2798-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3185.62" data-doi="10.1007/s00122-015-2593-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vos</span>  <span class="cit-name-given-names">P. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Uitdewilligen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Voorrips</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Visser</span>  <span class="cit-name-given-names">R. G. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Eck</span>  <span class="cit-name-given-names">H. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Development and analysis of a 20K SNP array for potato (Solanum tuberosum): an insight into the breeding history.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">128</span>: <span class="cit-fpage">2387</span>–<span class="cit-lpage">2401</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-015-2593-y</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D128%26rft.spage%253D2387%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-015-2593-y%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-015-2593-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3185.63" data-doi="10.1038/ng1702"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pressoir</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Briggs</span>  <span class="cit-name-given-names">W. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bi</span>  <span class="cit-name-given-names">I. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yamasaki</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">A unified mixed-model method for association mapping that accounts for multiple levels of relatedness.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">203</span>–<span class="cit-lpage">208</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng1702</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DYu%26rft.auinit1%253DJ.%26rft.volume%253D38%26rft.issue%253D2%26rft.spage%253D203%26rft.epage%253D208%26rft.atitle%253DA%2Bunified%2Bmixed-model%2Bmethod%2Bfor%2Bassociation%2Bmapping%2Bthat%2Baccounts%2Bfor%2Bmultiple%2Blevels%2Bof%2Brelatedness.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1702%26rft_id%253Dinfo%253Apmid%252F16380716%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1702&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16380716&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3185.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000234953200015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-8.10.3185.64" data-doi="10.1038/ncomms1467"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tung</span>  <span class="cit-name-given-names">C. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eizenga</span>  <span class="cit-name-given-names">G. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">M. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ali</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genome-wide association mapping reveals a rich genetic architecture of complex traits in Oryza sativa.</span> <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">467</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ncomms1467</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BCommun%26rft.aulast%253DZhao%26rft.auinit1%253DK.%26rft.volume%253D2%26rft.spage%253D467%26rft.epage%253D467%26rft.atitle%253DGenome-wide%2Bassociation%2Bmapping%2Breveals%2Ba%2Brich%2Bgenetic%2Barchitecture%2Bof%2Bcomplex%2Btraits%2Bin%2BOryza%2Bsativa.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms1467%26rft_id%253Dinfo%253Apmid%252F21915109%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms1467&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21915109&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F10%2F3185.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/8/10/3185.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/10/3173" title="Accounting for Programmed Ribosomal Frameshifting in the Computation of Codon Usage Bias Indices" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/10/3203" title="Estimating Genetic Relatedness in Admixed Populations" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/8/10"><span>Volume 8 Issue 10, October 2018</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/8/10.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/8/10.cover-source.jpg?itok=p8rfSNcT" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 8 (10)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/10/3185.abstract" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-9oafa-02B5Dz2SMJ92qPao5X31UEcGgkQevQQtwWl7s" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/8/10/3185" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/8/10/3185" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/8/10/3185" class="active" data-icon-position="" data-hide-link-title="0">Linkage Disequilibrium and Evaluation of Genome-Wide Association Mapping Models in Tetraploid Potato</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/490969" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-vCQeQRzI_9iXeL-gXJhL8H0kyglOBfmCKNQDhEU8lw8" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/10/3185.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/8/10/3185" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-ztR15-X_LhheRjyYpRdDXe9RPGJnqaEJP0c05g2mW3s" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/8/10/3185" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/10/3185.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490969" id="node490969--2" data-pisa="ggg;8/10/3185" data-pisa-master="ggg;g3.118.200377" data-apath="/ggg/8/10/3185.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Linkage Disequilibrium and Evaluation of Genome-Wide Association Mapping Models in Tetraploid Potato</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-8373-5509" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Sanjeev Kumar Sharma</span>, <span class="highwire-citation-author" data-delta="1">Katrin MacKenzie</span>, <span class="highwire-citation-author" data-delta="2">Karen McLean</span>, <span class="highwire-citation-author" data-delta="3">Finlay Dale</span>, <span class="highwire-citation-author" data-delta="4">Steve Daniels</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0002-7752-5740" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Glenn J. Bryan</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3185-3202;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200377 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node490969--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Sanjeev Kumar Sharma </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The James Hutton Institute, Dundee DD2 5DA, UK</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sanjeev%2BKumar%2BSharma%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sharma%20SK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASanjeev%2BKumar%2BSharma%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-8373-5509" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Sanjeev Kumar Sharma</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:glenn.bryan@hutton.ac.uk" class="" data-icon-position="" data-hide-link-title="0">glenn.bryan@hutton.ac.uk</a>
<a href="mailto:sanjeev.sharma@hutton.ac.uk" class="" data-icon-position="" data-hide-link-title="0">sanjeev.sharma@hutton.ac.uk</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Katrin MacKenzie </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Biomathematics and Statistics Scotland (BioSS), Dundee DD2 5DA, UK</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Katrin%2BMacKenzie%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=MacKenzie%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKatrin%2BMacKenzie%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Karen McLean </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The James Hutton Institute, Dundee DD2 5DA, UK</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Karen%2BMcLean%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=McLean%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKaren%2BMcLean%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Finlay Dale </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The James Hutton Institute, Dundee DD2 5DA, UK</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Finlay%2BDale%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dale%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFinlay%2BDale%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Steve Daniels </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Cygnet PB Ltd, Cambridge, CB21 6AS, UK</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Steve%2BDaniels%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Daniels%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASteve%2BDaniels%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Glenn J. Bryan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The James Hutton Institute, Dundee DD2 5DA, UK</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Glenn%2BJ.%2BBryan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bryan%20GJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGlenn%2BJ.%2BBryan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-7752-5740" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Glenn J. Bryan</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:glenn.bryan@hutton.ac.uk" class="" data-icon-position="" data-hide-link-title="0">glenn.bryan@hutton.ac.uk</a>
<a href="mailto:sanjeev.sharma@hutton.ac.uk" class="" data-icon-position="" data-hide-link-title="0">sanjeev.sharma@hutton.ac.uk</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/8/10/3185" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/8/10/3185&amp;title=Linkage%20Disequilibrium%20and%20Evaluation%20of%20Genome-Wide%20Association%20Mapping%20Models%20in%20Tetraploid%20Potato" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/8/10/3185&amp;title=Linkage%20Disequilibrium%20and%20Evaluation%20of%20Genome-Wide%20Association%20Mapping%20Models%20in%20Tetraploid%20Potato" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/8/10/3185&amp;title=Linkage%20Disequilibrium%20and%20Evaluation%20of%20Genome-Wide%20Association%20Mapping%20Models%20in%20Tetraploid%20Potato" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/8/10/3185&amp;text=Linkage%20Disequilibrium%20and%20Evaluation%20of%20Genome-Wide%20Association%20Mapping%20Models%20in%20Tetraploid%20Potato" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/8/10/3185&amp;title=Linkage%20Disequilibrium%20and%20Evaluation%20of%20Genome-Wide%20Association%20Mapping%20Models%20in%20Tetraploid%20Potato" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/8/10/3185&amp;t=Linkage%20Disequilibrium%20and%20Evaluation%20of%20Genome-Wide%20Association%20Mapping%20Models%20in%20Tetraploid%20Potato" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/8/10/3185&amp;title=Linkage%20Disequilibrium%20and%20Evaluation%20of%20Genome-Wide%20Association%20Mapping%20Models%20in%20Tetraploid%20Potato" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/8/10/3185&amp;title=Linkage%20Disequilibrium%20and%20Evaluation%20of%20Genome-Wide%20Association%20Mapping%20Models%20in%20Tetraploid%20Potato" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/490969/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490969" id="node490969--3" data-pisa="ggg;8/10/3185" data-pisa-master="ggg;g3.118.200377" data-apath="/ggg/8/10/3185.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Linkage Disequilibrium and Evaluation of Genome-Wide Association Mapping Models in Tetraploid Potato</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-8373-5509" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Sanjeev Kumar Sharma</span>, <span class="highwire-citation-author" data-delta="1">Katrin MacKenzie</span>, <span class="highwire-citation-author" data-delta="2">Karen McLean</span>, <span class="highwire-citation-author" data-delta="3">Finlay Dale</span>, <span class="highwire-citation-author" data-delta="4">Steve Daniels</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0002-7752-5740" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Glenn J. Bryan</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3185-3202;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200377 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node490969--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Sanjeev Kumar Sharma </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The James Hutton Institute, Dundee DD2 5DA, UK</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sanjeev%2BKumar%2BSharma%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sharma%20SK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASanjeev%2BKumar%2BSharma%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-8373-5509" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Sanjeev Kumar Sharma</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:glenn.bryan@hutton.ac.uk" class="" data-icon-position="" data-hide-link-title="0">glenn.bryan@hutton.ac.uk</a>
<a href="mailto:sanjeev.sharma@hutton.ac.uk" class="" data-icon-position="" data-hide-link-title="0">sanjeev.sharma@hutton.ac.uk</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Katrin MacKenzie </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Biomathematics and Statistics Scotland (BioSS), Dundee DD2 5DA, UK</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Katrin%2BMacKenzie%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=MacKenzie%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKatrin%2BMacKenzie%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Karen McLean </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The James Hutton Institute, Dundee DD2 5DA, UK</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Karen%2BMcLean%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=McLean%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKaren%2BMcLean%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Finlay Dale </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The James Hutton Institute, Dundee DD2 5DA, UK</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Finlay%2BDale%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dale%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFinlay%2BDale%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Steve Daniels </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Cygnet PB Ltd, Cambridge, CB21 6AS, UK</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Steve%2BDaniels%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Daniels%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASteve%2BDaniels%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Glenn J. Bryan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The James Hutton Institute, Dundee DD2 5DA, UK</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Glenn%2BJ.%2BBryan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bryan%20GJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGlenn%2BJ.%2BBryan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-7752-5740" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Glenn J. Bryan</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:glenn.bryan@hutton.ac.uk" class="" data-icon-position="" data-hide-link-title="0">glenn.bryan@hutton.ac.uk</a>
<a href="mailto:sanjeev.sharma@hutton.ac.uk" class="" data-icon-position="" data-hide-link-title="0">sanjeev.sharma@hutton.ac.uk</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=3185&amp;rft.epage=3202&amp;rft.atitle=Linkage%20Disequilibrium%20and%20Evaluation%20of%20Genome-Wide%20Association%20Mapping%20Models%20in%20Tetraploid%20Potato&amp;rft.volume=8&amp;rft.issue=10&amp;rft.date=2018-10-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Sharma%2C+Sanjeev+Kumar&amp;rft.au=MacKenzie%2C+Katrin&amp;rft.au=McLean%2C+Karen&amp;rft.au=Dale%2C+Finlay&amp;rft.au=Daniels%2C+Steve&amp;rft.au=Bryan%2C+Glenn+J."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/490969/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/490969/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/490969/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/490969/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/490969/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/490969/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/490969/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/490969/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/490969/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/490969/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/490969/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/490969/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/8/10/3185" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-9" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results and Discussion</a></li><li class="even"><a href="#sec-18" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Conclusions</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/8/10/3185.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/8/10/3185.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/8/10/3185.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/8/10/3185.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
