<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/7/9/3059" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/7/9/3059.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/7/9/3059.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/7/9/3059.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/7/9/3059.atom" />
<meta name="HW.pisa" content="ggg;7/9/3059" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Whole-Genome Sequencing and Concordance Between Antimicrobial Susceptibility Genotypes and Phenotypes of Bacterial Isolates Associated with Bovine Respiratory Disease" />
<meta name="DC.Identifier" content="10.1534/g3.117.1137" />
<meta name="DC.Date" content="2017-09-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2017 Owen et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Extended laboratory culture and antimicrobial susceptibility testing timelines hinder rapid species identification and susceptibility profiling of bacterial pathogens associated with bovine respiratory disease, the most prevalent cause of cattle mortality in the United States. Whole-genome sequencing offers a culture-independent alternative to current bacterial identification methods, but requires a library of bacterial reference genomes for comparison. To contribute new bacterial genome assemblies and evaluate genetic diversity and variation in antimicrobial resistance genotypes, whole-genome sequencing was performed on bovine respiratory disease–associated bacterial isolates ( Histophilus somni , Mycoplasma bovis , Mannheimia haemolytica , and Pasteurella multocida ) from dairy and beef cattle. One hundred genomically distinct assemblies were added to the NCBI database, doubling the available genomic sequences for these four species. Computer-based methods identified 11 predicted antimicrobial resistance genes in three species, with none being detected in M. bovis . While computer-based analysis can identify antibiotic resistance genes within whole-genome sequences (genotype), it may not predict the actual antimicrobial resistance observed in a living organism (phenotype). Antimicrobial susceptibility testing on 64 H. somni , M. haemolytica , and P. multocida isolates had an overall concordance rate between genotype and phenotypic resistance to the associated class of antimicrobials of 72.7% ( P &lt; 0.001), showing substantial discordance. Concordance rates varied greatly among different antimicrobial, antibiotic resistance gene, and bacterial species combinations. This suggests that antimicrobial susceptibility phenotypes are needed to complement genomically predicted antibiotic resistance gene genotypes to better understand how the presence of antibiotic resistance genes within a given bacterial species could potentially impact optimal bovine respiratory disease treatment and morbidity/mortality outcomes." />
<meta name="DC.Contributor" content="Joseph R. Owen" />
<meta name="DC.Contributor" content="Noelle Noyes" />
<meta name="DC.Contributor" content="Amy E. Young" />
<meta name="DC.Contributor" content="Daniel J. Prince" />
<meta name="DC.Contributor" content="Patricia C. Blanchard" />
<meta name="DC.Contributor" content="Terry W. Lehenbauer" />
<meta name="DC.Contributor" content="Sharif S. Aly" />
<meta name="DC.Contributor" content="Jessica H. Davis" />
<meta name="DC.Contributor" content="Sean M. O’Rourke" />
<meta name="DC.Contributor" content="Zaid Abdo" />
<meta name="DC.Contributor" content="Keith Belk" />
<meta name="DC.Contributor" content="Michael R. Miller" />
<meta name="DC.Contributor" content="Paul Morley" />
<meta name="DC.Contributor" content="Alison L. Van Eenennaam" />
<meta name="article:published_time" content="2017-09-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Whole-Genome Sequencing and Concordance Between Antimicrobial Susceptibility Genotypes and Phenotypes of Bacterial Isolates Associated with Bovine Respiratory Disease" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Extended laboratory culture and antimicrobial susceptibility testing timelines hinder rapid species identification and susceptibility profiling of bacterial pathogens associated with bovine respiratory disease, the most prevalent cause of cattle mortality in the United States. Whole-genome sequencing offers a culture-independent alternative to current bacterial identification methods, but requires a library of bacterial reference genomes for comparison. To contribute new bacterial genome assemblies and evaluate genetic diversity and variation in antimicrobial resistance genotypes, whole-genome sequencing was performed on bovine respiratory disease–associated bacterial isolates (&lt;i&gt;Histophilus somni&lt;/i&gt;, &lt;i&gt;Mycoplasma bovis&lt;/i&gt;, &lt;i&gt;Mannheimia haemolytica&lt;/i&gt;, and &lt;i&gt;Pasteurella multocida&lt;/i&gt;) from dairy and beef cattle. One hundred genomically distinct assemblies were added to the NCBI database, doubling the available genomic sequences for these four species. Computer-based methods identified 11 predicted antimicrobial resistance genes in three species, with none being detected in &lt;i&gt;M. bovis&lt;/i&gt;. While computer-based analysis can identify antibiotic resistance genes within whole-genome sequences (genotype), it may not predict the actual antimicrobial resistance observed in a living organism (phenotype). Antimicrobial susceptibility testing on 64 &lt;i&gt;H. somni&lt;/i&gt;, &lt;i&gt;M. haemolytica&lt;/i&gt;, and &lt;i&gt;P. multocida&lt;/i&gt; isolates had an overall concordance rate between genotype and phenotypic resistance to the associated class of antimicrobials of 72.7% (&lt;i&gt;P&lt;/i&gt; &amp;lt; 0.001), showing substantial discordance. Concordance rates varied greatly among different antimicrobial, antibiotic resistance gene, and bacterial species combinations. This suggests that antimicrobial susceptibility phenotypes are needed to complement genomically predicted antibiotic resistance gene genotypes to better understand how the presence of antibiotic resistance genes within a given bacterial species could potentially impact optimal bovine respiratory disease treatment and morbidity/mortality outcomes.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2017/09/01" />
<meta name="citation_mjid" content="ggg;7/9/3059" />
<meta name="citation_id" content="7/9/3059" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/7/9/3059" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/7/9/3059.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/7/9/3059.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/7/9/3059.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.117.1137" />
<meta name="citation_pmid" content="28739600" />
<meta name="citation_volume" content="7" />
<meta name="citation_issue" content="9" />
<meta name="citation_num_pages" content="13" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="3059" />
<meta name="citation_lastpage" content="3071" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Joseph R. Owen" />
<meta name="citation_author_institution" content="Department of Animal Science, University of California, Davis, California 95616" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-7821-3012" />
<meta name="citation_author" content="Noelle Noyes" />
<meta name="citation_author_institution" content="Department of Microbiology, Immunology, and Pathology, College of Veterinary Medicine and Biomedical Sciences, Colorado State University, Fort Collins, Colorado 80523" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-6149-1008" />
<meta name="citation_author" content="Amy E. Young" />
<meta name="citation_author_institution" content="Department of Animal Science, University of California, Davis, California 95616" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-4181-4233" />
<meta name="citation_author" content="Daniel J. Prince" />
<meta name="citation_author_institution" content="Department of Animal Science, University of California, Davis, California 95616" />
<meta name="citation_author" content="Patricia C. Blanchard" />
<meta name="citation_author_institution" content="California Animal Health and Food Safety Laboratory, School of Veterinary Medicine, University of California, Davis, Tulare, California 93274" />
<meta name="citation_author" content="Terry W. Lehenbauer" />
<meta name="citation_author_institution" content="Veterinary Medicine Teaching and Research Center, School of Veterinary Medicine, University of California, Davis, Tulare, California 93274" />
<meta name="citation_author_institution" content="Department of Population Health and Reproduction, School of Veterinary Medicine, University of California, Davis, California 95616" />
<meta name="citation_author" content="Sharif S. Aly" />
<meta name="citation_author_institution" content="Veterinary Medicine Teaching and Research Center, School of Veterinary Medicine, University of California, Davis, Tulare, California 93274" />
<meta name="citation_author_institution" content="Department of Population Health and Reproduction, School of Veterinary Medicine, University of California, Davis, California 95616" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-0330-5013" />
<meta name="citation_author" content="Jessica H. Davis" />
<meta name="citation_author_institution" content="Veterinary Medicine Teaching and Research Center, School of Veterinary Medicine, University of California, Davis, Tulare, California 93274" />
<meta name="citation_author" content="Sean M. O’Rourke" />
<meta name="citation_author_institution" content="Department of Animal Science, University of California, Davis, California 95616" />
<meta name="citation_author" content="Zaid Abdo" />
<meta name="citation_author_institution" content="Department of Microbiology, Immunology, and Pathology, College of Veterinary Medicine and Biomedical Sciences, Colorado State University, Fort Collins, Colorado 80523" />
<meta name="citation_author" content="Keith Belk" />
<meta name="citation_author_institution" content="Department of Animal Sciences, College of Agricultural Sciences, Colorado State University, Fort Collins, Colorado 80523" />
<meta name="citation_author" content="Michael R. Miller" />
<meta name="citation_author_institution" content="Department of Animal Science, University of California, Davis, California 95616" />
<meta name="citation_author" content="Paul Morley" />
<meta name="citation_author_institution" content="Department of Clinical Sciences, College of Veterinary Medicine and Biomedical Sciences, Colorado State University, Fort Collins, Colorado 80523" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-8138-2714" />
<meta name="citation_author" content="Alison L. Van Eenennaam" />
<meta name="citation_author_institution" content="Department of Animal Science, University of California, Davis, California 95616" />
<meta name="citation_author_institution" content="Veterinary Medicine Teaching and Research Center, School of Veterinary Medicine, University of California, Davis, Tulare, California 93274" />
<meta name="citation_author_email" content="alvaneenennaam@ucdavis.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-1562-162X" />
<meta name="citation_reference" content="citation_journal_title=J. Clin. Microbiol.;citation_author=A. Agyekum;citation_author=A. Fajardo-Lubian;citation_author=X. Ai;citation_author=A. N. Ginn;citation_author=Z. Zong;citation_title=Predictability of phenotype in relation to common betalactam resistance mechanisms in Escherichia coli and Klebsiella pneumoniae.;citation_pages=1243-1250;citation_volume=15;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Veterinary microbiology;citation_journal_abbrev=Vet Microbiol;citation_author=T. W. Alexander;citation_author=S. R. Cook;citation_author=L. J. Yanke;citation_title=A multiplex polymerase chain reaction assay for the identification of Mannheimia haemolytica, Mannheimia glucosida and Mannheimia ruminalis.;citation_pages=165-175;citation_volume=130;citation_year=2008;citation_issue=1-2;citation_pmid=18308486;citation_doi=10.1016/j.vetmic.2008.01.001" />
<meta name="citation_reference" content="citation_journal_title=J. Clin. Microbiol.;citation_author=M. W. Allard;citation_author=E. Strain;citation_author=D. Melka;citation_author=K. Bunning;citation_author=S. M. Musser;citation_title=Practical value of food pathogen traceability through building a whole-genome sequencing network and database.;citation_volume=54;citation_year=2016;citation_pmid=27008877;citation_doi=10.1128/JCM.00081-16" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=S. F. Altschul;citation_author=W. Gish;citation_author=D. J. Lipman;citation_title=Basic local alignment search tool.;citation_pages=403-410;citation_volume=215;citation_year=1990;citation_issue=3;citation_pmid=2231712;citation_doi=10.1006/jmbi.1990.9999" />
<meta name="citation_reference" content="citation_journal_title=Journal of computational biology :  a journal of computational molecular cell biology;citation_journal_abbrev=J Comput Biol;citation_author=A. Bankevich;citation_author=S. Nurk;citation_author=D. Antipov;citation_author=A. A. Gurevich;citation_author=M. Dvorkin;citation_title=SPAdes: a new genome assembly algorithm and its applications to single-cell sequencing.;citation_pages=455-477;citation_volume=19;citation_year=2012;citation_issue=5;citation_pmid=22506599;citation_doi=10.1089/cmb.2012.0021" />
<meta name="citation_reference" content="citation_journal_title=Euro Surveill.;citation_author=B. M. Berenger;citation_author=C. Berry;citation_author=T. Peterson;citation_author=P. Fach;citation_author=S. Delannoy;citation_title=The utility of multiple molecular methods including whole genome sequencing as tools to differentiate Escherichia coli O157:H7 outbreaks.;citation_pages=30073;citation_volume=20;citation_year=2015;citation_doi=10.2807/1560-7917.ES.2015.20.47.30073" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=L. S. Burall;citation_author=C. J. Grim;citation_author=M. K. Mammel;citation_author=A. R. Datta;citation_title=Whole genome sequence analysis using JSpecies tool establishes clonal relationships between Listeria monocytogenes strains from epidemiologically unrelated Listeriosis outbreaks.;citation_pages=e0150797;citation_volume=11;citation_year=2016;citation_pmid=26950338;citation_doi=10.1371/journal.pone.0150797" />
<meta name="citation_reference" content="citation_journal_title=Antimicrob. Agents Chemother.;citation_author=A. Carattoli;citation_author=E. Zankari;citation_author=A. García-Fernández;citation_author=L. M. Voldby;citation_author=O. Lund;citation_title=In silico detection and typing of plasmids using PlasmidFinder and plasmid multilocus sequence typing.;citation_volume=58;citation_year=2014;citation_pmid=24777092;citation_doi=10.1128/AAC.02412-14" />
<meta name="citation_reference" content="citation_journal_title=Antimicrob. Agents Chemother.;citation_author=R. Card;citation_author=J. Zhang;citation_author=P. Das;citation_author=C. Cook;citation_author=N. Woodford;citation_title=Evaluation of an expanded microarray for detecting antimicrobial resistance genes in a broad range of gram-negative bacterial pathogens.;citation_volume=57;citation_year=2013;citation_pmid=23129055;citation_doi=10.1128/AAC.01223-12" />
<meta name="citation_reference" content="citation_journal_title=J. Bacteriol.;citation_author=J. F. Challacombe;citation_author=A. J. Duncan;citation_author=T. S. Brettin;citation_author=D. Bruce;citation_author=O. Chertkov;citation_title=Complete genome sequence of Haemophilus somnus (Histophilus somni) strain 129Pt and comparison to Haemophilus ducreyi 35000HP and Haemophilus influenzae Rd.;citation_volume=189;citation_year=2007;citation_pmid=17172329;citation_doi=10.1128/JB.01422-06" />
<meta name="citation_reference" content="citation_journal_title=Proc. Am. Soc. Anim. Sci. West. Sec.;citation_author=N. K. Chirase;citation_author=L. W. Green;citation_title=Influence of oral natural interferon-a on performance and rectal temperature of newly received beef steers.;citation_pages=411-414;citation_volume=51;citation_year=2000" />
<meta name="citation_reference" content="citation_journal_title=J. Antimicrob. Chemother.;citation_author=P. T. Clausen;citation_author=E. Zankari;citation_author=F. M. Aarestrup;citation_author=O. Lund;citation_title=Benchmarking of methods for identification of antimicrobial resistance genes in bacterial whole genome data.;citation_volume=71;citation_year=2016;citation_pmid=27365186;citation_doi=10.1093/jac/dkw184" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=M. L. Clawson;citation_author=R. W. Murray;citation_author=M. T. Sweeney;citation_author=M. D. Apley;citation_author=K. D. DeDonder;citation_title=Genomic signatures of Mannheimia haemolytica that associate with the lungs of cattle with respiratory disease, an integrative conjugative element, and antimicrobial resistance genes.;citation_pages=982;citation_volume=17;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Veterinary microbiology;citation_journal_abbrev=Vet Microbiol;citation_author=G. H. D’Amours;citation_author=T. I. Ward;citation_author=M. R. Mulvey;citation_author=R. R. Read;citation_author=D. W. Morck;citation_title=Genetic diversity and tetracycline resistance genes of Histophilus somni.;citation_pages=362-372;citation_volume=150;citation_year=2011;citation_issue=3-4;citation_pmid=21482045" />
<meta name="citation_reference" content="citation_journal_title=Genome Announcements;citation_journal_abbrev=Genome Announc;citation_author=K. W. Davenport;citation_author=H. E. Daligault;citation_author=T. D. Minogue;citation_author=K. A. Bishop-Lilly;citation_author=D. C. Bruce;citation_title=Complete Genome Sequence of Type Strain Pasteurella multocida subsp. multocida ATCC 43137;citation_pages=e01070-14-e01070-14;citation_volume=2;citation_year=2014;citation_issue=5;citation_pmid=25342682;citation_doi=10.1128/genomeA.01070-14" />
<meta name="citation_reference" content="citation_journal_title=Anim. Health Res. Rev.;citation_author=K. D. DeDonder;citation_author=M. D. Apley;citation_title=A literature review of antimicrobial resistance in pathogens associated with bovine respiratory disease.;citation_pages=125-134;citation_volume=16;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Psychometrika;citation_journal_abbrev=Psychometrika;citation_author=A. L. Edwards;citation_title=Note on the correction for continuity in testing the significance of the difference between correlated proportions.;citation_pages=185-187;citation_volume=13;citation_year=1948;citation_issue=3;citation_pmid=18885738" />
<meta name="citation_reference" content="citation_journal_title=Genome Announc.;citation_author=C. Eidam;citation_author=A. Poehlein;citation_author=M. G. Brenner;citation_author=K. Kadlec;citation_author=H. Liesegang;citation_title=Complete genome sequence of Mannheimia haemolytica strain 42548 from a case of bovine respiratory disease.;citation_pages=e00318-e13;citation_volume=1;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=J. Antimicrob. Chemother.;citation_author=C. Eidam;citation_author=A. Poehlein;citation_author=A. Leimbach;citation_author=G. B. Michael;citation_author=K. Kadlec;citation_title=Analysis and comparative genomics of ICEMh1, a novel integrative and conjugative element (ICE) of Mannheimia haemolytica.;citation_volume=70;citation_year=2015;citation_pmid=25239467;citation_doi=10.1093/jac/dku361" />
<meta name="citation_reference" content="citation_journal_title=BMC Med. Res. Methodol.;citation_author=M. W. Fagerland;citation_author=S. Lydersen;citation_author=P. Laake;citation_title=The McNemar test for binary matched-pairs data: mid-p and asymptotic are better than exact conditional.;citation_pages=91-98;citation_volume=13;citation_year=2013;citation_pmid=23848987;citation_doi=10.1186/1471-2288-13-91" />
<meta name="citation_reference" content="citation_title=National Antimicrobial Resistance Monitoring System – Enteric Bacteria (NARMS): 2011 Executive Report;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Genome;citation_author=S. Fitz-Gibbon;citation_author=A. Hipp;citation_author=K. Pham;citation_author=P. Manos;citation_author=V. L. Sork;citation_title=Phylogenomic inferences from reference-mapped and de novo assembled short- read sequence data using RADseq sequencing of California white oaks (Quercus subgenus Quercus).;citation_year=2017;citation_doi=10.1139/gen-2016&amp;ndash;0202" />
<meta name="citation_reference" content="citation_journal_title=Can. Vet. J.;citation_author=R. W. Fulton;citation_author=A. W. Confer;citation_title=Laboratory test descriptions for bovine respiratory disease diagnosis and their strengths and weaknesses: gold standards for diagnosis, do they exist?;citation_pages=754-761;citation_volume=53;citation_year=2012;citation_pmid=23277642" />
<meta name="citation_reference" content="citation_journal_title=Public Health Genomics;citation_author=M. W. Gilmour;citation_author=M. Graham;citation_author=A. Reimer;citation_author=G. Van Domselaar;citation_title=Public health genomics and the new molecular epidemiology of bacterial pathogens.;citation_pages=25-30;citation_volume=16;citation_year=2013;citation_pmid=23548714;citation_doi=10.1159/000342709" />
<meta name="citation_reference" content="citation_journal_title=The Veterinary clinics of North America. Food animal practice;citation_journal_abbrev=Vet Clin North Am Food Anim Pract;citation_author=D. Griffin;citation_author=M. Chengappa;citation_author=J. Kuszak;citation_author=D. S. McVey;citation_title=Bacterial pathogens of the bovine respiratory disease complex.;citation_pages=381-394;citation_volume=26;citation_year=2010;citation_issue=2;citation_pmid=20619191;citation_doi=10.1016/j.cvfa.2010.04.004" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=M. Gouy;citation_author=S. Guindon;citation_author=O. Gascuel;citation_title=SeaView version 4: a multiplatform graphical user interface for sequence alignment and phylogenetic tree building.;citation_volume=27;citation_year=2010;citation_pmid=19854763;citation_doi=10.1093/molbev/msp259" />
<meta name="citation_reference" content="citation_journal_title=Genome Announc.;citation_author=G. P. Harhay;citation_author=S. Koren;citation_author=A. M. Phillippy;citation_author=D. S. McVey;citation_author=J. Kuszak;citation_title=Complete closed genome sequences of Mannheimia haemolytica serotypes A1 and A6, isolated from cattle.;citation_pages=e00188-e13;citation_volume=1;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Genome Announc.;citation_author=M. J. Hauglund;citation_author=F. M. Tatum;citation_author=D. O. Bayles;citation_author=S. K. Maheswaran;citation_author=R. E. Briggs;citation_title=Genome sequences of Mannheimia haemolytica serotype A1 strains D153 and D193 from bovine pneumonia.;citation_pages=e00848-e13;citation_volume=1;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Genome Announc.;citation_author=M. J. Hauglund;citation_author=F. M. Tatum;citation_author=D. O. Bayles;citation_author=S. K. Maheswaran;citation_author=R. E. Briggs;citation_title=Genome sequences of Mannheimia haemolytica serotype A2 isolates D171 and D35, recovered from bovine pneumonia.;citation_pages=e00093-e15;citation_volume=3;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Genome Announc.;citation_author=M. J. Hauglund;citation_author=F. M. Tatum;citation_author=D. O. Bayles;citation_author=S. K. Maheswaran;citation_author=R. E. Briggs;citation_title=Genome sequences of serotype A6 Mannheimia haemolytica isolates D174 and D38 recovered from bovine pneumonia.;citation_pages=e00086-e15;citation_volume=3;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Genome Announc.;citation_author=M. P. Heaton;citation_author=G. P. Harhay;citation_author=T. P. Smith;citation_author=J. L. Bono;citation_author=C. G. Chitko-McKown;citation_title=Complete closed genome sequences of a Mannheimia haemolytica serotype A1 leukotoxin deletion mutant and its wild-type parent strain.;citation_pages=e00417-e15;citation_volume=3;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Genome Med.;citation_author=M. Inouye;citation_author=H. Dashnow;citation_author=L. Raven;citation_author=M. B. Schultz;citation_author=B. J. Pope;citation_title=SRST2: rapid genomic surveillance for public health and hospital microbiology labs.;citation_pages=90-106;citation_volume=6;citation_year=2014;citation_pmid=25422674;citation_doi=10.1186/s13073-014-0090-6" />
<meta name="citation_reference" content="citation_journal_title=Diagn. Microbiol. Infect. Dis.;citation_author=J. M. Janda;citation_author=S. A. Abbot;citation_title=Culture-independent diagnostic testing: have we opened Pandora’s box for good?;citation_pages=171-176;citation_volume=80;citation_year=2014;citation_doi=10.1016/j.diagmicrobio.2014.08.001" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=B. Jia;citation_author=A. R. Raphenya;citation_author=B. Alcock;citation_author=N. Waglechner;citation_author=P. Guo;citation_title=CARD 2017: expansion and model-centric curation of the comprehensive antibiotic resistance database.;citation_volume=45;citation_year=2017;citation_pmid=27789705;citation_doi=10.1093/nar/gkw1004" />
<meta name="citation_reference" content="citation_journal_title=Antimicrob. Agents Chemother.;citation_author=K. Kadlec;citation_author=M. G. Brenner;citation_author=M. T. Sweeney;citation_author=E. Brzuszkiewicz;citation_author=H. Liesegang;citation_title=Molecular basis of macrolide, triamilide, and lincosamide resistance in Pasteurella multocida from bovine respiratory disease.;citation_volume=55;citation_year=2011;citation_pmid=21402855;citation_doi=10.1128/AAC.00092-11" />
<meta name="citation_reference" content="citation_journal_title=Journal of Antimicrobial Chemotherapy;citation_journal_abbrev=J Antimicrob Chemother;citation_author=J. A. Karlowsky;citation_author=G. Verma;citation_author=G. G. Zhanel;citation_author=D. J. Hoban;citation_title=Presence of ROB-1 {beta}-lactamase correlates with cefaclor resistance among recent isolates of Haemophilus influenzae;citation_pages=871-875;citation_volume=45;citation_year=2000;citation_issue=6;citation_pmid=10837442;citation_doi=10.1093/jac/45.6.871" />
<meta name="citation_reference" content="citation_journal_title=Anim. Sci. J.;citation_author=K. Kawai;citation_author=H. Higuchi;citation_author=H. Iwano;citation_author=A. Iwakuma;citation_author=K. Onda;citation_title=Antimicrobial susceptibilities of Mycoplasma isolated from bovine mastitis in Japan.;citation_pages=96-99;citation_volume=85;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Journal of Antimicrobial Chemotherapy;citation_journal_abbrev=J Antimicrob Chemother;citation_author=C. Kehrenberg;citation_author=S. A. Salmon;citation_author=J. L. Watts;citation_author=S. Schwarz;citation_title=Tetracycline resistance genes in isolates of Pasteurella multocida, Mannheimia haemolytica, Mannheimia glucosida and Mannheimia varigena from bovine and swine respiratory disease: intergeneric spread of the tet(H) plasmid pMHT1;citation_pages=631-640;citation_volume=48;citation_year=2001;citation_issue=5;citation_pmid=11679552;citation_doi=10.1093/jac/48.5.631" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=W. J. Kent;citation_title=BLAT–the BLAST-like alignment tool.;citation_volume=12;citation_year=2002;citation_pmid=11932250;citation_doi=10.1101/gr.229202. Article published online before March 2002" />
<meta name="citation_reference" content="citation_journal_title=J. Clin. Microbiol.;citation_author=C. L. Klima;citation_author=R. Zaheer;citation_author=S. R. Cook;citation_author=C. W. Booker;citation_author=S. Hendrick;citation_title=Pathogens of bovine respiratory disease in North American feedlots conferring multidrug resistance via integrative conjugative elements.;citation_volume=52;citation_year=2014;citation_pmid=24478472;citation_doi=10.1128/JCM.02485-13" />
<meta name="citation_reference" content="citation_journal_title=J. Vet. Med. Sci.;citation_author=L. C. Kong;citation_author=D. Goa;citation_author=B. Y. Jia;citation_author=Z. Wang;citation_author=Y. H. Gao;citation_title=Antimicrobial susceptibility and molecular characterization of macrolide resistance of Mycoplasma bovis isolates from multiple provinces in China.;citation_pages=293-296;citation_volume=78;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=S. Koren;citation_author=G. P. Harhay;citation_author=T. P. Smith;citation_author=J. L. Bono;citation_author=D. M. Harhay;citation_title=Reducing assembly complexity of microbial genomes with single-molecule sequencing.;citation_pages=R101;citation_volume=14;citation_year=2013;citation_pmid=24034426;citation_doi=10.1186/gb-2013-14-9-r101" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=T. Koressaar;citation_author=M. Remm;citation_title=Enhancements and modifications of primer design program Primer3.;citation_volume=23;citation_year=2007;citation_pmid=17379693;citation_doi=10.1093/bioinformatics/btm091" />
<meta name="citation_reference" content="citation_journal_title=BMC Bioinformatics;citation_author=T. S. Korneliussen;citation_author=A. Albrechtsen;citation_author=R. Nielsen;citation_title=ANGSD: analysis of next generation sequencing data.;citation_pages=356;citation_volume=15;citation_year=2014;citation_pmid=25420514;citation_doi=10.1186/s12859-014-0356-4" />
<meta name="citation_reference" content="citation_journal_title=Trends Genet.;citation_author=C. U. Koser;citation_author=M. J. Ellington;citation_author=S. J. Peacock;citation_title=Whole genome sequencing to control antimicrobial resistance.;citation_pages=401-407;citation_volume=30;citation_year=2014;citation_pmid=25096945;citation_doi=10.1016/j.tig.2014.07.003" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=B. Langmead;citation_author=S. Salzberg;citation_title=Fast gapped-read alignment with Bowtie 2.;citation_pages=357-359;citation_volume=9;citation_year=2012;citation_issue=4;citation_pmid=22388286;citation_doi=10.1038/nmeth.1923" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=B. Handsaker;citation_author=A. Wysoker;citation_author=T. Fennell;citation_author=J. Ruan;citation_title=The sequence alignment/map (SAM) format and SAMtools.;citation_volume=25;citation_year=2009;citation_pmid=19505943;citation_doi=10.1093/bioinformatics/btp352" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=Y. Li;citation_author=H. Zheng;citation_author=Y. Liu;citation_author=Y. Jiang;citation_author=J. Xin;citation_title=The complete genome sequence of Mycoplasma bovis strain Hubei-1.;citation_pages=e20999-e20999;citation_volume=6;citation_year=2011;citation_issue=6;citation_pmid=21731639;citation_doi=10.1371/journal.pone.0020999" />
<meta name="citation_reference" content="citation_journal_title=Journal of Bacteriology;citation_journal_abbrev=J. Bacteriol.;citation_author=W. Liu;citation_author=M. Yang;citation_author=Z. Xu;citation_author=H. Zheng;citation_author=W. Liang;citation_title=Complete Genome Sequence of Pasteurella multocida HN06, a Toxigenic Strain of Serogroup D;citation_pages=3292-3293;citation_volume=194;citation_year=2012;citation_issue=12;citation_pmid=22628516;citation_doi=10.1128/JB.00215-12" />
<meta name="citation_reference" content="citation_journal_title=Journal of Antimicrobial Chemotherapy;citation_journal_abbrev=J Antimicrob Chemother;citation_author=D. M. Livermore;citation_author=T. G. Winstanley;citation_author=K. P. Shannon;citation_title=Interpretative reading: recognizing the unusual and inferring resistance mechanisms from resistance phenotypes;citation_pages=87-102;citation_volume=48;citation_year=2001;citation_issue=90001;citation_pmid=11420342;citation_doi=10.1093/jac/47.1.87" />
<meta name="citation_reference" content="citation_journal_title=PeerJ;citation_author=W. J. Love;citation_author=T. W. Lehenbauer;citation_author=P. H. Kass;citation_author=A. L. Van Eenennaam;citation_author=S. S. Aly;citation_title=Development of a novel clinical scoring system for on-farm diagnosis of bovine respiratory disease in pre-weaned dairy calves.;citation_pages=e238;citation_volume=2;citation_year=2014;citation_pmid=24482759;citation_doi=10.7717/peerj.238" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=J. W. Marsh;citation_author=M. G. Krauland;citation_author=J. S. Nelson;citation_author=J. L. Schlackman;citation_author=A. M. Brooks;citation_title=Genomic epidemiology of an endoscope-associated outbreak of Klebsiella pneumoniae Carbapenemase (KPC)-producing K. pneumoniae.;citation_pages=e0144310;citation_volume=10;citation_year=2015;citation_pmid=26637170;citation_doi=10.1371/journal.pone.0144310" />
<meta name="citation_reference" content="citation_journal_title=Journal of veterinary internal medicine / American College of Veterinary Internal Medicine;citation_journal_abbrev=J Vet Intern Med;citation_author=F. P. Maunsell;citation_author=A. R. Woolums;citation_author=D. Francoz;citation_author=R. F. Rosenbusch;citation_author=D. L. Step;citation_title=Mycoplasma bovis infections in cattle.;citation_pages=772-783;citation_volume=25;citation_year=2011;citation_issue=4;citation_pmid=21745245;citation_doi=10.1111/j.1939-1676.2011.0750.x" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=B. J. May;citation_author=Q. Zhang;citation_author=L. L. Li;citation_author=M. L. Paustian;citation_author=T. S. Whittam;citation_title=Complete genomic sequence of Pasteurella multocida, Pm70.;citation_volume=98;citation_year=2001;citation_pmid=11248100;citation_doi=10.1073/pnas.051634598" />
<meta name="citation_reference" content="citation_journal_title=The Veterinary clinics of North America. Food animal practice;citation_journal_abbrev=Vet Clin North Am Food Anim Pract;citation_author=S. M. McGuirk;citation_title=Disease management of dairy calves and heifers.;citation_pages=139-153;citation_volume=24;citation_year=2008;citation_issue=1;citation_pmid=18299036;citation_doi=10.1016/j.cvfa.2007.10.003" />
<meta name="citation_reference" content="citation_journal_title=J. Antimicrob. Chemother.;citation_author=G. B. Michael;citation_author=K. Kadlec;citation_author=M. T. Sweeney;citation_author=E. Brzuszkiewicz;citation_author=H. Liesegang;citation_title=ICEPmu1, an integrative conjugative element (ICE) of Pasteurella multocida: structure and transfer.;citation_pages=91-100;citation_volume=67;citation_year=2011;citation_pmid=22001176" />
<meta name="citation_reference" content="citation_journal_title=BMC Res. Notes;citation_author=M. Mohammed;citation_author=M. Cormican;citation_title=Whole genome sequencing provides possible explanations for the difference in phage susceptibility among two Salmonella Typhimurium phage types (DT8 and DT30) associated with a single foodborne outbreak.;citation_pages=728;citation_volume=8;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=EBioMedicine;citation_author=M. M. Mustapha;citation_author=J. W. Marsh;citation_author=M. G. Krauland;citation_author=J. O. Fernandez;citation_author=A. P. de Lemos;citation_title=Genomic epidemiology of hypervirulent serogroup W, ST-11 Neisseria meningitidis.;citation_pages=1447-1455;citation_volume=2;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=A. Mutreja;citation_author=D. W. Kim;citation_author=N. R. Thomson;citation_author=T. R. Connor;citation_author=J. H. Lee;citation_title=Evidence for several waves of global transmission in the seventh cholera pandemic.;citation_pages=462-465;citation_volume=477;citation_year=2011;citation_issue=7365;citation_pmid=21866102;citation_doi=10.1038/nature10392" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=H. L. Neibergs;citation_author=C. M. Seabury;citation_author=A. J. Wojtowicz;citation_author=Z. Wang;citation_author=E. Scraggs;citation_title=Susceptibility loci revealed for bovine respiratory disease complex in pre-weaned Holstein calves.;citation_pages=1164;citation_volume=15;citation_year=2014;citation_pmid=25534905;citation_doi=10.1186/1471-2164-15-1164" />
<meta name="citation_reference" content="citation_journal_title=J. Vet. Intern. Med.;citation_author=N. R. Noyes;citation_author=K. M. Benedict;citation_author=S. P. Gow;citation_author=C. W. Booker;citation_author=S. J. Hannon;citation_title=Mannheimia haemolytica in feedlot cattle: prevalence of recovery and associations with antimicrobial use, resistance, and health outcomes.;citation_pages=705-713;citation_volume=29;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Gene;citation_author=Z. Peng;citation_author=W. Liang;citation_author=W. Liu;citation_author=B. Wu;citation_author=B. Tang;citation_title=Genomic characterization of Pasteurella multocida HB01, a serotype A bovine isolate from China.;citation_pages=85-93;citation_volume=581;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=J. Qi;citation_author=A. Guo;citation_author=P. Cui;citation_author=Y. Chen;citation_author=R. Mustafa;citation_title=Comparative geno-plasticity analysis of Mycoplasma bovis HB0801 (Chinese isolate).;citation_pages=e38239-e38239;citation_volume=7;citation_year=2012;citation_issue=5;citation_pmid=22693604;citation_doi=10.1371/journal.pone.0038239" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=S. Siddaramappa;citation_author=J. F. Challacombe;citation_author=A. J. Duncan;citation_author=A. F. Gillaspy;citation_author=M. Carson;citation_title=Horizontal gene transfer in Histophilus somni and its role in the evolution of pathogenic strain 2336, as determined by comparative genomic analyses.;citation_pages=570-570;citation_volume=12;citation_year=2011;citation_pmid=22111657;citation_doi=10.1186/1471-2164-12-570" />
<meta name="citation_reference" content="citation_author=D. A. Street;citation_author=K. R. Petersen;citation_author=A. T. Gerritsen;citation_author=S. S. Hunter;citation_author=M. L. Settles;citation_pages=523-524;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=VetScan.;citation_author=W. Ullah;citation_author=M. Abubakar;citation_author=M. J. Arshed;citation_title=Differentiation of closely related vaccinal strains of Pasteurella multocida using polymerase chain reaction (PCR).;citation_pages=36;citation_volume=4;citation_year=2009" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=A. Untergrasser;citation_author=I. Cutcutache;citation_author=T. Koressaar;citation_author=J. Ye;citation_author=B. C. Faircloth;citation_title=Primer3 - new capabilities and interfaces.;citation_volume=40;citation_year=2012;citation_pmid=22730293;citation_doi=10.1093/nar/gks596" />
<meta name="citation_reference" content="citation_journal_title=Infect. Immun.;citation_author=K. S. Wise;citation_author=M. J. Calcutt;citation_author=M. F. Foecking;citation_author=K. Roske;citation_author=R. Madupu;citation_title=Complete genome sequence of Mycoplasma bovis type strain PG45 (ATCC 25523).;citation_pages=982-983;citation_volume=79;citation_year=2010" />
<meta name="citation_reference" content="citation_journal_title=J. Antimicrob. Chemother.;citation_author=E. Zankari;citation_author=H. Hasman;citation_author=S. Cosentino;citation_author=M. Vestergaard;citation_author=S. Rasmussen;citation_title=Identification of acquired antimicrobial resistance genes.;citation_volume=67;citation_year=2012;citation_pmid=22782487;citation_doi=10.1093/jac/dks261" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Whole-Genome Sequencing and Concordance Between Antimicrobial Susceptibility Genotypes and Phenotypes of Bacterial Isolates Associated with Bovine Respiratory Disease" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9.cover-source.jpg" />
<meta name="twitter:description" content="Extended laboratory culture and antimicrobial susceptibility testing timelines hinder rapid species identification and susceptibility profiling of bacterial pathogens associated with bovine respiratory disease, the most prevalent cause of cattle mortality in the United States. Whole-genome sequencing offers a culture-independent alternative to current bacterial identification methods, but requires a library of bacterial reference genomes for comparison. To contribute new bacterial genome assemblies and evaluate genetic diversity and variation in antimicrobial resistance genotypes, whole-genome sequencing was performed on bovine respiratory disease–associated bacterial isolates ( Histophilus somni , Mycoplasma bovis , Mannheimia haemolytica , and Pasteurella multocida ) from dairy and beef cattle. One hundred genomically distinct assemblies were added to the NCBI database, doubling the available genomic sequences for these four species. Computer-based methods identified 11 predicted antimicrobial resistance genes in three species, with none being detected in M. bovis . While computer-based analysis can identify antibiotic resistance genes within whole-genome sequences (genotype), it may not predict the actual antimicrobial resistance observed in a living organism (phenotype). Antimicrobial susceptibility testing on 64 H. somni , M. haemolytica , and P. multocida isolates had an overall concordance rate between genotype and phenotypic resistance to the associated class of antimicrobials of 72.7% ( P &lt; 0.001), showing substantial discordance. Concordance rates varied greatly among different antimicrobial, antibiotic resistance gene, and bacterial species combinations. This suggests that antimicrobial susceptibility phenotypes are needed to complement genomically predicted antibiotic resistance gene genotypes to better understand how the presence of antibiotic resistance genes within a given bacterial species could potentially impact optimal bovine respiratory disease treatment and morbidity/mortality outcomes." />
<meta name="og-title" property="og:title" content="Whole-Genome Sequencing and Concordance Between Antimicrobial Susceptibility Genotypes and Phenotypes of Bacterial Isolates Associated with Bovine Respiratory Disease" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/7/9/3059" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Extended laboratory culture and antimicrobial susceptibility testing timelines hinder rapid species identification and susceptibility profiling of bacterial pathogens associated with bovine respiratory disease, the most prevalent cause of cattle mortality in the United States. Whole-genome sequencing offers a culture-independent alternative to current bacterial identification methods, but requires a library of bacterial reference genomes for comparison. To contribute new bacterial genome assemblies and evaluate genetic diversity and variation in antimicrobial resistance genotypes, whole-genome sequencing was performed on bovine respiratory disease–associated bacterial isolates ( Histophilus somni , Mycoplasma bovis , Mannheimia haemolytica , and Pasteurella multocida ) from dairy and beef cattle. One hundred genomically distinct assemblies were added to the NCBI database, doubling the available genomic sequences for these four species. Computer-based methods identified 11 predicted antimicrobial resistance genes in three species, with none being detected in M. bovis . While computer-based analysis can identify antibiotic resistance genes within whole-genome sequences (genotype), it may not predict the actual antimicrobial resistance observed in a living organism (phenotype). Antimicrobial susceptibility testing on 64 H. somni , M. haemolytica , and P. multocida isolates had an overall concordance rate between genotype and phenotypic resistance to the associated class of antimicrobials of 72.7% ( P &lt; 0.001), showing substantial discordance. Concordance rates varied greatly among different antimicrobial, antibiotic resistance gene, and bacterial species combinations. This suggests that antimicrobial susceptibility phenotypes are needed to complement genomically predicted antibiotic resistance gene genotypes to better understand how the presence of antibiotic resistance genes within a given bacterial species could potentially impact optimal bovine respiratory disease treatment and morbidity/mortality outcomes." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9.cover-source.jpg" />
    <title>Whole-Genome Sequencing and Concordance Between Antimicrobial Susceptibility Genotypes and Phenotypes of Bacterial Isolates Associated with Bovine Respiratory Disease | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"r3owSD2_hiKOEto4AS8drqC_mOm56pZGQBOvk1vh0f4","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"486354","apath":"\/ggg\/7\/9\/3059.atom","pisa":"ggg;7\/9\/3059","ac":{"\/ggg\/7\/9\/3059.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/7\/9\/3059.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;7\/9\/3059"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;7\/9\/3059"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/7\/9\/3059":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022\\\\\\\/NFJqKi+ayve1bTT6g4kXyB86INsYTEYlBZ4x4Jmq9GlS+nUVKh3CGB3P94TH27BZ6pXAE1mwuHwgtV3T1zy29EXQRDJqcU9NPzZsa6amQU6q6afktJ9PmXQOWcgXPxbLLnChtFMzXIaEg+Y9o05CZDv+2dINA84DHY6ux3JuQHZiCaxIdyrtyoPWpv\\\\\\\/F5liA+S\\\\\\\/Y9JeEDWBI2T9dQSL05X6s4855Ka4eyRa7pBFSEQnG9Q8Y8cEStTpq63CWnHA+330ZGXlHAurrNVxNVJ1bvu7nULh3DSCAD7oxA7AdKVF7xe6bGzJAVww02qvPFDkSTOV+N784vOrrim78L3BOlKrDZ0TAlq5plCwKpBmPt+dZ2tHkw8rd2oSOCRU56OeZS4u9qJoi2hflYInq5EsE5IyysHexRHiIHNoMk+eW9by0rtQD3jnMMJXkxNmVM5QMkSf1Pu2YAuEtYv9YBknb8TXolzsAliCu\\\\\\\/xrqVBqIj54HlshDgbElr+z7VipjavTPf+OvbF0mWOe+Ok2XZPCF6aJ2zDLQj5cZI8cFjclR4tkEAzdRXJp1vOMTpf3k7Xxiv0CRo9Wo4hiUqzcDXMKbJvGCV5HLIpavxpgvEpwk5ztgZKInPhxnqwMDtC70yme9NLbJ6ToAwt+d6c0TBTDZmTsESw6eaEuMr3BFpKp4L7icmlZAdVFIoYkXuwrzTCBPf3pmvIavn8So0f9PMQeQw\\\\\\\/bxIY0xaOB+3s4xN7K4uAyXwLJgmdHnm62\\\\\\\/psAlSQAQJD2xdNJj+J\\\\\\\/zfKaE0u1LlGZw5Kp+bJDJ+swZPJrcD7pef11LedD6qYb+R0BKJ+jaggldQjbqMPH9+0RFGHP5ufwXGRaTqYY8aj2JcgQJ3XHGf\\\\\\\/UXhCxaoYnRO6RillLY+tkQC3L2KOGz+8\\\\\\\/GDbYh9BG\\\\\\\/uualXC8KCjJoOIH9aMjKZvR53vRkUouRhVYXhcgz21tlw9VMoyuIlv6dZe5ZrAp\\\\\\\/ph9+khEY9oF5Gz4sCfs0KfquUo0gwJFLkCCqg9vhRf+17zHnAV9YITtBfWX6cF+oH72Udl5+T+HZ4ITTDxTVG4FkWrXW1KvGNbq9SVqh3TZS9H4hai8U2owCUoiKxAiOxj0M47SMy\\\\\\\/kcoSxTFmxe24ZOlqfdDlOUkVO5aNQ4dOcB\\\\\\\/064O7hDD3M3hyB5xrClmp0qayIJUAo0BspTfFtSB70rIZWgQYB6gFPwxurS880YobVdkVU7V3N7wLue1A+\\\\\\\/3HAjQbQ75q+L0bu+FfbNczdsD5OyS2+PTPyurr+\\\u0022,\\\u0022iv\\\u0022:\\\u0022hhszTQWpzse5yjM\\\\\\\/sKSr4Q==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00229febb5b026a5e97a61302e228c446646a2f535643b9f7b8a92043c2bf62bf9b6\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022TqUu2S4u+XD2qonO6WP276bDHqknhm8mkTxERtfNzXjlSZwXC4Xf3gLUZhDpR2F8yOxk6XZs3u19GzPL8QWBiUoZdVkC7lIokr7qmg6hyP5p1smTwp6N7PJg1X1ubpHra+ZMI2KTuH2FwJoHsvbzkHVqmZbhZrBbCQ3oAcNBg+2TyngECEJ2x7HwxWPTrtt4wtX7kkhw67AFZRwwokUhR3Qwhpua602s6ndMaiPIcFafF16lHXpbISwmJWyKqvup1S7Q+v9NmGLR3fWAR5xjyGaO8oAKHPZBVSrmEadXef19CvnOTxHhGExuCy790I2Y2JaZhZ5NhSBh1r8Ngn0WPMpi7s\\\\\\\/AMf2sVFmxu59FGSDLdegzPjuuGgfzwdX7+rPQ+U6JiZCGvI8P67I7Spu93OV6vnnDw9y4oVBKoMT6ASjIYcFIRzxuKYoziQeU9OYj96RF0MI0y360ZZdKpadnoZUdJs\\\\\\\/TEx8PvnnTupWN3Vk01evuWwusRwCEiv7L+4Pj8UO5XOgT7M6q4tvlvfO0\\\\\\\/8Rs\\\\\\\/HlGARARb7myDGDsRqQcgOVZ7OA2yGSPn71h6QG426FfdSzuNQCDHqS7pghkcZUEBobFI\\\\\\\/pn06vXR2OJrCOxkl4ENWjO3KB50UcpVjH0ZkKo86t\\\\\\\/U6G9JUTUV8OSVD3KiXQX5ZlO6VNmF7PY8UQ2MYJ\\\\\\\/ck57qDkfrpw31JKRFBHDLpAYz5ebuVr1\\\\\\\/y1Jg+gOmuLCe7Ey3SbrVgXb+OVa8SqsygXXcKakHskfig6xHtskvWxk2eBemcHv9PkF6qksCQzDul\\\\\\\/jwzhSg+xI0dnTvgAk9oqNg2Kj348J8pcDKkspOoSX55DG+8ocrLzw4knevzZB5RxdR8dG5gZj4ARF00KnLhbVLTxB+5r4zCsNkCZ1OcNgC+SnPGj4R5kbMNgmzX3o\\\\\\\/1kpl02+exW0V2trz0ry0kxzKanQlDne8RPvVcjweAzGrN9rfgRhd67rdx\\\\\\\/CXeycKxGclz\\\\\\\/NN7oefNhQCAf1K0vMdOeyQ+FP7NSHkjLpNF3+RVTBWbc4p8gjM5O3iLcucD2q45RQ5CXklra3QjjphKk+MfO5th9yUJUImyTCJN4IXZuInSZUUfjLtREEWPgMqAlBmQMEKRB1alRCxDHK7Yfzr4zb9a3UeNC9Bns2IQWhYEjRum4MaIbN2U9hMnh3q3UtCLwD7pgUVaU2Ux\\\\\\\/iHcFp8LTuO2Xbej5sl8Oo0H0kgmBlHJQckYGcjhY40fMjwG5RIrQY8GHBJkWCBQgR\\\\\\\/xr5fpKTN2JszRJoq\\\\\\\/Soo+siEM2JNYUAigHGaQtbXv2TV8y5Nu8Q2lFuO3RcKyJdxiB\\\\\\\/mYnwqFWq5Oq96cuUZIUuLZCb43Q9G9w3WrFsl7y4MtdN084cqck3Q1eEt1SkYUhXquDiGkbx1+Y2vAu\\\\\\\/TIjzdK4svNXQLkjQLp2goTmZA8hchn7dgy2A\\\\\\\/vmExL48kom9yclQoGzJ7b2uCvhGB3Sz2R9MP\\\\\\\/3hPTxDMvVpNk3e0CUzLqncl8YJjfEWBNyPWsJPnCcR0g+LZq2\\\\\\\/VUwIzTv0PajJf3T6IVoehWw1LIAN1HnkcjqNUFj58UpxGA8Lxvdeqe8HkQWmoRPz1njFHYy3aZ9CxOB8BM60wkW65LbvFhg2dKTgM0iKnAArObiY7CynbaELefAAlMwJG9de5eA\\\\\\\/1hd2kK2PKU+4ZH4G7iHWYW71kIbOjPOPmLsgVo0egkX53rJZltlrGoC3eRw2iMPWsgdvI428eUPiuZgDsigQQSNFEZACRXR66uLL5BgZTnepGD\\\\\\\/HtBzgED8LZAAKLmdxw2p0dj3vd3chlXQK0WuD5nzXDEYcsNRkKoN7Kzq2aNEl2oWjphQXirq\\\\\\\/oZ8IvOdFIM9lyqNBbK3fl3LtBIL45jMqOjUOO3pQfQMe5NjMeHLQhmKtiehNdApC578EcwV7qT62APeyk5TjCo2KXm1T1HHqH1ZjwTcW\\\\\\\/nMkAYxXeqlpky+PVTRZ6wc7R9Xagna177krsWT79LD8gMopb6dAg6PiHGuE4Ac13kRuN24wK+0hnDM+QCC+ooFHlE5dOIeLgZr1Mba0kD4Qk7y6j\\\\\\\/FyBfAd7sJqXoyfL704+TtmeXsOOT\\\\\\\/X77OD2ocu4i5tO3dmPQyXXF54ekMVdkkp1HGGQSJGiOFKlEBtuhW6banKbJv9uQo7g4JjSDIl0PdEqFVbW2gU8BWj+SVUefC2Ql1SoCl5n+NBN6k++T\\\\\\\/QTXX+\\\\\\\/akMLyB\\\\\\\/9mTNHxpvko7XeT3KypEcFmDYe87SnAb1OyT8Lz2avL\\\\\\\/JZ7IDj3XdayNIqaZ7G5KsJ\\\\\\\/Cqjfv0riwSL9L5pcVDVyiJRXh0A\\\\\\\/uzpJzLRK2PdJIwHei6epKd14L5wPsNvbPa25nYC3dIlOOCRqlv7rrFPvtW7gtib0MEjAtv7Ub25G8cToW7qj1cx4M6Elhj8M1DHUXwnP\\\\\\\/TOkilfX+4zFMnzTRIOC5TQ3g8XLvCdXfcPaQWoeFW7\\\\\\\/58bR216Jtoq8jVFKRogxLUdCixoQjn53rpdWRNb1sSSX8NpXUdynGWnCqrY1FNtxBLWWun36oU5SiYZsr6WUs+xWqJnVTufridfLYpcGFLQHHhK1X3FU0wz0jTDaP5CfPo9ULAKFTSgkFtjRDxTIBSxH3+Y8dURNI4ntkUsq4Gzpot+SQHLOW5ph2LDGA\\\\\\\/UOQTU+uz+f8Epz\\\\\\\/jHqjZ62ETDwzaAYgAlFPA6QpSIpvXVP+KBcHNq+LhlFx2L+bHrvr8DkeHt2DaBh88ulnAFRDPPuoJLOAkTbOi89jdS6XhtkU93yU+8ELWfsuisUOgLpf5r2Ca6nDHfMt72h+XVebBXzi4kkbft6\\\\\\\/RkWzPpo4G6nf0josok3f0zXdYZ63n9i+lROxfM02EURjOk52e3vUXv8k+FjNF\\\\\\\/hfEs3c55OaEc6d+sEJo51E4AtyzgQHvPd9olNIWeyh66nfo0IBIe3p0rRqI2zV5jUvlU94Cd6qkAmYO5c\\\\\\\/ugb3qR7shsoSG9kXM0Ytz3REWx6QU\\\\\\\/Fmh0nXNolovkjtn9aCE46IhJAWnUFp5ln1ynKuANItf7fP1YejRAT4PdNaB55uzXnwWGq5wHlfGt1bAwbarS27VS3\\\\\\\/RM8\\\\\\\/EUJYV8\\\\\\\/o3G6CYgGyQlq7YM8agqzRGHNgmLsKVvF6MG3Cc8M9ak8xmsrOGOlFhXZdeNR\\\\\\\/\\\\\\\/J+zh038TLVyGN1XHIxx+iMivSsO5rag\\\\\\\/k7cKadG3v43FhcVmr6p7E\\\\\\\/EXyX7VWk6vW1SgIuSqz5B3hv22pIqVKrgDK\\\\\\\/mfEBDgXXIFiSS65ENw6LBGU4kzmO1l4PB\\\\\\\/am2JV5z2lJeleDQmeU8DBMS9kNZQ9IR+UUaE9YxQVWRAW6BRQraYVbROm1P9WVOhiIRDREQvYHo4ohoGzJMg3yUxU8D+olJ8nwzDG25RP9rUzGcYSkoMHRiKbPOCicEFLFsZKTDL3cl4b3jOQTLhV\\\\\\\/JCrJypLOXFflS6W15uux4hzl1Hoa0KUHHot6V4l8HB3wU\\\\\\\/huN2eEznwFsUcECq2WKOqXu\\\\\\\/rHW8zqlPlIRzL6agDEeZazHHy8R7nFESXfylXLudvH\\\\\\\/IfxhU7+Tt+IpOFzk5S30GJ8yaKEao\\\\\\\/p8UQp45hfAt1jQaI+z0vlwOJD77rnDuJkR7OKLG+vauqXZP3TgDjmnWNvMutj7kjGdaYYsfYG+ReahVdMnmVhQ3KSd51WdqqqL46Hsz5jFMKdhp2Ha1buntri5793wl+A2qA7+t\\\\\\\/MqxqVEaqPbiBviMVKGK7aWHsH8aawVv2nboNDZaow8vBIqyFotsbv7EfrK7cAJkahCyUq8P2lKa5jJVOYbT7QBqwAPKYjui9Q\\\\\\\/0bp9vYFy2TLr89cyZIKLzkfUw7ooxwF+83cLWsgY8jCFRtE1HiifEN7IvyVfB77S+dvy729yP\\\\\\\/lgPTMuRZ3kFZCICHEjzE4K+l74mmAGYDgfNTFfDcL8NhdwljPnhoXa+deXpBbEeZ0kYI\\\\\\\/UISg5ZCVPTRg+yd8clClQI2kSsrRN14hkPjztut0L+eRDCVUypBXCWYKCR116g0oB4S0ASnWiLhbwqySY0U8RozFQG+M7QnJ0U9PMgrcW71rX50o2HmGZKKSjGhakRzGt1lFGMR9+Lq1Lj4LjEN2uI31PMDpHUpcq6IFwkAOi5ouHFccgqXXLvMjj7YAM+CZcY2ZUbbxTDmPCaac8Vf3r0PSbt0OOgd08bimoEqhmaBY3fTwfEHWwGglhAruMPTYAIuoMovCV\\\\\\\/8ZvpDVzolUrOKyhEaXQYjraLm1TqT2WneMFu0vgt3lYjkdG7LQ9Q5RvJSs+EiQQr968QnB2O49n8duTwPOuiAKQxjlvufvJE1PJal77SMbQshuKxoJ5b0KdnQaop5fsPQwNAaDreZ73tN55Cl6aDcHJt6elUJdcBaBHDOJ6rHqa1BZp10UF6ygPiVz4mCpOLTyBk8SCckv1XLMV8h+Xd9PLIJx8llEJlNTc8o8YZpCutA5vpg6SB+GlsOHZ5dvDtf+f+yaoakDLFBaRRKVcudWfmfIFQ1BU9MyXHlgo08dCmrFbQ20CDldquk1ZjBXEJjZ8x2NhByQgCkKcELvoMWuDN0kgdVXbsTjVF9dUP+kqUKXnffTbBFzsNfLLoBov8hG0lHomzf6IZnoewHO2YwCxhrW8ltBpsLHcG0uYNdg5ZVFFOPIsN\\\\\\\/aAKX2mjow1EcF2hgjTU7ivy1d6CDj3sburUMvRAoONlZKDPPW5EUx7SEglQ0L37zz8Kw+RIOzqlAAcDFEI5m\\\\\\\/KamLhrS\\\\\\\/8\\\\\\\/4bVMi9xTIpU8MQB0FOEv9CTxOGqUB2aJa2MRugqc4vg5zVYb3lmSmVcXY4O6T4btarScgkqFB4o6nttLQriUCFmcRLJtDo\\\\\\\/zA6EtbHlrzm3Et3jRJgCRccD+9Y7Lbit5xpxSfiAAqzZc5GcVd6KlTAO66miwf\\\\\\\/VrCCBuDelWZRhMPQ5KCdG+xNtzWwdvGgf3VgItQt+jFnxX9f7H4hnzU7cRSsWbXzBSE+9IbcsUhZxFzQjf8YOnNWHiDDNk72RmPt90UsOVUZdrozxpDsq\\\\\\\/TbAHYiKryMJ4kiKXH7chnhZmIJzdaz0Esecrr5EisdisIpKd\\\\\\\/1HjO43W6BX\\\\\\\/cnH2FavFixOnej1g2XBwoOPvum238r8shj5SP3gqeJ6atYvIWuin2UEqdMa\\\\\\\/uJpnglGtAPBB8jedDc8uod2GczSNrSAYNZSQerEtw4nUTSuDBpOJjAv5JUSHiOpotYSAzSPWhqJUrwRvu54hiTRh+AvfrT\\\\\\\/rq5FVfB4xi0pLsO4M7peIVpGIL4logS84H08Us1b0lggGMuWQPK70je51rKx9kL8YGmfoaOehV8x\\\\\\\/k\\\\\\\/i1WlWn99Vi2UrlcXXoPUOMzYv36wlXtFU3K0P8QjyO6yHghw3L6zltRRA0B475wC9kONGYIlQxJ6HoL7S+kHvVWjD\\\\\\\/PzgOGyN0xnOHn0WN8N09grD9CqOvp5GGApTU7BKCb6y5xRqdrSxGkeWHK5\\\\\\\/KZP5P6qURtmGzfRXA\\\\\\\/w0kF65lLF7pudGyF43BHFrK9cTNjVU9LNIDmyTZe8hsUS2zfo6auOzYDisW7tppyYYi8lDr54pqVw8HEb78LegGEM+4D24imIVRqWO4aH2ek8Va\\\\\\\/E76uwOwdvqPWOEoX\\\\\\\/xUWgktLmKBgrLdna+iMpNgfTwu77Ak4M+Sv8UUB0G3DDobocH8ehlJlViUAy6xiVW9dyobNEpHNJLwRbEKBKJ\\\\\\\/0sdTH7WtamCjCWdrRUrpsLtaFLSOEUaW6YvT2xikS07ULqISfx9NRcOgTii15pJOwTcC7klDr4JPwVXcZHwajU77v31osIgqSm2PfaaZt2Qu67hAt3i\\\\\\\/6ktcE95v5rJGbnYPYtp6A4AaojHDTyyRgu6sMS6RbiCeL9\\\\\\\/m9o69KwLdC8Pd0ASKDofsKqmakuM3YxTmGdIs0uIaZDDgIJuqviK0JgHN1ycgS5lREfBEu0rhbUNKdrkJL4fojGctajdHbX0TEVXkQpBPrTkH+tZBywAVhF9PmVfNQw+Yawvpq8khY20UZ514QCgDvI5XX+LxPRsI4fwNB1WUbZYZAGirGZLriUGKxIngFQzhWjfxBD3drYMh6lX87LdNx+ZXsL5iewSwcflCy4hDMrImlF9OioRsG\\\\\\\/7kBeKLAuNPrjutIrVuNezy+QP1MxdpJvfPMPDXj\\\\\\\/lfqY9ahAfYMeyjn6imyi7cBZIA5xxbSpV4D94ODx1zAp+qlf4+5GX8dbM1Y\\\\\\\/Urlfuo5NQW3DUB979kpSbsgy4dg2A7IxVAsfrJL3c87GphFHLywdxFSPY0wPVd3Qxk9lglRyeVWpUTIwciteP1uaYHe2FHvtRjEd3z\\\\\\\/XZ3Ze6YROAUHYKYKvxymvMf393z9eU4fuy9W1pLAeTM9E5r1bVe2adPJ2+MZ8bmkdnWP\\\\\\\/bJsbL29pkP4XwsRqQKLkNSgr2W4\\\\\\\/vJx3DVXrRMo6s\\\\\\\/fgnsmakaoQBNNKtGpV\\\\\\\/GiEWAh\\\\\\\/u2f7lKIqmcjnA4cDtEeu7+j15z2ZWrmu13bonTY1YaOVmGLKFMq62Go3HDJZQ0OevHcGrLsLbkGsJeOALswhVjUwJ4MShUwXmInYVuMMsXa4JqklSTUo4owutZXlPw0G41rrJRgpStRLHPc1yjg7Hla6rPUC87h7PbcxECRnNEkUDYkMccS0AWX7\\\\\\\/Bbw18KPHvs\\\\\\\/fV37S4vI7OS8f0wDSsCvCV3Z7ebB4Fu5MQ1Q75cOzi5Bs29e2RRnDHeH9nDLN9IR3jmlB35d\\\\\\\/XddXvsy5K0U5x7Rnyhw7BBvnUorcX0j2lu\\\\\\\/UwbzaR1gjXoI+4Hp8BJ07BanfoPOUE38Fk9GRRUsQ+2IQMJKnKLRCc8qMi4NbceXc4Mwk4LQkTiRv6yIxMjhKbgz7KS83rPj7QuXqaB8zdk9kIfxcad8YCSlZbBvUVfhhBP34z\\\\\\\/LlqkE4BBP00pMQWhpT4gay6wui1gpGF0c6PYYaQz2VScT+uSwaABo\\\\\\\/biqPR8yNEQdZFtcz5TzK0L1QVHHZpXF4ahs7qHPfNOSl3FCanqF29t4f4pB2SQrc7C6RMszOUZCK6b72\\\\\\\/\\\\\\\/NcENyVN2dr8aNZLu7lbNnkZRb4HNLGlIUssazseDw542lyCpPDHIkr\\\\\\\/C73exdSehg6lQE4Wgw\\\\\\\/XSS7HrLVzFA\\\\\\\/+qbKlVkZ6KSZJHaJ8YnM1z+FEf5YeCzSNS4VDGi1PtpyAy8oi6sKRf5eC57Lz6\\\\\\\/4fPRVD\\\\\\\/kxZjHBT+vDZMPOqTv030+OIrgVJ3qjhGsWa0lGxHd6GUcJWMtXpElBcLKn7n3EHoQtu9DAvNJKMd2IVBBUJoMjK7NV+RbEP1SUiJubASU6Z6P8FcuekR5aaSCDSHKRfFjJStcLTMN\\\\\\\/xtblZC\\\\\\\/i08J94\\\\\\\/XYzXwzVrKeg7rVS1XGHEsWrODyll8tBpAhWDQPBsRmklICsTCjK16BwUNvGsZvABRJYBIdKN4u0lZhjIMndC87GCdBPQYC7tG4d1z0dJn8wxzXrdcQMR1imgdSBeN9XF62GypOqHEsXm0M6DW\\\\\\\/H5c6SXxMFj4+Xo4pvnXlGHA+gaeLPaIcrlwXdUA6zeQOEpCnzEGZr0yZs2biQkg8c\\\\\\\/C\\\\\\\/dubDmqTVDWidFM9Qg56ndXfSAb9628\\\\\\\/o5eXSaFvDUlfQHC2sm4kcidkGn\\\\\\\/e3oLSYHyBtaPfcqcW0L4OfkaPRsfKO+0DJMSOyJxXYq8bJty6ZEptR5MK97v7HQ+qeAqR51YxHGD\\\\\\\/0EMgBlx3SVKuBmQYksL+4QuVEDNkI190V2Um3T0NDXAn6PXAfW4Sk0+8mWENIxlN5ZNQTV2cPyce9nbaKtk5ajla4k4nntj2AwU6jQqG9q72+yfwlIYF2zWWIp3bevhu3vfJMi7Pqo358htuXnGKN7h7753F3H+7wwFMMALF2PQNRyOm+7BEXeUETvjs2zSLrqDQr9u70s5+EQmtBst7luZOtUrNUU8ujBi\\\\\\\/FUN+PjDNAfmzHe6\\\\\\\/htaY7fGetM2VzVVg2+EWpmyvDDyJ\\\\\\\/RI1VyZ\\\\\\\/BElp6hrNv74fapn7a7k5sswajCpJriwzDGS1gYAtP6I4ITi8r73kFTQsz55MCpC7Hh3xQlEH48zj00DA4gXyN59NDRXQR49afGiLPdgt31MJW8bk93T+5bZnfS1zrEnkbpaE8DF2tZwD8gRkUSzrvymx9ySG1S3aEvQqzl8ZjE5eIW2VvVRdwWh5WBoep48JR78oEyRxd8uAoCCbmDcMmNeJ\\\\\\\/ECQb7QBBGekhfoWeJkBeo4wv\\\\\\\/3RLY238ODSTYF7d2rayER40Lx09oRHn+7AsVyyz25JAy6CLQWesYfTr0ZqUGpdauDK1kJKitD87OETogFHllFTyHU2hnGBSwjl4mfQuhEe0b8QYsxsL7HHDwduEfjM5UIXORYccIgwtk9YQ+cOfsNMxPKWk8hQEdjzx1MAEBvjMLGHit9U0NCRd\\\\\\\/JiVK+p+BGnRFbwEHAgJ7focKHaEV+FlMCKUh3+hLeE\\\\\\\/YMWnHMGlOqPvDfCACiXSxJdORjBwkl6b4I111PX09JgxmULWXmvsEbakwaS08Y9o4kV28fWb7ZM523wDHYkppd26A2HNW50DaOW35GCZZjoIZKTdNx8Qv0tg+rO5OK2qZHy2Uekaq29QCU4mCKc2O2bgKQVCz+uEZpYcqiU4\\\\\\\/lXpDZ6X+x39Mr1x\\\\\\\/89kdphMG\\\\\\\/Y6bLjUqLWOj8ivMCuP5g7ica2ZAFOhHhpbd+jy4MfNKEJ9vKFvp78YRiy88Sik++Qnvm655nngT1KXXLdpUjLB3+i7qt+iNjOASsIkGHtjepDX+SNYwwbVJIb3p18unltEo786Z36xZzPPQIl89mMDUDezXmmxII0Pnua\\\\\\\/5TSgCGNyV\\\\\\\/W3ilQPqzkI9WhN9X7WEbnam1ef1gUgUDD9jkgb7+561KJfI4x9WHPy3ZHHCvD4NADKxYt\\\\\\\/clz2OkrLEoCYDXYf8dnKgwW0\\\\\\\/V1oRCRBRAWTC\\\\\\\/J7bYak+N5i0x1NeH8XXjiUuSz4lkRQCEQKmHR9jYUqq9745Xvl4a5TSrYBe1VeDB5tfLp516FCXbd+f2oYsUzpt2XyQgwdhNNfNfxKO2K8ecMoC1iDSINodvs6Ec\\\\\\\/HsvjI69mFoCC\\\\\\\/+0MDvdX+GzkTCHReCyo+JPncii7U4ZTaKzdavklIUq\\\\\\\/uyhXJKUkMMyNWUDyfZaOWn3NeO6TlpKIPP9HZ\\\\\\\/f67ONi7D5P9RokIR5vm3wydz4vNkI9z2LyYd67DafEPbvwK00EHjPgT7yBieriJ1MBA0qP\\\\\\\/Sv7H2juMVUNIKKyDoV9s7x+qWTCz33rG7i3+w48nK6xbS9pj0239mmOn173lh2mHwMjC+3X6T3fLoUsBMX+iZAgy1J1EOJTEDkKuD3fdfgWFVXgvPFehJJlKg0RmE7UkJWzqVOJOBAYPYmpexyThRtDX6SSSwMZlW8OGdOINoU5iSYbS0jcj7yW9U9HZEy3EZBtAilHAPS8HAUB1+EmpdG+wDrV\\\\\\\/B\\\\\\\/4KYRjiHMA5H+aUUOXE5DUGTronDZESyTZJnHUVU7xKEv\\\\\\\/YF9UEAFeDe6pMx10GGAt6l2o5BIwmlggpuJ5yRqueYgQ8LgQcbk56nBIFcZultWgccDdRIoV4AvLBA1nH632HjL0fUX0MsPPXLXd+az4QeT4AQxfUSecMNaeOK9M05pfAOxhzQXDylxXJMwx8MIlju0AQJyU9n5NTFztL8PYfeHbBVoHpNO5jxxuQxTi3Ww4G30EtlGcYmwMIC58MDjzRfveFxDVsEtj+Wh+4MjPa\\\\\\\/Ve4RJB\\\\\\\/g+YW6xSE\\\\\\\/dMubPbsgKX69U1PRI3nh7Jfe6nZKo2ykJUbUJ+VkyyUGctGyEyuxwrT5VQ4WdAU0xK6t2Qu8EnTgCJbFcnochuMXhUyO\\\\\\\/sW3s\\\\\\\/ac3oe+F3rWHL3REBoMmet0ZsTMsZL3phPrtFUnR2G0fDSDwIlLX9hNwHRIfyRHuIYFV3pfw3BvpF\\\\\\\/vPBC2AG4TDKnBlvEf8CmgmTXavA1JkLMpAWTr1n7crjrzpeuXPWvXJWAJZJUsnqT5fGwcrcO2mseKSYbK2Q6OQXaE4kv60bpHMSBQp\\\\\\\/ELplmLPFt5qQGoGwXYk\\\\\\\/8tFMKU0fhgARuOTt2AB97NUu1GMt9CfpyT7UJUBSMto48ohY8mw0u9Pvt59KeD98F3YzlRYOr2zahIT\\\\\\\/7vTvfo+hmYqZ+67iBl1YrBx4fY5\\\\\\\/cwdf9bKX21C7+B93dJ8P\\\\\\\/SQJ0kWcWsJl35oX3wBKieJkqyJ+HZOS5LnYM2m9tAb9dFGrJi4\\\\\\\/Is9RfygZ0YK8iwpNvN9tkGTbpVKOoRC+xKx09yz59o4HelOcxQbqMdu8Dy9xw5RP9H2+B7EX0NFA49znETbeal3LrfT14bL7+dlepEF00wKYPybMtLkyq90aETpsEiEQGCFQBLkAefUcL97JdVR8HFWO3oU1xr8uwiEesXMrSdC2LodkzrcUWONiOml4XD1Dd0f1i6i0qxx6QHGvt8Lne21bXoFCF8RrUaSFkOuxNzuQqwdqu1S\\\\\\\/oA0Epm\\\\\\\/bnVwfK6+3bIyn1NmlLagjXj\\\\\\\/xHQlt3uYYaKy2B9j70YCh32p6raU0yP7fj\\\\\\\/BoEDxEEP83XHagVKYVpAgMJIDOM2lS\\\\\\\/5QntDrzVBGHyKjfjCIvwUL\\\\\\\/ebj5cfGSvoXrGzAvVQpcRO4aarPTFUEXfDQ\\\\\\\/qFKbYWsWufBEKNB6gf91hIrrSvxSaCjm+sg5jmHCKAmEdQRaQ1GX63jvVJJep9woUsJsyygeQMr0nbkcxQazxFcIslEie4a6+hPT+CNhfGD2uuKeoy0U+JPWknNpHAstukGbjy5sF\\\\\\\/QNHHGnAV25VF75e+n19nz9mtHGEbV4MUugP1D1SBYuIPW+vJ+Wi5yGVD67Q5VdmpnvTu7v477RL3RJlq0D3Z1\\\\\\\/LN3Cfxq4r5f+9rCVOCnUEWaSHbKLlqIva8elbRA\\\\\\\/BEYtbIky49Fk3FxsElt\\\\\\\/nSwhYqWzsY2uFihzb\\\\\\\/4IougtPgMN+JIPimPJu83JXTgEfgzzEndI5EsH3v+7\\\\\\\/iavl+6DaTS2mpMxoXzOiVKLrHUFyPu\\\\\\\/GiBSt2zRg3dzBNI8HKYc\\\\\\\/XYiilAL9XAsI9iUWSok9p\\\\\\\/dboXvtbEYmf8ZniDGnR\\\\\\\/\\\\\\\/ZKDXk5oXXIDWE2TEXUu7HCA37ptmJOj3fP04ccRDKXZ7Z6wdZRDl84vnG0kqW\\\\\\\/g\\\\\\\/mAw64GbIPcpx344GW9VmrUNSStDIYg88OIdz5jgtTE8y7nBAgVmH7VJWgNoOhgU\\\\\\\/Z9f3kteR7IM0k0wF59+nvODRwmiqrfEI0utRI33AgGgGENjjcUgPqkvxKlB8FYeECm\\\\\\\/kPiL39DLX71hJ+PRe8fw5G5+F1huymHMWde0mZVODkYm78B9mcrm26pwDRljjgLzSc2gLwh73RqEy868Zuf9q0ubu33EX7PXxpsjNvwSD6v6Hmo051demg0+ACyV1AxnKOMxOW7TrQf8AVz\\\\\\\/x2m1Oknzfzss+vF8xG2zsQ6bVt\\\\\\\/NK1xUnlCc\\\\\\\/5n4qUPKMiPyFq5267cLM\\\\\\\/BGLXdUqtUjH8x+jmzI\\\\\\\/CL0Dtj65cM8UiN2Zd95Q6dy4vvnSBJL4dkMpAz0iQ7bifONRI261taB+GRd3BgAMXmLirMjtC\\\\\\\/Tn\\\\\\\/Or3yDLb8nssztOjJRbS10XBGZyBIVVbk\\\\\\\/gmdG7Y\\\\\\\/iQEarslYsuqLR52ZE1jCHDtfkAbrmNnycxfsPic+zJrjzHQ\\\\\\\/tvn8OE8xWMhJq8QxZrAeA6UpiAncLrsmYs1+MneR9lY3KFivnEFK1pyy57f+V5mK6n\\\\\\\/6BA3jPSAVGgGzThcupsIwM7lzdAAaVSfi6m\\\\\\\/a47fdmaNqk37ZPjQuRT\\\\\\\/nhaYFhDg7s8cPb50YkEPgL5pxyNcbn9quaUfUovzBH6pT\\\\\\\/Nniz4sYkq+J+rvE89mpMtMX4sLH5tiy\\\\\\\/2JRxrPnAVmC6I5xGt9vQzOi7Dhh9hwSziOiNfnQ2\\\\\\\/Xng49faqeP8cqxD1F9lozMsONDBHSvA3XuCd5rHz+c6rT8crinnU7tmhU3xD7PT4od5yGGXwN0sblY2SMB8XAy5l3xh9qCgggK2stu9ZachSedKUPfyF6mfnL+TOkdzEo6QKV5iN9qgVeHTTKCx1ZsPzQSeDS9LxNKVlBjgKGX6V0R1h+PRfMyg\\\\\\\/txzi3di8KSfUgpRMgLk0oS5kF0H1qvXM5pW\\\\\\\/clS7+e1AQP1RVj8clhh5Oj1m7PcdjlRZRX4NTubY1Is+oqRIC+6b7dk1\\\\\\\/5jS0kKBXijvbaBFO2gQVufjSS\\\\\\\/2\\\\\\\/gcZ4k585h2MfKn93zUsZ8L0nnPh4xx8cgOEAN\\\\\\\/zi6Pwz\\\\\\\/kUIzsGQR\\\\\\\/5xX+I2W+seRptoiWZzN+9t\\\\\\\/IhRxRCUgeZZI8bFb1LQRQkp+Z2zdoro9NqGg30wwt9M4dIfmSquBCFRoVZxd1Cx1vZinqx2SsRFgBaug1rDyR2lg\\\\\\\/f74IXQtov3rkKEEUyqqBVeHDyuH1etdzFj98LW1QruAoHLB+DVRZ\\\\\\\/c5IjPSR9\\\\\\\/1NKU1\\\\\\\/idiLzNQnoycWN5zzqUBc2S1PZGu+9Y8m9o\\\\\\\/8Ag5hDg6M4xHJlr+DFyPKzKAFXLswvsuAA4QDZO9PnOgBV169bauauC7ZsV8NdoIeo9V+j4vyGziYiQ1Tm0bELKyOog3hfSJuvYXczI+z5yqR2\\\\\\\/M86i+4tc+1sneQvAkuTOqflf8Ai5CSda4aXTBARRNtbM7i9EQR5lih63we3RTR5ZrRRSom5ihgaVe0rnRcZEKylUlHToUBM0BONs\\\\\\\/b9sLtKSUzHSQ08bBvNZoodMAQdQTh3Nz+rFkOlOPjBbvnBrCrvEEP3ilcKuKsqFBAmAa5DQsZubYZWcj1aaRdxaGSs36\\\\\\\/2yXSryHnkDmJ+ujH4E+9dJur2G+nSFrNTuuO8Bc7CubizqOlez40bnNH37GcRBMNO7E\\\\\\\/wecFVv+uRi6dzbjOcaCOKseXFifVlCYvswz+QcgberKQjxBIxht0uXYMVqX2yT0r4QbWd\\\\\\\/5QB0xuA+xQ5SDomURgymb5uaB8FQ4C3GDs2MbjKeFWUL+l4cY\\\\\\\/x5p6pGR6+q1j23Kxm9n1bcC+xzPo7lNBaPM96V+bd3L00WDVeALzrUzuoZL\\\\\\\/NDRQP9olE4kpMDYaxPyiaC54diPK6j4R2SHKeLyp08jX\\\\\\\/8WIpfCb+4Mi8U2vqMmcoT+XC5UdR4kum4+GRkGFAPYOAKXoLMqAHHkVc4DYeWQCO\\\\\\\/FZ0xOPkED3lrzyBGLDLCvfRLcyzZN8e813VJz5Ne4c8IVhLFqWOQenrfd4G00+pi9O5jJW5Teh9n+f+f9rFwXMVrgahxrJh3ZdrzJK7q7PZiPiteUiS1Yf\\\\\\\/mFYGoAwmZjobm0XnyboPtoi8jEtwP91r0PA80KVjj39Mj0nzen\\\\\\\/fGSaYfymLFGGnYfrLT7zY6KEV7Q87IvbimkkUSDy27v9j3HIum9hKYq3y02IXJ+L+SSJGbHObWShyjbNZEZNoyhZVgi6kxgCmVmeXyLHCPWhzgL8hFQqSKy0rrX8NzVFUJ5SRQrfoOvdCm7VzbCkSIQWG513oq3wP2iaLUQNXuJiXf53uJrPhPKwbgBWLQekUoUbPMjuS82khkiwjobwbG8HXPbQNrWYqpv8hIxQEkj5XzRcGqvQDNk3Q2K9I4KDs\\\\\\\/R6RQTAjB3vFy+B0xgtTiXB71qqNhq9ehJmene2l5xQ18RtTWlSfK74P5uKVkWG4xibv+faYDjTPhukXKuy718BH9l27vksOlHBG2cJ31HwtXtOLJqi\\\\\\\/MAk3Jpdaq3+9oZuOJgLqTNB3LMbMK8RdWqDZpcdSPCtrv6nOBJ7+r9MpsbJDKQCauL4MhzMyKpgJ9iCMz4vXA0y6qrIblNp5Agat2HjkZkOJM2qHhhFXgQKWeCpQyqz6MxqUTuTbNzLYY69x1p5y7NaPSETajbkIE6kEQMtSXZW5TZYoUiVoeSPxQu0Q2kARnHSQUbJhOaYsYFjyKAMcTTcM2x092uDr4jwGp1MGnxp4wLHIq44N19663GN0K1NBuY+3mRmzf6exmXwYbhTvJM0gdt0BaZUnWf+\\\\\\\/1Kkaxe\\\\\\\/ZnUh0dl5j5uP+BrbxaIvYpsgWp+x0QPYYWSVGj6Kuo7LEGbmaOvoFqEKYG5oSQN+qKKtDaemOyTnc9uW93pbv6QZJKMY1zMn6BSDzg4zw8yfZFX+LNRxcr1snKgJ\\\\\\\/ekfsCnu57nzDh1dPOuG68VfM+cUstbKH+hxcykeX1\\\\\\\/jXD0ilepOgX+F8OcSmHyTFFL78EFt\\\\\\\/5Ry+y9X7CnXWTTn4QuCSvbQ91AiOpwsFiqr6KPkNsGpFFFTLGZqcteH3l6q8oQT1cfS6lNfGRFfi\\\\\\\/6FMBVnz7DXnDIMq4F4JVRNFrSboEWnEd9ECL8BLqdA+0B9r2wti7qdcZkOeg5tlKZK\\\\\\\/xYf5+YLb5MYrg5Nzett9mPy8L4l1zCintGw3slMpCmTihoQfusUFZQgjWj\\\\\\\/R257Go20fN5GjKyLzCu3V+5PC1EqfAE5EhFEnhVuMooc0IyEyTkZmAZhseMTP+MM9R\\\\\\\/Kk0ikv8eix1kCRSO4BkfqZDbPG5urcUh3rd3pP3b5hJQiPGWME9rTgBbRA2f\\\\\\\/qyp\\\\\\\/hjuVzUtu7ObgwIzEvuVcn+r3uxNlcj\\\\\\\/6kYIgJ8hCd9ET3SY76P0VIKXZDQniSzsPRaSj+gaGd1T\\\\\\\/IQsQ8HelC6lQWNBfJU6v3jAqZnP5482bMoHEwWhHkU5h4KLEWqTASHEExio\\\\\\\/YMD\\\\\\\/btzbK3\\\\\\\/ju0NdyTIacTub5psOg3bRIjL5Cj1P9aOYoKyheZv1QVxFSqRMUb3BA8diVVZrEcQ8SyCp06DrGFqSwbsvmqx82\\\\\\\/2pOChsA2sQ2Y0XrV+k0G3LymGfkIuG0kdsTIrpaCzN9ziOefBEZisNethycvyRUDQlxuEYnKhJvu3Yh0RQVI+nXT3IwihvhU23jjcWI\\\\\\\/X3jflLuCCFHIkB2br9VGwzAxBgqdXsw55u4AIbnRreqNuLdvCN6r42\\\\\\\/r4kMiz3gPmQ6SvkguM2oHhOnJX4pzDy91wsBhNGs2aOM4gyxfovA1gL\\\\\\\/LdYR8XZoBZBOfm1cXXLrNqV0BqOiUQ4T0VoAsasyvfB34OR21yZv4QEpRrbf8yQDpiYHo9voUxv8jxmH4AtVWAFE9JcPSfCFcuambwXzPMAjGuAmjzMpKFhi2pBednmmjL0oLXyO5O6+MH828MEuhSjQvJ+UpwfUyaXk4CwcRK86s2+2Q\\\\\\\/RQPdWGdpX5fvLBreoi+FhSqOeG4nmL8V3aeopYHGNOcELEDowm3o+Jp\\\\\\\/Gw5yxhm4YtTT1B2cA0EB26W+ooNWyuxNCsOWK8kmpBPCw1bHvnPu2Tf3cHiqQlgMPzY6FHTyThlcUpVNP036nMT3ig9zlug5AnxxT5vMY6Xgu2g7YjWzDDsbDkg7YkrCy1ZwB6aZsNgxpPet6DKA4bMUx\\\\\\\/nA83bbKr7aWZZtn1TEx19K0\\\\\\\/\\\\\\\/JHX\\\\\\\/EOKcOeKelAUaDUtfd6GxYm7dEYnnYAzkQ2XxuPOKWlkURvP8qQdk2x\\\\\\\/OceGyvS186Us3uoPc2m5Jt9NtJdoTQ8m8bAevS\\\\\\\/jnACAkCvpJtI4QGUIqp\\\\\\\/QC7zHLa388J18ZKwd7vuUK2Ac0Vt03qcWi7EVMrKevnYHjjugmP0Sn0bEHWR1qPfloG9FfQQLP0DdPJYnrn1dd0uA\\\\\\\/TkA4Tq1uSryCrD3FAX6ogsdSMPK7KuY3ZWGL3YdbMoiEYY+bmknIexQQe3ZN\\\\\\\/Ue1VAHumBdmy63T7Mi\\\\\\\/nS7HSwoiAltbbSSVKuZoTMLQr32ZhYpoquCm4xI4lqxEi38+96OgLMddiwa3AwRyI09+Qij4XccyBSY9sCqg74JrdqtPodPnT1bhjKvu5hvi1\\\\\\\/nuiCvVw8S4GrHsJpiY95HteKSm89lhlBuTWqchRc6ygQhflmHTj+xgxmhSri3jOXsaKS1i3Ye48pcwtsBFFYbel1G0kT69XH3TVEnV1V7UcIp74ZFPYnz22g551bX0xVaqFljKIH2ZJvwV1kfucyQctskKD5k1+3G8ym1HQ+9CatLbjOgnf0Pb51a6kgFiJXGsVjqsQpMe5p\\\\\\\/Z0VpjxTxFCIuzlZEUVFPeduzDtiCKaeTZAN9klzvH7tGRIdMNQXrZpUw3y9k\\\\\\\/XCtBToVEaaLvkPeW+D0O\\\\\\\/tKVdwdA9d1kP65AxGbp0b1RDTmDPnuKHR4QP8AISDAoQuAc0VwP2wU+SZsQ1VGZ7RJ42sMbi7OvZr55V6qCO\\\\\\\/iMf4\\\\\\\/MzI8plrm3Y3Ii\\\\\\\/1JIHwjqcHBuCSxG6w90vuan2RuTNAcfIc3ixtfI9L\\\\\\\/N1GoHPbVtk+SKeePDzbJxdC9QdH\\\\\\\/yKtg\\\\\\\/lYylOvStqAvsHs7m1Fq8hpekegrH7cRHSUmj\\\\\\\/W0LszcpvjmBFtaXGaDzKEgAQUrORxdGCLTHsHwv1qnXh1Fz\\\\\\\/Ediy+3yUThHtbEc2La2+DMewIDHRbPamO\\\\\\\/ThQyLd5kngfDpeoOQ0nYxKyDqpPOMhAqFMqBOBELpFYRrh\\\\\\\/0uqnzEiFs1BlHDoIWQiBug2bR3XTTGX2RxV9s3nkcvY9\\\\\\\/NpDoKP\\\\\\\/hMHqJU7xJQTS3Avm8ys3y+i4aZTokp6wbVfZw\\\\\\\/VpZAOuC\\\\\\\/l0ouA5B5vIX34TK87B0+yTzO\\\\\\\/RSdhO1ko3Yg1TevdqVbrPd9dMlTzriYC0cJ1V\\\\\\\/x+bM+tyQUyHJz78Fu7vxHC5uKntZIQFMLy3csYUgGEG8HKuU0ysnGqx7OryDIyf00aFm7ToHgs7Th1YdsSPSddzXrhOk1B\\\\\\\/Os7OPM9BCvANmC4MJEwIulzYPEAa9CDrJUxld+D1WIAGOGpChv8hMJi8iDaP5WcmF2T3\\\\\\\/cV8b6gsSe9Vis\\\\\\\/XBw4DqAvgcs2DELfBbJErVviXAICysiyTqQfcFkfDVSCOX15d8oiQ8p6JwRcorWhx8Yz+ujF3kUmbuGx17he7Be355tBAwHkoDlIEIWrGeXi\\\\\\\/Uzsd7ao+izMMWoTjZ+551lGAPGaY1zEvJpbRk2FBxl1hskDJg6Xj7fxIz7Y475nBnSkHmVChnP3fujR0Bv77w69WRypLi8yZKOOYh0KTsM\\\\\\\/UL9EBIv4OYhZB\\\\\\\/Djxl08nhHbQU4ZB7HSp8YoUofa63KXdSpFcPGbhJXRtnQLVFG+Z9HkiFY3UHg7yY25VfXN\\\\\\\/\\\\\\\/fbyG+pBJNBXvPLCd\\\\\\\/IKxq3LEAGvPup46CJYunhT0q0unV7R0Yl+5gtt0p7lrU6bUeR7t\\\\\\\/ou2ZloSTp8wXqahVFIrHwsif3fNBW1kLZi+mov2isajGwUu70tX9LZ1Q\\\\\\\/0VReJWhhtrPUjfnPjXXmDo4bzBrSJeHNyu7Oc25Vx9JT2yuhIRNHwHApbgV8wZ62VHYC5Dd2YbEBGRxxxahHh0EUmRTqLE2G2MYFXv4U90FLuQHMnKad8xQz+eMbeLcva3I96ejXCWKTQazN5NORHfxEg6b1L8v5bglUhAuljZKZSew68NMyUMD\\\\\\\/LxywKFia87xb\\\\\\\/bhWKeIXwiEJqgjtbCQdtc7olXvmRII\\\\\\\/bI23er+t63MerA4MNKNqTrJApw14WmI\\\\\\\/YnhELW5Xwr+X8bbEzyT0UZP3dwHjkBw1sTYtQgcSXFflI1QB8bSL6TkpMGQTOd8yCT6qq4jK8KCfq2srkFXWbTiNfBs+k4W3tXAAG33YJWNV5ASXBcuL5HckO4a2iIpmK0lnAKVtE8tGFxOGgt4sQvJiMg3koRQfXj2expcje0\\\\\\\/BEFW+Eni86MiGPef0piuA2HoWOHyPWlEoDAny3THF+Yn0k0HMjud8pYAh2\\\\\\\/K3w\\\\\\\/jKoAccK9owpKIRjdLYXuaxmujMghy2mQDprzgucWzPV1XDZZIg9ByOGi183G5kxOlqllY4Nljls1bhITgmck65bSl8mIEps9MQcmL1Yx2WXk3jrGfNjAxd4rUBvOj52wft3zUZE2RHV4OF7bGcuk7AaMZdRktZEfXxMEsUwRc638eXTGQUUlIPz4QftmcZIw3kiZkqZIpPY33mdolJUolYrpZ8+mTwXcub+4LDh6MWrXRqycRjfbMFdZb8DUQFo75iju+9z65xt+Dq3I+t+90h0TPg1kefdQH8SSvI8KFPBN+AHe7DF4tnrxI7dTIHn+9COAnqxDb8MOlrCHAC1z9AiMES\\\\\\\/LpTlP8Ckdvy8tHmFHykGWsr4k3MKsHecQkd5HkWCyMkev1ZXuz\\\\\\\/875AXPtScAzGcljEgueh5n9m1xakvXVBAWXFUMJ51XLDpLeNkXXbiOGQGjpt9jaFWvfVoyr+JAMVncB6Vv\\\\\\\/\\\\\\\/bkmZiqKw\\\\\\\/6zERU96X2wjrgnfehXmbaD8ORXmAlNaWq8egMDZ723nuaON5FhkPgWCmzIq0qCmjxVQIrpIhq8W9hMEvQI9x5Vjn2ka5QZDOMGhR0lSCjuvVpkT4lGOI0DXuFBwRP6KghrNDlRd7YTbav+zd52lZKb1sEPSDbTWRqfLa+FHEv43k6rmzFk3lTMENjOiUnIO7op8z+hsxtl333N9VIQ\\\\\\\/S+Nz4qFJx70OhRQa2iaM7QzboTovcZrp804M6MAuuKAopGW\\\\\\\/C3NBDGFUrs1hALnB9gQSHyWs9Ca20DTW8aJ514bSQPs6rmpMqRxdXlD1QQw+0PEy+5lZgx3Lg7L679ARCHP4t12LJLoEfBHTwKhlwZtMJt\\\\\\\/GSosF7LV3q+9qI9pRHno891VWf5IftOe0P4XbGi6wZ0d8Ma\\\\\\\/H79\\\\\\\/ZvFLpZFNdFEUMvdIb0fZ2XFtJHfH7vRSdODF5rPok\\\\\\\/Y3sn79+Xeruv3a5p8ejEOrMYTzvXmcT4ZEiDWvJPZf3YKGZ\\\\\\\/xbSrrafq\\\\\\\/44k+BleBppMxczmyrSrxoiK9GcDfoIqflAt7DSmOdbOEiTqPbtDNwy6w3gVxYgSP+bA\\\\\\\/RQ9Mk3o2kl1jcBJM5A42eXRU47ovePEUmssceid7AcC9lZkOkGj3ou3uC7qU8v5FoJXRmfi2YIXMEjvS0nPBrPcWmHL17YNeYRX8C5Iy64kxCEAbasn8kDCbcH29VZBB2pMG9qqxHoLulGfmrU0kN++KtOd74IzjfKB0fuEvgXLWbJr96xfl6Yqr1AZqme\\\\\\\/hpOm92\\\\\\\/bg72rsMB0+1g33uwdOwsE1rYRPTVoSmCauIa4\\\\\\\/G44HyNkiKq6067qwdFTnbdaF+d3VmHyldo3Of6KtnLngBUKNRxNkCx8mis1DhprmDbzPJ3hF9VsyqNmxJUt26yO3P1smI3TXHaCrerDXO2lrZHJDlakbQ2WPaNzV7EIHRBOah6VyHxsxdiIFVkHcgW7R83kVgukMLRJT+ljZnGzl7NMYBsz\\\\\\\/Z5CYnyIpEX04bVhz7YS868YGfmj5oSGPvlL0yOVpNRSpKfGwWHx37DSMHCaH\\\\\\\/U4kFeOpa1ehQGuv6SJa0Q08ZZ4heiam+fObBfAHXKXtnkxlB4zPJoHYwc4WkIL9MNhX0vSfSW4m73t7vhWa81zVoJQtDXUeOw\\\\\\\/17IwbTpMsCs5o7+\\\\\\\/\\\\\\\/m+4\\\\\\\/I\\\\\\\/FrmM1BTO\\\\\\\/hjD5W436MoLwstyLCIsJwM2Y69\\\\\\\/7N3ABypuAQwdppna8af0edah721LN5fVONJxCFCq1J\\\\\\\/Hd0bohRvqAO\\\\\\\/3cGDo7qEE5GdLN11eg0Z9QzMu7xay2xN6ho3HBE2ttRD9oF0UsiQK2dbQI+HIhH+IT4I9YL\\\\\\\/O4zCFfduJfloTrFNFp3hhRew8sAUEwbPWy1PPzVtCBsass5TXrizDISI1IKO1vuTOztAZLS6wig0\\\\\\\/w1909KCnLLkMdYzh4cil04UlNFF+WBdpf78Yz9Mt35X+WsrEIlcMHsnlx+zTimdb0fS4+CqUt0q8gfEN1SwLfO6al21F8AkE14bZQNFACgfdAuQi4CgNlbPyRz4yqLpPlkV2BPiNXrr9HmvNBb1jgtNOq\\\\\\\/BPtJxUFW9hAXp4Jmp8AomsZSG4mVTxHAAnDPd5YjojYOkQbj7sugHFvsSCyFVnf2tul4j0TW71Kd8VvcCn1fYuGIP2N6C2F+CEMqkEdaVQB0Mv1jCTYUTRyZz1iWf4JMD3eW08CyeQ8XJV2O5VaknAHkhDTGQV\\\\\\\/C5JNn6IXxZcSW+ENCttqfXX0D52dXZL9Zdq6MC09DHPM00Q7xM57e\\\\\\\/bvGnC\\\\\\\/ynjZquhzttQQQrDH3yT84l16FYsOa+X6kWsIESS+qQGvUlaEXOPTx2mV2A2Fs67VKqo7yW5UnViKlc91RV0JG+b1Ia9OyjO+mIZsovhqRq7dLqVhWT4muddztOiuYtQDnTAZwr6MgK1hywAxow3pdpFn57QC\\\\\\\/cVXo04CO3BRe6QHPSP6b2uyQ5n6d3Lhlmf6nNsxcmGbfnKLRTaNduCDbxGS5gVTrASIqntjcvatbTp5uGjQKkM+JUoVBojvwe0+DVlFPRo50zk0oZiwbFpzup2ktWVfB1n1qZaJXdiYOXeVFvHKvidssVjKypxUZHx6I84o0nB+ZkBVdJfAm0UKHFMjsmTq8HgOpP25UHYd14yDBJxfdN5uW3fXiAZ5H2zkGdpII4eorfMu5vDJk4Upv3pokjrP06X2UT4raNAGgHbjudfq5gDUsqEOEvKCm\\\\\\\/uQ97o7YuPFonieCp77D7iGg5TnBbzCEh4+3uJdlS9hrCgSuEpmY1upS4JgaN3x7GfbxYNYmtAbMBbLbgD\\\\\\\/oMyyc6R5Mt4rfKR7dQPf0PBE6w5uaHm0\\\\\\\/bIGsCa6yr5xOJQW5TOsr8FnNGxcMr5mR0DJsJfYawcfBA65EuzEJ+kW7vDjz+rF3YQinOdhETApuO70LFfw\\\\\\\/jTiZ4IRpJXBPQSF09T34pdBzrRqHt5Zq+Ti9SFs\\\\\\\/nFsLTLgjmMoejm+gAdW8ge0ynZhjiu6cjuvUnpifrjdbSStpTSLOle4fMIod15QE\\\\\\\/M5iLjPoOX1j38sz2fn8y5mLvdAeOf1vbV207uTglnIV4zkF\\\\\\\/FwnVS\\\\\\\/DPyHmA\\\\\\\/J8ppQ5POdnuEorpLx9ZOJvQ2YNqipvqdj+FHT5dup5YLh0T7zJ5ZH3Uo8cUqFe2zbc847ZUmspbIkdPgg28Z4hjDI5dQO6VfEjHl+1frsl8R7O2lBZ3Yoga6FrRExz5I3z9RKosDnps6NSB9BhBOItjqvNbUN95jjjGu7yuAHIAVm5in0mxC7LdVlRCeL\\\\\\\/97oaxP+5ZS2bUN\\\\\\\/bTmFe7Spl2tzAYnukncH5nETdwrcD9FjIVV11h2yEsq5CAt9W+WaxOVnZWG3L9eyyIam9wyahZrTPLzi4Mlb\\\\\\\/6PCYcpGOzlAiPgKdb0zjylN9UBgL62PjMhDsw79KLjnaGT4xMdGg05gt4CxjwrdE29saNjJ6jETYLvnUVcVxAWEDQ3WUsIxdwm5vWFL7GUcU1WWPXw63qYR+T\\\\\\\/SI6bdTdFoaBIDn\\\\\\\/mgLj1HBM5PhIKAfAxPJvxIQ10NUNMDpz\\\\\\\/+vCRrKkCFYn++WRP0\\\\\\\/ADGAdWGz3ZyHnZF2L4oaYQ7SK+TJaSLI+dzHmxJsDr+h3Ab9y4DcY8AqEF+UIYWc+tN59wo9cVQWGfhYRZrmEV9XRI2V\\\\\\\/+imVVkDRSWoTsjg7p+7ZZjG2rjYWKsD0OSJ9zgTgWbf7+eFuZHBB2STxgPq3JI9TS6E8cJEBeph2TKpZ7OCg0AoPVOi2T5+nVboo5kejtu42ALrjfVCvnjCaZtmGNXrIkj5I5i3xiBuklalOtQZL6mtRGrv9C8CfHYQELR0FRTvVGU4fBq1XMEEtO8TZKqkncDmN1B6maEXI4tIH2QdUrETWCeukZZGQTidq55pporsFIRDE99YXQo6qI24cceXWBMjHhoxqAqY0LFz0iR7b2mZRuGIl8BWTXmayKYaP6KK7Pe36MZFqrGfCr4wYqBnSGnaJo6szAzSS2vcJHNXQnBvd7Yk9Dra\\\\\\\/7omSTkDgZOkjiD3hSYAi64S64Gx0gyCX3E3lOCseRdDFWclE5rJ8G8A9y29spIkp03DJM085mWekdYo0SJb8+nvy3h4jdgsNomwS0MSQenu\\\\\\\/v2NeI4RZOVoShBR91qCOVCcau8e9dVIB0+N3qfDv07cJa4BCIJiR5kl\\\\\\\/psmmacBPx7igZ26ZRaQNiEt4VK48HPW+GtncMfOofT67Bvp4O3Sv0o6RxMbcFoTwKrcn3pjF2goBm7o3rmAqtoFhUm567Iw3oVHwa4ObNUnq67iLCwpZ5SeMmHEbRnRshlcm\\\\\\\/NN4A4yqqp+Chf5blSg8qnrIpGyMI9y0JzpPcs\\\\\\\/otkmTa5YUTKdDL\\\\\\\/wH\\\\\\\/AtHBO5MooiijqbtWrfjNlJ6esesGdvGcZNUVQSZKoDjxcf25ETr6F5WSdF9JQQE3Vfh0mKFxJ4rqr6Vb6vKzh\\\\\\\/O1SbejRDYcX3xEioDo49D1DabW+B5yvZtdk8KU5oUgSDNoIx7Jgnh1drzgNOeJSa+SSdRrbMY1NfBluF52hom3N8YrjjKLlD5K0qQFFjWva0bk5LbM8iNGlYgWIbe8cJTKOYTVASVZNTLRcovBeOQiL7XSvyWQiHkMjlvxt+sSLBldachFdhbFsJHqO98zVdprQawAx8j21qIHWxyHqhIObNdjmNPGuSdUb3AXogito1mu\\\\\\\/o6FsvScgMCNIT3AGkuzVcj8IMMq6HcNyGin6Wgq79RvSHobgD5LP8EEumtn6L6yn+WiGvY7Uqwn+Rkk00B7Ssn3D9Ig8RHAwYwA09oI4uYazVzvKwECKxJ1HjgpS2HgkR3xZcBRTnDiGbow2B6g+apuZbEwSIPAGrot47B4rb9Dvl1NwoFRbcDDXB1h0NicqmUtFehAD9X+WDXBrYxtHTZ\\\\\\\/7XY20EYkyDAC96\\\\\\\/uamMBgZnw4l+xjfmKiB5VlfZqphCHIsOhBmdTRCb1PtNQSN8rqwpYfhQkqa641tvJe3qu7qgbCfXJ28aPZ5M0o4RMF4KCZ3hUp+cQ0tNZbpYh2\\\\\\\/96Nm+ke2ptC+fHEhQuTGhVys8ySPQ2uB3rAe6Cz9DMdrZrpRvpoiiKbEfMNF6mm7+XDXI1W5hW1jez\\\\\\\/3mZBnp1qEbIfpeUgHolDdWqrWITJ12CJpTE15kiqvr+Bosiv5kX9qvhEQwzarF3qxKolgd+ekjSMWDVgEC8N2OHvA2+guw0Ko+3Dwi6XcoHB+6W8pXcaWnON5bAkqpM08+Fd\\\\\\\/QsIa0I8mXenF+wlyllfEC8N5lU\\\\\\\/U4BDQ9+DAFzlpZ69uUA1zJKEigRR2tEVQ8xrSzG\\\\\\\/k66PoerbJqxmYXvGrf17RD+RZplzx4aZe7+MVLrcVusdR6i\\\\\\\/bMBKo73SYDZEuhRyYa5RSHbxvKWNoeOMGp5F4XxW2GnMyAC6Uq\\\\\\\/BlYioAsObME29gp3qmESo\\\\\\\/fF1gm4WVtlpKvVxlkdgbjJlXd98i75HjlfwflOcipXtC3Ipexj2mtaw1OOdLIBzDpPcbMjO9\\\\\\\/NUEG7e9MUU+V8eUA9q8b3ya6mZXOD7uXFXY5Po7fxJPgro1B0usF6uqCYMgAAiZWViqGxNKvTCpD\\\\\\\/81jF\\\\\\\/E97qzebtsw0asxXOBGUcyi9buHO6wEMObDWJs8cixlWUIcwK3W98x70HuSfUhxSHJWXvarYjs3BePSLbKykngK+X\\\\\\\/OHviZLPal254RO2icCTmj1wq0AUqufQuU\\\\\\\/SUjtXryjzIz6VWPlwZlItwPnFhs9awO8nxaSXZHpnhQKq0pTnhTjC6VVd1tXYD08G9ayFwCoDBnB6VUvU5KBZWGQC5AT9aN9jjbDx3LNbQ31NcaZQMcnyT3AncvEKBwVO0Z25ouxLh96Ao2qR\\\\\\\/53fNHkWYujDC9pPPELEjQxCiqbNxWqInWvKD8sYvlwemvCXMtPaYVR2lG11c8sOaZzECSmErm2Abp3MK5IbFy4f+YNBVhAI7rdjTMcbx\\\\\\\/zILHq7pCxH+SRyQAYFHYU1KET81h3eJ88ucNyislPlEbWM1zASZGgDoJqLUbA25SN0VHrbgviiXfB8olH97+HJnXLv906ww+ndtZzR03zuR15hmOOpMfFrl1xrksXIDbSqzckQzFJhl\\\\\\\/MWlER0H2Cf6CzcLccgBrt\\\\\\\/QmFcwnG9QHLsCzIG+Dny22ptfPjSjTzQa8RQrn7qADgnfljWXpjRcxkSJkWX5CgYk3bGAbdTyEbZRO7EYcX2nG4OWaybR0JLINYP+BpB5OfO41eqzZx60Sgf0Cck0TGf\\\\\\\/oTi5121AaGZY81jzZKcY8gGkVR4k0syaAEgL1we7awLm80XBLM2Mg\\\\\\\/iabY4+iAk4vf0cpyS2M+Fews34fVhwBdX367xlgF3PYgAyQuckQhBzbmIXVzFfoO2roKQUHFoEYZ77Kp9vW593ROWbZKfk\\\\\\\/X5c\\\\\\\/9VofJZ2wuEOO6bUS5oQQx4mG1\\\\\\\/M7mOck1b4pW3P0WD+Aq01BoZ2MIlFFMDkEWwqQMGfzcWk9zAMTO97u\\\\\\\/QlR4WmmsWosGYrHldGOxGAVD9fLLAKPFrIZwiOGMrrf\\\\\\\/KS35cQhISbJHeiSOfLXUX1iLKgjXwJ36wYn4UH8IKvaRGhCZS85Fs1jZNu1r6JkcuJNnJMNFX+HHVvc1+9mXGWhwdgkfDGWiR5\\\\\\\/caLaRS56VUmeazDtS1t3diQ6nttSIbDk35oi5SyAyJW\\\\\\\/aYNID+JFxIC\\\\\\\/2KpJLsLlkWgnV5S8G7o1Vv5mWfxgUyL5tUV7i1HF7Us9Vi\\\\\\\/x+WL5g+gvg0inbXT4I16rH8QQk29wsNVvCVsJMrOgAoglcpi1ZAqzcaJTLaZHm8bbhMB4tO2jE4n3w2lzGrvXli8frzqqJSMiJy+xT5yCTJ2pYzsHpVI3TSV05O6DBb12MVjDcaAWz4ttuv4B+wJ8pkMlqfNW\\\\\\\/NrFm+DZRnzqlz4awHwOIxhUTMrjwcl5u1t03AvogOS\\\\\\\/F2YuYmG\\\\\\\/U3jFVbiU9CmrzJ\\\\\\\/Z1QNtIRuMuNgyMZDaHcY0C0hyQXdikWJmMsNM1RTfrM+ibJyQPvyL6lg5oXqSONO+oMXEP28UJ5xsk9EUNv6MmaGAM9+uIgc\\\\\\\/3LJxvZB2lOI8ce6jEzL8BhAQbKJrQt8mfn8tc49irMA+Disgmz3oJ+COJFxt3rs2vd5IaUxqu1fyGHuvlN2fWjeSP4MrgRukAQXDiKYYSQdG9ITjRE2FAVtf++5mlRtbie1ONu2MK09GcVrCH1kxQg4gaGQsKqCaclRNTnGYb9UbnZbmuHgVlDVEoDoU71ZDVY7M3c+IuKPEILKyWCEJ2FZKCs5N10a\\\\\\\/xnUVt+SgYVV12vIcSd\\\\\\\/HcnxBe2hn81TIvInodfUuDdmXua9Y3pJUwXrVP7j4DTfrJCFE4DcClRtfpGf5BVFslj7tA9j+lHFdeGcFOWATLIXf\\\\\\\/Tk\\\\\\\/hzCvNp5TPqQuQZJGHkwzRr2O2zWonErcuoed0pbws8Jfdz8cAPRrrNKJY08ROWSbe8KjGgFdpu7ag1bgAbkKcLaQtZ+yY45tZsx8y1cpXXPI48gBdfpZYgEw8eVvMb1f\\\\\\\/JXIan1ZT0Mz6jMP6MBvmMr1V0bUGRdYPigcuKOHON7TTHwBfTruebdClNYkfU\\\\\\\/v0KRFr6VNRRhNnomd8yZiVBlsi2FiqAL9ptQTEZHON\\\\\\\/gN3eXZuUuiPgpcKUv3n1dDdXMYDGeA55T70vBkHIZ9JHsvFQH\\\\\\\/9iYgKn+KWDIGYrvFcVgsw843keUzOHbZwF9o43nyZPbi+CyIh+gsonkuXBc+cERv8MQJrDs8xO0rYg6c1ly0NA+w2FJ2e32QxhJ+zQzGnvmFD4GdWsa2xPgv13qU+ADZC5RH6nWGbnGoCje4Ck\\\\\\\/pHZ55I5X3qM\\\\\\\/ukNNcE7FkUewTUIDJuNFzzogrvtP3OPSHTvL30t4I3ICM0jd4YitDM16Tm2po7n6xrfztdrPEA9H5uZ6gmmOLheZmPEHc+vxU3soF3KNtksIFSame\\\\\\\/AutVlJJ57PQ5qSEIzpDb1\\\\\\\/0HcmNwxinENBk5B9zsXNLZS5j2fl8Wf9qgwMGFNQDqlYsFEd26DfkOMeSLBTae+NVBCqoFHmHqQnbjkW3fTOjNhagzVL2F8HwscnfRzAkjFAsiWXtxF1LNLhOlmh2iJa3tgvEOVtteWMPiYLVSCae5yM\\\\\\\/Csmoh0p8+zArzp9mTTHREbsB47IUfzZYljQviVC0w716XQ9RUWg88XjhsSZ0o96yJQAoGE2vjd85IHlsow7Gna3\\\\\\\/ByNb+\\\\\\\/bc5BzeBBFnBGWR808tGM0W8yE5DmX823JhgTHF78oJIONikDj01GXlkhqS9ygqdxA4aw4AqMyFOM1f4N5RsJHdaQrVSimwVMbUJYuGNpuhMSoIbkTo6VyrkX0ZB+qWXTSeyHVB1GnHqwwF07oWmIY7JNk27iLtOkbfetRQTFp8mzn4zo3BBTwTo36PnPyKgAl81Xpi7cpdrxVmSMp4n6GqF4dHP7qIYgLlRtG3zxgs09jkMbHFknKClwuEBKBN\\\\\\\/pHiRwNAXO4M\\\\\\\/PKyA4jRoFEman1\\\\\\\/nv1XIiLtkeD6Wekv4qZPtiAemAOdvxP9lj4WcaDE2DNaIrr9bZHWn4G6zLZDpUvobx5tiKKhYFO0IjUAHF+2UiycnEbbEGJ8F8oiWJAlI9ND2XvCdBlBh5gr83Ll18yR1DRQEAP9emJel5Yd5KLmKStVjyVM5pcp22bg6E8uTLQWqApzj1hjmSY0xWYcbP8mBw5NGPv83iBLUFupbE\\\\\\\/wyjwMgKiwg+nAA+SBS+PlBTEzez7b4VvZ7wloh5FclBBI\\\\\\\/uNEMOGivnsRapuUYQ6nR9L8+58Asa99EIzLOTGb4mgJ8EPYoI1ib8xEZsuKJTc8EwupmxMeDEKpKnoq7hEh3PljxJgfnlrHltcFAVUTrzagmv4UtM9RVunP\\\\\\\/ophc6mHZYhxBbEimUqlSdWR\\\\\\\/\\\\\\\/tV0lwwyMUZqS9rwKO7bfaX3Ske1uUa20PfYO7XDl6w3lHtFnEBVv7eQuANDewrsQ3+7X2r7TRYjYYsa0xVHIuXAsNFeFMK7EA5nF6c6Wjcg0ZhD7233Fr4pujV7nivitfOgcNnULQH7cMz4FsiAYA07Vk\\\\\\\/w\\\\\\\/aR\\\\\\\/B+Ccl8btp+N\\\\\\\/JFNF1ScVkrumiuHoOC8EOglu+vcQmTFGWq2Mj14Mj1bKuskVfRVM7\\\\\\\/VSJSc\\\\\\\/eIQWlSgJXdUOYtL1MuqYeGZGaYLpThGGxtM5SlBiFwlaE\\\\\\\/mq9OaG5Xi0VDMbrJ5wOsbXSE8+ZPIdzIUkuqBKs7UggZ6l\\\\\\\/BtQTaQP8QRALuLo6YA5Vv\\\\\\\/vqDXvrmIFXu2mJm+nCWypZfpgFSjOFSKNL6iJTvQ10qaDgoAouJhixgzOppvQ1Kx9uDqwZCleYzSJLxVcRC543njKnujKut8xHISi9XD0q8WqYVc4c0FFgrgjA\\\\\\\/xu2QH8SMk6DPK4WrZkvKupag9Y+9NUf+j0S40LIGFi3see2RM1T1CLDzJjDaB600tmv5\\\\\\\/QpeSTkDGECBka5lhuP0ROrJdF3zzsbEZNhpnd6Bod76wStX3c\\\\\\\/Mt9xeVukbYNp50HLxU3rrUKxUUpIEcFRY9GoHvtrQxH2mlsFVRF\\\\\\\/s4sk\\\\\\\/D932Z4tDf9oxE4CkzdVL5ihfnSWW7\\\\\\\/LXa7y725AfpurWNj1uHlfqLo4M289l3ycohLzqdoX1T7pRwLqbiUAiUk+Ztpd6JGoUco1CVthxk\\\\\\\/YgT31zPBAVpXpaz+atBiQijVin0HPs\\\\\\\/Ju\\\\\\\/d4mhVb8iq6MN8tvxkmEPQrmgRLmuRAl6xyRndLgyitDeurFTmf990dqqxo2Z7uW3kGgQ0dmlZmeh0UqY2wnGHKHAX2cMMA7HI3FxTY4CcpfL4DyOAkz80rYcsPaBc5j8sf+\\\\\\\/whd5lqJdSS6a0n8z4qXfzlsGhXBylWzhxoHpZ6k3oIfSaidBIEEmSLW7JCHTJDtD2cC+aFx0wWz4460mdI3qu0Maj5NU6M6r6EwQRMqcxbfU6H5wsKNeCxXLEFfRzrFpYy\\\\\\\/Oo3ThXDyibjYICEaetmjdG5nwHJ3z2T6eUdvDvhnGaAXMQ7Xk\\\\\\\/MJx3r02VoXumroLD4XS40YZiqLre9+8bUxUaaznUPiAPD1ctCr0bIkAICnCeshiTSIQPRLqjrT7QXvoLjm1E\\\\\\\/DgzumtWjZvFTpU4d5LXk2IDXPOHMurUhb16Sj\\\\\\\/sGpoLWv0I+LEgA6POABXw3\\\\\\\/DBTydva6UKoxdpAqWGdK4V\\\\\\\/+UEGkZsfXE\\\\\\\/jeoRdSUZj8zLshQIbpFPz\\\\\\\/AhBrwExUzIgyB2neg30dzsJNpe\\\\\\\/JTbndyaK9beDJAW+TYLIhBNKjlnqm65dP0WGSGyWR\\\u0022,\\\u0022iv\\\u0022:\\\u0022bhTeFpuJWgmwWq29zgTIDQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002214a9b56ad6626be80b6a9969d15eea41a436dab927146495db62ed9eb6150ed7\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022lzPjQYAsbpfj7E7WjSPVyr8ru+ZNHiOSw6kmVR4vppvCVP7KM1vIZrFMCXRN3V+dcewuDQmXghmyM9z3ep49D5H8lna7Eymx3ApgDZM197cFD6CmhyGROuTLtDA0nK61nhvcRAw3v62i5w0WeNmtK0+znCmRCUq9zDS0NY0dooqAyX\\\\\\\/tU0BSRzxv+5WKpZp9wUXHCy6vp6Ys43gSNoUE\\\\\\\/NfcUtL+WeKX9VNjf62\\\\\\\/LrlbT2yROtBipuKLl1ezp0Dkh1+LcLIg7cJBtPxpRGy7cUKH4RwBn5H4gY0BfKmj2xyR3knrzE1ZX+s7f8x1ItuVEDXYFn680cVmF2LGnq\\\\\\\/ahXd68by1LINXrH+axLr3bNW9JhikZOlG711cPEujSQclHjr7uh0pDg6WmDXK5zMoQw3FI\\\\\\\/uNs5a3BFbfHoqVuaDumDCFX\\\\\\\/P1DEllyC\\\\\\\/oNdlbR8nlQiAiBUpYgS5lg3GHEvNg0YAZMTzrGOUDmrpTansD3K6ItA2MSc8I5D5aEg2mFydPNkYS4cpufqs9LGXk5Nq9\\\\\\\/Zev4XuFFs31rM6U9+MW4fmZm6WEg06hh3GxXOJhSjrLNAxcX5D5UEEw4q4Si96oCYdsd3v5PzmmCgyK8KDvCc2gq1OuNhrHOMmy3kBzyfKdtFGwqJqUDCvkWjUeftl9mFqG9fhlbd0ELL3wTpVO75Y6HmM2aJ+IZiJovs+TC7fOySLk1EgftxFCQOui1mbgAEzJY05yn2C0y1Y7A\\\\\\\/+Z75ouu3swbMU2aUaOp832jXtoaoVHSSQRQ7E+opPSi2HzeInhHqHOBM8lVhhI7mpBi11zJKHs4up5QNhE4ZzGwunmNgQp9Fn1TAu8L4roSt1p0PDgPhZI8QKzARHuvI\\\\\\\/5It6qMK2uPj6g8zcsJMwfD6PZSEBsjdPMKtYHyoaRVTl2f9k+kCi12gRQ8doit2f4HweNOJeaIQL+ahnYL+0IecCWJb1pEA4OtnZF8bAjPfZXBto0yd42zCa12k4Hum6oiLawp7V77C+dpUhKaaElYom26+ubr0HDqInOhK3GFRSD76tx+agk0Dwhhkfh\\\\\\\/rwLCzonmzrGKq4k\\\\\\\/\\\\\\\/LwqZii0cBZAWnrQ\\\\\\\/14R5Sbjm+jO1fdBgr3sWq\\\\\\\/nYcsyVQBzqo1ukxT98rpAYX9YrVXVBOh0YAwtNJ\\\\\\\/iJP7WV+r9H7Rl37VTboQAN910ucuTfI90+g09c0iCfy0H\\\\\\\/0b1+X0qgVBqlTYa3Sw1eBD42bzjVDGoNGRG3QkVrEyYPdcILpDVNHhFZPkqB+tYKKirMtluuug2je7\\\\\\\/jfbY9hrpyjpmhQJM2x3kG09KFgeG+LRyIi4x3EAbqLBbYGkym4IYRqh0iikj0fLqv5dIewQn\\\\\\\/MGk\\\\\\\/v3QEWjJIyTDBCiyOl92ANx\\\\\\\/0UicaMjjl\\\\\\\/SMthXoAxA6m+Wlt+SH8aWmq8HpDpvFh\\\\\\\/Af8PM2CBhgY3JJLmCQaA5FivkXleJWrzwEkAWCLHs7OjS9bHtU\\\\\\\/Ht2MK2dfR4BQ8zwDuOpNKaQykXOt3krPkI0sR69MRP1q0hBeuNHia7cuvWERJJr3opUN2zmsF3mKFXt5GnLbWIplLBdJQ\\\\\\\/QNor3MBB7WyILTdnd5AIQNqtykKQLtjAocQ\\\\\\\/RcInGNK8KAC2vVNQEU9qhajjRnyoxJqPQFNiiSLbzslc6nL5hqPbLuXmp8oH0NWtCPVFVtqI6NUGKtlrdUrxFu9ZLZeUPENuPu+ImqTU+ict2\\\\\\\/PqXzmxX9LFBZihfzA0LPUJDJzptn36VW0EgHsVsf36Ryzz41SiQP2ayeuRPy3JP10Y\\\\\\\/chpUgQsFM5cZAp3MwlEIkAvl4CPHlVJRQveF+SQ6N4vVup+EwJ8rt4IXgyeqFYOpyMfk51Of\\\\\\\/B8IdS\\\\\\\/SiX\\\\\\\/+fHuVeh43CXwCbSCR1vmncRhjF5CmRVZQnN302BGdJUkiSfRR6SA+lKJ69aptJMeHisJTZ2gCUjelSLJxHv06ehdv7lGk19XhHQ\\\\\\\/guATbgm4BbWh6PXC4+OlNSFCwQ9G9n3VD1lHKnz5XOCFCc\\\\\\\/ksS6MED6k244=\\\u0022,\\\u0022iv\\\u0022:\\\u00225h7yiCZOJe3U4mlOkwu\\\\\\\/rw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002275978f5d6ae54292e6c1fa8d6ed6d78e79668f0b2aa6f9d0c5e283fe3cf6c806\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-486354 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/9/3059" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-SObVffeY0_EvpOT_Nvz5xzS4IJX1-GKSyvMihvH-z9Q" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/9/3059" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-50ZReW76DTRKo-yiQQdDA0euKlbcMr3TxCWbpvFlhhw" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/9/3047" title="Annotated Draft Genome Assemblies for the Northern Bobwhite (Colinus virginianus) and the Scaled Quail (Callipepla squamata) Reveal Disparate Estimates of Modern Genome Diversity and Historic Effective Population Size" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/9/3073" title="Genome-Wide Transcriptional Dynamics in the Companion Bacterial Symbionts of the Glassy-Winged Sharpshooter (Cicadellidae: Homalodisca vitripennis) Reveal Differential Gene Expression in Bacteria Occupying Multiple Host Organs" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="486354" id="node486354" data-pisa="ggg;7/9/3059" data-pisa-master="ggg;g3.117.1137" data-apath="/ggg/7/9/3059.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Whole-Genome Sequencing and Concordance Between Antimicrobial Susceptibility Genotypes and Phenotypes of Bacterial Isolates Associated with Bovine Respiratory Disease</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-7821-3012" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Joseph R. Owen</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-6149-1008" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Noelle Noyes</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-4181-4233" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Amy E. Young</span>, <span class="highwire-citation-author" data-delta="3">Daniel J. Prince</span>, <span class="highwire-citation-author" data-delta="4">Patricia C. Blanchard</span>, <span class="highwire-citation-author" data-delta="5">Terry W. Lehenbauer</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0003-0330-5013" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Sharif S. Aly</span>, <span class="highwire-citation-author" data-delta="7">Jessica H. Davis</span>, <span class="highwire-citation-author" data-delta="8">Sean M. O’Rourke</span>, <span class="highwire-citation-author" data-delta="9">Zaid Abdo</span>, <span class="highwire-citation-author" data-delta="10">Keith Belk</span>, <span class="highwire-citation-author" data-delta="11">Michael R. Miller</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="12"><a href="http://orcid.org/0000-0001-8138-2714" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Paul Morley</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="13"><a href="http://orcid.org/0000-0003-1562-162X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Alison L. Van Eenennaam</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3059-3071;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.1137 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node486354" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Joseph R. Owen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Animal Science, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joseph%2BR.%2BOwen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Owen%20JR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJoseph%2BR.%2BOwen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-7821-3012" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Joseph R. Owen</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Noelle Noyes </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Microbiology, Immunology, and Pathology, College of Veterinary Medicine and Biomedical Sciences, Colorado State University, Fort Collins, Colorado 80523</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Noelle%2BNoyes%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Noyes%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ANoelle%2BNoyes%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-6149-1008" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Noelle Noyes</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Amy E. Young </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Animal Science, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Amy%2BE.%2BYoung%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Young%20AE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAmy%2BE.%2BYoung%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-4181-4233" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Amy E. Young</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Daniel J. Prince </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Animal Science, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniel%2BJ.%2BPrince%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Prince%20DJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADaniel%2BJ.%2BPrince%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Patricia C. Blanchard </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>California Animal Health and Food Safety Laboratory, School of Veterinary Medicine, University of California, Davis, Tulare, California 93274</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Patricia%2BC.%2BBlanchard%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Blanchard%20PC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APatricia%2BC.%2BBlanchard%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Terry W. Lehenbauer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Veterinary Medicine Teaching and Research Center, School of Veterinary Medicine, University of California, Davis, Tulare, California 93274</div><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Population Health and Reproduction, School of Veterinary Medicine, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Terry%2BW.%2BLehenbauer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lehenbauer%20TW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATerry%2BW.%2BLehenbauer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Sharif S. Aly </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Veterinary Medicine Teaching and Research Center, School of Veterinary Medicine, University of California, Davis, Tulare, California 93274</div><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Population Health and Reproduction, School of Veterinary Medicine, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sharif%2BS.%2BAly%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Aly%20SS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASharif%2BS.%2BAly%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-0330-5013" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Sharif S. Aly</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Jessica H. Davis </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Veterinary Medicine Teaching and Research Center, School of Veterinary Medicine, University of California, Davis, Tulare, California 93274</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jessica%2BH.%2BDavis%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Davis%20JH&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJessica%2BH.%2BDavis%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Sean M. O’Rourke </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Animal Science, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sean%2BM.%2BO%E2%80%99Rourke%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=O%E2%80%99Rourke%20SM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASean%2BM.%2BO%25E2%2580%2599Rourke%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Zaid Abdo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Microbiology, Immunology, and Pathology, College of Veterinary Medicine and Biomedical Sciences, Colorado State University, Fort Collins, Colorado 80523</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zaid%2BAbdo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Abdo%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AZaid%2BAbdo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Keith Belk </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>Department of Animal Sciences, College of Agricultural Sciences, Colorado State University, Fort Collins, Colorado 80523</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Keith%2BBelk%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Belk%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKeith%2BBelk%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Michael R. Miller </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Animal Science, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BR.%2BMiller%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Miller%20MR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichael%2BR.%2BMiller%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-name">Paul Morley </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>Department of Clinical Sciences, College of Veterinary Medicine and Biomedical Sciences, Colorado State University, Fort Collins, Colorado 80523</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Paul%2BMorley%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Morley%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APaul%2BMorley%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-8138-2714" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Paul Morley</a></li></ul></div><div class="author-tooltip-13"><div class="author-tooltip-name">Alison L. Van Eenennaam </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Animal Science, University of California, Davis, California 95616</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Veterinary Medicine Teaching and Research Center, School of Veterinary Medicine, University of California, Davis, Tulare, California 93274</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alison%2BL.%2BVan%2BEenennaam%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Van%20Eenennaam%20AL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAlison%2BL.%2BVan%2BEenennaam%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-1562-162X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Alison L. Van Eenennaam</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:alvaneenennaam@ucdavis.edu" class="" data-icon-position="" data-hide-link-title="0">alvaneenennaam@ucdavis.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/7/9/3059" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:486354" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:486354/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/9/3059.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:486354" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:486354/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/9/3059.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:486354" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:486354/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/9/3059.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:486354" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:486354/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/7/9/3059.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:486354" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:486354/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Extended laboratory culture and antimicrobial susceptibility testing timelines hinder rapid species identification and susceptibility profiling of bacterial pathogens associated with bovine respiratory disease, the most prevalent cause of cattle mortality in the United States. Whole-genome sequencing offers a culture-independent alternative to current bacterial identification methods, but requires a library of bacterial reference genomes for comparison. To contribute new bacterial genome assemblies and evaluate genetic diversity and variation in antimicrobial resistance genotypes, whole-genome sequencing was performed on bovine respiratory disease–associated bacterial isolates (<em>Histophilus somni</em>, <em>Mycoplasma bovis</em>, <em>Mannheimia haemolytica</em>, and <em>Pasteurella multocida</em>) from dairy and beef cattle. One hundred genomically distinct assemblies were added to the NCBI database, doubling the available genomic sequences for these four species. Computer-based methods identified 11 predicted antimicrobial resistance genes in three species, with none being detected in <em>M. bovis</em>. While computer-based analysis can identify antibiotic resistance genes within whole-genome sequences (genotype), it may not predict the actual antimicrobial resistance observed in a living organism (phenotype). Antimicrobial susceptibility testing on 64 <em>H. somni</em>, <em>M. haemolytica</em>, and <em>P. multocida</em> isolates had an overall concordance rate between genotype and phenotypic resistance to the associated class of antimicrobials of 72.7% (<em>P</em> &lt; 0.001), showing substantial discordance. Concordance rates varied greatly among different antimicrobial, antibiotic resistance gene, and bacterial species combinations. This suggests that antimicrobial susceptibility phenotypes are needed to complement genomically predicted antibiotic resistance gene genotypes to better understand how the presence of antibiotic resistance genes within a given bacterial species could potentially impact optimal bovine respiratory disease treatment and morbidity/mortality outcomes.</p></div><ul class="kwd-group"><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/histophilus-somni" class="hw-term hw-article-keyword hw-article-keyword-histophilus-somni" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Histophilus somni</em></a></li><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/mycoplasma-bovis" class="hw-term hw-article-keyword hw-article-keyword-mycoplasma-bovis" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Mycoplasma bovis</em></a></li><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/mannheimia-haemolytica" class="hw-term hw-article-keyword hw-article-keyword-mannheimia-haemolytica" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Mannheimia haemolytica</em></a></li><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/pasteurella-multocida" class="hw-term hw-article-keyword hw-article-keyword-pasteurella-multocida" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Pasteurella multocida</em></a></li></ul><p id="p-3">Historically, bacterial pathogens were identified by biochemical, antigen, or PCR-based methods, with results taking from days to weeks. Culture-independent diagnostic testing (CIDT), typically used to identify slow-growing agents or those that cannot be cultured <em>in vitro</em> (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Janda and Abbot 2014</a>), is gaining attention due to the availability of whole-genome sequence (WGS) data. WGS can reveal a pathogen’s identity and uncover other clinically useful information typically garnered through multiple diagnostic tests (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Gilmour <em>et al.</em> 2013</a>).</p><p id="p-4">Employment of WGS for pathogen identification in <em>Salmonella</em> (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Mohammed and Cormican 2015</a>), <em>Klebsiella</em> <em>pneumoniae</em> (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Marsh <em>et al.</em> 2015</a>), <em>Neisseria</em> <em>meningitidis</em> (<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Mustapha <em>et al.</em> 2015</a>), and <em>Escherichia coli</em> (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Berenger <em>et al.</em> 2015</a>) outbreaks demonstrated the effectiveness of this approach, but highlighted significant limitations due to the paucity of published pathogen genome sequences. It was recognized that broad sampling and establishing a global genomic database from isolates would be required to understand the genetic diversity and distribution of traits within a pathogen population (<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Mutreja <em>et al.</em> 2011</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Koser <em>et al.</em> 2014</a>).</p><p id="p-5">CIDT could be useful in characterizing pathogens associated with bovine respiratory disease (BRD) complex, the most prevalent and costly cause of mortality for cattle in the United States (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Chirase and Green 2000</a>; <a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">USDA 2011a</a>). Several bacterial pathogens are associated with BRD, with <em>Histophilus somni</em>, <em>Mycoplasma bovis</em>, <em>Mannheimia haemolytica</em>, and <em>Pasteurella multocida</em> being the most common (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Griffin <em>et al.</em> 2010</a>). Complete bacterial genomes are available for two strains of <em>H. somni</em>, five of <em>M. bovis</em>, 10 of <em>M. haemolytica</em>, and 11 of <em>P. multocida</em> (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/486484/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/486484/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Known reference sequences for the species sequenced in this study</span><div class="sb-div caption-clear"></div></div></div><p id="p-6">It is important to consider antimicrobial resistance and the spread of antimicrobial resistance genes (ARG) in treatment decisions associated with common diseases such as BRD (<a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">Koser <em>et al.</em> 2014</a>). Understanding the epidemiology of antimicrobial resistance in BRD pathogens in healthy cattle, as well as treated and untreated sick cattle, is crucial to understanding the role of resistance in BRD treatment failure. In addition, the increasing use of WGS in antimicrobial resistance surveillance necessitates a better understanding of the degree of concordance between phenotypic susceptibility and resistance genotype.</p><p id="p-7">The primary objective of this study was to perform WGS on bacterial isolates collected from untreated BRD-affected beef and dairy cattle, and unaffected dairy cattle, to contribute additional bacterial genome sequences that could be used for CIDT in the future. Secondary study objectives were to perform bioinformatic analyses and susceptibility testing to analyze concordance between phenotypic susceptibility and resistance genotype. Identification of distinguishing genomic characteristics, such as ARG, of different strains of the major BRD pathogens may facilitate a better understanding of these pathogens and their host interactions with the goal of identifying improved treatment methods specific to the host species, ARG genotype, and pathogen combination.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Cattle sampling</h3><p id="p-8">Personnel at a northern California feedlot and a central California dairy calf nursery cared for the animals used in this study. In the calf nursery, preweaned Holstein dairy calves 22–60 d of age, who had never been treated for BRD nor treated with antimicrobials for any reason within the previous 10 d, were enrolled in a matched case-control study between July 2011 and January 2012, as described previously (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Love <em>et al.</em> 2014</a>; <a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Neibergs <em>et al.</em> 2014</a>). Upon morning observation, BRD suspect calves and matched adjacent control calves were scored using a clinical scoring system that assigns numerical values from 0 to 3 based on severity of symptoms for rectal temperature, cough, nasal discharge, ocular discharge, ear or head tilt or droop, and fecal consistency, with a score of ≥5 classified as a case (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">McGuirk 2008</a>). Guarded deep nasopharyngeal swabs were collected from cases and control calves and placed into 1 ml of bacterial transport media (<em>Brucella</em> broth with 15% glycerol) for the detection of bacteria as described (<a id="xref-ref-53-2" class="xref-bibr" href="#ref-53">Love <em>et al.</em> 2014</a>). All samples were stored on wet ice in the dark until completion of sampling and delivery to the lab for aerobic bacteria and <em>Mycoplasma</em> culturing (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Fulton and Confer 2012</a>). Similar procedures were followed for beef cattle diagnosed as BRD cases on a California feedlot. Sampling was performed in compliance with approved Institutional Animal Care and Use protocols at the University of California, Davis (protocols 16431 and 17974).</p></div><div id="sec-3" class="subsection"><h3>Bacterial isolates</h3><p id="p-9">Aerobic cultures from dairy calves were set up on sheep blood agar and chocolate agar plates and incubated at 37° in 7% CO<sub>2</sub> for 2–4 d. Swabs for <em>Mycoplasma</em> isolation were swished in <em>Mycoplasma</em> D broth and incubated aerobically at 37° for 2–3 d, then plated to <em>Mycoplasma</em> N plates and incubated at 37° in 7% CO<sub>2</sub> for up to 7 d. Bacteria from dairy cattle were isolated by morphology and subjected to testing for species identification at the California Animal Health and Food Safety Laboratory System Tulare laboratory. Urease detection, triple sugar iron agar, oxidase, indole production, hemolysis on sheep blood agar, and Gram stain testing were performed on <em>M. haemolytica</em>, <em>P. multocida</em>, and <em>H. somni</em> isolates. β-Galactosidase testing was performed on <em>P. multocida</em> and <em>M. haemolytica</em> and trehalose fermentation testing was performed on all <em>M. haemolytica</em> samples. Catalase and motility were also performed on <em>H. somni</em> isolates. <em>M. bovis</em> samples were confirmed as <em>Mycoplasma</em> by colony morphology, Dienes stain, and digitonin susceptibility prior to identification.</p><p id="p-10">Ninety-four isolates of the four target organisms were frozen at −80°. Isolates were preferentially selected from cases rather than controls, generally from samples with moderate to large numbers of the organism in pure or predominant culture, and selection was spread out over the 6 months of the study. Four nonhemolytic <em>M. haemolytica</em> isolates were identified and removed from further analysis. <em>Mycoplasma</em> spp. were picked based on rapid growth and classic appearance for <em>M. bovis</em>, with only one morphotype present to prevent mixed cultures. In total, 74 case samples and 20 control samples were submitted for sequencing: <em>H. somni</em> (<em>n</em> = 16; 13 cases, three controls), <em>M. bovis</em> (<em>n</em> = 20; 17 cases, three controls), <em>M. haemolytica</em> (<em>n</em> = 28; 20 cases, eight controls), and <em>P. multocida</em> (<em>n</em> = 30; 24 cases, six controls). Species-identified isolates from dairy cattle were streaked onto various media and cultured at 37° for 1–5 d, depending on the species (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). A single colony was added to 5 ml of PPLO broth for <em>M. bovis</em> isolates and BHI broth for the other three species, and incubated at 37° for 1–5 d prior to DNA isolation.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/486543/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/486543/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Growth conditions for each of the four species used during this study</span><div class="sb-div caption-clear"></div></div></div><p id="p-12">One swab from each of the beef cattle sampled was plated on sheep blood agar, Columbia agar, and PPLO agar plates, and cultured at 37° for 1–5 d. Ten single colonies were isolated by morphology, replated, and cultured as described. A total of 10 samples were submitted from beef feedlot cattle: <em>M. haemolytica</em> (<em>n</em> = 2; two cases) and <em>P. multocida</em> (<em>n</em> = 8; eight cases).</p></div><div id="sec-4" class="subsection"><h3>DNA isolation</h3><p id="p-13">A 2.5 ml culture was centrifuged at 13,000 rpm for 5 min for <em>H. somni</em>, <em>M. haemolytica</em>, and <em>P. multocida</em>, and for 20 min for <em>M. bovis</em>. The supernatant was discarded and the pellet was resuspended in 200 μl of PBS. DNA was extracted using the QIAamp Cador Pathogen Mini Kit (Qiagen Inc., Valencia, CA) according to the manufacturer’s instructions.</p></div><div id="sec-5" class="subsection"><h3>PCR identification</h3><p id="p-14">PCR with species-specific primers was used to identify the four bacterial species from isolates obtained from beef cattle, and to confirm the biochemical (aerobic) and morphologic (<em>Mycoplasma</em>) identification of the isolates from the dairy cattle. Primers were designed in Primer3 (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Koressaar and Remm 2007</a>; <a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Untergrasser <em>et al.</em> 2012</a>) to targeted genes specific to each species (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>). PCR protocols for <em>M. haemolytica</em> (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Alexander <em>et al.</em> 2008</a>) and <em>P. multocida</em> (<a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Ullah <em>et al.</em> 2009</a>) were performed as described previously. Protocols for <em>H. somni</em> and <em>M. bovis</em> were developed during this study. PCR was performed on an Eppendorf Mastercycler (Eppendorf, Hamburg, Germany) with 1× GoTAQ Green Master Mix (Promega Biosciences LLC, San Luis Obispo, CA), 10 μM of each primer and 10 ng of DNA for 5 min at 95°, 35 cycles of 30 sec at 95°, 30 sec at 56°, and 45 sec at 72°, followed by 10 min at 72°. Products were visualized on a 1.5% agarose gel using a ChemiDoc-It<sup>TS2</sup> Imager (UVP, LLC, Upland, CA), purified using the QIAquick PCR Purification Kit (Qiagen), Sanger sequenced (<a href="http://www.davissequencing.com">www.davissequencing.com</a>) and aligned to their respective reference genomes using Bowtie2–default v2.2.8 (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Langmead and Salzberg 2012</a>) to verify that the appropriate target regions were amplified.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/486549/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/486549/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Primers and PCR amplification protocols used in this study</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-6" class="subsection"><h3>Library construction and sequencing</h3><p id="p-15">The PCR-confirmed DNA samples were sonicated on a Diagenode Bioruptor Pico Ultrasonicator (Diagenode Inc., Denville, NJ) to shear the DNA into 250 bp fragments. Next-generation sequencing libraries were created using the Kapa Genomic DNA Library Preparation kit (Kapa Biosystems, Wilmington, MA). The NEBNext Multiplex Oligos for Illumina kit (New England BioLabs Inc., Ipswich, MA) was used for indexing. The samples were sequenced (whole-genome shotgun, paired-end, 100 bp) on an Illumina HiSeq2500 sequencer at the Vincent J. Coates Genomics Sequencing Laboratory at UC Berkeley (supported by NIH S10 Instrumentation grants S10RR029668 and S10RR027303). A total of 104 isolates were whole-genome sequenced, along with five technical replicates.</p></div><div id="sec-7" class="subsection"><h3>Read cleanup and de novo assembly</h3><p id="p-16">Raw reads were run through expHTS v0.0.Oct212015 (<a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Street <em>et al.</em> 2015</a>), which screens for contaminants using a mapped based approach, de-duplicates reads using Super-Deduper v2.0, performs quality trimming using Sickle v1.40, overlaps paired-end reads using Flash2 v2.2.00, and performs a quality analysis. Cleaned reads were then <em>de novo</em> assembled using the SPAdes assembler v3.5.0 (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bankevich <em>et al.</em> 2012</a>) by running through the expHTS pipeline. Contigs were aligned to GenBank using BLAST (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Altschul <em>et al.</em> 1990</a>).</p></div><div id="sec-8" class="subsection"><h3>Mapping assembly and genetic diversity</h3><p id="p-17">Available reference sequences were obtained from NCBI for each of the four species of interest (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). Sequence reads from each isolate were aligned to their respective reference sequences using Bowtie2–default v2.2.8. SAM files were converted to BAM files and sorted using SAMtools v1.3 (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Li <em>et al.</em> 2009</a>). Consensus sequences were called using ANGSD v0.915 (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Korneliussen <em>et al.</em> 2014</a>) using -doFasta 3, which uses the base with the highest effective depth, and a minimum MAPQ score = 20. Average coverage for each consensus sequence was determined using SAMtools Depth v1.3. Consensus sequences were aligned and genetic distance trees, which were bootstrapped 100 times, were generated using SeaView v4.6.1 (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Gouy <em>et al.</em> 2010</a>). <em>M. haemolytica</em> sample sequences mapped to reference sequence USA-ARS-USMARC-183 were genotyped using previously described methods (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Clawson <em>et al.</em> 2016</a>). Statistical significance of genotype and the probability of being a case or control was assessed using the Fisher’s exact test.</p></div><div id="sec-9" class="subsection"><h3>Antimicrobial resistance</h3><p id="p-18">Raw reads and <em>de novo</em> assembled contigs from each sample were aligned to the Resfinder database (v3-20-2017; <a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">Zankari <em>et al.</em> 2012</a>) using Short Read Sequence Typing for Bacterial Pathogens (SRST2; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Inouye <em>et al.</em> 2014</a>) v0.2.0 and ABRicate v0.5-dev (<a href="https://github.com/tseemann/abricate">https://github.com/tseemann/abricate</a>), respectively, using default settings to identify resistance genes. For gene variants that confer resistance, <em>de novo</em> assembled contigs were aligned to the Comprehensive Antibiotic Resistance Database (CARD; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Jia <em>et al.</em> 2017</a>) variant model database using the BLAST-like alignment tool (BLAT; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Kent 2002</a>) v35, and visually inspected for resistance-conferring SNPs. Raw reads from each sample were aligned to the PlasmidFinder database (v3-23-2017; Carattolli <em>et al.</em> 2014) using the SRST2 v0.2.0 using default settings. The NCBI reference sequences were aligned to the Resfinder database using ABRicate.</p></div><div id="sec-10" class="subsection"><h3>Susceptibility testing</h3><p id="p-19">Microdilution susceptibility testing of isolates was performed at the Kansas State Veterinary Diagnostic Laboratory using the Sensititre bovine/porcine plate (Thermo Scientific; plate code BOPO6F, Supplemental Material, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.1137/-/DC1/TableS1.xlsx">Table S1</a>), which tests for susceptibility to 18 drugs across nine antimicrobial classes: (i) β-lactam [Ampicillin (AMP), Ceftiofur (CEFT), Penicillin (PEN)]; (ii) tetracycline [Chlortetracycline (CHLOR), Oxytetracycline (OXY)]; (iii) fluoroquinolone [Danofloxacin (DAN), Enrofloxacin (ENRO)]; (iv) phenicol [Florfenicol (FLOR)]; (v) aminoglycoside [Gentamicin (GEN), Neomycin (NEO), Spectinomycin (SPEC)]; (vi) sulfonamide [Sulfadimethoxine (SUL)]; (vii) trimethoprim/sulfonamide [Trimethoprim-Sulfamethoxazole (TMS)]; (viii) macrolides-lincosamides (MLS) [Clindamycin (CLIN), Tilmicosin (TIL), Tulathromycin (TUL) Tylosin tartrate (TYL)]; and (ix) pleuromutilans [Tiamulin (TIA)] (<a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>). Breakpoints were chosen based on cattle-specific Clinical and Laboratory Standards Institute (CLSI) guidelines (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">CLSI 2013</a>) when possible. CLSI swine-specific breakpoints were used for AMP and TIA, canine-specific breakpoints for CLIN and canine- and equine-specific breakpoints for GEN, and human-specific breakpoints for SUL and TMS. The CLSI breakpoints were not available for NEO and TYL, and therefore human-specific breakpoints from the National Antimicrobial Resistance Monitoring System were used (<a id="xref-table-wrap-4-2" class="xref-table" href="#T4">Table 4</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">FDA 2013</a>). Wherever possible, breakpoints for the actual pathogens tested in this study were utilized. Based on these breakpoints, isolates were classified as susceptible, intermediate, or resistant. For analysis purposes, susceptible and intermediate isolates were considered to be susceptible. Crude prevalence of phenotypic resistance for each drug tested was calculated by bacterial species and across all three species. In addition, prevalence of multidrug resistant (MDR) phenotypes (defined as resistance to more than two antimicrobial drug classes) was calculated and MDR patterns were assessed.</p><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/486587/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/486587/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">Mapping of identified resistance genes to antimicrobial drugs included on the broth microdilution phenotypic susceptibility panel</span><div class="sb-div caption-clear"></div></div></div><p id="p-30">In order to compare phenotypic and genotypic susceptibility results, identified resistance genes were mapped to the antimicrobial drug classes to which they are known to confer resistance (<a id="xref-table-wrap-4-3" class="xref-table" href="#T4">Table 4</a>). Proportion of isolates with concordant and discordant genotype–phenotype results were calculated for each antimicrobial drug and organism.</p><p id="p-31">Statistical significance of discordance was assessed using the mid-<em>P</em> exact for binomial paired data with a two-tailed <em>P</em>-value (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Edwards 1948</a>), unless there were at least 20 discordant observations or the marginal counts were equal to zero, in which case McNemar’s test with continuity correction was used (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Fagerland <em>et al.</em> 2013</a>). If there was a lack of observations in any of the genotype–phenotype concordance–discordance combinations, 0.5 was added to the count in order to enable calculation of the test statistic. OpenEpi was used for all calculations (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Dean <em>et al.</em> 2013</a>). A two-tailed <em>P</em>-value was used and the α level was 0.05.</p></div><div id="sec-11" class="subsection"><h3>Data availability</h3><p id="p-32">Sequence reads are available in the NCBI Sequence Read Archive as Bioproject PRJNA306895 and SRA accession number SRP095705. <em>De novo</em> assemblies are available as NCBI whole genome shotgun sequences (accession numbers: LQCX01-LQGS01). GenBank accession numbers are SRR5133037–SRR5133136. Additional supporting data are provided in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.1137/-/DC1/FigureS1.tiff">Figure S1</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.1137/-/DC1/FigureS2.tiff">Figure S2</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.1137/-/DC1/FigureS3.tiff">Figure S3</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.1137/-/DC1/FigureS4.tiff">Figure S4</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.1137/-/DC1/FigureS5.tiff">Figure S5</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.1137/-/DC1/FigureS6.tiff">Figure S6</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.1137/-/DC1/FileS1.xlsx">File S1</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.1137/-/DC1/TableS1.xlsx">Table S1</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.1137/-/DC1/TableS2.xlsx">Table S2</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.1137/-/DC1/TableS3.xlsx">Table S3</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.1137/-/DC1/TableS4.xlsx">Table S4</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.1137/-/DC1/TableS5.xlsx">Table S5</a>.</p></div></div><div class="section results" id="sec-12"><h2 class="">Results</h2><div id="sec-13" class="subsection"><h3>Sequencing and assembly</h3><p id="p-33">Laboratory-confirmed isolates of <em>H. somni</em> (<em>n</em> = 16; 13 cases, three controls), <em>M. bovis</em> (<em>n</em> = 20; 17 cases, three controls), <em>M. haemolytica</em> (<em>n</em> = 28; 20 cases, eight controls), and <em>P. multocida</em> (<em>n</em> = 30; 24 cases, six controls) from dairy cattle; and <em>M. haemolytica</em> (<em>n</em> = 2; two cases) and <em>P. multocida</em> (<em>n</em> = 8; eight cases) from beef cattle were sequenced, and raw reads were processed and assembled (<a id="xref-table-wrap-5-1" class="xref-table" href="#T5">Table 5</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.1137/-/DC1/TableS2.xlsx">Table S2</a>). <em>Staphylococcus aureus</em> contamination was identified in one of the <em>P. multocida</em> samples isolated from an affected dairy calf and from two of the beef cattle samples, as well as in one of the beef cattle <em>M. haemolytica</em> samples. These samples were excluded from further analysis. A total of 100 sequences—16 <em>H. somni</em> (all dairy), 20 <em>M. bovis</em> (all dairy), 29 <em>M. haemolytica</em> (28 dairy, one beef), and 35 <em>P. multocida</em> (29 dairy, six beef)—were selected for further analysis. Raw reads from these 100 samples were mapped to each of their respective reference sequences and subjected to further analysis.</p><div id="T5" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/486402/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/486402/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 5</span> <span class="caption-title">Average assembly statistics for each species sequenced in this study from <em>de novo</em> assembled contigs</span><div class="sb-div caption-clear"></div></div></div><p id="p-35">Of the 29 <em>M. haemolytica</em> samples, 22 were classified as genotype 2 and the other seven were classified as genotype 1, as recently described by <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Clawson <em>et al.</em> (2016)</a>. Of the 22 samples classified as genotype 2, 19 were obtained from cases (86.4%; <em>P</em> &lt; 0.01); while of the seven samples classified as genotype 1, five were obtained from controls (71.4%; <em>P</em> &lt; 0.01). Samples classified as genotype 2 have largely been associated with those taken from BRD-affected animals, while samples classified as genotype 1 have been associated with nonclinical isolates of <em>M. haemolytica</em>. There have been some genotype 1 samples obtained from BRD-affected animals, as well as genotype 2 samples obtained from non-BRD–affected animals (<a id="xref-ref-13-3" class="xref-bibr" href="#ref-13">Clawson <em>et al.</em> 2016</a>).</p><p id="p-36">Genetic distance trees were generated to visualize the genetic distance between the genomic sequences and mapped assemblies. Overall, the sequence assemblies for the samples collected in this study increased the genetic diversity for available WGS for three out of the four species analyzed, <em>H. somni</em>, <em>M. bovis</em>, and <em>P. multocida</em> (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). Five of the <em>H. somni</em> samples contained sequences that were genomically distant from both available reference sequences, while the other 11 samples were genomically distant from reference sequence 129PT (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.1137/-/DC1/FigureS1.tiff">Figure S1</a>a). While the <em>M. bovis</em> samples showed little variation among themselves, all 20 samples were genomically distant from the five reference sequences (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.1137/-/DC1/FigureS1.tiff">Figure S1</a>b). The <em>M. haemolytica</em> sample population was the only species that revealed little genetic variation between the samples collected and the available reference sequences (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.1137/-/DC1/FigureS1.tiff">Figure S1</a>c). The seven samples classified as genotype 1 clustered together away from the other 22 samples (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.1137/-/DC1/FigureS1.tiff">Figure S1</a>c). The <em>P. multocida</em> sample population showed significant variation between the samples collected and six of the 11 available reference sequences (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.1137/-/DC1/FigureS1.tiff">Figure S1</a>d). For samples aligned to <em>P. multocida</em> reference sequences 3480, HN06, OH1905, OH4807, PMTB2.1, and Pm70, the study samples clustered together away from the reference sequence, showing a large distance between the sample group and the reference sequences. For the remaining five reference sequences, 36950, ATCC_43137, HB01, HB03, and Pm-3, there was little genetic diversity between the samples collected and the reference sequences.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/9/3059/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genetic distance trees from mapped assemblies of the four bacterial species, Histophilus somni aligned to reference sequence 2336, Mycoplasma bovis aligned to Hubei1, Mannheimia haemolytica aligned to reference sequence M42548, and Pasteurella multocida aligned to reference sequence 36950. Red circle, beef cattle; blue box, reference sequence; green box, M. haemolytica genotype 1 (Clawson et al. 2016). Scale bar represents number of substitutions per site." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-43988265" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Genetic distance trees from mapped assemblies of the four bacterial species, &lt;em&gt;Histophilus somni&lt;/em&gt; aligned to reference sequence 2336, &lt;em&gt;Mycoplasma bovis&lt;/em&gt; aligned to Hubei1, &lt;em&gt;Mannheimia haemolytica&lt;/em&gt; aligned to reference sequence M42548, and &lt;em&gt;Pasteurella multocida&lt;/em&gt; aligned to reference sequence 36950. Red circle, beef cattle; blue box, reference sequence; green box, &lt;em&gt;M. haemolytica&lt;/em&gt; genotype 1 (Clawson &lt;em&gt;et al.&lt;/em&gt; 2016). Scale bar represents number of substitutions per site.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/9/3059/F1.medium.gif" width="440" height="372"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/7/9/3059/F1.medium.gif" width="440" height="372"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/9/3059/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/9/3059/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/486554" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-37" class="first-child">Genetic distance trees from mapped assemblies of the four bacterial species, <em>Histophilus somni</em> aligned to reference sequence 2336, <em>Mycoplasma bovis</em> aligned to Hubei1, <em>Mannheimia haemolytica</em> aligned to reference sequence M42548, and <em>Pasteurella multocida</em> aligned to reference sequence 36950. Red circle, beef cattle; blue box, reference sequence; green box, <em>M. haemolytica</em> genotype 1 (<a id="xref-ref-13-4" class="xref-bibr" href="#ref-13">Clawson <em>et al.</em> 2016</a>). Scale bar represents number of substitutions per site.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-14" class="subsection"><h3>ARGs</h3><p id="p-38">Raw read and <em>de novo</em> assembled contigs were aligned to the Resfinder database using SRST2 and ABRicate, respectively. <a id="xref-table-wrap-4-4" class="xref-table" href="#T4">Table 4</a> lists known ARG detected and the antimicrobial(s) to which the genes can confer resistance. Phenotypic antimicrobial susceptibility testing results were obtained for 64 isolates (13 <em>H. somni</em>, 26 <em>M. haemolytica</em>, and 25 <em>P. multocida</em>) for 18 antimicrobials from nine classes. The remaining three samples of <em>H. somni</em>, three samples of <em>M. haemolytica</em> and 10 samples of <em>P. multocida</em> failed to grow in the susceptibility testing positive control media and were excluded from analysis. Using <em>in silico</em> ARG prediction methods, 11 total ARG were detected, with two genes specific to <em>H. somni</em> isolates (<em>ermF</em> and <em>dfrA14</em>), two genes specific to <em>M. haemolytica</em> isolates (<em>ROB-1</em> and <em>cat2</em>), and the other seven detected in <em>H. somni</em>, <em>M. haemolytica</em>, and <em>P. multocida</em> (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). No ARG were detected in any of the <em>M. bovis</em> samples, and therefore, these samples were not subjected to susceptibility testing. Raw reads were mapped to the PlasmidFinder database using SRST2 to determine whether ARG present were on plasmids or integrated into the genome. No plasmids containing known ARG sequences were detected using this method. Additionally, alignment of <em>de novo</em> assembled contigs to the CARD protein variant model database did not reveal any variants in genes known to confer resistance to antimicrobials.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/9/3059/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Prevalence of resistance genes among isolates of Histophilus somni (N = 16), Mannheimia haemolytica (N = 29), and Pasteurella multocida (N = 35), as well as available reference sequences. +, present; &#x2212;, absent." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-43988265" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Prevalence of resistance genes among isolates of &lt;em&gt;Histophilus somni&lt;/em&gt; (&lt;em&gt;N&lt;/em&gt; = 16), &lt;em&gt;Mannheimia haemolytica&lt;/em&gt; (&lt;em&gt;N&lt;/em&gt; = 29), and &lt;em&gt;Pasteurella multocida&lt;/em&gt; (&lt;em&gt;N&lt;/em&gt; = 35), as well as available reference sequences. +, present; &#x2212;, absent.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/9/3059/F2.medium.gif" width="440" height="267"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/7/9/3059/F2.medium.gif" width="440" height="267"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/9/3059/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/9/3059/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/486428" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-39" class="first-child">Prevalence of resistance genes among isolates of <em>Histophilus somni</em> (<em>N</em> = 16), <em>Mannheimia haemolytica</em> (<em>N</em> = 29), and <em>Pasteurella multocida</em> (<em>N</em> = 35), as well as available reference sequences. +, present; −, absent.</p><div class="sb-div caption-clear"></div></div></div><div id="sec-15" class="subsection"><h4>H. somni ARG:</h4><p id="p-40"><em>In silico</em> ARG identification was performed for the two <em>H. somni</em> reference sequences 2336 and 129PT (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>). No ARG were predicted in reference sequence 129PT and only <em>tetH</em> was predicted in reference sequence 2336. Nine ARG were detected in the <em>H. somni</em> isolates (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.1137/-/DC1/FigureS2.tiff">Figure S2</a>). Two aminoglycoside resistance genes that confer resistance to the drugs KAN and NEO (<em>aph3-Ia</em>) and streptomycin (<em>strA</em>, <em>strB</em>) were detected. Two macrolide-lincosamide-streptogramin B (MLS<sub>b</sub>) resistance genes (<em>erm42</em> and <em>ermF</em>) were also detected, as was a phenicol resistance gene (<em>floR</em>), a sulfonamide resistance gene (<em>sul2</em>), a trimethoprim resistance gene (<em>dfrA14</em>), and a tetracycline resistance gene (<em>tetH</em>) (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a>). There was a 79.2% concordance rate between genes identified using raw reads and <em>de novo</em> assembled contigs.</p></div><div id="sec-16" class="subsection"><h4>M. haemolytica ARG:</h4><p id="p-41">Different ARG were identified across <em>M. haemolytica</em> reference sequences. The genes <em>aph3-Ia</em>, <em>strB</em>, <em>sul2</em>, <em>tetH</em> and the β-lactamase gene, <em>ROB-1</em>, were detected in reference sequence M42548; <em>ROB-1</em> was detected in reference sequence 89010807N_lktA; <em>tetH</em> was detected in reference sequence 89010807N_lktA and reference sequence 89010807N; no ARG were detected in any of the other available reference sequences (<em>n</em> = 7, <a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2</a>). Seven of the eight ARG detected in <em>M. haemolytica</em> isolates (<em>aph3-Ia</em>, <em>erm42</em>, <em>floR</em>, <em>strA</em>, <em>strB</em>, <em>sul2</em>, and <em>tetH</em>) were also found in <em>H. somni</em> and <em>P. multocida</em> (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2</a>). No ARG were detected in the only <em>M. haemolytica</em> sample sequenced from BRD-affected beef cattle (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.1137/-/DC1/FigureS3.tiff">Figure S3</a>). Two of the genes (<em>erm42</em> and <em>floR</em>) were not detected in any reference sequence but have been previously reported (<a id="xref-ref-13-5" class="xref-bibr" href="#ref-13">Clawson <em>et al.</em> 2016</a>). There was an 89.3% concordance rate between genes identified using raw reads and <em>de novo</em> assembled contigs. There was complete linkage of ARG <em>aph3-Ia</em>, <em>strA</em> and <em>strB</em> in 28 out of 29 samples, showing homology to the conserved backbone of ICE<em>Mh1</em> and ICE<em>Pmu1</em> (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.1137/-/DC1/FigureS3.tiff">Figure S3</a>). The presence of the conserved ICE backbone has been associated with samples classified as genotype 2. Here, 21 out of 22 samples classified as genotype 2 contained regions homologous to the ICE backbone and all seven samples classified as genotype 1 also contained regions homologous to the ICE backbone. Additionally, the phenicol resistance gene <em>cat2</em> was identified in six out of 29 samples. Six of the seven samples classified as genotype 1 contained the <em>cat2</em> ARG, while the other 22 samples, classified as genotype 2, did not contain the <em>cat2</em> ARG (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.1137/-/DC1/FigureS3.tiff">Figure S3</a>).</p></div><div id="sec-17" class="subsection"><h4>P. multocida ARG:</h4><p id="p-42">The available <em>P. multocida</em> reference sequences had similar ARG variation to that seen in the <em>M. haemolytica</em> reference sequences. All seven of the ARG (<em>tetH</em>, <em>aph3-Ia</em>, <em>erm42</em>, <em>strA</em>, <em>strB</em>, <em>floR</em>, and <em>sul2</em>) detected in the <em>P. multocida</em> samples were also detected in reference sequence 36950 (<a id="xref-ref-58-2" class="xref-bibr" href="#ref-58">Michael <em>et al.</em> 2011</a>). The gene <em>tetB</em> was detected in reference sequence HB01, but no ARG were detected in any of the other nine reference sequences (<a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2</a>). Seven of the 11 ARG were detected in the <em>P. multocida</em> samples originating from dairy cattle (<a id="xref-fig-2-7" class="xref-fig" href="#F2">Figure 2</a>). Sequences from two samples obtained from beef cattle contained four and five of seven ARG, <em>tetH</em>, <em>aph3-Ia</em>, <em>strA</em>, <em>strB</em> and <em>tetH</em>, <em>aph3-Ia</em>, <em>erm42</em>, <em>strA</em>, <em>strB</em>, respectively, but there were no ARG detected in the other four samples obtained from beef cattle (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.1137/-/DC1/FigureS4.tiff">Figure S4</a>). One of the 35 samples contained the β-lactamase gene, <em>ROB-1</em>. There was a 76.5% concordance rate between genes identified using raw reads and <em>de novo</em> assembled contigs. Like <em>M. haemolytica</em> samples, there was complete linkage of ARG <em>aph3-Ia</em>, <em>strA</em> and <em>strB</em> in 31 out of 35 samples, showing homology to the conserved backbone of ICE<em>Mh1</em> and ICE<em>Pmu1</em>.</p></div></div><div id="sec-18" class="subsection"><h3>Phenotypic susceptibility testing results</h3><p id="p-43">All 64 isolates displayed phenotypic resistance to NEO at a resistant breakpoint of ≥32 μg/ml (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.1137/-/DC1/FileS1.xlsx">File S1</a> and <a id="xref-table-wrap-6-1" class="xref-table" href="#T6">Table 6</a>). All 25 of the <em>P. multocida</em> isolates were also resistant to TYL, OXY, SUL, and CLIN, while all 26 of the <em>M. haemolytica</em> were resistant to OXY (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). None of the isolates displayed phenotypic resistance to SPEC or ENRO, and a single <em>H. somni</em> isolate was resistant to CEFT (<a id="xref-table-wrap-6-2" class="xref-table" href="#T6">Table 6</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.1137/-/DC1/TableS3.xlsx">Table S3</a>). <em>M. haemolytica</em> was the only organism that displayed resistance to TUL (five out of 26 isolates). None of the isolates from any species were susceptible to all drugs tested; 61 were MDR (defined as exhibiting phenotypic resistance to more than two antimicrobial classes tested) and the remaining three were resistant to only two classes of antimicrobials (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.1137/-/DC1/FigureS5.tiff">Figure S5</a>).</p><div id="T6" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/486508/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/486508/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 6</span> <span class="caption-title">Concordance of phenotypic and susceptibility results for all three organisms (64 isolates)</span><div class="sb-div caption-clear"></div></div></div><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/9/3059/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Prevalence of phenotypic resistance to all drugs tested among isolates of Histophilus somni (N = 13), Mannheimia haemolytica (N = 26), and Pasteurella multocida (N = 25)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-43988265" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Prevalence of phenotypic resistance to all drugs tested among isolates of &lt;em&gt;Histophilus somni&lt;/em&gt; (&lt;em&gt;N&lt;/em&gt; = 13), &lt;em&gt;Mannheimia haemolytica&lt;/em&gt; (&lt;em&gt;N&lt;/em&gt; = 26), and &lt;em&gt;Pasteurella multocida&lt;/em&gt; (&lt;em&gt;N&lt;/em&gt; = 25).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/9/3059/F3.medium.gif" width="440" height="251"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/7/9/3059/F3.medium.gif" width="440" height="251"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/9/3059/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/9/3059/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/486523" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-45" class="first-child">Prevalence of phenotypic resistance to all drugs tested among isolates of <em>Histophilus somni</em> (<em>N</em> = 13), <em>Mannheimia haemolytica</em> (<em>N</em> = 26), and <em>Pasteurella multocida</em> (<em>N</em> = 25).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-19" class="subsection"><h3>Genotype–phenotype concordance</h3><p id="p-46">Concordance rates for antimicrobial resistance genotype and phenotype varied widely for each antimicrobial drug (<a id="xref-table-wrap-6-3" class="xref-table" href="#T6">Table 6</a>). Across all three organisms, ENRO and NEO exhibited 100% concordance, with all isolates displaying phenotypic susceptibility and none containing genes typically associated with resistance to these antimicrobials, <em>i.e.</em>, adenylyltransferases AAD(3″) and AAD(9) for SPEC and <em>parC</em>, <em>parE</em>, <em>gyrA</em>, and <em>gyrB</em> for ENRO. As with NEO, SPEC also exhibited high genotype–phenotype concordance (98%), but in contrast with NEO, this concordance was driven by presence of <em>aph3-Ia</em> and corresponding phenotypic resistance. Concordance to OXY was also very high across all three organisms; of the 64 isolates tested, 62 displayed phenotypic resistance to OXY and contained the <em>tetH</em> resistance gene. All three organisms exhibited statistically significant phenotype–genotype discordance for CHLOR and TIL (<em>P</em> &lt; 0.05; <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.1137/-/DC1/TableS3.xlsx">Table S3</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.1137/-/DC1/TableS4.xlsx">Table S4</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.1137/-/DC1/TableS5.xlsx">Table S5</a>). CHLOR discordance was driven by phenotypic susceptibility in the presence of <em>tetH</em> (opposite the results for OXY). For <em>H. somni</em> and <em>P. multocida</em>, TIL discordance resulted from phenotypic susceptibility in the presence of <em>erm</em>F and/or <em>erm</em>42. Interestingly, <em>erm42</em> has previously been identified in MDR <em>P. multocida</em> isolates from cases of BRD, but has yet to be described in <em>H. somni</em> (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Kadlec <em>et al.</em> 2011</a>). Unlike TIL discordance in <em>H. somni</em> and <em>P. multocida</em>, discordance in <em>M. haemolytica</em> was due largely to phenotypic resistance without identification of a known macrolide resistance gene.</p><p id="p-47"><em>H. somni</em> and <em>M. haemolytica</em> demonstrated significant discordance for CLIN, however this discordance was driven by different dynamics (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.1137/-/DC1/TableS3.xlsx">Table S3</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.1137/-/DC1/TableS4.xlsx">Table S4</a>). Within <em>H. somni</em>, discordance resulted from phenotypic susceptibility in the presence of both <em>erm</em>42 and <em>erm</em>F, whereas phenotypic resistance and a lack of any known macrolide resistance genes drove the discordance in <em>M. haemolytica</em>.</p><p id="p-48"><em>H. somni</em> and <em>P. multocida</em> exhibited significant discordance for FLOR, which in both organisms resulted from phenotypic susceptibility despite presence of the <em>floR</em> resistance gene. In addition, TUL was significantly discordant in both <em>H. somni</em> and <em>P. multocida</em> (<em>P</em> &lt; 0.05; <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.1137/-/DC1/TableS3.xlsx">Table S3</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.1137/-/DC1/TableS4.xlsx">Table S4</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.1137/-/DC1/TableS5.xlsx">Table S5</a>), in both cases due to presence of <em>erm42</em> and/or <em>ermF</em> without corresponding phenotypic resistance.</p><p id="p-49">Phenotype and genotype results for DAN and TMS were significantly discordant in both <em>M. haemolytica</em> and <em>P. multocida</em> (<em>P</em> &lt; 0.05; <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.1137/-/DC1/TableS4.xlsx">Table S4</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.1137/-/DC1/TableS5.xlsx">Table S5</a>). For DAN, this discordance was due to phenotypic resistance without identification of a known danofloxacin resistance gene. Conversely, TMS discordance was driven by phenotypic susceptibility despite presence of the <em>sul2</em> gene.</p><p id="p-50">In addition, <em>H. somni</em> exhibited significant discordance for GENT, with 10 out of 13 isolates displaying phenotypic resistance but none containing a resistance gene known to confer resistance to this drug. All 10 of these isolates contained <em>aph3-Ia</em>, but this gene is typically associated with resistance to the aminoglycosides kanamycin, NEO, and paromomycin within <em>Enterobacteriaceae</em> spp. (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Livermore <em>et al.</em> 2001</a>). The breakpoint used for GENT was developed for <em>Pseudomonas aeruginosa</em> and <em>Enterobacteriaceae</em> in canines and equids, and <em>Actinobacillus</em> spp. in equids (<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">CLSI 2013</a>), and it is unknown whether these breakpoints are appropriate for <em>H. somni</em> in cattle.</p><p id="p-51"><em>M. haemolytica</em> isolates also displayed statistically significant discordance for both CEFT and TYL (<em>P</em> &lt; 0.05; <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.1137/-/DC1/TableS4.xlsx">Table S4</a>). In the case of TYL, 25 of the 26 isolates were phenotypically resistant, but only six of these 25 isolates contained <em>erm42</em>. CEFT discordance resulted from the fact that none of the isolates whose genomes contained <em>ROB-1</em> experienced CEFT resistance due to the presence of this β-lactamase gene.</p><p id="p-52"><em>P. multocida</em> isolates were also significantly discordant for the pleuromutilin TIA (<em>P</em> &lt; 0.05; <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.1137/-/DC1/TableS5.xlsx">Table S5</a>), with 21 out of 25 isolates displaying phenotypic resistance despite lack of an identified pleuromutilin resistance gene within the isolates’ genomes.</p><p id="p-53">Macrolide resistance is of particular concern, as BRD is typically treated with either tetracyclines or macrolides. We identified two MLS<sub>b</sub> resistance genes within the 64 samples, <em>erm42</em> and <em>ermF</em>. The genotype–phenotype concordance results differed greatly by the combination of gene, bacteria, and macrolide drug in question, as illustrated in <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>. When averaged across all three bacterial species, the genotype–phenotype discordance rates were statistically significant for all four macrolides tested (<a id="xref-table-wrap-6-4" class="xref-table" href="#T6">Table 6</a>). However, discordance rates and significance varied by organism (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.1137/-/DC1/TableS3.xlsx">Table S3</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.1137/-/DC1/TableS4.xlsx">Table S4</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.1137/-/DC1/TableS5.xlsx">Table S5</a>). Additionally, there was not a strong correlation between the number of ARG detected in each sample and the number of antimicrobials that each sample was resistant to (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.1137/-/DC1/FigureS6.tiff">Figure S6</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/9/3059/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Heatmap showing concordance (blue) and discordance (red) between erm42 and ermF presence (black) or absence (gray) and phenotypic resistance to the four macrolide drugs included in the susceptibility testing panel (columns), by isolate (rows) within organism. CLIN, clindamycin; TIL, tilmicosin; TUL, tulathromycin; TYL, tylosin tartrate." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-43988265" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Heatmap showing concordance (blue) and discordance (red) between &lt;em&gt;erm42&lt;/em&gt; and &lt;em&gt;ermF&lt;/em&gt; presence (black) or absence (gray) and phenotypic resistance to the four macrolide drugs included in the susceptibility testing panel (columns), by isolate (rows) within organism. CLIN, clindamycin; TIL, tilmicosin; TUL, tulathromycin; TYL, tylosin tartrate.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/9/3059/F4.medium.gif" width="440" height="257"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/7/9/3059/F4.medium.gif" width="440" height="257"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/9/3059/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/9/3059/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/486561" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-54" class="first-child">Heatmap showing concordance (blue) and discordance (red) between <em>erm42</em> and <em>ermF</em> presence (black) or absence (gray) and phenotypic resistance to the four macrolide drugs included in the susceptibility testing panel (columns), by isolate (rows) within organism. CLIN, clindamycin; TIL, tilmicosin; TUL, tulathromycin; TYL, tylosin tartrate.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-20"><h2 class="">Discussion</h2><div id="sec-21" class="subsection"><h3>Genome assembly and diversity</h3><p id="p-55">Additional WGS from BRD pathogens are needed to help understand genetic diversity and its role in pathogenesis, genotype–phenotype associations, and the relationship between genomic features and clinical disease. This study doubled the number of available genomic sequences for these four species. In addition to expanding current WGS databases, standardization of genome analysis methods for genomic epidemiological interpretation is important (<a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Gilmour <em>et al.</em> 2013</a>). As we have shown in this study, results obtained from <em>de novo</em> assemblies <em>vs.</em> consensus sequences constructed from mapping to reference genomes can be highly variable. This is demonstrated not only by the differences in genetic distances for each consensus sequence called when mapped to their respective reference sequences (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.1137/-/DC1/FigureS1.tiff">Figure S1</a>), but also by variability in ARG (<a id="xref-fig-2-8" class="xref-fig" href="#F2">Figure 2</a>). However, there are many benefits to using a reference-based mapping approach, particularly when many samples need to be sequenced and analyzed. One such advantage is the computing power needed for consensus mapping is much lower as compared to <em>de novo</em> assembly and phylogenomic analyses using mapped assemblies yield similar results to those produced by <em>de novo</em> assemblies (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Fitz-Gibbon <em>et al.</em> 2017</a>). Genome assembly via mapping of reads to reference sequences enabled us to efficiently characterize variation within our study population, even if there is a slight bias toward the reference sequences (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.1137/-/DC1/FigureS1.tiff">Figure S1</a>).</p><p id="p-56">Little variation was detected between samples mapped to each of the 10 <em>M. haemolytica</em> reference sequences. This is likely due to similarity in strains from cattle across the Midwest and California, as nine of the 10 reference sequences were obtained from cattle in the Midwest. Similarly, <em>P. multocida</em> samples clustered much more closely to reference sequences 3480, HN06, OH1905, OH4807, PMTB2.1, and Pm70, as each of these samples were obtained from cattle in California or the Midwest. In contrast, <em>P. multocida</em> samples in our study clustered more distantly from reference sequences 36950, ATCC_43137, HB01, HB03, and Pm-3, which were obtained from cattle in Germany or China. In this dataset, there was a clear distinction between <em>P. multocida</em> isolates collected from the beef and dairy cattle populations with the exception of one sample. However, it is important to note that only six of the 35 <em>P. multocida</em> samples analyzed were obtained from BRD-affected beef cattle. Further investigations with larger sample sizes from broader geographical ranges will be required to validate these observations.</p></div><div id="sec-22" class="subsection"><h3>Predicting antimicrobial resistance</h3><p id="p-57">The ability to predict ARG <em>in silico</em> would have a significant impact on our ability to diagnose and treat pathogenic diseases and monitor the spread of ARG (<a id="xref-ref-47-3" class="xref-bibr" href="#ref-47">Koser <em>et al.</em> 2014</a>). In a study of animals previously treated with antimicrobials, resistance was identified in 72% of <em>M. haemolytica</em> and 50% of <em>P. multocida</em> isolates, with 13 ARG present in <em>H. somni</em> (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Klima <em>et al.</em> 2014</a>). Other studies in healthy cattle suggest that the prevalence of antimicrobial resistance in BRD-associated bacteria may be relatively low (<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Noyes <em>et al.</em> 2015</a>). <em>M. haemolytica</em> and <em>P. multocida</em> harbor integrative and conjugative elements that enable the transfer of gene blocks between the species (<a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Klima <em>et al.</em> 2014</a>) and confer resistance to antimicrobials (<a id="xref-ref-58-3" class="xref-bibr" href="#ref-58">Michael <em>et al.</em> 2011</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Eidam <em>et al.</em> 2015</a>). <em>M. bovis</em> lacks a cell wall and is resistant to drugs that target the cell wall of Gram-negative bacteria; to date no ARG have been found in any strain of <em>M. bovis</em> (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Maunsell <em>et al.</em> 2011</a>).</p><p id="p-58">In this study, we detected 11 antibiotic resistance genes within the sample population, including three aminoglycoside resistance genes, two macrolide-lincosamide-streptogramin resistance genes, two phenicol and one each of β-lactam, sulfonamide, fluoroquinolone, tetracycline, and trimethoprim resistance genes. Tetracyclines and macrolides are often used as first-line agents in the treatment of BRD. The prevalence of <em>tetH</em> and <em>erm42</em> were relatively high in this population, and the presence of these ARG within BRD-associated bacterial isolates could render BRD treatment more challenging (<a id="xref-ref-52-2" class="xref-bibr" href="#ref-52">Livermore <em>et al.</em> 2001</a>). However, prevalence of resistance to antimicrobials typically used as second- or third-line treatments was relatively low, <em>i.e.</em>, <em>floR</em> and <em>cat2</em> for florfenicol. As a result, several alternative, commonly used antimicrobials for BRD treatment (<a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">USDA 2011b</a>) could still be effective based on our findings.</p><p id="p-59">Phenotypic antimicrobial resistance and the presence of ARG have both been demonstrated for <em>H. somni</em>, <em>M. haemolytica</em>, and <em>P. multocida</em> (<a id="xref-ref-42-3" class="xref-bibr" href="#ref-42">Klima <em>et al.</em> 2014</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">DeDonder and Apley 2015</a>). It is known that antimicrobials that target the cell wall, such as β-lactams, are not effective against <em>M. bovis</em> due to the lack of a cell wall. While we did not identify ARG in any of the <em>M. bovis</em> genomes in this study, phenotypic resistance within this organism has previously been demonstrated (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Kawai <em>et al.</em> 2014</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Kong <em>et al.</em> 2015</a>). Each of the other three species has shown resistance to tetracyclines (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Kehrenberg <em>et al.</em> 2001</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">D’Amours <em>et al.</em> 2011</a>), which is the primary resistance gene found in <em>H. somni</em> samples in previous studies, but the majority of resistance genes previously found have been in <em>P. multocida</em> and <em>M. haemolytica</em>.</p><p id="p-60">Previous studies have shown macrolide-lincosamide-streptogramin, kanamycin, NEO, and streptomycin resistance in <em>H. somni</em> samples (<a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">DeDonder and Apley 2015</a>), but there were no ARG found in <em>H. somni</em> reference sequences other than <em>tetH</em>. However, recent studies have shown the presence of seven out of nine ARG found in <em>H. somni</em> samples during this study (<a id="xref-ref-42-4" class="xref-bibr" href="#ref-42">Klima <em>et al.</em> 2014</a>), showing the variation of ARG found in different strains. Recent studies showing the presence of known ARG, as well as phenotypic antimicrobial resistance, have primarily been done in diseased animals from feedlots around the Midwest, while the reference sequence samples were obtained from diseased animals in California. Our study shows a similarity in ARG to the recent study of samples obtained from feedlots in Texas, Nebraska, and Alberta, Canada (<a id="xref-ref-42-5" class="xref-bibr" href="#ref-42">Klima <em>et al.</em> 2014</a>). Importantly, eight of nine ARG found in <em>H. somni</em> and three of eight ARG found in <em>M. haemolytica</em> were not found in any of the previously available reference genome sequences.</p><p id="p-61">It is important to note that there was slightly more ARG detected in each sample using raw reads compared to ARG detected using <em>de novo</em> assembled contigs. Using raw reads for gene detection has been shown to be superior to assembly-based approaches. This is primarily because assembly-based approaches are dependent on the quality of the sequencing, the assembly method used, and the quality of the assembly itself (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Clausen <em>et al.</em> 2016</a>).</p><p id="p-62">The grouping of ARG within each sample is not surprising, as these genes would likely increase the survival rate of the bacteria, thus representing a substantial fitness benefit and eventually becoming fixed in the population (<a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Griffin <em>et al.</em> 2010</a>). The fact that three out of four species of bacteria associated with BRD share many of the same ARG is likewise unsurprising, as infection is highly associated with the presence of multiple pathogens (<a id="xref-ref-27-3" class="xref-bibr" href="#ref-27">Griffin <em>et al.</em> 2010</a>). Additionally, previous studies have shown there is a high incidence of horizontal gene transfer of ARG-containing plasmids between <em>M. haemolytica</em> and <em>P. multocida</em> (<a id="xref-ref-42-6" class="xref-bibr" href="#ref-42">Klima <em>et al.</em> 2014</a>), and the presence of the integrative and conjugative elements found in these two species have been associated with isolates recovered from BRD-affected cattle (<a id="xref-ref-62-2" class="xref-bibr" href="#ref-62">Neibergs <em>et al.</em> 2014</a>). Here, we show that 97% of <em>M. haemolytica</em> samples and 86% <em>P. multocida</em> samples obtained from BRD-affected cattle contain regions homologous to the ICE backbone. It was interesting that no ARG were detected in five of the seven samples collected from untreated beef cattle, other than the two <em>P. multocida</em> samples. While both <em>M. haemolytica</em> and <em>P. multocida</em> are known to exist commensally in the respiratory tract of animals not affected by BRD (<a id="xref-ref-62-3" class="xref-bibr" href="#ref-62">Neibergs <em>et al.</em> 2014</a>), the significance of these findings needs to be substantiated by a larger number of more diverse samples.</p><p id="p-63">While <em>in silico</em> analysis can be used to identify ARG within WGS data, it does not reveal the functionality or effectiveness of the ARG <em>in vivo</em>. Therefore, antimicrobial susceptibility testing should be performed alongside <em>in silico</em> methods in order to understand the expression of these ARG, at least within laboratory testing conditions. In addition, extensive metadata should be collected for each sample, including the eventual clinical outcome for the animal from which the sample was taken. These types of data would help us to understand how knowledge of the presence of ARG within a given genome could potentially predict treatment and morbidity/mortality outcomes.</p></div><div id="sec-23" class="subsection"><h3>Genotype–phenotype concordance</h3><p id="p-64">The genotype–phenotype results from this study demonstrate the high degree of variability in concordance rates between different drugs, genes, and organisms. The ability of antimicrobial resistance genotype to predict phenotypic susceptibility has been under much discussion recently, particularly in the context of increased use of WGS as part of national food safety monitoring systems (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Allard <em>et al.</em> 2016</a>; Burrell <em>et al.</em> 2016). Some of the discordance rates presented in this study were higher than rates previously reported (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Card <em>et al.</em> 2013</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Agyekum <em>et al.</em> 2016</a>). However, the available literature does not include results for the three organisms included in this study, which suggests that genotype–phenotype concordance rates may be highly species-specific. Alternatively, the breakpoints that we utilized for this study may not have been appropriate for <em>H. somni</em>, <em>M. haemolytica</em>, and <em>P. multocida</em>, especially since many of the breakpoints were not formulated for the relevant drug-bacterium-host combinations (<a id="xref-table-wrap-4-5" class="xref-table" href="#T4">Table 4</a>). It has been shown that when changing the specific parameters of drug-bacterium-host, the CLSI-approved breakpoints become invalid and in cases were phenotypic resistance was seen, but no known ARG was detected, it becomes difficult to discern actual phenotypic resistance from inappropriate reporting (<a id="xref-ref-18-3" class="xref-bibr" href="#ref-18">DeDonder and Apley 2015</a>). However, even among breakpoints that were relevant, we found varying levels of genotype–phenotype concordance. Host- and organism-appropriate breakpoints were used for ENRO, TUL, FLOR, CHLOR, OXY, CEFT, and SPEC. While concordance rates were &gt;98% for ENRO, OXY, and SPEC, there was significant discordance for CEFT, CHLOR, FLOR, and TUL (<em>P</em> &lt; 0.05; <a id="xref-table-wrap-6-5" class="xref-table" href="#T6">Table 6</a>). Therefore, issues concerning breakpoints do not fully account for the relatively high rates of genotype–phenotype discordance observed.</p><p id="p-65">In many cases, the presence of a resistance gene correlated well with certain drugs within the relevant antimicrobial class, but not others. For instance, within <em>M. haemolytica</em> isolates the presence of the <em>ROB-1</em> gene correlated well with phenotypic resistance to AMP and PEN, but not CEFT, to which there was no phenotypic resistance. Therefore, these results suggest that <em>ROB-1</em> may not cause resistance to third-generation cephalosporins in <em>M. haemolytica</em>; a similar finding has been demonstrated with <em>ROB-1</em> in Canadian <em>Haemophilus influenzae</em> human isolates (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Karlowsky <em>et al.</em> 2000</a>). Phenotypic NEO resistance was present across all 64 isolates, and 63 of the isolates also contained <em>aph3-Ia</em>, suggesting that this gene may be a very good predictor of neomycin resistance in <em>H. somni</em>, <em>M. haemolytica</em>, and <em>P. multocida</em>.</p><p id="p-66">Resistance to TYL was nearly 100% within <em>M. haemolytica</em> and <em>P. multocida</em> isolates. However, presence of the <em>erm42</em> gene was only predictive of resistance in <em>P. multocida</em> (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>). Therefore, additional unrecognized mechanisms or genes are likely responsible for resistance in the other two species, although inappropriate breakpoints cannot be discounted. In contrast, the presence of <em>erm42</em> predicted TUL resistance well within <em>M. haemolytica</em>, in that all five resistant isolates contained <em>erm42</em> and none of the susceptible isolates did. However, the presence of <em>erm42</em> in all 25 <em>P. multocida</em> isolates did not result in TUL resistance (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4</a>). These results highlight the importance of considering the specific antimicrobial drug, resistance gene, organism, and host when investigating the ability of genotype to predict phenotype (or vice versa).</p><p id="p-67">Results of our study demonstrate the complexity of genome-wide dynamics in mediation of resistance, and emphasize the fact that simple presence/absence of individual genes does not always predict the behavior of a given organism under varying levels of antimicrobial drug exposures. More research is needed to develop reliable and accurate genomic predictors of antimicrobial resistance, and this research will likely need to be targeted toward specific drug-bacterium-host combinations. Until such genomic predictors are identified and refined, the scientific, medical, public health, and regulatory communities should be cognizant of the potential limitations of relying solely on genomic data to guide treatment and programmatic or regulatory decisions. Such a conservative approach is especially relevant to the veterinary community, where WGS of pathogens with high animal health importance has been limited.</p><p id="p-68">The sequences produced as part of this work represent a first step toward increased understanding of genome-mediated antimicrobial resistance dynamics within these important cattle pathogens. However, the variability in our results highlights the continuing criticality of collecting pertinent host, production, and health outcome information (<em>i.e.</em>, metadata) for each isolate. Without such information, the scientific community will be unable to evaluate the implications of positive identification of resistance genes within WGS data, and improve BRD morbidity/mortality outcomes by predicting the most effective treatment regime for BRD based on the specific gene-bacterium-host combination.</p></div></div><div class="section" id="sec-24"></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-69">Library preparation was performed at the University of California Davis DNA Technologies Core Facility. The authors would like to thank the University of California Davis Bioinformatics Core Facility for assistance with data analysis. The authors would also like to thank Paul Rossitto, James Moller, Bryan Welly, Ellen Lai, and Nadia Torok for assistance with sample collection, and Julie Cleek, Toni Ortiz, and Muhammad Ilyas for bacterial work-up and biochemical identification. Dairy cattle samples were collected as part of the US Department of Agriculture, Agriculture and Food Research Initiative competitive grant no. 2011-68004-30367. The sequencing and antimicrobial resistance phenotyping was funded by a grant from the Russell L. Rustici Rangeland and Cattle Research Endowment. </p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-14"><p id="p-70">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.1137/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.1137/-/DC1</a>.</p></li><li class="fn" id="fn-15"><p id="p-71"><em>Communicating editor: M. Watson</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2017-06-15"><span class="received-label">Received </span>June 15, 2017.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2017-07-10"><span class="accepted-label">Accepted </span>July 10, 2017.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2017 Owen <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3059.1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Agyekum</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fajardo-Lubian</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ai</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ginn</span>  <span class="cit-name-given-names">A. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zong</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Predictability of phenotype in relation to common betalactam resistance mechanisms in Escherichia coli and <em>Klebsiella pneumoniae</em>.</span> <abbr class="cit-jnl-abbrev">J. Clin. Microbiol.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">1243</span>–<span class="cit-lpage">1250</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BClin.%2BMicrobiol.%26rft.volume%253D15%26rft.spage%253D1243%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3059.2" data-doi="10.1016/j.vetmic.2008.01.001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Alexander</span>  <span class="cit-name-given-names">T. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cook</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yanke</span>  <span class="cit-name-given-names">L. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">A multiplex polymerase chain reaction assay for the identification of <em>Mannheimia haemolytica</em>, <em>Mannheimia glucosida</em> and <em>Mannheimia ruminalis</em>.</span> <abbr class="cit-jnl-abbrev">Vet. Microbiol.</abbr> <span class="cit-vol">130</span>: <span class="cit-fpage">165</span>–<span class="cit-lpage">175</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVeterinary%2Bmicrobiology%26rft.stitle%253DVet%2BMicrobiol%26rft.aulast%253DAlexander%26rft.auinit1%253DT.%2BW.%26rft.volume%253D130%26rft.issue%253D1-2%26rft.spage%253D165%26rft.epage%253D175%26rft.atitle%253DA%2Bmultiplex%2Bpolymerase%2Bchain%2Breaction%2Bassay%2Bfor%2Bthe%2Bidentification%2Bof%2BMannheimia%2Bhaemolytica%252C%2BMannheimia%2Bglucosida%2Band%2BMannheimia%2Bruminalis.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.vetmic.2008.01.001%26rft_id%253Dinfo%253Apmid%252F18308486%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.vetmic.2008.01.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18308486&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3059.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3059.3" data-doi="10.1128/JCM.00081-16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Allard</span>  <span class="cit-name-given-names">M. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Strain</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Melka</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bunning</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Musser</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Practical value of food pathogen traceability through building a whole-genome sequencing network and database.</span> <abbr class="cit-jnl-abbrev">J. Clin. Microbiol.</abbr> <span class="cit-vol">54</span>: <span class="cit-fpage">1975</span>–<span class="cit-lpage">1983</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BClin.%2BMicrobiol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJCM.00081-16%26rft_id%253Dinfo%253Apmid%252F27008877%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamNtIjtzOjU6InJlc2lkIjtzOjk6IjU0LzgvMTk3NSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvOS8zMDU5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3059.4" data-doi="10.1006/jmbi.1990.9999"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Altschul</span>  <span class="cit-name-given-names">S. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gish</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lipman</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">Basic local alignment search tool.</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">215</span>: <span class="cit-fpage">403</span>–<span class="cit-lpage">410</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DAltschul%26rft.auinit1%253DS.%2BF.%26rft.volume%253D215%26rft.issue%253D3%26rft.spage%253D403%26rft.epage%253D410%26rft.atitle%253DBasic%2Blocal%2Balignment%2Bsearch%2Btool.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fjmbi.1990.9999%26rft_id%253Dinfo%253Apmid%252F2231712%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/jmbi.1990.9999&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2231712&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3059.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1990ED16700008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3059.5" data-doi="10.1089/cmb.2012.0021"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bankevich</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nurk</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Antipov</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gurevich</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dvorkin</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">SPAdes: a new genome assembly algorithm and its applications to single-cell sequencing.</span> <abbr class="cit-jnl-abbrev">J. Comput. Biol.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">455</span>–<span class="cit-lpage">477</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bcomputational%2Bbiology%2B%253A%2B%2Ba%2Bjournal%2Bof%2Bcomputational%2Bmolecular%2Bcell%2Bbiology%26rft.stitle%253DJ%2BComput%2BBiol%26rft.aulast%253DBankevich%26rft.auinit1%253DA.%26rft.volume%253D19%26rft.issue%253D5%26rft.spage%253D455%26rft.epage%253D477%26rft.atitle%253DSPAdes%253A%2Ba%2Bnew%2Bgenome%2Bassembly%2Balgorithm%2Band%2Bits%2Bapplications%2Bto%2Bsingle-cell%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1089%252Fcmb.2012.0021%26rft_id%253Dinfo%253Apmid%252F22506599%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1089/cmb.2012.0021&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22506599&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3059.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3059.6" data-doi="10.2807/1560-7917.ES.2015.20.47.30073"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Berenger</span>  <span class="cit-name-given-names">B. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berry</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peterson</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fach</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Delannoy</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">The utility of multiple molecular methods including whole genome sequencing as tools to differentiate <em>Escherichia coli</em> O157:H7 outbreaks.</span> <abbr class="cit-jnl-abbrev">Euro Surveill.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">30073</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEuro%2BSurveill.%26rft.volume%253D20%26rft.spage%253D30073%26rft_id%253Dinfo%253Adoi%252F10.2807%252F1560-7917.ES.2015.20.47.30073%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2807/1560-7917.ES.2015.20.47.30073&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-7.9.3059.7" data-doi="10.1371/journal.pone.0150797"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Burall</span>  <span class="cit-name-given-names">L. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grim</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mammel</span>  <span class="cit-name-given-names">M. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Datta</span>  <span class="cit-name-given-names">A. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Whole genome sequence analysis using JSpecies tool establishes clonal relationships between Listeria monocytogenes strains from epidemiologically unrelated Listeriosis outbreaks.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">e0150797</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D11%26rft.spage%253De0150797%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0150797%26rft_id%253Dinfo%253Apmid%252F26950338%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0150797&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26950338&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3059.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-7.9.3059.8" data-doi="10.1128/AAC.02412-14"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Carattoli</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zankari</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">García-Fernández</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Voldby</span>  <span class="cit-name-given-names">L. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lund</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">In silico detection and typing of plasmids using PlasmidFinder and plasmid multilocus sequence typing.</span> <abbr class="cit-jnl-abbrev">Antimicrob. Agents Chemother.</abbr> <span class="cit-vol">58</span>: <span class="cit-fpage">3895</span>–<span class="cit-lpage">3903</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAntimicrob.%2BAgents%2BChemother.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FAAC.02412-14%26rft_id%253Dinfo%253Apmid%252F24777092%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiYWFjIjtzOjU6InJlc2lkIjtzOjk6IjU4LzcvMzg5NSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvOS8zMDU5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3059.9" data-doi="10.1128/AAC.01223-12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Card</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Das</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cook</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Woodford</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Evaluation of an expanded microarray for detecting antimicrobial resistance genes in a broad range of gram-negative bacterial pathogens.</span> <abbr class="cit-jnl-abbrev">Antimicrob. Agents Chemother.</abbr> <span class="cit-vol">57</span>: <span class="cit-fpage">458</span>–<span class="cit-lpage">465</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAntimicrob.%2BAgents%2BChemother.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FAAC.01223-12%26rft_id%253Dinfo%253Apmid%252F23129055%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiYWFjIjtzOjU6InJlc2lkIjtzOjg6IjU3LzEvNDU4IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy85LzMwNTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-7.9.3059.10" data-doi="10.1128/JB.01422-06"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Challacombe</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Duncan</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brettin</span>  <span class="cit-name-given-names">T. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bruce</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chertkov</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Complete genome sequence of <em>Haemophilus somnus</em> (<em>Histophilus somni</em>) strain 129Pt and comparison to <em>Haemophilus ducreyi</em> 35000HP and <em>Haemophilus influenzae</em> Rd.</span> <abbr class="cit-jnl-abbrev">J. Bacteriol.</abbr> <span class="cit-vol">189</span>: <span class="cit-fpage">1890</span>–<span class="cit-lpage">1898</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBacteriol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJB.01422-06%26rft_id%253Dinfo%253Apmid%252F17172329%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6MTA6IjE4OS81LzE4OTAiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzkvMzA1OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3059.11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chirase</span>  <span class="cit-name-given-names">N. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Green</span>  <span class="cit-name-given-names">L. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Influence of oral natural interferon-a on performance and rectal temperature of newly received beef steers.</span> <abbr class="cit-jnl-abbrev">Proc. Am. Soc. Anim. Sci. West. Sec.</abbr> <span class="cit-vol">51</span>: <span class="cit-fpage">411</span>–<span class="cit-lpage">414</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BAm.%2BSoc.%2BAnim.%2BSci.%2BWest.%2BSec.%26rft.volume%253D51%26rft.spage%253D411%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3059.12" data-doi="10.1093/jac/dkw184"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Clausen</span>  <span class="cit-name-given-names">P. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zankari</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aarestrup</span>  <span class="cit-name-given-names">F. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lund</span>  <span class="cit-name-given-names">O.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Benchmarking of methods for identification of antimicrobial resistance genes in bacterial whole genome data.</span> <abbr class="cit-jnl-abbrev">J. Antimicrob. Chemother.</abbr> <span class="cit-vol">71</span>: <span class="cit-fpage">2484</span>–<span class="cit-lpage">2488</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAntimicrob.%2BChemother.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjac%252Fdkw184%26rft_id%253Dinfo%253Apmid%252F27365186%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/jac/dkw184&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27365186&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3059.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3059.13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Clawson</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Murray</span>  <span class="cit-name-given-names">R. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sweeney</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Apley</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">DeDonder</span>  <span class="cit-name-given-names">K. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genomic signatures of <em>Mannheimia haemolytica</em> that associate with the lungs of cattle with respiratory disease, an integrative conjugative element, and antimicrobial resistance genes.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">982</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D17%26rft.spage%253D982%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-book" id="cit-7.9.3059.14"><div class="cit-metadata unstructured">CLSI, 2013 <em>Performance Standards for Antimicrobial Disk and Dilution Susceptibility Tests for Bacteria Isolated From Animals</em>, Ed. 4. Approved Standard VET01–A4. CLSI, Wayne, PA.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3059.15"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">D’Amours</span>  <span class="cit-name-given-names">G. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ward</span>  <span class="cit-name-given-names">T. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mulvey</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Read</span>  <span class="cit-name-given-names">R. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morck</span>  <span class="cit-name-given-names">D. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genetic diversity and tetracycline resistance genes of <em>Histophilus somni</em>.</span> <abbr class="cit-jnl-abbrev">Vet. Microbiol.</abbr> <span class="cit-vol">150</span>: <span class="cit-fpage">362</span>–<span class="cit-lpage">372</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVeterinary%2Bmicrobiology%26rft.stitle%253DVet%2BMicrobiol%26rft.aulast%253DD%2527Amours%26rft.auinit1%253DG.%2BH.%26rft.volume%253D150%26rft.issue%253D3-4%26rft.spage%253D362%26rft.epage%253D372%26rft.atitle%253DGenetic%2Bdiversity%2Band%2Btetracycline%2Bresistance%2Bgenes%2Bof%2BHistophilus%2Bsomni.%26rft_id%253Dinfo%253Apmid%252F21482045%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=21482045&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3059.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-7.9.3059.16" data-doi="10.1128/genomeA.01070-14"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Davenport</span>  <span class="cit-name-given-names">K. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daligault</span>  <span class="cit-name-given-names">H. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Minogue</span>  <span class="cit-name-given-names">T. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bishop-Lilly</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bruce</span>  <span class="cit-name-given-names">D. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Complete genome sequence of type strain <em>Pasteurella multocida</em> subsp. multocida ATCC 43137.</span> <abbr class="cit-jnl-abbrev">Genome Announc.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">e01070-14</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BAnnouncements%26rft.stitle%253DGenome%2BAnnounc%26rft.aulast%253DDavenport%26rft.auinit1%253DK.%2BW.%26rft.volume%253D2%26rft.issue%253D5%26rft.spage%253De01070-14%26rft.epage%253De01070-14%26rft.atitle%253DComplete%2BGenome%2BSequence%2Bof%2BType%2BStrain%2BPasteurella%2Bmultocida%2Bsubsp.%2Bmultocida%2BATCC%2B43137%26rft_id%253Dinfo%253Adoi%252F10.1128%252FgenomeA.01070-14%26rft_id%253Dinfo%253Apmid%252F25342682%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiZ2EiO3M6NToicmVzaWQiO3M6MTM6IjIvNS9lMDEwNzAtMTQiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzkvMzA1OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-web" id="cit-7.9.3059.17"><div class="cit-metadata unstructured">Dean, A. G., K. M. Sullivan, and M. M. Soe, 2013 OpenEpi: Open Source Epidemiologic Statistics for Public Health, 2013. Available at: <a href="http://www.OpenEpi.com">www.OpenEpi.com</a>. Accessed: February 26, 2017.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3059.18"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">DeDonder</span>  <span class="cit-name-given-names">K. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Apley</span>  <span class="cit-name-given-names">M. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">A literature review of antimicrobial resistance in pathogens associated with bovine respiratory disease.</span> <abbr class="cit-jnl-abbrev">Anim. Health Res. Rev.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">125</span>–<span class="cit-lpage">134</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnim.%2BHealth%2BRes.%2BRev.%26rft.volume%253D16%26rft.spage%253D125%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3059.19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Edwards</span>  <span class="cit-name-given-names">A. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1948</span> <span class="cit-article-title">Note on the “correction for continuity” in testing the significance of the difference between correlated proportions.</span> <abbr class="cit-jnl-abbrev">Psychometrika</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">185</span>–<span class="cit-lpage">187</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPsychometrika%26rft.stitle%253DPsychometrika%26rft.aulast%253DEdwards%26rft.auinit1%253DA.%2BL.%26rft.volume%253D13%26rft.issue%253D3%26rft.spage%253D185%26rft.epage%253D187%26rft.atitle%253DNote%2Bon%2Bthe%2Bcorrection%2Bfor%2Bcontinuity%2Bin%2Btesting%2Bthe%2Bsignificance%2Bof%2Bthe%2Bdifference%2Bbetween%2Bcorrelated%2Bproportions.%26rft_id%253Dinfo%253Apmid%252F18885738%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=18885738&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3059.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-7.9.3059.20"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eidam</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poehlein</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brenner</span>  <span class="cit-name-given-names">M. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kadlec</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liesegang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Complete genome sequence of <em>Mannheimia haemolytica</em> strain 42548 from a case of bovine respiratory disease.</span> <abbr class="cit-jnl-abbrev">Genome Announc.</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">e00318-e13</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BAnnounc.%26rft.volume%253D1%26rft.spage%253De00318-e13%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3059.21" data-doi="10.1093/jac/dku361"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eidam</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poehlein</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leimbach</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Michael</span>  <span class="cit-name-given-names">G. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kadlec</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Analysis and comparative genomics of ICE<em>Mh1</em>, a novel integrative and conjugative element (ICE) of <em>Mannheimia haemolytica</em>.</span> <abbr class="cit-jnl-abbrev">J. Antimicrob. Chemother.</abbr> <span class="cit-vol">70</span>: <span class="cit-fpage">93</span>–<span class="cit-lpage">97</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAntimicrob.%2BChemother.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjac%252Fdku361%26rft_id%253Dinfo%253Apmid%252F25239467%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/jac/dku361&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25239467&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3059.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3059.22" data-doi="10.1186/1471-2288-13-91"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fagerland</span>  <span class="cit-name-given-names">M. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lydersen</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Laake</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The McNemar test for binary matched-pairs data: mid-p and asymptotic are better than exact conditional.</span> <abbr class="cit-jnl-abbrev">BMC Med. Res. Methodol.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">91</span>–<span class="cit-lpage">98</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BMed.%2BRes.%2BMethodol.%26rft.volume%253D13%26rft.spage%253D91%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2288-13-91%26rft_id%253Dinfo%253Apmid%252F23848987%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2288-13-91&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23848987&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3059.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-book" id="cit-7.9.3059.23"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">FDA</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-source">National Antimicrobial Resistance Monitoring System – Enteric Bacteria (NARMS): 2011 Executive Report</span>. <span class="cit-publ-name">U.S. Department of Health and Human Services, Food and Drug Administration</span>, <span class="cit-publ-loc">Rockville, MD</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3059.24" data-doi="10.1139/gen-2016&amp;ndash;0202"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fitz-Gibbon</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hipp</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pham</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Manos</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sork</span>  <span class="cit-name-given-names">V. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Phylogenomic inferences from reference-mapped and de novo assembled short- read sequence data using RADseq sequencing of California white oaks (Quercus subgenus Quercus).</span> <abbr class="cit-jnl-abbrev">Genome</abbr><span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1139/gen-2016–0202</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%26rft_id%253Dinfo%253Adoi%252F10.1139%252Fgen-2016%2526ndash%253B0202%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1139/gen-2016&amp;ndash;0202&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3059.25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fulton</span>  <span class="cit-name-given-names">R. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Confer</span>  <span class="cit-name-given-names">A. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Laboratory test descriptions for bovine respiratory disease diagnosis and their strengths and weaknesses: gold standards for diagnosis, do they exist?</span> <abbr class="cit-jnl-abbrev">Can. Vet. J.</abbr> <span class="cit-vol">53</span>: <span class="cit-fpage">754</span>–<span class="cit-lpage">761</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCan.%2BVet.%2BJ.%26rft.volume%253D53%26rft.spage%253D754%26rft_id%253Dinfo%253Apmid%252F23277642%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=23277642&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3059.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3059.26" data-doi="10.1159/000342709"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gilmour</span>  <span class="cit-name-given-names">M. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Graham</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reimer</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van Domselaar</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Public health genomics and the new molecular epidemiology of bacterial pathogens.</span> <abbr class="cit-jnl-abbrev">Public Health Genomics</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">25</span>–<span class="cit-lpage">30</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPublic%2BHealth%2BGenomics%26rft.volume%253D16%26rft.spage%253D25%26rft_id%253Dinfo%253Adoi%252F10.1159%252F000342709%26rft_id%253Dinfo%253Apmid%252F23548714%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1159/000342709&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23548714&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3059.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3059.27" data-doi="10.1016/j.cvfa.2010.04.004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Griffin</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chengappa</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuszak</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McVey</span>  <span class="cit-name-given-names">D. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Bacterial pathogens of the bovine respiratory disease complex.</span> <abbr class="cit-jnl-abbrev">Vet. Clin. North Am. Food Anim. Pract.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">381</span>–<span class="cit-lpage">394</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BVeterinary%2Bclinics%2Bof%2BNorth%2BAmerica.%2BFood%2Banimal%2Bpractice%26rft.stitle%253DVet%2BClin%2BNorth%2BAm%2BFood%2BAnim%2BPract%26rft.aulast%253DGriffin%26rft.auinit1%253DD.%26rft.volume%253D26%26rft.issue%253D2%26rft.spage%253D381%26rft.epage%253D394%26rft.atitle%253DBacterial%2Bpathogens%2Bof%2Bthe%2Bbovine%2Brespiratory%2Bdisease%2Bcomplex.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cvfa.2010.04.004%26rft_id%253Dinfo%253Apmid%252F20619191%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cvfa.2010.04.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20619191&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3059.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3059.28" data-doi="10.1093/molbev/msp259"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gouy</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guindon</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gascuel</span>  <span class="cit-name-given-names">O.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">SeaView version 4: a multiplatform graphical user interface for sequence alignment and phylogenetic tree building.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">221</span>–<span class="cit-lpage">224</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsp259%26rft_id%253Dinfo%253Apmid%252F19854763%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msp259&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19854763&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3059.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000273704400003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-7.9.3059.29"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Harhay</span>  <span class="cit-name-given-names">G. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koren</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Phillippy</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McVey</span>  <span class="cit-name-given-names">D. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuszak</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Complete closed genome sequences of <em>Mannheimia haemolytica</em> serotypes A1 and A6, isolated from cattle.</span> <abbr class="cit-jnl-abbrev">Genome Announc.</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">e00188-e13</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BAnnounc.%26rft.volume%253D1%26rft.spage%253De00188-e13%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-7.9.3059.30"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hauglund</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tatum</span>  <span class="cit-name-given-names">F. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bayles</span>  <span class="cit-name-given-names">D. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maheswaran</span>  <span class="cit-name-given-names">S. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Briggs</span>  <span class="cit-name-given-names">R. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genome sequences of <em>Mannheimia haemolytica</em> serotype A1 strains D153 and D193 from bovine pneumonia.</span> <abbr class="cit-jnl-abbrev">Genome Announc.</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">e00848-e13</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BAnnounc.%26rft.volume%253D1%26rft.spage%253De00848-e13%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-7.9.3059.31"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hauglund</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tatum</span>  <span class="cit-name-given-names">F. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bayles</span>  <span class="cit-name-given-names">D. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maheswaran</span>  <span class="cit-name-given-names">S. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Briggs</span>  <span class="cit-name-given-names">R. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span><span class="cit-comment">a</span> <span class="cit-article-title">Genome sequences of <em>Mannheimia haemolytica</em> serotype A2 isolates D171 and D35, recovered from bovine pneumonia.</span> <abbr class="cit-jnl-abbrev">Genome Announc.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">e00093-e15</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BAnnounc.%26rft.volume%253D3%26rft.spage%253De00093-e15%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-7.9.3059.32"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hauglund</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tatum</span>  <span class="cit-name-given-names">F. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bayles</span>  <span class="cit-name-given-names">D. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maheswaran</span>  <span class="cit-name-given-names">S. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Briggs</span>  <span class="cit-name-given-names">R. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span><span class="cit-comment">b</span> <span class="cit-article-title">Genome sequences of serotype A6 <em>Mannheimia haemolytica</em> isolates D174 and D38 recovered from bovine pneumonia.</span> <abbr class="cit-jnl-abbrev">Genome Announc.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">e00086-e15</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BAnnounc.%26rft.volume%253D3%26rft.spage%253De00086-e15%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-7.9.3059.33"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heaton</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harhay</span>  <span class="cit-name-given-names">G. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">T. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bono</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chitko-McKown</span>  <span class="cit-name-given-names">C. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Complete closed genome sequences of a <em>Mannheimia haemolytica</em> serotype A1 leukotoxin deletion mutant and its wild-type parent strain.</span> <abbr class="cit-jnl-abbrev">Genome Announc.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">e00417-e15</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BAnnounc.%26rft.volume%253D3%26rft.spage%253De00417-e15%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3059.34" data-doi="10.1186/s13073-014-0090-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Inouye</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dashnow</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raven</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schultz</span>  <span class="cit-name-given-names">M. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pope</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">SRST2: rapid genomic surveillance for public health and hospital microbiology labs.</span> <abbr class="cit-jnl-abbrev">Genome Med.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">90</span>–<span class="cit-lpage">106</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BMed.%26rft.volume%253D6%26rft.spage%253D90%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13073-014-0090-6%26rft_id%253Dinfo%253Apmid%252F25422674%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13073-014-0090-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25422674&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3059.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3059.35" data-doi="10.1016/j.diagmicrobio.2014.08.001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Janda</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abbot</span>  <span class="cit-name-given-names">S. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Culture-independent diagnostic testing: have we opened Pandora’s box for good?</span> <abbr class="cit-jnl-abbrev">Diagn. Microbiol. Infect. Dis.</abbr> <span class="cit-vol">80</span>: <span class="cit-fpage">171</span>–<span class="cit-lpage">176</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDiagn.%2BMicrobiol.%2BInfect.%2BDis.%26rft.volume%253D80%26rft.spage%253D171%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.diagmicrobio.2014.08.001%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.diagmicrobio.2014.08.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3059.36" data-doi="10.1093/nar/gkw1004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jia</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raphenya</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alcock</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Waglechner</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">CARD 2017: expansion and model-centric curation of the comprehensive antibiotic resistance database.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">D566</span>–<span class="cit-lpage">D573</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkw1004%26rft_id%253Dinfo%253Apmid%252F27789705%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkw1004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27789705&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3059.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3059.37" data-doi="10.1128/AAC.00092-11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kadlec</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brenner</span>  <span class="cit-name-given-names">M. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sweeney</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brzuszkiewicz</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liesegang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Molecular basis of macrolide, triamilide, and lincosamide resistance in <em>Pasteurella multocida</em> from bovine respiratory disease.</span> <abbr class="cit-jnl-abbrev">Antimicrob. Agents Chemother.</abbr> <span class="cit-vol">55</span>: <span class="cit-fpage">2475</span>–<span class="cit-lpage">2477</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAntimicrob.%2BAgents%2BChemother.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FAAC.00092-11%26rft_id%253Dinfo%253Apmid%252F21402855%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiYWFjIjtzOjU6InJlc2lkIjtzOjk6IjU1LzUvMjQ3NSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvOS8zMDU5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3059.38" data-doi="10.1093/jac/45.6.871"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Karlowsky</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Verma</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhanel</span>  <span class="cit-name-given-names">G. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoban</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Presence of ROB-1 betalactamase correlates with cefaclor resistance among recent isolates of <em>Haemophilus influenzae</em>.</span> <abbr class="cit-jnl-abbrev">J. Antimicrob. Chemother.</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">871</span>–<span class="cit-lpage">875</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BAntimicrobial%2BChemotherapy%26rft.stitle%253DJ%2BAntimicrob%2BChemother%26rft.aulast%253DKarlowsky%26rft.auinit1%253DJ.%2BA.%26rft.volume%253D45%26rft.issue%253D6%26rft.spage%253D871%26rft.epage%253D875%26rft.atitle%253DPresence%2Bof%2BROB-1%2B%257Bbeta%257D-lactamase%2Bcorrelates%2Bwith%2Bcefaclor%2Bresistance%2Bamong%2Brecent%2Bisolates%2Bof%2BHaemophilus%2Binfluenzae%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjac%252F45.6.871%26rft_id%253Dinfo%253Apmid%252F10837442%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/jac/45.6.871&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10837442&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3059.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000087938700021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3059.39"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kawai</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Higuchi</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Iwano</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Iwakuma</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Onda</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Antimicrobial susceptibilities of <em>Mycoplasma</em> isolated from bovine mastitis in Japan.</span> <abbr class="cit-jnl-abbrev">Anim. Sci. J.</abbr> <span class="cit-vol">85</span>: <span class="cit-fpage">96</span>–<span class="cit-lpage">99</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnim.%2BSci.%2BJ.%26rft.volume%253D85%26rft.spage%253D96%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3059.40" data-doi="10.1093/jac/48.5.631"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kehrenberg</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salmon</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Watts</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schwarz</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Tetracycline resistance genes in isolates of <em>Pasteurella multocida</em>, <em>Mannheimia haemolytica</em>, <em>Mannheimia glucosida</em> and <em>Mannheimia varigena</em> from bovine and swine respiratory disease: intergeneric spread of the tet(H) plasmid pMHT1.</span> <abbr class="cit-jnl-abbrev">J. Antimicrob. Chemother.</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">631</span>–<span class="cit-lpage">640</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BAntimicrobial%2BChemotherapy%26rft.stitle%253DJ%2BAntimicrob%2BChemother%26rft.aulast%253DKehrenberg%26rft.auinit1%253DC.%26rft.volume%253D48%26rft.issue%253D5%26rft.spage%253D631%26rft.epage%253D640%26rft.atitle%253DTetracycline%2Bresistance%2Bgenes%2Bin%2Bisolates%2Bof%2BPasteurella%2Bmultocida%252C%2BMannheimia%2Bhaemolytica%252C%2BMannheimia%2Bglucosida%2Band%2BMannheimia%2Bvarigena%2Bfrom%2Bbovine%2Band%2Bswine%2Brespiratory%2Bdisease%253A%2Bintergeneric%2Bspread%2Bof%2Bthe%2Btet%2528H%2529%2Bplasmid%2BpMHT1%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjac%252F48.5.631%26rft_id%253Dinfo%253Apmid%252F11679552%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/jac/48.5.631&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11679552&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3059.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000172342800006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3059.41" data-doi="10.1101/gr.229202. Article published online before March 2002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kent</span>  <span class="cit-name-given-names">W. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">BLAT–the BLAST-like alignment tool.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">656</span>–<span class="cit-lpage">664</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.229202.%2BArticle%2Bpublished%2Bonline%2Bbefore%2BMarch%2B2002%26rft_id%253Dinfo%253Apmid%252F11932250%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjEyLzQvNjU2IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy85LzMwNTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3059.42" data-doi="10.1128/JCM.02485-13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Klima</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zaheer</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cook</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Booker</span>  <span class="cit-name-given-names">C. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hendrick</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Pathogens of bovine respiratory disease in North American feedlots conferring multidrug resistance via integrative conjugative elements.</span> <abbr class="cit-jnl-abbrev">J. Clin. Microbiol.</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">438</span>–<span class="cit-lpage">448</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BClin.%2BMicrobiol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJCM.02485-13%26rft_id%253Dinfo%253Apmid%252F24478472%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czozOiJqY20iO3M6NToicmVzaWQiO3M6ODoiNTIvMi80MzgiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzkvMzA1OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3059.43"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kong</span>  <span class="cit-name-given-names">L. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goa</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jia</span>  <span class="cit-name-given-names">B. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gao</span>  <span class="cit-name-given-names">Y. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Antimicrobial susceptibility and molecular characterization of macrolide resistance of <em>Mycoplasma bovis</em> isolates from multiple provinces in China.</span> <abbr class="cit-jnl-abbrev">J. Vet. Med. Sci.</abbr> <span class="cit-vol">78</span>: <span class="cit-fpage">293</span>–<span class="cit-lpage">296</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BVet.%2BMed.%2BSci.%26rft.volume%253D78%26rft.spage%253D293%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-7.9.3059.44" data-doi="10.1186/gb-2013-14-9-r101"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Koren</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harhay</span>  <span class="cit-name-given-names">G. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">T. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bono</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harhay</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Reducing assembly complexity of microbial genomes with single-molecule sequencing.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">R101</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D14%26rft.spage%253DR101%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2013-14-9-r101%26rft_id%253Dinfo%253Apmid%252F24034426%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2013-14-9-r101&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24034426&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3059.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3059.45" data-doi="10.1093/bioinformatics/btm091"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Koressaar</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Remm</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Enhancements and modifications of primer design program Primer3.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">1289</span>–<span class="cit-lpage">1291</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtm091%26rft_id%253Dinfo%253Apmid%252F17379693%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btm091&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17379693&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3059.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000247348300015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3059.46" data-doi="10.1186/s12859-014-0356-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Korneliussen</span>  <span class="cit-name-given-names">T. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Albrechtsen</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">ANGSD: analysis of next generation sequencing data.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">356</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BBioinformatics%26rft.volume%253D15%26rft.spage%253D356%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12859-014-0356-4%26rft_id%253Dinfo%253Apmid%252F25420514%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12859-014-0356-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25420514&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3059.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3059.47" data-doi="10.1016/j.tig.2014.07.003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Koser</span>  <span class="cit-name-given-names">C. U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ellington</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peacock</span>  <span class="cit-name-given-names">S. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Whole genome sequencing to control antimicrobial resistance.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">401</span>–<span class="cit-lpage">407</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BGenet.%26rft.volume%253D30%26rft.spage%253D401%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2014.07.003%26rft_id%253Dinfo%253Apmid%252F25096945%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tig.2014.07.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25096945&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3059.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3059.48" data-doi="10.1038/nmeth.1923"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Langmead</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salzberg</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Fast gapped-read alignment with Bowtie 2.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">357</span>–<span class="cit-lpage">359</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DLangmead%26rft.auinit1%253DB.%26rft.volume%253D9%26rft.issue%253D4%26rft.spage%253D357%26rft.epage%253D359%26rft.atitle%253DFast%2Bgapped-read%2Balignment%2Bwith%2BBowtie%2B2.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1923%26rft_id%253Dinfo%253Apmid%252F22388286%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1923&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22388286&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3059.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000302218500017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3059.49" data-doi="10.1093/bioinformatics/btp352"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Handsaker</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wysoker</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fennell</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The sequence alignment/map (SAM) format and SAMtools.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">2078</span>–<span class="cit-lpage">2079</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp352%26rft_id%253Dinfo%253Apmid%252F19505943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp352&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19505943&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3059.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268808600014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-7.9.3059.50" data-doi="10.1371/journal.pone.0020999"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zheng</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xin</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The complete genome sequence of <em>Mycoplasma bovis</em> strain Hubei-1.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e20999</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DLi%26rft.auinit1%253DY.%26rft.volume%253D6%26rft.issue%253D6%26rft.spage%253De20999%26rft.epage%253De20999%26rft.atitle%253DThe%2Bcomplete%2Bgenome%2Bsequence%2Bof%2BMycoplasma%2Bbovis%2Bstrain%2BHubei-1.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0020999%26rft_id%253Dinfo%253Apmid%252F21731639%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0020999&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21731639&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3059.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-7.9.3059.51" data-doi="10.1128/JB.00215-12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zheng</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liang</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Complete genome sequence of <em>Pasteurella multocida</em> HN06, a toxigenic strain of serogroup D.</span> <abbr class="cit-jnl-abbrev">J. Bacteriol.</abbr> <span class="cit-vol">194</span>: <span class="cit-fpage">3292</span>–<span class="cit-lpage">3293</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBacteriology%26rft.stitle%253DJ.%2BBacteriol.%26rft.aulast%253DLiu%26rft.auinit1%253DW.%26rft.volume%253D194%26rft.issue%253D12%26rft.spage%253D3292%26rft.epage%253D3293%26rft.atitle%253DComplete%2BGenome%2BSequence%2Bof%2BPasteurella%2Bmultocida%2BHN06%252C%2Ba%2BToxigenic%2BStrain%2Bof%2BSerogroup%2BD%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJB.00215-12%26rft_id%253Dinfo%253Apmid%252F22628516%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6MTE6IjE5NC8xMi8zMjkyIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy85LzMwNTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3059.52" data-doi="10.1093/jac/47.1.87"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Livermore</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Winstanley</span>  <span class="cit-name-given-names">T. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shannon</span>  <span class="cit-name-given-names">K. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Interpretative reading: recognizing the unusual and inferring resistance mechanisms from resistance phenotypes.</span> <abbr class="cit-jnl-abbrev">J. Antimicrob. Chemother.</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">87</span>–<span class="cit-lpage">102</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BAntimicrobial%2BChemotherapy%26rft.stitle%253DJ%2BAntimicrob%2BChemother%26rft.aulast%253DLivermore%26rft.auinit1%253DD.%2BM.%26rft.volume%253D48%26rft.issue%253D90001%26rft.spage%253D87%26rft.epage%253D102%26rft.atitle%253DInterpretative%2Breading%253A%2Brecognizing%2Bthe%2Bunusual%2Band%2Binferring%2Bresistance%2Bmechanisms%2Bfrom%2Bresistance%2Bphenotypes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjac%252F47.1.87%26rft_id%253Dinfo%253Apmid%252F11420342%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/jac/47.1.87&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11420342&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3059.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000170182300012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3059.53" data-doi="10.7717/peerj.238"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Love</span>  <span class="cit-name-given-names">W. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lehenbauer</span>  <span class="cit-name-given-names">T. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kass</span>  <span class="cit-name-given-names">P. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van Eenennaam</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aly</span>  <span class="cit-name-given-names">S. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Development of a novel clinical scoring system for on-farm diagnosis of bovine respiratory disease in pre-weaned dairy calves.</span> <abbr class="cit-jnl-abbrev">PeerJ</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">e238</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPeerJ%26rft.volume%253D2%26rft.spage%253De238%26rft_id%253Dinfo%253Adoi%252F10.7717%252Fpeerj.238%26rft_id%253Dinfo%253Apmid%252F24482759%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.7717/peerj.238&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24482759&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3059.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3059.54" data-doi="10.1371/journal.pone.0144310"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Marsh</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krauland</span>  <span class="cit-name-given-names">M. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nelson</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schlackman</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brooks</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genomic epidemiology of an endoscope-associated outbreak of <em>Klebsiella pneumoniae</em> Carbapenemase (KPC)-producing <em>K. pneumoniae</em>.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">e0144310</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D10%26rft.spage%253De0144310%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0144310%26rft_id%253Dinfo%253Apmid%252F26637170%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0144310&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26637170&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3059.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3059.55" data-doi="10.1111/j.1939-1676.2011.0750.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Maunsell</span>  <span class="cit-name-given-names">F. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Woolums</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Francoz</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosenbusch</span>  <span class="cit-name-given-names">R. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Step</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title"><em>Mycoplasma bovis</em> infections in cattle.</span> <abbr class="cit-jnl-abbrev">J. Vet. Intern. Med.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">772</span>–<span class="cit-lpage">783</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bveterinary%2Binternal%2Bmedicine%2B%252F%2BAmerican%2BCollege%2Bof%2BVeterinary%2BInternal%2BMedicine%26rft.stitle%253DJ%2BVet%2BIntern%2BMed%26rft.aulast%253DMaunsell%26rft.auinit1%253DF.%2BP.%26rft.volume%253D25%26rft.issue%253D4%26rft.spage%253D772%26rft.epage%253D783%26rft.atitle%253DMycoplasma%2Bbovis%2Binfections%2Bin%2Bcattle.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1939-1676.2011.0750.x%26rft_id%253Dinfo%253Apmid%252F21745245%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1939-1676.2011.0750.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21745245&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3059.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-7.9.3059.56" data-doi="10.1073/pnas.051634598"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">May</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">L. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paustian</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Whittam</span>  <span class="cit-name-given-names">T. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Complete genomic sequence of <em>Pasteurella multocida</em>, Pm70.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">98</span>: <span class="cit-fpage">3460</span>–<span class="cit-lpage">3465</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.051634598%26rft_id%253Dinfo%253Apmid%252F11248100%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiI5OC82LzM0NjAiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzkvMzA1OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3059.57" data-doi="10.1016/j.cvfa.2007.10.003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McGuirk</span>  <span class="cit-name-given-names">S. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Disease management of dairy calves and heifers. Disease management of dairy calves and heifers.</span> <abbr class="cit-jnl-abbrev">Vet. Clin. North Am. Food Anim. Pract.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">139</span>–<span class="cit-lpage">153</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BVeterinary%2Bclinics%2Bof%2BNorth%2BAmerica.%2BFood%2Banimal%2Bpractice%26rft.stitle%253DVet%2BClin%2BNorth%2BAm%2BFood%2BAnim%2BPract%26rft.aulast%253DMcGuirk%26rft.auinit1%253DS.%2BM.%26rft.volume%253D24%26rft.issue%253D1%26rft.spage%253D139%26rft.epage%253D153%26rft.atitle%253DDisease%2Bmanagement%2Bof%2Bdairy%2Bcalves%2Band%2Bheifers.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cvfa.2007.10.003%26rft_id%253Dinfo%253Apmid%252F18299036%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cvfa.2007.10.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18299036&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3059.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-2" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3059.58"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Michael</span>  <span class="cit-name-given-names">G. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kadlec</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sweeney</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brzuszkiewicz</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liesegang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">ICEPmu1, an integrative conjugative element (ICE) of <em>Pasteurella multocida</em>: structure and transfer.</span> <abbr class="cit-jnl-abbrev">J. Antimicrob. Chemother.</abbr> <span class="cit-vol">67</span>: <span class="cit-fpage">91</span>–<span class="cit-lpage">100</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAntimicrob.%2BChemother.%26rft.volume%253D67%26rft.spage%253D91%26rft_id%253Dinfo%253Apmid%252F22001176%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=22001176&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3059.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000300833700014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3059.59"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mohammed</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cormican</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Whole genome sequencing provides possible explanations for the difference in phage susceptibility among two <em>Salmonella Typhimurium</em> phage types (DT8 and DT30) associated with a single foodborne outbreak.</span> <abbr class="cit-jnl-abbrev">BMC Res. Notes</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">728</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BRes.%2BNotes%26rft.volume%253D8%26rft.spage%253D728%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3059.60"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mustapha</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marsh</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krauland</span>  <span class="cit-name-given-names">M. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernandez</span>  <span class="cit-name-given-names">J. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Lemos</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genomic epidemiology of hypervirulent serogroup W, ST-11 <em>Neisseria meningitidis</em>.</span> <abbr class="cit-jnl-abbrev">EBioMedicine</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">1447</span>–<span class="cit-lpage">1455</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEBioMedicine%26rft.volume%253D2%26rft.spage%253D1447%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3059.61" data-doi="10.1038/nature10392"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mutreja</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">D. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thomson</span>  <span class="cit-name-given-names">N. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Connor</span>  <span class="cit-name-given-names">T. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Evidence for several waves of global transmission in the seventh cholera pandemic.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">477</span>: <span class="cit-fpage">462</span>–<span class="cit-lpage">465</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DMutreja%26rft.auinit1%253DA.%26rft.volume%253D477%26rft.issue%253D7365%26rft.spage%253D462%26rft.epage%253D465%26rft.atitle%253DEvidence%2Bfor%2Bseveral%2Bwaves%2Bof%2Bglobal%2Btransmission%2Bin%2Bthe%2Bseventh%2Bcholera%2Bpandemic.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature10392%26rft_id%253Dinfo%253Apmid%252F21866102%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature10392&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21866102&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3059.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000295080500040&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3059.62" data-doi="10.1186/1471-2164-15-1164"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Neibergs</span>  <span class="cit-name-given-names">H. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seabury</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wojtowicz</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scraggs</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Susceptibility loci revealed for bovine respiratory disease complex in pre-weaned Holstein calves.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">1164</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D15%26rft.spage%253D1164%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-15-1164%26rft_id%253Dinfo%253Apmid%252F25534905%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-15-1164&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25534905&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3059.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3059.63"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Noyes</span>  <span class="cit-name-given-names">N. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Benedict</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gow</span>  <span class="cit-name-given-names">S. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Booker</span>  <span class="cit-name-given-names">C. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hannon</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title"><em>Mannheimia haemolytica</em> in feedlot cattle: prevalence of recovery and associations with antimicrobial use, resistance, and health outcomes.</span> <abbr class="cit-jnl-abbrev">J. Vet. Intern. Med.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">705</span>–<span class="cit-lpage">713</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BVet.%2BIntern.%2BMed.%26rft.volume%253D29%26rft.spage%253D705%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-7.9.3059.64"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Peng</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liang</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tang</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genomic characterization of <em>Pasteurella multocida</em> HB01, a serotype A bovine isolate from China.</span> <abbr class="cit-jnl-abbrev">Gene</abbr> <span class="cit-vol">581</span>: <span class="cit-fpage">85</span>–<span class="cit-lpage">93</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGene%26rft.volume%253D581%26rft.spage%253D85%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-7.9.3059.65" data-doi="10.1371/journal.pone.0038239"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Qi</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cui</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mustafa</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Comparative geno-plasticity analysis of <em>Mycoplasma bovis</em> HB0801 (Chinese isolate).</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e38239</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DQi%26rft.auinit1%253DJ.%26rft.volume%253D7%26rft.issue%253D5%26rft.spage%253De38239%26rft.epage%253De38239%26rft.atitle%253DComparative%2Bgeno-plasticity%2Banalysis%2Bof%2BMycoplasma%2Bbovis%2BHB0801%2B%2528Chinese%2Bisolate%2529.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0038239%26rft_id%253Dinfo%253Apmid%252F22693604%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0038239&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22693604&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3059.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-7.9.3059.66" data-doi="10.1186/1471-2164-12-570"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Siddaramappa</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Challacombe</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Duncan</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gillaspy</span>  <span class="cit-name-given-names">A. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carson</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Horizontal gene transfer in <em>Histophilus somni</em> and its role in the evolution of pathogenic strain 2336, as determined by comparative genomic analyses.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">570</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DSiddaramappa%26rft.auinit1%253DS.%26rft.volume%253D12%26rft.spage%253D570%26rft.epage%253D570%26rft.atitle%253DHorizontal%2Bgene%2Btransfer%2Bin%2BHistophilus%2Bsomni%2Band%2Bits%2Brole%2Bin%2Bthe%2Bevolution%2Bof%2Bpathogenic%2Bstrain%2B2336%252C%2Bas%2Bdetermined%2Bby%2Bcomparative%2Bgenomic%2Banalyses.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-12-570%26rft_id%253Dinfo%253Apmid%252F22111657%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-12-570&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22111657&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3059.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-confproc" id="cit-7.9.3059.67"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Street</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Petersen</span>  <span class="cit-name-given-names">K. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gerritsen</span>  <span class="cit-name-given-names">A. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hunter</span>  <span class="cit-name-given-names">S. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Settles</span>  <span class="cit-name-given-names">M. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> expHTS: analysis of high throughput sequence data in an experimental framework. Proceedings of the 6<sup>th</sup> ACM Conference on Bioinformatics, Computational Biology and Health Informatics, Atlanta, GA. pp. <span class="cit-fpage">523</span>–<span class="cit-lpage">524</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3059.68"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ullah</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abubakar</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arshed</span>  <span class="cit-name-given-names">M. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Differentiation of closely related vaccinal strains of <em>Pasteurella multocida</em> using polymerase chain reaction (PCR).</span> <abbr class="cit-jnl-abbrev">VetScan.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">36</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVetScan.%26rft.volume%253D4%26rft.spage%253D36%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3059.69" data-doi="10.1093/nar/gks596"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Untergrasser</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cutcutache</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koressaar</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ye</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Faircloth</span>  <span class="cit-name-given-names">B. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Primer3 - new capabilities and interfaces.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">e115</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgks596%26rft_id%253Dinfo%253Apmid%252F22730293%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gks596&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22730293&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3059.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-web" id="cit-7.9.3059.70"><div class="cit-metadata unstructured">USDA, 2011a National Agricultural Statistics Service Agricultural Statistics Board (NASS SAB) Cattle Death Loss. Available at: <a href="http://usda.mannlib.cornell.edu/MannUsda/viewDocumentInfo.do?documentID=1625">http://usda.mannlib.cornell.edu/MannUsda/viewDocumentInfo.do?documentID=1625</a>. Accessed: February 26, 2017.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-book" id="cit-7.9.3059.71"><div class="cit-metadata unstructured">USDA, 2011b <em>Part IV: Health and Health Management on U.S. Feedlots with a Capacity of 1,000 or More Head. USDA–APHIS–VS–CEAH–NAHMS</em>. Fort Collins, CO #638.0913.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-7.9.3059.72"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wise</span>  <span class="cit-name-given-names">K. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Calcutt</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Foecking</span>  <span class="cit-name-given-names">M. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roske</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Madupu</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Complete genome sequence of <em>Mycoplasma bovis</em> type strain PG45 (ATCC 25523).</span> <abbr class="cit-jnl-abbrev">Infect. Immun.</abbr> <span class="cit-vol">79</span>: <span class="cit-fpage">982</span>–<span class="cit-lpage">983</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInfect.%2BImmun.%26rft.volume%253D79%26rft.spage%253D982%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference  in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3059.73" data-doi="10.1093/jac/dks261"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zankari</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hasman</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cosentino</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vestergaard</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rasmussen</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Identification of acquired antimicrobial resistance genes.</span> <abbr class="cit-jnl-abbrev">J. Antimicrob. Chemother.</abbr> <span class="cit-vol">67</span>: <span class="cit-fpage">2640</span>–<span class="cit-lpage">2644</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAntimicrob.%2BChemother.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjac%252Fdks261%26rft_id%253Dinfo%253Apmid%252F22782487%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/jac/dks261&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22782487&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3059.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000309923800014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/7/9/3059.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/9/3047" title="Annotated Draft Genome Assemblies for the Northern Bobwhite (Colinus virginianus) and the Scaled Quail (Callipepla squamata) Reveal Disparate Estimates of Modern Genome Diversity and Historic Effective Population Size" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/9/3073" title="Genome-Wide Transcriptional Dynamics in the Companion Bacterial Symbionts of the Glassy-Winged Sharpshooter (Cicadellidae: Homalodisca vitripennis) Reveal Differential Gene Expression in Bacteria Occupying Multiple Host Organs" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/7/9"><span>Volume 7 Issue 9, September 2017</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/7/9.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/7/9.cover-source.jpg?itok=WuYpGT8r" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 7 (9)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/9/3059" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-vh1Vj4i3fbstPuhrmdkKcxopnItGzG_QjWwpuBdr8BA" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/7/9/3059" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/7/9/3059" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/7/9/3059" class="active" data-icon-position="" data-hide-link-title="0">Whole-Genome Sequencing and Concordance Between Antimicrobial Susceptibility Genotypes and Phenotypes of Bacterial Isolates Associated with Bovine Respiratory Disease</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/486354" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-z6chyVOLPTXj28IDYYUYCJCE61-bF1vChgUFqeD493w" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/9/3059.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/7/9/3059" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-Rsm4xY4s4Vo7mf98YOTfRqkIHzG0zCmFP_m_Irch1qA" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/7/9/3059" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/9/3059.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="486354" id="node486354--2" data-pisa="ggg;7/9/3059" data-pisa-master="ggg;g3.117.1137" data-apath="/ggg/7/9/3059.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Whole-Genome Sequencing and Concordance Between Antimicrobial Susceptibility Genotypes and Phenotypes of Bacterial Isolates Associated with Bovine Respiratory Disease</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-7821-3012" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Joseph R. Owen</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-6149-1008" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Noelle Noyes</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-4181-4233" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Amy E. Young</span>, <span class="highwire-citation-author" data-delta="3">Daniel J. Prince</span>, <span class="highwire-citation-author" data-delta="4">Patricia C. Blanchard</span>, <span class="highwire-citation-author" data-delta="5">Terry W. Lehenbauer</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0003-0330-5013" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Sharif S. Aly</span>, <span class="highwire-citation-author" data-delta="7">Jessica H. Davis</span>, <span class="highwire-citation-author" data-delta="8">Sean M. O’Rourke</span>, <span class="highwire-citation-author" data-delta="9">Zaid Abdo</span>, <span class="highwire-citation-author" data-delta="10">Keith Belk</span>, <span class="highwire-citation-author" data-delta="11">Michael R. Miller</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="12"><a href="http://orcid.org/0000-0001-8138-2714" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Paul Morley</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="13"><a href="http://orcid.org/0000-0003-1562-162X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Alison L. Van Eenennaam</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3059-3071;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.1137 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node486354--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Joseph R. Owen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Animal Science, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joseph%2BR.%2BOwen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Owen%20JR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJoseph%2BR.%2BOwen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-7821-3012" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Joseph R. Owen</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Noelle Noyes </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Microbiology, Immunology, and Pathology, College of Veterinary Medicine and Biomedical Sciences, Colorado State University, Fort Collins, Colorado 80523</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Noelle%2BNoyes%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Noyes%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ANoelle%2BNoyes%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-6149-1008" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Noelle Noyes</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Amy E. Young </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Animal Science, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Amy%2BE.%2BYoung%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Young%20AE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAmy%2BE.%2BYoung%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-4181-4233" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Amy E. Young</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Daniel J. Prince </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Animal Science, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniel%2BJ.%2BPrince%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Prince%20DJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADaniel%2BJ.%2BPrince%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Patricia C. Blanchard </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>California Animal Health and Food Safety Laboratory, School of Veterinary Medicine, University of California, Davis, Tulare, California 93274</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Patricia%2BC.%2BBlanchard%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Blanchard%20PC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APatricia%2BC.%2BBlanchard%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Terry W. Lehenbauer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Veterinary Medicine Teaching and Research Center, School of Veterinary Medicine, University of California, Davis, Tulare, California 93274</div><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Population Health and Reproduction, School of Veterinary Medicine, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Terry%2BW.%2BLehenbauer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lehenbauer%20TW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATerry%2BW.%2BLehenbauer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Sharif S. Aly </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Veterinary Medicine Teaching and Research Center, School of Veterinary Medicine, University of California, Davis, Tulare, California 93274</div><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Population Health and Reproduction, School of Veterinary Medicine, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sharif%2BS.%2BAly%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Aly%20SS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASharif%2BS.%2BAly%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-0330-5013" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Sharif S. Aly</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Jessica H. Davis </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Veterinary Medicine Teaching and Research Center, School of Veterinary Medicine, University of California, Davis, Tulare, California 93274</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jessica%2BH.%2BDavis%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Davis%20JH&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJessica%2BH.%2BDavis%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Sean M. O’Rourke </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Animal Science, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sean%2BM.%2BO%E2%80%99Rourke%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=O%E2%80%99Rourke%20SM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASean%2BM.%2BO%25E2%2580%2599Rourke%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Zaid Abdo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Microbiology, Immunology, and Pathology, College of Veterinary Medicine and Biomedical Sciences, Colorado State University, Fort Collins, Colorado 80523</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zaid%2BAbdo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Abdo%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AZaid%2BAbdo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Keith Belk </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>Department of Animal Sciences, College of Agricultural Sciences, Colorado State University, Fort Collins, Colorado 80523</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Keith%2BBelk%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Belk%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKeith%2BBelk%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Michael R. Miller </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Animal Science, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BR.%2BMiller%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Miller%20MR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichael%2BR.%2BMiller%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-name">Paul Morley </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>Department of Clinical Sciences, College of Veterinary Medicine and Biomedical Sciences, Colorado State University, Fort Collins, Colorado 80523</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Paul%2BMorley%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Morley%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APaul%2BMorley%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-8138-2714" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Paul Morley</a></li></ul></div><div class="author-tooltip-13"><div class="author-tooltip-name">Alison L. Van Eenennaam </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Animal Science, University of California, Davis, California 95616</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Veterinary Medicine Teaching and Research Center, School of Veterinary Medicine, University of California, Davis, Tulare, California 93274</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alison%2BL.%2BVan%2BEenennaam%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Van%20Eenennaam%20AL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAlison%2BL.%2BVan%2BEenennaam%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-1562-162X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Alison L. Van Eenennaam</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:alvaneenennaam@ucdavis.edu" class="" data-icon-position="" data-hide-link-title="0">alvaneenennaam@ucdavis.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/7/9/3059" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/7/9/3059&amp;title=Whole-Genome%20Sequencing%20and%20Concordance%20Between%20Antimicrobial%20Susceptibility%20Genotypes%20and%20Phenotypes%20of%20Bacterial%20Isolates%20Associated%20with%20Bovine%20Respiratory%20Disease" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/7/9/3059&amp;title=Whole-Genome%20Sequencing%20and%20Concordance%20Between%20Antimicrobial%20Susceptibility%20Genotypes%20and%20Phenotypes%20of%20Bacterial%20Isolates%20Associated%20with%20Bovine%20Respiratory%20Disease" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/7/9/3059&amp;title=Whole-Genome%20Sequencing%20and%20Concordance%20Between%20Antimicrobial%20Susceptibility%20Genotypes%20and%20Phenotypes%20of%20Bacterial%20Isolates%20Associated%20with%20Bovine%20Respiratory%20Disease" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/7/9/3059&amp;text=Whole-Genome%20Sequencing%20and%20Concordance%20Between%20Antimicrobial%20Susceptibility%20Genotypes%20and%20Phenotypes%20of%20Bacterial%20Isolates%20Associated%20with%20Bovine%20Respiratory%20Disease" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/7/9/3059&amp;title=Whole-Genome%20Sequencing%20and%20Concordance%20Between%20Antimicrobial%20Susceptibility%20Genotypes%20and%20Phenotypes%20of%20Bacterial%20Isolates%20Associated%20with%20Bovine%20Respiratory%20Disease" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/7/9/3059&amp;t=Whole-Genome%20Sequencing%20and%20Concordance%20Between%20Antimicrobial%20Susceptibility%20Genotypes%20and%20Phenotypes%20of%20Bacterial%20Isolates%20Associated%20with%20Bovine%20Respiratory%20Disease" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/7/9/3059&amp;title=Whole-Genome%20Sequencing%20and%20Concordance%20Between%20Antimicrobial%20Susceptibility%20Genotypes%20and%20Phenotypes%20of%20Bacterial%20Isolates%20Associated%20with%20Bovine%20Respiratory%20Disease" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/7/9/3059&amp;title=Whole-Genome%20Sequencing%20and%20Concordance%20Between%20Antimicrobial%20Susceptibility%20Genotypes%20and%20Phenotypes%20of%20Bacterial%20Isolates%20Associated%20with%20Bovine%20Respiratory%20Disease" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/486354/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="486354" id="node486354--3" data-pisa="ggg;7/9/3059" data-pisa-master="ggg;g3.117.1137" data-apath="/ggg/7/9/3059.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Whole-Genome Sequencing and Concordance Between Antimicrobial Susceptibility Genotypes and Phenotypes of Bacterial Isolates Associated with Bovine Respiratory Disease</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-7821-3012" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Joseph R. Owen</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-6149-1008" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Noelle Noyes</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-4181-4233" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Amy E. Young</span>, <span class="highwire-citation-author" data-delta="3">Daniel J. Prince</span>, <span class="highwire-citation-author" data-delta="4">Patricia C. Blanchard</span>, <span class="highwire-citation-author" data-delta="5">Terry W. Lehenbauer</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0003-0330-5013" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Sharif S. Aly</span>, <span class="highwire-citation-author" data-delta="7">Jessica H. Davis</span>, <span class="highwire-citation-author" data-delta="8">Sean M. O’Rourke</span>, <span class="highwire-citation-author" data-delta="9">Zaid Abdo</span>, <span class="highwire-citation-author" data-delta="10">Keith Belk</span>, <span class="highwire-citation-author" data-delta="11">Michael R. Miller</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="12"><a href="http://orcid.org/0000-0001-8138-2714" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Paul Morley</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="13"><a href="http://orcid.org/0000-0003-1562-162X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Alison L. Van Eenennaam</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3059-3071;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.1137 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node486354--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Joseph R. Owen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Animal Science, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joseph%2BR.%2BOwen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Owen%20JR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJoseph%2BR.%2BOwen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-7821-3012" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Joseph R. Owen</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Noelle Noyes </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Microbiology, Immunology, and Pathology, College of Veterinary Medicine and Biomedical Sciences, Colorado State University, Fort Collins, Colorado 80523</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Noelle%2BNoyes%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Noyes%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ANoelle%2BNoyes%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-6149-1008" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Noelle Noyes</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Amy E. Young </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Animal Science, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Amy%2BE.%2BYoung%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Young%20AE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAmy%2BE.%2BYoung%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-4181-4233" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Amy E. Young</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Daniel J. Prince </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Animal Science, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniel%2BJ.%2BPrince%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Prince%20DJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADaniel%2BJ.%2BPrince%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Patricia C. Blanchard </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>California Animal Health and Food Safety Laboratory, School of Veterinary Medicine, University of California, Davis, Tulare, California 93274</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Patricia%2BC.%2BBlanchard%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Blanchard%20PC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APatricia%2BC.%2BBlanchard%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Terry W. Lehenbauer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Veterinary Medicine Teaching and Research Center, School of Veterinary Medicine, University of California, Davis, Tulare, California 93274</div><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Population Health and Reproduction, School of Veterinary Medicine, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Terry%2BW.%2BLehenbauer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lehenbauer%20TW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATerry%2BW.%2BLehenbauer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Sharif S. Aly </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Veterinary Medicine Teaching and Research Center, School of Veterinary Medicine, University of California, Davis, Tulare, California 93274</div><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Population Health and Reproduction, School of Veterinary Medicine, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sharif%2BS.%2BAly%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Aly%20SS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASharif%2BS.%2BAly%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-0330-5013" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Sharif S. Aly</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Jessica H. Davis </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Veterinary Medicine Teaching and Research Center, School of Veterinary Medicine, University of California, Davis, Tulare, California 93274</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jessica%2BH.%2BDavis%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Davis%20JH&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJessica%2BH.%2BDavis%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Sean M. O’Rourke </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Animal Science, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sean%2BM.%2BO%E2%80%99Rourke%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=O%E2%80%99Rourke%20SM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASean%2BM.%2BO%25E2%2580%2599Rourke%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Zaid Abdo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Microbiology, Immunology, and Pathology, College of Veterinary Medicine and Biomedical Sciences, Colorado State University, Fort Collins, Colorado 80523</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zaid%2BAbdo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Abdo%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AZaid%2BAbdo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Keith Belk </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>Department of Animal Sciences, College of Agricultural Sciences, Colorado State University, Fort Collins, Colorado 80523</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Keith%2BBelk%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Belk%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKeith%2BBelk%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Michael R. Miller </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Animal Science, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BR.%2BMiller%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Miller%20MR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichael%2BR.%2BMiller%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-name">Paul Morley </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>Department of Clinical Sciences, College of Veterinary Medicine and Biomedical Sciences, Colorado State University, Fort Collins, Colorado 80523</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Paul%2BMorley%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Morley%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APaul%2BMorley%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-8138-2714" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Paul Morley</a></li></ul></div><div class="author-tooltip-13"><div class="author-tooltip-name">Alison L. Van Eenennaam </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Animal Science, University of California, Davis, California 95616</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Veterinary Medicine Teaching and Research Center, School of Veterinary Medicine, University of California, Davis, Tulare, California 93274</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alison%2BL.%2BVan%2BEenennaam%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Van%20Eenennaam%20AL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAlison%2BL.%2BVan%2BEenennaam%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-1562-162X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Alison L. Van Eenennaam</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:alvaneenennaam@ucdavis.edu" class="" data-icon-position="" data-hide-link-title="0">alvaneenennaam@ucdavis.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=3059&amp;rft.epage=3071&amp;rft.atitle=Whole-Genome%20Sequencing%20and%20Concordance%20Between%20Antimicrobial%20Susceptibility%20Genotypes%20and%20Phenotypes%20of%20Bacterial%20Isolates%20Associated%20with%20Bovine%20Respiratory%20Disease&amp;rft.volume=7&amp;rft.issue=9&amp;rft.date=2017-09-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Owen%2C+Joseph+R.&amp;rft.au=Noyes%2C+Noelle&amp;rft.au=Young%2C+Amy+E.&amp;rft.au=Prince%2C+Daniel+J.&amp;rft.au=Blanchard%2C+Patricia+C.&amp;rft.au=Lehenbauer%2C+Terry+W.&amp;rft.au=Aly%2C+Sharif+S.&amp;rft.au=Davis%2C+Jessica+H.&amp;rft.au=O%E2%80%99Rourke%2C+Sean+M.&amp;rft.au=Abdo%2C+Zaid&amp;rft.au=Belk%2C+Keith&amp;rft.au=Miller%2C+Michael+R.&amp;rft.au=Morley%2C+Paul&amp;rft.au=Van+Eenennaam%2C+Alison+L."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/486354/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/486354/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/486354/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/486354/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/486354/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/486354/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/486354/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/486354/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/486354/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/486354/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/486354/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/486354/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/7/9/3059" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-12" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-20" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/7/9/3059.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/7/9/3059.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/7/9/3059.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/7/9/3059.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
