<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/7/9/2919" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/7/9/2919.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/7/9/2919.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/7/9/2919.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/7/9/2919/embed/mml-math-5.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/7/9/2919.atom" />
<meta name="HW.pisa" content="ggg;7/9/2919" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Habitat Predicts Levels of Genetic Admixture in Saccharomyces cerevisiae" />
<meta name="DC.Identifier" content="10.1534/g3.117.041806" />
<meta name="DC.Date" content="2017-09-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2017 Tilakaratna and Bensasson. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Genetic admixture can provide material for populations to adapt to local environments, and this process has played a crucial role in the domestication of plants and animals. The model yeast, Saccharomyces cerevisiae , has been domesticated multiple times for the production of wine, sake, beer, and bread, but the high rate of admixture between yeast lineages has so far been treated as a complication for population genomic analysis. Here, we make use of the low recombination rate at centromeres to investigate admixture in yeast using a classic Bayesian approach and a locus-by-locus phylogenetic approach. Using both approaches, we find that S. cerevisiae from stable oak woodland habitats are less likely to show recent genetic admixture compared with those isolated from transient habitats such as fruits, wine, or human infections. When woodland yeast strains do show recent genetic admixture, the degree of admixture is lower than in strains from other habitats. Furthermore, S. cerevisiae populations from oak woodlands are genetically isolated from each other, with only occasional migration between woodlands and local fruit habitats. Application of the phylogenetic approach suggests that there is a previously undetected population in North Africa that is the closest outgroup to the European S. cerevisiae , including the domesticated Wine population. Careful testing for admixture in S. cerevisiae leads to a better understanding of the underlying population structure of the species and will be important for understanding the selective processes underlying domestication in this economically important species." />
<meta name="DC.Contributor" content="Viranga Tilakaratna" />
<meta name="DC.Contributor" content="Douda Bensasson" />
<meta name="article:published_time" content="2017-09-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Habitat Predicts Levels of Genetic Admixture in Saccharomyces cerevisiae" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Genetic admixture can provide material for populations to adapt to local environments, and this process has played a crucial role in the domestication of plants and animals. The model yeast, &lt;i&gt;Saccharomyces cerevisiae&lt;/i&gt;, has been domesticated multiple times for the production of wine, sake, beer, and bread, but the high rate of admixture between yeast lineages has so far been treated as a complication for population genomic analysis. Here, we make use of the low recombination rate at centromeres to investigate admixture in yeast using a classic Bayesian approach and a locus-by-locus phylogenetic approach. Using both approaches, we find that &lt;i&gt;S. cerevisiae&lt;/i&gt; from stable oak woodland habitats are less likely to show recent genetic admixture compared with those isolated from transient habitats such as fruits, wine, or human infections. When woodland yeast strains do show recent genetic admixture, the degree of admixture is lower than in strains from other habitats. Furthermore, &lt;i&gt;S. cerevisiae&lt;/i&gt; populations from oak woodlands are genetically isolated from each other, with only occasional migration between woodlands and local fruit habitats. Application of the phylogenetic approach suggests that there is a previously undetected population in North Africa that is the closest outgroup to the European &lt;i&gt;S. cerevisiae&lt;/i&gt;, including the domesticated Wine population. Careful testing for admixture in &lt;i&gt;S. cerevisiae&lt;/i&gt; leads to a better understanding of the underlying population structure of the species and will be important for understanding the selective processes underlying domestication in this economically important species.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2017/09/01" />
<meta name="citation_mjid" content="ggg;7/9/2919" />
<meta name="citation_id" content="7/9/2919" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/7/9/2919" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/7/9/2919.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/7/9/2919.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/7/9/2919.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.117.041806" />
<meta name="citation_pmid" content="28696926" />
<meta name="citation_volume" content="7" />
<meta name="citation_issue" content="9" />
<meta name="citation_num_pages" content="11" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="2919" />
<meta name="citation_lastpage" content="2929" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Viranga Tilakaratna" />
<meta name="citation_author_institution" content="Faculty of Life Sciences, University of Manchester, M13 9PT, UK" />
<meta name="citation_author" content="Douda Bensasson" />
<meta name="citation_author_institution" content="Department of Plant Biology, University of Georgia, Athens, Georgia 30602" />
<meta name="citation_author_institution" content="Institute of Bioinformatics, University of Georgia, Athens, Georgia 30602" />
<meta name="citation_author_email" content="dbensasson@uga.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-9015-2707" />
<meta name="citation_reference" content="citation_journal_title=Nat. Commun.;citation_author=P. Almeida;citation_author=C. Gonçalves;citation_author=S. Teixeira;citation_author=D. Libkind;citation_author=M. Bontrager;citation_title=A Gondwanan imprint on global diversity and domestication of wine and cider yeast Saccharomyces uvarum.;citation_pages=4044;citation_volume=5;citation_year=2014;citation_pmid=24887054" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=P. Almeida;citation_author=R. Barbosa;citation_author=P. Zalar;citation_author=Y. Imanishi;citation_author=K. Shimizu;citation_title=A population genomics insight into the Mediterranean origins of wine yeast domestication.;citation_pages=5412-5427;citation_volume=24;citation_year=2015;citation_doi=10.1111/mec.13341" />
<meta name="citation_reference" content="citation_author=D. Amberg;citation_author=D. Burke;citation_author=J. Strathern;citation_year=2005" />
<meta name="citation_reference" content="citation_title=Molecular Markers, Natural History and Evolution;citation_author=J. Avise;citation_year=1994" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol. Evol.;citation_author=R. Barbosa;citation_author=P. Almeida;citation_author=S. V. B. Safar;citation_author=R. O. Santos;citation_author=P. B. Morais;citation_title=Evidence of natural hybridization in Brazilian wild lineages of Saccharomyces cerevisiae.;citation_volume=8;citation_year=2016;citation_pmid=26782936;citation_doi=10.1093/gbe/evv263" />
<meta name="citation_reference" content="citation_journal_title=Funct. Ecol.;citation_author=M. Beekman;citation_author=F. L. W. Ratnieks;citation_title=Long-range foraging by the honey-bee, Apis mellifera L.;citation_pages=490-496;citation_volume=14;citation_year=2000" />
<meta name="citation_reference" content="citation_journal_title=BMC evolutionary biology [electronic resource];citation_journal_abbrev=BMC Evol Biol;citation_author=D. Bensasson;citation_title=Evidence for a high mutation rate at rapidly evolving yeast centromeres.;citation_pages=211-211;citation_volume=11;citation_year=2011;citation_pmid=21767380;citation_doi=10.1186/1471-2148-11-211" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=D. Bensasson;citation_author=M. Zarowiecki;citation_author=A. Burt;citation_author=V. Koufopanou;citation_title=Rapid Evolution of Yeast Centromeres in the Absence of Drive;citation_pages=2161-2167;citation_volume=178;citation_year=2008;citation_issue=4;citation_pmid=18430941;citation_doi=10.1534/genetics.107.083980" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=J. K. Bonfield;citation_author=K. F. Smith;citation_author=R. Staden;citation_title=A new DNA sequence assembly program;citation_pages=4992-4999;citation_volume=23;citation_year=1995;citation_issue=24;citation_pmid=8559656;citation_doi=10.1093/nar/23.24.4992" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=Y. Brandvain;citation_author=A. M. Kenney;citation_author=L. Flagel;citation_author=G. Coop;citation_author=A. L. Sweigart;citation_title=Speciation and introgression between Mimulus nasutus and Mimulus guttatus.;citation_pages=e1004410;citation_volume=10;citation_year=2014;citation_pmid=24967630;citation_doi=10.1371/journal.pgen.1004410" />
<meta name="citation_reference" content="citation_journal_title=Ecol. Lett.;citation_author=C. C. Buser;citation_author=R. D. Newcomb;citation_author=A. C. Gaskett;citation_author=M. R. Goddard;citation_title=Niche construction initiates the evolution of mutualistic interactions.;citation_pages=1257-1264;citation_volume=17;citation_year=2014;citation_pmid=25041133;citation_doi=10.1111/ele.12331" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=J. M. Cherry;citation_author=E. L. Hong;citation_author=C. Amundsen;citation_author=R. Balakrishnan;citation_author=G. Binkley;citation_title=Saccharomyces genome database: the genomics resource of budding yeast.;citation_pages=D700-D705;citation_volume=40;citation_year=2011;citation_pmid=22110037" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=R. Corbett-Detig;citation_author=R. Nielsen;citation_title=A hidden Markov model approach for simultaneously estimating local ancestry and admixture time using next generation sequence data in samples of arbitrary ploidy.;citation_pages=e1006529;citation_volume=13;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Am. Nat.;citation_author=J. A. Coyne;citation_author=B. Milstead;citation_title=Long-distance migration of Drosophila. 3. Dispersal of D. melanogaster Alleles from a Maryland Orchard.;citation_pages=70-82;citation_volume=130;citation_year=1987;citation_doi=10.1086/284698" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=G. A. Cromie;citation_author=K. E. Hyma;citation_author=C. L. Ludlow;citation_author=C. Garmendia-Torres;citation_author=T. L. Gilbert;citation_title=Genomic sequence diversity and population structure of Saccharomyces cerevisiae assessed by RAD-seq.;citation_pages=2163-2171;citation_volume=3;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=FEMS Yeast Res.;citation_author=F. A. Cubillos;citation_author=E. J. Louis;citation_author=G. Liti;citation_title=Generation of a large set of genetically tractable haploid and diploid Saccharomyces strains.;citation_volume=9;citation_year=2009;citation_pmid=19840116;citation_doi=10.1111/j.1567-1364.2009.00583.x" />
<meta name="citation_reference" content="citation_journal_title=Front. Microbiol.;citation_author=S. Dashko;citation_author=P. Liu;citation_author=H. Volk;citation_author=L. Butinar;citation_author=J. Piškur;citation_title=Changes in the relative abundance of two Saccharomyces species from Oak forests to wine fermentations.;citation_pages=215;citation_volume=7;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=S. Diezmann;citation_author=F. S. Dietrich;citation_title=Saccharomyces cerevisiae: population divergence and resistance to oxidative stress in clinical, domesticated and wild isolates.;citation_pages=e5317-e5317;citation_volume=4;citation_year=2009;citation_issue=4;citation_pmid=19390633;citation_doi=10.1371/journal.pone.0005317" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=S. Doniger;citation_author=H. Kim;citation_author=D. Swain;citation_author=D. Corcuera;citation_author=M. Williams;citation_title=A catalog of neutral and deleterious polymorphism in yeast.;citation_pages=e1000183-e1000183;citation_volume=4;citation_year=2008;citation_issue=8;citation_pmid=18769710;citation_doi=10.1371/journal.pgen.1000183" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=C. Eberlein;citation_author=J. B. Leducq;citation_author=C. R. Landry;citation_title=The genomics of wild yeast populations sheds light on the domestication of man’s best (micro) friend.;citation_pages=5309-5311;citation_volume=24;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Clin. Infect. Dis.;citation_author=A. Enache-Angoulvant;citation_author=C. Hennequin;citation_title=Invasive Saccharomyces infection: a comprehensive review.;citation_volume=41;citation_year=2005;citation_pmid=16267727;citation_doi=10.1086/497832" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=G. Evanno;citation_author=S. Regnaut;citation_author=J. Goudet;citation_title=Detecting the number of clusters of individuals using the software STRUCTURE: a simulation study.;citation_pages=2611-2620;citation_volume=14;citation_year=2005;citation_issue=8;citation_pmid=15969739;citation_doi=10.1111/j.1365-294X.2005.02553.x" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=D. Falush;citation_author=M. Stephens;citation_author=J. K. Pritchard;citation_title=Inference of Population Structure Using Multilocus Genotype Data: Linked Loci and Correlated Allele Frequencies;citation_pages=1567-1587;citation_volume=164;citation_year=2003;citation_issue=4;citation_pmid=12930761" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=J. C. Fay;citation_author=J. A. Benavides;citation_title=Evidence for domesticated and wild populations of Saccharomyces cerevisiae.;citation_pages=e5;citation_volume=1;citation_year=2005;citation_doi=10.1371/journal.pgen.0010005" />
<meta name="citation_reference" content="citation_journal_title=Molecular Biology and Evolution;citation_journal_abbrev=Mol Biol Evol;citation_author=J. Felsenstein;citation_author=G. A. Churchill;citation_title=A Hidden Markov Model approach to variation among sites in rate of evolution;citation_pages=93-104;citation_volume=13;citation_year=1996;citation_issue=1;citation_pmid=8583911;citation_doi=10.1093/oxfordjournals.molbev.a025575" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_author=B. Gallone;citation_author=J. Steensels;citation_author=T. Prahl;citation_author=L. Soriaga;citation_author=V. Saels;citation_title=Domestication and divergence of Saccharomyces cerevisiae Beer yeasts.;citation_pages=1397-1410.e16;citation_volume=166;citation_year=2016;citation_pmid=27610566;citation_doi=10.1016/j.cell.2016.08.020" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=V. Gayevskiy;citation_author=S. Klaere;citation_author=S. Knight;citation_author=M. R. Goddard;citation_title=ObStruct: a method to objectively analyse factors driving population structure using Bayesian ancestry profiles.;citation_pages=e85196;citation_volume=9;citation_year=2014;citation_pmid=24416362;citation_doi=10.1371/journal.pone.0085196" />
<meta name="citation_reference" content="citation_journal_title=FEMS Yeast Res.;citation_author=M. R. Goddard;citation_author=D. Greig;citation_title=Saccharomyces cerevisiae: a nomadic yeast with no niche?;citation_volume=15;citation_year=2015;citation_pmid=25725024;citation_doi=10.1093/femsyr/fov009" />
<meta name="citation_reference" content="citation_journal_title=Environmental Microbiology (Print);citation_journal_abbrev=Environmental Microbiology (Print);citation_author=M. R. Goddard;citation_author=N. Anfang;citation_author=R. Tang;citation_author=R. C. Gardner;citation_author=C. Jun;citation_title=A distinct population of Saccharomyces cerevisiae in New Zealand: evidence for local dispersal by insects and human-aided global dispersal in oak barrels.;citation_pages=63-73;citation_volume=12;citation_year=2010;citation_issue=1;citation_pmid=19691498;citation_doi=10.1111/j.1462-2920.2009.02035.x" />
<meta name="citation_reference" content="citation_journal_title=Curr. Biol.;citation_author=M. Gonçalves;citation_author=A. Pontes;citation_author=P. Almeida;citation_author=R. Barbosa;citation_author=M. Serra;citation_title=Distinct domestication trajectories in top-fermenting beer yeasts and wine yeasts.;citation_pages=2750-2761;citation_volume=26;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=M. Gouy;citation_author=S. Guindon;citation_author=O. Gascuel;citation_title=SeaView Version 4: a multiplatform graphical user interface for sequence alignment and phylogenetic tree building.;citation_volume=27;citation_year=2010;citation_pmid=19854763;citation_doi=10.1093/molbev/msp259" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=K. Harris;citation_author=R. Nielsen;citation_title=The genetic cost of Neanderthal introgression.;citation_volume=203;citation_year=2016;citation_pmid=27038113;citation_doi=10.1534/genetics.116.186890" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol. Resour.;citation_author=M. J. Hubisz;citation_author=D. Falush;citation_author=M. Stephens;citation_author=J. K. Pritchard;citation_title=Inferring weak population structure with the assistance of sample group information.;citation_pages=1322-1332;citation_volume=9;citation_year=2009;citation_pmid=21564903;citation_doi=10.1111/j.1755-0998.2009.02591.x" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=M. B. Hufford;citation_author=P. Lubinksy;citation_author=T. Pyhäjärvi;citation_author=M. T. Devengenzo;citation_author=N. C. Ellstrand;citation_title=The genomic signature of crop-wild introgression in Maize.;citation_pages=e1003477;citation_volume=9;citation_year=2013;citation_pmid=23671421;citation_doi=10.1371/journal.pgen.1003477" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=K. E. Hyma;citation_author=J. C. Fay;citation_title=Mixing of vineyard and oak-tree ecotypes of Saccharomyces cerevisiae in North American vineyards.;citation_pages=2917-2930;citation_volume=22;citation_year=2013;citation_doi=10.1111/mec.12155" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=M. Kellis;citation_author=N. Patterson;citation_author=M. Endrizzi;citation_author=B. Birren;citation_author=E. S. Lander;citation_title=Sequencing and comparison of yeast species to identify genes and regulatory elements.;citation_pages=241-254;citation_volume=423;citation_year=2003;citation_issue=6937;citation_pmid=12748633;citation_doi=10.1038/nature01644" />
<meta name="citation_reference" content="citation_journal_title=FEMS Yeast Res.;citation_author=S. J. Knight;citation_author=M. R. Goddard;citation_title=Sporulation in soil as an overwinter survival strategy in Saccharomyces cerevisiae.;citation_volume=16;citation_year=2016;citation_pmid=26568201;citation_doi=10.1093/femsyr/fov102" />
<meta name="citation_reference" content="citation_journal_title=Philosophical Transactions of the Royal Society B: Biological Sciences;citation_journal_abbrev=Phil Trans R Soc B;citation_author=V. Koufopanou;citation_author=J. Hughes;citation_author=G. Bell;citation_author=A. Burt;citation_title=The spatial scale of genetic differentiation in a model organism: the wild yeast Saccharomyces paradoxus;citation_pages=1941-1946;citation_volume=361;citation_year=2006;citation_issue=1475;citation_pmid=17028086;citation_doi=10.1098/rstb.2006.1922" />
<meta name="citation_reference" content="citation_journal_title=Nat. Microbiol.;citation_author=J. B. Leducq;citation_author=L. Nielly-Thibault;citation_author=G. Charron;citation_author=C. Eberlein;citation_author=J. P. Verta;citation_title=Speciation driven by hybridization and chromosomal plasticity in a wild yeast.;citation_pages=15003;citation_volume=1;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=G. Liti;citation_author=D. M. Carter;citation_author=A. M. Moses;citation_author=J. Warringer;citation_author=L. Parts;citation_title=Population genomics of domestic and wild yeasts.;citation_pages=337-341;citation_volume=458;citation_year=2009;citation_issue=7236;citation_pmid=19212322;citation_doi=10.1038/nature07743" />
<meta name="citation_reference" content="citation_journal_title=Curr. Biol.;citation_author=C. L. Ludlow;citation_author=G. A. Cromie;citation_author=C. Garmendia-Torres;citation_author=A. Sirr;citation_author=M. Hays;citation_title=Independent origins of yeast associated with coffee and cacao fermentation.;citation_pages=965-971;citation_volume=26;citation_year=2016;citation_pmid=27020745;citation_doi=10.1016/j.cub.2016.02.012" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=F. B. Marshall;citation_author=K. Dobney;citation_author=T. Denham;citation_author=J. M. Capriles;citation_title=Evaluating the roles of directed breeding and gene flow in animal domestication.;citation_volume=111;citation_year=2014;citation_pmid=24753599;citation_doi=10.1073/pnas.1312984110" />
<meta name="citation_reference" content="citation_journal_title=Oecologia;citation_author=L. Nunney;citation_title=The colonization of oranges by the Cosmopolitan Drosophila.;citation_pages=552-561;citation_volume=108;citation_year=1996;citation_doi=10.1007/BF00333733" />
<meta name="citation_reference" content="citation_author=E. Paradis;citation_year=2011" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=J. E. Pool;citation_author=R. B. Corbett-Detig;citation_author=R. P. Sugino;citation_author=K. A. Stevens;citation_author=C. M. Cardeno;citation_title=Population genomics of Sub-Saharan Drosophila melanogaster: African diversity and Non-African admixture.;citation_pages=e1003080;citation_volume=8;citation_year=2012;citation_pmid=23284287;citation_doi=10.1371/journal.pgen.1003080" />
<meta name="citation_reference" content="citation_journal_title=Journal of Molecular Evolution;citation_journal_abbrev=Journal of Molecular Evolution;citation_author=D. Posada;citation_author=K. A. Crandall;citation_title=The effect of recombination on the accuracy of phylogeny estimation.;citation_pages=396-402;citation_volume=54;citation_year=2002;citation_issue=3;citation_pmid=11847565;citation_doi=10.1007/s00239-001-0034-9" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. K. Pritchard;citation_author=M. Stephens;citation_author=P. Donnelly;citation_title=Inference of Population Structure Using Multilocus Genotype Data;citation_pages=945-959;citation_volume=155;citation_year=2000;citation_issue=2;citation_pmid=10835412;citation_doi=10.1111/j.1471-8286.2005.01155.x" />
<meta name="citation_reference" content="citation_journal_title=Current biology :  CB;citation_journal_abbrev=Curr Biol;citation_author=M. Reuter;citation_author=G. Bell;citation_author=D. Greig;citation_title=Increased outbreeding in yeast in response to dispersal by an insect vector.;citation_pages=R81-R83;citation_volume=17;citation_year=2007;citation_issue=3;citation_pmid=17276903;citation_doi=10.1016/j.cub.2006.11.059" />
<meta name="citation_reference" content="citation_journal_title=Ecol. Evol.;citation_author=H. A. Robinson;citation_author=A. Pinharanda;citation_author=D. Bensasson;citation_title=Summer temperature can predict the distribution of wild yeast populations.;citation_pages=1236-1250;citation_volume=6;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Genomics;citation_author=F. Rosenzweig;citation_author=G. Sherlock;citation_title=Editorial: experimental evolution: prospects and challenges.;citation_pages=v-vi;citation_volume=104;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Molecular Biology and Evolution;citation_journal_abbrev=Mol Biol Evol;citation_author=N. Saitou;citation_author=M. Nei;citation_title=The neighbor-joining method: a new method for reconstructing phylogenetic trees;citation_pages=406-425;citation_volume=4;citation_year=1987;citation_issue=4;citation_pmid=3447015;citation_doi=10.1099/ijs.0.01472-0" />
<meta name="citation_reference" content="citation_journal_title=Appl. Environ. Microbiol.;citation_author=J. P. Sampaio;citation_author=P. Gonçalves;citation_title=Natural populations of Saccharomyces kudriavzevii in Portugal are associated with Oak Bark and are sympatric with S. cerevisiae and S. paradoxus.;citation_volume=74;citation_year=2008;citation_pmid=18281431;citation_doi=10.1128/AEM.02396-07" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=S. Sankararaman;citation_author=S. Mallick;citation_author=M. Dannemann;citation_author=K. Prüfer;citation_author=J. Kelso;citation_title=The genomic landscape of Neanderthal ancestry in present-day humans.;citation_pages=354-357;citation_volume=507;citation_year=2014;citation_pmid=24476815;citation_doi=10.1038/nature12961" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=D. R. Scannell;citation_author=O. A. Zill;citation_author=A. Rokas;citation_author=C. Payen;citation_author=M. J. Dunham;citation_title=The awesome power of yeast evolutionary genetics: new genome sequences and strain resources for the Saccharomyces sensu stricto Genus.;citation_pages=11-25;citation_volume=1;citation_year=2011;citation_pmid=22384314;citation_doi=10.1534/g3.111.000273" />
<meta name="citation_reference" content="citation_journal_title=Conserv. Genet.;citation_author=M. K. Schwartz;citation_author=K. S. McKelvey;citation_title=Why sampling scheme matters: the effect of sampling scheme on landscape genetic results.;citation_pages=441;citation_volume=10;citation_year=2009;citation_doi=10.1007/s10592-008-9622-1" />
<meta name="citation_reference" content="citation_journal_title=FEMS Yeast Res.;citation_author=P. D. Sniegowski;citation_author=P. G. Dombrowski;citation_author=E. Fingerman;citation_title=Saccharomyces cerevisiae and Saccharomyces paradoxus coexist in a natural woodland site in North America and display different levels of reproductive isolation from European conspecifics.;citation_volume=1;citation_year=2002;citation_pmid=12702333;citation_doi=10.1111/j.1567-1364.2002.tb00048.x" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. Stamatakis;citation_title=RAxML Version 8: a tool for phylogenetic analysis and post-analysis of large phylogenies.;citation_volume=30;citation_year=2014;citation_pmid=24451623;citation_doi=10.1093/bioinformatics/btu033" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=I. Stefanini;citation_author=L. Dapporto;citation_author=J. L. Legras;citation_author=A. Calabretta;citation_author=M. Di Paola;citation_title=Role of social wasps in Saccharomyces cerevisiae ecology and evolution.;citation_volume=109;citation_year=2012;citation_pmid=22847440;citation_doi=10.1073/pnas.1208362109" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=Q. M. Wang;citation_author=W. Q. Liu;citation_author=G. Liti;citation_author=S. A. Wang;citation_author=F. Y. Bai;citation_title=Surprisingly diverged populations of Saccharomyces cerevisiae in natural environments remote from human activity.;citation_pages=5404-5417;citation_volume=21;citation_year=2012;citation_issue=22;citation_pmid=22913817;citation_doi=10.1111/j.1365-294X.2012.05732.x" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=C. W. Whitfield;citation_author=S. K. Behura;citation_author=S. H. Berlocher;citation_author=A. G. Clark;citation_author=J. S. Johnston;citation_title=Thrice Out of Africa: Ancient and Recent Expansions of the Honey Bee, Apis mellifera;citation_pages=642-645;citation_volume=314;citation_year=2006;citation_issue=5799;citation_pmid=17068261;citation_doi=10.1126/science.1132772" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Habitat Predicts Levels of Genetic Admixture in Saccharomyces cerevisiae" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9.cover-source.jpg" />
<meta name="twitter:description" content="Genetic admixture can provide material for populations to adapt to local environments, and this process has played a crucial role in the domestication of plants and animals. The model yeast, Saccharomyces cerevisiae , has been domesticated multiple times for the production of wine, sake, beer, and bread, but the high rate of admixture between yeast lineages has so far been treated as a complication for population genomic analysis. Here, we make use of the low recombination rate at centromeres to investigate admixture in yeast using a classic Bayesian approach and a locus-by-locus phylogenetic approach. Using both approaches, we find that S. cerevisiae from stable oak woodland habitats are less likely to show recent genetic admixture compared with those isolated from transient habitats such as fruits, wine, or human infections. When woodland yeast strains do show recent genetic admixture, the degree of admixture is lower than in strains from other habitats. Furthermore, S. cerevisiae populations from oak woodlands are genetically isolated from each other, with only occasional migration between woodlands and local fruit habitats. Application of the phylogenetic approach suggests that there is a previously undetected population in North Africa that is the closest outgroup to the European S. cerevisiae , including the domesticated Wine population. Careful testing for admixture in S. cerevisiae leads to a better understanding of the underlying population structure of the species and will be important for understanding the selective processes underlying domestication in this economically important species." />
<meta name="og-title" property="og:title" content="Habitat Predicts Levels of Genetic Admixture in Saccharomyces cerevisiae" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/7/9/2919" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Genetic admixture can provide material for populations to adapt to local environments, and this process has played a crucial role in the domestication of plants and animals. The model yeast, Saccharomyces cerevisiae , has been domesticated multiple times for the production of wine, sake, beer, and bread, but the high rate of admixture between yeast lineages has so far been treated as a complication for population genomic analysis. Here, we make use of the low recombination rate at centromeres to investigate admixture in yeast using a classic Bayesian approach and a locus-by-locus phylogenetic approach. Using both approaches, we find that S. cerevisiae from stable oak woodland habitats are less likely to show recent genetic admixture compared with those isolated from transient habitats such as fruits, wine, or human infections. When woodland yeast strains do show recent genetic admixture, the degree of admixture is lower than in strains from other habitats. Furthermore, S. cerevisiae populations from oak woodlands are genetically isolated from each other, with only occasional migration between woodlands and local fruit habitats. Application of the phylogenetic approach suggests that there is a previously undetected population in North Africa that is the closest outgroup to the European S. cerevisiae , including the domesticated Wine population. Careful testing for admixture in S. cerevisiae leads to a better understanding of the underlying population structure of the species and will be important for understanding the selective processes underlying domestication in this economically important species." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9.cover-source.jpg" />
    <title>Habitat Predicts Levels of Genetic Admixture in Saccharomyces cerevisiae | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"AHAa8-HlythSbm_O-pYOfzZpoSqgow2-K0Yz0JxKWIA","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"486369","apath":"\/ggg\/7\/9\/2919.atom","pisa":"ggg;7\/9\/2919","ac":{"\/ggg\/7\/9\/2919.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/7\/9\/2919.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;7\/9\/2919"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;7\/9\/2919"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/7\/9\/2919":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022GC4HMgPVzdQ71XgMvelnOJO6JZj1v3ANIiNdfJoOuZg6AYFRJyVCRpHRHyDkrdBcYuUBBWl5vQtsXbZCciQlJbufimUOmLRlZa32zBBv\\\\\\\/DzpDzCkT9+M+69PghRSwa1WmtoPBtoLY2IJif9y+qspbuxJ7yT2QA4tgEONqO8qtXLnBSx3P8u5zZJkoOhXJkMvnbU8yGdhDtDPydfapXYU4VWOTuEaqCWOEwTSB82wuB3NjJ\\\\\\\/U+lqv3lvgsNi7xoF0J6zEqKZI8dj5Jk2bkbdX83IDWWsfgkB220+qF5Xyt+OfBM5bBv\\\\\\\/T+AGDic32x0MmdjnL5OXuQzFkqjHe1gcVt3L3i6WvT1P0\\\\\\\/A7+EnZlUT5JALo0tFF5K\\\\\\\/rZVg9GmiovRKIthCppAI2DcsTtUDRbXVGE4PKBm0i7u33jtmmRY1849+VWY\\\\\\\/o6JRgC7U8npCG3H\\\\\\\/EupP\\\\\\\/LSoHSvEUA6UfxAbh6DhxchmtSyhQfnMA4Hq3tenW89zkJ16hnNzDfktJw3\\\\\\\/Q6dRDyc4sV2UiKpuuLHk6TY6fdu+L9N0wSbp3zCqGXOPhL2gGOTBLFaowD4Eh7a606MMI0akmIRjugRFOLYY9GCQoVjWerLoYkK+aIrNBPv0l4NNpFgN0icaVxNVS39pZ1XnRI4WT7Wzqm2r4fWpIz8+82tvbApk2cNE60G+BDszmNIlxr6r3uLa9\\\\\\\/mIt65ffbiEc2HKjr9HKHk852sNAlFO4s8\\\\\\\/PsJ\\\\\\\/ZXvC9xTH1X7gyumaXSri17w7AJIQtYsR5WuAKIbgBlS8czBjpD7qmYXJjZyMFjtdia0nCl7GCqWDItJ\\\\\\\/uPH8s3BBllnxrEy50YvPVBMSIDGByFdIAGPJdjgirUlVTrergQCAsAudH+4O9jtBL3lBjXYoQv0npzVbn7NhnsCCnQKybUsblo9s93G1pEvBIdnoT4DQBW5I9gIJuKZlMdSa88DUx9oGptNqJL0IbOVtZ6KABeaq1j6oxOostzUKwDytkGyBP9sElaOvU7RGWWRw\\\\\\\/in5FwvGMWRK2K8sHvtzV8+6h7P3nFprij2c5c4Ip8JhODzz2QKYnGXaxNztvD4OVjKFGsnnbX6mbySTXOILKsjW\\\\\\\/ETno9NW8jMhJRfnN+E\\\\\\\/jh4lX8qFZv9LMy\\\\\\\/f1ornI6ongv9zuKnAyLrua+0uKh+gfqON7zAZD8ev90lMCyoXB2pR3v4YWQZ\\\\\\\/ihMWczDKpNmRisxryWYmNwe224u8ens+r75hXfT0KEfBDF9YqsW+Vjvm6vtBWfax58otCYs+aDohZW4qCK\\\u0022,\\\u0022iv\\\u0022:\\\u0022L++pnfXqZkVQ44Oq+n\\\\\\\/GKw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00220834b0bba3e8afecfd58c05dec934e7fd58862d1fb54a0a556e17c3e631f384a\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022HVjyCK9FJfF6XLpXC9xSO6Rxh9fP6KyleIfbfB453qCNDa3su54SWwr9hr1ogROqLQniNYj3t2x4VES7zAgRIMkGU8scF2lZiMWeqLbQnk0l3n8nb2yBO3iR4EYF3O5ojeC+g\\\\\\\/iuE7S7QQY1LfQ2XiYyq22tn6VfmH96T0FoYjikrhG0g9d6Gx1E+cvrXLytZi7VdN631sPHQV65mLwlpsL1GSCmcA+iWGBscxYW3oF7PyPfY7kEjQgaJx4YEuUISdDlXZuwntpENzk2CrBNUWIz7GA3f7IoT2vDe1J+fsv5pYGxoRzsSX2DTG9I4dR1YcMKSI92Ls9AbHCXNfLq+NJK9OqHGdh+hmg5GG2rl8MV+jJIuPiPG5DkQUihHkK0KdnNH0wIKq7d9GJZBAI5dvg9rhoBWaKNcjxhTMOyIir6ZjfBkaeTFZNFHvvUgtRk47wkM7CtbzmccQ3HKCIK4eikeiYd4h2w4zkPzyTQZF7tYfb32CfOJ0oqVU1Bb4TkN48LfDENqy+4\\\\\\\/r+zxUbdtet6Z2F35MAqHKquEcspJj6llTTcTQ1d68s8oj0s8XKDaTLj9JbRD8Dp1sOxcdphth6O7tjb6GA4wzOLvl4ESol70OxKRWrCjYcKKKFM48GzP5p51YLJzGCzP\\\\\\\/XDotaMoZAQ6ZcG4hgQMqOv+pjz5nKJU13vClBBnar+9bImQS2zh1ikWjQFSt9A6hQ1Ev\\\\\\\/EtFKozVSMQhU0CtA5AmiWHTSxVsOga3gZQuYVAR1D2171YRaqUSLGkj0K5Qb\\\\\\\/FzQpS3VHmVazkqzvDePH9sK5tTMNqShZLonXl1WssdqolRbBBTyyqLUKdgQPZtq4new5d8ZPu2+924xrPVx\\\\\\\/8YnE\\\\\\\/fG2yqZJXDQBZssf6rxOTXCbw9qWwDPahsQkNpSr4SYe6jyO5PdbT2EYC\\\\\\\/dDj3pvHF\\\\\\\/eGLONZ4bAxOKWKLfOPI9rAExKnyzuAbenwPdpDvEcLxjf34P0+yd\\\\\\\/Z7jqywBuPxeXIx6aJBuqqRs9LCmHwlp4CSkiyU1LFgW\\\\\\\/CX\\\\\\\/rT3kJSNBrAzU3BYtdElk5aE8quW0rO5pCIo7kkxZSmTzwBcLk4gln1PIdjUNkj\\\\\\\/Lmlr25gmouk+P3zj7gCpgz\\\\\\\/kmqoT3lnTBrHcvj\\\\\\\/mUr8AcZUTc7495+6TmRrZNwEOFBdTxj+IbteoED4UOTjviLDcyTSBSvn4XUBOCRAI+3\\\\\\\/i09VljX+i6BG+ED9kXZq78uN1s9\\\\\\\/BUJO8WOfh\\\\\\\/BfP99jmucWbbISkAnOrakfJeNLbnqM7QQN9WGBqBLp9lIyVBBXb0M3btzt4qtSLXF\\\\\\\/BdcfuzDY3ZiX\\\\\\\/s1iIUPWv7lYTOu08VAwP7yt0+Vq4ESi1puveLT855Y8uhyAfhWH2Aw\\\\\\\/4aJ9z3o0gWgbOG30TaCH8ZxE5uANR4FMS70wIz4aTakWUQ3pjFCjHySQmcEVmsiwIfm5bCd0oB7GzD6EhjrJzmKF5aNvvdtk4LCkNNfzq+S+8MpKNBuC2H57cUkMtcr\\\\\\\/BLWRh\\\\\\\/0nllN4V7vXeE9EPQBmfv2hkjqNq0fJtD7nTr2tkJUN0b485fGECnEfWkfhwhCSGKGRwfkh6gxbD1Cfs1FqZpINNFP4SDRKrEgCJKyPsWAGNt8Ugi1byHk30UBsd5WYUyaUb3o++CTAmOWp3faj1LyBWWyfUqGjdAQ0hwwAm4ZCw+FZlGNLKDCXqDTDhx+zCEjnTAJWKLShTxWcnGXaT9dQk5kTN99GlnpwWJiXnOt3UbpH1z0oTRYKwM9I95xFJntp5adl2ta47WU7ZkLYMHK1LZWsbl\\\\\\\/vXJ1jLZ8mA3TBVdkzQouMbqt5VeCW\\\\\\\/AMJlQTSylMr2tdPZMpGgkyqwp\\\\\\\/q6\\\\\\\/IBWdbMRnQMzqM4oJjxW3YsPy91pzKcKHjXIMkDjDAl4w1wz2iizH\\\\\\\/HnZtxTzGSASu8WW+hSlxekASKfOIZzJQxNPpM5N\\\\\\\/TGUmUyjoqNm+cDBqJAcDWa8K\\\\\\\/U3gayNFTgpAvODjAcZvLl4wk3+IcVJKcxKvSYNnqKZw6dlNezHT4Z6CYNSfNzs3kHun6SVBo6a4pM5rQckFCYiIhJSpL\\\\\\\/55NzpwjhinF6ES2cSauU1tPnYowIER3Fwb+PVwkUCuK1doPQOi4EkwYc7FYW\\\\\\\/mwdpzGVLrvIc\\\\\\\/gaxbdzAM7kaSI3WWi5aSmePVKiK8t6BAOXvG4KOCN7V66Nq\\\\\\\/O3FV+ExHDte+OfJtRpqD3KCaSSqAckbTA\\\\\\\/kqH\\\\\\\/BR5UwLf0N0jJuniBlA32POwez5Gg3ObR4dvNrExs49oQLV3vDObeZT58TPD66Ey4slVP5YjI7SRkw6VnA+6+ZUO7vBxIWg4u5VpGlcCX+c9MwvLo4XPHApV6j\\\\\\\/TTDg2kI85OccUTAiwYDCQUy0FDwj5siwT4oOhzgcHtJz0TCI8cCs3oMhFfvgWxV3AHj4IxjxhdY\\\\\\\/R\\\\\\\/jel7ZMfH9S+1U7goYDIomGKvBjfwCeHq5rEyDSCLYMB6bL0y+j5UeexikvdR1pF2fKOYLpGZudzqqzSVgfnDaG+RzJdUtzIG1iYeRrN8IpukTkbiCuebvWhbSbw8AbFJ\\\\\\\/oi4RqOO69xBZIprvpFG+UmjWo5lAW6o4DGX8Y9onup+1YfynuKgpAOB1XARZoujWgheCzAqRi5m6d68tB7u0Y2Dnhct+i5UfsAwQ7A19xR0hYaOizUmYYD3pCrmWT\\\\\\\/0p0XD1i3daweDH2tFsi5ZvtraAq6UO83nSSRYHWJBBZVAud8B7qZwqVimUXluVn27OCMB+IUk8BoN1FQS4GBz9vrZrTvgBW+\\\\\\\/FibzcU8Dfin0Zu8lIb5TRcCNVDLP0M0ZnKRjJSkZIdhH0aQZ7Q+RQdPtOovknyiF7AtCWKqS+5AaLrCppaR1o8d6lJl0eP2tzRaFucS\\\\\\\/giYTybgX\\\\\\\/pwKun2Pw+Nh2YeXoSkldYpERz5x2G66gWpky+ehL5VPWw069tWMkxppM0+fArkWTui36Ubx+qQrOHBvkFBpElvyPw5vkRjSfiOq9kmFy+6t3e6VX3kFwRMJMEr1tZ7sslX8Cxi9BWO63a71g2qDfYoCxvxnrzqN6fwaDFUO2NFey6q1ZTc5zRvmu++E6gI3Vdy16j7iBxdBkwk7wZdBopmxu\\\\\\\/0bSsDNp95PFHjEMPyPF7yehV1Ztoq\\\\\\\/AKxf98OtjTiQNOP000GmzoexQoBJQ97XuNyI8XJtaRAWDQUzZQGH+VBAGX6mGXZMmXmM\\\\\\\/CdbDeNKLBifITOcMzM\\\\\\\/nroCroEazLR8gWs6s5qSVIxYiQo5mqfTTNXbK3Wtkd\\\\\\\/UYeIrYd5MR8QjlH3ypfimhHTQdzirW0UdsDVsPHg+1HQ8RyBiI7pMVey6kmVUNoF4RKTrqTKcBaKIPiSLPLk6j9PkwqZ59Fr4fNXB14fRiKB0lbuvwXiSKAyWa46MIyzsYMIji5zWYjV+Jj7fWMf7Xcbt8DM32gBE0U+W9TZjAy2IBWDRYisowvv9vXMAiQOraf\\\\\\\/h26uEpvsv+mepdLeEEKg3j2D+JNDKVPq3lf14R9Iu8mPAjBbXPs\\\\\\\/1BJd6uFGuM\\\\\\\/\\\\\\\/gLFBw4DDxmuWpgYh6uxqegCoOq7UEK9g3g8YmxuaDQCijkiMp2D7EHhozZcCuCi8Co+b7Y5uctpY\\\\\\\/YS+BZbGP3aWNLAkvvQJ9KGDuq9CmpmmoJHrPPKhCBMVl\\\\\\\/QdHz3qI+OQNU5KiH2EzQXOCtS\\\\\\\/i9RpvHAj6KtVL8KCmU8XhIaj4y9pcquw8f6H7xQdYpvH0YPnAdbGMZBdU1IhfsDBzo7Cpz\\\\\\\/eX20bI+qGTPVYhFAxcrAH6GenzRLpbHyJh9MVJu1Bs1QqfnaqClPerH\\\\\\\/pevm2Qkzh\\\\\\\/TK2oI5bRMUStQYXggR\\\\\\\/2JgfEKyvg+WA1yr9OawEu9sRSCNkRiMdUX44EHkyVfp8Io7xMrICfuXICSAD+wnSctQvsSmmWG3GQ5jc89cGPYBb7qM4caRm+izULTf+0Wf4lzqg7G+2a1Jk5N04vVr77ouheXGk2YDxCu9sPfwdlreo5dLJug+RIuf0P817pG7D6023ewrMkW5hlX4ruomP7QxH2WSEE3sUxeED3WE4nC1K\\\\\\\/gGdVkX0yMITUe4RlRpfPvj88ASEenPQByydOJfFeOc7ZT0SjDNdEef\\\\\\\/6yAQNuqw\\\\\\\/Phbz3cE78sDRa8+spA9kBpS5lF2VyNWVVTjOdmzOEvIP5\\\\\\\/JDMWiANYtZik1y7T+4JsyvfNgU6t8gYOKh6YtoAL2fokRmL6Eh8roIFrCS\\\\\\\/J0r3R8nvw0nRT416dREsfkCDlOuxOvt\\\\\\\/RfeMIPCCIERNoWNOSFUAtFOvtpvrC0zB3Ct0hCY+flwqhJdobxPvFZXXkbMD3hs5dBe5ENYTbKrfVxZWqkRkgssQlsQJUGh+pLW7GOTxwpTm\\\\\\\/GBNTkS\\\\\\\/Tyh5fNUfjOIAvnQpc7pPcqK48UZaUroW+QU9OpyKyVqkhOofQz9NT5x2S7sH1Noznc+Geo2oC3JfBGVDl5NWdECqq+Ttr5tvXIzH0d4xjH5Pwg5NyPFnOvNIraxOlE+h7AQ\\\\\\\/4hU72V3SylGBFj8p6TDJFjhRV4ZIAOXhPohUliLybcdHYTbk\\\\\\\/ZMrrgwJXrrOjCGTG7QrDaCJ7t0ggyS1sV9sb7F6UPeIRTBvgfLa6VGwGK2oBMbSxcTFTX8WLEoHk3T7nXQkklfCbcj9OkvCWkM\\\\\\\/5RF7V+08\\\\\\\/AyXJ+MWQsPM6brCNkRAf9xodp6fbM7\\\\\\\/EeIuAMbEqvQl63utYhO7C5MZwRIHmSEqhfXXVY2ECunnKxKboQh1Nt73gUx1kvWPg2CHT+bZJrQvGPVL25Eb\\\\\\\/nPGKIRWKsru9\\\\\\\/51\\\\\\\/WuiLiKYWizYntJhv9nly1G\\\\\\\/UzgUvI164ZAP1wSjdLX8CO0AHPv9+egV7\\\\\\\/s8TIpsWIwg6gUVVUBCsl0x4IHwnvj1Ya9sfCbH1Xnf+BmBiitqjaMyTsEwbYDtmzodYgtR02HyKrqOQkvPyywLPZf\\\\\\\/S6qSSj7F5BgyhhPHTxiq4ZtwAm3S5jdXnmumlY7PhhVTfXvXU5EFR9E7aH7s4JklGJJM+31A5F\\\\\\\/UxB8Bvs+T9F\\\\\\\/D83HHz8ExJoO3gJtSVmGZv9h\\\\\\\/kGTcOHIseFLPhQE0DcNVxb249m7pQQqOD1CfY\\\\\\\/YOVEB2lnU8+fGpbErqq3mzdg4MxLWfQEQID+lwN23N51GIlN+p10HsXr3Kx5fBn8lZRZ1Qa\\\\\\\/jM4fw7YSDna6LWdU4lt550sR0WLmUo2DdNrFrh\\\\\\\/R6LUYYCjX\\\\\\\/cod1xs7\\\\\\\/qDD7tar5BGVQVA8GzoDwsHqV7e5BMbDxXLC0\\\\\\\/dcbMsOZ6mhMIXZpTWpPoEF8+yCa8zSNYYGn3jgy2PKqwABOX5wS0rWpNTWvKRovy5Iiwexf\\\\\\\/r7dXOLUV\\\\\\\/aEDacEjM2B\\\\\\\/SqySc3RvxGkfF9MNQ97SgHon+IL0yLjKZZbRa4Oa7PupLNI+za11QNsHSe9TbW3IO53N\\\\\\\/x7HxpEWImAPs0ewaTAfzuP8RlYhOeMu\\\\\\\/NeychPohCCxNFGYOd4Wt1FDtnAYE33gdFSAZY8w9X7kngzRmrAyKo6EvK\\\\\\\/WC5fRgy\\\\\\\/dkeOhxD4EF+btJUPrJEv5AzKNJK8sLmv8rTbNUCi0FMf8d\\\\\\\/q3iX7jucooGtwsPpRWGLTeam6m2fglYiaS3pjPMvW\\\\\\\/aCIIkI48GMFXSYEX0BGbmGfvmW+E2bXwEixTiTYRyJQOKXObGmywFm8zv\\\\\\\/rOHFzP9aspAG2rZh8LBSE25MeRGMV6SYJ2021sAEwc7G37QJKBgzeJeP\\\\\\\/aof8xtKTh\\\\\\\/EIKqLb3HqevBesaw8UrXwdfTzZ3AIyWHdlZv0zoVLO6UENbXmsTluJfWZdycacdPd7TGvB\\\\\\\/eMZvj\\\\\\\/aU6o68BXyo6Q8z\\\\\\\/tJ9pZDfSxk7xRPP15FbS+csFVo20rl7PdTfrYhEPkvmE1faiT8gVU3DmYnVZeqe5ip5znZ1uaPit+898FyX8fFBgvV\\\\\\\/eFDQyWvBeDpJFmGFG14k3UCxU5XaqnZR2XhOKYTYbOSsT2AM8TKXARw3d7kyk8lBKNUpGS7b1WWIjyYAk6xbraSxX\\\\\\\/EkvDswrpnxaNhuSCw5m76ZnTwBvEnXWwgiYB+bHeWdTUYxOzX9JN1DqG63J9yUiz5l7hXrSkPY2be5fpnE5zLmE4LEbgNDIHjdN0kJsgAfr62lEG4TPGsDAwauuY0vPYtE\\\\\\\/gimHWpTlKz378HoeLaRi6wUflBU7moLZyvXUJIFTyuCdOGyYdqLrTT2AXvjmLuV\\\\\\\/gInvP21Qw5y5LzwuHLYmBs1Xp\\\\\\\/4gKyNh36AYAVFBffJU27BddlHjB4lde2ZXLElKJTCwVQ04UjncR1COMxgn4ieAS2XETEhw4owJhosm9vqwiXjS9DkvZm1Hi0RwwGR0W6yoLWbYWSHF9J9NBP4aydI+TzYhYlxa+LAWCbIykqZK75BwXClvIlXYm\\\\\\\/9xzZMhkPhD+Vewn4p7cVRQtmt3fMhGHZoh5PotJx2ayBsLS9uXXq5iu6KnOE+drH3UzZNVNFkDjxyX7Wh\\\\\\\/SnG6RMQ8NcAOJ0D8H3L+pQjWOqgaptDQd+KR9rIMOO+b8bRtHrEneHApb2N6Zgo9GuhvjPMeyNkOifJ9BtUo\\\\\\\/NSsU5gMfgZ9A2ns6CMsiuRfiN9oApAm3kIv9681afrOPWBu5MpjpYErUZYfSNx7DUjUnIxyyBHph6rTvl5NbviPolSdtaV7Rmuv\\\\\\\/knvoYk3lq25LGqs624qy9MXGAPyN2untNEYukupL0nXeipIpqvbkZdDICnC7APmXvu3tKFGnRFdYWWgE5yh4vO7lnfR1pDuL89S+U2teDunBJrB91H0uGEbCz2ZMJ14QCb429CwcMzkfh5swdcbdCpkevKNvqp7fbpxd5RnS9mJbXo5fhzc213RGyIfHMB67ddrgBL0GQFbcZStCfbj6pWPac\\\\\\\/qc69J642IBeSrX2iXgoOZUng70xVCUlYbuc5hTa9xoMjzK6Ok1NEw8xmTExZpzEotaGBPGKO5OnstrYjwLW7VIVHWWyEMT\\\\\\\/s2Eaj24339B9d1r2RX7JzXf5+jsJEkl8s5cvwotTqd9hmeBYTwEmUnHJ4XOOWLrPkDTd+CCSryQ6N22hq3xubMj5ybFoTBBIj5l9OZMo1PAtMnZlwb6AEb7KBKOPmRPVoGVvWiDWPHFFzsvBIaJezpez8ieC6ZJ19PlCbr8tsJdSefKz8kC8rrdHD+JBEB37+lhvy7unTncqLZp+M2lJSoGzKDYQRVFWgBWcyhVArfMZ+kdpbk53PuXyxFFRC3rxPIiyVsYS68xFTltoqZtxNcdhaQIcIKUNRshxAGuCsmy6Qlf1Rk\\\\\\\/iFFaCBbLE9Eau0cpeAOr2Md+zab\\\\\\\/sZm+w7toxS3sTLEoZKwTZKTdTDxx+DzwZUwEKNlztazLAbgrYOFGVxm7Osy6y0CPD1KDyUlIw+jU0pD5yya5n+g61qbZqbytvMFJui1B3R\\\\\\\/cKOvqUXl\\\\\\\/mftuWd9PRB3oK5g7e4j2ALQajGk9mypnNFl34c5onxiLU\\\\\\\/OBD5CSR3I2IrmzagC8zOYSJtjdhCddh14RbXf\\\\\\\/3afXEd6ZBDpL4gY3y1eoyhEGVyINKRUttda6fpq0Ag4bXIunyymXxEs66eTNwOTSU9dL1B+kNN0lZBmsN0b7LkVyo28s1ad7Jzo+ck4wJytLwmTgHPkwAtawl39pEJFK+Xtxp0uBOP2L1i0xN6dTHR7CmVZD9o8j2o7Qahm8\\\\\\\/hK0hRrDPbs0a6bFSeSDU\\\\\\\/1pwRec7vIVxYjs\\\\\\\/EeEitZmc6SAZmqRqNWbPb\\\\\\\/3EjwMKbA6I6Ry8Skr+XTebCsTpfWbFhj\\\\\\\/Chopz1cIBGsqBXx\\\\\\\/i\\\\\\\/hWjNcSGTxfJysPnlzoTwvpKUDLxoqVq+tmLJzf4SSSj\\\\\\\/d0TA7EXVtyGNUQVCLAyarjZHR465jwiJVGywBrtb5PYLW9qJNwELIroIAMUHS20fP9JeKpO0AdMQaTPlQBuVP16RppWWcpc\\\\\\\/mQf0js\\\\\\\/UANf0Qi4u6ph7ZSYPMv1Qo3ZbtNQc6ZPQVFSCnGmhpwuC4nH7v9JpOZnO4kiPPO5VO\\\\\\\/erP\\\\\\\/bMJQn2snoIdBKPthddayWNmARGNv0I9LLqTYkvYUoj66AQ\\\\\\\/dEfsCWEG8Q+z5XzAbS\\\\\\\/c6FHvew5tA9\\\\\\\/8PK4XmTQY6sVJXnI3bTr1phcag99WWbqQNaXLu+cSY6c76iBIwVThgNYqbn5uirdjBhsGYMKrzf9zjHfhzhbDmcwwHGVTuIU0RFdnitLBQpJtmSLniO1VlK54gUgDCSSEunhDG6pvVaxcrbFocK\\\\\\\/0ybnzvQLno1mLxt\\\\\\\/cCc0r+wozcCXky3t1WWDfWKq9BEXA0xLPt3sRkfB5gmiFVDZ2iE4KrRBgvmojRGJjVJdOT8uSK3y5ziQxbqFscBd1CjERq7nz1U4K9dP\\\\\\\/ixnjuSg9BJL75abCaKQuSKQSe\\\\\\\/+RClNVzBmgclMHpCo\\\\\\\/EJsHq+V9mFVa4BYa6TS5YiDbY8U610wdZ7ignPUbaSQeE5U+P2k5tLwbyg7zGTcG8KElptFNuARlKZ9hUCx5fCU\\\\\\\/tjN5ngnII9mPdUGw9YwAnqeEaO5xxWZZ3iawAGtSf2wftqkfmO3uwxM1vZVQDHupil2bp6VatT2gZnUaQSQGW0x1yTCOkyloZ7PRq6MNXgj0M6PSe6gsaXJ4nzOOwwLeX0HpwWoPdzfW3QmOZSKoKAuo54Yn68IfGZLlQgoeO0sabv3hrAQ483soXFxkMVoYxx0NCoKdbHRClmGSNDeCpI6NcujLjjlcFUBs5Y6to+cHulRLq9QdvjxW1Io5ucvNpYhvmpIMTbsz1tqLIaEQBCYE\\\\\\\/cOG8uUGfSvQRkC6r+JFcEpNrVUyEwXg\\\\\\\/gMFkfC+jSeUL\\\\\\\/NTMsvw7cjGhEZxXM9dn4OZtgBxs7PhAZszTd+TiArtD9jwdnWttV9APiIzva8wWfs1qPUq+5n1Z4ijdWfoXQz2W\\\\\\\/L6ldMjsM0Rf0WdRJEgC0usXWCF42V\\\\\\\/TGPr1YqkCE7E6hT98ijeOSsp5QJ\\\\\\\/LG\\\\\\\/wAXoBocDTP3v0NZbXDqJmp4Cv36SrLJnYC\\\\\\\/AePbNyJ07sZDqJSyuscqsTX28IA5tKFMSqC6vJJN8LYkqW8tjmn+t1Fw8\\\\\\\/P9hbUaTE9HvsMTe6vP6EMpS8k52BmQeca3UqN9Kcw4TQNmfnOD2GRC1EUlDBmyq0\\\\\\\/7t63UDIShsj+aTIiaTHyvI1vpMZhQK0kkTLogE\\\\\\\/cEaoO7Q6QOypCObJMEt+AbA3F+5Z203uHB772JXOVrYn28V6WMSyTd49ZGDMIWmuy0p9PEAN77Xao+RKitPQYWq2OaMcEXPUO+VedQ\\\\\\\/GzhO6rKw4mYFzKQS5ydV61QTAJN1b0nTga+xIKItJljoRMyx7xBuLQc9PeLhywSZv1GTYzKdMtoR5F6H4u4LZlNke76bakQ1lZapdrv\\\\\\\/G5\\\\\\\/bcM6wPks5r7JYB6+UQqw+0voH0vPMN61zVxdtb4roHE8woFJZYH05lxRy23pW2ZTKJDUTfgJIyijU5YAdb5YB9d8KIUcODOl\\\\\\\/0ctP7pGJ3nZoT82UYjRVPrM1geHDrEhoa3OJ+1IXFq+HasJIlXICAPYRzBFxzhZbxvGzXgfvOA5VcfS3ACRA62vEXYaIhES1\\\\\\\/vLR8b0w71wot1usjfPZAsnSTKN5wN9PYKMY1gld3AKsI3Sgx\\\\\\\/C\\\\\\\/ugJASfNIC3KAz6\\\\\\\/ioqM7dEwMAxjzYLTcfqK9y0YVqZNzAUcFO0Fzw9cdnUANgGACIoVLdE0JHrwecnyHlmElakdy+kDyuqHqG7lvR7bLiCyjMHK5m4V0O+VczheFFP3P800jkT2fH4B4IU5PdE8HghA0VpW5hZ7n8nMXmzFdFuZ8DeNg7j1p5epJhEmXUJ4n66Ynxfve4xnogO1gvmpe6c7oTdbv85c\\\\\\\/jmkZZJpx1Pfq46TQk5itparis3xFyjsnSa95emAufRyFY3YP\\\\\\\/eJ+kdob9oLqjBCYSMlzR0VY2PgO4Y9K\\\\\\\/35JMnowpOhoNE7vCjeIIWoXhduBziDnM5qhL3nsgp+G2tZaqlrg0D\\\\\\\/R2\\\\\\\/DY3gx0QlAaHn6+lDvZZCdzuaqIcmhUz51hj2Bvb5a6ERGFnbh6d\\\\\\\/6RiaHe+W1wJziyvqrWbMRxRoDuXBQe4XAwb38DRy4u8s6PbKmfypmKKelxIFWP1qKY1u4KGzt3+sHM8bUKfupRm6SJSuvW6A\\\\\\\/jmTzRfeuBhreqTPwr4FVcU53wY8rJXHghdZcX\\\\\\\/oVKC4mNTbXKVnmMcLbiEwPDsMGHMMERXM2hTXxl41p1KFYQDamcmAh0eaoAvMB2WVfAqw9kDpRBZQvsSRsvx5e4tSV+enlQF7NnLu6bqu90M4rviymJ6dfPkfW8lYA9etImfWjSHBMB78pJxYY2hvYtLVhv5oc2j2NjB+cAnsnq\\\\\\\/FycVZ1aQenj6c9gnwlwm4PHP6CuEsEyo4Uv3WITND2pidzArDUv+dT4DLoJeD8L90RGgHn8aeQ7g43cp+Y0c1TuGkqs5T2KP1GK9jC7V+31Wp2IXozzofLlKZVruoGJW9\\\\\\\/+Ti712vSfk9ZwCBKr9gcY7T+G4DU3qqmoMTnrc6XmzDwb2Xq2pGXu6kblOjRG3Y7vqNfSlm\\\\\\\/k++O1z\\\\\\\/EnCUW3Yv6GJjWrd6kErxQpY9vmHVhfgxGeQyIQ0atWGg4Bs1PnOLDLbMr0kORKdtZk3fGGtWIk+9SvawiytE9eHG56gsxvaGV0a728XoXvkmeVk4VuEW1W\\\\\\\/ZPpz46qBOAgY+RGkxfvQEO9MeXdM82ZblIGmT22mVItV1kHtHOTo5m2Y\\\\\\\/gUHsJQpsXQsx\\\\\\\/8vyHIlRDRj3YPQuCC4cRJV+pZTKABSHOWCe+18facudIPXwxgt78Vn0d7154z5yHqA+IpBfWbnMvgPg6xDTun5L58ZoD\\\\\\\/E999OxldPhJYgiZsqe1iLfM1akzOKRQ5JfsirkXs42pz3TS3628ZBXKBLwjAeTbBdj1SpZF772eQCTFOSe5zajDXP5feVWgaCOnNeISrdUmJewoVb7oPHrU+nl75hk6kmO+96owTQHQr41MgImVgDXMkvfJgje0zHK1xGnnciMuSy4csB4gCPaEecpEFLTBD+ItwULdt+5EF9bYoHusB0n5F7Bze8P9V4t2TUfVcVSz7F8+enHKbEb6qELezNKY1DgRC7XYBDulLL87tbtptJC\\\\\\\/Q\\\\\\\/jE+EobmZY1iQ5m4AhMRZIMQsD9m33Bppe1G0s\\\\\\\/Ecq0t1zpU9eZtY46kU2KEAa5IGtQrKJhm\\\\\\\/ihqhKAMHcciSRLBxSSZVp57csHMAUctESL6vjQGrgKN1mnEpDe\\\\\\\/Njwx1za7J\\\\\\\/jMn1HM3TYk6hFfLPLHiwhEd6DLh6xtMu2EBt4ZPK1kClDAzD5Sf0gdf5fCea+\\\\\\\/+fGPI2GHocwy\\\\\\\/ZjWezpZowZ3w09BHNN+4uyfSfhlf\\\\\\\/Za1s8luQ+mnBfLT9WCaeTLDOlszq2vhZ8FYoGF3dOqecHPqmQR4KJY9PaQp3V1jLFC+5rSkZ0hj8NGOpRRC8Kp\\\\\\\/FAsjmIv3coJu7htnqQd9+wPUjK3Bujv+6JaSEbg2jSA9EfUUnbUu7v6dkfQJA8PoEicAYbwxDCwb6\\\\\\\/ymm64Q4C+8uJAOtxQz34HCIJVKXMEqNXMnXYal9tC\\\\\\\/mxaDWjgLF3PIszDQmDxkqH\\\\\\\/h8O3ynHgjJiTAnulMt6fDgogizzx7r5ttP9f9s4c4JTJ+qJ4EpRrTQqGPS2e\\\\\\\/vWT5uO5vXPaKLZU\\\\\\\/199hQ32NwCujZQTaYkedC4UGQXCj5Lhnk0\\\\\\\/ZrTOcEJjLACcD7gQ1+aYOT7NCgyoHbiKjZjje9BCv+MmNMs7MYXhsBqOFGDp8cerHO9VG9I8T2+YWIu59JtAZlGXebw8Sna3JrdWD+rzzWZfMJ4iGfW6q9hBts6PlvShknjSdiX2E6p+FUt1eayK\\\\\\\/bfD7MfKGlQVyQY17+AxjnbF8XVc1VaTCXBWVm2EV+sxsvjqvqWWMYb7skop37Y2Wa\\\\\\\/TgzQlZg\\\\\\\/gPET7rnQJj6mpv7UaPkTv2a5bVjdG4xfqTSTdmvr7ms6pPfUicEdELDwO\\\\\\\/sKB139hZ1L6GOVrY8mZT3N4stql+dBQdDpMT9hfj\\\\\\\/y+fRYj3eJVkkUARZXF73LbUUViPXE+r9jefrAKXopRe9NChAQ53KV+kI5KnT3RVoF3TkroN70SQyq05DqcpWvQXd8VwnBPU45+\\\\\\\/Ikzw7uaan5ovp9ci\\\\\\\/BesORgaj5nUyo0YttRviQ+9c3pQ0vTHRaZORMLTIJi2yrsNrY7HpoaOUrMWUjRZxOUWKlRaZBhe+3DhhX2RswtCZekSIGLuqEyjxitZExbdfRbCwYt+I6UD\\\\\\\/D0q5fYQQLpSwuvW4f3UUdGGLliCvOCEDye7gaXi0ZNle+RIMrYq65AECzcGPFNCPrSQXlQbAFFv70M2Bww8ioTP\\\\\\\/MlmXbj3QHRXTIloiISYJYeR\\\\\\\/yuc4wucgzqdN\\\\\\\/0I+sLuz+XwRHfsD3JO84+yMdjYYeX7f4ifdPv15RGXBUtDF3\\\\\\\/1oV79IdVtZjHlvkjG+zDVyestXhq6lFTlc9qlCSUmmvBx9P4+ZAzO1M02vHMmJR1Kp1s8DQYBTm5a8j7vyHRmF3vXP+krAWrtp6WtLvTnd3kRQWRKa9w7VDeGh1zNekOSLN3LXfge3tmcLV7vBQAJUUZWMdZJRTfxMOLGOYfTmhzh6zKy7I5vuE4Nl04W9S\\\\\\\/i7WMAf74NmggmKUP2yu2l6cGzwrTZE6QyTTNWDkStb8ykMhCQXuyVoQsmKEF\\\\\\\/ve7tfrjYg6NO7TWL5oXakcVJWZ4SabdSevC3B+K\\\\\\\/mJBBw0DEUsfzFSd57jdPwXm8N1nzSE9Jx4ZCkUqHVYB6NciGNYPh64GmpPRgZ7KOCyJpNiiC7FbTFzqN8jhwPoFyoj4BxZhGuUSbHHvMpakhlvzNnCCJ6Go9Oyqoa5M7b65adE8PWYEggXVUGUB8evgoewluEtAupWO6b7gz2x4uQI3ktiAeVbax8WTquepFvfJLbkvvJtu6SPZXJwP8Z7IpFQCwGi6snB8LQE\\\\\\\/l2eNM1s9nuOMIBSmJ4FddxAvXB90\\\\\\\/9MlALMcSgwaGIiy2WTBgsX4Sc9o4722m8WJXyp3L8BVrCieuTn5Nw6qu3hcD5SdFNKjyiunmrdqGJyrxdRHQxzbkYtRcDTXpPJxpD9UP2HdtZMuhu3QEnysrZ2jRx0xxAPjAzX7I+3BLaCzwkYfNJgNfCFh1A5mm9IlnCWb6WLXGSoCoIICrGUOQNbOzNquikCLIND\\\\\\\/WcUC4JZf\\\\\\\/D84eA8ZZKcc43FTqVCppuFmdxgd3Zclhlz1AHpUMJrwfdEdV3pkWpz6ccCrJCMYej1LDHkkB79oXs6mNlxDqe+cfEH\\\\\\\/4PZr9u39ucSFeeFehBoRCjVhSK7P1N+wA10D5ZYM9JkT4VHJgPRCdUDh1VqIZ+t6MCRIBIyj1yl\\\\\\\/bACoiLtj0JoL8v80WujzFobLaPk41eABtEay6E2iTVBVSB8Mm0oD7iUGpI2B26II8HtKVkCVIUmluvxfukM213hYeHY3fylMT2QjWGl0k9kwcVBeKN2uGrfXcU6mA3VZb9dRSDBxxuSauzxK0qY\\\\\\\/p4An5FyE4ifu2VZ5v5Dnme77Nhe+688YUh+YyrS8tiUwf1T1XYqoh1erwJTkUOaFUSl8vc+nH1tY8yE66I52x\\\\\\\/OsdmT1HfFM0EBB+eYxeEVOHeA5vK+vTzJJnwI2y42JlM0u1Sz9uevp32bo91D2bb81qtZJiZ\\\\\\\/cLcWCs7TdiQCaeN9yFTwsNAhETjDyvkkn2OXsJR7JP7s+LburjXtXzY1Wz42qW1RFIGyuxUsz4v02HBPv1B02A6EXPNghZSJCtXiMSURx7J48yZa04tUnfTbvfcgaw4oUpBDKYsrw4ovKEAF2lpOrlxshxTDJwFRXVbt3US74lOelumjIm6l0pV4PvF6xWwUFAg3QawQCd73slqc0L7+ZIdgDAB9fl5mfY9u+nGrgjplGghf04232B8F2kfc3dPwGFFyF3UL4cItaT5UEemRSODVIre08MUS0u1IEihUmFoToOFDTmx3WFB30To8X8i+3i7b3ryKzXmv8YfP68PTd1l0jv7iwQSqngACOJE0yuCY2DQA9GsWoGkKMEe0a41MuEx+CW5yP\\\\\\\/vfbnK6hVYYnFx\\\\\\\/GPJW5NG50uEeZE8OmnQBZUSiCp7XDVeLOkjjJveuA5CuSe7xLtpv\\\\\\\/PNRVOZ3j+keI8vA08RhfAjt7\\\\\\\/laukqqHSLxrm8y8Z5MsK463Tt3XHYAp3JUMcCYJIjbygDeqwp4I7ZG2iLAhDV1Zce0F2U\\\\\\\/N5uEXHuApJWdTiBtQmPUkTvHyyThPumm7xkbRXZ08O2Bir+lkfLiidSYzWxRv6ulu+v7LXOF8dylxpImpElazqvNZXdImgJxHE6lOShwgEK9o6cJV0Nd2IqDO7yE4tuddpTwgP3BPTxGk\\\\\\\/hKxOQ6ZYCYmRwuOXISKEcx69bupr0yFo1iaXsXOZhG\\\\\\\/8wlrNnJj0F3LfJKLFylUhG+I0PiY68+UJIetq6KAvwddAU8Ow8u+\\\\\\\/h2ovfI2czHep8mBCYe2DBwRa0zmj0BFZMIe3eUYv2GmBvJSwPmHCIwIF3EkST277VA0NE9N9vmNJ2aKKWtBK1WJg0fvnGxT1vnIHdZNshzUU8pj4TNkzyRHCDXoyaqKu8X0kJvEzmC6PqpW\\\\\\\/WRdEtOf0iduazxDpUoGROXxbHmd0L1lBnD5opy1ia2MrDBNnvCu7wHrcgUwVPddZzDhNqGhJuULFekrW9tSW0+UFY6FWQb\\\\\\\/kFHQCv6vYQOoQ4uhPHIoDrYMpo+Z5VHVmdGhZyRJfUjKT\\\\\\\/UOl7ch+qViIJL6NIbWGDI7z49PaSQX3zHs1KE2PlR4laBoruK0UvMZhe4hpeAvTkKu8hPGARcRu2USB3mx8awvK6BD9GNoOpVnq5DA3CdfFana2FsHSu8a8EMmd5GC1F6um08VUTWXv05ylYz6mmHOH3sWDwLHxKlKJ04zysv+gL9DhDzco31LNsV7TyaKOwAHRK1bytFS5pcB3kG8usvOE8fQ7nhdL5NOI9WzABt1XHEKNHHj+5gAyl1S0sLK3Q+CN8XTg+PCTVYXTZ5Z1z7bmP+T7g4Mz5VFHxivnX1M3\\\\\\\/3Vg7acQ69XAhtmR85uAg21gEcujdG2STQHQJZE99Khgmv8TEzevfL9azpy8OzF6DfV0D+Z8GsOpb958wwO6oiLtAHEAVKaNMza9hTo7QoS3bts0bzDLOBQTbU7cT8QOCJO9AHZnYFvQRQgUh5uo4VqWGTy8tlVJmOg83Po2izbixDO6WMiKrFsJulyepNZq9WlC4E\\\\\\\/PPTdv4ovrf54OOzT06hYYhgGoU+2g08sfGSlakIrzzkBq20MbWBgEaPznn3ZOzDI0iGwuNb0QeFPAwmZNds+DD09gTaYPQv3wTwdvRoOUEaVLcsUix+PQisEzFZXTOPX1Zb7yz8KpRQcyhkzPBFJoJsGcD5xsnDhAKmCpXC18SIyio67l3AKkD+5h2AooobWOF7i7SPDHZDC7jyyjb9VXJvRVi5DNtkAw5rPiGMs9JO4WxJPnM8q9WWwG739n0Ssq6fz2+mRDRKOvSLbx1SHtq8Adm9LxE1U4jcatMQ6wM6wiq5Hzzd4irAkoTmUWdd8\\\\\\\/Vy7lbrs8JM1NsMOeq9YCWQNGAy1Zhyu8ZTZ6SdKMyFptWtYcCHUarfs99cHVydiW3586e6NpNsGEP+E42UducgcvlZv1nSr07pR9w9kCVmc3J1uTTlipow7RjT7WbVU4eJcm\\\\\\\/EhuEvOFbJU3\\\\\\\/PLyH05KGo475BgAg2umLczlEmOD5+4jCXLhdOLwoOTMvWrE+sAoMYdBVaJNHMubFTGh46qxEHEW5mwYyyYuuX3+R5XIwqzfZJKKn28q45DQOtXBMsea5R0YNvH8uADJa1QpUeUNFfcJBk+CdFiNhoxRBYoHWA8Sn+8VSq4R7\\\\\\\/LUON4ZQSqXa83u0C4xh1TTp3PugQDjI4U2RGA7VES5Vf6mJ1QgS9opC8hS7D1Fy8C0XYkfSA71tGf1Je2+yusjhrnv70ihBetgD3x40Vo+HlM8ANDC0KTcs1ciQ\\\\\\\/4gmjr1BZ5KnzWLvSzCrDcd6gec9nqclWSTbD32fJ0lxmimyu7f\\\\\\\/hmSWCp5vbryCQ+GxVqNb70ezvAqBrkHCybEevRWqU9yoALkPhPCkiE5iiNKZgKJ41PuUaLA0ADK7U+umQOdBVrOx0AueclftzpxyeCE7tww8bUAXyFPc4vCihrcdsard+hdTLtCKlQr\\\\\\\/1hvQtFBfWh9uVFOoWW+\\\\\\\/J+lnC7gF97nAM0UKNh\\\\\\\/tnHXgU\\\\\\\/9OlrsFrGUk6eWXR5kC41dpFc7g3IM\\\\\\\/EuNLFPSDugo37XLorLdD7PjmGRz7SbPmXGQevZQAtU2lySu2IRbxGlljK5Ar06LmnNL\\\\\\\/8+ThqPAUWUsUoZQSL2hNB1giMzX4yCSWuDKPNIztbSTvDuvWyqbClOpzcJI7135+u8+Xggk+fAFq+HyhcS3BDnyrPqBpT8wF2w6aTue7iXHyHxuz4J0FBwyloA6wLcUchYTMp3MrJHl6k\\\\\\\/zxw\\\\\\\/5\\\\\\\/4mw9CgqIJpOyTQtolo\\\\\\\/ANu+uz9RMxyJC1j97Vjv8HGNoJOPCc88SZxMhlQDLKOUyYIk89tqYZ\\\\\\\/iLGiMbOi4jKsTdmpKQvjbub+09VgmgdMHAZ0op0ULK28ENavqm9IvqF4+UmO6pGDa4nzEWX5lCpaPeTa7wlp52pcYPEtIDWVhnoiYo4Iye9sVD9aIjaf9iBdOYn3l+KIsWdO2YyLAyAehRFhmHoJqL2BIFsv8GhKFpJoMsKSn6NsmNLx\\\\\\\/hb4Rd6uO+w\\\\\\\/9Gefk11LXUT0E7Qlwg2LZ9wi2fSd3w59duVoCnxPdh8OlL3GCQjiWK2znTigo6CUfLIHDZYji52k8AVtK67YO1Zr\\\\\\\/+NY5XHVq5HADeaYMOqe\\\\\\\/y2DNqXVY9VMhGnN\\\\\\\/QlpCWX9RNQRd4LRx0mNuJbChm\\\\\\\/Rs5l0673wF86b+1+CQAq19q+JHarsL5Z+u+MQ7mi8HNp1ka0IDr5PpeeJxNz7v4CElNRwi9f5QsF1p7yPbFhb5AIbDgTiRhxxdnfGNdJoEeExPdbRqLJ\\\\\\\/OkQI9QB+rLwuQhvfiE8IJ3RcFO\\\\\\\/x2XY568ppqmfNY8EnbuDwDqc+cJ6aqtnRPULOM8ZsU41\\\\\\\/pZoDe7+Si71Aq\\\\\\\/GJ5EET9vI8LcqlfOLvdYgU4dV3j1ukpsAMNsaTEeRzVObeP5olcYFPEJd3U\\\\\\\/5Fx6hAWgUVp7eyfbPIv2qPl7juDGeSUyHuXP1KFWWewF7rDz9IdOX1pTu9SBS+9fL9CWdu20b9iMBTZ+RTHLoHl1L\\\\\\\/HQcCim\\\\\\\/qsRWVAT8iKpCN6728GAolm5DlAx3VaABa2LK7Vl6JeujgNlvIynHqErqDS1Z594cwDzbuyP5f2ZYcxCh0rScrv7BmW4Dsqlond520itm9P7y35tjk7Rz6mu39Tf1mvpL7S7fmy5EmmajufkgaHEEzTIdE03KquFXtB513oUUEdHHjOWZU1\\\\\\\/1c2voxU+5dyu7eIwmQ8DKjXCV5RtKelVgaZXnztOnGhJyyxGxmqcSPvzHC1x2EH5o5Quq0GXiS7oS91k6lt8gkjU6XGifdXQs63iRkZ3HmMre\\\\\\\/yZB72a1oLKWL0\\\\\\\/yPY26bLw+G6yfXFFzBp+MLgxgxMc5VjszHiqBOylsSqF\\\\\\\/ZRviFkTv3qraY1OOmHJ9u3ct6TnCyqNWDgOMPSHM65JH9afJmdRLOpIqOCij7EhQCOZ0tyzG6Caf4qToaPIm5I2h6MvoQJQG2g8Q57gjc53DPJIUjULR43TLcpmgL9XtrsCr2bkwEYm3VNUa4OIMAfuPjAv1AR3X\\\\\\\/i7v4pgVRdGkIguvHquVUxD32rDRvsR+tKD8yXa7CTlQ6d4RIOei3wuzM7UWNUuocz25KaYd7UKqCkFa8U431A7Yh5RWIkWfJY4mBiZykiohZSYXCKBsbaciUcq89tzcyVtzLdn9kZ9CVNYbkAfI9VVyrSdhWdh8cUEXq7xvWbyql6sneR8oppSSBGh5cdXN0kgpZh50hv1kXgs0icVqBp3vWyKa5g6TzKrT9KcYwj0rM\\\\\\\/1UT6j3YJ79rCrNO+Q6fKLg9ix7MkF0DbwlZ5bpKcqpzeKSKnPGsJvnliXvpwYTaS4RaCDqpJBcHM\\\\\\\/ZL6BqtMIAUNFJX7RRg0MToAX24h6w\\\\\\\/sp8fjh6\\\\\\\/u17++dsIY3cj4Pqcc7rmkMn1vELtPlwlvOE5ofYIdF42Fc8i1\\\\\\\/hxCkbrrK7cKrYQIjHOuWOAgX66SN+0y5wRf8grilPL\\\\\\\/pai35JvODbv75Zkl1axuc\\\\\\\/Pd2hOHN207PA5LIDD1J7SGZ1Rw7+avNDqBRG0ievtp\\\\\\\/4Z\\\\\\\/QZx4EDmn\\\\\\\/GAIPsMLSZpNRO236v\\\\\\\/gaunB4KGDPwW7Gdb263hNbvLKDJQ+hH61qZwZN72+K5wdhXGmMu80R8Vh\\\\\\\/sa61BQuD2w8ayKIjP65wQi0mLvL\\\\\\\/vhBZ9s3O350yM16IepmOi3cdJcVhhkDasSD5bisY5clq2gayqqdi9AguZfkA536+tSXRj66489mraxI9Pblf6kCYQzmH4D7lc09IHqGT4sA2E7wM1A\\\\\\\/iTeTfwT7125yWgnIFk2s0NuqtNE1yqgjPwd08Tk0DBzASC+gl\\\\\\\/1nJwMas\\\\\\\/o7LYVp3mQTBygqKJ9\\\\\\\/2aEJbI6Ui0Ip+BfiEXISz2MRKv3+R0sceOKqLpvqjD\\\\\\\/qcPsQRKHFsl9C5Xp1mN\\\\\\\/h7k1x9RXY+fLWU2921STB9oCGQsjbsPms31uwjhZOmNRZ7xqnnmfFahzyKu0O2noi0G\\\\\\\/U2tPi6w3vxJ1TWu9O+++PmHDN+bgBih\\\\\\\/VgsOd7tD7W3JPs3guFiyZg\\\\\\\/Dn6jEdykzF6Mw7WFUDHNOYGV1\\\\\\\/BulMPQJL0Pj1az0KRX2yPyHuicuuoYVjgH9Q8xKLpzwQCax28wtF7aP\\\\\\\/EL2stouafLJxK6PThRRpgiMLBZuhXZKoX34EvcwHmYUKOCCz\\\\\\\/Oh3Ire9aM8AHc9lDrNRWR0ZMzdLnxT76oCgsqSX6RX5rDEr1npG7mW\\\\\\\/Xe8qbLC6rG\\\\\\\/9WJl8sH01bPpduk922pARIlhD02tYQdoxvjLFoI\\\\\\\/JKqp28etYRl3ihrXNIqKTHc9KHN6NRe6VN9Zjo8fYnSg3TA7I6YvTTP4pE3lsHcnhQkRuF2Lgu40QKAOW9Px2CpP+\\\\\\\/Q0qglAGLSSCg08wIUb\\\\\\\/exhTqp2jygArXWVAdcxAORJ5dYGhnUxKaBaPfoNnZRbE7kaGyGj3Y3AHSbP\\\\\\\/sl7Syt2tBxvsbr+7VA7T2gYuZrSLgFGZJJ949qPaFR6wSlNPz+K\\\\\\\/bMi4pSh8fW8xbdQIdJa6b5QOkwHRxQcYq8U+2NPhnFtw4p+1PYfYravh1rvoyHF2PWFytivvsrxH2JZucvvbwdi1PmeVFldL0+XGD6cmKkOrzM44lyasHBHSJRlcaXBCMud5sYt\\\\\\\/FlTsq6RS9IxXSL5MsdPUZd+1XPMPzn865xd2Gn7ihfXB1rp6KwFtTKNl10jF3S0ugJ+4rieIkYJQ9cVW4ou0yq392qvlTZqbgU1eJeX3iwzEtBNe58IuyxkFsIRL+pkKsz+\\\\\\\/LUjpCl0AA0T0gxf+TH+Fj+uMDRCyNRolkBSIvXBhUFx00cNjkrDVnF\\\\\\\/sOTA77JyAbUI4sX84ItUNV7bdAbH2e8zNbFANKzUamsjJygpae9D3oRasmVkrBljIH34WxDfmNM8JOHp6pJEuo96QgAqJuJnY1WS2lsNBhjfm2TrscO1ccPDBi7q1ak1WNKuweeO9nLkmMkw3WAsBM5xIHVfGVPAEZx5w7b7T7x48o4b+Z+SeVoAq80JWyrurCKkrqHntTzcXsXVOV3vpvFVqltYXSCLOREWDfv+3v12i5h86wb1f6E0kWyqn090tNQzg7yXCDF1Ps2zRc+k4cl2LOgRI9ixGG1e2L6itgQ9neP8W+dGeYLQ3mFnevM\\\\\\\/A3U3wvwOce0VKi02yF89G1cgz81Q76JjFXTtktvVC2upMSQv+jOMnZDZworcQ0SZipz2mSbDcOnWxlZGnqQ2NttKrVloRDu60KAYZy5F\\\\\\\/IbiXD8Ctyg0l60CnGvwCe1hxRAHO62xVjPaNHRbwwIRM5cdr\\\\\\\/L9mkbZg0EjA10ejti8rgciW4jWrEkWxifYr6hoqLVV5UfIr991Yl050o\\\\\\\/dni4I3gRX25+NefKiHHhqg9eN8+TTVwq6Hy6kfv7QMhfq\\\\\\\/TVUKiE3qCtJurdVLZfNyK9FgpGyqJFHniz0RACdHreYVuQQe0A8YvzpEHUMuS0788dJAofsnfDHsE8ldSFfzKbMaX+G6twM11FHkZtzDTBq8NjnZejAPdJfa0+zq4Q46gDWNz0APcPTvTXe9Mb8\\\\\\\/5vr7lEs71NGVLl1QprSfaZm2mcg4nrf9l93wdLBLwTwZYiOggMrkq3+7PFgt15JCpUsCAetDzY+mT3EkD+JAbcMDYIB5hw6Gv6q9c1NPBiqculg8DS\\\\\\\/jlO9OPAiUUmWJvHXrsxzqR\\\\\\\/eIiGTWAtEdMJ9A5rrb0It\\\\\\\/7D6LX7T0AsZmggXnew7RjCBBx4rpvVbbVYKENW0qbNFtMm9JpYBD8+xiH4ivkydn+ZawyuGQyryV4YoYClM3g++iZEj5tWjBO6D1kONcWnXdrKDZviRo2R4K3GR33TEC+VarxYdpeack2H06xTKCLMs\\\\\\\/JZRKQPyXbyLXnw2wpdKjuyXb0B5r2F3Lu1fndZaAHm0pwDFHETDsh22Y0OKHHIPvICBOZ\\\\\\\/uUWN4euEqtJ1d3+PGlCfGsNSd\\\\\\\/2pWCrazsAarYdZoZe6SK+4GczS+\\\\\\\/YoCSIm+JQUMLev5I\\\\\\\/JhGQovT3UvSfJFIRdlT7skLZWnGtXOynQOuC4J60fh3Zz7\\\\\\\/r2aUKdjux8El6ENUFSrmOm2+ikqWDbU2YCmdanwkhwDbncyjd3ixwaJ2P\\\\\\\/a5GHCTkEg2RhKPf8Lv7eLKzlK7nJbAjUZqdTvGoY1X2kk3W6jOR35CsuOgxtthJhFPGHjabePJsIZLXLHMX8fBd2bWGgL\\\\\\\/xGKF0ngxiCpyKtjVKn0rErNeweK2+86lQLqJOrXXxvSSmyoegMXHOc5XmTWkiI8XuzhkN2DV4b0VPwPYVDasO55aRPxueNs9JNwF38LJkNyIfngpR21pokMgMjZw3MvYQ3btioFzQinP5wDEHW1z1p0Y+0YpIj+J72cb2yHlfwrfo07umIi3dvzI6W1RlYsNtxtxHxmsGjtSAzkP2SnxPGvJmwdmkB6HmHG0Pdaqtf0qw42BYBh6viWlRs4++IZcHus+HvcFymjN\\\\\\\/8kfVSU+nF0mm8jXIpOWXgE86IMHmtxbln8jjzxIW7OOJWJz\\\\\\\/0KYannOyCA8uSC9GjTHieQ+gTFMgjj0DOMbwnGB\\\\\\\/lWjJM4jVnseS8whfaNpQmSdmx1YYSE49w5MmuQ7ZA4Z3IahOcT7jf8LJ49DM\\\\\\\/2hPeVzF7qb1pPBMv+SCEXBdRR4HRVlwld7BhgCa5Umx6jolhOcWUpRirEh7VIAqUzzaR3QPGk\\\\\\\/FH\\\\\\\/1KWJABgnafkhhJyeu1Z6UYKSYQWn23Cr+ZGPGKpHACz4pNEg1Hdb1bECh+PTyS9JjrkbKazDuZYTHbKejjl+315koDmSFDFY+QRRrjx6+twG4wV3SHW6bjPvVnwhY9YzI8zdj9X5GajgdqpYpwPkybvvCCNU8o9hRXIrJ3P67CejEPHvDOJMzjS1pIZ7p01jUJmNOX5n5u3zHAPwaE7Qb0QwPoxcsqswQ03u5AY11+qFtAT1PZgaOAxp9j18Q+NKRLlFdQ+or9SiWGzQgkvfFgQbh\\\\\\\/8fj2Y+qOBDpeTjZgwqlrtltG6rUoTxXcF1FSJCTvHpC48D0\\\\\\\/Ly2gdjC\\\\\\\/ybVhA\\\\\\\/wofu7Ouenc4ZMdbGRm429RCHpRoOsA81mKcKuX7eld5PwJtSbIlpfqGT+X9dm+yh023gl39wZfBEd8X0g1HyjhDTRIzQPJ13JluP+vOPLXFrsWgzfb\\\\\\\/DD7JrjBDUPz3ddFElhN9s6pR5QLUyYLljg+xJ\\\\\\\/\\\\\\\/kRNNZ1cU3cElPhwUk82vqoEuC3jFpL8k2FoLOwDJQYXMv1ClIziNfwH0nwd2bropv4AbiNSJ4P8xJSPUAXuWP6NY1HS5tmJj5y0QfAO4YgMp5A0GKsS5ovc5WrXbgOyXScQrAh1k0Rc9gFMdFGc9Rnwm5+J2AR69cJIjpCLkjuTblk8CNrhiH6sQXeR\\\\\\\/yndw5wcMzrveAh0V5hpcMOCpmaA77pJXCf+iMymmEg5d9wPvx78YYSM8ARv0SkdLzY5psKsG2SCBPrJQc2NU+0mWjTlEtVdVUTIk6QTX5q3L6eTvsjtHP2b+\\\\\\\/vensgLZoJXYgELgjWdpa9BQ1jF\\\\\\\/3iGbuKjlC\\\\\\\/wvb5wfwDMPovSYbQmGHSARImUufEtZNlAF4HeC69vJhIbZH1Lvk3pHlxM\\\\\\\/HxiGlU5Ti\\\\\\\/kATI0RkR3fDdf0a6BEkfOZ7ub79\\\\\\\/EN0aH4X8CXxjHHRtMXT5RRGr8jA0Q0A1JKBN\\\\\\\/X9FDKpMuO8mJ44BgcP\\\\\\\/ebplTTb2Gya0MYpPYQs11y2SLmg34an+fgfl9AJ8Nlgu4piOndfoeflPzVrKdq7+77nvhmksJ4BVQstI+I1rhyIADUtLuOH2bnaQAJcaLXm72Q8eEiZJsP3RcbuE12p51mz8aSij6sPakklC8KvqlSX7UbXUL82vF2tWTOoxwMi4CJYyGbszZc\\\\\\\/esztiPRkmZPUl7ekZHWPlD7lo0kEJA4iptjjVcmtXkx6tYccFAS0Np9URDDaX2DUBpkrwcrYQ8s9hvXUNsbO0ZeGXnhCfTZgK2uEBzlcxKYhxYVCoKuqztkXGhw7p812MJdC0kC9D2iWA6hAEoT1X38YUeYe7t5Bzi3jj0jWJdGVeAtliM8JHhkh6jhbhLLpC62jmOg\\\\\\\/Kt\\\\\\\/1Ssa\\\\\\\/HrSxm2aQ3dtYpqw\\\\\\\/uJQZRY9p3rk9+EvHSTtPRZhhWUJwXbvfkvEygaGs\\\\\\\/7Ttdfk\\\\\\\/xaq3q1e6KWq4lBk9NqoeM76sq0g9TzTJ6hb9Wx684z9qeRqKaJaR20qCFBOxDxOLXVkdpCd\\\\\\\/PcdxGfJHZ8VAqHRZ5O+lzWzbTXYW\\\\\\\/m21oO\\\\\\\/m93hBRFLzOqIqc7yfOkJ13Mcnm2xoswCTO1QalE+wHGlz8awQ4s5REC4QdVjhBwYRK3v2a1R2sdVwQSYYw5cY\\\\\\\/jzCyUPK7t1nN0TS2SPrdpQ2h3xhtIB9nLXvaj45fsEMGbAMt6PMbqTk1ZN4i\\\\\\\/WaCulhcmt3E8lGR\\\\\\\/IGRjXGyTDNSm9VqMMH2V7bO+apKRQf08erdKKFiGcZBbc3I\\\\\\\/JqXSr+bixL+YRJ6sJ8bE43kIugjf6vaNTUvduwmyfbMSTIv2uX2Z2FYnXvUdEuXqk1qkq6Dm6sJuY0an8rfBRzEQ\\\\\\\/28o1k2B67ud+3Kjr\\\\\\\/HtVabWpMbS2rNYPCC\\\\\\\/UHHrB840zx4SgA5fOox7URCnaarvd9hKLXukP71\\\\\\\/OJ2lX3ytL4lA6iDLZbIDdC7H\\\\\\\/RhWWfasNUItdiHBUCpzNPfo9ZOWRm+F0fyUM\\\\\\\/s5mmdOoXf6K4RF1Me10anuig2oUcUZ99+AASrHAQG6Zf8s1uj9qv+ZeqD6irAUR+l31VihL5Lx6YrptXUSJ5\\\\\\\/yutPtZ4H8\\\\\\\/MNwpToQKmPGNiRrrl2kOdapuUq39yJgeAllwqO9rbJzNLozjrRq7Cy9nlyv1Ji1KJ1hXATfaiMJ70jFajQD2MZ82rbSHXxWWLcHmfNAkshZKGnHk0fMVoCAfftC1U4zKO7vVm33hQjmXXf0oPubYfWNZOS6snaA7w48oK8xMRTRgwmTK0TkEP\\\\\\\/xhLbfyNnG0j1BP8ixM\\\\\\\/IvP4WpoLOGwbuHzIZ10lcWg2L\\\\\\\/+j3SXRVK360vdCniTgkcNT3Yp9HLwi6whAMZ+TXLVZxGiUAnSHM+7x1IwWwaxEUeooHhydBraUk2IPSCnQxxEEHFfAtFhIPpfi7u7SyYurp8J1\\\\\\\/zM60LEObTvAxFIiiXYY+OWu+PsajA9NgqxrxvZmh4\\\\\\\/ueY7EJGCs2tc5BNOAOL5gbnc6fEcvCCiLfx+BH2r9U9KNVX0GR0Dmw7gd2\\\\\\\/Qu31ZMIn7OLDmGq13elKX8WHzquNyyC4HZ0j0V6gpsXkntI0bVISsz6punESANCYONqAI2b1zMN+1jp7iNlOoMQtelH9S8kBXz5GxlDTLkvPGNOF98LnGbOVlxJYbMgUOTsUiswrGAVFagICHnjUQu+BiFCRvjoETXyFRcFq\\\\\\\/WSYA9VwpZToAEd+l9g2dBzDi3oCqd8E\\\\\\\/pK\\\\\\\/3GHQrVQJj8q8crEMUfLmGQPkItJSw1bWpb3xCCVd4CU4leaw5TjysFYdKvstO3o7cU1BL2meHmrZHnhGJI1Bjtd1xKskPUOdKYl9ugkvoKh2\\\\\\\/bA+z4299AE+LedgwChb7FZUUvbbAIK5x\\\\\\\/mDFjxnnlahG1QcA+I4Zvf\\\\\\\/9BV0HmesYBib7sOwWKl8Yqdm8d+1Dj6L2oNh6LBRSQFjore9fjrochFBHz5zQ5F1lNm6iCNP18xPqUvBYMbjCO0vn08mOF\\\\\\\/Lt6\\\\\\\/U84thC4H951RkOWSVZafb4aiW44P8RipTTDIdiI6k0Q5xD3POEQYhUchVxTfylby0FQgKbv6XKL8ccByA9PlhPEtP\\\\\\\/enCokWmjJrn6LcnVdKUqodNFV2yx4rbHS2I8JIouIQ0XwfHKiq9GJVuTZcG100SAC212Do4GQy2e1RGGHn6zRSiv9KqO391y3QwME901mE9iNXWKZ2i+yTrXvfleEZeKlomun2oJyRa6Aa0iqOxAbpY8udFSFvFGMG3obyYqWIRuf5flUVHEb3H1Gxkk11Yg4+pcM5ABR+zfzJfkC2r6P6jdT2Gkw\\\\\\\/VPvi8IvuiJ9+eIyZnh9fmJ7rFQxo7XxE71scCozQtBsoGFWRghxm\\\\\\\/VHDgVYYM47+kjeCk68Lhmf7ebfuf7m6FMtQxVCMlviG2DkRj5Fot1eRNn7hGC1v8Pl7CE971d\\\\\\\/FFnUBJRpBpxWEv3fHxHe5exII3kTVJymzBlpvVmcf3SruWlkvewZZCvDKC7FP0vB8Z0m42D7Ad4H5gpDn9lR0E5wsNMAVZoVKQe\\\\\\\/ql7f48JtvlFN\\\\\\\/UWdQXfSX6vJLbtQlJ1QsAW2nBaVEZ6yEn0OC70UvzuXf3vKp1Ql9WWx3GLjTGyE3ZGkYdCiXkmD+s1NPrR\\\\\\\/j8hbQDg9eT1NA3xBXS\\\\\\\/i\\\\\\\/scbIk\\\\\\\/3YW8xVWDKYVRRwlF3xNIqrggHFzgC8eba5PmDl+XTpUZ9bFn2l6rgltFiTnVu3eiaWEOWLlVj2MupR5qqDMYFpHAiq2hJHRdFlSCMIGyyL2lMK1dPWN6KMG6Ui8TmJdq2ykx5rfnjdoFPXtUiKpJweMoVQ7zel9qzkF45up7lJLLX+1QUU8WUJvA7xyGaAc5E4U7z3vBtwxBvqoQZG\\\\\\\/mN++VpoKaxaJ18BeCRp\\\\\\\/kBPrP5c7vVyQzxfyFIfbUue7lqR9bVvUZAMo+o3iyB1bHvZBbQwgErQ6cEaS6GQmEVpTNSPv2rqWUrFbHtKKimd0b3l076FTPsGPHG3Ex2KfXpUfe0O5gqQxCvCESHSKT6IIV5iiCCetyr2Dr9RhzgNS0n2T39Dt9YWMTgxvZF1Q4IGVVJB4arDCxs5NRmUV5B\\\\\\\/jwTNwTb9sI47gEkqRYgAvwy\\\\\\\/dI8gDHioEaQjOJkZmfW6EpuYqes5QqaHXeWBV+Y+vvWskoWNirD80H+udCw5JaEapvB0Vn7Fird8u0EXuZ+her3UOsVBJjkDB6xQeWSgTro5yA4ykfPD\\\\\\\/dolo4rMXm6+k\\\\\\\/QPu6aHCjp3q4DMg2T09OO6\\\\\\\/Jm5Zaglf64PHXkC7AEsxw1dlqT8LsysIKnuWr1LwMyWFOB6tW5A5MaHomwVEG0NdvRGkTx7ZgZ8oV6VXazSLfoBpTnd5dfNs2fWT2\\\\\\\/pKtWAI1GYxl+t2q++400S\\\\\\\/EoG8akm7aKtxq0lRGUvLzDmQbHuIKGw5yLqlbbW+KFChLo6P1nX7Gy6yUGJ+hXV\\\\\\\/qnrvNcLqCp4tCrH68pOwHZ9I4jNiX9osHunOnN\\\\\\\/HkyCuktn07eyqueG8s033xODTZXEUYXdBiRPc646Pg+z3b4N8U+0r6MOFuTLkdJ+OhipCX4SUBvHShoz1cC4FU6ugBroYHzhQE1huN96BecuI7aRse4QLM3phNyh8bN0wTWrY\\\\\\\/B3UT\\\\\\\/MFCM3vPIlyQaQnzh1Jd25QeR1nfTno9fLChIeqK\\\\\\\/BOQ5l9XwH+vg9yiS1kkcYwW1IhVCvSx\\\\\\\/f0CtDS3vJ+Ggf0yxHn3b6rLFnHpT8geD0bw54gnZvU8Z+JlYUHSbAPAWxFA94zvBlvbFlN8402XaPIS+ahObPqmVh5PcGkiGBMbQ+RqOMpBBCuq1OnqYQ8xnKekEgFGG0oIorZFMWY1zKLb+eo7V\\\\\\\/AihsSAWo7hrzxdJILCIPiY1U+XAearTNOLkbXcT+RdiwSKvNY6RUjZb6S8\\\\\\\/CnjCOV9xvCxkEuQe3e8s+Yw9rc3boqDK\\\\\\\/DuYAfTEUGKAxiEbpIkd4M9gwfeOmLIozQ98XJl5b0tGNeubSrRjjzu2b4OrwvmFT7RkKEuHgHHMwZFZOMxZF\\\\\\\/C69pN6v0cSqLhQQRXHqCx3kT0C89P8HfXafANzdo92nNV1IpOLf6USSOgV6so4\\\\\\\/kRvZXPXWgS5MmUthQp1HFvs\\\\\\\/00ARlYTlBoUemLW5e9s8XF7uerCxevOXGC1j7V8NFoIJbO2Mk+gcDbIl8MFqjzMjfsPTyhbN5ONBnjBVRd79jvLmOBmG3uOTKJUwQxI1DOO342qlaaH28MDY90Yr2P+5VnCM3Mt+XuRdQbTSSKpyzAdPCXBRlLZ1b9OrIpYOtOw9FMm87qSdHBtTS+21yu\\\\\\\/Q0QAFXsLZGGCKFOpPdWdBA7GHpkKSb7H7O1ew06RwuSsk\\\\\\\/YG4IPliTs++ud1YbaVjGFy9AmphWv5FSQFh+5ZXCVFBk7bDCyVG4\\\\\\\/uVyxZcYlvnpdH5nvscOGYLZ5r8vvOUXtQBBhCiLTGyDYrZYQWYNxpMr8Sok2ljhxbHrggjacsmdHyCh\\\\\\\/EMFXV1H17iWoyj4XYoxNnIulfOcO7+yCrqXaRsxTFBwPmB\\\\\\\/xP97XORPlnoOMWfrfeF8CLM7Ts6HJ+bagkIpECdnm74qVbZNykTaKwaj+7WnbpPwRa+t2azDBncgCp+8zbjzPO1gQhxCmG5KFKIedFpcEBlpWQGKZ0q7FNTX28b\\\\\\\/\\\\\\\/KCIffObVxFrwh0GSA3MPE1Fm9eeAn\\\\\\\/sqvv95taIlruATLsE0inK\\\\\\\/q3x3U4NRzhgedOB38HPhd9ESEE1nKnMkbuu4Wh3GGwNfrc7fu7dhK55FPlbUhUaRrYAsVA3ckZ2N1ciIMccHJ0CvHRzf4ABesHQu5B7IvXTAC\\\\\\\/nD7UeJo6JZxQ\\\\\\\/5NqEA\\\\\\\/ydMNsUizeXC8QdxpIzDg49bw3Rv1DMCHJnoVfVZV+ae+v2OTxv2c01WSyjuWHTuaxZkGt3XoPh+OPM+wBMndWdZycGV3MAnU2Uaa2Pcpt+g0d7gNk4t23cVtvUp3suvF3YxNj3Ez\\\\\\\/\\\\\\\/\\\\\\\/U1SBY7wrOGdziWdQ9Mkp5IbEbfxfig\\\\\\\/cRAcRYqTARntmDTLNIl40IXNDUVFqsKGmCn\\\\\\\/FA+tCzuhuppP2g0wkwDYj90BMgDtyza2OH1oKm5TjDo6YNnwMQf0aUrba+9jCtQ3qD0g8cl2vhwDfQhLmc52oBvKvywkVJNls\\\\\\\/NZaMA2APCkI3mfUXr6QKgtpMAl2valEaNiHGMGERFFH6VC6LnY4L8A3\\\\\\\/mCqJ8J2YICIsWyDgKBvsILD4+wkz40WR+GgBWy4h4es2WhAxUvngH+8Pny4y5lx6sf+v6VdaPcZHdfWTHmhxK5vb2TySkXXiIj7HiqjJT0g\\\\\\\/1LZT3lk4BPLF4kbnM06T5WYHBWxkOa2bRNB\\\\\\\/QHLmjmPvNYJwgqw1UF\\\\\\\/WyPMlZTFOLUgoDxa0CbCni+Je4GyzEWe8BLD3S4L8wZbQwI+pLV0WzoFns4Dzdyi17vCpVYErzrZiB3Crgq6evCI0qV0t7byz\\\\\\\/WS5d7HSCVgsbb0g7eQp7jN\\\\\\\/\\\\\\\/UUC6iv\\\\\\\/FDmrVDdT5Vb3Q67wTZFh8b\\\u0022,\\\u0022iv\\\u0022:\\\u0022uGL+k1FQrjZXfEY2Km6M3A==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022e32d21e535198fd34727923353eb9644cdd7e7804fb6b4837b0300ddcd1f8a17\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022CnFXFpZiaNUBoZv9w1FxfB0KSFl4\\\\\\\/o3bzqToqHP\\\\\\\/NYzR86ehcy8tnDhADah1AoNI8BJ\\\\\\\/uGNH2Eio56HHQ60InJD3b6ppvM7VGAy3iRkAp3D5LJrLZKVZtvMkaD2wbRf1R8JiplOnbEtFPT5yCsKlxNuPp3eER972rtb0vh\\\\\\\/18v6hBPFEjpz\\\\\\\/NlbmzoYhxA6JdHz9nq25NRtJQjbFzadwZaNMXfpcaqUimWbvGxF2nObeu8pEOrttLT5k9qAAVOtR23kPXouOm8vkBac6awo4SXGQeNjnbxsTr59ky2aprLcHFG3oRelQxMOgJ4g9hzL5r7cUYJn4tOLLTmpCmJQaXsz0c\\\\\\\/xq55cjK4+V5nwuCzfBgJJx\\\\\\\/R8MyiHiOduuGqBtFIWWh8aEsSbHanwU+ROaJ\\\\\\\/bLdCx\\\\\\\/Bh4f9M00ISR6UnKSQ\\\\\\\/EmDr1\\\\\\\/0u28k9xeQkm0tZC7ADGKGJFMcZIAPZ9kbSpjePvqq+3DdNhjaBwBeY6I9Qko2AwsaN5355QjUWLe\\\\\\\/wRaIzRfmhZpbyz7VFJnuNfjD\\\\\\\/OVJe33zcts1H9DZvxIbKonY4SAWilf9GDygj9LEIp2BUIg+FSsWS0QaB3PVfylCWNEiYlbzAxSFvPB+Nug+A7Sh+Pn8repFOiH1kuK2gjVi8OTRIDhQGy0Uw3fIfp3\\\\\\\/VjG\\\\\\\/lqivewV2i9glGYKUbQGgIgHiGG32twwHurwKVFCy0sLFP9QT0XDKSZjaaS\\\\\\\/fGU1dxPo7fKzO5l3pd1jVxFtJZfnFa3keqoVsFDFEC4w1M2shUE0AluK3QEyrlTnIXpM0gerUFz7bDHuobzrC0g5oN0NMEZl2\\\\\\\/IRAm3arV+Xr9\\\\\\\/x8cBm3pKhsjcHBG5N0pFzf01DMgRUxGm+nsMdGegxUmxce7CeDJyFmQIXDjzLDeGOiasAH8QQyCemtFWyTxVRuNe1k1ImY6NvBzi2BIEO7M8FaQwWyZQ6bf+Ok+LvAnrXdbveEm34HfclG1BwZeC9i7QJwzOoP\\\\\\\/BPrzG5AzeJZSsBvu8j4Nv0bpnDT1YRa+ykO8S9bxhxPfiBd1eqdf1ZrTz1vKW1ldIacAy+6f0zY9qvhH6ZIW9ZqV8XKkWzYGd\\\\\\\/IIE4OKa\\\\\\\/eYXdU7w2\\\\\\\/Ho2+afvNFpHNA4nZVC7LphrV+aynTDn+oKgAEkKmbYjxFqOStxc1II3KrkTpyA+2hBac0yrLRQQc27J7SJNQuGJ\\\\\\\/4QGgxOJSKZXDyn1EP6jF\\\\\\\/\\\\\\\/i8x082afL9CxVxVKSj+ONKRZaTuZ4ZgMlbDfsDIqlYSyD6dlOHjRCYlN10h\\\\\\\/Jbq0XfV\\\\\\\/CY13pPslzZOVZJuzeeFRmlyH2rxWdPB+NRM3MepdHl\\\\\\\/kLXR19UpMN5P14JQo2cFiJwXQX5yNpvnV3IQV8qC25qfBsqRQ+uSLHUYqxvN2hTiQ5Ujwl5eEdjseSMX5CuqjSE\\\\\\\/jxcS7k7YvGU8+VG9HtOrc\\\\\\\/GTIoMPanoGMhJLwCnc5tNcPp0Xgj8Y2qax88ECjmRFVkwYeW2AQKTfI4f1eW8HzHR31rZ7am5l3c9I9D8iFUIjeSZnOeWKYnq5nF2EzVUKxKY+GaT66SkB2ooUTCnCsWyxJFGzgcWS\\\\\\\/7q5I8bcwc1FbPPbHb1JvYUTJjANA7tK5ROH8U55A7ftxFqUOYropkBKqQpZdFOVVCSXau5U9d5+KtPRBDFPaITBy3EHreKHkvGpaurulTG2X7oeVHiwcvLiprES\\\\\\\/55IXuFsD+xaxDSzVHizmvN7IO3nNpga7W8Ws92HThVXCQKPPSpsRGaLOpdgsBdd4C3mTaAOrVVBbJPAevDYmKagx\\\\\\\/qHm1jE+75OrKOElnyuHNlwT066vu5hCljRFtiCTvJR4xaM5BQy\\\\\\\/eYMelOw1NEdtF8U3hlrSf9WQh\\\\\\\/KBrmaCyO7Q+dN02e6THUmIbxKdeYTfipcCjTMjWTMJfrGwm9t3oTWUXLpdFiVEyhm1UqbVKYu+sfdws7TQn2GsdDHw7Fi2qP6ZvPDslidIdIlYMZf+U036dkCObZs9+0ZaQMinsPThjS9inJDnNubCUTSjE4hy1o7eFHnEcVOZDf12S\\\u0022,\\\u0022iv\\\u0022:\\\u00229OsTFzMNY2s4\\\\\\\/4fQFv0SBw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022f340de3b4c1ee1c4e516499d5ad66e2824ec0d16a916dd5aa1a1bc9ade22b44e\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-486369 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/9/2919" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-9gEPgGaewNid8uLJn_iuSrS0Tscw3IZxUQAhJMAPkD0" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/9/2919" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-9JzDhUycdGJAmeihgfprY9bXgBH5bMy1b9CxDy77-Jw" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/9/2907" title="Genome-Wide Screen for Genes Involved in Caenorhabditis elegans Developmentally Timed Sleep" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/9/2931" title="ASPsiRNA: A Resource of ASP-siRNAs Having Therapeutic Potential for Human Genetic Disorders and Algorithm for Prediction of Their Inhibitory Efficacy" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="486369" id="node486369" data-pisa="ggg;7/9/2919" data-pisa-master="ggg;g3.117.041806" data-apath="/ggg/7/9/2919.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Habitat Predicts Levels of Genetic Admixture in <em>Saccharomyces cerevisiae</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Viranga Tilakaratna</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-9015-2707" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Douda Bensasson</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2919-2929;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.041806 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node486369" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Viranga Tilakaratna </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Faculty of Life Sciences, University of Manchester, M13 9PT, UK</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Viranga%2BTilakaratna%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tilakaratna%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AViranga%2BTilakaratna%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Douda Bensasson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Plant Biology, University of Georgia, Athens, Georgia 30602</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Institute of Bioinformatics, University of Georgia, Athens, Georgia 30602</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Douda%2BBensasson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bensasson%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADouda%2BBensasson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-9015-2707" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Douda Bensasson</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:dbensasson@uga.edu" class="" data-icon-position="" data-hide-link-title="0">dbensasson@uga.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/7/9/2919" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:486369" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:486369/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/9/2919.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:486369" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:486369/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/9/2919.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:486369" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:486369/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/9/2919.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:486369" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:486369/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/7/9/2919.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:486369" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:486369/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Genetic admixture can provide material for populations to adapt to local environments, and this process has played a crucial role in the domestication of plants and animals. The model yeast, <em>Saccharomyces cerevisiae</em>, has been domesticated multiple times for the production of wine, sake, beer, and bread, but the high rate of admixture between yeast lineages has so far been treated as a complication for population genomic analysis. Here, we make use of the low recombination rate at centromeres to investigate admixture in yeast using a classic Bayesian approach and a locus-by-locus phylogenetic approach. Using both approaches, we find that <em>S. cerevisiae</em> from stable oak woodland habitats are less likely to show recent genetic admixture compared with those isolated from transient habitats such as fruits, wine, or human infections. When woodland yeast strains do show recent genetic admixture, the degree of admixture is lower than in strains from other habitats. Furthermore, <em>S. cerevisiae</em> populations from oak woodlands are genetically isolated from each other, with only occasional migration between woodlands and local fruit habitats. Application of the phylogenetic approach suggests that there is a previously undetected population in North Africa that is the closest outgroup to the European <em>S. cerevisiae</em>, including the domesticated Wine population. Careful testing for admixture in <em>S. cerevisiae</em> leads to a better understanding of the underlying population structure of the species and will be important for understanding the selective processes underlying domestication in this economically important species.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/admixture" class="hw-term hw-article-keyword hw-article-keyword-admixture" rel="nofollow">admixture</a></li><li class="kwd"><a href="/keyword/mosaic" class="hw-term hw-article-keyword hw-article-keyword-mosaic" rel="nofollow">mosaic</a></li><li class="kwd"><a href="/keyword/outcrossing" class="hw-term hw-article-keyword hw-article-keyword-outcrossing" rel="nofollow">outcrossing</a></li><li class="kwd"><a href="/keyword/introgression" class="hw-term hw-article-keyword hw-article-keyword-introgression" rel="nofollow">introgression</a></li><li class="kwd"><a href="/keyword/hybridization" class="hw-term hw-article-keyword hw-article-keyword-hybridization" rel="nofollow">hybridization</a></li><li class="kwd"><a href="/keyword/yeast" class="hw-term hw-article-keyword hw-article-keyword-yeast" rel="nofollow">yeast</a></li></ul><p id="p-3">The wine yeast <em>Saccharomyces cerevisiae</em> is one of the most economically important model organisms, and is used by humans around the world to produce alcohol and to ferment foods (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Fay and Benavides 2005</a>; <a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Wang <em>et al.</em> 2012</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Gallone <em>et al.</em> 2016</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Gonçalves <em>et al.</em> 2016</a>). <em>S. cerevisiae</em> is also found in the wild on fruits, flowers, on the bark of trees in oak woodlands, and can occur as a commensal or pathogen of humans (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Sniegowski <em>et al.</em> 2002</a>; <a id="xref-ref-59-2" class="xref-bibr" href="#ref-59">Wang <em>et al.</em> 2012</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Hyma and Fay 2013</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Goddard <em>et al.</em> 2010</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Cromie <em>et al.</em> 2013</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Dashko <em>et al.</em> 2016</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Goddard and Greig 2015</a>). <em>S. cerevisiae</em> shares the oak woodland habitat with all other <em>Saccharomyces</em> species, suggesting that the woodland habitat is ancestral for the species (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Eberlein <em>et al.</em> 2015</a>). The presence of <em>S. cerevisiae</em> on a broad range of habitats compared to its closest relatives makes it an ideal model for molecular ecology, especially because many genome-wide technologies are developed and tested first on <em>S. cerevisiae</em>, providing a wealth of supporting resources to help interpret ecological patterns (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Cherry <em>et al.</em> 2011</a>).</p><p id="p-4">However, the presence of genetic admixture among natural strains of yeast presents a challenge for the use of <em>S. cerevisiae</em> as a model in population genomics (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Liti <em>et al.</em> 2009</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Almeida <em>et al.</em> 2015</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Ludlow <em>et al.</em> 2016</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Barbosa <em>et al.</em> 2016</a>). Indeed, genetic admixture also complicates the population genomic analysis of model plants (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Hufford <em>et al.</em> 2013</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Brandvain <em>et al.</em> 2014</a>) and animals (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Pool <em>et al.</em> 2012</a>), including humans (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Sankararaman <em>et al.</em> 2014</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Harris and Nielsen 2016</a>). In addition to informing population genomic analysis, the study of genetic admixture can also reveal signatures of selective processes in natural (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Brandvain <em>et al.</em> 2014</a>) and human commensal populations (<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Hufford <em>et al.</em> 2013</a>; <a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">Pool <em>et al.</em> 2012</a>). Introgressions from natural to domesticated populations can allow adaptation of crops to local habitats (<a id="xref-ref-34-3" class="xref-bibr" href="#ref-34">Hufford <em>et al.</em> 2013</a>) and has probably played an important role in animal domestication (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Marshall <em>et al.</em> 2014</a>). The study of genetic admixture with deleterious effects in natural populations also has potential applications in conservation biology (<a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Harris and Nielsen 2016</a>). In the case of <em>S. cerevisiae</em>, analysis of genetic admixture could potentially reveal mechanisms of adaptation to industrial applications and the human body, as well as the connectivity of natural populations within and between habitats.</p><p id="p-5">Past studies have employed a number of different approaches to test whether strains are “mosaics” (genetically admixed), which precludes comparison among studies or samples (<a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Liti <em>et al.</em> 2009</a>; <a id="xref-ref-59-3" class="xref-bibr" href="#ref-59">Wang <em>et al.</em> 2012</a>; <a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Cromie <em>et al.</em> 2013</a>; <a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Almeida <em>et al.</em> 2015</a>; <a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Barbosa <em>et al.</em> 2016</a>). The precise definitions differ among studies, but in general, admixed <em>S. cerevisiae</em> strains are identified as (i) those that have long branches in phylogenetic analyses and do not occur in well-supported clades with other strains (<a id="xref-ref-40-3" class="xref-bibr" href="#ref-40">Liti <em>et al.</em> 2009</a>; <a id="xref-ref-59-4" class="xref-bibr" href="#ref-59">Wang <em>et al.</em> 2012</a>), or (ii) those that are not assigned to distinct populations using a Bayesian clustering method (<a id="xref-ref-40-4" class="xref-bibr" href="#ref-40">Liti <em>et al.</em> 2009</a>; <a id="xref-ref-2-3" class="xref-bibr" href="#ref-2">Almeida <em>et al.</em> 2015</a>; <a id="xref-ref-59-5" class="xref-bibr" href="#ref-59">Wang <em>et al.</em> 2012</a>; <a id="xref-ref-15-3" class="xref-bibr" href="#ref-15">Cromie <em>et al.</em> 2013</a>; <a id="xref-ref-5-3" class="xref-bibr" href="#ref-5">Barbosa <em>et al.</em> 2016</a>). In all cases, conclusions are based on analysis of data from multiple loci concatenated into a single alignment. One drawback of both the phylogenetic and Bayesian approaches as they are usually implemented is that strains belonging to populations that are poorly represented in a sample may incorrectly be defined as mosaics.</p><p id="p-6">Here, we test for differences in the levels of admixture among <em>S. cerevisiae</em> found in different habitats using two different approaches. We employ the most commonly used Bayesian test to detect admixture in our study strains (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Pritchard <em>et al.</em> 2000</a>). In addition, we develop a locus-by-locus phylogenetic approach in which a strain only tests positive for admixture when its different loci are assigned with good statistical support to differing populations. We avoid the complicating effects of selection and recombination in our analysis by using sequences for the point centromeres of <em>S. cerevisiae</em>, which are easy to sequence, are neutrally and rapidly evolving, and have low recombination rates (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Bensasson <em>et al.</em> 2008</a>). For our sampling strategy, we focus on woodland and fruit habitats in proximity to each other and compare these naturally occurring strains to a worldwide panel of <em>S. cerevisiae</em> (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). Using complete DNA sequence for the centromeres and the flanking DNA of all 16 chromosomes from 80 <em>S. cerevisiae</em> strains, we show differences between habitats in levels of genetic admixture and that oak woodland populations are more isolated than those from other habitats.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/486397/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/486397/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Summary of the 80 <em>S. cerevisiae</em> strains used in this study</span><div class="sb-div caption-clear"></div></div></div><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Yeast strains and DNA sequencing</h3><p id="p-10">We analyzed DNA sequence data from all 16 centromeres for 80 <em>S. cerevisiae</em> strains (Supplemental Material, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041806/-/DC1/FileS1.pdf">File S1</a> and <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). Centromere sequences were already available for 33 strains (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Bensasson 2011</a>; GenBank: <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=HQ339369&amp;atom=%2Fggg%2F7%2F9%2F2919.atom">HQ339369</a>–<a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=HQ339877&amp;atom=%2Fggg%2F7%2F9%2F2919.atom">HQ339877</a>), and we reuse these data here. These previously-reported sequences were obtained from monosporic derivatives, which we expect to be completely homozygous in all parts of the genome except at the MAT locus (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Bensasson 2011</a>).</p><p id="p-11">For the remaining 47 strains, we generated monosporic derivatives by sporulating yeast and isolating single spores as described in <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Amberg <em>et al.</em> (2005)</a>. DNA was extracted, amplified, and sequenced from the monosporic derivatives using the extraction, PCR, and DNA sequencing conditions described in <a id="xref-ref-7-3" class="xref-bibr" href="#ref-7">Bensasson (2011)</a>. DNA sequence reads were assembled into a consensus DNA sequence for each strain at each locus using Staden version 1.7.0 (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Bonfield <em>et al.</em> 1995</a>) and its quality was assessed using Phred (version: 0.020425.c) as described in <a id="xref-ref-7-4" class="xref-bibr" href="#ref-7">Bensasson (2011)</a>. Low-quality bases at the ends of consensus sequences were trimmed and any other bases with a Phred-scaled quality score below q40 were masked. The methods used here and the q40 filter ensure a very low base-calling error rate (<a id="xref-ref-7-5" class="xref-bibr" href="#ref-7">Bensasson 2011</a>). The resulting centromere sequences are available in GenBank (<a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=KT206234&amp;atom=%2Fggg%2F7%2F9%2F2919.atom">KT206234</a>–<a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=KT206982&amp;atom=%2Fggg%2F7%2F9%2F2919.atom">KT206982</a>). Sequences were manually aligned and visualized in SeaView 4.0 (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Gouy <em>et al.</em> 2010</a>).</p></div><div id="sec-3" class="subsection"><h3>Phylogenetic analysis</h3><p id="p-12">Alignments for all 16 centromere loci were concatenated into a single long alignment using a custom perl script (alcat.pl). Genetic distances between DNA sequences were estimated and analyzed using the ape package (version 3.5 <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Paradis 2011</a>) in R (version 3.3.0). More specifically, we estimated genetic distance using the F84 model (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Felsenstein and Churchill 1996</a>) as implemented in dist.dna and constructed a neighbor joining tree (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Saitou and Nei 1987</a>) from these distances using nj (<a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Paradis 2011</a>). DNA sequence data were bootstrapped using boot.phylo with 10,000 replicates to test the statistical support of the clades obtained. The resulting phylogram with associated bootstrap values was visualized and colored using plot.phylo.</p><p id="p-13">A second phylogenetic analysis was performed, after excluding strains showing recent genetic admixture (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041806/-/DC1/FileS1.pdf">File S1</a>). We conducted a phylogenetic analysis of the concatenated alignment of data for all loci using the maximum likelihood approach implemented in RAxML (version 8.2.4; <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Stamatakis 2014</a>). We used a general time reversible model with a γ distribution to estimate rate heterogeneity at sites from the data (GTRGAMMA), and a rapid bootstrap analysis for 10,000 bootstrap replicates with a search for the best-scoring Maximum Likelihood tree in the same RAxML run.</p></div><div id="sec-4" class="subsection"><h3>Population structure analysis</h3><p id="p-14">We tested for population structure within our sample of 80 yeast strains using the software package <em>structure</em> (version 2.3.4) with a model taking account of linkage between polymorphisms at the same locus (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Falush <em>et al.</em> 2003</a>), and assuming that the DNA sequences were haploid. Using <em>structure</em>, we estimated the most likely number of populations to explain the data (K) by varying K from 1 to 10 and visualizing the results. The linkage model allows individuals to show admixture between the K different populations (INFERALPHA 1), and we ran it with default parameters: a burnin of 10,000 steps followed by a run length of 20,000 steps. In pilot experiments, we found that increasing the length of the burnin from 10,000 to 100,000 did not alter our conclusions. Runs were repeated for each value of K five times. There are multiple methods for deciding the value of K that best describes the data (<a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">Pritchard <em>et al.</em> 2000</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Evanno <em>et al.</em> 2005</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Hubisz <em>et al.</em> 2009</a>); here, we use the method recommended by Pritchard and others in the original <em>structure</em> publication (<a id="xref-ref-47-3" class="xref-bibr" href="#ref-47">Pritchard <em>et al.</em> 2000</a>) and more recently in <a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Hubisz <em>et al.</em> (2009</a>). More specifically, we identified the most likely models across the 50 runs and from these we selected the model with the lowest value of K (<a id="xref-ref-47-4" class="xref-bibr" href="#ref-47">Pritchard <em>et al.</em> 2000</a>; <a id="xref-ref-33-3" class="xref-bibr" href="#ref-33">Hubisz <em>et al.</em> 2009</a>). We checked that all the models that were similarly likely had higher values of <em>K</em>. In addition, we verified that the most likely models with larger values of K resulted in groupings of strains with respect to their sampling location that are similar to the best model discussed in the <em>Results</em>.</p><p id="p-15">The <em>structure</em> package was run using three perl scripts available at <a href="https://github.com/bensassonlab/scripts">https://github.com/bensassonlab/scripts</a>: (i) structureInfile.pl converts sequence alignments (in fasta format) to <em>structure</em> input files that summarize bases at variable sites; (ii) structureShell.pl runs <em>structure</em> one time for each value of K in a specified range (from 1 to 10 in this study); and (iii) structurePrint.pl plots the structure results as barplots using R and allows for user specification of colors (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/9/2919/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Identification of seven S. cerevisiae populations when including the subpopulations within Europe and the USA. A breakdown of population assignments defined by the most likely models estimated by structure analysis. Reference strains for the detection of admixture in subsequent analyses are highlighted with a &#x201C;*&#x201D;. (a) Five major global populations identified in the worldwide sample. Most of our worldwide sample of 80 strains can be assigned to five distinct populations: &#x201C;1&#x201D; is the Malaysia population (blue), &#x201C;2&#x201D; is the population from Sake (red), and &#x201C;3&#x201D; is the West Africa population (orange), as well as strains from Europe (brown) and the USA (green). structure also invoked some gene flow from a sixth (K = 6) population (yellow); however, we did not encounter any strains that clearly represent this population. (b) Two subpopulations identified in Europe. Analysis of the 23 strains that were isolated from European oak trees or were previously assigned to a European Wine population (Liti et al. 2009) shows that there are at least two S. cerevisiae subpopulations in Europe (K = 3): population &#x201C;4&#x201D; from wine (purple) and population &#x201C;5&#x201D; from European Oaks (green). (c) Two subpopulations identified in the USA. Analysis of the 24 strains isolated from oaks in the USA shows that there are two S. cerevisiae subpopulations in the USA (K = 2): population &#x201C;6&#x201D; from Pennsylvanian Oaks (dark green) and population &#x201C;7&#x201D; from North Carolina Oaks (light green)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1671202662" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Identification of seven &lt;em&gt;S. cerevisiae&lt;/em&gt; populations when including the subpopulations within Europe and the USA. A breakdown of population assignments defined by the most likely models estimated by &lt;em&gt;structure&lt;/em&gt; analysis. Reference strains for the detection of admixture in subsequent analyses are highlighted with a &#x201C;*&#x201D;. (a) Five major global populations identified in the worldwide sample. Most of our worldwide sample of 80 strains can be assigned to five distinct populations: &#x201C;1&#x201D; is the Malaysia population (blue), &#x201C;2&#x201D; is the population from Sake (red), and &#x201C;3&#x201D; is the West Africa population (orange), as well as strains from Europe (brown) and the USA (green). &lt;em&gt;structure&lt;/em&gt; also invoked some gene flow from a sixth (&lt;em&gt;K&lt;/em&gt; = 6) population (yellow); however, we did not encounter any strains that clearly represent this population. (b) Two subpopulations identified in Europe. Analysis of the 23 strains that were isolated from European oak trees or were previously assigned to a European Wine population (Liti &lt;em&gt;et al.&lt;/em&gt; 2009) shows that there are at least two &lt;em&gt;S. cerevisiae&lt;/em&gt; subpopulations in Europe (K = 3): population &#x201C;4&#x201D; from wine (purple) and population &#x201C;5&#x201D; from European Oaks (green). (c) Two subpopulations identified in the USA. Analysis of the 24 strains isolated from oaks in the USA shows that there are two &lt;em&gt;S. cerevisiae&lt;/em&gt; subpopulations in the USA (K = 2): population &#x201C;6&#x201D; from Pennsylvanian Oaks (dark green) and population &#x201C;7&#x201D; from North Carolina Oaks (light green).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/9/2919/F1.medium.gif" width="427" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/7/9/2919/F1.medium.gif" width="427" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/9/2919/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/9/2919/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/486459" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-16" class="first-child">Identification of seven <em>S. cerevisiae</em> populations when including the subpopulations within Europe and the USA. A breakdown of population assignments defined by the most likely models estimated by <em>structure</em> analysis. Reference strains for the detection of admixture in subsequent analyses are highlighted with a “*”. (a) Five major global populations identified in the worldwide sample. Most of our worldwide sample of 80 strains can be assigned to five distinct populations: “1” is the Malaysia population (blue), “2” is the population from Sake (red), and “3” is the West Africa population (orange), as well as strains from Europe (brown) and the USA (green). <em>structure</em> also invoked some gene flow from a sixth (<em>K</em> = 6) population (yellow); however, we did not encounter any strains that clearly represent this population. (b) Two subpopulations identified in Europe. Analysis of the 23 strains that were isolated from European oak trees or were previously assigned to a European Wine population (<a id="xref-ref-40-6" class="xref-bibr" href="#ref-40">Liti <em>et al.</em> 2009</a>) shows that there are at least two <em>S. cerevisiae</em> subpopulations in Europe (K = 3): population “4” from wine (purple) and population “5” from European Oaks (green). (c) Two subpopulations identified in the USA. Analysis of the 24 strains isolated from oaks in the USA shows that there are two <em>S. cerevisiae</em> subpopulations in the USA (K = 2): population “6” from Pennsylvanian Oaks (dark green) and population “7” from North Carolina Oaks (light green).</p><div class="sb-div caption-clear"></div></div></div><p id="p-17">Our preliminary phylogenetic analysis showed well-supported subpopulations within Europe and the USA (at least 95% bootstrap support for all four subpopulations in <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>) that are missed by the initial <em>structure</em> analysis (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1a</a>). Following the recommendations in the <em>structure</em> documentation for identifying subpopulations, we repeated the above <em>structure</em> analysis of 50 runs (K = 1 to 10, 5 replicates) on two subsets of the data: (i) on the 23 strains that were either assigned to the European Wine population by <a id="xref-ref-40-7" class="xref-bibr" href="#ref-40">Liti <em>et al.</em> (2009)</a> or were isolated from European oak trees (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1b</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041806/-/DC1/FileS1.pdf">File S1</a>); and (ii) on the 24 strains isolated from oaks in the USA (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1c</a> and <a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>). The worldwide sample of strains used here (<a id="xref-table-wrap-1-5" class="xref-table" href="#T1">Table 1</a>) and by others (<a id="xref-ref-40-8" class="xref-bibr" href="#ref-40">Liti <em>et al.</em> 2009</a>) includes too few strains in the remaining three populations to permit testing for further subpopulations. <a id="xref-ref-40-9" class="xref-bibr" href="#ref-40">Liti <em>et al.</em> (2009)</a> assigned only three strains to the “Malaysia” population, three strains to the “Sake” population, and two strains to the “West Africa” population, and our analysis of the same worldwide sample of strains supports these population designations (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1</a> and <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/9/2919/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Neighbor joining distance analysis of all strains shows subpopulations within the USA and Europe. Bootstrap support was estimated from 10,000 bootstrap replicates and support is shown as a percentage for clades with over 70% support. Strain names and clades are colored according to populations defined by the structure analysis shown in Figure 1A. Bootstrap support for the European and USA populations is &lt; 70%; however, there is strong bootstrap support (&#x2265; 95%) for subpopulations within the USA and Europe." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1671202662" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Neighbor joining distance analysis of all strains shows subpopulations within the USA and Europe. Bootstrap support was estimated from 10,000 bootstrap replicates and support is shown as a percentage for clades with over 70% support. Strain names and clades are colored according to populations defined by the &lt;em&gt;structure&lt;/em&gt; analysis shown in Figure 1A. Bootstrap support for the European and USA populations is &lt; 70%; however, there is strong bootstrap support (&#x2265; 95%) for subpopulations within the USA and Europe.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/9/2919/F2.medium.gif" width="440" height="424"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/7/9/2919/F2.medium.gif" width="440" height="424"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/9/2919/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/9/2919/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/486544" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-18" class="first-child">Neighbor joining distance analysis of all strains shows subpopulations within the USA and Europe. Bootstrap support was estimated from 10,000 bootstrap replicates and support is shown as a percentage for clades with over 70% support. Strain names and clades are colored according to populations defined by the <em>structure</em> analysis shown in <a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1A</a>. Bootstrap support for the European and USA populations is &lt; 70%; however, there is strong bootstrap support (≥ 95%) for subpopulations within the USA and Europe.</p><div class="sb-div caption-clear"></div></div></div><p id="p-19">In order to test whether the <em>structure</em> analyses make meaningful population predictions, we used ObStruct (version 1.0; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Gayevskiy <em>et al.</em> 2014</a>). ObStruct tests whether ancestry estimates (<a id="xref-fig-1-7" class="xref-fig" href="#F1">Figure 1</a>) were correlated with the population assignments that we expect based on the geographic locations shown in <a id="xref-table-wrap-1-6" class="xref-table" href="#T1">Table 1</a> or on past assignment to the European Wine population (<a id="xref-fig-1-8" class="xref-fig" href="#F1">Figure 1c</a>; <a id="xref-ref-40-10" class="xref-bibr" href="#ref-40">Liti <em>et al.</em> 2009</a>). We were able to apply ObStruct to the best <em>structure</em> models because no sampling location information was used in generating them (<a id="xref-fig-1-9" class="xref-fig" href="#F1">Figure 1</a>). For our ObStruct analysis of the initial model generated by <em>structure</em> (<a id="xref-fig-1-10" class="xref-fig" href="#F1">Figure 1a</a>), we included only our 49 study strains and excluded the worldwide panel of 31 strains, many of which had uncertain sampling locations (<a id="xref-table-wrap-1-7" class="xref-table" href="#T1">Table 1</a>).</p></div><div id="sec-5" class="subsection"><h3>Testing for recent genetic admixture</h3><p id="p-20">In order to test for genetic admixture between populations, we first defined distinct <em>S. cerevisiae</em> populations on the basis of <em>structure</em> and phylogenetic analyses (<a id="xref-fig-1-11" class="xref-fig" href="#F1">Figure 1</a> and <a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a>). From each of the seven different populations identified in this way, we chose two reference strains to define each population (the minimum number that <em>structure</em> needs to estimate allele frequencies within a population). Where possible, reference strains were chosen from strains that were assigned to a particular population in an independent study (<a id="xref-ref-40-11" class="xref-bibr" href="#ref-40">Liti <em>et al.</em> 2009</a>). <a id="xref-ref-40-12" class="xref-bibr" href="#ref-40">Liti <em>et al.</em> (2009)</a> assigned more than two strains to the Malaysian, Sake, and Wine populations, and in these cases, for our reference strains we selected strains isolated from the habitat described by the population name: strains isolated from wine for the Wine population or from sake for the Sake population, and in cases where more than two strains fit these criteria, we selected the strains with the highest estimates of genetic ancestry for their population. For example, for the Wine population, we chose strains DBVPG1106m and L1374m, which were previously assigned to the Wine population by <a id="xref-ref-40-13" class="xref-bibr" href="#ref-40">Liti <em>et al.</em> (2009)</a> and were isolated from wine. For the populations that were not identified by <a id="xref-ref-40-14" class="xref-bibr" href="#ref-40">Liti <em>et al.</em> (2009)</a>, European Oak and North Carolina Oak, reference strains were those with the highest estimates of genetic ancestry for their respective population (<a id="xref-fig-1-12" class="xref-fig" href="#F1">Figure 1, b and c</a>).</p><p id="p-21">After removing the 14 reference strains, the remaining 66 strains in this study were assigned to populations or defined as showing recent genetic admixture using two independent approaches. The first approach uses <em>structure</em> to estimate levels of admixture for every strain with reference strains to define distinct populations using the USEPOPINFO option (<a id="xref-ref-47-5" class="xref-bibr" href="#ref-47">Pritchard <em>et al.</em> 2000</a>). It was necessary to define populations in this way because the initial runs of <em>structure</em> missed the subpopulations within Europe and North America. Preliminary analysis of the results of our initial 50 replicate runs of <em>structure</em> suggested that, in cases where population substructure was missed (European Oak and North Carolina Oak populations), the admixture invoked to explain the divergence of these populations from the rest was inconsistent. For example, in different replicate runs of <em>structure</em>, admixture from a single other population was invoked to explain the divergence of European Oak strains from the Wine population, but the source of the admixture was inconsistent: sometimes Malaysia, Sake, or West Africa. From this, we concluded that admixture is sometimes invoked by <em>structure</em> to explain divergence between populations. Therefore, we developed a second locus-by-locus phylogenetic test for admixture that only classifies strains as admixed if they show well-supported similarity to multiple populations, and are thus more conservative. Details of both approaches are described in the following sections.</p><div id="sec-6" class="subsection"><h4>Detecting admixture using structure:</h4><p id="p-22">We ran <em>structure</em> using the same parameters as we used for defining populations above, except that we used the USEPOPINFO model in <em>structure</em> (<a id="xref-ref-47-6" class="xref-bibr" href="#ref-47">Pritchard <em>et al.</em> 2000</a>) to estimate ancestry for the 66 <em>S. cerevisiae</em> strains of unknown origin (POPFLAG = 0) by prespecifying the population of origin for the 14 reference strains (POPFLAG = 1). We set the number of populations to seven (K = 7), and selected the breakdown of ancestry components for each strain from the most likely model out of 20 independent runs. For consistency across our two approaches, we defined a strain as “admixed” if its proportion of ancestry to a single population was &lt; 0.94 (equivalent to 1 out of 16 centromere loci being from a different population).</p></div><div id="sec-7" class="subsection"><h4>Detecting admixture using a locus-by-locus phylogenetic approach:</h4><p id="p-23">We also performed locus-by-locus phylogenetic analyses of all 80 strains, assigning each of the 66 nonreference strains at each locus to a population according to which of the 14 reference strains it grouped with. We used custom perl scripts to run the phylogenetic analysis using the ape package in R (chrbychr.pl and chrbychr2.pl). For each locus, we constructed a neighbor joining tree of genetic distances using the F84 model, used 10,000 bootstrap replicates to assess statistical support for each clade, and output a text summary of the strains found in each clade using the prop.part tool of boot.phylo in ape (<a id="xref-ref-44-3" class="xref-bibr" href="#ref-44">Paradis 2011</a>). We only considered clades that had at least 70% bootstrap support. For each locus, strains found in clades with reference strains from only one population were assigned to the same population as the reference strains. Because of the limited phylogenetic resolution at some loci, we also assigned strains in clades with reference strains only from the European Oak or Wine populations as “European” and those in clades with North Carolina or Pennsylvania reference strains as “USA.” In cases where a sequence does not group with those of reference strains belonging to a single population, its population status at that locus is classed as “undefined.” The more general classifications of European or USA do not conflict with subpopulation classifications within those groups, and loci classed as undefined do not conflict with classifications at any other loci. We then compared population predictions across all loci for a strain, and if a strain was assigned to a population at a locus that conflicted with the population assignment at any of the other loci, then that strain was defined as showing genetic admixture. For example, SDO8s1 was isolated from a North Carolina oak, and has a CEN4 sequence from the Wine clade, but it has five loci that are in the same clade as the North Carolina Oak reference strains (the remaining loci were undefined or in the USA clade; Figure S1). Therefore, this strain shows admixture between Wine and North Carolina Oak populations.</p><p id="p-24">In most cases (55 out of 66 strains), the two approaches resulted in the same population and admixture assignments. Fewer strains were defined as admixed using the locus-by-locus phylogenetic approach (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>), because a locus is classed as undefined when there is insufficient data at a locus for population assignment. Therefore, we used this more conservative approach to decide which strains to exclude from the final phylogenetic analysis.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/486387/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/486387/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title"><em>S. cerevisiae</em> from trees in oak woodlands show less genetic admixture than those from other habitats</span><div class="sb-div caption-clear"></div></div></div><p id="p-26">As an experimental control to test whether the number and choice of reference strains might affect our results and conclusions, we also tested for recent genetic admixture using both the <em>structure</em> and the locus-by-locus phylogenetic approach, but with a much longer list of 38 reference strains. These 38 strains included all the strains assigned to five populations in an independent study (<a id="xref-ref-40-15" class="xref-bibr" href="#ref-40">Liti <em>et al.</em> 2009</a>), and the strains assigned with the highest confidence (ancestry <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2919/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2919/embed/mml-math-5.gif"/></noscript></span></span>) to the North Carolina or European Oak subpopulations. Our findings and conclusions were unchanged in these control runs, except that we necessarily missed cases of genetic admixture in strains that were included in this larger set of reference strains. In addition, we tested the effect of simply using the two strains with the highest estimates of genetic ancestry for each population as reference strains. Using the locus-by-locus phylogenetic approach, this resulted in a different classification for only one strain (K11 was classed as admixed and not a Sake strain), and designations for four strains were affected by the choice of reference strains with the <em>structure</em> approach. In all cases, there were only negligible effects on the proportions of strains that were admixed from different habitats, and none of our findings or conclusions were affected. We did not use these control analyses further.</p></div></div><div id="sec-8" class="subsection"><h3>Data availability</h3><p id="p-27">DNA sequences determined for this study are available in GenBank: <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=KT206234&amp;atom=%2Fggg%2F7%2F9%2F2919.atom">KT206234</a>–<a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=KT206982.&amp;atom=%2Fggg%2F7%2F9%2F2919.atom">KT206982.</a> Perl scripts are available at <a href="https://github.com/bensassonlab/scripts">https://github.com/bensassonlab/scripts</a>, and locus-by-locus alignments and tree file data are available at <a href="https://github.com/bensassonlab/data">https://github.com/bensassonlab/data</a>. Most of the yeast strains used in this study are available from the National Collection of Yeast Cultures in the U.K. or the Portuguese Yeast Culture Collection.</p></div></div><div class="section results" id="sec-9"><h2 class="">Results</h2><div id="sec-10" class="subsection"><h3>Seven genetically distinct populations of S. cerevisiae</h3><p id="p-28">We generated complete sequence data for whole centromeres from all 16 chromosomes of 47 <em>S. cerevisiae</em> strains from oak woodlands and fruit in the USA and Europe. We compared these sequences with a similar dataset previously described in <a id="xref-ref-7-6" class="xref-bibr" href="#ref-7">Bensasson (2011)</a> that includes 33 strains collected worldwide (<a id="xref-table-wrap-1-8" class="xref-table" href="#T1">Table 1</a>). The data reused here showed that, in <em>S. cerevisiae</em>, centromeres are small (up to 125 bp long), rapidly and neutrally evolving, and have low recombination rates (<a id="xref-ref-7-7" class="xref-bibr" href="#ref-7">Bensasson 2011</a>). By analyzing centromeres and their flanking DNA, every yeast chromosome is represented in our analysis. This strategy also minimizes the challenges to phylogenetic and population structure analysis presented by recombination and positive selection (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Avise 1994</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Posada and Crandall 2002</a>; <a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">Brandvain <em>et al.</em> 2014</a>) that we would expect in a genome-wide analysis.</p><p id="p-29">Using the Markov Chain Monte Carlo approach implemented in the program <em>structure</em> (<a id="xref-ref-47-7" class="xref-bibr" href="#ref-47">Pritchard <em>et al.</em> 2000</a>; <a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Falush <em>et al.</em> 2003</a>), we analyzed data from 895 segregating sites in 13 kb of centromere sequence and identified five main populations in our sample of 80 strains (<a id="xref-fig-1-13" class="xref-fig" href="#F1">Figure 1a</a>). Although <em>structure</em> used no sampling location information, the assignment of strains to these five populations (ancestry estimates) was highly correlated with geographic sampling location (ObStruct: <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2919/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2919/embed/mml-math-6.gif"/></noscript></span></span>). These populations also recapitulate those previously described in the worldwide sample of 33 strains: Malaysia, Sake, West Africa, Europe, and the USA (<a id="xref-ref-40-16" class="xref-bibr" href="#ref-40">Liti <em>et al.</em> 2009</a>). Although these five populations are evident in our phylogenetic analysis of the same dataset, there is not good statistical support for the European and USA clades. However, there do appear to be well-supported distinct subpopulations of <em>S. cerevisiae</em> within both Europe and the USA (bootstrap values <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2919/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2919/embed/mml-math-7.gif"/></noscript></span></span> <a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2</a>). These subpopulations are not identified by <em>structure</em> in any of the models we obtained, even when invoking a larger number of populations. Therefore, we used a hierarchical approach as recommended in the <em>structure</em> documentation to test for population substructure within a subset of European strains (<span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2919/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2919/embed/mml-math-8.gif"/></noscript></span></span> <a id="xref-fig-1-14" class="xref-fig" href="#F1">Figure 1b</a>) and within strains from the USA (<span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2919/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2919/embed/mml-math-9.gif"/></noscript></span></span> <a id="xref-fig-1-15" class="xref-fig" href="#F1">Figure 1c</a>). This analysis revealed two subpopulations within Europe (Wine and European Oak; ObStruct: <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2919/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2919/embed/mml-math-10.gif"/></noscript></span></span>), and two subpopulations within the USA (Pennsylvania Oak and North Carolina Oak; ObStruct: <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2919/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2919/embed/mml-math-11.gif"/></noscript></span></span>). Overall, using this hierarchical <em>structure</em> approach, we identified a total of seven populations, and these populations were also represented with well-supported clades in our phylogenetic analysis (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2</a>).</p></div><div id="sec-11" class="subsection"><h3>A conservative test for recent genetic admixture</h3><p id="p-30">Using the seven populations identified above, we used two approaches to test for genetic admixture in each strain in our dataset. The first approach applies the standard software, <em>structure</em>, in a way that is similar to that used by others in the yeast research community to estimate levels of genetic admixture given a panel of reference strains. The initial <em>structure</em> analysis that we used to define yeast populations gave inconsistent results invoking admixture from different populations into Europe or North America from one replicate run to the next (see <em>Materials and Methods</em>). Therefore, we also developed a second locus-by-locus phylogenetic approach that would only invoke admixture for a strain if it has haplotypes at loci that are assigned with statistical confidence to differing known populations.</p><p id="p-31">Analysis of admixture results for individual strains showed that 55 out of 66 strains were defined concordantly by the two methods (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041806/-/DC1/FileS1.pdf">File S1</a>). In most of the remaining 11 cases, only <em>structure</em> invoked admixture in European strains (nine strains, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041806/-/DC1/FileS1.pdf">File S1</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041806/-/DC1/FileS2.tsv">File S2</a>). Investigation of all nine strains where only <em>structure</em> invokes admixture suggests that these are likely to be false positives because there was no strong or consistent phylogenetic support for their similarity to multiple populations (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041806/-/DC1/FileS2.tsv">File S2</a>). For some of these strains, <em>structure</em> appeared to invoke admixture to explain genetic divergence from known populations. For example, DBVPG1853m, a strain from North Africa, is most similar to European populations of <em>S. cerevisiae</em> yet it is also somewhat diverged from them (<a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2</a> and <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). Locus-by-locus analysis showed that it is diverged from European strains, and most similar to the Wine or European populations (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041806/-/DC1/FileS2.tsv">File S2</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041806/-/DC1/FileS3.pdf">File S3</a>). For this strain, <em>structure</em> invokes admixture between the Wine, European Oak, and Sake populations; however, we see no evidence for greater similarity to European Oak or Sake than other populations at any single locus (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041806/-/DC1/FileS3.pdf">File S3</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/9/2919/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Phylogenetic analysis of nonadmixed strains reveals structured populations with occasional migration. We excluded 22 recently admixed strains out of the 80 strains in the original dataset, and estimated the phylogeny using a maximum likelihood approach with bootstrap support shown for clades with &gt; 70% support out of 10,000 bootstrap replicates. Clades are colored according to their genetic population, and strains are labeled with a colored dot according to their habitat. DBVPG1853m, which is not clearly assigned to a European population in this maximum likelihood analysis, appeared most similar to the Wine population in the locus-by-locus analysis. Strains isolated from oak trees are similar to strains isolated from the same woodland and distinct from those isolated from other regions. There is some migration of S. cerevisiae from the North Carolina Oak population onto North Carolina grapes (prefixed &#x201C;ARN&#x201D;). In addition, all three strains isolated from oak trees in Aldeia das Dez in Portugal (prefixed &#x201C;ZP57&#x201D;) had migrated from the Wine population to trees." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1671202662" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Phylogenetic analysis of nonadmixed strains reveals structured populations with occasional migration. We excluded 22 recently admixed strains out of the 80 strains in the original dataset, and estimated the phylogeny using a maximum likelihood approach with bootstrap support shown for clades with &gt; 70% support out of 10,000 bootstrap replicates. Clades are colored according to their genetic population, and strains are labeled with a colored dot according to their habitat. DBVPG1853m, which is not clearly assigned to a European population in this maximum likelihood analysis, appeared most similar to the Wine population in the locus-by-locus analysis. Strains isolated from oak trees are similar to strains isolated from the same woodland and distinct from those isolated from other regions. There is some migration of &lt;em&gt;S. cerevisiae&lt;/em&gt; from the North Carolina Oak population onto North Carolina grapes (prefixed &#x201C;ARN&#x201D;). In addition, all three strains isolated from oak trees in Aldeia das Dez in Portugal (prefixed &#x201C;ZP57&#x201D;) had migrated from the Wine population to trees.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/9/2919/F3.medium.gif" width="440" height="437"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/7/9/2919/F3.medium.gif" width="440" height="437"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/9/2919/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/9/2919/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/486395" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-32" class="first-child">Phylogenetic analysis of nonadmixed strains reveals structured populations with occasional migration. We excluded 22 recently admixed strains out of the 80 strains in the original dataset, and estimated the phylogeny using a maximum likelihood approach with bootstrap support shown for clades with &gt; 70% support out of 10,000 bootstrap replicates. Clades are colored according to their genetic population, and strains are labeled with a colored dot according to their habitat. DBVPG1853m, which is not clearly assigned to a European population in this maximum likelihood analysis, appeared most similar to the Wine population in the locus-by-locus analysis. Strains isolated from oak trees are similar to strains isolated from the same woodland and distinct from those isolated from other regions. There is some migration of <em>S. cerevisiae</em> from the North Carolina Oak population onto North Carolina grapes (prefixed “ARN”). In addition, all three strains isolated from oak trees in Aldeia das Dez in Portugal (prefixed “ZP57”) had migrated from the Wine population to trees.</p><div class="sb-div caption-clear"></div></div></div><p id="p-33">In other cases, similarity between subpopulations can also lead to the potentially incorrect inference of admixture using the <em>structure</em> approach. For example, the wine strain RM11 consistently clustered with Wine or European strains at all loci where there was sufficient data to assign a population using the phylogenetic approach (12 loci, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041806/-/DC1/FileS2.tsv">File S2</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041806/-/DC1/FileS3.pdf">File S3</a>). However, <em>structure</em> invoked admixture between the European Oak and Wine populations, even though there is no evidence that RM11 is more similar to the European Oak population than the Wine population at any loci (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041806/-/DC1/FileS3.pdf">File S3</a>).</p><p id="p-34">Two strains (SDO8s1 and SDO9s1) were defined as admixed using the locus-by-locus phylogenetic approach, but not using the <em>structure</em> approach. For both strains, we found strong evidence for admixture from the Wine population to the North Carolina Oak population at a single locus (CEN4, bootstrap support of at least 98%, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041806/-/DC1/FileS2.tsv">File S2</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041806/-/DC1/FileS3.pdf">File S3</a>). Using the <em>structure</em> approach, we also detected some admixture from Wine to these two North Carolina Oak strains (4.7 and 5.5%), but these levels were below our threshold for defining admixture (one locus out of 16, 6.25%). This analysis of the differences in admixture calls between the two methods suggests that the locus-by-locus phylogenetic method is less likely to call false positives, while still being sensitive enough to detect low levels of admixture within a genome. Therefore, we used the admixture definitions from the locus-by-locus phylogenetic method for subsequent analyses.</p></div><div id="sec-12" class="subsection"><h3>Genetic admixture is high in transient and low in stable habitats of S. cerevisiae</h3><p id="p-35">Because <em>S. cerevisiae</em> occurs in a broad range of habitats, we can use it as a model organism to test whether there is an effect of habitat on levels of genetic admixture. Fruit, flowers, and insects represent habitats that are transient for yeast (<a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Goddard and Greig 2015</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Knight and Goddard 2016</a>) and <em>S. cerevisiae</em> also occurs in humans only transiently (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Enache-Angoulvant and Hennequin 2005</a>). In contrast, trees are probably undisturbed for decades or even centuries (<a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Knight and Goddard 2016</a>). Therefore, we classified habitats in the worldwide sample of 80 strains according to whether they represent transient natural habitats such as (i) fruit or (ii) human infections, whether their provenance was less well defined from (iii) fermentations, or (iv) soil and unknown sources, or (v) whether they were sampled from well-defined stable woodland habitats (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>).</p><p id="p-36">We found that the proportion of admixed strains is lower for strains from oak woodlands (11% of 36 strains) than for those of other habitats (<a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>) using both the <em>structure</em> (Fisher’s exact test, <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2919/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2919/embed/mml-math-12.gif"/></noscript></span></span>) and the locus-by-locus phylogenetic approaches (Fisher’s exact test, <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2919/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2919/embed/mml-math-13.gif"/></noscript></span></span>). When we excluded the <em>S. cerevisiae</em> from oaks or other trees, the proportion of admixed strains is similar across all habitats (59% of 44 strains, <em>structure</em> analysis: <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2919/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2919/embed/mml-math-14.gif"/></noscript></span></span> 41% of 44 strains, locus-by-locus analysis: <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2919/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2919/embed/mml-math-15.gif"/></noscript></span></span>), suggesting that it is mainly in the woodland habitat that genetic admixture is peculiarly low (<a id="xref-table-wrap-2-4" class="xref-table" href="#T2">Table 2</a>). Therefore, our data suggest that levels of genetic admixture are high in transient and low in stable habitats (<a id="xref-table-wrap-2-5" class="xref-table" href="#T2">Table 2</a>).</p><p id="p-37">Many of the admixed <em>S. cerevisiae</em> strains defined using the locus-by-locus phylogenetic method (8 out of 22 strains) showed a complex pattern of admixture involving at least three of the seven defined populations (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041806/-/DC1/FileS2.tsv">File S2</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041806/-/DC1/FileS3.pdf">File S3</a>). This suggests that the admixture we see in <em>S. cerevisiae</em> is not simply a consequence of recent hybridization resulting in the asexual descendants of F1 individuals. Furthermore, we were able to detect admixture only at a single locus for eight strains, which suggests that in some cases backcrossing has occurred between hybrids and strains from single populations. All four admixed strains from woodlands were isolated from North Carolina Oaks and showed admixture at a single locus, whereas admixture at only single loci occurred less often in strains from other habitats (4 out of 18 strains; Fisher’s exact test, <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2919/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2919/embed/mml-math-16.gif"/></noscript></span></span>). The admixed haplotypes observed in these four North Carolina Oak strains were all identical to (Wine or Malaysian) haplotypes from strains that were isolated from nearby North Carolina grapes. This implies that our results are explained by recent genetic admixture between North Carolina woodlands and local vineyards and not by incomplete lineage sorting. When strains showed admixture at only one locus, the admixture occurred at different loci for nonwoodland strains, but for the four strains from North Carolina Oaks they all showed evidence of admixture at the same locus (CEN4). Indeed, the CEN4 sequence was identical for three of these strains, suggesting that some of this admixture seen in oak strains does not result from independent events, and therefore that we could be overestimating the frequency of admixture in the oak habitat. Overall, the pattern of admixture observed suggests that the degree of admixture, as well as the frequency of admixture, could be lower in oak woodland habitats than in strains from other habitats, and that when admixture does occur it probably originates from nearby yeast strains on fruit.</p></div><div id="sec-13" class="subsection"><h3>Distinct populations and low migration in woodland habitats</h3><p id="p-38">The inclusion of DNA sequences that show recent genetic admixture can lead to incorrect phylogenetic estimation, especially when admixture occurred recently between diverged populations (<a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">Posada and Crandall 2002</a>), as it does in this study. Thus, to better understand the true relationships between <em>S. cerevisiae</em> populations, it is necessary to reduce the effects of recent genetic admixture as much as possible. Therefore, we performed a phylogenetic analysis of the 58 strains in our dataset that did not show recent genetic admixture in our locus-by-locus phylogenetic analysis (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>). Maximum likelihood phylogenetic reconstruction then showed that all but one of these 58 strains can be assigned to the seven known populations. This tree shows none of the long branch mosaic lineages reported in previous genome-wide analysis (<a id="xref-ref-40-17" class="xref-bibr" href="#ref-40">Liti <em>et al.</em> 2009</a>), suggesting that our admixture filtering was successful.</p><p id="p-39">One strain isolated from white teff grain in North Africa (DBVPG1853m) is not similar to any of the seven study populations. The bootstrap support in the analysis of pooled data from all loci (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a>) and in locus-by-locus analyses (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041806/-/DC1/FileS3.pdf">File S3</a>) suggests that this strain may be a strain from an undersampled population that represents the closest outgroup to the Wine and European Oak populations of <em>S. cerevisiae</em>.</p><p id="p-40">In addition, by overlaying habitat on our tree, phylogenetic analysis in the absence of admixture permits the identification of strains that have potentially migrated between habitats (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3</a>). Yeast strains from different oak woodlands mostly form distinct populations and differ from the strains of other habitats (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3</a>) and we found no evidence for the migration of yeast between oak woodland populations (<a id="xref-fig-3-6" class="xref-fig" href="#F3">Figure 3</a>). However, there is evidence for the migration of yeast (i) from the North Carolina Oak population into a local vineyard, (ii) from the Wine population to oak trees, and (iii) from the Wine population to a medley of regions and substrates (<a id="xref-fig-3-7" class="xref-fig" href="#F3">Figure 3</a>). Together, these observations suggest that strains from the Wine population migrate more between continents and habitats than the strains with woodland genotypes.</p></div></div><div class="section discussion" id="sec-14"><h2 class="">Discussion</h2><p id="p-41">The wine yeast <em>S. cerevisiae</em> has tremendous potential as a model for molecular ecology because it occurs naturally in several distinct habitats including fruit, flowers, insects, and the bark of oak trees (<a id="xref-ref-56-3" class="xref-bibr" href="#ref-56">Sniegowski <em>et al.</em> 2002</a>; <a id="xref-ref-59-6" class="xref-bibr" href="#ref-59">Wang <em>et al.</em> 2012</a>; <a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Hyma and Fay 2013</a>; <a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Goddard <em>et al.</em> 2010</a>; <a id="xref-ref-15-4" class="xref-bibr" href="#ref-15">Cromie <em>et al.</em> 2013</a>; <a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Dashko <em>et al.</em> 2016</a>). <em>S. cerevisiae</em> is one of several model organisms whose genomes show evidence of recent genetic admixture from diverged populations (<a id="xref-ref-40-18" class="xref-bibr" href="#ref-40">Liti <em>et al.</em> 2009</a>; <a id="xref-ref-2-5" class="xref-bibr" href="#ref-2">Almeida <em>et al.</em> 2015</a>; <a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Ludlow <em>et al.</em> 2016</a>; <a id="xref-ref-34-4" class="xref-bibr" href="#ref-34">Hufford <em>et al.</em> 2013</a>; <a id="xref-ref-10-4" class="xref-bibr" href="#ref-10">Brandvain <em>et al.</em> 2014</a>; <a id="xref-ref-45-3" class="xref-bibr" href="#ref-45">Pool <em>et al.</em> 2012</a>; <a id="xref-ref-53-2" class="xref-bibr" href="#ref-53">Sankararaman <em>et al.</em> 2014</a>). Therefore, <em>S. cerevisiae</em> provides an excellent model to test an important question in molecular ecology: whether genetic admixture differs among habitats.</p><p id="p-42">Using two approaches for defining admixed strains in a systematic and quantitative way, we show that patterns of genetic admixture differ between habitats (<a id="xref-table-wrap-2-6" class="xref-table" href="#T2">Table 2</a>). Yeast strains from oak woodlands are less likely to show recent genetic admixture than those from other habitats (<a id="xref-table-wrap-2-7" class="xref-table" href="#T2">Table 2</a>), and when it does occur the degree of admixture is lower and from fewer populations (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041806/-/DC1/FileS2.tsv">File S2</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041806/-/DC1/FileS3.pdf">File S3</a>). Consistent with our results, admixture has been noticed in the past for human-associated <em>S. cerevisiae</em> strains (<a id="xref-ref-15-5" class="xref-bibr" href="#ref-15">Cromie <em>et al.</em> 2013</a>; <a id="xref-ref-59-7" class="xref-bibr" href="#ref-59">Wang <em>et al.</em> 2012</a>) and for those fermenting cacao and coffee (<a id="xref-ref-41-3" class="xref-bibr" href="#ref-41">Ludlow <em>et al.</em> 2016</a>), but there has been no compelling evidence reported previously for intraspecific admixture in yeast from oak woodlands.</p><p id="p-43">By analyzing neutrally evolving centromeres with low recombination (<a id="xref-ref-7-8" class="xref-bibr" href="#ref-7">Bensasson 2011</a>), we minimized the complications of recombination and natural selection on our analysis. Using only these centromere sequences, our results recapitulate genome-wide analyses that identified a second population of <em>S. cerevisiae</em> in Europe that represents the closest wild relatives to the <em>S. cerevisiae</em> Wine population (<a id="xref-ref-15-6" class="xref-bibr" href="#ref-15">Cromie <em>et al.</em> 2013</a>; <a id="xref-ref-2-6" class="xref-bibr" href="#ref-2">Almeida <em>et al.</em> 2015</a>), and the identification of two North American populations by <a id="xref-ref-15-7" class="xref-bibr" href="#ref-15">Cromie <em>et al.</em> (2013)</a>. We were also better able to resolve the North American lineages using only centromere sequences compared to a previous analysis using genome-wide data (<a id="xref-ref-40-19" class="xref-bibr" href="#ref-40">Liti <em>et al.</em> 2009</a>). Therefore, our analysis of centromere sequences appears to have sufficient power to detect the lineages identified by genomic studies.</p><p id="p-44">We developed a locus-by-locus test to complement the use of <em>structure</em>, which is the standard Bayesian method used to estimate admixture (<a id="xref-ref-47-8" class="xref-bibr" href="#ref-47">Pritchard <em>et al.</em> 2000</a>). A comparison of admixture calls using the two different approaches suggests that <em>structure</em> will sometimes invoke admixture to explain the divergence of a strain from defined populations, or it could incorrectly invoke admixture between genetically similar populations (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041806/-/DC1/FileS2.tsv">File S2</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041806/-/DC1/FileS3.pdf">File S3</a>). Indeed, by using a locus-by-locus phylogenetic approach in addition to <em>structure</em>, we detect evidence for a distinct North African population (represented by DBVPG1853m in <a id="xref-fig-3-8" class="xref-fig" href="#F3">Figure 3</a>) that was previously treated as an admixed strain in population genomic analyses (<a id="xref-ref-40-20" class="xref-bibr" href="#ref-40">Liti <em>et al.</em> 2009</a>; <a id="xref-ref-2-7" class="xref-bibr" href="#ref-2">Almeida <em>et al.</em> 2015</a>; <a id="xref-ref-15-8" class="xref-bibr" href="#ref-15">Cromie <em>et al.</em> 2013</a>; <a id="xref-ref-5-4" class="xref-bibr" href="#ref-5">Barbosa <em>et al.</em> 2016</a>). These findings are consistent with the sensitivity of <em>structure</em> and similar programs to sampling scheme when data show isolation by distance (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Schwartz and McKelvey 2009</a>). Molecular ecology and population genomic analyses of <em>S. cerevisiae</em> have mostly used only <em>structure</em> on concatenated alignments from multiple loci to define admixed strains in order to better understand population structure in this species (<a id="xref-ref-59-8" class="xref-bibr" href="#ref-59">Wang <em>et al.</em> 2012</a>; <a id="xref-ref-2-8" class="xref-bibr" href="#ref-2">Almeida <em>et al.</em> 2015</a>; <a id="xref-ref-5-5" class="xref-bibr" href="#ref-5">Barbosa <em>et al.</em> 2016</a>). Our analysis suggests the need for a more thorough investigation of admixture in yeast population genomic data using alternative methods.</p><p id="p-45">When we removed admixed strains from our phylogenetic analyses, it became clear that woodland populations are distinct from one another, even when they occurred relatively close together in the Eastern USA (<a id="xref-fig-3-9" class="xref-fig" href="#F3">Figure 3</a>). Given that we were unable to detect any migration (<a id="xref-fig-3-10" class="xref-fig" href="#F3">Figure 3</a>) or admixture (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041806/-/DC1/FileS2.tsv">File S2</a>) between woodland yeast populations, it seems that yeasts in this ancestral habitat tend to be genetically isolated. Previous reports have suggested distinct oak-associated strains in the primeval forests of China (<a id="xref-ref-59-9" class="xref-bibr" href="#ref-59">Wang <em>et al.</em> 2012</a>) and Brazil (<a id="xref-ref-5-6" class="xref-bibr" href="#ref-5">Barbosa <em>et al.</em> 2016</a>). Our findings suggest that genetic isolation is not only a characteristic of Chinese and Brazilian forest populations, but that even strains from trees in Pennsylvania may be genetically isolated from North Carolina tree strains.</p><p id="p-46">Although we do not detect gene flow between oak woodlands, we do find evidence for both migration and admixture between the human-associated Wine population and woodland populations (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041806/-/DC1/FileS2.tsv">File S2</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041806/-/DC1/FileS3.pdf">File S3</a>, and <a id="xref-table-wrap-2-8" class="xref-table" href="#T2">Table 2</a>). This supports past reports of migration of <em>S. cerevisiae</em> between vineyards and local oak trees (<a id="xref-ref-29-3" class="xref-bibr" href="#ref-29">Goddard <em>et al.</em> 2010</a>; <a id="xref-ref-35-3" class="xref-bibr" href="#ref-35">Hyma and Fay 2013</a>). It also mirrors the situation for <em>D. melanogaster</em> where human-associated populations show higher admixture than populations from nonurban regions, and there has been recent genetic admixture from cosmopolitan to ancestral populations (<a id="xref-ref-45-4" class="xref-bibr" href="#ref-45">Pool <em>et al.</em> 2012</a>).</p><p id="p-47">Transient habitats like fruit only exist for a few weeks and therefore must have been colonized recently by yeast (<a id="xref-ref-28-3" class="xref-bibr" href="#ref-28">Goddard and Greig 2015</a>; <a id="xref-ref-37-3" class="xref-bibr" href="#ref-37">Knight and Goddard 2016</a>). Fruit flies, wasps, bees, and other insects carry live <em>S. cerevisiae</em> in their guts and are therefore likely dispersal vectors for the migration of yeast to fruit (<a id="xref-ref-29-4" class="xref-bibr" href="#ref-29">Goddard <em>et al.</em> 2010</a>; <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Stefanini <em>et al.</em> 2012</a>; <a id="xref-ref-15-9" class="xref-bibr" href="#ref-15">Cromie <em>et al.</em> 2013</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Buser <em>et al.</em> 2014</a>). These insects visit multiple fruits and flowers, can fly long distances (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Coyne and Milstead 1987</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Beekman and Ratnieks 2000</a>), and Drosophilids and honey bees at least have recently expanded to cosmopolitan distributions (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Nunney 1996</a>; <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Whitfield <em>et al.</em> 2006</a>; <a id="xref-ref-45-5" class="xref-bibr" href="#ref-45">Pool <em>et al.</em> 2012</a>). Insects associated with fruit bring together <em>S. cerevisiae</em> strains from diverged populations (<a id="xref-ref-58-2" class="xref-bibr" href="#ref-58">Stefanini <em>et al.</em> 2012</a>). Furthermore, the spores of multiple strains of <em>S. cerevisiae</em> can survive passage through the guts of <em>Drosophila melanogaster</em> and the survivors are much more likely to undergo mating, and therefore admixture, than uneaten yeasts (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Reuter <em>et al.</em> 2007</a>). Thus, if yeasts are primarily dispersed by insect vectors, yeasts from transient habitats are more likely to have cosmopolitan distributions and to show recent genetic admixture, as we observe here.</p><p id="p-48">In contrast, oak tree bark is less nutrient rich (<a id="xref-ref-28-4" class="xref-bibr" href="#ref-28">Goddard and Greig 2015</a>) and is therefore likely to attract fewer flying insects than rotting fruit. Consistent with this expectation, young oak trees have fewer yeast on their bark than older trees (<a id="xref-ref-49-3" class="xref-bibr" href="#ref-49">Robinson <em>et al.</em> 2016</a>), suggesting that stable colonization of oak could be occurring over a period of years rather than weeks. Our observation of genetic isolation and low genetic admixture in oak woodland populations is therefore consistent with the lower migration distances and slower colonization expected for oak trees compared to fruit. It is also consistent with the lack of genetic admixture and the isolation by distance seen in <em>S. paradoxus</em>, which is the closest relative of <em>S. cerevisiae</em> and has been studied almost exclusively from woodlands (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Koufopanou <em>et al.</em> 2006</a>; <a id="xref-ref-40-21" class="xref-bibr" href="#ref-40">Liti <em>et al.</em> 2009</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Leducq <em>et al.</em> 2016</a>). In addition, the degree of divergence that we observe between oak woodland populations may increase with geographic distance even in <em>S. cerevisiae</em>: North Carolina oak strains differ from Pennsylvania oak strains, while differing more from European oak strains (<a id="xref-fig-3-11" class="xref-fig" href="#F3">Figure 3</a>)</p><p id="p-49"><em>S. cerevisiae</em> is especially attractive as a model organism for molecular ecology and population genomics because of the resources already available for understanding its molecular evolution (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Kellis <em>et al.</em> 2003</a>; <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Scannell <em>et al.</em> 2011</a>), molecular biology (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Cherry <em>et al.</em> 2011</a>), experimental evolution (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Rosenzweig and Sherlock 2014</a>), and for testing predictions in the laboratory (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Cubillos <em>et al.</em> 2009</a>). Our study shows that the application of better methods for detecting genetic admixture in genomic data from woodland <em>S. cerevisiae</em> (<a id="xref-ref-2-9" class="xref-bibr" href="#ref-2">Almeida <em>et al.</em> 2015</a>; <a id="xref-ref-5-7" class="xref-bibr" href="#ref-5">Barbosa <em>et al.</em> 2016</a>) could lead to the generation of population genomic data that are unlikely to break the assumptions of most population genetic analyses. Therefore, with more thorough testing for admixture and filtering, <em>S. cerevisiae</em> is likely to be an excellent model for population genomic analysis, despite its complex historical association with humans. There is some evidence that introgressions between species could confer adaptive traits in <em>S. cerevisiae</em> (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Doniger <em>et al.</em> 2008</a>) and <em>S. uvarum</em> (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Almeida <em>et al.</em> 2014</a>). Population genomic analysis of intraspecific genetic admixture in maize revealed that gene flow from ancestral populations led to the adaptation of domesticated crops to the Mexican highlands (<a id="xref-ref-34-5" class="xref-bibr" href="#ref-34">Hufford <em>et al.</em> 2013</a>). Given that <em>S. cerevisiae</em> is employed in a broad range of industries, including the production of wine, sake, beer, chocolate, and cacao, it will be especially interesting to apply new tools to study genome-wide patterns of admixture (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Corbett-Detig and Nielsen 2017</a>) to reveal whether the genetic admixture seen among populations in <em>S. cerevisiae</em> plays a similar adaptive role in domestication.</p></div><div class="section" id="sec-15"></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-50">We thank Casey Bergman, Kelly Dyer, David Hall, and two anonymous reviewers for helpful comments on the manuscript, and Stephanie Diezmann, Paul Sniegowski, Greg Wray, Anne Rouse, José Paulo Sampaio, and Eladio Barrio for providing strains. This work was supported by the Natural Environment Research Council (NERC) through a NERC Fellowship to D.B. (grant number NE/D008824/1) (<a href="http://www.nerc.ac.uk">http://www.nerc.ac.uk</a>).</p><p id="p-51">Author contributions: D.B. conceived and designed the research, V.T. performed the laboratory work, and D.B. analyzed the data and wrote the manuscript.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-5"><p id="p-52">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041806/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.041806/-/DC1</a>.</p></li><li class="fn" id="fn-6"><p id="p-53"><em>Communicating editor: J. C. Fay</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2017-03-30"><span class="received-label">Received </span>March 30, 2017.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2017-06-22"><span class="accepted-label">Accepted </span>June 22, 2017.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2017 Tilakaratna and Bensasson</li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.2919.1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Almeida</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gonçalves</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Teixeira</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Libkind</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bontrager</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A Gondwanan imprint on global diversity and domestication of wine and cider yeast <em>Saccharomyces uvarum</em>.</span> <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">4044</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun.%26rft.volume%253D5%26rft.spage%253D4044%26rft_id%253Dinfo%253Apmid%252F24887054%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=24887054&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F2919.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.2919.2" data-doi="10.1111/mec.13341"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Almeida</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barbosa</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zalar</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Imanishi</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shimizu</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">A population genomics insight into the Mediterranean origins of wine yeast domestication.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">5412</span>–<span class="cit-lpage">5427</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D24%26rft.spage%253D5412%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.13341%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.13341&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-book" id="cit-7.9.2919.3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Amberg</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burke</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Strathern</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <em>Methods in Yeast Genetics: A Cold Spring Harbor Laboratory Course Manual</em>. <span class="cit-publ-name">Cold Spring Harbor Laboratory Press</span>, <span class="cit-publ-loc">Cold Spring Harbor, NY</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-book" id="cit-7.9.2919.4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Avise</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-source"><em>Molecular Markers</em>, <em>Natural History and Evolution</em></span>. <span class="cit-publ-name">Chapman &amp; Hall</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.2919.5" data-doi="10.1093/gbe/evv263"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Barbosa</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Almeida</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Safar</span>  <span class="cit-name-given-names">S. V. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Santos</span>  <span class="cit-name-given-names">R. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morais</span>  <span class="cit-name-given-names">P. B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Evidence of natural hybridization in Brazilian wild lineages of <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Genome Biol. Evol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">317</span>–<span class="cit-lpage">329</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevv263%26rft_id%253Dinfo%253Apmid%252F26782936%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/gbe/evv263&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26782936&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F2919.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.2919.6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Beekman</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ratnieks</span>  <span class="cit-name-given-names">F. L. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Long-range foraging by the honey-bee, Apis mellifera L.</span> <abbr class="cit-jnl-abbrev">Funct. Ecol.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">490</span>–<span class="cit-lpage">496</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFunct.%2BEcol.%26rft.volume%253D14%26rft.spage%253D490%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.2919.7" data-doi="10.1186/1471-2148-11-211"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bensasson</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Evidence for a high mutation rate at rapidly evolving yeast centromeres.</span> <abbr class="cit-jnl-abbrev">BMC Evol. Biol.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">211</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bevolutionary%2Bbiology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BEvol%2BBiol%26rft.aulast%253DBensasson%26rft.auinit1%253DD.%26rft.volume%253D11%26rft.spage%253D211%26rft.epage%253D211%26rft.atitle%253DEvidence%2Bfor%2Ba%2Bhigh%2Bmutation%2Brate%2Bat%2Brapidly%2Bevolving%2Byeast%2Bcentromeres.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2148-11-211%26rft_id%253Dinfo%253Apmid%252F21767380%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2148-11-211&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21767380&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F2919.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.2919.8" data-doi="10.1534/genetics.107.083980"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bensasson</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zarowiecki</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burt</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koufopanou</span>  <span class="cit-name-given-names">V.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Rapid evolution of yeast centromeres in the absence of drive.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">178</span>: <span class="cit-fpage">2161</span>–<span class="cit-lpage">2167</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DBensasson%26rft.auinit1%253DD.%26rft.volume%253D178%26rft.issue%253D4%26rft.spage%253D2161%26rft.epage%253D2167%26rft.atitle%253DRapid%2BEvolution%2Bof%2BYeast%2BCentromeres%2Bin%2Bthe%2BAbsence%2Bof%2BDrive%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.083980%26rft_id%253Dinfo%253Apmid%252F18430941%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3OC80LzIxNjEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzkvMjkxOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.2919.9" data-doi="10.1093/nar/23.24.4992"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bonfield</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">K. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Staden</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">A new DNA sequence assembly program.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">4992</span>–<span class="cit-lpage">4999</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DBonfield%26rft.auinit1%253DJ.%2BK.%26rft.volume%253D23%26rft.issue%253D24%26rft.spage%253D4992%26rft.epage%253D4999%26rft.atitle%253DA%2Bnew%2BDNA%2Bsequence%2Bassembly%2Bprogram%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F23.24.4992%26rft_id%253Dinfo%253Apmid%252F8559656%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/23.24.4992&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8559656&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F2919.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1995TP24300008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.2919.10" data-doi="10.1371/journal.pgen.1004410"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brandvain</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kenney</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Flagel</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coop</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sweigart</span>  <span class="cit-name-given-names">A. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Speciation and introgression between <em>Mimulus nasutus</em> and Mimulus guttatus.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">e1004410</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D10%26rft.spage%253De1004410%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1004410%26rft_id%253Dinfo%253Apmid%252F24967630%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1004410&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24967630&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F2919.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.2919.11" data-doi="10.1111/ele.12331"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Buser</span>  <span class="cit-name-given-names">C. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Newcomb</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gaskett</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Niche construction initiates the evolution of mutualistic interactions.</span> <abbr class="cit-jnl-abbrev">Ecol. Lett.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">1257</span>–<span class="cit-lpage">1264</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcol.%2BLett.%26rft.volume%253D17%26rft.spage%253D1257%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fele.12331%26rft_id%253Dinfo%253Apmid%252F25041133%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/ele.12331&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25041133&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F2919.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.2919.12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cherry</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hong</span>  <span class="cit-name-given-names">E. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amundsen</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Balakrishnan</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Binkley</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Saccharomyces genome database: the genomics resource of budding yeast.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">D700</span>–<span class="cit-lpage">D705</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft.volume%253D40%26rft.spage%253DD700%26rft_id%253Dinfo%253Apmid%252F22110037%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=22110037&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F2919.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000298601300106&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.2919.13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Corbett-Detig</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">A hidden Markov model approach for simultaneously estimating local ancestry and admixture time using next generation sequence data in samples of arbitrary ploidy.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">e1006529</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D13%26rft.spage%253De1006529%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.2919.14" data-doi="10.1086/284698"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Coyne</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Milstead</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span> <span class="cit-article-title">Long-distance migration of Drosophila. 3. Dispersal of <em>D. melanogaster</em> Alleles from a Maryland Orchard.</span> <abbr class="cit-jnl-abbrev">Am. Nat.</abbr> <span class="cit-vol">130</span>: <span class="cit-fpage">70</span>–<span class="cit-lpage">82</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BNat.%26rft.volume%253D130%26rft.spage%253D70%26rft_id%253Dinfo%253Adoi%252F10.1086%252F284698%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/284698&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.2919.15"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cromie</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hyma</span>  <span class="cit-name-given-names">K. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ludlow</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garmendia-Torres</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gilbert</span>  <span class="cit-name-given-names">T. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genomic sequence diversity and population structure of <em>Saccharomyces cerevisiae</em> assessed by RAD-seq.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">2163</span>–<span class="cit-lpage">2171</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft.volume%253D3%26rft.spage%253D2163%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.2919.16" data-doi="10.1111/j.1567-1364.2009.00583.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cubillos</span>  <span class="cit-name-given-names">F. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Louis</span>  <span class="cit-name-given-names">E. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liti</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Generation of a large set of genetically tractable haploid and diploid <em>Saccharomyces</em> strains.</span> <abbr class="cit-jnl-abbrev">FEMS Yeast Res.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">1217</span>–<span class="cit-lpage">1225</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEMS%2BYeast%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1567-1364.2009.00583.x%26rft_id%253Dinfo%253Apmid%252F19840116%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1567-1364.2009.00583.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19840116&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F2919.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000271264400009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.2919.17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dashko</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Volk</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Butinar</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Piškur</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Changes in the relative abundance of two <em>Saccharomyces</em> species from Oak forests to wine fermentations.</span> <abbr class="cit-jnl-abbrev">Front. Microbiol.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">215</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BMicrobiol.%26rft.volume%253D7%26rft.spage%253D215%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-7.9.2919.18" data-doi="10.1371/journal.pone.0005317"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Diezmann</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dietrich</span>  <span class="cit-name-given-names">F. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title"><em>Saccharomyces cerevisiae</em>: population divergence and resistance to oxidative stress in clinical, domesticated and wild isolates.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e5317</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DDiezmann%26rft.auinit1%253DS.%26rft.volume%253D4%26rft.issue%253D4%26rft.spage%253De5317%26rft.epage%253De5317%26rft.atitle%253DSaccharomyces%2Bcerevisiae%253A%2Bpopulation%2Bdivergence%2Band%2Bresistance%2Bto%2Boxidative%2Bstress%2Bin%2Bclinical%252C%2Bdomesticated%2Band%2Bwild%2Bisolates.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0005317%26rft_id%253Dinfo%253Apmid%252F19390633%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0005317&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19390633&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F2919.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.2919.19" data-doi="10.1371/journal.pgen.1000183"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Doniger</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Swain</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Corcuera</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Williams</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">A catalog of neutral and deleterious polymorphism in yeast.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e1000183</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DDoniger%26rft.auinit1%253DS.%2BW.%26rft.volume%253D4%26rft.issue%253D8%26rft.spage%253De1000183%26rft.epage%253De1000183%26rft.atitle%253DA%2Bcatalog%2Bof%2Bneutral%2Band%2Bdeleterious%2Bpolymorphism%2Bin%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000183%26rft_id%253Dinfo%253Apmid%252F18769710%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000183&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18769710&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F2919.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.2919.20"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eberlein</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leducq</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Landry</span>  <span class="cit-name-given-names">C. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">The genomics of wild yeast populations sheds light on the domestication of man’s best (micro) friend.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">5309</span>–<span class="cit-lpage">5311</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D24%26rft.spage%253D5309%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.2919.21" data-doi="10.1086/497832"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Enache-Angoulvant</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hennequin</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Invasive <em>Saccharomyces</em> infection: a comprehensive review.</span> <abbr class="cit-jnl-abbrev">Clin. Infect. Dis.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">1559</span>–<span class="cit-lpage">1568</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DClin.%2BInfect.%2BDis.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F497832%26rft_id%253Dinfo%253Apmid%252F16267727%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/497832&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16267727&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F2919.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000233018600003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.2919.22" data-doi="10.1111/j.1365-294X.2005.02553.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Evanno</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Regnaut</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goudet</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Detecting the number of clusters of individuals using the software structure: a simulation study.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">2611</span>–<span class="cit-lpage">2620</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DEvanno%26rft.auinit1%253DG.%26rft.volume%253D14%26rft.issue%253D8%26rft.spage%253D2611%26rft.epage%253D2620%26rft.atitle%253DDetecting%2Bthe%2Bnumber%2Bof%2Bclusters%2Bof%2Bindividuals%2Busing%2Bthe%2Bsoftware%2BSTRUCTURE%253A%2Ba%2Bsimulation%2Bstudy.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2005.02553.x%26rft_id%253Dinfo%253Apmid%252F15969739%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2005.02553.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15969739&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F2919.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000229961500029&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.2919.23"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Falush</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pritchard</span>  <span class="cit-name-given-names">J. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Inference of population structure using multilocus genotype data: linked loci and correlated allele frequencies.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">164</span>: <span class="cit-fpage">1567</span>–<span class="cit-lpage">1587</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DFalush%26rft.auinit1%253DD.%26rft.volume%253D164%26rft.issue%253D4%26rft.spage%253D1567%26rft.epage%253D1587%26rft.atitle%253DInference%2Bof%2BPopulation%2BStructure%2BUsing%2BMultilocus%2BGenotype%2BData%253A%2BLinked%2BLoci%2Band%2BCorrelated%2BAllele%2BFrequencies%26rft_id%253Dinfo%253Apmid%252F12930761%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2NC80LzE1NjciO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzkvMjkxOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.2919.24" data-doi="10.1371/journal.pgen.0010005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fay</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Benavides</span>  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Evidence for domesticated and wild populations of <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">e5</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D1%26rft.spage%253De5%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.0010005%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.0010005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.2919.25" data-doi="10.1093/oxfordjournals.molbev.a025575"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Felsenstein</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Churchill</span>  <span class="cit-name-given-names">G. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">A hidden Markov Model approach to variation among sites in rate of evolution.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">93</span>–<span class="cit-lpage">104</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft.stitle%253DMol%2BBiol%2BEvol%26rft.aulast%253DFelsenstein%26rft.auinit1%253DJ%26rft.volume%253D13%26rft.issue%253D1%26rft.spage%253D93%26rft.epage%253D104%26rft.atitle%253DA%2BHidden%2BMarkov%2BModel%2Bapproach%2Bto%2Bvariation%2Bamong%2Bsites%2Bin%2Brate%2Bof%2Bevolution%26rft_id%253Dinfo%253Adoi%252F10.1093%252Foxfordjournals.molbev.a025575%26rft_id%253Dinfo%253Apmid%252F8583911%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/oxfordjournals.molbev.a025575&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8583911&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F2919.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1996TP07600011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.2919.26" data-doi="10.1016/j.cell.2016.08.020"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gallone</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Steensels</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Prahl</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Soriaga</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Saels</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Domestication and divergence of <em>Saccharomyces cerevisiae</em> Beer yeasts.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">166</span>: <span class="cit-fpage">1397</span>–<span class="cit-lpage">1410.e16</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D166%26rft.spage%253D1397%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2016.08.020%26rft_id%253Dinfo%253Apmid%252F27610566%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2016.08.020&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27610566&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F2919.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.2919.27" data-doi="10.1371/journal.pone.0085196"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gayevskiy</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klaere</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Knight</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">ObStruct: a method to objectively analyse factors driving population structure using Bayesian ancestry profiles.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e85196</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De85196%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0085196%26rft_id%253Dinfo%253Apmid%252F24416362%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0085196&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24416362&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F2919.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.2919.28" data-doi="10.1093/femsyr/fov009"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Greig</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title"><em>Saccharomyces cerevisiae</em>: a nomadic yeast with no niche?</span> <abbr class="cit-jnl-abbrev">FEMS Yeast Res.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">fov009</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEMS%2BYeast%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Ffemsyr%252Ffov009%26rft_id%253Dinfo%253Apmid%252F25725024%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/femsyr/fov009&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25725024&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F2919.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.2919.29" data-doi="10.1111/j.1462-2920.2009.02035.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anfang</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tang</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gardner</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jun</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">A distinct population of <em>Saccharomyces cerevisiae</em> in New Zealand: evidence for local dispersal by insects and human-aided global dispersal in oak barrels.</span> <abbr class="cit-jnl-abbrev">Environ. Microbiol.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">63</span>–<span class="cit-lpage">73</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEnvironmental%2BMicrobiology%2B%2528Print%2529%26rft.stitle%253DEnvironmental%2BMicrobiology%2B%2528Print%2529%26rft.aulast%253DGoddard%26rft.auinit1%253DM.%2BR.%26rft.volume%253D12%26rft.issue%253D1%26rft.spage%253D63%26rft.epage%253D73%26rft.atitle%253DA%2Bdistinct%2Bpopulation%2Bof%2BSaccharomyces%2Bcerevisiae%2Bin%2BNew%2BZealand%253A%2Bevidence%2Bfor%2Blocal%2Bdispersal%2Bby%2Binsects%2Band%2Bhuman-aided%2Bglobal%2Bdispersal%2Bin%2Boak%2Bbarrels.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1462-2920.2009.02035.x%26rft_id%253Dinfo%253Apmid%252F19691498%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1462-2920.2009.02035.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19691498&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F2919.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000274234200006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.2919.30"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gonçalves</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pontes</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Almeida</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barbosa</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Serra</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Distinct domestication trajectories in top-fermenting beer yeasts and wine yeasts.</span> <abbr class="cit-jnl-abbrev">Curr. Biol.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">2750</span>–<span class="cit-lpage">2761</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BBiol.%26rft.volume%253D26%26rft.spage%253D2750%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.2919.31" data-doi="10.1093/molbev/msp259"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gouy</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guindon</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gascuel</span>  <span class="cit-name-given-names">O.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">SeaView Version 4: a multiplatform graphical user interface for sequence alignment and phylogenetic tree building.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">221</span>–<span class="cit-lpage">224</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsp259%26rft_id%253Dinfo%253Apmid%252F19854763%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msp259&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19854763&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F2919.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000273704400003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.2919.32" data-doi="10.1534/genetics.116.186890"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Harris</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">The genetic cost of Neanderthal introgression.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">203</span>: <span class="cit-fpage">881</span>–<span class="cit-lpage">891</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.116.186890%26rft_id%253Dinfo%253Apmid%252F27038113%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjAzLzIvODgxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy85LzI5MTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.2919.33" data-doi="10.1111/j.1755-0998.2009.02591.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hubisz</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Falush</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pritchard</span>  <span class="cit-name-given-names">J. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Inferring weak population structure with the assistance of sample group information.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol. Resour.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">1322</span>–<span class="cit-lpage">1332</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%2BResour.%26rft.volume%253D9%26rft.spage%253D1322%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1755-0998.2009.02591.x%26rft_id%253Dinfo%253Apmid%252F21564903%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1755-0998.2009.02591.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21564903&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F2919.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268855000004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.2919.34" data-doi="10.1371/journal.pgen.1003477"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hufford</span>  <span class="cit-name-given-names">M. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lubinksy</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pyhäjärvi</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Devengenzo</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ellstrand</span>  <span class="cit-name-given-names">N. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The genomic signature of crop-wild introgression in Maize.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e1003477</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D9%26rft.spage%253De1003477%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003477%26rft_id%253Dinfo%253Apmid%252F23671421%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003477&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23671421&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F2919.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.2919.35" data-doi="10.1111/mec.12155"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hyma</span>  <span class="cit-name-given-names">K. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fay</span>  <span class="cit-name-given-names">J. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Mixing of vineyard and oak-tree ecotypes of <em>Saccharomyces cerevisiae</em> in North American vineyards.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">2917</span>–<span class="cit-lpage">2930</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D22%26rft.spage%253D2917%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.12155%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.12155&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000319675300006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.2919.36" data-doi="10.1038/nature01644"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kellis</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Patterson</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Endrizzi</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Birren</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lander</span>  <span class="cit-name-given-names">E. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Sequencing and comparison of yeast species to identify genes and regulatory elements.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">423</span>: <span class="cit-fpage">241</span>–<span class="cit-lpage">254</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DKellis%26rft.auinit1%253DM.%26rft.volume%253D423%26rft.issue%253D6937%26rft.spage%253D241%26rft.epage%253D254%26rft.atitle%253DSequencing%2Band%2Bcomparison%2Bof%2Byeast%2Bspecies%2Bto%2Bidentify%2Bgenes%2Band%2Bregulatory%2Belements.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature01644%26rft_id%253Dinfo%253Apmid%252F12748633%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature01644&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12748633&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F2919.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000182853100033&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.2919.37" data-doi="10.1093/femsyr/fov102"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Knight</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Sporulation in soil as an overwinter survival strategy in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">FEMS Yeast Res.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">fov102</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEMS%2BYeast%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Ffemsyr%252Ffov102%26rft_id%253Dinfo%253Apmid%252F26568201%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/femsyr/fov102&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26568201&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F2919.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.2919.38" data-doi="10.1098/rstb.2006.1922"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Koufopanou</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hughes</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bell</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burt</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">The spatial scale of genetic differentiation in a model organism: the wild yeast <em>Saccharomyces paradoxus</em>.</span> <abbr class="cit-jnl-abbrev">Philos. Trans. R. Soc. Lond. B Biol. Sci.</abbr> <span class="cit-vol">361</span>: <span class="cit-fpage">1941</span>–<span class="cit-lpage">1946</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhilosophical%2BTransactions%2Bof%2Bthe%2BRoyal%2BSociety%2BB%253A%2BBiological%2BSciences%26rft.stitle%253DPhil%2BTrans%2BR%2BSoc%2BB%26rft.aulast%253DKoufopanou%26rft.auinit1%253DV.%26rft.volume%253D361%26rft.issue%253D1475%26rft.spage%253D1941%26rft.epage%253D1946%26rft.atitle%253DThe%2Bspatial%2Bscale%2Bof%2Bgenetic%2Bdifferentiation%2Bin%2Ba%2Bmodel%2Borganism%253A%2Bthe%2Bwild%2Byeast%2BSaccharomyces%2Bparadoxus%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frstb.2006.1922%26rft_id%253Dinfo%253Apmid%252F17028086%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Njoicm95cHRiIjtzOjU6InJlc2lkIjtzOjEzOiIzNjEvMTQ3NS8xOTQxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy85LzI5MTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.2919.39"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Leducq</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nielly-Thibault</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Charron</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eberlein</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Verta</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Speciation driven by hybridization and chromosomal plasticity in a wild yeast.</span> <abbr class="cit-jnl-abbrev">Nat. Microbiol.</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">15003</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BMicrobiol.%26rft.volume%253D1%26rft.spage%253D15003%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.2919.40" data-doi="10.1038/nature07743"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liti</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carter</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moses</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Warringer</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parts</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Population genomics of domestic and wild yeasts.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">458</span>: <span class="cit-fpage">337</span>–<span class="cit-lpage">341</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DLiti%26rft.auinit1%253DG.%26rft.volume%253D458%26rft.issue%253D7236%26rft.spage%253D337%26rft.epage%253D341%26rft.atitle%253DPopulation%2Bgenomics%2Bof%2Bdomestic%2Band%2Bwild%2Byeasts.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature07743%26rft_id%253Dinfo%253Apmid%252F19212322%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature07743&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19212322&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F2919.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000264285600041&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.2919.41" data-doi="10.1016/j.cub.2016.02.012"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ludlow</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cromie</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garmendia-Torres</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sirr</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hays</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Independent origins of yeast associated with coffee and cacao fermentation.</span> <abbr class="cit-jnl-abbrev">Curr. Biol.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">965</span>–<span class="cit-lpage">971</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BBiol.%26rft.volume%253D26%26rft.spage%253D965%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cub.2016.02.012%26rft_id%253Dinfo%253Apmid%252F27020745%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cub.2016.02.012&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27020745&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F2919.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.2919.42" data-doi="10.1073/pnas.1312984110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Marshall</span>  <span class="cit-name-given-names">F. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dobney</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Denham</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Capriles</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Evaluating the roles of directed breeding and gene flow in animal domestication.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">111</span>: <span class="cit-fpage">6153</span>–<span class="cit-lpage">6158</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1312984110%26rft_id%253Dinfo%253Apmid%252F24753599%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTExLzE3LzYxNTMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzkvMjkxOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.2919.43" data-doi="10.1007/BF00333733"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nunney</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">The colonization of oranges by the Cosmopolitan <em>Drosophila</em>.</span> <abbr class="cit-jnl-abbrev">Oecologia</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">552</span>–<span class="cit-lpage">561</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DOecologia%26rft.volume%253D108%26rft.spage%253D552%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00333733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00333733&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-book" id="cit-7.9.2919.44"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Paradis</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <em>Analysis of Phylogenetics and Evolution with R</em>, <span class="cit-edition">Ed. 2</span>. <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.2919.45" data-doi="10.1371/journal.pgen.1003080"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pool</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Corbett-Detig</span>  <span class="cit-name-given-names">R. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sugino</span>  <span class="cit-name-given-names">R. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stevens</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cardeno</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Population genomics of Sub-Saharan <em>Drosophila melanogaster</em>: African diversity and Non-African admixture.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e1003080</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D8%26rft.spage%253De1003080%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003080%26rft_id%253Dinfo%253Apmid%252F23284287%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003080&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23284287&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F2919.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.2919.46" data-doi="10.1007/s00239-001-0034-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Posada</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crandall</span>  <span class="cit-name-given-names">K. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">The effect of recombination on the accuracy of phylogeny estimation.</span> <abbr class="cit-jnl-abbrev">J. Mol. Evol.</abbr> <span class="cit-vol">54</span>: <span class="cit-fpage">396</span>–<span class="cit-lpage">402</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.stitle%253DJournal%2Bof%2BMolecular%2BEvolution%26rft.aulast%253DPosada%26rft.auinit1%253DD.%26rft.volume%253D54%26rft.issue%253D3%26rft.spage%253D396%26rft.epage%253D402%26rft.atitle%253DThe%2Beffect%2Bof%2Brecombination%2Bon%2Bthe%2Baccuracy%2Bof%2Bphylogeny%2Bestimation.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00239-001-0034-9%26rft_id%253Dinfo%253Apmid%252F11847565%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00239-001-0034-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11847565&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F2919.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000173928400011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.2919.47" data-doi="10.1111/j.1471-8286.2005.01155.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pritchard</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Donnelly</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Inference of population structure using Multilocus genotype data.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">155</span>: <span class="cit-fpage">945</span>–<span class="cit-lpage">959</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DPritchard%26rft.auinit1%253DJ.%2BK.%26rft.volume%253D155%26rft.issue%253D2%26rft.spage%253D945%26rft.epage%253D959%26rft.atitle%253DInference%2Bof%2BPopulation%2BStructure%2BUsing%2BMultilocus%2BGenotype%2BData%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1471-8286.2005.01155.x%26rft_id%253Dinfo%253Apmid%252F10835412%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTU1LzIvOTQ1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy85LzI5MTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.2919.48" data-doi="10.1016/j.cub.2006.11.059"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Reuter</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bell</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Greig</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Increased outbreeding in yeast in response to dispersal by an insect vector.</span> <abbr class="cit-jnl-abbrev">Curr. Biol.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">R81</span>–<span class="cit-lpage">R83</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bbiology%2B%253A%2B%2BCB%26rft.stitle%253DCurr%2BBiol%26rft.aulast%253DReuter%26rft.auinit1%253DM.%26rft.volume%253D17%26rft.issue%253D3%26rft.spage%253DR81%26rft.epage%253DR83%26rft.atitle%253DIncreased%2Boutbreeding%2Bin%2Byeast%2Bin%2Bresponse%2Bto%2Bdispersal%2Bby%2Ban%2Binsect%2Bvector.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cub.2006.11.059%26rft_id%253Dinfo%253Apmid%252F17276903%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cub.2006.11.059&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17276903&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F2919.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000244164700008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-3" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.2919.49"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Robinson</span>  <span class="cit-name-given-names">H. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pinharanda</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bensasson</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Summer temperature can predict the distribution of wild yeast populations.</span> <abbr class="cit-jnl-abbrev">Ecol. Evol.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">1236</span>–<span class="cit-lpage">1250</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcol.%2BEvol.%26rft.volume%253D6%26rft.spage%253D1236%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.2919.50"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rosenzweig</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sherlock</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Editorial: experimental evolution: prospects and challenges.</span> <abbr class="cit-jnl-abbrev">Genomics</abbr> <span class="cit-vol">104</span>: <span class="cit-fpage">v</span>–<span class="cit-lpage">vi</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenomics%26rft.volume%253D104%26rft.spage%253Dv%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.2919.51" data-doi="10.1099/ijs.0.01472-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Saitou</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nei</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span> <span class="cit-article-title">The neighbor-joining method: a new method for reconstructing phylogenetic trees.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">406</span>–<span class="cit-lpage">425</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft.stitle%253DMol%2BBiol%2BEvol%26rft.aulast%253DSaitou%26rft.auinit1%253DN%26rft.volume%253D4%26rft.issue%253D4%26rft.spage%253D406%26rft.epage%253D425%26rft.atitle%253DThe%2Bneighbor-joining%2Bmethod%253A%2Ba%2Bnew%2Bmethod%2Bfor%2Breconstructing%2Bphylogenetic%2Btrees%26rft_id%253Dinfo%253Adoi%252F10.1099%252Fijs.0.01472-0%26rft_id%253Dinfo%253Apmid%252F3447015%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1099/ijs.0.01472-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=3447015&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F2919.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1987J406700007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-7.9.2919.52" data-doi="10.1128/AEM.02396-07"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sampaio</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gonçalves</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Natural populations of <em>Saccharomyces kudriavzevii</em> in Portugal are associated with Oak Bark and are sympatric with <em>S. cerevisiae</em> and <em>S. paradoxus</em>.</span> <abbr class="cit-jnl-abbrev">Appl. Environ. Microbiol.</abbr> <span class="cit-vol">74</span>: <span class="cit-fpage">2144</span>–<span class="cit-lpage">2152</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAppl.%2BEnviron.%2BMicrobiol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FAEM.02396-07%26rft_id%253Dinfo%253Apmid%252F18281431%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiYWVtIjtzOjU6InJlc2lkIjtzOjk6Ijc0LzcvMjE0NCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvOS8yOTE5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.2919.53" data-doi="10.1038/nature12961"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sankararaman</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mallick</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dannemann</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Prüfer</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kelso</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The genomic landscape of Neanderthal ancestry in present-day humans.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">507</span>: <span class="cit-fpage">354</span>–<span class="cit-lpage">357</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D507%26rft.spage%253D354%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature12961%26rft_id%253Dinfo%253Apmid%252F24476815%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature12961&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24476815&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F2919.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000333029000033&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.2919.54" data-doi="10.1534/g3.111.000273"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Scannell</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zill</span>  <span class="cit-name-given-names">O. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rokas</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Payen</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dunham</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The awesome power of yeast evolutionary genetics: new genome sequences and strain resources for the <em>Saccharomyces sensu stricto</em> Genus.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">11</span>–<span class="cit-lpage">25</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft.volume%253D1%26rft.spage%253D11%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.111.000273%26rft_id%253Dinfo%253Apmid%252F22384314%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/g3.111.000273&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22384314&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F2919.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.2919.55" data-doi="10.1007/s10592-008-9622-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schwartz</span>  <span class="cit-name-given-names">M. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McKelvey</span>  <span class="cit-name-given-names">K. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Why sampling scheme matters: the effect of sampling scheme on landscape genetic results.</span> <abbr class="cit-jnl-abbrev">Conserv. Genet.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">441</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DConserv.%2BGenet.%26rft.volume%253D10%26rft.spage%253D441%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10592-008-9622-1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10592-008-9622-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000263869100016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.2919.56" data-doi="10.1111/j.1567-1364.2002.tb00048.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sniegowski</span>  <span class="cit-name-given-names">P. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dombrowski</span>  <span class="cit-name-given-names">P. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fingerman</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title"><em>Saccharomyces cerevisiae</em> and <em>Saccharomyces paradoxus</em> coexist in a natural woodland site in North America and display different levels of reproductive isolation from European conspecifics.</span> <abbr class="cit-jnl-abbrev">FEMS Yeast Res.</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">299</span>–<span class="cit-lpage">306</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEMS%2BYeast%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1567-1364.2002.tb00048.x%26rft_id%253Dinfo%253Apmid%252F12702333%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1567-1364.2002.tb00048.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12702333&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F2919.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.2919.57" data-doi="10.1093/bioinformatics/btu033"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stamatakis</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">RAxML Version 8: a tool for phylogenetic analysis and post-analysis of large phylogenies.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">1312</span>–<span class="cit-lpage">1313</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu033%26rft_id%253Dinfo%253Apmid%252F24451623%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btu033&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24451623&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F2919.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000336095100024&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.2919.58" data-doi="10.1073/pnas.1208362109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stefanini</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dapporto</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Legras</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Calabretta</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Di Paola</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Role of social wasps in <em>Saccharomyces cerevisiae</em> ecology and evolution.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">109</span>: <span class="cit-fpage">13398</span>–<span class="cit-lpage">13403</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1208362109%26rft_id%253Dinfo%253Apmid%252F22847440%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA5LzMzLzEzMzk4IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy85LzI5MTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.2919.59" data-doi="10.1111/j.1365-294X.2012.05732.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Q. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">W. Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liti</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bai</span>  <span class="cit-name-given-names">F. Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Surprisingly diverged populations of <em>Saccharomyces cerevisiae</em> in natural environments remote from human activity.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">5404</span>–<span class="cit-lpage">5417</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DWang%26rft.auinit1%253DQ.%2BM.%26rft.volume%253D21%26rft.issue%253D22%26rft.spage%253D5404%26rft.epage%253D5417%26rft.atitle%253DSurprisingly%2Bdiverged%2Bpopulations%2Bof%2BSaccharomyces%2Bcerevisiae%2Bin%2Bnatural%2Benvironments%2Bremote%2Bfrom%2Bhuman%2Bactivity.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2012.05732.x%26rft_id%253Dinfo%253Apmid%252F22913817%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2012.05732.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22913817&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F2919.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000310583800005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.2919.60" data-doi="10.1126/science.1132772"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Whitfield</span>  <span class="cit-name-given-names">C. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Behura</span>  <span class="cit-name-given-names">S. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berlocher</span>  <span class="cit-name-given-names">S. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clark</span>  <span class="cit-name-given-names">A. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnston</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Thrice out of Africa: ancient and recent expansions of the Honey Bee, <em>Apis mellifera</em>.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">314</span>: <span class="cit-fpage">642</span>–<span class="cit-lpage">645</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DWhitfield%26rft.auinit1%253DC.%2BW.%26rft.volume%253D314%26rft.issue%253D5799%26rft.spage%253D642%26rft.epage%253D645%26rft.atitle%253DThrice%2BOut%2Bof%2BAfrica%253A%2BAncient%2Band%2BRecent%2BExpansions%2Bof%2Bthe%2BHoney%2BBee%252C%2BApis%2Bmellifera%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1132772%26rft_id%253Dinfo%253Apmid%252F17068261%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMTQvNTc5OS82NDIiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzkvMjkxOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/7/9/2919.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/9/2907" title="Genome-Wide Screen for Genes Involved in Caenorhabditis elegans Developmentally Timed Sleep" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/9/2931" title="ASPsiRNA: A Resource of ASP-siRNAs Having Therapeutic Potential for Human Genetic Disorders and Algorithm for Prediction of Their Inhibitory Efficacy" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/7/9"><span>Volume 7 Issue 9, September 2017</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/7/9.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/7/9.cover-source.jpg?itok=WuYpGT8r" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 7 (9)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/9/2919" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-4uOwIel4z5jibARrHbIrsgWUEMjzgn_Y7slYW0SnrH0" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/7/9/2919" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/7/9/2919" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/7/9/2919" class="active" data-icon-position="" data-hide-link-title="0">Habitat Predicts Levels of Genetic Admixture in Saccharomyces cerevisiae</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/486369" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-4YA3SefynEn3iR2vVj0O3sIkFil5rZuKXVUV9Kyc8i8" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/9/2919.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/7/9/2919" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-tUdlLzULWvjItVdspX2G_sO118v7DU2jnxyhjn6ZhB8" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/7/9/2919" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/9/2919.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="486369" id="node486369--2" data-pisa="ggg;7/9/2919" data-pisa-master="ggg;g3.117.041806" data-apath="/ggg/7/9/2919.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Habitat Predicts Levels of Genetic Admixture in <em>Saccharomyces cerevisiae</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Viranga Tilakaratna</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-9015-2707" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Douda Bensasson</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2919-2929;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.041806 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node486369--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Viranga Tilakaratna </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Faculty of Life Sciences, University of Manchester, M13 9PT, UK</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Viranga%2BTilakaratna%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tilakaratna%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AViranga%2BTilakaratna%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Douda Bensasson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Plant Biology, University of Georgia, Athens, Georgia 30602</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Institute of Bioinformatics, University of Georgia, Athens, Georgia 30602</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Douda%2BBensasson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bensasson%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADouda%2BBensasson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-9015-2707" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Douda Bensasson</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:dbensasson@uga.edu" class="" data-icon-position="" data-hide-link-title="0">dbensasson@uga.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/7/9/2919" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/7/9/2919&amp;title=Habitat%20Predicts%20Levels%20of%20Genetic%20Admixture%20in%20Saccharomyces%20cerevisiae" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/7/9/2919&amp;title=Habitat%20Predicts%20Levels%20of%20Genetic%20Admixture%20in%20Saccharomyces%20cerevisiae" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/7/9/2919&amp;title=Habitat%20Predicts%20Levels%20of%20Genetic%20Admixture%20in%20Saccharomyces%20cerevisiae" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/7/9/2919&amp;text=Habitat%20Predicts%20Levels%20of%20Genetic%20Admixture%20in%20Saccharomyces%20cerevisiae" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/7/9/2919&amp;title=Habitat%20Predicts%20Levels%20of%20Genetic%20Admixture%20in%20Saccharomyces%20cerevisiae" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/7/9/2919&amp;t=Habitat%20Predicts%20Levels%20of%20Genetic%20Admixture%20in%20Saccharomyces%20cerevisiae" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/7/9/2919&amp;title=Habitat%20Predicts%20Levels%20of%20Genetic%20Admixture%20in%20Saccharomyces%20cerevisiae" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/7/9/2919&amp;title=Habitat%20Predicts%20Levels%20of%20Genetic%20Admixture%20in%20Saccharomyces%20cerevisiae" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/486369/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="486369" id="node486369--3" data-pisa="ggg;7/9/2919" data-pisa-master="ggg;g3.117.041806" data-apath="/ggg/7/9/2919.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Habitat Predicts Levels of Genetic Admixture in <em>Saccharomyces cerevisiae</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Viranga Tilakaratna</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-9015-2707" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Douda Bensasson</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2919-2929;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.041806 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node486369--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Viranga Tilakaratna </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Faculty of Life Sciences, University of Manchester, M13 9PT, UK</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Viranga%2BTilakaratna%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tilakaratna%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AViranga%2BTilakaratna%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Douda Bensasson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Plant Biology, University of Georgia, Athens, Georgia 30602</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Institute of Bioinformatics, University of Georgia, Athens, Georgia 30602</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Douda%2BBensasson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bensasson%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADouda%2BBensasson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-9015-2707" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Douda Bensasson</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:dbensasson@uga.edu" class="" data-icon-position="" data-hide-link-title="0">dbensasson@uga.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=2919&amp;rft.epage=2929&amp;rft.atitle=Habitat%20Predicts%20Levels%20of%20Genetic%20Admixture%20in%20Saccharomyces%20cerevisiae&amp;rft.volume=7&amp;rft.issue=9&amp;rft.date=2017-09-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Tilakaratna%2C+Viranga&amp;rft.au=Bensasson%2C+Douda"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/486369/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/486369/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/486369/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/486369/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/486369/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/486369/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/486369/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/486369/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/486369/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/486369/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/486369/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/486369/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/7/9/2919" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-9" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-14" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/7/9/2919.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/7/9/2919.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/7/9/2919.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/7/9/2919.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
