<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/7/9/3229" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/7/9/3229.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/7/9/3229.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/7/9/3229.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/7/9/3229/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/7/9/3229.atom" />
<meta name="HW.pisa" content="ggg;7/9/3229" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Accelerating Wright–Fisher Forward Simulations on the Graphics Processing Unit" />
<meta name="DC.Identifier" content="10.1534/g3.117.300103" />
<meta name="DC.Date" content="2017-09-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2017 Lawrie. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Forward Wright–Fisher simulations are powerful in their ability to model complex demography and selection scenarios, but suffer from slow execution on the Central Processor Unit (CPU), thus limiting their usefulness. However, the single-locus Wright–Fisher forward algorithm is exceedingly parallelizable, with many steps that are so-called “embarrassingly parallel,” consisting of a vast number of individual computations that are all independent of each other and thus capable of being performed concurrently. The rise of modern Graphics Processing Units (GPUs) and programming languages designed to leverage the inherent parallel nature of these processors have allowed researchers to dramatically speed up many programs that have such high arithmetic intensity and intrinsic concurrency. The presented GPU Optimized Wright–Fisher simulation, or “GO Fish” for short, can be used to simulate arbitrary selection and demographic scenarios while running over 250-fold faster than its serial counterpart on the CPU. Even modest GPU hardware can achieve an impressive speedup of over two orders of magnitude. With simulations so accelerated, one can not only do quick parametric bootstrapping of previously estimated parameters, but also use simulated results to calculate the likelihoods and summary statistics of demographic and selection models against real polymorphism data, all without restricting the demographic and selection scenarios that can be modeled or requiring approximations to the single-locus forward algorithm for efficiency. Further, as many of the parallel programming techniques used in this simulation can be applied to other computationally intensive algorithms important in population genetics, GO Fish serves as an exciting template for future research into accelerating computation in evolution. GO Fish is part of the Parallel PopGen Package available at: &lt;http://dl42.github.io/ParallelPopGen/.&gt;" />
<meta name="DC.Contributor" content="David S. Lawrie" />
<meta name="article:published_time" content="2017-09-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Accelerating Wright–Fisher Forward Simulations on the Graphics Processing Unit" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Forward Wright–Fisher simulations are powerful in their ability to model complex demography and selection scenarios, but suffer from slow execution on the Central Processor Unit (CPU), thus limiting their usefulness. However, the single-locus Wright–Fisher forward algorithm is exceedingly parallelizable, with many steps that are so-called “embarrassingly parallel,” consisting of a vast number of individual computations that are all independent of each other and thus capable of being performed concurrently. The rise of modern Graphics Processing Units (GPUs) and programming languages designed to leverage the inherent parallel nature of these processors have allowed researchers to dramatically speed up many programs that have such high arithmetic intensity and intrinsic concurrency. The presented GPU Optimized Wright–Fisher simulation, or “GO Fish” for short, can be used to simulate arbitrary selection and demographic scenarios while running over 250-fold faster than its serial counterpart on the CPU. Even modest GPU hardware can achieve an impressive speedup of over two orders of magnitude. With simulations so accelerated, one can not only do quick parametric bootstrapping of previously estimated parameters, but also use simulated results to calculate the likelihoods and summary statistics of demographic and selection models against real polymorphism data, all without restricting the demographic and selection scenarios that can be modeled or requiring approximations to the single-locus forward algorithm for efficiency. Further, as many of the parallel programming techniques used in this simulation can be applied to other computationally intensive algorithms important in population genetics, GO Fish serves as an exciting template for future research into accelerating computation in evolution. GO Fish is part of the Parallel PopGen Package available at: http://dl42.github.io/ParallelPopGen/.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2017/09/01" />
<meta name="citation_mjid" content="ggg;7/9/3229" />
<meta name="citation_id" content="7/9/3229" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/7/9/3229" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/7/9/3229.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/7/9/3229.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/7/9/3229.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.117.300103" />
<meta name="citation_pmid" content="28768689" />
<meta name="citation_volume" content="7" />
<meta name="citation_issue" content="9" />
<meta name="citation_num_pages" content="8" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="3229" />
<meta name="citation_lastpage" content="3236" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="David S. Lawrie" />
<meta name="citation_author_institution" content="Los Angeles, California 90034" />
<meta name="citation_author_email" content="dlawrie@alumni.stanford.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-6782-9986" />
<meta name="citation_reference" content="citation_journal_title=Trends Genet.;citation_author=C. Bank;citation_author=G. B. Ewing;citation_author=A. Ferrer-Admettla;citation_author=M. Foll;citation_author=J. D. Jensen;citation_title=Thinking too positive? revisiting current methods of population genetic selection inference.;citation_pages=540-546;citation_volume=30;citation_year=2014;citation_pmid=25438719;citation_doi=10.1016/j.tig.2014.09.010" />
<meta name="citation_reference" content="citation_journal_abbrev=Curr Genomics;citation_author=A. Carvajal-Rodriguez;citation_title=Simulation of genes and genomes forward in time.;citation_pages=58-61;citation_volume=11;citation_year=2010;citation_issue=1;citation_pmid=20808525;citation_doi=10.2174/138920210790218007" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=L. Cebamanos;citation_author=A. Gray;citation_author=I. Stewart;citation_author=A. Tenesa;citation_title=Regional heritability advanced complex trait analysis for GPU and traditional parallel architectures.;citation_volume=30;citation_year=2014;citation_pmid=24403537;citation_doi=10.1093/bioinformatics/btt754" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=G. K. Chen;citation_title=A scalable and portable framework for massively parallel variable selection in genetic association studies.;citation_volume=28;citation_year=2012;citation_pmid=22238272;citation_doi=10.1093/bioinformatics/bts015" />
<meta name="citation_reference" content="citation_journal_title=Front. Genet.;citation_author=G. K. Chen;citation_author=Y. Guo;citation_title=Discovering epistasis in large scale genetic association studies by exploiting graphics cards.;citation_pages=266;citation_volume=4;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=G. K. Chen;citation_author=K. Wang;citation_author=A. H. Stram;citation_author=E. M. Sobel;citation_author=K. Lange;citation_title=Mendel-GPU: haplotyping and genotype imputation on graphics processing units.;citation_volume=28;citation_year=2012;citation_pmid=22954633;citation_doi=10.1093/bioinformatics/bts536" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=A. J. Coffman;citation_author=P. H. Hsieh;citation_author=S. Gravel;citation_author=R. N. Gutenkunst;citation_title=Computationally efficient composite likelihood statistics for demographic inference.;citation_volume=33;citation_year=2016;citation_pmid=26545922;citation_doi=10.1093/molbev/msv255" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=G. Ewing;citation_author=J. Hermisson;citation_title=MSMS: a coalescent simulation program including recombination, demographic structure and selection at a single locus.;citation_volume=26;citation_year=2010;citation_pmid=20591904;citation_doi=10.1093/bioinformatics/btq322" />
<meta name="citation_reference" content="citation_journal_title=Proc. R. Soc. Edinb.;citation_author=R. A. Fisher;citation_title=The distribution of gene ratios for rare mutations.;citation_pages=205-220;citation_volume=50;citation_year=1930" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=N. R. Garud;citation_author=P. W. Messer;citation_author=E. O. Buzbas;citation_author=D. A. Petrov;citation_title=Recent selective sweeps in North American Drosophila melanogaster show signatures of soft sweeps.;citation_pages=e1005004;citation_volume=11;citation_year=2015;citation_pmid=25706129;citation_doi=10.1371/journal.pgen.1005004" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=R. N. Gutenkunst;citation_author=R. D. Hernandez;citation_author=S. H. Williamson;citation_author=C. D. Bustamante;citation_title=Inferring the joint demographic history of multiple populations from multidimensional SNP frequency data.;citation_pages=e1000695-e1000695;citation_volume=5;citation_year=2009;citation_issue=10;citation_pmid=19851460;citation_doi=10.1371/journal.pgen.1000695" />
<meta name="citation_reference" content="citation_journal_title=GPU Gems;citation_author=M. Harris;citation_author=S. Sengupta;citation_author=J. D. Owens;citation_title=Parallel prefix sum (scan) with CUDA.;citation_pages=851-876;citation_volume=3;citation_year=2007" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=R. D. Hernandez;citation_title=A flexible forward simulator for populations subject to selection and demography.;citation_volume=24;citation_year=2008;citation_pmid=18842601;citation_doi=10.1093/bioinformatics/btn522" />
<meta name="citation_reference" content="citation_journal_title=Nat. Rev. Genet.;citation_author=S. Hoban;citation_author=G. Bertorelle;citation_author=O. E. Gaggiotti;citation_title=Computer simulations: tools for population and evolutionary genetics.;citation_pages=110-122;citation_volume=13;citation_year=2012;citation_pmid=22230817" />
<meta name="citation_reference" content="citation_journal_title=Swarm Evol. Comput.;citation_author=J. Hofmann;citation_author=S. Limmer;citation_author=D. Fey;citation_title=Performance investigations of genetic algorithms on graphics cards.;citation_pages=33-47;citation_volume=12;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_journal_abbrev=Bioinformatics;citation_author=R. R. Hudson;citation_title=Generating samples under a Wright-Fisher neutral model of  genetic variation;citation_pages=337-338;citation_volume=18;citation_year=2002;citation_issue=2;citation_pmid=11847089;citation_doi=10.1093/bioinformatics/18.2.337" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol. Evol.;citation_author=B. C. Jackson;citation_author=J. L. Campos;citation_author=P. R. Haddrill;citation_author=B. Charlesworth;citation_author=K. Zeng;citation_title=Variation in the intensity of selection on codon bias over time causes contrasting patterns of base composition evolution in drosophila.;citation_pages=102-123;citation_volume=9;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=VLDB Endow.;citation_author=S. Jha;citation_author=B. He;citation_author=M. Lu;citation_author=X. Cheng;citation_author=H. P. Huynh;citation_title=Improving main memory hash joins on intel xeon phi processors: an experimental approach.;citation_pages=642-653;citation_volume=8;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=J. Appl. Probab.;citation_author=M. Kimura;citation_title=Diffusion models in population genetics.;citation_pages=177-232;citation_volume=1;citation_year=1964;citation_doi=10.2307/3211856" />
<meta name="citation_reference" content="citation_journal_abbrev=BMC Res Notes;citation_author=P. Klus;citation_author=S. Lam;citation_author=D. Lyberg;citation_author=M. S. Cheung;citation_author=G. Pullan;citation_title=BarraCUDA - a fast short read sequence aligner using graphics processing units.;citation_pages=27-27;citation_volume=5;citation_year=2012;citation_pmid=22244497;citation_doi=10.1186/1756-0500-5-27" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=E. Koch;citation_author=J. Novembre;citation_title=A temporal perspective on the interplay of demography and selection on deleterious variation in humans.;citation_volume=7;citation_year=2017;citation_pmid=28159863;citation_doi=10.1534/g3.117.039651" />
<meta name="citation_reference" content="citation_journal_title=J. Parallel Distrib. Comput.;citation_author=D. Komatitsch;citation_author=D. Michéa;citation_author=G. Erlebacher;citation_title=Porting a high-order finite-element earthquake modeling application to NVIDIA graphics cards using CUDA.;citation_pages=451-460;citation_volume=69;citation_year=2009;citation_doi=10.1016/j.jpdc.2009.01.006" />
<meta name="citation_reference" content="citation_journal_title=Mol. Phylogenet. Evol.;citation_author=L. Kubatko;citation_author=P. Shah;citation_author=R. Herbei;citation_author=M. A. Gilchrist;citation_title=A codon model of nucleotide substitution with selection on synonymous codon usage.;citation_pages=290-297;citation_volume=94;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Concurr. Comput.;citation_author=S. Limmer;citation_author=D. Fey;citation_title=Comparison of common parallel architectures for the execution of the island model and the global parallelization of evolutionary algorithms.;citation_pages=e3797;citation_volume=29;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Comptes Rendus De L’Academie Des Sciences Series I Mathematics;citation_author=J. Lions;citation_author=Y. Maday;citation_author=G. Turinici;citation_title=A “parareal” in time discretization of PDE’s.;citation_pages=661-668;citation_volume=332;citation_year=2001" />
<meta name="citation_reference" content="citation_journal_abbrev=IEEE/ACM Trans Comput Biol Bioinform;citation_author=W. Liu;citation_author=B. Schmidt;citation_author=W. Muller-Wittig;citation_title=CUDA-BLASTP: accelerating BLASTP on CUDA-enabled graphics hardware.;citation_pages=1678-1684;citation_volume=8;citation_year=2011;citation_issue=6;citation_pmid=21339531;citation_doi=10.1109/TCBB.2011.33" />
<meta name="citation_reference" content="citation_journal_title=BMC Bioinformatics;citation_author=Y. Liu;citation_author=A. Wirawan;citation_author=B. Schmidt;citation_title=CUDASW++ 3.0: accelerating smith-waterman protein database search by coupling CPU and GPU SIMD instructions.;citation_pages=117;citation_volume=14;citation_year=2013;citation_pmid=23557111;citation_doi=10.1186/1471-2105-14-117" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=R. Luo;citation_author=T. Wong;citation_author=J. Zhu;citation_author=C. Liu;citation_author=X. Zhu;citation_title=SOAP3-dp: fast, accurate and sensitive GPU-based short read aligner.;citation_pages=e65632;citation_volume=8;citation_year=2013;citation_pmid=23741504;citation_doi=10.1371/journal.pone.0065632" />
<meta name="citation_reference" content="citation_journal_title=bioRxiv;citation_title=Strong selection at the level of codon usage bias: evidence against the Li-Bulmer model.;citation_author=H. E. Machado;citation_author=D. S. Lawrie;citation_author=D. A. Petrov;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=P. W. Messer;citation_title=SLiM: simulating evolution with selection and linkage.;citation_volume=194;citation_year=2013;citation_pmid=23709637;citation_doi=10.1534/genetics.113.152181" />
<meta name="citation_reference" content="citation_author=P. Micikevicius;citation_pages=79-84;citation_year=2009" />
<meta name="citation_reference" content="citation_author=C. Montemuiño;citation_author=A. Espinosa;citation_author=J. Moure;citation_author=G. Vera-Rodríguez;citation_author=S. Ramos-Onsins;citation_pages=321-330;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Journal of mathematical biology;citation_journal_abbrev=J Math Biol;citation_author=T. Nagylaki;citation_title=The strong-migration limit in geographically structured populations.;citation_pages=101-114;citation_volume=9;citation_year=1980;citation_issue=2;citation_pmid=7365330;citation_doi=10.1007/BF00275916" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=S. Nakagome;citation_author=G. Alkorta-Aranburu;citation_author=R. Amato;citation_author=B. Howie;citation_author=B. M. Peter;citation_title=Estimating the ages of selection signals from different epochs in human history.;citation_volume=33;citation_year=2016;citation_pmid=26545921;citation_doi=10.1093/molbev/msv256" />
<meta name="citation_reference" content="citation_journal_title=Queue;citation_author=J. Nickolls;citation_author=I. Buck;citation_author=M. Garland;citation_author=K. Skadron;citation_title=Scalable parallel programming with CUDA.;citation_pages=40-53;citation_volume=6;citation_year=2008" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=D. Ortega-Del Vecchyo;citation_author=C. D. Marsden;citation_author=K. E. Lohmueller;citation_title=PReFerSim: fast simulation of demography and selection under the poisson random field model.;citation_volume=32;citation_year=2016;citation_pmid=27436562;citation_doi=10.1093/bioinformatics/btw478" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=S. A. Sawyer;citation_author=D. L. Hartl;citation_title=Population genetics of polymorphism and divergence.;citation_pages=1161-1176;citation_volume=132;citation_year=1992;citation_issue=4;citation_pmid=1459433" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=C. Song;citation_author=G. K. Chen;citation_author=R. C. Millikan;citation_author=C. B. Ambrosone;citation_author=E. M. John;citation_title=A genome-wide scan for breast cancer risk haplotypes among African American women.;citation_pages=e57298;citation_volume=8;citation_year=2013;citation_pmid=23468962" />
<meta name="citation_reference" content="citation_journal_abbrev=Comput Sci Eng;citation_author=J. E. Stone;citation_author=D. Gohara;citation_author=G. Shi;citation_title=OpenCL: A Parallel Programming Standard for Heterogeneous Computing Systems.;citation_pages=66-72;citation_volume=12;citation_year=2010;citation_issue=3;citation_pmid=21037981" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=M. A. Suchard;citation_author=A. Rambaut;citation_title=Many-core algorithms for statistical phylogenetics.;citation_volume=25;citation_year=2009;citation_pmid=19369496;citation_doi=10.1093/bioinformatics/btp244" />
<meta name="citation_reference" content="citation_journal_abbrev=J Comput Graph Stat;citation_author=M. A. Suchard;citation_author=Q. Wang;citation_author=C. Chan;citation_author=J. Frelinger;citation_author=A. Cron;citation_title=Understanding GPU Programming for Statistical Computation: Studies in Massively Parallel Massive Mixtures.;citation_pages=419-438;citation_volume=19;citation_year=2010;citation_issue=2;citation_pmid=20877443;citation_doi=10.1198/jcgs.2010.10016" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=K. R. Thornton;citation_title=A C++ template library for efficient forward-time population genetic simulation of large populations.;citation_volume=198;citation_year=2014;citation_pmid=24950894;citation_doi=10.1534/genetics.114.165019" />
<meta name="citation_reference" content="citation_journal_title=Comput. Fluids;citation_author=B. Tutkun;citation_author=F. O. Edis;citation_title=A GPU application for high-order compact finite difference scheme.;citation_pages=29-35;citation_volume=55;citation_year=2012" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=P. D. Vouzis;citation_author=N. V. Sahinidis;citation_title=GPU-BLAST: using graphics processors to accelerate protein sequence alignment.;citation_volume=27;citation_year=2011;citation_pmid=21088027;citation_doi=10.1093/bioinformatics/btq644" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=S. Williamson;citation_author=A. Fledel-Alon;citation_author=C. D. Bustamante;citation_title=Population Genetics of Polymorphism and Divergence for Diploid Selection Models With Arbitrary Dominance;citation_pages=463-475;citation_volume=168;citation_year=2004;citation_issue=1;citation_pmid=15454557;citation_doi=10.1534/genetics.103.024745" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=S. Wright;citation_title=The Distribution of Gene Frequencies Under Irreversible Mutation;citation_pages=253-259;citation_volume=24;citation_year=1938;citation_issue=7;citation_pmid=16577841;citation_doi=10.1073/pnas.24.7.253" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=K. Zhao;citation_author=X. Chu;citation_title=G-BLASTN: accelerating nucleotide alignment by graphics processors.;citation_volume=30;citation_year=2014;citation_pmid=24463183;citation_doi=10.1093/bioinformatics/btu047" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=C. Zhou;citation_author=X. Lang;citation_author=Y. Wang;citation_author=C. Zhu;citation_title=gPGA: GPU accelerated population genetics analyses.;citation_pages=e0135028;citation_volume=10;citation_year=2015" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Accelerating Wright–Fisher Forward Simulations on the Graphics Processing Unit" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9.cover-source.jpg" />
<meta name="twitter:description" content="Forward Wright–Fisher simulations are powerful in their ability to model complex demography and selection scenarios, but suffer from slow execution on the Central Processor Unit (CPU), thus limiting their usefulness. However, the single-locus Wright–Fisher forward algorithm is exceedingly parallelizable, with many steps that are so-called “embarrassingly parallel,” consisting of a vast number of individual computations that are all independent of each other and thus capable of being performed concurrently. The rise of modern Graphics Processing Units (GPUs) and programming languages designed to leverage the inherent parallel nature of these processors have allowed researchers to dramatically speed up many programs that have such high arithmetic intensity and intrinsic concurrency. The presented GPU Optimized Wright–Fisher simulation, or “GO Fish” for short, can be used to simulate arbitrary selection and demographic scenarios while running over 250-fold faster than its serial counterpart on the CPU. Even modest GPU hardware can achieve an impressive speedup of over two orders of magnitude. With simulations so accelerated, one can not only do quick parametric bootstrapping of previously estimated parameters, but also use simulated results to calculate the likelihoods and summary statistics of demographic and selection models against real polymorphism data, all without restricting the demographic and selection scenarios that can be modeled or requiring approximations to the single-locus forward algorithm for efficiency. Further, as many of the parallel programming techniques used in this simulation can be applied to other computationally intensive algorithms important in population genetics, GO Fish serves as an exciting template for future research into accelerating computation in evolution. GO Fish is part of the Parallel PopGen Package available at: &lt;http://dl42.github.io/ParallelPopGen/.&gt;" />
<meta name="og-title" property="og:title" content="Accelerating Wright–Fisher Forward Simulations on the Graphics Processing Unit" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/7/9/3229" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Forward Wright–Fisher simulations are powerful in their ability to model complex demography and selection scenarios, but suffer from slow execution on the Central Processor Unit (CPU), thus limiting their usefulness. However, the single-locus Wright–Fisher forward algorithm is exceedingly parallelizable, with many steps that are so-called “embarrassingly parallel,” consisting of a vast number of individual computations that are all independent of each other and thus capable of being performed concurrently. The rise of modern Graphics Processing Units (GPUs) and programming languages designed to leverage the inherent parallel nature of these processors have allowed researchers to dramatically speed up many programs that have such high arithmetic intensity and intrinsic concurrency. The presented GPU Optimized Wright–Fisher simulation, or “GO Fish” for short, can be used to simulate arbitrary selection and demographic scenarios while running over 250-fold faster than its serial counterpart on the CPU. Even modest GPU hardware can achieve an impressive speedup of over two orders of magnitude. With simulations so accelerated, one can not only do quick parametric bootstrapping of previously estimated parameters, but also use simulated results to calculate the likelihoods and summary statistics of demographic and selection models against real polymorphism data, all without restricting the demographic and selection scenarios that can be modeled or requiring approximations to the single-locus forward algorithm for efficiency. Further, as many of the parallel programming techniques used in this simulation can be applied to other computationally intensive algorithms important in population genetics, GO Fish serves as an exciting template for future research into accelerating computation in evolution. GO Fish is part of the Parallel PopGen Package available at: &lt;http://dl42.github.io/ParallelPopGen/.&gt;" />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9.cover-source.jpg" />
    <title>Accelerating Wright–Fisher Forward Simulations on the Graphics Processing Unit | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"puc6C21qA5FGSh-obSEzDjftcSc53mOdU_NjlqiKz1c","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"486371","apath":"\/ggg\/7\/9\/3229.atom","pisa":"ggg;7\/9\/3229","ac":{"\/ggg\/7\/9\/3229.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/7\/9\/3229.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;7\/9\/3229"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;7\/9\/3229"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/7\/9\/3229.full":true,"\/content\/7\/9\/3229":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00226ICF83TwQz0suzmJCV5COuF+vk\\\\\\\/NGOYNxbYttHsCJY\\\\\\\/N6cXcR3\\\\\\\/+HzgmoEmEszhMj3xJ20aOtJiQ2QzD3ZxKByW1IhHjEIPOTD4\\\\\\\/P2W9QqPrqnHTJgGXWFvb91jcOuxtYKZ7cRNbECXgnMjxK9wwo3vXzPRvhtBLyTFN3bwuPYEBVSjzcaSIFQHIvcVDQRjWsJdAAShH8Wr9RFd1ksycjqJVjO9zt6HoJxIUUzjvorwzMe6l5\\\\\\\/YKEBx+0yFPCYJCJmYG4lepRYT3VdqRXn+1u0GIokzp+T0AS\\\\\\\/WkOAopTJZijvdxhLy1fr8xLITxZ2TlYu\\\\\\\/XoV8Es13yAIRyzXdUnqp3t5puMUTB9CjgKyIPT4oLN1Q6Eg6A94c0zb1nGFGZLNd0laYIukzzol1RcI\\\\\\\/Drt4VIIe\\\\\\\/iWU9cfrefvJFFRXv1joMv1zuWvWBUMF08YPkRhdzkyN5SwCz3u6PhfLnKa+qmqut1fvI1xLz2jxxKfwuvS9QeaDle6\\\\\\\/sMB3oGTihbn5Xrz7uO\\\\\\\/y8JTnf9hIwbpZKhG8Poa+ByX5svHld2+mDOM0hnzp7zD9zErAGIuhsGtBaZPAWVXhvuQtW8uxQ9zZOezyMK1EJ5XNjCvvwb5N2EpTARDDC1lvSuDW9CluFgXQhMD2k+mtMIiMI5weqD2edP7uLsIT53KyeZRbIZXQfDfsR2B6RMk5sLw7E4MM79rlhDkCWQeDHbgeXh3V39C7V9WJnC4isSsx7Y4lg+vbeMSUG6f6ZiF6VrBFPDXteipBCLunLZISd\\\\\\\/mxtr0yjVhRL8C3xsCywPfzh3HgWL3u5Pr+cUTUAIymORz5SJr08BaElf7+TBcXteJghHvC0+Qt4cEVZKNgZdsEqN+SywgKBVtZuR10Evexnw0AzqGjajbAJJQ+Zc5BCy4WU75sZR7gZVdQWBYsBxmHNGTIsKJVTWjLhBsPSibOjMKIll6PTQCz0vI4BpfdwZYIPjLgN7SNTElnRzsFWuFnuAjFB7WF63lw1ZGjeFGHgmvjuUw1AwgkqY7bqdRd9jJkMkdvsp4ES56YQsbszKn3gvnESb61U8DAm8FL7cjG81a8vRTzg8FAz6GDlrqERBK\\\\\\\/6T76g3Qy1EldHNHhdQCU45Oy32dDVWBOm6x8NPX9eVyNCQWsqhnKh8NuY0HYfSPhQi4piPlPd2N5qOpdFQZkRr971g4EU7sDEqxTicyHDJPTueZzShLi5k8Xm5+Za09aGTSpSvT3gfja2unfd0l9OPxXVWvQTpw529jqG73k6QadwrS\\\\\\\/2\\\u0022,\\\u0022iv\\\u0022:\\\u0022rQbxtvLj3CnoZSHcYF5KUQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002288d72c3aafb2a16ddef94528f24d04ff6e366e5a143a5f785367f92c494ecf1f\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00228ned71KAO\\\\\\\/EvMtoaV1WrbhpAHvHlMIwK2rPeF5P9nE+WSGTfN0uZ2j\\\\\\\/6DgyHx6Cdf7192Qc1AvTakG6aZRc1hDlxm5TuWrtD8TVwfkqiWHnBwYAmILoZ\\\\\\\/Ucu3D7XpzOv++Zy0l9KC6Yq5c+gmr1GonO+kJEZKx1fWkaKMGmOFkKf4rkL8H3nomEL1lRX6yPbUiGQEUJixPkEi9M\\\\\\\/6DbG0GGMRfG7WWxmvspVycSqk8EkE\\\\\\\/gOV2vjZRhN6mp\\\\\\\/ZrLwiqZQKgvwUXfbr6m31lqnfA2eyoy2a6+vKpMuSfPirxDzam66E2Q6wI4Ex+sdcH8S4UrHymTP8Qf+ocgu5cXUSX+t5Ut7hADdru3uquJ1mwRZcOEePoRJ47r\\\\\\\/xRjHnl24v9o2ZERTtWm23KBAvvIvWCM+tHeWjS2ktsiCj7LFQBfWoUd3EMSCA6+Wl9Fc22Ohk1M6UKLYoJmusnGEi3bKpsN0GB0zbYbpNoy\\\\\\\/yuem\\\\\\\/AJJ6p0\\\\\\\/d1sVAtQi2x90CE4z0ColV3\\\\\\\/ymra2uIu6fCf+U\\\\\\\/wVmNlEqJdRb3YKU1Lbpwb2PUIO8dzfjqgHjCdyhEbkEzHU2\\\\\\\/9KcFFHDxT+nmNAEbxYmG0IhOU232eBw1MaIYP4OgzQwFdlPlz4m\\\\\\\/QvK3f+iw1cPJUcbc1DgxsBOR0EtQD13p+FpWF17jFe6kIZs4ppYv6dJc4rCGkxzJx7+Gm59U9e\\\\\\\/1PIsphUPvL7YfU1qw3znHm9ouVw9EMy17zba7phYU8zmvz\\\\\\\/6LGMAuslbBHWDDqMN5i90erZW+9iCm60iaAbt6o0ZPe+yLbvn6yvkpE9LaS4LbV7kEbETllC33h3jiJCLjoLuArNAZrEOfr3PSdRB9xbOMnFdFg2vk7c7mgtf7jZIC1D3SCBbBwLPjGaG5AG9QsXDRKAJic1Hn2k0VWODXVAquXX20oUqTJ5iXfSpSVKw7ZQuFhjLl\\\\\\\/HHlssbFzxZ5wlP4ETGv+pKH8Pv4Q709r6MZMiP0Q0ZofRH+8dEHw476jhtaYWAwgCpOWR1vQifUNrgoORNLDRPwoXwpsuhHQ9k\\\\\\\/3bjgJNUP6PtF31gBMvZtQavvRZVqij1EEzpnIZ9B59K+RUZ0E60QyDsYZqsENArUdgjsoOpfXixmtVKbFHjlTqthfz1uG7JqP3sRMwUeO60DRRs7DubW5pkdN11lTeUR+oxXr0U8zUrdCZzoI66paAiwwRsQkC3eCy1gZ1AypaYwsMW33s+htnQ62t5Xujv4gvp6Lo06R2fhsahLgxWldjEQ33AiYnV0eANbteKX6mG5S7fwXhrt6FPJa2fl2+ZU10Fyq1G9ydkVCPh4FsdZ04HJqhyCe6pR1G2g4zBx+sKJXA7DYcumQbMYP++dPPc6TBy8cvd3+hcq9dtfuQyA+CwmV4y8krCo11mFuyEKusa1Em6hCMFx0OifCaQNHPpUDl6bW6wd4+NWNjJoUUDcG66voZI8EO0rhzBUWF49Kzmli5XI9byfBpTVc4hs0emDscA3zRIzDo6RytL4bZBmCzxKXNNQDqu5oDeeEeDMGFDf1jO0AprjfuI7+LWQCM4b7eWL6WDmS0VW+SbPo9MANI5yba31rUm6yrMvh7dL6dbY6lkFNn+sscqXvuBA7FWy+LSYGZuZZJEK3dEocModKHP29HGpu+kU+yIUuqXk7hs4kvUGzqdXON4BjdiMQic5bNe0c6WtTtjt9tH8noifDaJqLs4xqxPJ3sNOi0AckT8OOCVtgWlhfmhtIkax8PiD7kwgX+yk3Nqcb2\\\\\\\/AlsKZrroXFPXkwX8EfSYrkn57eilfsC0DOV3PIMixi5uWX5oZ7gvs2ecTWxlWnMsGmJRvc6lCNPpu22Ff3sMwWBGaAhGrmPgMf14y1+Xl20QK2hHHK5j2Uc3UbyPAvzEcDBUXZqT\\\\\\\/gHwFnnYVNM1213JO0+GVYp0n0rl\\\\\\\/NfLp7W\\\\\\\/Og0NSIJ7xd4yNOPl4Hu1GbcKrbObk0SDq82Zw\\\\\\\/cjLm+fM3wx05Ct1Rv+VvcAUg\\\\\\\/g4kFmTTdEJjp8PDteeP2YHfFbNYo3olMiCVqRrxWOQoVeCPvfeMjUUoax6EcSH8PsIpC2tYPR6As+vxfXKu5XgXyLTSvkaxUAJwGP5OQUr7p0EOY4439Zo5eWmxyEqnwUwvssShvLo76uuuHJbqt+wZJz8NiPD9zMqK+1NlF+U9bY2fTiEPreNT4F939\\\\\\\/B+0Ge1vZKXORPKrrhW+2YupFyi3Dy0\\\\\\\/fT7Frzk1DFUvgOEt4VQjqcqnjlTw6bESSbrjMal5HWpC5HNxq9Ld\\\\\\\/KJjkd\\\\\\\/43VOxDn7dwEkdc7B8jrAbPd8GfS5kr8HDTXFT2XOYME\\\\\\\/tvdxpO6ABwUjGbBF6HjAGq\\\\\\\/wB2MGabd1caU4MO4FOasCmz7puAh0bXzj2wGPN\\\\\\\/PGui7hbrrCrcfyhZU9SCbNwJWAWbYLLl\\\\\\\/llOEds90xrZuEj3fD82\\\\\\\/zioqKBrEkBR6E17sLcRB9N92QVJLrEfYSx3mqVQqjFXk4aKVMZSTwxAL+zhrp7T1ijymLyQ7ww8V6dOsLSQoJe0BP1dq0xggyY5ZqpwOAUdh1m0cwSh5esUBuZCaEuOjC5u6ipoMIqRimXbmq8wPDiG37DpgP22zlkTQ4nfY8fIMRJD8aVHqbF5AIM+wJ55Lgo1aC6Sxn4adkDT\\\\\\\/nSTigsS57YufKVJ6dhXZ+Lg1zs6RXwBF+WtvuRv\\\\\\\/4gnItY7zM5kwwfiAouxAcw03sP9vOjlIJqz4atGTtEUvYP8HeFdzfOe\\\\\\\/YFlWtAfMjsW+HhzpkOoNKbBUEBzWvCG5GTOUY8K+M0b5G6usyENgGmW8MU4gwm517wiiUtvYHdJK057kOsTORtVtG1YJtZELSfXO9lQMMHKrmABiw8GDklh+4qeM2ygr69JjmwBjTBtENI6h+guauho9TBnEo12O4pWt+rUQbzpNhaXBzvOETHHPpbmtfUSEcSqLxCH6MveMANcJlMW\\\\\\\/B8vw5j3feoCArIIEtYOesYRA\\\\\\\/ZgHdVCaiY5ZnxfA6M0qK18JMXAFMO29zx3lwePiyos4\\\\\\\/MJNWgUxU8orYqFHxuuVkGVqADU9FBuJdh+HOvT27KzLAUK94F1iTOGEpEWxU9fReC0ePsnGn54BJ4rmwrCvV1EhvoFwdGXw8Ji\\\\\\\/MQuzjzSoyEUKWqVLrQpkAF3tjBP1Db6RXbg4jgXFUu0HAKatg90gqRxR5tBQNyhdsuhkxic7xlAkZy+umMrKriLUmIyKbZ2EU0thZubzegYNRmVxNt4iNX96c3TDDhcuGKFLvut7\\\\\\\/lIzeK22qR66m8Lbdz8IIKrcHGsv+YSfRVQJxEhwFumnvJC9\\\\\\\/4rmJUPaRes47fNzon2PdX63jrps1Yq5QGa5f6dXcTWmd8\\\\\\\/EfjUl1ZpyXTGVz0Q3bUj3yYR2ioKWJxqMkpnhjoUCv5u8kxZJaedYiHHYyw7zpIjV1+vPMK70k+hDRBUSVo6kyAiQ7diWRJi3icacLzdA+Wu0ax+RRwMNjAs7BpsfxRHv39dCXEuiRyDGo8Oaf1wqiuJYDqHYTnKnbEzPcxe+FcyJSxlCeINrkq0bp7V0gcYo+q7s8W7+mbrivZYIKxyGIdV1rdP9c4N1BneRBjcP8GLlHB7mOWfNTEJWMURZPFdPoqJEdRZgesRvHHlrChP7vNJ2OnUFZWwKbzYI437pTnqrMrLh8RMlP1949rM3euWMV0oOSMJoS4CGgrweh7eXKqTC2OyM0KDX0IkAwSr+wRvx7JVw\\\\\\\/VgjM4Z1RbhspEAIwGriu7AL11vNSRh+m68AyEtp0OH8uEPRZ1IJV56hDP+S\\\\\\\/7fnBQlm1kCq3mnKbh1GfnIri\\\\\\\/P7sUbAbTSy06GzlWi\\\\\\\/A7OFkRoqAhZ+i1bxXvWoheu28fCSIkKNlYdegYxtsT5d7FdD+PsI9oOZi5koqVumiQmMd6kd\\\\\\\/YQGGdgMEKo32DINH6eX7\\\\\\\/iJ7ahe1v3im3oQdFUASGaehTlSoF5MGJtUvUMvT37YZZ4AkjfWeDr5oUUdajSJxeQPtSIrxktxc5rY0ZEKB1ZD3aSxi7q2apHSUGXjg3OMXuXxAVTEvYIsBLZ1kQDk6zDoUZnonsyU5nNOYZ7vgH9TebfuggtRhZLNKylacR2eFDrDFAjExIm2LZHhhoJnJ5cFjjVE32gc44\\\\\\\/2l6YWZB\\\\\\\/6Urh0JPC4h7LVE4cdhwc3Sj4isKBmiCwdZvV5eXPo7oqzJuCiF+T9O\\\\\\\/b6Xyr5PsaKxo+z2IZToYHQCkRXZnA4ajPRHImJqJ+v\\\\\\\/er2c6ryDW69+s\\\\\\\/j3+EJF95yC2jadaRVrRQ1tNtH0FLe+RxBGze6lP92r8pPbkINCB5180+d0nellh2AX8qyZ4EBscKz9SoE3Bv6upniz6v88Ggk4V9ZJE8uvBcSYKaIRQodjtDpzYbnydTqFWIvdjI0UKI+ngXRlIHdUaKOCFsE4QA7w7lilhIUeIdF4tXms6LpEJbe7ubvwOoCm7PE8BnMHI6J8Tp+rpliNhkEmgbSDNkM3pG+TIsMyGFSZlF65h2S6cr3hy0vBgxhJeo4KSfUMo\\\\\\\/mNsDQoea1ZYeS6fSyy+bGgUT77cPD4yk1Ajm9G5+YHpHmHRVWxFPICXbeq87\\\\\\\/snfnFi069OjAeQa4CWnBASHCtb2Gj3UqH4XFTTocuwmIhg4AtPxJ8zatsGVq193t3PpeFdOshiokYFC01+xtmNM9pJgx\\\\\\\/\\\\\\\/Z5uuvFUlUnf9d2YNOlzQiESMN\\\\\\\/cUG6BmHTbhMvL5ZR\\\\\\\/v+9zI2QmJ0KdUqR7s\\\\\\\/jp3KuT1fXFcti496KG6GAm45tcPu1KQ52CteBGkoiyDy0+QeTGn6fJyHcO0KDA6wkU659eVLllC0zo62k11o7fdN3vFNqSlhnFP9QPwkawWGJj\\\\\\\/VOMLa07riGFTbBkmgzel+YcJEE3LSu\\\\\\\/cjPr6Q+CADQrV\\\\\\\/dqcw9fMaueni9dX1wpOjHyI7ysHR5Cyctupa2OFqqWjTxS9E3xBRK7yEeO9Q1SOtXLs1Iy0lx4iP2O1caro0OsgWL3Yz\\\\\\\/ivBnPsor7Vp47PyVl+i1uiRIw5\\\\\\\/3wCBO1QppRcSHM1R7tkzJCwgaZU\\\\\\\/XSPJ8vBOi7YI1cjolmO8kg0dVIiNVH+ClHZUVubg0RumbLHys64y2aSf85\\\\\\\/yBnJjuPB0XOsrfTEVr5WM4FpMlkHchRFZbx4KLzJ8G9Jyyz5gFrVgeDVtyDB0fK1go7CCSiC0mAGk88Mv6jg5uNuInzrLT8aI+OOSXetksGXMZ6\\\\\\\/O31ZYlvOi2F1Uy6xPbPGnqApeei3\\\\\\\/6zrqqY73Ov8UuKshJJNtPG\\\\\\\/\\\\\\\/eh1pNSbOyt8VN2KSGlh9wf5fW\\\\\\\/qzsnqV0\\\\\\\/zNEZ1QHFPkiysSBfd3YzQl\\\\\\\/eCKi0XUdRhMdAvB9LfARfdtLqwulvUm05JtFvaHkyMV9fS5aNgcBf\\\\\\\/cV+X0\\\\\\\/qTgQbw0wrqBU1YY2mULj6KTlC3CZhosW+\\\\\\\/YgNiI2XMuGcfBcKSinYwHo9V705+KTwIfVjrzxl5+09EVOYxh5C6OcJFFN8IroXYXVDwjLQmkIh3PWl\\\\\\\/6cFKvonlzcBomkx8JxJW5UERUrKs4yrcESw3q1LEOjbENu6KL0nhCUkSTsXxmTaMVPhsIdBLiBKE2FHumwVNfKpWm6+Ax1pUClGKmFDW5rqp0g9iXHtG13Gpczq0BMZn1s4uAg144gD8FWG+HNAh3U+0wYsJndpVNAHVpeykAUHz51GlCYsEYDJPz9ybL9BGf+7nB7ae3ye78iwotUBmqZ3a38dfxW9hXE+TFfzpMswkbLTGMC4eR3plMtu7Wna6vLLeRFpwXwtCINrO\\\\\\\/jN7BV9aNEOcyS1sij3RNWzQeXL123kraYbLrjp\\\\\\\/dVk4EI3h+PkRw1YPFi1IzKufa7L9QL9sWJfYUWSlYOWhKVfNKuPouCRHpMN3cwkRDQV3iYurXqkLMeH0LO6tArJcA6Y622ef24HpPmaMdI7l6nKeceV38+s+sVwXk48zIsq6eKFMgeeTKgizbYA0iBYzEF6umcc6cMZgqWWhR9\\\\\\\/8Vt66CXdWzbC3WSZic6p6nXYDGRWR8yAjQpnB7fP8C7InCShEHe\\\\\\\/OGUML\\\\\\\/ckRGOH7ma7HzwrIB9pvVjS2Yz+n\\\\\\\/cSRf6VHAgmW7rhPjzLKxYDnLayb+2rINbL15bVPIzECgVZ6+1nUYsdbp0jXid4TZ\\\\\\\/nhlHwUngqrxo6bXXxEsPndq8DdNgD+aUuXppQSTe5l7U4UrOTDEcQ2jwkHJ+UHZPJA1o5Q2sconm5wz0TH+6j+tFHCtdGDvCFzk+9nbPuVHiLsfDQdGSELO4z7ayIFFd3DH4w\\\\\\\/REEt28TG4oXD6keBXqpuA6mvlUXouXIKVA+JNc6dcmpwV2tDKcKF8HtFnyqKlxHRuVV5QjWdhgIJLdDXgjgblVofUU1mRxgS\\\\\\\/mAQ7qZbWAvFXE\\\\\\\/SgUtNBr6v6zaDHN1pY8LpcPjJYzMNkVoHFmdtcjOc+hy6WEUvgJ3aTTcfd26MNKQsoj6BoGUuP8OOA1lEaJEF2sjR05notFymlQ7SGZrGbXCRLZwW49GnXtSc19Rz7HMbNmcIna3HYklI5IYp\\\\\\\/geGryvG2CwCimKh5Wg6QyVnurif9XLmzPLzIdlk87KxGuAEsfZL2ER82O4ANTKT91lAhaShF+YXYlvYedm6l2QMN2Af\\\\\\\/rTrdHdBG6tfcXHFoozstSJwTHoq8\\\\\\\/TicYPDzPzigI\\\\\\\/vqKruY9bVEvsFpbKiUrI7vzGDLmfbU8Bd8udz2dsJm0h+ConQXNNUWADJP34g74mrrqoO8mn6B3CTDPdv\\\\\\\/byfn2VMY2P28+IBFP0xfWC5FakF39KCRJV8ZIllpYIZI\\\\\\\/WaoquLDqYHgX3DEeLIPcZLUhsfH9H8gcPgdp9UaLdedNLtTyevmmazzaeIeBA+EsXf8+fN65TzOnDnRweGkXW33fKrAkYlvbvALRuybQVih0iFvCSvQI5r94ZupLE1rZNablhvnpprgJnBow1y7i4cTqwka1oDvykax4EJP9jmtrEDAv34+XXR2uIFyZmbFnsJzNM6LiqoPPZPug6CBvltTCfD1adHnoP\\\\\\\/e5j6JMn5yoaoAMZGnX+GpOhXg\\\\\\\/EThuM40g981azvblI7q6q3m7Hz8ZQtcefXhF6Z+YgmKRWWOaS1MJKOJjYknZuKCVl5kbeZ9u5n5\\\\\\\/2rywGhKE32\\\\\\\/zLcG52hDyXPrZ03zWdtAfNuWlpyp\\\\\\\/FRBVD4Ja0pErhk5cwSi34WCTizeXEtaX9xIZMzPobbBvan87QAtLRxTCVOvp21mGeyDugTzDJqv0+AnjbGcARh8I\\\\\\\/hJwsoTezFoH2Ef1v4efSTyqrhLnzPW0KpE9B+IyPLMS7+Xg02PdpUGDGkpMJk2KqQDqAra5oxooH8c9A9UXN\\\\\\\/DqjZPXcYeFVoTFPQKNCeIXJF9id\\\\\\\/lHCMCA7mhUcPDtu1HwrvbM4UWy1MKCkrH2J0mt04Pl5J7wf7HAn\\\\\\\/JiwaH4ifkOl+eZoAI765iTomX6+OXqCLXML2waYiCXfkurjvgjAcKjMNtWQ6X7OIGmTdig89tI4KrF8RyGoJnO\\\\\\\/RQ0Z0JbfdZcdRgpmrdIhhMv4IEVeQfW12vPNt1V3voPmThOQwcAobu+JgN6rzEgTmR7\\\\\\\/v94xbfT5SG4c8mlquYsQD9HOrxD5lHJFT8PprG9Fsf\\\\\\\/gxxWTRoWC3hqpWFy711MRKzfnk6o0LMmP\\\\\\\/h4QDfhDK79Y2f4vLgepX5ozVIeneDW3TnlJg7KHKvcW3pcEPLN6SLve9PzS1Yk\\\\\\\/wyr5o72Z83Yru7yX1hw3cx8CYm+8WeFxMAmHWeON7o9Lbj8pm1SQ53i4wUBLDwZLjiH8cRE\\\\\\\/zQAqZ3iJHd06HfhCmFFEqjW9sNu5nB4tpWrzur7klkisbqQIWwlrAd6ImOtFYs\\\\\\\/p4+KFe37FyOcrYKuq5xsvK+LRrSx1qaQr4FJ73DGamqA\\\\\\\/Ns7+nhMhFlIgjq\\\\\\\/7rKndTKluFhLu9zSEu7YC5L1XORbg\\\\\\\/s3GT+gw7sF\\\\\\\/2jqJRhHlvQmJu5mf7V9lAI3BDTMlIzAJ6ggy6cWoFf5raVjDLcJOAh6TEfrpsB+dCDHhxxIf7EImpNEUQkeGbso\\\\\\\/maW+9BqOvHoHw2v8n67M18f14+SDkyvrRBTYUqRFwdvnaZLddDLQHj6QQkY8e+1clr1nIgU6IghQPYs0fb2ZJ8gMatLlPfU9mbf0KefK1NKnLRrbZA6U07BjG8FRbRjieSbTaCkWA7Cg+4Wuel84hR\\\\\\\/GHYPPLFNzDmaHynoi0C6GmRBgGs6LJR19uCFOW2JiRbI+oUJWqSgkAhdvM0PP6OchQgkZILl0zqah16tGXc\\\\\\\/rO5Bw0xp6asb5p9TyJmSl3Ei5EoNMXO1aa5FSDf5bukSMDdYtjl+TK3tA83dl94TeVUry9C40Vv713GT+AaTXLysaFcxKg4qPSerlt+M1bee51Wg2swdNgR+PcWWo5GgvmgrFWzZFb1zhjvCuIjm+YDfxPNtCGyPLw2UW61hi6GnjjUkXbXtyGETUn8Eb3WnXbB81fJUZQTJjCGaKp0C7uMvVGdn63\\\\\\\/2PrnfXw7FVVmZwCkr9OFpqStr6thc1g+PYs\\\\\\\/hvw7HCJpJSd7rQiYcFwS3Tst2rkPUcU5kUBT14k3k4n6M0O0qAokCaklwB06GHS+5IiFECTv89ojysJIoWNyq4rmcRMsLjFufCmkM0oNseMBHzkwcPkF308OPtZ9GZHDvHCkT+pBVxv5JquahBjfcFEqphn\\\\\\\/iMFrwVayu3ahbnz0eF51ukv\\\\\\\/Buxvuc8pJHGGPMEPGRiR7N4eOl92WrINNiUgM1DhXCPRCBP0ZXLb8+cjr7vy8EOGc2BAR1rxfkXcerGBKKg5cWJHR7a6l\\\\\\\/8OIocExC8PEGPlHUq6RKiBtVqPI+GSwBgV1kF+6fL6KBJru7HuQPiNO6msmxNrOx9rLw4hBC97i7mV48uSKL2UqjZtzVDdcliPv+32r9Eq2SR5R+czlb\\\\\\\/vBd\\\\\\\/Nr7dA6ta8MLkmYukxMx1nSSxlxG298zR99oqMOWV9+yjivyqdxdLv+jK4DBYuE8zc997xXqQenNZPwFSqIooEhHE\\\\\\\/H+v3RNyyRfF4YxhPXg3UBGQzGOl6blxPwWF5rP1LUQUQT7vARdo8XqmJjGCo6iWe6GMHsVzxuX7sYbIZ4NkfdCwmbegzUrNk+cusWauXZ2pvNyRfVRo1o1xFjyh66ek52Vn0v0t1HybCzA7Ub4iGoQGxoX9jssDBtjKDo1cQUPldEnyGM+XwCPXVua4XAt+4mIsM9Oq3sqtuVncNqvXgr1CDuCaKV7Z0os9+MLajhUjCK8usQwGcZiq9JMm4Io2\\\\\\\/hJblurjl\\\\\\\/3QqDZjv5jg\\\\\\\/gSHcP9IVc7gvPZwbvazMqQ+VCScWRgkwL8dczf31PCwxtS1Z5xOjdXueoDitxfohSLHW0+BIy+2NIT\\\\\\\/xhLXQ9wqJoX3Vc8uqpSgTQNSYFXXHbN7SKyhzJAc58jXBhqZRu00yJgFlccB54SuF+e0iw2lvBJf0yg+P5dZ4m\\\\\\\/5RRuPBpg9ueh5B4RkL3pjlzFudQTcFm3K\\\\\\\/2MI2hOKHZunvL\\\\\\\/kOsM\\\\\\\/kELmsaUSXXyxCLQMsDhkWJ\\\\\\\/4VwedZ4s+zUu4xAS2qWSbgzZXz\\\\\\\/ymVr0GM9qCMlEbChRwfCkWh9ALFcEo43zVg81Xzy3GI5c7pbwPhu0X3E\\\\\\\/W7\\\\\\\/De78n9m1NlfKJVpzfgQzwYLE+SIyJ1Vff1JALtgg6FWaTfl\\\\\\\/wkIkjOIWBk28b\\\\\\\/q3s3p4e+9ljioiH2jfpKji02awrCBJGkwH7yOH7MYIHy\\\\\\\/X2LeE1Gak3NaIxYOEomdUyxJmbIvDLgZlPzbUDRBzpgjHKGUyTAlgiteaSfjJWArnhVlzNq3G\\\\\\\/fyyteQ0P7Rl9T2lxEiD9bjbPi6eYQSbzQ7cp6DoQb0h5I3kjpOeyh3ebPuX4kpXrQyG1mpaBklvLojoBiHU0hDzS7owsSejc5n73yuSE\\\\\\\/dBdUJC\\\\\\\/FOgWfNYcYXVTbg2twlJb5wPMQGq1\\\\\\\/6A7eJXWCeDV4KeMT+ZVTPxt2qy1R8cf5ZONPMDhETVAIzb1mXlTb3WzkxM\\\\\\\/3lU2S2jolpY0V8e0REZp6qzyEIjTGB1MYvYw7l7BUY3ETU+gv8x\\\\\\\/uoT8lw5w1Ny+sY2toA3IpLcsiYLSIUVj+SxYCM5XCLL43p9Ti1D6pM3xiYVmns73ryaO\\\\\\\/w3hPl+Lkicm\\\\\\\/SnoqlaT\\\\\\\/C6Lw7KiPtO\\\\\\\/QOd2QFbocS1LQ+HY1RLs2OioMqlQOd5lITXQt03kUcdO6A67CgN1LBAMPcB4G8o1Fi+ckVPTomAQZkcvIuKExBDA6+G+cZeP15kxfV\\\\\\\/3BN6Xem14pFy6NjZUT\\\\\\\/qyPI4T8jloJgqRBwjCGdijL5QUXCbmQtC4RXMMmIByLEeNsCA6Uh\\\\\\\/cKkB9qiM7wVNWlIadb32xtFkmR6YR9+FuIELvM9xxoyAIvsnLsjnkSnDciCX6TbX+rhm333QEwXC4Isdgc58qcFK7vY+ApkwgHl+LXDOwcy074YpBuX1mIMZ6g5ZDv1gyVlTlhUZiEJq9DREd0z1OYjj14h5p2YMpnBenAZelYK5QnMxdi6qbfughz6LNhEYhKMGOwCGode8GkExpWpMDaY32d3r56K8G7JNV+TQjP9cyedldxPydE2Uhz2fwL+TNQVBceDw7JhWrpVl+t5aoOZDCQWHjBCZQU0BudZUoRUwLLqGbB1aMYLwIcHopONqZAtAqTuRjvpROCe0OJ+Dd4yaYw\\\\\\\/PyN07537sdteWdz8AZvvxgpWaMaNcXE2w0tkrfFeZQ67K6WFQFPOPQEF\\\\\\\/0tKJMLB9wjZ49pUlVXuxJ\\\\\\\/2rHNTzapLy4JUXsnRmGKITI6AYfdsLisFVsUnpqLNpmt2uy2aSnhuALG9QJsW92uznFcvQ1FbAw0WiQkZHSd\\\\\\\/xNsmO+As8hSWtFCOZMYDFHp8rLTA3Gi62+uIVfE9mjGw8w5KP9PvtK674lkAUMufCF3ngeev75gYOxxtAldonmwVgP35uFEUOud+uq0m5dVLxwgnC5hXH3mN4P1kh+WM47J6\\\\\\\/FCJPIUWY2y7+kImW4PEUDryB8RaHwHj\\\\\\\/IAGvds+TiZ2BGCE68gv6WtobZ\\\\\\\/uuCSYrI60afeHH9phVjVdTIXQ8EcNCo226EwtHUE7rQnslJjFZBfHepv2j9fUf1uPjbc+\\\\\\\/oPCIF1NmevCKDb\\\\\\\/mAOb1APed5R1jUzjMAqwOsH4Cg5WMXdyUazy\\\\\\\/1+CRLyxXN9qqH6vCMq31EO8qJ0F3PN4JyussWijb\\\\\\\/qK5zWlsYnjGBsC3f8nRCvQSHhajO+qvAH5i6Vfd\\\\\\\/jvFVHZPvu8J+\\\\\\\/rqlwP0kkDtz\\\\\\\/SIUWDXPClg7dV6PCd2Re0aRNGe54LUfYTYHWh2JrY9k5xEIYVCOX8CwXBfRIxwuqfiSGPWPbn8ac+zJUR\\\\\\\/xkr1u2yaPVWb6tWq8GBYHKXGHZ1VUfBUCvxpafOeDx0Mx6xN\\\\\\\/+wwjLYXoi34LnmBY4JfYUeXSUcW9Fzdmghoj4ol7X+m6RZXxaSDddPvohqSYLPLGuzQjNn9KkdHLynjbYIlR\\\\\\\/GqI34T4LoDwSSPk5mFbCCIOzoqCupHsLI50LKtInumpJmY\\\\\\\/IqXVBuZyOCyd6fYseT6McsqYkASO4\\\\\\\/Wj0EM1pZyQXLg0DO07xnMt5y5AIi2HPIanHmFQxXHBBeZFEPPIb14MfRDOS2J\\\\\\\/TWUZNWQ5w8dEj5eU1oeNFQ7mViSyDE+a+2oMIPwRXNSH3NSwoXdtFN0KiU\\\\\\\/bPDfxfitXzqZn44sPf0VT+rRn0AbTSMb3hXkfmzIhXugBxaUTQMj9lRjxodCtTMmTMJtfbgVZvQcNYDxbkkuoKkxswtaZBPTVSks9sOHQO2r8RXrTEDXrcFj0MpHdjzf9VcLIRBwBWsvhaUfZ7gkzOUoN\\\\\\\/g8RT8p5pJ8MvxAJLdHIx3LFQWF1h\\\\\\\/QLuytDaNnA6E1HVAlaqRn66PS\\\\\\\/Vh3wfy3aPgrHtvEyidSzoL9Bgffi\\\\\\\/q+15cqDhmiKfysFn5pQaSQgSqcvEaeHgwvmvKBdxiRXlfpvynQ2bcSO0F0JU5M+jVDtHB8Ki4WNEt2kO6vo2V0HmYSi2OacWLuCzSBzmig\\\\\\\/QC8BkYXb5IK3ecM9PINoJhGb0P\\\\\\\/4EFe7MyjsBVSzycKtmrCWswQ80KT4F6XSsbXf0Kfigv5hPJD5jTApEpE\\\\\\\/ZLBRQDbfK4n34QKX0Sv2S47YtKlTSn6fRIWNXQGtAjKFLL\\\\\\\/G5t9rsysJm79hS6Gm2MP257g5kTTfUFKxo6g7dzN2OnUxKtm6nBeij5D7HpfpDY5q+Ohr4JmZHtQsRWVhJETPO71jNN7lZqrJiEe7DFHSnuwo30wBPwb8ICdxoqGwzUEnex\\\\\\\/b5Z4OaZtGChlR3\\\\\\\/pmy2qnFJ0O9hW4OQUs9oghSdqbgIbEOF0ssx\\\\\\\/nWoyx6J347m7MwnLn\\\\\\\/YE2MWuQmx19\\\\\\\/TpCLgdB+jeNrGhi85RPLXltoMHRqHlnVVH+nbXSCptbJSQwRTmpGJkIwyybO1Y47jkdD3T5ar4zPMqwnlmKfoKaeh+K7afRHKJQzvgZYMm9VCb3h4K8ZcJpYWpMbNulyQRURLulAqOFh9IEOoGz1fpuUcs1C0j2Ggtzn+eA9eT285DkCCCohPma\\\\\\\/dW4RTzbJ1FF\\\\\\\/e27Dh0MooZRSawTQmvE5fj0Mi1UoGQAnMWzSJDKtaSenR8BLq+8My63sOtdVTAYaO7qtjjqO8hF0Pj0C5xaCj\\\\\\\/OADCaL55kmAOn9b2yWRuREy6tsMeu9fgmcCjnB\\\\\\\/AKujqr1YbtspSlR1VCoXCxmITHdnMXUG69T6KkuOhHnXawZTrhlTBmgf8cbCHDgh\\\\\\\/N+oWFKy3+06tXrX74BSYBCxqhhoS8k+2cP190Tipv0bWtLhmBp2uf3EFJwObJ+JTY9d5TWHJH27DQbos2BFYl60zhLkisue1pUCy3t+ePGmC25OeVY1C1D6RNNvyXt5V1DKKJ\\\\\\\/fqJyL3tYB2mVZmwCXnBBlrSZIHrAUSrMQbr4NZvASe5i0QdzQPNqB2HhHbkVG+AwL1JGy8vJW+RkJV8vxX0TKbkgrrqW+SCaNLe7csIJ8Wjawj9gK7IMtLk+gJmXdCerErLxtukbOgq+RRX\\\\\\\/BMHW5E4QbGimcZVcyvd+DsvbR\\\\\\\/RIDIM3wHi8isIMVK5Sf+6JZKCNlnUhA+DCysBbuR9oWZVj\\\\\\\/cZ3wpCCzxMnFprsyfdKrq8Xen2QC6j4APqP14+FmhKHRW9143l7+I5JIDwQHcseSPnCryU3tsj6HmAbiuvyBmvaRtmPTd0rIgZROVp3+DMwLGkz7bEeHmh9V0hYdz7PXcDtEYoO8u6XQ2xWzJTkKAvoOWXyqRhIVfAEQS\\\\\\\/J+5nVX0ReOYzD0UJ7EkiIrGw\\\\\\\/cOpp536uKVrfC43cC0561nkI7sR5pJoAKDbXDBBknIfCvC\\\\\\\/ERZHuCgbU+ySYriudC8G5yvobzL+GwKhdcl5WNNYO78w3Ksc+kLaaB\\\\\\\/o7iQVRNqCoVI+j7GZZaCHG23D9v4KCbLeGCq5rUylaaoVT23N6QkAJ\\\\\\\/yzeifv8T5EEfndKGLGv8s6qL1RwCWIhvnovw9aNYLcjfbdTtJM7bLlElUUPuRbHVrSSNjZB+yub15Gp\\\\\\\/RHonJk6y9JJTxFGZIqVCKQ7fJNeAK0rYkslLrB+aDQWL0Ail0h2k0mkbPNnT43vp4eikQAb6rnrUg0807SERdYtDsMHfN6HVtOXzR5ch\\\\\\\/Tmwk0eYhbnOmwO1LUa4N6RUWsJZ4eeoHOS7VXmQtGXeHYnIQhfopi+kZ22XYkyY6S4WLyes9zb2S4XQ9jQw9bqkSKqWJ3VyRpxbdEdYF2vIg7dps+Tb+9xfbLIS0NtTG0eDvSr3gDl1MxnGyP9ZWj\\\\\\\/tdwNOVslOiYgZFb+uN2C+1wCMaflVc87Ou+nN1OuNAfLthrQSIqnQ+d7it9wojhhIc5idLqZmpugaBn1kidv0GUHkn5ZAMl\\\\\\\/OoN7OJ5h7lrD4SDRAwJAJiFXspQ0KzBmZgjhn1QqC9PIiBIhy8Dfb0wvF\\\\\\\/f85VL4PL+FMM7lkLISEl+Bp6lv9xjfUQqi69fTOic7R3x+gA\\\\\\\/oOS\\\\\\\/VWnO\\\\\\\/RgItr8B34Uxxu7tFBgVwNlfyBqiZ\\\\\\\/T6ovtXQbT0eg4fWrZhDtT+u7055JsimpH76Nlf6nSDxowMuXo3pdEp+n3jyU\\\\\\\/2+bAc1MlDtCYIhBSIsSdSBlPKKaiomj3+yM72CIOOHpaIqmVy9Sn4+rABfV2Gd4R5xakNoVos32fH1oTlWbyuXjwsDQKMoxZDzDB\\\\\\\/gylUGrcp0iM\\\\\\\/lfelKQDChipTzpDoPcOQsRPpwOtFrBxKKMAHa+u7c5pSGKQOPFKdNZ\\\\\\\/CFcrOfYafRu9+hF7+ViHgVJC+VLyCs8sZg2\\\\\\\/aYbzYfOTMQU8zBlAwBBkBBeByYa2FHK6pIs2HPgfVHU\\\\\\\/Alwk47wB\\\\\\\/KrlKzwRIyMtJxW0KVUtONNb8BMqtWmgtU+AER+XqLAH0mcjvLMfMZT8q6Jqa3RfU\\\\\\\/\\\\\\\/DajaOd8NBEK8OO2kQSdi4rh5ijCbWZyZMA69LQuvsDhG2QJuVHeoW8+Ke5NNCwk32xcXnzPJryKdthyNOUpljVyn+l1W\\\\\\\/c6XxiH\\\\\\\/ZXjfy0hYLmfnrAhdE8bCFNo9MbkKisDM2Ac5WpL33XpY2i7L7F3Kuf7alxFsHMjfN34YYTrAP93qt0hMpzqzMwZF3hflecKfjg\\\\\\\/Rk9G\\\\\\\/jZmYEZ4K0K6FA7K7PX5FIhTpXGO5TfNtGgEyjNtjzp6YZLpzjQKtRiWrOU70d4G+TjZrR\\\\\\\/vGKAWDSMNn68xphqePrgTXu\\\\\\\/kKAePJ1k7iT+uSeOxAiQQbJwQfITnGfh1tNA9ccjPCWu\\\\\\\/2gJt\\\\\\\/lCWTLLTHaLpwe+3iXrMyTfwdQDxUlHst\\\\\\\/QaM7yD+VLSn0ZfHO5MKCinbAwC5m+9a\\\\\\\/M0movgTD9sYzNXXaISM2\\\\\\\/JAdUg0mAp6eMJGBQZWhF0Qzry6ZuJ6hEH\\\\\\\/dg5fcHVBsuoUcvht8HNWPF4TA12HEjm7GjxkW5KC77vyi0waDn\\\\\\\/FG2FdDgr9u\\\\\\\/BF0833AK0H9c6lk\\\\\\\/ernlfmKWqNniOCbvjfmzrq4oqmDwfCmQk5bDlB0IKgEMcDNJKDS9juTndYsrD9pqU4xdb5Hwo5DCiyfUFGrFSd\\\\\\\/Rs240zUvD8\\\\\\\/kV\\\\\\\/pWMcXga3cLLxHUOt9U7kZzeRAmMAuiViFYNdT1rXZF4lXl3deQVNuKZUfce4fquVjqF0fhb5Rd22i8ZtF60oHG27AHjjmLm3jyFqhsXsL4+\\\\\\\/y3OHNrBdEsFAqQv8cECvsMOjR3g5nBNJ27yGabLosjMAZrcTFYcunyJrbtt+YVs\\\\\\\/Mv1GZS3D02r7VtZ1hceKnEBtLSe50VXZJ+zJ8NINkqMtHS+t1kXfZMYQnSIWuRXEIedBmp3nBIqD2PdYYG9Sy55q7OA1ODFhQqw4axKIIDz0Nw+ZLKYMvuTGzNgBR1KuY7zDCQBW86CKk8qFqGOozZ9tTGiCSpIyfEplW48D++TSL83qrOzOoUSSX5jFFSZvpXpmuMYWQ0am5JUYp8wPFQlbIZLdWhxQks\\\\\\\/RM6vdJV5+DaRAfZrwLN0ryRtMGnyUYEwwv6tVxRG8EYzj28WOcp0KOKILM5OZpMAsXzKedmSLRJwbKOE744M2Ecz6Q0oxvTeZ8mnmXsX\\\\\\\/g9KhSjKfQT3stqiMDCal5yDwBYJqkm7z2hj3STBOfMkQJ6oasZVPPdNh6iAvtGg0ysTf6DjTkf3k3EF+PH858jjX2FwLrPczzivepyMBSDJitutjciL8d6o7FkVKwKnmdayO1mydP4WHbv3N86S6slBFJVIT5+kz1tgBvOnlIR+dQJkYMY\\\\\\\/yOLoVPZveWKa\\\\\\\/JJWa1X8beGOYRss9yJrra9apJ1A6U89lVN+S2Jp3nvbMeUiCwRA\\\\\\\/YliNViyl\\\\\\\/l7eI0gh4TyOfJBHWjJpl1zYtTUkC3WQVjADX8EqbA4B69CRns8HYt82cZZFv4+QMlfH9qPwc0kZnGQDe7cHEPA1+vM9nH79KgFcwAzw4QswEFm5spdrCDLqDLtaqHSzZFX+KpSyMonealeLvahso4NcqQxRkEX0A4njNI5tpYKucjJXzivkivUNHUm12q\\\\\\\/Z+K5jJmHzKeChMbblOlSmYncre+m78rM2IL6wOb\\\\\\\/5quLuETb7mKUIMCp8VKzcjNB6x1N62HkCDBWG+QcVBEyzlv3yKxdd6p0KYLAMoxkgJ\\\\\\\/XkzRZ9eZaPYavJEWioXNL2UQSXrWBGoxHPh5xJOuNeVEVNvp9QWvNuKbjyZECuyQpjFJApFedYLAfy9+p0QPOEG7Ec7M+5Q6lzXbseG9XEEpyD0tWDqvbb9waOVnDuYxdm4RB4duypyKXdEkrXrYH5hOVYafatoxm0hKPBP4quXKH41SZkPGvAg\\\\\\\/G9PnGPnqsDSwrAngvCXjsIX9KVBkP1WyVgq5gL69X4xGlF+P8OdCpckS1RY6jliHFbOEo\\\\\\\/Kq9IlJXTUMJlGwrCkDU0UCYQbTkvdGrItWK268ds71IXCgF+KI8LsuSKqIuRfHKuyFadWvLL3UorTDV\\\\\\\/g\\\\\\\/IkFCiszIql0aym6YDtqNyZ8JA71vw33QM0WJLgkYLzpd\\\\\\\/Slv01Y1fA+dDP2L9BCgqAkUlt21P0dqhNK1KivVgIgTZmJzyVXUs06kFRhdHs3tsxGImtKtgCQmv4MFgDdvrL6b\\\\\\\/LNsLOgWOyyIlja4Jt1eueW3PUIzv\\\\\\\/xGMor0T9oOsddxiVkHZ0t1YFnrJ4u5TalGy6GyaZaFtcs9zVXq2QlZ20Sje+oB7az4RXfetRbd8gkHrKjRY1S4+S3Ddw1Mk2o17o76ltkKI6Qg68P+havV\\\\\\\/MYum\\\\\\\/jgBjuG8eNj9b\\\\\\\/zs5lpjmyfdHK67bmrqouig4hoezKlD7CEkHi9IQAk+ROQRo301iLSDd0nzcM58sb2QKTmQJg7T4kDr9NxeaCBdyyWNJZqpxM1phYKCa7uNWK3mQ6FyfV60oCdFjXVYJmRmQyipgKaW+KABz+NXvY4xCl6kQWY\\\\\\\/CchYtNSYUFeJ03dYOSepQJ6g6f1hw4jmaOywr2nSXmmBei\\\\\\\/8i3GIuE28b\\\\\\\/VuJbLTfxLnjNl9uPVthtoBWAfnL1\\\\\\\/e+KYBhQCJFUqAjaUcbkeJIGfl5C0j40HRMlG+5psZpXLUxQArWQh9vZ6Wp5OG1dXQ14hBPe7bsjCjz7lvvbD8y0hSH7B37fZ85g2TDSi4KoQPOPATCw0+545rmc459L\\\\\\\/mp4Wuy2\\\\\\\/IAZCPwDZoBdNicfXgEdCMAaURJTxhgJ9GEORHKCkgfSenq0REkBJNVpYbW9Je7TYo7yZkSmtszGJIe5UUoXATWiC2cN\\\\\\\/3XC23tmmvKqBPuRKPCkKI1t++W9RS5Zgkoo7V\\\\\\\/fIoaAcglmboqyZEL5WJ630MAP7FB8ujgdVA0gUGvzBGaHFvbJNyNPgLavg8Lb4g2dRgHpsqrvjapYPS1LYl572ebpC5pjjMODiYepYanVNinr2usY48BgKlHY+pdg2Hkk6ikozndX0g2QUG1TNusEKJb8mP\\\\\\\/QnDvAe4Jsg+5x5jE1pZzl0VtvOMP1mbzQSTjhByeyXHtV+SKoxSzQWV+o6kXJ1gMw9bWBBcQVIfUja1+E\\\\\\\/JNg7qZES\\\\\\\/1bo3uhDMSyZVsOk1iRAYgwG4RPmWaIiYJRSJl9ZTIflRSERSta2sJ\\\\\\\/9+QrSR0XNsewgtRBNxjlJErRZOSeQg\\\\\\\/76Es+yaH2\\\\\\\/oFRaShlpSBkDD2P0V2uvrpkdVBqIVGR6vQ4bhgzS5doQ0eLSluqMnQEmhMoSdK4weHyMFlXlVsz8r5XiDCwqhyrxVj5a\\\\\\\/Lh0R1YPHk0AQoEqS+cjbyZ+dV6BOUSAz4QIHi8EiARuUEoMF5zStHEYoqtA8LB9UYJgotnUfedb4iIc4b1bD3Agvu6XsFeiJr1Eo194MTaxrmbOlk\\\\\\\/AA2jKhFzCyCB9zdsTDQoXSh+WhZD8ugvNalNdEKlWaVbbKqGxcjNwwUz297Kw1ADRdhxJzm+V6jEa5+no1RvqMo4VD2Qwqk2sG4yLj0UtdzxcOXNbtABUFQVhNf7KGMEoGfjejs\\\\\\\/l9s4er5P0f2gbhxd9Ouc1T58fSKUlKdEOyxjbsevApogp1byNczE92R+QIbSQBUm+q\\\\\\\/oEE4Uy42UrpkvBbH480TpLce9mJ3ggNj10LUft3RSaSGwRPEgZGps9DOtvl9LeAIE9wIp1uE5bseRG0FEVcoyGXwWJTCy++2G8jckTPhbVaKRUkoIuv9hjplRiZQh6vqjH3xkh6Fi26Mgrse2o0ypwILWGoXVLQfRVYNmLShYWfY+CktXd7UozjRwTtSYy95qv85F4oGoxIMYVIrYante8MjsWj7kx7LK7XswTET0n2A5JlefWTtH7C14zjiy25BgjdV3x5qItuQHUXaykpLrr8rfqqgwrrkEWfXHiG\\\\\\\/JivZAbGvQCEC+beKY6OJTcVH3e8zoudn4oEc0FzHc6cLNauZyiMgzQgxe3M8r6tNIua0rAv5yaK8FagOdJs9cgkqdeiNqCDzsGr\\\\\\\/LnwMuj+Y4MjSM+Q2K6fBZhZKOYvQSdgGt4R43aFz+2\\\\\\\/nBVUa0fdPZ7M7mEHAuuuCJitY8lZgfBjTZoW+1avYnBIBty7dH4RZfByW2z9R0TfGAagqfKB6HwtvF4s7u79PLfXi3FrlfX3JjJN1p1xdW\\\\\\\/yUTGZW6MA4RnLlfFgCDjJ9hdw8tS\\\\\\\/GGvZO2wbYrphmXPSw78wh14g5CtySOcS3QkUGGtgU28+TBUnNKw7u08E5nu+fqIY7C4PHT4DYrkcMA96ZnPBU7IMWYjlcE1+usljnU4cJ+61hHaVgEyiBFeF988vBwPdw62nCZsc0bnjolsxxRYexhTf77E0R6c3MdeLY394atHYyrxSbTBTcUjMYN+5rVbfrSGBNMa1mB+nSr\\\\\\\/4uwUFQp1Z4gUc62bRggUashqI40lCsFE66uhNk0g+OtC7KlNc8szD7DuX+NZuSMiTMZuL7K7SuOC4LiPMRw1R3vBOM9M\\\\\\\/lpV8Y2naERw5Rnw95iVnd1\\\\\\\/z77ChIYgWgUDO4U9DyaaP\\\\\\\/QfOLKufUCAzYT3L7lqFwe8uLiS\\\\\\\/uaVwTTYDMwlakoaAar15SIXhOcue2cz9SS7EjJkWSk4fmzdrsh7MoYneauteUBOH\\\\\\\/PhwaLdboNcNK4oKQwBdIaBWkYDLrn\\\\\\\/DMezHLIuG7eLrFfXEgETgfm9V3Fp4MwEaIcDTcWGrlfq2UPpMR3+wt7XGAQUe0Sw4WJcxFRcg8iFwYZvINNMhG\\\\\\\/ds8cma7bdUYQJPi0wb+EIceKzJBEhMibCuFegDx3Y\\\\\\\/8wCEzYU71aqp5O4hJe\\\\\\\/jt\\\\\\\/ABIPvXARkr13fgKEVKiX8lfm8sgs4Pwk3dengiR1JglTRK8lE7SBgSvplmDjDi2mIIAPPYAZFy+pLkMSlVXAC9nmr1VBvuA8SbAZsgaRZGC0TsfRK1VJJmEnmpqD0QPfxu4Nz6b8+OYNAGYmDFduF\\\\\\\/z82D\\\\\\\/pTHxh8bJmy6J60So1LS+vx8QPy3C4fIQ1dRq\\\\\\\/wpl4qoB8V1JaELQTWlgBJe9UVUz+0GliYZAzRFC\\\\\\\/BXDoVqhAnvWbE11lbQ7eX7h+aTlNprlSyDpgIRj2L8ZF7ZhG4vFyVnPpUaL+wnmgLicUEci31ATChYNHCyAU3mpJ1eEni78wcOyfKhVUYfqZdH1AcDA46PVFYdh76h20Hgtkg4nVZ2J166IKW1twoFEabB48fUSTQXV1R7L4RyEbIVscT7SYRZCp7xFBo2H5eLTu5fmCRP1jAzhk5Nh3TIdG7LIwoUpezGeuu4QywjzMUB2H03z1EAmIwth9EY2iUxUM8J8RaYYbzrzaennDC2KuNNWYE8wU\\\\\\\/qwyVfugLQ8BJpnb\\\\\\\/dhWtcN2YYSycNAIfVqAbpIIv0ulJSGQvEyUrzLxv9mD1aIBXfLgJtQMEmqmbqRyny5eA2hh6qlRToLfVGbNVauC8iYHllc8YI9ro9VoRh2+JBHbTIld2jgYHcQxf1plQWy\\\\\\\/JKaf4zIg5KfF7LIgH5Sh\\\\\\\/Fyl8RUpN3eQ1ybNPjH+xJ8X9aTieOkBTd0AIgxnH\\\\\\\/xO\\\\\\\/Q6BG52QFS77HymlTzo2MMSYY84\\\\\\\/24Ee9MylS4v+5cXv4\\\\\\\/Ar1gnYf1p8pwTbJzMoIzY7f8Qgf1fJjJx3RUjYsDzUDN3kJhNcqHXHf\\\\\\\/xouB49Nm5cZHyxfGjEDxxiDJprXdL67qpyUFzH6Ln+zHlLzgn7ArG98oCpW9DQpJrsQ0q65l1WMC8aAL+8VUXYRVAp3oc\\\\\\\/d6Kf1qievdUdmiLD2gHPimIm+XYef+KuvWJYCLuHxzdXSHuoG9HW\\\\\\\/I\\\\\\\/Lm7c69J1CD7POCG6rXVDw8QlcG3f\\\\\\\/pZ61fzVqj\\\\\\\/Z18xOT9jva2j07en8qtAbdfVYi8UXDBycN\\\\\\\/PqKsZt4gCpg58+r24GJequ1CEd\\\\\\\/Fx4FvMOCq8hIR4mvjs2uWvyJcyr9hv7RzojC4VA41VhWovGBUNCBimv5Hi1eI6N7NAFWjP7rfIMz\\\\\\\/cd5WdXepCpeowKtwrrzuayp5HXIz9Vhu8uSo2l42bqQ1CImi7p\\\\\\\/Wz+8gA8ykC90tedwJjVgEVBShdoerHPy7F79c+IatZO91\\\\\\\/KVZOUg1TXlbOZ4xcNrRuY0+XELxOvr0qMaV1cBXrcUcqtgL4uERy8m3DJ+gKCv7kJTSXULjcbNWzDfgRZvkmc9EffbjIHf+97D3NhkpWDhTILhip5NImpZ0zcA886+YOzT0Gc2nbI7EDouI+bNag+SNa7EmqvYWmRSEYmy35FXOsl34pCDuxKqh8kb\\\\\\\/0h+T\\\\\\\/aN7Nq6JdyGdjwEzVqXpCqI6E9ZyVBGXZn04i2Lwtnb4J+EbbgWJxgKlSCE5r7Jauc3UvM77BvU8h5d1DNWag2IXJkVJJNvqWbW3Q3v4iO4ImkJhZq9DDnKT\\\\\\\/97XVpx9WiZkKfK8\\\\\\\/bDhjbBmBS62uO6gD1DF6jyvV56Nub5VA+fODqagSGmQxTJHbYp5Bm1JhsaN5QCmclMOc6OwC55plZMnuVOymyl\\\\\\\/caLRmbbwaE6oh55rw99T+AbBkBBZgIZn6vO\\\\\\\/D3fVyCzA1ZQ8blhKNI53kvir3q5yH54LtrnFqG79mkT7kGUVv73tawXK5AkNbJRBYzKOO4DbkdhsRXQycQaaf+gBLR2svgIyuZTOw24yr9ti4b1AGMvKW27mXDdAvkEbp6rkzlKKsjGT403ua9I9qf3cbJvjaBCF2mcDnPXbyIlnQ2RhgyzLyiwe7Zgb1cQ8BrgaUYwU0a7HA3yaUCliTBZEQ6iv9NYOjOkWKq+lG+jQyu\\\\\\\/Q2Y4eUAaSzlafeQENlOzTTZaYA6xH783\\\\\\\/3NRPMSnk2N9OqDnHp9hmQdTEllOQLvh+WpRB5aTvJLDCJy74HR05\\\\\\\/SKBMTqNV2yh6QgtWYbFmCZIWF3sIIlrGWR0vP+mu+NRsY0YydgLkHYbnRiFk2\\\\\\\/Tyu\\\\\\\/m6wJr8ny1bQeOTKmzt4jZJt1iiCohuBp9VzSIS1VhlDJ3o8UF+OU6FzB3\\\\\\\/L5FSHCZjh2s+85GjHU2GvGPQXqz1562kHKqLuyBqUbicjikZT0JwsjBwwxwrYLZrWiXwXpF3vmc0GjR1nR0H4NAAsUe7uizDS4nuzk\\\\\\\/J+6Bz6t0ZAqv5iAZoeiKAiubLo3esp5I9QoOcOpaVb9FUf6EhQSYZvq6eB73q8HQ3rmmETEy63kdha6iLaas1OjNf0Sx+JulANl9kba95IxFqDXxpXvkZArWnPHqvPSz3mvynq+sNAOfFESdgcUqUTqESsjqu4Kl1Cho8Ob+maLZVqqcB76Cj84YP6VXCShUhmMvok3gvAyhBh7DOFoeWps\\\\\\\/ZMO9QjdORUzvzQnQ3NOJYlqWJd1av2lB6OctOivvBUqYcTl4I8cgFtiYZNwdhDGLsWtxEuqRBUl7fAIkUWw1qrPZlMJghsMEhfp8xz7Jy0SkN3WPZ9gJrvykaPJxV2UDB1zSrpQZfkYTePFKnIRv6UfL651lMfyoSpmpWuhaDvfUw4ukHvch2\\\\\\\/\\\\\\\/bU\\\\\\\/+aakGDaxDzNhR0vvUujggWvbwiZiRFuirSWgsVLTETev2gArzu+yJIFbkD4W0vZatkA9evctNk5jVjWxl3d0rBDTb3tXmOqTI71DQgU7i42dGMSfJSgdzDvvIEDYiqA3H56SSD21OAP+zkaM12p\\\\\\\/vc\\\\\\\/zTpUE4sRBaeQ3AVqMAcBQLs5K95O8KFj0RIG2vGLrF7oW1GdyJMDeEvRF3SsJOHzd5hYEqWT5ntaT2xtqggoH3zGtnaBbmhHiOybin\\\\\\\/JF\\\\\\\/ucGtxC75603X\\\\\\\/t1JPzosIs3piSjFK6+wAd7UTEvCEa1u7aXITtatnXsAfMdBON5BseY+eDxpvB0YAeE8rrgInSN1wMEsXb2H4Ab7B29kNBNV10vrJ4LmPHCFFoUqC+di9XWNnnkhuxFdMZnZTSdg3MvS3kGwi2Nz0LVayCxYll8f+duaGVLdxwCie8XmioD3sYL4QLirSGM9wGmzjHYUAsJd5VpTLU3\\\\\\\/p3Y1daqK4iwy8GZdnlwY5RTmgu\\\\\\\/rcn6un0lu3gy9Lif5x8LiXKFqwOxkRxB+IjSdUFe61eJ8TLQ5h7XwVsX3wFSJU9AlwNz6a4gadM+1xG4ftuG2Xdtt8mX5CWRRyMhfc1khPlhpFm4MjrdwMq2z4aGFUguR6GLR0yKxQuRxdSKiMVf86HtWGQ8VGd64X+jQ\\\\\\\/\\\\\\\/B5bKuuIgxlzpeAwwfx3eiVsswFkBEupYVtNkFiJSwRTHtpKrFwgm46P3tpSc5XzqiXj6REe\\\\\\\/CWfijNcQBYY1k71UE2lKQBFBClWgf3i4SMr3S0cudTK5UI5W50zVaT7KwNygw0SRPXdFeyvsSB14KGTMmKYYQX1Jje6fPeAGN8fYSkIzpu1SuOKblMmvrHGO\\\\\\\/mDLSbP8peiAg+dr8acZkF5ljz6qXIiVz6iKKT5dsqOILUy\\\\\\\/3wntFfvbyRzXbIC9gHMmj3iLTvq4oMWh6mJ5z4jLLC5xMaI+DbfqJJ1mAHsN8NUNw6n+uuf9s9gwnGx7s8fJED4cQ1nRCTHv3SnCB+X44q\\\\\\\/ESuD68uwI1qV3He6fbecvrjFXPeau+kGtuNpGsJczQsdEaCMy0CDpu6VSyOd7uxxHxFKQSZk\\\\\\\/jjAd7H9Ja9SyAD1BShd01mx3fvpxtQSgmE0rsVw8EMKKOnr7MwTg2QW0HhMukyUqyo+kWOGlMXpRPfc+Ds48vdJgH0ncHa22B58QsC0PF7KjdoPOm8lohdEFw81uIB+MxsfzvXo0mJvtbJGtlAVnGTT+r+gXLdvCWSbu+bxX8wtZQ3bAyh0dmKnXvJ5gQxfoD1nMnoBg1BO7Edodwj5usdPnvgIXR8yupeeSKP+sicbcqVh7Og95k5cE2EDiuzRkc2xMjHht49bhKChf1sJXn4xFHub4TwzDWNSXspCruXCHS7R8qTTOYKEH9PYXWp3lUs1Qy9R4zUgwrNvPw9QlscQSctmoA+pYBuI\\\\\\\/pJU8SZ1wMaRVSP77kl5kKMSv4UJW3BdGUs1tIa5o5E4NXgraehVPiid9q++HddISz0VqN83Qx4CUbKTEGNMcI3+8ZRW0S7Q\\\\\\\/kfiPSvzedv9azBpxWAAdBzK7kKUvHSCKplan9WXxlVV2BON83r0NSt4InNrKpx9NxwiKxH0OJrz1Y7BtNtOCKRvE4R\\\\\\\/7izLsLgZy32EBV00kNIv1iZ4dqSAfzWSFPeg8pjIqmWvgF3Y61kv1DKT3CG8xFgWjgCSjx821N1GeRfpY+LAC5WVxnvHmC65G7FbatT1Eu3ZofB4nOf2xn6YVJ5HMilpY\\\\\\\/RGAFQT4La77VE3lLyl9dUuv89qV\\\\\\\/7OWHzX3U+Ru1xF6CCW1qKWwq8PbMl0na3QIGdLlJIfpsxiH4cEO+Q7qERwHQdq+yduV\\\\\\\/ddESsJdAnBcBnx+XU3nVtJ0eCnurDV3TysvXawiHET39P6snRczolIEuF9k4sTvK3\\\\\\\/AaACDC\\\\\\\/kRQZmfFWzFRJvwWYLtMX+zhPip+tM\\\\\\\/XGQekFT6n6EYoJhdRYuIWeF12i1sGrlHe6SvufUYvHB+u5fYUSWbg0C5KwZ1JJLipF7KSLpaibqjv9Z9y7uu9+Ga\\\\\\\/mTf6yUZSDuYgfw6JcPfE9IBzmQSJ3LJEJmQ5CVZ7o+OHYsDbsCzvnKs331VPW\\\\\\\/4puUtuKu8xqVW4eiG5M3YSODCMQFNIHXmhve4iLM8s\\\\\\\/Z9W2ZoG4Sl\\\\\\\/+ckr9csVvElrTkYIM4StQ1WnSr\\\\\\\/jlMLbGJPb0MPxfgQywBUpOxGLPuU5tDugUAEfSGJ4fynab2dLz\\\\\\\/uKH4GFyX1AE1PJvp5XhFYBTHhJERW2n7Iwsz9pUcSqnSm2TEcpnlkHkm8SN2bqeGhn9Im\\\\\\\/\\\\\\\/XUvY+sOtTD+AOOobxwqI5PW2ZsSNNy6xh491Q5y0vK0fRwIrWVgTVe3Hy8hnrRmMsqEAMYyz4uJt4JiitMdcY8\\\\\\\/0LZX7\\\\\\\/alDYMJKXtpPwC05i17TrFH1AeR8+OnJDiO4NbnrEN9V6VLJFHPYag3YQeIs15eLE6L2egRxlR6YCsniO007cm7VxTzYGMgD1Osw0ogPwNejBvq1NZarS3hjjuSvEqGlYwt1h8fP\\\\\\\/Anij2b7qm7kfUqC42KMHaAP7tZzapq2b\\\\\\\/KoHsYbIpCtNlgh6Ygi9702qV288q\\\\\\\/qzIcrGYcwEpZwhfCLLYzBIaVMPmhegorxhlWIzPYNQ1FMbm+KE3gReuRpfjmQAk\\\\\\\/w2c11VR6cQmmf5X5kHGqyN0jIP0MzdCLJ5wbxsFnu\\\\\\\/sFkhkcVQXa1ZACx2sE4WMD0bNrefhfzyYULTt9EuBY07tAPK3CCjWMBGRkfSRNQA1v02KfgquMKNN20QQMHVbHyCENuIURy6JhgHs1iNzYY2LjC6ai6pnR24IJOsP2CYTgBIZmP1UXo2\\\\\\\/wHT3JvB7EEE+4sGgVATPb+aGfwTGPeHLx9LiOPXJRUp8In5m0lc\\\\\\\/Uuam77B5QzYvBPCtSlehUqES4ltBcQVBEHe5CF1zoQdkYBcz3\\\\\\\/6n5JblsyinQMOrAFKpnx0a1wI5US5Sh6gn2azHCeFMqVslo8CAZtDXtvoaLOuBFFJ2qU5NfjN6akUMrZUT2wCdWAd+rUSC26unufaKYoqaXCJhJm5MqjmxplsAJXpYRM8MQ5LQ4Q0HtkB3sH21hebeY7yZMKh5hFRY7yPAO4UA11l8uLFhDMh\\\\\\\/lZ99HXoo79sxgJNU5ZvqpmBYj3W2bMzdcijiRGjUxBlhiY6HS31UTnDPouEX321tEy+qlsW5isBO4yIPLm4iHrOU3SBABSXt0oVIx9s+CkpbAX6kCnNYcTjBvMk2Fw0qsIcz\\\\\\\/CjPC\\\\\\\/b1V3h9r\\\\\\\/TyVNuV62xZgQEbZnyXJQKZGE1p0o6xuyKnI8qa5srDXZZXm4QdXZ+RjXg8ufmyguhvKpKcxLqeZOd6tJZ5itigkcZDDiS1ukOoud9hfVGLw2uy8pglDvQ2xab1BuAOlkQF6kheCGh5H7dFeapq63T7BRtlRUZXe+OUZkhw3YVmZg7\\\\\\\/MYPXMAnTQg7pLFOdtl31D1HMcqKjb\\\\\\\/jbzuShKg1szDCv2a7dDv+dRhvYN9kD1fG5\\\\\\\/wPq3SbUxZvWweES6P0T5GBHsvZNHfAEWpX+sp9W3aHmLQkRTDS9U6687F5RkmZn9zJ4tFR84QntVDSQy3JS8+enDNsRR31DVSPvCUSiWdFByQN64rMeLD3dRSZ1c6ATZYiwQ6VIfR86ZKIa+uUlyWj6BdGa4TdoCVtTEjIrlCkeEp1D3NEZTdTzhKE8KXnNmTtbEZEJNWbQYxOGHcJZr+K+HCDB1DgVcj7wAvsklEWZTdzYc9lohNu7\\\\\\\/nvMyUU0MXR3wg7V1Z1FkoxqufIXxaAxTWmHaSpVZRw3YGWRQCu5liXoaXPNeTDoaszjtcqfqMod2oZ72JmsJQxywGImtnf6SLNZKRT34dHEq6zwgXpAwZXTACh0SAv7o8+3hhUUMRhOljCC5bO94HZotNxsC8cJkuTg8WGXgQBVIi3kfi2YzFal4Pgbuq0wuYD6SB9DOCkto9RuwSPTfAlkt+ltM6ho0S\\\\\\\/JyFqPP\\\\\\\/x8uXmqvthzajdSJykKlxH4hTLCVL\\\\\\\/Nhdqoz0H2VUDMk3oNyXjgvzH1U1J5l41aWZJhH8VxZKpMkdzlYJWQ9VjlculYhMwb\\\\\\\/CQj3sQWgT3ouMp55+yIhTs\\\\\\\/W9EGPofxjcmBCZ4a8qHXB\\\\\\\/DUrhNsbpYx7KNLLIh+VSvHNxQJcDUFT0QOHJsgBEXiIibay9zqJSYE0csYyoP5gH5Hl0UMqW\\\\\\\/Eaxn535yFpz+6o7jQVyMseirHkhyaduUav0h1lZrGS8fJY7NIxgASNquyTOWB4\\\\\\\/FX6wed\\\\\\\/yYZb9ulTc40x1VlvPqHuP1cbp\\\\\\\/iXhuCxslghYpkivCKfn0dVqWpnnCWL3cHPMQw0RvKzPcfivBCj59AzQbfxm88YExsQt2M\\\\\\\/nKtol5xej12uiXDqeUKkMVOn7LePtAXnkqkhUaOL\\\\\\\/oaGW2xtO98fVu0ilR5M82lOwcDet5X60ZM1GO5q94i1a139XTP+D4EBZYyG64bJfjuvH22nXz1I4i5eNeNTbh3JaETJihxqsVMlgS8dxyEI4LWylZmxHJ26NbFL56lxyf7hlpqjzMPYl2xGfl10HzuJ6o+MiS49JdkmSN9WmDZOwZJ59KOI4+YHiIZAOf3002g\\\\\\\/VCTtiV2kA+rHjP19qN5Uh0rClYTBDWYeBOC0EsYem\\\\\\\/LFXWDMa5lQ02uEum6endeEYQq6uZJJh7hFd5KOXDpCbeoyRwqRDFm+H4e6IZO1sHxyoCe+ZuE+T2YX1l0O4RBWdkslHktz3Ep1nhlfaFRkPPLQhXkDUlJcMEvcFhoLJBGfH2Lk5mcNg+4kJ1FfwCxADfGSNzRpLuQI9BKl6c7LvJRe3\\\\\\\/hphLMEsrYRkYdCjh0xXzWffsYs9tfXp+iObvbvJieKHa1eyTZq+5Dj\\\\\\\/5ERsz5mzTB5\\\\\\\/R2NKJz9rNo8FfKt542D9FDJ7nokWixmIUxDw\\\\\\\/uKKDdPZy\\\\\\\/J2hkrItmnVQV1Uh9lbBONuv5dQESyfErRSkZgUkwOANHzaCF+6al6tP+YnQsc3DaNZ8XRHp5a6F4KeWodN2xazO8Bsi2MYPpBkUjE9Epkq1jCAnz7U8b2sBqZwxla7tHFpGrIADlqXkTkEhAUSi4bnCVQ1WHgSGhiy1fy+xITKQtoxCFYU1+7hjNC7SHgBXRJ\\\\\\\/9w1s7lM1w6eVHSJWmBoIoOGfj12s0EMCtaRPI+hCtCesYO5FE+y1HfgmJQk0wCm8hL89b423mNY1EKn3evWrolJaB9xv9kbCahMKFT8TaciuzidVfk0ouILvr6SzwcLIlRcBemPEPphGuzDBPcA1dXt5YBdmU\\\\\\\/xQGauFiySAy1+MgxBvI9tqJegZ8Wuz7SZGHhck77JHQYQNsVcVqsdXmk4roBgIUp4rls8E3S2pkSlclgFKMOqXuMX0vFw9odg6tbaMuVYv+rtOVkb6ZsLhyWdyHczt+iRsXUAw+2EhPxgCm0ofRgg8wtuXwRxXUMLpbq0f9iRZHEG\\\\\\\/VTUDEy1sHhLIGLcgjhH3q+XN0ZurC8S5+wgfHqOVF+34NbBRkGYACc0fsTu1aWwIL9r1J4p5kgEiiJvrBmcf8MnkQR8wbeKrvuZJRb17gAlTNLDKlkab652edzBLbevuIvdGQoG6gW+GE9w2r47z3f9IkgULW2OcMGP\\\\\\\/ghkuPWWv\\\\\\\/nrd1S1C6roUQbSTP9KgmQQwS0BZ2K4WQzInS9q2fsxFc0zY3oiYZ+Bk0sEgbbxwGl71LhSFZt6K6UV3vqhVzAq9XRx+6CbV2V+5xP+RvSjN0ca+T4m2BQUNU\\\\\\\/xGxi39jU+8RpoQwV9UlxGXKTz6CbcAxytNNUDF8wrsYbknRcpaOqF0xabkeIJbTz9F2y6GfNb9TMhZ231plwLoPyc\\\\\\\/8O9yCGqpksUpQN3nI0rpKLvSFdKRWpW\\\\\\\/IJTLWO19CxPZVmr6AKhjodfq4\\\\\\\/rLzgwN0C67z1idl8B82p8QRt4yTyClufwFMUkmutpR\\\\\\\/2+ZOOvkdQVtvH9g611b7uyYqaN6tjng8AKU\\\\\\\/YNgJkI0\\\\\\\/GcK4H\\\\\\\/P4NuP4WWhI0mtTMTV+JLdHJHKBKNavCjVAHfB\\\\\\\/vK\\\u0022,\\\u0022iv\\\u0022:\\\u0022+d0woBaWZqJdfky9guh+OQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00222d19d38f182ee08e6dc38d8163bc5c351101c59800fcc842642be4b2cf75416f\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022IRYUioEPetBdS1bmOzvWmEYY7pa703mlAge0J7\\\\\\\/Vf7tD3ZvcImrieoL\\\\\\\/nVcVW9lrgfT7AjbfkHszKg7uETfcg9wYTtZ\\\\\\\/9qxcg0JbK5fl0uRfXSIggZaQiWo8iTrdfbfcPPqg8eNaM8uJVF9KMGy0uF75bfC+RpgzjYhexTc78b4W7Qd91TJQD7ra04\\\\\\\/pZlCczRr+i6lB3g27k5rPr2+K7DgDokjDmMTnFnU1hUXADuWZ9Os\\\\\\\/8g2Sk93gElBbmkkjLk0WPnfGpqBH7fXupC286nzbs3BPvRoLYTjHGjoAIC2nbnENS9m5BHnLqaJGTz\\\\\\\/pQ6UNb5SCAHblRKOdzuYljztXXOESTggyQ6w0mgzOp0pc6IwtEc+PHImVkg0PK32P4YoMKya0jeLCFovaOTbtZs\\\\\\\/UyckSEgEGfYQP6pb6ku14mW3fG88K8c+v1qobbhUpNlqxxXow6c3DTd6Ti2eR2XtLg5yhxwsy55zJgdbIFgaDw\\\\\\\/XfdMQpHeDjlOu\\\\\\\/XIL2g7EJPhd5xL3kzHnIYCBLgwp7mgUCLFa1RlKRxYINDekv\\\\\\\/aKlBvZnTuWOJhg7mMmV3CdHlO8745WZRVlsPGQpQtyyCGc5AdicPM1R+9AOWWYV+i7KuELcZy89JT0SUtfuHYUDxCCP+CKwxNrOsHyUcqgWRlNc6ALR9cCNYklVEF3RJmdJqUfSfw28v6SN9gT1Uo0LEPflYWK+kQyq9GJtj6Wqodf9NRo1TWJ\\\\\\\/J604xhCnmolnKuoPlPqQnuO0IWiFwsWAPG+MJXU\\\\\\\/whJBV3Eca4UFs\\\\\\\/4aZ0zPYAu5PFlmx415wVqcYgYA4cHUbVzk5eEL\\\\\\\/mRAX\\\\\\\/QrydRyO2TgjhEVuCRrk66FsNopA6cQwxvhFqYBO0MbboylE8BPrfY49q2FbUeuy4CjBnhcEiBoENjDyDVgDCvv331e+JgMQrXNYuL\\\\\\\/ErWVF3gzyzgB6jNY6UymCBozV\\\\\\\/eLUxLQWMPloI0g2bP+7Nb29HQUe3vWZ7SUhuMQEnPzD5XefvyLQ7wvm7rLtaAdk6uO20p7UeuucLnH1byRptWqm1hn6KW\\\\\\\/Dr4l7G40X8I8lLole8kU9dK9dhA\\\\\\\/cGj\\\\\\\/Gmj3lrXZPM+AhxkWayXAFVCC7d3RSJysSCV1SGOOMuwTq1OGATCF+md9dshxe+vgf3sq198j0GL+XbM11CVR9R+FJejZWwNo3vAYDMTqh6p0IR7q0497UKBsc5t4EV8Yl3lR8n7166jXxN\\\\\\\/b7jZmb8bUQrUJX7StTU3go6ixmZxwWt9eO4fmiNOnqzbulIoAXgvH8HgK2h2GzJ2yzU+vbZBxSqePNZb0FcWck6CFHEGJuU0klvWTLWq28IFi60XXMdYT63OiUWUzRowzSz\\\\\\\/4eJsM1KSBN+Vny2bLpFyorQOwhS5KvNqz2J4\\\\\\\/PjezHXy3OtoBsv0yGeLIx4\\\\\\\/3c8566z9nk2x2bupGQwBPP6HN+bQJ8HORDW0lNaf3RqbrDflfPV8iazCeN1niUVJWCJIjX4K6bF7LeVg8x2Or2Ha\\\\\\\/3ydg0ehYdQhjAMmfwUgfNTFqnFnuv8qxqyZaYy2vJJ8XqK6fw58aq8ovH9L0qJ8AqyEAn1VJZYhg6\\\\\\\/CKqamjRtPRHGiSMY1uMjvwpisIqbg5z4glYY7JGgMwufJIKOmsbWkk\\\\\\\/dG5H\\\\\\\/vjhAE7\\\\\\\/Jw58wj91JAL8ArQMYlWqbOtp1Dm4OoNLRjYWp4JQE0n4+yR2tJXqWxq4RmEpI7430kgVL9nNZ+iOy7m4LZrKgUycKjNp3nZui38VwX9NBNBctaBTajk0xePTLUTnyyz6e7hkrgQplZH8J0o+nzwTkifduJmbz1fk9PhzQfDIdoBroN89iXADmfbjnZn7cftGjVdS3xFv\\\\\\\/V6bDva9zJ8FSvd+EhknWelX9K+mKL86YLEV1avI2BqYWhwS4d+zNakdl6GHFZC\\\\\\\/GYzW2qd8ZjAZThAMkJNg6kuZ+lVjo7rzCBPX8oT7kqQOzEAoG04GvyrN2nw+4QuQE4aTPupcxQJRXdfg4Aw2UgqEbsTm7oi1NZ+SPE=\\\u0022,\\\u0022iv\\\u0022:\\\u0022Z1bIqtaCGQ7Kvv7HmBlH4g==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022203186e7aeb1847601206e67b5b373c7c2f123669fb3ce74cf17037dc0743f11\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-486371 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/9/3229" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-jnXHZ2fnH0TA-kugvuXS7kQ9xUjHcnQgkMNqtRUKd9Q" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/9/3229" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-W5JkfStV-d6PMxeT0AURAJtiiu0o6bnS9avmiSj_B6s" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/9/3217" title="Trans-ancestry Fine Mapping and Molecular Assays Identify Regulatory Variants at the ANGPTL8 HDL-C GWAS Locus" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/10/3237" title="Meeting Report on Experimental Approaches to Evolution and Ecology Using Yeast and Other Model Systems" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="486371" id="node486371" data-pisa="ggg;7/9/3229" data-pisa-master="ggg;g3.117.300103" data-apath="/ggg/7/9/3229.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Accelerating Wright–Fisher Forward Simulations on the Graphics Processing Unit</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-6782-9986" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>David S. Lawrie</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3229-3236;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300103 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node486371" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">David S. Lawrie </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Los Angeles, California 90034</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BS.%2BLawrie%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lawrie%20DS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADavid%2BS.%2BLawrie%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-6782-9986" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for David S. Lawrie</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:dlawrie@alumni.stanford.edu" class="" data-icon-position="" data-hide-link-title="0">dlawrie@alumni.stanford.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/7/9/3229" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:486371" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:486371/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/9/3229.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:486371" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:486371/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/9/3229.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:486371" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:486371/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/9/3229.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:486371" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:486371/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/7/9/3229.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:486371" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:486371/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Forward Wright–Fisher simulations are powerful in their ability to model complex demography and selection scenarios, but suffer from slow execution on the Central Processor Unit (CPU), thus limiting their usefulness. However, the single-locus Wright–Fisher forward algorithm is exceedingly parallelizable, with many steps that are so-called “embarrassingly parallel,” consisting of a vast number of individual computations that are all independent of each other and thus capable of being performed concurrently. The rise of modern Graphics Processing Units (GPUs) and programming languages designed to leverage the inherent parallel nature of these processors have allowed researchers to dramatically speed up many programs that have such high arithmetic intensity and intrinsic concurrency. The presented GPU Optimized Wright–Fisher simulation, or “GO Fish” for short, can be used to simulate arbitrary selection and demographic scenarios while running over 250-fold faster than its serial counterpart on the CPU. Even modest GPU hardware can achieve an impressive speedup of over two orders of magnitude. With simulations so accelerated, one can not only do quick parametric bootstrapping of previously estimated parameters, but also use simulated results to calculate the likelihoods and summary statistics of demographic and selection models against real polymorphism data, all without restricting the demographic and selection scenarios that can be modeled or requiring approximations to the single-locus forward algorithm for efficiency. Further, as many of the parallel programming techniques used in this simulation can be applied to other computationally intensive algorithms important in population genetics, GO Fish serves as an exciting template for future research into accelerating computation in evolution. GO Fish is part of the Parallel PopGen Package available at: <a href="http://dl42.github.io/ParallelPopGen/.">http://dl42.github.io/ParallelPopGen/.</a></p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/gpu" class="hw-term hw-article-keyword hw-article-keyword-gpu" rel="nofollow">GPU</a></li><li class="kwd"><a href="/keyword/wright%E2%80%93fisher-model" class="hw-term hw-article-keyword hw-article-keyword-wright&#x2013;fisher-model" rel="nofollow">Wright–Fisher model</a></li><li class="kwd"><a href="/keyword/simulation" class="hw-term hw-article-keyword hw-article-keyword-simulation" rel="nofollow">simulation</a></li><li class="kwd"><a href="/keyword/population-genetics" class="hw-term hw-article-keyword hw-article-keyword-population-genetics" rel="nofollow">population genetics</a></li></ul><p id="p-3">The GPU is commonplace in today’s consumer and workstation computers and provides the main computational throughput of the modern supercomputer. A GPU differs from a computer’s CPU in a number of key respects, but the most important differentiating factor is the number and type of computational units. While a CPU for a typical consumer laptop or desktop will contain anywhere from two to four very fast, complex cores, GPU cores are in contrast relatively slow and simple. However, there are typically hundreds to thousands of these slow and simple cores in a single GPU. Thus, CPUs are low latency processors that excel at the serial execution of complex, branching algorithms. Conversely, the GPU architecture is designed to provide high computational bandwidth, capable of executing many arithmetic operations in parallel.</p><p id="p-4">The historical driver for the development of GPUs was increasingly realistic computer graphics for computer games. However, researchers quickly latched on to their usefulness as tools for scientific computation, particularly for problems that were simply too time consuming on the CPU due to sheer number of operations that had to be computed but where many of those operations could in principle be computed simultaneously. Eventually programming languages were developed to exploit GPUs as massive parallel processors and, over time, the GPU hardware has likewise evolved to be more capable for both graphics and computational applications.</p><p id="p-5">Population genetics analysis of single nucleotide polymorphisms (SNPs) is exceptionally amenable to acceleration on the GPU. Beyond the study of evolution itself, such analysis is a critical component of research in medical and conservation genetics, providing insight into the selective and mutational forces shaping the genome as well as the demographic history of a population. One of the most common analysis methods is the site frequency spectrum (SFS), a histogram where each bin is a count of how many mutations are at a given frequency in the population.</p><p id="p-6">SFS analysis is based on the precepts of the Wright–Fisher process (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Fisher 1930</a>; <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Wright 1938</a>), which describes the probabilistic trajectory of a mutation’s frequency in a population under a chosen evolutionary scenario. The defining characteristic of the Wright–Fisher process is forward time, nonoverlapping, discrete generations with random genetic drift modeled as a binomial distribution dependent on the population size and the frequency of a mutation (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Fisher 1930</a>; <a id="xref-ref-52-2" class="xref-bibr" href="#ref-52">Wright 1938</a>). On top of this foundation can be added models for selection, migration between populations, mate choice and inbreeding, and linkage between different loci, etc. For simple scenarios, an approximate analytical expression for the expected proportion of mutations at a given frequency in the population, the expected SFS, can be derived (<a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">Fisher 1930</a>; <a id="xref-ref-52-3" class="xref-bibr" href="#ref-52">Wright 1938</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Kimura 1964</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Sawyer and Hartl 1992</a>; <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Williamson <em>et al.</em> 2004</a>). This expectation can then be compared to the observed SFS of real data, allowing for parameter fitting and model testing (<a id="xref-ref-51-2" class="xref-bibr" href="#ref-51">Williamson <em>et al.</em> 2004</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Machado <em>et al.</em> 2017</a>). However, more complex scenarios do not have tractable analytical solutions, approximate or otherwise. One approach is to simulate the Wright–Fisher process forward in time to build the expected frequency distribution or other population genetic summary statistics (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Hernandez 2008</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Messer 2013</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Thornton 2014</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Ortega-Del Vecchyo <em>et al.</em> 2016</a>). Because of the flexibility inherent in its construction, the Wright–Fisher forward simulation can be used to model any arbitrarily complex demographic and selection scenario (<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Hernandez 2008</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Carvajal-Rodriguez 2010</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Hoban <em>et al.</em> 2012</a>; <a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Messer 2013</a>; <a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">Thornton 2014</a>; <a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Ortega-Del Vecchyo <em>et al.</em> 2016</a>). Unfortunately, because of the computational cost, the use of such simulations to analyze polymorphism data are often prohibitively expensive in practice (<a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Carvajal-Rodriguez 2010</a>; <a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Hoban <em>et al.</em> 2012</a>). The coalescent looking backward in time and approximations to the forward single-locus Wright–Fisher algorithm using diffusion equations provide alternative, computationally efficient methods of modeling polymorphism data (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Hudson 2002</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Gutenkunst <em>et al.</em> 2009</a>). However, these effectively limit the selection and demographic models that can be ascertained and approximate the Wright–Fisher forward process (<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Gutenkunst <em>et al.</em> 2009</a>; <a id="xref-ref-3-3" class="xref-bibr" href="#ref-3">Carvajal-Rodriguez 2010</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Ewing and Hermisson 2010</a>; <a id="xref-ref-18-3" class="xref-bibr" href="#ref-18">Hoban <em>et al.</em> 2012</a>). Thus, by speeding up forward simulations, we can use more complex and realistic demographic and selection models to analyze within-species polymorphism data.</p><p id="p-7">Single-locus Wright–Fisher simulations based on the Poisson Random Field model (<a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Sawyer and Hartl 1992</a>) ignore linkage between sites and simulate large numbers of individual mutation frequency trajectories forward in time to construct the expected SFS. Exploiting the naturally parallelizable nature of the single-locus Wright–Fisher algorithm, these forward simulations can be greatly accelerated on the GPU. Written in the programming language CUDA (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Nickolls <em>et al.</em> 2008</a>), a C/C++ derivative for NVIDIA GPUs, GO Fish allows for accurate, flexible simulations of SFS at speeds orders of magnitude faster than comparative serial programs on the CPU. As a programming library, GO Fish can be run in standalone scripts or integrated into other programs to accelerate single-locus Wright–Fisher simulations used by those tools.</p><div class="section methods" id="sec-1"><h2 class="">Methods</h2><div id="sec-2" class="subsection"><h3>Algorithm</h3><p id="p-8">In a single-locus Wright–Fisher simulation, a population of individuals can be represented by the set of mutations segregating in that population, specifically by the frequencies of the mutant, derived alleles in the population. Under the Poisson Random Field model, these mutations are completely independent of each other and new mutational events only occur at nonsegregating sites in the genome (<em>i.e.</em>, no multiple hits) (<a id="xref-ref-43-3" class="xref-bibr" href="#ref-43">Sawyer and Hartl 1992</a>).</p><p id="p-9"><a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a> sketches the algorithm for a typical, serial Wright–Fisher simulation, starting with the initialization of an array of mutation frequencies. From one discrete generation time step to the next, the change in any given mutation’s frequency is dependent on the strength of selection on that mutation, migration from other populations, the percent of inbreeding, and genetic drift. Unlike the others listed, inbreeding is not directly a force for allele frequency change, but rather it modifies the effectiveness of selection and drift. Frequencies of 0 (lost) and 1 (fixed) are absorbing boundaries such that if a mutation becomes fixed or lost across all extant populations, it is removed from the next generation’s mutation array. New mutations arising stochastically throughout the genome are then added to the mutation array of the offspring generation, replacing those mutations lost and fixed by selection and drift. As the offspring become the parents of the next generation, the cycle repeats until the final generation of the simulation.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/9/3229/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Serial Wright&#x2013;Fisher algorithm. Mutations are the &#x201C;unit&#x201D; of simulation for the single-locus Wright&#x2013;Fisher algorithm. Thus, a generation of organisms is represented by an array of mutations and their frequency in the (each) population (if there are multiple in the simulation). There are several options for how to initialize the mutation array to start a simulation: a blank mutation array, the output of a previous simulation run, or mutation&#x2013;selection equilibrium (for details, see File S1). Simulating each discrete generation first consists of calculating the new allele frequency of each mutation, one at a time, where those mutations that become lost or fixed are discarded. Next, new mutations are added to the array, again, one at a time. The resulting offspring array of mutation frequencies becomes the parent array of the next generation and the cycle is repeated until the end of the simulation when the final mutation array is output." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1678968880" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Serial Wright&#x2013;Fisher algorithm. Mutations are the &#x201C;unit&#x201D; of simulation for the single-locus Wright&#x2013;Fisher algorithm. Thus, a generation of organisms is represented by an array of mutations and their frequency in the (each) population (if there are multiple in the simulation). There are several options for how to initialize the mutation array to start a simulation: a blank mutation array, the output of a previous simulation run, or mutation&#x2013;selection equilibrium (for details, see File S1). Simulating each discrete generation first consists of calculating the new allele frequency of each mutation, one at a time, where those mutations that become lost or fixed are discarded. Next, new mutations are added to the array, again, one at a time. The resulting offspring array of mutation frequencies becomes the parent array of the next generation and the cycle is repeated until the end of the simulation when the final mutation array is output.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/9/3229/F1.medium.gif" width="440" height="251"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/7/9/3229/F1.medium.gif" width="440" height="251"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/9/3229/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/9/3229/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/486452" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-10" class="first-child">Serial Wright–Fisher algorithm. Mutations are the “unit” of simulation for the single-locus Wright–Fisher algorithm. Thus, a generation of organisms is represented by an array of mutations and their frequency in the (each) population (if there are multiple in the simulation). There are several options for how to initialize the mutation array to start a simulation: a blank mutation array, the output of a previous simulation run, or mutation–selection equilibrium (for details, see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300103/-/DC1/FileS1.pdf">File S1</a>). Simulating each discrete generation first consists of calculating the new allele frequency of each mutation, one at a time, where those mutations that become lost or fixed are discarded. Next, new mutations are added to the array, again, one at a time. The resulting offspring array of mutation frequencies becomes the parent array of the next generation and the cycle is repeated until the end of the simulation when the final mutation array is output.</p><div class="sb-div caption-clear"></div></div></div><p id="p-11">While the details of how a GPU organizes computational work are quite intricate (<a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Nickolls <em>et al.</em> 2008</a>), the vastly oversimplified version is that a serial set of operations is called a thread and the GPU can execute many such threads in parallel. With completely unlinked sites, every simulated mutation frequency trajectory is independent of every other mutation frequency trajectory in the simulation. Therefore, the single-locus Wright–Fisher algorithm is trivially parallelized by simply assigning a thread to each mutation in the mutation array; when simulating each discrete generation, both calculating the new frequency of alleles in the next generation and adding new mutations to next generation are embarrassingly parallel operations (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>). This is the parallel ideal because no communication across threads is required to make these calculations. A serial algorithm has to calculate the new frequency of each mutation one by one, and the problem is multiplied where there are multiple populations as these new frequencies have to be calculated for each population. For example, in a simulation with 100,000 mutations in a given generation and three populations, 300,000 sequential passes through the functions governing migration, selection, and drift are required. However, in the parallel version, this huge number of iterations can theoretically be compressed to a single step in which all the new frequencies for all mutations are computed simultaneously. Similarly, if there are 5000 new mutations in a generation, a serial algorithm has to add each of those 5000 new mutations one at a time to the simulation. The parallel algorithm can, in theory, add them all at once. Of course, a GPU only has a finite number of computational resources to apply to a problem and thus this ideal of executing all processes in a single time step is never truly realizable for a problem of any substantial size. Even so, parallelizing migration, selection, drift, and mutation on the GPU results in dramatic speedups relative to performing those same operations serially on the CPU. This is the main source of GO Fish’s improvement over serial, CPU-based Wright–Fisher simulations.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/9/3229/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Common parallel algorithms. Illustrative examples of three classes of common parallel algorithms implemented using simple operations and an eight-element, integer array. (A) Embarrassingly parallel algorithms are those that can be computed independently and thus simultaneously on the GPU. The given example, adding one to every element of an array, can be done concurrently to all array elements. In GO Fish, calculating new mutation frequencies and adding new mutations to population are both embarrassingly parallel operations. (B) Reduce is a fundamental parallel algorithm in which all the elements of an array are reduced to a single value using a binary operator, such as in the above summation over the example array (Harris 2007a). This algorithm takes advantage of the fact that in each time step half of the sums can be done independently while synchronized communication is necessary to combine the results of previous time steps. In total, log2(8) = 3 time steps are required to reduce the example array. (C) Compact is a multi-step algorithm that allows one to filter arrays on the GPU (Billeter et al. 2009). In an embarrassingly parallel step, the algorithm presented above first creates a new Boolean array of those elements that passed the filter predicate (e.g., x &gt; 1). Then a &#x201C;scan&#x201D; is performed on the Boolean array. Scan is similar in concept to reduce, wherein for each time step half of the binary operations are independent, but it is a more complex parallel algorithm that creates a running sum over the array rather than condensing the array to a single value [see Harris et al. (2007b)]. This running sum determines the new index of each element in the original array being filtered and the size of the new array. Those elements that passed the filter are then scattered to their new indices of the now smaller, filtered array. Compact is used in GO Fish to filter out fixed and lost mutations. GPU, Graphics Processing Unit." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1678968880" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Common parallel algorithms. Illustrative examples of three classes of common parallel algorithms implemented using simple operations and an eight-element, integer array. (A) Embarrassingly parallel algorithms are those that can be computed independently and thus simultaneously on the GPU. The given example, adding one to every element of an array, can be done concurrently to all array elements. In GO Fish, calculating new mutation frequencies and adding new mutations to population are both embarrassingly parallel operations. (B) Reduce is a fundamental parallel algorithm in which all the elements of an array are reduced to a single value using a binary operator, such as in the above summation over the example array (Harris 2007a). This algorithm takes advantage of the fact that in each time step half of the sums can be done independently while synchronized communication is necessary to combine the results of previous time steps. In total, log&lt;sub&gt;2&lt;/sub&gt;(8) = 3 time steps are required to reduce the example array. (C) Compact is a multi-step algorithm that allows one to filter arrays on the GPU (Billeter &lt;em&gt;et al.&lt;/em&gt; 2009). In an embarrassingly parallel step, the algorithm presented above first creates a new Boolean array of those elements that passed the filter predicate (&lt;em&gt;e.g.&lt;/em&gt;, &lt;em&gt;x&lt;/em&gt; &gt; 1). Then a &#x201C;scan&#x201D; is performed on the Boolean array. Scan is similar in concept to reduce, wherein for each time step half of the binary operations are independent, but it is a more complex parallel algorithm that creates a running sum over the array rather than condensing the array to a single value [see Harris &lt;em&gt;et al.&lt;/em&gt; (2007b)]. This running sum determines the new index of each element in the original array being filtered and the size of the new array. Those elements that passed the filter are then scattered to their new indices of the now smaller, filtered array. Compact is used in GO Fish to filter out fixed and lost mutations. GPU, Graphics Processing Unit.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/9/3229/F2.medium.gif" width="440" height="286"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/7/9/3229/F2.medium.gif" width="440" height="286"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/9/3229/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/9/3229/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/486579" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-12" class="first-child">Common parallel algorithms. Illustrative examples of three classes of common parallel algorithms implemented using simple operations and an eight-element, integer array. (A) Embarrassingly parallel algorithms are those that can be computed independently and thus simultaneously on the GPU. The given example, adding one to every element of an array, can be done concurrently to all array elements. In GO Fish, calculating new mutation frequencies and adding new mutations to population are both embarrassingly parallel operations. (B) Reduce is a fundamental parallel algorithm in which all the elements of an array are reduced to a single value using a binary operator, such as in the above summation over the example array (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Harris 2007a</a>). This algorithm takes advantage of the fact that in each time step half of the sums can be done independently while synchronized communication is necessary to combine the results of previous time steps. In total, log<sub>2</sub>(8) = 3 time steps are required to reduce the example array. (C) Compact is a multi-step algorithm that allows one to filter arrays on the GPU (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Billeter <em>et al.</em> 2009</a>). In an embarrassingly parallel step, the algorithm presented above first creates a new Boolean array of those elements that passed the filter predicate (<em>e.g.</em>, <em>x</em> &gt; 1). Then a “scan” is performed on the Boolean array. Scan is similar in concept to reduce, wherein for each time step half of the binary operations are independent, but it is a more complex parallel algorithm that creates a running sum over the array rather than condensing the array to a single value [see <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Harris <em>et al.</em> (2007b)</a>]. This running sum determines the new index of each element in the original array being filtered and the size of the new array. Those elements that passed the filter are then scattered to their new indices of the now smaller, filtered array. Compact is used in GO Fish to filter out fixed and lost mutations. GPU, Graphics Processing Unit.</p><div class="sb-div caption-clear"></div></div></div><p id="p-13">One challenge to the parallelization of the Wright–Fisher algorithm is the treatment of mutations that become fixed or lost. When a mutation reaches a frequency of 0 (in all populations, if multiple) or 1 (in all populations, if multiple), that mutation is forever lost or fixed. Such mutations are no longer of interest to maintain in memory or process from one generation to the next. Without removing lost and fixed mutations from the simulation, the number of mutations being stored and processed would simply continue to grow as new mutations are added each generation. When processing mutations one at a time in the serial algorithm, removing mutations that become lost or fixed is as trivial as simply not adding them to the next generation and shortening the mutation array in the next generation by one each time. This becomes more difficult when processing mutations in parallel. As stated before: the different threads for different mutations do not communicate with each other when calculating the new mutation frequencies simultaneously. Therefore, any given mutation/thread has no knowledge of how many other mutations have become lost or fixed that generation. This in turn means that when attempting to remove lost and fixed mutations while processing mutations in parallel, there is no way to determine the size of the next generation’s mutation array or where in the offspring array each mutation should be placed.</p><p id="p-14">One solution to the above problems is the algorithm “compact” (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Billeter <em>et al.</em> 2009</a>), which can filter out lost and fixed mutations while still taking advantage of the parallel nature of GPUs (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2C</a>). However, compaction is not embarrassingly parallel, as communication between the different threads for different mutations is required, and it involves a lot of moving elements around in GPU memory rather than intensive computation. Thus, it is a less efficient use of the GPU as compared to calculating allele frequencies. As such, a nuance in optimizing GO Fish is how frequently to remove lost and fixed mutations from the active simulation. Despite the fact that computation on such mutations is wasted, calculating new allele frequencies is so fast that not filtering out lost and fixed mutations every generation and temporarily leaving them in the simulation actually results in faster runtimes. Eventually of course, the sheer number of lost and fixed mutations overwhelms even the GPU’s computational bandwidth and they must be removed. How often to compact for optimal simulation speed can be ascertained heuristically and is dependent on the number of mutations each generation in the simulation and the attributes of the GPU the simulation is running on. <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a> illustrates the algorithm for GO Fish, which combines parallel implementations of migration, selection, drift, and mutation with a compacting step run every X generations and again before the end of the simulation.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/9/3229/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="GO Fish algorithm. Both altering the allele frequencies of mutations from parent to child generation and adding new mutations to the child generation are embarrassingly parallel operations (see Figure 2A) that are greatly accelerated on the GPU. Further, as independent operations, adding new mutations and altering allele frequencies can be done concurrently on the GPU. In comparison to serial Wright&#x2013;Fisher simulations (Figure 1), GO Fish includes an extra compact step (see Figure 2C) to remove fixed and lost mutations every X generations. Until compaction, the size of the mutation array grows by the number of new mutations added each generation. Before the simulation ends, the program compacts the mutation array one final time. GPU, Graphics Processing Unit." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1678968880" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;GO Fish algorithm. Both altering the allele frequencies of mutations from parent to child generation and adding new mutations to the child generation are embarrassingly parallel operations (see Figure 2A) that are greatly accelerated on the GPU. Further, as independent operations, adding new mutations and altering allele frequencies can be done concurrently on the GPU. In comparison to serial Wright&#x2013;Fisher simulations (Figure 1), GO Fish includes an extra compact step (see Figure 2C) to remove fixed and lost mutations every X generations. Until compaction, the size of the mutation array grows by the number of new mutations added each generation. Before the simulation ends, the program compacts the mutation array one final time. GPU, Graphics Processing Unit.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/9/3229/F3.medium.gif" width="367" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/7/9/3229/F3.medium.gif" width="367" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/9/3229/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/9/3229/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/486536" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-15" class="first-child">GO Fish algorithm. Both altering the allele frequencies of mutations from parent to child generation and adding new mutations to the child generation are embarrassingly parallel operations (see <a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2A</a>) that are greatly accelerated on the GPU. Further, as independent operations, adding new mutations and altering allele frequencies can be done concurrently on the GPU. In comparison to serial Wright–Fisher simulations (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>), GO Fish includes an extra compact step (see <a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2C</a>) to remove fixed and lost mutations every X generations. Until compaction, the size of the mutation array grows by the number of new mutations added each generation. Before the simulation ends, the program compacts the mutation array one final time. GPU, Graphics Processing Unit.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-3" class="subsection"><h3>The population genetics model of GO Fish</h3><p id="p-16">A more detailed description of the implementation of the Wright–Fisher algorithm underlying GO Fish, with derivations of the equations below, can be found in Supplemental Material, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300103/-/DC1/FileS1.pdf">File S1</a>. <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a> provides a glossary of the variables used in the simulation.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/486429/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/486429/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Glossary of simulation terms</span><div class="sb-div caption-clear"></div></div></div><p id="p-21">The simulation can start with an empty initial mutation array, with the output of a previous simulation run, or with the frequencies of the initial mutation array in mutation–selection equilibrium. Starting a simulation as a blank canvas provides the most flexibility in the starting evolutionary scenario. However, to reach an equilibrium start point requires a “burn-in,” which may be quite a large number of generations (<a id="xref-ref-41-3" class="xref-bibr" href="#ref-41">Ortega-Del Vecchyo <em>et al.</em> 2016</a>). To save time, if a starting scenario is shared across multiple simulations, then the postburn-in mutation array can be simulated beforehand, stored, and input as the initial mutation array for the next set of simulations. Alternatively, the simulation can be initialized in a calculable, approximate mutation–selection equilibrium state, allowing the simulation of the evolutionary scenario of interest to begin essentially immediately. λ<sub>μ</sub>(<em>x</em>) is the expected (mean) number of mutations at a given frequency, <em>x</em>, in the population at mutation–selection equilibrium and can be calculate via the following equation:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3229/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3229/embed/mml-math-1.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3229/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3229/embed/mml-math-2.gif"/></noscript></span><span class="disp-formula-label">(1)</span></span>The derivation for Equation 1 can be found in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300103/-/DC1/FileS1.pdf">File S1</a> (Equation 1–6 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300103/-/DC1/FileS1.pdf">File S1</a>). The numerical integration required to calculate λ<sub>μ</sub>(<em>x</em>) has been parallelized and accelerated on the GPU. To start the simulation, the actual number of mutations at each frequency is determined by draws from the Inverse Poisson distribution with mean and variance λ<sub>μ</sub>(<em>x</em>). This numerical initialization routine can handle most of the equilibrium evolutionary scenarios the main simulation is capable of itself, a major exception being those cases with migration between multiple populations. Given the number of cases covered by the above integration technique, this is likely to be the primary method to start a GO Fish simulation in a state of mutation–selection equilibrium.</p><p id="p-22">After initialization begins the cycle of adding new mutations to the population and calculating new frequencies for currently segregating mutations. The number of new mutations introduced in each population <em>j</em>, for each generation <em>t</em> is Poisson distributed with mean <em>N<sub>e</sub></em>μ<em>L</em> in accordance with the assumptions of the Poisson Random Field Model. These new mutations start at frequency 1/<em>N<sub>e</sub></em> in the simulation. Meanwhile, the SNP frequencies of the extant mutations in the current generation <em>t</em> and population <em>j</em> are modified by the forces of migration (I.), selection (II.), and drift (III.) to produce the new frequencies of those mutations in generation <em>t + 1</em>.<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3229/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3229/embed/mml-math-3.gif"/></noscript></span></span>I. GO Fish uses a conservative model of migration (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Nagylaki 1980</a>) where the new allele frequency, <em>x<sub>mig</sub></em>, in population <em>j</em> is the average of the allele frequency in all the populations weighted by the migration rate from each population to population <em>j</em>. II. Selection further modifies the expected frequency of the mutations in population <em>j</em> according to Equation 2 below:<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3229/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3229/embed/mml-math-4.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3229/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3229/embed/mml-math-5.gif"/></noscript></span><span class="disp-formula-label">(2)</span></span>The derivation for Equation 2 can be found in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300103/-/DC1/FileS1.pdf">File S1</a> (Equation 8–13 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300103/-/DC1/FileS1.pdf">File S1</a>). The variable <em>x<sub>mig,sel</sub></em> represents the expected frequency of an allele in generation <em>t + 1</em>. III. Drift, which is modeled as a binomial random deviation with mean <em>N<sub>e</sub>x<sub>mig,sel</sub></em> and variance <em>N<sub>e</sub>x<sub>mig,sel</sub></em>(1-<em>x<sub>mig,sel</sub></em>), then acts on top of the deterministic forces of migration and selection to produce the ultimate frequency of the allele in the next generation, <em>t + 1</em>, in population <em>j</em>, <em>x<sub>t+1,j</sub></em>. Then the cycle repeats.</p></div><div id="sec-4" class="subsection"><h3>Data availability</h3><p id="p-23">The library of parallel APIs, the Parallel PopGen Package, of which GO Fish is a member, is hosted on GitHub at <a href="https://github.com/DL42/ParallelPopGen">https://github.com/DL42/ParallelPopGen</a>. In the Git repository, the code generating <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>, <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300103/-/DC1/FigureS1.pdf">Figure S1</a> is in the folders examples/example_speed (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>) and examples/example_dadi (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300103/-/DC1/FigureS1.pdf">Figure S1</a>) along with example outputs. The companion serial Wright–Fisher simulation, serial_SFS_code.cpp, is provided in examples/examples_speed, as is a help file, serial_SFS_code_help.txt, and makefile, makefile_serial. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300103/-/DC1/TableS1.xlsx">Table S1</a>, referenced in <a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4</a>, can also be found under the folder documentation/ in the GitHub repository uploaded as excel file 780M_980_Intel_speed_results.xlsx. The API manual is at <a href="http://dl42.github.io/ParallelPopGen/">http://dl42.github.io/ParallelPopGen/</a>.</p></div></div><div class="section results-discussion" id="sec-5"><h2 class="">Results and Discussion</h2><p id="p-24">To test the speed improvements from parallelizing the Wright–Fisher algorithm, GO Fish was compared to a serial Wright–Fisher simulation written in C++. Each program was run on two computers: an iMac and a self-built Linux-box with equivalent Intel Haswell CPUs, but very different NVIDIA GPUs. Constrained by the thermal and space requirements of laptops and all-in-one machines, the iMac’s NVIDIA 780M GPU (1536 cores at 823 MHz) is slower and older than the NVIDIA 980 (2048 cores at 1380 MHz) in the Linux-box. For a given number of simulated populations and number of generations, a key driver of execution time is the number of mutations in the simulation. Thus, many different evolutionary scenarios will have similar runtimes if they result in similar numbers of mutations being simulated each generation. As such, to benchmark the acceleration provided by parallelization and GPUs, the programs were run using a basic evolutionary scenario while varying the number of expected mutations in the simulation. The utilized scenario is a simple, neutral simulation, starting in mutation–selection equilibrium, of a single, haploid population with a constant population size of 200,000 individuals over 1000 generations and a mutation rate of 1 × 10<sup>−9</sup> mutations per generation per individual per site. With these other parameters held constant, varying the number of sites in the simulation adjusts the number of expected mutations for each of the benchmark simulations.</p><p id="p-25">As shown in <a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4</a>, accelerating the Wright–Fisher simulation on a GPU results in massive performance gains on both an older, mobile GPU like the NVIDIA 780M and a newer, desktop-class NVIDIA 980 GPU. For example, when simulating the frequency trajectories of ∼500,000 mutations over 1000 generations, GO Fish takes ∼0.2 sec to run on a 780M as compared to ∼18 sec for its serial counterpart running on the Intel i5/i7 CPU (@3.9 GHz), a speedup of 88-fold. On a full, modern desktop GPU like the 980, GO Fish runs this scenario ∼176× faster than the strictly serial simulation and only takes ∼0.1 sec to run. As the number of mutations in the simulation grows, more work is tasked to the GPU and the relative speedup of GPU to CPU increases logarithmically. Eventually though, the sheer number of simulated SNPs saturates even the computational throughput of the GPUs, producing linear increases in runtime for increasing SNP counts, like for serial code. Thus, eventually, there is a flattening of the fold performance gains. This plateau occurs earlier for 780M than for the more powerful 980 with its more and faster cores. Executed serially on the CPU, a huge simulation of ∼4 × 10<sup>7</sup> SNPs takes roughly 24 min to run <em>vs.</em> only ∼13/5.7 sec for GO Fish on the 780M/980, an acceleration of &gt; 109/250-fold. While not benchmarked here, the parallel Wright–Fisher algorithm is also trivial to partition over multi-GPU setups in order to further accelerate simulations.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/9/3229/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Performance gains on GPU relative to CPU. The above figure plots the relative performance of GO Fish, written in CUDA, to a basic, serial Wright&#x2013;Fisher simulation written in C++. The two programs were run both on a 2013 iMac with an NVIDIA GeForce GTX 780M mobile GPU, 1536 at 823 MHz cores, (black line) and an Intel Core i7 4771 CPU at 3.9 GHz and a self-built Linux-box with a factory-overclocked NVIDIA GeForce GTX 980 GPU, 2048 at 1380 MHz cores, and an Intel Core i5 4690K CPU at 3.9 GHz (red line). Full compiler optimizations (&#x2212;O3 &#x2013;fast-math) were applied to both serial and parallel programs. Each dot represents a simulation run plotted by the number of SNPs in its final generation. The serial program was run on the &#x223C;10,000, &#x223C;100,000, and &#x223C;1 &#xD7; 106 SNP scenarios. As the speed of the CPU-based program is linear on the number of simulated SNPs, the resulting runtimes of 0.4, 3.9, and 38.7 sec were then linearly rescaled to estimate runtimes for serial simulations with differing numbers of final SNPs. The two Intel processors have identical speeds on single-threaded, serial tasks, which also allows for direct comparison between the two GPU results. Consumer GPUs like the 780M and 980 need to warm up from idle and load the CUDA context. So, to obtain accurate runtimes on the GPU, GO Fish timings were done after 10 runs had finished and then the average of another 10 runs was taken for each data point. The GO Fish compacting rate was hand-optimized for each number of simulated SNPs, for each processor (Table S1). CPU, Central Processor Unit; GPU, Graphics Processing Unit; SNP, single nucleotide polymorphism." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1678968880" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Performance gains on GPU relative to CPU. The above figure plots the relative performance of GO Fish, written in CUDA, to a basic, serial Wright&#x2013;Fisher simulation written in C++. The two programs were run both on a 2013 iMac with an NVIDIA GeForce GTX 780M mobile GPU, 1536 at 823 MHz cores, (black line) and an Intel Core i7 4771 CPU at 3.9 GHz and a self-built Linux-box with a factory-overclocked NVIDIA GeForce GTX 980 GPU, 2048 at 1380 MHz cores, and an Intel Core i5 4690K CPU at 3.9 GHz (red line). Full compiler optimizations (&#x2212;O3 &#x2013;fast-math) were applied to both serial and parallel programs. Each dot represents a simulation run plotted by the number of SNPs in its final generation. The serial program was run on the &#x223C;10,000, &#x223C;100,000, and &#x223C;1 &#xD7; 10&lt;sup&gt;6&lt;/sup&gt; SNP scenarios. As the speed of the CPU-based program is linear on the number of simulated SNPs, the resulting runtimes of 0.4, 3.9, and 38.7 sec were then linearly rescaled to estimate runtimes for serial simulations with differing numbers of final SNPs. The two Intel processors have identical speeds on single-threaded, serial tasks, which also allows for direct comparison between the two GPU results. Consumer GPUs like the 780M and 980 need to warm up from idle and load the CUDA context. So, to obtain accurate runtimes on the GPU, GO Fish timings were done after 10 runs had finished and then the average of another 10 runs was taken for each data point. The GO Fish compacting rate was hand-optimized for each number of simulated SNPs, for each processor (Table S1). CPU, Central Processor Unit; GPU, Graphics Processing Unit; SNP, single nucleotide polymorphism.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/9/3229/F4.medium.gif" width="440" height="304"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/7/9/3229/F4.medium.gif" width="440" height="304"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/9/3229/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/9/3229/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/486576" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-26" class="first-child">Performance gains on GPU relative to CPU. The above figure plots the relative performance of GO Fish, written in CUDA, to a basic, serial Wright–Fisher simulation written in C++. The two programs were run both on a 2013 iMac with an NVIDIA GeForce GTX 780M mobile GPU, 1536 at 823 MHz cores, (black line) and an Intel Core i7 4771 CPU at 3.9 GHz and a self-built Linux-box with a factory-overclocked NVIDIA GeForce GTX 980 GPU, 2048 at 1380 MHz cores, and an Intel Core i5 4690K CPU at 3.9 GHz (red line). Full compiler optimizations (−O3 –fast-math) were applied to both serial and parallel programs. Each dot represents a simulation run plotted by the number of SNPs in its final generation. The serial program was run on the ∼10,000, ∼100,000, and ∼1 × 10<sup>6</sup> SNP scenarios. As the speed of the CPU-based program is linear on the number of simulated SNPs, the resulting runtimes of 0.4, 3.9, and 38.7 sec were then linearly rescaled to estimate runtimes for serial simulations with differing numbers of final SNPs. The two Intel processors have identical speeds on single-threaded, serial tasks, which also allows for direct comparison between the two GPU results. Consumer GPUs like the 780M and 980 need to warm up from idle and load the CUDA context. So, to obtain accurate runtimes on the GPU, GO Fish timings were done after 10 runs had finished and then the average of another 10 runs was taken for each data point. The GO Fish compacting rate was hand-optimized for each number of simulated SNPs, for each processor (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300103/-/DC1/TableS1.xlsx">Table S1</a>). CPU, Central Processor Unit; GPU, Graphics Processing Unit; SNP, single nucleotide polymorphism.</p><div class="sb-div caption-clear"></div></div></div><p id="p-27">Tools employing the single-locus Wright–Fisher framework are widely used in population genetics analyses to estimate selection coefficients and infer demography [see <a id="xref-ref-14-3" class="xref-bibr" href="#ref-14">Gutenkunst <em>et al.</em> (2009)</a>, <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Garud <em>et al.</em> (2015)</a>, <a id="xref-ref-41-4" class="xref-bibr" href="#ref-41">Ortega-Del Vecchyo <em>et al.</em> (2016)</a>, <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Jackson <em>et al.</em> (2017)</a>, <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Kim <em>et al.</em> (2017)</a>, <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Koch and Novembre (2017)</a>, <a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Machado <em>et al.</em> (2017)</a> for examples]. Often, these tools employ either a numerically solved diffusion approximation, or even the simple analytical function, to generate the expected SFS of a given evolutionary scenario, which can then be used to calculate the likelihood of producing an observed SFS. The model parameters of the evolutionary scenario are then fitted to the data by maximizing the composite likelihood. With GO Fish, forward simulation can generate the expected spectra. To validate these expected spectra, the results of GO Fish simulations were compared against δaδi (<a id="xref-ref-14-4" class="xref-bibr" href="#ref-14">Gutenkunst <em>et al.</em> 2009</a>) for a complex evolutionary scenario involving a single population splitting into two, exponential growth, selection, and migration (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5</a>). The spectra generated by each program are identical. Interestingly, the two programs also had essentially identical runtimes for this scenario and hardware (<a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5</a>). In general, the relative compute time will vary depending on the simulation size and population scaling for GO Fish, the grid size and time-step for δaδi (<a id="xref-ref-14-5" class="xref-bibr" href="#ref-14">Gutenkunst <em>et al.</em> 2009</a>), and the simulation scenario and computational hardware for both.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/9/3229/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Validation of GO Fish simulation results against &#x3B4;a&#x3B4;i. A complex demographic scenario was chosen as a test case to compare the GO Fish simulation against an already established site frequency spectrum (SFS) method, &#x3B4;a&#x3B4;i (Gutenkunst et al. 2009). The demographic model is from a &#x3B4;a&#x3B4;i code example for the Yoruba-Northern European (AF-EU) populations. Using &#x3B4;a&#x3B4;i parameterization to describe the model, the ancestral population, in mutation&#x2013;selection equilibrium, undergoes an immediate expansion from Nref to 2Nref individuals. After time T1 (= 0.005) the population splits into two with a constant, equivalent migration, mEU-AF (= 1) between the now split populations. The second (EU) population undergoes a severe bottleneck of 0.05Nref when splitting from the first (AF) population, followed by exponential growth over time T2 (= 0.045) to size 5Nref. The SFS (black dashed line) above is of weakly deleterious, codominant mutations (2Nrefs = &#x2212;2, h = 0.5) where 1001 samples were taken of the EU population. The spectrum was then normalized by the number of segregating sites. The corresponding GO Fish parameters for the evolutionary scenario, given a mutation rate of 1 &#xD7; 10&#x2212;9 per site, 2 &#xD7; 109 sites, and an initial population size, Nref, of 10,000, are: T1 = 0.005 &#xD7; 2Nref = 100 generations, T2 = 900 generations, mEU-AF = 1/(2Nref) = 0.00005, 2Nrefs = &#x2212;4, h = 0.5, and F = 0. As in &#x3B4;a&#x3B4;i, the population size/time can be scaled together and the simulation will generate the same normalized spectra (Gutenkunst et al. 2009). Using the aforementioned parameters, a GO Fish simulation ends with &#x223C;3 &#xD7; 106 mutations, of which &#x223C;560,000 are sampled in the SFS. The red line reporting GO Fish results is the average of 50 such simulations; the dispersion of those 50 simulations is reported in Figure S1. Each simulation run on the NVIDIA GeForce GTX 980 took roughly the same time to generate the SFS as &#x3B4;a&#x3B4;i did [grid size = (110,120,130), time-step = 10&#x2212;3] on the Intel Core i7 4771, just &lt; 0.7 sec." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1678968880" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Validation of GO Fish simulation results against &#x3B4;a&#x3B4;i. A complex demographic scenario was chosen as a test case to compare the GO Fish simulation against an already established site frequency spectrum (SFS) method, &#x3B4;a&#x3B4;i (Gutenkunst &lt;em&gt;et al.&lt;/em&gt; 2009). The demographic model is from a &#x3B4;a&#x3B4;i code example for the Yoruba-Northern European (AF-EU) populations. Using &#x3B4;a&#x3B4;i parameterization to describe the model, the ancestral population, in mutation&#x2013;selection equilibrium, undergoes an immediate expansion from &lt;em&gt;N&lt;sub&gt;ref&lt;/sub&gt;&lt;/em&gt; to 2&lt;em&gt;N&lt;sub&gt;ref&lt;/sub&gt;&lt;/em&gt; individuals. After time &lt;em&gt;T&lt;sub&gt;1&lt;/sub&gt;&lt;/em&gt; (= 0.005) the population splits into two with a constant, equivalent migration, &lt;em&gt;m&lt;sub&gt;EU-AF&lt;/sub&gt;&lt;/em&gt; (= 1) between the now split populations. The second (EU) population undergoes a severe bottleneck of 0.05&lt;em&gt;N&lt;sub&gt;ref&lt;/sub&gt;&lt;/em&gt; when splitting from the first (AF) population, followed by exponential growth over time &lt;em&gt;T&lt;sub&gt;2&lt;/sub&gt;&lt;/em&gt; (= 0.045) to size 5&lt;em&gt;N&lt;sub&gt;ref&lt;/sub&gt;&lt;/em&gt;. The SFS (black dashed line) above is of weakly deleterious, codominant mutations (2&lt;em&gt;N&lt;sub&gt;ref&lt;/sub&gt;s&lt;/em&gt; = &#x2212;2, &lt;em&gt;h&lt;/em&gt; = 0.5) where 1001 samples were taken of the EU population. The spectrum was then normalized by the number of segregating sites. The corresponding GO Fish parameters for the evolutionary scenario, given a mutation rate of 1 &#xD7; 10&lt;sup&gt;&#x2212;9&lt;/sup&gt; per site, 2 &#xD7; 10&lt;sup&gt;9&lt;/sup&gt; sites, and an initial population size, &lt;em&gt;N&lt;sub&gt;ref&lt;/sub&gt;&lt;/em&gt;, of 10,000, are: &lt;em&gt;T&lt;sub&gt;1&lt;/sub&gt;&lt;/em&gt; = 0.005 &#xD7; 2&lt;em&gt;N&lt;sub&gt;ref&lt;/sub&gt;&lt;/em&gt; = 100 generations, &lt;em&gt;T&lt;sub&gt;2&lt;/sub&gt;&lt;/em&gt; = 900 generations, &lt;em&gt;m&lt;sub&gt;EU-AF&lt;/sub&gt;&lt;/em&gt; = 1/(2&lt;em&gt;N&lt;sub&gt;ref&lt;/sub&gt;&lt;/em&gt;) = 0.00005, 2&lt;em&gt;N&lt;sub&gt;ref&lt;/sub&gt;s&lt;/em&gt; = &#x2212;4, &lt;em&gt;h&lt;/em&gt; = 0.5, and &lt;em&gt;F&lt;/em&gt; = 0. As in &#x3B4;a&#x3B4;i, the population size/time can be scaled together and the simulation will generate the same normalized spectra (Gutenkunst &lt;em&gt;et al.&lt;/em&gt; 2009). Using the aforementioned parameters, a GO Fish simulation ends with &#x223C;3 &#xD7; 10&lt;sup&gt;6&lt;/sup&gt; mutations, of which &#x223C;560,000 are sampled in the SFS. The red line reporting GO Fish results is the average of 50 such simulations; the dispersion of those 50 simulations is reported in Figure S1. Each simulation run on the NVIDIA GeForce GTX 980 took roughly the same time to generate the SFS as &#x3B4;a&#x3B4;i did [grid size = (110,120,130), time-step = 10&lt;sup&gt;&#x2212;3&lt;/sup&gt;] on the Intel Core i7 4771, just &lt; 0.7 sec.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/9/3229/F5.medium.gif" width="440" height="304"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/7/9/3229/F5.medium.gif" width="440" height="304"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/9/3229/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/9/3229/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/486475" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-28" class="first-child">Validation of GO Fish simulation results against δaδi. A complex demographic scenario was chosen as a test case to compare the GO Fish simulation against an already established site frequency spectrum (SFS) method, δaδi (<a id="xref-ref-14-6" class="xref-bibr" href="#ref-14">Gutenkunst <em>et al.</em> 2009</a>). The demographic model is from a δaδi code example for the Yoruba-Northern European (AF-EU) populations. Using δaδi parameterization to describe the model, the ancestral population, in mutation–selection equilibrium, undergoes an immediate expansion from <em>N<sub>ref</sub></em> to 2<em>N<sub>ref</sub></em> individuals. After time <em>T<sub>1</sub></em> (= 0.005) the population splits into two with a constant, equivalent migration, <em>m<sub>EU-AF</sub></em> (= 1) between the now split populations. The second (EU) population undergoes a severe bottleneck of 0.05<em>N<sub>ref</sub></em> when splitting from the first (AF) population, followed by exponential growth over time <em>T<sub>2</sub></em> (= 0.045) to size 5<em>N<sub>ref</sub></em>. The SFS (black dashed line) above is of weakly deleterious, codominant mutations (2<em>N<sub>ref</sub>s</em> = −2, <em>h</em> = 0.5) where 1001 samples were taken of the EU population. The spectrum was then normalized by the number of segregating sites. The corresponding GO Fish parameters for the evolutionary scenario, given a mutation rate of 1 × 10<sup>−9</sup> per site, 2 × 10<sup>9</sup> sites, and an initial population size, <em>N<sub>ref</sub></em>, of 10,000, are: <em>T<sub>1</sub></em> = 0.005 × 2<em>N<sub>ref</sub></em> = 100 generations, <em>T<sub>2</sub></em> = 900 generations, <em>m<sub>EU-AF</sub></em> = 1/(2<em>N<sub>ref</sub></em>) = 0.00005, 2<em>N<sub>ref</sub>s</em> = −4, <em>h</em> = 0.5, and <em>F</em> = 0. As in δaδi, the population size/time can be scaled together and the simulation will generate the same normalized spectra (<a id="xref-ref-14-7" class="xref-bibr" href="#ref-14">Gutenkunst <em>et al.</em> 2009</a>). Using the aforementioned parameters, a GO Fish simulation ends with ∼3 × 10<sup>6</sup> mutations, of which ∼560,000 are sampled in the SFS. The red line reporting GO Fish results is the average of 50 such simulations; the dispersion of those 50 simulations is reported in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300103/-/DC1/FigureS1.pdf">Figure S1</a>. Each simulation run on the NVIDIA GeForce GTX 980 took roughly the same time to generate the SFS as δaδi did [grid size = (110,120,130), time-step = 10<sup>−3</sup>] on the Intel Core i7 4771, just &lt; 0.7 sec.</p><div class="sb-div caption-clear"></div></div></div><p id="p-29">For maximum-likelihood and Bayesian statistics as for parametric bootstraps and C.I.s, hundreds, thousands, and even tens of thousands of distinct parameter values may need to be simulated to yield the needed statistics for a given model. Multiplying this by the need to often consider multiple evolutionary models as well as nonparametric bootstrapping of the data, a single serial simulation run on a CPU taking only 18 sec, as in the simple simulation of ∼500,000 SNPs presented in <a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4</a>, can add up to hours, even days of compute time. Moreover, and in contrast to the approximating analytical or numerical solutions typically employed, simulating the expected SFS introduces random noise around the “true” SFS of the scenario being modeled. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300103/-/DC1/FigureS1.pdf">Figure S1</a> demonstrates how increasing the number of simulated SNPs increases the precision of the simulation and therefore of the ensuing likelihood calculations. Simulating tens of millions of SNPs, wherein a single run on the CPU can take nearly half-an-hour, can be imperative to obtain a high-precision SFS needed for certain situations. Thus, the speed boost from parallelization on the GPU in calculating the underlying, expected SFS greatly enhances the practical utility of simulation for many current data analysis approaches. The speed and validation results demonstrate that, now with GO Fish, one can not only track allele trajectories in record time, but also generate SFS by using forward simulations in roughly the same time-frame as by solving diffusion equations. Just as importantly, GO Fish achieves the increase in performance without sacrificing flexibility in the evolutionary scenarios that it is capable of simulating.</p><p id="p-30">GO Fish can simulate mutations across multiple populations for comparative population genomics, with no limits to the number of populations allowed. Population size, migration rates, inbreeding, dominance, and mutation rate are all user-specifiable functions capable of varying over time and between different populations. Selection is likewise a user-specifiable function parameterized not only by generation and population, but also by allele frequency, allowing for the modeling of frequency-dependent selection as well as time-dependent and population-specific selection. By tuning the inbreeding and dominance parameters, GO Fish can simulate the full range of single-locus dynamics for both haploids and diploids with everything from outbred to inbred populations and overdominant to underdominant alleles. GPU-accelerated Wright–Fisher simulations thus provide extensive flexibility to model unique and complex demographic and selection scenarios beyond what many current site frequency spectrum analysis methods can employ.</p><p id="p-31">Paired with a coalescent simulator, GO Fish can also accelerate the forward simulation component in forward-backward approaches [see <a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Ewing and Hermisson (2010)</a> and <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Nakagome <em>et al.</em> (2016)</a>]. In addition, GO Fish is able to track the age of mutations in the simulation, providing an estimate of the distribution of the allele ages, or even the age by frequency distribution, for mutations in an observed SFS. Further, the age of mutations is one element of a unique identifier for each mutation in the simulation, which allows the frequency trajectory of individual mutations to be tracked through time. This ability to sample ancestral states and then track the mutations throughout the simulation can be used to contrast the population frequencies of polymorphisms from ancient DNA with those present in modern populations for powerful population genetics analyses (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Bank <em>et al.</em> 2014</a>). By accelerating the single-locus forward simulation on the GPU, GO Fish broadens the capabilities of SFS analysis approaches in population genetic studies.</p><p id="p-32">Across the field of population genetics and evolution, there exist a wide range of computationally intensive problems that could benefit from parallelization. The algorithms presented and discussed in <a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2</a> represent a subset of the essential parallel algorithms, which more complex algorithms modify or build upon. Applications of these parallel algorithms are already wide-ranging in bioinformatics: motif finding (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Ganesan <em>et al.</em> 2010</a>), global and local DNA and protein alignment (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Vouzis and Sahinidis 2011</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Liu <em>et al.</em> 2011</a>, <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">2013</a>; <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Zhao and Chu 2014</a>), short read alignment and SNP calling (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Klus <em>et al.</em> 2012</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Luo <em>et al.</em> 2013</a>), haplotyping and the imputation of genotypes (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Chen <em>et al.</em> 2012</a>), analysis for genome-wide association studies (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Chen 2012</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Song <em>et al.</em> 2013</a>), and mapping phenotype to genotype and epistatic interactions across the genome (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Chen and Guo 2013</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Cebamanos <em>et al.</em> 2014</a>). In molecular evolution, the basic algorithms underlying the building of phylogenetic trees and analyzing sequence divergence between species have likewise been GPU-accelerated (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Suchard and Rambaut 2009</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Kubatko <em>et al.</em> 2016</a>). Further, there are parallel methods for general statistical and computational methods, like Markov Chain Monte Carlo and Bayesian analysis, useful in computational evolution and population genetics (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Suchard <em>et al.</em> 2010</a>; <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Zhou <em>et al.</em> 2015</a>). GO Fish is itself part of the larger Parallel PopGen Package, a planned compendium of tools for accelerating the calculation of many different population genetics statistics on the GPU, including SFS and likelihoods. This larger package is currently under development; the results in <a id="xref-fig-5-5" class="xref-fig" href="#F5">Figure 5</a> make use of a prototype library, Spectrum, to generate SFS statistics from GO Fish simulations on the GPU.</p><p id="p-33">Future work on the single-locus Wright–Fisher algorithm will include extending the parallel structure of GO Fish to allow for multiple alleles as well as multiple mutational events at a site, relaxing one of the key assumptions of the Poisson Random Field (<a id="xref-ref-43-4" class="xref-bibr" href="#ref-43">Sawyer and Hartl 1992</a>). At present, neither running simulations with long divergence times between populations nor any scenario where the number of extant mutations in the simulation rises to too high a proportion of the total number of sites is theoretically consistent with the Poisson Random Field model underpinning the current version of GO Fish. Beyond GO Fish, solving Wright–Fisher diffusion equations in programs like δaδi (<a id="xref-ref-14-8" class="xref-bibr" href="#ref-14">Gutenkunst <em>et al.</em> 2009</a>) can likewise be sped up through parallelization on the GPU (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Lions <em>et al.</em> 2001</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Komatitsch <em>et al.</em> 2009</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Micikevicius 2009</a>; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Tutkun and Edis 2012</a>).</p><p id="p-34">Unfortunately, while the effects of linkage and linked selection across the genome can be mitigated in analyses using a single-locus framework (<a id="xref-ref-14-9" class="xref-bibr" href="#ref-14">Gutenkunst <em>et al.</em> 2009</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Coffman <em>et al.</em> 2016</a>; <a id="xref-ref-34-3" class="xref-bibr" href="#ref-34">Machado <em>et al.</em> 2017</a>), these effects cannot be examined and measured while assuming independence among sites. Expanding from the study of independent loci to modeling the evolution of haplotypes and chromosomes, simulations with the coalescent framework or forward Wright–Fisher algorithm with linkage can also be accelerated on GPUs. The coalescent approach has already been shown to benefit from parallelization over multiple CPU cores [see <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Montemuiño <em>et al.</em> (2014)</a>]. While <a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Montemuiño <em>et al.</em> (2014)</a> achieved their speed boost by running multiple independent simulations concurrently, they noted that parallelizing the coalescent algorithm itself may also accelerate individual simulations over GPUs (<a id="xref-ref-37-3" class="xref-bibr" href="#ref-37">Montemuiño <em>et al.</em> 2014</a>). Likewise, multiple independent runs of the full forward simulation with linkage can be run concurrently over multiple cores and the individual runs might themselves be accelerated by parallelization of the forward algorithm. The forward simulation with linkage has many embarrassingly parallel steps, as well as those that can be refactored into one of the core parallel algorithms. The closely related genetic algorithm, used to solve difficult optimization problems, has already been parallelized and, under many conditions, greatly accelerated on GPUs (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Pospichal <em>et al.</em> 2010</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Hofmann <em>et al.</em> 2013</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Limmer and Fey 2016</a>). However, not all algorithms will benefit from parallelization and execution on GPUs; the real-world performance of any parallelized algorithm will depend on the details of the implementation (<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Hofmann <em>et al.</em> 2013</a>; <a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Limmer and Fey 2016</a>). While the extent of the performance increase will vary from application to application, each of these represent key algorithms whose potential acceleration could provide huge benefits for the field (<a id="xref-ref-3-4" class="xref-bibr" href="#ref-3">Carvajal-Rodriguez 2010</a>; <a id="xref-ref-18-4" class="xref-bibr" href="#ref-18">Hoban <em>et al.</em> 2012</a>).</p><p id="p-35">These potential benefits extend to lowering the cost barrier for students and researchers to run intensive computational analyses in population genetics. The GO Fish results demonstrate how powerful even an older, mobile GPU can be at executing parallel workloads, which means that GO Fish can be run on everything from GPUs in high-end compute clusters to a GPU in a personal laptop and still achieve a great speedup over traditional serial programs. A batch of single-locus Wright–Fisher simulations that might have taken a 100 CPU-hr or more to complete on a cluster can be done, with GO Fish, in 1 hr on a laptop. Moreover, graphics cards and massively parallel processors in general are evolving quickly. While this paper has focused on NVIDIA GPUs and CUDA, the capability to take advantage of the massive parallelization inherent in the Wright–Fisher algorithm is the key to accelerating the simulation and in the high-performance computing market there are several avenues to achieve the performance gains presented here. For instance, OpenCL is another popular low-level language for parallel programming and can be used to program NVIDIA, AMD, Altera, Xilinx, and Intel solutions for massively parallel computation, which include GPUs, CPUs, and even Field Programmable Gate Arrays (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Stone <em>et al.</em> 2010</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Czajkowski <em>et al.</em> 2012</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Jha <em>et al.</em> 2015</a>). The parallel algorithm of GO Fish can be applied to all of these tools. Whichever platform(s) or language(s) researchers choose to utilize, the future of computation in population genetics is massively parallel and exceedingly fast.</p></div><div class="section" id="sec-6"></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-36">The author would like to thank Nandita Garud, Heather Machado, Philipp Messer, Kathleen Nguyen, Sergey Nuzhdin, Peter Ralph, Kevin Thornton, and two anonymous reviewers for providing feedback and helpful suggestions to improve this paper.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-5"><p id="p-37">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300103/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300103/-/DC1</a>.</p></li><li class="fn" id="fn-6"><p id="p-38"><em>Communicating editor: K. Thornton</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2017-05-11"><span class="received-label">Received </span>May 11, 2017.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2017-07-27"><span class="accepted-label">Accepted </span>July 27, 2017.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2017 Lawrie</li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3229.1" data-doi="10.1016/j.tig.2014.09.010"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bank</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ewing</span>  <span class="cit-name-given-names">G. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferrer-Admettla</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Foll</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jensen</span>  <span class="cit-name-given-names">J. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Thinking too positive? revisiting current methods of population genetic selection inference.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">540</span>–<span class="cit-lpage">546</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BGenet.%26rft.volume%253D30%26rft.spage%253D540%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2014.09.010%26rft_id%253Dinfo%253Apmid%252F25438719%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tig.2014.09.010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25438719&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3229.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-confproc" id="cit-7.9.3229.2"><div class="cit-metadata unstructured">Billeter, M., O. Olsson, and U. Assarsson, 2009 Efficient stream compaction on wide SIMD many-core architectures. Association for Computing Machinery Proceedings of the Conference on High Performance Graphics 2009, New Orleans, pp. 159–166.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3229.3" data-doi="10.2174/138920210790218007"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Carvajal-Rodriguez</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Simulation of genes and genomes forward in time.</span> <abbr class="cit-jnl-abbrev">Curr. Genomics</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">58</span>–<span class="cit-lpage">61</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DCurr%2BGenomics%26rft.aulast%253DCarvajal-Rodriguez%26rft.auinit1%253DA.%26rft.volume%253D11%26rft.issue%253D1%26rft.spage%253D58%26rft.epage%253D61%26rft.atitle%253DSimulation%2Bof%2Bgenes%2Band%2Bgenomes%2Bforward%2Bin%2Btime.%26rft_id%253Dinfo%253Adoi%252F10.2174%252F138920210790218007%26rft_id%253Dinfo%253Apmid%252F20808525%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2174/138920210790218007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20808525&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3229.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000273742500009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3229.4" data-doi="10.1093/bioinformatics/btt754"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cebamanos</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gray</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stewart</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tenesa</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Regional heritability advanced complex trait analysis for GPU and traditional parallel architectures.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">1177</span>–<span class="cit-lpage">1179</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtt754%26rft_id%253Dinfo%253Apmid%252F24403537%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btt754&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24403537&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3229.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3229.5" data-doi="10.1093/bioinformatics/bts015"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">G. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A scalable and portable framework for massively parallel variable selection in genetic association studies.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">719</span>–<span class="cit-lpage">720</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbts015%26rft_id%253Dinfo%253Apmid%252F22238272%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bts015&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22238272&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3229.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000300986600019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3229.6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">G. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Discovering epistasis in large scale genetic association studies by exploiting graphics cards.</span> <abbr class="cit-jnl-abbrev">Front. Genet.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">266</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BGenet.%26rft.volume%253D4%26rft.spage%253D266%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3229.7" data-doi="10.1093/bioinformatics/bts536"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">G. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stram</span>  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sobel</span>  <span class="cit-name-given-names">E. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lange</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Mendel-GPU: haplotyping and genotype imputation on graphics processing units.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">2979</span>–<span class="cit-lpage">2980</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbts536%26rft_id%253Dinfo%253Apmid%252F22954633%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bts536&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22954633&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3229.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3229.8" data-doi="10.1093/molbev/msv255"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Coffman</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hsieh</span>  <span class="cit-name-given-names">P. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gravel</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gutenkunst</span>  <span class="cit-name-given-names">R. N.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Computationally efficient composite likelihood statistics for demographic inference.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">591</span>–<span class="cit-lpage">593</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsv255%26rft_id%253Dinfo%253Apmid%252F26545922%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msv255&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26545922&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3229.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-confproc" id="cit-7.9.3229.9"><div class="cit-metadata unstructured">Czajkowski, T. S., U. Aydonat, D. Denisenko, J. Freeman, M. Kinsner <em>et al.</em>, 2012 From OpenCL to high-performance hardware on FPGAs. Institute of Electrical and Electronics Engineers 2012 22nd International Conference on Field Programmable Logic and Applications (FPL), Oslo, Norway, pp. 531–534.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3229.10" data-doi="10.1093/bioinformatics/btq322"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ewing</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hermisson</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">MSMS: a coalescent simulation program including recombination, demographic structure and selection at a single locus.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">2064</span>–<span class="cit-lpage">2065</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq322%26rft_id%253Dinfo%253Apmid%252F20591904%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btq322&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20591904&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3229.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000280703500024&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3229.11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fisher</span>  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1930</span> <span class="cit-article-title">The distribution of gene ratios for rare mutations.</span> <abbr class="cit-jnl-abbrev">Proc. R. Soc. Edinb.</abbr> <span class="cit-vol">50</span>: <span class="cit-fpage">205</span>–<span class="cit-lpage">220</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BR.%2BSoc.%2BEdinb.%26rft.volume%253D50%26rft.spage%253D205%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-confproc" id="cit-7.9.3229.12"><div class="cit-metadata unstructured">Ganesan, N., R. D. Chamberlain, J. Buhler, and M. Taufer, 2010 Accelerating HMMER on GPUs by implementing hybrid data and task parallelism. Association for Computing Machinery Proceedings of the First ACM International Conference on Bioinformatics and Computational Biology, Niagara Falls, pp. 418–421.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3229.13" data-doi="10.1371/journal.pgen.1005004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Garud</span>  <span class="cit-name-given-names">N. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Messer</span>  <span class="cit-name-given-names">P. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buzbas</span>  <span class="cit-name-given-names">E. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Petrov</span>  <span class="cit-name-given-names">D. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Recent selective sweeps in North American Drosophila melanogaster show signatures of soft sweeps.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">e1005004</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D11%26rft.spage%253De1005004%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1005004%26rft_id%253Dinfo%253Apmid%252F25706129%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1005004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25706129&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3229.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3229.14" data-doi="10.1371/journal.pgen.1000695"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gutenkunst</span>  <span class="cit-name-given-names">R. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hernandez</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Williamson</span>  <span class="cit-name-given-names">S. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bustamante</span>  <span class="cit-name-given-names">C. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Inferring the joint demographic history of multiple populations from multidimensional SNP frequency data.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e1000695</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DGutenkunst%26rft.auinit1%253DR.%2BN.%26rft.volume%253D5%26rft.issue%253D10%26rft.spage%253De1000695%26rft.epage%253De1000695%26rft.atitle%253DInferring%2Bthe%2Bjoint%2Bdemographic%2Bhistory%2Bof%2Bmultiple%2Bpopulations%2Bfrom%2Bmultidimensional%2BSNP%2Bfrequency%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000695%26rft_id%253Dinfo%253Apmid%252F19851460%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000695&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19851460&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3229.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-web" id="cit-7.9.3229.15"><div class="cit-metadata unstructured">Harris, M., 2007a Optimizing parallel reduction in CUDA. NVIDIA Developer Technology 2(4). [ONLINE]. Available at: <a href="http://developer.download.nvidia.com/compute/cuda/1.1-Beta/x86_website/projects/reduction/doc/reduction.pdf">http://developer.download.nvidia.com/compute/cuda/1.1-Beta/x86_website/projects/reduction/doc/reduction.pdf</a>. Accessed: August 12, 2017.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3229.16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Harris</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sengupta</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Owens</span>  <span class="cit-name-given-names">J. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span><span class="cit-comment">b</span> <span class="cit-article-title">Parallel prefix sum (scan) with CUDA.</span> <abbr class="cit-jnl-abbrev">GPU Gems</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">851</span>–<span class="cit-lpage">876</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGPU%2BGems%26rft.volume%253D3%26rft.spage%253D851%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3229.17" data-doi="10.1093/bioinformatics/btn522"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hernandez</span>  <span class="cit-name-given-names">R. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">A flexible forward simulator for populations subject to selection and demography.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">2786</span>–<span class="cit-lpage">2787</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtn522%26rft_id%253Dinfo%253Apmid%252F18842601%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btn522&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18842601&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3229.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000261168600023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3229.18"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hoban</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bertorelle</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gaggiotti</span>  <span class="cit-name-given-names">O. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Computer simulations: tools for population and evolutionary genetics.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">110</span>–<span class="cit-lpage">122</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BGenet.%26rft.volume%253D13%26rft.spage%253D110%26rft_id%253Dinfo%253Apmid%252F22230817%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=22230817&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3229.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3229.19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hofmann</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Limmer</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fey</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Performance investigations of genetic algorithms on graphics cards.</span> <abbr class="cit-jnl-abbrev">Swarm Evol. Comput.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">33</span>–<span class="cit-lpage">47</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSwarm%2BEvol.%2BComput.%26rft.volume%253D12%26rft.spage%253D33%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3229.20" data-doi="10.1093/bioinformatics/18.2.337"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hudson</span>  <span class="cit-name-given-names">R. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Generating samples under a wright-fisher neutral model of genetic variation.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">337</span>–<span class="cit-lpage">338</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DHudson%26rft.auinit1%253DR.%2BR.%26rft.volume%253D18%26rft.issue%253D2%26rft.spage%253D337%26rft.epage%253D338%26rft.atitle%253DGenerating%2Bsamples%2Bunder%2Ba%2BWright-Fisher%2Bneutral%2Bmodel%2Bof%2B%2Bgenetic%2Bvariation%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252F18.2.337%26rft_id%253Dinfo%253Apmid%252F11847089%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/18.2.337&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11847089&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3229.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000174028100021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3229.21"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jackson</span>  <span class="cit-name-given-names">B. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Campos</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haddrill</span>  <span class="cit-name-given-names">P. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zeng</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Variation in the intensity of selection on codon bias over time causes contrasting patterns of base composition evolution in drosophila.</span> <abbr class="cit-jnl-abbrev">Genome Biol. Evol.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">102</span>–<span class="cit-lpage">123</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%2BEvol.%26rft.volume%253D9%26rft.spage%253D102%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3229.22"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jha</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">He</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheng</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huynh</span>  <span class="cit-name-given-names">H. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Improving main memory hash joins on intel xeon phi processors: an experimental approach.</span> <abbr class="cit-jnl-abbrev">VLDB Endow.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">642</span>–<span class="cit-lpage">653</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVLDB%2BEndow.%26rft.volume%253D8%26rft.spage%253D642%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-web" id="cit-7.9.3229.23"><div class="cit-metadata unstructured">Kim, B. Y., C. D. Huber, and K. E. Lohmueller, 2017 Inference of the distribution of selection coefficients for new nonsynonymous mutations using large samples. Genetics DOI: <a href="https://doi.org/10.1534/genetics.116.197145">https://doi.org/10.1534/genetics.116.197145</a>.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3229.24" data-doi="10.2307/3211856"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kimura</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1964</span> <span class="cit-article-title">Diffusion models in population genetics.</span> <abbr class="cit-jnl-abbrev">J. Appl. Probab.</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">177</span>–<span class="cit-lpage">232</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAppl.%2BProbab.%26rft.volume%253D1%26rft.spage%253D177%26rft_id%253Dinfo%253Adoi%252F10.2307%252F3211856%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/3211856&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3229.25" data-doi="10.1186/1756-0500-5-27"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Klus</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lam</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lyberg</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheung</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pullan</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">BarraCUDA - a fast short read sequence aligner using graphics processing units.</span> <abbr class="cit-jnl-abbrev">BMC Res. Notes</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">27</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DBMC%2BRes%2BNotes%26rft.aulast%253DKlus%26rft.auinit1%253DP.%26rft.volume%253D5%26rft.spage%253D27%26rft.epage%253D27%26rft.atitle%253DBarraCUDA%2B-%2Ba%2Bfast%2Bshort%2Bread%2Bsequence%2Baligner%2Busing%2Bgraphics%2Bprocessing%2Bunits.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1756-0500-5-27%26rft_id%253Dinfo%253Apmid%252F22244497%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1756-0500-5-27&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22244497&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3229.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3229.26" data-doi="10.1534/g3.117.039651"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Koch</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Novembre</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">A temporal perspective on the interplay of demography and selection on deleterious variation in humans.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">1027</span>–<span class="cit-lpage">1037</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.117.039651%26rft_id%253Dinfo%253Apmid%252F28159863%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjg6IjcvMy8xMDI3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy85LzMyMjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3229.27" data-doi="10.1016/j.jpdc.2009.01.006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Komatitsch</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Michéa</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Erlebacher</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Porting a high-order finite-element earthquake modeling application to NVIDIA graphics cards using CUDA.</span> <abbr class="cit-jnl-abbrev">J. Parallel Distrib. Comput.</abbr> <span class="cit-vol">69</span>: <span class="cit-fpage">451</span>–<span class="cit-lpage">460</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BParallel%2BDistrib.%2BComput.%26rft.volume%253D69%26rft.spage%253D451%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jpdc.2009.01.006%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jpdc.2009.01.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3229.28"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kubatko</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shah</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Herbei</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gilchrist</span>  <span class="cit-name-given-names">M. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">A codon model of nucleotide substitution with selection on synonymous codon usage.</span> <abbr class="cit-jnl-abbrev">Mol. Phylogenet. Evol.</abbr> <span class="cit-vol">94</span>: <span class="cit-fpage">290</span>–<span class="cit-lpage">297</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BPhylogenet.%2BEvol.%26rft.volume%253D94%26rft.spage%253D290%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3229.29"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Limmer</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fey</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Comparison of common parallel architectures for the execution of the island model and the global parallelization of evolutionary algorithms.</span> <abbr class="cit-jnl-abbrev">Concurr. Comput.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">e3797</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DConcurr.%2BComput.%26rft.volume%253D29%26rft.spage%253De3797%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3229.30"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lions</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maday</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Turinici</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">A “parareal” in time discretization of PDE’s.</span> <abbr class="cit-jnl-abbrev">Comptes Rendus De L’Academie Des Sciences Series I Mathematics</abbr> <span class="cit-vol">332</span>: <span class="cit-fpage">661</span>–<span class="cit-lpage">668</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DComptes%2BRendus%2BDe%2BL%253FAcademie%2BDes%2BSciences%2BSeries%2BI%2BMathematics%26rft.volume%253D332%26rft.spage%253D661%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3229.31" data-doi="10.1109/TCBB.2011.33"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schmidt</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Muller-Wittig</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">CUDA-BLASTP: accelerating BLASTP on CUDA-enabled graphics hardware.</span> <abbr class="cit-jnl-abbrev">IEEE/ACM Trans. Comput. Biol. Bioinform.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">1678</span>–<span class="cit-lpage">1684</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DIEEE%252FACM%2BTrans%2BComput%2BBiol%2BBioinform%26rft.aulast%253DLiu%26rft.auinit1%253DW.%26rft.volume%253D8%26rft.issue%253D6%26rft.spage%253D1678%26rft.epage%253D1684%26rft.atitle%253DCUDA-BLASTP%253A%2Baccelerating%2BBLASTP%2Bon%2BCUDA-enabled%2Bgraphics%2Bhardware.%26rft_id%253Dinfo%253Adoi%252F10.1109%252FTCBB.2011.33%26rft_id%253Dinfo%253Apmid%252F21339531%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1109/TCBB.2011.33&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21339531&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3229.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3229.32" data-doi="10.1186/1471-2105-14-117"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wirawan</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schmidt</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">CUDASW++ 3.0: accelerating smith-waterman protein database search by coupling CPU and GPU SIMD instructions.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">117</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BBioinformatics%26rft.volume%253D14%26rft.spage%253D117%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-14-117%26rft_id%253Dinfo%253Apmid%252F23557111%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-14-117&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23557111&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3229.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3229.33" data-doi="10.1371/journal.pone.0065632"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Luo</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wong</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">SOAP3-dp: fast, accurate and sensitive GPU-based short read aligner.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e65632</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D8%26rft.spage%253De65632%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0065632%26rft_id%253Dinfo%253Apmid%252F23741504%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0065632&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23741504&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3229.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3229.34"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Machado</span>  <span class="cit-name-given-names">H. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lawrie</span>  <span class="cit-name-given-names">D. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Petrov</span>  <span class="cit-name-given-names">D. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Strong selection at the level of codon usage bias: evidence against the Li-Bulmer model.</span> <abbr class="cit-jnl-abbrev">bioRxiv</abbr> DOI: https://doi.org/10.1101/106476.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3229.35" data-doi="10.1534/genetics.113.152181"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Messer</span>  <span class="cit-name-given-names">P. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">SLiM: simulating evolution with selection and linkage.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">194</span>: <span class="cit-fpage">1037</span>–<span class="cit-lpage">1039</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.152181%26rft_id%253Dinfo%253Apmid%252F23709637%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5NC80LzEwMzciO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzkvMzIyOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-confproc" id="cit-7.9.3229.36"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Micikevicius</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> 3D finite difference computation on GPUs using CUDA. Association for Computing Machinery Proceedings of 2nd Workshop on General Purpose Processing on Graphics Processing Units, pp. <span class="cit-fpage">79</span>–<span class="cit-lpage">84</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-other" id="cit-7.9.3229.37"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Montemuiño</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Espinosa</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moure</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vera-Rodríguez</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ramos-Onsins</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> msPar: a parallel coalescent simulator. Springer Euro-Par 2013: Parallel Processing Workshops, pp. <span class="cit-fpage">321</span>–<span class="cit-lpage">330</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3229.38" data-doi="10.1007/BF00275916"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nagylaki</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">1980</span> <span class="cit-article-title">The strong-migration limit in geographically structured populations.</span> <abbr class="cit-jnl-abbrev">J. Math. Biol.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">101</span>–<span class="cit-lpage">114</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmathematical%2Bbiology%26rft.stitle%253DJ%2BMath%2BBiol%26rft.aulast%253DNagylaki%26rft.auinit1%253DT.%26rft.volume%253D9%26rft.issue%253D2%26rft.spage%253D101%26rft.epage%253D114%26rft.atitle%253DThe%2Bstrong-migration%2Blimit%2Bin%2Bgeographically%2Bstructured%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00275916%26rft_id%253Dinfo%253Apmid%252F7365330%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00275916&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7365330&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3229.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1980JR08600001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3229.39" data-doi="10.1093/molbev/msv256"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nakagome</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alkorta-Aranburu</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amato</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Howie</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peter</span>  <span class="cit-name-given-names">B. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Estimating the ages of selection signals from different epochs in human history.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">657</span>–<span class="cit-lpage">669</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsv256%26rft_id%253Dinfo%253Apmid%252F26545921%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msv256&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26545921&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3229.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3229.40"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nickolls</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buck</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garland</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Skadron</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Scalable parallel programming with CUDA.</span> <abbr class="cit-jnl-abbrev">Queue</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">40</span>–<span class="cit-lpage">53</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DQueue%26rft.volume%253D6%26rft.spage%253D40%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3229.41" data-doi="10.1093/bioinformatics/btw478"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ortega-Del Vecchyo</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marsden</span>  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lohmueller</span>  <span class="cit-name-given-names">K. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">PReFerSim: fast simulation of demography and selection under the poisson random field model.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">3516</span>–<span class="cit-lpage">3518</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtw478%26rft_id%253Dinfo%253Apmid%252F27436562%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btw478&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27436562&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3229.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-book" id="cit-7.9.3229.42"><div class="cit-metadata unstructured">Pospichal, P., J. Jaros, and J. Schwarz, 2010 Parallel genetic algorithm on the CUDA architecture, pp. 442–451 in <em>Applications of Evolutionary Computation</em>, <em>EvoApplications 2010</em>, edited by C. Di Chio Lecture Notes in Computer Science. Springer, Berlin.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3229.43"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sawyer</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hartl</span>  <span class="cit-name-given-names">D. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">Population genetics of polymorphism and divergence.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">132</span>: <span class="cit-fpage">1161</span>–<span class="cit-lpage">1176</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DSawyer%26rft.auinit1%253DS.%2BA.%26rft.volume%253D132%26rft.issue%253D4%26rft.spage%253D1161%26rft.epage%253D1176%26rft.atitle%253DPopulation%2Bgenetics%2Bof%2Bpolymorphism%2Band%2Bdivergence.%26rft_id%253Dinfo%253Apmid%252F1459433%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjEzMi80LzExNjEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzkvMzIyOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3229.44"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">G. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Millikan</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ambrosone</span>  <span class="cit-name-given-names">C. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">John</span>  <span class="cit-name-given-names">E. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">A genome-wide scan for breast cancer risk haplotypes among African American women.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e57298</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D8%26rft.spage%253De57298%26rft_id%253Dinfo%253Apmid%252F23468962%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=23468962&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3229.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3229.45"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stone</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gohara</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shi</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">OpenCL: a parallel programming standard for heterogeneous computing systems.</span> <abbr class="cit-jnl-abbrev">Comput. Sci. Eng.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">66</span>–<span class="cit-lpage">73</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DComput%2BSci%2BEng%26rft.aulast%253DStone%26rft.auinit1%253DJ.%2BE.%26rft.volume%253D12%26rft.issue%253D3%26rft.spage%253D66%26rft.epage%253D72%26rft.atitle%253DOpenCL%253A%2BA%2BParallel%2BProgramming%2BStandard%2Bfor%2BHeterogeneous%2BComputing%2BSystems.%26rft_id%253Dinfo%253Apmid%252F21037981%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=21037981&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3229.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3229.46" data-doi="10.1093/bioinformatics/btp244"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Suchard</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rambaut</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Many-core algorithms for statistical phylogenetics.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1370</span>–<span class="cit-lpage">1376</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp244%26rft_id%253Dinfo%253Apmid%252F19369496%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp244&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19369496&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3229.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000266109500005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3229.47" data-doi="10.1198/jcgs.2010.10016"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Suchard</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chan</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Frelinger</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cron</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Understanding GPU programming for statistical computation: studies in massively parallel massive mixtures.</span> <abbr class="cit-jnl-abbrev">J. Comput. Graph. Stat.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">419</span>–<span class="cit-lpage">438</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DJ%2BComput%2BGraph%2BStat%26rft.aulast%253DSuchard%26rft.auinit1%253DM.%2BA.%26rft.volume%253D19%26rft.issue%253D2%26rft.spage%253D419%26rft.epage%253D438%26rft.atitle%253DUnderstanding%2BGPU%2BProgramming%2Bfor%2BStatistical%2BComputation%253A%2BStudies%2Bin%2BMassively%2BParallel%2BMassive%2BMixtures.%26rft_id%253Dinfo%253Adoi%252F10.1198%252Fjcgs.2010.10016%26rft_id%253Dinfo%253Apmid%252F20877443%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1198/jcgs.2010.10016&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20877443&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3229.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3229.48" data-doi="10.1534/genetics.114.165019"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thornton</span>  <span class="cit-name-given-names">K. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A C++ template library for efficient forward-time population genetic simulation of large populations.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">198</span>: <span class="cit-fpage">157</span>–<span class="cit-lpage">166</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.165019%26rft_id%253Dinfo%253Apmid%252F24950894%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk4LzEvMTU3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy85LzMyMjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3229.49"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tutkun</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Edis</span>  <span class="cit-name-given-names">F. O.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A GPU application for high-order compact finite difference scheme.</span> <abbr class="cit-jnl-abbrev">Comput. Fluids</abbr> <span class="cit-vol">55</span>: <span class="cit-fpage">29</span>–<span class="cit-lpage">35</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DComput.%2BFluids%26rft.volume%253D55%26rft.spage%253D29%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3229.50" data-doi="10.1093/bioinformatics/btq644"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vouzis</span>  <span class="cit-name-given-names">P. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sahinidis</span>  <span class="cit-name-given-names">N. V.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">GPU-BLAST: using graphics processors to accelerate protein sequence alignment.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">182</span>–<span class="cit-lpage">188</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq644%26rft_id%253Dinfo%253Apmid%252F21088027%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btq644&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21088027&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3229.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000286215200006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3229.51" data-doi="10.1534/genetics.103.024745"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Williamson</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fledel-Alon</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bustamante</span>  <span class="cit-name-given-names">C. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Population genetics of polymorphism and divergence for diploid selection models with arbitrary dominance.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">168</span>: <span class="cit-fpage">463</span>–<span class="cit-lpage">475</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DWilliamson%26rft.auinit1%253DS.%26rft.volume%253D168%26rft.issue%253D1%26rft.spage%253D463%26rft.epage%253D475%26rft.atitle%253DPopulation%2BGenetics%2Bof%2BPolymorphism%2Band%2BDivergence%2Bfor%2BDiploid%2BSelection%2BModels%2BWith%2BArbitrary%2BDominance%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.103.024745%26rft_id%253Dinfo%253Apmid%252F15454557%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTY4LzEvNDYzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy85LzMyMjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3229.52" data-doi="10.1073/pnas.24.7.253"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1938</span> <span class="cit-article-title">The distribution of gene frequencies under irreversible mutation.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">253</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DWright%26rft.auinit1%253DS.%26rft.volume%253D24%26rft.issue%253D7%26rft.spage%253D253%26rft.epage%253D259%26rft.atitle%253DThe%2BDistribution%2Bof%2BGene%2BFrequencies%2BUnder%2BIrreversible%2BMutation%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.24.7.253%26rft_id%253Dinfo%253Apmid%252F16577841%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo4OiIyNC83LzI1MyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvOS8zMjI5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3229.53" data-doi="10.1093/bioinformatics/btu047"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chu</span>  <span class="cit-name-given-names">X.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">G-BLASTN: accelerating nucleotide alignment by graphics processors.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">1384</span>–<span class="cit-lpage">1391</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu047%26rft_id%253Dinfo%253Apmid%252F24463183%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btu047&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24463183&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3229.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3229.54"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">gPGA: GPU accelerated population genetics analyses.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">e0135028</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D10%26rft.spage%253De0135028%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/7/9/3229.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/9/3217" title="Trans-ancestry Fine Mapping and Molecular Assays Identify Regulatory Variants at the ANGPTL8 HDL-C GWAS Locus" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/10/3237" title="Meeting Report on Experimental Approaches to Evolution and Ecology Using Yeast and Other Model Systems" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/7/9"><span>Volume 7 Issue 9, September 2017</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/7/9.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/7/9.cover-source.jpg?itok=WuYpGT8r" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 7 (9)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/9/3229.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-wza3crAicoJJ-O2ymJU-ROnlm_JHJm-nRHjm8MDKTyE" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/7/9/3229" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/7/9/3229" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/7/9/3229" class="active" data-icon-position="" data-hide-link-title="0">Accelerating Wright–Fisher Forward Simulations on the Graphics Processing Unit</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/486371" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-gOQcMZDMrZJOmsmcCIp_HJonFNjaqUxHJ3oGmJBSHaI" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/9/3229.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/7/9/3229" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-p93u3_3D6HvtaqRJF81SICwegG6Fv-xW0jjuzny5c_o" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/7/9/3229" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/9/3229.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="486371" id="node486371--2" data-pisa="ggg;7/9/3229" data-pisa-master="ggg;g3.117.300103" data-apath="/ggg/7/9/3229.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Accelerating Wright–Fisher Forward Simulations on the Graphics Processing Unit</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-6782-9986" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>David S. Lawrie</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3229-3236;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300103 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node486371--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">David S. Lawrie </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Los Angeles, California 90034</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BS.%2BLawrie%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lawrie%20DS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADavid%2BS.%2BLawrie%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-6782-9986" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for David S. Lawrie</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:dlawrie@alumni.stanford.edu" class="" data-icon-position="" data-hide-link-title="0">dlawrie@alumni.stanford.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/7/9/3229" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/7/9/3229&amp;title=Accelerating%20Wright%E2%80%93Fisher%20Forward%20Simulations%20on%20the%20Graphics%20Processing%20Unit" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/7/9/3229&amp;title=Accelerating%20Wright%E2%80%93Fisher%20Forward%20Simulations%20on%20the%20Graphics%20Processing%20Unit" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/7/9/3229&amp;title=Accelerating%20Wright%E2%80%93Fisher%20Forward%20Simulations%20on%20the%20Graphics%20Processing%20Unit" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/7/9/3229&amp;text=Accelerating%20Wright%E2%80%93Fisher%20Forward%20Simulations%20on%20the%20Graphics%20Processing%20Unit" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/7/9/3229&amp;title=Accelerating%20Wright%E2%80%93Fisher%20Forward%20Simulations%20on%20the%20Graphics%20Processing%20Unit" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/7/9/3229&amp;t=Accelerating%20Wright%E2%80%93Fisher%20Forward%20Simulations%20on%20the%20Graphics%20Processing%20Unit" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/7/9/3229&amp;title=Accelerating%20Wright%E2%80%93Fisher%20Forward%20Simulations%20on%20the%20Graphics%20Processing%20Unit" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/7/9/3229&amp;title=Accelerating%20Wright%E2%80%93Fisher%20Forward%20Simulations%20on%20the%20Graphics%20Processing%20Unit" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/486371/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="486371" id="node486371--3" data-pisa="ggg;7/9/3229" data-pisa-master="ggg;g3.117.300103" data-apath="/ggg/7/9/3229.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Accelerating Wright–Fisher Forward Simulations on the Graphics Processing Unit</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-6782-9986" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>David S. Lawrie</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3229-3236;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300103 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node486371--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">David S. Lawrie </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Los Angeles, California 90034</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BS.%2BLawrie%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lawrie%20DS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADavid%2BS.%2BLawrie%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-6782-9986" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for David S. Lawrie</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:dlawrie@alumni.stanford.edu" class="" data-icon-position="" data-hide-link-title="0">dlawrie@alumni.stanford.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=3229&amp;rft.epage=3236&amp;rft.atitle=Accelerating%20Wright%E2%80%93Fisher%20Forward%20Simulations%20on%20the%20Graphics%20Processing%20Unit&amp;rft.volume=7&amp;rft.issue=9&amp;rft.date=2017-09-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Lawrie%2C+David+S."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/486371/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/486371/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/486371/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/486371/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/486371/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/486371/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/486371/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/486371/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/486371/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/486371/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/486371/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/486371/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/7/9/3229" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Methods</a></li><li class="odd"><a href="#sec-5" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results and Discussion</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/7/9/3229.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/7/9/3229.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/7/9/3229.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/7/9/3229.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
