<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/7/9/3185" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/7/9/3185.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/7/9/3185.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/7/9/3185.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/7/9/3185/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/7/9/3185.atom" />
<meta name="HW.pisa" content="ggg;7/9/3185" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Ras/MAPK Modifier Loci Revealed by eQTL in Caenorhabditis elegans" />
<meta name="DC.Identifier" content="10.1534/g3.117.1120" />
<meta name="DC.Date" content="2017-09-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2017 Sterken et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="The oncogenic Ras/MAPK pathway is evolutionarily conserved across metazoans. Yet, almost all our knowledge on this pathway comes from studies using single genetic backgrounds, whereas mutational effects can be highly background dependent. Therefore, we lack insight in the interplay between genetic backgrounds and the Ras/MAPK-signaling pathway. Here, we used a Caenorhabditis elegans RIL population containing a gain-of-function mutation in the Ras/MAPK-pathway gene [let-60][1] and measured how gene expression regulation is affected by this mutation. We mapped eQTL and found that the majority (∼73%) of the 1516 detected cis -eQTL were not specific for the [let-60][1] mutation, whereas most (∼76%) of the 898 detected trans -eQTL were associated with the [let-60][1] mutation. We detected six eQTL trans -bands specific for the interaction between the genetic background and the mutation, one of which colocalized with the polymorphic Ras/MAPK modifier [amx-2][2] . Comparison between transgenic lines expressing allelic variants of [amx-2][2] showed the involvement of [amx-2][2] in 79% of the trans -eQTL for genes mapping to this trans -band. Together, our results have revealed hidden loci affecting Ras/MAPK signaling using sensitized backgrounds in C. elegans . These loci harbor putative polymorphic modifier genes that would not have been detected using mutant screens in single genetic backgrounds.

 [1]: http://www.wormbase.org/db/get?name=WBGene00002335;class=Gene
 [2]: http://www.wormbase.org/db/get?name=WBGene00000138;class=Gene" />
<meta name="DC.Contributor" content="Mark G. Sterken" />
<meta name="DC.Contributor" content="Linda van Bemmelen van der Plaat" />
<meta name="DC.Contributor" content="Joost A. G. Riksen" />
<meta name="DC.Contributor" content="Miriam Rodriguez" />
<meta name="DC.Contributor" content="Tobias Schmid" />
<meta name="DC.Contributor" content="Alex Hajnal" />
<meta name="DC.Contributor" content="Jan E. Kammenga" />
<meta name="DC.Contributor" content="Basten L. Snoek" />
<meta name="article:published_time" content="2017-09-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Ras/MAPK Modifier Loci Revealed by eQTL in Caenorhabditis elegans" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;The oncogenic Ras/MAPK pathway is evolutionarily conserved across metazoans. Yet, almost all our knowledge on this pathway comes from studies using single genetic backgrounds, whereas mutational effects can be highly background dependent. Therefore, we lack insight in the interplay between genetic backgrounds and the Ras/MAPK-signaling pathway. Here, we used a &lt;i&gt;Caenorhabditis elegans&lt;/i&gt; RIL population containing a gain-of-function mutation in the Ras/MAPK-pathway gene &lt;i&gt;let-60&lt;/i&gt; and measured how gene expression regulation is affected by this mutation. We mapped eQTL and found that the majority (∼73%) of the 1516 detected &lt;i&gt;cis&lt;/i&gt;-eQTL were not specific for the &lt;i&gt;let-60&lt;/i&gt; mutation, whereas most (∼76%) of the 898 detected &lt;i&gt;trans&lt;/i&gt;-eQTL were associated with the &lt;i&gt;let-60&lt;/i&gt; mutation. We detected six eQTL &lt;i&gt;trans&lt;/i&gt;-bands specific for the interaction between the genetic background and the mutation, one of which colocalized with the polymorphic Ras/MAPK modifier &lt;i&gt;amx-2&lt;/i&gt;. Comparison between transgenic lines expressing allelic variants of &lt;i&gt;amx-2&lt;/i&gt; showed the involvement of &lt;i&gt;amx-2&lt;/i&gt; in 79% of the &lt;i&gt;trans&lt;/i&gt;-eQTL for genes mapping to this &lt;i&gt;trans&lt;/i&gt;-band. Together, our results have revealed hidden loci affecting Ras/MAPK signaling using sensitized backgrounds in &lt;i&gt;C. elegans&lt;/i&gt;. These loci harbor putative polymorphic modifier genes that would not have been detected using mutant screens in single genetic backgrounds.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2017/09/01" />
<meta name="citation_mjid" content="ggg;7/9/3185" />
<meta name="citation_id" content="7/9/3185" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/7/9/3185" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/7/9/3185.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/7/9/3185.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/7/9/3185.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.117.1120" />
<meta name="citation_pmid" content="28751501" />
<meta name="citation_volume" content="7" />
<meta name="citation_issue" content="9" />
<meta name="citation_num_pages" content="9" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="3185" />
<meta name="citation_lastpage" content="3193" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Mark G. Sterken" />
<meta name="citation_author_institution" content="Laboratory of Nematology, Wageningen University and Research, 6708PB, The Netherlands" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-7119-6213" />
<meta name="citation_author" content="Linda van Bemmelen van der Plaat" />
<meta name="citation_author_institution" content="Laboratory of Nematology, Wageningen University and Research, 6708PB, The Netherlands" />
<meta name="citation_author" content="Joost A. G. Riksen" />
<meta name="citation_author_institution" content="Laboratory of Nematology, Wageningen University and Research, 6708PB, The Netherlands" />
<meta name="citation_author" content="Miriam Rodriguez" />
<meta name="citation_author_institution" content="Laboratory of Nematology, Wageningen University and Research, 6708PB, The Netherlands" />
<meta name="citation_author" content="Tobias Schmid" />
<meta name="citation_author_institution" content="Institute of Molecular Life Sciences, University of Zurich, CH-8057, Switzerland" />
<meta name="citation_author_institution" content="Ph.D. Program in Molecular Life Sciences, University of Zurich, CH-8057, Switzerland" />
<meta name="citation_author" content="Alex Hajnal" />
<meta name="citation_author_institution" content="Institute of Molecular Life Sciences, University of Zurich, CH-8057, Switzerland" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-4098-3721" />
<meta name="citation_author" content="Jan E. Kammenga" />
<meta name="citation_author_institution" content="Laboratory of Nematology, Wageningen University and Research, 6708PB, The Netherlands" />
<meta name="citation_author_email" content="jan.kammenga@wur.nl" />
<meta name="citation_author_email" content="l.b.snoek@uu.nl" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-4822-4436" />
<meta name="citation_author" content="Basten L. Snoek" />
<meta name="citation_author_institution" content="Laboratory of Nematology, Wageningen University and Research, 6708PB, The Netherlands" />
<meta name="citation_author_email" content="jan.kammenga@wur.nl" />
<meta name="citation_author_email" content="l.b.snoek@uu.nl" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-5321-2996" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=R. Alberts;citation_author=P. Terpstra;citation_author=Y. Li;citation_author=R. Breitling;citation_author=J. P. Nap;citation_title=Sequence polymorphisms cause many false cis eQTLs.;citation_pages=e622-e622;citation_volume=2;citation_year=2007;citation_issue=7;citation_pmid=17637838;citation_doi=10.1371/journal.pone.0000622" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=G. J. Beitel;citation_author=S. G. Clark;citation_author=H. R. Horvitz;citation_title=Caenorhabditis elegans ras gene let-60 acts as a switch in the pathway of vulval induction.;citation_pages=503-509;citation_volume=348;citation_year=1990;citation_issue=6301;citation_pmid=2123303;citation_doi=10.1038/348503a0" />
<meta name="citation_reference" content="citation_journal_title=Ann. Stat.;citation_author=Y. Benjamini;citation_author=D. Yekutieli;citation_title=The control of the false discovery rate in multiple testing under dependency.;citation_pages=1165-1188;citation_volume=29;citation_year=2001;citation_pmid=17909820;citation_doi=10.1214/aos/1013699998" />
<meta name="citation_reference" content="citation_journal_title=Journal of biology (Online);citation_journal_abbrev=J Biol;citation_author=A. B. Byrne;citation_author=M. T. Weirauch;citation_author=V. Wong;citation_author=M. Koeva;citation_author=S. J. Dixon;citation_title=A global analysis of genetic interactions in Caenorhabditis elegans.;citation_pages=8-8;citation_volume=6;citation_year=2007;citation_issue=3;citation_pmid=17897480;citation_doi=10.1186/jbiol58" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=C. H. Chandler;citation_author=S. Chari;citation_author=D. Tack;citation_author=I. Dworkin;citation_title=Causes and consequences of genetic background effects illuminated by integrative genomic analysis.;citation_volume=196;citation_year=2014;citation_pmid=24504186;citation_doi=10.1534/genetics.113.159426" />
<meta name="citation_reference" content="citation_journal_title=Development;citation_journal_abbrev=Development;citation_author=Q. Ch’ng;citation_author=C. Kenyon;citation_title=egl-27 generates anteroposterior patterns of cell fusion in C. elegans by regulating Hox gene expression and Hox protein function;citation_pages=3303-3312;citation_volume=126;citation_year=1999;citation_issue=15;citation_pmid=10393110" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=A. Cho;citation_author=J. Shin;citation_author=S. Hwang;citation_author=C. Kim;citation_author=H. Shim;citation_title=WormNet v3: a network-assisted hypothesis-generating server for Caenorhabditis elegans.;citation_volume=42;citation_year=2014;citation_pmid=24813450;citation_doi=10.1093/nar/gku367" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=S. Cowley;citation_author=H. Paterson;citation_author=P. Kemp;citation_author=C. J. Marshall;citation_title=Activation of MAP kinase kinase is necessary and sufficient for PC12 differentiation and for transformation of NIH 3T3 cells.;citation_pages=841-852;citation_volume=77;citation_year=1994;citation_issue=6;citation_pmid=7911739;citation_doi=10.1016/0092-8674(94)90133-3" />
<meta name="citation_reference" content="citation_journal_title=PLoS biology;citation_journal_abbrev=PLoS Biol;citation_author=F. Duveau;citation_author=M. A. Felix;citation_title=Role of pleiotropy in the evolution of a cryptic developmental variation in Caenorhabditis elegans.;citation_pages=e1001230-e1001230;citation_volume=10;citation_year=2012;citation_issue=1;citation_pmid=22235190;citation_doi=10.1371/journal.pbio.1001230" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=M. Elvin;citation_author=L. B. Snoek;citation_author=M. Frejno;citation_author=U. Klemstein;citation_author=J. E. Kammenga;citation_title=A fitness assay for comparing RNAi effects across multiple C. elegans genotypes.;citation_pages=510-510;citation_volume=12;citation_year=2011;citation_pmid=22004469;citation_doi=10.1186/1471-2164-12-510" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=M. Francesconi;citation_author=B. Lehner;citation_title=The effects of genetic variation on gene expression dynamics during development.;citation_pages=208-211;citation_volume=505;citation_year=2014;citation_pmid=24270809;citation_doi=10.1038/nature12772" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=M. B. Gerstein;citation_author=Z. J. Lu;citation_author=E. L. Van Nostrand;citation_author=C. Cheng;citation_author=B. I. Arshinoff;citation_title=Integrative analysis of the Caenorhabditis elegans genome by the modENCODE project.;citation_volume=330;citation_year=2010;citation_pmid=21177976;citation_doi=10.1126/science.1196914" />
<meta name="citation_reference" content="citation_journal_title=Wiley Interdiscip. Rev. Dev. Biol.;citation_author=R. H. Gokhale;citation_author=A. W. Shingleton;citation_title=Size control: the developmental physiology of body and organ size regulation.;citation_pages=335-356;citation_volume=4;citation_year=2015;citation_pmid=25808999;citation_doi=10.1002/wdev.181" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=M. Han;citation_author=P. W. Sternberg;citation_title=let-60, a gene that specifies cell fates during C. elegans vulval induction, encodes a ras protein.;citation_pages=921-931;citation_volume=63;citation_year=1990;citation_issue=5;citation_pmid=2257629;citation_doi=10.1016/0092-8674(90)90495-Z" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=T. W. Harris;citation_author=J. Baran;citation_author=T. Bieri;citation_author=A. Cabunoc;citation_author=J. Chan;citation_title=WormBase 2014: new views of curated biology.;citation_volume=42;citation_year=2014;citation_pmid=24194605;citation_doi=10.1093/nar/gkt1063" />
<meta name="citation_reference" content="citation_journal_title=Development;citation_journal_abbrev=Development;citation_author=M. A. Herman;citation_author=Q. Ch’ng;citation_author=S. M. Hettenbach;citation_author=T. M. Ratliff;citation_author=C. Kenyon;citation_title=EGL-27 is similar to a metastasis-associated factor and controls cell polarity and cell migration in C. elegans;citation_pages=1055-1064;citation_volume=126;citation_year=1999;citation_issue=5;citation_pmid=9927605" />
<meta name="citation_reference" content="citation_journal_title=Trends in genetics :  TIG;citation_journal_abbrev=Trends Genet;citation_author=R. C. Jansen;citation_author=J. P. Nap;citation_title=Genetical genomics: the added value from segregation.;citation_pages=388-391;citation_volume=17;citation_year=2001;citation_issue=7;citation_pmid=11418218;citation_doi=10.1016/S0168-9525(01)02310-1" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=H. S. Jiang;citation_author=Y. C. Wu;citation_title=LIN-3/EGF promotes the programmed cell death of specific cells in Caenorhabditis elegans by transcriptional activation of the pro-apoptotic gene egl-1.;citation_pages=e1004513;citation_volume=10;citation_year=2014;citation_pmid=25144461;citation_doi=10.1371/journal.pgen.1004513" />
<meta name="citation_reference" content="citation_journal_title=FEBS J.;citation_author=J. E. Kammenga;citation_title=The background puzzle: how identical mutations in the same gene lead to different disease symptoms.;citation_year=2017;citation_doi=10.1111/febs.14080." />
<meta name="citation_reference" content="citation_journal_title=Trends in genetics :  TIG;citation_journal_abbrev=Trends Genet;citation_author=J. E. Kammenga;citation_author=P. C. Phillips;citation_author=M. De Bono;citation_author=A. Doroszuk;citation_title=Beyond induced mutants: using worms to study natural variation in genetic pathways.;citation_pages=178-185;citation_volume=24;citation_year=2008;citation_issue=4;citation_pmid=18325626;citation_doi=10.1016/j.tig.2008.01.001" />
<meta name="citation_reference" content="citation_journal_title=Mol. Cell. Oncol.;citation_author=E. Kyriakakis;citation_author=M. Markaki;citation_author=N. Tavernarakis;citation_title=Caenorhabditis elegans as a model for cancer research.;citation_pages=975027e;citation_volume=2;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=I. Lee;citation_author=B. Lehner;citation_author=T. Vavouri;citation_author=J. Shin;citation_author=A. G. Fraser;citation_title=Predicting genetic modifier loci using functional gene networks.;citation_volume=20;citation_year=2010;citation_pmid=20538624;citation_doi=10.1101/gr.102749.109" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=B. Lehner;citation_author=C. Crombie;citation_author=J. Tischler;citation_author=A. Fortunato;citation_author=A. G. Fraser;citation_title=Systematic mapping of genetic interactions in Caenorhabditis elegans identifies common modifiers of diverse signaling pathways.;citation_pages=896-903;citation_volume=38;citation_year=2006;citation_issue=8;citation_pmid=16845399;citation_doi=10.1038/ng1844" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=Y. Li;citation_author=O. A. Alvarez;citation_author=E. W. Gutteling;citation_author=M. Tijsterman;citation_author=J. Fu;citation_title=Mapping determinants of gene expression plasticity by genetical genomics in C. elegans.;citation_pages=e222-e222;citation_volume=2;citation_year=2006;citation_issue=12;citation_pmid=17196041;citation_doi=10.1371/journal.pgen.0020222" />
<meta name="citation_reference" content="citation_journal_title=Trends in genetics :  TIG;citation_journal_abbrev=Trends Genet;citation_author=Y. Li;citation_author=R. Breitling;citation_author=R. C. Jansen;citation_title=Generalizing genetical genomics: getting added value from environmental perturbation.;citation_pages=518-524;citation_volume=24;citation_year=2008;citation_issue=10;citation_pmid=18774198;citation_doi=10.1016/j.tig.2008.08.001" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=S. J. Mansour;citation_author=W. T. Matten;citation_author=A. S. Hermann;citation_author=J. M. Candia;citation_author=S. Rong;citation_title=Transformation of mammalian cells by constitutively active MAP kinase kinase;citation_pages=966-970;citation_volume=265;citation_year=1994;citation_issue=5174;citation_pmid=8052857;citation_doi=10.1126/science.8052857" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=J. Montojo;citation_author=K. Zuberi;citation_author=H. Rodriguez;citation_author=F. Kazi;citation_author=G. Wright;citation_title=GeneMANIA Cytoscape plugin: fast gene function predictions on the desktop.;citation_volume=26;citation_year=2010;citation_pmid=20926419;citation_doi=10.1093/bioinformatics/btq562" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=W. Niu;citation_author=Z. J. Lu;citation_author=M. Zhong;citation_author=M. Sarov;citation_author=J. I. Murray;citation_title=Diverse transcription factor binding features revealed by genome-wide ChIP-seq in C. elegans.;citation_volume=21;citation_year=2011;citation_pmid=21177963;citation_doi=10.1101/gr.114587.110" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=H. Ogata;citation_author=S. Goto;citation_author=K. Sato;citation_author=W. Fujibuchi;citation_author=H. Bono;citation_title=KEGG: kyoto encyclopedia of genes and genomes.;citation_volume=27;citation_year=1999;citation_pmid=9847135;citation_doi=10.1093/nar/27.1.29" />
<meta name="citation_reference" content="citation_journal_title=Nat. Rev. Genet.;citation_author=A. B. Paaby;citation_author=M. V. Rockman;citation_title=Cryptic genetic variation: evolution’s hidden substrate.;citation_pages=247-258;citation_volume=15;citation_year=2014;citation_pmid=24614309;citation_doi=10.1038/nrg3688" />
<meta name="citation_reference" content="citation_journal_title=eLife;citation_author=A. B. Paaby;citation_author=A. G. White;citation_author=D. D. Riccardi;citation_author=K. C. Gunsalus;citation_author=F. Piano;citation_title=Wild worm embryogenesis harbors ubiquitous polygenic modifier variation.;citation_volume=4;citation_year=2015;citation_doi=10.7554/eLife.09178" />
<meta name="citation_reference" content="citation_journal_title=Seminars in cell &amp; developmental biology;citation_journal_abbrev=Semin Cell Dev Biol;citation_author=I. A. Prior;citation_author=J. F. Hancock;citation_title=Ras trafficking, localization and compartmentalized signalling.;citation_pages=145-153;citation_volume=23;citation_year=2012;citation_issue=2;citation_pmid=21924373;citation_doi=10.1016/j.semcdb.2011.09.002" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=M. V. Rockman;citation_author=S. S. Skrovanek;citation_author=L. Kruglyak;citation_title=Selection at Linked Sites Shapes Heritable Phenotypic Variation in C. elegans;citation_pages=372-376;citation_volume=330;citation_year=2010;citation_issue=6002;citation_pmid=20947766;citation_doi=10.1126/science.1194208" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=T. Schmid;citation_author=L. B. Snoek;citation_author=E. Frohli;citation_author=M. L. van der Bent;citation_author=J. Kammenga;citation_title=Systemic regulation of RAS/MAPK signaling by the serotonin metabolite 5-HIAA.;citation_pages=e1005236;citation_volume=11;citation_year=2015;citation_pmid=25978500;citation_doi=10.1371/journal.pgen.1005236" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=H. S. Seidel;citation_author=M. V. Rockman;citation_author=L. Kruglyak;citation_title=Widespread genetic incompatibility in C. elegans maintained by balancing selection.;citation_volume=319;citation_year=2008;citation_pmid=18187622;citation_doi=10.1126/science.1151107" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=P. Shannon;citation_author=A. Markiel;citation_author=O. Ozier;citation_author=N. S. Baliga;citation_author=J. T. Wang;citation_title=Cytoscape: A Software Environment for Integrated Models of Biomolecular Interaction Networks;citation_pages=2498-2504;citation_volume=13;citation_year=2003;citation_issue=11;citation_pmid=14597658;citation_doi=10.1101/gr.1239303" />
<meta name="citation_reference" content="citation_journal_title=Methods (San Diego, Calif.);citation_journal_abbrev=Methods;citation_author=G. K. Smyth;citation_author=T. Speed;citation_title=Normalization of cDNA microarray data.;citation_pages=265-273;citation_volume=31;citation_year=2003;citation_issue=4;citation_pmid=14597310;citation_doi=10.1016/S1046-2023(03)00155-5" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_title=Contribution of trans regulatory eQTL to cryptic genetic variation in C. elegans.;citation_author=B. Snoek;citation_author=M. Sterken;citation_author=R. Bevers;citation_author=R. Volkers;citation_author=A. van’t Hof;citation_volume=18;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Front. Genet.;citation_author=L. B. Snoek;citation_author=I. R. Terpstra;citation_author=R. Dekter;citation_author=G. Van den Ackerveken;citation_author=A. J. Peeters;citation_title=Genetical genomics reveals large scale genotype-by-environment interactions in Arabidopsis thaliana.;citation_pages=317;citation_volume=3;citation_year=2012;citation_pmid=23335938" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=L. B. Snoek;citation_author=K. J. Van der Velde;citation_author=D. Arends;citation_author=Y. Li;citation_author=A. Beyer;citation_title=WormQTL–public archive and analysis web portal for natural variation data in Caenorhabditis spp.;citation_volume=41;citation_year=2013;citation_pmid=23180786;citation_doi=10.1093/nar/gks1124" />
<meta name="citation_reference" content="citation_journal_title=Current biology :  CB;citation_journal_abbrev=Curr Biol;citation_author=F. Solari;citation_author=J. Ahringer;citation_title=NURD-complex genes antagonise Ras-induced vulval development in Caenorhabditis elegans.;citation_pages=223-226;citation_volume=10;citation_year=2000;citation_issue=4;citation_pmid=10704416;citation_doi=10.1016/S0960-9822(00)00343-2" />
<meta name="citation_reference" content="citation_journal_title=Development;citation_journal_abbrev=Development;citation_author=F. Solari;citation_author=A. Bateman;citation_author=J. Ahringer;citation_title=The Caenorhabditis elegans genes egl-27 and egr-1 are similar to MTA1, a member of a chromatin regulatory complex, and are redundantly required for embryonic patterning;citation_pages=2483-2494;citation_volume=126;citation_year=1999;citation_issue=11;citation_pmid=10226007" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=P. B. Tan;citation_author=M. R. Lackner;citation_author=S. K. Kim;citation_title=MAP kinase signaling specificity mediated by the LIN-1 Ets/LIN-31 WH transcription factor complex during C. elegans vulval induction.;citation_pages=569-580;citation_volume=93;citation_year=1998;citation_issue=4;citation_pmid=9604932;citation_doi=10.1016/S0092-8674(00)81186-1" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_author=R. G. Tepper;citation_author=J. Ashraf;citation_author=R. Kaletsky;citation_author=G. Kleemann;citation_author=C. T. Murphy;citation_title=PQM-1 complements DAF-16 as a key transcriptional regulator of DAF-2-mediated development and longevity.;citation_pages=676-690;citation_volume=154;citation_year=2013;citation_pmid=23911329;citation_doi=10.1016/j.cell.2013.07.006" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=O. A. Thompson;citation_author=L. B. Snoek;citation_author=H. Nijveen;citation_author=M. G. Sterken;citation_author=R. J. M. Volkers;citation_title=Remarkably divergent regions punctuate the genome assembly of the Caenorhabditis elegans Hawaiian strain CB4856.;citation_volume=200;citation_year=2015;citation_pmid=25995208;citation_doi=10.1534/genetics.115.175950" />
<meta name="citation_reference" content="citation_journal_title=Current medicinal chemistry;citation_journal_abbrev=Curr Med Chem;citation_author=K. F. Tipton;citation_author=S. Boyce;citation_author=J. O’Sullivan;citation_author=G. P. Davey;citation_author=J. Healy;citation_title=Monoamine oxidases: certainties and uncertainties.;citation_pages=1965-1982;citation_volume=11;citation_year=2004;citation_issue=15;citation_pmid=15279561;citation_doi=10.2174/0929867043364810" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=K. J. van der Velde;citation_author=M. de Haan;citation_author=K. Zych;citation_author=D. Arends;citation_author=L. B. Snoek;citation_title=WormQTLHD–a web database for linking human disease to natural variation data in C. elegans.;citation_volume=42;citation_year=2014;citation_pmid=24217915;citation_doi=10.1093/nar/gkt1044" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=A. Vinuela;citation_author=L. B. Snoek;citation_author=J. A. Riksen;citation_author=J. E. Kammenga;citation_title=Genome-wide gene expression regulation as a function of genotype and age in C. elegans.;citation_volume=20;citation_year=2010;citation_pmid=20488933;citation_doi=10.1101/gr.102160.109" />
<meta name="citation_reference" content="citation_journal_title=BMC Biol.;citation_author=R. J. Volkers;citation_author=L. B. Snoek;citation_author=C. J. Hubar;citation_author=R. Coopman;citation_author=W. Chen;citation_title=Gene-environment and protein-degradation signatures characterize genomic and phenotypic diversity in wild Caenorhabditis elegans populations.;citation_pages=93;citation_volume=11;citation_year=2013;citation_pmid=23957880;citation_doi=10.1186/1741-7007-11-93" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_author=V. Vu;citation_author=A. J. Verster;citation_author=M. Schertzberg;citation_author=T. Chuluunbaatar;citation_author=M. Spensley;citation_title=Natural variation in gene expression modulates the severity of mutant phenotypes.;citation_pages=391-402;citation_volume=162;citation_year=2015;citation_pmid=26186192;citation_doi=10.1016/j.cell.2015.06.037" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=M. A. West;citation_author=H. van Leeuwen;citation_author=A. Kozik;citation_author=D. J. Kliebenstein;citation_author=R. W. Doerge;citation_title=High-density haplotyping with microarray-based expression and single feature polymorphism markers in Arabidopsis.;citation_volume=16;citation_year=2006;citation_pmid=16702412;citation_doi=10.1101/gr.5011206" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=Y. Wu;citation_author=M. Han;citation_title=Suppression of activated Let-60 ras protein defines a role of Caenorhabditis elegans Sur-1 MAP kinase in vulval differentiation.;citation_pages=147-159;citation_volume=8;citation_year=1994;citation_issue=2;citation_pmid=8299935;citation_doi=10.1101/gad.8.2.147" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=M. Zahurak;citation_author=G. Parmigiani;citation_author=W. Yu;citation_author=R. B. Scharpf;citation_author=D. Berman;citation_title=Pre-processing Agilent microarray data.;citation_pages=142-142;citation_volume=8;citation_year=2007;citation_pmid=17472750;citation_doi=10.1186/1471-2105-8-142" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=P. Zipperlen;citation_author=K. Nairz;citation_author=I. Rimann;citation_author=K. Basler;citation_author=E. Hafen;citation_title=A universal method for automated gene mapping.;citation_pages=R19-R19;citation_volume=6;citation_year=2005;citation_issue=2;citation_pmid=15693948;citation_doi=10.1186/gb-2005-6-2-r19" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Ras/MAPK Modifier Loci Revealed by eQTL in Caenorhabditis elegans" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9.cover-source.jpg" />
<meta name="twitter:description" content="The oncogenic Ras/MAPK pathway is evolutionarily conserved across metazoans. Yet, almost all our knowledge on this pathway comes from studies using single genetic backgrounds, whereas mutational effects can be highly background dependent. Therefore, we lack insight in the interplay between genetic backgrounds and the Ras/MAPK-signaling pathway. Here, we used a Caenorhabditis elegans RIL population containing a gain-of-function mutation in the Ras/MAPK-pathway gene [let-60][1] and measured how gene expression regulation is affected by this mutation. We mapped eQTL and found that the majority (∼73%) of the 1516 detected cis -eQTL were not specific for the [let-60][1] mutation, whereas most (∼76%) of the 898 detected trans -eQTL were associated with the [let-60][1] mutation. We detected six eQTL trans -bands specific for the interaction between the genetic background and the mutation, one of which colocalized with the polymorphic Ras/MAPK modifier [amx-2][2] . Comparison between transgenic lines expressing allelic variants of [amx-2][2] showed the involvement of [amx-2][2] in 79% of the trans -eQTL for genes mapping to this trans -band. Together, our results have revealed hidden loci affecting Ras/MAPK signaling using sensitized backgrounds in C. elegans . These loci harbor putative polymorphic modifier genes that would not have been detected using mutant screens in single genetic backgrounds.

 [1]: http://www.wormbase.org/db/get?name=WBGene00002335;class=Gene
 [2]: http://www.wormbase.org/db/get?name=WBGene00000138;class=Gene" />
<meta name="og-title" property="og:title" content="Ras/MAPK Modifier Loci Revealed by eQTL in Caenorhabditis elegans" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/7/9/3185" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="The oncogenic Ras/MAPK pathway is evolutionarily conserved across metazoans. Yet, almost all our knowledge on this pathway comes from studies using single genetic backgrounds, whereas mutational effects can be highly background dependent. Therefore, we lack insight in the interplay between genetic backgrounds and the Ras/MAPK-signaling pathway. Here, we used a Caenorhabditis elegans RIL population containing a gain-of-function mutation in the Ras/MAPK-pathway gene [let-60][1] and measured how gene expression regulation is affected by this mutation. We mapped eQTL and found that the majority (∼73%) of the 1516 detected cis -eQTL were not specific for the [let-60][1] mutation, whereas most (∼76%) of the 898 detected trans -eQTL were associated with the [let-60][1] mutation. We detected six eQTL trans -bands specific for the interaction between the genetic background and the mutation, one of which colocalized with the polymorphic Ras/MAPK modifier [amx-2][2] . Comparison between transgenic lines expressing allelic variants of [amx-2][2] showed the involvement of [amx-2][2] in 79% of the trans -eQTL for genes mapping to this trans -band. Together, our results have revealed hidden loci affecting Ras/MAPK signaling using sensitized backgrounds in C. elegans . These loci harbor putative polymorphic modifier genes that would not have been detected using mutant screens in single genetic backgrounds.

 [1]: http://www.wormbase.org/db/get?name=WBGene00002335;class=Gene
 [2]: http://www.wormbase.org/db/get?name=WBGene00000138;class=Gene" />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9.cover-source.jpg" />
    <title>Ras/MAPK Modifier Loci Revealed by eQTL in Caenorhabditis elegans | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"3Eackd1NLjB2suEti-5F4ShyoW1b7P5_uvW-iNryW0U","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"486349","apath":"\/ggg\/7\/9\/3185.atom","pisa":"ggg;7\/9\/3185","ac":{"\/ggg\/7\/9\/3185.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/7\/9\/3185.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;7\/9\/3185"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;7\/9\/3185"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/7\/9\/3185":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00227x3sv+OWxhCWQEF6Rr4FUgsXG5wJrp4UmbDdYTi9Sv+7DoRpMeEJeg7qIYJoNudx\\\\\\\/egbMUYCF3vtSHnDOrtb7mg0v4Sn3iRMl0CiSPzMaNLiG5K6Cwgab0I2AyzNduJRB3OCRusNPqiBJo5MM4iANqO4RP0UP0TzsoHkew98M62\\\\\\\/hy5zwBAEd9JhR4jW9zuDeKt3neIXp5VrgqpU44VZo\\\\\\\/znLf1bfBGDFcRUGkxxoPNmr\\\\\\\/y2mp0HWlMptERN91IoqSYTQXxMbVREUeg3qCEGAC7slJsmK\\\\\\\/6kGhaHAUvHPYwTdqZKVQM33dfu2kneJ1lZ7CRZlkO\\\\\\\/JMx3jGvosZ+nZq6bD\\\\\\\/zYOOiEKeEyfjCvmlHhKDN622DdDsUxVkvOjb4jjNY9gv\\\\\\\/suZxR29kW3u36IfgMc0SGvbTeHmIAZDrIyVy3grLu3U7eftiOA7MT3JMyNlSSqVdYSaa6b2PV+7D1SLyhoV+v5ZE+M+4zm4yboZnLXZJTpO\\\\\\\/tgGP5V5CqO4Ghqd19ODyZ1Q8YoMCup9CysH1SaUg1qtR1\\\\\\\/YCbzp3RbdcNOEr8RiT\\\\\\\/DZw\\\\\\\/V1807PEW4GZ\\\\\\\/Vo4MyJJbsx0fxkrzlxFSL+wHRUAw9SFRhByK26scgX31AaCfL3d2W6FRZozMDkJF3Sdf+vxaugIQxMxXdZuSWNSpw9hApAE\\\\\\\/LSHQR818ivasUcpdhVWAQ9h54Qyq5NxgXipQPFx\\\\\\\/fGRDP7dHJ1ztOFm2oRJLGcwdMFnVFUO\\\\\\\/k\\\\\\\/jXRgRXNO2YqTC46tOEsTLc83RGv8gdQbnKwUAokJqqDLE7Det3dQL1XPm1k\\\\\\\/rCmTF\\\\\\\/lJtu3sKrPAi31IMgCLjjVlXsPVSSTcEV3XDivCAAHaNC39EsFEBa+oEScnrv9RI6efWFsEzBMirei7Kw98CGvP5EDwnCFkIgbeJcSOyoP2K1sNKYIFNDpCX3n5rMsGMjbpTkjsT6Wb4cUljD60vyBXISK2TF3KZQ5eyg5nlq58isTu5peOn1M2rVWjkiGQskVeRrCgErpK9h1cPx14V3DZLdlFONbOhyTYZatr9IEIuqm4SnEIzwjVl0r8VFwD3CZcuHIOmkg0ThrLbe6CmOHlUqGZ\\\\\\\/hTRdtqYnNjhXVOPsvTCM3427BV9k3tokw3FuVL1az5Nt6iqGHDnJBLzcG8zJGeBUuZEt0caMibcX4tKLc5AeGUd9PsqZ162zOPi9eAUpdI7HKl0EoD0W+Tlyq\\\\\\\/BsXd776JQDAuZG+UU66FWoT9C2R7o0f7dBqODRzMtiBJ8wOT59RnvewOUHD\\\u0022,\\\u0022iv\\\u0022:\\\u0022Gpmw70jJnmCu41Gk41fwiA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022a55fd2c9b6cccff76287b553cb06e11f598746b9da51090f4196e1bc08b007b7\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022OnqR6vuTT22P\\\\\\\/nACSYHgaYwPsvts1eNdNAsP64z+6i4jgLS7XaMGOy5iRSvA3rERtjIu4mtkMofY1g5WZlJCbdzWzz8cJIE8XTadu1VGoBNqH3NcVB4khnURQeO9540FlXpwJ0XzzFPVfTpKP\\\\\\\/POgBOU6a+vJS4UOzJ+jVoYhc9t8IFIMQN7A5OEln3K4L5TBKdaOjjZq0x2Wo4PD7MVp5pBHzA6Cnoob9u5Xb9BBtKkVqp78fiebNRIveWbI1Xd+x7fX41zhKIrW5PVGGtpSn7WEU+HHQ1HOXou7k6me9qWNxUqIOFMPTkv4GIzZ9me17oJcjwVzZZZF\\\\\\\/+BEizvL6vCdOJS5vPNBvcXNSvS0+3XwgDvI774U4VG1ZI5tYqSpthdA2wBwskkd5rOAb+92JKL1bjRBVwkjp7I14H3I8c0rRX25fhmqCTXSBFEy8ULKlFvaCp25XIUUkHT7+j3+C8n46NcEmBrs\\\\\\\/H48MUBO49gjNSJyE\\\\\\\/cV1Am5Pbc6s5t6gT0gQlX33dKnUCXhMNxuDH+5Iuy5EjBq13pW8JkCS3ZLsAr24o5bR9G9909YnAsarzcHuI+5BfKogDdqROLYvs5DUPIsuQT\\\\\\\/fYaO1wfZcBHUYtis7ev4nKnlrlSCdMuYn12iZHsrWoyImHTkTxOZBKXyXxjhYq5EiYKk1R+n7H5YSSRpOj7lDbsEthyN1n50jfm3d99ry\\\\\\\/aIY2UPo2V+ES3tLMlmMj+m8kFXeT\\\\\\\/AQsKkVEl9olmsEdUjfDh6CRtkzB7J9Pv\\\\\\\/N2I7j5DBdmehC6p7tu0SEb9RY6ulwlyD7Bh7qAPjWl1C8MdGYYqk\\\\\\\/8AlvRoMND86TKOXlWccCqhsslA9k5Seolfdt922haD7GK6OFRsiGRU+S31wn1RR+0b6NROKKSYqZkBQ1kpZpeNOo3nhjBtLBmyhUsArlijqj471ol3aCqVbJaB6X4GHN8DT9pdhAwRQfc6QSVVl5HtxqTX\\\\\\\/F6Rbnf13H+URImx8rRxV6t54P4AukyK8YfXW+wkYV5Mbk+xoLP1ngqFt416hn30IlSaPIBHLJSk7NqAL52Yc2d48O2wqygwOasOJlH5Gb8Hoay2ZUPe\\\\\\\/FtsrkAdmeQNF4af87Dge8rxmrOJae7esBI\\\\\\\/6mP\\\\\\\/h75AOEsw5zN+xBfygy6jW9CZu71P8bUB15Dvq1lqcSB1FtOZK0PlWMRrmkJyxSD5dZY8FKY\\\\\\\/nH2oPxn+wY7BBI+ZF59Jx0lhKRYmwky\\\\\\\/pDakaP9YyMBTG0TRMbavgAaqtOuRgItcItmIOa4ovcL7i8bhSxuMQ9ipADVBxO3cf7DBHTcDDm+69T8buiU7+6BIU0e97QKjc18ZVpdmFbGNjU5b5gRzLjxiUGCfn5my+LnkEVVeQzMJxmcntO3UizUPSycmR67sjl\\\\\\\/hU1nMYVJ0QHXiXV2OAJ+E5x5OBzUAhBhoPv8iqE6wcrsRBeM1h44rCsBUMum5yDjkexdfiZTgy\\\\\\\/HNR1d4I774t2fk0BJcgBMPH37E89t\\\\\\\/YVQ7C6gy2lYlXTQH+Q6KqIkUzzb0iTIJze1lWX6FhHT6O+DWFi4LpCc+NPiEYyR5x5KlrfXfHjLuOWbd1spWkLqHvq0\\\\\\\/0\\\\\\\/8YNr7mSL2aXII5bC6mesttpsAO6HgJofzp9r3PmBtC0D7ayKS8LPLzmm7KMcIPx8OQCwKaM8voUsO0vPQw0cGTVhiheHD14CPk\\\\\\\/FqbLLfQIDXyzwPBE+wEq57cSMELx6IlS36VqBfnM7ZBojI1zCP6xAjM7MylKY\\\\\\\/FRydsubXgrbjpkIwMElZ+Gce7AwPVWszZDladnVKeozE4C\\\\\\\/BXmPbJpXhwSYM\\\\\\\/d0X9BVmYPq0RR+\\\\\\\/OJyiFjwmjtpiU1BFpnGyCMBPmisZVc2C8yr8ehq1yiRFKtrmyD\\\\\\\/icL48y9L0BqWGe9lnQxAvDVaZ2K22OLnW3IyIwD\\\\\\\/HdgE6cCsu\\\\\\\/Cxm73SOBiAJxgED1CCXHVrM29bP+ZizhJY098+xpj0wKprVua3yLlCUZOrpCF2UdaAVnJDHcKrMqZMyeVlUoxo5x1UpozKLu+GpiVwa\\\\\\\/2sBSGnSWndv6e7qvk5\\\\\\\/mUxMUC7NNfbzp9loujjXu71vsF836s+uqyQUJjfLNoQDbwLDrv0koqEMELK4TnercX6D4gBxI8Uti1r9MAVAqU8mF7LMkw1\\\\\\\/lDPbz8nz\\\\\\\/DGfluvggHLm7pOmPAFButUtqOR2eZkDzubBNsKdCV+ApNGE3uh3WXwml7Q\\\\\\\/yP6g5nU7yGh0dB0U7e7tofeOEpPcSJ9aMJvOa+H2MnmuSU36bXH6SgLFTk8cg3LXvFzjA5MIVN9oO0Bdh3Ir40yvwYD8yHtuO2d+ux\\\\\\\/TswdBpyhyxxsGzKJEd6x5V5BRLphvfpPfYIoxgRTcGLNmOWSYUNpET6YR5x\\\\\\\/aX3BjrQCDcmCjsQh4iZd\\\\\\\/wwKLonxmyJAq9Rdc1uyLTsgrVbo3gZ\\\\\\\/fGEVi80QjrUBI31LjGnTOgqFbVbS\\\\\\\/4B56m2COoUHIPcfDoIbqd1aKFsMq7tKXVSyvuIKmL0H9cbelufbOGfpe2IGgCO8PfBdjqlGQPN1BCUyeBsxpCAm7mLKOIcsAUJOP0fLrMEyGr9oYGD1FznUkTAaO\\\\\\\/AxkA3XbC194JgFdlu19H\\\\\\\/88siEppEUexE\\\\\\\/C\\\\\\\/tpUSbtQfgauGx+IWdExRfUMnu56MVnc8QzKgR3x1pfz\\\\\\\/9UpH1ys6scy47OHE+awqmHFl1C8mgjXYqjGuTBFvu4ODpSibAvVsWwGoOsYUHe9iKmxeMhxhZMBEhCPkCOxmbvFxysuXjh5DhJwN9yW1Wrx7x3wrstLKxohKt6e0f9TtQUrbQ++NovmAtDFfEQWKyiS7lQscxEwGTGQROT2ejhtcbT5TD7BNoZP251ilqPPSrjrbR0qFp\\\\\\\/6qkUc+ZupktDhPv0rifO1TAHee0tI4+HdrwWWkJqJ9ShLr12sIMbS425ZiT8KS9K\\\\\\\/BHWrYdnnSoRGpxdXRj1VscE3MCHi\\\\\\\/aqQ\\\\\\\/ekvdZ6AiFN\\\\\\\/mgdciVN26EonNoi4j27a7vppy4l4AwMbVfSb5tGrzNnfT48P0RD4uZYWojJO0rgNvDYjbm2\\\\\\\/OQQEfeQaEfTDdAC0mC2OjvYG\\\\\\\/Ci8kN5dW2aKW0kigRl7hwaolzkaPrF7QHufpTUfGjQONqcWE6G8OrGrWtJMioHgWqsz+dKFx\\\\\\\/7VQopXkzws8J\\\\\\\/elLsSTTUOn+4ryR8iws4YdYbR7BRXLkhGAjTujhg+qM0d0FBdDxNiZ9c03\\\\\\\/pM4EuL32sIZOAsdrCTzmaxTOiwALr7NO19shsRhB8xoVE8yXNgQ\\\\\\\/BQGuoj34ljHYoGbHFxtiuAG61eArPFjvNTV1wI6nJwnTjjn8IkQ\\\\\\\/L0IQn85Mt17R1I6HWsNvvzU6gTOOtxrLeUbzdZYMCxxBoQdj12YHvFnRf1tXo1GlFW4Y78hzpf1wZOt1mtkPnz7OMoOybUYXyNxZYkYVzyFSeZy4mTCUPGCy3i3xKjXVAVSyc1sK1TuvOa9qW1bSxj2FfkwceWz9y8cxWBDAo+vD1N3jDrl452rxIF1n\\\\\\\/OZcyR1F50ZdWdI8muQRXvfYOn8x08MxQyVvHn6RX\\\\\\\/anbgobWrGs94C0LPJvlTaKAQ9VODuMPOT1fVyd6yuruy8MoDMKmcBj12ASpvX\\\\\\\/zDqAMcrVG1ImtRRZzgwkfxs0jeCilxtId+9DH0oYkHyL18HtrsUfrub9HYY\\\\\\\/dEtQCTCY9\\\\\\\/AIaREkvyCmLM0b3OHVWR6TCBRb1wIZqD2qaDWJAddhOdvp5PF2tGxAC8hYQdUW4LPO2bqfTLsEQgxv723pg6mji\\\\\\\/UgFvBF\\\\\\\/xTvcOzcNO6xrPrEFWBVQElNgNYxuYKI6LNEgC3ROp4hUQehHHXTkesmBzQMFKqC8HFE7J33bcyC3kfERjP9zFU860CLj7TKoA3Y4\\\\\\\/tZVF34q1gKFv8Mq4A8rsinsOIZVAtmQjmg6A2HivvPwMq\\\\\\\/N\\\\\\\/VkqY7DBRWuPCiq1kU0UYVaLFC8Vekt\\\\\\\/7OkxlvUMpYhugcYP3Td8GixrYmSg8Jdp7GE8GD+8MoRVLvGQmmXmpO\\\\\\\/0UAW4ruiaEq3uTMbcY5jaoVgqL9ULSHVai1xJ0MECN1dhtaZ\\\\\\\/kjWu1uOImgNetnavtQY+mdbgznhcP0K5vgkUKNLOO5IukXsbqHaxF0MPESYZQn+Jxk9crXgMXXLMCyBCXFOWWkLgZeDFd22LcKaFwI7+ndmo\\\\\\\/SvY+0QMMOyYeiATG+cgNmcrrfbvxM+zKhWW\\\\\\\/c83g6ScUb3KzHRPxf22V7GskXiSXv1gquAFi00U0xw4E6gqGcg5X4otnVb6dJLRunVPAv3nZYENGtXLcp8WHys7zELdwDFoXDFb5OPhLx1BvRenIxo67A40xTzZv+\\\\\\\/3WnZqPJ74kHy96sIs\\\\\\\/3UPtLbw8i0GkVlJoXcqeu9EiNq5vnkqOab2ZPRSG7kQ6NXuDjHhCtU8AeEbMTRnCU+CrEoKIBlR2kJeddFDYD0zGuLFJKWVEialAZyP+hPg1Pp0s4JZjIU7T+M91ZJjMKbMTBomMpPCtCsoGWdtO6n8k86AhPDyjJKpN4o8\\\\\\\/XCXYVhf798CrRBSIIvfcIScToQjzHGdguIAYV1Ryjd6E0bvHNHUEhghDroA0+3fJD4r1glJ2dX\\\\\\\/T3lcBwMqaJsXjD0ygzPk0XfOETjAHf4TLg2q0WJpgIjaFRVtPeTMxELylOdnf61iAvG\\\\\\\/wRcSHfkZ9SddkTVAhMra\\\\\\\/HZFeH+aOky8oXmQMBfilidFMzJQ3N25lAVr6FrrkQpbaGKvZpMcx6cOXHLBEmupNKB5p2KTm13eqZVqk4zEOw5UaAgax8gIy0jySoeAQ1J4RilOgLIguw0TDCJiojanNpok8UFBB\\\\\\\/DTxExPvfAQDNKtwlCDqLe3Uayk+PJEqy12isgO4T5FrHU1stScLzO6pbZlPQgAtOjd8WaHCqlHEmXVf\\\\\\\/L8GwoSkdECiqpc2JABN2TxfKVUSY3ogsNsFh2DK0LEIBLUOsjMSHAXHY+siaZ76drRNj\\\\\\\/jZ9A5uzivd\\\\\\\/6zyXTt30U7rduQ9EfAaHRJklt7Po5LOpMpF5\\\\\\\/CfdbCHT9sBDnNDR6v6Zn1TeI9\\\\\\\/A2XOcjG8ILyECiJ6OVwM9M\\\\\\\/D3gHQ4mJEUyor0lkVxkaZWI+P8KiGVAMsccsW5NcAuUbME\\\\\\\/T5+9EMC2Hy5oube5Zzu2Ra6\\\\\\\/QV0P61t52EH4rumeIiiVulQwIGjOMpDWE2OPa+Egrr2G+WdUgKcxAu3KB75A10vJW+tdyzfohi5BtXmsX7ezsZNYz8122sqlRSSqjX1vTSJQAI4k2f0ggsYkkVuF9A2TwEhZ5XKdovt0HuUfZK07zSoLV5GT7zBr+3dQ2uutAy93+xdwqJX1vEZaLPymC7fbVUM7TNyLyTQvKy1VXgLRIeY4ssKQ2tmo2hCctil0e6xGrdM0F4FdI1NhMIc8aoHttwBJ23Ltaw8b98EWo7Ps1BJV3WLLJnb6KqrRc1ecT\\\\\\\/o2XQwbw\\\\\\\/OahZeOsX3\\\\\\\/nxrMHc5dDT9TiLTZSSUptCxRz98ErtB4A\\\\\\\/iK6pUWv5klF1mSmAnRge64AX86OW\\\\\\\/VbL6md5I5f1\\\\\\\/lORdNSEmJcXgdKHOWADZTAXrAqdJvpGcjwdXM+cvv1lc7oY7UINXe4olIRRlQqSli5JWV3EJ9sYx2bs0Vygui4C3NzVQe99346x6lrS35dwF5Z0pNmifrTjsA58Qgv+9fzHsSHoPK2hPCVWa+Y4\\\\\\\/CsoYhOYGbySS7F3Tdz\\\\\\\/RJvgeDQPvVLD+cTC\\\\\\\/1sv8KUFDqSQmPvGp\\\\\\\/n0oWgGh6PYy4V81fDX8U1EXLumfAzWMQEpdzcJS+zGrdlTzXR22xtyOKKlG6uVaWsTh0rlaOI3Pk\\\\\\\/pVAylTPs8bpD1tzIayN0m2ukchj\\\\\\\/NRa0JGwbl1l7bZPTdfJP0lmLAJqXU7+A4rETRzatbRnBkZLUO1oIqPgs0auUCBtHqEmYw9gugLQfmTOXhhsPpFQ33z7UiaM0h0zE57AcOURAvLE\\\\\\\/q38DXGp10B3ZJ26gUCRtJIFw7bPsCyMkPH2eNhNCLyY7s0xw2inClzzXx+m6QDAI2v3o\\\\\\\/6JBYcwnC7rIwaBy2MyNQIcPClm2YIqWMseraojZrWYW59Xgq+tj4z0csB\\\\\\\/2RoZPn2K2+mBCf2+5TUEunkt7GnEHwAzshUkT1\\\\\\\/Q6d+byO2r8PzoIRFacXmoBbSxL7DjS5UCjRnEVJuld6zmfDDlShLxXoPglVkgKEdoj41ZyIMwcGPTlVVYTJRagrbALaDj2fDcrb1LUjHrLfYDc+zBJn+7QeqqF\\\\\\\/YgFDYCQSqxw6aFRvwzx\\\\\\\/lZSLYbaox+\\\\\\\/mi9MD7BUgIV5KcdthqhgXAOUvgL7OuHnc6qnm5LuKRZe\\\\\\\/nIDiGJ4S9MR0rvwEfaxOrAH3q4SE7SfD+6fOnxACTCHYrF6a3LFatL0PWHL7d82U8lnFhvQ2FwB+xlCu\\\\\\\/ha6OyqbJziKvf1w4u4qG8whIpSDHxmOuSLRMuhn7u9P0cSUPrmRe8ah+MLiDtJmT\\\\\\\/SSe8n\\\\\\\/\\\\\\\/mFMA2FXOz0g3onBMtOPlanM\\\\\\\/fh90SLKaSfstMpFp9xtMpKSqEaKiAI4W2FXMPFy9EcN3SJyAbhdwheQcesKrdNoktTsEMuZR0FljRW0bgs7paxMZzAwOudJ46eW31jbY4iE\\\\\\\/c7nA0eHLUoahQsFnu+tNMCRZN74Lu1vmmY+JasM4tV5zxUIvupXjGbTajS1HgLjaB1UvHNnZ6onQJ39011b6pdbMBOhKKZARtNDKi7G1\\\\\\\/8ZaQEmne60dTP+qpXYnC9ZWLJIa8tZISi+jQbjUzeKhZyx76jECGWEouYEgw3y7+EQJsmQWSgOs2hazwoQgd7QNE\\\\\\\/7cHTZ1mfJeAeVBfqBXKrspgWWf0R\\\\\\\/qEtlz0yMgU0c+kvFcUv9EGF7TTjCldQA1n9DI63rs2cf+gEb0xS9S\\\\\\\/AVCVOOzFhPzn2+K\\\\\\\/7JZroBi6h+SYgltJVuXPz\\\\\\\/9+ZLTMe62nYPtCEUVwNSHMiYbgvHAEC0fJFQ1juEzFGL4TRVEMIwl5nX7PD8GvJE6nZ7wzk60AUGgY6xVEQ8DaZ52C61JW+Rf+4xyXvO8Zo+1aiQ\\\\\\\/el5W5yrV\\\\\\\/9BsA0iURxI7G6ris129ImoT2v1z\\\\\\\/Felv9qtfZnSVdlavOSN3FqrXaHFzSGEeBjNi184HSORfsXC0ltfamJYfWDaxrPlUaAbPELBmg6XvcOpqLchXLkay9xhXzUzuAlPivSMiqv0soxNOygDzzufyWh8oUq0+tzfFeVibf6SI86likvbvjSZk+0y+r\\\\\\\/WJjl4i+ROwerC380WXeRQ\\\\\\\/mRTbQHA+zSVR1gTrwZqItIfndajANPvdMLCf+bju8OpT8gHT9s7Wi3OYf\\\\\\\/KaKNG62e8RTGx0iKDMDQLWpO1pLK6yVTrLsJBXNLW8Pvz3kSxg5EM\\\\\\\/cNdjd6cjs4njcoljIfxB\\\\\\\/tdIpVAVg+pS6j8KSH02XuHQBjJpFXxa3A8O7PwHp\\\\\\\/6m7V66huhRodq4vBd98ZBTRONqygmqTwYhCiKBC9QgqmExVA6HqbG4hudMiNHu6WMfVTEwkS2qTCL03ly7E5fXyYSd2MAfiLjFaVxb5oQ6wHpg+fPpaG8IF3tbHlbk6VH2udoK1CozOA0REwgTfmRWmbKPFBxOg8n4T0cYZaNZOaJPQW\\\\\\\/2hhI1pZxE+tzLqVfiEElkoJpOnFxwhXEUzRXAWPevSQp3kABtiFice8ee1HegyGWYpFt2GlCvOks4oNTw75Zyout5nFlg8SLZKXdJI01kSMt2SAaGbo\\\\\\\/g41xM0RFdvkGfiYdUpgh8LG0PqPGeWuYXKion3X2XB8qE+fYFBwaIktKGAiN0jsHk8kP6D8OH1LSmFyvqx9hmELzBWHGfiPts7GcmCeiqPTSMiL5IYyhyD5kYNsz00P69Q5yZL9MQesbkO+BrEEl8pmhSrAQebFKv6rcOnA7x1Kj4k5\\\\\\\/5RfzPEAaAiq1Qj1D6lvrDA7LC4PScz6W2PkoQ+rV6Eq\\\\\\\/+Tyuj8n4nxi+ZCdWVSjR4+FMAgTD+zvgZMRbifziDSFcN85bYabWZLzhmPOrSqexpHvQamMNXLFzhhmlikU5JxMZzK3WB7oIRIi2ppaNYsQTHGF3\\\\\\\/oIzaIvZ6DNRp1o6FzlGeAIzuO4ujxXkRgMf1alUBdInLb5dC8hwHDmVGMTtYRR1aduRrMxnKo1VnxA7oJGfQYv5gWN1wimxJAz5vz0IqjmmgP4VIdmHD55LjN1r1aOjUX3nS\\\\\\\/Z2C3aKrRbclSY9ly\\\\\\\/eCICWHqd\\\\\\\/btsBxJGWfWC6lmJdhjisUgm31as5TF0cNZDX\\\\\\\/7oHQE5lHKMvxZalwt\\\\\\\/hXbwsCKPV\\\\\\\/R6F6X8UZDocgIUqOlgNsYYessSTVF1wNnkWDD\\\\\\\/MxoqHK+cHNCHNACXazO14lTT6JOqngIRbWDEJ\\\\\\\/Urmnz+i2VSmT4E7wn1hefderROSLqnL7bUs29xHES91kO+nRFJO8fhKG1GpCap3Xw9DupSVIw9ckFT5pzzwqqp64ZyJIq4FGTTjOXpZ4ABDAi1UQK63obsOgws54uxgN4VtjlTgN6l6oTG2U7eaCnGQxPIPofs1KDQgYiUooXF7bmkNmdV9HiiqCDjLD7M2NzWzXTK1LHPTtq764+WrIcc\\\\\\\/gc2x89FZ0K9YRDNi6WfwvVpKhvU\\\\\\\/x\\\\\\\/fCATv0HuHThQ\\\\\\\/5DvHFVhm6e+h7dENChKGbWFNaVg5iLeSJ8tj2OZq9gmBpBDGKERTiP\\\\\\\/QaACn+0GGTpDKG3URaK4+ig56YdLeClP1zcRJ4Zb7iNN4SwWuWKzTo8\\\\\\\/zxoWD9wHEo\\\\\\\/rGr63slWjaT9yJXk+aca9M4pmjPkDGIwHIZCvelU5Vm6F4PvokR74GsgSGYFX\\\\\\\/5mY8JSUUv5X8E7qkKnP8Uk5Vgeqij6Oh2EaVYjzaS6JnU\\\\\\\/KqF3R4axT73OY\\\\\\\/HlCZoAXdQJocph8mAWWOPqVug4+QIiKqdB30tScaC8BObGk0NCUzlrkPJ3tGjgriXcYmjONC4543wOkNj\\\\\\\/DFOd3f9PmFmsHQqArXbInHHlsVDUsHj6mScWNR2TrPwOpGv2VJ+Q25ZF2zb2+H+hVj8yzJwvA8\\\\\\\/m8p0BNOcmKYox43X8G13EvL475B6JnuIV2zc6PGZVK0PjEKgXTkdqpQuf2mnaacPNMfO7qi5KPzlQVQh0j6yAYQ5gznM1WtbG8ydqmPPp+uppKD2FD+ibV+tTVgb4kDlOG7MNCO2i33kTIq5536B2q7HXGugZTSXb1xJ\\\\\\\/WVAxQgfNsxKmMeXey5UdR8ygiDFN8AVRIHjvpWkqox5OzGuN73bx7mU5uYNp3Sj0LBMt4BFIdZX+GfhLUdDqTQkSWg\\\\\\\/Bfo16FfXbEygHGZ0ghAYIKc6DfmcZGIOmcrmd8hAcsmHFwsdtW\\\\\\\/pYEqp7ZC4UZ0d\\\\\\\/ojHO545tlCG8RPcmKhBOCes2CWYybpWbG3dPTOZoXbpbW1UQZQq3dAy3W8EkxtHc20lzWdv\\\\\\\/Kl1uVVVQesXonKbCcNsmabJkf+7RL\\\\\\\/KUWyfdCZ0HUQgA3+VZqYr3dsT02kbuWvpcN7NEGsgHXZ2P4cUjyUGkdZyh9VPvRivAxHQuzd46RE05o4NXX9oWgJN4MvWtJHk+0gNPcfn3h3R7JfJtV48icz57DykyZjtGlP1aORUj8kYxtTyc973tEIZmGV5gdJ4gbSGmdRQomXyobqId0qe4ceCB9zCljHtzEbE73Vgciix\\\\\\\/iVj2A1onXBUS2GZcEh8OEogtCox71BoaI2scZNZWhnyDVHysgCaHXkniT+phWgpXzpxGNlP8crlS08FF1taIpJqHrsTc0Jn+SV78QV7QBAgWvumb7zZYAJpjDyr6yUmfhqYgyOXRR4+iONR3VILwl9D\\\\\\\/iRNknJKWkI1reOEu80ilYfgIx\\\\\\\/2xt0Iok34jPviY51Lb6pXv7xVvMwghkMTlh2ksQINhsaF3KiceUFQkeoeIn+59OHKcM5IlYixVHvWcXY2BL5yswmWvxR9c5G7pxjw65kHyBvYPCHY7HBNwdP19qsRPgKEUI4WE8pjNZV8JeNKjCzEtdDIxHikipmJqkSRLydO29nGR6jNuAArcP2COYyNFr\\\\\\\/iczClP4WStmEL35mlf1Ma\\\\\\\/Qdfa7p7ouv0yqbGFr6ViG7nIe9rDzqB2WFArgA95gHiyOfOgF1V5ORf8MrJic4b17ovkO+FB0LWfHTEce3t7t96VThCZBq+tDC7YGTKkqTkwgERapd\\\\\\\/zUosLZFnf\\\\\\\/cRR9BLz+nH7hLiPr3l5Yz6NS\\\\\\\/oK+B03i0m8DbhFJH2qtX+FwYLEpx8JQz4u\\\\\\\/F6v\\\\\\\/LPH1eSazOHLo0DFfU\\\\\\\/dc+JI6wDGlyQn+Nmn5OKkXsNS6beAqn8YEcsZfja+b5AjfTUWpXclFTDiNOctZfWJdn18S6100EGYvpRA5JC2xG92luGtcfLlp73Ob4zoUsbSPAXwQMLusZ8K6BIsgykz41LgZvTBpE55piPZkOq5YoyEiM31Xga3yMhoTm9x4sAPNAS4qWDKv\\\\\\\/VTM+5NHNM1r36e++BRo8Ny9gVwL0O6FZYlBeunxDYamVonbaBFN17Abfy+QZWIXIjBodembB3QjizqEWtvIQVAgvzgqbBazb\\\\\\\/SbrSKwStblr7DLXjObavyIi1OGsLBcZ1FfRXK57yiDoP2ZwzlIeX9CWXfSxIan7ig\\\\\\\/QwvKdtAcxeazPkKKY16Ly+Y4szzcs7nE\\\\\\\/BT0LmV1rn77\\\\\\\/37WKoQwIABUG6S7pSahm5gV9ZcLH7vw5z0+3uYxYryQT65okac20MF4J3H655yfo2My3StaokTKAlgUI2XYEVYdx2Zke69g4lbh+X8PQi3NWQfcVJQInxwxBapJDVGWxRs9LEWKJFcQ6ag8Q6jSzUhEHRaXWb4Sd+LRyin2xtj8E2AMe73gGJ3yH4q64YXbJiDmpqLmXby4uW9uEY7tq3Ikm2gHiVKvursIo7HfoT4rctYsUHrUjaNRjF5uWNB36zem\\\\\\\/T2L\\\\\\\/Ua7gC+VZwAWdFukNYAheAbXkD2dgEy52naxbN66fF4Tz6RazbOikrlFjdiyMWK+2hMroQOI\\\\\\\/n6LC7pOOYlVVyGQ2ZVrlhnbgjvBXlpbXcZUKsWyNkxL+Eb75MCiOOJ+Ow3badTprNkfMvtGzmTa7FTQkbCa+TvIbBOhjlsv+Z7kWRD6nQayG8g8pIwBncAWff6JBBn+XVH3nDQKvltEDmY2\\\\\\\/nd1K59UVTUtiHRgmXFxRLI30dG4f6eorgyjMeC0Jq2z9mUGwbuKWi9fmUQcKkauZjAogZAwHyRA82HBAVZK9rAhnXhPDRzGIqGaLkwA19SnJ41bq0DPSoVdsTcxrkcMEtcEqYNo2ojoKlQjKpVmblAxnzTJlmWT9Tenr8KWY9ba4h9KhHbyZ2AXTtg0NaCn5v24t6yqPnUWwVUM8GM44kZvkqD8jqLjP+AVrBLvtJzdkCo9KgtKRif\\\\\\\/8GTJQpFgCubXxiHPJHImKnqPNFHjNvXMaSvSnDAlioNZyZ+VAOGIRGG8G6OTi2xkAEZ+vlQJnpvuFkyd77Fi9vzvir4OZ3JH4+NQeYLBgrngY5x3SQAJ+f7xMlWY+Iod\\\\\\\/bZkyfqB7MUr1eKP1ggPim5NowiurW8+5Xwfhlzc4MeqyEU3gIaw9g2P265diKuXuXbC9W3fE4pObhbUcaC27ESocHriS\\\\\\\/HnMJGNYW0Uzr\\\\\\\/SucGXl6jwNkdP2JEWLvNBd6QIrv5MLj7CxzzqFDmwBej3TH8HxHNcTYD2x\\\\\\\/+i2kM0PzfQJh6og1xJ\\\\\\\/H9Y\\\\\\\/SHPwGvE7ZeI05ticA0femLsb4m183PL8AAwnx5Wwm7a14i\\\\\\\/p0fUCU1fGJ4IJ9fhgiJpj4TqezKTPHEw1XXuRDHmykIKcqc2qkIu\\\\\\\/Da74WkWrTlsD89mp7QtnCPLryYD8oQl7u6kiAzmK+6YKfUTR6FOSNXhXYTED2s7NPhviP2\\\\\\\/q07mgvJEQrKkdP1Cqx0JWav6dVL7DUAWtiRvNCsMunk7TrHKID2EjAzKqspx7Nd7Xd5NK\\\\\\\/KT2QLc2lVXqG2VEc+PFrkv5YO7TOVeEsvVZrzAKEjk8IIE9Qx8CLeZ6cD\\\\\\\/sriPcONwlpXea9uFa4GL4xmtGw4tzJM7PLMiX1DcGC4lnpB0T017anScELz1gO7tXRMTHfZ2EoG24W2iWq1PhPBs96ESzVJbOZ8Ar7ZcrvIpN6LrwFBmqSJCU13d0BEFerY\\\\\\\/KDQEQSVgQ929aVF4kPaK6rc0mX68YzcSu+0mFC6z7Semb+WSoehnTFIz013UqLozFZF5KTAmwH9ICcRS8GdaddMmXt\\\\\\\/HwMi43DOhZQmh2\\\\\\\/\\\\\\\/5il4RjJPnRBas\\\\\\\/iRJ3XFIo0SZKGrWj0zC+obfqeDjT\\\\\\\/Cn5x9+PDw5CmsrY0l2k376u2ugjFwxE57\\\\\\\/FNthucXxC2Hd6FUrb9MM7I1eHW+uZAd5E1bMyw1SkMg\\\\\\\/QAJmI563QvvGfXKCjwhylbDlUPwu7yiWscgWLvkvPJBay9\\\\\\\/Xai2qKlRwAYxBQ1m2kJQx+6jRHX3iWOXolQMPvw+LqFJombDwY6bjLw5WnCT2LyLJHjRCT6I99E7PJfQaXlBnUQ825RRrvnzH7Y7J+Bx1Ag02PE3pWM3prxICsfBG\\\\\\\/4a6cLLjMc1ZJo8Ywf9iMi81i2DtWXLeINZpuoHBSG0BeA2dXPZvQZuhfCCtyupFx5x8X69JQPRicoRSZUMjygpNpd2DOo+RCyC1uJPrWDp+yR8R3BME+dhub8qDzSNZQ3zk4oL+W+VXsbil6rWOCWd1Y93aZ0MQb5MkEtFWQf5CmyVbXAQokIX+h7jGA\\\\\\\/34mKlYsXCknnQSDmcX6QzbMLB8Eko8oCZCjqvai1NrUlGgZKNAmAiDKaoDeSpMdE1crjHlILv2Z4YPkk7PhFb1VTcOG\\\\\\\/ZTAchKzMy4+PFwvBXJi9ET+ZrfAytoUFLeTxc0K+C8vgqtyXiNH4PxUvvyUne7mGvqY5GUEsmSYs9QqHTuF2C8Kgrt\\\\\\\/27wNTZjFFGHSmvAJCOj+kukvfSV1K3KtLT9oH7\\\\\\\/VzZaXKGal6wfnZem3a5788a9QWm7UpgLTly44B5IpVNAdfbkAYhPxmKCWFdJHCE5HNB1T02T7PLtSfKSfjXCJJ2AcsLTOrvI7TMoBeyIWr7EKlPiGB1FNn3JF60ijVxeok63SMm9CBpqBN4qI6vUdu5xzk\\\\\\\/HkDQ\\\\\\\/1FE+TfyUuh5PKe6+HRLImkYi9u8s4fEa1kudb\\\\\\\/bDanHCz+3\\\\\\\/YPTosDt3MCnJcfMpV9VZHbBqOV\\\\\\\/XmhiJqfPiCBO3pwzwjgz9aNlA+pWxJcIoPuIK6WrgY2M87b3NOFhd4mg4WYKtlCaSwHRHX4KWJVy1oNQYdZcPd\\\\\\\/Ug8WetNy4XHvoeXzjLKfyYn67uZqlPlWza24dyVm4rhS4JJKPpC5vz2wGsIKaepWJYm8rcbtRwy1pLxPGSNp3cjjT67Wl2zT+lRxykVVt\\\\\\\/aJtdTzVvbiJ3vWOu7QzjDDKHk69GqUiJIse1oPHgNnXIHL7D1s3Zfk1w1tQkvyDcRB3Ef3VbYjr0D3rwJm6rlHLXsEmyPA+2KQQXzKaCx0oJRzQUMxg9okivxzO5EbtnnQ4dcIRoWTO0T+X\\\\\\\/kUTPr0LZQZJMFovbRg0WOQrPlSCm4Ug7G1sbW3esoHuakUY75HkZQ\\\\\\\/WrnOdVF3T5GKStDqHYX7vPx0nTva6Q3uaQhKs957n6ZZFEBHZqRru7HxWY8sTp9q076rpy4G7oYlVwwaujc58VgOV6Qyka6JXl95SgAi3fWuDk+q761pLkXB7vGAxyC4bk6jBbm\\\\\\\/O0TQzjo7ZP\\\\\\\/c81WooxCO1CxA2Kdn7L6YFvn0z6yPFPPiszm3HXQM5w0ibCUEg\\\\\\\/C6hIJ1pQ8JQuup\\\\\\\/qHWpNKrOKpmPmgsNx\\\\\\\/AQLFg3FiBl2Nzb1QFNtm3EQuKaK+yduNXI3FseBUrr6LCv\\\\\\\/Pp78TmzEeLKn3dPjZI2ZJfCU0iJ\\\\\\\/W2QL561N2uskHCaflNZ6Gshzy+TWq2hn3KsA3vD52cQcwK5Qgy5EdI\\\\\\\/lQ1uAp07G0cljUUwSViVFytPNSjzmHnef+yH7S6I334PP55yXCQEHoV3wvaqtsYpe4n3gU7kft2Xy8BTxhfl5rxddcbvYrHtGAJN0TG9qeYOBgBQz9u+1umOI\\\\\\\/YycpvG08bP8nbb7Pl1Xczi7bvIuNTbiQVAVMCyuMPAqC+z8Qjijt6\\\\\\\/QAviCCHw58z4E3Xa6yNjR10YLo+XoyLQUGeyppH6Dp8\\\\\\\/Tj1W1H9Fmn+6J073pNOLjGWlIxEX\\\\\\\/PypLGGCN0iqiZi7qhALBTlIZQ0QZgDXWM479JaG6JsPVWbCyIrHq7rJP0DzHgXZbDwz9Ua3frWgNh66E2NYYZ33ewZIpfx97JeFtDvJPqIeo4dBTLhCtIK4Li\\\\\\\/iBJfiv4L5rCDoJMxL1k4Clegi+++QLUF0mTiEgbCPjIxSbDvf5OlmNXndPVQkuK3DW8DBTXV9YVDcyGzE5IgW9FXXOzvL6zWbLMSsWoAOez7rreMTYIVL7BgRiyNQIFYL05YV0xCWyEWYmF7bqUrF0mkUiZMzNUQJUybgZLzYWPbOpmvk7ptxZPQu5WASBrEBKJ0Ahi00qUrh\\\\\\\/V2MySjNCT1v\\\\\\\/tevv9dre5NZuKo8NzmN21K\\\\\\\/DEDZesb9HRuB9No4lqWPb+wbuX533qgPGQGR\\\\\\\/VNkHnDZP2qNfFRAqwj0uByFxnt0y5vA7QJqI0EmIhFeWgVlI1lcwfogGdZ0ebH3niAdz7toPTS6lB7+xwEk2980ZCUy4Nj1LieVvsU8tHtYw9LRRwqe9ukkz+N9D6fl5zvwkWFQSc2NOInhPc9Sx2c\\\\\\\/mKYNAbLmmLXUDoo5wk8DvbV0xt9qtdYSgKoN\\\\\\\/wgQJSW5BDYyp9W2UbbH4s14l5LNgM+FysgK0OLxQbvE3x9BJOYAx3NqgznjlUtHRmbhxkq7uJ8QonqLQs5gxbX\\\\\\\/aYxzWsPjf97zQVp\\\\\\\/ae6Ueo\\\\\\\/dBlek1Cuf+d83WiVx\\\\\\\/BNKgeCJqmv3Kgm\\\\\\\/LOwnSq7A17K50QJet+OvIqRZnIoUqfOsMZxAYWhb3Fogg8mFGC4UZQDwOgjsuFXVkE7n+3pUJFmySF6zZuIwJfA\\\\\\\/PPDk2Rs\\\\\\\/Zmf80MkoCQ61G8b5DqRsmOwAclSF+92X\\\\\\\/NGDEVXUOVaLBvER7xYP9QbTU53WdYtVcF0KC7EWRjeLG9ai1ZnpAmJtqhbEjtvUM2FbL111ZDSrAo06tLvsT11igzDxTWV194xFv57jHNXy9OisTCqLY2Vs6XtIHHHk+ySOEsd+hpLxqGkPoTWrLYZGyGk5wcZwmMVifBrvFqWqKqiqL9qEAtWP9FqflvJvN+IR9qmY90bNAQ8Oa3Zt8V374zKjxwMBnZuo2WOstLWqZwwaoQ+m3KiE6KU9ywWZSthwwBZE\\\\\\\/u0TEzlkq5q8IxpPsfOQkUwE0rmT5m2xZhDWok0VtlqVNBfWQ7c6ZCpqthjaa11lDIHiDh2MM71RrG2zoPBFboTOA28srKycqswC3tk9inG0d+7\\\\\\\/yhPt2PST6qQY7p8ugipZj5zkAlEPXSmT\\\\\\\/GaIgdtNScRuVdhRw9poSkQTOUF5BZkV5BILWmH3arWwJ3g8Fwch9sL6QsRI\\\\\\\/5tKFiU4YojkPF\\\\\\\/fef2T0dcYLAFNlEt6L4wWFrtMkU6oL2gvV2oSV9PB9M5Xew2Cv+ZCoO0G1rRq\\\\\\\/INYwSD7Iqq7yLz0njnHELtPQuNLYsJsY\\\\\\\/+3ZKIe5aOXlOAVEKvVRrDVVil8KtSKkbY9zyVmOV1qw+nmymlLsPovD015KfZKrGjuFXsFI6XkzwM5LDALrqEY1HiWTgNCcxEovzsfQWR+sYSwjEsCWnmfInHEaYV1kw6kxROEGIKO5NGk6FMdZx3fc1NBr17N7pF1+iw+Dmfrbj4slmyaMjQvYBAOXNNSbiMjLR0PWTLIDadatdEPMhdVrwoSGpzOVSpzSovw2quMO+xPi48SX0xt9Nd3hexQSznJl2nGSbTA0EFLu4mWFG7fSjSrLwLB5c1uOoDmrowfnPxQi4qUkIc1ac3O9bKYy0rmQQPxlmjV\\\\\\\/7KUkxoUpsCdVnlhuDCjrY8XfyodAN6s0TdqqKed1\\\\\\\/SkMbPwc5sb3oov9wAJ0hY4J6Ue9lS6RMoSn\\\\\\\/SXS6uDQ9Xy6v10Jn2IwY732RIPb4hyxiv\\\\\\\/idL6qISAU1LVdgrvyTvkY3Bvnd\\\\\\\/SeVpEjDygVQlJnbxFy6o9PUyfZaN799hQcbEJzpN4Vsd0TcDy7ERU5v7MyVax4d6UT8soNaPRdCra9kkSO8xYFJIteFB0fJE2dNjoCp6zIVg\\\\\\\/GmFOQoYMfHThhb0P809iwJsfElDWrcKndoJ5xiVmGW2o5YF2QYBEXGQ1ISvBsNhhMhgBn3F5Voe\\\\\\\/pmRDJ14MUtakIiBpRSjewi78Oc39bOp1ODjuvCdx1GfNp1hcQu3EULfBgdJK4VQYmvrCvg9skzlYKMMVDZFKi\\\\\\\/eZPqQoQxpYYEUUcdlIMXO6mgat5jU\\\\\\\/AZiR91Xh\\\\\\\/9iwwY1tJ7ed6yWi41Atiz1lg6Rc23x1g+SbxkQobO2Y+7fyHnZb\\\\\\\/7du2LbeZpQ4eFCX\\\\\\\/K42yhUCp1W9nHA1GBTvGapbY5CXgqgFwKGpncrB5YAiy1VuqoBDgHGQ49nJj4sk3Uj9qDxldtNdV5pnpKmKQDoIMGh\\\\\\\/v1mkOimWX5YPotv8pX4kFP\\\\\\\/8QFXCPx2ut4bD7LVElUNHfWpmoAYyredye3\\\\\\\/IGyRpbh7z+t4xRThProqky499eG24c33KOYvogLv4iMXb3QZ2RWzx33CY3JS+Ni6JuV1Uy59NeQrLg4X4USnRaWCBCwNiGYTTA7RmBGRymSXFenM\\\\\\\/+bWv3rM6tRdjHOemgtROICpL0FOdy2itnZLLxxAc80sswGY5XmtKQPiKrXPmc+icuSK5m8OtNMrnNn7+3Ee5fOeCZQ5fYsyepvQvJ3+ZLTtZ+cxxBYyRjlLX\\\\\\\/IgeIPz0TQOUlH2NV8N1m9Nz3v1e+D9+hABHIf6mSmB9RZKBnyJUKmIQ+qexkSfqDcAjE61Qsf8P3AYsr3UHM\\\\\\\/fRXehbelJprpcxC2aak6+BY2\\\\\\\/M15nuuOYJHMJVzmpU2JWF0prWjGGBseM7SVWNwlK0khesdutLDpxSU7AhOOSYJwMEc95jT57XwwAAGEPEDCGYk+Kkhi4hhpkp232zaBdQAEJ2yjEsblklbX96WiBURpi1suDglnKiBx4+is91M5nskBdZc4pU3re7InoAcJ4UniHf66uCV7q2n1N\\\\\\\/7\\\\\\\/VM2swSgUeEpqOqOQXsP4S2iaYec0IbteYTlHSElhZYtWsXdg5jvnoZYjhEJnK7ByBBSfdg7X5R14cNdK8tCQVq5jagZmjRN3bcEfXx3nv8HTEIc57\\\\\\\/VwyZRdkEq8QyOBMFm4szlnPi8cZXxvikUfBVcusptDzTpRvjy1dZ5QL+LS1AbgBlE0REn1Lm++gLZi3AgBjZmuUaOwKK1KeEwnfiCaNkWaa9BUn5e3jT0H7d3kT500AiwAZHk5+kDPvg\\\\\\\/X2L80gQ2nx8qx2FmmcoX8WtEaD0Hgeye+553vg6t1HfNXsg3lVLDNKnG01ryHQHzxHtH26JuA4efIyBzY8Yy9+jGCXd9PMpbkkRI75SFiHfYT8RJXPB8dLBUan65YS7iB\\\\\\\/Q+i6MhGYkKpkBHnlm2LuUxUw4cbnbfXVkovFbAsXkn4wndNgjDPKgikz9panS0a0k6KoVNQQDx6gw4eb3SCNdhR47n4WHFQ7HrMhnE88o6lnwLxCxO5nGNVea+wmkWcaGBZjEAzclqIj05ugKriMcKfmEqlCDSh5q8VxpE+NzVB9FHrhRyWS\\\\\\\/25+oCKr+SQg1Htdw7I6Ae7MSgraO5vNOzcXJJ\\\\\\\/I9ONNELzhr6u4y383FOy8nfWD6nkCWKI+xbRZKzctTIq+S8XkaYX7GUoR+5K1hTPCj10HoGu7tJ4AToH0hsWD+FijleqYj9u1+N9+HX+sWdGbpqWqoe3OMLIxuL01IcB21k4YFbp1bjMziBZTcXr7Em2r5lZ0EedmgMXmlbVHfvkb2dCpoXoJQZRi8HKN3oTMZibgixatfypqFxXPYwjYzsjzZk\\\\\\\/uOeHPhfv14cjxBgdIRrlgh\\\\\\\/DcqfhEDLF62SEb4Bk429YtwQ\\\\\\\/7UgXcpKc3t6V4ogqlLhQNxqGf3wNkyVplnJIf8TsjLI28kW0nc2DeXFAVHAVxJQYwydKthdOyREanYnqO4zSYSkAE7WCzHgzTcZn3wsBLYuMVc5yYXH1gDUOlZXTXXngJKh\\\\\\\/\\\\\\\/xbkL4qVo26E7BvUDJJlKl2OcZhMwATE3Wht0ofgLIzYt6n8WZIa+x4QMDAhf5lj3mPhxIARZi304TyR3kd7MVBbPD2DQ3Xb67tR5fsAPBq46FhEBhScy3epUSjoBEEpvTMDovGGyuIYfoWLKKmj+n4bMJ1XnmpiSDIXEPxvGsfVqtcTbcQSUYu\\\\\\\/KRzxv\\\\\\\/\\\\\\\/7T0OsZNLWURvTO3TLMTa3dp2c51VEJoTTEY5EAWcNdDcRBOBFt+o4NR5eC4AUiTyj9kOZUq7uPk2rkxA86UppnwHoysh6oES3DDbT7G8LEfjeal9l4NEFC9QPhpWMBLWndEnXII0UHagJP0lfn5mloAzLNDEJOhiFy09Tq6ETr13QfzlBjpLKgeEtI2JMuTmZaVcnSaFoVHjO2FpjU0jA2CYlEz7TERfpWvuhzG\\\\\\\/O53vEZx04TSV9ydkqPqTXoxPrVEXwm+m\\\\\\\/01LsmQRlM+sXt8A9cRDXlgllMGE17ioyvocpnTDb2jtz8S5pIENP4j8vLN6i2ueS4wLHFqz7AUGMVb+5ciZlU+T78vpQ5ykH6yjQ9MQfhVN8Iu8uHh9FV\\\\\\\/C1RycKbxN4syFOna8r5sw6eSWXoOiHTUMyRfFWPSkNlwC5VuzJxqHw1APQbw4kpKQZduDkJmALhePo25rkuWahn+5UXJM3P9cs8IAx4dofu4sYiO\\\\\\\/8l+gdW06oqA84rESI14t716u\\\\\\\/a6gJtTmPsR8XfRtz7QfrlOvRIFx5VEd5uJus\\\\\\\/wF2RoQo+Aey7v7r7t7U2PEA0Fmub3saAtFrISOxA2MQ0CLpmjDBkyP\\\\\\\/ZSIJAYbQHAagj0ttKy5z0n\\\\\\\/aS2NCUfErGM4h3lfZ+YaXZcWW0lz+cdSKqhxXMXI9jMMgcHfVv0rXdtTc8RyhmdbEQjKcOnUtABLmPV3vFyesEEShxgo8Aq8474D7C69tTnF4gl8T095cBdasjAFxInFMZEB4S6Iw2mszHMd\\\\\\\/3UzHJDd+jDzeG2zNBglKmYVBcfZ+heibNuxsk9UcHHWgUyM6OUF0XACXwhnNp6yfQ4X+ixC0mHaAsm99W8unPF2iJsBhJgku42PUhj3X4s9oyaJEF3H3HvEDdGDkLrFvcOtc42aKJV9Do19RFmFpSqpvtDSMfUBe9M0fOVyxmnuE\\\\\\\/BMWLO2iJQcNmBVJxlXShlXp9zqc2wGARwPkVlw\\\\\\\/7RCu81RxRDQVzXEgjiQ\\\\\\\/77bNtVXzDxusiemyqU0eVNzXx9WJ42I0iuDhqHAPvV8sLX4J1twMpoaO4Eh8G+7WdfhPNJDol9Lwfrzgq65NcXP0CUgeQQ4nsrm7pSyf2xcdJBonWnZq8R4oT1oYpLy1a1Q40qlkefEEHcoPTBtU4Glvt3N\\\\\\\/xKG6pvUhh3MXyZdoEri4v+Hsw1rp\\\\\\\/m6Y2rBtvu1s\\\\\\\/pEnkQDCA2h7V3gUiG+yoVcaJQZZyBTdtJ2Kp9KLmyFF2ZWKyUyirsfWxpkDHkK0Fl2R4glM9sxmyHXRazSh0+Omc3Upun3fmTXeUL5\\\\\\\/+hFoVUCjBOxr4HjJvE+vZBXWlKCl7DbozUoXa6zNiVSv3j2c1pbRzyNtM79g8huon+0TP98C5\\\\\\\/QrAAeg4hd1B7K89MVPNgQEfWIPHYwLNYnqMfptXfhppb8xfd1uHoitAB8722RuCQNQ1Ngd2WtJgcfNaFEBbpt17u9zWbOL8CgOJPVKnKxpTEXgppVb1h8Vh9yh7Tw541hcupk0PbFdLAp25QtXlp1F5lSAJ+24Dl7kY+1mNi\\\\\\\/F1Qqxv8fZasz8RPNXDfrXDuVjBMKJsSSIgJLTOJ4dU5GvdjDSmldg4+g3nog+rop2vIDUnqMyLxHjOJWkCQe2a0IYJly5vQ3JOJwq0KLDg8K\\\\\\\/eiCP8Iu3qAC82SlnWB7XsSo2wyCJZqePZhhQJiZY5rDIQ2VVIGmlx+N5cThSuFu2quYSHjozWIVBtcXou+Xf3DCNiXnu+DnXVYeXYrNmZUfxxvCH6qAMYH6O2oMIz3xD0kylNQEHxv5MN2aglUdK3ievUrHHDUP2nXKF70ilfuHf1x246b0V1eFeWau\\\\\\\/jQMsTOwnL6Om7pmEHc6DDn+hT2DpSJO6Xu1FY3BbC8iQyy4KeJEXY4Q6Fazi9tef8w\\\\\\\/U2oLeb4kYYV3OF\\\\\\\/YxaO8WxpcjLfl8C5pLEwYyibAcd2ClxO7Ozd4QdACwZg4dwmeG1f+pUC7nbEh3rSeBrrztvT6usRbBvvGKl7T4JFpu6KgCx1aEOhG2G3SSgE2g+GXffOqjfWSHbX66vfx0Wxq1KVaja6x3kF37JJV2qt+yJ3eJttlkqKCzodFLKJ+uci+rqmYk8HNmv1Y0ePYCDaEai7yLJhWiMjPAdJ5S\\\\\\\/BOkarHzUOcy+Wyn8SqRspKSk\\\\\\\/qV0jf+k+++S3l3fv4IxMqWtRPI1FFHiU+hnUEjZx4lXdX8yyHPp1JoUDUIgI\\\\\\\/bo8aBGkF1fajtlt8aBhYd8Ujx3NCo9QywL2HAm4LQpDlrrLGXKmm\\\\\\\/wRLXNiSKvkBMMzkYYIWh9IksYQiJ6nkxYqgAOl3C0kkN69qwxeQPf\\\\\\\/bUYRTLbodpU5nVDuy94EE6f2JQcXvWqeW\\\\\\\/VC0zh9rtBVI98YES34XiKsadtK9YTlR4fQwqgqp0CVKIPYG7qd43xpzZNeq3ciab8Tf4n01+kY0afpUJ9\\\\\\\/XWdR362Batqi+IgieP\\\\\\\/y4djrBd5DaNH9mwAU29LRB35OW0e4Q+q8tUkFU12sc1kOgd1S62qXkA\\\\\\\/DsWERMLr86U6miA8JrQvR\\\\\\\/4y8xkLuJUYt+A1LdgC5vsYTHdny2iAVERwuoUWuqjOe3w7UB9qkxC8kFMa0OquGcRYvm8eUv6QCwRh8i7CS9nYUing0ANi5nm\\\\\\\/A6yZrmL2W7PbbERk1DkigyqwwSArCivqnrrRfAwvxoOTUUFxmZGGEWdiut1vPm+GaQgKKgG6ZFY4gkY59FUzhH9urknR5LmHgMbET2tPdjbYTHp83a3He9u1QRNwulP6XJoCR1Zcnz8YfSaUQKiY1sUTjmJ1yexGYosLkpC4ovdiTwTcfD8kdGC3G4yM\\\\\\\/8TD\\\\\\\/9nJuwn4x210sy2tlT3hwl0wzL4Si+Qnca17vMNI9Wk2CGX2CzcBcQvbqkboOw0SGXPQV2lNlgT6vZu\\\\\\\/Rs8epF+iLfXU8n5DREZbdOCnaJNt5KsiaRcj+IFLN3czgxgLydlVY1wQGtpBIdMH\\\\\\\/gHLWUhF\\\\\\\/G5m9\\\\\\\/MZAqQ8+iwCZxcEQCnKjx\\\\\\\/4HNMBpl1zbVkwHkmM39OUlNV\\\\\\\/NufGNAG7\\\\\\\/wiMxGzt9OZzGwQSx0EAl9k2Rmi3gdi4PLeoUM7K0tdGiE7teqSSZWZ2Xl5pNU621+DdduxyBze9vv9NHPPWEitGQk\\\\\\\/\\\\\\\/E05rqK7eAf9FV4HbcYVB+yz11IW\\\\\\\/2K9somEfvZyxAxD59ntcolbfdOwS63sc+JbaHhkz1FKYsXnwRpxE4B8lRvjWhCOy7Im4Q1YohkTCS0hijlGLYzOPazg3o1wglngOIUEM4OEAsD16pyOW8uHxb\\\\\\\/sL9QFexOo5MuZubYsGiF3iuhcOaERukEfTaoNzKJmkjHSZdsrurmRT2fZWjEz\\\\\\\/fLRb9A\\\\\\\/zFboOAJcZ0Jha3lP1\\\\\\\/Oie8X2gO6cv5DWG7MZ4GDNUJY4FBeYCxSZjaDGRhNpOzqEUQzFiGFbzuuFw1UpLTgHo2x2J1u0DMfmN\\\\\\\/j2WwhdjRlILO11gtYxnHHOOgjcj1El24jrwwN\\\\\\\/yAVgHM3ngdxL4HTrCHoF6BHfPK+7cYZ623twRThS8wn6A4lnEEhh6KGu0YlYw86uZYKxDdOPkAiHQb57Q7Urq5QiV9xrsEnAoWhzZRLqEOtdSl1q2msxzJGvaULGBrrIc+PpxoxND3\\\\\\\/rnaNN8fwAa5o8SJfuUzw7BhUce50n+a\\\\\\\/q6HvMrakW4dvZqWtxU9aWkh\\\\\\\/T9Hc5fOl4xc76zvWqYOL9i7B+r\\\\\\\/JLqc\\\\\\\/XKpxcX2mtYa7y5PUxtkHlxmCMEfT1gRdNX2BQ8UPqF9Vz3YPIS7e2PSJZCDgspGs8DU7lTx36C3Onh9TFE5W7A1T6mSLl1QsVJ3zijijweSF0dTex0NF40Th2AbNi1n2KvuE1o7I21Tll1\\\\\\\/p35FvXdgaIW3abmJt+cQdk+\\\\\\\/GzX24+nceMZIKbWxUzBJKJX0lJ\\\\\\\/Z8Vj0tX7L48Ksn9WJSItr+v86qMZG5W9EdBmvUPGA\\\\\\\/igr4bmY5FH4e7Hr4i6CqFWQhLPaFIhrK4Xc3l1TkSLkkEfYBnrTZpkP1oxjtByDm3t4HdTnb7MzlTxX2+PcjjS67yuOdknARkr1oIXZllSw7DYpfY5qI1ieqUpwvnk0HROsNPZBhIceRwFsUvMm44g\\\\\\\/vGqrtpXTxdV3gZi\\\\\\\/68\\\\\\\/XlGE+oqqPhZnH04zdmgTZWn0MRXItxSYfHu+0kdBBq+AFDHrQOTTPL8oVM0u7t2kYVU6IrGt14YWSWF2ltzx3Uo80ciQCKzISz0tG4wkLxrOpzachlF5gFu8\\\\\\\/70NSnm7VUgh5\\\\\\\/qR2PlW2QS1mjuEebn5pg3ts\\\\\\\/zz+p+d0geswZqxoHxC18+a2WHYIXaYlL4psP6hqB\\\\\\\/vuK8zEWMgAg9MGXr+yVqk0sLoUNBijLibbxUu5nATjSSUTEsLsuMWbunpQCQkagXu2HbfKq6pJkEZsoQSRzw8epI33C4RJLcJfYJEYEc6euRSS2epXc0NJFYM0tjHt81tODRtt\\\\\\\/A1p57k1xV\\\\\\\/N4KIsl5+IZJkX4Pr051HkuqRsBqds7YFRTiDfD+y+8x6VpbAxy7Q2CzEDOIeVbvzhhwKL0TNPkZ4aoTIoVRh0oGDaebNBP76EkAOdkFiu7wkBl3EkPchxx2vJJVB\\\\\\\/0kPtXScOF19hWCRWsD+bcPPqDLQvNu7z7cFyAVJU0A\\\\\\\/qvP4lt6bxKnKuTtmxhGdEFzomJDB8nzwOLAgog9tEQ8VF5GURZ0A6gQPoJSM6EAqF3aUhxsfStvVbuKsvlHFSqjuMrvovtx9RwqppL+f4whrlnmDkzZURduhHGc\\\\\\\/BfNDW2iFmjklEc1qvpUPDr843AgXUHv2LFVKMBcvehO5T8\\\\\\\/nVY+bvOzOIZr0+4PXQZc+XYFEVMuso+1ZMrXPWp9ZxArbkxUzIUQMF+UsqXPtO+3s1L596wlSiQa9Y0Z4OopWBoD51LqkoGErS5BF55RuXjbum2gw0mwhV+W+vQLgnbHathX7jjKJY7T8lmiuqXyDa8fZA9VoFaCweGW\\\\\\\/FE5NHllsf8RB5mG6r376NuW4uqjPykC8RyaMjfkCmwegqx3bfzh9wxOcBjiBiInBzNETEX5vVrUdSCbizrMHd\\\\\\\/4VEhvk6vrJ7Jv8CQyws\\\\\\\/Pd239qEb6Ks+1gv5o8R11HhIfoK3BKKn78\\\\\\\/ByRs3SikGc0JrKJ347twjUpLloRqn8uqkZSFoQoCUHpo4K7GMzSco2HbPRjkPBTsa9w0VOtNB\\\\\\\/BIKHypyYupgJqOET4LMCfAOOFWDl6zpHDeBeXwet674RAEDcN++9Cz5FAimL+rXmNkXFGXVdM3yfANKqqU74GcD9qEUgOoYoMLRpZJ6dTp1xyhRoTI2u1id6T2h65pss6P\\\\\\\/5B\\\\\\\/xlkG7eed0uL0Y6+emaQK8VZ3+6ArQUBkZv1vZi5XPDvqz0haRyTkTv7ouYXS7kLXTy8FuiVl\\\\\\\/kz5LbCL8Acpe3VbZ5Qexxi05B\\\\\\\/Pfod+FVcTma7\\\\\\\/mvtfnOUTNhqqPD6jqMBsYzOVunXfc1yo\\\\\\\/ydkWrXWB6AsCveMMmNhnFgFixECq13aRMiTvlXKDvrLcyoO80G7I4sFG\\\\\\\/qZcxd9RHHVmgTvbDKTcKNVG8SvtBEaath7JHiqMdG+pDAedYH0lo3CYCU9asUonk1Pr9aHbHoj0VquCIYgzDa1mlXRqHeV7AW9mFN1GnoeiBTTp\\\\\\\/g3mRt4xWU\\\\\\\/2PV+QgOL5A\\\\\\\/o0mklK8l1f0Sh2E4hHkiT8A7qw25lMnvmvL6Aqm3D4IiPw7VdJ\\\\\\\/L4wMOsmRee0Ob0pR4SGUBvSIciV+cw1C\\\\\\\/hYZYLN8Dz7Fwadm7D0M\\\\\\\/lq6hmqxi14x8htDLAPaH6Nmm+DSkxEGQU1fMztb\\\\\\\/q\\\\\\\/tbHFKzyslZx\\\\\\\/wLC8KDS9SFuMp35uNk5oU1TLqpnj740MjxVz8riX+JRXZW4\\\\\\\/xB1f5q47QiZSKLTZtMLsH6cAm\\\\\\\/GfUqlqeoqOmMmDXrm1bP+DdRJPFlX5bmyRnKCsGJfAQ1TqxV6DKWBacgt1ldMdpZQspTiKUqp6dFAnJ06yiKC5dRZf4fEXgpJuHa154yY68UUrCvyJKmeJp4+Qb2rS+eKfDFCj77PzW\\\\\\\/C7ESdTD5b2X8Cr9HDYapQ5uCw4PG+Wq+9xTo9YcWQ3vPH6z8AJwv\\\\\\\/eNokCK\\\\\\\/vvi+UBSRA9C4qYgKygWeb0OT57\\\\\\\/fE8guwuIG1eBuqZpTyU2Tv\\\\\\\/8T69pw6qG6cZry7WTMmLsFcCO0kDOZZCAgWthDZsCehdgTvvlQ6uxITOa\\\\\\\/lT7APp9xMvsM2CY8em0RpWNcV1rgj1Jlbr\\\\\\\/ihoHaMQfgdT6a+uG4A\\\\\\\/bBcMdFLfaxNL0vDOlvWxnNNSR5PWWAyLHg2akrpvZrvAQCuICpWdbtt00nEEWXLrl1y7tUsrK6Hvxg2lPp1BQgujrLsxBqvvwHnTg+Dv3qaudYW8Wbzbm0OdiMl8xtTCfbgN0Yfate516mjE\\\\\\\/w6fpuoroREzhgtA8iYYggDeT8RAPqFJ60d+kMoNSkFK1WRzMSgfF2j9rUCmqDI2r1uqBuXjVR6KJVAl1PsCwb5gumolosfpLt4xHBtqy+7AVoUxjndWNj9NBiAGu1GphwbG9ukB750K4cExxKSPjDItFoXukYPHN\\\\\\\/6YTouAGICrEAUJfZyDnKKXHq5pAUz+lKvPzWIJLRR+P9QRr2WUuwx9NYu5D32nEmpSgTE6Z1ZtF+HD5N6V3tkYUlr2OLpXHrGcYZRNzIjtdgZl\\\\\\\/JqlGC42EXXKo17TXC57Xe\\\\\\\/\\\\\\\/uY4wguDGqqRFpHmhmz6fE9qCQkkacDz+kNv3Hg6KGgehzih0SBWalv\\\\\\\/OlvBUP493NlbzNVeycQ8CS0chnUEmlu8GK1GzB08I\\\\\\\/5LGgYGQWhjhqsEITyjmY2xl1gEOc3udgMxiIjAbZUK88KY4lXu6\\\\\\\/jnIyQvNw2QCrIs2xaC6Diqyf9JA3b8Yg8jULjVQ8mmNWRxCdWGKqCOSeAlxCD8qCLolVFCBD1MAb5OSnvNgJ9IxALEDVxdwQv3pGZZvxftgu1Itd5wJjQbnjXZegzTq5MOeMCkhAG9CxwIgZXRg4HpZGgVacFEI7GNgmFlRjsTMV2Qr5Hq5deeqXvlMElt\\\\\\\/4gZsnuYviJ\\\\\\\/IpE7CaymS2rRc7hyzW7GAoFmzt0zORAI4LnFWlh2NBW5ZALLY+EAEK1b3+n7guLqVM\\\\\\\/WEJal0Zv6ibCRBriirkLyx+emcd0UgBdK2Vohgt90AUqnyhp4Hg423Svr3qcTcqWLvj9fU+RL4nFlHHcxNcNsr+gnif4EVg0F4\\\\\\\/LoSN0tLT0TaW\\\\\\\/lHYJP1wK4buvYoHyoV46I9cJtyJW3ITslenwW5q0UxXidC8cf0hvfnVi4ncSWmu+d8JOYEk6d6faOA8Xqfo5yNuX16cqKoqPKx8epW6hroPFTjYrDbTRf9xp\\\\\\\/l0ESaFmX1F9IbNY6gF2ycU4solv+W2Bi0FuNeKQ\\\\\\\/h7fx69sqGiHmzOcHrtGv3iaKrzJN1k5Hy+i7U+uUelP00odkV3IxrCwmb9n8IcSaczwetkZGp9oAKFIvc3+MBx7SxdNlnq7zXRN3QtgtPK4bS16Vjoi9R+hdKfKX6qSD56RE4rOtfXink2QdfZ9wYRJiRuIMzg+Rko4gXBrxhDGWKc0bIkPBlt7CPXdNg6j27NNtogpwsaJO+ps\\\\\\\/qTBndJYEcZVWhKTS+egJyHokDVZdRH+Clwiuu4cKo5qsc3CYGsboMIOzQYQhvBRJLjFyD+hOXk0zHFGQHtmkIPDzaosjGj261UgM5v3dBem8bBorLcHj7dXMgrJmC22XjQW5IjEEZl9MxQX7debjW1Z+G6irIFGpJ9w\\\\\\\/EbR0Dlga8Re\\\\\\\/FftEKnwJu2SHVsIBZIjcvif6MfcE03sRiREPxjDdMxLhn\\\\\\\/zviqSWTS3HdxW4mtplOBoMRUC\\\\\\\/NYXx0GS128AqhMCGmJu5CB1w0K5nqAMdw7LIXiG8JLj84tHV55HrmtOeGuzHlzfKfC6vlQf3+uOOWmGcqDOpsE3BsqFt14vFcrpWINJLrlBV6O\\\\\\\/v0oj9t3mjknWIb0SQUXxmS6LP+HreyQLJ9+jwO\\\\\\\/jLqIdN8gKDmweu2jg0uMkXXh+hFLSGYex4sAvi2ukRX\\\\\\\/+EIrKflVF0Sbl\\\\\\\/A8JGiaT0mFm\\\\\\\/cs4BZru6lHWSB3R348vH5IQUY4dcSHhvyy67oeK+WW+gl9dvILT\\\\\\\/32mjl5oeKgOBzxAxDzlgRk6cho1bCm6+KbkmTI8ApdcoS1HZG\\\\\\\/+ApiNya6F9u6tNuxwddmmAnThzKGv1H0bKmJoKj69Hab9BBXwr9zIxQPRNhGIuK4K0NlMuynGW9E3r8gJuBWIQPPT1f4FJ3B6kpTaB7omUYoedGZcqkaagKSU2C83Q25Fgam2MaRLyBX+jpVZAgWSgiHJDGbt9sYr9x4CWcz6qPbhoFhfnFzW1TZqmCyBqNfXOpgGdr64EN3DApLCUDoNN7f9gw71NAI6QazCl1eKN4iPbHQnr3upIU+dVGhupMyu7LWSeoF5KXn92ENrwbitaaO7lELHWJzeTf+QjvfoCg4i3ImbHLySJfhmM22ePtv200nsRUWF3CCxLOgt0B4DEeYTeJMmqzOEKXfjW3K+rgOeW9H36Ksz3P8Sz+NfpXkt8KPPoVXkf4LJULkOeJd84DLxvbcFJCJhHTMA9iIncCiiBQq4PJWbICV5pxyIMflDHvP7aZJUfMA\\\\\\\/JM9icwAJp7qYCUvh29115\\\\\\\/mEGUm\\\\\\\/A8iASsaWAsSiKBT5yeSPoXzbr1Tb3pS1mDpDmZ4LkKl\\\\\\\/R3yVh9v8tyVFFC2cCeRzVu1sRonB+Sva3c8omS6YcEO2UwjPp04eiduCnlHquwYTqoyzwR2uKlFVFdcGublWP4fjANJEbqheZSOiz\\\\\\\/y1Dqw45eou8Duxdab4frvqMK+UoY5gvucg76gjtshzRec0Gey1WPmoCeeYyIoJ0Rp\\\\\\\/caKVlER0RBJCKImfy3JoY77iyrNA6dKj77D+XvJJSwnxFfR9fnTMm+vWheKlDGK\\\\\\\/JvlMvivhlyqws26+n\\\\\\\/ywpioO1u9mWfgpAwcGcjvvU6fdR8zU4hzoV+kw5YxODAIYQMvAkHCuWn2\\\\\\\/g9wiUyrfDKb33R41uhAHkS6GflHczNysH+oSgZT9j5ay4cbsG0SvH1KX61h\\\\\\\/ub8bFlBc7E1gYpGFOLvfgrhmWBak2PBfdIBsB9OSKuoyP08Z77et1AaxKzratklXjkpzNkVq5EUmwJ55sqdf12D\\\\\\\/MwkmS9BKg70Heq5l7ZjImlIncC5Hmq2JqdNjw+do++RnIkvnMz6b3WTx0CpkEqr75nPxdNP2LZx6r1OO3Ol\\\\\\\/itVeVjgULRN2G+pscGSlpNMqwD145ZJDF0LYeowqFtATI7Q2frM5HHfDX5ncf0F8aQ5jApxQkV8bDAECFWv3BkBECFuI9Z4iJMuxiXL996oyvngxOUHV8O0Q7L0Yd8\\\\\\\/KLgX+vbxA5X\\\\\\\/ZYM56H2p73HCA0sb66PJJkI8gdNu7SBvf+K3qV9QylCtYdvd7Umb\\\\\\\/KRscun6Jyw9\\\\\\\/tEL5GWmFW9SPZ94VoQ8A2PJzr+RB0ZwgqDBkSl0ekm+4+7hRHPnfjCfmJDkM1VdcsxjXcC3G9lV+B7985sgFJsTnD37wpg==\\\u0022,\\\u0022iv\\\u0022:\\\u0022ShDB8xDbRcZe0ZzYuuDdPQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002293a5bb3e29e54553a050011123c6990f5709f437f33776959096dcb8e93dc222\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022szhyP+LmFQoCWBFVh5KFXCL1MKA8eT3HAJTp977mrrnH5PmkhthS9Vq36CsPCZFMqChaqxaguW1RED+Kc53eszXQ4WxA\\\\\\\/mcBJuvWRzie+sYBhS7dd\\\\\\\/iifGiUwfMTXFCkb\\\\\\\/p8PZPjwVFuP03PwGL\\\\\\\/3cGOdMMgPVM3yapTP91v4NHavoDiThVIeHzJDCbG7mw9RBiF4YvO7Fn5eT2juOGiOJUv5QFTx6dnRxBVVU8Usnd8H0mkMnawl28R8jQraKdDDQTu60za4ELswW6Dpc0hJw51zCzwozxXgRclteIk+mlnHMC6nIhkDItqZaRjlBxA2tdQpQK7pln3JG+wtKxaW+IWZqjHoADtg7xrUU1mdJOYfpiB\\\\\\\/BLQ9cGI3sa9ADGHb245eetWfNht4N2rXXmmx6IDIRsMAGTgY0BSSeGnPO52\\\\\\\/\\\\\\\/4c46BInAP7kkmPMrfzBKwkOgefDvt+4XcLyusJuwA5cd+0PnpxKwFsdHOGP1S2cJ\\\\\\\/gWju\\\\\\\/asPYp4MPeGDLfEx36y1JXqnTuBVQb3bUJACs+0MY\\\\\\\/2WdAH\\\\\\\/Yex9KbmX1iipgVHYYCSULL1oibU0UUfmF1zblPItxzZk04CfIf8htcgOW0rS5wZi+m1w7bF5lmhPmji6h\\\\\\\/Go5IDGVgi3xfM55yNXDZtl9DzrH5dOz\\\\\\\/VSlgihbxi3iNV4NpzKdmNkmmWgactnRDahRyPiBLal\\\\\\\/t9edP+lenYs0eJ\\\\\\\/7dFnyAdBzV\\\\\\\/vFJ6fc\\\\\\\/AUYFmEBtE+BazCG\\\\\\\/eU4uTryOyPY3R6ovC46CC35MdBcHs1XhYG7nfQqmz5tb3a8ZaDMveAgM5Dztd\\\\\\\/CDdJUu82C3svcG6CKpaRfUpPGcvxXcWGcTw7RqhXv25506x888THFoUXBSR7D+FMjLXTa8DnMyW0xBNN5OrpAdkGdp8XWF7QnaxHAkq1tQ5XfWzwKZ+sHKhJsxuiu2CvPA6pVz47DuYBZz8+Od6eFAwBhZizKYT4XE0p6j\\\\\\\/\\\\\\\/HK0fKV9gHjSa6YQRv3JQ8az6jG\\\\\\\/8TJNQjThG2+w51j33LZRYSQjdnQBF8FPQMSgiI72GKvMb+PU64i3yKwDyCEpGcowIVCWubZiUy1fCUjqDEU1BVPHGI+gVLPdrkIlvZxp5ZdVLa+Y4VijEZjnuUC9\\\\\\\/I1yFDTSurIiWrd5el7+yrHKOZ\\\\\\\/4I\\\\\\\/a60VjtsvDgAOYyBfRnd8sE0Plp+Y5yHe+dpRaKqVn9q\\\\\\\/abq6lLDIQ6f1y6eSLSazzdJ4fukG+B53S\\\\\\\/lMBg+g96S3xUlWcQ6+X4vCxaqSCUSumgnOILQu8664mXS5og4qSv8UyF9cim3dviYK\\\\\\\/Ohl9HlCQyccUyzB5QW7Hc\\\\\\\/iHzHt1DY\\\\\\\/mWmzl2Jqy6qd1Jp1gNWflPfioc2ty6Dmsc\\\\\\\/stMPw96ZNPRg8UZ+cm0OOXLt8KawJ5q1\\\\\\\/PpiUCoJ8ak8WFK2uM9A9MLrx8oEYtTG9eDjkjWbSl7hspHBjf41UINCC2BC+RVAWTbVkS+lyv2xUcIRMC91nmGEwxeJlNS00BeVZbgVFQnykVT1qj3zZBU7TjEAr9ixBH0fwjwVsWtxGft5MSJR9CR86KWhlDMqKBRXG2yRW99mnzfwAAyR0333n1Skk14OnLXFw13yqn5O4cFetqw9zIydP+tLfa6Dha2bWRTwITd7ppcjb8ds9YfPwU0AM+HkzdjrnPhxxipL9BEZZk5pGzQr3NaVSBPZwpNcBsVawMZ5kW5qRi52eaenH+EyeOgu3FhBg0sz9oT0eNiIPdu3eURhAARzodtVVFkifEXAaOxJUOatJJMxgKpz082dJlLcs4CFr7VTWMV\\\\\\\/Fcdy7k33Ffd\\\\\\\/un8sH8rbNh1h7Xrxkx3icf++vNIRxaRO6jBTAsTy9G65bdW9NcVVA0FD1cHwlMexyTkFda+dc2R2XhyXPlffhKP4eJ+6LkkomORI4oAPMAhPH265HocSzmAPlUTbNSIWRH8LLa2i+ISiyXG8T6MuXgr4m2MkarLSAky4vAoY7vRFPHMRWJJvnuaJy42qsVw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022xHAJ7Sr6KChKETaUYW+j\\\\\\\/g==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022ca41f9d071e95d4acf345f67d0ef8fd36fdedbd56568ca3a46c206f7901aaed5\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-486349 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/9/3185" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-pOwpgQwUe7legOc_h3o75J5DNbckNTg3jd14KxiVGQ0" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/9/3185" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-kG9rNpcp3brsBQfvGA485F3CmfoE4evn8EXltpSbzfw" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/9/3177" title="The Stress-Inducible Peroxidase TSA2 Underlies a Conditionally Beneficial Chromosomal Duplication in Saccharomyces cerevisiae" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/9/3195" title="The Integrated Genomic Architecture and Evolution of Dental Divergence in East African Cichlid Fishes (Haplochromis chilotes x H. nyererei)" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="486349" id="node486349" data-pisa="ggg;7/9/3185" data-pisa-master="ggg;g3.117.1120" data-apath="/ggg/7/9/3185.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Ras/MAPK Modifier Loci Revealed by eQTL in <em>Caenorhabditis elegans</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-7119-6213" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Mark G. Sterken</span>, <span class="highwire-citation-author" data-delta="1">Linda van Bemmelen van der Plaat</span>, <span class="highwire-citation-author" data-delta="2">Joost A. G. Riksen</span>, <span class="highwire-citation-author" data-delta="3">Miriam Rodriguez</span>, <span class="highwire-citation-author" data-delta="4">Tobias Schmid</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0002-4098-3721" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Alex Hajnal</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0003-4822-4436" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jan E. Kammenga</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="7"><a href="http://orcid.org/0000-0001-5321-2996" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Basten L. Snoek</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3185-3193;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.1120 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node486349" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Mark G. Sterken </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Laboratory of Nematology, Wageningen University and Research, 6708PB, The Netherlands</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mark%2BG.%2BSterken%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sterken%20MG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMark%2BG.%2BSterken%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-7119-6213" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Mark G. Sterken</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Linda van Bemmelen van der Plaat </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Laboratory of Nematology, Wageningen University and Research, 6708PB, The Netherlands</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Linda%2Bvan%2BBemmelen%2Bvan%2Bder%2BPlaat%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=van%20Bemmelen%20van%20der%20Plaat%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALinda%2Bvan%2BBemmelen%2Bvan%2Bder%2BPlaat%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Joost A. G. Riksen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Laboratory of Nematology, Wageningen University and Research, 6708PB, The Netherlands</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joost%2BA.%2BG.%2BRiksen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Riksen%20JA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJoost%2BA.%2BG.%2BRiksen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Miriam Rodriguez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Laboratory of Nematology, Wageningen University and Research, 6708PB, The Netherlands</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Miriam%2BRodriguez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rodriguez%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMiriam%2BRodriguez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Tobias Schmid </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Institute of Molecular Life Sciences, University of Zurich, CH-8057, Switzerland</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Ph.D. Program in Molecular Life Sciences, University of Zurich, CH-8057, Switzerland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tobias%2BSchmid%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Schmid%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATobias%2BSchmid%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Alex Hajnal </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Institute of Molecular Life Sciences, University of Zurich, CH-8057, Switzerland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alex%2BHajnal%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hajnal%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAlex%2BHajnal%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-4098-3721" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Alex Hajnal</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Jan E. Kammenga </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Laboratory of Nematology, Wageningen University and Research, 6708PB, The Netherlands</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jan%2BE.%2BKammenga%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kammenga%20JE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJan%2BE.%2BKammenga%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-4822-4436" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jan E. Kammenga</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jan.kammenga@wur.nl" class="" data-icon-position="" data-hide-link-title="0">jan.kammenga@wur.nl</a>
<a href="mailto:l.b.snoek@uu.nl" class="" data-icon-position="" data-hide-link-title="0">l.b.snoek@uu.nl</a></span></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Basten L. Snoek </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Laboratory of Nematology, Wageningen University and Research, 6708PB, The Netherlands</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Basten%2BL.%2BSnoek%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Snoek%20BL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABasten%2BL.%2BSnoek%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-5321-2996" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Basten L. Snoek</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jan.kammenga@wur.nl" class="" data-icon-position="" data-hide-link-title="0">jan.kammenga@wur.nl</a>
<a href="mailto:l.b.snoek@uu.nl" class="" data-icon-position="" data-hide-link-title="0">l.b.snoek@uu.nl</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/7/9/3185" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:486349" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:486349/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/9/3185.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:486349" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:486349/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/9/3185.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:486349" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:486349/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/9/3185.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:486349" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:486349/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/7/9/3185.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:486349" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:486349/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">The oncogenic Ras/MAPK pathway is evolutionarily conserved across metazoans. Yet, almost all our knowledge on this pathway comes from studies using single genetic backgrounds, whereas mutational effects can be highly background dependent. Therefore, we lack insight in the interplay between genetic backgrounds and the Ras/MAPK-signaling pathway. Here, we used a <em>Caenorhabditis elegans</em> RIL population containing a gain-of-function mutation in the Ras/MAPK-pathway gene <em><a href="http://www.wormbase.org/db/get?name=WBGene00002335;class=Gene">let-60</a></em> and measured how gene expression regulation is affected by this mutation. We mapped eQTL and found that the majority (∼73%) of the 1516 detected <em>cis</em>-eQTL were not specific for the <em><a href="http://www.wormbase.org/db/get?name=WBGene00002335;class=Gene">let-60</a></em> mutation, whereas most (∼76%) of the 898 detected <em>trans</em>-eQTL were associated with the <em><a href="http://www.wormbase.org/db/get?name=WBGene00002335;class=Gene">let-60</a></em> mutation. We detected six eQTL <em>trans</em>-bands specific for the interaction between the genetic background and the mutation, one of which colocalized with the polymorphic Ras/MAPK modifier <em><a href="http://www.wormbase.org/db/get?name=WBGene00000138;class=Gene">amx-2</a></em>. Comparison between transgenic lines expressing allelic variants of <em><a href="http://www.wormbase.org/db/get?name=WBGene00000138;class=Gene">amx-2</a></em> showed the involvement of <em><a href="http://www.wormbase.org/db/get?name=WBGene00000138;class=Gene">amx-2</a></em> in 79% of the <em>trans</em>-eQTL for genes mapping to this <em>trans</em>-band. Together, our results have revealed hidden loci affecting Ras/MAPK signaling using sensitized backgrounds in <em>C. elegans</em>. These loci harbor putative polymorphic modifier genes that would not have been detected using mutant screens in single genetic backgrounds.</p></div><ul class="kwd-group"><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/caenorhabditis-elegans" class="hw-term hw-article-keyword hw-article-keyword-caenorhabditis-elegans" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Caenorhabditis elegans</em></a></li><li class="kwd"><a href="/keyword/rasmapk" class="hw-term hw-article-keyword hw-article-keyword-ras-mapk" rel="nofollow">RAS/MAPK</a></li><li class="kwd"><a href="/keyword/eqtl" class="hw-term hw-article-keyword hw-article-keyword-eqtl" rel="nofollow">eQTL</a></li><li class="kwd"><a href="/keyword/genetic-background" class="hw-term hw-article-keyword hw-article-keyword-genetic-background" rel="nofollow">genetic background</a></li></ul><p id="p-3">The Ras/MAPK pathway is highly conserved across s metazoans and regulates a wide range of physiological responses, such as cell proliferation, apoptosis, cell differentiation, and tissue morphogenesis (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Gokhale and Shingleton 2015</a>). In humans, activating (“gain-of-function”) mutations in HRas and KRas are strong tumor initiating mutations (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Prior and Hancock 2012</a>). Activation of MAP kinase components in model organisms has been shown to cause cell transformation and is implicated in tumorigenesis (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Cowley <em>et al.</em> 1994</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Mansour <em>et al.</em> 1994</a>). As a key pathway in vertebrates and invertebrates, Ras/MAPK signaling has been thoroughly studied in model organisms. Genetic studies in the model nematode <em>Caenorhabditis elegans</em> have provided insight into <em><a href="http://www.wormbase.org/db/get?name=WBGene00002335;class=Gene">let-60</a></em> Ras/MAPK signaling. Activated <a href="http://www.wormbase.org/db/get?name=WBGene00002335;class=Gene">LET-60</a>, a member of the GTP-binding RAS family (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Beitel <em>et al.</em> 1990</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Han and Sternberg 1990</a>), induces the phosphorylation of <a href="http://www.wormbase.org/db/get?name=WBGene00003030;class=Gene">LIN-45</a> (a Raf ortholog), <a href="http://www.wormbase.org/db/get?name=WBGene00003186;class=Gene">MEK-2</a> (a MAPK kinase), and <a href="http://www.wormbase.org/db/get?name=WBGene00003401;class=Gene">MPK-1</a> (an ERK ortholog) (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Wu and Han 1994</a>). After phosphorylation, <a href="http://www.wormbase.org/db/get?name=WBGene00003401;class=Gene">MPK-1</a> enters the nucleus where it regulates many genes by phosphorylation of transcription factors (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Tan <em>et al.</em> 1998</a>). Additionally, <em><a href="http://www.wormbase.org/db/get?name=WBGene00002335;class=Gene">let-60</a></em> activation underlies programmed cell death in <em>C. elegans</em> (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Jiang and Wu 2014</a>).</p><p id="p-4">In <em>C. elegans</em> almost all studies on <em><a href="http://www.wormbase.org/db/get?name=WBGene00002335;class=Gene">let-60</a></em> activation have been conducted with mutant screens using single genetic backgrounds (<em>i.e.</em>, a mutation in one genotype, in this case Bristol <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a>). However, the phenotype of induced mutations can vary widely depending on the genetic background (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Duveau and Felix 2012</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Chandler <em>et al.</em> 2014</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Schmid <em>et al.</em> 2015</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Kammenga 2017</a>). Induced mutations in one genetic background do not reveal the allelic effects that segregate in natural populations and contribute to phenotypic variation (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Kammenga <em>et al.</em> 2008</a>). At the moment, we lack insight into how genetic background effects modulate activated Ras/MAPK signaling and which genetic modifiers are involved in the underlying genetic architecture of gene expression.</p><p id="p-5">Here we go beyond mutant screens of <em><a href="http://www.wormbase.org/db/get?name=WBGene00002335;class=Gene">let-60</a>(gf)</em> in a single genetic background in <em>C. elegans</em> by incorporating the effects of multiple genetic backgrounds. This provides the opportunity to explore the genetic variation for identifying novel modifiers of the Ras/MAPK pathway. We recently mapped modifiers affecting Ras/MAPK signaling associated with vulval development in a population of recombinant inbred lines derived from a cross between wild-type Hawaiian <a href="http://www.wormbase.org/db/get?name=CB4856;class=Strain">CB4856</a> and Bristol <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a> (<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Schmid <em>et al.</em> 2015</a>). The lines were sensitized by introgression of the G13E gain-of-function mutation (<em><a href="http://www.wormbase.org/db/get?name=WBVar00089919;class=Variation">n1046</a></em>) in the Ras gene <em><a href="http://www.wormbase.org/db/get?name=WBGene00002335;class=Gene">let-60</a></em> [mutation introgression recombinant inbred lines (miRILs)]. This mutation is analogous to mutations causing excess cell division in human tumors (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Kyriakakis <em>et al.</em> 2015</a>). Hawaiian <a href="http://www.wormbase.org/db/get?name=CB4856;class=Strain">CB4856</a> males were crossed with Bristol <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a> <em><a href="http://www.wormbase.org/db/get?name=WBGene00002335;class=Gene">let-60</a></em> mutants. Random segregation of the two parental genomes was allowed, except for the <em><a href="http://www.wormbase.org/db/get?name=WBGene00002335;class=Gene">let-60</a></em> mutation which was kept homozygous from the F2 generation onwards. After 10 generations of self-fertilization, to drive all regions to homozygosity, independent miRILs were successfully obtained, each carrying the mutation. Quantitative trait loci (QTL) analysis on the vulval index (VI) of these miRILs in combination with allele-swapping experiments revealed the polymorphic monoamine oxidase A (MAOA) gene <em><a href="http://www.wormbase.org/db/get?name=WBGene00000138;class=Gene">amx-2</a></em> as a negative regulator of Ras/MAPK signaling (<a id="xref-ref-34-3" class="xref-bibr" href="#ref-34">Schmid <em>et al.</em> 2015</a>).</p><p id="p-6">Here, we extended the study by <a id="xref-ref-34-4" class="xref-bibr" href="#ref-34">Schmid <em>et al.</em> (2015)</a> by studying the transcriptional architecture of the same <em><a href="http://www.wormbase.org/db/get?name=WBGene00002335;class=Gene">let-60</a>(gf)</em>-sensitized miRILs. First, we measured the transcriptome of 33 miRILs using microarrays. Second, to gain further insight into the underlying molecular mechanisms, pathways, and new modifiers, we mapped gene expression QTL (eQTL). eQTL are polymorphic loci underlying variation in gene transcript abundances and can be used to detect loci which affect many transcript levels and transcriptional pathways (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Jansen and Nap 2001</a>). We found that the <em><a href="http://www.wormbase.org/db/get?name=WBGene00002335;class=Gene">let-60</a>(gf)</em> mutation mainly reveals novel <em>trans</em>-eQTL, which are eQTL distant from the regulated gene. In contrast, <em>cis</em>-eQTL, genes that colocalize with the eQTL, were mostly independent of the <em><a href="http://www.wormbase.org/db/get?name=WBGene00002335;class=Gene">let-60</a>(gf)</em> mutation. Of the <em>trans</em>-eQTL, 77 genes have an eQTL in a <em>trans</em>-band (eQTL hotspot) colocalizing with <em><a href="http://www.wormbase.org/db/get?name=WBGene00000138;class=Gene">amx-2</a></em>. Comparing the transcriptional profiles of transgenic lines with the <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a> <em><a href="http://www.wormbase.org/db/get?name=WBGene00000138;class=Gene">amx-2</a></em> allele with the <a href="http://www.wormbase.org/db/get?name=CB4856;class=Strain">CB4856</a> <em><a href="http://www.wormbase.org/db/get?name=WBGene00000138;class=Gene">amx-2</a></em> allele showed the involvement of <em><a href="http://www.wormbase.org/db/get?name=WBGene00000138;class=Gene">amx-2</a></em> in 79% (61/77) of the genes mapping to this locus. Through network-assisted gene expression analysis, we found evidence that <em><a href="http://www.wormbase.org/db/get?name=WBGene00000138;class=Gene">amx-2</a></em> indirectly affects gene expression mapping to this <em>trans</em>-band.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>General methods and strains used</h3><p id="p-7">The strains <a href="http://www.wormbase.org/db/get?name=MT2124;class=Strain">MT2124</a> [<a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a> background with the <em>let60(<a href="http://www.wormbase.org/db/get?name=WBVar00089919;class=Variation">n1046</a>)</em> gain-of-function mutation] and <a href="http://www.wormbase.org/db/get?name=CB4856;class=Strain">CB4856</a> were used, as were 33 miRILs described previously (<a id="xref-ref-34-5" class="xref-bibr" href="#ref-34">Schmid <em>et al.</em> 2015</a>). A file with the strains and the genetic map has been included (Supplemental Material, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.1120/-/DC1/TableS1.xlsx">Table S1</a>). Furthermore, transgenic lines <em><a href="http://www.wormbase.org/db/get?name=WBGene00000138;class=Gene">amx-2</a>(lf);Si[<a href="http://www.wormbase.org/db/get?name=WBGene00000138;class=Gene">amx-2</a>(Bristol)];<a href="http://www.wormbase.org/db/get?name=WBGene00002335;class=Gene">let-60</a>(gf)</em> and <em><a href="http://www.wormbase.org/db/get?name=WBGene00000138;class=Gene">amx-2</a>(lf);Si[<a href="http://www.wormbase.org/db/get?name=WBGene00000138;class=Gene">amx-2</a>(Hawaii)];<a href="http://www.wormbase.org/db/get?name=WBGene00002335;class=Gene">let-60</a>(gf)</em> containing the <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a> or <a href="http://www.wormbase.org/db/get?name=CB4856;class=Strain">CB4856</a> allele of <em><a href="http://www.wormbase.org/db/get?name=WBGene00000138;class=Gene">amx-2</a></em> in a <em><a href="http://www.wormbase.org/db/get?name=WBGene00000138;class=Gene">amx-2</a></em>(lf);<em><a href="http://www.wormbase.org/db/get?name=WBGene00002335;class=Gene">let-60</a></em>(gf) background were used in a confirmation experiment. For details regarding the construction of these lines, see <a id="xref-ref-34-6" class="xref-bibr" href="#ref-34">Schmid <em>et al.</em> (2015)</a>.</p><p id="p-8">Strains were maintained on NGM agar seeded with <a href="http://www.wormbase.org/db/get?name=OP50;class=Strain">OP50</a> bacteria at 20°, strains were age synchronized by bleaching and were harvested 72 hr postsynchronization. One sample consisted of multiple pooled, nonstarved populations from approximately four separate 9-cm NGM plates.</p></div><div id="sec-3" class="subsection"><h3>RNA isolation, cDNA synthesis, and cRNA synthesis</h3><p id="p-9">The RNA of the samples was isolated using the RNEasy Micro Kit from Qiagen (Hilden, Germany), following the “Purification of Total RNA from Animal and Human Tissues” protocol with a modified lysing procedure. As prescribed, 150 µl RLT buffer and 295 µl RNAse-free water were used to lyse the samples, but with an addition of 800 µg/ml proteinase K and 1% β-mercaptoethanol. This suspension was incubated for 30 min at 55° and 1000 rpm in a Thermomixer (Eppendorf, Hamburg, Germany). Thereafter the protocol as supplied by the manufacturer was followed.</p><p id="p-10">For gene expression analysis, 200 ng of RNA (as quantified by NanoDrop) was used in the “Two-Color Microarray-Based Gene Expression Analysis; Low Input Quick Amp Labeling” protocol, version 6.0, from Agilent (Agilent Technologies, Santa Clara, CA).</p></div><div id="sec-4" class="subsection"><h3>Microarray hybridization and scanning</h3><p id="p-11">The Agilent <em>C. elegans</em> (V2) Gene Expression Microarray 4X44K slides were used to measure gene expression. As recommended by the manufacturer, the samples were hybridized for 17 hr and scanned by an Agilent High Resolution C Scanner. The intensities were extracted using Agilent Feature Extraction Software (version 10.7.1.1). The data were normalized using the Limma package in “R” (version 3.3.0 x64). For within-array normalization, the “Loess” method was used and the “Quantile” method for between-array normalization (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Zahurak <em>et al.</em> 2007</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Smyth and Speed 2003</a>). Unless mentioned otherwise, the log<sub>2</sub> transformed intensities were used in subsequent analysis.</p></div><div id="sec-5" class="subsection"><h3>Expanding the genetic map</h3><p id="p-12">Previously, the strains were genotyped with 73 fragment length polymorphism (FLP) markers (as described in <a id="xref-ref-34-7" class="xref-bibr" href="#ref-34">Schmid <em>et al.</em> 2015</a> and <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Zipperlen <em>et al.</em> 2005</a>) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.1120/-/DC1/TableS2.docx">Table S2</a>).</p><p id="p-13">To increase the resolution of the genetic map, <em>cis</em>-eQTL of a large <em>C. elegans</em> eQTL experiment were used to further pinpoint the crossovers (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Rockman <em>et al.</em> 2010</a>). The gene expression of the miRILs was transformed to the mean of the two parental lines (R) by<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3185/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3185/embed/mml-math-1.gif"/></noscript></span></span>where <em>Y</em> stands for the untransformed intensities of strain <em>x</em> and spot <em>i</em> (1, 2, 3, ..., 45220). The obtained values were correlated to the <em>cis</em>-eQTL effects from <a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Rockman <em>et al.</em> (2010)</a>. Since the current study uses a newer version of the <em>C. elegans</em> microarray, only the spots that occurred in both designs were compared. The expression markers were generated following a procedure that is a variation of the method described in <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">West <em>et al.</em> (2006)</a>. Per 20 consecutive <em>cis</em>-eQTL, the correlation between the N2-eQTL effect and the transformed intensities (R) were calculated. In this way, the correlation would be negative if the genotype stemmed from <a href="http://www.wormbase.org/db/get?name=CB4856;class=Strain">CB4856</a>, and positive if the genotype stemmed from <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a>.</p><p id="p-14">In this way, 424 gene expression markers were generated per strain. To control for quality, the markers were filtered for calling the correct genotype in the four <a href="http://www.wormbase.org/db/get?name=MT2124;class=Strain">MT2124</a> replicas, the four <a href="http://www.wormbase.org/db/get?name=CB4856;class=Strain">CB4856</a> replicas, and in &gt;50% of the samples an absolute correlation &gt;0.5 was required. This pruning resulted in 204 reliable gene expression markers. The quality of these markers was controlled by predicting the genotype by expression in RIL AH2244, in which the gene expression was measured twice. There, we found that 10/204 selected gene expression markers were in disagreement with the genotype. This could be reduced to 0 by comparing only markers with an absolute correlation &gt;0.6. Therefore, all the correlations with an absolute value &gt;0.6 were assigned the predicted marker, which corresponds to an error rate of &lt;0.01 per strain. The genotypes of the remaining markers were manually inferred from the surrounding markers. Furthermore, the genotypes at the ends of the chromosomes were inferred from the distal most-assigned marker. This brings the size of the resulting map to 289 markers. The correlations and assigned genotypes can be found in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.1120/-/DC1/TableS3.xlsx">Table S3</a>.</p></div><div id="sec-6" class="subsection"><h3>Evaluating the genetic map</h3><p id="p-15">The 289 marker set was analyzed by correlation analysis for markers describing unique crossover events and to see if there are any strong linkages between chromosomes (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.1120/-/DC1/FigureS1.pdf">Figure S1</a>). This led to the selection of 247 markers indicating the border of a crossover event. To reduce the chances of false positives, we only used markers where the frequency of the least-occurring genotype in the population was &gt;15%. It has to be noted that this excluded most of chromosome IV [as the genotype was predominantly <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a> due to selection for strains including the <em><a href="http://www.wormbase.org/db/get?name=WBGene00002335;class=Gene">let-60</a>(gf)</em> mutation (<a id="xref-ref-34-8" class="xref-bibr" href="#ref-34">Schmid <em>et al.</em> 2015</a>)]. Furthermore, also the <em><a href="http://www.wormbase.org/db/get?name=WBGene00077563;class=Gene">peel-1</a></em>/<em><a href="http://www.wormbase.org/db/get?name=WBGene00021463;class=Gene">zeel-1</a></em> region on chromosome I was excluded for the same reason (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Seidel <em>et al.</em> 2008</a>).</p></div><div id="sec-7" class="subsection"><h3>eQTL mapping and threshold determination</h3><p id="p-16">Mapping of eQTL mapping was done in “R” (version 3.3.0 x64) and the gene expression data were fitted to the linear model<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3185/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3185/embed/mml-math-2.gif"/></noscript></span></span>where <em>y</em> is the log<sub>2</sub>-normalized intensity as measured by microarray of spot <em>i</em> (<em>i</em> = 1, 2, ..., 45220) of miRIL <em>j</em>. This is explained by the genotype (either <a href="http://www.wormbase.org/db/get?name=CB4856;class=Strain">CB4856</a> or <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a>) on marker location <em>x</em> (<em>x</em> = 1, 2, ..., 247) of miRIL <em>j</em>.</p><p id="p-17">A permutation approach was used to determine an empirical false discovery rate (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Snoek <em>et al.</em> 2017</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Vinuela <em>et al.</em> 2010</a>). First, the log<sub>2</sub>-normalized intensities were randomly distributed per gene over the genotypes. This randomized data set was tested using the eQTL mapping model. This procedure was repeated 10 times. The threshold was determined using<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3185/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3185/embed/mml-math-3.gif"/></noscript></span></span>where, at a specific significance level, the false discoveries (FDS) were the averaged outcome of the permutations and read discoveries (RDS) were the outcome of the eQTL mapping. The value of <em>m</em><sub>0,</sub> the number of true null hypotheses tested, was 45220 RDS; and for the value of <em>m</em>, the number of hypotheses tested, the number of spots (45220) was taken. Because this study only used a limited set of strains, a more lenient threshold of 0.1 was taken as the <em>q</em>-value (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Benjamini and Yekutieli 2001</a>), which resulted in a threshold of −log<sub>10</sub>(<em>p</em>) &gt; 3.2.</p></div><div id="sec-8" class="subsection"><h3>Statistical power calculations</h3><p id="p-18">The statistical power of the mapping was determined at the significance threshold of −log<sub>10</sub>(<em>p</em>) &gt; 3.2. Using the genetic map (33 strains with 247 markers), 10 QTL were simulated per marker location, explaining 20–80% of the variation (in increments of 5%). Next to a peak, random variation was also introduced, based on a standard normal distribution (μ = 0 and σ = 1). Peaks with corresponding explanatory power were simulated in this random variation (<em>e.g.</em>, a peak size of 1 corresponds to 20% explained variation). Furthermore, random data without peaks was also generated. These simulated data were mapped as described above and for each simulated peak it was determined: (i) if it was detected correctly, (ii) how precise the effect size estimation was, and (iii) how precise the location determined was. An overview of the results can be found in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.1120/-/DC1/TableS4.xlsx">Table S4</a>.</p></div><div id="sec-9" class="subsection"><h3>eQTL analysis</h3><p id="p-19">The distinction between <em>cis</em>- and <em>trans</em>-eQTL was made on the distance between the gene and the eQTL peak. Given the relatively low number of unique recombinations (due to the limited set of strains), the <em>cis</em>-eQTL window was set at 2 Mb. This means that if a gene lies within 2 Mb of the QTL peak or the confidence interval, it is called a <em>cis</em>-eQTL. The confidence interval was based on a −log<sub>10</sub>(<em>p</em>) drop of 1.5 compared to the peak.</p><p id="p-20"><em>Trans</em>-bands were identified based on a Poisson distribution of the mapped <em>trans</em>-eQTL (as in <a id="xref-ref-33-3" class="xref-bibr" href="#ref-33">Rockman <em>et al.</em> 2010</a>). The number of <em>trans</em>-eQTL was counted per 1-Mb bin, leading to the identification of 52 bins with <em>trans</em>-eQTL. Since we mapped 1149 <em>trans</em>-eQTL peaks (spots) to 52 bins, we expected 22.10 <em>trans</em>-eQTL per bin with <em>trans</em>-eQTL assigned to it. Based on a Poisson distribution, any bin linking &gt;30 spots with a <em>trans</em>-eQTL would have a significance of <em>P</em> &lt; 0.05, and &gt;38 <em>trans</em>-eQTL would yield a significance of <em>P</em> &lt; 0.001. In this way, six <em>trans</em>-band loci were identified.</p><p id="p-21">Comparative analysis against previous eQTL experiments was done on remapped experiments downloaded from WormQTL (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Snoek <em>et al.</em> 2013</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">van der Velde <em>et al.</em> 2014</a>) and EleQTL (<a href="http://www.bioinformatics.nl/EleQTL">http://www.bioinformatics.nl/EleQTL</a>). This data set consists of four different experiments representing nine different conditions. The first set contains eQTL in two temperature conditions, 16° and 24°, measured in the L3 stage (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Li <em>et al.</em> 2006</a>). The second set contains eQTL over three life stages: L4 juvenile animals grown at 24°, reproducing adult animals (96 hr) grown at 24°, and aging animals (214 hr) grown at 24° (<a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">Vinuela <em>et al.</em> 2010</a>). The third set contains eQTL from a single experimental condition (young adults grown at 20°) measured on a large RIL panel (<a id="xref-ref-33-4" class="xref-bibr" href="#ref-33">Rockman <em>et al.</em> 2010</a>). The fourth set contains eQTL from three experimental conditions over the course of a heat-shock treatment: a control condition (L4 animals grown for 48 hr at 20°), a heat-shock condition (L4 animals grown for 46 hr at 20° and exposed to 2 hr of 35°), and a recovery condition (similar to heat shock, only followed by 2 hr at 20°) (<a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Snoek <em>et al.</em> 2017</a>). Each of these experiments was compared at FDR = 0.05 to the eQTL mapped at FDR = 0.10 in the <em><a href="http://www.wormbase.org/db/get?name=WBGene00002335;class=Gene">let-60</a>(gf)</em>-sensitized miRILs.</p><p id="p-22">Since the data set of <a id="xref-ref-33-5" class="xref-bibr" href="#ref-33">Rockman <em>et al.</em> (2010)</a> was used for expanding the genetic map, analysis was also conducted excluding this study. The reason for excluding this set from analysis was that a bias could be introduced in overlap with the <em>cis</em>-eQTL. The main conclusion that <em>cis</em>-eQTL are less unique than <em>trans</em>-eQTL also stands with this analysis. Excluding the Rockman <em>et al.</em> data, resulted in detection of 999/1516 (65.9%) <em>cis</em>-eQTL and 171/898 (19.0%) <em>trans</em>-eQTL present in previous experiments.</p></div><div id="sec-10" class="subsection"><h3>Enrichment analysis</h3><p id="p-23">Gene group enrichment analysis was done using a hypergeometric test on the unique genes (not on the spots) with the following criteria: Bonferroni-corrected <em>P</em> &lt; 0.05; size of the category, <em>n</em> &gt; 3; and size of the overlap, <em>n</em> &gt; 2.</p><p id="p-24">The following databases were used: the WormBase (<a href="http://www.wormbase.org">www.wormbase.org</a>) WS220 gene class annotations, WS256 GO-annotation, anatomy terms, phenotypes, RNAi phenotypes, developmental stage expression, and disease-related genes (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Harris <em>et al.</em> 2014</a>); the MODENCODE release 32 transcription-factor binding sites (<a href="http://www.modencode.org">www.modencode.org</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Gerstein <em>et al.</em> 2010</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Niu <em>et al.</em> 2011</a>), which were mapped to transcription start sites (according to <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Tepper <em>et al.</em> 2013</a>); and the KEGG (Kyoto Encyclopedia of Genes and Genomes) pathway release 65.0 (<a href="http://www.genome.jp/kegg/">www.genome.jp/kegg/</a>) (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Ogata <em>et al.</em> 1999</a>).</p></div><div id="sec-11" class="subsection"><h3>amx-2 allelic comparison</h3><p id="p-25">Four independent transgenic lines, two of <em><a href="http://www.wormbase.org/db/get?name=WBGene00000138;class=Gene">amx-2</a>(lf);Si[<a href="http://www.wormbase.org/db/get?name=WBGene00000138;class=Gene">amx-2</a>(Bristol)];<a href="http://www.wormbase.org/db/get?name=WBGene00002335;class=Gene">let-60</a>(gf)</em> and two of <em><a href="http://www.wormbase.org/db/get?name=WBGene00000138;class=Gene">amx-2</a>(lf);Si[<a href="http://www.wormbase.org/db/get?name=WBGene00000138;class=Gene">amx-2</a>(Hawaii)];<a href="http://www.wormbase.org/db/get?name=WBGene00002335;class=Gene">let-60</a>(gf)</em>, were used to investigate the effect of the <em><a href="http://www.wormbase.org/db/get?name=WBGene00000138;class=Gene">amx-2</a></em> <a href="http://www.wormbase.org/db/get?name=CB4856;class=Strain">CB4856</a> and <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a> allele on gene expression. These lines contain single-copy insertions on LGII of the <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a> or <a href="http://www.wormbase.org/db/get?name=CB4856;class=Strain">CB4856</a> alleles of <em><a href="http://www.wormbase.org/db/get?name=WBGene00000138;class=Gene">amx-2</a></em> in an <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a> <em><a href="http://www.wormbase.org/db/get?name=WBGene00000138;class=Gene">amx-2</a></em>(lf); <em><a href="http://www.wormbase.org/db/get?name=WBGene00002335;class=Gene">let-60</a></em>(gf) background. The effect of different alleles on gene expression was measured by microarray for each independent transgenic line and compared to the effects of the eQTL mapping to the <em>trans</em>-band closest to the position of <em><a href="http://www.wormbase.org/db/get?name=WBGene00000138;class=Gene">amx-2</a></em>.</p></div><div id="sec-12" class="subsection"><h3>Connectivity network analysis</h3><p id="p-26">To investigate the connectivity and function of the affected genes we used WormNet (version 3) (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Cho <em>et al.</em> 2014</a>) and GeneMANIA plug-in for Cytoscape (version 3.4.0) (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Montojo <em>et al.</em> 2010</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Shannon <em>et al.</em> 2003</a>). WormNet was used to investigate enrichment in connectivity within groups of genes. For example, groups of coexpressed genes mapping to the same <em>trans</em>-band were assumed to share the same regulator. Further evidence for this coexpression and regulation can be found in WormNet if these genes are significantly more connected than by chance. GeneMANIA was used to find the closest neighbors, those genes with the most connections, of <em><a href="http://www.wormbase.org/db/get?name=WBGene00000138;class=Gene">amx-2</a></em> and <em><a href="http://www.wormbase.org/db/get?name=WBGene00002335;class=Gene">let-60</a></em> to locate the genes by which <em><a href="http://www.wormbase.org/db/get?name=WBGene00000138;class=Gene">amx-2</a></em> modifies RAS signaling.</p></div><div id="sec-13" class="subsection"><h3>Data availability</h3><p id="p-27">All strains used can be requested from the authors. The transcriptome data sets generated and the mapped eQTL profiles can be interactively accessed via <a href="http://www.bioinformatics.nl/EleQTL">http://www.bioinformatics.nl/EleQTL</a>. Moreover, the raw transcriptomic data are also available at ArrayExpress (E-MTAB-5856).</p></div></div><div class="section results" id="sec-14"><h2 class="">Results</h2><div id="sec-15" class="subsection"><h3>eQTL mapping in miRIL population</h3><p id="p-28">Using microarrays, we measured gene expression levels in 33 miRILs sensitized for RAS/MAPK signaling by introgression of a <em><a href="http://www.wormbase.org/db/get?name=WBGene00002335;class=Gene">let-60</a></em> (gf) mutation in a segregated <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a>/<a href="http://www.wormbase.org/db/get?name=CB4856;class=Strain">CB4856</a> genetic background (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.1120/-/DC1/FigureS2.pdf">Figure S2</a>) (<a id="xref-ref-34-9" class="xref-bibr" href="#ref-34">Schmid <em>et al.</em> 2015</a>). Analysis of the statistical power of this population with a genetic map of 247 informative markers showed that we can detect 77% of the QTL explaining 40% of the variation (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.1120/-/DC1/TableS4.xlsx">Table S4</a>). We detected 2303 genes (represented by 3226 array spots) with at least one eQTL (FDR = 0.1, −log<sub>10</sub>(<em>p</em>) &gt; 3.2; <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>, <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.1120/-/DC1/TableS5.xlsx">Table S5</a>). For 1516 of these genes, a <em>cis</em>-eQTL was found, indicating regulation from within a window of 2 Mb around the affected gene. For 898 genes a <em>trans</em>-eQTL was found, showing regulation more distal from the affected transcript. Most <em>cis</em>-eQTL (1074 out of 1516; ∼71%) show a positive effect for the <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a> allele; whereas for the <em>trans</em>-eQTL, the positive and negative effects were equally distributed over the <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a> and <a href="http://www.wormbase.org/db/get?name=CB4856;class=Strain">CB4856</a> allele (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/9/3185/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="eQTL mapping in the let-60(gf) miRIL population. On top, the cis-trans plot is shown. On the horizontal axes, the positions of the eQTL are plotted along the six chromosomes. The location of the genes with an eQTL is plotted on the vertical axis. The black &#x25CF;&#x2019;s represent cis-eQTL (lying within 2 Mb of the regulated gene), the green &#x25CF;&#x2019;s represent trans-eQTL (eQTL lying elsewhere). The gray horizontal lines indicate the confidence interval of the QTL location [based on a 1.5 drop in &#x2212;log10(p)]. The bottom histogram shows the number of eQTL per genomic location. Note that on chromosome IV, hardly any eQTL are mapped, this is because of linkage of the let-60(gf) mutation (which is located at IV: 11.7 Mb)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-469627366" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;eQTL mapping in the &lt;em&gt;let-60(gf)&lt;/em&gt; miRIL population. On top, the &lt;em&gt;cis-trans&lt;/em&gt; plot is shown. On the horizontal axes, the positions of the eQTL are plotted along the six chromosomes. The location of the genes with an eQTL is plotted on the vertical axis. The black &#x25CF;&#x2019;s represent &lt;em&gt;cis&lt;/em&gt;-eQTL (lying within 2 Mb of the regulated gene), the green &#x25CF;&#x2019;s represent &lt;em&gt;trans&lt;/em&gt;-eQTL (eQTL lying elsewhere). The gray horizontal lines indicate the confidence interval of the QTL location [based on a 1.5 drop in &#x2212;log&lt;sub&gt;10&lt;/sub&gt;(p)]. The bottom histogram shows the number of eQTL per genomic location. Note that on chromosome IV, hardly any eQTL are mapped, this is because of linkage of the &lt;em&gt;let-60(gf)&lt;/em&gt; mutation (which is located at IV: 11.7 Mb).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/9/3185/F1.medium.gif" width="350" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/7/9/3185/F1.medium.gif" width="350" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/9/3185/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/9/3185/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/486571" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-29" class="first-child">eQTL mapping in the <em>let-60(gf)</em> miRIL population. On top, the <em>cis-trans</em> plot is shown. On the horizontal axes, the positions of the eQTL are plotted along the six chromosomes. The location of the genes with an eQTL is plotted on the vertical axis. The black ●’s represent <em>cis</em>-eQTL (lying within 2 Mb of the regulated gene), the green ●’s represent <em>trans</em>-eQTL (eQTL lying elsewhere). The gray horizontal lines indicate the confidence interval of the QTL location [based on a 1.5 drop in −log<sub>10</sub>(p)]. The bottom histogram shows the number of eQTL per genomic location. Note that on chromosome IV, hardly any eQTL are mapped, this is because of linkage of the <em>let-60(gf)</em> mutation (which is located at IV: 11.7 Mb).</p><div class="sb-div caption-clear"></div></div></div><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/486485/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/486485/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">The number of genes with an eQTL</span><div class="sb-div caption-clear"></div></div></div><p id="p-32">The eQTL were not equally distributed across the genome. We detected different clustered <em>trans</em>-eQTL as “hotspots” or <em>trans</em>-bands. <em>Trans</em>-bands are frequently found in eQTL experiments and indicate a locus with one or multiple alleles affecting the expression of many distant genes. To identify the <em>trans</em>-bands in our experiment, we calculated the chance of overrepresentation of <em>trans</em>-eQTL per marker, using a Poisson distribution (as in <a id="xref-ref-33-6" class="xref-bibr" href="#ref-33">Rockman <em>et al.</em> 2010</a>). This method was applied per 1-Mb window on the genome. The <em>trans</em>-bands that were identified in adjacent windows were merged. This way, six <em>trans</em>-bands were identified (Poisson distribution, <em>P</em> &lt; 0.001; <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>).</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/486384/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/486384/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title"><em>Trans</em>-bands detected in the sensitized miRILs</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-16" class="subsection"><h3>Specificity of let-60 eQTL</h3><p id="p-35">By comparative analysis of different environments and/or populations, it has been shown that eQTL <em>trans</em>-bands can be population, environment, or age specific (for example, see <a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Li <em>et al.</em> 2006</a>). To investigate if and which eQTL are specifically found in the sensitized miRILs, we compared the detected eQTL to the eQTL found in nonsensitized RILs (<a id="xref-ref-24-3" class="xref-bibr" href="#ref-24">Li <em>et al.</em> 2006</a>; <a id="xref-ref-33-8" class="xref-bibr" href="#ref-33">Rockman <em>et al.</em> 2010</a>; <a id="xref-ref-48-4" class="xref-bibr" href="#ref-48">Vinuela <em>et al.</em> 2010</a>; <a id="xref-ref-38-4" class="xref-bibr" href="#ref-38">Snoek <em>et al.</em> 2017</a>). These data sets contain eQTL mapped over nine different conditions (see <em>Materials and Methods</em>), which were compared at an FDR = 0.05 with the eQTL mapped in the sensitized miRILs. We found that ∼73% (1112 out of 1516) of the genes with a <em>cis</em>-eQTL in the miRIL population were also found in one or more of the previous studies (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.1120/-/DC1/FigureS3.tiff">Figure S3</a>B). When taking the allelic effects into account, of the 1074 genes with a <em>cis</em>-eQTL that gave a higher expression linked to the <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a> allele, 816 (∼76%) were found in other studies. For the 456 genes with a <em>cis</em>-eQTL that gave a higher expression linked to the <a href="http://www.wormbase.org/db/get?name=CB4856;class=Strain">CB4856</a> allele, 310 (∼68%) were found in other studies. This shows that the majority of the <em>cis</em>-eQTL detected in the miRIL population can be also be detected in other experiments.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/9/3185/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="eQTL detected in previous experiments. The eQTL mapped in the let-60(gf) RIL population were compared to eQTL mapped in three published C. elegans eQTL studies, representing nine different conditions (Li et al. 2006; Rockman et al. 2010; Vinuela et al. 2010; Snoek et al. 2017). The histogram shows how many of the eQTL mapped in this study were found back in these independent conditions. Whereas the majority of cis-eQTL was mapped in previous studies, there was only a minority of trans-eQTL that had been mapped." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-469627366" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;eQTL detected in previous experiments. The eQTL mapped in the &lt;em&gt;let-60(gf)&lt;/em&gt; RIL population were compared to eQTL mapped in three published &lt;em&gt;C. elegans&lt;/em&gt; eQTL studies, representing nine different conditions (Li &lt;em&gt;et al.&lt;/em&gt; 2006; Rockman &lt;em&gt;et al.&lt;/em&gt; 2010; Vinuela &lt;em&gt;et al.&lt;/em&gt; 2010; Snoek &lt;em&gt;et al.&lt;/em&gt; 2017). The histogram shows how many of the eQTL mapped in this study were found back in these independent conditions. Whereas the majority of &lt;em&gt;cis&lt;/em&gt;-eQTL was mapped in previous studies, there was only a minority of &lt;em&gt;trans&lt;/em&gt;-eQTL that had been mapped.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/9/3185/F2.medium.gif" width="440" height="352"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/7/9/3185/F2.medium.gif" width="440" height="352"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/9/3185/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/9/3185/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/486590" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-36" class="first-child">eQTL detected in previous experiments. The eQTL mapped in the <em>let-60(gf)</em> RIL population were compared to eQTL mapped in three published <em>C. elegans</em> eQTL studies, representing nine different conditions (<a id="xref-ref-24-4" class="xref-bibr" href="#ref-24">Li <em>et al.</em> 2006</a>; <a id="xref-ref-33-9" class="xref-bibr" href="#ref-33">Rockman <em>et al.</em> 2010</a>; <a id="xref-ref-48-5" class="xref-bibr" href="#ref-48">Vinuela <em>et al.</em> 2010</a>; <a id="xref-ref-38-5" class="xref-bibr" href="#ref-38">Snoek <em>et al.</em> 2017</a>). The histogram shows how many of the eQTL mapped in this study were found back in these independent conditions. Whereas the majority of <em>cis</em>-eQTL was mapped in previous studies, there was only a minority of <em>trans</em>-eQTL that had been mapped.</p><div class="sb-div caption-clear"></div></div></div><p id="p-37">In contrast, for the <em>trans</em>-eQTL detected in our experiment, not taking location into account, we only found ∼24% (214 out of 898) of the genes had a <em>trans</em>-eQTL in at least one of the previous experiments (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.1120/-/DC1/FigureS3.tiff">Figure S3</a>C). To further compare the <em>trans</em>-eQTL overlap, it was investigated whether the <em>trans</em>-eQTL colocalized on the same chromosome. Using that criterion, ∼9% (80 out of 898) of the genes with a <em>trans</em>-eQTL were found in a previous study. This showed that the majority of the genes with a <em>trans</em>-eQTL were specifically detected in the sensitized miRIL population. This observation also extended to the <em>trans</em>-bands. To test the specificity of the <em>trans</em>-bands, we counted the number of times a gene within the miRIL <em>trans</em>-bands had a colocating <em>trans</em>-eQTL in other studies. All <em>trans</em>-bands were found to be specific for the <em><a href="http://www.wormbase.org/db/get?name=WBGene00002335;class=Gene">let-60</a>(gf)</em> miRILs (Poisson distribution, <em>P</em> &lt; 0.001; <a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>). This provides additional evidence that these <em>trans</em>-bands are specific for interaction between the genetic background and the <em><a href="http://www.wormbase.org/db/get?name=WBGene00002335;class=Gene">let-60</a>(gf)</em> mutation.</p><p id="p-38">Altogether, we identified 404 genes with a novel <em>cis</em>-eQTL and 684 genes with a novel <em>trans</em>-eQTL (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>). As our method can be biased by the way the genetic map was constructed (using expression differences, see <em>Materials and Methods</em>), we conducted the same analyses using the original FLP map. Our conclusions on the detection of novel <em>cis</em>- and <em>trans</em>-eQTL were not affected by the different genetic map (for details, see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.1120/-/DC1/FileS1.docx">File S1</a>). This implies that a substantial majority of <em>trans</em>-eQTL showed a mutation-dependent induction. This is also illustrated by showing the eQTL that were consistently found (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3B</a>), which mainly shows the <em>cis</em>-diagonal- and a few <em>trans</em>-eQTL.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/9/3185/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="eQTL mapped in the let-60(gf) miRIL population compared to other studies (Li et al. 2006; Rockman et al. 2010; Vinuela et al. 2010; Snoek et al. 2017). As in Figure 2, the cis- and trans-eQTL are plotted. (A) eQTL only found in the miRIL population. (B) eQTL found over multiple studies." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-469627366" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;eQTL mapped in the &lt;em&gt;let-60(gf)&lt;/em&gt; miRIL population compared to other studies (Li &lt;em&gt;et al.&lt;/em&gt; 2006; Rockman &lt;em&gt;et al.&lt;/em&gt; 2010; Vinuela &lt;em&gt;et al.&lt;/em&gt; 2010; Snoek &lt;em&gt;et al.&lt;/em&gt; 2017). As in Figure 2, the &lt;em&gt;cis&lt;/em&gt;- and &lt;em&gt;trans&lt;/em&gt;-eQTL are plotted. (A) eQTL only found in the miRIL population. (B) eQTL found over multiple studies.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/9/3185/F3.medium.gif" width="440" height="266"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/7/9/3185/F3.medium.gif" width="440" height="266"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/9/3185/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/9/3185/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/486514" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-39" class="first-child">eQTL mapped in the <em>let-60(gf)</em> miRIL population compared to other studies (<a id="xref-ref-24-5" class="xref-bibr" href="#ref-24">Li <em>et al.</em> 2006</a>; <a id="xref-ref-33-10" class="xref-bibr" href="#ref-33">Rockman <em>et al.</em> 2010</a>; <a id="xref-ref-48-6" class="xref-bibr" href="#ref-48">Vinuela <em>et al.</em> 2010</a>; <a id="xref-ref-38-6" class="xref-bibr" href="#ref-38">Snoek <em>et al.</em> 2017</a>). As in <a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a>, the <em>cis</em>- and <em>trans</em>-eQTL are plotted. (A) eQTL only found in the miRIL population. (B) eQTL found over multiple studies.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-17" class="subsection"><h3>Functional allelic differences</h3><p id="p-40">Enrichment analysis on groups of genes with an eQTL mapping to the same <em>trans</em>-band can be used to uncover the functional consequences of the allelic variation at the selected locus. Enrichment analysis was done on GO-terms, KEGG, anatomy terms, disease phenotypes, development expression, phenotypes, transcription-factor binding sites, and RNAi phenotypes (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.1120/-/DC1/TableS6.xlsx">Table S6</a>).</p><p id="p-41">The genes with <em>cis</em>-eQTL were enriched for genes in the gene classes <em>math</em>, <em>bath</em>, <em>btb</em>, and <em>fbx</em>. These classes contain many genes that function as intra- and extracellular sensors involved in gene–environment interactions, and were found highly polymorphic between <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a> and <a href="http://www.wormbase.org/db/get?name=CB4856;class=Strain">CB4856</a>, but also between other wild isolates (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Thompson <em>et al.</em> 2015</a>; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Volkers <em>et al.</em> 2013</a>). The genes with <em>trans</em>-eQTL with higher expression linked to <a href="http://www.wormbase.org/db/get?name=CB4856;class=Strain">CB4856</a> loci were enriched for genes related to development in general and gonad development specifically. These enrichments were likely to stem from the <em>trans</em>-bands on chromosome I: 12–15 Mb and II: 3–6 Mb specifically. These two <em>trans</em>-bands mainly consisted of genes with a higher expression linked to the <a href="http://www.wormbase.org/db/get?name=CB4856;class=Strain">CB4856</a> locus and were enriched for genes associated with reproduction and/or development.</p></div><div id="sec-18" class="subsection"><h3>Allelic effect of amx-2 on gene expression</h3><p id="p-42">The <em>trans</em>-band on chromosome I: 12–15 Mb colocalizes with a previously identified QTL for vulval induction, for which <em><a href="http://www.wormbase.org/db/get?name=WBGene00000138;class=Gene">amx-2</a></em> was identified as polymorphic regulator (<a id="xref-ref-34-10" class="xref-bibr" href="#ref-34">Schmid <em>et al.</em> 2015</a>). To determine if the <em>trans</em>-eQTL mapping to the <em><a href="http://www.wormbase.org/db/get?name=WBGene00000138;class=Gene">amx-2</a></em> locus were affected by the allelic difference between <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a> and <a href="http://www.wormbase.org/db/get?name=CB4856;class=Strain">CB4856</a>, we measured gene expression in transgenic lines containing single-copy insertions of either allele of <em><a href="http://www.wormbase.org/db/get?name=WBGene00000138;class=Gene">amx-2</a></em> in a <em><a href="http://www.wormbase.org/db/get?name=WBGene00000138;class=Gene">amx-2</a>(lf)</em>; <em><a href="http://www.wormbase.org/db/get?name=WBGene00002335;class=Gene">let-60</a>(gf)</em> background. For each allele, two independent strains were created and the transcriptome was measured using microarrays (<a id="xref-ref-34-11" class="xref-bibr" href="#ref-34">Schmid <em>et al.</em> 2015</a>). From these measurements, the allelic effects of <em>amx</em>-2 on gene expression were determined and compared with the <em>trans</em>-eQTL effects measured in the miRIL population. These were found to correlate significantly (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.1120/-/DC1/FigureS4.pdf">Figure S4</a>; R<sup>2</sup> ∼0.34; <em>P</em> &lt; 2 × 10<sup>−8</sup>). Specifically, 61 of the 77 genes with an eQTL colocalizing with <em><a href="http://www.wormbase.org/db/get?name=WBGene00000138;class=Gene">amx-2</a></em> showed the same allelic effect in the transgenic strains carrying the two <em><a href="http://www.wormbase.org/db/get?name=WBGene00000138;class=Gene">amx-2</a></em> variants. Furthermore, analysis of the effect directions showed that the <em>trans</em>-band on chromosome I (<a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>) consists of two separate <em>trans</em>-bands (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.1120/-/DC1/FigureS4.pdf">Figure S4</a>); the first colocalizing with <em><a href="http://www.wormbase.org/db/get?name=WBGene00000138;class=Gene">amx-2</a></em> and the second lying more distal on chromosome I.</p><p id="p-43">To further investigate the mechanism by which <em><a href="http://www.wormbase.org/db/get?name=WBGene00000138;class=Gene">amx-2</a></em> modifies <em><a href="http://www.wormbase.org/db/get?name=WBGene00002335;class=Gene">let-60</a></em> Ras/MAPK signaling, we used the well-established <em>C. elegans</em> connectivity networks WormNet (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Cho <em>et al.</em> 2014</a>) and GeneMANIA (<a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Montojo <em>et al.</em> 2010</a>; <a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Shannon <em>et al.</em> 2003</a>). WormNet showed that the genes mapping to the <em><a href="http://www.wormbase.org/db/get?name=WBGene00000138;class=Gene">amx-2</a></em> locus were more connected than expected by chance (<em>P</em> &lt; 3 × 10<sup>−13</sup>). Visualizing the connectivity of these genes together with <em><a href="http://www.wormbase.org/db/get?name=WBGene00000138;class=Gene">amx-2</a></em> and <em><a href="http://www.wormbase.org/db/get?name=WBGene00002335;class=Gene">let-60</a></em> using GeneMANIA identified one major gene cluster (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). As expected, genes affected by the <em>trans</em>-band were found to be highly connected and at the core of the cluster, indicating a shared biological function. In contrast, 25% of the genes with a <em>cis</em>-eQTL at the <em>trans</em>-band location were not connected, and those that were connected had only one to three connections. Interestingly, <em><a href="http://www.wormbase.org/db/get?name=WBGene00002335;class=Gene">let-60</a></em> was at the core of the cluster of genes with a <em>trans</em>-eQTL, whereas <em><a href="http://www.wormbase.org/db/get?name=WBGene00000138;class=Gene">amx-2</a></em> was at the periphery. We found six genes, <em><a href="http://www.wormbase.org/db/get?name=WBGene00003025;class=Gene">lin-40</a></em> (also called <em><a href="http://www.wormbase.org/db/get?name=WBGene00003025;class=Gene">egr-1</a></em>), <em><a href="http://www.wormbase.org/db/get?name=WBGene00001194;class=Gene">egl-27</a></em>, <em><a href="http://www.wormbase.org/db/get?name=WBGene00007028;class=Gene">trr-1</a></em>, <em><a href="http://www.wormbase.org/db/get?name=WBGene00007042;class=Gene">pbrm-1</a></em>, <em><a href="http://www.wormbase.org/db/get?name=WBGene00000448;class=Gene">ceh-26</a></em> (also called <em><a href="http://www.wormbase.org/db/get?name=WBGene00000448;class=Gene">pros-1</a></em>), and <em><a href="http://www.wormbase.org/db/get?name=WBGene00001259;class=Gene">emb-5</a></em>, that were previously found to have a genetic interactions with <em><a href="http://www.wormbase.org/db/get?name=WBGene00002335;class=Gene">let-60</a></em> (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Lee <em>et al.</em> 2010</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Lehner <em>et al.</em> 2006</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Byrne <em>et al.</em> 2007</a>). These could be the genes through which <em><a href="http://www.wormbase.org/db/get?name=WBGene00000138;class=Gene">amx-2</a></em> affects RAS/MAPK signaling.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/9/3185/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Interaction network of genes colocalizing with amx-2. Genes with an eQTL colocalizing with amx-2 were selected together with amx-2 (yellow) and let-60 (pink). Interactions were obtained from GeneMANIA (Montojo et al. 2010; Shannon et al. 2003). Node color indicates cis in black and trans in green. Node shape indicates eQTL effect, N2 &gt; CB4856 as a &#x25CB; and CB4856 &gt; N2 as a &#x25A1;. Node border color gradient indicates expression in amx-2 transgenic lines (blue N2-amx-2-allele &gt; CB4856-amx-2 allele, red CB4856-amx-2 allele &gt; N2 amx-2 allele). Node size indicates number of edges connected. Edges in blue show coexpression links and edges in pink show genetic interactions as found by Lee et al. (2010), Lehner et al. (2006), and Byrne et al. (2007). Genes connected by genetic interactions have boldface gene names (as well as amx-2)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-469627366" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Interaction network of genes colocalizing with &lt;em&gt;amx-2&lt;/em&gt;. Genes with an eQTL colocalizing with &lt;em&gt;amx-2&lt;/em&gt; were selected together with &lt;em&gt;amx-2&lt;/em&gt; (yellow) and &lt;em&gt;let-60&lt;/em&gt; (pink). Interactions were obtained from GeneMANIA (Montojo &lt;em&gt;et al.&lt;/em&gt; 2010; Shannon &lt;em&gt;et al.&lt;/em&gt; 2003). Node color indicates &lt;em&gt;cis&lt;/em&gt; in black and &lt;em&gt;trans&lt;/em&gt; in green. Node shape indicates eQTL effect, N2 &gt; CB4856 as a &#x25CB; and CB4856 &gt; N2 as a &#x25A1;. Node border color gradient indicates expression in &lt;em&gt;amx-2&lt;/em&gt; transgenic lines (blue N2-&lt;em&gt;amx-2&lt;/em&gt;-allele &gt; CB4856-&lt;em&gt;amx-2&lt;/em&gt; allele, red CB4856-&lt;em&gt;amx-2&lt;/em&gt; allele &gt; N2 &lt;em&gt;amx-2&lt;/em&gt; allele). Node size indicates number of edges connected. Edges in blue show coexpression links and edges in pink show genetic interactions as found by Lee &lt;em&gt;et al.&lt;/em&gt; (2010), Lehner &lt;em&gt;et al.&lt;/em&gt; (2006), and Byrne &lt;em&gt;et al.&lt;/em&gt; (2007). Genes connected by genetic interactions have boldface gene names (as well as &lt;em&gt;amx-2&lt;/em&gt;).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/9/3185/F4.medium.gif" width="440" height="330"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/7/9/3185/F4.medium.gif" width="440" height="330"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/9/3185/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/9/3185/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/486389" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-44" class="first-child">Interaction network of genes colocalizing with <em>amx-2</em>. Genes with an eQTL colocalizing with <em>amx-2</em> were selected together with <em>amx-2</em> (yellow) and <em>let-60</em> (pink). Interactions were obtained from GeneMANIA (<a id="xref-ref-27-3" class="xref-bibr" href="#ref-27">Montojo <em>et al.</em> 2010</a>; <a id="xref-ref-36-3" class="xref-bibr" href="#ref-36">Shannon <em>et al.</em> 2003</a>). Node color indicates <em>cis</em> in black and <em>trans</em> in green. Node shape indicates eQTL effect, N2 &gt; CB4856 as a ○ and CB4856 &gt; N2 as a □. Node border color gradient indicates expression in <em>amx-2</em> transgenic lines (blue N2-<em>amx-2</em>-allele &gt; CB4856-<em>amx-2</em> allele, red CB4856-<em>amx-2</em> allele &gt; N2 <em>amx-2</em> allele). Node size indicates number of edges connected. Edges in blue show coexpression links and edges in pink show genetic interactions as found by <a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Lee <em>et al.</em> (2010)</a>, <a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Lehner <em>et al.</em> (2006)</a>, and <a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Byrne <em>et al.</em> (2007)</a>. Genes connected by genetic interactions have boldface gene names (as well as <em>amx-2</em>).</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-19"><h2 class="">Discussion</h2><p id="p-45">Ras/MAPK signaling is strongly affected by gain-of-function mutations in <em><a href="http://www.wormbase.org/db/get?name=WBGene00002335;class=Gene">let-60</a></em>, as shown by the strong effect on vulva development in <em>C. elegans</em> (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Beitel <em>et al.</em> 1990</a>). The allele <em><a href="http://www.wormbase.org/db/get?name=WBGene00002335;class=Gene">let-60</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00089919;class=Variation">n1046</a>)</em> hyper-activates Ras/MAPK signaling, leading to a multivulva phenotype and the differentiation of more than three vulval precursor cells. In the miRIL population, which carries a <em><a href="http://www.wormbase.org/db/get?name=WBGene00002335;class=Gene">let-60</a>(<a href="http://www.wormbase.org/db/get?name=WBVar00089919;class=Variation">n1046</a>)</em> mutation in a segregating <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a>/<a href="http://www.wormbase.org/db/get?name=CB4856;class=Strain">CB4856</a> background, a wide range of VI was found. While in the full <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a> background, VI of the <em><a href="http://www.wormbase.org/db/get?name=WBGene00002335;class=Gene">let-60</a>(gf)</em> mutation was 3.7 induced vulval cells on average, the VI in the miRIL population varied from 3.0 to 5.7 induced vulval cells, illustrating the strong modulatory effect of the genetic background on the mutant phenotype (<a id="xref-ref-34-12" class="xref-bibr" href="#ref-34">Schmid <em>et al.</em> 2015</a>). By measuring gene expression across the different miRILs, we gained insight into the genetic background effects on the transcriptional architecture of <em><a href="http://www.wormbase.org/db/get?name=WBGene00002335;class=Gene">let-60</a>(gf)</em>-sensitized miRILs.</p><div id="sec-20" class="subsection"><h3>The let-60(gf) mutation affects gene expression in trans</h3><p id="p-46">To our knowledge, this is the first study where a mutation, introgressed into a panel of RILs, is used to investigate the genetic architecture of transcript variation by eQTL analysis. Unfortunately, the direct effect of the <em><a href="http://www.wormbase.org/db/get?name=WBGene00002335;class=Gene">let-60</a>(gf)</em> allele on the genetic background is impossible to determine in this population, since there is no population with the same genetic background lacking the <em><a href="http://www.wormbase.org/db/get?name=WBGene00002335;class=Gene">let-60</a>(gf)</em> mutation. Therefore, we used previously published eQTL studies in <em>C. elegans</em> as an estimation for the miRIL-specific eQTL. We found that introgression of <em><a href="http://www.wormbase.org/db/get?name=WBGene00002335;class=Gene">let-60</a>(gf)</em> leads to an overrepresentation of specific <em>trans</em>-eQTL, while the <em>cis</em>-eQTL showed a high overlap with eQTL observed in the absence of <em><a href="http://www.wormbase.org/db/get?name=WBGene00002335;class=Gene">let-60</a>(gf)</em> allele. Although our study is based on a relatively small number of strains, the verification of identified eQTL against other eQTL studies in <em>C. elegans</em> shows that at least 73% of the identified <em>cis</em>-eQTL were previously discovered. This supports our power analysis, showing that our study has enough statistical power to detect eQTL with larger effects.</p><p id="p-47">Given that <em>cis</em>-eQTL are most likely genes with polymorphisms in or near the gene itself, it is expected that most have been previously discovered. In support of this, most of the <em>cis</em>-eQTL genes show higher expression in <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a>, making it likely that a relatively large proportion of these <em>cis</em>-eQTL indeed stem from polymorphisms causing mis-hybridization on the microarrays instead of true expression differences (<a id="xref-ref-33-11" class="xref-bibr" href="#ref-33">Rockman <em>et al.</em> 2010</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Alberts <em>et al.</em> 2007</a>). Therefore, most of the <em>cis</em>-eQTL will not represent gene expression changes that can be linked to the <em><a href="http://www.wormbase.org/db/get?name=WBGene00002335;class=Gene">let-60</a>(gf)</em> mutation in the miRIL population. That is also exactly what we found when constructing the regulatory network by connecting <em>cis</em>- and <em>trans</em>-eQTL to <em><a href="http://www.wormbase.org/db/get?name=WBGene00002335;class=Gene">let-60</a></em>; <em>cis</em>-eQTL are only loosely, or not at all, connected to the network.</p><p id="p-48">Compared to <em>cis</em>-eQTL, the <em>trans</em>-eQTL identified in the miRIL population were hardly found in previous eQTL studies in <em>C. elegans</em> (only 24% were previously detected). However, it should be noted that <em>trans</em>-eQTL are themselves are also environment and age dependent (<a id="xref-ref-48-7" class="xref-bibr" href="#ref-48">Vinuela <em>et al.</em> 2010</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Francesconi and Lehner 2014</a>; <a id="xref-ref-24-6" class="xref-bibr" href="#ref-24">Li <em>et al.</em> 2006</a>; <a id="xref-ref-38-7" class="xref-bibr" href="#ref-38">Snoek <em>et al.</em> 2017</a>), therefore it is possible that not all the <em>trans</em>-eQTL are indeed <em><a href="http://www.wormbase.org/db/get?name=WBGene00002335;class=Gene">let-60</a></em> induced. For example, they might be age related (<a id="xref-ref-48-8" class="xref-bibr" href="#ref-48">Vinuela <em>et al.</em> 2010</a>). Another aspect that influences detection of <em>trans</em>-eQTL is the power of the study, as <em>trans</em>-eQTL explain less of the variation than <em>cis</em>-eQTL. However, by being stringent in the overlap (requiring only one match over nine different environments), it is likely that most of these eQTL would be detected. This makes it likely that many of the identified <em>trans</em>-eQTL are specifically linked to an effect of the <em><a href="http://www.wormbase.org/db/get?name=WBGene00002335;class=Gene">let-60</a>(gf)</em> mutation in the genetic background. This extends the idea that environmental perturbations can reveal additional genetic variation (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Li <em>et al.</em> 2008</a>) by perturbing the genetic environment (<a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Kammenga <em>et al.</em> 2008</a>; <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Duveau and Felix 2012</a>; <a id="xref-ref-34-13" class="xref-bibr" href="#ref-34">Schmid <em>et al.</em> 2015</a>). The advantage of mutational perturbation and perturbation via induced responses in eQTL studies (for example, see <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Snoek <em>et al.</em> 2012</a> and <a id="xref-ref-38-8" class="xref-bibr" href="#ref-38">Snoek <em>et al.</em> 2017</a>) is that it contextualizes the transcriptional response. Yet, as the phenotypic influence of <em><a href="http://www.wormbase.org/db/get?name=WBGene00002335;class=Gene">let-60</a>(gf)</em> in the miRIL population was investigated before, we know it results in a phenotype. In the study presented here, the context is the <em><a href="http://www.wormbase.org/db/get?name=WBGene00002335;class=Gene">let-60</a>(gf)</em> mutation, which results in a phenotype where VI is affected. It is therefore interesting that the <em>trans</em>-bands colocalize with QTL mapped for VI in the same miRIL population: <em>trans</em>-band I: 12–15 Mb overlaps with QTL1b, <em>trans</em>-band II: 7–8 Mb with QTL2, and <em>trans</em>-band V: 13–14 Mb with QTL3 (<a id="xref-ref-34-14" class="xref-bibr" href="#ref-34">Schmid <em>et al.</em> 2015</a>). This colocalization adds to the plausibility that the novel <em>trans</em>-eQTL detected in the miRIL population are indeed due to the <em><a href="http://www.wormbase.org/db/get?name=WBGene00002335;class=Gene">let-60</a>(gf)</em> mutation and its interaction with the genetic background.</p></div><div id="sec-21" class="subsection"><h3>A trans-band on chromosome I links to variation in the amx-2 gene</h3><p id="p-49">Using the same <em><a href="http://www.wormbase.org/db/get?name=WBGene00002335;class=Gene">let-60</a>(gf)</em> miRIL population, we have previously identified the polymorphic MAOA <em><a href="http://www.wormbase.org/db/get?name=WBGene00000138;class=Gene">amx-2</a></em> as a background modifier that negatively regulates the Ras/MAPK pathway and the VI phenotype (<a id="xref-ref-34-15" class="xref-bibr" href="#ref-34">Schmid <em>et al.</em> 2015</a>). The <em>trans</em>-band on chromosome I colocalizes with the <em><a href="http://www.wormbase.org/db/get?name=WBGene00000138;class=Gene">amx-2</a></em> QTL for VI [QTL1b (<a id="xref-ref-34-16" class="xref-bibr" href="#ref-34">Schmid <em>et al.</em> 2015</a>)], it is therefore possible that the modifier <em><a href="http://www.wormbase.org/db/get?name=WBGene00000138;class=Gene">amx-2</a></em> also affects gene expression. More specifically, the <em>trans</em>-band on chromosome I originally identified in the miRILs consists of two separate regulatory loci. The <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a> allelic effect at the <em><a href="http://www.wormbase.org/db/get?name=WBGene00000138;class=Gene">amx-2</a></em> locus is mostly negative; whereas for the more distal part of the original <em>trans</em>-band, the <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a> allelic effect is mostly positive. Yet, it seems unlikely that <em><a href="http://www.wormbase.org/db/get?name=WBGene00000138;class=Gene">amx-2</a></em> is a direct gene expression effector. As <em><a href="http://www.wormbase.org/db/get?name=WBGene00000138;class=Gene">amx-2</a></em> is a mitochondrial MAOA, a catalyzer of neuropeptide oxidative deamination, it will probably not influence gene expression directly (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Tipton <em>et al.</em> 2004</a>). Placing the <em>trans</em>-eQTL of the <em><a href="http://www.wormbase.org/db/get?name=WBGene00000138;class=Gene">amx-2</a> trans</em>-band in a gene interaction network supports this line of reasoning: although <em><a href="http://www.wormbase.org/db/get?name=WBGene00002335;class=Gene">let-60</a></em> is in the center of the <em>trans</em>-eQTL, <em><a href="http://www.wormbase.org/db/get?name=WBGene00000138;class=Gene">amx-2</a></em> is only peripheral.</p><p id="p-50">By measuring gene expression in transgenic lines expressing the <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a> or <a href="http://www.wormbase.org/db/get?name=CB4856;class=Strain">CB4856</a> allelic variants of <em><a href="http://www.wormbase.org/db/get?name=WBGene00000138;class=Gene">amx-2</a></em> in an <em><a href="http://www.wormbase.org/db/get?name=WBGene00000138;class=Gene">amx-2</a>(lf);<a href="http://www.wormbase.org/db/get?name=WBGene00002335;class=Gene">let-60</a>(gf)</em> genetic background, we attempted to link the allelic effect of <em><a href="http://www.wormbase.org/db/get?name=WBGene00000138;class=Gene">amx-2</a></em> to the <em>trans</em>-eQTL mapping to the <em><a href="http://www.wormbase.org/db/get?name=WBGene00000138;class=Gene">amx-2</a></em> locus. As a significant correlation between the expression differences in the transgenic lines and the <em>trans</em>-eQTL effects mapping to the <em><a href="http://www.wormbase.org/db/get?name=WBGene00000138;class=Gene">amx-2</a></em> locus is found, these <em>trans</em>-eQTL can be confidently linked to allelic variation in <em><a href="http://www.wormbase.org/db/get?name=WBGene00000138;class=Gene">amx-2</a></em>. As discussed before, it is unlikely that <em><a href="http://www.wormbase.org/db/get?name=WBGene00000138;class=Gene">amx-2</a></em> is the direct cause of the transcriptional variation, but rather it acts through an indirect mechanism. One route of effect might be through <em><a href="http://www.wormbase.org/db/get?name=WBGene00000138;class=Gene">amx-2</a></em>-mediated degradation of serotonin (5-HT) to 5-HIAA, which both affect VI (as discussed in <a id="xref-ref-34-17" class="xref-bibr" href="#ref-34">Schmid <em>et al.</em> 2015</a>). Subsequently, the affected VI might result in different gene expression levels. It is interesting to remark that the <em><a href="http://www.wormbase.org/db/get?name=WBGene00000138;class=Gene">amx-2</a> trans</em>-band is characterized by downregulation of expression related to the <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a> genotype as well as a decreased VI for that genotype.</p><p id="p-51">As this places <em><a href="http://www.wormbase.org/db/get?name=WBGene00000138;class=Gene">amx-2</a></em> in the causal chain of events, we think it is most likely that <em><a href="http://www.wormbase.org/db/get?name=WBGene00000138;class=Gene">amx-2</a></em> does not affect gene expression directly but via another gene involved in Ras/MAPK signaling, possibly directly linked to <em><a href="http://www.wormbase.org/db/get?name=WBGene00002335;class=Gene">let-60</a></em>. Therefore, we hypothesize that <em><a href="http://www.wormbase.org/db/get?name=WBGene00000138;class=Gene">amx-2</a></em> affects Ras/MAPK signaling via one or more of the six previously found genes with a genetic interaction with <em><a href="http://www.wormbase.org/db/get?name=WBGene00002335;class=Gene">let-60</a></em>: <em><a href="http://www.wormbase.org/db/get?name=WBGene00003025;class=Gene">lin-40</a></em> (also called <em><a href="http://www.wormbase.org/db/get?name=WBGene00003025;class=Gene">egr-1</a></em>), <em><a href="http://www.wormbase.org/db/get?name=WBGene00001194;class=Gene">egl-27</a></em>, <em><a href="http://www.wormbase.org/db/get?name=WBGene00007028;class=Gene">trr-1</a></em>, <em><a href="http://www.wormbase.org/db/get?name=WBGene00007042;class=Gene">pbrm-1</a></em>, <em><a href="http://www.wormbase.org/db/get?name=WBGene00000448;class=Gene">ceh-26</a></em> (also called <em><a href="http://www.wormbase.org/db/get?name=WBGene00000448;class=Gene">pros-1</a></em>), and <em><a href="http://www.wormbase.org/db/get?name=WBGene00001259;class=Gene">emb-5</a></em>. For example, <em><a href="http://www.wormbase.org/db/get?name=WBGene00001194;class=Gene">egl-27</a></em> and <em><a href="http://www.wormbase.org/db/get?name=WBGene00003025;class=Gene">lin-40</a></em> both have a <em>trans</em>-eQTL mapping to the <em><a href="http://www.wormbase.org/db/get?name=WBGene00000138;class=Gene">amx-2</a></em> locus, and are the two MTA (metastasis-associated protein) homologs found in <em>C. elegans</em> (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Ch’ng and Kenyon 1999</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Herman <em>et al.</em> 1999</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Solari <em>et al.</em> 1999</a>). The proteins act in the NURD chromatin remodeling complex, which has previously been shown to antagonize Ras-induced vulval development (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Solari and Ahringer 2000</a>). Chromatin remodeling provides a more likely mechanism of action than the molecular role of <em><a href="http://www.wormbase.org/db/get?name=WBGene00000138;class=Gene">amx-2</a></em> itself.</p></div><div id="sec-22" class="subsection"><h3>Implications for understanding the Ras/MAPK pathway</h3><p id="p-52">How can our results help form a better understanding of the Ras/MAPK pathway? Our previous investigation on VI in the miRIL population resulted in the identification of three QTL harboring polymorphic Ras-signaling modifiers. Expanding our research to genetic variation affecting gene expression in a <em><a href="http://www.wormbase.org/db/get?name=WBGene00002335;class=Gene">let-60</a>(gf)</em>-sensitized RIL population uncovered six <em>trans</em>-bands (or “eQTL hotspots”). As mentioned before, these <em>trans</em>-bands overlap with the QTL mapped for VI in <a id="xref-ref-34-18" class="xref-bibr" href="#ref-34">Schmid <em>et al.</em> (2015)</a>. There are three more <em>trans</em>-bands that do not overlap with QTL for VI, but are also likely to represent modifier loci of the Ras/MAPK pathway, possibly underlying other Ras/MARK-associated phenotypic differences.</p><p id="p-53">The main advantage of studying the genetic architecture of gene expression in the miRIL population is that it creates more insight in the genes and pathways affected by allelic variation acting on the Ras/MAPK pathway. It identifies hidden genetic variation; genetic variants that are unlocked under altered environmental conditions or when the genetic background is modified (for a review, see <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Paaby and Rockman 2014</a> and <a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Kammenga 2017</a>). Identification of background modifiers of disease pathways is important for gaining insight into individual-based differences of disease contraction. Mutant phenotypes can be strongly affected by the genetic background. For example, large variation in traits between different backgrounds in many different mutated genes has been observed in <em>C. elegans</em> (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Paaby <em>et al.</em> 2015</a>; <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Vu <em>et al.</em> 2015</a>; <a id="xref-ref-9-3" class="xref-bibr" href="#ref-9">Duveau and Felix 2012</a>). A major discovery was that this variation in mutant phenotypes could be predicted from gene expression variation (<a id="xref-ref-50-2" class="xref-bibr" href="#ref-50">Vu <em>et al.</em> 2015</a>). These results are in line with the discovery of <em>trans</em>-bands at the location of each VI QTL. Furthermore, results presented here (and previously in <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Elvin <em>et al.</em> 2011</a>) link variations in individual gene expression levels to enhancing or diminishing the severity of a Mendelian disorder caused by a so-called “major gene.” More specifically, these differences seem to stem from a couple of modifier loci harboring polymorphic regulators.</p><p id="p-54">So far, we only explored the <em><a href="http://www.wormbase.org/db/get?name=WBGene00000138;class=Gene">amx-2</a> trans</em>-band, and it is likely that the other loci also contain polymorphic modifiers of the Ras/MAPK pathway. Genetic perturbation by <em><a href="http://www.wormbase.org/db/get?name=WBGene00002335;class=Gene">let-60</a>(gf)</em> leads to a strong increase in specific <em>trans</em>-acting eQTL organized over six <em>trans</em>-bands, thus supporting the involvement of multiple genes as modifiers. Given the fact that we detected <em><a href="http://www.wormbase.org/db/get?name=WBGene00000138;class=Gene">amx-2</a></em> both via an indirect transcriptional response, but also mechanistically, we feel confident that the detected loci indeed harbor other modifiers. A single gene mutation apparently has the capacity to unlock a huge number of novel interactions controlled by many genes across different genetic backgrounds. Further studies should be conducted to identify and characterize these underlying polymorphic regulators.</p></div><div id="sec-23" class="subsection"><h3>Conclusion</h3><p id="p-55">Introgression of a mutation in a segregating genetic background allows for identification of polymorphic modifier loci. By introducing a <em><a href="http://www.wormbase.org/db/get?name=WBGene00002335;class=Gene">let-60</a>(gf)</em> mutation in a <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a>/<a href="http://www.wormbase.org/db/get?name=CB4856;class=Strain">CB4856</a> genetic background, in the form of the miRIL population, and measuring the transcriptome, we identified six <em>trans</em>-bands specific for the <em><a href="http://www.wormbase.org/db/get?name=WBGene00002335;class=Gene">let-60</a>(gf)</em> miRIL population. The majority of <em>trans</em>-eQTL are specific for the miRIL population, showing that genetic variation in gene expression can be specifically modified by a background mutation. Therefore, genetic perturbation can be viewed as analogous to environmental perturbation, which also results in specific <em>trans</em>-eQTL. We demonstrated the involvement of <em><a href="http://www.wormbase.org/db/get?name=WBGene00000138;class=Gene">amx-2</a></em> and the allelic variation between <a href="http://www.wormbase.org/db/get?name=N2;class=Strain">N2</a> and <a href="http://www.wormbase.org/db/get?name=CB4856;class=Strain">CB4856</a> in gene expression variation originating from chromosome I. Yet, we think it is unlikely that <em><a href="http://www.wormbase.org/db/get?name=WBGene00000138;class=Gene">amx-2</a></em> directly affects gene expression variation. Instead, we prefer the hypothesis that allelic variation in <em><a href="http://www.wormbase.org/db/get?name=WBGene00000138;class=Gene">amx-2</a></em> indirectly affects gene expression, possibly through the NURD complex.</p></div></div><div class="section" id="sec-24"></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-56">The authors thank Harm Nijveen for making our data available in EleQTL. M.R., T.S., A.H., and J.E.K. were funded by the European Community’s Health Seventh Framework Program (FP7/2007-2013) under grant 222936. B.L.S. was funded by ERASysbio-plus ZonMW project GRAPPLE: Iterative modeling of gene regulatory interactions underlying stress, disease and ageing in <em>C. elegans</em> (project 90201066); and The Netherlands Organization for Scientific Research (project no. 823.01.001).</p><p id="p-57">Author contributions: A.H., J.E.K., and B.L.S. conceived and designed the experiments. J.A.G.R., M.R., and T.S. conducted the experiments. M.G.S., L.v.B.v.d.P., and B.L.S. conducted transcriptome and main analyses. M.G.S., A.H., J.E.K., and B.L.S. wrote the manuscript.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-5"><p id="p-58">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.1120/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.1120/-/DC1</a>.</p></li><li class="fn" id="fn-6"><p id="p-59"><em>Communicating editor: D-J. de Koning</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2017-04-18"><span class="received-label">Received </span>April 18, 2017.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2017-07-22"><span class="accepted-label">Accepted </span>July 22, 2017.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2017 Sterken <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3185.1" data-doi="10.1371/journal.pone.0000622"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Alberts</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Terpstra</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Breitling</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nap</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Sequence polymorphisms cause many false cis eQTLs.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">e622</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DAlberts%26rft.auinit1%253DR.%26rft.volume%253D2%26rft.issue%253D7%26rft.spage%253De622%26rft.epage%253De622%26rft.atitle%253DSequence%2Bpolymorphisms%2Bcause%2Bmany%2Bfalse%2Bcis%2BeQTLs.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000622%26rft_id%253Dinfo%253Apmid%252F17637838%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0000622&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17637838&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3185.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3185.2" data-doi="10.1038/348503a0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Beitel</span>  <span class="cit-name-given-names">G. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clark</span>  <span class="cit-name-given-names">S. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Horvitz</span>  <span class="cit-name-given-names">H. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">Caenorhabditis elegans ras gene let-60 acts as a switch in the pathway of vulval induction.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">348</span>: <span class="cit-fpage">503</span>–<span class="cit-lpage">509</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DBeitel%26rft.auinit1%253DG.%2BJ.%26rft.volume%253D348%26rft.issue%253D6301%26rft.spage%253D503%26rft.epage%253D509%26rft.atitle%253DCaenorhabditis%2Belegans%2Bras%2Bgene%2Blet-60%2Bacts%2Bas%2Ba%2Bswitch%2Bin%2Bthe%2Bpathway%2Bof%2Bvulval%2Binduction.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F348503a0%26rft_id%253Dinfo%253Apmid%252F2123303%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/348503a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2123303&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3185.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3185.3" data-doi="10.1214/aos/1013699998"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Benjamini</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yekutieli</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">The control of the false discovery rate in multiple testing under dependency.</span> <abbr class="cit-jnl-abbrev">Ann. Stat.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">1165</span>–<span class="cit-lpage">1188</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BStat.%26rft.volume%253D29%26rft.spage%253D1165%26rft_id%253Dinfo%253Adoi%252F10.1214%252Faos%252F1013699998%26rft_id%253Dinfo%253Apmid%252F17909820%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1214/aos/1013699998&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17909820&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3185.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000172838100012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3185.4" data-doi="10.1186/jbiol58"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Byrne</span>  <span class="cit-name-given-names">A. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weirauch</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wong</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koeva</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dixon</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">A global analysis of genetic interactions in Caenorhabditis elegans.</span> <abbr class="cit-jnl-abbrev">J. Biol.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">8</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bbiology%2B%2528Online%2529%26rft.stitle%253DJ%2BBiol%26rft.aulast%253DByrne%26rft.auinit1%253DA.%2BB.%26rft.volume%253D6%26rft.issue%253D3%26rft.spage%253D8%26rft.epage%253D8%26rft.atitle%253DA%2Bglobal%2Banalysis%2Bof%2Bgenetic%2Binteractions%2Bin%2BCaenorhabditis%2Belegans.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fjbiol58%26rft_id%253Dinfo%253Apmid%252F17897480%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/jbiol58&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17897480&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3185.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3185.5" data-doi="10.1534/genetics.113.159426"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chandler</span>  <span class="cit-name-given-names">C. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chari</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tack</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dworkin</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Causes and consequences of genetic background effects illuminated by integrative genomic analysis.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">196</span>: <span class="cit-fpage">1321</span>–<span class="cit-lpage">1336</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.159426%26rft_id%253Dinfo%253Apmid%252F24504186%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5Ni80LzEzMjEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzkvMzE4NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3185.6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ch’ng</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kenyon</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">egl-27 generates anteroposterior patterns of cell fusion in C-elegans by regulating Hox gene expression and Hox protein function.</span> <abbr class="cit-jnl-abbrev">Development</abbr> <span class="cit-vol">126</span>: <span class="cit-fpage">3303</span>–<span class="cit-lpage">3312</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopment%26rft.stitle%253DDevelopment%26rft.aulast%253DCh%2527ng%26rft.auinit1%253DQ%26rft.volume%253D126%26rft.issue%253D15%26rft.spage%253D3303%26rft.epage%253D3312%26rft.atitle%253Degl-27%2Bgenerates%2Banteroposterior%2Bpatterns%2Bof%2Bcell%2Bfusion%2Bin%2BC.%2Belegans%2Bby%2Bregulating%2BHox%2Bgene%2Bexpression%2Band%2BHox%2Bprotein%2Bfunction%26rft_id%253Dinfo%253Apmid%252F10393110%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZGV2ZWxvcCI7czo1OiJyZXNpZCI7czoxMToiMTI2LzE1LzMzMDMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzkvMzE4NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3185.7" data-doi="10.1093/nar/gku367"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cho</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shin</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hwang</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shim</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">WormNet v3: a network-assisted hypothesis-generating server for Caenorhabditis elegans.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">W76</span>–<span class="cit-lpage">W82</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgku367%26rft_id%253Dinfo%253Apmid%252F24813450%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gku367&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24813450&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3185.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000339715000014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3185.8" data-doi="10.1016/0092-8674(94)90133-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cowley</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paterson</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kemp</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marshall</span>  <span class="cit-name-given-names">C. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Activation of map kinase kinase is necessary and sufficient for Pc12 differentiation and for transformation of Nih 3t3 cells.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">77</span>: <span class="cit-fpage">841</span>–<span class="cit-lpage">852</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DCowley%26rft.auinit1%253DS.%26rft.volume%253D77%26rft.issue%253D6%26rft.spage%253D841%26rft.epage%253D852%26rft.atitle%253DActivation%2Bof%2BMAP%2Bkinase%2Bkinase%2Bis%2Bnecessary%2Band%2Bsufficient%2Bfor%2BPC12%2Bdifferentiation%2Band%2Bfor%2Btransformation%2Bof%2BNIH%2B3T3%2Bcells.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0092-8674%252894%252990133-3%26rft_id%253Dinfo%253Apmid%252F7911739%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0092-8674(94)90133-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7911739&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3185.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1994NT33100009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3185.9" data-doi="10.1371/journal.pbio.1001230"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Duveau</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Felix</span>  <span class="cit-name-given-names">M. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Role of pleiotropy in the evolution of a cryptic developmental variation in Caenorhabditis elegans.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">e1001230</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DDuveau%26rft.auinit1%253DF.%26rft.volume%253D10%26rft.issue%253D1%26rft.spage%253De1001230%26rft.epage%253De1001230%26rft.atitle%253DRole%2Bof%2Bpleiotropy%2Bin%2Bthe%2Bevolution%2Bof%2Ba%2Bcryptic%2Bdevelopmental%2Bvariation%2Bin%2BCaenorhabditis%2Belegans.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.1001230%26rft_id%253Dinfo%253Apmid%252F22235190%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.1001230&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22235190&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3185.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3185.10" data-doi="10.1186/1471-2164-12-510"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Elvin</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Snoek</span>  <span class="cit-name-given-names">L. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Frejno</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klemstein</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kammenga</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A fitness assay for comparing RNAi effects across multiple C. elegans genotypes.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">510</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DElvin%26rft.auinit1%253DM.%26rft.volume%253D12%26rft.spage%253D510%26rft.epage%253D510%26rft.atitle%253DA%2Bfitness%2Bassay%2Bfor%2Bcomparing%2BRNAi%2Beffects%2Bacross%2Bmultiple%2BC.%2Belegans%2Bgenotypes.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-12-510%26rft_id%253Dinfo%253Apmid%252F22004469%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-12-510&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22004469&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3185.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3185.11" data-doi="10.1038/nature12772"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Francesconi</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lehner</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The effects of genetic variation on gene expression dynamics during development.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">505</span>: <span class="cit-fpage">208</span>–<span class="cit-lpage">211</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D505%26rft.spage%253D208%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature12772%26rft_id%253Dinfo%253Apmid%252F24270809%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature12772&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24270809&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3185.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3185.12" data-doi="10.1126/science.1196914"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gerstein</span>  <span class="cit-name-given-names">M. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">Z. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van Nostrand</span>  <span class="cit-name-given-names">E. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheng</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arshinoff</span>  <span class="cit-name-given-names">B. I.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Integrative analysis of the Caenorhabditis elegans genome by the modENCODE project.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">330</span>: <span class="cit-fpage">1775</span>–<span class="cit-lpage">1787</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1196914%26rft_id%253Dinfo%253Apmid%252F21177976%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMzAvNjAxMi8xNzc1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy85LzMxODUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3185.13" data-doi="10.1002/wdev.181"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gokhale</span>  <span class="cit-name-given-names">R. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shingleton</span>  <span class="cit-name-given-names">A. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Size control: the developmental physiology of body and organ size regulation.</span> <abbr class="cit-jnl-abbrev">Wiley Interdiscip. Rev. Dev. Biol.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">335</span>–<span class="cit-lpage">356</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DWiley%2BInterdiscip.%2BRev.%2BDev.%2BBiol.%26rft.volume%253D4%26rft.spage%253D335%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fwdev.181%26rft_id%253Dinfo%253Apmid%252F25808999%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/wdev.181&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25808999&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3185.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3185.14" data-doi="10.1016/0092-8674(90)90495-Z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Han</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sternberg</span>  <span class="cit-name-given-names">P. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">let-60, a gene that specifies cell fates during C. elegans vulval induction, encodes a ras protein.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">63</span>: <span class="cit-fpage">921</span>–<span class="cit-lpage">931</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DHan%26rft.auinit1%253DM.%26rft.volume%253D63%26rft.issue%253D5%26rft.spage%253D921%26rft.epage%253D931%26rft.atitle%253Dlet-60%252C%2Ba%2Bgene%2Bthat%2Bspecifies%2Bcell%2Bfates%2Bduring%2BC.%2Belegans%2Bvulval%2Binduction%252C%2Bencodes%2Ba%2Bras%2Bprotein.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0092-8674%252890%252990495-Z%26rft_id%253Dinfo%253Apmid%252F2257629%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0092-8674(90)90495-Z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2257629&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3185.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1990EL75900007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3185.15" data-doi="10.1093/nar/gkt1063"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Harris</span>  <span class="cit-name-given-names">T. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baran</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bieri</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cabunoc</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">WormBase 2014: new views of curated biology.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">D789</span>–<span class="cit-lpage">D793</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkt1063%26rft_id%253Dinfo%253Apmid%252F24194605%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkt1063&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24194605&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3185.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000331139800116&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3185.16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Herman</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ch’ng</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hettenbach</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ratliff</span>  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kenyon</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">EGL-27 is similar to a metastasis-associated factor and controls cell polarity and cell migration in C-elegans.</span> <abbr class="cit-jnl-abbrev">Development</abbr> <span class="cit-vol">126</span>: <span class="cit-fpage">1055</span>–<span class="cit-lpage">1064</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopment%26rft.stitle%253DDevelopment%26rft.aulast%253DHerman%26rft.auinit1%253DM.%26rft.volume%253D126%26rft.issue%253D5%26rft.spage%253D1055%26rft.epage%253D1064%26rft.atitle%253DEGL-27%2Bis%2Bsimilar%2Bto%2Ba%2Bmetastasis-associated%2Bfactor%2Band%2Bcontrols%2Bcell%2Bpolarity%2Band%2Bcell%2Bmigration%2Bin%2BC.%2Belegans%26rft_id%253Dinfo%253Apmid%252F9927605%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZGV2ZWxvcCI7czo1OiJyZXNpZCI7czoxMDoiMTI2LzUvMTA1NSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvOS8zMTg1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3185.17" data-doi="10.1016/S0168-9525(01)02310-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jansen</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nap</span>  <span class="cit-name-given-names">J. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Genetical genomics: the added value from segregation.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">388</span>–<span class="cit-lpage">391</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DJansen%26rft.auinit1%253DR.%2BC.%26rft.volume%253D17%26rft.issue%253D7%26rft.spage%253D388%26rft.epage%253D391%26rft.atitle%253DGenetical%2Bgenomics%253A%2Bthe%2Badded%2Bvalue%2Bfrom%2Bsegregation.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0168-9525%252801%252902310-1%26rft_id%253Dinfo%253Apmid%252F11418218%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0168-9525(01)02310-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11418218&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3185.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000169674900006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3185.18" data-doi="10.1371/journal.pgen.1004513"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">H. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">Y. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">LIN-3/EGF promotes the programmed cell death of specific cells in Caenorhabditis elegans by transcriptional activation of the pro-apoptotic gene egl-1.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">e1004513</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D10%26rft.spage%253De1004513%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1004513%26rft_id%253Dinfo%253Apmid%252F25144461%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1004513&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25144461&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3185.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3185.19" data-doi="10.1111/febs.14080."><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kammenga</span>  <span class="cit-name-given-names">J. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">The background puzzle: how identical mutations in the same gene lead to different disease symptoms.</span> <abbr class="cit-jnl-abbrev">FEBS J.</abbr><span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">DOI:</span><span class="cit-pub-id cit-pub-id-doi">10.1111/febs.14080.</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEBS%2BJ.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Ffebs.14080.%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/febs.14080.&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3185.20" data-doi="10.1016/j.tig.2008.01.001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kammenga</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Phillips</span>  <span class="cit-name-given-names">P. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">De Bono</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doroszuk</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Beyond induced mutants: using worms to study natural variation in genetic pathways.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">178</span>–<span class="cit-lpage">185</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DKammenga%26rft.auinit1%253DJ.%2BE.%26rft.volume%253D24%26rft.issue%253D4%26rft.spage%253D178%26rft.epage%253D185%26rft.atitle%253DBeyond%2Binduced%2Bmutants%253A%2Busing%2Bworms%2Bto%2Bstudy%2Bnatural%2Bvariation%2Bin%2Bgenetic%2Bpathways.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2008.01.001%26rft_id%253Dinfo%253Apmid%252F18325626%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tig.2008.01.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18325626&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3185.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000255346300005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3185.21"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kyriakakis</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Markaki</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tavernarakis</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Caenorhabditis elegans as a model for cancer research.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Oncol.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">e975027</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BCell.%2BOncol.%26rft.volume%253D2%26rft.spage%253D975027e%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3185.22" data-doi="10.1101/gr.102749.109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lehner</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vavouri</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shin</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fraser</span>  <span class="cit-name-given-names">A. G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Predicting genetic modifier loci using functional gene networks.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">1143</span>–<span class="cit-lpage">1153</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.102749.109%26rft_id%253Dinfo%253Apmid%252F20538624%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjIwLzgvMTE0MyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvOS8zMTg1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3185.23" data-doi="10.1038/ng1844"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lehner</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crombie</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tischler</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fortunato</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fraser</span>  <span class="cit-name-given-names">A. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Systematic mapping of genetic interactions in Caenorhabditis elegans identifies common modifiers of diverse signaling pathways.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">896</span>–<span class="cit-lpage">903</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DLehner%26rft.auinit1%253DB.%26rft.volume%253D38%26rft.issue%253D8%26rft.spage%253D896%26rft.epage%253D903%26rft.atitle%253DSystematic%2Bmapping%2Bof%2Bgenetic%2Binteractions%2Bin%2BCaenorhabditis%2Belegans%2Bidentifies%2Bcommon%2Bmodifiers%2Bof%2Bdiverse%2Bsignaling%2Bpathways.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1844%26rft_id%253Dinfo%253Apmid%252F16845399%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1844&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16845399&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3185.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000239325700018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3185.24" data-doi="10.1371/journal.pgen.0020222"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alvarez</span>  <span class="cit-name-given-names">O. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gutteling</span>  <span class="cit-name-given-names">E. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tijsterman</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Mapping determinants of gene expression plasticity by genetical genomics in C. elegans.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">e222</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DLi%26rft.auinit1%253DY.%26rft.volume%253D2%26rft.issue%253D12%26rft.spage%253De222%26rft.epage%253De222%26rft.atitle%253DMapping%2Bdeterminants%2Bof%2Bgene%2Bexpression%2Bplasticity%2Bby%2Bgenetical%2Bgenomics%2Bin%2BC.%2Belegans.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.0020222%26rft_id%253Dinfo%253Apmid%252F17196041%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.0020222&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17196041&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3185.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3185.25" data-doi="10.1016/j.tig.2008.08.001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Breitling</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jansen</span>  <span class="cit-name-given-names">R. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Generalizing genetical genomics: getting added value from environmental perturbation.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">518</span>–<span class="cit-lpage">524</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DLi%26rft.auinit1%253DY.%26rft.volume%253D24%26rft.issue%253D10%26rft.spage%253D518%26rft.epage%253D524%26rft.atitle%253DGeneralizing%2Bgenetical%2Bgenomics%253A%2Bgetting%2Badded%2Bvalue%2Bfrom%2Benvironmental%2Bperturbation.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2008.08.001%26rft_id%253Dinfo%253Apmid%252F18774198%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tig.2008.08.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18774198&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3185.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000260756300009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3185.26" data-doi="10.1126/science.8052857"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mansour</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Matten</span>  <span class="cit-name-given-names">W. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hermann</span>  <span class="cit-name-given-names">A. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Candia</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rong</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Transformation of mammalian cells by constitutively active MAP kinase kinase.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">265</span>: <span class="cit-fpage">966</span>–<span class="cit-lpage">970</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DMansour%26rft.auinit1%253DS.%26rft.volume%253D265%26rft.issue%253D5174%26rft.spage%253D966%26rft.epage%253D970%26rft.atitle%253DTransformation%2Bof%2Bmammalian%2Bcells%2Bby%2Bconstitutively%2Bactive%2BMAP%2Bkinase%2Bkinase%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.8052857%26rft_id%253Dinfo%253Apmid%252F8052857%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIyNjUvNTE3NC85NjYiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzkvMzE4NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3185.27" data-doi="10.1093/bioinformatics/btq562"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Montojo</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zuberi</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rodriguez</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kazi</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">GeneMANIA Cytoscape plugin: fast gene function predictions on the desktop.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">2927</span>–<span class="cit-lpage">2928</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq562%26rft_id%253Dinfo%253Apmid%252F20926419%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btq562&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20926419&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3185.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000283919800028&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3185.28" data-doi="10.1101/gr.114587.110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Niu</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">Z. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhong</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sarov</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Murray</span>  <span class="cit-name-given-names">J. I.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Diverse transcription factor binding features revealed by genome-wide ChIP-seq in C. elegans.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">245</span>–<span class="cit-lpage">254</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.114587.110%26rft_id%253Dinfo%253Apmid%252F21177963%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjIxLzIvMjQ1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy85LzMxODUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3185.29" data-doi="10.1093/nar/27.1.29"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ogata</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goto</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sato</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fujibuchi</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bono</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">KEGG: kyoto encyclopedia of genes and genomes.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">29</span>–<span class="cit-lpage">34</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F27.1.29%26rft_id%253Dinfo%253Apmid%252F9847135%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/27.1.29&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9847135&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3185.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000077983000006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3185.30" data-doi="10.1038/nrg3688"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Paaby</span>  <span class="cit-name-given-names">A. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rockman</span>  <span class="cit-name-given-names">M. V.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Cryptic genetic variation: evolution’s hidden substrate.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">247</span>–<span class="cit-lpage">258</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BGenet.%26rft.volume%253D15%26rft.spage%253D247%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3688%26rft_id%253Dinfo%253Apmid%252F24614309%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3688&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24614309&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3185.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3185.31" data-doi="10.7554/eLife.09178"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Paaby</span>  <span class="cit-name-given-names">A. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">White</span>  <span class="cit-name-given-names">A. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Riccardi</span>  <span class="cit-name-given-names">D. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gunsalus</span>  <span class="cit-name-given-names">K. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Piano</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Wild worm embryogenesis harbors ubiquitous polygenic modifier variation.</span> <abbr class="cit-jnl-abbrev">eLife</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e09178</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DeLife%26rft_id%253Dinfo%253Adoi%252F10.7554%252FeLife.09178%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NToiZWxpZmUiO3M6NToicmVzaWQiO3M6MTA6IjQvMC9lMDkxNzgiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzkvMzE4NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3185.32" data-doi="10.1016/j.semcdb.2011.09.002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Prior</span>  <span class="cit-name-given-names">I. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hancock</span>  <span class="cit-name-given-names">J. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Ras trafficking, localization and compartmentalized signalling.</span> <abbr class="cit-jnl-abbrev">Semin. Cell Dev. Biol.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">145</span>–<span class="cit-lpage">153</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSeminars%2Bin%2Bcell%2B%2526%2Bdevelopmental%2Bbiology%26rft.stitle%253DSemin%2BCell%2BDev%2BBiol%26rft.aulast%253DPrior%26rft.auinit1%253DI.%2BA.%26rft.volume%253D23%26rft.issue%253D2%26rft.spage%253D145%26rft.epage%253D153%26rft.atitle%253DRas%2Btrafficking%252C%2Blocalization%2Band%2Bcompartmentalized%2Bsignalling.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.semcdb.2011.09.002%26rft_id%253Dinfo%253Apmid%252F21924373%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.semcdb.2011.09.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21924373&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3185.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3185.33" data-doi="10.1126/science.1194208"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rockman</span>  <span class="cit-name-given-names">M. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Skrovanek</span>  <span class="cit-name-given-names">S. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kruglyak</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Selection at linked sites shapes heritable phenotypic variation in C. elegans.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">330</span>: <span class="cit-fpage">372</span>–<span class="cit-lpage">376</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DRockman%26rft.auinit1%253DM.%2BV.%26rft.volume%253D330%26rft.issue%253D6002%26rft.spage%253D372%26rft.epage%253D376%26rft.atitle%253DSelection%2Bat%2BLinked%2BSites%2BShapes%2BHeritable%2BPhenotypic%2BVariation%2Bin%2BC.%2Belegans%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1194208%26rft_id%253Dinfo%253Apmid%252F20947766%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMzAvNjAwMi8zNzIiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzkvMzE4NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3185.34" data-doi="10.1371/journal.pgen.1005236"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schmid</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Snoek</span>  <span class="cit-name-given-names">L. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Frohli</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van der Bent</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kammenga</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Systemic regulation of RAS/MAPK signaling by the serotonin metabolite 5-HIAA.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">e1005236</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D11%26rft.spage%253De1005236%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1005236%26rft_id%253Dinfo%253Apmid%252F25978500%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1005236&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25978500&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3185.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3185.35" data-doi="10.1126/science.1151107"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Seidel</span>  <span class="cit-name-given-names">H. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rockman</span>  <span class="cit-name-given-names">M. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kruglyak</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Widespread genetic incompatibility in C. elegans maintained by balancing selection.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">319</span>: <span class="cit-fpage">589</span>–<span class="cit-lpage">594</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1151107%26rft_id%253Dinfo%253Apmid%252F18187622%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMTkvNTg2My81ODkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzkvMzE4NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3185.36" data-doi="10.1101/gr.1239303"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shannon</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Markiel</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ozier</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baliga</span>  <span class="cit-name-given-names">N. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Cytoscape: a software environment for integrated models of biomolecular interaction networks.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">2498</span>–<span class="cit-lpage">2504</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DShannon%26rft.auinit1%253DP.%26rft.volume%253D13%26rft.issue%253D11%26rft.spage%253D2498%26rft.epage%253D2504%26rft.atitle%253DCytoscape%253A%2BA%2BSoftware%2BEnvironment%2Bfor%2BIntegrated%2BModels%2Bof%2BBiomolecular%2BInteraction%2BNetworks%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.1239303%26rft_id%253Dinfo%253Apmid%252F14597658%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIxMy8xMS8yNDk4IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy85LzMxODUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3185.37" data-doi="10.1016/S1046-2023(03)00155-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Smyth</span>  <span class="cit-name-given-names">G. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Speed</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Normalization of cDNA microarray data.</span> <abbr class="cit-jnl-abbrev">Methods</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">265</span>–<span class="cit-lpage">273</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2B%2528San%2BDiego%252C%2BCalif.%2529%26rft.stitle%253DMethods%26rft.aulast%253DSmyth%26rft.auinit1%253DG.%2BK.%26rft.volume%253D31%26rft.issue%253D4%26rft.spage%253D265%26rft.epage%253D273%26rft.atitle%253DNormalization%2Bof%2BcDNA%2Bmicroarray%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1046-2023%252803%252900155-5%26rft_id%253Dinfo%253Apmid%252F14597310%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S1046-2023(03)00155-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14597310&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3185.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000186547800002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3185.38"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Snoek</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sterken</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bevers</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Volkers</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van’t Hof</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Contribution of trans regulatory eQTL to cryptic genetic variation in C. elegans.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">18</span>: 500.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3185.39"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Snoek</span>  <span class="cit-name-given-names">L. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Terpstra</span>  <span class="cit-name-given-names">I. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dekter</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van den Ackerveken</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peeters</span>  <span class="cit-name-given-names">A. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genetical genomics reveals large scale genotype-by-environment interactions in Arabidopsis thaliana.</span> <abbr class="cit-jnl-abbrev">Front. Genet.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">317</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BGenet.%26rft.volume%253D3%26rft.spage%253D317%26rft_id%253Dinfo%253Apmid%252F23335938%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=23335938&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3185.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3185.40" data-doi="10.1093/nar/gks1124"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Snoek</span>  <span class="cit-name-given-names">L. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van der Velde</span>  <span class="cit-name-given-names">K. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arends</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beyer</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">WormQTL–public archive and analysis web portal for natural variation data in Caenorhabditis spp.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">D738</span>–<span class="cit-lpage">D743</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgks1124%26rft_id%253Dinfo%253Apmid%252F23180786%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gks1124&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23180786&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3185.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3185.41" data-doi="10.1016/S0960-9822(00)00343-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Solari</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ahringer</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">NURD-complex genes antagonise Ras-induced vulval development in Caenorhabditis elegans.</span> <abbr class="cit-jnl-abbrev">Curr. Biol.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">223</span>–<span class="cit-lpage">226</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bbiology%2B%253A%2B%2BCB%26rft.stitle%253DCurr%2BBiol%26rft.aulast%253DSolari%26rft.auinit1%253DF.%26rft.volume%253D10%26rft.issue%253D4%26rft.spage%253D223%26rft.epage%253D226%26rft.atitle%253DNURD-complex%2Bgenes%2Bantagonise%2BRas-induced%2Bvulval%2Bdevelopment%2Bin%2BCaenorhabditis%2Belegans.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0960-9822%252800%252900343-2%26rft_id%253Dinfo%253Apmid%252F10704416%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0960-9822(00)00343-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10704416&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3185.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000088977600024&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3185.42"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Solari</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bateman</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ahringer</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">The Caenorhabditis elegans genes egl-27 and egr-1 are similar to MTA1, a member of a chromatin regulatory complex, and are redundantly required for embryonic patterning.</span> <abbr class="cit-jnl-abbrev">Development</abbr> <span class="cit-vol">126</span>: <span class="cit-fpage">2483</span>–<span class="cit-lpage">2494</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopment%26rft.stitle%253DDevelopment%26rft.aulast%253DSolari%26rft.auinit1%253DF%26rft.volume%253D126%26rft.issue%253D11%26rft.spage%253D2483%26rft.epage%253D2494%26rft.atitle%253DThe%2BCaenorhabditis%2Belegans%2Bgenes%2Begl-27%2Band%2Begr-1%2Bare%2Bsimilar%2Bto%2BMTA1%252C%2Ba%2Bmember%2Bof%2Ba%2Bchromatin%2Bregulatory%2Bcomplex%252C%2Band%2Bare%2Bredundantly%2Brequired%2Bfor%2Bembryonic%2Bpatterning%26rft_id%253Dinfo%253Apmid%252F10226007%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZGV2ZWxvcCI7czo1OiJyZXNpZCI7czoxMToiMTI2LzExLzI0ODMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzkvMzE4NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3185.43" data-doi="10.1016/S0092-8674(00)81186-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tan</span>  <span class="cit-name-given-names">P. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lackner</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">S. K.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">MAP kinase signaling specificity mediated by the LIN-1 Ets/LIN-31 WH transcription factor complex during C. elegans vulval induction.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">93</span>: <span class="cit-fpage">569</span>–<span class="cit-lpage">580</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DTan%26rft.auinit1%253DP.%2BB.%26rft.volume%253D93%26rft.issue%253D4%26rft.spage%253D569%26rft.epage%253D580%26rft.atitle%253DMAP%2Bkinase%2Bsignaling%2Bspecificity%2Bmediated%2Bby%2Bthe%2BLIN-1%2BEts%252FLIN-31%2BWH%2Btranscription%2Bfactor%2Bcomplex%2Bduring%2BC.%2Belegans%2Bvulval%2Binduction.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0092-8674%252800%252981186-1%26rft_id%253Dinfo%253Apmid%252F9604932%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0092-8674(00)81186-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9604932&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3185.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000073722200010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3185.44" data-doi="10.1016/j.cell.2013.07.006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tepper</span>  <span class="cit-name-given-names">R. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ashraf</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kaletsky</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kleemann</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Murphy</span>  <span class="cit-name-given-names">C. T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">PQM-1 complements DAF-16 as a key transcriptional regulator of DAF-2-mediated development and longevity.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">154</span>: <span class="cit-fpage">676</span>–<span class="cit-lpage">690</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D154%26rft.spage%253D676%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2013.07.006%26rft_id%253Dinfo%253Apmid%252F23911329%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2013.07.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23911329&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3185.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3185.45" data-doi="10.1534/genetics.115.175950"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">O. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Snoek</span>  <span class="cit-name-given-names">L. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nijveen</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sterken</span>  <span class="cit-name-given-names">M. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Volkers</span>  <span class="cit-name-given-names">R. J. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Remarkably divergent regions punctuate the genome assembly of the Caenorhabditis elegans Hawaiian strain CB4856.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">200</span>: <span class="cit-fpage">975</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.115.175950%26rft_id%253Dinfo%253Apmid%252F25995208%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjAwLzMvOTc1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy85LzMxODUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3185.46" data-doi="10.2174/0929867043364810"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tipton</span>  <span class="cit-name-given-names">K. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boyce</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">O’Sullivan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davey</span>  <span class="cit-name-given-names">G. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Healy</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Monoamine oxidases: certainties and uncertainties.</span> <abbr class="cit-jnl-abbrev">Curr. Med. Chem.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">1965</span>–<span class="cit-lpage">1982</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bmedicinal%2Bchemistry%26rft.stitle%253DCurr%2BMed%2BChem%26rft.aulast%253DTipton%26rft.auinit1%253DK.%2BF.%26rft.volume%253D11%26rft.issue%253D15%26rft.spage%253D1965%26rft.epage%253D1982%26rft.atitle%253DMonoamine%2Boxidases%253A%2Bcertainties%2Band%2Buncertainties.%26rft_id%253Dinfo%253Adoi%252F10.2174%252F0929867043364810%26rft_id%253Dinfo%253Apmid%252F15279561%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2174/0929867043364810&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15279561&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3185.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3185.47" data-doi="10.1093/nar/gkt1044"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">van der Velde</span>  <span class="cit-name-given-names">K. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Haan</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zych</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arends</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Snoek</span>  <span class="cit-name-given-names">L. B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">WormQTLHD–a web database for linking human disease to natural variation data in C. elegans.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">D794</span>–<span class="cit-lpage">D801</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkt1044%26rft_id%253Dinfo%253Apmid%252F24217915%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkt1044&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24217915&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3185.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3185.48" data-doi="10.1101/gr.102160.109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vinuela</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Snoek</span>  <span class="cit-name-given-names">L. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Riksen</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kammenga</span>  <span class="cit-name-given-names">J. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genome-wide gene expression regulation as a function of genotype and age in C. elegans.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">929</span>–<span class="cit-lpage">937</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.102160.109%26rft_id%253Dinfo%253Apmid%252F20488933%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjIwLzcvOTI5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy85LzMxODUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3185.49" data-doi="10.1186/1741-7007-11-93"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Volkers</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Snoek</span>  <span class="cit-name-given-names">L. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hubar</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coopman</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Gene-environment and protein-degradation signatures characterize genomic and phenotypic diversity in wild Caenorhabditis elegans populations.</span> <abbr class="cit-jnl-abbrev">BMC Biol.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">93</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BBiol.%26rft.volume%253D11%26rft.spage%253D93%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1741-7007-11-93%26rft_id%253Dinfo%253Apmid%252F23957880%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1741-7007-11-93&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23957880&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3185.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3185.50" data-doi="10.1016/j.cell.2015.06.037"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vu</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Verster</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schertzberg</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chuluunbaatar</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spensley</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Natural variation in gene expression modulates the severity of mutant phenotypes.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">162</span>: <span class="cit-fpage">391</span>–<span class="cit-lpage">402</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D162%26rft.spage%253D391%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2015.06.037%26rft_id%253Dinfo%253Apmid%252F26186192%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2015.06.037&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26186192&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3185.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3185.51" data-doi="10.1101/gr.5011206"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">West</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Leeuwen</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kozik</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kliebenstein</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doerge</span>  <span class="cit-name-given-names">R. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">High-density haplotyping with microarray-based expression and single feature polymorphism markers in Arabidopsis.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">787</span>–<span class="cit-lpage">795</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.5011206%26rft_id%253Dinfo%253Apmid%252F16702412%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE2LzYvNzg3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy85LzMxODUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3185.52" data-doi="10.1101/gad.8.2.147"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Han</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Suppression of activated Let-60 ras protein defines a role of Caenorhabditis elegans Sur-1 MAP kinase in vulval differentiation.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">147</span>–<span class="cit-lpage">159</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DWu%26rft.auinit1%253DY%26rft.volume%253D8%26rft.issue%253D2%26rft.spage%253D147%26rft.epage%253D159%26rft.atitle%253DSuppression%2Bof%2Bactivated%2BLet-60%2Bras%2Bprotein%2Bdefines%2Ba%2Brole%2Bof%2BCaenorhabditis%2Belegans%2BSur-1%2BMAP%2Bkinase%2Bin%2Bvulval%2Bdifferentiation.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.8.2.147%26rft_id%253Dinfo%253Apmid%252F8299935%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6NzoiOC8yLzE0NyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvOS8zMTg1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3185.53" data-doi="10.1186/1471-2105-8-142"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zahurak</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parmigiani</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scharpf</span>  <span class="cit-name-given-names">R. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berman</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Pre-processing Agilent microarray data.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">142</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DZahurak%26rft.auinit1%253DM.%26rft.volume%253D8%26rft.spage%253D142%26rft.epage%253D142%26rft.atitle%253DPre-processing%2BAgilent%2Bmicroarray%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-8-142%26rft_id%253Dinfo%253Apmid%252F17472750%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-8-142&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17472750&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3185.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3185.54" data-doi="10.1186/gb-2005-6-2-r19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zipperlen</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nairz</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rimann</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Basler</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hafen</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">A universal method for automated gene mapping.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">R19</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DZipperlen%26rft.auinit1%253DP.%26rft.volume%253D6%26rft.issue%253D2%26rft.spage%253DR19%26rft.epage%253DR19%26rft.atitle%253DA%2Buniversal%2Bmethod%2Bfor%2Bautomated%2Bgene%2Bmapping.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2005-6-2-r19%26rft_id%253Dinfo%253Apmid%252F15693948%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2005-6-2-r19&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15693948&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3185.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/7/9/3185.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/9/3177" title="The Stress-Inducible Peroxidase TSA2 Underlies a Conditionally Beneficial Chromosomal Duplication in Saccharomyces cerevisiae" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/9/3195" title="The Integrated Genomic Architecture and Evolution of Dental Divergence in East African Cichlid Fishes (Haplochromis chilotes x H. nyererei)" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/7/9"><span>Volume 7 Issue 9, September 2017</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/7/9.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/7/9.cover-source.jpg?itok=WuYpGT8r" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 7 (9)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/9/3185" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-HnrxDsTwRfxPDdv1OOZbVkcdGTwKFYhoJB0KJtOLMJA" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/7/9/3185" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/7/9/3185" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/7/9/3185" class="active" data-icon-position="" data-hide-link-title="0">Ras/MAPK Modifier Loci Revealed by eQTL in Caenorhabditis elegans</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/486349" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-k0EwVMShrin2-VMGYpbkE-MiHaep-8t0BXQvMs41BLE" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/9/3185.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/7/9/3185" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-5w99pWwfTjEX-9op7X-HmQY7PJWLGfURLfcWzNlRoRk" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/7/9/3185" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/9/3185.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="486349" id="node486349--2" data-pisa="ggg;7/9/3185" data-pisa-master="ggg;g3.117.1120" data-apath="/ggg/7/9/3185.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Ras/MAPK Modifier Loci Revealed by eQTL in <em>Caenorhabditis elegans</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-7119-6213" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Mark G. Sterken</span>, <span class="highwire-citation-author" data-delta="1">Linda van Bemmelen van der Plaat</span>, <span class="highwire-citation-author" data-delta="2">Joost A. G. Riksen</span>, <span class="highwire-citation-author" data-delta="3">Miriam Rodriguez</span>, <span class="highwire-citation-author" data-delta="4">Tobias Schmid</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0002-4098-3721" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Alex Hajnal</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0003-4822-4436" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jan E. Kammenga</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="7"><a href="http://orcid.org/0000-0001-5321-2996" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Basten L. Snoek</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3185-3193;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.1120 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node486349--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Mark G. Sterken </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Laboratory of Nematology, Wageningen University and Research, 6708PB, The Netherlands</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mark%2BG.%2BSterken%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sterken%20MG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMark%2BG.%2BSterken%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-7119-6213" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Mark G. Sterken</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Linda van Bemmelen van der Plaat </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Laboratory of Nematology, Wageningen University and Research, 6708PB, The Netherlands</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Linda%2Bvan%2BBemmelen%2Bvan%2Bder%2BPlaat%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=van%20Bemmelen%20van%20der%20Plaat%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALinda%2Bvan%2BBemmelen%2Bvan%2Bder%2BPlaat%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Joost A. G. Riksen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Laboratory of Nematology, Wageningen University and Research, 6708PB, The Netherlands</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joost%2BA.%2BG.%2BRiksen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Riksen%20JA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJoost%2BA.%2BG.%2BRiksen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Miriam Rodriguez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Laboratory of Nematology, Wageningen University and Research, 6708PB, The Netherlands</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Miriam%2BRodriguez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rodriguez%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMiriam%2BRodriguez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Tobias Schmid </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Institute of Molecular Life Sciences, University of Zurich, CH-8057, Switzerland</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Ph.D. Program in Molecular Life Sciences, University of Zurich, CH-8057, Switzerland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tobias%2BSchmid%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Schmid%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATobias%2BSchmid%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Alex Hajnal </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Institute of Molecular Life Sciences, University of Zurich, CH-8057, Switzerland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alex%2BHajnal%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hajnal%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAlex%2BHajnal%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-4098-3721" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Alex Hajnal</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Jan E. Kammenga </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Laboratory of Nematology, Wageningen University and Research, 6708PB, The Netherlands</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jan%2BE.%2BKammenga%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kammenga%20JE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJan%2BE.%2BKammenga%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-4822-4436" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jan E. Kammenga</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jan.kammenga@wur.nl" class="" data-icon-position="" data-hide-link-title="0">jan.kammenga@wur.nl</a>
<a href="mailto:l.b.snoek@uu.nl" class="" data-icon-position="" data-hide-link-title="0">l.b.snoek@uu.nl</a></span></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Basten L. Snoek </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Laboratory of Nematology, Wageningen University and Research, 6708PB, The Netherlands</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Basten%2BL.%2BSnoek%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Snoek%20BL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABasten%2BL.%2BSnoek%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-5321-2996" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Basten L. Snoek</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jan.kammenga@wur.nl" class="" data-icon-position="" data-hide-link-title="0">jan.kammenga@wur.nl</a>
<a href="mailto:l.b.snoek@uu.nl" class="" data-icon-position="" data-hide-link-title="0">l.b.snoek@uu.nl</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/7/9/3185" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/7/9/3185&amp;title=Ras/MAPK%20Modifier%20Loci%20Revealed%20by%20eQTL%20in%20Caenorhabditis%20elegans" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/7/9/3185&amp;title=Ras/MAPK%20Modifier%20Loci%20Revealed%20by%20eQTL%20in%20Caenorhabditis%20elegans" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/7/9/3185&amp;title=Ras/MAPK%20Modifier%20Loci%20Revealed%20by%20eQTL%20in%20Caenorhabditis%20elegans" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/7/9/3185&amp;text=Ras/MAPK%20Modifier%20Loci%20Revealed%20by%20eQTL%20in%20Caenorhabditis%20elegans" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/7/9/3185&amp;title=Ras/MAPK%20Modifier%20Loci%20Revealed%20by%20eQTL%20in%20Caenorhabditis%20elegans" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/7/9/3185&amp;t=Ras/MAPK%20Modifier%20Loci%20Revealed%20by%20eQTL%20in%20Caenorhabditis%20elegans" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/7/9/3185&amp;title=Ras/MAPK%20Modifier%20Loci%20Revealed%20by%20eQTL%20in%20Caenorhabditis%20elegans" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/7/9/3185&amp;title=Ras/MAPK%20Modifier%20Loci%20Revealed%20by%20eQTL%20in%20Caenorhabditis%20elegans" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/486349/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="486349" id="node486349--3" data-pisa="ggg;7/9/3185" data-pisa-master="ggg;g3.117.1120" data-apath="/ggg/7/9/3185.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Ras/MAPK Modifier Loci Revealed by eQTL in <em>Caenorhabditis elegans</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-7119-6213" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Mark G. Sterken</span>, <span class="highwire-citation-author" data-delta="1">Linda van Bemmelen van der Plaat</span>, <span class="highwire-citation-author" data-delta="2">Joost A. G. Riksen</span>, <span class="highwire-citation-author" data-delta="3">Miriam Rodriguez</span>, <span class="highwire-citation-author" data-delta="4">Tobias Schmid</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0002-4098-3721" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Alex Hajnal</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0003-4822-4436" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jan E. Kammenga</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="7"><a href="http://orcid.org/0000-0001-5321-2996" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Basten L. Snoek</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3185-3193;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.1120 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node486349--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Mark G. Sterken </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Laboratory of Nematology, Wageningen University and Research, 6708PB, The Netherlands</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mark%2BG.%2BSterken%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sterken%20MG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMark%2BG.%2BSterken%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-7119-6213" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Mark G. Sterken</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Linda van Bemmelen van der Plaat </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Laboratory of Nematology, Wageningen University and Research, 6708PB, The Netherlands</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Linda%2Bvan%2BBemmelen%2Bvan%2Bder%2BPlaat%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=van%20Bemmelen%20van%20der%20Plaat%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALinda%2Bvan%2BBemmelen%2Bvan%2Bder%2BPlaat%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Joost A. G. Riksen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Laboratory of Nematology, Wageningen University and Research, 6708PB, The Netherlands</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joost%2BA.%2BG.%2BRiksen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Riksen%20JA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJoost%2BA.%2BG.%2BRiksen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Miriam Rodriguez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Laboratory of Nematology, Wageningen University and Research, 6708PB, The Netherlands</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Miriam%2BRodriguez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rodriguez%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMiriam%2BRodriguez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Tobias Schmid </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Institute of Molecular Life Sciences, University of Zurich, CH-8057, Switzerland</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Ph.D. Program in Molecular Life Sciences, University of Zurich, CH-8057, Switzerland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tobias%2BSchmid%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Schmid%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATobias%2BSchmid%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Alex Hajnal </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Institute of Molecular Life Sciences, University of Zurich, CH-8057, Switzerland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alex%2BHajnal%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hajnal%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAlex%2BHajnal%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-4098-3721" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Alex Hajnal</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Jan E. Kammenga </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Laboratory of Nematology, Wageningen University and Research, 6708PB, The Netherlands</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jan%2BE.%2BKammenga%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kammenga%20JE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJan%2BE.%2BKammenga%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-4822-4436" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jan E. Kammenga</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jan.kammenga@wur.nl" class="" data-icon-position="" data-hide-link-title="0">jan.kammenga@wur.nl</a>
<a href="mailto:l.b.snoek@uu.nl" class="" data-icon-position="" data-hide-link-title="0">l.b.snoek@uu.nl</a></span></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Basten L. Snoek </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Laboratory of Nematology, Wageningen University and Research, 6708PB, The Netherlands</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Basten%2BL.%2BSnoek%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Snoek%20BL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABasten%2BL.%2BSnoek%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-5321-2996" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Basten L. Snoek</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jan.kammenga@wur.nl" class="" data-icon-position="" data-hide-link-title="0">jan.kammenga@wur.nl</a>
<a href="mailto:l.b.snoek@uu.nl" class="" data-icon-position="" data-hide-link-title="0">l.b.snoek@uu.nl</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=3185&amp;rft.epage=3193&amp;rft.atitle=Ras/MAPK%20Modifier%20Loci%20Revealed%20by%20eQTL%20in%20Caenorhabditis%20elegans&amp;rft.volume=7&amp;rft.issue=9&amp;rft.date=2017-09-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Sterken%2C+Mark+G.&amp;rft.au=van+Bemmelen+van+der+Plaat%2C+Linda&amp;rft.au=Riksen%2C+Joost+A.+G.&amp;rft.au=Rodriguez%2C+Miriam&amp;rft.au=Schmid%2C+Tobias&amp;rft.au=Hajnal%2C+Alex&amp;rft.au=Kammenga%2C+Jan+E.&amp;rft.au=Snoek%2C+Basten+L."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/486349/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/486349/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/486349/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/486349/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/486349/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/486349/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/486349/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/486349/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/486349/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/486349/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/486349/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/486349/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/7/9/3185" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-14" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-19" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/7/9/3185.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/7/9/3185.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/7/9/3185.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/7/9/3185.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
