<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/7/9/2957" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/7/9/2957.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/7/9/2957.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/7/9/2957.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/7/9/2957/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/7/9/2957.atom" />
<meta name="HW.pisa" content="ggg;7/9/2957" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Reduced Protein Expression in a Virus Attenuated by Codon Deoptimization" />
<meta name="DC.Identifier" content="10.1534/g3.117.041020" />
<meta name="DC.Date" content="2017-09-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2017 Jack et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="A general means of viral attenuation involves the extensive recoding of synonymous codons in the viral genome. The mechanistic underpinnings of this approach remain unclear, however. Using quantitative proteomics and RNA sequencing, we explore the molecular basis of attenuation in a strain of bacteriophage T7 whose major capsid gene was engineered to carry 182 suboptimal codons. We do not detect transcriptional effects from recoding. Proteomic observations reveal that translation is halved for the recoded major capsid gene, and a more modest reduction applies to several coexpressed downstream genes. We observe no changes in protein abundances of other coexpressed genes that are encoded upstream. Viral burst size, like capsid protein abundance, is also decreased by half. Together, these observations suggest that, in this virus, reduced translation of an essential polycistronic transcript and diminished virion assembly form the molecular basis of attenuation." />
<meta name="DC.Contributor" content="Benjamin R. Jack" />
<meta name="DC.Contributor" content="Daniel R. Boutz" />
<meta name="DC.Contributor" content="Matthew L. Paff" />
<meta name="DC.Contributor" content="Bartram L. Smith" />
<meta name="DC.Contributor" content="James J. Bull" />
<meta name="DC.Contributor" content="Claus O. Wilke" />
<meta name="article:published_time" content="2017-09-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Reduced Protein Expression in a Virus Attenuated by Codon Deoptimization" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;A general means of viral attenuation involves the extensive recoding of synonymous codons in the viral genome. The mechanistic underpinnings of this approach remain unclear, however. Using quantitative proteomics and RNA sequencing, we explore the molecular basis of attenuation in a strain of bacteriophage T7 whose major capsid gene was engineered to carry 182 suboptimal codons. We do not detect transcriptional effects from recoding. Proteomic observations reveal that translation is halved for the recoded major capsid gene, and a more modest reduction applies to several coexpressed downstream genes. We observe no changes in protein abundances of other coexpressed genes that are encoded upstream. Viral burst size, like capsid protein abundance, is also decreased by half. Together, these observations suggest that, in this virus, reduced translation of an essential polycistronic transcript and diminished virion assembly form the molecular basis of attenuation.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2017/09/01" />
<meta name="citation_mjid" content="ggg;7/9/2957" />
<meta name="citation_id" content="7/9/2957" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/7/9/2957" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/7/9/2957.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/7/9/2957.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/7/9/2957.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.117.041020" />
<meta name="citation_pmid" content="28698233" />
<meta name="citation_volume" content="7" />
<meta name="citation_issue" content="9" />
<meta name="citation_num_pages" content="12" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="2957" />
<meta name="citation_lastpage" content="2968" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Benjamin R. Jack" />
<meta name="citation_author_institution" content="Department of Integrative Biology, The University of Texas at Austin, Texas 78712" />
<meta name="citation_author_institution" content="Center for Computational Biology and Bioinformatics, The University of Texas at Austin, Texas 78712" />
<meta name="citation_author_institution" content="Institute for Cellular and Molecular Biology, The University of Texas at Austin, Texas 78712" />
<meta name="citation_author_email" content="benjamin.r.jack@gmail.com" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-7777-1653" />
<meta name="citation_author" content="Daniel R. Boutz" />
<meta name="citation_author_institution" content="Institute for Cellular and Molecular Biology, The University of Texas at Austin, Texas 78712" />
<meta name="citation_author" content="Matthew L. Paff" />
<meta name="citation_author_institution" content="Department of Integrative Biology, The University of Texas at Austin, Texas 78712" />
<meta name="citation_author_institution" content="Center for Computational Biology and Bioinformatics, The University of Texas at Austin, Texas 78712" />
<meta name="citation_author_institution" content="Institute for Cellular and Molecular Biology, The University of Texas at Austin, Texas 78712" />
<meta name="citation_author" content="Bartram L. Smith" />
<meta name="citation_author_institution" content="Department of Integrative Biology, The University of Texas at Austin, Texas 78712" />
<meta name="citation_author_institution" content="Center for Computational Biology and Bioinformatics, The University of Texas at Austin, Texas 78712" />
<meta name="citation_author_institution" content="Institute for Cellular and Molecular Biology, The University of Texas at Austin, Texas 78712" />
<meta name="citation_author" content="James J. Bull" />
<meta name="citation_author_institution" content="Department of Integrative Biology, The University of Texas at Austin, Texas 78712" />
<meta name="citation_author_institution" content="Center for Computational Biology and Bioinformatics, The University of Texas at Austin, Texas 78712" />
<meta name="citation_author_institution" content="Institute for Cellular and Molecular Biology, The University of Texas at Austin, Texas 78712" />
<meta name="citation_author" content="Claus O. Wilke" />
<meta name="citation_author_institution" content="Department of Integrative Biology, The University of Texas at Austin, Texas 78712" />
<meta name="citation_author_institution" content="Center for Computational Biology and Bioinformatics, The University of Texas at Austin, Texas 78712" />
<meta name="citation_author_institution" content="Institute for Cellular and Molecular Biology, The University of Texas at Austin, Texas 78712" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-7470-9261" />
<meta name="citation_reference" content="citation_journal_title=Journal of Bacteriology;citation_journal_abbrev=J. Bacteriol.;citation_author=S. Aksoy;citation_author=C. L. Squires;citation_author=C. Squires;citation_title=Translational coupling of the trpB and trpA genes in the Escherichia coli tryptophan operon.;citation_pages=363-367;citation_volume=157;citation_year=1984;citation_issue=2;citation_pmid=6319355" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=N. J. Atkinson;citation_author=J. Witteveldt;citation_author=D. J. Evans;citation_author=P. Simmonds;citation_title=The influence of CpG and UpA dinucleotide frequencies on RNA virus replication and characterization of the innate cellular pathways underlying virus attenuation and enhanced replication.;citation_volume=42;citation_year=2014;citation_pmid=24470146;citation_doi=10.1093/nar/gku075" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=T. Barrett;citation_author=S. E. Wilhite;citation_author=P. Ledoux;citation_author=C. Evangelista;citation_author=I. F. Kim;citation_title=NCBI GEO: archive for functional genomics data sets—update.;citation_volume=41;citation_year=2013;citation_pmid=23193258;citation_doi=10.1093/nar/gks1193" />
<meta name="citation_reference" content="citation_journal_title=J. Stat. Softw.;citation_author=D. Bates;citation_author=M. Mächler;citation_author=B. Bolker;citation_author=S. Walker;citation_title=Fitting linear mixed-effects models using lme4.;citation_pages=1-48;citation_volume=67;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Comput Biol;citation_author=E. W. Birch;citation_author=N. A. Ruggero;citation_author=M. W. Covert;citation_title=Determining Host Metabolic Limitations on Viral Replication via Integrated Modeling and Experimental Perturbation.;citation_pages=e1002746-e1002746;citation_volume=8;citation_year=2012;citation_issue=10;citation_pmid=23093930;citation_doi=10.1371/journal.pcbi.1002746" />
<meta name="citation_reference" content="citation_journal_title=Nat. Biotechnol.;citation_author=N. L. Bray;citation_author=H. Pimentel;citation_author=P. Melsted;citation_author=L. Pachter;citation_title=Near-optimal probabilistic RNA-seq quantification.;citation_pages=525-527;citation_volume=34;citation_year=2016;citation_pmid=27043002;citation_doi=10.1038/nbt.3519" />
<meta name="citation_reference" content="citation_journal_title=Journal of Theoretical Biology;citation_journal_abbrev=Journal of Theoretical Biology;citation_author=J. J. Bull;citation_title=Optimality models of phage life history and parallels in disease evolution.;citation_pages=928-938;citation_volume=241;citation_year=2006;citation_issue=4;citation_pmid=16616205;citation_doi=10.1016/j.jtbi.2006.01.027" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=J. J. Bull;citation_author=R. H. Heineman;citation_author=C. O. Wilke;citation_title=The phenotype-fitness map in experimental evolution of phages.;citation_pages=e27796-e27796;citation_volume=6;citation_year=2011;citation_issue=11;citation_pmid=22132144;citation_doi=10.1371/journal.pone.0027796" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=J. J. Bull;citation_author=I. J. Molineux;citation_author=C. O. Wilke;citation_title=Slow fitness recovery in a codon-modified viral genome.;citation_volume=29;citation_year=2012;citation_pmid=22532576;citation_doi=10.1093/molbev/mss119" />
<meta name="citation_reference" content="citation_journal_title=Journal of Virology;citation_journal_abbrev=J. Virol.;citation_author=C. C. Burns;citation_author=J. Shaw;citation_author=R. Campagnoli;citation_author=J. Jorba;citation_author=A. Vincent;citation_title=Modulation of Poliovirus Replicative Fitness in HeLa Cells by Deoptimization of Synonymous Codon Usage in the Capsid Region;citation_pages=3259-3272;citation_volume=80;citation_year=2006;citation_issue=7;citation_pmid=16537593;citation_doi=10.1128/JVI.80.7.3259-3272.2006" />
<meta name="citation_reference" content="citation_journal_title=J. Virol.;citation_author=C. C. Burns;citation_author=R. Campagnoli;citation_author=J. Shaw;citation_author=A. Vincent;citation_author=J. Jorba;citation_title=Genetic inactivation of poliovirus infectivity by increasing the frequencies of CpG and UpA dinucleotides within and across synonymous capsid region codons.;citation_volume=83;citation_year=2009;citation_pmid=19605476;citation_doi=10.1128/JVI.00508-09" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=G. Cannarozzi;citation_author=N. N. Schraudolph;citation_author=M. Faty;citation_author=P. von Rohr;citation_author=M. T. Friberg;citation_title=A role for codon order in translation dynamics.;citation_pages=355-367;citation_volume=141;citation_year=2010;citation_issue=2;citation_pmid=20403329;citation_doi=10.1016/j.cell.2010.02.036" />
<meta name="citation_reference" content="citation_journal_title=G3;citation_author=N. Cecchini;citation_author=M. Schmerer;citation_author=I. J. Molineux;citation_author=R. Springman;citation_author=J. J. Bull;citation_title=Evolutionarily stable attenuation by genome rearrangement in a virus.;citation_pages=1389-1397;citation_volume=3;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=F. F. V. Chevance;citation_author=K. T. Hughes;citation_title=Case for the genetic code as a triplet of triplets.;citation_volume=114;citation_year=2017;citation_pmid=28416671;citation_doi=10.1073/pnas.1614896114" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=J. R. Coleman;citation_author=D. Papamichail;citation_author=S. Skiena;citation_author=B. Futcher;citation_author=E. Wimmer;citation_title=Virus Attenuation by Genome-Scale Changes in Codon Pair Bias;citation_pages=1784-1787;citation_volume=320;citation_year=2008;citation_issue=5884;citation_pmid=18583614;citation_doi=10.1126/science.1155761" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=J. J. Dunn;citation_author=F. W. Studier;citation_title=Complete nucleotide sequence of bacteriophage T7 DNA and the locations of T7 genetic elements.;citation_pages=477-535;citation_volume=166;citation_year=1983;citation_issue=4;citation_pmid=6864790;citation_doi=10.1016/0022-2836(83)90062-1" />
<meta name="citation_reference" content="citation_journal_title=Biotechnology and Bioengineering;citation_journal_abbrev=Biotechnology and Bioengineering;citation_author=D. Endy;citation_author=D. Kong;citation_author=J. Yin;citation_title=Intracellular kinetics of a growing virus: a genetically structured simulation for bacteriophage T7.;citation_pages=375-389;citation_volume=55;citation_year=1997;citation_issue=2;citation_pmid=18636496;citation_doi=10.1002/(SICI)1097-0290(19970720)55:2&lt;375::AID-BIT15&gt;3.0.CO;2-G" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=K. Fredrick;citation_author=M. Ibba;citation_title=How the sequence of a gene can tune its translation.;citation_pages=227-229;citation_volume=141;citation_year=2010;citation_issue=2;citation_pmid=20403320;citation_doi=10.1016/j.cell.2010.03.033" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_author=C. E. Gamble;citation_author=C. E. Brule;citation_author=K. M. Dean;citation_author=S. Fields;citation_author=E. J. Grayhack;citation_title=Adjacent codons act in concert to modulate translation efficiency in yeast.;citation_pages=679-690;citation_volume=166;citation_year=2016;citation_doi=10.1016/j.cell.2016.05.070" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=S. Guyader;citation_author=C. L. Burch;citation_title=Optimal foraging predicts the ecology but not the evolution of host specialization in bacteriophages.;citation_pages=e1946-e1946;citation_volume=3;citation_year=2008;citation_issue=4;citation_pmid=18414655;citation_doi=10.1371/journal.pone.0001946" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_journal_abbrev=Evolution;citation_author=R. H. Heineman;citation_author=J. J. Bull;citation_title=Testing optimality with experimental evolution: lysis time in a bacteriophage.;citation_pages=1695-1709;citation_volume=61;citation_year=2007;citation_issue=7;citation_pmid=17598749;citation_doi=10.1111/j.1558-5646.2007.00132.x" />
<meta name="citation_reference" content="citation_journal_title=Molecular microbiology;citation_journal_abbrev=Mol Microbiol;citation_author=K. Hellmuth;citation_author=G. Rex;citation_author=B. Surin;citation_author=R. Zinck;citation_author=J. E. G. McCarthy;citation_title=Translational coupling varying in efficiency between different pairs of genes in the central region of the atp operon of Escherichia coli.;citation_pages=813-824;citation_volume=5;citation_year=1991;citation_issue=4;citation_pmid=1830358;citation_doi=10.1111/j.1365-2958.1991.tb00754.x" />
<meta name="citation_reference" content="citation_journal_title=PLoS Comput. Biol.;citation_author=J. R. Houser;citation_author=C. Barnhart;citation_author=D. R. Boutz;citation_author=S. M. Carroll;citation_author=A. Dasgupta;citation_title=Controlled measurement and comparative analysis of cellular components in E. coli reveals broad regulatory changes in response to glucose starvation.;citation_pages=e1004400;citation_volume=11;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=S. Kosuri;citation_author=J. R. Kelly;citation_author=D. Endy;citation_title=TABASCO: A single molecule, base-pair resolved gene expression simulator.;citation_pages=480-480;citation_volume=8;citation_year=2007;citation_pmid=18093293;citation_doi=10.1186/1471-2105-8-480" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=G. Kudla;citation_author=A. W. Murray;citation_author=D. Tollervey;citation_author=J. B. Plotkin;citation_title=Coding-Sequence Determinants of Gene Expression in Escherichia coli;citation_pages=255-258;citation_volume=324;citation_year=2009;citation_issue=5924;citation_pmid=19359587;citation_doi=10.1126/science.1170160" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=P. Lesage;citation_author=C. Chiaruttini;citation_author=M. Graffe;citation_author=J. Dondon;citation_author=M. Milet;citation_title=Messenger RNA secondary structure and translational coupling in the Escherichia coli operon encoding translation initiation factor IF3 and the ribosomal proteins, L35 and L20.;citation_pages=366-386;citation_volume=228;citation_year=1992;citation_issue=2;citation_pmid=1453449;citation_doi=10.1016/0022-2836(92)90827-7" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_author=G. W. Li;citation_author=D. Burkhardt;citation_author=C. Gross;citation_author=J. S. Weissman;citation_title=Quantifying absolute protein synthesis rates reveals principles underlying allocation of cellular resources.;citation_pages=624-635;citation_volume=157;citation_year=2014;citation_pmid=24766808;citation_doi=10.1016/j.cell.2014.02.033" />
<meta name="citation_reference" content="citation_title=The Bacteriophages;citation_title=The T7 group;citation_author=I. J. Molineux;citation_pages=277-301;citation_year=2006" />
<meta name="citation_reference" content="citation_journal_title=Journal of Virology;citation_journal_abbrev=J. Virol.;citation_author=S. Mueller;citation_author=D. Papamichail;citation_author=J. R. Coleman;citation_author=S. Skiena;citation_author=E. Wimmer;citation_title=Reduction of the Rate of Poliovirus Protein Synthesis through Large-Scale Codon Deoptimization Causes Attenuation of Viral Virulence by Lowering Specific Infectivity;citation_pages=9687-9696;citation_volume=80;citation_year=2006;citation_issue=19;citation_pmid=16973573;citation_doi=10.1128/JVI.00738-06" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=S. Mueller;citation_author=J. R. Coleman;citation_author=D. Papamichail;citation_author=C. B. Ward;citation_author=A. Nimnual;citation_title=Live attenuated influenza virus vaccines by computer-aided rational design.;citation_pages=723-726;citation_volume=28;citation_year=2010;citation_issue=7;citation_pmid=20543832;citation_doi=10.1038/nbt.1636" />
<meta name="citation_reference" content="citation_journal_title=PLoS Pathog.;citation_author=A. Nougairede;citation_author=L. De Fabritus;citation_author=F. Aubry;citation_author=E. A. Gould;citation_author=E. C. Holmes;citation_title=Random codon re-encoding induces stable reduction of replicative fitness of chikungunya virus in primate and mosquito cells.;citation_pages=1-18;citation_volume=9;citation_year=2013;citation_doi=10.1371/journal.ppat.1003800" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=D. S. Oppenheim;citation_author=C. Yanofsky;citation_title=TRANSLATIONAL COUPLING DURING EXPRESSION OF THE TRYPTOPHAN OPERON OF ESCHERICHIA COLI;citation_pages=785-795;citation_volume=95;citation_year=1980;citation_issue=4;citation_pmid=6162715" />
<meta name="citation_reference" content="citation_journal_title=Journal of Theoretical Biology;citation_journal_abbrev=Journal of Theoretical Biology;citation_author=Z. Patwa;citation_author=L. Wahl;citation_title=The impact of host-cell dynamics on the fixation probability for lytic viruses.;citation_pages=799-810;citation_volume=259;citation_year=2009;citation_issue=4;citation_pmid=19464301;citation_doi=10.1016/j.jtbi.2009.05.008" />
<meta name="citation_reference" content="citation_journal_title=Nat. Rev. Genet.;citation_author=J. B. Plotkin;citation_author=G. Kudla;citation_title=Synonymous but not the same: the causes and consequences of codon bias.;citation_pages=32-42;citation_volume=12;citation_year=2010;citation_pmid=21102527;citation_doi=10.1038/nrg2899" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=X. Qu;citation_author=J. D. Wen;citation_author=L. Lancaster;citation_author=H. F. Noller;citation_author=C. Bustamante;citation_title=The ribosome uses two active mechanisms to unwind messenger RNA during translation.;citation_pages=118-121;citation_volume=475;citation_year=2011;citation_issue=7354;citation_pmid=21734708;citation_doi=10.1038/nature10126" />
<meta name="citation_reference" content="citation_title=R: A Language and Environment for Statistical Computing;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=J. R. Soc. Interface;citation_author=A. Raveh;citation_author=M. Margaliot;citation_author=E. D. Sontag;citation_author=T. Tuller;citation_title=A model for competition for ribosomes in the cell.;citation_pages=e1002127;citation_volume=13;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Comput Biol;citation_author=S. Reuveni;citation_author=I. Meilijson;citation_author=M. Kupiec;citation_author=E. Ruppin;citation_author=T. Tuller;citation_title=Genome-scale analysis of translation elongation with a ribosome flow model.;citation_pages=e1002127-e1002127;citation_volume=7;citation_year=2011;citation_issue=9;citation_pmid=21909250;citation_doi=10.1371/journal.pcbi.1002127" />
<meta name="citation_reference" content="citation_journal_title=Journal of Biological Chemistry;citation_journal_abbrev=J Biol Chem;citation_author=G. Rex;citation_author=B. Surin;citation_author=G. Besse;citation_author=B. Schneppe;citation_author=J. E. McCarthy;citation_title=The mechanism of translational coupling in Escherichia coli. Higher order structure in the atpHA mRNA acts as a conformational switch regulating the access of de novo initiating ribosomes;citation_pages=18118-18127;citation_volume=269;citation_year=1994;citation_issue=27;citation_pmid=7517937" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=D. Schümperli;citation_author=K. McKenney;citation_author=D. A. Sobieski;citation_author=M. Rosenberg;citation_title=Translational coupling at an intercistronic boundary of the Escherichia coli galactose operon.;citation_pages=865-871;citation_volume=30;citation_year=1982;citation_issue=3;citation_pmid=6754091;citation_doi=10.1016/0092-8674(82)90291-4" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=M. Scott;citation_author=C. W. Gunderson;citation_author=E. M. Mateescu;citation_author=Z. Zhang;citation_author=T. Hwa;citation_title=Interdependence of Cell Growth and Gene Expression: Origins and Consequences;citation_pages=1099-1102;citation_volume=330;citation_year=2010;citation_issue=6007;citation_pmid=21097934;citation_doi=10.1126/science.1192588" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=P. Shah;citation_author=M. A. Gilchrist;citation_title=Explaining complex codon usage patterns with selection for translational efficiency, mutation bias, and genetic drift.;citation_volume=108;citation_year=2011;citation_pmid=21646514;citation_doi=10.1073/pnas.1016719108" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=Y. Shao;citation_author=N. Wang;citation_title=Bacteriophage adsorption rate and optimal lysis time.;citation_volume=180;citation_year=2008;citation_pmid=18757924;citation_doi=10.1534/genetics.108.090100" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=S. H. Shen;citation_author=C. B. Stauft;citation_author=O. Gorbatsevych;citation_author=Y. Song;citation_author=C. B. Ward;citation_title=Large-scale recoding of an arbovirus genome to rebalance its insect vs. mammalian preference.;citation_volume=112;citation_year=2015;citation_pmid=25825721;citation_doi=10.1073/pnas.1502864112" />
<meta name="citation_reference" content="citation_journal_title=Mol. Cell. Proteomics;citation_author=J. C. Silva;citation_author=M. V. Gorenstein;citation_author=G. Z. Li;citation_author=J. P. C. Vissers;citation_author=S. J. Geromanos;citation_title=Absolute quantification of proteins by LCMSE: a virtue of parallel MS acquisition.;citation_volume=5;citation_year=2006;citation_pmid=16219938;citation_doi=10.1074/mcp.M500230-MCP200" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=R. A. Spanjaard;citation_author=J. van Duin;citation_title=Translational reinitiation in the presence and absence of a Shine and Dalgarno sequence;citation_pages=5501-5508;citation_volume=17;citation_year=1989;citation_issue=14;citation_pmid=2668889;citation_doi=10.1093/nar/17.14.5501" />
<meta name="citation_reference" content="citation_journal_title=Virology;citation_journal_abbrev=Virology;citation_author=R. Springman;citation_author=M. Badgett;citation_author=I. Molineux;citation_author=J. Bull;citation_title=Gene order constrains adaptation in bacteriophage T7.;citation_pages=141-152;citation_volume=341;citation_year=2005;citation_issue=1;citation_pmid=16081122;citation_doi=10.1016/j.virol.2005.07.008" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=T. Tian;citation_author=H. M. Salis;citation_title=A predictive biophysical model of translational coupling to coordinate and control protein expression in bacterial operons.;citation_volume=43;citation_year=2015;citation_pmid=26117546;citation_doi=10.1093/nar/gkv635" />
<meta name="citation_reference" content="citation_journal_title=Journal of Bacteriology;citation_journal_abbrev=J. Bacteriol.;citation_author=M. Y. Torgov;citation_author=D. M. Janzen;citation_author=M. K. Reddy;citation_title=Efficiency and Frequency of Translational Coupling between the Bacteriophage T4 Clamp Loader Genes;citation_pages=4339-4343;citation_volume=180;citation_year=1998;citation_issue=17;citation_pmid=9721267" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=T. Tuller;citation_author=A. Carmi;citation_author=K. Vestsigian;citation_author=S. Navon;citation_author=Y. Dorfan;citation_title=An evolutionarily conserved mechanism for controlling the efficiency of protein translation.;citation_pages=344-354;citation_volume=141;citation_year=2010;citation_issue=2;citation_pmid=20403328;citation_doi=10.1016/j.cell.2010.03.031" />
<meta name="citation_reference" content="citation_journal_title=Elife;citation_author=F. Tulloch;citation_author=N. J. Atkinson;citation_author=D. J. Evans;citation_author=M. D. Ryan;citation_author=P. Simmonds;citation_title=RNA virus attenuation by codon pair deoptimisation is an artefact of increases in CpG/UpA dinucleotide frequencies.;citation_volume=3;citation_year=2014;citation_pmid=25490153;citation_doi=10.7554/eLife.04531" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=J. Vind;citation_author=M. A. Sørensen;citation_author=M. D. Rasmussen;citation_author=S. Pedersen;citation_title=Synthesis of proteins in Escherichia coli is limited by the concentration of free ribosomes. Expression from reporter genes does not always reflect functional mRNA levels.;citation_pages=678-688;citation_volume=231;citation_year=1993;citation_issue=3;citation_pmid=7685825;citation_doi=10.1006/jmbi.1993.1319" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=J. Vizcaíno;citation_author=A. Csordas;citation_author=N. del Toro;citation_author=J. A. Dianes;citation_author=J. Griss;citation_title=2016 update of the PRIDE database and its related tools.;citation_volume=44;citation_year=2016;citation_pmid=26527722;citation_doi=10.1093/nar/gkv1145" />
<meta name="citation_reference" content="citation_journal_title=Evol. Ecol.;citation_author=N. Wang;citation_author=D. E. Dykhuizen;citation_author=L. B. Slobodkin;citation_title=The evolution of phage lysis timing.;citation_pages=545-558;citation_volume=10;citation_year=1996;citation_doi=10.1007/BF01237884" />
<meta name="citation_reference" content="citation_title=ggplot2: Elegant Graphics for Data Analysis.;citation_author=H. Wickham;citation_year=2009" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=E. Wimmer;citation_author=S. Mueller;citation_author=T. M. Tumpey;citation_author=J. K. Taubenberger;citation_title=Synthetic viruses: a new opportunity to understand and prevent viral disease.;citation_pages=1163-1172;citation_volume=27;citation_year=2009;citation_issue=12;citation_pmid=20010599;citation_doi=10.1038/nbt.1593" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=Y. Xu;citation_author=P. Ma;citation_author=P. Shah;citation_author=A. Rokas;citation_author=Y. Liu;citation_title=Non-optimal codon usage is a mechanism to achieve circadian clock conditionality.;citation_pages=116-120;citation_volume=495;citation_year=2013;citation_pmid=23417065;citation_doi=10.1038/nature11942" />
<meta name="citation_reference" content="citation_journal_title=Journal of Bacteriology;citation_journal_abbrev=J. Bacteriol.;citation_author=L. You;citation_author=P. F. Suthers;citation_author=J. Yin;citation_title=Effects of Escherichia coli Physiology on Growth of Phage T7 In Vivo and In Silico;citation_pages=1888-1894;citation_volume=184;citation_year=2002;citation_issue=7;citation_pmid=11889095;citation_doi=10.1128/JB.184.7.1888-1894.2002" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=M. Zhou;citation_author=J. Guo;citation_author=J. Cha;citation_author=M. Chae;citation_author=S. Chen;citation_title=Non-optimal codon usage affects expression, structure and function of clock protein FRQ.;citation_pages=111-115;citation_volume=495;citation_year=2013;citation_pmid=23417067;citation_doi=10.1038/nature11833" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=Z. Zhou;citation_author=Y. Dang;citation_author=M. Zhou;citation_author=L. Li;citation_author=Ch. Yu;citation_title=Codon usage is an important determinant of gene expression levels largely through its effects on transcription.;citation_volume=113;citation_year=2016;citation_pmid=27671647;citation_doi=10.1073/pnas.1606724113" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=H. Zur;citation_author=T. Tuller;citation_title=Predictive biophysical modeling and understanding of the dynamics of mRNA translation and its evolution.;citation_volume=44;citation_year=2016;citation_pmid=27591251;citation_doi=10.1093/nar/gkw764" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Reduced Protein Expression in a Virus Attenuated by Codon Deoptimization" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9.cover-source.jpg" />
<meta name="twitter:description" content="A general means of viral attenuation involves the extensive recoding of synonymous codons in the viral genome. The mechanistic underpinnings of this approach remain unclear, however. Using quantitative proteomics and RNA sequencing, we explore the molecular basis of attenuation in a strain of bacteriophage T7 whose major capsid gene was engineered to carry 182 suboptimal codons. We do not detect transcriptional effects from recoding. Proteomic observations reveal that translation is halved for the recoded major capsid gene, and a more modest reduction applies to several coexpressed downstream genes. We observe no changes in protein abundances of other coexpressed genes that are encoded upstream. Viral burst size, like capsid protein abundance, is also decreased by half. Together, these observations suggest that, in this virus, reduced translation of an essential polycistronic transcript and diminished virion assembly form the molecular basis of attenuation." />
<meta name="og-title" property="og:title" content="Reduced Protein Expression in a Virus Attenuated by Codon Deoptimization" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/7/9/2957" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="A general means of viral attenuation involves the extensive recoding of synonymous codons in the viral genome. The mechanistic underpinnings of this approach remain unclear, however. Using quantitative proteomics and RNA sequencing, we explore the molecular basis of attenuation in a strain of bacteriophage T7 whose major capsid gene was engineered to carry 182 suboptimal codons. We do not detect transcriptional effects from recoding. Proteomic observations reveal that translation is halved for the recoded major capsid gene, and a more modest reduction applies to several coexpressed downstream genes. We observe no changes in protein abundances of other coexpressed genes that are encoded upstream. Viral burst size, like capsid protein abundance, is also decreased by half. Together, these observations suggest that, in this virus, reduced translation of an essential polycistronic transcript and diminished virion assembly form the molecular basis of attenuation." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9.cover-source.jpg" />
    <title>Reduced Protein Expression in a Virus Attenuated by Codon Deoptimization | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"76rKvYo4CMmj6MZLb4nEtcmQHNhOhtZmqE6hiEz9CAk","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"486347","apath":"\/ggg\/7\/9\/2957.atom","pisa":"ggg;7\/9\/2957","ac":{"\/ggg\/7\/9\/2957.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/7\/9\/2957.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;7\/9\/2957"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;7\/9\/2957"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/7\/9\/2957":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022DhAbGloVAX3NUZS5vAb6bUAsrEfSEkgT4E1Cbx4pN919yQ9QAIWe\\\\\\\/5923xJ\\\\\\\/roZgdUjiul\\\\\\\/Uq1t\\\\\\\/ow93Jz8aMa2JJ7baMY3fuuA7rZd9sDlE3JCQddJPwvCEP6PpUbtxUe9QTtoQzQ\\\\\\\/LBJnvvzRrzs5FM3VhhjepVtsPVVPMb2z1PBwQUK6awSKf6tH53ggsswg9bXWovFCEf79J+zs74wiSjMd5g7w\\\\\\\/tkq\\\\\\\/cPFvOuQPdvrV0I1aQnTiWu35y0+lulQjO9wr76UPb47C50m6Gblro\\\\\\\/gojGg+AqWJbYoLiPh5o\\\\\\\/k3rB1MU4TMxT+YcQAKZ39uxoUvYuFycbQZ7\\\\\\\/13FFW825xzc7xIPJ\\\\\\\/Kn0ny4h8bnXlg57ZT\\\\\\\/H1Cs5Rbr36JzYJmkf4Ce2d8mnrwoHlDPEWELpaAdjuRNLtyPNTSiW0WybQYAhq7WTIikfseuyJ390umJT0tsI6t+zhjL8fxzP66Plnh\\\\\\\/p5T4HFKxaD1SNp3Za1B589Eh9B\\\\\\\/ymoYX7JeKee6VsnulhiOGWKRO\\\\\\\/qm9SJwLDyIz6mTIwNaEWMUT2vn0oR\\\\\\\/DXr7y7DDgtKS2Zgb1W8uCb83M\\\\\\\/czVLGAJeVuGiRNj5JCaTQd2XckLzVpfArjMdR6RlDufvBH4thEQucKYkHeN0YrMV1P3ieEOqbazfdRcOEhxBNMVYRlU+jqUbF4WYj8wzAqVxSVds3jUrx2nEllW\\\\\\\/lyEjLYiVdvDX4OswG1FRt1ZUDD3CdSFxF+R9I+wH6r2ZUVrHAYWBMajbIN\\\\\\\/Q3T1l045PvqoUNJd+d2Z6z7Ayo1pDbEJfaXwl\\\\\\\/nl3tsHlYnsm9VzvIghSux9nD6X8G8J6FmSPq0EB9\\\\\\\/e5x6+fErIaThm87FdvNda95\\\\\\\/m6mqtylt8nAxwupsigoYQq++iyg8TWGawcRsG2vSJpESGpwUtGEeQAYvAjb\\\\\\\/0O7Vqyy2+BT07ltcoPCYLgBimf+9pW4\\\\\\\/feCc9eRlmtoyG29UV1kETSKtYZnpg\\\\\\\/JEXW8X9Xjrp37Uy2nFVn72Tvz29ojKuLGtS\\\\\\\/4grI37lcE1KPbEYewDnsIb2M++CzFeANJjX6urP69Y9jyQ2\\\\\\\/mkJoxxsU7xx+TvWJ5G1J6TfXnX1AaF9Y1qL93Vckt9M0D9Fk\\\\\\\/WwprKNn3CunjW530W82\\\\\\\/UMYsTInyxork6iHlpgvmcFuG\\\\\\\/b8WaOUpZpzvHO2dhjV7Zc9uyYSdCfUnVaKluoudhhxe3jWqk4ewyu+IWabW\\\\\\\/r+iFhcKuMd2smyQOeGAxVtQWMvNxeJnf1AVRJG7IFB+\\\\\\\/hYRe\\\u0022,\\\u0022iv\\\u0022:\\\u0022jXpBI4WdidFrnhzOqguNpA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022cb795b432850030c793c324178ea26df54ba2e52ff4b0a43a585a847e69d62ea\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022TH6hzMfwuDr\\\\\\\/KnZM+hEjMMIk2Wy2BP\\\\\\\/Apy9Zc9SF0xqVJNnn67WxG\\\\\\\/Fmjr1g3ajJlAMIHlgaC4g+1NgMGgSUrRZ6mrVZw67snv1sg8i\\\\\\\/xFAVPsYJH8wXuUCl33fXRF2ZQ+0S14IgEZWsV8CmAAJs6jhWhb57k9Is2erih2wx06rUABy9kO5lz44YNEUYn6iDtcGTuRgaPXfpFsCicVbgYWrqk55CPvyBYlh\\\\\\\/ED+\\\\\\\/JPNoHtfAP3GjB\\\\\\\/RFdBcsLnlWOXVnPoohCnM48RVzRLfRrerxsAgy\\\\\\\/f1JaTY\\\\\\\/xIb\\\\\\\/VRHW7uRHyeMRjQJqIUL5fdw2OCT7jD57FFPgQ6JZCXQuTCGl7UZqj+McgHIvU03NimnWXx8rowbG+w7JEgpiMMHSDF0JKd8D7aJQ7VvwhsberViyLLsZvRxgJ92TN50jpMgUxPVxzrrdvcTyGjfHNX5n31OqMxBnKtqBVlGxd4tWY41anm4pM3AWOXNpH5ymuveE7e708j1i7u6iDrIL2s8VhcVM1t6sjYPml2kOi4bH8zqZVyCAYTFXUxvxQVvoOmJtqFkeuf98YzguQ0fFat9mcthMec+Q0CVqT0X50QzuojquUlmSWxm43RkeH\\\\\\\/NJUQ4qu2HhNYcFIKJA0jIjNyQAFfi3QmDBp\\\\\\\/tqb4Fh5PP\\\\\\\/f7Y7IdPFuBnvUcLgyN+ebJLLQSiZmxL9cskDOfbS1lPVLdIs5MhosCD1pbkxt7icmag3zExcVble2CL8qc7oMzQC20xOCS5fbvfix2Zlp\\\\\\\/eW2u+KjjOcMeSzIIc9PBwNJR8QcWR4adBMsAm6nTv73B62v9yK69y2vc9MKWnh9a76sLeRLgVYfUnuB9O4AcTjWOq+vpK\\\\\\\/TfPzHL\\\\\\\/YbKyQkhAqL28PjDIWF+rbktB\\\\\\\/COG2r6xjIiKIOE1XmdPFRMHDZERq+242SpORQZSR8ETRlQG9GZWKV+jENRCzVGSEQAmXnd9HKdVAii7EQU6GElRO5lKPsjs3ur0J\\\\\\\/AFVvqJlG4R3QTdvCIhLhDdevqVYF8o86CSDCG2XzpjB4myadyj6Tuju2Ce0d1NmxVt6QEXIN9dJyUcg3KNGbEQgy+x+YX2Iry0c+8wE4BtzEc7P6+KgA05UIWo0HLtXb0nU\\\\\\\/aVnL5vT8Emh5RmioGiM8AsmPo32UWg388ZQYeLuiUj7G5Ti25\\\\\\\/b42gZOjvq9rsOh45PtHhq9YZkghNzNAwh3D3jv49QZ\\\\\\\/WkehSdCLhBDIGZ9el2kimMtWN3aMK9cuSd1ayFoL2XNAu4USLATSCtq3mNVPewmZvYEJEeFvbRvsyUQ0GZlyj0ZI7FHGaJkxS4nKc7hhr17SJdB3b5T5BXxnP+YwYpVtT\\\\\\\/9Ykj2Zp9fgZ1BJdwK09xwHzsZG8+BrY9g3fIZEYBAGJuM9icJflmaj\\\\\\\/EmOdkYXF\\\\\\\/MfXIjDPMS8K9numuJ4X1wsIcitIGbVj+3OneJi\\\\\\\/0Y9KdEw5MQYbDBXY8sJMhq\\\\\\\/gXT4sGhdDNIkpCkZMGMglL90eKHu5JA2IG+rneRgYy7hkisVd\\\\\\\/ITSIGLy6gOKWFCAN+BIqe2XmJBgf6xhE\\\\\\\/3qtFjxhF25NrOdPPyPdEoVplCtWBk6lr\\\\\\\/Nn\\\\\\\/NaGIn9zJsxXOjBDhUoUDndFt8lm\\\\\\\/jGD4FXS4fHALDf+VEF2aHMYiBdmsz0OrxbaeI9rWf0D4h+KMebRjqIcWc+zPGO6DH+ckXLYFUe7kxSqd2mV8A8nJvx177ze7GE3TRqnnGNt+oihLEtzghwyHQzpsbc2KtS7mAbUOcqndUaqfi40U3VVMYw9c5oPyDcuWrdlfcS3FFpzvejX0NDlCtilkU2oxmu\\\\\\\/YEEFZ23hWLXuHxzKSP1D4dDCXHerdGvhQGQ5C\\\\\\\/KkdJpdKRVbBWwv+PkUjj3gPtt8lrzDkjlIA7qahg\\\\\\\/GDrtU4wL4VOwQUR7ObHET1HILuhyVQkDLim7dM8G6PVWBjrCsO0Z5nUWvc5fIuISPRXuIgYU9Om78Y0MAv4yG39WBi7leDNIQ2Z8Jh9SMPnFt27FXyimQsnVBQqe3E98ocU3KAOharFgG8KM2D3jQbCrXwzw39U2cCBSYCaK2vQL3rWQ\\\\\\\/p59W9VnZcERF0ZH6QDAfACkZ9wytLO9nMqqtkA468dco\\\\\\\/AdY2EuJ\\\\\\\/1s2nipY6qjBWM+aMeM+gEfZHUw6JwCvR7aTutUqWcdZJU+V9thU1jBS+DTjnBVV90VxBTSteGyEjsoIJ0D1DuhyDXKU5jObXwU\\\\\\\/1H+wpIktUvf2GZjOOIIOyte0FE\\\\\\\/9g9qcJ9Q6CVtDquu\\\\\\\/MxrfIr+JsgXmwUYmlc5FfNtaPEOkrSE0UouFiT8VF7nzjIi5OylkP\\\\\\\/0Tr8F\\\\\\\/j4gaBzuVmP3D+QRI34DuqwjfHYxas8DpPTNfef\\\\\\\/eiaF0G8EjnPJPaSW93CepQ2ONeR76AO5HhIw8N4ivsHVUFFYQ42cijCnQ7z12cfAXt7Dqg6gJ2\\\\\\\/bNTJRrbV2ILEgPDgMbh5y1cpq+snGXH9b0XzUBSI8QuO57HOieieIkVijxs8NO2f4z7\\\\\\\/UzrcbM38No2BqFHXbZsRCqhmacpUW9xJ0O9cTI2xjDWik1su58ypuA+FOeW7yNsSUjVAJxAqzcCanV3P8SQj5elSTaRZp1\\\\\\\/6kBZfJsGTKCGgZKbKrsoTKJWZKX6y8DFU7q966r09hJtXkUPl+phk3Jj8iFU2277f03PDEVyjF7aGpi1Tn2u8LaL7v9pIc3mgSsJxiGDRaYNTVaiAoBCquD8ubsxfe5mvOyZJ81nP4X6Gi+6CgGFCqmNfwi+NhsvT6EuUtJmxeD5jmH3DM7r7UZA\\\\\\\/CkrnKch\\\\\\\/LO18uCG\\\\\\\/I2GcBmzxjYaQBk72JPbgJYZtFgDzc5JvjCwfv2nd6x83zqQdPSACENxNg\\\\\\\/kjHuZKomXneSTSi2QBzAKdL6LN5\\\\\\\/vbYIsAzWKY1GaQY5VDuEEAd6+6c1lek5Bdh3rneD+KbKGfgMSCfTMkVikR\\\\\\\/vN9brQiL57GuJSS5rtBWhFiKfFMMZOmGX8qSad1WujN4BltdWMF5tfH5iU40C\\\\\\\/C0jkOVjL+JgjNj0iftpRWT9G0J07t6KNmWOIQUK3hZPan3Bnh25mWEdAroqnY6IK0HKxf4rpth4Q0YRwTyr6b8SKAr5pHq8gYW7GRUXgV95e5BZ4mmQAxDscceGkHGaEAg\\\\\\\/6S7vNWdZUmS7uQI7fQMP6LYX9ZaHnwudyA53B\\\\\\\/w3qyqyDP76FqJyuujRBRyWJvPdKAEu7R+HXQF9Vg1gb1HI\\\\\\\/zwLieJMx1k2qc9Cfk2+HZVT9mKEe87ZPUoVePGiVMnKUT8bc71HuH4bpDSnlRcsuk89Pws3qkOubtDcpWNkAYG0pjE4zEkmYabxhA8trB9zrrwlsqZgWvfBTtzeXvtaNCjKMzesj5v3gPrnzrdltBCMJDQkNW15I97HDzfJcbCnC2UjQA3z75sQ0dwFaWGHMu5UGoIrGGA2S\\\\\\\/LkFOSvmoW35lPf6A30coNcoFfbLgRqZaDa1xqpSyU\\\\\\\/+naoShm\\\\\\\/hJB38JAxMrEXMiisGxasLkBzVK24tEPFl9yH03eHgBoeDpmdp43AjTEEJIg+nDfyi7C0OsMT32AEclIZMeNNNTkVhgBhVc5gkeZJnqsawnVJBsNsNvndh+pAlsSKgfEAqdpFfO+UJGRfXe7\\\\\\\/7cRQTRNZhRqvw58k\\\\\\\/XjvVWIgrqi\\\\\\\/FTm93d6X\\\\\\\/4o7+\\\\\\\/a4dlR8gaGEW6tVcsjbqmaR0zkQ6ENh1Zde74hQP1PhweyjSJeqCN+AQb1n7dSzC2wKQDebcP3+71iLHWlrqJ1wgdkiPYh5ZIoRi7izUwe4A4SdI2kX3rcM1PVjZH+Zf7db0m\\\\\\\/TrssLfjeezEzod0w7IQ8IRWQjFi8a8e\\\\\\\/EmvTDzQ+FbbJgivQYk9QAsoW226OauakpyclZ22T+iYO0vuaTIGA6p021NWzJrRvtoI\\\\\\\/2gMgU\\\\\\\/5gou0qiAffNbf1ERk+5SCPMqsoJBBuVZoiD7qXbkDXnS88hb5sxk6KbJu8+X1tgiJtXXJw926NeTWILiT8+FyiiddcKRQjwlBVgq6jAspK8EnHma2dqjhyLnhvlcMq0C\\\\\\\/ZBhE134LO27gwqVXdgr0uKYMWBLi4p3LCSx+nHYt62KtvdntTq+YzRwMtgutP8B9nOjDWP9lmxse\\\\\\\/esM\\\\\\\/HGv4ber8CN1ApFlFrrYTarsoe0ez9ReMwgGD\\\\\\\/QHI+FD59FlrJLzelj48hzJ0Q\\\\\\\/YxOiBQ4WnrIK8\\\\\\\/1hyLTUWG7vH82rdbi3Ocz94WP0P1gD7A0\\\\\\\/b7RrCU\\\\\\\/SzPNU0\\\\\\\/r1ZXwl\\\\\\\/dmJuM6CZsBV9ftyi7jAWClXzzueY7mpsrh1LhzSguBbP0x7qsGFdifcyt7MDW\\\\\\\/JI4RiRa\\\\\\\/kAFXewiM9mExKUM\\\\\\\/BES+nFCFOYkK8L0uFQxoF4\\\\\\\/LC7GG\\\\\\\/1JuYQiAOg107beBku5yNMmabnQ2ulvCSFwVo2wMxkVfYKKvxZ5C1nk03\\\\\\\/UOJ0lLj\\\\\\\/YdlxycBSEDsa5LVD\\\\\\\/qox\\\\\\\/2d2u4+ed9QFpaOmdUv4yLNKSxJf9VK8kYc981PaNZ8nyRD3q\\\\\\\/7VPsbpC+Sh3BdLoYHMxyiy0MLQfXoxV774dDE4URYbwkQtxEDQxD7BmI077qdeml9TUTRqXv4G0Kv9\\\\\\\/KoQGgM3qt\\\\\\\/V7ClwPSuOGZ6vvXPJjWWYbqzq+KF89okfsiYuSuq2hWDyCuqIL1Ky839oI8mxbkzwgE+AGbqfFoxIA36JmptsaWhPY6Kv8eSVpN17XleegfRg2hbLiaL\\\\\\\/OpwJN0tkfEcYGlsPuxApAuMTRn8dPk27eHLWr6RHm\\\\\\\/1xwXBIPrI+1K1INdyoxz3eevpqgEEwjU6yG+h2r\\\\\\\/jOefLDV8EQRunqjH+cs2PezouF3Li3hnHfvedZzwPOdqY5BmWgN60nYDXPild2IjywfpYXflnwzhq00tZ76ehIU70upqaqVEYjOe1m8isTcOm+0t+b45FjTbp1X2SKoIXQtrFpVgy\\\\\\\/S5VoHrd5ZH5ManLB56wBQbBcJX2aEOyMjlQd\\\\\\\/npGjuNVZcGCVW6i4gSrLhp655sF3g1LWsYlJ4+vLvPiSGCuAd2NzL9ZsAhjWphzmwnxF7o0AZq2zstX3J8seZRjSX9NpYIgqjxSBnqlo3ANUDqi3DhtWyxS7n8Sb+OrsnR\\\\\\\/DarrRJk88dwiDe+iHCy2wAfkA6MWcd8HsVU8aGSK0CXUtPQkKvGrX0Feo7KdJFynpZYDZxt5B\\\\\\\/2rmumRU71XJfgRzNFJxdFq3YKg\\\\\\\/r7qC0gcgABVN1bEczbC\\\\\\\/BYSWvE2iRuzE5fW\\\\\\\/TFXI\\\\\\\/slyp46JMpUqdT\\\\\\\/\\\\\\\/UrbrDcoGUfUjle1xgjJKfBulBoWsFbOhP6D43EKJ\\\\\\\/svdOslwOVhPKfmzlAJD2RSlVIvIpNfgZfXasmW3YeI3Arh24oUK81n\\\\\\\/AvjktCwP8+fFgL93tQLsZXMqUmauR2qYI8YVs1APo03vuaRoNzjq\\\\\\\/BPEKU39qmStKYBk9dMwoE5NE2OugasXmusHZRworZjXUvHKoH7fc9ufqWnKlwG4\\\\\\\/3iST7MoXPOTAl5rjOrlEoy2v\\\\\\\/dkxP+kp6C+gKF8nFpNNCc4eP3ewCWJmUAdB3bNAmlcmvTQdDHi7XlbFEr+nkby49\\\\\\\/4APCw+XBKHyEVB0G1zPks2qqyojFrpdEfkcRetlx3I\\\\\\\/AcwyYnObd+liIrAlEFxkmHgfaFopsEya1RDUfShcQWPpbgK+1cVgO7i4VGRTZvo1ZH8OAYmjjsVwU6NL0vO+lnv7MmvX7eoTo\\\\\\\/+EVGOXXIDMGhgxJsxj7Uf3e7MYq\\\\\\\/3f5aXr\\\\\\\/yyYPcpFYMzHG6nO5FNii0BQ95mduEvG4pkE2b5JPVKHHaSO5+0hBW505hs1i5uHbp+jhfaaDDimKrYw8NJSQ7cK8OD45qO5ISUuMgghJN8LFwcgUg\\\\\\\/DoxV8H6pxV3aBKxVO0W8lu8mJhqoGxNIS9FORsHYOrQTo8BkZufzyM6EirJatlMWfEbqa8KHXUNU7ccR391gbQVBY90KfxTQKu7Bm\\\\\\\/ImBmf418fTcQFv74RdWM1GF4jMXLGVNWO82YyoQzUFMJcCccpxfk6LFjNuhdYCEcCZtwMbh6Q\\\\\\\/SuEP4upEfPlgyzG11JWClyoXdUhHqnPr4loavlbOvWxA1QyfOAtPBSOkRqkwshVgm8FKXVWac32Xns0\\\\\\\/KQn\\\\\\\/+iZ6zg3yVqWUob8sJW5GVkx9gbfw2NANbPJk2Qyago3vLJRGjrkbIN9FgQ24WHEWSsHBYz2ixHAgrdNXUe9b1op5pTz0fz4z9A+XSD+tj8iHMKSBpWUTbfxr7Es6rAMpq3eNyVAcpuYHb6PlhQ\\\\\\\/yjvuSssG9Mst3+QVJE383+to\\\\\\\/aVn9evY9GPETOz2O5RfM+Oxm9xULj2BpyS8G5UXd1xE94ucrUcyFIqpssLpQpck0SilLnhDvBKL\\\\\\\/6bFFW4ZATI6qkjgM\\\\\\\/5jPMt9+YTBv2hAmc4bEiNoFeVxauSsMBs3rBb6YZCGnD8mv5MwSl7BVM0nYHsOhUyE\\\\\\\/y6Bup0fCLM8+OiFprCTqQbH5FxJZqHueu6HTZTAOKgcHKI5cOcuLai5K9YrhasDdd7GzCCIjrtOxScSLYMYktiWGL6bLFZwnZAabZHVuJNexEkdhE28wILFIEoFa6hdOgWuxXTTokhT1ZjK\\\\\\\/KlYpWkdrMy+555VdAcU3EszvvqEU5tmWgTq5eKbHOr5RFvhnE1gToVOFUoewR+JjevDPf\\\\\\\/szxO2gM2xpLu6JjxJBdcbf\\\\\\\/fdDtg\\\\\\\/DJLdGHz3GINmxPplDFyTdBhXv88c7lx5T2zunCF1WYrM9jOx+PvZYabFDKF+7HY2PQWXc2QWN\\\\\\\/LwlPD5DGQ3QMMrOPVfTFHjKUwqR4\\\\\\\/n6oDyKfkgSax5PDh\\\\\\\/7z7YgeF1YkYu9pEhCtQ+l+wiouN8ulgxb7\\\\\\\/tFOu7kB0Udf+YNmMoPP6VIFNzZePJFcE38hFjX6Dgpu35A2NSzFYC6GZe74F0a0q9Hxp9wGGkT3EwlQ+rzbQvMc4Pg1BTy2NKyUyeENISSxFR1M0tks6cwD8GYvnmUrXN8\\\\\\\/HXBGEirzRgbKSpZg5mivhCHaaUrIwxVVTBWSrzLqLamWmmYcx8JbAOXBaUYxJV4x8+sLWgJ5Gh0YOj0+LBnflmoSGPmxtPmnHHHLPdrVZ85UAjry1kl\\\\\\\/3z2iJfPELRByLwEhq83KtTT9SreSg1V2e0+7W6jUz0uLZzwbQNnJw2RXcAnLXolvts+s08x1r6DN4QDICTnxqMwqjfEagWr0H4sjhHKrroRq2iZefiBWPvt7yYggZHezPaEhhCyRa5O4XI2b4M5UodljAI8DalJm0NZO0yqJDMpy1ypgpqPxEyJQlSZEc1j0ES3jzbrHKpe0aa9Qg78O0R\\\\\\\/fuwdNhmxK2bNiioA9JwH0As9bgonE+ZvpRnLuF7j8cMk\\\\\\\/VxYtmw9RPBLjLzdmD0H6QYvKGhWfd8SIwHOLpjrwV6rHZN1PBDZwt3r9+7PZTm9+swKUU06hwVjOCiswqYcP9MSIFRtE\\\\\\\/Ym6t7GtvZU7KiFbNZ2r4sfNFsgbHGltgHRXgp71mpmiyHv8vI91c+IU9mth\\\\\\\/7J8V6LgMmT9XSsqpUWgDbI48ylpbz4vkB87L\\\\\\\/hN3ZKPrQtJEZprhGeuxiR4cRovd7IbgX0RHtoGzTmPKp1iTBib9lsKa9dCStCsxUvFB2\\\\\\\/NUGtlmHCjkpbo8YR4wRFZNX3FUIXb7XCtH1CrIYZ6f+c1I1OaXUgk7GoJ3UF0Z9wVnpuyEhDx5nizxfBKDPObXX3L4XwtNLBwxMUnXskBEaGMYd8N2urnR5dF8\\\\\\\/LCeOqtQV2jyjJ+tMNNbaTA8xAgPIa3zL+STZd4hx2w6lZbZR8poHd9Qj20HRfg1Vlj8jtMP2s+t1O\\\\\\\/YEKlACIZSTLaTsvXUq7vUhOEMIEthulIS7PI9FIZsf4TUCAhO5pE64ZgN7aLsMkDVvRfRETPuR8yuTXpKPHX1pMonYnESENqoER0fhUB\\\\\\\/JBd7i8PfdLAW1lMWs1VmCOP3cMALCw73qE1MBLqxwDHrHSIit21HMCfQNytWP1UfrIagLYxcFYQlO\\\\\\\/2zlHoimI1\\\\\\\/wfZ3hmkj2EDPcraFvLfBkVKS8KRWKjcxXCfQwhFsygzmoVRitxuUN7YqSxsK5mEke1cVg5ysX8h\\\\\\\/vZ4EXrhwAoSelWeq\\\\\\\/gv5CPFD4wwhcGp5MNz+7dlHuR0U77YD4r7nB8jEAEobeu8cEnhqoxPCajTXwkOuo1rkZKzENMWnsL05vR5kEBJWM9EUBjowB+iZ+CX3Rv8dDL\\\\\\\/XHFmzZ7zxVRjrnXv0Rno3EYXeERg7kGgrATazk392amjFmMJWjwvoKh+yUMSVefmEUmHXp5DDV+synxRHlc5WseXyYPTPjHfJEubeNIVcuILIfD7QbRDF2hGmN9YJbAXN+PNtQmA\\\\\\\/SDIDdvmFhCnHnJeYEbsq4L8fZXLlHjvfRIp0os9NtnV9Jd6CruKcLeUh2re77hZ4R\\\\\\\/KD6i5DpTWKclxpWsFXL\\\\\\\/YFeD0VW7nZfrtnc2Y7gM+tJUjoangJFbyMgRF7wM7\\\\\\\/NTqAwQYFncmhK8fm3keW016\\\\\\\/svaBquNPFEnC\\\\\\\/g8B9zadXIs4Us5zhJLa1t+SXSsBi9PMW0vaQdtmxjK1\\\\\\\/hSjuBavFDrAdHfWSCMaGbzOa9dhU29GRr7Bxq3gLLOoFgdfvve3GYB9FCn9cVicQWd9Q3Z70sOyKDWWnQu8yIi3NVbVFNWYUA1L0KLnzv+7WWNPaywwpr1894RzgkVUykzcexpeKNYs011vZBt98LXMTAPWq\\\\\\\/rdt7Rx+Fu9scaudw1mkqGjQEpm8u\\\\\\\/tHsF12QrBee1JfMuBwaYkegxKJpbDtaAg+LN8bQoTgymjWtIlCH9e9UUAl+65vhNwSZGhpfz8qbA0i\\\\\\\/nPFS0hsJ3SYSTAJoi5sk7LjtBpDeyKcn2Of3i3BeuaEoROIOTw70fmLvW2EiBAm1ca0QZAEfESIHnfg3WxYr\\\\\\\/W7dXuU1KTYySYiOxjYGLhNQhdATlXQSkKB3nsR20WzVSHIX3n7eMzl2jZFU7s2BK0OHB7AN59RVrPX\\\\\\\/lUPb4NPkFPprPAVS0d52WDQM9n3Nvx9LKg8cFOpbYF6q0yOxRhUgnq8BObR1aEau+zYXONv9XBvO36Y9WZNJbZJbRbx\\\\\\\/6iMKyEtKNg1tJu7XOg79m86o+BX5rqLIFjDmRNBpjqxP6mFJMgwRLiW\\\\\\\/Kx9mdmKthj1uEYuBJWEENK5yODBEoZpG8OdauEJDO3UJXvxM0CPYzJHo2nnskl06jSkSnUvSezpnlFDJcFQXCNtkvWWkwYS5FvNctFi0DZ6gu8dzH6n0QYp\\\\\\\/opW3+x4W6W0y8XAjYD\\\\\\\/IkLqI1FsaqbINFSDFPy3COEgfhoC1mq6hr7RKelG0yBRNFNVgGQzXsl0R4iaMxMBUG5Qd7JmNnCNj1Fc7SNc2KgzmpvWR8DxzfnGmlRHe11Jn06XYeWTn4wlu81OXSHbI0EB\\\\\\\/BRe3AdW\\\\\\\/mgRol06XuvVYBvZDQtB+KaeA9LaS07CdzvssS+YtzIbRHGJBsc83LaKuxlVsytGHuN6zBoSW4gnCd1w\\\\\\\/qoYsYJ51w\\\\\\\/0\\\\\\\/OyhQMrnTpobMrxYvp4PnIhh+vBTh3PUEuitz0m9cf2Mp95XupcqHNqCRA0yCEDfM\\\\\\\/T5imqR3h0pfO9SFvQsliFh8sWTshsn1y1OiOcXzwrWSHlw977NOOwnCa0vkU8eIfe9QzjPKKMJ3s\\\\\\\/fIvuTEH5\\\\\\\/L4NTlmOO\\\\\\\/3mSG\\\\\\\/aKdP7XWxZrGO+kpAmh8laTwNz1\\\\\\\/NtWzFamAaLfQpxCLblS3KSk3MvtLoQdqtzyj0WKUqbfDBQ3uitaEgQy7Y1Z9cRUxovASBFM+8lk9vSEYjPEraUZfwYfuRpb7FRy5h8gcWc+66NN+o8bHoCx7C\\\\\\\/gqoYoV4X2kf9qZJxinlW+S8Q8JnHa6cMXJnTWPQPHul5+XTgWhZThsv+zxeybwyryT5q+cfRIcq6y\\\\\\\/FPfXVNm7d5kak5Ry\\\\\\\/u4AhcXu2xta3RgkQhjh2rqM6kuzvZ6kutsgz+zbKgT1qPVV+YTdO\\\\\\\/HePFb+OXmWj3K+ZTkwOrzYu0leaOHHo+Hegr+a9+t662klnOgJvLh4u5MRAehJ8PAm+S5gAId4ymfsVdv9AdW+hmoMZuluYCD9tKJo0g0SJ9fvf2woCON\\\\\\\/8+fmq0rIGF0Ocbz3eVzk0GashUaskS8gSxQ58ZFpkF\\\\\\\/FDd4tutnm83laK8PWF8v\\\\\\\/BZdVok0nz2WvdP3Mc1IS+SYSWv86Rg0KRwY8qnL6Bzkhurr0CNSRd6Mis+MySYp3sEnPsO6z5YJR4yO81ZAeqEb1HIT6QSttVrh5Ocl9e\\\\\\\/42hoyLx7NQKoD\\\\\\\/iKkaXmg6h51ehueZ9hNwCF+OtS\\\\\\\/ik4gZKTeVPaleImoMhrOzSsu8rRyiFoO+B+7j2ta2msTRcUWIfdmiU\\\\\\\/U0vGFRgiqSdRUJOMICYi7lXyEUimDcaNSzyZIdUAaWw8O8ndDDTMo3FIFtPNo1hydJjDMIk6FaCLkO4JCOE7hI0Xa1j1Idg4\\\\\\\/ld9imWP8mq36BdwBZPDtGPG0Xs559Cdrhr0pBpFIRk8sjEZSM2Vha+0bjMiHBEtpiCQMGQNK4kdizk+UGELQ8KOAasq4rAgRqOZsrvNEyIkX5gMfEgy+8ofbSLuBEYOdr32543gLut1+Zfc+eSx5ENUEJDVL9zoNeehY9baqDsQoxVrZGp6YwSRHU5uzBSiTrVUbV9R6e7B2+euT8sLq0Wgu3I6qQf3w3XmmwkrcE4rAdJQgcNW0z2AHGLMaQ91yNhcTIGHNc19g3spjORv9d11WlJTupqMgUBp5xqTW7gZEglRb2U0\\\\\\\/FYVEWxoaclRt6QG11dmdtHRozbCnciTzGdq4ptWuvaFSoBtQdL0QvP14hlutpsgjwLw8zCdXF6u0pf95B6nWJ4VgQ8TsYaDVL7eqdVuERQgyqKPI2gTqwsj61Cvbjx2fxn+vKRw2rzIa1Nve5Iho9Mfkp03j9+ql2CEDRvUjNsG3WWEFr7opAXb4KU\\\\\\\/8UPdjAkEy9fJm2ZZLculHEbrhrCqThsGJhxTjWpo99ewTAEfEwqGomWFFd0iyZBS1CtONiGdfE3AEpggPB\\\\\\\/eF\\\\\\\/jj1IjC7lhoR4WYjWBRyxcoOgE7en176Ucjq2Hiug+rHN3n2vRzb5ORVCD31ZcatEY\\\\\\\/2B0C+KUQf8AEEVC9qRoIVYmT1pZc3Gb+vEqu4cRUAJp4yTau2XPfKU6WJzxJmZ4UvyhNai0gqeUx7iM+JZ9bAbfGUjcLYkU0aSoEEAEh72+AoCvcFEvRbmOSOKKgxIAKXMnjUTMx\\\\\\\/9pHX+EwQIUaKNHPPEaf35wVtAtjnlGkRKnVQxgcpxRcmLDhTJsoKYRp2yoJCB6AFb3PyeH85FUxtYkjRlPtG9xno3scq2ff9Mzh5FLoOifydfdx5Ljvh0OBBadaWNU3Bn0J0wM566dOqGdhivAlYO9QFA+jL4rzf4MGrOajLgkC6+GhP9V\\\\\\\/v2l93f3C\\\\\\\/h21VL4kxILqrAYlVVKQzLVaXCMqXGFLTbRKZcE0fYwwSetbExv6+A2E6kr84QG6IfKjxTbW9PjNnF3yIxX8brYUisb1ucYnnCR9aVZQH2t5IAaC3GIbilbVpJi+Z1OQoIhvdzDysNI7guV4Vs\\\\\\\/VKIZU\\\\\\\/aHeNXSgHDyedi4HuYAN34tVwsv1I4J7m3NNP1RwsDJk4mezLTbipQ4RdV8fi9G5Dyte9HbvwS1XbQjM2GkmiN\\\\\\\/d4ZXejEi4Z7YsMZCBdazdkx\\\\\\\/qXKBnAnQA9O0TY0CsQEVJKANUGsp3wzS7eE8KelZflwqq9IH3E0a9MR3JwkGp0x4ubCfTHYkr095S0ja97cNV4gx9+XhLn95NDYvEKG6rnY6a2XJ+vnNGfWpssTjpuZ3bc3ZpZ+kKdYRlJB4cPvjpD3x3TRhVfSzR0VZtBV1X17SDXJgJg5AtYHoUHfTHLlBnSTXJse87fYoPJfd1Xt2CEfin3rdUF5Mr3q\\\\\\\/9RYpP2zaPd1MUB+\\\\\\\/aj0NHGMX0W+yfi8X3aH9PqA1+C0BXDTxUbWI+K7ltcRjjWCRvwBcI\\\\\\\/yWWJ9pozI\\\\\\\/Ct1tMVirkEhVOTn+5rWpixXJlxAZIqMUoN1lQzdEradUX0QomGriGa5ehqgtyIXI\\\\\\\/LIfPRJFcj\\\\\\\/jEtulyRv0r31PChIEfW\\\\\\\/onAizMmP58vQaaGvAZxKBOYFZXrH4VUZOxKIyRoiwI69PF2N0E3EFbVawuhTdF+NBXvH+SkWh6kDpduEnHzd3WPmkmXV5vWX7SiT3N5VRzNJzqKHCf\\\\\\\/ws5o0aLzg5tg19GwsjJb5+tUYwNGnrtOtB8ZkbcoCoCE6Agk+FZA8GuSdrzp\\\\\\\/b4Usui7UT0NIINSndCLSwowC+moka\\\\\\\/lMhQn5CgcvqMenlI2F3TGZq5R9D7vzirgL8Uqx9\\\\\\\/5JnMXnqr8C7SkhtvxKWUOFrtzfeIqC+9BW3B4vb1Ah9SJwX1JcEW9voeUuXDbMuzI8Gpmfe6YgolO0sOOy7iNaSxcuZf5affoqOMgJStS9WmYUSQoWFN\\\\\\\/0\\\\\\\/EBC09e8olXIuN16pDUoEAeMJhW8uRyLdd0tFmfCpmEz+rdtU0pia4h2ujKkrDpW0rZjDHj5UYvzsvAxCMmwrNzwmkSlg+sb1qfG\\\\\\\/pyPMJqiHV9yxpZHvRHR8\\\\\\\/guP5u2xkJQFqCUO6Hlvw0JjN2G847np+O9BZTpXDbHUD75SYZiITEl0Fnp4HZlUwA3tDWBFWvzfNbF9B3WiX64bQtHvzv6xGGQ+RAZzmhyZ\\\\\\\/QC\\\\\\\/oHTTcu1iQoAMFO5l8dAGX0WGJZQ\\\\\\\/\\\\\\\/XbenHy5pj8ZNxPuwn\\\\\\\/mI56+PqIPa09GsQB4yGMOKewHDj2pHOeLU5RkB9YNa3K4cQWUQ7MLrc\\\\\\\/8btlZYgrarX3WFo\\\\\\\/mnSphr7ZG4ljt4VFzQtcU6XXsz+J9D2PxKwSaV0ayWdvRU6J7Jj93XYBaWZ0Vzfa9wCQuICHMx\\\\\\\/wNAHLsfqPNFWBblTseiGxMq32VPiyoUav0a8+UAMOsCuIr6OWGkvOP1riRdke3tq\\\\\\\/a5Cw7okP+uW+OWhRfF70g0Jww3cLJOJbqLyvvtPVNjoDXoeAckbRTyHzL+U1cxxH63xSuii27Dbdf5QNtxkDE9lK761V+I\\\\\\\/bSKCfsB+AD0FIanRuI+WdTSUmOunVA4f1qnrn1qm1a\\\\\\\/JyKWWx66EQMftuTCQJ\\\\\\\/wET1PFY2bs92aKNqnHk549e3v72AXMVGu2TcEmoKuZxRrKe\\\\\\\/P4WWvN8lRmBVel2nmYPCZWxHMRuRAMBeaCQawRJrDeW8vXqQtGmNNc5F3L4\\\\\\\/BOd9m4kyr+ZiMMhMXbMlUBkWJCUtnHzIQiOGlX41svX75UJqEqGDJgidv15\\\\\\\/oqTono1h0Zu+AX38NGZP+mJl3jdYTVrN2LA0xYkMsWMygCtus0+t0CBaFuPE3Fj2s0rFVhPPNtNFiFWvZF8GF\\\\\\\/PZr\\\\\\\/tb+XWBBtezn0fXNPd5KsEnlhFSigiRsvTdv7grSRYSBYmm+PIsd+fXRlxYMMuqsAroj9qXmzGrAerlqmJcmmZaq+7ha6aMmm5qbNkZlTNs7NjHD370UhIuGJMXWIrZWCNywq90kyQZn0SvMuGXJBzI2FJWPinnnXUbijvb+mVo60eACzw0xIu00dTltt6ngc6AiV45xzUIybWmknr55CaIkBm7mo2pDK3HATA6JE8MWs7oMVEdkOxSN\\\\\\\/F+hP+TsbThREZGSOWwi\\\\\\\/X4uuyTBj1ve9eARGjB3v6Uej+rA0bIuFwSJEGsGW\\\\\\\/JRtHenVk9hBDes8s6EbahmWOZLW3jIj3FPvV6UHcq8J0lK\\\\\\\/wPgA9pNyUGHZ2nAaTxyDTUqD0uxMGPr6B++JFvX5JspAf+3600VLdh9xr2rwziD8TYvGjPH8J5Z+IfHrfBoq67bb7Nmsi4s5LqqESUDsCuguA7rG3Q+qIgYi\\\\\\\/tZ\\\\\\\/iLL3e+wdbAuv0Sabr\\\\\\\/x33UYiSn6vTcdYv\\\\\\\/b\\\\\\\/t8z5HYfyy7EBvPHeKuMYvioDYfCM6imdZpWQF0L9xOi78QM3n52cDQ7uue3TFSZVOhhLexQX7v3iPl\\\\\\\/Yc0aT3TuBvoLVjpM73e1J1iIa9GnC8ZcJp5U7wEb6cJJ1\\\\\\\/vwI7rGA31Nmob7ghkXO9iqY1ksYWwIR13mEyIOXXEDmPvogZs+dWw50DwyAOzT3n16kkicNyN45A44aO8dgkubmdsXDb\\\\\\\/\\\\\\\/\\\\\\\/Vg50\\\\\\\/b7\\\\\\\/SbKsiKClzN1c1StITLorWs9O+TGD3g8hiKJf7pn2k+MNMqg\\\\\\\/iD\\\\\\\/yuq1nb5mUN88+9cq53GJgX0vFGui6vYzRGUEoB2zh9boJ7uN4CJ8jn+4\\\\\\\/QuCt4YVxjiSzQ8Mqk8ICvvIHod57EZ7n9iurGIm+jQ6N5+QXevV+Xcc5HOzzLzvZO9UVqh6QgSaTcIYrdDk6QS3qVWzbYmEkW+IdUrA7gjtEVge7d9Oi5OBCSEjqr\\\\\\\/R8+N+34x9nMYD4fEmM4LE3ocLXUPihAHL73T1bqIQu8o\\\\\\\/loso\\\\\\\/koE8wKQWfKqPzCN\\\\\\\/0GoO5LYB+H4x9z5qjBXOhLUY\\\\\\\/8L4zT80BT0iTUELprkXNzMM2ax6QwBThu0tkt4rS9gdQQye81lf0U09mQJhFCNyfTVIKZQClVrmFcvkHs\\\\\\\/lyQ1GthsWCBY9JiXEgE233hy+kvFY1hJqBFa2UpddxDi9vib3WwqUKzaMxha4iNUEyZEwFRnzrO6N5ZJAWmZra6mCuuNzXeglAnIjziRcTffdlj0vAqqH\\\\\\\/taZcCgnr2cI65+9VBC7\\\\\\\/HLDRUUioaC3+lA0+9nrPsfZWnFuAclls6EG5fzT9TqObqtJIVpAbTsTkQJak\\\\\\\/6orqBKEsL+iq1Sz2wiNv383cFCgdZSs3LceI6J1Cv5KLgtucc5RSU+n8XOjTbjqMPue\\\\\\\/FNgwZagWp4S0\\\\\\\/q7jRWBBxFOSE5DSTnCjfVVQcCt01aI6kYbRJwZwOhfd3MmxhP7p72bh9YkK9fkbawOsrcvrMOlwtEv+AdbCWZA4ZC0lPUlo75J4gQIxmBHtpEvQkjOXQn3\\\\\\\/fLjxMWdILL5d4pNB2J1M\\\\\\\/BXwMF8vcglpy2gZ8QfDoRHwipmKm5YNjIc3lQW+O2sLVCC\\\\\\\/J+7uYRFbgDMewp8D7xvZuI93FpXGxtvWTTbvgDFVogps8ut7+46BiICenX0Ip\\\\\\\/6Q9b4enW8KP1qJs\\\\\\\/7kkRha38DlRwLCqRed9DU3NuAlw9Ie2nmqMgGfemsOKgxySWDv+fhK5IQU7zWd7UEwdCyoyKt21w5x2MZIOuj+2FZblqIMahN6c26f8x+WKhJfc97H7elh7cE0FfnDJ4OVZdM855BwZ1TzKAMCgm5yCVqV8m1ezTENd7lTR3tloCwo2Ep79ujitJRtErccd0Q3RVd3tZWh4YGntUggXgQKcAOmPgF5wml1LWNJvauMeZ6orzVRFQvNBOHyqtiuLsMe7lyq5vcOec6CDGYwiYr7p6N8nGx1zLFNweqssiYhvlaDTRdO1Q9zEWbj23lOmPJYH9SrvJVdxjZq6FttqztOB64v8Hm\\\\\\\/qnyfkQqN+KanKL14QsCGIe9NihActsk1NlCO3+ImeWyzB8VOGNKwlzNa3vy3Ti17oEVOVVTquSg9yxPkeIGj0sOKTjS1xSix+tXsrdtPEQNbXJ8b84yvnhRP4j5TrqYNpr4F0ita3cz7F6fCjN5MNwmnQFlpXG4KM92SjbdONduM1yxpeNEVQpaq0oGZP7qJSpf8NqVUjolSw2H9yVye4Jf7ksZPmy\\\\\\\/F5H+MyGzClHqXTsxBpLhtok\\\\\\\/klp9\\\\\\\/LVULujMiw\\\\\\\/iwJPzO2X7OcKQwfFdiqt+ILaoQ4Vk7ReP9s+ynCxKO575Lf6UevquM2c1p0ghMrWRarEy0xfo8cmpOvkY5wArWzPUcPXxBFuBR42wg2SXF9Vve5rrOYMukFoUlnAyWWMsn1BHRBFW81hNhvE++x3f6hE22d4tgnRIyuPMt9Z9kYrdU46FP1v13X0hS6+NggQESAjnBeR2He7LhLoN6o8FQ1XhyuU1IBpNZWHXVO3T7IY9C+j2\\\\\\\/Wclvbm90uhfrXWgSh6ycIyZXO3JG67gyuApQFC4AdsVlx1e9mU1sXFMHz90kxF6hkg+o0X4CO70eLRIE0N2VjmJ6QBb+kLpKnUI62512sN3yChAqzeSW8OYO9TCXrCx+PC+pxI9V6o9noOwjlxUzZsfg1cSlmBO8BZbb8xnWx72P\\\\\\\/ZrpncI5mchh6OIez7CdSckmscTofJ0c1I+9220WtyhBN41Tm0hVsmlKbkuIykam6gj\\\\\\\/GXXM91FZlh82XrGDrYlleXW3DHrhtPb\\\\\\\/vMcCeIvljCSz2ELqmpCvjXp52P7Sw37nuT3FyLgMoZJ2AiznAEN6SJMStds1aTZa7PmVzV7+ChSRiy8ETgtLeUTfNVS2B702bZ3iiG7rSZotPxTZJqn11qWeEZL4RqP3MNC\\\\\\\/2+ej+iiky9ulOnKa4Euv4LqqFUCJEEURKD96blraw+8\\\\\\\/zOv0EEiuupQimkmZacbQl72UgA+cKe93WDTGmSRALIQtyP2CGRMHZCJH08UeVX+G1X4p+FD71X9H2huZo8+1GMz9fZgU7qPJFRtce24YPTKFmI823lEdnIqclsAX0sMoa3RXLg2LD0qNjC11CGxd4Du\\\\\\\/iEgW3MO5EKguF+pQzB5HZK70Qa72M0E3zNBV2aPxPE\\\\\\\/A66ZaWQXN5yVenelmPjd2V62f44sjFcs3S4vFZqRrGSZygYi884VFXonsrIzCFElg\\\\\\\/VUJOZG+THgCKysuZ8MhFUcLRU69iDDVPDmrJ+NRIJK3SgOFvgrRnEXClFuJDtEku2qfr7rYSoeUXISj5CuasWowYqH9VHHWnlFkB5CdvBh2owj4oHOPp4+2OpNE909KzFzEVBTrUk2p0duHVCaf2BXKwsrRv+\\\\\\\/N7hpjvegU6tpO5ij2A+1KNveMKusn6Ige3FxhBqOdTBwybCbtpW0PV7RdJ47n\\\\\\\/KjqzHUoSd3cu46MKOppxDGePgxoKN8Ia1olrjaXR6sbCFo4hd5lfet6dZgIHgkV4FpjudsoC9EymEqikIcWMShhesWQk4z1678VEr6yqgW2yXMakrT3NNIiBr4ejL4c5L\\\\\\\/EUfwFXit+msgDv0L+BkFWjYMhfps4GQ36i8s0PasHX8I\\\\\\\/BvkfIUZqPoxNWS7YKfdW1uOa1waX9p1p3Z2pc4zqs6DM51QBSRRGbsmy6mQhJEtEHVyX7A3qD0wa5rFSoLMGspN5RUT2R3Dy6DmqSzih3DyFlIvK4yNRu5T3fRAoAcwqn6ZAiGzvXPOeAPD8J3R5bU+iDYZWLNrfNnJUaGq5\\\\\\\/3YkMwlLR2yENK7WI4Y49SOFzIpdQ5pAMh0aEz6CdL5ve5Cma4edGuY++x9vxmEdrIRlwfz+qyQo+6g47VVlBDsip2HrrWfZsFEv8xoqQd8tEKhQeh0vWLeQVceRTekQEVehvM7TrAS8f9M2jk4U9o3e2vYkphZDx6y56Su17FuopjjfP3s3HiitxJuuOIOxexpP\\\\\\\/Yt45w6DVQCrRyfzOwuxM1qZgFxxDDuNoIEN8K3cvHfDEHREATcn8glqiQsGTgpn\\\\\\\/CTVZOFY7HD+Sx6ZsQDTZywpgN3qjF\\\\\\\/IeDimS0nepecEIbxnNn7hqb5FujWeGKu1tpbNdrKe9JThM7IQilzKLj\\\\\\\/wZyFnv7+pU5ohUPCPvPfcJThKuXEuvveR2eWmjKwl6wUfz5qhVXKB\\\\\\\/GfDtZf4R\\\\\\\/4AT9cYzMVoUq0jX2zAYBP24C3w5u49cWA7E+e6xc9o0gJmwZT2tlhF1NtljnEBP989wOStN5IH2ugJ0baJsqpfsjemlBlUyp\\\\\\\/coLmIZpCUXdyN2miMTJpce61E3etbw641Z9XYP5V0jnGcjL6sH0pPYQgrX5rq1WBEwfRLCvxhBPj+sk5GnUizKxUYy\\\\\\\/T++AFdkUx4zkNQPMCj8uOnxzDeQipKsmlGJvoeNeuRLkNrbffmHQ8H2e9oL+Xg8fpWLr4KibsSl6rZQxQfKqPrPvRCZNMH8jPhtLCXIBINjnY8WFXFSAx17mHVczZjAILNXuIvzRnQS8SUqw\\\\\\\/a0rBW+4FHvCM0gGAPtVkPBFZj4obBCpNsH0oR5SI8ZMEMU4nb1BIjH617gjJjiju8PbENfAVXhFCCifql1HueOfen+3Jx301X+BeTG+SkDm8BRVxnN8XgSMen7uluriBAC5M6q3HkfGUtNhrquabe3PF9vLOi+xFEE0KQNTjSuM3x7vSnq3PXpX2vj7Bn8b\\\\\\\/0A50s1m7XFSiysJh5QuvgDsZ515XjiNXrZKAD+t5EBsiKAe2zu2hNCh0Zc\\\\\\\/px2bpNOk4Vt1KZH9khmhIhd6UyxXAbrW7xo6c7yRRTvKJfmgBq+0sAnEQabkV5eOhLAKRzs6K7M39+3+3QGUyRVC93TNXk+C0z0TMqs4aK\\\\\\\/Z1A2isiexkf4qDjZhZ6eQEE6WMha7Rm0SFIOJw9b3ozJaEk8FoJu2oK\\\\\\\/SfK0yf\\\\\\\/0+ojzuQgOTvIVA\\\\\\\/Y1pKFcoY\\\\\\\/7L4OkiGRRXv7kxAbfWXb+1DqEMOEXU6fQFt5E8b1qAh4yAv+FgBSZee4nWvo\\\\\\\/kz5RWMp+3I3yFK2xloDCZFipV8POB7xW7kNMRP7B3azfrEZ0mCgexxNAeUtxVBxe2E1khdleSjSEE4tS0pEEfs1WS\\\\\\\/69C38H0E9LOv7sCvRn9\\\\\\\/RMCskQx0z\\\\\\\/U5vjoH8hRhna+mptEb37HQMNWLFN\\\\\\\/WFHODcgCGN6sPUb8hRCPqMChht6PVl1QBq7E406sU7blR+GrsP27PDZkKKMvl1Agg4DxOCjsWo46EtfGz9XqmJtJwBXa2DYCgjj3Rr4IuzaOZtI5YO7tpfGd09o1zS51RX4zbpfhkKzggh2ofy8vJbIUXQpmXV1N+jChGSCuDOfd48qYTtZyctPdIr3tblGqelZvT0LGUISfubEDFsoblQwfPD\\\\\\\/nCmYxUR+8rK+By6Blaa7gZUOyf4ZcPm6Ox5wdlmgri5bgame\\\\\\\/hKrbTdwDlwcYSlmFIeHMTld4N3oB2B25Vp3JpX0tGaDwlijcT34omi9IjPVmolpKPy8j+uxusq81S6Jtkf+rpuLX4P4kjeDaGrNjHzHULiJSS6dXd8epGDf93K4Sz1URGZLtGfujg2EjH5Ol2vQiVNKHxXOIhhPhnfx\\\\\\\/FDzRRjrRtmB8KYK6EYQnJ1AOW7cMLV+2OegvLfLa4httarrgYY4beiNI5gwtU2vZst12ODyMQ2wJ5LFHNUk3ouWrrYaGiRXxBfF\\\\\\\/XxoLlSI9duMyJLfewztNuDQd3wTn3tNHVmjTmUBgX8Wx9dy2P5SoyiRxHpjVqvzGhz0yuSO9n1bXl\\\\\\\/Qq37XMABJ+9nXlBz2VzTfJrVStAa\\\\\\\/+Rs3gWqm5b7kI3GRtDVxyH7yiWitRyrMpdCZJggsvOjIwVxWadnEsYCuP6M9eDiyttzGAVyZZno0sT6tkBDQF74L3EyA0GhXXjRFprxOdzo2WeV6+O9ax8VAympwT6PSpl8v8GQIU7tpfqwWYsAHAg5iJokdRJgpkcBXZcqdirjvIgf5k+5a3BckSMNKY+U9CfpJb1AsZ+QWrXejaW\\\\\\\/bk2+WDoELL05L\\\\\\\/3d72U7lnqbeWQUJmJ\\\\\\\/7VwkC5q+FnN438lrj\\\\\\\/ye4C+E6qpb92x4Qr0yuwTE+73JY+JEJfrl08oUUo\\\\\\\/AFug64PHth7zD15t+RQcH+aKbHpnabf+AgxbhMOIM2Iuo+MOBkSCfc9u7XjarHv\\\\\\\/9WwPqLRe+TOZF7Miw9IB4uiDj5AYtihCZhiVotauAThl17ZUhGNW1YIWk9wfGIEAi6nYMKED35WLEVF\\\\\\\/y9FUiOoQepZwSlqkGW0SqBnjHJSAmykcPT3iGhVxZwuLh0116rHUbAQgNHMSYzmftcHhnlUFWGUdxnrzAUHZP3\\\\\\\/379IQu1gSuU82jv\\\\\\\/XNX3IbmM5bRi+x\\\\\\\/bfmzJcF4n3dAA5LMt9y91w9Sxq\\\\\\\/IPb1Lr3ssrwfJ4SlxZlXr5YpLvCFPHDx2EgzIy1uEyMDVMR86VPbWdLjvkftAM7XdU2W3heyAgoVVmkjxAzBzffTmFZJdon4hgvX2ecKDFHHSK6TaES6HU2gt5+X\\\\\\\/aKiQFO2+uC7yVdTQHq3BGrv\\\\\\\/20Zvk\\\\\\\/E7tGhaTIPKU1aWiU+00E1kTUanSTvfLSDfdhzISqgLUvTQTTAcnfcu\\\\\\\/mBh3YMsMTf\\\\\\\/yOXAk\\\\\\\/rgGWHpLwGg+tGUOlUD25CSwV2llAdDAw0JJH3HOEdeUscZgExyTmoXluBgoiu6oG7mIMPcwjRCzhLy\\\\\\\/PokEjmDNNFCXJdxovqRWxLXi99mtH1ya0kRkKwSBY7BmiyB5s6xBWjH042ClrVg3VtWJEY40LVvT\\\\\\\/rPZV88Ym4eC3fU57xTmBXnNbjcXBbqSSkT+BuCWgo0+IMwdCVm2prRsH\\\\\\\/9TTSXf8VRKYruYNN9JLoJ6r+2EwIzTcpfFRhr8n1cMe+TZdarVFs0iGKPc39KVAocWTLDF+bADKPa29neOhK0Vuaj+lXWiRExU3V9AbJhUPi7VUpfdS7XBAmrt1hYZQ66WQD52+oI\\\\\\\/Swj8W9hIxFwA+5Zcve7G1nDzahhWR+M71RpA83z+Cm\\\\\\\/oIzrFy6Y6olfPZPntTMPK9TtS+qKlV4BLacGO2J924\\\\\\\/TxpHMBG+EaJq2IXY0sQIe0j7xsEn2vgzGn2rb6e+g2IcvYVLPfJMFAVsCpvTBsphUYXG1h\\\\\\\/8EgfPHPTbJDTKz8m\\\\\\\/mnY7yXbJD7VFElJ3qPJ+M\\\\\\\/pt98rYjphf4o3W\\\\\\\/w7QcZcrze7lotZpm64zOjCyj7v5br9X\\\\\\\/nQhs9D4wMI3W1Z\\\\\\\/0JvcjkPB\\\\\\\/vNaJVJgvILp1G\\\\\\\/lINU0QAAmqJRfzOjs9uE2E9+kualq8wmTESG7cF1xq5M+kb8HelIZOg7aucCNRdQZYPQAPXJmrBQ4AMi2blzSaN8Mi+DhroB8Fb3JB+m7CHxMhRSlvS0NKKMiFXzckbKJ8svhSG3\\\\\\\/6St\\\\\\\/aA7lRdznqGEBBU6Th7Tyfzl8rZH0z7naoBHLak3NCbPqJKTVTceaTmOmRZQ7EdUBJEM7xqzoWAz1WyzyXmh5211+Gnj0u\\\\\\\/AMUHXtHpXGNoZFkJ\\\\\\\/TiNvx4fRPMRbRGIy+2APhDzWZlwwdCyLtxRDRMaY0IOYZ4bCVTrbelQXPg\\\\\\\/iUGlSFHOMTY0N851QAESPiJ5w70qoSbINb791R3xLbaVIcLy5vhaaMWffJWEIKLb7C2h0+XlXCfXY55\\\\\\\/tDZOY7roUOW1NEjP1C+FGiFq3PlG8LVAGl4KBynpRIIp+jqx8ZonSWQWlNGTymerNfQnbYZw600O5OUk8Id\\\\\\\/Hlkf4KMzCJ9GK9B7rk2ZVmrdTiUYqD8WNh3pHpjjyRoNwZodQohGhZ29eUFvL7OjYcyqdE0C2DZ6UpJTvKvon9n0Inc+V4610zzCY4mmABfhTgzEoPSTVpI2Vn4SzkqFaaVloq1gF2ZruB38oHd9\\\\\\\/VLXNh2BvRnSiCwt+IUxgNgK9uho5JdPG2ES9FQ4xD1MBRm3v598A2YsR0cCriqQsA682XqTMQPmL\\\\\\\/Hee9gNA4zmdIjJsfypjvIO1sDUBh79\\\\\\\/tfb9uwrE9G\\\\\\\/oUNrYiIyd5lAGhY3hVGC0estpb9+qYd1hpPBylQX6GtQYQQahavUyZdMRHinS\\\\\\\/Wb8f7OWpbS4gtOwehhqXXuheiKIE2J9tONZB70vNUVv03RXflMD51inw7KEpF2Z0E\\\\\\\/qmHEpRMpJbbw\\\\\\\/DxbDTvIWPsLIJu671Mm37VC9sKODHuKfap96+Ogzg19HmBCFVRbpAXdvUrVaHXYwLUH+Vz81EDDzoWSBNKX9XfGzQ0sE4\\\\\\\/acX19affdqR77TcK62I20k616n1Q9rfpC9YjnntPD7t3RNSFLTcMunf1KPBDdgPzJYNy2i7am8VMYU56MNLj6QMrp5UKVT+h\\\\\\\/xkkJhVJqLZDC8HU2GqIvtRMNjeVJp03zIxkwjOH8x01BidcUHH3LLt+DktFjfZenwMtsRXn08ooDoiLgLaTjR+L+Dz4HSnlunS\\\\\\\/qPWJGwAltkTb0k7HdbTE\\\\\\\/qr\\\\\\\/sY7D+aA02a+nWIfKW8gaDYOeWp\\\\\\\/NC\\\\\\\/ilIR1Dme\\\\\\\/HP6\\\\\\\/rqzfIfqZqS+SLlwza6P0gi8hEgjFPEbcyw12odOyVGOy4IEVtWeLUfVfZDyDb+Q6Kirwo3\\\\\\\/m\\\\\\\/pWzd2N3AKewASwWUL4UrU22KZW2WWtCFuFu80c5WoIoIuHuMdOtrtODlI\\\\\\\/IQuZ3hRn53h\\\\\\\/Qk5myKK\\\\\\\/75KVo4lyJwYPNcwvVENMYhl\\\\\\\/52jvl+HYQ0YJ5oUfkRI9jN8hArHu318itHqBwr+HVT9uMhASxbBnFm1FyJBjCiRzqsOoUUZnca5qIvB+qB+nibCWQWw90BG9RPT9Czh8P7p9Qtmbc0r31iwdw8lbRBPWAh1J4UX9JNv1+ixSaZV7SnAdxpI+R7UCOZBDSc8YrTGeDJK+zWnYHiGJMUEF6OaNuN7J6Pl6NgBMhNebiVvyL5UeeSpeWqlydY2KE2Qs83UvLvOy4vyWePep+ZnqpTM8VufK9PqR\\\\\\\/Td1vaLB4K6uBLs4nIrfPrGI6pkGY8I62xcr3Z\\\\\\\/ui9Tvdlrbp8GOGZVOMUpxZ9e4+jJr7610Tjap0qaepGYVe9lopngKXLI79FvBIEQ3U4CSmeqPXyTDLeOkraYMgGG8sJSLCT8lRA8Fuj+uvymLTjscqJiJzgBS6E+uALaEJ8HqcBiYu9Bc5jYXccC3nn\\\\\\\/DrssJGGXLDFO\\\\\\\/1KJINOkRAC38GvWHmlwVcB6ritleThXTqvuNfM1HwL\\\\\\\/0gTrSFeqaH3BSSbDTAF0qEE3MZqpmrevntw\\\\\\\/pDYlgN0E9SbtC5PEngi920VSTT50BxWtf8k4Eg1LdaZR37JLr9jzqmIAO6D1t2Z1+pwdHV9A\\\\\\\/7an2rDlfJqxb5TMYNcQgaY3F47Da0A2v+r\\\\\\\/Nni7YMpXjXVdABCsRplJdyIbAjK23WNu29HjT45d7bIhlgR0\\\\\\\/TBmiHhbKIXrMBFeZePpoA8HJgJRNoNYAnrQ8RgYAiRVc17wtPVDExyaTh3CeopPt73HTLzH3\\\\\\\/dGAoy7AH\\\\\\\/7NhdR\\\\\\\/YdM+IqAYdabkiQ6rwTB\\\\\\\/5lXuwlmKh9X396rk6ORDYr6dKUFuUu+G2UqQa3ULHbBFsqGLXjjpB7yoAkF6k98MRmmmCau+dgVZOIgtcZSNtNtCp7pHkcMJzBW4GaGDhpnwTiTkm7E2JuMALJS2TY6eFYvzuhdLeSQUKSkqPH6s1ckoMghKl8zY9xUeJ+L5kPxCqHAZyrmqp5e9fO1aP1HhysynsdfDRBtIOpZ1IDo5QnXMxEBdlW7BsIZbeyyUbpIe4jaOR303j2VDfkAN\\\\\\\/MIKUCTS+nnM4nIw9Tng9EEgIyt9SSfJNhU\\\\\\\/B8ep\\\\\\\/J1ysEARRkbQTYQiPnC\\\\\\\/Ld3lWMlyLkqs0+wAGTX20lR44vtNAdOVD2gr03u+6jTgt5l+ZJzCrx6tTWNHugiZtBQEhdhoavb+nZeMIqTF\\\\\\\/OzvsRjWeIZbZawEwVVmqRFiuPCO\\\\\\\/ydUCpRIicTWD5f2vxVM\\\\\\\/JY1dxvnEVGmySpb0uH4w1+xnJ3pENbWhpd6+A3KKHSe0kBE7YMNR3s+9iY7ald5xiu7Yy70Wc9T4IV03fyatfu5joLFfnReDcizJWwQRGtapMLEsuRQHBd1Bk74UrWYWd1iQUsWEtAek3b7RwwTyG3ypLRHDiiGZr0ppKplVd5v4ZXs8zn9t2FBn9312m11hft0iWj6Pgp+QlbAsYiPSjqhCULv5UKHAV+7D6XpaigLEgWUgaP2A4YrdqVMzxfaJh\\\\\\\/hMp8pBJQ0OpG\\\\\\\/Ioe6wTaiyFVkPMbXQVgkp+NB7FVaaqKu7cT3tUnHuqVVlEDj6zn+PxQRKHWaHbV10EHN3jXVpCcPegQVAqimBTYfseJxbevDvy31jhl\\\\\\\/Z2YeuU6DJotHJw9W5QR63kbvv8ufSqI50OfGwviiNb\\\\\\\/leQyUxO4REp+P2XeletyGix3ZJszj0WF\\\\\\\/CeA7UE6gZZS1Yrwkz6+Mz\\\\\\\/KudqG\\\\\\\/GFwM2GZsowwpHR809b6BrE6oyKSszWtCHWo6BPqDTykjdCZP7yT0w9KCKOgz\\\\\\\/2D77xsv94cLwRUEMvny3i29ZxNXywqe0qqe9hvWKZUIXhaq8I+EjmIoj6emPBScL7QWvnMAjjTAxDJjZA6+APehON2hZ\\\\\\\/JaY376ewQgOVhycVHbC3hqNbfXmAFlwSXlcfkXbGoKfqe1VG5PJNj3WT2Wz1e4xloJzY+rP5DEm+15rMh0Kfvtr+1v5Cdqfm0tQgeZW25\\\\\\\/vuPY6REkEI4GxEK+160+YO0tawZTtx3RDEU8PAIt9S+Fb2WZvPc5O9XHpFjdmpjtwKeGaukcNxmqUshFiS50Tk76ANKHegWt7rRyN7CSCNwy2Ss2SFi4+BY8Z4UBcqr8fx7A834CkmMyTUJbPMZUT1+VBveb0AnrDovafVY4P78cElXqwjNZiloxIKsek4fb1N7\\\\\\\/o3SUiy5jZMxb8tTv5+o\\\\\\\/BmQtv2kqsbWNwGzXqoBE6VgFlXCQJfGGru69uW5wHbnbInMSIuf3s9rvyFDGPHkJokAyoSMFj75rbt7bNLgFelWmQ3yzILV6tpu2X5CudevNDX4NXN0+aQy9mIN\\\\\\\/zvzoQpwq0MO\\\\\\\/py188gy9H5fV+ZBS9CywgYopjibd+HzS9gE2a94h+7DOViy6Pq56JkBwCcbZKbRJK+g4ujXq80Iac4G\\\\\\\/nMS9adn6OkGP1VJzo3xBzzXWFtjW3jM0YR+Xy\\\\\\\/dyQ0venFwhjSySzNH8SHQoOefWsmuKAmi2YHutkpWTG2tiFI7SjjCTsBQ0e815g5\\\\\\\/fdWmCRrduMi1bFlfcASzx2yWHEEN9wExWBC6t4gFnsbD6dacSDZdy2mJjufgOya69CrRXtsv3qXjCRtT58oINmmmTZWhdqvD2x+5wc8YncbdSxskkMYkprptTjsQYLOGquMNtqvffBSYien7I2lvTpn2uXRvMHOI0GnCP5IvMxOm\\\\\\\/uaQ7mjDnNsdNh5zecf9UwwPrGaOxtB+RRU5riY1kyJy3cRwG\\\\\\\/Al\\\\\\\/bMYei38TmLvvOfNsu+krcnFLL9NlF+z\\\\\\\/6N8\\\\\\\/6vVsht9ZngJZTjSiumOtFgJQSOXJxfBYTzL0fva24YCzb4tOpOxFJul403jw6pR7QfnrXMRxgivzFqxTI6dNiUsEwktt\\\\\\\/ZOOlhw17YOSN42gtdVE\\\\\\\/72mF1fptIdWNHBtV126OiSr9QuzK4VsqeBw57yLFUVegRCsdf26ip7n1KYFHqm+NUnt9mCEO5DbgVEZFHSkidcFnstiCoOnXmWLQnqRugqpkzBMtTkTs\\\\\\\/l1JxirkMgjhnWRc7I9tgenH6g1Uh9fuTwbm26OjR7BnfnKWLbYuxwVZz6ho1MBXp21MeRZTaAjaRtCOMdYi9m+IRkyHVnR3NnZ6fjZgAHIt7y09c37oVSQ\\\\\\\/Nr3UUC25PbXwdvyQ30HIcURa6S1IvvugePTai93+PxR8gsKiFqIcB6+S8mAxFRMtkjzaULI1jctgpw1l6loCUOfklo7JWYM8y4IuBPpBWpFMNuRt4ncLpbty3lD6B6xbH1CLJWONsObzdjVovAHg7HmfgzEslahitsTUtW3x1EWfsyembZMUOQC0RiwTrrCUabHMuWOg4PBR1D5QuYuMyiSHWokcmtSZ5qEONG6LEYfb4aS9TVbscK+09R7eBeXCKxJuiPaw9FfGfAtPY3oGA3MGiDRvmmEFQAYrEZGG+fBoEOr9Bo748v2+BdjT12Ew4EWSXHo8ttSf5iDvvDQlNtvOUtFPrU8c7jW0XUCCI+hdHEghrbR7O\\\\\\\/OE\\\\\\\/LF4W4w1JOzCwDSMmmx0XYsN6\\\\\\\/rAs1hej60P8Adq1NKxB\\\\\\\/yeGJi1bo\\\\\\\/6lEV2bFFo5227gOk7FpjMmc2jY4+wR\\\\\\\/mQP41mdky+u2towkFp4EqDmz5nX+U1ahjOYcS8\\\\\\\/xtv3MrE2MKCLMQrNVt\\\\\\\/+2ia6SzErFuUPx\\\\\\\/IAJ83wBykAt975XClgd7RqJdMVcdqbDw1Wj9ADzhMxZv\\\\\\\/cZGXAbbHh7UdSzu692NmzZR8ZvBlcgH3tLt9+baXt0Z9BSTd0X3\\\\\\\/XPcrvWaLA\\\\\\\/mRWr0dizItaYDAH9aZVMBnWbbjaGmutXdLfXzFzM1U3qwAPFxfoWmhsFob7qP5xoVbxr68g\\\\\\\/BS3Bhhy8gOQpgX\\\\\\\/6VHOH07KKWGTSDjAJaVmR0A+xXctwbXtpT1Lv\\\\\\\/0aDVyybT9D0vQMVMnrH2T6C7Ni7QOT93QHY1EhyFf7F8QWq28Fq14BTLQj5PFrnj4wgrR2nkc6ZDccHwsJbFknlKYjqWuNjUTFxrx2CR7hJM8MFnPnfYU1YB1akJPoT5wUfbHbYz5HdCJqaVbw5gtiVYrduLQY7q46JMgTZp8NZ0aRmzOIdbq2AXiFhlMSQmmEyqTk+Nn7fhD+J+nnuA++fjMQ+0TJ0Wc\\\\\\\/G104FI5+oCdqQLw\\\\\\\/Ez1LbUCwqO99S6aaHGZ++1WtBYqmMfmt\\\\\\\/0t\\\\\\\/lwMNLVjFsIkbL42bi7Hw+m22x7TbE2nbnu2Js0hpI+4Tx2+io4SEnkYitwrDJEaHbWeSvmHBAtmXIb4cPq5w8G6ao2qPYZFQyGeooKEk99EEQQnhxOOEU2HGgp0CWpQiyQeYHLyDrYwVDu1gNpviXDaDG8DrQwzPOd1BYKHA9l5kFcxkPL37yHdv5D5mQfYHhvoGMssi\\\\\\\/Ih\\\\\\\/8YVu45pXULHozKkc9SMvmrGxHV9JhmXa3JzdR6854Kv7pSQvziyR410FjTeGOzsjuelQB+oqMS1N0K2tmKwpIYujkZkF\\\\\\\/2H9pYrerv35\\\\\\\/G0qwRDuDSjrKIIDtLlJ5k5n3mHp9bdyZYYsW0wL7Lrvw1gN907Fg00e3dmUvRX1bZ9wXrf2Vn5iyR\\\\\\\/e4TNf6iQpZgkdH0ZUAI2m6zSlXZ0BhYkL38Y\\\\\\\/cQ0h7h5qsFnncPAbTcEMypcHd9mYV1DIhzrZxN\\\\\\\/ctJvfeoWJvZ79PPuxjgbVWchRzlc1bUCMePQDogil1NvdyS0N4wq9Idb8vE4VqvAx3ZzdIC5phXtaJ\\\\\\\/Wk\\\\\\\/dy3WmaNHdY64cYJmhPN\\\\\\\/496ODktwg0PpP9fe\\\\\\\/sCbVulwzc6OxB05l6scDDc5FPooAiySHfwDwrmIgAXUmcZqAOfVp\\\\\\\/e8uei6BABOXIjzabj9f6Bf9wa19hbZGYcS8qYQHCSCAcdLFItNRWhp95jyYrmi8JHMKbvpqdjtNd7ZojqXsB6d8+\\\\\\\/vI\\\\\\\/n38IQyjAnurcLNEW5c44hNYa\\\\\\\/pe2PQa5ndBAHjCvWGvcR4+SiXeuaG+XsoRzYqLxfu4TN+VoTs80jcmiSHr69xjeQIRMyY2c2uARw4RqHQo5fKEJnnVDjY983fKlJtAf+YQbh9AuHo8zvor\\\\\\\/JqZoMTsUIk9MszQBcDr+pO0Lf9KTc9PUvcjj7AjC2JTr9QXUqcVLLK6Ltw\\\\\\\/I9YwzJfEHiHA\\\\\\\/ozsgPjS2IhZ9m6c6TS\\\\\\\/CK99atlYUe\\\\\\\/MHFZuhkKkF1+AP83zOPizV9FkRk9HM\\\\\\\/l\\\\\\\/n6gzuVwlZWlSA05MAcQwYBkhMggp5dBKG0Z55FllIkrm4I9bv74NlEIyQA7V+yfq6eX4rdrw5wq212v1qQZhnQ0n\\\\\\\/YUMOZMlsZp\\\\\\\/b\\\\\\\/m+Ytk2Z8g7UpSwSg6VKfX2omwrLfpn2W1qTl3nDtWTK7IwOzEJMgfW+mvPNRPStvF1ZF1Di3ESWY+w6wi2dMmIMzbbGeaQFWH2IG1hC90H9d4ekxAjSiFHgyH2TPRwxQWjT9tuBI1IP1Ec3pi6JQn6Axi9L6oZbtHwC9wRnEbquXhiAXakBLi3YkiokdKuzeZPVa0iO4gHucZjsKjegruWm8\\\\\\\/8vGD1tugpb2N8+Yfwa5gZkkGEiVIaElqm73itiGZppZReofwSP4kJMExssDW9xWITQg9UETn9tU8e4T5+bFXw9AXFg0h3xICfeQ8wJYQduucPgMynYc7WC2b35JPE+BMXpCCQHtQu1eY0k4DrSQ6GqVO2zczHszLSEECBmnnwWoh1gktPPZe+iz+jm73jBoo3mtjecNeVi2ZE9VTSOdN\\\\\\\/RAe7tPpc+Kn5ILEMfXobAfnYUs+VW8kqK6nZ2NeKL5JlfsYWW\\\\\\\/JRVbBufL6X\\\\\\\/IuObSvjhA==\\\u0022,\\\u0022iv\\\u0022:\\\u00222SN3cCy3N\\\\\\\/FVmhVM0TKoFw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022fa0a6401fc9779696e4742b6b18d9f99ca45a62158696d86a4e862df600ce6e9\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022QedTfaLv+fNliStkn6AT81Yo25mFxZ80LYAMASS\\\\\\\/8e9ZCLHhBzwlGGqiUfEzr9YhuDjPHgiQKE8pmRwhvDFmQzUq49UlrOJA5H58py+in4rLwu1yLtuPUaAnHKoAFXUEZaPSXBzb\\\\\\\/SKcxSM1ml1vhxoch0betNcbnlDUuwVPkpLaCtwZa1WAfZ+2\\\\\\\/2RGOzdnQMJQAfMMMa80ZgleoP4djNnWJGA3syMKDEQIN+T\\\\\\\/jtPTVXUbH1wV13V\\\\\\\/dk4SfUaXyZC8VdXUUsca2mo5jYPu7efTqQ2ISNc8cuhvgV6ZOul9Uxf9SFdTWChsMVD1+6jVkQwVFmIrIUne4QMKzCCA176ridVIhMU12gQkP8l+smyLIPf5JQFsWNurjZPxhzf8eKX43de2uERjzF5TYCgV0dE9uRXvIob+UTo9Ak7r0+RF0z\\\\\\\/cg4OVjVO2sDFDINV7qQ2Rgm\\\\\\\/fLaQWgc3OZ\\\\\\\/pCSriukxambfzyVFRU2G3QIyvciM4EHbv1K3eo4bB3wsQeiK\\\\\\\/MV9\\\\\\\/u8vSw4ILCyiIgtUz0AF3cCz8g+EFkgUqB7JEDRj8lMGBH66AJDYaZoU9u0D8GsmY7XvfuYhReANbw\\\\\\\/+oWCsJkN7x1Hcp73kxtxrwP576pYrBmcnP2Zo5pkz2TkqK8ZQKtzloVtVbGkSVEvQTU4Q6bmwtzDHDyGBqFKE1ks7FNzoRpGyjOTIfil7XgpM7KRbeLI8tRmXJte6H6ruBkLbZNHrCwRh1+TIaVmaTFhIYYCkv42C5pdbC5rDH69og83xmnE5zuK4+LdixlwkpzA4D7bsvwObWzgZvlqbUxq6d7eFkQcP+idHkayJ3RVRvSPpkAJG4H6Z8tPIrOZn5a7yyVWtuPiOGO+vnb3595WJQk4SA4IH6YRU1Ihs4fnV\\\\\\\/1sWL89vU2ANGFQxsJq9W2QtMHClpzY2OF2BAlkBoQsiaZA+2zStGtzmfhRgiJ8C+laCUQRLdXRgr9wsWHLLpN\\\\\\\/tWciRa+cznVsdpyuKUb\\\\\\\/riDGA+RygnxoHqG6YRPw\\\\\\\/3bg56CNB\\\\\\\/0Lwi+9t88TChoeuwLBrTf00yrZxe4STGy\\\\\\\/mDKNQXzv3rZCPHO1g9qqcgU6N5iqnJyYug9o3XMZp8TiNnn9ClXpy0sZ5okBTYgxUA6RVo3ilVtCgMHTtBVQ4nnUICE5B0l85kJV7boqndbbskoIL9mYv+0Zwp6GC0YQo7b29ChBLRlIMnAXo4DZxU1gSwtTCmGzRsk2luawnZ03HrL\\\\\\\/6uwi0CElbH29QWoW0ymkuSVP28d8B+6ze+UAEGMrYMo0IptSD3xOlG7s+iLWXYBG0HUOvfhUYZU6Bm0d2jAFJ+umDT7IfG2uhb2ac6v6L51EuSU+MGBcTW+eh\\\\\\\/HN0SJMSuo8wWEOGlQTg\\\\\\\/JGVnO62YjKLrr16vGFhjYVI3NocVxblz9iqUIa8+vp4D2zgyBBSRMo\\\\\\\/C8pbkjit6G3zAfI7qTkt4IncFvemPNb3IOkVIpbRcV75zrMWqV1Opbv02izXK1wPADJkT76uJVkOVwDK3ePB\\\\\\\/KVve\\\\\\\/ZsIHmPi8qC1jw41xOGhxZZOa8iKEeauczxOrd\\\\\\\/w\\\\\\\/BBFr\\\\\\\/qv3XT+ULt08\\\\\\\/lBXa5esGGeAsCFRZorHT3VO3lofECUc8PzUubg8eT5GT4t4E7HvWTmH8l1+TmXw4yfmupP+7nNLm8IGBqJ7fpJs05\\\\\\\/dl5L2Y5Lo5ilJrE348C9L91JwXRMY8KUjwoOMdP1GA8a1G3YiYR5SD38WThR9ws4+qbRDS2G1lZFOezA7jxXSXWKtJXhgTdGXhgpCH4Hb1DlbLWwKaEPo9zrRZP6P+GFku4UfVq1\\\\\\\/Y6dCVy2WFlgvd\\\\\\\/1dp0bWyu5rnP\\\\\\\/A1k0rvM9fKfZBrQdNC2uiDOeIYu5050JbsW5r7OYIBcJc62vymQXGn2ylciKJgyzgsfMO7lk6yex65qXMVhUXxBGuLpSyLTQWoY1KI3+gp74VGdLRhsy+\\\\\\\/mOu645yf\\\\\\\/Dva9ms8bvS8CFAEP6j6W17jGw1ILHCZ798SQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022qXpYG9kFtScueIqFy7Ba8A==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022d3d77cf0466c3163c44bb047a95d9934fb234635f58107cdfaeb57df0682784f\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-486347 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/9/2957" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-CuMftNTTsL_iwDgf3e2grrb82phJI8UO8P4AMz0KYA0" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/9/2957" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-C48bigUBR3yEc1usTnGMOhi7tYV6dW2kNx872eZ3bLE" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/9/2945" title="Functional Profiling of Transcription Factor Genes in Neurospora crassa" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/9/2969" title="The Caenorhabditis elegans Female-Like State: Decoupling the Transcriptomic Effects of Aging and Sperm Status" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="486347" id="node486347" data-pisa="ggg;7/9/2957" data-pisa-master="ggg;g3.117.041020" data-apath="/ggg/7/9/2957.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Reduced Protein Expression in a Virus Attenuated by Codon Deoptimization</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-7777-1653" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Benjamin R. Jack</span>, <span class="highwire-citation-author" data-delta="1">Daniel R. Boutz</span>, <span class="highwire-citation-author" data-delta="2">Matthew L. Paff</span>, <span class="highwire-citation-author" data-delta="3">Bartram L. Smith</span>, <span class="highwire-citation-author" data-delta="4">James J. Bull</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0002-7470-9261" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Claus O. Wilke</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2957-2968;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.041020 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node486347" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Benjamin R. Jack </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Integrative Biology, The University of Texas at Austin, Texas 78712 </div><div class='author-affiliation'><span class='nlm-sup'>†</span>Center for Computational Biology and Bioinformatics, The University of Texas at Austin, Texas 78712</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Institute for Cellular and Molecular Biology, The University of Texas at Austin, Texas 78712</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Benjamin%2BR.%2BJack%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jack%20BR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABenjamin%2BR.%2BJack%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-7777-1653" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Benjamin R. Jack</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:benjamin.r.jack@gmail.com" class="" data-icon-position="" data-hide-link-title="0">benjamin.r.jack@gmail.com</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Daniel R. Boutz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Institute for Cellular and Molecular Biology, The University of Texas at Austin, Texas 78712</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniel%2BR.%2BBoutz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Boutz%20DR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADaniel%2BR.%2BBoutz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Matthew L. Paff </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Integrative Biology, The University of Texas at Austin, Texas 78712 </div><div class='author-affiliation'><span class='nlm-sup'>†</span>Center for Computational Biology and Bioinformatics, The University of Texas at Austin, Texas 78712</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Institute for Cellular and Molecular Biology, The University of Texas at Austin, Texas 78712</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Matthew%2BL.%2BPaff%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Paff%20ML&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMatthew%2BL.%2BPaff%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Bartram L. Smith </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Integrative Biology, The University of Texas at Austin, Texas 78712 </div><div class='author-affiliation'><span class='nlm-sup'>†</span>Center for Computational Biology and Bioinformatics, The University of Texas at Austin, Texas 78712</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Institute for Cellular and Molecular Biology, The University of Texas at Austin, Texas 78712</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bartram%2BL.%2BSmith%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Smith%20BL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABartram%2BL.%2BSmith%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">James J. Bull </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Integrative Biology, The University of Texas at Austin, Texas 78712 </div><div class='author-affiliation'><span class='nlm-sup'>†</span>Center for Computational Biology and Bioinformatics, The University of Texas at Austin, Texas 78712</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Institute for Cellular and Molecular Biology, The University of Texas at Austin, Texas 78712</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=James%2BJ.%2BBull%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bull%20JJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJames%2BJ.%2BBull%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Claus O. Wilke </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Integrative Biology, The University of Texas at Austin, Texas 78712 </div><div class='author-affiliation'><span class='nlm-sup'>†</span>Center for Computational Biology and Bioinformatics, The University of Texas at Austin, Texas 78712</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Institute for Cellular and Molecular Biology, The University of Texas at Austin, Texas 78712</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Claus%2BO.%2BWilke%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wilke%20CO&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AClaus%2BO.%2BWilke%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-7470-9261" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Claus O. Wilke</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/7/9/2957" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:486347" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:486347/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/9/2957.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:486347" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:486347/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/9/2957.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:486347" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:486347/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/7/9/2957.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:486347" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:486347/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">A general means of viral attenuation involves the extensive recoding of synonymous codons in the viral genome. The mechanistic underpinnings of this approach remain unclear, however. Using quantitative proteomics and RNA sequencing, we explore the molecular basis of attenuation in a strain of bacteriophage T7 whose major capsid gene was engineered to carry 182 suboptimal codons. We do not detect transcriptional effects from recoding. Proteomic observations reveal that translation is halved for the recoded major capsid gene, and a more modest reduction applies to several coexpressed downstream genes. We observe no changes in protein abundances of other coexpressed genes that are encoded upstream. Viral burst size, like capsid protein abundance, is also decreased by half. Together, these observations suggest that, in this virus, reduced translation of an essential polycistronic transcript and diminished virion assembly form the molecular basis of attenuation.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/bacteriophage-t7" class="hw-term hw-article-keyword hw-article-keyword-bacteriophage-t7" rel="nofollow">bacteriophage T7</a></li><li class="kwd"><a href="/keyword/codon-deoptimization" class="hw-term hw-article-keyword hw-article-keyword-codon-deoptimization" rel="nofollow">codon deoptimization</a></li><li class="kwd"><a href="/keyword/recoding" class="hw-term hw-article-keyword hw-article-keyword-recoding" rel="nofollow">recoding</a></li><li class="kwd"><a href="/keyword/translation" class="hw-term hw-article-keyword hw-article-keyword-translation" rel="nofollow">translation</a></li><li class="kwd"><a href="/keyword/viral-attenuation" class="hw-term hw-article-keyword hw-article-keyword-viral-attenuation" rel="nofollow">viral attenuation</a></li></ul><p id="p-3">A recent methodological advance in the development of attenuated viral vaccines has been the design of genomes with hundreds of synonymous codon changes that are collectively suboptimal (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Burns <em>et al.</em> 2006</a>, <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">2009</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Mueller <em>et al.</em> 2006</a>, <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">2010</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Coleman <em>et al.</em> 2008</a>; <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Wimmer <em>et al.</em> 2009</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Bull <em>et al.</em> 2012</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Nougairede <em>et al.</em> 2013</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Shen <em>et al.</em> 2015</a>). The nature of the design varies from the replacement of common codons with rare codons to merely shuffling existing codons to create uncommon codon pairs. By maintaining wild-type protein sequences, the recoding method retains the antigenic profile of the wild type while reducing viral growth rate and virulence. Based on the premise that silent codon changes have individually small effects, the method should not only allow the tunable crippling of viral growth to an arbitrary degree but also profoundly retard the reevolution of high viral fitness. Both predictions have been supported empirically. Thus, poliovirus, influenza, several arboviruses, and a bacteriophage all exhibit quantitative fitness declines after synonymous codon changes (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Burns <em>et al.</em> 2006</a>; <a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Mueller <em>et al.</em> 2006</a>, <a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">2010</a>; <a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Coleman <em>et al.</em> 2008</a>; <a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Nougairede <em>et al.</em> 2013</a>; <a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Shen <em>et al.</em> 2015</a>; <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Bull <em>et al.</em> 2012</a>), and fitness recovery during viral growth over hundreds of generations is at best slow (<a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">Burns <em>et al.</em> 2006</a>; <a id="xref-ref-15-3" class="xref-bibr" href="#ref-15">Coleman <em>et al.</em> 2008</a>; <a id="xref-ref-9-3" class="xref-bibr" href="#ref-9">Bull <em>et al.</em> 2012</a>; <a id="xref-ref-31-3" class="xref-bibr" href="#ref-31">Nougairede <em>et al.</em> 2013</a>).</p><p id="p-4">The broad success of attenuation from synonymous codon changes in different viruses and with different designs could arise from a common underlying mechanism. Yet, there is ongoing debate about how synonymous codon changes affect fitness and thus what that mechanism could be. One popular hypothesis is that codon usage controls translational efficiency, in turn affecting the rate of protein synthesis (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Fredrick and Ibba 2010</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Plotkin and Kudla 2010</a>; <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Tuller <em>et al.</em> 2010</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Shah and Gilchrist 2011</a>; <a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Zur and Tuller 2016</a>). Under this hypothesis, highly expressed and functionally important genes are encoded by optimal codons to increase translational efficiency. However, a simple codon “optimality” model cannot explain the attenuation attained by merely shuffling codons; since the abundance of the different codons is not being changed by shuffling, the suboptimality must be due to something besides codon abundance. Likewise, some highly expressed genes in cyanobacteria and <em>Neurospora</em> have nonoptimal encodings (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Xu <em>et al.</em> 2013</a>; <a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Zhou <em>et al.</em> 2013</a>). In other cases, codon usage determines expression through transcription, not translation (<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Zhou <em>et al.</em> 2016</a>). In the face of so many seemingly contradictory observations, further advances in this research program would benefit from the identification of a common mechanism for viral attenuation, or at least benefit from the demonstration that different mechanisms are involved.</p><p id="p-5">Resolving the basis of attenuation in viral systems is partly hampered by the sequence of life-history steps between the initial effects of codon changes and the final emergence of assembled virions. The initial impact may lie in transcription or translation of one or more genes, but the effect on the number of virions will depend on which proteins are limiting during assembly. The same modification of an early life-history stage may have different fitness effects in different viruses. Ultimately, it may be necessary to interpret the genome engineering in the context of a system-wide, comprehensive model of the viral life cycle. Such is our motivation.</p><p id="p-6">In the bacterial virus T7, recoding the major capsid protein (gene <em>10A</em>) with synonymous codons reduced the fitness of the phage (<a id="xref-ref-9-4" class="xref-bibr" href="#ref-9">Bull <em>et al.</em> 2012</a>). To engineer T7, codons in gene <em>10A</em> that were highly utilized in <em>Escherichia coli</em> (the T7 host) were replaced with codons that were underutilized in the host. The major capsid protein, which forms the head of the T7 phage particle, is the most abundant and highly expressed phage protein (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Dunn and Studier 1983</a>). In the phage genome with the most extensive set of gene <em>10A</em> synonymous codon replacements, the fitness was 35.7 doublings/hr compared to 43.2 doublings/hr in the wild type (<a id="xref-ref-9-5" class="xref-bibr" href="#ref-9">Bull <em>et al.</em> 2012</a>). This difference translates to a 180-fold decline in descendants produced per hour. In total, 182 codons were changed, just over half the number of codons in the major capsid gene; in addition, the first 20 codons were not altered to avoid disrupting translation initiation processes.</p><p id="p-7">After adapting the recoded phage for 1000 generations, fitness increased to 38.7, a recovery of almost half the initial deficit (on a log scale). Thus, recoding gene <em>10A</em> induced a moderately stable fitness reduction. A mere nine nucleotide changes were responsible for the fitness recovery, and seven fell outside the recoded region, shedding little light on the underlying mechanism of fitness reduction.</p><p id="p-8">Here, we apply new methods to continue exploration of the T7 attenuation. Using the same strains and designs as the prior study (<a id="xref-ref-9-6" class="xref-bibr" href="#ref-9">Bull <em>et al.</em> 2012</a>), our purpose is to develop a comprehensive model of the way silent codon changes cause reduced fitness. As part of this effort, we propose and test three mechanistic models that could explain the fitness reduction in recoded T7. In testing those models, we apply proteomic methods, RNA sequencing, and various phenotypic measures in a systems approach to understanding the basis of attenuation. We find that recoding gene <em>10A</em> reduces protein abundances of gene <em>10</em> and also of several downstream genes. From there, we address the impact of protein abundances on viral fitness components (burst size and lysis time), ultimately connecting these measurements to a model that describes actual fitness.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Gene 10 nomenclature</h3><p id="p-9">Gene <em>10</em> is translated in two forms, A and B. Form A is 344 amino acids and is formally denoted the major capsid protein. Form B (the minor capsid protein) is not essential and results from a ribosomal frameshift at the end of A and is 397 amino acids. In the engineering, all codon changes were within <em>10A</em> and thus also within <em>10B</em>. Moreover, since most peptide fragments coming from the minor and major capsid proteins ambiguously mapped to both proteins, abundances of 10A and 10B were not differentiated using our proteomic methods and were combined. We followed a similar procedure for our RNA-sequencing analyses. To simplify notation, we merely refer to the recoded gene as <em>10</em> and the affected A and B proteins as capsid protein.</p></div><div id="sec-3" class="subsection"><h3>Bacteriophage T7 strains and E. coli hosts</h3><p id="p-10">The host for all experiments was IJ1133 [<em>E. coli</em> K-12, F-ΔlacX74 thiΔ (mcrC-mrr)102::Tn10]. T7 strains used in this study come from previous work (<a id="xref-ref-9-7" class="xref-bibr" href="#ref-9">Bull <em>et al.</em> 2012</a>). An isolate of T7<sub>61</sub> (a population adapted to grow optimally on IJ1133 specifically through serial passage) was first deleted of its gene <em>10</em>, then recombined over a plasmid carrying a different gene <em>10</em> engineered to contain a low fraction (0.1) of preferred codons, with 182 codon changes. The recombinant, denoted here as the recoded strain, could be identified by its ability to grow without complementation. The evolved strain was initiated from the recoded phage and adapted over 800–1000 generations [strain L1 from <a id="xref-ref-9-8" class="xref-bibr" href="#ref-9">Bull <em>et al.</em> (2012)</a>]. The wild-type strain in this study was derived from the recoded strain after recombination over a plasmid containing wild-type gene <em>10</em>, then grown out for 6 hr of serial transfer on IJ1133. Fitness of this “wild-type” strain was approximately the same as that of the ancestral population (T7<sub>61</sub>).</p></div><div id="sec-4" class="subsection"><h3>Burst size and lysis time</h3><p id="p-11">Lysis time and burst size assays were performed as previously described (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Heineman and Bull 2007</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Bull <em>et al.</em> 2011</a>). The initial infection steps were identical for both assays. Briefly, 10<sup>8</sup> phage (MOI = 0.1) were added to a 10 ml culture of exponentially growing cells (37° with agitation), incubated for 3 min, and subsequently diluted 10<sup>4</sup>-fold to prevent further adsorption. For lysis times, phage were plated at various time points between 4 and 18 min (after initial infection) to monitor changes in titer; lysis time was taken as the time of the first significant increase in titer.</p><p id="p-12">To determine burst size, initial density of phage-infected cells was determined by plating phage before and after treatment with chloroform 5 and 6 min after initial infection. Cells infected with phage at the time of chloroform treatment do not produce viable phage, so only free phage will form plaques, allowing for the determination of phage-infected cells at these times. Final phage titers were obtained at 15, 16, and 17 min by plating chloroform-treated samples. Burst size was then calculated as the phage titer at the end time points divided by the number of initial phage-infected cells.</p></div><div id="sec-5" class="subsection"><h3>RNA sequencing</h3><p id="p-13"><em>E. coli</em> was grown in LB broth to a concentration of 10<sup>8</sup> cells/ml at 37° with agitation, then infected with phage at an MOI of 2.5. At 1, 5, and 9 min postinfection, 2 ml of bacterial suspension were removed from the phage-infected cultures and pelleted in a microcentrifuge. Pellets were either flash frozen in liquid nitrogen or immediately used for downstream processes (RNA extraction or protein preparation for proteomics). RNA was isolated using Trizol reagent, following the manufacturer’s protocol. Library preparation and sequencing was performed by the University of Texas Genome Sequencing and Analysis Facility using Illumina NextSequation 500 (SR75).</p><p id="p-14">Since gene <em>10B</em> is a readthrough product of gene <em>10A</em>, we excluded the gene <em>10B</em> transcript from the reference transcriptome. RNA-sequencing reads were quantified using Kallisto (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Bray <em>et al.</em> 2016</a>) and <em>E. coli</em> K-12 (NCBI: U00096.3) and T7 (NCBI: NC_001604.1) reference genomes. For analyses of the recoded and evolved T7 strains, the gene <em>10</em> sequence was replaced with the recoded sequence (supplementary file S1 in <a id="xref-ref-9-9" class="xref-bibr" href="#ref-9">Bull <em>et al.</em> 2012</a>,) in the reference genome. A population of T7-infected <em>E. coli</em> has no core set of stably expressed genes with which to normalize during differential expression analysis. Therefore, all transcript abundance estimates (transcripts per million) were normalized to the total cellular transcript abundance (including both T7 and <em>E. coli</em> transcripts). Differential expression analysis was only possible within genes across treatments, but not between genes in the same treatment.</p></div><div id="sec-6" class="subsection"><h3>Proteomics</h3><p id="p-15">Proteomics was performed as previously described in <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Houser <em>et al.</em> (2015)</a>. In brief, T7-infected <em>E. coli</em> cell pellets (prepared as described in the <em>RNA sequencing</em> section above) were resuspended in 50 mM Tris-HCl, pH 8.0, 10 mM DTT. 2,2,2-trifluoroethanol (Sigma-Aldrich) was added to 50% (v/v) final concentration and samples were incubated at 56° for 45 min. Following incubation, iodoacetamide was added to a concentration of 25 mM and samples were incubated at room temperature in the dark for 30 min. Samples were diluted 10-fold with 2 mM CaCl<sub>2</sub>, 50 mM Tris-HCl, pH 8.0. Samples were digested with trypsin (Pierce) at 37° for 5 hr. Digestion was quenched by adding formic acid to 1% (v/v). Tryptic peptides were bound, washed, and eluted from HyperSep C18 SpinTips (Thermo Fisher Scientific). Eluted peptides were dried by speed-vac and resuspended in Buffer C (5% acetonitrile, 0.1% formic acid) for analysis by LC-MS/MS.</p><p id="p-16">For LC-MS/MS analysis, peptides were subjected to separation by C18 reverse phase chromatography on a Dionex Ultimate 3000 RSLCnano UHPLC system (Thermo Fisher Scientific). Peptides were loaded onto an Acclaim C18 PepMap RSLC column (Dionex; Thermo Fisher Scientific) and eluted using a 5–40% acetonitrile gradient over 250 min at 300 nl/min flow rate. Eluted peptides were directly injected into an Orbitrap Elite mass spectrometer (Thermo Fisher Scientific) by nano-electrospray and subjected to data-dependent tandem mass spectrometry, with full precursor ion scans (MS1) collected at 60,000 resolution. Monoisotopic precursor selection and charge-state screening were enabled, with ions of charge &gt;+1 selected for collision-induced dissociation. Up to 20 fragmentation scans (MS2) were collected per MS1. Dynamic exclusion was active with 45-sec exclusion for ions selected twice within a 30-sec window.</p><p id="p-17">We assigned each peptide to a protein or protein group (in the case of ambiguous peptides which map to multiple proteins) using Proteome Discoverer (Thermo Fisher Scientific) and REL606 and T7 reference proteomes (NCBI: NC_012967, NC_001604.1) concatenated with a database of contaminant proteins (<a href="http://www.biochem.mpg.de/5111795/maxquant">http://www.biochem.mpg.de/5111795/maxquant</a>). We selected the top three most-abundant peptides by peak area for each protein. We averaged these peptide peak areas across technical replicates to obtain a protein-abundance estimate (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Silva <em>et al.</em> 2006</a>). All protein-abundance estimates were normalized to the total <em>E. coli</em> protein content of the sample.</p><p id="p-18">To determine if gene <em>10</em> C-terminal peptides were more common than N-terminal peptides in the recoded T7 strain compared to the wild-type strain, we compared the fit of the following two models [given here in notation from the lme4 R package (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bates <em>et al.</em> 2015</a>)]:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2957/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2957/embed/mml-math-1.gif"/></noscript></span><span class="disp-formula-label">(1)</span></span><span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2957/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2957/embed/mml-math-2.gif"/></noscript></span><span class="disp-formula-label">(2)</span></span>where count is the number of peptides of type peptide, strain is the strain of T7, and location is the location of peptide within gene <em>10</em>. We assume peptide to be a random effect, given by the (1|peptide) term. The term strain:location indicates interaction between strain and location. Thus, we compare a model in which the location of a peptide and the strain interact, and a model in which there is no such interaction. If N-terminal peptides were less prevalent than C-terminal peptides in the recoded T7 strain, we would expect Equation 2 to provide a better fit than Equation 1.</p></div><div id="sec-7" class="subsection"><h3>Models of translational coupling</h3><div id="sec-8" class="subsection"><h4>Biophysical model:</h4><p id="p-19">Secondary structure near the ribosome binding site (RBS) can inhibit translation initiation. On polycistronic transcripts, this secondary structure can be disrupted by ribosomes completing translation of an upstream gene, thus increasing translation initiation rates. Estimates of this relative increase in translation initiation of gene <em>11</em> due to translation of gene <em>10</em> was predicted using the Operon Calculator (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Tian and Salis 2015</a>).</p></div><div id="sec-9" class="subsection"><h4>Mathematical model:</h4><p id="p-20">To model the effects of translational coupling on protein production, we first assume a polycistronic transcript with three genes <em>a</em>, <em>b</em>, and <em>c</em>. We write the effective initiation rate <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2957/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2957/embed/mml-math-3.gif"/></noscript></span></span> of gene <em>a</em> as<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2957/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2957/embed/mml-math-4.gif"/></noscript></span><span class="disp-formula-label">(3)</span></span>where <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2957/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2957/embed/mml-math-5.gif"/></noscript></span></span> is the aggregate initiation rate of <em>a</em> and <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2957/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2957/embed/mml-math-6.gif"/></noscript></span></span> is the translation elongation rate of gene <em>a</em>. We assume that if the initiation rate ever exceeds the elongation rate, ribosomes will quickly back up on the transcript and make elongation the rate-limiting step of translation. Thus, in our model, the elongation rate can never be exceeded by the aggregate translation initiation rate. For gene <em>a</em>, the aggregate initiation rate <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2957/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2957/embed/mml-math-7.gif"/></noscript></span></span> is simply the <em>de novo</em> initiation rate because there are no genes upstream of <em>a</em>.</p><p id="p-21">The rate at which ribosomes complete translation of gene <em>a</em>, equivalent to the production rate of protein A, is defined as<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2957/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2957/embed/mml-math-8.gif"/></noscript></span><span class="disp-formula-label">(4)</span></span>at steady state. In this context, the steady state assumption means that all three protein products are being produced continuously at the equilibrium rate.</p><p id="p-22">For the effective translation initiation rate <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2957/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2957/embed/mml-math-9.gif"/></noscript></span></span> of gene <em>b</em>, we similarly write<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2957/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2957/embed/mml-math-10.gif"/></noscript></span><span class="disp-formula-label">(5)</span></span>where <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2957/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2957/embed/mml-math-11.gif"/></noscript></span></span> is the aggregate translation initiation rate due to upstream-dependent reinitiation and <em>de novo</em> initiation, and <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2957/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2957/embed/mml-math-12.gif"/></noscript></span></span> is the translation elongation rate of <em>b</em>. We define the aggregate translation initiation rate<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2957/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2957/embed/mml-math-13.gif"/></noscript></span><span class="disp-formula-label">(6)</span></span>where <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2957/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2957/embed/mml-math-14.gif"/></noscript></span></span> is the rate of upstream translating ribosomes reinitiating on gene <em>b</em>, and <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2957/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2957/embed/mml-math-15.gif"/></noscript></span></span> is the rate of ribosomes initiating <em>de novo</em> on gene <em>b</em>.</p><p id="p-23">Lastly we define reinitiation and <em>de novo</em> initiation rates on gene <em>b</em> as follows:<span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2957/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2957/embed/mml-math-16.gif"/></noscript></span><span class="disp-formula-label">(7)</span></span><span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2957/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2957/embed/mml-math-17.gif"/></noscript></span><span class="disp-formula-label">(8)</span></span>where <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2957/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2957/embed/mml-math-18.gif"/></noscript></span></span> is the rate of ribosomes flowing from upstream translation of gene <em>a</em>, and <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2957/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2957/embed/mml-math-19.gif"/></noscript></span></span> represents the proportion of that ribosome flow reinitiating on gene <em>b</em>. We assume that the rate of <em>de novo</em> initiation depends, in part, on upstream ribosomes relaxing secondary structure around the RBS of gene <em>b</em>. Thus, the rate of <em>de novo</em> initiation is given by the upstream ribosome flow <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2957/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2957/embed/mml-math-20.gif"/></noscript></span></span> scaled by some constant <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2957/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2957/embed/mml-math-21.gif"/></noscript></span></span> (facilitated binding), and by a constant rate <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2957/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2957/embed/mml-math-22.gif"/></noscript></span></span> that does not depend on upstream ribosome flow.</p><p id="p-24">We can simplify the effective initiation rate to<span class="disp-formula" id="disp-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2957/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2957/embed/mml-math-23.gif"/></noscript></span><span class="disp-formula-label">(9)</span></span>We simplify the effective initiation rate further by defining a coupling constant,<span class="disp-formula" id="disp-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2957/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2957/embed/mml-math-24.gif"/></noscript></span><span class="disp-formula-label">(10)</span></span>which accounts for both the effects of facilitated binding and ribosome reinitiation. The final effective initiation rate of <em>b</em> is defined as<span class="disp-formula" id="disp-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2957/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2957/embed/mml-math-25.gif"/></noscript></span><span class="disp-formula-label">(11)</span></span>Similar to the protein production rate of gene <em>a</em>, we define the protein production of gene <em>b</em><span class="disp-formula" id="disp-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2957/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2957/embed/mml-math-26.gif"/></noscript></span><span class="disp-formula-label">(12)</span></span>at steady state.</p><p id="p-25">The effective initiation rate of gene <em>c</em> is similar to that of gene <em>b</em><span class="disp-formula" id="disp-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2957/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2957/embed/mml-math-27.gif"/></noscript></span><span class="disp-formula-label">(13)</span></span><span class="disp-formula" id="disp-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2957/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2957/embed/mml-math-28.gif"/></noscript></span><span class="disp-formula-label">(14)</span></span>where <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2957/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2957/embed/mml-math-29.gif"/></noscript></span></span> is the rate of ribosomes flowing from the end of upstream gene <em>b</em>, and <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2957/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2957/embed/mml-math-30.gif"/></noscript></span></span> represents the proportion of that ribosome flow reinitiating on gene <em>c</em>. The rate of <em>de novo</em> initiation dependent on upstream ribosome flow (facilitated binding) is given by <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2957/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2957/embed/mml-math-31.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2957/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2957/embed/mml-math-32.gif"/></noscript></span></span> is the <em>de novo</em> initiation rate independent of upstream ribosome flow. Again, <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2957/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2957/embed/mml-math-33.gif"/></noscript></span></span> is a coupling constant that incorporates the effects of both facilitated binding and ribosome reinitiation.</p></div></div><div id="sec-10" class="subsection"><h3>Statistical software and plots</h3><p id="p-26">All statistical tests were conducted using the R language (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">R Core Team 2014</a>). All plots were generated using the ggplot2 package (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Wickham 2009</a>).</p></div><div id="sec-11" class="subsection"><h3>Data availability</h3><p id="p-27">Raw RNA reads are available at NCBI GEO (ID: GSE96573) (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Barrett <em>et al.</em> 2013</a>). The mass spectrometry proteomics data have been deposited to the ProteomeXchange Consortium via the PRIDE (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Vizcaíno <em>et al.</em> 2016</a>) partner repository with the data set identifier PXD006502. All processed data and scripts are available at <a href="https://github.com/benjaminjack/phage_attenuation">https://github.com/benjaminjack/phage_attenuation</a>.</p></div></div><div class="section results" id="sec-12"><h2 class="">Results</h2><div id="sec-13" class="subsection"><h3>Codon deoptimization reduces capsid protein abundances</h3><p id="p-28">As codon deoptimization is thought to affect translational efficiency, we propose three models in which recoding gene <em>10</em> (capsid protein) affects protein abundances in T7. In model 1, codon deoptimization slows translation of gene <em>10</em> and reduces the abundance of the capsid protein only. In model 2, deoptimization depletes the ribosome pool by creating high ribosomal densities on capsid protein transcripts, thus reducing translation of all viral proteins late in the infection cycle (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Vind <em>et al.</em> 1993</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Birch <em>et al.</em> 2012</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Raveh <em>et al.</em> 2016</a>). In model 3, codon deoptimization has intermediate effects between models 1 and 2: translation is impaired for gene <em>10</em> and downstream genes but not for upstream genes. The expectation in model 3 arises because T7 produces many polycistronic transcripts, leading to translational coupling of gene <em>10</em> and genes immediately downstream. Translational coupling has been observed in bacterial operons but has not been considered in the context of codon deoptimization (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Oppenheim and Yanofsky 1980</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Schümperli <em>et al.</em> 1982</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Aksoy <em>et al.</em> 1984</a>; <a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">Tian and Salis 2015</a>). All three models assume that deoptimizing gene <em>10</em> will reduce the abundance of at least the capsid protein.</p><p id="p-29">To differentiate between our three proposed models, we compared the T7 proteome during infection among wild-type, attenuated, and evolved phages using mass spectrometry-based protein quantitation. T7 is thought to encode 58–60 proteins, but only 19 are essential, and many have no known function (<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Dunn and Studier 1983</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Molineux 2006</a>) (<em>e.g.</em>, some are thought to be homing endonucleases, selfish elements). All genes are encoded on the same strand, and expression order is linear; genes are numbered in order with essential genes having integral numbers (<em>1</em>–<em>19</em>) and nonessential genes having fractional numbers. The genome is divided into three expression groups. Class-I genes are the first to enter and are expressed from promoters at the entering end of the genome, transcribed by the host RNA polymerase (RNAP). The phage RNAP gene (numbered gene <em>1</em>) is the last of the class-I genes and the first essential gene. All other genes are expressed from phage promoters, but nearly all transcripts are polycistronic as there is only one terminator for phage RNAP (immediately after gene <em>10</em>), and there are only 17 phage promoters (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). Gene <em>1.1</em> is the first class-II gene (<a id="xref-ref-16-3" class="xref-bibr" href="#ref-16">Dunn and Studier 1983</a>). [Note that genes <em>1.1</em>–<em>1.3</em> are sometimes also considered to be part of class I, because they are transcribed by both <em>E. coli</em> and T7 RNAP (<a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Molineux 2006</a>).] Gene <em>6.5</em> is the first class-III gene.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/9/2957/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="T7 produces many polycistronic transcripts. The bar across the top shows the T7 genome, and each class is shown in a different color. Horizontal lines represent transcripts. Dashed vertical lines represent RNase cleavage sites, where R3.8, R4.7, R6.5, and R18.5 are strong cleavage sites. R13 is a weak RNase cleavage site. The solid vertical line represents the terminator T&#x3D5;. Genes 11 and 12 are only ever expressed as a product of readthrough of T&#x3D5;, indicated by the readthrough transcript. Only transcripts containing class-III genes are shown. Not all read-through products are shown. Redrawn from Dunn and Studier (1983)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-180513883" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;T7 produces many polycistronic transcripts. The bar across the top shows the T7 genome, and each class is shown in a different color. Horizontal lines represent transcripts. Dashed vertical lines represent RNase cleavage sites, where R&lt;em&gt;3.8&lt;/em&gt;, R&lt;em&gt;4.7&lt;/em&gt;, R&lt;em&gt;6.5&lt;/em&gt;, and R&lt;em&gt;18.5&lt;/em&gt; are strong cleavage sites. R&lt;em&gt;13&lt;/em&gt; is a weak RNase cleavage site. The solid vertical line represents the terminator T&#x3D5;. Genes &lt;em&gt;11&lt;/em&gt; and &lt;em&gt;12&lt;/em&gt; are only ever expressed as a product of readthrough of T&#x3D5;, indicated by the readthrough transcript. Only transcripts containing class-III genes are shown. Not all read-through products are shown. Redrawn from Dunn and Studier (1983).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/9/2957/F1.medium.gif" width="440" height="186"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/7/9/2957/F1.medium.gif" width="440" height="186"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/9/2957/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/9/2957/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/486502" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-30" class="first-child">T7 produces many polycistronic transcripts. The bar across the top shows the T7 genome, and each class is shown in a different color. Horizontal lines represent transcripts. Dashed vertical lines represent RNase cleavage sites, where R<em>3.8</em>, R<em>4.7</em>, R<em>6.5</em>, and R<em>18.5</em> are strong cleavage sites. R<em>13</em> is a weak RNase cleavage site. The solid vertical line represents the terminator Tϕ. Genes <em>11</em> and <em>12</em> are only ever expressed as a product of readthrough of Tϕ, indicated by the readthrough transcript. Only transcripts containing class-III genes are shown. Not all read-through products are shown. Redrawn from <a id="xref-ref-16-4" class="xref-bibr" href="#ref-16">Dunn and Studier (1983)</a>.</p><div class="sb-div caption-clear"></div></div></div><p id="p-31">Because the wild-type phage lyses the cell at ∼11 min after infection (<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Heineman and Bull 2007</a>; <a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Bull <em>et al.</em> 2011</a>), the proteome of infected hosts was sampled at 1, 5, and 9 min after phage addition to the culture; infection of cells is neither immediate nor synchronous upon phage addition to the culture, so these times are approximate postinfection values. By 9 min after infection, ∼50 of the known or predicted T7 proteins were detected. All samples recovered ∼4000 <em>E. coli</em> proteins. Since proteins have a much longer half-life than transcripts, and T7 has no known mechanism of degrading <em>E. coli</em> proteins (<a id="xref-ref-28-3" class="xref-bibr" href="#ref-28">Molineux 2006</a>), we assumed that <em>E. coli</em> protein abundances remained constant over the 9 min infection, and normalized the phage protein abundances to that of <em>E. coli</em> (<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Houser <em>et al.</em> 2015</a>). Thus, we report all phage protein abundances as a proportion of <em>E. coli</em> protein content.</p><p id="p-32">Abundances of the major capsid protein (a product of gene <em>10</em>) were of primary interest, as gene <em>10</em> is the most highly expressed phage gene and is also the one deoptimized. Gene <em>10</em> comprises two protein products: the major capsid protein (gp10A) and the much less abundant minor capsid protein (gp10B). The minor capsid protein is produced after a frameshift and stop codon readthrough of <em>10A</em> and, except for the C-terminal ∼53 amino acids, is identical in sequence to the major capsid protein. Thus, our proteomic methods have limited ability to distinguish between the two protein products, so we combine abundance estimates into a single capsid protein measurement (see <em>Materials and Methods</em>). By 9 min after infection, capsid protein abundances in the attenuated strain were about half of those in the wild type (<span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2957/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2957/embed/mml-math-34.gif"/></noscript></span></span> paired <em>t</em>-test; <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). The capsid protein abundance for the evolved strain was intermediate.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/9/2957/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Recoding gene 10 reduces capsid protein abundances. We measured protein abundance at 1, 5, and 9 min after infection. In the recoded (attenuated) strain, protein abundance for capsid protein after 9 min of infection is half of that of the wild type ( paired t-test). The evolved strain also has significantly lower levels of capsid protein after 9 min. Each point represents a single measurement, and lines connect biological replicates." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-180513883" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Recoding gene &lt;em&gt;10&lt;/em&gt; reduces capsid protein abundances. We measured protein abundance at 1, 5, and 9 min after infection. In the recoded (attenuated) strain, protein abundance for capsid protein after 9 min of infection is half of that of the wild type (&lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-35&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; paired &lt;em&gt;t&lt;/em&gt;-test). The evolved strain also has significantly lower levels of capsid protein after 9 min. Each point represents a single measurement, and lines connect biological replicates.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/9/2957/F2.medium.gif" width="440" height="215"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/7/9/2957/F2.medium.gif" width="440" height="215"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/9/2957/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/9/2957/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/486574" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-33" class="first-child">Recoding gene <em>10</em> reduces capsid protein abundances. We measured protein abundance at 1, 5, and 9 min after infection. In the recoded (attenuated) strain, protein abundance for capsid protein after 9 min of infection is half of that of the wild type (<span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2957/F2/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2957/F2/embed/mml-math-35.gif"/></noscript></span></span> paired <em>t</em>-test). The evolved strain also has significantly lower levels of capsid protein after 9 min. Each point represents a single measurement, and lines connect biological replicates.</p><div class="sb-div caption-clear"></div></div></div><p id="p-34">Recoding of gene <em>10</em> could have reduced capsid protein abundance by reducing rates of translation elongation, thereby increasing the likelihood of ribosome stalling and fall-off. If ribosome fall-off were the dominant mechanism by which protein abundance was reduced, attenuation should have been accompanied by an excess of short peptides from the N-terminal end. Alternatively, if translation was slowed down without ribosome fall-off, a uniform distribution of peptides should be observed across the capsid protein. When mapping individual peptides recovered from the mass spectrometry proteomics, no systematic change was observed in the distribution of peptides across the protein (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). Thus, the recoded phage strains produced complete capsid protein, but in smaller quantities than that of the wild type.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/9/2957/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Peptide abundances are uniformly distributed across the capsid protein in the recoded T7 strain. Shown are the abundances of individual peptides within the capsid protein of the recoded strain, relative to the wild type. Data are from 9 min after infection. If translation consistently terminated before the stop codon in recoded gene 10, we would expect the relative peptide abundance to systematically vary with the location of the peptide. Therefore, we tested a model that includes an interaction between strain and peptide location, and a model that includes no interaction (see Materials and Methods). We found that a model in which an interaction is included between the strain and peptide location fits the peptide count data no better than a model without interaction (  respectively). Thus, in the recoded strain, there is no evidence of early translation termination." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-180513883" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Peptide abundances are uniformly distributed across the capsid protein in the recoded T7 strain. Shown are the abundances of individual peptides within the capsid protein of the recoded strain, relative to the wild type. Data are from 9 min after infection. If translation consistently terminated before the stop codon in recoded gene &lt;em&gt;10&lt;/em&gt;, we would expect the relative peptide abundance to systematically vary with the location of the peptide. Therefore, we tested a model that includes an interaction between strain and peptide location, and a model that includes no interaction (see &lt;em&gt;Materials and Methods&lt;/em&gt;). We found that a model in which an interaction is included between the strain and peptide location fits the peptide count data no better than a model without interaction (&lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-36&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-37&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; respectively). Thus, in the recoded strain, there is no evidence of early translation termination.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/9/2957/F3.medium.gif" width="440" height="397"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/7/9/2957/F3.medium.gif" width="440" height="397"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/9/2957/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/9/2957/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/486596" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-35" class="first-child">Peptide abundances are uniformly distributed across the capsid protein in the recoded T7 strain. Shown are the abundances of individual peptides within the capsid protein of the recoded strain, relative to the wild type. Data are from 9 min after infection. If translation consistently terminated before the stop codon in recoded gene <em>10</em>, we would expect the relative peptide abundance to systematically vary with the location of the peptide. Therefore, we tested a model that includes an interaction between strain and peptide location, and a model that includes no interaction (see <em>Materials and Methods</em>). We found that a model in which an interaction is included between the strain and peptide location fits the peptide count data no better than a model without interaction (<span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2957/F3/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2957/F3/embed/mml-math-36.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2957/F3/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2957/F3/embed/mml-math-37.gif"/></noscript></span></span> respectively). Thus, in the recoded strain, there is no evidence of early translation termination.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-14" class="subsection"><h3>Codon deoptimization reduced some other class-III protein abundances</h3><p id="p-36">If recoding a highly expressed gene saturates the pool of ribosomes by slowing translation of the recoded gene, the rate of translation of all T7 genes could ultimately decline (model 2, above). Models of T7 replication in <em>E. coli</em>, and limited experimental data, are consistent with protein synthesis being the rate-limiting step of T7 replication (although the evidence is at best weak and indirect) (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Endy <em>et al.</em> 1997</a>; <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">You <em>et al.</em> 2002</a>). Moreover, depletion of free ribosomes is common in <em>E. coli</em> transformed with highly expressed heterologous genes (<a id="xref-ref-52-2" class="xref-bibr" href="#ref-52">Vind <em>et al.</em> 1993</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Scott <em>et al.</em> 2010</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Reuveni <em>et al.</em> 2011</a>; <a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Raveh <em>et al.</em> 2016</a>).</p><p id="p-37">The data allow us to measure other T7 protein abundances over time. 9 min postinfection, abundances of class-III proteins gp11, gp12, gp13, gp14, and gp15 in the recoded strain were lower than that of the wild type [false discovery rate (FDR) &lt; 0.1, FDR-corrected <em>t</em>-test] (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a> and <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). These proteins are mostly or all structural: tail tubular proteins (gp11 and gp12), two internal virion proteins (gp14 and gp15), and a protein of unknown function but required for incorporation of other essential proteins in the virion (gp13). The T7 endonuclease (gp3), encoded by a class-II gene, also showed reduced abundance in the recoded strain. Abundances of these class-III proteins in the evolved strain again fall somewhere between that of the wild-type and recoded strains. Together, these results demonstrate that recoding gene <em>10</em> reduced the protein abundances of the capsid protein and several proteins encoded downstream of the capsid protein. The false discovery threshold we employed allows for one or two of those downstream proteins identified to be false positives, but the majority are likely real.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/9/2957/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Recoding of gene 10 reduces abundances of capsid protein (gp10), of five other proteins encoded immediately downstream of 10 (gp11&#x2013;15), and of T7 endonuclease (gp3). The bacteriophage T7 genome contains many polycistronic transcripts. Genes 11 and 12 are always transcribed with gene 10, following a read through of the T&#x3D5; terminator. Genes 13, 14, and 15 may also share the same transcript as gene 10, although these will be less common because of an RNase cleavage site between genes 12 and 13, and a promoter before gene 13. The panels show the relative protein abundances corresponding to these genes, in addition to gene 3, for all wild-type, recoded, and evolved strains at the 9-min time point. All of these genes, with the exception of gene 3, are class-III genes, expressed late in the T7 life cycle. In addition to lower abundance of capsid protein (gp10), all protein products from the five genes immediately downstream of 10 are also suppressed ( FDR-corrected paired t-test). These proteins are tail tubular proteins (gp11 and gp12), probable virion-associated protein (gp13), and two internal virion proteins (gp14 and gp15). Gene 3, which codes for the T7 endonuclease, also has a reduced abundance in the recoded strain. Protein abundances for the evolved strain fall somewhere between that of the wild-type and the recoded strains. Each point represents a single measurement, and lines connect biological replicates." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-180513883" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Recoding of gene &lt;em&gt;10&lt;/em&gt; reduces abundances of capsid protein (gp10), of five other proteins encoded immediately downstream of &lt;em&gt;10&lt;/em&gt; (gp11&#x2013;15), and of T7 endonuclease (gp3). The bacteriophage T7 genome contains many polycistronic transcripts. Genes &lt;em&gt;11&lt;/em&gt; and &lt;em&gt;12&lt;/em&gt; are always transcribed with gene &lt;em&gt;10&lt;/em&gt;, following a read through of the T&#x3D5; terminator. Genes &lt;em&gt;13&lt;/em&gt;, &lt;em&gt;14&lt;/em&gt;, and &lt;em&gt;15&lt;/em&gt; may also share the same transcript as gene &lt;em&gt;10&lt;/em&gt;, although these will be less common because of an RNase cleavage site between genes &lt;em&gt;12&lt;/em&gt; and &lt;em&gt;13&lt;/em&gt;, and a promoter before gene &lt;em&gt;13&lt;/em&gt;. The panels show the relative protein abundances corresponding to these genes, in addition to gene &lt;em&gt;3&lt;/em&gt;, for all wild-type, recoded, and evolved strains at the 9-min time point. All of these genes, with the exception of gene &lt;em&gt;3&lt;/em&gt;, are class-III genes, expressed late in the T7 life cycle. In addition to lower abundance of capsid protein (gp10), all protein products from the five genes immediately downstream of &lt;em&gt;10&lt;/em&gt; are also suppressed (&lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-38&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; FDR-corrected paired &lt;em&gt;t&lt;/em&gt;-test). These proteins are tail tubular proteins (gp11 and gp12), probable virion-associated protein (gp13), and two internal virion proteins (gp14 and gp15). Gene &lt;em&gt;3&lt;/em&gt;, which codes for the T7 endonuclease, also has a reduced abundance in the recoded strain. Protein abundances for the evolved strain fall somewhere between that of the wild-type and the recoded strains. Each point represents a single measurement, and lines connect biological replicates.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/9/2957/F4.medium.gif" width="339" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/7/9/2957/F4.medium.gif" width="339" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/9/2957/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/9/2957/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/486476" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-38" class="first-child">Recoding of gene <em>10</em> reduces abundances of capsid protein (gp10), of five other proteins encoded immediately downstream of <em>10</em> (gp11–15), and of T7 endonuclease (gp3). The bacteriophage T7 genome contains many polycistronic transcripts. Genes <em>11</em> and <em>12</em> are always transcribed with gene <em>10</em>, following a read through of the Tϕ terminator. Genes <em>13</em>, <em>14</em>, and <em>15</em> may also share the same transcript as gene <em>10</em>, although these will be less common because of an RNase cleavage site between genes <em>12</em> and <em>13</em>, and a promoter before gene <em>13</em>. The panels show the relative protein abundances corresponding to these genes, in addition to gene <em>3</em>, for all wild-type, recoded, and evolved strains at the 9-min time point. All of these genes, with the exception of gene <em>3</em>, are class-III genes, expressed late in the T7 life cycle. In addition to lower abundance of capsid protein (gp10), all protein products from the five genes immediately downstream of <em>10</em> are also suppressed (<span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2957/F4/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2957/F4/embed/mml-math-38.gif"/></noscript></span></span> FDR-corrected paired <em>t</em>-test). These proteins are tail tubular proteins (gp11 and gp12), probable virion-associated protein (gp13), and two internal virion proteins (gp14 and gp15). Gene <em>3</em>, which codes for the T7 endonuclease, also has a reduced abundance in the recoded strain. Protein abundances for the evolved strain fall somewhere between that of the wild-type and the recoded strains. Each point represents a single measurement, and lines connect biological replicates.</p><div class="sb-div caption-clear"></div></div></div><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/486417/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/486417/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Proteins in which abundance differs significantly (FDR <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2957/T1/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2957/T1/embed/mml-math-39.gif"/></noscript></span></span> FDR-corrected paired <em>t</em>-test) between the wild-type and recoded strains of T7</span><div class="sb-div caption-clear"></div></div></div><p id="p-40">Whereas proteins encoded downstream of the recoded gene <em>10</em> showed decreased abundances, those encoded immediately upstream of gene <em>10</em> were not obviously affected. Gene <em>9</em>, immediately upstream of <em>10</em>, encodes the highly expressed scaffold protein but this protein showed no difference in abundance between wild-type and recoded strains (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>). Under the ribosome-depletion hypothesis (model 2), we expected all genes expressed at the same time as gene <em>10</em> to be suppressed, thus including gene <em>9</em>. Indeed, because of the high levels of expression of <em>9</em> and the consequent ease of measuring it with our proteomics methods, we reject the ribosome-depletion model as the basis of attenuation.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/9/2957/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Gene 9, encoding the highly expressed scaffold protein, shows no detectable difference in protein abundance between wild-type and recoded strains. Genes 9 (left) and 10 (right) are both class-III genes that are expressed at approximately the same time in the T7 life cycle. All abundances are from 9 min postinfection. Each point represents a single measurement, and lines connect biological replicates." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-180513883" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Gene &lt;em&gt;9&lt;/em&gt;, encoding the highly expressed scaffold protein, shows no detectable difference in protein abundance between wild-type and recoded strains. Genes &lt;em&gt;9&lt;/em&gt; (left) and &lt;em&gt;10&lt;/em&gt; (right) are both class-III genes that are expressed at approximately the same time in the T7 life cycle. All abundances are from 9 min postinfection. Each point represents a single measurement, and lines connect biological replicates.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/9/2957/F5.medium.gif" width="440" height="240"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/7/9/2957/F5.medium.gif" width="440" height="240"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/9/2957/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/9/2957/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/486385" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-41" class="first-child">Gene <em>9</em>, encoding the highly expressed scaffold protein, shows no detectable difference in protein abundance between wild-type and recoded strains. Genes <em>9</em> (left) and <em>10</em> (right) are both class-III genes that are expressed at approximately the same time in the T7 life cycle. All abundances are from 9 min postinfection. Each point represents a single measurement, and lines connect biological replicates.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-15" class="subsection"><h3>E. coli mRNA transcripts declined after bacteriophage T7 infection</h3><p id="p-42">A reduction in capsid protein abundance in the attenuated virus could be explained by fewer transcripts from the gene. Of course, a reduction in transcription is not expected from a change in codon usage, and indeed, the design left the 5′ end of the gene unaltered specifically to keep transcription and translation initiation unchanged. Nonetheless, RNA sequences were obtained from phage-infected <em>E. coli</em> at 1, 5, and 9 min after infection to see if the decline in protein was accompanied by a decline in transcripts. Transcript abundances were normalized to the total RNA abundance within a sample, excluding rRNA and tRNA. Both phage and <em>E. coli</em> transcripts were included.</p><p id="p-43">Gene <em>10</em> transcript abundance increased over time, being highest at the final 9-min time point. Yet no significant heterogeneity was observed among the wild-type, recoded, and evolved strain transcripts of gene <em>10</em> (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6A</a>). This observation seems to rule out a transcriptional cause of capsid protein reduction.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/9/2957/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Transcript abundances of individual T7 genes change over time. Three biological replicates each are shown for wild-type, attenuated, and evolved strains. Transcript abundances are presented relative to the total mRNA content in the sample. Each column represents a different time point after infection. Both gene 10 and gene 1 follow the expected expression patterns of class-I and class-III genes, respectively. Each point represents a single measurement, and lines connect biological replicates. (A) Transcripts of gene 10 increase over time. There are no detectable differences in gene 10 mRNA abundances between the attenuated and wild-type T7 strains. (B) Transcripts of gene 1 increase from 1 to 5 min, then decrease from 5 to 9 min. Again, there are no detectable differences in gene 1 mRNA abundances between the attenuated and wild-type T7 strains." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-180513883" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Transcript abundances of individual T7 genes change over time. Three biological replicates each are shown for wild-type, attenuated, and evolved strains. Transcript abundances are presented relative to the total mRNA content in the sample. Each column represents a different time point after infection. Both gene &lt;em&gt;10&lt;/em&gt; and gene &lt;em&gt;1&lt;/em&gt; follow the expected expression patterns of class-I and class-III genes, respectively. Each point represents a single measurement, and lines connect biological replicates. (A) Transcripts of gene &lt;em&gt;10&lt;/em&gt; increase over time. There are no detectable differences in gene &lt;em&gt;10&lt;/em&gt; mRNA abundances between the attenuated and wild-type T7 strains. (B) Transcripts of gene &lt;em&gt;1&lt;/em&gt; increase from 1 to 5 min, then decrease from 5 to 9 min. Again, there are no detectable differences in gene &lt;em&gt;1&lt;/em&gt; mRNA abundances between the attenuated and wild-type T7 strains.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/9/2957/F6.medium.gif" width="440" height="326"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/7/9/2957/F6.medium.gif" width="440" height="326"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/9/2957/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/9/2957/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/486568" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-44" class="first-child">Transcript abundances of individual T7 genes change over time. Three biological replicates each are shown for wild-type, attenuated, and evolved strains. Transcript abundances are presented relative to the total mRNA content in the sample. Each column represents a different time point after infection. Both gene <em>10</em> and gene <em>1</em> follow the expected expression patterns of class-I and class-III genes, respectively. Each point represents a single measurement, and lines connect biological replicates. (A) Transcripts of gene <em>10</em> increase over time. There are no detectable differences in gene <em>10</em> mRNA abundances between the attenuated and wild-type T7 strains. (B) Transcripts of gene <em>1</em> increase from 1 to 5 min, then decrease from 5 to 9 min. Again, there are no detectable differences in gene <em>1</em> mRNA abundances between the attenuated and wild-type T7 strains.</p><div class="sb-div caption-clear"></div></div></div><p id="p-45">As validation of the RNA-sequencing methods, transcription analyses were extended to other properties of the T7 and <em>E. coli</em> transcriptomes. Gene <em>1</em> (T7 RNAP) transcript abundance decreased over time as a proportion of total transcripts (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6B</a>), consistent with previously observed class-I gene expression timing (<a id="xref-ref-28-4" class="xref-bibr" href="#ref-28">Molineux 2006</a>). Likewise, the relative abundance of T7 transcripts to <em>E. coli</em> increased sharply, consistent with established mechanisms by which T7 shuts off host transcription and degrades the host genome (<a id="xref-ref-28-5" class="xref-bibr" href="#ref-28">Molineux 2006</a>). At 1 min after infection, T7 transcripts comprised &lt;1% of the transcript pool of infected <em>E. coli</em> (<a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7</a>). By 5 min after infection, T7 transcripts made up &gt;75% of the transcript pool. By 9 min postinfection, this proportion reached ∼95%. Although it is not possible to assess changes in absolute transcript abundances (see <em>Materials and Methods</em>), the data require some combination of host-cell transcripts being degraded rapidly, or T7 synthesizing transcripts so rapidly that they quickly dwarf the pool of <em>E. coli</em> transcripts. No strain-specific trends were detected.</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/9/2957/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="T7 transcripts increase relative to E. coli transcripts over time. T7 transcripts ultimately constitute most of the mRNA content of infected E. coli. The top row shows total E. coli transcript abundances, and the bottom row shows total T7 abundances. Points show individual experiments, while lines connect measurements from the same biological replicate. Each bar represents the mean relative mRNA abundance for a given strain and time point. Transcript abundances are shown relative to the total mRNA content of the sample. Wild-type, recoded, and evolved strains are shown, and there are no detectable differences between the three strains." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-180513883" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;T7 transcripts increase relative to &lt;em&gt;E. coli&lt;/em&gt; transcripts over time. T7 transcripts ultimately constitute most of the mRNA content of infected &lt;em&gt;E. coli&lt;/em&gt;. The top row shows total &lt;em&gt;E. coli&lt;/em&gt; transcript abundances, and the bottom row shows total T7 abundances. Points show individual experiments, while lines connect measurements from the same biological replicate. Each bar represents the mean relative mRNA abundance for a given strain and time point. Transcript abundances are shown relative to the total mRNA content of the sample. Wild-type, recoded, and evolved strains are shown, and there are no detectable differences between the three strains.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/9/2957/F7.medium.gif" width="440" height="271"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/7/9/2957/F7.medium.gif" width="440" height="271"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/9/2957/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/9/2957/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/486583" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-46" class="first-child">T7 transcripts increase relative to <em>E. coli</em> transcripts over time. T7 transcripts ultimately constitute most of the mRNA content of infected <em>E. coli</em>. The top row shows total <em>E. coli</em> transcript abundances, and the bottom row shows total T7 abundances. Points show individual experiments, while lines connect measurements from the same biological replicate. Each bar represents the mean relative mRNA abundance for a given strain and time point. Transcript abundances are shown relative to the total mRNA content of the sample. Wild-type, recoded, and evolved strains are shown, and there are no detectable differences between the three strains.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-16" class="subsection"><h3>Downstream effects of recoding support a model of translational coupling</h3><p id="p-47">The proteomics suggest that translation of some downstream genes are specifically depressed by the recoding of <em>10</em>. This effect on downstream genes might involve translational coupling, in which a stalling of translation over <em>10</em> delays translation of genes further down on the same transcript. Translational coupling often occurs when multiple genes are encoded on a single transcript with little intergenic space, such as in bacterial operons and viruses (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Lesage <em>et al.</em> 1992</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Hellmuth <em>et al.</em> 1991</a>; <a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Schümperli <em>et al.</em> 1982</a>; <a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Oppenheim and Yanofsky 1980</a>; <a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Aksoy <em>et al.</em> 1984</a>; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Torgov <em>et al.</em> 1998</a>). Translational coupling is a plausible process for sets of T7 genes because most transcripts include multiple genes (<a id="xref-ref-16-5" class="xref-bibr" href="#ref-16">Dunn and Studier 1983</a>). The T7 class-III promoters precede genes <em>6.5</em> (the first class-III gene), <em>9</em>, <em>10</em>, <em>13</em>, and <em>17</em>; some transcripts with <em>9</em> and <em>10</em> will thus include earlier genes, but many will not (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>). Although a phage-specific terminator between <em>10</em> and <em>11</em> aborts most (but clearly not all) <em>10</em> transcripts before <em>11</em>, all transcripts with <em>11</em> and <em>12</em> necessarily include <em>10</em>. Thus, translational coupling would operate for <em>11</em> and <em>12</em> if many of the ribosomes on those genes first translated <em>10</em>. Translational coupling beyond <em>12</em> is less plausible, however. The combination of an RNAse-III site between <em>12</em> and <em>13</em> and a promoter before <em>13</em> will mean that many or most transcripts with <em>13</em> do not include <em>12</em>. So we expect substantially higher levels of translational coupling of <em>11</em> and <em>12</em> with <em>10</em>, but far less between <em>10</em> and <em>13</em>.</p><p id="p-48">Polycistronic transcripts are necessary for translational coupling, but not sufficient. It must also be the case that downstream genes contain structured, inaccessible RBSs that only initiate translation in the presence of upstream translating ribosomes (<a id="xref-ref-48-3" class="xref-bibr" href="#ref-48">Tian and Salis 2015</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Rex <em>et al.</em> 1994</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Qu <em>et al.</em> 2011</a>). Ribosomes that reach the stop codon of one gene often expose the RBS of the next gene on the transcript and then reinitiate translation on that downstream gene (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Spanjaard and van Duin 1989</a>). Moreover, this exposure of the RBS also facilitates binding from the free ribosome pool (<a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Rex <em>et al.</em> 1994</a>). Support for the coupling model was evaluated from secondary structure predictions and <em>in silico</em> predictions of ribosome binding (<a id="xref-ref-48-4" class="xref-bibr" href="#ref-48">Tian and Salis 2015</a>). Due to limitations of software-based methods, we only tested the coupling of genes <em>10</em> and <em>11</em>. With translational coupling, translation initiation rates are predicted to be ∼6 times greater for gene <em>11</em> than they would if gene <em>11</em> occurred on a single-gene transcript. This supports a model in which at least genes <em>10</em> and <em>11</em> are translationally coupled.</p><p id="p-49">As the inference of translational coupling here is indirect and tentative, additional insight was sought from a mathematical model. The model assumed three genes on a single transcript in the order <em>a</em>, <em>b</em>, and <em>c</em> (<a id="xref-fig-8-1" class="xref-fig" href="#F8">Figure 8A</a>, Equations 3–14 in <em>Materials and Methods</em>). All genes were assumed to be the same length. In the model, protein production rates (<em>i.e.</em>, the rate at which ribosomes complete translation) depended on translation initiation and translation elongation rates. The model allowed varying degrees of coupling, with a coupling constant <em>y</em>, which accounts for both ribosome reinitiation and facilitated binding. The model also allowed for translation initiation independent of any coupling effects, given by <em>w</em>. (In the results presented here, <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2957/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2957/embed/mml-math-40.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2957/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2957/embed/mml-math-41.gif"/></noscript></span></span> See <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a> and <em>Materials and Methods</em> for all parameters). In a system with no coupling, translation initiation of gene <em>b</em> did not depend on translation of gene <em>a</em>, and likewise between genes <em>b</em> and <em>c</em>. In a strongly coupled system, translation initiation of gene <em>b</em> depended almost entirely on the rate at which ribosomes complete translation of gene <em>a</em>. Translation of gene <em>c</em> similarly depended on gene <em>b</em> in a strongly coupled system. Thus, we explored how relative rates of protein production depend on both coupling between genes and elongation rates within genes.</p><div id="F8" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/9/2957/F8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="When genes are translationally coupled, the translation rate of upstream genes can affect translation rates of downstream genes. (A) Model of translational coupling. We assume three genes (a, b, and c) are expressed in a polycistronic transcript and are translationally coupled, such that the translation initiation rates of one gene depend partially on the rate of ribosomes reaching the stop codon of the previous gene. Typically, translation initiation is the rate-limiting step in translation. All genes are assumed to be the same length. Translation initiation of gene a is given as  and the translation elongation rates of genes a, b, and c, are given by   and , respectively. The proportion of ribosomes initiating due to upstream translation activity (coupling constant) is given as  and  for genes b and c, respectively. The rate of de novo initiation is given by  and  (B) Recoding gene b decreases the rate of translation elongation. Upon recoding gene b, we hypothesize that ribosomes will accumulate on the gene b transcript and downstream ribosomal densities will decrease. (C) As translation elongation rates increase, the rate of translation of downstream genes also increases. The translation rate of b and c both increase as the translation elongation rate of gene b increases. When the elongation rate exceeds the initiation rate, initiation becomes rate limiting and protein production no longer increases. The left, middle, and right panels show models with coupling constants (y, where ) of 0.1, 0.5, and 1.2, respectively. A larger coupling constant indicates that downstream genes are less likely to initiate translation in the absence of upstream translating ribosomes. As coupling increases, production of protein C becomes more sensitive to translation elongation rates of gene b. The rows show different rates of independent translation initiation (w, where ). The greater the independent initiation rate, the lower the sensitivity of protein-C production to changes in translation elongation of gene b." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-180513883" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;When genes are translationally coupled, the translation rate of upstream genes can affect translation rates of downstream genes. (A) Model of translational coupling. We assume three genes (&lt;em&gt;a&lt;/em&gt;, &lt;em&gt;b&lt;/em&gt;, and &lt;em&gt;c&lt;/em&gt;) are expressed in a polycistronic transcript and are translationally coupled, such that the translation initiation rates of one gene depend partially on the rate of ribosomes reaching the stop codon of the previous gene. Typically, translation initiation is the rate-limiting step in translation. All genes are assumed to be the same length. Translation initiation of gene &lt;em&gt;a&lt;/em&gt; is given as &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-42&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; and the translation elongation rates of genes &lt;em&gt;a&lt;/em&gt;, &lt;em&gt;b&lt;/em&gt;, and &lt;em&gt;c&lt;/em&gt;, are given by &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-43&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-44&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; and &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-45&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;, respectively. The proportion of ribosomes initiating due to upstream translation activity (coupling constant) is given as &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-46&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; and &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-47&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; for genes &lt;em&gt;b&lt;/em&gt; and &lt;em&gt;c&lt;/em&gt;, respectively. The rate of &lt;em&gt;de novo&lt;/em&gt; initiation is given by &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-48&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; and &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-49&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; (B) Recoding gene &lt;em&gt;b&lt;/em&gt; decreases the rate of translation elongation. Upon recoding gene &lt;em&gt;b&lt;/em&gt;, we hypothesize that ribosomes will accumulate on the gene &lt;em&gt;b&lt;/em&gt; transcript and downstream ribosomal densities will decrease. (C) As translation elongation rates increase, the rate of translation of downstream genes also increases. The translation rate of &lt;em&gt;b&lt;/em&gt; and &lt;em&gt;c&lt;/em&gt; both increase as the translation elongation rate of gene &lt;em&gt;b&lt;/em&gt; increases. When the elongation rate exceeds the initiation rate, initiation becomes rate limiting and protein production no longer increases. The left, middle, and right panels show models with coupling constants (&lt;em&gt;y&lt;/em&gt;, where &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-50&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;) of 0.1, 0.5, and 1.2, respectively. A larger coupling constant indicates that downstream genes are less likely to initiate translation in the absence of upstream translating ribosomes. As coupling increases, production of protein C becomes more sensitive to translation elongation rates of gene &lt;em&gt;b&lt;/em&gt;. The rows show different rates of independent translation initiation (&lt;em&gt;w&lt;/em&gt;, where &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-51&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;). The greater the independent initiation rate, the lower the sensitivity of protein-C production to changes in translation elongation of gene &lt;em&gt;b&lt;/em&gt;.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 8" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/9/2957/F8.medium.gif" width="425" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 8" src="https://www.g3journal.org/content/ggg/7/9/2957/F8.medium.gif" width="425" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/9/2957/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 8" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/9/2957/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/486482" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 8</span> <p id="p-50" class="first-child">When genes are translationally coupled, the translation rate of upstream genes can affect translation rates of downstream genes. (A) Model of translational coupling. We assume three genes (<em>a</em>, <em>b</em>, and <em>c</em>) are expressed in a polycistronic transcript and are translationally coupled, such that the translation initiation rates of one gene depend partially on the rate of ribosomes reaching the stop codon of the previous gene. Typically, translation initiation is the rate-limiting step in translation. All genes are assumed to be the same length. Translation initiation of gene <em>a</em> is given as <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2957/F8/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2957/F8/embed/mml-math-42.gif"/></noscript></span></span> and the translation elongation rates of genes <em>a</em>, <em>b</em>, and <em>c</em>, are given by <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2957/F8/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2957/F8/embed/mml-math-43.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2957/F8/embed/mml-math-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2957/F8/embed/mml-math-44.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2957/F8/embed/mml-math-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2957/F8/embed/mml-math-45.gif"/></noscript></span></span>, respectively. The proportion of ribosomes initiating due to upstream translation activity (coupling constant) is given as <span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2957/F8/embed/mml-math-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2957/F8/embed/mml-math-46.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2957/F8/embed/mml-math-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2957/F8/embed/mml-math-47.gif"/></noscript></span></span> for genes <em>b</em> and <em>c</em>, respectively. The rate of <em>de novo</em> initiation is given by <span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2957/F8/embed/mml-math-48.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2957/F8/embed/mml-math-48.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2957/F8/embed/mml-math-49.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2957/F8/embed/mml-math-49.gif"/></noscript></span></span> (B) Recoding gene <em>b</em> decreases the rate of translation elongation. Upon recoding gene <em>b</em>, we hypothesize that ribosomes will accumulate on the gene <em>b</em> transcript and downstream ribosomal densities will decrease. (C) As translation elongation rates increase, the rate of translation of downstream genes also increases. The translation rate of <em>b</em> and <em>c</em> both increase as the translation elongation rate of gene <em>b</em> increases. When the elongation rate exceeds the initiation rate, initiation becomes rate limiting and protein production no longer increases. The left, middle, and right panels show models with coupling constants (<em>y</em>, where <span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2957/F8/embed/mml-math-50.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2957/F8/embed/mml-math-50.gif"/></noscript></span></span>) of 0.1, 0.5, and 1.2, respectively. A larger coupling constant indicates that downstream genes are less likely to initiate translation in the absence of upstream translating ribosomes. As coupling increases, production of protein C becomes more sensitive to translation elongation rates of gene <em>b</em>. The rows show different rates of independent translation initiation (<em>w</em>, where <span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2957/F8/embed/mml-math-51.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2957/F8/embed/mml-math-51.gif"/></noscript></span></span>). The greater the independent initiation rate, the lower the sensitivity of protein-C production to changes in translation elongation of gene <em>b</em>.</p><div class="sb-div caption-clear"></div></div></div><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/486584/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/486584/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Parameters for three-gene model of translational coupling</span><div class="sb-div caption-clear"></div></div></div><p id="p-51">To simulate codon deoptimization of one gene in the transcript, we varied the translation elongation rate of gene <em>b</em> only (<a id="xref-fig-8-2" class="xref-fig" href="#F8">Figure 8</a>). We considered reductions in translation rate within a two- to threefold range. This reduction in translation elongation is compatible with prior studies of codon deoptimization in <em>E. coli</em> (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Kudla <em>et al.</em> 2009</a>). We also varied the coupling constant <em>y</em>. Under strong translational coupling (<span class="inline-formula" id="inline-formula-54"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2957/embed/mml-math-68.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2957/embed/mml-math-68.gif"/></noscript></span></span>), the translation initiation rates of a gene depended mostly on the rate of ribosomes moving through the stop codon of an upstream gene. Thus, the relative rate of protein A, B, and C production (assuming that initiation is slower than elongation in each gene) depended on the translation initiation rate of gene <em>a</em>. In turn, when the translation elongation rate was less than the initiation rate for gene <em>b</em>, translation elongation became rate limiting. Under these elongation rate-limited conditions, the production rates of proteins B and C only increased as the elongation rate of <em>b</em> increased, while A production rates remained unaffected (<a id="xref-fig-8-3" class="xref-fig" href="#F8">Figure 8C</a>, right panels). Conversely, in a weakly coupled model (<span class="inline-formula" id="inline-formula-55"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2957/embed/mml-math-69.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2957/embed/mml-math-69.gif"/></noscript></span></span>), even if elongation rates were slower than initiation rates in gene <em>b</em>, production of protein C was only weakly affected by recoding of gene <em>b</em> (<a id="xref-fig-8-4" class="xref-fig" href="#F8">Figure 8C</a>, left panels). In partially coupled models (<span class="inline-formula" id="inline-formula-56"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2957/embed/mml-math-70.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2957/embed/mml-math-70.gif"/></noscript></span></span> <a id="xref-fig-8-5" class="xref-fig" href="#F8">Figure 8B</a>, middle panels), the production rate of protein C also increased as elongation rates increased until surpassing initiation rates for gene <em>b</em>. However, this increase in the rate of C protein production was smaller than the increase observed on a strongly coupled model. Increasing the independent translation initiation rate <em>w</em> decreases the dependence of protein-C production on gene <em>b</em> elongation rates (<a id="xref-fig-8-6" class="xref-fig" href="#F8">Figure 8C</a>, top row to bottom row). This model demonstrated that recoding a gene in a translationally coupled set of genes affects the protein production rates of downstream, but not upstream, genes.</p></div><div id="sec-17" class="subsection"><h3>Connecting proteomics to fitness</h3><p id="p-52">In previous work, the T7 with a recoded capsid gene had been found to have a fitness of 35.7 doublings/hr, compared to a value of 43.2 for the wild type. Here we consider whether and how the altered proteomics might lead to this fitness reduction. The connection from proteomics to fitness spans two steps: (i) identify the phage life-history components affected by the recoding and evaluate whether that change is compatible with the proteomics, and then (ii) assess whether the magnitude of altered fitness components is compatible with overall fitness.</p><p id="p-53">In the growth conditions used for our assays, fitness is determined by cell density and three phage properties: burst size, lysis time, and adsorption rate (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Wang <em>et al.</em> 1996</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Guyader and Burch 2008</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Shao and Wang 2008</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Bull 2006</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Patwa and Wahl 2009</a>; <a id="xref-ref-8-3" class="xref-bibr" href="#ref-8">Bull <em>et al.</em> 2011</a>). Burst size refers to the average number of viral particles released from each infected cell. Burst sizes and lysis times were estimated here for the wild-type and recoded phages (there was no expectation that adsorption rate would be affected, which depends on the presence of tail fibers, the product of gene <em>17</em>). Although the same cell line was used here as in previous studies (IJ1133), new cell preparations were used, so quantitative agreement with past estimates of burst size and lysis time is not expected, but proportional differences should scale across different cell preparations. No difference in lysis time was observed between strains, but burst size was reduced almost 50% with the recoding (<a id="xref-fig-9-1" class="xref-fig" href="#F9">Figure 9</a>).</p><div id="F9" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/9/2957/F9.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Recoding affects burst size, but not lysis time. Burst size in recoded T7 strain is lower than that of the wild-type and evolved strains, while lysis time is indistinguishable for all three strains. (A) Five replicates each of burst-size measurements are shown for the wild-type, recoded, and evolved T7 strains. Points represent individual measurements in plaque forming units (PFUs), and each set of measurements in a replicate are connected by a line. Bars show the mean burst size for each strain. Burst size of the recoded strain is smaller than that of the evolved ( paired t-test) and wild-type ( paired t-test) strains. (B) Lysis time for the three different strains is shown. Points represent individual measurements, and bars represent the mean lysis time for each strain. There were no significant differences between the lysis times of each strain ( ANOVA)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-180513883" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Recoding affects burst size, but not lysis time. Burst size in recoded T7 strain is lower than that of the wild-type and evolved strains, while lysis time is indistinguishable for all three strains. (A) Five replicates each of burst-size measurements are shown for the wild-type, recoded, and evolved T7 strains. Points represent individual measurements in plaque forming units (PFUs), and each set of measurements in a replicate are connected by a line. Bars show the mean burst size for each strain. Burst size of the recoded strain is smaller than that of the evolved (&lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-71&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; paired &lt;em&gt;t&lt;/em&gt;-test) and wild-type (&lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-72&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; paired &lt;em&gt;t&lt;/em&gt;-test) strains. (B) Lysis time for the three different strains is shown. Points represent individual measurements, and bars represent the mean lysis time for each strain. There were no significant differences between the lysis times of each strain (&lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-73&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; ANOVA).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 9" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/9/2957/F9.medium.gif" width="440" height="215"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 9" src="https://www.g3journal.org/content/ggg/7/9/2957/F9.medium.gif" width="440" height="215"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/9/2957/F9.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 9" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/9/2957/F9.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/486458" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 9</span> <p id="p-54" class="first-child">Recoding affects burst size, but not lysis time. Burst size in recoded T7 strain is lower than that of the wild-type and evolved strains, while lysis time is indistinguishable for all three strains. (A) Five replicates each of burst-size measurements are shown for the wild-type, recoded, and evolved T7 strains. Points represent individual measurements in plaque forming units (PFUs), and each set of measurements in a replicate are connected by a line. Bars show the mean burst size for each strain. Burst size of the recoded strain is smaller than that of the evolved (<span class="inline-formula" id="inline-formula-57"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2957/F9/embed/mml-math-71.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2957/F9/embed/mml-math-71.gif"/></noscript></span></span> paired <em>t</em>-test) and wild-type (<span class="inline-formula" id="inline-formula-58"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2957/F9/embed/mml-math-72.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2957/F9/embed/mml-math-72.gif"/></noscript></span></span> paired <em>t</em>-test) strains. (B) Lysis time for the three different strains is shown. Points represent individual measurements, and bars represent the mean lysis time for each strain. There were no significant differences between the lysis times of each strain (<span class="inline-formula" id="inline-formula-59"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2957/F9/embed/mml-math-73.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/2957/F9/embed/mml-math-73.gif"/></noscript></span></span> ANOVA).</p><div class="sb-div caption-clear"></div></div></div><p id="p-55">The proportional reduction in burst size is nearly the same as that for the reduction in capsid protein abundance at 9 min. This reduction in burst size is no doubt caused by the reduction in capsid protein. There is perhaps little basis for arguing that the reductions should match quantitatively, but the agreement between the two numbers poses no dilemma.</p><p id="p-56">The second step in connecting proteomics with fitness is to consider whether a 50% reduction in burst size (with no change in lysis time) is compatible with the observed fitness reduction of 7.5 doublings/hr. The mapping of phage fitness components onto total fitness has been addressed in detail (<a id="xref-ref-8-4" class="xref-bibr" href="#ref-8">Bull <em>et al.</em> 2011</a>). From numerical trials in that study, a 50% reduction in burst size is compatible with a fitness reduction of the magnitude observed here (table 1, lines L1 and L5, in <a id="xref-ref-8-5" class="xref-bibr" href="#ref-8">Bull <em>et al.</em> 2011</a>).</p></div></div><div class="section discussion" id="sec-18"><h2 class="">Discussion</h2><p id="p-57">In the decade since the first proposals to attenuate viruses by synonymous codon substitutions, it has been established that the method works in many viruses and offers many advantages over earlier methods of attenuation. Yet the mechanism by which silent codon changes attenuate not only remains elusive but seems less clear now than it did at the start. Nor is it clear that a single mechanism underlies the attenuation in different systems. Here, we extended previous work on a bacteriophage system in which the encoding of rare codons in the major capsid gene reduced fitness. Our goal was to refine an understanding of the molecular basis of the attenuation.</p><p>The capsid protein (encoded by gene <em>10</em>) is the most abundant protein produced during the infection cycle of bacteriophage T7. Deoptimizing 50% of gene <em>10</em> codons reduced fitness (<a id="xref-ref-9-10" class="xref-bibr" href="#ref-9">Bull <em>et al.</em> 2012</a>). In exploring the underlying molecular mechanism by which the recoding has this effect, our primary result is that the protein product of gene <em>10</em> is reduced almost 50% by the end of the infection cycle, but protein abundance of genes immediately downstream of gene <em>10</em> are also depressed. The differences in protein abundance are not reflected in transcript levels, so it appears that the suppression of protein levels lies in translation. The evidence thus supports a simple interpretation of the fitness impact of recoding the major capsid gene:</p><ol class="list-ord " id="list-1"><li id="list-item-1"><p id="p-59">Capsid protein is expressed at a reduced level, as are a few downstream genes.</p></li><li id="list-item-2"><p id="p-60">Burst size is correspondingly reduced ∼50% with no change in lysis time, compatible with the observed reduction in total fitness.</p></li></ol><p>Taken together, these two observations indicate that the reduction of capsid protein abundance is a consequence of recoding and that this reduction is a plausible intracellular cause of viral attenuation.</p><p id="p-61">One mechanism we entertained to explain the altered proteomics of the recoded phage is saturation of the ribosomes with gene <em>10</em> transcripts. Such a model requires that the production of all T7 proteins would decline late in the infection cycle for the recoded phage. Within the limits of resolution, the proteomics rule out an overall reduction in T7 protein production, indicating that reductions are limited to the recoded <em>10</em> and a few downstream genes. This study may provide the first indication that translational effects of the recoding extend beyond the recoded genes. There was also no evidence for aborted gene <em>10</em> polypeptides in the recoded strains, as might occur from ribosomes stalled on gene <em>10</em> transcripts. An obvious next step is to extend these analyses to ribosome profiling, which would directly indicate whether the recoding does tie up ribosomes on gene <em>10</em> (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Li <em>et al.</em> 2014</a>).</p><p id="p-62">The means by which synonymous codon replacement attenuates may be more straightforward for phage T7 than for eukaryotic viruses. Several lines of evidence suggest that the eukaryotic virus attenuation by synonymous codon changes is from the creation of CpG dinucleotides (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Burns <em>et al.</em> 2009</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Atkinson <em>et al.</em> 2014</a>; <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Tulloch <em>et al.</em> 2014</a>); indeed, especially powerful evidence to support this interpretation is that evolutionary reversions of attenuated viruses disproportionately reverse CpGs (<a id="xref-ref-10-4" class="xref-bibr" href="#ref-10">Burns <em>et al.</em> 2006</a>). In contrast, T7 evolutionary reversions did not exhibit any signature suggestive of a dinucleotide basis for attenuation (<a id="xref-ref-9-11" class="xref-bibr" href="#ref-9">Bull <em>et al.</em> 2012</a>). Nonetheless, we expect that the mechanism of attenuation in T7 will apply across other viruses, even if it is not the only mechanism operating in those viruses.</p><p id="p-63">If our interpretation is correct for the mechanism underlying the effect of the recoding, an evolutionary response to overcome the effect might be duplication of the promoter immediately upstream of gene <em>10</em>. Such a duplication would increase the number of <em>10</em> transcripts and, in the absence of ribosome saturation, would increase the amount of capsid protein. A duplication of a class-II promoter was observed during adaptation of a different phage (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Springman <em>et al.</em> 2005</a>), adding credence to the possibility of such an outcome. Yet the mutational origin of a promoter duplication may be highly dependent on surrounding sequences, so perhaps not feasible for all promoters.</p><p id="p-64">We propose that translational coupling may explain why expression of genes downstream of <em>10</em> is suppressed by the recoding. Part of that inference is based on a mathematical model of translation. That model is necessarily simplified, however, and there are some obvious improvements needed to increase its realism. First, it assumes a fixed quantity of transcripts, when we know from our RNA-sequencing results that T7 transcripts increase rapidly during infection. Second, the model assumes a per-gene translation elongation rate, but does not model individual codons. A more sophisticated model that includes codon-level detail and the T7 life cycle would be needed to predict the fitness effects of codon deoptimization. Several life-cycle and molecular models of T7 have achieved limited success in predicting the phenotypic effects of genome manipulations (<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Endy <em>et al.</em> 1997</a>; <a id="xref-ref-58-2" class="xref-bibr" href="#ref-58">You <em>et al.</em> 2002</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Kosuri <em>et al.</em> 2007</a>; <a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Birch <em>et al.</em> 2012</a>), but none enable codon modifications or complex translation mechanisms such as coupling. The proteomics and RNA-sequencing data generated in this study should be useful in future high-resolution modeling studies that scale from the molecular level to that of viral fitness.</p><p id="p-65">Although it is tempting to interpret the slowed translation as a consequence of using rare codons, which then use rare tRNAs, some recoding strategies used in other genomes suggest alternative possibilities. Codon deoptimization of GFP in <em>E. coli</em> initially yielded a range of protein expression effects, but these effects were eventually attributed to changes in mRNA secondary structure in the first 28 codons of the GFP sequence (<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Kudla <em>et al.</em> 2009</a>). Codon changes beyond these first 28 codons had a weak effect on protein expression (<a id="xref-ref-25-3" class="xref-bibr" href="#ref-25">Kudla <em>et al.</em> 2009</a>). In our designs, we explicitly excluded these 5′-end codons from modification. Moreover, some attenuation designs with influenza virus, poliovirus, and arboviruses have achieved attenuation by merely shuffling codons within a gene or genome to create rare codon pairs (<a id="xref-ref-15-4" class="xref-bibr" href="#ref-15">Coleman <em>et al.</em> 2008</a>; <a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">Burns <em>et al.</em> 2009</a>; <a id="xref-ref-30-3" class="xref-bibr" href="#ref-30">Mueller <em>et al.</em> 2010</a>; <a id="xref-ref-31-4" class="xref-bibr" href="#ref-31">Nougairede <em>et al.</em> 2013</a>; <a id="xref-ref-44-3" class="xref-bibr" href="#ref-44">Shen <em>et al.</em> 2015</a>). Since the numbers of each codon are not changed in those designs, the mechanism cannot be one of simple tRNA abundance. One possibility is suggested by recent work in yeast and <em>Salmonella</em>, whereby the degree of codon clustering is important to rapid translation (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Cannarozzi <em>et al.</em> 2010</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Gamble <em>et al.</em> 2016</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Chevance and Hughes 2017</a>). We did not attempt to control for codon-pair bias in our recoded T7 constructs. Further experiments will be needed to determine if the <em>E. coli</em> translation machinery is sensitive to changes in codon-pair bias.</p><p id="p-66">The approach developed here should help elucidate other mechanisms of viral attenuation. For example, the timing of gene expression appears important to fitness: a reciprocal exchange of some middle and late genes had some major fitness effects, and those effects were not recovered on long-term adaptation (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Cecchini <em>et al.</em> 2013</a>). Ultimately, we envision a future in which an understanding of viral life history at the molecular level enables facile engineering of arbitrary fitness and alternative vaccine designs.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-67">We thank I. J. Molineux for advice on and insight into T7. This work was supported in part by National Institutes of Health grants R01 GM088344, National Science Foundation Cooperative agreement no. DBI-0939454 (BEACON Center), and Army Research Office grant W911NF-12-1-0390. The Texas Advanced Computing Center provided high-performance computing resources. J.J.B. is supported as the University of Texas Miescher Regents Professor.</p><p id="p-68">Author contributions: J.J.B., C.O.W., and D.R.B. conceived and designed the experiments and provided reagents and materials. D.R.B., M.L.P., and B.L.S. performed the experiments. B.R.J., D.R.B., M.L.P., and B.L.S. analyzed the data. B.R.J., C.O.W., and J.J.B. developed the model. J.J.B., C.O.W., D.R.B., M.L.P., B.L.S., and B.R.J. wrote the paper. The authors report no conflicts of interest.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-2"><p id="p-69"><em>Communicating editor: B. J. Andrews</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2017-03-02"><span class="received-label">Received </span>March 2, 2017.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2017-06-27"><span class="accepted-label">Accepted </span>June 27, 2017.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2017 Jack <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.2957.1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Aksoy</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Squires</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Squires</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">1984</span> <span class="cit-article-title">Translational coupling of the trpB and trpA genes in the <em>Escherichia coli</em> tryptophan operon.</span> <abbr class="cit-jnl-abbrev">J. Bacteriol.</abbr> <span class="cit-vol">157</span>: <span class="cit-fpage">363</span>–<span class="cit-lpage">367</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBacteriology%26rft.stitle%253DJ.%2BBacteriol.%26rft.aulast%253DAksoy%26rft.auinit1%253DS%26rft.volume%253D157%26rft.issue%253D2%26rft.spage%253D363%26rft.epage%253D367%26rft.atitle%253DTranslational%2Bcoupling%2Bof%2Bthe%2BtrpB%2Band%2BtrpA%2Bgenes%2Bin%2Bthe%2BEscherichia%2Bcoli%2Btryptophan%2Boperon.%26rft_id%253Dinfo%253Apmid%252F6319355%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6OToiMTU3LzIvMzYzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy85LzI5NTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.2957.2" data-doi="10.1093/nar/gku075"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Atkinson</span>  <span class="cit-name-given-names">N. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Witteveldt</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Evans</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Simmonds</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The influence of CpG and UpA dinucleotide frequencies on RNA virus replication and characterization of the innate cellular pathways underlying virus attenuation and enhanced replication.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">4527</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgku075%26rft_id%253Dinfo%253Apmid%252F24470146%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gku075&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24470146&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F2957.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.2957.3" data-doi="10.1093/nar/gks1193"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Barrett</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilhite</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ledoux</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Evangelista</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">I. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">NCBI GEO: archive for functional genomics data sets—update.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">D991</span>–<span class="cit-lpage">D995</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgks1193%26rft_id%253Dinfo%253Apmid%252F23193258%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gks1193&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23193258&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F2957.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000312893300141&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.2957.4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bates</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mächler</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bolker</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Walker</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Fitting linear mixed-effects models using lme4.</span> <abbr class="cit-jnl-abbrev">J. Stat. Softw.</abbr> <span class="cit-vol">67</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">48</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BStat.%2BSoftw.%26rft.volume%253D67%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.2957.5" data-doi="10.1371/journal.pcbi.1002746"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Birch</span>  <span class="cit-name-given-names">E. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruggero</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Covert</span>  <span class="cit-name-given-names">M. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Determining host metabolic limitations on viral replication via integrated modeling and experimental perturbation.</span> <abbr class="cit-jnl-abbrev">PLoS Comput. Biol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e1002746</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BComput%2BBiol%26rft.aulast%253DBirch%26rft.auinit1%253DE.%2BW.%26rft.volume%253D8%26rft.issue%253D10%26rft.spage%253De1002746%26rft.epage%253De1002746%26rft.atitle%253DDetermining%2BHost%2BMetabolic%2BLimitations%2Bon%2BViral%2BReplication%2Bvia%2BIntegrated%2BModeling%2Band%2BExperimental%2BPerturbation.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.1002746%26rft_id%253Dinfo%253Apmid%252F23093930%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.1002746&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23093930&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F2957.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.2957.6" data-doi="10.1038/nbt.3519"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bray</span>  <span class="cit-name-given-names">N. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pimentel</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Melsted</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pachter</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Near-optimal probabilistic RNA-seq quantification.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">525</span>–<span class="cit-lpage">527</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BBiotechnol.%26rft.volume%253D34%26rft.spage%253D525%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.3519%26rft_id%253Dinfo%253Apmid%252F27043002%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.3519&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27043002&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F2957.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.2957.7" data-doi="10.1016/j.jtbi.2006.01.027"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bull</span>  <span class="cit-name-given-names">J. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Optimality models of phage life history and parallels in disease evolution.</span> <abbr class="cit-jnl-abbrev">J. Theor. Biol.</abbr> <span class="cit-vol">241</span>: <span class="cit-fpage">928</span>–<span class="cit-lpage">938</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BTheoretical%2BBiology%26rft.stitle%253DJournal%2Bof%2BTheoretical%2BBiology%26rft.aulast%253DBull%26rft.auinit1%253DJ.%2BJ.%26rft.volume%253D241%26rft.issue%253D4%26rft.spage%253D928%26rft.epage%253D938%26rft.atitle%253DOptimality%2Bmodels%2Bof%2Bphage%2Blife%2Bhistory%2Band%2Bparallels%2Bin%2Bdisease%2Bevolution.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jtbi.2006.01.027%26rft_id%253Dinfo%253Apmid%252F16616205%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jtbi.2006.01.027&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16616205&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F2957.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000239842600020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.2957.8" data-doi="10.1371/journal.pone.0027796"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bull</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heineman</span>  <span class="cit-name-given-names">R. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilke</span>  <span class="cit-name-given-names">C. O.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The phenotype-fitness map in experimental evolution of phages.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e27796</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DBull%26rft.auinit1%253DJ.%2BJ.%26rft.volume%253D6%26rft.issue%253D11%26rft.spage%253De27796%26rft.epage%253De27796%26rft.atitle%253DThe%2Bphenotype-fitness%2Bmap%2Bin%2Bexperimental%2Bevolution%2Bof%2Bphages.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0027796%26rft_id%253Dinfo%253Apmid%252F22132144%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0027796&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22132144&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F2957.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.2957.9" data-doi="10.1093/molbev/mss119"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bull</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Molineux</span>  <span class="cit-name-given-names">I. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilke</span>  <span class="cit-name-given-names">C. O.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Slow fitness recovery in a codon-modified viral genome.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">2997</span>–<span class="cit-lpage">3004</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmss119%26rft_id%253Dinfo%253Apmid%252F22532576%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/mss119&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22532576&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F2957.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.2957.10" data-doi="10.1128/JVI.80.7.3259-3272.2006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Burns</span>  <span class="cit-name-given-names">C. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shaw</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Campagnoli</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jorba</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vincent</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Modulation of poliovirus replicative fitness in HeLa cells by deoptimization of synonymous codon usage in the capsid region.</span> <abbr class="cit-jnl-abbrev">J. Virol.</abbr> <span class="cit-vol">80</span>: <span class="cit-fpage">3259</span>–<span class="cit-lpage">3272</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DBurns%26rft.auinit1%253DC.%2BC.%26rft.volume%253D80%26rft.issue%253D7%26rft.spage%253D3259%26rft.epage%253D3272%26rft.atitle%253DModulation%2Bof%2BPoliovirus%2BReplicative%2BFitness%2Bin%2BHeLa%2BCells%2Bby%2BDeoptimization%2Bof%2BSynonymous%2BCodon%2BUsage%2Bin%2Bthe%2BCapsid%2BRegion%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.80.7.3259-3272.2006%26rft_id%253Dinfo%253Apmid%252F16537593%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6IjgwLzcvMzI1OSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvOS8yOTU3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.2957.11" data-doi="10.1128/JVI.00508-09"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Burns</span>  <span class="cit-name-given-names">C. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Campagnoli</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shaw</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vincent</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jorba</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genetic inactivation of poliovirus infectivity by increasing the frequencies of CpG and UpA dinucleotides within and across synonymous capsid region codons.</span> <abbr class="cit-jnl-abbrev">J. Virol.</abbr> <span class="cit-vol">83</span>: <span class="cit-fpage">9957</span>–<span class="cit-lpage">9969</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BVirol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.00508-09%26rft_id%253Dinfo%253Apmid%252F19605476%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI4My8xOS85OTU3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy85LzI5NTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.2957.12" data-doi="10.1016/j.cell.2010.02.036"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cannarozzi</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schraudolph</span>  <span class="cit-name-given-names">N. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Faty</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">von Rohr</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Friberg</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">A role for codon order in translation dynamics.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">141</span>: <span class="cit-fpage">355</span>–<span class="cit-lpage">367</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DCannarrozzi%26rft.auinit1%253DG.%26rft.volume%253D141%26rft.issue%253D2%26rft.spage%253D355%26rft.epage%253D367%26rft.atitle%253DA%2Brole%2Bfor%2Bcodon%2Border%2Bin%2Btranslation%2Bdynamics.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2010.02.036%26rft_id%253Dinfo%253Apmid%252F20403329%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2010.02.036&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20403329&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F2957.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000276738400024&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.2957.13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cecchini</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schmerer</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Molineux</span>  <span class="cit-name-given-names">I. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Springman</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bull</span>  <span class="cit-name-given-names">J. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Evolutionarily stable attenuation by genome rearrangement in a virus.</span> <abbr class="cit-jnl-abbrev">G3</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">1389</span>–<span class="cit-lpage">1397</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%26rft.volume%253D3%26rft.spage%253D1389%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.2957.14" data-doi="10.1073/pnas.1614896114"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chevance</span>  <span class="cit-name-given-names">F. F. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hughes</span>  <span class="cit-name-given-names">K. T.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Case for the genetic code as a triplet of triplets.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">114</span>: <span class="cit-fpage">4745</span>–<span class="cit-lpage">4750</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1614896114%26rft_id%253Dinfo%253Apmid%252F28416671%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTE0LzE4LzQ3NDUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzkvMjk1Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.2957.15" data-doi="10.1126/science.1155761"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Coleman</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Papamichail</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Skiena</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Futcher</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wimmer</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Virus attenuation by genome-scale changes in codon pair bias.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">320</span>: <span class="cit-fpage">1784</span>–<span class="cit-lpage">1787</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DColeman%26rft.auinit1%253DJ.%2BR.%26rft.volume%253D320%26rft.issue%253D5884%26rft.spage%253D1784%26rft.epage%253D1787%26rft.atitle%253DVirus%2BAttenuation%2Bby%2BGenome-Scale%2BChanges%2Bin%2BCodon%2BPair%2BBias%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1155761%26rft_id%253Dinfo%253Apmid%252F18583614%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMjAvNTg4NC8xNzg0IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy85LzI5NTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.2957.16" data-doi="10.1016/0022-2836(83)90062-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dunn</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Studier</span>  <span class="cit-name-given-names">F. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1983</span> <span class="cit-article-title">Complete nucleotide sequence of bacteriophage T7 DNA and the locations of T7 genetic elements.</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">166</span>: <span class="cit-fpage">477</span>–<span class="cit-lpage">535</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DDunn%26rft.auinit1%253DJ.%2BJ.%26rft.volume%253D166%26rft.issue%253D4%26rft.spage%253D477%26rft.epage%253D535%26rft.atitle%253DComplete%2Bnucleotide%2Bsequence%2Bof%2Bbacteriophage%2BT7%2BDNA%2Band%2Bthe%2Blocations%2Bof%2BT7%2Bgenetic%2Belements.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0022-2836%252883%252990062-1%26rft_id%253Dinfo%253Apmid%252F6864790%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0022-2836(83)90062-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=6864790&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F2957.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1983QW37900002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.2957.17" data-doi="10.1002/(SICI)1097-0290(19970720)55:2&lt;375::AID-BIT15&gt;3.0.CO;2-G"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Endy</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kong</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yin</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Intracellular kinetics of a growing virus: a genetically structured simulation for bacteriophage T7.</span> <abbr class="cit-jnl-abbrev">Biotechnol. Bioeng.</abbr> <span class="cit-vol">55</span>: <span class="cit-fpage">375</span>–<span class="cit-lpage">389</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiotechnology%2Band%2BBioengineering%26rft.stitle%253DBiotechnology%2Band%2BBioengineering%26rft.aulast%253DEndy%26rft.auinit1%253DD.%26rft.volume%253D55%26rft.issue%253D2%26rft.spage%253D375%26rft.epage%253D389%26rft.atitle%253DIntracellular%2Bkinetics%2Bof%2Ba%2Bgrowing%2Bvirus%253A%2Ba%2Bgenetically%2Bstructured%2Bsimulation%2Bfor%2Bbacteriophage%2BT7.%26rft_id%253Dinfo%253Adoi%252F10.1002%252F%2528SICI%25291097-0290%252819970720%252955%253A2%253C375%253A%253AAID-BIT15%253E3.0.CO%253B2-G%26rft_id%253Dinfo%253Apmid%252F18636496%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/(SICI)1097-0290(19970720)55:2&lt;375::AID-BIT15&gt;3.0.CO;2-G&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18636496&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F2957.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1997XH85500015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.2957.18" data-doi="10.1016/j.cell.2010.03.033"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fredrick</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ibba</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">How the sequence of a gene can tune its translation.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">141</span>: <span class="cit-fpage">227</span>–<span class="cit-lpage">229</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DFredrick%26rft.auinit1%253DK.%26rft.volume%253D141%26rft.issue%253D2%26rft.spage%253D227%26rft.epage%253D229%26rft.atitle%253DHow%2Bthe%2Bsequence%2Bof%2Ba%2Bgene%2Bcan%2Btune%2Bits%2Btranslation.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2010.03.033%26rft_id%253Dinfo%253Apmid%252F20403320%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2010.03.033&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20403320&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F2957.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000276738400013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.2957.19" data-doi="10.1016/j.cell.2016.05.070"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gamble</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brule</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dean</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fields</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grayhack</span>  <span class="cit-name-given-names">E. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Adjacent codons act in concert to modulate translation efficiency in yeast.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">166</span>: <span class="cit-fpage">679</span>–<span class="cit-lpage">690</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D166%26rft.spage%253D679%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2016.05.070%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2016.05.070&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.2957.20" data-doi="10.1371/journal.pone.0001946"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guyader</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burch</span>  <span class="cit-name-given-names">C. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Optimal foraging predicts the ecology but not the evolution of host specialization in bacteriophages.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">e1946</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DGuyader%26rft.auinit1%253DS.%26rft.volume%253D3%26rft.issue%253D4%26rft.spage%253De1946%26rft.epage%253De1946%26rft.atitle%253DOptimal%2Bforaging%2Bpredicts%2Bthe%2Becology%2Bbut%2Bnot%2Bthe%2Bevolution%2Bof%2Bhost%2Bspecialization%2Bin%2Bbacteriophages.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001946%26rft_id%253Dinfo%253Apmid%252F18414655%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0001946&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18414655&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F2957.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.2957.21" data-doi="10.1111/j.1558-5646.2007.00132.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heineman</span>  <span class="cit-name-given-names">R. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bull</span>  <span class="cit-name-given-names">J. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Testing optimality with experimental evolution: lysis time in a bacteriophage.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">61</span>: <span class="cit-fpage">1695</span>–<span class="cit-lpage">1709</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.stitle%253DEvolution%26rft.aulast%253DHeineman%26rft.auinit1%253DR.%2BH.%26rft.volume%253D61%26rft.issue%253D7%26rft.spage%253D1695%26rft.epage%253D1709%26rft.atitle%253DTesting%2Boptimality%2Bwith%2Bexperimental%2Bevolution%253A%2Blysis%2Btime%2Bin%2Ba%2Bbacteriophage.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1558-5646.2007.00132.x%26rft_id%253Dinfo%253Apmid%252F17598749%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1558-5646.2007.00132.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17598749&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F2957.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000247576400014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.2957.22" data-doi="10.1111/j.1365-2958.1991.tb00754.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hellmuth</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rex</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Surin</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zinck</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCarthy</span>  <span class="cit-name-given-names">J. E. G.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">Translational coupling varying in efficiency between different pairs of genes in the central region of the atp operon of <em>Escherichia coli</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Microbiol.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">813</span>–<span class="cit-lpage">824</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bmicrobiology%26rft.stitle%253DMol%2BMicrobiol%26rft.aulast%253DHellmuth%26rft.auinit1%253DK.%26rft.volume%253D5%26rft.issue%253D4%26rft.spage%253D813%26rft.epage%253D824%26rft.atitle%253DTranslational%2Bcoupling%2Bvarying%2Bin%2Befficiency%2Bbetween%2Bdifferent%2Bpairs%2Bof%2Bgenes%2Bin%2Bthe%2Bcentral%2Bregion%2Bof%2Bthe%2Batp%2Boperon%2Bof%2BEscherichia%2Bcoli.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2958.1991.tb00754.x%26rft_id%253Dinfo%253Apmid%252F1830358%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-2958.1991.tb00754.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1830358&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F2957.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.2957.23"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Houser</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barnhart</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boutz</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carroll</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dasgupta</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Controlled measurement and comparative analysis of cellular components in <em>E. coli</em> reveals broad regulatory changes in response to glucose starvation.</span> <abbr class="cit-jnl-abbrev">PLoS Comput. Biol.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">e1004400</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BComput.%2BBiol.%26rft.volume%253D11%26rft.spage%253De1004400%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.2957.24" data-doi="10.1186/1471-2105-8-480"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kosuri</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kelly</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Endy</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">TABASCO: a single molecule, base-pair resolved gene expression simulator.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">480</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DKosuri%26rft.auinit1%253DS.%26rft.volume%253D8%26rft.spage%253D480%26rft.epage%253D480%26rft.atitle%253DTABASCO%253A%2BA%2Bsingle%2Bmolecule%252C%2Bbase-pair%2Bresolved%2Bgene%2Bexpression%2Bsimulator.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-8-480%26rft_id%253Dinfo%253Apmid%252F18093293%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-8-480&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18093293&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F2957.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.2957.25" data-doi="10.1126/science.1170160"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kudla</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Murray</span>  <span class="cit-name-given-names">A. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tollervey</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Plotkin</span>  <span class="cit-name-given-names">J. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Coding-sequence determinants of gene expression in <em>Escherichia coli</em>.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">324</span>: <span class="cit-fpage">255</span>–<span class="cit-lpage">258</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DKudla%26rft.auinit1%253DG.%26rft.volume%253D324%26rft.issue%253D5924%26rft.spage%253D255%26rft.epage%253D258%26rft.atitle%253DCoding-Sequence%2BDeterminants%2Bof%2BGene%2BExpression%2Bin%2BEscherichia%2Bcoli%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1170160%26rft_id%253Dinfo%253Apmid%252F19359587%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjQvNTkyNC8yNTUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzkvMjk1Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.2957.26" data-doi="10.1016/0022-2836(92)90827-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lesage</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chiaruttini</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Graffe</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dondon</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Milet</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">Messenger RNA secondary structure and translational coupling in the <em>Escherichia coli</em> operon encoding translation initiation factor IF3 and the ribosomal proteins, L35 and L20.</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">228</span>: <span class="cit-fpage">366</span>–<span class="cit-lpage">386</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DLesage%26rft.auinit1%253DP.%26rft.volume%253D228%26rft.issue%253D2%26rft.spage%253D366%26rft.epage%253D386%26rft.atitle%253DMessenger%2BRNA%2Bsecondary%2Bstructure%2Band%2Btranslational%2Bcoupling%2Bin%2Bthe%2BEscherichia%2Bcoli%2Boperon%2Bencoding%2Btranslation%2Binitiation%2Bfactor%2BIF3%2Band%2Bthe%2Bribosomal%2Bproteins%252C%2BL35%2Band%2BL20.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0022-2836%252892%252990827-7%26rft_id%253Dinfo%253Apmid%252F1453449%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0022-2836(92)90827-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1453449&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F2957.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1992KA67600009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.2957.27" data-doi="10.1016/j.cell.2014.02.033"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">G. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burkhardt</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gross</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weissman</span>  <span class="cit-name-given-names">J. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Quantifying absolute protein synthesis rates reveals principles underlying allocation of cellular resources.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">157</span>: <span class="cit-fpage">624</span>–<span class="cit-lpage">635</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D157%26rft.spage%253D624%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2014.02.033%26rft_id%253Dinfo%253Apmid%252F24766808%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2014.02.033&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24766808&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F2957.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000335392100012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-book" id="cit-7.9.2957.28"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Molineux</span>  <span class="cit-name-given-names">I. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">The T7 group</span>, pp. <span class="cit-fpage">277</span>–<span class="cit-lpage">301</span> in <span class="cit-source"><em>The Bacteriophages</em></span>, edited by <span class="cit-ed"><span class="cit-name-surname">Calendar</span>  <span class="cit-name-given-names">R.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Abedon</span>  <span class="cit-name-given-names">S. T.</span></span>. <span class="cit-publ-name">Oxford University Press</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.2957.29" data-doi="10.1128/JVI.00738-06"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mueller</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Papamichail</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coleman</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Skiena</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wimmer</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Reduction of the rate of poliovirus protein synthesis through large-scale codon deoptimization causes attenuation of viral virulence by lowering specific infectivity.</span> <abbr class="cit-jnl-abbrev">J. Virol.</abbr> <span class="cit-vol">80</span>: <span class="cit-fpage">9687</span>–<span class="cit-lpage">9696</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DMueller%26rft.auinit1%253DS.%26rft.volume%253D80%26rft.issue%253D19%26rft.spage%253D9687%26rft.epage%253D9696%26rft.atitle%253DReduction%2Bof%2Bthe%2BRate%2Bof%2BPoliovirus%2BProtein%2BSynthesis%2Bthrough%2BLarge-Scale%2BCodon%2BDeoptimization%2BCauses%2BAttenuation%2Bof%2BViral%2BVirulence%2Bby%2BLowering%2BSpecific%2BInfectivity%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.00738-06%26rft_id%253Dinfo%253Apmid%252F16973573%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI4MC8xOS85Njg3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy85LzI5NTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.2957.30" data-doi="10.1038/nbt.1636"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mueller</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coleman</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Papamichail</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ward</span>  <span class="cit-name-given-names">C. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nimnual</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Live attenuated influenza virus vaccines by computer-aided rational design.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">723</span>–<span class="cit-lpage">726</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DMueller%26rft.auinit1%253DS.%26rft.volume%253D28%26rft.issue%253D7%26rft.spage%253D723%26rft.epage%253D726%26rft.atitle%253DLive%2Battenuated%2Binfluenza%2Bvirus%2Bvaccines%2Bby%2Bcomputer-aided%2Brational%2Bdesign.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.1636%26rft_id%253Dinfo%253Apmid%252F20543832%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.1636&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20543832&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F2957.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000279723900029&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.2957.31" data-doi="10.1371/journal.ppat.1003800"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nougairede</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">De Fabritus</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aubry</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gould</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holmes</span>  <span class="cit-name-given-names">E. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Random codon re-encoding induces stable reduction of replicative fitness of chikungunya virus in primate and mosquito cells.</span> <abbr class="cit-jnl-abbrev">PLoS Pathog.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">18</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BPathog.%26rft.volume%253D9%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1003800%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1003800&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.2957.32"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Oppenheim</span>  <span class="cit-name-given-names">D. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yanofsky</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">1980</span> <span class="cit-article-title">Translational coupling during expression of the tryptophan operon of <em>Escherichia coli</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">95</span>: <span class="cit-fpage">785</span>–<span class="cit-lpage">795</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DOppenheim%26rft.auinit1%253DD.%2BS.%26rft.volume%253D95%26rft.issue%253D4%26rft.spage%253D785%26rft.epage%253D795%26rft.atitle%253DTRANSLATIONAL%2BCOUPLING%2BDURING%2BEXPRESSION%2BOF%2BTHE%2BTRYPTOPHAN%2BOPERON%2BOF%2BESCHERICHIA%2BCOLI%26rft_id%253Dinfo%253Apmid%252F6162715%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiOTUvNC83ODUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzkvMjk1Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.2957.33" data-doi="10.1016/j.jtbi.2009.05.008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Patwa</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wahl</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The impact of host-cell dynamics on the fixation probability for lytic viruses.</span> <abbr class="cit-jnl-abbrev">J. Theor. Biol.</abbr> <span class="cit-vol">259</span>: <span class="cit-fpage">799</span>–<span class="cit-lpage">810</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BTheoretical%2BBiology%26rft.stitle%253DJournal%2Bof%2BTheoretical%2BBiology%26rft.aulast%253DPatwa%26rft.auinit1%253DZ.%26rft.volume%253D259%26rft.issue%253D4%26rft.spage%253D799%26rft.epage%253D810%26rft.atitle%253DThe%2Bimpact%2Bof%2Bhost-cell%2Bdynamics%2Bon%2Bthe%2Bfixation%2Bprobability%2Bfor%2Blytic%2Bviruses.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jtbi.2009.05.008%26rft_id%253Dinfo%253Apmid%252F19464301%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jtbi.2009.05.008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19464301&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F2957.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.2957.34" data-doi="10.1038/nrg2899"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Plotkin</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kudla</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Synonymous but not the same: the causes and consequences of codon bias.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">32</span>–<span class="cit-lpage">42</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BGenet.%26rft.volume%253D12%26rft.spage%253D32%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2899%26rft_id%253Dinfo%253Apmid%252F21102527%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2899&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21102527&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F2957.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.2957.35" data-doi="10.1038/nature10126"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Qu</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wen</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lancaster</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Noller</span>  <span class="cit-name-given-names">H. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bustamante</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The ribosome uses two active mechanisms to unwind messenger RNA during translation.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">475</span>: <span class="cit-fpage">118</span>–<span class="cit-lpage">121</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DQu%26rft.auinit1%253DX.%26rft.volume%253D475%26rft.issue%253D7354%26rft.spage%253D118%26rft.epage%253D121%26rft.atitle%253DThe%2Bribosome%2Buses%2Btwo%2Bactive%2Bmechanisms%2Bto%2Bunwind%2Bmessenger%2BRNA%2Bduring%2Btranslation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature10126%26rft_id%253Dinfo%253Apmid%252F21734708%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature10126&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21734708&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F2957.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000292461300057&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-book" id="cit-7.9.2957.36"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">R Core Team</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-source"><em>R: A Language and Environment for Statistical Computing</em></span>, <span class="cit-publ-name">R Foundation for Statistical Computing</span>, <span class="cit-publ-loc">Vienna, Austria</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.2957.37"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Raveh</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Margaliot</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sontag</span>  <span class="cit-name-given-names">E. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tuller</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">A model for competition for ribosomes in the cell.</span> <abbr class="cit-jnl-abbrev">J. R. Soc. Interface</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">e1002127</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BR.%2BSoc.%2BInterface%26rft.volume%253D13%26rft.spage%253De1002127%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.2957.38" data-doi="10.1371/journal.pcbi.1002127"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Reuveni</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meilijson</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kupiec</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruppin</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tuller</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genome-scale analysis of translation elongation with a ribosome flow model.</span> <abbr class="cit-jnl-abbrev">PLoS Comput. Biol.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e1002127</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BComput%2BBiol%26rft.aulast%253DReuveni%26rft.auinit1%253DS.%26rft.volume%253D7%26rft.issue%253D9%26rft.spage%253De1002127%26rft.epage%253De1002127%26rft.atitle%253DGenome-scale%2Banalysis%2Bof%2Btranslation%2Belongation%2Bwith%2Ba%2Bribosome%2Bflow%2Bmodel.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.1002127%26rft_id%253Dinfo%253Apmid%252F21909250%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.1002127&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21909250&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F2957.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.2957.39"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rex</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Surin</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Besse</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schneppe</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCarthy</span>  <span class="cit-name-given-names">J. E.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">The mechanism of translational coupling in <em>Escherichia coli</em>. Higher order structure in the atpHA mRNA acts as a conformational switch regulating the access of de novo initiating ribosomes.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">269</span>: <span class="cit-fpage">18118</span>–<span class="cit-lpage">18127</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBiological%2BChemistry%26rft.stitle%253DJ%2BBiol%2BChem%26rft.aulast%253DRex%26rft.auinit1%253DG.%26rft.volume%253D269%26rft.issue%253D27%26rft.spage%253D18118%26rft.epage%253D18127%26rft.atitle%253DThe%2Bmechanism%2Bof%2Btranslational%2Bcoupling%2Bin%2BEscherichia%2Bcoli.%2BHigher%2Border%2Bstructure%2Bin%2Bthe%2BatpHA%2BmRNA%2Bacts%2Bas%2Ba%2Bconformational%2Bswitch%2Bregulating%2Bthe%2Baccess%2Bof%2Bde%2Bnovo%2Binitiating%2Bribosomes%26rft_id%253Dinfo%253Apmid%252F7517937%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyNjkvMjcvMTgxMTgiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzkvMjk1Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.2957.40" data-doi="10.1016/0092-8674(82)90291-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schümperli</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McKenney</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sobieski</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosenberg</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1982</span> <span class="cit-article-title">Translational coupling at an intercistronic boundary of the <em>Escherichia coli</em> galactose operon.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">865</span>–<span class="cit-lpage">871</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DSch%25C3%25BCmperli%26rft.auinit1%253DD.%26rft.volume%253D30%26rft.issue%253D3%26rft.spage%253D865%26rft.epage%253D871%26rft.atitle%253DTranslational%2Bcoupling%2Bat%2Ban%2Bintercistronic%2Bboundary%2Bof%2Bthe%2BEscherichia%2Bcoli%2Bgalactose%2Boperon.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0092-8674%252882%252990291-4%26rft_id%253Dinfo%253Apmid%252F6754091%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0092-8674(82)90291-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=6754091&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F2957.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1982PM41000023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.2957.41" data-doi="10.1126/science.1192588"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Scott</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gunderson</span>  <span class="cit-name-given-names">C. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mateescu</span>  <span class="cit-name-given-names">E. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hwa</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Interdependence of cell growth and gene expression: origins and consequences.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">330</span>: <span class="cit-fpage">1099</span>–<span class="cit-lpage">1102</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DScott%26rft.auinit1%253DM.%26rft.volume%253D330%26rft.issue%253D6007%26rft.spage%253D1099%26rft.epage%253D1102%26rft.atitle%253DInterdependence%2Bof%2BCell%2BGrowth%2Band%2BGene%2BExpression%253A%2BOrigins%2Band%2BConsequences%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1192588%26rft_id%253Dinfo%253Apmid%252F21097934%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMzAvNjAwNy8xMDk5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy85LzI5NTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.2957.42" data-doi="10.1073/pnas.1016719108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shah</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gilchrist</span>  <span class="cit-name-given-names">M. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Explaining complex codon usage patterns with selection for translational efficiency, mutation bias, and genetic drift.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">10231</span>–<span class="cit-lpage">10236</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1016719108%26rft_id%253Dinfo%253Apmid%252F21646514%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA4LzI1LzEwMjMxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy85LzI5NTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.2957.43" data-doi="10.1534/genetics.108.090100"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shao</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Bacteriophage adsorption rate and optimal lysis time.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">180</span>: <span class="cit-fpage">471</span>–<span class="cit-lpage">482</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.108.090100%26rft_id%253Dinfo%253Apmid%252F18757924%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTgwLzEvNDcxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy85LzI5NTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.2957.44" data-doi="10.1073/pnas.1502864112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shen</span>  <span class="cit-name-given-names">S. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stauft</span>  <span class="cit-name-given-names">C. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gorbatsevych</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ward</span>  <span class="cit-name-given-names">C. B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Large-scale recoding of an arbovirus genome to rebalance its insect <em>vs.</em> mammalian preference.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">112</span>: <span class="cit-fpage">4749</span>–<span class="cit-lpage">4754</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1502864112%26rft_id%253Dinfo%253Apmid%252F25825721%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTEyLzE1LzQ3NDkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzkvMjk1Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.2957.45" data-doi="10.1074/mcp.M500230-MCP200"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Silva</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gorenstein</span>  <span class="cit-name-given-names">M. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">G. Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vissers</span>  <span class="cit-name-given-names">J. P. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Geromanos</span>  <span class="cit-name-given-names">S. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Absolute quantification of proteins by LCMSE: a virtue of parallel MS acquisition.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Proteomics</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">144</span>–<span class="cit-lpage">156</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BCell.%2BProteomics%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fmcp.M500230-MCP200%26rft_id%253Dinfo%253Apmid%252F16219938%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoibWNwcm90IjtzOjU6InJlc2lkIjtzOjc6IjUvMS8xNDQiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzkvMjk1Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.2957.46" data-doi="10.1093/nar/17.14.5501"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Spanjaard</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Duin</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">Translational reinitiation in the presence and absence of a Shine and Dalgarno sequence.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">5501</span>–<span class="cit-lpage">5508</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DSpanjaard%26rft.auinit1%253DR.%2BA.%26rft.volume%253D17%26rft.issue%253D14%26rft.spage%253D5501%26rft.epage%253D5508%26rft.atitle%253DTranslational%2Breinitiation%2Bin%2Bthe%2Bpresence%2Band%2Babsence%2Bof%2Ba%2BShine%2Band%2BDalgarno%2Bsequence%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F17.14.5501%26rft_id%253Dinfo%253Apmid%252F2668889%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/17.14.5501&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2668889&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F2957.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1989AG83900006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.2957.47" data-doi="10.1016/j.virol.2005.07.008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Springman</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Badgett</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Molineux</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bull</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Gene order constrains adaptation in bacteriophage T7.</span> <abbr class="cit-jnl-abbrev">Virology</abbr> <span class="cit-vol">341</span>: <span class="cit-fpage">141</span>–<span class="cit-lpage">152</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirology%26rft.stitle%253DVirology%26rft.aulast%253DSpringman%26rft.auinit1%253DR.%26rft.volume%253D341%26rft.issue%253D1%26rft.spage%253D141%26rft.epage%253D152%26rft.atitle%253DGene%2Border%2Bconstrains%2Badaptation%2Bin%2Bbacteriophage%2BT7.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virol.2005.07.008%26rft_id%253Dinfo%253Apmid%252F16081122%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.virol.2005.07.008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16081122&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F2957.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000232498900014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.2957.48" data-doi="10.1093/nar/gkv635"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tian</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salis</span>  <span class="cit-name-given-names">H. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">A predictive biophysical model of translational coupling to coordinate and control protein expression in bacterial operons.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">7137</span>–<span class="cit-lpage">7151</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkv635%26rft_id%253Dinfo%253Apmid%252F26117546%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkv635&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26117546&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F2957.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.2957.49"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Torgov</span>  <span class="cit-name-given-names">M. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Janzen</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reddy</span>  <span class="cit-name-given-names">M. K.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Efficiency and frequency of translational coupling between the bacteriophage T4 clamp loader genes.</span> <abbr class="cit-jnl-abbrev">J. Bacteriol.</abbr> <span class="cit-vol">180</span>: <span class="cit-fpage">4339</span>–<span class="cit-lpage">4343</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBacteriology%26rft.stitle%253DJ.%2BBacteriol.%26rft.aulast%253DTorgov%26rft.auinit1%253DM.%2BY.%26rft.volume%253D180%26rft.issue%253D17%26rft.spage%253D4339%26rft.epage%253D4343%26rft.atitle%253DEfficiency%2Band%2BFrequency%2Bof%2BTranslational%2BCoupling%2Bbetween%2Bthe%2BBacteriophage%2BT4%2BClamp%2BLoader%2BGenes%26rft_id%253Dinfo%253Apmid%252F9721267%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6MTE6IjE4MC8xNy80MzM5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy85LzI5NTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.2957.50" data-doi="10.1016/j.cell.2010.03.031"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tuller</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carmi</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vestsigian</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Navon</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dorfan</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">An evolutionarily conserved mechanism for controlling the efficiency of protein translation.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">141</span>: <span class="cit-fpage">344</span>–<span class="cit-lpage">354</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DTuller%26rft.auinit1%253DT.%26rft.volume%253D141%26rft.issue%253D2%26rft.spage%253D344%26rft.epage%253D354%26rft.atitle%253DAn%2Bevolutionarily%2Bconserved%2Bmechanism%2Bfor%2Bcontrolling%2Bthe%2Befficiency%2Bof%2Bprotein%2Btranslation.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2010.03.031%26rft_id%253Dinfo%253Apmid%252F20403328%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2010.03.031&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20403328&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F2957.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000276738400023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.2957.51" data-doi="10.7554/eLife.04531"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tulloch</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Atkinson</span>  <span class="cit-name-given-names">N. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Evans</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ryan</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Simmonds</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">RNA virus attenuation by codon pair deoptimisation is an artefact of increases in CpG/UpA dinucleotide frequencies.</span> <abbr class="cit-jnl-abbrev">Elife</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">e04531</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DElife%26rft_id%253Dinfo%253Adoi%252F10.7554%252FeLife.04531%26rft_id%253Dinfo%253Apmid%252F25490153%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NToiZWxpZmUiO3M6NToicmVzaWQiO3M6MTA6IjMvMC9lMDQ1MzEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzkvMjk1Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.2957.52" data-doi="10.1006/jmbi.1993.1319"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vind</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sørensen</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rasmussen</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pedersen</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Synthesis of proteins in <em>Escherichia coli</em> is limited by the concentration of free ribosomes. Expression from reporter genes does not always reflect functional mRNA levels.</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">231</span>: <span class="cit-fpage">678</span>–<span class="cit-lpage">688</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DVind%26rft.auinit1%253DJ.%26rft.volume%253D231%26rft.issue%253D3%26rft.spage%253D678%26rft.epage%253D688%26rft.atitle%253DSynthesis%2Bof%2Bproteins%2Bin%2BEscherichia%2Bcoli%2Bis%2Blimited%2Bby%2Bthe%2Bconcentration%2Bof%2Bfree%2Bribosomes.%2BExpression%2Bfrom%2Breporter%2Bgenes%2Bdoes%2Bnot%2Balways%2Breflect%2Bfunctional%2BmRNA%2Blevels.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fjmbi.1993.1319%26rft_id%253Dinfo%253Apmid%252F7685825%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/jmbi.1993.1319&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7685825&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F2957.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1993LJ64100013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.2957.53" data-doi="10.1093/nar/gkv1145"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vizcaíno</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Csordas</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">del Toro</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dianes</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Griss</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">2016 update of the PRIDE database and its related tools.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">D447</span>–<span class="cit-lpage">D456</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkv1145%26rft_id%253Dinfo%253Apmid%252F26527722%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkv1145&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26527722&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F2957.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.2957.54" data-doi="10.1007/BF01237884"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dykhuizen</span>  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Slobodkin</span>  <span class="cit-name-given-names">L. B.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">The evolution of phage lysis timing.</span> <abbr class="cit-jnl-abbrev">Evol. Ecol.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">545</span>–<span class="cit-lpage">558</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvol.%2BEcol.%26rft.volume%253D10%26rft.spage%253D545%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF01237884%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF01237884&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1996VJ62700007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-book" id="cit-7.9.2957.55"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wickham</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite> <span class="cit-pub-date">2009</span> <span class="cit-source"><em>ggplot2: Elegant Graphics for Data Analysis</em>.</span> <span class="cit-publ-name">Springer-Verlag</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.2957.56" data-doi="10.1038/nbt.1593"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wimmer</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mueller</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tumpey</span>  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taubenberger</span>  <span class="cit-name-given-names">J. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Synthetic viruses: a new opportunity to understand and prevent viral disease.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">1163</span>–<span class="cit-lpage">1172</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DWimmer%26rft.auinit1%253DE.%26rft.volume%253D27%26rft.issue%253D12%26rft.spage%253D1163%26rft.epage%253D1172%26rft.atitle%253DSynthetic%2Bviruses%253A%2Ba%2Bnew%2Bopportunity%2Bto%2Bunderstand%2Band%2Bprevent%2Bviral%2Bdisease.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.1593%26rft_id%253Dinfo%253Apmid%252F20010599%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.1593&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20010599&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F2957.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000272708300033&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.2957.57" data-doi="10.1038/nature11942"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shah</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rokas</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Non-optimal codon usage is a mechanism to achieve circadian clock conditionality.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">495</span>: <span class="cit-fpage">116</span>–<span class="cit-lpage">120</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D495%26rft.spage%253D116%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature11942%26rft_id%253Dinfo%253Apmid%252F23417065%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature11942&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23417065&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F2957.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000316039800051&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.2957.58" data-doi="10.1128/JB.184.7.1888-1894.2002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">You</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Suthers</span>  <span class="cit-name-given-names">P. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yin</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Effects of <em>Escherichia coli</em> physiology on growth of phage T7 in vivo and in silico.</span> <abbr class="cit-jnl-abbrev">J. Bacteriol.</abbr> <span class="cit-vol">184</span>: <span class="cit-fpage">1888</span>–<span class="cit-lpage">1894</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBacteriology%26rft.stitle%253DJ.%2BBacteriol.%26rft.aulast%253DYou%26rft.auinit1%253DL.%26rft.volume%253D184%26rft.issue%253D7%26rft.spage%253D1888%26rft.epage%253D1894%26rft.atitle%253DEffects%2Bof%2BEscherichia%2Bcoli%2BPhysiology%2Bon%2BGrowth%2Bof%2BPhage%2BT7%2BIn%2BVivo%2Band%2BIn%2BSilico%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJB.184.7.1888-1894.2002%26rft_id%253Dinfo%253Apmid%252F11889095%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6MTA6IjE4NC83LzE4ODgiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzkvMjk1Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.2957.59" data-doi="10.1038/nature11833"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cha</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chae</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Non-optimal codon usage affects expression, structure and function of clock protein FRQ.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">495</span>: <span class="cit-fpage">111</span>–<span class="cit-lpage">115</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D495%26rft.spage%253D111%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature11833%26rft_id%253Dinfo%253Apmid%252F23417067%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature11833&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23417067&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F2957.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000316039800050&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.2957.60" data-doi="10.1073/pnas.1606724113"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">Ch.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Codon usage is an important determinant of gene expression levels largely through its effects on transcription.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">113</span>: <span class="cit-fpage">E6117</span>–<span class="cit-lpage">E6125</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1606724113%26rft_id%253Dinfo%253Apmid%252F27671647%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTEzLzQxL0U2MTE3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy85LzI5NTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.2957.61" data-doi="10.1093/nar/gkw764"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zur</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tuller</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Predictive biophysical modeling and understanding of the dynamics of mRNA translation and its evolution.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">9031</span>–<span class="cit-lpage">9049</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkw764%26rft_id%253Dinfo%253Apmid%252F27591251%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkw764&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27591251&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F2957.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/7/9/2957.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/9/2945" title="Functional Profiling of Transcription Factor Genes in Neurospora crassa" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/9/2969" title="The Caenorhabditis elegans Female-Like State: Decoupling the Transcriptomic Effects of Aging and Sperm Status" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/7/9"><span>Volume 7 Issue 9, September 2017</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/7/9.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/7/9.cover-source.jpg?itok=WuYpGT8r" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 7 (9)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/9/2957" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-6sXZZGjaQ3GQ34z8CmOn-3UTlaFT4WG-nxo-tzgfpys" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/7/9/2957" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/7/9/2957" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/7/9/2957" class="active" data-icon-position="" data-hide-link-title="0">Reduced Protein Expression in a Virus Attenuated by Codon Deoptimization</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/486347" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-1iNpGfxR8NbQle0jdXy97ZZNHDPzK8WApB6rgqK3X24" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/9/2957.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/7/9/2957" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-mLmozMOSHR75f520OefhiQIUOM28TajJJ2JXRULRETo" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/7/9/2957" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/9/2957.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="486347" id="node486347--2" data-pisa="ggg;7/9/2957" data-pisa-master="ggg;g3.117.041020" data-apath="/ggg/7/9/2957.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Reduced Protein Expression in a Virus Attenuated by Codon Deoptimization</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-7777-1653" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Benjamin R. Jack</span>, <span class="highwire-citation-author" data-delta="1">Daniel R. Boutz</span>, <span class="highwire-citation-author" data-delta="2">Matthew L. Paff</span>, <span class="highwire-citation-author" data-delta="3">Bartram L. Smith</span>, <span class="highwire-citation-author" data-delta="4">James J. Bull</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0002-7470-9261" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Claus O. Wilke</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2957-2968;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.041020 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node486347--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Benjamin R. Jack </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Integrative Biology, The University of Texas at Austin, Texas 78712 </div><div class='author-affiliation'><span class='nlm-sup'>†</span>Center for Computational Biology and Bioinformatics, The University of Texas at Austin, Texas 78712</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Institute for Cellular and Molecular Biology, The University of Texas at Austin, Texas 78712</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Benjamin%2BR.%2BJack%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jack%20BR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABenjamin%2BR.%2BJack%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-7777-1653" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Benjamin R. Jack</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:benjamin.r.jack@gmail.com" class="" data-icon-position="" data-hide-link-title="0">benjamin.r.jack@gmail.com</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Daniel R. Boutz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Institute for Cellular and Molecular Biology, The University of Texas at Austin, Texas 78712</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniel%2BR.%2BBoutz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Boutz%20DR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADaniel%2BR.%2BBoutz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Matthew L. Paff </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Integrative Biology, The University of Texas at Austin, Texas 78712 </div><div class='author-affiliation'><span class='nlm-sup'>†</span>Center for Computational Biology and Bioinformatics, The University of Texas at Austin, Texas 78712</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Institute for Cellular and Molecular Biology, The University of Texas at Austin, Texas 78712</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Matthew%2BL.%2BPaff%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Paff%20ML&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMatthew%2BL.%2BPaff%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Bartram L. Smith </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Integrative Biology, The University of Texas at Austin, Texas 78712 </div><div class='author-affiliation'><span class='nlm-sup'>†</span>Center for Computational Biology and Bioinformatics, The University of Texas at Austin, Texas 78712</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Institute for Cellular and Molecular Biology, The University of Texas at Austin, Texas 78712</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bartram%2BL.%2BSmith%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Smith%20BL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABartram%2BL.%2BSmith%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">James J. Bull </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Integrative Biology, The University of Texas at Austin, Texas 78712 </div><div class='author-affiliation'><span class='nlm-sup'>†</span>Center for Computational Biology and Bioinformatics, The University of Texas at Austin, Texas 78712</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Institute for Cellular and Molecular Biology, The University of Texas at Austin, Texas 78712</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=James%2BJ.%2BBull%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bull%20JJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJames%2BJ.%2BBull%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Claus O. Wilke </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Integrative Biology, The University of Texas at Austin, Texas 78712 </div><div class='author-affiliation'><span class='nlm-sup'>†</span>Center for Computational Biology and Bioinformatics, The University of Texas at Austin, Texas 78712</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Institute for Cellular and Molecular Biology, The University of Texas at Austin, Texas 78712</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Claus%2BO.%2BWilke%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wilke%20CO&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AClaus%2BO.%2BWilke%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-7470-9261" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Claus O. Wilke</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/7/9/2957" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/7/9/2957&amp;title=Reduced%20Protein%20Expression%20in%20a%20Virus%20Attenuated%20by%20Codon%20Deoptimization" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/7/9/2957&amp;title=Reduced%20Protein%20Expression%20in%20a%20Virus%20Attenuated%20by%20Codon%20Deoptimization" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/7/9/2957&amp;title=Reduced%20Protein%20Expression%20in%20a%20Virus%20Attenuated%20by%20Codon%20Deoptimization" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/7/9/2957&amp;text=Reduced%20Protein%20Expression%20in%20a%20Virus%20Attenuated%20by%20Codon%20Deoptimization" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/7/9/2957&amp;title=Reduced%20Protein%20Expression%20in%20a%20Virus%20Attenuated%20by%20Codon%20Deoptimization" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/7/9/2957&amp;t=Reduced%20Protein%20Expression%20in%20a%20Virus%20Attenuated%20by%20Codon%20Deoptimization" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/7/9/2957&amp;title=Reduced%20Protein%20Expression%20in%20a%20Virus%20Attenuated%20by%20Codon%20Deoptimization" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/7/9/2957&amp;title=Reduced%20Protein%20Expression%20in%20a%20Virus%20Attenuated%20by%20Codon%20Deoptimization" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/486347/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="486347" id="node486347--3" data-pisa="ggg;7/9/2957" data-pisa-master="ggg;g3.117.041020" data-apath="/ggg/7/9/2957.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Reduced Protein Expression in a Virus Attenuated by Codon Deoptimization</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-7777-1653" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Benjamin R. Jack</span>, <span class="highwire-citation-author" data-delta="1">Daniel R. Boutz</span>, <span class="highwire-citation-author" data-delta="2">Matthew L. Paff</span>, <span class="highwire-citation-author" data-delta="3">Bartram L. Smith</span>, <span class="highwire-citation-author" data-delta="4">James J. Bull</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0002-7470-9261" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Claus O. Wilke</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2957-2968;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.041020 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node486347--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Benjamin R. Jack </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Integrative Biology, The University of Texas at Austin, Texas 78712 </div><div class='author-affiliation'><span class='nlm-sup'>†</span>Center for Computational Biology and Bioinformatics, The University of Texas at Austin, Texas 78712</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Institute for Cellular and Molecular Biology, The University of Texas at Austin, Texas 78712</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Benjamin%2BR.%2BJack%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jack%20BR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABenjamin%2BR.%2BJack%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-7777-1653" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Benjamin R. Jack</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:benjamin.r.jack@gmail.com" class="" data-icon-position="" data-hide-link-title="0">benjamin.r.jack@gmail.com</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Daniel R. Boutz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Institute for Cellular and Molecular Biology, The University of Texas at Austin, Texas 78712</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniel%2BR.%2BBoutz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Boutz%20DR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADaniel%2BR.%2BBoutz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Matthew L. Paff </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Integrative Biology, The University of Texas at Austin, Texas 78712 </div><div class='author-affiliation'><span class='nlm-sup'>†</span>Center for Computational Biology and Bioinformatics, The University of Texas at Austin, Texas 78712</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Institute for Cellular and Molecular Biology, The University of Texas at Austin, Texas 78712</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Matthew%2BL.%2BPaff%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Paff%20ML&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMatthew%2BL.%2BPaff%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Bartram L. Smith </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Integrative Biology, The University of Texas at Austin, Texas 78712 </div><div class='author-affiliation'><span class='nlm-sup'>†</span>Center for Computational Biology and Bioinformatics, The University of Texas at Austin, Texas 78712</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Institute for Cellular and Molecular Biology, The University of Texas at Austin, Texas 78712</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bartram%2BL.%2BSmith%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Smith%20BL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABartram%2BL.%2BSmith%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">James J. Bull </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Integrative Biology, The University of Texas at Austin, Texas 78712 </div><div class='author-affiliation'><span class='nlm-sup'>†</span>Center for Computational Biology and Bioinformatics, The University of Texas at Austin, Texas 78712</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Institute for Cellular and Molecular Biology, The University of Texas at Austin, Texas 78712</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=James%2BJ.%2BBull%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bull%20JJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJames%2BJ.%2BBull%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Claus O. Wilke </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Integrative Biology, The University of Texas at Austin, Texas 78712 </div><div class='author-affiliation'><span class='nlm-sup'>†</span>Center for Computational Biology and Bioinformatics, The University of Texas at Austin, Texas 78712</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Institute for Cellular and Molecular Biology, The University of Texas at Austin, Texas 78712</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Claus%2BO.%2BWilke%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wilke%20CO&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AClaus%2BO.%2BWilke%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-7470-9261" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Claus O. Wilke</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=2957&amp;rft.epage=2968&amp;rft.atitle=Reduced%20Protein%20Expression%20in%20a%20Virus%20Attenuated%20by%20Codon%20Deoptimization&amp;rft.volume=7&amp;rft.issue=9&amp;rft.date=2017-09-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Jack%2C+Benjamin+R.&amp;rft.au=Boutz%2C+Daniel+R.&amp;rft.au=Paff%2C+Matthew+L.&amp;rft.au=Smith%2C+Bartram+L.&amp;rft.au=Bull%2C+James+J.&amp;rft.au=Wilke%2C+Claus+O."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/486347/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/486347/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/486347/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/486347/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/486347/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/486347/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/486347/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/486347/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/486347/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/486347/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/486347/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/486347/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/7/9/2957" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-12" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-18" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/7/9/2957.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/7/9/2957.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/7/9/2957.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
