<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/7/9/3091" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/7/9/3091.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/7/9/3091.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/7/9/3091.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/7/9/3091.atom" />
<meta name="HW.pisa" content="ggg;7/9/3091" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="3′-Terminated Overhangs Regulate DNA Double-Strand Break Processing in Escherichia coli" />
<meta name="DC.Identifier" content="10.1534/g3.117.043521" />
<meta name="DC.Date" content="2017-09-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2017 Đermić et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Double-strand breaks (DSBs) are lethal DNA lesions, which are repaired by homologous recombination in Escherichia coli . To study DSB processing in vivo , we induced DSBs into the E. coli chromosome by γ-irradiation and measured chromosomal degradation. We show that the DNA degradation is regulated by RecA protein concentration and its rate of association with single-stranded DNA (ssDNA). RecA decreased DNA degradation in wild-type, recB , and recD strains, indicating that it is a general phenomenon in E. coli . On the other hand, DNA degradation was greatly reduced and unaffected by RecA in the recB1080 mutant (which produces long overhangs) and in a strain devoid of four exonucleases that degrade a 3′ tail (ssExos). 3′–5′ ssExos deficiency is epistatic to RecA deficiency concerning DNA degradation, suggesting that bound RecA is shielding the 3′ tail from degradation by 3′–5′ ssExos. Since 3′ tail preservation is common to all these situations, we infer that RecA polymerization constitutes a subset of mechanisms for preserving the integrity of 3′ tails emanating from DSBs, along with 3′ tail’s massive length, or prevention of their degradation by inactivation of 3′–5′ ssExos. Thus, we conclude that 3′ overhangs are crucial in controlling the extent of DSB processing in E. coli . This study suggests a regulatory mechanism for DSB processing in E. coli , wherein 3′ tails impose a negative feedback loop on DSB processing reactions, specifically on helicase reloading onto dsDNA ends." />
<meta name="DC.Contributor" content="Edyta Đermić" />
<meta name="DC.Contributor" content="Davor Zahradka" />
<meta name="DC.Contributor" content="Dušica Vujaklija" />
<meta name="DC.Contributor" content="Siniša Ivanković" />
<meta name="DC.Contributor" content="Damir Đermić" />
<meta name="article:published_time" content="2017-09-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="3′-Terminated Overhangs Regulate DNA Double-Strand Break Processing in Escherichia coli" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Double-strand breaks (DSBs) are lethal DNA lesions, which are repaired by homologous recombination in &lt;i&gt;Escherichia coli&lt;/i&gt;. To study DSB processing &lt;i&gt;in vivo&lt;/i&gt;, we induced DSBs into the &lt;i&gt;E. coli&lt;/i&gt; chromosome by γ-irradiation and measured chromosomal degradation. We show that the DNA degradation is regulated by RecA protein concentration and its rate of association with single-stranded DNA (ssDNA). RecA decreased DNA degradation in wild-type, &lt;i&gt;recB&lt;/i&gt;, and &lt;i&gt;recD&lt;/i&gt; strains, indicating that it is a general phenomenon in &lt;i&gt;E. coli&lt;/i&gt;. On the other hand, DNA degradation was greatly reduced and unaffected by RecA in the &lt;i&gt;recB1080&lt;/i&gt; mutant (which produces long overhangs) and in a strain devoid of four exonucleases that degrade a 3′ tail (ssExos). 3′–5′ ssExos deficiency is epistatic to RecA deficiency concerning DNA degradation, suggesting that bound RecA is shielding the 3′ tail from degradation by 3′–5′ ssExos. Since 3′ tail preservation is common to all these situations, we infer that RecA polymerization constitutes a subset of mechanisms for preserving the integrity of 3′ tails emanating from DSBs, along with 3′ tail’s massive length, or prevention of their degradation by inactivation of 3′–5′ ssExos. Thus, we conclude that 3′ overhangs are crucial in controlling the extent of DSB processing in &lt;i&gt;E. coli&lt;/i&gt;. This study suggests a regulatory mechanism for DSB processing in &lt;i&gt;E. coli&lt;/i&gt;, wherein 3′ tails impose a negative feedback loop on DSB processing reactions, specifically on helicase reloading onto dsDNA ends.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2017/09/01" />
<meta name="citation_mjid" content="ggg;7/9/3091" />
<meta name="citation_id" content="7/9/3091" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/7/9/3091" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/7/9/3091.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/7/9/3091.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/7/9/3091.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.117.043521" />
<meta name="citation_pmid" content="28710290" />
<meta name="citation_volume" content="7" />
<meta name="citation_issue" content="9" />
<meta name="citation_num_pages" content="12" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="3091" />
<meta name="citation_lastpage" content="3102" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Edyta Đermić" />
<meta name="citation_author_institution" content="Department of Plant Pathology, Faculty of Agriculture, University of Zagreb, 10000, Croatia" />
<meta name="citation_author" content="Davor Zahradka" />
<meta name="citation_author_institution" content="Division of Molecular Biology, Ruđer Bošković Institute, 10000 Zagreb, Croatia" />
<meta name="citation_author" content="Dušica Vujaklija" />
<meta name="citation_author_institution" content="Division of Molecular Biology, Ruđer Bošković Institute, 10000 Zagreb, Croatia" />
<meta name="citation_author" content="Siniša Ivanković" />
<meta name="citation_author_institution" content="Division of Molecular Medicine, Ruđer Bošković Institute, 10000 Zagreb, Croatia" />
<meta name="citation_author" content="Damir Đermić" />
<meta name="citation_author_institution" content="Division of Molecular Biology, Ruđer Bošković Institute, 10000 Zagreb, Croatia" />
<meta name="citation_author_email" content="dermic@irb.hr" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=D. G. Anderson;citation_author=S. C. Kowalczykowski;citation_title=The recombination hot spot chi is a regulatory element that switches the polarity of DNA degradation by the RecBCD enzyme.;citation_pages=571-581;citation_volume=11;citation_year=1997;citation_issue=5;citation_pmid=9119222;citation_doi=10.1101/gad.11.5.571" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=D. G. Anderson;citation_author=S. C. Kowalczykowski;citation_title=The translocating RecBCD enzyme stimulates recombination by directing RecA protein onto ssDNA in a chi-regulated manner.;citation_pages=77-86;citation_volume=90;citation_year=1997;citation_issue=1;citation_pmid=9230304;citation_doi=10.1016/S0092-8674(00)80315-3" />
<meta name="citation_reference" content="citation_journal_title=Journal of Biological Chemistry;citation_journal_abbrev=J Biol Chem;citation_author=D. G. Anderson;citation_author=J. J. Churchill;citation_author=S. C. Kowalczykowski;citation_title=A Single Mutation, RecBD1080A, Eliminates RecA Protein Loading but Not Chi Recognition by RecBCD Enzyme;citation_pages=27139-27144;citation_volume=274;citation_year=1999;citation_issue=38;citation_pmid=10480929;citation_doi=10.1074/jbc.274.38.27139" />
<meta name="citation_reference" content="citation_journal_title=Microbiology and Molecular Biology Reviews;citation_journal_abbrev=Microbiol. Mol. Biol. Rev.;citation_author=B. J. Bachmann;citation_title=Pedigrees of some mutant strains of Escherichia coli K-12.;citation_pages=525-557;citation_volume=36;citation_year=1972;citation_issue=4;citation_pmid=4568763" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=P. R. Bianco;citation_author=L. R. Brewer;citation_author=M. Corzett;citation_author=R. Balhorn;citation_author=Y. Yeh;citation_title=Processive translocation and DNA unwinding by individual RecBCD enzyme molecules.;citation_pages=374-378;citation_volume=409;citation_year=2001;citation_issue=6818;citation_pmid=11201750;citation_doi=10.1038/35053131" />
<meta name="citation_reference" content="citation_journal_title=Int. J. Radiat. Biol.;citation_author=S. E. Bresler;citation_author=L. A. Noskin;citation_author=N. A. Kuzovleva;citation_author=I. G. Noskina;citation_title=Nature of the damage to Escherichia coli DNA induced by gamma radiation.;citation_pages=289-300;citation_volume=36;citation_year=1979;citation_doi=10.1080/09553007914551061" />
<meta name="citation_reference" content="citation_journal_title=Journal of Biological Chemistry;citation_journal_abbrev=J Biol Chem;citation_author=R. S. Brody;citation_author=K. G. Doherty;citation_author=P. D. Zimmerman;citation_title=Processivity and kinetics of the reaction of exonuclease I from Escherichia coli with polydeoxyribonucleotides;citation_pages=7136-7143;citation_volume=261;citation_year=1986;citation_issue=16;citation_pmid=3519606" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=F. N. Capaldo;citation_author=S. D. Barbour;citation_title=DNA content, synthesis and integrity in dividing and non-dividing cells of rec- strains of Escherichia coli K12.;citation_pages=53-66;citation_volume=91;citation_year=1975;citation_issue=1;citation_pmid=1102696;citation_doi=10.1016/0022-2836(75)90371-X" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=J. J. Churchill;citation_author=D. G. Anderson;citation_author=S. C. Kowalczykowski;citation_title=The RecBC enzyme loads RecA protein onto ssDNA asymmetrically and independently of chi , resulting in constitutive recombination activation;citation_pages=901-911;citation_volume=13;citation_year=1999;citation_issue=7;citation_pmid=10197989;citation_doi=10.1101/gad.13.7.901" />
<meta name="citation_reference" content="citation_journal_title=Biochimie;citation_author=A. J. Clark;citation_title=recA operator mutations and their usefulness.;citation_pages=669-675;citation_volume=64;citation_year=1982;citation_pmid=6291636;citation_doi=10.1016/S0300-9084(82)80108-9" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=P. Dabert;citation_author=S. D. Ehrlich;citation_author=A. Gruss;citation_title=Chi sequence protects against RecBCD degradation of DNA in vivo.;citation_pages=12073-12077;citation_volume=89;citation_year=1992;citation_issue=24;citation_pmid=1465442;citation_doi=10.1073/pnas.89.24.12073" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Microbiology;citation_journal_abbrev=Nat Rev Microbiol;citation_author=M. J. Daly;citation_title=A new perspective on radiation resistance based on Deinococcus radiodurans.;citation_pages=237-245;citation_volume=7;citation_year=2009;citation_issue=3;citation_pmid=19172147;citation_doi=10.1038/nrmicro2073" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=D. Đermić;citation_title=Functions of multiple exonucleases are essential for cell viability, DNA repair and homologous recombination in recD mutants of Escherichia coli.;citation_volume=172;citation_year=2006;citation_pmid=16452142;citation_doi=10.1534/genetics.105.052076" />
<meta name="citation_reference" content="citation_journal_title=Adv. Genomics Genet.;citation_author=D. Đermić;citation_title=Double-strand break repair mechanisms in Escherichia coli: recent insights.;citation_pages=35-42;citation_volume=5;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Research in Microbiology (Paris);citation_journal_abbrev=Research in Microbiology (Paris);citation_author=D. Đermić;citation_author=E. Halupecki;citation_author=D. Zahradka;citation_author=M. Petranović;citation_title=RecBCD enzyme overproduction impairs DNA repair and homologous recombination in Escherichia coli.;citation_pages=304-311;citation_volume=156;citation_year=2005;citation_issue=3;citation_pmid=15808933;citation_doi=10.1016/j.resmic.2004.10.005" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=D. Đermić;citation_author=D. Zahradka;citation_author=M. Petranović;citation_title=Exonuclease requirements for recombination of λ-phage in recD mutants of Escherichia coli.;citation_volume=173;citation_year=2006;citation_pmid=16702415;citation_doi=10.1534/genetics.106.060426" />
<meta name="citation_reference" content="citation_journal_title=Biochimie;citation_author=D. Đermić;citation_author=E. Đermić;citation_author=D. Zahradka;citation_author=M. Petranović;citation_author=N. Lerš;citation_title=Gamma-irradiated RecD overproducers become permanent recB-/recC- phenocopies for extrachromosomal DNA processing due to prolonged titration of RecBCD enzyme on damaged Escherichia coli chromosome.;citation_pages=379-386;citation_volume=88;citation_year=2006" />
<meta name="citation_reference" content="citation_journal_title=Microbiology and Molecular Biology Reviews;citation_journal_abbrev=Microbiol. Mol. Biol. Rev.;citation_author=M. S. Dillingham;citation_author=S. C. Kowalczykowski;citation_title=RecBCD Enzyme and the Repair of Double-Stranded DNA Breaks;citation_pages=642-671;citation_volume=72;citation_year=2008;citation_issue=4;citation_pmid=19052323;citation_doi=10.1128/MMBR.00020-08" />
<meta name="citation_reference" content="citation_journal_title=J. Biol. Chem.;citation_author=M. S. Dillingham;citation_author=M. R. Webb;citation_author=S. C. Kowalczykowski;citation_title=Bipolar DNA translocation contributes to highly processive DNA unwinding by RecBCD enzyme.;citation_volume=280;citation_year=2005;citation_pmid=16041061;citation_doi=10.1074/jbc.M505520200" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=I. Ivančić-Baće;citation_author=P. Peharec;citation_author=S. Moslavac;citation_author=N. Škrobot;citation_author=E. Salaj-Šmic;citation_title=RecFOR Function Is Required for DNA Repair and Recombination in a RecA Loading-Deficient recB Mutant of Escherichia coli;citation_pages=485-494;citation_volume=163;citation_year=2003;citation_issue=2;citation_pmid=12618388" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=S. Ivanković;citation_author=D. Đermić;citation_title=DNA end resection controls the balance between homologous and illegitimate recombination in Escherichia coli.;citation_pages=e39030-e39030;citation_volume=7;citation_year=2012;citation_issue=6;citation_pmid=22720024" />
<meta name="citation_reference" content="citation_journal_title=DNA Repair (Amst.);citation_author=S. Ivanković;citation_author=D. Vujaklija;citation_author=D. Đermić;citation_title=Nucleolytic degradation of 3′-ending overhangs is essential for DNA-end resection in RecA-loading deficient recB mutants of Escherichia coli.;citation_pages=56-65;citation_volume=57;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Molecular microbiology;citation_journal_abbrev=Mol Microbiol;citation_author=M. E. Jockovich;citation_author=R. S. Myers;citation_title=Nuclease activity is essential for RecBCD recombination in Escherichia coli.;citation_pages=949-962;citation_volume=41;citation_year=2001;citation_issue=4;citation_pmid=11532156;citation_doi=10.1046/j.1365-2958.2001.02573.x" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=R. Johnson;citation_author=V. Borde;citation_author=M. J. Neale;citation_author=A. Bishop-Bailey;citation_author=M. North;citation_title=Excess single-stranded DNA inhibits meiotic double-strand break repair.;citation_pages=e223-e223;citation_volume=3;citation_year=2007;citation_issue=11;citation_pmid=18081428;citation_doi=10.1371/journal.pgen.0030223" />
<meta name="citation_reference" content="citation_journal_title=J. Mol. Biol.;citation_author=M. Kohiyama;citation_author=V. Contremoulins;citation_author=X. Baudin;citation_title=Trashing of single-stranded DNA generated during processing of arrested replication fork in E. coli.;citation_pages=4837-4844;citation_volume=425;citation_year=2013;citation_pmid=23810902;citation_doi=10.1016/j.jmb.2013.06.027" />
<meta name="citation_reference" content="citation_journal_title=Biochemistry;citation_author=F. Korangy;citation_author=D. A. Julin;citation_title=Kinetics and processivity of ATP hydrolysis and DNA unwinding by the RecBC enzyme from Escherichia coli.;citation_pages=4873-4880;citation_volume=32;citation_year=1993;citation_pmid=8387820;citation_doi=10.1021/bi00069a024" />
<meta name="citation_reference" content="citation_journal_title=Biochemistry;citation_author=F. Korangy;citation_author=D. A. Julin;citation_title=Efficiency of ATP hydrolysis and DNA unwinding by the RecBC enzyme from Escherichia coli.;citation_pages=9552-9560;citation_volume=33;citation_year=1994;citation_pmid=8068630;citation_doi=10.1021/bi00198a022" />
<meta name="citation_reference" content="citation_journal_title=Microbiology and Molecular Biology Reviews;citation_journal_abbrev=Microbiol. Mol. Biol. Rev.;citation_author=A. Kuzminov;citation_title=Recombinational Repair of DNA Damage in Escherichia coli and Bacteriophage lambda;citation_pages=751-813;citation_volume=63;citation_year=1999;citation_issue=4;citation_pmid=10585965" />
<meta name="citation_reference" content="citation_journal_title=Journal of Bacteriology;citation_journal_abbrev=J. Bacteriol.;citation_author=A. Kuzminov;citation_author=F. W. Stahl;citation_title=Stability of linear DNA in recA mutant Escherichia coli cells reflects ongoing chromosomal DNA degradation.;citation_pages=880-888;citation_volume=179;citation_year=1997;citation_issue=3;citation_pmid=9006046;citation_doi=10.1128/jb.179.3.880-888.1997" />
<meta name="citation_reference" content="citation_journal_title=EMBO J.;citation_author=A. Kuzminov;citation_author=E. Shabtach;citation_author=F. W. Stahl;citation_title=χ-sites in combination with RecA protein increase survival of linear DNA in Escherichia coli by inactivating exoV activity of RecBCD nuclease.;citation_pages=2764-2776;citation_volume=13;citation_year=1994;citation_pmid=8026461" />
<meta name="citation_reference" content="citation_journal_title=Journal of Biological Chemistry;citation_journal_abbrev=J Biol Chem;citation_author=P. E. Lavery;citation_author=S. C. Kowalczykowski;citation_title=Biochemical basis of the constitutive repressor cleavage activity of recA730 protein. A comparison to recA441 and recA803 proteins;citation_pages=20648-20658;citation_volume=267;citation_year=1992;citation_issue=29;citation_pmid=1400384" />
<meta name="citation_reference" content="citation_journal_title=Biochimie.;citation_author=J. W. Little;citation_title=Mechanism of specific LexA cleavage-autodigestion and the role of RecA coprotease.;citation_pages=411-422;citation_volume=73;citation_year=1991;citation_pmid=1911941;citation_doi=10.1016/0300-9084(91)90108-D" />
<meta name="citation_reference" content="citation_journal_title=EcoSal Plus;citation_title=The DNA exonucleases of Escherichia coli.;citation_author=S. T. Lovett;citation_volume=4;citation_year=2011" />
<meta name="citation_reference" content="citation_title=A Short Course in Bacterial Genetics;citation_author=J. H. Miller;citation_year=1992" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=A. Miranda;citation_author=A. Kuzminov;citation_title=Chromosomal Lesion Suppression and Removal in Escherichia coli via Linear DNA Degradation;citation_pages=1255-1271;citation_volume=163;citation_year=2003;citation_issue=4;citation_pmid=12702673" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=K. Morimatsu;citation_author=S. C. Kowalczykowski;citation_title=RecFOR proteins load RecA protein onto gapped DNA to accelerate DNA strand exchange: a universal step of recombinational repair.;citation_pages=1337-1347;citation_volume=11;citation_year=2003;citation_issue=5;citation_pmid=12769856;citation_doi=10.1016/S1097-2765(03)00188-6" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=K. Morimatsu;citation_author=S. C. Kowalczykowski;citation_title=RecQ helicase and RecJ nuclease provide complementary functions to resect DNA for homologous recombination.;citation_volume=111;citation_year=2014;citation_pmid=25411316;citation_doi=10.1073/pnas.1420009111" />
<meta name="citation_reference" content="citation_journal_title=Critical Reviews in Biochemistry and Molecular Biology;citation_journal_abbrev=CRIT REV BIOCHEM MOL BIOL;citation_author=N. S. Persky;citation_author=S. T. Lovett;citation_title=Mechanisms of recombination: lessons from E. coli.;citation_pages=347-370;citation_volume=43;citation_year=2008;citation_issue=6;citation_pmid=19016098;citation_doi=10.1080/10409230802485358" />
<meta name="citation_reference" content="citation_journal_title=Mutat. Res.;citation_author=J. Repar;citation_author=N. Briški;citation_author=M. Buljubašić;citation_author=K. Zahradka;citation_author=D. Zahradka;citation_title=Exonuclease VII is involved in “reckless” DNA degradation in UV-irradiated Escherichia coli.;citation_pages=96-104;citation_volume=750;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Journal of Bacteriology;citation_journal_abbrev=J. Bacteriol.;citation_author=R. Rinken;citation_author=B. Thoms;citation_author=W. Wackernagel;citation_title=Evidence that recBC-dependent degradation of duplex DNA in Escherichia coli recD mutants involves DNA unwinding.;citation_pages=5424-5429;citation_volume=174;citation_year=1992;citation_issue=16;citation_pmid=1322885;citation_doi=10.1128/jb.174.16.5424-5429.1992" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=M. Sassanfar;citation_author=J. W. Roberts;citation_title=Nature of the SOS-inducing signal in Escherichia coli. The involvement of DNA replication.;citation_pages=79-96;citation_volume=212;citation_year=1990;citation_issue=1;citation_pmid=2108251;citation_doi=10.1016/0022-2836(90)90306-7" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=C. A. Schneider;citation_author=W. S. Rasband;citation_author=K. W. Eliceiri;citation_title=NIH Image to ImageJ: 25 years of image analysis.;citation_pages=671-675;citation_volume=9;citation_year=2012;citation_issue=7;citation_pmid=22930834;citation_doi=10.1038/nmeth.2089" />
<meta name="citation_reference" content="citation_journal_title=Elife;citation_author=C. Shee;citation_author=B. D. Cox;citation_author=F. Gu;citation_author=E. M. Luengas;citation_author=M. C. Joshi;citation_title=Engineered proteins detect spontaneous DNA breakage in human and bacterial cells.;citation_volume=2;citation_year=2013;citation_pmid=24171103;citation_doi=10.7554/eLife.01222" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=H. Shinagawa;citation_author=H. Iwasaki;citation_author=T. Kato;citation_author=A. Nakata;citation_title=RecA protein-dependent cleavage of UmuD protein and SOS mutagenesis.;citation_pages=1806-1810;citation_volume=85;citation_year=1988;citation_issue=6;citation_pmid=3126496;citation_doi=10.1073/pnas.85.6.1806" />
<meta name="citation_reference" content="citation_journal_title=Journal of Bacteriology;citation_journal_abbrev=J. Bacteriol.;citation_author=K. Skarstad;citation_author=E. Boye;citation_title=Degradation of individual chromosomes in recA mutants of Escherichia coli.;citation_pages=5505-5509;citation_volume=175;citation_year=1993;citation_issue=17;citation_pmid=8366035;citation_doi=10.1128/jb.175.17.5505-5509.1993" />
<meta name="citation_reference" content="citation_journal_title=Cold Spring Harbor Perspectives in Biology;citation_journal_abbrev=Cold Spring Harb. Perspect. Biol.;citation_author=L. S. Symington;citation_title=End Resection at Double-Strand Breaks: Mechanism and Regulation;citation_pages=a016436-a016436;citation_volume=6;citation_year=2014;citation_issue=8;citation_pmid=25085909;citation_doi=10.1101/cshperspect.a016436" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=L. N. Truong;citation_author=Y. Li;citation_author=L. Z. Shi;citation_title=Microhomology-mediated end joining and homologous recombination share the initial end resection step to repair DNA double-strand breaks in mammalian cells.;citation_volume=110;citation_year=2013;citation_pmid=23610439;citation_doi=10.1073/pnas.1213431110" />
<meta name="citation_reference" content="citation_journal_title=Journal of Bacteriology;citation_journal_abbrev=J. Bacteriol.;citation_author=K. M. Ulmer;citation_author=R. F. Gomez;citation_author=A. J. Sinskey;citation_title=Ionizing radiation damage to the folded chromosome of Escherichia coli K-12: sedimentation properties of irradiated nucleoids and chromosomal deoxyribonucleic acid.;citation_pages=475-485;citation_volume=138;citation_year=1979;citation_issue=2;citation_pmid=374388" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=M. B. Vallerga;citation_author=S. F. Mansilla;citation_author=M. B. Federico;citation_author=A. P. Bertolin;citation_author=V. Gottifredi;citation_title=Rad51 recombinase prevents Mre11 nuclease-dependent degradation and excessive PrimPol-mediated elongation of nascent DNA after UV irradiation.;citation_volume=112;citation_year=2015;citation_pmid=26627254;citation_doi=10.1073/pnas.1508543112" />
<meta name="citation_reference" content="citation_title=Single-Stranded DNA Binding Proteins;citation_title=Detecting posttranslational modifications of bacterial SSB proteins;citation_author=D. Vujaklija;citation_author=B. Maček;citation_pages=205-218;citation_year=2012" />
<meta name="citation_reference" content="citation_journal_title=Radiation research;citation_journal_abbrev=Radiat Res;citation_author=S. S. Wallace;citation_title=Enzymatic processing of radiation-induced free radical damage in DNA.;citation_pages=S60-S79;citation_volume=150;citation_year=1998;citation_issue=5 Suppl;citation_pmid=9806610;citation_doi=10.2307/3579809" />
<meta name="citation_reference" content="citation_journal_title=Nat. Rev. Microbiol.;citation_author=D. B. Wigley;citation_title=Bacterial DNA repair: recent insights into the mechanism of RecBCD, AddAB and AdnAB.;citation_pages=9-13;citation_volume=11;citation_year=2013;citation_pmid=23202527;citation_doi=10.1038/nmicro2917" />
<meta name="citation_reference" content="citation_journal_title=J. Bacteriol.;citation_author=N. S. Willets;citation_author=A. J. Clark;citation_title=Characteristics of some multiply recombination-deficient strains of Escherichia coli.;citation_pages=53-66;citation_volume=100;citation_year=1969" />
<meta name="citation_reference" content="citation_journal_title=J. Biol. Chem.;citation_author=H. Q. Xu;citation_author=E. Deprez;citation_author=A. H. Zhang;citation_author=P. Tauc;citation_author=M. M. Ladjimi;citation_title=The Escherichia coli RecQ helicase functions as a monomer.;citation_volume=278;citation_year=2003;citation_pmid=12805371;citation_doi=10.1074/jbc.M303581200" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=M. Yu;citation_author=J. Souaya;citation_author=D. A. Julin;citation_title=Identification of the nuclease active site in the multifunctional RecBCD enzyme by creation of a chimeric enzyme.;citation_pages=797-808;citation_volume=283;citation_year=1998;citation_issue=4;citation_pmid=9790841;citation_doi=10.1006/jmbi.1998.2127" />
<meta name="citation_reference" content="citation_journal_title=J. Bacteriol.;citation_author=K. Zahradka;citation_author=M. Buljubašić;citation_author=M. Petranović;citation_author=D. Zahradka;citation_title=Roles of ExoI and SbcCD nucleases in “reckless” DNA degradation in recA mutants of Escherichia coli.;citation_volume=191;citation_year=2009;citation_pmid=19074388;citation_doi=10.1128/JB.01877-07" />
<meta name="citation_reference" content="citation_journal_title=Cold Spring Harb. Perspect. Biol.;citation_author=A. Zelensky;citation_author=R. Kanaar;citation_author=C. Wyman;citation_title=Mediators of homologous DNA pairing.;citation_volume=6;citation_year=2014;citation_pmid=25301930;citation_doi=10.1101/cshperspect.a016451" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="3′-Terminated Overhangs Regulate DNA Double-Strand Break Processing in Escherichia coli" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9.cover-source.jpg" />
<meta name="twitter:description" content="Double-strand breaks (DSBs) are lethal DNA lesions, which are repaired by homologous recombination in Escherichia coli . To study DSB processing in vivo , we induced DSBs into the E. coli chromosome by γ-irradiation and measured chromosomal degradation. We show that the DNA degradation is regulated by RecA protein concentration and its rate of association with single-stranded DNA (ssDNA). RecA decreased DNA degradation in wild-type, recB , and recD strains, indicating that it is a general phenomenon in E. coli . On the other hand, DNA degradation was greatly reduced and unaffected by RecA in the recB1080 mutant (which produces long overhangs) and in a strain devoid of four exonucleases that degrade a 3′ tail (ssExos). 3′–5′ ssExos deficiency is epistatic to RecA deficiency concerning DNA degradation, suggesting that bound RecA is shielding the 3′ tail from degradation by 3′–5′ ssExos. Since 3′ tail preservation is common to all these situations, we infer that RecA polymerization constitutes a subset of mechanisms for preserving the integrity of 3′ tails emanating from DSBs, along with 3′ tail’s massive length, or prevention of their degradation by inactivation of 3′–5′ ssExos. Thus, we conclude that 3′ overhangs are crucial in controlling the extent of DSB processing in E. coli . This study suggests a regulatory mechanism for DSB processing in E. coli , wherein 3′ tails impose a negative feedback loop on DSB processing reactions, specifically on helicase reloading onto dsDNA ends." />
<meta name="og-title" property="og:title" content="3′-Terminated Overhangs Regulate DNA Double-Strand Break Processing in Escherichia coli" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/7/9/3091" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Double-strand breaks (DSBs) are lethal DNA lesions, which are repaired by homologous recombination in Escherichia coli . To study DSB processing in vivo , we induced DSBs into the E. coli chromosome by γ-irradiation and measured chromosomal degradation. We show that the DNA degradation is regulated by RecA protein concentration and its rate of association with single-stranded DNA (ssDNA). RecA decreased DNA degradation in wild-type, recB , and recD strains, indicating that it is a general phenomenon in E. coli . On the other hand, DNA degradation was greatly reduced and unaffected by RecA in the recB1080 mutant (which produces long overhangs) and in a strain devoid of four exonucleases that degrade a 3′ tail (ssExos). 3′–5′ ssExos deficiency is epistatic to RecA deficiency concerning DNA degradation, suggesting that bound RecA is shielding the 3′ tail from degradation by 3′–5′ ssExos. Since 3′ tail preservation is common to all these situations, we infer that RecA polymerization constitutes a subset of mechanisms for preserving the integrity of 3′ tails emanating from DSBs, along with 3′ tail’s massive length, or prevention of their degradation by inactivation of 3′–5′ ssExos. Thus, we conclude that 3′ overhangs are crucial in controlling the extent of DSB processing in E. coli . This study suggests a regulatory mechanism for DSB processing in E. coli , wherein 3′ tails impose a negative feedback loop on DSB processing reactions, specifically on helicase reloading onto dsDNA ends." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9.cover-source.jpg" />
    <title>3′-Terminated Overhangs Regulate DNA Double-Strand Break Processing in Escherichia coli | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"iaeNJMN-RSp6vxNm20UDTalVrw4k3wiHEzDl6w5hfOk","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"486373","apath":"\/ggg\/7\/9\/3091.atom","pisa":"ggg;7\/9\/3091","ac":{"\/ggg\/7\/9\/3091.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/7\/9\/3091.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;7\/9\/3091"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;7\/9\/3091"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/7\/9\/3091":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022rHwvsEVwe6TpkdGUgg6Xk5JY+0rAu5+bkm+N3+ynVRYrQi1XERoFCbzCn+UK4CSiCe+8WcXxGog6DMt5n0QPiHxapo9b5beenniJ+AtnwXmWmnfmYjMNbyyOFBoDlAPq4fIfg7kSbaxNqZmRsU8mZsyMvJd02TZItG\\\\\\\/8qs9u\\\\\\\/bqJmuhtPLIWc\\\\\\\/SiuCgPk\\\\\\\/X2ORpHA7qvlrs8ghEeTilPWW+SvSPgqY+vqTv6nvfDIn\\\\\\\/dOl2F91s1hwTm3fqBbW+2myZUT4r+ZEPYe6s2AUW+Zi3gsJOH2VVKDrYgnHgqn7F+PkycGlzFOGxwVTKdLwbiMNfbWDc5eXHxEH1Gqqw90N3uKjg0wbMFbzQ80xqjiNtICn6kk4U0SCHaOEGhe5HRK\\\\\\\/xhPBc00o7j8gm9+dSf0TTRsnQyLAYIUJENM36JN8m+wSYTDpCyXedcKquj4JD1NtzKVMvi4ZwRmGAk3j9Anmr1cnmLodoCVReLgcUbHOly\\\\\\\/uJrBnCWuHyaWwXlPwhuyQaXdsxFiPalSmvZXDHA17wRyhz2oWRfyYUxKIh3hjOP2HduKBX\\\\\\\/M2AQbE9cuT3KdRAn7B\\\\\\\/pppCmCFcgqy7FXmprux2WCGtTzQBJ2kTwYrBzc9LGTzyHz1de3PSCB0C\\\\\\\/anPJrcT4+N+llTQb0+3eV7hkd2L5cVrD3FZxQoro3AcfTOi5Bk7mDW+dCT3T3VHZ2zGN48JDKXmQgdmGQOsLFu7n\\\\\\\/w+Df2Fe0oRMffOEutVZcPZq0KgdYXVcBhCHaTGBZJs3o49KiN\\\\\\\/3sT4\\\\\\\/ueGbVPa+BxJ7OOwdUWqqX35xA9LE7Kh3siegWVpd0Ifgv8mn+SggOZNELCrq2hB8X1cvQsBhsaAbU6FMh5\\\\\\\/ziXn3VVaFsCfiRwuJX665kW7UaR7R3Ek9zbdypJ6H72EBS9VnD6x4CjcpyFbQgbKwIZ0yeV4rWy0nTp6bpbRoLZvwK1hjsYABMA7yfoHCfZmZNOJASW+ER4jwx3np+U9hc+hws92wOQg1UwwUTUdrd5AElb4t5wpuJ6Lbo16EYsNlpS7GnjuAP5Shc1eLt\\\\\\\/0576UB8yclz\\\\\\\/E+FurfrY2yysbeGG\\\\\\\/Oh6afqypfi0wy\\\\\\\/nNlR+pwEJwhtTamVd\\\\\\\/yYKSuqGdaAi+WdQj80v\\\\\\\/99c2cFU9Z7+6lR9RBRJKW3oDwPWiyywlLnz384aul34\\\\\\\/KYETNv\\\\\\\/dZhUWeNedl5GvEa+Soluhio6JkVLabmsX+n6RsXlNxwXscXyuvKufkIBFn3tQieeAB+hXwR0HfZ1ApIg5MvSrN06Tb\\\u0022,\\\u0022iv\\\u0022:\\\u0022h0XiKiKMv8qU4N12v0FelA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00226eb73da684b2509172a84f3e8e5f709fc1b99857c18ce407e97c9240ab490aba\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022GcT8I9SXNWkYpsJak4BMM1b042CR6X0ScjKKWfMNEQB+gwPWYXQTEgA7+Dbi0lA2vpGeSZdE\\\\\\\/XvQWBe4FD+UZb0Y70cHBEgVCQTbKdw9ult9rtFYl6kHtsxzxUVvk4UwSB4\\\\\\\/jk7htCFEXvt8VppNtQC+XuoQfsKdZHDJVxdrZkSugAsOQJ1friTk23rWMnQi\\\\\\\/oRwH7gKiToXMIxy68Mi5w2RAxlKbPH6CEmZmaws5yVHewKIqHyF12GYmE\\\\\\\/CahqwVc5Cy+h7dxVuA\\\\\\\/bQIgzS\\\\\\\/R6c7SeLMHwLU7YhQah2yPoDb6KIhGriFNVdrDbQl2yesoE76IY1dzrF85wzsB97OOJCHrvKZsxLO1xMANzcDfn7qGPTuvJs9s3cB+nk\\\\\\\/oHKTlg6BfjJweT00rOB0X7KEr9yqno9RDe5064MhNmmgr+k65JxvX\\\\\\\/eXNeoWCjUC6A54gOcG43ODHcFbsdnArjZAWuufmU65+wAbT3HzquAL\\\\\\\/bcBzF5zuT3rowxBbpNiXW90X4AHDE+r2kd1AbFPrzctK8DZP7N84mIw85yLw7XvNuuottKUg6hqHWM6CqexyG6XqzHst5vBDsWaBLOlK2ekaBVFdL3LWqoImrROsBEHnjQg3RMyq6pMrrkU0hqTGSof7ig7Y5CLpHXUZy848yk1k80mmnFC5wIuVKRqQxHIQgdVxpYloCY+5dUgWRz2So63fwfwjUMEvZWSas7zzIxsFacRoeGMZDXF10thvv3V0HvdNj\\\\\\\/3clkzIH4r5D7KcRMKl6ioCcWZPEXlXO4o+LQsCfh28PE8LDs\\\\\\\/COjKJCA+ri1WDm3CFs1UgBNMfnOWGMECju\\\\\\\/URVdPsRu3beY0wUmLNOkVBY1FMFJDQbVaqcwbBhBeJqhapgkSD48mn0g5VvSOOifsen3\\\\\\\/qDevhHoq2f9caIQ9oEGFJqdjH0hZbUXjDd2g3aOsBi4rZ+UiInlCp3cMY4u5fUwSSY\\\\\\\/lo77kL57DWuNLJGlUM0IboQe1IOdwtbiUgS7f9GjeWA9bA3DH27g0Sy+iUX7fmo4Q8VmvFAy2FUM3ygqyQm4G5Ecl9Zf68BSavwG2hjLUeORIKfaiIpAyZsSy2I6ROBhNe\\\\\\\/719ipC2qQ\\\\\\\/He+9MIyAoAqINugYKrZLSuK1TdF0Nx6ZBoL8LvkRQFLLczUCEALWuBx67Ofse\\\\\\\/HBP315qELGcLE8uIWRSY3vK4JJpG2srP0CGms8fOF6BUGH5UmAreayPGaobGuM+9C73I3I\\\\\\\/QqKEXEkM+JMZkG3KQ\\\\\\\/fqh7+W0N6XnsRF+DpM2WqgdNLD2B3UjqNUgpNAvcMRgu\\\\\\\/jNQh3KJcA7gPx5IZsu0Yh3NKlGDWGVqHGI82Hc1we7nQRIoIFdAtVD6mgWu8\\\\\\\/5dWHw64nZZbY84uezf7az3TfIusNxy5X8Ua1ZDkPlCaq38AzeQ106TYaKW0qTTdqkCP9\\\\\\\/DYoAALdCkiHcuDq3Yzp2VGFcAUWohjcSJAYvY16mtoYpw1J5qWtswqWARq0ZBCrBqgKuKsDfRjzM4BL7dHSDiBjdZZlV2UEevzcynl4XE05ZMDFOnMTeCBbFzg3VvHOav2+Q8tgTgIR19x+17XPVgMG1ONB1IS9c4qtbmmViTmhMUxFRhErfrVrD8g4CgT+o2KWywyhDbe3tYtkpG40kdmPrBmhvDLpEsX7rV3vodOR2HKf8cwQrCWKJQQ4uPROe7\\\\\\\/hLNl0AGSVxz8OPjSfOcrOKXoY4X6sZJck14RNbSPOEtkQOyM1pbySV7vfnjrO2l+LPs3tya8pVNLaY3On85Dcnp7DZ3VKmsMBspCkF\\\\\\\/7TSxNue2lmR2u1iPXRQEpS5ygmjxIXYKgUbsOC9AubkZdPws2EqSSx58ExpIuS4nqGo1h616HDuocfz160hP3ZalImf\\\\\\\/TgCPzikuoRirLqA\\\\\\\/NdK6TmrQzwccSb2SXCcB1+mZDtkkVaCC\\\\\\\/6ZD\\\\\\\/Oas9EgllQBCGe99ZLJI1qziMl0ZxEv\\\\\\\/sDg\\\\\\\/LLMsOaPsJ68oocPS2ekxFH7q2Sl7OCl1BsgUvfGsYh1jY\\\\\\\/++z042lcZbDsm58wcThxMkvkWYwPrKTOz9rXUo+rpq2LAZnl8lfPNF67JK3LBdV1FPOvAD6JEOcv56Tg0VzcpB1ziH++D1lUFqqPSJK4vJ9\\\\\\\/zGTeXUfcheqc1\\\\\\\/VsljgLtoGimTQET9jKYf8rLUkT1fwlMws+F9h5vQLrZR1g712VWrHxbS6yP93V3EhbZ8ifgkbPuICzAR8GkHL\\\\\\\/B+76Hm2kcjuZTvGuIAgChfDkGqcyQNOAuHeEliL8iwfbztcvxDOArUlfll8+68z9JSKFO0Ex\\\\\\\/nRpZ9dhAXW7XzsSb9A76cduXwc6tKFOktNFvitR7eZLBTkIBGm8SoPL7fJLdJGpJCVAMm4UUt+hg4FlIUur2laz1hzylRCoxwEEIkDldAtATgaOJQL1GxeSS\\\\\\\/ckXLUA8851sr1NTZTtQXxCYlqqo5Q6zNOzGEcsOLk8BR57MSPlWVNwembF5he3hiAYGY661vVLDx9Wz0YK0W8gMMuSTdZypCNEY7PaDqY0mVS7BuW7\\\\\\\/fKVOgw4voBw48suNZears\\\\\\\/1vNO+U4K2kWgL71jsYsv0KCKSKAYtylKVevdbmN0ASVWbVuDTQOsA3LmJKx7kELyu5yRAgcnsjRzCX61smtu6v181Z+c2sl8ggFiWe8t7zr2XY5nHeunb8tLj+iCT6RTqKU87TyELKQHvX\\\\\\\/EwpBJzdoNSkxAoazsfSBOO3yb2APwqaxeRwIuxE8D+MO50yp\\\\\\\/zgVAsoXUd\\\\\\\/f2lrg4nA5ahs4KUcB8SCFoJw1P\\\\\\\/TIitOqk1IJfdYokRzewc9BbA6beZE0xufdPyGQVmrJElFJRrUu2A3sYHR5zdNMfNjWmjMyRZdHnNbC9sfR8N9rmEnsct5g+YYeZOrHME5EqDGyOojvXcZLYqNjjJT3\\\\\\\/KyLGDv8j6RmFICKl36hCvE3f9bNcLFjQRyff9ZXcFK0buS2GemdMhQXOZBwlG1PRijAw\\\\\\\/SSTTLCMKrIVwrFPVwPWyHT8EEdshoWPSGUMMbdcqgtCyPR4AgtN+tq1EVx+lSucJmssb3rxfyt0d0ZiH\\\\\\\/aWF3pkmXtPqeNoLguDc5ODlBewOu1DZxr41BSP3MOvPMGubI9wjqY\\\\\\\/Xj\\\\\\\/DYQ875Wmj89nNoL3cYICVztClfARegBfjKewitREXbXdWOQScNeX8lrLEIZOet0SvE0PNV4ZC7GJhFZHNydNlKnmZgz+5AM\\\\\\\/sVbwYlFbkwKMacsVZ60vdcck5HDVIKuk5714qZU9QJ4bve1eiM09npoNyxgRUp9+N7h0VJJyvxdfWBpP0\\\\\\\/7zq0q\\\\\\\/XIKgkyofX7CTV0lJUzYR+Q9U4B\\\\\\\/3e8butgyqcpfTcssg7MIL3VwdlA3NXikitgSYvQMjbh5Tj483PnceNQdg7NGvtewMmMC2xMlbD97Bf3mJEZctebF9VfN4V65hEHDeOuU550EOJnB\\\\\\\/O7YSSJ52LMDjI7eM14BWotix3ejuF4X2B6kubdDh8NJynKQjxhDNl+Kb\\\\\\\/8Vos81nW3k\\\\\\\/OL2qGIWn0vPADdRjeFb43Pq0egS4JuMd+KBCzxXgqh4SQ\\\\\\\/gsx6HQmBoXM2AUV+PM\\\\\\\/4hhu3v+KRBTvVNDVJ582Ty1RIbJn7Xd0lpRpPFywuc+5exDj2tei0AaALIUZTShHXKN2ZFiWJdLOFWBAwcQmuLHjpcMbVzsdQWVXkaaiB3H6VF\\\\\\\/bcAX0M+IZSN76zUQAJuoFmzLLph98+cROniEhZjKvBQcHZwquuMhRHqgd+ZsCbHAx0RKVjjAhbJZtJUMyf64Py94uyxPdi2qqTCmLpyaKN3zFQ1vLx+L8idcATVDFHC4SVXtTV1986GcQCBQScXWeBTKi+DTvhEyrMgUhGf6wh3mnIrT9gtfznMIkEQoxjwjladSl6XXnovcMrGJi8ttNQqmFaIoZsmvVVD+wz7oVDhYSoK8d3P8MPYwPdregCaozrIM+oEc+I\\\\\\\/y3LLbkAnSEzl5tHnsxlVGFgPuZ7GRSenfbzLr9acZWdqF5I2d+nDTQ2dSVdBuxs9zL+pGQlznMF4vxjzPbFqSyaSECs2FohDZmN7prqbyd0Gl2pe7kKrCl0AxSFlTSrzDXswXCYLU\\\\\\\/4p6\\\\\\\/pDMi0n1Ry1TudCi8m\\\\\\\/\\\\\\\/4J7VwLd9wqXPITl2u9bkJ4FFaVXSV7i+HAmoccdBSKiBNZIah\\\\\\\/u+YguPG7hcKQS0GaOECMjBCfxjoZQum8GVvl71OXGmW69HhG5+qVOTJKnhsSsaSx1+ZdLQjqIvDdoiKI3Ggrok1dCfp0p5KwHhDF3dI4YEI1AkurgAHtsCq44rQ5zYhM4bJZFnJ9HeFGHk+Drt+Mh546LdBcJMZbvGN4tKG6ryR89obdTnIb1oFc2nWP9r5GduJIFYGN9BKId2GOAiec5AwuWssVTllAcK8GIUsuz4dTX5ChdWRX8YiIoaYXJtf\\\\\\\/ySYg2VngqyUKkWsoi0xAN\\\\\\\/M9c35XAqw04\\\\\\\/tm2YXkHeSRXmYT\\\\\\\/8bPbM1lbepe4BvTnD\\\\\\\/loGrLSj5YpwT0oRMzCA37kpHnEpxaol7yB7JM8NJXZowbpkCaJhpWtUuL\\\\\\\/yfD0aX40KyBvI5xMNGEChHq3nl8tWOelwm\\\\\\\/wWOK62dZXQT8hjuzgA8CgvMMsXQYDNxdmtKAz7skB5Z2h\\\\\\\/38Q9Bua0TXb8Iikmlrb23L7i5IgCbAA\\\\\\\/4Fq4pw7quzDGaHkisgXdsEuW7jdFDOMbtIwMUZV4jtlUQIIwrBY57gSvAJ9i76XLloywhUUFjgntdwBpxYmNNkabFhyPGUkKyC9+q3mY5xyWxkEcFdZT6CBwgkWh1eFhCs\\\\\\\/xqDG4HucY+CL217qi6FadPl02OqvsX5Mi0TVbvwMxsMDOIqWt7ZXsIQnfUjHEy4bUc0A3EnoGpzJN3RomUNa3tbT8TTC2MXWZ6PomBVN9qbS1urOuX6esR5llc25UWyubGADDoiyoUGBFkq1SS5NPGLKA6aV8\\\\\\\/1ZC39q9Sj9s5dZRuZjHkYFvz+thEQS3vASytK49CPpgzCwjCRIGBzsEb5J5fpIj7MHiWQK0px5WVSBvnx0edkGbtlSU6RvQOYJhFEo+4scLpiL47Vmb6NxT30rA8NFC1adFw+AWAuqPPWLfIIXIZq1ZZLR34lk7l\\\\\\\/9LzIobmF2CaKkUI6Bn1gjkhyOPpoXkcyJVWIYIrPNlwwph\\\\\\\/oxy\\\\\\\/aUtb9bnec6+HIL9yN\\\\\\\/4kk5I3BHEY6pefA3vtQI1\\\\\\\/dEu4R5jGhlPvWeq3D\\\\\\\/VCC9XwDD+kcN79ZcXC5xSWC\\\\\\\/tQGKbSTJuanSa58tCd\\\\\\\/tK8oAHdPqvnX9WolXxSvOtHwoujFc9Jr1OUoOWHnUu5B0SN+hpzLjdX\\\\\\\/HyUbrcZ1vMGLrT3FmFgHGe2xuola1sfCVq7U4m8vqxd+Y\\\\\\\/K4yWv9EoKXkpmEy5ENaYFVN5mfMsGez8ZA+jmojiz6mtf9KCFa9uFaH3j7nV4yc14vYEqH0304gw5o\\\\\\\/BQ56groiHlph3jFx4rEmiT5tWLR63oBJEt1G9BeTK8g0cAFtdI1OhYNu+KmhPimLfuyCb9DN+6cC1MzHjww\\\\\\\/K1kPStIkK4rU3IXpSXIdfXy2eq+ZHc\\\\\\\/ZPNNeEZ8F5rVbXnTRoZsJiPonfYliNh+fMsBonzGjMJXQm6C+KCmQ70R0s9rsgMasXCcSSv7cp6yZ4Lm4j8tU507GBfvu4I25Gj7lacMmnwExHJ8VxZvmp0kxDLsUytljBQyCgLFFQUL2HSSr6Qo2y498cujrY1jQHwQEz5plFkHb+7Tg1Fm7XJt6SvPNGGEkmroTH2C36EJ8dkiOcSLrBEhUg162EGUcksri1yP5PIK6kleKufpP0syiKETjzDd6gaexypUbe2FLB6v1txwyz+JvypMBzEx1P+zMo\\\\\\\/9YuU4T5AjDH5Re7Q2FVmOfNcUcIoFnideLF2wOChc2EGVH1beaDuZJbxjGtedPzsqNSL9IPaZZ5G18E1dPjUgzb9geqgEbuxdBx68UNoNCsUN91Y9361GHTplq8\\\\\\\/jJbmRhGgT\\\\\\\/95AFfLaMC6Hy3ocsLhn\\\\\\\/yNZ1xcpVHOKwE97XwDgRx5fatCF3rW1MwZGVvEVabsftDT8m475eQCakmD5+RpIgbEF9c4fM41IhOk6NZ21I0t64RiO2Gc\\\\\\\/nPqAxdlo+ZTtTlfD0S20twaFAIKMXlnNoVfkO1YdaHKZjkb5CI5KShMnUh0uGoC\\\\\\\/S0kh0\\\\\\\/nOMufAt+6TtFBJnO3ihkOiQaOZKZJBhrFxhTPF+Glh8kBl3CsuoNb2NQDz+MKVjWwPukE3wDGFVdde8OlF1neMSx3KSO7XUGOJCJv5b9jywBAjBogUVpLlJc\\\\\\\/EXW\\\\\\\/i6ua\\\\\\\/4+g4vUYSegYOJQmOZFMBjp0YGDedYeg8V+VBAe\\\\\\\/01ZUcg4vTX9Fk3QV8qwCqMQm0q7L5RHG+pWYqq5yNC0Vm8SdfiH\\\\\\\/5daBwQRixrgfrwqELbS6XaJJkVzxCq9HJJMqu3a3DVGEz4ELPiSp1lvszGa+qt4ZJ11rLSMlCLF+c\\\\\\\/sOwdrJMpfbumsxCw7nRKcK7zazjFboxo26WcLrU0NvbAIHrlL9v8O8ZVvFSVG0NdSbLXhilhGtYgz9B64HKMuPw7CGX0oZyQVWlD07ztaXTpJ+Mcwb1fpdw9lKauaqJRPhDWRwnZJGrGoVPBDwz4hryy2aKzR8vabnTbX6Z1tRxzZZ3QGbWRrsTtVF35vjsTx5lhiBkw7rsIJCxdpqK+tFhf3KboJUGG57Oc16921WufvD7BhDzNQdTPnyj6pf9ZXcv37KPAS6gQPwRyj4y4QU7PwJ1wE3YL7itPzZ\\\\\\\/nRbuRUpNu8zREU4vt2hO25A6n7YSsTh8zj\\\\\\\/IR8Tpf2GFW9M+9E1TomPMTv7MQJQEhzXZk+NvIYCfrMKyrmpXzDEk6E00Uqf0+EU\\\\\\\/aJlFiAOMw8Czqhx7yCJFjwWNwc9Ekaa15+kx9XT9CmLBPq4JxVmGpxsq01m0SLtqywHadTb2PuRCy\\\\\\\/mmjzbBavK0qV92D69Npn5hE1dbZSUXUuebyRHNV3CJsJAYGP5r\\\\\\\/Qd7pUwwz0\\\\\\\/pb1QRXG61JXS8izptfj+loY1ML0G6WfoRm+L37YcXuwPLxFNt\\\\\\\/tRuz2d8V9RoiGHIzKN\\\\\\\/b\\\\\\\/pMUECe846UKZlyzTNyq8gHCFz7QcNZjJ94QaarP9ra2EjFP9vhJo3D1sjrjb4CmhJTfTcMXo7cym4j4bSD\\\\\\\/8h8OsYPU6SuXXoKrHNFe+qbscb1DaeGfty4lUgHl7q3ix6AAYj6XZfkpzNWciFpN7bRcDci9Lb+8CWbu\\\\\\\/FzO9C15PCXPixCIeqfRBLt60BtpgUvGpQCOlof8uLW\\\\\\\/jrIRWIjGO5tekEqIe7gXHh9hWMpqs\\\\\\\/ZOp77\\\\\\\/QHFGew1aevBrxNtN1kxF+3aj\\\\\\\/+SPBKpyN3YRGEFxPbLhXHcXap125zqO1gXinj4IjvGP0KoZv7\\\\\\\/EUPzal4nJCxN3nTOkFQtHWIGCr82s3UQ8J\\\\\\\/aeDSQVWNXxNM4AjnKAdNqoHs9imlNUR76XzvpEaHD7tMKe8cIanY1Tl1siCKUEV9uDrhN1FXJVVMGwLpFgizfPJoWsL81qITFiitp03ufmv2EeGwRnV\\\\\\\/58DDVmSEFJJhQF\\\\\\\/PVdiresLLkrUN4QL+BWE1XhhsOXN\\\\\\\/WcMYAgVl6XBjdWWOVUhHXY+ESLICF8s\\\\\\\/9JEejSXAwEAi5XRkgKpVXPK76ssOUdNM\\\\\\\/gU\\\\\\\/5tsnkKoPUBpY\\\\\\\/bbPzx6ewF7wrxGe+8Gld\\\\\\\/vUol7FS\\\\\\\/W+PK+2RqXd9P5cd0kRY4ADkWzcobtq3qayXJYcGU3jGuQwPpdL1KdAIKmjY0zp0Y2cNlbopudZzadPAIOUEYxNLr9fzfFGY8cCLjqTEMzzDq+1nvUn76bys3T7NSAyhYJS4RoczXSpBmrqUJqSaqnZ5RO+Ir2W5J7fChnBfLJCJNs3qD3FalTPuVe+Qi1GimnqbwAyIb0t0T\\\\\\\/vrM38pyRSi847u\\\\\\\/gI4Ec2b7948Tz8ahj94iUMFa0YRnqzJCSGm9FmtZY6a6MkqovOejNqW5OgyL+Awe\\\\\\\/bF1YyfTo4QU6tOCzB\\\\\\\/hxID+16hpRGHPnLL4CgNfJl6KZ3Cj8vba4x77iaCvesZ8NKS+fHcOPF\\\\\\\/iGmpe8\\\\\\\/i\\\\\\\/hMlino7nLaDuQo5BJnT9YBzkwp64MThVTTrqH3odLQGyIgTvr5sPow9MfeCjEV70NnK1yFGrm7Yd4RBfKyi6Eju71rJg7QXQ0opUj+IVbwldqTDGbWmpMjnTEH\\\\\\\/RLDl1skBIBV34k7smMnlpBXO5kmrPbVnX2IMwiYuseV5mxKfLyc8qfDmfMRbql0+g4uH3Zw9ykmIG0ZjElMZPCQNCowXZZvaXhNV5usUuf7gtoXySSyfC+K3Zukn8hT5Yzk2vhJ7if6LV\\\\\\\/XCKlHUdXuSxYKZooBAZdvI+uNfVJ1CwE1bet6GRvBG+pWybQMegKr+3+aEc\\\\\\\/dvJW3MHaUxn3i8uWxwZ8UhPk5Z6fcQGWp6ZqUcXipJEJpwgLeY3FK1E9gX88wcmT3JD4rkBa5VKmbDvewJwPEksSciepg8Mgg658Zirr6p+cdVz6ROTAtD1evPv\\\\\\\/VCyU0auF6EwtK5lB0R8x4kQHqg+5MeY6X\\\\\\\/ecm4lR+CjwtVZQNdC\\\\\\\/2mrh\\\\\\\/F3iTwWl39aArzTbjU7kOUI9n5QQ7LsrByk2eENz9Lz0+BWvRCPwatFpIBKJl+lNMRCJl+HGoIUoP8uMekMVUN6QvmSsACzMOrqgD8PizZyDosorkD20j\\\\\\\/bLjH3QetVuldbVFd2xTV+\\\\\\\/uQnPMDIwic2sq29IPJ9lmIWjC5A9y8IZ0pj\\\\\\\/ksUCBk4wlYL3K4o6BeqFKZ6RmHSiBAZ6XfFe0OtZgL0I0hJnUUAxITQ9X1i2RX3QGpfeef3EQyZ3XyPZBIS6ZjDmR6JiS72862JvZzPlk5B65Q3hJWl5xZPzTcjf81ACBz8xW3REemflvsEZYlY9wup0W4B7w2zcxFArBug6zFuAkaNqFxehSV\\\\\\\/MwKdo\\\\\\\/yX6yTpbI5yTmEi4lLalCNPDpBwiAuZSP+A2KiSAnfYxFskErz7hLjvWcPEURgmROQjKeUE6he6iDIitS\\\\\\\/EO33NImicTrsiESoch3irq+ZiZDVqozNJiifns71fFaYcT+Bgl7HVs96xEBY0JNrwpGXUxwhhJV6oxAFTHoLtzmPlPa6bXRI1M9WCDmUR6gLbY7iSc3YVIE2DkhL1rnZ8iJQdHSq59Harx1N7IdiP05wuzVQzaN6Qlt+eZBK3gR\\\\\\\/bdyDu7SwM7MAU231HAQnR0QHPC75bvNxSMEjWKYXCRMHdU0aTv+YnbvYjnpA2zMG4++qBk0JkjGj1JqDvnzjLvUPar5h\\\\\\\/InlSOivQRESH5Dj5h4J3SnbQFD6egM4+0Vd5z2p8fzriMVyOl5QVfFImdpgRWxP4sSP8Gq4q1LodELczXpq+aTwDQAQPPpXEZlBN+zPO5y7Ag\\\\\\\/kkVMDOjQzo9qzixkA4wQ793Cue681JqeDmIJqubYe7j0Q59M2fMoGgvRDIShe8Mg0vPiyvZecQtxREqEB89eM+nd345G1+zKNB9fG3O5XcjtULCRpN+cT9AAvs2NzS3MwXo\\\\\\\/2HLAp2n+h14Yy6ZwbfiruL\\\\\\\/v3VKmN04k4+FXAb2YX5jb8tCRYxOBJYHBOzfA6Qnk5\\\\\\\/hwBSLJ4MTsSeXLIlcfiNqdI\\\\\\\/hdBVR0x08BVZvYJBq2Ln40QxcWg0EqNRZHEK9YtePOUrw\\\\\\\/x\\\\\\\/eA+lzkxJeVNitpHLRUPVOp\\\\\\\/1IyIczTK10YBYGQt57rsEmFIMtlEMzC8wKwbLZ+yoCgwJ3xNRB\\\\\\\/MMq2UJWfAj4DzihDLdf0FPUY8p9Nm7FgH\\\\\\\/lbVE2sIkSOn2+EOTBClUKrK4bxCyc7fLBWjTGCj9sJZWN7Z7AxiT9NXt6\\\\\\\/NLAWSPObqdiKI+8IZ6am7fZEObjkoD5Ttef0u+X4774zvh7OrnLg3LFonDLlQ7Cgna0VFPG8X1P5E9ya6dNxHytGXUD3SM8YtiXalMsgnrzsr4C7ZnzEbGM1UN1g1LQGnymgBxtuHQL9K7GsVOg\\\\\\\/xcpiWGIzXLnpKa0Ol\\\\\\\/zdG+c9sruqJOC4lLzJy2tBw769Nj7ktq\\\\\\\/Fq\\\\\\\/UsEUX2wSzC0m7L7BNhTMl9N5Ep9wizcLu4oiwGgcOSbMiEMh471o5CllK9AQ1AG\\\\\\\/HDWid7YdoNZQBcnKnIAXzQrBqGJ8ZI+C6N\\\\\\\/SAgrikangY040ZZAgw80wZ5shjGooHi23tX5ug1jMfqVt8R6exU+ywGi8XPrLCUTeJMoqZ83pHo1l21ak8dHL+eAKW0NYP6ZVQYmaU1DMnEo1Pp9CP3WRgufvh1VUs8uVyseNlkhZJqDrjgqJFxLLyGt58giT5r+VoMUWZjxO882NGIwnQENMo\\\\\\\/3u53Qj7Fs99XjNHJdP2WaIqqCLjvxS5C8HtrN9nLUnrsZ4W6Ilo1BVWEqWh3poYpVzhKq84Pj1syVe0JJJwXw+8cpj0z\\\\\\\/HNzSbP0fO444Waznrh+6a23WrWtusLyAXmUq+N\\\\\\\/lGTgyZpuGTbfnZE3PlEdraQLR1Uc6n3dMR0HmvIBWt+bqnhbvsKX1lpLAbXXgYHIeOMQ+ecZ77Mh4g8uWKuN5mRGs\\\\\\\/aER9POEogEFDQbzpb2LFgeTwqPz6vZUW\\\\\\\/iSpbSWQwLVIQKjCYc8tSsAjZyR2FWy1E3EgzpBB\\\\\\\/esV6nG6OL1LplPq3gRA9q8NmYxiYKkfkgbGlS+DpiJ9XicykH1exwpzxR1czycYHkLwiLMxWfde9wP83tOoKE\\\\\\\/aOr+h9FQCkCAAo\\\\\\\/91nuD7AEaNZAYSinLsIKPARxX+ArTWn2X9JPZF+rLZNwj12rPBxBkxUne4E3FHKUT6NJp1yrqve1PpDjTgfG3FsuFMUHoZL6e9OK7wzCl7YkRax9SxeCrHuk933eFLvREOE9Odwi5b0QI6EOeak98yuXfIrYkDVwlwVKw4UB7J9UkC4nDZxdSD7T\\\\\\\/9LstNzemD9N82piHzQ9YgajNX55Dr1EtYaV9p48F9MlI2jwteO\\\\\\\/Lnk4hbGBb5llAxTCezHvl9j4sR1SQfat9W8He+49NPhjppm5dy6Qlp7OiQ\\\\\\\/7QTRY\\\\\\\/F0oZQaA8tRH2OWNSLRUwm9tfx49rB0oX5SMpQ0+uLCIwXdLb9brb4nYJ3xka\\\\\\\/omSpPlQWsS\\\\\\\/B\\\\\\\/QMsW93KCjv0tMggU2lWb8sVUv+J0gSGyFqLLXMyP4NBaOyWs4chfahU0Uot33tHoL6G4\\\\\\\/\\\\\\\/uaq9kaUmhJotxKejWWs\\\\\\\/FySMbStJc1FyXVhqtmdantez\\\\\\\/RES4ycJMkQ2gyYTyuL1HZggtK\\\\\\\/7M476Xz4AK0Ig\\\\\\\/VU0YvOXZ+lh9Rb5Gdsw06SKgifkaJbn4AmvHJp0fcEEdBQd+6Qmwqc8zOVi7X\\\\\\\/O88D77lYlZ2\\\\\\\/jPcTNJl2aMWa5XcHzR5rChfPcVIRSym+Z\\\\\\\/530Od4c7bCUB2jn8nAM4s0oyX02F6+ytgJjdHpxoGT1Q2+PiuQUV4HatjOSS0WmSHg82RYeC3Tuc5nbUuTysU6SmNnPWV3f+24Qw\\\\\\\/Ad8hyiFZp+JHRNgNMejoeDMHpMIZdr+eWNcm4c+A1sWKqOl\\\\\\\/ZGlW1Wynetdrt+ICs+YH\\\\\\\/TkQWB85c7hqHxdWb0v+LaeNg0dCcsVDvEZJ2\\\\\\\/gyjM+Pn5zj7vxc8ro+QPG71SKWOt9gxRgwLOz9vv8NIYEDIKGIfedQDOlwohlKWulouNxNZvBjO\\\\\\\/GVEtwCEzCXT92UD5MQJqdLIqr83dMW5TVkeDx025KwVwQYc4wfkIyVve2r\\\\\\\/JQqNAgQtmY0BLHjlGcJmCVCJXDG0DmIBkRKq00nT\\\\\\\/BgdXWSxwjR2oBaL1XGUv5bsvfxwKD8fAyQJjdwqCR+CaIPjt3UqSkuxTLkWzsA4Z41aVYNPMgVTdGbk0mYq+EuILZCRcKoyKlGGiiG5o+pEIT6OT0woT979BesB0g5AKdnelTvA6PANku5a+EZcXU5qVxuOIwaAFvqh68hdl4sKaKCrA\\\\\\\/rRnFchN\\\\\\\/OCHO\\\\\\\/zGCq0I\\\\\\\/6hBVtVU8JQEESfFMRjJ1+otwWiJAtIpNojB5D7KimjuVc6OIcyab1rBw+MLKH0Y5Sr5bntpAVtCSLwoD7XZCtjLgHob6dXsu3tHZtXbtsErGMg0385AwPLCIvA2CN4rp8Hphi5sMK2+Ff+X2eaLGm9wO3ixlF9eALV+CkfOSG+sFfR91sehdaOhf0Lj5VvvuQ8ZEoSpOSvjHWPfF1L0maJTHVW5Fmou7K8fUBVgCiNgXaENTVrO6HqmuxaWWbnnEssu+hM4cj6Lp0E16HSogPxtTTG3z1GBC7SqMhZIsw9Wl5ndi8VoZP0OQTjcRA32xqHim3AqK92mW87Uf536zSRuMjvbBF0edTnW3WHzFSshc9iGLdXxtcKbIjTOtN8+KI7DjnP+zVKZIgsSsUu4O6IAz\\\\\\\/T+6Xl3LHc6TBgVBQAJMVKpC4mW3\\\\\\\/pJEpol5ctST5cTNjV6FMYcbFSr+e28NbGNkISPfBBVZ51VMQcvK4geshMCNNc+9lZ+GXW+fZancvzV1LEkpyIpxc1bf0vN3Qfzkfe1kgp8QGGWLRR8JsLR+JM1EmiTCD7GsxuF8jdxD7PmyfYZn6NRoeDaKdey5CKu7uAfcqtsL2m9+DeMmE9dyTCXZilLy2dVYrZ9DHT+VSITu8PVTQ7wKZG4hinhGW8+2\\\\\\\/msYGhuRbJ7bwKyp2heV+yi7Oam1Ee0zfWfP6etRlI7Oaf\\\\\\\/I6VTYYHWHQIgvaSmXCJhZ+U94PfqktwH27FSnjmYDSdxV94naa8W62gojlG4Pk1t9GdakX+WfuXcNHVed5dbhcqjw0I9eJT3eXAc0w65dVyUd2Y4drTZ6h0yxXQ6gTefZMuKFKow8FOv11miGcbBEUlCOXfCiQwNzzMJpZqDO\\\\\\\/WJMA2t6Jaa3dUDFViTURyE30AGXnNk0At2UXLOP0vZHNsluaUXeVuGbzlxj+WCh\\\\\\\/5CM1+orehWORc69qblqv+UUdk0SgdqOGrng4M8g5ea6x5RkLz1mzOvLC90SN5PU9JQdn0Vo5Khm3IQ\\\\\\\/\\\\\\\/RzWtiBk5xuolayl+fiWXcE3UA1CSchViPXpp9n3jtZdhmFelEcNAXl5ztf\\\\\\\/0B40VhcUtU4I0w3v+8P1D1mrojqARYkQ6vnVUl2Qfonlx7xdDAEoZwtfTU9PSjeJMUEBE\\\\\\\/zvU59bRtzjptFE6Plsz5kODsYnCT2H1WiuOwt\\\\\\\/qu4TGDfw\\\\\\\/5pPpZ6BdmwylYj\\\\\\\/1\\\\\\\/c0MC2Rc1NDyi4e6Bc9ekn0ehPsPwto2u+RsiMhhrzeyXGe4ejHayODET+ExVMW33hWPh+o8L9yzEcbd95B0UmQRQSETVtxQzYHSAhdCqV0Gdn\\\\\\\/l74Afcph72TfGKHtWGfqGv2fDVucSatHZ1bMDJzMPz8m4MXVLiDvwZCI75DFpuNjLkuWC4vZD5VZB59psBNnz8qi9B2pRSRBf+kVC8vqVS8U1J30QMTZ6ZJF+M69p1ZiDmWHrTj3ZiYu5wO6ko+Wm\\\\\\\/8abkhpAvV81LfzumtnJVHKSGTnnFJXx2QY34BvE7tEceIX9a9HeYbeHN33dMyAQMykqYPR6Tvfl+we8a3czkOC8KRccDYaHaes\\\\\\\/rAYk9+gbO6yi\\\\\\\/GCYFXtp9WHoBAvzHOIRC7c6DMlOv2R57SV7eyJBbLrMufp7z+F+kBPQMa3oC5X3E6oQcoEupE7XnO4bvaFasJdhQoM0EGRtfZl6AW6TGAAp1HxhxzYHrRqjFwKlUufkHyQG2bz\\\\\\\/1LIqxuKh2qWCbUe6pER1aLXF8Mwuh7JepcC\\\\\\\/yvT28uU4JrdUYuICDzRUpu+q4Hi04sWYzyjM6RicsfWBbvu09QZn9vq7TVZNHpP5PfWfMuHwiCgt8CwPZpGobE5buJ\\\\\\\/ytoRQrwF8C\\\\\\\/Kjy7Errzc0hJxzHD6ER\\\\\\\/rX8ti1dsXWRGuR8iIRnPxXwmNMsuhap4dFC9yZVuVlOjbJ5YA0gntcpp5Pxa\\\\\\\/wFHkaxZIGAF7T6cJMYtcAiBn97G0Bo8AYUYlIm6IlROsWJyOMM11G14NDYGnJP\\\\\\\/VZiFb5zGnRe261ms+UagFD2mbLn\\\\\\\/1I\\\\\\\/oUHASBg67qq8ONFFMibA0sLIHBFS0qpMt5JzhuOVtYubYINLnCrbZL6o1XBRmKL3FywkYgswuU25xJ+lKmb85gZH+wBngbdOrHdXFVqbgYLekFQ0QkxMktw3QS5QcN5JlkgSP8VzbrO22iQb53uEe49CXXTw69ajtQT5FNo5wVv\\\\\\\/TbJj+N6SWA5gUIMnQY8U1PtGxs05WdKnP4vwIGAKKbQHZvSAZEhs4wr7KI4r2KoSy8WwdXaIe1aUr6\\\\\\\/Ea9au2uon8kXpW9pAxUOugnWaYk+1FGibskD8teYeqiz9iQkUFl9j0ZqECRV3Xi3Xe8qFgPkB7rwh9\\\\\\\/Vz3pmng5isweibhRBjWsKtZ4oUuykylvr2olZPjrZq2jbZ4nTydf+lkg2AEGQkvxqeE6NUHGQqeOEc3qeqjy4QVSZs8mDOwphn3e1P38NqZSwqiHN\\\\\\\/bEYS5zkwlMN5JY7eQ5h1LAA9i+4TuVxZA+XLe2wqSkM\\\\\\\/02mMcapJ7nte7kTRLXNKw2jwq2C8t9pcEEhrfsIgYiUgceVJ5IUFzy7llE5YRNwQJp9ulFpIONy1DBFZwHMxTZlMA8N0DT8yDeE5\\\\\\\/65u\\\\\\\/9FOfyP2urKlrCv0Ue1rHXO4dNHj3Ry1aUb0gR5b1g7287aX5eSfKvY8TNqM9qN4nXZbszJb9dAqDk6lcjMG8aQZYCnT7fbo33eJIYKNmcgc0LMP72ZFcjCPGa1k+sLSWcPHUt1fpYMGRPjtJQ7JiAC\\\\\\\/qO28QF3O7pM99ujrbEvyP7oVJKucj2b5trcTZ5Y2S0C9Ady\\\\\\\/j97\\\\\\\/UxmLeyuDQWWg8VISyfe79NtXKKLnpLBijSdE+uv0eoY35rDUGvlC8xq\\\\\\\/0PHu\\\\\\\/i0eI6IQxzxD3bkui3buTQ25cqsCIn10PI\\\\\\\/2EiTgD9Gz+Rpa1XvdByiIttEy8M3OKa\\\\\\\/MA2BegSr+hNEz2MxT\\\\\\\/1JkM64Iab6aVEEDoW67MXpxrH5g4hye5aV913xZJVgOxMrMfXqVXpn87dyRz9zSlZd7w9spIbnZGVOxZsgOI4fZ7cZNbiTPw41zqNqvJA9Ehp26IbbZ3fPqg617bDbbLQr0FEfG583GKIreRqT5rI+dBSMpu2\\\\\\\/YejO7wekhQHiAutXCDrHqXgdHOeOvDoMt3P5cL7FTujcV71\\\\\\\/1cDcGRiLMu8o5eRsJbOK5ULRO4ClqTJU3a8Zlg8iMIqNXuHIasez2YLguUr3lhFXbkSuzWsPlrz6+hi93K01gr3CeSaZaUhJiMEMAwVioKp6WACdco8tv\\\\\\\/4Xbp8wJjsQORKaWB9Movnf9nRyAG+9zo7keCxbQa\\\\\\\/b7t\\\\\\\/M6Pg2bcKra0tAixoiRm21OCViKFVFRBuzuKgZ4MhgBez3OnZ0TYnZtMzH0Pama6bSJ+wyhTqBIHK5zwnZjs+dwozqAYSp+Qu5uDaywU8xSZyV5h2hSp2wzpqPAugr+3zqjVD2jbAvHmI8mtp2W9dgeHlDITfAs\\\\\\\/5lEBi76hs99PpxgrLezgiI7MKGEkzYv\\\\\\\/DqXhLErYj0u14p+fwD6sG8H9be6w2Di+Te73taZmWqpEqwrUHeOWy8dtxItj8RbFIvvlDQkIAcQGsfySHdA1HzHfSG8jYL+nF9T1h0QD3BiM2Z8ZW2kPqFnKoV9H+PtIBnSAuBREJC7gGkIpokY9Khltn7531mlK43wg8e8UGiBd0DmS2rqSIsRfSBwcjzMFXwrRN4z42ofnGSAAx\\\\\\\/2XtnnuX1CQORkZszJCSmWRg0PAFg4IConeIEf8QKWvnv\\\\\\\/BIGj3RvsvyrY6JEap1us+hr4hRdPj0XbCioITDWZFUtWsxjb9v0bV4uKYZNHE8tdxFfHzPR6vFgg2g6ZWWGeaoQO7hGQ9qPFijGJ8p+dFewnEX8EpeUPPsnlt0nQNG5VgWZG9QNwkjJWnQLKa8M4RzFuYX\\\\\\\/hfYcuJslzoShL8UO+AnOiYhLzsGu2NDW3AiZMUawGd\\\\\\\/qWbLxiTWPmDGZUrUSXmvM+nYs7k4B5yI6wFhLHlcYfRQjFc\\\\\\\/aBjAemlP5ldA4WpIf\\\\\\\/d4tpGxOcWJkLxQ2LE\\\\\\\/7imXZt8\\\\\\\/+TnxSDg2KlG0Ik06Ki+1TOkIitKe9SKUFfw4F9OPLCKRvc37zVGXslv1ycOS6MKVrC6utosXZfwuUQDWNIJWdunp97rHaNgeFH8YAIkenWnbBiMJMwqsw0utA8EJ9IHqWMvVFVZLcxrVgGemXyUwGIH4YvVOwqPKf7QPFEL4yRZ7dN6B4J\\\\\\\/cgsDBcYPJbbyaaBQqC6u\\\\\\\/ePJHxjBO8gC\\\\\\\/e6STZhKQK1IuRGEJU6KfW2vk0ZKO2WciwSi2PNNxzA7eGSM+sEZZ5jQMJwQsHoSVqLA1JXI+0qBnaK\\\\\\\/6fW66eLcVD6ilmRoJ\\\\\\\/IQqu\\\\\\\/TeOigeJTHSrtOF0sAFLSYbwvWBCMTbjvSUtTVvjNJcLJEcTQsmtmpKJibjgor9P25Bcpz+6qeyGXRT7UB4NCN2xIX634go613TGET71FIRUpIWZ3WL3tX\\\\\\\/Q7sKyZ2JWAmd4sYlLA4dD+ZMYeEubFAKZjHc6QZ9QKcmYeDt9ni5KCJFrJCPdZCW3wYmMVPfB6Hray0RLHPN0xIN9AMwr2hBecLR3H+tkMmrfFl9LfAftqmbORTHG5hbpK3VxAw+c63dZNwh7uZJ\\\\\\\/BF7H4MrCdZLa7kPvQinz0I\\\\\\\/SZ7Bu9RInddZz1UYZc3B+Qm9IRKkd0SMFA4qbAl9C6kSNrjqmExZW2G\\\\\\\/5Gim0gZ\\\\\\\/IiFvTE+sgovftYazaOBo7P2\\\\\\\/gvx5+mOLHDsDtqqMhhx44R98gJPX+dmJr8NGpdh6Hkj8FeefrCvQ1cdSX6EXeGID5sIa4du0juEM1vHc\\\\\\\/v8CXn0w7Eb+LLwG+u30zBDY2HbdqyPRlW9UqlnYHX\\\\\\\/5uENP4R6c+ZfXGJxbW2JP3wotYi3uEuexyNhieeMppSCF8elEMNiiRxyRJKkErvCKsSUi39tkbrje0tcF65d4QzcRPu3cQWQP78XHPluDAbHvoQXdM8eNIktIcxMKYHeF+8XB68cLJUOS58sbbLPvRRIZ3QZVn5tIDti60p0QLd3Qi83QJErDGPGFv98hm5uFLQj41Nv8CniEdhH06d3bfHge7zdNPt3+3jUU4CQIIuYvIdfJmPmwyqMUlKQyZUzu7kH0b8KfY9eCvyg6ygOcGLlssqkEycPoS3EDW2Ys1ghaLDIBx9NQcXHfrit0QTSne+1tdEDHUYuIUpWf2xEsLlK8IMD3vbEeAGKb08AdzOM7Ao9gmqUMWSnBIEqpvnAKpPskNQ\\\\\\\/mxcCjZdg2h5zwUvbKoTIRvuZoJSSyMwUOP3FFDsYMK9v\\\\\\\/4C1WJ9AA3+kC8VqBU4PbQlj+LlvAPB+1IoLCpCig4AGhtwPK4GhqRmgLmQSdgYMgPRS9iufxy0YEsuk1cvpz7CyfboDj1QHGfywh9SLruu2L\\\\\\\/ovotD\\\\\\\/m6iyt93mTb9t49Sx2C0cN4avWbxBSq7R3Uark2ENB6rLbWtOUhhZEPtJHmesUOY1WBSV\\\\\\\/8Vck2zUaBxetIXHi2Z+J+Y5XvnjsTrgT7RldwOQrfosl5u68LtAdsUjL2OG\\\\\\\/FbeNDYiwEsaeWkLnaz5E5ic\\\\\\\/9J\\\\\\\/4HngAimbt9SeDvnII67ZuZBz+Vy4Xjds2p77nzxgDglJth7pOvYDwy12iIWpYIUtU\\\\\\\/rbGBRPDEDipwAgZacQ20AyaCzuOHtIa6hqDxeM5z26SKB2nmqCkx9dRkXt4gTtXxS5YADFXChNB0gJnDc2vGgtzKdNa6BNOzw9dDfM\\\\\\\/iT9\\\\\\\/SbV09Zfmj3hDoI0qZGL+OQvbV0wSFKrMQ+ULee2CY2wXozRqLOZ0BY5rGJ\\\\\\\/DcZtgN1K6IcOY5r\\\\\\\/bBy1Ktsd+R4YaVVJcDlOfyCMd0KSgV3L1eC\\\\\\\/89SuQ0qWpHUH8aAsOPLyU0wjeBVr19LsjBtf6AdVbnGrddLCQJbMat9Dy9EWsjraycgviNWbA0r9nT9Z6ONYn7o7O8OiIEZapOPmRUfN4edLVVTwdP6\\\\\\\/1tw0noc0YfLHLC+g6VPgL1UThco2VBxS7lUcwx9yWKZwKsyfj\\\\\\\/bbWsIy5ixEvx+nVGplYafWoltt\\\\\\\/Pl+m04TEb\\\\\\\/0+JYjfL5eZ8ptg3+IEt9+56n\\\\\\\/bLfhAp8uXuK5XTVN2JxVWPjgFZNnPYo\\\\\\\/fZxq+fBzPfv1NSo9Zc+RkecUtqZ1O9XhYDNzXwSCZz5nAcE2iYjErX4FbyKDPfibad1bKJJq8+w3+YGK93BSUdypbU93v+Q9PdjarBKgoilHu9RXnJ6eZnn3xw2HJlnqM+kJoNGS2tXc8ys1uhpXcAOIGv6MYnUwq50PBhQCmsURRjJKUNDEYstYX70ZfkWhtCM2Dypt2TLtr5ITU6xcJTa0GgN7d6VtdiKigRaOMKHA7iHHBU5+RmAoc3U2IXAnQ\\\\\\\/Uxw+pr4SPXmao+Ir34oUYES2PLGT42xUE0nnJeNVtBy1ZCihr2rf6xbWXhF4CM8wy2nQG9kBml5yM8+00t2T+UUYgwKxeVLCuKmSgDZmg\\\\\\\/Kxj6Pp84e1H6WxF8DuVZ1lcivDNYHqmWfVXjsseHn076SmLuHSPxy2m\\\\\\\/PrEFHqJITNs8D20oM+jhl9w1vyZaV8iVNjTLT\\\\\\\/yj29p0woNzp\\\\\\\/l5E7sqa8Wc6YCBOYEqxope8tdH7sci\\\\\\\/YckiOWkgXpfI40GgKl9NKZ0UDEzlsejVv465+mkkhBpv3g4fdX6Jl93k726m\\\\\\\/8JFxVPLnzOWHO1ilEeocJ3wpbcmd0B4k2lIu3ftilCsFAJebbovNKnLvGu2zF1JIR7PvRNiGgrNuASnIbC6fsPjHyZjjLDb24wxnyDDQAnmvEBtnsJyWxCcJlgCKO5MpLykRcRwXLhKpworWpg4qzPzaqghyIxnWKZjE8Gd0F13KMK7os0PV7KRYLgFXcBjFY+xRuZ56z2lfYpwh3DYvlpwieZX2PSJdt\\\\\\\/AHkqP2MwiSSfRAqRxaKVQVJ\\\\\\\/9dTns8DHFEho8Yq7si4I1+gGBwidWKMVBypBHfLLdntgrfKohemalaTK0xqBQhGMEBgHcawRMyoSbaaheqmYJsarUnVac7Rt2z8CeY8oxQFuH2U1J2iyd7FwYIE3RIIcFi72rcPPKmggIByIieyS6UXfRbHokad4hbzp2+tTUEkYHYe\\\\\\\/tXmDB63P7qgzYpvbBPj2+BglKZ+rsv21Ys92ZmjWRw5AvKB0v0uBfKYUTWX4sZcZgxq2Hycs0b8NBVMixwKl5nnZZHi6a5FKApqylIYEdHCoOQ8A1jXZNFXV5poQ2LzQJDaIH1MpYQfUdfps66NPFFdxxfQjzt0yFu2Q4wiTf5fOhsMNb+tkulc4a+nXBxrJF6CRdLbATpuu\\\\\\\/OjQhBac4Mx1NurkW1Rs0wH23S+qb2XVXCnUdTd8V7iX\\\\\\\/WJs\\\\\\\/8hL1QY9fz7pSBPUkvqBsLC7TbYDKyk\\\\\\\/AavqMI+MvXbJTVrEnhAx4N8uD5v3Vmwfl+wf6meUwmi9P3UJzsyOwVWldGlYzyfuaArPBzehHg6kb9KS5A\\\\\\\/fURUN9QB09gGmjoUXHWYeExEp2KiwsOUA4Sg3pRkejB4pz6mvavTFTg57Lz3H6rqiqO+2cRxT6GMvVFSyD8wOLIkUfrvbuiEGX81QkvxN0J7hc\\\\\\\/9CuPbQjNR1rJFDO+3Rapm4jyYdluPSzinK0oNdobLj\\\\\\\/gF5m2nw4hV0w05nKgPU68zeYkVwd9PTDD6yNLJQB+4OxwmAFWduA9oESJcxBv8ZjOkO7pHK8SUKKznxkTvCmSfHEihQnkCjRJtIA7xqmtHe5otBrWVvRxuIHbvvc1TCDRgyGN8xhSmEVHY2zLK5ZgyTYXBK4MLnnf2zhNZE4zaPNFzyc63yLYNvDhDTxVNm071p74RSuP9uuaAxs7mr5gUWRo\\\\\\\/7okdT2SML8M5wvFWbY4Ege26lSlRpA\\\\\\\/jMEnPV\\\\\\\/kqkG+o\\\\\\\/5+ei2uLQasy+ujN1g+iKExMNul5Y5cW5+op19YlQeAxYv1dcoAWyjSuBdAN8o0LbvSt80w2dTmu6d4MR0znL7s7dpqU+9DmHJxHRcabAM5Gwn4eqyB4YIXk69xtQYeLArIRFntr3WXB5pf7aQMYcfvc5WzMaU5q9K+74WYon6\\\\\\\/HEV68sGc5ihNvI35MlV2TkggR+OKUPJcVsc1n3WoSrj\\\\\\\/eOnybrkw\\\\\\\/71aXkjPvdxYs6gLAMdQubYRcYcRvA0gdDdLi0UZa2fsU90Vr33KGwploq3P5UQI6kp9n2Rbx9HLSaqwJuyfcV8NGqt35fbkEuyh6heaE5LFjkSMalrKUgq0bYBlo2WDfXX\\\\\\\/aaxQ8zb2v6yGr79BmLbgwHoAOAfHMWieCQrjx1b41Cj3hPehqCmAalho061IWrY\\\\\\\/IIIi+9YzhfYzrUmWmh+qRQBPS691jY3KM\\\\\\\/04EGPM6vzLE1BiFF7IAtPRvttxhhNaahoAHTSxkdwFayPwBIWoYTR9I4KFdrM7LNnKVCMLARd8tkIz7lj\\\\\\\/ReIfNKFsRMlNzImeMt70gaHsSmhIKqgYClYiH+YN15uPQYjy2WE7CpK17+Rk+JNiP0RxJPNO7DZW+uOVVIUPU6DmOTHrihmEvwk71CpsXIyh3DdxEADTRp3WQffnuf7zANZQlyuAB4+\\\\\\\/j2weDdzTgofR6FtvirSSIQi+fayBctUOM8Z3D4rkm3aatyyjx7DNH77nEVOjBuetURFrrjoMUg9FM\\\\\\\/wubvyA65\\\\\\\/L7+R5RuAWrk\\\\\\\/6rRxUmBySX49hLYlvtYKTfmJUfeqnfX48M8aGwl6URMG39YdCNJeb4xQLzx2P\\\\\\\/3hMWh3ACytX\\\\\\\/NN1vADq1txycjo\\\\\\\/pBhHGEeRHOLrHXaaUdTzytwFDYAKEUJ+9bBHt8Puxj\\\\\\\/b5aH2HVWl1Zsz65yZdp4IEwd3bwseMf7IETbzb4W7t7QIzcgp4XB8nF2+GNcmfmiZX5LoIkD8fKAHL+U5ZKtsJa6eaym1Yiu92QCXqXoN57kVyv39gsZ2Rrffl7YUWsWr1i0fTKtMCoe5JA6+BDAp8PM74dlZR4DfVdJl3putljDi3EDqBGsuyG7OEqi2gsReQDd8wp1QxkhA8rJip4tOtshHRfixzksSGwJJaXOceVOFos4U2nTIwPNeFkq6wIEsokdhF6wD3O+rssm2yrsCUH9\\\\\\\/3FP9iVk2XH+EzZhcoVdsLUjsxEozZiu67GsuOl9puBA8XY4fKLAfJUQyiv25u6DbL+bF8KRLFJKLp4TkfobSd7\\\\\\\/JmwMw8VYQ9356VwBavDLGyHUcQbzhSC3mlmickraEtJ0sJDzID94C898y3\\\\\\\/jTVm0sY5ju9QLWnACZ+OZv5G0gjvwoLAuJAHxrtUCSuE3duhTy7LNosJysKnurfkxm3au+fQeXN24IhGGZ+6hyARjowc796jpeFO4XWa+XfNtZ6AT7ooP55UaJUmYrTSa9Qu0dWddMDCSGZlsqZfsOzPF22ty7xpWMF4+BNjdcfIaWqiuV50YaUhBlstlro\\\\\\\/5fT75ssHUroXJDVAnA9hYSXyMviWqHIhMQ49F67ehSfHOORGm7KI3kImmJ\\\\\\\/an2vrPkoo5RHTTcNcNMYBC7uWyAq1\\\\\\\/q1Syb3uZb4lYAo8n3g1Q+AQBwur8K\\\\\\\/bwNcLy\\\\\\\/nuea3oCDIgAvbay8dt1d7YIMMzGdfaGUBI2PCTj7kWOhWGpb0AtOu3dS3QZi0R9TKdb34SA59lB0NOXfOLolfAVqDs5jmL86I1oLEkT5VWNxSPjneGWv+bpYTz1YrvJlj04oCx5UWru5LMo97Nf+B2kzlZisG1Bwo2tM7yxqWRHEF6moucFhJBgP2zMGbA1AuWi3z526dRV7tLC1MTBW38Id5MtF7op7EXSLJjhwsV31dwT5R75BT4zlTIv8FrUDvBlZu+UEBOMcoJHthUBQLkXXzPQ6ryDPtfE8V\\\\\\\/hguUh6zWodh7FUU+6TWCRBOb0uIDuVKHFooC\\\\\\\/pl2UJsFiQQFhjWXuZ+6nO7ni5RjqAEpAMVyG2oILtR9K\\\\\\\/ozA8fPqn7G9QtePPP1u8XqcBogGNDzLqkZdefHboDZzf36RHeY3QPUDbw+F3Pc+HVFvfhA7NZXsMjL9Oegjw6Ncgj7+ogIsv\\\\\\\/s03mJaQ5DHw3dTQsfqs1gzkh+aI++u6itfbTiU\\\\\\\/3kyXYYx3VTdsyeyFCNrDyowqsyLdgfJA8r7TbHWv27HJwwnd1T3r+TztHG\\\\\\\/toUWK60Rk5vqQzqnkVX2T9m1kmfhwriHIgrsXJ855u0i9bnJid1piHsUajDopeuFS1qwktU9laPD7kw8JIXYFiBftmqB\\\\\\\/jfYB6MDNtvnfScOwUinuta4Fzhw+8XLnP1siGyp3xIVeOBidm65Hy8eWcV+bGauRaVXCahD014R1iwiH1VwXmPbfwFH8Zyr+pzxTHLjhnZ84OvHiQnVGKftcOLI3aUOjSi+aWtKIOfEcAWcn7Tg7FldlYqKN2IiHuzAlzLqTbWeIjmbxTKjPEhSNQ0wnwqZIgBA9lgWQgmwLqPIHu9DLdPIxWk3Rhe97ganfo1GP4WVrrR9bdOt8QWZKR\\\\\\\/Fv+VB8NlXOEZn0vpG9p2v84UO1DhxLzAbmJ2vcj4ruYpGCnqGeQEtAg7QMq4cbO9v2aPbJiViGAgovfQG7ALSkPH8npx6fPqn1GU0TVGT1568bARLU5VqzpGOhOmeRvXuqpMTcJoF8jdL2dqH\\\\\\\/4ZYIKL3u6K0csWtiISugvqBHgeAYDXbERotRRzrkiW9sAuKxx6yHTL8MmFMVpPRTnIDp66vG41rZgp\\\\\\\/L97VbUcKtkGJNr8lJ+H47Dn+51lHOqJFuHbEUs94HqF2AcgMAvVWmYFKFM6rg\\\\\\\/n2n5yPgTL10I499WOvtwPdygc8O5o63zgukU8ktTsn2D4NIs7kVie3i\\\\\\\/1T\\\\\\\/qDGqWinmQGmE0yZrtDtZROFq3XrQmnamZ3uw3rq6z3Um\\\\\\\/Gw8RyXbNRtN9oqTcDbd5CQTqdCRzkN26GcQ0xzoHU\\\\\\\/isIvN37s5oZOIXXB6dEBK0v8QW4yH+5572yYKfKoKPlotxjXupHJQ8C6NN3l776JORqCV6ZFFQvD02v5EgSZKSGiZsVUJoa2WgFPQvM9UKPUJnLuEXFaIfcdCQ\\\\\\\/jYgXIYzVQrQt1kbK1ntOM4MWJBMSGwu9kZ3JE8i5GZev\\\\\\\/4NCqH9YvKwvCQsT79IkKkGXf+MRY4Ghez4odX8C0rScKd6\\\\\\\/sVE5u9Tx8ctGHHFu8y\\\\\\\/pIcUShA98UQjZ1kDg9hnP0Pl55n1kyofCd4XIx5byW5oh2PhSzbkYvYJD1CRYWpNbF6rmBWh50v3mlUO7kszGBkp7477yqZ98eYOp2LPmOPGp5mMpCZWO9ibca7Daiq+etT5m\\\\\\\/AlF1uZUZumXCaJ7xa+cOHEPq\\\\\\\/o9J+ROB9DX7L4XjNkfQJt5N62l8ytTeaj7he8cu81rsdYuvFaCY3wE6IVDBfST4CXn+oW5vAsf5Itgr7YyMEB0UYOdTcbrUxH18UL1CAh7uI6VI+Zjnt3SlM6URbfClQvMJsO7ZjqQCQ9S3jDPulX9u8GnGEwNbAywbj3Z1p1CGszhCefxOCBs06hqDF0gia1qu6y3e5I9zfNrK1+t9kbEv5WNPj9CJFxkyD4mHCXvQljIRF1zJEJUtknusmRNXpmmPnR506dcuDuWROTBVti5e2IaVSlgT5A1qwFcEL2XY06w2jeNvb\\\\\\\/gClnJ2psVneVnXqh4\\\\\\\/f9F2lbZXg6uYUK10Hzw449Ui37sK2ZycCyIt61dKAr8jWRKZI0pZKZtDbStw6vDL4SaZnFz4qw+eyc1papsHqicKx16nnfYqHcYxu5HCZSTo5x6QotETAJO4Oze0l2AurPnPDSQ\\\\\\\/\\\\\\\/qtENb26RjCDjuitXqH5lwbZuaU8UGkFbPT51FVe+toAyZxvDPlE7PdX88IVNEVloVkjoHs7RxFa+VaC6r0GW5cf1JCQiK+nqPGI5ypJwaadv\\\\\\\/73ynnE3VCG9d5V6BmYCJuyZG7VJFhDvRqUhB5gnUeTk7IHQJyqKV\\\\\\\/EvGDwQwraYYEplWtwFcgCQeGR3Owgat2iKCCJfCS4tBj+wkONC7qK7INgKhPrz1Ihz4fJvWipEES+CTyWXYYNs0fFqYCpzCtSRZ6lcSAfTL47ci5TXgMYqT\\\\\\\/qKXDcwQdekPFqngj\\\\\\\/Tnc1m8\\\\\\\/TIOItk7QKKeT2182jtgkJgOftcrHo1I26kR5UGAnPdO\\\\\\\/gZVJ5Lxl6\\\\\\\/gRjVevxCzniYs83tGp00L3IF6bzQnieUbpLzMwrS9ce4iWFMu6TCIyf0GOCt8B7x\\\\\\\/WP7gPAN57pThrejxhc\\\\\\\/7yz8bzONoE+yvcUg+Ozqv9fWgeBgsatnJLWUSbyxzBuEVhxiSZPFzzSoIlPYT1lPnvnC2j3z58BrQ2tPl7lEMwth3AVo1u8IPBAEc99bvPiETA3n4zGAWc+1KzjF2f7HEsOqG4Bk5gAlhDuWMrj6MJjIIWLhdMzdbqSiBzqpL0+BGgFDPfrvftReZUVZXMgjjWpj2PXfNEsohpcjaLCPWsy3ZTLSFuLsRr7oe3BMD+rsgzu2STU8qZMIbv8XDiQPQFyZC+rb3n7ywvm9Xdp8PbeIJnzHYBH7OyMpnhE0vxQHE6tbpzauMT7s72xSG9CYgsWLKQxnJ3J7FjtsPFgGPQHbvsF6Q5ukmPU6MvCgD3Te2f2hdYnWgo+nw3CJcXXtCbqXgZTN1NlG18NQXVY++dsY41m2wEw4qXCBXeKrCBwlUhAl4CCNUN6Mqm0vUC1SPSmQOyZMjFQbA7SrzmcnUxenmHv8p4w+pH9\\\\\\\/fiJRBT9DaBcNHgDvIhRsaK1iP28f6tOFVXAwNLNZU\\\\\\\/WOC7Y2UCp86b\\\\\\\/XEVTImgevR+oFO3ELG+g+GVX7uWWLcJszmCiXn3BroWH1i2Uh\\\\\\\/oza9atybJHfBGYn2\\\\\\\/HLZeXqwK98kiCEaSmazAddTWXAHEiEqX9vK+E4e8eoZgWxh21uoA6vyx71kI5GBW\\\\\\\/3tHotRqrXCwHZuf7uFiWDA7mx3Tn0oKfSFalkEmfSJyIO9lfNGcdHEogseMmiVvzDl\\\\\\\/LB6j43VcgBfuEnaZIW61MwmUNdHiFVTe2t4VGNMJx\\\\\\\/ErF8+xct+nNZuUw9e7YHa\\\\\\\/MshnDla4VX6igF9R7jlwZ5qFOmCIjgHg3PWgo80kpnbdNL\\\\\\\/5Czi5vtOF974HpqEh\\\\\\\/TCdff+GdldiPkbLPwk5RpJzPPpaJznEg0mXab8W9oN+M5JMk4EA0E5KrsvZirFxEafxQ\\\\\\\/g3S+X\\\\\\\/bLUpn9Zfy5hcOvmM9\\\\\\\/wjSrMPK8hh\\\\\\\/xb4KtlGsjjrmoUmAnIYwbYywMnQ7pfJKIZdFgXXVAfCDZCYuQeFLe7NUNfSh1YQX7vhXEOb2WIWONJFFqW\\\\\\\/Cvjvt7jsXpx6ZxzbxTpTVjNdHyx8hvOkAv1X2MmuIH6aRxvQ2PWF2v0CkDrb5wVRaQi52CVJUrykF6uGnzyRs8Z1aVvoxRX4Zb4pcaWAqoJb6DUwZ0gKCPUCeff22P3uIkhQMfHgUM6iNvzbOyRDcLJ56wQNzxiwzDS9f5CVGAQ+sM0vCmRQFXrdN8Tgi9wXzg7joHUmA4PIuktbIbUeFwPr6I85f3TaiahwzbLcmX6txyZCOQI5h\\\\\\\/EsDW9ZSyskc1Ojkr8Ob62b4Iz5PliY2KBQjH0d01Q+FldtsFZ6Zd9+t1KkKWPYdxCJRdXaiNuJE0V+XJkH8465fTh8ZWXnVeJuI60iFI\\\\\\\/RxvvvGEpxPvNr7XfBW8EcqeEAg3\\\\\\\/T9V10iWDsKLOC6A4vXGqb9mAZ18EXiwAXL3Zgcr0jpM2Gxw52V9f7Il+2YY1kubBE5hMXi4ryNtL41x4vG3+DIs20BZY7XDIOPuthsGeYE5sKcbQ\\\\\\\/ioUpgDYxY0r1Z0dQrtdmS1vnvuNz+iYXw87E1L9Ww9EMMwjsIHh9M6iK7Po+JnPi8Za\\\\\\\/ac1GNz32v+0xSet0p30kWMQpLRgHH+QSKYqDN9kJ5P5uuHsQkea42hgtDsS41LtPqWezNI\\\\\\\/XeMmtQ8t7kMxK3Pwj5oKXLh4ykNLAbY0Z789Kew93iXF+RxX4gI\\\\\\\/lNbrzuaG7K7NFB4Utf1T+R72\\\\\\\/Gc4kvqJW0RJx6vvWA+YhQPHWZPBBjbYIisA2VjliQ+rItWk1IUC8jc7\\\\\\\/vAmJstcsCrGi5DGJW5X69yLJbP30Zn2iZvateyS4afUUdO0\\\\\\\/7PzPATD1bTgGhwI7PiB9sh16G9VzHrjttVjRpjP20XUms+FBTSkIokwnOxScZBNnntrSgaN4VzyZyJX2\\\\\\\/PuKPJrxV9Hos77L\\\\\\\/swp8OvPQPotwbmDMnNuuapziLfxKQEC80akmti2Hhus0T85UmK8qoiepcdbqlfcpXOYspQ+0ODY42lti2hC4CzymQJcdgwsrMy1Dl1kV5G0Uh1o8EQMcEFK4rKrR0T3w3ns3PW9jmy+GqUw0FL2ln1OmbkefsnRfww2\\\\\\\/Y8\\\\\\\/MytqurfHIkvtjI4ZN6cx8WKUjCOszbs21tRhT\\\\\\\/ycL6rinkO82n9RFPyPL5vSs0ZQmyGvvA5klO0cx+Evnblxpk9dsK8okmCnWg27g5t+O3qWDsuSDzdEYupmz\\\\\\\/QgSIAHXfMX4Ys7iWbhmYH5+wvGSYzG2LAk6bbiekMsIgIWJFDkM8ctKYrd0p802qpVVxAGq5f+Sl85DsUHWGdPcAZnz9Y0ujS9vvemn3M5Dn4umtWw9Sgq\\\\\\\/KtByQq61ovj90K9VNPmZ7CY7DPhjJr6CSVUIZVBKD2LB3GeIDtpL8YWTa6DRTsnilj5wFyWcsHyh\\\\\\\/lKqkrkIgRLehyiIk6cScpD5EN7o\\\\\\\/qmcXxNyaO1ur0OQuRaN51HJmprJsGUbcGcZFvtHdZ1EO9vUT24UF4Ckqt8LIzGCJvgiwg2dauXjZlD7cTChms9vNA5U4B\\\\\\\/L1fYJ0Qu\\\\\\\/H6l\\\\\\\/PbsNvF5JYGP0cUOPHNL0harjldACWaLqHV6aX+wNOXp3yhNCB9s1EOTAIlXbjEEhgZOmTm4JV40+sDrKH92ljsGwo4aZpjizmRYc7VJto2QpGS8nH0tFjSLlHZVB\\\\\\\/MGfQmiFDVswduQFkoFpRM763hMyjnDjrgxqbcIZOrItyZ\\\\\\\/LSE9VKjyFeIF6ArkE+VLhZxYiXL8Wl4ETAti+jsqnw7Fa0CagNTFg647KZge3vMvkI0ZzdmDj2lfzpFXQRdQZfQrjG59C2E06+xd\\\\\\\/7pDbL8cLlkxovNgehEpZwgOlmyAg2BYldCrs68j36x9sFg1zjHhKRayyCEZKPolzwg1X8baNd8gyfapIOKv+3y0dvdtJDcx4IffB74h4lXN+U4zvE+qUvlUUpO1UNRbxRUxOgslwvJf1m3vXcxES\\\\\\\/5slvpjPGdlv0eTTauhi9OLCjTR++yHpxKO17qhaun6mPvaqIOrwd2jBNxmHTJbGoC53EloIUgS2vjy9c58fEu5DysgFveIcvVLd3A4FpqGlYTGyYBnmouEww5qtsi3zmYY3\\\\\\\/1wHvFJzvpTv2uyJ\\\\\\\/sc2S8Ht3iTbjcpljF4chPkC4NgYspM51GUt7\\\\\\\/GGJQimhaWVs\\\\\\\/y2ICPxIqW7fn8R2K8UUF9kDFPIVwtKqpf4vOfLabsypvl17zWNb0UTDRoUM\\\\\\\/uxdfJqkIKRpfJimVSmDJcHncPGMNDqtOKCGHfd\\\\\\\/EgflvHwlGFvWM9xRf1ulEAcPNJIFOylfxnzwMHThMtmHOqL6leMPLrCAeIqlGsGViXrsaCNNWoqEpC98M2\\\\\\\/uSHX95pzmXrbFCtLv4wm7Ss4DOfYZCH3rVhj+\\\\\\\/mJLpNDj8uaNK0JJKshdmC0Io5dZmPzzHK+R6lAO\\\\\\\/tuqq97KhRUVguOLc4+nMX2kSlOSqZtcK9CX1xIZpYzq6l\\\\\\\/8s5JMaC+5uQcO+b6djf2dxGYOb9yrMYy4cYqM+bIYtnAPARjVR09JvZt0\\\\\\\/fH7KUkBaKceyLZcmhwrLY6E6+k79A\\\\\\\/TYiAGzjQGrc9FRtMBvJ9NAFs0YwaVuwiw1jOC+OpeHNB0eUh7GK\\\\\\\/ORuhm2ovW9vvTQmslmfozCg3GKIT0\\\\\\\/pFq0LoucjweFMEFZe5Xo1MsZ6EedA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022tO+A1ltzBkD3gPFm\\\\\\\/jCFXA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002253d5c417776349ed6ae06db71f240604dcf8be1c4a0d836a0a0bc93f8e203514\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022\\\\\\\/CIzWVRQcwTc\\\\\\\/BcDBxPrrDfktm5BXISkdaom34TAgNUv5TiaqLF9VZt98HBhJYtWXqeXQZZV\\\\\\\/Ts6wxQVPJe6R6bWXs06jLi2mFukfBP2cJFxcgBYfAbSCIPz049CdKgAZU7w\\\\\\\/jTZRu5kUAu+wDXWQc9U2QYa\\\\\\\/gOMGSNoUHI0TnhFsLBlWpANNbYLjpObAU\\\\\\\/j7r060qwzdE0SXdAdbAOoYqB92ZtadHtZAlXFX0Bn6lGiEdJlPrvhU1ieAeOCM1jvP5QwOmasDjXnLVa5qnjOwk8X6Kk9\\\\\\\/rI1Nj5zGa9\\\\\\\/dVIn2RfdJ6kNoOwLPegf240pEfk\\\\\\\/SQckFv+dsnuHT7kpxwvfXkGm0MbAMm7sO5glFIyG6Ru8iqGrCJW4pqORtvxYvqKVQrWOTNLSnydMAVigikU\\\\\\\/Chqo6aMFnxCkcmjDWALSC+4wn5\\\\\\\/mcGACvnmWGUuW+LNuhax8WyRu4F\\\\\\\/+38zhLMLviKMjTqW30zVkf8sK7Rc1Sd9Ns6TH2hYxbM\\\\\\\/m0kIZJ1POT5Hf2syJ+os67tqspkcSKmamB86G6DmT1dDjY73SA0mcWCxlTmeFocJsMNQwpppq4uKJYtbg6aHsuX1bHZf0uZNreazdKpTafUUzxiuWWwC7FDrNezbxj+XHCcPN6tyGrE6zybsZe73zpz52VHOYn4JWrRnUvfY5EwGU2GHHt9ONsDxSKAkyLbGbEobaRjVMLTioG1ZiHlPRh7K\\\\\\\/wEYPV1cdLNdwc1Sdr8ThI\\\\\\\/l9AD26fQ0C4y8rUESund9eQX7jR0fOvlAV3VmAMdNKiPcjVyUZlKTpAqzh1zOfoZPw4G225ii5EWQ4AIVnkbVrZWr+0typjer9Z4UAtcnPQ8CKAZ779LlCBUrrqLiFazKNST9EU5AJZ5XbGtDhRccJp4FCqcxjYkzHvsh6KPEBs47gwkPr\\\\\\\/1CXSmDAKDNAc+pAggeDQBARU6TCRn5SPR3oY+aIUvAf4hJiBhiUqHALG\\\\\\\/ogutXi2PXgf1m\\\\\\\/gQci2+nz5PGiNQqLbEEzIOS4FI43iHC8bMqOAwb7gJIuyRXx1cnAW8bNLyOzJJSOcsgRmkuavkII9YUnCL5M+ciyn0iPrHkjrS7f4TrGeYJgsXZqmY8ZGsoYWmJK\\\\\\\/bvU2B0ftAB8Df66osLBV3RrPUa7Fd5ntsx\\\\\\\/SbHPhgjqMsqp2+E9Yy6hTIWpgVYfOYTeHSsPM+0X0P7dd0pg+WnfJxgiB\\\\\\\/o7rDiWevTQQhv30b0M16IFiKA6HOsqvf1Ia9jAlpDe1vS8np2BAwZgvLgdypXJgaIQuRhYXjjG7kQiFCpHmHjyPI8gUP9zNd1fTF5nRL3UQ+QJ3aX2zVJ0tU1D2j+MTZs0uu04Xsm+LLnmPruBZYikdZL4E3Ls3Bo7tQ4weMRJYNBVMaMmzJ7T63fDzKVN4rA8bSUNuJS7ViA0E4EoragKU44li7kofElekGKUit3n1hEfU7\\\\\\\/JNDljknVgCkPhjgiuAV5tE38N8U+Lo7C0ZDRlsgPa8pBli0l0sImOmqMTHpl0GZo2V5e7BkXsS4enjt9Mc8+GQ651WakLm8lkB4INDcVSjGB857oMxjvUglSjF3D33BBsyuYlSRO38L+iJfv0h13XK+jzcVzJeyaRSqWF2xUvphJrIyM3b+\\\\\\\/YhrWEvqVv3WyA7cOM9Nj+PrIblqCwFRa620uzoY1tvKYkv9W33+9wWMmTIRz7qgDXV55epuuCoMLApXE065u0lxpzkSsx\\\\\\\/gd9Z+rnp+yDcVIU05HKIb80GI\\\\\\\/S\\\\\\\/hCqMe7R3DhgSqEfTA5kWoiFvIdcLHeHdX8auewZxsgoyvlw4g7ZdCsXB3Y15vLKPwyenzvrcFTJ85gKEKG4bxaEBQ\\\\\\\/zfgjrZDwbUw+Lga75+1+5jRgzzWDEpgpc3YXXc+rPBaIVeG+FwizbmhdsxFpWEYUQGZ8fD651vgtbL5V7cUAw2rA6UitcW0U3N5xf1X9q+qcAW\\\\\\\/Jx2nuJu\\\\\\\/4P\\\\\\\/hahpDXPp\\\\\\\/Itlx+RxD6miZoZm7BiZ4dX6BfdVQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022Y8cJO+KYeFS2JRJQ4sYlsw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022dc5b94db9982017895f28e1690e3d773a5a0e5b79d9413f3bd2f583c9d2d8474\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-486373 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/9/3091" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-UUqrEh1qUj_FLDT1fu0zLVShK8GlH1smb9vEKGzJJZ4" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/9/3091" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-dDJ6M1cJ7OmpoadoE5VAAInJaVNjHDduWbEAlNZQE84" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/9/3083" title="Regulation of Small Mitochondrial DNA Replicative Advantage by Ribonucleotide Reductase in Saccharomyces cerevisiae" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/9/3103" title="Application of Response Surface Methods To Determine Conditions for Optimal Genomic Prediction" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="486373" id="node486373" data-pisa="ggg;7/9/3091" data-pisa-master="ggg;g3.117.043521" data-apath="/ggg/7/9/3091.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">3′-Terminated Overhangs Regulate DNA Double-Strand Break Processing in <em>Escherichia coli</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Edyta Đermić</span>, <span class="highwire-citation-author" data-delta="1">Davor Zahradka</span>, <span class="highwire-citation-author" data-delta="2">Dušica Vujaklija</span>, <span class="highwire-citation-author" data-delta="3">Siniša Ivanković</span> and <span class="highwire-citation-author" data-delta="4">Damir Đermić</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3091-3102;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.043521 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node486373" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Edyta Đermić </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Pathology, Faculty of Agriculture, University of Zagreb, 10000, Croatia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Edyta%2B%C4%90ermi%C4%87%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=%C4%90ermi%C4%87%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEdyta%2B%25C4%2590ermi%25C4%2587%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Davor Zahradka </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Division of Molecular Biology, Ruđer Bošković Institute, 10000 Zagreb, Croatia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Davor%2BZahradka%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zahradka%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADavor%2BZahradka%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Dušica Vujaklija </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Division of Molecular Biology, Ruđer Bošković Institute, 10000 Zagreb, Croatia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Du%C5%A1ica%2BVujaklija%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Vujaklija%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADu%25C5%25A1ica%2BVujaklija%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Siniša Ivanković </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Division of Molecular Medicine, Ruđer Bošković Institute, 10000 Zagreb, Croatia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sini%C5%A1a%2BIvankovi%C4%87%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ivankovi%C4%87%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASini%25C5%25A1a%2BIvankovi%25C4%2587%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Damir Đermić </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Division of Molecular Biology, Ruđer Bošković Institute, 10000 Zagreb, Croatia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Damir%2B%C4%90ermi%C4%87%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=%C4%90ermi%C4%87%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADamir%2B%25C4%2590ermi%25C4%2587%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:dermic@irb.hr" class="" data-icon-position="" data-hide-link-title="0">dermic@irb.hr</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/7/9/3091" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:486373" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:486373/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/9/3091.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:486373" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:486373/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/9/3091.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:486373" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:486373/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/7/9/3091.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:486373" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:486373/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">Double-strand breaks (DSBs) are lethal DNA lesions, which are repaired by homologous recombination in <em>Escherichia coli</em>. To study DSB processing <em>in vivo</em>, we induced DSBs into the <em>E. coli</em> chromosome by γ-irradiation and measured chromosomal degradation. We show that the DNA degradation is regulated by RecA protein concentration and its rate of association with single-stranded DNA (ssDNA). RecA decreased DNA degradation in wild-type, <em>recB</em>, and <em>recD</em> strains, indicating that it is a general phenomenon in <em>E. coli</em>. On the other hand, DNA degradation was greatly reduced and unaffected by RecA in the <em>recB1080</em> mutant (which produces long overhangs) and in a strain devoid of four exonucleases that degrade a 3′ tail (ssExos). 3′–5′ ssExos deficiency is epistatic to RecA deficiency concerning DNA degradation, suggesting that bound RecA is shielding the 3′ tail from degradation by 3′–5′ ssExos. Since 3′ tail preservation is common to all these situations, we infer that RecA polymerization constitutes a subset of mechanisms for preserving the integrity of 3′ tails emanating from DSBs, along with 3′ tail’s massive length, or prevention of their degradation by inactivation of 3′–5′ ssExos. Thus, we conclude that 3′ overhangs are crucial in controlling the extent of DSB processing in <em>E. coli</em>. This study suggests a regulatory mechanism for DSB processing in <em>E. coli</em>, wherein 3′ tails impose a negative feedback loop on DSB processing reactions, specifically on helicase reloading onto dsDNA ends.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/dna-degradation" class="hw-term hw-article-keyword hw-article-keyword-dna-degradation" rel="nofollow">DNA degradation</a></li><li class="kwd"><a href="/keyword/exonuclease-activity" class="hw-term hw-article-keyword hw-article-keyword-exonuclease-activity" rel="nofollow">exonuclease activity</a></li><li class="kwd"><a href="/keyword/reca-protein" class="hw-term hw-article-keyword hw-article-keyword-reca-protein" rel="nofollow">RecA protein</a></li><li class="kwd"><a href="/keyword/single-strand-specific-exonucleases-0" class="hw-term hw-article-keyword hw-article-keyword-single-strand-specific-exonucleases" rel="nofollow">single-strand-specific exonucleases</a></li><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/recb1080-mutant" class="hw-term hw-article-keyword hw-article-keyword-recb1080-mutant" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">recB1080</em> mutant</a></li></ul><p id="p-4">A DSB is an adverse DNA lesion that has to be repaired in order for a cell to survive. DSBs are repaired in all living organisms by either mutagenic nonhomologous end joining or by much more universally distributed and precise homologous recombination (HR). During HR, a single 3′-terminated strand is produced from each of two double-strand DNA (dsDNA) ends of a DSB by a process called DNA end resection, wherein a combination of helicase and nuclease activities result in degradation of complementary 5′-terminated strands (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Symington 2014</a>). The 3′-end overhangs emanating from a DSB are bound by a recombinase protein, thus creating the central recombination intermediate, the nucleoprotein filament. A recombinase nucleoprotein filament searches for an intact homologous sequence and invades it, hence restoring continuity of genomic information.</p><p id="p-5">Since evolutionarily conserved recombinase proteins [RecA, RadA, and Rad51 (Dmc1) from bacteria, archaea, and eukaryotes, respectively] have a lower affinity of binding to ssDNA than their cognate ssDNA-binding proteins SSB/RPA, a recombination-mediator class of proteins (RecBCD and RecFOR proteins in bacteria and BRCA2, PALB2, and Rad52 in eukaryotes) facilitates recombinase polymerization on ssDNA (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Zelensky <em>et al.</em> 2014</a>).</p><p id="p-6">In addition to its role in HR, the RecA nucleoprotein filament in <em>Escherichia coli</em> serves as a coprotease to promote autocatalytic cleavage of the LexA repressor, leading to induction of an SOS response (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Little 1991</a>). RecA also activates a mutagenic DNA polymerase V during SOS induction (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Shinagawa <em>et al.</em> 1988</a>).</p><p id="p-7">In bacteria, both helicase and nuclease activities for DNA end resection are provided by the functionally related RecBCD, AddAB, and AdnAB enzymes (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Wigley 2013</a>). In <em>E. coli</em>, the RecBCD enzyme binds to a flush dsDNA end and unwinds a duplex molecule with its fast and processive helicase activity (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Dillingham and Kowalczykowski 2008</a>). Both of the unwound strands are degraded by a single nuclease center of the enzyme (residing in its RecB subunit) (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Yu <em>et al.</em> 1998</a>) until the enzyme encounters a regulatory octanucleotide sequence designated χ. Interaction with χ changes RecBCD’s behavior so that it ceases degradation of the 3′-terminated strand, while continuing DNA unwinding and degradation of the 5′-terminated strand (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Anderson and Kowalczykowski 1997a</a>). Also, the χ-modified RecBCD starts facilitating RecA polymerization onto the post-χ 3′ strand, hence producing a RecA-nucleoprotein filament (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Anderson and Kowalczykowski 1997b</a>). In this way, χ switches RecBCD enzyme degradation activity into a repair activity.</p><p id="p-8">DSB repair in <em>E. coli</em> is active even in the absence of RecBCD due to RecQ helicase unwinding of duplex DNA, RecJ exonuclease trimming of ssDNA tails (ssExo) from the 5′-end, and RecFOR proteins mediating RecA polymerization onto the unwound 3′ overhangs. This pathway is operative only when ssExos that degrade 3′-terminated overhangs [<em>e.g.</em>, Exonuclease I (ExoI) and SbcCD] are inactive, which enables stabilization of the recombinogenic substrate (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Persky and Lovett 2008</a>).</p><p id="p-9">There are some mutants that show only partial function of RecBCD, yet are DSB repair proficient as well. When RecBCD lacks its RecD subunit, the resulting RecBC enzyme shows reduced helicase rate and processivity compared to RecBCD, and is completely devoid of nuclease activity and χ interaction (<a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Dillingham and Kowalczykowski 2008</a>). In the <em>recD</em> mutant, RecBC enzyme unwinds duplex DNA and constitutively loads RecA protein onto the unwound 3′ tail (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Churchill <em>et al.</em> 1999</a>), while its 5′ complement is trimmed by RecJ and Exonuclease VII (ExoVII) ssExos (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Đermić 2006</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Đermić <em>et al.</em> 2006a</a>).</p><p id="p-10">The <em>recB1080</em> mutation renders the RecBCD enzyme deficient in nuclease and RecA loading activity, whereas the enzyme’s binding to DNA as well as rate and processivity of its helicase activity is unaffected (<a id="xref-ref-55-2" class="xref-bibr" href="#ref-55">Yu <em>et al.</em> 1998</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Anderson <em>et al.</em> 1999</a>). <em>In vitro</em>, the RecB<sup>1080</sup>CD enzyme unwinds the linear DNA duplex, producing full length, RecA-free ss tails in the presence of SSB (<a id="xref-ref-55-3" class="xref-bibr" href="#ref-55">Yu <em>et al.</em> 1998</a>; <a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Anderson <em>et al.</em> 1999</a>). In the <em>recB1080</em> mutant, the 5′-ended tail is clipped by RecJ and ExoVII ssExos, while its 3′ complement is covered with RecA protein with the help of RecFOR proteins (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Jockovich and Myers 2001</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Ivančić-Baće <em>et al.</em> 2003</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Ivanković <em>et al.</em> 2017</a>). The <em>recB1080</em> mutant is recombination proficient; however, the efficiency of HR depends on trimming of its excessively long 3′ tails, and is lower than in wild-type bacteria (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Ivanković and Đermić 2012</a>; <a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Ivanković <em>et al.</em> 2017</a>). HR is not regulated by χ in the <em>recB1080</em> mutant (<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Jockovich and Myers 2001</a>).</p><p id="p-11">An <em>E. coli</em> mutant lacking RecA protein is HR-deficient and is therefore unable to repair DSBs, which is reflected in its extreme sensitivity to various genotoxic agents and reduced viability (∼60% of the wild-type) (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Kuzminov 1999</a>). This relatively substantial decrease in viability is due to the exonuclease (ExoV) activity of the RecBCD enzyme (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Miranda and Kuzminov 2003</a>), which is actually unregulated, or “reckless”, in a <em>recA</em> mutant (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Willets and Clark 1969</a>). In this mutant, RecBCD degrades DNA duplexes with free dsDNA ends (either damaged or linear molecules introduced exogenously) so heavily that a fraction of the <em>recA</em> mutant population is devoid of a chromosome due to its complete degradation (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Capaldo and Barbour 1975</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Skarstad and Boye 1993</a>).</p><p id="p-12">In this study, we have characterized processing of DSBs introduced synchronously into a radioactively-labeled <em>E. coli</em> chromosome by γ-irradiation. Ionizing radiation, such as γ-rays, induce DSBs in DNA by either directly transferring energy to it or indirectly, by creating reactive oxygen species in the cell’s cytoplasm that then damage DNA; during repair of these clustered lesions, DSBs are produced (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Bresler <em>et al.</em> 1979</a>; <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Wallace 1998</a>). DSB processing was assessed by measuring degradation of the fragmented chromosome. We show that DNA degradation in γ-irradiated <em>E. coli</em> is inhibited by RecA protein concentration and its ssDNA binding affinity. In fact, we show that binding of RecA to ssDNA is sufficient to protect the DNA duplex from degradation. However, DNA degradation as well as its RecA dependence was diminished in cells with preserved 3′ overhangs. Our results suggest that 3′ overhangs exert their influence on DSB processing by inhibiting helicase reloading onto dsDNA ends.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Bacterial strains, media, growth conditions, phage plating, and microscopy</h3><p id="p-13">We used AB1157, a standard HR-, DNA repair-, and DNA degradation-proficient strain (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bachmann 1972</a>). Its derivatives were constructed by P1 transduction, as described earlier (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Miller 1992</a>), and are listed in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>. Bacteria were grown at 37° in LB medium and on LB agar plates (<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Miller 1992</a>), supplemented with antibiotics when required.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/486446/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/486446/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Bacterial strains used in this study</span><div class="sb-div caption-clear"></div></div></div><p id="p-15">Exponentially growing bacteria (OD<sub>600</sub>∼0.3) were mixed with serially diluted phage T4 <em>2</em> stock and incubated for 15 min at 37°. Soft LB agar was added to each mixture, the mixtures were poured onto LB plates, and incubated for 24 hr at 37°. Determinations were repeated three times.</p><p id="p-16">Bacterial cells (grown identically to those for DNA degradation experiments, except for lacking [<sup>3</sup>H]thymidine) and their chromosomes were visualized in three independent experiments by combined phase-contrast and fluorescent microscopy, as described earlier (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Zahradka <em>et al.</em> 2009</a>).</p></div><div id="sec-3" class="subsection"><h3>γ-irradiation</h3><p id="p-17">Bacteria were exposed to various doses of γ-rays from a <sup>60</sup>Co source, with a dose rate of ∼2.2 Gy s<sup>−1</sup>. For a bacterial survival assay, bacterial cultures were grown to midexponential phase (OD<sub>600</sub>∼0.3), serially diluted in 67 mM phosphate buffer (pH 7.0), and aliquots spread onto LB plates. The plates were immediately irradiated at room temperature and then incubated at 37° for 24–48 hr.</p></div><div id="sec-4" class="subsection"><h3>Chromosomal DNA degradation</h3><p id="p-18">We used the procedure described earlier (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Đermić <em>et al.</em> 2005</a>, <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">2006b</a>). The cells were grown overnight in LB medium supplemented with 0.07 MBq ml<sup>−1</sup> [<sup>3</sup>H]thymidine (specific activity 962 GBq mmol<sup>−1</sup>; Amersham, UK) and 100 µg ml<sup>−1</sup> deoxyadenosine. Unincorporated [<sup>3</sup>H]thymidine was washed out with phosphate buffer; the cultures were suspended in a double volume of fresh LB medium and divided into two counterparts. One served as an unirradiated control, while the other was irradiated with 400 Gy at 0°. After irradiation, the cultures were incubated at 37° and at intervals duplicate samples were spread onto Whatman filters pretreated with 0.3 M NaOH. The filters were allowed to dry at room temperature, and then were suspended for 30 min in 10% trichloroacetic acid and twice in 5% trichloroacetic acid at 4°. Trichloroacetic acid precipitates high molecular weight DNA, while the low molecular weight DNA is washed away. The filters were then washed in 1:1 solution of ether and ethanol at 4° for 30 min, and then in ether at room temperature. Acid-precipitable radioactivity of the filters represents the amount of the intact chromosomal DNA, and was measured by scintillation counting (Liquid Scintillation Analyzer, Tri-Carb 2810 TR, Perkin Elmer). On average, 1500–3000 cpm was measured in unirradiated samples.</p><p id="p-19">The frequency of genomic DSBs inflicted by γ-irradiation ranges from 0.004 to 0.01 DSBs/Gy/Mbp in various organisms (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Daly 2009</a>). The dose of 400 Gy is therefore expected to produce at least eight DSBs per <em>E. coli</em> chromosome. This is in accord with a measured rate of DSB induction in the <em>E. coli</em> chromosome of 2.7 DSBs per 100 Gy (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Ulmer <em>et al.</em> 1979</a>) (which would amount to ∼10 DSBs at 400 Gy), and also with the prediction of ∼9 DSBs per cell at 400 Gy (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Shee <em>et al.</em> 2013</a>), based on direct detection of DSBs <em>in vivo</em> using a fluorescent DSB binding protein.</p></div><div id="sec-5" class="subsection"><h3>Preparation of cell-free extracts and western analysis</h3><p id="p-20">Bacterial cultures (grown identically as those for DNA degradation experiments, except for lacking [<sup>3</sup>H]thymidine) were split into two parts, one of which was irradiated with 400 Gy at 0°. After irradiation, both cultures were incubated at 37° for an additional 35 min. Cell-free extracts were prepared as previously described (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Vujaklija and Maček 2012</a>), with some modifications. Briefly, the cells were harvested from 20 ml of culture by centrifugation at 5000 × <em>g</em> and washed with 25 mM Tris-HCl pH 8.3, 1 mM NaCl, 1 mM EDTA, 1 mM DTT, and 0.5 mM PMSF. The cells were suspended in the same buffer and disrupted by sonication. Cell debris was removed by centrifugation at 12000 × <em>g</em> and the supernatant was used as a cell-free extract. Next, 15 µg of total protein were loaded into each lane and resolved by SDS-PAGE under reducing conditions in 10% gels. After electrophoresis, the separated proteins were transferred to an Amersham Hybond-P PVDF membrane (GE Healthcare). The membrane was stained with Amido Black (Sigma Aldrich, St. Louis) for protein visualization. This was a loading control and allowed determination of protein transfer efficiency across the blot. Western analysis performed as described (<a id="xref-ref-50-2" class="xref-bibr" href="#ref-50">Vujaklija and Maček 2012</a>) was used to estimate the relative concentrations of RecA protein in each sample. RecA was detected on western blots with the polyclonal anti-RecA antibody ab63797 (Abcam) diluted 1:6000. Antibody binding was visualized with peroxidase-coupled goat anti-rabbit antibody diluted 1:30,000 and Amersham ECLTM Western Blotting Reagent Pack (GE Healthcare). The specificity of the antibody was examined using an <em>E. coli</em> strain lacking the <em>recA</em> gene. Two biological replicates of each strain were analyzed by immunoblotting. ImageJ, a Java-based image processing program (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Schneider <em>et al.</em> 2012</a>), was applied to analyze western blot signal intensity.</p></div><div id="sec-6" class="subsection"><h3>Data availability</h3><p id="p-21">Strains are available upon request. All the data necessary for confirming the conclusions presented in this paper are represented in the paper.</p></div></div><div class="section results" id="sec-7"><h2 class="">Results</h2><p id="p-22">Synchronous DSBs were induced into <em>E. coli</em> DNA by γ-irradiation and their processing was monitored by measuring degradation of the fragmented chromosome, which was labeled with [<sup>3</sup>H]thymidine.</p><div id="sec-8" class="subsection"><h3>RecA inhibits DNA degradation in γ-irradiated E. coli</h3><p id="p-23"><a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a> shows a time course of chromosomal DNA degradation in <em>E. coli</em> irradiated with 400 Gy of γ-rays. Irradiated wild-type strain AB1157 degraded ∼30% of its DNA during ∼60 min, after which degradation ceased and the remaining DNA was stable for another ∼60 min of postirradiation incubation at 37° (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1A</a>). This result is in accord with previous studies (<a id="xref-ref-15-3" class="xref-bibr" href="#ref-15">Đermić <em>et al.</em> 2005</a>, <a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">2006b</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/9/3091/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="(A) DNA degradation in &#x3B3;-irradiated E. coli depends on RecA protein concentration and its rate of association with ssDNA. Bacterial cultures were divided into two counterparts; one served as a control (open symbols), while the other was irradiated with 400 Gy (closed symbols). The cultures contained [3H]thymidine-labeled chromosome; kinetics of their degradation were monitored during incubation at 37&#xB0;. AB1157 (&#x25A1;,&#x25AA;); &#x2206;recA (&#x25CB;,&#x25CF;); lexA3 (&#x25B5;,&#x25B4;); recAo281 (+,); lexA3 recAo281 (&#x25BF;,&#x25BE;); recA730 (&#x25CA;,&#x2666;); and recA730 lexA3 (&#xD7;,&#x2297;). Each value is a mean of three independent experiments, with error bars representing SD. (B) Survival of &#x3B3;-irradiated bacteria. Wild-type strain AB1157 (&#x25AA;) and its recA (&#x25CF;), lexA3 (h), recAo281 (&#x25BE;), lexA3 recAo281 (&#x25BF;), recA730 (&#x2666;), and recA730 lexA3 (&#x25CA;) derivatives. Fraction survival is given as a fraction of the unirradiated control. Each value is a mean of three independent experiments, with error bars representing SD. ssDNA, single-stranded DNA; wt, wild-type." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-894880217" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;(A) DNA degradation in &#x3B3;-irradiated &lt;em&gt;E. coli&lt;/em&gt; depends on RecA protein concentration and its rate of association with ssDNA. Bacterial cultures were divided into two counterparts; one served as a control (open symbols), while the other was irradiated with 400 Gy (closed symbols). The cultures contained [&lt;sup&gt;3&lt;/sup&gt;H]thymidine-labeled chromosome; kinetics of their degradation were monitored during incubation at 37&#xB0;. AB1157 (&#x25A1;,&#x25AA;); &lt;em&gt;&#x2206;recA&lt;/em&gt; (&#x25CB;,&#x25CF;); &lt;em&gt;lexA3&lt;/em&gt; (&#x25B5;,&#x25B4;); &lt;em&gt;recAo281&lt;/em&gt; (+,); &lt;em&gt;lexA3 recAo281&lt;/em&gt; (&#x25BF;,&#x25BE;); &lt;em&gt;recA730&lt;/em&gt; (&#x25CA;,&#x2666;); and &lt;em&gt;recA730 lexA3&lt;/em&gt; (&#xD7;,&#x2297;). Each value is a mean of three independent experiments, with error bars representing SD. (B) Survival of &#x3B3;-irradiated bacteria. Wild-type strain AB1157 (&#x25AA;) and its &lt;em&gt;recA&lt;/em&gt; (&#x25CF;), &lt;em&gt;lexA3&lt;/em&gt; (h), recA&lt;sub&gt;o&lt;/sub&gt;281 (&#x25BE;), &lt;em&gt;lexA3 recA&lt;sub&gt;o&lt;/sub&gt;281&lt;/em&gt; (&#x25BF;), &lt;em&gt;recA730&lt;/em&gt; (&#x2666;), and &lt;em&gt;recA730 lexA3&lt;/em&gt; (&#x25CA;) derivatives. Fraction survival is given as a fraction of the unirradiated control. Each value is a mean of three independent experiments, with error bars representing SD. ssDNA, single-stranded DNA; wt, wild-type.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/9/3091/F1.medium.gif" width="440" height="266"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/7/9/3091/F1.medium.gif" width="440" height="266"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/9/3091/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/9/3091/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/486448" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-24" class="first-child">(A) DNA degradation in γ-irradiated <em>E. coli</em> depends on RecA protein concentration and its rate of association with ssDNA. Bacterial cultures were divided into two counterparts; one served as a control (open symbols), while the other was irradiated with 400 Gy (closed symbols). The cultures contained [<sup>3</sup>H]thymidine-labeled chromosome; kinetics of their degradation were monitored during incubation at 37°. AB1157 (□,▪); <em>∆recA</em> (○,●); <em>lexA3</em> (▵,▴); <em>recAo281</em> (+,<span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3091/F1/embed/inline-graphic-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3091/F1/embed/inline-graphic-1.gif"/></noscript></span>); <em>lexA3 recAo281</em> (▿,▾); <em>recA730</em> (◊,♦); and <em>recA730 lexA3</em> (×,⊗). Each value is a mean of three independent experiments, with error bars representing SD. (B) Survival of γ-irradiated bacteria. Wild-type strain AB1157 (▪) and its <em>recA</em> (●), <em>lexA3</em> (h), recA<sub>o</sub>281 (▾), <em>lexA3 recA<sub>o</sub>281</em> (▿), <em>recA730</em> (♦), and <em>recA730 lexA3</em> (◊) derivatives. Fraction survival is given as a fraction of the unirradiated control. Each value is a mean of three independent experiments, with error bars representing SD. ssDNA, single-stranded DNA; wt, wild-type.</p><div class="sb-div caption-clear"></div></div></div><p id="p-25">On the other hand, a <em>recA</em> null mutant (DE586) showed much stronger, continuous DNA degradation that resulted in degradation of ∼95% of its cellular DNA after ∼120 min postirradiation incubation (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1A</a>). Therefore, we confirmed a “reckless” DNA degradation in the <em>recA</em> mutant.</p><p id="p-26">Since RecA protein regulates SOS induction and is itself regulated by SOS, we wanted to assess the effect of the SOS response on DNA degradation in γ-irradiated <em>E. coli</em>. For that, we made the SOS system in AB1157 uninducible by introducing a <em>lexA3</em> allele, coding for a noncleavable SOS repressor. By western analysis, we estimated the relative concentrations of RecA protein in the wild-type cells that were or were not exposed to γ-rays. Our result revealed an increase of around threefold the basal RecA concentration in the irradiated wild-type strain (35 min after irradiation) compared to the unirradiated control (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). The lower level of SOS induction compared to the previous study (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Sassanfar and Roberts 1990</a>) is likely due to short postirradiation incubation time and different SOS-inducing agents used. As expected, no change of (basal) RecA concentration was observed in the <em>lexA3</em> mutant upon irradiation (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2A</a>). As shown in <a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1A</a>, the <em>lexA3</em> mutant also showed continuous DNA degradation during ∼120 min postirradiation incubation, but the amount of degradation was lower than in the <em>recA</em> mutant, with ∼20% of its DNA being spared. This result shows that the SOS system influences DNA degradation in γ-irradiated <em>E. coli</em>. In order to check which function of the SOS is important for suppression of DNA degradation, we made use of <em>recA<sub>o</sub>281</em>, an operator mutation that enables SOS-independent constitutive overexpression of RecA protein (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Clark 1982</a>). RecA concentration was ∼4 fold higher in a <em>recA<sub>o</sub>281</em> mutant compared to the unirradiated wild-type strain, while being about the same in their irradiated counterparts (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2B</a>). DNA degradation was similar in the <em>recA<sub>o</sub>281</em> mutant and wild-type strain (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1A</a>). A <em>lexA3 recA<sub>o</sub>281</em> mutant is SOS-deficient, yet expresses similarly elevated (around fourfold) concentration of RecA protein (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2B</a>). After 2 hr of postirradiation incubation, it degraded ∼40% of its DNA, which is similar to degradation in the wild-type strain, whereas about threefold more of its DNA was preserved compared to DNA in the irradiated single <em>lexA3</em> mutant (<a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1A</a>). This suggests that RecA concentration is important for suppressing DNA degradation, while other parts of the SOS response have a minor effect. Therefore, our results reveal an inverse relationship between RecA concentration and DNA degradation in irradiated <em>E. coli</em>, which led us to conclude that RecA protein suppresses DNA degradation by mass effect.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/9/3091/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Immunoblotting analysis of relative RecA protein concentrations in E. coli (A and B). Protein samples obtained from the cells that were irradiated with 400 Gy of &#x3B3;-rays and subsequently incubated for 35 min at 37&#xB0; are marked with *. PVDF membranes stained with Amido Black were used as a loading control (lower panels). SDS-PAGE gels and PVDF membranes are from one experiment, while graph depicts mean and SD (error bars) from multiple replicates (numbered in parentheses) from two independent experiments. PVDF, polyvinylidene fluoride; SDS-PAGE, sodium dodecyl sulfate polyacrylamide gel electrophoresis; wt, wild-type." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-894880217" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Immunoblotting analysis of relative RecA protein concentrations in &lt;em&gt;E. coli&lt;/em&gt; (A and B). Protein samples obtained from the cells that were irradiated with 400 Gy of &#x3B3;-rays and subsequently incubated for 35 min at 37&#xB0; are marked with *. PVDF membranes stained with Amido Black were used as a loading control (lower panels). SDS-PAGE gels and PVDF membranes are from one experiment, while graph depicts mean and SD (error bars) from multiple replicates (numbered in parentheses) from two independent experiments. PVDF, polyvinylidene fluoride; SDS-PAGE, sodium dodecyl sulfate polyacrylamide gel electrophoresis; wt, wild-type.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/9/3091/F2.medium.gif" width="440" height="202"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/7/9/3091/F2.medium.gif" width="440" height="202"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/9/3091/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/9/3091/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/486406" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-27" class="first-child">Immunoblotting analysis of relative RecA protein concentrations in <em>E. coli</em> (A and B). Protein samples obtained from the cells that were irradiated with 400 Gy of γ-rays and subsequently incubated for 35 min at 37° are marked with *. PVDF membranes stained with Amido Black were used as a loading control (lower panels). SDS-PAGE gels and PVDF membranes are from one experiment, while graph depicts mean and SD (error bars) from multiple replicates (numbered in parentheses) from two independent experiments. PVDF, polyvinylidene fluoride; SDS-PAGE, sodium dodecyl sulfate polyacrylamide gel electrophoresis; wt, wild-type.</p><div class="sb-div caption-clear"></div></div></div><p id="p-28">Next, we wanted to determine how the RecA rate of association with ssDNA influences DNA degradation in irradiated cells. For that, we used a <em>recA730</em> mutant, which produces the RecAE38K mutant protein that shows an increased rate of association with ssDNA and hence competes more efficiently with SSB protein for binding to ssDNA (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Lavery and Kowalczykowski 1992</a>). When introduced into AB1157, the <em>recA730</em> mutation reduced DNA degradation in the resulting strain ∼1.5-fold (<a id="xref-fig-1-7" class="xref-fig" href="#F1">Figure 1A</a>). This effect can be caused by an increased affinity of the RecAE38K for ssDNA, but also by an increased concentration of the enzyme in that mutant due to constitutive SOS induction. RecAE38K concentration in unirradiated and irradiated cells of the <em>recA730</em> mutant was around four- and eightfold higher, respectively, than the RecA concentration in an unirradiated wild-type strain (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2</a>). We measured the effect of the increased affinity of RecAE38K for ssDNA by using a <em>lexA3 recA730</em> mutant, which produces a basal level of RecA E38K that is equal to that of RecA in the <em>lexA3</em> mutant (<a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2A</a>). The <em>lexA3 recA730</em> mutant showed greatly reduced (about twofold) DNA degradation compared to that in the <em>lexA3</em> mutant (<a id="xref-fig-1-8" class="xref-fig" href="#F1">Figure 1A</a>), revealing that RecA dependency of DNA degradation is proportionate to RecA’s rate of association with ssDNA. In order to check whether the protective effect of RecAE38K protein on DNA degradation is indeed due to its high affinity to ssDNA, and not to its possible nonspecific binding to dsDNA, we determined the titer of T4 <em>2</em> phage in <em>recA730</em> and <em>recA730 recB</em> mutants. The T4 <em>2</em> mutant phage genome is a linear dsDNA with free, blunt ends and hence susceptible to RecBCD binding and nucleolytic degradation. If RecAE38K would bind to dsDNA, hence blocking the access and activity of RecBCD on it, the titer of T4 <em>2</em> phage should increase. In comparison to a <em>recB</em> mutant (which cannot degrade the T4 <em>2</em> phage genome and therefore enables maximal phage yield), the T4 <em>2</em> phage plating efficiencies in the <em>recA730</em> and wild-type strains were 0.0019 ± 0.001 and 0.0017 ± 0.0007, respectively, which is significantly different (<em>P</em> = 0.001, two-tailed <em>t</em>-test) from the <em>recB</em> derivative of <em>recA730</em> mutant that had 0.94 ± 0.19 of <em>recB’s</em> titer (<em>n</em> = 3), suggesting that RecBCD inhibits phage propagation in the <em>recA730</em> mutant and that phage genome metabolism is otherwise not impaired in that mutant. Hence, we infer that RecAE38K protein does not interfere with degradation of purely dsDNA.</p><p id="p-29">The difference in degradation between <em>recA730</em> and <em>recA730 lexA3</em> mutants (<a id="xref-fig-1-9" class="xref-fig" href="#F1">Figure 1A</a>) is likely mostly due to different concentrations of the RecAE38K protein in these strains. The same, moderate amount of DNA degradation observed in the <em>lexA3 recA730</em> and <em>lexA3 recA<sub>o</sub>281</em> mutants (<a id="xref-fig-1-10" class="xref-fig" href="#F1">Figure 1A</a>) indicates that, for DNA degradation inhibition, the higher RecA rate of association with ssDNA can compensate for lower RecA concentration, and that SOS induction is not required <em>per se</em> for protection of cellular DNA from degradation.</p><p id="p-30">In summary, we have shown that RecA protein protects a fragmented chromosome from degradation in γ-irradiated wild-type <em>E. coli</em>. The protective activity depends on RecA concentration and its rate of association with ssDNA.</p></div><div id="sec-9" class="subsection"><h3>DNA degradation is distributed uniformly in wild-type cells</h3><p id="p-31">The DNA degradation experiments used in this study provide bulk population data that do not give us information on degradation distribution in a population. To assess the distribution of DNA degradation in a cell population, we visualized bacteria by microscopy and determined the fraction of cells lacking DAPI staining material in a population of wild-type (AB1157) and <em>recA</em> bacteria after 90 min postirradiation incubation. Unirradiated populations of wild-type and <em>recA</em> cells contained 0.055 ± 0.001% and 8.87 ± 0.93% anucleate cells, respectively (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). On the other hand, wild-type and <em>recA</em> populations γ-irradiated with 400 Gy had 0.49 ± 0.10% and 84.1 ± 2.4% of cells lacking a DAPI signal, respectively, after 90 min incubation (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>). These data suggest that DNA degradation in wild-type cells is tightly regulated since it is very uniform, so that ∼35% of their degraded cellular DNA (<a id="xref-fig-1-11" class="xref-fig" href="#F1">Figure 1A</a>) is reflected in &lt;1% of chromosomeless cells. Conversely, only ∼16% of irradiated <em>recA</em> cells contained DNA, which is consistent with heavy, uncontrolled DNA degradation in them, sparing just ∼5% of their DNA (<a id="xref-fig-1-12" class="xref-fig" href="#F1">Figure 1A</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/9/3091/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="DNA degradation is strictly controlled in wild-type, but not in recA cells. A fraction of anucleate cells was determined in wild-type (AB1157) and recA populations, which were either unirradiated (left side of the panel) or irradiated with 400 Gy of &#x3B3;-rays and then incubated for 90 min at 37&#xB0; (right side of the panel). Cells were fixed with osmium tetroxide, their DNA stained with DAPI, and visualized by fluorescence microscopy. An anucleated cell count in an analyzed population from three experiments is represented in a table. DAPI, 4&#x2019;,6-diamidino-2-phenylindole; wt, wild-type." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-894880217" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;DNA degradation is strictly controlled in wild-type, but not in &lt;em&gt;recA&lt;/em&gt; cells. A fraction of anucleate cells was determined in wild-type (AB1157) and &lt;em&gt;recA&lt;/em&gt; populations, which were either unirradiated (left side of the panel) or irradiated with 400 Gy of &#x3B3;-rays and then incubated for 90 min at 37&#xB0; (right side of the panel). Cells were fixed with osmium tetroxide, their DNA stained with DAPI, and visualized by fluorescence microscopy. An anucleated cell count in an analyzed population from three experiments is represented in a table. DAPI, 4&#x2019;,6-diamidino-2-phenylindole; wt, wild-type.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/9/3091/F3.medium.gif" width="377" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/7/9/3091/F3.medium.gif" width="377" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/9/3091/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/9/3091/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/486575" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-32" class="first-child">DNA degradation is strictly controlled in wild-type, but not in <em>recA</em> cells. A fraction of anucleate cells was determined in wild-type (AB1157) and <em>recA</em> populations, which were either unirradiated (left side of the panel) or irradiated with 400 Gy of γ-rays and then incubated for 90 min at 37° (right side of the panel). Cells were fixed with osmium tetroxide, their DNA stained with DAPI, and visualized by fluorescence microscopy. An anucleated cell count in an analyzed population from three experiments is represented in a table. DAPI, 4’,6-diamidino-2-phenylindole; wt, wild-type.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-10" class="subsection"><h3>Role of RecA protein in survival of γ-irradiated cells</h3><p id="p-33">Since degradation of a fragmented chromosome reflects DSB processing reactions, we wanted to relate it to a DSB repair process. Therefore, we determined survival of γ-irradiated bacteria as a measure of the efficiency of their DSB repair. As expected, the <em>recA</em> mutant showed an extreme sensitivity to γ-rays; at 90 Gy dose, its survival was more than three orders of magnitude lower than that of the wild-type strain AB1157 (<a id="xref-fig-1-13" class="xref-fig" href="#F1">Figure 1B</a>). The <em>lexA3</em> mutant had somewhat higher survival than the <em>recA</em> mutant (<a id="xref-fig-1-14" class="xref-fig" href="#F1">Figure 1B</a>), which is not surprising considering that it retains basal RecA concentration. Survival of the <em>lexA3 recA<sub>o</sub>281</em> mutant was considerably higher than that of the <em>lexA3</em> mutant (also due to higher RecA concentration), whereas the <em>recA<sub>o</sub>281</em> mutant had the same survival as the wild-type strain (<a id="xref-fig-1-15" class="xref-fig" href="#F1">Figure 1B</a>).</p><p id="p-34">While the <em>recA730</em> mutant had essentially the same survival as the wild-type strain, its <em>lexA3</em> derivative showed extreme sensitivity to γ-rays, similar to that of the <em>recA</em> and <em>lexA3</em> mutants (<a id="xref-fig-1-16" class="xref-fig" href="#F1">Figure 1B</a>). Since the <em>recA730 lexA3</em> mutant showed highly reduced DNA degradation compared to the <em>recA</em> (and <em>lexA3)</em> mutant, whereas it displayed almost identical γ-survival (compare <a id="xref-fig-1-17" class="xref-fig" href="#F1">Figure 1, A and B</a>), we conclude that reducing “reckless” degradation is insufficient to enable DNA repair.</p><p id="p-35">A comparison of γ-survivals shows that the important factors for higher survival of γ-irradiated cells are: higher RecA protein concentration (hence the higher survival of <em>recA<sub>o</sub>281 lexA3</em> than <em>lexA3</em> and <em>recA730 lexA3</em> mutants) and SOS induction (causing increased survival of <em>recA<sub>o</sub>281</em> compared to the <em>recAo281 lexA3</em> mutant).</p></div><div id="sec-11" class="subsection"><h3>RecA inhibition of DNA degradation is independent of DNA polymerase V</h3><p id="p-36">In <em>E. coli</em>, RecA is required for three processes, namely for HR, SOS induction, and activation of mutagenic DNA polymerase V (PolV) (see <em>Introduction</em>). Since we have shown that RecA’s function in inhibition of DNA degradation does not necessarily correspond to its role in DSB repair and SOS induction, we wanted to determine its relationship with PolV activation. For that goal, we determined DNA degradation and γ-survival of a PolV-deficient <em>umuDC</em> mutant and its <em>recA</em> derivative. As shown in <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>, the PolV-deficient mutant had mildly increased DNA degradation compared to the wild-type strain, while its γ-survival was the same as that of the wild-type strain. Furthermore, no difference was observed in DNA degradation and survival of <em>umuDC recA</em> and <em>recA</em> mutants (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>). These results indicate that the RecA protective role during DNA degradation does not depend on PolV.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/9/3091/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="DNA degradation is unrelated to DNA polymerase V. Kinetics of [3H]thymidine-labeled DNA degradation (A), and survival (B) of &#x3B3;-irradiated wild-type strain AB1157 (&#x25AA;) and its umuDC (&#x25BE;), recA (&#x25B4;), and umuDC recA (&#x2666;) derivatives. For DNA degradation assay, bacterial cultures were divided into two fractions; one served as a control (open symbols), while the other was irradiated with 400 Gy (closed symbols). Fraction survival is given as the fraction of the unirradiated control. Each value is a mean of three independent experiments, with error bars representing SD. wt, wild-type." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-894880217" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;DNA degradation is unrelated to DNA polymerase V. Kinetics of [&lt;sup&gt;3&lt;/sup&gt;H]thymidine-labeled DNA degradation (A), and survival (B) of &#x3B3;-irradiated wild-type strain AB1157 (&#x25AA;) and its &lt;em&gt;umuDC&lt;/em&gt; (&#x25BE;), &lt;em&gt;recA&lt;/em&gt; (&#x25B4;), and &lt;em&gt;umuDC recA&lt;/em&gt; (&#x2666;) derivatives. For DNA degradation assay, bacterial cultures were divided into two fractions; one served as a control (open symbols), while the other was irradiated with 400 Gy (closed symbols). Fraction survival is given as the fraction of the unirradiated control. Each value is a mean of three independent experiments, with error bars representing SD. wt, wild-type.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/9/3091/F4.medium.gif" width="440" height="273"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/7/9/3091/F4.medium.gif" width="440" height="273"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/9/3091/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/9/3091/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/486477" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-37" class="first-child">DNA degradation is unrelated to DNA polymerase V. Kinetics of [<sup>3</sup>H]thymidine-labeled DNA degradation (A), and survival (B) of γ-irradiated wild-type strain AB1157 (▪) and its <em>umuDC</em> (▾), <em>recA</em> (▴), and <em>umuDC recA</em> (♦) derivatives. For DNA degradation assay, bacterial cultures were divided into two fractions; one served as a control (open symbols), while the other was irradiated with 400 Gy (closed symbols). Fraction survival is given as the fraction of the unirradiated control. Each value is a mean of three independent experiments, with error bars representing SD. wt, wild-type.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-12" class="subsection"><h3>DNA degradation in recB and recD mutants is inhibited by RecA</h3><p id="p-38">We wanted to determine whether the RecA inhibition of DNA degradation is restricted to RecBCD-catalyzed reactions, or whether it is a more general phenomenon. For this aim, we determined DNA degradation in γ-irradiated <em>recB</em> and <em>recD</em> mutants. No degradation was observed in these mutants (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5, A and B</a>), which is not surprising considering their ExoV<sup>−</sup> phenotype. However, their <em>recA</em> derivatives did degrade their damaged DNA. A <em>recB recA</em> mutant degraded its DNA continuously during postirradiation incubation, with degradation reaching a level close to that of the wild-type strain after 130 min (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5A</a>). This result indicates that RecA protects DNA from degradation in a <em>recB</em> null mutant. Since in the <em>recB</em> mutant dsDNA ends are processed by the RecQ helicase, we evaluated the role of RecQ in DNA degradation in the <em>recB recA</em> mutant. A triple <em>recB recA recQ</em> mutant degraded ∼10% of its DNA after 130 min incubation, which is about twofold less compared to its parental RecQ<sup>+</sup> strain (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5A</a>), indicating that a helicase activity participates in DSB processing reactions that result in degradation of a shattered <em>E. coli</em> chromosome. At the same time, the <em>recQ</em> mutation did not affect DNA degradation in wild-type or <em>recA</em> and <em>recB</em> null mutants (<a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5A</a>), suggesting that the RecQ helicase role is specifically pronounced in the <em>recB recA</em> mutant.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/9/3091/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="DNA degradation in recB (A) and recD (B) mutants is inhibited by RecA protein. Bacterial cultures were divided into two counterparts; one served as a control (open symbols), while the other was irradiated with 400 Gy (closed symbols). The cultures contained [3H]thymidine-labeled chromosome; kinetics of its degradation was monitored during incubation at 37&#xB0;. (A) AB1157 (&#x25A1;,&#x25AA;); recQ (&#x25CB;,&#x25CF;); recB (&#x25B5;,&#x25B4;); recB recQ (+,); recB recA (&#x25BF;,&#x25BE;); recB recA recQ (&#x25CA;,&#x2666;); recQ recA (&#xD7;,&#x2297;), and recA (- - -). (B) AB1157 (&#x25A1;,&#x25AA;); recD (&#x25CB;,&#x25CF;); recD recA (&#x25B5;,&#x25B4;); and ExoI&#x2212; ExoVII&#x2212; ExoX&#x2212; SbcCD&#x2212; RecA&#x2212; RecD&#x2212; (&#x25BF;,&#x25BE;). Each value is a mean of three independent experiments, with error bars representing SD. wt, wild-type." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-894880217" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;DNA degradation in &lt;em&gt;recB&lt;/em&gt; (A) and &lt;em&gt;recD&lt;/em&gt; (B) mutants is inhibited by RecA protein. Bacterial cultures were divided into two counterparts; one served as a control (open symbols), while the other was irradiated with 400 Gy (closed symbols). The cultures contained [&lt;sup&gt;3&lt;/sup&gt;H]thymidine-labeled chromosome; kinetics of its degradation was monitored during incubation at 37&#xB0;. (A) AB1157 (&#x25A1;,&#x25AA;); &lt;em&gt;recQ&lt;/em&gt; (&#x25CB;,&#x25CF;); &lt;em&gt;recB&lt;/em&gt; (&#x25B5;,&#x25B4;); &lt;em&gt;recB recQ&lt;/em&gt; (+,); &lt;em&gt;recB recA&lt;/em&gt; (&#x25BF;,&#x25BE;); &lt;em&gt;recB recA recQ&lt;/em&gt; (&#x25CA;,&#x2666;); &lt;em&gt;recQ recA&lt;/em&gt; (&#xD7;,&#x2297;), and &lt;em&gt;recA&lt;/em&gt; (- - -). (B) AB1157 (&#x25A1;,&#x25AA;); &lt;em&gt;recD&lt;/em&gt; (&#x25CB;,&#x25CF;); &lt;em&gt;recD recA&lt;/em&gt; (&#x25B5;,&#x25B4;); and ExoI&lt;sup&gt;&#x2212;&lt;/sup&gt; ExoVII&lt;sup&gt;&#x2212;&lt;/sup&gt; ExoX&lt;sup&gt;&#x2212;&lt;/sup&gt; SbcCD&lt;sup&gt;&#x2212;&lt;/sup&gt; RecA&lt;sup&gt;&#x2212;&lt;/sup&gt; RecD&lt;sup&gt;&#x2212;&lt;/sup&gt; (&#x25BF;,&#x25BE;). Each value is a mean of three independent experiments, with error bars representing SD. wt, wild-type.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/9/3091/F5.medium.gif" width="440" height="270"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/7/9/3091/F5.medium.gif" width="440" height="270"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/9/3091/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/9/3091/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/486401" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-39" class="first-child">DNA degradation in <em>recB</em> (A) and <em>recD</em> (B) mutants is inhibited by RecA protein. Bacterial cultures were divided into two counterparts; one served as a control (open symbols), while the other was irradiated with 400 Gy (closed symbols). The cultures contained [<sup>3</sup>H]thymidine-labeled chromosome; kinetics of its degradation was monitored during incubation at 37°. (A) AB1157 (□,▪); <em>recQ</em> (○,●); <em>recB</em> (▵,▴); <em>recB recQ</em> (+,<span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3091/F5/embed/inline-graphic-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3091/F5/embed/inline-graphic-2.gif"/></noscript></span>); <em>recB recA</em> (▿,▾); <em>recB recA recQ</em> (◊,♦); <em>recQ recA</em> (×,⊗), and <em>recA</em> (- - -). (B) AB1157 (□,▪); <em>recD</em> (○,●); <em>recD recA</em> (▵,▴); and ExoI<sup>−</sup> ExoVII<sup>−</sup> ExoX<sup>−</sup> SbcCD<sup>−</sup> RecA<sup>−</sup> RecD<sup>−</sup> (▿,▾). Each value is a mean of three independent experiments, with error bars representing SD. wt, wild-type.</p><div class="sb-div caption-clear"></div></div></div><p id="p-40">A <em>recD recA</em> mutant (DE595) also showed constitutive DNA degradation; ∼40% of its DNA was degraded after 2 hr of incubation, thus exceeding the degradation level observed in the wild-type strain (<a id="xref-fig-5-5" class="xref-fig" href="#F5">Figure 5B</a>). The RecBC enzyme is a nuclease-free helicase whose unwound products are subject to the activity of ssExos (<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Đermić 2006</a>; <a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Đermić <em>et al.</em> 2006a</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Rinken <em>et al.</em> 1992</a>), which results in limited degradation of the unwound DNA (<a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Rinken <em>et al.</em> 1992</a>). An earlier study (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Kuzminov and Stahl 1997</a>) has shown degradation of linearized plasmid DNA in a <em>recD recA</em> but not in a <em>recB recA</em> mutant.</p><p id="p-41">In summary, we have shown that the RecA inhibition of DNA degradation is not only restricted to RecBCD-expressing wild-type bacteria, but also applies to the <em>recB</em> and <em>recD</em> mutants, suggesting that it is a more general phenomenon in <em>E. coli</em>. Also, we have shown that RecA-controlled DNA degradation involves a helicase activity.</p></div><div id="sec-13" class="subsection"><h3>DNA degradation in the recB1080 mutant is not inhibited by RecA protein</h3><p id="p-42">To further characterize the interplay of a helicase activity and RecA polymerization in DNA degradation, we made use of a <em>recB1080</em> mutant strain RIK174, which produces the RecB<sup>1080</sup>CD enzyme. This enzyme is a fast and processive helicase, but is also nuclease-free and unable to load RecA onto the unwound 3′ tail (<a id="xref-ref-55-4" class="xref-bibr" href="#ref-55">Yu <em>et al.</em> 1998</a>; <a id="xref-ref-3-3" class="xref-bibr" href="#ref-3">Anderson <em>et al.</em> 1999</a>). <em>In vitro</em>, the enzyme unwinds the linear DNA duplex, releasing full length, RecA-free ss tails (<a id="xref-ref-55-5" class="xref-bibr" href="#ref-55">Yu <em>et al.</em> 1998</a>; <a id="xref-ref-3-4" class="xref-bibr" href="#ref-3">Anderson <em>et al.</em> 1999</a>). γ-irradiated RIK174 degraded ∼25% of its DNA after 240 min of incubation, which was ∼70% of DNA degradation in the wild-type bacteria (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>). However, a <em>recB1080 recA</em> mutant showed a similarly low level of DNA degradation as its RecA<sup>+</sup> parental strain (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6</a>), suggesting that DNA degradation in the <em>recB1080</em> mutant is not inhibited by the RecA protein. This result indicates that, during processing of DSBs by a powerful nuclease-free helicase, long unwound tails prevent excessive DNA degradation, in effect relieving the RecA regulation of DNA degradation.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/9/3091/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="DNA degradation in the recB1080 mutant is weak and uninhibited by RecA protein. Bacterial cultures were divided into two counterparts; one served as a control (open symbols), while the other was irradiated with 400 Gy (closed symbols). The cultures contained [3H]thymidine-labeled chromosome; kinetics of its degradation was monitored during incubation at 37&#xB0;. AB1157 (&#x25A1;,&#x25AA;); recB1080 (&#x25B5;,&#x25B4;); and recB1080 recA (&#x25BF;,&#x25BE;). Each value is a mean of three independent experiments, with error bars representing SD. wt, wild-type." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-894880217" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;DNA degradation in the &lt;em&gt;recB1080&lt;/em&gt; mutant is weak and uninhibited by RecA protein. Bacterial cultures were divided into two counterparts; one served as a control (open symbols), while the other was irradiated with 400 Gy (closed symbols). The cultures contained [&lt;sup&gt;3&lt;/sup&gt;H]thymidine-labeled chromosome; kinetics of its degradation was monitored during incubation at 37&#xB0;. AB1157 (&#x25A1;,&#x25AA;); &lt;em&gt;recB1080&lt;/em&gt; (&#x25B5;,&#x25B4;); and &lt;em&gt;recB1080 recA&lt;/em&gt; (&#x25BF;,&#x25BE;). Each value is a mean of three independent experiments, with error bars representing SD. wt, wild-type.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/9/3091/F6.medium.gif" width="342" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/7/9/3091/F6.medium.gif" width="342" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/9/3091/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/9/3091/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/486490" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-43" class="first-child">DNA degradation in the <em>recB1080</em> mutant is weak and uninhibited by RecA protein. Bacterial cultures were divided into two counterparts; one served as a control (open symbols), while the other was irradiated with 400 Gy (closed symbols). The cultures contained [<sup>3</sup>H]thymidine-labeled chromosome; kinetics of its degradation was monitored during incubation at 37°. AB1157 (□,▪); <em>recB1080</em> (▵,▴); and <em>recB1080 recA</em> (▿,▾). Each value is a mean of three independent experiments, with error bars representing SD. wt, wild-type.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-14" class="subsection"><h3>Inactivation of 3′–5′ ssExos greatly reduces DNA degradation and makes it independent of RecA</h3><p id="p-44">It was shown earlier that inactivation of ExoI, SbcCD, and ExoVII, ssExos that degrade 3′ overhangs, prevents “reckless” DNA degradation in a <em>recA</em> mutant (<a id="xref-ref-56-2" class="xref-bibr" href="#ref-56">Zahradka <em>et al.</em> 2009</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Repar <em>et al.</em> 2013</a>). We made a quadruple mutant deficient in ExoI, SbcCD, ExoVII, and Exonuclease X (ExoX), ssExos that trim 3′ overhangs (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Lovett 2011</a>), and measured its DNA degradation. The mutant degraded ∼10% of its DNA after 130 min of incubation (<a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7</a>), which is close to the amount of DNA degraded in its unirradiated part (<a id="xref-fig-7-2" class="xref-fig" href="#F7">Figure 7</a>), and is greatly reduced compared to degradation in its ssExo<sup>+</sup> parental strain. On the other hand, DNA degradation in a strain lacking 5′–3′ ssExos RecJ and ExoVII (<a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Lovett 2011</a>) was similar to that in wild-type bacteria amounting to 31 ± 4% of genomic DNA after 130 min of incubation (<a id="xref-fig-7-3" class="xref-fig" href="#F7">Figure 7</a>).</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/9/3091/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="DNA degradation in a mutant lacking 3&#x2032;&#x2013;5&#x2032; ssExos is very weak and not inhibited by RecA protein. Bacterial cultures were divided into two fractions; one served as a control (open symbols) while the other was irradiated with 400 Gy (closed symbols). The cultures contained [3H]thymidine-labeled chromosome; kinetics of its degradation were monitored during incubation at 37&#xB0;. AB1157 (&#x25A1;,&#x25AA;); ExoI&#x2212; ExoVII&#x2212; ExoX&#x2212; SbcCD&#x2212; (&#x25B5;,&#x25B4;); ExoI&#x2212; ExoVII&#x2212; ExoX&#x2212; SbcCD&#x2212; RecA&#x2212; (&#x25BF;,&#x25BE;); and RecJ&#x2212; ExoVII&#x2212; (&#x25CB;,&#x25CF;). Each value is a mean of three independent experiments, with error bars representing SD. wt, wild-type." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-894880217" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;DNA degradation in a mutant lacking 3&#x2032;&#x2013;5&#x2032; ssExos is very weak and not inhibited by RecA protein. Bacterial cultures were divided into two fractions; one served as a control (open symbols) while the other was irradiated with 400 Gy (closed symbols). The cultures contained [&lt;sup&gt;3&lt;/sup&gt;H]thymidine-labeled chromosome; kinetics of its degradation were monitored during incubation at 37&#xB0;. AB1157 (&#x25A1;,&#x25AA;); ExoI&lt;sup&gt;&#x2212;&lt;/sup&gt; ExoVII&lt;sup&gt;&#x2212;&lt;/sup&gt; ExoX&lt;sup&gt;&#x2212;&lt;/sup&gt; SbcCD&lt;sup&gt;&#x2212;&lt;/sup&gt; (&#x25B5;,&#x25B4;); ExoI&lt;sup&gt;&#x2212;&lt;/sup&gt; ExoVII&lt;sup&gt;&#x2212;&lt;/sup&gt; ExoX&lt;sup&gt;&#x2212;&lt;/sup&gt; SbcCD&lt;sup&gt;&#x2212;&lt;/sup&gt; RecA&lt;sup&gt;&#x2212;&lt;/sup&gt; (&#x25BF;,&#x25BE;); and RecJ&lt;sup&gt;&#x2212;&lt;/sup&gt; ExoVII&lt;sup&gt;&#x2212;&lt;/sup&gt; (&#x25CB;,&#x25CF;). Each value is a mean of three independent experiments, with error bars representing SD. wt, wild-type.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/9/3091/F7.medium.gif" width="340" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/7/9/3091/F7.medium.gif" width="340" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/9/3091/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/9/3091/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/486539" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-45" class="first-child">DNA degradation in a mutant lacking 3′–5′ ssExos is very weak and not inhibited by RecA protein. Bacterial cultures were divided into two fractions; one served as a control (open symbols) while the other was irradiated with 400 Gy (closed symbols). The cultures contained [<sup>3</sup>H]thymidine-labeled chromosome; kinetics of its degradation were monitored during incubation at 37°. AB1157 (□,▪); ExoI<sup>−</sup> ExoVII<sup>−</sup> ExoX<sup>−</sup> SbcCD<sup>−</sup> (▵,▴); ExoI<sup>−</sup> ExoVII<sup>−</sup> ExoX<sup>−</sup> SbcCD<sup>−</sup> RecA<sup>−</sup> (▿,▾); and RecJ<sup>−</sup> ExoVII<sup>−</sup> (○,●). Each value is a mean of three independent experiments, with error bars representing SD. wt, wild-type.</p><div class="sb-div caption-clear"></div></div></div><p id="p-46">Since the mutant lacking four 3′–5′ ssExos is an ExoV<sup>−</sup> phenocopy with respect to DNA degradation, although it expresses an intact RecBCD enzyme, we checked the enzyme’s nuclease activity by assessing its ability to inhibit growth of a T4 <em>2</em> phage. Relative to the titer on the <em>recB</em> mutant, T4 <em>2</em> phage plating efficiency on the wild-type strain was 0.0067 ± 0.0043, and that on the 3′–5′ ssExos-deficient mutant was 0.019 ± 0.012 (<em>n</em> = 3), which is not significantly different (<em>P</em> = 0.170, two-tailed <em>t</em>-test). This result indicates that RecBCD is proficient in binding to and degrading phage DNA in the mutant lacking 3′–5′ ssExos.</p><p id="p-47">A <em>recA</em> derivative of the quadruple ExoI<sup>−</sup> SbcCD<sup>−</sup> ExoVII<sup>−</sup> ExoX<sup>−</sup> mutant showed about the same, low-level ∼10% degradation as its RecA<sup>+</sup> counterpart (<a id="xref-fig-7-4" class="xref-fig" href="#F7">Figure 7</a>), suggesting that DSB processing in bacteria with a preserved 3′ tail is not affected by RecA.</p><p id="p-48">Similarly, DNA degradation in a <em>recD recA</em> mutant was greatly abolished by inactivation of the four 3′–5′ ssExos (<a id="xref-fig-5-6" class="xref-fig" href="#F5">Figure 5B</a>). Therefore, we conclude that the constitutive chromosome degradation observed in the irradiated <em>recD recA</em> mutant is dependent on degradation of 3′-ended ss overhangs by the 3′–5′ ssExos.</p><p id="p-49">Hence, our results indicate that 3′ overhangs suppress DSB processing, especially when spared from 3′–5′ ssExos.</p></div></div><div class="section discussion" id="sec-15"><h2 class="">Discussion</h2><p id="p-50">To gain better insight into the <em>in vivo</em> processing of DSBs, we followed the degradation of radioactively-labeled chromosomal DNA of γ-irradiated <em>E. coli</em>. DSBs in that bacterium are repaired by the RecBCD enzyme, which upon binding to a (nearly blunt) dsDNA end unwinds the DNA duplex and degrades both of the unwound strands. Only after interaction with the χ sequence does the enzyme starts a resection process, meaning that it continues its degradation of the 5′ strand while ceasing trimming of the 3′ strand. Therefore, DNA degradation is an essential and indivisible part of DSB processing in <em>E. coli</em> (there is no situation in which RecBCD activity on DNA is nuclease-free), and by assessing DNA degradation one can indeed get an insight into DSB processing. RecBCD enzyme is the strongest DNase in <em>E. coli</em> (<a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Kuzminov 1999</a>), yet its nuclease activity is augmented in bacteria lacking the RecA protein, wherein it becomes unregulated (“reckless”), leading to complete degradation of a chromosome (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Capaldo and Barbour 1975</a>; <a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">Skarstad and Boye 1993</a>). “Reckless” degradation in <em>recA</em> null mutants is attributed to either impaired χ regulation of the RecBCD enzyme (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Kuzminov <em>et al.</em> 1994</a>; <a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Kuzminov and Stahl 1997</a>) or to a lack of RecA protection of the frayed ends of a processed DNA molecule (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Dabert <em>et al.</em> 1992</a>). We have shown here that recruitment of RecA onto ssDNA inhibits degradation at DSBs in wild-type, <em>recB</em>, and <em>recD</em> genetic backgrounds, indicating that the RecA protection of DNA from degradation is a general phenomenon in <em>E. coli</em>. Since there is no χ activity in the <em>recB</em> and <em>recD</em> mutants, and yet degradation of their DNA is still inhibited by RecA, we infer that the physical protection by RecA binding on (3′-terminated) ss overhangs is the main mechanism for RecA inhibition of DNA degradation. A recent study indicated that a RecA-ssDNA complex is resistant to degradation by nucleases (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Kohiyama <em>et al.</em> 2013</a>). Analogously, a human RecA ortholog, RAD51 recombinase, prevents excessive DNA nucleolytic degradation in UV-irradiated human cells (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Vallerga <em>et al.</em> 2015</a>), indicating conservation of the inhibitory role of recombinase proteins in DNA degradation, thus signifying its importance.</p><p id="p-51">However, our results do not exclude the possibility that RecBCD’s χ activity is indeed impaired in the <em>recA</em> mutant, nor do they determine the possible contribution of each of the two mechanisms to DNA protection in that mutant.</p><p id="p-52">Interestingly, our results show that the ability of the RecA protein to protect DNA from degradation differs from its role in DSB repair. The <em>recA730 lexA3</em> mutant has greatly suppressed DNA degradation compared with a <em>lexA3</em> mutant [both contain about the same concentration of RecA(E38K) protein], while having similar γ-survival. This phenotype is not unique as the <em>recB</em> mutant has it too; it also has RecA-inhibited DNA degradation and extremely poor survival. Thus, our results suggest that the RecA role in DNA degradation regulation is less challenging than its role in DSB repair; the former is about preventing 3′–5′ ssExos from degrading the 3′ tail (for which, binding of a couple of RecA molecules would likely suffice), while the latter requires production of a functional RecA nucleofilament. Furthermore, <em>recB</em> and <em>recD</em> null mutants share poor DNA degradation (suppressed by RecA), while the former is deficient for DSB repair and the latter is proficient. Also, <em>recA730 lexA3</em> and <em>recA<sub>o</sub>281 lexA3</em> mutants have about the same amount of DNA degradation, while their γ-survivals greatly differ.</p><p id="p-53">However, we revealed two notable exceptions in the RecA-imposed control of DNA degradation, which enabled further insight into the mechanism of DSB processing regulation in <em>E. coli</em>. Namely, inactivation of four ssExos (ExoI, ExoVII, ExoX, and SbcCD) that degrade a 3′ tail (<a id="xref-ref-33-3" class="xref-bibr" href="#ref-33">Lovett 2011</a>) greatly reduced DNA degradation in both RecA<sup>+</sup> and RecA<sup>−</sup> bacteria, making them an ExoV<sup>−</sup> phenocopy, even though the RecBCD enzyme is active in these cells and able to degrade DNA (as it prevented proliferation of the T4 <em>2</em> phage, whose genome is blunt-ended). Since RecBCD poorly degrades fragmented chromosomes in cells lacking 3′–5′ ssExos while retaining ExoV activity, we infer that the enzyme is affected in binding to DNA, <em>i.e.</em>, creation of blunt DNA ends is suppressed. A previous study showed that a strain lacking three 3′–5′ ssExos (ExoI, ExoVII, and SbcCD) retains nearly wt capacity for DNA repair, whereas its <em>recA</em> derivative had inhibited DNA degradation (ExoV<sup>−</sup>), thus indicating that these ssExos are not required for blunting the initial irradiation-produced dsDNA ends although being critical for “reckless” DNA degradation (<a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Repar <em>et al.</em> 2013</a>). Because 3′–5′ ssExos inactivation is epistatic to RecA deficiency for DNA degradation, we conclude that RecA inhibits DNA degradation by preventing trimming of 3′-terminated overhangs by 3′–5′ ssExos; hence, when these ssExos are absent, RecA becomes dispensable. Therefore, in cells lacking 3′–5′ ssExos, DSB processing is inhibited by preserved 3′ overhangs, irrespective of RecA protein (<a id="xref-fig-8-1" class="xref-fig" href="#F8">Figure 8A</a>). Conversely, RecA deficiency is epistatic to 3′–5′ ssExos deficiency for DNA repair, thus additionally emphasizing differences in roles of the RecA protein in DNA repair and DNA degradation in <em>E. coli</em>.</p><div id="F8" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/9/3091/F8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Processing of a dsDNA end in several E. coli genetic backgrounds. (A) In wt bacteria, dsDNA processing is initiated by binding of RecBCD enzyme onto it, which unwinds the DNA duplex while degrading both unwound strands. Upon interaction with the &#x3C7; site, the enzyme stops the 3&#x2032; strand degradation, while continuing DNA unwinding and degradation of the 5&#x2032; strand. &#x3C7;-modified RecBCD also facilitates RecA loading onto a post-&#x3C7; 3&#x2032; overhang (a). In the recA null mutant, RecBCD degrades both unwound strands before eventually being released from DNA, which might be independent of &#x3C7;, leaving a protruding 3&#x2032; tail. The 3&#x2032; tail prevents subsequent reloading of RecBCD onto a processed dsDNA end unless/until 3&#x2032;&#x2013;5&#x2032; ssExos degrade it (b). Since both RecA+ and RecA&#x2212; cells have similar, low DNA degradation when 3&#x2032;&#x2013;5&#x2032; ssExos are inactive [blocked (b) path], these exonucleases apparently act on a post-&#x3C7; 3&#x2032; tail in wt cells [hence path (b) may apply to wt cells too]. (B) In recA derivatives of recB and recD mutants, dsDNA end processing starts with RecQ and RecBC helicase unwinding of the DNA duplex, respectively. The unwound 3&#x2032;- and 5&#x2032;-terminated strands are degraded by 3&#x2032;&#x2013;5&#x2032; and 5&#x2032;&#x2013;3&#x2032; ssExos, respectively. The degradation of the 5&#x2032; strand is apparently faster than that of its complementary 3&#x2032; strand (otherwise recD mutant would not be HR proficient) and thus the latter remains after helicase detachment from DNA. The protruding 3&#x2032; tail is then degraded by 3&#x2032;&#x2013;5&#x2032; ssExos (a), therefore enabling creation of a (nearly) blunt DNA end and hence repeated loading of RecQ or RecBC helicase. In RecA+ variants of recB and recD, RecA binds to the 3&#x2032; tail (b) and prevents RecQ reloading onto the DNA end in the recB mutant, while blocking its trimming and recreation of the blunt DNA end in the recD mutant and hence inhibits RecBC reloading. dsDNA, double-stranded DNA; ssExos, exonucleases that degrade a 3&#x2032; tail; wt, wild-type." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-894880217" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Processing of a dsDNA end in several &lt;em&gt;E. coli&lt;/em&gt; genetic backgrounds. (A) In wt bacteria, dsDNA processing is initiated by binding of RecBCD enzyme onto it, which unwinds the DNA duplex while degrading both unwound strands. Upon interaction with the &#x3C7; site, the enzyme stops the 3&#x2032; strand degradation, while continuing DNA unwinding and degradation of the 5&#x2032; strand. &#x3C7;-modified RecBCD also facilitates RecA loading onto a post-&#x3C7; 3&#x2032; overhang (a). In the &lt;em&gt;recA&lt;/em&gt; null mutant, RecBCD degrades both unwound strands before eventually being released from DNA, which might be independent of &#x3C7;, leaving a protruding 3&#x2032; tail. The 3&#x2032; tail prevents subsequent reloading of RecBCD onto a processed dsDNA end unless/until 3&#x2032;&#x2013;5&#x2032; ssExos degrade it (b). Since both RecA&lt;sup&gt;+&lt;/sup&gt; and RecA&lt;sup&gt;&#x2212;&lt;/sup&gt; cells have similar, low DNA degradation when 3&#x2032;&#x2013;5&#x2032; ssExos are inactive [blocked (b) path], these exonucleases apparently act on a post-&#x3C7; 3&#x2032; tail in wt cells [hence path (b) may apply to wt cells too]. (B) In &lt;em&gt;recA&lt;/em&gt; derivatives of &lt;em&gt;recB&lt;/em&gt; and &lt;em&gt;recD&lt;/em&gt; mutants, dsDNA end processing starts with RecQ and RecBC helicase unwinding of the DNA duplex, respectively. The unwound 3&#x2032;- and 5&#x2032;-terminated strands are degraded by 3&#x2032;&#x2013;5&#x2032; and 5&#x2032;&#x2013;3&#x2032; ssExos, respectively. The degradation of the 5&#x2032; strand is apparently faster than that of its complementary 3&#x2032; strand (otherwise &lt;em&gt;recD&lt;/em&gt; mutant would not be HR proficient) and thus the latter remains after helicase detachment from DNA. The protruding 3&#x2032; tail is then degraded by 3&#x2032;&#x2013;5&#x2032; ssExos (a), therefore enabling creation of a (nearly) blunt DNA end and hence repeated loading of RecQ or RecBC helicase. In RecA&lt;sup&gt;+&lt;/sup&gt; variants of &lt;em&gt;recB&lt;/em&gt; and &lt;em&gt;recD&lt;/em&gt;, RecA binds to the 3&#x2032; tail (b) and prevents RecQ reloading onto the DNA end in the &lt;em&gt;recB&lt;/em&gt; mutant, while blocking its trimming and recreation of the blunt DNA end in the &lt;em&gt;recD&lt;/em&gt; mutant and hence inhibits RecBC reloading. dsDNA, double-stranded DNA; ssExos, exonucleases that degrade a 3&#x2032; tail; wt, wild-type.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 8" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/9/3091/F8.medium.gif" width="440" height="314"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 8" src="https://www.g3journal.org/content/ggg/7/9/3091/F8.medium.gif" width="440" height="314"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/9/3091/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 8" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/9/3091/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/486506" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 8</span> <p id="p-54" class="first-child">Processing of a dsDNA end in several <em>E. coli</em> genetic backgrounds. (A) In wt bacteria, dsDNA processing is initiated by binding of RecBCD enzyme onto it, which unwinds the DNA duplex while degrading both unwound strands. Upon interaction with the χ site, the enzyme stops the 3′ strand degradation, while continuing DNA unwinding and degradation of the 5′ strand. χ-modified RecBCD also facilitates RecA loading onto a post-χ 3′ overhang (a). In the <em>recA</em> null mutant, RecBCD degrades both unwound strands before eventually being released from DNA, which might be independent of χ, leaving a protruding 3′ tail. The 3′ tail prevents subsequent reloading of RecBCD onto a processed dsDNA end unless/until 3′–5′ ssExos degrade it (b). Since both RecA<sup>+</sup> and RecA<sup>−</sup> cells have similar, low DNA degradation when 3′–5′ ssExos are inactive [blocked (b) path], these exonucleases apparently act on a post-χ 3′ tail in wt cells [hence path (b) may apply to wt cells too]. (B) In <em>recA</em> derivatives of <em>recB</em> and <em>recD</em> mutants, dsDNA end processing starts with RecQ and RecBC helicase unwinding of the DNA duplex, respectively. The unwound 3′- and 5′-terminated strands are degraded by 3′–5′ and 5′–3′ ssExos, respectively. The degradation of the 5′ strand is apparently faster than that of its complementary 3′ strand (otherwise <em>recD</em> mutant would not be HR proficient) and thus the latter remains after helicase detachment from DNA. The protruding 3′ tail is then degraded by 3′–5′ ssExos (a), therefore enabling creation of a (nearly) blunt DNA end and hence repeated loading of RecQ or RecBC helicase. In RecA<sup>+</sup> variants of <em>recB</em> and <em>recD</em>, RecA binds to the 3′ tail (b) and prevents RecQ reloading onto the DNA end in the <em>recB</em> mutant, while blocking its trimming and recreation of the blunt DNA end in the <em>recD</em> mutant and hence inhibits RecBC reloading. dsDNA, double-stranded DNA; ssExos, exonucleases that degrade a 3′ tail; wt, wild-type.</p><div class="sb-div caption-clear"></div></div></div><p id="p-55">Another situation where DNA degradation is unaffected by RecA concerns the <em>recB1080</em> mutant, whose <em>recA</em> derivative had about the same, low level of DNA degradation as its parental RecA<sup>+</sup> strain. A combination of fast and processive helicase activity and lack of nuclease and RecA loading activities of RecB<sup>1080</sup>CD enzyme likely results in long, RecA-free overhangs in the <em>recB1080</em> mutant (as discussed, <a id="xref-ref-21-3" class="xref-bibr" href="#ref-21">Ivanković and Đermić 2012</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Đermić 2015</a>), analogously to what was observed <em>in vitro</em> (<a id="xref-ref-55-6" class="xref-bibr" href="#ref-55">Yu <em>et al.</em> 1998</a>; <a id="xref-ref-3-5" class="xref-bibr" href="#ref-3">Anderson <em>et al.</em> 1999</a>). In this mutant, RecA loading is achieved by RecFOR proteins, which perform it from an ss-ds DNA junction (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Morimatsu and Kowalczykowski 2003</a>), <em>i.e.</em>, from the 5′-end of that tens of kb-long 3′-terminated tail (as discussed, <a id="xref-ref-21-4" class="xref-bibr" href="#ref-21">Ivanković and Đermić 2012</a>). Therefore, ssExos of 3′–5′ polarity likely need to digest very long ss stretches in order to reach a growing RecA nucleofilament. This is a heavy task because that ss region is overly reactive; it may get engaged in inter- and intramolecular transactions, resulting in illegitimate recombination intermediates (<a id="xref-ref-21-5" class="xref-bibr" href="#ref-21">Ivanković and Đermić 2012</a>) and ds secondary structures, as well as ds products of reannealing with the complementary 5′ tail, respectively. All of them may inhibit the 3′–5′ ssExos, hence increasing the longevity of the 3′-terminated tail. Furthermore, ExoI, the most potent 3′–5′ ssExo (<a id="xref-ref-33-4" class="xref-bibr" href="#ref-33">Lovett 2011</a>), degrades the 3′ tail with an ∼275 nt s<sup>−1</sup> rate (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Brody <em>et al.</em> 1986</a>), which is around four- to eightfold slower than RecBCD helicase (and also less processive) (see below). Therefore, excessive length of the unwound 3′ strands and the mechanism of RecA loading onto them in the <em>recB1080</em> mutant seemingly protects them from degradation by ssExos, thus making RecA protection dispensable. In fact, nucleolytic degradation of 3′ overhangs by 3′–5′ ssExos is essential for survival of the <em>recB1080</em> mutant (<a id="xref-ref-22-3" class="xref-bibr" href="#ref-22">Ivanković <em>et al.</em> 2017</a>).</p><p id="p-56">Since 3′ tail preservation is common to all the aforementioned situations, we conclude that 3′ overhangs emanating from DSBs are crucial in limiting the extent of DSB processing in <em>E. coli</em>. How do 3′ tails regulate DSB processing? Our results reveal a gradient in intensity of DNA degradation in the <em>recA</em> derivatives of wild-type, <em>recD</em>, and <em>recB</em> strains (∼95, ∼40, and ∼20% degraded DNA, respectively), which unsurprisingly correlates with the activities of their respective DSB processing helicases RecBCD, RecBC, and RecQ. The RecBCD helicase/nuclease processes at least 30 kb of DNA at ∼1000–2000 bp s<sup>−1</sup> <em>in vitro</em> (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bianco <em>et al.</em> 2001</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Dillingham <em>et al.</em> 2005</a>), the RecBC helicase is about fourfold slower and less processive than RecBCD (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Korangy and Julin 1993</a>, <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">1994</a>), and the RecQ helicase is even slower (∼2 bp s<sup>−1</sup>, <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Xu <em>et al.</em> 2003</a>) and less processive than RecBC. However, this proportionality of DNA degradation with the potency of the DSB-processing helicase is disrupted in the <em>recB1080</em> mutant, whose RecB<sup>1080</sup>CD enzyme is equally as powerful a helicase as RecBCD (<a id="xref-ref-3-6" class="xref-bibr" href="#ref-3">Anderson <em>et al.</em> 1999</a>); and yet, paradoxically, DNA degradation in its <em>recA</em> derivative is very weak, not stronger than that in the parental RecA<sup>+</sup> strain, and certainly weaker than those observed in the <em>recA recB</em> and <em>recA recD</em> mutants. Extensive unwound tails produced in <em>recB1080</em> cells by the powerful RecB<sup>1080</sup>CD helicase may certainly prevent its reloading onto dsDNA ends because RecBCD and its variants (RecBC and RecB<sup>1080</sup>CD) bind exclusively onto a (nearly) blunt dsDNA end (<a id="xref-ref-18-3" class="xref-bibr" href="#ref-18">Dillingham and Kowalczykowski 2008</a>). On the other hand, continuous DNA degradation for &gt; 2 hr, resulting in a 20–40% degraded genome (1–2 Mbp) in <em>recA</em> derivatives of <em>recB</em> and <em>recD</em> mutants (<a id="xref-fig-8-2" class="xref-fig" href="#F8">Figure 8B</a>) reflects perpetual processing of DSBs in those bacteria, which is possible only if their slow and poorly processive helicases (RecQ and RecBC) are repeatedly loaded onto the processed DNA ends. As a consequence, DSB processing catalyzed by repeated reloading of a weak helicase, such as RecQ or RecBC, turns out to be more vigorous than that accomplished by a single-round activity of the powerful helicase RecB<sup>1080</sup>CD. Consequently, we infer that RecA prevents reloading of RecQ and RecBC helicases in <em>recB</em> and <em>recD</em> mutants, respectively. Inactivation of 3′–5′ ssExos in the <em>recD</em> mutant alleviates the RecA inhibition of DNA degradation, indicating that RecA inhibits DNA degradation by protecting 3′ tails from being degraded by 3′–5′ ssExos. Preserved 3′ tails in turn inhibit RecBC reloading onto the processed DNA end. Lack of DNA degradation in the <em>recB</em> mutant indicates that an optimal substrate for the RecQ binding, a dsDNA end with a 3′ protrusion (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Morimatsu and Kowalczykowski 2014</a>), becomes inaccessible to RecQ when the tail is bound by RecA. Furthermore, our results show that 3′–5′ ssExos enable DNA degradation in wild-type cells, suggesting that RecBCD-catalyzed DSB processing is achieved through repeated reloading of the enzyme, which depends on the removal of the 3′ tails. This indicates that, in wild-type <em>E. coli</em>, 3′–5′ ssExos regularly trim RecBCD-produced post-χ 3′ tails (<a id="xref-fig-8-3" class="xref-fig" href="#F8">Figure 8A</a>).</p><p id="p-57">However, there is an alternative explanation for the stimulatory effect of 3′–5′ ssExos on DNA degradation. Namely, if RecBCD would release long, acid-precipitable oligonucleotides while processing the DNA duplex (since it has endonucleolytic rather than exonucleolytic activity), 3′–5′ ssExos may be actively trimming those oligonucleotides, thus making them acid-soluble. In this way, low acid-soluble DNA content observed in irradiated 3′–5′ ssExos-deficient cells would mask ongoing DNA degradation in them. However, this hypothesis may be ruled out since both the un- and UV-irradiated ExoI<sup>−</sup> ExoVII<sup>−</sup> SbcCD<sup>−</sup> RecA<sup>−</sup> mutants have preserved genomic DNA, as assessed by DAPI staining and genome restriction (<a id="xref-ref-39-3" class="xref-bibr" href="#ref-39">Repar <em>et al.</em> 2013</a>), suggesting a lack of RecBCD-catalyzed “reckless” DNA degradation.</p><p id="p-58">We show here that DNA degradation in <em>E. coli</em> is inhibited by either DSB-processing helicase inactivation (<em>e.g.</em>, by <em>recB</em> mutation in otherwise wild-type cells and by RecQ inactivation in the <em>recB recA</em> mutant), or by preservation of 3′ tails, again leading to the inability of these helicases to reload on the processed dsDNA ends. Our results therefore indicate that repeated helicase loading is the main determinant of the extent of DSB processing, with 3′ overhang metabolism being the crucial factor in helicase reloading. We describe three ways by which the availability of dsDNA ends to a helicase is controlled by 3′ tails: (i) RecA polymerization onto 3′ tails either directly inhibits RecQ loading onto them in the <em>recB</em> null mutant or inhibits their degradation by the 3′–5′ ssExos, thus preventing creation of blunt dsDNA ends and, consequently, reloading of RecBC(D) onto them; (ii) the appearance of blunt dsDNA ends is prevented by inactivation of the 3′–5′ ssExos, hence making shielding RecA binding onto such a protected 3′ overhang dispensable; and (iii) in the <em>recB1080</em> mutant, lengthy DSB-derived 3′ overhangs are protected from 3′–5′ ssExos in a RecA-independent manner (instead, this is likely achieved by their involvement in transactions that produce ds regions in them), hence preventing recreation of blunt dsDNA ends. Similarly, excessively long 3′ overhangs produced during resection inhibit meiotic DSB repair in eukaryotes (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Johnson <em>et al.</em> 2007</a>).</p><p id="p-59">Our collective results indicate that resection of a dsDNA end in <em>E. coli</em> proceeds until a 3′ tail of sufficient length and stability is formed, which then inhibits further end processing by preventing reloading of a DSB-processing helicase. Factors that facilitate 3′ tails’ longevity involve: (i) 3′ tails protection from degradation by 3′–5′ ssExos, by either “insulating” them with the RecA protein or by inactivation of these 3′–5′ ssExos, and (ii) their excessive length.</p><p id="p-60">The 3′ tail regulation of DSB processing that we describe here is reminiscent of DNA end-resection in eukaryotes, with a short 3′ overhang produced during initial DSB resection by the Rad50 and Mre11 nuclease (orthologs of SbcCD) directing a resected end toward HR and microhomology-mediated end joining pathways and away from the nonhomologous end joining repair pathway (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Truong <em>et al.</em> 2013</a>). Similarly, the length of (and RecA binding to) the 3′ overhangs created during DNA end resection determines the equilibrium of HR and illegitimate recombination pathways in <em>E. coli</em> (<a id="xref-ref-21-6" class="xref-bibr" href="#ref-21">Ivanković and Đermić 2012</a>). Furthermore, SbcCD and its eukaryotic orthologs are analogously required to enable (re)loading of the main DSB processing machines [bacterial RecBCD and eukaryotic EXOI (ExoI)/BLM DNA2 (Sgs1 Dna2)] that perform long-range dsDNA end processing (as discussed recently, <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Đermić 2015</a>).</p><p id="p-61">Hence, one can note a common regulatory mechanism for DSB processing, with a 3′ tail produced during end resection acting as the main supervisory element by imposing a negative feedback loop on subsequent processing reactions. This mechanism ensures that processing of dsDNA ends, continues until stable and utilizable 3′ tails are produced that enable the efficient repair of DSBs in both bacteria and eukaryotes.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-62">We are grateful to Amir Dubravić, Mary Sopta, and Nikola Paić for their help with manuscript preparation. This study was funded by the Croatian Science Foundation, project HRZZ-IP-11-2013-2978.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-3"><p id="p-63"><em>Communicating editor: R. A. Sclafani</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2017-05-19"><span class="received-label">Received </span>May 19, 2017.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2017-07-11"><span class="accepted-label">Accepted </span>July 11, 2017.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2017 Đermić <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3091.1" data-doi="10.1101/gad.11.5.571"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Anderson</span>  <span class="cit-name-given-names">D. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kowalczykowski</span>  <span class="cit-name-given-names">S. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span><span class="cit-comment">a</span> <span class="cit-article-title">The recombination hot spot, Chi, is a regulatory element that switches the polarity of DNA degradation by the RecBCD enzyme.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">571</span>–<span class="cit-lpage">581</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DAnderson%26rft.auinit1%253DD%2BG%26rft.volume%253D11%26rft.issue%253D5%26rft.spage%253D571%26rft.epage%253D581%26rft.atitle%253DThe%2Brecombination%2Bhot%2Bspot%2Bchi%2Bis%2Ba%2Bregulatory%2Belement%2Bthat%2Bswitches%2Bthe%2Bpolarity%2Bof%2BDNA%2Bdegradation%2Bby%2Bthe%2BRecBCD%2Benzyme.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.11.5.571%26rft_id%253Dinfo%253Apmid%252F9119222%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6ODoiMTEvNS81NzEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzkvMzA5MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3091.2" data-doi="10.1016/S0092-8674(00)80315-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Anderson</span>  <span class="cit-name-given-names">D. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kowalczykowski</span>  <span class="cit-name-given-names">S. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span><span class="cit-comment">b</span> <span class="cit-article-title">The translocating RecBCD enzyme stimulates recombination by directing RecA protein onto ssDNA in a χ-regulated manner.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">90</span>: <span class="cit-fpage">77</span>–<span class="cit-lpage">86</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DAnderson%26rft.auinit1%253DD.%2BG.%26rft.volume%253D90%26rft.issue%253D1%26rft.spage%253D77%26rft.epage%253D86%26rft.atitle%253DThe%2Btranslocating%2BRecBCD%2Benzyme%2Bstimulates%2Brecombination%2Bby%2Bdirecting%2BRecA%2Bprotein%2Bonto%2BssDNA%2Bin%2Ba%2Bchi-regulated%2Bmanner.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0092-8674%252800%252980315-3%26rft_id%253Dinfo%253Apmid%252F9230304%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0092-8674(00)80315-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9230304&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3091.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1997XL36200010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3091.3" data-doi="10.1074/jbc.274.38.27139"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Anderson</span>  <span class="cit-name-given-names">D. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Churchill</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kowalczykowski</span>  <span class="cit-name-given-names">S. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">A single mutation, RecB<sub>D1080A</sub>, eliminates RecA protein loading but not Chi recognition by RecBCD enzyme.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">274</span>: <span class="cit-fpage">27139</span>–<span class="cit-lpage">27144</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBiological%2BChemistry%26rft.stitle%253DJ%2BBiol%2BChem%26rft.aulast%253DAnderson%26rft.auinit1%253DD.%2BG.%26rft.volume%253D274%26rft.issue%253D38%26rft.spage%253D27139%26rft.epage%253D27144%26rft.atitle%253DA%2BSingle%2BMutation%252C%2BRecBD1080A%252C%2BEliminates%2BRecA%2BProtein%2BLoading%2Bbut%2BNot%2BChi%2BRecognition%2Bby%2BRecBCD%2BEnzyme%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.274.38.27139%26rft_id%253Dinfo%253Apmid%252F10480929%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyNzQvMzgvMjcxMzkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzkvMzA5MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3091.4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bachmann</span>  <span class="cit-name-given-names">B. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1972</span> <span class="cit-article-title">Pedigrees of some mutant strains of <em>Escherichia coli</em> K-12.</span> <abbr class="cit-jnl-abbrev">Bacteriol. Rev.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">525</span>–<span class="cit-lpage">557</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMicrobiology%2Band%2BMolecular%2BBiology%2BReviews%26rft.stitle%253DMicrobiol.%2BMol.%2BBiol.%2BRev.%26rft.aulast%253DBachmann%26rft.auinit1%253DB%2BJ%26rft.volume%253D36%26rft.issue%253D4%26rft.spage%253D525%26rft.epage%253D557%26rft.atitle%253DPedigrees%2Bof%2Bsome%2Bmutant%2Bstrains%2Bof%2BEscherichia%2Bcoli%2BK-12.%26rft_id%253Dinfo%253Apmid%252F4568763%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo0OiJtbWJyIjtzOjU6InJlc2lkIjtzOjg6IjM2LzQvNTI1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy85LzMwOTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3091.5" data-doi="10.1038/35053131"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bianco</span>  <span class="cit-name-given-names">P. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brewer</span>  <span class="cit-name-given-names">L. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Corzett</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Balhorn</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yeh</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Processive translocation and DNA unwinding by individual RecBCD enzyme molecules.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">409</span>: <span class="cit-fpage">374</span>–<span class="cit-lpage">378</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DBianco%26rft.auinit1%253DP.%2BR.%26rft.volume%253D409%26rft.issue%253D6818%26rft.spage%253D374%26rft.epage%253D378%26rft.atitle%253DProcessive%2Btranslocation%2Band%2BDNA%2Bunwinding%2Bby%2Bindividual%2BRecBCD%2Benzyme%2Bmolecules.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F35053131%26rft_id%253Dinfo%253Apmid%252F11201750%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/35053131&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11201750&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3091.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3091.6" data-doi="10.1080/09553007914551061"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bresler</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Noskin</span>  <span class="cit-name-given-names">L. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuzovleva</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Noskina</span>  <span class="cit-name-given-names">I. G.</span></span></li></ol><cite>, <span class="cit-pub-date">1979</span> <span class="cit-article-title">Nature of the damage to <em>Escherichia coli</em> DNA induced by gamma radiation.</span> <abbr class="cit-jnl-abbrev">Int. J. Radiat. Biol.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">289</span>–<span class="cit-lpage">300</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInt.%2BJ.%2BRadiat.%2BBiol.%26rft.volume%253D36%26rft.spage%253D289%26rft_id%253Dinfo%253Adoi%252F10.1080%252F09553007914551061%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/09553007914551061&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3091.7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brody</span>  <span class="cit-name-given-names">R. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doherty</span>  <span class="cit-name-given-names">K. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zimmerman</span>  <span class="cit-name-given-names">P. D.</span></span></li></ol><cite>, <span class="cit-pub-date">1986</span> <span class="cit-article-title">Processivity and kinetics of the reaction of exonuclease I from Escherichia coli with polydeoxyribonucleotides.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">261</span>: <span class="cit-fpage">7136</span>–<span class="cit-lpage">7143</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBiological%2BChemistry%26rft.stitle%253DJ%2BBiol%2BChem%26rft.aulast%253DBrody%26rft.auinit1%253DR.%2BS.%26rft.volume%253D261%26rft.issue%253D16%26rft.spage%253D7136%26rft.epage%253D7143%26rft.atitle%253DProcessivity%2Band%2Bkinetics%2Bof%2Bthe%2Breaction%2Bof%2Bexonuclease%2BI%2Bfrom%2BEscherichia%2Bcoli%2Bwith%2Bpolydeoxyribonucleotides%26rft_id%253Dinfo%253Apmid%252F3519606%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjExOiIyNjEvMTYvNzEzNiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvOS8zMDkxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3091.8" data-doi="10.1016/0022-2836(75)90371-X"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Capaldo</span>  <span class="cit-name-given-names">F. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barbour</span>  <span class="cit-name-given-names">S. D.</span></span></li></ol><cite>, <span class="cit-pub-date">1975</span> <span class="cit-article-title">DNA content, synthesis and integrity in dividing and non-dividing cells of rec- strains of <em>Escherichia coli</em> K12.</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">53</span>–<span class="cit-lpage">66</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DCapaldo%26rft.auinit1%253DF.%2BN.%26rft.volume%253D91%26rft.issue%253D1%26rft.spage%253D53%26rft.epage%253D66%26rft.atitle%253DDNA%2Bcontent%252C%2Bsynthesis%2Band%2Bintegrity%2Bin%2Bdividing%2Band%2Bnon-dividing%2Bcells%2Bof%2Brec-%2Bstrains%2Bof%2BEscherichia%2Bcoli%2BK12.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0022-2836%252875%252990371-X%26rft_id%253Dinfo%253Apmid%252F1102696%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0022-2836(75)90371-X&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1102696&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3091.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3091.9" data-doi="10.1101/gad.13.7.901"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Churchill</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anderson</span>  <span class="cit-name-given-names">D. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kowalczykowski</span>  <span class="cit-name-given-names">S. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">The RecBC enzyme loads RecA protein onto ssDNA asymmetrically and independently of Chi, resulting in constitutive recombination activation.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">901</span>–<span class="cit-lpage">911</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DChurchill%26rft.auinit1%253DJ.%2BJ.%26rft.volume%253D13%26rft.issue%253D7%26rft.spage%253D901%26rft.epage%253D911%26rft.atitle%253DThe%2BRecBC%2Benzyme%2Bloads%2BRecA%2Bprotein%2Bonto%2BssDNA%2Basymmetrically%2Band%2Bindependently%2Bof%2Bchi%2B%252C%2Bresulting%2Bin%2Bconstitutive%2Brecombination%2Bactivation%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.13.7.901%26rft_id%253Dinfo%253Apmid%252F10197989%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6ODoiMTMvNy85MDEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzkvMzA5MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3091.10" data-doi="10.1016/S0300-9084(82)80108-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Clark</span>  <span class="cit-name-given-names">A. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1982</span> <span class="cit-article-title"><em>recA</em> operator mutations and their usefulness.</span> <abbr class="cit-jnl-abbrev">Biochimie</abbr> <span class="cit-vol">64</span>: <span class="cit-fpage">669</span>–<span class="cit-lpage">675</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochimie%26rft.volume%253D64%26rft.spage%253D669%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0300-9084%252882%252980108-9%26rft_id%253Dinfo%253Apmid%252F6291636%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0300-9084(82)80108-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=6291636&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3091.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3091.11" data-doi="10.1073/pnas.89.24.12073"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dabert</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ehrlich</span>  <span class="cit-name-given-names">S. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gruss</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">χ sequence protects against RecBCD degradation of DNA <em>in vivo</em>.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">89</span>: <span class="cit-fpage">12073</span>–<span class="cit-lpage">12077</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DDabert%26rft.auinit1%253DP.%26rft.volume%253D89%26rft.issue%253D24%26rft.spage%253D12073%26rft.epage%253D12077%26rft.atitle%253DChi%2Bsequence%2Bprotects%2Bagainst%2BRecBCD%2Bdegradation%2Bof%2BDNA%2Bin%2Bvivo.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.89.24.12073%26rft_id%253Dinfo%253Apmid%252F1465442%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiODkvMjQvMTIwNzMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzkvMzA5MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3091.12" data-doi="10.1038/nrmicro2073"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Daly</span>  <span class="cit-name-given-names">M. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A new perspective on radiation resistance based on <em>Deinococcus radiodurans</em>.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Microbiol.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">237</span>–<span class="cit-lpage">245</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BMicrobiology%26rft.stitle%253DNat%2BRev%2BMicrobiol%26rft.aulast%253DDaly%26rft.auinit1%253DM.%2BJ.%26rft.volume%253D7%26rft.issue%253D3%26rft.spage%253D237%26rft.epage%253D245%26rft.atitle%253DA%2Bnew%2Bperspective%2Bon%2Bradiation%2Bresistance%2Bbased%2Bon%2BDeinococcus%2Bradiodurans.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrmicro2073%26rft_id%253Dinfo%253Apmid%252F19172147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrmicro2073&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19172147&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3091.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000263361000015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3091.13" data-doi="10.1534/genetics.105.052076"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Đermić</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Functions of multiple exonucleases are essential for cell viability, DNA repair and homologous recombination in <em>recD</em> mutants of <em>Escherichia coli</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">172</span>: <span class="cit-fpage">2057</span>–<span class="cit-lpage">2069</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.105.052076%26rft_id%253Dinfo%253Apmid%252F16452142%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3Mi80LzIwNTciO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzkvMzA5MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3091.14"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Đermić</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Double-strand break repair mechanisms in <em>Escherichia coli:</em> recent insights.</span> <abbr class="cit-jnl-abbrev">Adv. Genomics Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">35</span>–<span class="cit-lpage">42</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAdv.%2BGenomics%2BGenet.%26rft.volume%253D5%26rft.spage%253D35%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-2" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3091.15" data-doi="10.1016/j.resmic.2004.10.005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Đermić</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Halupecki</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zahradka</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Petranović</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">RecBCD enzyme overproduction impairs DNA repair and homologous recombination in <em>Escherichia coli</em>.</span> <abbr class="cit-jnl-abbrev">Res. Microbiol.</abbr> <span class="cit-vol">156</span>: <span class="cit-fpage">304</span>–<span class="cit-lpage">311</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DResearch%2Bin%2BMicrobiology%2B%2528Paris%2529%26rft.stitle%253DResearch%2Bin%2BMicrobiology%2B%2528Paris%2529%26rft.aulast%253DDermi%25C4%2587%26rft.auinit1%253DD.%26rft.volume%253D156%26rft.issue%253D3%26rft.spage%253D304%26rft.epage%253D311%26rft.atitle%253DRecBCD%2Benzyme%2Boverproduction%2Bimpairs%2BDNA%2Brepair%2Band%2Bhomologous%2Brecombination%2Bin%2BEscherichia%2Bcoli.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.resmic.2004.10.005%26rft_id%253Dinfo%253Apmid%252F15808933%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.resmic.2004.10.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15808933&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3091.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3091.16" data-doi="10.1534/genetics.106.060426"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Đermić</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zahradka</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Petranović</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span><span class="cit-comment">a</span> <span class="cit-article-title">Exonuclease requirements for recombination of λ-phage in <em>recD</em> mutants of <em>Escherichia coli</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">173</span>: <span class="cit-fpage">2399</span>–<span class="cit-lpage">2402</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.106.060426%26rft_id%253Dinfo%253Apmid%252F16702415%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3My80LzIzOTkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzkvMzA5MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3091.17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Đermić</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Đermić</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zahradka</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Petranović</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lerš</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span><span class="cit-comment">b</span> <span class="cit-article-title">Gamma-irradiated RecD overproducers become permanent <em>recB<sup>-</sup>/recC</em><sup>-</sup> phenocopies for extrachromosomal DNA processing due to prolonged titration of RecBCD enzyme on damaged <em>Escherichia coli</em> chromosome.</span> <abbr class="cit-jnl-abbrev">Biochimie</abbr> <span class="cit-vol">88</span>: <span class="cit-fpage">379</span>–<span class="cit-lpage">386</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochimie%26rft.volume%253D88%26rft.spage%253D379%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3091.18" data-doi="10.1128/MMBR.00020-08"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dillingham</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kowalczykowski</span>  <span class="cit-name-given-names">S. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">RecBCD enzyme and the repair of double-stranded DNA breaks.</span> <abbr class="cit-jnl-abbrev">Microbiol. Mol. Biol. Rev.</abbr> <span class="cit-vol">72</span>: <span class="cit-fpage">642</span>–<span class="cit-lpage">671</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMicrobiology%2Band%2BMolecular%2BBiology%2BReviews%26rft.stitle%253DMicrobiol.%2BMol.%2BBiol.%2BRev.%26rft.aulast%253DDillingham%26rft.auinit1%253DM.%2BS.%26rft.volume%253D72%26rft.issue%253D4%26rft.spage%253D642%26rft.epage%253D671%26rft.atitle%253DRecBCD%2BEnzyme%2Band%2Bthe%2BRepair%2Bof%2BDouble-Stranded%2BDNA%2BBreaks%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMMBR.00020-08%26rft_id%253Dinfo%253Apmid%252F19052323%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoibW1iciI7czo1OiJyZXNpZCI7czo4OiI3Mi80LzY0MiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvOS8zMDkxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3091.19" data-doi="10.1074/jbc.M505520200"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dillingham</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Webb</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kowalczykowski</span>  <span class="cit-name-given-names">S. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Bipolar DNA translocation contributes to highly processive DNA unwinding by RecBCD enzyme.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">280</span>: <span class="cit-fpage">37069</span>–<span class="cit-lpage">37077</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M505520200%26rft_id%253Dinfo%253Apmid%252F16041061%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyODAvNDQvMzcwNjkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzkvMzA5MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3091.20"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ivančić-Baće</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peharec</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moslavac</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Škrobot</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salaj-Šmic</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">RecFOR function is required for DNA repair and recombination in a RecA loading-deficient <em>recB</em> mutant of <em>Escherichia coli</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">163</span>: <span class="cit-fpage">485</span>–<span class="cit-lpage">494</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DIvancic-Bace%26rft.auinit1%253DI.%26rft.volume%253D163%26rft.issue%253D2%26rft.spage%253D485%26rft.epage%253D494%26rft.atitle%253DRecFOR%2BFunction%2BIs%2BRequired%2Bfor%2BDNA%2BRepair%2Band%2BRecombination%2Bin%2Ba%2BRecA%2BLoading-Deficient%2BrecB%2BMutant%2Bof%2BEscherichia%2Bcoli%26rft_id%253Dinfo%253Apmid%252F12618388%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTYzLzIvNDg1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy85LzMwOTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3091.21"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ivanković</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Đermić</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">DNA end resection controls the balance between homologous and illegitimate recombination in <em>Escherichia coli</em>.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e39030</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DIvankovic%26rft.auinit1%253DS.%26rft.volume%253D7%26rft.issue%253D6%26rft.spage%253De39030%26rft.epage%253De39030%26rft.atitle%253DDNA%2Bend%2Bresection%2Bcontrols%2Bthe%2Bbalance%2Bbetween%2Bhomologous%2Band%2Billegitimate%2Brecombination%2Bin%2BEscherichia%2Bcoli.%26rft_id%253Dinfo%253Apmid%252F22720024%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=22720024&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3091.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3091.22"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ivanković</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vujaklija</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Đermić</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Nucleolytic degradation of 3′-ending overhangs is essential for DNA-end resection in RecA-loading deficient <em>recB</em> mutants of <em>Escherichia coli</em>.</span> <abbr class="cit-jnl-abbrev">DNA Repair (Amst.)</abbr> <span class="cit-vol">57</span>: <span class="cit-fpage">56</span>–<span class="cit-lpage">65</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDNA%2BRepair%2B%2528Amst.%2529%26rft.volume%253D57%26rft.spage%253D56%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3091.23" data-doi="10.1046/j.1365-2958.2001.02573.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jockovich</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Myers</span>  <span class="cit-name-given-names">R. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Nuclease activity is essential for RecBCD recombination in <em>Escherichia coli</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Microbiol.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">949</span>–<span class="cit-lpage">962</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bmicrobiology%26rft.stitle%253DMol%2BMicrobiol%26rft.aulast%253DJockovich%26rft.auinit1%253DM.%2BE.%26rft.volume%253D41%26rft.issue%253D4%26rft.spage%253D949%26rft.epage%253D962%26rft.atitle%253DNuclease%2Bactivity%2Bis%2Bessential%2Bfor%2BRecBCD%2Brecombination%2Bin%2BEscherichia%2Bcoli.%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1365-2958.2001.02573.x%26rft_id%253Dinfo%253Apmid%252F11532156%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1365-2958.2001.02573.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11532156&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3091.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000170904400015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3091.24" data-doi="10.1371/journal.pgen.0030223"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Borde</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neale</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bishop-Bailey</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">North</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Excess single-stranded DNA inhibits meiotic double-strand break repair.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">e223</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DJohnson%26rft.auinit1%253DR.%26rft.volume%253D3%26rft.issue%253D11%26rft.spage%253De223%26rft.epage%253De223%26rft.atitle%253DExcess%2Bsingle-stranded%2BDNA%2Binhibits%2Bmeiotic%2Bdouble-strand%2Bbreak%2Brepair.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.0030223%26rft_id%253Dinfo%253Apmid%252F18081428%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.0030223&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18081428&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3091.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3091.25" data-doi="10.1016/j.jmb.2013.06.027"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kohiyama</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Contremoulins</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baudin</span>  <span class="cit-name-given-names">X.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Trashing of single-stranded DNA generated during processing of arrested replication fork in <em>E. coli</em>.</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">425</span>: <span class="cit-fpage">4837</span>–<span class="cit-lpage">4844</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BMol.%2BBiol.%26rft.volume%253D425%26rft.spage%253D4837%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jmb.2013.06.027%26rft_id%253Dinfo%253Apmid%252F23810902%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jmb.2013.06.027&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23810902&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3091.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3091.26" data-doi="10.1021/bi00069a024"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Korangy</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Julin</span>  <span class="cit-name-given-names">D. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Kinetics and processivity of ATP hydrolysis and DNA unwinding by the RecBC enzyme from <em>Escherichia coli</em>.</span> <abbr class="cit-jnl-abbrev">Biochemistry</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">4873</span>–<span class="cit-lpage">4880</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochemistry%26rft.volume%253D32%26rft.spage%253D4873%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fbi00069a024%26rft_id%253Dinfo%253Apmid%252F8387820%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1021/bi00069a024&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8387820&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3091.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1993LB64700024&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3091.27" data-doi="10.1021/bi00198a022"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Korangy</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Julin</span>  <span class="cit-name-given-names">D. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Efficiency of ATP hydrolysis and DNA unwinding by the RecBC enzyme from <em>Escherichia coli</em>.</span> <abbr class="cit-jnl-abbrev">Biochemistry</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">9552</span>–<span class="cit-lpage">9560</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochemistry%26rft.volume%253D33%26rft.spage%253D9552%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fbi00198a022%26rft_id%253Dinfo%253Apmid%252F8068630%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1021/bi00198a022&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8068630&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3091.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1994PC54500022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3091.28"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kuzminov</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Recombinational repair of DNA damage in <em>Escherichia coli</em> and bacteriophage λ.</span> <abbr class="cit-jnl-abbrev">Microbiol. Mol. Biol. Rev.</abbr> <span class="cit-vol">63</span>: <span class="cit-fpage">751</span>–<span class="cit-lpage">813</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMicrobiology%2Band%2BMolecular%2BBiology%2BReviews%26rft.stitle%253DMicrobiol.%2BMol.%2BBiol.%2BRev.%26rft.aulast%253DKuzminov%26rft.auinit1%253DA.%26rft.volume%253D63%26rft.issue%253D4%26rft.spage%253D751%26rft.epage%253D813%26rft.atitle%253DRecombinational%2BRepair%2Bof%2BDNA%2BDamage%2Bin%2BEscherichia%2Bcoli%2Band%2BBacteriophage%2Blambda%26rft_id%253Dinfo%253Apmid%252F10585965%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoibW1iciI7czo1OiJyZXNpZCI7czo4OiI2My80Lzc1MSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvOS8zMDkxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3091.29" data-doi="10.1128/jb.179.3.880-888.1997"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kuzminov</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stahl</span>  <span class="cit-name-given-names">F. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Stability of linear DNA in <em>recA</em> mutant <em>Escherichia coli</em> cells reflects ongoing chromosomal DNA degradation.</span> <abbr class="cit-jnl-abbrev">J. Bacteriol.</abbr> <span class="cit-vol">179</span>: <span class="cit-fpage">880</span>–<span class="cit-lpage">889</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBacteriology%26rft.stitle%253DJ.%2BBacteriol.%26rft.aulast%253DKuzminov%26rft.auinit1%253DA.%26rft.volume%253D179%26rft.issue%253D3%26rft.spage%253D880%26rft.epage%253D888%26rft.atitle%253DStability%2Bof%2Blinear%2BDNA%2Bin%2BrecA%2Bmutant%2BEscherichia%2Bcoli%2Bcells%2Breflects%2Bongoing%2Bchromosomal%2BDNA%2Bdegradation.%26rft_id%253Dinfo%253Adoi%252F10.1128%252Fjb.179.3.880-888.1997%26rft_id%253Dinfo%253Apmid%252F9006046%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6OToiMTc5LzMvODgwIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy85LzMwOTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3091.30"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kuzminov</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shabtach</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stahl</span>  <span class="cit-name-given-names">F. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">χ-sites in combination with RecA protein increase survival of linear DNA in <em>Escherichia coli</em> by inactivating exoV activity of RecBCD nuclease.</span> <abbr class="cit-jnl-abbrev">EMBO J.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">2764</span>–<span class="cit-lpage">2776</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEMBO%2BJ.%26rft.volume%253D13%26rft.spage%253D2764%26rft_id%253Dinfo%253Apmid%252F8026461%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=8026461&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3091.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1994NT76100004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3091.31"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lavery</span>  <span class="cit-name-given-names">P. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kowalczykowski</span>  <span class="cit-name-given-names">S. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">Biochemical basis of the constitutive repressor cleavage activity of recA730 protein. A comparison to recA441 and recA803 proteins.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">267</span>: <span class="cit-fpage">20648</span>–<span class="cit-lpage">20658</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBiological%2BChemistry%26rft.stitle%253DJ%2BBiol%2BChem%26rft.aulast%253DLavery%26rft.auinit1%253DP.%2BE.%26rft.volume%253D267%26rft.issue%253D29%26rft.spage%253D20648%26rft.epage%253D20658%26rft.atitle%253DBiochemical%2Bbasis%2Bof%2Bthe%2Bconstitutive%2Brepressor%2Bcleavage%2Bactivity%2Bof%2BrecA730%2Bprotein.%2BA%2Bcomparison%2Bto%2BrecA441%2Band%2BrecA803%2Bproteins%26rft_id%253Dinfo%253Apmid%252F1400384%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyNjcvMjkvMjA2NDgiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzkvMzA5MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3091.32" data-doi="10.1016/0300-9084(91)90108-D"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Little</span>  <span class="cit-name-given-names">J. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">Mechanism of specific LexA cleavage-autodigestion and the role of RecA coprotease.</span> <abbr class="cit-jnl-abbrev">Biochimie.</abbr> <span class="cit-vol">73</span>: <span class="cit-fpage">411</span>–<span class="cit-lpage">422</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochimie.%26rft.volume%253D73%26rft.spage%253D411%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0300-9084%252891%252990108-D%26rft_id%253Dinfo%253Apmid%252F1911941%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0300-9084(91)90108-D&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1911941&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3091.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1991FT39300010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3091.33"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lovett</span>  <span class="cit-name-given-names">S. T.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The DNA exonucleases of <em>Escherichia coli</em>.</span> <abbr class="cit-jnl-abbrev">EcoSal Plus</abbr> <span class="cit-vol">4</span>. Available at: .<a href="http://www.asmscience.org/content/journal/ecosalplus/10.1128/ecosalplus.4.4.7">http://www.asmscience.org/content/journal/ecosalplus/10.1128/ecosalplus.4.4.7</a></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-book" id="cit-7.9.3091.34"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">J. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-source"><em>A Short Course in Bacterial Genetics</em></span>. <span class="cit-publ-name">Cold Spring Harbor Laboratory Press</span>, <span class="cit-publ-loc">Cold Spring Harbor, NY</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3091.35"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Miranda</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuzminov</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Chromosomal lesion suppression and removal in <em>Escherichia coli</em> via linear DNA degradation.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">163</span>: <span class="cit-fpage">1255</span>–<span class="cit-lpage">1271</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMiranda%26rft.auinit1%253DA.%26rft.volume%253D163%26rft.issue%253D4%26rft.spage%253D1255%26rft.epage%253D1271%26rft.atitle%253DChromosomal%2BLesion%2BSuppression%2Band%2BRemoval%2Bin%2BEscherichia%2Bcoli%2Bvia%2BLinear%2BDNA%2BDegradation%26rft_id%253Dinfo%253Apmid%252F12702673%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2My80LzEyNTUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzkvMzA5MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3091.36" data-doi="10.1016/S1097-2765(03)00188-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Morimatsu</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kowalczykowski</span>  <span class="cit-name-given-names">S. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">RecFOR proteins load RecA protein onto gapped DNA to accelerate DNA strand exchange: a universal step of recombinational repair.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">1337</span>–<span class="cit-lpage">1347</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DMorimatsu%26rft.auinit1%253DK.%26rft.volume%253D11%26rft.issue%253D5%26rft.spage%253D1337%26rft.epage%253D1347%26rft.atitle%253DRecFOR%2Bproteins%2Bload%2BRecA%2Bprotein%2Bonto%2Bgapped%2BDNA%2Bto%2Baccelerate%2BDNA%2Bstrand%2Bexchange%253A%2Ba%2Buniversal%2Bstep%2Bof%2Brecombinational%2Brepair.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1097-2765%252803%252900188-6%26rft_id%253Dinfo%253Apmid%252F12769856%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S1097-2765(03)00188-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12769856&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3091.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000183139400022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3091.37" data-doi="10.1073/pnas.1420009111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Morimatsu</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kowalczykowski</span>  <span class="cit-name-given-names">S. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">RecQ helicase and RecJ nuclease provide complementary functions to resect DNA for homologous recombination.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">111</span>: <span class="cit-fpage">E5133</span>–<span class="cit-lpage">E5142</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1420009111%26rft_id%253Dinfo%253Apmid%252F25411316%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTExLzQ4L0U1MTMzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy85LzMwOTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3091.38" data-doi="10.1080/10409230802485358"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Persky</span>  <span class="cit-name-given-names">N. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lovett</span>  <span class="cit-name-given-names">S. T.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Mechanisms of recombination: lessons from <em>E. coli</em>.</span> <abbr class="cit-jnl-abbrev">Crit. Rev. Biochem. Mol. Biol.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">347</span>–<span class="cit-lpage">370</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCritical%2BReviews%2Bin%2BBiochemistry%2Band%2BMolecular%2BBiology%26rft.stitle%253DCRIT%2BREV%2BBIOCHEM%2BMOL%2BBIOL%26rft.aulast%253DPersky%26rft.auinit1%253DN.%2BS.%26rft.volume%253D43%26rft.issue%253D6%26rft.spage%253D347%26rft.epage%253D370%26rft.atitle%253DMechanisms%2Bof%2Brecombination%253A%2Blessons%2Bfrom%2BE.%2Bcoli.%26rft_id%253Dinfo%253Adoi%252F10.1080%252F10409230802485358%26rft_id%253Dinfo%253Apmid%252F19016098%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/10409230802485358&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19016098&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3091.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000261766600001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3091.39"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Repar</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Briški</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buljubašić</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zahradka</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zahradka</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Exonuclease VII is involved in “reckless” DNA degradation in UV-irradiated <em>Escherichia coli</em>.</span> <abbr class="cit-jnl-abbrev">Mutat. Res.</abbr> <span class="cit-vol">750</span>: <span class="cit-fpage">96</span>–<span class="cit-lpage">104</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMutat.%2BRes.%26rft.volume%253D750%26rft.spage%253D96%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3091.40" data-doi="10.1128/jb.174.16.5424-5429.1992"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rinken</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thoms</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wackernagel</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">Evidence that <em>recBC</em>-dependent degradation of duplex DNA in <em>Escherichia coli recD</em> mutants involves DNA unwinding.</span> <abbr class="cit-jnl-abbrev">J. Bacteriol.</abbr> <span class="cit-vol">174</span>: <span class="cit-fpage">5424</span>–<span class="cit-lpage">5429</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBacteriology%26rft.stitle%253DJ.%2BBacteriol.%26rft.aulast%253DRinken%26rft.auinit1%253DR.%26rft.volume%253D174%26rft.issue%253D16%26rft.spage%253D5424%26rft.epage%253D5429%26rft.atitle%253DEvidence%2Bthat%2BrecBC-dependent%2Bdegradation%2Bof%2Bduplex%2BDNA%2Bin%2BEscherichia%2Bcoli%2BrecD%2Bmutants%2Binvolves%2BDNA%2Bunwinding.%26rft_id%253Dinfo%253Adoi%252F10.1128%252Fjb.174.16.5424-5429.1992%26rft_id%253Dinfo%253Apmid%252F1322885%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6MTE6IjE3NC8xNi81NDI0IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy85LzMwOTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3091.41" data-doi="10.1016/0022-2836(90)90306-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sassanfar</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roberts</span>  <span class="cit-name-given-names">J. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">Nature of the SOS-inducing signal in <em>Escherichia coli</em>. The involvement of DNA replication.</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">212</span>: <span class="cit-fpage">79</span>–<span class="cit-lpage">96</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DSassanfar%26rft.auinit1%253DM.%26rft.volume%253D212%26rft.issue%253D1%26rft.spage%253D79%26rft.epage%253D96%26rft.atitle%253DNature%2Bof%2Bthe%2BSOS-inducing%2Bsignal%2Bin%2BEscherichia%2Bcoli.%2BThe%2Binvolvement%2Bof%2BDNA%2Breplication.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0022-2836%252890%252990306-7%26rft_id%253Dinfo%253Apmid%252F2108251%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0022-2836(90)90306-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2108251&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3091.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1990CV28600011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3091.42" data-doi="10.1038/nmeth.2089"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schneider</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rasband</span>  <span class="cit-name-given-names">W. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eliceiri</span>  <span class="cit-name-given-names">K. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">NIH Image to ImageJ: 25 years of image analysis.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">671</span>–<span class="cit-lpage">675</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DSchneider%26rft.auinit1%253DC.%2BA.%26rft.volume%253D9%26rft.issue%253D7%26rft.spage%253D671%26rft.epage%253D675%26rft.atitle%253DNIH%2BImage%2Bto%2BImageJ%253A%2B25%2Byears%2Bof%2Bimage%2Banalysis.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.2089%26rft_id%253Dinfo%253Apmid%252F22930834%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.2089&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22930834&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3091.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000305942200020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3091.43" data-doi="10.7554/eLife.01222"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shee</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cox</span>  <span class="cit-name-given-names">B. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gu</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luengas</span>  <span class="cit-name-given-names">E. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Joshi</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Engineered proteins detect spontaneous DNA breakage in human and bacterial cells.</span> <abbr class="cit-jnl-abbrev">Elife</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">e01222</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DElife%26rft_id%253Dinfo%253Adoi%252F10.7554%252FeLife.01222%26rft_id%253Dinfo%253Apmid%252F24171103%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NToiZWxpZmUiO3M6NToicmVzaWQiO3M6MTA6IjIvMC9lMDEyMjIiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzkvMzA5MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3091.44" data-doi="10.1073/pnas.85.6.1806"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shinagawa</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Iwasaki</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kato</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nakata</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1988</span> <span class="cit-article-title">RecA protein-dependent cleavage of UmuD protein and SOS mutagenesis.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">85</span>: <span class="cit-fpage">1806</span>–<span class="cit-lpage">1810</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DShinagawa%26rft.auinit1%253DH.%26rft.volume%253D85%26rft.issue%253D6%26rft.spage%253D1806%26rft.epage%253D1810%26rft.atitle%253DRecA%2Bprotein-dependent%2Bcleavage%2Bof%2BUmuD%2Bprotein%2Band%2BSOS%2Bmutagenesis.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.85.6.1806%26rft_id%253Dinfo%253Apmid%252F3126496%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiI4NS82LzE4MDYiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzkvMzA5MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3091.45" data-doi="10.1128/jb.175.17.5505-5509.1993"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Skarstad</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boye</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Degradation of individual chromosomes in <em>recA</em> mutants of <em>Escherichia coli</em>.</span> <abbr class="cit-jnl-abbrev">J. Bacteriol.</abbr> <span class="cit-vol">175</span>: <span class="cit-fpage">5505</span>–<span class="cit-lpage">5509</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBacteriology%26rft.stitle%253DJ.%2BBacteriol.%26rft.aulast%253DSkarstad%26rft.auinit1%253DK.%26rft.volume%253D175%26rft.issue%253D17%26rft.spage%253D5505%26rft.epage%253D5509%26rft.atitle%253DDegradation%2Bof%2Bindividual%2Bchromosomes%2Bin%2BrecA%2Bmutants%2Bof%2BEscherichia%2Bcoli.%26rft_id%253Dinfo%253Adoi%252F10.1128%252Fjb.175.17.5505-5509.1993%26rft_id%253Dinfo%253Apmid%252F8366035%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6MTE6IjE3NS8xNy81NTA1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy85LzMwOTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3091.46" data-doi="10.1101/cshperspect.a016436"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Symington</span>  <span class="cit-name-given-names">L. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">End resection at double-strand breaks: mechanism and regulation.</span> <abbr class="cit-jnl-abbrev">Cold Spring Harb. Perspect. Biol.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">a016436</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCold%2BSpring%2BHarbor%2BPerspectives%2Bin%2BBiology%26rft.stitle%253DCold%2BSpring%2BHarb.%2BPerspect.%2BBiol.%26rft.aulast%253DSymington%26rft.auinit1%253DL.%2BS.%26rft.volume%253D6%26rft.issue%253D8%26rft.spage%253Da016436%26rft.epage%253Da016436%26rft.atitle%253DEnd%2BResection%2Bat%2BDouble-Strand%2BBreaks%253A%2BMechanism%2Band%2BRegulation%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fcshperspect.a016436%26rft_id%253Dinfo%253Apmid%252F25085909%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6ImNzaHBlcnNwZWN0IjtzOjU6InJlc2lkIjtzOjExOiI2LzgvYTAxNjQzNiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvOS8zMDkxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3091.47" data-doi="10.1073/pnas.1213431110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Truong</span>  <span class="cit-name-given-names">L. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shi</span>  <span class="cit-name-given-names">L. Z.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Microhomology-mediated end joining and homologous recombination share the initial end resection step to repair DNA double-strand breaks in mammalian cells.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">110</span>: <span class="cit-fpage">7720</span>–<span class="cit-lpage">7725</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1213431110%26rft_id%253Dinfo%253Apmid%252F23610439%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTEwLzE5Lzc3MjAiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzkvMzA5MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3091.48"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ulmer</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gomez</span>  <span class="cit-name-given-names">R. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sinskey</span>  <span class="cit-name-given-names">A. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1979</span> <span class="cit-article-title">Ionizing radiation damage to the folded chromosome of <em>Escherichia coli</em> K-12: sedimentation properties of irradiated nucleoids and chromosomal deoxyribonucleic acid.</span> <abbr class="cit-jnl-abbrev">J. Bacteriol.</abbr> <span class="cit-vol">138</span>: <span class="cit-fpage">475</span>–<span class="cit-lpage">485</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBacteriology%26rft.stitle%253DJ.%2BBacteriol.%26rft.aulast%253DUlmer%26rft.auinit1%253DK%2BM%26rft.volume%253D138%26rft.issue%253D2%26rft.spage%253D475%26rft.epage%253D485%26rft.atitle%253DIonizing%2Bradiation%2Bdamage%2Bto%2Bthe%2Bfolded%2Bchromosome%2Bof%2BEscherichia%2Bcoli%2BK-12%253A%2Bsedimentation%2Bproperties%2Bof%2Birradiated%2Bnucleoids%2Band%2Bchromosomal%2Bdeoxyribonucleic%2Bacid.%26rft_id%253Dinfo%253Apmid%252F374388%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6OToiMTM4LzIvNDc1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy85LzMwOTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3091.49" data-doi="10.1073/pnas.1508543112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vallerga</span>  <span class="cit-name-given-names">M. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mansilla</span>  <span class="cit-name-given-names">S. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Federico</span>  <span class="cit-name-given-names">M. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bertolin</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gottifredi</span>  <span class="cit-name-given-names">V.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Rad51 recombinase prevents Mre11 nuclease-dependent degradation and excessive PrimPol-mediated elongation of nascent DNA after UV irradiation.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">112</span>: <span class="cit-fpage">E6624</span>–<span class="cit-lpage">E6633</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1508543112%26rft_id%253Dinfo%253Apmid%252F26627254%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTEyLzQ4L0U2NjI0IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy85LzMwOTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-book" id="cit-7.9.3091.50"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vujaklija</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maček</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Detecting posttranslational modifications of bacterial SSB proteins</span>, pp. <span class="cit-fpage">205</span>–<span class="cit-lpage">218</span> in <span class="cit-source"><em>Single-Stranded DNA Binding Proteins</em></span> in Methods in Molecular Biology, edited by <span class="cit-ed"><span class="cit-name-surname">Keck</span>  <span class="cit-name-given-names">J.</span></span>. <span class="cit-publ-name">Humana Press, Copyright Holder Springer Science+Business Media, LLC, NY</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3091.51" data-doi="10.2307/3579809"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wallace</span>  <span class="cit-name-given-names">S. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Enzymatic processing of radiation-induced free radical damage in DNA.</span> <abbr class="cit-jnl-abbrev">Radiat. Res.</abbr> <span class="cit-vol">150</span>: <span class="cit-fpage">S60</span>–<span class="cit-lpage">S79</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRadiation%2Bresearch%26rft.stitle%253DRadiat%2BRes%26rft.aulast%253DWallace%26rft.auinit1%253DS.%2BS.%26rft.volume%253D150%26rft.issue%253D5%2BSuppl%26rft.spage%253DS60%26rft.epage%253DS79%26rft.atitle%253DEnzymatic%2Bprocessing%2Bof%2Bradiation-induced%2Bfree%2Bradical%2Bdamage%2Bin%2BDNA.%26rft_id%253Dinfo%253Adoi%252F10.2307%252F3579809%26rft_id%253Dinfo%253Apmid%252F9806610%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/3579809&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9806610&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3091.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000076698300008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3091.52" data-doi="10.1038/nmicro2917"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wigley</span>  <span class="cit-name-given-names">D. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Bacterial DNA repair: recent insights into the mechanism of RecBCD, AddAB and AdnAB.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Microbiol.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">9</span>–<span class="cit-lpage">13</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BMicrobiol.%26rft.volume%253D11%26rft.spage%253D9%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmicro2917%26rft_id%253Dinfo%253Apmid%252F23202527%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmicro2917&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23202527&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3091.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3091.53"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Willets</span>  <span class="cit-name-given-names">N. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clark</span>  <span class="cit-name-given-names">A. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1969</span> <span class="cit-article-title">Characteristics of some multiply recombination-deficient strains of <em>Escherichia coli</em>.</span> <abbr class="cit-jnl-abbrev">J. Bacteriol.</abbr> <span class="cit-vol">100</span>: <span class="cit-fpage">53</span>–<span class="cit-lpage">66</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBacteriol.%26rft.volume%253D100%26rft.spage%253D53%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3091.54" data-doi="10.1074/jbc.M303581200"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">H. Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Deprez</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tauc</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ladjimi</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">The <em>Escherichia coli</em> RecQ helicase functions as a monomer.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">278</span>: <span class="cit-fpage">34925</span>–<span class="cit-lpage">34933</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M303581200%26rft_id%253Dinfo%253Apmid%252F12805371%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyNzgvMzcvMzQ5MjUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzkvMzA5MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3091.55" data-doi="10.1006/jmbi.1998.2127"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Souaya</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Julin</span>  <span class="cit-name-given-names">D. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Identification of the nuclease active site in the multifunctional RecBCD enzyme by creation of a chimeric enzyme.</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">283</span>: <span class="cit-fpage">797</span>–<span class="cit-lpage">808</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DYu%26rft.auinit1%253DM.%26rft.volume%253D283%26rft.issue%253D4%26rft.spage%253D797%26rft.epage%253D808%26rft.atitle%253DIdentification%2Bof%2Bthe%2Bnuclease%2Bactive%2Bsite%2Bin%2Bthe%2Bmultifunctional%2BRecBCD%2Benzyme%2Bby%2Bcreation%2Bof%2Ba%2Bchimeric%2Benzyme.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fjmbi.1998.2127%26rft_id%253Dinfo%253Apmid%252F9790841%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/jmbi.1998.2127&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9790841&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3091.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000076854700008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3091.56" data-doi="10.1128/JB.01877-07"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zahradka</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buljubašić</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Petranović</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zahradka</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Roles of ExoI and SbcCD nucleases in “reckless” DNA degradation in <em>recA</em> mutants of <em>Escherichia coli</em>.</span> <abbr class="cit-jnl-abbrev">J. Bacteriol.</abbr> <span class="cit-vol">191</span>: <span class="cit-fpage">1677</span>–<span class="cit-lpage">1687</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBacteriol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJB.01877-07%26rft_id%253Dinfo%253Apmid%252F19074388%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6MTA6IjE5MS81LzE2NzciO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzkvMzA5MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3091.57" data-doi="10.1101/cshperspect.a016451"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zelensky</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kanaar</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wyman</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Mediators of homologous DNA pairing.</span> <abbr class="cit-jnl-abbrev">Cold Spring Harb. Perspect. Biol.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">a016451</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCold%2BSpring%2BHarb.%2BPerspect.%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fcshperspect.a016451%26rft_id%253Dinfo%253Apmid%252F25301930%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6ImNzaHBlcnNwZWN0IjtzOjU6InJlc2lkIjtzOjEyOiI2LzEyL2EwMTY0NTEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzkvMzA5MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/7/9/3091.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/9/3083" title="Regulation of Small Mitochondrial DNA Replicative Advantage by Ribonucleotide Reductase in Saccharomyces cerevisiae" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/9/3103" title="Application of Response Surface Methods To Determine Conditions for Optimal Genomic Prediction" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/7/9"><span>Volume 7 Issue 9, September 2017</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/7/9.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/7/9.cover-source.jpg?itok=WuYpGT8r" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 7 (9)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/9/3091" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Zfi7p4ozpt2jV4tQc225K7dLsjZXws4zb-xRMdTBLNA" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/7/9/3091" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/7/9/3091" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/7/9/3091" class="active" data-icon-position="" data-hide-link-title="0">3′-Terminated Overhangs Regulate DNA Double-Strand Break Processing in Escherichia coli</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/486373" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-SIiVUpUvisJdHe0GiplQhwG6VGiK4zLjwhezNgJWBrI" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/9/3091.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/7/9/3091" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-svofNlQp66urpMuf1wuOLfxdWFywpoJeeMgD3M63Reo" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/7/9/3091" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/9/3091.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="486373" id="node486373--2" data-pisa="ggg;7/9/3091" data-pisa-master="ggg;g3.117.043521" data-apath="/ggg/7/9/3091.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">3′-Terminated Overhangs Regulate DNA Double-Strand Break Processing in <em>Escherichia coli</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Edyta Đermić</span>, <span class="highwire-citation-author" data-delta="1">Davor Zahradka</span>, <span class="highwire-citation-author" data-delta="2">Dušica Vujaklija</span>, <span class="highwire-citation-author" data-delta="3">Siniša Ivanković</span> and <span class="highwire-citation-author" data-delta="4">Damir Đermić</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3091-3102;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.043521 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node486373--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Edyta Đermić </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Pathology, Faculty of Agriculture, University of Zagreb, 10000, Croatia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Edyta%2B%C4%90ermi%C4%87%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=%C4%90ermi%C4%87%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEdyta%2B%25C4%2590ermi%25C4%2587%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Davor Zahradka </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Division of Molecular Biology, Ruđer Bošković Institute, 10000 Zagreb, Croatia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Davor%2BZahradka%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zahradka%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADavor%2BZahradka%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Dušica Vujaklija </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Division of Molecular Biology, Ruđer Bošković Institute, 10000 Zagreb, Croatia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Du%C5%A1ica%2BVujaklija%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Vujaklija%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADu%25C5%25A1ica%2BVujaklija%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Siniša Ivanković </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Division of Molecular Medicine, Ruđer Bošković Institute, 10000 Zagreb, Croatia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sini%C5%A1a%2BIvankovi%C4%87%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ivankovi%C4%87%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASini%25C5%25A1a%2BIvankovi%25C4%2587%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Damir Đermić </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Division of Molecular Biology, Ruđer Bošković Institute, 10000 Zagreb, Croatia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Damir%2B%C4%90ermi%C4%87%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=%C4%90ermi%C4%87%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADamir%2B%25C4%2590ermi%25C4%2587%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:dermic@irb.hr" class="" data-icon-position="" data-hide-link-title="0">dermic@irb.hr</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/7/9/3091" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/7/9/3091&amp;title=3%E2%80%B2-Terminated%20Overhangs%20Regulate%20DNA%20Double-Strand%20Break%20Processing%20in%20Escherichia%20coli" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/7/9/3091&amp;title=3%E2%80%B2-Terminated%20Overhangs%20Regulate%20DNA%20Double-Strand%20Break%20Processing%20in%20Escherichia%20coli" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/7/9/3091&amp;title=3%E2%80%B2-Terminated%20Overhangs%20Regulate%20DNA%20Double-Strand%20Break%20Processing%20in%20Escherichia%20coli" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/7/9/3091&amp;text=3%E2%80%B2-Terminated%20Overhangs%20Regulate%20DNA%20Double-Strand%20Break%20Processing%20in%20Escherichia%20coli" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/7/9/3091&amp;title=3%E2%80%B2-Terminated%20Overhangs%20Regulate%20DNA%20Double-Strand%20Break%20Processing%20in%20Escherichia%20coli" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/7/9/3091&amp;t=3%E2%80%B2-Terminated%20Overhangs%20Regulate%20DNA%20Double-Strand%20Break%20Processing%20in%20Escherichia%20coli" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/7/9/3091&amp;title=3%E2%80%B2-Terminated%20Overhangs%20Regulate%20DNA%20Double-Strand%20Break%20Processing%20in%20Escherichia%20coli" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/7/9/3091&amp;title=3%E2%80%B2-Terminated%20Overhangs%20Regulate%20DNA%20Double-Strand%20Break%20Processing%20in%20Escherichia%20coli" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/486373/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="486373" id="node486373--3" data-pisa="ggg;7/9/3091" data-pisa-master="ggg;g3.117.043521" data-apath="/ggg/7/9/3091.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">3′-Terminated Overhangs Regulate DNA Double-Strand Break Processing in <em>Escherichia coli</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Edyta Đermić</span>, <span class="highwire-citation-author" data-delta="1">Davor Zahradka</span>, <span class="highwire-citation-author" data-delta="2">Dušica Vujaklija</span>, <span class="highwire-citation-author" data-delta="3">Siniša Ivanković</span> and <span class="highwire-citation-author" data-delta="4">Damir Đermić</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3091-3102;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.043521 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node486373--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Edyta Đermić </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Pathology, Faculty of Agriculture, University of Zagreb, 10000, Croatia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Edyta%2B%C4%90ermi%C4%87%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=%C4%90ermi%C4%87%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEdyta%2B%25C4%2590ermi%25C4%2587%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Davor Zahradka </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Division of Molecular Biology, Ruđer Bošković Institute, 10000 Zagreb, Croatia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Davor%2BZahradka%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zahradka%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADavor%2BZahradka%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Dušica Vujaklija </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Division of Molecular Biology, Ruđer Bošković Institute, 10000 Zagreb, Croatia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Du%C5%A1ica%2BVujaklija%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Vujaklija%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADu%25C5%25A1ica%2BVujaklija%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Siniša Ivanković </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Division of Molecular Medicine, Ruđer Bošković Institute, 10000 Zagreb, Croatia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sini%C5%A1a%2BIvankovi%C4%87%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ivankovi%C4%87%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASini%25C5%25A1a%2BIvankovi%25C4%2587%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Damir Đermić </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Division of Molecular Biology, Ruđer Bošković Institute, 10000 Zagreb, Croatia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Damir%2B%C4%90ermi%C4%87%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=%C4%90ermi%C4%87%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADamir%2B%25C4%2590ermi%25C4%2587%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:dermic@irb.hr" class="" data-icon-position="" data-hide-link-title="0">dermic@irb.hr</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=3091&amp;rft.epage=3102&amp;rft.atitle=3%E2%80%B2-Terminated%20Overhangs%20Regulate%20DNA%20Double-Strand%20Break%20Processing%20in%20Escherichia%20coli&amp;rft.volume=7&amp;rft.issue=9&amp;rft.date=2017-09-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=%C4%90ermi%C4%87%2C+Edyta&amp;rft.au=Zahradka%2C+Davor&amp;rft.au=Vujaklija%2C+Du%C5%A1ica&amp;rft.au=Ivankovi%C4%87%2C+Sini%C5%A1a&amp;rft.au=%C4%90ermi%C4%87%2C+Damir"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/486373/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/486373/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/486373/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/486373/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/486373/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/486373/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/486373/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/486373/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/486373/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/486373/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/486373/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/486373/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/7/9/3091" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-7" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-15" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/7/9/3091.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/7/9/3091.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/7/9/3091.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
