<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/7/9/3169" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/7/9/3169.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/7/9/3169.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/7/9/3169.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/7/9/3169.atom" />
<meta name="HW.pisa" content="ggg;7/9/3169" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Genome-Wide SNP Discovery and Analysis of Genetic Diversity in Farmed Sika Deer (Cervus nippon) in Northeast China Using Double-Digest Restriction Site-Associated DNA Sequencing" />
<meta name="DC.Identifier" content="10.1534/g3.117.300082" />
<meta name="DC.Date" content="2017-09-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2017 Ba et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Sika deer are an economically valuable species owing to their use in traditional Chinese medicine, particularly their velvet antlers. Sika deer in northeast China are mostly farmed in enclosure. Therefore, genetic management of farmed sika deer would benefit from detailed knowledge of their genetic diversity. In this study, we generated over 1.45 billion high-quality paired-end reads (288 Gbp) across 42 unrelated individuals using double-digest restriction site-associated DNA sequencing (ddRAD-seq). A total of 96,188 (29.63%) putative biallelic SNP loci were identified with an average sequencing depth of 23×. Based on the analysis, we found that the majority of the loci had a deficit of heterozygotes (FIS &gt;0) and low values of Hobs, which could be due to inbreeding and Wahlund effects. We also developed a collection of high-quality SNP probes that will likely be useful in a variety of applications in genotyping for cervid species in the future." />
<meta name="DC.Contributor" content="Hengxing Ba" />
<meta name="DC.Contributor" content="Boyin Jia" />
<meta name="DC.Contributor" content="Guiwu Wang" />
<meta name="DC.Contributor" content="Yifeng Yang" />
<meta name="DC.Contributor" content="Gilead Kedem" />
<meta name="DC.Contributor" content="Chunyi Li" />
<meta name="article:published_time" content="2017-09-01" />
<meta name="article:section" content="Genomic Selection" />
<meta name="citation_title" content="Genome-Wide SNP Discovery and Analysis of Genetic Diversity in Farmed Sika Deer (Cervus nippon) in Northeast China Using Double-Digest Restriction Site-Associated DNA Sequencing" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Sika deer are an economically valuable species owing to their use in traditional Chinese medicine, particularly their velvet antlers. Sika deer in northeast China are mostly farmed in enclosure. Therefore, genetic management of farmed sika deer would benefit from detailed knowledge of their genetic diversity. In this study, we generated over 1.45 billion high-quality paired-end reads (288 Gbp) across 42 unrelated individuals using double-digest restriction site-associated DNA sequencing (ddRAD-seq). A total of 96,188 (29.63%) putative biallelic SNP loci were identified with an average sequencing depth of 23×. Based on the analysis, we found that the majority of the loci had a deficit of heterozygotes (F&lt;sub&gt;IS&lt;/sub&gt; &amp;gt;0) and low values of H&lt;sub&gt;obs&lt;/sub&gt;, which could be due to inbreeding and Wahlund effects. We also developed a collection of high-quality SNP probes that will likely be useful in a variety of applications in genotyping for cervid species in the future.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2017/09/01" />
<meta name="citation_mjid" content="ggg;7/9/3169" />
<meta name="citation_id" content="7/9/3169" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/7/9/3169" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/7/9/3169.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/7/9/3169.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/7/9/3169.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.117.300082" />
<meta name="citation_pmid" content="28751500" />
<meta name="citation_volume" content="7" />
<meta name="citation_issue" content="9" />
<meta name="citation_num_pages" content="8" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Genomic Selection" />
<meta name="citation_firstpage" content="3169" />
<meta name="citation_lastpage" content="3176" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Hengxing Ba" />
<meta name="citation_author_institution" content="State Key Laboratory for Molecular Biology of Special Wild Economic Animals, Institute of Special Wild Economic Animals and Plants, Chinese Academy of Agricultural Sciences, Changchun 130112, China" />
<meta name="citation_author" content="Boyin Jia" />
<meta name="citation_author_institution" content="State Key Laboratory for Molecular Biology of Special Wild Economic Animals, Institute of Special Wild Economic Animals and Plants, Chinese Academy of Agricultural Sciences, Changchun 130112, China" />
<meta name="citation_author" content="Guiwu Wang" />
<meta name="citation_author_institution" content="State Key Laboratory for Molecular Biology of Special Wild Economic Animals, Institute of Special Wild Economic Animals and Plants, Chinese Academy of Agricultural Sciences, Changchun 130112, China" />
<meta name="citation_author" content="Yifeng Yang" />
<meta name="citation_author_institution" content="State Key Laboratory for Molecular Biology of Special Wild Economic Animals, Institute of Special Wild Economic Animals and Plants, Chinese Academy of Agricultural Sciences, Changchun 130112, China" />
<meta name="citation_author" content="Gilead Kedem" />
<meta name="citation_author_institution" content="GENEWIZ Inc., South Plainfield, New Jersey 07080" />
<meta name="citation_author" content="Chunyi Li" />
<meta name="citation_author_institution" content="State Key Laboratory for Molecular Biology of Special Wild Economic Animals, Institute of Special Wild Economic Animals and Plants, Chinese Academy of Agricultural Sciences, Changchun 130112, China" />
<meta name="citation_author_email" content="lichunyi1959@163.com" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=A. J. Amaral;citation_author=H. J. Megens;citation_author=H. H. Kerstens;citation_author=H. C. Heuven;citation_author=B. Dibbits;citation_title=Application of massive parallel sequencing to whole genome SNP discovery in the porcine genome.;citation_pages=374-374;citation_volume=10;citation_year=2009;citation_pmid=19674453;citation_doi=10.1186/1471-2164-10-374" />
<meta name="citation_reference" content="citation_journal_title=Mitochondrial DNA;citation_author=H. Ba;citation_author=F. Yang;citation_author=X. Xing;citation_author=C. Li;citation_title=Classification and phylogeny of sika deer (Cervus nippon) subspecies based on the mitochondrial control region DNA sequence using an extended sample set.;citation_pages=373-379;citation_volume=26;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Mitochondrial DNA A DNA Mapp. Seq. Anal.;citation_author=H. Ba;citation_author=L. Wu;citation_author=Z. Liu;citation_author=C. Li;citation_title=An examination of the origin and evolution of additional tandem repeats in the mitochondrial DNA control region of Japanese sika deer (Cervus Nippon).;citation_pages=276-281;citation_volume=27;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=S. W. Baxter;citation_author=J. W. Davey;citation_author=J. S. Johnston;citation_author=A. M. Shelton;citation_author=D. G. Heckel;citation_title=Linkage mapping and comparative genomics using next-generation RAD sequencing of a non-model organism.;citation_pages=e19315-e19315;citation_volume=6;citation_year=2011;citation_issue=4;citation_pmid=21541297;citation_doi=10.1371/journal.pone.0019315" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=J. Catchen;citation_author=P. A. Hohenlohe;citation_author=S. Bassham;citation_author=A. Amores;citation_author=W. A. Cresko;citation_title=Stacks: an analysis tool set for population genomics.;citation_pages=3124-3140;citation_volume=22;citation_year=2013;citation_pmid=23701397;citation_doi=10.1111/mec.12354" />
<meta name="citation_reference" content="citation_journal_title=G3;citation_author=J. M. Catchen;citation_author=A. Amores;citation_author=P. Hohenlohe;citation_author=W. Cresko;citation_author=J. H. Postlethwait;citation_title=Stacks: building and genotyping Loci de novo from short-read sequences.;citation_pages=171-182;citation_volume=1;citation_year=2011" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=J. M. DaCosta;citation_author=M. D. Sorenson;citation_title=Amplification biases and consistent recovery of loci in a double-digest RAD-seq protocol.;citation_pages=e106713;citation_volume=9;citation_year=2014;citation_pmid=25188270;citation_doi=10.1371/journal.pone.0106713" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=J. W. Davey;citation_author=P. A. Hohenlohe;citation_author=P. D. Etter;citation_author=J. Q. Boone;citation_author=J. M. Catchen;citation_title=Genome-wide genetic marker discovery and genotyping using next-generation sequencing.;citation_pages=499-510;citation_volume=12;citation_year=2011;citation_issue=7;citation_pmid=21681211;citation_doi=10.1038/nrg3012" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=M. A. DePristo;citation_author=E. Banks;citation_author=R. Poplin;citation_author=K. V. Garimella;citation_author=J. R. Maguire;citation_title=A framework for variation discovery and genotyping using next-generation DNA sequencing data.;citation_pages=491-498;citation_volume=43;citation_year=2011;citation_issue=5;citation_pmid=21478889;citation_doi=10.1038/ng.806" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=J. C. Dohm;citation_author=C. Lottaz;citation_author=T. Borodina;citation_author=H. Himmelbauer;citation_title=Substantial biases in ultra-short read data sets from high-throughput DNA sequencing.;citation_volume=36;citation_year=2008;citation_pmid=18660515;citation_doi=10.1093/nar/gkn425" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=S. H. Eck;citation_author=A. Benet-Pages;citation_author=K. Flisikowski;citation_author=T. Meitinger;citation_author=R. Fries;citation_title=Whole genome sequencing of a single Bos taurus animal for single nucleotide polymorphism discovery.;citation_pages=R82-R82;citation_volume=10;citation_year=2009;citation_issue=8;citation_pmid=19660108;citation_doi=10.1186/gb-2009-10-8-r82" />
<meta name="citation_reference" content="citation_journal_title=Human heredity;citation_journal_abbrev=Hum Hered;citation_author=J. Graffelman;citation_author=J. M. Camarena;citation_title=Graphical tests for Hardy-Weinberg equilibrium based on the ternary plot.;citation_pages=77-84;citation_volume=65;citation_year=2008;citation_issue=2;citation_pmid=17898538;citation_doi=10.1159/000108939" />
<meta name="citation_reference" content="citation_journal_title=Anim. Genet.;citation_author=L. F. Groeneveld;citation_author=J. A. Lenstra;citation_author=H. Eding;citation_author=M. A. Toro;citation_author=B. Scherf;citation_title=Genetic diversity in farm animals–a review.;citation_pages=6-31;citation_volume=41;citation_year=2010;citation_pmid=20500753;citation_doi=10.1111/j.1365-2052.2010.02038.x" />
<meta name="citation_reference" content="citation_journal_title=Molecular phylogenetics and evolution;citation_journal_abbrev=Mol Phylogenet Evol;citation_author=S. Guha;citation_author=S. P. Goyal;citation_author=V. K. Kashyap;citation_title=Molecular phylogeny of musk deer: a genomic view with mitochondrial 16S rRNA and cytochrome b gene.;citation_pages=585-597;citation_volume=42;citation_year=2007;citation_issue=3;citation_pmid=17158073;citation_doi=10.1016/j.ympev.2006.06.020" />
<meta name="citation_reference" content="citation_journal_title=Syst. Biol.;citation_author=A. Hassanin;citation_author=E. J. Douzery;citation_title=Molecular and morphological phylogenies of ruminantia and the alternative position of the moschidae.;citation_volume=52;citation_year=2003;citation_pmid=12746147;citation_doi=10.1080/10635150390192726;citation_georef=2005040515" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=G. D. Haynes;citation_author=E. K. Latch;citation_title=Identification of novel single nucleotide polymorphisms (SNPs) in deer (Odocoileus spp.) using the BovineSNP50 BeadChip.;citation_pages=e36536-e36536;citation_volume=7;citation_year=2012;citation_issue=5;citation_pmid=22590559;citation_doi=10.1371/journal.pone.0036536" />
<meta name="citation_reference" content="citation_journal_title=Dongwuxue Yanjiu;citation_author=Y. He;citation_author=Z. H. Wang;citation_author=X. M. Wang;citation_title=Genetic diversity and population structure of a Sichuan sika deer (Cervus sichuanicus) population in Tiebu Nature Reserve based on microsatellite variation.;citation_pages=528-536;citation_volume=35;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=H. H. Kerstens;citation_author=R. P. Crooijmans;citation_author=A. Veenendaal;citation_author=B. W. Dibbits;citation_author=A. W. T. F. Chin;citation_title=Large scale single nucleotide polymorphism discovery in unsequenced genomes using second generation high throughput sequencing technology: applied to turkey.;citation_pages=479-479;citation_volume=10;citation_year=2009;citation_pmid=19835600;citation_doi=10.1186/1471-2164-10-479" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=R. H. Kraus;citation_author=H. H. Kerstens;citation_author=P. Van Hooft;citation_author=R. P. Crooijmans;citation_author=J. J. Van Der Poel;citation_title=Genome wide SNP discovery, analysis and evaluation in mallard (Anas platyrhynchos).;citation_pages=150-150;citation_volume=12;citation_year=2011;citation_pmid=21410945" />
<meta name="citation_reference" content="citation_journal_title=J. Hered.;citation_author=J. Krojerova-Prokesova;citation_author=M. Barancekova;citation_author=I. Voloshina;citation_author=A. Myslenkov;citation_author=J. Lamka;citation_title=Dybowski’s sika deer (Cervus nippon hortulorum): genetic divergence between natural primorian and introduced Czech populations.;citation_volume=104;citation_year=2013;citation_pmid=23454911;citation_doi=10.1093/jhered/est006" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=B. Langmead;citation_author=S. L. Salzberg;citation_title=Fast gapped-read alignment with Bowtie 2.;citation_pages=357-359;citation_volume=9;citation_year=2012;citation_issue=4;citation_pmid=22388286;citation_doi=10.1038/nmeth.1923" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=S. Q. Le;citation_author=R. Durbin;citation_title=SNP detection and genotyping from low-coverage sequencing data on multiple diploid samples.;citation_volume=21;citation_year=2011;citation_pmid=20980557;citation_doi=10.1101/gr.113084.110" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_title=A statistical framework for SNP calling, mutation discovery, association mapping and population genetical parameter estimation from sequencing data.;citation_volume=27;citation_year=2011;citation_pmid=21903627;citation_doi=10.1093/bioinformatics/btr509" />
<meta name="citation_reference" content="citation_journal_title=Chin. Sci. Bull.;citation_author=X. P. Lu;citation_author=F. W. Wei;citation_author=M. Li;citation_author=G. Yang;citation_author=H. Liu;citation_title=Genetic diversity among Chinese sika deer (Cervus nippon) populations and relationships between Chinese and Japanese sika deer.;citation_pages=433-440;citation_volume=51;citation_year=2006" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=N. Malhis;citation_author=S. J. M. Jones;citation_title=High quality SNP calling using Illumina data at shallow coverage.;citation_volume=26;citation_year=2010;citation_pmid=20190250;citation_doi=10.1093/bioinformatics/btq092" />
<meta name="citation_reference" content="citation_title=Sika Deer in Mainland China;citation_title=Sika deer: biology and management of native and introduced populations;citation_author=E. D. R. McCullough;citation_author=S. Takatsuki;citation_author=K. Kaji;citation_pages=526-531;citation_year=2009" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=N. Patterson;citation_author=A. L. Price;citation_author=D. Reich;citation_title=Population structure and eigenanalysis.;citation_pages=2074-2093;citation_volume=2;citation_year=2006" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=B. K. Peterson;citation_author=J. N. Weber;citation_author=E. H. Kay;citation_author=H. S. Fisher;citation_author=H. E. Hoekstra;citation_title=Double digest RADseq: an inexpensive method for de novo SNP discovery and genotyping in model and non-model species.;citation_pages=e37135-e37135;citation_volume=7;citation_year=2012;citation_issue=5;citation_pmid=22675423;citation_doi=10.1371/journal.pone.0037135" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol. Resour.;citation_author=J. H. Powell;citation_author=S. J. Amish;citation_author=G. D. Haynes;citation_author=G. Luikart;citation_author=E. K. Latch;citation_title=Candidate adaptive genes associated with lineage divergence: identifying SNPs via next-generation targeted resequencing in mule deer (Odocoileus hemionus).;citation_pages=1165-1172;citation_volume=16;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. K. Pritchard;citation_author=M. Stephens;citation_author=P. Donnelly;citation_title=Inference of Population Structure Using Multilocus Genotype Data;citation_pages=945-959;citation_volume=155;citation_year=2000;citation_issue=2;citation_pmid=10835412;citation_doi=10.1111/j.1471-8286.2005.01155.x" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=C. M. Seabury;citation_author=E. K. Bhattarai;citation_author=J. F. Taylor;citation_author=G. G. Viswanathan;citation_author=S. M. Cooper;citation_title=Genome-wide polymorphism and comparative analyses in the white-tailed deer (Odocoileus virginianus): a model for conservation genomics.;citation_pages=e15811-e15811;citation_volume=6;citation_year=2011;citation_issue=1;citation_pmid=21283515;citation_doi=10.1371/journal.pone.0015811" />
<meta name="citation_reference" content="citation_title=Deer of China: Biology and Management;citation_author=H. L. Sheng;citation_author=N. Ohtaishi;citation_year=1993" />
<meta name="citation_reference" content="citation_journal_title=Mamm. Genome;citation_author=J. Szyda;citation_author=M. Fraszczak;citation_author=M. Mielczarek;citation_author=R. Giannico;citation_author=G. Minozzi;citation_title=The assessment of inter-individual variation of whole-genome DNA sequence in 32 cows.;citation_pages=658-665;citation_volume=26;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Zoological science;citation_journal_abbrev=Zoolog Sci;citation_author=H. Takiguchi;citation_author=K. Tanaka;citation_author=K. Ono;citation_author=A. Hoshi;citation_author=M. Minami;citation_title=Genetic variation and population structure of the Japanese sika deer (Cervus nippon) in the Tohoku District based on mitochondrial D-loop sequences.;citation_pages=433-436;citation_volume=29;citation_year=2012;citation_issue=7;citation_pmid=22775251" />
<meta name="citation_reference" content="citation_journal_title=Zoological science;citation_journal_abbrev=Zoolog Sci;citation_author=H. B. Tamate;citation_author=A. Okada;citation_author=M. Minami;citation_author=N. Ohnishi;citation_author=H. Higuchi;citation_title=Genetic Variations Revealed by Microsatellite Markers ina Small Population of the Sika Deer (Cervus nippon)on Kinkazan Island, Northern Japan.;citation_pages=47-53;citation_volume=17;citation_year=2000;citation_issue=1;citation_pmid=18494571;citation_doi=10.2108/zsj.17.47" />
<meta name="citation_reference" content="citation_journal_title=Journal of Heredity;citation_journal_abbrev=J Hered;citation_author=S. Thevenon;citation_author=L. T. Thuy;citation_author=L. V. Ly;citation_author=F. Maudet;citation_author=A. Bonnet;citation_title=Microsatellite Analysis of Genetic Diversity of the Vietnamese Sika Deer (Cervus nippon pseudaxis);citation_pages=11-18;citation_volume=95;citation_year=2004;citation_issue=1;citation_pmid=14757725;citation_doi=10.1093/jhered/esh001" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=W. Wang;citation_author=H. Yan;citation_author=J. Yu;citation_author=J. Yi;citation_author=Y. Qu;citation_title=Discovery of genome-wideSNPs by RAD-seqand the genetic diversity of captive hog deer (Axis porcinus).;citation_pages=e0174299;citation_volume=12;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Biol. Conserv.;citation_author=H. Wu;citation_author=Q. H. Wan;citation_author=S. G. Fang;citation_title=Two genetically distinct units of the Chinese sika deer (Cervus nippon): analyses of mitochondrial DNA variation.;citation_pages=183-190;citation_volume=119;citation_year=2004" />
<meta name="citation_reference" content="citation_journal_title=Forensic Science International;citation_journal_abbrev=Forensic Science International;citation_author=H. Wu;citation_author=Q. H. Wan;citation_author=S. G. Fang;citation_author=S. Y. Zhang;citation_title=Application of mitochondrial DNA sequence analysis in the forensic identification of Chinese sika deer subspecies.;citation_pages=101-105;citation_volume=148;citation_year=2005;citation_issue=2-3;citation_pmid=15639603" />
<meta name="citation_reference" content="citation_journal_title=Zhong yao cai = Zhongyaocai = Journal of Chinese medicinal materials;citation_journal_abbrev=Zhong Yao Cai;citation_author=P. J. Wu;citation_author=E. D. Zhang;citation_title=[The resource conservation and utilization of wild Sika deer in China];citation_pages=552-554;citation_volume=24;citation_year=2001;citation_issue=8;citation_pmid=11715188" />
<meta name="citation_reference" content="citation_journal_title=bioRxiv;citation_author=L. Zhu;citation_author=X. Zhang;citation_author=C. Deng;citation_author=J. Ding;citation_author=Y. Ren;citation_title=Comparative genomics and metagenomics analyses of endangered Père David’s deer (Elaphurus davidianus) provide insights into population recovery.;citation_year=2016;citation_doi=10.1101/073528" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Genome-Wide SNP Discovery and Analysis of Genetic Diversity in Farmed Sika Deer (Cervus nippon) in Northeast China Using Double-Digest Restriction Site-Associated DNA Sequencing" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9.cover-source.jpg" />
<meta name="twitter:description" content="Sika deer are an economically valuable species owing to their use in traditional Chinese medicine, particularly their velvet antlers. Sika deer in northeast China are mostly farmed in enclosure. Therefore, genetic management of farmed sika deer would benefit from detailed knowledge of their genetic diversity. In this study, we generated over 1.45 billion high-quality paired-end reads (288 Gbp) across 42 unrelated individuals using double-digest restriction site-associated DNA sequencing (ddRAD-seq). A total of 96,188 (29.63%) putative biallelic SNP loci were identified with an average sequencing depth of 23×. Based on the analysis, we found that the majority of the loci had a deficit of heterozygotes (FIS &gt;0) and low values of Hobs, which could be due to inbreeding and Wahlund effects. We also developed a collection of high-quality SNP probes that will likely be useful in a variety of applications in genotyping for cervid species in the future." />
<meta name="og-title" property="og:title" content="Genome-Wide SNP Discovery and Analysis of Genetic Diversity in Farmed Sika Deer (Cervus nippon) in Northeast China Using Double-Digest Restriction Site-Associated DNA Sequencing" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/7/9/3169" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Sika deer are an economically valuable species owing to their use in traditional Chinese medicine, particularly their velvet antlers. Sika deer in northeast China are mostly farmed in enclosure. Therefore, genetic management of farmed sika deer would benefit from detailed knowledge of their genetic diversity. In this study, we generated over 1.45 billion high-quality paired-end reads (288 Gbp) across 42 unrelated individuals using double-digest restriction site-associated DNA sequencing (ddRAD-seq). A total of 96,188 (29.63%) putative biallelic SNP loci were identified with an average sequencing depth of 23×. Based on the analysis, we found that the majority of the loci had a deficit of heterozygotes (FIS &gt;0) and low values of Hobs, which could be due to inbreeding and Wahlund effects. We also developed a collection of high-quality SNP probes that will likely be useful in a variety of applications in genotyping for cervid species in the future." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9.cover-source.jpg" />
    <title>Genome-Wide SNP Discovery and Analysis of Genetic Diversity in Farmed Sika Deer (Cervus nippon) in Northeast China Using Double-Digest Restriction Site-Associated DNA Sequencing | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__Ta-UXINC0YONAj3hv7M-Srfc-_B4Wn0KATTOQGieTps__agTSMicuoC50kRvu-VcTNoPubLJgflHIUuLq_Jt5dMY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"Bl2tqatjIMpSfucW6RpM_S1Atc3D9oNOqavox9YPEBk","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"486356","apath":"\/ggg\/7\/9\/3169.atom","pisa":"ggg;7\/9\/3169","ac":{"\/ggg\/7\/9\/3169.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/7\/9\/3169.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;7\/9\/3169"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;7\/9\/3169"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/7\/9\/3169":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022x5cWFAxKkzGtA8WHOrkq+WmlK\\\\\\\/tw226Ch7LT4\\\\\\\/znHopXVaW84HJlVX7QQ4Q0YuBGHPsyXawbsQWtbB85vwJHk06xHAi2ie8wbfaFB7GntNEGaSjacHY5GMm1OpSQEDALzs8moiDY1OvOFrT+\\\\\\\/3GxajbtGoRG5mEXc\\\\\\\/EkIBPmkMnO\\\\\\\/B1pul7LiuzAF4gnJ5Ollu0J5KvZpUtsuSrj6HyszDaeInXknOxV9u1lO9CbZZeX+HCo+GSxw6qqPfpeAjKP+v6SQr1Jeg+z6TSDL2OLk1NVd5H7jYVsn3DdFdKdtguK1b+UffQ8JwOf6PM4gTIWTekVYEj3bb9IOYCwyuoprp75ci+mZGEk6UaNfDyMja\\\\\\\/QKGfovf8jfRiwCJUG+wBJwSZfhxzS5hhlkbo9ENNIdKVlNE9\\\\\\\/4PiUSP1yCNsEsQEhY\\\\\\\/EgIvUUQY1JVFeeuF8LcOxK2qWrYIjsQggAfpGUkKbhzmXvhF\\\\\\\/NUxLScZiOELIIHOnbd54j7vkD46g6ocmBTT6s1QiOb9YTmp0N2C9N1iVlcu4agpaKx8Imergdj\\\\\\\/fkqHMBD+nW+LVy4aq01ZfMFRZQy80rhz71I2+zyWirl236lMbBhR932pEhFa+AF9yIdJb5yZnkfg7jQ96lIvL3eTvp2sjMIR\\\\\\\/OIO2Y2upWykI6HuKCh14lhYRCZGqlD8seSgDMcQOjJIONL5K+HyusI\\\\\\\/JkMY+su\\\\\\\/5vrgKWBpyhat3\\\\\\\/8jatfGZ7P+kD07krOR7gm2tkqreiYSJ6s\\\\\\\/ygKBnmf\\\\\\\/vkmdh2aoYlYEJmoz+2XeJeFuODsq+jnIgI3FdKzM5ybGDkgu4TwNuiWHC\\\\\\\/apwcU+tW9KlD8AkML5KKuWaT1Pg010OttkjCRGsuASRGpnlRALgVo0O8nqYEix6EecmEbbrpmZZ4rQWorUDk5OvId24dIFC4q8dXyzt+GjZToft5kLOGCx4h09Cluc50cKwSg1uBLM9xZuGY+6857buobq6tOgfQ8b1u8V2ag2X192f7foKOxyyU6uWcGJ+SNwsXnRtJqTBVwIkEWnFdOdbJRLaJ6BnKylHA\\\\\\\/DqXH0aq0Di+BNnfE4PD1tWHByy5hXyVz\\\\\\\/x9SZ1ApKcIqLNnxBEcNU1KrUzuxYaRnDCUug9tsNMfuKADRWgceN\\\\\\\/Yu6joK\\\\\\\/wXaqg5tunPWdX6Z\\\\\\\/dTLAPjE8pblGm8qXxv9j9U4P95xTeqhm\\\\\\\/hQgDc8ZFW6Uwgv6oOZIMrAXAKYUR31PG9qHS7Ards\\\\\\\/U+iDmhIgq6IbHD44lwe8pnh4mK5TNB\\\\\\\/mETtyFUA\\\u0022,\\\u0022iv\\\u0022:\\\u0022wONC6cWA8gajJMBmeV+QEg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00229a0c7b302f6753174734521ce37fc83aebbf6900e2bd2a3994a1f4aa880d962b\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022I8LGLjTI4\\\\\\\/7ROA3bGmYE880GNvOs0hh7H9uDJDypGmdKU3eiHgK1p9UgZsu7shVif\\\\\\\/xkPEPzqztvA3BzEJTkqrgPmrTX+HmlyBiQCttutX7++Fx7TWmtPsEIxrMUCb3rUsyDbvKLgXerDklEGxf1Z+h\\\\\\\/ho88osaA2n8FFGK9rdPjTVZsqyoF6QKWj4ht0+Lv2FaA6FJDtLFHPR0q\\\\\\\/0V1DC\\\\\\\/5ZzTMSWCHWNhkCzeoP7UoEOTO5bwzhfQXoBptWnWm1s0w+jMQ3R6me\\\\\\\/23LG8vp\\\\\\\/jBJnNzHjBrmy8qbNTZncXkBlqz\\\\\\\/OsAWj\\\\\\\/oUprMj1kvXVLVux4cLaGkP52SVwnUpDXwWHI1OZscqWaSqYomtbmqpsslbV9qWG7hKO6yDW9k9CMGAsu3AzfS8lNlLc\\\\\\\/Tb8KtAO7RYdEvuHkevYVka5LX30GQ3sWHo8Tx7\\\\\\\/SL9OQpXy12ZWFpKbO6pkBJexZuHkEuR964q8zt5+vPwvsy1551B1SSUigjxWQIf8Qay7K6NmdSK4gGtR9d8gaimat4+y+pnYy0ua\\\\\\\/mBDQ+wmPnwZgCVaKs5xUfkbcAob10T6MvbFH\\\\\\\/7PK7LbiZDC5VHsv9+egx3RR0EBNegTHBAjeIh5XOUC+9rWwYN6CC1HDs\\\\\\\/IoxVgIQVMcwUKIleIGluyVbt7WhFYm2+El7iujZ7U0TP\\\\\\\/v3R+E8GzcdpzeI0hunqf35JLvG48sSMruH7pHH1gZDe7P51DUbCflBDwcne6fnQShawYTmuA7MR5a2FkJD1jH9oBm2b78hG\\\\\\\/Dlcvoegu2h9Bh97zLnvMiTYGLTZmNMXwDzxioWotZ4qCpqazg\\\\\\\/wFEyBG\\\\\\\/BWTlMCLyzgDCCdYgy9koCaRgDdNbL5CL76Y5ktYRpUOZIjSDRkQP6y29b\\\\\\\/19zMAF397kSauuhx+eIlSd+tG1f2XVbI+Gnb7Cl7JS9j9zy83BMIA1TA8lu5rynv3f1w0OIG6PeTUuoKblJ2XKLRqGD0n6M\\\\\\\/+RpzK6IMq6QMI0DRYGOLebSG9Ns1rlWAVWgF69Uoe49FVW1rmO859HLd7FdEvTE0YK11s\\\\\\\/OvFA3knlaMlNqdgdHcKfdxNjQGwWv1zv23Tw55ZGr1d6A2Rgz8+p9wElVsDVqpNUA3e49JtQ0JiE4dz7xGDKKuvVteMD\\\\\\\/Og35DIB6+TZkgq8OSqj\\\\\\\/JRhl\\\\\\\/4uKMiwMdVUXJW17uWWXSW+N\\\\\\\/qtw3kTDeE1\\\\\\\/PMuR3YAjT17hSUl9UBFngtzSZR3qo+nFTJ4xsVyrClVoLHaYtXekBx7EfPqR1gWiURrbhaR5AxtNcjkk9RKR4578YoMGt1mXwXjnN7X4LtHyF2N+\\\\\\\/RxiFl2\\\\\\\/Ez6tpS1qCcfJ6HRPemiGw\\\\\\\/DZNjDQiuqisWPLgfnn6HUdQwYKDbtzWSLLzK27oWgVyw7vHPfbw7c4zwlHhb5Qq89WOCTDEhWhFJk\\\\\\\/1EuVQ+iDwxEbNAlV9f1HZTPBIzmBT8R8W10LIITPEL\\\\\\\/LRtP6OIE4+uRkaydByPuywFDeeQP+QI9cMwEWe0\\\\\\\/szBY9ERwKFt1yQytGGQf1LeqvL1\\\\\\\/jSHn1DCnIwBMT486W9R8EpCmjC\\\\\\\/ZQC5oOYuPLVCmaE4Y2IDkjiL1c+BmgkRRVrYrQoSYSpNq5tfKzjGSN9CKB8GxwuBw\\\\\\\/vQ6fgJI17bPBIr3xGsSmCVrwk6b2y7lGxLdtwGMbP4jaUoCI1vgwABo3kOhgoKzaG2Bz+m0fSnYQvsEjH15R895nr4SyhDCp+MZLCOXU3hTnJt8MTasCdAwH1cFgD6nhNJiBQGkB5BYt00oNPwjJVAAkDsnFiBZX3akG9otPQU4zKomN37lX\\\\\\\/9DaO6+z+LXqEFJgjqtcJyuqiQt6glHP7vWgvJtfVtyiRdIdXYIAFpNXiWckDMjLnNkZyc+QBYQ8wV91kPb+OK2AdiB2SYtFoijxfPmS\\\\\\\/jfQXNw0wVwN2wqSKABRRH7ComvGVDevcp8DYAdd3\\\\\\\/IyPezZKJgQIGGLPRjlDIbIM2hQBScmJLCZquT789XOwoBgTarGfczMdjNpfKZG9XyYO+dbR+if3u2hn72gayTZV1S0\\\\\\\/V8rAHFvFeWofN1YLOwggJh0H1nTtGniv6Zd\\\\\\\/mIDu3WLSNMaRrPgwyJvXhf41le7HDOSFkOCAniKtlh4yEXWaE1Gre685DZCmvkLZs7iFNv\\\\\\\/X8+2xvbceggm4AbLfKq2pwu2flBF3PqbK1GOpb82bO5PLa4Jy7fpnDDzcnE9U8KKQdB7XdCOZRKLOY0GBdkmeCAkDUARYFQ00hfW7UOBW2PW2Bd1a1GNMgVlqmwrms8uTCqlUwLwL7kBkRLXMS6ud0ZL25byARxCLUHtQl3QPyN1xuiUx7VWcTBl6yfjt\\\\\\\/YpwYOkqtIZFe+QWYmenRIwY4sfDOIl0WeZlEBHL8mQ3cE+gtrHnsG3wc+AIXklJYJzkz\\\\\\\/YD7eFfQlzy9k8nZvbfds+Yk36SOyfqolWlItEYx0IXxfyniDHC2c0JK31MW5lB3Kenrc5JaM\\\\\\\/bpGWJuYLHIcU6wxPH70bmbnjkRNrHB5EcLgDmAJ+O+AJmjRMIRw3XlpUyhqukhqmFvLMT1ckDxYBVQIl\\\\\\\/HTdvbBBaQi\\\\\\\/JTDUX+v9gdJESTgtd4VgfDffNXKbgviq\\\\\\\/Kfxi0TR40O8z80EEQZc0yfA8qIGyj2hcVHnBEcFAcoU+nVz7aP9fEhQNyN85B1jm8v\\\\\\\/PJfjePi7GGavxbSWrzqdKCFvsK4CNlnl9tUY\\\\\\\/Ub1H\\\\\\\/gPyLIbpmfQ0Ehok+bIncYLHvqaXfaD+ZaSZd0CxTtpsqOGvTQfqYrObxAxzbsnhbbJy4jG030g3pwdCMYOQw0dLavYJIk8MIk5MTNVj2p3DKYIxi6YCkOunAuR7tqhJ\\\\\\\/1+yTljiNRN7nPyn4Il\\\\\\\/+O3mjqI+g8Oj85qxOZSgKTx5p41ok7ukQw5QKDhrTa1fqc0pPAMYT0l8k99s7WVcmg7GADLzSvMaJehoQ9\\\\\\\/pNP+Ke8dZWKQTyFg6KCdNoHKgEFZD8Ah87uKPv28jolNZWut60ulkT0B0TXDOTXA3xsCE6VHomOWNdZfqA9qIwbP2O0PywnRA9Ub1uLlwyoRtXerJtFYPfdc4gS+pmET5jNCtGySlmY8DBsQBPmwVW4PazI79MSvNkSOdI5AsyvTePXMg5gOWNCe7zS8+9+hIYrFWjBtnfF1l7fetoOWv9J553KZlWkf4tBYwCmpPIQMc9emIJF+fmEhrIEeCJS7aqrnMBE2cotjT\\\\\\\/IplZhcnuRLRUkaooSRBPYeJ+NzDApGbdoYiKxNnJzov\\\\\\\/ydiKFwEdtM67AksMb0dAV9AFJYe1J1F8ydxn5DlAnvO3EljEe0\\\\\\\/Th1j\\\\\\\/ZDr7zyyd+exbZrG45CxFWoBnfqYm9wb\\\\\\\/aVGTdf1SM9PrIW0qTN\\\\\\\/mL0Ig638WIM0ZxEa1BFPFEoOPuTuG77VU5O6zINUdAx1NFOftUXYV7EsUJgI4oLaNK35w141D0K6v+LC3vwyYHmLBeeJmHIrUu2dZ9SNwb94r621PhcyKlOeGYUmc05MXE6psuwi3NmIjQkoJ75vdCmtq\\\\\\\/r+vbFH2C2\\\\\\\/AzScL5RU8+rOj70Ytfu70M6tCBvd9EPfsiDgXOGd6abCKlDq2oJJFTyrP0nqvbvAUMmevsYtMzxRo55IDJjN7zDxM+3lxqZn8Nhf7Kf0e6tm0tCLlFdddHbkVhMJ+CI9Lv7bMYC6OKcR0dBlEgQNLzJvFY5oxiEW86j5BgxdDSG+cDUAVfn8NnYHdIyQCAGQUSakWJMyxKpmlLesWldxRkxUolWZpgKPWBuqs3OchWKoHXif1JFGHKKA6JVGvSfaG5zUMzU5CsKOHUChAYroyMrJpx2qJIkzPM6d9LSnDYbXTNC5VFUCiNPQzz93KvVVJbebqItLK7weXtYILumBOyd1Qi1yQlYkTf8\\\\\\\/lNrs6g4ancTvzMlbN01qRWXF7+Q9+SO\\\\\\\/9MhXGWHhQHyHWvxbZylPgS\\\\\\\/I\\\\\\\/95UTQHj7rao5\\\\\\\/ucahPm7GuqOLJC3scJmnVBKguPwC14RWMBWB+h3b8Lc9B4u4KMa2NISxIQondj3MslMw1we3F\\\\\\\/R5C62uz72KjZabckclke9ELmVi6IFrQt9RPPpq+rWoFH719UsPAIIhMh5hxEl5MBnhaXp9FqxgJM\\\\\\\/jVkT69\\\\\\\/5GIN1wbjizCYUd2WqFae1IZgZzqyFOauvlz\\\\\\\/QzrTLOHtqb\\\\\\\/2D3kA\\\\\\\/mFUQ\\\\\\\/AptKo4MG1f9ZQgSEMUsI1tfjsQK+f4xcSWeEmh4whht8W10UMYb1YrMZxFXwUgk1p6H1mNw7vhAISw5DUelg\\\\\\\/RAeVtqWkwTWCjLrXifqwAeWQa0No8cyNhtxIKvJz2OP9nqrimHh6fHyQbGDcACEpqkCComOm4gZp9tvxvP5Acp\\\\\\\/lMvGHeCK\\\\\\\/WR1wNO3zpS7dHpDbMSlTx9wgLMSxeJn+EsB+oLrFjIJbuG0vffS2nhvUSv6pCNrI\\\\\\\/8oq0qBR5bjB8hkqIinEuklZ36HASTnC5\\\\\\\/GdZZJne620+P3WFVpuuLfCk+oJxwgFlBSzP103ijP+Yf5avXgB5HvD5s29mE0Pe7KZFeGrlEK0hmiOuIJTeI5YQ+CijjK006fIFauwegp1WoqXRLJfBNKgNIrbCN1o4SiQGdYb\\\\\\\/y7NONQhDHNxpDvfAIIqej26EcsDUWf49KpNbIa+7TzPA6naNcxR9IQwxx+XPpMsxRU6af5lpH3Q8wF2esIxnmxsxG54wsVuABYbEaIR1IPxVRTSB6TEJ\\\\\\\/bb30r\\\\\\\/2afWFgE15Xx3FlQOdHFzf1DKAh+1YGM3DlLgKhi6YszR0AhsvSiCYfsU3+85woypyEclgLx6g3Ux+lPZus7WwRG\\\\\\\/TaeFrlOoTKxr0McH0Kvhmc9zohPJwhgEf4oBoRZsnYQRllfxH2Bv+wt53usiOqf5+Sk47GjMPQuM8vFzxKvwMNTY\\\\\\\/oMOSAhpzMuRWdYBP0lTAyasgDNaCh1eMr5eZSPPp+ChhiVTwewOe4MDVOE2JFC7oLonhcSgZGkQyhfcRRfoXi8CHk0nxhkiJUzLi4BmFIPcVH3jekSYPXl40jGco6Pr+Df1w1SrNGmx6TYH+yI\\\\\\\/EGqGtUbgdgkON0nOiLXW8pAVOhqqjwPD51Jywqg9l9p39oHpaSY33Ga4F+uhdGDedc+rGraCRsmlgpdUWJyHRyPl295sh6dXRUX7AFKOTEiEzwMYJubQg+098EEidwCycNI8ubDflYex4ViUnVc28zAzCBHMAx4ehoZEAmCreVO+Rh39TcD\\\\\\\/iY\\\\\\\/9OWwOC39k0IdQwPOI5VAkKmj4SmddePsHf1QgT5GCPXS5GC2DAaIPmP1x+QlC+7IKHmIm15ugvxU9h2jgU+DmhJP9M6mgxtphuKIThTXWacTwBQMDudPVw4cEQoHL5D3MWTrmGmab\\\\\\\/Guvj7avLUDWvAf1MoSRE8l\\\\\\\/6s8uHiArpGlXi0OAHYhlr1qqgV1vJoNaPSzePYVL1UClJjBzORHnvYddbmlYNROshEdrBRu\\\\\\\/1fmLuvbKkBI0KfBXoTEc1yTfV4vz6I4nex\\\\\\\/N1C5OP+jTIe9h+wLvrBfzxxT17hCsMakC8XOLQaVIHreTyYngqnf6v6p\\\\\\\/dB8oFcWkws5iFapQOwa8rx2XHqANLbEwDCGPXj7+UWcPhyyOQ7c0F4JJKLngOJvOFMqWImwwNDqs1qEUowrxkBolo8\\\\\\\/TyLZH6X1cCd1R7xvpqpZdfwrCgHAi2fVZk1iDiQxJUWJeU3JhvlQRELhpPNYj8Da1XmHQlzq+qFpHW4lRWDBF8ugo0PfOl3kcyZFVEGaBaMq5CXakLGsmZM6NGwFV0auEXc7W0ri1sMA9KqyyX4JEJf9YW9jexA9Wrr4TAwSzQEzmRf082sYmTXmAI8PzD6QKykQlb1datJuRsDXFSBxun9IfTGQgbvBiE3Ora47TIpPzEouGCt5N+2dV0hZVwVi+JGPCI5rSDYl4UJLoMX6xjld9tbnWQIKV2vFrDLaZBL7tawuMKKxVVNhV442OuFB7pX+OpJJE+CxjPAsyq9kLYAPLJF3SzOrz1kM6\\\\\\\/WWTdx68sQfnLwRRbSdoOaCIqo67\\\\\\\/XMhBnq3nMbP9nFjP4zO5pH\\\\\\\/n3EGzDE2ypv633ghigNSbRE5nBmwUbtWMfTEofcBetygXvZz6DpDAteC0cYjVh5KSgU6mNvngj8W27FjkkIAUFZtoZhAcMznpMQT05Ll1VK\\\\\\\/oBUxwE6ui4C4tEvQEoKHAZZKXnWRrRQysZRRw9UNDc6ICma3bo3YfywL3TZQC4KU9\\\\\\\/WeF\\\\\\\/DpvtAJVlva2v2bryRFqiFf2EieyD80Qv4+Kr8m5HQonZxR1gqK33Dx+3wRhPdeEB9HoQmevYt4PkKjbbJL3db+9nP9hYr4Z\\\\\\\/45vsLnYaItE2jHeeJY4SkOnxwZM+SSO3DSNrEGwFeYaN3PUsTTpW\\\\\\\/nb7SlCtLFHg27MlhQ7JRZHX6lx7rpH8t7jgQla\\\\\\\/zfvjtM9YXEpaQADkSw2SKpytDeodP+J4B\\\\\\\/cjkY1+VPGmr4tDTe8F7D4vqUlICznRC2X+ArGgF4TDTFymh9ccFFiXQyikmW3WUtaGGuTcj6m5vV97WpPmpGF4DfalUvj6UCRG4cIyCdU6bU7\\\\\\\/Ebtnqe9dsBvGmPcIYl4W09LZXYexDMYHqiIDPGxL9TwYLALr\\\\\\\/RLl\\\\\\\/9dA7RY+VRDycfFzF6g7\\\\\\\/mz\\\\\\\/RnD2tDFojwGjiqiYImc0qGXNZJa4jqc40NdI6obettr5Ca2l3\\\\\\\/Z4+OPNwquR59leMUzn1Y4i8xULtqfi3ZTrdzq9shTEQO2J7Pq2pGELfDaYIpBM3YUPVO8n3N5ZMIbdRJF3S9HV8rbyuVBLRIZrVDQ1kQDs1lodlNNoFrhein1jgn\\\\\\\/SJlXYPH1YfRxl7Dt\\\\\\\/yyRtOmAHvDJPRRvKBQhTCJ6688\\\\\\\/SwtgEre7kEjiPpHIpsBk0FtVkvHyo6fFbAtkk+tYzfMS1aUnSsLG7rs\\\\\\\/it58uLGgq0xU8ygOBb78bDVgR\\\\\\\/k1gHS0AwPmjTr0G1bsJwWrv7jVdcUnGp4uuM0BNB\\\\\\\/iqOmd\\\\\\\/Huhielmjv3kmkMlrZ+sF\\\\\\\/oDNcEzdpRwcv8S2vtTJfbXhSQzz0Mh\\\\\\\/Xg8iSC09\\\\\\\/D7bkkX9rGD3AKYeywIEqr+hk2Z9SvVxoWEWmapIFx4NJlN5axMjv\\\\\\\/Aghd5whx\\\\\\\/\\\\\\\/LXFX1YeZ0DUjkCoHiYLaSRMS++pQHNITowVJdXswh8sqMVcfuPQx8CWuotlLQUtFCXHHgLbc5JV5nfJHIXbqSa1NTryLE1B4Voe3sNVxp6YWNjHtnR9ji5Gfmj51Fi5tJAL3OGDYS\\\\\\\/ou86NVn5kT\\\\\\\/fNZkVrmrhg9cs9CKicKX3vTSHXZ45EvR4u5BMk\\\\\\\/q\\\\\\\/RPnV60JLkR4i+ckTYnKftDsXw3Gw2LPdG6AL1Iy7aboR41v6rcdVpRW1bXcmCSj847T634ejkhoIoA87ux0tt9icUvG2bzE3oQYqR4wUydRaUn95aeEcelLyB7xg9p0hZ\\\\\\\/rkyeHkCMo0r4wOngfePd9HCWsw1BHXUrPbN82JDnP1IcCLRFtY+PvEQ14s\\\\\\\/0Wjk\\\\\\\/OlcHdbb08fRIWDkmdHGf9kxo385yCYJzL4uFYZ\\\\\\\/9BRP6vaySMS4Anz8mERfxhg9Rp4jcG64nUQH9MYB4C1AYBB1k+xCpazTJ1VNmEhQtT9Mzbvz4pcFYNl7gkcoy87s8sW4c59xLFr9vZHgnLZKgBV+37Gr21KLl8CWY3AZrXXQN0Cf5xkit3ntRG8iCgw30KTsbl8wA3o37gd8SnAXK537uMa3Om+JZZKBIqsWuf9HRcBkqLF9eZvaG\\\\\\\/++ayUllvOCQkz3TQM\\\\\\\/2AWj7hWkWXq3fSwLII\\\\\\\/utCXR7ExDulg54J1u5kBZjedR82iaMZxgpmdAsLxtkQ5T\\\\\\\/HUZYN1RfXKlfrhs+gWow\\\\\\\/VCGIoycpG7EvLltjLqPC67sQOjewV4SvZmIC8IpAgap7QjWE0mjPCIKkvf5yLF1HIlMt1wnmviTmVmb3X4rG3CYkkYy4ME\\\\\\\/GAcB3+EgLNHKvKhOzPsdu4mP1kT5kbSaWOXIvNBhaKdwBLEq56xbgxAWzk7RDpI6dcuWJ2EX2obouqTxs8AZSduUHmelHQnk0cL3COExpS31aiC3z3YQuLvXlKy26XUm+eXAQzCDUyLL2pB4CEpTQNWVh+8zzX5D8pYwSEEgluJNEu48lGFMRHGcwHk+4WAsP2sKLDRvWQyVZ83pL\\\\\\\/bmcy2wscu7rWyj9SlzDZ1zJKhK6z5esU6cC1BI\\\\\\\/28wrpLUrFv4pFFxw5w6nA9gHJSCTcSxPuT0MUEVfEDcu703uKvs1MSapwDVXAFJYL2lsw5kz1c7yO3tMQi530sKT7qLFJfur37Hftg0hQmddJoF7i\\\\\\\/joGTA+TO8hY6ol3tfKEhfASW+Uqds0Eztxr5wmRzs\\\\\\\/WElnkX8r9mcugD0zbSr6PGF\\\\\\\/zxrxwPcExd60zEIIeD0OcNbFTRPvOix+n6ZuXZ4IKxI+eOrqakPcXtqWqbKGefDS3rZdKuSIdnjUupf9w7ISB1uZbn2H\\\\\\\/gysgPPxmtvLYOlWW4i7PutZzyq8JpPAcp+d0DimH\\\\\\\/TjuVxoD\\\\\\\/j2n+YXM5FnZF+JfCIc3XF+LwJH0xTh\\\\\\\/CNxBPid1P3jnvENv211hNwgORJY7iZe1BXIoKL3jId0pF6AappfcgTqNjjosRFM5VkNOl5K3nAjfugQ+EjRITA\\\\\\\/RGRzGf62i6fl7HeOtTpXz6oMODdDeGxFVMOdyj3nLJfnstS5NJE+JJZQy4lNWGOlkmdl9hFMtryq+WXKjcTLpdOy6sGtsTyu9LDlD1uU9BFrqKrAYr6ou0awd1DO5Uzvb34S\\\\\\\/JhUmBvGCkFsC\\\\\\\/dNQbuvX8gWfi5Mm89D71LanTNDC3+eXwODCjbC7Yp8INvb20TyQQTjNJbgGyySIqqsyzWbk4TiX7kaSCROnClwLbAsWZq\\\\\\\/xf+56oNno+9h616wKgiEHLAgLMB+33omlsqcY8WRZIYU9+w+GdA3GFef6WaYOvnRHGajlHO6qB+YTXsqDuWufrzQDEN4xeIkpbi8yN0qig+RXRx5Gx+gZeWIoG9lpJ1TOgjAqc3Ie43QW5nyE3JAYKOKnwjyXedioBvwO1lxo9II0T2RFOprfUXdiA9OYlDYfXU1VzNt\\\\\\\/k4WlfJPkcwChjbjDlgGhAhMkg\\\\\\\/MlFTlhrMsNWJAT3fODvjqusb0QpSq17T7g9oqUtvSzQtm7JcqwPFxflnPZ3quQHwjuFVRJpmwGWYYxRuTj46FaO9x+PLSOyRYjpmfbu4feIlhFEYPIRMrEydazwGKjrTu1S9sUEvyZmeGQA0YXPhk1PRwrIv1sG7HPnFm6eQ94nVBrxKhbDDmsCW4mFJkVmw0HZx9NcyjkODhWqqQsRPLn5AgY\\\\\\\/BRCHskda4\\\\\\\/8LIMGtgqG7\\\\\\\/sfBjKG58lcVmuIsNQnQnVf0PDSpb8rj6gm6CabhaNnE51\\\\\\\/xJvFxqCU7V9ZRNXff+PWoHRxmaVvPOco7mAtX0FzJTK1OnK6rSQNCYj3CGjpDXSeUuSNO3tfr24x7zyfvXcpGaR9\\\\\\\/GLJnEPbKFM5h9s1cPfaIi7z0zfZdnf5XEUcapLTIxj7IATVmaIen7NIr\\\\\\\/CBEy+iY3gG8YIK5EJz76IGTeMOBKxsBy+94LaI\\\\\\\/EK+lWqMaT5IZKyTjXUB1+Lv2+l6V+NTnsCRQFppuRoHbMAnPMwJZFAXtbu7zaGZjwxw2PFYV1UqAj5Iz7sc2cPB2YP5LfSsehjbufz7vWnuenu1zGCA8\\\\\\\/x9k3Exz0xJADNiaHQM\\\\\\\/OoL3aprvzgQi6UsYBI0N3orarvhavyQuFCIeBppsFbkskkVdNaI7k2E97LksiiORV2uDB\\\\\\\/MBKwed6EUJqRHKDN2O6vk9X\\\\\\\/CcYkWJ0mV1mCnI+1xSTHvu11mXCHhydiZcDt1MXtrhaEzFSVLT0LBnQI3YB1Mk\\\\\\\/gsoIqZCqYs1FTfDYAe0a2Mu3Jf9TkmGP\\\\\\\/Emx9wDUycVBXBFrJlmBliCfqWmSuK1TB1AdkbZQiehVPbHDtzFB5qbcbzveyVPt4PcYUkZYkVSf843YP9D8R\\\\\\\/94YmEokT\\\\\\\/TVXmwuEpfAALLzSMy\\\\\\\/X4vFpBVH8+IY8pRk\\\\\\\/Q28vn8omymRCFCFHAqGOxEeSUeezgg\\\\\\\/M1qf8Ee\\\\\\\/NIyZrUeTkYsAV5LK9YGgzTRCkCIWYyycrmN7\\\\\\\/Nxn\\\\\\\/Ngd056SDfJ9n4lLWxkkCgjMqUMoRapaOeH8dz1x1IkAAxHH\\\\\\\/LzvUqFgqQ8OPShUzjPmB3B4gKjxsKSHIoUxkbQWapbLd2zdwMEUEzQPFSR4n4Y+5viV4h+oo5zC1ZbHocreK2b8G5d0m2PJaBbH1bn3EHJHyTjSwaAFO4tcqz65O52WThLOxBsp8dtZdY\\\\\\\/394RqoID7\\\\\\\/+frzUMcA5d8+eMQkW\\\\\\\/vMGHtP1aXSXi+9EhspfKZtfxoSuwhmQK52bShOcGQF\\\\\\\/tsrdMmDH8cRRIxBKzncvdr+PHFey5Gm7LTB9pgXFZ9BjaOXple2r4DhQlemHtyaoMa\\\\\\\/yGKxbnY1FVlDTgo66OOIKLt2B90y0qDkCC0hPb2E53I+3LIY5JSk3RBF0\\\\\\\/2HS8CXAlq+ar06NpihAzf4YxPvfhKCQ8M\\\\\\\/NlPFCD98kWccKudz+fl7as43uuRWDeThoFVlsrGlXFNb6YVweu94oqzwNklSDyMgeBD+ubjps\\\\\\\/TinwmwUet3s+mikd\\\\\\\/DDl8g945tFvERzlUNCCobniOyeR+OzfNWDXYNejC52C6TRjbR1RirlAvj7aZBYN2u1V2qoN30py0LcCnBGVuciWPa83FTrZvFGRqP9+2N8DB\\\\\\\/H8EwUMdxl1anx3eMUqiF79\\\\\\\/SpK\\\\\\\/NybRTF8v3Fh1E4iRC46KhPKZcPzBvETs16CdIAqLpWXSRwS2JEUacm2n5Pdz36C21CbETqJYM7UHJUKLd5G88RCr25f5DSAOGHH3DwsEbsYmiU9aFD2Yi4sYGlvPXdStRzi44m+VgCGOduKEvkT1UfQbLQ+y\\\\\\\/nwKbJdxyZYcEVM+4oyTwQSXnoRNW8ycrcaLJ7DnnoVbnKfENFrsBCcmyHSJPt9WbytLLCoPMYvj74HVW5SQoD1KwEnG60BcBlDKYvmO+6jFvthcCGUbhg1U4TmSujpLn\\\\\\\/LizF4aZ9kfnMeifSd+dWzjXKY+vwPjUBYjfmUuMpXszxWD+ViLVLxlkWv9QHjcz6ib4QULoKaCPMqcQaRpMMMW0oqWK\\\\\\\/\\\\\\\/ILbfNe\\\\\\\/tc6vZbkLANhuV0GkgtkzQTHcaQzbDaapMfkjHLj02JiKlQHPpcs1mUvOVFyrFxYomQiNXt3KzoVI2HstreeRRn5BBxjUsGm+Xc5Fm\\\\\\\/nF4Q6vxBBnv0E7gb\\\\\\\/Bal4mtdjlZhNqkv7Qc\\\\\\\/PzugZpxMYZcz5y9sXONDvQBvn5ca1iSxoURw4kUW2RhFU7Vqg6RgHivFeYTkZUUjznqL06Uhn4DbGvDzIMdpSYjSmLMu78k\\\\\\\/WovzXjpQxKiNz3ipsQ4DUVL+BkKBCLx2h50tAenHa2RyDJFTbXKSQzAbNZobzhRUCSNFckIigwZeTUeOcouFHpmAQIWQ4KNrY32NziFnNymfYEJb+iaFMg2gpVCg\\\\\\\/wqRSO3iTGCKJi1R3t6++2a\\\\\\\/KH4e+m89s4ttZasuHuIo3fh0x7DbfwuTupvHsbT9wwcHyapdNNEY9ojGpDkXuAHqCN3pvsMlcnQdJ2PsT3LTxSZ8KC3jKbdRujFbFfcLgoNCUw1SJcl2c6C2c\\\\\\\/ZTx6tj2sYTlNuQSksBW0LgqW1gl22+6ZmWnyFpWo1Fsdg0x7E0MKF0jk45howZ4pO\\\\\\\/YigpHB41TEpyhn1Y1g27V3UJ6MKS+dcO6JtaZ1Cg0TNTo+Q1PlFjmPc08byBwaNNSoVgmDrGEz2FhSJ9Zr0f7y+l2Ll2mIDRoyX1bLxNC5SYS3d+wHg2SDxaulvMsfVj7SoteCQZE0p0Oha6xaijdyT4uHdVOZPCVpuxZFXvy0qq1glraRB7ew5OsikspPNTIIDbzyqSL34fSre4Nxnww32eYzexicpeVnS7QmZ3xz1lgJ3rZmjg8xhoQqv+KaLcNPP9ZYtatpLdMhGX64BSmOceseqz2CQABsk1jzZjaOYifnyYFA+lija4KwN\\\\\\\/ClCxYcvUv\\\\\\\/YMZrcGGLNS4pLudGCZFrvCpEwQ8ZXZ3dsR6xSLkD3uyrnIGJa7v\\\\\\\/NBjv3AMGs8zsai9FJTxFG\\\\\\\/EIWlWHwc59ITMrpG3UxMse1yoMsKniEehFp4Xnu2g\\\\\\\/faDqhWi2BKsl6c1kS3Id1bvkI1RjyQ+9uTvbxWtN\\\\\\\/OYAx32ngjfnClLqU6O0LXyiBRrFkyqr8rAsIfUPradsC2vp9G5UDrSSwg518aV89w4yJARGGeAot9pKXUIZW1+j1D3m63iHgNZPjpm2IO6tx1K5LIJPtXkA8eSklX1S0WHaYWOtnwaPAnGk\\\\\\\/Wgss+GTSjxbetuPmMT0bt+J4sxw9WHoXeMmffOYjECxkNXQFQNpn0fTCupl\\\\\\\/7QMqDAkGvqSRHoXUSW02PlGhjRaVO4I02PU\\\\\\\/2eDTh+dHNVKGv5sGp3\\\\\\\/Q4CblaiNdjTm8JmGCwWQsZUdczCnnk59eEWRwIWugmeF\\\\\\\/NMC9\\\\\\\/dvaJKpZnb2exPCDQsgyeMROb2aEWvxC2yng7wjDqplCfmJKyx8FamNDxxP\\\\\\\/XpsVx2l+zLEbpFSFKNgwUCIHSihHvFfd+JwiO9XGu5BCALdRyW1AkDvaqDEU90HEcOhdMpwo0Yd7MaRAle3bU6Rh7sHTVVYTSaTGf\\\\\\\/\\\\\\\/QyJmX\\\\\\\/rJFGFjM1g4kQAmNTDoiGy1hCSxo2UinUj+RuyC1G3ZC7OKq+b27s32lhO2Ndrc3RtYg0mZtvwOEX9bhAIefz+1sSePorJznJcbqJtiwX1HO6wp7tkOj0\\\\\\\/ek3uuVuDc+KYhazOodhqscAE4WEh0cLidek934dC+JIdyZGPZhh6l5av7WX+uttULFTKlBSyWLPxFW1yC2e8kZxBD+TnyU5MMdSj9xA9wxreHgEGKPRLLDw3zORPQGlL9gx+0ov1M3QreOnVD5s0qJer24o5T7ef9glNvEb2QLlzwxPvsqTTkXUhEfbhiaQtH4j6xaqLKE0HV6h\\\\\\\/Lz4jhYXkrfIFWiMtIYOa\\\\\\\/fZSAM4BRtsvUSYOwbxH0OSgj1O45z6aSktdOgtZ1DnaAJ\\\\\\\/+aJ\\\\\\\/qpXYJXYWm70T5pwXKMIWbRislIH1mM+pPJAeaflcmGs1tgHU+Um6HCKFv95JcvCAy+byj+xde667SnvSZay+\\\\\\\/Xg7KCoBriV4u9wsJO5NFb+vXEv90bXY\\\\\\\/S8U3tzqHSOXVGWXnfvrXH3Hqn6R2j4BVanGu1Er8SpML8wPEnN6SC5sN2SzeoQPCC\\\\\\\/VJRwIguKrrmqN8jEQFGGjM7yE\\\\\\\/oulwrO6BXrUqqV6YD18pXRUQurECmZLONr4eJFywJPW6UTCseoCH3iuB99k\\\\\\\/5FTAGsMgJA8q+f85T1XMErQEBi5ViaJH0oF3cVv03tVKsdf4pmhBo1qxJMVXpPhITqu8sy0rz6FeiaRp97AIwQpQ6eu5kip0ymixcdcYPHYj6t5pWLASJYSB7lB1uSsFb0\\\\\\\/bsFa9uIxZZHmNfusrFgnp08MOaBzSdho0OOnAupzM1b7HjfVkE4lg\\\\\\\/k7YQsi64H4QnkgCdOEcUAaldCKHLfld0xy5296JHE0ZVvLBxICi+3vUyuTNzOUJ8cTkvLCfxHu30I4GPrzJ1gmegUjM9gPOkfnZ7rkIofj1CemwGbuD0vSpccWUDd0c7fg\\\\\\\/lUnhBiJ4KZWVHobMVZY30YVfWKjnOVqwZmYJhe\\\\\\\/O3pvvEvL9U995SdPlOvCfaBoWSqfgF0DYLxFfQu1USLo+fKb6hQ5B6yZFtFab0SU681fPOM7sgunZI4FLJsM\\\\\\\/HqdZUnuRcv6jM0VusV0zv6lfBknI4jOGSutpdLZb\\\\\\\/bbmY++XdIz4aLvTdF820m39nuD0Yt6n9+RM+9gMFYE6PPqimU99WgRoV9V0ybM3EeC5tsjVdgBTcNEXswdJnYshwYDgAnxgjw3uqCu9QFRmNMtGeyYyico7K0x2aVtz5+N+DpATYYJu6ed5OSF5+eNt41oAflSJcoeWTAodePoArAhmFe7RTOAHu5p3ZIBuOmUnc+1aK8JXjA1bS9vqyLKPvrMLqn2lbBEo0NwR3fcXVU5Q3hV\\\\\\\/PPFAyAzWhwWc9LB3123d9M9\\\\\\\/JyKcnkRILUQmUtiCf6T8gKdJsChgeHlVv1Ia8L\\\\\\\/t54EuN1eXR+9qUJIRqG7199ab+UBmj7N0U2coI+sC1KR0ishtW1TzEYKExBZkqLV7kzkW74rJkUafuJBUngUZpA0J\\\\\\\/064ddMbhtC5oR9I5YrMxiqXqL6Vm4b9Ll\\\\\\\/EDw5hedWZuTDalAusjt+NVB\\\\\\\/3OpAEqg6XmfxsLjMoj4XqTlHM5mSXYfeQqfL9ivhut8qterGmFUK2yL7UNxZ+Q5wFCXmeDlEa9cKVZE22FjKh1SeOWXUvcz+7IKiwekG\\\\\\\/dkLLVJ50kp3wh\\\\\\\/9LxHG15Qie8v0DKcf+pBVjf6TfuSSAxlRIWvpkNFUqEu9BG7pM2BCht+p7WANRglMh0cur23BpPb+42rC1hNLKpUEv2Ewo5\\\\\\\/XJSP8HHl1yjyU8m5BiH2dVBHrBg7k6dPBKE6c4HAfpobCUkqV8Ulg1G9Yx34B5jPJbzfrjqf5jiUnQ6WBRR6\\\\\\\/OtuQ\\\\\\\/Zggxn1gzEGzFAyOuNVaYbzUAYjyGqsWm9nl0XHm6wFMqcJkzEfFVmPBWwZ0+gGtc4EQRGpxaObnggBoP4ECY8rhTj\\\\\\\/uDmVLkGB3xPnluNm0tCUIKQvjvHM+8M0v69wq2bvt0cwSseBIwXGJ1H6NOYwUyrzCN7M6wqiNBcyL8qNBbaf4VnYXRzFQeqwAehvX+HPKPwEz6yVQjkeTmNxtHZTF\\\\\\\/nMPpczkVQ1pbCZ9BU0+9z1JjU+arxx0P9GEG4v1raO7C3BD83hqBeVsyHUGpJ7BKjKdzBnYfHgdarqH8C1aSflJt7gf0B49PYBl3AkgTpr68mOOKkZK7PD+YRDj\\\\\\\/nKyJVdGrBS10wzW7m82ddTfI+ppSGw986HfPHbnA21g\\\\\\\/u62y53aKQjmqUXEtHFKt1yL+jfEFUMp1vdyPrWI+IUVoqkRwJ2jriu2PYT3MoTENY0rqyfjKDFDvU1ITi1BnV7lSOi\\\\\\\/XTa9h1zNaiG7KFpNqEcr54TtJD5wjeb7TRK1\\\\\\\/wEGi9FaWcDlgKLnTsYadF9Cbo+9Qo7ye3ypOo2nPjOQ7iERiTceJ6HGQAMp6XF3m6gtDR8fnZIA6\\\\\\\/Sqc3Qzbw2IMnXr4wTNuf5b8jIk4tH5gjRaxCjdXdc4DjinPdSAkJOw3MLUQD\\\\\\\/7xWccs7jN7eFzp5tcJaMUFY+sou+e3BhswMgc19WWV8RqisttnFUGZjhewctldOwTNckp0c1imf5YcvVcaXDoqWq6OZChXlr8z5mWje9DeCURRAufZFPO8EFKyRhfGYZB+ufPjfRQRIdLjNaJFRCzEkE6FXomK9RSX9eqjNx3OvAlCFxCEAE2Ot\\\\\\\/jCmzW5SqAEXn0HDJ9nriaQDUvx3XkKL4ziSa8B7LHL0hG2e9DoIEvCEU99taMb8a3faV\\\\\\\/DIrdFsfR2WtyTCGCNGZFEn5lMUk8dyDwR5nIwIWI4Y+jHeUgVcJHFo6hyElZGTSQgjc30hXjngMbDG6RX8\\\\\\\/lEERoBQDj9R8pk4AB1vxXmM7fE3GQqateNviBGCV9G6RkGPdjhYP9rN0AiHOpdU5pSXm9iJqBHzSSvXtAH\\\\\\\/o0zPMDk7tPwNuWOt59R5CprYSddeF6pFx+5mDGwQN7s67JoAQXIktSjNDHdwmb68GkwNNX5xuXCNhSMD4qyP1nJjY4PeDe6Bq8yY9DfYMWq1SJUTBhVJWThY+HABqeuWHRgF5n04NKJKMQw6ARS2a5v5fWekJsfmeKpRqsP1xgEJhJ92lQI88XpX5yIlK4tZDduN1MSwOheGZpgYKTg08raBfr7UriDCjVa7kGbgM7rAgvc\\\\\\\/+VRGNZoU4NyTShhSq1JFpCY39L1fqrFPiA2dlQSUA\\\\\\\/n15NEq8ESHgYwRhyJ8ArSP4egbBC9cM1YKqEkiYNfT8Y81aowkIsVk+pw0Vcx8B9MBZJGZfUvVNGVyQErTAWtirJmauTGItE1jj+V+\\\\\\\/mp2oli+d\\\\\\\/ScjjU\\\\\\\/eQh3LVr8qms5Ge58sRoEAKCT35zPoRNXvhYXf1cIyD\\\\\\\/6e\\\\\\\/0c9kx3ouPwVctPjFVk0MP898wSDGbUzMwmrzPtxrDMIwqWyb9HLKRSSELg\\\\\\\/e\\\\\\\/LwuXm\\\\\\\/TlwjtA8ce8VkmgFkXKCiNd1BAog+Vgee7dCBGObdbnR4SUoH8LT9s\\\\\\\/weI1GH6IHfuSRAAPcmcJurqfRowe8tyFbTFmm50r9qWeC7QqAkDXNWpWJK5mCKlj4OLNgf25ADftgMoGyOrSp2ugA3nV86FkjUfxEurqOq1DPdhXXrVqBTLLnOdMAgv65bj2IJQWHYd401dhHTDwF99b65jmElXip70tgqI0eg9a7K9yuo9IPsKsfcpnWZvNGq1oN3H3kqwwxPBm+0rgraDy\\\\\\\/E97e66onDst59u1LB6hckW4Se7jw+KKsju5CWYXorZohN2I04utrmi39AIWwUp0wUvRpASPa4Hc6B6C6QlXrT51QsQbQ5XjT92mOBFRgKDYiTPC5nSCZm\\\\\\\/sp7awUJ8v0kktLRIrr\\\\\\\/b2AA455S8hfHzRAoR5IGo5UOg0+TcgEUGg9Q34kqvQyWY1keeblarqtilV7uZG2rDjwnM4F0eQv56oStK+G1X8Pr\\\\\\\/Wfz2+lhu+cyw2E2K4tjPD8jg5kbIceowGW6E+MqrXL9KQnEzb6qJC0JRDZdBxcwUqitisGdn2e4KKivtyUW10+rrQ+PzVGtv7R4pjFhXB68RE5aLXJOUL\\\\\\\/GeVLrCttbQZZLK74+vF6+Huc2iweZTnKTeInx1OyFA6+7et0nT4ULtnyy07Ol1XqOBOIaOK+PrMUV22U1nC7XLmbff4L\\\\\\\/YD4WfoRh+43Mx4D3ALA1lmk2XovVcb9fbyM\\\\\\\/4ygDTYD6CkHgkfL8azmA\\\\\\\/fpH3PnxPOUQru0oUVQ90BNcgSzm4fLZAH1r1DyKfVf5S22J62lZqs22e7yKsPjLwpUC3jK7QbJuaX+cDKQQuYFkgQovwzoQS4pUrldkcyET3N2Htx8Fqkp48sgE5DFz6FwS+bTb5GYfwlosusgRmPf4KpzFP11hHq2wwHNUWHkQaw\\\\\\\/KiipHGS5+p5BJl8gpM\\\\\\\/jdoACJTx12rmmGAjoQaPImRs6xdTzAU\\\\\\\/tkb5naFyaUmy+3jPx6ue8h9M3IqpQsH7Cq97h193TsUXofErgAnUIuBg58Cr5hZLm66WnWju3O83gh\\\\\\\/OIDDX\\\\\\\/QG5ExeIg4Fhrm18mTdt3uihYs0D0jtzUIfaL63ta7L\\\\\\\/pt6BJBRAWgNhSyQz8yaSaCpdGuYELk373SEZc\\\\\\\/DpIo1NzEU5OxG7KAxCiPinMgo2Chpx6dg6jyNBEcfaFFXsp\\\\\\\/jwDlj79OljtQaDJ9VAXZvlWWtt8mxhM866KWCAo8edcbybw5UsxKQZHy2tjRHyrZ+kWYYsPDyPnjFsqumZklMmuDpmqB5e+sqTLp\\\\\\\/QgIcm1QHKtnNSCbpFYPzWnIf48Z\\\\\\\/iGkaTDxGWvo1lTvSw6zZb\\\\\\\/PeDrhtogFpWW\\\\\\\/Ok\\\\\\\/s4F7gNQxVh0xPTGvkkXM3\\\\\\\/d\\\\\\\/NzvS2RB5LBTwohhv9C1FgSY7V+PTE5iEeRyFyk5jnl7nJ5mJgUnN7TAl+WDccW981D8yHYx5al4uBCoQgGHI\\\\\\\/0Q9aBilywrisX8EWbeCyMA0lvEukaiJx4ez\\\\\\\/9uM8kbZGzVP\\\\\\\/EpWjbrABMat8BY2uZHdfqz4zzcYNTZZ8m3jyAR7HJAmdVR83gLPUWMZXkKYv50e+GvWzXlsx8XfRj2+T8IhsSu4SQ0vq+19QRViILL59d1vzvbEH4+rESqghr77kODYLv8uBjjSqP5OiSMDHtKbIAjlLO5e9FkCXzPSV57GplBDv7Afr\\\\\\\/YYQlAM0qfc6sMxfZtXaaG3G3Hn01J0qcJCSNtzlH1Ue7HXa4l32Y2YFCAK3kPKh9tvJtN3HXIB1hD2\\\\\\\/bIDtXWfl1qVKiPhfSzfyW1JIwJ1\\\\\\\/iokbjZN9NOLxPgODkPef\\\\\\\/2CkaSH+QDIfNAPTjY\\\\\\\/\\\\\\\/4hOTyiaHsDwoV\\\\\\\/y9sizGX0HuSA+LJRlYaHvKSv5nqGuhVGgjz2O166\\\\\\\/20EDi4OwHOBJp+fzidLHZ8+j+r2UCTuu7Ooblo9P2mqd1T5QbwzJnNdTwGhuhhzV8csszr0Qlks7tPeZ5zLpENhI13xjiLpLpG3TkGR2CDVJyXNq8+cT6Eo4kBXGKSXo3Irtvt0j1g61xYyp4NuYzraV1+MTbvhl68wl\\\\\\\/\\\\\\\/ZwHUNDa0o3Yv7UmtXZww87RwBq3e5HeiU27WwQSCP5dAzJDLdakRVNRNlUSfQiz5EvpLsmcFuKuifJSsvPqniuGpSVUhUqDuntrKwRhyFuH7iFGeC2m6Qb44YBvvvcDfEDOvT+epjdxwvT4YPbho+fqQbcexGRcpSIdcPAN7MXj0neGJ3GoirrW1JdPiqEHWJuD+gQfmFY0QCbcXlqJKIAljMBjBC\\\\\\\/YrOFMXiUU1pi9DwtPqXSKDPXzZ4GPpn\\\\\\\/+q1bjseijUBVZ7xoO3nDY+tCrnSj8C\\\\\\\/ATBz\\\\\\\/5MX44NQPI68XZorOc+5ue3CiymVh\\\\\\\/GrW4hNkrorIJBcqPniNJrSRF7YSNp1Dn+je+3sdfEpVvr2HVNoMxA0BJPT3BQyKHrFzFI197covT3UySmWz3Ghb7oUTGvMqEcgeW0nc9IOgApcwCPR44BiQmw43TOd7yTCXQmZUH\\\\\\\/b0a0KzSiUVK2Tc0rVu41wsagMaX1kaRBJ2Z0Z+j6y2rbT9XQ8B6CrmVeh\\\\\\\/2SWTwuGV8+QIQR\\\\\\\/AAvAKOQv2R7S+As0OkqTrxn0OQGtsg1SstnoYQEVUEks6VWqKlUqV8zZS8fJ834XAKncoKrGuunU4JatDT5porkRPcbGOKIBCy43m+nGZIovxqRUGN8rCuesGpQCnisgZxTRtSqzRN\\\\\\\/FhfZ0ghFSyB\\\\\\\/\\\\\\\/K40Nhk4+Khs\\\\\\\/HeaT8DhVYz6iDidQ0bveWBYBwF9PSw3s0d2zDls1H4aPacciaweRgEJxp25vpwrWOlKmA5VmonMBHxNNFk5ZTQbtvo08UD6R+dhQ\\\\\\\/1sVFbf1+w\\\\\\\/UPdCY6W2EgCypTfoAP9kMdwMi4UyVVDSm4i2P6rWnpxO0CPYQOQLU6BN94g4EpjEJuN7GDd\\\\\\\/6p1QyhIrqRgDA24VkEAKpQAjR+4mHeCwoXyDavkeMNyHBE14Rt\\\\\\\/yjHr9Z6cAydVTa9Sleqpqt3voHNHUEzGy7Qui9f81EIGSPnvy19dz6EKodV1i9h2nIi57p1U6pcPQrDn5dElcNnagV8HO6uG4nH65xb8jhH+mNYl+vpdy7UR41im0Mb0nE7fIkbfxPWfMQ5th3sY4wJQb91vuBNO9Q9nD3TScCBAk3McZ\\\\\\\/VwMJFzyejtzDPSAHzCFMA4yUzRr+8Lr573IkQnyUMpVYQsi7owy0Cdj+cAgkHguHLQxiyyfnOjTopS6SNNAmRlZzxbXwOTqsfOnc2SQsk8N+wuIALvMWU50A4us4jDJ1mbl1cmmuBWdLOVKGCfqnZYMSf4GNxkPvVcHkYlP7f0ydTfWDlTGgqc39KLsVcaxTyoS7ayVdTXVhhWjQJ7Hp56H8LnRIbumLllsRx8s2kTqPLriZi\\\\\\\/4V81sM4wZGePx8dVJNcvqqI2iTNYAw\\\\\\\/\\\\\\\/Xic1Y3JJxajQEzABNidiVlC+L3FncOGT6Hxau33Rxc40zHhIW+HrYa64xsQjVm2vV5itNI5+vw9KIjQXaOZJei4wWksw1YIaEA1rSG3PjyHLCS8IaW7hzZkcfWO\\\\\\\/gUvz3kU7IIYAPn122lghCu6r+WE75PuadQUs6nhU6pd8peYfpGcXGJMIB2oQhYsk3GTzPZq2J7QQ7pjTJS4M4UZ3t1jcufT+3vruuVpWWziRMKzo4CKC+VkRWuE+sK9aIjnqzsenh1o5ew2WzBWBtCPVuDgmp30hexSYMyORGpZsHgp23RVImB57V9goUdf8PFlIDttDCGCYMDgpz\\\\\\\/aFyPuiv8vUxDOLCKHX1lH2O1MKgHXkJZwV\\\\\\\/gQq9YrBA9PEWFsxmyvbDGSwXpbDKfH3m0ddRby2X3EqKGavumfVOnqA3scNkDWtsMdacnzFbPHtyGRGSPZLkJ7ZAXWc\\\\\\\/3exNB7cR86olXda7G36wOH7f\\\\\\\/vCizMnbHqVdFtoHCYjakCcyvT2d8aRD+DoJQCAq3+gsal2kXX2MsBNlSriF99oGpDbrejdbdt1TN3uqtnPcEXiqz0Uncb3W8FFgK6m6qZ8kY4hpXf5NG6CvZ0Fg1pq26X\\\\\\\/zJG2\\\\\\\/+DLrcMxPTnWqiV26eMlVDryqY2MEkxes7+tlidpAZnc\\\\\\\/uMicE8BN1Kdn7t6o7c0SPtCZY\\\\\\\/8eM5Z6+CXkcQDSG6UvQCGRxLfqBTOWuBX0nc8PrXN51xgMbYvNOT7EaPS9PlEQ2NrsfRYVP9hll98lntmg64GGqYgh5CKiHVY6GXgDoOfm5p6M6ZMalPaHxnFXu1ArHj33VAJK5Y+xvhbY38t5PrMgrTQ1HFalvV3C8OV044ZbClOerg5OZeYb5ozYHEuizLcCCvxtuKBD9Z1VX6wAh5TOYy5Uz6CJjkAh57huNbvhl9ngMEHldxcX7nOdeu2\\\\\\\/g\\\\\\\/K\\\\\\\/4VQTrf2sfFddvUKsdLZ+OnsofAwISarZJLi8u3hpIcNaL2iDskHIWgOQ1o5oMXHmY1tw3nObseZ+bh\\\\\\\/7bonnlBG+7ZMMqaH5t\\\\\\\/FkI55sXJYk49VNNJmvgmBe\\\\\\\/btqqhSCuNcEbVH0K+xpPrNT0znq386e7JC7B\\\\\\\/t+hycN3nPIX5li9oOStzXH59A0WpYskqPc0cDef7ijgZQbDitQCnF74NqSlzeYmUSAKABnsBTDHS+m+IscoHNbAJUQT5XlccTfDnGmCQJgvnFQc89yX2jNtBBJF045szzYJ7NtVxgD7gzvcS7HHdVDQqq7bt66kOoUyQLNbwWStYlV9nhXyMcT2Y2UgRHp8UYCrrRTslwkAtwu3PUjqsdRsB2OBZbCpxz9XyPJ5229gW0wgqkf9JhQC4qYOiy1YMpuQG2eNj11VRSVFVXxzb9onpMjkIEMo4u+PlAJKL\\\\\\\/JSdc1lyOYAqPon\\\\\\\/FXriuIXYDSqZfTHIvMRswvVUEu6L070gXnR3c8oLUbwT8FrvXCM8cmuD\\\\\\\/HIwk+5kD4dAJVEV\\\\\\\/MQL\\\\\\\/4fI3h5XTTBb76OGl3zde5Z+3GVWw4O\\\\\\\/nqg4StU1Ua2wCKfmnzxVlsfJDh7IBp40DPKveoS+vTNaXx6kNjVL71LkfspYuCFuZr0RYWRt2kzUjchQUd08v3wNp55C0MBW2dIsOrRPn0\\\\\\\/X\\\\\\\/lA0fbg0\\\\\\\/Cq4Z1xwa5B\\\\\\\/yhxz0L7z8GVtH0ZhiNhusSUPPchMyL7YC962\\\\\\\/+P\\\\\\\/6UMe4a2jvpIqpg8Q031+fc5Jx4DePjwzKhh9bkwMequZk\\\\\\\/0mUYmU0FrNdKnegjwKSK1K+JpN\\\\\\\/lVs1\\\\\\\/13s0vDbgE2Blpi87OwBTqHjRrdbrG1LNV70ZFlR7fIdPY5C1+hP1OBGdL+iv\\\\\\\/t36AlEpGPF3ZdFD+rDt0S2PTFXzhfxJd4jNSNElhLzDpONbL38PySGEtrs47rlKnQSY0A+7B8HW1X0vtVyFhPKPZsqUyaPsukSYyTX3MVEhBs2YzoBJZ+ixxnL6LiWoKfCRWl6CEbeTeUlFob2I6jlNS0q\\\\\\\/Oo+D\\\\\\\/0wU6NkU3IIHfWbRLra3LQ8YAYsHLF0+L4FE9HkXm7vLaI3iMpkJUEGSMk0MbIpjHo6oDIxGfARU3B77he5AUsoaVIjKT6Rvkivjtp\\\\\\\/IARQEeYidFUkXwNJnAO0ssWUZcq9u\\\\\\\/7F+BF8Z4fnH9FLWlFJn3Q1d7CokoskBtP6Vlh85l30o7jEXy0h3j6vWeRV6wBRjJqLRIObb1WdIrRRBjM9P6ZYscxDThkE\\\\\\\/S+Vhl4BQH66SW3ghm8EvO0b9JKvkHrytLvuIrzI\\\\\\\/eftPIBNQYOOI5gOqzKv3cvYPy0L1AUeshRSIIIeJd\\\\\\\/kwGBfmrF+bZ+YCznPeVA0EvdZa0XA1tJgUT1iZvL1gay4MxhzYJ17D7CWRulp\\\\\\\/3REdqH\\\\\\\/TriS7j0me+JilXf+Nc1+EdnDHmKgfSIvzsVZV57F2ukItHZO9ZiWFqAy1LaykE5ravXbj8mjYUK1tya23UQUarY2YQCmFeF57p3ZwyKngEQai6Vj1uIho4vHHYroMlqUJM\\\\\\\/qZ1CiKfeZUC\\\\\\\/p6wLi5NulN33unPrmqdzZfr9VZyH0+vkJOfuqDVDXsKaErqQ4FZl6XCHzr1B1WZ6QWi1QA9aGIAmfEg5L0pQncQiiXOasO\\\\\\\/+YsOEr9CmVb+q4v04J2LWhMUV5tJiGb+lpxFU1CCodw9o+ykH186owzO9YkYT8S4PUWm07e5OVX9LdTEvGwfc8IC442urFhol2cL8bKy9WAr9aC1T0NgzLfNbunee9w+gaQbptgehVXY7uqscFOJHz6\\\\\\\/nVgrYiWE3P6+L8ONtgPWcLVxWyrwbREwe7wHJFlqFTlKfJ9NiZV6B+uhhYyJY+Pw4MOWZ2BpwJyHdcdxyUT\\\\\\\/0Q0nmR9sMnPV4xgsBpec16IwxQSBN53oM7Wi10zh7xbXnwYP1GON3jkoW5Y+XBXxcLwfs5zsHAPapLbz0J8IE1iZ10SfZrbpDfheQvb+HXq2AW0aYPa7IPsvwm1uou+feToy9L6Sug+qSyaGSJELYOBeplZFgd9UOmmrEBK02j7eFlyn9Xlj7IHS4KymGpyCpKPzC6sFRgk79693SlbWcgL7XeN2THMP59yP97UWee6dr3bYvnSHuJhlqwwZRWERxlV8x39q2n\\\\\\\/SGX+QxXTqLzZAFLZOe+cVoSKJy2LD3RSZcYJUnJytEx8\\\\\\\/mEIG6JNus80VE2+iV3zEUSyfnDen8A6K+zcJ655OfLgRluZm0+81X1r6UIyo4lqgnv5tDTbU+900HnLhZZBZkaqXdDCvoWpGO\\\\\\\/gIUojm3o3C5Q3KpSJemOz9qjuiYDohAXeUHoqJGYuttYyfgWPhbAJj0RpRVVMIRIxb3IWjyG6JzfQojtMmvs8cSI+cuGAa3br9UjpS\\\\\\\/ANbG9JTbc52ZeQ8XT3WAbXDiIgtLwFmrd8deW7sC+UTAllmrFZQS21+2h60WivpysWo1Ua+WsKcnHGxvTPiy7LhhI9crEyXk77vawaOkAS3aFuEI7Xi\\\\\\\/8bPfYU1seEGmK\\\\\\\/MTXb2lfMiBtW6JTP5vb33dWvoYCg\\\\\\\/mLwEVEWkLsHWIwyiuwWG\\\\\\\/fHZ7P3HwWjntNgpWsCcOiYahdzfc7OyJBieU1EWhjjDI4dJ0X2znRbtBlH8vYQv6mfvN7moOgx60tS2TJ5NEyzbyVPfsYaWkDHgDpiQ+ZKZrWDSN1cIn9J+BQCY00v+LU5KVVcADZ+KcgR+TEbcawFKmKFDv53j+HcbBOGTNVs723Gy\\\\\\\/49JIExK2sIDlkOtK17wIrgGDwS6Oil7LeAfu8Li3g0JKJgTzs+g9LDR5iDHEg2CufgPsQJq7W070PMvgq10f633l9yS0\\\\\\\/CMnxMrqLqM0bcJEin3lcoDfUxeiAqUjYHrvEdZ8VvVd7UXqsHIUeslTkbLjnBGT+95PZk2Av7zxsjbbPQ3Mu0mqqseIZ8RfP3GlbMoUnnCbxTH7sNbO8ykq4jmZDxNOMdQcC\\\\\\\/f0\\\\\\\/9edLlY\\\\\\\/CHf329hcFKGhm6CWhWc0s3JiyxMDUPTpIJor+AUt275O2SEfy+kFCiExVe7ZQFFVmwvFTDqfxGTiSkO\\\\\\\/fVhHaWKRaBVjL6IeKhcwwd6e0WU6MAyYn3LIlB+A9quKnpdsn6EzmbTeNobV75PQPtfr50\\\\\\\/SD8Xyo8gDtTx14eqK0Uig9WwFdV+QPp5zUMVmxIpZ1R6VAPrKlpnxkHamQBI7EJchUE2sB9W9IDE4NC6Wknn\\\\\\\/gpHAJBHUOLE5BxJVn6\\\\\\\/09pWzYuzCW1wjdfQtg431idaHsRZ1yXSIMM55pXQVV8iC1LamVDRY7TMS9EjcF\\\\\\\/ORt9olXErp+uMG\\\\\\\/0IcPBK8YlTy8dk7OsWIs3D78QX0OybQu\\\\\\\/RgCV3zw3dDcKwhI+oSwRIw1Xb3vX+1TAh5LQ0zlvRPnap2SxhBSRRXyeH\\\\\\\/gBdsq1g5raGIa3g2dqrGmojfx4e+fA7GS8SkKATzzOuUcoaHXpVbz8zpvFboF5FFPSvViqJBR+CFgddchrCcD4UfjHRq6Wo6kkcHg+DdAdvVKuyeZJqdC4PeFNlnBzmYFDZkx+a6lOiXRDaJvEMlHSO6KF\\\\\\\/VO\\\\\\\/P9mWdK683ZLqBcg3+jQeeapLP3BkTIiynZSnPSXuaIV2dxM+aauMr7sK04Y1+wDLc8nUzeZ7PWkB5hJMv+1txKr0ATAb5mupF9v8H\\\\\\\/gSVFHQ2RnbTg28FuUxYzJlmaeKRZBDcBIglyqQyRhlOD+pWk3iIk8tIRDBemTcgdZQ1gTtluwdK\\\\\\\/+LgABSJk5xT9FsH3F6OdfcWyL5vhS0VT9Gk0LbAAbUUhMXNGE+KIMRdWJGWzJx6NzRZPkzPMkR3TpfOtty\\\\\\\/mVdYFHJ7jstVTpufgncKNSvPZppNzFCNeQRTZd6Yw9RxuJst2X+7uOmf8CvOXJ5b0NB8LxGLF3BG7erwqMb7Buj90npfkYlBaj0ItXpxGurUY+o1FQSSC\\\\\\\/LYbk0WrAYhVtAX9B6h7YiHZuRgF5rWIMu5853p9NB9N4kDm0r4tC\\\\\\\/o4rUcnx9ILnHJJ3CnTIg4JDKW9pd9nhgCGsc72cHjwBB2oG7776zdI94OsXLq6Do1gyeeRUDZtn2j25K5lzVENSLjV+zH3DnciVyufOzUDrn+BeNYPVBzBq7hwOQzvb+MHCS04UfvFi40BkzVyDUxvFp3u+ZzcWwVxT8sMtmXmhlVEAgUT4dQCVZWs3jlo5mUlraJr7OTnXEXwh3u0LqAwu7hRUZUVErJnSx65uXSxpVwcuX0HcXsMVDReas7h55N4gZLbuvXuDMpMAlIuOTUH7FhowiHZOswJZ0FWTrlMOWVtPLul7b200DLn8yGuS4qIOF\\\\\\\/3BoSz6QlRfDInfJ3vKAftx\\\\\\\/ZFShxSYjbPesb+kE8iwsRQqZimXdXlYM3PSE0EQfqIqs0TfbfZ2pRP+96sQtnzC648L0aK3wXt4kx7hLbFFnXerDPs956OLbAYC5alsuPM8wm0JXepoz5bB5cUAlDR1O+sGl0i19iYFegMglUqRRgLltQT8K9HcgAxz83lYKwZMcKPKtkr5WMlIbjMQxqW9BOxQ0ovqzWkKBYTLkiZmSbn2BU1\\\\\\\/c6ued54j5J\\\\\\\/HtoKiKVAwz5i8fs5J3f2XcVPVeMOy25luHBb8WftTwGrNVGf4pUFRz54EL+xZH+2nGigcEQooUl7IoJ0zEGoeKzebgbA1STTIzzaWzveoNjBx1kd7m1sIjJk4+pyi2fwSb8+J4Daq+MArH0Oq4g22HGdn2rnCUIiVYzIi0DRljA14afmx95Y8lEGjc6ODc7AZbsOgQevD0yH3sCRF1PKentVP8kCNb3ibRubRGkvvZ2OqsUvRsouKWb\\\\\\\/\\\\\\\/Ysmj44tAFT7gd56hg0hqnvEZR2oc78ANVYWG7FlHMGYa6lRB8vUhZ9TpVtYVeg2DaO8kIK9Q5JkMwefe6YyHti1q0AjpCnB8KTocUeGjLD07y5HDNrAxiluggX2EBhmtQ8ohUeAkcjtxHKo4kPXPr6jls6MOZwvHBQlX3DmK6qansGacNGo0360IJad+V2qE5AW7dqGwUEHB9Rv31MQfOcTJWMRSEpoafvDAbiwsD2FENuvOWVXC9BAgnlRJ3gUaIn3Bjox9n7RPRUmDrm6qZgxL6r5A2O4ei+mVXFWqA80J7a7fjBzEQPNhbxlm2XMp1lA5p3xm5KEVcaM3AFlLGDfiMb47Q8cxrfVCmNcviTlW7+MpXl0KSsuLhtDDS353iZ3ObMp7Y56YmiBpXerbfStUhNewpxVLYsSOFhrqX\\\\\\\/+bycE65RRJ2yf9RXtd+9ymoOmzuoSWkhDjg2OUqA18tKTXV8qiYEWMGndOjz4R\\\\\\\/C5QrWuem7E005PfqsmlSwHBiEzt9GTeNU6VXV0vkPhhB0d3\\\\\\\/ULSOlFs7482nPMGv0SMSCkUcVAu\\\\\\\/mI0rxrEC+clx997NkaNGCzfA0DCEoPXWq0OIXdcKylEIE3tucdgLmvb1L6cWRHyiESUys3HLpj+0MAMiZeu6BomUNDo584w6WFF5oQIAIYXiWhX3i\\\\\\\/I77evFj3tTHWbeIc7lVkhTRDo4F3UM6PM1tbO3upCNVZKem4fazHUj8II987DMu4\\\\\\\/O2u74wA\\\\\\\/\\\\\\\/l9dmakGVH5+ybo0txmqAfA3XAv\\\\\\\/dvuqB5t0Elg0qm+l+iIhyQehPiFmCQyWWAdojplGcnRyAK\\\\\\\/asuq1Us7zVFODrpjos7kIS7HJn9YjbyH1wF4C2yDYRFUh2akByQKynK0gj4+ioAqkbaqSkq\\\\\\\/GSxfcajjmmMdI7a53oXAEC1WlNaZLcKwbanwIwMrL6qw8siw\\\\\\\/2LixvvPWzBdo41srHXMYJf1bO02Q3jMNhWDvGLOMnGTss6o1ATUGvkblcGKBTLt35vnSS21wHiw19tsb2g2bs7Y1fLlU+aLuevSnWXr\\\\\\\/pVIHjTcguquKImLhMGZL5M8NyJGxpzhPKgw+2EOHTBhrjlPzLs6KOqKcReZJKdSmS8Eg3Qu6MFgKZ9ULOJEFNe7sqcH9xW4JaANqN8O4YHkBUaRhKEaqgR3JCVHDZODcUhQy5xWjYHzdgxEXqiJmkvxsCFH+RShKAst3LcXJFTx\\\\\\\/g4IJexKEMCQyxtI8YGCD3F5SlijUeYO1JI6SgQG59RqLvo9HyY5tOTojkCkT+FXQue+MP9qkGvP7wXkNnTXbzpMX0LVqqLlT8TyamoHegzlrAdU0BYHWlaZSVpKq9jNqJzyhazOFP6Y1pj2oqPnu0YaLh3KwllOY9gnIh7\\\\\\\/RYoTYWzJuQaEXav18uhx269+Umh1Rdskw4nJYBvBdsZLKRRz9h2WtgQUbZmrouj6f5380gzK8wJ+1BtEcAB3TazcdYFVjo\\\\\\\/dKFuyC75qfZDC1RBoWi7CoXEHHFBrYSsozxPmpi\\\\\\\/H9W6F37PR1Jbr2IW8JpLRXSSGB\\\\\\\/u4S3ReLE1zIJ8XLa7h7iZSQ1TWnvHn91ElyD6g5sRXuD0+N8elP0XxsP9H6DeZY3N51Qbz76l9ACBVJPIIth324vVZu4Gh2p3byYOhXULuLRX2clSEMo1D+88TOb69dKtL7k48ELhetW56cs7ph7uCZzvmDFxiA9\\\\\\\/2fR6ADQIvn+JPGmi5wFOatNWPXEIB5dFWdTz6mo9TeJAhS0Rvk9LxOlZaNTcKnFWKNHsYB3vt5BhRWflYmb0cir076XzzKhKaqhu32GPR6eJqUTGbRwJx\\\\\\\/w1WwhEo2p0\\\\\\\/Xtms07up6inAK87eCHa\\\\\\\/aebmF735lTPnm2D24U9kDNhjOY\\\\\\\/VVhsEQFmAiU+kdkkW+JCLtjp3K58wUuym\\\\\\\/bUe9ElXw0MdWP9zqWvGMBgegLKoBpBCgH5X68DxORyNi1jkyxYBjJBzzyyptJmzI1rZg\\\\\\\/xSvxUwb9s5vB7r4gh9PtzybkmylzFdyh1YqCXCSjjzVoHcB4qTtm\\\\\\\/OYo8xvxkuBqyrKrBCAKQ7GqqJUBfyA5AHqn5ni77YTczaINfsOxgTtNH00U43HximueNle+eXeTEcsE9pw3XrKNQBPAl7ZsaGboXfDt5Uefk414I5ktDRPYPkSVOKn9mRu7UtmzcbfJoTwxoA+6Q38RKCWS7qbOpuCyj31NZRxNHFI5H0laRt5aTytuAUfMLxkRfe30A3eZ52rvnnf1tKUKyV\\\\\\\/CA00JWWNNNlLnRlJBMbyo6VtZyu8\\\\\\\/O8KA9lZIShgjAstYutt6\\\\\\\/SYyonoc5gIjIouwKa3oEJ4Oyrc0a4u9dhiZ7FOQlM5gwmdsHtn1HDWG1dpuaIpysJuQcAgLw2X\\\\\\\/NaTilcNHkTJWBg\\\\\\\/vuOm8217zB92idSmCjbv8\\\\\\\/ESCJn2ges\\\\\\\/00NqI5DD1cNU4Jj5QGRA5TLSxTFQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022j01pa6n38R1y9PrXlIu0Yg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00228146468d121ec074b24816f4d97bff8f364170634c673c1bd4f32456685452af\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022dcEM7ekhBYi9D8tpYC32fMZYM41FaTiGT76g4+7pKe4c99tr9kTK8w4vJEewrAKi3MLmDlDiIoZRPMIJ3qckvtMLBvppE9dLk3KdP3tsXfWZ3bV4HQLwtqPSuKXWbCIc7GVKBsSTGcTKO64k3X1dc+6TeHsiPmPrgezp0JJu\\\\\\\/IvxIyAZV09TbGrlPApVeB2\\\\\\\/bsCUgX5\\\\\\\/6wnLW\\\\\\\/NJjZQGs0Qg64J\\\\\\\/0\\\\\\\/vLH5GVefXR2m69KAs\\\\\\\/gzJGFEo7RdnZeKZPlOzC\\\\\\\/Ffhf\\\\\\\/abFRHSq+sVG1w74kC51fi\\\\\\\/CbLAYxkqfCVMqbBucliFMXPjC4UWqAHy0uxqq4iTu\\\\\\\/GYHqS5rM9UAoC\\\\\\\/lhlxp+gOqiSNgH2lqTGZZDVVbMUOw5XK0BNKDqth5WvlqgieGAF9x7TPf7Gmyyc\\\\\\\/fb107F4Yqsej71sPkjYgh4NOQPckym1oPT6GMcfRh30LmWxSJblHjRlUNlhxIaAhYVjzG5K\\\\\\\/7n8rGSQbTs4mXYg4jSqRqc2y0EGE9fp8xhzpRTVaK8E6VfcKIslI1UmP5KmOaW4pEw3guIecTiwtcrBAoYnjbAsRxRu8arlU58m79nveVbfSTAKh0tQS+t9xDM4lmzMjONBbWZn+SQzwfUS4vbs99p7vQsb5OWOkTeIZKG+8cXbhvADLrE8JUrW2fxP3zy9IRI9dnczYvz71g0OYg4TtWrtoa6o3kQc9obM\\\\\\\/9Xp4rTMoLpiWRwx5otUlrxSs2+1aVFpixCgpS\\\\\\\/H74l4i8+U7oSgP3HUtycyCBVjTtTCCtPkpSDMiSwG54m9Arzaj70IDCqbFjIvdH9hf3lkOSaIkQ\\\\\\\/YQV0vsxX+94eZQkI6zU\\\\\\\/gxyiAJo6QAmzhyxTGJl0LmCbUVj6+SxVQDkV8GbByaxEaG1pRdFwv9QsT9jKHVlXOYC9I5jaNIYzXNZ9U+iOI6PfsskfBTympEKXFQd2yZ\\\\\\\/DuQ\\\\\\\/+2gHGvqbkRzlm+eosFzXbvHMeBljyopKMLoM7fHWS8NIp4raUtmjVEfI1wTdBbsUl3Jh2d0un\\\\\\\/I5r\\\\\\\/YskTwWZu9sLPvDb\\\\\\\/YFjpxCZtzZSKABkpHxOJ9xmCAJi9YPMTuFCUjlcRABE778rO3rhz1rH0auYRLdskJpnnUDt3VjDbT3TFjnEJ3wI2kvcRBYunYbgiauGjI3Wqv9Pj5McEFQXZKQCGTssBmqoCpr0CJTBuv61K5QFzNVd+ItpIGLPwVD8gYi1Nu1HXTh0c297HPwtxI2m2pcUPkBT0RJL5DoQYbVIiN8JcZfYsOfh5Lj\\\\\\\/lmU9NoWeaAh3EBcr4g\\\\\\\/KFufxY9o+WKAigIUHSlaQGgKnprae6WEBrGdwzjniZvysLFQkmKgQW1jY0wLaIYzkjnfx4nO9CFXRNFm\\\\\\\/11g\\\\\\\/aMSNGYgRznavuFMR6kYKPYRU1JHdHaeVbb2CW5n6gRUT9AzbHZIXPpqOYThQ0ZPiDAyNkT04k4K1j5tj5ERZVYhJfrWUIJVSxXMOEooh0LXTRXwTf+E\\\\\\\/CxP+0s6nyde7iWXDe0RA6TAU5h1TE9QqW6wjrTR1u\\\\\\\/1d9I9D6S4clUZp0fwB3uIBzfWgkEvKwGs+Zi+zbR8Qrm8+gD+DxoErxYls3uSQ5jgf9F4qi49Y8OwqBMWpJ4dmIx+R0a5rMG1gN5LzNmfNMYwkKyBh6CedNUToISbs448fYgsrbJA7EcA7LtG+CWDCx6Zr6JLYjnDnnLMA2Ya9LVoBPyJvRa1bCSPK3EpF1FFYqYy6CqiTXPOFCnwWyBpkBTrV\\\\\\\/cASO1uTtepByOPkcW16iw8h\\\\\\\/8dmcHMIdhTpG2A0Z3bUvITz7rlxwbYAmd6PhAH9d8GJbjGSIln4e5Xw62ho7HDn7vQ\\\\\\\/cc2iXHgwKmjtb4qn+O6b8r0Tj8oCpRPO+rin2ulJuBFAYCv6s62quhfojmbCCRCDsRn1xoW2MP3avdifDLRJXMIUDmYksLzWSpliebqQuE0E38O6\\\\\\\/PHkhkRJkB7fInOyXpmEXtNnAHFSW80notSQhX2yCpCty4pHG8lw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022\\\\\\\/I4Sj8Kc19REF14MT9xxfw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022ee8688c7e64bc6e8640a127b3a42162823c875ceee00a0eebe4f42c2b178f898\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-486356 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-genomic-selection">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/9/3169" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-EDs7_DpPjYaFWuO4NUyfcrql9kHAdaq-VeiA3qoCRNs" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/9/3169" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-1IPH6CMM7LgwXqgTVamrWpOw--PpUokmimDOJXKUGj8" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/9/3157" title="The Douglas-Fir Genome Sequence Reveals Specialization of the Photosynthetic Apparatus in Pinaceae" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/9/3177" title="The Stress-Inducible Peroxidase TSA2 Underlies a Conditionally Beneficial Chromosomal Duplication in Saccharomyces cerevisiae" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="486356" id="node486356" data-pisa="ggg;7/9/3169" data-pisa-master="ggg;g3.117.300082" data-apath="/ggg/7/9/3169.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genome-Wide SNP Discovery and Analysis of Genetic Diversity in Farmed Sika Deer (<em>Cervus nippon</em>) in Northeast China Using Double-Digest Restriction Site-Associated DNA Sequencing</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Hengxing Ba</span>, <span class="highwire-citation-author" data-delta="1">Boyin Jia</span>, <span class="highwire-citation-author" data-delta="2">Guiwu Wang</span>, <span class="highwire-citation-author" data-delta="3">Yifeng Yang</span>, <span class="highwire-citation-author" data-delta="4">Gilead Kedem</span> and <span class="highwire-citation-author" data-delta="5">Chunyi Li</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3169-3176;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300082 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node486356" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Hengxing Ba </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>State Key Laboratory for Molecular Biology of Special Wild Economic Animals, Institute of Special Wild Economic Animals and Plants, Chinese Academy of Agricultural Sciences, Changchun 130112, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hengxing%2BBa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ba%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHengxing%2BBa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Boyin Jia </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>State Key Laboratory for Molecular Biology of Special Wild Economic Animals, Institute of Special Wild Economic Animals and Plants, Chinese Academy of Agricultural Sciences, Changchun 130112, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Boyin%2BJia%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jia%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABoyin%2BJia%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Guiwu Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>State Key Laboratory for Molecular Biology of Special Wild Economic Animals, Institute of Special Wild Economic Animals and Plants, Chinese Academy of Agricultural Sciences, Changchun 130112, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Guiwu%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGuiwu%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Yifeng Yang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>State Key Laboratory for Molecular Biology of Special Wild Economic Animals, Institute of Special Wild Economic Animals and Plants, Chinese Academy of Agricultural Sciences, Changchun 130112, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yifeng%2BYang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yang%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYifeng%2BYang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Gilead Kedem </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>GENEWIZ Inc., South Plainfield, New Jersey 07080</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gilead%2BKedem%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kedem%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGilead%2BKedem%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Chunyi Li </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>State Key Laboratory for Molecular Biology of Special Wild Economic Animals, Institute of Special Wild Economic Animals and Plants, Chinese Academy of Agricultural Sciences, Changchun 130112, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Chunyi%2BLi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Li%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AChunyi%2BLi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:lichunyi1959@163.com" class="" data-icon-position="" data-hide-link-title="0">lichunyi1959@163.com</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/7/9/3169" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:486356" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:486356/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/9/3169.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:486356" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:486356/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/9/3169.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:486356" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:486356/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/9/3169.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:486356" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:486356/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/7/9/3169.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:486356" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:486356/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Sika deer are an economically valuable species owing to their use in traditional Chinese medicine, particularly their velvet antlers. Sika deer in northeast China are mostly farmed in enclosure. Therefore, genetic management of farmed sika deer would benefit from detailed knowledge of their genetic diversity. In this study, we generated over 1.45 billion high-quality paired-end reads (288 Gbp) across 42 unrelated individuals using double-digest restriction site-associated DNA sequencing (ddRAD-seq). A total of 96,188 (29.63%) putative biallelic SNP loci were identified with an average sequencing depth of 23×. Based on the analysis, we found that the majority of the loci had a deficit of heterozygotes (F<sub>IS</sub> &gt;0) and low values of H<sub>obs</sub>, which could be due to inbreeding and Wahlund effects. We also developed a collection of high-quality SNP probes that will likely be useful in a variety of applications in genotyping for cervid species in the future.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/sika-deer" class="hw-term hw-article-keyword hw-article-keyword-sika-deer" rel="nofollow">sika deer</a></li><li class="kwd"><a href="/keyword/ddrad-seq" class="hw-term hw-article-keyword hw-article-keyword-ddrad-seq" rel="nofollow">ddRAD-seq</a></li><li class="kwd"><a href="/keyword/snp-discovery" class="hw-term hw-article-keyword hw-article-keyword-snp-discovery" rel="nofollow">SNP discovery</a></li><li class="kwd"><a href="/keyword/genetic-diversity" class="hw-term hw-article-keyword hw-article-keyword-genetic-diversity" rel="nofollow">genetic diversity</a></li><li class="kwd"><a href="/keyword/stacks" class="hw-term hw-article-keyword hw-article-keyword-stacks" rel="nofollow">STACKS</a></li><li class="kwd"><a href="/keyword/genpred" class="hw-term hw-article-keyword hw-article-keyword-genpred" rel="nofollow">GenPred</a></li><li class="kwd"><a href="/keyword/shared-data-resources" class="hw-term hw-article-keyword hw-article-keyword-shared-data-resources" rel="nofollow">Shared Data Resources</a></li><li class="kwd"><a href="/keyword/genomic-selection" class="hw-term hw-article-keyword hw-article-keyword-genomic-selection" rel="nofollow">Genomic Selection</a></li></ul><p id="p-3">Sika deer are an economically valuable species, with some body parts, such as antler, blood, penis, and placenta, used in traditional Chinese medicine. In order to continue the tradition, people have captured sika deer from the wild on a large scale in Jilin province in the northeastern part of China since the Qing dynasty (c. 1733) (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Sheng and Ohtaishi 1993</a>). These animals were gradually introduced into many other regions of China. The original captive population in northeast China is generally considered to be the source of the entire sika deer population on farms throughout China (<a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Sheng and Ohtaishi 1993</a>).</p><p id="p-4">Almost all sika deer raised on farms in northeast China are used for antler (velvet antler) production (<a id="xref-ref-32-3" class="xref-bibr" href="#ref-32">Sheng and Ohtaishi 1993</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Wu <em>et al.</em> 2004</a>). China is currently one of the largest producers of velvet antler in the world and sika deer farming in China is still thriving. Nonetheless, there is still concern about the deterioration of farm-bred sika deer (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Wu and Zhang 2001</a>). In recent decades, people have been trying to improve velvet antler yield through long-term improved management on farms (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">McCullough <em>et al.</em> 2009</a>). Whether a healthy genetic diversity can be maintained is an open question. Therefore, any genetic management requires detailed knowledge of the genetic diversity.</p><p id="p-5">The genetic diversity of the sika deer wild population has been investigated using two major sources of genomic variation. Mitochondrial genes, such as cytochrome b and the control region, have proven highly informative for investigations into the classification and phylogeny of sika deer with maternal haplogroups (<a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Wu <em>et al.</em> 2004</a>, <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">2005</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Lu <em>et al.</em> 2006</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Takiguchi <em>et al.</em> 2012</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Krojerova-Prokesova <em>et al.</em> 2013</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Ba <em>et al.</em> 2015</a>, <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">2016</a>). Autosomal microsatellites have been used extensively to estimate levels of genetic diversity (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Tamate <em>et al.</em> 2000</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Thevenon <em>et al.</em> 2004</a>; <a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Krojerova-Prokesova <em>et al.</em> 2013</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">He <em>et al.</em> 2014</a>). Although highly polymorphic markers can be very informative, in particular when assessing recent demographic bottlenecks (endangered and captive-bred species), the previous data are not sufficient to provide a good understanding of the level of genetic diversity in the farmed population in China. This is the first study to report levels of genetic diversity within sika deer using single nucleotide polymorphism (SNP) markers on a global scale.</p><p id="p-6">Their genomic abundance and amenability to cost-effective, high-throughput genotyping, has meant that SNPs are now the most widely used class of genetic markers. Use of genome-wide SNPs constitutes an important genetic tool for investigating genetic diversity in livestock animals (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Groeneveld <em>et al.</em> 2010</a>). Although genome-wide SNP discovery in white-tailed deer (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Seabury <em>et al.</em> 2011</a>) and hog deer (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Wang <em>et al.</em> 2017</a>) has been reported, development of this tool in deer has been hampered in part by the substantial duplicate regions in large parts of the deer genome, and by the lack of a reference genome sequence. Fortunately, double-digest restriction site-associated DNA sequencing (ddRAD-seq) technology (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Peterson <em>et al.</em> 2012</a>) can provide a flexible and inexpensive platform for the simultaneous discovery of tens of thousands of SNP markers in model and nonmodel organisms. While several SNP genotyping tools have been developed for SNP discovery, including GATK (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">DePristo <em>et al.</em> 2011</a>), STACKS (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Catchen <em>et al.</em> 2011</a>, <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">2013</a>), SAMtools (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Li 2011</a>) and RADtools (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Baxter <em>et al.</em> 2011</a>), STACKS is recommended for ddRAD-seq projects, as it uses a multinomial-based likelihood model to call SNPs, which incorporates a bounded SNP model.</p><p id="p-7">In this study, we employed the ddRAD-seq technology to achieve the first genome-wide SNP discovery for farmed sika deer that are widely distributed in northeast China. The STACKS pipeline was applied for SNP calling. Using newly developed markers, we investigated the genetic diversity of farmed populations. We also developed a collection of SNP probes for genotyping in the future.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Sample collection</h3><p id="p-8">A total of 42 unrelated animals were selected from eight farms (farm codes: AD, CB, DF, NM, SP, SY, XF, and XK) in northeast China, with six from NM and XK farms and five from the other six farms. Whole-blood samples were extracted from the jugular vein using EDTA vacuum tubes and were stored at −20 °C until DNA extraction. The genomic DNA extractions from whole blood were performed using a blood DNA kit (Qiagen) according to the manufacturer’s instructions. Each DNA sample was evaluated by gel electrophoresis for the presence of high-molecular-weight DNA and then stored at −80°C until ddRAD-seq library construction.</p></div><div id="sec-3" class="subsection"><h3>ddRAD-seq library preparation and sequencing</h3><p id="p-9">The procedure was performed as described in previous studies (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">DaCosta and Sorenson 2014</a>) with some modifications. First, the double-digest reactions were carried out in a volume of 25 μl containing ∼0.8 μg of genomic DNA, 5U of PstI and MseI, and 1× buffer 3.1 (NEB). The reaction mixture was incubated at 37 °C for 2 hr and 65 °C for 30 min Amplification and sequencing adapters with a unique barcode (5 or 6 bp) were ligated on to the digested DNA. Each sample was then amplified via PCR in a 50 μl reaction volume, containing 70–100 ng of adaptor-ligated DNA fragments, and amplified with 22 cycles following the manufacturer’s protocol. Samples were run on a 2% agarose gel, and DNA in the 300–450 bp size range (with indices and adaptors) was excised using a gel extraction kit (Qiagen). Each sample library was pooled in equal amounts and quantified using Agilent 2100 (Agilent Technologies) and real-time quantitative PCR, and then paired-end 101 bp sequencing was performed using the Illumina HiSeq4000 platform (BGI, Shenzhen, China).</p></div><div id="sec-4" class="subsection"><h3>Available sika deer genome reference</h3><p id="p-10">The available sika deer genome reference (SK-REF) included 387,939 contigs, comprising a total of 2.73 Gbp. All sequences were longer than 500 bp, with a N50 length of 38.4 kbp. The percentage of repeat sequences detected by Repeatmasker-4.0.6 (<a href="http://www.repeatmasker.org">http://www.repeatmasker.org</a>) was 44.14% and the GC level was 42.11% (H. Ba, unpublished data).</p></div><div id="sec-5" class="subsection"><h3>RAD-seq data analysis and SNP identification</h3><p id="p-11">Given the main quality score parameters (-c -q -r) of the process_radtags program in STACKS v.1.41 (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Catchen <em>et al.</em> 2011</a>, <a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">2013</a>), raw reads were filtered and separated by barcode. The filtered paired-end reads were uniquely mapped to SK-REF using BOWTIE v2.0 (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Langmead and Salzberg 2012</a>) (disallowing gaps and suppressing unpaired and discordant alignments), followed by SAMtools v1.2 (<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Li 2011</a>) to convert to a BAM file. Putative orthologous loci were assembled for each animal using the BAM file as the input of the pstacks function with the minimum depth of coverage above 5. Two mismatches were allowed between animals when assembling the catalog of ddRAD loci using the cstacks program. Matches of individual ddRAD loci to the catalog were searched using the sstacks program. Loci that have low depth of coverage or high sequencing error will exhibit poor log likelihood scores that are highly negative. The rxstacks program (-conf_filter -conf_lim 0.75 -prune_haplo -lnl_lim -8.0) was used to make corrections to genotype calls in individuals based on data from a population-wide examination. Subsequently, the cstacks and sstacks programs were again run to rebuild and match to the catalog. The populations program was used to filter the dataset to contain SNPs found in at least 75% of individuals and with a minor allele frequency ≥0.05.</p></div><div id="sec-6" class="subsection"><h3>Estimates of genetic diversity within population</h3><p id="p-12">Observed heterozygosity (H<sub>obs</sub>), expected heterozygosity (H<sub>exp</sub>) and inbreeding coefficient of an individual relative to the subpopulation (F<sub>IS</sub>) were calculated using the populations program in STACKS v.1.41. Deviation from the Hardy–Weinberg equilibrium (HWE) was assessed by performing a Fisher’s exact test with the HardyWeinberg R package (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Graffelman and Camarena 2008</a>) for each SNP marker. The False discovery rate correction (Benjamini–Hochberg) was performed using the p.adjust program in the R package.</p></div><div id="sec-7" class="subsection"><h3>Population structure analysis</h3><p id="p-13">Bayesian clustering analysis implemented in STRUCTURE 2.3.4 (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Pritchard <em>et al.</em> 2000</a>) was used for estimating the number of populations/groups (<em>k</em>) represented by the dataset. Three iterations were run per <em>K</em> (number of populations) for <em>K</em> = 2 or <em>K</em> = 3 using an admixture model. Each run consisted of a burn-in of 100,000 MCMC steps, followed by 500,000 replications. Population structure was also examined by carrying out principal component analysis using SMARTPCA within EIGENSOFT (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Patterson <em>et al.</em> 2006</a>).</p></div><div id="sec-8" class="subsection"><h3>Development of SNP probes for genotyping assay</h3><p id="p-14">A custom script was run to develop high-quality SNP probes. In summary, sufficient 50 bp flanking sequences on either side of the SNPs were derived from the SK-REF genome and then were filtered according to four filtration criteria: (a) no repetitive sequences; (b) SNP-free within flanking sequence; (c) one of two alleles in accordance with the base in the reference; (d) flanking sequence on the SK-REF ≥50bp.</p></div><div id="sec-9" class="subsection"><h3>Data availability</h3><p id="p-15">Raw (adapter trimmed) Illumina ddRAD-seq sequences: NCBI Short Read Archive (project accession: SRP105008). Statistics describing different properties of each sequenced individual and all high-quality SNP probes are available in the Supplemental Material, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300082/-/DC1/TableS1.xlsx">Table S1</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300082/-/DC1/TableS2.csv">Table S2</a>.</p></div></div><div class="section results" id="sec-10"><h2 class="">Results</h2><div id="sec-11" class="subsection"><h3>ddRAD sequencing</h3><p id="p-16">After trimming the barcodes and filtering low-quality bases, ddRAD sequencing generated a total of over 1.45 billion high-quality paired-end reads (288 Gbp) across the 42 animals. Of these clean reads, &gt;0.93 billion (65.63%) were aligned uniquely to the SK-REF genome. As variation among animals is often observed in a pooled ddRAD sequencing library, the number of aligned reads per individual ranged from 0.55 to 5.3 million (1.01–9.78 Gbp, with an average of 4.1 Gbp) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300082/-/DC1/TableS1.xlsx">Table S1</a>).</p></div><div id="sec-12" class="subsection"><h3>SNP discovery</h3><p id="p-17">Using the Stacks pipeline, we initially obtained 7,576,300 candidates of the ddRAD loci from all individuals. Quality filtering (see Methods) reduced this to a total of 324,564 (4.28%), which were shared by &gt;75% of the individuals, corresponding to genomic size of 0.032 Gbp (∼1.1% of the genome) (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). Within these 324,564 ddRAD loci of ∼100 bp, we detected 96,188 (29.63%) putative biallelic SNPs with an average sequencing depth of 23×. For each individual, the number of SNPs varied from 56,388 (58.6%) to 89,845 (93.4%), and the sequencing depth ranged from 9× to 50×. The number of identified heterozygous SNPs varied from 11,780 to 22,641 with an average of 19,078, corresponding to 0.38–0.71 heterozygous SNPs per kilobase pair with an average of 0.61 detected per individual animal (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300082/-/DC1/TableS1.xlsx">Table S1</a>). With a read rate within individuals averaging 83%, this suggests that the rate could be ∼0.74 per kbp.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/486449/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/486449/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Filtering results for STACKS pipeline</span><div class="sb-div caption-clear"></div></div></div><p id="p-18">Through evaluation of SNP frequency, substitution types and sequencing depth, the results showed that: (a) SNP frequency decreased from 5′ to 3′ end in all loci excluding both ends; however, transition/transversion (Ts/Tv) ratios were very stably distributed over the loci except for the last five positions at the 3′ end (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>); (b) frequency of A/T substitution is relatively less than other types of substitutions (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1B</a>), which could be attributed to lower coverage in AT-rich region; (c) minor alleles had relatively lower sequencing depth, but the difference between the depths of the minor and major alleles was not statistically significant (Wilcox paired test <em>P</em> value = 0.056) (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1C</a>); (d) a majority of loci contained one SNP and only 1733 (2.14%) were identified with &gt;2 SNPs (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1D</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/9/3169/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Evaluation of putative SNP quality. (A) Distribution of SNP frequency on ddRAD tag positions. Distribution of the identified Ts/Tv is also illustrated. (B) Distribution of six types of substitutions. (C) Distribution of SNP frequency in major allele, minor allele, and ddRAD loci against their sequencing depth. (D) Distribution of SNP frequency of the number of SNPs in ddRAD loci tags." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1059704979" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Evaluation of putative SNP quality. (A) Distribution of SNP frequency on ddRAD tag positions. Distribution of the identified Ts/Tv is also illustrated. (B) Distribution of six types of substitutions. (C) Distribution of SNP frequency in major allele, minor allele, and ddRAD loci against their sequencing depth. (D) Distribution of SNP frequency of the number of SNPs in ddRAD loci tags.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/9/3169/F1.medium.gif" width="440" height="319"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/7/9/3169/F1.medium.gif" width="440" height="319"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/9/3169/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/9/3169/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/486540" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-19" class="first-child">Evaluation of putative SNP quality. (A) Distribution of SNP frequency on ddRAD tag positions. Distribution of the identified Ts/Tv is also illustrated. (B) Distribution of six types of substitutions. (C) Distribution of SNP frequency in major allele, minor allele, and ddRAD loci against their sequencing depth. (D) Distribution of SNP frequency of the number of SNPs in ddRAD loci tags.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-13" class="subsection"><h3>Genetic diversity</h3><p id="p-20">Genetic parameters evaluated using putative SNPs included MAF, H<sub>obs</sub>, H<sub>exp</sub>, F<sub>IS</sub>, and deviation from HWE for the studied population. The distribution of MAF showed that the SNPs with MAF &lt;0.15 were overrepresented at nearly half of the SNPs (49.0%), and SNPs were evenly distributed in high-MAF bins ranging from 0.25 to 0.5 (29%) (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>). The average MAF of all SNP loci was 0.20.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/9/3169/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Population genetic parameters for putative SNP loci. (A) Frequency of SNPs across the MAF bins. The line depicts the cumulative density function across the MAF bins. (B) Frequency of SNPs across the heterozygosity bins. (C) Frequency of SNPs across the FIS bins." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1059704979" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Population genetic parameters for putative SNP loci. (A) Frequency of SNPs across the MAF bins. The line depicts the cumulative density function across the MAF bins. (B) Frequency of SNPs across the heterozygosity bins. (C) Frequency of SNPs across the F&lt;sub&gt;IS&lt;/sub&gt; bins.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/9/3169/F2.medium.gif" width="440" height="282"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/7/9/3169/F2.medium.gif" width="440" height="282"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/9/3169/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/9/3169/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/486393" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-21" class="first-child">Population genetic parameters for putative SNP loci. (A) Frequency of SNPs across the MAF bins. The line depicts the cumulative density function across the MAF bins. (B) Frequency of SNPs across the heterozygosity bins. (C) Frequency of SNPs across the F<sub>IS</sub> bins.</p><div class="sb-div caption-clear"></div></div></div><p id="p-22">Of 96,188 SNPs, 11,876 (12.35%, at <em>P</em> ≤0.05) were shown to deviate from HWE. Following application of the Benjamini–Hochberg correction, 8205 (5.53%) SNPs remained significant (<em>P</em> ≤0.05). The excess of H<sub>obs</sub> values was observed in the frequency distribution from 0 to 0.15, and the deficit was from 0.4 to 0.5 (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2B</a>). The deficit of heterozygotes (F<sub>IS</sub> &gt;0) is shown in <a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2C</a>, indicating a high level of inbreeding. The average H<sub>obs</sub>, H<sub>exp</sub>, and F<sub>IS</sub> calculated across all 96,188 SNPs were 0.24, 0.28, and 0.16, respectively.</p><div id="sec-14" class="subsection"><h4>Genetic structure:</h4><p id="p-23">In order to assess whether there is a stratification within the farmed sika deer population, population structure was estimated using all SNPs (96,188) across &gt;32 animals (≥75%). Since the STRUCTURE model assumes that the loci are independent and at HWE, we also performed STRUCTURE analysis on a filtered SNP dataset (13,739) at HWE (<em>P</em> ≥0.05) with one SNP on each ddRAD loci across &gt;38 animals (≥90%). Graphic displays of the consistent results from two SNP datasets provided a meaningful explanation of the genetic structure and levels of admixture for the farmed populations (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3, A and C</a>). At <em>K</em> = 2, the two clusters were distinct; four animals (SP1, SP3, SP4, and SP5) were clearly distinguished and, of these, SP3 and SP4 represented an admixture pattern (two clusters of genetic background). At <em>K</em> = 3, another cluster of two animals (AD5 and DF4) was evident. Similar results were also observed using the PCA approach based on these two SNP datasets. Although PC1 and PC2 accounted for only 2.4% and 1.9% of total variation, respectively, four animals (SP1, SP3, SP4, and SP5) were clearly distinguishable based on PC1, and two animals (AD5 and DF4) based on PC2 (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3, B and D</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/9/3169/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Population genetic structure. (A) STRUCTURE analysis for K = 2 and 3 for 96,188 SNPs. (B) PCA analysis for 96,188 SNPs. (C) STRUCTURE analysis for K = 2 and 3 for 13,739 SNPs. (D) PCA analysis for 13,739 SNPs." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1059704979" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Population genetic structure. (A) STRUCTURE analysis for &lt;em&gt;K&lt;/em&gt; = 2 and 3 for 96,188 SNPs. (B) PCA analysis for 96,188 SNPs. (C) STRUCTURE analysis for &lt;em&gt;K&lt;/em&gt; = 2 and 3 for 13,739 SNPs. (D) PCA analysis for 13,739 SNPs.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/9/3169/F3.medium.gif" width="440" height="292"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/7/9/3169/F3.medium.gif" width="440" height="292"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/9/3169/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/9/3169/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/486455" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-24" class="first-child">Population genetic structure. (A) STRUCTURE analysis for <em>K</em> = 2 and 3 for 96,188 SNPs. (B) PCA analysis for 96,188 SNPs. (C) STRUCTURE analysis for <em>K</em> = 2 and 3 for 13,739 SNPs. (D) PCA analysis for 13,739 SNPs.</p><div class="sb-div caption-clear"></div></div></div></div></div><div id="sec-15" class="subsection"><h3>SNP probes</h3><p id="p-25">We selected 27,009 high-quality SNPs probes according to four filtration criteria. The Ts/Tv ratio was 3.72. The MAF frequency distribution was accordant between the 96,188 and 27,009 SNP datasets (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>), suggesting that the repetitive sequences have no effect on the change of MAF frequency distribution for the discovered SNPs. These SNP probes (101 bp) were matched with the bovine genome assembly (UMD3.1) via BLASTN, and their alignment resulted in 58.42% (15,778/27,009) probes producing 50,447 <em>E</em>-value informative hits (<em>E</em> value 1e<sup>−20</sup>). As expected, 96.24% (15,185/15,778) of these matched SNP probes produced one unique alignment to a bovine genome, with an average identity of 94.39%. Only 3.76% (593/15,778) probes hit multiple cattle genome positions (35,262).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/9/3169/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparison of SNP frequency distribution between all putative SNPs and high-quality SNPs." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1059704979" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Comparison of SNP frequency distribution between all putative SNPs and high-quality SNPs.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/9/3169/F4.medium.gif" width="440" height="335"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/7/9/3169/F4.medium.gif" width="440" height="335"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/9/3169/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/9/3169/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/486396" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-26" class="first-child">Comparison of SNP frequency distribution between all putative SNPs and high-quality SNPs.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-16"><h2 class="">Discussion</h2><p id="p-27">The ddRAD-seq technique is an efficient and cost-effective means of SNP discovery, providing thousands of high-quality SNPs, even in the absence of an available genome sequence (<a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Peterson <em>et al.</em> 2012</a>). Using the ddRAD-seq technique, we reported here the first genome-wide set of 96,188 novel SNPs in sika deer from 42 unrelated animals selected from eight farms.</p><p id="p-28">In order to evaluate the application of SNPs reported in our study, the SNP quality was screened based on the data characteristics of next-generation sequencing technology. Sequencing errors are often found at the 3′ end of the sequence reads (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Dohm <em>et al.</em> 2008</a>). If a substantial proportion of the SNPs in the dataset were the result of sequencing errors, the number of SNPs toward the end of the reads should increase. However, our results showed that the number of SNPs identified actually decreased from the 5′ to the 3′ end (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1A</a>), indicating that base errors at the 3′ end were not identified as SNPs. An underrepresentation at both ends of the reads (positions 1–3 and 95–100) was attributed to the <em>Mse</em>I and <em>Pst</em>I restriction sites and the effect of barcode trimming. However, the Ts/Tv ratio is very stably distributed over the loci except at the very ends of the reads, and was not affected by the relatively higher base error rate at the ends of the reads.</p><p id="p-29">The expected Ts/Tv ratio can vary with the targeted region (<em>i.e.</em>, whole genome, exon, specific genes) and can also vary greatly in the CpG islands and GC content of the region (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Dohm <em>et al.</em> 2008</a>). For example, the Ts/Tv ratio of 3.5 is typical of coding regions in human genomes (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Le and Durbin 2011</a>). Higher Ts/Tv ratios are also observed in other reduced representation libraries for SNP discovery (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Kraus <em>et al.</em> 2011</a>; <a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Le and Durbin 2011</a>). Therefore, it is not unexpected that the observed Ts/Tv ratio is relatively higher in the SNP dataset. The number of A/T substitutions is relatively lower than other types of substitutions (<a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1B</a>). This could be attributed to the fact that higher coverage needs to be attained to discover real SNPs in AT-rich regions (<a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">Dohm <em>et al.</em> 2008</a>). Matching the base content of a restriction site to a reference genome can also influence substitution types (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Davey <em>et al.</em> 2011</a>).</p><p id="p-30">To date, the SK-REF database comprises ∼90% of the sika deer genome, and is incomplete. The unique alignment of ddRAD reads does not guarantee that there is no other similar sequence in the remainder of the genome. This type of genome mapping could result in abnormally high coverage in the paralogous or duplicated sequence, which will likely generate a large number of false-positive SNPs with excessive heterozygotes (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Malhis and Jones 2010</a>). Our results show that the depths of almost all alleles fall into a reasonable range (less than threefold average depth, 42× for minor alleles and 60× for major alleles) (<a id="xref-fig-1-7" class="xref-fig" href="#F1">Figure 1C</a>). This is an acceptable depth range for SNP discovery that had been identified in other reduced representation libraries of sequencing projects, including cattle (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Eck <em>et al.</em> 2009</a>), porcine (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Amaral <em>et al.</em> 2009</a>) and turkey (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Kerstens <em>et al.</em> 2009</a>). We did not observe excess heterozygotes, in which the number of SNPs with H<sub>obs</sub> &gt;0.5 is relatively lower in the SNP dataset (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2B</a>). In addition, only 1733 (2.14%) ddRAD loci were identified with &gt;2 SNPs, which was also a good indicator of correct homologous loci matching (<a id="xref-fig-1-8" class="xref-fig" href="#F1">Figure 1D</a>).</p><p id="p-31">The average frequency of heterozygous SNPs per kilobase pair is 0.74 in the genome of sika deer individuals. As a comparison, this value is slightly higher than that of milu (a highly endangered species that is susceptible to genetic drift and inbreeding because of small population size), but almost &lt;2 times those of cow and panda (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). The relatively low level of genetic variability in the farmed population could be due to ongoing inbreeding within a narrow gene pool and genetic drift. Additionally, the 42 individuals sequenced were from eight farmed populations that may be small and divergent in allele frequencies. For instance, almost 5.53% (8205) of SNPs (Benjamini–Hochberg correction <em>P</em> ≤0.05) deviate from HWE, which may be caused by the different subpopulations sampled. The higher values of F<sub>IS</sub> and the lower-than-expected values of H<sub>obs</sub> could be also explained by subpopulation structure (Wahlund effect). The STUCTURE and PCA results further provide evidence of subpopulation structure in farmed sika deer, despite the lack of overall variation and the fact that the PC1 and PC2 accounted for such a small proportion (2.4% and 1.9%, respectively) of the total variation. The small-scale farming population and the application of the technique of artificial insemination over a long period have encouraged inbreeding and genetic drift for velvet antler production. Such population events also compromise the genetic structure of the original population. Hence, our results could help in investigating the structure of farmed populations so such disadvantages can be better managed.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/486589/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/486589/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Information of SNP frequency among four species</span><div class="sb-div caption-clear"></div></div></div><p id="p-33">SNP markers are not only readily applied to population genetic analyses, but also suitable for parentage testing and assessment of breed composition. Genome-wide association analysis may offer the opportunity to identify the genomic regions and mutations that underpin production traits such as velvet antler weight. One attractive feature of association studies is that pedigrees are not necessary, so potentially a larger number of the farmed animals may be amenable to this type of analysis (<a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Seabury <em>et al.</em> 2011</a>). However, high-quality phenotypes are critical, and this will require major investment.</p><p id="p-34">There may be potential to apply genomic selection for the genetic improvement of farmed sika deer. The small effective population size may make this an attractive proposition. The option of introducing a different subpopulation would also enable the exploitation of heterosis (hybrid vigor).</p><p id="p-35">Overall, we have discovered ∼27,009 high-quality SNPs in 1% of the sika genome using ddRAD-seq technology. A total of 58% (15,778/27,009) probes were uniquely matched on to the bovine genome (BLASTN E value 1e<sup>−20</sup>), which could be an indication of the sharing of SNP genotypes between the genomes. Only 593 SNP probes of sika deer could be aligned multiple times to the bovine genome, which may represent a duplicated and/or expanded region on the cattle genome. In comparison, <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Haynes and Latch (2012)</a> successfully genotyped a lower proportion of loci (39%) in mule deer using BovineSNP50 BeadChip, and <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Powell <em>et al.</em> (2016)</a> directly captured 60% exon data from bovine genome on mule deer genome, matching rates almost concordant with our data. As deer and cattle diverged 27–32 MYA (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Hassanin and Douzery 2003</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Guha <em>et al.</em> 2007</a>), the matching of sika deer SNPs discovered in the bovine genome would likely be useful to study genomic evolution and phylogenetic analysis across species within Cervidae, given the high cross-species matching success rate. Although, to date, there is no valuable reference genome within the Cervidae family and map positions are unknown, SNP assays derived from nonrepetitive loci (contigs) could be designed for high-density genotyping across cervid species, providing a reasonable genome analysis tool for future research.</p></div><div class="section" id="sec-17"></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-36">We thank Peter Fennessy and Jia Yankai for reading the paper and giving valuable comments. This work was funded by the Chinese National Science Foundation (31402035) and partially funded by Jilin Province Science Foundation (20170101003JC). </p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-2"><p id="p-37">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300082/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300082/-/DC1</a>.</p></li><li class="fn" id="fn-3"><p id="p-38"><em>Communicating editor: D-J. de Koning</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2017-06-12"><span class="received-label">Received </span>June 12, 2017.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2017-07-18"><span class="accepted-label">Accepted </span>July 18, 2017.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2017 Ba <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3169.1" data-doi="10.1186/1471-2164-10-374"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Amaral</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Megens</span>  <span class="cit-name-given-names">H. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kerstens</span>  <span class="cit-name-given-names">H. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heuven</span>  <span class="cit-name-given-names">H. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dibbits</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Application of massive parallel sequencing to whole genome SNP discovery in the porcine genome.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">374</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DAmaral%26rft.auinit1%253DA.%2BJ.%26rft.volume%253D10%26rft.spage%253D374%26rft.epage%253D374%26rft.atitle%253DApplication%2Bof%2Bmassive%2Bparallel%2Bsequencing%2Bto%2Bwhole%2Bgenome%2BSNP%2Bdiscovery%2Bin%2Bthe%2Bporcine%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-10-374%26rft_id%253Dinfo%253Apmid%252F19674453%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-10-374&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19674453&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3169.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3169.2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ba</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xing</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Classification and phylogeny of sika deer (Cervus nippon) subspecies based on the mitochondrial control region DNA sequence using an extended sample set.</span> <abbr class="cit-jnl-abbrev">Mitochondrial DNA</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">373</span>–<span class="cit-lpage">379</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMitochondrial%2BDNA%26rft.volume%253D26%26rft.spage%253D373%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3169.3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ba</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">An examination of the origin and evolution of additional tandem repeats in the mitochondrial DNA control region of Japanese sika deer (Cervus Nippon).</span> <abbr class="cit-jnl-abbrev">Mitochondrial DNA A DNA Mapp. Seq. Anal.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">276</span>–<span class="cit-lpage">281</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMitochondrial%2BDNA%2BA%2BDNA%2BMapp.%2BSeq.%2BAnal.%26rft.volume%253D27%26rft.spage%253D276%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3169.4" data-doi="10.1371/journal.pone.0019315"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Baxter</span>  <span class="cit-name-given-names">S. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davey</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnston</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shelton</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heckel</span>  <span class="cit-name-given-names">D. G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Linkage mapping and comparative genomics using next-generation RAD sequencing of a non-model organism.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e19315</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DBaxter%26rft.auinit1%253DS.%2BW.%26rft.volume%253D6%26rft.issue%253D4%26rft.spage%253De19315%26rft.epage%253De19315%26rft.atitle%253DLinkage%2Bmapping%2Band%2Bcomparative%2Bgenomics%2Busing%2Bnext-generation%2BRAD%2Bsequencing%2Bof%2Ba%2Bnon-model%2Borganism.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0019315%26rft_id%253Dinfo%253Apmid%252F21541297%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0019315&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21541297&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3169.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3169.5" data-doi="10.1111/mec.12354"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Catchen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hohenlohe</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bassham</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amores</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cresko</span>  <span class="cit-name-given-names">W. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Stacks: an analysis tool set for population genomics.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">3124</span>–<span class="cit-lpage">3140</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D22%26rft.spage%253D3124%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.12354%26rft_id%253Dinfo%253Apmid%252F23701397%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.12354&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23701397&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3169.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000319675300022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3169.6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Catchen</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amores</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hohenlohe</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cresko</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Postlethwait</span>  <span class="cit-name-given-names">J. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Stacks: building and genotyping Loci de novo from short-read sequences.</span> <abbr class="cit-jnl-abbrev">G3</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">171</span>–<span class="cit-lpage">182</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%26rft.volume%253D1%26rft.spage%253D171%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3169.7" data-doi="10.1371/journal.pone.0106713"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">DaCosta</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorenson</span>  <span class="cit-name-given-names">M. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Amplification biases and consistent recovery of loci in a double-digest RAD-seq protocol.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e106713</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De106713%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0106713%26rft_id%253Dinfo%253Apmid%252F25188270%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0106713&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25188270&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3169.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3169.8" data-doi="10.1038/nrg3012"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Davey</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hohenlohe</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Etter</span>  <span class="cit-name-given-names">P. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boone</span>  <span class="cit-name-given-names">J. Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Catchen</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genome-wide genetic marker discovery and genotyping using next-generation sequencing.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">499</span>–<span class="cit-lpage">510</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DDavey%26rft.auinit1%253DJ.%2BW.%26rft.volume%253D12%26rft.issue%253D7%26rft.spage%253D499%26rft.epage%253D510%26rft.atitle%253DGenome-wide%2Bgenetic%2Bmarker%2Bdiscovery%2Band%2Bgenotyping%2Busing%2Bnext-generation%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3012%26rft_id%253Dinfo%253Apmid%252F21681211%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3012&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21681211&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3169.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3169.9" data-doi="10.1038/ng.806"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">DePristo</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banks</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poplin</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garimella</span>  <span class="cit-name-given-names">K. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maguire</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A framework for variation discovery and genotyping using next-generation DNA sequencing data.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">491</span>–<span class="cit-lpage">498</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DDepristo%26rft.auinit1%253DM.%2BA.%26rft.volume%253D43%26rft.issue%253D5%26rft.spage%253D491%26rft.epage%253D498%26rft.atitle%253DA%2Bframework%2Bfor%2Bvariation%2Bdiscovery%2Band%2Bgenotyping%2Busing%2Bnext-generation%2BDNA%2Bsequencing%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.806%26rft_id%253Dinfo%253Apmid%252F21478889%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.806&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21478889&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3169.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000289972600023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3169.10" data-doi="10.1093/nar/gkn425"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dohm</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lottaz</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Borodina</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Himmelbauer</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Substantial biases in ultra-short read data sets from high-throughput DNA sequencing.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">e105</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkn425%26rft_id%253Dinfo%253Apmid%252F18660515%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkn425&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18660515&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3169.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3169.11" data-doi="10.1186/gb-2009-10-8-r82"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eck</span>  <span class="cit-name-given-names">S. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Benet-Pages</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Flisikowski</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meitinger</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fries</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Whole genome sequencing of a single Bos taurus animal for single nucleotide polymorphism discovery.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">R82</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DEck%26rft.auinit1%253DS.%26rft.volume%253D10%26rft.issue%253D8%26rft.spage%253DR82%26rft.epage%253DR82%26rft.atitle%253DWhole%2Bgenome%2Bsequencing%2Bof%2Ba%2Bsingle%2BBos%2Btaurus%2Banimal%2Bfor%2Bsingle%2Bnucleotide%2Bpolymorphism%2Bdiscovery.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2009-10-8-r82%26rft_id%253Dinfo%253Apmid%252F19660108%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2009-10-8-r82&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19660108&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3169.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3169.12" data-doi="10.1159/000108939"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Graffelman</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Camarena</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Graphical tests for Hardy-Weinberg equilibrium based on the ternary plot.</span> <abbr class="cit-jnl-abbrev">Hum. Hered.</abbr> <span class="cit-vol">65</span>: <span class="cit-fpage">77</span>–<span class="cit-lpage">84</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHuman%2Bheredity%26rft.stitle%253DHum%2BHered%26rft.aulast%253DGraffelman%26rft.auinit1%253DJ.%26rft.volume%253D65%26rft.issue%253D2%26rft.spage%253D77%26rft.epage%253D84%26rft.atitle%253DGraphical%2Btests%2Bfor%2BHardy-Weinberg%2Bequilibrium%2Bbased%2Bon%2Bthe%2Bternary%2Bplot.%26rft_id%253Dinfo%253Adoi%252F10.1159%252F000108939%26rft_id%253Dinfo%253Apmid%252F17898538%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1159/000108939&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17898538&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3169.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3169.13" data-doi="10.1111/j.1365-2052.2010.02038.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Groeneveld</span>  <span class="cit-name-given-names">L. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lenstra</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eding</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Toro</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scherf</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genetic diversity in farm animals–a review.</span> <abbr class="cit-jnl-abbrev">Anim. Genet.</abbr> <span class="cit-vol">41</span>(<span class="cit-supplement">Suppl. 1</span>): <span class="cit-fpage">6</span>–<span class="cit-lpage">31</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnim.%2BGenet.%26rft.volume%253D41%26rft.spage%253D6%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2052.2010.02038.x%26rft_id%253Dinfo%253Apmid%252F20500753%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-2052.2010.02038.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20500753&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3169.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3169.14" data-doi="10.1016/j.ympev.2006.06.020"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guha</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goyal</span>  <span class="cit-name-given-names">S. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kashyap</span>  <span class="cit-name-given-names">V. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Molecular phylogeny of musk deer: a genomic view with mitochondrial 16S rRNA and cytochrome b gene.</span> <abbr class="cit-jnl-abbrev">Mol. Phylogenet. Evol.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">585</span>–<span class="cit-lpage">597</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bphylogenetics%2Band%2Bevolution%26rft.stitle%253DMol%2BPhylogenet%2BEvol%26rft.aulast%253DGuha%26rft.auinit1%253DS.%26rft.volume%253D42%26rft.issue%253D3%26rft.spage%253D585%26rft.epage%253D597%26rft.atitle%253DMolecular%2Bphylogeny%2Bof%2Bmusk%2Bdeer%253A%2Ba%2Bgenomic%2Bview%2Bwith%2Bmitochondrial%2B16S%2BrRNA%2Band%2Bcytochrome%2Bb%2Bgene.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ympev.2006.06.020%26rft_id%253Dinfo%253Apmid%252F17158073%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ympev.2006.06.020&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17158073&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3169.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3169.15" data-doi="10.1080/10635150390192726"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hassanin</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Douzery</span>  <span class="cit-name-given-names">E. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Molecular and morphological phylogenies of ruminantia and the alternative position of the moschidae.</span> <abbr class="cit-jnl-abbrev">Syst. Biol.</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">206</span>–<span class="cit-lpage">228</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSyst.%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1080%252F10635150390192726%26rft_id%253Dinfo%253Apmid%252F12746147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/10635150390192726&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2005040515&amp;link_type=GEOREF" class="cit-ref-sprinkles cit-ref-sprinkles-georef"><span>GeoRef</span></a><a href="/lookup/external-ref?access_num=12746147&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3169.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000181839200004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3169.16" data-doi="10.1371/journal.pone.0036536"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Haynes</span>  <span class="cit-name-given-names">G. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Latch</span>  <span class="cit-name-given-names">E. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Identification of novel single nucleotide polymorphisms (SNPs) in deer (Odocoileus spp.) using the BovineSNP50 BeadChip.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e36536</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DHaynes%26rft.auinit1%253DG.%2BD.%26rft.volume%253D7%26rft.issue%253D5%26rft.spage%253De36536%26rft.epage%253De36536%26rft.atitle%253DIdentification%2Bof%2Bnovel%2Bsingle%2Bnucleotide%2Bpolymorphisms%2B%2528SNPs%2529%2Bin%2Bdeer%2B%2528Odocoileus%2Bspp.%2529%2Busing%2Bthe%2BBovineSNP50%2BBeadChip.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0036536%26rft_id%253Dinfo%253Apmid%252F22590559%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0036536&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22590559&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3169.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3169.17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">He</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Z. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">X. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genetic diversity and population structure of a Sichuan sika deer (Cervus sichuanicus) population in Tiebu Nature Reserve based on microsatellite variation.</span> <abbr class="cit-jnl-abbrev">Dongwuxue Yanjiu</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">528</span>–<span class="cit-lpage">536</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDongwuxue%2BYanjiu%26rft.volume%253D35%26rft.spage%253D528%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3169.18" data-doi="10.1186/1471-2164-10-479"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kerstens</span>  <span class="cit-name-given-names">H. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crooijmans</span>  <span class="cit-name-given-names">R. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Veenendaal</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dibbits</span>  <span class="cit-name-given-names">B. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chin</span>  <span class="cit-name-given-names">A. W. T. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Large scale single nucleotide polymorphism discovery in unsequenced genomes using second generation high throughput sequencing technology: applied to turkey.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">479</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DKerstens%26rft.auinit1%253DH.%2BH.%26rft.volume%253D10%26rft.spage%253D479%26rft.epage%253D479%26rft.atitle%253DLarge%2Bscale%2Bsingle%2Bnucleotide%2Bpolymorphism%2Bdiscovery%2Bin%2Bunsequenced%2Bgenomes%2Busing%2Bsecond%2Bgeneration%2Bhigh%2Bthroughput%2Bsequencing%2Btechnology%253A%2Bapplied%2Bto%2Bturkey.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-10-479%26rft_id%253Dinfo%253Apmid%252F19835600%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-10-479&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19835600&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3169.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3169.19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kraus</span>  <span class="cit-name-given-names">R. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kerstens</span>  <span class="cit-name-given-names">H. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van Hooft</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crooijmans</span>  <span class="cit-name-given-names">R. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van Der Poel</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genome wide SNP discovery, analysis and evaluation in mallard (Anas platyrhynchos).</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">150</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DKraus%26rft.auinit1%253DR.%2BH.%26rft.volume%253D12%26rft.spage%253D150%26rft.epage%253D150%26rft.atitle%253DGenome%2Bwide%2BSNP%2Bdiscovery%252C%2Banalysis%2Band%2Bevaluation%2Bin%2Bmallard%2B%2528Anas%2Bplatyrhynchos%2529.%26rft_id%253Dinfo%253Apmid%252F21410945%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=21410945&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3169.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3169.20" data-doi="10.1093/jhered/est006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Krojerova-Prokesova</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barancekova</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Voloshina</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Myslenkov</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lamka</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Dybowski’s sika deer (Cervus nippon hortulorum): genetic divergence between natural primorian and introduced Czech populations.</span> <abbr class="cit-jnl-abbrev">J. Hered.</abbr> <span class="cit-vol">104</span>: <span class="cit-fpage">312</span>–<span class="cit-lpage">326</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BHered.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjhered%252Fest006%26rft_id%253Dinfo%253Apmid%252F23454911%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/jhered/est006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23454911&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3169.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3169.21" data-doi="10.1038/nmeth.1923"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Langmead</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salzberg</span>  <span class="cit-name-given-names">S. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Fast gapped-read alignment with Bowtie 2.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">357</span>–<span class="cit-lpage">359</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DLangmead%26rft.auinit1%253DB.%26rft.volume%253D9%26rft.issue%253D4%26rft.spage%253D357%26rft.epage%253D359%26rft.atitle%253DFast%2Bgapped-read%2Balignment%2Bwith%2BBowtie%2B2.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1923%26rft_id%253Dinfo%253Apmid%252F22388286%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1923&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22388286&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3169.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000302218500017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3169.22" data-doi="10.1101/gr.113084.110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Le</span>  <span class="cit-name-given-names">S. Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durbin</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">SNP detection and genotyping from low-coverage sequencing data on multiple diploid samples.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">952</span>–<span class="cit-lpage">960</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.113084.110%26rft_id%253Dinfo%253Apmid%252F20980557%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjIxLzYvOTUyIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy85LzMxNjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3169.23" data-doi="10.1093/bioinformatics/btr509"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A statistical framework for SNP calling, mutation discovery, association mapping and population genetical parameter estimation from sequencing data.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">2987</span>–<span class="cit-lpage">2993</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtr509%26rft_id%253Dinfo%253Apmid%252F21903627%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btr509&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21903627&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3169.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000296099300009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3169.24"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">X. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wei</span>  <span class="cit-name-given-names">F. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Genetic diversity among Chinese sika deer (Cervus nippon) populations and relationships between Chinese and Japanese sika deer.</span> <abbr class="cit-jnl-abbrev">Chin. Sci. Bull.</abbr> <span class="cit-vol">51</span>: <span class="cit-fpage">433</span>–<span class="cit-lpage">440</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DChin.%2BSci.%2BBull.%26rft.volume%253D51%26rft.spage%253D433%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3169.25" data-doi="10.1093/bioinformatics/btq092"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Malhis</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">S. J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">High quality SNP calling using Illumina data at shallow coverage.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">1029</span>–<span class="cit-lpage">1035</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq092%26rft_id%253Dinfo%253Apmid%252F20190250%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btq092&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20190250&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3169.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000276737100008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-book" id="cit-7.9.3169.26"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McCullough</span>  <span class="cit-name-given-names">E. D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Takatsuki</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kaji</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Sika deer: biology and management of native and introduced populations</span>, pp. <span class="cit-fpage">526</span>–<span class="cit-lpage">531</span> in <span class="cit-source"><em>Sika Deer in Mainland China</em></span>, edited by <span class="cit-ed"><span class="cit-name-surname">McCullough</span>  <span class="cit-name-given-names">D. R.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">Z.-G.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">C.-W.</span></span>. <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">Tokyo</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3169.27"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Patterson</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Price</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reich</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Population structure and eigenanalysis.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">2074</span>–<span class="cit-lpage">2093</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D2%26rft.spage%253D2074%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=000243482100012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3169.28" data-doi="10.1371/journal.pone.0037135"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Peterson</span>  <span class="cit-name-given-names">B. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weber</span>  <span class="cit-name-given-names">J. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kay</span>  <span class="cit-name-given-names">E. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fisher</span>  <span class="cit-name-given-names">H. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoekstra</span>  <span class="cit-name-given-names">H. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Double digest RADseq: an inexpensive method for de novo SNP discovery and genotyping in model and non-model species.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e37135</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DPeterson%26rft.auinit1%253DB.%2BK.%26rft.volume%253D7%26rft.issue%253D5%26rft.spage%253De37135%26rft.epage%253De37135%26rft.atitle%253DDouble%2Bdigest%2BRADseq%253A%2Ban%2Binexpensive%2Bmethod%2Bfor%2Bde%2Bnovo%2BSNP%2Bdiscovery%2Band%2Bgenotyping%2Bin%2Bmodel%2Band%2Bnon-model%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0037135%26rft_id%253Dinfo%253Apmid%252F22675423%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0037135&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22675423&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3169.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3169.29"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Powell</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amish</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haynes</span>  <span class="cit-name-given-names">G. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luikart</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Latch</span>  <span class="cit-name-given-names">E. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Candidate adaptive genes associated with lineage divergence: identifying SNPs via next-generation targeted resequencing in mule deer (Odocoileus hemionus).</span> <abbr class="cit-jnl-abbrev">Mol. Ecol. Resour.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">1165</span>–<span class="cit-lpage">1172</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%2BResour.%26rft.volume%253D16%26rft.spage%253D1165%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3169.30" data-doi="10.1111/j.1471-8286.2005.01155.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pritchard</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Donnelly</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Inference of population structure using multilocus genotype data.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">155</span>: <span class="cit-fpage">945</span>–<span class="cit-lpage">959</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DPritchard%26rft.auinit1%253DJ.%2BK.%26rft.volume%253D155%26rft.issue%253D2%26rft.spage%253D945%26rft.epage%253D959%26rft.atitle%253DInference%2Bof%2BPopulation%2BStructure%2BUsing%2BMultilocus%2BGenotype%2BData%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1471-8286.2005.01155.x%26rft_id%253Dinfo%253Apmid%252F10835412%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTU1LzIvOTQ1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy85LzMxNjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3169.31" data-doi="10.1371/journal.pone.0015811"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Seabury</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bhattarai</span>  <span class="cit-name-given-names">E. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Viswanathan</span>  <span class="cit-name-given-names">G. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cooper</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genome-wide polymorphism and comparative analyses in the white-tailed deer (Odocoileus virginianus): a model for conservation genomics.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e15811</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DSeabury%26rft.auinit1%253DC.%2BM.%26rft.volume%253D6%26rft.issue%253D1%26rft.spage%253De15811%26rft.epage%253De15811%26rft.atitle%253DGenome-wide%2Bpolymorphism%2Band%2Bcomparative%2Banalyses%2Bin%2Bthe%2Bwhite-tailed%2Bdeer%2B%2528Odocoileus%2Bvirginianus%2529%253A%2Ba%2Bmodel%2Bfor%2Bconservation%2Bgenomics.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0015811%26rft_id%253Dinfo%253Apmid%252F21283515%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0015811&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21283515&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3169.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-book" id="cit-7.9.3169.32"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sheng</span>  <span class="cit-name-given-names">H. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ohtaishi</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-source"><em>Deer of China: Biology and Management</em></span>. <span class="cit-publ-name">Elsevier Science</span>, <span class="cit-publ-loc">Amsterdam</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-7.9.3169.33"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Szyda</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fraszczak</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mielczarek</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Giannico</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Minozzi</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">The assessment of inter-individual variation of whole-genome DNA sequence in 32 cows.</span> <abbr class="cit-jnl-abbrev">Mamm. Genome</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">658</span>–<span class="cit-lpage">665</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMamm.%2BGenome%26rft.volume%253D26%26rft.spage%253D658%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3169.34"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Takiguchi</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tanaka</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ono</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoshi</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Minami</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genetic variation and population structure of the Japanese sika deer (Cervus nippon) in the Tohoku District based on mitochondrial D-loop sequences.</span> <abbr class="cit-jnl-abbrev">Zoolog. Sci.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">433</span>–<span class="cit-lpage">436</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DZoological%2Bscience%26rft.stitle%253DZoolog%2BSci%26rft.aulast%253DTakiguchi%26rft.auinit1%253DH.%26rft.volume%253D29%26rft.issue%253D7%26rft.spage%253D433%26rft.epage%253D436%26rft.atitle%253DGenetic%2Bvariation%2Band%2Bpopulation%2Bstructure%2Bof%2Bthe%2BJapanese%2Bsika%2Bdeer%2B%2528Cervus%2Bnippon%2529%2Bin%2Bthe%2BTohoku%2BDistrict%2Bbased%2Bon%2Bmitochondrial%2BD-loop%2Bsequences.%26rft_id%253Dinfo%253Apmid%252F22775251%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=22775251&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3169.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3169.35" data-doi="10.2108/zsj.17.47"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tamate</span>  <span class="cit-name-given-names">H. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Okada</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Minami</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ohnishi</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Higuchi</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Genetic variations revealed by microsatellite markers ina small population of the sika deer (Cervus nippon)on Kinkazan Island, northern Japan.</span> <abbr class="cit-jnl-abbrev">Zoolog. Sci.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">47</span>–<span class="cit-lpage">53</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DZoological%2Bscience%26rft.stitle%253DZoolog%2BSci%26rft.aulast%253DTamate%26rft.auinit1%253DH.%2BB.%26rft.volume%253D17%26rft.issue%253D1%26rft.spage%253D47%26rft.epage%253D53%26rft.atitle%253DGenetic%2BVariations%2BRevealed%2Bby%2BMicrosatellite%2BMarkers%2Bina%2BSmall%2BPopulation%2Bof%2Bthe%2BSika%2BDeer%2B%2528Cervus%2Bnippon%2529on%2BKinkazan%2BIsland%252C%2BNorthern%2BJapan.%26rft_id%253Dinfo%253Adoi%252F10.2108%252Fzsj.17.47%26rft_id%253Dinfo%253Apmid%252F18494571%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2108/zsj.17.47&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18494571&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3169.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3169.36" data-doi="10.1093/jhered/esh001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thevenon</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thuy</span>  <span class="cit-name-given-names">L. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ly</span>  <span class="cit-name-given-names">L. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maudet</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bonnet</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Microsatellite analysis of genetic diversity of the Vietnamese sika deer (Cervus nippon pseudaxis).</span> <abbr class="cit-jnl-abbrev">J. Hered.</abbr> <span class="cit-vol">95</span>: <span class="cit-fpage">11</span>–<span class="cit-lpage">18</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BHeredity%26rft.stitle%253DJ%2BHered%26rft.aulast%253DThevenon%26rft.auinit1%253DS.%26rft.volume%253D95%26rft.issue%253D1%26rft.spage%253D11%26rft.epage%253D18%26rft.atitle%253DMicrosatellite%2BAnalysis%2Bof%2BGenetic%2BDiversity%2Bof%2Bthe%2BVietnamese%2BSika%2BDeer%2B%2528Cervus%2Bnippon%2Bpseudaxis%2529%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjhered%252Fesh001%26rft_id%253Dinfo%253Apmid%252F14757725%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/jhered/esh001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14757725&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3169.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000188797900002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3169.37"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yan</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yi</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Qu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Discovery of genome-wideSNPs by RAD-seqand the genetic diversity of captive hog deer (Axis porcinus).</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">e0174299</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D12%26rft.spage%253De0174299%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3169.38"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wan</span>  <span class="cit-name-given-names">Q. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fang</span>  <span class="cit-name-given-names">S. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Two genetically distinct units of the Chinese sika deer (Cervus nippon): analyses of mitochondrial DNA variation.</span> <abbr class="cit-jnl-abbrev">Biol. Conserv.</abbr> <span class="cit-vol">119</span>: <span class="cit-fpage">183</span>–<span class="cit-lpage">190</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiol.%2BConserv.%26rft.volume%253D119%26rft.spage%253D183%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3169.39"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wan</span>  <span class="cit-name-given-names">Q. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fang</span>  <span class="cit-name-given-names">S. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">S. Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Application of mitochondrial DNA sequence analysis in the forensic identification of Chinese sika deer subspecies.</span> <abbr class="cit-jnl-abbrev">Forensic Sci. Int.</abbr> <span class="cit-vol">148</span>: <span class="cit-fpage">101</span>–<span class="cit-lpage">105</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DForensic%2BScience%2BInternational%26rft.stitle%253DForensic%2BScience%2BInternational%26rft.aulast%253DWu%26rft.auinit1%253DH.%26rft.volume%253D148%26rft.issue%253D2-3%26rft.spage%253D101%26rft.epage%253D105%26rft.atitle%253DApplication%2Bof%2Bmitochondrial%2BDNA%2Bsequence%2Banalysis%2Bin%2Bthe%2Bforensic%2Bidentification%2Bof%2BChinese%2Bsika%2Bdeer%2Bsubspecies.%26rft_id%253Dinfo%253Apmid%252F15639603%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=15639603&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3169.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3169.40"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">E. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">The resource conservation and utilization of wild sika deer in China.</span> <abbr class="cit-jnl-abbrev">Zhong Yao Cai</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">552</span>–<span class="cit-lpage">554 (in Chinese)</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DZhong%2Byao%2Bcai%2B%253D%2BZhongyaocai%2B%253D%2BJournal%2Bof%2BChinese%2Bmedicinal%2Bmaterials%26rft.stitle%253DZhong%2BYao%2BCai%26rft.aulast%253DWu%26rft.auinit1%253DP.%26rft.volume%253D24%26rft.issue%253D8%26rft.spage%253D552%26rft.epage%253D554%26rft.atitle%253D%255BThe%2Bresource%2Bconservation%2Band%2Butilization%2Bof%2Bwild%2BSika%2Bdeer%2Bin%2BChina%255D%26rft_id%253Dinfo%253Apmid%252F11715188%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=11715188&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3169.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-7.9.3169.41" data-doi="10.1101/073528"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Deng</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ding</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ren</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Comparative genomics and metagenomics analyses of endangered Père David’s deer (Elaphurus davidianus) provide insights into population recovery.</span> <abbr class="cit-jnl-abbrev">bioRxiv</abbr><span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">DOI: </span><span class="cit-pub-id cit-pub-id-doi">10.1101/073528</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DbioRxiv%26rft_id%253Dinfo%253Adoi%252F10.1101%252F073528%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvcnhpdiI7czo1OiJyZXNpZCI7czo4OiIwNzM1Mjh2MSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvOS8zMTY5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/7/9/3169.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/9/3157" title="The Douglas-Fir Genome Sequence Reveals Specialization of the Photosynthetic Apparatus in Pinaceae" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/9/3177" title="The Stress-Inducible Peroxidase TSA2 Underlies a Conditionally Beneficial Chromosomal Duplication in Saccharomyces cerevisiae" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/7/9"><span>Volume 7 Issue 9, September 2017</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/7/9.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/7/9.cover-source.jpg?itok=WuYpGT8r" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 7 (9)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/9/3169" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-ukXC2pnaeUUJ1nRonwB5YT9Wa-rgwmxYzXye9-XWsCE" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/genomic-selection" class="" data-icon-position="" data-hide-link-title="0">Genomic Selection</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/7/9/3169" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/7/9/3169" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/7/9/3169" class="active" data-icon-position="" data-hide-link-title="0">Genome-Wide SNP Discovery and Analysis of Genetic Diversity in Farmed Sika Deer (Cervus nippon) in Northeast China Using Double-Digest Restriction Site-Associated DNA Sequencing</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/486356" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-r1vko7Oy926n7XvxaO7KvEMdDpCQCrJpXgzlkJkTQUA" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/9/3169.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/7/9/3169" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-WisBSsCRPsckxjK_N0gSAc0utXEu8BCH_E5-ZW_P1SQ" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/7/9/3169" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/9/3169.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="486356" id="node486356--2" data-pisa="ggg;7/9/3169" data-pisa-master="ggg;g3.117.300082" data-apath="/ggg/7/9/3169.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genome-Wide SNP Discovery and Analysis of Genetic Diversity in Farmed Sika Deer (<em>Cervus nippon</em>) in Northeast China Using Double-Digest Restriction Site-Associated DNA Sequencing</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Hengxing Ba</span>, <span class="highwire-citation-author" data-delta="1">Boyin Jia</span>, <span class="highwire-citation-author" data-delta="2">Guiwu Wang</span>, <span class="highwire-citation-author" data-delta="3">Yifeng Yang</span>, <span class="highwire-citation-author" data-delta="4">Gilead Kedem</span> and <span class="highwire-citation-author" data-delta="5">Chunyi Li</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3169-3176;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300082 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node486356--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Hengxing Ba </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>State Key Laboratory for Molecular Biology of Special Wild Economic Animals, Institute of Special Wild Economic Animals and Plants, Chinese Academy of Agricultural Sciences, Changchun 130112, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hengxing%2BBa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ba%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHengxing%2BBa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Boyin Jia </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>State Key Laboratory for Molecular Biology of Special Wild Economic Animals, Institute of Special Wild Economic Animals and Plants, Chinese Academy of Agricultural Sciences, Changchun 130112, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Boyin%2BJia%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jia%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABoyin%2BJia%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Guiwu Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>State Key Laboratory for Molecular Biology of Special Wild Economic Animals, Institute of Special Wild Economic Animals and Plants, Chinese Academy of Agricultural Sciences, Changchun 130112, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Guiwu%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGuiwu%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Yifeng Yang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>State Key Laboratory for Molecular Biology of Special Wild Economic Animals, Institute of Special Wild Economic Animals and Plants, Chinese Academy of Agricultural Sciences, Changchun 130112, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yifeng%2BYang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yang%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYifeng%2BYang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Gilead Kedem </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>GENEWIZ Inc., South Plainfield, New Jersey 07080</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gilead%2BKedem%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kedem%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGilead%2BKedem%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Chunyi Li </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>State Key Laboratory for Molecular Biology of Special Wild Economic Animals, Institute of Special Wild Economic Animals and Plants, Chinese Academy of Agricultural Sciences, Changchun 130112, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Chunyi%2BLi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Li%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AChunyi%2BLi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:lichunyi1959@163.com" class="" data-icon-position="" data-hide-link-title="0">lichunyi1959@163.com</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/7/9/3169" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/7/9/3169&amp;title=Genome-Wide%20SNP%20Discovery%20and%20Analysis%20of%20Genetic%20Diversity%20in%20Farmed%20Sika%20Deer%20%28Cervus%20nippon%29%20in%20Northeast%20China%20Using%20Double-Digest%20Restriction%20Site-Associated%20DNA%20Sequencing" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/7/9/3169&amp;title=Genome-Wide%20SNP%20Discovery%20and%20Analysis%20of%20Genetic%20Diversity%20in%20Farmed%20Sika%20Deer%20%28Cervus%20nippon%29%20in%20Northeast%20China%20Using%20Double-Digest%20Restriction%20Site-Associated%20DNA%20Sequencing" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/7/9/3169&amp;title=Genome-Wide%20SNP%20Discovery%20and%20Analysis%20of%20Genetic%20Diversity%20in%20Farmed%20Sika%20Deer%20%28Cervus%20nippon%29%20in%20Northeast%20China%20Using%20Double-Digest%20Restriction%20Site-Associated%20DNA%20Sequencing" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/7/9/3169&amp;text=Genome-Wide%20SNP%20Discovery%20and%20Analysis%20of%20Genetic%20Diversity%20in%20Farmed%20Sika%20Deer%20%28Cervus%20nippon%29%20in%20Northeast%20China%20Using%20Double-Digest%20Restriction%20Site-Associated%20DNA%20Sequencing" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/7/9/3169&amp;title=Genome-Wide%20SNP%20Discovery%20and%20Analysis%20of%20Genetic%20Diversity%20in%20Farmed%20Sika%20Deer%20%28Cervus%20nippon%29%20in%20Northeast%20China%20Using%20Double-Digest%20Restriction%20Site-Associated%20DNA%20Sequencing" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/7/9/3169&amp;t=Genome-Wide%20SNP%20Discovery%20and%20Analysis%20of%20Genetic%20Diversity%20in%20Farmed%20Sika%20Deer%20%28Cervus%20nippon%29%20in%20Northeast%20China%20Using%20Double-Digest%20Restriction%20Site-Associated%20DNA%20Sequencing" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/7/9/3169&amp;title=Genome-Wide%20SNP%20Discovery%20and%20Analysis%20of%20Genetic%20Diversity%20in%20Farmed%20Sika%20Deer%20%28Cervus%20nippon%29%20in%20Northeast%20China%20Using%20Double-Digest%20Restriction%20Site-Associated%20DNA%20Sequencing" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/7/9/3169&amp;title=Genome-Wide%20SNP%20Discovery%20and%20Analysis%20of%20Genetic%20Diversity%20in%20Farmed%20Sika%20Deer%20%28Cervus%20nippon%29%20in%20Northeast%20China%20Using%20Double-Digest%20Restriction%20Site-Associated%20DNA%20Sequencing" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/486356/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="486356" id="node486356--3" data-pisa="ggg;7/9/3169" data-pisa-master="ggg;g3.117.300082" data-apath="/ggg/7/9/3169.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genome-Wide SNP Discovery and Analysis of Genetic Diversity in Farmed Sika Deer (<em>Cervus nippon</em>) in Northeast China Using Double-Digest Restriction Site-Associated DNA Sequencing</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Hengxing Ba</span>, <span class="highwire-citation-author" data-delta="1">Boyin Jia</span>, <span class="highwire-citation-author" data-delta="2">Guiwu Wang</span>, <span class="highwire-citation-author" data-delta="3">Yifeng Yang</span>, <span class="highwire-citation-author" data-delta="4">Gilead Kedem</span> and <span class="highwire-citation-author" data-delta="5">Chunyi Li</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3169-3176;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300082 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node486356--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Hengxing Ba </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>State Key Laboratory for Molecular Biology of Special Wild Economic Animals, Institute of Special Wild Economic Animals and Plants, Chinese Academy of Agricultural Sciences, Changchun 130112, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hengxing%2BBa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ba%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHengxing%2BBa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Boyin Jia </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>State Key Laboratory for Molecular Biology of Special Wild Economic Animals, Institute of Special Wild Economic Animals and Plants, Chinese Academy of Agricultural Sciences, Changchun 130112, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Boyin%2BJia%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jia%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABoyin%2BJia%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Guiwu Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>State Key Laboratory for Molecular Biology of Special Wild Economic Animals, Institute of Special Wild Economic Animals and Plants, Chinese Academy of Agricultural Sciences, Changchun 130112, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Guiwu%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGuiwu%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Yifeng Yang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>State Key Laboratory for Molecular Biology of Special Wild Economic Animals, Institute of Special Wild Economic Animals and Plants, Chinese Academy of Agricultural Sciences, Changchun 130112, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yifeng%2BYang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yang%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYifeng%2BYang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Gilead Kedem </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>GENEWIZ Inc., South Plainfield, New Jersey 07080</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gilead%2BKedem%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kedem%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGilead%2BKedem%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Chunyi Li </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>State Key Laboratory for Molecular Biology of Special Wild Economic Animals, Institute of Special Wild Economic Animals and Plants, Chinese Academy of Agricultural Sciences, Changchun 130112, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Chunyi%2BLi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Li%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AChunyi%2BLi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:lichunyi1959@163.com" class="" data-icon-position="" data-hide-link-title="0">lichunyi1959@163.com</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=3169&amp;rft.epage=3176&amp;rft.atitle=Genome-Wide%20SNP%20Discovery%20and%20Analysis%20of%20Genetic%20Diversity%20in%20Farmed%20Sika%20Deer%20%28Cervus%20nippon%29%20in%20Northeast%20China%20Using%20Double-Digest%20Restriction%20Site-Associated%20DNA%20Sequencing&amp;rft.volume=7&amp;rft.issue=9&amp;rft.date=2017-09-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Ba%2C+Hengxing&amp;rft.au=Jia%2C+Boyin&amp;rft.au=Wang%2C+Guiwu&amp;rft.au=Yang%2C+Yifeng&amp;rft.au=Kedem%2C+Gilead&amp;rft.au=Li%2C+Chunyi"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/486356/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/486356/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/486356/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/486356/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/486356/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/486356/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/486356/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/486356/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/486356/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/486356/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/486356/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/486356/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490720" id="node490720" data-pisa="ggg;8/9/3019" data-pisa-master="ggg;g3.118.200430" data-apath="/ggg/8/9/3019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/3019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">A Bayesian Decision Theory Approach for Genomic Selection</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490724" id="node490724" data-pisa="ggg;8/9/2889" data-pisa-master="ggg;g3.118.200311" data-apath="/ggg/8/9/2889.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/2889" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Optimising Genomic Selection in Wheat: Effect of Marker Density, Population Size and Population Structure on Prediction Accuracy</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490727" id="node490727" data-pisa="ggg;8/9/3039" data-pisa-master="ggg;g3.118.200435" data-apath="/ggg/8/9/3039.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/3039" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">BGGE: A New Package for Genomic-Enabled Prediction Incorporating Genotype × Environment Interaction Models</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Genomic%20Selection" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Genomic Selection</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/7/9/3169" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-10" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-16" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/7/9/3169.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/7/9/3169.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/7/9/3169.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/7/9/3169.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
