<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/7/9/3203" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/7/9/3203.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/7/9/3203.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/7/9/3203.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/7/9/3203.atom" />
<meta name="HW.pisa" content="ggg;7/9/3203" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="A Functional Link Between Bir1 and the Saccharomyces cerevisiae Ctf19 Kinetochore Complex Revealed Through Quantitative Fitness Analysis" />
<meta name="DC.Identifier" content="10.1534/g3.117.300089" />
<meta name="DC.Date" content="2017-09-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2017 Makrantoni et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="The chromosomal passenger complex (CPC) is a key regulator of eukaryotic cell division, consisting of the protein kinase Aurora B/[Ipl1][1] in association with its activator (INCENP/[Sli15][2]) and two additional proteins (Survivin/[Bir1][3] and Borealin/[Nbl1][4]). Here, we report a genome-wide genetic interaction screen in Saccharomyces cerevisiae using the [bir1][3]-17 mutant, identifying through quantitative fitness analysis deletion mutations that act as enhancers and suppressors. Gene knockouts affecting the [Ctf19][5] kinetochore complex were identified as the strongest enhancers of [bir1][3]-17 , while mutations affecting the large ribosomal subunit or the mRNA nonsense-mediated decay pathway caused strong phenotypic suppression. Thus, cells lacking a functional [Ctf19][5] complex become highly dependent on [Bir1][3] function and vice versa. The negative genetic interaction profiles of [bir1][3]-17 and the cohesin mutant [mcd1][6]-1 showed considerable overlap, underlining the strong functional connection between sister chromatid cohesion and chromosome biorientation. Loss of some [Ctf19][5] components, such as [Iml3][7] or [Chl4][8], impacted differentially on [bir1][3]-17 compared with mutations affecting other CPC components: despite the synthetic lethality shown by either [iml3][7] ∆ or [chl4][8] ∆ in combination with [bir1][3]-17 , neither gene knockout showed any genetic interaction with either [ipl1][1]-321 or [sli15][2]-3 . Our data therefore imply a specific functional connection between the [Ctf19][5] complex and [Bir1][3] that is not shared with [Ipl1][1].

 [1]: http://www.yeastgenome.org/locus/S000006130/overview
 [2]: http://www.yeastgenome.org/locus/S000000360/overview
 [3]: http://www.yeastgenome.org/locus/S000003849/overview
 [4]: http://www.yeastgenome.org/locus/S000029704/overview
 [5]: http://www.yeastgenome.org/locus/S000005939/overview
 [6]: http://www.yeastgenome.org/locus/S000002161/overview
 [7]: http://www.yeastgenome.org/locus/S000000311/overview
 [8]: http://www.yeastgenome.org/locus/S000002662/overview" />
<meta name="DC.Contributor" content="Vasso Makrantoni" />
<meta name="DC.Contributor" content="Adam Ciesiolka" />
<meta name="DC.Contributor" content="Conor Lawless" />
<meta name="DC.Contributor" content="Josefin Fernius" />
<meta name="DC.Contributor" content="Adele Marston" />
<meta name="DC.Contributor" content="David Lydall" />
<meta name="DC.Contributor" content="Michael J. R. Stark" />
<meta name="article:published_time" content="2017-09-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="A Functional Link Between Bir1 and the Saccharomyces cerevisiae Ctf19 Kinetochore Complex Revealed Through Quantitative Fitness Analysis" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;The chromosomal passenger complex (CPC) is a key regulator of eukaryotic cell division, consisting of the protein kinase Aurora B/Ipl1 in association with its activator (INCENP/Sli15) and two additional proteins (Survivin/Bir1 and Borealin/Nbl1). Here, we report a genome-wide genetic interaction screen in &lt;i&gt;Saccharomyces cerevisiae&lt;/i&gt; using the &lt;i&gt;bir1-17&lt;/i&gt; mutant, identifying through quantitative fitness analysis deletion mutations that act as enhancers and suppressors. Gene knockouts affecting the Ctf19 kinetochore complex were identified as the strongest enhancers of &lt;i&gt;bir1-17&lt;/i&gt;, while mutations affecting the large ribosomal subunit or the mRNA nonsense-mediated decay pathway caused strong phenotypic suppression. Thus, cells lacking a functional Ctf19 complex become highly dependent on Bir1 function and vice versa. The negative genetic interaction profiles of &lt;i&gt;bir1-17&lt;/i&gt; and the cohesin mutant &lt;i&gt;mcd1-1&lt;/i&gt; showed considerable overlap, underlining the strong functional connection between sister chromatid cohesion and chromosome biorientation. Loss of some Ctf19 components, such as Iml3 or Chl4, impacted differentially on &lt;i&gt;bir1-17&lt;/i&gt; compared with mutations affecting other CPC components: despite the synthetic lethality shown by either &lt;i&gt;iml3&lt;/i&gt;∆ or &lt;i&gt;chl4&lt;/i&gt;∆ in combination with &lt;i&gt;bir1-17&lt;/i&gt;, neither gene knockout showed any genetic interaction with either &lt;i&gt;ipl1-321&lt;/i&gt; or &lt;i&gt;sli15-3&lt;/i&gt;. Our data therefore imply a specific functional connection between the Ctf19 complex and Bir1 that is not shared with Ipl1.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2017/09/01" />
<meta name="citation_mjid" content="ggg;7/9/3203" />
<meta name="citation_id" content="7/9/3203" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/7/9/3203" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/7/9/3203.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/7/9/3203.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/7/9/3203.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.117.300089" />
<meta name="citation_pmid" content="28754723" />
<meta name="citation_volume" content="7" />
<meta name="citation_issue" content="9" />
<meta name="citation_num_pages" content="13" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="3203" />
<meta name="citation_lastpage" content="3215" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Vasso Makrantoni" />
<meta name="citation_author_institution" content="Centre for Gene Regulation and Expression, University of Dundee, DD1 5EH, UK" />
<meta name="citation_author_institution" content="Wellcome Trust Centre for Cell Biology, University of Edinburgh, EH9 3BF, UK" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-0668-6157" />
<meta name="citation_author" content="Adam Ciesiolka" />
<meta name="citation_author_institution" content="Institute for Cell and Molecular Biosciences, Newcastle University, NE2 4HH, UK" />
<meta name="citation_author" content="Conor Lawless" />
<meta name="citation_author_institution" content="Institute for Cell and Molecular Biosciences, Newcastle University, NE2 4HH, UK" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-4186-8506" />
<meta name="citation_author" content="Josefin Fernius" />
<meta name="citation_author_institution" content="Wellcome Trust Centre for Cell Biology, University of Edinburgh, EH9 3BF, UK" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-9582-4242" />
<meta name="citation_author" content="Adele Marston" />
<meta name="citation_author_institution" content="Wellcome Trust Centre for Cell Biology, University of Edinburgh, EH9 3BF, UK" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-3596-9407" />
<meta name="citation_author" content="David Lydall" />
<meta name="citation_author_institution" content="Institute for Cell and Molecular Biosciences, Newcastle University, NE2 4HH, UK" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-2478-085X" />
<meta name="citation_author" content="Michael J. R. Stark" />
<meta name="citation_author_institution" content="Centre for Gene Regulation and Expression, University of Dundee, DD1 5EH, UK" />
<meta name="citation_author_email" content="m.j.r.stark@dundee.ac.uk" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-9086-191X" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. G. Addinall;citation_author=M. Downey;citation_author=M. Yu;citation_author=M. K. Zubko;citation_author=J. Dewar;citation_title=A genomewide suppressor and enhancer analysis of cdc13–1 reveals varied cellular processes influencing telomere capping in Saccharomyces cerevisiae.;citation_volume=180;citation_year=2008;citation_pmid=18845848;citation_doi=10.1534/genetics.108.092577" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=S. G. Addinall;citation_author=E. M. Holstein;citation_author=C. Lawless;citation_author=M. Yu;citation_author=K. Chapman;citation_title=Quantitative fitness analysis shows that NMD proteins and many other protein complexes suppress or enhance distinct telomere cap defects.;citation_pages=e1001362-e1001362;citation_volume=7;citation_year=2011;citation_issue=4;citation_pmid=21490951;citation_doi=10.1371/journal.pgen.1001362" />
<meta name="citation_reference" content="citation_title=Methods in yeast genetics. A Cold Spring Harbor Laboratory Course Manual;citation_author=D. C. Amberg;citation_author=D. J. Burke;citation_author=J. N. Strathern;citation_year=2005" />
<meta name="citation_reference" content="citation_journal_title=J. Biol. Chem.;citation_author=M. Bando;citation_author=Y. Katou;citation_author=M. Komata;citation_author=H. Tanaka;citation_author=T. Itoh;citation_title=Csm3, Tof1, and Mrc1 form a heterotrimeric mediator complex that associates with DNA replication forks.;citation_volume=284;citation_year=2009;citation_pmid=19819872;citation_doi=10.1074/jbc.M109.065730" />
<meta name="citation_reference" content="citation_journal_title=Current biology :  CB;citation_journal_abbrev=Curr Biol;citation_author=D. L. Beach;citation_author=J. Thibodeaux;citation_author=P. Maddox;citation_author=E. Yeh;citation_author=K. Bloom;citation_title=The role of the proteins Kar9 and Myo2 in orienting the mitotic spindle of budding yeast.;citation_pages=1497-1506;citation_volume=10;citation_year=2000;citation_issue=23;citation_pmid=11114516;citation_doi=10.1016/S0960-9822(00)00837-X" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=J. J. Benschop;citation_author=N. Brabers;citation_author=D. van Leenen;citation_author=L. V. Bakker;citation_author=H. W. van Deutekom;citation_title=A consensus of core protein complex compositions for Saccharomyces cerevisiae.;citation_pages=916-928;citation_volume=38;citation_year=2010;citation_issue=6;citation_pmid=20620961;citation_doi=10.1016/j.molcel.2010.06.002" />
<meta name="citation_reference" content="citation_journal_title=The Journal of Cell Biology;citation_journal_abbrev=JCB;citation_author=V. Berlin;citation_author=C. A. Styles;citation_author=G. R. Fink;citation_title=BIK1, a protein required for microtubule function during mating and mitosis in Saccharomyces cerevisiae, colocalizes with tubulin;citation_pages=2573-2586;citation_volume=111;citation_year=1990;citation_issue=6;citation_pmid=2277073;citation_doi=10.1083/jcb.111.6.2573" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=S. Biggins;citation_title=The Composition, Functions, and Regulation of the Budding Yeast Kinetochore;citation_pages=817-846;citation_volume=194;citation_year=2013;citation_issue=4;citation_pmid=23908374;citation_doi=10.1534/genetics.112.145276" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=S. Biggins;citation_author=F. F. Severin;citation_author=N. Bhalla;citation_author=I. Sassoon;citation_author=A. A. Hyman;citation_title=The conserved protein kinase Ipl1 regulates microtubule binding to kinetochores in budding yeast;citation_pages=532-544;citation_volume=13;citation_year=1999;citation_issue=5;citation_pmid=10072382;citation_doi=10.1101/gad.13.5.532" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Cell;citation_author=K. A. Blake-Hodek;citation_author=L. Cassimeris;citation_author=T. C. Huffaker;citation_title=Regulation of microtubule dynamics by Bim1 and Bik1, the budding yeast members of the EB1 and CLIP-170 families of plus-end tracking proteins.;citation_volume=21;citation_year=2010;citation_pmid=20392838;citation_doi=10.1091/mbc.E10-02-0083" />
<meta name="citation_reference" content="citation_journal_title=Nature Cell Biology;citation_journal_abbrev=Nature Cell Biology;citation_author=L. J. Bock;citation_author=C. Pagliuca;citation_author=N. Kobayashi;citation_author=R. A. Grove;citation_author=Y. Oku;citation_title=Cnn1 inhibits the interactions between the KMN complexes of the yeast kinetochore.;citation_pages=614-624;citation_volume=14;citation_year=2012;citation_issue=6;citation_pmid=22561345;citation_doi=10.1038/ncb2495" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=E. I. Boyle;citation_author=S. Weng;citation_author=J. Gollub;citation_author=H. Jin;citation_author=D. Botstein;citation_title=GO:TermFinder–open source software for accessing gene ontology information and finding significantly enriched gene ontology terms associated with a list of genes.;citation_volume=20;citation_year=2004;citation_pmid=15297299;citation_doi=10.1093/bioinformatics/bth456" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=C. S. Campbell;citation_author=A. Desai;citation_title=Tension sensing by Aurora B kinase is independent of survivin-based centromere localization.;citation_pages=118-121;citation_volume=497;citation_year=2013;citation_pmid=23604256;citation_doi=10.1038/nature12057" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=J. M. Cherry;citation_author=E. L. Hong;citation_author=C. Amundsen;citation_author=R. Balakrishnan;citation_author=G. Binkley;citation_title=Saccharomyces genome database: the genomics resource of budding yeast.;citation_volume=40;citation_year=2012;citation_pmid=22110037;citation_doi=10.1093/nar/gkr1029" />
<meta name="citation_reference" content="citation_journal_title=Nature structural &amp; molecular biology;citation_journal_abbrev=Nat Struct Mol Biol;citation_author=U. S. Cho;citation_author=S. C. Harrison;citation_title=Ndc10 is a platform for inner kinetochore assembly in budding yeast.;citation_pages=48-55;citation_volume=19;citation_year=2012;citation_issue=1;citation_pmid=22139014;citation_doi=10.1038/nsmb.2178" />
<meta name="citation_reference" content="citation_journal_title=Genes Dev.;citation_author=P. De Wulf;citation_author=A. D. McAinsh;citation_author=P. K. Sorger;citation_title=Hierarchical assembly of the budding yeast kinetochore from multiple subcomplexes.;citation_volume=17;citation_year=2003;citation_pmid=14633972;citation_doi=10.1101/gad.1144403" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=J. Fernius;citation_author=A. L. Marston;citation_title=Establishment of cohesion at the pericentromere by the Ctf19 kinetochore subcomplex and the replication fork-associated factor, Csm3.;citation_pages=e1000629-e1000629;citation_volume=5;citation_year=2009;citation_issue=9;citation_pmid=19730685;citation_doi=10.1371/journal.pgen.1000629" />
<meta name="citation_reference" content="citation_journal_title=Curr. Biol.;citation_author=J. Fernius;citation_author=O. O. Nerusheva;citation_author=S. Galander;citation_author=L. Alves Fde;citation_author=J. Rappsilber;citation_title=Cohesin-dependent association of Scc2/4 with the centromere initiates pericentromeric cohesion establishment.;citation_pages=599-606;citation_volume=23;citation_year=2013;citation_pmid=23499533;citation_doi=10.1016/j.cub.2013.02.022" />
<meta name="citation_reference" content="citation_journal_title=J. Cell Biol.;citation_author=S. Fink;citation_author=K. Turnbull;citation_author=A. Desai;citation_author=C. S. Campbell;citation_title=An engineered minimal chromosomal passenger complex reveals a role for INCENP/Sli15 spindle association in chromosome biorientation.;citation_volume=216;citation_year=2017;citation_pmid=28314741;citation_doi=10.1083/jcb.201609123" />
<meta name="citation_reference" content="citation_journal_title=J. Cell Biol.;citation_author=M. K. Gardner;citation_author=J. Haase;citation_author=K. Mythreye;citation_author=J. N. Molk;citation_author=M. Anderson;citation_title=The microtubule-based motor Kar3 and plus end-binding protein Bim1 provide structural support for the anaphase spindle.;citation_volume=180;citation_year=2008;citation_pmid=18180364;citation_doi=10.1083/jcb.200710164" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=D. Gietz;citation_author=A. St Jean;citation_author=R. A. Woods;citation_author=R. H. Schiestl;citation_title=Improved method for high efficiency transformation of intact yeast cells;citation_pages=1425-1425;citation_volume=20;citation_year=1992;citation_issue=6;citation_pmid=1561104;citation_doi=10.1093/nar/20.6.1425" />
<meta name="citation_reference" content="citation_journal_title=J. Cell Biol.;citation_author=A. N. Gillis;citation_author=S. Thomas;citation_author=S. D. Hansen;citation_author=K. B. Kaplan;citation_title=A novel role for the CBF3 kinetochore-scaffold complex in regulating septin dynamics and cytokinesis.;citation_volume=171;citation_year=2005;citation_pmid=16330709;citation_doi=10.1083/jcb.200507017" />
<meta name="citation_reference" content="citation_journal_title=Yeast (Chichester, England);citation_journal_abbrev=Yeast;citation_author=A. L. Goldstein;citation_author=J. H. McCusker;citation_title=Three new dominant drug resistance cassettes for gene disruption in Saccharomyces cerevisiae.;citation_pages=1541-1553;citation_volume=15;citation_year=1999;citation_issue=14;citation_pmid=10514571;citation_doi=10.1002/(SICI)1097-0061(199910)15:14&lt;1541::AID-YEA476&gt;3.0.CO;2-K" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=D. Hanahan;citation_author=R. A. Weinberg;citation_title=Hallmarks of cancer: the next generation.;citation_pages=646-674;citation_volume=144;citation_year=2011;citation_issue=5;citation_pmid=21376230;citation_doi=10.1016/j.cell.2011.02.013" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=H. Haruki;citation_author=J. Nishikawa;citation_author=U. K. Laemmli;citation_title=The anchor-away technique: rapid, conditional establishment of yeast mutant phenotypes.;citation_pages=925-932;citation_volume=31;citation_year=2008;citation_issue=6;citation_pmid=18922474;citation_doi=10.1016/j.molcel.2008.07.020" />
<meta name="citation_reference" content="citation_journal_title=Yeast (Chichester, England);citation_journal_abbrev=Yeast;citation_author=P. Hentges;citation_author=B. Van Driessche;citation_author=L. Tafforeau;citation_author=J. Vandenhaute;citation_author=A. M. Carr;citation_title=Three novel antibiotic marker cassettes for gene disruption and marker switching in Schizosaccharomyces pombe.;citation_pages=1013-1019;citation_volume=22;citation_year=2005;citation_issue=13;citation_pmid=16200533;citation_doi=10.1002/yea.1291" />
<meta name="citation_reference" content="citation_journal_title=Cell Rep.;citation_author=S. M. Hinshaw;citation_author=S. C. Harrison;citation_title=An Iml3-Chl4 heterodimer links the core centromere to factors required for accurate chromosome segregation.;citation_pages=29-36;citation_volume=5;citation_year=2013;citation_pmid=24075991;citation_doi=10.1016/j.celrep.2013.08.036" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=M. A. Hoyt;citation_author=T. Stearns;citation_author=D. Botstein;citation_title=Chromosome instability mutants of Saccharomyces cerevisiae that are defective in microtubule-mediated processes.;citation_pages=223-234;citation_volume=10;citation_year=1990;citation_issue=1;citation_pmid=2403635;citation_doi=10.1128/MCB.10.1.223" />
<meta name="citation_reference" content="citation_journal_title=The Journal of Cell Biology;citation_journal_abbrev=JCB;citation_author=M. A. Hoyt;citation_author=L. He;citation_author=K. K. Loo;citation_author=W. S. Saunders;citation_title=Two Saccharomyces cerevisiae kinesin-related gene products required for mitotic spindle assembly;citation_pages=109-120;citation_volume=118;citation_year=1992;citation_issue=1;citation_pmid=1618897;citation_doi=10.1083/jcb.118.1.109" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=M. A. Hoyt;citation_author=J. P. Macke;citation_author=B. T. Roberts;citation_author=J. R. Geiser;citation_title=Saccharomyces cerevisiae PAC2 Functions With CIN1, 2 and 4 in a Pathway Leading to Normal Microtubule Stability;citation_pages=849-857;citation_volume=146;citation_year=1997;citation_issue=3;citation_pmid=9215891" />
<meta name="citation_reference" content="citation_journal_title=Current biology :  CB;citation_journal_abbrev=Curr Biol;citation_author=V. B. Indjeian;citation_author=A. W. Murray;citation_title=Budding yeast mitotic chromosomes have an intrinsic bias to biorient on the spindle.;citation_pages=1837-1846;citation_volume=17;citation_year=2007;citation_issue=21;citation_pmid=17980598;citation_doi=10.1016/j.cub.2007.09.056" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=A. A. Jeyaprakash;citation_author=U. R. Klein;citation_author=D. Lindner;citation_author=J. Ebert;citation_author=E. A. Nigg;citation_title=Structure of a Survivin-Borealin-INCENP core complex reveals how chromosomal passengers travel together.;citation_pages=271-285;citation_volume=131;citation_year=2007;citation_issue=2;citation_pmid=17956729;citation_doi=10.1016/j.cell.2007.07.045" />
<meta name="citation_reference" content="citation_journal_title=J. Cell Biol.;citation_author=J. Kang;citation_author=I. M. Cheeseman;citation_author=G. Kallstrom;citation_author=S. Velmurugan;citation_author=G. Barnes;citation_title=Functional cooperation of Dam1, Ipl1, and the inner centromere protein (INCENP)-related protein Sli15 during chromosome segregation.;citation_volume=155;citation_year=2001;citation_pmid=11724818;citation_doi=10.1083/jcb.200105029" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=S. A. Kawashima;citation_author=Y. Yamagishi;citation_author=T. Honda;citation_author=K. Ishiguro;citation_author=Y. Watanabe;citation_title=Phosphorylation of H2A by Bub1 prevents chromosomal instability through localizing shugoshin.;citation_volume=327;citation_year=2010;citation_pmid=19965387;citation_doi=10.1126/science.1180189" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=B. M. Kiburz;citation_author=D. B. Reynolds;citation_author=P. C. Megee;citation_author=A. L. Marston;citation_author=B. H. Lee;citation_title=The core centromere and Sgo1 establish a 50-kb cohesin-protected domain around centromeres during meiosis I;citation_pages=3017-3030;citation_volume=19;citation_year=2005;citation_issue=24;citation_pmid=16357219;citation_doi=10.1101/gad.1373005" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=J. L. Koh;citation_author=H. Ding;citation_author=M. Costanzo;citation_author=A. Baryshnikova;citation_author=K. Toufighi;citation_title=DRYGIN: a database of quantitative genetic interaction networks in yeast.;citation_volume=38;citation_year=2010;citation_pmid=19880385;citation_doi=10.1093/nar/gkp820" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Molecular cell biology;citation_journal_abbrev=Nat Rev Mol Cell Biol;citation_author=F. Lampert;citation_author=S. Westermann;citation_title=A blueprint for kinetochores - new insights into the molecular mechanics of cell division.;citation_pages=407-412;citation_volume=12;citation_year=2011;citation_issue=7;citation_pmid=21633384;citation_doi=10.1038/nrm3133" />
<meta name="citation_reference" content="citation_journal_title=Trends in cell biology;citation_journal_abbrev=Trends Cell Biol;citation_author=M. A. Lampson;citation_author=I. M. Cheeseman;citation_title=Sensing centromere tension: Aurora B and the regulation of kinetochore function.;citation_pages=133-140;citation_volume=21;citation_year=2011;citation_issue=3;citation_pmid=21106376;citation_doi=10.1016/j.tcb.2010.10.007" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=D. Liu;citation_author=G. Vader;citation_author=M. J. Vromans;citation_author=M. A. Lampson;citation_author=S. M. Lens;citation_title=Sensing chromosome bi-orientation by spatial separation of aurora B kinase from kinetochore substrates.;citation_volume=323;citation_year=2009;citation_pmid=19150808;citation_doi=10.1126/science.1167000" />
<meta name="citation_reference" content="citation_journal_title=Yeast (Chichester, England);citation_journal_abbrev=Yeast;citation_author=M. S. Longtine;citation_author=A. McKenzie;citation_author=D. J. Demarini;citation_author=N. G. Shah;citation_author=A. Wach;citation_title=Additional modules for versatile and economical PCR-based gene deletion and modification in Saccharomyces cerevisiae.;citation_pages=953-961;citation_volume=14;citation_year=1998;citation_issue=10;citation_pmid=9717241;citation_doi=10.1002/(SICI)1097-0061(199807)14:10&lt;953::AID-YEA293&gt;3.0.CO;2-U" />
<meta name="citation_reference" content="citation_journal_title=Mol. Cell. Biol.;citation_author=V. Makrantoni;citation_author=M. J. Stark;citation_title=Efficient chromosome biorientation and the tension checkpoint in Saccharomyces cerevisiae both require Bir1.;citation_volume=29;citation_year=2009;citation_pmid=19528231;citation_doi=10.1128/MCB.01911-08" />
<meta name="citation_reference" content="citation_journal_title=EMBO J.;citation_author=F. Malvezzi;citation_author=G. Litos;citation_author=A. Schleiffer;citation_author=A. Heuck;citation_author=K. Mechtler;citation_title=A structural basis for kinetochore recruitment of the Ndc80 complex via two distinct centromere receptors.;citation_volume=32;citation_year=2013;citation_pmid=23334295;citation_doi=10.1038/emboj.2012.356" />
<meta name="citation_reference" content="citation_journal_title=J. Cell Biol.;citation_author=B. D. Manning;citation_author=J. G. Barrett;citation_author=J. A. Wallace;citation_author=H. Granok;citation_author=M. Snyder;citation_title=Differential regulation of the Kar3p kinesin-related protein by two associated proteins, Cik1p and Vik1p.;citation_volume=144;citation_year=1999;citation_pmid=10087265;citation_doi=10.1083/jcb.144.6.1219" />
<meta name="citation_reference" content="citation_journal_title=Mol. Cell. Biol.;citation_author=A. L. Marston;citation_title=Shugoshins: tension-sensitive pericentromeric adaptors safeguarding chromosome segregation.;citation_volume=35;citation_year=2015;citation_pmid=25452306;citation_doi=10.1128/MCB.01176-14" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=A. L. Marston;citation_author=W. H. Tham;citation_author=H. Shah;citation_author=A. Amon;citation_title=A genome-wide screen identifies genes required for centromeric cohesion.;citation_volume=303;citation_year=2004;citation_pmid=14752166;citation_doi=10.1126/science.1094220" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=M. L. Mayer;citation_author=S. P. Gygi;citation_author=R. Aebersold;citation_author=P. Hieter;citation_title=Identification of RFC(Ctf18p, Ctf8p, Dcc1p): an alternative RFC complex required for sister chromatid cohesion in S. cerevisiae.;citation_pages=959-970;citation_volume=7;citation_year=2001;citation_issue=5;citation_pmid=11389843;citation_doi=10.1016/S1097-2765(01)00254-4" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Cell;citation_author=M. L. Mayer;citation_author=I. Pot;citation_author=M. Chang;citation_author=H. Xu;citation_author=V. Aneliunas;citation_title=Identification of protein complexes required for efficient sister chromatid cohesion.;citation_volume=15;citation_year=2004;citation_pmid=14742714;citation_doi=10.1091/mbc.E03-08-0619" />
<meta name="citation_reference" content="citation_journal_title=J. Biol. Chem.;citation_author=C. Mayr;citation_author=K. Richter;citation_author=H. Lilie;citation_author=J. Buchner;citation_title=Cpr6 and Cpr7, two closely related Hsp90-associated immunophilins from Saccharomyces cerevisiae, differ in their functional properties.;citation_volume=275;citation_year=2000;citation_pmid=10942767;citation_doi=10.1074/jbc.M005251200" />
<meta name="citation_reference" content="citation_journal_title=Molecular Biology of the Cell;citation_journal_abbrev=Mol. Biol. Cell;citation_author=R. K. Miller;citation_author=S. C. Cheng;citation_author=M. D. Rose;citation_title=Bim1p/Yeb1p Mediates the Kar9p-dependent Cortical Attachment of Cytoplasmic Microtubules;citation_pages=2949-2959;citation_volume=11;citation_year=2000;citation_issue=9;citation_pmid=10982392;citation_doi=10.1091/mbc.11.9.2949" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Cell;citation_author=Y. Nakajima;citation_author=R. G. Tyers;citation_author=C. C. Wong;citation_author=J. R. Yates;citation_author=D. G. Drubin;citation_title=Nbl1p: A Borealin/Dasra/CSC-1-like protein essential for Aurora/Ipl1 complex function and integrity in Saccharomyces cerevisiae.;citation_volume=20;citation_year=2009;citation_pmid=19158380;citation_doi=10.1091/mbc.E08-10-1011" />
<meta name="citation_reference" content="citation_journal_title=Annual review of genetics;citation_journal_abbrev=Annu Rev Genet;citation_author=K. Nasmyth;citation_author=C. H. Haering;citation_title=Cohesin: its roles and mechanisms.;citation_pages=525-558;citation_volume=43;citation_year=2009;citation_pmid=19886810;citation_doi=10.1146/annurev-genet-102108-134233" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Cell;citation_author=T. M. Ng;citation_author=W. G. Waples;citation_author=B. D. Lavoie;citation_author=S. Biggins;citation_title=Pericentromeric sister chromatid cohesion promotes kinetochore biorientation.;citation_volume=20;citation_year=2009;citation_pmid=19605555;citation_doi=10.1091/mbc.E09-04-0330" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=K. Ohkuni;citation_author=R. Abdulle;citation_author=A. H. Tong;citation_author=C. Boone;citation_author=K. Kitagawa;citation_title=Ybp2 associates with the central kinetochore of Saccharomyces cerevisiae and mediates proper mitotic progression.;citation_pages=e1617-e1617;citation_volume=3;citation_year=2008;citation_issue=2;citation_pmid=18286174;citation_doi=10.1371/journal.pone.0001617" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=J. Ortiz;citation_author=O. Stemmann;citation_author=S. Rank;citation_author=J. Lechner;citation_title=A putative protein complex consisting of Ctf19, Mcm21, and Okp1 represents a missing link in the budding yeast kinetochore;citation_pages=1140-1155;citation_volume=13;citation_year=1999;citation_issue=9;citation_pmid=10323865;citation_doi=10.1101/gad.13.9.1140" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Cell;citation_author=I. Pot;citation_author=V. Measday;citation_author=B. Snydsman;citation_author=G. Cagney;citation_author=S. Fields;citation_title=Chl4p and iml3p are two new members of the budding yeast outer kinetochore.;citation_volume=14;citation_year=2003;citation_pmid=12589047;citation_doi=10.1091/mbc.E02-08-0517" />
<meta name="citation_reference" content="citation_journal_title=The Journal of Cell Biology;citation_journal_abbrev=JCB;citation_author=D. M. Roof;citation_author=P. B. Meluh;citation_author=M. D. Rose;citation_title=Kinesin-related proteins required for assembly of the mitotic spindle;citation_pages=95-108;citation_volume=118;citation_year=1992;citation_issue=1;citation_pmid=1618910;citation_doi=10.1083/jcb.118.1.95" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=S. Ruchaud;citation_author=M. Carmena;citation_author=W. C. Earnshaw;citation_title=The chromosomal passenger complex: one for all and all for one.;citation_pages=230-231;citation_volume=131;citation_year=2007;citation_issue=2;citation_pmid=17956723;citation_doi=10.1016/j.cell.2007.10.002" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=S. L. Rutherford;citation_title=Between genotype and phenotype: protein chaperones and evolvability.;citation_pages=263-274;citation_volume=4;citation_year=2003;citation_issue=4;citation_pmid=12671657;citation_doi=10.1038/nrg1041" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=S. Sandall;citation_author=F. Severin;citation_author=I. X. McLeod;citation_author=J. R. Yates;citation_author=K. Oegema;citation_title=A Bir1-Sli15 complex connects centromeres to microtubules and is required to sense kinetochore tension.;citation_pages=1179-1191;citation_volume=127;citation_year=2006;citation_issue=6;citation_pmid=17174893;citation_doi=10.1016/j.cell.2006.09.049" />
<meta name="citation_reference" content="citation_journal_title=The EMBO Journal;citation_journal_abbrev=EMBO J.;citation_author=S. Santaguida;citation_author=A. Musacchio;citation_title=The life and miracles of kinetochores;citation_pages=2511-2531;citation_volume=28;citation_year=2009;citation_issue=17;citation_pmid=19629042;citation_doi=10.1038/emboj.2009.173" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=W. S. Saunders;citation_author=M. A. Hoyt;citation_title=Kinesin-related proteins required for structural integrity of the mitotic spindle.;citation_pages=451-458;citation_volume=70;citation_year=1992;citation_issue=3;citation_pmid=1643659;citation_doi=10.1016/0092-8674(92)90169-D" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Cell;citation_author=M. M. Shimogawa;citation_author=P. O. Widlund;citation_author=M. Riffle;citation_author=M. Ess;citation_author=T. N. Davis;citation_title=Bir1 is required for the tension checkpoint.;citation_volume=20;citation_year=2009;citation_pmid=19056681;citation_doi=10.1091/mbc.E08-07-0723" />
<meta name="citation_reference" content="citation_journal_title=Dev. Cell;citation_author=A. Suzuki;citation_author=B. L. Badger;citation_author=X. Wan;citation_author=J. G. DeLuca;citation_author=E. D. Salmon;citation_title=The architecture of CCAN proteins creates a structural integrity to resist spindle forces and achieve proper Intrakinetochore stretch.;citation_pages=717-730;citation_volume=30;citation_year=2014;citation_pmid=25268173;citation_doi=10.1016/j.devcel.2014.08.003" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=T. U. Tanaka;citation_author=N. Rachidi;citation_author=C. Janke;citation_author=G. Pereira;citation_author=M. Galova;citation_title=Evidence that the Ipl1-Sli15 (Aurora kinase-INCENP) complex promotes chromosome bi-orientation by altering kinetochore-spindle pole connections.;citation_pages=317-329;citation_volume=108;citation_year=2002;citation_issue=3;citation_pmid=11853667;citation_doi=10.1016/S0092-8674(02)00633-5" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Molecular cell biology;citation_journal_abbrev=Nat Rev Mol Cell Biol;citation_author=T. U. Tanaka;citation_author=M. J. Stark;citation_author=K. Tanaka;citation_title=Kinetochore capture and bi-orientation on the mitotic spindle.;citation_pages=929-942;citation_volume=6;citation_year=2005;citation_issue=12;citation_pmid=16341079;citation_doi=10.1038/nrm1764" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=B. J. Thomas;citation_author=R. Rothstein;citation_title=Elevated recombination rates in transcriptionally active DNA.;citation_pages=619-630;citation_volume=56;citation_year=1989;citation_issue=4;citation_pmid=2645056;citation_doi=10.1016/0092-8674(89)90584-9" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Cell;citation_author=S. Thomas;citation_author=K. B. Kaplan;citation_title=A Bir1p-Sli15p kinetochore passenger complex regulates septin organization during anaphase.;citation_volume=18;citation_year=2007;citation_pmid=17652458;citation_doi=10.1091/mbc.E07-03-0201" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=H. Tourriere;citation_author=G. Versini;citation_author=V. Cordon-Preciado;citation_author=C. Alabert;citation_author=P. Pasero;citation_title=Mrc1 and Tof1 promote replication fork progression and recovery independently of Rad53.;citation_pages=699-706;citation_volume=19;citation_year=2005;citation_issue=5;citation_pmid=16137625;citation_doi=10.1016/j.molcel.2005.07.028" />
<meta name="citation_reference" content="citation_journal_title=Elife;citation_author=K. F. Verzijlbergen;citation_author=O. O. Nerusheva;citation_author=D. Kelly;citation_author=A. Kerr;citation_author=D. Clift;citation_title=Shugoshin biases chromosomes for biorientation through condensin recruitment to the pericentromere.;citation_pages=e01374;citation_volume=3;citation_year=2014;citation_pmid=24497542;citation_doi=10.7554/eLife.01374" />
<meta name="citation_reference" content="citation_journal_title=BMC genetics [electronic resource];citation_journal_abbrev=BMC Genet;citation_author=S. Wee;citation_author=B. Hetfeld;citation_author=W. Dubiel;citation_author=D. A. Wolf;citation_title=Conservation of the COP9/signalosome in budding yeast.;citation_pages=15-15;citation_volume=3;citation_year=2002;citation_pmid=12186635;citation_doi=10.1186/1471-2156-3-15" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Cell;citation_author=P. O. Widlund;citation_author=J. S. Lyssand;citation_author=S. Anderson;citation_author=S. Niessen;citation_author=J. R. Yates;citation_title=Phosphorylation of the chromosomal passenger protein Bir1 is required for localization of Ndc10 to the spindle during anaphase and full spindle elongation.;citation_volume=17;citation_year=2006;citation_pmid=16381814;citation_doi=10.1091/mbc.E05-07-0640" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=E. A. Winzeler;citation_author=D. D. Shoemaker;citation_author=A. Astromoff;citation_author=H. Liang;citation_author=K. Anderson;citation_title=Functional Characterization of the S. cerevisiae Genome by Gene Deletion and Parallel Analysis;citation_pages=901-906;citation_volume=285;citation_year=1999;citation_issue=5429;citation_pmid=10436161;citation_doi=10.1126/science.285.5429.901" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=H. Xu;citation_author=C. Boone;citation_author=H. L. Klein;citation_title=Mrc1 Is Required for Sister Chromatid Cohesion To Aid in Recombination Repair of Spontaneous Damage;citation_pages=7082-7090;citation_volume=24;citation_year=2004;citation_issue=16;citation_pmid=15282308;citation_doi=10.1128/MCB.24.16.7082-7090.2004" />
<meta name="citation_reference" content="citation_journal_title=Current biology :  CB;citation_journal_abbrev=Curr Biol;citation_author=E. Yeh;citation_author=J. Haase;citation_author=L. V. Paliulis;citation_author=A. Joglekar;citation_author=L. Bond;citation_title=Pericentric chromatin is organized into an intramolecular loop in mitosis.;citation_pages=81-90;citation_volume=18;citation_year=2008;citation_issue=2;citation_pmid=18211850;citation_doi=10.1016/j.cub.2007.12.019" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=H. J. Yoon;citation_author=J. Carbon;citation_title=Participation of Bir1p, a member of the inhibitor of apoptosis family, in yeast chromosome segregation events;citation_pages=13208-13213;citation_volume=96;citation_year=1999;citation_issue=23;citation_pmid=10557299;citation_doi=10.1073/pnas.96.23.13208" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="A Functional Link Between Bir1 and the Saccharomyces cerevisiae Ctf19 Kinetochore Complex Revealed Through Quantitative Fitness Analysis" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9.cover-source.jpg" />
<meta name="twitter:description" content="The chromosomal passenger complex (CPC) is a key regulator of eukaryotic cell division, consisting of the protein kinase Aurora B/[Ipl1][1] in association with its activator (INCENP/[Sli15][2]) and two additional proteins (Survivin/[Bir1][3] and Borealin/[Nbl1][4]). Here, we report a genome-wide genetic interaction screen in Saccharomyces cerevisiae using the [bir1][3]-17 mutant, identifying through quantitative fitness analysis deletion mutations that act as enhancers and suppressors. Gene knockouts affecting the [Ctf19][5] kinetochore complex were identified as the strongest enhancers of [bir1][3]-17 , while mutations affecting the large ribosomal subunit or the mRNA nonsense-mediated decay pathway caused strong phenotypic suppression. Thus, cells lacking a functional [Ctf19][5] complex become highly dependent on [Bir1][3] function and vice versa. The negative genetic interaction profiles of [bir1][3]-17 and the cohesin mutant [mcd1][6]-1 showed considerable overlap, underlining the strong functional connection between sister chromatid cohesion and chromosome biorientation. Loss of some [Ctf19][5] components, such as [Iml3][7] or [Chl4][8], impacted differentially on [bir1][3]-17 compared with mutations affecting other CPC components: despite the synthetic lethality shown by either [iml3][7] ∆ or [chl4][8] ∆ in combination with [bir1][3]-17 , neither gene knockout showed any genetic interaction with either [ipl1][1]-321 or [sli15][2]-3 . Our data therefore imply a specific functional connection between the [Ctf19][5] complex and [Bir1][3] that is not shared with [Ipl1][1].

 [1]: http://www.yeastgenome.org/locus/S000006130/overview
 [2]: http://www.yeastgenome.org/locus/S000000360/overview
 [3]: http://www.yeastgenome.org/locus/S000003849/overview
 [4]: http://www.yeastgenome.org/locus/S000029704/overview
 [5]: http://www.yeastgenome.org/locus/S000005939/overview
 [6]: http://www.yeastgenome.org/locus/S000002161/overview
 [7]: http://www.yeastgenome.org/locus/S000000311/overview
 [8]: http://www.yeastgenome.org/locus/S000002662/overview" />
<meta name="og-title" property="og:title" content="A Functional Link Between Bir1 and the Saccharomyces cerevisiae Ctf19 Kinetochore Complex Revealed Through Quantitative Fitness Analysis" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/7/9/3203" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="The chromosomal passenger complex (CPC) is a key regulator of eukaryotic cell division, consisting of the protein kinase Aurora B/[Ipl1][1] in association with its activator (INCENP/[Sli15][2]) and two additional proteins (Survivin/[Bir1][3] and Borealin/[Nbl1][4]). Here, we report a genome-wide genetic interaction screen in Saccharomyces cerevisiae using the [bir1][3]-17 mutant, identifying through quantitative fitness analysis deletion mutations that act as enhancers and suppressors. Gene knockouts affecting the [Ctf19][5] kinetochore complex were identified as the strongest enhancers of [bir1][3]-17 , while mutations affecting the large ribosomal subunit or the mRNA nonsense-mediated decay pathway caused strong phenotypic suppression. Thus, cells lacking a functional [Ctf19][5] complex become highly dependent on [Bir1][3] function and vice versa. The negative genetic interaction profiles of [bir1][3]-17 and the cohesin mutant [mcd1][6]-1 showed considerable overlap, underlining the strong functional connection between sister chromatid cohesion and chromosome biorientation. Loss of some [Ctf19][5] components, such as [Iml3][7] or [Chl4][8], impacted differentially on [bir1][3]-17 compared with mutations affecting other CPC components: despite the synthetic lethality shown by either [iml3][7] ∆ or [chl4][8] ∆ in combination with [bir1][3]-17 , neither gene knockout showed any genetic interaction with either [ipl1][1]-321 or [sli15][2]-3 . Our data therefore imply a specific functional connection between the [Ctf19][5] complex and [Bir1][3] that is not shared with [Ipl1][1].

 [1]: http://www.yeastgenome.org/locus/S000006130/overview
 [2]: http://www.yeastgenome.org/locus/S000000360/overview
 [3]: http://www.yeastgenome.org/locus/S000003849/overview
 [4]: http://www.yeastgenome.org/locus/S000029704/overview
 [5]: http://www.yeastgenome.org/locus/S000005939/overview
 [6]: http://www.yeastgenome.org/locus/S000002161/overview
 [7]: http://www.yeastgenome.org/locus/S000000311/overview
 [8]: http://www.yeastgenome.org/locus/S000002662/overview" />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9.cover-source.jpg" />
    <title>A Functional Link Between Bir1 and the Saccharomyces cerevisiae Ctf19 Kinetochore Complex Revealed Through Quantitative Fitness Analysis | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"GIMMeHS8wpjpkKWovSx9JS5foQB_t6i0bXDutbcPrGc","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"486361","apath":"\/ggg\/7\/9\/3203.atom","pisa":"ggg;7\/9\/3203","ac":{"\/ggg\/7\/9\/3203.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/7\/9\/3203.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;7\/9\/3203"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;7\/9\/3203"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/7\/9\/3203.supplemental":true,"\/content\/7\/9\/3203":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022j9uLPMWm2ftNOzZrB08O1iFKDtq8heHFVErhMYOU3UUb9dp6vk01N7oF\\\\\\\/+nkBh8DzHPC9ysmWB80de8VAXnmpD2DXrf2v3O7SwIZGQ6el20SSw4NgmvL17zACYhnggeKebvNfroocXo1P7EK+tlhvQ3HlGtCuV8Q7mvGszzbV6oDlwN7mN4Eu1zs195ax42tCM6S3wnSIao61HQoGEU\\\\\\\/tV1UlcUSFJijuHFXA8Z8OvBrKnUBvzkCnue0CXLOsWd4ieZ55iaIx0l6eSjQn+8eAK7lN0ZnnUg5LhUIqgz2vbceDdbmIuTmCdduNxiTaO5Xu8C1vWFWGPGo5jaLmaFxPqL4D1\\\\\\\/tBH5pawxuxCCPG8EhKrpABHpmRFZpT688SXYEccevyY1p1vdH83G1vkb3n44EEtVUAUX\\\\\\\/XusHn1dulCa117bHSi2jQkMC35v0NVtlfDWspxmN2L9cWx0KPPg5KSk6RZ1h1+k6czciESidIRbiKlgLmrorWcCE69VoJ0dJhRX917Pju0cJlTIDtMcmWa73L\\\\\\\/LLp9eEIdTtf6WbemLKHsqoeHxcLljnaC0Sjee1HbOAVTB2vTyWR0ZOxqJ80I6TqDWUTg8ATKrBxcDekLhYAOx\\\\\\\/NSv40JK\\\\\\\/cr\\\\\\\/7+m\\\\\\\/C2UzYV3j1kciOD7EpIFLdWhM3VOV1ZMDVdfpCtnUFgtcu38SpXs91rVRBI2kyXL9ju8vecfxpj2f5jg3xnp4nlV0QlLdJm\\\\\\\/AdluBeqHdkrgB5X6Eolk\\\\\\\/1Dwh2cJswNqZvthzZMulqLRpoeBEG3egtuWCUafY+CMQ8n7xnVkL45yQeSGIDVyDzDFXAbog6u7aqe8AO6BfqqHLlH21PNi90NjDyNAtb7ODm5Xs\\\\\\\/UfYzQYwKtrev8e2Mi2AChWOIcJhnYejPSTscK376FXbJJzGOOcSApgfvLr0D\\\\\\\/1rbOKqpeW2WlyB70zL9LFtk4H\\\\\\\/AY1CoqjROslmOrbiuLhxyMdn3OfbwtnRy0WtF8pxZASKar4RaLPitkt1FdoF3ChpQjyzPY46CERzAw\\\\\\\/gj1a73DwtUTOVu1pWk3q4XjhaeesxMlyWMK7yJyEKGhR5U4BkVwJiyI7X2dx+QjQlzz\\\\\\\/4ts+szThtuJb5xEQn+xB7DWC1g1\\\\\\\/oIiNgEAsOgbf\\\\\\\/xL7eynsuOtpkrAMHt5vcXKaY18zhhJuSHecddlZxM1OaPcuFdUvEmcYUC446sRsoriS16GUbU2KmF4t\\\\\\\/g\\\\\\\/5cdbF3y813yJPBneGGFXThvwoEq8IsqXyi+zs0efKKwJ0DaCKWpyPfr\\\u0022,\\\u0022iv\\\u0022:\\\u0022utbX7JiX78GOdbaYugeU1w==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022c0b1e37e8984104d7a48091f774748e8773de8730540950996864040278fc02b\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022Q5947qe2MWy1dDslSyKdAoip6Qmsk8jfpoyPWRbiNHHLopCk+XyCEmlVqNh23vaZavdiPb4xhZmy4JUrF777wgzZVSutrnn2ny9d9SCjTwEGbNfsOHFjHLZ\\\\\\\/5kv6W3LHOdVy5w2xqMpRpC3Cql9whAofDZ+y8n+LjbSFogfpF0ciEH0l5Ij5j47eUov7OBwbUKLocWI9rBkvOQePU25r05l5m6OS4rBtUF9OzKV+OnWMqT5viPEs3hLvELBP\\\\\\\/53zuZzd1JBjSlPSQFUPWlZDt2nWel59ZF8XzY05obCM\\\\\\\/SHvLeOAQfdDPPV6b6tnNpZaTr65dTN4l7gIjRfJWnp19ItW3t\\\\\\\/Ys\\\\\\\/O4o4JMwLyCpGooQSsWQ8wIbW9tkUQCTPGC6oWu8+Xx0O9egLdYyUxMrXyf8TTpuJHOXYx+DgqISw9sF+GIQmCo4vwf60RDMbQHUcqMbUjxU3dae5I70ASNOTCDNppN3+LwsnsRasGDI3iLlcYzJAMPZI32rFPXq0WNxpqYdDZNXk5sLOGIDf8CRGvKfN13gVZG6y1gNmdHZbvCr0WLOpbAmBoGGkKdScQmKH\\\\\\\/YDLG6qWJXHARuLgJ5QfAnGmSt5488mRdPEcx5riE\\\\\\\/p04r\\\\\\\/crXuoGXFOEKAIh\\\\\\\/tQLBga4XFUN0elTZKmVwqsGzitFM4hT4Pd+cF0ILcnv\\\\\\\/EOIACNf2N9TBH4gfOt1QjeVctxhJxaB7LvlqisjxtAqgezuA9XdxJh6sH8GV3sFwSW3SSsmVJ1lkppJmeAR1bdQLNSQ80dOWZI0wCUrh5LFmo6XAihLbwboy0IVrsl28Eg4B\\\\\\\/lhE1SfNmblLwAspqbtfZs04QzIZg+tqCy+YwDEEHChCvpb8jrUTVUrodgzeN+raYuMbRFbozzOtZl3lTZOSb1yWLApPo7beKNfIKuXqylwE55mZqEbltGsGHwH8R0dXrf7dqPVFrakMzv8krI7xeB4q0+RvvLLFIaq0ItJbcpgjb4wjYutao+CglhTmmhpQ6\\\\\\\/TqFdXio5O+3fkxGDYmIRbw\\\\\\\/nADQ7sBFiH7bHPgaUd\\\\\\\/nITLxhj+uyqRH+jUKepQEleLRTUt8dsANOjEqF2\\\\\\\/bnW17kesLt2bhuo+fAEVJL+vkzrtVrtcR6bE2pWv0+Xcgf8dXwlb79xJpXooolo\\\\\\\/eUtw68cTyeSvTrNpKFtYZs6f3M\\\\\\\/9spuC25OESmruvLSVbzai8JECSRtp3bwV6xfPfFZkFBD25wbWxfcfznIEKjbaEHIo0tbGrRstScjDWeQ97L6wOd\\\\\\\/xNhJ91WdyPZJ9rVE8kCy2zyP2pPzPiPG+sUqIKHMB3s8d8KDXwXqWQN7KmzKDh+0+YUI5w8q2RJmyrjSs9C\\\\\\\/s9P\\\\\\\/eYuEOJ906qF8T2W8Q\\\\\\\/64ZH9bmmnCEBNsHIsYFTaFoIUbDW5PrQb5+W9vxpnsnIga6S5oq8xr9AV+te7DWJuZ6pCc\\\\\\\/WPRbPnQl3KhBphXQUAQgUOLDQ0GwaRX5NpMl1zy14J6WgyZOOYdpL9EdoF+ff60yxPogru3v2dGt0NZ\\\\\\\/ER6urG0rbklzfKfZ8NBTGvM7m2byfRKRyW7U9YUXGJQYeijnLQuE5lDzcVtMDKaR50kC+IGrdbeG5lYhv14mBUn\\\\\\\/uIFhNNUM+1IR94f9ZnZfV9R+z9MUQVI6dKAo\\\\\\\/ViQJKopJ6XJxELdcaR9h8T5w+QyzfNGUBu20PuLOK9wiCQyG\\\\\\\/3aB+B+3Dwd0aeHfTpKP6PpT7mtu7zCUuj\\\\\\\/bEVr6lMnRRKpqDt8JN1Qec9s47zCIDY3NN79D5aNZ\\\\\\\/bIOmLqYIeSUuv7dhAudnAZLM3tHeUC2vDqZq3CoKxrJK31xVT0i25DpnbQsP2bYNoOrCthEbvWP2jrJgZUXDfNcO8Cvr\\\\\\\/Msc5Oui1JzzJzJCAFrggthpzyBaNVmaFBuPCpsktXkfLs4drq2UxJ248XIjVD8dLGxy\\\\\\\/MBJ3cXNXTAHhSPnA0UQRMWCejbBSXo2ttWwa3q14y2ZttEknihn+k5IAM6PiCzG807ytvQXTzpGPeuANEZG+jlGCRQqUYYZy4XPRn+nvvDZHGBZTDKfkwxOfK2oI2pr+vxuPALt50f3BWifjbUn0OyteBQ73jv+Y8dCH9Ot0W\\\\\\\/gZ05aRY+T\\\\\\\/dbier2OTTCzv\\\\\\\/uZq3OFSQ44AXbDNQQylq9SzeuzIvWgs7hIA+2sY4RSgdeccJHPVdXlYhgw2l9Ef2V2Q4Kq21t+KckjXam9c+FPIuL4\\\\\\\/fw25oXHiF9XPhRkJG\\\\\\\/2cQ0sP2ftJG83ZKmb2ncn7U18GAgiuXnnJXA37OYrVA7Us4Fypj5tIU4AYC+Yh\\\\\\\/N+ZR\\\\\\\/4AoYiuWdN1ld8qdwlLwU+QYCY1KwT04cbNOsMX6UKjVYUZvU2SkpsSWftw8NPFvc44lcIbXIG07YGCgFpZyefneYcobvUcQIZDwRyQ98RMnwxxBT1C14z7iDQ0YGFAZeno19OYp1HxxT0t6Y92nudR9eDPWFe\\\\\\\/yahV5fpkbtyqvGkOx9SME22dyqczP7ahV1eZnGdKXFrKzYx\\\\\\\/TUWltCK\\\\\\\/PRrUzjHP4yLyeeUUYoRVvTZjWG7GhD0lm02jMBn5TBILAqet1eukBGZrLwj7+OUepxg4lksAlDLW4mqa73Zo3h5SB43ioU6lRBUd6E6x\\\\\\\/Qc6R9AQiYD7thRtFv8oSWyHmsW7K4XkN7Ps4OrExI+0X8nIJAj\\\\\\\/JRaxk5bYFrqH+BOVAAb1vpM5gzSvj1bwa3ccQf\\\\\\\/Ui35THdw5T\\\\\\\/kC+QkQuGWlgcs\\\\\\\/3y3icz1p3I6hqQDqfHEFqS39BL6IU\\\\\\\/jCgJOrdQ9jPYHqxoyL\\\\\\\/9FYJ8msiZYDFNvqJL+7wDoG3wBcI\\\\\\\/mD6cZSGAsf6mEWwci6k8mJof72NW864y9KWUG613Kka4CDoEjKInHukpq6wl8QF2O6+O8dJixgEZH5oXFvhZCS34plehEWqUcYs\\\\\\\/W6f5pPhVkgof8eXWZ5N4ZNCI78qdZp1d7KfB15cFEaXTA\\\\\\\/FNQxuyycGLbM4CxJcUdBLK9efkVrUhIZvJ1L3tErtoCy256rVFypEheTQvKFMv78O3muCBx952sCRbqhzeCV7eZSzhQl8H4I5aJxmSFz7ELEUtufS1c9MzN0qPkrwEBj1IJdklXazcjyHsGJkD69kqZTq5SaKDdFYcspaxRDerslyXAxTq3yfP8cPFOVcdEGeCWB9SD5SttNPCJEHaEldsc0c1YO23IizjV5tSfvTiKNavgGsesEAAEoI4+kp5oqQl2NCldJw6P40IOnczjIvy78NP68XpsllDCmyM4qddReyxvy3h4cKmnioJDzkhk6zJUK6uFzkoO7kpr4Z2fk2B+m7sBNy1CTqwP1VOM0q6KSNwCY6n78oWZTMsCDgC9CxetdlZ6siJqhn5uEY3+m6gp6H9TbtBFz2aYtYWiTE\\\\\\\/rQxMGII8Ln6XRGdal0FSpOBhg05xiRLY5baiw1pI1+CuvxgaNlMD8jZ7jj4Do09o0ARBq5Q2UsxH6S6hEPCzdb6zhWpZHiT9QA4dD1ByCm8iMe6cy+YyxEvnSkYJGy9z9pjiHKF+hvldZ9Gaa+GaBHLAsCM+67EOlc9C9JQ6pN9iNCRcibAYkNvuZMo6ZzKogPiRMTH9lURJwFkIpTnllJYgA70OW6dWwkF6KEq0gIVlBk0YWqm9c34j0kiz5kH4Tlud5GVoFx99iXl91NlXMzy3kHCop84U+VRgylJ1Je66\\\\\\\/0W7jKZCF6mPh5Zj0jqrDrfjKI8UpEo\\\\\\\/8aA3nijTZsE8zhpX3WKuwFQrDNKSkUxS\\\\\\\/i6hEGV6aUSZRD2bF0m9sJxY6N12cvcleonKr2D\\\\\\\/f+Dml4jlk906EusX87quIwz2BH45HtPrqJfpXQVoEICoLHeGga7PD0MqUniNszkgWl0k4bd8LpZVjKS5Z2eEeT3T\\\\\\\/BfdL1r5bjTgihVHrdFRucPDswwYOVxsGzdcWZDs8qZTueWrmHbpwLB7jGzp6luW7rKyInYwe\\\\\\\/pjVCNQFLct16vI+JNzCj\\\\\\\/IMn+b2RFAVhz82ztqnVnjsaNo5srfuawEwTrcJXKywPrqUopRc8G8k9OYKwNI2Ut6ZVpkrEZBo4wscMhaYavQZ0frkU2SKKWQK0DCZHrD\\\\\\\/7S0SprDR4Wm4tGYyxJnrCKjUqx7tL3QizuvvFpANfAVdAqDkcjUZEzfgcVPSMjyOic7Dg7QB0++xLPAFaBIFQcycAAu9kvdBQMnu2ZwsUlu9lPMPrkOtmgA8nFWu8Gx92s2AP9Nl7gm86t8X5ViI4BkjHrNzNKJ6gZHH6qloEmNDywMeZPYk7xIzMj57PFHxJu3G46VmxDhX9f0KfgbBzTJjtzJTnyAx5p4R7RR22DRFCajXxmHva3apA0\\\\\\\/CM7PVBqHRWYFfugE2vuePjFStZ60TUpEMpHNJ9l2dJ6uH6hZao9JcehQcoC9uRoHJ9dpWDNukKHfvKasCjVvXvsMB7PjhK6vdYuRQY3v+MDjbEd1UT44aF+znislpvv0Vk\\\\\\\/BMkWUplpNf47w30XvLXQsMZx0PqtJoSRvwLPm6oOQSAjnxTIKXFOAI\\\\\\\/xUKWfGFU3sIVwNOElxZxxw9ZZWrmG8Rh\\\\\\\/f4C9\\\\\\\/t6KoiLnpprRVXmYJIOdcT80qAGeSYGKyJXW8YHoYgTOdN+gorayA+JYBnTtGpwddHk0XnOnfG+DT\\\\\\\/CFZwAVrkuKEnkF0gfHqXl1uu1O+Hp97mEmlrWsGHdv67yFIetpcKxyw1uQkqL3G64B4VkPN\\\\\\\/Wtp9XAMsJkF807\\\\\\\/++6QH25oTuani0SyPCmJeL8HFizz4ixiZwgR4+f8F\\\\\\\/5x8olis0VJWVVmItEOOd+ZL22cb3mJ+ElJ8y5u2VAwI4hEnvZZ\\\\\\\/sTR397cOaxCTkaFLhgrXz98IxAYtQzr\\\\\\\/gMyiv9QqFurl+dS6G7WZy68r13qZMA0XyZFRTzDO7uqbxJapPWeB8sPaFnDvw82WHKG+ncxm0XklitiA13RprqgnYif9mPMWG+SaoI361kqWtoAw0Ihkni2GOIyIVgHH5p+oFtSUv2U8rXhKJjtE4SyE1kK2sg86xlb04z3ycXQqbZEOAlBdK+urprzfuhM+SxBB0rCCs0gVnoDdqCWAxuPvZ3E4Ye58wwuCcH1qEr5mQ4gia6Pv7tiZ678aXKloIB6+8v8CVYcdC9XAvHUixahrUkZ92QIpKMdmu5VFQ+hYiBYnTkxByLfzUFgZHjYoBl9xWRlKtzmdv72nZhMhV2FQ1xP4JuVUpgTpTs6aZTIcsrTR6K+qW+QNKG2iBnQC6JiqNF4mf4266PVEQcfHXIV5fvMOL\\\\\\\/zwuPrqIxsWRPwPImNC1o\\\\\\\/mkC+XLuClkybMueIExP01Hj+MclT78FfgOsNlLGynzOmpM3\\\\\\\/I6GTjcqPiwNH+JLph0OwITogmwcd62lTuY0EOBQIJfBm8lY6yjVdpImdOfPi4Km\\\\\\\/hE3GXCivUq1jHAeAeAO7VQ+pGClU1mHzndLtjlsYVxY10Mds+1RWnsL1yDfgfMi1ScQZm6U5VqDqyTVFZhtKuvYtbOr2qS4TXeLBUYWl3ulD6+BzFAmeE9uVNoZOCDtOiip2HA1Pxm\\\\\\\/K7RHXoham2EKd5EDRSCWeKH5b9zi5PQurHrQOyMAy8vgNQVqjykUCA6w1j9wKCgE56j08EYS9t1v\\\\\\\/anQptPd5P\\\\\\\/m6X7tLOFRn3kFf3ztaxb9JTQOPH8qS9azzry\\\\\\\/UP9+poUpoeE9RF5Xsv3aAy+FzI3sCZrW6F2EtzvEYXGk5qAp3wDt3kYbpy6mb98I4GemlVanlGCVI+SaaHSTfM4FiMbUM3QTZiH7v41w+q2\\\\\\\/w1uQNa0XfFCd36yW\\\\\\\/RS87L3qAPHX+qmppddrRFYz8CkvSsUveITJ2PxjhxQICWYFwWBsXL3mA16nfMEJQdsDRYx\\\\\\\/Q98M\\\\\\\/DHnjyX5c8SuJapygVjgd4PD+BVL7ahLJrUjUNklrBLgztejKGojrL0+lxqcfYG29wNl\\\\\\\/H8f4NW4sDeYc8mT7Qm5vHJZFZJaStPjzEYcuzYMFYSHoN\\\\\\\/PioDPmDcNGIjrQWQsjECf9EfgbbrAyTcuWGWtuVXvhnBPH2pEJOo+taeWKHOFIh98w2MRZM6OiekUqENSuyht+AfE4Ec4vqh7JxVvtA+29cHBmRw1ttezASx8giGO90NHt6xhh1SJ\\\\\\\/zV0O43Ww3XkADh7QwjLlmHH6tiOPDpBLL6PHIBU7OLFTZb9ezQHy6\\\\\\\/DLDc\\\\\\\/DkCvxoU0bW5DtVq7bLmeE9\\\\\\\/14xO9G7hEAUJh1LifC7O7wZHtFr723fNRpHOgB1Urmxkx6ERtEyU5B6vnl83MrGUFndGVeUvIphst1eeKLbIZxCKs4oCayGNZ5OhJs+9T7SmXjnQt3NV7WLj+mezbtJQGAWHdWum6\\\\\\\/1bY+m2W4otNOI3uLMh6xjst5EvXknH4\\\\\\\/xA7iXcJjxGFKp9bkl9Volx8wUgzEDwBmNc2q9AfunaKvtwve282u59o9kB3WkH1FYPSxnFje5V4HdGWsxASfNy+Dp2LgFfBhsJ1iDoSHJGsGgonxDonsGfGUH+MKZLl0qcG6lPvXUOYMtsafX4H1p\\\\\\\/poZKyhOj\\\\\\\/0VQTGKNUAETQOF83mqTdryUcdKcloPKUEdyav0cLV04NlEXUBynHIzR0Q7L05EWc5Jp4HXFgaYrs6NETHKpeF8YLWuV5\\\\\\\/9+rnd7+5hk8SYSlZlJsAn9W9bwrDimQuro\\\\\\\/hRyAYlD2rX7MUm2aURglltB946Tz6c8i+FOLV3SkxT3Un41dJwgCcDz+OD57qPnr\\\\\\\/vFXsXslyxeFQjdh8oyIj90ECI85EQddHemBXy3WuKtfF+VdMIvxNbHLMEKCvYBNLKSADenRr\\\\\\\/N7Nj2P8I4JxfgPvA2Dr78hWc17v1jVSj3EedxD+Gvpk8WtlY\\\\\\\/XFQktZV63aefzqhMQCbKfhvuP69vnMf5PyjO5vWDSKmUjacFGCig2esUbNS+N5xUdEwEhhscISlre7935dYOmgoUhmS6HeKuZCBp7FP6mWp7xAtNgSrEgsf+ECIkIn1bG0eXJrhHRf0yE9TAezgshPq69gr+bIJXaJt1nu3KQXA3lBWUX5Bmci9vE9h6uxqwtjQsbYR0ioSruiBbXbksG3TJ7x4fm93bPvfdSeljYeiIgXLOoyWcrrOU8bD3sbtaSWqVDt39N+3w\\\\\\\/UuhEpe7WSd1GnKLPI6ezZHdLJwGTTwUxy3SsxykJq\\\\\\\/fxl5WH5lmE+CH81kMbv5BkMwBqNlUeumyFq7OS7i\\\\\\\/TkM6f7UR6PMpL98zxh3OWDoJnIqBmAkw0gcd4SKB+of+rKbx+SjudMF8vF4C\\\\\\\/IVZDORsMcpJDNLktKkzsjCOUrp0RUD9Fl8eRrNwI9mcZRJzKT3mPY+8Vj\\\\\\\/ECcRJSCBVOJT\\\\\\\/RngtxmiRj0TZWhmavzZmyFjkznFBfi55XBpTpqDJsdK5xUQs6zJc0NxkjUmu6LK+3LL51flWzyVMqPo1bH0IsoUelaLqyV5rCeq5Zl2g3u7HwQiRsuBljZEQHOJSamOiGe+FTfy4g1oFC3bThe07YVthwlWb6uaGtDjUDSHafoUbsSS\\\\\\\/flU699N3SfKhCcOBT3lvic2VyoCQZp0ErwpSdfDWR2kT8WEZgvzN6qvZctiv+jIyEJsL8W8sA8C3DZe1AHo4BLJDRomF\\\\\\\/UsfKzJkmC7ANdNb7gdLEb0QSab\\\\\\\/oXBm9yTD9HQrZk5nu4xEbG1CYeGBdtE1Gqj\\\\\\\/BOi\\\\\\\/\\\\\\\/ZhMNnD6LXJFVj2E7KJjRY0oMz9HOTmSp0GgT5C76hQGRn8N5wH4t4shQcvR+L1\\\\\\\/+V0qgqlN\\\\\\\/rRGSHTzY2eCsyzaNMvyClFttFuwlrU3i+laV4KNd3R5h3J83bv4iKNu21bGLcaBuCdSLoiXEa\\\\\\\/ewbUCc7mlyIDTPor+UBSq78zvc4FZTinf4Ya3mnwwNfc3lx0Tho6f2ITdmjI84sRLHCJ4RJ2sjy6uVBs8osN\\\\\\\/FW0DhX3EEEaD7+nW96vAyeqFBE5lG1VuPJpYZ6jCoTnciq1rx7aw9L5vT9dFEeY3Fny4BnWhgNW\\\\\\\/OzCCJXs\\\\\\\/DtZX0HmfGvNkqQbiJ\\\\\\\/kXiP9dks202EE7UQk9JXnne4bXs9cZiIsTqqnOr2NAXm34S4B3D02mZQGv1vexsgOa\\\\\\\/vXwJpK8FC+rZSbyI\\\\\\\/TMvIsRn2XEFEQ0ohybXTmEdhULlw70UkvhAI6m7Y1uAeEAc4axoxQn0HFit7QP2Vhybrj6w5FvWTjvETwlMg1NnPzmUQynHEoFJLONPGDCqbzrUMVwH7vW7P1H\\\\\\\/gwwZIhpKaATpMFOQWaZrwOzo98ki+zt+XnLo591h9oFHwwnGfgNosTkIjNyIi4JJEKhEP5yGwxfhMFpQF+X5WLYrWmzmt1mugrAiQcq4p8NxvU5ZQyxL6gKWfvW4+XpOJ7+kSGiBAEl7J5N+Dh1d61CghW40Ur1U2m04hgL9dqeOrvfeIeU5\\\\\\\/GBpAaf52ZStC6z5lQJnxloAF0nZCL3S2FGhm8XXKFYDlfLVoHqHWDJeEJ+L1tOYNwXEoil+ch+3jKw6737yxb3JndmCg1klijDMOX1qx5fQpi7XSN5KieZqA7Fgyz2wA\\\\\\\/4GN9+mpYb\\\\\\\/VLuo3KaDClqINZQVaMXJNpTW7Pl2PxQLwY0Q0dAIYZh5HjmRQT9iOnWpbMsvAffJizT7BduLPzU7CDUl1qsvKVPxOVTmVYoL1KHvlKhUhnzP9YIEqfGWRZT3GfpDAj+zw8+BU3tXg2LhcvPxjFEEMIoycoC+LK9GPXLdu47q7godzUefVZctsrS2Ku2XUjtMQm69S+hi1NyaZzPDSaAx6M1cCqvdnBUh9tYLQVh9mhxWaVUZBK7NjYOEUeN7YwAk3zBI0RSzX78l5tfHwUCYTKd5OB2VE+LVVnti81x6xRtI5NHkLkadBQvIzTbxlLxL9HLBuIDV1jqpw+UIUsjfCqOtzKtBoeeHgxL+M402C2gJ\\\\\\\/zaahAJLhrM3LF\\\\\\\/1VbzoXEsYnZ582kTuTLXWBngXTnjt27yUGdLM3KNQmeIOwVQ6SWurAfO9ZTKMHQi9o8DhnuLrB7zEFCMDKPG3cCp10CZv5QfZdDmjwwh2v0nstynX3GyqLB\\\\\\\/GjZTJfqdAionyA3xqLnKDoKwI13Gh9efygteBNxCwYlb\\\\\\\/p4TsXgKOjrioTXscyes+tM\\\\\\\/oaX1Ssv4jZBlCML6N8V7Fg6A5tukIJBTLrDZ266kAfZUOJyC2HPqdbewAXnfKi\\\\\\\/aT8xgHupsq8LlpOL3nEncBMOiiPX1vw++TpOZBJ5oUo2gXQ3gbyakwtm21j8UyNTnIP9wzh3OcwK78vZkXzWp8mXJhtfYZ8V7sa7FsV7Nf2o\\\\\\\/vIs8JzQsCDKMp0nRJNPOxK+1qjQxyrP6Un891EqYrlsHm8LUR3a0wUiz3T5PXSA9df9vPsyypvkDbHx9Nr\\\\\\\/zScIrzU3Bt9i5F\\\\\\\/o1GvOgsKqsMjBUx7qLFqT0YDHytjcVNlqLTrV\\\\\\\/QdVU2mXTHTq9a5YPNLvj3YAuPq03\\\\\\\/n6skkmJ6Negt6stLSPqy6noLmu4lwWw7PdDnWo6wGetJXEhuLDLpCbhkpZsEXnKKf+Dd7uN3bnfy4H4PdyQLyO\\\\\\\/QT9buXUFwIvuScUFcnFsizi719nWYPhwOUDmpqTTLbGq2D4oz434zLtk\\\\\\\/3J8FZxPY6f7Q4ESFZ48ePZyGJsHqXZoEqh5V1kx+5vQmtJ7Lq73WaqguX6AZMLOmgQ8ri2mxrW+pGZiLjU2k3u7rSiw2SdmUx27JUBp8exyZwjG\\\\\\\/00FMvtA2edDjyignzlPkCxHzPSdNNOPrHkTwirAYDXAqw90YMStVsYnVqxMT4QKpbge+at+A0vF4HK9lYm+FkZY9GQlqMoTtAcwkU2XN2z00kJzBoJYg7bu+R\\\\\\\/yU4Royc4jnjBvihQObGyX4xGo+wRxuHj4IKwuw+ONutpwJ7q16D\\\\\\\/tQvhmBwVs9stx0P+NBA3VX876fHj2b4RzWYW79bi+iZwaBMU3N364VC+QIfEUr8zBVx\\\\\\\/oBpwNqPvbVJZe3DjfM0dPNdKoREf1zhhdMnsbKYh8ZQav7nhfE410XBPFBqkdniBMYXEk3zIYFGWjIwHZ3eTtCYCWVIytyTS2cLeB0MZPmc1wR4saHILYBslDNGoeVoYeHazVyPyZB87D2q3IW7poEiBlxyTleazRZNVQZ+MdsaoZjrm2cb9SnPKNdaviYyin7TDN2XhzNG\\\\\\\/CYWeJy+to3+fezqR5xUbM7rlIQyostAqlzCyMqJkqHupDFqOiaYxZnbJxtSKox2axZ4Q8jiq8QS5SRU7TjnaqgY2ZBgZ+2k89xGzt7yCTILcD9psTIabWpm0knPOhwph0aslnsSgT7CGCIx5yGv6o2EwJ58lSU\\\\\\\/3670l3JEiOBk6DmJqphIHKBdhyoV43wHxLVqmLFOxqJykMoRBfZSeOuaSi1T9lKrE4y5aVu2lcH4woyzWJu9tsVQxWXDN1VxJ17+iDuek6PvV0guXG9UqVQOvcGEl+QFLI8QNBoFJIFInTqzATczSL6ZCIUvK1RlVfcQsWApYMYPwQD8BwtB3j6YFCqiemD3+P5Ce5xYN5\\\\\\\/2Sm0z7N4OGrWPISED9BMNbFwmRXPFS+IsNigK7e3ngyZmacJ4BQ1t7mBmtKNcmlifTICEzl4+mmUIIsDlphmeVB1YrPbD4kTzRGGARC7CplwM06bKh5gvYUxJCyfKLX0DM9kxUJImzc0HCOWEJCBJ9tITWTy4j4LkpeqO7KaTSY4zcjvRK5NPwA8KUkEJp5z\\\\\\\/Sr6g3lVhqJSzlWrzMRI7vHXy8YGCTojL8rKrbfUP18dkpO1KVJ0hBLiPdeMFXi24vIeiujr1nE6tEpjvFY\\\\\\\/V3jiNdn7gtjA3CwLGd3I6JmEn4ETptXx14J0u++q2urgl2zp8n8ZlNpR267gPKLtYup8BDkgjZEOcwjfyl7l7d2bqEe\\\\\\\/vTH9R3iOHqXTEsk4CXVNcuE1MMkHa0ZBdUbo+G0yZGx9ks4A484ws7CNpMPLF+nFPrfvrBoAwfxNEkPgVNPVognBbvv4Xpwa4Do9inIGk7jYYBM2\\\\\\\/SJVvk0qD+6bzQ1FNC7BpRv1m7kp0oKpeupwbjyUkeXwy8nynr8jsyxc9wuP\\\\\\\/+GS2lFBg6eWMQB2twb\\\\\\\/wQ8s40FYnvgNdx49ktaVa7qORKanxHKVEeBxswJTDCp6Bulf9EUJhyOcFIvf6wmd+2uJ\\\\\\\/OSGTkimjNIGAEmYTRRHd292GfTXSjWqj908LPrrcdH8\\\\\\\/CCYFT9gWC9\\\\\\\/Kcdu45jlBzlEwkm66UYdXM1F7Xadzwdqzmh74ZRaD48aYV3JdXeuz221\\\\\\\/26mqlkoyGHuoyeKDvf5c4koeCIyvMc6M\\\\\\\/5bmfH3cefwmwx0n0pLxF5lQJ6ytwa0Ep6j\\\\\\\/10euMHPsAuavzAfwPW5zcDxnfI+EaARWTw0Z9F\\\\\\\/HT+j4MMLWy+x+MoIVksJAtUtWaS4hqLfwDuFyec4bClScn9nlZABS3ffgZG7bKBqA0jLL63XSr3dje\\\\\\\/h51FgNkytCzufTbHRLLI3VtEsGbk\\\\\\\/HR40dbPJB576eovf19LJp\\\\\\\/4Efi+c8DQ\\\\\\\/Hi3XovXOflulFNPcmHA7Ape6nZRs333jnd4YCVpYcUM1eX1Zd2+7XFLGS++eBQSSi83d7\\\\\\\/qrTPQk5fW6VU22kPcW4X0\\\\\\\/kuYwJq7gU6FW+\\\\\\\/NnyKhV0Jy6EM37Ah2uoLhAj8IXGKMmJ2fo21MxlYvTEO90Vjeyp2N2HdvvIJK5RkqJf\\\\\\\/tfN5BZulVYOdN\\\\\\\/KMgu5bnQWnwB+lgwqjudmWI7lzmZUW6lo4tP9dlK+unaMF+rP2PLUeH+y5lDjc4dhKLE9DmBNPiSf18dLHHS5b2+Lso\\\\\\\/9JSwwxwZCR\\\\\\\/i2ErehN6zE6b6H90+TjFWu\\\\\\\/Th2PU92oN4WDA5ATMC6BziMAnlQj4SzLYEwNGQ03Me2kq5mE09aiiw2xYR2vEwv2R9SwInAqzZnCnmK8DyPIA\\\\\\\/Ksj4AWVDUvRtljPHnZ0NrFH9a+8m\\\\\\\/jwQhPgZN0\\\\\\\/9BRg3pPpM2klG+vWvt3i7padKtFUuVxYBjiHceiIrqBAdQChCDQtHoNZ42eeVp6b1yfxaC58PehBUlPj9aR\\\\\\\/8dM+NthjGecMn0NDFOD5UeRTYomCy6wbbs1B8zLIq\\\\\\\/JJN3dUQh0FoGA+ZP1yg2v5x4yYxc4jZsS+UB\\\\\\\/ZY30P1+PIpEOiOE\\\\\\\/N\\\\\\\/TYVfcbkf\\\\\\\/zoDedcZTQxrCUrHF1jl1xOTvT\\\\\\\/EH+T5gP9s4Evc8VGMKMBXXdLAvQTQ7ITqJ\\\\\\\/wxb8XzOgTAGEBBo8nLHs13N6SPNefywCguyCLCI8Bvn5kQEbyEVWyTLf27cFXCfc\\\\\\\/isjDgr8C6qfqI5qCpS1qCY9symGrCtEwtd5Zuf9JH3+mg74XAvH8mHFRpQIiV5Hkzd1rLDt0yOkP6A5+32vLqum9uAgWSmKKFhgTb4LrXlwClu8oD1XrZ0BEZSwxEDGSBYYB4KZZRpao1S\\\\\\\/ElNtN55Kmk9tEtv8gbMOwwpJnXGEzGzoBHqbbF6ejfpDXfVeH9GUXqFOncAD1col+OxGIJzGeWTdJfOcqIo6DxwbsTUJUW2FTxTlMI0RvJAtT6tLDPDAwwG1pDBHY5PoqB2yT1zUYm70EfH5tvv7UD4RN5WyIg8rG7WPHkDj4nf8noYo1Jr1yiakF+a8UOdLvRPoAwJhKyI5ClToLI6V23FT2B3Sfhueozx3M0e63pncgSJOlW2DfHThUWMQ5S6bFVDXHIpLsKQKCsXROL\\\\\\\/OMsIEmf0Y6jMxMS+5geqww06nl5PGwZuUojoFqWj4FqghuhzS0QhAooa9qw4QXYcmy7Ajq4bEY12okuTE9Gz6xdEAyXmyuxSpOlzrRlfb2pVA5DKJV1q3BOPmv\\\\\\\/K7h4DFLFgYVpLGtKl3+IdnzmcXsrXvswUFQ6leJkYXOeuzrJF4RIy4pepc8rohDOF9WbblYC0XPJN5CsZgT6hlKdu7Rgfaz3G1YYmixbfi\\\\\\\/jeYDtJ0HTEuG3sYzy49rLnSHrzPUveDJh5BwlI+L38iynyA0NJ0i8w7dRMpeha6CXnzluGjUuC8PilBdZm4UIBOB+87si2Sf6OUQFzsX\\\\\\\/8Ea35DxO4S7CgO3ptsyIt1oW4OOqUF1L1bZiJ2E1osPfrA+3mf14jAZ6Jeearn3bIM89w164uZqoVmPwmPWod0LkFAtkmYrvDsgsOl0uh1tGZxJdMy25YooLY5IcR3+08tp16Fh1\\\\\\\/FWTAujmlYkShYdylT\\\\\\\/J17PEC7LOr\\\\\\\/9me88nmvTmKTn2qy7agyXCduRJ78WECNPwYbuQBpb84ijkqvLFYkJNElfUXNcjVUPSgBPB++hXPg7rnArF1jq5KZrFnaLhGBLZ5zZIKqNbj17FXxGCUr2tA+WlxfhXAIOCsHUqm+RoeqYUA0DSU9dnf\\\\\\\/bGNIk\\\\\\\/LCE4GawnnepXe+hQ\\\\\\\/zY5GYx1a\\\\\\\/OUFSWcwiwiZ8u1IV1310WC3sLgoN4m2oBf0opcILYjrGVuDbAR2fejO6JFktF6qKyHIT+dj5fW8zA4AdqOQadfJgyHcRAUcXI5N9rw+\\\\\\\/xKs\\\\\\\/F7siRtBbo4P8rTH1uvn9XuU82WfiQJsJLINsZpCSVAyXgxauO99\\\\\\\/2dZ7CicET\\\\\\\/uNybINwDGCYSNk54cO82aBFhm4n8cmEqV6Ax0eeTDjDXSJ4oZ95EM+0RHk+7D4C\\\\\\\/weIToIVsJSkCsyzxl7WK8Ulo+60kbCzkKJO0ddx2gfONCjdTEh632JDPm6FAXpoMqC0yWF+p40lnnpE1Qym5GJpmU6LJB2q4zMxaTyvZXb4ExeHHsFkBnmDrRk2uGKkbKJVIcCGlubpoTD+0qRa\\\\\\\/E\\\\\\\/6dZ1mRJ9pgWzny\\\\\\\/kd8MkeLO3CZyZFRceuloIF0QmURwVXjj9e6zuyGa1LLk\\\\\\\/quIftffDfLhgdbl8G6SMgsVHlfCn2MpgenTAPqtbJij\\\\\\\/gGVdEvoVqBnpzVebHXlQdj3\\\\\\\/iScyPKvmdzn\\\\\\\/KGNwpPq5m9tbW9ZOkixWaLHuzBzAco44\\\\\\\/IOv\\\\\\\/JRSCfvrTmvpQI\\\\\\\/IfdP9h2FbBwXwdXTu01+zm8u2j9xWumprgnEtar\\\\\\\/y0hDvNSfzjYgBc+Ijleal9ZyCIYIw4nzsJXHfzcCkqAargQ0l66V\\\\\\\/YlutxampZXSyJLVIBaRifC0zMIA6PRnc\\\\\\\/MW6zMkKvAe3m2zsrr1fDqjQ2dBYFiDev8v772MRddya4lnmLHTcWcQmBLIkjUpbVWOAoxtYBSVUrLqLkYv1wpwacXtzT7he2a0ebn8WBxqs9KMrQb2GdA6xibkPOB0Kri\\\\\\\/0Hw0dUpgnPQq\\\\\\\/6AjtW0Zbl4qpWKb1EYnNXpRIf4\\\\\\\/uAtCll2G\\\\\\\/yyMnu5AzdNFK+L2h\\\\\\\/rh+aImBK1ODVrdyVmviPsCCVPTx1uv\\\\\\\/lQpddPQauj88khQjX8anqxAZKSPjmmbntxylptY0ZhXYSOtD9xdTroai4Fd7AwuQmWFKqc7p8Y7\\\\\\\/9ut2HdR5k7qJ\\\\\\\/A7KHUm68vRuEc8RZ9AcLKH9PrFWkn11bG9ctlzUU+cIO0ueslTvdFO3OCfIJfB4w\\\\\\\/f3NDC5ITrcPKcH2wXuBsDUB\\\\\\\/WPVosAZEoOAe8KGALmX\\\\\\\/hDSA3qP59KK3X1LJYq7NzRFZXge7\\\\\\\/CtQuoeujA4sGFddnSpflaIapPckNaejAuz6yG51hOZ5jzjSZ6y\\\\\\\/JtQKvuYKARJTQMNS7iJF4\\\\\\\/lwjWkwtGokO04O6LpGYz8wQ1Uo2W0rEVVpFtHbv0O3\\\\\\\/CAQmo9opXimt9Z5zRSY3nthS5nL\\\\\\\/Bah8AmrYo+0ritRlY2LNDjuxbTXgLp55fAM\\\\\\\/tpecqO9HSJVzRgtLlBSiA2\\\\\\\/T6gOFaAXGe81yfSWKhbeTDtJsJmgfnOyRyEVSPP3RI4JHXMutZatnjXoZVdXQmLattXym81dxLuvka+HkL2n9AEbvIH1Fm4Niivn+E7lIlzuPpQG\\\\\\\/nf\\\\\\\/\\\\\\\/l9a\\\\\\\/qgz+ucaROuJZsPSS9yB72QWw46uME8PoENp7+Js8NFB8IIrCPtzUHpWbLCujO6XiGpB+DiQkE7ZYKRWO0PZnytr7PsM9IOyBHxz2gAwQBwsjcmDKg3zz4nTsx41T8uFI+rnopibjC65rfw2KLoA7XFV52Oa1FKJc5yy0pJipGGQprKfxowAmAtgn+xXFQXYnRv59k0sVdCEAwQy6HJiBuY9E8378sow3GWFy3XHKIj+Si\\\\\\\/Aq0NWzQ6nuv0HbL+92IYgHPpvomeu7lPpsJW8QsH5DEQV7e6pPw8XTGibOTaUakMvAK2uy7cUfcrrXJZokhEH8tgBHFchyizF1Sa19wIyxwP9nRKkoG1C8YWHcFn4RPI3enGrP7tKRuWMLXNWGu5ANy0Anypm84GNQv+UmsLucJPszdjWz3hLU6dDRU9ZAuJor9sotSCtlf\\\\\\\/ggIbK+H2NsgTB0ONaLSiRi+IYp4CxQTrrpLOUzs6r1wVxMYloZIja5uObJVvgybfW7W0R6AZzRWlBGKsKXi8euG\\\\\\\/dkJGmERJ6MBQCKXQnfjfLry\\\\\\\/i3HEBrOnXT66i3dJueU08IYGyFFxlufwQGAY5vpFlmYJL2Ir\\\\\\\/jnc9UOYmlPnQGx527HOXhOUrK+nl3qhXxVobtbx2Maowd3cAZDc+lOAXPjH+GT3wbSmvdXMoZEdf2gLL\\\\\\\/JVmSnZxjwsBHvGXjbjHwTykZf9kr5\\\\\\\/uRyJAJq0agHP0ZXOi3lVUFpy9b0GJpdnsM5XBbkVmT7YouIg19JyevuI32YjulvDmHWiF8OEnfSIhGaHnLJ7KLB+j36MwubqAq80BHiKvKqcGqQzTFpfVuUbkX3i7bAc4sgXVimjDUhlIK4GK+XgCY52IEifZ2FRMjoeISNWcG9Jz5yJZn4SiZRbbDe+EnJJlac7yGKnuuMie+ejqVQ7F5\\\\\\\/BaSbSrkTNypsSp3U6KdCz49834eoXtTxYXi3V1WyHMFrOtoorZL2CiuC8+4DTHpOj5VBZ0lp9goDdbebs7e8RpPjloq1gRwHsMeCbpin3wBJ1aXKCPgiHYvTJ3L8fp8Z22tg\\\\\\\/o+w2gTeBvIMTgqmbrnal1IwFocirDmHeYwTKRAyYJOWgLbOG\\\\\\\/La0az26MQJifdxDE0otKqvlSYiNk\\\\\\\/8187Kle0XyRG0qLF6VFE42QQeXPptApf\\\\\\\/uWbOTCSIZ1eabhTLXMycYO3iDOHF3vqTzItOkYkav4YNlJJlpXPD1KpkKDspDpowB9TXaqeoWIvZPIQqIVfEBDtCDYE64Oi70zj28CRqlJYbO1t9J9NnArTRlqDTSL16l9G\\\\\\\/8oq8VP17OVeDR8\\\\\\\/sFJWhJ7babFu2jKFt\\\\\\\/ZU4Wxmg+jgsn3IlaUVsgSobhTzXuGyjTg1BHy9v2azdpHEzNjIXHGcp7yB9O\\\\\\\/xA04I1daSpBZp+nU7zX4n111B6j7z2vUFBfBkmH0t1SxaJSz5x7w\\\\\\\/whddhjh19gM6i3xfg6fPa2bUwZyEYBC6VylHrb67vskObp5\\\\\\\/1Fjl5uHbQDwZnpkRBpFoyZ5g2swGENASsKZmyU1\\\\\\\/ZTTW9o6k1UWpdFNiP8ht5rsDF2Pk4KVFsjMWyM4SFjPOlk9ky5ieqaIHwgE6Ofn1gS+2APTv8U4Dm6tQHeZHfDdi0ykUGDqL11EUjx95up60d3amWWoAJ4\\\\\\\/KJ6ylDXCuulPUwFoKvoc6dnm6dkSK1HcoUrHl3Gdsn5OKBnwAwTpXY9Poedyr4HTQSosSH92DWf4UpSy+O3pmH7hCH0oGXCgg5rLwe2wmDWe7qkXzEhjkxTbeNhOLeFpcNTDv\\\\\\\/PB9ST1TKlaKrfBTkZREEBKKTzqAaTPtFOI\\\\\\\/OW3YBHYv2+rw4UnKGhT2BtSncpmonxTPwP5m+b5HCVhLwn2nVwq1vrbqsCcohoWPLh\\\\\\\/EIIhwb4zIdut72SixmfEFoakhR+fiIEr7jfpPofCx887eUv8TLr2jjRP3+8Amt07534oes9XAoqR+K7zCilGR4Qo\\\\\\\/7qaRnqs3mArwEzQGI3Ov996DSPx8XqWPCQVcw+seAhHs8EfJjDndCj2PfGipqtRq+6xX5WCjEXZYY3srqeDExzwf8AK1bE6jiDFawM4+alqGSkHfDsAzNiGdSnQeXTZG5e6m5AHr\\\\\\\/tqFtoYCH03my3lvJlVOo94tM8VxI\\\\\\\/a2iRri+xqRNbDCcHLTFDcBFJPS4miI8xcYN6BI9haDtVTfchQyAccidCW6h5jpDXEoGGdWUVjRfGuKQbptO7hdR3I0No0MTn7NolcCQ8VqFBeYxk9brCnjdOQ08bRCqTMjmjH0E\\\\\\\/sstcJQvBijklBTLC5ndR0HTEk7Yed6qJ31s18e+4\\\\\\\/UqLkrfElULwgxJzByLoSRkmwzKHJMh97YX0jCoJihOc4RlCxTKM9bq\\\\\\\/6UM\\\\\\\/aiAP8ywC8YfdOpDcT11LSGTHh1IzPuaBQnQSzjguLUe2OG1D8M9jakKDce71grgpBvzfdRrOVT2gTYW7lhM1RSGPWhHGnx2RQfcX47EIFQUGVjnKBXkOoqIKE1wFaCV60wh8XVWKp4cTcVRlRKwJyI4VxFXQU5dJQVWUuQRNJVApoCgKNVbTckxYHFLZ4pxOGXabmSetDrYj6pxrMwy1Wecjthm45Nv4qE+CzRvoDU5vhmaGXassGanHztwQfs9LkRyz+KyNICS5HPFUhp38i61JdF9KTYLRWSOJe3p7zpYczuHOSdeJ4oAB1r1y+zdVFrKQyFirO1mukgaoUPRFiXZx2Vz6b0H2OCB4JHV3SlhfihYffLhXKSkrHz9YLrnGcSZb34GyD+scPaXi3ONpoxcEOvM3hR0NAwHf9iStifTyLyZt80a5pUPoZS7InHU7Vnjqii+GWzbObSKhjOsd0rn1Xq4gtHjGB6MRgyy\\\\\\\/w\\\\\\\/w7s8czajFSzZnWoKxw5viLVonPCj1oUQ96mR0CBF+1D8+RbctiyYbJZvAAxWeFVjoF1M8aifYjAzToBdv\\\\\\\/6sTTg5jILlbI0RhewjJV+kE4tLZH1H6zlmYy5OowXjOc5haQgS4XU6pAmevuHaA10AEnKpmWPIkb3soFrkXw\\\\\\\/8vL0fgN9sHiaNn7+\\\\\\\/LXtGZUAv4T3DbzDstwcQUT3flz9bcgHPrOeUUl+3Idkfpq4Gsy5E3U6q6MHTxtXnb2aI3KE+8Dsh9bgBpRy9AVBu6jHZNINdDRa7z4k0rjYjNneOa6d3vfKGB+\\\\\\\/YnJlEP7+ZvbsHFsm39MHKYHE55QlOirYQpq8Hkp\\\\\\\/8In75ePYPsy8Wn5yNsmbKPmKM9EXzMEthyyl9cCUL52ECMdnKzKRnStuVSW8VWh6aAbNW33ksN7DfOVRV\\\\\\\/kswiIddoFhxzYur\\\\\\\/fcoGTc7M3o7q3ceDG1UjfHIBUpbpL0Oe8njWpNGLIaurXNe\\\\\\\/Rh4zJVduP6Qd+Pc3S59SUklLW1sZMBED5qAN98pnRTXlbdAOHYLoInWJWF8cY6y+YWAymA1pPjeHNQCna60bdBsL1mNe2Te+5hkW2qe4loGGjqFghi6Xf3fy24kJsKrMOZ\\\\\\\/mM9D+\\\\\\\/R5Em+y1OPtTLeydSa+gUFQdhkfN2uf1m9p52o4RrZrhIaWuZCUIIcWg1YxvOT6FTAvUNvOzWkGSfJFzGIcm03Bze\\\\\\\/ehICZI9yrgz84f2CkStqYDRB89h6zmFqp0vzdoCtKrfdYnVq3SrCOgyEr2IbBhjcKEicYjbzbjSjh5oTQ1sFROr9aNCpZeyLXfWMmMAS\\\\\\\/WtD1U+ylpVxPGJK9BZnxDnQwbGeD4MBM6U3j1DWY0ZcqeciSYx3+2aX7l5kFfkaPDPbwKYfUHVNsvEfnlgSMc7JI7LgE14OZUDy0Gl+XPPktW330GJH8c+F4o2GA60hDg731iFX4RGVg6E7WzoTcJSfHOVf3VI8NmcOWW9Hk6HpgtJhEzRtkzDxWm1VTdhoV\\\\\\\/tafG26oddOeOwGgxMDTtKrqU1BWoMK4bxDp4ePnpPutN\\\\\\\/\\\\\\\/sIRebVfHxJpgXbTgJSnAmzPJPcklvQXAoaI15QJ2Cq11cMdVaoUC30zpY6YxobqGl8szpXgKOrv81+J88KzalfU3wKwOAGdsZMBeaP78szY655glaFef6uf1a8Q6b42WFv2fLPWT+jmsiaJoAUuoEvFRh\\\\\\\/XmRG1GiCYOpEwGInLAQ0ySilgCuOKtePy9\\\\\\\/TP7ZcehQoeN47X8Mox4BbVDoh4GkZPx\\\\\\\/Qvzjxxem9IjKvvGbKBLnLd2SOImfoo84Knc85csG+B2PzY26q9c6z+dKgCil5EaLBP88HsMSz0sbWRSWfS0xBdnJJhZCcyUVIg0PimzHKwC9ktjbYcjkhAr48SB0vzTw\\\\\\\/GtwxtXv0FxKgOStgGtsfL+xasOuSpBoeqHB30hgIB3tdZoEYXgFEdwAcxQ+xBqJSJLK9zsOgvt5+IEnl0QIVfHWzlGO9BCTLCSoiDtgr2ZquwaIQ\\\\\\\/M2A3qt\\\\\\\/qyfc\\\\\\\/uDulXy\\\\\\\/\\\\\\\/lzYzhKnAbSdefmlvt5bj79JYrKKr79emgWQ28TI5eDFe7HfNsmJMFFDLCaP6bHCeLr1b2jPcZQVJcThQnk2PUmUambBMoq2K9au+jlDToT+PCnr59YdBTFjVqd0VSNlqzkrtgqM7W1SwnZGTMcDO5Hs2OLn341NmU8ggv+1TJVapJkhQnEm40qT5NX7LCmoejwlIJTRRGzRliDyGf5KMdsG4wXUZlAhlFshWhhN210Ciwvo8DYDolUvJ4BUj+rOSiBgPwaeuatfe+UfTBlWXgkEKeIf2GvWsOU2A7G+J45utxiYQFORrMu4XR3aYo1G0ce\\\\\\\/hpLY7vvsjsMsqTrLXOP61AvlETHh\\\\\\\/Q56aoiTL5SWhim0XNbu3AbPCwZhzx3J8EjUh8\\\\\\\/gmfzLLi3gVAGgbnYqTpZBGkF4tXPfTWwkcbSw+WoKyxJi8TJhb75wC0WNNKFlCzOqGjf4IrbN1WvT8LxrZaig8syFwRrvBuetDwB4Osebi1Xx8tOWTxYQdFV92b7+h+cr3lyb7SnxEDRzr3zCh5HXHBKAsj3hn7\\\\\\\/ZWLpc\\\\\\\/E7FUSQvvxx9TtgVS2U5hL91NEU2KABwTk3FaeoAWo4AVT32Ml4kB9Xkl0RyLdMWQ8pu1qNZZjYGzcc5PQvrBrLZVYCiI64ETgCu2o1OBelOnML3LX+LQduHQ1uWV9eI\\\\\\\/H1X2D+Ztgdj+mIMyDjON2d4PTW1m1313D\\\\\\\/w5qfuHRpeX00VhnU8ZLccfTShz+9Iq7S8IOTj59Ezu3qnA02bweoOwpulxB27+JGLtJ1HQc\\\\\\\/HnT6ka9G+7FkSxWuNfYUI2W7fzlGGZ8yY4qUoOY2+vS9HkMdx4r+aINbUxjq0REls0b8gCM1P5SgTUZPYGb5e+0mpreyumK+poRUTzGauc2gGl4As\\\\\\\/BATTnIPZugnlLtEW2zCt3C58lngVH5LIjd\\\\\\\/jm+eWVGIKl5tvJPg6IMqdgvjtablnTXbjoyRps8EjfIHA+6zZVGmr6sx9Uqq0e6B7TwEfwNygSr8AucP\\\\\\\/H3gm\\\\\\\/STKnLECyFfu2lhFt4uZDEM8Vwd0Q8c9F9fq5CODTGr1BGaUmiBxq2tvNFs\\\\\\\/nd4ecvRxIX2K0cH3LdiYPNq4LAfwn7jx6UNLBFJ\\\\\\\/w\\\\\\\/jsFGdGk9HZ83hhrzLSIiloRoS5PMYsCPyE6\\\\\\\/vKNElCJw5qqb1CTa1kCVqXsIlg1qS3MIPDHLrsWxZsNUHtQmjHISiyXwKjmEGe0N7RbOYQoeKJWJCepjfoHFUNrZ5rVcoqkFpIUGZQw1x1ngMj61a1tFl8X6LAZb7E8nCmtvgswVK9x2O2o8mMFk3nT\\\\\\\/7DLiCzlJSLjKscyBFEjMLI8Pz5IJJZSBeZK4SonilgI77PYboYkFvq+StCap9hx8QTFXYndghbQjMIUND0VIJiqlzqd\\\\\\\/49AiMOFViDCOcOgPkZu4LQB3Pf7UHi6EY462nOTjzGL\\\\\\\/hRrqJ0WRQHYtEGPK93\\\\\\\/Mti\\\\\\\/h5Xwtpn4IEw0lUu1gaegrOQ0zlqKZYvLFoTY5nakrS+hAKKKix3IOxY+8HLNtNRnmv789kw3ZhUePtn3nQtPCJsQKwotPzV1riaeV\\\\\\\/YBrNR3tzvKiICeU\\\\\\\/OIBn\\\\\\\/Yd1UuvvEf+BOqQ+t0NJa0jvELHy4ePFCc5llUtaADgnIO9qbKY5\\\\\\\/bBsfp2uhseh+mC5Yrbi9LOXQ4FJTvtOhqqVNH0PpG1DvAN7BO5FVrUggQy4lFzT8oazLJ9P10XlQpncXbn\\\\\\\/K+l7weNbw65K6M9zYDgeVd6t6iS5Wzzd1rIiSY1owWo2bmHjGUyGrbmniV54woab2\\\\\\\/Rn1vlZyyrbKj09VUA6zkTx8BfgOIzixAqC2GVC4u8bOFYkt37iK1LU80giDerhnvHf\\\\\\\/ahh2JZ68IjnoIgGP4Fu4EczeAsQYopXv6rnBUYHQUHpjyed8Vg5g7pBn0qkSwj\\\\\\\/hDuHfeXRfWbneFuya\\\\\\\/\\\\\\\/lfIhssSIJ62uPwCdnflgvdh4JGREHnSJGglc1pJMuIczID93ara7fSK4lNqrtvuAawsFWDN\\\\\\\/QxFwXZR2MvMwjjYTBQGa26r9QP0zUQ2jHTWoUOm79TCHWvDBgfc4B\\\\\\\/CYI+8mI3MwGOSgIYuZ4vNAexufWIo\\\\\\\/fCe5Q7tU36PDdIDIGvftz4wcXyRIiM4VTMCF7G3iEwr5TtmG4guiXK\\\\\\\/KzMkvRYL9l8zOx68L4hJXvY7rpJEQDbBQQGEMg0yG\\\\\\\/Bxj0cqmopQTRPuxZ\\\\\\\/teHfGYe8MgyiQ0XOnG0faRXeaXAVQ3pDSqYtFD9sXCYOmI+lO+EOC+KZbdGw08joSNiKTOgrNCVEGOGNUZGok3BZa0lsULwv8f8UGIroNVH7NYwNQ+FJfs8MgYl2\\\\\\\/SwS1NC5wXUQ8LLdZ3Fg4zUDsP6jaUBjL\\\\\\\/LAnwAEG7zV2qfX7CvuJpY8XVHrNpiTkMB3S1idPxMEB0VHOLwCt0sF1PAH0+vIckan4QdhEIbQxHWkFp+wFl5csDM4WSg6zOgAJipXZyHOR2lZEMpNrvTa6uxVPd8QmbUlreObhzhm0ki38\\\\\\\/+gbtt\\\\\\\/a0MfJy3Y0OEVFXpcAKyPHDKBnT5qvx8gZsviF9NR0CmzsLZUKdx5AJhgBYkapYdvDOnMYLN8MGkpa4gPSli5k+8jJCtsBxUw2XnZ4p6yORzWQJPU9MTMiGg+aJ8O\\\\\\\/+Rv6r9PjL+f1pjUL4pkp6DoSKDS\\\\\\\/W1Y+EZDs5XoFAQYvlFv6diJgqpGiPvPAWNAt\\\\\\\/UIBpWykpnivousHjaIH10brrK8toF\\\\\\\/YwPFQL+4p4\\\\\\\/vSBIr63doN\\\\\\\/3sXGVQa4rKpo0bKKcdXcOH3UmwxmndQYFIyQ9h88JhVAAsuLIbpfcsFxp\\\\\\\/sFCdlqUlnds2oHkJV4\\\\\\\/2Wj9UwD2Lu\\\\\\\/n9MJZZ71VQ9hIfR\\\\\\\/UyehCbDSP9v97qLz3RsoCTBHjU3dTcqOqhd8wOSWzYdDikxlerBa76\\\\\\\/y4Dd4Bg1gXjE2HGSqmkwN9TlNVlkjQ+B6XLA7++1frtYEOqaHB2VJ+utt1TuQ0WvcWE734juorLBNtmb2s8GhbqmthF+rQY\\\\\\\/7FC6+0ZwxmGmfhp6QmdLjJq8MuMjhmplTRVnuC4kqvb+w4vf9ylT8FtJJOmrVBLlGFkQaqIHO8Scpbzgvml8QGlJFMbOQ1B7j8YHEI2fwpOR0fZ6pFj\\\\\\\/7VOFdC314CVAah5aZLYaszNISqF1PhM9Ti5KK\\\\\\\/\\\\\\\/y7A1WdZIAP55ncj5ea4kbG8FfgEEl3ShyDCFozkkDusYR\\\\\\\/tbeX2Kp2LZZOC8yD+0c9qRou2vycoHPTcVLQVSaER+\\\\\\\/qNx3NwKVPTfWHkXn3Fva7prGuE71CNdIYJOYU3ixJCB3iQnwOMVFWx\\\\\\\/u\\\\\\\/ZXsMhh6RNGD6bdv7QO4B7MiJNmDrhk0ns4Dfe0DlI2AMiGypHH8g4hy20Vw3bZ3aCUSUQba1fEaCqha7WmMJZ7Ow5T8sKXMlTRWU5AU47SKnUTc8+Whe6qZg\\\\\\\/MNs\\\\\\\/u85XmDVG6ftXQogHhx9klVMObBXJnYPGeoAMxP2IYznMdkGznHgsP9GGOP4IIU2XeIYI+03EJ7\\\\\\\/umMmaEzhfjFGAWCnMYu+6RGFTzqRCnz2zUQ0WaiAEx7viP1Huz\\\\\\\/VThK\\\\\\\/GYfAzVZhTin7Q4NG\\\\\\\/IP9SNESCqlVL\\\\\\\/uO2lc5TdbBU0v8B3Q4TLyPv338lbu\\\\\\\/YKXUNGiUYK+xnoD4wzzwfh\\\\\\\/T\\\\\\\/QMl7oqDeLxow4giR3gds0aTPNQaPeO9aRtni6IhudRpDbRwdRgcIZmA8NRiT21eWBWs0u7iPc97b43+QS1pco4ROLy8u+2oWnvDkSWKstVva4FNm+7DqJ37Fj7uxzDwuLN5\\\\\\\/H++5VXArAEKEOJ0YNdmNPZQ0Cva0ZuPJjxmUzR3jbG93cd5xyANmzhTcikWstFDxRQnBFMtEQlb5XTb0iHWZAapvKgeXq8NWo6gohmQAS7ZDlwEhtPwRgaNECy3pPwHgyv+vo7GjWSKLieAnMKY3SxrQIf2qY93UuWSvZi9TkRR2bijSzqGsSoNmvpKWmMGQSHggHAJhl8Ui\\\\\\\/+QrSTn\\\\\\\/0GdHnJAKHB9DsZ0U\\\\\\\/fe6aCoalmtKh5CUNn3BwYp1iosXC+4X9h++BzcwEOXjEZ1ZTJ\\\\\\\/AfsFQJcJ2Kw6cCN9RrVg6j0TwfqzCTeGaZtSPwTGnKDv2RmS7B2jp5+QSJPsJDWkfehvzymKQDrLSM2Zf2xaLh+\\\\\\\/7hmyRltRBlHMAweTmBzb0XXRtbZqDrWOi8BgJulz9E7+7gOmNyzDlcqVYrAiUwPnfjgpW1aMTGe+f4vbZhV1n7Q4kBNLm8xLdzJcAhuL1fC2dnxRKQa4ixZMNQ72wDFpDwbrnf2uzHbyqeuUvACPkG6zaqqkcs+s\\\\\\\/D2maXrb1MxxT9LTM0PMUhHs\\\\\\\/wjyDtMplCrojDG1oC7aDZ27tFzXjrUwVVMsGMmb73RZCaxitTIUfMmEaBLPYNzVFsfRO3qfnVmhbx8z7AYvhjvaycl9qhFPjrUqdmm8sfZBiy6Pb+PP+DSPxt5BnJyAm9DZtGLAv8zpkN0Ai6n8l\\\\\\\/0WpSzMIru4xlwirQMnx7qdnKqW1sI7zGrZW86iUSwW+BJx9aK7uJCsU1upJesXOgMsPCroMbVU45w6G2zCPHtvygD6JdbpvmKlnno3caLnftcSIyRhb5XPpSuZsxt0L50u7P8n2fG4gtF2HY6lFPylxVFiIdhe1eT\\\\\\\/O0FXL3T\\\\\\\/8RccaoLjQqKionxkbO6AlE381sUaVvXJQgKGVhdcaWqT1bU79cypLx5cDFoCCHk009tTlMyYZ3VbPRLnpvtOPTLbcvE0oSstW70eEKzvHd6D0wlFp7Lxx\\\\\\\/9ySVeWT2lV5C+CugRqSXi1yfhmMMCaYWplnRbWGyCOvtB9wUo1V53lUSufIoXUgWkOyG+ba4iM\\\\\\\/2sFu5PS1VUxYJ4VwVtIc48Y4Bj0GyvB78LXvzszASpBNPsgeYXJenef5z1G1j7OLV+qxawtRrt8UeSALDMxuHmbcm454JLfeHZttcahkNdAPdzeX+7okSsI1FsOyYwYkAKFn+jsyHAwH1nK4jXPBR6gZCDt0IxY4rGNsgyuwLRli6Zg5b4ML+gLMnq3GNGalmrat\\\\\\\/dj7kEXtSE\\\\\\\/R+tpryxNad5FaQi0WlUGbE4ODmSEHRZ10UW2L\\\\\\\/FS3Dad+5wcp+qX\\\\\\\/ITuNyZ3cgT+9ptU0WAl\\\\\\\/RLX5bBpYF3GyDwlCdqtTlllICooSpzp07WiaYjDtMuUrDwnABP+N\\\\\\\/VFqMUJZlPTPBRVZVGU7zO575Kz4MWzxLda5zceHZ42i1miyiCTwNRJXu7sQRjgjlqKSf24oHI0FvxJwKeY5zCSGJyPxZTcKYZqiOC9EYg8dOz8dl9SQoMJAH19i9IyZWk3X4PsWa9z+Sh23MQ7rikCXrmCp3ovW40gTjNZkvbXGJNd1UsjIwzug7lzx\\\\\\\/ot9CQq23lVww0PP2z3QQkLe\\\\\\\/QlwhiT1Owm2WwcCuO9OxN5CrRRYXWCkYhdD+61uDgXe44tFXyZG4thOoh2sHvvqWuUG8QSboI3\\\\\\\/GcTji6qff5wAq78IFnx+hy7sp9EBjehSqvokc17Tc\\\\\\\/qMPpMyPYMwX03zyKf9y9chq8CreC4Qm1FetiW+MyO6IvxHP520MH+c\\\\\\\/Zt2YMK6bZKpBqhUA4zx63h7WyXJtCsjlKDov4Zr6DZy13zznYSzbMFmnRRdk6fUrOgItkM4VC2d6y+1EQr392sXmS38Tsqu1C6uL0IklonJ9scfxdgKdTXAxLwJfiO16IAiZSHVcM4qDKh29IKZ0QagCa\\\\\\\/mBkk8hrGPEltKt8fhS73XnP0e6\\\\\\\/KjCDvT61MFY6wVenSHTHxIQYmjUXWg\\\\\\\/kMh8Cfmds2wDmprSOzDoEI\\\\\\\/uCHPfa0aO\\\\\\\/HHJ3v0sOoBO4WcjhXvegSw7L+n2ge\\\\\\\/LWMaQyFb2yLdsfWVA2nWYltfRXx9dZGxfUaPgImaYITtTmst4QVKrJAyEGew0eTDRuD9Cx1wzB+ej+XwiIO74SyGfFIkVmKhOsNkaoTH8E8PD6s+tRKf77eX4aR2GyphYuTSOAcSZxRFyAbyKA5VMj4ijVN6IlaO3atEXwJOXZvCz0b5eirJQgOp\\\\\\\/iYsShe8AM9QL+nvjf5DI1tgfCyViEiAEyOzZzTQJQMNg9QY1yA571cphTMIzBHK0MPz9E1PZ1uLmkSnis1goYl0c4aSILgqPHJ94IpovRAwv7DxUCwRtp6QmyH7PIsgTk2YrqgpWs9EBy8ioHpKjjNXDQooYQZD7tKZcj9BMchHRnz7\\\\\\\/+5b22GOQf6vKaL5Q0BaWGuGrGO5djyWfNuOOehckCDVsNkHuDeXPeRWy0fPi\\\\\\\/dVwfw42J9TqhtEz1zVZfSB\\\\\\\/TMnWa86sH\\\\\\\/xefqXS+XJ4b\\\\\\\/LlRQNkTRqEzVNeMAsOmpBX6wnmbTINk9HtoGwuh5ZP4G4cHe\\\\\\\/sM3Qf07SpkYOZDYd\\\\\\\/xKgtQ9yLBOMUGBb03a+2qBXNEP7AWdBxCslNVQHZOEOa7wSQJPwALu4GTuw13VrIlIovSPnpullv2VVpNBtENI3tot08\\\\\\\/Pz4TmyUTevZ0+hUgi2fgN4Lap659u0Cwif5SGl5vrDrJCZv4zEAFrOdcp9p454sMxhPgweCjpCC4EuToRQT66MyqES5AsvDFdkuGbSHa76krlsozFrLzKXOYmIgA5IotamebKFT7HBuoDEPWNc73nENFPkDRcRaiojH4jkJyxgk1loAiBHVveZ2vt3FYFXXwno6sxfroXyUTAcDGg2nyHdibYN9Qdow5lX8tqWJXE\\\\\\\/71RqyaTeBe1cfeOdPm20LCi025GEAQ2YfGlu1LLUwF8YubKQFEOWs+AJKSAyyExLS3Gv98cAmEGEDCp59FQ2reRC8Qz2TQF2qu2yrakoTuyn0vaS7QeSsbE7w+nAiiivovSd74rfQ9aUqyBVGoFEemzc3OLsfedaZ\\\\\\\/WHmJmh8qo+SIqHh1muoE4kYY2+uq+p\\\\\\\/+0mrjrpQ9UY\\\\\\\/tQPMZDWuHKtOrSC9JfkBiyKNroKPYGl00daMipJZqQHVVmBO94jnTQx0Xrx8lyxPyR5yrpvLCAbnchgFNgHbwwldLH8qrltyF4I6B3sCsM8g7RNOKRKH7EeEb8Zx05pY6hxF64ow\\\\\\\/xa1lQ8fYAVndIRgY78JdyM86L6rJK4KJWMyV1gU4mCw3Ekj6rpxc95q1zSw\\\\\\\/h3ycXXweiD4exClqcmbf3Muv5pH+rbma5M5QVghbJLqcOWEUBjEyujG6EcM6SDR8fne5eP0t9xQ8o+awHkOYFdHWrPNixktV5lQCBY26n1Pml0kX8WSbdf60T\\\\\\\/4g3\\\\\\\/LDOouT3ONcAFx2nom6n2v0Qp+bUj\\\\\\\/ZJiR1Liz\\\\\\\/Eca9\\\\\\\/MvOiW9PDy55oSIJIs0TcwEIkGB4Zb8iswLIo\\\\\\\/ZO5TJElat1Cfnl\\\\\\\/KciEbN8Z+5k+tmEISr+7KI4Ksl5XeYaKpQxgK30ki\\\\\\\/Lr3opfSS1wLl9SGpYESZ9rxagag8n6q9FSbxNC5LjAa9qgLiu6DB2SbYFt\\\\\\\/Chax1eBvngb0lq0ukvV9gtA5OzGX9IZ0DyB4E1j1vZViwn9DK8De8fOybt8ZdhuNQPPaxPImAvY7ds1r0KhLtHgsUzXmXWlHCkS1CskqUExyAu684ThtXSO1sZL6ONbVmL45+bEdBBpSx2wncANy8jVZAgsMaRDOxyHAaMl3pWLuX2jPlGSBunTmQGYFXapTzpf0gTAr6VhLQ+yWvzWCn0+d9Sog9T1EVm3ungcl4YGwYvivTRCwUf2YG7i+e3yq7d0QpX0u\\\\\\\/bhjNHcrH887SvNJUAWNg3Xa10UrRlhkW1i9BOTZyTkfH+BbygNLpGqAEvLHMPy6d5GTmn1i9faexPrV\\\\\\\/0DqbeZZscgSzV7gC3iRT07vdzB\\\\\\\/TnGVe+igxpAkxaVz6p6pbSA+D33xJjaOliZ7Teb589D0czFyOMbgIu0GViM29eio0oHey7CASo0\\\\\\\/uusUHfmJ+JZmXEBB0pkyi12xO2mYeoavCOigeKrm813qLtOYIMOytc9qqubbwYuDCELvBFYoKhEqGxQA2UcqNd6W3Sq8UgIW49wXo\\\\\\\/yBEXBTJOhscCABPvBTAW5y933yx5T\\\\\\\/p69R1GsTCzPweCb55ykViCxh0BA6qnwhX+DKg9I1vdyabblFgbhwjdomGwiV3nV6ahDwWvlaE7IcfBRfTGak35aWgZptj5k0DuenEjUOr55B0L2BthcKPPxD5EhmcizM1q5dCApY9OSxZ4q7xltL3UZV4wPKSPcuO9yCN6llvsTX7kljOcSVewYcI8vSGT2jqr8O9DZvsnubKBFnJjfb09z9BmqYnejg2YIbjfaTkN0OMgzIfE+aGMRVoTOMmJdKyUxlqYJEDqBpVRIxONVtIs1X\\\\\\\/Q31lr9PmY8\\\\\\\/nrO6c\\\\\\\/VWdU\\\\\\\/d1S3qlNft+M7TNX0FMztpGKSRJ5dRXD0f0ltU0+lNTQYdgJkhbvJoXBA4TPiNmKRiF2Ountcq4XuYTCmM5tZbIQTs+5aMNakF2T+1bgjndhUngRRsT8A2PB\\\\\\\/SsQdWeQMycHNxYRWxIhUtSHqIFmIaTAjKFHKOyitOkEcyWTUoLPE9nvYxewA2H7CK6IQJrMrWxVUSc7wheVWVmwMmYo38bsskOXzZBw3mse+U2SlmkCgYKcAA81CoS56Jee9Fgu5OJvb4HheIkS92S94edFlNFwOHRWaxsqHQkuQH6REfFjb7iSIlbja2iEa1\\\\\\\/gl1+TLetrpVqrwVH5M2TGxQ6z2YvQE1JTpJeCpOBaG+KUl35fZ0IE4QfjdQrl6QdzZVDgOzksjxJODPxi2VzlZiXPHmUvP77W7LyI49CjqfQqfjJgFQ5lj4J0Lo\\\\\\\/MCKYpyA8P7WnqN6TT474=\\\u0022,\\\u0022iv\\\u0022:\\\u0022HKttutCj39Qaz4vmcW8Y0A==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002289eb7df2f6d0c8a75e9ee1f194d9102205056490cdc858d19a295c5ab6fc8b3d\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022vVYg226Yh8d8I83rx24j+FkwkxGV8vi1jceD0v4rFINqsaz00Kw1IYxeKeJNBQONblk9Cnz+pRHJuQAdD\\\\\\\/4w+h8fzWOYMnfWK1QeFVGIgSsKB5esb3LIdNUMyd2TmHokAqoskan9kEp\\\\\\\/Dh0wy5Azvja8rJIZs0M9xkgHpCai+YoKR0TNW1QAFIwLV2lzY83xKHunXDdwbc8ywIZsrOW3gCYxqIPND8TppV1I+eUSQ1WW5y78cGxfsz6rm71fo4puLiN+5xcA2U\\\\\\\/7\\\\\\\/J0v+qNGzT7sUa\\\\\\\/XDvOzpg3imbT9tijZm2ajAPAYwKQXF7Dwt0kQ7ZdWxpKMIyheP0atdFFCKKGsWyMwWsBVb25dsFjmV3icOyKT+3poEO3y5pkA97\\\\\\\/rO9uOAZkoxwv6lrYOUMf0Ji7Ggk9MlMyEMswGR6ynX0LOIhYBCZWxPiAKEwyXC90TYV07f\\\\\\\/O8xFn1QX6NISJHES8ZO6jHGGe2qiceYSG2yBtw5UGxgu1cLmY4c9XoKUbiXN0ZP2s1sXCmRQxzFCckOcpDuv637fYuYY3hGRS6gmiyo4uob4Em3kxLD1VQztrB6DWOv6rjVE\\\\\\\/LV4NWyzsi6SnuUBRB7\\\\\\\/pnzJrCaXZRs1TqEAw55hpSUduVBx2Qd3Vzrc6WWgAGG\\\\\\\/KRCBERIKiWhJuLGVpU5JCljbqYQILTUwkqWTWop1AtkOw4t3bWlD07jvVXJxD21iw5w8JMDFeUVZtdpZPsQurVL1b5qjwiHc3El+CX4xs0he0ff\\\\\\\/wFsNDyAYOhxFQLMURhtKWEzQ4c9zP\\\\\\\/TXcCAuA2pk0nzI34W4\\\\\\\/Xh4fdk1g6nwoGkwHcnskgSS1WwHmYsUKCbg6f9RYG5VJ\\\\\\\/xo5Xd2QRqtQjkWumiR5GI+6M+Sxxbo\\\\\\\/TWEwZ\\\\\\\/edahWazjXgYkHNw6TW7T0\\\\\\\/ACZ6mvCplponKe0bpdoLSe9haqvEbwNSGjlipPxkKIJ3kUOPctYowbGmziyY8V8+ijDV2Y6smMf8PtOoVpWbH6f5WpEHhgy4bZWLYJ5Aj6nsYRQwl+PFQectYCSjZKwgAEGrmMHf4w09A222vM21FHsVOEYxs2qC0aJ4+iXJZ8\\\\\\\/7PKPjNu4LKwSZr8jxRezobG06vSQqOk+Dc1qJdcVku6ogRSplvLHluFpnWfME8vCKV8PgpfkmBDtUbRfh3l5xP+kNOW1DnCx8sNskkYm0xT\\\\\\\/Cxs9hoTraDUqnEKzwNcE3uw+4Okv47r2udIryogsz3g4WqPjkOB1crG\\\\\\\/8fO9uynB+SZR2\\\\\\\/4Hhi8AhLbQewlhOdfCjSZ+y+Y8nK9r8a7PwclqGW5UvdTuX4ZHZyxKUiyYTICjrw3Xf6rqkiQWGY0UheKhjLeEgLWzEujTjlue7A1aBYoBU0ufBmENfRirRSwpE1nLXG4e0CxV4b3afwW5tSW1HL3Tg+S7GhxebTMXidsoJzT\\\\\\\/EBgzqpInM0ilolOV4NMfyBjkgkPRydXBHoAkY4+ySAV+11DqsbaFIGpo6mrfVJSoJvjPXHBDu91wnCn6DwXK9TYbCHmfvDd5qjgMetUryfic3vT9XlLGQmRWoQo+aCmdoMNvvzYud\\\\\\\/lGj2XL4lgiPvJPw1nUziUF\\\\\\\/HmxhDNt1UJ2nBYLUJCPd5FqogfFhqSxfWd\\\\\\\/oiq9+NWBBXG99Ixk3ScRbkSJMv8U8yFOsZLe4yPoYXOor2sBQt38goKngpTXObdtygVefzYqJqYoApXkjogKg1QzdPtNjS6Au8UMHD8zzphpQnYJOLCGfUpFbYDjE51TG85cXjRaK9zCLk7zeKRHigLCdeRmS6z+Q2095GavPHMeGKd2OSYpT71aqRqzVZ6kwhrMmHCXiVehmZHmU+x9Fp9Kqf1IuX+op0JrcBRamO8ckJaXfieW3+6QtZLnzHpr6HlkSKicE49ZsFzJxfoQeQI\\\\\\\/to4wllHzrLgsHe2J8SSr9mljpOqK11y0S1vfl\\\\\\\/DT7SDuFMwnSn4X70y5gzatc73HmGYWxMZfH0PZ2fOz115CgG1rdFuaVhL9vaWjoDJaMiSHJwSiTWFMpCdE5V4Mvw2I3AWwwfHby7gbhX3K\\\\\\\/FyLlkQEMBSVTwHAb6hFZjHHx6myxTP6jQVf35OtTE6Xbjuj+n4qmf+42QkKFRW3qzegSXhaaYqd6IWg==\\\u0022,\\\u0022iv\\\u0022:\\\u0022kgyXEOPBJaF\\\\\\\/XK923wSVTw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002252d5db3a2687d49edb103840281b519de88344d4ce1a574d1f165130eda899ee\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-486361 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/9/3203" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-68OrHjzih-IuP7HeCJ6kUFgjkKtv9aqKp6cTQGqeQ7s" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/9/3203" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Od1eWu3vbTcS5emxxfM__n5FR_wJ8Nd7PjDDL1mfFkU" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/9/3195" title="The Integrated Genomic Architecture and Evolution of Dental Divergence in East African Cichlid Fishes (Haplochromis chilotes x H. nyererei)" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/9/3217" title="Trans-ancestry Fine Mapping and Molecular Assays Identify Regulatory Variants at the ANGPTL8 HDL-C GWAS Locus" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="486361" id="node486361" data-pisa="ggg;7/9/3203" data-pisa-master="ggg;g3.117.300089" data-apath="/ggg/7/9/3203.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Functional Link Between Bir1 and the <em>Saccharomyces cerevisiae</em> Ctf19 Kinetochore Complex Revealed Through Quantitative Fitness Analysis</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-0668-6157" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Vasso Makrantoni</span>, <span class="highwire-citation-author" data-delta="1">Adam Ciesiolka</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-4186-8506" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Conor Lawless</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-9582-4242" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Josefin Fernius</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0002-3596-9407" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Adele Marston</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0003-2478-085X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>David Lydall</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0001-9086-191X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Michael J. R. Stark</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3203-3215;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300089 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node486361" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Vasso Makrantoni </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Centre for Gene Regulation and Expression, University of Dundee, DD1 5EH, UK</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Wellcome Trust Centre for Cell Biology, University of Edinburgh, EH9 3BF, UK</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Vasso%2BMakrantoni%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Makrantoni%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AVasso%2BMakrantoni%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-0668-6157" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Vasso Makrantoni</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Adam Ciesiolka </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Institute for Cell and Molecular Biosciences, Newcastle University, NE2 4HH, UK</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Adam%2BCiesiolka%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ciesiolka%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAdam%2BCiesiolka%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Conor Lawless </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Institute for Cell and Molecular Biosciences, Newcastle University, NE2 4HH, UK</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Conor%2BLawless%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lawless%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AConor%2BLawless%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-4186-8506" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Conor Lawless</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Josefin Fernius </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Wellcome Trust Centre for Cell Biology, University of Edinburgh, EH9 3BF, UK</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Josefin%2BFernius%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fernius%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJosefin%2BFernius%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-9582-4242" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Josefin Fernius</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Adele Marston </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Wellcome Trust Centre for Cell Biology, University of Edinburgh, EH9 3BF, UK</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Adele%2BMarston%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Marston%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAdele%2BMarston%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-3596-9407" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Adele Marston</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">David Lydall </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Institute for Cell and Molecular Biosciences, Newcastle University, NE2 4HH, UK</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BLydall%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lydall%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADavid%2BLydall%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-2478-085X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for David Lydall</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Michael J. R. Stark </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Centre for Gene Regulation and Expression, University of Dundee, DD1 5EH, UK</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BJ.%2BR.%2BStark%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Stark%20MJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMichael%2BJ.%2BR.%2BStark%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-9086-191X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Michael J. R. Stark</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:m.j.r.stark@dundee.ac.uk" class="" data-icon-position="" data-hide-link-title="0">m.j.r.stark@dundee.ac.uk</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/7/9/3203" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:486361" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:486361/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/9/3203.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:486361" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:486361/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/9/3203.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:486361" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:486361/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/9/3203.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:486361" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:486361/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/7/9/3203.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:486361" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:486361/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-5">The chromosomal passenger complex (CPC) is a key regulator of eukaryotic cell division, consisting of the protein kinase Aurora B/<a href="http://www.yeastgenome.org/locus/S000006130/overview">Ipl1</a> in association with its activator (INCENP/<a href="http://www.yeastgenome.org/locus/S000000360/overview">Sli15</a>) and two additional proteins (Survivin/<a href="http://www.yeastgenome.org/locus/S000003849/overview">Bir1</a> and Borealin/<a href="http://www.yeastgenome.org/locus/S000029704/overview">Nbl1</a>). Here, we report a genome-wide genetic interaction screen in <em>Saccharomyces cerevisiae</em> using the <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17</em> mutant, identifying through quantitative fitness analysis deletion mutations that act as enhancers and suppressors. Gene knockouts affecting the <a href="http://www.yeastgenome.org/locus/S000005939/overview">Ctf19</a> kinetochore complex were identified as the strongest enhancers of <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17</em>, while mutations affecting the large ribosomal subunit or the mRNA nonsense-mediated decay pathway caused strong phenotypic suppression. Thus, cells lacking a functional <a href="http://www.yeastgenome.org/locus/S000005939/overview">Ctf19</a> complex become highly dependent on <a href="http://www.yeastgenome.org/locus/S000003849/overview">Bir1</a> function and vice versa. The negative genetic interaction profiles of <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17</em> and the cohesin mutant <em><a href="http://www.yeastgenome.org/locus/S000002161/overview">mcd1</a>-1</em> showed considerable overlap, underlining the strong functional connection between sister chromatid cohesion and chromosome biorientation. Loss of some <a href="http://www.yeastgenome.org/locus/S000005939/overview">Ctf19</a> components, such as <a href="http://www.yeastgenome.org/locus/S000000311/overview">Iml3</a> or <a href="http://www.yeastgenome.org/locus/S000002662/overview">Chl4</a>, impacted differentially on <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17</em> compared with mutations affecting other CPC components: despite the synthetic lethality shown by either <em><a href="http://www.yeastgenome.org/locus/S000000311/overview">iml3</a></em>∆ or <em><a href="http://www.yeastgenome.org/locus/S000002662/overview">chl4</a></em>∆ in combination with <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17</em>, neither gene knockout showed any genetic interaction with either <em><a href="http://www.yeastgenome.org/locus/S000006130/overview">ipl1</a>-321</em> or <em><a href="http://www.yeastgenome.org/locus/S000000360/overview">sli15</a>-3</em>. Our data therefore imply a specific functional connection between the <a href="http://www.yeastgenome.org/locus/S000005939/overview">Ctf19</a> complex and <a href="http://www.yeastgenome.org/locus/S000003849/overview">Bir1</a> that is not shared with <a href="http://www.yeastgenome.org/locus/S000006130/overview">Ipl1</a>.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/bir1" class="hw-term hw-article-keyword hw-article-keyword-bir1" rel="nofollow">Bir1</a></li><li class="kwd"><a href="/keyword/chromosome-biorientation" class="hw-term hw-article-keyword hw-article-keyword-chromosome-biorientation" rel="nofollow">Chromosome biorientation</a></li><li class="kwd"><a href="/keyword/kinetochore" class="hw-term hw-article-keyword hw-article-keyword-kinetochore" rel="nofollow">Kinetochore</a></li><li class="kwd"><a href="/keyword/iml3-chl4-complex-0" class="hw-term hw-article-keyword hw-article-keyword-iml3-chl4-complex" rel="nofollow">Iml3-Chl4 complex</a></li><li class="kwd"><a href="/keyword/yeast" class="hw-term hw-article-keyword hw-article-keyword-yeast" rel="nofollow">yeast</a></li></ul><p id="p-6">To maintain genomic integrity, it is essential that every chromosome be faithfully transmitted to both progeny during cell division. Genomic instability is a characteristic of cancer cells, and chromosome number alterations (aneuploidy) caused by gain or loss of chromosomes are thought to be one of the driving forces behind tumor progression (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Hanahan and Weinberg 2011</a>). To help ensure accurate chromosome segregation, sister chromatids generated by DNA replication are held together by protein complexes termed cohesin. The sister kinetochores, multiprotein complexes assembled at sister centromeres to mediate their attachment to microtubules (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Lampert and Westermann 2011</a>; <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Santaguida and Musacchio 2009</a>), become linked to microtubules emanating from opposite spindle poles as they align on the mitotic spindle during metaphase (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Nasmyth and Haering 2009</a>). This state of attachment (chromosome biorientation) ensures that when cohesin is removed as cells enter anaphase, sister chromatids are pulled in opposite directions and each daughter receives exactly one copy of each chromosome (<a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Tanaka <em>et al.</em> 2005</a>).</p><p id="p-7">Aurora B protein kinase has emerged over the past 15 yr as a key regulator promoting chromosome biorientation (<a id="xref-ref-65-2" class="xref-bibr" href="#ref-65">Tanaka <em>et al.</em> 2005</a>). Although there is an intrinsic bias favoring bioriented attachment of sister chromatids to the mitotic spindle that is most readily seen when the spindle pole bodies (SPBs) have already separated (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Indjeian and Murray 2007</a>; <a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Verzijlbergen <em>et al.</em> 2014</a>), achievement of biorientation is not automatic and attachment errors occur that would lead to chromosome mis-segregation if they were left uncorrected. Aurora B kinase corrects such errors, promoting detachment of incorrect attachments through phosphorylation of proteins at the kinetochore, such that correct attachments have a chance to replace them (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Liu <em>et al.</em> 2009</a>; <a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Tanaka <em>et al.</em> 2002</a>). Aurora B/<a href="http://www.yeastgenome.org/locus/S000006130/overview">Ipl1</a> kinase forms part of the chromosomal passenger complex (CPC; see <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Ruchaud <em>et al.</em> 2007</a>) together with three other conserved proteins (yeast names in parentheses): INCENP (<a href="http://www.yeastgenome.org/locus/S000000360/overview">Sli15</a>), Survivin (<a href="http://www.yeastgenome.org/locus/S000003849/overview">Bir1</a>), and Borealin (<a href="http://www.yeastgenome.org/locus/S000029704/overview">Nbl1</a>). INCENP, Survivin, and Borealin associate via a triple helical interaction (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Jeyaprakash <em>et al.</em> 2007</a>) and INCENP contains a domain that binds to and activates Aurora B kinase (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Kang <em>et al.</em> 2001</a>). Error correction by <a href="http://www.yeastgenome.org/locus/S000006130/overview">Ipl1</a> kinase is an essential process and yeast cells show massive mis-segregation of chromosomes during division in its absence (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Biggins <em>et al.</em> 1999</a>). In addition to the CPC, efficient chromosome biorientation also requires the accumulation of cohesin around the centromere in yeast (pericentromeric cohesin), as well as pericentromeric condensin and the protein <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1</a>, which also interacts with this region of yeast chromosomes (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Marston 2015</a>). The contribution of pericentromeric cohesin, condensin, and <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1</a> may be to enforce a geometry that underlies the intrinsic bias toward chromosome biorientation (<a id="xref-ref-69-2" class="xref-bibr" href="#ref-69">Verzijlbergen <em>et al.</em> 2014</a>), while <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1</a> may be needed to sense when sister kinetochores are under tension from the mitotic spindle, thereby indicating that they are correctly bioriented (<a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Marston 2015</a>). The CPC-mediated error correction mechanism has generally been considered to involve inner centromere–localized Aurora B/<a href="http://www.yeastgenome.org/locus/S000006130/overview">Ipl1</a> (see <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Lampson and Cheeseman 2011</a>). In yeast, CPC interaction with the inner centromere is targeted by <a href="http://www.yeastgenome.org/locus/S000003849/overview">Bir1</a> through its interactions with <a href="http://www.yeastgenome.org/locus/S000003372/overview">Ndc10</a> (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Cho and Harrison 2012</a>; <a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">Yoon and Carbon 1999</a>) and with histone H2A phosphorylated on Ser-121 by <a href="http://www.yeastgenome.org/locus/S000003420/overview">Bub1</a> kinase (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Kawashima <em>et al.</em> 2010</a>). However, the importance of inner centromeric localization of the CPC has recently been called into question by the surprising finding that the <a href="http://www.yeastgenome.org/locus/S000006130/overview">Ipl1</a>-<a href="http://www.yeastgenome.org/locus/S000000360/overview">Sli15</a> complex in yeast can still provide error correction, even in the absence of <a href="http://www.yeastgenome.org/locus/S000003849/overview">Bir1</a> or <a href="http://www.yeastgenome.org/locus/S000029704/overview">Nbl1</a>, if it is delocalized from kinetochores by deletion of the first 228 residues of <a href="http://www.yeastgenome.org/locus/S000000360/overview">Sli15</a> that normally anchor <a href="http://www.yeastgenome.org/locus/S000006130/overview">Ipl1</a>-<a href="http://www.yeastgenome.org/locus/S000000360/overview">Sli15</a> to <a href="http://www.yeastgenome.org/locus/S000003849/overview">Bir1</a> and <a href="http://www.yeastgenome.org/locus/S000029704/overview">Nbl1</a> (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Campbell and Desai 2013</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Fink <em>et al.</em> 2017</a>; <a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Jeyaprakash <em>et al.</em> 2007</a>).</p><p id="p-8">We previously generated a temperature-sensitive allele (<em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17</em>) supporting normal proliferation and chromosome biorientation at 26°, but which fails to proliferate and shows a chromosome biorientation defect at 37° (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Makrantoni and Stark 2009</a>). <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17</em> contains 11 point alterations within the C-terminus half of the protein, seven of which are localized within the C-terminus 297 residues of <a href="http://www.yeastgenome.org/locus/S000003849/overview">Bir1</a> that can provide its essential function (<a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">Widlund <em>et al.</em> 2006</a>). Five point alterations are within the C-terminus 228 residues of <a href="http://www.yeastgenome.org/locus/S000003849/overview">Bir1</a> that interact strongly with both <a href="http://www.yeastgenome.org/locus/S000029704/overview">Nbl1</a> and <a href="http://www.yeastgenome.org/locus/S000000360/overview">Sli15</a> (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Nakajima <em>et al.</em> 2009</a>), and two lie within residues 889–941 that correspond to a domain proposed to form the triple helical interaction that is conserved in the human CPC (<a id="xref-ref-32-3" class="xref-bibr" href="#ref-32">Jeyaprakash <em>et al.</em> 2007</a>; <a id="xref-ref-50-2" class="xref-bibr" href="#ref-50">Nakajima <em>et al.</em> 2009</a>). One of these (L924S) affects a hydrophobic residue that is directly involved in the triple helical interaction, and is also mutated in two other conditional <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a></em> alleles (<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Shimogawa <em>et al.</em> 2009</a>). Thus <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17</em> is likely to affect the interaction of the mutant protein with the other CPC components, although we have not examined this directly. Some of the point mutations are also located within a region of <a href="http://www.yeastgenome.org/locus/S000003849/overview">Bir1</a> that is known to interact with <a href="http://www.yeastgenome.org/locus/S000003372/overview">Ndc10</a> (<a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Thomas and Kaplan 2007</a>).</p><p id="p-9">To understand better the role of <a href="http://www.yeastgenome.org/locus/S000003849/overview">Bir1</a> and the proteins and processes with which it interacts, we carried out a genome-wide synthetic interaction screen using the <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17</em> mutant. We found that the <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17</em> mutant is strongly enhanced by mutations affecting components of the <a href="http://www.yeastgenome.org/locus/S000005939/overview">Ctf19</a> kinetochore complex, including <a href="http://www.yeastgenome.org/locus/S000002662/overview">Chl4</a> and <a href="http://www.yeastgenome.org/locus/S000000311/overview">Iml3</a>, and in the W303 background both <em><a href="http://www.yeastgenome.org/locus/S000002662/overview">chl4</a></em>∆ and <em><a href="http://www.yeastgenome.org/locus/S000000311/overview">iml3</a></em>∆ are synthetic lethal with <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17</em>. Surprisingly, the synthetic lethal interactions between <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17</em> and either <em><a href="http://www.yeastgenome.org/locus/S000002662/overview">chl4</a></em>∆ or <em><a href="http://www.yeastgenome.org/locus/S000000311/overview">iml3</a></em>∆ are specific to <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17</em> and were not seen with either <em><a href="http://www.yeastgenome.org/locus/S000000360/overview">sli15</a>-3</em> or <em><a href="http://www.yeastgenome.org/locus/S000006130/overview">ipl1</a>-321</em>, which each confer a much stronger Ts<sup>−</sup> phenotype than <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17</em> (<a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Makrantoni and Stark 2009</a>). The yeast <a href="http://www.yeastgenome.org/locus/S000005939/overview">Ctf19</a> complex is a group of inner kinetochore proteins that are analogous to the CCAN complex of metazoan kinetochores (<a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Lampert and Westermann 2011</a>; <a id="xref-ref-60-2" class="xref-bibr" href="#ref-60">Santaguida and Musacchio 2009</a>). Most yeast <a href="http://www.yeastgenome.org/locus/S000005939/overview">Ctf19</a> complex components are nonessential for proliferation, although gene knockouts confer elevated chromosome mis-segregation and reduced association of cohesin, condensin, <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1</a>, and <a href="http://www.yeastgenome.org/locus/S000006130/overview">Ipl1</a> with centromere-proximal (pericentromeric) chromatin (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Kiburz <em>et al.</em> 2005</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Fernius and Marston 2009</a>; <a id="xref-ref-69-3" class="xref-bibr" href="#ref-69">Verzijlbergen <em>et al.</em> 2014</a>). Our data therefore imply a specific functional connection between <a href="http://www.yeastgenome.org/locus/S000003849/overview">Bir1</a> and these two <a href="http://www.yeastgenome.org/locus/S000005939/overview">Ctf19</a> complex components that is critical for CPC function, and supports the notion that the <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17</em> mutation affects CPC function in a fundamentally different way to <em><a href="http://www.yeastgenome.org/locus/S000006130/overview">ipl1</a></em> mutations that simply reduce its ability to phosphorylate its targets. Our findings are consistent with the notion that delocalization of the CPC from kinetochores may make cells more dependent on mechanisms involving the <a href="http://www.yeastgenome.org/locus/S000005939/overview">Ctf19</a> complex that impart the intrinsic bias toward biorientation.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Yeast strains and general methods</h3><p id="p-10">Basic yeast methods, growth media, and routine recombinant DNA methodology were performed as previously described (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Amberg <em>et al.</em> 2005</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Gietz <em>et al.</em> 1992</a>). Unless stated otherwise, all yeast strains used in this study (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>) are derivatives of W303-1a (<a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Thomas and Rothstein 1989</a>) and have the following markers: <em><a href="http://www.yeastgenome.org/locus/S000005654/overview">ade2</a>-1 <a href="http://www.yeastgenome.org/locus/S000005728/overview">his3</a>-11</em>, <em>15 <a href="http://www.yeastgenome.org/locus/S000000523/overview">leu2</a>-3</em>, <em>112 <a href="http://www.yeastgenome.org/locus/S000002414/overview">trp1</a>-1 <a href="http://www.yeastgenome.org/locus/S000000747/overview">ura3</a>-1 <a href="http://www.yeastgenome.org/locus/S000000789/overview">can1</a>-100 <a href="http://www.yeastgenome.org/locus/S000002701/overview">ssd1</a>-d2</em> Gal<sup>+</sup>. However, synthetic interactions screening was performed as previously described, using the BY strain background for reasons of strain compatibility with the genome-wide gene knockout collection (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Addinall <em>et al.</em> 2011</a>). To verify genetic interactions detected in the BY background, deletion strains were made W303 background by using the pFA6a-HIS3MX6 cassette as previously described (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Longtine <em>et al.</em> 1998</a>), and then crossed with <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17</em> in the same background. Deletion of <em><a href="http://www.yeastgenome.org/locus/S000005938/overview">IRC15</a></em> was performed such that the last eight codons of <em><a href="http://www.yeastgenome.org/locus/S000005939/overview">CTF19</a></em> (which overlap with <em><a href="http://www.yeastgenome.org/locus/S000005938/overview">IRC15</a></em>) were retained.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/486597/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/486597/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Yeast strains</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-3" class="subsection"><h3>Chromatin immunoprecipitation</h3><p id="p-13">Cohesin association with centromeric, pericentromeric, and arm sequences from chromosome IV was assessed in strains expressing HA-tagged <a href="http://www.yeastgenome.org/locus/S000002161/overview">Mcd1</a> using chromatin immunoprecipitation with anti-HA antibody (clone 12CA5) followed by qPCR analysis, performed as previously described (<a id="xref-ref-17-5" class="xref-bibr" href="#ref-17">Fernius and Marston 2009</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Fernius <em>et al.</em> 2013</a>), using a Roche LightCycler and Express SYBR Green reagent (Invitrogen). PCR primers are listed in Supplemental Material <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300089/-/DC1/TableS1.pdf">Table S1</a>.</p></div><div id="sec-4" class="subsection"><h3>Quantitative fitness analysis using bir1-17</h3><p id="p-14">To generate a <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17</em> strain in the S288C background suitable for synthetic gene array (SGA) screening, Y1082 was first transformed with a PCR fragment amplified from pFA6a-HpHMX6 (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Hentges <em>et al.</em> 2005</a>) using primers #536 and #537 (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300089/-/DC1/TableS1.pdf">Table S1</a>), such that the <em>Hph</em> marker replaced the region 236–297 bp upstream of the <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">BIR1</a></em> open reading frame (ORF), generating VMY165. This strain was next transformed with a PCR fragment made using primers #208 and #575 (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300089/-/DC1/TableS1.pdf">Table S1</a>) to amplify the <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17</em>::<em>NatMX</em> construct from VMY26, in which the region 49-71 bases downstream of the <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17</em> ORF was replaced by the <em>NatMX</em> marker from pAG25 (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Goldstein and McCusker 1999</a>). This generated VMY179, from which the <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17</em> region was amplified and sequenced to verify presence of all of the base changes in <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17</em> (<a id="xref-ref-41-3" class="xref-bibr" href="#ref-41">Makrantoni and Stark 2009</a>). <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300089/-/DC1/FigureS1.jpeg">Figure S1</a> shows the doubly marked <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17</em> locus generated. Flanking <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17</em> with two different markers and then selecting for both during SGA greatly reduced the possibility that <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17</em> could be separated from its markers by recombination. VMY179 grew normally at temperatures below 37° but, while still clearly temperature-sensitive, showed some growth at 37°, particularly when arrayed by pinning.</p><p id="p-15">SGA analysis by crossing VMY179 to the systematic yeast gene deletion collection was performed as already described (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Addinall <em>et al.</em> 2008</a>, <a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">2011</a>). Quantitative fitness analysis (QFA) of the double mutants to generate strain fitnesses and genetic interaction strengths (GIS) was performed as previously reported (<a id="xref-ref-2-3" class="xref-bibr" href="#ref-2">Addinall <em>et al.</em> 2011</a>), by screening at 20, 27, and 37°. Fitness and GIS calculations, including <em>t</em>-tests for the significance of GIS, were carried out and plots were generated using the QFA R package (version 0.0-43; <a href="http://qfa.r-forge.r-project.org/">http://qfa.r-forge.r-project.org/</a>). Four replicate <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17</em> strain (VMY179) crosses and eight replicate control strain DLY4242 (<em><a href="http://www.yeastgenome.org/locus/S000000747/overview">ura3</a></em>::<em>NatMX</em>) crosses were analyzed. QFA data are summarized in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300089/-/DC1/FileS2.xlsx">Tables S2–S4</a> in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300089/-/DC1/FileS2.xlsx">File S2</a>, following removal of all genes tightly linked to the query <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17</em> mutation (<em>i.e.</em>, located within 20 kb of <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17</em> on chromosome X) and a standard set of genes related to the genetic selections used in SGA that are therefore incompatible with SGA (<em><a href="http://www.yeastgenome.org/locus/S000002580/overview">ARG82</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000000871/overview">ARG5,6</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000001060/overview">ARG4</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000003607/overview">ARG2</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000003624/overview">ARG3</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000004565/overview">ARG81</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000004645/overview">ARG80</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000004666/overview">ARG7</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000005419/overview">ARG1</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000005500/overview">ARG8</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000000452/overview">HIS7</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000000535/overview">HIS4</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000001921/overview">HIS2</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000000857/overview">HIS1</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000001282/overview">HIS6</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000001378/overview">HIS5</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000000523/overview">LEU2</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000002977/overview">LEU1</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000001044/overview">LEU5</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000004443/overview">LEU3</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000005048/overview">LEU4</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000005634/overview">LEU9</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000000319/overview">LYS2</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000002289/overview">LYS21</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000002341/overview">LYS20</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000002441/overview">LYS14</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000002642/overview">LYS4</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000003122/overview">LYS5</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000001356/overview">LYS12</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000001473/overview">LYS1</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000005333/overview">LYS9</a></em>, and <em><a href="http://www.yeastgenome.org/locus/S000004641/overview">CCS1</a></em>). To identify potentially significant phenotypic enhancers and suppressors, double mutants with <em>q</em>-value [false discovery rate (FDR) corrected <em>p</em>-value ≤ 0.05] and either a negative GIS (enhancers) or positive GIS (suppressors) were selected (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300089/-/DC1/FileS2.xlsx">Tables S5–S8</a> in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300089/-/DC1/FileS2.xlsx">File S2</a>). GO terms enriched in the enhancer and suppressor gene subsets were determined using the GO Term Finder (version 0.83; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Boyle <em>et al.</em> 2004</a>). as implemented by the <em>Saccharomyces</em> Genome Database (SGD; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Cherry <em>et al.</em> 2012</a>, queried December 2016) with a <em>p</em>-value cut-off of ≤0.01. All recognized <em>Saccharomyces cerevisiae</em> nuclear-encoded ORFs within the systematic deletion collection, but lacking the <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17</em>–linked genes and the SGA-incompatible genes listed above, were used as the background set for determining GO term enrichment (4235 genes; the full list is shown in each of <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300089/-/DC1/FileS2.xlsx">Table S2</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300089/-/DC1/FileS2.xlsx">Table S3</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300089/-/DC1/FileS2.xlsx">Table S4</a> in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300089/-/DC1/FileS2.xlsx">File S2</a>). For further analysis of strong negative genetic interactors (GIS ≤ −25) identified at 20 or 27°, growth of the individual control and <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17</em> double mutants was examined. Where at least three out of four <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17 yfg</em>∆ replicates failed to grow but at least six out of eight control <em><a href="http://www.yeastgenome.org/locus/S000000747/overview">ura3</a></em>::<em>NatMX yfg</em>∆ replicates grew, then the high negative GIS was considered to represent a synthetic lethal interaction (<em>yfg</em>∆: your favorite gene deletion; used to indicate one of the ∼4200 viable yeast gene deletions from the systematic deletion collection). Fitness plots were generated from the QFA data using iRVis (<a href="http://qfa.r-forge.r-project.org/visTool/">http://qfa.r-forge.r-project.org/visTool/</a>), which is a part of the QFA software package, with significant negative and positive genetic interactors (<em>i.e.</em>, <em>q</em>-values of ≤ 0.05 defined by <em>t</em>-test) colored blue and red, respectively.</p></div><div id="sec-5" class="subsection"><h3>Data availability</h3><p id="p-16">Yeast strains are available on request. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300089/-/DC1/FileS1.docx">File S1</a> contains detailed descriptions of all supplemental files. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300089/-/DC1/FileS2.xlsx">File S2</a> contains <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300089/-/DC1/FileS2.xlsx">Tables S2–S8</a>. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300089/-/DC1/FileS3.xlsx">File S3</a> summarizes GO analysis of <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-1</em>7 enhancers and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300089/-/DC1/FileS4.xlsx">File S4</a> summarizes GO analysis of <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17</em> suppressors. The authors state that all data necessary for confirming the conclusions presented in the article are represented fully within the article and the supplemental material.</p></div></div><div class="section results" id="sec-6"><h2 class="">Results</h2><div id="sec-7" class="subsection"><h3>QFA identifies gene deletions that interact with bir1-17</h3><p id="p-17">To identify enhancers and suppressors of <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17</em> that might indicate specific functional interactions, we used SGA technology to cross <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17</em> to the collection of ∼4200 viable systematic gene deletion strains, followed by QFA (<a id="xref-ref-2-4" class="xref-bibr" href="#ref-2">Addinall <em>et al.</em> 2011</a>) to identify suppressing or enhancing genetic interactions. <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17</em> was originally isolated in the W303 genetic background and confers a recessive, temperature-sensitive growth defect that is clearly evident at 37° (<a id="xref-ref-41-4" class="xref-bibr" href="#ref-41">Makrantoni and Stark 2009</a>). To investigate the best approach for identifying <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17</em> suppressors and enhancers, we first compared growth of the S288C background control and <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17</em> strains after arraying by either pinning or spotting. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300089/-/DC1/FigureS2.jpeg">Figure S2</a> shows that when strains were arrayed by pinning, the fitness defect of <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17</em> was hard to detect even at higher incubation temperatures (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300089/-/DC1/FigureS2.jpeg">Figure S2</a>A). In contrast, when dilutions of the two strains were arrayed by spotting onto the screening plates then the fitness defect of <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17</em> was readily detectable at 37° (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300089/-/DC1/FigureS2.jpeg">Figure S2</a>B). We therefore carried out QFA analysis by spotting rather than pinning the arrays of control and <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17</em> double mutants generated by SGA.</p><p id="p-18">Double <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17 yfg</em>∆ mutants were generated at 23°. QFA was subsequently performed by spotting out and screening growth at 20, 27, and 37°, calculating the GIS and <em>q</em>-value (FDR-corrected <em>p</em>-value) for each double mutant to indicate the magnitude of the genetic interaction between <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17</em> and each <em>yfg</em>∆ gene knockout and its statistical significance, respectively (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300089/-/DC1/FileS2.xlsx">Tables S2–S4</a> in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300089/-/DC1/FileS2.xlsx">File S2</a>; data ranked in ascending order of GIS, starting with the most negative interactions). Since <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17</em> cells spotted at 37° show a clear growth defect, we chose to focus primarily on this dataset. A <em>q</em>-value threshold of ≤0.05 was applied to identify a subset of statistically significant genetic interactions at each screening temperature. QFA data were summarized in the form of fitness plots, in which the fitness of each <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17 yfg</em>∆ strain is plotted against the fitness of the corresponding control <em><a href="http://www.yeastgenome.org/locus/S000000747/overview">ura3</a></em>∆ <em>yfg</em>∆ strain.</p><p id="p-19"><a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a> shows the fitness plot for <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17 yfg</em>∆ mutants screened at 37°, indicating all statistically significant enhancers (negative GIS; blue triangles) and suppressors (positive GIS; red triangles). <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300089/-/DC1/FileS2.xlsx">Table S5</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300089/-/DC1/FileS2.xlsx">Table S8</a> in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300089/-/DC1/FileS2.xlsx">File S2</a> list the statistically significant <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17</em> enhancers and suppressors, respectively, identified by screening at 37°, while <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300089/-/DC1/FileS2.xlsx">Table S6</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300089/-/DC1/FileS2.xlsx">Table S7</a> in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300089/-/DC1/FileS2.xlsx">File S2</a> present the statistically significant enhancers at 20 and 27° for comparison. In the fitness plot, the dashed gray line indicates where points should lie when deletion mutants show identical fitness in combination with either <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17</em> or the control <em><a href="http://www.yeastgenome.org/locus/S000000747/overview">ura3</a></em>∆ mutation (the line of equal growth). The regression line of the actual data points is indicated by the solid line. Downward displacement of this regression line away from the line of equal growth (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>) is consistent with the temperature-sensitive phenotype of <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17</em> that was clearly evident following spotting out under QFA conditions (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300089/-/DC1/FigureS2.jpeg">Figure S2</a>), and is in contrast to the fitness plots from the 20 and 27° screens (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300089/-/DC1/FigureS3.jpeg">Figure S3</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/9/3203/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Fitness plot of bir1-17 double mutants at 37&#xB0;. Following four replicate crosses of bir1-17 with the yeast genome knockout collection, quantitative fitness analysis of each bir1-17 yfg&#x2206; (&#x201C;your favorite gene deletion&#x201D;) strain was carried out at 37&#xB0; and mean fitness plotted against the mean fitness observed from eight replicates of a control cross between a ura3&#x2206; strain and the knockout collection. Gene deletions that significantly enhanced (blue triangles) or suppressed (red triangles) the growth defect of a bir1-17 strain are indicated, with all other nonsignificant deletions indicated as gray circles. A significant interaction was defined as one with a q-value (FDR-corrected p-value; see Addinall et al. 2011) &#x2264; 0.05, with enhancers having a negative genetic interaction strength (GIS) and suppressors having a positive GIS. The line of equal growth (gray dashed) and a population model of expected fitness under the assumption of genetic independence (solid gray; a regression line based on all the data points) are also indicated. The blue lines show the average position of his3&#x2206; strains as a proxy for wild-type growth." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-401112292" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Fitness plot of &lt;em&gt;bir1-17&lt;/em&gt; double mutants at 37&#xB0;. Following four replicate crosses of &lt;em&gt;bir1-17&lt;/em&gt; with the yeast genome knockout collection, quantitative fitness analysis of each &lt;em&gt;bir1-17 yfg&lt;/em&gt;&#x2206; (&#x201C;your favorite gene deletion&#x201D;) strain was carried out at 37&#xB0; and mean fitness plotted against the mean fitness observed from eight replicates of a control cross between a &lt;em&gt;ura3&lt;/em&gt;&#x2206; strain and the knockout collection. Gene deletions that significantly enhanced (blue triangles) or suppressed (red triangles) the growth defect of a &lt;em&gt;bir1-17&lt;/em&gt; strain are indicated, with all other nonsignificant deletions indicated as gray circles. A significant interaction was defined as one with a &lt;em&gt;q&lt;/em&gt;-value (FDR-corrected &lt;em&gt;p&lt;/em&gt;-value; see Addinall &lt;em&gt;et al.&lt;/em&gt; 2011) &#x2264; 0.05, with enhancers having a negative genetic interaction strength (GIS) and suppressors having a positive GIS. The line of equal growth (gray dashed) and a population model of expected fitness under the assumption of genetic independence (solid gray; a regression line based on all the data points) are also indicated. The blue lines show the average position of &lt;em&gt;his3&lt;/em&gt;&#x2206; strains as a proxy for wild-type growth.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/9/3203/F1.medium.gif" width="440" height="406"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/7/9/3203/F1.medium.gif" width="440" height="406"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/9/3203/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/9/3203/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/486505" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-20" class="first-child">Fitness plot of <em>bir1-17</em> double mutants at 37°. Following four replicate crosses of <em>bir1-17</em> with the yeast genome knockout collection, quantitative fitness analysis of each <em>bir1-17 yfg</em>∆ (“your favorite gene deletion”) strain was carried out at 37° and mean fitness plotted against the mean fitness observed from eight replicates of a control cross between a <em>ura3</em>∆ strain and the knockout collection. Gene deletions that significantly enhanced (blue triangles) or suppressed (red triangles) the growth defect of a <em>bir1-17</em> strain are indicated, with all other nonsignificant deletions indicated as gray circles. A significant interaction was defined as one with a <em>q</em>-value (FDR-corrected <em>p</em>-value; see <a id="xref-ref-2-5" class="xref-bibr" href="#ref-2">Addinall <em>et al.</em> 2011</a>) ≤ 0.05, with enhancers having a negative genetic interaction strength (GIS) and suppressors having a positive GIS. The line of equal growth (gray dashed) and a population model of expected fitness under the assumption of genetic independence (solid gray; a regression line based on all the data points) are also indicated. The blue lines show the average position of <em>his3</em>∆ strains as a proxy for wild-type growth.</p><div class="sb-div caption-clear"></div></div></div><p id="p-21">To look in an unbiased manner for relationships between genes identified as statistically significant enhancers and suppressors of <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17</em> at 37°, we searched for GO terms within the process, function, and component ontologies that showed significant enrichment in the enhancers and suppressors. We focused primarily on strong interactions (|GIS| ≥ 25; mapped onto the 37° fitness plot in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300089/-/DC1/FigureS4.jpeg">Figure S4</a>), but also searched using enhancers and suppressors with |GIS| ≥ 10 for comparison. These data are summarized in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300089/-/DC1/FileS3.xlsx">File S3</a> (enhancers) and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300089/-/DC1/FileS4.xlsx">File S4</a> (suppressors). We also manually examined the position on the 37° fitness plot of the members of each of the core protein complexes defined by <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Benschop <em>et al.</em> (2010)</a>, as a means of identifying consistent patterns of genetic interaction with <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17</em> that might reflect functional interactions between <a href="http://www.yeastgenome.org/locus/S000003849/overview">Bir1</a> and specific cellular processes. <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a> summarizes the proportion of <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17</em> enhancers or suppressors identified at each screening temperature. Around 3% of <em>yfg</em>∆ knockouts strongly enhanced <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17</em> in the 37° screen (GIS ≤ −25), while 2.2% of knockouts strongly suppressed (GIS ≥ 25). This was in contrast to the 20 and 27° screens, where the proportion of strong interactions was far less.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/486519/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/486519/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Summary of <em>bir1-17</em> suppressors and enhancers</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-8" class="subsection"><h3>Phenotypic enhancers of bir1-17</h3><p id="p-23">It was quickly apparent from looking at the <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17</em> enhancers identified at 37° (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300089/-/DC1/FileS2.xlsx">Table S5</a> in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300089/-/DC1/FileS2.xlsx">File S2</a>) that knockouts of almost any of the nonessential components of the <a href="http://www.yeastgenome.org/locus/S000005939/overview">Ctf19</a> kinetochore complex (<em><a href="http://www.yeastgenome.org/locus/S000002662/overview">CHL4</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000004373/overview">CTF3</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000005939/overview">CTF19</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000000311/overview">IML3</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000005938/overview">IRC15</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000006250/overview">MCM16</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000002726/overview">MCM21</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000003896/overview">MCM22</a></em>, and <em><a href="http://www.yeastgenome.org/locus/S000002791/overview">NKP1</a></em>; see <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Biggins 2013</a>) strongly enhanced <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17</em>; only <em><a href="http://www.yeastgenome.org/locus/S000004307/overview">NKP2</a></em> was not identified as an enhancer. <em><a href="http://www.yeastgenome.org/locus/S000005938/overview">IRC15</a></em> is included here because of the overlap of its full-length deletion with <em><a href="http://www.yeastgenome.org/locus/S000005939/overview">CTF19</a></em>, as discussed below. All of these knockouts except <em><a href="http://www.yeastgenome.org/locus/S000002791/overview">nkp1</a></em>∆ fell into the strong enhancer category (<em>i.e.</em>, GIS ≤ −25), and five out of the six strongest enhancers were members of the <a href="http://www.yeastgenome.org/locus/S000005939/overview">Ctf19</a> complex (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300089/-/DC1/FileS2.xlsx">Table S5</a> in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300089/-/DC1/FileS2.xlsx">File S2</a>). Only <em><a href="http://www.yeastgenome.org/locus/S000002726/overview">MCM21</a></em> and <em><a href="http://www.yeastgenome.org/locus/S000005939/overview">CTF19</a></em>, encoding the two nonessential members of the COMA subcomplex (<a href="http://www.yeastgenome.org/locus/S000000415/overview">Ame1</a> and <a href="http://www.yeastgenome.org/locus/S000003411/overview">Okp1</a> are both essential; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">De Wulf <em>et al.</em> 2003</a>; <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Ortiz <em>et al.</em> 1999</a>), were among the strongest enhancer of <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17</em> at all three screening temperatures. The other members of the <a href="http://www.yeastgenome.org/locus/S000005939/overview">Ctf19</a> complex were either weaker enhancers at 20° and 27° than at 37°, or in some cases, not identified as significant enhancers at all. It is striking that in contrast to the two lower temperature screens, the 37° screen placed knockouts of genes encoding all the <a href="http://www.yeastgenome.org/locus/S000005939/overview">Ctf19</a> complex components except <em><a href="http://www.yeastgenome.org/locus/S000002791/overview">NKP1</a></em> and <em><a href="http://www.yeastgenome.org/locus/S000004307/overview">NKP2</a></em> in a tight cluster at the lower right of the plot, indicating that the knockouts had little or no defect in the control (<em><a href="http://www.yeastgenome.org/locus/S000000747/overview">ura3</a></em>∆) background, but in contrast had a very strong defect when combined with <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17</em> (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300089/-/DC1/FigureS5.jpeg">Figure S5</a>). Thus, screening at 37° when double mutants are close to the <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17</em> maximum permissive temperature greatly assisted in identifying genetic interactions between the core components of the <a href="http://www.yeastgenome.org/locus/S000005939/overview">Ctf19</a> complex and <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17</em>.</p><p id="p-24">GO analysis of the strong <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17</em> enhancers (GIS ≤ −25) identified at 37° (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300089/-/DC1/FileS2.xlsx">Table S5</a> in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300089/-/DC1/FileS2.xlsx">File S2</a>) highlighted a number of specific GO terms concerned with chromosome segregation, sister chromatid cohesion, and microtubule-based processes, as well as more general terms including nuclear division and chromosome organization that encompassed many of the strongest negative genetic interactions (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300089/-/DC1/FileS3.xlsx">File S3</a>). In addition to the <a href="http://www.yeastgenome.org/locus/S000005939/overview">Ctf19</a> complex, this analysis highlighted genes encoding all the <em>S. cerevisiae</em> kinesin-like proteins except <em><a href="http://www.yeastgenome.org/locus/S000000159/overview">KIP1</a></em> (<em>i.e.</em>, <em><a href="http://www.yeastgenome.org/locus/S000006076/overview">KIP2</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000003184/overview">KIP3</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000006345/overview">KAR3</a></em>, and <em><a href="http://www.yeastgenome.org/locus/S000000787/overview">CIN8</a></em>), <em><a href="http://www.yeastgenome.org/locus/S000006174/overview">VIK1</a></em> and <em><a href="http://www.yeastgenome.org/locus/S000004811/overview">CIK1</a></em> (encoding <a href="http://www.yeastgenome.org/locus/S000006345/overview">Kar3</a>-associated proteins; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Manning <em>et al.</em> 1999</a>), <em><a href="http://www.yeastgenome.org/locus/S000000534/overview">BIK1</a></em> (a plus-end tracking protein related to CLIP-170; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Berlin <em>et al.</em> 1990</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Blake-Hodek <em>et al.</em> 2010</a>), <em><a href="http://www.yeastgenome.org/locus/S000004593/overview">TUB3</a></em> (α-tubulin), <em><a href="http://www.yeastgenome.org/locus/S000005876/overview">CIN1</a></em> (encoding a β-tubulin folding factor; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Hoyt <em>et al.</em> 1990</a>, <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">1997</a>) and <em><a href="http://www.yeastgenome.org/locus/S000006190/overview">KAR9</a></em> (encoding a spindle positioning factor; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Beach <em>et al.</em> 2000</a>; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Miller <em>et al.</em> 2000</a>), although <em><a href="http://www.yeastgenome.org/locus/S000006190/overview">kar9</a></em>∆ was inviable at 20 and 37° but was synthetic lethal with <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17</em> at 27°. <em><a href="http://www.yeastgenome.org/locus/S000000818/overview">bim1</a></em>∆, removing another plus-end tracking protein that binds to <a href="http://www.yeastgenome.org/locus/S000006190/overview">Kar9</a> and is also involved in spindle positioning (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Beach <em>et al.</em> 2000</a>; <a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">Miller <em>et al.</em> 2000</a>), was also clearly a strong <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17</em> enhancer at the two lower temperatures, although it fell outside our statistical significance cut-off at 37°. Thus cells lacking proper <a href="http://www.yeastgenome.org/locus/S000003849/overview">Bir1</a> function appear to become particularly dependent on the normal functioning of kinesins and microtubules.</p><p id="p-25">GO analysis, together with consideration of the core protein complexes defined by <a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Benschop <em>et al.</em> (2010)</a>, also identified knockouts of genes encoding any of the three members of the <a href="http://www.yeastgenome.org/locus/S000001234/overview">Ctf8</a>/<a href="http://www.yeastgenome.org/locus/S000004683/overview">Ctf18</a>/<a href="http://www.yeastgenome.org/locus/S000000521/overview">Dcc1</a> complex that is related to replication factor C (RFC<sup><a href="http://www.yeastgenome.org/locus/S000004683/overview">Ctf18</a></sup>) as enhancers of <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17</em> (although <em><a href="http://www.yeastgenome.org/locus/S000004683/overview">ctf18</a></em>∆ fell just outside our cut-offs of <em>q</em> ≤ 0.05 and GIS ≤ −25 for significant strong enhancers). All three members of the <a href="http://www.yeastgenome.org/locus/S000005217/overview">Tof1</a>/<a href="http://www.yeastgenome.org/locus/S000000566/overview">Mrc1</a>/<a href="http://www.yeastgenome.org/locus/S000004651/overview">Csm3</a> complex that acts at stalled replication forks to promote sister chromatid cohesion (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bando <em>et al.</em> 2009</a>; <a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Tourriere <em>et al.</em> 2005</a>; <a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">Xu <em>et al.</em> 2004</a>) were also highlighted (although <em><a href="http://www.yeastgenome.org/locus/S000005217/overview">tof1</a></em>∆ fell just outside our <em>q</em>-value cut-off). Like the <a href="http://www.yeastgenome.org/locus/S000005217/overview">Tof1</a> complex, the <a href="http://www.yeastgenome.org/locus/S000001234/overview">Ctf8</a> complex is also involved in sister chromatid cohesion (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Mayer <em>et al.</em> 2001</a>, <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">2004</a>), highlighting the link between this process and CPC-dependent error correction. Examination of the core protein complexes also revealed negative enhancement of <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17</em> by those viable knockouts in the systematic collection that affected RFC<sup><a href="http://www.yeastgenome.org/locus/S000000975/overview">Rad24</a></sup> (<em><a href="http://www.yeastgenome.org/locus/S000000975/overview">RAD24</a></em>), RFC<sup><a href="http://www.yeastgenome.org/locus/S000005670/overview">Elg1</a></sup> (<em><a href="http://www.yeastgenome.org/locus/S000005670/overview">ELG1</a></em>), a PCNA-like clamp (<em><a href="http://www.yeastgenome.org/locus/S000005895/overview">RAD17</a></em> and possibly <em><a href="http://www.yeastgenome.org/locus/S000006115/overview">DDC1</a></em>) and DNA polymerase epsilon [<em><a href="http://www.yeastgenome.org/locus/S000000482/overview">DPB3</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000000481/overview">YBR277c</a></em> (::<em><a href="http://www.yeastgenome.org/locus/S000000482/overview">DPB3</a></em>), and <em><a href="http://www.yeastgenome.org/locus/S000002528/overview">DPB4</a></em> if the <em>q</em>-value cut-off is relaxed].</p><p id="p-26">Broadening the GO analysis to consider enhancers with a GIS ≤ −10 did not change this overall picture but gave greater emphasis to some categories such as genes involved in chromatin modification (components of the Set3C, ISW, Compass, Ada, and Rpd3S complexes), histone exchange (<em><a href="http://www.yeastgenome.org/locus/S000000435/overview">SWC5</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000004505/overview">VPS71</a></em>, and <em><a href="http://www.yeastgenome.org/locus/S000002893/overview">VPS72</a></em>), tRNA wobble uridine modification (components of Elongator, <em><a href="http://www.yeastgenome.org/locus/S000000018/overview">ATS1</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000001593/overview">KTI12</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000001270/overview">URM1</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000001153/overview">UBA4</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000005063/overview">NCS2</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000003179/overview">NCS6</a></em>, and <a href="http://www.yeastgenome.org/locus/S000001736/overview">SAP190</a>), iron transport (<em><a href="http://www.yeastgenome.org/locus/S000004662/overview">FET3</a></em> and <em><a href="http://www.yeastgenome.org/locus/S000000947/overview">FTR1</a></em>), and peroxisomal function. Multiple components in each of these categories were strong, significant enhancers of <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17</em> at 37° (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300089/-/DC1/FileS2.xlsx">Table S5</a> in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300089/-/DC1/FileS2.xlsx">File S2</a>). While a potential functional connection between <a href="http://www.yeastgenome.org/locus/S000003849/overview">Bir1</a> and either iron transport or peroxisomes is not at all obvious, many of these other enhancers may function indirectly by altering the pattern of expression of proteins that have a direct functional relationship with <a href="http://www.yeastgenome.org/locus/S000003849/overview">Bir1</a>. Other notable strong enhancers not falling into any of the above-mentioned categories included knockouts of the CENP-T-related kinetochore component <em><a href="http://www.yeastgenome.org/locus/S000001942/overview">CNN1</a></em> that interacts with the <a href="http://www.yeastgenome.org/locus/S000001406/overview">Ndc80</a> kinetochore subcomplex (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Bock <em>et al.</em> 2012</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Malvezzi <em>et al.</em> 2013</a>) and the <em><a href="http://www.yeastgenome.org/locus/S000004206/overview">CPR6</a></em> peptidyl-prolyl <em>cis</em>-trans isomerase and its interacting protein encoded by <em><a href="http://www.yeastgenome.org/locus/S000005553/overview">STI1</a></em> (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Mayr <em>et al.</em> 2000</a>). <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300089/-/DC1/FigureS6.jpeg">Figure S6</a> shows these groups of genes mapped onto the 37° fitness plot as an indicator of how consistently they interacted with <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17</em> in the screen.</p></div><div id="sec-9" class="subsection"><h3>Phenotypic suppressors of bir1-17</h3><p id="p-27">QFA carried out at 37° identified many potential suppressors of <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17</em> that caused relief of the temperature-sensitive growth defect, including 94 strong suppressors (GIS ≥ 25; <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300089/-/DC1/FileS2.xlsx">Table S8</a> in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300089/-/DC1/FileS2.xlsx">File S2</a>). GO analysis of these genes (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300089/-/DC1/FileS4.xlsx">File S4</a>) revealed highly significant enrichment for genes encoding components of the large ribosomal subunit (LRSU) or proteins involved in its rRNA processing and assembly, as well as for genes involved in mRNA catabolism, and in particular nonsense-mediated mRNA decay (NMD). This was further supported by our analysis of the core protein complexes of <a id="xref-ref-6-3" class="xref-bibr" href="#ref-6">Benschop <em>et al.</em> (2010)</a>. Thus four out of the six strongest suppressors and almost one-third of all the strong (GIS ≥ 25) suppressors could be assigned roles either in ribosome biogenesis or as components of the ribosome. Strikingly, the vast majority of knockouts affecting the ribosome were specific for the LRSU: we isolated 27 <em>RPL</em> gene deletions (removing LRSU proteins) but only three <em>RPS</em> gene deletions (removing small ribosomal subunit proteins) as statistically significant <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17</em> strong suppressors. Taking the unbiased approach of mapping all genes annotated as <em>RPL</em> and <em>RPS</em> in SGD onto the 37° fitness plot, we confirmed that genes in these two groups in general behaved very differently when knocked out and combined with <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17</em> (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300089/-/DC1/FigureS7.jpeg">Figure S7</a>, compare A and B), while <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300089/-/DC1/FigureS7.jpeg">Figure S7</a>C shows that the vast majority of all gene knockouts affecting either biogenesis of the LRSU or LRSU components caused phenotypic suppression of <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17</em>.</p><p id="p-28">Knockouts of the NMD genes <em><a href="http://www.yeastgenome.org/locus/S000002614/overview">EBS1</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000001119/overview">NMD2</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000004685/overview">NAM7</a></em>, and <em><a href="http://www.yeastgenome.org/locus/S000003304/overview">UPF3</a></em> were all within the strong suppressor set (<em><a href="http://www.yeastgenome.org/locus/S000004685/overview">nam7</a></em>∆ was the second strongest suppressor), while the remaining components of the NMD pathway present in the systematic deletion collection (<em><a href="http://www.yeastgenome.org/locus/S000004118/overview">DCN1</a></em> and <em><a href="http://www.yeastgenome.org/locus/S000004355/overview">NMD4</a></em>) were also statistically significant suppressors but falling just below our GIS ≥ 25.0 cut-off (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300089/-/DC1/FileS2.xlsx">Table S8</a> in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300089/-/DC1/FileS2.xlsx">File S2</a>). <em><a href="http://www.yeastgenome.org/locus/S000004390/overview">SKI2</a></em> and <em><a href="http://www.yeastgenome.org/locus/S000006393/overview">SKI3</a></em>, encoding two components of the Ski complex that are involved in a variety of RNA decay processes, including NMD and processes mediated by the exosome, were also identified. The remaining functionally related genes (<em><a href="http://www.yeastgenome.org/locus/S000005527/overview">RRP6</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000005602/overview">SKI7</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000003181/overview">SKI8</a></em>, and <a href="http://www.yeastgenome.org/locus/S000001506/overview">YKL023w</a>) were also suppressors falling slightly below our GIS cut-off. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300089/-/DC1/FigureS8.jpeg">Figure S8</a>A summarizes how the SKI, NMD, and exosome gene knockouts mapped onto the 37° fitness plot.</p><p id="p-29">Although GO terms directly related to cell division were not specifically highlighted by our GO analysis even when suppressors with GIS ≥ 10 were included, one gene encoding a kinetochore component (<em><a href="http://www.yeastgenome.org/locus/S000003028/overview">YBP2</a></em>) was found within the top 30 suppressors (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300089/-/DC1/FileS4.xlsx">File S4</a>). Mapping the core protein complexes defined by <a id="xref-ref-6-4" class="xref-bibr" href="#ref-6">Benschop <em>et al.</em> (2010)</a> onto the 37° fitness plot also identified mutations affecting the COP9 signalosome as suppressors of <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17</em>. The COP9 signalosome removes the NEDD8 homolog <a href="http://www.yeastgenome.org/locus/S000002546/overview">Rub1</a> from the yeast cullin <a href="http://www.yeastgenome.org/locus/S000002290/overview">Cdc53</a> and is required for cell cycle regulation at the G1/S boundary (<a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Wee <em>et al.</em> 2002</a>). Thus, both <em><a href="http://www.yeastgenome.org/locus/S000004627/overview">csi1</a></em>∆ and <em><a href="http://www.yeastgenome.org/locus/S000001333/overview">pci8</a></em>∆ were strong significant suppressors, while <em><a href="http://www.yeastgenome.org/locus/S000002586/overview">csn9</a></em>∆, <em><a href="http://www.yeastgenome.org/locus/S000002375/overview">rri1</a></em>∆, and <em><a href="http://www.yeastgenome.org/locus/S000005477/overview">rri2</a></em>∆ were suppressors that fell just outside our GIS and/or <em>q</em>-value cut-offs (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300089/-/DC1/FigureS8.jpeg">Figure S8</a>B).</p><p id="p-30">The basis for suppression by each of these classes of genes is not yet clear, but suppression by the first two groups of gene knockout is most likely related to alterations in the expression of proteins caused by changes in mRNA stability or alterations in the ribosome population. Since <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17</em> is a temperature-sensitive loss of function mutant, it may be that these knockouts lead to elevated <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">Bir1-17</a></em> protein levels at higher temperatures that can compensate for the effect of the mutations it contains. In a previous QFA analysis, both NMD gene deletions and <em>RPL</em> (but not <em>RPS</em>) gene deletions were also found to suppress the temperature-sensitivity of a <em><a href="http://www.yeastgenome.org/locus/S000002379/overview">cdc13</a>-1</em> query mutation (<a id="xref-ref-2-6" class="xref-bibr" href="#ref-2">Addinall <em>et al.</em> 2011</a>). In this case, the NMD deletions were shown to operate through affecting levels of another protein (<a href="http://www.yeastgenome.org/locus/S000002489/overview">Stn1</a>), which like <a href="http://www.yeastgenome.org/locus/S000002379/overview">Cdc13</a>, is specifically involved in telomere function (<a id="xref-ref-2-7" class="xref-bibr" href="#ref-2">Addinall <em>et al.</em> 2011</a>). Conversely, both NMD and <em>RPL</em> gene deletions enhanced the phenotype of <em><a href="http://www.yeastgenome.org/locus/S000004897/overview">yku70</a></em>∆ in a parallel QFA analysis (<a id="xref-ref-2-8" class="xref-bibr" href="#ref-2">Addinall <em>et al.</em> 2011</a>). It is therefore possible that loss of either the NMD pathway or specific <em>RPL</em> genes may lead to generalized effects on gene expression that can suppress or enhance specific mutations through effects on expression of functionally related genes.</p></div><div id="sec-10" class="subsection"><h3>The genetic enhancement profiles of bir1-17 and ipl1-321 show both common and distinct features, many of which are shared with mcd1-1</h3><p id="p-31">Since <a href="http://www.yeastgenome.org/locus/S000006130/overview">Ipl1</a> functions together with <a href="http://www.yeastgenome.org/locus/S000000360/overview">Sli15</a>, <a href="http://www.yeastgenome.org/locus/S000003849/overview">Bir1</a>, and <a href="http://www.yeastgenome.org/locus/S000029704/overview">Nbl1</a> within the yeast CPC complex, it might be expected that strains with loss-of-function mutations in each of these genes would show synthetic lethality or strong negative genetic interactions with a common set of nonessential gene knockouts that reflect their shared functional roles. Comprehensive analysis has yet to be performed with conditional alleles of either <em><a href="http://www.yeastgenome.org/locus/S000000360/overview">sli15</a></em> or <em><a href="http://www.yeastgenome.org/locus/S000029704/overview">nbl1</a></em>, but our study now enables comparison of the strong genetic enhancers of both <em><a href="http://www.yeastgenome.org/locus/S000006130/overview">ipl1</a>-321</em> and <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17</em> to be made. <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a> shows that 13 of the strong genetic interactions of <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17</em> established in our work are indeed shared with the known synthetic lethal interactions of <em><a href="http://www.yeastgenome.org/locus/S000006130/overview">ipl1</a>-321</em> (colored yellow and red). Overlap between strong negative genetic interactors of <em><a href="http://www.yeastgenome.org/locus/S000006130/overview">ipl1</a>-321</em> and those of the cohesin mutant <em><a href="http://www.yeastgenome.org/locus/S000002161/overview">mcd1</a>-1</em> (also called <em><a href="http://www.yeastgenome.org/locus/S000002161/overview">scc1</a>-73</em>) was noted previously (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Ng <em>et al.</em> 2009</a>) and is also the case here for <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17</em>, with nine strong negative genetic interactions shared by all three mutant alleles (colored red in <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>). These data underline the strong functional links between sister chromatid cohesion, chromosome biorientation, and faithful sister chromatid segregation that operate during cell division.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/9/3203/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparison of the strong negative genetic interactors of ipl1-321, bir1-17, and mcd1-1 revealed by SGA analysis. Genes showing synthetic lethality or strong negative genetic interaction with two or more of ipl1-321, bir1-17, or mcd1-1 are connected to the relevant query mutations by lines that are color-coded according to the number of shared interactions as follows: yellow, negative genetic interactors shared by ipl1-321 and bir1-17; dark blue, negative genetic interactors shared by ipl1-321 and mcd1-1; light blue, negative genetic interactors shared by bir1-17 and mcd1-1; and red, negative genetic interactors shared by all three query genes. The gray box with dashed outline encloses genes encoding components of the Ctf19 complex. Strong negative genetic interactors shown in the diagram were defined as follows: ipl1-321, all genes identified by Ng et al. (2009) as ipl1-321 negative genetic interactors together with additional genes listed in SGD (Cherry et al. 2012, queried December 2016); mcd1-1, all genes identified by Ng et al. (2009) or present in the DRYGIN database (Koh et al. 2010, queried December 2016) as mcd1-1 negative genetic interactors, together with additional genes listed in SGD (Cherry et al. 2012, queried December 2016); bir1-17, all strong negative enhancers identified in this work at any of the three screening temperatures that were shared with at least one of the other two mutations (ipl1-321 or mcd1-1). Note that we find that deletion of CTF19 is synthetic lethal with ipl1-321 as shown (100% inviability of 25 ctf19::KanMX ipl1-321 spores in 31 tetrads from a W303 background ctf19::KanMX &#xD7; ipl1-321 cross where single ctf19::KanMX and ipl1-321 segregants showed 100% and &gt;97% viability, respectively). IRC15 is included within the Ctf19 complex because its knockout also affects CTF19: see Confirmation of the negative genetic interactions between bir1-17 and the Ctf19, Ctf8-Ctf18-Dcc1 and Csm3-Mrc1-Tof1 complexes in W303 for details." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-401112292" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Comparison of the strong negative genetic interactors of &lt;em&gt;ipl1-321&lt;/em&gt;, &lt;em&gt;bir1-17&lt;/em&gt;, and &lt;em&gt;mcd1-1&lt;/em&gt; revealed by SGA analysis. Genes showing synthetic lethality or strong negative genetic interaction with two or more of &lt;em&gt;ipl1-321&lt;/em&gt;, &lt;em&gt;bir1-17&lt;/em&gt;, or &lt;em&gt;mcd1-1&lt;/em&gt; are connected to the relevant query mutations by lines that are color-coded according to the number of shared interactions as follows: yellow, negative genetic interactors shared by &lt;em&gt;ipl1-321&lt;/em&gt; and &lt;em&gt;bir1-17&lt;/em&gt;; dark blue, negative genetic interactors shared by &lt;em&gt;ipl1-321&lt;/em&gt; and &lt;em&gt;mcd1-1&lt;/em&gt;; light blue, negative genetic interactors shared by &lt;em&gt;bir1-17&lt;/em&gt; and &lt;em&gt;mcd1-1&lt;/em&gt;; and red, negative genetic interactors shared by all three query genes. The gray box with dashed outline encloses genes encoding components of the Ctf19 complex. Strong negative genetic interactors shown in the diagram were defined as follows: &lt;em&gt;ipl1-321&lt;/em&gt;, all genes identified by Ng &lt;em&gt;et al.&lt;/em&gt; (2009) as &lt;em&gt;ipl1-321&lt;/em&gt; negative genetic interactors together with additional genes listed in SGD (Cherry &lt;em&gt;et al.&lt;/em&gt; 2012, queried December 2016); &lt;em&gt;mcd1-1&lt;/em&gt;, all genes identified by Ng &lt;em&gt;et al.&lt;/em&gt; (2009) or present in the DRYGIN database (Koh &lt;em&gt;et al.&lt;/em&gt; 2010, queried December 2016) as &lt;em&gt;mcd1-1&lt;/em&gt; negative genetic interactors, together with additional genes listed in SGD (Cherry &lt;em&gt;et al.&lt;/em&gt; 2012, queried December 2016); &lt;em&gt;bir1-17&lt;/em&gt;, all strong negative enhancers identified in this work at any of the three screening temperatures that were shared with at least one of the other two mutations (&lt;em&gt;ipl1-321&lt;/em&gt; or &lt;em&gt;mcd1-1&lt;/em&gt;). Note that we find that deletion of &lt;em&gt;CTF19&lt;/em&gt; is synthetic lethal with &lt;em&gt;ipl1-321&lt;/em&gt; as shown (100% inviability of 25 &lt;em&gt;ctf19&lt;/em&gt;::&lt;em&gt;KanMX ipl1-321&lt;/em&gt; spores in 31 tetrads from a W303 background &lt;em&gt;ctf19&lt;/em&gt;::&lt;em&gt;KanMX&lt;/em&gt; &#xD7; &lt;em&gt;ipl1-321&lt;/em&gt; cross where single &lt;em&gt;ctf19&lt;/em&gt;::&lt;em&gt;Kan&lt;/em&gt;MX and &lt;em&gt;ipl1-321&lt;/em&gt; segregants showed 100% and &gt;97% viability, respectively). &lt;em&gt;IRC15&lt;/em&gt; is included within the Ctf19 complex because its knockout also affects &lt;em&gt;CTF19&lt;/em&gt;: see Confirmation of the negative genetic interactions between bir1-17 and the Ctf19, Ctf8-Ctf18-Dcc1 and Csm3-Mrc1-Tof1 complexes in W303 for details.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/9/3203/F2.medium.gif" width="440" height="379"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/7/9/3203/F2.medium.gif" width="440" height="379"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/9/3203/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/9/3203/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/486392" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-32" class="first-child">Comparison of the strong negative genetic interactors of <em>ipl1-321</em>, <em>bir1-17</em>, and <em>mcd1-1</em> revealed by SGA analysis. Genes showing synthetic lethality or strong negative genetic interaction with two or more of <em>ipl1-321</em>, <em>bir1-17</em>, or <em>mcd1-1</em> are connected to the relevant query mutations by lines that are color-coded according to the number of shared interactions as follows: yellow, negative genetic interactors shared by <em>ipl1-321</em> and <em>bir1-17</em>; dark blue, negative genetic interactors shared by <em>ipl1-321</em> and <em>mcd1-1</em>; light blue, negative genetic interactors shared by <em>bir1-17</em> and <em>mcd1-1</em>; and red, negative genetic interactors shared by all three query genes. The gray box with dashed outline encloses genes encoding components of the Ctf19 complex. Strong negative genetic interactors shown in the diagram were defined as follows: <em>ipl1-321</em>, all genes identified by <a id="xref-ref-52-2" class="xref-bibr" href="#ref-52">Ng <em>et al.</em> (2009)</a> as <em>ipl1-321</em> negative genetic interactors together with additional genes listed in SGD (<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Cherry <em>et al.</em> 2012</a>, queried December 2016); <em>mcd1-1</em>, all genes identified by <a id="xref-ref-52-3" class="xref-bibr" href="#ref-52">Ng <em>et al.</em> (2009)</a> or present in the DRYGIN database (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Koh <em>et al.</em> 2010</a>, queried December 2016) as <em>mcd1-1</em> negative genetic interactors, together with additional genes listed in SGD (<a id="xref-ref-14-3" class="xref-bibr" href="#ref-14">Cherry <em>et al.</em> 2012</a>, queried December 2016); <em>bir1-17</em>, all strong negative enhancers identified in this work at any of the three screening temperatures that were shared with at least one of the other two mutations (<em>ipl1-321</em> or <em>mcd1-1</em>). Note that we find that deletion of <em>CTF19</em> is synthetic lethal with <em>ipl1-321</em> as shown (100% inviability of 25 <em>ctf19</em>::<em>KanMX ipl1-321</em> spores in 31 tetrads from a W303 background <em>ctf19</em>::<em>KanMX</em> × <em>ipl1-321</em> cross where single <em>ctf19</em>::<em>Kan</em>MX and <em>ipl1-321</em> segregants showed 100% and &gt;97% viability, respectively). <em>IRC15</em> is included within the Ctf19 complex because its knockout also affects <em>CTF19</em>: see Confirmation of the negative genetic interactions between bir1-17 and the Ctf19, Ctf8-Ctf18-Dcc1 and Csm3-Mrc1-Tof1 complexes in W303 for details.</p><div class="sb-div caption-clear"></div></div></div><p id="p-33">Remarkably, the <em><a href="http://www.yeastgenome.org/locus/S000006130/overview">ipl1</a>-321</em> and <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17</em> alleles each individually show at least as many strong negative genetic interactions with <em><a href="http://www.yeastgenome.org/locus/S000002161/overview">mcd1</a>-1</em> that they do not share with each other. For example, strains lacking several nonessential components of the <a href="http://www.yeastgenome.org/locus/S000005939/overview">Ctf19</a> complex that were identified in this study as the strongest negative enhancers of <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17</em> are also synthetic lethal with, or strong enhancers of <em><a href="http://www.yeastgenome.org/locus/S000002161/overview">mcd1</a>-1</em>, but not <em><a href="http://www.yeastgenome.org/locus/S000006130/overview">ipl1</a>-321</em>. Conversely, both <em><a href="http://www.yeastgenome.org/locus/S000006130/overview">ipl1</a>-312</em> and <em><a href="http://www.yeastgenome.org/locus/S000002161/overview">mcd1</a>-1</em> show synthetic lethality with the spindle assembly checkpoint gene knockouts (<em><a href="http://www.yeastgenome.org/locus/S000003420/overview">BUB1</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000005552/overview">BUB3</a></em>), whereas these mutations are only moderate negative genetic interactors of <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17</em> at 27°, consistent with our earlier finding that <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17</em> does not require a functional checkpoint for viability (<a id="xref-ref-41-5" class="xref-bibr" href="#ref-41">Makrantoni and Stark 2009</a>). The substantial differences between the profiles of <em><a href="http://www.yeastgenome.org/locus/S000006130/overview">ipl1</a>-321</em> and <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17</em>, and in particular between those subsets of interactions that are shared with <em><a href="http://www.yeastgenome.org/locus/S000002161/overview">mcd1</a>-1</em>, support the notion that the <em><a href="http://www.yeastgenome.org/locus/S000006130/overview">ipl1</a>-321</em> and <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17</em> mutations confer distinct effects on CPC function, although some of these differences could reflect either false positives or false negatives in one or another SGA screen. However, the apparent overlap between the strong enhancers of <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17</em> and <em><a href="http://www.yeastgenome.org/locus/S000002161/overview">mcd1</a>-1</em> is nonetheless striking. We therefore next individually assessed a selection of the <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17</em> genetic interactors identified in our screen both to verify the interactions and to address more systematically whether some interactions were really specific to <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17</em> and not shared with <em><a href="http://www.yeastgenome.org/locus/S000006130/overview">ipl1</a>-321</em>. As a more robust approach to verification, we chose to do this in the W303 genetic background in which <em><a href="http://www.yeastgenome.org/locus/S000006130/overview">ipl1</a>-321</em> and <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17</em> were both isolated (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Biggins <em>et al.</em> 1999</a>; <a id="xref-ref-41-6" class="xref-bibr" href="#ref-41">Makrantoni and Stark 2009</a>), and in which much of the work on CPC-mediated error correction in yeast has been carried out.</p></div><div id="sec-11" class="subsection"><h3>Confirmation of the negative genetic interactions between bir1-17 and the Ctf19, Ctf8-Ctf18-Dcc1, and Csm3-Mrc1-Tof1 complexes in W303</h3><p id="p-34">To verify a selection of the interactions identified in the <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17</em> QFA screen, we deleted a range of hits in the W303 background and carried out tetrad analysis following crosses with <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17</em>. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300089/-/DC1/TableS9.pdf">Table S9</a> summarizes those crosses where <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17 yfg</em>∆ strains were found to be unconditionally lethal. In instances where such double mutants were viable, their relative fitness was examined by spotting out equivalent serial dilutions of strains on agar plates and assessing growth at different temperatures (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300089/-/DC1/FigureS9.jpeg">Figure S9</a>). <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300089/-/DC1/TableS10.pdf">Table S10</a> summarizes both sets of data and shows that, although some interactions could be readily verified in W303, other interactions could not.</p><p id="p-35">A number of conclusions can be drawn from these data. First, all five knockouts of core <a href="http://www.yeastgenome.org/locus/S000005939/overview">Ctf19</a> complex components that we tested from within the group of strong <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17</em> enhancers were synthetic lethal with <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17</em> in the W303 background, as was the knockout of the one example of the kinesin group that we tested (<em><a href="http://www.yeastgenome.org/locus/S000000787/overview">cin8</a></em>∆). <em><a href="http://www.yeastgenome.org/locus/S000002791/overview">nkp1</a></em>∆ (identified as a weaker enhancer in our screen) showed no clear enhancement of <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17</em> in W303, although deletion of its paralog <em><a href="http://www.yeastgenome.org/locus/S000004307/overview">NKP2</a></em> caused strong enhancement of <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17</em>. This verifies in W303 all five core <a href="http://www.yeastgenome.org/locus/S000005939/overview">Ctf19</a> complex components we identified by QFA as strong negative genetic interactors of <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17</em>. Remarkably, the genetic interactions seen in W303 were actually stronger (synthetic lethality) compared with the QFA screen (strong enhancement). The unconditional lethality of either <em><a href="http://www.yeastgenome.org/locus/S000000311/overview">iml3</a></em>∆ or <em><a href="http://www.yeastgenome.org/locus/S000002662/overview">chl4</a></em>∆ with <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17</em> in W303 (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>) is particularly striking in this regard, since the QFA screen only identified these knockouts as enhancers at 37° (see Tables S5–S7 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300089/-/DC1/FileS2.xlsx">File S2</a>). Although we identified <em><a href="http://www.yeastgenome.org/locus/S000001942/overview">cnn1</a></em>∆ (deleting the yeast homolog of the kinetochore protein CENP-T) as a strong enhancer in the screen, this could not be reproduced in the W303 background. Second, negative genetic interactions between both the <em><a href="http://www.yeastgenome.org/locus/S000001234/overview">Ctf8</a></em>-<em><a href="http://www.yeastgenome.org/locus/S000004683/overview">Ctf18</a></em>-<em><a href="http://www.yeastgenome.org/locus/S000000521/overview">Dcc1</a></em> complex (synthetic lethality with <em><a href="http://www.yeastgenome.org/locus/S000000521/overview">dcc1</a></em>∆) and the <em><a href="http://www.yeastgenome.org/locus/S000004651/overview">Csm3</a></em>-<em><a href="http://www.yeastgenome.org/locus/S000000566/overview">Mrc1</a></em>-<em><a href="http://www.yeastgenome.org/locus/S000005217/overview">Tof1</a></em> complex (negative genetic interaction with <em><a href="http://www.yeastgenome.org/locus/S000000566/overview">mrc1</a></em>∆) could be confirmed in the W303 background, although the strong negative enhancement by loss of <a href="http://www.yeastgenome.org/locus/S000004651/overview">Csm3</a> seen by QFA could not be recapitulated. Finally, of the <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17</em> enhancers identified that fell into the other functional groups discussed above, three could be verified in W303 as <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17</em> enhancers (<em><a href="http://www.yeastgenome.org/locus/S000004376/overview">iki3</a></em>∆, <em><a href="http://www.yeastgenome.org/locus/S000004505/overview">vps71</a></em>∆, and <em><a href="http://www.yeastgenome.org/locus/S000004897/overview">yku70</a></em>∆) but several others affecting the COMPASS (<em><a href="http://www.yeastgenome.org/locus/S000004005/overview">bre2</a></em>∆, <em><a href="http://www.yeastgenome.org/locus/S000006059/overview">spp1</a></em>∆), ISW (<em><a href="http://www.yeastgenome.org/locus/S000000966/overview">chd1</a></em>∆) and Rpd3S (<em><a href="http://www.yeastgenome.org/locus/S000005364/overview">sin3</a></em>∆) complexes showed little or no enhancement of <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17</em> in W303.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/9/3203/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genetic interactions of iml3&#x2206; and chl4&#x2206; with bir1-17, ipl1, and sli15 mutations in the W303 genetic background. (A) iml3&#x2206; and chl4&#x2206; each show synthetic lethality with bir1-17. Progeny from five tetrads are shown, indicating the relevant genotypes of viable progeny and the deduced genotypes of inviable progeny. (B) iml3&#x2206; and chl4&#x2206; are viable when combined with ipl1-2, ipl1-321, sli15-3, and sli15&#x2206;2-228. Equivalent 10-fold dilutions of representative single and double mutants were grown at 26 or 35&#xB0; for 2 d. Although the strong temperature-sensitive phenotype of ipl1-2, ipl1-321, and sli15-3 is clearly evident at 35&#xB0;, all double mutant combinations involving these alleles grew normally at 26&#xB0;. (C) iml3&#x2206; and chl4&#x2206; are viable when combined with either sli15&#x2206;2-228 or nbl1-6 but show synthetic negative genetic interaction with both. While iml3&#x2206; sli15&#x2206;2-228 double mutants grew normally at 26&#xB0;, chl4&#x2206; sli15&#x2206;2-228 grew poorly, and both iml3&#x2206; sli15&#x2206;2-228 and chl4&#x2206; sli15&#x2206;2-228 strains showed temperature sensitivity at 35&#xB0; in comparison to the corresponding single mutant strains. iml3&#x2206; nbl1-6 and chl4&#x2206; nbl1-6 strains were also viable, but unlike the three individual mutant strains, were unable to grow at 35&#xB0;." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-401112292" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Genetic interactions of &lt;em&gt;iml3&lt;/em&gt;&#x2206; and &lt;em&gt;chl4&lt;/em&gt;&#x2206; with &lt;em&gt;bir1-17&lt;/em&gt;, &lt;em&gt;ipl1&lt;/em&gt;, and &lt;em&gt;sli15&lt;/em&gt; mutations in the W303 genetic background. (A) &lt;em&gt;iml3&lt;/em&gt;&#x2206; and &lt;em&gt;chl4&lt;/em&gt;&#x2206; each show synthetic lethality with &lt;em&gt;bir1-17&lt;/em&gt;. Progeny from five tetrads are shown, indicating the relevant genotypes of viable progeny and the deduced genotypes of inviable progeny. (B) &lt;em&gt;iml3&lt;/em&gt;&#x2206; and &lt;em&gt;chl4&lt;/em&gt;&#x2206; are viable when combined with &lt;em&gt;ipl1-2&lt;/em&gt;, &lt;em&gt;ipl1-321&lt;/em&gt;, &lt;em&gt;sli15-3&lt;/em&gt;, and &lt;em&gt;sli15&lt;/em&gt;&#x2206;&lt;em&gt;2-228&lt;/em&gt;. Equivalent 10-fold dilutions of representative single and double mutants were grown at 26 or 35&#xB0; for 2 d. Although the strong temperature-sensitive phenotype of &lt;em&gt;ipl1-2&lt;/em&gt;, &lt;em&gt;ipl1-321&lt;/em&gt;, and &lt;em&gt;sli15-3&lt;/em&gt; is clearly evident at 35&#xB0;, all double mutant combinations involving these alleles grew normally at 26&#xB0;. (C) &lt;em&gt;iml3&lt;/em&gt;&#x2206; and &lt;em&gt;chl4&lt;/em&gt;&#x2206; are viable when combined with either &lt;em&gt;sli15&lt;/em&gt;&#x2206;&lt;em&gt;2-228&lt;/em&gt; or &lt;em&gt;nbl1-6&lt;/em&gt; but show synthetic negative genetic interaction with both. While &lt;em&gt;iml3&lt;/em&gt;&#x2206; &lt;em&gt;sli15&lt;/em&gt;&#x2206;&lt;em&gt;2-228&lt;/em&gt; double mutants grew normally at 26&#xB0;, &lt;em&gt;chl4&lt;/em&gt;&#x2206; &lt;em&gt;sli15&lt;/em&gt;&#x2206;&lt;em&gt;2-228&lt;/em&gt; grew poorly, and both &lt;em&gt;iml3&lt;/em&gt;&#x2206; &lt;em&gt;sli15&lt;/em&gt;&#x2206;&lt;em&gt;2-228&lt;/em&gt; and &lt;em&gt;chl4&lt;/em&gt;&#x2206; &lt;em&gt;sli15&lt;/em&gt;&#x2206;&lt;em&gt;2-228&lt;/em&gt; strains showed temperature sensitivity at 35&#xB0; in comparison to the corresponding single mutant strains. &lt;em&gt;iml3&lt;/em&gt;&#x2206; &lt;em&gt;nbl1-6&lt;/em&gt; and &lt;em&gt;chl4&lt;/em&gt;&#x2206; &lt;em&gt;nbl1-6&lt;/em&gt; strains were also viable, but unlike the three individual mutant strains, were unable to grow at 35&#xB0;.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/9/3203/F3.medium.gif" width="440" height="194"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/7/9/3203/F3.medium.gif" width="440" height="194"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/9/3203/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/9/3203/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/486460" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-36" class="first-child">Genetic interactions of <em>iml3</em>∆ and <em>chl4</em>∆ with <em>bir1-17</em>, <em>ipl1</em>, and <em>sli15</em> mutations in the W303 genetic background. (A) <em>iml3</em>∆ and <em>chl4</em>∆ each show synthetic lethality with <em>bir1-17</em>. Progeny from five tetrads are shown, indicating the relevant genotypes of viable progeny and the deduced genotypes of inviable progeny. (B) <em>iml3</em>∆ and <em>chl4</em>∆ are viable when combined with <em>ipl1-2</em>, <em>ipl1-321</em>, <em>sli15-3</em>, and <em>sli15</em>∆<em>2-228</em>. Equivalent 10-fold dilutions of representative single and double mutants were grown at 26 or 35° for 2 d. Although the strong temperature-sensitive phenotype of <em>ipl1-2</em>, <em>ipl1-321</em>, and <em>sli15-3</em> is clearly evident at 35°, all double mutant combinations involving these alleles grew normally at 26°. (C) <em>iml3</em>∆ and <em>chl4</em>∆ are viable when combined with either <em>sli15</em>∆<em>2-228</em> or <em>nbl1-6</em> but show synthetic negative genetic interaction with both. While <em>iml3</em>∆ <em>sli15</em>∆<em>2-228</em> double mutants grew normally at 26°, <em>chl4</em>∆ <em>sli15</em>∆<em>2-228</em> grew poorly, and both <em>iml3</em>∆ <em>sli15</em>∆<em>2-228</em> and <em>chl4</em>∆ <em>sli15</em>∆<em>2-228</em> strains showed temperature sensitivity at 35° in comparison to the corresponding single mutant strains. <em>iml3</em>∆ <em>nbl1-6</em> and <em>chl4</em>∆ <em>nbl1-6</em> strains were also viable, but unlike the three individual mutant strains, were unable to grow at 35°.</p><div class="sb-div caption-clear"></div></div></div><p id="p-37">Although we reproduced the negative genetic interaction between <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17</em> and <em><a href="http://www.yeastgenome.org/locus/S000005938/overview">irc15</a></em>∆ in W303 using a precise ORF knockout identical to that in the systematic collection used for the QFA screen (data not shown), this knockout also deletes the last eight sense codons of <em><a href="http://www.yeastgenome.org/locus/S000005939/overview">CTF19</a></em>, with which <em><a href="http://www.yeastgenome.org/locus/S000005938/overview">IRC15</a></em> overlaps on the opposite strand. Using a construct that removed the first 332 codons of the 499-codon <em><a href="http://www.yeastgenome.org/locus/S000005938/overview">IRC15</a></em> ORF and thus left the <em><a href="http://www.yeastgenome.org/locus/S000005939/overview">CTF19</a></em> ORF intact, we found no genetic interaction with <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17</em> (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300089/-/DC1/FigureS9.jpeg">Figure S9</a>A), and so we conclude that <em><a href="http://www.yeastgenome.org/locus/S000005938/overview">irc15</a></em>∆ was identified in our QFA screen because the systematic knockout interferes with <a href="http://www.yeastgenome.org/locus/S000005939/overview">Ctf19</a> function. It is also therefore likely that other mitotic phenotypes reported for <em><a href="http://www.yeastgenome.org/locus/S000005938/overview">irc15</a></em>∆ mutants may result from effects on the overlapping <em><a href="http://www.yeastgenome.org/locus/S000005939/overview">CTF19</a></em> gene rather than indicating functional consequences of <a href="http://www.yeastgenome.org/locus/S000005938/overview">Irc15</a> loss.</p></div><div id="sec-12" class="subsection"><h3>bir1-17, but not ipl1-321, is dependent on the Iml3-Chl4 subcomplex for viability</h3><p id="p-38">Given that we identified several verifiable strong negative genetic interactions in our QFA analysis between <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17</em> and <a href="http://www.yeastgenome.org/locus/S000005939/overview">Ctf19</a> complex gene knockouts that were not seen in a similar <em><a href="http://www.yeastgenome.org/locus/S000006130/overview">ipl1</a>-321</em> screen (<a id="xref-ref-52-4" class="xref-bibr" href="#ref-52">Ng <em>et al.</em> 2009</a>), we looked in more detail at <em><a href="http://www.yeastgenome.org/locus/S000000311/overview">iml3</a></em>∆ and <em><a href="http://www.yeastgenome.org/locus/S000002662/overview">chl4</a></em>∆. When <em><a href="http://www.yeastgenome.org/locus/S000000311/overview">iml3</a></em>∆ and <em><a href="http://www.yeastgenome.org/locus/S000002662/overview">chl4</a></em>∆ strains were crossed with <em><a href="http://www.yeastgenome.org/locus/S000006130/overview">ipl1</a>-321</em>, <em><a href="http://www.yeastgenome.org/locus/S000006130/overview">ipl1</a>-2</em>, and <em><a href="http://www.yeastgenome.org/locus/S000000360/overview">sli15</a>-3</em> mutants (all in the W303 background), viable double mutants were readily isolated in each case and grew essentially normally at 26°, which is permissive for all three Ts<sup>–</sup> alleles (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3B</a>). Thus <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17</em> strains show complete dependence on functional <a href="http://www.yeastgenome.org/locus/S000000311/overview">Iml3</a> and <a href="http://www.yeastgenome.org/locus/S000002662/overview">Chl4</a> in W303, whereas <em><a href="http://www.yeastgenome.org/locus/S000006130/overview">ipl1</a>-321</em>, <em><a href="http://www.yeastgenome.org/locus/S000006130/overview">ipl1</a>-2</em>, and <em><a href="http://www.yeastgenome.org/locus/S000000360/overview">sli15</a>-3</em> strains are virtually independent of <a href="http://www.yeastgenome.org/locus/S000000311/overview">Iml3</a> and <a href="http://www.yeastgenome.org/locus/S000002662/overview">Chl4</a> for normal proliferation. This confirms that at least some of the differences between the genetic interaction profiles of <em><a href="http://www.yeastgenome.org/locus/S000006130/overview">ipl1</a>-321</em> and <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17</em> are genuine and implies that there is a fundamental difference in the way that the <em><a href="http://www.yeastgenome.org/locus/S000006130/overview">ipl1</a></em> and <em><a href="http://www.yeastgenome.org/locus/S000000360/overview">sli15</a>-3</em> mutations affect CPC function in comparison with <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17</em>.</p><p id="p-39">The <em><a href="http://www.yeastgenome.org/locus/S000000360/overview">sli15</a></em>(∆NT) allele removes the first 228 residues of <a href="http://www.yeastgenome.org/locus/S000000360/overview">Sli15</a> and thereby prevents its interaction with <a href="http://www.yeastgenome.org/locus/S000003849/overview">Bir1</a>. Surprisingly, this is not lethal as anticipated, despite the delocalization of <a href="http://www.yeastgenome.org/locus/S000006130/overview">Ipl1</a> from kinetochores as a result of the truncation (<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Campbell and Desai 2013</a>), leading to the idea that while targeting of the CPC to the kinetochore may be important for efficient chromosome biorientation, it is not essential for it to occur. However, the combination of <em><a href="http://www.yeastgenome.org/locus/S000000360/overview">sli15</a></em>(∆NT) with either <em><a href="http://www.yeastgenome.org/locus/S000005939/overview">ctf19</a></em>∆ or <em><a href="http://www.yeastgenome.org/locus/S000002726/overview">mcm21</a></em>∆ was almost lethal (<a id="xref-ref-13-3" class="xref-bibr" href="#ref-13">Campbell and Desai 2013</a>). We therefore crossed <em><a href="http://www.yeastgenome.org/locus/S000000360/overview">sli15</a></em>(∆NT) with <em><a href="http://www.yeastgenome.org/locus/S000000311/overview">iml3</a></em>∆ and <em><a href="http://www.yeastgenome.org/locus/S000002662/overview">chl4</a></em>∆ and, although the double mutants could be obtained, in contrast to <em><a href="http://www.yeastgenome.org/locus/S000000360/overview">sli15</a>-3</em> (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3B</a>), there was a clear negative genetic interaction in both cases and the <em><a href="http://www.yeastgenome.org/locus/S000002662/overview">chl4</a></em>∆ <em><a href="http://www.yeastgenome.org/locus/S000000360/overview">sli15</a></em>(∆NT) double mutant was inviable at 35°, a temperature at which each single mutant grew normally (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3C</a>). Thus <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17</em> and <em><a href="http://www.yeastgenome.org/locus/S000000360/overview">sli15</a></em>(∆NT) share a clear negative genetic interaction with loss of <a href="http://www.yeastgenome.org/locus/S000000311/overview">Iml3</a> or <a href="http://www.yeastgenome.org/locus/S000002662/overview">Chl4</a> that is not seen in <em><a href="http://www.yeastgenome.org/locus/S000006130/overview">ipl1</a></em> mutants or <em><a href="http://www.yeastgenome.org/locus/S000000360/overview">sli15</a>-3</em>. Double <em><a href="http://www.yeastgenome.org/locus/S000029704/overview">nbl1</a>-6 <a href="http://www.yeastgenome.org/locus/S000002662/overview">chl4</a></em>∆ and <em><a href="http://www.yeastgenome.org/locus/S000029704/overview">nbl1</a>-6 <a href="http://www.yeastgenome.org/locus/S000000311/overview">iml3</a></em>∆ mutants could also be obtained, but again showed a strong negative genetic interaction at 35° or above (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3C</a>). In summary, mutations that affect the targeting of the CPC [<em><a href="http://www.yeastgenome.org/locus/S000029704/overview">nbl1</a>-6</em>, <em><a href="http://www.yeastgenome.org/locus/S000000360/overview">sli15</a>(</em>∆<em>NT)</em> and <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17</em>] show strong negative genetic interactions with loss of either <em><a href="http://www.yeastgenome.org/locus/S000000311/overview">IML3</a></em> or <em><a href="http://www.yeastgenome.org/locus/S000002662/overview">CHL4</a></em>, whereas mutations primarily affecting CPC’s protein kinase activity (<em><a href="http://www.yeastgenome.org/locus/S000006130/overview">ipl1</a>-2</em>, <em><a href="http://www.yeastgenome.org/locus/S000006130/overview">ipl1</a>-321</em>, and <em><a href="http://www.yeastgenome.org/locus/S000000360/overview">sli15</a>-3</em>) do not.</p></div><div id="sec-13" class="subsection"><h3>bir1-17 does not reduce accumulation of pericentromeric cohesin</h3><p id="p-40">The <a href="http://www.yeastgenome.org/locus/S000005939/overview">Ctf19</a> kinetochore complex is important for establishment of pericentromeric cohesion, while <a href="http://www.yeastgenome.org/locus/S000004651/overview">Csm3</a> is needed for ensuring that pericentromeric cohesin functions to hold sister centromeres together and shows an additive phenotype with mutations affecting the <a href="http://www.yeastgenome.org/locus/S000005939/overview">Ctf19</a> complex (<a id="xref-ref-17-6" class="xref-bibr" href="#ref-17">Fernius and Marston 2009</a>). We therefore examined whether the <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17</em> mutant might also have a defect in the accumulation of pericentromeric cohesin that could explain its strong negative genetic interactions with knockouts affecting the <a href="http://www.yeastgenome.org/locus/S000005939/overview">Ctf19</a> and <em><a href="http://www.yeastgenome.org/locus/S000004651/overview">Csm3</a></em>-<em><a href="http://www.yeastgenome.org/locus/S000000566/overview">Mrc1</a></em>-<em><a href="http://www.yeastgenome.org/locus/S000005217/overview">Tof1</a></em> complexes. After synchronizing cells in G1 and then releasing them at 37° in the presence of benomyl and nocodazole, association of cohesin at the centromere, pericentromere, and arm regions of chromosome IV was quantified in metaphase-arrested cells by chromatin immune precipitation using HA-tagged <a href="http://www.yeastgenome.org/locus/S000002161/overview">Mcd1</a>. Relative to a wild-type control, the <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17</em> strain showed no obvious defect in the association of cohesin with the centromeric and pericentromeric regions (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). In contrast, a <em><a href="http://www.yeastgenome.org/locus/S000002662/overview">chl4</a></em>∆ strain showed a clear deficiency in the level of cohesin at the centromere and pericentromere but not in accumulation of cohesin on the chromosome arm, as found previously (<a id="xref-ref-17-7" class="xref-bibr" href="#ref-17">Fernius and Marston 2009</a>). Thus defective accumulation of cohesin around the centromere in <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17</em> is unlikely to provide an explanation for its strong negative interaction with other mutations affecting pericentromeric cohesion. However, we cannot exclude the possibility that, as in <em><a href="http://www.yeastgenome.org/locus/S000004651/overview">csm3</a></em> mutants (<a id="xref-ref-17-8" class="xref-bibr" href="#ref-17">Fernius and Marston 2009</a>), the pericentromeric cohesin that accumulates is not fully functional.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/9/3203/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Accumulation of cohesin at the centromere and in the pericentromeric region is not defective in bir1-17. Analysis of Mcd1-6HA in wild-type, bir1-17, and chl4&#x2206; cells, first synchronized in G1 with &#x3B1;-factor at 25&#xB0; and then released for 3 h at temperatures either permissive (25&#xB0;; P) or restrictive (37&#xB0;; R) for bir1-17 in the presence of nocodazole and benomyl to induce a metaphase arrest. (A) Mcd1 association with the centromeric (CEN), pericentromeric (PERICEN), and arm (ARM) regions of chromosome IV in the two mutant strains relative to the wild-type strain was examined by chromatin immunoprecipitation (ChIP) using an anti-HA antibody. The mean of three independent experiments is shown with error bars indicating the SE. (B) FACS analysis of DNA content confirming synchronization in mitotic metaphase at either temperature." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-401112292" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Accumulation of cohesin at the centromere and in the pericentromeric region is not defective in &lt;em&gt;bir1-17&lt;/em&gt;. Analysis of Mcd1-6HA in wild-type, &lt;em&gt;bir1-17&lt;/em&gt;, and &lt;em&gt;chl4&lt;/em&gt;&#x2206; cells, first synchronized in G1 with &#x3B1;-factor at 25&#xB0; and then released for 3 h at temperatures either permissive (25&#xB0;; P) or restrictive (37&#xB0;; R) for &lt;em&gt;bir1-17&lt;/em&gt; in the presence of nocodazole and benomyl to induce a metaphase arrest. (A) Mcd1 association with the centromeric (CEN), pericentromeric (PERICEN), and arm (ARM) regions of chromosome IV in the two mutant strains relative to the wild-type strain was examined by chromatin immunoprecipitation (ChIP) using an anti-HA antibody. The mean of three independent experiments is shown with error bars indicating the SE. (B) FACS analysis of DNA content confirming synchronization in mitotic metaphase at either temperature.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/9/3203/F4.medium.gif" width="352" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/7/9/3203/F4.medium.gif" width="352" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/9/3203/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/9/3203/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/486409" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-41" class="first-child">Accumulation of cohesin at the centromere and in the pericentromeric region is not defective in <em>bir1-17</em>. Analysis of Mcd1-6HA in wild-type, <em>bir1-17</em>, and <em>chl4</em>∆ cells, first synchronized in G1 with α-factor at 25° and then released for 3 h at temperatures either permissive (25°; P) or restrictive (37°; R) for <em>bir1-17</em> in the presence of nocodazole and benomyl to induce a metaphase arrest. (A) Mcd1 association with the centromeric (CEN), pericentromeric (PERICEN), and arm (ARM) regions of chromosome IV in the two mutant strains relative to the wild-type strain was examined by chromatin immunoprecipitation (ChIP) using an anti-HA antibody. The mean of three independent experiments is shown with error bars indicating the SE. (B) FACS analysis of DNA content confirming synchronization in mitotic metaphase at either temperature.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-14" class="subsection"><h3>Sgo1 overexpression does not relieve the requirement for Chl4 and Iml3 in bir1-17 strains</h3><p id="p-42">Since <a href="http://www.yeastgenome.org/locus/S000000311/overview">Iml3</a> and <a href="http://www.yeastgenome.org/locus/S000002662/overview">Chl4</a> are also needed for pericentromeric accumulation of <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1</a>, which is involved both in CPC recruitment and the bias of sister kinetochores to form bioriented attachments (<a id="xref-ref-69-4" class="xref-bibr" href="#ref-69">Verzijlbergen <em>et al.</em> 2014</a>), we tested whether boosting <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1</a> expression could suppress the synthetic lethality between <em><a href="http://www.yeastgenome.org/locus/S000000311/overview">iml3</a></em>∆ or <em><a href="http://www.yeastgenome.org/locus/S000002662/overview">chl4</a></em>∆ and <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17</em>. To do this and to provide a platform for further analysis of the defect in the double mutants, we utilized the “anchor away” system, in which proteins that function within the nucleus can be excluded by the addition of rapamycin, thereby triggering conditional loss of function (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Haruki <em>et al.</em> 2008</a>). This was carried out in the context of a <em><a href="http://www.yeastgenome.org/locus/S000003827/overview">TOR1</a>-1</em> background so that cells were resistant to growth inhibition by rapamycin, and rapamycin-induced effects can therefore be ascribed solely to nuclear exclusion of the protein of interest. To verify the use of this approach with the <a href="http://www.yeastgenome.org/locus/S000005939/overview">Ctf19</a> kinetochore complex, we tagged each of the two essential <a href="http://www.yeastgenome.org/locus/S000005939/overview">Ctf19</a> complex members (<a href="http://www.yeastgenome.org/locus/S000000415/overview">Ame1</a> and <a href="http://www.yeastgenome.org/locus/S000003411/overview">Okp1</a>) with FRB-GFP and demonstrated that cells could no longer grow when rapamycin was added, while strains in which the nonessential <a href="http://www.yeastgenome.org/locus/S000000311/overview">Iml3</a> and <a href="http://www.yeastgenome.org/locus/S000002662/overview">Chl4</a> were FRB-tagged allowed robust growth on rapamycin (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300089/-/DC1/FigureS10.jpeg">Figure S10</a>A). This analysis also confirmed that nuclear exclusion of <a href="http://www.yeastgenome.org/locus/S000000311/overview">Iml3</a> and <a href="http://www.yeastgenome.org/locus/S000002662/overview">Chl4</a> did not interfere with other essential components of the kinetochore, for example through them “piggy-backing” out of the nucleus with the tagged protein. Microscopy of both the <em><a href="http://www.yeastgenome.org/locus/S000000311/overview">Iml3</a></em>-FRB-GFP and <em><a href="http://www.yeastgenome.org/locus/S000002662/overview">Chl4</a></em>-FRB-GFP strains confirmed that kinetochore localization of either tagged protein was lost within 50 min of rapamycin treatment (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300089/-/DC1/FigureS10.jpeg">Figure S10</a>B).</p><p id="p-43">To test whether boosting <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1</a> levels could reverse the synthetic lethality between <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17</em> and loss of <a href="http://www.yeastgenome.org/locus/S000000311/overview">Iml3</a> or <a href="http://www.yeastgenome.org/locus/S000002662/overview">Chl4</a>, <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17</em> was introduced into the <em><a href="http://www.yeastgenome.org/locus/S000000311/overview">Iml3</a></em>-FRB-GFP and <em><a href="http://www.yeastgenome.org/locus/S000002662/overview">Chl4</a></em>-FRB-GFP strains, and then a galactose-inducible <em>GAL-<a href="http://www.yeastgenome.org/locus/S000005599/overview">SGO1</a></em> construct introduced. As shown in <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>, addition of rapamycin to the growth medium recapitulated the synthetic lethal phenotype of <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17</em> with <em><a href="http://www.yeastgenome.org/locus/S000000311/overview">iml3</a></em>∆ or <em><a href="http://www.yeastgenome.org/locus/S000002662/overview">chl4</a></em>∆ gene knockouts. The rapamycin-induced lethality was not overcome by induction of the <em><a href="http://www.yeastgenome.org/locus/S000005599/overview">SGO1</a></em> overexpression construct. Furthermore, inducing <em><a href="http://www.yeastgenome.org/locus/S000005599/overview">SGO1</a></em> expression appeared, if anything, somewhat detrimental to the growth of strains lacking nuclear <a href="http://www.yeastgenome.org/locus/S000000311/overview">Iml3</a> or <a href="http://www.yeastgenome.org/locus/S000002662/overview">Chl4</a>. <a href="http://www.yeastgenome.org/locus/S000000311/overview">Iml3</a> and <a href="http://www.yeastgenome.org/locus/S000002662/overview">Chl4</a> both have a specific role in ensuring correct sister chromatid separation in meiosis II that is not necessarily shared with other <a href="http://www.yeastgenome.org/locus/S000005939/overview">Ctf19</a> components (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Marston <em>et al.</em> 2004</a>), but <a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a> confirms that the synthetic lethality seen in genetic crosses (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300089/-/DC1/TableS9.pdf">Table S9</a>) is also seen in mitotically proliferating cells.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/9/3203/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Boosting Sgo1 expression does not overcome the requirement for Chl4 and Iml3 in bir1-17 strains. Strains with the indicated genotypes (in a TOR1-1 background) were grown on YPAD medium containing 2% raffinose and 2% galactose to induce expression of GAL-SGO1 where present, either in the absence (left panels) or presence (right panels) of 10 &#xB5;g/ml rapamycin to induce nuclear exclusion of Iml3 or Chl4. Plates were photographed after 2 d of growth at 25&#xB0;. Several other independent isolates of each ura3::GAL-SGO1 strain showed the same properties." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-401112292" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Boosting Sgo1 expression does not overcome the requirement for Chl4 and Iml3 in &lt;em&gt;bir1-17&lt;/em&gt; strains. Strains with the indicated genotypes (in a &lt;em&gt;TOR1-1&lt;/em&gt; background) were grown on YPAD medium containing 2% raffinose and 2% galactose to induce expression of &lt;em&gt;GAL-SGO1&lt;/em&gt; where present, either in the absence (left panels) or presence (right panels) of 10 &#xB5;g/ml rapamycin to induce nuclear exclusion of Iml3 or Chl4. Plates were photographed after 2 d of growth at 25&#xB0;. Several other independent isolates of each &lt;em&gt;ura3&lt;/em&gt;::&lt;em&gt;GAL-SGO1&lt;/em&gt; strain showed the same properties.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/9/3203/F5.medium.gif" width="440" height="318"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/7/9/3203/F5.medium.gif" width="440" height="318"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/9/3203/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/9/3203/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/486494" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-44" class="first-child">Boosting Sgo1 expression does not overcome the requirement for Chl4 and Iml3 in <em>bir1-17</em> strains. Strains with the indicated genotypes (in a <em>TOR1-1</em> background) were grown on YPAD medium containing 2% raffinose and 2% galactose to induce expression of <em>GAL-SGO1</em> where present, either in the absence (left panels) or presence (right panels) of 10 µg/ml rapamycin to induce nuclear exclusion of Iml3 or Chl4. Plates were photographed after 2 d of growth at 25°. Several other independent isolates of each <em>ura3</em>::<em>GAL-SGO1</em> strain showed the same properties.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-15"><h2 class="">Discussion</h2><p id="p-45">Here, we present the results of a QFA screen using the temperature-sensitive <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17</em> mutant that is defective in chromosome biorientation to identify both enhancers and suppressors of <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17</em>. By screening close to the maximum permissive temperature for <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17</em> at 37°, many strong enhancers and suppressors of <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17</em> were identified. Furthermore, QFA performed following spotting, was easily able to identify fitness differences between strains that pinning (SGA) could not (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300089/-/DC1/FigureS2.jpeg">Figure S2</a>). The strong enhancer set contained groups of genes involved in chromosome segregation, sister chromatid cohesion, and microtubule-based processes, consistent with the known function of <a href="http://www.yeastgenome.org/locus/S000003849/overview">Bir1</a> within the CPC, along with additional categories. Our QFA analysis was therefore successful in identifying strong genetic interactions with a subset of genes that, like <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">BIR1</a></em>, are involved in processes related to chromosome segregation. In contrast, only a very small number of enhancers were identified that affected fitness at 20 and/or 27°, temperatures that are fully permissive for <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17</em>, and overall only four true synthetic lethal interactions were observed (with <em><a href="http://www.yeastgenome.org/locus/S000006345/overview">kar3</a></em>∆, <em><a href="http://www.yeastgenome.org/locus/S000005324/overview">coq2</a></em>∆, <em><a href="http://www.yeastgenome.org/locus/S000001695/overview">sac1</a></em>∆, and <em><a href="http://www.yeastgenome.org/locus/S000006228/overview">yme1</a></em>∆) within the statistically significant strong enhancers that we identified. With the exception of <em><a href="http://www.yeastgenome.org/locus/S000006345/overview">kar3</a></em>∆ (see above), none of these genes show any obvious functional connection with <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">BIR1</a></em>.</p><p id="p-46">The negative genetic interaction profiles of <em><a href="http://www.yeastgenome.org/locus/S000006130/overview">ipl1</a>-321</em>, <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17</em>, and <em><a href="http://www.yeastgenome.org/locus/S000002161/overview">mcd1</a>-1</em> showed considerable overlap, consistent with the importance of sister chromatid cohesion for chromosome biorientation, and the <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17</em> screen highlighted the importance of all the core components of the <a href="http://www.yeastgenome.org/locus/S000005939/overview">Ctf19</a> kinetochore complex in cells where <a href="http://www.yeastgenome.org/locus/S000003849/overview">Bir1</a> function is compromised. All of the interactions between <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17</em> and <a href="http://www.yeastgenome.org/locus/S000005939/overview">Ctf19</a> complex members and several of the other interactions identified by QFA involving the kinesin <a href="http://www.yeastgenome.org/locus/S000000787/overview">Cin8</a>, the <a href="http://www.yeastgenome.org/locus/S000005217/overview">Tof1</a> complex, RFC<sup><a href="http://www.yeastgenome.org/locus/S000004683/overview">Ctf18</a></sup>, and tRNA wobble uridine modification were independently verified in the W303 background. Despite the expected overlap that we saw between the enhancers of <em><a href="http://www.yeastgenome.org/locus/S000006130/overview">ipl1</a>-321</em> and <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17</em>, given that <a href="http://www.yeastgenome.org/locus/S000006130/overview">Ipl1</a> and <a href="http://www.yeastgenome.org/locus/S000003849/overview">Bir1</a> function together within the CPC, we nonetheless identified several genes, of which <em><a href="http://www.yeastgenome.org/locus/S000000311/overview">IML3</a></em> and <em><a href="http://www.yeastgenome.org/locus/S000002662/overview">CHL4</a></em> are notable examples, that become essential only in <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17</em> and not in <em><a href="http://www.yeastgenome.org/locus/S000006130/overview">ipl1</a>-321</em> and thus support the notion that the <em><a href="http://www.yeastgenome.org/locus/S000006130/overview">ipl1</a></em> and <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a></em> mutations affect CPC function in somewhat different ways.</p><p id="p-47">Unlike the enhancers, GO analysis of the <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17</em> suppressors identified by QFA did not highlight chromosome or microtubule-based functions, but instead identified genes involved in mRNA catabolism (principally the NMD pathway) or genes encoding LRSU proteins as <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17</em> suppressors. While we cannot exclude the possibility that these suppressors impinge directly on kinetochore function, the CPC or sister chromatid cohesion, we consider it more likely that they relieve the temperature-sensitivity of <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17</em> by altering the expression level of one or more proteins that are relevant to <a href="http://www.yeastgenome.org/locus/S000003849/overview">Bir1</a> function, and that these suppressors therefore act in a less direct manner. Loss of the NMD pathway and components of the LRSU have been isolated as suppressors in other unrelated SGA screens, emphasizing their potential to act pleiotropically (<a id="xref-ref-2-9" class="xref-bibr" href="#ref-2">Addinall <em>et al.</em> 2011</a>). These components may therefore play a role in restricting the viability of partial loss of function mutations in a wide range of essential genes, countering the mutational buffering capacity provided by heat shock proteins that allows such mutations to survive (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Rutherford 2003</a>).</p><p id="p-48">A single kinetochore protein gene knockout (<em><a href="http://www.yeastgenome.org/locus/S000003028/overview">ybp2</a></em>∆) was identified as a suppressor of <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17</em>. <a href="http://www.yeastgenome.org/locus/S000003028/overview">Ybp2</a> shows interactions with the COMA and <a href="http://www.yeastgenome.org/locus/S000001406/overview">Ndc80</a> kinetochore subcomplexes and in its absence there are changes in the interactions between members of the KNL [Knl1 (<a href="http://www.yeastgenome.org/locus/S000003061/overview">Spc105</a>)-<a href="http://www.yeastgenome.org/locus/S000001406/overview">Ndc80</a>-Mis12] network in the kinetochore (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Ohkuni <em>et al.</em> 2008</a>). This is reminiscent of <em><a href="http://www.yeastgenome.org/locus/S000001942/overview">cnn1</a></em>∆, which we identified as a strong enhancer of <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17</em>: <a href="http://www.yeastgenome.org/locus/S000001942/overview">Cnn1</a> also interacts with the <a href="http://www.yeastgenome.org/locus/S000001406/overview">Ndc80</a> complex and <em><a href="http://www.yeastgenome.org/locus/S000001942/overview">cnn1</a></em>∆ also affects interactions within the KNM network within the kinetochore (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Bock <em>et al.</em> 2012</a>). Identification of <em><a href="http://www.yeastgenome.org/locus/S000001942/overview">CNN1</a></em> and <em><a href="http://www.yeastgenome.org/locus/S000003028/overview">YBP2</a></em> as strong genetic interactors of <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">BIR1</a></em> might therefore indicate a role of the CPC in modulating interactions within the KNM network and is consistent with the notion proposed by <a id="xref-ref-11-3" class="xref-bibr" href="#ref-11">Bock <em>et al.</em> (2012)</a> that <a href="http://www.yeastgenome.org/locus/S000001942/overview">Cnn1</a> and <a href="http://www.yeastgenome.org/locus/S000003028/overview">Ybp2</a> could act in overlapping pathways that regulate KNM interactions during the cell cycle.</p><p id="p-49">Although the <a href="http://www.yeastgenome.org/locus/S000005939/overview">Ctf19</a> complex is a component of the kinetochore, it is now clear that it plays a key role in providing a signal for the deposition of pericentromeric cohesin on chromosomes (<a id="xref-ref-17-9" class="xref-bibr" href="#ref-17">Fernius and Marston 2009</a>), which in turn leads to the recruitment of <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1</a> and condensin (<a id="xref-ref-69-5" class="xref-bibr" href="#ref-69">Verzijlbergen <em>et al.</em> 2014</a>). The pericentromeric region has a specialized structure (<a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">Yeh <em>et al.</em> 2008</a>) and it is likely that the accumulation of cohesin and condensin in this region is part of a mechanism that provides an intrinsic bias toward bioriented attachment of sister chromatids to spindle microtubules. <a href="http://www.yeastgenome.org/locus/S000000311/overview">Iml3</a> and <a href="http://www.yeastgenome.org/locus/S000002662/overview">Chl4</a> form a heterodimer (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Hinshaw and Harrison 2013</a>) that is a peripheral component of the <a href="http://www.yeastgenome.org/locus/S000005939/overview">Ctf19</a> complex, based on the assembly dependencies that have been established for <a href="http://www.yeastgenome.org/locus/S000005939/overview">Ctf19</a> complex (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Pot <em>et al.</em> 2003</a>). However, loss of <a href="http://www.yeastgenome.org/locus/S000000311/overview">Iml3</a> or <a href="http://www.yeastgenome.org/locus/S000002662/overview">Chl4</a> is sufficient to disrupt the association of cohesin, condensin, and <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1</a> with the pericentromere, despite not affecting the interaction of other <a href="http://www.yeastgenome.org/locus/S000005939/overview">Ctf19</a> complex members with the kinetochore. Why then, should loss of <a href="http://www.yeastgenome.org/locus/S000000311/overview">Iml3</a> and <a href="http://www.yeastgenome.org/locus/S000002662/overview">Chl4</a> only be a problem in combination with <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17</em> and not <em><a href="http://www.yeastgenome.org/locus/S000006130/overview">ipl1</a>-321</em>? In <em><a href="http://www.yeastgenome.org/locus/S000006130/overview">ipl1</a>-321</em>, the mutation affects the catalytic subunit of the CPC (<a id="xref-ref-9-3" class="xref-bibr" href="#ref-9">Biggins <em>et al.</em> 1999</a>) but it can most likely still be targeted to the kinetochore through its interactions with <a href="http://www.yeastgenome.org/locus/S000000360/overview">Sli15</a>, <a href="http://www.yeastgenome.org/locus/S000029704/overview">Nbl1</a>, and <a href="http://www.yeastgenome.org/locus/S000003849/overview">Bir1</a>. While the <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17</em> mutation does reduce <a href="http://www.yeastgenome.org/locus/S000006130/overview">Ipl1</a>-dependent kinase activity, it also causes significant delocalization of <a href="http://www.yeastgenome.org/locus/S000006130/overview">Ipl1</a> from the kinetochore (<a id="xref-ref-41-7" class="xref-bibr" href="#ref-41">Makrantoni and Stark 2009</a>), and under circumstances where <a href="http://www.yeastgenome.org/locus/S000006130/overview">Ipl1</a> is delocalized, we now know that the <a href="http://www.yeastgenome.org/locus/S000005939/overview">Ctf19</a> complex becomes essential for viability (<a id="xref-ref-13-4" class="xref-bibr" href="#ref-13">Campbell and Desai 2013</a>). Thus, although kinase function is reduced in <em><a href="http://www.yeastgenome.org/locus/S000006130/overview">ipl1</a>-321</em>, because it can be targeted to the kinetochore it may be sufficient to overcome loss of any intrinsic bias toward biorientation that requires <a href="http://www.yeastgenome.org/locus/S000000311/overview">Iml3</a> and <em><a href="http://www.yeastgenome.org/locus/S000002662/overview">Chl4</a></em>-dependent accumulation of cohesin, condensin, and <a href="http://www.yeastgenome.org/locus/S000005599/overview">Sgo1</a> at the pericentromere. Conversely, in <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17</em> strains as in <em><a href="http://www.yeastgenome.org/locus/S000029704/overview">nbl1</a>-6</em> strains (<a id="xref-ref-50-3" class="xref-bibr" href="#ref-50">Nakajima <em>et al.</em> 2009</a>) and <em><a href="http://www.yeastgenome.org/locus/S000000360/overview">sli15</a></em>(∆NT) strains (<a id="xref-ref-13-5" class="xref-bibr" href="#ref-13">Campbell and Desai 2013</a>), delocalization of the CPC from kinetochores may make CPC-dependent error correction less efficient, and cells may now rely much more on the intrinsic bias toward biorientation that ultimately relies on <a href="http://www.yeastgenome.org/locus/S000005939/overview">Ctf19</a> complex-dependent events at the pericentromere. Interestingly, the intrinsic bias toward biorienting chromosomes is greater, and hence the need for CPC-mediated error correction much lower, when microtubule attachment occurs after the SPBs have separated (<a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Indjeian and Murray 2007</a>). This feature may explain the negative genetic interactions between <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17</em> and either <em><a href="http://www.yeastgenome.org/locus/S000006345/overview">kar3</a></em>∆ or <em><a href="http://www.yeastgenome.org/locus/S000000787/overview">cin8</a></em>∆. These two genes encode motor proteins that are involved in spindle pole separation and both knockouts lead to short spindles (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Gardner <em>et al.</em> 2008</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Hoyt <em>et al.</em> 1992</a>; <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Roof <em>et al.</em> 1992</a>; <a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Saunders and Hoyt 1992</a>), which may reduce the intrinsic biorientation bias and lead to a much greater requirement for CPC-mediated error correction.</p><p id="p-50">Since <em><a href="http://www.yeastgenome.org/locus/S000005939/overview">ctf19</a></em>∆ mutant kinetochores lack both <a href="http://www.yeastgenome.org/locus/S000000311/overview">Iml3</a> and <a href="http://www.yeastgenome.org/locus/S000002662/overview">Chl4</a> (<a id="xref-ref-55-2" class="xref-bibr" href="#ref-55">Pot <em>et al.</em> 2003</a>), the synthetic lethality between <em><a href="http://www.yeastgenome.org/locus/S000005939/overview">ctf19</a></em>∆ and <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17</em> could, in principle, be explained solely on the basis of loss of <a href="http://www.yeastgenome.org/locus/S000002662/overview">Chl4</a> and <a href="http://www.yeastgenome.org/locus/S000000311/overview">Iml3</a>, and this could also be the case for some or all of the other deletions of <a href="http://www.yeastgenome.org/locus/S000005939/overview">Ctf19</a> components that share this phenotype. Why then, might loss of some core <a href="http://www.yeastgenome.org/locus/S000005939/overview">Ctf19</a> complex components such as <a href="http://www.yeastgenome.org/locus/S000002726/overview">Mcm21</a> also lead to inviability in the <em><a href="http://www.yeastgenome.org/locus/S000006130/overview">ipl1</a>-321</em> mutant? Perhaps absence of components such as <a href="http://www.yeastgenome.org/locus/S000002726/overview">Mcm21</a> leads to a significantly greater loss of the intrinsic bias toward bi-orientation, or alternatively, these inner components of the <a href="http://www.yeastgenome.org/locus/S000005939/overview">Ctf19</a> complex may have additional roles in kinetochore function, as proposed for their higher eukaryotic counterparts (<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Suzuki <em>et al.</em> 2014</a>), which are separate from their requirement to signal cohesin and condensin deposition at the pericentromere and that lead to reduced kinetochore function when they are absent.</p><p id="p-51">Although we can account for the known phenotypes of the <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17</em> mutation based on <a href="http://www.yeastgenome.org/locus/S000003849/overview">Bir1</a> being a component of the yeast CPC (<a id="xref-ref-41-8" class="xref-bibr" href="#ref-41">Makrantoni and Stark 2009</a>), it is possible that some of the genetic interactions we have found might reflect roles of <a href="http://www.yeastgenome.org/locus/S000003849/overview">Bir1</a> that are independent of it being part of the canonical CPC (<em>i.e.</em>, the <em><a href="http://www.yeastgenome.org/locus/S000006130/overview">Ipl1</a></em>-<em><a href="http://www.yeastgenome.org/locus/S000000360/overview">Sli15</a></em>-<em><a href="http://www.yeastgenome.org/locus/S000029704/overview">Nbl1</a></em>-<em><a href="http://www.yeastgenome.org/locus/S000003849/overview">Bir1</a></em> complex). It has been reported that a significant fraction of <a href="http://www.yeastgenome.org/locus/S000003849/overview">Bir1</a> is present in a complex with <a href="http://www.yeastgenome.org/locus/S000000360/overview">Sli15</a> (and possibly also <a href="http://www.yeastgenome.org/locus/S000029704/overview">Nbl1</a>) that do not contain <a href="http://www.yeastgenome.org/locus/S000006130/overview">Ipl1</a> (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Sandall <em>et al.</em> 2006</a>; <a id="xref-ref-67-2" class="xref-bibr" href="#ref-67">Thomas and Kaplan 2007</a>) and this complex has been implicated both in septin dynamics (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Gillis <em>et al.</em> 2005</a>; <a id="xref-ref-67-3" class="xref-bibr" href="#ref-67">Thomas and Kaplan 2007</a>) and as a tension sensor at the kinetochore (<a id="xref-ref-59-2" class="xref-bibr" href="#ref-59">Sandall <em>et al.</em> 2006</a>). We could not find any genes annotated in SGD as being involved in septin function among the <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17</em> enhancers, although five such genes (<em><a href="http://www.yeastgenome.org/locus/S000001157/overview">DMA1</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000001531/overview">ELM1</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000002717/overview">GIC2</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000005653/overview">RGA1</a></em>, and <em><a href="http://www.yeastgenome.org/locus/S000003291/overview">SPR3</a></em>) were identified as weak <em><a href="http://www.yeastgenome.org/locus/S000003849/overview">bir1</a>-17</em> suppressors (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300089/-/DC1/FileS2.xlsx">Table S8</a> in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300089/-/DC1/FileS2.xlsx">File S2</a>) and may relate to the <a href="http://www.yeastgenome.org/locus/S000006130/overview">Ipl1</a>-independent role in septin behavior proposed for <a href="http://www.yeastgenome.org/locus/S000003849/overview">Bir1</a> (<a id="xref-ref-67-4" class="xref-bibr" href="#ref-67">Thomas and Kaplan 2007</a>). If <a href="http://www.yeastgenome.org/locus/S000000360/overview">Sli15</a>-Bir1 does constitute some form of tension-sensing linkage as proposed by <a id="xref-ref-59-3" class="xref-bibr" href="#ref-59">Sandall <em>et al.</em> (2006)</a>, then it is also possible that the interactions we find with the <a href="http://www.yeastgenome.org/locus/S000005939/overview">Ctf19</a> complex could, in part, reflect a requirement for pericentromeric cohesion in promoting tension-sensing.</p></div><div class="section" id="sec-16"></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-52">We thank Georjana Barnes for providing the <em><a href="http://www.yeastgenome.org/locus/S000029704/overview">nbl1</a>-6</em> mutant, and Charlie Boone for providing yeast strains. This research was supported by project grant BB/G003440/1 from the Biotechnology and Biological Sciences Research Council, awarded to M.J.R.S.; by the Wellcome Trust through a Senior Research Fellowship to A.M. (090903, 107827); core funding for the Wellcome Centre for Cell Biology (092076, 203149); and a VIP award (092416/Z/10/Z). We also gratefully acknowledge additional support from the Wellcome Trust (083524/Z/07/Z).</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-7"><p id="p-53">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300089/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300089/-/DC1</a>.</p></li><li class="fn" id="fn-8"><p id="p-54"><em>Communicating editor: S. Jaspersen</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2017-05-14"><span class="received-label">Received </span>May 14, 2017.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2017-07-25"><span class="accepted-label">Accepted </span>July 25, 2017.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2017 Makrantoni <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-4">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3203.1" data-doi="10.1534/genetics.108.092577"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Addinall</span>  <span class="cit-name-given-names">S. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Downey</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zubko</span>  <span class="cit-name-given-names">M. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dewar</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">A genomewide suppressor and enhancer analysis of <em>cdc13–1</em> reveals varied cellular processes influencing telomere capping in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">180</span>: <span class="cit-fpage">2251</span>–<span class="cit-lpage">2266</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.108.092577%26rft_id%253Dinfo%253Apmid%252F18845848%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE4MC80LzIyNTEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzkvMzIwMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3203.2" data-doi="10.1371/journal.pgen.1001362"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Addinall</span>  <span class="cit-name-given-names">S. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holstein</span>  <span class="cit-name-given-names">E. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lawless</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chapman</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Quantitative fitness analysis shows that NMD proteins and many other protein complexes suppress or enhance distinct telomere cap defects.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e1001362</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DAddinall%26rft.auinit1%253DS.%2BG.%26rft.volume%253D7%26rft.issue%253D4%26rft.spage%253De1001362%26rft.epage%253De1001362%26rft.atitle%253DQuantitative%2Bfitness%2Banalysis%2Bshows%2Bthat%2BNMD%2Bproteins%2Band%2Bmany%2Bother%2Bprotein%2Bcomplexes%2Bsuppress%2Bor%2Benhance%2Bdistinct%2Btelomere%2Bcap%2Bdefects.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1001362%26rft_id%253Dinfo%253Apmid%252F21490951%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1001362&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21490951&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-book" id="cit-7.9.3203.3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Amberg</span>  <span class="cit-name-given-names">D. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burke</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Strathern</span>  <span class="cit-name-given-names">J. N.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-source"><em>Methods in yeast genetics. A Cold Spring Harbor Laboratory Course Manual</em></span>, <span class="cit-publ-name">Cold Spring Harbor Laboratory</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3203.4" data-doi="10.1074/jbc.M109.065730"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bando</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Katou</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Komata</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tanaka</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Itoh</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Csm3, Tof1, and Mrc1 form a heterotrimeric mediator complex that associates with DNA replication forks.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">284</span>: <span class="cit-fpage">34355</span>–<span class="cit-lpage">34365</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M109.065730%26rft_id%253Dinfo%253Apmid%252F19819872%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyODQvNDkvMzQzNTUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzkvMzIwMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3203.5" data-doi="10.1016/S0960-9822(00)00837-X"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Beach</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thibodeaux</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maddox</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yeh</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bloom</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">The role of the proteins Kar9 and Myo2 in orienting the mitotic spindle of budding yeast.</span> <abbr class="cit-jnl-abbrev">Curr. Biol.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">1497</span>–<span class="cit-lpage">1506</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bbiology%2B%253A%2B%2BCB%26rft.stitle%253DCurr%2BBiol%26rft.aulast%253DBeach%26rft.auinit1%253DD.%2BL.%26rft.volume%253D10%26rft.issue%253D23%26rft.spage%253D1497%26rft.epage%253D1506%26rft.atitle%253DThe%2Brole%2Bof%2Bthe%2Bproteins%2BKar9%2Band%2BMyo2%2Bin%2Borienting%2Bthe%2Bmitotic%2Bspindle%2Bof%2Bbudding%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0960-9822%252800%252900837-X%26rft_id%253Dinfo%253Apmid%252F11114516%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0960-9822(00)00837-X&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11114516&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000165819000017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3203.6" data-doi="10.1016/j.molcel.2010.06.002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Benschop</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brabers</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Leenen</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bakker</span>  <span class="cit-name-given-names">L. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Deutekom</span>  <span class="cit-name-given-names">H. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">A consensus of core protein complex compositions for <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">916</span>–<span class="cit-lpage">928</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DBenschop%26rft.auinit1%253DJ.%2BJ.%26rft.volume%253D38%26rft.issue%253D6%26rft.spage%253D916%26rft.epage%253D928%26rft.atitle%253DA%2Bconsensus%2Bof%2Bcore%2Bprotein%2Bcomplex%2Bcompositions%2Bfor%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2010.06.002%26rft_id%253Dinfo%253Apmid%252F20620961%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2010.06.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20620961&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000279297500016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3203.7" data-doi="10.1083/jcb.111.6.2573"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Berlin</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Styles</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fink</span>  <span class="cit-name-given-names">G. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">BIK1, a protein required for microtubule function during mating and mitosis in <em>Saccharomyces cerevisiae</em>, colocalizes with tubulin.</span> <abbr class="cit-jnl-abbrev">J. Cell Biol.</abbr> <span class="cit-vol">111</span>: <span class="cit-fpage">2573</span>–<span class="cit-lpage">2586</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BJournal%2Bof%2BCell%2BBiology%26rft.stitle%253DJCB%26rft.aulast%253DBerlin%26rft.auinit1%253DV.%26rft.volume%253D111%26rft.issue%253D6%26rft.spage%253D2573%26rft.epage%253D2586%26rft.atitle%253DBIK1%252C%2Ba%2Bprotein%2Brequired%2Bfor%2Bmicrotubule%2Bfunction%2Bduring%2Bmating%2Band%2Bmitosis%2Bin%2BSaccharomyces%2Bcerevisiae%252C%2Bcolocalizes%2Bwith%2Btubulin%26rft_id%253Dinfo%253Adoi%252F10.1083%252Fjcb.111.6.2573%26rft_id%253Dinfo%253Apmid%252F2277073%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamNiIjtzOjU6InJlc2lkIjtzOjEwOiIxMTEvNi8yNTczIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy85LzMyMDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3203.8" data-doi="10.1534/genetics.112.145276"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Biggins</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The composition, functions, and regulation of the budding yeast kinetochore.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">194</span>: <span class="cit-fpage">817</span>–<span class="cit-lpage">846</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DBiggins%26rft.auinit1%253DS.%26rft.volume%253D194%26rft.issue%253D4%26rft.spage%253D817%26rft.epage%253D846%26rft.atitle%253DThe%2BComposition%252C%2BFunctions%252C%2Band%2BRegulation%2Bof%2Bthe%2BBudding%2BYeast%2BKinetochore%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.145276%26rft_id%253Dinfo%253Apmid%252F23908374%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk0LzQvODE3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy85LzMyMDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3203.9" data-doi="10.1101/gad.13.5.532"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Biggins</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Severin</span>  <span class="cit-name-given-names">F. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bhalla</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sassoon</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hyman</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">The conserved protein kinase Ipl1 regulates microtubule binding to kinetochores in budding yeast.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">532</span>–<span class="cit-lpage">544</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DBiggins%26rft.auinit1%253DS.%26rft.volume%253D13%26rft.issue%253D5%26rft.spage%253D532%26rft.epage%253D544%26rft.atitle%253DThe%2Bconserved%2Bprotein%2Bkinase%2BIpl1%2Bregulates%2Bmicrotubule%2Bbinding%2Bto%2Bkinetochores%2Bin%2Bbudding%2Byeast%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.13.5.532%26rft_id%253Dinfo%253Apmid%252F10072382%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6ODoiMTMvNS81MzIiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzkvMzIwMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3203.10" data-doi="10.1091/mbc.E10-02-0083"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Blake-Hodek</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cassimeris</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huffaker</span>  <span class="cit-name-given-names">T. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Regulation of microtubule dynamics by Bim1 and Bik1, the budding yeast members of the EB1 and CLIP-170 families of plus-end tracking proteins.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Cell</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">2013</span>–<span class="cit-lpage">2023</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1091%252Fmbc.E10-02-0083%26rft_id%253Dinfo%253Apmid%252F20392838%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xjZWxsIjtzOjU6InJlc2lkIjtzOjEwOiIyMS8xMi8yMDEzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy85LzMyMDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3203.11" data-doi="10.1038/ncb2495"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bock</span>  <span class="cit-name-given-names">L. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pagliuca</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kobayashi</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grove</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oku</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Cnn1 inhibits the interactions between the KMN complexes of the yeast kinetochore.</span> <abbr class="cit-jnl-abbrev">Nat. Cell Biol.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">614</span>–<span class="cit-lpage">624</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2BCell%2BBiology%26rft.stitle%253DNature%2BCell%2BBiology%26rft.aulast%253DBock%26rft.auinit1%253DL.%2BJ.%26rft.volume%253D14%26rft.issue%253D6%26rft.spage%253D614%26rft.epage%253D624%26rft.atitle%253DCnn1%2Binhibits%2Bthe%2Binteractions%2Bbetween%2Bthe%2BKMN%2Bcomplexes%2Bof%2Bthe%2Byeast%2Bkinetochore.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncb2495%26rft_id%253Dinfo%253Apmid%252F22561345%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncb2495&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22561345&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000304599900011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3203.12" data-doi="10.1093/bioinformatics/bth456"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Boyle</span>  <span class="cit-name-given-names">E. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weng</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gollub</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jin</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Botstein</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">GO:TermFinder–open source software for accessing gene ontology information and finding significantly enriched gene ontology terms associated with a list of genes.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">3710</span>–<span class="cit-lpage">3715</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbth456%26rft_id%253Dinfo%253Apmid%252F15297299%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bth456&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15297299&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000225786600064&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3203.13" data-doi="10.1038/nature12057"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Campbell</span>  <span class="cit-name-given-names">C. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Desai</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Tension sensing by Aurora B kinase is independent of survivin-based centromere localization.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">497</span>: <span class="cit-fpage">118</span>–<span class="cit-lpage">121</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D497%26rft.spage%253D118%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature12057%26rft_id%253Dinfo%253Apmid%252F23604256%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature12057&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23604256&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000318221500045&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3203.14" data-doi="10.1093/nar/gkr1029"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cherry</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hong</span>  <span class="cit-name-given-names">E. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amundsen</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Balakrishnan</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Binkley</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title"><em>Saccharomyces</em> genome database: the genomics resource of budding yeast.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">D700</span>–<span class="cit-lpage">D705</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkr1029%26rft_id%253Dinfo%253Apmid%252F22110037%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkr1029&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22110037&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000298601300106&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3203.15" data-doi="10.1038/nsmb.2178"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cho</span>  <span class="cit-name-given-names">U. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harrison</span>  <span class="cit-name-given-names">S. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Ndc10 is a platform for inner kinetochore assembly in budding yeast.</span> <abbr class="cit-jnl-abbrev">Nat. Struct. Mol. Biol.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">48</span>–<span class="cit-lpage">55</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bstructural%2B%2526%2Bmolecular%2Bbiology%26rft.stitle%253DNat%2BStruct%2BMol%2BBiol%26rft.aulast%253DCho%26rft.auinit1%253DU.%2BS.%26rft.volume%253D19%26rft.issue%253D1%26rft.spage%253D48%26rft.epage%253D55%26rft.atitle%253DNdc10%2Bis%2Ba%2Bplatform%2Bfor%2Binner%2Bkinetochore%2Bassembly%2Bin%2Bbudding%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnsmb.2178%26rft_id%253Dinfo%253Apmid%252F22139014%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nsmb.2178&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22139014&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3203.16" data-doi="10.1101/gad.1144403"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">De Wulf</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McAinsh</span>  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorger</span>  <span class="cit-name-given-names">P. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Hierarchical assembly of the budding yeast kinetochore from multiple subcomplexes.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">2902</span>–<span class="cit-lpage">2921</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2BDev.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.1144403%26rft_id%253Dinfo%253Apmid%252F14633972%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6MTA6IjE3LzIzLzI5MDIiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzkvMzIwMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3203.17" data-doi="10.1371/journal.pgen.1000629"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fernius</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marston</span>  <span class="cit-name-given-names">A. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Establishment of cohesion at the pericentromere by the Ctf19 kinetochore subcomplex and the replication fork-associated factor, Csm3.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e1000629</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DFernius%26rft.auinit1%253DJ.%26rft.volume%253D5%26rft.issue%253D9%26rft.spage%253De1000629%26rft.epage%253De1000629%26rft.atitle%253DEstablishment%2Bof%2Bcohesion%2Bat%2Bthe%2Bpericentromere%2Bby%2Bthe%2BCtf19%2Bkinetochore%2Bsubcomplex%2Band%2Bthe%2Breplication%2Bfork-associated%2Bfactor%252C%2BCsm3.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000629%26rft_id%253Dinfo%253Apmid%252F19730685%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000629&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19730685&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3203.18" data-doi="10.1016/j.cub.2013.02.022"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fernius</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nerusheva</span>  <span class="cit-name-given-names">O. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Galander</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alves Fde</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rappsilber</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Cohesin-dependent association of Scc2/4 with the centromere initiates pericentromeric cohesion establishment.</span> <abbr class="cit-jnl-abbrev">Curr. Biol.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">599</span>–<span class="cit-lpage">606</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BBiol.%26rft.volume%253D23%26rft.spage%253D599%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cub.2013.02.022%26rft_id%253Dinfo%253Apmid%252F23499533%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cub.2013.02.022&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23499533&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3203.19" data-doi="10.1083/jcb.201609123"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fink</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Turnbull</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Desai</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Campbell</span>  <span class="cit-name-given-names">C. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">An engineered minimal chromosomal passenger complex reveals a role for INCENP/Sli15 spindle association in chromosome biorientation.</span> <abbr class="cit-jnl-abbrev">J. Cell Biol.</abbr> <span class="cit-vol">216</span>: <span class="cit-fpage">911</span>–<span class="cit-lpage">923</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BCell%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1083%252Fjcb.201609123%26rft_id%253Dinfo%253Apmid%252F28314741%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamNiIjtzOjU6InJlc2lkIjtzOjk6IjIxNi80LzkxMSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvOS8zMjAzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3203.20" data-doi="10.1083/jcb.200710164"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gardner</span>  <span class="cit-name-given-names">M. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haase</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mythreye</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Molk</span>  <span class="cit-name-given-names">J. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anderson</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">The microtubule-based motor Kar3 and plus end-binding protein Bim1 provide structural support for the anaphase spindle.</span> <abbr class="cit-jnl-abbrev">J. Cell Biol.</abbr> <span class="cit-vol">180</span>: <span class="cit-fpage">91</span>–<span class="cit-lpage">100</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BCell%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1083%252Fjcb.200710164%26rft_id%253Dinfo%253Apmid%252F18180364%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamNiIjtzOjU6InJlc2lkIjtzOjg6IjE4MC8xLzkxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy85LzMyMDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3203.21" data-doi="10.1093/nar/20.6.1425"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gietz</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">St Jean</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Woods</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schiestl</span>  <span class="cit-name-given-names">R. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">Improved method for high efficiency transformation of intact yeast cells.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">1425</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DGietz%26rft.auinit1%253DD.%26rft.volume%253D20%26rft.issue%253D6%26rft.spage%253D1425%26rft.epage%253D1425%26rft.atitle%253DImproved%2Bmethod%2Bfor%2Bhigh%2Befficiency%2Btransformation%2Bof%2Bintact%2Byeast%2Bcells%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F20.6.1425%26rft_id%253Dinfo%253Apmid%252F1561104%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/20.6.1425&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1561104&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1992HM42000042&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3203.22" data-doi="10.1083/jcb.200507017"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gillis</span>  <span class="cit-name-given-names">A. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thomas</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hansen</span>  <span class="cit-name-given-names">S. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kaplan</span>  <span class="cit-name-given-names">K. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">A novel role for the CBF3 kinetochore-scaffold complex in regulating septin dynamics and cytokinesis.</span> <abbr class="cit-jnl-abbrev">J. Cell Biol.</abbr> <span class="cit-vol">171</span>: <span class="cit-fpage">773</span>–<span class="cit-lpage">784</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BCell%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1083%252Fjcb.200507017%26rft_id%253Dinfo%253Apmid%252F16330709%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamNiIjtzOjU6InJlc2lkIjtzOjk6IjE3MS81Lzc3MyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvOS8zMjAzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3203.23" data-doi="10.1002/(SICI)1097-0061(199910)15:14&lt;1541::AID-YEA476&gt;3.0.CO;2-K"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goldstein</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCusker</span>  <span class="cit-name-given-names">J. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Three new dominant drug resistance cassettes for gene disruption in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Yeast</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">1541</span>–<span class="cit-lpage">1553</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DYeast%2B%2528Chichester%252C%2BEngland%2529%26rft.stitle%253DYeast%26rft.aulast%253DGoldstein%26rft.auinit1%253DA.%2BL.%26rft.volume%253D15%26rft.issue%253D14%26rft.spage%253D1541%26rft.epage%253D1553%26rft.atitle%253DThree%2Bnew%2Bdominant%2Bdrug%2Bresistance%2Bcassettes%2Bfor%2Bgene%2Bdisruption%2Bin%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1002%252F%2528SICI%25291097-0061%2528199910%252915%253A14%253C1541%253A%253AAID-YEA476%253E3.0.CO%253B2-K%26rft_id%253Dinfo%253Apmid%252F10514571%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/(SICI)1097-0061(199910)15:14&lt;1541::AID-YEA476&gt;3.0.CO;2-K&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10514571&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000083104100010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3203.24" data-doi="10.1016/j.cell.2011.02.013"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hanahan</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weinberg</span>  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Hallmarks of cancer: the next generation.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">144</span>: <span class="cit-fpage">646</span>–<span class="cit-lpage">674</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DHanahan%26rft.auinit1%253DD.%26rft.volume%253D144%26rft.issue%253D5%26rft.spage%253D646%26rft.epage%253D674%26rft.atitle%253DHallmarks%2Bof%2Bcancer%253A%2Bthe%2Bnext%2Bgeneration.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2011.02.013%26rft_id%253Dinfo%253Apmid%252F21376230%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2011.02.013&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21376230&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000288007100007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3203.25" data-doi="10.1016/j.molcel.2008.07.020"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Haruki</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nishikawa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Laemmli</span>  <span class="cit-name-given-names">U. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">The anchor-away technique: rapid, conditional establishment of yeast mutant phenotypes.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">925</span>–<span class="cit-lpage">932</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DHaruki%26rft.auinit1%253DH.%26rft.volume%253D31%26rft.issue%253D6%26rft.spage%253D925%26rft.epage%253D932%26rft.atitle%253DThe%2Banchor-away%2Btechnique%253A%2Brapid%252C%2Bconditional%2Bestablishment%2Bof%2Byeast%2Bmutant%2Bphenotypes.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2008.07.020%26rft_id%253Dinfo%253Apmid%252F18922474%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2008.07.020&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18922474&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000259721600016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3203.26" data-doi="10.1002/yea.1291"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hentges</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van Driessche</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tafforeau</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vandenhaute</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carr</span>  <span class="cit-name-given-names">A. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Three novel antibiotic marker cassettes for gene disruption and marker switching in <em>Schizosaccharomyces pombe</em>.</span> <abbr class="cit-jnl-abbrev">Yeast</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">1013</span>–<span class="cit-lpage">1019</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DYeast%2B%2528Chichester%252C%2BEngland%2529%26rft.stitle%253DYeast%26rft.aulast%253DVallazza-Deschamps%26rft.auinit1%253DG.%26rft.volume%253D22%26rft.issue%253D13%26rft.spage%253D1013%26rft.epage%253D1019%26rft.atitle%253DThree%2Bnovel%2Bantibiotic%2Bmarker%2Bcassettes%2Bfor%2Bgene%2Bdisruption%2Band%2Bmarker%2Bswitching%2Bin%2BSchizosaccharomyces%2Bpombe.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fyea.1291%26rft_id%253Dinfo%253Apmid%252F16200533%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/yea.1291&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16200533&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000232785800001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3203.27" data-doi="10.1016/j.celrep.2013.08.036"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hinshaw</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harrison</span>  <span class="cit-name-given-names">S. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">An Iml3-Chl4 heterodimer links the core centromere to factors required for accurate chromosome segregation.</span> <abbr class="cit-jnl-abbrev">Cell Rep.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">29</span>–<span class="cit-lpage">36</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BRep.%26rft.volume%253D5%26rft.spage%253D29%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.celrep.2013.08.036%26rft_id%253Dinfo%253Apmid%252F24075991%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.celrep.2013.08.036&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24075991&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000326152100005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3203.28" data-doi="10.1128/MCB.10.1.223"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hoyt</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stearns</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Botstein</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">Chromosome instability mutants of <em>Saccharomyces cerevisiae</em> that are defective in microtubule-mediated processes.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">223</span>–<span class="cit-lpage">234</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DHoyt%26rft.auinit1%253DM%2BA%26rft.volume%253D10%26rft.issue%253D1%26rft.spage%253D223%26rft.epage%253D234%26rft.atitle%253DChromosome%2Binstability%2Bmutants%2Bof%2BSaccharomyces%2Bcerevisiae%2Bthat%2Bare%2Bdefective%2Bin%2Bmicrotubule-mediated%2Bprocesses.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.10.1.223%26rft_id%253Dinfo%253Apmid%252F2403635%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjg6IjEwLzEvMjIzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy85LzMyMDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3203.29" data-doi="10.1083/jcb.118.1.109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hoyt</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">He</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Loo</span>  <span class="cit-name-given-names">K. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Saunders</span>  <span class="cit-name-given-names">W. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">Two <em>Saccharomyces cerevisiae</em> kinesin-related gene products required for mitotic spindle assembly.</span> <abbr class="cit-jnl-abbrev">J. Cell Biol.</abbr> <span class="cit-vol">118</span>: <span class="cit-fpage">109</span>–<span class="cit-lpage">120</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BJournal%2Bof%2BCell%2BBiology%26rft.stitle%253DJCB%26rft.aulast%253DHoyt%26rft.auinit1%253DM.%2BA.%26rft.volume%253D118%26rft.issue%253D1%26rft.spage%253D109%26rft.epage%253D120%26rft.atitle%253DTwo%2BSaccharomyces%2Bcerevisiae%2Bkinesin-related%2Bgene%2Bproducts%2Brequired%2Bfor%2Bmitotic%2Bspindle%2Bassembly%26rft_id%253Dinfo%253Adoi%252F10.1083%252Fjcb.118.1.109%26rft_id%253Dinfo%253Apmid%252F1618897%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamNiIjtzOjU6InJlc2lkIjtzOjk6IjExOC8xLzEwOSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvOS8zMjAzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3203.30"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hoyt</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Macke</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roberts</span>  <span class="cit-name-given-names">B. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Geiser</span>  <span class="cit-name-given-names">J. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title"><em>Saccharomyces cerevisiae PAC2</em> functions with <em>CIN1</em>, <em>2</em> and <em>4</em> in a pathway leading to normal microtubule stability.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">146</span>: <span class="cit-fpage">849</span>–<span class="cit-lpage">857</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DHoyt%26rft.auinit1%253DM.%2BA.%26rft.volume%253D146%26rft.issue%253D3%26rft.spage%253D849%26rft.epage%253D857%26rft.atitle%253DSaccharomyces%2Bcerevisiae%2BPAC2%2BFunctions%2BWith%2BCIN1%252C%2B2%2Band%2B4%2Bin%2Ba%2BPathway%2BLeading%2Bto%2BNormal%2BMicrotubule%2BStability%26rft_id%253Dinfo%253Apmid%252F9215891%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTQ2LzMvODQ5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy85LzMyMDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3203.31" data-doi="10.1016/j.cub.2007.09.056"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Indjeian</span>  <span class="cit-name-given-names">V. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Murray</span>  <span class="cit-name-given-names">A. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Budding yeast mitotic chromosomes have an intrinsic bias to biorient on the spindle.</span> <abbr class="cit-jnl-abbrev">Curr. Biol.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">1837</span>–<span class="cit-lpage">1846</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bbiology%2B%253A%2B%2BCB%26rft.stitle%253DCurr%2BBiol%26rft.aulast%253DIndjeian%26rft.auinit1%253DV.%2BB.%26rft.volume%253D17%26rft.issue%253D21%26rft.spage%253D1837%26rft.epage%253D1846%26rft.atitle%253DBudding%2Byeast%2Bmitotic%2Bchromosomes%2Bhave%2Ban%2Bintrinsic%2Bbias%2Bto%2Bbiorient%2Bon%2Bthe%2Bspindle.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cub.2007.09.056%26rft_id%253Dinfo%253Apmid%252F17980598%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cub.2007.09.056&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17980598&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3203.32" data-doi="10.1016/j.cell.2007.07.045"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jeyaprakash</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klein</span>  <span class="cit-name-given-names">U. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lindner</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ebert</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nigg</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Structure of a Survivin-Borealin-INCENP core complex reveals how chromosomal passengers travel together.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">131</span>: <span class="cit-fpage">271</span>–<span class="cit-lpage">285</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DJeyaprakash%26rft.auinit1%253DA.%2BA.%26rft.volume%253D131%26rft.issue%253D2%26rft.spage%253D271%26rft.epage%253D285%26rft.atitle%253DStructure%2Bof%2Ba%2BSurvivin-Borealin-INCENP%2Bcore%2Bcomplex%2Breveals%2Bhow%2Bchromosomal%2Bpassengers%2Btravel%2Btogether.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2007.07.045%26rft_id%253Dinfo%253Apmid%252F17956729%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2007.07.045&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17956729&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000250507700018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3203.33" data-doi="10.1083/jcb.200105029"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheeseman</span>  <span class="cit-name-given-names">I. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kallstrom</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Velmurugan</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barnes</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Functional cooperation of Dam1, Ipl1, and the inner centromere protein (INCENP)-related protein Sli15 during chromosome segregation.</span> <abbr class="cit-jnl-abbrev">J. Cell Biol.</abbr> <span class="cit-vol">155</span>: <span class="cit-fpage">763</span>–<span class="cit-lpage">774</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BCell%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1083%252Fjcb.200105029%26rft_id%253Dinfo%253Apmid%252F11724818%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamNiIjtzOjU6InJlc2lkIjtzOjk6IjE1NS81Lzc2MyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvOS8zMjAzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3203.34" data-doi="10.1126/science.1180189"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kawashima</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yamagishi</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Honda</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ishiguro</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Watanabe</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Phosphorylation of H2A by Bub1 prevents chromosomal instability through localizing shugoshin.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">327</span>: <span class="cit-fpage">172</span>–<span class="cit-lpage">177</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1180189%26rft_id%253Dinfo%253Apmid%252F19965387%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjcvNTk2Mi8xNzIiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzkvMzIwMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3203.35" data-doi="10.1101/gad.1373005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kiburz</span>  <span class="cit-name-given-names">B. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reynolds</span>  <span class="cit-name-given-names">D. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Megee</span>  <span class="cit-name-given-names">P. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marston</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">B. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">The core centromere and Sgo1 establish a 50-kb cohesin-protected domain around centromeres during meiosis I.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">3017</span>–<span class="cit-lpage">3030</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DKiburz%26rft.auinit1%253DB.%2BM.%26rft.volume%253D19%26rft.issue%253D24%26rft.spage%253D3017%26rft.epage%253D3030%26rft.atitle%253DThe%2Bcore%2Bcentromere%2Band%2BSgo1%2Bestablish%2Ba%2B50-kb%2Bcohesin-protected%2Bdomain%2Baround%2Bcentromeres%2Bduring%2Bmeiosis%2BI%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.1373005%26rft_id%253Dinfo%253Apmid%252F16357219%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6MTA6IjE5LzI0LzMwMTciO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzkvMzIwMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3203.36" data-doi="10.1093/nar/gkp820"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Koh</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ding</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Costanzo</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baryshnikova</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Toufighi</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">DRYGIN: a database of quantitative genetic interaction networks in yeast.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">D502</span>–<span class="cit-lpage">D507</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkp820%26rft_id%253Dinfo%253Apmid%252F19880385%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkp820&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19880385&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000276399100081&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3203.37" data-doi="10.1038/nrm3133"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lampert</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Westermann</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A blueprint for kinetochores - new insights into the molecular mechanics of cell division.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Mol. Cell Biol.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">407</span>–<span class="cit-lpage">412</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BMolecular%2Bcell%2Bbiology%26rft.stitle%253DNat%2BRev%2BMol%2BCell%2BBiol%26rft.aulast%253DLampert%26rft.auinit1%253DF.%26rft.volume%253D12%26rft.issue%253D7%26rft.spage%253D407%26rft.epage%253D412%26rft.atitle%253DA%2Bblueprint%2Bfor%2Bkinetochores%2B-%2Bnew%2Binsights%2Binto%2Bthe%2Bmolecular%2Bmechanics%2Bof%2Bcell%2Bdivision.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrm3133%26rft_id%253Dinfo%253Apmid%252F21633384%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrm3133&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21633384&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3203.38" data-doi="10.1016/j.tcb.2010.10.007"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lampson</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheeseman</span>  <span class="cit-name-given-names">I. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Sensing centromere tension: Aurora B and the regulation of kinetochore function.</span> <abbr class="cit-jnl-abbrev">Trends Cell Biol.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">133</span>–<span class="cit-lpage">140</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bcell%2Bbiology%26rft.stitle%253DTrends%2BCell%2BBiol%26rft.aulast%253DLampson%26rft.auinit1%253DM.%2BA.%26rft.volume%253D21%26rft.issue%253D3%26rft.spage%253D133%26rft.epage%253D140%26rft.atitle%253DSensing%2Bcentromere%2Btension%253A%2BAurora%2BB%2Band%2Bthe%2Bregulation%2Bof%2Bkinetochore%2Bfunction.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tcb.2010.10.007%26rft_id%253Dinfo%253Apmid%252F21106376%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tcb.2010.10.007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21106376&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000288730600001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3203.39" data-doi="10.1126/science.1167000"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vader</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vromans</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lampson</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lens</span>  <span class="cit-name-given-names">S. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Sensing chromosome bi-orientation by spatial separation of aurora B kinase from kinetochore substrates.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">323</span>: <span class="cit-fpage">1350</span>–<span class="cit-lpage">1353</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1167000%26rft_id%253Dinfo%253Apmid%252F19150808%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMjMvNTkxOS8xMzUwIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy85LzMyMDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3203.40" data-doi="10.1002/(SICI)1097-0061(199807)14:10&lt;953::AID-YEA293&gt;3.0.CO;2-U"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Longtine</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McKenzie</span>  <span class="cit-name-given-names">A.</span>  <span class="cit-name-suffix">III</span>.</span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Demarini</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shah</span>  <span class="cit-name-given-names">N. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wach</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Additional modules for versatile and economical PCR-based gene deletion and modification in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Yeast</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">953</span>–<span class="cit-lpage">961</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DYeast%2B%2528Chichester%252C%2BEngland%2529%26rft.stitle%253DYeast%26rft.aulast%253DLongtine%26rft.auinit1%253DM.%2BS.%26rft.volume%253D14%26rft.issue%253D10%26rft.spage%253D953%26rft.epage%253D961%26rft.atitle%253DAdditional%2Bmodules%2Bfor%2Bversatile%2Band%2Beconomical%2BPCR-based%2Bgene%2Bdeletion%2Band%2Bmodification%2Bin%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1002%252F%2528SICI%25291097-0061%2528199807%252914%253A10%253C953%253A%253AAID-YEA293%253E3.0.CO%253B2-U%26rft_id%253Dinfo%253Apmid%252F9717241%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/(SICI)1097-0061(199807)14:10&lt;953::AID-YEA293&gt;3.0.CO;2-U&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9717241&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000075005100008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3203.41" data-doi="10.1128/MCB.01911-08"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Makrantoni</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stark</span>  <span class="cit-name-given-names">M. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Efficient chromosome biorientation and the tension checkpoint in <em>Saccharomyces cerevisiae</em> both require Bir1.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">4552</span>–<span class="cit-lpage">4562</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BCell.%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.01911-08%26rft_id%253Dinfo%253Apmid%252F19528231%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjEwOiIyOS8xNi80NTUyIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy85LzMyMDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3203.42" data-doi="10.1038/emboj.2012.356"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Malvezzi</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Litos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schleiffer</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heuck</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mechtler</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">A structural basis for kinetochore recruitment of the Ndc80 complex via two distinct centromere receptors.</span> <abbr class="cit-jnl-abbrev">EMBO J.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">409</span>–<span class="cit-lpage">423</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEMBO%2BJ.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Femboj.2012.356%26rft_id%253Dinfo%253Apmid%252F23334295%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZW1ib2pubCI7czo1OiJyZXNpZCI7czo4OiIzMi8zLzQwOSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvOS8zMjAzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3203.43" data-doi="10.1083/jcb.144.6.1219"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Manning</span>  <span class="cit-name-given-names">B. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barrett</span>  <span class="cit-name-given-names">J. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wallace</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Granok</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Snyder</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Differential regulation of the Kar3p kinesin-related protein by two associated proteins, Cik1p and Vik1p.</span> <abbr class="cit-jnl-abbrev">J. Cell Biol.</abbr> <span class="cit-vol">144</span>: <span class="cit-fpage">1219</span>–<span class="cit-lpage">1233</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BCell%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1083%252Fjcb.144.6.1219%26rft_id%253Dinfo%253Apmid%252F10087265%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamNiIjtzOjU6InJlc2lkIjtzOjEwOiIxNDQvNi8xMjE5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy85LzMyMDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3203.44" data-doi="10.1128/MCB.01176-14"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Marston</span>  <span class="cit-name-given-names">A. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Shugoshins: tension-sensitive pericentromeric adaptors safeguarding chromosome segregation.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">634</span>–<span class="cit-lpage">648</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BCell.%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.01176-14%26rft_id%253Dinfo%253Apmid%252F25452306%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjg6IjM1LzQvNjM0IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy85LzMyMDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3203.45" data-doi="10.1126/science.1094220"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Marston</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tham</span>  <span class="cit-name-given-names">W. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shah</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amon</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">A genome-wide screen identifies genes required for centromeric cohesion.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">303</span>: <span class="cit-fpage">1367</span>–<span class="cit-lpage">1370</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1094220%26rft_id%253Dinfo%253Apmid%252F14752166%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMDMvNTY2Mi8xMzY3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy85LzMyMDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3203.46" data-doi="10.1016/S1097-2765(01)00254-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mayer</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gygi</span>  <span class="cit-name-given-names">S. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aebersold</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hieter</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Identification of RFC(Ctf18p, Ctf8p, Dcc1p): an alternative RFC complex required for sister chromatid cohesion in <em>S. cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">959</span>–<span class="cit-lpage">970</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DMayer%26rft.auinit1%253DM.%2BL.%26rft.volume%253D7%26rft.issue%253D5%26rft.spage%253D959%26rft.epage%253D970%26rft.atitle%253DIdentification%2Bof%2BRFC%2528Ctf18p%252C%2BCtf8p%252C%2BDcc1p%2529%253A%2Ban%2Balternative%2BRFC%2Bcomplex%2Brequired%2Bfor%2Bsister%2Bchromatid%2Bcohesion%2Bin%2BS.%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1097-2765%252801%252900254-4%26rft_id%253Dinfo%253Apmid%252F11389843%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S1097-2765(01)00254-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11389843&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000169039700005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3203.47" data-doi="10.1091/mbc.E03-08-0619"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mayer</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pot</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aneliunas</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Identification of protein complexes required for efficient sister chromatid cohesion.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Cell</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">1736</span>–<span class="cit-lpage">1745</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1091%252Fmbc.E03-08-0619%26rft_id%253Dinfo%253Apmid%252F14742714%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xjZWxsIjtzOjU6InJlc2lkIjtzOjk6IjE1LzQvMTczNiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvOS8zMjAzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3203.48" data-doi="10.1074/jbc.M005251200"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mayr</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Richter</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lilie</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buchner</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Cpr6 and Cpr7, two closely related Hsp90-associated immunophilins from <em>Saccharomyces cerevisiae</em>, differ in their functional properties.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">275</span>: <span class="cit-fpage">34140</span>–<span class="cit-lpage">34146</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M005251200%26rft_id%253Dinfo%253Apmid%252F10942767%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyNzUvNDQvMzQxNDAiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzkvMzIwMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3203.49" data-doi="10.1091/mbc.11.9.2949"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">R. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheng</span>  <span class="cit-name-given-names">S. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rose</span>  <span class="cit-name-given-names">M. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Bim1p/Yeb1p mediates the Kar9p-dependent cortical attachment of cytoplasmic microtubules.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Cell</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">2949</span>–<span class="cit-lpage">2959</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Bof%2Bthe%2BCell%26rft.stitle%253DMol.%2BBiol.%2BCell%26rft.aulast%253DMiller%26rft.auinit1%253DR.%2BK.%26rft.volume%253D11%26rft.issue%253D9%26rft.spage%253D2949%26rft.epage%253D2959%26rft.atitle%253DBim1p%252FYeb1p%2BMediates%2Bthe%2BKar9p-dependent%2BCortical%2BAttachment%2Bof%2BCytoplasmic%2BMicrotubules%26rft_id%253Dinfo%253Adoi%252F10.1091%252Fmbc.11.9.2949%26rft_id%253Dinfo%253Apmid%252F10982392%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xjZWxsIjtzOjU6InJlc2lkIjtzOjk6IjExLzkvMjk0OSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvOS8zMjAzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3203.50" data-doi="10.1091/mbc.E08-10-1011"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nakajima</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tyers</span>  <span class="cit-name-given-names">R. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wong</span>  <span class="cit-name-given-names">C. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yates</span>  <span class="cit-name-given-names">J. R.</span>  <span class="cit-name-suffix">III</span>.</span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Drubin</span>  <span class="cit-name-given-names">D. G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Nbl1p: A Borealin/Dasra/CSC-1-like protein essential for Aurora/Ipl1 complex function and integrity in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Cell</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">1772</span>–<span class="cit-lpage">1784</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1091%252Fmbc.E08-10-1011%26rft_id%253Dinfo%253Apmid%252F19158380%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xjZWxsIjtzOjU6InJlc2lkIjtzOjk6IjIwLzYvMTc3MiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvOS8zMjAzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3203.51" data-doi="10.1146/annurev-genet-102108-134233"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nasmyth</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haering</span>  <span class="cit-name-given-names">C. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Cohesin: its roles and mechanisms.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Genet.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">525</span>–<span class="cit-lpage">558</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bgenetics%26rft.stitle%253DAnnu%2BRev%2BGenet%26rft.aulast%253DNasmyth%26rft.auinit1%253DK.%26rft.volume%253D43%26rft.spage%253D525%26rft.epage%253D558%26rft.atitle%253DCohesin%253A%2Bits%2Broles%2Band%2Bmechanisms.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-genet-102108-134233%26rft_id%253Dinfo%253Apmid%252F19886810%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev-genet-102108-134233&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19886810&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000273580300022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3203.52" data-doi="10.1091/mbc.E09-04-0330"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ng</span>  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Waples</span>  <span class="cit-name-given-names">W. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lavoie</span>  <span class="cit-name-given-names">B. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Biggins</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Pericentromeric sister chromatid cohesion promotes kinetochore biorientation.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Cell</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">3818</span>–<span class="cit-lpage">3827</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1091%252Fmbc.E09-04-0330%26rft_id%253Dinfo%253Apmid%252F19605555%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xjZWxsIjtzOjU6InJlc2lkIjtzOjEwOiIyMC8xNy8zODE4IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy85LzMyMDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3203.53" data-doi="10.1371/journal.pone.0001617"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ohkuni</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abdulle</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tong</span>  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boone</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kitagawa</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Ybp2 associates with the central kinetochore of <em>Saccharomyces cerevisiae</em> and mediates proper mitotic progression.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">e1617</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DOhkuni%26rft.auinit1%253DK.%26rft.volume%253D3%26rft.issue%253D2%26rft.spage%253De1617%26rft.epage%253De1617%26rft.atitle%253DYbp2%2Bassociates%2Bwith%2Bthe%2Bcentral%2Bkinetochore%2Bof%2BSaccharomyces%2Bcerevisiae%2Band%2Bmediates%2Bproper%2Bmitotic%2Bprogression.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001617%26rft_id%253Dinfo%253Apmid%252F18286174%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0001617&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18286174&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3203.54" data-doi="10.1101/gad.13.9.1140"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ortiz</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stemmann</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rank</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lechner</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">A putative protein complex consisting of Ctf19, Mcm21, and Okp1 represents a missing link in the budding yeast kinetochore.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">1140</span>–<span class="cit-lpage">1155</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DOrtiz%26rft.auinit1%253DJ.%26rft.volume%253D13%26rft.issue%253D9%26rft.spage%253D1140%26rft.epage%253D1155%26rft.atitle%253DA%2Bputative%2Bprotein%2Bcomplex%2Bconsisting%2Bof%2BCtf19%252C%2BMcm21%252C%2Band%2BOkp1%2Brepresents%2Ba%2Bmissing%2Blink%2Bin%2Bthe%2Bbudding%2Byeast%2Bkinetochore%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.13.9.1140%26rft_id%253Dinfo%253Apmid%252F10323865%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6OToiMTMvOS8xMTQwIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy85LzMyMDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3203.55" data-doi="10.1091/mbc.E02-08-0517"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pot</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Measday</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Snydsman</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cagney</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fields</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Chl4p and iml3p are two new members of the budding yeast outer kinetochore.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Cell</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">460</span>–<span class="cit-lpage">476</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1091%252Fmbc.E02-08-0517%26rft_id%253Dinfo%253Apmid%252F12589047%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xjZWxsIjtzOjU6InJlc2lkIjtzOjg6IjE0LzIvNDYwIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy85LzMyMDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3203.56" data-doi="10.1083/jcb.118.1.95"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Roof</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meluh</span>  <span class="cit-name-given-names">P. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rose</span>  <span class="cit-name-given-names">M. D.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">Kinesin-related proteins required for assembly of the mitotic spindle.</span> <abbr class="cit-jnl-abbrev">J. Cell Biol.</abbr> <span class="cit-vol">118</span>: <span class="cit-fpage">95</span>–<span class="cit-lpage">108</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BJournal%2Bof%2BCell%2BBiology%26rft.stitle%253DJCB%26rft.aulast%253DRoof%26rft.auinit1%253DD.%2BM.%26rft.volume%253D118%26rft.issue%253D1%26rft.spage%253D95%26rft.epage%253D108%26rft.atitle%253DKinesin-related%2Bproteins%2Brequired%2Bfor%2Bassembly%2Bof%2Bthe%2Bmitotic%2Bspindle%26rft_id%253Dinfo%253Adoi%252F10.1083%252Fjcb.118.1.95%26rft_id%253Dinfo%253Apmid%252F1618910%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamNiIjtzOjU6InJlc2lkIjtzOjg6IjExOC8xLzk1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy85LzMyMDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3203.57" data-doi="10.1016/j.cell.2007.10.002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ruchaud</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carmena</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Earnshaw</span>  <span class="cit-name-given-names">W. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">The chromosomal passenger complex: one for all and all for one.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">131</span>: <span class="cit-fpage">230</span>–<span class="cit-lpage">231</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DRuchaud%26rft.auinit1%253DS.%26rft.volume%253D131%26rft.issue%253D2%26rft.spage%253D230%26rft.epage%253D231%26rft.atitle%253DThe%2Bchromosomal%2Bpassenger%2Bcomplex%253A%2Bone%2Bfor%2Ball%2Band%2Ball%2Bfor%2Bone.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2007.10.002%26rft_id%253Dinfo%253Apmid%252F17956723%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2007.10.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17956723&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000250507700012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3203.58" data-doi="10.1038/nrg1041"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rutherford</span>  <span class="cit-name-given-names">S. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Between genotype and phenotype: protein chaperones and evolvability.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">263</span>–<span class="cit-lpage">274</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DRutherford%26rft.auinit1%253DS.%2BL.%26rft.volume%253D4%26rft.issue%253D4%26rft.spage%253D263%26rft.epage%253D274%26rft.atitle%253DBetween%2Bgenotype%2Band%2Bphenotype%253A%2Bprotein%2Bchaperones%2Band%2Bevolvability.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg1041%26rft_id%253Dinfo%253Apmid%252F12671657%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg1041&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12671657&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000181931400012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3203.59" data-doi="10.1016/j.cell.2006.09.049"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sandall</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Severin</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McLeod</span>  <span class="cit-name-given-names">I. X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yates</span>  <span class="cit-name-given-names">J. R.</span>  <span class="cit-name-suffix">III</span>.</span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oegema</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">A Bir1-Sli15 complex connects centromeres to microtubules and is required to sense kinetochore tension.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">127</span>: <span class="cit-fpage">1179</span>–<span class="cit-lpage">1191</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DSandall%26rft.auinit1%253DS.%26rft.volume%253D127%26rft.issue%253D6%26rft.spage%253D1179%26rft.epage%253D1191%26rft.atitle%253DA%2BBir1-Sli15%2Bcomplex%2Bconnects%2Bcentromeres%2Bto%2Bmicrotubules%2Band%2Bis%2Brequired%2Bto%2Bsense%2Bkinetochore%2Btension.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2006.09.049%26rft_id%253Dinfo%253Apmid%252F17174893%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2006.09.049&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17174893&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000242991000016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3203.60" data-doi="10.1038/emboj.2009.173"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Santaguida</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Musacchio</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The life and miracles of kinetochores.</span> <abbr class="cit-jnl-abbrev">EMBO J.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">2511</span>–<span class="cit-lpage">2531</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BEMBO%2BJournal%26rft.stitle%253DEMBO%2BJ.%26rft.aulast%253DSantaguida%26rft.auinit1%253DS.%26rft.volume%253D28%26rft.issue%253D17%26rft.spage%253D2511%26rft.epage%253D2531%26rft.atitle%253DThe%2Blife%2Band%2Bmiracles%2Bof%2Bkinetochores%26rft_id%253Dinfo%253Adoi%252F10.1038%252Femboj.2009.173%26rft_id%253Dinfo%253Apmid%252F19629042%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZW1ib2pubCI7czo1OiJyZXNpZCI7czoxMDoiMjgvMTcvMjUxMSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvOS8zMjAzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3203.61" data-doi="10.1016/0092-8674(92)90169-D"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Saunders</span>  <span class="cit-name-given-names">W. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoyt</span>  <span class="cit-name-given-names">M. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">Kinesin-related proteins required for structural integrity of the mitotic spindle.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">70</span>: <span class="cit-fpage">451</span>–<span class="cit-lpage">458</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DSaunders%26rft.auinit1%253DW.%2BS.%26rft.volume%253D70%26rft.issue%253D3%26rft.spage%253D451%26rft.epage%253D458%26rft.atitle%253DKinesin-related%2Bproteins%2Brequired%2Bfor%2Bstructural%2Bintegrity%2Bof%2Bthe%2Bmitotic%2Bspindle.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0092-8674%252892%252990169-D%26rft_id%253Dinfo%253Apmid%252F1643659%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0092-8674(92)90169-D&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1643659&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1992JH12400010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3203.62" data-doi="10.1091/mbc.E08-07-0723"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shimogawa</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Widlund</span>  <span class="cit-name-given-names">P. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Riffle</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ess</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">T. N.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Bir1 is required for the tension checkpoint.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Cell</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">915</span>–<span class="cit-lpage">923</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1091%252Fmbc.E08-07-0723%26rft_id%253Dinfo%253Apmid%252F19056681%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xjZWxsIjtzOjU6InJlc2lkIjtzOjg6IjIwLzMvOTE1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy85LzMyMDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3203.63" data-doi="10.1016/j.devcel.2014.08.003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Suzuki</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Badger</span>  <span class="cit-name-given-names">B. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wan</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">DeLuca</span>  <span class="cit-name-given-names">J. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salmon</span>  <span class="cit-name-given-names">E. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The architecture of CCAN proteins creates a structural integrity to resist spindle forces and achieve proper Intrakinetochore stretch.</span> <abbr class="cit-jnl-abbrev">Dev. Cell</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">717</span>–<span class="cit-lpage">730</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDev.%2BCell%26rft.volume%253D30%26rft.spage%253D717%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.devcel.2014.08.003%26rft_id%253Dinfo%253Apmid%252F25268173%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.devcel.2014.08.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25268173&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3203.64" data-doi="10.1016/S0092-8674(02)00633-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tanaka</span>  <span class="cit-name-given-names">T. U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rachidi</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Janke</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pereira</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Galova</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Evidence that the Ipl1-Sli15 (Aurora kinase-INCENP) complex promotes chromosome bi-orientation by altering kinetochore-spindle pole connections.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">317</span>–<span class="cit-lpage">329</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DTanaka%26rft.auinit1%253DT.%2BU.%26rft.volume%253D108%26rft.issue%253D3%26rft.spage%253D317%26rft.epage%253D329%26rft.atitle%253DEvidence%2Bthat%2Bthe%2BIpl1-Sli15%2B%2528Aurora%2Bkinase-INCENP%2529%2Bcomplex%2Bpromotes%2Bchromosome%2Bbi-orientation%2Bby%2Baltering%2Bkinetochore-spindle%2Bpole%2Bconnections.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0092-8674%252802%252900633-5%26rft_id%253Dinfo%253Apmid%252F11853667%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0092-8674(02)00633-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11853667&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000173801600005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3203.65" data-doi="10.1038/nrm1764"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tanaka</span>  <span class="cit-name-given-names">T. U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stark</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tanaka</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Kinetochore capture and bi-orientation on the mitotic spindle.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Mol. Cell Biol.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">929</span>–<span class="cit-lpage">942</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BMolecular%2Bcell%2Bbiology%26rft.stitle%253DNat%2BRev%2BMol%2BCell%2BBiol%26rft.aulast%253DTanaka%26rft.auinit1%253DT.%2BU.%26rft.volume%253D6%26rft.issue%253D12%26rft.spage%253D929%26rft.epage%253D942%26rft.atitle%253DKinetochore%2Bcapture%2Band%2Bbi-orientation%2Bon%2Bthe%2Bmitotic%2Bspindle.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrm1764%26rft_id%253Dinfo%253Apmid%252F16341079%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrm1764&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16341079&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000233774700013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3203.66" data-doi="10.1016/0092-8674(89)90584-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thomas</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rothstein</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">Elevated recombination rates in transcriptionally active DNA.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">56</span>: <span class="cit-fpage">619</span>–<span class="cit-lpage">630</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DThomas%26rft.auinit1%253DB.%2BJ.%26rft.volume%253D56%26rft.issue%253D4%26rft.spage%253D619%26rft.epage%253D630%26rft.atitle%253DElevated%2Brecombination%2Brates%2Bin%2Btranscriptionally%2Bactive%2BDNA.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0092-8674%252889%252990584-9%26rft_id%253Dinfo%253Apmid%252F2645056%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0092-8674(89)90584-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2645056&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1989T405100011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3203.67" data-doi="10.1091/mbc.E07-03-0201"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thomas</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kaplan</span>  <span class="cit-name-given-names">K. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">A Bir1p-Sli15p kinetochore passenger complex regulates septin organization during anaphase.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Cell</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">3820</span>–<span class="cit-lpage">3824</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1091%252Fmbc.E07-03-0201%26rft_id%253Dinfo%253Apmid%252F17652458%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xjZWxsIjtzOjU6InJlc2lkIjtzOjEwOiIxOC8xMC8zODIwIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy85LzMyMDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3203.68" data-doi="10.1016/j.molcel.2005.07.028"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tourriere</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Versini</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cordon-Preciado</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alabert</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pasero</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Mrc1 and Tof1 promote replication fork progression and recovery independently of Rad53.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">699</span>–<span class="cit-lpage">706</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DTourri%25C3%25A8re%26rft.auinit1%253DH.%26rft.volume%253D19%26rft.issue%253D5%26rft.spage%253D699%26rft.epage%253D706%26rft.atitle%253DMrc1%2Band%2BTof1%2Bpromote%2Breplication%2Bfork%2Bprogression%2Band%2Brecovery%2Bindependently%2Bof%2BRad53.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2005.07.028%26rft_id%253Dinfo%253Apmid%252F16137625%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2005.07.028&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16137625&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000231692900011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3203.69" data-doi="10.7554/eLife.01374"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Verzijlbergen</span>  <span class="cit-name-given-names">K. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nerusheva</span>  <span class="cit-name-given-names">O. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kelly</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kerr</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clift</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Shugoshin biases chromosomes for biorientation through condensin recruitment to the pericentromere.</span> <abbr class="cit-jnl-abbrev">Elife</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">e01374</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DElife%26rft.volume%253D3%26rft.spage%253De01374%26rft_id%253Dinfo%253Adoi%252F10.7554%252FeLife.01374%26rft_id%253Dinfo%253Apmid%252F24497542%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.7554/eLife.01374&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24497542&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3203.70" data-doi="10.1186/1471-2156-3-15"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wee</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hetfeld</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dubiel</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wolf</span>  <span class="cit-name-given-names">D. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Conservation of the COP9/signalosome in budding yeast.</span> <abbr class="cit-jnl-abbrev">BMC Genet.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">15</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenetics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenet%26rft.aulast%253DWee%26rft.auinit1%253DS.%26rft.volume%253D3%26rft.spage%253D15%26rft.epage%253D15%26rft.atitle%253DConservation%2Bof%2Bthe%2BCOP9%252Fsignalosome%2Bin%2Bbudding%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2156-3-15%26rft_id%253Dinfo%253Apmid%252F12186635%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2156-3-15&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12186635&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3203.71" data-doi="10.1091/mbc.E05-07-0640"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Widlund</span>  <span class="cit-name-given-names">P. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lyssand</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anderson</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Niessen</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yates</span>  <span class="cit-name-given-names">J. R.</span>  <span class="cit-name-suffix">III</span>.</span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Phosphorylation of the chromosomal passenger protein Bir1 is required for localization of Ndc10 to the spindle during anaphase and full spindle elongation.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Cell</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">1065</span>–<span class="cit-lpage">1074</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1091%252Fmbc.E05-07-0640%26rft_id%253Dinfo%253Apmid%252F16381814%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xjZWxsIjtzOjU6InJlc2lkIjtzOjk6IjE3LzMvMTA2NSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzcvOS8zMjAzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-7.9.3203.72" data-doi="10.1126/science.285.5429.901"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Winzeler</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shoemaker</span>  <span class="cit-name-given-names">D. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Astromoff</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anderson</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Functional characterization of the <em>S. cerevisiae</em> genome by gene deletion and parallel analysis.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">285</span>: <span class="cit-fpage">901</span>–<span class="cit-lpage">906</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DWinzeler%26rft.auinit1%253DE.%2BA.%26rft.volume%253D285%26rft.issue%253D5429%26rft.spage%253D901%26rft.epage%253D906%26rft.atitle%253DFunctional%2BCharacterization%2Bof%2Bthe%2BS.%2Bcerevisiae%2BGenome%2Bby%2BGene%2BDeletion%2Band%2BParallel%2BAnalysis%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.285.5429.901%26rft_id%253Dinfo%253Apmid%252F10436161%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIyODUvNTQyOS85MDEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzkvMzIwMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference  in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3203.73" data-doi="10.1128/MCB.24.16.7082-7090.2004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boone</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klein</span>  <span class="cit-name-given-names">H. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Mrc1 is required for sister chromatid cohesion to aid in recombination repair of spontaneous damage.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">7082</span>–<span class="cit-lpage">7090</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DXu%26rft.auinit1%253DH.%26rft.volume%253D24%26rft.issue%253D16%26rft.spage%253D7082%26rft.epage%253D7090%26rft.atitle%253DMrc1%2BIs%2BRequired%2Bfor%2BSister%2BChromatid%2BCohesion%2BTo%2BAid%2Bin%2BRecombination%2BRepair%2Bof%2BSpontaneous%2BDamage%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.24.16.7082-7090.2004%26rft_id%253Dinfo%253Apmid%252F15282308%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjEwOiIyNC8xNi83MDgyIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy85LzMyMDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference  in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3203.74" data-doi="10.1016/j.cub.2007.12.019"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yeh</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haase</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paliulis</span>  <span class="cit-name-given-names">L. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Joglekar</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bond</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Pericentric chromatin is organized into an intramolecular loop in mitosis.</span> <abbr class="cit-jnl-abbrev">Curr. Biol.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">81</span>–<span class="cit-lpage">90</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bbiology%2B%253A%2B%2BCB%26rft.stitle%253DCurr%2BBiol%26rft.aulast%253DYeh%26rft.auinit1%253DE.%26rft.volume%253D18%26rft.issue%253D2%26rft.spage%253D81%26rft.epage%253D90%26rft.atitle%253DPericentric%2Bchromatin%2Bis%2Borganized%2Binto%2Ban%2Bintramolecular%2Bloop%2Bin%2Bmitosis.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cub.2007.12.019%26rft_id%253Dinfo%253Apmid%252F18211850%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cub.2007.12.019&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18211850&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3203.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000252693900024&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference  in text" id="ref-75">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3203.75" data-doi="10.1073/pnas.96.23.13208"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yoon</span>  <span class="cit-name-given-names">H. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carbon</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Participation of Bir1p, a member of the inhibitor of apoptosis family, in yeast chromosome segregation events.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">96</span>: <span class="cit-fpage">13208</span>–<span class="cit-lpage">13213</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DYoon%26rft.auinit1%253DH.-J.%26rft.volume%253D96%26rft.issue%253D23%26rft.spage%253D13208%26rft.epage%253D13213%26rft.atitle%253DParticipation%2Bof%2BBir1p%252C%2Ba%2Bmember%2Bof%2Bthe%2Binhibitor%2Bof%2Bapoptosis%2Bfamily%252C%2Bin%2Byeast%2Bchromosome%2Bsegregation%2Bevents%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.96.23.13208%26rft_id%253Dinfo%253Apmid%252F10557299%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiOTYvMjMvMTMyMDgiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzkvMzIwMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/7/9/3203.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/9/3195" title="The Integrated Genomic Architecture and Evolution of Dental Divergence in East African Cichlid Fishes (Haplochromis chilotes x H. nyererei)" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/9/3217" title="Trans-ancestry Fine Mapping and Molecular Assays Identify Regulatory Variants at the ANGPTL8 HDL-C GWAS Locus" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/7/9"><span>Volume 7 Issue 9, September 2017</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/7/9.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/7/9.cover-source.jpg?itok=WuYpGT8r" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 7 (9)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/9/3203.supplemental" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-HHkR5Y7lTAoaF1Jjbp1YJoOPd5AkZ2dlv__9cbUpMzs" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/7/9/3203" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/7/9/3203" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/7/9/3203" class="active" data-icon-position="" data-hide-link-title="0">A Functional Link Between Bir1 and the Saccharomyces cerevisiae Ctf19 Kinetochore Complex Revealed Through Quantitative Fitness Analysis</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/486361" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-7o5QyHtEC6r6SDlf4cIu8zySYuCBFnAQECGEuQLEu_0" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/9/3203.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/7/9/3203" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-iT14T3LWpN8sydOTfT82zN4FPWoUFkcU5pVNHajYmS0" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/7/9/3203" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/9/3203.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="486361" id="node486361--2" data-pisa="ggg;7/9/3203" data-pisa-master="ggg;g3.117.300089" data-apath="/ggg/7/9/3203.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Functional Link Between Bir1 and the <em>Saccharomyces cerevisiae</em> Ctf19 Kinetochore Complex Revealed Through Quantitative Fitness Analysis</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-0668-6157" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Vasso Makrantoni</span>, <span class="highwire-citation-author" data-delta="1">Adam Ciesiolka</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-4186-8506" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Conor Lawless</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-9582-4242" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Josefin Fernius</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0002-3596-9407" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Adele Marston</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0003-2478-085X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>David Lydall</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0001-9086-191X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Michael J. R. Stark</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3203-3215;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300089 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node486361--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Vasso Makrantoni </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Centre for Gene Regulation and Expression, University of Dundee, DD1 5EH, UK</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Wellcome Trust Centre for Cell Biology, University of Edinburgh, EH9 3BF, UK</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Vasso%2BMakrantoni%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Makrantoni%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AVasso%2BMakrantoni%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-0668-6157" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Vasso Makrantoni</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Adam Ciesiolka </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Institute for Cell and Molecular Biosciences, Newcastle University, NE2 4HH, UK</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Adam%2BCiesiolka%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ciesiolka%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAdam%2BCiesiolka%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Conor Lawless </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Institute for Cell and Molecular Biosciences, Newcastle University, NE2 4HH, UK</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Conor%2BLawless%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lawless%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AConor%2BLawless%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-4186-8506" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Conor Lawless</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Josefin Fernius </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Wellcome Trust Centre for Cell Biology, University of Edinburgh, EH9 3BF, UK</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Josefin%2BFernius%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fernius%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJosefin%2BFernius%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-9582-4242" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Josefin Fernius</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Adele Marston </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Wellcome Trust Centre for Cell Biology, University of Edinburgh, EH9 3BF, UK</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Adele%2BMarston%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Marston%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAdele%2BMarston%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-3596-9407" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Adele Marston</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">David Lydall </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Institute for Cell and Molecular Biosciences, Newcastle University, NE2 4HH, UK</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BLydall%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lydall%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADavid%2BLydall%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-2478-085X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for David Lydall</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Michael J. R. Stark </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Centre for Gene Regulation and Expression, University of Dundee, DD1 5EH, UK</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BJ.%2BR.%2BStark%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Stark%20MJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMichael%2BJ.%2BR.%2BStark%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-9086-191X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Michael J. R. Stark</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:m.j.r.stark@dundee.ac.uk" class="" data-icon-position="" data-hide-link-title="0">m.j.r.stark@dundee.ac.uk</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/7/9/3203" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/7/9/3203&amp;title=A%20Functional%20Link%20Between%20Bir1%20and%20the%20Saccharomyces%20cerevisiae%20Ctf19%20Kinetochore%20Complex%20Revealed%20Through%20Quantitative%20Fitness%20Analysis" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/7/9/3203&amp;title=A%20Functional%20Link%20Between%20Bir1%20and%20the%20Saccharomyces%20cerevisiae%20Ctf19%20Kinetochore%20Complex%20Revealed%20Through%20Quantitative%20Fitness%20Analysis" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/7/9/3203&amp;title=A%20Functional%20Link%20Between%20Bir1%20and%20the%20Saccharomyces%20cerevisiae%20Ctf19%20Kinetochore%20Complex%20Revealed%20Through%20Quantitative%20Fitness%20Analysis" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/7/9/3203&amp;text=A%20Functional%20Link%20Between%20Bir1%20and%20the%20Saccharomyces%20cerevisiae%20Ctf19%20Kinetochore%20Complex%20Revealed%20Through%20Quantitative%20Fitness%20Analysis" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/7/9/3203&amp;title=A%20Functional%20Link%20Between%20Bir1%20and%20the%20Saccharomyces%20cerevisiae%20Ctf19%20Kinetochore%20Complex%20Revealed%20Through%20Quantitative%20Fitness%20Analysis" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/7/9/3203&amp;t=A%20Functional%20Link%20Between%20Bir1%20and%20the%20Saccharomyces%20cerevisiae%20Ctf19%20Kinetochore%20Complex%20Revealed%20Through%20Quantitative%20Fitness%20Analysis" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/7/9/3203&amp;title=A%20Functional%20Link%20Between%20Bir1%20and%20the%20Saccharomyces%20cerevisiae%20Ctf19%20Kinetochore%20Complex%20Revealed%20Through%20Quantitative%20Fitness%20Analysis" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/7/9/3203&amp;title=A%20Functional%20Link%20Between%20Bir1%20and%20the%20Saccharomyces%20cerevisiae%20Ctf19%20Kinetochore%20Complex%20Revealed%20Through%20Quantitative%20Fitness%20Analysis" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/486361/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="486361" id="node486361--3" data-pisa="ggg;7/9/3203" data-pisa-master="ggg;g3.117.300089" data-apath="/ggg/7/9/3203.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Functional Link Between Bir1 and the <em>Saccharomyces cerevisiae</em> Ctf19 Kinetochore Complex Revealed Through Quantitative Fitness Analysis</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-0668-6157" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Vasso Makrantoni</span>, <span class="highwire-citation-author" data-delta="1">Adam Ciesiolka</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-4186-8506" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Conor Lawless</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-9582-4242" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Josefin Fernius</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0002-3596-9407" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Adele Marston</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0003-2478-085X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>David Lydall</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0001-9086-191X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Michael J. R. Stark</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3203-3215;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300089 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node486361--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Vasso Makrantoni </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Centre for Gene Regulation and Expression, University of Dundee, DD1 5EH, UK</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Wellcome Trust Centre for Cell Biology, University of Edinburgh, EH9 3BF, UK</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Vasso%2BMakrantoni%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Makrantoni%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AVasso%2BMakrantoni%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-0668-6157" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Vasso Makrantoni</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Adam Ciesiolka </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Institute for Cell and Molecular Biosciences, Newcastle University, NE2 4HH, UK</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Adam%2BCiesiolka%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ciesiolka%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAdam%2BCiesiolka%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Conor Lawless </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Institute for Cell and Molecular Biosciences, Newcastle University, NE2 4HH, UK</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Conor%2BLawless%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lawless%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AConor%2BLawless%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-4186-8506" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Conor Lawless</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Josefin Fernius </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Wellcome Trust Centre for Cell Biology, University of Edinburgh, EH9 3BF, UK</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Josefin%2BFernius%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fernius%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJosefin%2BFernius%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-9582-4242" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Josefin Fernius</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Adele Marston </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Wellcome Trust Centre for Cell Biology, University of Edinburgh, EH9 3BF, UK</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Adele%2BMarston%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Marston%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAdele%2BMarston%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-3596-9407" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Adele Marston</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">David Lydall </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Institute for Cell and Molecular Biosciences, Newcastle University, NE2 4HH, UK</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BLydall%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lydall%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADavid%2BLydall%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-2478-085X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for David Lydall</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Michael J. R. Stark </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Centre for Gene Regulation and Expression, University of Dundee, DD1 5EH, UK</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BJ.%2BR.%2BStark%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Stark%20MJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMichael%2BJ.%2BR.%2BStark%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-9086-191X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Michael J. R. Stark</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:m.j.r.stark@dundee.ac.uk" class="" data-icon-position="" data-hide-link-title="0">m.j.r.stark@dundee.ac.uk</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=3203&amp;rft.epage=3215&amp;rft.atitle=A%20Functional%20Link%20Between%20Bir1%20and%20the%20Saccharomyces%20cerevisiae%20Ctf19%20Kinetochore%20Complex%20Revealed%20Through%20Quantitative%20Fitness%20Analysis&amp;rft.volume=7&amp;rft.issue=9&amp;rft.date=2017-09-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Makrantoni%2C+Vasso&amp;rft.au=Ciesiolka%2C+Adam&amp;rft.au=Lawless%2C+Conor&amp;rft.au=Fernius%2C+Josefin&amp;rft.au=Marston%2C+Adele&amp;rft.au=Lydall%2C+David&amp;rft.au=Stark%2C+Michael+J.+R."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/486361/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/486361/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/486361/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/486361/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/486361/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/486361/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/486361/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/486361/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/486361/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/486361/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/486361/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/486361/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/7/9/3203" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-6" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-15" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/7/9/3203.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/7/9/3203.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/7/9/3203.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/7/9/3203.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
