<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/7/9/3103" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/7/9/3103.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/7/9/3103.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/7/9/3103.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/7/9/3103/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/7/9/3103.atom" />
<meta name="HW.pisa" content="ggg;7/9/3103" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Application of Response Surface Methods To Determine Conditions for Optimal Genomic Prediction" />
<meta name="DC.Identifier" content="10.1534/g3.117.044453" />
<meta name="DC.Date" content="2017-09-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2017 Howard et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="An epistatic genetic architecture can have a significant impact on prediction accuracies of genomic prediction (GP) methods. Machine learning methods predict traits comprised of epistatic genetic architectures more accurately than statistical methods based on additive mixed linear models. The differences between these types of GP methods suggest a diagnostic for revealing genetic architectures underlying traits of interest. In addition to genetic architecture, the performance of GP methods may be influenced by the sample size of the training population, the number of QTL, and the proportion of phenotypic variability due to genotypic variability (heritability). Possible values for these factors and the number of combinations of the factor levels that influence the performance of GP methods can be large. Thus, efficient methods for identifying combinations of factor levels that produce most accurate GPs is needed. Herein, we employ response surface methods (RSMs) to find the experimental conditions that produce the most accurate GPs. We illustrate RSM with an example of simulated doubled haploid populations and identify the combination of factors that maximize the difference between prediction accuracies of best linear unbiased prediction (BLUP) and support vector machine (SVM) GP methods. The greatest impact on the response is due to the genetic architecture of the population, heritability of the trait, and the sample size. When epistasis is responsible for all of the genotypic variance and heritability is equal to one and the sample size of the training population is large, the advantage of using the SVM method vs. the BLUP method is greatest. However, except for values close to the maximum, most of the response surface shows little difference between the methods. We also determined that the conditions resulting in the greatest prediction accuracy for BLUP occurred when genetic architecture consists solely of additive effects, and heritability is equal to one." />
<meta name="DC.Contributor" content="Réka Howard" />
<meta name="DC.Contributor" content="Alicia L. Carriquiry" />
<meta name="DC.Contributor" content="William D. Beavis" />
<meta name="article:published_time" content="2017-09-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Application of Response Surface Methods To Determine Conditions for Optimal Genomic Prediction" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;An epistatic genetic architecture can have a significant impact on prediction accuracies of genomic prediction (GP) methods. Machine learning methods predict traits comprised of epistatic genetic architectures more accurately than statistical methods based on additive mixed linear models. The differences between these types of GP methods suggest a diagnostic for revealing genetic architectures underlying traits of interest. In addition to genetic architecture, the performance of GP methods may be influenced by the sample size of the training population, the number of QTL, and the proportion of phenotypic variability due to genotypic variability (heritability). Possible values for these factors and the number of combinations of the factor levels that influence the performance of GP methods can be large. Thus, efficient methods for identifying combinations of factor levels that produce most accurate GPs is needed. Herein, we employ response surface methods (RSMs) to find the experimental conditions that produce the most accurate GPs. We illustrate RSM with an example of simulated doubled haploid populations and identify the combination of factors that maximize the difference between prediction accuracies of best linear unbiased prediction (BLUP) and support vector machine (SVM) GP methods. The greatest impact on the response is due to the genetic architecture of the population, heritability of the trait, and the sample size. When epistasis is responsible for all of the genotypic variance and heritability is equal to one and the sample size of the training population is large, the advantage of using the SVM method &lt;i&gt;vs.&lt;/i&gt; the BLUP method is greatest. However, except for values close to the maximum, most of the response surface shows little difference between the methods. We also determined that the conditions resulting in the greatest prediction accuracy for BLUP occurred when genetic architecture consists solely of additive effects, and heritability is equal to one.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2017/09/01" />
<meta name="citation_mjid" content="ggg;7/9/3103" />
<meta name="citation_id" content="7/9/3103" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/7/9/3103" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/7/9/3103.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/7/9/3103.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/7/9/3103.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.117.044453" />
<meta name="citation_pmid" content="28720710" />
<meta name="citation_volume" content="7" />
<meta name="citation_issue" content="9" />
<meta name="citation_num_pages" content="11" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="3103" />
<meta name="citation_lastpage" content="3113" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Réka Howard" />
<meta name="citation_author_institution" content="Department of Statistics, University of Nebraska, Lincoln, Nebraska 68583" />
<meta name="citation_author_email" content="rekahoward@unl.edu" />
<meta name="citation_author" content="Alicia L. Carriquiry" />
<meta name="citation_author_institution" content="Department of Statistics, Iowa State University, Ames, Iowa 50011" />
<meta name="citation_author" content="William D. Beavis" />
<meta name="citation_author_institution" content="Department of Agronomy, Iowa State University, Ames, Iowa 50011" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=R. Bernardo;citation_title=Prediction of Maize Single-Cross Performance Using RFLPs and Information from Related Hybrids;citation_pages=20-25;citation_volume=34;citation_year=1994;citation_issue=1;citation_doi=10.2135/cropsci1994.0011183X003400010003x" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=R. Bernardo;citation_author=J. M. Yu;citation_title=Prospects for Genomewide Selection for Quantitative Traits in Maize;citation_pages=1082-1090;citation_volume=47;citation_year=2007;citation_issue=3;citation_doi=10.2135/cropsci2006.11.0690" />
<meta name="citation_reference" content="citation_journal_abbrev=Talanta;citation_author=M. A. Bezerra;citation_author=R. E. Santelli;citation_author=E. P. Oliveira;citation_author=L. S. Villar;citation_author=L. A. Escaleira;citation_title=Response surface methodology (RSM) as a tool for optimization in analytical chemistry.;citation_pages=965-977;citation_volume=76;citation_year=2008;citation_issue=5;citation_pmid=18761143;citation_doi=10.1016/j.talanta.2008.05.019" />
<meta name="citation_reference" content="citation_journal_title=J. R. Stat. Soc. B.;citation_author=G. E. P. Box;citation_author=D. R. Cox;citation_title=An analysis of transformation (with discussion).;citation_pages=211-246;citation_volume=26;citation_year=1964" />
<meta name="citation_reference" content="citation_journal_title=J. R. Stat. Soc. Ser. A Stat. Soc.;citation_author=G. E. P. Box;citation_author=K. G. Wilson;citation_title=On the experimental attainment of optimum conditions;citation_pages=1-45;citation_volume=13;citation_year=1951" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=K. J. Clowers;citation_author=R. F. Lyman;citation_author=T. F. Mackay;citation_author=T. J. Morgan;citation_title=Genetic variation in senescence marker protein-30 is associated with natural variation in cold tolerance in Drosophila.;citation_pages=103-113;citation_volume=92;citation_year=2010;citation_issue=2;citation_pmid=20515514;citation_doi=10.1017/S0016672310000108" />
<meta name="citation_reference" content="citation_title=Quantitative Genetics, Genomics and Plant Breeding;citation_title=Complexity, quantitative traits and plant breeding: a role for simulation modelling in the genetic improvement of crops;citation_author=M. Cooper;citation_author=D. W. Podlich;citation_author=K. P. Micallef;citation_author=O. S. Smith;citation_author=N. M. Jensen;citation_pages=143-166;citation_year=2002" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=G. de los Campos;citation_author=D. Gianola;citation_author=G. J. M. Rosa;citation_author=K. A. Weigel;citation_author=J. Crossa;citation_title=Semi-parametric genomic-enabled prediction of genetic values using reproducing kernel Hilbert spaces methods.;citation_pages=295-308;citation_volume=92;citation_year=2010;citation_issue=4;citation_pmid=20943010;citation_doi=10.1017/S0016672310000285" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=J. B. Endelman;citation_title=Ridge regression and other kernels for genomic selection with R package rrBLUP.;citation_pages=250-255;citation_volume=4;citation_year=2011" />
<meta name="citation_reference" content="citation_title=The Elements of Statistical Learning: Data Mining, Inference, and Prediction;citation_author=T. Hastie;citation_author=R. Tibshirani;citation_author=J. Friedman;citation_year=2009" />
<meta name="citation_reference" content="citation_title=Statistical Genetics and Plant Breeding;citation_title=Selection index and expected genetic advance;citation_author=C. R. Henderson;citation_pages=141-163;citation_year=1963" />
<meta name="citation_reference" content="citation_journal_title=Biometrics;citation_author=C. R. Henderson;citation_author=O. Kempthorne;citation_author=S. R. Searle;citation_author=C. M. von Krosigk;citation_title=The estimation of environmental and genetic trends from records subject to culling.;citation_pages=192-218;citation_volume=15;citation_year=1959;citation_doi=10.2307/2527669" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=N. Heslot;citation_author=J. Jannink;citation_author=M. E. Sorrells;citation_title=Perspectives for genomic selection applications and research in plants.;citation_pages=1-12;citation_volume=55;citation_year=2015;citation_doi=10.2135/cropsci2014.03.0249" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=R. Howard;citation_author=A. L. Carriquiry;citation_author=W. D. Beavis;citation_title=Parametric and nonparametric statistical methods for genomic selection of traits with additive and epistatic genetic architectures.;citation_pages=1027-1046;citation_volume=4;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Proc. Nat. Acad. Sci.;citation_author=W. Huang;citation_author=S. Richards;citation_author=M. A. Carbone;citation_author=D. Zhu;citation_author=R. R. H. Anholt;citation_title=Epistasis dominates the genetic architecture of Drosophila quantitative traits.;citation_volume=109;citation_year=2012;citation_pmid=22949659;citation_doi=10.1073/pnas.1213423109" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=C. H. Kao;citation_author=Z. B. Zeng;citation_title=Modeling Epistasis of Quantitative Trait Loci Using Cockerham's Model;citation_pages=1243-1261;citation_volume=160;citation_year=2002;citation_issue=3;citation_pmid=11901137" />
<meta name="citation_reference" content="citation_journal_title=J. Stat. Softw.;citation_author=A. Karatzoglou;citation_author=A. Smola;citation_author=K. Hornik;citation_author=A. Zeileis;citation_title=Kernlab an S4 package for kernel methods in R.;citation_pages=9;citation_volume=11;citation_year=2004" />
<meta name="citation_reference" content="citation_journal_title=WIREs Comp Stat.;citation_author=A. I. Khuri;citation_author=S. Mukhopadhyay;citation_title=Response Surface Methodology.;citation_pages=128-149;citation_volume=2;citation_year=2010" />
<meta name="citation_reference" content="citation_author=K. Koyamada;citation_author=K. Saki;citation_author=T. Itoch;citation_pages=2909-2912;citation_volume=Vol. 2;citation_year=2004" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=R. Lande;citation_author=R. Thompson;citation_title=Efficiency of marker-assisted selection in the improvement of quantitative traits.;citation_pages=743-756;citation_volume=124;citation_year=1990;citation_issue=3;citation_pmid=1968875" />
<meta name="citation_reference" content="citation_journal_title=Ecology;citation_author=W. W. Menke;citation_title=Response surface methodology applications to biological data from field measurements.;citation_pages=920-923;citation_volume=54;citation_year=1973" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=T. H. E. Meuwissen;citation_author=B. J. Hayes;citation_author=M. E. Goddard;citation_title=Prediction of Total Genetic Value Using Genome-Wide Dense Marker Maps;citation_pages=1819-1829;citation_volume=157;citation_year=2001;citation_issue=4;citation_pmid=11290733" />
<meta name="citation_reference" content="citation_journal_title=Technometrics;citation_author=R. H. Myers;citation_author=A. I. Khuri;citation_author=W. H. Carter;citation_title=Response surface methodology: 1966–1988.;citation_pages=137-157;citation_volume=31;citation_year=1989;citation_doi=10.2307/1268813" />
<meta name="citation_reference" content="citation_author=T. H. Naylor;citation_pages=80-95;citation_year=1969" />
<meta name="citation_reference" content="citation_author=V. Vapnik;citation_year=1995" />
<meta name="citation_reference" content="citation_journal_title=Electron. J. Biotechnol.;citation_author=X. Zhang;citation_author=J. Zhou;citation_author=W. Fu;citation_author=Z. Li;citation_author=J. Zhong;citation_title=Response surface methodology used for statistical optimization of jiean-peptide production by Bacillus subtillis.;citation_pages=1-12;citation_volume=13;citation_year=2010" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=Y. Zhao;citation_author=M. Gowda;citation_author=W. Liu;citation_author=T. Wurschum;citation_author=H. P. Maurer;citation_title=Accuracy of genomic selection in European maize elite breeding populations.;citation_pages=769-776;citation_volume=124;citation_year=2012;citation_issue=4;citation_pmid=22075809;citation_doi=10.1007/s00122-011-1745-y" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Application of Response Surface Methods To Determine Conditions for Optimal Genomic Prediction" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9.cover-source.jpg" />
<meta name="twitter:description" content="An epistatic genetic architecture can have a significant impact on prediction accuracies of genomic prediction (GP) methods. Machine learning methods predict traits comprised of epistatic genetic architectures more accurately than statistical methods based on additive mixed linear models. The differences between these types of GP methods suggest a diagnostic for revealing genetic architectures underlying traits of interest. In addition to genetic architecture, the performance of GP methods may be influenced by the sample size of the training population, the number of QTL, and the proportion of phenotypic variability due to genotypic variability (heritability). Possible values for these factors and the number of combinations of the factor levels that influence the performance of GP methods can be large. Thus, efficient methods for identifying combinations of factor levels that produce most accurate GPs is needed. Herein, we employ response surface methods (RSMs) to find the experimental conditions that produce the most accurate GPs. We illustrate RSM with an example of simulated doubled haploid populations and identify the combination of factors that maximize the difference between prediction accuracies of best linear unbiased prediction (BLUP) and support vector machine (SVM) GP methods. The greatest impact on the response is due to the genetic architecture of the population, heritability of the trait, and the sample size. When epistasis is responsible for all of the genotypic variance and heritability is equal to one and the sample size of the training population is large, the advantage of using the SVM method vs. the BLUP method is greatest. However, except for values close to the maximum, most of the response surface shows little difference between the methods. We also determined that the conditions resulting in the greatest prediction accuracy for BLUP occurred when genetic architecture consists solely of additive effects, and heritability is equal to one." />
<meta name="og-title" property="og:title" content="Application of Response Surface Methods To Determine Conditions for Optimal Genomic Prediction" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/7/9/3103" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="An epistatic genetic architecture can have a significant impact on prediction accuracies of genomic prediction (GP) methods. Machine learning methods predict traits comprised of epistatic genetic architectures more accurately than statistical methods based on additive mixed linear models. The differences between these types of GP methods suggest a diagnostic for revealing genetic architectures underlying traits of interest. In addition to genetic architecture, the performance of GP methods may be influenced by the sample size of the training population, the number of QTL, and the proportion of phenotypic variability due to genotypic variability (heritability). Possible values for these factors and the number of combinations of the factor levels that influence the performance of GP methods can be large. Thus, efficient methods for identifying combinations of factor levels that produce most accurate GPs is needed. Herein, we employ response surface methods (RSMs) to find the experimental conditions that produce the most accurate GPs. We illustrate RSM with an example of simulated doubled haploid populations and identify the combination of factors that maximize the difference between prediction accuracies of best linear unbiased prediction (BLUP) and support vector machine (SVM) GP methods. The greatest impact on the response is due to the genetic architecture of the population, heritability of the trait, and the sample size. When epistasis is responsible for all of the genotypic variance and heritability is equal to one and the sample size of the training population is large, the advantage of using the SVM method vs. the BLUP method is greatest. However, except for values close to the maximum, most of the response surface shows little difference between the methods. We also determined that the conditions resulting in the greatest prediction accuracy for BLUP occurred when genetic architecture consists solely of additive effects, and heritability is equal to one." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9.cover-source.jpg" />
    <title>Application of Response Surface Methods To Determine Conditions for Optimal Genomic Prediction | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings, {"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"PlR6Z5UnJOy6B9tNAboSVTAn8OhgI_oqFiOEAWQ2ygQ","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"486351","apath":"\/ggg\/7\/9\/3103.atom","pisa":"ggg;7\/9\/3103","ac":{"\/ggg\/7\/9\/3103.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/7\/9\/3103.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;7\/9\/3103"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;7\/9\/3103"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/7\/9\/3103.abstract":true,"\/content\/7\/9\/3103":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022FCUp+I2FYoE7aOVqDLrO7jvUSgbOYgxlxHDBvgT71kQgMeJRzBqM8FKfHylpVANmhh9CjAF7yialeP7hoAnbUxfszjEtXtucJUcX2V7QJQ+FRjfvaMgVp+8Xa69G4rxBAAjtTpwHsIJMd6kf5+Ch5OAE08AGKEdle5R8I+EFVSOQ9P6rX9T2\\\\\\\/rMjh8vASd+KXVH5TbKm1c+vyEQ3bOgXiX1dXQQO37f5g0WnFK7uEPr\\\\\\\/z2FkSr2WxHaqJIDLDTLSghNanqVfuPMJVKINzdnPOzEMSCaqeyfxJ9PhtQcwf8iIfvF2E2cAoA8PvOqxixzPQznZwKmoGdWsx4N5oExQmBrM\\\\\\\/DCbCqBaW7MSDhMNbTXy6cY6IpRfeeg5Q1qfBfITK5t\\\\\\\/akSub0L9AUf17G4ys+8Ep0ZOF0Fg8zFXCHZ+sSutb+ERKOgtXdCymKRJJ0GNHpXU+75Pi\\\\\\\/MO9t3\\\\\\\/VCOX35QjPkkbfzHhewzujshMCbP7QlZ2vwNCDNdFGsIERdOih\\\\\\\/YWBWH0y2\\\\\\\/cEKiEAxfErVf+9XL0bTwDsUZ6sckQTM6EZMI6EoxXAVn8rx\\\\\\\/Q3oDp8po9F6Gb4nWYcvu4H0MwmarfJdei5jpQ6F5pHIRND1qCyTJy6adfLlkdQr7nwiSWKHTgWRsg2nmVS3TNkd\\\\\\\/Qrnmsm03r7wORPLI7uYrpgBpVU+pQm26myLTp6p10dYdGPNdLZX2BBfNfdibIjPFYAtN+gYQ2Px\\\\\\\/PmXzV5tl2+jfZSPZKhSO2oH6Pa3wLzPMcjiM050LATzfqb0VQZapz3ZMpDq7Gfb7+MjrnSpqdg\\\\\\\/G\\\\\\\/vduqi0amLJq9j0c0TzGKnHPIo\\\\\\\/SVzWSoax6BsNzpz5rQYShLiHwEgy35DBao6GAtwqPPeRZHoKK6JsgADVj1\\\\\\\/40c\\\\\\\/8IH2xtjzIwXrCUffU72oo1p\\\\\\\/Pkqmsp3V6gazf86ABhb5n5QCyQMIelV5I0fljMR0yvezMdKv8oBv6RARmHmvCWdQEIXA3T6iH1XzWdXQX2oSjVNKaA+wdMFTmrYVrHIqnaCNEGwvJ20IKCl4lbI0pO\\\\\\\/aMWrXEeFbFKg5Ws35Z\\\\\\\/eZO+zZcqKyDVUR1\\\\\\\/a\\\\\\\/YjSdWkGsa+HY8R9RcKDL6\\\\\\\/MlDawvBy8lyWnpJW7DLr3zFUWN2Mjr89dYL6k0TInrd3i7Nz0KrjnLu+haeV8Ff+BLR66uEEClQpCsKT8q91bpbCc6JMnt0sWI5hRFU8cOR\\\\\\\/YmLvF36ejF4EMCTgBR75outbXbjNfBkCHw9\\\\\\\/Wqs0qmhri5mM\\\\\\\/iEYz0cvMb7Be\\\u0022,\\\u0022iv\\\u0022:\\\u00227PcAnf0660GlBXis58HgTw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022af12f72400f3a58a6fd43c71317695182f47e6ab9ddc9b28bb6c03c36c0526bf\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022Pt4iu\\\\\\\/QjwNX0hWBKhb4WMcjkbZcMpXLAZMJLNfUDQOpqTA6n1rcWvcf3T4YT0hzt9ydqEjHzbZ84g+MnnKLVglZ99iqq8wGcfM5lEAhfJ8ERC\\\\\\\/Mu3IoBOdDd4tsfx1wHbIVq4UatMiUn7nAflOxNwEW023whe2hL0i0g25DHaBUJa5NVyVwyVbXAsyCrUG23YFZ8eRcxkt4NlHOWfe3v4TCP\\\\\\\/6K7ORl7POJpOD70TJ2NV4igBCtQe+Sq9OTUMAmgfN7l2Ur9Yz\\\\\\\/SSt8t3QahDldQon9G0dw4Ww8UZkezWi4KYzjf9HK1KkbzC\\\\\\\/aaIVSbeGdaQpa7tJ34r33iaxeaVHeUVc5aN\\\\\\\/73eVzNnt3EdS0CMB8LHUhORBfwZEfGu6vS4s9\\\\\\\/B3o4fh14uov7iG6TiKGtP\\\\\\\/5bMDjK61FX9AnaCPQb8pOxa7EQLX+GohSjVO9mcRCf9T+QKTaeID5O7Lokh96nCCUWg3zSbS\\\\\\\/bMR+098qwicXbRXVDPzIfwe674WNgp4qJA28RllKeLpfW8Ez3GtKi4ee7gBbwYcldsh6MauXYMD2i7XZAj8+JEAJxzBnF2QgFEjnVFiwpoBJrtPun0MjlvL8LJvz9v1FsJM\\\\\\\/QPKZrGtD1eqsPFRTLo1ouq2HIbu0uimmUkmUYdj16uhtPEiVWOvjnK\\\\\\\/rXmbz1a3uUjeukK1vRr9P8KHl3F3EJYWrgDQQ96eF2iSBr9nZgVtHmNeT9dmG22+ihbiMo9ZR7\\\\\\\/s5G6uhIsqiuWkIOTz7bFvjldJhWyPaIPjK6jXnMr\\\\\\\/+X6w0OOrjinjLqzCboaXxC1DC+5mGpNuYsnpBzpqf6omfpF6fsFHFTc84Q3lTFT0jrnDe3Z+R0j0C+llGuMI95yGOh3NFy+tfAnp7juhxnp4kpfWZhJuJ46TCVKcyw1L3XB0DGLqDLq1tTiKYqpn6QhVUlo1xA2U4Eg135Lpv4Jn0K91gk5fgDv1J4CF4NkDKeHbgfGEfWC7HfvC1yE9ah9M8mJzHEFiM8ftT3AYykN17Pa4EeTpGABy+VHzrDADvKY8l+h2JNL8apZew2SzbVGYOXU+21Cqgr\\\\\\\/XECR9MpxwIxbS0VPoSHvCUpV+xKlkWWrDVd5Q17LyUTSOCqT2Dnn0VJqTDWpClbwY9zdqMr00N5ioesGw1UvBtLysSn4QWx1AcHcSl\\\\\\\/UprIAlCXUVUtuRezRATVY1TiQeg4dAxdNVosmjrnZX0IRt7nEdNJAr3xpJBM2sqDuz6u6wG1nh16+gcuoF1jdUHsPAU7eYm+UeCYqusm6Db7eAhZF1RkfXNNjmy3eAJ7VJKc0Rz9Uh7aqufM7M5OPBML3\\\\\\\/VubSORHYUJ1bomtM\\\\\\\/8aaGdDBGuc8dyzNa4QNXyGxcIcQTGnvSqXGdawep+7cf4l2bzdgRfzi4hbwcGKTI7k5qR3aROwRH0SZwZXItpwH53ZAjEP803R+OMjAkscNG7etAMJQOhkm75l\\\\\\\/Xih7vp3T95vf49284BQfDqEjvQbQELEyCs9Skyh9iIu3MiZ7Nudx\\\\\\\/mW1aqL3kRlXiZ2zIr6yisU5v5N8cWBNbKHlN2OlxYebKRqSuynRNptJ24YhEICl+cIO6ilkG1LeGkeQMRcasRkf9BJr0kOPvD96V4WEhWa4c22F6YkOM+mLolNTpEg7Jk0iFnoEzD90MPYrKMKOyosbAAR2S2yHcy2czI26ZdPISqrILs3fcgUky+3xEdBoRDblksuoDOXS5f2XRq8Hls39pN0v\\\\\\\/PlAPg\\\\\\\/Xh4XwOEwCUX06SjUkM3yNESkjoMSTXCLUZO853eLDa620dM1jZblswKKjwuKsHfMBsXyZNrf+CKq\\\\\\\/IMBD18SynVZs5eIPGZ\\\\\\\/B\\\\\\\/wM1QY+VThty328ilYTU55xLi9Tao9fBJsi17c2QhWdsm\\\\\\\/64eixj+XvQ6GKlxjur\\\\\\\/wbcGOpofBzFGzVNiALZTO0qFmI\\\\\\\/UynZZNjM9jYPD7CD1Oi+lkT58uM11LDQak\\\\\\\/1wSpQZD9paY5y+t9JkmHAJi\\\\\\\/\\\\\\\/p4ZDX76SO45rg+cWJArjHQDs6NiaU9nrrFmNfBKcYVBcKLH\\\\\\\/\\\\\\\/\\\\\\\/eeXEUZGqjKNnhpsmzcyzWI8\\\\\\\/23KOwtjN4RZDVtyyBEr70Sv3ovEDpuF9rwMXfMhbcYzDBoavo1G5dHALoZJKrvUFgxfOQI03GrLQN35S3iRiT\\\\\\\/qP8vz638QFUKgdPbFGS0QwT9soi1zBWMOrKEiBExVPZ+6fP0PFHOnJvqPl9L7L3M9D\\\\\\\/tq4BdBlEBvzs+uAvLgyIsdZ+fn0z8jrC58g6BwF8nwJD9WFXR32WqnJXVNbNGtf2HzxinD3OmikawmTmhDFV8aEdwwmOGNFnghlte2t8MkVYkRhqiG\\\\\\\/OsnILIRQEGb2pyriRnCzHOIa7ZRMjfqoZHJ5Sm6qoARqById\\\\\\\/yLMT10Jd77QIs4Xz4pkTgst+\\\\\\\/QMM14JVdT8C7CU7L47TEyaCpsYV\\\\\\\/xYcNK7rzUkKevbDN82+dJneE2JmuUO9YD+bKNVjefyAoZr9TCEQLULjgxzuAP4z2yHCtqIWxk0A1dXNQ6KkT5T0lOP1CYwszFyuX6p6UMyZDwCg\\\\\\\/+Y9Z0aXbrcmAKiExzi4qKInK9gwa\\\\\\\/MKaS8KVpQJWLd6Yl8SkmhqOFGFd2uSs8hmRyY5OWlD6K5C+UqKU\\\\\\\/LoKwGgQMnhsANSHW3MK+Z1ybA5oQmjUJrn6c3kNSGOBE04PawahiwWLKUtNMp4Nv1mA6pTRCsPaKmaqZHCjcM3UQwz3Cll8NJa+ydPywx5PEVC9o3mYFFnNpJjlKV8VIENiZx0oTvdMLjUhLumzZA9vG8vChm1Le5dC9T6CYtthedyiEgCiGV9AvjCDDwR98ed0pNTNdIrZwGhsNwevfuv0AeLjvo6LSB69d3OfhVXgu8bLfYrxrYZIONQ3iV3NK4OaBOKWRiV6leEOwrKcP9N2f8ArrXsSXXkd6kgxK4EqPODvfHDkIwn+lRhxEZryZBbOpcdm3yLDaUzpulY8NFakwXo49yx\\\\\\\/zEZ0ZHglOayZMB3dqGwIEo8Cql4LG+f8W4FT\\\\\\\/I0qS7NQMOQ2Lwpobnp80CTVrgWWE9QzTb3WTnPSJ9bR\\\\\\\/GQ4aAUb2Iin659jNV9DLc5qDdBTSucaACZnZxcUOotN5z5lxiQFs\\\\\\\/po6oAZxjFOMrmjLjXWORtAXnzG0ysTRBJoWi6ExIO0tdjCISEWnbMU7WKD618YKciqUrxilIhR2Myx2wlfZmdw7jKhYvoF7h0eelNreWhScVvRwXu\\\\\\\/3Rw1g4vZvm2p2qprWV88R+LFqrPb\\\\\\\/OZcn6Th5RXa+RLwLXss9b61aAP5N5tzS7jXXhJLZEvaV5nlFOh6fkajPLxjviOSmE394sxU\\\\\\\/3Apym1ESHqWM964oDOkOhpJkcxzb0o0wHSni5SoSCgAoaAi0Bob2DgH1+UchwTvScEh7XEpHQDp7m2NmUnSyu9uPxyMcvDtiRJVNuEihdoktA8DA8KhPE6XI9tuW9V6Jkg6dlmsiEhnriVg9EwardB5sEIgrGY+8mA0q9+uqeweXnZjaD0TssOxEu4dn7RHUbyNH2pK61D2m0erJw5FXuzrlwBJBaV0NqlH7TcREetoF8s9WmLFS4uaf1zlYYvVaEsJQF45BaiOP1Ho809rYx7P1nFsclDs0brwhtzacBu6FZGxNoSa5XZfQeQXfnxqkpWv1q8UrlsarxjEPamoKQ6Z6Tm6IUttfJrq2wgJ3BzWUVJkwdy\\\\\\\/LiRPcnNjpV3MfXsoc6s47kITYdUE7aWVJPLbYnAAC2IYqOsup1zpqbDfuCehU3nyEhtW86o2xQNxNWnUigbFsLQsrweQmw2DJn3Ak0lkQHqFsU2p8r5\\\\\\\/NINkPX4lI6castr6QpupTu\\\\\\\/R+rcNuO6ap1HB7iu6zzstb2Nxl81XShRKNi65rXozI+pUgTuL1uNq00HdjbhSyS+CAzvbTAZkVwOFPLAhl\\\\\\\/lLIHRatuJG6bNZgSsjHWYVP8BG\\\\\\\/gpj\\\\\\\/tNPUgrvc40r7WN3fMw2mr+qWQsQCRfSia4eWgHf29MD7ZnmWHGQUGM7+QScpZ+pRhHibsZhov7xAVXeUz7k2i5xEdWL9+eIz3ZhkZenrYPNtpChWdDgLkPOJLCcqTAlPxsEGUm2TeSTCQXI+rXyo68yMLMxsJ+Nk7cfQBLhkRXSIHmC4sR4VNFsygdhIoI\\\\\\\/ZWPEWfAWKuZTPVm9yiMjX\\\\\\\/JdrBKxgSsYTohlkfN2jVdBOwFNt2iY\\\\\\\/VdIfqGOnZIXy\\\\\\\/4IszDhAGkR+2uBh6ayJNoy4hKWWQHStS+v2pktUhnDmTytT8WeYl\\\\\\\/wq\\\\\\\/GhnPlIvWQ4ocUXMZHAbU6xZRzdbijAvvV46MZukvlx6B+4K0j53ppUX9oleNWuS\\\\\\\/K8NtjTSBT5m7\\\\\\\/sKAhm5zkdzRundZW9+WMX7q6UqAeVPU7JQc6MuJY\\\\\\\/u2Uqn3Eu+BssG0n+Uz+LBS7HG0zUIINnp6RD4nZxrda3rU4qyj2+z9nsF9094NrLAx4Hm8B7r12UNLNLHW7fKCbpG3gU8zNH\\\\\\\/T1W5b8IjIizRQwQTiCiLgVoL0p8o2Euv4DcggKW3BPMWiusvTFffGu4sgfQyHNUGVUPY63SbpwSAFGn8cpqxQ7Q+6VixKYuBKR1RfqV3J7h0zF+ala5407lSC05EnfX+GEWEgyiURLsRnbD+TZM3wyq5yohkvBGiq7TsJJhSKrBWrMTTER5\\\\\\\/Jirl2IHbK8w9wX4P7LzJk0DEQPcz1e7S7qCQoSX1rv\\\\\\\/2tv7H50csShLZuhNZsBWxQm3z8qEpvVfbhH0VpzaLo3BLDoCJXqlS4+FmOl3gff6rfJPy\\\\\\\/jkFDD0YIZshTTGSB6bBaHLNbHBKO8srmqezHcDn+2\\\\\\\/AZZ+rTOdfInw1n6N6i8sLF27bMpVXI9TKCdHNMP7PdipFY494nqV58MX4JPXgAyZUzQDwB4sHV3E\\\\\\\/UT0jC7L9O3L4ahFS82m8EKnUMUl8NbGQYw6heK2y0xok4aEbq7EuTB32GhymgodsgsDD3+PodEHo3orK1jfq1dE4CYVNNMt4BcX12wOLTFkvHVGA7+QmtXXCrBCQfOyBmza0zBM9shcQsJvQ4l842M3jgqZLxjo5h9OVkQF\\\\\\\/FylUk3sR4L2nK7+LMpDwcMHU4+eCdd59x1ECjr0ckX5XxJ\\\\\\\/qYrzBSALB6mwFbDVlm8oUf\\\\\\\/31jzVo3IzZmRjmONH4b5G5yXRALBACI87QBZk57QoIW3Eho5JySYM0P69zy7Eh5Oo+rxLyCTUirAweFZh+RK2ZRhEmzGNXSSRYsK29p6W+jbKdn5YlBjgDJrYJLcpbcjoHPijh3OzUO6EYj5SlNDXmgYokYmdiY1LmerBtCJa8I30sT1EpWxWLuBrFgQ9dT5kvkwt+fxIZSc5PPcSVVC7Ed420UH3B2c9kuuEuckEjFYZ57BnSvWcj3PvWq26hGuhgl3ty6\\\\\\\/tMvPC1Sd8bQfWjfJrFfVZuOgPgtXwuW9kW7uIa7gJPwDjQF9VmgByHCJhwrxwn0tYtTNYYqEg\\\\\\\/5+IPaJx\\\\\\\/D6gRGKHvVSaTjcqMjuucdSRoZvF82u9yVwyrtxKczWfxLj6ERJQkDV\\\\\\\/p\\\\\\\/WI6IAoOBiC1\\\\\\\/TygHRCEeWSooSEY\\\\\\\/uZ\\\\\\\/tOFbQxc1Xg\\\\\\\/55+wo889oe64QSoETLITx1jihDLo5Hikdc5iX3ij1V2lukTMSCPsQC2WzjoLdJMCNQA9lJlid7Lg3SeUsut8yadaAWxOb+IsDv5mz3XLMWe+LCav1A8J9hYHIdkZTqTnqUycdrp0IZ9CzL0EisBrISwhggePxkXU3CwYMdb\\\\\\\/VwqjJawDBQiKnRarJTEb\\\\\\\/p2qGgzXxkAT8Nru8DChpRYmaki0N4fQ9uK6eg+2\\\\\\\/ZUedz5453aMcLnZV6xXfDoUn9tX\\\\\\\/4P2irHsvK4UhPQLpJtTgdno1wjg054ujjRltf98G6wpJ+BavNWdJlUdsu23+0GVhDsMGpHSMyRwsk\\\\\\\/hqRyBcUAd6Iu6tLTDKw2yCfnoNAEDAvgCH8EKkZIrUb2b8VZ9\\\\\\\/8+gLpimD0LVhkyhvzdznBsnuaMhpG1FRDDT3pKMfBJYVU8iUuqE6R6DKoeK4tE3e6qj\\\\\\\/Q4GeMJAfo80AC31YHA8WStDTEKXhdej2DRmA6CFWsh3M+AwyAoq\\\\\\\/5djAPpo1Ip+PRtElwnNOxOh8GygPv6nNgehI5ql7kZzVdhdnCPhtLc2W0iMMDa\\\\\\\/FKdcWqUGVoRYZWPs1GAKfM7nXO11hMzNbsIzWRUz\\\\\\\/Mq8ob1jw69IrpxnhTXLVEJsVWwK4B7gGVRCoS9N87FSPPLZzkeUvIFMaPBzQ7SuQQYo8VInYkmGlGXVSqjODV9pqcHHLprlOh3ekBfhfylAyH67O4I7U5DfhWzVwrG1sJzOJQ9tbPgJRhNwhqXlSLz1RUyhtGihAeNreSyuOcbqPBCuZm1OrkAKaImPaA3craRQjSdNkP8K2e5kRMgS3NJbRqoLyNbpwF7qS2hs4V2NXZwFRjbsBowpbwoE5HNahaNkNApBjF2\\\\\\\/bV6LpEtXlcBokZkNUq56qx5P1sgDd5sNSajpR7DoEUIarUZ+govO0WppVZbquHqSM1em0nilX+7TBrkbhturJP3x4AXzJuGaVRR4dUjNSxEH6LQcXFBdqdJzIiUmPF+LlADCKe12xvUpo1thPqDsqIk47k38rN2p6gNaldWz2yMSrwdthg+hHGJ2QO8xWhcTO9sjo3ucZQSsJUftcG+PLy3013GIH6DWQyT87X5BNGxIder\\\\\\\/WrTL3JIS1ypkdKFRx\\\\\\\/6wxd+LnIc9zQZyD3xKnMSCpd7x4grsKoj2AKZy9L3nQtOLA7rZSa6ly2fObkykS6liUWC47toQjanPXjLl79dyXL8a2V\\\\\\\/Crqy8Pu3hHVwrHVc0zeLaqv4GK4fOVwfM5hUpLImYn45D4CQNLJRT53j7COSjLE4HLP\\\\\\\/oV1pRCujvRJN2eGPJgEvGH1hAYWe0x7pZS41PD3rP1RrpBXxeGn7sXBdV\\\\\\\/Aa2XFK7Ge5A3Om3zUuWHU\\\\\\\/dJnVFgxLl57Q\\\\\\\/JSLzGaptCC1y4u+cG5URIl\\\\\\\/Ijm8OEN7uladiyZXYVQpgh\\\\\\\/GVe\\\\\\\/JygcnqQdVzweESfTEu0vdO6nBKrSna4vHh1JVYrL2xgZdqbXwBHv5VjXb0iRuvp8XEDPU50wsbhyPY5gNGs5Xe1cb5WydETFfVUjibxp3+mEknSI5wpuZL6Uy5cxnsGJXzRHM6wpXlG+3vFvTzIdyPEG29P8\\\\\\\/BYBRvow10SYVRCUNzHwEA4K\\\\\\\/6UQghL3dMnLnIyoGUgKU7UVGTD3BMMxPOZTKsr\\\\\\\/3AF5PDVBqPoKbJuXo858Wz0t0pi07\\\\\\\/zntuUJBwL9GSCRLb4I6kDrolFiIcyH+hW13T80ihYlPA7cMCGXwrd2FYrTEOx3sewoCilg1wincAcFnRIWZpdnikFVNjc3GEc9U5g4V9Gwh01vWgjfUZKcddfYN60JYcVArWd+EOkgH6p1qJUmHBmewXrj2lr0rIoba5vNGkNNVBEKyUS8BcMFTfPVQ6oqoIhli4dAu72MAZ3EXG9i9c76vTs\\\\\\\/myAWqV0\\\\\\\/E7P8vaSlS7mbNuKqoQqlePBwBarEYaM+juM9Mb49JfPafhzD8lKPLVZLQjN7wG0ziixPaRP31bo6nYkuxN26gqKBAevtXAX+mUb6XqmxsZTKr8vFWACLbpM4VwNeAbgvT2bvBYOaa0eF4h1655J7dsZTmMsVPHLAlNimX\\\\\\\/K9dmVvtLP9R+hZOWazzGsYIiloXBPClcX2YwrtwEYLrZML8itsRGWuyo4+Fyip6Y4zGWfKD7T+p7Tz7Qiw6p5dJNC\\\\\\\/wTOhEyJ0oXW5AJ8BzrgSbwGVsvisosoSwUk6t5sprP0hoj\\\\\\\/D3QIwA7vkD8hUaQ2wfi6EUUWqB4DMPbIzGKZfbCNsZ4ru5WPyiGbJevFUfiIA+fTUVMeo4inJuVP5ISu2QxASCxw7H\\\\\\\/FCnn\\\\\\\/ZuER49JtqGEl4w2KtK\\\\\\\/DNkAnv3ZF2j42dd7k3fAu4Q1Treh1zX\\\\\\\/hKl9T8WcE\\\\\\\/eNFMDneX+Y8cwCabLlaFH71hxZlwwcJwkyn5+Y4U5Om5dR9Z\\\\\\\/XlDvIf1CmvMMo3O\\\\\\\/2GiCh1r6MjnFCEt6ZERAl1buNtiHOyjgadQuDQoVXkoKL69ACeR8dJMVuzVUZBOBJGBSx+aS+uLETYZA46JwhQZqw8uAwSQkvh8Oj\\\\\\\/mTbKe\\\\\\\/KQF1sfIbn5j17PfW\\\\\\\/0kY7WwbPMWYrESXNBQsJf6TilD2yAQoWgpckA0ZNlqxMG3Fn1ouGX2VVlC7znPGLuiDYiTXmjmacVGwW+psQXzN5Vst56TDtLFG1F0IWQHgtFfRxrzn0lq\\\\\\\/UpnkJeLY0x38+ZhVGtL83utBofQMMGS7ms7+2cKS65N7sXBAqvVzNnQ4quR09IDRoT1A9zBfRk8tt5ke\\\\\\\/9ajFKPCyfBlORYWLgaOVV6w0xUVSdGrL4lJl5bBDo5j8CynKO\\\\\\\/L\\\\\\\/FaRpktb9dTls9pQChR727hqLqL0hQYZl\\\\\\\/Xr+6T08NzxoD+wUngwLzpE9FTvFPig9ztcxypebx+Nm\\\\\\\/mXENmC4rI6+WFDaQ6I3EkP1c\\\\\\\/nkh7kRwSebtutpJkkwZml2uEVA7CVj9KQcAQF34W2p0qhJzLhBTJu8b4VQW+XoqSQ9S\\\\\\\/gFiv8C4hi6Oju9PKyI9laA6n8+3hevuMTXcYjW65BwsQo4sbKlT6kPwL5vUfgFlBkkBRvOPzdMo5HpudfIFERKVNSzn6SZd5WIYfTwC\\\\\\\/Vbi8+x9\\\\\\\/1NzPxv12L4IxFwpYnjB3OCayDcDznVKiLD1WZkA9DdXjw2xeoTt2mvSiWDg9LcK967UxDmZM9b0WmHa8IUUuf8unB58WC7VybJFsHv0IOdvGdaNwSSmZkaPLNkZx\\\\\\\/Zy1YaOIyn3MeO4dCRbdmE845KedzckzlNO9eqYZ4CHPefvzgz65elZxo3ovKYCk426FmUZQtbyFahJScgtk1jswcJW0Dm\\\\\\\/56jBfKnXnlbxg150\\\\\\\/ZJ6H9qO8p+eyKr9FaHPMxtL+Xnvno2ma\\\\\\\/h\\\\\\\/LH2G2UCjnjyE\\\\\\\/THAUx7mrB2pj1xNjD8mH5Phnt5sbocfuOMY1q6HSBm60N6YXzbLL+GhP7NitmAihFo2QsYRCh7OgW9JjgpDcK2Sg5o83+G0pvmhkeN0OR0YO3kIUMkTpLYY\\\\\\\/\\\\\\\/y\\\\\\\/oFhyhsctZPmZM4kYjvH6ZQsqxREwFklHlxNlGM30JDO6EPU4G\\\\\\\/syBJ2HBJdaWGzUf7MSddtHClG+QtnTrnMelTvLzrlmBpB5h81a8NZ7wxu0lZX7FxQGLNbw8708xWQNW08bWG4FUIyrr0dmjHpoF5xfbKhtNTF7Kg7JgUAylB9EWysHGeE\\\\\\\/04x976ImI9qIcrbTKgV3UfG+Ap58ycLEc8QYBBn7vZObpiZT1FZsv9dHgnNsXgdZAFSGgVYzDWFlMMo7lmMGYjcirPPiOe6RTj+hfoWkJx96OVSqbnqcwb03PsqJFxsJ85SH4WYWYRT0066D\\\\\\\/+olMPFnCmdl6sShY26EwHJ9SEV\\\\\\\/LuJK70PRIaDfobX4h2Q1qIHKDVCNGQ+jyJcB7TUDlfj3Krh4qHv0Ia0aWPQIJpacghF1IFz3Bb\\\\\\\/3rQAAI4uiR7pHSX\\\\\\\/k7b7f0Cw7MPXSatvTXQnWaieM1KCq7wy2uwDdM7a5C05rqWLILWj0ii3yJo0wNuzKiHOJ0u3htAGl+X5duvgSXvFlytqPQuzlV\\\\\\\/GOefNsPI1J0rfKol9ePtzqGhbPjeMrQ924y68hljxio68lHlkaQoH2F1B\\\\\\\/d4fNSvKLmRsCKJfuuPKLVCkK9JXWcKTAJQRp8xbgdSPoEEb7dOFc6XNqIhRFB9tV1Tnow6P7GYoAZXIKpD9BsaFJrg+cKyQZDZNfxRPsc63eWYX4TqzCcRjEabgHrFnKRW0OZgAilQiV0wXyWujoYcVz9NBfiPg9Wer\\\\\\\/NV0r0mxqgzW3oBKDwJEGJsQNrWbP3PkhzLRKX4EiHLhaZamKtblZ03G8\\\\\\\/rhv8jmwRz2GobMGwS6UW6uzO7bJtk9OQQQN3SJbbAm4HS+OuZ4cbI\\\\\\\/EzZ0tq\\\\\\\/nu49\\\\\\\/AcrvLQuz6OEjh652bPQDppQwH4ubHfEOq+\\\\\\\/zup\\\\\\\/VS7\\\\\\\/LEo4CHVfPPrtf46soNJLJaSxybvNtcTct6\\\\\\\/EpZVAMo7TD1mGJ8Ln3XgOkdJaYEEMjMsaK0IdavX+Ho5hLhWJdgtiIkKTRkJ8djadJOJkdFbscTUGJOsL4OeglnIWMx2FwM80X4oXXITJ8aZ88giFTFgTshmvOBQczkR7kB4GWTCOtgvtLqiQM+1HRd81UNXclFDngH9wlhPXwWAF5G9ZzN2W0qzS59UtRbO\\\\\\\/1Qa5EZQqpW3XEreQoZ6la4e0tC\\\\\\\/q1JYc91tf4dFcu8LvocnQ1uscpNbvZmgF4wKCp5yoZYtNSjlcIeYxJJVai\\\\\\\/omYwIqsuyI0reNS7Tv12OlMINeMmJ4EzMYVQtBvjF7nY98JeN9PtvPdOoTbsJ5fnYQ7R6Qa07H7EjODtz6JTZZXLCaBV5FWEie+kZLX8xH05cdmq8h5i049k0fRk6Blun3lcZlzJGmsXh2Yubs5l\\\\\\\/yq6I0aiXHN3vUkGHAm3BvQPjytXEqH2EYRofUZDLxpYIiUD8UZQ6cZIGYq8ssbq8af9hYvtt7A3bs4N72iTItK4\\\\\\\/sKQxEpyhjKlj2xdAwzxmnlu256NwEj0e+Ci2HlFAkvRZwlAGWtxeHk2dP66M4ImWIzgHH2fnfy27NnhhtCr5N8PG7n8a3X3NiABctYEgmp8uAYQ81k0lGJkIGYxR\\\\\\\/78EGXtctoVbOVoNiSb4NrzsGbifKOAlpd2TXp3AjW\\\\\\\/DPrgbBJSm0K24YgFqabEgl+MYkPIbAqw7IntmCQLfyI1M0vNgzAcJ8qVstfEnk\\\\\\\/UrXRc\\\\\\\/g7JcoDTVhHnkaK7WO+J3Zgg47yf0eYzrxtzJ8lbTsG3Fll9lBNJdGmd01osh9cJjTMZ8KOKR\\\\\\\/fE2Uw1ud7y2DNHFquVqqkahU4B90pii00Hf\\\\\\\/4nOKVDgQDk8HfAZ\\\\\\\/rDTv2Z86MYIK\\\\\\\/TxSqseOxbwytgbolJWnX9v0nuse1Xi76GcrVHl\\\\\\\/Ot\\\\\\\/JE+TaE9q5F2Br8huWzNHNVKSPrdwF435C1bxqWKGRiiWDWZjDe29RpRbAt1xIeGh28QIx1LtNPhMiKNwJUEYiT3q2lwmNYo\\\\\\\/dvPs1yR4rJHK6DxtxDkXrZvopNMTwKEmYIlU2OR669YNizP6hpPqaXqne6yoTyhsXidebq9+fRnA25Ak6TlywuylM+lFLshE8DJZEAbv4hGMEmICTZpiPJ38ub7dRDZs8vcLYHlCdukoUDp9DzyWWAEsYB+vXqccet1i9Vq4PWDFdaDz9v1ISYTv71coGf6AFSEtYl+d9RvyZ0\\\\\\\/pM\\\\\\\/KBoU96xMGTtzD4si9pgZhOfTZibnHm9IHQREiItdn7GYv9HTTXyrFuOJN2uVteFBgNifcQnKuwFnxOLxnZJD47yyewBAfJl\\\\\\\/JMxYAbeDqq0be+66a\\\\\\\/fjCfOW\\\\\\\/5gMrNEb1jAT9oz21TildzgTaHRuM0YhLt12OZCVPuZAUuwIWDIi7U6cF\\\\\\\/PtSm4LSrIPyzlLo9fz1RHxr9gyBYhYkvPDdEhUXgfH1+7+\\\\\\\/bJK\\\\\\\/b241guQWMOfqncrBGbhvF7kkQLbu8+MEgS4OsMy5qomaOa3I8hRv+OCby6wnTWg0DcN\\\\\\\/AQYGUknquaFW7xvLwyjSRb3xfIbSC6egHDj5TvcH6rrlOuP+r1Q6OaLt4d5wm\\\\\\\/wgcZ0a88D2PnlAvqHo77dttJPO8uaizbcqXU4IiRiYxuwT0ye633wmoJNK2tCJOPDgRVr67FrGHkSxiDIDup4L8C8ozINT7aHyzUEU0Kzp59SQgZOAWuA4sg4NlzRbGlZBwikoEDyCk8TLQKr4f9pdqgv+44cnVVnoChaDZhNyhNtl59eRazO21x4Jz94NuJWbdm3Ifv6aGyMgfgUq1jUsrtfewHumTN0lQ9pGdeJ8NZz2CwG8TFZpFLl0ihOpHP2\\\\\\\/S7x3bhdnzvVoWEcpuJ5mvgIXJFNqXzsRdDbjhI1hTMeGv9L1maCnRUBEsNO4uPMnP\\\\\\\/vO4M8rfRQz9nC6c\\\\\\\/fbjIdYfxfSEXZVwMUt9OUUxzK6BNUrobAPBiWHTsSJ9YGUXjvIMiNEPNtcJuAQ7sD89WxQOQ5GIYxPt7dKE2SNqIGFaoprX+8NPIGWIqVOXw3pAigacqrEntlrVpVDLoLsd9yJfVLXitnjtcJNhgN9GgP+X5jIOjzqyKheXSNTyAyxX0JQPtb38zDVj6B89IMOhbXudCWzCcXd\\\\\\\/Cxifpk4uaIeQSNSIbXLDL0zt9mMgTFWGBiyfcWv2SR\\\\\\\/Rj1IhGdsyRB4Hwk7uXYMmukhVCQXJT+QBzYQTIh+vWKNvqSVSCghptzytE+T\\\\\\\/gfjGxIwndt1jLL30wWixy28ycJjqL4CQQtWfFBvZX7WtGNJCZi9xW8DMfPh\\\\\\\/ynbLTmfdPfgBlnfEbGFSApZW9frYTuvwNmmE+kIqbL+R5xN9v\\\\\\\/qE+c+vyctaleF5goocTJfjvXWf\\\\\\\/NqL\\\\\\\/uXfYXMBQ7SsLEnm4iuCzwNjeaMxlnYPrpIA3ax2bdRHSfh9\\\\\\\/UqeRcOT1huQ5rG7bBFAGVM6y2tWHDVcA1kqwKakJ9GtY9yeYDiQUpI6GF5d9Z6dP0Mb5CxSGoW5xeI6NpjFB\\\\\\\/fknRBLPHsrwL8a+bo9sNcAkpFCObSpMfqYd9a6auQ6uaBXGfZh+ijBvqjdjF9dDfzSvVr8chTW5bxn0ksnsVN9gk4hLTG98H9G6BbQfwi2F1r4ETtgf3WpNUtM7DF2jCosVcCbLrUqXt4LEjwG3h5NUIJ5Tj3zR8x8mP7Uc\\\\\\\/uoH\\\\\\\/W9exoVBsIXNCcOYU8uHsmMSsLpRnSTcF0B3zp2GVunXj2Sl8WJKC3Bv9kdYGE4\\\\\\\/Hzfp16uBw1zMskzS3tHcPJylX1u2j8N7LuoiOUIcgytuF\\\\\\\/KzWNzy8wWKa3qBFpZbyw\\\\\\\/P2wsE2JhQyzFtDfMhjgegB2SLHyJ7ubSuK2iHX3m5U+jQlROuz4M936a11CeMvMjj6qTqC+\\\\\\\/pC7KCI\\\\\\\/KzIJtRYMW\\\\\\\/AdGYzt0fr6hZsxasNWG75Gq4Hh\\\\\\\/v4YFjnGIKzSS5ApdKZw3yoX2wlURI5grzBkfttx4vsQd9QZQvMIA3Tmd9lulTfi7icG9ClwflpllJXyDRQbVWDJgRq1yx5yPDx\\\\\\\/LCX4mX7ZLswVVDeb620XqDVqaGpYOwSpEd89QORCkz\\\\\\\/EFzitzZnL4e4qfvPPn9JbtC1jHpDVL2noJL8tgNLS3NiKQ1gekWHAz3oowRb+IxAiAnuJYalCkp5hLar74Ujf4d2YllRmVjolI7cqLdYCFwF2YEOCAsdjUsu3mdSyPcYR6q4lP2c9LVFk8qeTOujg3lcBTmuXGCioaZ4kPF7TD5qa20BDzhjnBfOtfpgDsFqkmX5BiMqzdkZUujqaWs7G6g34kPjK0HuuaQjcIYZgCjvNSSAwFSN6hLmFhF88BEA4ZSsCHVi5Z+IsY1EkuvMMS7iBnhs6yNCGUYMxZKeoLM6HT97b6L8vxfGhxCAy31M89fWJ4mtIMSLG7d9hMH\\\\\\\/05kyV9tXxB3L6\\\\\\\/sVTiQGZ4jmFbgBDocwthTEZ5+4kWP62Ilz67OplRsHAXbc5rz0\\\\\\\/+pTQ1ZAcQiBWtsMBkY+A1xltbKvBwKsVUKyO9r96VoU8gwll0q8BVdSf1HihwcvPMbgiKEqUXnCoZ8vpDv7f8MoPHWuCwNRtao1YvpXSvuFvr92UwN0FkqD7L0invlMu\\\\\\\/QM82V6OwrqhPjszKe\\\\\\\/LZim0YwPRi67YG24em\\\\\\\/R9fspH6l7Tuky+qSEgCZyYIWclBfKysMjIrkXB8SfSfxTpfnufFxEuHGSpsmAcSFBrjEkSM7MOONKZDMN0GUviGy+pq\\\\\\\/HuyqgvlKwfqJoqqgPj+4+2V6PqEOOPQU9MQ6RTJLLPf7mRhIWMWn5MCNiv+UZYWpNR4cTh0HloqVJN3EfJmLwRrbNq26Fd6y294VytaLLO+ZfrATOUSG7\\\\\\\/+AUt6xhecMmsC0wUOpC8oaqDy1wR7vuR2DYEcnxEYJ4q4B3G5bAO\\\\\\\/30JuKs638tj1+TitMysIgP0v7sK0AQ5176Ew73CZ3ffAAEVxnUTIZhySB67pUt\\\\\\\/3Ju+urngbSZR170zIE5IuWxGaCEB0bPXfXnf8MUKB8Qj3Nb1qVo1e3wtwWSuPTiBMGhNWwfioRh8NvNJSq22xUeNR1sy2p5q4n5ciiOhH0\\\\\\\/O30kV2QXN1KjCUEQ\\\\\\\/80p8Jw3iQOAtGItVPqsLCi6YYOve+EsdPiUGitIQL8PBe5iV9bYr3RnIxsHkGgZGRLTrWqWJHGYg2a1TSiv9FJIKiCxKwEgRm4xBfOXtMF1vd+IqGve4UZQHSTZHizqYpR+FAF6B5gzf897i+TKzuDkFh8ZKgWni\\\\\\\/5sD9L2Y0MVSd32yHfToc8UUu6cT\\\\\\\/0geq8VM17duujGb+g9q6zB0iLNK0ZlVQUQOWclxoxbwgDasd6\\\\\\\/FwKvUAO0tdQ5T3leTSrdfcIMqBWEYwvsyXPpNQQNpDVIitKHyvS22OT+7o+SBEOlLtk8pLhLGu254wKpojJq3C42lHfC6FmPSD47gYGCo2ePKY077ssd6q2lO\\\\\\\/N2qax0uBv6joup09W91t+Z10GhSZPeubzrfMpVBSWwX89xntYPDanev4AkgECZm6\\\\\\\/BkVXrCwYxS4FzOy1A1OAZV3adcf\\\\\\\/KKl8l2+6p9WaQcckjQ\\\\\\\/J5bnUWr2lIDaRzYRkmzGNYYi5zEWCKp0WoieZ0Cb2sr5J2Spr\\\\\\\/HSUnqAmXxfM3Q+zxjES+TnChOQ1S3LSL2CUbA0vROpv9kLV8am4s7tF4AjgTBnaKYq4LlJoBEf5HhcWZK5FAtvcrx5KnLX1LPrI\\\\\\\/rj75S\\\\\\\/kyXxiNiMDMEwU9Ppp9ql8iRRiK6+XxFseil2nJAG8z56IO93S5L8sPROIPphNVXGkfQC+YUPIN1rHoVA6udZ8x8\\\\\\\/LAEA5ts4c+PA495AmCXxxiTkjQ+P\\\\\\\/5VyE1jAGMfq9lQ7SphFb7kPMY85PeSRCUh\\\\\\\/aE941m7nn5Q2S7KSLQhXBJGIjBEndo8Gd8P6vdArQptib+ZhEy8\\\\\\\/gMI6nS9HXquorFlM9D6TpMi2AqS6rxg7auPxoqYxBZ69Hc2wKuRfCMuT0DqK7JbWX8WRpCehA2SgRUT2MlG7dMFIJNJpFN4akaLCh72hnSyuDdnFlzP6Cmly4Map6PLSjMEkxizGQ+mzfTRpir2Qjxd7+Hozkyi4MJuOOQ7GF2AKXGhFa8S1pZMxJ3dg4fxQ\\\\\\\/jy7dBosVucbKU2eS6oXkcgoXTPv23Cr7QpeaTgFiIgfhkFNLX7Fg0SpHgiGf3PinJsfXG9JMEeKCjoNrULIaLOuRJMwOjhW7YbeefqCkygB\\\\\\\/qOP9ieC0RXLk5c6mj2RO5stqxuOVnHtjSoF4kKaeT1nXp1Cp2bENdCFrLc5uaowFVmwvTwJDt1Wh6y3u4XnUHWUH106Smz8Uq2k5aKwbKPxIKSBPzPBnWU71TMv9l\\\\\\\/CRLgqbtrjkeSzEJa4+9P11efpm7MTWWfltUE1cjajTSSSiKtbOCn96cC6dbI0S0mSVH+8MIxQ4JnyHfG3YsH9\\\\\\\/X01BTEYUxpGHYahTHbdXf47iA2XistSYRjt50Utpfs4Y3sJyr0+aecaU2Z9J\\\\\\\/QZ0o3Viw3naRfAEoXkSynymVoINfqUpnVspn3MonXOlYIzjpxTfaPfTDkoyL2M1Ymei9CIYWEsLvi2ilts4YBw4g3PyRTozwcMF5PI0dO6\\\\\\\/Hk7GLaE5IvEWnJ1oVkX3jRxCmzIdtGGAVr+ke1o+Lfjzy7x3ukDOgL8ZvMYFbbivTEHGsKg7umUddQJ9ZpGNktul7BsK2hebQ5iZFWXYoD833Z39cOnQKa8qPV8VGPgavNweAT4XPP0fMf6V0ma5zG3aPTps\\\\\\\/s4xDZ2QWxWhKvdofqfam8NVlMCrwxTKtgnhMDutt47rvdf1InD4ZA\\\\\\\/PjN2NtlrBagLZoGzWmG1BFiKsJGm0UWm3kC28UCeRjCdK1Ag+y6Hg2kCAecL21LQvXFTuy+gDxjCUzSw1M6vLcXyHudjnOPSUz9N2qUDxiKs9anZpAaYCc39IfXCpilsAQBVp0aa1ringpTLSLFv0Zsho7KWaCbg0F+5eW76dJvcosG1fXuOPqzibvvwpcw6V6PWiy3ACAHOyiZ8L\\\\\\\/FzL+Jaipy8cdzg+qRz\\\\\\\/sb6VG461dYDl9mAE7qFQ4\\\\\\\/72MHEbYYJyVggKUfLU80TWUthL\\\\\\\/wgTsyRYNgH3rZtQ2RgDbsvskKjdDek7avJvL4k+3\\\\\\\/px5ChUxUkolr\\\\\\\/taFwNxqw8RbZvP8k5+PErU7j5A0e0DrFk+8bzGb+RhUmzkwxBCo6QwB67r\\\\\\\/8WaVoM4JYmi3lmONXHkBHLPkdYKWg59S1aHfCmPh6AJ7aVGISLlYBVTIstCfxMGwNaZZ5ne+Ehmy+Q5M9puytOIO\\\\\\\/SjJbFJc2Q0CH3HPqz9afIWr5rYPNrFh518GQ07v6zWFJF68LbARUymhYnS7xtuClVunzyvIEeYw1T5GPL0qsuBIOKa4uAtMxpGHCagW\\\\\\\/V92ZEL5F9LWwAHcWsDBA6BDHR8\\\\\\\/JSuX2Nbu9vVyu5IFpu5G+dY74YqQKPFlqhPPd5F\\\\\\\/shOlCoOkO6757ROIRDalY\\\\\\\/q5WiyQSHIimFjoW2CKyymD5orfUPzKxZdTOVtmXztUnkBgmZ7jqUiMeVYe7pLRKhhI\\\\\\\/nqFcMBkW4RuOvyNrC2ZtF2HQUggbCZEKTS9Oy49evUAlCkkjPj4D0jAzCBGGYyJbu9zqM\\\\\\\/f7LzrOqSXQseAnDC+5m8K1nrNNZ7z1R8wRKoNQaTy5w3WkIVbSqoWNY12ePfTUq01HAIqKkxWeUY8YXxhc5ILF2x4jktS+YPtWZwXR9H4Kl1R5AYfiAESxSBZgf4gdqr+D3Iw1B1y0wCwmEP\\\\\\\/ReIIMcUJBR8vEdgLWolihVbjUuFHJCAliSi+VFf0iedhZZT3eDVDEefMzX9Q+i8xdJMYAACkv8Hyg\\\\\\\/VOLbFE2zsXieqk5AKWt\\\\\\\/rWr6MoHNJxFsyI7npdcg54HaU7jGqYMozFAYDu47hobT7OzzWKS4Iv\\\\\\\/ScSpiFZcUug+hYGpHDHzyjn5tUvWrQsUgsmm\\\\\\\/O2ZSoO8VPrepTAi+utXS+7Y83S9D4ECdS06gt\\\\\\\/AyQBYl9cOnFDVRGWIjnkskz5lD8v\\\\\\\/Mql9GIZ5gHB1dQQ7l1MR9pHJhywMu23+N4SfrElJorlxwGj1FPPMQl2NzzEOH2h5EqQvVDoy4CXLxQ6YOcdaQ7dbeg1ylk\\\\\\\/hbFrI3aNds0SOQf5lf\\\\\\\/AfYWcUKvNPznnUlj1U3tD\\\\\\\/ROCuswoL2ujuv0yTvFSePKW+T6byFlgipKi0avZk9ubgNpVjx\\\\\\\/73rwbyvs6MMoJrkOFKjs7t2kKN8cT5s+ks0gYaWONv3aSw5mUfIDDOBmP+0x4AU+ObcP6gr9LGWp1GtCoexQu7dFNVsbBncItCSBcYeWpy1xd\\\\\\\/1j94oqTeA6fvChG9jo3xYkdV6cOFYNWjuJ4UWZj4eMRNmVgAJ4XHiQnw05yXuhCxYnmofCpOVrV8kTDRDLOXznQ4aY+fZzpExjDKt+suTr+ymeJZbIe7yT\\\\\\\/q1HITvE9wX9kKTr7QZtW\\\\\\\/vndXH2K1Iee6BKD7MLF4wGfktobPPYzy81WdP7dY8fYrj1ZmX6fxDkXTSSu1sDQn9L2NpELc0ykARGLDgCy5d2gKWb7pro1MzYfBNQeeakws+JvKHRFqSs0hqIGUY5vJuLvbINvCY87UlBoirOXmXe1Fe7uEDk4vTtEpsuWGSQEeejg+NgP\\\\\\\/Ecr5QNFJgM8vnHAWrxn8Q47FP9k6IzzIzGFQeeFPg4L+AZHsGLuADuuYbmYZlzUIHklvHDNCzX5PXUS3sbRd5BS31vWmjBhi9GHi0tClphU2AtrLGArXPiE3oG8b3ptnz2q3ZGi0k0S6NEuclNaMw9e37EUTUMT6mLUvE60NFN2bx2JnVJbiyHDPxj4fN8GSfoNPqkHxEz3rE2UGSqp8O1v\\\\\\\/cOtfi9Ac\\\\\\\/IAt1QiRxy0nUyws\\\\\\\/RGvvgY+y1TxWbeKMRjfN0SyVo\\\\\\\/htp6OeeyTOj1rLYvmMwOe3IuBV+9Dk+iICrOeAi4CCi9wvuffkNOj\\\\\\\/JESJD2xmbOGATOsjvnOwzpUB40rWctJqmoSngeTs3lgCNJ7MzYYKC+R92IxGB7kgVpKRwPR3hdoTQQ63PyX3ME4dAGkxj2WV98GmtO0q2QjuC0XGfFkCxihD6TJ\\\\\\\/zdt\\\\\\\/dujYQ4JlJxz+m6jpiPfF7WE6ERfzsQYgCh9il6WwtnrEmSF402mrm7OIEWkfSIngd8SwrUCDgzDglY\\\\\\\/rh2Yq2GT9HytQP\\\\\\\/wWVdw11Yeu2x740eumDlahwK8DPqsiIUxbiEQvBxmfBRTSInCbyf2ih6YGVQWH7TOFxXQK1jJeTB1Aq+XiKVL7Z6FuHQnKVBboDOF922JSygbr4Nv+4g1ZuoeFHoBqEsaWqIvy\\\\\\\/lJ5O4ecnjjYnpYukIywQo\\\\\\\/C+h1a7h55\\\\\\\/BguOO3xinAyVfJCvApQ4r5+ajE0ZfIS2V76mlO2uXpR1lUwcOPGTtbOX4UPBJpPf1o\\\\\\\/Cdd2TVT12sPD27tO5qV8nAnpkSXoadGmF9baqCajxBxEma812fZzsmzxKDmL9hC+BV1586R4TbEaB7wmR54Nh082feDrhARIf9ZljbmFar8nWU2dnw1VTmKAMaU4RnKgF6RTg5JVHO\\\\\\\/QF2eSUSH325sAZ1JGAJn+eMyUK1T7dM36w4bm7OhOvArI\\\\\\\/KhxYvci1bzaKkXm+I0FW48QevpB1zX\\\\\\\/q57x9+fJMZp9QhfG9RoT\\\\\\\/963fEwgPptWgHClQUTlaiB9szuwT7g3G2MDETh1coa\\\\\\\/6jvJTRTZyGyMLaYTjlOaew7zuOeHZ2uaxKJIIgnywF1hsny\\\\\\\/uL\\\\\\\/LtyWQGyvcEISX5A7NXn1vc6FRCiTtCJ9GLAtH2yYb0uhZGhlmHQ4hGmj1isxV4Xfic0cV13kXrJgcY09BwkSVbcNEAtk7cJNxU5IlNRifWG+\\\\\\\/T5nRauuNtx2LrJ0uqcs2oOn5CIXBevwor\\\\\\\/y9Y9xs0X8nk0P7t3wb9X5w3JGvc\\\\\\\/kzfZloQjzmwl40WiGwAJOtvWgvu9BHnOqu7p+uHOkV5L8h4xnavrchsOt7p\\\\\\\/10EmS+iMn\\\\\\\/lpn9\\\\\\\/Uhtm7tieW4JgYruD3zDOi8fpVNZCsGEXLWoDHdMOfIw4pspph\\\\\\\/xodeXScf3NgF6+4D4XoHn2bJN9mBUBNEZssnjbzirrrdi5+WlwtkWhjHV0cQC5FfeT0cBK0sT0wO5pt4Lo+Dig5ypePi841kHJ2PJ5eB5XUeDIHRV\\\\\\\/qzRGbEu\\\\\\\/fJwkbt2g\\\\\\\/3qfQOFBElTzMUbMcBko+HuwGdjaH+N6FWBzPsZF5FWd7hoci7hIac8qH+c6ZsNBeUd3vHi13\\\\\\\/SilpzrQmnmY7Lwjix2fzO0SN4PfBqM6q+P2W1YlV7v07dri+7tXt3zQ6++XddsZFL3tx+CmkpkRwhPkUeNkcSmANjR8xVU3VIxB4yz4jVr71HfchvzZl6fXHhPWIrk11W\\\\\\\/Wa6Y8DTNA2WG\\\\\\\/mdaWBGSfDgZfi4mHW7hAkSCHLRYrcSIGv+gzbgEgt9pPrZpZ4UR+28ZPE9C528jIuUUbU8w3B92oENR0I7zZCCkDcrIXJPgqfN67llcQOQHAcKsxhG+7E8N\\\\\\\/NsCySEM5b6kepaWtTp6jTS2GcQfGzqzyGA8KGCgsdfEf0jYYck5d0724agTWJvIrOQhHjKvY+t8ckzPrdQQerLIiLFQCNmNYx+PQg+p9PXMpqgbQOeVVpNwfnfJpYg9B6T8HxMzY+324YF4DjCrkyQF3iv5GKrw7Vk\\\\\\\/ZyfDSAIf8t0DGzpfnl2+0fURwwuRFhrP1ZoaaPfE\\\\\\\/Fmaz4GZ8fQlQ1Hn3W4eGhdE\\\\\\\/vRxxUr9mWvKcTWdkLfZBFJ5adwLLNAx+GgJ+ycBwEd9A13qDwqYbaj0LMPoaby1DCUu2C8JZE4upmMi3PF0DLRk2PSbTBs39TpacGnJpwZ9fZM8aoMAtf93V3WrZuNrM7CqYORBaG3JAl1O8JVHcQTs6sHoTCX76e+yzMLqwieGeHJYw5JvluIC4lKjwRl1ayWkVyeWFzwK4SHJBvkQQ43XPVkyYC\\\\\\\/s6Bl1fswrMaGlBkHmFkE+UHlNVsBgzphsSZw8hQX9MTG0fj6ZxQPDXw33pZIQQLW\\\\\\\/S1cUutURD\\\\\\\/rgNQadPNt98jA3RANCFF6LjfvrqKRZJGNYdOSjwF\\\\\\\/znILDxAiidjIAfnJ7T0bR\\\\\\\/RAM86m\\\\\\\/mBj2lNRtmXqvRGQ0F9mwSKd1CTUBGpaZrASNBrFhJoB0c+tdZV0Vwe+zUvkp3a1yHaUqUE4j\\\\\\\/dMdCWYuDYpkxAwUeh6nxpoSGt9+s73SlVwb93Gy5SICGFD7zqALFOC3SlYYUdDTXtNI5w8QTW21h+xXOYJ6S+F+wv3pj9C0wJRBdbxBXyDbvg6NS6MT3qs+K18yoTG78jhappulDTzRWKhVLVGwI0DWTpo20Kx7sFSbWz\\\\\\\/VL+GfTsgaU69Og3fmn8byn7LYTfL\\\\\\\/w27buFU0k2N1pqxjapJS0beWXojfR6dqwWUvf\\\\\\\/mf4jJ8no1C1oZUpGdO\\\\\\\/UybZTHYNxKxkPshGMVs7x7EEAbDphB1UVS10EhOzXs05IGPzgfyyzfYFYyR0r1lYW2+tjIkNf0S+RlxpXPu\\\\\\\/2ve\\\\\\\/3bypnkgiT9GYT2n0QjWcQM6VQu0jWYZcmTbgAA9T3tdOw7U2XXOOuFlOSCzONJhN3mYZbpHu8sPvgoJWwRo63yjt93xwUs4WcgBtZsKXw8aebUt3l3LQQxbZ5ch2oSag48N2UgOgQ\\\\\\\/xVTPcRM3g7PUjwJwtgN3k4L9psW2qPl2tPkzM8JYQfxu4z9S\\\\\\\/yD17e6YGWzxwiL0nX972NErPme4qlvuxB63vXk\\\\\\\/RhS6wBTqXAN8o9jm3C4WcMK4XoBJHziXC\\\\\\\/B3bnCqz+D0bBROtu3fWR32OX\\\\\\\/\\\\\\\/SAQZv9mrKybX3x2vb5RTn9MatRkR\\\\\\\/luXd+YJ7XALiws8dTEBsniIxLU0zykZ+jJ4P8lVbx0hBuTMZx\\\\\\\/tNz0hJ6pQAenj97fugwt+RXLICGbfdjyDU98hzedtJ4NBGNJCEqYb3NNa2VXo+L+N2VUROxh0JGm9XdDlr110IRLyybufQdIllDkw2jQk2vAHWBmoLJWlhpgsnvAMNEJd4o3Bar9S43FXMFNk8ViV09jYKJbZfIrKvwx2EY1QDGvs0bxmwfbwzK51+q3gnJtgttRH8C5J3oACEMuLZMW5LqW7EOPvR6+iS2g\\\\\\\/Fjmbe0OmPBbc\\\\\\\/5i3e1LZtp7Rj+YhHfsRQu2donXTJ\\\\\\\/zIUQ8JOlSzPxarccuty58+6T4LlrXSPCX7H3S6Yn4F9lhyr90tRqIhhT\\\\\\\/X5YRdn40sE9f16J3VjlwpPg1\\\\\\\/ZeNw\\\\\\\/VWgWSXmzXq2XNYVjSsLLqqCk9Us0q\\\\\\\/85bU4vp3erZnuW5pe\\\\\\\/nJ5OBEwI37rjYX1z75KUjE4CzfygAeXhGCz+haH8ZQOll9aJPSfcYnwDFZ+MAFaDWvfbMHqX91C9qXYF\\\\\\\/2+7YMm81xXwzUGXWMARBkQoz50DO8\\\\\\\/trlTxP7\\\\\\\/Qp6Oi8OCdeLR1JQ4RI3cEBqofFCO6VELFQUWslR6DI8PzZTQGlQfEmyC2TDEyd4mPeAm1m0asPWrc7KftegLPINSrMzLILlD9OMBEKSTGnjl+88FVj6BfvQ+wKstM9leftbDDNJSrK\\\\\\\/ZEoIB00yS6OgUqXQOlSNe50a9Yb0Cia+2Yd7Do7bV0cda2NLts\\\\\\\/WLIHoIIepXRZ1+U6MlNhrUBxFKISzkyFwYdpPsPFdM4hvN1qRnRUJB60BnU0YAyKYNnC\\\\\\\/eMiDzhAUYl3a0berjtmeA1Afn4wERx2hdx+XAm1AG7dOgVNq4HLQNgEr06AQMfMw3Bu7ckZ\\\\\\\/5mlY6b1RiKwgs5arrzeEbpY6CdEDjCCJLoeMKqjaQDU78PDGoEW7C9FAjc6xhgjJ0wLSOV3on2KfMdkMVleNHxEkW\\\\\\\/rcHLDAYjgROqXcjfkAls8QBWPcpVBywxlgSBmwH\\\\\\\/ofXxYu4gJZG+HVMphhhr+IAhIdznhW4Vfco5Dm\\\\\\\/jekZnWreHU2R4RbDSO9y1rxGT+JrFCwMsb1MOmCP9bNOtCarf4h7rtxL7xitbzIjdLTH4uGDCXCBaTjQc+v2WKeokNUdRqdH8kooT6vrOnCfV++E\\\\\\\/1LBrMP1pDFb4c6Nf4QyCN7Er+NNOFj+Xqfn2R9aGf7BglWZHYnu7jJ9pTt5QF6tvO4WItN83vXps\\\\\\\/9BDlWJbovl6rLP7nzYwRfJf0zg58Ns3sEBsasjLXLrQXh6pVp3DzMsT+NFyY1LsK26\\\\\\\/5IruwsWcYM8KrfgsisJIbSfanDscaK6e7Q1T7Sstz\\\\\\\/FpIZHKqfNm0xO4reIG7k2xNMBWJVZYWm6b4pOsm2tvffQe3ZyV3RNXTGVnO+SQkURTm909BTVnfZDV\\\\\\\/pgSleLFpR9eQtxdH+hAaLlgDn4qv+w7hFUmrebrv5tkObSmbHouNqXa+oNUsa7IUZVvceEMKQBkgflzp+d3pqKW1E3rxdMaZXgq4q5eiaRqVCAE6LuOMGaTlDQbgDEs9mdTFX22uz3z4ukVgaEGNKCfe+u+xj9vv7R73kx+NO4\\\\\\\/VcP+s3Rywn9LNYuHhw1ARqJQgQ2hIv3oNI4TtyOi8NafMtNRcwO9meMk6a\\\\\\\/9\\\\\\\/P32DajqlL5ghKpruTyt8Rg0yh5QQs0xcuYvyiYorxj05JpycVvyUiuy6Ohpc+emWEe4TPqCAA86ZL0I+Eq\\\\\\\/1ES30ynClEV8PFHEv7GAVcGLXHWVZQFq9Wsu180eckXWHCT1V8tp9A8kIQbjh71LVbaKEzZdv1lRdYLlqjnmTkmxL6pcqA6kmqyfiLi2+xfkTjgwYQD6tEIGti+c6jYbJvmwzv652YZIUkOsV3yMc9vP3ONRdSYibed8WIOx8+i\\\\\\\/JpXj3H9IQ5ysGzoJit7NCxP0W4pfEttnraAzd4vcdWOwhjUGBcI8MIbFbH+LsWmy264GQ2ejONzjkzaJlE9EUW1WnZPAxUtFyUuYzOij4dzW+UN2sMvzaRlq79Yk7XMUayi\\\\\\\/Ys7ySWQH07KvHNDQeBDo3vp7D6VGIv72tBHW8aKgnZUbgzWJc8MHE+5FTVtSMEoJyizXa6xr5DDDZOJv2s4M4bzxOJq1vdLuYSKpmw9opHiYDjQvw8vfyUyAlXaC1d7wbNBvzDqcSZPVYBk0mrH09fMak\\\\\\\/f48Xe0eHBg2M8rJK0fY606gBSPKSq0EUU73lZrd4Nrw\\\\\\\/eQgf0qn5VkyOdmAC5n7H8fAneLADA8LFIS2lc\\\\\\\/SWC9+F3Qz4mltwnVbx678L81M4N9LEcZWUifrV29Yabh5tCtspiR5VT5SQlO07TW\\\\\\\/0DGtuNJAgHqEptVqAT3H5m4N+YAyNpuXscxzlqJPdptdwMr8GmiNGhSSTC2B74KgWwmd5+DjE2pj5iPDwBvEPp095Ezldux65mu7jMEW40DOdy6hZjdQkMq7ClDnGlUYIJexlAsLw3kBtj++p+Df+U1UgXWDsBEG2VRfEfY+G3DNhtU2u3S1T1LCloDfECRzNyy9mUxI3jdldN5ez6e8yaItp7gQ23ZC8HllHgUD3d7waRyHDA3IWtrywbFTSdSWvS4Anr2Ub15nyj9B+Hq+HuL8KzE4dOVk2oy2hvFV2bXAMbduZf3dD8QwbkVXTnpWnc4AkOOPMx1ubzcSTtmhu+KH8Yf3UFhLLxM7PnLV4Id8iHsZ2FUXnaHjNcBQkRim8k4QRrBMwX87yvJToBnAuGtVN5oLt7hxC1ZxMNUtaWlBPqc6vj9zp+R8Z09nohNjlbiSIWFAMbF7es+xADXuJ8px722qA5D+ebKZA0VIvXzgm9v26ocWOKCnf6iOXpUfXZf4jnq5iWnjSSS7vTnY+gF5Qn8H7RqJFoDJKQ2P80sER5e7Q4jDNTyDle6UIRxi+iIOk4rvdAgN93BClRoK+bPRiLXdz+0ektTxl0aY+9itJXl+akGyskxCGcnUZYKVRpO1sqxfDb4TqjbgJZ18vrdTPmeXQ1EhHx5ZX6i9mH7jhK2ubMptJ6LeJqKLSUrPxqut0+QlQcODSLP0tVn7RSF10X8sxLEYUIOwB0PzNu9uDmjbM0\\\\\\\/dVTPJ+7w1m03zkI1xAEIYVr12KQPjCklY+IUct3QOVjvBm7TleLvDk4OZVReypYgS5Ig36n1pMmsZS8E3CcoJuZOeSUmy6x1beyV8wPZ7p2kdiSjFiIzGgTryRwrXwtwZM6ohKs6\\\\\\\/A3qcd3iKtbwGI1K+f3z0CHnqFF5pF\\\\\\\/W2WerpAEZwRHqA1F\\\\\\\/SewfZJXBFdgqTqPY4\\\\\\\/ixFtmJ59OkEhDJHXV5gpvGTaU+f2byS5q9hcwPneDSIMTbU+\\\\\\\/VzUlhVLiShCkSBzmBxjuRtwRTaUf1wZPrTzUtuAZryzLjdWPj8mKTKfa7Z7mU\\\\\\\/W8GOYujN5HmMuPIZcJYuGfIWCUElXJ0y3DkMv6gOdVJR68mwRLU942g+0HLpd2Yzpow1LEIxQs3s5GFMu9ZKWphfVh6hkSFnnReVkkzTmRHI6O\\\\\\\/IwPxdkJmjgC7dPVZ2GDmWPxB0Y7p+t+IHcgNc4ajG7Zt5qiBjGmw1xsgclke8NCXzseeX38vJ0+jliN2\\\\\\\/wZPnM06zAJL1o5tM4tVEaa0glexejUly8oWX2jQhX\\\\\\\/\\\\\\\/F6ey2pkiC99LzgX5paKTmzImL3Xv+fDFUF1K3FmOj2xpXKEq7V8affccORRKyldpE9Xdd3GSpgzHQZfQRV6RecNEu9gLE+5XWusnBSzc2jBf8qKjq4YhPlI3Ej2HQ\\\\\\\/OubQloJekrSPu4SzzreqwcaGMhVLnfLPI\\\\\\\/KaQw6wD6Aha5029dejBUPHxLLgm6wJm3S68sjX29G7dVxKTHUuEJXQi+5w35r7OHi+OwEFboh51y2ROKzw+v5vc5vs65qq5bJ6u3vRadVjnZS19NMDnkttwDvxoaCQW3c+pYhz4ZYJd1rSTr6Zx4EbQtF1ztPHok49Ia\\\\\\\/xUwtBygbfVFJ9i2FA4Ny4Y8Sx9KI2+XVwuSm4Mpo0PAO06hcjzX3U7J\\\\\\\/hJG2wiwStfopIeKN+kp4UdrblExzts6wPlrOU71WnOTcg0Qx5jmyV1MMw2aYT4Ix8eVksL5x7IjzbqrYw\\\\\\\/sGGOn3RBymfiamllGRHvvBptM4\\\\\\\/ZxDjHILrznr69v6LJDKgREol6jkYQxH+GCmV2HcjWhNCT9lI7J5N4713utytX8ArwUYN1sfPTeYVoaFi1t8fxNzPPt1GxuqnKX+ofurPZ\\\\\\\/ZiIxuvIv2WHWc7BKh\\\\\\\/uZOWMw9OlUZ\\\\\\\/bonkWFANexTDgh2VIjnHUV4T\\\\\\\/4tMI+ydkJwKoF3eFtqOpJYKTj45\\\\\\\/CRWCx0Qp6pyq\\\\\\\/7WdFDVQiixTkLGJpfOWeQopoIMLA0Yd33w0gD6OQ+9qWxhO\\\\\\\/ZLlZ3U\\\\\\\/Fesh6akXRpd5LbzpEQ7RgEarUBiQX13zl5GGs4AtjIRNJ4ItoFuI559k1yByD\\\\\\\/lReGIeVSXQaOQ52RzmYznL1GtDm9olBBdg7ce5hhgM3VPUjjvRDp2tNhIaAxN8jTAquyMbkmIXgsnEzXXKOK5DOZjvz69b+T20b3vFpDM31KJI2z+c7ohm+WwaSdPzYDdq1W09U45lBYRiNetIGtIdq7u3Aq5Vw3W3Yx7ARjZ1NOhzJweV8TWFXx\\\\\\\/FURgVJpCTDyH8y+1p+jqzyfudO4NcHFzHRaG5Ae+7zNgWFMJcWe4MWbA763+ZA2FKXqFNyW29uPsDlh33m+edSruwbABul35c80OUHGNw65+Y0+iM2ytTL899ry3Kqqwp+nCAtJ4hhWTSQf+Px4EWHDkzujD4cl7lo9TFPeq0vPFR4o41T+1+SJVna+KeSjo1nIbevGiFSTyifleYlPaGbvrOGnmOhCyquGL\\\\\\\/Z4CIrxbrkgvLRyvqWRwPNRlriVsJ7uVFiERdzXhK4KQqnL1xY2VrYHOUcBYRFhUKn7UZZC5wW00cUGALaJlZY2Id\\\\\\\/4OCOUraMtN5GWOtElWpJtok0R0IBL2g5WknFIbEC6Y9EUYQEFVcTnEessP4YDs7mr9x5j+24t+pB9lNu+ooKqHlvQQ5GR82HzXHs6C6mN3BxSe1Pp0ys5a3ny4EXX5moLkl+Vt\\\\\\\/qat6fgCuiV1S8iAZVTYjUBpZdS6BSFOHGFr4wHxIgu9+ItBKcyebC7s46zSPG\\\\\\\/Z9265tS7dNMcT+571goe6q3JhLyDrL5H2nEuQ6Sy7ytlavFKtQqotgImR+1fIv1bvJX4\\\\\\\/Sms6msw8KCfe9eVFcd1Gd1RVRPpNboIdVGKkf925lQ1URExasmBMlBv8QPShwDQNWT1nEql27X8OJ+c6nmUbfHu7mebn+VvY7kqdLuu7HZ7dUg2pbt7JH+\\\\\\\/1BjJoAf9FxA5DpyKnFfM1gIIf32yIsQONFMrtVJCnIKniIoVwk\\\\\\\/B3Oc2JFDPX+jPmfPLJ3NCTgG6IsPICXypTTYrOAVdsG4MIsMd3eEeobEyX0YnsWYsu0WUG35wXKcgzZOGPyMl+VxFuirJL8GZvLrOQXRp4Yy5gFQhcrEw7UEJ9Lxg2cmH9zpfnZXW4tb5k\\\\\\\/c74DpBUGChJ4\\\\\\\/BIItxSyvtUvwybJR0w6m6OSYLkQIgBaVnRkHfr8FK6eeY6ZOan41yyZP\\\\\\\/DuwjEGYWtEHvZZ2jX0V4GJ0MuycTL\\\\\\\/5jKqOs65EO9fCSVziLo9d3eJcem2zRZf7N1P\\\\\\\/TpqOV\\\\\\\/wKLnR7lpJnOdPDZ6qWS593\\\\\\\/4DV2+t2JTkoFXOHBa36B6CbHVsICJxZo6VfrGq+hW0R0ldgA\\\\\\\/AgaCCP1LjSNg7VZlnuZHAeKCFA9VLxJV2GwywW6dHqgB3D07rjmIQQm4CQTm0mbzYF1oPjljjeJPWnSu6ZNSBtogiX98L4efxetiXi1r06pQLa86YTohfL6ZsCu+V6P87YAaltF0oo\\\\\\\/fsSroImXqfPObHdX+J+IMsEJD3njm3Oa7Ov9CUfd98XeHO5evK8WAw9GtxOwulDVOArdOmvVXJyZTcGY4kzKMMd3JrKc5Yzpt53BArJOI2yZQsIQMftUFYv6GoGHP9ZGSz8RdPaB+tcIH7Wy46jcNYH0ZobZaT+yXnj3i7685BuAsBCQEuSGJyGCHtLurwxz2IBy6ZfO0KXyhVDgpk5OXvfaLNjgCyhgBSO+3ReksXlbQ1JzFLJUCimuRgkRRIrZjSjkmMN7sSDN6tcQAyCr1bZqnW\\\\\\\/Ujwc+Mf9UJshmEkZede8sfC\\\\\\\/8MTtT95fhym3bGeBviUq4GI10Tbs+3Yeogcl4FNU17kyacBT76WnE5CCLAJl3aSvVHef8fJbBDEki2Ch6x08ZCIU9VoDo4tmiOQRJyoM1DPW44r\\\\\\\/jD1XrVudadfYa7gTHe0UggUcIMtH\\\\\\\/bR0IEZGFC1ao1caHKVmuGEaMgsxP0BAhKbCLyGz7hxNAvekLK0luLj9aKXbebONBQUNC\\\\\\\/HO35d0n9KSHesj4iPyolPMOy0ul4xgGsZAwyyPy5GyXE2H8O90vXgAZGejmPxuCnHOLLJIaBf9+ThyHcSkbRUkc0Pk8zTTdNWAOY8L\\\\\\\/DPLhc6CHZyOSsCoS+Ms+sbUBxWf43Esn8weN01kP78QvXzL6RIOnKUYC4G\\\\\\\/hHJXwEhuNcrkiw\\\\\\\/psDCMuxQjpoJB2Ho5YyRZORI4gwOprDPmphSOo1lB3wIAG2XC5hivi5mPJLYZL9leTmYp+5GIeOMj76TuCzgE1kStSW\\\\\\\/IDm5868CMOz9pfBHxt99RNtcXy97Wynjgg8\\\\\\\/1fuhDsBH3PZR2f3X1B9Rd2rc4+MpSkrFkTqBpV9TmUxa25st5kOLCA1R1cgPotH\\\\\\\/mf3wxrxeDgAEn\\\\\\\/JzX2zQBa5NzarLSuWmY28+O+ftUgVUiPtKN5CrVO0u\\\\\\\/96+nh\\\\\\\/7f0puWotzjcAkiA7gVyw7mSiUN3uNNYuyRJ4DHKuCZz\\\\\\\/7Y7KgkDqoKyhNAyzSJ2VOvW471jVZ+XyBIYJP4AxBx4cLkGX3N4brXytgosw\\\\\\\/GfdWLV+X80YBe2fhSL+SevM\\\u0022,\\\u0022iv\\\u0022:\\\u0022aMsrowDPAv7SV+vIQ83HHw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022a2daefae7d70e40f2868602d7fb9b88e604b854c54c41967fbc4669f923fa9fc\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022o4NTZDCicLqz09jVsc7i0MRLuvoEjbRfZmHd\\\\\\\/1y1xkzG5x+8ga7kZHO5w4OVCqBdowc35l1YJ+S1qIYRrjLJ3Lt7tzXbtlDPSrQUWVZA7q2ltyPsxX0nx55GNYdzntXd3hmf+0+gn6U6G9EruQYa3F3ZeQ1hB2o4seQ6feHe56AmzxMfG3iaUCSskPKLPDyg0turjbl\\\\\\\/PRZxHFlmw37PgWKEthBC85HRDnKLaHCs\\\\\\\/FPR7WWYuhRZrE0bdyMr4RUjm9a0q16QToMRh+0NzXpn2TY73aB4WF2u5Xwltv6r5zbPf1OBQur4vRdaFKl8DuNbl5u5O3FQyqMyZm7SC5\\\\\\\/WGA+ZqIInZy\\\\\\\/2\\\\\\\/GsTcdM\\\\\\\/x\\\\\\\/UqYbttchh4c5fTcWY248SusODzAGFEnJBCds4rpI736C6d5WbSd4AWdTcauPaGkVYNgFoJfR+vLGbQKSWxVgJQj2SBH8qrHE2J\\\\\\\/aRgMcI4VVbuBn3jTZiL9aBmy3GY4iuY0\\\\\\\/pHgBdN2beX5s7qDhunD7dVhi3r+ChV+XZk0+in3y7rsJda7mFIR71nDVKGwJoJayUJOFEixB7SVcZPC5pVoJXGGStvYK1zncLNixyOtSOf882N2z7dBbxEs3XNjw3Vwf9KIm+KG0pesJhqwU\\\\\\\/VaJfw1T6UhmEwt04LaDmpLbMJ9gAxPsBHqcurTzehpNX3KEZWzsCDCvlG+wOTxEHZjjy8nZZAz9UbPBOVs9Ja6QIeDaeaCsoklWEKrdAoSYLywbpjonFMlrZ7WbPB9RQL87XH1bNImYFhI9tJv1RaDJrFMsIFbzRupzNmwRh3TJS1Dm1tPd0A1j1oSO+BrCE+4zRc5Xa\\\\\\\/7xnu\\\\\\\/HI5dNcvztQLfLg9BXx9ImqiR7OKD5BlK0a6Dg\\\\\\\/f7eEWW7YsWEE7Eyf8WxPqY8taV0IYf3ovKxoFWGavojvjioFRD7wHi4JeQev2GfqWwn5i3eYdd1Db27NcEA9iPkTt5fzYaeBw9ImXCjl626\\\\\\\/T46UR1nqZL+Rk9DDoRo1L81dVaVkpeXpP8vbbuHuKZgH7JbBFLxeFSfrWwjBAs1COL8Ue35+apjxbckTvAMXyvLyx+gCvU3IIITd+BONfPp2vaOzFiJsd\\\\\\\/t4VIMqSUy5pH4xtCxL56P\\\\\\\/ZgYkMQtS7Km4cVYXsFbL+YCEB2l6+wSW9psB81ilzemctWg50MD3EGIhNRt\\\\\\\/+hc9gWJb566Yfwgnt5KInL2gmjutb2c6Oh2QvBBk4u04o7PHlvcMS7igvkA8d\\\\\\\/m7QR5lo8UVlExG4rx60S71jAhV1UoVMxS1k\\\\\\\/1yTdVTi\\\\\\\/BiTo2a5uFVUgpYOxA50239Xff8kHj6czpIJcBdIQfht\\\\\\\/J0YHgJsH5BSGd2v44KImaCd4TioLG9wRXSnjUJj0XXpy8BdBm\\\\\\\/8fdslU69odR9lZfgfUAGkJtDMwZbVhovnOxfDTIp2\\\\\\\/6vzROApg65cfAcWn\\\\\\\/KK18Cj+l3j0Zj8MriGBrjyNfS6vZfAUDMo96WW59\\\\\\\/zTQYRYX8eMDmtUAt7nEgUvDivh68eaafFIcc8ZlKLNHVZwR0IY93SkIbooBIWVI5r7LI2j+PwOSMqQBIkX+t3AFQy4KdV7\\\\\\\/bZSle54XjUpKMgqCIwvKy2deVr29N58FM6zleQmKp2Ocb0SGGWhXlWssz98cD4ocXlR+B4P09YD13OIeNqNfha03JcJkYVPHeN0Y+yHHknFlwog7GM3LsNwtWAZxadef61uFYn5yy1q0yn\\\\\\\/sZH1xIN4rW3lCVAguLxcmGmQxF1dLcfA33ThTUL7iE8yPjOjRrl5oeIzdQA6w9CfuDwSbWU4R9JktJA8vVaLbBnh1b+Urnkv2+KvKTo7C6FJb8A6q4atA0XK6MFl+kmM0g8wALsmju2rRww6UUYwkLxS0lcJvO+O8ooGzAhcZEwlW0onCjyVQ2YosPw6jGu0yZF\\\\\\\/\\\\\\\/FAL7VRBshSlJmSJP+slR1MT0apYqHkKZbpVGph++GTptEZocZWPOMqToIEoCDa7PH037a9aIt0BptPDjt+eRtYLm8vL7tg57oRpV0=\\\u0022,\\\u0022iv\\\u0022:\\\u0022O9o7oUF9L0OuWILdHbuZkw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022192fc97e1cee95b998afd0989f7a40fe364a27e05ae8109d2aa3676ae74ea69d\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-486351 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/9/3103" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-9-ZScQatzwK7NJuB5lzLgeDmZ2aNDGGKiZ1HN_pVN-g" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/9/3103" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-0QN-5_rOgcVk7-aJ1F2dO7PCafQFTSOxfTdpRAvgIFw" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/9/3091" title="3′-Terminated Overhangs Regulate DNA Double-Strand Break Processing in Escherichia coli" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/9/3115" title="A Large Deletion in the NSDHL Gene in Labrador Retrievers with a Congenital Cornification Disorder" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="486351" id="node486351" data-pisa="ggg;7/9/3103" data-pisa-master="ggg;g3.117.044453" data-apath="/ggg/7/9/3103.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Application of Response Surface Methods To Determine Conditions for Optimal Genomic Prediction</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Réka Howard</span>, <span class="highwire-citation-author" data-delta="1">Alicia L. Carriquiry</span> and <span class="highwire-citation-author" data-delta="2">William D. Beavis</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3103-3113;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.044453 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node486351" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Réka Howard </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Statistics, University of Nebraska, Lincoln, Nebraska 68583</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=R%C3%A9ka%2BHoward%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Howard%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AR%25C3%25A9ka%2BHoward%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:rekahoward@unl.edu" class="" data-icon-position="" data-hide-link-title="0">rekahoward@unl.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Alicia L. Carriquiry </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Statistics, Iowa State University, Ames, Iowa 50011</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alicia%2BL.%2BCarriquiry%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Carriquiry%20AL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlicia%2BL.%2BCarriquiry%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">William D. Beavis </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Agronomy, Iowa State University, Ames, Iowa 50011</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=William%2BD.%2BBeavis%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Beavis%20WD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AWilliam%2BD.%2BBeavis%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/7/9/3103" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:486351" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:486351/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/9/3103.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:486351" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:486351/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/9/3103.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:486351" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:486351/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/7/9/3103.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:486351" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:486351/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/7/9/3103.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:486351" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:486351/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">An epistatic genetic architecture can have a significant impact on prediction accuracies of genomic prediction (GP) methods. Machine learning methods predict traits comprised of epistatic genetic architectures more accurately than statistical methods based on additive mixed linear models. The differences between these types of GP methods suggest a diagnostic for revealing genetic architectures underlying traits of interest. In addition to genetic architecture, the performance of GP methods may be influenced by the sample size of the training population, the number of QTL, and the proportion of phenotypic variability due to genotypic variability (heritability). Possible values for these factors and the number of combinations of the factor levels that influence the performance of GP methods can be large. Thus, efficient methods for identifying combinations of factor levels that produce most accurate GPs is needed. Herein, we employ response surface methods (RSMs) to find the experimental conditions that produce the most accurate GPs. We illustrate RSM with an example of simulated doubled haploid populations and identify the combination of factors that maximize the difference between prediction accuracies of best linear unbiased prediction (BLUP) and support vector machine (SVM) GP methods. The greatest impact on the response is due to the genetic architecture of the population, heritability of the trait, and the sample size. When epistasis is responsible for all of the genotypic variance and heritability is equal to one and the sample size of the training population is large, the advantage of using the SVM method <em>vs.</em> the BLUP method is greatest. However, except for values close to the maximum, most of the response surface shows little difference between the methods. We also determined that the conditions resulting in the greatest prediction accuracy for BLUP occurred when genetic architecture consists solely of additive effects, and heritability is equal to one.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/mixed-models" class="hw-term hw-article-keyword hw-article-keyword-mixed-models" rel="nofollow">mixed models</a></li><li class="kwd"><a href="/keyword/machine-learning" class="hw-term hw-article-keyword hw-article-keyword-machine-learning" rel="nofollow">machine learning</a></li><li class="kwd"><a href="/keyword/genomic-prediction" class="hw-term hw-article-keyword hw-article-keyword-genomic-prediction" rel="nofollow">genomic prediction</a></li><li class="kwd"><a href="/keyword/epistasis" class="hw-term hw-article-keyword hw-article-keyword-epistasis" rel="nofollow">epistasis</a></li></ul><p id="p-3">Genomic selection (GS) is an approach for improving quantitative traits through the use of genomic prediction (GP) techniques which use information provided by phenotypic values and genotypic information for individuals, lines, varieties, or hybrids in a training set to predict phenotypic values of individuals, lines, varieties, or hybrids with only genotypic information (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Meuwissen <em>et al.</em> 2001</a>). Using GP, it is possible to improve the accuracy of prediction relative to the traditional phenotypic and marker assisted selection (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Lande and Thompson 1990</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Bernardo and Yu 2007</a>). GS can increase genetic gain by increasing selection intensity because many more individuals can be assigned phenotypic values than budgets will support through field assays (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Heslot <em>et al.</em> 2015</a>). There have been many statistical methods proposed for GP, and there are numerous articles evaluating these methods for sampled populations under various conditions (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">de los Campos <em>et al.</em> 2010</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Zhao <em>et al.</em> 2012</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Howard <em>et al.</em> 2014</a>). The relative performance of the methods depends on the attributes of the training population, including sample sizes of the training and validation populations, marker density, narrow sense heritability, <em>etc</em>. Using simulation models, these attributes can be varied and their impact on prediction accuracies has been evaluated.</p><p id="p-4">In a previous publication (<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Howard <em>et al.</em> 2014</a>), we simulated phenotypic and genotypic information for <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-1.gif"/></noscript></span></span> and backcross populations for traits with heritabilities of 0.30 and 0.70. Half of the simulated data sets had only additive genetic effects, and the other half had only two-way epistatic genetic effects among 30 loci. All simulated data had phenotypic values for 1000 individuals, and genotypic values for 2000 biallelic markers. Using the simulated data, we compared the performance of 10 best linear unbiased prediction (BLUP) and four machine learning methods in terms of prediction accuracy. The measure of accuracy reported was the Pearson correlation coefficient between the simulated phenotypic value and the predicted phenotypic value. Results showed that genetic architecture had the greatest impact on estimated accuracies: machine learning methods provided higher correlations between predictions and simulated values if the genetic architectures consisted of epistatic genetic effects. BLUP methods provided no ability to predict if the genetic architecture of the trait consisted solely of epistatic effects. The results suggest an analytical diagnostic that could reveal the underlying unknown genetic architecture of a trait in experimental data. Explicitly, a comparison of estimated prediction accuracies for a given phenotype using both types of methods could reveal whether additive or epistatic effects dominate the genetic architecture of a trait. However, we did not elaborate on the conditions under which the diagnostic could be employed. It is possible that there are many conditions that could result in large differences between estimated prediction accuracies of algorithmic and linear-model methods, or it is also possible that there are very few conditions under which the differences are large. Our purpose was to systematically investigate combinations of factors that could affect the estimated prediction accuracies of both linear model-based GP and algorithmic-based GP.</p><p id="p-5">GP accuracies could be influenced by sample size, number of markers, number of QTL, epistasis, and proportion of phenotypic variance attributed to variability among genotypes (heritability in the broad sense). Further, it is possible that interactions among the factors influence GP. Our objective herein is to report a strategy for identifying conditions under which the diagnostic could be employed. We determined this by constructing the response surface for accuracies of linear-model and machine learning GP methods as determined by sample size, number of markers, number of QTL, degree of epistasis, and heritability. We chose ridge regression BLUP and support vector machine (SVM) as representatives of linear-model and machine learning GP methods because <a id="xref-ref-14-3" class="xref-bibr" href="#ref-14">Howard <em>et al.</em> (2014)</a> demonstrated that these provided the most consistent accuracies among mixed linear-model and machine learning approaches. After constructing the response surface of estimated prediction accuracies for these five factors we employed the steepest ascent response surface method (RSM) to demonstrate experimental efficiencies that can be gained when evaluating conditions in which GP accuracies are maximized. The steepest ascent RSM is a technique that is useful for guiding the choice of factor levels to identify the optimal condition of a variable (response) dependent on several input variables. RSMs are used in many areas of science to design experiments that will identify combinations of factors that lead to an optimum response. In this manuscript, we are extending this concept to the design of simulation studies to identify combinations of genetic architecture and input factors that maximize the difference between prediction accuracies for different GP models. The intent here is to evaluate the sensitivity of the GP models to the underlying genetic architecture and design factors using RSM.</p><p id="p-6">Before communicating the methods and results for the diagnostic we provide background information on RSMs. Next, we describe how the response surface for GP accuracies was simulated, and last, we illustrate how the steepest ascent RSM can be applied to efficiently determine the specific combination of factors that maximize estimated prediction accuracies of GP.</p><div class="section other1" id="sec-1"><h2 class="">Background</h2><div id="sec-2" class="subsection"><h3>Response surfaces and approximation methods</h3><p id="p-7">RSMs are used to approximate functional relationships between a response variable <em>y</em> and a set of design variables (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Khuri and Mukhopadhyay 2010</a>) which can be used to find the combination of factor levels for which the response variables are optimized. In this context, the term optimize refers to either maximize or minimize. RSMs were first introduced by <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Box and Wilson (1951)</a>, and are used in many experimental disciplines, including physical, biological, environmental, and chemical sciences; engineering; and economics although we are unaware of prior work on RSMs for evaluation of statistical techniques. The primary advantage of RSMs is that the number of experimental treatment combinations required to find the optimum experimental conditions can be (much) less than the total number of treatment combinations composing the entire response surface (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Myers and Montgomery 1995</a>).</p><p id="p-8">To illustrate the response surface for two variables, we simulated hypothetical yield data which are influenced by temperature and drought. Average daily temperature is simulated to be between <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-2.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-3.gif"/></noscript></span></span> and drought is between <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-4.gif"/></noscript></span></span> and 4 standard precipitation index (SPI). Negative values for the drought index indicate conditions that are dryer than normal. The model we used to simulate yield <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-5.gif"/></noscript></span></span> <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a> shows the response surface, and <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1B</a> shows the contour plot of the simulated yield. The simulation was performed in R (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">R Development Core Team 2008</a>, <a href="http://www.r-project.org">http://www.r-project.org</a>) and the code can be found in Supplemental Material, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.044453/-/DC1/FileS1.txt">File S1</a>.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/9/3103/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="(A) The response surface of yield in relation to temperature and drought. (B) The contour plot (level curves) of the response surface of yield." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1752734692" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;(A) The response surface of yield in relation to temperature and drought. (B) The contour plot (level curves) of the response surface of yield.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/9/3103/F1.medium.gif" width="440" height="202"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/7/9/3103/F1.medium.gif" width="440" height="202"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/9/3103/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/9/3103/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/486491" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-9" class="first-child">(A) The response surface of yield in relation to temperature and drought. (B) The contour plot (level curves) of the response surface of yield.</p><div class="sb-div caption-clear"></div></div></div><p id="p-10">For these simulated data it is clear that the yield is maximized when temperature is between <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-6.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-7.gif"/></noscript></span></span> and drought is ∼2 SPI. In most situations, the true response surface is unknown, and is influenced by more than two design variables where visualization of the data are difficult.</p><p id="p-11">A model for the relationship between response <em>y</em> and the <em>p</em> design variables, <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-8.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-9.gif"/></noscript></span></span>…, <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-10.gif"/></noscript></span></span> can be written in the form<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-11.gif"/></noscript></span><span class="disp-formula-label">(1)</span></span>where <em>f</em> is the true, unknown response function and <em>ϵ</em> is the error term (<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Myers and Montgomery 1995</a>). The error term is often assumed to have a normal distribution with mean 0 and variance <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-12.gif"/></noscript></span></span> although other distributions can be modeled. If we assume that <em>ϵ</em> has a distribution that has mean 0, then the expected value of the response in terms of the natural variables can be written as<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-13.gif"/></noscript></span><span class="disp-formula-label">(2)</span></span><span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-14.gif"/></noscript></span><span class="disp-formula-label">(3)</span></span><span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-15.gif"/></noscript></span><span class="disp-formula-label">(4)</span></span>The design variables <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-16.gif"/></noscript></span></span> are commonly referred to as natural variables because they have the natural units of the measurements. In an RSM the natural variables are transformed into coded variables: <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-17.gif"/></noscript></span></span> All of the coded variables have mean 0 and the same variances. It is convenient to code the low level of the factor variables as <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-18.gif"/></noscript></span></span> and the high level as 1.</p><p id="p-12">Since the true response function is unknown, we have to approximate <em>f</em>. Under standard smoothness assumptions, a low-order polynomial function provides a good local approximation to the true <em>f</em>. For example, a first-order main effects model can be written as<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-19.gif"/></noscript></span><span class="disp-formula-label">(5)</span></span>where <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-20.gif"/></noscript></span></span> are coded variables, .. is the unknown intercept, and <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-21.gif"/></noscript></span></span> are the unknown regression coefficients. Equation 5 is called a main effects linear model because it only contains the linear effects of the <em>p</em> factors on the response with no interaction terms. When the model in Equation 5 includes interactions, we call it the first-order model with interaction, and write it as<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-22.gif"/></noscript></span><span class="disp-formula-label">(6)</span></span>In situations such as those illustrated in <a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>, a second-order model can be also used to model the unknown response function <em>f</em>. A second-order polynomial provides a good local approximation to almost any surface because it can have different functional forms and it is easy to estimate its parameters. In general, the second-order model can be written as<span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-23.gif"/></noscript></span><span class="disp-formula-label">(7)</span></span>Let <em>β</em> denote the vector of unknown regression coefficients with dimension depending on the model. With an interaction term or a second-order model we can introduce curvature into the estimated surface. The model equation can be written in a concise matrix notation as<span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-24.gif"/></noscript></span><span class="disp-formula-label">(8)</span></span>where <strong>y</strong> is an <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-25.gif"/></noscript></span></span> vector of observations, <strong>X</strong> is an <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-26.gif"/></noscript></span></span> dimensional matrix of the levels of the coded explanatory variables, <em>β</em> is a <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-27.gif"/></noscript></span></span> vector of the coefficients, and <em>ϵ</em> is an <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-28.gif"/></noscript></span></span> vector of the random error terms. An ordinary least-squared estimator of the model coefficients can be written as<span class="disp-formula" id="disp-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-29.gif"/></noscript></span><span class="disp-formula-label">(9)</span></span>with the variance–covariance matrix of <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-30.gif"/></noscript></span></span> having the form<span class="disp-formula" id="disp-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-31.gif"/></noscript></span><span class="disp-formula-label">(10)</span></span>where <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-32.gif"/></noscript></span></span> is the variance-covariance matrix of <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-33.gif"/></noscript></span></span> To estimate regression coefficients, <em>β</em> in the response function requires data from experiments designed to meet the objective (<a id="xref-ref-23-3" class="xref-bibr" href="#ref-23">Myers and Montgomery 1995</a>). If the objective is to approximate the response surface, a frequently used treatment design is the factorial. For example, temperature and the degree of drought could be two factors affecting yield, but the number of factors could be more than two and the possible values per factor can be qualitative, quantitative, and numerous. For the example illustrated in <a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1</a>, a reasonable full factorial model would consist of <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-34.gif"/></noscript></span></span> factor combinations. To observe a response at each factor combination when there are two levels for the <em>p</em> factors, <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-35.gif"/></noscript></span></span> unreplicated treatment combinations are required and the design is called the <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-36.gif"/></noscript></span></span> factorial design. When <em>p</em> is large and the range of possible values of each factor is also large, finding the combination of the <em>p</em> factors needed to approximate the response surface increases exponentially. For example, three factors with two levels each requires <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-37.gif"/></noscript></span></span> treatment combinations; whereas if the number of factors is five, the number of factor combinations is <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-38.gif"/></noscript></span></span> At least some of the treatment combinations would need to be replicated if we want an estimate of the variance of residuals, <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-39.gif"/></noscript></span></span></p></div><div id="sec-3" class="subsection"><h3>Efficient designs to find the optimal response</h3><p id="p-13">Often the research objective is to identify conditions where the response is optimal, not to characterize the entire response surface. For such an objective, an experimental strategy needs only to determine the combinations of factors that optimize the response. For example, it is possible to find the maximum yield due to temperature and drought with as few as six, instead of 20, treatment combinations. Conceptually, these strategies are based on the sequential evaluation of subsets of the full factorial design. The key is to design the subsets of four-factor combinations in a manner that will maximize the information about the direction and distance to the optimum from data collected on the responses.</p><p id="p-14">Choice of initial subsets of factor combinations are based on recognition that in a <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-40.gif"/></noscript></span></span> factorial design, <em>p</em> degrees of freedom from a total <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-41.gif"/></noscript></span></span> are used to estimate the main effects while the remaining degrees of freedom are used to estimate interactions among the factors. In an initial subset of factor combinations, it is reasonable to assume that the second-order and nonlinear aspects of the response surface are not of initial interest and use a fractional factorial design. For example, consider the half of the full factorial design also known as the <span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-42.gif"/></noscript></span></span> design for a response that is influenced by three factors, <em>e.g.</em>, temperature, drought, and fertilizer, denoted <span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-43.gif"/></noscript></span></span> and <em>C</em>. The half-factorial design in this case requires <span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-44.gif"/></noscript></span></span> treatment combinations. Even though this first exploratory experiment is less expensive than the complete factorial experiment, the fractional-factorial design will involve aliasing of factors. In other words, some effects are confounded and cannot be estimated independently. In this case of the <span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-45.gif"/></noscript></span></span> design, the main effects are confounded with the two-factor interactions; a subsequent experiment is needed to disentangle the confounded effects. Explicitly, consider all of the treatment combinations <span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-46.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-47.gif"/></noscript></span></span> in a <span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-48.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-48.gif"/></noscript></span></span> design (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). The factorial effects are <span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-49.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-49.gif"/></noscript></span></span> where <em>I</em> is the identity column used to estimate the average response. The − symbol stands for a low level (level 1) of a factor, and + stands for a high level (level 2) of a factor.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/486497/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/486497/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Combinations and factorial effects for a <span class="inline-formula" id="inline-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/T1/embed/mml-math-50.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/T1/embed/mml-math-50.gif"/></noscript></span></span> design</span><div class="sb-div caption-clear"></div></div></div><p id="p-16">The identity column, <em>I</em>, is always <span class="inline-formula" id="inline-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-53.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-53.gif"/></noscript></span></span> and we can write<span class="disp-formula" id="disp-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-54.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-54.gif"/></noscript></span><span class="disp-formula-label">(11)</span></span>Equation 11 is called the defining relation for the design, and represents the relationship between the identity and a factorial effect, which determines the aliasing pattern. Multiplying both sides of (11) by <em>C</em> results in<span class="disp-formula" id="disp-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-55.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-55.gif"/></noscript></span><span class="disp-formula-label">(12)</span></span>However, the square of any column (factorial effect) is the identity <em>I</em>, so we get<span class="disp-formula" id="disp-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-56.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-56.gif"/></noscript></span><span class="disp-formula-label">(13)</span></span>Using the defining relation, the factorial effect of factor <em>C</em> for every treatment combination can be determined. To create a <span class="inline-formula" id="inline-formula-44"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-57.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-57.gif"/></noscript></span></span> fractional-factorial design we can consider the treatment combinations where the <span class="inline-formula" id="inline-formula-45"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-58.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-58.gif"/></noscript></span></span> factorial effect has a + sign. Note that these are <span class="inline-formula" id="inline-formula-46"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-59.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-59.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-47"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-60.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-60.gif"/></noscript></span></span> treatment combinations listed in the top half of <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>. Also, we can consider the four treatment combinations where the corresponding <span class="inline-formula" id="inline-formula-48"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-61.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-61.gif"/></noscript></span></span> factorial effect has a − sign. We call this the complementary fraction. These treatment combinations are the <span class="inline-formula" id="inline-formula-49"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-62.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-62.gif"/></noscript></span></span> and (1), thus illustrating the confounding interaction effects with the main effects. It does not matter which fraction we take because both belong to the same family of treatment designs.</p><p id="p-17">After designing and conducting the fractional-factorial experiment for the four treatment combinations, at least four responses will provide information about the next set of treatments that should be conducted to increase the value of the response. One algorithm to do so is <em>steepest ascent</em> (or <em>steepest descent</em>). Steepest ascent is a sequential approach for finding the maximum response, where we search for a region of the factor space where the response is improved. The method of steepest ascent has three main steps (<a id="xref-ref-23-4" class="xref-bibr" href="#ref-23">Myers and Montgomery 1995</a>):</p><ol class="list-ord " id="list-1"><li id="list-item-1"><p id="p-18">Design of factor combinations with replicates.</p></li><li id="list-item-2"><p id="p-19">Model building.</p></li><li id="list-item-3"><p id="p-20">Sequential experimentation.</p></li></ol><p id="p-21">Since the method of steepest ascent is a sequential procedure, the three steps are typically repeated until the optimum response is obtained. That is, steepest ascent can involve several experiments consisting of subsets of factor combinations that lead to the maximum response. The subsets of factor combinations in each experiment depend on the estimates of the regression coefficients of the model from prior experiments. For illustration, consider a first-order regression model<span class="disp-formula" id="disp-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-63.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-63.gif"/></noscript></span><span class="disp-formula-label">(14)</span></span>Changing values of <span class="inline-formula" id="inline-formula-50"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-64.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-64.gif"/></noscript></span></span> relative to the other factors depends on the estimated regression coefficient <span class="inline-formula" id="inline-formula-51"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-65.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-65.gif"/></noscript></span></span> The magnitude of <span class="inline-formula" id="inline-formula-52"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-66.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-66.gif"/></noscript></span></span> provides the rate or number of steps relative to the <span class="inline-formula" id="inline-formula-53"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-67.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-67.gif"/></noscript></span></span> coordinate, and the sign of <span class="inline-formula" id="inline-formula-54"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-68.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-68.gif"/></noscript></span></span> tells us the direction for the next set of factor levels. If, for example, the magnitude of <span class="inline-formula" id="inline-formula-55"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-69.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-69.gif"/></noscript></span></span> is twice as much as the magnitude of <span class="inline-formula" id="inline-formula-56"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-70.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-70.gif"/></noscript></span></span> then <span class="inline-formula" id="inline-formula-57"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-71.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-71.gif"/></noscript></span></span> will change twice as fast as <span class="inline-formula" id="inline-formula-58"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-72.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-72.gif"/></noscript></span></span> for the same change in factor levels.</p><p id="p-22">Simple first-order models are typically used in the initial experiments. Unless the surface is complex or the initial fractional factorial uses levels that are far from the optimal region, only a few steps will be needed to move quickly into “the neighborhood” of the optimum response. As factor combinations approach the optimum, second-order models that include interaction terms and curvature are employed to determine more accurate approximations of the underlying surface.</p></div></div><div class="section methods" id="sec-4"><h2 class="">Methods</h2><div id="sec-5" class="subsection"><h3>Simulated response surface</h3><p id="p-23">Doubled haploid (DH) populations were simulated using R (<a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">R Development Core Team 2008</a>, <a href="http://www.r-project.org">http://www.r-project.org</a>). The R code for performing the simulations, and the predictions, can be found in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.044453/-/DC1/FileS2.txt">File S2</a>. Since our objective is to demonstrate application of steepest ascent to determine the maximum response, we first simulated the entire response surface using a list of five factors (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>) with operability regions specified by factor levels (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>). For the number of segregating progeny, number of markers, number of QTL, and level of heritability three levels were evaluated, and for epistasis five levels were evaluated. Epistasis at level 0 means that all of the genetic variance is additive, 0.5 epistasis means that half of the genetic variance is additive and the other half is epistatic. Thus, the response surface is characterized by the <span class="inline-formula" id="inline-formula-59"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-73.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-73.gif"/></noscript></span></span> factor combinations (<a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>).</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/486381/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/486381/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Possible values for <em>n</em>, <em>m</em>, <em>QTL</em>, <em>epi</em>, and <em>h</em></span><div class="sb-div caption-clear"></div></div></div><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/486421/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/486421/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Specification of the factors including <em>n</em>, <em>m</em>, <em>QTL</em>, <em>epi</em>, and <em>h</em></span><div class="sb-div caption-clear"></div></div></div><p id="p-26">The simulated genomes of the DHs had 10 chromosomes, each having the same length. The markers were distributed throughout the genome in such a way that each chromosome had the same number of markers equally spaced along the length of each chromosome. There were no missing genotypic values and no missing phenotypic values. The recombination rate was simulated as a function of the number of the marker loci within a linkage group. The phenotypic values are simulated based on the model described as<span class="disp-formula" id="disp-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-74.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-74.gif"/></noscript></span><span class="disp-formula-label">(15)</span></span>where Pheno is a vector of length <em>n</em> (<em>n</em> is the number of segregating progeny), <span class="inline-formula" id="inline-formula-60"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-75.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-75.gif"/></noscript></span></span> is an (<span class="inline-formula" id="inline-formula-61"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-76.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-76.gif"/></noscript></span></span>)-dimensional additive incidence matrix where <em>q</em> is the number of QTL, <strong>a</strong> is a <em>q</em>-long vector of additive effects, <span class="inline-formula" id="inline-formula-62"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-77.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-77.gif"/></noscript></span></span> is the (<span class="inline-formula" id="inline-formula-63"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-78.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-78.gif"/></noscript></span></span>)-dimensional epistatic incidence matrix, <strong>epi</strong> is a 2<em>q</em>-long vector of epistatic effects, and <em>ϵ</em> is an <em>n</em>-long vector of random errors. <em>ϵ</em> has a normal distribution with mean of 0 and variance determined by h (proportion of phenotypic variance due to genetic variance) and the genetic variance. <strong>a</strong> and <strong>epi</strong> are simulated in such a way that <span class="inline-formula" id="inline-formula-64"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-79.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-79.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-65"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-80.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-80.gif"/></noscript></span></span> where <em>a</em> and <span class="inline-formula" id="inline-formula-66"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-81.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-81.gif"/></noscript></span></span> are constants, and <strong>I</strong> is the identity vector of length specified by the model.</p><p id="p-27">The genetic variance is defined as<span class="disp-formula" id="disp-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-82.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-82.gif"/></noscript></span><span class="disp-formula-label">(16)</span></span>where <span class="inline-formula" id="inline-formula-67"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-83.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-83.gif"/></noscript></span></span> is the additive genetic variance and <span class="inline-formula" id="inline-formula-68"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-84.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-84.gif"/></noscript></span></span> is the epistatic genetic variance. The genotypic values were coded according to Cockerham’s model (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Kao and Zeng 2002</a>), and epistatic interactions were simulated between pairs of neighboring QTL. Thus, we only considered two-way gene interactions among QTL. The proportion of genetic variance explained by the additive and epistatic parts can be specified by the user of the simulation software, and is determined by using Equation 16.</p><p id="p-28">To determine the response, <em>i.e.</em>, accuracy of prediction at all 405 factor and factor level combinations, phenotypes were predicted using ridge regression BLUP and SVM methods. BLUP is a parametric statistical procedure for prediction consisting of a random effect term for the marker genotypes (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Henderson <em>et al.</em> 1959</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Henderson 1963</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Bernardo 1994</a>; <a id="xref-ref-14-4" class="xref-bibr" href="#ref-14">Howard <em>et al.</em> 2014</a>). SVM is a nonparametric machine learning technique that can model the relationship between the marker values and the phenotypes using a linear or a nonlinear mapping function (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Vapnik 1995</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Hastie <em>et al.</em> 2009</a>; <a id="xref-ref-14-5" class="xref-bibr" href="#ref-14">Howard <em>et al.</em> 2014</a>). Specifically, we used RRBLUP (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Endelman 2011</a>) and SVM (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Karatzoglou <em>et al.</em> 2004</a>) implemented in R (<a id="xref-ref-26-3" class="xref-bibr" href="#ref-26">R Development Core Team 2008</a>, <a href="http://www.r-project.org">http://www.r-project.org</a>). Predictive accuracies were estimated using cross-validation, where the data were divided into training and testing sets. Let <span class="inline-formula" id="inline-formula-69"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-85.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-85.gif"/></noscript></span></span> denote the true phenotypic values at factor combination <em>i</em>, and let <span class="inline-formula" id="inline-formula-70"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-86.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-86.gif"/></noscript></span></span> denote the estimated phenotypic values at factor combination <em>i</em>. Accuracy of prediction <span class="inline-formula" id="inline-formula-71"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-87.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-87.gif"/></noscript></span></span> is defined as the correlation between the true phenotypic values and the predicted phenotypic values (<a id="xref-ref-14-6" class="xref-bibr" href="#ref-14">Howard <em>et al.</em> 2014</a>). Prediction accuracies were estimated at each treatment combination with 500 replications consisting of 20 different genotypic–phenotypic data sets, and within each data set we divided the marker and phenotypic data into 25 different training–testing subsets. The training–testing data sets were created in a such way that a random <span class="inline-formula" id="inline-formula-72"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-88.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-88.gif"/></noscript></span></span> of the individuals belong to the testing set, and the remaining <span class="inline-formula" id="inline-formula-73"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-89.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-89.gif"/></noscript></span></span> belong to the training set. We looked at the difference in prediction accuracy for SVM and BLUP techniques, and the prediction accuracy for BLUP.</p></div><div id="sec-6" class="subsection"><h3>Data availability</h3><p id="p-29">The simulation code that produces <a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1</a> can be found in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.044453/-/DC1/FileS1.txt">File S1</a>, and the code for simulations and predictions can be found in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.044453/-/DC1/FileS2.txt">File S2</a>.</p></div></div><div class="section results" id="sec-7"><h2 class="">Results</h2><div id="sec-8" class="subsection"><h3>Response surfaces</h3><p id="p-30">Three response surfaces were generated: one based on prediction accuracies of BLUP, <span class="inline-formula" id="inline-formula-74"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-90.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-90.gif"/></noscript></span></span> a second based on prediction accuracies of SVM, <span class="inline-formula" id="inline-formula-75"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-91.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-91.gif"/></noscript></span></span> and the third based on the differences of the prediction accuracies, <span class="inline-formula" id="inline-formula-76"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-92.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-92.gif"/></noscript></span></span> The left panel of <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a> is a histogram of the differences in the prediction accuracies, <span class="inline-formula" id="inline-formula-77"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-93.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-93.gif"/></noscript></span></span> and the right panel is a histogram for <span class="inline-formula" id="inline-formula-78"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-94.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-94.gif"/></noscript></span></span> The histograms are based on average prediction accuracies from 500 replicates for all 405 factor combinations (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.044453/-/DC1/TableS1.xlsx">Table S1</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/7/9/3103/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Histogram of differences of prediction accuracies between SVM and BLUP shown in the left panel. Histogram of prediction accuracies for BLUP shown in the right panel." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1752734692" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Histogram of differences of prediction accuracies between SVM and BLUP shown in the left panel. Histogram of prediction accuracies for BLUP shown in the right panel.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/7/9/3103/F2.medium.gif" width="440" height="217"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/7/9/3103/F2.medium.gif" width="440" height="217"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/7/9/3103/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/7/9/3103/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/486498" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-31" class="first-child">Histogram of differences of prediction accuracies between SVM and BLUP shown in the left panel. Histogram of prediction accuracies for BLUP shown in the right panel.</p><div class="sb-div caption-clear"></div></div></div><p id="p-32">Most factor combinations produce similar prediction accuracies (left panel of <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>), although there are treatment combinations where SVM is more accurate than BLUP. The maximum difference between SVM and BLUP is 0.3, and this occurs when the number of segregating progeny is 2000 (which is the maximum we considered), the number of markers is 100 (which is the minimum we considered), h is one, and the proportion of epistatic variance relative to the total genotypic variance is one. For only <span class="inline-formula" id="inline-formula-79"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-95.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-95.gif"/></noscript></span></span> of the treatment combinations, the difference between the prediction accuracies for SVM and BLUP is &gt;0.20, and for all of these cases epistasis and h are at their maximum limits, and the numbers of segregating progenies are 2000. Note that <span class="inline-formula" id="inline-formula-80"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-96.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-96.gif"/></noscript></span></span> has a wide range of values depending on the treatment combinations (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a>). The maximum <span class="inline-formula" id="inline-formula-81"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-97.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-97.gif"/></noscript></span></span> is 0.80 when the number of segregating progeny is 2000, h is at its maximum limit, and the proportion of epistatic variability accounting for the genetic variance is zero. For <span class="inline-formula" id="inline-formula-82"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-98.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-98.gif"/></noscript></span></span> of the treatment combinations, the prediction accuracy is &gt;0.80 and in all those cases the number of segregating progeny is 2000, h is one, and the proportion of epistatic variability accounting for the genetic variance is zero. There was no pattern detected for the number of markers and the number of QTL (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.044453/-/DC1/TableS1.xlsx">Table S1</a>).</p></div><div id="sec-9" class="subsection"><h3>Steepest ascent to determine factor combinations for optimal responses</h3><p id="p-33">In the steepest ascent RSM, our goal is to find the combination of factor levels associated with the optimal response <em>y</em> without evaluating every possible factor combination of the response surface. A full factorial design using five factors with two factor levels would imply <span class="inline-formula" id="inline-formula-83"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-99.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-99.gif"/></noscript></span></span> experimental treatment combinations. Several different starting points for the factors could influence the results. Initial levels for each factor had no impact on the final estimate of the maximum response (data not shown). However, initial values did impact the number of subsets of factor combinations that were needed to reach the maximum. For the sake of brevity, we provide results of the dynamic decision process for only one set of initial values that were located furthest from the optimum (<a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>). Let <span class="inline-formula" id="inline-formula-84"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-100.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-100.gif"/></noscript></span></span> be the difference between the accuracy of prediction using the BLUP and the SVM method. Let <span class="inline-formula" id="inline-formula-85"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-101.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-101.gif"/></noscript></span></span> be the prediction accuracy for the BLUP method. Since the calculations for SVM are comparable to the calculations for BLUP, we only included <span class="inline-formula" id="inline-formula-86"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-102.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-102.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-87"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-103.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-103.gif"/></noscript></span></span> as responses. The response depends on the set of design variables <span class="inline-formula" id="inline-formula-88"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-104.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-104.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-89"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-105.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-105.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-90"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-106.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-106.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-91"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-107.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-107.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-92"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-108.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-108.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-93"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-109.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-109.gif"/></noscript></span></span> is the number of individuals in the simulated DH population, <span class="inline-formula" id="inline-formula-94"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-110.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-110.gif"/></noscript></span></span> is the number of markers, <span class="inline-formula" id="inline-formula-95"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-111.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-111.gif"/></noscript></span></span> is the number of QTL, <span class="inline-formula" id="inline-formula-96"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-112.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-112.gif"/></noscript></span></span> is the proportion of genetic variability due to epistasis, and <span class="inline-formula" id="inline-formula-97"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-113.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-113.gif"/></noscript></span></span> is the proportion of phenotypic variance due to genetic variance. A model can be written as<span class="disp-formula" id="disp-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-114.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-114.gif"/></noscript></span><span class="disp-formula-label">(17)</span></span>where <em>y</em> is the response; <em>f</em> is the unknown, possibly complex response function, which depends on the design variables <span class="inline-formula" id="inline-formula-98"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-115.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-115.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-99"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-116.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-116.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-100"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-117.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-117.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-101"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-118.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-118.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-102"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-119.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-119.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-103"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-120.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-120.gif"/></noscript></span></span> iid <span class="inline-formula" id="inline-formula-104"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-121.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-121.gif"/></noscript></span></span> The expected value of the response function can be written as<span class="disp-formula" id="disp-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-122.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-122.gif"/></noscript></span><span class="disp-formula-label">(18)</span></span><span class="disp-formula" id="disp-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-123.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-123.gif"/></noscript></span><span class="disp-formula-label">(19)</span></span>Initially, use a first-order polynomial to approximate the response function, <em>f</em>, so that<span class="disp-formula" id="disp-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-124.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-124.gif"/></noscript></span><span class="disp-formula-label">(20)</span></span>where <span class="inline-formula" id="inline-formula-105"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-125.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-125.gif"/></noscript></span></span> is the intercept, <span class="inline-formula" id="inline-formula-106"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-126.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-126.gif"/></noscript></span></span> is the regression coefficient associated with the number of individuals, <span class="inline-formula" id="inline-formula-107"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-127.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-127.gif"/></noscript></span></span> is the regression coefficient associated with the number of markers, <span class="inline-formula" id="inline-formula-108"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-128.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-128.gif"/></noscript></span></span> is the regression coefficient associated with the number of QTL, <span class="inline-formula" id="inline-formula-109"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-129.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-129.gif"/></noscript></span></span> is the regression coefficient associated with the proportion of genetic variation due to epistasis, and <span class="inline-formula" id="inline-formula-110"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-130.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-130.gif"/></noscript></span></span> is the regression coefficient associated with the proportion of phenotypic variability due to genotypic variability.</p><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/486529/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/486529/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">Starting values for the factors <em>ind</em>, <em>m</em>, <em>QTL</em>, <em>epi</em>, and <em>h</em> in terms of natural units and coded units</span><div class="sb-div caption-clear"></div></div></div><p id="p-35">Average responses <span class="inline-formula" id="inline-formula-111"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-131.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-131.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-112"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-132.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-132.gif"/></noscript></span></span> and their difference from 500 replicates of the half-fractional factorial of 16 factor combinations (<a id="xref-table-wrap-5-1" class="xref-table" href="#T5">Table 5</a>) were used to determine subsequent subsets of factor combinations that should be close to the optimum response.</p><div id="T5" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/486431/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/486431/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 5</span> <span class="caption-title">Mean accuracy of BLUP, mean accuracy of SVM, and the difference between the mean accuracy of SVM and mean accuracy of BLUP for 16 combinations of factors</span><div class="sb-div caption-clear"></div></div></div><p id="p-36">The mean accuracy difference between the SVM and BLUP, and the estimates of the regression coefficients in terms of the coded units for the response <span class="inline-formula" id="inline-formula-113"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-133.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-133.gif"/></noscript></span></span> for the half factorial were<span class="disp-formula" id="disp-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-134.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-134.gif"/></noscript></span><span class="disp-formula-label">(21)</span></span>Based on the estimated coefficients, increasing the number of progeny, the proportion of total genetic variance due to epistasis, and the heritability, and decreasing the number of markers and QTL should improve the response. The coefficients with the smallest magnitudes include number of markers and QTL. Thus, these are not as influential as the other factors in terms of the next set of factor combinations which will improve the response (<span class="inline-formula" id="inline-formula-114"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-135.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-135.gif"/></noscript></span></span>).</p><p id="p-37">To determine the next set of factor combinations, we define a basis and calculate the step sizes (increments) for each factor. <a id="xref-table-wrap-6-1" class="xref-table" href="#T6">Table 6</a> shows the low and high levels (level 1 and level 2) of the factors, the average of the two levels of the factors (which is the <em>base</em> in later calculations), and the distance between the average and either of the initial values (which is used for calculating the coordinates of the response surface which need to be evaluated next).</p><div id="T6" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/486500/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/486500/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 6</span> <span class="caption-title">Levels of factors, average of the levels of the factors, and half of the difference between the levels of factors</span><div class="sb-div caption-clear"></div></div></div><p id="p-38">For establishing which treatment combinations need to be evaluated next to find the maximum response, the step size of the input variables needs to be determined. First, an input variable needs to be chosen (called the basis) which will also influence the step size of the other variables. It is beneficial to choose a variable for which the most information is available, but it is also a common practice to choose the basis with the largest absolute value of the estimated regression coefficient.</p><p id="p-39">Because in our example the fitted model has the largest absolute estimated coefficient for epistasis, epistasis is the basis. We choose the step size for epistasis to be 0.25 because previous research indicated that the maximum response (<span class="inline-formula" id="inline-formula-117"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-138.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-138.gif"/></noscript></span></span>) can be found when epistasis is high. The step size will only determine the number of additional experiments we have to run to reach the optimum. If we were to specify a smaller step size for epistasis, we would move more slowly on the surface by evaluating more treatment combinations. However, when the surface is not smooth and is complex, it might be beneficial to choose a step size for the first input variable that is small. The choice of the step size of the first input variable is determined by the researcher, and it influences the step size of the other input variables.</p><p id="p-40">The step size of 0.25 for epistasis in natural units corresponds to <span class="inline-formula" id="inline-formula-118"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-139.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-139.gif"/></noscript></span></span> in coded units (the value of 0.15 comes from <a id="xref-table-wrap-6-2" class="xref-table" href="#T6">Table 6</a>). The step sizes for the other input variables in coded units are<span class="disp-formula" id="disp-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-140.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-140.gif"/></noscript></span><span class="disp-formula-label">(22)</span></span>explicitly:<span class="disp-formula" id="disp-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-141.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-141.gif"/></noscript></span><span class="disp-formula-label">(23)</span></span>for individuals,<span class="disp-formula" id="disp-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-142.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-142.gif"/></noscript></span><span class="disp-formula-label">(24)</span></span>for markers,<span class="disp-formula" id="disp-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-143.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-143.gif"/></noscript></span><span class="disp-formula-label">(25)</span></span>for QTL, and<span class="disp-formula" id="disp-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-144.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-144.gif"/></noscript></span><span class="disp-formula-label">(26)</span></span>for h.</p><p id="p-41">To determine the coordinates of the response surface (<em>i.e.</em>, experimental conditions) that need to be evaluated next, we need to convert the step sizes in coded units into natural units. The step size in natural units can be calculated as the product of <span class="inline-formula" id="inline-formula-119"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-145.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-145.gif"/></noscript></span></span> and the corresponding step size in coded units (<a id="xref-table-wrap-7-1" class="xref-table" href="#T7">Table 7</a>). For example, for the number of individuals the step size in natural units is calculated as <span class="inline-formula" id="inline-formula-120"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-146.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-146.gif"/></noscript></span></span> The base in natural units for the input variables corresponds to a base of zero in coded units. <span class="inline-formula" id="inline-formula-121"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-147.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-147.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-122"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-148.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-148.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-123"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-149.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-149.gif"/></noscript></span></span> … indicate the direction and magnitude of change for each factor which can be used for the next set of experimental conditions; and base<span class="inline-formula" id="inline-formula-124"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-150.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-150.gif"/></noscript></span></span> base<span class="inline-formula" id="inline-formula-125"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-151.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-151.gif"/></noscript></span></span> base<span class="inline-formula" id="inline-formula-126"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-152.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-152.gif"/></noscript></span></span> … specify the coordinates of the response surface that need to be evaluated next (<a id="xref-table-wrap-8-1" class="xref-table" href="#T8">Table 8</a>).</p><div id="T7" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/486414/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/486414/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 7</span> <span class="caption-title">Base, step size in natural units, and the coordinates of the steepest ascent for the number of individuals, number of markers, number of QTL, proportion of epistasis, and the degree of heritability to determine the second set of factor combinations for response <span class="inline-formula" id="inline-formula-127"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/T7/embed/mml-math-153.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/T7/embed/mml-math-153.gif"/></noscript></span></span></span><div class="sb-div caption-clear"></div></div></div><div id="T8" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/486386/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/486386/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 8</span> <span class="caption-title">Coordinates of the steepest ascent for the number of individuals, number of markers, number of QTL, proportion of epistasis, and the degree of heritability for the additional runs when the response is the mean accuracy difference between the SVM and BLUP, and the corresponding mean accuracy for BLUP, SVM, and for SVM–BLUP</span><div class="sb-div caption-clear"></div></div></div><p id="p-42">After adjusting for the operability regions of the five factors, the second set of experimental conditions for <span class="inline-formula" id="inline-formula-140"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-166.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-166.gif"/></noscript></span></span> and associated responses indicated that the increased number of individuals, proportion of genetics due to epistasis, and h, and the decreased number of markers and QTL produced improved responses. As the number of segregating progeny, the proportion of epistasis, and h increase, the advantage of using the SVM method instead of BLUP increases. The total number of factor combinations evaluated to find the maximum <span class="inline-formula" id="inline-formula-141"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-167.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-167.gif"/></noscript></span></span> was &lt;25, instead of the 405 which would be required to describe the entire surface (<em>Response surfaces</em>).</p><p id="p-43">The estimated regression coefficients for the averaged response <span class="inline-formula" id="inline-formula-142"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-168.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-168.gif"/></noscript></span></span> are listed in the estimated regression line:<span class="disp-formula" id="disp-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-169.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-169.gif"/></noscript></span><span class="disp-formula-label">(27)</span></span>indicating that increasing the number of progeny, number of QTL, and h, and decreasing the number of markers and the proportion of genetic variance due to epistasis should increase the response. The estimated coefficients are the smallest for the number of markers and QTL, so these factors are not as influential as the other factors. Because the largest estimated coefficient in absolute value is associated with epistasis, the basis is epistasis and the corresponding step size is <span class="inline-formula" id="inline-formula-143"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-170.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-170.gif"/></noscript></span></span> for the response of accuracy of prediction using BLUP. Note that the step size of the basis has a negative value here because the estimated regression coefficient for epistasis is negative. The step size for epistasis <span class="inline-formula" id="inline-formula-144"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-171.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-171.gif"/></noscript></span></span> in natural units corresponds to <span class="inline-formula" id="inline-formula-145"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-172.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-172.gif"/></noscript></span></span> in coded units. For the other four factors, the step sizes in coded units are<span class="disp-formula" id="disp-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-173.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-173.gif"/></noscript></span><span class="disp-formula-label">(28)</span></span>for individuals,<span class="disp-formula" id="disp-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-174.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-174.gif"/></noscript></span><span class="disp-formula-label">(29)</span></span>for markers,<span class="disp-formula" id="disp-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-175.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-175.gif"/></noscript></span><span class="disp-formula-label">(30)</span></span>for QTL, and<span class="disp-formula" id="disp-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-176.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-176.gif"/></noscript></span><span class="disp-formula-label">(31)</span></span>for heritability.</p><p id="p-44">Factor levels for the next set of analyses resulted in decreasing factor levels for epistasis and number of markers (<a id="xref-table-wrap-9-1" class="xref-table" href="#T9">Table 9</a> and <a id="xref-table-wrap-10-1" class="xref-table" href="#T10">Table 10</a>) for maximizing <span class="inline-formula" id="inline-formula-146"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-177.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-177.gif"/></noscript></span></span> The optimum response for averaged <span class="inline-formula" id="inline-formula-147"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-178.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-178.gif"/></noscript></span></span> occurs when the genetic variance is explained by purely the additive effects. For both responses (<span class="inline-formula" id="inline-formula-148"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-179.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-179.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-149"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-180.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-180.gif"/></noscript></span></span>), the response increases when the number of segregating progeny and heritability increase. For both responses, it is beneficial to have markers which explain the phenotypic variation, and it does not improve prediction accuracies when markers which are not associated with the phenotypic variability are added. The total number of factor combinations needed to find the maximum responses for <span class="inline-formula" id="inline-formula-150"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-181.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-181.gif"/></noscript></span></span> was 21, ∼5% of what would be required to describe the entire response surface.</p><div id="T9" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/486552/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/486552/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 9</span> <span class="caption-title">Base, step size in natural units, and the coordinates of the steepest ascent for the number of individuals, number of markers, number of QTL, proportion of epistasis, and the degree of heritability for response <span class="inline-formula" id="inline-formula-151"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/T9/embed/mml-math-182.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/T9/embed/mml-math-182.gif"/></noscript></span></span></span><div class="sb-div caption-clear"></div></div></div><div id="T10" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/486437/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/486437/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 10</span> <span class="caption-title">Coordinates of the steepest ascent for the number of individuals, number of markers, number of QTL, proportion of epistasis, and the degree of heritability for the additional runs when the response is the mean accuracy for BLUP, and the corresponding mean accuracy for BLUP</span><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-10"><h2 class="">Discussion</h2><div id="sec-11" class="subsection"><h3>Some considerations about the factors</h3><p id="p-45">In 2014, we conjectured that differences between estimated prediction accuracies of linear-model and algorithmic methods could be used as a computational diagnostic for an epistatic genetic architecture. <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Huang <em>et al.</em> (2012)</a> had previously alluded to failure of linear model-based methods as an indicator of epistatic genetic architectures. Thus, we had an obligation to investigate the hypothesis using a more thorough and systematic approach. We learned that the response surface for the computational diagnostic, (<span class="inline-formula" id="inline-formula-157"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-188.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-188.gif"/></noscript></span></span>), is flat except in the vicinity of maximum values for the proportional contribution of epistasis to genetic variance, and the proportional contribution of genetic variance to phenotypic variability. To our knowledge there are no known quantitative traits that exhibit such genetic architectures.</p><p id="p-46">Before designing the experiments for the steepest ascent/descent procedure, it is important to evaluate some initial experiments where we determine which factors might be important and which can be excluded from the model. We also have to define the region in which the factors can affect the response, also known as the operability region.</p><p id="p-47">The initial choices of the factors we consider and their range of values can have a large impact on the speed of approaching the optimal response, and whether the optimum is reached. For example, for the factor temperature we can choose the range to be between <span class="inline-formula" id="inline-formula-158"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-189.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-189.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-159"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-190.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-190.gif"/></noscript></span></span> or we can convert to the Celsius scale which will lead to a range between <span class="inline-formula" id="inline-formula-160"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-191.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-191.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-161"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-192.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-192.gif"/></noscript></span></span> The estimated regression coefficients will be different depending on the temperature scale. Using different ranges of factors only influences the magnitude of the regression coefficients, not the sign of the regression coefficients. This implies that using a different range for the factors would not modify the direction in which we are moving along the path, but it would change the speed of the movement relative to the scale used.</p><p id="p-48">Another design aspect to consider is the choice of the metric for the response. Especially when the range of the response is large, it is useful to transform the response. One of the most commonly used transformations is the Box–Cox power transformation (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Box and Cox 1964</a>). The transformed response, <em>w</em> is defined as<span class="disp-formula" id="disp-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-193.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-193.gif"/></noscript></span><span class="disp-formula-label">(32)</span></span>where <em>y</em> is the untransformed response and <em>λ</em> is the power parameter. For example, <span class="inline-formula" id="inline-formula-162"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-194.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-194.gif"/></noscript></span></span> results in a reciprocal transformation, <span class="inline-formula" id="inline-formula-163"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-195.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-195.gif"/></noscript></span></span> results in a square root transformation, and since <span class="inline-formula" id="inline-formula-164"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-196.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-196.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-165"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-197.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-197.gif"/></noscript></span></span> results in a logarithmic transformation. The power parameter <em>λ</em> can be estimated via maximum likelihood. This rank-preserving transformation is also useful when we need to stabilize the variance of the response, or when the residual variance does not satisfy the normal assumptions. It is possible that the response surface has multiple peaks, and then more than one combination of the design variables satisfies the condition for having the optimal response. It is also likely that the range of starting values do not include the global peaks, but in the procedure of steepest descent (or ascent) we would arrive in the required range.</p></div><div id="sec-12" class="subsection"><h3>Application of RSMs for evaluation of GP methods</h3><p id="p-49">Originally, RSMs (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Box and Wilson 1951</a>) were developed to find combinations of controlled conditions to maximize output of industrial processes (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Naylor 1969</a>). <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Myers <em>et al.</em> (1989)</a> summarized the extensive applications of RSMs for systems engineering; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Bezerra <em>et al.</em> (2008)</a> summarized applications in analytical chemistry; and RSMs have been applied to systems of interest to biologists, including pharmaceutical production (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Koyamada <em>et al.</em> 2004</a>) and fermentation (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Zhang <em>et al.</em> 2010</a>). Our own experiences in consulting have revealed a misperception that RSMs can only be applied to systems where the factors and factor levels can be explicitly planned and controlled as fixed-effect treatments. However, RSMs have been shown to be efficient for determining optimal conditions of complex systems where many of the factors are represented as random samples or unknown factors, <em>e.g.</em>, ecosystem impacts on growth and development of individuals (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Menke 1973</a>). Herein, we demonstrated that the steepest ascent RSM also provides an efficient approach to identify conditions that affect prediction accuracies of a machine learning and a BLUP method. The purpose here is not to conduct many different experiments, but to evaluate the sensitivity of the GP models to the underlying genetic architecture and design factors with a limited number of experiments. To our knowledge, this is the first report of RSM for GP methods and a demonstration of how to find factor combinations that are responsible for maximizing prediction accuracies using an efficient number of experimental analyses. Further, we demonstrated that the steepest ascent RSM also provided useful information about the response surface without the burden of evaluating every possible combination of factors and factor levels.</p><p id="p-50">Even though we demonstrated RSM using GP methods, the methodology can be applied during development of any novel data analyses or computational methods, although this proposition requires more research. There are some aspects of our application of an RSM that need to be emphasized. First, steepest ascent represents only one of many RSMs, and use of a fractional factorial represents only one of many efficient experimental designs. We chose these because our goal was to find a set of factor levels in which the responses were maximized and we had some prior information about possible factors. The set of factors that we investigate might not be the only set associated with maximizing responses. For example, in case we find two sets of factor levels that are associated with the maximum response, suggesting that there might be a ridge of maximum responses, then a future ridge analysis RSM could be justified. Also, if we had some prior knowledge that our maximum responses were likely to occur at intermediate levels for our sets of factors, the central composite design would provide a more efficient design than the half-fractional factorial for the initial set.</p><p id="p-51">Before designing and conducting an initial experiment, knowledge about factors and factor levels should be incorporated into defining the operability regions, <em>i.e.</em>, ranges of values of the factors that are attainable. From prior publications, we were aware that genetic architecture of the trait (number of QTL, epistasis among QTL, and interactions of QTL with environments), population structure, relationships between training and validation data sets, number of genetic markers, heritability, and number of individuals (lines, varieties, and hybrids) could affect the predictive ability of GP methods. We were primarily interested in the potential of the computational diagnostic (<span class="inline-formula" id="inline-formula-166"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-198.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/7/9/3103/embed/mml-math-198.gif"/></noscript></span></span>) for detection of epistatic genetic architectures involving a large, but finite, number of QTL (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Clowers <em>et al.</em> 2010</a>; <a id="xref-ref-14-7" class="xref-bibr" href="#ref-14">Howard <em>et al.</em> 2014</a>). Further investigation is needed on larger numbers of QTL especially for traits like yield or biomass where the infinitesimal model is more appropriate. Also, since it is likely that the diagnostic will detect interactions of QTL with environments, we decided to avoid this confounded interpretation for this investigation. The operability region for epistasis was restricted to two-way interactions. Higher order interactions were not considered because <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Cooper <em>et al.</em> (2002)</a> pointed out that response to selection on an adaptive surface is unlikely to happen if the average number of interactions among QTL is much more than two. Crops have been responding to selection for at least 100 yr. Since it is trivial for simulation to produce any level of epistasis, the two-way interactions were simulated to represent the full range of contributions to genetic variance. Likewise, the operability region for heritability was simulated to represent the full range of contributions to phenotypic variance. We also kept the population structure and relationships among training and validation subsets consistent and simple to avoid confounded interpretations of the proposed diagnostic.</p><p id="p-52">With the emergence of genotyping-by-sequencing technologies, the operability region for the number of markers could have been much larger; however, given maximal linkage disequilibrium (LD) in the population structure, the operability region need only include sufficient numbers to assure some redundancy among genotypic information. Indeed, we found that if the LD between markers and QTL is complete, then prediction accuracies are not improved with any additional markers. This outcome will likely not change with more complex population structures for finite numbers of segregating QTL.</p><p id="p-53">The operability region for number of individuals was based on a possible number of DH progeny with application of DH technologies to <em>F</em><sub>2</sub> progeny from a single cross of inbred lines in a crop such as maize. While 2000 DH progeny are possible, producing such numbers is unlikely without significant resource allocations. Thus, from a practical perspective, it is highly unlikely that the diagnostic can be employed routinely because the maximum values for the diagnostic occurred with large numbers of progeny in which the genetic architecture is comprised entirely of epistatic variance and broad sense heritability is unity. Also, the large difference in accuracy between SVM and BLUP occurs at the extreme of pure epistatic variance, which is a genetic architecture that has not been described for any trait. Thus, the reader should be aware that the GP models can have a similar accuracy in cases when the genetic architecture consists of a high proportion of epistatic variance, and only comparing prediction accuracies might not be a useful method to infer genetic architecture.</p><p id="p-54">Lastly, the reader will note that prediction accuracy results are not exactly the same as results previously reported by <a id="xref-ref-14-8" class="xref-bibr" href="#ref-14">Howard <em>et al.</em> (2014)</a>. In the prior report, we simulated <em>F</em><sub>2</sub> and backcross populations where the linkage groups had different lengths. This report is based on DH lines with linkage groups that consist of the same numbers of marker loci per linkage group and the same recombination among adjacent marker loci within linkage groups. Consequently, the LD among the simulated QTL was not the same in the two studies. The inconsistencies between the two reports are small, but the results suggest that LD among QTL, population structures, and genomic architecture also will influence prediction accuracies. In particular, it is likely that because of LD among QTL in our first set of simulations we unintentionally produced a second source of nonlinear interactions, <em>i.e.</em>, pseudooverdominance.</p></div></div><div class="section" id="sec-13"></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-55">We thank the editor and the reviewers for their valuable comments, which helped us improve the manuscript. This work was supported in part by USDA CRIS IOW4314 and the Department of Agronomy at Iowa State University through the GF Sprague Endowment.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-5"><p id="p-56">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.044453/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.044453/-/DC1</a>.</p></li><li class="fn" id="fn-6"><p id="p-57"><em>Communicating editor: J. B. Holland</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2017-06-20"><span class="received-label">Received </span>June 20, 2017.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2017-07-11"><span class="accepted-label">Accepted </span>July 11, 2017.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2017 Howard <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3103.1" data-doi="10.2135/cropsci1994.0011183X003400010003x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bernardo</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Prediction of maize single-cross performance using RFLPs and information from related hybrids.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">20</span>–<span class="cit-lpage">25</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DBernardo%26rft.auinit1%253DR.%26rft.volume%253D34%26rft.issue%253D1%26rft.spage%253D20%26rft.epage%253D25%26rft.atitle%253DPrediction%2Bof%2BMaize%2BSingle-Cross%2BPerformance%2BUsing%2BRFLPs%2Band%2BInformation%2Bfrom%2BRelated%2BHybrids%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci1994.0011183X003400010003x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci1994.0011183X003400010003x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1994MV45600003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3103.2" data-doi="10.2135/cropsci2006.11.0690"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bernardo</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Prospects for genomewide selection for quantitative traits in maize.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">1082</span>–<span class="cit-lpage">1090</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DBernardo%26rft.auinit1%253DR.%26rft.volume%253D47%26rft.issue%253D3%26rft.spage%253D1082%26rft.epage%253D1090%26rft.atitle%253DProspects%2Bfor%2BGenomewide%2BSelection%2Bfor%2BQuantitative%2BTraits%2Bin%2BMaize%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2006.11.0690%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2006.11.0690&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000247424200020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3103.3" data-doi="10.1016/j.talanta.2008.05.019"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bezerra</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Santelli</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oliveira</span>  <span class="cit-name-given-names">E. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Villar</span>  <span class="cit-name-given-names">L. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Escaleira</span>  <span class="cit-name-given-names">L. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Response surface methodology (RSM) as a tool for optimization in analytical chemistry.</span> <abbr class="cit-jnl-abbrev">Talanta</abbr> <span class="cit-vol">76</span>: <span class="cit-fpage">965</span>–<span class="cit-lpage">977</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DTalanta%26rft.aulast%253DBezerra%26rft.auinit1%253DM.%2BA.%26rft.volume%253D76%26rft.issue%253D5%26rft.spage%253D965%26rft.epage%253D977%26rft.atitle%253DResponse%2Bsurface%2Bmethodology%2B%2528RSM%2529%2Bas%2Ba%2Btool%2Bfor%2Boptimization%2Bin%2Banalytical%2Bchemistry.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.talanta.2008.05.019%26rft_id%253Dinfo%253Apmid%252F18761143%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.talanta.2008.05.019&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18761143&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3103.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000259750800001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3103.4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Box</span>  <span class="cit-name-given-names">G. E. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cox</span>  <span class="cit-name-given-names">D. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1964</span> <span class="cit-article-title">An analysis of transformation (with discussion).</span> <abbr class="cit-jnl-abbrev">J. R. Stat. Soc. B.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">211</span>–<span class="cit-lpage">246</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BR.%2BStat.%2BSoc.%2BB.%26rft.volume%253D26%26rft.spage%253D211%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3103.5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Box</span>  <span class="cit-name-given-names">G. E. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilson</span>  <span class="cit-name-given-names">K. G.</span></span></li></ol><cite>, <span class="cit-pub-date">1951</span> <span class="cit-article-title">On the experimental attainment of optimum conditions</span>. <abbr class="cit-jnl-abbrev">J. R. Stat. Soc. Ser. A Stat. Soc.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">45</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BR.%2BStat.%2BSoc.%2BSer.%2BA%2BStat.%2BSoc.%26rft.volume%253D13%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3103.6" data-doi="10.1017/S0016672310000108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Clowers</span>  <span class="cit-name-given-names">K. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lyman</span>  <span class="cit-name-given-names">R. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mackay</span>  <span class="cit-name-given-names">T. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morgan</span>  <span class="cit-name-given-names">T. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genetic variation in senescence marker protein-30 is associated with natural variation in cold tolerance in Drosophila.</span> <abbr class="cit-jnl-abbrev">Genet Res (Camb).</abbr> <span class="cit-vol">92</span>: <span class="cit-fpage">103</span>–<span class="cit-lpage">113</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DClowers%26rft.auinit1%253DK.%2BJ.%26rft.volume%253D92%26rft.issue%253D2%26rft.spage%253D103%26rft.epage%253D113%26rft.atitle%253DGenetic%2Bvariation%2Bin%2Bsenescence%2Bmarker%2Bprotein-30%2Bis%2Bassociated%2Bwith%2Bnatural%2Bvariation%2Bin%2Bcold%2Btolerance%2Bin%2BDrosophila.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672310000108%26rft_id%253Dinfo%253Apmid%252F20515514%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672310000108&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20515514&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3103.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000279152300003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-book" id="cit-7.9.3103.7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cooper</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Podlich</span>  <span class="cit-name-given-names">D. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Micallef</span>  <span class="cit-name-given-names">K. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">O. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jensen</span>  <span class="cit-name-given-names">N. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Complexity, quantitative traits and plant breeding: a role for simulation modelling in the genetic improvement of crops</span>, pp. <span class="cit-fpage">143</span>–<span class="cit-lpage">166</span> in <span class="cit-source"><em>Quantitative Genetics</em>, <em>Genomics and Plant Breeding</em></span>, edited by <span class="cit-ed"><span class="cit-name-surname">Kang</span>  <span class="cit-name-given-names">M. S.</span></span>. <span class="cit-publ-name">CAB International</span>, <span class="cit-publ-loc">Wallingford, United Kingdom</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3103.8" data-doi="10.1017/S0016672310000285"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosa</span>  <span class="cit-name-given-names">G. J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weigel</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Semi-parametric genomic-enabled prediction of genetic values using reproducing kernel Hilbert spaces methods.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">92</span>: <span class="cit-fpage">295</span>–<span class="cit-lpage">308</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DDE%2BLos%2BCampos%26rft.auinit1%253DG.%26rft.volume%253D92%26rft.issue%253D4%26rft.spage%253D295%26rft.epage%253D308%26rft.atitle%253DSemi-parametric%2Bgenomic-enabled%2Bprediction%2Bof%2Bgenetic%2Bvalues%2Busing%2Breproducing%2Bkernel%2BHilbert%2Bspaces%2Bmethods.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672310000285%26rft_id%253Dinfo%253Apmid%252F20943010%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672310000285&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20943010&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3103.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3103.9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Endelman</span>  <span class="cit-name-given-names">J. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Ridge regression and other kernels for genomic selection with R package rrBLUP.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">250</span>–<span class="cit-lpage">255</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D4%26rft.spage%253D250%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-book" id="cit-7.9.3103.10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hastie</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tibshirani</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Friedman</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-source"><em>The Elements of Statistical Learning: Data Mining</em>, <em>Inference</em>, <em>and Prediction</em></span>. <span class="cit-publ-name">Springer-Verlag</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-book" id="cit-7.9.3103.11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Henderson</span>  <span class="cit-name-given-names">C. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1963</span> <span class="cit-article-title">Selection index and expected genetic advance</span>, pp. <span class="cit-fpage">141</span>–<span class="cit-lpage">163</span> in <span class="cit-source">Statistical Genetics and Plant Breeding</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Hanson</span>  <span class="cit-name-given-names">W. D.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Robinson</span>  <span class="cit-name-given-names">H. F.</span></span>. <span class="cit-publ-name">NAS-NRC Publication</span>, <span class="cit-publ-loc">Washington DC</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3103.12" data-doi="10.2307/2527669"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Henderson</span>  <span class="cit-name-given-names">C. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kempthorne</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Searle</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">von Krosigk</span>  <span class="cit-name-given-names">C. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1959</span> <span class="cit-article-title">The estimation of environmental and genetic trends from records subject to culling.</span> <abbr class="cit-jnl-abbrev">Biometrics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">192</span>–<span class="cit-lpage">218</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiometrics%26rft.volume%253D15%26rft.spage%253D192%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2527669%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2527669&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1959WX52200002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3103.13" data-doi="10.2135/cropsci2014.03.0249"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heslot</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Perspectives for genomic selection applications and research in plants.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">55</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">12</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D55%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2014.03.0249%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2014.03.0249&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3103.14"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Howard</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carriquiry</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beavis</span>  <span class="cit-name-given-names">W. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Parametric and nonparametric statistical methods for genomic selection of traits with additive and epistatic genetic architectures.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">1027</span>–<span class="cit-lpage">1046</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft.volume%253D4%26rft.spage%253D1027%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3103.15" data-doi="10.1073/pnas.1213423109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Richards</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carbone</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anholt</span>  <span class="cit-name-given-names">R. R. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Epistasis dominates the genetic architecture of Drosophila quantitative traits.</span> <abbr class="cit-jnl-abbrev">Proc. Nat. Acad. Sci.</abbr> <span class="cit-vol">109</span>: <span class="cit-fpage">15553</span>–<span class="cit-lpage">15559</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNat.%2BAcad.%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1213423109%26rft_id%253Dinfo%253Apmid%252F22949659%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA5LzM5LzE1NTUzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy85LzMxMDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3103.16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kao</span>  <span class="cit-name-given-names">C. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zeng</span>  <span class="cit-name-given-names">Z. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Modeling epistasis of quantitative trait loci using Cockerham's model.</span> <abbr class="cit-jnl-abbrev">Genetics.</abbr> <span class="cit-vol">160</span>: <span class="cit-fpage">1243</span>–<span class="cit-lpage">1261</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DKao%26rft.auinit1%253DC.-H.%26rft.volume%253D160%26rft.issue%253D3%26rft.spage%253D1243%26rft.epage%253D1261%26rft.atitle%253DModeling%2BEpistasis%2Bof%2BQuantitative%2BTrait%2BLoci%2BUsing%2BCockerham%2527s%2BModel%26rft_id%253Dinfo%253Apmid%252F11901137%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2MC8zLzEyNDMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzkvMzEwMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3103.17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Karatzoglou</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smola</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hornik</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zeileis</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Kernlab an S4 package for kernel methods in R.</span> <abbr class="cit-jnl-abbrev">J. Stat. Softw.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">9</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BStat.%2BSoftw.%26rft.volume%253D11%26rft.spage%253D9%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3103.18"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Khuri</span>  <span class="cit-name-given-names">A. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mukhopadhyay</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Response Surface Methodology.</span> <abbr class="cit-jnl-abbrev">WIREs Comp Stat.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">128</span>–<span class="cit-lpage">149</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DWIREs%2BComp%2BStat.%26rft.volume%253D2%26rft.spage%253D128%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-confproc" id="cit-7.9.3103.19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Koyamada</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Saki</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Itoch</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> Parameter optimization technique using the response surface methodlogy. Engineering in Medicine and Biology Society 26th Annual IEEE Conference, <span class="cit-vol">Vol. 2</span>, pp. <span class="cit-fpage">2909</span>–<span class="cit-lpage">2912</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3103.20"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lande</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">Efficiency of marker assisted selection in the improvement of quantitative traits.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">124</span>: <span class="cit-fpage">743</span>–<span class="cit-lpage">756</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DLande%26rft.auinit1%253DR.%26rft.volume%253D124%26rft.issue%253D3%26rft.spage%253D743%26rft.epage%253D756%26rft.atitle%253DEfficiency%2Bof%2Bmarker-assisted%2Bselection%2Bin%2Bthe%2Bimprovement%2Bof%2Bquantitative%2Btraits.%26rft_id%253Dinfo%253Apmid%252F1968875%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTI0LzMvNzQzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNy85LzMxMDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3103.21"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Menke</span>  <span class="cit-name-given-names">W. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1973</span> <span class="cit-article-title">Response surface methodology applications to biological data from field measurements.</span> <abbr class="cit-jnl-abbrev">Ecology</abbr> <span class="cit-vol">54</span>: <span class="cit-fpage">920</span>–<span class="cit-lpage">923</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcology%26rft.volume%253D54%26rft.spage%253D920%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3103.22"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>  <span class="cit-name-given-names">T. H. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Prediction of total genetic value using genome-wide dense marker maps.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">157</span>: <span class="cit-fpage">1819</span>–<span class="cit-lpage">1829</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMeuwissen%26rft.auinit1%253DT.%2BH.%2BE.%26rft.volume%253D157%26rft.issue%253D4%26rft.spage%253D1819%26rft.epage%253D1829%26rft.atitle%253DPrediction%2Bof%2BTotal%2BGenetic%2BValue%2BUsing%2BGenome-Wide%2BDense%2BMarker%2BMaps%26rft_id%253Dinfo%253Apmid%252F11290733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1Ny80LzE4MTkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy83LzkvMzEwMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-book" id="cit-7.9.3103.23"><div class="cit-metadata unstructured">Myers, R. H., and D. C. Montgomery, 1995 <em>Response Surface Methodology: Process and Product Optimization Using Designed Experiments</em> (Wiley Series in Probability and Statistics). Wiley, New York.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3103.24" data-doi="10.2307/1268813"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Myers</span>  <span class="cit-name-given-names">R. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Khuri</span>  <span class="cit-name-given-names">A. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carter</span>  <span class="cit-name-given-names">W. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">Response surface methodology: 1966–1988.</span> <abbr class="cit-jnl-abbrev">Technometrics</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">137</span>–<span class="cit-lpage">157</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTechnometrics%26rft.volume%253D31%26rft.spage%253D137%26rft_id%253Dinfo%253Adoi%252F10.2307%252F1268813%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/1268813&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-book" id="cit-7.9.3103.25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Naylor</span>  <span class="cit-name-given-names">T. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1969</span> <em>The Design of Computer Simulation Experiments</em>, pp. <span class="cit-fpage">80</span>–<span class="cit-lpage">95</span>. <span class="cit-publ-name">Duke University Press</span>, <span class="cit-publ-loc">Durham, NC</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-web" id="cit-7.9.3103.26"><div class="cit-metadata unstructured">R Development Core Team, 2008 <em>R: A Language and Environment for Statistical Computing</em>. R Foundation for Statistical Computing, Vienna, Austria. Available at: <a href="http://www.R-project.org/">http://www.R-project.org/</a>.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-book" id="cit-7.9.3103.27"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vapnik</span>  <span class="cit-name-given-names">V.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <em>The Nature of Statistical Learning Theory</em>, <span class="cit-edition">Ed. 2</span>. <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3103.28"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhong</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Response surface methodology used for statistical optimization of jiean-peptide production by Bacillus subtillis.</span> <abbr class="cit-jnl-abbrev">Electron. J. Biotechnol.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">12</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DElectron.%2BJ.%2BBiotechnol.%26rft.volume%253D13%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-7.9.3103.29" data-doi="10.1007/s00122-011-1745-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gowda</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wurschum</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maurer</span>  <span class="cit-name-given-names">H. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Accuracy of genomic selection in European maize elite breeding populations.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">124</span>: <span class="cit-fpage">769</span>–<span class="cit-lpage">776</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DZhao%26rft.auinit1%253DY.%26rft.volume%253D124%26rft.issue%253D4%26rft.spage%253D769%26rft.epage%253D776%26rft.atitle%253DAccuracy%2Bof%2Bgenomic%2Bselection%2Bin%2BEuropean%2Bmaize%2Belite%2Bbreeding%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-011-1745-y%26rft_id%253Dinfo%253Apmid%252F22075809%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-011-1745-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22075809&amp;link_type=MED&amp;atom=%2Fggg%2F7%2F9%2F3103.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/7/9/3103.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/9/3091" title="3′-Terminated Overhangs Regulate DNA Double-Strand Break Processing in Escherichia coli" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/7/9/3115" title="A Large Deletion in the NSDHL Gene in Labrador Retrievers with a Congenital Cornification Disorder" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/7/9"><span>Volume 7 Issue 9, September 2017</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/7/9.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/7/9.cover-source.jpg?itok=WuYpGT8r" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 7 (9)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/7/9/3103.abstract" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-RW3ChVRUTpz8BHBBqITfbHYM2St7yLO3Kamr_aXgzu0" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/7/9/3103" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/7/9/3103" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/7/9/3103" class="active" data-icon-position="" data-hide-link-title="0">Application of Response Surface Methods To Determine Conditions for Optimal Genomic Prediction</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/486351" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-eQ7VmjOy_oYXhV60eGBQRqJPFA54cKCXHVMVwWjr_KM" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/9/3103.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/7/9/3103" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-1R4RMun8DKw_yLFMXLoP1LS7Fz9Z4Ew7qz0QxT6Rn0c" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/7/9/3103" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/7/9/3103.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="486351" id="node486351--2" data-pisa="ggg;7/9/3103" data-pisa-master="ggg;g3.117.044453" data-apath="/ggg/7/9/3103.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Application of Response Surface Methods To Determine Conditions for Optimal Genomic Prediction</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Réka Howard</span>, <span class="highwire-citation-author" data-delta="1">Alicia L. Carriquiry</span> and <span class="highwire-citation-author" data-delta="2">William D. Beavis</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3103-3113;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.044453 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node486351--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Réka Howard </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Statistics, University of Nebraska, Lincoln, Nebraska 68583</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=R%C3%A9ka%2BHoward%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Howard%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AR%25C3%25A9ka%2BHoward%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:rekahoward@unl.edu" class="" data-icon-position="" data-hide-link-title="0">rekahoward@unl.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Alicia L. Carriquiry </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Statistics, Iowa State University, Ames, Iowa 50011</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alicia%2BL.%2BCarriquiry%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Carriquiry%20AL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlicia%2BL.%2BCarriquiry%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">William D. Beavis </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Agronomy, Iowa State University, Ames, Iowa 50011</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=William%2BD.%2BBeavis%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Beavis%20WD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AWilliam%2BD.%2BBeavis%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/7/9/3103" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/7/9/3103&amp;title=Application%20of%20Response%20Surface%20Methods%20To%20Determine%20Conditions%20for%20Optimal%20Genomic%20Prediction" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/7/9/3103&amp;title=Application%20of%20Response%20Surface%20Methods%20To%20Determine%20Conditions%20for%20Optimal%20Genomic%20Prediction" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/7/9/3103&amp;title=Application%20of%20Response%20Surface%20Methods%20To%20Determine%20Conditions%20for%20Optimal%20Genomic%20Prediction" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/7/9/3103&amp;text=Application%20of%20Response%20Surface%20Methods%20To%20Determine%20Conditions%20for%20Optimal%20Genomic%20Prediction" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/7/9/3103&amp;title=Application%20of%20Response%20Surface%20Methods%20To%20Determine%20Conditions%20for%20Optimal%20Genomic%20Prediction" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/7/9/3103&amp;t=Application%20of%20Response%20Surface%20Methods%20To%20Determine%20Conditions%20for%20Optimal%20Genomic%20Prediction" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/7/9/3103&amp;title=Application%20of%20Response%20Surface%20Methods%20To%20Determine%20Conditions%20for%20Optimal%20Genomic%20Prediction" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/7/9/3103&amp;title=Application%20of%20Response%20Surface%20Methods%20To%20Determine%20Conditions%20for%20Optimal%20Genomic%20Prediction" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/486351/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="486351" id="node486351--3" data-pisa="ggg;7/9/3103" data-pisa-master="ggg;g3.117.044453" data-apath="/ggg/7/9/3103.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Application of Response Surface Methods To Determine Conditions for Optimal Genomic Prediction</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Réka Howard</span>, <span class="highwire-citation-author" data-delta="1">Alicia L. Carriquiry</span> and <span class="highwire-citation-author" data-delta="2">William D. Beavis</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2017 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 7 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3103-3113;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.044453 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node486351--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Réka Howard </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Statistics, University of Nebraska, Lincoln, Nebraska 68583</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=R%C3%A9ka%2BHoward%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Howard%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AR%25C3%25A9ka%2BHoward%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:rekahoward@unl.edu" class="" data-icon-position="" data-hide-link-title="0">rekahoward@unl.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Alicia L. Carriquiry </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Statistics, Iowa State University, Ames, Iowa 50011</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alicia%2BL.%2BCarriquiry%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Carriquiry%20AL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlicia%2BL.%2BCarriquiry%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">William D. Beavis </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Agronomy, Iowa State University, Ames, Iowa 50011</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=William%2BD.%2BBeavis%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Beavis%20WD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AWilliam%2BD.%2BBeavis%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=3103&amp;rft.epage=3113&amp;rft.atitle=Application%20of%20Response%20Surface%20Methods%20To%20Determine%20Conditions%20for%20Optimal%20Genomic%20Prediction&amp;rft.volume=7&amp;rft.issue=9&amp;rft.date=2017-09-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Howard%2C+R%C3%A9ka&amp;rft.au=Carriquiry%2C+Alicia+L.&amp;rft.au=Beavis%2C+William+D."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/486351/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/486351/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/486351/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/486351/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/486351/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/486351/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/486351/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/486351/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/486351/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/486351/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/486351/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/486351/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/7/9/3103" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Background</a></li><li class="odd"><a href="#sec-4" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Methods</a></li><li class="even"><a href="#sec-7" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="odd"><a href="#sec-10" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/7/9/3103.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/7/9/3103.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/7/9/3103.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/7/9/3103.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
