<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/2/9/1085" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/2/9/1085.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/2/9/1085.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/2/9/1085.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/2/9.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/2/9/1085.atom" />
<meta name="HW.pisa" content="ggg;2/9/1085" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Predicting Adaptive Phenotypes From Multilocus Genotypes in Sitka Spruce (Picea sitchensis) Using Random Forest" />
<meta name="DC.Identifier" content="10.1534/g3.112.002733" />
<meta name="DC.Date" content="2012-09-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2012 Holliday et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Climate is the primary driver of the distribution of tree species worldwide, and the potential for adaptive evolution will be an important factor determining the response of forests to anthropogenic climate change. Although association mapping has the potential to improve our understanding of the genomic underpinnings of climatically relevant traits, the utility of adaptive polymorphisms uncovered by such studies would be greatly enhanced by the development of integrated models that account for the phenotypic effects of multiple single-nucleotide polymorphisms (SNPs) and their interactions simultaneously. We previously reported the results of association mapping in the widespread conifer Sitka spruce ( Picea sitchensis ). In the current study we used the recursive partitioning algorithm ‘Random Forest’ to identify optimized combinations of SNPs to predict adaptive phenotypes. After adjusting for population structure, we were able to explain 37% and 30% of the phenotypic variation, respectively, in two locally adaptive traits—autumn budset timing and cold hardiness. For each trait, the leading five SNPs captured much of the phenotypic variation. To determine the role of epistasis in shaping these phenotypes, we also used a novel approach to quantify the strength and direction of pairwise interactions between SNPs and found such interactions to be common. Our results demonstrate the power of Random Forest to identify subsets of markers that are most important to climatic adaptation, and suggest that interactions among these loci may be widespread." />
<meta name="DC.Contributor" content="Jason A. Holliday" />
<meta name="DC.Contributor" content="Tongli Wang" />
<meta name="DC.Contributor" content="Sally Aitken" />
<meta name="article:published_time" content="2012-09-01" />
<meta name="article:section" content="Genomic Selection" />
<meta name="citation_title" content="Predicting Adaptive Phenotypes From Multilocus Genotypes in Sitka Spruce (Picea sitchensis) Using Random Forest" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Climate is the primary driver of the distribution of tree species worldwide, and the potential for adaptive evolution will be an important factor determining the response of forests to anthropogenic climate change. Although association mapping has the potential to improve our understanding of the genomic underpinnings of climatically relevant traits, the utility of adaptive polymorphisms uncovered by such studies would be greatly enhanced by the development of integrated models that account for the phenotypic effects of multiple single-nucleotide polymorphisms (SNPs) and their interactions simultaneously. We previously reported the results of association mapping in the widespread conifer Sitka spruce (&lt;i&gt;Picea sitchensis&lt;/i&gt;). In the current study we used the recursive partitioning algorithm ‘Random Forest’ to identify optimized combinations of SNPs to predict adaptive phenotypes. After adjusting for population structure, we were able to explain 37% and 30% of the phenotypic variation, respectively, in two locally adaptive traits—autumn budset timing and cold hardiness. For each trait, the leading five SNPs captured much of the phenotypic variation. To determine the role of epistasis in shaping these phenotypes, we also used a novel approach to quantify the strength and direction of pairwise interactions between SNPs and found such interactions to be common. Our results demonstrate the power of Random Forest to identify subsets of markers that are most important to climatic adaptation, and suggest that interactions among these loci may be widespread.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2012/09/01" />
<meta name="citation_mjid" content="ggg;2/9/1085" />
<meta name="citation_id" content="2/9/1085" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/2/9/1085" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/2/9/1085.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/2/9/1085.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/2/9/1085.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.112.002733" />
<meta name="citation_pmid" content="22973546" />
<meta name="citation_volume" content="2" />
<meta name="citation_issue" content="9" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Genomic Selection" />
<meta name="citation_firstpage" content="1085" />
<meta name="citation_lastpage" content="1093" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Jason A. Holliday" />
<meta name="citation_author_institution" content="Department of Forest Resources and Environmental Conservation, Virginia Polytechnic Institute and State University, Blacksburg, Virginia 24061, and" />
<meta name="citation_author_email" content="jah1@vt.edu" />
<meta name="citation_author" content="Tongli Wang" />
<meta name="citation_author_institution" content="Department of Forest Sciences, University of British Columbia, Vancouver, BC, Canada, V6T 1Z4" />
<meta name="citation_author" content="Sally Aitken" />
<meta name="citation_author_institution" content="Department of Forest Sciences, University of British Columbia, Vancouver, BC, Canada, V6T 1Z4" />
<meta name="citation_reference" content="citation_journal_title=Silvae Genet.;citation_author=M. R. Ahuja;citation_author=D. B. Neale;citation_title=Evolution of genome size in conifers;citation_pages=126-137;citation_volume=54;citation_year=2005" />
<meta name="citation_reference" content="citation_journal_title=Evolutionary Applications;citation_author=S. N. Aitken;citation_author=S. Yeaman;citation_author=J. A. Holliday;citation_author=T. Wang;citation_author=S. Curtis-McLane;citation_title=Adaptation, migration or extirpation: climate change outcomes for tree populations;citation_pages=95-111;citation_volume=1;citation_year=2008;citation_pmid=25567494;citation_doi=10.1111/j.1752-4571.2007.00013.x" />
<meta name="citation_reference" content="citation_journal_title=Trends in Ecology &amp; Evolution;citation_journal_abbrev=Trends in Ecology &amp; Evolution;citation_author=R. D. H. Barrett;citation_author=D. Schluter;citation_title=Adaptation from standing genetic variation.;citation_pages=38-44;citation_volume=23;citation_year=2008;citation_issue=1;citation_pmid=18006185;citation_doi=10.1016/j.tree.2007.09.008" />
<meta name="citation_reference" content="citation_journal_title=Mach. Learn.;citation_author=L. Breiman;citation_title=Random forests;citation_pages=5-32;citation_volume=45;citation_year=2001;citation_doi=10.1023/A:1010933404324" />
<meta name="citation_reference" content="citation_journal_title=Genetic epidemiology;citation_journal_abbrev=Genet Epidemiol;citation_author=A. Bureau;citation_author=J. Dupuis;citation_author=K. Falls;citation_author=K. L. Lunetta;citation_author=B. Hayward;citation_title=Identifying SNPs predictive of phenotype using random forests.;citation_pages=171-182;citation_volume=28;citation_year=2005;citation_issue=2;citation_pmid=15593090;citation_doi=10.1002/gepi.20041" />
<meta name="citation_reference" content="citation_journal_title=Trans. Computat. Biol. Bioinformatics;citation_author=C. C. M. Chen;citation_author=H. Schwender;citation_author=J. Keith;citation_author=R. Nunkesser;citation_author=K. Mengersen;citation_title=Methods for identifying SNP interactions: a review on variations of logic regression, Random Forest and Bayesian logistic regression;citation_pages=1580-1591;citation_volume=8;citation_year=2011;citation_doi=10.1109/TCBB.2011.46" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=H. J. Cordell;citation_title=Detecting gene-gene interactions that underlie human diseases.;citation_pages=392-404;citation_volume=10;citation_year=2009;citation_issue=6;citation_pmid=19434077;citation_doi=10.1038/nrg2579" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. da Silva;citation_author=M. Coetzer;citation_author=R. Nedellec;citation_author=C. Pastore;citation_author=D. E. Mosier;citation_title=Fitness epistasis and constraints on adaptation in a human immunodeficiency virus type 1 protein region;citation_pages=293-303;citation_volume=185;citation_year=2010;citation_pmid=20157005;citation_doi=10.1534/genetics.109.112458" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. J. Eckert;citation_author=A. D. Bower;citation_author=J. L. Wegrzyn;citation_author=B. Pande;citation_author=K. D. Jermstad;citation_title=Asssociation genetics of coastal Douglas-fir (Pseudotsuga menziesii var. menziesii, Pinaceae). I. Cold hardiness-related traits;citation_pages=1289-1302;citation_volume=182;citation_year=2009;citation_pmid=19487566;citation_doi=10.1534/genetics.109.102350" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. J. Eckert;citation_author=J. L. Wegrzyn;citation_author=B. Pande;citation_author=K. D. Jermstad;citation_author=J. M. Lee;citation_title=Multilocus patterns of nucleotide diversity and divergence reveal positive selection at candidate genes related to cold hardiness in coastal Douglas-fir (Pseudotsuga menziesii var. menziesii);citation_volume=183;citation_year=2009;citation_pmid=19596906;citation_doi=10.1534/genetics.109.103895" />
<meta name="citation_reference" content="citation_journal_title=New Phytologist;citation_journal_abbrev=New Phytologist;citation_author=A. J. Eckert;citation_author=J. L. Wegrzyn;citation_author=W. P. Cumbie;citation_author=B. Goldfarb;citation_author=D. A. Huber;citation_title=Association genetics of the loblolly pine (Pinus taeda, Pinaceae) metabolome.;citation_pages=890-902;citation_volume=193;citation_year=2012;citation_issue=4;citation_pmid=22129444;citation_doi=10.1111/j.1469-8137.2011.03976.x" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=M. W. Feldman;citation_author=F. B. Christiansen;citation_author=L. D. Brooks;citation_title=Evolution of recombination in a constant environment;citation_pages=4838-4841;citation_volume=77;citation_year=1980;citation_issue=8;citation_pmid=16592864;citation_doi=10.1073/pnas.77.8.4838" />
<meta name="citation_reference" content="citation_journal_title=BMC genetics [electronic resource];citation_journal_abbrev=BMC Genet;citation_author=B. A. Goldstein;citation_author=A. E. Hubbard;citation_author=A. Cutler;citation_author=L. F. Barcellos;citation_title=An application of Random Forests to a genome-wide association dataset: methodological considerations &amp;amp; new findings.;citation_pages=49-49;citation_volume=11;citation_year=2010;citation_pmid=20546594" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. C. Gonzalez-Martinez;citation_author=E. Ersoz;citation_author=G. R. Brown;citation_author=N. C. Wheeler;citation_author=D. B. Neale;citation_title=DNA sequence variation and selection of tag single-nucleotide polymorphisms at candidate genes for drought-stress response in Pinus taeda L;citation_pages=1915-1926;citation_volume=172;citation_year=2006;citation_pmid=16387885;citation_doi=10.1534/genetics.105.047126" />
<meta name="citation_reference" content="citation_journal_title=New Phytologist;citation_journal_abbrev=New Phytologist;citation_author=S. C. Gonzalez-Martinez;citation_author=K. V. Krutovsky;citation_author=D. B. Neale;citation_title=Forest-tree population genomics and adaptive evolution.;citation_pages=227-238;citation_volume=170;citation_year=2006;citation_issue=2;citation_pmid=16608450;citation_doi=10.1111/j.1469-8137.2006.01686.x" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. C. Gonzalez-Martinez;citation_author=N. C. Wheeler;citation_author=E. Ersoz;citation_author=C. D. Nelson;citation_author=D. B. Neale;citation_title=Association genetics in Pinus taeda L. I. Wood property traits;citation_pages=399-409;citation_volume=175;citation_year=2007;citation_pmid=17110498" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_journal_abbrev=Heredity;citation_author=S. C. Gonzalez-Martinez;citation_author=D. Huber;citation_author=E. Ersoz;citation_author=J. M. Davis;citation_author=D. B. Neale;citation_title=Association genetics in Pinus taeda L. II. Carbon isotope discrimination.;citation_pages=19-26;citation_volume=101;citation_year=2008;citation_issue=1;citation_pmid=18478029;citation_doi=10.1038/hdy.2008.21" />
<meta name="citation_reference" content="citation_journal_title=Tree Genet. Genomes;citation_author=D. Grattapaglia;citation_author=M. D. V. Resende;citation_title=Genomic selection in forest tree breeding;citation_pages=241-255;citation_volume=7;citation_year=2011;citation_doi=10.1007/s11295-010-0328-4" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_journal_abbrev=Bioinformatics;citation_author=L. W. Hahn;citation_author=M. D. Ritchie;citation_author=J. H. Moore;citation_title=Multifactor dimensionality reduction software for detecting gene-gene and gene-environment interactions;citation_pages=376-382;citation_volume=19;citation_year=2003;citation_issue=3;citation_pmid=12584123;citation_doi=10.1093/bioinformatics/btf869" />
<meta name="citation_reference" content="citation_journal_title=Ecology;citation_journal_abbrev=Ecology;citation_author=A. Hamann;citation_author=T. L. Wang;citation_title=Potential effects of climate change on ecosystem and tree species distribution in British Columbia.;citation_pages=2773-2786;citation_volume=87;citation_year=2006;citation_issue=11;citation_pmid=17168022;citation_doi=10.1890/0012-9658(2006)87[2773:PEOCCO]2.0.CO;2" />
<meta name="citation_reference" content="citation_journal_title=Can. J. Forest Res.;citation_author=M. Hannerz;citation_author=S. N. Aitken;citation_author=J. N. King;citation_author=S. Budge;citation_title=Effects of genetic selection for growth on frost hardiness in western hemlock;citation_pages=509-516;citation_volume=29;citation_year=1999;citation_doi=10.1139/x99-019" />
<meta name="citation_reference" content="citation_journal_title=New Phytologist;citation_journal_abbrev=New Phytologist;citation_author=J. A. Holliday;citation_author=K. Ritland;citation_author=S. N. Aitken;citation_title=Widespread, ecologically relevant genetic markers developed from association mapping of climate-related traits in Sitka spruce (Picea sitchensis).;citation_pages=501-514;citation_volume=188;citation_year=2010;citation_issue=2;citation_pmid=20663060;citation_doi=10.1111/j.1469-8137.2010.03380.x" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=P. K. Ingvarsson;citation_author=M. V. Garcia;citation_author=V. Luquez;citation_author=D. Hall;citation_author=S. Jansson;citation_title=Nucleotide polymoirphism and phenotypic associations within and around the phytochrome B2 locus in European aspen (Populus tremula, Salicaceae);citation_pages=2217-2226;citation_volume=178;citation_year=2008;citation_pmid=18245834;citation_doi=10.1534/genetics.107.082354" />
<meta name="citation_reference" content="citation_journal_title=Brief.Funct. Genomics;citation_author=J. L. Jannink;citation_author=A. J. Lorenz;citation_author=H. Iwata;citation_title=Genomic selection in plant breeding: from theory to practice;citation_pages=166-177;citation_volume=9;citation_year=2010" />
<meta name="citation_reference" content="citation_journal_title=BMC Bioinformatics;citation_author=R. Jiang;citation_author=W. W. Tang;citation_author=X. B. Wu;citation_author=W. H. Fu;citation_title=A random forest approach to the detection of epistatic interactions in case-control studies;citation_pages=S65;citation_volume=10;citation_year=2009;citation_pmid=19208169;citation_doi=10.1186/1471-2105-10-S1-S65" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=A. Kremer;citation_author=V. Le Corre;citation_title=Decoupling of differentiation between traits and their underlying genes in response to divergent selection;citation_pages=375-385;citation_volume=108;citation_year=2011;citation_pmid=21915150" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=K. V. Krutovsky;citation_author=D. B. Neale;citation_title=Nucleotide diversity and linkage disequilibrium in cold-hardiness- and wood quality-related candidate genes in Douglas fir;citation_volume=171;citation_year=2005;citation_pmid=16157674;citation_doi=10.1534/genetics.105.044420" />
<meta name="citation_reference" content="citation_journal_title=Am. Nat.;citation_author=R. G. Latta;citation_title=Differentiation of allelic frequencies at quantitative trait loci affecting locally adaptive traits;citation_pages=283-292;citation_volume=151;citation_year=1998;citation_pmid=18811359;citation_doi=10.1086/286119" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=V. Le Corre;citation_author=A. Kremer;citation_title=Genetic Variability at Neutral Markers, Quantitative Trait Loci and Trait in a Subdivided Population Under Selection;citation_pages=1205-1219;citation_volume=164;citation_year=2003;citation_issue=3;citation_pmid=12871925" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=V. Le Corre;citation_author=A. Kremer;citation_title=The genetic differentiation at quantitative trait loci under local adaptation.;citation_pages=1548-1566;citation_volume=21;citation_year=2012;citation_issue=7;citation_pmid=22332667;citation_doi=10.1111/j.1365-294X.2012.05479.x" />
<meta name="citation_reference" content="citation_journal_title=R News;citation_author=A. Liaw;citation_author=M. Wiener;citation_title=Classification and regression by randomForest;citation_pages=18-22;citation_volume=2;citation_year=2002" />
<meta name="citation_reference" content="citation_journal_title=BMC genetics [electronic resource];citation_journal_abbrev=BMC Genet;citation_author=K. L. Lunetta;citation_author=L. B. Hayward;citation_author=J. Segal;citation_author=P. Van Eerdewegh;citation_title=Screening large-scale association study data: exploiting interactions using random forests.;citation_pages=32-32;citation_volume=5;citation_year=2004;citation_pmid=15588316;citation_doi=10.1186/1471-2156-5-32" />
<meta name="citation_reference" content="citation_journal_title=Genetic epidemiology;citation_journal_abbrev=Genet Epidemiol;citation_author=D. J. Lunn;citation_author=J. C. Whittaker;citation_author=N. Best;citation_title=A Bayesian toolkit for genetic association studies.;citation_pages=231-247;citation_volume=30;citation_year=2006;citation_issue=3;citation_pmid=16544290;citation_doi=10.1002/gepi.20140" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=X.-F. Ma;citation_author=D. Hall;citation_author=K. R. St Onge;citation_author=S. Jansson;citation_author=P. K. Ingvarsson;citation_title=Genetic differentiation, clinal variation and phenotypic associations with growth cessation across the populus tremula photoperiodic Pathway;citation_volume=186;citation_year=2010;citation_pmid=20805554;citation_doi=10.1534/genetics.110.120873" />
<meta name="citation_reference" content="citation_journal_title=J. Biogeogr.;citation_author=J. R. Malcolm;citation_author=A. Markham;citation_author=R. P. Neilson;citation_author=M. Garaci;citation_title=Estimated migration rates under scenarios of global climate change;citation_pages=835-849;citation_volume=29;citation_year=2002;citation_doi=10.1046/j.1365-2699.2002.00702.x" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=R. L. Malmberg;citation_author=S. Held;citation_author=A. Waits;citation_author=R. Mauricio;citation_title=Epistasis for fitness-related quantitative traits in Arabidopsis thaliana grown in the field and in the greenhouse;citation_pages=2013-2027;citation_volume=171;citation_year=2005;citation_pmid=16157670;citation_doi=10.1534/genetics.105.046078" />
<meta name="citation_reference" content="citation_journal_title=Ecology;citation_author=J. S. McLachlan;citation_author=J. S. Clark;citation_author=P. S. Manos;citation_title=Molecular indicators of tree migration capacity under rapid climate change;citation_pages=2088-2098;citation_volume=86;citation_year=2005;citation_doi=10.1890/04-1036" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=T. H. E. Meuwissen;citation_author=B. J. Hayes;citation_author=M. E. Goddard;citation_title=Prediction of Total Genetic Value Using Genome-Wide Dense Marker Maps;citation_pages=1819-1829;citation_volume=157;citation_year=2001;citation_issue=4;citation_pmid=11290733" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_journal_abbrev=Heredity;citation_author=M. Mimura;citation_author=S. N. Aitken;citation_title=Adaptive gradients and isolation-by-distance with postglacial migration in Picea sitchensis.;citation_pages=224-232;citation_volume=99;citation_year=2007;citation_issue=2;citation_pmid=17487214;citation_doi=10.1038/sj.hdy.6800987" />
<meta name="citation_reference" content="citation_journal_title=Journal of Theoretical Biology;citation_journal_abbrev=Journal of Theoretical Biology;citation_author=J. H. Moore;citation_author=J. C. Gilbert;citation_author=C. T. Tsai;citation_author=F. T. Chiang;citation_author=T. Holden;citation_title=A flexible computational framework for detecting, characterizing, and interpreting statistical patterns of epistasis in genetic studies of human disease susceptibility.;citation_pages=252-261;citation_volume=241;citation_year=2006;citation_issue=2;citation_pmid=16457852;citation_doi=10.1016/j.jtbi.2005.11.036" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=S. Myles;citation_author=J. Peiffer;citation_author=P. J. Brown;citation_author=E. S. Ersoz;citation_author=Z. W. Zhang;citation_title=Association mapping: critical considerations shift from genotyping to experimental design;citation_volume=21;citation_year=2009;citation_pmid=19654263;citation_doi=10.1105/tpc.109.068437" />
<meta name="citation_reference" content="citation_journal_title=Trends in plant science;citation_journal_abbrev=Trends Plant Sci;citation_author=D. B. Neale;citation_author=O. Savolainen;citation_title=Association genetics of complex traits in conifers.;citation_pages=325-330;citation_volume=9;citation_year=2004;citation_issue=7;citation_pmid=15231277;citation_doi=10.1016/j.tplants.2004.05.006" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=A. Nemri;citation_author=S. Atwell;citation_author=A. M. Tarone;citation_author=Y. S. Huang;citation_author=K. Zhao;citation_title=Genome-wide survey of Arabidopsis natural variation in downy mildew resistance using combined association and linkage mapping;citation_volume=107;citation_year=2010;citation_pmid=20479233;citation_doi=10.1073/pnas.0913160107" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_author=H. A. Orr;citation_title=The population genetics of adaptation: the distribution of factors fixed during adaptive evolution;citation_pages=935-949;citation_volume=52;citation_year=1998;citation_doi=10.2307/2411226" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=H. A. Orr;citation_title=The evolutionary genetics of adaptation: a simulation study.;citation_pages=207-214;citation_volume=74;citation_year=1999;citation_issue=3;citation_pmid=10689798;citation_doi=10.1017/S0016672399004164" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=E. A. Ortlund;citation_author=J. T. Bridgham;citation_author=M. R. Redinbo;citation_author=J. W. Thornton;citation_title=Crystal structure of an ancient protein: evolution by conformational epistasis;citation_volume=317;citation_year=2007;citation_pmid=17702911;citation_doi=10.1126/science.1142819" />
<meta name="citation_reference" content="citation_journal_title=Theoretical population biology;citation_journal_abbrev=Theor Popul Biol;citation_author=S. P. Otto;citation_author=M. W. Feldman;citation_title=Deleterious mutations, variable epistatic interactions, and the evolution of recombination.;citation_pages=134-147;citation_volume=51;citation_year=1997;citation_issue=2;citation_pmid=9169238;citation_doi=10.1006/tpbi.1997.1301" />
<meta name="citation_reference" content="citation_journal_title=J. Exp. Zoolog. B Mol. Dev. Evol.;citation_author=M. Pavlicev;citation_author=E. A. Norgard;citation_author=G. L. Fawcett;citation_author=J. M. Cheverud;citation_title=EVOLUTION OF PLEIOTROPY: EPISTATIC INTERACTION PATTERN SUPPORTS A MECHANISTIC MODEL UNDERLYING VARIATION IN GENOTYPE-PHENOTYPE MAP;citation_pages=371-385;citation_volume=316B;citation_year=2011;citation_doi=10.1002/jez.b.21410" />
<meta name="citation_reference" content="citation_journal_title=Annu. Rev. Ecol. Evol. Syst.;citation_author=R. J. Petit;citation_author=A. Hampe;citation_title=Some evolutionary consequences of being a tree;citation_pages=187-214;citation_volume=37;citation_year=2006;citation_doi=10.1146/annurev.ecolsys.37.091305.110215" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=P. C. Phillips;citation_title=Epistasis--the essential role of gene interactions in the structure and evolution of genetic systems.;citation_pages=855-867;citation_volume=9;citation_year=2008;citation_issue=11;citation_pmid=18852697;citation_doi=10.1038/nrg2452" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. K. Pritchard;citation_author=M. Stephens;citation_author=P. Donnelly;citation_title=Inference of population structure using multilocus genotype data;citation_pages=945-959;citation_volume=155;citation_year=2000;citation_pmid=10835412" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=T. Quesada;citation_author=V. Gopal;citation_author=W. P. Cumbie;citation_author=A. J. Eckert;citation_author=J. L. Wegrzyn;citation_title=Association mapping of quantitative disease resistance in a natural population of loblolly pine (Pinus taeda L.);citation_pages=677-686;citation_volume=186;citation_year=2010;citation_pmid=20628037;citation_doi=10.1534/genetics.110.117549" />
<meta name="citation_reference" content="citation_journal_title=New Phytologist;citation_journal_abbrev=New Phytologist;citation_author=M. F. R. Resende;citation_author=P. Munoz;citation_author=J. J. Acosta;citation_author=G. F. Peter;citation_author=J. M. Davis;citation_title=Accelerating the domestication of trees using genomic selection: accuracy of prediction models across ages and environments.;citation_pages=617-624;citation_volume=193;citation_year=2012;citation_issue=3;citation_pmid=21973055;citation_doi=10.1111/j.1469-8137.2011.03895.x" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology Resources;citation_author=F. Rousset;citation_title=GENEPOP’007: a complete re-implementation of the GENEPOP software for Windows and Linux;citation_pages=103-106;citation_volume=8;citation_year=2008;citation_pmid=21585727;citation_doi=10.1111/j.1471-8286.2007.01931.x" />
<meta name="citation_reference" content="citation_journal_title=Annu. Rev. Ecol. Evol. Syst.;citation_author=O. Savolainen;citation_author=T. Pyhajarvi;citation_author=T. Knurr;citation_title=Gene flow and local adaptation in tees;citation_pages=595-619;citation_volume=38;citation_year=2007;citation_doi=10.1146/annurev.ecolsys.38.091206.095646" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=P. Shannon;citation_author=A. Markiel;citation_author=O. Ozier;citation_author=N. S. Baliga;citation_author=J. T. Wang;citation_title=Cytoscape: A Software Environment for Integrated Models of Biomolecular Interaction Networks;citation_pages=2498-2504;citation_volume=13;citation_year=2003;citation_issue=11;citation_pmid=14597658;citation_doi=10.1101/gr.1239303" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=J. D. Storey;citation_author=R. Tibshirani;citation_title=Statistical significance for genomewide studies;citation_volume=100;citation_year=2003;citation_pmid=12883005;citation_doi=10.1073/pnas.1530509100" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=M. von Korff;citation_author=J. Leon;citation_author=K. Pillen;citation_title=Detection of epistatic interactions between exotic alleles introgressed from wild barley (H. vulgare ssp. spontaneum).;citation_pages=1455-1464;citation_volume=121;citation_year=2010;citation_issue=8;citation_pmid=20617300;citation_doi=10.1007/s00122-010-1401-y" />
<meta name="citation_reference" content="citation_journal_title=Trends Ecol. Evol.;citation_author=M. J. Wade;citation_author=R. G. Winther;citation_author=A. F. Agrawal;citation_author=C. J. Goodnight;citation_title=Alternative definitions of epistasis: dependence and interaction;citation_pages=498-504;citation_volume=16;citation_year=2001;citation_doi=10.1016/S0169-5347(01)02213-3" />
<meta name="citation_reference" content="citation_journal_title=New Phytologist;citation_journal_abbrev=New Phytologist;citation_author=J. L. Wegrzyn;citation_author=A. J. Eckert;citation_author=M. Choi;citation_author=J. M. Lee;citation_author=B. J. Stanton;citation_title=Association genetics of traits controlling lignin and cellulose biosynthesis in black cottonwood (Populus trichocarpa, Salicaceae) secondary xylem.;citation_pages=515-532;citation_volume=188;citation_year=2010;citation_issue=2;citation_pmid=20831625;citation_doi=10.1111/j.1469-8137.2010.03415.x" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=J. M. Yu;citation_author=G. Pressoir;citation_author=W. H. Briggs;citation_author=I. V. Bi;citation_author=M. Yamasaki;citation_title=A unified mixed-model method for association mapping that accounts for multiple levels of relatedness.;citation_pages=203-208;citation_volume=38;citation_year=2006;citation_issue=2;citation_pmid=16380716;citation_doi=10.1038/ng1702" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=Y. Zhang;citation_author=J. S. Liu;citation_title=Bayesian inference of epistatic interactions in case-control studies.;citation_pages=1167-1173;citation_volume=39;citation_year=2007;citation_issue=9;citation_pmid=17721534;citation_doi=10.1038/ng2110" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Predicting Adaptive Phenotypes From Multilocus Genotypes in Sitka Spruce (Picea sitchensis) Using Random Forest" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/2/9.cover-source.jpg" />
<meta name="twitter:description" content="Climate is the primary driver of the distribution of tree species worldwide, and the potential for adaptive evolution will be an important factor determining the response of forests to anthropogenic climate change. Although association mapping has the potential to improve our understanding of the genomic underpinnings of climatically relevant traits, the utility of adaptive polymorphisms uncovered by such studies would be greatly enhanced by the development of integrated models that account for the phenotypic effects of multiple single-nucleotide polymorphisms (SNPs) and their interactions simultaneously. We previously reported the results of association mapping in the widespread conifer Sitka spruce ( Picea sitchensis ). In the current study we used the recursive partitioning algorithm ‘Random Forest’ to identify optimized combinations of SNPs to predict adaptive phenotypes. After adjusting for population structure, we were able to explain 37% and 30% of the phenotypic variation, respectively, in two locally adaptive traits—autumn budset timing and cold hardiness. For each trait, the leading five SNPs captured much of the phenotypic variation. To determine the role of epistasis in shaping these phenotypes, we also used a novel approach to quantify the strength and direction of pairwise interactions between SNPs and found such interactions to be common. Our results demonstrate the power of Random Forest to identify subsets of markers that are most important to climatic adaptation, and suggest that interactions among these loci may be widespread." />
<meta name="og-title" property="og:title" content="Predicting Adaptive Phenotypes From Multilocus Genotypes in Sitka Spruce (Picea sitchensis) Using Random Forest" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/2/9/1085" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Climate is the primary driver of the distribution of tree species worldwide, and the potential for adaptive evolution will be an important factor determining the response of forests to anthropogenic climate change. Although association mapping has the potential to improve our understanding of the genomic underpinnings of climatically relevant traits, the utility of adaptive polymorphisms uncovered by such studies would be greatly enhanced by the development of integrated models that account for the phenotypic effects of multiple single-nucleotide polymorphisms (SNPs) and their interactions simultaneously. We previously reported the results of association mapping in the widespread conifer Sitka spruce ( Picea sitchensis ). In the current study we used the recursive partitioning algorithm ‘Random Forest’ to identify optimized combinations of SNPs to predict adaptive phenotypes. After adjusting for population structure, we were able to explain 37% and 30% of the phenotypic variation, respectively, in two locally adaptive traits—autumn budset timing and cold hardiness. For each trait, the leading five SNPs captured much of the phenotypic variation. To determine the role of epistasis in shaping these phenotypes, we also used a novel approach to quantify the strength and direction of pairwise interactions between SNPs and found such interactions to be common. Our results demonstrate the power of Random Forest to identify subsets of markers that are most important to climatic adaptation, and suggest that interactions among these loci may be widespread." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/2/9.cover-source.jpg" />
    <title>Predicting Adaptive Phenotypes From Multilocus Genotypes in Sitka Spruce (Picea sitchensis) Using Random Forest | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"e7j0BeEN4VCtaRv1yPTKG6zcobGu0v_zZSb2R9uXH_o","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459328","apath":"\/ggg\/2\/9\/1085.atom","pisa":"ggg;2\/9\/1085","ac":{"\/ggg\/2\/9\/1085.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/2\/9\/1085.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;2\/9\/1085"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;2\/9\/1085"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/2\/9\/1085":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022hgA1ALPH6PcJW8N9JauLwGk4pdVT8roWQ\\\\\\\/wF4R1bYkJTgSbkeJmkXH805wuZrS75BxnvWWOndCZiOMNs08\\\\\\\/Y38KRd5JMe\\\\\\\/475XsuKotIU05im490u2M3mYN8Yqfrpso01vKuLRQYcnhwAnAMorCad98yMhOk1kMvNN2BNbGZlUGVSJyXzUKcJ\\\\\\\/qwTuG8EkRBfdzW5bApwwGlUNPkSDaHbWNeTlqkhn6k+h7SodiCZINUogeE\\\\\\\/UeYJ8100P3AnzsueIA6pLMVEwxPtHG88JoriQh1Z2UXOEnnwlipqZ3rRqDHwVPGU2H9tvm3KljVJfFGxYdsMn04BICQm9\\\\\\\/i9b38\\\\\\\/bU03rJzPm4sziuYrxVJmpQLnJc\\\\\\\/YYITk1iNwwrW0UU+7TFxUIbyVJpPKlatkeo1eq\\\\\\\/qwDGP3kRS4fEev36E7otrRhexUYazYIF1XfMITaGY84HM\\\\\\\/oqOjo5EXcJCg1yb5eAG4fVWB+9XH73YHx7O2qlZJng8EUYBsn1WXL0lqOZHv+2zAZbpLNZ3fpsLRwswJ2SpkMRUslC252Ew6+L9MjtcHFq5aCdvrOANxN50gWNkwbYiZ+loaVciVbICsGI48Fov3EMiGqwjc8IftOH+nRw5vB\\\\\\\/WHQOdmwkK+286tph6dIO9GQG8pyinwEw53BefQpL2MSJuT91vjwO0N0DQ40dyEg5k4RklzJhCNwT9cUTK1hSsl1FTEEtw8QuiHUnmEo2yQP4+dXhn3zzJp6TMOkW3a0g+eM4YHUBAKlls6QNUoiCzpDrKvJYkO7enQWB6PsFeLHJ5o61EvJuztCLjz7ZV0dVEEQHdgaX206xjvNc0bLXJRLseryiqcRG\\\\\\\/yO\\\\\\\/sAC7z5Ku8C8JyUS3cFSvUBqU7b8qdzBTsF4JQFMXdf0W3DmqiejOzRFe3nuB26DGjx3Z9RrSax\\\\\\\/xbyaNgJnJUuWXyMBY0m71QsFJF88fUsS19msjD5DC6CpZNZZo7b\\\\\\\/Vsrh3G6qkjgEHbM12zFqnQfD9d8i0\\\\\\\/pxsCt95\\\\\\\/wxPCL8\\\\\\\/7i+C0U0j1d\\\\\\\/PcDBwfb3u9XaFyixmgvdLO\\\\\\\/ON\\\\\\\/p+bBm1qfjyiHclOdQxZiAULBz6wJ9fAasyZG1ILuqJ7NHGNHypJ3dXfOiGO5UVbWyN9Ey3NPBGZtTPkGjJifBpVuavNmXMunDTsapkLAu\\\\\\\/7Nh\\\\\\\/wEGmva3M9AkNkIxpKHtDr85N14X1+Et5TO8mQ5990xeeR3K5S7H0WIHY04dUitnV1\\\\\\\/2s+mhCjKhzizCAHCGnItd7s86fKkKyDIs6vKWhrlNvTR\\\u0022,\\\u0022iv\\\u0022:\\\u0022KP6DyymSG3ytIGADqVYUVw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022fd0eaee10febb5a5c8ef5c6347409d304c8deab9f47407ae0291028ce1522797\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022A4ouVONLv3DiS+2RN2OPI\\\\\\\/7sL4XbZPIsDWUOvUYCuU2xemigPQaJ4CTbHJgaVvXbUMasb7atG160ARSMyqvxNXCR1AcmT\\\\\\\/06m6ei5AnX0XqBdsu+A8UiEDVnCJJZGXCb9vpx9QG+Aeg7POOHCisNoyaVuaZb2lwlqMNxGxMrMH8mcV3Y2VkvelkVFNMaSIpm58fJlz1YG1l2ffZ6uLG\\\\\\\/v91ntDsGKYuQL4SDcP9j+Pm+Yt6Imx7rTWlIx8+WEgJ9hWCBdcKhEvpX6q8HHh2R0czW9ISBFnwL168m6HulQNNWPP83tj\\\\\\\/d8NO1sSIKEKb9S1+SIK8A9bLXKwWxW1ZG7Axo6ZA5CKqgDH3yzR+769DMl3rcknCyLrvU\\\\\\\/RhcbXAuu2JMTN\\\\\\\/R9YLpSw\\\\\\\/X8sZLsN0bZUbGeNGQpJmjDUxhyp1vgjXKIoKObvWjZ594TtYHsCXlRTWpSZBV0h3AK+cUZDMboZwQ3yap5uq1JFD6+RGeeSESS5dA1wyTelzBpVxS6Ap9gpCEgOlFTikANKzEVls\\\\\\\/lpLTJxBcyEJ28v58CFp5lBD\\\\\\\/37CPqCr8pt9s75r+4br7W+ER\\\\\\\/83E6BohZOwjEcpMmkILeXYhDM+\\\\\\\/9fggurRiB5okowgja8vHPspcjYBtsImMYQSOBd5lcA2t8ljRAbHZ0stzUIEx+mbbGPPTZ6EONKPM8nCvadiwLtOULVaTikuhRmbUnTkjaEhTqUvdFsORaSnb5NJ+OQgp1xRzdaloC8mf4oZrMUrgaLfEp2ftU\\\\\\\/QMPqVLp4hFyCF33yLJB2dosCK8eiqQc37Hc\\\\\\\/7zx9MFd+aHttTiVtp7Ddf0CAzSEri08xj3aB22stt47UKAuVF7iY7Ttl6hv3QPNgeN0+n2VrlaBQXcxcIL5Y+\\\\\\\/k7baM1cLmCJ8\\\\\\\/r\\\\\\\/IsWX5iL8vxRyn6tJm0WHTT\\\\\\\/IobI6hz1AAef971F7VZNXZIBJQVt0dl\\\\\\\/Nt7F9lzntKRiCG+Vk+QtBYCSv\\\\\\\/RwYBehZB3227G+8lVUFkx69pRKijYnLRWUbfwN\\\\\\\/X430Dn9sFs1ykspFVrxURJOnYKmVy9at0MVdHfHnTg7bdEtn4zKV0DYeAxZtQD6MYZt7HoEtZB3EOCRMMYSeR0trdX\\\\\\\/kYjiT2sQ4XdhSF+lQRXPSZE25dQ8qoQM2jMtECfC\\\\\\\/PrKVA+s+6S6MIYv\\\\\\\/7mKI6unS6cw7mP7irmNon9tG48YWsX4ykupgaf\\\\\\\/\\\\\\\/5\\\\\\\/h7fJ1jzA5l8q0JG403PPvTdoeE4Eo8Jm\\\\\\\/ljFqLxNHQaEaqhbrJD+z1aHO\\\\\\\/cM3RpWBlXmD97jHg5qLM1mgcSL8C3b0jZh+c3Y\\\\\\\/\\\\\\\/LtLha2rYsmrcrUSyN\\\\\\\/BrgwGBhfmB2MHv+SZZY8K91\\\\\\\/OqPhUS1GDeTxImp6rEFlu8GNUbHgQMHRe8angcXkNVWgoz3bgLqHthqXI8catXf4MwmpRwcg+P4p23p8gHXsYzU6qlmnQ6wNICuU5IqRDnEUl7LjSIdFCS8pTJjigvXHa4qy8a5Efqd9z0hP9iAk7J49RqDonQQ5h8rqWZgbXA2PD1nGf6CJyAKA\\\\\\\/AZPV9jQdTLeai7Zt1WGQEgUdUyWywp9syT9GxQ5qNl1XiW8SuvbNWrwp2JvzEa82xACVY6Pv1tyAx7rA0lSo\\\\\\\/QhOdsJQNDERy\\\\\\\/j\\\\\\\/s0zs3MAsvhTKKIpwaZPvM2QmL1I6BghIwHmvMySshZEl3+2eEEigEQ9DRO6pKeSVSepv4ukjlSlQpeYrBTYPyqzYYKnLEcKaxLDJcLho5gb2JW5vUBSeUoUWkFUI2FBcJ6nx1z5Z8NAHMvc6+y9AneWb+ooh4x79tVhf7ew3WMmMsfRva8q0ZnkoE6LfnA0oJlUMNBQrvS7Um32lFCiH6wxQ4KgWfs38Wi80+FPDO1Fm+XQhfMg+H+\\\\\\\/GRn\\\\\\\/\\\\\\\/KIcamm48sgrswUACQ7UBrXmNwrVDS67NiPVmwePnH0\\\\\\\/IrkNh\\\\\\\/1jwNzg5hhtqqezkIMjBStxk3HHegyi5cFYmhwsUNx3kwAXWRYy4y6V74PH6cHLGTxN9zX\\\\\\\/VfGQ2zZvoLXhWcjsgInUQY+uO0uagm4HbVGHX3h+IwA81BPwUHQjQA2w13Alibiwfnklp0I9clLMBjswYhCu+kMy9R32zTRmGsgCxLgPWLTfyJur6ZkH9sbBNu3EVSXtJOJalYOarggwPgi1l2P6jMDwnyddGUCL2bYj26Y2TyJczJXizeD4HkfWThOTdq\\\\\\\/k6d\\\\\\\/eARHuvUxEjWl7miTn6E1sfkIgrMSMKoukOd9B1lBiAkvz+taFziqTHI7k26dmYKiq5HLRzZyfONP+lAjk3Z5UxF4QH4Pga6\\\\\\\/YLwRGZciP0FHZpVXwpyVFTgFL6j93LR6IqJpC4Ds+XEsqdFLdnM7Y+LRrQJM3fYip7BTZHdPlFGfbHoDbuGgACXw64b59PweLjqz2wPkxGKLRCoK6sXQncWP+6XBw4LPn\\\\\\\/owlqZgbRa\\\\\\\/Gv3KMYUOStWIvGfpnxrX0fKKQEUseRoDdpDPz8Ful8KnxCsMKrJQXGaO8mlka73iSIkWSnIalZPNjirnPMvwsI489XjDThADgPuTuHvjUbZ2\\\\\\\/pUIjY+hWWMpqFS+mQeB7Mi2FCsoqQ13qbMjpXX1Pq\\\\\\\/m9DbG6dh3WTmUpJDuA+jeDPkv+5u\\\\\\\/6RNRUFOmehDRkotNds2wshWr6DZOMhOjRgExQfSIlnVqYMcYzseb1RpU1aURC7D+WWY4YFY3j2ubmpbRUFqj2bZagOzrr5kgF4YRRYKkj6CcUibnQif4dfRn+1X+ImD1gqmnacxrtUTSpwLt4G\\\\\\\/ruuuGRVqHMKZThpNjJHqT\\\\\\\/xeIwcSSZI5GlRv9Xo1WzvkAbd5bHAxVfnRcu7mbj\\\\\\\/md5gtO7s+BAgr7bQJMP\\\\\\\/taoLw+MjjxJiDVT1DLiXV3r1C8\\\\\\\/yYmAiTWz8K8qO9DfJcYAJDUR4syYuB\\\\\\\/9qNBgszjK8CgIhtpKvyhdyws0qrLnUc7Uie8NXBPgxUo2U3NvJyXKNehvieALTfju\\\\\\\/N4tIlC9qvj\\\\\\\/ELu30bZqu25+21RowCKIQ84JbNhOEHsOmq+9xN4NG0V+huZQEFtPL0HHrcaowPZItACZnjmlR8VtUx3o7YGDRtIjVyjStDC11qj5N0teB678Vl6vyx2XThKrrtPVnrqpmg64OQJhvS2JuYsBZB\\\\\\\/\\\\\\\/HCJPQvE91bbl\\\\\\\/4lSLcIordLWYb7e\\\\\\\/Rj4UWBvoihnrqcjPGvg234m21e+loyHD0zIEic+jDEEe0SqJBVgKEKgg4WMCJFdxEyuS3t3wD2fPwslv+hbQsdkzikJvW1WfjAEXU40p5N2DjSpGg52XT6HKwJT71XdCE7xoMo1apVkPCxUSzzqJI2qJ4NwOWW3mCSNoVCIQTcLDhIrVgEOynAnU7SbsuGwrJ2oXvoKYvDG0thxnq5E0IdOnLw0RJgprxNrSgK2OCrgc4P9aolBffiKB8YzrM\\\\\\\/8jAuvoSdkKzVx7VczBLkpZeXolA5C5CyQdKQ4Gc2fEpX3DVRrLpNpnNNqx5kjx4JVAzHRtGlF+E6H0xzoOzSJKB2zGuwxGGFdcEGpLTNZcuIya\\\\\\\/Bo+XbKicH55OcBn\\\\\\\/6C\\\\\\\/Fv0rJkoM6Q0XAaJr3VQK756lfa8bINKEQDkHUT2dDkyS2YLU+Rpiaw6xZ7lcB9gQLwUHR5u9p9ODQzL5\\\\\\\/i57pnakU6wXpO6DEUI1g+aXnusp7naN\\\\\\\/orScpR6YRCgh3XUHnwMqS5\\\\\\\/WRKYMrG0Ppps29iTh1pCgATZo3k3s\\\\\\\/g5d77CupWukH8FZFct9x\\\\\\\/qVSXkzKuliVZsHfd6X44VcPLGPeYGV8E4g82klDqcBcu1a2cho6g3kPdx\\\\\\\/92slfcm2LtkHaxobSgn9f2bXAjPjOD3P86vDANIBLpzXvn8V8qpXVTC+7x7g4uI1xdTss5yoJ8cfpxcISumE8btueSI9rUmdEXxHfmAQXdO4Ep3P6cOoWsbDWH1Fy53m9GcKzzxodJm\\\\\\\/Q2JcjuWGDLye6U0WVeQxY9f744FD88CV6CDEEKPMJqvATSNBNNaaxA4dOcz9I2JQvR4OTvVNEmLIE3hjlIKB0uvYk8Gq7gaUUMaZp5V9MOf5\\\\\\\/VJ+f0BYA1npLS4HUPjoscG3CTCh+bj+Mo4UD2zSNR8B3ZxRgoVjwT2GTMp1fp55POx4ky5XPhGhNVQuzl31a6ECKkhOcOJS3ZpBtovl2JmHuwD\\\\\\\/Vyno63p6EbLQS7303FQWXXZIkzT\\\\\\\/foGIHScWk2GBzw1QJrcZ30t8aAcAfAKPNYkZ+trlkwBeA74uKrtjWl9yVFPph\\\\\\\/tli\\\\\\\/NBmf6\\\\\\\/DHV+yH6sTRnPzCK+QkRtZEGca426A\\\\\\\/1Sug12w39ZLahzD5zk04WvCgkd0\\\\\\\/JwvafeaVQpH5OW2eG9KuGYqGWcmp2lahU99EjZqi2CbmDsghvxKtMxraewKnunEAP4\\\\\\\/BzDZ9olOl3LZHg8ZpE3DIg\\\\\\\/pg7AU1xk69vL80bb3qcFKiItorbGBQwR3EIUURSYb4bSjzyTX46OeBJPP1D\\\\\\\/5seqpvpuzeqJ85R9\\\\\\\/cItwOsIvG0Ff3aOWNmrs7P4GIK0rzI+BZk+pKv9wv6s8aiXCkZmz9NNI0e2SrcXV+sjhbIciYsiAAQnjoST8Z1Dr9f2dURbpHj60qVcbirzxXEtUifTdkcYgLS3Zj5DkY5ZOa0puIROOg67zk4+dGOUJnqbq+WAM6taz6OwguuK+qLarvXJEq8pvWFFJJ\\\\\\\/RgbCw6OEST9DGl6pME70Retx5cJZzw\\\\\\\/ho57vOOgHXkF3prOfMZF+brHVK\\\\\\\/atC3xxEwC8Vqaj5G6o+1qamnDedDclH62fcxALvZHeQ5pl0a1kRdihZCYg6ckVScmJQycUZ1d8OOGXV1X8FZtl+SCjVqpj38MMUlQcWxNopuUUibK\\\\\\\/u9UdgNGbQm3zNebZbw2iiXb1MAeQ+BF4XwDkhTsvt4gtsrItos2usTgOXDxQjJcfY2PlLMFJMk7RpAItFPpSkYkVnEeyUVofzBHOxlA4lexP+mOfyKPLVGL3UXrvtFjTuWEjP+q+3XYyMeNokrqJR6mFSqpw++QNT5Ri3GRsXf43kmz3oGQffl00hSzemnDh0FZW+PCruJXdDGfcKn4QsYxk5u9NvYt0FhJLie7LVS\\\\\\\/jFawseCqQsx1LH9+vHZMK2BLYkUzT378MJ98yOEPPWouLRqK\\\\\\\/4ozVj8Wb+aqobOhACedqRmUoUiqKZnLpMQ9DbE95hg1\\\\\\\/MPJOFkn4avu4DG4f0MGTdi30TAAvDk88jzlcDg97pgSI9Es\\\\\\\/82AVFB390v6a\\\\\\\/l2iX7OyVWB9SToNO+BAa\\\\\\\/ptEwootxdUH4d5LIxk4aYVOMQ0QuJ6gX9jBAqf1xXB2eS9t8Yh6MUxtHIKnyMF1jOfggJDRhiQR5Xi73YNerV4quysJRB7qtkSGMfUJQ97e3qAro7ops7BfupqSXa6pFYqG284U+ytKaVCLtb+XYgKHHTGKyAmgBP24LWttBvLu7w7qrM5verUV69pLnkzHcV5rlGHEzqMueAkG88+Ahs6dr4FG3QOzv9ewVO+6RknG8bRIQM35+PoKZOdIEL2OCh+khFkeauL55\\\\\\\/s7u96PYXG60jfFqa0gg+mhbFL\\\\\\\/XlsrqCRrEelru4VJnLJT9cUdg4AnJXOyaHqeRPZsftm5\\\\\\\/B\\\\\\\/Q\\\\\\\/VyjbIu\\\\\\\/ksaqxVO+s891uLUEYU2c4BRpEWYymEH4t50lR0VtHN9ooSkGNc7ivBKFlufGfgRAUIzKysVK8Vvwh07Y\\\\\\\/UMRismDJOU8xCNz2pij8QBFEf9puWPgSCttW5\\\\\\\/b3W9fguSIZh180CNFbE2yN9\\\\\\\/Lv8rvdTl+Sg26Me99oeyhzSbRj8fn3f7C4SZoYqq4JAlmAlIqcTFcZyIwikgdwBMnwOyQ5NAa8P0BGKDy1Fl7eM7xtf+xOe85EIMgZd2TtbaRSiM\\\\\\\/PxO6Iup8ROvBdmDIAgPYUgtvC8oLPJQKQ8HKFXZC0XhWiIwSffpEWsnJV2GYxen0ExSl1UR1csWXZbxkEYT5xM7m6DE6ZRsOtjHta7VxLu5l9\\\\\\\/gb+XT1Mdzrcg8CtWr8+b8D\\\\\\\/uFI46pldPY279s6kWYPZC8bz0CPNpnvWPgJfkoWYqXk7cFilCO0R9woWbKY\\\\\\\/tOAFJGQtZMqwcVLr0JWom\\\\\\\/jrTw6E2DKaw1qvFx2jByxSIF2OWx6GSjEFvtn\\\\\\\/cDDAhY6H7cG1lgQzbKo9leLvJoJAzBuW4+LMibPw9sHJ\\\\\\\/fPX7pl\\\\\\\/lDr1aal6CmT4+7zdc7sNdzbqNy1eK9kWJcTGD3hJqof1EzO2BEAwv3\\\\\\\/C72W8K35V2D7O2DOKNkOloX+gaLCnAYtCa44NBljWW8jA7GD8nbRGVJiGqCpqJIhXbL4mAH8KEzVOZQFS05u3IcikKYFL+iPfIv7zVAw+x3OIgxZDUojkmtmUiLCzK5HyCWMn\\\\\\\/rFlNdeoQiohxpLjsbi+uiyHjQlFzuq0O5+Omx5FpyDx9LqYiOKOH6i4kjXC9hQ01LKRUuxBa9Lsff5CEuzA8vw+boR8lKOzcm4yxjyeP1rIHerhtDP6otwgsawVqse3k1hplcTFg4JQfgX\\\\\\\/dUX1Irs8gy+nJ3KjbVN9GgTirWZGnhC9woclACZEIpajhBjQmyp7LSQqlp\\\\\\\/W0Jr2\\\\\\\/5QEk8o0R4ynrltsO0wO33rOZRUs0cZ9sq3+Y4wOhl\\\\\\\/\\\\\\\/DIbJbY+KQ12V0k7D01Y9aw3+INyWedHr218WfXcGkMCfjhzyuwEEnIwXyIO+qiKGQPNOYfo6LZzLwNywupJjKhGEQ+LhBMFt\\\\\\\/ojHEiommyaLpo+YJUPNixlZCcV4T\\\\\\\/659TLZgXLOOqh7lDHb8ZuVMVcxXXRa5U0vQdlNYiC1U3EuFkbkIvHsr43ZjO9xnd4A0J0kJuoG\\\\\\\/Vos0kyI6JUG4YsKIQ8TJQxEooN8AERnxJxbzLDifULt7s4c52u6+023ED9aFbpgd5luQRy4Jh6F5pEfGKfFyJqDh3KA303+jgvEyd4BfxSPMZx2Iro4frZUqt6VvJRXupcrkupSLf5BCutRbCCP2uBdvR5PGCxg+3j2SQYCbN+lZ3f9V2OEXJPMqWcHIFmL0ppxhLMsZvPmfde83+I8HAe7edINGVm4GTvajrKdLrGMsYv2+Z4tr+\\\\\\\/FF0oaYB+UgPtZcy0u8wSNO2kMN1BjrqdpVJoEje2DcCnX5BJ\\\\\\\/d4LqvonsOm6WytQLZCL6ZTc1rnP000s2xZJNtF2OmFUQaod48slOsZXSHZrZMR04w7m9ybQwaCcKM2AQ\\\\\\\/bzHTLCzQS+AVSPysCNJCGVxQoHttAcN0c17i8m1BlF5dv5OMyXX0OQTlA5djU6RteRgWUSerCD4khVB6osnMwmlduWsRRQa4b+pj2Jg0nyeR9ma8z\\\\\\\/gWjJI533dG6AuDPZj4h71xDI+b65Se8ZL8dPbDkVLCZR9Afk\\\\\\\/H07BfdEKKsrb7SvVR2Y8\\\\\\\/qqoPvTIW7j6sNfgLYVKXhTiPwEM2ysL1vq5rPv0ERnz3PLouDiS8srgyb4G9j8ZFNdc8s31oZKx+H4RZSCoqpMPwa81PTXE8YDvlUVTLLIc39KZykpqDsQOw3MnqUZaLGv5q+Q7Ar62TpQPF6hN9fOjuX\\\\\\\/e+C04Qg6MH1Vnt2hvcP9H\\\\\\\/iUFRYOByL+pyWNjXmuC7RqP88ZxHIKngVaNlD0n78gBZhd4ZE2QiUkQMD3hZcecQnL\\\\\\\/grMA7qtj4+9a1+yJcRoWUuZZnAkY3jdUqgeaiH13ZUnXEJTU+555jL4PNeTFyaS2y17mgD2dfJnJB1iqJC8cBLeTpXGP9c+BrvGs5nmyWOS9gAI2ydDZga9N58oA44X2LuBEJxzHHgIPEi2SQSd8wP+BOhb+5g41TSW3bmt6kYXnYq1qp\\\\\\\/pHboOkLIX+zzRFf9nlVmoCeKpxno3XkMJRrwaTpXTmmP8Tsj4nJsTA8D6cf2+\\\\\\\/cS9ddV1hqSP7Ta3XjvUT56VnS+1MVTsc+gswj0T6jmioU6zm\\\\\\\/ypol241f3meUHKDirbKpzIQ6V6cDRKo8\\\\\\\/io47ohxnzQZVq3xkdSPGiWMgFsAH2O7MDYxLLk6hgbnIeEtvF7rrsP76oZocomNslay5fwhrqGO3ZRutJ48oVnCsRfdCjGFNfAA5ohh1h+KHIlEPTiORYhOOcwEcNzxgS3Ysi5mW9SRWHHuFZpmW5Ik0DhcbWylF3DnTnIgXHfv5iQTSXbzI2HXwGk4f0\\\\\\\/IX4e4vpTYDlxao+tMxv8r0f+65femdvDQ\\\\\\\/Fxv8iKDzHNY1QM63zIoFbWqsbh\\\\\\\/zap\\\\\\\/C0ISWZPBlLKhUBKqPXHr3FjL3BiFnLPvRpvcpbIS20orcs9jDprS\\\\\\\/hdy2k+y1L6RYDAw6P9L2CE33+v5VuhxipXp7ez9AWRsNVv\\\\\\\/uBmJGsC3ic3p+b2KsPgI63swdUPJXmkbS7BLw+yRpSG6eMBOPyUhx30OxOI36hgQWgl2kgzLQHqC8x\\\\\\\/jvSn\\\\\\\/dC09SmXsZFhoqaQuYF4ewy6J59dva1YBl0Ss\\\\\\\/SlUD28qIHHzuNCS7xT9nYVxsrHnLV5wYNYZebpjI41TBOFbmUgxAeVXwZ7gIwUO5K\\\\\\\/UDFm0SgliwjrR6wj6Zv4RC83ulIfDj4zLQsSiGhMRmcJKvL4YhZ5iRFEH7\\\\\\\/R5qMMpBq7yg+KWaoeic55mypnm0R4EJECpyrPZvW7mKCsLPAxtvfGn9v\\\\\\\/EKfQgCt1G1teaRSxPNUmnkXC1ZEfwbNsflltm63rKXoN\\\\\\\/aIeY08RPRSJYIHDyc5R9NrQrU39JTkUMVfEavMq1NKXZluteHIPYmNvzCEAmJw6wzxCPWaml\\\\\\\/Cz413z4bvMuinhzaJLFc5F1DsU\\\\\\\/5EblfVORadbOTlJWBQURZrWewyH1yVt0ulU8SBLj4eJjA4pZgMZJcqYcI0S2yZK1aUWl6urxYkWaiE0+ODA8XS+4HAfVxMopsJJcbMaPsgmIGL4sMJEIJBF89g+MPo1drnBFF6TX00YdY9uwIztZ2iWdaoZMBD20O4oeni5u400YcWKR78x4ZwPhyiwlI7eaRXGDxpv15hIkpvNshZszJHKq2LR9x8FrlGid5skic0\\\\\\\/U92qbe7Pa8UB\\\\\\\/6Gix4c+lMUGOYabtuTJqPisQudhLkfWAY6prwnUPmna5rvZ7lf8257sAEWnhWxtzNKydWsz68fGl3nazKFfPghz\\\\\\\/aaXO7RurbgwAcDQZFPImZvPi1VO3iWXBdalmARogH8elSyKHson6SFDXd5GRKaQODDij1gjvbJlXasRDP50i8v9epcR24Dx60MeOdrWP2nw9CZWk+lgcN+eX\\\\\\\/ALK4mwteCrR6a8d2Q07aeMM25MphgJeQV+JU3gN7\\\\\\\/CspMLzQwk\\\\\\\/pLixvDhE48HlwcIRXe8ney36O4q8vlQDZdXcb9q7aveMb+Sw51zVUZtmgR9AfZC5Ed5deFMl4RyzgcJTXHnyCf7D45tK+bY\\\\\\\/iZAuNd5BH8yIqufmdeEMR3F+IookFk5HbsAIDpS\\\\\\\/o\\\\\\\/PP\\\\\\\/jbRXn3PfboRxEhIfATvNgrr4FszNPP1Vm5K\\\\\\\/m4mjmwLBQMzAuNcOBY5i0w6KJT1H74qorPn+3NhaLYxnov\\\\\\\/lOZH4B7aiJfK8c2QciDVWZvyQSTjpqpPeuowNbf3PK0obcRQuUvNVmsfT+sM4p4Me88tLqz3SX5hQGYRo9QMYOGtEQCc+JweEQy4MP\\\\\\\/IM5JPAAOq3IlbDxVjJTjOt6HXPAk5s8doc9KxoyPog\\\\\\\/56wxdPXpAGVsmar91xPS538gY1lm4LmdN4oGeUwBn0AyDDCoPtlMUhAkJH7lka9+mdVdlhaxSpGaLk9zbv\\\\\\\/e105BB09RtBIWvkstR7qgJ\\\\\\\/943U65wkPx1tXQK39iuCGmVE9Id1AM8NGqx5bJYbDGbOdzuFe2ceZhPGiVtIPD30tjkAXPRXRaFMUH8kcDFjUVzfKsnUnm5tBIj0ufzHLreFuVS4csLW3x2FXViTPSYPLWZ0yqRvMgKI4riGOnIkpUig02sRfBC3955SQVL7Dvz\\\\\\\/pCPumOS9xNhbYUy9g6MxQE56qSj7qlB9n05XH5m4n0zGL8xQP0Du46W2SBMT5LNFi0MBp5587X1cf5LgzeTy60OfdlgTxmw+o4Ydh+2IiAUXJxZHAYn5zbls8oBqKSwfPC27G80UGJQGUMQJKXlLOsvTr7+XBIQy9UKP+wcWbLOBKK95TD3msc3hgd7lZ0\\\\\\\/FG+4rqCzHOBFaniQZ7b65+pho86YMcze55\\\\\\\/E4zN2zYCz4bVhY42erNidoIjFN+1XkIuPrX6StZM4tAXdsy5IPdAHSDkk6gEB8CAb5VNoPBI0hzWSFMFmt9BeYVhlpuCOb2fsi343igdZ97uR+DTQ3rAgQZhOT7HWAL5yYQrVH7dXx0GZaFx7S8OVDspheYw96DY\\\\\\\/QPpCaf9ZbLzi9mOdURBNkgBgEtmAzFKiFgNez0y6FCh5yosjPBF+Ww4CUTFzpqODsGUDroaY77UdaaqQDpJ09ig8dl8lQi0wJI42FQJ8r27IOeXcwg33Bdrj3oACjVHp8sPFzoT7eJpPAoIZkwJb6eeloQqgwSH4WjSjAu\\\\\\\/Yafm9LqeC\\\\\\\/b+o4nEp3xdV8R2wNozffFIFOEY\\\\\\\/eyDT9IcwjL\\\\\\\/jg2PK9lAcRuJDlCXx6als0zvdmyLXBxH\\\\\\\/9cy4TQcOlhzMpUX+WT4nIxC6xoyuQ\\\\\\\/Du1C5GR\\\\\\\/g6hEzPQevG7BH6cRwWayjj\\\\\\\/7cQN+I2KmHvZKDky8ZyXiBU\\\\\\\/hUw7XmsTxv8ajmiOcqzhnemd51oLAqRFgEiaJiRQZSJUmpdFqLvVZ2pmn9XA\\\\\\\/OF4OMv9AMFoAXtGKbRqXqwUcs4ahDLD1aw3DlpxCSLcwRfcuVQxO5x04cGZ0bGroXdDXt9NDECnCyJ76o6KPQ+yBba7O14QSJtCv6x+XBz9vrdlvwtINKWSHq\\\\\\\/sjHljeRVoQqvMp2PcVvlpicBuL3PbOWKGCFw\\\\\\\/tyQeo5Ge90\\\\\\\/nwW8If6Nrr3YVypEsL8IXNUAwNRdVak3emwv6Fg6VT8yPaBQ4Vy2WqPbMf\\\\\\\/DhDeteo\\\\\\\/q2b0DTcYshKGh7wq6W7D7HMPWfLb83nK9QX1txQUjnrCh5ZOus69zgAI5L+WEggCcg6TVwivOu98toagKwDahm8XY936qoM9snSHyOj0yF63GFMQ47UshZSiTUKQV7o7d2TQB11NWRWLbmYe+ekpVg7eUW2MgxFFkRz\\\\\\\/3KBdX\\\\\\\/AOG+CTzAAv4s+b4UCwXQCYAvJrN412H36VKW14zvlvwqQfou4xH2zLz8f1iW1hEcRwX0dLeCUZdYJMFMc+LgPiXaDJ9ZYI3ASxTC2Ecm9UjgFMXhePaOpGQhpmtpdalgHABcoL4kkPkQC3Sy5TEtj0rW6UciQ1saoo5C+0Iov8teleJ5Kv6m\\\\\\\/z77TjA2OlTJ7S+Wa7\\\\\\\/nfTkq6yQg686RnrSQt9R5oS8Yo5k+7knje+N\\\\\\\/6cKV8s+AzxXYSrg3uDHfU9\\\\\\\/YBgOynSTspZcEsX305SQQi55Q8THq4C1OnQRloNpixUUnxw11IJCWqYu\\\\\\\/b\\\\\\\/bEDiTK9i0\\\\\\\/8BDvqhIPxlu2nA\\\\\\\/xVKEEFPrxRbi3fUvh84zBra5ivvbCx2sy0kGw1Gi06f5pTYirj+FR2qXmHRkcko0NzOd0XS930cFk5nVnJxC2Z0TCJ0wzQv6HuZJhHzorpwCDsOetqBf7K4GPZym4rRm+nYcmwrU9KdJ+D5Xx2B4Uy9BHuj2EzcvbfiUPOQc3rX\\\\\\\/4OpvGKsl8PiHCxNxAi2P4s57bF5CtsaxKwnx\\\\\\\/UKjZQJsKDrvjtI17MUVFAnDoE4OWrnZ8ka522fZYIGn4XxKz\\\\\\\/mBJbbU4zE9M9BCijOm+OakNFZBFvylz+U6pCNHIU+\\\\\\\/lBRxDMCpiv89Bjft3ccBTi7TYbsVK5iti0KD3o\\\\\\\/lDjIngt2zg9W1hi3Cgywty\\\\\\\/ZiIHdq7VaKii5PfVxibL3MAr\\\\\\\/XXi2T3Vn7fRM1oPiELltC+31LJbEAbmbmwmGjLDiI7vQRZPiRzzV5nyj7m4wRJQes+iRpY6dfEG832xCyFWPO2SVlcXYl\\\\\\\/MB7+WtbjXnSI2IkqQi+gLzy339THOdUVc9iZ1PBbMJ41Qli4bIrixGfUUaV5GcjEN3A4EpiWl8NQITqicHhgEOjr9oQoRM1EAbzsVF8E\\\\\\\/Pq3hj737xOdyuw7L3V8lwT1n0q1qOxDcgoCXYwIw7GsVL0u6On+US2KSW2f83SSSlphT1hSb7A0RCVxxxED1KtgLZ3glfdGY7zM51Zo1rFFEloXt82jwEaXo\\\\\\\/YwDF6rWw6gTR4Egd5j058nanjXctMBSZt1TMGNc6jBKEnk+j0MTaG0dub5v6Ph1euKZxUy\\\\\\\/bHEoYj1rryktIYJ8d28IgkxAWzXE8C+gGnYuQZy7HS+DaWcQ1rwanrjlbtmLWbHslbz4GHP1oDtFqmsC7QwLGXBhVQG6u2pbbUdWHK+sNm4g+p+jgnbxv98Z88vWcolzCesylALlEBXFwJ661ZJxEpOYCZg7MLuzMDRflVGtZitsXPtdLla21EdrSly2caJoZnZWfpFIbs61Sh17ccD4+Jpwflsa42NnbT3S8g9SarAAC10ISKLLvCzp3Umi+NE+7bf5w6VDjJhLwqb577opJr2XJVSm48+HoIVj7i2V+tCgQESgVv2mVs734zUpkS011f5yE+Y12Am1Lt+kBVJVKgMim0u8nxQ5wg+FYZM4M\\\\\\\/8GvpvVu8qCIQImU8\\\\\\\/E+HfegfjGIEK0HkrCAI\\\\\\\/a3q894OVLYXr3HUg6mi0NF9AuZ9oPQlOlOuaou0rIlM3r8dTMjg8CY7nXkyMUN\\\\\\\/LfgqaGmHGL97bYg6Lw7SgNWv7icCvWSUWiUXpfuUsTLHT1pHXk+2iGNcQLcjUfLg41p\\\\\\\/vENIbRmSBociwMGH9be4HFyxJ9H5pQnYCDjGu3pi0a8BGwlJxfqYGHPqsFZH1Z1DCmQb6O\\\\\\\/RpB5LXEp10k59v6Tpth8iUV51kjeC+YZrezn6VSpvrjeH\\\\\\\/YE0BksbaU\\\\\\\/t+8x93iSMinXtGCqlpGi9zniazV2EhxJu9pIC9OCsd4tSNVKXz8MH41UNSgwK\\\\\\\/2me3NB4ohCRcRUgeXa0tpM0kKGq2fGCxcs0DgaVC0xb5bv8kXG9EsjBbUkQ7vT+HuKpli3Thim0v\\\\\\\/RgbR2Lu7wQHLh7JlUjFUdhCLh923KYtrFNCaJZPEy4xQEgHKt0FjA6vz6G8tc846kaqkDNHvwblPuP\\\\\\\/o18FMRpU3vYUjxz6FE9TfRltvYZ60KfLnX+0GHkYlQL3l1SPyWJX42lMoJeSZerBPIpO5OBlyq7C0\\\\\\\/yAFc9D2A2ZzAZXmiXPFiJqhS7vPQCYfSASfn5j98SCIL1MW6nB3JfSZotvt4J8GKpjB457Gr4OHKgCKRK2P4WNf20fTru6IR8GvjEeQAeq\\\\\\\/JX03Ra1Ygzp9gdNgT2rzabEdV0SzHAZxmUSw42GjyRzCrgk0C0x8I4buNTef1UMvE1\\\\\\\/kCsIRsriVKD4Obx+Iywsjt9yx2kgUGvMjw8NUg8EMMiAu129Y0Me9P7QlsaMnn\\\\\\\/vaQ4pBTmbntY7R1oJFPCvQ0alYNs5f5QyYZlSPxG8\\\\\\\/cRxJWtdmeyf9+t6hpVKIZhDKkkGKm3wGYRyhZDBMM0U8YztnzZxwPryERhz\\\\\\\/l\\\\\\\/REDEC89rh5HpDhHXQ++mtj\\\\\\\/7P1faCHiOIP\\\\\\\/HIvq6n5Wydikp56n9hOmwCWqL0AxngkhuI7\\\\\\\/M3jMpkdehwHPDlfP4\\\\\\\/ZArrzILBUmEVZ7VHBv5vkfqlqpE4oYUiZPtwCEtFp7YyrrtsQ5VmXQKEE9dwF3ENjtZH+GnI7VjYQPMUVln5FPj9awG6AJzF2M6n\\\\\\\/31mU4wxXr2OSkdaHI5UgZGYKDUSSKdMvI\\\\\\\/eDUsYC6jph7fUmFtVkwDKrNZSPwyrbqZNmS82nm\\\\\\\/vstp7aqzspyjwlAMEl61xARJ5icQZZBPRNeXSzcrXIEBL7Z93CvXm15j+WNbMjBy7zgVy9N54c1vtOqbZfFs4MnwNaKd9ewNbfpZS48M8mDgqI3AlrXaSdk2GQazN\\\\\\\/u7EDtbKZyS8W3u\\\\\\\/B1qgLkNm3A08sn00Q+eKjyxsqJxJOmIP7IeC1n1MJPXLHKa4nTqYLMPLTq8Qf0GAzno1SJB2l0KuO3GPYCZsNvow25\\\\\\\/SLKw2VQJu+qpYha4H82C\\\\\\\/MMaRFZyaeLcdxr1fSGeUMrp6RCs2C\\\\\\\/g7nQ30lC14NHYQHpP15k8dj1uqpzNJ9xM+u8AsKH7xdZq7Av\\\\\\\/4Pcuq4p8ChfXAb+9Wdzn5HludY+ezIyWIacqZhKbhpyXwx4HzSD9+ZASIXUoFkAXKURzvqXwOsmAfN6QFb7R7xPY+0fw02yDwyzRAeWkknCVMRzc4SfsTqRns8\\\\\\\/5Bl3olHZyOEXiLCVVDqeN26elcvQs1MmvKkoAlGKvWaIT\\\\\\\/lx0Sfd6IAH+ywNf1SunC2FJD4RCdzEsbg\\\\\\\/y0wp7KyOzZbG0V4euw2dIGXgMWPPsP7UnFrvSPoBYgQNpj8hpuI6YvC2DoIcEWxlG1XcrNIBYxI\\\\\\\/s10aZnvfzGqmGAU1WhifnuHi+JDUkVVxSZOXzKuVn8DeIN7QTv6\\\\\\\/qcNCktp54sMO76VJ6x9BpKrvX+NSDy2XFCPT3xBdUoMDw6JtMvkxYvAysf9EDjrMUSXkaVID13V91ZsWxeC9ds9k6NhQWY5Ea4mS\\\\\\\/SbOlEemXAKK6mjBaDhOvdCnN5AAea4IJMLxk6\\\\\\\/ok6AigQATxykyjCS\\\\\\\/+T16fgtdfxOv\\\\\\\/8yiZQFDt4xrJRyqRnrl8knF5T9Fbmuk+49zPwPZ7RNGJ0DZnB0a7SM4cvTBpQIMkJw2jsQ26tFP0CUZTw1ITcEVEqSlYHYXaVaYzs0aVVmPwB+ghYFvMQ\\\\\\\/MHqITJRB5sEcwx9OVB4NA7Z8EqGzPYzOFQnaOBfmtxN\\\\\\\/b6l\\\\\\\/51UeuFE29x3AHrII9iDaQA\\\\\\\/OENN6rf0lfRNqlw7awH6CexZLzrku7ePv18m5Ft0A4TF8xdCfsySnkFReC6UWQEZNDfaqgMZpMNJ+\\\\\\\/gmmH8LDWojCdufEmSDh17uduFLhsnGgg2+gCgonJ0kJQcLZw9c8FZq97\\\\\\\/brW+H8ZWwFW4fid2lczjbAri9SnpX+H9+izpRXszumlzcq88GIbZAzAv20PXDMuqN8Jkwr15ItCFaGbcJna0geFM9AoMp0jM1IK6+7W5ggVm+s\\\\\\\/xKr2AOKJ4tqR28r6GtqlM8P6BmwXpvfQmwKyRmqjBuRfo4LGb+IZ84HORl4hOhQ6Xl6TcqgbeMQJ1DK+TS7f8eeYQOHBytZY8EbJqGXCQFuNvFbTgAB\\\\\\\/g19o4LjMtH7RNh29y8c8D+JrB6x5\\\\\\\/d29+7Xyblc+VcFFnzeTsL+uKZBo1ZdoVjp7+ir2etw+ebUD6BwZj+6YghyAmL0rc2Z5P1Kt4k24v4bNUndl7ryPy1z3vmX5Pm14AOZtBLjpdkohm6Ia5GlkRIlWI7YNrLYk1ou+tRuHHiDcfZ8dpPBANHUb+5qYWusheL5A1w3s9p8Ud5jTTv5Yqj8RsqtyLBYVCtxJ3kTPKi3aLWXy3u3OX3F9YMLEbUs0r5pOUnk9hcotinTrYr2xGUn341XtOJ8smFsM4aLizGLfWj9P32ReTcbCX0DUQU5WeFGpl8wwnaTeSXx7A1ylEogpyVYvlvAFzlmxnxH9n6WNfAQPfvPCDjCJJ9ZvyNQ+IPJKXaPIQdrBAafOBneAAPi4fq+12vdIYyA9GkQ9ggOagkgRLXUIHKmR+njarVt4Nac\\\\\\\/KyYCXrDunQuoJRNTcqYRKtQHYvtERGoFaM49hyX60goz4wpg6Sn4o\\\\\\\/swbmOxCcGqz3PQWCCZAiLx4kW9Vl10Gc9eaa\\\\\\\/8Kg+XCuaYi12Uh5tsaO2NKpP9lF5vqDFwLasiJMOyIy7MrtyDyQcfKzal8ffSD6AQAJIVfBfTkLGbW17ydrFdApb7TGBp54OL0LUS3kVVK9mU037OTE9HJbIojqVMMQm+TwCSKWiDE4XnyCtmuBcMdkGNLF850W0\\\\\\\/Lg2+Y\\\\\\\/GD+g9t1qmzGKGhM5uV+kKM2ZlPctQrJTy5mJhQXmkS2xQqf5rJxbKGYpWRXD8CxESqqXGmGqLgco9o7aZUhw9RWb\\\\\\\/ezT9n8O+bo4W7if2uMo767yFbl\\\\\\\/\\\\\\\/3xwDyOx2Jz+CmBroOrxjtDZEaXNGRf\\\\\\\/nBkZFbe2cWfKqvO\\\\\\\/wL7Xa26U5v0X4Aa7sLVmQBp3MT5Hkuo6vLSp5CR8K7DmdBREjX+djCcwHMJraVgIDl\\\\\\\/TZZpkDG15m+dana6iI4exqJTCdLne1WnKvABAtz\\\\\\\/rgFWr5ycfIxazKVJMiODTlKWN13DOaJO99LjBU1rehfmk71JjeAcKrv8mktw1f7On6Fx98H+DuEIhBYZMZA7xJz3rpitskn0jdD\\\\\\\/Ie+N56NgeoRFCZ0IkmxvBVrs2JCUapOVsTYRgYhka9RBsnlFMUhug2nkD2y0R3XliFElCYzzKqbJHp5IC31GFK2+e63KX9\\\\\\\/l2IMEVLGKfRuhl43AQJB7R85HK7NuqSgDgq2j0TupECEOjCvk7dprHtNamLmrBkiERD3JUB4q2NYfv0gdLp8fSSrHMGeL4IH1HfGHBGFgzNQSqhhEBFW9j8vpBUkCjDC6vas1NKAbA8N3aTtb06WeAPJDwDMMx4FmszMwqepyEhDdZkcTUFACyzITFkErQtLuz8OcHqNS7WCVUHvmvEeRv\\\\\\\/GttJJpPchKvPDyJMjuVrnMrW+7udyy\\\\\\\/YJQ2f9ceRJKvE9Brjtutv81Qf0aZ6uEEy7t0vbwKa0fFtIiUyt+ikYFuY\\\\\\\/tSvdTU1k8dTML+kZpF\\\\\\\/RxUltDJecA2OreWpD10QdYYT+5Zk6LyvyvfTkonG8Kuf8NiP09yfPx8cRv2LSXhpj1HKNbjKadHJ2scan6ZScyig1EIegm+F+RKcuK+vyWvQVdY0oyo5Pfezq2Lf66GHo0dzQUJIx\\\\\\\/gtuNRlGQkATG\\\\\\\/ipz\\\\\\\/epfsKDbOKcVLzgNqjUZMZJfH2StEDxELnRoupHIEeu5Bop6LdYKbyGwu9rqp\\\\\\\/CJ+cjTtRepMxZyTzj6dg6WcLiqoBXKlzVY96\\\\\\\/dY7rJEOBlYmfLVvwWNnfv8kiEi9vGJ2sNzt+4Y9oPIspfqL+kiJbAvhE15PpGefpfH+Kd6j1iGLBh2jg0pEcwRXITUz43YQDgNVxJnUZE6a0wnfqDmmgkQ2qeOmA8RnbGkMq1vGsELlZWclk16LiWdQgCh6kyHChKtTRWOZRpcEI1sOS01TUSY1Tomq2Do9ZHn38vmBuzqO0wRKSbDRYSO7X4hCim4dyaurFNl4AwWb3E+F7kFthuUnidG5ImtsKFa1Bcm6yUpFCHp0F+Vcxs7cb+3iR1SZ\\\\\\\/aSXMlg0h64rA5iVqjygRFNeU\\\\\\\/MxxsxcMQLcdyZX\\\\\\\/UTt\\\\\\\/uQ+c\\\\\\\/qjTogPL4OamxqduY6Jy9KXQFzUXeIDmBhGLn6OJ8kCw9MPkj3H3Ss97mbXUD6EZM3CqEgypAchjU+58nUsXQBb+ZWa\\\\\\\/FLODQ40bnLLMNbZ+\\\\\\\/o4T3ur9vfLWnIVPZGu99IlUoekw4XJyAPvUurC9RpAfLpW1qPHBF4zrj+V87U6aM7cH1UAPcMMhQ9b+OvndTMAFNhM8RELdcvkbmHr2UeJUahVehkXj0+BhNplcqjEUbejltxTH+l9qwgbpGpXJTHNhO4fXoGokWVF0Vkx7b2y12MU7Id+KFzZrhn9GJbmk9Lro3JGxBHyLdF0sEfwXnShm\\\\\\\/J24Ocpe\\\\\\\/ygXWfSqWrQm1R00+VrZMH+9KAUCNUw+3I90MzBJEhQ9KXlSG5ua6kOIM80\\\\\\\/iYRnAGftsRY8Pg1Ee6vLUB6R6WJgdAoqyyWW9x88JTv7wg8i5djJG1ZVzPQ06AygBNlkGS83WpLFhrIJJlplHTB8l0oeRwB7rLbjV39ike8bS6OypYQAIH78YPMlIqmDN71FMpHMxGdkOpFlD55CIub9\\\\\\\/OwVPPnNkoxq27ZX1TYi1ZqzogX1oioV6yRtAPq8GC+V3kcomqL3UpPcH5prTHuUZYbHI0TzefiUSh8sV5ylXKzmvxERx170n+bk8cBvevzapYq+gDV0vBarC8AIJQJbz63XBSqQxjTB5W\\\\\\\/vSEDNX0didmnT683bpF5tmdmZDSwsGvK9mVVGxIdZrdzzoTsml8jvOuNj\\\\\\\/FtYHTunYYDcHrjpxeRSpwjB+HMczFlImlDTF8hrHE12LRgS0aN39yeeimdBLLXxpSVNvVQdxZW3P35BNRvDFJ\\\\\\\/EnJUMBaa1X\\\\\\\/LG8eOiMwCToVeGJp3RBQH7q+te1gaa98XWhaN\\\\\\\/Xy0LIIdvgUf0f9wptUC\\\\\\\/c6LjeenoNEi0B95wdxsAZ+pfHyTRSBTOKSM00WE6VnEh2astUX2IBaPRlUr1o6BK5V1cmCe6405mgMIX92f6Kj\\\\\\\/Soxo6DI5lazydjurDF8bKoWn6PCnDqm1rp5Zb5PawKRbG7+aevuzYLs3hz0HhAzyZe9Q2d+ox0tDaDVUOOx8ChW1CT68S0qeozz1BNd6OsXBTQ+XQSG1sM\\\\\\\/S4A3YrnwByh0RRh0i3AceFaiJTStEtinDu9gXrgKAOqpOINQRfx67X\\\\\\\/mDZPUDCNtaKS8o9tOlG4inKGtvK934AbjaUyQ06pcTzndbgZQ4iU2yY2iRUTRn\\\\\\\/9aJG6mVJ9aAq9LgDwvl9jLa6r+cBIGa98CWfDipjDMQHSXUBzvNy74mAVtt445546PyQjRibWpqHTpCjmyCnbCpo30yNLT+v+CbQdPW7tF0Jy8Mt0hraSXw5oJnGqfjFJESR36CBfxXRp1CPWJ5jH2I+8LTC+8BMKBx1NeZ+7dbrpT+CPMKR1uWbS5UKWYzVoAO8eGuvrVFVwbPdeIvtAK6KXH59tHUhXXva2xVT2\\\\\\\/8kT0CysIc7ejBSHTYDOSEyg+kFkwqCLean6+izW3I7iTrjRfwMiBQjACNFdTAUNip4NRy\\\\\\\/fRtHEmnWdBCd2TmlZFw8OaKIlk9bJxPJT6DWOkWDJftiY1JvpV2vM22gYTEcyyJlskx8pu7ApCZOQvoci+1J5Hu9DJy85Oh6VGVoRcmZ\\\\\\\/nvOBe7+nkNbFnVSzGKawDaYCFuv7VfAkkoPiv3Qpdg6G\\\\\\\/1e3YJ7DYhO1Bde3RQbEqRBBfIiPywwVP9yTAbOiPfx7p1jDslQJ+YLhSX6j4V+iKRahhXutm83zrvsMkNE7v7k4HSfXXHUa1ZMjUR+wIWJABsKBc1XzU\\\\\\\/WgZxKTRTQh8h3ALnbCmreIe11Iuf1QMFFeiIVo7GnKRVWn0vBJIt8ZT9fsg+Q5B5\\\\\\\/TGmYDAiXgMKa4qa9f32Kd\\\\\\\/Nsylf7Cfku4J\\\\\\\/QXNmvjjlUikknuGa8htWIbkISZbTVGbGAoAYHuz9TOcjBEKHoNDPDVxVk3yhzwzdgJO7aQBoV+t0ddD22lkOXJlqtl+HqaPHS9oK+phtVM47P4TqFFxb9qjcydBR\\\\\\\/o\\\\\\\/RW08L85HsSce6aJY64GIOFOhOD8FrZcuRM8nTbZpyVNTSW36uNrxBjo\\\\\\\/CVRRlDCEPHWzJnwzzmxydncxlTf3aEwBOqQAvhKJIxNzAWjDDRlLUBYPMt2QOyZa3kGDgbUzLxq3bj2jnWRsoICtkSXl4zkLNC4YpdEwL0gCThdVf1bbeyTAcL4aIq1mBaeUnfSLQp32xOF+lI2udbcdCq8Q9Iis8j05lh\\\\\\\/EHlU\\\\\\\/JYJNmSbz090biJFRhhC6clUdCvdVpUUGLGRvEhnmj0kLikrBx5C\\\\\\\/Mslbb0uM628eB\\\\\\\/lXUInBlEoQQtjGUaerZ7nMBWWh2ti4VRcwY+xs6NI1MkYfAO+3rsKatyaTCZVMH0N\\\\\\\/oOXxB\\\\\\\/s5Avop1qyeY\\\\\\\/JtnM+GIDGeZU30zfwS6f5HDuIUPZqbZaoolphS0txKPTEukzfeqV4kDMeKXQrMSMPcXK8cSadCLbPeE28scbWCVnmnGO3RanqPkxNHW3eVhXeQrOlV\\\\\\\/oBr6iwgdeFyrg1gDW4xiOGfceas7+q84LHomZZTBZMBj+MrLRfd7xDMsKBzOCTXojSt73oWCJj8susngEwq4X1GS1wMG5vTplpCJvTxmDYcvkgvri0+mAa9dAVih0UcS4ikviQsTeXXTXpc7g2NJtOouPa+R5Fo\\\\\\\/TWfjru5Oo1q6nOdx6lPE+iJIehGhnC2Dot1yaYVFfSO9KtXVhGSTOZrCg6YGbQG6F1TzoR8QW6bi8j8exdGQiMPZWcGguJAcPgaPzz2DCePnoqrTCU0FHvP6Vul\\\\\\\/6smbSzvQv8ToLfigyxIhhQ\\\\\\\/LF5+uipVact1mj9iswUyodMm5Sw5tv9Yo3+l3omLgFMioyN4mx6BAzW2Sbv4HuFNlk0ksOnKgvY\\\\\\\/0shUlKBEulZ5mO8K5rGQFrEWMtpRADVOmJkBTm5NzFm2i85GLBEfBHi\\\\\\\/Kk1zwlXpJYFayMUu\\\\\\\/nkmTT3xR6Qgf1iPs+JVnUqfBybpIEi9NJR\\\\\\\/ZYyj5TUuYowUwtOiAkvK+2zo1oOdA7ikbtGY5PX\\\\\\\/9dAWay56AytdfXRjacxUR\\\\\\\/8BmGFp3A8+JrqT\\\\\\\/rWB+6Ilx5TmmWnfkIcqylTOUT5N5hIRNTyguCQZ90xIE0pK6mUX\\\\\\\/YSKODzuVfUEavwa8x3tPiahOWaqH220beSA1+tTnoH8\\\\\\\/xTjP9e1HkSLClwfn2cShWXS5jJIi4UcXsCmJXUwl+rQchiMzZMfgyb7vZ5S4KC26ZH7Nhg1Wa1\\\\\\\/VaHRnYLGe75c8dMt79Hl97lzahfKvTJERTwmNjWiYi1KFKcuXBqp1Yhdnay+KIxUwoa4PUHaMrwDXb3qCQ3ZKBv8UPyGXDHCHIDbIeXtJh7EdpUtsaGSvOaPcLAcV1BqFRlZUgT0Tu4vZThCmkYvX\\\\\\\/EO17PuO7W3xPqmZeP8kdJfHWfGS6GCpem8nO7M8HrpN\\\\\\\/kpMrk\\\\\\\/oPfsyVqCosqPk1xEVXf2dN8JJDa47qSjfb7Yf+vMb6sycrxLrvJUv7+Svc7JwYqb0yd5n2XGFJblXfggZZYV4s6O9pAZX4piR\\\\\\\/nMIW+WwZ9SJYkg6zLlsl2Fq0NLvlhBBm2xwquuWxZU6U2FBpCLGFUmcaZEHg7peOb15DpFhb9bnCUsWFdxJk8yTpwsyJ4mzmOjSXc46H+T2K5WjrFHx2otkm8ZY+JIg15FkZ\\\\\\\/J002zKgJcrPezNG3Bb\\\\\\\/zDQF9Ax5mI0mI3JRFcd3r5lDcB1gJTes2a\\\\\\\/0aVI9vzbMSp1dKoCgoc19BcDfvUJNYjGAVndVYgBl9hlgk+woMYUf\\\\\\\/IzvmLd4EgtGdkJ0BFg5L0xxwsWASRQi8qDSjEeMMHdww+0l\\\\\\\/4cOPXcJ7smOPLkaXV0e\\\\\\\/TXfTlwg+QsMnZ0ZyuLm0KBhUCtQUU0qeMIYLr9FZqXDoVoKOgGG3d\\\\\\\/JtkZV+1e3cTS0clT+g\\\\\\\/u7ujIhyo3AkSujCQRPeke067sh1y7LE5kmBHuQZx8MSJk\\\\\\\/+re+19a\\\\\\\/dein+d\\\\\\\/zihCMXl11mQQXyCVhk7UL\\\\\\\/cFWRoBKg6G3MRsr6AwP6VXfTxng2uXN26RtHpAFc5y8621b6Ivr4Hf8W696eWz6vajSGYUVD0QUmmEIovBnFzVIEyRdn5pIjlBvtnEhC3WTVIGHh9oxOSd3eiZzCU3XLkhtVZ1YncaqcfQAIYY+6vAWCGGRjU6TFlp7Ga6iiLlKNaFBxWHQswkpHBPjVT0bh4Yrpouuq5\\\\\\\/8nUEgJhWuhzx6kURqdXgu2rDC4pznFKjdlneDwvhRl\\\\\\\/cW7vx3Cz3uKSpfS2wUlyFIMRrvNEHMH50mKKSMXz00tpYaeWzk0+7\\\\\\\/UqqC+kfJ0ZHcOzEwGOUtz0SwCK6CBNCbAIS8zCmpAynmqgCsYcHcDd8cWc13E8y7rk2R+LeQtqKLQ\\\\\\\/bSIokHjOv\\\\\\\/ZePE4mdAZXNgCYzgv2\\\\\\\/lk5R7TFd53nmKEQWfTsS0S0glkHLR5n0Z4GGIvTvA\\\\\\\/jKJsCCkasVRexjSN+phU6DsFHqRaL3u8xSPtdHMmY5eB19TTbhZxDhZTOI8\\\\\\\/dQUGDs1g5MsvQd4BoW9AojHI6hVfBVQgaGI9742sNO7tIWu1\\\\\\\/YPu5k7w4bQInLfJqayIO2sqVWiFC8\\\\\\\/59r5fHxN0DBR5oxiZ7AKkdB9J7Zf85sgMTnzeaAf9PbiRfGxZ2InydVths1mHQuT\\\\\\\/q0\\\\\\\/bUoCgYaSs9xFXAfIUJVLBZFaD9L90UoXL\\\\\\\/Dkqzs7Q+0pyH91UN0TW4XzNRkoTk7SJ3Tb\\\\\\\/fJasqABD3HhK36a\\\\\\\/XO0xGJq4Cl87P4I38C8\\\\\\\/LFuFd7w5NOS+E0oPuSRFmV0Ru2yZRxJaGtTsJlnqfRH+ODUIhLaSwalmb3SgGWQq79vaskT2OIeLVVltxcGOOHINkOH6F3ppxE8vrB6Ln9xwWxOGfl8y5g+yTABDAB07j1+jDHJG4Fn2J7v2kpPOfvVAi6LIH9kp8vedbqnaqU4AKk4tM0250ifFFpBXr3Dzyehwyz4cUhi8yE\\\\\\\/qNv9nGwMJ7Pxfm5N9a7msc\\\\\\\/tk1PziM55GPfMbsZ9Kr6ReMEY2JKy4GDyF1Pkb1CYqELidKo+KqSig5c9YfVqgVp27knQ9gmzmaNeRJ\\\\\\\/qOdqytJWnYMhsdKKpenuXSHyRPqbxYZHvoA6za6Jt8\\\\\\\/Xzdd5VfBgGbGpKadgXT+IvZC+C6cUV73WXNo6pCv2u0VA9do81Q9jqUpqgdvW4hD1IRgG2H+LZeeNnavZpJsMiTMjTiiMOKDqKtbvQKj14j1fDYIqOjZU8qqXNEZb36cfQHU0mSTwYoPSvtkltNDBeh16Z4wUBilGJxcVzlWJfYEvsykZTN+wSXM8SSi622e5vQ2J6rHVwx9+DOBZKNcRlxXpkEjBMnW1f5xJWi\\\\\\\/01rPGOoSqfgMFatkppoOM37MUzPUBL6dtaxkh6uSf4F7D9on7XpdVpsnng\\\\\\\/Zcgx9j2SX9RBYibp8u7+E8BwZOZO3YistdJKyslWkoxO5M6W\\\\\\\/5H9wBNcDQ3D6q+D8yAUeSxn\\\\\\\/hYhu34CgUGersDAZddaCHroA0dBf2NSTKmYkIffNKJJkpo5dhviBM22vkmZI\\\\\\\/jQF8ZDFYG77zz0ob0X9GwczR84mUNIKWdaMI\\\\\\\/8eaqxuQi5jU3Srb+DNiqK3LUxyTJ0jAsJ\\\\\\\/z1O+NGiLu7+2JRX9bJ7x\\\\\\\/eyBgnJjyu0gUAENZMilsv+PyX6XpyZPNM0y6HXqzT8MkRufdgtHR6paZC7jqUdMj8z6Nfbw3PBdF8UI4YX3pnXzDPhXmUl2PPgf7f0UDHLSLAZlOhRGY1g04kYvL6tzkfUFsbhKUdXy2SFFZyIG7xx0LZ5CS0h11XjYF\\\\\\\/SOvZBAg\\\\\\\/qr1zAKsrPz2Ch1b5yrZ95ZbY3G2aafJ57dcKqNNycGmS25HX\\\\\\\/H3M+ru3rRw2HxRU5hZusDGM\\\\\\\/ql0Ux6wYunKF4rxHwtp\\\\\\\/YuIXRosMAxNMeMGAWyqCAQSW0UNxIcVvsy46xfdWYNaIFC+QmFy+Rp\\\\\\\/Pit58DQ90bMVy6wuxk99v+ZpOpp3JIEJoim7D8xoUJ+ryEt2GyvOkPZdmtxpT9\\\\\\\/aDMkZbD5zey4txj0eZ3QNqzeWMnSjBeVnsFiYDyGljIbdB7DbkKmtaBmcYbZMUXnd63h2AFXRtre9RV3cIXrf77K1NCtfo4mQ5yybvBa4A0oKYdoPlKQhDkTsR3y8ragT34aHS0LqaTUIzC74BoHQbi3QxGaurq7tUt1slRF65dPf5AXDyGsjc2m9pY3ROm2QE2mw7eTGlaiMSRajlQ4p2nMLFtt6HSap2aEM5MBGa27SNCdh1yEf9WKpq\\\\\\\/EV2YK6qIe9ymX0SzqoTuG29fOq\\\\\\\/0WUTIqqM+I56Dq\\\\\\\/KYwYe1sLZY+VfaP7VWIr5WaBWhbrbM+Fm+wzwDlhQ6P20m5Z\\\\\\\/nxYARPOnUQWi5YrO0NtVH7POYflAvLXnAOjslavciu9thpfRBWJW4zoZ3W7HY2wzJuraeOr7TJvujfMom2udn7gGHkrQzhqOOY8cpLXzX1fVJXPzmDT8rXlGlKAL4\\\\\\\/7XxsifZmVa4hdkHVAnn+OBt08ckEFIcodCi4fqz4+BGqT3nCRuiLM3uL86PnX55d8ZuFqJ96T16fnOUopo1AwoYVxoDzoy+ymuAISZvD3b\\\\\\\/lbHFrBA9DjpxRKtWlVN6obbnEpBJ3vJLnL5kM0kAR+lYtpTxOBGrLbsfLPA+x4VdJIFcSe1sT0nIMWdjbjD+jK+8QMRHRVYL2fJAfwz1I+ONtf0+bAAtsn1KQqv99cP6b\\\\\\\/\\\\\\\/QE05mn\\\\\\\/oXbfD3FlPilpQxDKwo0CI7iVaesRw9Q+k25JsKoqelO7Zg2GlVMt++KLZf9srO3KhFAAbxZ84WpOlvA\\\\\\\/LLvR+qVLV+aBM\\\\\\\/g6ZvXLGFaMEvFVFWh13HFMnaomIMWEwF10eqV2N7hFtpGi5fADUrk3YsYE2pbBa\\\\\\\/jzrOUrJ8vbDDfD1xGAl3QiFZzkbeouoAXoG40fxXvCsolGacDS\\\\\\\/qGcMQzuSphnyRYCnCBRjQ8X3\\\\\\\/TdauB7M0eoIPtM5R1D9cOpmr+dBjRyO6pRC1H1p5dFEx6QXWod2sfrlJE3PyXI9Ywd+32QqvERDZMBogTHpSLRF3Mpd+IQhrUNCrbX7RFB1sJIessGdh3+AFBDYkSW43xaMTiLWmHz81LQJPLnCUeZSChCSZvlyI18vaArFrY42LsEcobcBZGEAdbEYTol6MzmoddQwfu5R+ysWGkddrh83SIgCKol5fOfv0HPUCha8sm6U9tTAdOBGiWmR5cqrJcjoaXu8w2akLKAmFx9MKWZczeBdT0oiZJANiMYPXHPVZGwbVGKMUdfgNwFm4AYHxqdr\\\\\\\/BcUzWjoFoL9nGgTCuR3dANa9pPJnbPEATM5OT1X6W+kp6McaNA930nOOdiSDIlhQBwmdelnCeHoZJp3B4WR0V+aV+nKS92sILXzOzZg2ZDS39ErIi8fkyIAkkFTL370D6B8jH7NBYkCuzpqs0JWMrr5b463d\\\\\\\/O4GdEiJmdPeLH5XIZvgRlto2lyv8x9APl+Rs2G9DcpE65lD7FVBGXsRqosfkQtthLYPZiaruFDKl+JRfpVQFUhvEGvSPvVUXDrbPVJhG5DrSE7gSAYP6VhF2jgxAmZOTAwTOm\\\\\\\/5U0DzUN3a5gR1IPaqE6s5T+CNKcXPwgr1xaz93OuH5jnYxl5bkwpZhXKUGjLc1OCdWW9Ln+yqoYAkU0uS0DJjT\\\\\\\/iXST2RR0oSdmKtOzq3lefPRgKa89nFV+SlZEl4JE3B4xRqRL6Dug4UaSn1srpR6OXwAE9jD1Lv\\\\\\\/4UTnIA2iCydL+s2xBYHzsBY3RKgpwUgoeU7m9djHaW5TkvEtJmOjgMt6CW01VD6AzvYrsXo26JsKlm11Eyw4S4z2FjDQ7YtgOqVg0YFVjxvkSL0vTDBT8oL344xeC5hAbixBW9lc5AedUFfI+Jl4z+cqkRpUEmrOQfO0\\\\\\\/eMLXzW5gjqztq3OdArhO3LrP0I\\\\\\\/\\\\\\\/CqAS9WOQwEuQMJjZECSjhQbe8WM5iqPpufvWw9CwB65RELCmllh6XxhpPegZ7j6CJ2g2BfxQMMll0FUDqci9w1biybHJip1g1b4nLdLmJD8D\\\\\\\/aV0z8DeUC\\\\\\\/MIyAi8+4b65iox2pPYAleYbkbJnD1\\\\\\\/nSx2qr5c1jVPtBfY7Xzrp7wMyZ9Tsk87QyNhGk6ElBbwrE3CHOaGc7tMg78CQm2IkRz8B4VEFVXtWZAOomOdIaHQtKOSGc57IN\\\\\\\/vNo7TIA9ezgS+MTsNtAd8AS9eFnrAdcuuzOzaAtRmtYFJxEVUcDBmIaerT2QlTpL0egzGG3QyY7\\\\\\\/VOUS7P62KgLtPsxhUbpS0kXgc5JPVjlUWTAJZQaa4nnXFeVomIqjZWTnRVt97R+LMG8+RbnD9sD5BE7wAdlVr6\\\\\\\/E6GeDb\\\\\\\/CJhljLCZTRDjMADSMfrO6M3NnRv1yHIi1C+Cgk1VDJA8rZqAPf8mt0HgKL0Rys8dEUZlksM3N+IOgHDubTwGH+F0xjqoNZBtp5EPo2\\\\\\\/aiQQYe80RpV32yUNDHQw2kZdSONUJIxLT3bEVBpCUvqfNFe3PKqYox8di4sXRmtGfiY2djv5OTR\\\\\\\/Uqcm1CBYIdgCDw9xP2uBlb\\\\\\\/DUHiasm4a7utKwjNStLWOH6r0IJUgoBwkboZUjq+J796pbqRX6wCeJWXvfaK1SqZMpxE2kMEpEpmh9cgCW3PLSgbprcEO0r1\\\\\\\/01esr02Wqr4B30FUyv96fGrgWx5lO8+EobY70clEBiUUa5kGgXyeSnJ8TqAqeNo6TUXi\\\\\\\/UN2bJlpg6liDMI2r9B\\\\\\\/96IDKQNbGG1XDCQWwHrScdbOso71HLKjuWd5k\\\\\\\/1Xb+Jk59dtZWUtG2ei7hBOYGC2RlWo5sZMuhxgu1NNDcfeuVE7jJBVaYEhDgd8Oc040l\\\\\\\/eYjDIRkhPQLsTJyj7Rh0Py+vQBLduJpHuxhNWVXwdDyTHxqMNAQaPB3MSVrsqrypx1kbmSakyZLIiAdm3V0Dt\\\\\\\/1yGoUCo7fhi5ImLf\\\\\\\/k6EUZb+qufc1Wrz+BjS5G2XSpM74bhI7E6+WnWxPTlDmZqZBl4rE1JKEJhAaT+6bNgb4UYijGQml5VBveSe1JSdckdtaTqcnli+DqAHxP1+IB40RcF1nD\\\\\\\/vHCAbXGFQ\\\\\\\/anIrbdkIOr2RZLWMb2wn8oPLs6xxWbvOx61ILKTIuhZQr730efDxnAUDE12+piqFujpjMb0bWEZU\\\\\\\/xonVDhx6RAoYsetjisa8KLg503scxVIjHiQbZFCkkJD1oDL+y\\\\\\\/COXz0UbOYUp5DKuFwdiPVTarYVmRNRafP4T1Jq6ahTZKZmcNhFEhru9BBTPfrxlQzH1pfHumH0k7aYpPUWwMQveeMonYTnGFK3th6d4Zv+Tzyemd0Kd8ZcLV5x8mOKs9j3HDo5MBwWnSxNWWV7QbOCgKNT\\\\\\\/GlLLVmPURXAqUzb5rhpwZsB914a70eU\\\\\\\/Pewu5WcN7GL9uIrxY0wAx0xnzxB11Hfub7jrHvIgaQ7pm7Ja04r\\\\\\\/pie2MFotPfCj8b2isvrGg4qnSm9n8TaM8x8zt0TANksBcoevQr8jZVNCzMoZaDFNjXs\\\\\\\/IrDt7S0GeqjEqPCLw+djY0tL5S3gbwUv7nu7enuDCjj75Foc97JlKjddm+zby7jYK0EurL4MgP3ud9SvcK471o5oduMJvax\\\\\\\/VMbDcpa6lnhid62DH2hla8Uv+JjnVpgqZryZPwRtxsKO9I5XptfBW8qVztVJ+GNU8c3zfHFvSEFd7ttDugCoB88yoZ8Ad1iNkz18yyr7lAreRay6MWyc0D4UsLCx71oXYRGIExyyYWLTVpS8a5j85fTzLONB4IS41+HenG3XLWRBHSZQlOabkNOrJmtlZoWUY1OqzFWI70wKjM9VjCNgwlwy2nbC3Rmz08xztx5GzbROXjRavCoyj5uP7gixjYorfVLOBbKSMQbC0eylr2E4DQKUWdQ4txpjvy9X4uUMhyGRQJSPpXownd56sViq49tZPJMoBeo1whxguleq+J+vCzDLesCMZ+8KwX\\\\\\\/aNDAy2RUsRK5ihsWdXRnHilNaLgUBEx9tZENZCgb0l59u3jZeh+a4MAw8\\\\\\\/fcijhbk\\\\\\\/Rq26tuocKHzV\\\\\\\/UclCzURc+hqCscrSQS8t6585nENauLJupEqh3J0cv\\\\\\\/JYGsJXl+5SvW\\\\\\\/YPlOaSWJgHjy+Dhin6nCFDa3JXBz3lPG\\\\\\\/2vTLIQR0Z3R5ORs5U\\\\\\\/cZzbenhfB2pAoeWWn5S7zmgAxBOnvBJMWAldo66xvSbVOY\\\\\\\/xkBEMBraxbLzkQqoOrWRLiD\\\\\\\/g9mNgufnVETUQCXIdI\\\\\\\/s8zr0\\\\\\\/Axmx\\\\\\\/mF+G32nqxiHsy9RQ95N71iDCXo2XMQeOZM70HgPXaBzIySHnosUjgBnxWUWuwAyDdkbhpwetrUOa1bSd6Vz9pgIWQe7COLgc94nfS0un4sLs0WWs3\\\\\\\/XXEjz+BDFICrZpkfqoXkWlU+rwHqGPxdJoPt2oZesQ5+GsTPB9TLmxDv3VaPj5+q9mXuv8FQeZRHNuwbImOCpaXH8uPpsMd1T1KnalrVI+lDlTO13g6aLU03gaD3pbw4l7HH3JOalPLorAAAJxMG8mBwglszNsOmb07vCqh1RgVUh\\\\\\\/VIYy\\\\\\\/XftwY16gGQlX4Szn2iemTTkQKe3+R0W9lVhxswdYFqiJ55eYVmL9cIY+VLdr9rnnA8Y8r6c53sTvhksw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022WuQ74abaOdm\\\\\\\/K2XDld\\\\\\\/muw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022e36547190ec2bd5e24aa8f82cc1da14c55cbe260c45df33a77591d1061b2d6a7\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022P94jbUGREk0iaY9ZTvXKaBvPqxy0t\\\\\\\/\\\\\\\/y8boZkYqalRQ2pRkEsXZGlhKLQlLY3vYbm8SPYhnRgOTCUf8oc7mPXiX1Lq3zuljvMvjUK6qjPFOoXcRj\\\\\\\/Isbxg8D\\\\\\\/XbxtOWTUTScJH2TVBkTY\\\\\\\/pXnnQMHWOMpoOq0gyq8+lLraLaDJuXdRgWxwvrXPgI25OuNqDgifFxvvMRbYIgMAqEzw5olUV+wbBGFKV0cigqzm5cpPYapx5VwI1VpMU7hxKrW2igP9u8\\\\\\\/ALbijNlxnwrRFeoYRxAB6n7T1XpZ5Pq\\\\\\\/rAhoehmboN5fXM05XcQ8ytsb02+xUEShPZnPF2RvbDOI0P7tTS7ga\\\\\\\/NkaMdaChvoiND94vx5C7P6CE56rM3CmqhSQQiO29iQePlOmY0igSChr0gomJdy9cw8UmfyrEsKWrbr+IPsuBYLR6+bPCgIgLg1FBT0b3eLzuJdcFNWtKyLqOfsagmV7G\\\\\\\/PXOokiL0dhScGWayIya+xZ7H8RzMFUZxyNPiG9Fvg5pfljkabCbwRc242p4JGv0EYReaDkkeGM3P4ZKwtjFOi7Js6yBliG3PBhh7BAqyMU7vHqc8pd9ilEMFtbK59y+v6fkTbhYJ3svU2cFx8+zS2anMXYRdqfYaMYcnukA\\\\\\\/vS\\\\\\\/9I9eaoTiXKrOpfYz36QX\\\\\\\/\\\\\\\/PsK2RzXPnHNkPAmmMuam0C+IZfBGinBxoh5om16LmRo7oil2MhunjiKhZbBsB6H9dMkI87sajr+jhCUjB8NlnwN6gYDEiOyNbC7baPoJZIszvx8X7Ih+HxAmUlZCE5R8783N3C9WkSbUbI6yPgbSFK9PY7OGNLWVE\\\\\\\/rFWUlaA8wnmeOpJdmtY\\\\\\\/ntRGyJ5PyrXEEC4Mqvc1QA1aOBJ6TrHbvOwtAoQoZ+T+n65cbymVbsrJ826sjk8BzdPMN0w0vvcBme\\\\\\\/UR8JKzYO3fn3hT90hMwqM1omOZrQl1bR0whhD\\\\\\\/Yq0h9kQiNH+COtcZGLFPjHshG9FIY3ibbQbxZIwV23YBh10HqSDw6+5iwlPtFVgo91cwsWwR84f37N0R+dZbYY3gw\\\\\\\/BFiM7fiS\\\\\\\/wnm\\\\\\\/MuoPOraGsu83Jo12LADA0LAkFgTe5LfWTQMP0jX8s9r015cH2fit9SQBNoWqMrOHjM9HxrONazFIdOM8V2YNR2gm+8YZCRyRDOhEjiQH09DPZdsebAZnXnB9xoSO6hf1dZtrJ4BR2PDiymAcSqz2za7Dzp8og0+AF2+f7HdiGZLeVH7dZFRgnVEmHYAeY70oGCRwXDkUYhnd\\\\\\\/8dZ0pUvnmGxtNy\\\\\\\/lrjoDNoyBK6zcne4DhEVCh0sCs9z7nsidPZQDdtLNPGTBMPt+zlmfZPNQVx9RFNLgwf8IJyqRp2aQAo7AL5wyhf4SSMMi8O5Bjuat41BbTH6rroV3ULdYJtdcProWAojr26eeUXx9ZZCH6g62vOKddrgyssz9iAbXSo7DfomUiPQt8jJdB4BDRbbZKL+r0HM2dgucBxcjMmr+kEIyBoXWozgCYqnMK6V0Z9xxqlPIz8sn9V+gSYHlz6NHzPOJHHmtAj4Ms8xcIhMs2ktAtdaT4R6t+IETk8E6IxCa3vmGmoNkJOPUh4WysZfEepNyPlYmWvVBVqHCNWQaNSZjj6jMQ5Defs3n1TXeevXnNgmNmqfCF95tG5Fg2uDoEdT3O9Zn7Wp41f19bcsMnbkqpxUVuBoSNpgKcdFvuKPvLGSmUHn8\\\\\\\/RF93zhlRmedhJA1QUJn+Qh5iMn2xGNJ09VCB\\\\\\\/DHAfrFIDC1xNc7WUWvZjhoLYj4l2YZ8Ay3tex6mMN+RToojJZPkw8fZ\\\\\\\/1bfDhYSW\\\\\\\/08q2AZovn4W87\\\\\\\/8rC8QL01qNHo5lZlJ7ZiHKTXB8nsAIb7+AHO4NPXxF3TUfn\\\\\\\/32nj0OUMmkt+bAKATOpScGjzH7HEihxplUyoKYEwC2sbvatS2UurhwfHWv6BvuBSaD89fKjiUpcYkoJmCcUf9w4tieqi2L0LcdvTzAiWOjHRlZNSinH3OWF4A==\\\u0022,\\\u0022iv\\\u0022:\\\u0022vON1P3CS23NAI5O\\\\\\\/2WpgGA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002276109f69d2727a6f18b26d3e0f2ad42a843a18ce43e85b971eea059314f2f9b3\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459328 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-genomic-selection">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/2/9/1085" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-WNnQSJs_vONBdi-Wl6wuV0r-yXQNNHlxWkbMmWzmpCI" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/2/9/1085" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-dJjC6Pai8ZqOhtYlhvcz74FdGB3W6F8L1w8hLDIcl90" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/2/9/1129" title="Hog1 Controls Global Reallocation of RNA Pol II upon Osmotic Shock in Saccharomyces cerevisiae" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/2/10/1137" title="Competition Between Conjugation and M13 Phage Infection in Escherichia coli in the Absence of Selection Pressure: A Kinetic Study" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459328" id="node459328" data-pisa="ggg;2/9/1085" data-pisa-master="ggg;2/9/1085" data-apath="/ggg/2/9/1085.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Predicting Adaptive Phenotypes From Multilocus Genotypes in Sitka Spruce (<em>Picea sitchensis</em>) Using Random Forest</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Jason A. Holliday</span>, <span class="highwire-citation-author" data-delta="1">Tongli Wang</span> and <span class="highwire-citation-author" data-delta="2">Sally Aitken</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2012 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 2 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1085-1093;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.002733 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459328" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jason A. Holliday </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Forest Resources and Environmental Conservation, Virginia Polytechnic Institute and State University, Blacksburg, Virginia 24061, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jason%2BA.%2BHolliday%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Holliday%20JA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJason%2BA.%2BHolliday%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jah1@vt.edu" class="" data-icon-position="" data-hide-link-title="0">jah1@vt.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Tongli Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Forest Sciences, University of British Columbia, Vancouver, BC, Canada, V6T 1Z4</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tongli%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATongli%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Sally Aitken </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Forest Sciences, University of British Columbia, Vancouver, BC, Canada, V6T 1Z4</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sally%2BAitken%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Aitken%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASally%2BAitken%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/2/9/1085" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459328" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459328/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/2/9/1085.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459328" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459328/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/2/9/1085.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459328" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459328/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/2/9/1085.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459328" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459328/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/2/9/1085.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459328" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459328/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-4">Climate is the primary driver of the distribution of tree species worldwide, and the potential for adaptive evolution will be an important factor determining the response of forests to anthropogenic climate change. Although association mapping has the potential to improve our understanding of the genomic underpinnings of climatically relevant traits, the utility of adaptive polymorphisms uncovered by such studies would be greatly enhanced by the development of integrated models that account for the phenotypic effects of multiple single-nucleotide polymorphisms (SNPs) and their interactions simultaneously. We previously reported the results of association mapping in the widespread conifer Sitka spruce (<em>Picea sitchensis</em>). In the current study we used the recursive partitioning algorithm ‘Random Forest’ to identify optimized combinations of SNPs to predict adaptive phenotypes. After adjusting for population structure, we were able to explain 37% and 30% of the phenotypic variation, respectively, in two locally adaptive traits—autumn budset timing and cold hardiness. For each trait, the leading five SNPs captured much of the phenotypic variation. To determine the role of epistasis in shaping these phenotypes, we also used a novel approach to quantify the strength and direction of pairwise interactions between SNPs and found such interactions to be common. Our results demonstrate the power of Random Forest to identify subsets of markers that are most important to climatic adaptation, and suggest that interactions among these loci may be widespread.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/random-forest" class="hw-term hw-article-keyword hw-article-keyword-random-forest" rel="nofollow">Random Forest</a></li><li class="kwd"><a href="/keyword/adaptation" class="hw-term hw-article-keyword hw-article-keyword-adaptation" rel="nofollow">adaptation</a></li><li class="kwd"><a href="/keyword/association-mapping" class="hw-term hw-article-keyword hw-article-keyword-association-mapping" rel="nofollow">association mapping</a></li><li class="kwd"><a href="/keyword/epistasis" class="hw-term hw-article-keyword hw-article-keyword-epistasis" rel="nofollow">epistasis</a></li><li class="kwd"><a href="/keyword/phenology" class="hw-term hw-article-keyword hw-article-keyword-phenology" rel="nofollow">phenology</a></li><li class="kwd"><a href="/keyword/cold-hardiness" class="hw-term hw-article-keyword hw-article-keyword-cold-hardiness" rel="nofollow">cold hardiness</a></li><li class="kwd"><a href="/keyword/genpred" class="hw-term hw-article-keyword hw-article-keyword-genpred" rel="nofollow">GenPred</a></li><li class="kwd"><a href="/keyword/shared-data-resources" class="hw-term hw-article-keyword hw-article-keyword-shared-data-resources" rel="nofollow">shared data resources</a></li></ul><p id="p-5">The health of forest tree populations is inextricably linked with the ability of local populations to track phenotypic optimums enforced by their respective climates. With anthropogenic climate change substantially altering adaptive landscapes, tree populations are expected to suffer maladaptation, become sources rather than sinks of carbon, and ultimately be forced to adapt, migrate, or be extirpated (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Savolainen <em>et al.</em> 2007</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Aitken <em>et al.</em> 2008</a>). Paleoecological data suggest that migration rates may be insufficient to realize range shifts predicted by climate-based species distribution models (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Malcolm <em>et al.</em> 2002</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">McLachlan <em>et al.</em> 2005</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Hamann and Wang 2006</a>), and thus, the importance of adaptive evolution in response to new climatic regimes cannot be underestimated. To determine the potential for adaptation, we must first have an understanding of the genomic underpinnings of variation in climate-related phenotypic traits.</p><p id="p-6">The current method of choice for elucidating the genomic determinants of complex adaptive traits is association mapping (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Neale and Savolainen 2004</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Gonzalez-Martinez <em>et al.</em> 2006b</a>). Widespread temperate and boreal tree species are well suited to this approach, which has been successfully demonstrated in a variety of species (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Gonzalez-Martinez <em>et al.</em> 2007</a>, <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">2008</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Ingvarsson <em>et al.</em> 2008</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Eckert <em>et al.</em> 2009a</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Holliday <em>et al.</em> 2010</a>; <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Wegrzyn <em>et al.</em> 2010</a>). Conventional analytical approaches to association mapping studies estimate the marginal effect of individual polymorphisms on the trait of interest (<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Yu <em>et al.</em> 2006</a>), but these approaches are not designed to predict phenotypes from multilocus genotypes nor to detect interactions among loci. There has been increasing interest in the use of multiple regression-like models that take all single-nucleotide polymorphisms (SNPs) as predictor variables simultaneously. This approach is most frequently applied to mapping populations generated by controlled crosses for the purposes of selecting germplasm for deployment or further crosses in breeding programs—so-called “genomic selection.” The goal of these methods is not so much to identify trait-associated SNPs as it is to saturate the genome with markers such that most quantitative trait loci will be captured through linkage disequilibrium (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Meuwissen <em>et al.</em> 2001</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Jannink <em>et al.</em> 2010</a>). The extension of this approach to trees has been suggested and the first test cases recently reported (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Grattapaglia and Resende 2011</a>; <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Resende <em>et al.</em> 2012</a>)</p><p id="p-7">Although genomic selection has shown great promise in the fields of animal and plant breeding, it is not currently feasible as a tool for conservation genetics in natural populations because of the rapid decay of linkage disequilibrium (LD), which would necessitate hundreds of thousands to millions of markers be genotyped. Ideally, one would be able to identify a small core set of ecologically relevant markers that comprise much of the adaptive variation segregating in the wild, which could then be targeted by land managers to assess the adaptive portfolio of local populations. The decision tree algorithm ‘Random Forest’ (RF) (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Breiman 2001</a>) is an alternative method to uncover the relative importance of SNPs to the expression of adaptive traits and to predict the phenotypes of adaptive traits by accounting for both the cumulative effect of individual SNPs and the effect of all forms of interactions among SNPs without the need to define these terms in the model. The principle behind RF is to build decision trees (or regression trees in the case of continuous dependent variables) by categorizing observations using multiple predictor variables. In contrast to conventional decision and regression tree approaches, RF constructs a large number of trees (the ‘forest’) by introducing two layers of randomness—random bootstrap sampling of the data and random selection of a subset of predictors for splitting at each node—which improves predictive accuracy (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Cordell 2009</a>). The relative importance of individual SNPs is assessed by permutation: SNPs with no effect on phenotypes will not change the prediction of the tree when their values are permuted, whereas SNPs with explanatory power will. Several studies have used RF to detect loci involved in binary disease phenotypes in humans (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bureau <em>et al.</em> 2005</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Goldstein <em>et al.</em> 2010</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Chen <em>et al.</em> 2011</a>;), and one study extended the RF approach to quantitative variation in resistance to powdery mildew in <em>Arabidopsis</em> (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Nemri <em>et al.</em> 2010</a>).</p><p id="p-8">In addition to accounting for multilocus combined effects, RF can be used to test for interactions (<em>i.e.</em>, epistasis) among SNPs. Epistasis is a fundamental component of complex trait variation that may be an important driver of adaptation, but such effects are difficult to quantify because of the computational burden of testing all possible pairwise interactions for hundreds or thousands of loci (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Phillips 2008</a>). In the context of association mapping, several methods have been developed to detect interacting SNPs (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Hahn <em>et al.</em> 2003</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Lunn <em>et al.</em> 2006</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Moore <em>et al.</em> 2006</a>; <a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Zhang and Liu 2007</a>), but these methods are still computationally intensive (and in some cases intractable) for datasets comprising more than a few hundred SNPs (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Cordell 2009</a>). By contrast, RF offers an efficient framework to test for epistasis, which does not require that all possible pairwise interactions be specified in the model. Any predictor SNP in an RF model that has a significant interaction, even in the absence of a main effect, should be assigned high importance because the phenotype prediction will change when its values are permuted. <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Lunetta <em>et al.</em> (2004)</a> simulated interacting disease risk-associated SNPs and found that selecting SNPs on the basis of their importance value from an RF analysis lead to improved detection of epistasis. <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Jiang <em>et al.</em> (2009)</a> applied a sliding-window approach with RF to reduce the number SNPs tested for interactions in a case-control binary trait study involving a huge number of SNPs. However, to the best of our knowledge no report has been made that tested SNP interactions for quantitative traits using RF. In particular, no method is available to detect the directions of such interactions between each SNP pair.</p><p id="p-9">We recently completed an association study in Sitka spruce encompassing 202 candidate genes related to local adaptation, and identified numerous genotype-phenotype associations (<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Holliday <em>et al.</em> 2010</a>). Among 339 SNPs tested, 35 were significantly associated with either timing of budset or cold hardiness. The current paper uses the RF framework to analyze the same dataset by building an integrated model to predict phenotypes on the basis of their multilocus genotypes, to select a subset of SNPs with the greatest predictive power, and uses a novel approach to detect the extent and direction of epistasis among these predictive SNPs.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><p id="p-10">We previously reported association mapping of climate-related traits in Sitka spruce (<em>Picea sitchensis</em>) (<a id="xref-ref-22-3" class="xref-bibr" href="#ref-22">Holliday <em>et al.</em> 2010</a>). In this study, the Illumina GoldenGate platform was used to genotype a range-wide mapping population of 410 individuals, which were also measured for timing of budset (defined as the Julian date on which the first bud scales were visible, assessed weekly from July 7 through December 31, 2007) and cold hardiness (measured on December 1, 2007, using electrolyte leakage as a proxy for cell death after <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Hannerz <em>et al.</em> (1999)</a>). Sampled populations, their origins, and cluster assignments are described in supplementary materials (supporting information, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.002733/-/DC1/TableS1.xlsx">Table S1</a>). Substantial variation both within and among populations was observed for both of these traits. By using a mixed model approach that accounts for population structure, we found 35 SNP loci (from a panel of 339 successful SNP assays) to be associated with timing of autumn budset, autumn cold hardiness, or both.</p><p id="p-11">We used the data from <a id="xref-ref-22-4" class="xref-bibr" href="#ref-22">Holliday <em>et al.</em> (2010)</a> to build a predictive model for adaptive traits, for which dependent variables consisted of the two phenotypes described previously, and independent variables consisted of 339 SNPs that passed quality control and were polymorphic. Candidate gene names and genotype/phenotype data can be found in supplementary materials (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.002733/-/DC1/TableS2.xlsx">Table S2</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.002733/-/DC1/TableS3.xlsx">Table S3</a>, respectively). The procedure to build the random forest is as follows: (1) take a bootstrap sample of observations (<em>i.e.</em>, a random sample with replacement; ∼64% by default) from the total population, leaving the rest ‘out of bag’ (OOB) to be used for independent predictions and error checking; (2) at each successive node randomly sample a subset of SNPs (approximately the square-root of the total by default, but tuned for optimal values), and use the SNP that minimizes predictive error to split observations at that node into ‘branches’; (3) continue this process until there is no further improvement in predictive accuracy of terminal ‘leaves’; (4) repeat the aforementioned steps to build a large number of regression trees forming the random forest; (5) predict the phenotypes for the OOB samples through the trees for which they were OOB and take the average of the predictions across trees as the final phenotype prediction for a given sample. The importance value reflecting the relative contribution of each SNP to the model is estimated by randomly permuting its values and recalculating the predictive accuracy of the model. The difference of the model accuracy before and after the random permutations, averaged over all trees in the forest, tells us how important that predictor is for determining the outcome. The amount of variance explained by the predicted values relative to the total amount of variation in the observed phenotypes provides an estimate of the explanatory power of the model.</p><p id="p-12">Because population structure can increase type-one error rates (false-positives) in association studies, model predictions are more credible with population structure removed. <a id="xref-ref-22-5" class="xref-bibr" href="#ref-22">Holliday <em>et al.</em> (2010)</a> used <em>Structure</em> software (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Pritchard <em>et al.</em> 2000</a>) to show that three populations provides the most parsimonious solution to population subdivision across the species range in Sitka spruce. To remove the effect of population structure, we regressed the two phenotypic traits on estimates of cluster membership derived from replicate runs of <em>Structure</em> (that is, the Q matrix from <em>Structure</em>). The sums of the general mean and the residue of each observation from this regression were used as dependent variable for RF analysis. However, as Sitka spruce spans a wide latitudinal range, with latitude highly correlated with climatic variables, but is restricted to a narrow longitudinal range, population structure covaries strongly with climate gradients. The removal of the population structure may thus result in overadjustment (false-negatives) for the effect of local adaptation to climate. Therefore, phenotype prediction was conducted both with and without an adjustment for population structure.</p><p id="p-13">For this analysis, 1500 regression trees were constructed in the model, although no further improvement was found after the number of trees increased to 1000. The model was initially run with all 339 SNPs as predictors (the full model), followed by runs using various numbers of selected SNPs based on their relative importance values to optimize the model. In previous studies, the importance value of each SNP assigned by the full model was used to rank the SNPs. However, as RF accounts for interactions, the ranking changes when the combination of the SNPs changes (<em>i.e.</em>, adding or dropping one or more SNPs to the model). To consider the effect of interactions among the SNPs on phenotype prediction accuracy and on the ranking of their importance, we developed a backward purging approach to purge the least important SNP step by step. The minimum number of SNPs that can explain the maximum amount of variance in observed phenotypes was around 20. We thus applied this approach starting with a highly bifurcating model with the top-ranked 50 SNPs included to avoid excluding those SNPs with large interactions but insignificant main effects. At each step, the model run was repeated three times, and the SNP with the smallest importance value was eliminated from the model. The amount of variance explained by each successive model was averaged over the three runs. This step was repeated until there were only two SNPs remaining in the model (the minimum number of predictor variables that can be used).</p><p id="p-14">To validate the effectiveness of our approach in selection of most important SNPs for the adaptive traits, we compared the effect of our selected SNPs on prediction of phenotypes with randomly selected SNPs. For this, RF model was run with bootstrap samples of SNPs (<em>i.e.</em> random combinations) with sample sizes of 2, 5, 10, 15, and 20 SNPs. These were referred to as ‘random’ models. The random sampling and model runs were repeated 10 times with each sample size, from which the average percentages of the amount of variance explained were calculated for each sample size. All RF analyses were performed using the R version of RF (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Liaw and Wiener 2002</a>).</p><p id="p-15">Although RF accounts for epistasis without directly testing all possible pairwise combinations of loci, which is computationally efficient, it does not identify specific interacting SNPs. We have extended the method to enable direct testing of putatively interacting SNPs by removing SNPs with high importance values one-by-one and recalculating the importance values of remaining SNPs. If two SNPs interact, the absence of one of them would increase or decrease the importance values of the remaining SNP. Therefore, we can determine the amount and the direction (negative or positive) of the two-way interactions among important SNPs through examining the changes in their importance values when a SNP is removed from the model. We did this for each of the 20 most important SNPs to quantify both synergistic epistasis (<em>i.e.</em>, when the importance of a SNP decreases upon removal of another SNP) and disruptive epistasis (<em>i.e.</em>, when the importance value of a SNP increases upon removal of another SNP). This procedure was repeated five times, and the changes in the importance values were calculated for each locus in each run. The student's <em>T</em>-tests were applied to determine whether the changes in the importance values were significant. The <em>P</em>-values for the tests were adjusted for multiple testing using the <em>qvalue</em> package in R [<a href="http://www.r-project.org/">http://www.r-project.org/</a> (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Storey and Tibshirani 2003</a>]. Tests quantifying interactions among SNPs were carried out only for the dataset with population structure removed. Interactions networks were displayed using Cytoscape software (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Shannon <em>et al.</em> 2003</a>).</p><p id="p-16">To determine whether interacting SNPs were also in LD, we used our unphased genotypic data to calculate the probability that genotypes at pairs of loci are independent by using the composite LD test implemented in Genepop software (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Rousset 2008</a>). <em>P</em>-values obtained from this test were adjusted for the false-discovery rate (FDR).</p></div><div class="section results" id="sec-2"><h2 class="">Results</h2><div id="sec-3" class="subsection"><h3>Phenotype prediction</h3><p id="p-17">When all 339 SNPs were used as predictors, RF explained 32.7% of the phenotypic variation in budset timing (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>). When budset dates were adjusted for population structure using multiple regressions before running RF, this value decreased to 24% (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1B</a>). The predictive value of the model improved when only the 20 most important SNPs were used as predictors, which were selected using the backward purging approach described in the <em>Materials and Methods</em> (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1, C and D</a>). The percentage of phenotypic variance explained (PVE) increased to 44.6% and 37.2% for the population unadjusted and adjusted models, respectively. Accounting for population structure had a more dramatic effect on prediction of cold injury, for which the unadjusted model accounted for 41.7% of variation, whereas the adjusted model accounted for only 22.5% (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2, A and B</a>). When only the top 20 SNPs were again selected, PVE was improved by 8.4% and 7.6% for the uncorrected and corrected models, respectively (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2, C</a> and <a id="xref-fig-2-3" class="xref-fig" href="#F2">2D</a>). These results are consistent with an increase in noise contributed by additional uninformative SNPs, which is reflected in negative importance values for a substantial fraction of the 339 SNPs for both budset (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>) and cold hardiness (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3B</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/9/1085/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Relationship between observed budset dates (given as days from January 1) and those predicted by RF using all 339 SNPs (A) without adjusting for population structure and (B) with adjustment. The 20 SNPs with highest importance [% mean squared error (MSE)] were then used to similarly re-run RF both without adjusting for population structure (C) and with adjustment for population structure (D). All regressions were significant (P &lt; 0.001)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1190993121" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Relationship between observed budset dates (given as days from January 1) and those predicted by RF using all 339 SNPs (A) without adjusting for population structure and (B) with adjustment. The 20 SNPs with highest importance [% mean squared error (MSE)] were then used to similarly re-run RF both without adjusting for population structure (C) and with adjustment for population structure (D). All regressions were significant (&lt;em&gt;P&lt;/em&gt; &lt; 0.001).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/9/1085/F1.medium.gif" width="440" height="404"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1&#x2002;" src="https://www.g3journal.org/content/ggg/2/9/1085/F1.medium.gif" width="440" height="404"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/9/1085/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/9/1085/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461681" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1 </span> <p id="p-18" class="first-child">Relationship between observed budset dates (given as days from January 1) and those predicted by RF using all 339 SNPs (A) without adjusting for population structure and (B) with adjustment. The 20 SNPs with highest importance [% mean squared error (MSE)] were then used to similarly re-run RF both without adjusting for population structure (C) and with adjustment for population structure (D). All regressions were significant (<em>P</em> &lt; 0.001).</p><div class="sb-div caption-clear"></div></div></div><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/9/1085/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Relationship between observed cold injury (averaged across three test temperatures) and that predicted by RF using all 339 SNPs (A) without adjusting for population structure and (B) with adjustment. The 20 SNPs with highest importance (% MSE) were then used to similarly re-run RF both without adjusting for population structure (C) and with adjustment for population structure (D). All regressions were significant (P &lt; 0.001)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1190993121" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Relationship between observed cold injury (averaged across three test temperatures) and that predicted by RF using all 339 SNPs (A) without adjusting for population structure and (B) with adjustment. The 20 SNPs with highest importance (% MSE) were then used to similarly re-run RF both without adjusting for population structure (C) and with adjustment for population structure (D). All regressions were significant (&lt;em&gt;P&lt;/em&gt; &lt; 0.001).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/9/1085/F2.medium.gif" width="440" height="409"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2&#x2002;" src="https://www.g3journal.org/content/ggg/2/9/1085/F2.medium.gif" width="440" height="409"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/9/1085/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/9/1085/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461862" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2 </span> <p id="p-19" class="first-child">Relationship between observed cold injury (averaged across three test temperatures) and that predicted by RF using all 339 SNPs (A) without adjusting for population structure and (B) with adjustment. The 20 SNPs with highest importance (% MSE) were then used to similarly re-run RF both without adjusting for population structure (C) and with adjustment for population structure (D). All regressions were significant (<em>P</em> &lt; 0.001).</p><div class="sb-div caption-clear"></div></div></div><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/9/1085/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Importance values for all 339 SNPs in the study (vertical lines) for (A) budset date and (B) cold hardiness, adjusted for population structure." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1190993121" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Importance values for all 339 SNPs in the study (vertical lines) for (A) budset date and (B) cold hardiness, adjusted for population structure.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/9/1085/F3.medium.gif" width="440" height="177"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3&#x2002;" src="https://www.g3journal.org/content/ggg/2/9/1085/F3.medium.gif" width="440" height="177"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/9/1085/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/9/1085/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461739" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3 </span> <p id="p-20" class="first-child">Importance values for all 339 SNPs in the study (vertical lines) for (A) budset date and (B) cold hardiness, adjusted for population structure.</p><div class="sb-div caption-clear"></div></div></div><p id="p-21">The result of the backward purging approach suggests that for both budset and cold hardiness, the first three to five SNPs account for much of the variation, as PVE tended to reach a plateau with only incremental increases with the inclusion of additional SNPs (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4, A and B</a>, respectively). In contrast, randomly selected SNPs collectively had much less explanatory power (less than 5% in most cases) than the optimized models (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>), a result consistent regardless of the number of SNPs from 2 to 20 included in the model.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/9/1085/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Percentages of total variance explained in (A) bud set and (B) observed cold injury with the number of SNPs included in the RF models. SNPs were selected using optimized combinations of between two and 20 SNPs from all 339 SNPs (Top 20) and using bootstrapped random samples (Random)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1190993121" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Percentages of total variance explained in (A) bud set and (B) observed cold injury with the number of SNPs included in the RF models. SNPs were selected using optimized combinations of between two and 20 SNPs from all 339 SNPs (Top 20) and using bootstrapped random samples (Random).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/9/1085/F4.medium.gif" width="440" height="206"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4&#x2002;" src="https://www.g3journal.org/content/ggg/2/9/1085/F4.medium.gif" width="440" height="206"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/9/1085/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/9/1085/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461757" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4 </span> <p id="p-22" class="first-child">Percentages of total variance explained in (A) bud set and (B) observed cold injury with the number of SNPs included in the RF models. SNPs were selected using optimized combinations of between two and 20 SNPs from all 339 SNPs (Top 20) and using bootstrapped random samples (Random).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-4" class="subsection"><h3>Importance values for individual SNPs</h3><p id="p-23">Adjusting for population structure did not change the identity of the top two SNPs for budset, whereas for cold injury the top SNP before the adjustment decreased to fourth most important following population adjustment (data not shown). For budset, seven of the top 10 SNPs remained within the top 10 after adjustment for population structure, and all of the top 10 SNPs after adjustment were among the top 15 SNPs in the unadjusted model. For cold hardiness, eight of the top 10 SNPs were in common between the adjusted and unadjusted models, although the ranking changed in a few cases. The 10th most important SNP in the population adjusted model increased in importance from position 24 in the unadjusted model. Of the 339 predictor SNPs, 114 had negative importance values for the budset model (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3A</a>) and 105 had negative values for the cold hardiness model (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3B</a>).</p><p id="p-24">Although RF and the mixed linear model (MLM), previously used by <a id="xref-ref-22-6" class="xref-bibr" href="#ref-22">Holliday <em>et al.</em> (2010)</a> to identify genotype-phenotype associations, assessed the explanatory power of our SNP markers in different ways, the concordance between the two methods was high for the SNPs with strongest associations with phenotype. This concordance broke down, however, beyond the top 10 SNPs. For budset timing, after adjustment for population structure, eight of the top 10 SNPs identified using RF were also significant using the MLM, whereas only two of the next 10 were in concordance with the MLM (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5A</a>). For cold hardiness, six of the top 10 SNPs and two of the next 10 from the RF analysis were also significant using the MLM (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5B</a>).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/9/1085/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Importance values for top 20 SNPs for budset (A) and cold hardiness (B) identified by the initial run of RF that included all 339 SNPs. SNPs highlighted in dark gray had significant main effects when the mixed model (after Holliday et al. 2010) was used, and those highlighted in light gray were only significant using RF." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1190993121" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Importance values for top 20 SNPs for budset (A) and cold hardiness (B) identified by the initial run of RF that included all 339 SNPs. SNPs highlighted in dark gray had significant main effects when the mixed model (&lt;em&gt;after&lt;/em&gt; Holliday &lt;em&gt;et al.&lt;/em&gt; 2010) was used, and those highlighted in light gray were only significant using RF.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/9/1085/F5.medium.gif" width="440" height="223"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5&#x2002;" src="https://www.g3journal.org/content/ggg/2/9/1085/F5.medium.gif" width="440" height="223"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/9/1085/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/9/1085/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461793" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5 </span> <p id="p-25" class="first-child">Importance values for top 20 SNPs for budset (A) and cold hardiness (B) identified by the initial run of RF that included all 339 SNPs. SNPs highlighted in dark gray had significant main effects when the mixed model (<em>after</em> <a id="xref-ref-22-7" class="xref-bibr" href="#ref-22">Holliday <em>et al.</em> 2010</a>) was used, and those highlighted in light gray were only significant using RF.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-5" class="subsection"><h3>Tests for epistasis</h3><p id="p-26">By sequentially removing SNPs from the model and recalculating importance values for those that remained, we were able to quantify cases of both synergistic and antagonistic epistasis. We conducted this analysis using the top 20 SNPs based on importance values from the initial RF analysis with adjustment for population structure and followed by a backward purging approach described previously, and the resulting epistasis networks were plotted using Cytoscape (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figures 6</a> and <a id="xref-fig-7-1" class="xref-fig" href="#F7">7</a>) (<a id="xref-ref-56-2" class="xref-bibr" href="#ref-56">Shannon <em>et al.</em> 2003</a>). Numerous statistically significant interactions were identified among the top SNPs for each trait. A larger number of pairwise interactions were found for budset (45 interactions) than for cold hardiness (27 interactions). Interestingly, the SNPs with the highest importance values for both traits also were among the most highly connected in its respective network. For budset, <em>per6</em> (573) had eight interactions, whereas for cold hardiness, <em>xth1</em> (350) had five interactions. However, some SNPs with relatively small contributions to the model also exhibited high connectivity, including <em>gp5</em> (98), which for cold hardiness had eight interactions. Conversely, the second most important SNP for budset had only one interaction.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/9/1085/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Cytoscape interaction network showing significant (Q &lt; 0.05) pairwise epistatic interactions for budset among the top 20 SNPs measured by their importance values. Line width indicates the relative strength of the interaction; dashed lines indicate negative (antagonistic) epistasis, and solid lines indicate positive (synergistic) epistasis. Asterisks indicate significant genotypic linkage disequilibrium (Q &lt; 0.05) between interacting SNPs." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1190993121" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Cytoscape interaction network showing significant (Q &lt; 0.05) pairwise epistatic interactions for budset among the top 20 SNPs measured by their importance values. Line width indicates the relative strength of the interaction; dashed lines indicate negative (antagonistic) epistasis, and solid lines indicate positive (synergistic) epistasis. Asterisks indicate significant genotypic linkage disequilibrium (Q &lt; 0.05) between interacting SNPs.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/9/1085/F6.medium.gif" width="440" height="256"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6&#x2002;" src="https://www.g3journal.org/content/ggg/2/9/1085/F6.medium.gif" width="440" height="256"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/9/1085/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/9/1085/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461765" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6 </span> <p id="p-27" class="first-child">Cytoscape interaction network showing significant (Q &lt; 0.05) pairwise epistatic interactions for budset among the top 20 SNPs measured by their importance values. Line width indicates the relative strength of the interaction; dashed lines indicate negative (antagonistic) epistasis, and solid lines indicate positive (synergistic) epistasis. Asterisks indicate significant genotypic linkage disequilibrium (Q &lt; 0.05) between interacting SNPs.</p><div class="sb-div caption-clear"></div></div></div><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/9/1085/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Cytoscape interaction network showing significant (Q &lt; 0.05) pairwise epistatic interactions for cold hardiness among the top 20 SNPs measured by their importance values. Line width indicates the relative strength of the interaction; dashed lines indicate negative (antagonistic) epistasis, and solid lines indicate positive (synergistic) epistasis. Asterisks indicate significant genotypic linkage disequilibrium (Q &lt; 0.05) between interacting SNPs." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1190993121" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Cytoscape interaction network showing significant (Q &lt; 0.05) pairwise epistatic interactions for cold hardiness among the top 20 SNPs measured by their importance values. Line width indicates the relative strength of the interaction; dashed lines indicate negative (antagonistic) epistasis, and solid lines indicate positive (synergistic) epistasis. Asterisks indicate significant genotypic linkage disequilibrium (Q &lt; 0.05) between interacting SNPs.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/9/1085/F7.medium.gif" width="382" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7&#x2002;" src="https://www.g3journal.org/content/ggg/2/9/1085/F7.medium.gif" width="382" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/9/1085/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/9/1085/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461832" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7 </span> <p id="p-28" class="first-child">Cytoscape interaction network showing significant (Q &lt; 0.05) pairwise epistatic interactions for cold hardiness among the top 20 SNPs measured by their importance values. Line width indicates the relative strength of the interaction; dashed lines indicate negative (antagonistic) epistasis, and solid lines indicate positive (synergistic) epistasis. Asterisks indicate significant genotypic linkage disequilibrium (Q &lt; 0.05) between interacting SNPs.</p><div class="sb-div caption-clear"></div></div></div><p id="p-29">Finally, we calculated genotypic linkage disequilibrium for each pair of interacting SNPs. For budset, 10 of 45 SNP pairs that exhibited significant epistatic interactions were also in linkage disequilibrium (Q &lt; 0.05), and for cold hardiness, 13 of 27 interacting SNPs were in LD. Most interacting SNPs were found on different genes, and significant LD was therefore not likely due tight physical linkage in most cases, though our markers were not genetically mapped and we therefore do not know the physical proximity of the candidate genes. However, given the size of the spruce genome (∼18 Gb) (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Ahuja and Neale 2005</a>), it is likely that most interacting SNPs are separated by large physical distances or reside on different chromosomes.</p></div></div><div class="section discussion" id="sec-6"><h2 class="">Discussion</h2><p id="p-30">We used RF to identify subsets of SNPs that provide the greatest power to predict adaptive phenotypes in the conifer Sitka spruce. This integrated approach revealed that only a few SNPs are needed to capture a large portion of the quantitative variation in adaptive traits, a finding is somewhat surprising because the conventional analysis of variance approach usually identifies a large number of SNPs that contribute relatively small amounts of variation to the phenotype (Eckert <em>et al.</em> 2009a, <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">2012</a>; <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Quesada <em>et al.</em> 2010</a>; <a id="xref-ref-60-2" class="xref-bibr" href="#ref-60">Wegrzyn <em>et al.</em> 2010</a>). A possible explanation for this difference would be the inclusion of interactions in the RF model used in this study. Specifically, with our backward purging approach, higher-order interactions are considered in reducing the number of SNPs in the model, and direct tests for epistasis showed that it is rampant among these SNPs. The role of interactions in predicting phenotypes has increasingly been recognized (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Malmberg <em>et al.</em> 2005</a>; <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">von Korff <em>et al.</em> 2010</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Pavlicev <em>et al.</em> 2011</a>) although there are a paucity of data in plants. One study in Arabidopsis showed that the magnitude of epistatic effects on several fitness related traits was roughly double that of the effects of the additive QTL (<a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Malmberg <em>et al.</em> 2005</a>). Our results advance an empirical understanding of how populations may adapt to local climate across many loci and should facilitate the screening of climatically relevant adaptive variation in natural, breeding and deployment populations, and to developing species-specific genetic resource management and conservation strategies.</p><div id="sec-7" class="subsection"><h3>A small number of SNPs are sufficient to explain much of the variation in adaptive traits in Sitka spruce</h3><p id="p-31">Many temperate and boreal plant species have expanded from glacial refugia to occupy wide latitudinal ranges in a relatively short period of time, on the order of 10−15,000 years. Such expansions necessitate ongoing adaptation to novel climates, which for annual plants is not difficult to envision due to their short generation time. However, long-lived tree species present somewhat of an adaptive paradox in that they have a delay until reproductive maturity resulting in a generation time on the order of 20 to 50 years or longer, depending on ecological disturbance or harvesting rotations (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Petit and Hampe 2006</a>). This presents two impediments to local adaptation, namely, a lower effective mutation rate and fewer opportunities for allele frequency adjustments. How then have long-lived tree species been able to expand their ranges so rapidly while simultaneously tracking relative phenotypic optima so closely? Our results suggest that although the average effect of individual SNPs on trait variance is small, the genetic architecture of these traits is clearly finite. Of course, this study only accounts for a small fraction of the segregating variation present in the genome of spruce, and future work that capitalizes on next generation sequencing technologies will no doubt improve on the model we developed here.</p><p id="p-32">It should be noted that the variation we explain with this sample, as is the case in many genotype−phenotype association studies of natural tree species, comprises both within and among-population components. Hence, the relevant measure of quantitative genetic variance with which to compare our model is Q<sub>ST</sub> (the proportion of phenotypic variation attributable to differentiation among populations), which in Sitka spruce for the same populations was reported to be ∼0.89 for both cold hardiness and budset (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Mimura and Aitken 2007</a>). Our RF model therefore explains on the order of a third to a half of the variation present in the studied collection of populations. Adjusting for population structure reduced the explanatory power of the SNPs. Although it was necessary to make this adjustment to avoid false positives, Sitka spruce has a very wide latitudinal range but is essentially restricted to coastal habitat, and population structure in this species covaries with selective pressures related to climate. As such, removing the effect of population structure inevitably removes some of the SNP effects of interest, and hence reduces explanatory power (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Myles <em>et al.</em> 2009</a>). Taken together, our results would seem to support modern refinements of Fisher’s geometric model of adaptation, which emphasizes the importance of a few mutations of moderate effect and many mutations of small effect (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Orr 1998</a>, <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">1999</a>). We have previously shown that SNPs with phenotypic associations in this study segregate in most populations throughout the range of Sitka spruce, including populations well south of the maximum extent of glaciation, and they are therefore likely to be evolutionarily old (<a id="xref-ref-22-8" class="xref-bibr" href="#ref-22">Holliday <em>et al.</em> 2010</a>). Adaptation to anthropogenic climate change in this species may therefore proceed largely from standing variants, and our results suggest this adaptation may involve fewer loci than is often assumed by the infinitesimal model underlying classical quantitative genetics (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Barrett and Schluter 2008</a>). If this is the case, allele frequency adjustments within the loci described here, coupled with realistic migration rates and epistasis, may provide the means for tree populations to track the rapidly changing phenotypic optimums imposed by climate change.</p></div><div id="sec-8" class="subsection"><h3>Epistasis shapes adaptive traits in Sitka spruce</h3><p id="p-33">Phenotypes are the product of both genotype and environment according to the central dogma of quantitative genetics, and it is the goal of association genetics to localize genotypic effects while holding environment constant, as in a common garden. These genotypic effects can be further broken down into additive, dominance, and epistatic components. The first two effects can be easily tested in a conventional association analysis, but the computational burden of testing all possible pairwise and higher-order interactions limits our ability to determine the relative contribution of SNP-SNP interactions to complex trait variation. We took a novel three-stage approach to reduce the dimensionality of the data and directly test for epistasis. First, we built a model that took all 339 SNPs as possible predictors. Second, we paired this model down to include only the 20 SNPs with the highest importance values through a backward purging approach. Finally, we iteratively removed individual SNPs and recalculated the importance values of those that remained. This allowed us to build an interaction network describing both the strength and direction of pairwise interactions between adaptive SNPs. The principle behind this approach is that RF assigns an importance value to each SNP based on its contribution to the model either through its main or interactive effect, or both. Therefore, when the model is paired down to a smaller number of SNPs through the backward purging approach, the SNPs remained in the model represent the most important SNPs contributing to the phenotype. If a SNP interacts with other SNPs, the removal of this SNP will affect the importance values of the interacting SNPs. If the importance value of an interacting SNP is reduced, it suggests a positive interaction. Otherwise, the interaction is negative. However, because the magnitude of the interaction is determined based on the change in the importance values rather than the effect on the phenotype as estimated with traditional methods, these interactions only represent the relative importance of the epistasis.</p><p id="p-34">Somewhat surprisingly, all 20 of the most important SNPs for budset and 16 of 20 for cold injury exhibited at least one interaction. This result might be attributable to the backward purging approach we applied because it considered the effect of all levels of interactions for a SNP being removed from the model, which is different from other approaches reported (<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Jiang <em>et al.</em> 2009</a>; <a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Chen <em>et al.</em> 2011</a>). SNPs with the highest importance values (and which were significant in the mixed model association tests) for each trait tended to have more interactions than those with lower importance values. One strategy that has been used in the search for epistasis is to specify tests for interactions only among those SNPs with significant univariate associations. Our results suggest that this approach is likely to capture some of the genetic interactions relevant to complex trait variation, but would miss many interactions involving SNPs with no significant main effect. Although we only dealt with the top 20 SNPs from the RF analysis in our tests for epistasis, this is an arbitrary cutoff that could be relaxed to achieve a more comprehensive view of the interaction network governing complex traits.</p></div><div id="sec-9" class="subsection"><h3>Linkage disequilibrium between interacting loci</h3><p id="p-35">Epistasis is expected to produce stable LD between interacting loci (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Feldman <em>et al.</em> 1980</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Otto and Feldman 1997</a>). Although LD may also be generated by demographic phenomena such as migration, genetic drift, and single-locus selection (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Wade <em>et al.</em> 2001</a>), background levels of LD are low in most widespread tree species (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Krutovsky and Neale 2005</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Gonzalez-Martinez <em>et al.</em> 2006a</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Eckert <em>et al.</em> 2009b</a>). As such, LD between putatively interacting SNPs, particularly for intergenic interactions, provides additional evidence of <em>bona fide</em> epistasis. We therefore sought to determine whether putatively interacting SNPs in our study were also in linkage disequilibrium with one another. After correcting for multiple testing, we found that many, although not all, interacting SNPs exhibited LD. Interestingly, all but one of the interacting SNP pairs that also exhibited LD were positive interactions. This result reinforces the finding that negative epistasis promotes recombination, which in turn breaks down LD (<a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">Otto and Feldman 1997</a>).</p><p id="p-36">Although most SNPs in this study represented individual genetic loci presumably separated by some physical distance, and likely on different chromosomes in many cases (though we have no genetic map with which to test this assumption), there were a few cases of adjacent SNPs from the same gene among the top 20 in terms of importance values. One of the strongest interactions was for cold hardiness, between two SNPs separated by only 61bp in the <em>xth1</em> gene (SNPs 289 and 350). Each of these SNPs, as well as two others within <em>xth1</em>, were significantly associated with both budset and cold hardiness and explained between 3% and 5% of the variation in these traits (<a id="xref-ref-22-9" class="xref-bibr" href="#ref-22">Holliday <em>et al.</em> 2010</a>). Given the close physical proximity of these SNPs, we initially considered the possibility that LD was driving the associations, leading to a high PVE for all the SNPs adjacent to the quantitative trait nucleotide. However, given the strong epistasis for the two SNPs described above, as well as between SNPs 199 and 289 for budset, it appears more likely that each of these SNPs shape the phenotypic traits, alone and in combination. This result is not without precedent: previous studies have identified intragenic epistasis (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">da Silva <em>et al.</em> 2010</a>), and in one case, conformational changes in the protein products were directly linked to epistasis between allelic variants within the relevant gene (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Ortlund <em>et al.</em> 2007</a>).</p><p id="p-37">This study represents a significant step toward the goal of a comprehensive description of the genomic basis of local adaptation in widely distributed plant species in general, and conifer trees in particular. We conclude that dimensionality reduction using RF, coupled with iterative tests for pairwise interactions, is an efficient and fairly comprehensive approach to uncover SNPs and their interactions that shape adaptive traits, and this approach should be broadly applicable to other systems. The identification of a suite of markers that collectively explain a substantial fraction of the phenotypic variation in budset and cold hardiness in Sitka spruce provides an important resource for the conservation of genetic resources present in wild populations of spruce species, and prediction of ability to adapt to new climates. Future extensions of the RF approach may include the search for interpopulation allelic covariance that some theoretical and empirical work suggests may be responsible for high population differentiation in adaptive traits found in many tree species (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Latta 1998</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Le Corre and Kremer 2003</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Ma <em>et al.</em> 2010</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Kremer and Le Corre 2011</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Le Corre and Kremer 2012</a>). In trees, high Q<sub>ST</sub> is often accompanied by low F<sub>ST</sub> (<em>i.e.,</em> low genetic differentiation among populations), a result of high pollen-mediated gene flow, and this apparent paradox has yet to be reconciled by so-called F<sub>ST</sub>-outlier approaches that search for markers with unusually high levels of differentiation relative to the neutral distribution. <a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Kremer and Le Corre (2011)</a> show that in such situations, adaptation is likely the result of covariance between alleles at many loci, leading to strong clines in quantitative traits where clines in the underlying adaptive SNPs are weak and indistinguishable from the neutral distribution of F<sub>ST</sub>. Identifying allelic covariance among many possible candidate SNPs is a challenge, and the RF approach could be used to pare down the list of possible allelic combinations that lead to advantageous trait values for different populations. Such an analysis would likely be most successful if relative fitness were used as the response variable rather than the adaptive phenotypes themselves, since the optimal trait value for a given environment would not be known.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-38">We thank the two reviewers for helpful comments on a previous version of this manuscript. This work was supported by Genome British Columbia, Genome Canada, and the Province of British Columbia (funding proposals by S.A., J.A.H., and T.W.), by the Natural Science and Engineering Research Council of Canada (NSERC; grant to S.A.), by an NSERC Postgraduate Scholarship to J.A.H, and by Virginia Tech ‘Startup Funds’ to J.A.H.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.002733/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.002733/-/DC1</a></p></li><li class="fn" id="fn-3"><p id="p-39"><em>Communicating editor: R. W. Doerge</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2012-04-08"><span class="received-label">Received </span>April 8, 2012.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2012-07-03"><span class="accepted-label">Accepted </span>July 3, 2012.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2012 Holliday <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-3">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1085.1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ahuja</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neale</span>  <span class="cit-name-given-names">D. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Evolution of genome size in conifers</span>. <abbr class="cit-jnl-abbrev">Silvae Genet.</abbr> <span class="cit-vol">54</span>: <span class="cit-fpage">126</span>–<span class="cit-lpage">137</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSilvae%2BGenet.%26rft.volume%253D54%26rft.spage%253D126%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1085.2" data-doi="10.1111/j.1752-4571.2007.00013.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Aitken</span>  <span class="cit-name-given-names">S. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yeaman</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holliday</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Curtis-McLane</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Adaptation, migration or extirpation: climate change outcomes for tree populations</span>. <abbr class="cit-jnl-abbrev">Evolutionary Applications</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">95</span>–<span class="cit-lpage">111</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolutionary%2BApplications%26rft.volume%253D1%26rft.spage%253D95%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1752-4571.2007.00013.x%26rft_id%253Dinfo%253Apmid%252F25567494%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1752-4571.2007.00013.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25567494&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F9%2F1085.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000262783400009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1085.3" data-doi="10.1016/j.tree.2007.09.008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Barrett</span>  <span class="cit-name-given-names">R. D. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schluter</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Adaptation from standing genetic variation</span>. <abbr class="cit-jnl-abbrev">Trends Ecol. Evol.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">38</span>–<span class="cit-lpage">44</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2BEcology%2B%2526%2BEvolution%26rft.stitle%253DTrends%2Bin%2BEcology%2B%2526%2BEvolution%26rft.aulast%253DBarrett%26rft.auinit1%253DR.%2BD.%26rft.volume%253D23%26rft.issue%253D1%26rft.spage%253D38%26rft.epage%253D44%26rft.atitle%253DAdaptation%2Bfrom%2Bstanding%2Bgenetic%2Bvariation.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tree.2007.09.008%26rft_id%253Dinfo%253Apmid%252F18006185%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tree.2007.09.008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18006185&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F9%2F1085.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000252767300009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1085.4" data-doi="10.1023/A:1010933404324"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Breiman</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Random forests</span>. <abbr class="cit-jnl-abbrev">Mach. Learn.</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">5</span>–<span class="cit-lpage">32</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMach.%2BLearn.%26rft.volume%253D45%26rft.spage%253D5%26rft_id%253Dinfo%253Adoi%252F10.1023%252FA%253A1010933404324%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1023/A:1010933404324&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000170489900001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1085.5" data-doi="10.1002/gepi.20041"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bureau</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dupuis</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Falls</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lunetta</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayward</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Identifying SNPs predictive of phenotype using random forests</span>. <abbr class="cit-jnl-abbrev">Genet. Epidemiol.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">171</span>–<span class="cit-lpage">182</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetic%2Bepidemiology%26rft.stitle%253DGenet%2BEpidemiol%26rft.aulast%253DBureau%26rft.auinit1%253DA.%26rft.volume%253D28%26rft.issue%253D2%26rft.spage%253D171%26rft.epage%253D182%26rft.atitle%253DIdentifying%2BSNPs%2Bpredictive%2Bof%2Bphenotype%2Busing%2Brandom%2Bforests.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fgepi.20041%26rft_id%253Dinfo%253Apmid%252F15593090%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/gepi.20041&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15593090&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F9%2F1085.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000226568200006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1085.6" data-doi="10.1109/TCBB.2011.46"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">C. C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schwender</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Keith</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nunkesser</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mengersen</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Methods for identifying SNP interactions: a review on variations of logic regression, Random Forest and Bayesian logistic regression</span>. <abbr class="cit-jnl-abbrev">Trans. Computat. Biol. Bioinformatics</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">1580</span>–<span class="cit-lpage">1591</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrans.%2BComputat.%2BBiol.%2BBioinformatics%26rft.volume%253D8%26rft.spage%253D1580%26rft_id%253Dinfo%253Adoi%252F10.1109%252FTCBB.2011.46%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1109/TCBB.2011.46&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1085.7" data-doi="10.1038/nrg2579"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cordell</span>  <span class="cit-name-given-names">H. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Detecting gene-gene interactions that underlie human diseases</span>. <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">392</span>–<span class="cit-lpage">404</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DCordell%26rft.auinit1%253DH.%2BJ.%26rft.volume%253D10%26rft.issue%253D6%26rft.spage%253D392%26rft.epage%253D404%26rft.atitle%253DDetecting%2Bgene-gene%2Binteractions%2Bthat%2Bunderlie%2Bhuman%2Bdiseases.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2579%26rft_id%253Dinfo%253Apmid%252F19434077%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2579&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19434077&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F9%2F1085.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000266095200014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1085.8" data-doi="10.1534/genetics.109.112458"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">da Silva</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coetzer</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nedellec</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pastore</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mosier</span>  <span class="cit-name-given-names">D. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Fitness epistasis and constraints on adaptation in a human immunodeficiency virus type 1 protein region</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">185</span>: <span class="cit-fpage">293</span>–<span class="cit-lpage">303</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D185%26rft.spage%253D293%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.109.112458%26rft_id%253Dinfo%253Apmid%252F20157005%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/genetics.109.112458&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20157005&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F9%2F1085.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000281903800026&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1085.9" data-doi="10.1534/genetics.109.102350"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eckert</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bower</span>  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wegrzyn</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pande</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jermstad</span>  <span class="cit-name-given-names">K. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span><span class="cit-comment">a</span> <span class="cit-article-title">Asssociation genetics of coastal Douglas-fir (<em>Pseudotsuga menziesii</em> var. <em>menziesii</em>, Pinaceae). I. Cold hardiness-related traits</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">182</span>: <span class="cit-fpage">1289</span>–<span class="cit-lpage">1302</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D182%26rft.spage%253D1289%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.109.102350%26rft_id%253Dinfo%253Apmid%252F19487566%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/genetics.109.102350&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19487566&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F9%2F1085.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000270214000029&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1085.10" data-doi="10.1534/genetics.109.103895"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eckert</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wegrzyn</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pande</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jermstad</span>  <span class="cit-name-given-names">K. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span><span class="cit-comment">b</span> <span class="cit-article-title">Multilocus patterns of nucleotide diversity and divergence reveal positive selection at candidate genes related to cold hardiness in coastal Douglas-fir (<em>Pseudotsuga menziesii</em> var. menziesii)</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">183</span>: <span class="cit-fpage">289</span>–<span class="cit-lpage">298</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.109.103895%26rft_id%253Dinfo%253Apmid%252F19596906%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTgzLzEvMjg5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMi85LzEwODUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1085.11" data-doi="10.1111/j.1469-8137.2011.03976.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eckert</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wegrzyn</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cumbie</span>  <span class="cit-name-given-names">W. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goldfarb</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huber</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Association genetics of the loblolly pine (<em>Pinus taeda</em>, Pinaceae) metabolome</span>. <abbr class="cit-jnl-abbrev">New Phytol.</abbr> <span class="cit-vol">193</span>: <span class="cit-fpage">890</span>–<span class="cit-lpage">902</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytologist%26rft.stitle%253DNew%2BPhytologist%26rft.aulast%253DEckert%26rft.auinit1%253DA.%2BJ.%26rft.volume%253D193%26rft.issue%253D4%26rft.spage%253D890%26rft.epage%253D902%26rft.atitle%253DAssociation%2Bgenetics%2Bof%2Bthe%2Bloblolly%2Bpine%2B%2528Pinus%2Btaeda%252C%2BPinaceae%2529%2Bmetabolome.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1469-8137.2011.03976.x%26rft_id%253Dinfo%253Apmid%252F22129444%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1469-8137.2011.03976.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22129444&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F9%2F1085.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000299778300011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1085.12" data-doi="10.1073/pnas.77.8.4838"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Feldman</span>  <span class="cit-name-given-names">M. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Christiansen</span>  <span class="cit-name-given-names">F. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brooks</span>  <span class="cit-name-given-names">L. D.</span></span></li></ol><cite>, <span class="cit-pub-date">1980</span> <span class="cit-article-title">Evolution and recombination in a constant environment</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">77</span>: <span class="cit-fpage">4838</span>–<span class="cit-lpage">4841</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DFeldman%26rft.auinit1%253DM.%2BW.%26rft.volume%253D77%26rft.issue%253D8%26rft.spage%253D4838%26rft.epage%253D4841%26rft.atitle%253DEvolution%2Bof%2Brecombination%2Bin%2Ba%2Bconstant%2Benvironment%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.77.8.4838%26rft_id%253Dinfo%253Apmid%252F16592864%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiI3Ny84LzQ4MzgiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8yLzkvMTA4NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1085.13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goldstein</span>  <span class="cit-name-given-names">B. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hubbard</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cutler</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barcellos</span>  <span class="cit-name-given-names">L. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">An application of Random Forests to a genome-wide association dataset: methodological considerations &amp; new findings</span>. <abbr class="cit-jnl-abbrev">BMC Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">49</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenetics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenet%26rft.aulast%253DGoldstein%26rft.auinit1%253DB.%2BA.%26rft.volume%253D11%26rft.spage%253D49%26rft.epage%253D49%26rft.atitle%253DAn%2Bapplication%2Bof%2BRandom%2BForests%2Bto%2Ba%2Bgenome-wide%2Bassociation%2Bdataset%253A%2Bmethodological%2Bconsiderations%2B%2526amp%253B%2Bnew%2Bfindings.%26rft_id%253Dinfo%253Apmid%252F20546594%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=20546594&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F9%2F1085.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1085.14" data-doi="10.1534/genetics.105.047126"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gonzalez-Martinez</span>  <span class="cit-name-given-names">S. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ersoz</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">G. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wheeler</span>  <span class="cit-name-given-names">N. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neale</span>  <span class="cit-name-given-names">D. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span><span class="cit-comment">a</span> <span class="cit-article-title">DNA sequence variation and selection of tag single-nucleotide polymorphisms at candidate genes for drought-stress response in <em>Pinus taeda</em> L</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">172</span>: <span class="cit-fpage">1915</span>–<span class="cit-lpage">1926</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D172%26rft.spage%253D1915%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.105.047126%26rft_id%253Dinfo%253Apmid%252F16387885%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/genetics.105.047126&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16387885&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F9%2F1085.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000236668100046&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1085.15" data-doi="10.1111/j.1469-8137.2006.01686.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gonzalez-Martinez</span>  <span class="cit-name-given-names">S. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krutovsky</span>  <span class="cit-name-given-names">K. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neale</span>  <span class="cit-name-given-names">D. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span><span class="cit-comment">b</span> <span class="cit-article-title">Forest tree population genomics and adaptive evolution</span>. <abbr class="cit-jnl-abbrev">New Phytol.</abbr> <span class="cit-vol">170</span>: <span class="cit-fpage">227</span>–<span class="cit-lpage">238</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytologist%26rft.stitle%253DNew%2BPhytologist%26rft.aulast%253DGonzalez-Martinez%26rft.auinit1%253DS.%2BC.%26rft.volume%253D170%26rft.issue%253D2%26rft.spage%253D227%26rft.epage%253D238%26rft.atitle%253DForest-tree%2Bpopulation%2Bgenomics%2Band%2Badaptive%2Bevolution.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1469-8137.2006.01686.x%26rft_id%253Dinfo%253Apmid%252F16608450%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1469-8137.2006.01686.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16608450&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F9%2F1085.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000236248200007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1085.16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gonzalez-Martinez</span>  <span class="cit-name-given-names">S. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wheeler</span>  <span class="cit-name-given-names">N. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ersoz</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nelson</span>  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neale</span>  <span class="cit-name-given-names">D. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Association genetics in <em>Pinus taeda</em> L. I. Wood property traits</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">175</span>: <span class="cit-fpage">399</span>–<span class="cit-lpage">409</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D175%26rft.spage%253D399%26rft_id%253Dinfo%253Apmid%252F17110498%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=17110498&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F9%2F1085.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000244180300034&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1085.17" data-doi="10.1038/hdy.2008.21"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gonzalez-Martinez</span>  <span class="cit-name-given-names">S. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huber</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ersoz</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neale</span>  <span class="cit-name-given-names">D. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Association genetics in <em>Pinus taeda</em> L. II. Carbon isotope discrimination</span>. <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">101</span>: <span class="cit-fpage">19</span>–<span class="cit-lpage">26</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.stitle%253DHeredity%26rft.aulast%253DGonzalez-Martinez%26rft.auinit1%253DS.%2BC.%26rft.volume%253D101%26rft.issue%253D1%26rft.spage%253D19%26rft.epage%253D26%26rft.atitle%253DAssociation%2Bgenetics%2Bin%2BPinus%2Btaeda%2BL.%2BII.%2BCarbon%2Bisotope%2Bdiscrimination.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2008.21%26rft_id%253Dinfo%253Apmid%252F18478029%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2008.21&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18478029&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F9%2F1085.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000256832800004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1085.18" data-doi="10.1007/s11295-010-0328-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Grattapaglia</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Resende</span>  <span class="cit-name-given-names">M. D. V.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genomic selection in forest tree breeding</span>. <abbr class="cit-jnl-abbrev">Tree Genet. Genomes</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">241</span>–<span class="cit-lpage">255</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTree%2BGenet.%2BGenomes%26rft.volume%253D7%26rft.spage%253D241%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11295-010-0328-4%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11295-010-0328-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000288656800003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1085.19" data-doi="10.1093/bioinformatics/btf869"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hahn</span>  <span class="cit-name-given-names">L. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ritchie</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moore</span>  <span class="cit-name-given-names">J. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Multifactor dimensionality reduction software for detecting gene−gene and gene−environment interactions</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">376</span>–<span class="cit-lpage">382</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DHahn%26rft.auinit1%253DL.%2BW.%26rft.volume%253D19%26rft.issue%253D3%26rft.spage%253D376%26rft.epage%253D382%26rft.atitle%253DMultifactor%2Bdimensionality%2Breduction%2Bsoftware%2Bfor%2Bdetecting%2Bgene-gene%2Band%2Bgene-environment%2Binteractions%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtf869%26rft_id%253Dinfo%253Apmid%252F12584123%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIxOS8zLzM3NiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzIvOS8xMDg1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1085.20" data-doi="10.1890/0012-9658(2006)87[2773:PEOCCO]2.0.CO;2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hamann</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">T. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Potential effects of climate change on ecosystem and tree species distribution in British Columbia</span>. <abbr class="cit-jnl-abbrev">Ecology</abbr> <span class="cit-vol">87</span>: <span class="cit-fpage">2773</span>–<span class="cit-lpage">2786</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcology%26rft.stitle%253DEcology%26rft.aulast%253DHamann%26rft.auinit1%253DA.%26rft.volume%253D87%26rft.issue%253D11%26rft.spage%253D2773%26rft.epage%253D2786%26rft.atitle%253DPotential%2Beffects%2Bof%2Bclimate%2Bchange%2Bon%2Becosystem%2Band%2Btree%2Bspecies%2Bdistribution%2Bin%2BBritish%2BColumbia.%26rft_id%253Dinfo%253Adoi%252F10.1890%252F0012-9658%25282006%252987%255B2773%253APEOCCO%255D2.0.CO%253B2%26rft_id%253Dinfo%253Apmid%252F17168022%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1890/0012-9658(2006)87[2773:PEOCCO]2.0.CO;2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17168022&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F9%2F1085.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000242214800010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1085.21" data-doi="10.1139/x99-019"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hannerz</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aitken</span>  <span class="cit-name-given-names">S. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">King</span>  <span class="cit-name-given-names">J. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Budge</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Effects of genetic selection for growth on frost hardiness in western hemlock</span>. <abbr class="cit-jnl-abbrev">Can. J. Forest Res.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">509</span>–<span class="cit-lpage">516</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCan.%2BJ.%2BForest%2BRes.%26rft.volume%253D29%26rft.spage%253D509%26rft_id%253Dinfo%253Adoi%252F10.1139%252Fx99-019%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1139/x99-019&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1085.22" data-doi="10.1111/j.1469-8137.2010.03380.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Holliday</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ritland</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aitken</span>  <span class="cit-name-given-names">S. N.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Widespread, ecologically relevant genetic markers developed from association mapping of climate-related traits in Sitka spruce (<em>Picea sitchensis</em>)</span>. <abbr class="cit-jnl-abbrev">New Phytol.</abbr> <span class="cit-vol">188</span>: <span class="cit-fpage">501</span>–<span class="cit-lpage">514</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytologist%26rft.stitle%253DNew%2BPhytologist%26rft.aulast%253DHolliday%26rft.auinit1%253DJ.%2BA.%26rft.volume%253D188%26rft.issue%253D2%26rft.spage%253D501%26rft.epage%253D514%26rft.atitle%253DWidespread%252C%2Becologically%2Brelevant%2Bgenetic%2Bmarkers%2Bdeveloped%2Bfrom%2Bassociation%2Bmapping%2Bof%2Bclimate-related%2Btraits%2Bin%2BSitka%2Bspruce%2B%2528Picea%2Bsitchensis%2529.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1469-8137.2010.03380.x%26rft_id%253Dinfo%253Apmid%252F20663060%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1469-8137.2010.03380.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20663060&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F9%2F1085.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000282225000019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1085.23" data-doi="10.1534/genetics.107.082354"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ingvarsson</span>  <span class="cit-name-given-names">P. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garcia</span>  <span class="cit-name-given-names">M. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luquez</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hall</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jansson</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Nucleotide polymoirphism and phenotypic associations within and around the phytochrome B2 locus in European aspen (<em>Populus tremula</em>, Salicaceae)</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">178</span>: <span class="cit-fpage">2217</span>–<span class="cit-lpage">2226</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D178%26rft.spage%253D2217%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.082354%26rft_id%253Dinfo%253Apmid%252F18245834%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/genetics.107.082354&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18245834&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F9%2F1085.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000255239600031&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1085.24"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lorenz</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Iwata</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genomic selection in plant breeding: from theory to practice</span>. <abbr class="cit-jnl-abbrev">Brief.Funct. Genomics</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">166</span>–<span class="cit-lpage">177</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBrief.Funct.%2BGenomics%26rft.volume%253D9%26rft.spage%253D166%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1085.25" data-doi="10.1186/1471-2105-10-S1-S65"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tang</span>  <span class="cit-name-given-names">W. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">X. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">W. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A random forest approach to the detection of epistatic interactions in case-control studies</span>. <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">10</span>(<span class="cit-supplement">Suppl. 1</span>)<strong>:</strong> <span class="cit-fpage">S65</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BBioinformatics%26rft.volume%253D10%26rft.spage%253DS65%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-10-S1-S65%26rft_id%253Dinfo%253Apmid%252F19208169%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-10-S1-S65&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19208169&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F9%2F1085.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1085.26"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kremer</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Le Corre</span>  <span class="cit-name-given-names">V.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Decoupling of differentiation between traits and their underlying genes in response to divergent selection</span>. <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">375</span>–<span class="cit-lpage">385</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D108%26rft.spage%253D375%26rft_id%253Dinfo%253Apmid%252F21915150%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=21915150&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F9%2F1085.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1085.27" data-doi="10.1534/genetics.105.044420"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Krutovsky</span>  <span class="cit-name-given-names">K. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neale</span>  <span class="cit-name-given-names">D. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Nucleotide diversity and linkage disequilibrium in cold-hardiness- and wood quality-related candidate genes in Douglas fir</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">171</span>: <span class="cit-fpage">2029</span>–<span class="cit-lpage">2041</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.105.044420%26rft_id%253Dinfo%253Apmid%252F16157674%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3MS80LzIwMjkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8yLzkvMTA4NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1085.28" data-doi="10.1086/286119"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Latta</span>  <span class="cit-name-given-names">R. G.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Differentiation of allelic frequencies at quantitative trait loci affecting locally adaptive traits</span>. <abbr class="cit-jnl-abbrev">Am. Nat.</abbr> <span class="cit-vol">151</span>: <span class="cit-fpage">283</span>–<span class="cit-lpage">292</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BNat.%26rft.volume%253D151%26rft.spage%253D283%26rft_id%253Dinfo%253Adoi%252F10.1086%252F286119%26rft_id%253Dinfo%253Apmid%252F18811359%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/286119&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18811359&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F9%2F1085.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000072128600008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1085.29"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Le Corre</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kremer</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Genetic variability at neutral markers, quantitative trait loci and trait in a subdivided population under selection</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">164</span>: <span class="cit-fpage">1205</span>–<span class="cit-lpage">1219</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DLe%2BCorre%26rft.auinit1%253DV.%26rft.volume%253D164%26rft.issue%253D3%26rft.spage%253D1205%26rft.epage%253D1219%26rft.atitle%253DGenetic%2BVariability%2Bat%2BNeutral%2BMarkers%252C%2BQuantitative%2BTrait%2BLoci%2Band%2BTrait%2Bin%2Ba%2BSubdivided%2BPopulation%2BUnder%2BSelection%26rft_id%253Dinfo%253Apmid%252F12871925%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2NC8zLzEyMDUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8yLzkvMTA4NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1085.30" data-doi="10.1111/j.1365-294X.2012.05479.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Le Corre</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kremer</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The genetic differentiation at quantitative trait loci under local adaptation</span>. <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">1548</span>–<span class="cit-lpage">1566</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DLE%2BCorre%26rft.auinit1%253DV.%26rft.volume%253D21%26rft.issue%253D7%26rft.spage%253D1548%26rft.epage%253D1566%26rft.atitle%253DThe%2Bgenetic%2Bdifferentiation%2Bat%2Bquantitative%2Btrait%2Bloci%2Bunder%2Blocal%2Badaptation.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2012.05479.x%26rft_id%253Dinfo%253Apmid%252F22332667%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2012.05479.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22332667&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F9%2F1085.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000301933600004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1085.31"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liaw</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wiener</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Classification and regression by randomForest</span>. <abbr class="cit-jnl-abbrev">R News</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">18</span>–<span class="cit-lpage">22</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DR%2BNews%26rft.volume%253D2%26rft.spage%253D18%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1085.32" data-doi="10.1186/1471-2156-5-32"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lunetta</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayward</span>  <span class="cit-name-given-names">L. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Segal</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van Eerdewegh</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Screening large-scale association study data: exploiting interactions using random forests</span>. <abbr class="cit-jnl-abbrev">BMC Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">32</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenetics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenet%26rft.aulast%253DLunetta%26rft.auinit1%253DK.%2BL.%26rft.volume%253D5%26rft.spage%253D32%26rft.epage%253D32%26rft.atitle%253DScreening%2Blarge-scale%2Bassociation%2Bstudy%2Bdata%253A%2Bexploiting%2Binteractions%2Busing%2Brandom%2Bforests.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2156-5-32%26rft_id%253Dinfo%253Apmid%252F15588316%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2156-5-32&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15588316&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F9%2F1085.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1085.33" data-doi="10.1002/gepi.20140"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lunn</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Whittaker</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Best</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">A Bayesian toolkit for genetic association studies</span>. <abbr class="cit-jnl-abbrev">Genet. Epidemiol.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">231</span>–<span class="cit-lpage">247</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetic%2Bepidemiology%26rft.stitle%253DGenet%2BEpidemiol%26rft.aulast%253DLunn%26rft.auinit1%253DD.%2BJ.%26rft.volume%253D30%26rft.issue%253D3%26rft.spage%253D231%26rft.epage%253D247%26rft.atitle%253DA%2BBayesian%2Btoolkit%2Bfor%2Bgenetic%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fgepi.20140%26rft_id%253Dinfo%253Apmid%252F16544290%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/gepi.20140&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16544290&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F9%2F1085.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000236422100004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1085.34" data-doi="10.1534/genetics.110.120873"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">X.-F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hall</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">St Onge</span>  <span class="cit-name-given-names">K. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jansson</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ingvarsson</span>  <span class="cit-name-given-names">P. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genetic differentiation, clinal variation and phenotypic associations with growth cessation across the populus tremula photoperiodic Pathway</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">186</span>: <span class="cit-fpage">1033</span>–<span class="cit-lpage">1044</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.120873%26rft_id%253Dinfo%253Apmid%252F20805554%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE4Ni8zLzEwMzMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8yLzkvMTA4NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1085.35" data-doi="10.1046/j.1365-2699.2002.00702.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Malcolm</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Markham</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neilson</span>  <span class="cit-name-given-names">R. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garaci</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Estimated migration rates under scenarios of global climate change</span>. <abbr class="cit-jnl-abbrev">J. Biogeogr.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">835</span>–<span class="cit-lpage">849</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiogeogr.%26rft.volume%253D29%26rft.spage%253D835%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1365-2699.2002.00702.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1365-2699.2002.00702.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000177026400001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1085.36" data-doi="10.1534/genetics.105.046078"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Malmberg</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Held</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Waits</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mauricio</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Epistasis for fitness-related quantitative traits in Arabidopsis thaliana grown in the field and in the greenhouse</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">171</span>: <span class="cit-fpage">2013</span>–<span class="cit-lpage">2027</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D171%26rft.spage%253D2013%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.105.046078%26rft_id%253Dinfo%253Apmid%252F16157670%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/genetics.105.046078&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16157670&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F9%2F1085.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000234407100050&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1085.37" data-doi="10.1890/04-1036"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McLachlan</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clark</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Manos</span>  <span class="cit-name-given-names">P. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Molecular indicators of tree migration capacity under rapid climate change</span>. <abbr class="cit-jnl-abbrev">Ecology</abbr> <span class="cit-vol">86</span>: <span class="cit-fpage">2088</span>–<span class="cit-lpage">2098</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcology%26rft.volume%253D86%26rft.spage%253D2088%26rft_id%253Dinfo%253Adoi%252F10.1890%252F04-1036%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1890/04-1036&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000231075400016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1085.38"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>  <span class="cit-name-given-names">T. H. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Prediction of total genetic value using genome-wide dense marker maps</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">157</span>: <span class="cit-fpage">1819</span>–<span class="cit-lpage">1829</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMeuwissen%26rft.auinit1%253DT.%2BH.%2BE.%26rft.volume%253D157%26rft.issue%253D4%26rft.spage%253D1819%26rft.epage%253D1829%26rft.atitle%253DPrediction%2Bof%2BTotal%2BGenetic%2BValue%2BUsing%2BGenome-Wide%2BDense%2BMarker%2BMaps%26rft_id%253Dinfo%253Apmid%252F11290733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1Ny80LzE4MTkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8yLzkvMTA4NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1085.39" data-doi="10.1038/sj.hdy.6800987"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mimura</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aitken</span>  <span class="cit-name-given-names">S. N.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Adaptive gradients and isolation-by-distance with postglacial migration in <em>Picea sitchensis</em></span>. <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">99</span>: <span class="cit-fpage">224</span>–<span class="cit-lpage">232</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.stitle%253DHeredity%26rft.aulast%253DMimura%26rft.auinit1%253DM.%26rft.volume%253D99%26rft.issue%253D2%26rft.spage%253D224%26rft.epage%253D232%26rft.atitle%253DAdaptive%2Bgradients%2Band%2Bisolation-by-distance%2Bwith%2Bpostglacial%2Bmigration%2Bin%2BPicea%2Bsitchensis.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsj.hdy.6800987%26rft_id%253Dinfo%253Apmid%252F17487214%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/sj.hdy.6800987&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17487214&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F9%2F1085.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000248301900013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1085.40" data-doi="10.1016/j.jtbi.2005.11.036"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Moore</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gilbert</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tsai</span>  <span class="cit-name-given-names">C. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chiang</span>  <span class="cit-name-given-names">F. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holden</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">A flexible computational framework for detecting, characterizing, and interpreting statistical patterns of epistasis in genetic studies of human disease susceptibility</span>. <abbr class="cit-jnl-abbrev">J. Theor. Biol.</abbr> <span class="cit-vol">241</span>: <span class="cit-fpage">252</span>–<span class="cit-lpage">261</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BTheoretical%2BBiology%26rft.stitle%253DJournal%2Bof%2BTheoretical%2BBiology%26rft.aulast%253DMoore%26rft.auinit1%253DJ.%2BH.%26rft.volume%253D241%26rft.issue%253D2%26rft.spage%253D252%26rft.epage%253D261%26rft.atitle%253DA%2Bflexible%2Bcomputational%2Bframework%2Bfor%2Bdetecting%252C%2Bcharacterizing%252C%2Band%2Binterpreting%2Bstatistical%2Bpatterns%2Bof%2Bepistasis%2Bin%2Bgenetic%2Bstudies%2Bof%2Bhuman%2Bdisease%2Bsusceptibility.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jtbi.2005.11.036%26rft_id%253Dinfo%253Apmid%252F16457852%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jtbi.2005.11.036&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16457852&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F9%2F1085.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000239691300009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1085.41" data-doi="10.1105/tpc.109.068437"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Myles</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peiffer</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ersoz</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Association mapping: critical considerations shift from genotyping to experimental design</span>. <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">2194</span>–<span class="cit-lpage">2202</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.109.068437%26rft_id%253Dinfo%253Apmid%252F19654263%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjk6IjIxLzgvMjE5NCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzIvOS8xMDg1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1085.42" data-doi="10.1016/j.tplants.2004.05.006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Neale</span>  <span class="cit-name-given-names">D. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Savolainen</span>  <span class="cit-name-given-names">O.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Association genetics of complex traits in conifers</span>. <abbr class="cit-jnl-abbrev">Trends Plant Sci.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">325</span>–<span class="cit-lpage">330</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bplant%2Bscience%26rft.stitle%253DTrends%2BPlant%2BSci%26rft.aulast%253DNeale%26rft.auinit1%253DD.%2BB.%26rft.volume%253D9%26rft.issue%253D7%26rft.spage%253D325%26rft.epage%253D330%26rft.atitle%253DAssociation%2Bgenetics%2Bof%2Bcomplex%2Btraits%2Bin%2Bconifers.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tplants.2004.05.006%26rft_id%253Dinfo%253Apmid%252F15231277%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tplants.2004.05.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15231277&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F9%2F1085.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000223010600004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1085.43" data-doi="10.1073/pnas.0913160107"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nemri</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Atwell</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tarone</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">Y. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genome-wide survey of Arabidopsis natural variation in downy mildew resistance using combined association and linkage mapping</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">107</span>: <span class="cit-fpage">10302</span>–<span class="cit-lpage">10307</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0913160107%26rft_id%253Dinfo%253Apmid%252F20479233%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA3LzIyLzEwMzAyIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMi85LzEwODUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1085.44" data-doi="10.2307/2411226"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Orr</span>  <span class="cit-name-given-names">H. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">The population genetics of adaptation: the distribution of factors fixed during adaptive evolution</span>. <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">935</span>–<span class="cit-lpage">949</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.volume%253D52%26rft.spage%253D935%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2411226%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2411226&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000075753100001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1085.45" data-doi="10.1017/S0016672399004164"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Orr</span>  <span class="cit-name-given-names">H. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">The evolutionary genetics of adaptation: a simulation study</span>. <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">74</span>: <span class="cit-fpage">207</span>–<span class="cit-lpage">214</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DOrr%26rft.auinit1%253DH.%2BA.%26rft.volume%253D74%26rft.issue%253D3%26rft.spage%253D207%26rft.epage%253D214%26rft.atitle%253DThe%2Bevolutionary%2Bgenetics%2Bof%2Badaptation%253A%2Ba%2Bsimulation%2Bstudy.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672399004164%26rft_id%253Dinfo%253Apmid%252F10689798%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672399004164&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10689798&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F9%2F1085.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000085417000002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1085.46" data-doi="10.1126/science.1142819"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ortlund</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bridgham</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Redinbo</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thornton</span>  <span class="cit-name-given-names">J. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Crystal structure of an ancient protein: evolution by conformational epistasis</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">317</span>: <span class="cit-fpage">1544</span>–<span class="cit-lpage">1548</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1142819%26rft_id%253Dinfo%253Apmid%252F17702911%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMTcvNTg0NC8xNTQ0IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMi85LzEwODUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1085.47" data-doi="10.1006/tpbi.1997.1301"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Otto</span>  <span class="cit-name-given-names">S. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feldman</span>  <span class="cit-name-given-names">M. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Deleterious mutations, variable epistatic interactions, and the evolution of recombination</span>. <abbr class="cit-jnl-abbrev">Theor. Popul. Biol.</abbr> <span class="cit-vol">51</span>: <span class="cit-fpage">134</span>–<span class="cit-lpage">147</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheoretical%2Bpopulation%2Bbiology%26rft.stitle%253DTheor%2BPopul%2BBiol%26rft.aulast%253DOtto%26rft.auinit1%253DS.%2BP.%26rft.volume%253D51%26rft.issue%253D2%26rft.spage%253D134%26rft.epage%253D147%26rft.atitle%253DDeleterious%2Bmutations%252C%2Bvariable%2Bepistatic%2Binteractions%252C%2Band%2Bthe%2Bevolution%2Bof%2Brecombination.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Ftpbi.1997.1301%26rft_id%253Dinfo%253Apmid%252F9169238%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/tpbi.1997.1301&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9169238&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F9%2F1085.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1997WW85300005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1085.48" data-doi="10.1002/jez.b.21410"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pavlicev</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Norgard</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fawcett</span>  <span class="cit-name-given-names">G. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheverud</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Evolution of pleiotropy: epistatic interaction pattern supports a mechanistic model underlying variation in genotype-phenotype map</span>. <abbr class="cit-jnl-abbrev">J. Exp. Zoolog. B Mol. Dev. Evol.</abbr> <span class="cit-vol">316B</span>: <span class="cit-fpage">371</span>–<span class="cit-lpage">385</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BExp.%2BZoolog.%2BB%2BMol.%2BDev.%2BEvol.%26rft.volume%253D316B%26rft.spage%253D371%26rft.atitle%253DEVOLUTION%2BOF%2BPLEIOTROPY%253A%2BEPISTATIC%2BINTERACTION%2BPATTERN%2BSUPPORTS%2BA%2BMECHANISTIC%2BMODEL%2BUNDERLYING%2BVARIATION%2BIN%2BGENOTYPE-PHENOTYPE%2BMAP%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fjez.b.21410%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/jez.b.21410&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1085.49" data-doi="10.1146/annurev.ecolsys.37.091305.110215"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Petit</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hampe</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Some evolutionary consequences of being a tree</span>. <abbr class="cit-jnl-abbrev">Annu. Rev. Ecol. Evol. Syst.</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">187</span>–<span class="cit-lpage">214</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnu.%2BRev.%2BEcol.%2BEvol.%2BSyst.%26rft.volume%253D37%26rft.spage%253D187%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.ecolsys.37.091305.110215%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.ecolsys.37.091305.110215&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000243038500008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1085.50" data-doi="10.1038/nrg2452"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Phillips</span>  <span class="cit-name-given-names">P. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Epistasis—the essential role of gene interactions in the structure and evolution of genetic systems</span>. <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">855</span>–<span class="cit-lpage">867</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DPhillips%26rft.auinit1%253DP.%2BC.%26rft.volume%253D9%26rft.issue%253D11%26rft.spage%253D855%26rft.epage%253D867%26rft.atitle%253DEpistasis--the%2Bessential%2Brole%2Bof%2Bgene%2Binteractions%2Bin%2Bthe%2Bstructure%2Band%2Bevolution%2Bof%2Bgenetic%2Bsystems.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2452%26rft_id%253Dinfo%253Apmid%252F18852697%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2452&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18852697&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F9%2F1085.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000260162500010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1085.51"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pritchard</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Donnelly</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Inference of population structure using multilocus genotype data</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">155</span>: <span class="cit-fpage">945</span>–<span class="cit-lpage">959</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D155%26rft.spage%253D945%26rft_id%253Dinfo%253Apmid%252F10835412%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10835412&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F9%2F1085.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000087475100039&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1085.52" data-doi="10.1534/genetics.110.117549"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Quesada</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gopal</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cumbie</span>  <span class="cit-name-given-names">W. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eckert</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wegrzyn</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Association mapping of quantitative disease resistance in a natural population of loblolly pine (<em>Pinus taeda</em> L.)</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">186</span>: <span class="cit-fpage">677</span>–<span class="cit-lpage">686</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D186%26rft.spage%253D677%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.117549%26rft_id%253Dinfo%253Apmid%252F20628037%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/genetics.110.117549&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20628037&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F9%2F1085.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000282807400019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1085.53" data-doi="10.1111/j.1469-8137.2011.03895.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Resende</span>  <span class="cit-name-given-names">M. F. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Munoz</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Acosta</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peter</span>  <span class="cit-name-given-names">G. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Accelerating the domestication of trees using genomic selection: accuracy of prediction models across ages and environments</span>. <abbr class="cit-jnl-abbrev">New Phytol.</abbr> <span class="cit-vol">193</span>: <span class="cit-fpage">617</span>–<span class="cit-lpage">624</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytologist%26rft.stitle%253DNew%2BPhytologist%26rft.aulast%253DResende%26rft.auinit1%253DM.%2BF.%26rft.volume%253D193%26rft.issue%253D3%26rft.spage%253D617%26rft.epage%253D624%26rft.atitle%253DAccelerating%2Bthe%2Bdomestication%2Bof%2Btrees%2Busing%2Bgenomic%2Bselection%253A%2Baccuracy%2Bof%2Bprediction%2Bmodels%2Bacross%2Bages%2Band%2Benvironments.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1469-8137.2011.03895.x%26rft_id%253Dinfo%253Apmid%252F21973055%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1469-8137.2011.03895.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21973055&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F9%2F1085.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000298984900008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1085.54" data-doi="10.1111/j.1471-8286.2007.01931.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rousset</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">GENEPOP’007: a complete re-implementation of the GENEPOP software for Windows and Linux</span>. <abbr class="cit-jnl-abbrev">Molecular Ecology Resources</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">103</span>–<span class="cit-lpage">106</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2BResources%26rft.volume%253D8%26rft.spage%253D103%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1471-8286.2007.01931.x%26rft_id%253Dinfo%253Apmid%252F21585727%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1471-8286.2007.01931.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21585727&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F9%2F1085.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000253827100016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1085.55" data-doi="10.1146/annurev.ecolsys.38.091206.095646"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Savolainen</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pyhajarvi</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Knurr</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Gene flow and local adaptation in tees</span>. <abbr class="cit-jnl-abbrev">Annu. Rev. Ecol. Evol. Syst.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">595</span>–<span class="cit-lpage">619</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnu.%2BRev.%2BEcol.%2BEvol.%2BSyst.%26rft.volume%253D38%26rft.spage%253D595%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.ecolsys.38.091206.095646%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.ecolsys.38.091206.095646&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000252359800024&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1085.56" data-doi="10.1101/gr.1239303"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shannon</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Markiel</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ozier</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baliga</span>  <span class="cit-name-given-names">N. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Cytoscape: a software environment for integrated models of biomolecular interaction networks</span>. <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">2498</span>–<span class="cit-lpage">2504</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DShannon%26rft.auinit1%253DP.%26rft.volume%253D13%26rft.issue%253D11%26rft.spage%253D2498%26rft.epage%253D2504%26rft.atitle%253DCytoscape%253A%2BA%2BSoftware%2BEnvironment%2Bfor%2BIntegrated%2BModels%2Bof%2BBiomolecular%2BInteraction%2BNetworks%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.1239303%26rft_id%253Dinfo%253Apmid%252F14597658%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIxMy8xMS8yNDk4IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMi85LzEwODUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1085.57" data-doi="10.1073/pnas.1530509100"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Storey</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tibshirani</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Statistical significance for genomewide studies</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">100</span>: <span class="cit-fpage">9440</span>–<span class="cit-lpage">9445</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1530509100%26rft_id%253Dinfo%253Apmid%252F12883005%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTAwLzE2Lzk0NDAiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8yLzkvMTA4NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1085.58" data-doi="10.1007/s00122-010-1401-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">von Korff</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leon</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pillen</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Detection of epistatic interactions between exotic alleles introgressed from wild barley (<em>H. vulgare ssp. spontaneum</em>)</span>. <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">121</span>: <span class="cit-fpage">1455</span>–<span class="cit-lpage">1464</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253Dvon%2BKorff%26rft.auinit1%253DM.%26rft.volume%253D121%26rft.issue%253D8%26rft.spage%253D1455%26rft.epage%253D1464%26rft.atitle%253DDetection%2Bof%2Bepistatic%2Binteractions%2Bbetween%2Bexotic%2Balleles%2Bintrogressed%2Bfrom%2Bwild%2Bbarley%2B%2528H.%2Bvulgare%2Bssp.%2Bspontaneum%2529.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-010-1401-y%26rft_id%253Dinfo%253Apmid%252F20617300%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-010-1401-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20617300&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F9%2F1085.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1085.59" data-doi="10.1016/S0169-5347(01)02213-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wade</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Winther</span>  <span class="cit-name-given-names">R. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Agrawal</span>  <span class="cit-name-given-names">A. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goodnight</span>  <span class="cit-name-given-names">C. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Alternative definitions of epistasis: dependence and interaction</span>. <abbr class="cit-jnl-abbrev">Trends Ecol. Evol.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">498</span>–<span class="cit-lpage">504</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BEcol.%2BEvol.%26rft.volume%253D16%26rft.spage%253D498%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0169-5347%252801%252902213-3%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0169-5347(01)02213-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000170534800010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1085.60" data-doi="10.1111/j.1469-8137.2010.03415.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wegrzyn</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eckert</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Choi</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stanton</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Association genetics of traits controlling lignin and cellulose biosynthesis in black cottonwood (<em>Populus trichocarpa</em>, Salicaceae) secondary xylem</span>. <abbr class="cit-jnl-abbrev">New Phytol.</abbr> <span class="cit-vol">188</span>: <span class="cit-fpage">515</span>–<span class="cit-lpage">532</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytologist%26rft.stitle%253DNew%2BPhytologist%26rft.aulast%253DWegrzyn%26rft.auinit1%253DJ.%2BL.%26rft.volume%253D188%26rft.issue%253D2%26rft.spage%253D515%26rft.epage%253D532%26rft.atitle%253DAssociation%2Bgenetics%2Bof%2Btraits%2Bcontrolling%2Blignin%2Band%2Bcellulose%2Bbiosynthesis%2Bin%2Bblack%2Bcottonwood%2B%2528Populus%2Btrichocarpa%252C%2BSalicaceae%2529%2Bsecondary%2Bxylem.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1469-8137.2010.03415.x%26rft_id%253Dinfo%253Apmid%252F20831625%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1469-8137.2010.03415.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20831625&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F9%2F1085.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000282225000020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1085.61" data-doi="10.1038/ng1702"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pressoir</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Briggs</span>  <span class="cit-name-given-names">W. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bi</span>  <span class="cit-name-given-names">I. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yamasaki</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">A unified mixed-model method for association mapping that accounts for multiple levels of relatedness</span>. <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">203</span>–<span class="cit-lpage">208</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DYu%26rft.auinit1%253DJ.%26rft.volume%253D38%26rft.issue%253D2%26rft.spage%253D203%26rft.epage%253D208%26rft.atitle%253DA%2Bunified%2Bmixed-model%2Bmethod%2Bfor%2Bassociation%2Bmapping%2Bthat%2Baccounts%2Bfor%2Bmultiple%2Blevels%2Bof%2Brelatedness.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1702%26rft_id%253Dinfo%253Apmid%252F16380716%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1702&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16380716&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F9%2F1085.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000234953200015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1085.62" data-doi="10.1038/ng2110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">J. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Bayesian inference of epistatic interactions in case-control studies</span>. <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">1167</span>–<span class="cit-lpage">1173</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DZhang%26rft.auinit1%253DY.%26rft.volume%253D39%26rft.issue%253D9%26rft.spage%253D1167%26rft.epage%253D1173%26rft.atitle%253DBayesian%2Binference%2Bof%2Bepistatic%2Binteractions%2Bin%2Bcase-control%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng2110%26rft_id%253Dinfo%253Apmid%252F17721534%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng2110&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17721534&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F9%2F1085.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000249122400030&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/2/9/1085.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/2/9/1129" title="Hog1 Controls Global Reallocation of RNA Pol II upon Osmotic Shock in Saccharomyces cerevisiae" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/2/10/1137" title="Competition Between Conjugation and M13 Phage Infection in Escherichia coli in the Absence of Selection Pressure: A Kinetic Study" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/2/9"><span>Volume 2 Issue 9, September 2012</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/2/9.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/2/9.cover-source.jpg?itok=JMb6jwWa" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 2 (9)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/2/9/1085" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-97GSyhFYgtq4QAcqJ2KivbW59qm1pYN2lUcdmZ-FjfI" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/genomic-selection" class="" data-icon-position="" data-hide-link-title="0">Genomic Selection</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/2/9/1085" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/2/9/1085" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/2/9/1085" class="active" data-icon-position="" data-hide-link-title="0">Predicting Adaptive Phenotypes From Multilocus Genotypes in Sitka Spruce (Picea sitchensis) Using Random Forest</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459328" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-BzppH025U5y3BLW-imGMbMSL63aovrr2-qqwM29BUAw" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/2/9/1085.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/2/9/1085" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form--Cat-WbAN2weNkza5uafzf0S8EMoSa3etP6dZr8U75Y" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/2/9/1085" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/2/9/1085.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459328" id="node459328--2" data-pisa="ggg;2/9/1085" data-pisa-master="ggg;2/9/1085" data-apath="/ggg/2/9/1085.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Predicting Adaptive Phenotypes From Multilocus Genotypes in Sitka Spruce (<em>Picea sitchensis</em>) Using Random Forest</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Jason A. Holliday</span>, <span class="highwire-citation-author" data-delta="1">Tongli Wang</span> and <span class="highwire-citation-author" data-delta="2">Sally Aitken</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2012 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 2 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1085-1093;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.002733 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459328--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jason A. Holliday </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Forest Resources and Environmental Conservation, Virginia Polytechnic Institute and State University, Blacksburg, Virginia 24061, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jason%2BA.%2BHolliday%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Holliday%20JA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJason%2BA.%2BHolliday%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jah1@vt.edu" class="" data-icon-position="" data-hide-link-title="0">jah1@vt.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Tongli Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Forest Sciences, University of British Columbia, Vancouver, BC, Canada, V6T 1Z4</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tongli%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATongli%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Sally Aitken </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Forest Sciences, University of British Columbia, Vancouver, BC, Canada, V6T 1Z4</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sally%2BAitken%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Aitken%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASally%2BAitken%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/2/9/1085" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/2/9/1085&amp;title=Predicting%20Adaptive%20Phenotypes%20From%20Multilocus%20Genotypes%20in%20Sitka%20Spruce%20%28Picea%20sitchensis%29%20Using%20Random%20Forest" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/2/9/1085&amp;title=Predicting%20Adaptive%20Phenotypes%20From%20Multilocus%20Genotypes%20in%20Sitka%20Spruce%20%28Picea%20sitchensis%29%20Using%20Random%20Forest" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/2/9/1085&amp;title=Predicting%20Adaptive%20Phenotypes%20From%20Multilocus%20Genotypes%20in%20Sitka%20Spruce%20%28Picea%20sitchensis%29%20Using%20Random%20Forest" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/2/9/1085&amp;text=Predicting%20Adaptive%20Phenotypes%20From%20Multilocus%20Genotypes%20in%20Sitka%20Spruce%20%28Picea%20sitchensis%29%20Using%20Random%20Forest" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/2/9/1085&amp;title=Predicting%20Adaptive%20Phenotypes%20From%20Multilocus%20Genotypes%20in%20Sitka%20Spruce%20%28Picea%20sitchensis%29%20Using%20Random%20Forest" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/2/9/1085&amp;t=Predicting%20Adaptive%20Phenotypes%20From%20Multilocus%20Genotypes%20in%20Sitka%20Spruce%20%28Picea%20sitchensis%29%20Using%20Random%20Forest" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/2/9/1085&amp;title=Predicting%20Adaptive%20Phenotypes%20From%20Multilocus%20Genotypes%20in%20Sitka%20Spruce%20%28Picea%20sitchensis%29%20Using%20Random%20Forest" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/2/9/1085&amp;title=Predicting%20Adaptive%20Phenotypes%20From%20Multilocus%20Genotypes%20in%20Sitka%20Spruce%20%28Picea%20sitchensis%29%20Using%20Random%20Forest" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459328/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459328" id="node459328--3" data-pisa="ggg;2/9/1085" data-pisa-master="ggg;2/9/1085" data-apath="/ggg/2/9/1085.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Predicting Adaptive Phenotypes From Multilocus Genotypes in Sitka Spruce (<em>Picea sitchensis</em>) Using Random Forest</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Jason A. Holliday</span>, <span class="highwire-citation-author" data-delta="1">Tongli Wang</span> and <span class="highwire-citation-author" data-delta="2">Sally Aitken</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2012 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 2 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1085-1093;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.002733 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459328--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jason A. Holliday </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Forest Resources and Environmental Conservation, Virginia Polytechnic Institute and State University, Blacksburg, Virginia 24061, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jason%2BA.%2BHolliday%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Holliday%20JA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJason%2BA.%2BHolliday%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jah1@vt.edu" class="" data-icon-position="" data-hide-link-title="0">jah1@vt.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Tongli Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Forest Sciences, University of British Columbia, Vancouver, BC, Canada, V6T 1Z4</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tongli%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATongli%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Sally Aitken </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Forest Sciences, University of British Columbia, Vancouver, BC, Canada, V6T 1Z4</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sally%2BAitken%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Aitken%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASally%2BAitken%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1085&amp;rft.epage=1093&amp;rft.atitle=Predicting%20Adaptive%20Phenotypes%20From%20Multilocus%20Genotypes%20in%20Sitka%20Spruce%20%28Picea%20sitchensis%29%20Using%20Random%20Forest&amp;rft.volume=2&amp;rft.issue=9&amp;rft.date=2012-09-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Holliday%2C+Jason+A.&amp;rft.au=Wang%2C+Tongli&amp;rft.au=Aitken%2C+Sally"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459328/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459328/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459328/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459328/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459328/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459328/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459328/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459328/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459328/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459328/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459328/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459328/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490720" id="node490720" data-pisa="ggg;8/9/3019" data-pisa-master="ggg;g3.118.200430" data-apath="/ggg/8/9/3019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/3019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">A Bayesian Decision Theory Approach for Genomic Selection</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490724" id="node490724" data-pisa="ggg;8/9/2889" data-pisa-master="ggg;g3.118.200311" data-apath="/ggg/8/9/2889.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/2889" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Optimising Genomic Selection in Wheat: Effect of Marker Density, Population Size and Population Structure on Prediction Accuracy</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490727" id="node490727" data-pisa="ggg;8/9/3039" data-pisa-master="ggg;g3.118.200435" data-apath="/ggg/8/9/3039.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/3039" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">BGGE: A New Package for Genomic-Enabled Prediction Incorporating Genotype × Environment Interaction Models</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Genomic%20Selection" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Genomic Selection</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/2/9/1085" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-2" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-6" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/2/9/1085.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/2/9/1085.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/2/9/1085.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/2/9/1085.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__c-64NG7CB_fwZbyex-B49GkX1AvVESfu7M9yxTNDG_o__Uvi1liJMEKbHtHHv_1fo_FCLvimvbq9jMwPcCPR364Q__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
