<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/2/9/1067" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/2/9/1067.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/2/9/1067.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/2/9/1067.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/2/9.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/2/9/1067/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/2/9/1067.atom" />
<meta name="HW.pisa" content="ggg;2/9/1067" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Uncovering Networks from Genome-Wide Association Studies via Circular Genomic Permutation" />
<meta name="DC.Identifier" content="10.1534/g3.112.002618" />
<meta name="DC.Date" content="2012-09-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2012 Cabrera et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Genome-wide association studies (GWAS) aim to detect single nucleotide polymorphisms (SNP) associated with trait variation. However, due to the large number of tests, standard analysis techniques impose highly stringent significance thresholds, leaving potentially associated SNPs undetected, and much of the trait genetic variation unexplained. Pathway- and network-based methodologies applied to GWAS aim to detect associations missed by standard single-marker approaches. The complex and non-random architecture of the genome makes it a challenge to derive an appropriate testing framework for such methodologies. We developed a rapid and simple permutation approach that uses GWAS SNP association results to establish the significance of pathway associations while accounting for the linkage disequilibrium structure of SNPs and the clustering of functionally related elements in the genome. All SNPs used in the GWAS are placed in a “circular genome” according to their location. Then the complete set of SNP association P values are permuted by rotation with respect to the genomic locations of the SNPs. Once these “simulated” P values are assigned, the joint gene P values are calculated using Fisher’s combination test, and the association of pathways is tested using the hypergeometric test. The circular genomic permutation approach was applied to a human genome-wide association dataset. The data consists of 719 individuals from the ORCADES study genotyped for ∼300,000 SNPs and measured for 51 traits ranging from physical to biochemical measurements. KEGG pathways (n = 225) were used as the sets of pathways to be tested. Our results demonstrate that the circular genomic permutations provide robust association P values. The non-permuted hypergeometric analysis generates ∼1400 pathway-trait combination results with an association P value more significant than P ≤ 0.05, whereas applying circular genomic permutation reduces the number of significant results to a more credible 40% of that value. The circular permutation software (“genomicper”) is available as an R package at &lt;http://cran.r-project.org/&gt;." />
<meta name="DC.Contributor" content="Claudia P. Cabrera" />
<meta name="DC.Contributor" content="Pau Navarro" />
<meta name="DC.Contributor" content="Jennifer E. Huffman" />
<meta name="DC.Contributor" content="Alan F. Wright" />
<meta name="DC.Contributor" content="Caroline Hayward" />
<meta name="DC.Contributor" content="Harry Campbell" />
<meta name="DC.Contributor" content="James F. Wilson" />
<meta name="DC.Contributor" content="Igor Rudan" />
<meta name="DC.Contributor" content="Nicholas D. Hastie" />
<meta name="DC.Contributor" content="Veronique Vitart" />
<meta name="DC.Contributor" content="Chris S. Haley" />
<meta name="article:published_time" content="2012-09-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Uncovering Networks from Genome-Wide Association Studies via Circular Genomic Permutation" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Genome-wide association studies (GWAS) aim to detect single nucleotide polymorphisms (SNP) associated with trait variation. However, due to the large number of tests, standard analysis techniques impose highly stringent significance thresholds, leaving potentially associated SNPs undetected, and much of the trait genetic variation unexplained. Pathway- and network-based methodologies applied to GWAS aim to detect associations missed by standard single-marker approaches. The complex and non-random architecture of the genome makes it a challenge to derive an appropriate testing framework for such methodologies. We developed a rapid and simple permutation approach that uses GWAS SNP association results to establish the significance of pathway associations while accounting for the linkage disequilibrium structure of SNPs and the clustering of functionally related elements in the genome. All SNPs used in the GWAS are placed in a “circular genome” according to their location. Then the complete set of SNP association &lt;i&gt;P&lt;/i&gt; values are permuted by rotation with respect to the genomic locations of the SNPs. Once these “simulated” &lt;i&gt;P&lt;/i&gt; values are assigned, the joint gene &lt;i&gt;P&lt;/i&gt; values are calculated using Fisher’s combination test, and the association of pathways is tested using the hypergeometric test. The circular genomic permutation approach was applied to a human genome-wide association dataset. The data consists of 719 individuals from the ORCADES study genotyped for ∼300,000 SNPs and measured for 51 traits ranging from physical to biochemical measurements. KEGG pathways (n = 225) were used as the sets of pathways to be tested. Our results demonstrate that the circular genomic permutations provide robust association &lt;i&gt;P&lt;/i&gt; values. The non-permuted hypergeometric analysis generates ∼1400 pathway-trait combination results with an association &lt;i&gt;P&lt;/i&gt; value more significant than &lt;i&gt;P&lt;/i&gt; ≤ 0.05, whereas applying circular genomic permutation reduces the number of significant results to a more credible 40% of that value. The circular permutation software (“genomicper”) is available as an R package at http://cran.r-project.org/.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2012/09/01" />
<meta name="citation_mjid" content="ggg;2/9/1067" />
<meta name="citation_id" content="2/9/1067" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/2/9/1067" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/2/9/1067.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/2/9/1067.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/2/9/1067.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.112.002618" />
<meta name="citation_pmid" content="22973544" />
<meta name="citation_volume" content="2" />
<meta name="citation_issue" content="9" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="1067" />
<meta name="citation_lastpage" content="1075" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Claudia P. Cabrera" />
<meta name="citation_author_institution" content="MRC Human Genetics Unit, Medical Research Council, Institute of Genetics and Molecular Medicine, University of Edinburgh, Edinburgh EH4 2XU, Scotland, United Kingdom" />
<meta name="citation_author" content="Pau Navarro" />
<meta name="citation_author_institution" content="MRC Human Genetics Unit, Medical Research Council, Institute of Genetics and Molecular Medicine, University of Edinburgh, Edinburgh EH4 2XU, Scotland, United Kingdom" />
<meta name="citation_author" content="Jennifer E. Huffman" />
<meta name="citation_author_institution" content="MRC Human Genetics Unit, Medical Research Council, Institute of Genetics and Molecular Medicine, University of Edinburgh, Edinburgh EH4 2XU, Scotland, United Kingdom" />
<meta name="citation_author" content="Alan F. Wright" />
<meta name="citation_author_institution" content="MRC Human Genetics Unit, Medical Research Council, Institute of Genetics and Molecular Medicine, University of Edinburgh, Edinburgh EH4 2XU, Scotland, United Kingdom" />
<meta name="citation_author" content="Caroline Hayward" />
<meta name="citation_author_institution" content="MRC Human Genetics Unit, Medical Research Council, Institute of Genetics and Molecular Medicine, University of Edinburgh, Edinburgh EH4 2XU, Scotland, United Kingdom" />
<meta name="citation_author" content="Harry Campbell" />
<meta name="citation_author_institution" content="Centre for Population Health Sciences, University of Edinburgh, Edinburgh EH8 9AG, Scotland, United Kingdom" />
<meta name="citation_author" content="James F. Wilson" />
<meta name="citation_author_institution" content="Centre for Population Health Sciences, University of Edinburgh, Edinburgh EH8 9AG, Scotland, United Kingdom" />
<meta name="citation_author" content="Igor Rudan" />
<meta name="citation_author_institution" content="Centre for Population Health Sciences, University of Edinburgh, Edinburgh EH8 9AG, Scotland, United Kingdom" />
<meta name="citation_author" content="Nicholas D. Hastie" />
<meta name="citation_author_institution" content="MRC Human Genetics Unit, Medical Research Council, Institute of Genetics and Molecular Medicine, University of Edinburgh, Edinburgh EH4 2XU, Scotland, United Kingdom" />
<meta name="citation_author" content="Veronique Vitart" />
<meta name="citation_author_institution" content="MRC Human Genetics Unit, Medical Research Council, Institute of Genetics and Molecular Medicine, University of Edinburgh, Edinburgh EH4 2XU, Scotland, United Kingdom" />
<meta name="citation_author" content="Chris S. Haley" />
<meta name="citation_author_institution" content="MRC Human Genetics Unit, Medical Research Council, Institute of Genetics and Molecular Medicine, University of Edinburgh, Edinburgh EH4 2XU, Scotland, United Kingdom" />
<meta name="citation_author_institution" content="The Roslin Institute and R(D)SVS, University of Edinburgh, Easter Bush EH25 9RG, Scotland, United Kingdom" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=Y. S. Aulchenko;citation_author=S. Ripke;citation_author=A. Isaacs;citation_author=C. M. van Duijn;citation_title=GenABEL: an R library for genome-wide association analysis;citation_volume=23;citation_year=2007;citation_pmid=17384015;citation_doi=10.1093/bioinformatics/btm108" />
<meta name="citation_reference" content="citation_journal_abbrev=BMC Med Genomics;citation_author=D. Ballard;citation_author=C. Abraham;citation_author=J. Cho;citation_author=H. Zhao;citation_title=Pathway analysis comparison using Crohn's disease genome wide association studies.;citation_pages=25-25;citation_volume=3;citation_year=2010;citation_pmid=20584322;citation_doi=10.1186/1755-8794-3-25" />
<meta name="citation_reference" content="citation_journal_title=Hum. Mol. Genet.;citation_author=S. E. Baranzini;citation_author=N. W. Galwey;citation_author=J. Wang;citation_author=P. Khankhanian;citation_author=R. Lindberg;citation_title=Pathway and network-based analysis of genome-wide association studies in multiple sclerosis;citation_volume=18;citation_year=2009;citation_pmid=19286671;citation_doi=10.1093/hmg/ddp120" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. Callegaro;citation_author=D. Basso;citation_author=S. Bicciato;citation_title=A locally adaptive statistical procedure (LAP) to identify differentially expressed chromosomal regions;citation_volume=22;citation_year=2006;citation_pmid=16951291;citation_doi=10.1093/bioinformatics/btl455" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=H. Caron;citation_author=B. v. Schaik;citation_author=M. d. Mee;citation_author=F. Baas;citation_author=G. Riggins;citation_title=The Human Transcriptome Map: Clustering of Highly Expressed Genes in Chromosomal Domains;citation_pages=1289-1292;citation_volume=291;citation_year=2001;citation_issue=5507;citation_pmid=11181992;citation_doi=10.1126/science.1056794" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=W. M. Chen;citation_author=G. R. Abecasis;citation_title=Family-based association tests for genomewide association scans.;citation_pages=913-926;citation_volume=81;citation_year=2007;citation_issue=5;citation_pmid=17924335;citation_doi=10.1086/521580" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=A. D’Addabbo;citation_author=O. Palmieri;citation_author=A. Latiano;citation_author=V. Annese;citation_author=S. Mukherjee;citation_title=RS-SNP: a random-set method for genome-wide association studies.;citation_pages=166-166;citation_volume=12;citation_year=2011;citation_pmid=21450072;citation_doi=10.1186/1471-2164-12-166" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=J. Dupuis;citation_author=C. Langenberg;citation_author=I. Prokopenko;citation_author=R. Saxena;citation_author=N. Soranzo;citation_title=New genetic loci implicated in fasting glucose homeostasis and their impact on type 2 diabetes risk.;citation_pages=105-116;citation_volume=42;citation_year=2010;citation_issue=2;citation_pmid=20081858;citation_doi=10.1038/ng.520" />
<meta name="citation_reference" content="citation_journal_title=PLOS Comput. Biol.;citation_author=T. C. Freeman;citation_author=L. Goldovsky;citation_author=M. Brosch;citation_author=S. van Dongen;citation_author=P. Maziere;citation_title=Construction, visualisation, and clustering of transcription networks from microarray expression data;citation_pages=e206;citation_volume=3;citation_year=2007;citation_doi=10.1371/journal.pcbi.0030206" />
<meta name="citation_reference" content="citation_journal_title=European journal of human genetics :  EJHG;citation_journal_abbrev=Eur J Hum Genet;citation_author=B. L. Fridley;citation_author=J. M. Biernacka;citation_title=Gene set analysis of SNP data: benefits, challenges, and future directions.;citation_pages=837-843;citation_volume=19;citation_year=2011;citation_issue=8;citation_pmid=21487444;citation_doi=10.1038/ejhg.2011.57" />
<meta name="citation_reference" content="citation_journal_title=Journal of Leukocyte Biology;citation_journal_abbrev=J. Leukoc. Biol.;citation_author=E. Garcia-Garcia;citation_author=C. Rosales;citation_title=Signal transduction during Fc receptor-mediated phagocytosis;citation_pages=1092-1108;citation_volume=72;citation_year=2002;citation_issue=6;citation_pmid=12488490" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=D. L. Ge;citation_author=K. L. Zhang;citation_author=A. C. Need;citation_author=O. Martin;citation_author=J. Fellay;citation_title=WGAViewer: software for genomic annotation of whole genome association studies;citation_volume=18;citation_year=2008;citation_pmid=18256235;citation_doi=10.1101/gr.071571.107" />
<meta name="citation_reference" content="citation_journal_title=New England Journal of Medicine;citation_journal_abbrev=NEJM;citation_author=M. H. Gollob;citation_author=M. S. Green;citation_author=A. S. L. Tang;citation_author=T. Gollob;citation_author=A. Karibe;citation_title=Identification of a Gene Responsible for Familial Wolff-Parkinson-White Syndrome;citation_pages=1823-1831;citation_volume=344;citation_year=2001;citation_issue=24;citation_pmid=11407343;citation_doi=10.1056/NEJM200106143442403" />
<meta name="citation_reference" content="citation_journal_title=J. R. Stat. Soc. Ser. C Appl. Stat.;citation_author=J. A. Hartigan;citation_author=M. A. Wong;citation_title=A K-means clustering algorithm;citation_pages=100-108;citation_volume=28;citation_year=1979;citation_doi=10.2307/2346830" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=L. A. Hindorff;citation_author=P. Sethupathy;citation_author=H. A. Junkins;citation_author=E. M. Ramos;citation_author=J. P. Mehta;citation_title=Potential etiologic and functional implications of genome-wide association loci for human diseases and traits;citation_volume=106;citation_year=2009;citation_pmid=19474294;citation_doi=10.1073/pnas.0903103106" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=P. Holmans;citation_author=E. K. Green;citation_author=J. S. Pahwa;citation_author=M. Ferreira;citation_author=S. M. Purcell;citation_title=Gene ontology analysis of GWA study data sets provides insights into the biology of bipolar disorder.;citation_pages=13-24;citation_volume=85;citation_year=2009;citation_issue=1;citation_pmid=19539887;citation_doi=10.1016/j.ajhg.2009.05.011" />
<meta name="citation_reference" content="citation_journal_title=Circulation;citation_author=S. A. Hunt;citation_author=W. T. Abraham;citation_author=M. H. Chin;citation_author=A. M. Feldman;citation_author=G. S. Francis;citation_title=ACC/AHA 2005 guideline update for the diagnosis and management of chronic heart failure in the adult;citation_volume=112;citation_year=2005;citation_pmid=16160202;citation_doi=10.1161/CIRCULATIONAHA.105.167586" />
<meta name="citation_reference" content="citation_journal_title=Schizophrenia research;citation_journal_abbrev=Schizophr Res;citation_author=P. Jia;citation_author=L. Wang;citation_author=H. Y. Meltzer;citation_author=Z. Zhao;citation_title=Common variants conferring risk of schizophrenia: a pathway analysis of GWAS data.;citation_pages=38-42;citation_volume=122;citation_year=2010;citation_issue=1-3;citation_pmid=20659789;citation_doi=10.1016/j.schres.2010.07.001" />
<meta name="citation_reference" content="citation_journal_title=Drug Design Development and Therapy;citation_author=R. Kones;citation_title=Rosuvastatin, inflammation, C-reactive protein, JUPITER, and primary prevention of cardiovascular disease - a perspective;citation_pages=383-413;citation_volume=4;citation_year=2010" />
<meta name="citation_reference" content="citation_journal_title=Circulation;citation_journal_abbrev=Circulation;citation_author=G. D. Lopaschuk;citation_author=W. C. Stanley;citation_title=Glucose Metabolism in the Ischemic Heart;citation_pages=313-315;citation_volume=95;citation_year=1997;citation_issue=2;citation_pmid=9008441;citation_doi=10.1161/01.CIR.95.2.313" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=T. A. Manolio;citation_author=F. S. Collins;citation_author=N. J. Cox;citation_author=D. B. Goldstein;citation_author=L. A. Hindorff;citation_title=Finding the missing heritability of complex diseases.;citation_pages=747-753;citation_volume=461;citation_year=2009;citation_issue=7265;citation_pmid=19812666;citation_doi=10.1038/nature08494" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=R. McQuillan;citation_author=A. L. Leutenegger;citation_author=R. Abdel-Rahman;citation_author=C. S. Franklin;citation_author=M. Pericic;citation_title=Runs of homozygosity in European populations.;citation_pages=359-372;citation_volume=83;citation_year=2008;citation_issue=3;citation_pmid=18760389;citation_doi=10.1016/j.ajhg.2008.08.007" />
<meta name="citation_reference" content="citation_journal_title=Cancer Res.;citation_author=I. Menashe;citation_author=D. Maeder;citation_author=M. Garcia-Closas;citation_author=J. D. Figueroa;citation_author=S. Bhattacharjee;citation_title=Pathway analysis of breast cancer genome-wide association study highlights three pathways and one canonical signaling cascade;citation_volume=70;citation_year=2010;citation_pmid=20460509;citation_doi=10.1158/0008-5472.CAN-09-4502" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=Mouse Genome Sequencing Consortium;citation_title=Initial sequencing and comparative analysis of the mouse genome.;citation_pages=520-562;citation_volume=420;citation_year=2002;citation_issue=6915;citation_pmid=12466850;citation_doi=10.1038/nature01262" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=X. J. Mu;citation_author=Z. J. Lu;citation_author=Y. Kong;citation_author=H. Y. K. Lam;citation_author=M. B. Gerstein;citation_title=Analysis of genomic variation in non-coding elements using population-scale sequencing data from the 1000 Genomes Project;citation_volume=39;citation_year=2011;citation_pmid=21596777;citation_doi=10.1093/nar/gkr342" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=C. O’Dushlaine;citation_author=E. Kenny;citation_author=E. A. Heron;citation_author=R. Segurado;citation_author=M. Gill;citation_title=The SNP ratio test: pathway analysis of genome-wide association datasets;citation_volume=25;citation_year=2009;citation_pmid=19620097;citation_doi=10.1093/bioinformatics/btp448" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=H. Ogata;citation_author=S. Goto;citation_author=K. Sato;citation_author=W. Fujibuchi;citation_author=H. Bono;citation_title=KEGG: Kyoto Encyclopedia of Genes and Genomes;citation_volume=27;citation_year=1999;citation_pmid=9847135;citation_doi=10.1093/nar/27.1.29" />
<meta name="citation_reference" content="citation_journal_title=European journal of human genetics :  EJHG;citation_journal_abbrev=Eur J Hum Genet;citation_author=G. Peng;citation_author=L. Luo;citation_author=H. Siu;citation_author=Y. Zhu;citation_author=P. Hu;citation_title=Gene and pathway-based second-wave analysis of genome-wide association studies.;citation_pages=111-117;citation_volume=18;citation_year=2010;citation_issue=1;citation_pmid=19584899;citation_doi=10.1038/ejhg.2009.115" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=S. F. Schaffner;citation_title=The X chromosome in population genetics.;citation_pages=43-51;citation_volume=5;citation_year=2004;citation_issue=1;citation_pmid=14708015;citation_doi=10.1038/nrg1247" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=M. Seifert;citation_author=M. Strickert;citation_author=A. Schliep;citation_author=I. Grosse;citation_title=Exploiting prior knowledge and gene distances in the analysis of tumor expression profiles with extended Hidden Markov Models;citation_volume=27;citation_year=2011;citation_pmid=21511716;citation_doi=10.1093/bioinformatics/btr199" />
<meta name="citation_reference" content="citation_journal_title=JAMA;citation_journal_abbrev=JAMA;citation_author=J. A. Towbin;citation_author=A. M. Lowe;citation_author=S. D. Colan;citation_author=L. A. Sleeper;citation_author=E. J. Orav;citation_title=Incidence, causes, and outcomes of dilated cardiomyopathy in children.;citation_pages=1867-1876;citation_volume=296;citation_year=2006;citation_issue=15;citation_pmid=17047217;citation_doi=10.1001/jama.296.15.1867" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=R. Versteeg;citation_author=B. D. C. van Schaik;citation_author=M. F. van Batenburg;citation_author=M. Roos;citation_author=R. Monajemi;citation_title=The human transcriptome map reveals extremes in gene density, intron length, GC content, and repeat pattern for domains of highly and weakly expressed genes;citation_volume=13;citation_year=2003;citation_pmid=12915492;citation_doi=10.1101/gr.1649303" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=J. B. Veyrieras;citation_author=S. Kudaravalli;citation_author=S. Y. Kim;citation_author=E. T. Dermitzakis;citation_author=Y. Gilad;citation_title=High-resolution mapping of expression-QTLs yields insight into human gene regulation.;citation_pages=e1000214-e1000214;citation_volume=4;citation_year=2008;citation_issue=10;citation_pmid=18846210;citation_doi=10.1371/journal.pgen.1000214" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=K. Wang;citation_author=M. Li;citation_author=M. Bucan;citation_title=Pathway-based approaches for analysis of genomewide association studies.;citation_pages=1278-1283;citation_volume=81;citation_year=2007;citation_issue=6;citation_pmid=17966091;citation_doi=10.1086/522374" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=K. Wang;citation_author=M. Li;citation_author=H. Hakonarson;citation_title=Analysing biological pathways in genome-wide association studies.;citation_pages=843-854;citation_volume=11;citation_year=2010;citation_issue=12;citation_pmid=21085203;citation_doi=10.1038/nrg2884" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=L. Wang;citation_author=P. Jia;citation_author=R. D. Wolfinger;citation_author=X. Chen;citation_author=B. L. Grayson;citation_title=An efficient hierarchical generalized linear mixed model for pathway analysis of genome-wide association studies;citation_volume=27;citation_year=2011;citation_pmid=21266443;citation_doi=10.1093/bioinformatics/btq728" />
<meta name="citation_reference" content="citation_journal_title=Genomics;citation_journal_abbrev=Genomics;citation_author=L. Wang;citation_author=P. Jia;citation_author=R. D. Wolfinger;citation_author=X. Chen;citation_author=Z. Zhao;citation_title=Gene set analysis of genome-wide association studies: methodological issues and perspectives.;citation_pages=1-8;citation_volume=98;citation_year=2011;citation_issue=1;citation_pmid=21565265;citation_doi=10.1016/j.ygeno.2011.04.006" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=Wellcome Trust Case Control Consortium;citation_title=Genome-wide association study of 14,000 cases of seven common diseases and 3,000 shared controls.;citation_pages=661-678;citation_volume=447;citation_year=2007;citation_issue=7145;citation_pmid=17554300;citation_doi=10.1038/nature05911" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=K. Zhang;citation_author=S. Cui;citation_author=S. Chang;citation_author=L. Zhang;citation_author=J. Wang;citation_title=i-GSEA4GWAS: a web server for identification of pathways/gene sets associated with traits by applying an improved gene set enrichment analysis to genome-wide association study;citation_volume=38;citation_year=2010;citation_pmid=20435672;citation_doi=10.1093/nar/gkq324" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Uncovering Networks from Genome-Wide Association Studies via Circular Genomic Permutation" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/2/9.cover-source.jpg" />
<meta name="twitter:description" content="Genome-wide association studies (GWAS) aim to detect single nucleotide polymorphisms (SNP) associated with trait variation. However, due to the large number of tests, standard analysis techniques impose highly stringent significance thresholds, leaving potentially associated SNPs undetected, and much of the trait genetic variation unexplained. Pathway- and network-based methodologies applied to GWAS aim to detect associations missed by standard single-marker approaches. The complex and non-random architecture of the genome makes it a challenge to derive an appropriate testing framework for such methodologies. We developed a rapid and simple permutation approach that uses GWAS SNP association results to establish the significance of pathway associations while accounting for the linkage disequilibrium structure of SNPs and the clustering of functionally related elements in the genome. All SNPs used in the GWAS are placed in a “circular genome” according to their location. Then the complete set of SNP association P values are permuted by rotation with respect to the genomic locations of the SNPs. Once these “simulated” P values are assigned, the joint gene P values are calculated using Fisher’s combination test, and the association of pathways is tested using the hypergeometric test. The circular genomic permutation approach was applied to a human genome-wide association dataset. The data consists of 719 individuals from the ORCADES study genotyped for ∼300,000 SNPs and measured for 51 traits ranging from physical to biochemical measurements. KEGG pathways (n = 225) were used as the sets of pathways to be tested. Our results demonstrate that the circular genomic permutations provide robust association P values. The non-permuted hypergeometric analysis generates ∼1400 pathway-trait combination results with an association P value more significant than P ≤ 0.05, whereas applying circular genomic permutation reduces the number of significant results to a more credible 40% of that value. The circular permutation software (“genomicper”) is available as an R package at &lt;http://cran.r-project.org/&gt;." />
<meta name="og-title" property="og:title" content="Uncovering Networks from Genome-Wide Association Studies via Circular Genomic Permutation" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/2/9/1067" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Genome-wide association studies (GWAS) aim to detect single nucleotide polymorphisms (SNP) associated with trait variation. However, due to the large number of tests, standard analysis techniques impose highly stringent significance thresholds, leaving potentially associated SNPs undetected, and much of the trait genetic variation unexplained. Pathway- and network-based methodologies applied to GWAS aim to detect associations missed by standard single-marker approaches. The complex and non-random architecture of the genome makes it a challenge to derive an appropriate testing framework for such methodologies. We developed a rapid and simple permutation approach that uses GWAS SNP association results to establish the significance of pathway associations while accounting for the linkage disequilibrium structure of SNPs and the clustering of functionally related elements in the genome. All SNPs used in the GWAS are placed in a “circular genome” according to their location. Then the complete set of SNP association P values are permuted by rotation with respect to the genomic locations of the SNPs. Once these “simulated” P values are assigned, the joint gene P values are calculated using Fisher’s combination test, and the association of pathways is tested using the hypergeometric test. The circular genomic permutation approach was applied to a human genome-wide association dataset. The data consists of 719 individuals from the ORCADES study genotyped for ∼300,000 SNPs and measured for 51 traits ranging from physical to biochemical measurements. KEGG pathways (n = 225) were used as the sets of pathways to be tested. Our results demonstrate that the circular genomic permutations provide robust association P values. The non-permuted hypergeometric analysis generates ∼1400 pathway-trait combination results with an association P value more significant than P ≤ 0.05, whereas applying circular genomic permutation reduces the number of significant results to a more credible 40% of that value. The circular permutation software (“genomicper”) is available as an R package at &lt;http://cran.r-project.org/&gt;." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/2/9.cover-source.jpg" />
    <title>Uncovering Networks from Genome-Wide Association Studies via Circular Genomic Permutation | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings, {"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"k656q8vnD-WpqmAM_O5sg87tEkTA008ure1LSXWlcSw","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459322","apath":"\/ggg\/2\/9\/1067.atom","pisa":"ggg;2\/9\/1067","ac":{"\/ggg\/2\/9\/1067.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/2\/9\/1067.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;2\/9\/1067"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;2\/9\/1067"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/2\/9\/1067":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022Q2InjnUYblMkXr3o70rf+hjFr7uobK+CouwcRHlttC4lJxrEZ60W5hXq6hmNExqc2OvEQJhxBvGWmOVyjrQYUk8\\\\\\\/76SlMZO0TQB4j2KEQIDkBZb51ULDI8myeHFl6q+ypq3PrZYgdgQqcbYLvM210HeoZO9oCvefsDsSlvwHcVPNMMszW8\\\\\\\/98SmyH8BTm0iD8pVxDD74i0FtmPuOxYaHl2y+arSKzR2reECQ7F3\\\\\\\/OSDhbSuzn1Q\\\\\\\/j4yJjyBK9Uh0B4NdLdmtGf\\\\\\\/APkq\\\\\\\/VxIErTdj\\\\\\\/97a9IGxBCy0h69ykaDB5Gk6eVAWEZmgbRTvNzyuynmqzR5UsIUw3kNg0HjqN5NRjgIgBGo9yhyRSEM6k52bSm1+z5ONXUePZO+0NVmFOJP5yedfePVBy9yPGsQORitFMXOwUQCK6dhPUd5l5dG7v+FWIycTAZtsyPm7q15kwfqO5PEPon11Ges5iB2VqpHbHhPcIzdzgc7Tn4oG+lkqeuI9K\\\\\\\/UuYGLTcHmlMF39MeNhDN6T3oRTXjZQxSoD7WXm4nLzY1amnC+jnH4VZ60E43fkXlefNCDJpIsgIxWlVMshteKznKWdQC2RP\\\\\\\/l6UqQScbylsVK\\\\\\\/G222watnuvC78hQe9fACJdNS+ywanBnU0BJIcmkClAbWCyZUtGlqlpxhoemBXHEX96QEg02EA5+in7wWdutE7TlHb4h\\\\\\\/eTpYfl6DS9kWjpMFpBrn6A\\\\\\\/wV59YAxPhoAr5FvEI2lMRdFsAa7PPxdvehrL0pt+6e6CPFw1lIhRBa3SztA7RQKCUutcR4Ji95LmfpxGPiipfXthGuyZIfP33+cKF7voQH\\\\\\\/3FfBK78xsQK0LdVURPnJK7ysFma\\\\\\\/+U8PRkQUjVux3dv7VbcV\\\\\\\/9WaiMvCTdLI9NzvkXqzm7rDOkEieNEnYarfmqT4l\\\\\\\/QVvh4l9eAEhXu4HIt\\\\\\\/cgqycLj8iWymtB74GNzDR6FZAAz\\\\\\\/BxuCIakOMjMYjcOg2L+K62ZpMjpDEci0CCkQvFxig+YgnpyZ3gQVwZq7hyJ2KoXRSi4ssQ9i1exxfXgTj7I5bPYIBikIMdS5eWHKiiMAYfLlGufJDegtALzXCnva5xhjfzrHf0tOkjqi5n0dJOjMD+L6hq8QxYrw0V1P+GnLaaARs9HeFpapfSgWzZXKU+RmK7ojMp3uQkPcFvpYDwKD2mFJ8OJIqZjBQPtJEBZIxIMKqeL35XgNj7ZXzppJ3jRy1iG5auYmAPz+B00MjQ+3g2KbFP4z4eGZVYQ\\\\\\\/YfxR8gX2jhxrhJZFmPnxT5\\\u0022,\\\u0022iv\\\u0022:\\\u0022TGtfJsErwjahsSms2taM7w==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00223f03196263cebefc2b23e03cf62d9428741ea0129daa08d1d59617c80493c74a\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022LgqWQQPrQud1F8JoTXUAp\\\\\\\/MH58XFu3y66m\\\\\\\/UBE4A0O\\\\\\\/OAOqtrh9DnJ2bK42a\\\\\\\/OtBnTRaWbdv2XsYBmUYAeJJo8k0KasV1BRjA16BQ2TOaqFLbqrFy\\\\\\\/jnpcG2ch99Q80svOWCai0s6wAQT6sXYwLPUrqz\\\\\\\/84IcG14e1WraU+Dj0fYQXa7iLpHxUFZD9Cv8n4SPvywlNnTaheGPmfxoElRdypvrH6nljQnlSFpRGSTYR2f6GCB+AScQZMnxURe+7L8TTDZ17csHe0T84tqQtWiHqpb7y4Q7S4Q4YRsYV6vwzEHsE33fSw+830G11YYlj6ouYDmctZok0PInt1gU43TXJhmSabPXWXXHaZedjrh69C0BImH98ljVyTmoJ4z9GLXU0m1jH582UIhSjFv+CZuyFd1qzUJ1SunUwd\\\\\\\/0+SZZ7aJOjvdqwX1Fy\\\\\\\/5BcFdWbnd81z56IRfodK3HZlUQoIlu3rVjz3SatZ6YOdeHLA+Ytw5Hv6dT1O7ocvA+9vzMhtfO4Ki8\\\\\\\/lYc2virrsvp8AFmiIyvSpp1HGiaS4hYdiCGoWA8aIZXEkcjQrra679YgQ\\\\\\\/FeK0QhjXBgcvg\\\\\\\/0\\\\\\\/oFCvcZWTML\\\\\\\/\\\\\\\/JRg80b2x62tJ7Vab4DONE22PqfgZpzPh0RZAPWe4MWYjLaiemRqmqAL3C4DnD4knTZKkUBZoK5XklaaCfAAUljOkw8lh4ZmhEO1qgNUbR\\\\\\\/ChpKBtKWD4Ps0VmIXIn2ES6RrboYeDzgEYW8CavuVl659Kn97Syk73RktAGSU9qe6F4jsDeVWvu0UQPrtHJ0RbpOUg+B47bKDmNoADsirdKuGDEJWdzI3aWDEJZXSHA\\\\\\\/Lfa+WpBi5f82zfgSyjd5x6HhstDPn\\\\\\\/Lm6iu4B0Iu8MXzpVY6c\\\\\\\/B2MsWCY3a9juhU4fbxfLPWiFyVFKcZgf5IJ6Xj4LZ4ZQ0EBarACBqs8mY+tE1WqpB9fC8QPFtdSbo4yJWggp8ffLMBcKVk5P1GqjeH7AR\\\\\\\/i40BRZ9dCOdENOIrqfQ0SsQ7vNl1ap3KTTP9jFfcIXraRgaU\\\\\\\/WmFYmF0fO\\\\\\\/2SOUnLJbYt8lB2JGrJb50MpAh4H4YeaW5gSFL+UeBDs1RewiJilFjl+ULecDHay130l7OJbgcv7\\\\\\\/hBpWhjNnyrbH7ENiCil\\\\\\\/GvyDVgd9nevkE82VDKSUpxoKY4v+hYIjrb4GJSU0wuHRYQF7ZKUwJYmHzsI6FL1vjvoSelU56QIDTcfZAkxGjrC2LGdZ2Qq2S7jAZQD64DGaEyV8WtKDdVoXH8nw9+TlqLkUAy4\\\\\\\/veUqddUHraQRN+OJz7epI5E9XFk5s2qs8uS6Vz\\\\\\\/hPWGaHSPgq7grN3yk8k4ysaJ9iAPb1c3GWDQkvgMSeCtljEqlcFlJKHWePlHnunkvlFiyxv0pXhMnNUhtUWmYO9AaCjAOAavHjfjTm8ZycjrXzpCeJAb3IYSLPxxntdh4O8VpSgrYrSqnrT4RokLuBrqvXbQPi+bFS+gd5QAIvQXvx0t8obk1VURYkuq\\\\\\\/RSSzDhKRZDET7c6J3XqnJDxOMQ1ulkjQb3RNUgsst0y2wBzG6eY1sL6QX4VkMAcUMzYRlSeRyBSqvCLAh9bqMEhUlYM1pX092Aa0WHOrhbzccJGqHiUcrc9xIMkK0Tj9R68BR3lFcoABicBpbvYUoqOjk4gaeXMWZ9gnWhmPwwPsGPbRtGRGKWsjyfFR7wpQ0FvRKCZxAWqcdbACyAqIQUZgT+4xqOqJFgtG\\\\\\\/OTQGLGRbzDPiHJlXZuUWoFelk9GSh+bcXtpXarF2Ag2C5pWeWDvkkDPu8dqHpRI3ME3EKjIWtGugzhKFnkeL5yHADkVYqPXY2c5dZ6tm6whkb65KPuBX343qBOaPeJktBy03X\\\\\\\/JoRPtheQDzitrwpmVNGc\\\\\\\/kaggyETX9ax5HvgBijpbHgZwKp1BKxP+YoZ5tDdOjXob425X4mWEBuUacChZwUGQDNUW+fN48l4+DOAIQmUNbnzPuILMW118pMP+nAUlGs4PNojhdPf3r61h1x7XxmPiknqaKn10ZYRJI803OOSCznsL\\\\\\\/IWA5ZwWiatyG4mGgqo3AVgqnPx0GjYtPdXqM7avX2dKsrgGftDZ\\\\\\\/T7WGZ2viCskm1uoU6Fk6IfZERlQiqXmvGeWvWo6KRvciBIGbYVDPTj2iD1zzIHgPskgUuRz34TlRve4otRS9VyJI0mzfNOUz\\\\\\\/oZ4tHa83daxMylEwKjtwIrW+ePX90SI8GlawaVihnFSr+5Aufxe+QjERhwc2ZvUnFdtL5xLrA2lwjDYUSlZjfE9vzEDzU9fkj09DYsEa4in+hgR24YxfOiHJL5q5wVi1KWWGcTbK2cyx683bQ269leyFzq0LZj7O3Nu9M7daeIvdiLz6PwQYli2ojgPku3nuAUgIlSoyZjz8qGsej8VI52J9oBsAtPd3ID65VLXAjoFMUqIWF0R9xBvT\\\\\\\/+49aqTPgmWFN2yGJDOl9aOo1MFQiUmVPB7dKkal62TFC9n\\\\\\\/1liT8GRE38l2nQQKBS5CBTeVq9I+eBMvXd8min4ltSrXmrc0MFNEcF0mvNbW6\\\\\\\/D8bM41TncGzs\\\\\\\/UQgK6One0Vrj3d+QKBG68DLbalbRzS3NXIaDs0Hzg2Fo7g6VzVgIR9NtejNJ9t7DIOLvVWBQwPEbdxwxEhKaowpuoG2zSYN1INYzh7miseiwiQiZG42E8cRbQ1nQcdJrkLZHLB+tp6XQKpy6T5O2fP7IQsuRJ1mH6uJIWUtZOlaZWYqdgFrNzQVnU1z0UQqpcoFH4zoVxOQqzEMrlIaY9xKIB5BBxKRQfYEThHBPNWpJQyA5jROy9IhbgX4EeCBY93L+4T2AmA4fFibLuMG0CZDE8LLHW7EQ9MbT5t8coEdQEMf5rvwIUmWHTFzFYp1ocx5iUUvVbK3h8hu2A\\\\\\\/EkJvPxZIuvP8bD43AjnH64looXL0Uzj3M2RGaBBg\\\\\\\/AMIMI\\\\\\\/OO8tn\\\\\\\/eSLAodCxhmRSSV6ognrUjtZjlgFWi4NnRMgAC\\\\\\\/GIe0mbKkecrdHHJXC1yqRiLOSSPI2ZQLIlWVfgSO9gywfak7HCWC6OgOpE\\\\\\\/iyIqTxVHl3PIMn7ypqxQF8tv8SULqO8D447eItRTZbdlapsG+ZlMtuISDspdR4ntOna5zaElITTD+Tv+7uiz734CMv706NWfUrKl0IIaLzeShplLdW2nvS7RKeRdYGZoveyZEo4daNAKdum\\\\\\\/u7xmsGuIQQpcrQY7HW6aFSM9WIDoyobTbZ9CQRc8qZigTUS1VN7VH4cHtSFwYqdN\\\\\\\/yM7baNU8DuSLV\\\\\\\/KLSnglY2DM7mwPBoS8umx2MZvC6WqXiAviOCf+\\\\\\\/s9ZycnbDkcCLfVmFRmD+yg4Z8AInwPZL83rU6H34upFULkQAcWofSf9CPTjnl10BMuTWbiQMKpfkfLK0WPskuw9iRzKiPrlkQZvrxOoiMWemTfloIz3+j8MQv94ifv4JQCASMaOncRrYnKt3EDp5HKbciR0FJdoYPmEL4YAkFlJNu6z8Le1fPwFiroPI\\\\\\\/Whe2g\\\\\\\/KN\\\\\\\/7xeGOflR0sjIUBYe9maNeqlhXjglJ1aqAVUPt403X81fk\\\\\\\/TUnsZPBtnbKh3mdhO2Qequ92MJkrHR3wjsBadg7sn\\\\\\\/W+Eipxdvz2KACUODCk5vQDvesQiilFrjM3bsUX8FMBq1SlFJ2M2CKLE3cSUfBTUlgx7FG7v1nI6My3uuoHOt0kZqvX7L2YnIUxnwIqcXOd7VOYvXEGU1iubhz9L20g8FoaYg1HnyEZr6p9zm2mUDNNWTzQfSTaxJ4vN+5DNSFUhKWpR68Y7RB6W4ejm3pzOEJ09ziQTGpiIJRoA9ZrocpMxlSFYymzRb0t\\\\\\\/jt0WOfGVjPxD\\\\\\\/RvZUzDRH2W9nO5v7Ygvq89QlGtIp70QdzamLqmlAQVMpMoDvZPU5cBlHHEsivbD3zd1SF7GWMQezBZOPLQJ6j+x2EUil0zFKh2r+IkstJIqykOnNsPlqbQQXOe9G9KJd9LT+RbI0IGTJn2vCT0Ipb6V6ldu6rJ\\\\\\\/1N1tU7Wiysu8nDVnpsFXO8nxH82PNbyARQ7Ua0OWQT2VzA8mRLOuI7mDGt2gcGi5GrF4I+H3QpETC0EAepJieOFLQC7SMWJ\\\\\\\/Sf2hcZX8f1C5yzKVOFc+sMTdywdMLHsxIS5XeBZgd6KmNNtmaDDm8C\\\\\\\/9WO0zCeoBvsY58OlplnMLVZpyuC7t4G1Z4wnzaJxJ32kdSpJLoYLofBpO0+Rrk2VbCjHIIFUFSZPts60L7DbHKaosZvFOpuPUaeeeaMDA2v4CbExRm2SD1pXp6dkRTLQmaQrRrF8KSj3\\\\\\\/HHonCp4zBCZSIRmB3pwiqn0viXr8k4W9t52D9SIoFKwU6yukMVJTnad+oJAS0dbXYO\\\\\\\/jFq4+2623Pzwj6Ul6TL8aQ5oQ8eZGBSYy3NPdlN4w89olzXtnTjYj4ZRf+2pjASigX2t5Ed37AfKR0BMtkglyfx+C2u5cW29JUlZqVk7LLYSQQrVflv9W9IYsZaXsRIl5dc3a3pCMNyovXYffm7ZbXgFItTOzIzNzdYNSsEz7KKJvI3W2CDtfbeQbBKs4E+YT5JnnVT0VUNLfqqh+Lqg+8nPqANxxbyeXYFoq5xQ5vFL5v5+fVz4j98EIYt0dgtB2ad8Vv+hsK5CsQRYqZgQjYBzcFJUtK7X4b8vZcZGLf5wfOnjvf3hMXIJGx44Lek6TA+8WyWAu4YZ9iV56yDwf8\\\\\\\/7i3ygszTzAV5fyt9TgpeGbrFHdmCCLD\\\\\\\/NMdkc01C9Xo9k538bJlJw\\\\\\\/m+cM3AbNEki2TXxMJgr\\\\\\\/w\\\\\\\/E02PyNA9GAkluSGVDCf6QNqdzrLqShBnD83hWFAqbk8asRWNS\\\\\\\/2LabF\\\\\\\/s8MZKmzNsRLhLmgD5J3b33z09IOYlYIURf+5lyK4+DmjUiZubSaKsSPDd\\\\\\\/9uI4RhXdzn4jcjtjFazqs5\\\\\\\/XSopQJ4efgKq4REoU9l\\\\\\\/3n79lec74HUcVXm2j07hif+wx1InUHP+8Xm3l2KfFQLWWPdAQNNVRa2gaDTjUcI26K1RV5eSaj4BggiqQU5Wp\\\\\\\/SxhDtMzQBZyCUgy8CpcqDcWAfv5G85MqyV506VsfGYozvUjwFE2\\\\\\\/ezxcytKe98hU3GVicCa+MsyMzuE4tc55Kx+b9\\\\\\\/zutM6iQRkCqiKfN5YHz02A3nJezEPknlslTL0EAoTQujegIQsLnNr3vSFMk5FScG3mur3BCX5uSbpE79yBALSyeBYqHLF2qfHRb5zbK8WVaoug4Yllyha+\\\\\\\/9cw5paKRxhqnhXJ3nVrivTqoMGz7Nefj7A4ltSvCPaO8snurzssPjMj4AEEvtABazWwbWZzbUk9YCQ487xvDu7hu8Xx24NLhl+RQUp9mBJA+XvjAAIlhtZsfFR2cPm3I+b7dyroXEMalDi2ElYS41HLmAPz74XiHDGN\\\\\\\/OCOZuiQSc5BxTs8GPoObmb+UdXqsbJ5bz229RqjgD74h5r1cneGV3WCb1d4bLrFrozJz3VaOamfLgvbex0KSvT5PNm1L2zIOU7rAjovFLlLC6kjUqyuaNI1Y\\\\\\\/8U454Wdn3UsEiqO8Otsrkp+ESsPlGXYRHT2I4XHfIdKC2Yy3sOXK7MB2mM3Vp3tx1jeqJX1LJeX3Dt5Ey+guXns6X+ijcYcRdRQ6Doe3WAH8C7v1opB5OeupbXN9QBElFvf6OQdyNAadoV0CEIUTZjCy6TbheN1y2cR6eIW1hAFqk5E+kdxv0qW47YekOGyuQT61kCh21aoyG94wp1zOk3Zvksvf0XXyoDF23ul17rZjnKECtmNGY5\\\\\\\/Buy5JZbk+quRYtC\\\\\\\/jJx5x94ABkqjonfknTv4Fk7UlI9gviJc0FlujDONhZwZfCcdZLlM4mRKsotOSWa2egLdtr0yqE+RK+b4n26eE+NyGEmcR2+ULnzTjnBksA9fLYVhPqBRHiwm\\\\\\\/9FOpeldYzcyzDKdQS7x7fYAg8McC7oszOgn\\\\\\\/EkwgT5bMhi+Cmc8RHtGu48rZJ52nO827fa4xIEZwfNNQFl6WM47SCJ5HvRh7BUY36WCZtLg5LTJeZ1LLx7rvE889kGPmIltyuFCBU4kAE9ZIke+wMAybZ4QJ+VN2BI6LXKDxwJGR5NTT680ds3hiYsdKbP8FoZUZDzRxQbRSzSQVeUYd+gS8FUkWadF0pegUHs8fmq5qOnLrVAa86T74Tjx+a+6IJanrV\\\\\\\/zLDihNB8l6Y5sLWowYrVTpc0xhDw\\\\\\\/jT8P8JUydXezPMxmm+Q9ez7+r5okkKRjvqwE10oQJadd68tL60LFWVh9aAHwwL0qRsjGFDmS2nq4rcZN5guI1C3YmAXSt\\\\\\\/Qt8UMwRp6jwwQ\\\\\\\/wfViiZiALWsWmpxN+6813nQwjkI4oG4aLLdVKfRlicbeIBbCAT0u4cispUvifNqOOy+QTiAK3G5lLP9rtaK1Wu70eMq+JJOzY+Um9pipO4lqpq053sIWdY7kfmTUYP8NYv+8Ix4nR3dRWcHniRWpCWTBgP9Z8CtlA6lRCL41c15PVEpYSAoFlOnnNEqBgTze7nO\\\\\\\/im+S+P1pU+0biGo9vvmTrF7BZml\\\\\\\/A80\\\\\\\/\\\\\\\/paPbOX8iZFCvyYNZKn22zDuEFrms98NotOD\\\\\\\/iFQpoJmiI1a5aqNYmRpUwHKZOKSD\\\\\\\/yotAIayNI7dQtK2MiAI\\\\\\\/jZM8Rs742WL2jsqHAA9pMfmJJWrp5G3CennZ+yJFIhiVRIH6gNLXP2pYqhttloNkhdmkcBoX7oMId02C72JXTfRYSHctJNyhY5IrEUoiFhcL0UHjgIEEY2L6NONsydcgXvGSKCtQG41ElJpz4L1ujswjUQi5d0lLp44T+9uMcYIj6U3DupAHxPHDv\\\\\\\/sAr6gOlZzjdHURjRYDgB9LN38M0CvfExTOjxbspqWsZK4sa5jOLpueIbqjKVpv619aHZfn3bn\\\\\\\/qQ9s3mIDeaMap6FFM\\\\\\\/IxMpOKcFLrl7UigrcljZmRtGpPvi51Nv94fX8gtZYNUCr2PBRHuXltgWz34p8ewnvj7Sszld7eLCjxzcHBVNROZemtsi9lnzb\\\\\\\/VgqT1c8sq3ouNGvyAmxd27h5jpye3D85wzdBrDfRxrFgqzpWRG\\\\\\\/74LIy+K1lEOoJInIkMYyqcx\\\\\\\/B7e5qD+3xc\\\\\\\/uXegHlpxjEHLdbdSMF9WLSA9LDCVrjU6MOnrU05Paf0yspbP4FVwpJfM7x2UsPm7J6rQOlL581R4m203yr5a59INgglHN6FWjsnrY6awZZ4rzfQLlKz3v+4wqyw+2MzmmiuuIolS0kU7qxD5Wu7xfvVzdOs6jTYjs3EftIIH0HSSM8v4xhQ5rHCv7dwKzUBzuQzIVPOQaidrazu9Qdcmi6seI3zgTvZTtvJmEOL5Aocf\\\\\\\/yumfoslBYtglusU1aJWAsuwZG4BrweYunH5aBQ7k24KzAXqWgAWpZ7KLrw2DBW1ZczF5cOc4K3rCuptLLhCu9CF1G5CaXNqCMIG0qP8bp90VdTjPmeKYC23Wn5D\\\\\\\/oTM4DougRWrludtw6FYIV4U9vA9PuJnn60gsLHvHmlCmb+8tgNN76McyCKgrk8qqFsdYTDmtYJs4A8lYUo5zjVanNcdxt7DAlcto4Qim3L5TJsP9yxAU5wVdxWlzbl4E9D4v2QC33284d8bQO+IOh4MUsyhYUaSNOIwJwoTvp6Qg8zmbeam8aPyV3dHukHgi\\\\\\\/K+56nm1WaYE+M2y1b3nvlmhS2LGvPAxCKPJn2z8aZCP6WB3U+Sz75maHnXGnk7D16QAPG5tfcNEgto9CQ39B5KEtTjI82XfHiAB5x83ZzdVBq9lOp5vnMzks\\\\\\\/Fdpasn30wHnMAE2dMWEEP\\\\\\\/NDYwtKnHFGlgDChM\\\\\\\/S6mb6Vp8ZeQXD6dli76ifMqWAwrz2wFeG+AefGcRgjn7vglMviQK+w7T5LJZJ1HLNj8GAHzsNFBgR4Dhvb9OAWyNh8yL9RCHKbYF2Pnx4n08ZZwrdmTKkW4+U8UPn0rV2MKbXumr1GBnk7uAJdfA5YXy8HUzYTpgW\\\\\\\/NXFCUJEh6sTUbt2XcaLrf7ng2nqhT1qesOTZQwJMC\\\\\\\/4Q8ihF1Vqur4RrGf5bmTOL4Zo85+HQ4luuPCVKUubD2jt374wj7inDcqZEf6XVO6xVdm7emqKRAFI0es\\\\\\\/x3PHWzpp+zvCBIHuyhtMZLADcw2FqmlP22uTO0kQBN7q1fxY4d\\\\\\\/u59RDe+ovVWrEaQ8bl0H\\\\\\\/1A\\\\\\\/vOvS68h4+FtNliv3sMQOHZmsNW84sadmj5Egky0xWPKNto2e+TT+3qsuUUDeO0VIbw0Iim4e4Ck\\\\\\\/fbzbdDcrpKB72PsEbVTymmLjAP7cDWKieRsgLgxFo8LoYE3H3yCtiGLHwnc3n8xopVx3EWIQ1DCF9qs02fkhKXIImSI3EmFprlrcrtMdjmtiw3tz1+t0\\\\\\\/zwSuFOF9El\\\\\\\/7GflhDxNOKnswqsmJ9r8wglzMgVSrSAbD7sZ1ouxkPhgAEzmWmMiuCdRWqVUm2fIAp7ag35D8C\\\\\\\/IDU2u+pswSD77IgyXt+brop46OQY66QDGJvCqD3om7OTiLojJ8YhZvje+K+He7eV0ZSRdSKXgpDJorQ3FMB78RrAy8CQ6LTP4B8kk\\\\\\\/Tyan54heYmnYcbwIzNDnMGCaEvM9wXFg\\\\\\\/3o\\\\\\\/Hc+cFQfTdRbv0Oj8cFduNO1WHCZejyZHch5CpxlS5Sh\\\\\\\/JUeJDsddDOu73mxSJipAT23bYZhAjwdyBumSBSUCwkQrrfNtF1CCuKFjDjUMTIJ1K0Gsf3neU20sD1KwToPKet9KM6QyzGfXPVkDyoDS5NRn68D9DU3uDqy35NR+0Hg9tQQ5yWI+4EbBDQxenbZ07S1kr3y9UDcsZjbA76EHZNH67huBxEihBpGds\\\\\\\/+AGbfA18ZE03zmfBcUE0VI2WTv8UTlWQwrwNWan2Kh1oaEzJEhitHVrmsW2W5EpUoUOFwzc03x7LvKMipMOP7kdvnb4+6QupbgAGncon2ioHo9EI9xXa6JPcNjXHRiVH8jRWjVWFXrY39r52GYdq\\\\\\\/FCCOP1wWHHxHFN6gPuCJJ3AsQAqrmBQ6sVqgvujDFIGR+boFAfZz\\\\\\\/\\\\\\\/dwCEy3MpZkLMJlVeyYhZ2wTk05EhCEXkMlP2apTix9AKSKuwo5kGmDbW1aXMNZezZgmZ7BJMRoO7gGNwwRXTmBTiQiU2THihdPzrMiY8J\\\\\\\/8SWtpiroNXfd\\\\\\\/owqAuVqGsqBzlvmLB8F+zKLfdHIMfTLxBP\\\\\\\/8GuB3Bg5MQdpTlZBlnSQqsDdy4\\\\\\\/M13HunHh+8MgOmTeGAylF2zmsfOdnbtiTtKZGbW76stv5edjVm7pm5tGRfaR91cDMKF+q7aiim6iKXzcyu3++NcKtLu+jpxE1mKI010fow70Mfm6DQTJ69Rs2p25esQKG8wV\\\\\\\/xTMeX7btRm4KKUM2mjCw\\\\\\\/tZZmsNP9Q+zX8ZXjcVDF\\\\\\\/cBZz08GrlyR5Ib6SCNIitY2RJhCgamOwWl6AS\\\\\\\/eFNUZW3JhGwNrFFERzys9vHUZebEhVCLSsMv3\\\\\\\/5IdM9DC\\\\\\\/m1uJaCVRTBfSZ4CvI+iZKCcdowLfz3gYCcdhCkxgWo0+ktWYKHIld5Dw+bStD+djK93rrugHR3QcoAqlkW3RcZS+DnEvZImuyW3cHaeLeuLHBLdP08OufDJfYdorfuR1P7T33U7kI9Y7DRyZbqe5tBBWxa4ix8G1QGIGNbprxqUDW5PI5TYQFSEjudl4k70HIa2uXM3oRYOkZEf57c8lgKM1dQE89chTx1zjFRy3ocutFSuVDN1lS1HyKe1Ko\\\\\\\/LFMZCnBkvlJzImM8qooqcHkAbH+kWGdQuZH3W1DPSUqUusqA2msGRuG6UJZWWo\\\\\\\/HV1pKOql9bMa+xbfPTN0+nracPXeZT4aJzPm9MDPypJKZ1z9Uq3sUXE8S5bsH8Z6vMqM9GEIjjsIEQe1uiQByJ\\\\\\\/xVYTYl9hMUWP9Qw402Lu+pGg\\\\\\\/MngVmvGwGyJg3OdHKmHbMGwTqquZuXgvDairZce8ho3OF75xyLrW5Iu3YITkPUESYwqgK+1lIGqoUmIxQGCVM3RCTj7Oag\\\\\\\/HCisK5WtmkUArdRoudbpgKcCAJS\\\\\\\/TrQ6HIHZrCPpyhRLNeiguOP\\\\\\\/1692wH8gGnWJiZqx4Hs3i1NXn5qeNONQip9LoyQV8lKfCPECG\\\\\\\/ZzwruYTwgIkcaIgbcijAVeGkRKrKaBpTPBpYKTI7hcTYGLXkutS8bLzlAcFC+gWRs0ryR1dtCPoChbW9KNhWjeM9pj4UzrzU\\\\\\\/EPlYDA4PVFDsoFSYIzxLSSGvJxyr3\\\\\\\/n0o9A3zJk3l\\\\\\\/9AdxVay+8+LWpxMMgjqAqeL\\\\\\\/ES7LQKSrgJ0FtYfqWoiCjTprab8IBfCV9W8QA2Lo9uTOdvymSBvMkO6bXRV570+TCPbah8Hl5YbkwxQnb+riqQSB+u1HJk5c7TuunYVrX7TrMrltZZhINVYQudbcMrCAh6irUvOlFIViWgQx6t37sXwc0eg1ey02CAneXtkh3DN6tvIOUdY\\\\\\\/QdWOGv7vxnYlsdHWd6QAcdiKvTpi4JTKXEYDYljNJyH7JtXRZsM91UWvVI+sKmuZuZfjzJVxTCG4+Ttkj5tBT\\\\\\\/tqZgetvSIPr3flMQN1kPEw9mV1xj3XMVXYbZ49JDI6uu9BQiFyqRXFpQaddAyP2C+I36JlbhbXsq4yDfvRiv1VaHAmAscw1Z8Ye5guUqRRsrGYhkq52Q2whXdvZXdfoc1a65Ukrxm0iya66\\\\\\\/p1\\\\\\\/aEBv7ch+V6TF788fNJ3nqaHUFue\\\\\\\/HCxlGo2+oMy2OHmG0RX2Zg2EyahdxrK11kmUAkN\\\\\\\/U60ehObiBiZi4dNgcef65Fw+\\\\\\\/u2Na33LP\\\\\\\/CVDggYpju9wIJjNZoO3jjmY3LY2lEOVL4dAsxJIoKb+akH+vZb7CHA2qWoBMSsj0bT71UFP2e7ZO8mwgc3G3mOzTsX\\\\\\\/DZpvotDBhf4x9jGFLgLHiNTvmv2WlDN1IlfC4QyheF2q5OTjZli+u5NvqeTHUwbKtlcvjE72oKtptShrRXetkOMlkRj7KKusdsTQGN1OywCx6m\\\\\\\/d7gWNFGJztH\\\\\\\/CcL4nYRuKFiB0mjN3ckoZj0s61wJ76SJBYAs7+rvlzW57zU838\\\\\\\/y5kqtZ0e7WVJ9qus9mts41C5kZs3DO1SI+IF4G4XoS8agwCSAfM+Z4FlV2k6A7mAQlBj0sFWr9gz3E3l8WKmQxQwzRI1ZT06d0Fs7KRKMM8aczryBPdIV0j2MPk+wWwvCPKcdBMq4XSPODmhas+mcs351YCGEqeJ\\\\\\\/yymjGwCQzhbxzEMd7f0tbCwirRqT76Dck5IjpMtkP\\\\\\\/xCjs2CCoJOLPdx0ldVh17jBJaRec5MQMR3C81WlAw6Jll\\\\\\\/lHNetvoyf4E3L6N2qC9quHcKtH9MXrzh2G3UKdd4DoZdkgkD750MSzgiezJ5B5NjKuvMUk75SD1fTenoC6gHVoB47mobZBryy+ktbOiy9EMY\\\\\\\/JSIz3gpumbn4V1nQwGeRhWI4Xj3Q+1fRrp2okVqSov+6\\\\\\\/J5duP6P803L30GvFxH6Jeg8abvN6l20I5fyLXjwVGbAU3LRHWvIQbrnCID6IB2KcPpEsrva5\\\\\\\/emsf5q85Vdfgv4DUdtB46MJjE4Yelov53JaFNYxNxRUolDGYciSOZpGl+QYOYwDAlYPhbLVxMOJ\\\\\\\/5jYiFcDPDm1Ho+cBL\\\\\\\/yQorA7uYKpXF30cMk47\\\\\\\/Lkh3aoP5JgpwiMMhMf7i5pwzWfb+g56x8YIsI4Yy4uG4oMbRLCPQdy+QzPh7pODQr7+GvKYi8S4+uruP9GyVbkjb1obzpS0zJoxCFPvtpsOB3Hj7T4Mx1gWnYGpU6DLX8tO23gzgCRRn3\\\\\\\/ZYdSjtWtUKeMOM19N6ZgpImDDwYZ6ueUbC8xa5JLCRUmlR7vpQebbpssAKys9+C\\\\\\\/ziinw75WZdt7PyZR65wpq4XSUr8IFulTIcIb8lxHwel2e1ug1Oq0DdiXbWHNXVpHS48CJ0KPjGSE4F1nkQUOaDdq53xt8IQTjwfVvpsFbFqNJXFRym7OkhzRoeT6vHg25T9pnvFE0zK6ZyI7HxdBjcgfl9hJLNL5pzrYeCnk6GefHKbkTB6UvpCVZZjhRWEV8LkWLzXi0S2WYwbs\\\\\\\/gUUAlcG3FP+C3kL+g3C2l8WY3XFOw5S5eNydNz2cNFpXOHYZAVVUmwnaCmu9c9+vKhqe+cLkCcf9kFOcTxbpOBOO+1o4PZF8NQJpe2tjhns8d\\\\\\\/XK3W8kZz5BE7oAQxxhO36UFz3lIkYZEUdBYrD0DkOReLLXQqo+7TkaZubG9jd\\\\\\\/KhgOwwqcNUABHJ0HUh4IZ0FEuFN0RBPJbHGIxdqT7ipwOG3aW3m5JOrWRSFOX1xWUYzgsIbt6xohui+wypa\\\\\\\/3esAnMT24gNPDn0C60+faWc3yH6hitnAoJzWctv5RL1ZmADH6DOgRvbtkxxyDy3mC37I3C3Yelk45rS9SUh+kR3iwTrQ3nPdemU5+Rt4DxlKLJkLZJxVl9ie61Lv+moEFwRgumRmUIUva8\\\\\\\/OPzcnoRav4QWfmjLbTdkgq7AfMknk6HwVr9xRdomWZenEARgcZVJX5NMjQid100g8k6cM3Hsb9WR3bhn50XMYb3GBtmvQnrUvueFZCLCtEP7yABZhzXviEa61+cda8APbis7z5J1D4iatSXOn3NTZw0Igr9zA55fqlQiexDupWwUeT+12svtPFNHA3SL+xTYVMHVSIa6uL0oqdo2UCoMu\\\\\\\/xRCs++tNgW7Y2\\\\\\\/\\\\\\\/c37JMZilltOcmBGcHbXOLOi5T13unUGbJ5HJV8OOqf84OYsogjQOPbLKo+f5hN0YWRMfOa2JS1ZFhL2gMh8CLG7sQYjJtKiIRG7Zn7oMeSBfODbVos+22xMrGYrNxkwhUcOHeqFPh6n5yzAaFG3tZRxqGKZC0GEG0ZOoUO0JzrGUK\\\\\\\/xtBrZjAnENsHarLShEz9cISImXShFZiGxWY7rATWatM6OQdrl6vgS8obQE6r7uW9zPXHWOt4QnCO8d37NOgrNplWUv1ZOH\\\\\\\/Vz2CLSThcRpnWItloJCrdL4\\\\\\\/YjWH7W3CD458eanZNjhGvRQk7u+dnrDCNc\\\\\\\/buVZiqjCzjGqZK3RfOSZkmW\\\\\\\/j+7XvGGljziWPs5ucwCdZFVC2Ms8y1uJJOWj5MWpAI3YU3F5R\\\\\\\/28zhiRoVwjpo3kzIf0+JjnXaKY45LoqLLLo37z0pMqc\\\\\\\/VmVFWuB5S9rpOFR7R86KkiwiyifOXVePK5joTL8zB6yJ7y4R\\\\\\\/MzxAI6L9EmEAQAqNQ\\\\\\\/0qdMaCzTeK5BZ+J1sohXQGPhLAX\\\\\\\/BbdA9X0JUCPMq+aBidD+QezUfeAIil2kCnLQaXyZDEH2Bt+wTnaabf+9kUrQOgU2X542ItpljHk3PiiEewSd37l0IcvNegajXiRoRmKuFHWYKaeLBhyEZ\\\\\\\/59mDWCaHKLFa+GgOKGtmX9Tf1Qd9p9GYH3aji7Lo9b34DztkkxfNayOfVdPgtaqqzk9sErjywe1Oi5p0qJetOYK7oqQIo5FSKqIvmEIhq0OlZJpX+KiYL2vbF4lA4CNV9BJ\\\\\\\/xsZ0bwS3DhWV4LfgZG9YO\\\\\\\/X4c+AVbsJF4RXOapIupn1DZIr58OblrUaqANbZnxstsgaEDVgIOHJAOsXWHE0JA1PaD\\\\\\\/pckTuN8NPLHqH2\\\\\\\/J14EpV12fO25BWfF3HROr55ZHmqvc3s42pMp4AkBddkm28ifxHd+NgI4vJtP0Be6YAVuXu0lU2hFk9CdqJTuoVcKi+0asyj+NE2\\\\\\\/RVKgwi3jAr92o9u7IapJHq1uAf\\\\\\\/ucDge8Jrcpk5FyVdkb3lpiqK73ZzRKqXI6mfz2W55QhVaURGjpbW+MxL0Jvtz1YkBoWOyOGm7lc9VlHrR41I87h+ESxUHNJ7L8+dSmi1VdgnK2C8Xm9qywy+EVfeNAhOTxG30lO9QHTNDE3ZVKFrYT528NWSJntWaPeeLJQ+vdE9omMdoHmVIiZS\\\\\\\/1c\\\\\\\/o24kwUX93kFZl+aeADcBgP9doYRGxkCzMQUOxcQ51\\\\\\\/wp2MOhAYFijtt5GgBMA32bldIBT69JEm8rSCKcFlJmejJYGQrE2gYuzHfd3BYChEjf793GekPwUoBP5jMRNw5BdUVvzGrgGHqOJQ7oRs4w8xzHUs6+IRb6eAEJCGgCv1jlH\\\\\\\/mdpi8xL4qb+20ejpHeoeB26Hw5bHMBA8nVdwVttNGhdQW6cQH6pQn+ctpvYuLMLC9mKnZn+BYJHb0H7BMuLA1KvAHX\\\\\\\/eehLpItRnB\\\\\\\/KUPGt+X\\\\\\\/OZpj6NWd1YHzNoxy2zpRLmTX1\\\\\\\/j0FOWWeCrVuomn\\\\\\\/+1xA+Pzzn6+kA57\\\\\\\/ulx\\\\\\\/nLKUskbxo\\\\\\\/ZAZMgYKzctZFMaHUlVarfL+QiFajUHSka\\\\\\\/D5p1UwgmVXp\\\\\\\/s\\\\\\\/dy9KUQC1Pu8navIt9c7L8rSIi\\\\\\\/vl395BXGIHS\\\\\\\/SqKcqeErOaJI0mKh4\\\\\\\/ubQYMvi8O5a+toNGiY2QCZsCCGeBA+owSZO3rumYb7cr0r60j2xChzmjrVl+\\\\\\\/jUJOZJNpGoOxfgHJjBBfOGFHF8O2qrJyuvujQvMpqLd\\\\\\\/hra2rb+\\\\\\\/w6jbMzAiw6qf6N55xw+EWZx751FM2oJcX+YrFiBn2voGD\\\\\\\/G5nmDdeUfC\\\\\\\/\\\\\\\/cZwAzfXPHekDNvHmz+bQ+1MTRQK\\\\\\\/oqfVPHAeVZ8fBmKKF8fT7GYSXomyqdPAjccXwUO\\\\\\\/F7OCNwJ8xVOMJTNWY6mzLyB1yuafFs2ug+okPFAYoDjE95h\\\\\\\/gw0gEUWUSBqTIpv7VP6hIi6mV+JOgGRdd6eOjGH0HqP\\\\\\\/b8BABRVHXKU2Xn6Y4ZtMLEEu7A0AR7LHQ04Ggsuz837WkqPFI2yfHVwr4gbRe\\\\\\\/l8C3cdldDWK1Ann8uhJsULjRSpUEuJvlzuhctP53h8ZhPQzeZCh9F2q96Rs6MuL0cktWUCfuilcHL8yXx1OUVZg1NJgeRuHklbiDZC4KH8i01OpXt8LzjfCKx33OVejJ9+t2FVr9Y3pndYK5u7vt4rXh4CH49orqcamZ9cvd93fxls8idY9tniQx2Kyhinbj0p9LiV3GGrVhazsBNV1xYhbOVuBEaIY04aNuZ0uvmiCD3fjkBJoyugJat+BPMn8QfyJNN08NSMirYRrnW5UU5RC4odlusAUKRV7RP8eplNMxJDUyzx\\\\\\\/yKRRWslgFe\\\\\\\/YVlqb4FwrOrQDyk311MeW8NIbA1TtgBiJMwAh0kaJZQGq8AZnY7z4M9SeK3wQ9XHEzaZ3tkfP4CFC5gD8jWeBNgHUAP\\\\\\\/7\\\\\\\/5OUB5FjJwlW37o2IKxSBlWwrdW0lm0GI1r7F2SpGcXsddk25tqUpj6QTLyeIffAdhtzDbUyoSBtONQmb2qLiA3l6uZrt7tJjG5\\\\\\\/zoLDceRUKUgVoGZeYqPdr9HMPDMi6uEoozom4gvhnyy\\\\\\\/lfboVN9Kvwn\\\\\\\/Yy1eQVQRGpW3UMOFtTBeBBHW\\\\\\\/3GloTRFhZd+vE4tKRh1HGqBzyBhbr5Ho\\\\\\\/uqBx\\\\\\\/pP5muaGKyef\\\\\\\/vpGD+pmngyK34msl6dXe7BU6zeOiuiaLTpr0igzpIVDOrKh0usLjawgTNNbqYrIDvrqcvJZAh4tdZKFViNiuygh+Lo2azv+wMbcCUA50AUmA5ksW9Dn0qMGlwoaNONLEn3nXD4AO8V89v9qX5KVOa0HaNfxQHYvzcrhizRe2OUoWaBJ0dL85xpj8eHET02XYU\\\\\\\/AK6PadbZgmbM2+TOQ+5x5ZRckoLHTYBqOKOMjAYTzyMHv+l+zZjyLgxtUjohUP9ALe5OyFDBRyqJ++oii8d1jUurKNqdFYv67LcMPm4sUE6aG9GfWDgh4VMrGFDNol8B9xiTH7JbYBR\\\\\\\/hAWdH4na09EDlJ57\\\\\\\/89ArlxwwQeumaV+akc17sU+IhREdNFxvFkZoJxS83l6DtqJ4ZVcQC3o5LR5Mlp37VlXDEVnk9K5ifLmQbLPKvOWbxFiUipcS8j2ygIoGOp\\\\\\\/aqDw\\\\\\\/8Ts5jX3rGQcuVh0lYEvpIkUyqTaUjYw1o5gIfKMymqx\\\\\\\/x1fP5\\\\\\\/RmdEHAsaYCrEJQwKFrFkG+rw77a+N0ARwuTfNR4R7Dd6ZGwlSw1naScDinDHPX9osyuJltKj6N5roHtRAUpso2wIFPMjkZthgrLoSBjJsP0Gg+esrmj4rxBmXAmN3l9DLXE\\\\\\\/ndAoyrjqD+1nss5GxSD6ORnZoPiTAyM1ofR6ZGKBKeGhbh7jbc9cSM+5z5+uOJcE\\\\\\\/7uktH+TdfEoZLMhpzCQF7qcdXEmv87OIfugpZeHlidthK9f18C5xV49+7yF5LukLRFdHhc6z2CXb\\\\\\\/MYwze62qr4r0AAM+3m12MalQNnPmLpKESgV5Zx0jd7TW6Fwc0Q3rBdNiUWkabNDCZAYYrEurdkBZNk4zvPi8YDsh\\\\\\\/9i6WxFnzgExuwPx0o3bhO4UnVTf2QkQRos83FRMaaHnVEIcttW8nwR2agOC9OZ4+BXzgJQ5Rl0vu\\\\\\\/KSnd5RmKUJgdKn96osEDEXKgY+49JjCgvooUzGEvslme0b4yz6LmyMjzYVrQeUKvJg3EhF70UoVVimvcgQX+R8Op0GnQxWSyxUcSblhgcxfWDBDhaVeGqD4XaYeA2Ifv4TYTSMvh3VZjNyisUKGGh4Lolgy\\\\\\\/nRj3hIpPBOskD7khI9XK23UKFqnfCluiALHLgiSGUIcv3qw042c73n4kQ1WABiojloB2DTE8aq1LGawUEAWmqBq3RD7GMkecgbYdnY+dIz1lCjjIrUeja7C0N8ZIhcXhQe5SMm03\\\\\\\/3ZnwdBp4X\\\\\\\/Blk3NYA8peezgsc0RVwjgUYbrKLX55\\\\\\\/3\\\\\\\/45Ks7Xm0LDssX7nC\\\\\\\/MW7ekZmJW3gsBSU1JhV59qhFxPBQn5fU5XUECrjQm83q4J84y9w4MQ5fQKMasi9uIxFlF7jk8yW3lyZOUQQatAR1GGyFksnKCSSc\\\\\\\/IBd+J3NKhg5+xQPnAUi4aqBd89QBsrbIiezs\\\\\\\/yu4db6QgCN9NQ7R78wwk3WnrtQCXNNpEDulGVwKPLibnQLgY1xc+\\\\\\\/bV0uL2qedkn5B2HgvhWLefGeUmxHiuGiheQmjf+qQARwVvXQubU+XcNUpYKEKHuPWt3Gnpk9VXCLZDEenp4LUR56OAUTk9MqBNMg0LcyML7AJECXkFdRywYPS0xlNW81trlsIJaqxx7\\\\\\\/AD1Wi8bA7qLV7g+R6U8\\\\\\\/h\\\\\\\/xPdLSN3GljT+L5XlUjIV19fuY135zJzR1Skgl5wQ6PXsogHq\\\\\\\/LVZNvj5yfCX0WUnEdTCEEgqh2PTtb30ksvzLNZ4PYFzwdmbu5lhO4P+hnGT7Su3Q6A0XRUMtoQWjLkuyYXlOQb1hUxyHcVC+iFuSIXfe8J2\\\\\\\/Z4H6CDroU3+If9wlqfs\\\\\\\/\\\\\\\/o+qwEulBGW6qL9uNb97oqMTPwqzxyfO5XsU4jgTpI4WZGyGeTBR7TxVyRPQ13IyzX5fMtJEOz52FOJKHBcGT8R\\\\\\\/lowmQ3RURNPK1DNSlQzejlMkXB2NIX8BzytQubPhENogmoWjDG6u+ckPzFuywTju9jsN6\\\\\\\/hLXuaS0SbPeOCjHODemL9zwGgdHl+3G0HWEHiDKZb7v\\\\\\\/+ue0V9Oxq4UykRppAE0inWRRugNxrjmIp19g5DQZ8g3irYhvSb1Gu5bAz2VbUBN7JuQ4uQ5T4zIlKA8r9n\\\\\\\/B7iqeAS8XVpgn2KfuJ67W+SNdpv5o2UsI3VOloxrLEZNwJz9\\\\\\\/c2Iwe4vKbTOPhmDipjfot3I7rdMpiS1L682gobkdYRtsGJoeeu6Vmmo3OGA95QY2f\\\\\\\/fHojKUpIKsBmdLfl1eudin79F26A9DFslz+aC0UIsaaY2ROb9qbJz5fPRSD8pg2gV7BttM7uPt7VX+hfBkgpt3iijGbwngUVHqAc50bTZudWhUEm+ZXrJzeJPeBWjUiQjCY3y0WXpRxzF+VNHmyFLpEqboUanDJENvLYKDYM6298hn\\\\\\\/3\\\\\\\/3L48c7fPwIO6lAkqgmdfUKOK5ye0aNZ5wKHthdaLPpZGw6jJF4l8NhWfT\\\\\\\/BsNmX\\\\\\\/gWNpLvp1691Hmq8WBvZ9QpXH+A+th5pY5eFqZg6dGNPixwxm4kFcbmQlINLasGA5qjLS4+SWJAucv\\\\\\\/tD1XW5cPOq\\\\\\\/DRH\\\\\\\/XzmMUh2nNbpgBdVCyNzTnTfKOQpoQLHCHOFkPOIF88ADDvyg6t5E3hYlzOlmmMKh6gryV9mM82eIWlUhRvlR1BktgJODh0cUdkjDzCjYALxoigcmPa56EPawrdpdm5xjfDURoEOaLhgh060W6uZJvOfpj3qVH6f8h\\\\\\\/f\\\\\\\/EJIqdsQ+EMjWEKU4BKBJ+uH5\\\\\\\/jNqh4Mr\\\\\\\/0Z\\\\\\\/oL66XEIWxMsRl4cdwiX6\\\\\\\/olJQYFmyZtlnoOMNBna9yi4fCidPasNnWfPgo1wZVnzLzecr9+5YHVZPhlTI+FnDaWrE5SImN\\\\\\\/2qAWtHgkKxmxw4FecwK\\\\\\\/mvlZ3PooZ9AINbiXkws5f0gn2EPdYzNFtv5gZDs\\\\\\\/sFd7oPASbE71n2E0H38\\\\\\\/\\\\\\\/WAGZbnx8v5daa4mNRVbkgxAxURnpmLZMmIQ15HItPWJE4yaZYVbHGpw2pr14ehf3fXObMR6S88gtlCIPxJ6U003UL11NkzzN\\\\\\\/SecNbmdI8GNsY4CXB6WpJm0CI7+EHlCcZWGx0017oJSY4B5QXj8Z\\\\\\\/iNSroWYKFOz9zDouvZ8on3PXtY9Wh+q5PgeC\\\\\\\/AlVBbJ4B0Ox2HYlS5bjvmQ159kCuGkQJEWWDaf9wB1pZAxxhTApacrmbDQcU4PWnKX1SOBo0mHNa0VLTsLwGS50TSIUM5pAuflKVCHu4h1wXe+\\\\\\\/i\\\\\\\/N8IJZHjORENcFZIP0uIjSa5PHarC+H0lzeQBgvAD2vg+bVZGGi2klpA2apXHpXs5\\\\\\\/qpUO1hdPQbcT3OXQVvEwxsWptQXY43fvgshenWE1PNc8zo3lKaCzqdawvMlcuznb3Kqzc0z7sVpB5GYEM9Mawfa2XeBseuFEGnUdjzsahbAXJ4J3Q5WV3O1dBQOexTGOw96uZXSkuwWxEgNXhZyWWfTeWqWoRS2eN37FA78xy899w+zOBO0XE1A2ht6rzc\\\\\\\/l5Eh0GBAAn6RzibBIygR84hQ0Wt20H8xnnRNRl67LvUQdbojKnTSN+kj5dyEi6RnzQfGDZAuhFZhge1TfZB7HmQxLr99CLei6WU6dITIn9zUYCRIOu5CwyLwFYArwKkWnyTFJ3XR+LizirqOu5bacl8VUuVMZdvrJfoeGp6fOlmmcnXuwfUTLX4J\\\\\\\/fELDRIejuiGTPVeInwAGYcb0Rw73xDQtRblDIEq33KH+gS4bv3LB0YUjvI67zL8RflbnaBnOJbXLbdgKg3Fsaom6mbj5bEUu2RKAO7pWKSs\\\\\\\/F9W+EcusT4+DPU7x3cN19eUQarDb+a\\\\\\\/cjT6EEZ64GCv6eAmRuiX8\\\\\\\/x05eB0cP2Hixv75i6OExYqpBtQaeSXHX1jDzL2FpysDkXlRhjje93l4l92BkK0lEzcl55HO9XTGzAj5aRf4EjnvSN7+E1VrMxdJmtjJEZDg+23fAGd6Gt0C4AiU9oYrAO8ks09F\\\\\\\/E9BKd47x60ZRl8d++\\\\\\\/+DoZ4dgDI9I8LFO7wJDZ430KbRJ03hdz0hPbLO3a7Rbbc0Jw\\\\\\\/8wb\\\\\\\/WRuIyYTXti9hYfQyexKUUtXidUP\\\\\\\/ArjMJaMY3GRMTGc48qbVgGBLIFK20ApAzmjnMXrWeYdDGE+E3VnV1JnSNN5a067xj2g1J82ylTb6ZyFp70byebwkjJqeGJR07NF5fa4cFiFrTMUSzqiBHBufFMAnLiibPDuCYQdRIen722p2e3G4iD1oRGSfh60YLspFKR3bOBWe31eKz28CobAB3EBkKmzj53HiwoeQ6v7XPvL\\\\\\\/KQUmFNuGP5DElpKP+VhNhIMcLixrYSEBDqaImZ+\\\\\\\/K+YDmehH3Xh6qMcl7yK1phgWB5EiPJJ+85DGxrUuC0siadFS\\\\\\\/rrrCBd3YY2iURCnzH6O2ikb7JVCuVyFhCsH67z4a4nmqkFh2U\\\\\\\/8KTmK3Aohu\\\\\\\/wdtq6i8sIGwjSnxetoSq5XGlNhJBL8aeyZunbm\\\\\\\/XsX4Uz8BK3TSmNw6EMiptOSCpccHMrn28fnASRHyrIl4fvkgig19i865nOBSbBmxgjAmbVBF8pJVLA3o\\\\\\\/FJbZlkoMTjPpM8n35i4g51JHPdOmeBi+VELn5Rytk8niEY7KB9Y01M3toWFDq5kWme1RxKBMuUsNGyyu7Th5DASULCHeRTO5evw4pV7TDpDMBiNjIPHnQthjqkvyFMam2U7adX1\\\\\\\/j7Vzyzw8Xgkyl2BY6a14iSz8ilk+hH30RzdzaWO0qX\\\\\\\/rQh1nTRpGNpRD4rmUnJUxVVVuF3UIIYj1ZO\\\\\\\/kqwd9OOV3qi8noG5yinBEx5UMLqlbgOxOXEMLEWi27qmnJ5s\\\\\\\/68B+GvNeEt1WMCzLsBTbTm+BX67Rvfl3AFgW7g8ZFDUb7rwrAWwVg\\\\\\\/63atsOya2+A4NFdhIMGzbGrtQoDy1K\\\\\\\/N\\\\\\\/Zn2KXnBf\\\\\\\/4Y7Xn3w8TBn2yuwgEPCO58+YhMF7gwEf5XKl7NDz86x8+2LtFhKBEOmJd6tyzjZ+rsXUZc65Rb8yHnI4lYxAU1SKU9B0ZHBS2Y2Sh2P5cOTvM0WsI2\\\\\\\/TAMagiSjweR17TevNeIYdk3ePP0mWKu6KpqFTouq97IhyTz8t74xyapkkcmESuZryazISaN5nhFe6jgdi+1v8J6tPTYQNr9sP0RKJDkVo8reuPgUluS4Kw1w4Q8cHt8ucM588sn2c3DU9\\\\\\\/GzGQy65OQLOP3Z\\\\\\\/wT7yXiMn951XLy3LhtCSvPaODkbs7M5xB2Ue5N1WcU5V+H0\\\\\\\/VOK16MCkayrxHoPrUcR2PEee8obZPgN4bqGIh6DZTTYoY\\\\\\\/L3AiPzdNYKj71aqXlIKbkfN\\\\\\\/7H6CX4XGRIsVnFc4ylz0NADYyPDfwjx1PW\\\\\\\/D1EcXA+QPnu9dOvYu13fb3bu4fZOgeMKl+epWL9gZzW2cCLZGelGt5Uw+GLxijFyCtmpm3bk6NNhZ5fa1pULw1Z8Cp14JnA2i+nWXnvWH9qBRCPs0b2TukaT1IIfCCJZrm9WnwpPWCU9AYBz8jaeVyTfxoEgOMB3UVf7UaKr1yAsq0xiEGvzRT8BV4VECB+FelCAj++5yxFA3bViXOAnkQCSSt8cplNoW3L0CWwSuaQ+UbM4X498Ez7v+cioP+OuUmSuJwv77Vi5I2dUWzrrYDvqVjnZythEWGck\\\\\\\/X1uh351cr200NSwU6pDzif+sGiHQeD6+WREZZKHuGNhjG6KCULEwyeGu4LV63OcsFvRu1pSmAfW2OQBpZKtR1J4QNsvuZdaKbdCxQvo+CAnDb4\\\\\\\/iqdt1QmERwbglXwSslMZn0tx7ZMJXq1Hvz0oC1XX4VMsVwNZxVfYina7A9cQPdOf4h3hDaLj\\\\\\\/BUWncUK4vPsMSHhzY1nKivZF6eVf3dAEg4WB6XKypKv4aMjEyn4GXg5Mmni78gtQj97h4yo7Ssp2y06Uwqqdj4iagav830zymdmVaObG7iljPjlMpHgjKawNP\\\\\\\/dsqdViVAiZimXGj0h3POboSqOTGLbsBPvBoz1NDI49otYBP82Ensj5umCvi\\\\\\\/phmbonbyvjmMMO7jIe0wqoLm\\\\\\\/GZHkx\\\\\\\/sPooJi3B1wU9GB1kTtf8t5HL6LQvk+cT3CIk8HYl02S7FnDDebGD63hWsEW03zyMm+gvnIdSf7b3OpHTNzQo8lgBHxWY\\\\\\\/VAvDPoer3Z8tARwRXDpRiOenTWge6uf73YEj9V\\\\\\\/P9wIp\\\\\\\/LWRXIspsajMFRLEqEpjJcqhdQe30vJAip7dveuF4t9eifKDs6P8QlU\\\\\\\/X8hoUEf6JTAi2KRwcdG1yuwHg21bhWWxrpgxSjDLoAGe2d11vPWXiE62QPl9WQqrn1i\\\\\\\/Z4+VGKI+YoXFIDp9kn7yZMuEbGZdfhjVBLqr9uokuGHLBM8q9J4iHvnT+3nO\\\\\\\/dklgu2na68wlMkZzMi1A4enjqZX8t7RXLEPeilWAFssRnhH\\\\\\\/Wj\\\\\\\/gbtodAn8NZSvZezZPUcybU9sonMfIphh7s98hN38+awsJWLPoaBShZFc1jM462ddhBLFOTWOKxySuUbUPYKirRlgMSrNxYObiyUE3r7kG9VdvSDzoY8tHBTRDDnssgRvNafCSHY+IvKZI+bUQx0ovQRJtkaqVrQnx5u2VK7bHI2109tW3XCQoAegvjx2I0YxjUhEO5aej\\\\\\\/BkyGatUbRWdFpoC9MxR5GTh6mrinp3Paa4b9PSDuBMbgfSqxE6NNVvxK3k7RSc85GusSoRTRLPS+jyZIo+G5CWJddjxBcFLOKYPCylJx0WX6oB04PMRijrDVEEjguRT8oDwPz7F7ZignMnlPLmYn4wiP96+GJiL6eGh4jooxEJJCJrsfBniXdwVo7UqqzdxekC8mGkGgmEiK61UFBTLrX5xTe3YWAUkDSn1iS++AJX09PRwxgFvXBEkBw2gu9C20gAiE9YAOlb3+jPc5HtMdnKsQ8odmt13KJSbUvn00RewHN+J2jrldaTC62hvkPRcn3n9X+4aGRm9or3HtshvRQtCl6Ut\\\\\\\/twRQN+wmI8EedL+M2hZyKKMeTLMRa2gNAC6uDLIGOWlgRaRbeOXv3RgT8wUYgc72qan\\\\\\\/X3bsRAx7cwKfFSHzjuQuFsplp5KovnmMVDazFc4feBsAWTXTktOh2W7TmfDg4eIPslcEICWd6Yj4IHjGJtqoDg+DuP2jpqlVfLklhd5rEqtl\\\\\\\/AJoBR2edfdKZKOJAYpoH0y3+hVDMb4+wHVyUFB5dRHQZVAJMyOxphcu0v7OxHs9EFIKz8VpJhJubxCfW8tOiXXAfdYaBZNlNvzVg\\\\\\\/hWoqg9Vd0fYb\\\\\\\/hOHAXIayWXaDIYIoZ7tbi6mMSzZ6VDEaaJ26zGbgZFGoIe4UoDXzU6hmQDQnH+eUQ3BX1itAg66ECjwYMApzR5H0OSWZrgkb3FECadOsaAXP\\\\\\\/pyAYWOTrkDKDiDuGStiS9LTOvOOdW1GBM\\\\\\\/oL\\\\\\\/4opPJMiHieNfwcf0kbFqdVtvDGibHjsXgKCJ6hOczvZYySdmlGIQUR+8Z+U2dvrRpRkStD6P\\\\\\\/Kf7s6bBo1KFq\\\\\\\/uo8AywaXXBoAUc0+cfGNtzx4sPMRd1bhmAqm1S\\\\\\\/HHRpLlVfLdeqkGI1mdItMauZYlRhUe+\\\\\\\/bDzwMEhG4i2g4wh9xd89D6tqS\\\\\\\/gw2sJ4ipb0Iz4Saur8ZndB\\\\\\\/ihipLSOGVc2A6tsHIV4MGCpbyen1Ei7XoB4rYK3XkDZ81pSWcAOwwuQN+7rtmQbDgalILD0ERobRHFq91dmlPmJ1WE7Cx31SE3K8hfici6YsJ1nEOwNS+FGXBDuqeqhOBZF9GGxRSeVcCeLxJiYoNasOWg95fQrafwhgyOFNJ3J3ZjtI6h+KrAuqZ7DAt8tYwrbj\\\\\\\/Z9kyO5Z1Mx7NpGNUa4saVvkTiyIizMjHWKyW+wsMjb5xcaEm5vRzTAyz68dOpsRtMp9UZoOZVPnqPyA+WffVROUMnYG8U+XmRPpnU\\\\\\\/qXjeRq8ger\\\\\\\/\\\\\\\/MXTIFkc+FSkPSF9DWfW2Vu7p8hMfIyr6sXlFJkJ92hsgq14ygwXh8E2dMLu6xiOcdn5N5fT3tclMd8TVOnxnOuXrnEw8HwrJcquE+AUnAeVt9I5OHHPeDfCpDggUk3lsm5vpX5clWTHTA0mQgr+CJ2u1o6t\\\\\\\/ef6ebtTQv397PylyTaovO0Os03DTR5hFeMFgZh8qR74Ne7555LMYXBDRoziJzPGfxL+XjWVxFTR7Vr8JKS6kCvhqENf8RIMlfqzLywNUX7Z\\\\\\\/8U3BKROadF8oJGHTxI\\\\\\\/OQaVxXlbj4PgACyEhObuEBeK3Y0\\\\\\\/BeuUiq7cOf\\\\\\\/pwXqCIeBpINnkY1QjDxyICMmFDyZM4z8\\\\\\\/LFz1te\\\\\\\/aGuELQkzpe09pzVjvnSq5aSNEnbj1iZMzQhH7sWbjF6YXXkDS4JcQQwATDyyo\\\\\\\/eD6V5QMrRJDPDHyKqP99HLIscVvlRtRc6L9dR4QN7BDBhspH7DzvEnyXn43Y1mOWx60Ak2SmylC5iYhkg8Y6KsO0KTTz7q9McXXcZ6uLS5v0ORQYaF1yPYbHybeZYLnpCn4WvmxlRXQpPuC4M\\\\\\\/jEI4C1AVJszHNs7Ut4OHNqGuBQR4qb902S+8weybxkABPXLX1598XHtej2n36TBoeH4CEBIyBvHuVUtrI4QikKSqfMFx4ExTyAYMYWs08Hu+589vL6VcjhP6f8e4hb8FkNp8zrQr11N9lboVVlHJ3bkiqU1D6ehd32VIHeJplR4eTz+0OH1eKaXIvfyVAY2FJjPUE9YH\\\\\\\/6jKpgS5twBQeO9bBFmKEr5nS0AAxswvPTygyndMP7DpEOEwAXs5LpLpHLBiNbt7JnArAX9rDk\\\\\\\/+D6pvkXF5ePMVbqNDD+7ghK+11sC2nJT7pSnPEWQNUT8FxRr8RdTwSeFYGjsMc\\\\\\\/ZK9tOAIVrdZApmY096dI\\\\\\\/AkI2Uu3HRHtyJ2D9qV110kAzGv6vWTMn8zF+ek5H4p7A+DdUuL\\\\\\\/051Zntm2YWl5a2JkEpF85sqweIRMBDjx1Lp+W2\\\\\\\/x9h8yFVoToHcxY0JFiHqwS2nv9LM26+AEYcxvNdnLqgL5DW55ihun86zBqUkOivkDRResw8aTaW4FT9jn+qPouIJOYsZl+YGV9xwxvL45DrsiO8TJS\\\\\\\/wo3p5JM0HivsnHL\\\\\\\/AYsy8LEGLJ1ZsL+GgCCZDXcB5JfIG7\\\\\\\/PLBbmFupIvsyFv2hkPlaZ50BmnZamqQLlUonVXiYNk8jI2hXNwJTkXfZlDBwpk\\\\\\\/cYhx5ISKr+LwbltQv4QPSkLCIY+E9vZP9tv\\\\\\\/SHbZQe+KSmKyVbOh4Rbntbc9cASjhS839+c8ZaKESZCq2z\\\\\\\/FBZPGHbrHm9e1kSIyLWrxDu7c7aPS0\\\\\\\/oX0tBQlBQYna2k30VDVm5kwAdY\\\\\\\/8vSPB7UFGPYLHYLY3hBFldivK1096xOEwhWVf8oaFATgdkBoKW1QAqiLOFIQXMrVJT1ZtOeSO7J+1FTK077WuxeEvbOr76y\\\\\\\/ChbONmd0WcwiezpVqUzd9f4eT6vWIaKS1tNYvZli68qnUga3nwXK9Y\\\\\\\/ur9O4WAo\\\\\\\/lLIg3HXd08BzmUww1glrYayRbXOChiqcJrz8mewQl\\\\\\\/vpG8Q85UkNMKT+U3vHXr2zgRV5oEC9BFGuVJlCuBh8xY1aefplEkP6gUzcNp12SCm3yRTZ0TkwL3tRAJLvFeXOByuO1x56331dadY0BWWzURRq5nJGeVhdUC+1HsByfv5c4aLGZlCpWZB0hQ7umuee98Q+bJ8DJkqm7zUe6EZP9VnW9ivIAvRpXF+7LJB1gUmYbfZc6lS4Dvjs5cFRB0KDdkRgJVHpS8jTYU1xs+67pP6cocSDSjrI\\\\\\\/yYF2P4EFreN+XVaDJDtUxf1n9ruLKYR2p2CQQFLiwCQFwW6u0+erZ0L6zGqdscqV156M31VNciNojt8axZjjj8Blts\\\\\\\/pg9EyJ7bLhvgvn7+pjCSoPzqyOV5ze03Kuy6RPSLcd47dO81wVJNmJvjZTK8dH8tjOpklRPoRwvbXTn\\\\\\\/0NRxXyARJ7wwgPjSweXGSyjq6cdNEYmoFiyYsvDw72hi5iw6bwpWapeeomxPktj0scVbrBDVlyyUcPKKAmpwBzLHzAiXvP+wL64eaGcNzADaOV4S0YFNZBMC4byTwdbrGNImcHjhXphIbHlVoQjSMg0OzB522NS0sksDPMZH5EpgQuOKvJKDcmdnSvSmvp1thG4C6lx4yko0FTL1gZrdmWl\\\\\\\/QIDmuB+qccAVGfaXVUMeMWHHbMxXw6dh9UR+VSaRUzoMcsHHJKMF6oUFIQvkvuWdek9jsaQt2rNgAZNDMy7YLbjJEO4eVXqhv5\\\\\\\/Vh5AQfN1nIzvMbFheyWbwPUmq5+KE\\\\\\\/GMng2U6k4lIWp+i5iomrnNaA+GAwSZSVbcxSKYdv0yOzsM7Lo0Sph9wRKpRYzuHldFtIAiwFJbhHBeAZvdDUYN0PcpWdnkc64FhzGmOwB0pSf6UflPKBIHJm5fhJu6lM0JD9WUMbp8zJ1wekxfbsl3cE11d+XLpYEc0o52a0NkOrcH7Pf02tmHHfJrqzzUZ5zoCwfulTpeH3MPrwcdos\\\\\\\/zN+e+T5tcupK6otY6WJkLoGDXcQqxW+xg2faZgTQTLD911dt8CdxmI5sRfXR2wLU7AyQsmxhhf9F6j5cJc7qOTfzu\\\\\\\/aTY288wC6tA4Y6XNT8g2T3MtaeAdACiKCxu4EA2DBsdydAKKueSitz6h2UlS58JxNFAamO\\\\\\\/kqqEztxbYGsFO7A0E50haroC1iXO4tDrkity5OVqQYJG5XlRC4FbyalKXMr3cbDi66aRSw\\\\\\\/EpKJaCRU6r\\\\\\\/A5Ih9CJTC\\\\\\\/flVAPXZ4h64QxcbubNrLWWam6vWwc57cukEf4ebeeV6JLFhYPFeIwSC64RCQHXnUxTH+fCnCLONBle4YEAs\\\\\\\/tTuKPbVsu4FPrab\\\\\\\/XHPjtKqDlszOW3dyHrQMy8BFTJc7CVdmaO+uGmoV5mdAO7lRjljOwly2ZL9tfrQyD7oF1g7uVeFWolJD2OfSCswcvRkmCDB1YmOE25QtaswC\\\\\\\/yAthiipPUFhE88sqbjt3\\\\\\\/GF7PoeNUq3JqxWNla6Nj9FRpJTmICef4If4Jg45JFaIQALBz8tnLYIil2xi45ZKSQ2F6EQ1RKUSdSynraGQvgJlE1Q1dh\\\\\\\/p+8gAYh4uX33qPVa4lHjFTZ3GD7a+cNSz1DQ34OEzZW436pN3FnqopebtajLki2Hfo6YDP0dXJZUrWOJiOemf1yUtnCI7H5dhBvZLGwRgRsyqBRd0UH84O3CZZqZmYrJ1M3rC4v\\\\\\\/Aq\\\\\\\/RqGOC+3Ud+PgZyNSkXRFMU4MGM90PW31nkoBSVh4EExEN\\\\\\\/GTFHR8\\\\\\\/E\\\\\\\/kLODq\\\\\\\/ri5WzQuhDjtJgROlhE66eH5i2ToP9cJqKwd5yEkKF\\\\\\\/iRGqt3LHy2OYw0jdqGKf4Jy2SXj4ZOubUWKLqbF3UdpCE6ONUdne1Mk+yyP2\\\\\\\/0HB6J\\\\\\\/DEvPYQpul7UBeHukuTJwhHnymDBJndU6teXNjMi2YOUaIz8SWFWk0r5AP4GEkspkMaeBWn2uOBEvujnvu7Vy28Y470dsbkpHqYmt2ZhffT66H3AJlQdT9n5rqRDXMNpgJePFVSH5eNmyAgqf9rTMij0cfH3aAwvc2tVqxw84u1RqkjH74jc0ED0Qaxs4SEr\\\\\\\/oDEyRh2ZJ1stmT6mHORC+gyDWx13WkotvB4cJJjuDhj5UtkJYH5D8DE4dWRD3GQXywf5P2cGoW5pBHjHlP+hy\\\\\\\/rVDCEx4xKW4JRs5SAlbqnzK2pQX2fa0x1p62b\\\\\\\/R84z5V38Z6wfkaJXzoWa\\\\\\\/l0m81wIZA0Ptmc9Ec8BBODBZoNmo4svVjYhdKEUFrskblbGFlUXa4f5P8v\\\\\\\/HQPXA96+IfOdYDWc551cLprcHovVhhQ2n9yJpIgIqIPRTY56DkaUDoguj7VejGB3+uXlXuPJ\\\\\\\/eawDrazktJ4WTF1J5qRKrz0XsuH0tjM4YDNq7U65dfuDzKbe9hTAzyIGF+FvLh0415IoqdzalUqJco03aiqNZnby6zIUtGoCUrI0UQWfw6Gx3DRXKoKOz6xaWjYF835INp+yapPsYHQyRxm0lsdkv8UPdOSIII0SDlDZSb+H0UD9gk\\\\\\\/dT\\\\\\\/tTA25osPNw2WXgAuJi2JMoJdQ1QnGnRgEPjcovQRvXGxSVfQ4ZZdTFai1YYfGsYX\\\\\\\/AUwQDeGH6c4h9tuXE1tE89B+lSK7Bmyx\\\\\\\/Mtj37LLN8JVh8kLY7\\\\\\\/OZCFOer208I0p\\\\\\\/lgggjlbwDfQeh+E8hd2q7lznuxZxbl6JouiexQC1UWE+tLkSAldcwSLrBFrUEXxqBvtk06BlBed+fyvcaIR++AvXdL82KpKZF2JuRrnQ0DJxnRi+jpU+YHg\\\\\\\/PYLzkwdcN4pEhykCGKfnMJQMuSiJ4xYP6R9+VuFSjGPM1jbvBgtsPSEzORjNd8MPFbg714Z2IwXQZXH22m3mQgKkws4yaTy6+g0RZNfpE8yI2IKRaFpxLrOje1rHxnuGsHq2XcfGBrTHxN7hEqX6uILg5iCCQrs5F3YiPdIv8tTxf02u5GZRmcIOYtSGOOt7Ozm3QHylE2Fw19Jx8JdW\\\\\\\/+xhcK6vs8HCE02Ux6ADzl0KKq+CCmfKDlEE4kWcg==\\\u0022,\\\u0022iv\\\u0022:\\\u0022yg6PpPjIHgIKuXQDVlqllA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022239d1105165ecff5be756203248efb6e2511cef09fd827b2d12da60046a41741\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022n9U2XtT75mTxzEUk4WpeBYIuIYZV9JWZY4Ouq8MNfHCZVJy9M1yQssaNb0Vtvzd\\\\\\\/Wjn1OtWXAOOcN10Glt+2XmgS9wfXMf7q9CcOqTIF5vLTseSW4cI3BftnzlD0RcE2\\\\\\\/HyTUiECvtUMaz4r5DjWryJUhWUTAQATs1I8nM9g2ytQHa7OmptqhNgN8CQcJwc5XB2JF7rRJNiFUAvm3gmtVFsmb7fIvag3fFu3k63cEE0xmbBWDzTW6FLY1j05DYpmxTWG1MasLD\\\\\\\/DUqthz9oyjxuV9ndb35aUois9JN5jBtgr7h3DZUN7v4g5HTmGYnXzSAXecHuDGGdZPcjdbDbfqvruuB5NU\\\\\\\/BGvJoRLBdPulA2hn3WUubv9Nwug0\\\\\\\/AHz+EECZseLoC8nH+Z6oGMl+tuF8rlXKJw6Vj5raH99rtI3Wt2UljuofmnYRYDp43voo++md4uHsrnum38n\\\\\\\/4Nav8ZiELfyWpT8GHzcU3OFILFwhV65s4mCtAloEJ5jG3aD+BvEUysor5VRNnNZ5MSk0YIw28E3NOnVWjVLOUUueE9w3aghjCs95JO2GUSUrY8V\\\\\\\/TZyEYbxhTRC7zBXiDOVSGVlvrz8TiYeIxgXRcbnQ+irZDTZ0Di4X\\\\\\\/8lSFre3eEhEZbYq\\\\\\\/TsCLqoBqCCzqAjfi39wWjlqYlMH4QSV+cqytRcTMuCkOC0IuN4vocfGhEYNNq0jJe5p7BNqalxB0zXN9evCP8Ud7u25zQq9RW1CdOVB89VilEaCe5iMPmp7V84MKMGhhVtklQojf+xGTMQHaEai4NTc1dYjx\\\\\\\/54NdEkuBHbsHRgEtZUuL0GBeXWzRNfecnTvP0WkbtQu+VwVmlW702yJVlrc17+Bleb5NA5ksO9l8XE51BQBXZ9oa5+RGaEajfbMzSQWjlCaNsct+7yj64Xb6Inqp\\\\\\\/uCXfpq9GWsZjwp44819UqHbVcPC85RAzWbAaddMQ2MpiFgkiHtuxtBR7cSGJE62Oo4EJAW0fsHd63huFNhTbF0Eej8c9\\\\\\\/YBEVolxSxoKP8DtzkD\\\\\\\/FHQfDwaB+E6ChBiE8FsTF47chlA3hSRZi6lTV5PFmolFCZy3KGUf2R0oKjM7EAEiphnQR7Txf95ru5sLyJH2LUALi1LsEl0x\\\\\\\/3MqvZaiEKa7\\\\\\\/oOCkGHoAKTuCSG6pBNnQGfkAH3DE1thy65VwOtnG6lQZPlPzc9ik23Vnr4OBREfAr6mdPWu2cYMGgNCbE15oI328oep2sKXcBVFd8TEO471S9EA+TYOwmMiWl3ffNM4ZhRI+oDsM1czJJdWBd1vU405pLY1WRFiHwKrHNlJYvmiEJbFhmuTC9EeSe2lvgzcRgI8D2q9cF\\\\\\\/mbM2ZuF3fd0NnQd9FYDA9\\\\\\\/in5h0tEzid0WOiN15FycpvPol9RWD\\\\\\\/vHBwq2SvTe5\\\\\\\/6Lpk4aqOUqtAHBl3ZSq5dJoGN6ncILdtJaUWhcIZvfKJjLq+1rKaO8U29V6CjccVuZLG94Dgzo6NdrEIK58BVj+IEFtMOjBpEoajMb6qYiZCGCNH7kq7X7Os4yU7+BMF+6o9o2kWta1BwaZswntlipAVhlOeDpE5QBkJgENa4LD7vkv7mtS2mkIR3NRURUi+BieZ3C9GxS0yGl4n8defykws3cf8ikS6Y1Fe8NDP2cm8lLPLSnG3R6AHs7Qw2YetEzT0crdxgMRQ6JOUOlnaTbXahqftAeFtP7k6UzF3knctPTB9W5icSh7yLbBWeifS74DwdxkZPHH9ryxjKbvdYXRoVhaWeyv1sFdTlFSeCQ7obWfJ1TYMPu03Qs88Do+tXMRhtEHSqQQXQNL+ZU8\\\\\\\/s921XriOfENNdLr\\\\\\\/ooS1bzfiT2vQ5nT3jCnCsU0Za58AbSSLUUmYDR\\\\\\\/sxd6PQ8eXCjquxaBdKol2eY2UQUwJpswgSRuIT3ZJzH\\\\\\\/DtXXhG13Crc0g\\\\\\\/KpZt0nCDHw05aC7TAVIZoIo0z6NOZ4S118ZgiQEqfFDzLB1fvQ\\\\\\\/\\\\\\\/4fMOAsRyPr8qKTMksrJJh0Rg==\\\u0022,\\\u0022iv\\\u0022:\\\u0022BsSDCUJzy6L1nZumowgaCQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022f92c1fbb9460b67853f9161b6e3354a6b52281279491e7900a5925b8cf96436a\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459322 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/2/9/1067" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Sz5Vc1fUJC4izpFSHphgOjcR7YeEpiE3pskE6KwryhM" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/2/9/1067" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-tO28yXQUvg0YXeB3Fx16LGLnmh4ixbq9r7sZ0nWnbCs" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/2/9/1057" title="Using Near-Infrared Spectroscopy to Resolve the Species, Gender, Age, and the Presence of Wolbachia Infection in Laboratory-Reared Drosophila" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/2/9/1077" title="Variant in the 5′ Untranslated Region of Insulin-Like Growth Factor 1 Receptor Is Associated With Susceptibility to Mastitis in Cattle" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459322" id="node459322" data-pisa="ggg;2/9/1067" data-pisa-master="ggg;2/9/1067" data-apath="/ggg/2/9/1067.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Uncovering Networks from Genome-Wide Association Studies via Circular Genomic Permutation</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Claudia P. Cabrera</span>, <span class="highwire-citation-author" data-delta="1">Pau Navarro</span>, <span class="highwire-citation-author" data-delta="2">Jennifer E. Huffman</span>, <span class="highwire-citation-author" data-delta="3">Alan F. Wright</span>, <span class="highwire-citation-author" data-delta="4">Caroline Hayward</span>, <span class="highwire-citation-author" data-delta="5">Harry Campbell</span>, <span class="highwire-citation-author" data-delta="6">James F. Wilson</span>, <span class="highwire-citation-author" data-delta="7">Igor Rudan</span>, <span class="highwire-citation-author" data-delta="8">Nicholas D. Hastie</span>, <span class="highwire-citation-author" data-delta="9">Veronique Vitart</span> and <span class="highwire-citation-author" data-delta="10">Chris S. Haley</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2012 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 2 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1067-1075;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.002618 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459322" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Claudia P. Cabrera </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>MRC Human Genetics Unit, Medical Research Council, Institute of Genetics and Molecular Medicine, University of Edinburgh, Edinburgh EH4 2XU, Scotland, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Claudia%2BP.%2BCabrera%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cabrera%20CP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AClaudia%2BP.%2BCabrera%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Pau Navarro </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>MRC Human Genetics Unit, Medical Research Council, Institute of Genetics and Molecular Medicine, University of Edinburgh, Edinburgh EH4 2XU, Scotland, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Pau%2BNavarro%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Navarro%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APau%2BNavarro%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Jennifer E. Huffman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>MRC Human Genetics Unit, Medical Research Council, Institute of Genetics and Molecular Medicine, University of Edinburgh, Edinburgh EH4 2XU, Scotland, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jennifer%2BE.%2BHuffman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Huffman%20JE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJennifer%2BE.%2BHuffman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Alan F. Wright </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>MRC Human Genetics Unit, Medical Research Council, Institute of Genetics and Molecular Medicine, University of Edinburgh, Edinburgh EH4 2XU, Scotland, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alan%2BF.%2BWright%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wright%20AF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlan%2BF.%2BWright%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Caroline Hayward </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>MRC Human Genetics Unit, Medical Research Council, Institute of Genetics and Molecular Medicine, University of Edinburgh, Edinburgh EH4 2XU, Scotland, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Caroline%2BHayward%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hayward%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACaroline%2BHayward%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Harry Campbell </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Centre for Population Health Sciences, University of Edinburgh, Edinburgh EH8 9AG, Scotland, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Harry%2BCampbell%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Campbell%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHarry%2BCampbell%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">James F. Wilson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Centre for Population Health Sciences, University of Edinburgh, Edinburgh EH8 9AG, Scotland, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=James%2BF.%2BWilson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wilson%20JF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJames%2BF.%2BWilson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Igor Rudan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Centre for Population Health Sciences, University of Edinburgh, Edinburgh EH8 9AG, Scotland, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Igor%2BRudan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rudan%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AIgor%2BRudan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Nicholas D. Hastie </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>MRC Human Genetics Unit, Medical Research Council, Institute of Genetics and Molecular Medicine, University of Edinburgh, Edinburgh EH4 2XU, Scotland, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nicholas%2BD.%2BHastie%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hastie%20ND&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANicholas%2BD.%2BHastie%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Veronique Vitart </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>MRC Human Genetics Unit, Medical Research Council, Institute of Genetics and Molecular Medicine, University of Edinburgh, Edinburgh EH4 2XU, Scotland, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Veronique%2BVitart%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Vitart%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AVeronique%2BVitart%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Chris S. Haley </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>MRC Human Genetics Unit, Medical Research Council, Institute of Genetics and Molecular Medicine, University of Edinburgh, Edinburgh EH4 2XU, Scotland, United Kingdom</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>The Roslin Institute and R(D)SVS, University of Edinburgh, Easter Bush EH25 9RG, Scotland, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Chris%2BS.%2BHaley%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Haley%20CS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChris%2BS.%2BHaley%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/2/9/1067" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459322" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459322/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/2/9/1067.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459322" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459322/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/2/9/1067.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459322" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459322/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/2/9/1067.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459322" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459322/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/2/9/1067.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459322" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459322/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">Genome-wide association studies (GWAS) aim to detect single nucleotide polymorphisms (SNP) associated with trait variation. However, due to the large number of tests, standard analysis techniques impose highly stringent significance thresholds, leaving potentially associated SNPs undetected, and much of the trait genetic variation unexplained. Pathway- and network-based methodologies applied to GWAS aim to detect associations missed by standard single-marker approaches. The complex and non-random architecture of the genome makes it a challenge to derive an appropriate testing framework for such methodologies. We developed a rapid and simple permutation approach that uses GWAS SNP association results to establish the significance of pathway associations while accounting for the linkage disequilibrium structure of SNPs and the clustering of functionally related elements in the genome. All SNPs used in the GWAS are placed in a “circular genome” according to their location. Then the complete set of SNP association <em>P</em> values are permuted by rotation with respect to the genomic locations of the SNPs. Once these “simulated” <em>P</em> values are assigned, the joint gene <em>P</em> values are calculated using Fisher’s combination test, and the association of pathways is tested using the hypergeometric test. The circular genomic permutation approach was applied to a human genome-wide association dataset. The data consists of 719 individuals from the ORCADES study genotyped for ∼300,000 SNPs and measured for 51 traits ranging from physical to biochemical measurements. KEGG pathways (n = 225) were used as the sets of pathways to be tested. Our results demonstrate that the circular genomic permutations provide robust association <em>P</em> values. The non-permuted hypergeometric analysis generates ∼1400 pathway-trait combination results with an association <em>P</em> value more significant than <em>P</em> ≤ 0.05, whereas applying circular genomic permutation reduces the number of significant results to a more credible 40% of that value. The circular permutation software (“genomicper”) is available as an R package at <a href="http://cran.r-project.org/">http://cran.r-project.org/</a>.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/gwas" class="hw-term hw-article-keyword hw-article-keyword-gwas" rel="nofollow">GWAS</a></li><li class="kwd"><a href="/keyword/pathway-based" class="hw-term hw-article-keyword hw-article-keyword-pathway-based" rel="nofollow">pathway-based</a></li><li class="kwd"><a href="/keyword/permutation-method" class="hw-term hw-article-keyword hw-article-keyword-permutation-method" rel="nofollow">permutation method</a></li><li class="kwd"><a href="/keyword/genomicper-r-package" class="hw-term hw-article-keyword hw-article-keyword-genomicper-r-package" rel="nofollow">genomicper R package</a></li><li class="kwd"><a href="/keyword/cardiac-disease" class="hw-term hw-article-keyword hw-article-keyword-cardiac-disease" rel="nofollow">cardiac disease</a></li></ul><p id="p-4">Genome-wide association studies (GWAS) have successfully identified many loci associated with complex traits and diseases (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Wellcome Trust Case Control Consortium 2007</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Hindorff <em>et al.</em> 2009</a>). However, the identified single nucleotide polymorphisms (SNP) passing the highly stringent significance thresholds set in these studies explain only a small proportion of the traits’ variation (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Manolio <em>et al.</em> 2009</a>). Investigating variants of modest size effects using a gene-set analysis approach has been proposed to identify some of the undetected variation. Gene-set methods aim to identify effects of groups of genes, which may not be individually significant but, when analyzed jointly, may have a detectable effect on the phenotype or disease of the organism under study (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Wang <em>et al.</em> 2007</a>).</p><p id="p-5">Most of the gene-set methodologies use random permutations to assess the statistical significance of the results. Typically, to generate a null distribution of pathway or gene-set association results, SNP or gene association <em>P</em> values are randomized post-GWAS [<em>e.g.</em>, gene ontology analysis ALLIGATOR performs gene resampling (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Holmans <em>et al.</em> 2009</a>), and i-GSEA4GWAS performs random SNP permutations (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Zhang <em>et al.</em> 2010</a>)] or, alternatively, phenotypic labels are randomized prior to GWAS [<em>e.g</em>,. SNP ratio test (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">O’Dushlaine <em>et al.</em> 2009)</a> and RS-SNP random-set (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">D’Addabbo <em>et al.</em> 2011)</a>]. However, SNP and gene-level permutations do not take into account the genomic structure, such as regional linkage disequilibrium (LD) and functional clustering of genes, effectively simulating a genome in which adjacent SNPs are uncorrelated by LD and genes are distributed randomly with respect to each other. Permutation approaches that ignore clustering generated by LD and functional co-location could create inappropriate test statistic null distributions. The possibility of performing case/control label permutations or other permutations prior to performing GWAS is limited as it requires the raw data which is often not available, and also given that each permutation is followed by the association analysis (GWAS) and subsequently with the gene-set testing, this results in a computationally expensive approach (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Wang <em>et al.</em> 2011a</a>). Furthermore, concern on the application and interpretation of gene-set–based methodologies has being raised and discussed previously (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Wang <em>et al.</em> 2010</a>, <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">2011b</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Fridley and Biernacka 2011</a>). Despite the lack of consensus on the most appropriate methodology and the warnings raised, GWAS gene-set approaches have being applied to diseases, such as breast cancer (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Menashe <em>et al.</em> 2010</a>), Crohn’s disease (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Ballard <em>et al.</em> 2010</a>), multiple sclerosis (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Baranzini <em>et al.</em> 2009</a>), and schizophrenia (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Jia <em>et al.</em> 2010</a>).</p><p id="p-6">We have developed a permutation approach that uses GWAS results (single SNP association <em>P</em> values) to establish the significance of gene sets and pathway associations. The objective was to develop a method that would increase our understanding of the pathways involved in various traits and to detect the effects that would have been missed by traditional single-marker analysis without generating an excess of false-positive pathway associations. We explore the performance of pathway-based approaches applied to GWAS using this new “circular genomic permutation” approach. This approach is equivalent to the simulation of pathways with the same number of genes as the pathway under study and with a structure that faithfully reflects that found in the genome with regard to conservation of LD and relative location of features (SNPs and genes), but with the SNP effects on the phenotype randomized and no real pathway effects. This allows us to obtain appropriate distributions of the test statistics under the null hypothesis of no association for every pathway/trait combination surveyed. We compare our approach with that of applying the hypergeometric test alone, which assumes that the null distribution is hypergeometric, or of generating a null distribution by random permutations at the SNP and the gene levels, which does not take full account of genomic architecture. In addition, we attempt to measure the variation explained by the SNPs acting under a pathway.</p><div class="section materials-methods" id="sec-1"><h2 class="">MATERIALS AND METHODS</h2><div id="sec-2" class="subsection"><h3>Genome-wide association data and analysis</h3><p id="p-7">The association values were obtained from the Orkney Complex Disease Study (ORCADES) (McQuillan <em>et al.</em> 2008).The cohort consisted of 719 individuals measured for 51 traits ranging from physical to biochemical measurements. These traits include blood pressure, height, waist circumference measurements, and cortisol and cholesterol levels, among others (for full descriptions, see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.002618/-/DC1/TableS1.xlsx">Table S1</a>). Individuals were genotyped using the Illumina HumanHap 300v2 array (which contains in excess of 300,000 SNP markers; <a href="http://www.illumina.com/">http://www.illumina.com/</a>). The SNP association <em>P</em> values were calculated for each trait as described here. Each trait was adjusted for sex and age, and the residuals were transformed to ensure their normal distribution using quantile normalization. The mixed linear model mmscore() function of the GenABEL package (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Aulchenko <em>et al.</em> 2007</a>) for R statistical software (<a href="http://www.R-project.org">http://www.R-project.org</a>) was used to test association between SNP and trait under an additive model for the SNP fixed effect. This score test for family-based association takes into account pedigree structure and allows unbiased estimations of SNP allelic effects when relatedness is present between individuals in the population sample (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Chen and Abecasis 2007</a>). The relationship matrix used in this analysis was generated by the ibs() function of GenABEL (using weight = “freq” option), which utilizes IBS genotype sharing to estimate the realized pairwise kinship coefficient.</p></div><div id="sec-3" class="subsection"><h3>SNP annotations</h3><p id="p-8">The genomic location of every SNP in the dataset was updated using the WGAViewer software annotation tool (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Ge <em>et al.</em> 2008</a>). Gene annotations (<em>i.e.</em>, gene identifiers, symbols, and locations) were extracted from the NCBI Gene database (<a href="http://www.ncbi.nlm.nih.gov/gene">http://www.ncbi.nlm.nih.gov/gene</a>; build.37.1). The SNP-to-gene annotation was performed by linking the SNP genomic location and the gene genomic locations. Two annotation datasets were created according to the physical distance of the SNPs to the genes. “Distance 0” refers to the dataset where the SNPs were annotated to a gene if they were located within the start and end of transcription of the gene (including SNPs falling within the introns of a gene). The second annotation annotated SNPs to genes if they were within a 20 kb distance of the gene(s), also from the start to end of transcription. SNPs from both datasets may be annotated to more than one gene, and many SNPs do not annotate to any gene.</p></div><div id="sec-4" class="subsection"><h3>Pathway analysis: gene <em>P</em> values and hypergeometric test</h3><p id="p-9">KEGG pathways were used as the predefined gene sets. The significance of the association of a pathway with each trait was evaluated using the hypergeometric test. Under the assumption of independent variables, the hypergeometric test gives the probability of having <em>x</em> number of “significant genes” belonging to the same pathway.<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/9/1067/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/9/1067/embed/mml-math-1.gif"/></noscript></span></span>where <em>m</em> represents the number of genes in the pathway, <em>K</em> is the number of significant genes in the pathway, <em>N</em> the total number of genes contained in the dataset, and <em>S</em> is the total number of significant genes in the dataset (according to their joint gene <em>P</em> values). To apply the hypergeometric test, we needed one <em>P</em> value per gene; however, the number of SNPs mapped to a specific gene varies across the whole dataset, and therefore, a variable number of SNP <em>P</em> values of association are assigned to each gene. The joint gene <em>P</em> value was calculated using the Fisher’s combination test as described by <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Peng <em>et al.</em> (2010)</a>.<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/9/1067/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/9/1067/embed/mml-math-2.gif"/></noscript></span></span>where <em>P<sub>i</sub></em> represents the SNP association <em>P</em> values, and <em>K</em> is the number of SNPs to be combined.</p><p id="p-10">For this study, we considered the genes to be significantly associated with a trait if their gene <em>P</em> value is less than α ≤ 0.05. The hypergeometric test was calculated for the 51 traits and for each of the 225 pathways in the KEGG database (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Ogata <em>et al.</em> 1999</a>).</p></div><div id="sec-5" class="subsection"><h3>Circular and random permutations</h3><p id="p-11">We used circular genomic permutations, gene-level random permutations, and SNP-level random permutations to obtain the distributions of the hypergeometric test pathway-association <em>P</em> values under the null hypothesis of no association. The three permutation types were performed for each test (for each pathway for each trait) and are described below.</p><div id="sec-6" class="subsection"><h4>Circular genomic permutations:</h4><p id="p-12">Our methodology performs the permutations at the SNP-level in a genomic manner. The complete set of SNP association <em>P</em> values are ordered according to their genomic position, first by chromosome and then by their location in the chromosome, including those SNPs genotyped in our data but not annotated to any gene. We consider the genome to be circular and ordered from chromosome 1 to chromosome X and restarting at chromosome 1 again. Then the complete set of SNP association <em>P</em> values are permuted by rotation with respect to their genomic locations, <em>i.e.</em>, a random number between 1 and the total number of SNPs is drawn, and the <em>P</em> value associated with the first SNP in the genome rotates to that of the random number-th SNP and all other <em>P</em> values rotate to the same degree to the corresponding SNPs. SNPs thus retain the same position with respect to each other but, at each permutation, gain new random values of association with adjacent <em>P</em> values showing similar patterns of correlation as found in the original data. For each permutation, the joint gene <em>P</em> values are calculated using Fisher’s combination test followed by the hypergeometric test (<a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Peng <em>et al.</em> 2010</a>). This process is repeated 10,000 times to obtain the test statistic distribution under the null hypothesis of no pathway effect on the phenotype.</p><p id="p-13">The pathway significance threshold was set at 5% of the distribution of the test statistics obtained from each pathway-trait test permutation; each pathway-trait test distribution consists of 10,000 permuted hypergeometric results. The hypergeometric-empirical <em>P</em> values were calculated from the ranked position of the hypergeometric-theoretical <em>P</em> value in the permutations.</p></div><div id="sec-7" class="subsection"><h4>Gene-level and SNP-level random permutations:</h4><p id="p-14">To explore how circular permutations perform compared with random permutations, alternative permutation strategies were applied to the distance 0 dataset. These permutations randomly attributed the observed trait-association <em>P</em> values to either SNPs or genes, as appropriate, without accounting for spatial (and other possible) correlations among them. Both random permutation procedures consisted of 10,000 runs on every trait for all the 225 pathways.</p><div id="sec-8" class="subsection"><h5>SNP-level random permutations:</h5><p id="p-15">The difference between the SNP-level random permutation and circular permutation is that the association <em>P</em> values are randomly given to <em>n</em> SNPs, where <em>n</em> is the total number of SNPs per gene for each trait in the initial GWAS. This simulates a situation where SNPs within a gene are not correlated due either to LD or to their collocation within a gene. Random selection of SNP <em>P</em> values is followed by the Fisher’s combination test and the hypergeometric test.</p></div><div id="sec-9" class="subsection"><h5>Gene-level random permutations:</h5><p id="p-16">For the gene-level random permutations, the gene <em>P</em> values where calculated once for the original (non-permuted) dataset. This permutation consists of re-sampling genes according to the number of genes represented in our dataset per pathway with their <em>P</em> values. This simulates a situation where genes within a pathway are not correlated due to any spatial or functional clustering within the genome.</p></div></div></div></div><div class="section results" id="sec-10"><h2 class="">RESULTS</h2><div id="sec-11" class="subsection"><h3>GWAS data annotations</h3><p id="p-17">After quality control and genome-wide analysis of the data, 318,235 SNP association <em>P</em> values were taken for further analyses. Approximately 42% of the SNPs were successfully annotated to a gene (∼137,808 SNPs) at distance 0. The percentage of annotated SNPs rises to ∼61% for the 20 kb annotation dataset. According to the NCBI gene statistics, currently there are ∼42,059 unique human gene identifiers (as of February 2012; <a href="http://www.ncbi.nlm.nih.gov/projects/Gene/gentrez_stats.cgi?TAXORG=9606">http://www.ncbi.nlm.nih.gov/projects/Gene/gentrez_stats.cgi?TAXORG=9606</a>). The numbers of unique gene identifiers observed in the datasets were 17,806 and 30,580 for distances 0 and 20 kb, respectively. The analyzed pathways have a total of 5859 unique genes. We found 4403 genes mapping to pathways in the distance 0 dataset, whereas in the 20 kb dataset, the number goes up to 5595 genes.</p></div><div id="sec-12" class="subsection"><h3>Hypergeometric test results</h3><p id="p-18">For each dataset, 11,475 hypergeometric tests were performed (from a combination of 225 pathways and 51 traits). At a <em>P</em> ≤ 0.05 threshold, we would expect ∼574 significant results by chance alone. The hypergeometric results of the distance 0 dataset produced 1423 tests below the 0.05 threshold; therefore, more than twice the number of significant tests than expected by chance was observed. The number of significant tests below the 0.05 threshold on the 20 kb annotation increased to 1805.</p></div><div id="sec-13" class="subsection"><h3>Circular genomic permutation results</h3><div id="sec-14" class="subsection"><h4>Distance 0:</h4><p id="p-19">The results based on the empirical pathway significance threshold found 536 pathway-trait tests to be statistically significant (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.002618/-/DC1/TableS2.xlsx">Table S2</a>). The range of the pathway significance thresholds observed across all the tests (all pathway-trait combinations) was very variable (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). For distance 0, the maximum empirical pathway significance threshold across tests corresponded to a <em>P</em> value of 0.175 (<em>i.e.</em>, for one pathway-trait combination, hypergeometric-empirical <em>P</em> values below 0.175 would be significant at the 0.05 level as determined empirically by circular permutation). However, for most of the pathway-trait combinations, the hypergeometric-theoretical <em>P</em> value of 0.05 was too liberal for the empirical threshold of 0.05. For the empirical <em>P</em> value threshold of 0.05, the inter-quartile distribution (<em>i.e.</em>, central 50% of the tests) corresponded to <em>P</em> values from ∼0.02 to ∼0.03. As a result, more than 60% of the hypergeometric tests that were classified as significant when using the hypergeometric test alone do not remain statistically significant at α (<em>P</em> ≤ 0.05) if the empirical pathway significance thresholds are used. The opposite was also observed for some trait-pathway combinations; thus, five results with the hypergeometric-theoretical <em>P</em> value less significant than α (<em>i.e.</em>, non-significant), are deemed to be significant according to the permutations. These results correspond to four different pathways and five different traits.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/9/1067/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Threshold distributions at distances 0 and 20 kb. Distance 0: The first quartile of the tests sets a threshold of &#x223C;0.02 and the third quartile is set at &#x223C;0.03, whereas the maximum value was equal to 0.175. The range of threshold distributions at 20 kb was smaller than that observed at distance 0. The first quartile had a threshold of &#x223C; 0.008, the third only went up to &#x223C;0.02, and the maximum threshold set was 0.093." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-903947176" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Threshold distributions at distances 0 and 20 kb. Distance 0: The first quartile of the tests sets a threshold of &#x223C;0.02 and the third quartile is set at &#x223C;0.03, whereas the maximum value was equal to 0.175. The range of threshold distributions at 20 kb was smaller than that observed at distance 0. The first quartile had a threshold of &#x223C; 0.008, the third only went up to &#x223C;0.02, and the maximum threshold set was 0.093.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/9/1067/F1.medium.gif" width="440" height="402"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1&#x2002;" src="https://www.g3journal.org/content/ggg/2/9/1067/F1.medium.gif" width="440" height="402"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/9/1067/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/9/1067/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461701" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1 </span> <p id="p-20" class="first-child">Threshold distributions at distances 0 and 20 kb. Distance 0: The first quartile of the tests sets a threshold of ∼0.02 and the third quartile is set at ∼0.03, whereas the maximum value was equal to 0.175. The range of threshold distributions at 20 kb was smaller than that observed at distance 0. The first quartile had a threshold of ∼ 0.008, the third only went up to ∼0.02, and the maximum threshold set was 0.093.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-15" class="subsection"><h4>Distance 20 kb:</h4><p id="p-21">A total of 594 tests were found to be significant at the empirical threshold (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.002618/-/DC1/TableS3.xlsx">Table S3</a>). The pathway significance thresholds were smaller than those observed at distance 0. The inter-quartile threshold range was from ∼0.008 up to ∼0.02, and the maximum pathway significance threshold set was equal to 0.093 (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>). Here four tests were also observed to be significant in the permutations but with a theoretical hypergeometric probability less significant than α (<em>P</em> ≥ 0.05). However, these tests are not the same as the ones detected by the distance 0 analysis. The analyses at distances 0 and 20 kb share a total of 198 (∼34%) significant tests (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.002618/-/DC1/TableS4.xlsx">Table S4</a>).</p></div></div><div id="sec-16" class="subsection"><h3>Hypergeometric test <em>vs.</em> circular permutations</h3><p id="p-22">The hypergeometric-theoretical <em>P</em> values were compared with those obtained by permutation analysis. The correlation between the hypergeometric-theoretical <em>P</em> values and the hypergeometric-empirical <em>P</em> values seemed to be very high (<em>i.e.</em>, 0.840 and 0.885 for the 0 kb and 20 kb analyses, respectively). The hypergeometric-empirical <em>P</em> values deviate notably from the estimated probability of the hypergeometric test; for most tests, the hypergeometric-empirical <em>P</em> values are larger (<em>i.e.</em>, less significant) than those obtained by the hypergeometric test (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/9/1067/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Hypergeometric-theoretical P values vs. hypergeometric-empirical P values at distances 0 and 20 kb. Visual representation of the relationship of the hypergeometric-theoretical P values (x-axis) compared with the hypergeometric-empirical P values from the permutations (y-axis). To assess the effect of the size of the pathways (i.e., number of genes in the pathway), P values are colored by pathway size. For this representation, pathways were clustered using k-means with eight groups. The legend represents the centers of that size group. (Top plots) The red line represents the trend that would be followed if the hypergeometric-empirical P values would match perfectly to those of the hypergeometric-theoretical P values. (Bottom plots) Close-up from the top plot, where the red lines are fixed at a 0.05 empirical threshold. The close-up graph represents all the tests below an arbitrary threshold set at 0.05 when using the hypergeometric test alone. The results below the line represent the significant results when applying circular permutations; results above the line are those which were not longer significant according to this method." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-903947176" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Hypergeometric-theoretical &lt;em&gt;P&lt;/em&gt; values &lt;em&gt;vs.&lt;/em&gt; hypergeometric-empirical &lt;em&gt;P&lt;/em&gt; values at distances 0 and 20 kb. Visual representation of the relationship of the hypergeometric-theoretical &lt;em&gt;P&lt;/em&gt; values (x-axis) compared with the hypergeometric-empirical &lt;em&gt;P&lt;/em&gt; values from the permutations (y-axis). To assess the effect of the size of the pathways (&lt;em&gt;i.e.&lt;/em&gt;, number of genes in the pathway), &lt;em&gt;P&lt;/em&gt; values are colored by pathway size. For this representation, pathways were clustered using k-means with eight groups. The legend represents the centers of that size group. (Top plots) The red line represents the trend that would be followed if the hypergeometric-empirical &lt;em&gt;P&lt;/em&gt; values would match perfectly to those of the hypergeometric-theoretical &lt;em&gt;P&lt;/em&gt; values. (Bottom plots) Close-up from the top plot, where the red lines are fixed at a 0.05 empirical threshold. The close-up graph represents all the tests below an arbitrary threshold set at 0.05 when using the hypergeometric test alone. The results below the line represent the significant results when applying circular permutations; results above the line are those which were not longer significant according to this method.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/9/1067/F2.medium.gif" width="440" height="367"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2&#x2002;" src="https://www.g3journal.org/content/ggg/2/9/1067/F2.medium.gif" width="440" height="367"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/9/1067/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/9/1067/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461799" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2 </span> <p id="p-23" class="first-child">Hypergeometric-theoretical <em>P</em> values <em>vs.</em> hypergeometric-empirical <em>P</em> values at distances 0 and 20 kb. Visual representation of the relationship of the hypergeometric-theoretical <em>P</em> values (x-axis) compared with the hypergeometric-empirical <em>P</em> values from the permutations (y-axis). To assess the effect of the size of the pathways (<em>i.e.</em>, number of genes in the pathway), <em>P</em> values are colored by pathway size. For this representation, pathways were clustered using k-means with eight groups. The legend represents the centers of that size group. (Top plots) The red line represents the trend that would be followed if the hypergeometric-empirical <em>P</em> values would match perfectly to those of the hypergeometric-theoretical <em>P</em> values. (Bottom plots) Close-up from the top plot, where the red lines are fixed at a 0.05 empirical threshold. The close-up graph represents all the tests below an arbitrary threshold set at 0.05 when using the hypergeometric test alone. The results below the line represent the significant results when applying circular permutations; results above the line are those which were not longer significant according to this method.</p><div class="sb-div caption-clear"></div></div></div><p id="p-24">To investigate further the causes of the difference between the hypergeometric-theoretical <em>P</em> values and the permutation results, we carried out linear regressions in which our observed variables were the hypergeometric-theoretical <em>P</em> values, the hypergeometric-empirical <em>P</em> values, and the difference between the hypergeometric-theoretical <em>P</em> values and the hypergeometric-empirical <em>P</em> values. The predictive variables used were functions of the number of genes, the number of SNPs, or the SNP-gene ratio in the pathways. These predictors were categorized into groups representing the size of the pathway (<em>i.e.</em>, amount of genes/SNPs/SNP-gene ratio on each pathway). For example, pathways containing from 1 to 25 genes would belong to category A, while pathways containing 26–50 genes would belong to category B. The pathways analyzed consisted approximately of 100 different pathway sizes according to the number of genes, ∼200 according to the SNPs, and ∼130 according to the SNP-gene ratio. To assign the pathways to their representative category, we used the <em>k-means</em> algorithm (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Hartigan and Wong 1979</a>) within R. Genes were grouped in 5 clusters, SNPs in 12, and the SNP-gene ratio into 7. The <em>r<sup>2</sup></em> results demonstrate that none of these three predictive variables has a strong impact on the theoretical or the empirical results (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). However, a high <em>r<sup>2</sup></em> was observed for the regression on the differences between theoretical and the empirical <em>P</em> values, where the <em>r<sup>2</sup></em> goes up to 0.431 for the number of SNPs being the predictive variable, where the difference between the methodologies is greater in pathways of extreme sizes (<em>i.e.</em>, very small pathways containing ∼23 SNPs or very large pathways ∼1,500 SNPs), implying that the number of SNPs in the pathways may influence the differences between the two methodologies. However, this was not surprising as the <em>P</em> values vary widely between the theoretical and the empirical approaches, especially for those extreme-sized pathways where more significant results (small <em>P</em> values) were observed by the theoretical-hypergeometric test, but when analyzed through permutations, these results obtain very large <em>P</em> values.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/461751/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/461751/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1 </span> <span class="caption-title"><em>r</em><sup>2</sup> results for hypergeometric-theoretical <em>P</em> values, hypergeometric-empirical <em>P</em> values, and <em>r</em><sup>2</sup> difference between the hypergeometric-theoretical <em>P</em> values and the hypergeometric-empirical <em>P</em> values</span><div class="sb-div caption-clear"></div></div></div><div id="sec-17" class="subsection"><h4>Random permutations vs. circular permutations:</h4><p id="p-26">We explored the distribution of individual tests of the permuted hypergeometric results (<em>i.e.</em>, distribution of <em>P</em> values for each pathway-trait combination for both random permutations and the circular permutations). <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a> shows these distributions for the glucose trait across three pathways and highlights the differences between them across permutation methods and pathways. Each plot represents the outcome of the 10,000 permuted hypergeometric tests.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/9/1067/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Permutation distributions. Three pathways and their hypergeometric P value permuted distributions for the glucose trait. Each individual plot represents the outcome of the 10,000 permuted hypergeometric tests per pathway. These three pathways were selected because they represent the three trends observed across all the analyzed pathways. The left column represents the circular genomic permutation P value distribution; the central column represents the SNP-level random permutation P value distribution, and the right column represents the gene-level random permutation P value distribution." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-903947176" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Permutation distributions. Three pathways and their hypergeometric &lt;em&gt;P&lt;/em&gt; value permuted distributions for the glucose trait. Each individual plot represents the outcome of the 10,000 permuted hypergeometric tests per pathway. These three pathways were selected because they represent the three trends observed across all the analyzed pathways. The left column represents the circular genomic permutation &lt;em&gt;P&lt;/em&gt; value distribution; the central column represents the SNP-level random permutation &lt;em&gt;P&lt;/em&gt; value distribution, and the right column represents the gene-level random permutation &lt;em&gt;P&lt;/em&gt; value distribution.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/9/1067/F3.medium.gif" width="440" height="258"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3&#x2002;" src="https://www.g3journal.org/content/ggg/2/9/1067/F3.medium.gif" width="440" height="258"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/9/1067/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/9/1067/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461731" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3 </span> <p id="p-27" class="first-child">Permutation distributions. Three pathways and their hypergeometric <em>P</em> value permuted distributions for the glucose trait. Each individual plot represents the outcome of the 10,000 permuted hypergeometric tests per pathway. These three pathways were selected because they represent the three trends observed across all the analyzed pathways. The left column represents the circular genomic permutation <em>P</em> value distribution; the central column represents the SNP-level random permutation <em>P</em> value distribution, and the right column represents the gene-level random permutation <em>P</em> value distribution.</p><div class="sb-div caption-clear"></div></div></div><p id="p-28">In <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>, we can observe the hypergeometric-empirical <em>P</em> values obtained through the various permutation methodologies compared with the hypergeometric-theoretical <em>P</em> values. Here, gene-level random permutation results seem almost identical to those predicted by the hypergeometric test. The correlation between the hypergeometric-theoretical <em>P</em> values and the hypergeometric-empirical <em>P</em> values for the gene-level random permutations was 0.99, and the maximum difference observed among all the 11,475 tests was only 0.02. This degree of similarity is not observed in the SNP-level permutations (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/9/1067/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Permutation methods. This plot compares the hypergeometric P values on the x-axis to the permutation P values in the y-axis. The three permutation methodologies are represented (circular genomic permutation, the SNP-level random permutation, and the gene-level random permutations)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-903947176" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Permutation methods. This plot compares the hypergeometric &lt;em&gt;P&lt;/em&gt; values on the x-axis to the permutation &lt;em&gt;P&lt;/em&gt; values in the y-axis. The three permutation methodologies are represented (circular genomic permutation, the SNP-level random permutation, and the gene-level random permutations).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/9/1067/F4.medium.gif" width="440" height="260"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4&#x2002;" src="https://www.g3journal.org/content/ggg/2/9/1067/F4.medium.gif" width="440" height="260"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/9/1067/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/9/1067/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461830" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4 </span> <p id="p-29" class="first-child">Permutation methods. This plot compares the hypergeometric <em>P</em> values on the x-axis to the permutation <em>P</em> values in the y-axis. The three permutation methodologies are represented (circular genomic permutation, the SNP-level random permutation, and the gene-level random permutations).</p><div class="sb-div caption-clear"></div></div></div></div></div><div id="sec-18" class="subsection"><h3>Variance explained by arrhythmogenic right ventricular cardiomyopathy: an example</h3><p id="p-30">To determine whether the pathway-based circular permutation approach detects more effects and can explain more variation within a trait than the traditional single-marker approach; the arrhythmogenic right ventricular cardiomyopathy (ARVC) pathway was selected to investigate how much variance was explained by the SNPs within the pathway. ARVC was selected because it was found to be highly significant in the distances 0 and 20 kb datasets for the trait glucose (hypergeometric-empirical <em>P</em> values of 0.0004 and 0.0006, respectively). The traits glucose and waist-to-height ratio were the only traits found to be significant for the ARVC pathway. Four other pathways were significant for the same two traits: cardiac muscle contraction, dilated cardiomyopathy (DCM), hypertrophic cardiomyopathy (HCM), and fc gamma R-mediated phagocytosis. <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a> is a representation of the significant pathways and traits related to ARVC. However, ARVC, HCM, and DCM share 50 genes in total. This represents 72% of the ARVC genes, 65% of HCM, and 63% DCM; there are no genes shared between the cardiovascular disease pathways and the fc-gamma R-mediated phagocytosis. The fc-gamma receptors are linked to the initiation of various signals (<em>e.g</em>., actin-cytoskeleton reorganization, and phagosomal membrane remodeling) (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Garcia-Garcia and Rosales 2002</a>).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/9/1067/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Significant tests related to the arrhythmogenic right ventricular cardiomyopathy (ARVC) pathway. Two traits (glucose and waist-to-height ratio) were found to be significant for the ARVC pathway. Both traits share a total of five pathways significant for both traits. Traits are represented by sphere nodes, and pathways are represented by the icosahedron nodes. The pathways are colored according to their pathway category. The link between the pathways and the traits (edges) represent the significant results obtained through the circular genomic permutation approach. Edges are colored from a blue-to-red scale, where blue represents the most significant results (i.e., the ARVC pathway and the trait glucose hypergeometric-empirical P value = 0.0004, whereas the linoleic acid metabolism and glucose hypergeometric-empirical P value = 0.044). Image produced using BioLayout Express (Freeman et al. 2007)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-903947176" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Significant tests related to the arrhythmogenic right ventricular cardiomyopathy (ARVC) pathway. Two traits (glucose and waist-to-height ratio) were found to be significant for the ARVC pathway. Both traits share a total of five pathways significant for both traits. Traits are represented by sphere nodes, and pathways are represented by the icosahedron nodes. The pathways are colored according to their pathway category. The link between the pathways and the traits (edges) represent the significant results obtained through the circular genomic permutation approach. Edges are colored from a blue-to-red scale, where blue represents the most significant results (&lt;em&gt;i.e.&lt;/em&gt;, the ARVC pathway and the trait glucose hypergeometric-empirical &lt;em&gt;P&lt;/em&gt; value = 0.0004, whereas the linoleic acid metabolism and glucose hypergeometric-empirical &lt;em&gt;P&lt;/em&gt; value = 0.044). Image produced using BioLayout Express (Freeman &lt;em&gt;et al.&lt;/em&gt; 2007).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/9/1067/F5.medium.gif" width="440" height="240"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5&#x2002;" src="https://www.g3journal.org/content/ggg/2/9/1067/F5.medium.gif" width="440" height="240"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/9/1067/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/9/1067/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461868" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5 </span> <p id="p-31" class="first-child">Significant tests related to the arrhythmogenic right ventricular cardiomyopathy (ARVC) pathway. Two traits (glucose and waist-to-height ratio) were found to be significant for the ARVC pathway. Both traits share a total of five pathways significant for both traits. Traits are represented by sphere nodes, and pathways are represented by the icosahedron nodes. The pathways are colored according to their pathway category. The link between the pathways and the traits (edges) represent the significant results obtained through the circular genomic permutation approach. Edges are colored from a blue-to-red scale, where blue represents the most significant results (<em>i.e.</em>, the ARVC pathway and the trait glucose hypergeometric-empirical <em>P</em> value = 0.0004, whereas the linoleic acid metabolism and glucose hypergeometric-empirical <em>P</em> value = 0.044). Image produced using BioLayout Express (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Freeman <em>et al.</em> 2007</a>).</p><div class="sb-div caption-clear"></div></div></div><p id="p-32">The mixed linear model implemented in the <em>polygenic</em>() function of GenABEL was used to calculate the variance explained by the SNPs with an association <em>P</em> value more significant than α (<em>P</em> ≤ 0.05) for the trait glucose and all traits measured for the ARVC pathway. The analysis was performed using the distance 0 annotation. A total of ∼2311 SNPs were mapped to ARVC, from which 175 were below an association <em>P</em> value of 0.05 for the glucose trait. The percentage of the variance explained was given by the difference between the total estimated heritability and the estimated heritability with fixed effects, divided by the total estimated heritability. The estimated percentage of the variance of the trait glucose explained by the SNPs in the ARVC pathway with <em>P</em> value more significant than α ≤ 0.05 was ∼24%. In addition, only three other traits had some of the variance explained by the same SNPs: ∼20% variance of Hba1c (glycated hemoglobin, a measure of the average glucose plasma levels over time); ∼18% of CRP (C-reactive protein, an independent biomarker for cardiovascular diseases) (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Kones 2010</a>); and ∼40% of CIMT (carotid intima-media thickness, a measure of atherosclerosis). The phenotypic correlations of Hba1c, CRP, and CIMT with the trait glucose are 0.37, 0.03, and 0.10, respectively. No other trait had any significant amount of its variance explained by these SNPs in the ARVC pathway.</p></div></div><div class="section discussion" id="sec-19"><h2 class="">DISCUSSION</h2><p id="p-33">In spite of the substantial developments in genetics and genomics, there are many unresolved questions, and the mechanisms driving complex diseases remain very poorly explained. Major advances have become available by sequencing genomes, characterizing their structure, and identifying the common elements across species (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Mouse Genome Sequencing Consortium 2002</a>). Despite these advances, there are still regions whose functions or mechanisms of action are still unknown or not fully understood; an example of this is a recent study that demonstrates natural selection also acting on non-coding regions (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Mu <em>et al.</em> 2011</a>). Furthermore, gene expression studies have identified chromosomal regions, groups of contiguous genes characterized by coordinated expression and similar transcriptional profiles (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Caron <em>et al.</em> 2001</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Versteeg <em>et al.</em> 2003</a>). This has led to the development of approaches to take into account physical locations and genomic distances (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Callegaro <em>et al.</em> 2006</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Seifert <em>et al.</em> 2011</a>).</p><p id="p-34">Here we propose a novel approach for testing association of gene sets taking genomic structure and non-independence among SNPs into account: the circular permutation method.</p><p id="p-35">The circular genomic permutation was applied to a small study (N = 719 individuals). Single-marker GWAS analysis detected 11 SNPs across three genomic regions and three traits more significant than 5 × 10<sup>−8</sup> (generally accepted GWAS threshold). The small number of individuals genotyped in this study results in low power to detect associations reaching the stringent genome-wide threshold. This in turn results in true associations remaining undetected. The aim of the gene-set analysis using circular genomic permutation is to detect associations missed by standard GWAS single-marker association analyses by identifying SNPs with joint effects on a phenotype. Although our study provides a large number of phenotypic data, a larger study would have the potential to yield results that give us a better biological insight of the gene sets.</p><p id="p-36">The methodology was applied to a study genotyped on a single platform. In instances where studies use a variety of platforms, imputation of genotypes at loci that are not present in all platforms, prior to the association analysis that will produce the association <em>P</em> values that are used in our gene-set analysis is recommended.</p><p id="p-37">We included the X chromosome in the analysis (and permutations). LD in chromosome X is higher than for the autosomes (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Schaffner 2004</a>). We do not expect this to introduce a substantial bias in our case where the number of SNPs on the X chromosome is not high (∼2% of the total number of SNPs). However, an alternative approach could be to perform the permutations at a chromosomal instead of at the genomic level.</p><p id="p-38">We compared the performance of our approach with that of two other, often used permutation methods (<em>i.e.</em>, gene-level and SNP-level random permutations). The hypergeometric-empirical <em>P</em> values of the gene-level random permutations agreed largely with the hypergeometric-theoretical <em>P</em> values because the number of significant genes is constant across all tests and the only source of variation is derived from how many of the re-sampled genes fall in pathways below the α threshold. Not all correlations of the initial SNP association <em>P</em> values are broken in this type of permutation, as the gene <em>P</em> values were derived from consecutive SNPs that were annotated to the same gene. In addition, the simulated pathways would mainly contain non-correlated gene <em>P</em> values, and therefore, the assumption of the hypergeometric test is not violated in this scenario. For the SNP-level random permutations, the SNPs are taken randomly from the genome and a new gene <em>P</em> value calculated from non-correlated SNPs, leading to different number of significant genes in the pathways and also in the universe (complete dataset), consequently generating more variation in the permuted hypergeometric results. In contrast, the circular genomic permutation, which is also performed at the SNP-level where the number of significant genes in the pathway and in the universe are not fixed (similarly to the SNP-level random permutations), seems to capture the genomic structure and shows how likely an outcome is expected according to this structure (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>).</p><p id="p-39">Finding a consensus between pathway-based approaches represents a challenge. Several methodologies demand a single association value per gene. Many of the methodologies select the strongest SNP signal of the gene, but doing so will overestimate the association for pathways with long genes. We chose to calculate a single gene <em>P</em> value through the Fisher’s combination test, although this may be conservative for the opposite reason—long genes with true associations may contain many SNPs with no functional effect that are not in LD with any functional variants. Another parameter to select is the window size to apply in the SNP-to-gene annotation. Based on the findings of <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Veyrieras <em>et al.</em> (2008)</a>, which estimated that 95% of the expression quantitative trait loci localize within the genes and within 20 kb from the genes, the 20 kb window-size annotation is becoming more popular (<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Holmans <em>et al.</em> 2009</a>; <a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Wang <em>et al.</em> 2011b</a>). A clear advantage of including a window region for the annotation is the number of SNPs that will be annotated to genes. Here the number of SNPs successfully annotated to genes increased by 20% when the annotation window was set to 20 kb, allowing the capture of 1192 genes mapped to pathways that were missed before but that may influence the trait. An undesired effect may be the addition of noise to the analyses, as some SNPs included in the larger window-size analyses will have no real effect on the trait, and the proportion of SNPs annotated to various genes that might be acting under different pathways may increase. The effect of the noise on the data may also be observed on the differences between the pathway significance thresholds set through permutations, where the distance 0 analysis sets on average a pathway significance threshold of ∼0.02–0.03, and the distance 20 kb, ∼0.008–0.02. The distribution of the 20 kb permuted-hypergeometric <em>P</em> values approximates to the distributions observed in the gene-level random distributions. The overlap between the significant results of the distances 0 and the 20 kb datasets was weak; only 198 tests out of 536 (0 kb) and 594 (20 kb) were observed as significant in both analyses, suggesting that the annotation has a substantial contribution to the pathway association results. Some instances in which the distances 0 and 20 kb analyses agreed included two tests of the five in distance 0 found significant through permutations but with a theoretical hypergeometric <em>P</em> &gt; 0.05. They were both confirmed as significant in the 20 kb analyses with a hypergeometric test <em>P</em> value of <em>P</em> = 0.04 and <em>P</em> = 0.001, and both also significant through the circular permutations of 20 kb.</p><p id="p-40">In both analyses (distances 0 and 20 kb), the correlation of the hypergeometric-empirical <em>P</em> values with the hypergeometric-theoretical <em>P</em> values is very strong. However, the strong correlation only shows the degree of similarity between the ranked order between the empirical <em>P</em> values and the theoretical <em>P</em> values. As shown in <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>, the empirical <em>P</em> values tend to be larger than those obtained only through the hypergeometric test, which means that the use of the hypergeometric test alone will lead to false-positive results.</p><p id="p-41">We also looked at the impact of the number of SNPs and the number of genes of the pathway. As shown in <a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a>, the hypergeometric-theoretical <em>P</em> values below 0.05 but not empirically significant at the same level included pathways of different sizes in terms of number of genes, and this pattern was also observed when the tests results were clustered by SNPs (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.002618/-/DC1/FigureS1.pdf">Figure S1</a>).</p><p id="p-42">We presented as an example the significant pathways and traits linked to one of the most significant pathway-trait associations found through this methodological study. The ARVC pathway groups genes involved in one type of cardiomyopathy (characterized by replacement of the myocardium by fibrofatty tissue). Cardiomyopathy is a dysfunction of the heart muscle highly associated with sudden death, especially in the young (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Towbin <em>et al.</em> 2006</a>). No scientific references were found for the link between ARVC and the trait glucose. However, a close form of inherited cardiomyopathy has been linked to the PRKAG2 gene, a master regulator of glucose and lipid metabolism (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Gollob <em>et al.</em> 2001</a>), and one of the major causes of dilated cardiomyopathy, the most common type of cardiomyopathy, is hypertension and ischemic cardiomyopathy (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Hunt <em>et al.</em> 2005</a>). The latter is also associated with abnormal glucose levels (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Lopaschuk and Stanley 1997</a>). Our results show how the three types of cardiomyopathy pathways (in KEGG) and the cardiac muscle contraction pathway are found significantly associated to the traits glucose and waist-to-height ratio. Furthermore, the catalog of published genome-wide association studies (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Hindorff <em>et al.</em> 2009</a>) reports a meta-analysis in which 17 genes were found significantly associated to fasting glucose (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Dupuis <em>et al.</em> 2010</a>). Among these reported genes, two genes (ADCY5 and TCF7L2) were also found annotated to ARVC and dilated cardiomyopathy pathways (<a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Ogata <em>et al.</em> 1999</a>). We present the variance explained by the SNPs as a preliminary result; because the SNPs chosen for the analysis were those whose <em>P</em> values of association were below 0.05, there is an expected degree of variance explained by the subset chosen for the analysis. Subsequently, the present analysis is likely to overestimate the variance explained by the SNPs, and a better methodology needs to be developed. Still, because of the small sample size of the study (n = 719), we do not claim to detect the link between the ARVC pathway and the trait glucose; to confirm this result, replication in another cohort would be required.</p><p id="p-43">The rigorous evaluation of pathway methodologies is very challenging, but the application of permutations can help in their assessment. Our results demonstrate that the circular genomic permutations provide robust associations. A thoughtful consideration of the advantages and disadvantages on the window-size annotation should be done according to the objective of the study. Due to the various possible sources of bias, keeping the annotation window size shorter than 20 kb is recommended. Our new methodology permits the spurious associations that are detected when not accounting for genomic structure to be discarded. According to the hypergeometric-theoretical results, ∼1,400 pathway trait results have a <em>P</em> ≤ 0.05; however, empirical results suggest that ∼60% of them could arise just by chance. In this study, we applied Fisher’s combination test and a hypergeometric test to test for pathway significance; however, this permutation approach can be applied to any SNP-to-gene or SNP-to-pathway <em>P</em> value estimation methodology and to any other gene sets or pathway resources.</p></div><div class="section ack" id="ack-1"><h2 class="">ACKNOWLEDGMENTS</h2><p id="p-44">We acknowledge the financial support provided by the MRC Core Fund and the Biotechnology and Biological Sciences Research Council. ORCADES was supported by the Scottish Executive Health Department, the Royal Society, and the European Union framework program 6 EUROSPAN project.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.002618/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.002618/-/DC1</a>.</p></li><li class="fn" id="fn-3"><p id="p-45"><em>Communicating editor: B. S. Yandell</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2012-03-15"><span class="received-label">Received </span>March 15, 2012.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2012-06-29"><span class="accepted-label">Accepted </span>June 29, 2012.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2012 Cabrera <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">LITERATURE CITED</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1067.1" data-doi="10.1093/bioinformatics/btm108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Aulchenko</span>  <span class="cit-name-given-names">Y. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ripke</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Isaacs</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Duijn</span>  <span class="cit-name-given-names">C. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">GenABEL: an R library for genome-wide association analysis</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">1294</span>–<span class="cit-lpage">1296</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtm108%26rft_id%253Dinfo%253Apmid%252F17384015%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjMvMTAvMTI5NCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzIvOS8xMDY3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1067.2" data-doi="10.1186/1755-8794-3-25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ballard</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abraham</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cho</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Pathway analysis comparison using Crohn’s disease genome wide association studies</span>. <abbr class="cit-jnl-abbrev">BMC Med. Genomics</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">25</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DBMC%2BMed%2BGenomics%26rft.aulast%253DBallard%26rft.auinit1%253DD.%26rft.volume%253D3%26rft.spage%253D25%26rft.epage%253D25%26rft.atitle%253DPathway%2Banalysis%2Bcomparison%2Busing%2BCrohn%2527s%2Bdisease%2Bgenome%2Bwide%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1755-8794-3-25%26rft_id%253Dinfo%253Apmid%252F20584322%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1755-8794-3-25&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20584322&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F9%2F1067.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1067.3" data-doi="10.1093/hmg/ddp120"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Baranzini</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Galwey</span>  <span class="cit-name-given-names">N. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Khankhanian</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lindberg</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Pathway and network-based analysis of genome-wide association studies in multiple sclerosis</span>. <abbr class="cit-jnl-abbrev">Hum. Mol. Genet.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">2078</span>–<span class="cit-lpage">2090</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHum.%2BMol.%2BGenet.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fhmg%252Fddp120%26rft_id%253Dinfo%253Apmid%252F19286671%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiaG1nIjtzOjU6InJlc2lkIjtzOjEwOiIxOC8xMS8yMDc4IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMi85LzEwNjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1067.4" data-doi="10.1093/bioinformatics/btl455"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Callegaro</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Basso</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bicciato</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">A locally adaptive statistical procedure (LAP) to identify differentially expressed chromosomal regions</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">2658</span>–<span class="cit-lpage">2666</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtl455%26rft_id%253Dinfo%253Apmid%252F16951291%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjIvMjEvMjY1OCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzIvOS8xMDY3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1067.5" data-doi="10.1126/science.1056794"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Caron</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schaik</span>  <span class="cit-name-given-names">B. v.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mee</span>  <span class="cit-name-given-names">M. d.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baas</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Riggins</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">The human transcriptome map: clustering of highly expressed genes in chromosomal domains</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">291</span>: <span class="cit-fpage">1289</span>–<span class="cit-lpage">1292</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DCaron%26rft.auinit1%253DH.%26rft.volume%253D291%26rft.issue%253D5507%26rft.spage%253D1289%26rft.epage%253D1292%26rft.atitle%253DThe%2BHuman%2BTranscriptome%2BMap%253A%2BClustering%2Bof%2BHighly%2BExpressed%2BGenes%2Bin%2BChromosomal%2BDomains%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1056794%26rft_id%253Dinfo%253Apmid%252F11181992%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyOTEvNTUwNy8xMjg5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMi85LzEwNjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1067.6" data-doi="10.1086/521580"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">W. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abecasis</span>  <span class="cit-name-given-names">G. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Family-based association tests for genomewide association scans</span>. <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">81</span>: <span class="cit-fpage">913</span>–<span class="cit-lpage">926</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DChen%26rft.auinit1%253DW.%2BM.%26rft.volume%253D81%26rft.issue%253D5%26rft.spage%253D913%26rft.epage%253D926%26rft.atitle%253DFamily-based%2Bassociation%2Btests%2Bfor%2Bgenomewide%2Bassociation%2Bscans.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F521580%26rft_id%253Dinfo%253Apmid%252F17924335%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/521580&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17924335&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F9%2F1067.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000250480900005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1067.7" data-doi="10.1186/1471-2164-12-166"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">D’Addabbo</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Palmieri</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Latiano</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Annese</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mukherjee</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">RS-SNP: a random-set method for genome-wide association studies</span>. <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">166</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DD%2527Addabbo%26rft.auinit1%253DA.%26rft.volume%253D12%26rft.spage%253D166%26rft.epage%253D166%26rft.atitle%253DRS-SNP%253A%2Ba%2Brandom-set%2Bmethod%2Bfor%2Bgenome-wide%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-12-166%26rft_id%253Dinfo%253Apmid%252F21450072%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-12-166&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21450072&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F9%2F1067.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1067.8" data-doi="10.1038/ng.520"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dupuis</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Langenberg</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Prokopenko</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Saxena</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Soranzo</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">New genetic loci implicated in fasting glucose homeostasis and their impact on type 2 diabetes risk</span>. <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">105</span>–<span class="cit-lpage">116</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DDupuis%26rft.auinit1%253DJ.%26rft.volume%253D42%26rft.issue%253D2%26rft.spage%253D105%26rft.epage%253D116%26rft.atitle%253DNew%2Bgenetic%2Bloci%2Bimplicated%2Bin%2Bfasting%2Bglucose%2Bhomeostasis%2Band%2Btheir%2Bimpact%2Bon%2Btype%2B2%2Bdiabetes%2Brisk.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.520%26rft_id%253Dinfo%253Apmid%252F20081858%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.520&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20081858&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F9%2F1067.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000274084400005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1067.9" data-doi="10.1371/journal.pcbi.0030206"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Freeman</span>  <span class="cit-name-given-names">T. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goldovsky</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brosch</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Dongen</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maziere</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Construction, visualisation, and clustering of transcription networks from microarray expression data</span>. <abbr class="cit-jnl-abbrev">PLOS Comput. Biol.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">e206</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLOS%2BComput.%2BBiol.%26rft.volume%253D3%26rft.spage%253De206%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.0030206%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.0030206&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1067.10" data-doi="10.1038/ejhg.2011.57"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fridley</span>  <span class="cit-name-given-names">B. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Biernacka</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Gene set analysis of SNP data: benefits, challenges, and future directions</span>. <abbr class="cit-jnl-abbrev">Eur. J. Hum. Genet.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">837</span>–<span class="cit-lpage">843</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEuropean%2Bjournal%2Bof%2Bhuman%2Bgenetics%2B%253A%2B%2BEJHG%26rft.stitle%253DEur%2BJ%2BHum%2BGenet%26rft.aulast%253DFridley%26rft.auinit1%253DB.%2BL.%26rft.volume%253D19%26rft.issue%253D8%26rft.spage%253D837%26rft.epage%253D843%26rft.atitle%253DGene%2Bset%2Banalysis%2Bof%2BSNP%2Bdata%253A%2Bbenefits%252C%2Bchallenges%252C%2Band%2Bfuture%2Bdirections.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fejhg.2011.57%26rft_id%253Dinfo%253Apmid%252F21487444%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ejhg.2011.57&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21487444&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F9%2F1067.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1067.11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Garcia-Garcia</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosales</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Signal transduction during Fc receptor-mediated phagocytosis</span>. <abbr class="cit-jnl-abbrev">J. Leukoc. Biol.</abbr> <span class="cit-vol">72</span>: <span class="cit-fpage">1092</span>–<span class="cit-lpage">1108</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BLeukocyte%2BBiology%26rft.stitle%253DJ.%2BLeukoc.%2BBiol.%26rft.aulast%253DGarcia-Garcia%26rft.auinit1%253DE.%26rft.volume%253D72%26rft.issue%253D6%26rft.spage%253D1092%26rft.epage%253D1108%26rft.atitle%253DSignal%2Btransduction%2Bduring%2BFc%2Breceptor-mediated%2Bphagocytosis%26rft_id%253Dinfo%253Apmid%252F12488490%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NToiamxldWIiO3M6NToicmVzaWQiO3M6OToiNzIvNi8xMDkyIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMi85LzEwNjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1067.12" data-doi="10.1101/gr.071571.107"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ge</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Need</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fellay</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">WGAViewer: software for genomic annotation of whole genome association studies</span>. <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">640</span>–<span class="cit-lpage">643</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.071571.107%26rft_id%253Dinfo%253Apmid%252F18256235%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE4LzQvNjQwIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMi85LzEwNjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1067.13" data-doi="10.1056/NEJM200106143442403"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gollob</span>  <span class="cit-name-given-names">M. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Green</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tang</span>  <span class="cit-name-given-names">A. S. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gollob</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Karibe</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Identification of a gene responsible for familial Wolff-Parkinson-White syndrome</span>. <abbr class="cit-jnl-abbrev">N. Engl. J. Med.</abbr> <span class="cit-vol">344</span>: <span class="cit-fpage">1823</span>–<span class="cit-lpage">1831</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BEngland%2BJournal%2Bof%2BMedicine%26rft.stitle%253DNEJM%26rft.aulast%253DGollob%26rft.auinit1%253DM.%2BH.%26rft.volume%253D344%26rft.issue%253D24%26rft.spage%253D1823%26rft.epage%253D1831%26rft.atitle%253DIdentification%2Bof%2Ba%2BGene%2BResponsible%2Bfor%2BFamilial%2BWolff-Parkinson-White%2BSyndrome%26rft_id%253Dinfo%253Adoi%252F10.1056%252FNEJM200106143442403%26rft_id%253Dinfo%253Apmid%252F11407343%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1056/NEJM200106143442403&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11407343&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F9%2F1067.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000169231000003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1067.14" data-doi="10.2307/2346830"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hartigan</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wong</span>  <span class="cit-name-given-names">M. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1979</span> <span class="cit-article-title">A K-means clustering algorithm</span>. <abbr class="cit-jnl-abbrev">J. R. Stat. Soc. Ser. C Appl. Stat.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">100</span>–<span class="cit-lpage">108</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BR.%2BStat.%2BSoc.%2BSer.%2BC%2BAppl.%2BStat.%26rft.volume%253D28%26rft.spage%253D100%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2346830%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2346830&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1067.15" data-doi="10.1073/pnas.0903103106"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hindorff</span>  <span class="cit-name-given-names">L. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sethupathy</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Junkins</span>  <span class="cit-name-given-names">H. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ramos</span>  <span class="cit-name-given-names">E. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mehta</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Potential etiologic and functional implications of genome-wide association loci for human diseases and traits</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">106</span>: <span class="cit-fpage">9362</span>–<span class="cit-lpage">9367</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0903103106%26rft_id%253Dinfo%253Apmid%252F19474294%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTA2LzIzLzkzNjIiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8yLzkvMTA2Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1067.16" data-doi="10.1016/j.ajhg.2009.05.011"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Holmans</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Green</span>  <span class="cit-name-given-names">E. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pahwa</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferreira</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Purcell</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Gene ontology analysis of GWA study data sets provides insights into the biology of bipolar disorder</span>. <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">85</span>: <span class="cit-fpage">13</span>–<span class="cit-lpage">24</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DHolmans%26rft.auinit1%253DP.%26rft.volume%253D85%26rft.issue%253D1%26rft.spage%253D13%26rft.epage%253D24%26rft.atitle%253DGene%2Bontology%2Banalysis%2Bof%2BGWA%2Bstudy%2Bdata%2Bsets%2Bprovides%2Binsights%2Binto%2Bthe%2Bbiology%2Bof%2Bbipolar%2Bdisorder.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2009.05.011%26rft_id%253Dinfo%253Apmid%252F19539887%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2009.05.011&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19539887&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F9%2F1067.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000267997600004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1067.17" data-doi="10.1161/CIRCULATIONAHA.105.167586"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hunt</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abraham</span>  <span class="cit-name-given-names">W. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chin</span>  <span class="cit-name-given-names">M. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feldman</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Francis</span>  <span class="cit-name-given-names">G. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, ;<span class="cit-pub-date">2005</span> <span class="cit-article-title">ACC/AHA 2005 guideline update for the diagnosis and management of chronic heart failure in the adult</span>. <abbr class="cit-jnl-abbrev">Circulation</abbr> <span class="cit-vol">112</span>: <span class="cit-fpage">e154</span>–<span class="cit-lpage">e235</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCirculation%26rft_id%253Dinfo%253Adoi%252F10.1161%252FCIRCULATIONAHA.105.167586%26rft_id%253Dinfo%253Apmid%252F16160202%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6MTQ6ImNpcmN1bGF0aW9uYWhhIjtzOjU6InJlc2lkIjtzOjExOiIxMTIvMTIvZTE1NCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzIvOS8xMDY3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1067.18" data-doi="10.1016/j.schres.2010.07.001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jia</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meltzer</span>  <span class="cit-name-given-names">H. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">Z.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Common variants conferring risk of schizophrenia: a pathway analysis of GWAS data</span>. <abbr class="cit-jnl-abbrev">Schizophr. Res.</abbr> <span class="cit-vol">122</span>: <span class="cit-fpage">38</span>–<span class="cit-lpage">42</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSchizophrenia%2Bresearch%26rft.stitle%253DSchizophr%2BRes%26rft.aulast%253DJia%26rft.auinit1%253DP.%26rft.volume%253D122%26rft.issue%253D1-3%26rft.spage%253D38%26rft.epage%253D42%26rft.atitle%253DCommon%2Bvariants%2Bconferring%2Brisk%2Bof%2Bschizophrenia%253A%2Ba%2Bpathway%2Banalysis%2Bof%2BGWAS%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.schres.2010.07.001%26rft_id%253Dinfo%253Apmid%252F20659789%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.schres.2010.07.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20659789&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F9%2F1067.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1067.19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kones</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Rosuvastatin, inflammation, C-reactive protein, JUPITER, and primary prevention of cardiovascular disease - a perspective</span>. <abbr class="cit-jnl-abbrev">Drug Design Development and Therapy</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">383</span>–<span class="cit-lpage">413</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDrug%2BDesign%2BDevelopment%2Band%2BTherapy%26rft.volume%253D4%26rft.spage%253D383%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1067.20" data-doi="10.1161/01.CIR.95.2.313"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lopaschuk</span>  <span class="cit-name-given-names">G. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stanley</span>  <span class="cit-name-given-names">W. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Glucose metabolism in the ischemic heart</span>. <abbr class="cit-jnl-abbrev">Circulation</abbr> <span class="cit-vol">95</span>: <span class="cit-fpage">313</span>–<span class="cit-lpage">315</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCirculation%26rft.stitle%253DCirculation%26rft.aulast%253DLopaschuk%26rft.auinit1%253DG.%2BD.%26rft.volume%253D95%26rft.issue%253D2%26rft.spage%253D313%26rft.epage%253D315%26rft.atitle%253DGlucose%2BMetabolism%2Bin%2Bthe%2BIschemic%2BHeart%26rft_id%253Dinfo%253Adoi%252F10.1161%252F01.CIR.95.2.313%26rft_id%253Dinfo%253Apmid%252F9008441%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6MTQ6ImNpcmN1bGF0aW9uYWhhIjtzOjU6InJlc2lkIjtzOjg6Ijk1LzIvMzEzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMi85LzEwNjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1067.21" data-doi="10.1038/nature08494"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Manolio</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Collins</span>  <span class="cit-name-given-names">F. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cox</span>  <span class="cit-name-given-names">N. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goldstein</span>  <span class="cit-name-given-names">D. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hindorff</span>  <span class="cit-name-given-names">L. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Finding the missing heritability of complex diseases</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">461</span>: <span class="cit-fpage">747</span>–<span class="cit-lpage">753</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DManolio%26rft.auinit1%253DT.%2BA.%26rft.volume%253D461%26rft.issue%253D7265%26rft.spage%253D747%26rft.epage%253D753%26rft.atitle%253DFinding%2Bthe%2Bmissing%2Bheritability%2Bof%2Bcomplex%2Bdiseases.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature08494%26rft_id%253Dinfo%253Apmid%252F19812666%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature08494&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19812666&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F9%2F1067.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000270547500027&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-2.9.1067.22" data-doi="10.1016/j.ajhg.2008.08.007"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McQuillan</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leutenegger</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abdel-Rahman</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Franklin</span>  <span class="cit-name-given-names">C. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pericic</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Runs of homozygosity in European populations</span>. <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">83</span>: <span class="cit-fpage">359</span>–<span class="cit-lpage">372</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DMcQuillan%26rft.auinit1%253DR.%26rft.volume%253D83%26rft.issue%253D3%26rft.spage%253D359%26rft.epage%253D372%26rft.atitle%253DRuns%2Bof%2Bhomozygosity%2Bin%2BEuropean%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2008.08.007%26rft_id%253Dinfo%253Apmid%252F18760389%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2008.08.007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18760389&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F9%2F1067.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000259307200007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1067.23" data-doi="10.1158/0008-5472.CAN-09-4502"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Menashe</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maeder</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garcia-Closas</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Figueroa</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bhattacharjee</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Pathway analysis of breast cancer genome-wide association study highlights three pathways and one canonical signaling cascade</span>. <abbr class="cit-jnl-abbrev">Cancer Res.</abbr> <span class="cit-vol">70</span>: <span class="cit-fpage">4453</span>–<span class="cit-lpage">4459</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCancer%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1158%252F0008-5472.CAN-09-4502%26rft_id%253Dinfo%253Apmid%252F20460509%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiY2FucmVzIjtzOjU6InJlc2lkIjtzOjEwOiI3MC8xMS80NDUzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMi85LzEwNjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1067.24" data-doi="10.1038/nature01262"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">Mouse Genome Sequencing Consortium</span>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Initial sequencing and comparative analysis of the mouse genome</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">420</span>: <span class="cit-fpage">520</span>–<span class="cit-lpage">562</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DWaterston%26rft.auinit1%253DR.%2BH.%26rft.volume%253D420%26rft.issue%253D6915%26rft.spage%253D520%26rft.epage%253D562%26rft.atitle%253DInitial%2Bsequencing%2Band%2Bcomparative%2Banalysis%2Bof%2Bthe%2Bmouse%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature01262%26rft_id%253Dinfo%253Apmid%252F12466850%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature01262&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12466850&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F9%2F1067.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000179611600053&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1067.25" data-doi="10.1093/nar/gkr342"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mu</span>  <span class="cit-name-given-names">X. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">Z. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kong</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lam</span>  <span class="cit-name-given-names">H. Y. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gerstein</span>  <span class="cit-name-given-names">M. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Analysis of genomic variation in non-coding elements using population-scale sequencing data from the 1000 Genomes Project</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">7058</span>–<span class="cit-lpage">7076</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkr342%26rft_id%253Dinfo%253Apmid%252F21596777%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiIzOS8xNi83MDU4IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMi85LzEwNjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1067.26" data-doi="10.1093/bioinformatics/btp448"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">O’Dushlaine</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kenny</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heron</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Segurado</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gill</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The SNP ratio test: pathway analysis of genome-wide association datasets</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">2762</span>–<span class="cit-lpage">2763</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp448%26rft_id%253Dinfo%253Apmid%252F19620097%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjUvMjAvMjc2MiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzIvOS8xMDY3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1067.27" data-doi="10.1093/nar/27.1.29"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ogata</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goto</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sato</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fujibuchi</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bono</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">KEGG: Kyoto Encyclopedia of Genes and Genomes</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">29</span>–<span class="cit-lpage">34</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F27.1.29%26rft_id%253Dinfo%253Apmid%252F9847135%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjc6IjI3LzEvMjkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8yLzkvMTA2Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1067.28" data-doi="10.1038/ejhg.2009.115"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Peng</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luo</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Siu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Gene and pathway-based second-wave analysis of genome-wide association studies. European journal of human genetics</span>. <abbr class="cit-jnl-abbrev">Eur. J. Hum. Genet.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">111</span>–<span class="cit-lpage">117</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEuropean%2Bjournal%2Bof%2Bhuman%2Bgenetics%2B%253A%2B%2BEJHG%26rft.stitle%253DEur%2BJ%2BHum%2BGenet%26rft.aulast%253DPeng%26rft.auinit1%253DG.%26rft.volume%253D18%26rft.issue%253D1%26rft.spage%253D111%26rft.epage%253D117%26rft.atitle%253DGene%2Band%2Bpathway-based%2Bsecond-wave%2Banalysis%2Bof%2Bgenome-wide%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fejhg.2009.115%26rft_id%253Dinfo%253Apmid%252F19584899%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ejhg.2009.115&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19584899&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F9%2F1067.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000272609900021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1067.29" data-doi="10.1038/nrg1247"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schaffner</span>  <span class="cit-name-given-names">S. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">The X chromosome in population genetics</span>. <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">43</span>–<span class="cit-lpage">51</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DSchaffner%26rft.auinit1%253DS.%2BF.%26rft.volume%253D5%26rft.issue%253D1%26rft.spage%253D43%26rft.epage%253D51%26rft.atitle%253DThe%2BX%2Bchromosome%2Bin%2Bpopulation%2Bgenetics.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg1247%26rft_id%253Dinfo%253Apmid%252F14708015%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg1247&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14708015&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F9%2F1067.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000187641100015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1067.30" data-doi="10.1093/bioinformatics/btr199"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Seifert</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Strickert</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schliep</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grosse</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Exploiting prior knowledge and gene distances in the analysis of tumor expression profiles with extended Hidden Markov Models</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">1645</span>–<span class="cit-lpage">1652</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtr199%26rft_id%253Dinfo%253Apmid%252F21511716%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjcvMTIvMTY0NSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzIvOS8xMDY3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1067.31" data-doi="10.1001/jama.296.15.1867"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Towbin</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lowe</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Colan</span>  <span class="cit-name-given-names">S. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sleeper</span>  <span class="cit-name-given-names">L. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Orav</span>  <span class="cit-name-given-names">E. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Incidence, causes, and outcomes of dilated cardiomyopathy in children</span>. <abbr class="cit-jnl-abbrev">JAMA</abbr> <span class="cit-vol">296</span>: <span class="cit-fpage">1867</span>–<span class="cit-lpage">1876</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJAMA%26rft.stitle%253DJAMA%26rft.aulast%253DTowbin%26rft.auinit1%253DJ.%2BA.%26rft.volume%253D296%26rft.issue%253D15%26rft.spage%253D1867%26rft.epage%253D1876%26rft.atitle%253DIncidence%252C%2Bcauses%252C%2Band%2Boutcomes%2Bof%2Bdilated%2Bcardiomyopathy%2Bin%2Bchildren.%26rft_id%253Dinfo%253Adoi%252F10.1001%252Fjama.296.15.1867%26rft_id%253Dinfo%253Apmid%252F17047217%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1001/jama.296.15.1867&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17047217&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F9%2F1067.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000241329300029&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1067.32" data-doi="10.1101/gr.1649303"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Versteeg</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Schaik</span>  <span class="cit-name-given-names">B. D. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Batenburg</span>  <span class="cit-name-given-names">M. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roos</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Monajemi</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">The human transcriptome map reveals extremes in gene density, intron length, GC content, and repeat pattern for domains of highly and weakly expressed genes</span>. <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">1998</span>–<span class="cit-lpage">2004</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.1649303%26rft_id%253Dinfo%253Apmid%252F12915492%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjEzLzkvMTk5OCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzIvOS8xMDY3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1067.33" data-doi="10.1371/journal.pgen.1000214"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Veyrieras</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kudaravalli</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">S. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dermitzakis</span>  <span class="cit-name-given-names">E. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gilad</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">High-resolution mapping of expression-QTLs yields insight into human gene regulation</span>. <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e1000214</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DVeyrieras%26rft.auinit1%253DJ.%2BB.%26rft.volume%253D4%26rft.issue%253D10%26rft.spage%253De1000214%26rft.epage%253De1000214%26rft.atitle%253DHigh-resolution%2Bmapping%2Bof%2Bexpression-QTLs%2Byields%2Binsight%2Binto%2Bhuman%2Bgene%2Bregulation.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000214%26rft_id%253Dinfo%253Apmid%252F18846210%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000214&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18846210&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F9%2F1067.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1067.34" data-doi="10.1086/522374"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bucan</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Pathway-based approaches for analysis of genomewide association studies</span>. <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">81</span>: <span class="cit-fpage">1278</span>–<span class="cit-lpage">1283</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DWang%26rft.auinit1%253DK.%26rft.volume%253D81%26rft.issue%253D6%26rft.spage%253D1278%26rft.epage%253D1283%26rft.atitle%253DPathway-based%2Bapproaches%2Bfor%2Banalysis%2Bof%2Bgenomewide%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F522374%26rft_id%253Dinfo%253Apmid%252F17966091%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/522374&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17966091&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F9%2F1067.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000251098800013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1067.35" data-doi="10.1038/nrg2884"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hakonarson</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Analysing biological pathways in genome-wide association studies</span>. <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">843</span>–<span class="cit-lpage">854</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DWang%26rft.auinit1%253DK.%26rft.volume%253D11%26rft.issue%253D12%26rft.spage%253D843%26rft.epage%253D854%26rft.atitle%253DAnalysing%2Bbiological%2Bpathways%2Bin%2Bgenome-wide%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2884%26rft_id%253Dinfo%253Apmid%252F21085203%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2884&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21085203&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F9%2F1067.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000284359000010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1067.36" data-doi="10.1093/bioinformatics/btq728"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jia</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wolfinger</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grayson</span>  <span class="cit-name-given-names">B. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span><span class="cit-comment">a</span> <span class="cit-article-title">An efficient hierarchical generalized linear mixed model for pathway analysis of genome-wide association studies</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">686</span>–<span class="cit-lpage">692</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq728%26rft_id%253Dinfo%253Apmid%252F21266443%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIyNy81LzY4NiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzIvOS8xMDY3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1067.37" data-doi="10.1016/j.ygeno.2011.04.006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jia</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wolfinger</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">Z.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span><span class="cit-comment">b</span> <span class="cit-article-title">Gene set analysis of genome-wide association studies: methodological issues and perspectives</span>. <abbr class="cit-jnl-abbrev">Genomics</abbr> <span class="cit-vol">98</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">8</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenomics%26rft.stitle%253DGenomics%26rft.aulast%253DWang%26rft.auinit1%253DL.%26rft.volume%253D98%26rft.issue%253D1%26rft.spage%253D1%26rft.epage%253D8%26rft.atitle%253DGene%2Bset%2Banalysis%2Bof%2Bgenome-wide%2Bassociation%2Bstudies%253A%2Bmethodological%2Bissues%2Band%2Bperspectives.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ygeno.2011.04.006%26rft_id%253Dinfo%253Apmid%252F21565265%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ygeno.2011.04.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21565265&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F9%2F1067.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000292625300001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1067.38" data-doi="10.1038/nature05911"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">Wellcome Trust Case Control Consortium</span>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Genome-wide association study of 14,000 cases of seven common diseases and 3,000 shared controls</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">447</span>: <span class="cit-fpage">661</span>–<span class="cit-lpage">678</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.volume%253D447%26rft.issue%253D7145%26rft.spage%253D661%26rft.epage%253D678%26rft.atitle%253DGenome-wide%2Bassociation%2Bstudy%2Bof%2B14%252C000%2Bcases%2Bof%2Bseven%2Bcommon%2Bdiseases%2Band%2B3%252C000%2Bshared%2Bcontrols.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature05911%26rft_id%253Dinfo%253Apmid%252F17554300%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature05911&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17554300&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F9%2F1067.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000247030700035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1067.39" data-doi="10.1093/nar/gkq324"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cui</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">i-GSEA4GWAS: a web server for identification of pathways/gene sets associated with traits by applying an improved gene set enrichment analysis to genome-wide association study</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">W90</span>–<span class="cit-lpage">W95</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkq324%26rft_id%253Dinfo%253Apmid%252F20435672%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjE0OiIzOC9zdXBwbF8yL1c5MCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzIvOS8xMDY3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/2/9/1067.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/2/9/1057" title="Using Near-Infrared Spectroscopy to Resolve the Species, Gender, Age, and the Presence of Wolbachia Infection in Laboratory-Reared Drosophila" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/2/9/1077" title="Variant in the 5′ Untranslated Region of Insulin-Like Growth Factor 1 Receptor Is Associated With Susceptibility to Mastitis in Cattle" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/2/9"><span>Volume 2 Issue 9, September 2012</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/2/9.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/2/9.cover-source.jpg?itok=JMb6jwWa" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 2 (9)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/2/9/1067" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-WNFAtaIEIWeCgpzKh53pW1tcyral7mi8mwyy8UOn3g8" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/2/9/1067" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/2/9/1067" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/2/9/1067" class="active" data-icon-position="" data-hide-link-title="0">Uncovering Networks from Genome-Wide Association Studies via Circular Genomic Permutation</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459322" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-TevdUU-PqB5oJwYb_7jAC4NkB7MGi8zIngEZjR_3xBo" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/2/9/1067.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/2/9/1067" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-pYFGNUlLcyh1-XvfOTPCKGU2TJry2Ql-zEFK07P8MM0" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/2/9/1067" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/2/9/1067.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459322" id="node459322--2" data-pisa="ggg;2/9/1067" data-pisa-master="ggg;2/9/1067" data-apath="/ggg/2/9/1067.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Uncovering Networks from Genome-Wide Association Studies via Circular Genomic Permutation</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Claudia P. Cabrera</span>, <span class="highwire-citation-author" data-delta="1">Pau Navarro</span>, <span class="highwire-citation-author" data-delta="2">Jennifer E. Huffman</span>, <span class="highwire-citation-author" data-delta="3">Alan F. Wright</span>, <span class="highwire-citation-author" data-delta="4">Caroline Hayward</span>, <span class="highwire-citation-author" data-delta="5">Harry Campbell</span>, <span class="highwire-citation-author" data-delta="6">James F. Wilson</span>, <span class="highwire-citation-author" data-delta="7">Igor Rudan</span>, <span class="highwire-citation-author" data-delta="8">Nicholas D. Hastie</span>, <span class="highwire-citation-author" data-delta="9">Veronique Vitart</span> and <span class="highwire-citation-author" data-delta="10">Chris S. Haley</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2012 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 2 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1067-1075;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.002618 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459322--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Claudia P. Cabrera </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>MRC Human Genetics Unit, Medical Research Council, Institute of Genetics and Molecular Medicine, University of Edinburgh, Edinburgh EH4 2XU, Scotland, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Claudia%2BP.%2BCabrera%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cabrera%20CP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AClaudia%2BP.%2BCabrera%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Pau Navarro </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>MRC Human Genetics Unit, Medical Research Council, Institute of Genetics and Molecular Medicine, University of Edinburgh, Edinburgh EH4 2XU, Scotland, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Pau%2BNavarro%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Navarro%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APau%2BNavarro%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Jennifer E. Huffman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>MRC Human Genetics Unit, Medical Research Council, Institute of Genetics and Molecular Medicine, University of Edinburgh, Edinburgh EH4 2XU, Scotland, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jennifer%2BE.%2BHuffman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Huffman%20JE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJennifer%2BE.%2BHuffman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Alan F. Wright </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>MRC Human Genetics Unit, Medical Research Council, Institute of Genetics and Molecular Medicine, University of Edinburgh, Edinburgh EH4 2XU, Scotland, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alan%2BF.%2BWright%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wright%20AF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlan%2BF.%2BWright%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Caroline Hayward </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>MRC Human Genetics Unit, Medical Research Council, Institute of Genetics and Molecular Medicine, University of Edinburgh, Edinburgh EH4 2XU, Scotland, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Caroline%2BHayward%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hayward%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACaroline%2BHayward%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Harry Campbell </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Centre for Population Health Sciences, University of Edinburgh, Edinburgh EH8 9AG, Scotland, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Harry%2BCampbell%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Campbell%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHarry%2BCampbell%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">James F. Wilson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Centre for Population Health Sciences, University of Edinburgh, Edinburgh EH8 9AG, Scotland, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=James%2BF.%2BWilson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wilson%20JF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJames%2BF.%2BWilson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Igor Rudan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Centre for Population Health Sciences, University of Edinburgh, Edinburgh EH8 9AG, Scotland, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Igor%2BRudan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rudan%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AIgor%2BRudan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Nicholas D. Hastie </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>MRC Human Genetics Unit, Medical Research Council, Institute of Genetics and Molecular Medicine, University of Edinburgh, Edinburgh EH4 2XU, Scotland, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nicholas%2BD.%2BHastie%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hastie%20ND&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANicholas%2BD.%2BHastie%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Veronique Vitart </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>MRC Human Genetics Unit, Medical Research Council, Institute of Genetics and Molecular Medicine, University of Edinburgh, Edinburgh EH4 2XU, Scotland, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Veronique%2BVitart%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Vitart%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AVeronique%2BVitart%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Chris S. Haley </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>MRC Human Genetics Unit, Medical Research Council, Institute of Genetics and Molecular Medicine, University of Edinburgh, Edinburgh EH4 2XU, Scotland, United Kingdom</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>The Roslin Institute and R(D)SVS, University of Edinburgh, Easter Bush EH25 9RG, Scotland, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Chris%2BS.%2BHaley%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Haley%20CS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChris%2BS.%2BHaley%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/2/9/1067" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/2/9/1067&amp;title=Uncovering%20Networks%20from%20Genome-Wide%20Association%20Studies%20via%20Circular%20Genomic%20Permutation" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/2/9/1067&amp;title=Uncovering%20Networks%20from%20Genome-Wide%20Association%20Studies%20via%20Circular%20Genomic%20Permutation" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/2/9/1067&amp;title=Uncovering%20Networks%20from%20Genome-Wide%20Association%20Studies%20via%20Circular%20Genomic%20Permutation" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/2/9/1067&amp;text=Uncovering%20Networks%20from%20Genome-Wide%20Association%20Studies%20via%20Circular%20Genomic%20Permutation" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/2/9/1067&amp;title=Uncovering%20Networks%20from%20Genome-Wide%20Association%20Studies%20via%20Circular%20Genomic%20Permutation" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/2/9/1067&amp;t=Uncovering%20Networks%20from%20Genome-Wide%20Association%20Studies%20via%20Circular%20Genomic%20Permutation" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/2/9/1067&amp;title=Uncovering%20Networks%20from%20Genome-Wide%20Association%20Studies%20via%20Circular%20Genomic%20Permutation" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/2/9/1067&amp;title=Uncovering%20Networks%20from%20Genome-Wide%20Association%20Studies%20via%20Circular%20Genomic%20Permutation" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459322/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459322" id="node459322--3" data-pisa="ggg;2/9/1067" data-pisa-master="ggg;2/9/1067" data-apath="/ggg/2/9/1067.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Uncovering Networks from Genome-Wide Association Studies via Circular Genomic Permutation</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Claudia P. Cabrera</span>, <span class="highwire-citation-author" data-delta="1">Pau Navarro</span>, <span class="highwire-citation-author" data-delta="2">Jennifer E. Huffman</span>, <span class="highwire-citation-author" data-delta="3">Alan F. Wright</span>, <span class="highwire-citation-author" data-delta="4">Caroline Hayward</span>, <span class="highwire-citation-author" data-delta="5">Harry Campbell</span>, <span class="highwire-citation-author" data-delta="6">James F. Wilson</span>, <span class="highwire-citation-author" data-delta="7">Igor Rudan</span>, <span class="highwire-citation-author" data-delta="8">Nicholas D. Hastie</span>, <span class="highwire-citation-author" data-delta="9">Veronique Vitart</span> and <span class="highwire-citation-author" data-delta="10">Chris S. Haley</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2012 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 2 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1067-1075;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.002618 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459322--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Claudia P. Cabrera </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>MRC Human Genetics Unit, Medical Research Council, Institute of Genetics and Molecular Medicine, University of Edinburgh, Edinburgh EH4 2XU, Scotland, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Claudia%2BP.%2BCabrera%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cabrera%20CP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AClaudia%2BP.%2BCabrera%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Pau Navarro </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>MRC Human Genetics Unit, Medical Research Council, Institute of Genetics and Molecular Medicine, University of Edinburgh, Edinburgh EH4 2XU, Scotland, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Pau%2BNavarro%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Navarro%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APau%2BNavarro%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Jennifer E. Huffman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>MRC Human Genetics Unit, Medical Research Council, Institute of Genetics and Molecular Medicine, University of Edinburgh, Edinburgh EH4 2XU, Scotland, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jennifer%2BE.%2BHuffman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Huffman%20JE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJennifer%2BE.%2BHuffman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Alan F. Wright </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>MRC Human Genetics Unit, Medical Research Council, Institute of Genetics and Molecular Medicine, University of Edinburgh, Edinburgh EH4 2XU, Scotland, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alan%2BF.%2BWright%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wright%20AF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlan%2BF.%2BWright%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Caroline Hayward </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>MRC Human Genetics Unit, Medical Research Council, Institute of Genetics and Molecular Medicine, University of Edinburgh, Edinburgh EH4 2XU, Scotland, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Caroline%2BHayward%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hayward%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACaroline%2BHayward%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Harry Campbell </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Centre for Population Health Sciences, University of Edinburgh, Edinburgh EH8 9AG, Scotland, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Harry%2BCampbell%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Campbell%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHarry%2BCampbell%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">James F. Wilson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Centre for Population Health Sciences, University of Edinburgh, Edinburgh EH8 9AG, Scotland, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=James%2BF.%2BWilson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wilson%20JF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJames%2BF.%2BWilson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Igor Rudan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Centre for Population Health Sciences, University of Edinburgh, Edinburgh EH8 9AG, Scotland, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Igor%2BRudan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rudan%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AIgor%2BRudan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Nicholas D. Hastie </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>MRC Human Genetics Unit, Medical Research Council, Institute of Genetics and Molecular Medicine, University of Edinburgh, Edinburgh EH4 2XU, Scotland, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nicholas%2BD.%2BHastie%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hastie%20ND&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANicholas%2BD.%2BHastie%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Veronique Vitart </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>MRC Human Genetics Unit, Medical Research Council, Institute of Genetics and Molecular Medicine, University of Edinburgh, Edinburgh EH4 2XU, Scotland, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Veronique%2BVitart%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Vitart%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AVeronique%2BVitart%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Chris S. Haley </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>MRC Human Genetics Unit, Medical Research Council, Institute of Genetics and Molecular Medicine, University of Edinburgh, Edinburgh EH4 2XU, Scotland, United Kingdom</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>The Roslin Institute and R(D)SVS, University of Edinburgh, Easter Bush EH25 9RG, Scotland, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Chris%2BS.%2BHaley%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Haley%20CS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChris%2BS.%2BHaley%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1067&amp;rft.epage=1075&amp;rft.atitle=Uncovering%20Networks%20from%20Genome-Wide%20Association%20Studies%20via%20Circular%20Genomic%20Permutation&amp;rft.volume=2&amp;rft.issue=9&amp;rft.date=2012-09-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Cabrera%2C+Claudia+P.&amp;rft.au=Navarro%2C+Pau&amp;rft.au=Huffman%2C+Jennifer+E.&amp;rft.au=Wright%2C+Alan+F.&amp;rft.au=Hayward%2C+Caroline&amp;rft.au=Campbell%2C+Harry&amp;rft.au=Wilson%2C+James+F.&amp;rft.au=Rudan%2C+Igor&amp;rft.au=Hastie%2C+Nicholas+D.&amp;rft.au=Vitart%2C+Veronique&amp;rft.au=Haley%2C+Chris+S."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459322/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459322/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459322/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459322/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459322/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459322/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459322/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459322/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459322/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459322/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459322/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459322/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/2/9/1067" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">MATERIALS AND METHODS</a></li><li class="odd"><a href="#sec-10" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">RESULTS</a></li><li class="even"><a href="#sec-19" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">DISCUSSION</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">ACKNOWLEDGMENTS</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">LITERATURE CITED</a></li></ul></div></li><li class="odd"><a href="/content/2/9/1067.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/2/9/1067.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/2/9/1067.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/2/9/1067.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
