<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/2/9/1103" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/2/9/1103.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/2/9/1103.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/2/9/1103.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/2/9.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/2/9/1103.atom" />
<meta name="HW.pisa" content="ggg;2/9/1103" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Mitochondrial Genome Evolution in a Single Protoploid Yeast Species" />
<meta name="DC.Identifier" content="10.1534/g3.112.003152" />
<meta name="DC.Date" content="2012-09-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2012 Jung et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Mitochondria are organelles, which play a key role in some essential functions, including respiration, metabolite biosynthesis, ion homeostasis, and apoptosis. The vast numbers of mitochondrial DNA (mtDNA) sequences of various yeast species, which have recently been published, have also helped to elucidate the structural diversity of these genomes. Although a large corpus of data are now available on the diversity of yeast species, little is known so far about the mtDNA diversity in single yeast species. To study the genetic variations occurring in the mtDNA of wild yeast isolates, we performed a genome-wide polymorphism survey on the mtDNA of 18 Lachancea kluyveri (formerly Saccharomyces kluyveri ) strains. We determined the complete mt genome sequences of strains isolated from various geographical locations (in North America, Asia, and Europe) and ecological niches ( Drosophila , tree exudates, soil). The mt genome of the NCYC 543 reference strain is 51,525 bp long. It contains the same core of genes as Lachancea thermotolerans , the nearest relative to L. kluyveri . To explore the mt genome variations in a single yeast species, we compared the mtDNAs of the 18 isolates. The phylogeny and population structure of L. kluyveri provide clear-cut evidence for the existence of well-defined geographically isolated lineages. Although these genomes are completely syntenic, their size and the intron content were found to vary among the isolates studied. These genomes are highly polymorphic, showing an average diversity of 28.5 SNPs/kb and 6.6 indels/kb. Analysis of the SNP and indel patterns showed the existence of a particularly high overall level of polymorphism in the intergenic regions. The dN/dS ratios obtained are consistent with purifying selection in all these genes, with the noteworthy exception of the [VAR1][1] gene, which gave a very high ratio. These data suggest that the intergenic regions have evolved very fast in yeast mitochondrial genomes.

 [1]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000007275" />
<meta name="DC.Contributor" content="Paul P. Jung" />
<meta name="DC.Contributor" content="Anne Friedrich" />
<meta name="DC.Contributor" content="Cyrielle Reisser" />
<meta name="DC.Contributor" content="Jing Hou" />
<meta name="DC.Contributor" content="Joseph Schacherer" />
<meta name="article:published_time" content="2012-09-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Mitochondrial Genome Evolution in a Single Protoploid Yeast Species" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Mitochondria are organelles, which play a key role in some essential functions, including respiration, metabolite biosynthesis, ion homeostasis, and apoptosis. The vast numbers of mitochondrial DNA (mtDNA) sequences of various yeast species, which have recently been published, have also helped to elucidate the structural diversity of these genomes. Although a large corpus of data are now available on the diversity of yeast species, little is known so far about the mtDNA diversity in single yeast species. To study the genetic variations occurring in the mtDNA of wild yeast isolates, we performed a genome-wide polymorphism survey on the mtDNA of 18 &lt;i&gt;Lachancea kluyveri&lt;/i&gt; (formerly &lt;i&gt;Saccharomyces kluyveri&lt;/i&gt;) strains. We determined the complete mt genome sequences of strains isolated from various geographical locations (in North America, Asia, and Europe) and ecological niches (&lt;i&gt;Drosophila&lt;/i&gt;, tree exudates, soil). The mt genome of the NCYC 543 reference strain is 51,525 bp long. It contains the same core of genes as &lt;i&gt;Lachancea thermotolerans&lt;/i&gt;, the nearest relative to &lt;i&gt;L. kluyveri&lt;/i&gt;. To explore the mt genome variations in a single yeast species, we compared the mtDNAs of the 18 isolates. The phylogeny and population structure of &lt;i&gt;L. kluyveri&lt;/i&gt; provide clear-cut evidence for the existence of well-defined geographically isolated lineages. Although these genomes are completely syntenic, their size and the intron content were found to vary among the isolates studied. These genomes are highly polymorphic, showing an average diversity of 28.5 SNPs/kb and 6.6 indels/kb. Analysis of the SNP and indel patterns showed the existence of a particularly high overall level of polymorphism in the intergenic regions. The dN/dS ratios obtained are consistent with purifying selection in all these genes, with the noteworthy exception of the &lt;i&gt;VAR1&lt;/i&gt; gene, which gave a very high ratio. These data suggest that the intergenic regions have evolved very fast in yeast mitochondrial genomes.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2012/09/01" />
<meta name="citation_mjid" content="ggg;2/9/1103" />
<meta name="citation_id" content="2/9/1103" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/2/9/1103" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/2/9/1103.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/2/9/1103.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/2/9/1103.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.112.003152" />
<meta name="citation_pmid" content="22973548" />
<meta name="citation_volume" content="2" />
<meta name="citation_issue" content="9" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="1103" />
<meta name="citation_lastpage" content="1111" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Paul P. Jung" />
<meta name="citation_author_institution" content="Department of Genetics, Genomics and Microbiology, University of Strasbourg/Centre National de la Recherche Scientifique, Unité Mixte de Recherche 7156, Strasbourg, France" />
<meta name="citation_author" content="Anne Friedrich" />
<meta name="citation_author_institution" content="Department of Genetics, Genomics and Microbiology, University of Strasbourg/Centre National de la Recherche Scientifique, Unité Mixte de Recherche 7156, Strasbourg, France" />
<meta name="citation_author" content="Cyrielle Reisser" />
<meta name="citation_author_institution" content="Department of Genetics, Genomics and Microbiology, University of Strasbourg/Centre National de la Recherche Scientifique, Unité Mixte de Recherche 7156, Strasbourg, France" />
<meta name="citation_author" content="Jing Hou" />
<meta name="citation_author_institution" content="Department of Genetics, Genomics and Microbiology, University of Strasbourg/Centre National de la Recherche Scientifique, Unité Mixte de Recherche 7156, Strasbourg, France" />
<meta name="citation_author" content="Joseph Schacherer" />
<meta name="citation_author_institution" content="Department of Genetics, Genomics and Microbiology, University of Strasbourg/Centre National de la Recherche Scientifique, Unité Mixte de Recherche 7156, Strasbourg, France" />
<meta name="citation_author_email" content="schacherer@unistra.fr" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=S. F. Altschul;citation_author=T. L. Madden;citation_author=A. A. Schäffer;citation_author=J. Zhang;citation_author=Z. Zhang;citation_title=Gapped BLAST and PSI-BLAST: a new generation of protein database search programs;citation_volume=25;citation_year=1997;citation_pmid=9254694;citation_doi=10.1093/nar/25.17.3389" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=C. Batini;citation_author=J. Lopes;citation_author=D. M. Behar;citation_author=F. Calafell;citation_author=L. B. Jorde;citation_title=Insights into the demographic history of African Pygmies from complete mitochondrial genomes;citation_volume=28;citation_year=2011;citation_pmid=21041797;citation_doi=10.1093/molbev/msq294" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol;citation_author=S. K. Behura;citation_title=Analysis of nuclear copies of mitochondrial sequences in honeybee (Apis mellifera) genome;citation_volume=24;citation_year=2007;citation_pmid=17404397;citation_doi=10.1093/molbev/msm068" />
<meta name="citation_reference" content="citation_journal_title=FEMS Yeast Res.;citation_author=C. Bouchier;citation_author=L. Ma;citation_author=S. Créno;citation_author=B. Dujon;citation_author=C. Fairhead;citation_title=Complete mitochondrial genome sequences of three Nakaseomyces species reveal invasion by palindromic GC clusters and considerable size expansion;citation_volume=9;citation_year=2009;citation_pmid=19758332;citation_doi=10.1111/j.1567-1364.2009.00551.x" />
<meta name="citation_reference" content="citation_journal_title=Current genetics;citation_journal_abbrev=Curr Genet;citation_author=G. D. Clark-Walker;citation_title=Contrasting mutation rates in mitochondrial and nuclear genes of yeasts versus mammals.;citation_pages=195-198;citation_volume=20;citation_year=1991;citation_issue=3;citation_pmid=1657417;citation_doi=10.1007/BF00326232" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=A. Delahodde;citation_author=V. Goguel;citation_author=A. M. Becam;citation_author=F. Creusot;citation_author=J. Perea;citation_title=Site-specific DNA endonuclease and RNA maturase activities of two homologous intron-encoded proteins from yeast mitochondria.;citation_pages=431-441;citation_volume=56;citation_year=1989;citation_issue=3;citation_pmid=2536593;citation_doi=10.1016/0092-8674(89)90246-8" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=D. R. Denver;citation_author=K. Morris;citation_author=M. Lynch;citation_author=L. L. Vassilieva;citation_author=W. K. Thomas;citation_title=High Direct Estimate of the Mutation Rate in the Mitochondrial Genome of Caenorhabditis elegans;citation_pages=2342-2344;citation_volume=289;citation_year=2000;citation_issue=5488;citation_pmid=11009418;citation_doi=10.1126/science.289.5488.2342" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=A. Dereeper;citation_author=V. Guignon;citation_author=G. Blanc;citation_author=S. Audic;citation_author=S. Buffet;citation_title=Phylogeny.fr: robust phylogenetic analysis for the non-specialist;citation_pages=465-469;citation_volume=36;citation_year=2008;citation_doi=10.1093/nar/gkn180" />
<meta name="citation_reference" content="citation_journal_title=Gene;citation_author=B. Dujon;citation_author=M. Belfort;citation_author=R. A. Butow;citation_author=C. Jacq;citation_author=C. Lemieux;citation_title=Mobile introns: definition of terms and recommended nomenclature;citation_pages=115-118;citation_volume=82;citation_year=1989;citation_pmid=2555261;citation_doi=10.1016/0378-1119(89)90035-8" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=R. C. Edgar;citation_title=MUSCLE: a multiple sequence alignment method with reduced time and space complexity.;citation_pages=113-113;citation_volume=5;citation_year=2004;citation_pmid=15318951;citation_doi=10.1186/1471-2105-5-113" />
<meta name="citation_reference" content="citation_journal_title=FEMS Yeast Res.;citation_author=M. A. Eldarov;citation_author=A. V. Mardanov;citation_author=A. V. Beletsky;citation_author=N. V. Ravin;citation_author=K. G. Skryabin;citation_title=Complete sequence and analysis of the mitochondrial genome of the methylotrophic yeast Hansenula polymorpha DL-1;citation_volume=11;citation_year=2011;citation_pmid=21545683;citation_doi=10.1111/j.1567-1364.2011.00736.x" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=J. L. Elson;citation_author=D. M. Turnbull;citation_author=N. Howell;citation_title=Comparative genomics and the evolution of human mitochondrial DNA: assessing the effects of selection.;citation_pages=229-238;citation_volume=74;citation_year=2004;citation_issue=2;citation_pmid=14712420;citation_doi=10.1086/381505" />
<meta name="citation_reference" content="citation_journal_title=FEBS Letters;citation_journal_abbrev=FEBS Letters;citation_author=F. Foury;citation_author=T. Roganti;citation_author=N. Lecrenier;citation_author=B. Purnelle;citation_title=The complete sequence of the mitochondrial genome of Saccharomyces cerevisiae.;citation_pages=325-331;citation_volume=440;citation_year=1998;citation_issue=3;citation_pmid=9872396;citation_doi=10.1016/S0014-5793(98)01467-7" />
<meta name="citation_reference" content="citation_journal_title=FEMS Yeast Res.;citation_author=C. Gaillardin;citation_author=C. Neuvéglise;citation_author=S. Kerscher;citation_author=J. M. Nicaud;citation_title=Mitochondrial genomes of yeasts of the Yarrowia clade;citation_pages=317-331;citation_volume=12;citation_year=2011" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=M. Gouy;citation_author=S. Guindon;citation_author=O. Gascuel;citation_title=SeaView version 4: a multiplatform graphical user interface for sequence alignment and phylogenetic tree building;citation_volume=27;citation_year=2010;citation_pmid=19854763;citation_doi=10.1093/molbev/msp259" />
<meta name="citation_reference" content="citation_journal_title=Syst. Biol.;citation_author=S. Guindon;citation_author=O. Gascuel;citation_title=A simple, fast, and accurate algorithm to estimate large phylogenies by maximum likelihood;citation_volume=52;citation_year=2003;citation_pmid=14530136;citation_doi=10.1080/10635150390235520" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=E. D. Gunnarsdóttir;citation_author=M. Li;citation_author=M. Bauchet;citation_author=K. Finstermeier;citation_author=M. Stoneking;citation_title=High-throughput sequencing of complete human mtDNA genomes from the Philippines;citation_volume=21;citation_year=2011;citation_pmid=21147912;citation_doi=10.1101/gr.107615.110" />
<meta name="citation_reference" content="citation_journal_title=PLoS biology;citation_journal_abbrev=PLoS Biol;citation_author=C. Haag-Liautard;citation_author=N. Coffey;citation_author=D. Houle;citation_author=M. Lynch;citation_author=B. Charlesworth;citation_title=Direct estimation of the mitochondrial DNA mutation rate in Drosophila melanogaster.;citation_pages=e204-e204;citation_volume=6;citation_year=2008;citation_issue=8;citation_pmid=18715119;citation_doi=10.1371/journal.pbio.0060204" />
<meta name="citation_reference" content="citation_journal_title=Trends in genetics :  TIG;citation_journal_abbrev=Trends Genet;citation_author=P. Haugen;citation_author=D. M. Simon;citation_author=D. Bhattacharya;citation_title=The natural history of group I introns.;citation_pages=111-119;citation_volume=21;citation_year=2005;citation_issue=2;citation_pmid=15661357;citation_doi=10.1016/j.tig.2004.12.007" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol. Evol.;citation_author=D. K. Howe;citation_author=C. F. Baer;citation_author=D. R. Denver;citation_title=High rate of large deletions in Caenorhabditis briggsae mitochondrial genome mutation processes;citation_volume=2;citation_year=2010;citation_pmid=20333220;citation_doi=10.1093/gbe/evp055" />
<meta name="citation_reference" content="citation_journal_title=Molecular &amp; general genetics :  MGG;citation_journal_abbrev=Mol Gen Genet;citation_author=A. Jacquier;citation_author=B. Dujon;citation_title=The intron of the mitochondrial 21S rRNA gene: distribution in different yeast species and sequence comparison between Kluyveromyces thermotolerans and Saccharomyces cerevisiae.;citation_pages=487-499;citation_volume=192;citation_year=1983;citation_issue=3;citation_pmid=6361491;citation_doi=10.1007/BF00392195" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=H. Jiang;citation_author=W. Guan;citation_author=D. Pinney;citation_author=W. Wang;citation_author=Z. Gu;citation_title=Relaxation of yeast mitochondrial functions after whole-genome duplication;citation_volume=18;citation_year=2008;citation_pmid=18669479;citation_doi=10.1101/gr.074674.107" />
<meta name="citation_reference" content="citation_journal_title=FEMS Yeast Res.;citation_author=P. P. Jung;citation_author=J. Schacherer;citation_author=J. L. Souciet;citation_author=S. Potier;citation_author=P. Wincker;citation_title=The complete mitochondrial genome of the yeast Pichia sorbitophila;citation_volume=9;citation_year=2009;citation_pmid=19594828;citation_doi=10.1111/j.1567-1364.2009.00540.x" />
<meta name="citation_reference" content="citation_journal_title=Current genetics;citation_journal_abbrev=Curr Genet;citation_author=P. P. Jung;citation_author=A. Friedrich;citation_author=J. L. Souciet;citation_author=V. Louis;citation_author=S. Potier;citation_title=Complete mitochondrial genome sequence of the yeast Pichia farinosa and comparative analysis of closely related species.;citation_pages=507-515;citation_volume=56;citation_year=2010;citation_issue=6;citation_pmid=20830585;citation_doi=10.1007/s00294-010-0318-y" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=P. Kosa;citation_author=M. Valach;citation_author=L. Tomaska;citation_author=K. H. Wolfe;citation_author=J. Nosek;citation_title=Complete DNA sequences of the mitochondrial genomes of the pathogenic yeasts Candida orthopsilosis and Candida metapsilosis: insight into the evolution of linear DNA genomes from mitochondrial telomere mutants.;citation_pages=2472-2481;citation_volume=34;citation_year=2006;citation_issue=8;citation_pmid=16684995;citation_doi=10.1093/nar/gkl327" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=S. Kurtz;citation_author=A. Phillippy;citation_author=A. L. Delcher;citation_author=M. Smoot;citation_author=M. Shumway;citation_title=Versatile and open software for comparing large genomes.;citation_pages=R12-R12;citation_volume=5;citation_year=2004;citation_issue=2;citation_pmid=14759262;citation_doi=10.1186/gb-2004-5-2-r12" />
<meta name="citation_reference" content="citation_journal_title=Human genomics;citation_journal_abbrev=Hum Genomics;citation_author=Y. Li;citation_author=Y. Hu;citation_author=L. Bolund;citation_author=J. Wang;citation_title=State of the art de novo assembly of human genomes from massively parallel sequencing data.;citation_pages=271-277;citation_volume=4;citation_year=2010;citation_issue=4;citation_pmid=20511140;citation_doi=10.1186/1479-7364-4-4-271" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=G. Liti;citation_author=D. M. Carter;citation_author=A. M. Moses;citation_author=J. Warringer;citation_author=L. Parts;citation_title=Population genomics of domestic and wild yeasts.;citation_pages=337-341;citation_volume=458;citation_year=2009;citation_issue=7236;citation_pmid=19212322;citation_doi=10.1038/nature07743" />
<meta name="citation_reference" content="citation_title=The Origins of Genome Architecture;citation_author=M. Lynch;citation_year=2007" />
<meta name="citation_reference" content="citation_journal_title=Genetica;citation_author=M. Lynch;citation_author=J. L. Blanchard;citation_title=DELETERIOUS MUTATION ACCUMULATION IN ORGANELLE GENOMES;citation_pages=29-39;citation_volume=102/103;citation_year=1998" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=M. Lynch;citation_author=B. Koskella;citation_author=S. Schaack;citation_title=Mutation pressure and the evolution of organelle genomic architecture.;citation_pages=1727-1730;citation_volume=311;citation_year=2006;citation_issue=5768;citation_pmid=16556832;citation_doi=10.1126/science.1118884" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=M. Lynch;citation_author=W. Sung;citation_author=K. Morris;citation_author=N. Coffey;citation_author=C. R. Landry;citation_title=A genome-wide view of the spectrum of spontaneous mutations in yeast;citation_volume=105;citation_year=2008;citation_pmid=18583475;citation_doi=10.1073/pnas.0803466105" />
<meta name="citation_reference" content="citation_journal_title=FEBS Journal;citation_journal_abbrev=FEBS J;citation_author=A. Merico;citation_author=P. Sulo;citation_author=J. Piskur;citation_author=C. Compagno;citation_title=Fermentative lifestyle in yeasts belonging to the Saccharomyces complex;citation_pages=976-989;citation_volume=274;citation_year=2007;citation_issue=4;citation_pmid=17239085;citation_doi=10.1111/j.1742-4658.2007.05645.x" />
<meta name="citation_reference" content="citation_journal_title=Journal of Bacteriology;citation_journal_abbrev=J. Bacteriol.;citation_author=K. Møller;citation_author=L. Olsson;citation_author=J. Piskur;citation_title=Ability for Anaerobic Growth Is Not Sufficient for Development of the Petite Phenotype in Saccharomyces kluyveri;citation_pages=2485-2489;citation_volume=183;citation_year=2001;citation_issue=8;citation_pmid=11274107;citation_doi=10.1128/JB.183.8.2485-2489.2001" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=R.I. Molnar;citation_author=G. Bartelmes;citation_author=I. Dinkelacker;citation_author=H. Witte;citation_author=R. J. Sommer;citation_title=Mutation rates and intraspecific divergence of the mitochondrial genome of Pristionchus pacificus;citation_volume=28;citation_year=2011;citation_pmid=21368317;citation_doi=10.1093/molbev/msr057" />
<meta name="citation_reference" content="citation_journal_title=Biochemistry. Biokhimiiï¸ aï¸¡;citation_journal_abbrev=Biochemistry (Mosc);citation_author=E. S. Naumova;citation_author=E. V. Serpova;citation_author=G. I. Naumov;citation_title=Molecular systematics of Lachancea yeasts.;citation_pages=1356-1362;citation_volume=72;citation_year=2007;citation_issue=12;citation_pmid=18205619;citation_doi=10.1134/S0006297907120097" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=C. Noutsos;citation_author=E. Richly;citation_author=D. Leister;citation_title=Generation and evolutionary fate of insertions of organelle DNA in the nuclear genomes of flowering plants;citation_pages=616-628;citation_volume=15;citation_year=2005;citation_issue=5;citation_pmid=15867426;citation_doi=10.1101/gr.3788705" />
<meta name="citation_reference" content="citation_journal_title=Int. J. Syst. Bacteriol.;citation_author=J. Piskur;citation_author=S. Smole;citation_author=C. Groth;citation_author=R. F. Petersen;citation_author=M. B. Pedersen;citation_title=Structure and genetic stability of mitochondrial genomes vary among yeasts of the genus Saccharomyces;citation_pages=1015-1024;citation_volume=3;citation_year=1998" />
<meta name="citation_reference" content="citation_journal_title=FEMS Yeast Res.;citation_author=P. V. Pramateftaki;citation_author=V. N. Kouvelis;citation_author=P. Lanaridis;citation_author=M. A. Typas;citation_title=The mitochondrial genome of the wine yeast Hanseniaspora uvarum: a unique genome organization among yeast/fungal counterparts;citation_volume=6;citation_year=2006;citation_pmid=16423073;citation_doi=10.1111/j.1567-1364.2005.00018.x" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. K. Pritchard;citation_author=M. Stephens;citation_author=P. Donnelly;citation_title=Inference of population structure using multilocus genotype data;citation_pages=945-959;citation_volume=155;citation_year=2000;citation_pmid=10835412" />
<meta name="citation_reference" content="citation_journal_title=FEMS Yeast Res.;citation_author=E. Procházka;citation_author=S. Poláková;citation_author=J. Piskur;citation_author=P. Sulo;citation_title=Mitochondrial genome from the facultative anaerobe and petite-positive yeast Dekkera bruxellensis contains the NADH dehydrogenase subunit genes;citation_volume=10;citation_year=2010;citation_pmid=20528950;citation_doi=10.1111/j.1567-1364.2010.00644.x" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=D. M. Ruderfer;citation_author=S. C. Pratt;citation_author=H. S. Seidel;citation_author=L. Kruglyak;citation_title=Population genomic analysis of outcrossing and recombination in yeast.;citation_pages=1077-1081;citation_volume=38;citation_year=2006;citation_issue=9;citation_pmid=16892060;citation_doi=10.1038/ng1859" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=J. Schacherer;citation_author=J. A. Shapiro;citation_author=D. M. Ruderfer;citation_author=L. Kruglyak;citation_title=Comprehensive polymorphism survey elucidates population structure of Saccharomyces cerevisiae.;citation_pages=342-345;citation_volume=458;citation_year=2009;citation_issue=7236;citation_pmid=19212320;citation_doi=10.1038/nature07670" />
<meta name="citation_reference" content="citation_journal_title=European journal of human genetics :  EJHG;citation_journal_abbrev=Eur J Hum Genet;citation_author=A. Schönberg;citation_author=C. Theunert;citation_author=M. Li;citation_author=M. Stoneking;citation_author=I. Nasidze;citation_title=High-throughput sequencing of complete human mtDNA genomes from the Caucasus and West Asia: high diversity and demographic inferences.;citation_pages=988-994;citation_volume=19;citation_year=2011;citation_issue=9;citation_pmid=21487439;citation_doi=10.1038/ejhg.2011.62" />
<meta name="citation_reference" content="citation_journal_title=RNA;citation_journal_abbrev=RNA;citation_author=E. R. Seif;citation_author=L. Forget;citation_author=N. C. Martin;citation_author=B. F. Lang;citation_title=Mitochondrial RNase P RNAs in ascomycete fungi: Lineage-specific variations in RNA secondary structure;citation_pages=1073-1083;citation_volume=9;citation_year=2003;citation_issue=9;citation_pmid=12923256;citation_doi=10.1261/rna.5880403" />
<meta name="citation_reference" content="citation_journal_title=Trends in microbiology;citation_journal_abbrev=Trends Microbiol;citation_author=L. Solieri;citation_title=Mitochondrial inheritance in budding yeasts: towards an integrated understanding.;citation_pages=521-530;citation_volume=18;citation_year=2010;citation_issue=11;citation_pmid=20832322;citation_doi=10.1016/j.tim.2010.08.001" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=S. Song;citation_author=Z. F. Pursell;citation_author=W. C. Copeland;citation_author=M. J. Longley;citation_author=T. A. Kunkel;citation_title=DNA precursor asymmetries in mammalian tissue mitochondria and possible contribution to mutagenesis through reduced replication fidelity;citation_volume=102;citation_year=2005;citation_pmid=15784738;citation_doi=10.1073/pnas.0500253102" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=J. L. Souciet;citation_author=B. Dujon;citation_author=C. Gaillardin;citation_author=M. Johnston;citation_author=P. V. Baret;citation_title=Comparative genomics of protoploid Saccharomycetaceae;citation_volume=19;citation_year=2009;citation_pmid=19525356;citation_doi=10.1101/gr.091546.109" />
<meta name="citation_reference" content="citation_journal_title=FEBS Letters;citation_journal_abbrev=FEBS Letters;citation_author=E. Talla;citation_author=V. Anthouard;citation_author=C. Bouchier;citation_author=L. Frangeul;citation_author=B. Dujon;citation_title=The complete mitochondrial genome of the yeast Kluyveromyces thermotolerans.;citation_pages=30-40;citation_volume=579;citation_year=2005;citation_issue=1;citation_pmid=15620687;citation_doi=10.1016/j.febslet.2004.10.106" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=M. Valach;citation_author=Z. Farkas;citation_author=D. Fricova;citation_author=J. Kovac;citation_author=B. Brejova;citation_title=Evolution of linear chromosomes and multipartite genomes in yeast mitochondria;citation_volume=39;citation_year=2011;citation_pmid=21266473;citation_doi=10.1093/nar/gkq1345" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=J. M. Wenzlau;citation_author=R. J. Saldanha;citation_author=R. A. Butow;citation_author=P. S. Perlman;citation_title=A latent intron-encoded maturase is also an endonuclease needed for intron mobility.;citation_pages=421-430;citation_volume=56;citation_year=1989;citation_issue=3;citation_pmid=2536592;citation_doi=10.1016/0092-8674(89)90245-6" />
<meta name="citation_reference" content="citation_journal_title=Methods in enzymology;citation_journal_abbrev=Methods Enzymol;citation_author=D. H. Williamson;citation_author=D. J. Fennell;citation_title=Visualization of yeast mitochondrial DNA with the fluorescent stain &amp;quot;DAPI&amp;quot;.;citation_pages=728-733;citation_volume=56;citation_year=1979;citation_pmid=459889;citation_doi=10.1016/0076-6879(79)56065-0" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=Z. Yang;citation_title=PAML 4: phylogenetic analysis by maximum likelihood;citation_volume=24;citation_year=2007;citation_pmid=17483113;citation_doi=10.1093/molbev/msm088" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Mitochondrial Genome Evolution in a Single Protoploid Yeast Species" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/2/9.cover-source.jpg" />
<meta name="twitter:description" content="Mitochondria are organelles, which play a key role in some essential functions, including respiration, metabolite biosynthesis, ion homeostasis, and apoptosis. The vast numbers of mitochondrial DNA (mtDNA) sequences of various yeast species, which have recently been published, have also helped to elucidate the structural diversity of these genomes. Although a large corpus of data are now available on the diversity of yeast species, little is known so far about the mtDNA diversity in single yeast species. To study the genetic variations occurring in the mtDNA of wild yeast isolates, we performed a genome-wide polymorphism survey on the mtDNA of 18 Lachancea kluyveri (formerly Saccharomyces kluyveri ) strains. We determined the complete mt genome sequences of strains isolated from various geographical locations (in North America, Asia, and Europe) and ecological niches ( Drosophila , tree exudates, soil). The mt genome of the NCYC 543 reference strain is 51,525 bp long. It contains the same core of genes as Lachancea thermotolerans , the nearest relative to L. kluyveri . To explore the mt genome variations in a single yeast species, we compared the mtDNAs of the 18 isolates. The phylogeny and population structure of L. kluyveri provide clear-cut evidence for the existence of well-defined geographically isolated lineages. Although these genomes are completely syntenic, their size and the intron content were found to vary among the isolates studied. These genomes are highly polymorphic, showing an average diversity of 28.5 SNPs/kb and 6.6 indels/kb. Analysis of the SNP and indel patterns showed the existence of a particularly high overall level of polymorphism in the intergenic regions. The dN/dS ratios obtained are consistent with purifying selection in all these genes, with the noteworthy exception of the [VAR1][1] gene, which gave a very high ratio. These data suggest that the intergenic regions have evolved very fast in yeast mitochondrial genomes.

 [1]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000007275" />
<meta name="og-title" property="og:title" content="Mitochondrial Genome Evolution in a Single Protoploid Yeast Species" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/2/9/1103" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Mitochondria are organelles, which play a key role in some essential functions, including respiration, metabolite biosynthesis, ion homeostasis, and apoptosis. The vast numbers of mitochondrial DNA (mtDNA) sequences of various yeast species, which have recently been published, have also helped to elucidate the structural diversity of these genomes. Although a large corpus of data are now available on the diversity of yeast species, little is known so far about the mtDNA diversity in single yeast species. To study the genetic variations occurring in the mtDNA of wild yeast isolates, we performed a genome-wide polymorphism survey on the mtDNA of 18 Lachancea kluyveri (formerly Saccharomyces kluyveri ) strains. We determined the complete mt genome sequences of strains isolated from various geographical locations (in North America, Asia, and Europe) and ecological niches ( Drosophila , tree exudates, soil). The mt genome of the NCYC 543 reference strain is 51,525 bp long. It contains the same core of genes as Lachancea thermotolerans , the nearest relative to L. kluyveri . To explore the mt genome variations in a single yeast species, we compared the mtDNAs of the 18 isolates. The phylogeny and population structure of L. kluyveri provide clear-cut evidence for the existence of well-defined geographically isolated lineages. Although these genomes are completely syntenic, their size and the intron content were found to vary among the isolates studied. These genomes are highly polymorphic, showing an average diversity of 28.5 SNPs/kb and 6.6 indels/kb. Analysis of the SNP and indel patterns showed the existence of a particularly high overall level of polymorphism in the intergenic regions. The dN/dS ratios obtained are consistent with purifying selection in all these genes, with the noteworthy exception of the [VAR1][1] gene, which gave a very high ratio. These data suggest that the intergenic regions have evolved very fast in yeast mitochondrial genomes.

 [1]: http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000007275" />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/2/9.cover-source.jpg" />
    <title>Mitochondrial Genome Evolution in a Single Protoploid Yeast Species | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"TZ8KYpwxxmhKeSF7_N3P8aCTYFN7lqi-oLFea_Ryh_0","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459324","apath":"\/ggg\/2\/9\/1103.atom","pisa":"ggg;2\/9\/1103","ac":{"\/ggg\/2\/9\/1103.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/2\/9\/1103.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;2\/9\/1103"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;2\/9\/1103"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/2\/9\/1103":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022kAzLZbtOfrvNyMlF06vpRO4GQnsUMcvByph4+7tJubIWSebkUXZcziulRjMyIHOxKK3TorhxGAQNzPQ+lyyUVYI77uhQwZoQthYff73C7X6emkzyIkIhIZjOpHHpr1rCwLxylJg\\\\\\\/dN0aqqeEoSgmZMfYpC0k5hAU0wwwqtwTDkTthdNpeAZXXMEIad4Ibj\\\\\\\/PvilLHKaJo7QOVGb7BpKlm3lISeoj7gCVa0W3DmFMBwDZlKujyWw6Ybz3RWpZTT0\\\\\\\/HX0ybTx+NUzHdZlW4gnt2b4kbuzaHyUnkUdODNo0EPjge1w11t19VbK9QN2QgY34LJt7QU1rIfFXHSGlNDi2kaFd8usvuv\\\\\\\/Tg+KWUNUFbLQOkImjj1Rw+q2ojm612to5iA7SjV3yNqDeFpi6p46hehJLVOAQkb8\\\\\\\/V1yOn8yDbUVxQFolsUGb07TGmeh0XPRLNwEu4bYI9nNZGYG5N5RzhsePteaJO+iPfCM+jJg0435nqkc7\\\\\\\/Zpl8\\\\\\\/C3xcgcPXAdhyh0bDyiKOXz77jqQ8j8s60rXnFE5H6G+YUihHLm9LJ2WbzB+mTfBn\\\\\\\/UkjvqUET+HBHI7gpsa1Roc6omkNplgElf+Bk0eOkddcxD53CZjjEZYgwEttkZQFaruAjZBWUuD+OwEbDxxjGRKhFodZo0gw\\\\\\\/RfOl+iwKj\\\\\\\/kR4\\\\\\\/iEMAmCYGU4Tr6J7J41eGjyCRBXI0o6ItVc5smrjCSwcLFkKPZDaGCdobZvZvRhd2OZZPBoMSiVvhf8O5nbNd0Axz8S83iHeaW5vXxiZjfbJFyvFdZqwXv8LE0tynXuXzuejFk0bFH0C8wGEgY2FgdZ9yzelhYejvgWZfAnH07TzaQ77sSaX14Hxy++8OLgr\\\\\\\/Wu1qmkJodRRr\\\\\\\/EAiF\\\\\\\/yLaWrs26FKsbqxfzOgVNV+1suzHtDISM9NZhhTqVROYUBcGuw895Vu4RbDp80jCs0YAR+FijHsPdwXvIeA4ckUqsJJkjIIA5eLI5MaOq0l7z54Lp+i\\\\\\\/+OT9iXUg6vLxHTK7NrgAC7OEk0hh3YEoJYDObHzqPliEEqT\\\\\\\/mrqgFIWc2HCKp0vQbEIyXRgX1bT20xyzlbM1mPtp6rzGYuYMtHJcYuolEk4rdz9mY9YtwbqLif+nPyGV4l0z04TWStRU7hEsix0tnyZefHbJRYdPuJjqJLimT\\\\\\\/LdqhUrUUTN0GHIPcfZIKENs4up10hqrG5uQOPeNIwqwGJqCn8BwozoC2T0ns\\\\\\\/nTPJvFSTaiDHOqm9G2E3exI04Z3qGnZhuB\\\\\\\/KK30J3W\\\\\\\/aEm+\\\u0022,\\\u0022iv\\\u0022:\\\u0022Pk4FVaEtOeqtMFwGSAHuQQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022658a72423a1cf96dfb8c88606acd00a48737a506ae3e6ca3c1afb889f7c5daf2\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022FKk9Qn98jl+ZtVNaEty4ysPayxzWWPo\\\\\\\/ofi9DBjZoZGpEjs24LyZF85bwfZrbukvvBwiGsTk1wnO8ud6Jbpk3T9LC27hJs4KWVvNNUghB8Dy80+Csvfi5cuwQN8TwjJJrTUj87mH1yt438ass\\\\\\\/rD6f3xtkQzduTimsGnTSLaTr6AIl+skPLWBUQRk8Fd6UuX5UXrM096OSdOMJkT9Nl0X50DuDe3EY8GJHHxZgpiOsZcUIRZaanOYmRPE7Zfyp\\\\\\\/8DCShxcwafJClz3KLhHnN7gsxqA03z2NGN4+ipVZz0FD375UG4FOHQvVRt+wle0rXykdWZM5pdvOtCEcCEHG81ecobYU3zq83sQWmkqeCouGj5pm3OjeNnZYqk3tgdPlGv3LVf1PwkEG29g14fXQVfOjOf2xuLHhGza1K2eU1dVRd5Q8taecaY4\\\\\\\/ACjs\\\\\\\/tT0EwMMuMAcDY4dYO3Es6sLsgFncsrj7VhE9RSwuwiCBNi7CL+GPlIyMYWcDZW\\\\\\\/YRM9M\\\\\\\/IDzmQI4VgqqDtLMTsk+xF20MmeC7chA8cEJDCziFGm9R51GYNoxxbpaTTfotBXKegJc7BrquYdHMg5AmVaIiQp48WCp95M1NJ0DqIdVpu4vsP7RMT4wcTtQYenTbc\\\\\\\/+FeixUvMu2uHZSeCQSEU6bGoX4z0SXwBmA1eL91LYpGGUTJFzjY+\\\\\\\/Xu2af9P6inZgC6vBbRm4AuX36Qin1RO361QObRIOnuOa\\\\\\\/mGy\\\\\\\/i9eLIQ4AxE9OZgpiSpO+8IP6uxE2gBNMlfOhKoyWN3LZLfnD22ZnSc0I6CJBIAR26qJjcKYVSVEQ1z8udl\\\\\\\/WBGDFt4VRNoY1M7bWP\\\\\\\/Dgh\\\\\\\/IFiY2eqVYWA+p3N3RlyBy4dqLurevvpVx7BrkEvkQ+qDVA2ZgE8VZgRMZudLz69qeLDcxvkS4Fb7CqjnDsrhB3Z4jVVfRcJJw7mdQ+zL6aW7Ov7B1Qwl5jEr7tsPDIq+LZhjJVg+NqHwkFercZ8cC7hKrCiJ+ancOdIsdSmwRGd3MbD5rWucRkbA+aYJgzuYwB8yG3cXcV2kHo1W6uGtp7he4phaTSC\\\\\\\/GodPcusAoCJuyfe+irs8TJ3Xh1\\\\\\\/Xw5BfULaOSCGyqcWtnJOQ1A\\\\\\\/avXcHbMmx469Y9j32FhphqumhNYiRDSVlM9XL4X2dbOZMqOzsU+wXncGoMOgBX0cUrLzGjh9AUH+h43CFyZe1x9ziP5H6tuf0P22cfL\\\\\\\/FgRqRY3b\\\\\\\/wiU6mq5SHKBnNqzW38525dWfFOOMS1e4llNQNLWvSJ5L4box0VcQO2CZKk9lLiXNsX8Ih9KxFaWeSQBbzp8tQNcY9Yc+9g9BloDVWyh2++2SkIZX3tWtoMBPpGS6mmtmqZlye6GIdSkEXRQ\\\\\\\/YZak711EE5OqYYIBHWgn2RCGG4VAcdS\\\\\\\/N9LSxD6Btw3\\\\\\\/jU8\\\\\\\/ddECzSs8GRrPnKGHrEyhro1LKA2pRqrugoOuSKFLA4mOYR05315tO2UUjn81za1\\\\\\\/rbz+CUTLtNkbEtsvFn9ezjZF6BPUYlTno+AoWZNhYmxdC97a253h7tC2+zqEj8iWEpwIJfjJXMo3Tv5TZPPexKrUT2h++FDWSpyhfVfhIfMe263HnjqUXOyPnkLW1L8kdLKAmPta4oJ3HiS7b3NnJayHhMo9JpwYpUESgrjKw+6b0e6cK3Up4m3jemO2QpEaQ1Di2ZUxRNdXxs1cqP3I\\\\\\\/4fEWzGWK0Cw9lOFFnd2g25DFD+qI3iALHDcyd3g5F4xbOuriRmbQa2EOfSK\\\\\\\/mKvDakkMBtnO\\\\\\\/SyrboQjAejzkI\\\\\\\/6sCd+R6Nji82ChLhOinpQglhlsU3jXCab\\\\\\\/kyD3cJzcwAhEXz\\\\\\\/SxKsVhftfuTSffeeP3lw+8n7h4Cvk0F0vn110mozDITKv12B0XE1xXBo\\\\\\\/oGIZgPd8ATVc8bKIleQWvUYDWe7YcN0L0kcerZyr08EXjYeLHaEOuRsDHVC3T4pGd\\\\\\\/kKGii3VP7rB1cirhXf8ExbGZ8h5CXLk8d8HclozFEZmIhueyyZgO+9XA3M62STeMDKcwMLanxP3y\\\\\\\/SHLY5FzFzq6K6bnozMlznM5B0Rnlod98Ruz1mXp9UfXfmk7GW4fXEyQIaGRqszoN4zUpjWxeyXLLWiD\\\\\\\/4wkjAz1h6fROE++ylhA8bpuXZgquZ\\\\\\\/jaj0BUEOU+Kzwp2\\\\\\\/zU13iS9wR04E3YCH0zYo3p0S5IJGHNs53e47NtaVLSCxhDgq+mXooWgvB5V5jzsjU9fGUidJgA19j60D2LFM7GkAVWb6VPTsQpwEFNBYGyGlc5IlPGk6GMye482nOtBjVYbi4ul0jSPIUOZT8nhxi4ps77XX3fMQLji7novdo6aHDGnJcExn\\\\\\\/oexHzwhRAWIbOWFPDTG+cfDui7aDoE6DE7u6Ri0xuCFiq8IfkwDmPzIU3cGqkRV\\\\\\\/wfF6pmLuEo1UxtgTz44ZMT2D5oqkIAE+aNZdb2gUMR5Ht8pkcFTHVSu54\\\\\\\/NOAShiVYDXqu99yHRa3OzzcW8qkW1M\\\\\\\/+iDpslb6uZDzq3l8roagoi5dzJ0W+EVVucPSK8ig+zMmzItixDA3V9v9ZdUXZwqUTw2DzG1wTMcaS3a5KJf2W1UVVyp45yemkKi2IMi9wsLU5u7rMasDuwwd01m2Pa0nczJuGrkRujNlFOpM3EY5lSjKyNVforNiI7COb3OzNG9CrHhsmiF7TX3sXc8+lR9jRNhVhpYt+TjulQfC2d5lY1W3p2wEDRITHtQ5wm5EYeVlx5PtlxJoy+6Zuaci0\\\\\\\/p0pcF5xUV9HA76n\\\\\\\/+Pqxqw0LOHXTuDErGcrhlLE7bV2K5COdeYS3vsHgd9hlKRP6J9NXZstOY9J30O+4dByiiHsHS596uu11a\\\\\\\/dkXKXxZE6pTKSIp9kiaHJ\\\\\\\/RZAJPnsO5vBsNkCjN32vVDmggVcMVEhn57Rho\\\\\\\/iRukGR6Q6vVIEJFIMrCgClDvxKfNsJAjBc0o9W\\\\\\\/6sJ5b5tRjMhg\\\\\\\/Kr7IndifgVXPMofG7fr9Ox0gONzLbMbS3OlBExMlMAP7M1tbCwegkDY15BtvpMmitJPkGVwfArgV7\\\\\\\/RRIq7YueptS0Osl7aQXvnw2by1JmwCHEiTChMzKSYcYe8WRQFtvuyw+\\\\\\\/\\\\\\\/IdaXOWxSHWO9Az3Ah+gWYT4A2ahaFFt9PTUFvEo0a4Rr5L3Uaofp6p57bawz+ewp281eV6kEZFxU\\\\\\\/ntKw8kQXhrmOGmfL34ey1dWEdEYuDpdxaOLgx3t+r6o9iG7qR+urOzwxG4f+iNj9uKExLkyc1qFgfazY6HU1gKeQ\\\\\\\/qMopAv2c1v\\\\\\\/hSoXvSJH1OwskkLWfsa0aEUtJrE0CqRv817k8p2iPFy2aVLBmMNRUYz0G57+l7RRGTsPzRTA5XvXxiSEwSil6MIBReL\\\\\\\/tFIK\\\\\\\/kjgunE092IN04+ApDtwFYF5AUuDpdyGsUCccLwcWaitH+Ze4LwNdA1BAnomtVkQCtd\\\\\\\/4Tg3qqTjIUGoAORKT+MPl1JAWi2XzjVP3QNRJBYa1hPfIXPtqkbBEr2QcuEN7XP5xAXz1VJH1ZVa07VMSSAlxh6Iw50YxP5dlDy7aVgsQBPx\\\\\\\/R2JcgjUKhZap5MR8n7KD\\\\\\\/\\\\\\\/\\\\\\\/3CYUDHpHB3GNHUzrgALh7QYiS1tA+JgPC9aJFTMCUaiek+YD1azZYbVeOlpaGJnbwgLnKYiPHGwwtTQAZEfhffxcTa1Ko6SF70tMQjqqbyfIiU5VumiJJ7oYUCxSbJ5\\\\\\\/txBymadFUPTwSm4uEAzl5A+3+nkvOD\\\\\\\/sG4xDqmEMMQxOUk70tHtLnDq3zyrJj3+UeYqlTx5+3lQkm6t5wYmqlKWHQFPr9R1kE694A8SwhDTMP98atTS+b\\\\\\\/vJbgax33On8MpYySeWsU68Nm2gJfDZTEvzJo6o\\\\\\\/luZUWSnpu+x58tIDm4RTuo7YPJXAe8gU44lTt5KuvZn3EY1DQlubTf80raGC+E5rMF2FGsquzHxh7PGnkD4\\\\\\\/GjLtp3fW9wIfA4c7X23GQb6d7\\\\\\\/WK2GdHCdoblW+LFjZgGCMiSGpw7TR\\\\\\\/UTD5xYwzve2G5L6dhiZsaCY8rFMMSVrS8UaHFZY2+VeV+vnp4E3A5n2T\\\\\\\/A1tbuA\\\\\\\/37MwYdPybvsD1Pf9nd1HmWUfo44XwipFgd85O6rbCfEugv9MpplMbidhj6JjvrXblj\\\\\\\/HYUWQH1n6pDk11zb7A0ZA00Y61TDkosYPyvyH2MTb2t\\\\\\\/Vj3t7NKJ+JgVDFbqcN79Nz9zSsPktD1ZLrgxVkcOwlDwY5+4fhk7i+JG3ZwRsQGsgPAsiWpI3JF\\\\\\\/OAV5f6FhMZf5iI1G4y2KyM6TujFVDHeKW1hf+KE7yuPLHe8xWtvDhAx7r+4jPqg2oHGf9hyO98+YFBqyoGOEESh6FbSwKNS7rJuTROgihpS16lXAcKLpun+3+bRq2ydpr0icPmiqorDRczgzDfY2WyaZHRCls0ciZo6RrQK2H\\\\\\\/gre0zBAcr94GA3HiW+8Qw3gyXvzzWGiu1kOZOFT2WlIh5+YGCWyVHvpi5M76QfzVKcUGtvO1pVly2oPInyBye9o0r+91L1HMbXZZGcpK8zYNoQwOExp4Ka2lJADQfwJfgeo+aWcvufuC+THTRpHxf0ykKautj9SwM0bgSk8r6X31VVtjGV68qSZhtP7UHeZSH\\\\\\\/+Q9C6cRyz6mc6mlHDLXIE7UrEEj\\\\\\\/5F\\\\\\\/II4zWhBYiwex8gH\\\\\\\/S6lWPfw1TlXrS6vOIyF6aeFO4F3I6F42391oG1Llib4Nxc59HkUGxy3nn4S7BKvKfIIueOXIFSMd1ELoeWdm01Anto+hNuk5a51ZyGfb1dp\\\\\\\/BwqKxIfbAlresBJ+9vb9UqAD4CvSreQggZCGXGffd+NJpOa2o1KRip\\\\\\\/g2Aa6ayNuZWSAZdpyTZpN0V+8LHPQqzIW0Sk5fsZwC0O9oVV4g0bv6vfGEKnlA6J2Iy0Sc0N71S6RKNesZBpIl0TZMCAzw5lhD67dAaAhMCekTV2Wrki+h1kLr2ItGYmc51JGNThplk6LD6AF2WDPo04ZDm33\\\\\\\/vZRbOROrIaYCZ6wghjGsQJZEk0Qeh1S4sSFFpy0KBSbkqteH+u0CsXbVNTUHfFSKIUMnWJBWnRxSTy5YkDQnDAlr7h28DAurTI3eEZxz0JDmJuIl78O6OtQcdcVOrmKNlXLseOKt61seXpojSEOqyMaE5c3vTkYV57nWvpZQZQw4YSsYf5A6YIN2Jc8IV42ZYFCOtapTX\\\\\\\/9ATde4jzRTfY+kekiMY1xZ1dPuygRS8gaXFlRCfiMP0+UOXtEQddmtOrmMCbJWvmSMeSrPS7EMG\\\\\\\/0YTTyZUcVw49dxzuj9zS\\\\\\\/Oo5AntNc03HArxYJuDEfGPlMbU6Xml93SSmtHfF7Dmvt9MG2VIe+Rb6CatQtc1lLvp8d2MNN1cALaRpJd7TcJd0oKtzcxr2y4C97IOhtkM0S4K\\\\\\\/QfH5Af0nHKMVCfqwt+CO7LcwwiupMcqt96DNFkU1lowmPxQT57f+T+IIJ7t\\\\\\\/wTffgQTTxRdUkEaFzUR4SamwbnpJpzYv8uzCWczQV3AdhyNFrB0x3X+xElCXMgWVtTd\\\\\\\/23JslQP0pcM36z9DEfWWNJo6utfzmzg9rb8VIu9npypZwpAkL+SvhlWmXKDMt3adGy5TLaMj5xnNeGPnxjsWPVc3dcxRue8Wn2JSrH94BsYAcRPTdIM+NOJipRxRMByjbg+qZkcxQljId\\\\\\\/bSEflNEXlw9pz9GwqQQ5dD\\\\\\\/wXWawTDowwva0Oz4WEB39Dx9SnFlKKrm1GUESXOCO3j7OemJPAYI011rTZXTJrHKwz+juDaSjG9218Q1XrW3PkyEVn3Ad3bwrMBEKkK0LGhMBjofUo7csT6e4dBNSH9VBVCJAkH2PjfUZ06LmwxQyCCWcq6kw\\\\\\\/16HAMAfqVf3onSGPJxClrOVw4NHjisy\\\\\\\/fXZ+rbkOY7zQzKEAruEGC+fyUrOsI0v2g7bSUqTu9vkdB+ef+GyH+K7RMv5w4OHTOvxUvzcocbS7Fb2Hp9\\\\\\\/MPOQTvMChQZmFHRaVEWms4dPhgMjZC6RZ3ZcJsy6HHVa3y+NHepL548mtJSP\\\\\\\/svgTdU4oPEcKzMZAy+ez3npTQwrq9klMXJNps9anpnoriokhUSe7YGx5SU3tyZTWUdrUqyXIP+SBKqwbDkKurKROVwL324XtlNuAGuA05fhVJzXRnDgtMcoIYoTpmPf6i7Ds13VRbDigOSTXP\\\\\\\/Brhji32yk2BTix4SnL9zYdaQyQBCd7cmnzkIhyhuc8Gk6wxOmXThaWYk8uod\\\\\\\/j6hLW+4ScyWndPPz0W6ib1kwF\\\\\\\/cJnFojngv4pdzsKiGSQUPspI0wzJgRZhYbNUu9GnZn+PCJaEwl5xZAVz7d3p2b+p4k\\\\\\\/Ze36WDOn8rabP\\\\\\\/LKboo6a3AqFYL1y1VnSCinLpj\\\\\\\/9sBdu7d6nG5AOjWdmiR9jVWsldErllvXdjQtHoqAVF66rH491t361J41qMXtAPXSMkjnNy0NycVaANmCnWeVHqL63mdKgFcSeurhzt0rD2LvsOMY95ySKw+3RcXkuJNPvu\\\\\\\/Ts8LB\\\\\\\/D1WK7C\\\\\\\/zfSD7V3KbH3EZwTH9ghsEmZHnHwcxnyzoHHbEIW6bHRzaJlgyLxczHb8w\\\\\\\/Lku5xyzq2a00RY9Fwbqz8LRrGGQplL01k\\\\\\\/6IrvETV1l3vlzv5IzdDjfZeX+3cdO+94yqFYRme5B+4s9esDkeXhT2JUplAKxK\\\\\\\/LpliJ+ndQ+EnLKm9ppW0gxaKqNRbjPZfF5N6qJVmNBROQ\\\\\\\/zkNWNrS9BkDI4K2RhyyyaK\\\\\\\/q3eAFwJVVtSO1C1MXntXMhecEtTUb58i0vJNJchBIyipe7EIde0E+pdUJxXyI1dM8l8aJI8X4O4M8nAb1r9HlIvqQ9BitK1XuSg1LHvnXkb3tez8DqZWvZwHHFEUMBs1UZ8DwbGIS33RTHLWKGykwjHuBD\\\\\\\/sRnnbVJPxDZHxAaMTN8MjLt7ilEAY+\\\\\\\/JxmmaTn84l5OV21JCnGk+56oMkHkIe3FqV5JQsfTt05T6Un47T1OXUJ4V2VmyZkV3\\\\\\\/YkN\\\\\\\/UeFUF1L8n0o89JimWOEqet2454LVEzGsLcz26JXdVEIfEaEKgPoz\\\\\\\/49qFTG3k9pGBHwP6PkWG0X24LODQgemDxiO3gWIGMKZexypnom2wlzcApg9EYOmA4MWl0ucLGZx+K\\\\\\\/JkngV9V\\\\\\\/14NNyUGiNGYuxda1\\\\\\\/ya8vv+j0MU0TuAeL7Fbssk8EYlkQ6KQS0aIherNopQNAgTKAkpNrzLd04iALrv+xImPhnEBOIpZKJDm9pEwor5Z8kM1scloHPFkhUWylQaayXqpxXl\\\\\\\/RUAOvcDcr7XrEe0R9iUw9HdJtvGTwj6+70nZMJBkqhDyPNbN2g+N8Dr79VepvCmCDFgrO8Ny0olSonQCvLW3emC1s+AD\\\\\\\/uo5EUZ+NX3FTYU5+OUbrKme+9ey9Zw5MPyxxdPflYcSUieUHuxAIF9d798qHN3anUADodzPgtakOo\\\\\\\/5F7f9FW7cUuVsirYHLjrP1vGGLm\\\\\\\/GihIZ3cNsy9Xjy41574VppUJxp39PcQLtmVgiBNfgjznZCi06fDMmG2zHik1YoYbnQCv6joalDhwvfzGRjlgzHbgS+OhdSVi8pUF+sIWoJ+txDQpmj8cJlgO5Kr1xJwsiWWHuaEhvzGawok1HXWBTpFji35uYr1BiewdpgJHP2le\\\\\\\/0cHgATOUYGlGlj6DE+WxIoYo10rCdq3B4YdlZLCWL3j\\\\\\\/hkS0vUAGixox+WfbpLQH4PaGkhh9awn+\\\\\\\/aOKMLTj8AIx28uku66dd1Fke5ERWfgzPQJD0O6YRMX5SHU6ancBGOPQOMEp2SUAaBQmUUqjPmKZWmwk10bsNgLotWQNoow5iykM+aDTu3rH8Gil8a58gjAdnSZDjuyeBYd+whP+XbK\\\\\\\/yAs5j7nU\\\\\\\/fCUVIUuEL6wwMfomHRcOBQChvIGdLBHvAKiOsewmjzsqjevPRr2kHvPTIpXXgawpjU+YPi\\\\\\\/RmlAKPuJJrQATFrY\\\\\\\/JMg1NosusJ770lT7ZUrYJY5TCjm5HG++TnpWw85L17N2xBVxNkGjmJ2InVL+IKTd9UkVETwIurY5BSmOVWzIE7pGqtA42S\\\\\\\/oxgCLfEY\\\\\\\/cOXtgCBbHjSh7C0QQAiccbnYakmU1N+7rI5ypZEn1tAMOzNowQKdqkqvKQ1uycyifdWWT\\\\\\\/SsOKb6eHhxLMjkJEJaaHZQIiTtb5mOooELQtYk\\\\\\\/nowpp3pmuPIge7P6zB95hvgQi16Ir5FAlJPtmqVEN61nix2fEUrxt4WvE0Y3QINzM7KPEsy+1ZMpGnFp4DGS7RYFksMfp+ffzPhS+1LD\\\\\\\/mqaj5gI1dYYFysR+I5aZlp5hNX2tHGiuodEO4bk3YVDyoOjv\\\\\\\/pW0Pbnf+Dt1aWz7kacgQWzxVRXjhreBBTMAwWHFYKd\\\\\\\/COHmHtebVycQtRpLW8aZzHg5tY6wuWYhfNC0bncooUL2RXwLSKfP0hE2KttM8JtORKf5JDaAWqJyvRjeQtCM3aXmhSvxiNk8gM4Tb1ROs2vCpAu+rHJw98nspx05+P9Ffv\\\\\\\/scSn+TbwIOBU\\\\\\\/fP0OcEXk\\\\\\\/FkJxa2pw1Svz+OcgmNVWIifCh+CJGmMjqWNZckLw+9hfX17J7x6maNMCfgaydFzeTJwodxue4mWKYE6\\\\\\\/a8WoN6V75CZzsrQyfTThXBGXJtJmYaunj2qngWBYne\\\\\\\/C3IxSJQ0UIaSziKY2qBUbTD7dut5BnFUP4wnL72BM2+xTPUu3QFaL360ZawIwjqzV9MhzeJtiqfEajnF+YoLzhM8l6WmMuf6fDP5+QsEaKa61YrXWEXzw3qleSB+bV8iRVlwUGe9qw\\\\\\\/6EnDaMgQrbyKA2vLD8pSZkpQGeU3\\\\\\\/V+FruyVX2gZLgViBH6YaEhKHHYbfgDO6OmgOYfDJAWO3XLgR2Vx\\\\\\\/W4xzDnLGX5fkdPCbWMIsU004jkIfCBOarGp1\\\\\\\/cxWfEzHc1rOtSmuhfLq2GuiIveXzDJY2DrKjTJm3HeThZ0\\\\\\\/nttVa61TY0RESgpynrLUYcmnWw1di4RqUXfWnlBgid0z0lWboWl6o\\\\\\\/OqocKOhhIGXzb\\\\\\\/wvJyi\\\\\\\/kq952ydkOF9N8IiGuOZDQHmcU4ZdubGPsFauycJ9NIrCOmx4bhDJpFMxR9SAptRBklbSVgT0B6RYoOJ5zp0vsBzU6fO3ZFZV8ZCzJu4AEb40eY+i3tasEP5IPmkUTF1ny1YyW4VyFf64mAZiFKmX7\\\\\\\/I6UReXhF9Bap0hj8se3wZrgoZOU4utHXGfwpdGoOF9jgvHAh+5Gn5mBgtFRX0uY4W35ZW6EuIqqd9tZL9BxU2devTYDV6fL+cJyF0CcBmtQB6oZ+fFZog64m93fubsa8dFtXbT2RBpcLloRYnfGD3vvB6RySNBvo5HhVTPi8nApBd2D9+lJ4u\\\\\\\/g0+MvA6EkrEH\\\\\\\/z42Jf4w+9khv4tAYafT+Vzb75tuymJ3amBWIdHBbQACqbzxs8hsg3Ntg2gPpULDf6xvOEV4pPLC1VY18T4Ry0e7aRX8Kz6qT328o36NI3pgxPh3Z32Tot8vKjWPoAOqi3YY0uRyabYVXAWqg++hTg6iRX2jEq8AWq2Pa4iY1OXfV2IutZitJ+28Nk61IgxVMueCkDy5mzyOx\\\\\\\/idCkdtosRuIH8wB12HopXvxfhDnbLqLFs2JBwm2NSrq6lBM3Q9c3gUwgzCKZ+HHpDz+CBniIrBFnBq5ewKOU3FZ+Lix0u\\\\\\\/wo6dtk5ZqN5+7\\\\\\\/nghkXufGiGhrr1pxYVF1GzfBiYqWgkLZqj2hkVbwVuU7grZ5UbcnoLD1we4oyijgagY\\\\\\\/wdIaKxRN3pGBulVtBfqs3D5oOqA7YGX+rSS\\\\\\\/gzdWipRbX9p+tGNkjd7ZOavWZFCE03bFWSGDLoeFqGmricyG4rY25vFJuQMSOdLkBVFmsldEE0w99rnseyBpFaE8RNUwuqRt\\\\\\\/tD1cjhi+A30db67EsdCRblOZlPpXA2\\\\\\\/vxK6to+Yx6nDpBD2HmmeBT8N8klSwXCYty8fVExFJvd4uO+zQCKvNwUqBI7FEulM2np55WVi6w+\\\\\\\/UHZTjv1U\\\\\\\/k\\\\\\\/XAAnGsNFoA7qoSEVagyPLsRRv2LZiWVO0hVBpGoLR+kzY+bVVyxrnS8XSa6V6z1SSR6\\\\\\\/WTmwmt5NRv5JNLqBhOUs8laA9Kqyu5\\\\\\\/bzAR085nuAKAmvrQ\\\\\\\/aERhZ5xVNOKr1uh4LsiZsoax3h3xue5Ih6bhfOVKXMONhnOSg\\\\\\\/hQ9K6vWNx729NMwt8i9s8GfIHCx9Cle12Tw7dHy0hzqCQvzR+5\\\\\\\/rLMwlY1303bTQiDpJESszzwvQ7ow84XytaHp8Isulay1NRWEoKcDn3wNm2+Sy6M4aD5mTIdkuBLB3Zg\\\\\\\/jL5q0DO7rl9QxczZKIK1DRpqV5j7IgY9HnW2s+P+fd0Y2oP0f8WQHq\\\\\\\/Z6ah4\\\\\\\/qwIj0HGbBzrPhh5RbmAf1ByWhIz2SFTGpRlPVrcGWx3Pwko7HX2tUyPSZGkTCLStLmbTC6CdV+65GCl\\\\\\\/eJJHamMUr20V4Uu9xGI64Q4wNK5phy+XQJaqjQ+9w4IFOLUtjk\\\\\\\/eHSGvKMkv9+\\\\\\\/nhGfXFSTSUStAA4xbiDmWVSFjsjZexVDf82q9q0hLU3Tlk2\\\\\\\/+I7Gm9amDIY38oVIaer0+GxNcAUw2u38MYPfkpvqQN5bg0Q31i6dBztDGl8Y47cYoGL\\\\\\\/qZJtwpvcdEaselCTMCSTdwHxSdUhzC7icsX3k0n3IoPW7Bzgf9YBgVHZWDXbU3teRX74Nw8q8TGsfrF\\\\\\\/dcQf1oHFepJ8ltBfmCV5ipnQg\\\\\\\/6n1zRdroTEh8N25PiolAfqXY37KmBqEfI5E5afslWmKGM4i8OdMEMnqI89W8laKnGMCVlGSwhUMnUsfCcJ3VDB135F4oU6eis\\\\\\\/GmJxTCPPv6LoInqaaogI565vJ6ayT\\\\\\\/fzBzUw4rDhqugGk44WotcRyB\\\\\\\/lnDunPmAe7E4OWSq\\\\\\\/7tucyYNY9gu\\\\\\\/uqpDot49IqdQpr5xGxqL19o5nj5\\\\\\\/IVLwE8AysmwgRSyryWYYVlt955ldv2DhDAEahCTBPZ1n6HSTQdKhIANwAOu\\\\\\\/4s\\\\\\\/Pi7+0Ro2JaLOayxao4SsiXSjKfNgQkNBekHZB5t6FKvc3cBqsLgZr4P+eiuJvoiedhf3A7XehoF7+ZROiSVzGT+VqSwu+65lC+TtccePrudQfTNK4EdFbkyqBDnhmKiu7pi0ihMRW3\\\\\\\/TY\\\\\\\/UZoTLK0n4JWMp39DFA+9Uvpn0BAPGieggRiltcTBLLylJK5zx5PykKKoWtO13MpJpp1MCWKgFN+RJ63cFmzFaxF+ABTnAPz4LOPEUhQNdZHITzrJ1ViV5heiPCwJdds9UOwYGcYYVgE+Jzx5sTeQ5X1BEYI0ZhIEmagsHtklyYj427ZGoRvNCJZ+sRQs6KmPd3cnkTb\\\\\\\/aZrmZYm\\\\\\\/8l+ErkMIAUY8P\\\\\\\/BSlHFSKdVrpz9bUVZPox8EKWdrAA\\\\\\\/NhJzjEWdJRScaCIjLcgWVp2LQGui6kFKDBx9KZtPtP+JEyU8Ci+yoNsOkC16F58sD29skecHbLczO+Bj80bC2HqsXFKbS54IpDgEnXadtzhnfKwR\\\\\\\/HAWaADNCWJPpA8I8CSI3ZboALXZouCuEuhNKEh4JTPe1vwqWjshJ63+72edfbSS24Wk1m\\\\\\\/xzrLbI4Hqr96LpwBBnhw2MAPs17y5nLElTSs9YVw2GKrD\\\\\\\/Ea\\\\\\\/E1BzMI84b4BJWduRHggk54nP18rNOc0a39WHeSWVTou+7YiqNUb9y2TiWc\\\\\\\/rULi6yBbbZXsCrcyKckbunpmDy0r2sFWI0cuuBpgOQrEOy8tK9QmAVDfGeOsJIEHduutK7bZLlgIRtqOw2fyJhzAFIyMw7gZOz7bYMIQQgWQNJgt2TWAhZWsUKagXlYHoCI22v1HGGDuDjy20rT+bWs4lrc\\\\\\\/zuApkewhYbtHKfJsFY7qXKPalPBs3x5gmv\\\\\\\/59x6xm6YgoMO7yZu8hzAYUc7hQJ\\\\\\\/o2BpyuTo9XP5cs\\\\\\\/qahLRSDPjG6mxBhC1sKh1NBRQTY1VuKbNwIQzp1UYZSWRLMkuPhTrAjqEzeWbXI6KBGX8DGNAlndowVQj3Z3Fojlsg\\\\\\\/G+m2zuGcWo01C2X\\\\\\\/\\\\\\\/HZOlVj\\\\\\\/VvREHA2b07T7CmMTi6QiWbGH6ikt84\\\\\\\/gbJO1PS2iVMFb345RiBsepKvTrmoBMfr\\\\\\\/9EKPZeihyKSG+bgtPHOW9XHXg2EL6nLo\\\\\\\/sXT8c8E\\\\\\\/WnkFu2tARLDNnrVZBD2+AEgyIrHRuWFc+m9iIsb2X0p3Fn3uoSaie3dSNYKMJL6hK\\\\\\\/W2La2etyexVwHyHYyTxteATLG9tjY7kkly2PJ2s\\\\\\\/kbk\\\\\\\/mm8cBcMDSsRDeCneGT7Aho2MhaoLtEVT3lrJfkqAY\\\\\\\/\\\\\\\/PE6CG4dZZOHNB\\\\\\\/O6ZAVNHsJjM8eElcpykdFkBpzGT+G+zwp2dkrYs7e7tDWyteC0UmdTq3tMLMOM9EBz5m7JWscEtH4lR6ZirET7E0ge\\\\\\\/pf35YBAwUxqoUH6DgTZkaj5Z7dunFiY\\\\\\\/8ycJDNQ5MsCDoRngyvV+pwti648zd8qZmlmY\\\\\\\/+fe0zuLpV09bcKiAl9EFmDgW31XQKBTHQ4YSg1\\\\\\\/Q4Quf6+3UD0cPPG2DFgLG8LR0YvKde+78l+Av8zf54lU7b+mhM7djYUBMPTU766VApaVVkRytTcve\\\\\\\/d55nUPwkMKbJgA8qkeAlURU8tQFUv+AU\\\\\\\/YKCkG6+dv4mQZgojOV1IbTOEMjdlgNcu+mUaZ1Ybnyim283oM7EE5\\\\\\\/3lszxjx7rMOPObjgU9D3d0m1ox6Bc8i3TBreTfJp3O8+LU2eire3c1F8UXI36O2Q+hYWOFNVEgRrie3zW7a60t++AGKsI+3utsa8dmg8jw7DLohJiApqxbgmq7HqgfIE7vywZD8DpGvY4Tvxt6FBxff7PVFPGYLXovF6R7a1ChT+KJMdPTTd0ZFgrWkQWU\\\\\\\/WLkFry2WqYklps8W6iK3GdVeVFl07rX\\\\\\\/YgHnZgsudls\\\\\\\/n9Nuywb4NdVf\\\\\\\/bfo8+7AVuwL0ILwlWTiUMdTaeccQzzEiC2ekHsg8pw5JHgl91BNEp5riywXQauA2WtYI0VrBEhtZPSeRiAMPMJPZD1AbUjLxENmQG9JTwSz7j3NSRPNCrWTry0UaO8F2S3Mll9XBOeRSZFEv\\\\\\\/+TKpWmbh7Le9iM5WYuBz1VAJNTp8gQn1hrJkIc\\\\\\\/MA0X7Uh5KhYJ+1DBPXkGmzPdpXbiM6pK6CsvgNQhBwMf9GaQV0od0cirKXZs3OeLucJbTZw04c6zm9shRxZWxrtwzpMDdiv120nw8HUKtQpoPvxlZvNMVIrIQ2vc2s4zS2g7FNKS70ZbvAqeUqJPs09g\\\\\\\/Dg\\\\\\\/EeFBgPkK4DxKXgTxRD0zx0eaKQVGt4n9t38S3yLZ0AnLCq9K\\\\\\\/cboFL9lLlBaEnhmEBVnduaT1h4D0fbDpEUfqRmoQ6GT7tDinEK0dz0l1izrNtaFonC4VJxo68by6xrp329L\\\\\\\/u17SjN4G+H3VrakWnsrRtQTkqXBZNC8C38rzBDgWxvfqAxGakbbn6yfUbsDZKYPE84H+wxaLkee4BJvMsfnjLSmSL0+QRujH\\\\\\\/Qc4MGZC7krD8qLA57DxNxr9oRfSbEbgxwRpPcJSRGIh7UO4xHhqIQ2gixexaZfe8GdkZr3eYHUJ+EdoLkKnrYxz0rBh1RstB6rIB4HCkSUWIJkPf8QhC1HpQesLGEkj5D72L701imjFID1\\\\\\\/SkaGmC5GPhsm+hcLrewWuJuHZbizfX41FgVMsgsMlSwCduGNfptl8ENtBK5Md3XHkY1bFX5j\\\\\\\/TPDv4\\\\\\\/XV7k31Xh285FAwOJucLEUTVylJtq86\\\\\\\/5oij7\\\\\\\/\\\\\\\/DgBgHbEL+rpJvFuu32YbsPRip+IQdBB4kdSv28bxVy3WlffBOtFAOYALZyAEDYRXxWtiJr1o6o6xkn0UI3W0mbOmPfUoIoTvvizH3ZsuNZCAZ6\\\\\\\/DSMIQwVsK1duhHl7qMyQcgut9BorkaVh857Q9ikSPaQyC7E59AIdArxbC9EcSqdkY6jwhe0vK51Ari8pL1WxKcZQRuYkfvJRoYZBXpf6vpqC0t6r9gqIyGmRgVq5GThpQrg\\\\\\\/0ZtKAVWf3IyCPN9j6YF2qgAEXPvruPLvHZgvw7IexZBR9P3iH3NItRGGF1UUb0xBeFBJq3rGJrMKfuSxYMHAtrPtTdVyfzXMq\\\\\\\/oR\\\\\\\/phu7z2DDMRBk8D67PtlYgTIDmo2cEEQYeWmo8VcoDa2QsKNLw26h\\\\\\\/MlDC6vLh\\\\\\\/OWqgsp6rbHXobha2MxdGzV\\\\\\\/KGhYyiNQC1FNEPhGpDjW9DTcnW2VH+P9ZFNTIZCr68RNcWzOmD3mFOXWSeGRteXOk9uvLYFVfUk7ADSyG8SBMlUANVI0rZfO0MDgx+pt2SmdMClHdIH0AXV2xC7b8MDrDOsbxJG2y9oT3uu3hdjKY8gRMmDU3mhqgD3dX1LIMuVqKMz7r7my2adsQ53bVVc58jG+9qpsiV9XjaqV7re0cgdCgiw+Hp7EBZ1\\\\\\\/QUYtlRmo15rEWID1QEZXdFW4ir\\\\\\\/\\\\\\\/gP85XMNkCUAFqKkTwQ00EaymC5J7wz4fFB4b1wWjeZQoYux+vOiq+GVQsgDqIF+FsgN\\\\\\\/yuoCDFcQSAwWhDrGSiMJRuRmU6jnhHho92sCv8FesC4N6xf2GBR\\\\\\\/MyJ3uwquwHiXZKHFkVFALd8wVLnvAmpRA4sgY4MAKvEV+ZQn6Sd4vziBR5oCYUYmyxVMEHvaqtP3NnvdnMIM2CWZuUbPTLJryDGSdDWZvFl+aS4v2JeNKjuvMke2mIDp5oj2uT6CcmNKOdydzc3rW7D0ifOj3rsQ2oHpMvmQ\\\\\\\/5I6pqZ9G7XCs7AjhV3Ku1xMiUaPeh9wSwWXYjo1tYGEsxsMhcYvS2u4Uf8z97WPVpWHsWe4BheBEPxcFVuUzluzRROcbl+WDdyKvvXI3FA66XYGfxu+pGAGVCPBo2hDVLfpkAjwbspfzEUlo4tMmSWOlCIu4U0Vrc2HW2PhCMot5u2xTYOF+tg6lcP6JSmx2OAsk40gae9qwZZL9hsrgOS1tEVhFRYe42XucRbIVPbPcwMySxE+0V0VJo+kgREDG3laOvBLxBCRL2EPJiZVciQoGyqIZxhiuTBcxDy2uzi1u2yOrjsqjhwhOCrFJ4p0s4Cto0grROzRnFUkMeOcE\\\\\\\/pOhsINh+8xLY1XUO\\\\\\\/WVyxyEa+Mp7Cpdwf+qfmjBOqcV+MdSW42ZvXEPjAE7AzutrHsoUDLWUYenjpT9poE8SnJydmN9fjSlph2u6dH6gQrhDa9OXe1mNKKeW7Rgp5+ZfJiZVyfi5UEWvUHUyIwhMxdjujXv2Mt7WN9YvJTumVGrWY38MEVxir4NOjlydQW9oY7+47ZUzv2m2pfxdIfz0sJmrhjjXZikHHjjpDEAG03\\\\\\\/Qzn\\\\\\\/aJRXKK9Jq5MFkvsoD8vxIMTG3zEqLhFAqTl5sO7XJ\\\\\\\/4wIWnByM4kdT7pI\\\\\\\/Q66XLrrOU6RDacG9rXwsjrbaMAjzWR4v+AckCWGFvo6dJ5qsZ2\\\\\\\/PAF1EoAbvJCty+ZUBFycXLJepyJeFDE0NOpbJWtPygdkFgEflSTl4EaUP30SmCtJsHwcMFx4N7kwcgMxt1SSVgBHzIgMctRXf38LQjo2oQg7ddXSgx5mifI0RM1zOjoMGiOV+vQUxv13T4k0ZUUM4K5VOECGPkfHKjI7+FcF7h8vQn8xFZ\\\\\\\/QIH1eBzCmpAqsghnIDzxTEnfE1TX9brxewtCpXb61FUhBJRmJgVHDRoOBxY7Ag+ztenBYvpJ5S7ga5arwjbio1XlGZkKIC4QyjMvA6fb6dHvN0oU6cHBMFR5EouhXmPXFQgBIL6H3S8H0NQyw8U7996f0OwvWZBTFh3snAeYGY7u9iczAp+YtQNZ3ESQJfh3QFdKD2LUYDAg1IObJXcOERmF7Uf\\\\\\\/tkkAj5Mu+ZTlDDAc+Je7OhF3mbPia5bL4WJxsLPqR5crwAOsQFPkt0DT4kSLDaicaXTzDAY4eexzJF9zEpYDBqN\\\\\\\/K7iJbRczSWHZOsfUi\\\\\\\/P6zevrJhrbUkArK3\\\\\\\/K9Sa6ZJ53YTzrKFCI85Bf1LtWDguLQIN2Q\\\\\\\/j1QNddiMFEcWjmlxkH5FTFQYvTK6Y+EJsQ4\\\\\\\/T7QOAZyYKovjBH1Ij2abTjkcV2sGORX8Iuv8AR5UeUsoSjVtkOcwwUBiQPoVmRGZbbfgP5gf0IdbdxRMoXadrfUSq7TFFf+OZSQsVN5XSpVYF9QbAvbtCa+9r8VvbUvR00fjbHneXBClHCNFpQ49byBHS13zvQCepX3Omup+dqdKtiFgoa8AXi2mf5i5uaNey4yBZ9OdOwTCZ80Q1pDR8A04XvkJXBop3Hn\\\\\\\/UymrCOdXfiXf1mFPtJn0d3qXiV6ka9la0u44faVyhoLumBdWtu6BNnjI2WAyBeyNiIT0i+hC7sQ6fGyjuyWnG1l2Hp3gSpOydH1NBFBGhscEELtmFI1GpU9dmyzUOeqzOzOMO988f5voLlP2j62W3fpmpPDy28XsHKp2\\\\\\\/w87QWdJe4vWpmrMlsx4gFqmK9TmrwNostcbnNKVtvNXP4NY8zhL9wRYsX0hGdZ6BCv3ETr4ubKyQtVyc5tdwUNJo7QNkrjLImpI1XaxiqBhb4JciswUo5Lggsb0ddJ\\\\\\\/2cEOfF41ZLT63UFRztOFgJFjN+wzrjr4fw6Vhz1ZHHqPw8unMxNegwtCQSi4P1ERg0jv24FfUICl9KKPgo1HzE6owBwzRmf2yDspu4WvIfJBOamFU4PWgcyr8o4f2bF1wbarkt615ytRbcTq\\\\\\\/lIOocg53ITA\\\\\\\/tjSO+Nukc7SFmXp9bVt27Cz0NNdOUsn38vtlBZOuMkxkj2NjAAVRAwuTbgffDPqwXyheAI94iEWbayr4XNq76NKAYUY48rdFuCjWuor75G14r9ECV\\\\\\\/sjtrEDWmB51Ph02RzMfjiHA354RJzp+wQMx0jeMC8u+h3iVY9zrtOrNd9r6\\\\\\\/AzJtM\\\\\\\/kKKEiCJCXp5mQDlEa+jEOAZeDR\\\\\\\/sCNVpiAhy9m0Q\\\\\\\/PIxI6JCdxWXsT8NPfqZESkBqGzMseLhY3gH8+rWP5zZAZf0RzilHUwK1RtoK9nyZ19KfgPK86fpSDx98pL3F73fdVhG35VGWze4N49XbJwM0awuNiGguqFkDm0MYRHk8XPq4Q9MTPKLTPWqabis1bpn35yvN6SYRWRPWbCMC9NQjR6\\\\\\\/pDat+ynzXuldQGB2uVWLrAPaDwmBA4ItgWKLvVLvRpvsJnsRRXJQBLEDWb7QDGO5ElmURaxxg\\\\\\\/N5GEdHnxuRCtFbG\\\\\\\/\\\\\\\/Bz4rJ+gHa75C2cdMa4Q5ZHEg4j9s6ctLofBSpZr\\\\\\\/9R5h8iwkHnla\\\\\\\/xV4c3drGaNSqHU5QsrZPKsBwXAV4KLMbF1FZzP1ObIKKrrf+7jJddlDHmoY2YkNZyE6UJnridHMXJM6\\\\\\\/r9Mk5KHGY3EbOlj8XSX\\\\\\\/WcJbBNV7J6KhW9ylH21GndKztbjZt1lyC5WQMv7\\\\\\\/W10mosH8oUPLvxOoMR2cLLXEa5EvJaMV4Jrni0fgvnyNHH3qf4NYsU5GR02gIhPzn9dXU6BC\\\\\\\/GDjGmm4\\\\\\\/Wr1uC3RspH0QR3BNBEJ\\\\\\\/Qn8aZVoxFMzwbTXZnVLnTC9J\\\\\\\/HNv7oCV\\\\\\\/HaIfU7zMim+Pn2FuZtwYBuitRVUde7jCXdCuoy4zdkhFK4AlDl0IH+zDbwOGdf8q1EERlVQNvOTeAkIzPfcZZi7HLvxPiRo2+Ijc8qbeYwsB6XDopZyD+60Zk9sELHPSpiqn+HmgqmIkb8A4T7lrdGoYMpOFF3VEJsgNO4rvIm21lo15sxa\\\\\\\/v02xt7zgEtxGmb2vkuFhOcUXSuHZGMybih0HVYWbRt8VsknwXS5WO0+66F5EWbQ0m4LiPKhXcCfc92RXZv45kQzElUvkmo\\\\\\\/EqvZ7g1y\\\\\\\/GV7PPh5yZh2d9mY1KTBoyQkMRLQyxF74R59p167wWMhwF8FzHUtCmfstq4hidvK8wJ3SoBNxtdlzMfVx8NphnjA8AYsfXRIfvU3KXFQEFZ0YjtOl4t8Li3Ro9KQpPsSpgeAO4+QLFZLiNW1cko4lgQcuOSkdvgRY9O7mCnBQgEEbPG1zbCBrh2wFTdRM5VtnRoYkfbNREML0QqIrb2L2mVj5U2\\\\\\\/xYSuZUpR6jIXyR+SZogcO\\\\\\\/LLHz+w31FjQrWMGNk\\\\\\\/wmrdvRnsJySpCpLsepI0W6MAywGfWi\\\\\\\/vUehquW4T7n7gjLkw8oPNPR43TObB7op4dhmIcJ7wDlnkfoFHf3\\\\\\\/+S1mwTQ1s7jUH6OWVovyVqWK+NEPt0FNaLEQvwrpuP0xH57FNyxQNtLCVKi3DshWHdDcpus\\\\\\\/rURGS+nxGsrFKRhb3px7g7lLBTdogh+KWG1bfXLe4uVNkJ7G7w4iFRBgJH0GMHlsJKP8WEcaoVHsQevkUlYNrT4wFk7R1bTv62vC0NOgCJVJYjKq\\\\\\\/d+xOQmY0DWL0u\\\\\\\/HYa8aCKA7DZpmWZvZFpYqStbaC9hIswMbN+hQZuiKxjsdRQavbSoRNRxIoDRUYMUAN4UzZ2FLqWzMC8YbB9iulgTAkPjIGIIw9WXj7FtYAE5cZEoesFDtCg5oTL41Y34NUMDkvpdFHyQNh1RqER8CUJWc2Ozr2qDh0d8+tYpsgS7EfZtrmSeI8DwSZGrzx6AdPk\\\\\\\/TmrsEoLqcSrfPHBaiLbQC4b2nEr8P2lEG7zfhJisa8vo16aVDUksyEYcZ+uO8IcTRs+0yB0dGi0ROr9m1TZPSV5hdaPoBkYGnM\\\\\\\/Ip+JXvImCK9nddy8RWUKVrmbzFgDSQ0toUiJzhZjAIxsdMu0zSLGfwz3z9TOHuSIXIPE2bXQVS6pzPedgtJB2tXTb+HRaX3E0RL1psFCCWxmSGkNoy7uYkDzNK54GPxgtHwhVqsQrc+6QtIxbGhcwRujYsQltUktpqX+cYvV7yGF0wDzHkvFsKwgBLcHERU5Fnucvova9PuDlqlho9IEPb+XUtGFcjR0Zt6AmHHn5vT1eFCjCTPiR8hgkisf\\\\\\\/3ISy0U1YE\\\\\\\/ZIdBLDNJVYKmOvhFZk5AyRqetxzlUudxCJ2EBGeEgzgAVPpLyZa9P2BIgDf1VHQomQGvp8957ohc9AiN\\\\\\\/G\\\\\\\/h399iixMP1COEsIqOgXwXamX7w3joPvJAX8tiSscmPdtjsTTvoFvEFip0i\\\\\\\/FJIevK3Iva0fhPHYxJeqnFsNMcIWVrEdcs65XI+oW6PAeNHdrZ4g3hpu4Z5VNrkbqshSTXDl7oanjA7PQZ1tBQw7tqoOJngBUSiUouW8sqOZ6ihz2dgaE1j+4TblLQ7s6qFajaFPOarklh85mN\\\\\\\/kpPOrJfdWtdwHDrNhwGQZGvneKdOXW9\\\\\\\/MsCw06Gcl5c+H0gZC3HMOsTUPTg+0KWBpuJV5eIAT1REVp74eWOiV\\\\\\\/+kJMeYlgOZFd4S40\\\\\\\/d+9r7MWxzirZPZ2aSBUdv3P5YmPria5bx5AzMHpRZ74FrsPdGOzC0Jqi2dovaPfLJSXwUMjhaTueNEJCkgi3fgbbrT1A7yfagQ5NOlirToH0QZVvioQxuHNTge\\\\\\\/CIwEJUXNlbs0cLWQmLDriIgRUOIOGEu57bHIMghZsREwzX8rKoW3gB+1gFZDemoQ8nSKzcxZ+EyxcCi\\\\\\\/IhTQ7lwgFOVqfaZq\\\\\\\/WyaSNNTVNr4\\\\\\\/4N1NXOES0vqsKPapiqno+liu2K\\\\\\\/63slNcCRqdGt2fYNpZMWh4eMmjZmVl9aDtsfiOYeTbtwoxk895ZZHIOQ9cohOs\\\\\\\/M8y4JxZWDpqg9etjzf7kUZbW1g0+haFVwjbthD5z2Hxh98pzKRFuFmjV+w+2kW7+KFL0FHKsjAwyY8YmJu6JHdFAsDgqSGW4pBqORQRYy1PuXFMTnz7vF5ub4QXQVeXw0Zf6Eusg0ZVVHOIvJV66tNO7xfXIOB0ZV17vdYTzhFyk51JArtwd6RpJrbQlfnNl8zar7JOy14FusxSUzMDZdzYWyn5MLE\\\\\\\/F9ZohdyXxZ2M6e3ovPhqqCUWyX3QYBSw+VVHdIo\\\\\\\/LYPGBGN2Ck8v9z4745tPS\\\\\\\/+OCcLvvcJc8SJjCXVyRXAF\\\\\\\/FZ6\\\\\\\/DezUFMV3CW6i7gYazRNoLbBt+vODvsh2aTTNvs1MJQtRa8j1Wve2bM0JBsj\\\\\\\/AIhREUoWuho\\\\\\\/ertyR2x9xD+BeFSCK5oJ1i0IpbGTT1UmuAxdJQ\\\\\\\/T+gWgLO6FyhHjx9LU3B4cbUj8NYJChoH2ugxrMSAdMjsQjU5zuFCFEGSJhdx5jFDSLHWgWZhDQYsQcZT0f00IzPxVqp82rs0DIKtoLS2jd6dmdgvattr4iD2AZF5OJg2ybqLADWT6FcCnYFWOxdgWPgij7O\\\\\\\/N\\\\\\\/rWgWwR7+SNad0gPrlC5Q4i67B5oYz6lhaChVz8G6AvZnwV\\\\\\\/CC233cb1+mqaC+Ze\\\\\\\/YhHqAibAugLsE3eRX3t4meq6SKcgPG08dQPwUSIQYcsvEgV5KW5\\\\\\\/JvR8gystnc\\\\\\\/9G7lCZ9cfaVpQNNNW2BjKJhDm6GeaGqZzlJyz+LopjBcFuZZ+\\\\\\\/MR0hd18rJ\\\\\\\/aPmy4Ui6gXlPiVEfJswlDg4ceQCKx9KyAePN++EKZip3l1Zf2hkiU7CCWXxF0ZgLZajxGEyjiqtxsQVJzVLcDr00+eLodTlj40PwA6KaTQkF2XAiA7h4JkuKri6+xTbmFeBlBZZeV+DI09jsgGEymvEyEsQkQ6MPUu48ZgxxrActqFSbRq9FN3yq9YfvHR23nkFuoNr9QKlm0vG1jIoaznV\\\\\\\/uISLaN8pPtqi2rHaMkLHgg8oRoYlE2SBNWBMPHQeU2GVXGhwDBD4h60XdouhF\\\\\\\/ZcwaJG4xxZZv3q6Owj5cMcqlyumvp620f1+F5yipddzj00YTKd9+nFKO1AXoAjj3jTgXva86a5upteTPkLZZJw0zuu0u34FyWvSrAO+N5qYhtlTxu4Gqv+bP\\\\\\\/ZJL11ebpeBhXwb83c3vM8aQmvgXp\\\\\\\/xtoUZZ45edwLFx9wIVs8XGlDQXoE6KlOohFrWIx\\\\\\\/Q9uV0usEYTfuUuig3L9qxgRNs4QSTTV2C4tO2LuWVKxXyBmXHATmDzXlMGfMjoFR6K1U\\\\\\\/HMD5j75\\\\\\\/OuIb9cPdBVGNvzk\\\\\\\/Fx1BHTexRmYD9gW\\\\\\\/gEHbXeJCJ4lq+lMRh\\\\\\\/w1bCQbVINbG9ApcAOXRUNG+EugA\\\\\\\/oqLVxbmbfhm7cb\\\\\\\/DhbsefBnQDM0GJPNB63OQP\\\\\\\/yV26gfBOz5tidV\\\\\\\/\\\\\\\/i0gX6tvDR4o8HUuzNb+7jtweC7+ZC7bmt3EzznhEp14a2dIPTIH+PR3+c+vNX6fewTNfB+9Ci6Nn4poebvF1RnBxDXkP+lOiH5W00XYKx7oeAhKWsiT6nLYujhcAiJLopAZWJXRmBo9piJkNQk6Mk+EhTubBKpbwv+w+kL+1W9dlpPZFYINB2IxeikXCpiDYqQ1d5loJkt3TVk8wLv5CI3dbstIPUEZKlKHGu1EPUKEwyl9\\\\\\\/UAnRl1iv6assKhrgZ1zhrV\\\\\\\/i7zR7JqLstyUirAcVpqiSe+yGbI9JTYJGsxOJIOk61DrZcvShtXAiQ2eecwyuUw655nYmY+UNSghbU9uK1YqnvUGoZZjHL\\\\\\\/I8JNWR9XOf3TNba9iiCo1N97mnR8GqCMEfDrhvTduFY1WpPB51LIIfge74SRwAYalXZKTxJ3hnDRuP1HC05nXVc1XGOFPdXIPhjHT26f+6g4Nx01IvRTxnACz77QXCNJ1d0I\\\\\\\/iuzU2\\\\\\\/Fjrz4OpznGCNR\\\\\\\/PKyg9vXq4UyXSuBNIRECyy\\\\\\\/vgF5noSxMj2IROgiYINWFDrn8T504YLeFqFd++32vbuX6Pdq1um69EFJAZid9FPiOUK4y1Ne3AaOzC0UMfSaefsaGyBRLzDzT63xG1+TgNNEnr4nbTGlFuKBVaShW534eWZpNsezBujCok3XTJw0eOxklHacfGJ9epkqqfm\\\\\\\/HNXFoiCCuF6HQwMEHiLKKKfD8kBJURHPJ8MkzL+DVXuYgsBFHs4aGSm+I\\\\\\\/UkZfC8uysJTRR79B3bdACGgAT\\\\\\\/xOBcYme0C4VQ6yK2s5cMxnTUOqfPA1sm98Z7munD0wN7qy3k95HOBqCAIEZwhMad+3aInWEXAIBfu8WkW14PNvP93S6ndzdanErjMIwenPN0SLsbxq5VSUBQ0J8Ww5VP1p6OHMdltL1RrYmQTL8PLNhhgi\\\\\\\/clSRkQgHS2DdZ+kODO4i4\\\\\\\/4J7YrWifB7K77VNgE1wGcekDw54PeHbldBPWrf8RvXCRHVg1H2XjRwcKiVFdaaewXzgPWQ9GCp71XBIM6xw3T5Haz3VNdr1n121XsAUIozdE3C\\\\\\\/PUEboAREol\\\\\\\/H0qkaNvy9mRlCtcFUg4VY7TkCZH7VashyRXntkCO\\\\\\\/2h46pb8KA7eJcP\\\\\\\/3sOn0JD0+L8UB\\\\\\\/gv\\\\\\\/hzYB68IUs7fq77ItfXJwDCx44grJqDj4TCSbKyOMtU7CQbZUmtIQWOdsJW\\\\\\\/EwfXic6G5Jw71iSpDCgcH5bcFpmgZn0ZCR7TmqS4fJqJFVmEl1pj8UPjXFHUvQGm1DscnbfkHRUrg8jhfVd2hRBJPaoItU\\\\\\\/BLCxySbdNJ14IIvoxoloX1\\\\\\\/8nbz9NiERLoAu+ESLzWVCm05+cu8T0s+eJisIGrerWVNrfJeEx315EmiY9MwWw\\\\\\\/m0IV38epuclPLrL1VuJ0ErwpDUUbTGd5J5QUZhgiY+QK6NsoyxZ9qkUw45KhtfRb8l+i8Q69DV6GvssNcEcp9sqCMUrROYEGwWAlZ2rrnP8sTxW9ePR5uUv9ru5ryN0ctsp5uj\\\\\\\/hy0GdBbWGTk7ru4Efo2D8iUj01KKPlUeyks4DhksbcZsexLuw741E3U7W\\\\\\\/SLYdUeHvK5VbPFaQynvDKmOoc\\\\\\\/jAxwC7WfPaGvQ\\\\\\\/\\\\\\\/0o73zYZp05qKDC57ft1kKuM12MCHywgGfo17UgAAioiEg42x7NzMXIltwmVVGIVkiqcE\\\\\\\/KEYV3Rif2fwsvOqJZQQyV7N\\\\\\\/7Vd3GoNTklFFAvj8nB+o444mRI9r3H+VtqxXFH57YWVpxSHH+oasvB2pdbiHqDLJ1p+JnRyN+wCDUXIO0GnGZWms3\\\\\\\/29nUpaG3yqJe97OMrPyOywbdP0PkviAi6O9sZ9fC3JdwhTQppmvPgziuyD0lTsf9EChUlzLRdStkeGxbuVjeMhlVbNxqlXVcS\\\\\\\/jhEf77a5HteWaEk0p7kGSSkeTRmza5XFXq\\\\\\\/na0y5\\\\\\\/cNPrQ812amySE9tM5UYi\\\\\\\/YYcU\\\\\\\/nSpG5pQ1TC8ZgJVtRBqt0RnfxpsKMaq8IeY9aRrU4Y2LyElWSswpc1aN2vO+wsU\\\\\\\/QhfoLO85YP+H9lWD03EQATyCQ5Y9LmvB2ag0rYobjys\\\\\\\/njgcFS3cg7fiOGRRz7Ba8bTeTV2mAlejT7Y4tZQnnPMkTRlIgw6LEwFAW9OGj7yes+ohcBTgC3jHSCLDxNkcdf9b7ze2SbtRd9956olGRo1gmYZYu3vEoRVg9ShKHbQrTeJjupnRXJ\\\\\\\/YVmBH9G5Pe5vscAN35Z6+ocHu8bfKw+LJvAxhuKmwj3lqiTzKPaEq34g\\\\\\\/QBhPoFK+myc0I7Ct2jQruQDZZofkQM9ysOTw4Uoc5ZrbQoRt+pwMw9FjuEcYWISO1vtA4o5XcYkTCxj\\\\\\\/huTy76ZHv0eEYqw+5FRv7YEGr3cX6dlhaQUkNwgqGQlPSGCkjOrRInuKu7MNMMRV9FT94dDxgCnUfPpdGvLdeIjjhpl6lf16uANdGY2NQf20ZEUd0tLSzHChQ3MTWebN7ZevA\\\\\\\/cdqZJ73Wtl2UdEFcWv4ZwHgsXfFd7YHM2EdtsLzdap7ha38LZTyyuwfiuiEPUSXTKwg9g7qasKlYD1MxWpXBFUvwXEbFduMmxYvgH9lx6u8ZngHiEdKSx4rTBTtSdg8KFKbMlCJgiJT3Lraz4Z5j1JGVzwUQDkSDlCY2sm6aayJN\\\\\\\/2OEGOAfYgY0TDDDE\\\\\\\/eMZXHwOcZ+ZcwEBVp+nmU\\\\\\\/UACJ1JXctCXNEc1DsF59wZesj82v3QT9bmuS7yQM\\\\\\\/8RMAQLt9vH78FDgrEJZCpxRfDk0hkZTeG23NB68Qqog7LZeuaTRCxqa12e3W4rsSEueYz1Lsb2mpFTDCSAffjHCxMV5+rx9D84NON3Dx0i0+6N2AhPUc6LMSS\\\\\\\/2lHAm5LHlMoZI2Vp5MYklLE+JW7h4tNYWdysCZDbnZMMzJ8uiRPolkH31zBabIYh08K+FTHw5oF3mONlYjmaoR0Cg30Vw69iLBvtEjlHuy4xocRpDrPUNcKjbN1qN7q9c3\\\\\\\/Y8BgSdWwbogiCS8U4nUCVz88XpC6m7hAXcukr\\\\\\\/zlRzMk8+QMrvU6olfN8JWr\\\\\\\/TEDh14XTiRCZQd\\\\\\\/YL2R83JABPNCJU5gLYFa5DcN4W8wXTrkdafdxwizeaiDo3xSI4aG7fVXHluGXtM9rN18xi9dP2KASswCpfazlVxMCb9iI5hF3zOBTSnofYgzlwO05Qtdlds4tFBwWeQn5W+cqshICuFD5rAgQeGBXDoJbikOW1b\\\\\\\/xHCYRGo4f8LVsRX\\\\\\\/zV1HbP+2CtyxyibV9N4nAD35HP7jbHB7Jzfci1EvOxlkVnvIuDPKcKaHfjXLr70hT+rb6wsK3mS2fnMz+LP9B5QvIlfj+dJsKMlVGpXVKosZNknMjp3ObleGgTs61TwFTkd5XdGhYVxYW9AeyJtE2vJIddyYw3JQ2puFT6vNqIk9J5tvreaKO\\\\\\\/+P3LHQj6Xg4gMNKYC3ZKtv7YS2jfop3K5nVmJK5cKgQqyR1l9gcXstO8yMGrfFP2Ewko0E9BBVoLaQal4B1JOAT52qVX6JN4m5zdqvfbW0y9YRNw21yYY7G+gY6R+SSzvMvXVrHJbmS2\\\\\\\/9a0I95SajJp0QhwvLXMHNr6P31\\\\\\\/A8cD52rFqyeWs7CXrKfoqaBuGw9sm0URA\\\\\\\/0DlmL6ucgU9at3Pfx+shrGL12vnqybe2wgj7C\\\\\\\/s6MhHlQwIJiTDhkDWbheWQ+Qej+Tq0R8bv5S88p3k+YqfReP\\\\\\\/zE6\\\\\\\/LXPgtf69aznigoNiIrGbe8AGkMf6TJE0Rh8vaTXsf9UDM0uxiiOdwzYIUGDKSfCVzYszUFRTCWXSg2C9GT5iqlUZopIZ3Zj7fIiHgqKAIL29CAYJSCXBbeqPQNnBNshhBXCYxFzAK9DaxmRcpvbVmdMoVZPsCYl2AxCSlCD6Tyoypa3RAGFn\\\\\\\/wWP3bivUn8OZTHrdcTx+CZy2XDgfRd6Tnwew9Z2O\\\\\\\/w\\\\\\\/4PGg\\\\\\\/ufytb3A5ar+JR3bMUg8wH5q83xCYNi5yQzBiZv6ompn9Q86VkqMCiWVglXmvhF8bV0t0GKsV9sO2acSwvyT6KIoKNNpByPQcBy+lzoVX4pJHhzaKvOehMfkcex\\\\\\\/\\\\\\\/TfXWh\\\\\\\/p5iuX80eVKAU251zB0SzCfA4TmiD0hgrWzL4OvqzJu+BK4Q9\\\\\\\/Vrvc6VYkH7nqoVk2w12ZaQeWY64Cf0VCn9QKwPI\\\\\\\/op66oofo\\\\\\\/xs25ByxUg1xyFB0mi32599oNES6UhDVpaAX0M0Q8xzN+Pfw0P1bGWJ9XT+zWebeiKJ7Kl\\\\\\\/TvZi+S2Snz0bNl4lLWKYRcyW8CHO03rPbjoz+XxythHmoPv3MWfQ40hUiClWW41n5xfFaFsgwRV1ztFzOqL4el\\\\\\\/BsUdt8UnKJX9ndwNPhV\\\\\\\/WhFsHWYuUTv0+7lGiWO\\\\\\\/9iXOgjARwLV07cvhx7\\\\\\\/CkvpqW7\\\\\\\/TwT8SFqAVPXHe4OjRICGFZtvjgj8HAxH1pZuEr7I5dqze2UYdkGVVeWZtj4R2jaXdgrhppN4SJoMdhVLoQ882owjmCtskDvCkQbUlxM40m7pUVivV8k0gCFtG8Slg5yg3EN0Qn715zwaDP1dANGEFIlcSf7MR4Croat6wAcKasDvsP1TB8cuCYUpZSPYpSuYzL7L2ManM05+nuX9GZFNTQkI7tzv9gRQTaOkfl\\\\\\\/cptECogHE8GLCVI9xLEU6OLwBMZVwAefVLRtrrN8MHRLSxYusgDufe8gQtx9Wwoc7FoejHrIxy7x0zijf5rhRaKv3N52O5I25paU+heY6AfURwcjdoMDmKmp3l\\\\\\\/aB007DXJya9A0Z4UkSitZUjyhTapQdeOoc+p2n5EOgCOUVGkypjSaiidXBHUgzUBg7la2Aw3j6p8PjsXmwiBvzirsatHhHlXA6euPzBFIPD\\\\\\\/wayCCldxZHSz5HF2lY4ZXIHS\\\\\\\/j1JUz3Wlo53jzOcKvz\\\\\\\/i35j83nP+mFCzsStxuIUiNgJTpqdRUMOpsXvKR1r2XBGw3x5OsIXHSu033Lh\\\\\\\/NCUkcEvKVvmemFEpUJqEaXrQiT8uHDwsNUvE+J6E8aQFBUrdWtk6YXxUDz8OiQyUJWj0gfXCh8GdgDZtVVnC5+hbuvn4dc1tw0U\\\\\\\/UOthdLspSUpWKdqCtB6Lkp9c12nts0L6DHve0Tn8HDzb+m2ys9Y\\\\\\\/NqxLSBNHvXjSe65EE1W63gSvVO5bJJhiQD6LGTxGOE1\\\\\\\/2i08B4SvTzM4YjgkPN3wIQp11L41dDucwBlD66VPbDeqw4SfX3xcxJVaJKBH5NawDVZ0\\\\\\\/2OcetTWA3k06nDsMfN7Noiar1RYdpaLHIFRa0DTcxrbglUvmItoMoKLe8BN\\\\\\\/PqAvPqsjazKg3BJWFD2D1TB2681KoPvgbCEORo+D9H\\\\\\\/5gMEoOO7EKBIrRbimRwbI7I9nl2tttj+xIZhkJP+JSSX0yhAzxYBrhWuTCTQ7gH2w7rnO2PZD+O60BLNqIlzQHwPQADKHUA8\\\\\\\/ObI4ZeQtzQAGwUtADFhRbqwXtqa7PekW+3M9tME5O4ge+XuSnCUGRXNVvMhGlw9XUQ4nl6SfLiJCygJ33baP3XVtqfXqtD9txh1FL4LRZBPMuc2+x2XWrx3iFRvfUJv34OJ+7N\\\\\\\/4r+To8meg3PDvXbOQUp4C1ZWR02hAWxt06+MQK7x9CqKuvDz7LOk7lyYVtInkTT2wvfFAolGWbVdFr8PS9X9aeGH5Nlg\\\\\\\/cRQV7WzgVV5OrY13t5jGJbw1lrT43Poefb1v+kXU14fVfmu5j7PatsW6FJgOMf\\\\\\\/oMZmC1DmMMT2B298nGc8TxPZwDLU7n4oGnKSrpT2mDjCyxDjJYS2d1NgUrYxle5APVK+hSXWrgqJgO1w4q+dLLMWzfx+Z\\\\\\\/FQkq9uTiGUchi1RyybFjeZQTUCzNPydk+yHwVIydfvLwW8OLJ\\\\\\\/uhF+x4ZmH43KJfsKr1Mr3veeXXpaduEIdvV2T36bF2XVYM4Dp3wU4vQ1iCjJflWxvZ1fo3BizMRUhs53CUVjmZMHLUhXvFeIgFOTAQjg73TtfSUTv2AatZyCIglJelSSY44iiXry\\\\\\\/ghj4azmxnl6R+17fityo0fnMyaeeWE+limteIGyEevD1kbRrnY4V3BHIlJHaaDYDD0PHTBMEeE240c95gKsPWhL3HCKHvMxK8FMqs8xiisJ2EwSctaxaqwQuehrnN9Qx4nQ7b5EjkkI7tYPc2qru9Lx8gDAfTTNq\\\\\\\/TcF\\\\\\\/VJer3qflH9+lGnwVNSYX\\\\\\\/59O0ueJnUmPCjAJC6LnK1ToT9D\\\\\\\/ngIcD6C1sGPt7E86CLPTxNZApuDXnkGPV\\\\\\\/DDEtT\\\\\\\/hbKdpaj\\\\\\\/45Mo6hEfniVI9EWIny++FP7mMAM62iQX10SLXS0Nux\\\\\\\/qftsDGdnqxQ8qSnvVFr7zg2KmpMtOgRWSS6QZjPH\\\\\\\/+iTWj2IqcldcPZuSzQzwb\\\\\\\/V6P7SB7xhCgrH78nlU6HRzQddbRmWM0G0XohNRWaaEIrpOL7D3rljTLu4r0lJqxzqXaCPB\\\\\\\/8BRzhOjCAa4\\\\\\\/zuumhLlOTF9x8zsMyzMNxNQlSsfMBHPCmxhxHSeKuIn058tbKcbx4MQ8IkelYT9l4mx8lqzS886rCTJgY++bPjYA10kIOO\\\\\\\/0nTzCvBw2MaBU7prWORNBD5E6rTKYaQPoROXE9D5hVlgcp8r\\\\\\\/0Uz5rLss59Sz6GpWSDougyUcUGoDghrWngGFqo623sDI8xgLO5PIWAN\\\\\\\/c5FbLebrq0rvD\\\\\\\/D0eyuIYDrcgGxaP5fg8ComOgS1bkrjLbudapGks0rVAGSK7FtPkIBkMk18UeisdbLGAboSb67WMOm8geHIZq5eCBaqHXeT3rBFux++r+kbeSvhq+EWpv7PHVaDaXDMIMNx5jHErWVW9UlqJpNkRzpDSC9A2t0bcfKMdWrRTNT4Tw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022k7JZsfPUapHqBfyCfmuI2Q==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022a5d5ea5bd134a2d756b55fcbc470070504780a331099ddd4c2c619cfbee69175\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u002260MwXmSlogwz5TGc2zGY2uzY25F\\\\\\\/nT8\\\\\\\/n8gKRHZ0nJlg+WR5GNn7lRL9ebrjAY5qIBDFD+CDnN1Zf4rFNVTyvxhbd2g9uDzhJHcIBLLR2HURjcFL3PLbrNTZvefiNCY4Dt+DXUfVZzlNVPD9O\\\\\\\/VlG1PEqFluc0a5nfJZ4qOuHQGory\\\\\\\/79L+DEGeUMo2wSNOcztERecHqK\\\\\\\/5pXK0F0E3qNlhNgJiwChcOlLkbqYuYFP\\\\\\\/2U3EYI5EkNOFULk9jwiNR8KW5GT1ybA8AkuHezlXMZhvYcYVg1KussKRW2FqYaLo3ckTtqKacGGhtbSkjY05E26plRTYSDtJ4E8bDNChtRZjMMe2bTEjLtsbywfTBwwvwPxNQbY7B0khuKh3yP7hZQLkimUy05hdMryjl46Yt4UZrQ+wk64bwGxQa3Weh9a1yxbKK1VA3ht4nu37zaO3VVWxvLv0KE7byF5QdjQ\\\\\\\/DeZjFFoX+36mw3FYp9ApLWtnXpawx\\\\\\\/gC1jmFI0w6JrPSkxAGycM4n45MR7Ib0oUxd4YAY0OkXm9q2y5WZp4jEGQ+2dyDsocFKBRMDAd2AOmx3\\\\\\\/TQN1RwVOOr1ENBb2To\\\\\\\/fLnf5eAtHXc6cTkP9T5ehSKgV0rD7P5un0mho+0LkPdNxwN2XTH4lXKBzB4u1riCJAbucJ6FTS3jr9vTtTDf63KKfybh+JFSsS8sYFMvgu\\\\\\\/3IEQIOj2GSvxLzinuZtcN5Ydju0Xyi0UJ88ylXvxSQZlEVzwskx7yHWvi7YieTbX8dRQg0KeP\\\\\\\/iDz\\\\\\\/Nw8XTVS1QgnBdlU98ubAuAD+bufIFDMOKDp8fwsO7mRTTJtZcQmLjo3d5J\\\\\\\/dcaF7ZJpKb+6VrqhnyI8s8GrEH2LgP\\\\\\\/bbbfQA83qMh7P4PdwFVaM\\\\\\\/GS7phQg6CXRhHYxu07Vi7hgwYzWVBP4UcGQvic+jKnrWMNOJAj0Rkp0tgqzrr1kcYpigTbb+2ALPhfaSBXXi7\\\\\\\/ptVjM4cmMaRZRcdb+pG\\\\\\\/9Nmq9eyxla+7omFofuo7PFITXQfVIAYLy1skFMvLhdPHP10du99eAcqhaxpQmvs9i93IaaZ7NQX8yPoWwo4yw1rtVf1uU2vGZWHbXSqmTDeIVteMD9\\\\\\\/ixjBM53VmI6uexlmhSRy9CVfcmXvDGo+l+2rrcN9C5DDpcX+KBzIkxX7E9U+sEhK3Fo4bQDsZq3Gq4LnW5g\\\\\\\/cA6iQtqRVzEycKhyhSUIaxz3Qqx828AsnFWZ3Jy25U3888R5f96KwVH96ecFEvCxGrapujmn8NAG3\\\\\\\/zhJNp4kP\\\\\\\/SInSnepgGi5tRnTg+J5zHoiGQ7lTL5ZxG9x+nT1hbnBjmIi85PEguZMwRoz749ifzENnTX8BPNWUg1Y4Jzi2xzFKgyCzxGa9MMp6oVVO1GUlU3Nyztyp4M2QSZs9lQOGY5vZZ9iSZvnehNmL59X5IrQtTUkP1pVskpQRjnPe6EBcov755aGrm+5L1+rL9i9IuIiArbvaUQllxfUWUXCmRUbX5unt+Pu+y8CY4fGrkByDd8X16DqQTNvDhx35N\\\\\\\/U+ddaswJCZb+xYYLjDiIrqaJQbhw4SAAUlN2jUhTowN5gkvN6yUk1Y9oWr5+vfpNfVJndjr8Sr23KeCvJ2Vf3cI6TPJM8bQq7btfP3c9ODZg5Owx35HWmbNVEXceZi4+QR2TtEG3mpAiN7vveWXZu8LrNj89OzGDg\\\\\\\/nPQbhmLMuquRcU\\\\\\\/jMEsObRrYlA2BFv+S3o04zaT7CiiIwguVKFt3kKlJ7Sb1kCbaYrFVtIM0nDlyd3drPpvxACeo+tZrTxb+g6cAPaiddSb1EGT3GZk4j4KAPRbotfa7EEP97swGXm5ZUcHNLosifsPqxs117dlfERFd+sNJNbUvQSDDF90EhFbYkAE\\\\\\\/4anPy7cEh+9sbMkDtKLGM1oAMlxlv2Uq3gbpLtKCpWww4oeQi7KLiJ7kI\\\\\\\/IRUij31uip7S+8C7FMPXH9f0GN8PfYtqqDUuX29V1td4gJyKkrCIeL0vS\\\\\\\/A==\\\u0022,\\\u0022iv\\\u0022:\\\u00228q7sNjo1w2TNAVP7b7gNWw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022d5d50e8b356d937941cd3cef4a1772e04e81c239919a6fcfb352b0b9575d66e1\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459324 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/2/9/1103" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-9t5DFW7-z6r46f4k13opwJD69JnYLzdFTGmLsjrHtd8" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/2/9/1103" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-FjvoWAHh2gm8GZUWiL70G6CDVe0pdHgJCUjq_9QtmNs" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/2/9/1095" title="An Assay to Detect In Vivo Y Chromosome Loss in Drosophila Wing Disc Cells" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/2/9/1113" title="Genetic Architecture of Migration-Related Traits in Rainbow and Steelhead Trout, Oncorhynchus mykiss" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459324" id="node459324" data-pisa="ggg;2/9/1103" data-pisa-master="ggg;2/9/1103" data-apath="/ggg/2/9/1103.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Mitochondrial Genome Evolution in a Single Protoploid Yeast Species</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Paul P. Jung</span>, <span class="highwire-citation-author" data-delta="1">Anne Friedrich</span>, <span class="highwire-citation-author" data-delta="2">Cyrielle Reisser</span>, <span class="highwire-citation-author" data-delta="3">Jing Hou</span> and <span class="highwire-citation-author" data-delta="4">Joseph Schacherer</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2012 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 2 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1103-1111;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.003152 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459324" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Paul P. Jung </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Genetics, Genomics and Microbiology, University of Strasbourg/Centre National de la Recherche Scientifique, Unité Mixte de Recherche 7156, Strasbourg, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Paul%2BP.%2BJung%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jung%20PP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APaul%2BP.%2BJung%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Anne Friedrich </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Genetics, Genomics and Microbiology, University of Strasbourg/Centre National de la Recherche Scientifique, Unité Mixte de Recherche 7156, Strasbourg, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anne%2BFriedrich%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Friedrich%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAnne%2BFriedrich%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Cyrielle Reisser </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Genetics, Genomics and Microbiology, University of Strasbourg/Centre National de la Recherche Scientifique, Unité Mixte de Recherche 7156, Strasbourg, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Cyrielle%2BReisser%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Reisser%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACyrielle%2BReisser%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Jing Hou </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Genetics, Genomics and Microbiology, University of Strasbourg/Centre National de la Recherche Scientifique, Unité Mixte de Recherche 7156, Strasbourg, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jing%2BHou%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hou%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJing%2BHou%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Joseph Schacherer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Genetics, Genomics and Microbiology, University of Strasbourg/Centre National de la Recherche Scientifique, Unité Mixte de Recherche 7156, Strasbourg, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joseph%2BSchacherer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Schacherer%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJoseph%2BSchacherer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:schacherer@unistra.fr" class="" data-icon-position="" data-hide-link-title="0">schacherer@unistra.fr</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/2/9/1103" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459324" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459324/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/2/9/1103.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459324" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459324/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/2/9/1103.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459324" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459324/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/2/9/1103.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459324" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459324/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/2/9/1103.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459324" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459324/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-4">Mitochondria are organelles, which play a key role in some essential functions, including respiration, metabolite biosynthesis, ion homeostasis, and apoptosis. The vast numbers of mitochondrial DNA (mtDNA) sequences of various yeast species, which have recently been published, have also helped to elucidate the structural diversity of these genomes. Although a large corpus of data are now available on the diversity of yeast species, little is known so far about the mtDNA diversity in single yeast species. To study the genetic variations occurring in the mtDNA of wild yeast isolates, we performed a genome-wide polymorphism survey on the mtDNA of 18 <em>Lachancea kluyveri</em> (formerly <em>Saccharomyces kluyveri</em>) strains. We determined the complete mt genome sequences of strains isolated from various geographical locations (in North America, Asia, and Europe) and ecological niches (<em>Drosophila</em>, tree exudates, soil). The mt genome of the NCYC 543 reference strain is 51,525 bp long. It contains the same core of genes as <em>Lachancea thermotolerans</em>, the nearest relative to <em>L. kluyveri</em>. To explore the mt genome variations in a single yeast species, we compared the mtDNAs of the 18 isolates. The phylogeny and population structure of <em>L. kluyveri</em> provide clear-cut evidence for the existence of well-defined geographically isolated lineages. Although these genomes are completely syntenic, their size and the intron content were found to vary among the isolates studied. These genomes are highly polymorphic, showing an average diversity of 28.5 SNPs/kb and 6.6 indels/kb. Analysis of the SNP and indel patterns showed the existence of a particularly high overall level of polymorphism in the intergenic regions. The dN/dS ratios obtained are consistent with purifying selection in all these genes, with the noteworthy exception of the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000007275">VAR1</a></em> gene, which gave a very high ratio. These data suggest that the intergenic regions have evolved very fast in yeast mitochondrial genomes.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/mitochondrial-dna" class="hw-term hw-article-keyword hw-article-keyword-mitochondrial-dna" rel="nofollow">mitochondrial DNA</a></li><li class="kwd"><a href="/keyword/intraspecific-diversity" class="hw-term hw-article-keyword hw-article-keyword-intraspecific-diversity" rel="nofollow">intraspecific diversity</a></li><li class="kwd"><a href="/keyword/population-structure" class="hw-term hw-article-keyword hw-article-keyword-population-structure" rel="nofollow">population structure</a></li><li class="kwd"><a href="/keyword/purifying-selection" class="hw-term hw-article-keyword hw-article-keyword-purifying-selection" rel="nofollow">purifying selection</a></li><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/lanchancea-kluyveri" class="hw-term hw-article-keyword hw-article-keyword-lanchancea-kluyveri" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Lanchancea kluyveri</em></a></li></ul><p id="p-5">The mitochondrion is a central organelle, which participates in key cellular functions such as respiration, metabolite biosynthesis, ion homeostasis, and apoptosis. These essential cellular processes are conserved in all eukaryotic organisms, from humans to yeasts. However, the mitochondrial DNA (mtDNA) has been found to show a greater rate of mutation than the nuclear DNA. The mtDNA mutation rate and spectrum have been determined by performing mutation-accumulation experiments on a wide range of model organisms, including <em>Caenorhabidtis elegans</em> (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Denver <em>et al.</em> 2000</a>), <em>Caenorhabidtis briggsae</em> (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Howe <em>et al.</em> 2010</a>), <em>Pristionchus pacificus</em> (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Molnar <em>et al.</em> 2011</a>), <em>Drosophila melanogaster</em> (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Haag-Liautard <em>et al.</em> 2008</a>), and <em>Saccharomyces cerevisiae</em> (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Lynch <em>et al.</em> 2008</a>). The mutation rates are high (around of 10<sup>−8</sup> to 10<sup>−7</sup> per site per generation) in all these species, but they vary from one species to another.</p><p id="p-6">Studies on yeasts, especially those belonging to the Hemiascomycetes phylum, have contributed considerably to our understanding of interspecific mitochondrial (mt) genome evolution. Complete mitochondrial genome sequences are now available on more than 40 hemiascomycetous species covering a broad evolutionary range (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Solieri 2010</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Gaillardin <em>et al.</em> 2011</a>). The availability of these genome sequences provided an opportunity for us to investigate the changes in the patterns of mtDNA organization and genome architecture from the evolutionary perspective. The size of mt genomes varies between yeast species, ranging, for example, from 11 to 85 kb in <em>Hanseniaspora uvarum</em> and <em>S. cerevisiae</em>, respectively (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Foury <em>et al.</em> 1998</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Pramateftaki <em>et al.</em> 2006</a>). Despite this high level of diversity, a core set of genes is conserved. This core contains the genes encoding the apocytochrome b (<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000007270">COB</a></em>), three subunits of ATP synthetase (<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000007268">ATP6</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000007267">ATP8</a></em>, and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000007274">ATP9</a></em>) and three subunits of cytochrome oxidase (<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000007260">COX1</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000007281">COX2</a></em>, and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000007283">COX3</a></em>). In addition, mtDNAs encode both large (<em>LSU</em>) and small (<em>SSU</em>) rRNA genes and a complete set of tRNA genes. However, some variability of the gene content is also observed among yeasts. Several genes, including those encoding <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000007275">VAR1</a></em> (a ribosomal protein), <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029023">RPM1</a></em> (the RNA subunit of the mitochondrial RNaseP), and the seven NADH:ubiquinone oxidoreductase (complex I) subunits do not occur consistently in all yeast mt genomes. The fact that the gene order varies significantly indicates that a large number of rearrangements has occurred with time (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bouchier <em>et al.</em> 2009</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Jung <em>et al.</em> 2009</a>, <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">2010</a>; <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Valach <em>et al.</em> 2011</a>). A few exceptions have been observed, however, where the synteny is conserved across species. The gene order is completely conserved between <em>Candida parapsilosis</em>, <em>Candida orthopsilosis</em>, and <em>Candida metapsilosis</em>, which are closely related species as well as between species of the <em>Yarrowia</em> clade, which are phylogenetically more distantly related species (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Kosa <em>et al.</em> 2006</a>; <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Gaillardin <em>et al.</em> 2011</a>).</p><p id="p-7">The many complete yeast mt genome sequences published by now have shown the existence of considerable variations and a wide range of genome architectures among species. Despite the existence of this ever-increasing body of data, relatively little is known so far about the mitochondrial genome variations existing within single yeast species. Due to the lack of intraspecific mitochondrial data, the evolution of mtDNA has not yet been completely elucidated. Because the differences between the mt genomes of various wild yeast isolates have not so far been documented, it was proposed here to investigate the patterns of mtDNA diversity present in a single yeast species by conducting a comprehensive survey on a set of <em>Lachancea kluyveri</em> (formerly known as <em>Saccharomyces kluyveri</em>) yeast isolates. <em>L. kluyveri</em> strains can be isolated from various sources such as soil, <em>Drosophila</em>, and tree exudates. Unlike <em>S. cerevisiae</em>, this species has not been domesticated (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Liti <em>et al.</em> 2009</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Schacherer <em>et al.</em> 2009</a>). In addition, <em>L. kluyveri</em> is a protoploid <em>Saccharomycetaceae</em>, which means that it diverged from the <em>S. cerevisiae</em> lineage before undergoing ancestral whole-genome duplication (WGD) and is therefore a pre-WGD yeast species (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Souciet <em>et al.</em> 2009</a>). Interestingly, in species belonging to the <em>Saccharomyces</em> genus that underwent WGD, the nuclear genes acting in the mitochondria show a greater rate of evolution than the other genes, which suggests that the mitochondrial functions have weakened in these post-WGD yeasts (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Jiang <em>et al.</em> 2008</a>). These findings can be correlated with the fact that most post-WGD species can live without a functional mt genome with their ability to generate petite mutants (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Merico <em>et al.</em> 2007</a>). Unlike the <em>Saccharomyces</em> species, <em>L. kluyveri</em> is petite-negative and mainly ferments sugar in the absence of oxygen (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Piskur <em>et al.</em> 1998</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Møller <em>et al.</em> 2001</a>). We therefore expected <em>L. kluyveri</em> to be a suitable model organism for studying intraspecific yeast mt genome evolution.</p><p id="p-8">Here, we present the complete mt genome sequences of 18 <em>L. kluyveri</em> strains, which were isolated from various geographical locations (in North America, Asia, and Europe) and ecological niches (<em>Drosophila</em>, tree exudates, and soil). We first assembled the mt genome of the NCYC 543 reference strain (also known as CBS 3082), the nuclear genome of which was recently sequenced (<a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">Souciet <em>et al.</em> 2009</a>). This genome is 51,525 bp long and contains the same set of 35 genes as those described in <em>Lachancea thermotolerans</em> (CBS 6340 strain), its closest relative (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Talla <em>et al.</em> 2005</a>). To provide a better picture of the genetic diversity among <em>L. kluyveri</em> species, we compared the mtDNA of these 18 isolates. This species is composed of two geographically defined lineages corresponding to the North American and European isolates. The genomes are syntenic, but the size of the mtDNA and the intron content differ. Whole-mtDNA genome analysis clearly showed the presence of a greater rate of SNPs and indels in the intergenic regions than in the coding regions. The dN/dS ratios obtained clearly suggested that purifying selection purged most nonsynonymous differences from the protein-coding genes. Only the dN/dS ratio obtained in the case of the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000007275">VAR1</a></em> gene was exceptionally high, possibly due to changes in the functional constraints to which this gene was exposed.</p><div class="section materials-methods" id="sec-1"><h2 class="">Material and Methods</h2><div id="sec-2" class="subsection"><h3>Strains and DNA preparation</h3><p id="p-9">Strains to be sequenced were selected so as to maximize the range of sources and the types of location from which they were isolated. The list of the 18 <em>L. kluyveri</em> strains used in this study, which were purchased from various yeast culture collections, is given in <a href="http://www.g3journal.orglookup/suppl/doi:10.1534/g3.112.003152/-/DC1/003152SI.pdf">supporting information</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003152/-/DC1/TableS1.pdf">Table S1</a>.</p><p id="p-10">Yeast cell cultures were grown overnight at 30° in 20 mL of YPD medium up to the early stationary phase. Total genomic DNAs were subsequently extracted using the QIAGEN Genomic-tip 100/G in line with the manufacturer’s instructions.</p></div><div id="sec-3" class="subsection"><h3>Sequencing and assembly</h3><p id="p-11">Genomic paired-end Illumina sequencing libraries were prepared with a mean insert size of 280 nt. The 18 libraries were multiplexed in 2 Illumina HiSeq2000 lanes for sequencing. Paired-end reads from 102 nt, six of which were dedicated to the multiplex tag, were obtained, amounting to 149,308,900 paired-end reads in all. FASTX-Toolkit (<a href="http://hannonlab.cshl.edu/fastx_toolkit/">http://hannonlab.cshl.edu/fastx_toolkit/</a>) was first used with options “-t 20 -l 50” to clean the reads. Two independent <em>de novo</em> assemblies (ace63 and ace75) were then constructed with each strain using SOAPdenovo, version 1.05 (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Li <em>et al.</em> 2010</a>), with two different Kmer sizes (soapdenovo63mer with option –K 63 and soapdenovo127mer with –K 75).</p></div><div id="sec-4" class="subsection"><h3>Mitochondrial reference sequence construction and annotation</h3><p id="p-12">The assemblies obtained with strain NCYC 543 were used to construct a mitochondrial reference sequence corresponding to the <em>L. kluyveri</em> species. We identified mitochondrial contigs and scaffolds within ace63 and ace75 assemblies by performing similarity searches with the BLAST suite of programs (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Altschul <em>et al.</em> 1997</a>) using <em>L. thermotolerans</em> mitochondrial gene sequences as the query (<a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">Talla <em>et al.</em> 2005</a>). A total number of 135 scaffolds and contigs were highlighted in these two independent assemblies, among which 98 were redundant. The 37 remaining sequences were compared with MUMmer 3.0 (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Kurtz <em>et al.</em> 2004</a>) to detect overlapping segments and the alignment of all these sequences was refined manually to obtain a single contig.</p><p id="p-13">The protein-coding genes as well as the large and small subunits of the ribosomal RNA genes were exactly located on the complete mitochondrial sequence by performing BLASTN searches with <em>L. thermotolerans</em> genes. The <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000007260">COX1</a></em> and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000007270">COB</a></em> intronic regions were refined manually based on the conservation of the protein sequences. The position of introns within the <em>LSU</em> genes was determined based on the boundaries defined for both <em>S. cerevisiae</em> and <em>L. thermotolerans</em> species (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Jacquier and Dujon 1983</a>).</p><p id="p-14">The same procedure was used to assemble and annotate the complete mitochondrial sequences of the 55-86.1, 77-1003, CBS 6547, and CBS 5828 strains for intraspecific comparison purposes. We obtained complete mitochondrial contigs for the 55-86.1 and 77-1003 strains. By contrast, there are one gap of 49 bp and two gaps of 30 and 40 bp in the scaffold of CBS 5828 and CBS 6547, respectively.</p><p id="p-15">Sequences are available under EMBL accession numbers <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=HE664110&amp;atom=%2Fggg%2F2%2F9%2F1103.atom">HE664110</a> for the NCYC 543 strain, <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=HE774680&amp;atom=%2Fggg%2F2%2F9%2F1103.atom">HE774680</a> for the 55-86.1 strain, HE664111 for the 77-1003 strain, HE774664 for the CBS 6547 strain, and HE664112 for the CBS 5828 strain.</p></div><div id="sec-5" class="subsection"><h3>Phylogenetic studies</h3><p id="p-16">The coding sequences of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000007268">ATP6</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000007267">ATP8</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000007274">ATP9</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000007270">COB</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000007260">COX1</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000007281">COX2</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000007283">COX3</a></em>, and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000007275">VAR1</a></em> genes were extracted from the initial ace63 assembly of the <em>L. kluyveri</em> strains by performing similarity searches with NCYC 543 gene sequences as the query. The sequences of all the genes were recovered in all strains except for the CBS 10368 strain, in which <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000007275">VAR1</a></em> could not be completed. The <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000007275">VAR1</a></em> gene was therefore removed from the phylogenetic analysis. The nucleotide sequences of these genes were automatically aligned with MUSCLE (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Edgar 2004</a>) and manually inspected before being concatenated.</p><p id="p-17">Based on these 5475 aligned positions, phylogenetic relationships among <em>L. kluyveri</em> strains were analyzed using neighbor-joining (using the Kimura two-parameter substitution model), maximum likelihood (using the Hasegawa-Kishino-Yano 85 substitution model), and Bayesian (10,000 generations, sampling trees every 10 generations, discarding the first 250 trees as a burn-in) methods. These analyses were performed with SeaView (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Gouy <em>et al.</em> 2010</a>), PhyML (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Guindon and Gascuel 2003</a>), and MrBayes (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Dereeper <em>et al.</em> 2008</a>), respectively. Bootstrap analyses (1000 replications) were used to assess the confidence level of each node for the Neighbor-joining method.</p></div><div id="sec-6" class="subsection"><h3>Selection and dN/dS ratios</h3><p id="p-18">The dN/dS ratios were calculated using CODEML model in PAML package version 4.4b (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Yang 2007</a>). We used a tree-based maximum-likelihood method. To summarize, coding sequence multi-alignments was generated for each of the eight protein-coding genes among the set of 18 <em>L. kluyveri</em>, as described previously. Based on the alignments, neighbor-joining trees were constructed using ClustalX and then labeled manually considering the branch-length and genetic proximity of the strains. Estimates of the dN/dS ratios were then calculated for each gene.</p></div><div id="sec-7" class="subsection"><h3>Population structure</h3><p id="p-19">The estimation of the number of population clusters was performed with the <em>Structure</em> program, version 2.3.1 (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Pritchard <em>et al.</em> 2000</a>), based on the 180 polymorphic positions detected in the seven protein-coding sequences previously used for the phylogenetic analyses. We ran the <em>Structure</em> program using the admixture model with the population number parameter <em>K</em> set from two to four, on 100,000 replicates after a burn-in of 100,000 iterations and did not incorporate any prior population information in these analyses.</p></div></div><div class="section results" id="sec-8"><h2 class="">Results</h2><p id="p-20">To investigate the intraspecific mtDNA variations present in yeast, we sequenced complete mitochondrial genomes from a set of 18 <em>L. kluyveri</em> isolates. These strains were isolated in various countries worldwide from ecological sources of various kinds (insect guts, tree exudates, and soil; <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003152/-/DC1/TableS1.pdf">Table S1</a>). The complete mt genome sequences were determined directly using high-throughput sequencing methods. Illumina libraries were constructed from genomic DNA isolated from the various strains and run on two lanes of a flow cell on an Illumina HiSeq 2000 using 102-bp paired-end sequencing methods. In this way, a coverage of 100% was obtained and the average read depth was ∼1145x.</p><div id="sec-9" class="subsection"><h3>Mitochondrial genome of <em>L. kluyveri</em></h3><p id="p-21">The nuclear genome of the reference NCYC 543 strain (also known as CBS 3082) was previously sequenced, but the mt genome was not sequenced (<a id="xref-ref-48-3" class="xref-bibr" href="#ref-48">Souciet <em>et al.</em> 2009</a>). This strain was isolated from the intestinal canal of <em>Drosophila pinicola</em> in the Yosemite National Park of California in the United States. We first assembled the mt genome. When we used SOAPdenovo, assembly of the reads yielded a single scaffold, which is 51,525 bp long and shows a GC content of about 15% (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003152/-/DC1/TableS2.pdf">Table S2</a>). This mitochondrial sequence is more than twice the size of that obtained in the closely related species <em>L. thermotolerans</em> [CBS 6340 strain (<a id="xref-ref-49-3" class="xref-bibr" href="#ref-49">Talla <em>et al.</em> 2005</a>)]. Despite this difference in size, the same set of 35 genes is encoded by the two genomes, both being transcribed from only one DNA strand (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). Among these 35 genes, eight are protein-coding genes and 27 are noncoding RNA genes. The protein-coding genes encode three subunits of the cytochrome c oxidase (<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000007260">COX1</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000007281">COX2</a></em>, and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000007283">COX3</a></em>), three subunits of the ATP synthase (<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000007268">ATP6</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000007267">ATP8</a></em>, and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000007274">ATP9</a></em>), the apocytochrome b (<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000007270">COB</a></em>) and a ribosomal protein (<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000007275">VAR1</a></em>). The noncoding RNA genes comprise two genes encoding the small and large RNA subunits of the ribosome (<em>SSU</em> and <em>LSU</em>, respectively), the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029023">RPM1</a></em> gene that codes for the RNA subunit of the RNaseP and 24 tRNA genes. The <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029023">RPM1</a></em> gene was detected using three conserved regions among ascomycete fungi, named CRI, CRIV, and CRV (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Seif <em>et al.</em> 2003</a>). This gene is located between tRNA<sup>Pro</sup> and tRNA<sup>Met</sup>: this location is the same as that previously found to occur in <em>S. cerevisiae</em> and <em>L. thermotolerans</em> (<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Foury <em>et al.</em> 1998</a>; <a id="xref-ref-49-4" class="xref-bibr" href="#ref-49">Talla <em>et al.</em> 2005</a>). The set of tRNA genes includes at least one tRNA for each of the 20 amino acids and sufficed to be able to decipher the fungal mitochondrial genetic code. This genetic code differs from standard code because mtDNA encodes a tRNA<sup>Trp</sup> able to decode UGA codon as it has been reported for several yeast species including <em>L. thermotolerans</em> (<a id="xref-ref-49-5" class="xref-bibr" href="#ref-49">Talla <em>et al.</em> 2005</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/9/1103/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Overview of the organization of the mtDNA of L. kluyveri. The mtDNA of L. thermotolerans is also shown so as to be able to compare the structure and the synteny. Protein-coding genes and tRNA and rRNA genes are presented in dark blue, red, and green, respectively. Coding introns are represented in light blue and noncoding introns in white. The RPM1 gene is presented in yellow." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-604681697" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Overview of the organization of the mtDNA of &lt;em&gt;L. kluyveri&lt;/em&gt;. The mtDNA of &lt;em&gt;L. thermotolerans&lt;/em&gt; is also shown so as to be able to compare the structure and the synteny. Protein-coding genes and tRNA and rRNA genes are presented in dark blue, red, and green, respectively. Coding introns are represented in light blue and noncoding introns in white. The &lt;em&gt;RPM1&lt;/em&gt; gene is presented in yellow.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/9/1103/F1.medium.gif" width="440" height="97"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1&#x2002;" src="https://www.g3journal.org/content/ggg/2/9/1103/F1.medium.gif" width="440" height="97"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/9/1103/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/9/1103/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461720" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1 </span> <p id="p-22" class="first-child">Overview of the organization of the mtDNA of <em>L. kluyveri</em>. The mtDNA of <em>L. thermotolerans</em> is also shown so as to be able to compare the structure and the synteny. Protein-coding genes and tRNA and rRNA genes are presented in dark blue, red, and green, respectively. Coding introns are represented in light blue and noncoding introns in white. The <em>RPM1</em> gene is presented in yellow.</p><div class="sb-div caption-clear"></div></div></div><p id="p-23">Complete analysis of gene conservation order between the mt genomes of <em>L. kluyveri</em> and <em>L. thermotolerans</em> showed that these two sequences are almost completely syntenic (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>). With the exception of the two tRNAs surrounding the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000007281">COX2</a></em> gene, these two genomes are colinear, which suggests that a very low rate of gross chromosomal rearrangements of mtDNA has occurred in the <em>Lachancea</em> clade (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>). Despite the high levels of synteny observed, comparisons between entire mt genomes of <em>L. kluyveri</em> and <em>L. thermotolerans</em> showed the existence of considerable genomic variability, especially in the noncoding sequences. This variability explains the size differences observed, for example. The twofold size difference is found to be correlated with the relative size of the intergenic regions, which ranges from 22.3% in <em>L. thermotolerans</em> to 41% in <em>L. kluyveri</em> (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003152/-/DC1/TableS2.pdf">Table S2</a>), as well as with the difference between the intron content of three genes: <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000007270">COB</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000007260">COX1</a></em>, and <em>LSU</em>. Unlike the <em>L. thermotolerans <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000007270">COB</a></em> gene, which is intronless, that of <em>L. kluyveri</em> harbors two introns. In addition, although the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000007260">COX1</a></em> genes of those two species both contain three introns, the third one is almost two times larger in <em>L. kluyveri</em> than in <em>L. thermotolerans</em> (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1</a>). All the introns present in the protein-coding genes of <em>L. kluyveri</em>, apart from the first intron in the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000007270">COB</a></em> gene, encode endonucleases belonging to the LAGLIDADG family of group I introns (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Haugen <em>et al.</em> 2005</a>). An intron was also detected in the <em>LSU</em> gene, with the same downstream boundary as that described in <em>S. cerevisiae</em> and an upstream boundary containing only one additional nucleotide (<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Jacquier and Dujon 1983</a>).</p></div><div id="sec-10" class="subsection"><h3>Intraspecific diversity in the mitochondrial coding regions</h3><p id="p-24">To explore the intraspecific variability, DNA sequence diversity in the coding region was first compared among 18 <em>L. kluyveri</em> isolates. A total number of 208 polymorphic positions (3.16% of the sites on the 6591 bp) showed a nucleotide substitution. The frequency of polymorphism is 0.0098 per bp on average. Among the 1101 SNPs identified in coding genes, 171 are nonsynonymous and 930 are synonymous (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003152/-/DC1/TableS3.pdf">Table S3</a>). This imbalance strongly suggests that purifying selection has purged the nonsynonymous differences from the protein-coding genes.</p><p id="p-25">To determine the effects of purifying selection, we then quantified the selective constraints present in the mt genomes by estimating the ratio of nonsynonymous (dN) to synonymous (dS) substitution rates ω = dN/dS. We calculated the average ω ratio in each of the coding genes (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). In all the cases studied, dN was lower than dS, signature of strong purifying selection of mitochondrial genes (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003152/-/DC1/FigureS1.pdf">Figure S1</a>). Median values of ω lower than 1 were obtained in all the genes, but the values differed from one gene to another (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>). The <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000007267">ATP8</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000007274">ATP9</a></em>, and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000007283">COX3</a></em> gene ratios were lower than average. In addition, the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000007268">ATP6</a></em> and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000007275">VAR1</a></em> genes were found to be characterized by a dN/dS ratio, which is well above average, giving a median value of 0.21 and 0.35, respectively. This finding might potentially be attributable either to positive selection or to a reduced constraint level on these genes. The results observed for <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000007275">VAR1</a></em> gene is interesting not only because of the elevated dN/dS ratio but also because we observed an exceptionally low dS value (the lowest in this set) and high dN value than average (the second highest in this set) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003152/-/DC1/FigureS1.pdf">Figure S1</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/9/1103/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Box-plot comparisons of the dN/dS ratio [&#x3C9;] estimated in the various mt genes and based on pairwise alignments." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-604681697" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Box-plot comparisons of the dN/dS ratio [&#x3C9;] estimated in the various mt genes and based on pairwise alignments.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/9/1103/F2.medium.gif" width="440" height="221"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2&#x2002;" src="https://www.g3journal.org/content/ggg/2/9/1103/F2.medium.gif" width="440" height="221"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/9/1103/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/9/1103/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461685" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2 </span> <p id="p-26" class="first-child">Box-plot comparisons of the dN/dS ratio [ω] estimated in the various mt genes and based on pairwise alignments.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-11" class="subsection"><h3>Intraspecific diversity of the intron content</h3><p id="p-27">The mitochondrial intron content is known to vary among closely related yeast species (<a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Jung <em>et al.</em> 2010</a>; <a id="xref-ref-14-3" class="xref-bibr" href="#ref-14">Gaillardin <em>et al.</em> 2011</a>). The results obtained here show that this is also the case between members of the same species (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003152/-/DC1/TableS2.pdf">Table S2</a>). All the introns present in protein-coding genes belong to the group I intron family and encode an endonuclease, with the exception of the first intron in the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000007270">COB</a></em> gene. In the various <em>L. kluyveri</em> mt genomes studied, it is worth noting that the intron content is highly variable in the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000007270">COB</a></em> and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000007260">COX1</a></em> genes (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003152/-/DC1/FigureS2.pdf">Figure S2</a>). The intron patterns seem to depend on the geographical origin of the strains. All the strains originating from North America contain the same two introns in the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000007270">COB</a></em> gene, whereas the number of introns ranges from two to four in the Eurasian strains (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003152/-/DC1/FigureS2.pdf">Figure S2</a>). The first intron detected in the North American strains (between amino acids 132 and 133) occurs in 9 of the 11 Eurasian strains, whereas the second one (between amino acids 168 and 169) occurs in all the latter strains. A supplementary intron was found to exist between amino acids 143 and 144 in five of the seven European strains and two of the four Asian strains. The Asian strains also contain a specific intron located between amino acids 237 and 238, which suggests that the pattern of evolution differed. The difference in the intron content is even more pronounced in the case of the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000007260">COX1</a></em> gene. The Eurasian isolates contain four to five introns, in different combinations depending on the strains. All the Asian strains contain a first intron composed of two LAGLIDADG motifs and lack the last intron occurring in the European strains, with the exception of CBS 10367. The intron pattern of the European strains seems to be well conserved in comparison with the Asian strains, which suggests that the latter strains may have evolved independently from European ones. More surprisingly, the North American strains (62-196, CBS 6545, and CBS 6547) contain an additional intron located between amino acids 300 and 301.</p><p id="p-28">All in all, these data clearly show that the intron content varied considerably between the isolates studied. This high rate of intraspecific intron variation suggests that the intron dynamics may contribute crucially to the structure of mitochondrial genomes.</p></div><div id="sec-12" class="subsection"><h3>Phylogeny of the <em>L. kluyveri</em> species</h3><p id="p-29">To shed light on the intraspecific mitochondrial evolution in <em>L. kluyveri</em> and compare the effects of geographical <em>vs.</em> ecological origins on strain diversity, we investigated the phylogenetic relationships among the strains surveyed. A phylogenic tree was built based on the concatenation of the aligned <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000007268">ATP6</a>-<a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000007267">ATP8</a>-<a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000007274">ATP9</a>-<a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000007260">COX1</a>-<a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000007281">COX2</a>-<a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000007283">COX3</a>-<a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000007270">COB</a></em> coding sequences. A standard neighbor-joining method was used to build a majority-rule consensus tree based on the 18 mt genomes under investigation (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). The phylogenetic tree clearly shows the existence of two tight clusters, which diverge from each other. We deduced that these two clusters consist of isolates of different geographical origins. The clades corresponding to these two clusters contain either the North American samples (clade 1) or the Eurasian samples (clade 2; <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>). High bootstrap values support the existence of the node between the two clusters. A similar topology was observed using two other methods (Bayesian and maximum likelihood methods) to build the tree (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003152/-/DC1/FigureS3.pdf">Figure S3</a>). This phylogeny strongly supports the hypothesis that the evolution of the <em>L. kluyveri</em> species depended on their geographical location. By contrast, the ecological environment does not seem to have been a key factor in the evolution of <em>L. kluyveri</em>, as several strains recovered from tree exudates were found to be present in both clades.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/9/1103/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Neighbor-joining tree of 18 L. kluyveri strains. The tree was constructed based on the concatenation of mt genes, corresponding to 5475 polymorphic positions. Numbers are bootstrap values based on 1000 replicates." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-604681697" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Neighbor-joining tree of 18 &lt;em&gt;L. kluyveri&lt;/em&gt; strains. The tree was constructed based on the concatenation of mt genes, corresponding to 5475 polymorphic positions. Numbers are bootstrap values based on 1000 replicates.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/9/1103/F3.medium.gif" width="440" height="410"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3&#x2002;" src="https://www.g3journal.org/content/ggg/2/9/1103/F3.medium.gif" width="440" height="410"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/9/1103/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/9/1103/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461734" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3 </span> <p id="p-30" class="first-child">Neighbor-joining tree of 18 <em>L. kluyveri</em> strains. The tree was constructed based on the concatenation of mt genes, corresponding to 5475 polymorphic positions. Numbers are bootstrap values based on 1000 replicates.</p><div class="sb-div caption-clear"></div></div></div><p id="p-31">In addition, single gene phylogenies based on the coding sequences of the two largest genes (<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000007270">COB</a></em> and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000007260">COX1</a></em>) as well as the evolution of the intron pattern suggest that clade 2 could be subdivided into subclades, the first of which corresponded to the Asian strains and the second to the European ones (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003152/-/DC1/FigureS4.pdf">Figure S4</a>). These differences probably emerged more recently than that observed between clades 1 and 2.</p></div><div id="sec-13" class="subsection"><h3>Population structure provides clear-cut evidence for geographical isolation</h3><p id="p-32">Phylogenetic trees do not show the true evolutionary history of strains, since they mask the effects of outcrossing among strains (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Ruderfer <em>et al.</em> 2006</a>). The population structure was therefore inferred here from the polymorphic sites across mitochondrial sequences. We used the model-based clustering algorithm implemented in the program <em>Structure</em> (<a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Pritchard <em>et al.</em> 2000</a>). This analysis largely confirmed the results of the phylogenetic analysis (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). The results obtained with this program were consistent with the existence of two main populations forming major geographical subgroups. The North American and Eurasian strains belong to two distinct well-defined populations (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>). Some SNPs represent private polymorphism within each population, resulting in a clear-cut separation between these two populations. The CBS 6547 strain is the only one that can be said to be a mixture of two populations when the K parameter (<em>i.e.</em> the number of predicted populations) was equal to two.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/9/1103/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Population structure of the 18 L. kluyveri strains. Cluster results from a structure analysis on 180 polymorphic sites. Each strain is represented by a single vertical bar, which is partitioned into K-colored segments that represent the strain&#x2019;s estimated ancestry proportion in each of the K clusters." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-604681697" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Population structure of the 18 &lt;em&gt;L. kluyveri&lt;/em&gt; strains. Cluster results from a structure analysis on 180 polymorphic sites. Each strain is represented by a single vertical bar, which is partitioned into K-colored segments that represent the strain&#x2019;s estimated ancestry proportion in each of the K clusters.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/9/1103/F4.medium.gif" width="390" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4&#x2002;" src="https://www.g3journal.org/content/ggg/2/9/1103/F4.medium.gif" width="390" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/9/1103/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/9/1103/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461683" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4 </span> <p id="p-33" class="first-child">Population structure of the 18 <em>L. kluyveri</em> strains. Cluster results from a structure analysis on 180 polymorphic sites. Each strain is represented by a single vertical bar, which is partitioned into K-colored segments that represent the strain’s estimated ancestry proportion in each of the K clusters.</p><div class="sb-div caption-clear"></div></div></div><p id="p-34">However, the increase in the K parameter shows that our collection contained two clean populations. One of them is composed of four North American strains (55-86.1, 77-1003, NCYC 543, and 62-1041), whereas the second one consists of four European strains (dd281a, CBS 2861, CBS 4568, and DBVPG 3108; <a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4</a>). The existence of these two populations is consistent with the phylogenetic distribution previously observed, as bootstrap values of 100 were associated with both ancestral nodes. By contrast, the other 10 strains are part of mixed-populations, as they seem to be mostly composed of a combination of 2 populations. The genomes of the other three North American strains (CBS 6545, 62-196, and CBS 6547) as well as those of all the Asian strains are mosaics showing a mixed architecture. The present structural analysis clearly shows that the Asian isolates studied here are not members of a clear population. These results show that the <em>L. kluyveri</em> species is structured and composed of two clean lineages, corresponding to the North American and European populations (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4</a>).</p></div><div id="sec-14" class="subsection"><h3>Whole mitochondrial genome comparisons</h3><p id="p-35">As suggested previously, the reason why the protein coding genes are well conserved is that purifying selection has eliminated many mutations. To investigate the mitochondrial genome-wide pattern of mitochondrial polymorphism, we compared the reference genome (NCYC 543) with four other mt genomes. Three of them belong to clade 1 (55-86.1, 77-1003, and CBS 6547) and one (CBS 5828) to clade 2 (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a>). Assembly of the reads yielded a single scaffold for each mt genome. As was to be expected, it turned out that exactly the same set of 35 genes is encoded by these mitochondrial genomes and that the synteny is perfectly conserved. The absence of rearrangements of the mtDNA in the <em>L. kluyveri</em> species is not surprising, in a view of the fact that practically no genomic rearrangements exist between <em>L. kluyveri</em> and <em>L. thermotolerans</em>.</p><p id="p-36">However, differences in size were observed between these mt sequences (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003152/-/DC1/TableS2.pdf">Table S2</a>): they range from 50,137 bp in the case of CBS 6547 to 53,726 bp in that of CBS 5828. This difference can be partly explained by the difference in the intron contents mentioned previously. The North American sequences were found to have the same intron content, whereas CBS 5828 harbors three additional introns. In addition, a decrease in the size of the intergenic region was observed between the distal strains, from 41% in the case of 77−103 to 28% in that of CBS 5828 (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003152/-/DC1/TableS2.pdf">Table S2</a>). This difference is mainly due to the variability of the region located between the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000007274">ATP9</a></em> and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000007281">COX2</a></em> genes, which decreases in size from almost 3 kb in the case of 77−1003 to 700 bp in that of CBS 5828.</p><p id="p-37">We also analyzed the genome-wide pattern of polymorphism (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). The great diversity observed in the intergenic regions contrasts with the existence of highly conserved coding regions. In the coding region, the SNP density ranges from 0 to 13.96 SNPs/kb in 55−86.1 and CBS 5828, respectively. By contrast, the SNP density is much greater in the intergenic regions, where it ranges from 0.31 to 82.7 SNPs/kb in the same genomes. We also detected a total number of 1196 insertion and deletion events (indels). Single-base indels are the most common size class, and about half of them are less than five bases long in size (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003152/-/DC1/FigureS5.pdf">Figure S5</a>). A few large deletions (up to 1695 bases in size) were detected in the CBS 6547 and CBS 5828 genomes. These strains are genetically more distantly related to the NCYC 543 reference strain than the others. Interestingly, a very large number of indels were detected in the intergenic regions. Among the 1196 indels observed, 1123 (93.9%) occur in intergenic regions, which account for only 57% of the genome on average, and only five occur in coding regions. These data suggest once again that most of the indels present in protein-coding genes were removed by selection. In addition, the density of the indels (0.19 indels/kb on average) is much lower than that of the SNPs (5.19 SNPs/kb on average) in the coding genes. This paucity of indels probably reflects the fact that most of the indels have resulted in frameshifts, and thus to a nonfunctional protein. By contrast, intergenic regions, which are mostly selectively neutral regions, show a very high density of SNPs (37.2 SNPs/kb on average) as well as indels (9.8 indels/kb on average). All in all, these data strongly suggest that most of the mutational events affecting the protein sequence have been purged by selection.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/461774/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/461774/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Polymorphic patterns in mitochondrial genomes of 4 <em>L. kluyveri</em> isolates</span><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-15"><h2 class="">Discussion</h2><p id="p-39">The availability of mt genomes from single isolates of various hemiascomycetous yeast species has improved our understanding of mtDNA evolution at different levels (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Eldarov <em>et al.</em> 2011</a>; <a id="xref-ref-24-3" class="xref-bibr" href="#ref-24">Jung <em>et al.</em> 2010</a>; <a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Kosa <em>et al.</em> 2006</a>; <a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Pramateftaki <em>et al.</em> 2006</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Prochazka <em>et al.</em> 2010</a>). Comparative genomic studies on these genomes have shown the existence of many differences in their structure, organization, and topology (<a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">Solieri 2010</a>). However, no comparisons have been carried out so far on single yeast species. Some whole-mtDNA genome studies recently shed interesting light on selection and evolution in humans (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Batini <em>et al.</em> 2011</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Elson <em>et al.</em> 2004</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Gunnarsdóttir <em>et al.</em> 2011</a>, <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Schönberg <em>et al.</em> 2011</a>). To investigate the genetic diversity and the patterns of mitochondrial genome evolution occurring in a single yeast species, we sequenced the complete mtDNA of 18 <em>L. kluyveri</em> isolates. In addition, the data we generated will help us to have a better understanding of phenotypic variation linked mtDNA diversity.</p><p id="p-40">One aim of this study was to assess the evolutionary history of the <em>L. kluyveri</em> species. Mitochondrial coding sequences of the 18 isolates were used to investigate the phylogeny by various methods. All the phylogenetic analyses clearly showed that the genomes sequenced comprised two tight clusters, corresponding to the North American and Eurasian isolates (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3</a>). To investigate the evolutionary history of these strains more closely, the population structure was also determined and the data obtained yielded some useful information about the populations in question (<a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4</a>). They also suggested that the genetic variations observed in <em>L. kluyveri</em> have a geographical basis. Evolutionary patterns based on geographical origin have previously been described, especially in the <em>S. paradoxus</em> species (<a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Liti <em>et al.</em> 2009</a>). Studies on the variability of this species clearly showed the existence of four populations originating from Europe, the Far East, America, and Hawaii (<a id="xref-ref-28-3" class="xref-bibr" href="#ref-28">Liti <em>et al.</em> 2009</a>). The evolution of <em>S. cerevisiae</em> strains was found on the contrary to be highly correlated with the ecological niches from which they originated rather than with their geographical origins (<a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Schacherer <em>et al.</em> 2009</a>). A clear-cut population structure was observed in the main ecological subgroups. These populations result from separate domestication events, but <em>S. cerevisiae</em> as a whole is not a domesticated species (<a id="xref-ref-28-4" class="xref-bibr" href="#ref-28">Liti <em>et al.</em> 2009</a>).</p><p id="p-41">All these data give an overall picture of the intraspecific variability of the yeast mt genome. In terms of their size and intron content, the yeast mt genomes of this species are highly variable (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003152/-/DC1/TableS2.pdf">Table S2</a>). The size ranged from 50.1 to 53.7 kb in CBS 6547 and CBS 5828, respectively. These differences may be attributable to either the intron content or the length of the intergenic regions. In fact, there exists a balance between the variations in these two regions. The size of intergenic regions varies between 15 to 21.2 kb, whereas the intron content varies both in number (between 6 and 9) and size (between 8.7 kb and 12.8 kb; <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003152/-/DC1/TableS2.pdf">Table S2</a>). The large genome size of the CBS 5828 isolate can be explained, for example, by the presence of nine introns, amounting to 12.8 kb. All the coding introns identified in the various yeast genomes studied here belong to the group I introns and encode endonucleases. It was established long time ago that group I introns are mobile elements, which can be gained or lost (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Delahodde <em>et al.</em> 1989</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Dujon <em>et al.</em> 1989</a>; <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Wenzlau <em>et al.</em> 1989</a>). The patterns of intron content observed in the wild isolates are interesting because they indicate that the mt genomes show a high level of intraspecific plasticity (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003152/-/DC1/FigureS2.pdf">Figure S2</a>). These patterns are correlated with the phylogeny of the strains and are therefore similar between strains originating from similar geographical locations. In addition to the difference in size and intron content, the gene content and order are conserved across the strains, as suggested by previous interspecific genome comparisons.</p><p id="p-42">Whole-mtDNA genome studies also provide a good means of addressing selection issues. These issues have been extensively studied in humans (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Batini <em>et al.</em> 2011</a>; <a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Gunnarsdóttir <em>et al.</em> 2011</a>; <a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Schönberg <em>et al.</em> 2011</a>). To investigate the patterns of polymorphism involved in mitochondrial genome evolution among <em>L. kluyveri</em> isolates, we quantified the selective constraints to which each of the mitochondrial genes was exposed by estimating ω. These ratios were found to be very low and consistent among the genes studied, with the noteworthy exception of the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000007275">VAR1</a></em> gene, which gave a very high ratio. The low dN/dS ratios of most of the other mitochondrial genes provide evidence for purifying selection, <em>i.e.</em>, most of the amino acid substitutions which occurred were deleterious and were therefore removed. Despite this purifying selection, we found definite evidence for relaxed purifying selection for two genes. First, the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000007268">ATP6</a></em> gene is characterized by a greater than average ω value (median value, 0.21). The <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000007268">ATP6</a></em> gene is one of three mitochondrial genes, the others being <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000007267">ATP8</a></em> and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000007274">ATP9</a></em>, which encode ATP synthase subunits. The most interesting point here, however, is the particularly high ω value obtained in the case of the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000007275">VAR1</a></em> gene (median value: 0.35). The <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000007275">VAR1</a></em> gene encodes a mitochondrial ribosomal protein of the small subunit. Interestingly, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000007275">VAR1</a></em> is not found in all the yeast mitochondrial genomes, and shows a scattered pattern of distribution among hemiascomycetous yeasts (<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Jung <em>et al.</em> 2009</a>). The most parsimonious explanation for the relaxation of the purifying selection should be a reduced functional constraint in the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000007275">VAR1</a></em> gene.</p><p id="p-43">Genome-wide patterns of SNPs and indels were also studied. Large disparities were noted between the distribution of the polymorphisms observed in the coding and intergenic regions (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). The mean density of nucleotide polymorphism was estimated at 5.2 SNPs/kb in the coding region and 37.2 SNPs/kb in the intergenic regions. This high figure is also consistent with the effects of purifying selection. This selection was also reflected in the high indel density in the intergenic regions (9.8 indels/kb on average), whereas they amounted to only 0.19 indels/kb on average in the coding regions. These data suggest once again that most of the indels in protein-coding genes were removed by selection. In addition, it is worth noting that most of the indels were found to be single-base or very small indels (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003152/-/DC1/FigureS5.pdf">Figure S5</a>). The fact that indels are mostly concentrated in homopolymers also suggests that sequence variations may have been caused by replication slippage events (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Behura 2007</a>, <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Noutsos <em>et al.</em> 2005</a>).</p><p id="p-44">The overall density of nucleotide polymorphism of the mt genomes studied here is very high, amounting 28.5 SNPs/kb on average, which is 10-fold greater than that previously observed in the nuclear genome of <em>S. cerevisiae</em> (2.8 SNPs/kb on average) (<a id="xref-ref-43-3" class="xref-bibr" href="#ref-43">Schacherer <em>et al.</em> 2009</a>). This finding seems to confirm that the mtDNA shows a higher mutation rate than the nuclear genome (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Clark-Walker 1991</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Lynch and Blanchard 1998</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Lynch <em>et al.</em> 2006</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Lynch 2007</a>). In <em>S. cerevisiae</em>, mutation-accumulation lines have been used to estimate the rate of nucleotide substitution in nuclear and mt genomes (<a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Lynch <em>et al.</em> 2008</a>). Nucleotide substitution rates have been estimated to be 37-fold greater in the mitochondria than in the nuclei. There are several explanations for this high mutation rate. First, mtDNA undergoes more replication and may accumulate more mutations than nuclear DNA explaining the hypermutability state of the organelle. In <em>S. cerevisiae</em>, the mtDNA genome is present normally at 20−100 copies per cell comprising 10–20% of the total cellular DNA (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Williamson and Fennell 1979</a>). Second, there is a low efficient of the DNA repair pathway (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Song <em>et al.</em> 2005</a>). Finally, the intracellular environment is more mutagenic (<a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Lynch 2007</a>).</p><p id="p-45">The intraspecific diversity of the yeast mt genome was described here in detail for the first time. As mentioned previously, the content and order of the genes have been completely conserved. By contrast, the size, intron content and the intergenic regions were found to vary from one isolates to another. The pattern of variation observed among the mitochondrial genes, with the exception of the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000007268">ATP6</a></em> and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000007275">VAR1</a></em> genes, is consistent with purifying selection. It would now be interesting to determine the rates and patterns of mtDNA in close relatives of this species such as <em>Lachancea cidri</em>, <em>Lachancea fermentati</em>, <em>Lachancea meyersii</em>, and <em>Lachancea waltii</em> (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Naumova <em>et al.</em> 2007</a>). Such a study could address the issues of selection and compare the patterns of mt sequence variation at two hierarchical levels: within species and among closely related species.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-46">We thank Michael Knop for kindly providing us with the dd281 <em>L. kluyveri</em> strain. We are most grateful to Vladimir Benes from the Gene Core Facilities EMBL (Heidelberg, Germany). We sincerely thank Bernard Dujon for helpful discussions and critical reading of the manuscript. This work was supported by an ANR grant (2010-BLAN-1606-05). C.R. is supported by a grant from CNRS and Région Alsace. J.S. is supported by an ANR grant (2011-JSV6-004-01).</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003152/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003152/-/DC1</a></p></li><li class="fn" id="fn-4"><p id="p-47"><em>Communicating editor: A. Rokas</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2012-05-22"><span class="received-label">Received </span>May 22, 2012.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2012-07-09"><span class="accepted-label">Accepted </span>July 9, 2012.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2012 Jung <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-3">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1103.1" data-doi="10.1093/nar/25.17.3389"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Altschul</span>  <span class="cit-name-given-names">S. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Madden</span>  <span class="cit-name-given-names">T. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schäffer</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Gapped BLAST and PSI-BLAST: a new generation of protein database search programs</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">3389</span>–<span class="cit-lpage">3402</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F25.17.3389%26rft_id%253Dinfo%253Apmid%252F9254694%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiIyNS8xNy8zMzg5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMi85LzExMDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1103.2" data-doi="10.1093/molbev/msq294"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Batini</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lopes</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Behar</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Calafell</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jorde</span>  <span class="cit-name-given-names">L. B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Insights into the demographic history of African Pygmies from complete mitochondrial genomes</span>. <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">1099</span>–<span class="cit-lpage">1110</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsq294%26rft_id%253Dinfo%253Apmid%252F21041797%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjk6IjI4LzIvMTA5OSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzIvOS8xMTAzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1103.3" data-doi="10.1093/molbev/msm068"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Behura</span>  <span class="cit-name-given-names">S. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Analysis of nuclear copies of mitochondrial sequences in honeybee (Apis mellifera) genome</span>. <abbr class="cit-jnl-abbrev">Mol. Biol. Evol</abbr>. <span class="cit-vol">24</span>: <span class="cit-fpage">1492</span>–<span class="cit-lpage">1505</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsm068%26rft_id%253Dinfo%253Apmid%252F17404397%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjk6IjI0LzcvMTQ5MiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzIvOS8xMTAzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1103.4" data-doi="10.1111/j.1567-1364.2009.00551.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bouchier</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Créno</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dujon</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fairhead</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Complete mitochondrial genome sequences of three <em>Nakaseomyces</em> species reveal invasion by palindromic GC clusters and considerable size expansion</span>. <abbr class="cit-jnl-abbrev">FEMS Yeast Res.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">1283</span>–<span class="cit-lpage">1292</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEMS%2BYeast%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1567-1364.2009.00551.x%26rft_id%253Dinfo%253Apmid%252F19758332%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZmVtc3lyIjtzOjU6InJlc2lkIjtzOjg6IjkvOC8xMjgzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMi85LzExMDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1103.5" data-doi="10.1007/BF00326232"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Clark-Walker</span>  <span class="cit-name-given-names">G. D.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">Contrasting mutation rates in mitochondrial and nuclear genes of yeasts <em>vs.</em> mammals</span>. <abbr class="cit-jnl-abbrev">Curr. Genet.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">195</span>–<span class="cit-lpage">198</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bgenetics%26rft.stitle%253DCurr%2BGenet%26rft.aulast%253DClark-Walker%26rft.auinit1%253DG.%2BD.%26rft.volume%253D20%26rft.issue%253D3%26rft.spage%253D195%26rft.epage%253D198%26rft.atitle%253DContrasting%2Bmutation%2Brates%2Bin%2Bmitochondrial%2Band%2Bnuclear%2Bgenes%2Bof%2Byeasts%2Bversus%2Bmammals.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00326232%26rft_id%253Dinfo%253Apmid%252F1657417%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00326232&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1657417&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F9%2F1103.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1103.6" data-doi="10.1016/0092-8674(89)90246-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Delahodde</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goguel</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Becam</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Creusot</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Perea</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">Site-specific DNA endonuclease and RNA maturase activities of two homologous intron-encoded proteins from yeast mitochondria</span>. <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">56</span>: <span class="cit-fpage">431</span>–<span class="cit-lpage">441</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DDelahodde%26rft.auinit1%253DA.%26rft.volume%253D56%26rft.issue%253D3%26rft.spage%253D431%26rft.epage%253D441%26rft.atitle%253DSite-specific%2BDNA%2Bendonuclease%2Band%2BRNA%2Bmaturase%2Bactivities%2Bof%2Btwo%2Bhomologous%2Bintron-encoded%2Bproteins%2Bfrom%2Byeast%2Bmitochondria.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0092-8674%252889%252990246-8%26rft_id%253Dinfo%253Apmid%252F2536593%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0092-8674(89)90246-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2536593&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F9%2F1103.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1989T189500013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1103.7" data-doi="10.1126/science.289.5488.2342"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Denver</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morris</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vassilieva</span>  <span class="cit-name-given-names">L. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thomas</span>  <span class="cit-name-given-names">W. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">High direct estimate of the mutation rate in the mitochondrial genome of <em>Caenorhabditis elegans</em></span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">289</span>: <span class="cit-fpage">2342</span>–<span class="cit-lpage">2344</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DDenver%26rft.auinit1%253DD.%2BR.%26rft.volume%253D289%26rft.issue%253D5488%26rft.spage%253D2342%26rft.epage%253D2344%26rft.atitle%253DHigh%2BDirect%2BEstimate%2Bof%2Bthe%2BMutation%2BRate%2Bin%2Bthe%2BMitochondrial%2BGenome%2Bof%2BCaenorhabditis%2Belegans%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.289.5488.2342%26rft_id%253Dinfo%253Apmid%252F11009418%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyODkvNTQ4OC8yMzQyIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMi85LzExMDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1103.8" data-doi="10.1093/nar/gkn180"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dereeper</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guignon</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blanc</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Audic</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buffet</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Phylogeny.fr: robust phylogenetic analysis for the non-specialist</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">465</span>–<span class="cit-lpage">469</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft.volume%253D36%26rft.spage%253D465%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkn180%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkn180&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1103.9" data-doi="10.1016/0378-1119(89)90035-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dujon</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Belfort</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Butow</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jacq</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lemieux</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">Mobile introns: definition of terms and recommended nomenclature</span>. <abbr class="cit-jnl-abbrev">Gene</abbr> <span class="cit-vol">82</span>: <span class="cit-fpage">115</span>–<span class="cit-lpage">118</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGene%26rft.volume%253D82%26rft.spage%253D115%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0378-1119%252889%252990035-8%26rft_id%253Dinfo%253Apmid%252F2555261%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0378-1119(89)90035-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2555261&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F9%2F1103.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1989AU55800011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1103.10" data-doi="10.1186/1471-2105-5-113"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Edgar</span>  <span class="cit-name-given-names">R. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">MUSCLE: a multiple sequence alignment method with reduced time and space complexity</span>. <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">113</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DEdgar%26rft.auinit1%253DR.%2BC.%26rft.volume%253D5%26rft.spage%253D113%26rft.epage%253D113%26rft.atitle%253DMUSCLE%253A%2Ba%2Bmultiple%2Bsequence%2Balignment%2Bmethod%2Bwith%2Breduced%2Btime%2Band%2Bspace%2Bcomplexity.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-5-113%26rft_id%253Dinfo%253Apmid%252F15318951%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-5-113&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15318951&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F9%2F1103.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1103.11" data-doi="10.1111/j.1567-1364.2011.00736.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eldarov</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mardanov</span>  <span class="cit-name-given-names">A. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beletsky</span>  <span class="cit-name-given-names">A. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ravin</span>  <span class="cit-name-given-names">N. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Skryabin</span>  <span class="cit-name-given-names">K. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Complete sequence and analysis of the mitochondrial genome of the methylotrophic yeast <em>Hansenula polymorpha</em> DL-1</span>. <abbr class="cit-jnl-abbrev">FEMS Yeast Res.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">464</span>–<span class="cit-lpage">472</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEMS%2BYeast%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1567-1364.2011.00736.x%26rft_id%253Dinfo%253Apmid%252F21545683%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZmVtc3lyIjtzOjU6InJlc2lkIjtzOjg6IjExLzYvNDY0IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMi85LzExMDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1103.12" data-doi="10.1086/381505"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Elson</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Turnbull</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Howell</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Comparative genomics and the evolution of human mitochondrial DNA: assessing the effects of selection</span>. <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">74</span>: <span class="cit-fpage">229</span>–<span class="cit-lpage">238</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DElson%26rft.auinit1%253DJ.%2BL.%26rft.volume%253D74%26rft.issue%253D2%26rft.spage%253D229%26rft.epage%253D238%26rft.atitle%253DComparative%2Bgenomics%2Band%2Bthe%2Bevolution%2Bof%2Bhuman%2Bmitochondrial%2BDNA%253A%2Bassessing%2Bthe%2Beffects%2Bof%2Bselection.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F381505%26rft_id%253Dinfo%253Apmid%252F14712420%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/381505&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14712420&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F9%2F1103.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000188860000004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1103.13" data-doi="10.1016/S0014-5793(98)01467-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Foury</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roganti</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lecrenier</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Purnelle</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">The complete sequence of the mitochondrial genome of <em>Saccharomyces cerevisiae</em></span>. <abbr class="cit-jnl-abbrev">FEBS Lett.</abbr> <span class="cit-vol">440</span>: <span class="cit-fpage">325</span>–<span class="cit-lpage">331</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEBS%2BLetters%26rft.stitle%253DFEBS%2BLetters%26rft.aulast%253DFoury%26rft.auinit1%253DF.%26rft.volume%253D440%26rft.issue%253D3%26rft.spage%253D325%26rft.epage%253D331%26rft.atitle%253DThe%2Bcomplete%2Bsequence%2Bof%2Bthe%2Bmitochondrial%2Bgenome%2Bof%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0014-5793%252898%252901467-7%26rft_id%253Dinfo%253Apmid%252F9872396%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0014-5793(98)01467-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9872396&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F9%2F1103.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000077595600016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1103.14"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gaillardin</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neuvéglise</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kerscher</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nicaud</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Mitochondrial genomes of yeasts of the <em>Yarrowia</em> clade</span>. <abbr class="cit-jnl-abbrev">FEMS Yeast Res.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">317</span>–<span class="cit-lpage">331</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEMS%2BYeast%2BRes.%26rft.volume%253D12%26rft.spage%253D317%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1103.15" data-doi="10.1093/molbev/msp259"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gouy</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guindon</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gascuel</span>  <span class="cit-name-given-names">O.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">SeaView version 4: a multiplatform graphical user interface for sequence alignment and phylogenetic tree building</span>. <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">221</span>–<span class="cit-lpage">224</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsp259%26rft_id%253Dinfo%253Apmid%252F19854763%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjg6IjI3LzIvMjIxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMi85LzExMDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1103.16" data-doi="10.1080/10635150390235520"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guindon</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gascuel</span>  <span class="cit-name-given-names">O.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">A simple, fast, and accurate algorithm to estimate large phylogenies by maximum likelihood</span>. <abbr class="cit-jnl-abbrev">Syst. Biol.</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">696</span>–<span class="cit-lpage">704</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSyst.%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1080%252F10635150390235520%26rft_id%253Dinfo%253Apmid%252F14530136%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Njoic3lzYmlvIjtzOjU6InJlc2lkIjtzOjg6IjUyLzUvNjk2IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMi85LzExMDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1103.17" data-doi="10.1101/gr.107615.110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gunnarsdóttir</span>  <span class="cit-name-given-names">E. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bauchet</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Finstermeier</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stoneking</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">High-throughput sequencing of complete human mtDNA genomes from the Philippines</span>. <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">11</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.107615.110%26rft_id%253Dinfo%253Apmid%252F21147912%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjY6IjIxLzEvMSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzIvOS8xMTAzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1103.18" data-doi="10.1371/journal.pbio.0060204"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Haag-Liautard</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coffey</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Houle</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Direct estimation of the mitochondrial DNA mutation rate in <em>Drosophila melanogaster</em></span>. <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e204</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DHaag-Liautard%26rft.auinit1%253DC.%26rft.volume%253D6%26rft.issue%253D8%26rft.spage%253De204%26rft.epage%253De204%26rft.atitle%253DDirect%2Bestimation%2Bof%2Bthe%2Bmitochondrial%2BDNA%2Bmutation%2Brate%2Bin%2BDrosophila%2Bmelanogaster.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.0060204%26rft_id%253Dinfo%253Apmid%252F18715119%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.0060204&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18715119&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F9%2F1103.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1103.19" data-doi="10.1016/j.tig.2004.12.007"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Haugen</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Simon</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bhattacharya</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">The natural history of group I introns</span>. <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">111</span>–<span class="cit-lpage">119</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DHaugen%26rft.auinit1%253DP.%26rft.volume%253D21%26rft.issue%253D2%26rft.spage%253D111%26rft.epage%253D119%26rft.atitle%253DThe%2Bnatural%2Bhistory%2Bof%2Bgroup%2BI%2Bintrons.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2004.12.007%26rft_id%253Dinfo%253Apmid%252F15661357%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tig.2004.12.007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15661357&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F9%2F1103.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000227082600009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1103.20" data-doi="10.1093/gbe/evp055"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Howe</span>  <span class="cit-name-given-names">D. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baer</span>  <span class="cit-name-given-names">C. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Denver</span>  <span class="cit-name-given-names">D. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">High rate of large deletions in Caenorhabditis briggsae mitochondrial genome mutation processes</span>. <abbr class="cit-jnl-abbrev">Genome Biol. Evol.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">29</span>–<span class="cit-lpage">38</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevp055%26rft_id%253Dinfo%253Apmid%252F20333220%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2JlIjtzOjU6InJlc2lkIjtzOjY6IjIvMC8yOSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzIvOS8xMTAzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1103.21" data-doi="10.1007/BF00392195"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jacquier</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dujon</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">1983</span> <span class="cit-article-title">The intron of the mitochondrial 21S rRNA gene: distribution in different yeast species and sequence comparison between <em>Kluyveromyces thermotolerans</em> and <em>Saccharomyces cerevisiae</em></span>. <abbr class="cit-jnl-abbrev">Mol. Gen. Genet.</abbr> <span class="cit-vol">192</span>: <span class="cit-fpage">487</span>–<span class="cit-lpage">499</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2B%2526%2Bgeneral%2Bgenetics%2B%253A%2B%2BMGG%26rft.stitle%253DMol%2BGen%2BGenet%26rft.aulast%253DJacquier%26rft.auinit1%253DA.%26rft.volume%253D192%26rft.issue%253D3%26rft.spage%253D487%26rft.epage%253D499%26rft.atitle%253DThe%2Bintron%2Bof%2Bthe%2Bmitochondrial%2B21S%2BrRNA%2Bgene%253A%2Bdistribution%2Bin%2Bdifferent%2Byeast%2Bspecies%2Band%2Bsequence%2Bcomparison%2Bbetween%2BKluyveromyces%2Bthermotolerans%2Band%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00392195%26rft_id%253Dinfo%253Apmid%252F6361491%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00392195&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=6361491&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F9%2F1103.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1983RV63300031&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1103.22" data-doi="10.1101/gr.074674.107"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guan</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pinney</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gu</span>  <span class="cit-name-given-names">Z.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Relaxation of yeast mitochondrial functions after whole-genome duplication</span>. <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">1466</span>–<span class="cit-lpage">1471</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.074674.107%26rft_id%253Dinfo%253Apmid%252F18669479%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjE4LzkvMTQ2NiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzIvOS8xMTAzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1103.23" data-doi="10.1111/j.1567-1364.2009.00540.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jung</span>  <span class="cit-name-given-names">P. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schacherer</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Souciet</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Potier</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wincker</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The complete mitochondrial genome of the yeast <em>Pichia sorbitophila</em></span>. <abbr class="cit-jnl-abbrev">FEMS Yeast Res.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">903</span>–<span class="cit-lpage">910</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEMS%2BYeast%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1567-1364.2009.00540.x%26rft_id%253Dinfo%253Apmid%252F19594828%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZmVtc3lyIjtzOjU6InJlc2lkIjtzOjc6IjkvNi85MDMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8yLzkvMTEwMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1103.24" data-doi="10.1007/s00294-010-0318-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jung</span>  <span class="cit-name-given-names">P. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Friedrich</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Souciet</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Louis</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Potier</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Complete mitochondrial genome sequence of the yeast <em>Pichia farinosa</em> and comparative analysis of closely related species</span>. <abbr class="cit-jnl-abbrev">Curr. Genet.</abbr> <span class="cit-vol">56</span>: <span class="cit-fpage">507</span>–<span class="cit-lpage">515</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bgenetics%26rft.stitle%253DCurr%2BGenet%26rft.aulast%253DJung%26rft.auinit1%253DP.%2BP.%26rft.volume%253D56%26rft.issue%253D6%26rft.spage%253D507%26rft.epage%253D515%26rft.atitle%253DComplete%2Bmitochondrial%2Bgenome%2Bsequence%2Bof%2Bthe%2Byeast%2BPichia%2Bfarinosa%2Band%2Bcomparative%2Banalysis%2Bof%2Bclosely%2Brelated%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00294-010-0318-y%26rft_id%253Dinfo%253Apmid%252F20830585%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00294-010-0318-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20830585&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F9%2F1103.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1103.25" data-doi="10.1093/nar/gkl327"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kosa</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valach</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tomaska</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wolfe</span>  <span class="cit-name-given-names">K. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nosek</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Complete DNA sequences of the mitochondrial genomes of the pathogenic yeasts <em>Candida orthopsilosis</em> and <em>Candida metapsilosis</em>: insight into the evolution of linear DNA genomes from mitochondrial telomere mutants</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">2472</span>–<span class="cit-lpage">2481</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DKosa%26rft.auinit1%253DP.%26rft.volume%253D34%26rft.issue%253D8%26rft.spage%253D2472%26rft.epage%253D2481%26rft.atitle%253DComplete%2BDNA%2Bsequences%2Bof%2Bthe%2Bmitochondrial%2Bgenomes%2Bof%2Bthe%2Bpathogenic%2Byeasts%2BCandida%2Borthopsilosis%2Band%2BCandida%2Bmetapsilosis%253A%2Binsight%2Binto%2Bthe%2Bevolution%2Bof%2Blinear%2BDNA%2Bgenomes%2Bfrom%2Bmitochondrial%2Btelomere%2Bmutants.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkl327%26rft_id%253Dinfo%253Apmid%252F16684995%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjk6IjM0LzgvMjQ3MiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzIvOS8xMTAzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1103.26" data-doi="10.1186/gb-2004-5-2-r12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kurtz</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Phillippy</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Delcher</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smoot</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shumway</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Versatile and open software for comparing large genomes</span>. <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">R12</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DKurtz%26rft.auinit1%253DS.%26rft.volume%253D5%26rft.issue%253D2%26rft.spage%253DR12%26rft.epage%253DR12%26rft.atitle%253DVersatile%2Band%2Bopen%2Bsoftware%2Bfor%2Bcomparing%2Blarge%2Bgenomes.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2004-5-2-r12%26rft_id%253Dinfo%253Apmid%252F14759262%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2004-5-2-r12&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14759262&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F9%2F1103.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1103.27" data-doi="10.1186/1479-7364-4-4-271"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bolund</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">State of the art de novo assembly of human genomes from massively parallel sequencing data</span>. <abbr class="cit-jnl-abbrev">Hum. Genomics</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">271</span>–<span class="cit-lpage">277</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHuman%2Bgenomics%26rft.stitle%253DHum%2BGenomics%26rft.aulast%253DLi%26rft.auinit1%253DY.%26rft.volume%253D4%26rft.issue%253D4%26rft.spage%253D271%26rft.epage%253D277%26rft.atitle%253DState%2Bof%2Bthe%2Bart%2Bde%2Bnovo%2Bassembly%2Bof%2Bhuman%2Bgenomes%2Bfrom%2Bmassively%2Bparallel%2Bsequencing%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1479-7364-4-4-271%26rft_id%253Dinfo%253Apmid%252F20511140%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1479-7364-4-4-271&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20511140&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F9%2F1103.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1103.28" data-doi="10.1038/nature07743"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liti</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carter</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moses</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Warringer</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parts</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Population genomics of domestic and wild yeasts</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">458</span>: <span class="cit-fpage">337</span>–<span class="cit-lpage">341</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DLiti%26rft.auinit1%253DG.%26rft.volume%253D458%26rft.issue%253D7236%26rft.spage%253D337%26rft.epage%253D341%26rft.atitle%253DPopulation%2Bgenomics%2Bof%2Bdomestic%2Band%2Bwild%2Byeasts.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature07743%26rft_id%253Dinfo%253Apmid%252F19212322%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature07743&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19212322&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F9%2F1103.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000264285600041&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-book" id="cit-2.9.1103.29"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-source">The Origins of Genome Architecture</span>, <span class="cit-publ-name">Sinauer Associates Inc</span>, <span class="cit-publ-loc">Sunderland, MA</span></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1103.30"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blanchard</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Deleterious mutation accumulation in organelle genomes</span>. <abbr class="cit-jnl-abbrev">Genetica</abbr> <span class="cit-vol">102/103</span>: <span class="cit-fpage">29</span>–<span class="cit-lpage">39</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetica%26rft.volume%253D102%252F103%26rft.spage%253D29%26rft.atitle%253DDELETERIOUS%2BMUTATION%2BACCUMULATION%2BIN%2BORGANELLE%2BGENOMES%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1103.31" data-doi="10.1126/science.1118884"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koskella</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schaack</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Mutation pressure and the evolution of organelle genome architecture</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">311</span>: <span class="cit-fpage">1727</span>–<span class="cit-lpage">1730</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DLynch%26rft.auinit1%253DM.%26rft.volume%253D311%26rft.issue%253D5768%26rft.spage%253D1727%26rft.epage%253D1730%26rft.atitle%253DMutation%2Bpressure%2Band%2Bthe%2Bevolution%2Bof%2Borganelle%2Bgenomic%2Barchitecture.%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1118884%26rft_id%253Dinfo%253Apmid%252F16556832%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMTEvNTc2OC8xNzI3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMi85LzExMDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1103.32" data-doi="10.1073/pnas.0803466105"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sung</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morris</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coffey</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Landry</span>  <span class="cit-name-given-names">C. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">A genome-wide view of the spectrum of spontaneous mutations in yeast</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">105</span>: <span class="cit-fpage">9272</span>–<span class="cit-lpage">9277</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0803466105%26rft_id%253Dinfo%253Apmid%252F18583475%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTA1LzI3LzkyNzIiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8yLzkvMTEwMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1103.33" data-doi="10.1111/j.1742-4658.2007.05645.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Merico</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sulo</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Piskur</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Compagno</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Fermentative lifestyle in yeasts belonging to the <em>Saccharomyces</em> complex</span>. <abbr class="cit-jnl-abbrev">FEBS J.</abbr> <span class="cit-vol">274</span>: <span class="cit-fpage">976</span>–<span class="cit-lpage">989</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEBS%2BJournal%26rft.stitle%253DFEBS%2BJ%26rft.aulast%253DMerico%26rft.auinit1%253DA.%26rft.volume%253D274%26rft.issue%253D4%26rft.spage%253D976%26rft.epage%253D989%26rft.atitle%253DFermentative%2Blifestyle%2Bin%2Byeasts%2Bbelonging%2Bto%2Bthe%2BSaccharomyces%2Bcomplex%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1742-4658.2007.05645.x%26rft_id%253Dinfo%253Apmid%252F17239085%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1742-4658.2007.05645.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17239085&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F9%2F1103.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1103.34" data-doi="10.1128/JB.183.8.2485-2489.2001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Møller</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Olsson</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Piskur</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Ability for anaerobic growth is not sufficient for development of the petite phenotype in <em>Saccharomyces kluyveri</em></span>. <abbr class="cit-jnl-abbrev">J. Bacteriol.</abbr> <span class="cit-vol">183</span>: <span class="cit-fpage">2485</span>–<span class="cit-lpage">2489</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBacteriology%26rft.stitle%253DJ.%2BBacteriol.%26rft.aulast%253DMoller%26rft.auinit1%253DK.%26rft.volume%253D183%26rft.issue%253D8%26rft.spage%253D2485%26rft.epage%253D2489%26rft.atitle%253DAbility%2Bfor%2BAnaerobic%2BGrowth%2BIs%2BNot%2BSufficient%2Bfor%2BDevelopment%2Bof%2Bthe%2BPetite%2BPhenotype%2Bin%2BSaccharomyces%2Bkluyveri%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJB.183.8.2485-2489.2001%26rft_id%253Dinfo%253Apmid%252F11274107%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6MTA6IjE4My84LzI0ODUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8yLzkvMTEwMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1103.35" data-doi="10.1093/molbev/msr057"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Molnar</span>  <span class="cit-name-given-names">R.I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bartelmes</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dinkelacker</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Witte</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sommer</span>  <span class="cit-name-given-names">R. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Mutation rates and intraspecific divergence of the mitochondrial genome of <em>Pristionchus pacificus</em></span>. <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">2317</span>–<span class="cit-lpage">2326</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsr057%26rft_id%253Dinfo%253Apmid%252F21368317%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjk6IjI4LzgvMjMxNyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzIvOS8xMTAzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1103.36" data-doi="10.1134/S0006297907120097"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Naumova</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Serpova</span>  <span class="cit-name-given-names">E. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Naumov</span>  <span class="cit-name-given-names">G. I.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Molecular systematics of <em>Lachancea</em> yeasts</span>. <abbr class="cit-jnl-abbrev">Biochemistry (Mosc.)</abbr> <span class="cit-vol">72</span>: <span class="cit-fpage">1356</span>–<span class="cit-lpage">1362</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochemistry.%2BBiokhimii%25EF%25B8%25A0a%25EF%25B8%25A1%26rft.stitle%253DBiochemistry%2B%2528Mosc%2529%26rft.aulast%253DNaumova%26rft.auinit1%253DE.%2BS.%26rft.volume%253D72%26rft.issue%253D12%26rft.spage%253D1356%26rft.epage%253D1362%26rft.atitle%253DMolecular%2Bsystematics%2Bof%2BLachancea%2Byeasts.%26rft_id%253Dinfo%253Adoi%252F10.1134%252FS0006297907120097%26rft_id%253Dinfo%253Apmid%252F18205619%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1134/S0006297907120097&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18205619&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F9%2F1103.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1103.37" data-doi="10.1101/gr.3788705"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Noutsos</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Richly</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leister</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Generation and evolutionary fate of insertions of organelle DNA in the nuclear genomes of flowering plants</span>. <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">616</span>–<span class="cit-lpage">628</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DNoutsos%26rft.auinit1%253DC.%26rft.volume%253D15%26rft.issue%253D5%26rft.spage%253D616%26rft.epage%253D628%26rft.atitle%253DGeneration%2Band%2Bevolutionary%2Bfate%2Bof%2Binsertions%2Bof%2Borganelle%2BDNA%2Bin%2Bthe%2Bnuclear%2Bgenomes%2Bof%2Bflowering%2Bplants%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.3788705%26rft_id%253Dinfo%253Apmid%252F15867426%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE1LzUvNjE2IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMi85LzExMDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1103.38"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Piskur</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smole</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Groth</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Petersen</span>  <span class="cit-name-given-names">R. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pedersen</span>  <span class="cit-name-given-names">M. B.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Structure and genetic stability of mitochondrial genomes vary among yeasts of the genus <em>Saccharomyces</em></span>. <abbr class="cit-jnl-abbrev">Int. J. Syst. Bacteriol.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">1015</span>–<span class="cit-lpage">1024</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInt.%2BJ.%2BSyst.%2BBacteriol.%26rft.volume%253D3%26rft.spage%253D1015%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1103.39" data-doi="10.1111/j.1567-1364.2005.00018.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pramateftaki</span>  <span class="cit-name-given-names">P. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kouvelis</span>  <span class="cit-name-given-names">V. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lanaridis</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Typas</span>  <span class="cit-name-given-names">M. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">The mitochondrial genome of the wine yeast <em>Hanseniaspora uvarum</em>: a unique genome organization among yeast/fungal counterparts</span>. <abbr class="cit-jnl-abbrev">FEMS Yeast Res.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">77</span>–<span class="cit-lpage">90</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEMS%2BYeast%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1567-1364.2005.00018.x%26rft_id%253Dinfo%253Apmid%252F16423073%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZmVtc3lyIjtzOjU6InJlc2lkIjtzOjY6IjYvMS83NyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzIvOS8xMTAzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1103.40"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pritchard</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Donnelly</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Inference of population structure using multilocus genotype data</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">155</span>: <span class="cit-fpage">945</span>–<span class="cit-lpage">959</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D155%26rft.spage%253D945%26rft_id%253Dinfo%253Apmid%252F10835412%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10835412&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F9%2F1103.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000087475100039&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1103.41" data-doi="10.1111/j.1567-1364.2010.00644.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Procházka</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poláková</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Piskur</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sulo</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Mitochondrial genome from the facultative anaerobe and petite-positive yeast <em>Dekkera bruxellensis</em> contains the NADH dehydrogenase subunit genes</span>. <abbr class="cit-jnl-abbrev">FEMS Yeast Res.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">545</span>–<span class="cit-lpage">557</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEMS%2BYeast%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1567-1364.2010.00644.x%26rft_id%253Dinfo%253Apmid%252F20528950%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZmVtc3lyIjtzOjU6InJlc2lkIjtzOjg6IjEwLzUvNTQ1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMi85LzExMDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1103.42" data-doi="10.1038/ng1859"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ruderfer</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pratt</span>  <span class="cit-name-given-names">S. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seidel</span>  <span class="cit-name-given-names">H. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kruglyak</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Population genomic analysis of outcrossing and recombination in yeast</span>. <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">1077</span>–<span class="cit-lpage">1081</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DRuderfer%26rft.auinit1%253DD.%2BM.%26rft.volume%253D38%26rft.issue%253D9%26rft.spage%253D1077%26rft.epage%253D1081%26rft.atitle%253DPopulation%2Bgenomic%2Banalysis%2Bof%2Boutcrossing%2Band%2Brecombination%2Bin%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1859%26rft_id%253Dinfo%253Apmid%252F16892060%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1859&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16892060&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F9%2F1103.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000240112100028&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1103.43" data-doi="10.1038/nature07670"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schacherer</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shapiro</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruderfer</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kruglyak</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Comprehensive polymorphism survey elucidates population structure of <em>Saccharomyces cerevisiae</em></span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">458</span>: <span class="cit-fpage">342</span>–<span class="cit-lpage">345</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DSchacherer%26rft.auinit1%253DJ.%26rft.volume%253D458%26rft.issue%253D7236%26rft.spage%253D342%26rft.epage%253D345%26rft.atitle%253DComprehensive%2Bpolymorphism%2Bsurvey%2Belucidates%2Bpopulation%2Bstructure%2Bof%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature07670%26rft_id%253Dinfo%253Apmid%252F19212320%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature07670&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19212320&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F9%2F1103.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000264285600042&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1103.44" data-doi="10.1038/ejhg.2011.62"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schönberg</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Theunert</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stoneking</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nasidze</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">High-throughput sequencing of complete human mtDNA genomes from the Caucasus and West Asia: high diversity and demographic inferences</span>. <abbr class="cit-jnl-abbrev">Eur. J. Hum. Genet.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">988</span>–<span class="cit-lpage">994</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEuropean%2Bjournal%2Bof%2Bhuman%2Bgenetics%2B%253A%2B%2BEJHG%26rft.stitle%253DEur%2BJ%2BHum%2BGenet%26rft.aulast%253DSchonberg%26rft.auinit1%253DA.%26rft.volume%253D19%26rft.issue%253D9%26rft.spage%253D988%26rft.epage%253D994%26rft.atitle%253DHigh-throughput%2Bsequencing%2Bof%2Bcomplete%2Bhuman%2BmtDNA%2Bgenomes%2Bfrom%2Bthe%2BCaucasus%2Band%2BWest%2BAsia%253A%2Bhigh%2Bdiversity%2Band%2Bdemographic%2Binferences.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fejhg.2011.62%26rft_id%253Dinfo%253Apmid%252F21487439%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ejhg.2011.62&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21487439&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F9%2F1103.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1103.45" data-doi="10.1261/rna.5880403"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Seif</span>  <span class="cit-name-given-names">E. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Forget</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">N. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lang</span>  <span class="cit-name-given-names">B. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Mitochondrial RNase P RNAs in ascomycete fungi: lineage-specific variations in RNA secondary structure</span>. <abbr class="cit-jnl-abbrev">RNA</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">1073</span>–<span class="cit-lpage">1083</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRNA%26rft.stitle%253DRNA%26rft.aulast%253DSEIF%26rft.auinit1%253DE.%2BR.%26rft.volume%253D9%26rft.issue%253D9%26rft.spage%253D1073%26rft.epage%253D1083%26rft.atitle%253DMitochondrial%2BRNase%2BP%2BRNAs%2Bin%2Bascomycete%2Bfungi%253A%2BLineage-specific%2Bvariations%2Bin%2BRNA%2Bsecondary%2Bstructure%26rft_id%253Dinfo%253Adoi%252F10.1261%252Frna.5880403%26rft_id%253Dinfo%253Apmid%252F12923256%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoicm5hIjtzOjU6InJlc2lkIjtzOjg6IjkvOS8xMDczIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMi85LzExMDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1103.46" data-doi="10.1016/j.tim.2010.08.001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Solieri</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Mitochondrial inheritance in budding yeasts: towards an integrated understanding</span>. <abbr class="cit-jnl-abbrev">Trends Microbiol.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">521</span>–<span class="cit-lpage">530</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bmicrobiology%26rft.stitle%253DTrends%2BMicrobiol%26rft.aulast%253DSolieri%26rft.auinit1%253DL.%26rft.volume%253D18%26rft.issue%253D11%26rft.spage%253D521%26rft.epage%253D530%26rft.atitle%253DMitochondrial%2Binheritance%2Bin%2Bbudding%2Byeasts%253A%2Btowards%2Ban%2Bintegrated%2Bunderstanding.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tim.2010.08.001%26rft_id%253Dinfo%253Apmid%252F20832322%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tim.2010.08.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20832322&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F9%2F1103.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000284493600006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1103.47" data-doi="10.1073/pnas.0500253102"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pursell</span>  <span class="cit-name-given-names">Z. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Copeland</span>  <span class="cit-name-given-names">W. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Longley</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kunkel</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">DNA precursor asymmetries in mammalian tissue mitochondria and possible contribution to mutagenesis through reduced replication fidelity</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">102</span>: <span class="cit-fpage">4990</span>–<span class="cit-lpage">4995</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0500253102%26rft_id%253Dinfo%253Apmid%252F15784738%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTAyLzE0LzQ5OTAiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8yLzkvMTEwMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1103.48" data-doi="10.1101/gr.091546.109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Souciet</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dujon</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gaillardin</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnston</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baret</span>  <span class="cit-name-given-names">P. V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Comparative genomics of protoploid <em>Saccharomycetaceae</em></span>. <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">1696</span>–<span class="cit-lpage">1709</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.091546.109%26rft_id%253Dinfo%253Apmid%252F19525356%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIxOS8xMC8xNjk2IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMi85LzExMDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1103.49" data-doi="10.1016/j.febslet.2004.10.106"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Talla</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anthouard</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bouchier</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Frangeul</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dujon</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">The complete mitochondrial genome of the yeast <em>Kluyveromyces thermotolerans</em></span>. <abbr class="cit-jnl-abbrev">FEBS Lett.</abbr> <span class="cit-vol">579</span>: <span class="cit-fpage">30</span>–<span class="cit-lpage">40</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEBS%2BLetters%26rft.stitle%253DFEBS%2BLetters%26rft.aulast%253DTalla%26rft.auinit1%253DE.%26rft.volume%253D579%26rft.issue%253D1%26rft.spage%253D30%26rft.epage%253D40%26rft.atitle%253DThe%2Bcomplete%2Bmitochondrial%2Bgenome%2Bof%2Bthe%2Byeast%2BKluyveromyces%2Bthermotolerans.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.febslet.2004.10.106%26rft_id%253Dinfo%253Apmid%252F15620687%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.febslet.2004.10.106&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15620687&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F9%2F1103.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000226170100006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1103.50" data-doi="10.1093/nar/gkq1345"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Valach</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Farkas</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fricova</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kovac</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brejova</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Evolution of linear chromosomes and multipartite genomes in yeast mitochondria</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">4202</span>–<span class="cit-lpage">4219</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkq1345%26rft_id%253Dinfo%253Apmid%252F21266473%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiIzOS8xMC80MjAyIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMi85LzExMDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1103.51" data-doi="10.1016/0092-8674(89)90245-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wenzlau</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Saldanha</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Butow</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Perlman</span>  <span class="cit-name-given-names">P. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">A latent intron-encoded maturase is also an endonuclease needed for intron mobility</span>. <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">56</span>: <span class="cit-fpage">421</span>–<span class="cit-lpage">430</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DWenzlau%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D56%26rft.issue%253D3%26rft.spage%253D421%26rft.epage%253D430%26rft.atitle%253DA%2Blatent%2Bintron-encoded%2Bmaturase%2Bis%2Balso%2Ban%2Bendonuclease%2Bneeded%2Bfor%2Bintron%2Bmobility.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0092-8674%252889%252990245-6%26rft_id%253Dinfo%253Apmid%252F2536592%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0092-8674(89)90245-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2536592&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F9%2F1103.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1989T189500012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1103.52" data-doi="10.1016/0076-6879(79)56065-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Williamson</span>  <span class="cit-name-given-names">D. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fennell</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1979</span> <span class="cit-article-title">Visualization of yeast mitochondrial DNA with the fluorescent stain “DAPI”</span>. <abbr class="cit-jnl-abbrev">Methods Enzymol.</abbr> <span class="cit-vol">56</span>: <span class="cit-fpage">728</span>–<span class="cit-lpage">733</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2Bin%2Benzymology%26rft.stitle%253DMethods%2BEnzymol%26rft.aulast%253DWilliamson%26rft.auinit1%253DD.%2BH.%26rft.volume%253D56%26rft.spage%253D728%26rft.epage%253D733%26rft.atitle%253DVisualization%2Bof%2Byeast%2Bmitochondrial%2BDNA%2Bwith%2Bthe%2Bfluorescent%2Bstain%2B%2526quot%253BDAPI%2526quot%253B.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0076-6879%252879%252956065-0%26rft_id%253Dinfo%253Apmid%252F459889%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0076-6879(79)56065-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=459889&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F9%2F1103.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1103.53" data-doi="10.1093/molbev/msm088"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">Z.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">PAML 4: phylogenetic analysis by maximum likelihood</span>. <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">1586</span>–<span class="cit-lpage">1591</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsm088%26rft_id%253Dinfo%253Apmid%252F17483113%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjk6IjI0LzgvMTU4NiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzIvOS8xMTAzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/2/9/1103.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/2/9/1095" title="An Assay to Detect In Vivo Y Chromosome Loss in Drosophila Wing Disc Cells" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/2/9/1113" title="Genetic Architecture of Migration-Related Traits in Rainbow and Steelhead Trout, Oncorhynchus mykiss" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/2/9"><span>Volume 2 Issue 9, September 2012</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/2/9.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/2/9.cover-source.jpg?itok=JMb6jwWa" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 2 (9)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/2/9/1103" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-ZASaE5v24JD3rq7WDOV2Yf3NO1E6t2z61cEnkxIJ6dM" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/2/9/1103" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/2/9/1103" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/2/9/1103" class="active" data-icon-position="" data-hide-link-title="0">Mitochondrial Genome Evolution in a Single Protoploid Yeast Species</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459324" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-8m0u0TeDWgM9Gb9jqFi5JRYMY7aOyefVjhTfDPhG8_o" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/2/9/1103.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/2/9/1103" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-eYSVEhSc-zwFnTFg5-M7Y0gkio40l5v-LcZVfTHnwlk" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/2/9/1103" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/2/9/1103.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459324" id="node459324--2" data-pisa="ggg;2/9/1103" data-pisa-master="ggg;2/9/1103" data-apath="/ggg/2/9/1103.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Mitochondrial Genome Evolution in a Single Protoploid Yeast Species</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Paul P. Jung</span>, <span class="highwire-citation-author" data-delta="1">Anne Friedrich</span>, <span class="highwire-citation-author" data-delta="2">Cyrielle Reisser</span>, <span class="highwire-citation-author" data-delta="3">Jing Hou</span> and <span class="highwire-citation-author" data-delta="4">Joseph Schacherer</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2012 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 2 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1103-1111;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.003152 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459324--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Paul P. Jung </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Genetics, Genomics and Microbiology, University of Strasbourg/Centre National de la Recherche Scientifique, Unité Mixte de Recherche 7156, Strasbourg, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Paul%2BP.%2BJung%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jung%20PP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APaul%2BP.%2BJung%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Anne Friedrich </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Genetics, Genomics and Microbiology, University of Strasbourg/Centre National de la Recherche Scientifique, Unité Mixte de Recherche 7156, Strasbourg, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anne%2BFriedrich%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Friedrich%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAnne%2BFriedrich%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Cyrielle Reisser </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Genetics, Genomics and Microbiology, University of Strasbourg/Centre National de la Recherche Scientifique, Unité Mixte de Recherche 7156, Strasbourg, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Cyrielle%2BReisser%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Reisser%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACyrielle%2BReisser%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Jing Hou </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Genetics, Genomics and Microbiology, University of Strasbourg/Centre National de la Recherche Scientifique, Unité Mixte de Recherche 7156, Strasbourg, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jing%2BHou%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hou%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJing%2BHou%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Joseph Schacherer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Genetics, Genomics and Microbiology, University of Strasbourg/Centre National de la Recherche Scientifique, Unité Mixte de Recherche 7156, Strasbourg, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joseph%2BSchacherer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Schacherer%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJoseph%2BSchacherer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:schacherer@unistra.fr" class="" data-icon-position="" data-hide-link-title="0">schacherer@unistra.fr</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/2/9/1103" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/2/9/1103&amp;title=Mitochondrial%20Genome%20Evolution%20in%20a%20Single%20Protoploid%20Yeast%20Species" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/2/9/1103&amp;title=Mitochondrial%20Genome%20Evolution%20in%20a%20Single%20Protoploid%20Yeast%20Species" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/2/9/1103&amp;title=Mitochondrial%20Genome%20Evolution%20in%20a%20Single%20Protoploid%20Yeast%20Species" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/2/9/1103&amp;text=Mitochondrial%20Genome%20Evolution%20in%20a%20Single%20Protoploid%20Yeast%20Species" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/2/9/1103&amp;title=Mitochondrial%20Genome%20Evolution%20in%20a%20Single%20Protoploid%20Yeast%20Species" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/2/9/1103&amp;t=Mitochondrial%20Genome%20Evolution%20in%20a%20Single%20Protoploid%20Yeast%20Species" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/2/9/1103&amp;title=Mitochondrial%20Genome%20Evolution%20in%20a%20Single%20Protoploid%20Yeast%20Species" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/2/9/1103&amp;title=Mitochondrial%20Genome%20Evolution%20in%20a%20Single%20Protoploid%20Yeast%20Species" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459324/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459324" id="node459324--3" data-pisa="ggg;2/9/1103" data-pisa-master="ggg;2/9/1103" data-apath="/ggg/2/9/1103.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Mitochondrial Genome Evolution in a Single Protoploid Yeast Species</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Paul P. Jung</span>, <span class="highwire-citation-author" data-delta="1">Anne Friedrich</span>, <span class="highwire-citation-author" data-delta="2">Cyrielle Reisser</span>, <span class="highwire-citation-author" data-delta="3">Jing Hou</span> and <span class="highwire-citation-author" data-delta="4">Joseph Schacherer</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2012 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 2 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1103-1111;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.003152 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459324--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Paul P. Jung </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Genetics, Genomics and Microbiology, University of Strasbourg/Centre National de la Recherche Scientifique, Unité Mixte de Recherche 7156, Strasbourg, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Paul%2BP.%2BJung%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jung%20PP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APaul%2BP.%2BJung%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Anne Friedrich </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Genetics, Genomics and Microbiology, University of Strasbourg/Centre National de la Recherche Scientifique, Unité Mixte de Recherche 7156, Strasbourg, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anne%2BFriedrich%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Friedrich%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAnne%2BFriedrich%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Cyrielle Reisser </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Genetics, Genomics and Microbiology, University of Strasbourg/Centre National de la Recherche Scientifique, Unité Mixte de Recherche 7156, Strasbourg, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Cyrielle%2BReisser%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Reisser%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACyrielle%2BReisser%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Jing Hou </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Genetics, Genomics and Microbiology, University of Strasbourg/Centre National de la Recherche Scientifique, Unité Mixte de Recherche 7156, Strasbourg, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jing%2BHou%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hou%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJing%2BHou%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Joseph Schacherer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Genetics, Genomics and Microbiology, University of Strasbourg/Centre National de la Recherche Scientifique, Unité Mixte de Recherche 7156, Strasbourg, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joseph%2BSchacherer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Schacherer%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJoseph%2BSchacherer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:schacherer@unistra.fr" class="" data-icon-position="" data-hide-link-title="0">schacherer@unistra.fr</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1103&amp;rft.epage=1111&amp;rft.atitle=Mitochondrial%20Genome%20Evolution%20in%20a%20Single%20Protoploid%20Yeast%20Species&amp;rft.volume=2&amp;rft.issue=9&amp;rft.date=2012-09-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Jung%2C+Paul+P.&amp;rft.au=Friedrich%2C+Anne&amp;rft.au=Reisser%2C+Cyrielle&amp;rft.au=Hou%2C+Jing&amp;rft.au=Schacherer%2C+Joseph"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459324/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459324/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459324/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459324/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459324/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459324/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459324/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459324/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459324/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459324/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459324/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459324/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/2/9/1103" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Material and Methods</a></li><li class="odd"><a href="#sec-8" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-15" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/2/9/1103.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/2/9/1103.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/2/9/1103.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/2/9/1103.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
