<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/2/9/1035" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/2/9/1035.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/2/9/1035.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/2/9/1035.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/2/9.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/2/9/1035/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/2/9/1035.atom" />
<meta name="HW.pisa" content="ggg;2/9/1035" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Location-Dependent Empirical Thresholds for Quantitative Trait Mapping" />
<meta name="DC.Identifier" content="10.1534/g3.112.003517" />
<meta name="DC.Date" content="2012-09-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2012 LaCombe et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="The Churchill-Doerge approach toward constructing empirical thresholds has received widespread use in the genetic mapping literature through the past 16 years. The method is valued for both its simplicity and its ability to preserve the genome-wide error rate at a prespecified level. However, the Churchill-Doerge method is not designed to maintain the local (comparison-wise) error rate at a constant level except in situations that are unlikely to occur in practice. In this article, we introduce the objective of preserving the local error rate at a constant level in the context of mapping quantitative trait loci in linkage populations. We derive a method that preserves the local error rate at a constant level, provide an application via simulation on a Hordeum vulgare population, and demonstrate evidence of the relationship between recombination and location bias. Furthermore, we indicate that this method is equivalent to the Churchill-Doerge method when several assumptions are satisfied." />
<meta name="DC.Contributor" content="Jason LaCombe" />
<meta name="DC.Contributor" content="Benjamin McClosky" />
<meta name="DC.Contributor" content="Steven Tanksley" />
<meta name="article:published_time" content="2012-09-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Location-Dependent Empirical Thresholds for Quantitative Trait Mapping" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;The Churchill-Doerge approach toward constructing empirical thresholds has received widespread use in the genetic mapping literature through the past 16 years. The method is valued for both its simplicity and its ability to preserve the genome-wide error rate at a prespecified level. However, the Churchill-Doerge method is not designed to maintain the local (comparison-wise) error rate at a constant level except in situations that are unlikely to occur in practice. In this article, we introduce the objective of preserving the local error rate at a constant level in the context of mapping quantitative trait loci in linkage populations. We derive a method that preserves the local error rate at a constant level, provide an application via simulation on a &lt;i&gt;Hordeum vulgare&lt;/i&gt; population, and demonstrate evidence of the relationship between recombination and location bias. Furthermore, we indicate that this method is equivalent to the Churchill-Doerge method when several assumptions are satisfied.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2012/09/01" />
<meta name="citation_mjid" content="ggg;2/9/1035" />
<meta name="citation_id" content="2/9/1035" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/2/9/1035" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/2/9/1035.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/2/9/1035.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/2/9/1035.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.112.003517" />
<meta name="citation_pmid" content="22973540" />
<meta name="citation_volume" content="2" />
<meta name="citation_issue" content="9" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="1035" />
<meta name="citation_lastpage" content="1039" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Jason LaCombe" />
<meta name="citation_author_institution" content="Nature Source Genetics, Ithaca, New York 14850" />
<meta name="citation_author_email" content="jlacombe@naturesourcegenetics.com" />
<meta name="citation_author" content="Benjamin McClosky" />
<meta name="citation_author_institution" content="Nature Source Genetics, Ithaca, New York 14850" />
<meta name="citation_author" content="Steven Tanksley" />
<meta name="citation_author_institution" content="Nature Source Genetics, Ithaca, New York 14850" />
<meta name="citation_reference" content="citation_title=QTL Cartographer;citation_author=C. J. Basten;citation_author=B. S. Weir;citation_author=Z.-B. Zeng;citation_year=2002" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_journal_abbrev=Bioinformatics;citation_author=K. Broman;citation_author=H. Wu;citation_author=S. Sen;citation_author=G. Churchill;citation_title=R/qtl: QTL mapping in experimental crosses;citation_pages=889-890;citation_volume=19;citation_year=2003;citation_issue=7;citation_pmid=12724300;citation_doi=10.1093/bioinformatics/btg112" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=G. Churchill;citation_author=R. Doerge;citation_title=Empirical threshold values for quantitative trait mapping.;citation_pages=963-971;citation_volume=138;citation_year=1994;citation_issue=3;citation_pmid=7851788" />
<meta name="citation_reference" content="citation_title=Multiple Testing Procedures with Applications to Genomics;citation_author=S. Dudoit;citation_author=M. J. van der Laan;citation_year=2007" />
<meta name="citation_reference" content="citation_author=D. S. Falconer;citation_author=T. F. C. Mackay;citation_year=1996" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_journal_abbrev=Heredity;citation_author=K. C. Falke;citation_author=M. Frisch;citation_title=Power and false-positive rate in QTL detection with near-isogenic line libraries.;citation_pages=576-584;citation_volume=106;citation_year=2011;citation_issue=4;citation_pmid=20683482;citation_doi=10.1038/hdy.2010.87" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=P. M. Hayes;citation_author=B. H. Liu;citation_author=S. J. Knapp;citation_author=F. Chen;citation_author=B. Jones;citation_title=Quantitative trait locus effects and environmental interaction in a sample of North American barley germplasm;citation_pages=392-401;citation_volume=87;citation_year=1993;citation_issue=3;citation_pmid=24190268;citation_doi=10.1007/BF01184929" />
<meta name="citation_reference" content="citation_journal_title=TAG Theoretical and Applied Genetics;citation_author=A. Kleinhofs;citation_author=A. Kilian;citation_author=M. Saghai Maroof;citation_author=R. M. Biyashev;citation_author=P. Hayes;citation_title=A molecular, isozyme and morphological map of the barley (Hordeum vulgare) genome;citation_pages=705-712;citation_volume=86;citation_year=1993;citation_issue=6" />
<meta name="citation_reference" content="citation_author=E. L. Lehmann;citation_author=J. P. Romano;citation_year=2005" />
<meta name="citation_reference" content="citation_year=1998" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=T. C. Lystig;citation_title=Adjusted P Values for Genome-Wide Scans;citation_pages=1683-1687;citation_volume=164;citation_year=2003;citation_issue=4;citation_pmid=12930772" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. Manichaikul;citation_author=A. Palmer;citation_author=Ś. Sen;citation_author=K. Broman;citation_title=Significance thresholds for quantitative trait locus mapping under selective genotyping;citation_pages=1963-1966;citation_volume=177;citation_year=2007;citation_issue=3;citation_pmid=17720902;citation_doi=10.1534/genetics.107.080093" />
<meta name="citation_reference" content="citation_author=D. E. Mather;citation_year=1995" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_journal_abbrev=Heredity;citation_author=B. U. Müller;citation_author=B. Stich;citation_author=H.-P. Piepho;citation_title=A general method for controlling the genome-wide type I error rate in linkage and association mapping experiments in plants.;citation_pages=825-831;citation_volume=106;citation_year=2011;citation_issue=5;citation_pmid=20959861;citation_doi=10.1038/hdy.2010.125" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=J. Peirce;citation_author=K. Broman;citation_author=L. Lu;citation_author=R. Williams;citation_title=A simple method for combining genetic mapping data from multiple crosses and experimental designs.;citation_pages=e1036-e1036;citation_volume=2;citation_year=2007;citation_issue=10;citation_pmid=17940600;citation_doi=10.1371/journal.pone.0001036" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=J. Peirce;citation_author=K. Broman;citation_author=L. Lu;citation_author=E. Chesler;citation_author=G. Zhou;citation_title=Genome Reshuffling for Advanced Intercross Permutation (GRAIP): simulation and permutation for advanced intercross population analysis.;citation_pages=e1977-e1977;citation_volume=3;citation_year=2008;citation_issue=4;citation_pmid=18431467;citation_doi=10.1371/journal.pone.0001977" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=S. R. Seaman;citation_author=B. Muller-Myhsok;citation_title=Rapid simulation of P values for product methods and multiple-testing adjustment in association studies.;citation_pages=399-408;citation_volume=76;citation_year=2005;citation_issue=3;citation_pmid=15645388;citation_doi=10.1086/428140" />
<meta name="citation_reference" content="citation_journal_title=Stat. Appl. Genet. Mol. Biol.;citation_title=Using alpha wisely: improving power to detect multiple QTL;citation_author=K. Simonsen;citation_author=L. McIntyre;citation_volume=3;citation_year=2004;citation_issue=1" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=J. D. Storey;citation_author=R. Tibshirani;citation_title=Statistical significance for genomewide studies;citation_volume=100;citation_year=2003;citation_issue=16;citation_pmid=12883005;citation_doi=10.1073/pnas.1530509100" />
<meta name="citation_reference" content="citation_title=MapQTL;citation_author=J. W. Van Ooijen;citation_year=2009" />
<meta name="citation_reference" content="citation_title=GenStat for Windows;citation_author=VSNInternational;citation_year=2011" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_journal_abbrev=Heredity;citation_author=W. H. Wei;citation_author=S. Knott;citation_author=C. S. Haley;citation_author=D. J. De Koning;citation_title=Controlling false positives in the mapping of epistatic QTL.;citation_pages=401-409;citation_volume=104;citation_year=2010;citation_issue=4;citation_pmid=19789566;citation_doi=10.1038/hdy.2009.129" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Location-Dependent Empirical Thresholds for Quantitative Trait Mapping" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/2/9.cover-source.jpg" />
<meta name="twitter:description" content="The Churchill-Doerge approach toward constructing empirical thresholds has received widespread use in the genetic mapping literature through the past 16 years. The method is valued for both its simplicity and its ability to preserve the genome-wide error rate at a prespecified level. However, the Churchill-Doerge method is not designed to maintain the local (comparison-wise) error rate at a constant level except in situations that are unlikely to occur in practice. In this article, we introduce the objective of preserving the local error rate at a constant level in the context of mapping quantitative trait loci in linkage populations. We derive a method that preserves the local error rate at a constant level, provide an application via simulation on a Hordeum vulgare population, and demonstrate evidence of the relationship between recombination and location bias. Furthermore, we indicate that this method is equivalent to the Churchill-Doerge method when several assumptions are satisfied." />
<meta name="og-title" property="og:title" content="Location-Dependent Empirical Thresholds for Quantitative Trait Mapping" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/2/9/1035" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="The Churchill-Doerge approach toward constructing empirical thresholds has received widespread use in the genetic mapping literature through the past 16 years. The method is valued for both its simplicity and its ability to preserve the genome-wide error rate at a prespecified level. However, the Churchill-Doerge method is not designed to maintain the local (comparison-wise) error rate at a constant level except in situations that are unlikely to occur in practice. In this article, we introduce the objective of preserving the local error rate at a constant level in the context of mapping quantitative trait loci in linkage populations. We derive a method that preserves the local error rate at a constant level, provide an application via simulation on a Hordeum vulgare population, and demonstrate evidence of the relationship between recombination and location bias. Furthermore, we indicate that this method is equivalent to the Churchill-Doerge method when several assumptions are satisfied." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/2/9.cover-source.jpg" />
    <title>Location-Dependent Empirical Thresholds for Quantitative Trait Mapping | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ZtPI6yltzkTaloU_9lwKK0o-wSRaDrvI72aIpAYv01U__yCSGhNVuCbGf6KLL54AUS64uP6kQfk1f_X5fJJp6-FY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__36Yiqb2U0AhDmqVjDsvW59h3D9k7d0px_jM-NJ9exZg__uqIT76aUscmJwriA5WgUcByCR2EI-ARNdlG2HGF7Oec__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings, {"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"uRY1WSJIUseoGcL4ksUUiOl9ZzPD_t9FBk28GQabDKo","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459316","apath":"\/ggg\/2\/9\/1035.atom","pisa":"ggg;2\/9\/1035","ac":{"\/ggg\/2\/9\/1035.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/2\/9\/1035.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;2\/9\/1035"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;2\/9\/1035"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/2\/9\/1035":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022R7HBBssF3WVJ0N4LugviT8X1E\\\\\\\/Y2wf\\\\\\\/u6mbeZZD8Nh+ZoWvn0t0blbFktYF+ePg5I+0DQv6jZ\\\\\\\/\\\\\\\/0\\\\\\\/d01gt77zZnmO\\\\\\\/PP\\\\\\\/Dc50\\\\\\\/I4SQOxNq5CB6\\\\\\\/N6qyBOZ2pgJsd0wyCQ6r8GqfTZKqYJhjFwrGMBdplElyhh+YW27J1zORiU1vUk3NQa\\\\\\\/HsqsHr0Z0XP+nk7\\\\\\\/16wTokCmtrg5V1p1BWjodeBdZZ3mXI\\\\\\\/c3MJU73p1oXE\\\\\\\/9AmO3wn6AJpWs4gsZnpXAPREUNdIdKW00iGDl9oHYXDEgQ5CmcUCoKV0u9GP2OF0GNqXsoBG2IUdxxwCoz1cY8UpgILdEiSsMNddzmQUa05gf+Tln5YzOXiIaoUk7kmZNn3lB7MQ4n2nfHPPZuM2YQgXD5BCJCi\\\\\\\/ja570NOQqGRU8DlABhzcOeLz3SizCNvzVZ0O7QDVzCzTXZSiM3Uc\\\\\\\/14ws75sIcBRWDigpg275MsS9k8dARzm2NxYKZfkc\\\\\\\/Jp61m6VGuK6LZlDxnPY5J7jXRc9aJV+v3mckY2OpVNPULGKmv5eRw3GShFT22u2hIxbEe4Z7EWYJWP+ELMeaI6D+Ao6V42je+y0CPpFrnqoNp2W5vDVT7+s9W72qcK5cMTNfJk6anWZ7WG4jfZpgsmJp5ODaDTfj+VS8miao6lRE6veZmu4VHmQFkKJJ2PM2ZjgvFRJVxQ1bMwdpVGYax5Oy1vc7RHnn8Y5PDkM15jYzNAnY7hWy4wEan\\\\\\\/YUt1hSMJTORJMzAWib20VlrjYITKuI31HS7fdHBk0nw\\\\\\\/Ws1KsBXduM8jcXb\\\\\\\/qS80lRZKXSbNowTgMPhBmgkuEZXNJVvhaFFalisn4Huc588NmWxAOffTixncJmHP8ygmTneU9RUrJTe5a+vvpOfmWV46C8HInF9dSKDe3HkopzC0CH\\\\\\\/\\\\\\\/Qd3TN4okOc8wxJy4le8zhd0dPJDtpTCvOEZNELbAoq3Qs+4I4JW0C5L6bZ+xfQBMEdRXP2eGvO1KNC9rsjkQmEFJ+dufHRUgAedKpN4qV6N1PmKSsgzhkw5XEwG8XgLp2ehP9K0Q7y+2LOjebxX8HaPlyYhRSjJbzxN07YIH535F0Nq9GDH0ONlzLh62W9qFFtoFpRwatcWiE49GxSfTKnkgO5dnVLbZjYF+m1GamSR\\\\\\\/LVrC1F4eNsGjndAnS\\\\\\\/dJAopIEDLz8wbGm3sAyE3d\\\\\\\/9ek+n0Qdh5mEctXC04PeqzSPuKoVW17kZcxAlOHOCAQ4t8VG97MUPwes6gTyPcqpXe4ehY\\\\\\\/WRUMNqVptVmU\\\\\\\/h\\\u0022,\\\u0022iv\\\u0022:\\\u0022WdvFihB\\\\\\\/fbVpglv2A2WE3Q==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002205efdb3c8cd83791a51a2df91839a7642fb296a32d50740268c79405e746c84a\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022v7\\\\\\\/BiBE+voUHkp115\\\\\\\/KA9APPll3wDkSaWUGn16yQDowJ\\\\\\\/YYSRLBPCLAMqguh2tIvgZ11HbXTnkp3Eqa0MX6YrrZ759NP7x4TjcKxP6RehexS7i4HEZwaq760vlXzf8jrZfx7n\\\\\\\/KFLaAuCCoWfNSpn9BGcA9F7MF4ZyW+s+h4+5i+4Aonttuvmqt2HT5cXAAIUIG+tVAnHDZ5E8+ZpwUL5x3Cku49Ff9BCtAWe+mzaeOP6PqxG6UylqjgMiC5fqTMgbP6KkSMkQNtL2TLBD5dtNlIIR2Yq\\\\\\\/bS5JDqKFdFgHFFcIKN5X458Bm2KZHFciJ9X4J6qJESOpkihoY7bE69EtlRuvXNMbn4lbNQWhF6Pm9b1DOzYre\\\\\\\/93sEszRuaAZGxyuWN75UGMwEGkcR3cETfNpSTojtgtef3rJNHGNKG+0lDh1n84szcdgDfTQcwJ7sI+3ejvE9Lw7pDlBSOIZgjVo+5IIlGhAUJXaoZyGGldesKPfMJ0XdYpcZ+aE7SK+wBElj1Qqtfr68gxiwUJpabhcCrIJ5R4sCzSPWVIW65f9HenJtvm6F\\\\\\\/qBBKTVj6spZvtiNDFHTp99rXGYH0vqndSbM5IHtqoWgp4RApjHqD7z0X3ssb1NdwIqSYxFbGaM1zG8mAUwYTf4mGUxgyqq6pq6HWcFxDsxMyizAaKSjVfjfE3DIPAnbRbfZPDWW4SZy9uAmgavHCx8wSJD6v6BwCQy4wKFL4MDzh70dJVKaxHZ4hE3kb0nk4r3863dU3QP9kdVAV\\\\\\\/sxpI2jWXWcge3xt+\\\\\\\/fAtleOhU6MxKJTQQKw6VRIBoMWjqnEEBo0qlDP5YCoSqDw4QukzuLJRlVei0YRESX1bZpJ\\\\\\\/rZAmqPXcL8rP2ITvt7bdT\\\\\\\/+fIP9tJc4g2+nyqLh31ED5YZfhIrin9Kh\\\\\\\/I28BzU\\\\\\\/AwncKMRIAKg08lQ4hQTadsLuksd2u4iNeLp6SImACnw2pgPX0mRMby2dFXtwia3qRTSA3qONjW\\\\\\\/YKMoBqLHQ7+cXyi3PqlUCVvkQuDz3Y8Da4VVLHmr3ErWFAxM7Igcu2AlYH62\\\\\\\/PHGedmxmWbYBg6vM8FrfzuXrcslj54IqZT2cUVGXIfZ3OOeIQ5B3qrjZ7rjGaK0+XvGdCrPK7YuiTZ3LelvjTK8kzR5bGzoCzyqd0m0XtwrnFxcbtryN1Dd5OxcyW4yvUOPIIlafewIS2UxYe2gPYC9GKLcU2CAqQuXfNv\\\\\\\/YErUxS3n3SLjF2Gs1JVI4l0ftsxxUiKlKPwfAtsWhusF8aXVek+6kpLiv8gDYBo9fs2I+QdH8cm2jJuMt7JruL35GEjRsDLvKQBEs8rgGKoalMJPmeGH\\\\\\\/59NwkEmMr7PXxfGpY0Fi5\\\\\\\/w+ks\\\\\\\/9NWDHwKijIZy7gbzr2EZSkVXktWXN9g\\\\\\\/mPrS1aSIkkpSePcCw5VAwzc4G9rTRVl\\\\\\\/ZoYtPuopSoqSYLfGtleS9hxMYCJGd636nAOXOyf7QRRwZNtQoUeZK+0mHTzOxHoDmpK+CO2D\\\\\\\/Lv1l9G+8irsXaxd5DycoJ1ajkyTUYaeDvJzcvjOEjuEPBvRB451Y5wy03yaS4WiiQ\\\\\\\/sa70Nq4Ub2KygRGZUOlkWo1\\\\\\\/goSHgxZIUHHRFECC8l98CVfT7F+pL1ZHqere2v95MQBgG9eGITE1LKi6BOUlfHk\\\\\\\/rsNpT5h6O1mU+o4duDOmetQiHzTTpH6TjE519hr7FLez+sywlLui2Z8MufjrWwL7HcRCzJZjU\\\\\\\/NY62CWgpg+yv8br3+5F+gAuj666YbEA5+GpYLzgswtwPR1Uf1Y9QNh2QoHAK0B79+f\\\\\\\/laIprjM48vC2MbR8JcPzkD9GdyqFDil6kykaT74RlMJFuVT+IxsB2io2bKZdpxkGxwWB6mhPLSl0Y8AQs8qANstrOze1tqV+F3dolIu2ILOO5YnYrFduHgNUd1NOuVf6+yAy2NGd1\\\\\\\/u2MmYpfVUGyYZCmeAYRv5QUhj66DFdu2mZItoWXIrtlqfoHh61vONaTXdN7DXshFXula8wejmk+i9qaA3zrF6ssXfr6jTe2tlsOx4226DoDbJtrXC5RS5GQWLPHfYJm5Figu0GVz\\\\\\\/EEmqA4kcr7nnwGplJGZ9NiaaOEPixonvSZicezOUP8H++TY8gTzvcYEBOAVqoU0fqUzlwxLEyr7GQy452B+AHBwxYtA78keJ63RGSim9Lz3bs6mGjsZu2+H+Ka9Yw2av+Odakl+o8xTObH\\\\\\\/06ktNT5Oh\\\\\\\/1+7hOdckgQ\\\\\\\/d0cvBn+\\\\\\\/TMVh2bF3rBGww4e2JUTkGRcSnSzDOfihoe4n2I1H436nX3j0av\\\\\\\/66AkL9P0gH2lmNPGdZ5qiGhGgnibQd9PqjUwtZhQOIY5gYGAK9dtCgpAoUtsazR6JAOJW1zVvlafWpg+7tTE\\\\\\\/qHxp83osoQNcx60mfqRNBwOaG9sNxszMAacluZziJ3pJiCJ7yEXvo\\\\\\\/g7VgndHBkB6hKFBcL0hdZqgbiOTFGiItsto+DYflH2WLUtzXlyQgvnq46oeaQz8fj9V+w6sMpDBjWK7onFm6xxkszWBfdErVc5pWA55Vfg3jsbr36A462Ii99Hsw6USzqkeikoMmY+N5sosAVcOWy2qtbKZaFIJeFkT7sE9MD66DfgNPn2fK\\\\\\\/eZPfJqGWNI7fe5P9P1DKVmC4JdVfaKEfWDkEnp2ptI3gnLS8vFGX1aeI0NznbUiImrAw\\\\\\\/YDsI\\\\\\\/XtvtkM7vHkAVTvNQlPcWDt5XUWFEYeYspdPrC9EofEkeyZuiM\\\\\\\/+EUYZrYmzr3ba3xVLsjHPBvu+REA2hMQ9fGQXycNT82wZBMxpVFfKp1ynXuYxeR6G0E8Q5vd7tLq9tsuC9MbZFcIgE9PN2YO6ZTsGumqIBxZcuvrFvttaaS6yASZTOqefFNAjPoXEXne2LwNFryZtlxWM0UDBrxXiwHsYKoYG59TSUlEwOJJbHl73TBFliedBW9OXwJiyW5elljwow8gWzVeSDrGWwQbK3wEADII5Tv\\\\\\\/7l82Dg7ZEfBGt\\\\\\\/nbuKZvs2k15U5XB9WrV0+qO6jup45sKcCPjbEIzUJoPryzCMisb3C9zpQbqIjtNCArfKtg4oaEyTLD1ICPOPjyoYQCjxQ824dEJxrFZtdmiUWbxxMSAMaZZNVQhKQfyqmLDtIUUYvuT78xylon+vf6SWLkTSb1sKyc8iNYGhuTrMDD96iuY+jqy4xgJjX9YerLdmxWao7uk+Tgx7UbaQUBGwGf3eBv\\\\\\\/aUWROG4dS2fAbePo4A8mI5znT2xyY9pPfzdZcde6yK5yKNTynLORA2IBLgPqR0XYO5pVKXIxVFr9BztLXpvxgDwQZDp3XdfBvSJuXDPR8xzTO9uqL+XrPDBl5cX4TUg\\\\\\\/pfdThfKQlyeMpMJGwyTEJYwfiIQjtX3MBqe1sEPc\\\\\\\/R\\\\\\\/RNNTlL2cQ4duWsotrWMuMkt1DURMhTLwYXd95fq7+S9g7o\\\\\\\/De1cMqTnMGXM5R918foHmMMcRw5GnmnlonHu4ZbuFOlUdPNuIRxjyeYONxTYqT\\\\\\\/8QY+cP3AOid\\\\\\\/Ygb4t\\\\\\\/XLcRCqvPwPDguYXVqisb6bbWNICkMe0fh0XX0zcTJYRE6wwI04+pIcTMIckjC3YxhdMh38UQXgAZIqJPCOwPU6ZdY6k0rHCxjU2KwQkCBF1JmrArld9fi5Sx8klgiOzvygOoXKAl4rsUuzkAfGpxA6OHcHZH2XINQ\\\\\\\/7q5wtCKNVGVOza3AuFx\\\\\\\/ypNpgmD5KBiRLpt3M3FitlWJpMQNmjWCjfGlX5Ft\\\\\\\/bACSLMG79wKtSkRdav\\\\\\\/udVWkgr5n+1BKe7YJw3bBtvzBb\\\\\\\/k9na857b5LLSeaUS8rw58pOaBcNiCFBM91yFJKd\\\\\\\/YnfKaXMKxL7IIEPBOpxOVCYdKsr9QisHBGF3F\\\\\\\/gso5\\\\\\\/\\\\\\\/6tsFIAnDRu\\\\\\\/mJwzm\\\\\\\/D470PWU\\\\\\\/1orc9fqbjKPrJH7bxkXj3xYEPHwTlbfdBprKaiDjAVQD3+d6QFhTtxNRcyMMH+NJd\\\\\\\/0QDJRDVT3KfTvO4tAiifudI5TVDMYNtwo0sY3uvz4B1qpQRDhlOLLlla8YlHM7pcLwOmNwxVNVRkzZUH1PZOGQ\\\\\\\/dB8hrTAq6gFHyH0uhURbKGkQ27A0ieLtWXl0jv3emH2pHq9JvdJiBALHNkDQY3Ttkwfbo\\\\\\\/ckaUKjH1aJhR3KlWv8OMByNAt7+JVYfqdvVfxlLQjy2\\\\\\\/aEMI01n\\\\\\\/ZrNO\\\\\\\/Ed6XkXRTYXTu8IAyh9ihJXmWWLySWJT6+Mkz1swMJTb196vjjIwFNwktsY1ib58ghzJ171UmHcoU0yaw9QFlTcekf2qPmsDng0SL3nQqcFC5kAzEc+WTdsIx4Yuc\\\\\\\/knqRJfh6xqATMBULOLM8TWeZtTmAS3EXiUuipuMO55JSbsxDjH6W\\\\\\\/gyLnK9Pvzm5UKuG7OrkVFp3FdEvEOfzYwCgGP6O7M9lQ7Ipu2JrTXeiCoIPnHhus5Q1DZL1JF8mvxGtNOsCA\\\\\\\/OQQuw4CgW4Zo4GC2Zb2PHPLA2JiVfq8XsE6j52RW7TungbJ5hgtO+txmk3p5nGbmgH016kz6P\\\\\\\/YJKCV\\\\\\\/yrmN9puduJV40LsBb8aZszWboQkGSYFmb9su7N52AVD7jJZMa4qPm0djfG7Q\\\\\\\/Xq8wk4DRT1KFHJphT+EnYk5kKBJ2fTu2ix2dH6PWihu3RJTEoEB9ePfIP1YPwItL7ptfOyLOE5oINKQpWK89ZPhQretI0cKh2gzRv586ztfa58LLxzhmcOizsUaWAA5KSahe4HUon2IhQAeIxWS5nvb+qu6H7zuKx1viumsA5P0iUFZkhECSucFMJfZYR8f7EtI0+qmWsoH9Ep3ODfUMkvHcIJL1sYBo7p3o5xBF2Pp4xJxQhcoIWfGc5L6RILI9+h1nHcVa0sobH5p+UQ8I26WVCs5GMEhXHaclc6eeH0qjYXykfRkR1Tw7wJTNz4juvhKuK6vlh87Oj6J1pHTB3xlFnewaJVGq7+QY1Ux2pim4rrJZTXh\\\\\\\/lrmNUFV2vENiWj6DP+hW8hVvnt6gGid1XjhgeU0tAhVKk7Zap+wVXjNdmVnMweOw3kZ9qZHVnfL7qoFKVU5Go5Q\\\\\\\/i+LY87QxbK8VHxn+uo4x9i4npPLWnpTtG4PEsm5LiywbAQZOGGwmgChGOn7+aYmM4KnaE05NEL1e+fuFglW9cRpGdiBcdyzg4Qrp+jmLFLDnBl\\\\\\\/yIvw5wZ8tln3tJrpKWhfZTiYwy9GTMlqmzv53LicvfWKuOP2RQVfTW21TK1VrwmgbovZWiyvc5Kz\\\\\\\/t8+qIHL6RpV+bfgqFDi5zwFd+yQ3g3\\\\\\\/ehrqVXp\\\\\\\/2ruKrXC7KacIPHf+dTTPbazECuxpCkVs52\\\\\\\/qJyCd3qzEzKdS4epjTR9cIvjvU+kkG6sFOAua8YGwFFYg6\\\\\\\/dYGm5xGOFiLorYDr5Ju6P3GNtID24Y7Yj+1nOy\\\\\\\/FVBb\\\\\\\/SFZkGIzHq\\\\\\\/l4lH2sO5v86KDBM4oPY0lPOxFytuPsdMmt693tbVAQw\\\\\\\/ZWYfwSI9YQ6rjZlk6RIChQX2fZvVEyTiApPuhWyfPUXjds+yH6PyccJYn7Zv3aboSiDiR4yC7Sh1DRXBHf4FlzKsTR99aW78DyanK9ly5XbcaQy4rA+Q5NUx+UDHwf11VjJ4k4WG\\\\\\\/gp5wndOwm+OaMNPFnWWsRzBM8JCngjdsTEq8e\\\\\\\/B\\\\\\\/f7GKlaVswbZmy1i8BQanNAOnsYUkfvWf69E\\\\\\\/VhaTRY4f+Ed+Ym+GYfNbisEIYVnJ0pGM9Y4xNc2+CubQB1sJtf5x5OUsfgON3bST8kbds9hKQjEek\\\\\\\/REzxzb0faJ8VyaDX8dVgBtwdWJ41LLJQAwoxh4BwdtQKJqHD2xT7Hv\\\\\\\/DcsCWgEcd50qOo5IOIKK8Zlbjc3AoPm4qWGFbs3ujmcmMcYPTNI4Vmu6M5+vjbsyvd7uP1y\\\\\\\/Iv1Dq8eiTtou0yz1k9AwbrtuV0wQv8SphIq7d93AoHCG5phQwbL0yv\\\\\\\/1MHjtr1IuXyBctL7JSGrU6t5oBSbIdHjEBQeWCCeb6mKQS4hj4AsG1AHtoOvN0oA+lJq78cSinVEc2n3ApGo2tLKejvkie20QPGvfc5OjnhWh1N7tafEcGg01eZvKXXSCk\\\\\\\/pXMClHkLroyTqrmaRIlHysjBDrY0dOLMcs0Lwa6md1l+SN3tHbnuv7Y4u\\\\\\\/dXDGpeslv0fjemNTakfbIgnV7jD06ngCPLM30MvE4wDtt6B5C\\\\\\\/1ewZqmI1PQD3a3eiX80OFq4H+FT+GHmZBgFqVnBep98SixW0doiZgCdO0PuzuujL3w9p7IZPUjwZOsWPBft9416B3kNMMxrJ+pxiyQKpvP9DpLMFNI3m8fLL06KcmCPxOIPbbZ7gqDM1oy\\\\\\\/t3ehyD4sWn3fSMkxgdVeC\\\\\\\/NxYn7o5Ar+KS7jDd1TFgig7cRsWulwqsw74kZtA6uI62Cm9IrRcN8A2xg9HrExffx7RE1rIa+\\\\\\\/QV7CAa4uM3+UVO1DXkj5MJlwh84jT8DIlVf1A9XzTgItLzIM0KaKs2OX9Cbrv\\\\\\\/XriRomoEM8TnxscNY0QJcN0Yb9lGVsGg9E4nwgLVAngrWZzfp\\\\\\\/u6zSFLvcnck43r98XIn01xb6nTdLsrv5yAmpaci8gzMXZkMXluBxOrVvNITDONiRq6RuEyR8rODRL1ICtsZDhG0AGo6LQUcMWuB2bftW5nJw2ql5EraaEKW\\\\\\\/4UwCrMQUSiF4YW7qgHzMUcxwODnZnljod4l28m6TweIqpk2UHE+uxpq0LC328\\\\\\\/9HoHn46W5houtuOL1ys2PZzjQWB1YbqBwiZFBdxepdmZEo1tlS9gs0CkuxsDZBBJzC6QUgu7MQdEh7SjCnnQ8MNLYabeJevRqf3xBCcm7dO2oZpJLAV9wdE8WLEaZqwcrBRvKRzreMRlmCJjrWvkUYvSK1ubDsuwRt+YMwB558gLUf0qBrhu45T+X\\\\\\\/cjJF3uW+oQeKw7kSq6epn0SV04IAJzAAw2lVnIAjucG7pi92km3OvznMBZnePQ7cgofYKRhllFEYsBDHcpA6vne+ol0rzsAcR6tq2+iSpPYPOORV5p5Ptz32vNGmHCraZ5Ax5rNCXREJl5l6dRFC4NYgmiWx85FtBB13GxnvyGBf8VT6zT47RlOiClFdtagPWIMPXHrGgVNxgSxzYIkPvvb5Ca4yNPUZU45xl7i1liK5JiErARbZbGAZkmeFZy9eOnd2BAQUuPTw1RZNtdzqqDoX47Sx5ugqmw+UTx0iuNrJ+2srNDZxFGIvDs+3o0tpr\\\\\\\/Rjph+FVrw8NMv0oyMoxQEhqgsfT+S4+\\\\\\\/BoAiRgJeCkkoLMFsVqAnCFwNkEk3yBV8bBNdbqcm0P+6537uARqz2xajKJkVXpig7hAUF86viqYzkkMJiG32Gqd0z7pcg73VrYeQTCBvb9v6W4kWfw9d+TpUaDNsxJ+EcZoPhgeIjak4ZvEY0uwHCip9lBpZzj+IfL67bH2yAX7qnTkC+sCOvW4j5l6z4EoXJolfaxCPdkBxlJhwZOnuau4r3iUlRmVuqp2tiTIncTOBgPds+4EWqSlGVJj9lFW\\\\\\\/0RyFo15Sgde+xE2dAIZEWtLzkVxkgu\\\\\\\/lDZFIjj6npLChcJDNZjd+vOA06SHeMZE2UzRr\\\\\\\/5BYfPBGhPkE\\\\\\\/OIKU6\\\\\\\/y1F2iYlVYDyL7c8n3EUBQGSTndobQhgON+Ko0lbYdeygTSBxliCvhcmzHrCaZ9ABkojOC3VYUhXQkXl6+ZQAlDESpRXVZLo1QWXJ3kNkUHISBoJa\\\\\\\/OqCzcNE8CcbmvHwJtkzPzZMl+k3ydRYiisvFegwXhCk4wWXqmrRdQyKHopH43PEwqjpjYAVRws5ck2Vr5dREfC1IAVwDZgYeITka42PoCiU6Epdc1rGGH81xKnLA9yIwVX74qmJbzKxwFl9lgIqDMaJC79GJCs9NhyIGWfy0oq2Ri01e1i9WMJFjdWvyQ+Mio0F4BCX0QP9FTVcMhII\\\\\\\/5CssZK3MM87ZVNi9gel9Js9ARVMt\\\\\\\/OiKb9tcs3KlLOUU0EtZicDLzkO8ZRD30pTyV+L8FleEv0k9oLWiRMcSng4verQEzA4jYi+BrtMbMFM81\\\\\\\/L5xS\\\\\\\/JAdZTTQC5a7zlsPgkoHoZ1ZMW+IZq+dE50Oq7okTlojQTPzsO1UftVSzqAXfqDAwO0TxgnjV5io1AthxSMQVyABeMyQdFXDWnL7Uxww\\\\\\\/eiGkNjcYuIzqs2jTOQ6WC61EIP5GTN6ImSdBz2U5rosRSVK1fk2zs6MhDTQkmdj0My58AHfWAjGXeNyZk9Z8D8Ylib0BXl3Ummrm05bIcCEowTBH\\\\\\\/MKSVcqX5qpX6XTkbheLI9JsvrcAp9Zdqxf0xHgtqekHz\\\\\\\/Pf6e3fs0jY1cZZxrovCk\\\\\\\/J7ynYXUGK70p6c9EomlagGTSglmpb0qoiSr+YV2W1yag4nXZLZNCcnBqGe0X0ytCLEu1LouK14wTK5bHBhwS\\\\\\\/xf+KazuLf00NvQ30f+SRpYqPCuyQEHE3fhCL6FkqJwQWz8iokw8AH2hrrTSA\\\\\\\/OfjkvDIwfzCt\\\\\\\/EMPus6e98eNXjr6C8MkehQ+sLswjqqX8BxRZhRLYM\\\\\\\/oExM4jp434HHpLqyCQQGZjye5RP+A2MseaNYIs2eOw4vt2moyXuvXef5jPNZ5RjnB6RATH8EG4yGLIHnZDrx68Fm7iQt2Jh3CC4drmKU+VDxaoUzpbuvCBJc+H8EcZID+RrZwbj8IczEWf7NBqSfBB9rhd+Y7GMG1Crj2kDuXoVXAwSOg0+Mt+eOSTGqOunyUkORg9dX0G7zIa6JBbXZbgh7qYuv2loa2tO8efREEpOewc3ATbYfH6WvzQKT24oQc+0Kj7nG5xoMhrraL8qNtr4kekSFkrx1aMuoSH9QYLu38rzxyH5kUl5WpoiayCG44KaWinw7NWuyV2tw5rQBITJOji8wjnVxPtYzhAplMAtkYDf1k3eKwaRhDrRybT6Y1HLAOIePbllPJmN9oAhY2KpuI1c7E98nRQawetK0WVGToIXqln0JgiVFQID8p95nf3AqHQ\\\\\\\/znephgTibNzruKxDJdBJeqVpF+oGsABzuRUZj4f9V6DbBBvk37NwcPHgo43cBiReVBCpvib39rlU2jdkH8zQWVyo0I5WY1fTWBDeY5xMXgcQur3cNsx4vFL1Tbnm8DGVZwZjNkpBcWltiPiAqkK4kLJXa9fuN6N5OhUSitk4Aw2PrOYElWeAAemux6Oy8JtuzyuSjBSBiSLtkCn3xegpjWkSyrE\\\\\\\/tha5MRd\\\\\\\/x0cXF5SIFCtvDztl8jl+x16VyxNZGzaLwIAvQpti4pyHpJpW\\\\\\\/3NbdRYoFb0FSKHBjYmfsmrSlFq8WC1sHIOk+1iGpflniHBtWaKK2tKs5XkmQ+sTpQfTdQzr3R0oBwePK\\\\\\\/2sy0K2aUBDzalS8+1Ggxer0HSrYOR\\\\\\\/0Z0oxqeOzJDcuuugM93HSnUC8RrCX0+K3hSiPQYoqhc5d08ij+DW8JPijqrdawQT\\\\\\\/XDd7Hb6IVmbPzb3vtU+AeHNLb+TW7JscZ8C70yMDECGvh11Js8qpEp1WCDEKVW4X\\\\\\\/NrizeXxAQ3omvW3IHm2tHY0\\\\\\\/0O2zjVsmg0izNXjMya5J4YjTPOxAPQwIceQGqRA4pDwQmsCxg55YcH+yhc5s1zPnDDEhCUaiyGK3B3h8hM0peSco8UOu9fxf6WZ0z4\\\\\\\/WjKi2uOnXZNaEwHQrDGfw7iUNoDxkrovXdMdB7lr754RUj0O2QxLSL\\\\\\\/J4m\\\\\\\/BtoMvVwfZVC9Qe1xUn4jFx0lWMcQXdfEWH7nUfzuxyv4rvNejJqafbEZqQav5+xiNAJquG5sWZg1NcXMwfuKx\\\\\\\/61IS+xxFBsegwSDOqPq2EEYviKdE2p0aIX0GPx04HBMWkK5k5rJ6SpkYyNRyDinZ\\\\\\\/dLinMIxUmlx73PVaG7v0xW4i4HWyFBaesPnGxg30SL4kYq+O+zlFj0VWiDOXfj336FY1mGBLvDghoQXae9JyyfMXplBtXuLutlWcUCDA8YCMzkd1vjDO66E7oeq\\\\\\\/jWtSAq1aUa504YPTy2iPSU7czcjeuAX\\\\\\\/lttBCf9V7Ha37OUfzvxd+mw4vcUaRG6f2ssQAM2sl7dUfksCX9UjbIpGBATqpXdss+HiadIjJ3qeubNHgVBNaeeydE2C46iOrICtZq6lgPrDmGd7MbLGI\\\\\\\/JdaSvQ58v++8nQ4HRH1bAA+LlZslPsZ80yB1Tfp9aNjtYqgb3u46vh5Z66Pnlc+Sl06ExAa9jNRC3N0q5NY7aa6zKIGnK3kgMDnJe4g6q7j9NnB+8WuXQu1sQKYEYTVqwHeLK2DS2\\\\\\\/avkkwm2yxbLjJ0jTHARKDeN1THt\\\\\\\/DA3KTqpRxb+OaFw6sbriLHLKy58XtIOrN4YQnbkexWxJkHuCSj7r5n\\\\\\\/YatqpwNI5L9UHRZ5HBq9wEXnGjflAJkWLlq6O+B90ps4bxJE3+Fd8OJT82yoEMHewwhqZ72Agbf1gb9\\\\\\\/tojmXQJfSF4nuMLM6MKqyTGNCSa0hwCH2yY2hLHSi7zszusFY9E\\\\\\\/SICiQn5bnmcxUQt1RCR9pPq1A60iHHbS45P63S7hwBq\\\\\\\/ym8JG\\\\\\\/IDy6Eyqskc7zUKSjOANPL+iak7FYGEH4m5EggWNujh1vjkQ\\\\\\\/MZDCvPI7auHWG88HIphPijAyvhFvu2ERYoW\\\\\\\/i1UVeKM8NBzJVc8W95fC97Q0kho6ly8Cq+unLPFDgZqf1aXOSCeKG+ZEnxD1clgvr6qcAWJCPrG0rUeIcvGAKFwEge9SjPf31lVCsi1GyriGS5h1eCh\\\\\\\/ML3zk83bc\\\\\\\/8A8UmAtWPrjZjG8YLgnWlnqKlZWBF\\\\\\\/GNoDuIwvWBeZxtklN8os8lwciVTZPZpJSiQs1xyip1cJcbb4UPrq8XK55xmVe8hDLiaVrjDmvWLJ3NKGkfAD6\\\\\\\/vHPijyCL0wZnYgtftR2ySLllpMb6bTJJuNjHZK6ZP15KKHgKvrGiTL5A0y2Ymd1vGPxGOMLUtUKoAJwWJT71YZK4VPkF6VC\\\\\\\/9S0Uk\\\\\\\/o1zLj8y2EbLpfk4zSMzLf6rgsbDiZJca6DFtdMZU9g68p00L3E\\\\\\\/SEyS0ASdfyYl1hMQj\\\\\\\/RSsegnpG+pfZUZbtTl7eTRWrCtGTKw367vbQ\\\\\\\/zeWH6xQMUvsdmrr\\\\\\\/TgsyaGC6TvfMNYMquHxoYV6hplz6boguHNx6LLuED8MrvPFH8o8HSlDmSG\\\\\\\/uKGb\\\\\\\/CAU5VIJq40wqNHDomXjdtfu99EdGdW01qj+\\\\\\\/AAAbIzvpVwM5\\\\\\\/B74NGZGgE\\\\\\\/iCILTYWFZBp6zOETfTUp164CAh+mZ391KvZ81NJLbLdLp5Pj4lSPdXhgKYNtpS14LUc\\\\\\\/1\\\\\\\/dAOkgCvUCl53yREX0QSJJiKN+nyuToo0VKTxnVa7KszNWwAyTGxUjqh8HSGAbY+4AbrlpbF2k1eNHWDa4wHsWu+mJTCja3Ag4wtDT6jjjY2eNIpsLIvBAtUcAOW9PGW\\\\\\\/HcRUGFxNvGsm2k42pJRz7xgmRwp\\\\\\\/KYBLmiUfGwM8FhqecTKuRe8ae0X4PVTJ6whbQWovwJS2I86Y7k3+qmFIoBUKY870HhfWETGrSwr8w8WEUlcLVWzt7vPGMM2Yp+VjEVI\\\\\\\/6THrSGoG6aTBBBv51bxZl4S24WgwqVSGOxTre188bVgCTJAT4zyiXEIPTcznctTNNIY5ajyxgsD0avkwryJe\\\\\\\/EF\\\\\\\/pjKIx5bEgRJCafLzzpt8xU06VrGa8bpW9gVjKJI+LDMDwjiu\\\\\\\/6hleJWKRd0gOpJ9Xhn\\\\\\\/HzavN0NHrlQJoyoYuMFqR\\\\\\\/xqlcjHlVOnD+KI7n8Gph\\\\\\\/980WP9ZAqNKC8peyNtqKXGA8PyMXBadHZ7cLalhc5mEHC+mWsq3CjO5fudX\\\\\\\/+gur\\\\\\\/\\\\\\\/H31vJwGJdZy7H\\\\\\\/ix\\\\\\\/fBPVBdxdQqEGZdfnrv9mlFY60Q52\\\\\\\/e6KAbUiEP9fs5Yu7jCdtMoqv6HXFlGMy42M+C6eHHuji2bzHCDUaWeMJA6aCRtvHjb8vNiuczxP2LOSr\\\\\\\/G1Axg4LQWkLVrOOC6eNb8UxcLn3SMiUgF99m9RjA70TtBG8c\\\\\\\/MrGxOUS\\\\\\\/RBpOQkyCYW7vcjuDtNrvM79eYJU2DsnIwyMjWFQN+QCw2h2TVOq6149jnQW6ocBKqPoh5iV8A7JkP16P1kgd6No\\\\\\\/qJ5ym9v3gk5f+ndMpZY3X+i57r2WRMIedL4QExvPKOC6TZPDvEB038P0OHoZjk\\\\\\\/2FvKlXpx+jpm9yZjeHmdBI1z8yFAO9keVng5lQAiVfRkoyIP5ndY\\\\\\\/58Fi9fBwprNVnbzrQKFleUNkzgtdyCecA4Wfo3CM3pOzX3JCN2rdRSylZxZ+nmZ0qcX4+DD7J3WewZIKIMoe9VsK7ZIdI9d04BfiBwB2xMWHpcIy\\\\\\\/PpKZP18vjoT4q0P24lpooQzld\\\\\\\/aqyJEZGLnw4zopcGYAjWiNBrb5bvCwjjqehQxMF9rKbaKC9a4KRHZfVl2XyQf1xmgTpm1agMKHwDxVu5i0d2um8w6pfdLLjdelwhA7RIKW67KtNpJdIcRrR7HD1lJTV9x20IPjwnIlOkQu9cefltTSMDbEiQc38RIlJGgku6FmA\\\\\\\/+vO9KIJKBGAPAvQKVs3vF\\\\\\\/mQR3PirbjWH1N5kw410attWMGIpA+vPJKrvGlzxWKI0AmkHrkqAPdAoobez3SFFjCAGzhLnsySJPHoLpktmkPrK8zz5IAPu7lQFSpnjHNs3rwYGzfVKtDbUC9AoE6nwbWQmA5gIH7URO5qoRKPom5vFSLztYs+QaNjBjB57qmJ3dUBbT+XI7dSTmxm6qpkYg9QUcCA4fIloHc+dH8F38i9pwUxpD9B+3r8+E7zzBhJe\\\\\\\/gfOBrWMl5\\\\\\\/JOa2yKTtycSkXaPiVu6+cuAlSwmvdPJrevEcAymwN2qMg49RnFL8kWv5zey5rry7ge5K1PW+0+MCzQxz1K0Z+f3F1+SZ0nXqmrYj8amFIRzoykZtytQUm+KKvdfRoAryteNRj6r4xaNbljxNNbiwwrILRqOPjSyn1vNxc20llcBl7jGKMsB6mILRZcKUKG2yMoQIO9bEthw+\\\\\\\/WTZU3L3YRXIat8er0KbHIPiRhOuxnruwLT8YyUXMFDkWT2P1xbYMA\\\\\\\/8P7o6MvqmdLD4qPL52tbd4+EHZDyeZHR0yoTBb9Pc2yZKjtrtU8rzM6KRxl1TgfjIM0fFKpYIdsfRJNU0FKR67Ytn0K6DwpuqzOnpo9oWowOnzMmguHjZTMNUDuYy2rtLdxBbJEqxPZA9cf9Aobio5zQ6ICcLShZkGlIEGC31rMJaLShzYwPU1igCpg4ItCvl+pFoxy6V+d1pI+vE8DN+sr\\\\\\\/Zs04RsT0HrTStzOThwPcRCU+E0UKYwlIAVgvj7+50jRaVckRRiOA6gRdOr72hGPcA0FNzpZTYmjjd0ySaxwKj9pX+ydyERhv\\\\\\\/5b0npudMqBd75cs2k0txXQNEgUdGYqgVeo1KOJvOajX4NKl3sVLm8fbfUrwiOLOjHFiwxLAW2OaQFZEBtHwVt950oxvWDJ4UsqeeLNRSDNaMngUqTzL0qmp7FkvwElzQb1x3FVHho78ughKIfM34w44cITN2sDkV1zX8969wDnm0SfAL+\\\\\\\/uL07Odd8y5zwizkYLhsnR+EYLr\\\\\\\/sptQLTKKnGR5wIeAGBGF7uB+WK4BtKZ+gNz5XUZ+HrpJeg\\\\\\\/YCejbANJq+Nm3sgnssPxygse0oHJEWbS5VfRbDx+sV70ramzq7ahg0fXCk3Ryr1xf3t7GgKvl95RMdOs5Zy8MlaTLjmAiWWnP0\\\\\\\/XhGKuZXdWCfW4ckAhDlvdhG23QPiW4AIXyXBjAaP7qLSCYJmboNJzv8hf0iSSzZ5Ax7iOXTySd7v5NiEzMWSuehyViuD0N8s\\\\\\\/LUWrhd2CmDjm\\\\\\\/Vvql0XKSWCIBasis8IFEwDapvlpycXopIywM4vFfwDlKqPVFY9Whtnte92IvzwHg7d5qBrcpDdvua3h2J84FOI8gL\\\\\\\/wll4t+uGwyWfHNcPMooJFMvJ1MnDtVlgSVpDGLsRsKCuHCypM1G3b2lBLR7+w8KXc+XuKe2Y9WQIuMKj8V2odM4FhuUS8Z60SezJxUJeVqVyg+bGb1OL6lwcwMydvJtcQV1ragXRNNW5FI5\\\\\\\/3tZ57fdcHQlFKIvX8iubAQZcHwWbL0awavBZfK9e\\\\\\\/W7t7CQdg1UIOyfbMeQcQS4xbRIC2YMbZVVBQ3CSBftBBv4H\\\\\\\/MK6g\\\\\\\/IkLypIuYk5UHJD0e8jzgSs7u95ZzPInJlWim+LKwYZWu3GjeldK093D5EV1fBBil3FqJ0UvVZUl+aUwsWlVCU0AGkdHiGz\\\\\\\/XVw9Ab5mgfNKFJSmrehPzsfwcStSIf\\\\\\\/Bu5M9U5nI+WQLCRmxiRMUHM2NgolIByATndVJvBREaZkHdU5MJQyfmIijLIKKXWTPgSwJlgrkNAOGcVhfMdfUunE9N819IFrBhQw9c5aQBieZXYP5hIsSAoOnilvrSw5dbUCLsAH+ZDuEU1lQ2dBfZunq2kEfIo\\\\\\\/Af4Q2zU04W8iFSWGq5CnBi0Jgmjhc3RGc\\\\\\\/vljWXc\\\\\\\/htxM1HVU1lD69Esi9IIoroyBkPok6IdJ1v4GOnJD0yFNsxDmQFVcGkm7Hqh7G0jcxWBLr2TPgGPKxWJEOUen9FeSH5j5CRIN1SZp0c0BoKTy+yd+sNq2q7581xuFCC\\\\\\\/v\\\\\\\/RGm+DFzz7zMfWUiFbb9nfi3YT553Cj49FZk5wh5WvJkYut7J6a4E8ebuRTKe6L46kYzPC7UlkghkORykPvvJcxeFHO0kU7+HBzNdFFs0CTCV+rAfP95TGYrahQIm31mxQuFYZJOD+YYENqJ24Niw4fVH2iFXtBA4Q0AqW6JE8b1xr2tXlYpmsRbczeqO74gSg3sXWgw8MW8horFiP6vMJbsFVFDguWqtxd7H3zcr3sfCwAtrRT2xL9aqyqSxHIWFrR9r5cPMDMwpBEapoWl9eskhQzkE1WB1Yiuhej42YRh3Lfz155POdsT0NDeiU+t2rQ4A6d2pQP3+9CH9PmvmAc8vTDXLzJhlwvIKdeLqfV8FBkye6kFR\\\\\\\/LKxWkEbBiPfx1aOviAmSgXKj5K5FQoLuPiA\\\\\\\/KN+YGP+i003aJDFUjJkhZTRqfFVrK0G3Pcb7BoUuG2r21gHN45pZJ1N+z+xfycno7OtSgYBFwZLR7wW3ubQfYl2j8lNKb4qvzFAfXBCl6Ja5cC1N5CqEHsM7NuroJ9Yb+XT\\\\\\\/p86shn1Uyv4lY\\\\\\\/hfLryKwsg9NKt8Ad660fwH8i8QWWNNpLxTq9vt+\\\\\\\/gAnAaG71XVCLhv1kC39Wobqpfl9LqAPgMYxVqhX7XZW\\\\\\\/zUw+kntlAopeHDvd6xu5W7GtgQmGB5sSm8uYap5CFmpLquHSJoki7Mfjmn5O2KHWl79BKU3S2iRHBaFAfzoia5DVCAzCVAhoQeU\\\\\\\/Jt6vrXKeIKERiCw3bKKkPYSq83KBAkQi7\\\\\\\/8dB9upvQsHaInyQDWVgAQroKOVWk6gw1BFwBqJp0JjJQ5r\\\\\\\/pvpm6XgHlQSL2prP9y5GIKShO8vMCD6wu5zLFztG12\\\\\\\/D2kF\\\\\\\/RLLprMGmT8XABepgyUptaKbj3R93kEJ\\\\\\\/Lhe3CVDhjpZm9UW+voHyrjBtdN5ulaLyZcK7\\\\\\\/AQy7MoIIoGasKETOr6HcQn61TQgW9uM4PdbWcF9mHVDyOTKZqTEdfDrh0UqRhyNmyD6mUxDRSRY0ZTNzR3TCKn2PzIqiqBqFpIyNVtw7++H0gVVFZZOYPwEwWthKvLIn9st9+1PpdAEERgKyuKSrijk4ESIgpDipDcxii733uHBA5xOyJMblLxorWRa0z6zgk7+71qTNUEsP37DDCuRwYcwlVJtOtR3gYtJ2Stwh56Sp0\\\\\\\/BX5uGZP4HiexYLlv5qKbCppmp8zo0RYlq5wbmFPUA1ptB+ycambG8XMC1foqKh0o55xzaZrS1ToL0G2+w8o6FAEE2IjlsRfvmqwaHVGvVQGl89RuLd\\\\\\\/oiUR3wMEzWH+HbJbA9kskLwTC45yi2JkpSIHVgrxC4McThAeikpjpXpEuaY\\\\\\\/XPA\\\\\\\/Jv04x3qRJ8YCEsigqbnhv0OdRPybCDYDhOe2uozmEFz6MfWgrzsYueMq7e3EVv7a99C1KnPX4WNwAUpjcWd7mxIkULPNkjrMaTIIuaBht6jriaA85IYEd6dOvk5JrbWjHckLcE5LvKS9H6U4IfR6GibLLFmMnIzE4sAmbLD+o4najFG9NTOQCaiW3AyLCU2zBbZx4ZCH3NHx1msx6qeZSEr0Ni6xttdNK4ggDqFPmxFlpeQZOQzdqKttXGSxCw\\\\\\\/o6\\\\\\\/Auh42JI6TUSMqHgPsKeDcDNJRp3BRrlD0pF5dOih2+Et5LiNgWNw5oC0OWqd8oC7uDeZ9aGhb9jO\\\\\\\/rEgr2fsLhftoouu\\\\\\\/5xMa8hMThGW0C36KIGq9iCk0tfWyaICUtbMSxZXH6C+6C\\\\\\\/cQNRYqMK9fkCwQAKt0aWU80CJwQoFosYQH63RGK4EnvNUKHTqrFpmRi3j2xdlaJt7iiK\\\\\\\/Zjkm6J98AaqDvKqgwbxzwZpsaH4foGID8KwWE\\\\\\\/nZIE3NRnYMWqbLA8Ik4RT8xS7UR4t5lf7A3HACeJs3BUJWkSHZi44DmAiu+sTIRBa2GBVU5MCwltKS4ttmwU2nA2xCsjZgctdUzAGUOMgPq00NzM7xXyivtuGxjNcIn6md11aRal0H4kFQ7lNbzhX5fV8s7dT+vzfZqqzbOg2TpDyeyJKn\\\\\\\/SAAo+30giKcNhRTWGsAMJwFcQJGUTgJUbW6Hkw+dnV0\\\\\\\/gPEVs9gLSajaUQoEzypNPKlveZucwkNNlVR9lpTOTCaWvivLv4EoLuQQoKpxZlxRAMPpgKiAdstE\\\\\\\/ofJv5QP0tsX6rFvTrwMql7DuzuXpqakpu54JwnS\\\\\\\/6JwceyZymYgwi+RDf5eJKfOAQoks9nZNQ6hCV0KkQlDedHSIWMrSuPoH9S9MGt422e8AgWbxkOTsHc43Jp+WinlYEhIwfdrfT2eVhPgWwCwBZ4YvtFAKH95aie3221hKZX5c7D1ryTb\\\\\\\/9\\\\\\\/6J8IqQeuKAM20o9C2zbw+MB194yTuMDYT41jzbGx4EYm+PbYpcgIwXIgnQi8hyedssEIpbnX5zyhupQ1NsyIH0P3wqyzRibMbcuLahSsAWIMRYzQwC4E5iI7gB99rUSRBITsaTpz5Rkl9Zk+\\\\\\\/j1lLGVrFWv1BXnS38pF1jobyU9ccJ84cYgvkU26F2qD2kp3BJqbzIN6ROeE8F8bc2kn8WjCegTevGm7BhCRYcffI99GG0j0f\\\\\\\/5SzCf3aUbb6pM72q8lPeZCGz0XKNstE8Xa0w8HMCP3T2yjzBzzSWRhr42vYlfie2TeSyfYK3Nunbz8LolIMbkQ8Kb8s4Lq0NQ7Je2qDb\\\\\\\/PFCoDRftHX+iqhlcC+iG7BqzQSI\\\\\\\/PZsvBJUWgBEKbcD7mC2M3\\\\\\\/8bi4eK25JOZq1li2ic4L3C2Qf+GGd8Jq09QizpJvCDzVWH2Cei5eK3hs90nhwhejt1EKNUdENT0VdIrQbOYU2mZShhAl7u6QolRhZz+wh1VdHO6PR7+jf7hrODqzbELncRmykZOMkNejSAgqq9Oqo+0xY3o5Jq\\\\\\\/k78M47Be05TYAFjtSv1aYMAYU9fEqATJB7zM7wCG7rcHUpFiYKKjzAk27GJTE4Oh0DF0PEfjc6mSbohXyOmQwImgjXP1UTn2nfh2tHCx0QSgox2nNsvAxlJ2LJWQgXRcNsdhXVpa1j13+66sOluUkNDeREM2aHGwm7yDARha0oveoLhRHlWPL+1AkZepswQZ\\\\\\\/q77YczdUjwHusWtBRuBjHeYuKBgYJmAkKBMDgCYHHytW9y2N9h173\\\\\\\/wo8utf2L4DwcKuAbhxBa7qREz0v2q9m6NFAHVfrLWK9lfqJ08WjLrgPZOwhcqaTWocsW5l1gz3FPh0iLyk2VCZP0nYegDVqZVPJXqkBg8ReU+9mObuF1uabi0GxBGuVKtHvbyb+uM+PsfmgQOFBJDBGgd4+FgSMvTGmtfoNqCr5fngsoJtyPWvp8MSw5+8JfkDS4c1Kd0JQkpIee58PZFaaCnEbu2PB4B8AHFmrQ\\\\\\\/ix3stMnewUv2fEWaKsfRIYkG4bvQDqQyb9osTHesOI9b0c+eMKseCEalXcTBL5ML\\\\\\\/3QlPUPmqrn9u8PhF7TfXdXO6xcuSjYe\\\\\\\/JF5DpxeIYFVosWkSn6ZCUFMu5v+iPyvGlOyiqh7oD5Fur25zn2CfB\\\\\\\/3ZwiMhX1ElKYB6HqU4FCV0S+Kypi9x3SGoeMPC5xX17z5ILf4Up6E5NrMB2ORGDC7VrvXDUoeP4djfKlgigi80HJrwpa4hIdTlX\\\\\\\/hAHh6FEhDZIOVZIQ8VuqzoEt6RFsAawhyfna0xJkjOVq46ufapquyEF3rhqyVIr1rtawPbtmMw7A4KneuflFQvTNth1N9OXlzca0amQ2eObSvTBIgGesLagbODiBsTNIV0epWGXQy3ROxgHNRetmSF1WiDRJjmHLB13MzL0RANl9ofSvD4FXsW+RgwrUlid6E5T6c4AJRe+bzHEH6v7CpYjH6lh1Ul50AFMuz6JYwdaoYyIVtUbdHo0TrAUMshgxaWkTS\\\\\\\/Ul\\\\\\\/jDWq6or67jwvcu397Qfgn6gVCqsMKGb9wQQgL\\\\\\\/STAMdFOMD95cKZLOsoiV9+3jscqvrcEBXUOa1L5oTNLeD4OH1NwCDFeOhqsUxOzXZiyXVEka4YvunpWS4r6vflDvHnsqyALfcfql75VzOaUJ8jp7IXCgoaw74DM4NVX4sXfg4tbKUN0gnlNXJBmcL2C53IWeXCDD9Bf\\\\\\\/iDV4Q\\\\\\\/8ZYicqiPwV+sMVHecoZUyxR79pX+3B2QVEj5ccam\\\\\\\/QpxCPr6n\\\\\\\/0mY57idaTN8cXBlstw\\\\\\\/1dFHP2bQtaY7vz8Fo8mn4dyJejwGAONe6lHZBOnkk01bPONU0rbSuW77kaZKoa0ehBdmMYB64rZUKOsMgYT6IHKkPFm88kZhN7rJylR2rkqbiTZFexm3g6e5wcOgx9IOGOob2VrE07iHDl5u1QNGqndQCki+3TUgpslRf2hPrCjkHNIKmwey5xLYJIeVv4O37xgbj5qk1KL6y7bZ2ZMawhV\\\\\\\/Yc\\\\\\\/kYLd97WnqmaP\\\\\\\/KT2nim7+\\\\\\\/WazgiFw8inSuyB2Q6fgtX3MFtUjwJn5jz\\\\\\\/78j0Nwa2dz0r5O3YIAcw36W9j29\\\\\\\/R\\\\\\\/Qeu1rOoB4lW9\\\\\\\/zZoyX1UXgGn5fOmtaHNYOQurpRH750oCPSGvEaW\\\\\\\/sCR7bgD\\\\\\\/lFWr\\\\\\\/lmfJG5yqoGK2AHLLmRCD3BMOivz0w5Kh7oc6zVV6ZGqMSjXhEJuWGqOl\\\\\\\/pFFP+swJUm4+zlBm8hJUL65Klz1IZAs\\\\\\\/XIJdow9tnAf0q7eAxdBSBUnCILDow7eyp3Y99g3JW4vzl6UCquXCxaBuzPTuEonY66kMFzT9Dis9jdX53s80nnl\\\\\\\/v5bVPTmTw1Xaw8pxLAxp\\\\\\\/Z\\\\\\\/niCB+CX9ttajBWrb\\\\\\\/2OwWew47P96Ce\\\\\\\/qQNdbuP5eLytsd1IBQdEL80Cp68FKoROl0nuw4JBxpGrw\\\\\\\/L5LlxJdmj9SpIbLcfD06WSx6GZ5eAU8S9QI6hf1C\\\\\\\/VyiKn9RuvDJXJ46Yr4LogEA05S73CzRBk5Afa0iLHQUrFKxJU\\\\\\\/6IQmxiEdCJxFBQFg\\\\\\\/cMS08lrIlTk3\\\\\\\/oGcfudl7Ma75l7aKjnwfNQsZ2gOaE8MDc9H2w0Y03XZzbSM9Bfcn7DygKbGBoV6ayLcKkEoYyftoHkv+s8xiQhY4ovDjxWCOfjc5jKq58Zb8H\\\\\\\/xMxtMIz6NBzXCP9b3sPDt8FZ6LCAzQdYETddQi62lIrWovHv3YICs1sXTDqb\\\\\\\/Jwm4Euxg1dj9a38Daa4xT8lOs5HhZmyt2mi8ffFP8CeSCTA2HJDSdiOTHb2cK0T0\\\\\\\/sgUn\\\\\\\/8yZC6WhrtFwLp1lrPe\\\\\\\/9JvTA7lvnkwxl4L8ftvsIFDomy3lNe+TJY+OAMsN+sWYMSQZUJXHk1z7qwhLj7pJIW1Ukma5P+j+sjIuA+zZH2AtzQdDUPggMUGtYXmbWjgkZIrvIh0mSDvg3lcDf7NtSqfp9Pc57FGGiUiob+VR\\\\\\\/5\\\\\\\/XG8YVZDLzWjn4MjwzvodzMmgE8uwHbbr9GMnX1MkvR0t+giWwsa5AYdl5URnRDLU06KzKGQScb5W1dr03G0r3qJOKxgF1mDRoTskCZY5T6Khxv040Bh8KWetphbLomSgUCl+bzOw2FepsqZk6ToK6OYFM7VJ7bhzIoF0ZBiGNzPMTYjVZ5XAfNE2dYNw5\\\\\\\/8E7ZZ5itIfDg2L2AQCknsn5WhxR1RE4MkjKBwnxbREH+hpjzPYGiizEgzdcVaqEffN0YyUkbzslPKWMlm+yu7sJWZ7a667D84I9UTt031e\\\\\\\/ZikbYX4QxlSGdUF92JGblc6JmwfIlsRQH5pALHzdsh\\\\\\\/BPVgmSJOhRiXo7SziH70BfgYsPDWyC\\\\\\\/AnrpYsirmgoFAYO9QTNXM+NErNJuLcZwed3ZOYF9M9Y8F+Ta9tX4cx5hailKWlqJ52AjmVZ\\\\\\\/WlfyIXbnKp4\\\\\\\/o1EVgcd+l6GDRJbyxWXEmiSBKocPKZcjrIckXw6HvohmQZ\\\\\\\/J+wGathAE24tzcB5G+1N8sD9rqs+V\\\\\\\/0chAO7P78I9VLo1PLu4T0eNoV5ja7DGd0KiEmBFXIQ12y1EryLX11Xsgb7BEIjVZa40AHZmMQmlKabkstCQXCilIIjXdN\\\\\\\/LnoD0mUrcC5KllvZBoKChtzS1DIy2QursGkMV8klRhbFB+YopjkzK65bcGzewaPui3UiknfSAW8ISuGFOEe3Adt7e3kTyQqSo1l37eB7YY\\\\\\\/IZv5Uc0wO\\\\\\\/ItJ+2qCOJ8aEJwndGH22HdgZZ8PQ5mFSmUfgZy0je0Tnax4PUr0YwiKEKbOd+FItoTx+bTwcWA3mi6Oy\\\\\\\/O0rB78+gEejD9MZFvefZ2BrBLG5kWfE2kHZ+dYTGXcogsbgKc9J9BvOxt7PrbmC3L1dUNKnPzcKuKma\\\\\\\/\\\\\\\/4d07gXzXvM6owLWqw2+gA9\\\\\\\/UY85iKaRhRiFlY+QQWA+lAEnS\\\\\\\/sfLc6i+ieqHapuk+GcAtw0Q3FWv0Qw8GsPW4rz+I40UwSPE\\\\\\\/Om+KoZTkGm4M3a1g+zAlvc1Egqvhkm\\\\\\\/qAuRJ7wrhESqa5FdxjKKxoBvlZFOKBB5LKZVGTWe9grnrQv28Cj6bJnz0cznNliW9AKV5pae4fHQce3n89QHe\\\\\\\/C1Gt2NGqgoH5CMOc\\\\\\\/7w+NbfdkQ30Jx236dEdc\\\\\\\/KLt3DqM1\\\\\\\/O2HeS6D72TURb8920Oa5EwvN3vtVrZCwuTkJ8XvMs+YDDIXcWmzqQ6vC3yyqfP4d37VHAkyPxDQo9DOPeegRlzuBByOuYqgYOZIKEqExwhf81G0Z2vjqj7uZWg8VHiiuUjfYhZdoJAXV0FvEVUnodt3SlUNOtGFdgXmA4hQgc89PlPuhGkEohTH6VFOV+wkhfCxY\\\\\\\/oSkNSM2QD4U\\\\\\\/P6vKJA0IP70+F\\\\\\\/hT9ZFheFtlxZ\\\\\\\/mW9AVG0ibxiMpbZZJ4VbB04RNYd7EUb3twXN\\\\\\\/SceK8Q8KZH9x1a+pvZ1kqjmqn0u6XiGlOLsQwdZMYtha62FxIEdHkOzBM7XJmG4zxWidknBH3IzcgQaieK8rlqRrKtv3ZOyCX6hctunDPsR9O4tUWY67H6v\\\\\\\/N24aZ7kAkPud0nQE2Ur5teKGj0t5qcRLWRU3M6MKt5XB\\\\\\\/6ez1H+fPRKRYMvLE\\\\\\\/SyYRd1Gkx5qzUtwc36dDs5JFzm1yaZBlbyWFFtki1l3idZjt9SzJEegbPztCh51lG8bcyVShr90k6l8Nj3uTeRNmmXeQXNoZP5E05lsKEtGv4vrJ1O8w2FRJ89Ostxck76O07JF2PffQ\\\\\\\/5RkRhxJoOUSdduz7pwF9CR5c1hVP+AxjSKlvmB1dz9W1HUMx8yGs\\\\\\\/ZChH28Ygq2FS4t6ij3\\\\\\\/MwRw9lqjP\\\\\\\/0Wnv4KHHDayE9fNRQQWCdH1dOPSHxoSI3ph0Gglj40gjmCfV0S\\\\\\\/ZwncnZXgQZpFmSgr3Ebj7OrKqP8FB2NgLacIRh2\\\\\\\/jrP+bNfAW6Alaab3gMJPVhCRKOBme5PWKqw53q73qioWwt+BVQW7IKFelRLmXf9M+3i\\\\\\\/8RM35qr6ZvnjH5TNFperJrr4WhaOXN2beY+FcqcSDXbeiF87uxcpPGiY\\\\\\\/VuxlMzDmYkuHN8UUftf5btctntyTmrS6eb\\\\\\\/+9vIEnt56GOeNa87iaDeusWdn11Yg9CtTRpSIbGOpO0klR5Gijl0VBU91DZj9pdZe5g4Go0bGvM1Bn45QNlRsP4rRooThUbyrSoC7rTMj+sLSVLN5DkrKUKWQiXyKe8lfwPhH\\\\\\\/z0d\\\\\\\/HGpornldZWqAzksabNd8yBp5jdXdTTqMXwjjeFqhcmf9w7NCRAXyz42HaTnte7PcsYeJjVJUQULDaY1XfvHnZEHUQJ\\\\\\\/LlBrfqGbHQy8Kbc2QHrLorXpLd6rtCEptvTICHFIgvypGmQIUcxhS8BW+UDFNsF2kwYPhuURLZcmnjhpY70hYF+VgFytd6dTR2nw+By9nB4RxaLjFkxUhayDvJ0cfU6zQMQ1cH18blww\\\\\\\/sA2VXtl+dISb0lhP6md45Qpf+wECvghlrK+BbbEM1vSyN4ciicTYx58A9l+ztm1DiH24LTbXgNTUgWdtlDMxgd7C6zZfmk28SU2h0Xu3\\\\\\\/v\\\\\\\/ZR77pNQDNQKVX9mSZFpvy2bbh66wPrOWcA6LI+RSmuqEEZfWAqzFaGu80nt2s+Yvq4X1HeBAi4YS5bau++LU131r6RsZKbeN\\\\\\\/0aF9OO8Wc32Y0C2Ux0KiYY6XBlF36ZmcdvGMaXqKfgZT2+B5zual89Dq+lFCWq7uUNtlwRgsfK3urDK0Nf1vCRDBU0NfU97G1KxIjy8DyW+bJAivXrYPTo7sKYOlyXviWD2drCmmKm1rydQwIxQKPPpoCERoOPN+TSNuRwlsJBF8EvcVTTOWF5D93DLf8eNIxaOtuCfoRKpI8D6BwR4pFrXT+bOyo0yKDOYPupuBY1e7avY0Fu4xpcZF1B6f25qXpeAv9OE2CBQQSkoFHjVk+V19K1ez\\\\\\\/bQsWAUyehwITxO5YzCV+Kaup55WTVy2IlqynKMT+i1zQW7K7hpG8pC0uzi3\\\\\\\/yUg\\\\\\\/+iW52AvFeTYLjjcSxbMrI4ioB1fEXe8\\\\\\\/G7\\\\\\\/y2oCnVPDwvvlTYWRhT+XW3kfUfKwSfe\\\\\\\/6ZxpxlHECXVKZqVc\\\\\\\/WxDSTsZ4EFuAQuYMzOJPBP6WXlGNRj3Ng7j8+7QWm\\\\\\\/fM3FImRdhFxWo7vKGFvR40P6vDvitDZFTZcV94rEtr01A6OuGUvov8Vb1+mYNkZY3TJ\\\\\\\/HVvoho00hEoD5Ebaup\\\\\\\/OcFGYsYEG6SmuuKYjA2ibxYYVZL0LbbNZIMph8mTGhGnMqhas1YAM5qrBXEa8Mr38QgVbSavpOala75y8swwJULAHS9mEGVCdmNWWY9ffwGSMf0VJuSwU58B19Di3KqPywwHfxvk581SDPejyZKtVV3Saau3bgTD7yVPeufou7prGOdeH94+3o7a47XeHTr2DwVNHoIP1tSTUcKuYK4cRQxzoA9+y3azyGLLEWvyw\\\\\\\/nwzQA+oGCo2DP2vWDdB9zKjI2RHC++IO5JHPl0w5NqIlTmB+XqkFLL6eNNaaDGkLfk5pS2rmjFSsYvtwiP42Ee3roDLy2PzSa906PSfq4ZZQDYJ3tM0aqQn6\\\\\\\/KDI\\\\\\\/QdkUs7Ot\\\\\\\/a9OQmWTgD81lHuXWlmbNmb6z3VcXXLfM\\\\\\\/UQy0PC2igA3JULlNrdtr5eyojk+8s2ZdH1imkqfcmZ3ZALhxUp4d686ZomGORryuwKHG8zmLqWFAH\\\\\\\/TOjNjnuB8\\\\\\\/Ld7GkHzYXkmoTGCoTL4EErySPR3jkw8pANl2PnsMMT+vk97gLctPug8L4gPNf9SfU4H6LnyBhCx1KEIP7jItT7kspq60zkIkP\\\\\\\/EJWh17evSM3SXthl6009rM9DF4j7tE8S56cXi+\\\\\\\/zzmfi3Gvj5XEtPKBtyKQ3puJpyJzOhv8Q4BNEAHVhA1gBxrXYqElizey1LF6n9V+xCRTPamBbBzqe9VsETdxU+GiTJex0RfRo7BL2OgZUqQ1TfZaDomrYnuqTLXHraIJ4gzGVxkLoN0lzjEqlVVmG0N+aKzg0fdbxudrJTu7wCtLCljYKRfvYMmKX1fCEDdBNraX9K\\\\\\\/KXUGqSFwMGxa7hosnNGQmdb0rw5k51EbQCJ6TEw5hxViuJd7T0CXJReZ0XQuOgQaU4qWC\\\\\\\/X+jcNVK4kqGy3IjLKuGstQWT89bXJdJ0bVQ0F\\\\\\\/HTwldDCQF+vaj0OpdKYUXiL0G\\\\\\\/42RYWIk3uHSfbNxp1bGEsE77kMS\\\\\\\/QWaLOTOlcGEGYmhO5dZ2LIQOhS2aua+jOmxR0eL1lUYkUy4cL9Ytr\\\\\\\/RExlN6IDKOUunX+Siz7W7nek7buTnXFloTHf8VEfey3hQ1CyuN9etP2lBaYl9cyP3IMhdlJauq1n0JZxzJh1GZe7hDOSX3BaSD2x+lQSn+aJJLLZXWqdgf\\\\\\\/\\\\\\\/IIK9dbXUovFaCM2QNs0NIfZy7b1YWSzlMXl59hRJcEYUnM+jh0PMdQPFOnpiBIBMZZ0DmFz7JSA8\\\\\\\/8qJxA3MUnrbxjl+56KOxSi8wD1N97gULTNL5qqhCJY67JER4HHoyKOTeWpx0zs\\\\\\\/+\\\\\\\/C01zxLpI1o9PtkuLuKqC1QTQWKNgc27lmg8q7oL\\\\\\\/uawQ+dgyNXsYWWDoLOFByKSG9E0\\\\\\\/RYhBVU9DAcKTT84EDXsDHhuvQwygGxL9ZfFFTreLHqfW7w+pRkcn13Dgva3Uwv\\\\\\\/qQkYk4I1s+VaNFtHzlYjPTzWMk5LO8bPoVUX2Jodd2syhmPDkFHw2002laU6sQ0LkocjQF47pfsvPCLfffnkMW4Y5B8fqPq3Brh+298Y2f+Hft9kv1dqYKfCnwVDT3OAU+dbEulBgwnmF8CbN\\\\\\\/rsr1+xfCCGdnbe4wWUZPtr6sAYdh8t31QdRC\\\\\\\/tcCnIfsIYtE3nc7BD7GjPDHMJVyM4IEMeg1dQbBqFg1uHXuC65PJPEVO4CAiXV\\\\\\\/+7b8eTTrkfmqfF38K3j38S\\\\\\\/NKwLxF8le6VKd0m+ukL1qVX+Ljh3cnHPRp72kge\\\\\\\/nUrNxOLLup\\\\\\\/1S8uE7If2EgfTT3VuIRbmKpVLdbJsJYSMbofXrZDiorJmGZuf+FvGKrjG9ZwyuDkJyc7xfMFi1yIXf2q9DacgMiJWwoqPayF6wkFpme56L6q9IJTHxRn1CpNejwG7Ji5il0BWLapNtYjBOPLm8KSGswjFhihXjtqwWtDpco+KBK\\\\\\\/64cWKoYGOqLaqPYxQjksndET09qU1Bl+ENaMDuejj+Vuj0ewGekvv+tbfiQouVX138bRCK+W6Slu6e9nGROb93f+wyEG08I6FGiOw9ZQOcn3WjTNpLUmGFFZwD5gpr8RwHJQN+2GLN+HC0tYqHtzmGr+Ns+w9sj0X60yyeckBv7aCFvAcb92GB0ZZT+FMfGneRa0sYML4nK8aLngGpT3Cb1\\\\\\\/lxRql+0xwuPoGzLdiYb5uMNZLTzSsQJqrrCR0PCz2UsGZ2hq5INs5L+Me6Cexz+ZTuhgGGgWm628tL8ld5VY7FXfL96SPbjGMSt3eA8i3oygSVOVC+4MLzC\\\\\\\/ddciXDg1sPzD68WNOBr6tnLLiQFZPNgOhS7UD2aWyBv+\\\\\\\/Y81fhXQVrIJ60to6oLEg8JdPqXf1cD53i2bZwYLnjvAYjQarZpSySiLL9hCytcux1yylS1mJUwcqjRhJERKNe1xLkdvT4qLIL2nGcv1ccnsWgLpvOCgTgE5FrBIP8R8eANS64nbRqpFpx380mbRZjid\\\\\\\/LjiRlS03z0Cdrwvruug\\\\\\\/HJX2NQfadgQN3oj9pZdtrDEzLhToWa8Csk+AtdbFcEVOFfhujAt7P7c435siOE5X2tgILP8C2svdWPY8EgMYKqMdDASFSphyjocvFji8JjyoikOXchuyGAHBt3PopyUNtTRh1G8Ldpfpqk2zrZp6T31BtYmpSpiV\\\\\\\/nwWju+cHNXElDSUd2zZ3FrHNEOGRL75o53iV7My3B0spEcnaCf8uk3klXI9pmhjkar2Sw+snXTfqUGTIkS+iP\\\\\\\/nC4kipm7krlIL5QAD7Rff3mo27y4lUcpqHob6JvOiZPNeaekxlMSE58ryStvWBXnBJJSDIVgZ0l3dNMxeKBPe6sgei96s2G32r4x1HyUtfXVAC3vrgKXvBIe6vY6ROfNKb5HRmi1DFkGMd9E0WD8hNZJq3deP\\\\\\\/RV5kF2sHjyp7UXAGGG5zGgyUKq+HmgrZx32iasHmk38OrnrZ5OOZOaUZE9iZfaMt8p6oVzEaIHvikXbmG9FEVSPg3X5zXkKShT9ClBMKHknTowfb+PvkmAtAHYrFMlVmzMLZxYMatpKvWXVTHwk8xCJM\\\\\\\/QogtNGMNpCQ4qLaNS+eM3DU10HjV0ggtS\\\\\\\/SMuy2w9IQLNMSAv3XmhqIYlICJawkqs9dVbP47+nsYf\\\\\\\/DJlHboxqtnpWR6oVRYnj4DYorLYj4F2li6Kx4K3SwlkBKojPe09id2+YsiTE0UQ1TX68Uej3dc\\\\\\\/ZEhukhrHDC8pRst1lr41GFCqwLOI2hcsJGq2bhao6jSbfQuGPUnxbMwhjZdtq9tNSDqL4R\\\\\\\/UjVcPVWuDvgJrYa+ZEkEGx+pBj0uJ+phqYNmEdicGXt2IQep0EdGQLhHnD+ubHkzYkfrwPlZC3adPhGfRga1cxbdJIAJzmK8Et0EaDrqoikA8zY4LGh8wuKWcrnIaSVAciL\\\\\\\/1vdg2zEnoAUWBx3aEjJTbFOCPZmuv4oT\\\\\\\/HKXFPcI90wa4p+FzPMX1W2mXj4UtfT\\\\\\\/039CzazmpZncQ0JvzUFWuPdopnzcf7Kouu5V94nFTFAiWIRKDSqUm+ylHj3GiDRNDJv95YqxjoRTiPjD7I\\\\\\\/jtoO2MQt\\\\\\\/0dFbnl7SA+qOTlZyuzedoDW1ydjop\\\\\\\/HKFeysMQrxtPttg+Sboqbiy4yxgqY07td0NojsjQBXDX5EpALA\\\\\\\/1p+OvnH7QrkOB3AwTUIs9E3tG3mhP3aUONHB6B0hDYC4hlR9tBZzILN6jVZ+FvHlpojXt7XwfOC+v2M\\\\\\\/s4bVtI2uIqnFhWEauerqr7XYRFJCZOPmZGPp4xGbR5y8PICfhrmUgYI1DCRrDceVygjrwiUYeNfl2Jjv+epHJ8Fh4WOLKPjQhFVKco7Egxq1GOlMU0t+bXyjxJVMrlkQe+JVRpPCuJOuAmP1jz8kI1zGvfsw6CBwSz5xVFnF87lWDz\\\\\\\/jFP2VgwvtWYaOh4\\\\\\\/K0p9Za42RkneWM4evOUqCL8r98GoRycxhvBQ1vvx0NzXwzKdSDtam+LDBZ9jEM6Hks4eA0GjTwiEqvMrZ9ZyP0sDMy3ToUmysMbeFw2iCT7GgELhZ8oSvYZwPXPQNSStIE6IzoYcCUs96kmP3M0fDyooES4psE38QDSRZvGixRuXixrel0UwesoZV6w7TN22SnMNHQzQAEpZ94AoL3fnlT2Fax5N5nNmtgmL2f15mHbXqQkfFpPGr+zum6\\\\\\\/afRtLrX3x5eQOmVEOWJXzMyh\\\\\\\/rl1hwmP8kXbmU5wUxOkZb5mFb2DPIanVBpqJHBqVse0EBEpCT45A31dN3ezxlKBhta6ozSsNN4S0RcRX5Ogu2hWjVvOjjM7uUz4DLzlDBcaf7y9YSF+J67FaKV\\\\\\\/3V17vW9uHdod9ZAif3LXRMQtbJS6BNCHwB9fBzBmhi8feSCcgnqKV7vCSSQ\\\\\\\/\\\\\\\/390+eCV9ck6zhZVRXTdkMliAYL940Za+aglDVD1y58\\\\\\\/stBNvFw0HI7tORVLc3GwcwVp8FUEHNsX3JzKRWu0CxxhIadK3UruUHnw4WVKsbCpoO2EiL5\\\\\\\/uVQlictdGVqURJoq1vKQLuncMfiPM0Cwiv6Nbf5uN2lQ09QVh4pdBJLCv3TAv6XGUsX12cLkSPGrOYF8aqvtwXwdegDYYPHXXTxgV9CwY4yZ8L8TuTGWrRltb0LtpCuPMscgXO7IpVdNDI8oT03G654drCH7ZsklIcMhBc0mKAOToTTYry6kOVMimisp83D2BbSimhdnI9bT6KzRQD+ANQqxyb80MOXFFf0MAjOonZUGbVQw+ZeA\\\\\\\/QndYs99JjsHV1kmK+O+mZjaDJHuyWUN8Rh52v3WHStDIm8JwlF\\\\\\\/apNFSn2LAIKhqf\\\u0022,\\\u0022iv\\\u0022:\\\u0022zR\\\\\\\/25BemcI595WnRIDS88w==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022b3b58f21de8a8a71744516575286a37d35b3b0402550d5c2e1b6740674c6b8ce\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022BDMqNy3FVBghzyBPvtoZCbqErTDwylfR42nvAt\\\\\\\/dyHI3qxNnRNFMVol6wCkAdyAVcETr9KbOKbDatlB5qYBJSzPh+sSktyR6T5u6MMMT\\\\\\\/QheVbkUPlYjMsy1LEiGHPfG9XJQJLBuJ2wO4cfxRR9oIbmUXtEsUdOUHoWG\\\\\\\/BeZY5DIxgV\\\\\\\/xygagaKhzmfXoV1SpNnvMcJ5JvWZ+v8ZlOKw2jg+h7lbRqhr58eePruDcEBcYExsmkHpJueN2mn5ndeRHb9tpVWZjKTEwgsE3g1cdoL2WdsOfiNGjSiwWae05300mkGnieJJiDnZw\\\\\\\/isJpiu2U7gMVygGFw\\\\\\\/u8NYauHUas7xxLjQsJv76g8XOfbNdvR0wSm5jexs4kbjCb7X3bcO+zvPhyCan1z71Udff1OuRDAhEkrNOywsIkeJHJ74Qb\\\\\\\/eO1jm3TMtkWzWpi7jNCGVNfb+ogQR\\\\\\\/6zXce8dshl5Kdn12tWrqkws5l8afYyJSfJNsy\\\\\\\/4WiOYwd0bmgAgohzOK+SNjTK+S8+58JQoYGptAf4KeSTbt6bBI44mp8wh8pL0ihOKW5Bq6Qf79iERi5MbgTSTXQ9I78r7zMrlac8GIqOAIC00cGhXjXK4KCY\\\\\\\/3HVQOjgFT0UDEw2A55wjzQdPFsSieYYlUmIJ09nazgmjT7MLhKhzEuqzXbiQxjXzKnmgx2Zk38c+HbYqz3mP\\\\\\\/dwun6KVFnuKRgYu8e7DmxM8BKnDNr6F57Uyi1GX1HGLhNr\\\\\\\/spw7t9szTxANuGKLIdvkY6THbIbRYxPuxFiB+qgTi\\\\\\\/Y\\\\\\\/4DAAMFkdwc5GYbF9\\\\\\\/\\\\\\\/WNnz+AeJrCjutMaaRbIuctPgFWp8F0Bfg6oiM+RoyZz7me8HdCCxyY9SwjI9DHfSXAzY8SdOYJmVhYFKe0wAV+EMpUT0NiHkQm2LFwfWXVS0YvTZDFOCE+W6JT4JTY0YixgNuKzCirUkDMbl7+On1NpcCpHa3Lrcnc4\\\\\\\/SPFqu5DDBtIHxiC4vpekIDpaJUev7p4vVIyi6JIWoh5fVOLA1fvN9jWDi3lZ3jtfs6ZKADhd0YxZwG9T1eDTSG1+eG\\\\\\\/rjiNVgiLZPEeZLxtf\\\\\\\/tMCiVx8h8PvG03XHc+q5MazBELrCixNm4XGX5cVYMEeMteDQG9qS7Fw825O8WniXYGVVkFHyKvxJ+n8SHNVXnaCWux0Qoqk+BzntY2iRD\\\\\\\/a+ppdY2FV0uas+SeXb0qkSEpQS3RV1f9zkj5eN7pdA88rg1FVJkFFq4C1HDUI7rtvydqi1UzqwAsxSV\\\\\\\/kBb3Uco3avELxLXH+\\\\\\\/0OwOTzfMeTscUTgghY9ECKGvhFiPAeid3MS04c3SqYoMIaM8fpdBN7XZ2zQ5xvU9+mjR+Fa+zLPhj9dJia\\\\\\\/DwiOX5VvAGdNZckQw\\\\\\\/L24noHJv8cnOmVkI0bsCeaOVKy7Hbbxy+p1+WznzdiClKdlHpl0UPmtYdRiyJ3LHHZh8XypJZTsSnYBnG8cM++X0yJZWhrJ3h7YdRLDiHQyEdNcDXnygcZc2ZOUnWryc\\\\\\\/PsFSSV6r7DqEaxYzD2P0nKikZ5OokcWTaFAJoeWXE5D6MgS\\\\\\\/tEC1n4yoGesNVryB0jEEqAqCoWRph4zxDPqXfRMjsopjA51jlG3gHrh8OxRbVDvZbgoFD1v1lXIHzZ\\\\\\\/eMQMQz61HrHlVMxN2+rvZcIZGZlbIJxzIUHgUsY9BvF53aDMRii00kfrehPNQdH0PK0ijTyuntYO\\\\\\\/1eXqJPClSk5y0QkNq2s9bYZSjTOc5uwGONUnKrQxr9fJJZ8xvRWMS8KdXV5tvYNYxF\\\\\\\/iFBo9rHlFYiYbsUfJnSR5wHx2znxtupRPIjoZqDlRe4ECBuyWIAa1CoX8bo30QmiPQfhnQ2hREG\\\\\\\/kCoKTVSPy5YJvrSdkRnoaJb5B1gm63xpCQLjvGZwDwHUKN4sF4hg0ERpO2KGTqyCXB\\\\\\\/jHeeIv4V35aroFZ8lU0q\\\\\\\/QRBRnnMKGSrPJNYRZIXqH7ghGhX9io3jcv62rn7o7XsbHHypMPRI6sAejyGyCUSaaT61icN3oI9NJ7atLRp8rIwpiG+uycla\\\u0022,\\\u0022iv\\\u0022:\\\u0022esdWPhc5XbfoJ9OWR\\\\\\\/M59w==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022bf870803f9be35909f07a8a25063a237dab941ff510c8ae21265f7358b62c0e5\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459316 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/2/9/1035" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-1FUxCr4gWEZee0rR3yoRR1dY-k1fOIA35iVEkG2Lyho" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/2/9/1035" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-F-Ed8agqjgpN9HzJl7RA_2nfQd6fOS_XyOCSg63AeNA" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/2/9/1027" title="Tandem-Repeat Patterns and Mutation Rates in Microsatellites of the Nematode Model Organism Pristionchus pacificus" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/2/9/1041" title="The Lsktm1 Locus Modulates Lung and Skin Tumorigenesis in the Mouse" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459316" id="node459316" data-pisa="ggg;2/9/1035" data-pisa-master="ggg;2/9/1035" data-apath="/ggg/2/9/1035.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Location-Dependent Empirical Thresholds for Quantitative Trait Mapping</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Jason LaCombe</span>, <span class="highwire-citation-author" data-delta="1">Benjamin McClosky</span> and <span class="highwire-citation-author" data-delta="2">Steven Tanksley</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2012 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 2 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1035-1039;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.003517 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459316" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jason LaCombe </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Nature Source Genetics, Ithaca, New York 14850</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jason%2BLaCombe%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=LaCombe%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJason%2BLaCombe%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jlacombe@naturesourcegenetics.com" class="" data-icon-position="" data-hide-link-title="0">jlacombe@naturesourcegenetics.com</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Benjamin McClosky </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Nature Source Genetics, Ithaca, New York 14850</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Benjamin%2BMcClosky%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=McClosky%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABenjamin%2BMcClosky%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Steven Tanksley </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Nature Source Genetics, Ithaca, New York 14850</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Steven%2BTanksley%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tanksley%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASteven%2BTanksley%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/2/9/1035" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459316" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459316/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/2/9/1035.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459316" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459316/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/2/9/1035.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459316" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459316/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/2/9/1035.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459316" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459316/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/2/9/1035.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459316" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459316/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-4">The Churchill-Doerge approach toward constructing empirical thresholds has received widespread use in the genetic mapping literature through the past 16 years. The method is valued for both its simplicity and its ability to preserve the genome-wide error rate at a prespecified level. However, the Churchill-Doerge method is not designed to maintain the local (comparison-wise) error rate at a constant level except in situations that are unlikely to occur in practice. In this article, we introduce the objective of preserving the local error rate at a constant level in the context of mapping quantitative trait loci in linkage populations. We derive a method that preserves the local error rate at a constant level, provide an application via simulation on a <em>Hordeum vulgare</em> population, and demonstrate evidence of the relationship between recombination and location bias. Furthermore, we indicate that this method is equivalent to the Churchill-Doerge method when several assumptions are satisfied.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/qtl" class="hw-term hw-article-keyword hw-article-keyword-qtl" rel="nofollow">QTL</a></li><li class="kwd"><a href="/keyword/empirical-threshold" class="hw-term hw-article-keyword hw-article-keyword-empirical-threshold" rel="nofollow">empirical threshold</a></li><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/adjusted-p-value" class="hw-term hw-article-keyword hw-article-keyword-adjusted-p-value" rel="nofollow">adjusted <em xmlns:default="http://www.w3.org/1999/xhtml">P</em>-value</a></li><li class="kwd"><a href="/keyword/local-error-rate" class="hw-term hw-article-keyword hw-article-keyword-local-error-rate" rel="nofollow">local error rate</a></li><li class="kwd"><a href="/keyword/multiple-hypothesis-testing" class="hw-term hw-article-keyword hw-article-keyword-multiple-hypothesis-testing" rel="nofollow">multiple hypothesis testing</a></li></ul><p id="p-5">The accurate detection of significant quantitative trait loci (QTL) remains an area of active research in the genetics mapping community (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Falke and Frisch 2011</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Manichaikul <em>et al.</em> 2007</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Müller <em>et al.</em> 2011</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Wei <em>et al.</em> 2010</a>). In studying linkage populations, the process of performing a genome scan to detect QTL may be interpreted within a hypothesis-testing framework. Traditionally, at each point in the genome a statistical test is performed, resulting in a <em>P</em>-value representing the evidence for the presence of a QTL at that location. As the number of analysis points grows, so does the number of statistical tests performed, and the chance of incorrectly declaring a QTL at a given marker (committing a Type-I error) increases.</p><p id="p-6">The multiple-testing problem is well studied in both the scientific and statistical literature (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Dudoit and van der Laan 2007</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Falconer and Mackay 1996</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Lehmann and Romano 2005</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Lynch and Walsh 1998</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Seaman and Muller-Myhsok 2005</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Storey and Tibshirani 2003</a>). While several different types of error rates may be considered (<em>e.g.</em>, the false positive rate or false discovery rate), this article is concerned with methods that provide control of the genome-wide error rate (GWER).<a id="xref-fn-2-1" class="xref-fn" href="#fn-2"><sup>2</sup></a> Of such methods, by far the most widely used is the permutation-based method of <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Churchill and Doerge (1994)</a>.</p><p id="p-7">We first briefly review the steps involve in the application of the Churchill-Doerge (CD) method (<a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Churchill and Doerge 1994)</a>. For demonstration purposes, we take the example of single marker analysis, although the discussion is also applicable to interval mapping. For each marker, a user seeks to compare the observed log-of-odds (LOD) score with some underlying distribution in order to calculate a <em>P</em>-value representing the level of statistical evidence present for a genetic effect. <a id="xref-ref-3-3" class="xref-bibr" href="#ref-3">Churchill and Doerge (1994)</a> propose comparing the LOD score at a marker to the empirical distribution of the maximum LOD score across the genome, under the null hypothesis of no QTL present in the genome (or a QTL present but not linked with the given marker.) The empirical distribution of the max LOD score under the null is estimated by repeatedly sampling the magnitude of the max LOD score across random permutations of the phenotype data.</p><p id="p-8">The CD method is attractive for several reasons. First, the method is simple to apply and is currently implemented in a range of popular QTL analysis software packages: R/qtl (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Broman <em>et al.</em> 2003</a>), GenStat (VSN International 2011), MapQTL (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Van Ooijen 2009</a>), QTL Cartographer (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Basten <em>et al.</em> 2002</a>), etc. Second, the method is empirically driven, and so will more accurately represent the characteristics of the observed data than other methods relying on parametric assumptions. Finally, the method can be demonstrated to strongly control the GWER at a prespecified level.</p><p id="p-9">However, under certain conditions, the CD method may result in upwardly or downwardly biased estimates for the Type-I error in specific regions of the genome—a property we refer to as location bias. Such behavior may be expected to occur when the location-conditioned distributions of the LOD (not max LOD) score are not identical. Several relevant examples are presented in <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Peirce <em>et al.</em> (2007</a>, <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">2008)</a>, and the respective authors provide evidence for the consideration of linkage as a possible causal factor of location bias.</p><p id="p-10">The primary focus of this article is in deriving a multiple-testing procedure that not only preserves the GWER but also maintains the local error rate at a constant level across the genome. The method presented possesses the property that Type-I errors are no more likely to occur at a given locus than at an alternative. For simplicity, we confine our attention to the technique of marker regression, although this work is easily extended to any approach in which a finite number of single hypotheses are considered (<em>e.g.</em>, interval mapping on a finite grid).</p><p id="p-11">The derived method is motivated by the following considerations. Suppose that the distribution of the maximum LOD score is not identical across the genome. That is, for two non-identical markers, it is possible that the distribution of the maximum LOD score at the first marker is not identical to the distribution of the max LOD score at the second marker. For a given marker, we propose a comparison of the associated LOD score with the distribution of the maximum LOD score at that marker. Thus the comparison is with a conditional—rather than a marginal—distribution.</p><p id="p-12">Proceeding with the comparison of the observed LOD scores with the appropriate conditional max LOD distributions, we seek to construct a threshold that preserves the GWER at a prespecified level, while maintaining the local error rate at a constant level.</p><div class="section methods" id="sec-1"><h2 class="">NOTATION AND METHODS</h2><p id="p-13">In this section, we introduce the mathematical notation relevant to the remainder of this work, and we present a derivation of the proposed method.</p><p id="p-14">Denote the number of markers in the genome scan as <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/9/1035/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/9/1035/embed/mml-math-1.gif"/></noscript></span></span>, and define <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/9/1035/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/9/1035/embed/mml-math-2.gif"/></noscript></span></span> to be the number of individuals for which we possess phenotype information. Throughout the remainder of this article, we assume that the observed genotype data are complete (<em>i.e.</em>, no missing values); a discussion on the implications of missing data are presented in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003517/-/DC1/FileS1.pdf">File S1</a>. We represent an observation of the maximum LOD as <em>M</em>, where <em>M</em> is defined as the pair <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/9/1035/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/9/1035/embed/mml-math-3.gif"/></noscript></span></span>, with <em>δ</em> the magnitude of the max LOD value and <em>λ</em> the location (marker index) of the max LOD value. A threshold <em>T</em> is defined as a sequence of non-negative values indexed from 1 to <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/9/1035/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/9/1035/embed/mml-math-4.gif"/></noscript></span></span>.</p><p id="p-15">We next present the candidate method—referred to as the location-dependent threshold (LDT) method—for constructing empirical thresholds similar to those of <a id="xref-ref-3-4" class="xref-bibr" href="#ref-3">Churchill and Doerge (1994)</a>, while accounting for local properties of the genome. Our goal is to construct a threshold <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/9/1035/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/9/1035/embed/mml-math-5.gif"/></noscript></span></span>such that for a desired GWER <em>γ</em>, <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/9/1035/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/9/1035/embed/mml-math-6.gif"/></noscript></span></span> The LDT method relies on generating observations of <em>M</em> under random permutations of the observed phenotypes. However, unlike the CD method, which relies on the marginal distribution of <em>δ</em>, the proposed procedure necessitates the derivation of estimates of the empirical distribution function of <em>δ</em> conditioned on <em>λ</em>. Estimating these conditional distributions requires that we generate many more samples (the number of samples denoted as <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/9/1035/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/9/1035/embed/mml-math-7.gif"/></noscript></span></span>) than the Churchill-Doerge procedure, enough so that <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/9/1035/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/9/1035/embed/mml-math-8.gif"/></noscript></span></span> for all <em>i</em>. Although the computational burden is significantly increased, the additional resources are needed to accurately estimate and account for location bias.</p><p id="p-16">To begin, let <em>T</em> be an arbitrary threshold and fix <em>γ</em> at the desired GWER. For each <em>i</em> define <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/9/1035/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/9/1035/embed/mml-math-9.gif"/></noscript></span></span>, and a function <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/9/1035/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/9/1035/embed/mml-math-10.gif"/></noscript></span></span> such that <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/9/1035/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/9/1035/embed/mml-math-11.gif"/></noscript></span></span> is the unique value satisfying</p><span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/9/1035/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/9/1035/embed/mml-math-12.gif"/></noscript></span></span><p id="p-17">Note that, for a collection of <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/9/1035/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/9/1035/embed/mml-math-13.gif"/></noscript></span></span> markers, <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/9/1035/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/9/1035/embed/mml-math-14.gif"/></noscript></span></span> exists if and only if <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/9/1035/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/9/1035/embed/mml-math-15.gif"/></noscript></span></span> otherwise, the equality in the above equation does not hold. Let <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/9/1035/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/9/1035/embed/mml-math-16.gif"/></noscript></span></span> be a sequence of location-dependent threshold values such that <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/9/1035/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/9/1035/embed/mml-math-17.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/9/1035/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/9/1035/embed/mml-math-18.gif"/></noscript></span></span> Then <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/9/1035/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/9/1035/embed/mml-math-19.gif"/></noscript></span></span>, the sequence generated by the evaluation of <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/9/1035/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/9/1035/embed/mml-math-20.gif"/></noscript></span></span> for all <em>i</em>, is a threshold that preserves the GWER at level <em>γ</em> (see <em>Appendix</em>).</p><p id="p-18">The role of the function <em>H</em> is to adjust the threshold in the presence of location bias. For example, when <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/9/1035/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/9/1035/embed/mml-math-21.gif"/></noscript></span></span> is greater than one, there exists a positive bias away from location <em>i</em>; by dividing by <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/9/1035/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/9/1035/embed/mml-math-22.gif"/></noscript></span></span> we are shrinking the numerator in <a id="xref-disp-formula-1-1" class="xref-disp-formula" href="#disp-formula-1">Equation 1</a> toward zero, and so must decrease the threshold accordingly. Thus, we adjust for the presence of a positive location bias by decreasing the corresponding significance threshold. Similarly, we adjust for the presence of a negative location bias by increasing the threshold.</p><div id="sec-2" class="subsection"><h3>Example of controlling the local error rate</h3><p id="p-19">As an example, suppose we observe genotype information at 34 markers for 200 individuals from a backcross population. A demonstration of the presence of location bias in this synthetic population is given in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/9/1035/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Plot of the empirical location distribution of &#x3BB; for a single chromosome of a synthetic double-haploid population of 200 individuals, with 34 markers. Observe that the distribution of &#x3BB; has a tendency toward smaller probabilities in areas of dense markers, while &#x3BB; has larger probabilities in areas of sparse markers." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1553014608" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Plot of the empirical location distribution of &lt;em&gt;&#x3BB;&lt;/em&gt; for a single chromosome of a synthetic double-haploid population of 200 individuals, with 34 markers. Observe that the distribution of &lt;em&gt;&#x3BB;&lt;/em&gt; has a tendency toward smaller probabilities in areas of dense markers, while &lt;em&gt;&#x3BB;&lt;/em&gt; has larger probabilities in areas of sparse markers.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/9/1035/F1.medium.gif" width="440" height="289"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1&#x2002;" src="https://www.g3journal.org/content/ggg/2/9/1035/F1.medium.gif" width="440" height="289"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/9/1035/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/9/1035/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461786" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1 </span> <p id="p-20" class="first-child">Plot of the empirical location distribution of <em>λ</em> for a single chromosome of a synthetic double-haploid population of 200 individuals, with 34 markers. Observe that the distribution of <em>λ</em> has a tendency toward smaller probabilities in areas of dense markers, while <em>λ</em> has larger probabilities in areas of sparse markers.</p><div class="sb-div caption-clear"></div></div></div><p id="p-21">Consider the problem of constructing a 20% genome-wide threshold for the sample of maximum LOD scores corresponding to this population; such a construction is given in <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>. The figure illustrates the difference between the CD method (the construction of a constant genome-wide threshold that places 20% of the observed max LOD scores above the line) and the LDT method, which distributes 20% of the observed max LOD scores above the line, while assigning a constant percentage above the threshold at each marker, and thus varies with location.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/9/1035/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Plot of the LOD score constructed for the synthetic population associated with Figure 1. Thresholds preserving the GWER at 20% are displayed for both the CD method and the LDT method. Note that the CD threshold remains constant across the genome, while the LDT threshold varies with location." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1553014608" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Plot of the LOD score constructed for the synthetic population associated with Figure 1. Thresholds preserving the GWER at 20% are displayed for both the CD method and the LDT method. Note that the CD threshold remains constant across the genome, while the LDT threshold varies with location.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/9/1035/F2.medium.gif" width="440" height="280"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2&#x2002;" src="https://www.g3journal.org/content/ggg/2/9/1035/F2.medium.gif" width="440" height="280"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/9/1035/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/9/1035/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461797" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2 </span> <p id="p-22" class="first-child">Plot of the LOD score constructed for the synthetic population associated with <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>. Thresholds preserving the GWER at 20% are displayed for both the CD method and the LDT method. Note that the CD threshold remains constant across the genome, while the LDT threshold varies with location.</p><div class="sb-div caption-clear"></div></div></div><p id="p-23">Although the CD method controls the GWER in a marginal sense, <em>e.g.</em>, <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/9/1035/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/9/1035/embed/mml-math-23.gif"/></noscript></span></span>, it is not difficult to show that the CD method does not control the joint location-magnitude in a local sense (in the presence of location bias.) For the proposed method, the GWER is preserved both in a global and local sense with regard to the joint location-magnitude:<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/9/1035/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/9/1035/embed/mml-math-24.gif"/></noscript></span></span></p></div><div id="sec-3" class="subsection"><h3>Physical linkage</h3><p id="p-24">In accordance with <a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Peirce <em>et al.</em> (2008)</a>, one potential justification for the presence of location bias is the existence of physical linkage between markers, <em>i.e.</em>, differences in recombination rates between markers. We illustrate the plausibility of such a hypothesis by considering the following experiment.</p><p id="p-25">Suppose that we have a set of 10 equally spaced markers, representing observations for 200 individuals from a backcross population. Consider a sequence of 11 simulations, such that for each simulation the recombination rate between each marker is incremented by 0.03 units (starting at a recombination rate of 0.01.) Thus for the first simulation, we observe 10 markers, with recombination rates of 0.01, while in the final simulation, the rates are increased to 0.31. For each simulation, we compute vectors of the alpha-values for 300 independent samples of genotype data. For each individual sample, we perform a Chi-squared test for deviation from the uniform distribution and present the associated box-plots in <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>. The results support the hypothesis that as the recombination rate increases, the less severe the deviation of the alphas vectors from uniformity (the presence of location bias decreases.) As the LDT method accounts for location bias, we can then make the case that the method at least partially accounts for differences in recombination rates between markers.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/9/1035/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Plot of the relationship between recombination rate and the presence of location bias. The figure illustrates that as the recombination rate between markers increases, the deviance from uniformity decreases, indicating a decrease in location bias. The red line indicates the 95th percentile for a Chi-squared statistic with nine degrees of freedom." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1553014608" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Plot of the relationship between recombination rate and the presence of location bias. The figure illustrates that as the recombination rate between markers increases, the deviance from uniformity decreases, indicating a decrease in location bias. The red line indicates the 95&lt;sup&gt;th&lt;/sup&gt; percentile for a Chi-squared statistic with nine degrees of freedom.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/9/1035/F3.medium.gif" width="440" height="297"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3&#x2002;" src="https://www.g3journal.org/content/ggg/2/9/1035/F3.medium.gif" width="440" height="297"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/9/1035/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/9/1035/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461804" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3 </span> <p id="p-26" class="first-child">Plot of the relationship between recombination rate and the presence of location bias. The figure illustrates that as the recombination rate between markers increases, the deviance from uniformity decreases, indicating a decrease in location bias. The red line indicates the 95<sup>th</sup> percentile for a Chi-squared statistic with nine degrees of freedom.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-4" class="subsection"><h3>Barley example</h3><p id="p-27">In this section, we illustrate the performance of the proposed method applied to a double haploid empirical barley population derived from a cross of ‘Steptoe’ and ‘Morex’ (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Kleinhofs <em>et al.</em> 1993</a>). Phenotype and genotype data were downloaded from the GrainGenes website (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Mather 1995</a>). For this example, we considered the average of grain yield across 16 environments (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">P. M. Hayes <em>et al.</em> 1993</a>) for 150 double haploid lines, with genotype data observed for 223 markers set across seven chromosomes.</p><p id="p-28">For the comparison, a single sample consisting of 1,000,000 observations of <em>M</em> was generated and used to construct both the CD and LDT thresholds. <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a> indicates the presence of an observable location bias. We calculate thresholds at the 0.01, 0.10, and 0.40 levels for both the CD and LDT methods. The data (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003517/-/DC1/FileS3.csv">File S3</a>) and R code (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003517/-/DC1/FileS2.zip">File S2</a>) for the analysis are provided.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/9/1035/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Plot of the empirical location distribution of &#x3BB; for the average yield trait of the barley population." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1553014608" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Plot of the empirical location distribution of &lt;em&gt;&#x3BB;&lt;/em&gt; for the average yield trait of the barley population.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/9/1035/F4.medium.gif" width="440" height="291"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4&#x2002;" src="https://www.g3journal.org/content/ggg/2/9/1035/F4.medium.gif" width="440" height="291"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/9/1035/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/9/1035/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461782" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4 </span> <p id="p-29" class="first-child">Plot of the empirical location distribution of <em>λ</em> for the average yield trait of the barley population.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-5"><h2 class="">DISCUSSION</h2><p id="p-30">The results of the threshold constructions for the barley example are presented in <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>. We first observe that the LDT thresholds appear centered about the thresholds provided by the CD method. This behavior is consistent with Equation 1, to the extent that if the location-conditioned distributions are all identical, the average of the LDT threshold across the genome will be the CD threshold.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/2/9/1035/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Plot of the LOD score constructed for the average yield trait of the barley population. Significance thresholds at the 0.01, 0.10, and 0.40 levels are displayed for both the CD method (in red) and the LDT method (in blue). The red, dashed, vertical lines indicate the markers associated with the highest LOD peaks on chromosomes 1 and 2, respectively. The range of the plot is restricted to better illustrate the performance of the two methods." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1553014608" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Plot of the LOD score constructed for the average yield trait of the barley population. Significance thresholds at the 0.01, 0.10, and 0.40 levels are displayed for both the CD method (in red) and the LDT method (in blue). The red, dashed, vertical lines indicate the markers associated with the highest LOD peaks on chromosomes 1 and 2, respectively. The range of the plot is restricted to better illustrate the performance of the two methods.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5&#x2002;" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/2/9/1035/F5.medium.gif" width="440" height="279"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5&#x2002;" src="https://www.g3journal.org/content/ggg/2/9/1035/F5.medium.gif" width="440" height="279"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/2/9/1035/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5&#x2002;" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/2/9/1035/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/461703" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5 </span> <p id="p-31" class="first-child">Plot of the LOD score constructed for the average yield trait of the barley population. Significance thresholds at the 0.01, 0.10, and 0.40 levels are displayed for both the CD method (in red) and the LDT method (in blue). The red, dashed, vertical lines indicate the markers associated with the highest LOD peaks on chromosomes 1 and 2, respectively. The range of the plot is restricted to better illustrate the performance of the two methods.</p><div class="sb-div caption-clear"></div></div></div><p id="p-32">Although the CD threshold may be viewed as the approximate average of the location-dependent threshold, in the event that an investigator desires to construct adjusted <em>P</em>-values, <em>e.g.</em>, (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Lystig 2003</a>), the two methods produce different results. However, it is important to note that the LDT thresholds only preserve the GWER at levels in the range <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/9/1035/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/9/1035/embed/mml-math-25.gif"/></noscript></span></span> to 1.</p><p id="p-33">For example, consider the marker associated with the highest LOD peak on chromosome 2 (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>). The CD-adjusted <em>P</em>-value is 0.3176, while the LDT-adjusted <em>P</em>-value is larger, at 0.4352. Similarly, marker on chromosome 1 has a CD-adjusted <em>P</em>-value of 0.8370, while the LDT-adjusted <em>P</em>-value is smaller at 0.5676 (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5</a>). In comparing the relative evidence for a QTL linked to the marker on chromosome 1 <em>vs.</em> chromosome 2 (<em>i.e.</em>, the ratio of the adjusted <em>P</em>-values), the LDT method results in twice as much relative evidence as that derived by the use of the CD method.</p><p id="p-34">While the presented location-dependent methodology has the potential to provide more accurate QTL detection (in the sense of preserving the local Type-I error rate at a constant level), the methodology is not without criticism. First, it should be recognized that the number of required permutations (<span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/9/1035/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/9/1035/embed/mml-math-26.gif"/></noscript></span></span>is much greater than the number required for the CD method. In accordance with <a id="xref-ref-3-5" class="xref-bibr" href="#ref-3">Churchill and Doerge (1994)</a>, we suggest enough so that there are at least 1000 observations of the max LOD score at each marker location. Thus, while Churchill and Doerge suggest a sample size of 1000 for their method (to maintain precision at the 0.05 level), the LDT method requires a suggested number of <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/9/1035/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/9/1035/embed/mml-math-27.gif"/></noscript></span></span>samples to account for moderate location bias.</p><p id="p-35">Also, one might suggest that for markers that never or rarely attain a maximum LOD score, it is not reasonable to compare observed LOD score with the distribution of the maximum. Rather, an investigator might choose to compare the observed score to a mixture distribution of LOD score order statistics, <em>e.g.</em>, <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Simonsen and McIntyre (2004)</a>.</p><p id="p-36">Furthermore, as previously indicated, the LDT thresholds only preserve the GWER at levels within a range determined by the observed location bias <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/9/1035/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/9/1035/embed/mml-math-28.gif"/></noscript></span></span>. When controlling the GWER at levels that approach the maximum of this range, the magnitude of the LDT-adjusted <em>P</em>-values will be driven entirely by the location parameter, as opposed to a mixture of the location and magnitude parameters.</p><p id="p-37">Although the above criticisms are valid, it remains desirable to consider QTL detection methods—within the hypothesis-testing framework—that preserve both the GWER as well as the local error rate at a constant level. Failing to account for location bias leaves an investigator susceptible to a varying and unquantified level of bias in the estimation of the Type-I error rate for different regions of the genome. It should also be noted that, in the event that the location-conditioned distributions are identical and no location bias exists, the LDT threshold is equivalent to the CD threshold. More generally, the proposed location-adjusted <em>P</em>-values are obtained through simple linear transformations of the <em>P</em>-values for the location-conditioned distributions (see <em>Appendix</em>.)</p></div><div class="section conclusions" id="sec-6"><h2 class="">CONCLUSION</h2><p id="p-38">The method of <a id="xref-ref-3-6" class="xref-bibr" href="#ref-3">Churchill and Doerge (1994)</a> provides investigators with an approach to QTL detection that maintains control of the GWER at a prespecified level. However, the presence of location bias has implications for the accurate assessment of local Type-I error rates throughout the genome.</p><p id="p-39">The introduction of the LDT method provides investigators with an approach that controls the GWER, while also maintaining the local error rate at a constant level. Through simulation, we have demonstrated evidence that the method accounts for the presence of physical linkage. We have provided an application of the method to an empirical barley data set to better illustrate the method’s performance on data derived under realistic experimental conditions. Finally, we have observed that, in the presence of identical location-conditioned distributions and the absence of location bias, the method is equivalent to the approach of Churchill and Doerge.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-40">The authors thank the referees for their helpful comments and suggestions. In particular, both the extension of the analysis to the case of missing data and the inclusion of several additional references have contributed to the clarity and completeness of this work.</p></div><div class="section app" id="app-1"><h2 class="">APPENDIX</h2><p id="p-41">In this appendix, we prove that the LDT method preserves the GWER at a prespecified level <em>γ</em>. First, choose <em>γ</em> such that <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/9/1035/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/9/1035/embed/mml-math-29.gif"/></noscript></span></span>Then,</p><div class="section" id="sec-7"><p id="p-42"><span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/9/1035/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/9/1035/embed/mml-math-30.gif"/></noscript></span></span></p><p id="p-43">In addition, the adjusted <em>P</em>-values for the method can be represented as a point-wise linear transform of the <em>P</em>-values for the location-conditioned distributions. Let <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/9/1035/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/9/1035/embed/mml-math-31.gif"/></noscript></span></span> represent the LOD score observed at marker <em>i</em>. Then,</p><p id="p-44"><span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/9/1035/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/2/9/1035/embed/mml-math-32.gif"/></noscript></span></span></p></div></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003517/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.112.003517/-/DC1</a></p></li><li class="fn" id="fn-2"><p id="p-2"><a class="rev-xref" href="#xref-fn-2-1">↵</a><span class="fn-label">2</span> Known in the statistical literature as the family-wise error rate (FWER).</p></li><li class="fn" id="fn-3"><p id="p-45"><em>Communicating editor: D. J. de Koning</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2012-01-03"><span class="received-label">Received </span>January 3, 2012.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2012-06-23"><span class="accepted-label">Accepted </span>June 23, 2012.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2012 LaCombe <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-3">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-book" id="cit-2.9.1035.1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Basten</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weir</span>  <span class="cit-name-given-names">B. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zeng</span>  <span class="cit-name-given-names">Z.-B.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-source">QTL Cartographer</span>. <span class="cit-publ-name">Department of Statistics, North Carolina State University</span>, <span class="cit-publ-loc">Raleigh, NC</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1035.2" data-doi="10.1093/bioinformatics/btg112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Broman</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sen</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Churchill</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">R/qtl: QTL mapping in experimental crosses</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">19</span>(<span class="cit-issue">7</span>): <span class="cit-fpage">889</span>–<span class="cit-lpage">890</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DBroman%26rft.auinit1%253DK.%2BW.%26rft.volume%253D19%26rft.issue%253D7%26rft.spage%253D889%26rft.epage%253D890%26rft.atitle%253DR%252Fqtl%253A%2BQTL%2Bmapping%2Bin%2Bexperimental%2Bcrosses%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtg112%26rft_id%253Dinfo%253Apmid%252F12724300%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIxOS83Lzg4OSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzIvOS8xMDM1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1035.3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Churchill</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doerge</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Empirical threshold values for quantitative trait mapping</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">138</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">963</span>–<span class="cit-lpage">971</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DChurchill%26rft.auinit1%253DG.%2BA.%26rft.volume%253D138%26rft.issue%253D3%26rft.spage%253D963%26rft.epage%253D971%26rft.atitle%253DEmpirical%2Bthreshold%2Bvalues%2Bfor%2Bquantitative%2Btrait%2Bmapping.%26rft_id%253Dinfo%253Apmid%252F7851788%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTM4LzMvOTYzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvMi85LzEwMzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-book" id="cit-2.9.1035.4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dudoit</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van der Laan</span>  <span class="cit-name-given-names">M. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-source">Multiple Testing Procedures with Applications to Genomics</span>. <span class="cit-publ-name">Springer Series in Statistics</span>. <span class="cit-publ-loc">Springer, New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-book" id="cit-2.9.1035.5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Falconer</span>  <span class="cit-name-given-names">D. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mackay</span>  <span class="cit-name-given-names">T. F. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-comment"><em>Introduction to Quantitative Genetics</em>, Ed. 4. Pearson Education Limited, Essex, England</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1035.6" data-doi="10.1038/hdy.2010.87"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Falke</span>  <span class="cit-name-given-names">K. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Frisch</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Power and false-positive rate in QTL detection with near-isogenic line libraries</span>. <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">106</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">576</span>–<span class="cit-lpage">584</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.stitle%253DHeredity%26rft.aulast%253DFalke%26rft.auinit1%253DK.%2BC.%26rft.volume%253D106%26rft.issue%253D4%26rft.spage%253D576%26rft.epage%253D584%26rft.atitle%253DPower%2Band%2Bfalse-positive%2Brate%2Bin%2BQTL%2Bdetection%2Bwith%2Bnear-isogenic%2Bline%2Blibraries.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2010.87%26rft_id%253Dinfo%253Apmid%252F20683482%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2010.87&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20683482&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F9%2F1035.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1035.7" data-doi="10.1007/BF01184929"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">B. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Knapp</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Quantitative trait locus effects and environmental interaction in a sample of North American barley germplasm</span>. <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">87</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">392</span>–<span class="cit-lpage">401</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D87%26rft.spage%253D392%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF01184929%26rft_id%253Dinfo%253Apmid%252F24190268%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF01184929&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24190268&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F9%2F1035.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1993MK06900016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1035.8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kleinhofs</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kilian</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Saghai Maroof</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Biyashev</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">A molecular, isozyme and morphological map of the barley (Hordeum vulgare) genome</span>. <abbr class="cit-jnl-abbrev">TAG Theoretical and Applied Genetics</abbr> <span class="cit-vol">86</span>(<span class="cit-issue">6</span>): <span class="cit-fpage">705</span>–<span class="cit-lpage">712</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG%2BTheoretical%2Band%2BApplied%2BGenetics%26rft.volume%253D86%26rft.spage%253D705%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-book" id="cit-2.9.1035.9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lehmann</span>  <span class="cit-name-given-names">E. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Romano</span>  <span class="cit-name-given-names">J. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-comment"><em>Testing Statistical Hypotheses</em>, Ed. 3. Springer Texts in Statistics (series). Springer, New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-book" id="cit-2.9.1035.10"><div class="cit-metadata"><cite><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">M.</span></span>, <span class="cit-auth"><span class="cit-name-surname">Walsh</span>  <span class="cit-name-given-names">B.</span></span>, <span class="cit-pub-date">1998</span> <span class="cit-comment"><em>Genetics and Analysis of Quantitative Traits</em>, Ed. 1. Sinauer Associates, Sunderland, MA</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1035.11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lystig</span>  <span class="cit-name-given-names">T. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Adjusted P values for genome-wide scans</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">164</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">1683</span>–<span class="cit-lpage">1687</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DLystig%26rft.auinit1%253DT.%2BC.%26rft.volume%253D164%26rft.issue%253D4%26rft.spage%253D1683%26rft.epage%253D1687%26rft.atitle%253DAdjusted%2BP%2BValues%2Bfor%2BGenome-Wide%2BScans%26rft_id%253Dinfo%253Apmid%252F12930772%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2NC80LzE2ODMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8yLzkvMTAzNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1035.12" data-doi="10.1534/genetics.107.080093"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Manichaikul</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Palmer</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sen</span>  <span class="cit-name-given-names">Ś.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Broman</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Significance thresholds for quantitative trait locus mapping under selective genotyping</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">177</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">1963</span>–<span class="cit-lpage">1966</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D177%26rft.spage%253D1963%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.080093%26rft_id%253Dinfo%253Apmid%252F17720902%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/genetics.107.080093&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17720902&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F9%2F1035.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000251368800058&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-web" id="cit-2.9.1035.13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mather</span>  <span class="cit-name-given-names">D. E.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-comment">GrainGenes map data report: barley, SxM basemap. Available at: <a href="http://wheat.pw.usda.gov/ggpages/SxM/">http://wheat.pw.usda.gov/ggpages/SxM/</a>. Accessed July 27, 2011</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1035.14" data-doi="10.1038/hdy.2010.125"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Müller</span>  <span class="cit-name-given-names">B. U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stich</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Piepho</span>  <span class="cit-name-given-names">H.-P.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A general method for controlling the genome-wide type I error rate in linkage and association mapping experiments in plants</span>. <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">106</span>(<span class="cit-issue">5</span>): <span class="cit-fpage">825</span>–<span class="cit-lpage">831</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.stitle%253DHeredity%26rft.aulast%253DMuller%26rft.auinit1%253DB.%2BU.%26rft.volume%253D106%26rft.issue%253D5%26rft.spage%253D825%26rft.epage%253D831%26rft.atitle%253DA%2Bgeneral%2Bmethod%2Bfor%2Bcontrolling%2Bthe%2Bgenome-wide%2Btype%2BI%2Berror%2Brate%2Bin%2Blinkage%2Band%2Bassociation%2Bmapping%2Bexperiments%2Bin%2Bplants.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2010.125%26rft_id%253Dinfo%253Apmid%252F20959861%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2010.125&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20959861&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F9%2F1035.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1035.15" data-doi="10.1371/journal.pone.0001036"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Peirce</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Broman</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Williams</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">A simple method for combining genetic mapping data from multiple crosses and experimental designs</span>. <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">2</span>(<span class="cit-issue">10</span>): <span class="cit-fpage">e1036</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DPeirce%26rft.auinit1%253DJ.%2BL.%26rft.volume%253D2%26rft.issue%253D10%26rft.spage%253De1036%26rft.epage%253De1036%26rft.atitle%253DA%2Bsimple%2Bmethod%2Bfor%2Bcombining%2Bgenetic%2Bmapping%2Bdata%2Bfrom%2Bmultiple%2Bcrosses%2Band%2Bexperimental%2Bdesigns.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001036%26rft_id%253Dinfo%253Apmid%252F17940600%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0001036&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17940600&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F9%2F1035.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-book" id="cit-2.9.1035.16" data-doi="10.1371/journal.pone.0001977"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Peirce</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Broman</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chesler</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite> , <span class="cit-pub-date">2008</span> <span class="cit-article-title">Genome Reshuffling for Advanced Intercross Permutation (GRAIP): simulation and permutation for advanced intercross population analysis</span>. <span class="cit-source">PLoS One</span> <span class="cit-vol">3</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">e1977</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DPeirce%26rft.auinit1%253DJ.%2BL.%26rft.volume%253D3%26rft.issue%253D4%26rft.spage%253De1977%26rft.epage%253De1977%26rft.atitle%253DGenome%2BReshuffling%2Bfor%2BAdvanced%2BIntercross%2BPermutation%2B%2528GRAIP%2529%253A%2Bsimulation%2Band%2Bpermutation%2Bfor%2Badvanced%2Bintercross%2Bpopulation%2Banalysis.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001977%26rft_id%253Dinfo%253Apmid%252F18431467%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0001977&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18431467&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F9%2F1035.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1035.17" data-doi="10.1086/428140"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Seaman</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Muller-Myhsok</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Rapid simulation of P values for product methods and multiple-testing adjustment in association studies</span>. <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">76</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">399</span>–<span class="cit-lpage">408</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DSeaman%26rft.auinit1%253DS.%2BR.%26rft.volume%253D76%26rft.issue%253D3%26rft.spage%253D399%26rft.epage%253D408%26rft.atitle%253DRapid%2Bsimulation%2Bof%2BP%2Bvalues%2Bfor%2Bproduct%2Bmethods%2Band%2Bmultiple-testing%2Badjustment%2Bin%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F428140%26rft_id%253Dinfo%253Apmid%252F15645388%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/428140&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15645388&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F9%2F1035.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000226851900003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1035.18"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Simonsen</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McIntyre</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Using alpha wisely: improving power to detect multiple QTL</span>. <abbr class="cit-jnl-abbrev">Stat. Appl. Genet. Mol. Biol.</abbr> <span class="cit-vol">3</span>(<span class="cit-issue">1</span>): <span class="cit-comment">Article1</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1035.19" data-doi="10.1073/pnas.1530509100"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Storey</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tibshirani</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Statistical significance for genomewide studies</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">100</span>(<span class="cit-issue">16</span>): <span class="cit-fpage">9440</span>–<span class="cit-lpage">9445</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1530509100%26rft_id%253Dinfo%253Apmid%252F12883005%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTAwLzE2Lzk0NDAiO3M6NDoiYXRvbSI7czoxODoiL2dnZy8yLzkvMTAzNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-book" id="cit-2.9.1035.20"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Van Ooijen</span>  <span class="cit-name-given-names">J. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-source">MapQTL</span>, <span class="cit-publ-name">Kyazma B.V.</span>, <span class="cit-publ-loc">Wageningen, Netherlands</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-book no-rev-xref" id="cit-2.9.1035.21"><div class="cit-metadata"><cite><span class="cit-auth cit-collab">VSNInternational</span>, <span class="cit-pub-date">2011</span> <span class="cit-source">GenStat for Windows</span>, <span class="cit-publ-name">VSN International</span>, <span class="cit-publ-loc">Hemel Hempstead, UK</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-2.9.1035.22" data-doi="10.1038/hdy.2009.129"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wei</span>  <span class="cit-name-given-names">W. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Knott</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haley</span>  <span class="cit-name-given-names">C. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">De Koning</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Controlling false positives in the mapping of epistatic QTL</span>. <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">104</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">401</span>–<span class="cit-lpage">409</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.stitle%253DHeredity%26rft.aulast%253DWei%26rft.auinit1%253DW.%2BH.%26rft.volume%253D104%26rft.issue%253D4%26rft.spage%253D401%26rft.epage%253D409%26rft.atitle%253DControlling%2Bfalse%2Bpositives%2Bin%2Bthe%2Bmapping%2Bof%2Bepistatic%2BQTL.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2009.129%26rft_id%253Dinfo%253Apmid%252F19789566%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2009.129&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19789566&amp;link_type=MED&amp;atom=%2Fggg%2F2%2F9%2F1035.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000275898200011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/2/9/1035.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/2/9/1027" title="Tandem-Repeat Patterns and Mutation Rates in Microsatellites of the Nematode Model Organism Pristionchus pacificus" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/2/9/1041" title="The Lsktm1 Locus Modulates Lung and Skin Tumorigenesis in the Mouse" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/2/9"><span>Volume 2 Issue 9, September 2012</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/2/9.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/2/9.cover-source.jpg?itok=JMb6jwWa" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 2 (9)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/2/9/1035" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-84Gq69JSkwU9SwlGGs74m6VcyW0I0MSpykWDf19Zk5Y" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/2/9/1035" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/2/9/1035" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/2/9/1035" class="active" data-icon-position="" data-hide-link-title="0">Location-Dependent Empirical Thresholds for Quantitative Trait Mapping</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459316" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-3HavIaktSZAOOYwsqSvz7IcRRov1X0SaauVwOGiyqyI" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/2/9/1035.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/2/9/1035" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-s1Rpl7fW_aBIWoXBxpr8ggZ7sqiKTlxI_AbFPbAmFEE" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/2/9/1035" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/2/9/1035.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459316" id="node459316--2" data-pisa="ggg;2/9/1035" data-pisa-master="ggg;2/9/1035" data-apath="/ggg/2/9/1035.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Location-Dependent Empirical Thresholds for Quantitative Trait Mapping</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Jason LaCombe</span>, <span class="highwire-citation-author" data-delta="1">Benjamin McClosky</span> and <span class="highwire-citation-author" data-delta="2">Steven Tanksley</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2012 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 2 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1035-1039;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.003517 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459316--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jason LaCombe </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Nature Source Genetics, Ithaca, New York 14850</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jason%2BLaCombe%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=LaCombe%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJason%2BLaCombe%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jlacombe@naturesourcegenetics.com" class="" data-icon-position="" data-hide-link-title="0">jlacombe@naturesourcegenetics.com</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Benjamin McClosky </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Nature Source Genetics, Ithaca, New York 14850</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Benjamin%2BMcClosky%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=McClosky%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABenjamin%2BMcClosky%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Steven Tanksley </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Nature Source Genetics, Ithaca, New York 14850</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Steven%2BTanksley%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tanksley%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASteven%2BTanksley%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/2/9/1035" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/2/9/1035&amp;title=Location-Dependent%20Empirical%20Thresholds%20for%20Quantitative%20Trait%20Mapping" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/2/9/1035&amp;title=Location-Dependent%20Empirical%20Thresholds%20for%20Quantitative%20Trait%20Mapping" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/2/9/1035&amp;title=Location-Dependent%20Empirical%20Thresholds%20for%20Quantitative%20Trait%20Mapping" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/2/9/1035&amp;text=Location-Dependent%20Empirical%20Thresholds%20for%20Quantitative%20Trait%20Mapping" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/2/9/1035&amp;title=Location-Dependent%20Empirical%20Thresholds%20for%20Quantitative%20Trait%20Mapping" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/2/9/1035&amp;t=Location-Dependent%20Empirical%20Thresholds%20for%20Quantitative%20Trait%20Mapping" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/2/9/1035&amp;title=Location-Dependent%20Empirical%20Thresholds%20for%20Quantitative%20Trait%20Mapping" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/2/9/1035&amp;title=Location-Dependent%20Empirical%20Thresholds%20for%20Quantitative%20Trait%20Mapping" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459316/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459316" id="node459316--3" data-pisa="ggg;2/9/1035" data-pisa-master="ggg;2/9/1035" data-apath="/ggg/2/9/1035.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Location-Dependent Empirical Thresholds for Quantitative Trait Mapping</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Jason LaCombe</span>, <span class="highwire-citation-author" data-delta="1">Benjamin McClosky</span> and <span class="highwire-citation-author" data-delta="2">Steven Tanksley</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">September 1, 2012 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 2 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 9 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1035-1039;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.112.003517 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459316--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jason LaCombe </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Nature Source Genetics, Ithaca, New York 14850</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jason%2BLaCombe%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=LaCombe%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJason%2BLaCombe%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jlacombe@naturesourcegenetics.com" class="" data-icon-position="" data-hide-link-title="0">jlacombe@naturesourcegenetics.com</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Benjamin McClosky </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Nature Source Genetics, Ithaca, New York 14850</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Benjamin%2BMcClosky%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=McClosky%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABenjamin%2BMcClosky%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Steven Tanksley </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Nature Source Genetics, Ithaca, New York 14850</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Steven%2BTanksley%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tanksley%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASteven%2BTanksley%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1035&amp;rft.epage=1039&amp;rft.atitle=Location-Dependent%20Empirical%20Thresholds%20for%20Quantitative%20Trait%20Mapping&amp;rft.volume=2&amp;rft.issue=9&amp;rft.date=2012-09-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=LaCombe%2C+Jason&amp;rft.au=McClosky%2C+Benjamin&amp;rft.au=Tanksley%2C+Steven"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459316/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459316/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459316/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459316/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459316/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459316/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459316/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459316/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459316/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459316/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459316/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459316/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/2/9/1035" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">NOTATION AND METHODS</a></li><li class="odd"><a href="#sec-5" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">DISCUSSION</a></li><li class="even"><a href="#sec-6" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">CONCLUSION</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#app-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">APPENDIX</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/2/9/1035.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/2/9/1035.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/2/9/1035.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/2/9/1035.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__c-64NG7CB_fwZbyex-B49GkX1AvVESfu7M9yxTNDG_o__Uvi1liJMEKbHtHHv_1fo_FCLvimvbq9jMwPcCPR364Q__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
