<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/9/4/1103" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/9/4/1103.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/9/4/1103.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/9/4/1103.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/9/4/1103.atom" />
<meta name="HW.pisa" content="ggg;9/4/1103" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Evolution of the Highly Repetitive PEVK Region of Titin Across Mammals" />
<meta name="DC.Identifier" content="10.1534/g3.118.200714" />
<meta name="DC.Date" content="2019-04-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2019 Muenzen et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Relation" content="https://doi.org/10.25387/g3.7544900" />
<meta name="DC.Description" content="The protein titin plays a key role in vertebrate muscle where it acts like a giant molecular spring. Despite its importance and conservation over vertebrate evolution, a lack of high quality annotations in non-model species makes comparative evolutionary studies of titin challenging. The PEVK region of titin—named for its high proportion of Pro-Glu-Val-Lys amino acids—is particularly difficult to annotate due to its abundance of alternatively spliced isoforms and short, highly repetitive exons. To understand PEVK evolution across mammals, we developed a bioinformatics tool, PEVK\_Finder, to annotate PEVK exons from genomic sequences of titin and applied it to a diverse set of mammals. PEVK\_Finder consistently outperforms standard annotation tools across a broad range of conditions and improves annotations of the PEVK region in non-model mammalian species. We find that the PEVK region can be divided into two subregions (PEVK-N, PEVK-C) with distinct patterns of evolutionary constraint and divergence. The bipartite nature of the PEVK region has implications for titin diversification. In the PEVK-N region, certain exons are conserved and may be essential, but natural selection also acts on particular codons. In the PEVK-C, exons are more homogenous and length variation of the PEVK region may provide the raw material for evolutionary adaptation in titin function. The PEVK-C region can be further divided into a highly repetitive region (PEVK-CA) and one that is more variable (PEVK-CB). Taken together, we find that the very complexity that makes titin a challenge for annotation tools may also promote evolutionary adaptation." />
<meta name="DC.Contributor" content="Kathleen Muenzen" />
<meta name="DC.Contributor" content="Jenna Monroy" />
<meta name="DC.Contributor" content="Findley R. Finseth" />
<meta name="article:published_time" content="2019-04-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Evolution of the Highly Repetitive PEVK Region of Titin Across Mammals" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;The protein titin plays a key role in vertebrate muscle where it acts like a giant molecular spring. Despite its importance and conservation over vertebrate evolution, a lack of high quality annotations in non-model species makes comparative evolutionary studies of titin challenging. The PEVK region of titin—named for its high proportion of Pro-Glu-Val-Lys amino acids—is particularly difficult to annotate due to its abundance of alternatively spliced isoforms and short, highly repetitive exons. To understand PEVK evolution across mammals, we developed a bioinformatics tool, PEVK_Finder, to annotate PEVK exons from genomic sequences of titin and applied it to a diverse set of mammals. PEVK_Finder consistently outperforms standard annotation tools across a broad range of conditions and improves annotations of the PEVK region in non-model mammalian species. We find that the PEVK region can be divided into two subregions (PEVK-N, PEVK-C) with distinct patterns of evolutionary constraint and divergence. The bipartite nature of the PEVK region has implications for titin diversification. In the PEVK-N region, certain exons are conserved and may be essential, but natural selection also acts on particular codons. In the PEVK-C, exons are more homogenous and length variation of the PEVK region may provide the raw material for evolutionary adaptation in titin function. The PEVK-C region can be further divided into a highly repetitive region (PEVK-CA) and one that is more variable (PEVK-CB). Taken together, we find that the very complexity that makes titin a challenge for annotation tools may also promote evolutionary adaptation.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2019/04/01" />
<meta name="citation_mjid" content="ggg;9/4/1103" />
<meta name="citation_id" content="9/4/1103" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/9/4/1103" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/9/4/1103.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/9/4/1103.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/9/4/1103.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.118.200714" />
<meta name="citation_pmid" content="30804022" />
<meta name="citation_volume" content="9" />
<meta name="citation_issue" content="4" />
<meta name="citation_num_pages" content="13" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="1103" />
<meta name="citation_lastpage" content="1115" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Kathleen Muenzen" />
<meta name="citation_author_institution" content="Keck Science Department, Claremont McKenna, Pitzer, and Scripps Colleges, Claremont, CA 91711" />
<meta name="citation_author_institution" content="Department of Biomedical Informatics and Medical Education, University of Washington, Seattle, WA 98105" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-0840-614X" />
<meta name="citation_author" content="Jenna Monroy" />
<meta name="citation_author_institution" content="Keck Science Department, Claremont McKenna, Pitzer, and Scripps Colleges, Claremont, CA 91711" />
<meta name="citation_author_email" content="ffinseth@kecksci.claremont.edu" />
<meta name="citation_author_email" content="jmonroy@kecksci.claremont.edu" />
<meta name="citation_author" content="Findley R. Finseth" />
<meta name="citation_author_institution" content="Keck Science Department, Claremont McKenna, Pitzer, and Scripps Colleges, Claremont, CA 91711" />
<meta name="citation_author_email" content="ffinseth@kecksci.claremont.edu" />
<meta name="citation_author_email" content="jmonroy@kecksci.claremont.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-5040-9627" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=S. F. Altschul;citation_author=W. Gish;citation_author=W. Miller;citation_author=E. W. Myers;citation_author=D. J. Lipman;citation_title=Basic local alignment search tool.;citation_pages=403-410;citation_volume=215;citation_year=1990;citation_issue=3;citation_pmid=2231712;citation_doi=10.1006/jmbi.1990.9999" />
<meta name="citation_reference" content="citation_journal_title=Circ. Res.;citation_author=M. L. Bang;citation_author=T. Centner;citation_author=F. Fornoff;citation_author=A. J. Geach;citation_author=M. Gotthardt;citation_title=The complete gene sequence of titin, expression of an unusual ≈700-kDa titin isoform, and its interaction with obscurin identify a novel Z-line to I-band linking system.;citation_volume=89;citation_year=2001;citation_pmid=11717165;citation_doi=10.1161/hh2301.100981" />
<meta name="citation_reference" content="citation_journal_title=Biophys. J.;citation_author=P. Bianco;citation_author=A. Nagy;citation_author=A. Kengyel;citation_author=D. Szatmári;citation_author=Z. Mártonfalvi;citation_title=Interaction forces between F-actin and titin PEVK domain measured with optical tweezers.;citation_volume=93;citation_year=2007;citation_pmid=17513381;citation_doi=10.1529/biophysj.107.106153" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=P. Bork;citation_author=E. V. Koonin;citation_title=Predicting functions from protein sequences--where are the bottlenecks?;citation_pages=313-318;citation_volume=18;citation_year=1998;citation_issue=4;citation_pmid=9537411;citation_doi=10.1038/ng0498-313" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=C. Burge;citation_author=S. Karlin;citation_title=Prediction of complete gene structures in human genomic DNA.;citation_pages=78-94;citation_volume=268;citation_year=1997;citation_issue=1;citation_pmid=9149143;citation_doi=10.1006/jmbi.1997.0951" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=M. Burset;citation_author=I. A. Seledtsov;citation_author=V. V. Solovyev;citation_title=Analysis of canonical and non-canonical splice sites in mammalian genomes;citation_pages=4364-4375;citation_volume=28;citation_year=2000;citation_issue=21;citation_pmid=11058137;citation_doi=10.1093/nar/28.21.4364" />
<meta name="citation_reference" content="citation_journal_title=Hum. Mutat.;citation_author=C. Chauveau;citation_author=J. Rowell;citation_author=A. Ferreiro;citation_title=A Rising Titan, TTN review and mutation update.;citation_pages=1046-1059;citation_volume=35;citation_year=2014;citation_pmid=24980681;citation_doi=10.1002/humu.22611" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=M. Chikina;citation_author=J. D. Robinson;citation_author=N. L. Clark;citation_title=Hundreds of Genes Experienced Convergent Shifts in Selective Pressure in Marine Mammals.;citation_volume=33;citation_year=2016;citation_pmid=27329977;citation_doi=10.1093/molbev/msw112" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=P. J. A. Cock;citation_author=T. Antao;citation_author=J. T. Chang;citation_author=B. A. Chapman;citation_author=C. J. Cox;citation_title=Biopython: freely available Python tools for computational molecular biology and bioinformatics.;citation_volume=25;citation_year=2009;citation_pmid=19304878;citation_doi=10.1093/bioinformatics/btp163" />
<meta name="citation_reference" content="Delaney, N., 2017 Dotplot. GitHub Repository, https://github.com/evolvedmicrobe/dotplot." />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=F. R. Finseth;citation_author=E. Bondra;citation_author=R. G. Harrison;citation_title=Selective constraint dominates the evolution of genes expressed in a novel reproductive gland.;citation_volume=31;citation_year=2014;citation_pmid=25193339;citation_doi=10.1093/molbev/msu259" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=F. R. Finseth;citation_author=Y. Dong;citation_author=A. Saunders;citation_author=L. Fishman;citation_title=Duplication and adaptive evolution of a key centromeric protein in mimulus, a genus with female meiotic drive.;citation_volume=32;citation_year=2015;citation_pmid=26104011;citation_doi=10.1093/molbev/msv145" />
<meta name="citation_reference" content="Forbes, J. G., A. J. Jin, K. Ma, G. Gutierrez-Cruz, W. L. Tsai, K. Wang, 2005 Titin PEVK segment: charge-driven elasticity of the open and flexible polyampholyte. J Muscle Res. Cell Motil. 26:291–301." />
<meta name="citation_reference" content="citation_journal_title=Circulation Research;citation_journal_abbrev=Circ. Res.;citation_author=A. Freiburg;citation_author=K. Trombitas;citation_author=W. Hell;citation_author=O. Cazorla;citation_author=F. Fougerousse;citation_title=Series of Exon-Skipping Events in the Elastic Spring Region of Titin as the Structural Basis for Myofibrillar Elastic Diversity;citation_pages=1114-1121;citation_volume=86;citation_year=2000;citation_issue=11;citation_pmid=10850961;citation_doi=10.1161/01.RES.86.11.1114" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=B. E. Galindo;citation_author=V. D. Vacquier;citation_author=W. J. Swanson;citation_title=Positive selection in the egg receptor for abalone sperm lysin.;citation_volume=100;citation_year=2003;citation_pmid=12676995;citation_doi=10.1073/pnas.0830022100" />
<meta name="citation_reference" content="citation_journal_title=FEBS Letters;citation_journal_abbrev=FEBS Letters;citation_author=M. Gautel;citation_author=D. Goulding;citation_title=A molecular map of titin/connectin elasticity reveals two different mechanisms acting in series.;citation_pages=11-14;citation_volume=385;citation_year=1996;citation_issue=1-2;citation_pmid=8641453;citation_doi=10.1016/0014-5793(96)00338-9" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=M. Goujon;citation_author=H. McWilliam;citation_author=W. Li;citation_author=F. Valentin;citation_author=S. Squizzato;citation_title=A new bioinformatics analysis tools framework at EMBL-EBI.;citation_volume=38;citation_year=2010;citation_pmid=20439314;citation_doi=10.1093/nar/gkq313" />
<meta name="citation_reference" content="citation_journal_title=Am. J. Physiol. Regul. Integr. Comp. Physiol.;citation_author=H. Granzier;citation_author=M. Radke;citation_author=J. Royal;citation_author=Y. Wu;citation_author=T. C. Irving;citation_title=Functional genomics of chicken, mouse, and human titin supports splice diversity as an important mechanism for regulating biomechanics of striated muscle.;citation_volume=293;citation_year=2007;citation_pmid=17522126;citation_doi=10.1152/ajpregu.00001.2007" />
<meta name="citation_reference" content="citation_journal_title=Journal of muscle research and cell motility;citation_journal_abbrev=J Muscle Res Cell Motil;citation_author=M. L. Greaser;citation_author=M. Berri;citation_author=C. M. Warren;citation_author=P. E. Mozdziak;citation_title=Species variations in cDNA sequence and exon splicing patterns in the extensible I-band region of cardiac titin: relation to passive tension.;citation_pages=473-482;citation_volume=23;citation_year=2002;citation_issue=5-6;citation_pmid=12785098;citation_doi=10.1023/A:1023410523184" />
<meta name="citation_reference" content="Greaser, M. L. 2001 Identification of new repeating motifs in titin. Proteins. 43:145–149." />
<meta name="citation_reference" content="citation_journal_title=Current opinion in cell biology;citation_journal_abbrev=Curr Opin Cell Biol;citation_author=C. C. Gregorio;citation_author=H. Granzier;citation_author=H. Sorimachi;citation_author=S. Labeit;citation_title=Muscle assembly: a titanic achievement?;citation_pages=18-25;citation_volume=11;citation_year=1999;citation_issue=1;citation_pmid=10047523;citation_doi=10.1016/S0955-0674(99)80003-9" />
<meta name="citation_reference" content="citation_journal_title=J. Biomed. Biotechnol.;citation_author=W. Guo;citation_author=S. J. Bharmal;citation_author=K. Esbona;citation_author=M. L. Greaser;citation_title=Titin diversity: alternative splicing gone wild.;citation_pages=1-8;citation_volume=2010;citation_year=2010;citation_pmid=20589071;citation_doi=10.1155/2010/180682" />
<meta name="citation_reference" content="citation_journal_title=Mol. Cell. Biomech.;citation_author=W. Herzog;citation_author=T. Leonard;citation_author=V. Joumaa;citation_author=M. DuVall;citation_author=A. Panchangam;citation_title=The three filament model of skeletal muscle stability and force production.;citation_pages=175-191;citation_volume=9;citation_year=2012;citation_pmid=23285733" />
<meta name="citation_reference" content="citation_journal_title=Front. Physiol.;citation_author=A. L. Hessel;citation_author=S. L. Lindstedt;citation_author=K. C. Nishikawa;citation_title=Physiological mechanisms of eccentric contraction and its appications: a role for the gian titin protein.;citation_pages=70;citation_volume=8;citation_year=2017;citation_doi=10.3389/fphys.2017.00070" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=A. L. Hughes;citation_author=M. Nei;citation_title=Nucleotide substitution at major histocompatibility complex class II loci: evidence for overdominant selection.;citation_pages=958-962;citation_volume=86;citation_year=1989;citation_issue=3;citation_pmid=2492668;citation_doi=10.1073/pnas.86.3.958" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_journal_abbrev=Bioinformatics;citation_author=T. Junier;citation_author=M. Pagni;citation_title=Dotlet: diagonal plots in a Web browser;citation_pages=178-179;citation_volume=16;citation_year=2000;citation_issue=2;citation_pmid=10842741;citation_doi=10.1093/bioinformatics/16.2.178" />
<meta name="citation_reference" content="citation_journal_title=Nat. Rev. Genet.;citation_author=E. K. Karlsson;citation_author=D. P. Kwiatkowski;citation_author=P. C. Sabeti;citation_title=Natural selection and infectious disease in human populations.;citation_pages=379-393;citation_volume=15;citation_year=2014;citation_pmid=24776769;citation_doi=10.1038/nrg3734" />
<meta name="citation_reference" content="citation_journal_title=FEBS Letters;citation_journal_abbrev=FEBS Letters;citation_author=M. S. Z. Kellermayer;citation_author=H. L. Granzier;citation_title=Calcium-dependent inhibition of in vitro thin-filament motility by native titin.;citation_pages=281-286;citation_volume=380;citation_year=1996;citation_issue=3;citation_pmid=8601441;citation_doi=10.1016/0014-5793(96)00055-5" />
<meta name="citation_reference" content="citation_journal_title=Annual review of genetics;citation_journal_abbrev=Annu Rev Genet;citation_author=E. V. Koonin;citation_title=Orthologs, Paralogs, and Evolutionary Genomics.;citation_pages=309-338;citation_volume=39;citation_year=2005;citation_pmid=16285863;citation_doi=10.1146/annurev.genet.39.073003.114725" />
<meta name="citation_reference" content="citation_journal_title=Annual Review of Ecology and Systematics;citation_journal_abbrev=Annu. Rev. Ecol. Syst.;citation_author=M. Kreitman;citation_author=H. Akashi;citation_title=Molecular Evidence for Natural Selection;citation_pages=403-422;citation_volume=26;citation_year=1995;citation_issue=1;citation_doi=10.1146/annurev.es.26.110195.002155" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=S. Kumar;citation_author=A. J. Filipski;citation_author=F. U. Battistuzzi;citation_author=S. L. Kosakovsky Pond;citation_author=K. Tamura;citation_title=Statistics and truth in phylogenomics.;citation_volume=29;citation_year=2012;citation_pmid=21873298;citation_doi=10.1093/molbev/msr202" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=S. Kumar;citation_author=G. Stecher;citation_author=K. Tamura;citation_title=MEGA7: Molecular Evolutionary Genetics Analysis Version 7.0 for Bigger Datasets.;citation_volume=33;citation_year=2016;citation_pmid=27004904;citation_doi=10.1093/molbev/msw054" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=S. Kumar;citation_author=G. Stecher;citation_author=S. B. Hedges;citation_title=TimeTree: A Resource for Timelines, Timetrees, and Divergence Times.;citation_volume=34;citation_year=2017;citation_doi=10.1093/molbev/msx116" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=S. Labeit;citation_author=B. Kolmerer;citation_title=Titins: Giant Proteins in Charge of Muscle Ultrastructure and Elasticity;citation_pages=293-296;citation_volume=270;citation_year=1995;citation_issue=5234;citation_pmid=7569978;citation_doi=10.1126/science.270.5234.293" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=D. Labeit;citation_author=K. Watanabe;citation_author=C. Witt;citation_author=H. Fujita;citation_author=Y. Wu;citation_title=Calcium-dependent molecular spring elements in the giant protein titin.;citation_volume=100;citation_year=2003;citation_pmid=14593205;citation_doi=10.1073/pnas.2235652100" />
<meta name="citation_reference" content="citation_journal_title=Am. J. Physiol. Cell Physiol.;citation_author=T. R. Leonard;citation_author=W. Herzog;citation_title=Regulation of muscle force in the absenve of actin-myosin-based cross-bridge interaction.;citation_volume=299;citation_year=2010;citation_pmid=20357181;citation_doi=10.1152/ajpcell.00049.2010" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_journal_abbrev=Heredity;citation_author=E. Lerat;citation_title=Identifying repeats and transposable elements in sequenced genomes: how to find your way through the dense forest of programs.;citation_pages=520-533;citation_volume=104;citation_year=2010;citation_issue=6;citation_pmid=19935826;citation_doi=10.1038/hdy.2009.165" />
<meta name="citation_reference" content="citation_journal_title=Annu. Rev. Physiol.;citation_author=W. A. Linke;citation_title=Titin Gene and Protein Functions in Passive and Active Muscle.;citation_pages=389-411;citation_volume=80;citation_year=2018;citation_doi=10.1146/annurev-physiol-021317-121234" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=W. A. Linke;citation_author=M. Ivemeyer;citation_author=P. Mundel;citation_author=M. R. Stockmeier;citation_author=B. Kolmerer;citation_title=Nature of PEVK-titin elasticity in skeletal muscle;citation_pages=8052-8057;citation_volume=95;citation_year=1998;citation_issue=14;citation_pmid=9653138;citation_doi=10.1073/pnas.95.14.8052" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=W. A. Linke;citation_author=M. Ivemeyer;citation_author=N. Olivieri;citation_author=B. Kolmerer;citation_author=J. C. Rüegg;citation_title=Towards a molecular understanding of the elasticity of titin.;citation_pages=62-71;citation_volume=261;citation_year=1996;citation_issue=1;citation_pmid=8760502;citation_doi=10.1006/jmbi.1996.0441" />
<meta name="citation_reference" content="citation_journal_title=Journal of structural biology;citation_journal_abbrev=J Struct Biol;citation_author=W. A. Linke;citation_author=M. Kulke;citation_author=H. Li;citation_author=S. Fujita-Becker;citation_author=C. Neagoe;citation_title=PEVK domain of titin: an entropic spring with actin-binding properties.;citation_pages=194-205;citation_volume=137;citation_year=2002;citation_issue=1-2;citation_pmid=12064946;citation_doi=10.1006/jsbi.2002.4468" />
<meta name="citation_reference" content="citation_journal_title=Nat. Struct. Mol. Biol.;citation_author=A. Manteca;citation_author=J. Schönfelder;citation_author=A. Alonso-Caballero;citation_author=M. J. Fertin;citation_author=N. Barruetabeña;citation_title=Mechanochemical evolution of the giant muscle protein titin as inferred from resurrected proteins.;citation_pages=652-657;citation_volume=24;citation_year=2017;citation_doi=10.1038/nsmb.3426" />
<meta name="citation_reference" content="citation_journal_title=J. Exp. Biol.;citation_author=J. A. Monroy;citation_author=K. L. Powers;citation_author=C. M. Pace;citation_author=T. Uyeno;citation_author=K. C. Nishikawa;citation_title=Effects of activation on the elastic properties of intact soleus muscles with a deletion in titin.;citation_volume=220;citation_year=2017;citation_pmid=27994045;citation_doi=10.1242/jeb.139717" />
<meta name="citation_reference" content="citation_journal_title=J. Cell Sci.;citation_author=A. Nagy;citation_author=P. Cacciafesta;citation_author=L. Grama;citation_author=A. Kengyel;citation_author=A. Málnási-Csizmadia;citation_title=Differential actin binding along the PEVK domain of skeletal muscle titin.;citation_volume=117;citation_year=2004;citation_pmid=15507486;citation_doi=10.1242/jcs.01501" />
<meta name="citation_reference" content="citation_journal_title=Proc. Biol. Sci.;citation_author=K. C. Nishikawa;citation_author=J. A. Monroy;citation_author=T. E. Uyeno;citation_author=S. H. Yeo;citation_author=D. K. Pai;citation_title=Is titin a “winding filament”? A new twist on muscle contraction.;citation_volume=279;citation_year=2012;citation_pmid=21900329;citation_doi=10.1098/rspb.2011.1304" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=N. A. O’Leary;citation_author=M. W. Wright;citation_author=J. R. Brister;citation_author=S. Ciufo;citation_author=D. Haddad;citation_title=Reference sequence (RefSeq) database at NCBI: current status, taxonomic expansion, and functional annotation.;citation_volume=44;citation_year=2016;citation_pmid=26553804;citation_doi=10.1093/nar/gkv1189" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=G. Parra;citation_author=E. Blanco;citation_author=R. Guigó;citation_title=GeneID in Drosophila;citation_pages=511-515;citation_volume=10;citation_year=2000;citation_issue=4;citation_pmid=10779490;citation_doi=10.1101/gr.10.4.511" />
<meta name="citation_reference" content="citation_journal_title=eLife;citation_author=R. Partha;citation_author=B. K. Chauhan;citation_author=Z. Ferreira;citation_author=J. D. Robinson;citation_author=K. Lathrop;citation_title=Subterranean mammals show convergent regression in ocular genes and enhancers, along with adaptation to tunneling.;citation_pages=e25884;citation_volume=6;citation_year=2017;citation_pmid=29035697;citation_doi=10.7554/eLife.25884" />
<meta name="citation_reference" content="Perutz, M. F., 1983 Species adaptation in a protein molecule. Mol. Biol. Evol. 1: 1–28." />
<meta name="citation_reference" content="citation_journal_title=Nat. Commun.;citation_author=D. D. Pervouchine;citation_author=S. Djebali;citation_author=A. Breschi;citation_author=C. A. Davis;citation_author=P. P. Barja;citation_title=Enhanced transcriptome maps from multiple mouse tissues reveal evolutionary constraint in gene expression.;citation_pages=5903;citation_volume=6;citation_year=2015;citation_pmid=25582907;citation_doi=10.1038/ncomms6903" />
<meta name="citation_reference" content="citation_journal_title=J. Exp. Biol.;citation_author=K. Powers;citation_author=K. Nishikawa;citation_author=V. Joumaa;citation_author=W. Herzog;citation_title=Decreased force enhancement in skeletal muscle sarcomeres with a deletion in titin.;citation_volume=219;citation_year=2016;citation_pmid=26944495;citation_doi=10.1242/jeb.132027" />
<meta name="citation_reference" content="citation_journal_title=J. Gen. Physiol.;citation_author=L. G. Prado;citation_author=I. Makarenko;citation_author=C. Andresen;citation_author=M. Krüger;citation_author=C. A. Opitz;citation_title=Isoform Diversity of Giant Proteins in Relation to Passive and Active Contractile Properties of Rabbit Skeletal Muscles.;citation_volume=126;citation_year=2005;citation_pmid=16230467;citation_doi=10.1085/jgp.200509364" />
<meta name="citation_reference" content="R Core Team, (2016). R: A language and environment for statistical computing. R Foundation for Statistical Computing, Vienna Austria, URL https://www.R-project.org/" />
<meta name="citation_reference" content="Revell, L., 2012 phytools: An R package for phylogenetic comparative biology (and other things). Methods Ecol. Evol. 3: 217–223." />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=A. A. Salamov;citation_author=V. V. Solovyev;citation_title=Ab initio Gene Finding in Drosophila Genomic DNA;citation_pages=516-522;citation_volume=10;citation_year=2000;citation_issue=4;citation_pmid=10779491;citation_doi=10.1101/gr.10.4.516" />
<meta name="citation_reference" content="citation_journal_title=Skelet. Muscle;citation_author=M. Savarese;citation_author=P. H. Jonson;citation_author=S. Huovinen;citation_author=L. Paulin;citation_author=P. Auvinen;citation_title=The complexity of titin splicing pattern in human adult skeletal muscles.;citation_pages=11;citation_volume=8;citation_year=2018;citation_doi=10.1186/s13395-018-0156-z" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=S. Schafer;citation_author=A. de Marvao;citation_author=E. Adami;citation_author=L. R. Fiedler;citation_author=B. Ng;citation_title=Titin-truncating variants affect heart function in disease cohorts and the general population.;citation_pages=46-53;citation_volume=49;citation_year=2017;citation_doi=10.1038/ng.3719" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=G. Schappacher-Tilp;citation_author=T. Leonard;citation_author=G. Desch;citation_author=W. Herzog;citation_title=A novel three-filament model of force generation in eccentric contraction of skeletal muscles.;citation_pages=e0117634;citation_volume=10;citation_year=2015;citation_pmid=25816319;citation_doi=10.1371/journal.pone.0117634" />
<meta name="citation_reference" content="citation_journal_abbrev=Gene;citation_author=R. D. Sleator;citation_title=An overview of the current status of eukaryote gene prediction strategies.;citation_pages=1-4;citation_volume=461;citation_year=2010;citation_issue=1-2;citation_pmid=20430068;citation_doi=10.1016/j.gene.2010.04.008" />
<meta name="citation_reference" content="citation_title=Gnomon-NCBI eukaryotic gene prediction tool;citation_author=A. Souvorov;citation_author=Y. Kaputsin;citation_author=B. Kiryutin;citation_author=V. Chetverin;citation_author=T. Tatusova;citation_year=2010" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=M. Stanke;citation_author=B. Morgenstern;citation_title=AUGUSTUS: a web server for gene prediction in eukaryotes that allows user-defined constraints;citation_pages=W465-W467;citation_volume=33;citation_year=2005;citation_issue=suppl_2;citation_pmid=15980513;citation_doi=10.1093/nar/gki458" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=P. R. H. Steinmetz;citation_author=J. E. M. Kraus;citation_author=C. Larroux;citation_author=J. U. Hammel;citation_author=A. Amon-Hassenzahl;citation_title=Independent evolution of striated muscles in cnidarians and bilaterians.;citation_pages=231-234;citation_volume=487;citation_year=2012;citation_issue=7406;citation_pmid=22763458;citation_doi=10.1038/nature11180" />
<meta name="citation_reference" content="citation_journal_title=Prog. Nat. Sci.;citation_author=K. Tamura;citation_author=M. Nei;citation_author=S. Kumar;citation_title=Prospects for inferring very large phylogenies by using the neighbor-joining method.;citation_volume=101;citation_year=2004;citation_pmid=15258291;citation_doi=10.1073/pnas.0404206101" />
<meta name="citation_reference" content="citation_journal_title=Journal of muscle research and cell motility;citation_journal_abbrev=J Muscle Res Cell Motil;citation_author=R. Tatsumi;citation_author=K. Maeda;citation_author=A. Hattori;citation_author=K. Takahashi;citation_title=Calcium binding to an elastic portion of connectin/titin filaments.;citation_pages=149-162;citation_volume=22;citation_year=2001;citation_issue=2;citation_pmid=11519738;citation_doi=10.1023/A:1010349416723" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=R. L. Tatusov;citation_author=E. V. Koonin;citation_author=D. J. Lipman;citation_title=A Genomic Perspective on Protein Families;citation_pages=631-637;citation_volume=278;citation_year=1997;citation_issue=5338;citation_pmid=9381173;citation_doi=10.1126/science.278.5338.631" />
<meta name="citation_reference" content="citation_journal_title=BioEssays :  news and reviews in molecular, cellular and developmental biology;citation_journal_abbrev=Bioessays;citation_author=P. Tompa;citation_title=Intrinsically unstructured proteins evolve by repeat expansion.;citation_pages=847-855;citation_volume=25;citation_year=2003;citation_issue=9;citation_pmid=12938174;citation_doi=10.1002/bies.10324" />
<meta name="citation_reference" content="citation_journal_title=Biophysical Journal;citation_journal_abbrev=Biophys. J;citation_author=K. Trombitás;citation_author=A. Redkar;citation_author=T. Centner;citation_author=Y. Wu;citation_author=S. Labeit;citation_title=Extensibility of Isoforms of Cardiac Titin: Variation in Contour Length of Molecular Subsegments Provides a Basis for Cellular Passive Stiffness Diversity;citation_pages=3226-3234;citation_volume=79;citation_year=2000;citation_issue=6;citation_pmid=11106626;citation_doi=10.1016/S0006-3495(00)76555-6" />
<meta name="citation_reference" content="citation_journal_title=Genomics, proteomics &amp; bioinformatics / Beijing Genomics Institute;citation_journal_abbrev=Genomics Proteomics Bioinformatics;citation_author=Z. Wang;citation_author=Y. Chen;citation_author=Y. Li;citation_title=A brief review of computational gene prediction methods.;citation_pages=216-221;citation_volume=2;citation_year=2004;citation_issue=4;citation_pmid=15901250;citation_doi=10.1016/S1672-0229(04)02028-5" />
<meta name="citation_reference" content="citation_journal_title=Journal of structural biology;citation_journal_abbrev=J Struct Biol;citation_author=C. C. Witt;citation_author=N. Olivieri;citation_author=T. Centner;citation_author=B. Kolmerer;citation_author=S. Millevoi;citation_title=A survey of the primary structure and the interspecies conservation of I-band titin's elastic elements in vertebrates.;citation_pages=206-215;citation_volume=122;citation_year=1998;citation_issue=1-2;citation_pmid=9724622;citation_doi=10.1006/jsbi.1998.3993" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=Z. Yang;citation_title=PAML 4: Phylogenetic Analysis by Maximum Likelihood.;citation_volume=24;citation_year=2007;citation_pmid=17483113;citation_doi=10.1093/molbev/msm088" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=Z. Yang;citation_author=W. S. W. Wong;citation_author=R. Nielsen;citation_title=Bayes Empirical Bayes Inference of Amino Acid Sites Under Positive Selection.;citation_volume=22;citation_year=2005;citation_pmid=15689528;citation_doi=10.1093/molbev/msi097" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=A. Yates;citation_author=W. Akanni;citation_author=M. R. Amode;citation_author=D. Barrell;citation_author=K. Billis;citation_title=Ensembl 2016.;citation_volume=44;citation_year=2016;citation_pmid=26687719;citation_doi=10.1093/nar/gkv1157" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=G. Zhang;citation_author=C. Li;citation_author=Q. Li;citation_author=B. Li;citation_author=D. M. Larkin;citation_title=Comparative genomics reveals insights into avian genome evolution and adaptation;citation_pages=1311-1320;citation_volume=346;citation_year=2014;citation_issue=6215;citation_pmid=25504712;citation_doi=10.1126/science.1251385" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=H. Zhao;citation_author=B. Ru;citation_author=E. C. Teeling;citation_author=C. G. Faulkes;citation_author=S. Zhang;citation_title=Rhodopsin molecular evolution in mammals inhabiting low light environments.;citation_pages=e8326-e8326;citation_volume=4;citation_year=2009;citation_issue=12;citation_pmid=20016835;citation_doi=10.1371/journal.pone.0008326" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Evolution of the Highly Repetitive PEVK Region of Titin Across Mammals" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4.cover-source.jpg" />
<meta name="twitter:description" content="The protein titin plays a key role in vertebrate muscle where it acts like a giant molecular spring. Despite its importance and conservation over vertebrate evolution, a lack of high quality annotations in non-model species makes comparative evolutionary studies of titin challenging. The PEVK region of titin—named for its high proportion of Pro-Glu-Val-Lys amino acids—is particularly difficult to annotate due to its abundance of alternatively spliced isoforms and short, highly repetitive exons. To understand PEVK evolution across mammals, we developed a bioinformatics tool, PEVK\_Finder, to annotate PEVK exons from genomic sequences of titin and applied it to a diverse set of mammals. PEVK\_Finder consistently outperforms standard annotation tools across a broad range of conditions and improves annotations of the PEVK region in non-model mammalian species. We find that the PEVK region can be divided into two subregions (PEVK-N, PEVK-C) with distinct patterns of evolutionary constraint and divergence. The bipartite nature of the PEVK region has implications for titin diversification. In the PEVK-N region, certain exons are conserved and may be essential, but natural selection also acts on particular codons. In the PEVK-C, exons are more homogenous and length variation of the PEVK region may provide the raw material for evolutionary adaptation in titin function. The PEVK-C region can be further divided into a highly repetitive region (PEVK-CA) and one that is more variable (PEVK-CB). Taken together, we find that the very complexity that makes titin a challenge for annotation tools may also promote evolutionary adaptation." />
<meta name="og-title" property="og:title" content="Evolution of the Highly Repetitive PEVK Region of Titin Across Mammals" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/9/4/1103" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="The protein titin plays a key role in vertebrate muscle where it acts like a giant molecular spring. Despite its importance and conservation over vertebrate evolution, a lack of high quality annotations in non-model species makes comparative evolutionary studies of titin challenging. The PEVK region of titin—named for its high proportion of Pro-Glu-Val-Lys amino acids—is particularly difficult to annotate due to its abundance of alternatively spliced isoforms and short, highly repetitive exons. To understand PEVK evolution across mammals, we developed a bioinformatics tool, PEVK\_Finder, to annotate PEVK exons from genomic sequences of titin and applied it to a diverse set of mammals. PEVK\_Finder consistently outperforms standard annotation tools across a broad range of conditions and improves annotations of the PEVK region in non-model mammalian species. We find that the PEVK region can be divided into two subregions (PEVK-N, PEVK-C) with distinct patterns of evolutionary constraint and divergence. The bipartite nature of the PEVK region has implications for titin diversification. In the PEVK-N region, certain exons are conserved and may be essential, but natural selection also acts on particular codons. In the PEVK-C, exons are more homogenous and length variation of the PEVK region may provide the raw material for evolutionary adaptation in titin function. The PEVK-C region can be further divided into a highly repetitive region (PEVK-CA) and one that is more variable (PEVK-CB). Taken together, we find that the very complexity that makes titin a challenge for annotation tools may also promote evolutionary adaptation." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4.cover-source.jpg" />
    <title>Evolution of the Highly Repetitive PEVK Region of Titin Across Mammals | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"gO7WShkU5RBreL-0MupQcF7BkdvOzMezoHSYumTMAD8","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"492618","apath":"\/ggg\/9\/4\/1103.atom","pisa":"ggg;9\/4\/1103","ac":{"\/ggg\/9\/4\/1103.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/9\/4\/1103.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;9\/4\/1103"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;9\/4\/1103"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/9\/4\/1103":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022IjNlekUqjJQKC5vuo3UD2L09Qe\\\\\\\/k0W318IZCQ8CUQFuSbPvfZ7uV7ckYm+B6G2ztXKq6w9yyjdXIf6x7xXtZ0lGuIMPvbie\\\\\\\/cM0HNDoRJkMkhfltltwei2KV5IX8JS5gwM2mmKajyYnrIasM0h7fQtTNQlNBlIdo7+sX2KXFfSvZe7xl7Y\\\\\\\/JrCaWtEXlf8kMO24uMkjy3ucu\\\\\\\/obPFprA2MSOHrrwmetyea6lzGKP7j\\\\\\\/CTFg4WcygH70htsDjY3HvjlMT1Ivz+eH3+j0mYkIFb9yAxV4aYTfYCxK0u5PTrLXqotyy9BFktj4ICMoqRbACfoTuA\\\\\\\/ZSqtjrOU0Cwnja+x8FnGIT6Q12Gpkw9GZgwIkcos7Bna9bXzoUsVzmE5TgC23qiAtxLu35bfg5OSmhAAc5qg7YM\\\\\\\/u6tcqYsAmxeDBNcjWeMShznrMAZptbqdKmM2PeblBdwEGJTF5G9sj0S0o5gyaKs24qJCeEIMqd7xCGXjgNkPCrh7J8\\\\\\\/VywaPqw80tW4gSMREyd+DTwcayuP\\\\\\\/iljGGfbAFZF6+AWGm9hX54db5NYij5oNwp9bSF3EPVJjUb3pQT5Vh0MGQKgVYP7b\\\\\\\/U01CN6Lq6Lzof1is0k7d3hpnLAf9+8lVx5vYXyynEkCxJy2xVBwfxKJIjYvaHgfG1ODhR8cSDOZm0bufxQFugkvwj3rsW+Js0LlW7is3tTSiI09OVgW6OO2\\\\\\\/5YLWz9HB0CxqxKsCu9uB1OsL9fs4Pryt7uIQfiPiiXNCGQSc4siuGk\\\\\\\/4tm\\\\\\\/mslsE5P3NTagT07D+3zXRSJY4itR246o6Xv\\\\\\\/MdMV2hEUrcK68TBIgM3oNfJvO\\\\\\\/iS\\\\\\\/H2SO8AhMWbhErwtaF\\\\\\\/dngntp\\\\\\\/rSTxXuMsOR57j4jnfT4e1zgahABL59VRuoZ+foePFfx\\\\\\\/boUpPbbVea1INHrG95OTPYPrq2ZTtmO6fW9t+1eM8SYvKGvuXtXORJ3+MkP0+6Hc7r2nBiZmlO43BdZxCovGBt4r5i+569TXB8P+IH1bT2TBpw80DfRruNiQkzPjNpuDdGQv02aJpzVoiHK2OFW0GPauctDjOBiCUQHIQL01vGJ7HsqU8kk\\\\\\\/Nw63Eddg8SdvjSpv4KsxNC68hYKsXx9IIa5WPYoIzsACMUhpuM6hQxCQDPOqO56XkESvrdMfl+vBmrlHIecZ6mWqvo5ZJJfBvCPV0wbzxLpFHNqF7z2wqi2geOHkupGRj+awJ9jq\\\\\\\/4W\\\\\\\/tEuc0ao6NuXQSZkAf9jAYVvo4L97RWao0gWf7y9a5UDDnfBu\\\u0022,\\\u0022iv\\\u0022:\\\u0022F\\\\\\\/OpykBh4GmCqRgW7xjr0A==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022e011fae60c116f73fb7547dd037910c38875412e5a1098f16c72692912523bf6\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00224CeaRUrYkeKq3+M\\\\\\\/PGh1BI9qJ5okwyOJM1EwG1Cv+GGdzyEOUQKF4TUyotJJE013mGCHZbsE45NbkSodYUyTw52atthgrpP6qKoHiJH3CPRRMDbsp7wcDGLHbVwCr0n\\\\\\\/pLYAltHPlVGJ3lyyFolBlx2ycE+FxiZp072KKCDY24vmgxnP\\\\\\\/zGD6wF5brNl3VzkBSqNHJ79om18bhu\\\\\\\/1mUW6opSBYnjBlaUAEa3yUsj+0xst9cbANFcgzd3jUFzmnvDphSJckVLzieqIG24IoqwnUoZcqoZpr74De2FkPyisGIE8U\\\\\\\/PnijhRuQ2nvLZMdKEJO41T+SnfarRZ3rdljTyPMJAOixxRK0sJsxN6QA6WXfpsNvz1GkTbtrLP8WpF+aZmXvRtDEv1MxSBuZ2dEyHOKCOUT4uK4ILqiDxTTWvrF2C18vyYQisLT+lC5mpHV5k7\\\\\\\/gm\\\\\\\/qZCQNDUj5uFlbOR089NZywgq\\\\\\\/atO0OxXWq\\\\\\\/Ramg1Ua1JERHKZjG5uFI3Au\\\\\\\/Yn1AmfAqad\\\\\\\/SzrcVqoaJLdKG8XHQo0KFyc524DULCm8xaVJX9U\\\\\\\/7\\\\\\\/dgPbciGVCqeCG5pIl+JHK+FGI+s7nJlpu1o372M+vbuuxXCi\\\\\\\/RP8bQu2k3vdWFvPDmf7Mjh43H6gKvlTEFIqHrIm9Dk3EC0xkUduslmpQYjbwHpiCNsQdwT+QvHxiPOnrQPlZjYeTsgYt7k2z8eWSfZivgB\\\\\\\/95bUuRIxMenjwAy1JAAR8XiFF\\\\\\\/9EjHsYR4U6m\\\\\\\/jLKQ2t8Tw9Ylf1g7y\\\\\\\/pFogYOq9A9fNN7Lgm6TFmA0083qhZ2YeZFYLyBvCkC4j25inNkMjCusmzhrB4o6HUrdJ54aKINDubTNj4sBHD1JRy2g0dOTEmCV2fwN2UHPfgDIjZKuuxsK9hbUp8DkHfHMX\\\\\\\/N8b4KBAclmMuw0iyYZPKWuvCQxUMiqueHNsYLS1OXXy4p+16qdzcEO4s2S4V14y1o\\\\\\\/6znmHwH41Ku\\\\\\\/w2gwc5Lk+4Q265wzs7ZKedKeOA55wJwrY9YbYPSlzZpqMILi1RjaOF4OlNHUH9tfSvzK9HJp949P74dRpCUeUrjYx3lmnaskZpJO7xlJGt+goVodLQvnB88n8L+gSlxHx0x4fglJKjySb\\\\\\\/U8YAqiNQpYAuiEAxsoG9Gzda8oAqKclF3zg8U8jQsfrC0G\\\\\\\/HY1z\\\\\\\/tKq5W+wvbxqFZDih\\\\\\\/qJxaVeWGLlNpfIyPUt9pTRDEZKRGCbIxJpPzhOWN6OWv4PCjuITY3u9LEsfz7qs5Dd7OFwU8F3bvzZh\\\\\\\/dciGUd\\\\\\\/tnhXJxsuZqq7Ifyh4wuZ8GPdo8jUQAacW3ebCpxg0exPP4a\\\\\\\/WWFQwDM9acODYZXMHfmGrGNNzCNsEPjGPr8Iu8RLwa5WSKzOj4DRQEp2Xg8PO55tCuzb1wHNhw3GwmSkZnZzn8OMcJtjCdI8v8+RPYgRlNhddyakvMhYC44J7JpStjf8OqpM3YYU02oXe\\\\\\\/cSImh3\\\\\\\/05MgmFR2pEqoN+hV9Av078TLJxGmbTvYuSbD1BLywUV98syjenkcyLiiSRasUlZxHe7eKXvfLXCDC84LrUMSn+U3LMDuGXQz7MACC7E6UCWTp01SUET\\\\\\\/bB\\\\\\\/N27GKQWgo+7pWwWs7ak3UJ8Bhdf4YLhEaTbSM\\\\\\\/4p901Jc7QMahqww0JgC6\\\\\\\/Dl46Aay3Q\\\\\\\/yiB8TBDNMhIpJ8rSeE4uU5X0aIMfdBY0NumJQZBwUiA5IGMf8VW+1RPcS0\\\\\\\/YRDcM6n8dg7ehQljsdgB0\\\\\\\/bh6H8DZRnOeXHz9LN63jEEc1HRoZm6kFFzI\\\\\\\/ciCw3qBE9cU1aIFtYJFROI0whOkOzq7qR\\\\\\\/TCMMUQWbfL1VAjbZZVTEMQ+D5lgevg2cEN7\\\\\\\/ZVsG\\\\\\\/TD8PpF6JADixPmtCH1pTbN9X8paybXgU9EjqVokQ6XBXsxaotamgvgJndVB4+aS1jQM5vRCVEQjEeJbJ52bkxUq+WBAkhW5+X7EHhROA0Cd6oUg+8A5sgSntDkicnqiUPzbeQr\\\\\\\/2Lu3ufd0KGd9IgVHg+tnmOxBuOWT6WAVRkQNmhjI5GDRtzZjOV+FwNbeHF9u\\\\\\\/fOSAKlEPaqUyuM0efeO9yBoOsvlXqfzRDBpanInjwFHwKFTI8RPHLmVjBD2zFU5Akbm1RH8WKTeugFjgcbVFB6wigM9EEDQ+qaYixEg\\\\\\\/Nx0Ds5SksjL3zQaF\\\\\\\/a6jmeG\\\\\\\/8woLbZd7dk5HUWm1EZLQUIXwNkxN3zqoBUIRowuX3ftI+X7SR\\\\\\\/u0rgLC3g6j\\\\\\\/K0TQz2Zeg7O3gkJJYljl+eVDWMKsKqGJXBtOCy+Mq3VoBriavTGdY7tfeQckGHIYAYtA64hnIHb5hrAKo15OrnTzJpZBiukm2d63FdVgj64TOHNwC0IrwPCZ0rk9My0FS86guEdq9TdD+vdTqumbR+9KQpoR2NwTeu6QkzqsmfqEf8wkEAd+wMNlJhBA0NJb8bS16LMhm\\\\\\\/dArC\\\\\\\/bT7aJiaUbenVRDf\\\\\\\/Uvky57xxd4XpinCxxmrYFnINUcy4i3Fu8uGsFHuUUogQrRBowsEWLjwJRcmtxajQa2N\\\\\\\/ZXeAEc01ubel4K4Zh\\\\\\\/TEqz4j08V\\\\\\\/LcRFSmvF7IkNR+XNIYPvsex4yWMzjsetb+mkNGJ0bUlw6JPWLs+IrbKad08iEj9zFbhFG+zrXhJVgaxxi\\\\\\\/qtVOHkH3jzRXH00k7dy6Vngbq5B24GctaBqBpm2XbpjkNRDNw1FOiKi4rahUKy898rqX5t0+yJssXJvY\\\\\\\/WmuXLPh5aztkxJKFdATPM4S+KVsmw4WU1d3ePzaalYOEz4pQAgQggdOAjcw1qsdsKeyEnrwdvU8S59bOtcmzXt+pzqyKEs+PaazYJF1Q5PGsveEidRYvJSSd5O1oNfw3pafRdmSrG4u+PcAoc7noAoCq72c9S62i+l\\\\\\\/JpOfJBNaUtgyv09rmeJSgr7oaES7UlLg0EJbwRWsE9aFM3WKkT+tFGGtGW2u7ACgeOX+Vc7h7f5ugH8hCG6Az3kHt4vqDJGkFp1ow015LcWUiEsWhcxXbEryren53StCrb6we2G0WYXxrawQt7J6gB1U3EIjoOTx\\\\\\\/3ity+aiFmJG3HGy+rs8cIg5KutkNO6HMojFgtpq3HB8ZAGG\\\\\\\/R3wPNRO2GabicmR2jLsUIPpTKEl9AS2t3MlOatsxawY224ZWnR\\\\\\\/M8bViJa953wKoh6favg8Eys26lHxqjLum5\\\\\\\/GrKuBywkvW2cTCEL5qkFj\\\\\\\/LTsK2+lRDnKCsLlu4wqsetntpYNg8IqYjE2p6Mg8rvJOsyEFwlKnL8I2mbHbuerbilRqEjaWjHCauslEd+XOkKhkLheTf9\\\\\\\/7ES0AiHGngabwH9ugPiAGjRnfONiWnQ3N6E6GEQfitXMIqYrteYZjr\\\\\\\/JyV0v3hFt47wnyd7\\\\\\\/WcdKHLRtxOrn\\\\\\\/ISxrKp\\\\\\\/Z5q0x5IZ0ExDR8A\\\\\\\/6QQNw+JzF6RiJ8yb052ZMNZ55L62qwcSP27M23wOdtFTScq6jjKC7qWwOoMu9eD5mKfdpx5NbRY+iSItprdQ9+LkybnoL4oJcLs3NxqJk8lkdw5v2HdtA4YcGMrysdjVaROA4UdpsKqIVzPor+6eF0rJ+GrujEQi0gejXoGmgKzTalf1xpmkfw+oA0ZBJbeOJkv9uArDcTEecimG6opJJaz3VYSoeQz8mQqT0CfcP9ntiICX2O0CSAVFfZo7JvkeTeb+h02qf3PQcbdP0QkxasKb+mVIhaT1TW1RgnbDcpfKXXABPkA6qHH21Hgte6Y3pcGQYCjwJCZ8Y8NjNrMV1nYtBjqDUhF\\\\\\\/ccbZSLGLRy1d+vRoQLdu1qrP8I\\\\\\\/0tTATyHaXKw6JyqBjUpx69j6qfGUGTPYSYf+Ss287SHbTS+Qycm29H3q5eQL37bnLeu\\\\\\\/to2tpC4pimg0f3OSF1Dy3jp4B3uClMG2z2KsKGKCF+EK+CM1xVpRT0GRqiMGIcmtTd7wraWG8VTl2u+x1+JrbRC8a+XnRQRvwpLIWNsEPx0pGp1FImnega6MZFv8iNDI+3E6OOg+hk4riKlvQD7GA+MXj0SITvWKfMfQU0xYqQJ5FhyoHlxnAW8oeP6o9ZADn8ch2OsTrhjIBVE5sILUM89hKIPUOhQ3qw3BOHvG7+k28zGVrexOOl6XGj83g8dx6m9gHrqVYsOEj7vrII47KAUqMKtFbOGM9fst1tWL3\\\\\\\/f0TecfilsbHjpVVnaTmgNWttEBu0GtG0kJz9L36kssESleQpoROyX8id1LU6NfLAmuGn6tUK1PIHpsycY5LkTtDZA5c5nM0Srx+mBEhYj9DF10E\\\\\\\/oRP4xwAwKqxFd78QL8uvh2QmpKrJAu05ByILJh6t6NLRta8VtQarktbfPfCMPazinfBpXlCgdIeirBzrq3Z2dDUwGi+3\\\\\\\/lkMWXGYRY5TMXIuG54C4H7Yr3IUeuaCR0F9wIflWnZENPPwxf5sUbF1FmLvEhkzo5f5tYK3oyv0fADBShGDm73ydrEXsqrKS7EoXFIYBXB08+VjrGt46XPPgiVe+e6rRVQ09PRDnx+118wxikilSERsTybbXLjVn4cMDM5TJLSgLpy9CUo0cmksVQStCYrgtWlp6hphITjCxcf77g6d3PzeSRuyfHFLXHyQZTG2\\\\\\\/J8SWKwd4erDWbAW3UWdzwewzL6bLqK+xhmID2\\\\\\\/iHP+tv833Pdvlx51sAtxBQiOzs8DYC5gtNy7kJlD4\\\\\\\/8IAVPhg7fEm0OEO+ICpJrIAys8gno8j4dgNLyfsbVCkj3cFJtrLjXfffzuQ6rRLL3mk\\\\\\\/9x5\\\\\\\/SPeYBw9cfLWwXUh2Loh\\\\\\\/QRL1W+NNMIH+Kgc\\\\\\\/1MjcuOq0+QKYbIny6Gn0S1vUU0ZyafzsT6YUiY6mVOq1uY\\\\\\\/Z1YNweYDGewMTQaACdcHX1TaG81OSjFFZTWHHGEKq4EJPZ1kwFcyI7D6oyUXlxeGDOb+9VGrz+ZMmVy8dqNWUmq9CaP\\\\\\\/9gRNGiQmUlpf62ffg3dGOaCldLEBCuXkQnhUBQ9CwtO6PeiFBjs7sq15C4FFjOlTqxYSt\\\\\\\/b3M7\\\\\\\/FhcOIfqa\\\\\\\/QgAL2PP2zx8ILlsinmGDsn81LtcIl9Tx1KW6MZz5Fm9xcdajt0Xrf4aKPw7ScjE2TW9B7ezUsY+ptvSUOFBxYhlbpsf+kwWI1b698Dtn1s5+LP9DpzS30HQxOqoGoAbGFdLtQHsyzCcgd2h5Qfx1nrgxeNV9F33ADmEOo5IHW+a07EMt4qn6cNCECvnF99rJQQc71OqJ602fMQnx2WNthuowToRsiEOg3RPyD7WhrhbOwqckg+o0NZB\\\\\\\/sWaHUpVTPCZi2Xpbbu4G7a9V9Tv9NaUj59+OvPUIsMGSdyYsmYeo+e6dnZYrUTwU1HkrOeeTPtPuSGHbfJbnxxEVOxiQVe7Bt19c3iO6nZqNE3hYNfTp6Mexu6E5\\\\\\\/anoQFwknL8U19Kt0VpNcFuXRW5Nio88fa0ZuikePJIv\\\\\\\/482buojTSkxbe9FBjpukKAGLQ8shJpE3fZF\\\\\\\/iA\\\\\\\/ay350j7B+K1+JyTpFAzIVumZkdq1GcAwIzwziYf6CQ6BJnI4JcmBZG\\\\\\\/TiDOaj\\\\\\\/7KpESs8HnF34xI0vGlUvBcNB27DfuHAAQGwC6n\\\\\\\/YyOGkEX34L9crSM0Yzfs6w95tXnzVovdz9e69WD0we5zT8fEkyMwbVC7H1pFylm61IGbblrheeOvMCJBRlJPC5+KmPI3Yj053+cozcE6w\\\\\\\/ptJXFL587ZvoHhJlRI7d0fUkJtcbvn0Grp2Eb5au\\\\\\\/oePnMsE28jilG7v6yeutEgIsY66H9QsqgbOe0m6xWC7Ajzm9s8qgu8mB1STGL6TQJfeWNi+akcDJq8YeTDMLh6Pww9opcS2nSRAS9h34g8vmHegGS2ANpLTTaDr7Xpwej7FJSehOZ1dORYE+NltZd5HgCXubobefVf+1FgU36n7cHh2t4Zg8gxHpziI8BmCYrsUfA9PDPcIpo6qlZpkpBH4rAtZSRFAje4iExnSG74LVdralJaPWaIiEbE7zP90SI07PbDOrpVI6n4ZZr87EfVl8e66squNMf20a0D1XA5LG6vrD40KG27Lar9rIZazuTOt4N\\\\\\\/VxxCIWJz2KeUiGWuFm8essuvQ6HCdlmcxy46NQy5E4NQLIjvmxl3xqx\\\\\\\/xCYaCW5xl0eOVckAUrkAjOn2dWqjO1L3qc6DXIu89THbRxK9DkCNiTlaUqNLTrX7IQ8zG5qjoyIG0OxhzBuTSGnOohXZdbrR7UQsJ5UBrzMPjky90tIvpT1b3WT8CW2WPYeOH7yoDDc0B1uyugfFLBCo5R2bEqI1Az\\\\\\\/fSfg\\\\\\\/6nnrvdt8x3Lk+2zbm8H\\\\\\\/zMbzdzpDQQdHtQpEzIOxs37Xads5JNe8Y5ZtB3FGrCKYtAWyxr0BTxNIoztEPHsbga2S8HF6HKOCjR1CbnbwEZnVbhkUUMHL9JL3PGujYlks+gQmClUUoWNYF3zUbHRztG8ibVrT6FAq3NbUKeGIOHWHH3qaslnFBwppHrymRFeq7IsuZZfuVt+\\\\\\\/81Q3uF6vRh+djdSz9iWpG7lz0eLseXSc6Jlrs8N6pRIUBJ\\\\\\\/oivqb4btgIkD8IUCqEzVdt1Uc1yzWiqj6jPAIH\\\\\\\/HCpPmEspGbHUTbO\\\\\\\/QKmc4VLIFs\\\\\\\/e44smVUV+Z0u9HGejC85p\\\\\\\/m2wAfQuBhurY00qhA\\\\\\\/UMfvsRc\\\\\\\/WeDqsiebEvyfw4A1O+Hr91zYa2y6p7F6F0YEv7nlQbbg8Zm+xsk7nFkOyvKgiz3ocwlm2PHyNAzvp\\\\\\\/Apkpo2Kn3FdFtVdDHTJ9qtMSwnGD3l5S64P5IAaTIDVZIMj4aYqzXByn0rdMr0kwoOiqUEZumcSYT1LCc8luQcjRee8Mg++RIuYaxdNIb8d1rlyRZSrNaYNd82Cfgc12VLsc91xEmgXIJ+MImxPvyH1AsY+xrir9\\\\\\\/9PLwT7aqOLnychu3tbB\\\\\\\/1YjrzyXqdRlMroPloUF+9BnUljn65VCPNfeS9yd8P3QKE45vJkjqEwJIg4sYOYNEJHMtkYm7IwJ4r2G1zwFJqPnqPWUx+3i74jr9H1o7OmtTSBGF59OcxqmIGFap3XQPzhCqQMRNFaxwx5WIpF45zB7+XEaA0KL18RjaYdgZiRKTvGSsDqXiAJNXkmd+I3JCAJABOWQwWZC+QMUIChErp4sjihAOaRHxbj5D5tpUDAVi3MpEj\\\\\\\/r0UE75Xtglavjuks5fGNCfe6QkMw9fe9LHldLgSRQyaz2sFIyH84UjeaNeZDAkoScZJrbcLojLKrX\\\\\\\/n\\\\\\\/YZNMoznJF7WinXf8p6MWlzCfIgLYgaAwnzPHpVCYIyAMtl8yS4gXUJijW1AzD4KdRvmBf69wxMDgF7+FkXTdUbvWBP2Hx7Xekj5AndKVxLM14YWIssZCDhBb0osXw0qQoCY7ZX\\\\\\\/dDX5ibIy4W9ZWNHvYKDHInkIYN0DY\\\\\\\/SeF4anKpr52Zl3jqbkTRPRnodiSl1GF3RGlNF6z5WDSW68EC1Ukn\\\\\\\/znC2PhLneA9gotWPBzluijEiKzLJd9QuRqjKkF4LqvoajW9Nhk6n9cqpVWYQSGWaL5kxJPxAjPbhAKUZmo65aFaEdUvfQ+ZxCHJnXRbtGYEHD9RoxBiypFzwo0Aq7vKWSvip+Of9VxF8MzmqmNP4vBZd4XM55hZaTmAYZg9XrfH+8J30YKzyhNtcstOqERnBMVtd\\\\\\\/s2zypw9X3CcomfeN1NsSwcAM1+ih2wownpX2vIb9SGLILaLxonEC9U6XccIQBcUAF3utyp\\\\\\\/IDEQmqnzE9dCCGmjnQyNEbSngAPQMyNrJZiwLCs33PzzChZeIvS6Yzi6X2xVBuK2QkXZT5n9xiFxSllnpJ1LVe9bnxjTuEcEuYMU1BcXqcxHA+DV9FvvJpXsVASTNv7C7xe+aq4yPBlMqgSxVsD2wae7wGVDekqP7eFHSXkdMKIKjsEt52KZJruQgB2q85\\\\\\\/FYHVjXWzoT6M5WaRHZ7X5HGlukpLqswkCVgNSdxP0K6Ci6p68lPYk9g8KTWgT7dn3L8wyftQXyKx\\\\\\\/jbfGTgQhdXTxPv0HwjZGG\\\\\\\/JMj4Vzmc3cRJgZ0UkQ5KxAjkXhDbLO0nVgkfDOgCEY2\\\\\\\/5K\\\\\\\/yug5Nq+tQYyoHWXHc8EHW8dEZoTWhnckI2dVJLWELnaL4IXC747L0YELE62iDHPrK57tPGv+1Wgdlhy4I6KGBzYEKfFyfZWVYRgflhyC2dHKxtDA0HLWt6aP9g6nPow9yuuGP1oY1Rs5H9MXsWQSQmaKwrKJXzpM17QTSr8LAhi6tbNV6ykzlarieCvzSn5WFpzAuGVkh06Bo9kcfpFoqu7x96FPFYF9TeOnHapFuXrHMHbNUsrqkAvfpc28ITkrfGG6ayU0gS+vtwg1fEojxMDI8\\\\\\\/4NXz3w6ENQoJlCOelvtN5i8DigZhu88rubdqeLbVUUEpVe9ycWq8okCdUGdgRYWycLxn\\\\\\\/Z8J3TeG6WLnFyKRbYVi1O0N7skMppVOq9PWRBHgNF0pg6iJPA+IXSaUeEMA\\\\\\\/M8ZzdnbY3jus539qPmp5E\\\\\\\/MBVO5BK63dTF9QdvOUtvvwaE5v43QjosEoTEjuoIGuVMrHgcfFZBugqjjg18VUWSeNkuC9Y\\\\\\\/IYV4T8iz0nuZ7loUaZ8cS0RT3Yatv5rAwvIBddcm5FLnpVtZaOteyL7pLL8k8eO4hMrxn+a3q+eZRgb4EmS\\\\\\\/2+ota9XX\\\\\\\/TgcmqUm0B1rae+eLz3WmHianrE430s6+ceGjZb1FPV4J3B5Ob3U\\\\\\\/9ICmnB8IAP+KqA9vuhG3iZpkkDRYm7VfihksibiAM0hFbXMgrBxHACHPHWXRew99w24NlgjlwM8jZJgb4KefKSYBrC12hhEoCM6Mm1MqB01gMmTjnqn97O4\\\\\\\/iRruGocN4LV2TjmrK+JJ5kgIF+CQyUX+tuARRR9Lp2KZkm1VpnHIugGpS8bTqogfSsuL7CY27F0bh8iKz+fUaRvt52qtrqw0sZsuaZN8XjiZYKSfoAaqsRWgCTp4jGb1Zo+fypZVh\\\\\\\/Sqp17Zyx1eRgLHk6Rz3GV3yzyl9Rk671Gw1htPo6mqPs93Wmw5+Q\\\\\\\/8EL8Lt0gh+Y0oCJzUDnUnn34\\\\\\\/BMyw8FN9tg7tgA\\\\\\\/FGCh2Pom4YCDuSLp9YTDIh69NjgC9sehlQfx2ZxNblNW0OSkFBcftXnJlteHAR+KzCocdgoLeV9\\\\\\\/JandDLaafO8fWWlm7q3mjRxAZp4FjURk2x90mYQl3ycrYHFVfMHnrOeI67P+D4+TFiLb3Pz8rCTB\\\\\\\/+zENEj3fr4l7esjxH\\\\\\\/w1o7qDugVun2O\\\\\\\/ORScXXNut3vuwk6oGgiAYeaQmX2K7UfVIQO818bFn3fNUPPhxmTWpEPDUvNqpQchiHUp4+JetlkJngyMNvsCFeK6lJQdhpu7z7pQ48qqRY2e7sRDQWs+xAWKLwIoB9lZ8d7XjnnXzuWFRTpsD68AeHT16gpRohei\\\\\\\/K+ojmBkTQ7UmMmnRujSljBslsd1yz7u9hhBn1xPHG1OlD2WEMaUohTOB2VWRAnK+\\\\\\\/gM1OpFFEc0w8kXHF01N37RsYjTCfooArk7f93ducoNW0zgWlow9Zsmqg2jmj3UUCpKY36I3swCW3rj7dXYZPxitNU5ifRRmfinvdLBSASL6mXP5wdSbLetgDWzR58p3M+zlhLNBABf3qTWTNsmAmOBCdpg8WcqVamVwMiopJJIOFwGqllTmcLZO70oUvRsPiIPCNvpk6ibSDUhL3E67wwWpoLyG5JJRcNVyw0n4XCMye4NT8gS9d1gWsE1ybtO7Fdjr8MEwwn3picoaulU1ryZ6MvcVgADd\\\\\\\/zX\\\\\\\/x\\\\\\\/5pLbYyXTUoUTaIyuB5QK4PyjQBWukK4bGr69qC\\\\\\\/NyeWzUL9LIWN4OnYHqPo8GFgsNWrQPHcV7+taGLTH3RCD4a4BAyLO8DBHnZEDBVPJTA5salq4l44\\\\\\\/zHZ8x4HAegbxquDynoBMOavGD+ZHdCDSMUnt+2hng1QvdNaNOHnAK0XY+lqurYJ11muIxZjenjYmlm2DkIA58EmJ9kAR4e3vN+dSF4fOv5XeN4Ewut0LRQp4rHFfgbXSV38n1Geu6oLY8dvuUu1vqstVgjQJ+5yfYxnFC3\\\\\\\/N4m97Ekd3nCgmnnwdhoKxzEabDJ0ljNEurWl4y\\\\\\\/YIVbbZwg61NGUlCd47cDAV5NfpskypWrluOzKTyfRpMuW3gay4lxHMJhp+iOGbyuuoH\\\\\\\/gCtak1GF0\\\\\\\/BgxIeziTVtRcrjvkU6L0XNaMHDdnPEXBWup0Y4Qlp3izbsWAhdcZUaNEQA1NwcvPv4ZsyEqqorAgqxguuPn+BrqU++6mUyydTeiqIdha4MVQ6w8TBMYB9nXQReT\\\\\\\/H8o32LsWQaT0a\\\\\\\/QWg8uuLH3NY6zSArGg2aQQ3BkdEgOP+zcBgk9vmaQlhQUyCTimpHHqfqIZNZc6eCSibG7UoPDEu5AKBsy8kij9etLUUsgGMbnbaO5fOF0kyWmk+lqa7rfiVWFWjpoud8NImA46htmsEpKKFvkaZ9+7QkCfUqtmd616mEOlrwZZhaHVCZ6khg3hm\\\\\\\/1m9Z52JiSkTMOgPrKTkfCW3ro5E9\\\\\\\/kgcUZzVboXihVBxBO57pLzqaOuTPxRNO8lFw3MhBAcKv3VlFnDUeFr\\\\\\\/x3IaGzf0ytwJpe7kC73vhwHtuBKBEhYKKKiJZwKBQvMJoUcJEcNqsgAwxBwuOOA9q7Io9Eltiz8JzzwaKeU0Ih8C9UpCosPJbjNTFRBTVmqHnTpcJ02IR9F7WRzXauGykA8ZPGB5R000WFrh+JXNu4hgYS8KJo7DGG2+Eol5\\\\\\\/KvvcO1Scr1EhMONrwNLb04tG2CCdX\\\\\\\/v6XW0\\\\\\\/jLHAD7aozdQKz9dulqXJ9NrDp36tU5Po6zWBn51ZM3tVa7AyvfARwXlSuSsAPQeNtxMPVAbN27WE6SRebZVnuyfvUhGv2ZXgCANtJzhmfsQu23Pvh9Mn1gtF\\\\\\\/+9NrCstpvzo16HDiRRTCsJ1qxL8CvBbTXR0qU\\\\\\\/kU29mSKwWdZMXjpXm9gyWagW2Ci++mHvnC+E18W02j5HBdxW9AtzSRWa220Go67qdFekRrgtjc4\\\\\\\/Biz0bi+VIywKgAJ3ufX0Bg1egIu1\\\\\\\/O0sf4KRnKIVYZjRWqLx8uJUBkd\\\\\\\/QR0CIutz8B3luivR4AWz8cINmC4p10DSJany9m5+gTMnOooaIdmFD\\\\\\\/wCmfLvEQ3nTbZsuZdVRsYhTvVS8gzl800BfD0vTX574gMnpP4ooIo5a7zvsBuxpeHMGBH79ScCX25S4OGx8FqbOZL8imumaqEd9ke1TjMyp3tycLPW9LRPP\\\\\\\/Z09\\\\\\\/Ku9yEHt\\\\\\\/mMYiTDy9XQMJZeRMlhTyPwU+tfmI1Jv5r8LGJaVRXUqEb8SAGR\\\\\\\/8fsjZuuTI9f1rh1Z\\\\\\\/nWCL5gjBkqZsiofK6YsJ03jS5U1\\\\\\\/KvwJIw9GwndSWQIjDUw6pGuAz5sx4a773sxP3AkmkzrVL4E5dHN1aA4mmGCrA2kYwT4KrkBpGHSLZ9eqEqFPBcX+9pXrMSZ3U8eoMAcdFA2YIJNYSdW0tV\\\\\\\/6bdqjFERWAZV6l+HZtCETg7TAnnX6fLiBl2L3q4OVcExX9ulOwtbtLPwNgKpZk\\\\\\\/mPsHtfduAL8QRa1+PMFdUOJnYGCu5jh20QofJT5BXmxiFfhuX223gh\\\\\\\/hIA4vU\\\\\\\/r68ITctcfso9r5RmitrOzauzZ3PR2bxWu422R\\\\\\\/AmwTt1lryoQ3neGk+z929uHar0iTC95EJqSgUV138UDGBnlYEnx\\\\\\\/9vdrtsX6WH+CfU0fyOZTl\\\\\\\/h\\\\\\\/wmZAwFdfXKrVEFbVEJiCvJ461LKVYzfsjI56E0CktkQbnbVNLSktEqzEDGGzj4O+9FdSFzkOA3MA1ZOyQAZgdXyvQHMKhl4fJ+eSAYBLmqaJ8laWYF3APWH+vcgz+efJ9G3saRUHHWE485wZF7YM6KGi3fGghdidnqC5lbuxsmfzjDjwcLx9QpCqAyAlC2Hyma9WSpfxYcnDbngdYvjJRv50trCy1kD34mJ44\\\\\\\/+1jGDNaH24FulxhY\\\\\\\/aJWa\\\\\\\/zkPs5PcSnX0gJaPn144mVxv6t1Nv6r6FM1xB3mgmISNYbmTzIijwcR+5LNUNZoAGpezwr4NRtXd\\\\\\\/Y3Zb25u00nIAb3vFE9w6Kav850dTuXiZwDnqUxg+0Vvqtk0y4DYc1bKe7PxrWo+2Tm5+a945JU8\\\\\\\/5lsAYw1Jbbp6a9G4bIoA2mfhJNIkfFwv3j+CY0uNMKtWvAm0QoA8GPYbiv61oyz52bn6PY1oiXNGnaSy9pbOTPlWPQDwU02iIDQeghA7a9DNX\\\\\\\/7RQ2yHTlie8\\\\\\\/1X7FR8oTUyac0Qc4xK+ht9HJAl+GODdxnqpl7oVRHdIsbWsfiZdgSc5FOVgPtixfv+bDtk71Cqk0NnqNPCK64cpoGZa\\\\\\\/ETARq\\\\\\\/9Vo9rQnhpvEmzbWiX\\\\\\\/02d\\\\\\\/g4ZU5nNuY4+pAOJi9gbcmEDDC7UcbHkZ+YV2p8cwbFkBMESR4LCh5+BW+rK7u1MfveW1vHJQVbdLwU92aXBhTrekEm8FwNg2xtO\\\\\\\/S\\\\\\\/vW03PKEy4VnQgR7py1bBfFi2cixYgc+8EY7xv2FH0H55fc0Ji00ASm7xUmOmY0Lg\\\\\\\/F6KhuHI38L9mTnTU1eWglmIXXOnaCvjYX9FuiAaKiCmj+iH2LpI1NPRPmCYR56vs1y6LcI9RH5k9nCN\\\\\\\/t5HfwI1Jc0JfQ96ywDHlOHGOpCcwB23kfRdHqFCMMbj8bW6BM0bcNnx\\\\\\\/sL\\\\\\\/DmhkcuRI9AI4+nbJKz6d9\\\\\\\/Y\\\\\\\/isKDJvmrMR8bYkkDg4E61aOTf1v1BU2ivAuPHljZIS0V20sqH+4zqJox7i566zOM\\\\\\\/ZEJb3tBbTBlJ8kurLMrRhxpDoCcrD\\\\\\\/AjcN1hnyk2CRJABuA8lFobgrjxOTmAoaE4u14zS6DD6ig0+h\\\\\\\/8kaY\\\\\\\/\\\\\\\/CHcObXtxWzzz0kaXRnspossvMym+2jr8lNEtimCYyOmm1vTuWZBvqzGpRnqMSOJ+NOa24PaMn7UBZo2WxvkffTe256\\\\\\\/jN1nongN7p\\\\\\\/w1LnuwstjSK3FdHRb2znhLtJ05oq5pGSlKcmbLvxDchXC3AklYfOuGpMdl+cGngHIjvi9X4v448TuvMok0GvooYptavQDmU5LkQkE1iDr53sA0y0EBlz5G7eAEC0cWVgBWp8trpZGW8rQG3pxqmT0YtOh1hVv1M9K+R2epXGfiTkhAwVKLsbQYprjeB2DabLuHNj6RHAyyEQ0DG0C4jsYiBrka8O3sHMEunMe9Ba\\\\\\\/Z4X6ZWPf7hFMCbBZ5y+JFeYAATgs2yqb\\\\\\\/u57I3sLaHbEhL7DC2oHO3xBaBtbIBh33hsDR4imqsSh1g67\\\\\\\/zpecG8fjBBjd\\\\\\\/RXfCi\\\\\\\/F\\\\\\\/bVYGizsaocPSGd09EUnvAbS0SIuiIPEiaUOT6tFQSNFeIIfPabll0N89iNgB9LuTAJoGMy5Aog9HCvMQdb3XWgDvwih+ioCj1E9YpVaUoDWr+6g+1Hr\\\\\\\/yCC2G3bDeHlYpEKOJJU42k78MRLEHuiKR45spp1dyw44K+tBX4foxzp1+sYV4q9g1OQL0lAJVOeG3dtcn3Szsp9tQtcm6ntokSdRKAwmeUL+q7LgaA2ZyMLDk\\\\\\\/WJ6X4rgACTWC4Oy++iUDFfgziCKxtydPx4zPHPJKTZgTQRXgG9f6eriWNac0MD5Da2dPUITYJY95HWBPWstPj\\\\\\\/ILEb2SpWw0JZWoaTIYl\\\\\\\/rFKyl0yaEARn2w7ObkKEu+kjmn6DPP6Xv3p1ETPDyWTPORPFKZ9xIdZtMvjH269hUf86G\\\\\\\/BCH8bKachQfGRfnd\\\\\\\/0zVENO4DHNkDP0P+WbjZrp4PixoXKH7Cr+8DpwU03bNoJn0X85oevFuTP\\\\\\\/c3xozydPJgfoCwBhRBqd54QFuJp0ZK5bjrQT2gaa28WAezI2CDLddTo\\\\\\\/5MW+WKxtICpeqL+XKbqHq+ZHCZ3xLWVwq\\\\\\\/S4UjztMbNRxuOv7UMnUB7Zd0teqpiroVfALuSjQEgeQ3OM3eLU2QkK7ettyrzg27LqQnb\\\\\\\/l7Vx3nNECwSgF0yn8t\\\\\\\/oGH0bO6IowoBhCNOwtrIHv356vQ3wJO18r0V217s71MqN30KDbWRme\\\\\\\/zyHWjU5qnC\\\\\\\/ORH1aG8dPNCPb86GZWjeRLlS35BBoIK8L1prg7WrKLIrroHjHdREwXzXCSTciLvXNlxhNdpn\\\\\\\/EidMCZqft3ttnNz24yufWLVBibc8jSKjecV6+39BWXAYBU0tjdUY\\\\\\\/mXlEgY0DdshBT9YJKy1vdpvLnxAC7yVajVBcGUsQ1zII\\\\\\\/1KbNpRpv1ZrgFAhAN+\\\\\\\/U0UauKTs5QsmPkcQmo9oZc+itx2choRk3BEkVhz9sBP60Iz+tEZgY+EgxMUqyJcJ3T5nktTsp\\\\\\\/yZFpJVv1JgQAD6I19LiNaXbuFdkU6d7MgxhzvdvcyyeuXUgZR0ns5K6M0phQyUvKQfM5imrs5khPUfFVFRGSq23j4CY6V60AqA7\\\\\\\/+aukhGqlam2gpfIEyHDcrkJglefvQCrmacjCG3b4QwLeDiRxTdSU6h+PFE64P7uCYweUXNl9vE3LHrnG1tzH4gl0Hz+Uix5Mmq4PrxDGeh2JCvTJejBaRdkajAxP\\\\\\\/EBOijAh3DxX0hgwnzbzv9I\\\\\\\/tmX5srpexrrhuzxknXfiXkqSOXnh6jiUjCDmdqe5x\\\\\\\/sOLREEFv+aVJb3ULPEWusJjUYSCtA2+oBKE0IywOCWCBVO4HcmOptFThD9S0h6dvpVS7581zfIeuwnHAXtLD3dq+yW2uMtKmvYLEiyC9cIeQwX0D5UYaVokO6s+7Kta7IagQK8k\\\\\\\/RKyW6NhEHgOAtk+Saztau5Oj5fCZRwwCOJNe3w98XuAbwGnVB+cTVWA7Npx6jBGhkQOABTPxV8+e0xo70QUmE6Rfrtx\\\\\\\/vM2Kr6cNWrE02flVmph1nGhNWKogLnIlR8KvF8lN+\\\\\\\/0as6sNICGmTsvtsUJXrhAi6Pv19UzoacE\\\\\\\/4sftKB67+KVUjdYu41vojQIwrxshUNaeBdV3L2ZKlR8AoHquO9cn4Vr0RZlAEDoF6g7WjDGaro59RD5gG60u4+wvpCOBtcI13ULblVhmapR0d1bFFwalUVM8J\\\\\\\/+qv\\\\\\\/eETCAlVe9NlOOqVP95c6WIxWfWfsPgGEJcg4Qs+I632w5FjuFJl8M1VjHgmS4+HlGSsdZsK6GI6uPPuUJ80aPa9t5vUGIGGKSWLhT419x5we6OZdY8BUgFPjnDnTyN\\\\\\\/g8k7fc6F9ww5PePMu+DbxXQjFUOpU5c7vHby6GLgOFvcS0Mlgr83Jd+WKhKAXM9RClXHH5FhZEnMYC\\\\\\\/530kV51R0xkR7Bbh1fs3JX1hEVVQHrkcP3keEvZFRj4Mme5X4rv2ZY4YCJw0tzabvHNnHrRxPeNtyTj3zpLgJE5zROvH1q6ov7klr6+dTTWOWr1tunuNUf6EzMG8JKS7\\\\\\\/3tCnUyAZAl9ET51Mw9+jbrmyT5uvnskll3Bpz7uUHFf3w64Hz2dPFdd9uG\\\\\\\/QiL2t6js4hO5S5CimTaCiX4eE55gEoaMPt7rmMoMUexnLQ0EFCQrZ9Kwz0QzfU\\\\\\\/7gVlJl9tjyD4GSuZKj6ClNgfWR41HG\\\\\\\/0z2g6UFPMaFgXezpV6ujwml1nfBfofo+a0\\\\\\\/kwNaJ4tNbYEmucTEOTRjUOSZQYmBmVVg5XTNNlH41C+2pSCl0BBqcUasblL\\\\\\\/VFnPVGLNGZgCl6aYA813WsO2+ohvHS8GxeAnaU2F0by0C4YkvEPu0jhyl06WbF\\\\\\\/Py6B9R81BeiIihXhYNYifHl\\\\\\\/BLzHpdFHEtOZyo9t92d+7+b6WpgzI6jBeVgEVj2H3yWqoRQB+TM5b53n4wJlZ2xPxSNDDYXz4CSgo4f15rfunH6iuHmJc8Xf5D15VgBKMjf\\\\\\\/6sB9uxDl9qZ13QkkV\\\\\\\/O6NLPuJJYPlwFk9wdnufFZtfkl3EvnPLR7qKNG9SInK6rqT1GgtmQOusNSGugQvqmaEW9PNZG\\\\\\\/WB6jvWeDgYtiXIjEt++9fME51SudV6x6DMpgpbNOWqYruYntB0CwuhwGtuCxWTS0XLu3fM4rHJtCQhLDqxTMIEwe5AQwPysieTfa3aaIF9elzWBtXSxL3hwrD8jTJV0oUJvuQUBS+KMlwZ5EyL3KF0J60wfXWZlGoufkk5FMYMkbN5TYm142fsG7x0VRD9D59JBt91e66iAfrLTsPkB\\\\\\\/xw+zcxSgsHbREhJYGYKumCXZzR1TvWuhcaOioV4AaQ148l20K5E\\\\\\\/RnMOvUZMgspzGzay2DBMqP0WD5sfkUGyzzicbn7esdp+GRGq+DDMW9FznXMoUkb0hQL9s5ZEsz60a6y\\\\\\\/3eB\\\\\\\/KrX+DvtjDEffUxtccykFoBahBGkkQqQ6q\\\\\\\/ckrn19Y31qWnKvpZvxUJ\\\\\\\/XV9QFlarkvshp\\\\\\\/CS\\\\\\\/6X9ecNW1n7smiwA3CqExk8taE4Sh2FVRKIgYA9SkRTE+gRQSzz4BNxx2B0Xp+K3s\\\\\\\/P8qFma3VDUkuh1hf84YwCqCufKa5UQPJAjAc5RkdnQJlZOnddJpNcZoA+15x8mxdr1ZKZJOolVArUwENuGM9OOYRLfT7cMLIBHJTeDPZawh6q3NFg6KhLFCGvjFTJGJnACYRkcx8Ckts3p0FAzqCJc9Av1UZpJu5MlNgTv6Z1\\\\\\\/py0+S88GfhBrset8og9Q3qwfj9x\\\\\\\/zSl8NRi\\\\\\\/QixE7Mc9XYdPIqcZ\\\\\\\/RQHCV\\\\\\\/rWaG04rv29Kwze\\\\\\\/XldtejynoWnYnUWijrZsKkfc4m6k++XZCfiDTox5QJk1yLWRqNvPIXp3zbEEv02ot6JDyNYgT3ySChraX9OpDpG8Hh9rjPw2TvqWOWoMBir+KvZHJHPju41AjmcQwqpLEulEJ5nHUEsfBhbjQRoehfMwWKrf\\\\\\\/7khMskruRcaFDzD2cw537LTOPUwFrHgjUJK3S69gCVu1O\\\\\\\/meHmaLmbd0GOxRoviFMKtRovMfykgdJuEV4sBpEchzDGGnywquUtz+ght5v5gZB9kO+STnFIYrsbklTJwwPnxqsMRXXhtH9wR+CBlCkcIXMWT2a+LLfnD4Ap\\\\\\\/JQq0jDdA\\\\\\\/jRHm4R8e3GX8GiMX2rPFA7jDUb3YQQSRHvgPt6n4\\\\\\\/05posjpP31X7BSUlMoOGfCnGR5se5+h3m2QKi9PRecuHNvv\\\\\\\/Faz\\\\\\\/QGuo8q98PI3RKo1QYC54O6h7Fe+gBGh6JwXhvXUPo3lvnL9hM+\\\\\\\/jZYrFufc5B6HRsTNJhk7fcQyTUMskd1Phr3TVKydrl\\\\\\\/MFbMymEzz4Gwiom1cl30aX+VAgYhxKdx+xXJOKvQbQeA0tEE4895Vpp3yk46+x7ylz8Bb7mnQvT3j4WM77qHjI+FS3mf+7QzlIIeuEuB+takxdZaJ6tY7Rh8ZSNUntsC+9KdZVMYUGr+eTrEQu\\\\\\\/KIARZRwgBJjO83Q5bzYxCUfE3bB\\\\\\\/5ffjXTLQGIh3t8KvABiiyO\\\\\\\/PJ2Rw7FnqbmKrRIpCg8rclroNJ0+IBXhX6kw+3gISQmVNEP2J+pnpfCmIWsLpzIBBxtGR7UNwoB6OEZF8\\\\\\\/uQqoEQsJ7Z5xyAeK5F5oQIR53c3Ekwo\\\\\\\/u\\\\\\\/PKtfUy3OqPYadifSwpFKY0B44yj\\\\\\\/HBM7CiJvN\\\\\\\/T\\\\\\\/HbLAz5dQKdpx5UBn74H+w+VevMsUYTsEGwXBUS7ty8SUq2zf4kDYXczrnYp9vsKBucgVHVw2BHjWHFPvKZh2V8a\\\\\\\/WFuM8W296TzHDGydX6Vn1NXYGGe9iPp1znyq2PxosAPt621Bu46YKB4X9uQ4Ajo8a\\\\\\\/+upa9WNVp0pW\\\\\\\/mgjPRR0yBSKZ9mhajDkex654XEzt20pfp\\\\\\\/EHPefMDtOTuJM5Xgxjy5yVaUBYoyExdjYfi+9\\\\\\\/6CDK1rbyq1Twr+AJJNv55d7SkX6srb9Q9qA\\\\\\\/lHfOqz4hO8B7\\\\\\\/OWhI+KczImXSRCCJC83Kg9iAMuK4LTJN7lgHIYeL2o8ifoz3a7cr\\\\\\\/r2G88A3YaAzy+72\\\\\\\/gTQ4tt+uwHoVhxYjEV1pGVJULd9oKa95WoSZR88QviAW1pw1PtM\\\\\\\/wRWg10J14o5qMAsfLd4ZKUoIHHq\\\\\\\/kSrS\\\\\\\/70gdbd7S9JGb9haAUXcivWmFHe19ZnpPWO0u8kknJN+IKLpiyioYhqk+UPOgiSOuVm0wuQIiZuB0ICHnogCBfCpkekXoND\\\\\\\/QP+W47ipzlh9MyFa13Lkv91nUjhbnZQuNLjaZ\\\\\\\/fL0N8jQtYnCD\\\\\\\/LYbYMuS+T4EmHs\\\\\\\/Dgofiw60CF+2Qp6DKnA7PeMp\\\\\\\/ENr\\\\\\\/lM9be7dKdoEGcxknDOu5r5FukvK+PavacCgHVoE1ku2ogRAfOZ8exDgtC9TBvLIlY1UIRikZhNWadrKhNZT6FhBQDGQuoI3mxBMLMZPa626+4DEik+Qw9hPJ3wYN22G627sZFgEUXvy6yWQnEdsjftUCELcpNK\\\\\\\/IK\\\\\\\/eTsXhw7F77UJJVSEiea5ps5kiVNG6qBc+ALry1wH5H6tFAX\\\\\\\/Q30Y+Ee6Ln9bJa8XLCWf8OMqcHTdblZJy\\\\\\\/1EbqYZA1cpOkHJHGLg6iEYieOZCAsh0kZ1Tn49mx8GaEQX+w+niuoaunuj\\\\\\\/NdeCducQpeRdWCrQZHnR\\\\\\\/ICsuMNVDm+uTotCC8CWtxI0OvVVGyvxoBdxfKm9bpfH0+gakc\\\\\\\/6Woax6I\\\\\\\/KV58iPtypojVQ0RJwTUbd\\\\\\\/izQ51FNyRFGIWZWUuTZ0NKw89N7wpbgnvPcPW8ljSrF09vhgzSj4eK0F2bw6bGbcIDfhd1rbYPBBcmujYPIYH4Jb4WLHXyj3BNaV9EA+e4Oz2aEnQOUVUrec0Cf6JKnUEPvJIM+jaCPL9GhyEcES\\\\\\\/5evoXzRzdgGVXTLuztdl8ZZLV4UmaZdfnkisjtMr\\\\\\\/wgWlP08I9Q7l5biNzhd5EA\\\\\\\/iHDIU7FhaK2mSTrNID5L2RBIB5QiFS6qEEvS+w3pog9KyLgtykZDDR8EgJi\\\\\\\/cSoevhw8MRV4Mb4Ab4BVZlRZipKjUfFWDc5kNpXjNzdGvXzKBUOysBHNclFir3qaQ4YXNBKFiOVDhqVU61Le27EMifGy9BAxrLcZ+MmrOqbvbKo+5tKXJDAL4MMk8hkRNCEtPrf4OEt\\\\\\\/zJUBtg5g1hZmFjt++1z6e0weAMyfOp\\\\\\\/BKxpTKXPL7StzGagaPRMP+KeU5QBEui5XLChH9dWjrrRNcXjxFAlCoCJl6GzOSU6rXlMvMnid5EsXXMR6mkgjNgLCDeh2S1M01lEFeRb8EK8UpiQN1owJcfiboc7JAFl+VDfzSfrfd2SLvVltBAJwLXhM5Cbl4Uq6WYR8JpITGDNzUYRUArhmY5OCn9N\\\\\\\/xpGMZo9lHgCJ+6NzQZvtGL1hAZ8PNNNeU\\\\\\\/BrS0mIbTJ\\\\\\\/56Cwh73zh8zeZ5sctQOffZRcqerFsQffhzj4fG+T5vf8b9DwJTc3s9b2Q6QwPjIETpfADljilZtSjMKLkbBX9SAFzCBojHL4S5j8FXw\\\\\\\/Ovi\\\\\\\/NCScy7mqvQSvlZJBOJER6HFbMe1dOQAquKzQFoDUHQTMNc4YZ3MGHCr+FodF7C37A5t3Zjs7klmZ\\\\\\\/mq8+qazMND+wrbK6nz0vR4AsI3eXkyqgBzmDPcJFH8\\\\\\\/OWXja0Sk5A98X64epcAPuWT8FfH2GIyWlaA\\\\\\\/WWCA9pPyS2eEHA9PrrY28i+4ikBWruHC91Am\\\\\\\/J9BwiJ4S\\\\\\\/IcZ9G2fA7FesOgKWjnI6kvX5lfUTtj5UAJpb6cgK0tSCdYXjmcl5IOTe6+O6L4JApy3tTEBfWvmXCg36ju43X8ViKtEg2crWClLdPrGiTUdyOeX5RKbgJtCyFPl+ktaOMnseqCpR\\\\\\\/r41uP6u3eklSp5hb3hxWN4PkPDQGwGMfgAbp8seaCsLzZyAdAn7B4\\\\\\\/xgnLmaO7xavQvv+gMh3LQm6FieMi9XGdxh\\\\\\\/FUmn+ZbG1VblTwtSD73pYkra8YVH8NbY1Ano3eaF++Ja8rH8BbW3wxhXvbQRGmcA7DA4A3aUywW\\\\\\\/AOIapADgwhKXN8zcNA3DN9H7tnRLIU1S0MWLWIUNNZffVqnIOkrNLE1Xq8hBuIbRXFg3\\\\\\\/hLbm2eg6coanxMKk8HU4a8gyaDkfh62HhZdjB6kk4nnvnqHPK3\\\\\\\/hSIG9UiZIVJDTkDDbn29C3zsV6YnlsPgVT2r8Mt5tprl5pVA4DkqMFtQ3psaYEItLq++ZbbjRub0V2lC78\\\\\\\/YUta53u3ZVJsHbhGeaVsCRUZ6ncdXn+9RIKrm3\\\\\\\/eYJEbSax4DLqhK1mDoDKXDj9YHZcbiTODSgwUX12IjPBAzakHpr\\\\\\\/W8UWTU0KfmoOUKmGkgNZZzPkl2zPer7JtP7C1O1I9pWtNjQOBL2Og2eqwoAxTDiIM5hq5dioNA58zBL+40BqflYd9UcN7hpq1OAk3VGKgKuQGYnUTMv8YGThiwZJiVtkqbQwGNdDOsd\\\\\\\/h0\\\\\\\/EJMZTqvbNSTEEiiLB6JqiFbqkvVDcn1ktROUEp1V2v9p2Gm9cGvyyvRKzrjg+4wUPUc7PlkWPfOHqXpqXm4bUiarZtw\\\\\\\/HKaEGpoW9ht3Y5hKMm91onilp11mxVXfjn+9LamMwiXy106XU4wwzj2l2PJhEAX9ytlJBJo8r\\\\\\\/wm75f4cxvVcpfMI98gkpBG\\\\\\\/+PQ4HBbHA\\\\\\\/PjqvVdSJAMQ8hNsfgU0hjAuyVC+1jZw+l8kBBR7pU3aIek5yUSI0RSFdhZhMJMs6Fg+DTYFOClJku8hPZVFfkZ2F1gyJa29uOV64eBhKNnVwJR+\\\\\\\/5xNwA70w+6+O1c66i+CS0B7uEa0NWWINja2h0Y66IGLhGWZzOCI+Zj7DsYfWB3jqT8MSG9TRDJEMAljJJbq5It09IIWXjmKQ2xFzOC0KQe7RGO4MtQUN8rcxNBkMN\\\\\\\/+HVwkmW5FsdEbTY6+SKjPq2TX1nFZFtlsipnfBn+bfZzJanI0qm4nsSmxAOyVuqnOTQR1rO6GN9sekz54l6Ifg4dGJWruICpuBtPqmeCLfUzGdhABp4sVFA\\\\\\\/SGJk4hV6HuMxLAjt9pYwA9XrAncBFcqnbeRCFnTPS63FvXTVgEZlFZryjRpDhkV5ld3rF5gCFrEIUAxQ7iqPFSW44TJOddgLvZu8z\\\\\\\/7MGV5rYgZI2NGgv3gXeu9ktPz+JZju438golRqv\\\\\\\/VR\\\\\\\/iAnBWArpSGoiwXnPD2fdKa\\\\\\\/xne41MnD+UQndH1+Qjd0P5RvYst7YXdO93PxxzAeLVCnr\\\\\\\/pjyVN6NmqC4ZPtF4pAt9aMZ\\\\\\\/OdPgEfSC3nndzfqUxJzy9uPEpz+3OPnRYB243A6D1vNx3Spns+9B4FZCknHxrGZklm3G\\\\\\\/zy9Qi98rkAs7bhDAx5udfax6CYclYXUDlaQ2lQCB0cpgy70F27pXQ\\\\\\\/yC41RejEA1ICuP3qEP06VCW3O4F2RZWvRCt8gVKeSRug8dBPZoPQ+NcqF8BNJhu\\\\\\\/HexT4ezWO1Padq7De3SRSPeSKK+4PmOLnymYwjcQ47H7w2hxg2M1TUjZ+VE8xUApafpLX1eA4eAVSE1L\\\\\\\/ijUHwJjH1EJg2iRTVTblSV2cmS\\\\\\\/gNh29uC+Kp\\\\\\\/K05c7T\\\\\\\/GWClvVjPxnlZXeyxX2m7yiBGHXq\\\\\\\/zreyLJsozQXY26nX030BoLXTo0v8qubN7XRYtZdR1rfNgHapK0hMwatRw5ZL1\\\\\\\/WlNj5hjUPxMfevkze7zjorKFLVFQLxjQ0K0\\\\\\\/lqKMgGh7eLfquJAwEHBeuVDRY1PSumTs0PrnEl545ew9SYx\\\\\\\/jiCRwxvy1FT9\\\\\\\/JsAARdg38E8O16r5bySart9eHpffg6RwljN0hKqYd5hMte+WIDBARLPqdnW\\\\\\\/8Vqp3yXQi9+PLDbx44E6A8qM765Vo+SaFer6eaokghmuL4sSbRFLOBmvxb+uwdXkUZkR7jM5VUeosSCQyn\\\\\\\/GEsjMzn\\\\\\\/yXETu\\\\\\\/4XCjmUhQnFMnGWObqoVph9J7QAqgKSwCF6GRyeZMSMEgVopbxbmweX9GtIHUUFYGs9\\\\\\\/TMqmH7Do51Gr7P0xvEm\\\\\\\/zIUTrEkpi\\\\\\\/79RcVEm7clUPhUGKjHD6AuU1adxMify\\\\\\\/MFeckfaDuVtBYrzCU6JbahGf9\\\\\\\/\\\\\\\/pwZN8SJomEDKDHGjWtKqOSU9BzsHByOVEwCyHSgQoKIrP6AlqsuVqWqGua+2bmJLHL\\\\\\\/DcvsXY7v7hQeaOktSi19+VbRJ1hIqp\\\\\\\/2shWNTH+nZXUbQrL2n4QyfA7Br+37ft0yWgYvhiObk1diZ15pYDMxNgHnSqX6JzzDOyLQIGM9LE3NHofeDvXZc9ShWL12napUCKQGguHGRyw9NtX3xUMoXu1Y9seDXhlTkx7w+\\\\\\\/p017RxqUKpQjnBeT3HCvDjNY1BMmENyQARHU1q7S3tTEtYxsVTaXYiVkpFXVM3NylwTktg4l7pbx0yvKeeq4ZM7HtJzBmXYQR2HyJhYAmQk\\\\\\\/1mWw3XjGRIU64y2CbLFWh1SCWoTvVQOlfA0t+zS\\\\\\\/hYBHPEO9IjjvKfKhZIHyQFU0qxz5hDNo\\\\\\\/4GCN+6t77qmdqMQX9szPyd5Lia+Kj51pxQ8r\\\\\\\/C\\\\\\\/hDTiHA2atNHeZmyg6Lzofzy2jY0wC2mrBDZaaR9QmGbC84VvI\\\\\\\/tYLGKb8Jww3m4qu4nUv6cgDV0AFg2DQUwCFJN5\\\\\\\/cTBUBIwFNrxXQiN89BSrTdXqp\\\\\\\/Egz5n4rGk8Xxy75egV7RbLaer0CZ3All5Kj9pGJOXlvfuKt5+NIItnt3E186t+i\\\\\\\/mGuhNreIhnNqDxgDhwcO4I5tHdYHe31+Sw61WEW7l7JfJnbnIRQLpD\\\\\\\/AnVULqH7wTABeKCCTAU92oQ2tw1DNjcsqf5VS+pS7cxf0a63ZsL6tDp2chDd242hAawRehJrFtptOkCGyPVfYIGzyUoyu+j3l9+u42xw7+90cMmaC8X9DR8IX3m930C2BCaG8knCqFUVuhrq1VWOuqn26HswnoXBWhP+aIwS79a3qgIn6lLAXgButGMDM\\\\\\\/+yGtt046cPZEVdRx856CL+fhjGecvYJzo8KLOP5QOxs1ysURIXa18MfrT9CT\\\\\\\/Ei6VwzWS1rSFE+m6fAT6f7zOFw9xsv53JJQZvfiNzxpk3zGt6QO88mKLIAMb+EAQAiEn+er4gotUlXc+oOgMdJxKLHaPYpCZN76fRqXob\\\\\\\/HrBlR9LQw0cnYrlUt1rw+Rb4DuCQp8N4W0UHo4rZWuCJeDmnDguiIRXA7tAvZcj8d5N8kvPa+EG2yVRvY4AmtD6xJm\\\\\\\/GODJsh86JJIHHqu0eLdtVB\\\\\\\/d0qIPCI9y5cL2J\\\\\\\/bBuGCH0WppbQ7d\\\\\\\/lc0f1Y44UPXjlnOjLuPFWD2SUVdPCspU+60ieU2CTcsJ8Dt+STaeN2zy1DaD\\\\\\\/+N2pgRYjdjkQR+aJ8yPcdCjzPoH+l1czVUfnxj12zO4W1kXT595ixv1vbR1T6q9m4zygV4YzI5Xr3zmcGgK5IUiO+\\\\\\\/fBP+gcfJwPLyWrkah\\\\\\\/ALG3X11z+m1vrjqs8m7rkQRDpn11jx7Z08vAy9LG6oaBHdsP\\\\\\\/7IFkOr57Vv3CY7ndb6Qs1dBq4EXp5yX3EyRbYb9Goa1guF+APwrJlMhdIyGHaOEjyaGOuW+y4hBaxz7DjETeAvBOlWGecNu8KyfLwsua86EHWFgOkq4h+tUmE+cUV4cPlXoumJ2BKCOqVZ3jCtqCqsnmh6dmKKDPYiYGooLFgnr6NebJ\\\\\\\/MX98Z4rMIn1039Lh289+1aLaso0MGPdy4rIvkmHfZsREkfWmmpbEpfWWgbn8oo7gIlRZqvyFfJCwjHxYn3REWVCIiiw\\\\\\\/BcjkylNYbBOtUTm+Wv9Hl2RrThv\\\\\\\/atnzRAVHo\\\\\\\/D+qegSevPcb9eRIzz25DTJOO3xPe8kjVhLB+Y6E3fpD3ea\\\\\\\/Kb4eJwDbR1SObK4GTnR\\\\\\\/iddaX5xwA3S0HWAekUnSgbd8o1JWwo6EewiSyT+GQT0ytJQ+hQhzdFdiEDYJUAinH9qkZhOkAkUD5il8+Vwq1aGGkKgY6PPG6yd\\\\\\\/dRX+zhE3SxFdWajVyyInEfUHwF8qd5OV6z2atv\\\\\\\/Jfd4GCkdUVUXdnlKfj2vihBDcbHddvAa+I26X4heF5llu+jJk\\\\\\\/l9sRmsNDWX0sHNX4gk+YYQXSDq0sn2E1O2WAoWKq0ZeqnAnpHs7KJtnXob2rbRJeXkDwSPNrUNo+9535pVKCR62tVIDxc\\\\\\\/Nj+XKRCR4DsGAUHf4ERDG3FUdAeof8JN3wfr7K98HYbi+rjZNjwW5upKFANmcvj48rD+Fr2yK8kZheE90RJCtuWMRJ9+NKRHXTdUT1kaEwPGlYKwZqWaRepDDlzhxqkd\\\\\\\/+MJXynN4ehxBdMjNmEmAw6TDEp6XQVgAwoWSqxqPUISWOPQtw9stS0eNzTaCVwIVu\\\\\\\/DfZmspRL4UIYf79l0h16zo8OCuLMl1dgI3S+b07pSX\\\\\\\/DSi1DRGM0A6fSuaq3moElhZIbLiXtopHbRBODPfnQq\\\\\\\/A5m1LK2uDoeY8dejJGcZozv1+EAJuDB5Jk0sImeZByTMJalNUwDet8Gl0C72kGVV0\\\\\\\/pLma\\\\\\\/DtFchySjuourQpvjHUWe9C4osOvWNAZyOTK+z7zNWG087i7WHgdcBqfgProY1sxqM83CVLlCTic74Oe5BV78L9uWttwLPfhJvvrwD5KyZpgAAhOAJDiPb1mI1Q352aQBcmh7ktg7Sy8drnePh4ZugBkjlXVw08Ai5ALWPVTiHv8SE\\\\\\\/89h2k5O+qzTtdhPL7iCGg5XuOzoA2vwDCrXS9+vw9LCdzxo0j8vzw5jPmt2\\\\\\\/wWHm4HhDTJhn+DKL5wNYcDGM3yv2wah3dRR7HRcbzCpeRm\\\\\\\/55SypbS+I7XlFOvZDQKii8dQ1o6oqCn92Pri3KAd+1g745tWp9ReakXMcPFOqVkxmvpOsdKdRxi+v3auUQAF9UaliL0NTHTBHw7hd1C9jpVoqCFRBhPti1zS0ETHP+pCnU7EbYpL+qcPDRqhLm7ZIMvDjLZOYEm2pmshgvk2PIbMXXSN2\\\\\\\/CabP9tAb\\\\\\\/xumB3k\\\\\\\/CPf1gjcWtWIprjQ2zRIkq8tAO9g4zlN+XWhvBQFSQxK6vY7kEeNr5zgCiQlGiQr\\\\\\\/BKoL1fRQotfnrO1eoDCYS12EgASbQLEPGi6oUmEfB3T36g47iqJcp\\\\\\\/4xXLqQnBDnBY6JYIdcoAcb4WWbq7TLiQajqw3Za490EeOIQKF6ja4qPHppv4tDTFVJvSrE4Zks6DRJzWDTQAlk7fCzS+22LWeRkzuxMK8iLwLwRyjsq\\\\\\\/GtAyL2PBR2DbsAPtMto6o2oQBPJovbG4ptPRawG9weBGc+xAtZiAxhmiNBjM4dGZBX4cPKsCiUldCdrisFyxzjjZWg1eshDWQvPYpWKEw\\\\\\\/fbIJvMJ+3HfMe4QdieoZgaYHJ4bjEk\\\\\\\/2zIJ6tBFpCYKXbKsUfSbo5DinVIT7YlgjMFgBzU+xVTVWtqDu23S4ovrIVGOPXSOMVJDg510mDDE2S1nDiuJZ0eTjDWsXg8ALKAcBhpGVL\\\\\\\/Pet1eRUeb29EYGKKDC\\\\\\\/KsyE+NCWvQ7J7XjxSYkaVS3RRY4QJ\\\\\\\/sj642OOLrY89vhOw5x2MFjui31mL66FxU0zoxNXHSl+f016nnyfQ8aT+2fcqoOEe6r8LPZqL\\\\\\\/Lp1G8F2d9FZfU3n2gjS3Q35UBBTndcl0HF\\\\\\\/cgnO6\\\\\\\/pkJqBYx7Y9bL4RK\\\\\\\/O5jIgBUFsi5e5rfuE+4ofv1QhtzjKKINBUMzUDHdmypbDJrBqXImq1Jamietgu3vftq+0fIo9PJmA++YjtWhmgEMFjvdedKiv1g4XpjWv+mhZs06m2x0CiYOfm4BvDa\\\\\\\/QeH+auDfhxpepe6zFpNOaSJ\\\\\\\/qfeqkcLlE6MT9T\\\\\\\/+VJmklnJ8MFU1h0AVjcrVrikkq0y4IuEX2B1ET4QICONRZlr\\\\\\\/6QkXT+8ZZ1xm+XHpdpqOegGVZ0pUF2uMzJllp112bRHhjy\\\\\\\/2KKuvtkSIgyeLZwYZMMeMfKoELpl9sDqY1LQZ7m32ktYo+0wsOwBt3VLkVJs7lAMkZ0PzcikcDnWvae5nWEjgZmI7bnBDUf2lJ68IccmKyAg9TotN9PXEhLa+w9\\\\\\\/6S5sszz5L\\\\\\\/qDg9osvtgyLbdmbezzm2Y8EiPh9c688zeG55tDQo4+pTWxX6VHr3MlDWP3JIeg3lmL4L1si624lbTVP9Bket\\\\\\\/s1fvy0JI5OzEZz715J4314YjvDbjX035ZJla\\\\\\\/YNlBJPNDF7uTKkj3BHcFme6mB+MN7+Vmy300QtG7ibl3TPIV6pStDB1YOeo0KPHlB6MPTgpPTJBUw3jH7avo4lRPLU\\\\\\\/9QSOc\\\\\\\/Yg2Ux465rouAx6X8GUrGhejwWs1qsijzg0pcqU1le\\\\\\\/U7wT4Y6havTEHasALoeRydboaun0H3vLSr74n2kyG9RXou4vljVVbTcL\\\\\\\/jXCIMxRZCgihbUARiDGLgbvALjdgsdep6nUt9Sweid+WJhiFhMFd8uMaj5oioKYjdNOsBjoUDVmrWTy+XipyhPzARgEWJAKAYtGVnVlj6P5XvwP1OT2gChoVfTdNTNTA7HtABhsUv1x3aphJnvT+rJDcIS91tvX0hgxoRP\\\\\\\/EHI02eCfOl41DqLir10jzwBso+LMrGpAvdLnWPYUAm35K9kWdXS6\\\\\\\/5XkFbDQK4HrWvG7FArpOiGr+hheh5GMCtnAEKn2\\\\\\\/lXlQzwN0AkgnbY5gAm8qIYOugo7yHXnsnhvyd5GZrw9gAWCm4i6SzbiFzaGmNajR8S08P+FfuEi\\\\\\\/o5TVcezW6TPaQjlwWAWbijKs6Tf\\\\\\\/Wk\\\\\\\/Beu0ySu0kxQFWVJQ1uzSsSfe\\\\\\\/GfX\\\\\\\/UUpiBt1Xwg5eccMbal98rtJMn+bt5SXeZyXzx0YRMf0BCttYoFJGST8jXSWafWBRFi6K0op1qUMmE8pcuvF9rmQSUeFFlkam6iNagSOX5VFgfWZFLIaAV7yH\\\\\\\/\\\\\\\/4qOf3XBscC5KrtWYDXH\\\\\\\/LpRpHt6vrPphMf6g+WIDXDxvW+DdFhIHhxK4nlc7YH0iGDHVqz99Fa4ZNzhM9LySKOqNB0lgvICHzJOg5S7Y3TscnghTz+NpeZqjj63EeC4yy063U\\\\\\\/gn5zrAGrnepehXZOeZZA8XN\\\\\\\/39UKkLb9n4HBpGlTl9K4aURmtHswr1243ulxH\\\\\\\/RH0L7tvRPEBUqRoH4CjB3iuzMdtZmPgciRwU1nyWLW6bS0V3KXjy7ke+0x87bzgruWxshF5cKl+NB2pDE1Ij1J\\\\\\\/9B60ML+Wh1t3C\\\\\\\/C4iLPE+Pzv3P7UosVMKxLEUcbJq0VPMQ3wHrpqetB20wAvdfx7ttCe0GSjDJ8wvB1W8\\\\\\\/VrraUzgITTeFLpLvlz8\\\\\\\/+Dn\\\\\\\/9xwTVixGRZBaWel3Bog1VrI6s99M37cwl4n82rp5oyp3X4iXHWwxaeue8qv1c9Oaif+2z2WgNcenHtJ7OFCiVTII6fDD1+SK5Nnox1nf\\\\\\\/cnvtHk5ug0SLQN4wABb5bzlC9P8eGOTxJotGPXR5lwoucbogR5hqgGWu34ktERbDvj5uF4v4LjHZuunXR8x052XfzML9O+hmIsmQQ984RwI0M0QTsY05e2wL\\\\\\\/kMVTLCjX5p2K8gAa2QeTeSTndVTB2eXL\\\\\\\/u2cycDMxSoLRt4RmjjdX9hA+VQelEXWV4mmxP2WrI\\\\\\\/dp6LDyVJL2bEAbIE8+1oO7DSko4BoCLklre1kD9m0QttvuBp\\\\\\\/rB9UQIbu83t4trMD0CFnHU0Z6QbzlveKucAP3m1f1vtWFPKK9aPuB9VfDC4znxdPZkC93Cr3zyoT21oDGY4aeWHUUOY9QPqcccXSN1FUDK2eVXB451AEV6CEu6KEq4ENbEJIMncXJtjTyi1Tvont81mrQXaxry3gMhZr4T3C0+wQ+Jm9QeWO6CMAGGL6qGE\\\\\\\/F6QzxRVtmUdrq6iXhb00lscKyPuMH8jqWOvAzeCqSQHzux1svI3rPUsk07eoaCeqWqtLw2s55vbpPzzZFze+C2xfJNmlYcAGsNjqP3pONTtNhOXpcqFCp52EV6yGh82XhsgBcoZ345zrUFmFIFZn9basVJTP7QyD\\\\\\\/lPlvOBv2Nt1GpgEgHfjdU2xOFN99IF1Bdm44zBpoAYSrMi\\\\\\\/U2wvG60Htqx\\\\\\\/Dgp\\\\\\\/KcXfyu5IWjSB7DAmx9zchoVp08p\\\\\\\/utoe5W6VFWuivmEQ3DZq4kK6SoFAcIarkenHuxa6VDlEVq9l8WxeCHzVdWojtSgV8KUyRKAseYi4RPYbZrw1DXB2sKSxhC+YN0w==\\\u0022,\\\u0022iv\\\u0022:\\\u0022I576Yj3LYwaJI6Rw8TF9zg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022e448f52121d0a597490f3a93d3ded0bd65c51cb210c9df10cbf5492e1f5ba13f\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022o9XRV4hj+Ka3aXyjthAgvBar3dlI4jw6lmiON5KQ7mNI5hD39I9PNpP3zRXASExCdWkk8qS6S8msOJ2dWaPS3Ki6mCvSX7CmP2+yhYD7tHZE69sQ4Bjfs7yosN7R+j2HL3Q932XnsczMQj2rxT6wm9PXnVmMA6zlE3oP2uJY27zGB7Jv26jcOJN87HRcuGpdV7eoaoaz1s\\\\\\\/\\\\\\\/3+TtIOZEDOfdnoqGtrES2zjlXBlT0fnL0hn+8SQwoNqoS9g7nOT4M63lYCXWV5FmAqZFgwy6I9ANsU+yZ0UwyFZ8tY9yxiMZffOAE0A6\\\\\\\/l2MceVb9o1OLMWggE29\\\\\\\/hFAW+cgAlySmACVijjDxVNkWMcFUtSFIN6\\\\\\\/\\\\\\\/YCH3E6kL\\\\\\\/K1y+q7y6ImQNiRX60DOlha4sUDsQR+5h51EVmhZ\\\\\\\/u1GEqsgIhTEIIyjQZIK4OgreG91ls8UWAqds36wn2apIfhVFGmGusdP87w4MBrlqDC7Wj+wV\\\\\\\/qS0bpUNRq\\\\\\\/UtbcypezyG1x4IpZ9nvfu3JNFCeqGQmoZMV8iqMhYypgbiiA+H56iGigPICPMCSOVuxaJqfW+5yacpiDOlS6jmLAgZGiHnhl75fTTYnJR\\\\\\\/a0XqgKr2W5HX1HeewOTiw7ivD6dTk2yi6np61aMfx\\\\\\\/PGHgl0GbPQS4L0G7xabDuNdHUkdn3ECI7MrUMiZ94m+A0+v+XJ\\\\\\\/MmD+jzr\\\\\\\/FqAZXr7qBCtYUo5gZLgFTfwghzgzcPVs2xmQaYQJVe5QkWhGErkYJqFblp6WkxwghFBu8aJZkn+Zn4GQkM6EU62K2pTFcsYIpzPiJwdFwz3qmnDb4tdfDJW8+0DZY\\\\\\\/8ccnLoikVSI\\\\\\\/A4Ls0hQ4g16zPiwi1gJTOLxB0bJQx4Sh2jQgJLNOm8cIlScQMxXVe4gnJEG33DoYn5kc+dM4yn+s31i+yc8\\\\\\\/XkaM08tK2kayyi3+ix52js\\\\\\\/n3XpcDYNn+PDTiElWN7VmAn0o4SxiMxdSVmh357xWXm1rGGDlqUjdvyyyyVGLe+C5YtSumTFiCALxegIUaWF6zxL3vW8Q5VrYxiruevnj+5VjYOIN0ofsL7iiTuaVTlnzF+MetL+xxiJPtNRHcylvXd+0z0M9n44p3zMP\\\\\\\/5YXBEPbidk4QaOW1ogCQhgySDuIOCwINspC+tlr2gFQRaqMqjv63vU6PALXqclzH923WvWfMktOxqEvd9GGIxOuSIwWVhKHRa47yo9+cM\\\\\\\/AO3CQ7xfvmqmKHqZmeOjiuTMiHmGFUB79+EZ8wxY7KHubtB\\\\\\\/4xJNH1hlOYR3p\\\\\\\/6G6W\\\\\\\/AAC5w5LZKk1M21exaaA8MP07FGunyrpsEfQf0LgWPCRIQ89jpW9PPIQPLF3XIZMjUkOx9kaQYuLz42TCPdjLHOVKEbYJMhr\\\\\\\/Qe9TrrRE0hc5TgJxu\\\\\\\/KS8TxgLxZNw+f0vh78pRjNfJgRO69Tm6fzx2usajTjlduflzQFEYyNcTnSwDISHWWJCRT2uDqN6GJgAclgMY1yJ8BHlqSz16p7V+ohFSrUQQFcwkeIaNC08cK29nm\\\\\\\/DhCmsJFw28OgiRs2lApC+o52m78a2EaBco5nsxB\\\\\\\/eDOh5CTsVNpOZJY5C+E5cu9ANM02qrPBbA0d9SDb95a\\\\\\\/KLM\\\\\\\/qdqZ0a+uf16VY3exO\\\\\\\/D\\\\\\\/NA0q9pdqKBojtutWSx3w\\\\\\\/YsOQ3y4Lfe9bQLtFgznkt\\\\\\\/YNvDLzr7N05jn3yFlrCd61F+06svqgXHOTrxDdlZo2xy2RqeBV+\\\\\\\/LczkD5+p\\\\\\\/HnjxrAqkfo5w8jLxUfohhg63P9loalhP8R8LGmqIco4pBxTXEp06DIpSabbyZtt5YtPAjYDzOM7raoxBmXjZc2oSoNryrQRetiDJpeex2YwhfYaPgk0uwLFWj9XefBsNSmPIUUyIxWXaqxRsYd3TkalsO0RPpZUE5s8hwgV+Q9BehJ3GLC0AmAqbRSGja1RLRMfFuo2phyfK3PiKJbQNiuU67EcBc+RzPJ00\\\\\\\/GGQ51ilAE5Wej5X63N1e5PPvZujc7ZIZ7e+LA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022Zm2K1MKOtfoMstU2vbSmOg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022c10c6cd3a2dd455624233ae2635a134995455a9b000c131599e8286c955d8dc1\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-492618 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/4/1103" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-0-O-QhZp9CgH8j3X3zZaanCV8CyexajFZRRtuEW5odc" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/4/1103" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-7IegvfByKYg2_MuiFp2K9Ly9Y23KYBIwZV9L-dCmv2M" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/4/1085" title="Identification and Functional Testing of Novel Interacting Protein Partners for the Stress Sensors Wsc1p and Mid2p of Saccharomyces cerevisiae" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/4/1117" title="Lost in Translation: On the Problem of Data Coding in Penalized Whole Genome Regression with Interactions" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="492618" id="node492618" data-pisa="ggg;9/4/1103" data-pisa-master="ggg;g3.118.200714" data-apath="/ggg/9/4/1103.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Evolution of the Highly Repetitive PEVK Region of Titin Across Mammals</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-0840-614X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Kathleen Muenzen</span>, <span class="highwire-citation-author" data-delta="1">Jenna Monroy</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-5040-9627" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Findley R. Finseth</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1103-1115;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200714 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node492618" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Kathleen Muenzen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Keck Science Department, Claremont McKenna, Pitzer, and Scripps Colleges, Claremont, CA 91711</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biomedical Informatics and Medical Education, University of Washington, Seattle, WA 98105</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kathleen%2BMuenzen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Muenzen%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKathleen%2BMuenzen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-0840-614X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Kathleen Muenzen</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jenna Monroy </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Keck Science Department, Claremont McKenna, Pitzer, and Scripps Colleges, Claremont, CA 91711</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jenna%2BMonroy%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Monroy%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJenna%2BMonroy%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:ffinseth@kecksci.claremont.edu" class="" data-icon-position="" data-hide-link-title="0">ffinseth@kecksci.claremont.edu</a>
<a href="mailto:jmonroy@kecksci.claremont.edu" class="" data-icon-position="" data-hide-link-title="0">jmonroy@kecksci.claremont.edu</a></span></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Findley R. Finseth </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Keck Science Department, Claremont McKenna, Pitzer, and Scripps Colleges, Claremont, CA 91711</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Findley%2BR.%2BFinseth%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Finseth%20FR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AFindley%2BR.%2BFinseth%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-5040-9627" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Findley R. Finseth</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:ffinseth@kecksci.claremont.edu" class="" data-icon-position="" data-hide-link-title="0">ffinseth@kecksci.claremont.edu</a>
<a href="mailto:jmonroy@kecksci.claremont.edu" class="" data-icon-position="" data-hide-link-title="0">jmonroy@kecksci.claremont.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/9/4/1103" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:492618" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:492618/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/4/1103.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:492618" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:492618/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/4/1103.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:492618" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:492618/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/4/1103.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:492618" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:492618/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/9/4/1103.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:492618" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:492618/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">The protein titin plays a key role in vertebrate muscle where it acts like a giant molecular spring. Despite its importance and conservation over vertebrate evolution, a lack of high quality annotations in non-model species makes comparative evolutionary studies of titin challenging. The PEVK region of titin—named for its high proportion of Pro-Glu-Val-Lys amino acids—is particularly difficult to annotate due to its abundance of alternatively spliced isoforms and short, highly repetitive exons. To understand PEVK evolution across mammals, we developed a bioinformatics tool, PEVK_Finder, to annotate PEVK exons from genomic sequences of titin and applied it to a diverse set of mammals. PEVK_Finder consistently outperforms standard annotation tools across a broad range of conditions and improves annotations of the PEVK region in non-model mammalian species. We find that the PEVK region can be divided into two subregions (PEVK-N, PEVK-C) with distinct patterns of evolutionary constraint and divergence. The bipartite nature of the PEVK region has implications for titin diversification. In the PEVK-N region, certain exons are conserved and may be essential, but natural selection also acts on particular codons. In the PEVK-C, exons are more homogenous and length variation of the PEVK region may provide the raw material for evolutionary adaptation in titin function. The PEVK-C region can be further divided into a highly repetitive region (PEVK-CA) and one that is more variable (PEVK-CB). Taken together, we find that the very complexity that makes titin a challenge for annotation tools may also promote evolutionary adaptation.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/titin" class="hw-term hw-article-keyword hw-article-keyword-titin" rel="nofollow">titin</a></li><li class="kwd"><a href="/keyword/pevk-region" class="hw-term hw-article-keyword hw-article-keyword-pevk-region" rel="nofollow">PEVK region</a></li><li class="kwd"><a href="/keyword/comparative-genomics" class="hw-term hw-article-keyword hw-article-keyword-comparative-genomics" rel="nofollow">comparative genomics</a></li><li class="kwd"><a href="/keyword/gene-prediction" class="hw-term hw-article-keyword hw-article-keyword-gene-prediction" rel="nofollow">gene prediction</a></li><li class="kwd"><a href="/keyword/molecular-evolution" class="hw-term hw-article-keyword hw-article-keyword-molecular-evolution" rel="nofollow">molecular evolution</a></li></ul><p id="p-3">One goal of modern biology is to connect the underlying molecular structure of a protein with physiological function. Studies that compare protein sequences in an evolutionary context can illuminate the regions of proteins that are most essential, under functional constraint, or responding to natural selection (<em>e.g.</em>, <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Perutz 1983</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Hughes and Nei 1989</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Kreitman and Akashi 1995</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Galindo <em>et al.</em> 2003</a>; <a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">Zhao <em>et al.</em> 2009</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Finseth <em>et al.</em> 2015</a>). With the advent of next generation sequencing, the focus of such studies has often moved beyond single gene analyses to comparing entire genomes or transcriptomes (<em>e.g.</em>, <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Finseth <em>et al.</em> 2014</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Karlsson <em>et al.</em> 2014</a>; <a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">Zhang <em>et al.</em> 2014</a>; <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Pervouchine <em>et al.</em> 2015</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Chikina <em>et al.</em> 2016</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Partha <em>et al.</em> 2017</a>). Yet, genes that are large, repetitive, and/or poorly annotated can be left behind by this approach. One such example is titin (TTN), a giant filamentous protein expressed in the muscles of all bilaterian metazoans (<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Steinmetz <em>et al.</em> 2012</a>) that plays a key role in muscle elasticity (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Linke 2018</a>) and is linked to various human muscular diseases (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Savarese <em>et al.</em> 2016</a>).</p><p id="p-4">At nearly 4,000 kD and greater than 1 micrometer in length, titin (also known as connectin) is among the largest proteins found in vertebrates (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Bang <em>et al.</em> 2001</a>). Titin plays a fundamental role in vertebrate striated muscle, where it acts as a giant molecular spring responsible for passive and active muscle elasticity (reviewed in <a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Linke 2018</a>). Titin spans an entire half sarcomere from z-disk to m-line and its I-band region is composed of three domains: the proximal tandem Ig segment; the unique N2A (skeletal muscle), N2B (cardiac muscle), or N2BA (cardiac muscle) sequence; and the PEVK region (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Linke <em>et al.</em> 1996</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Gregorio <em>et al.</em> 1999</a>). Together, these act as serially linked springs (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Labeit and Kolmerer 1995</a>).</p><p id="p-5">The general role titin plays in sarcomere structure is conserved among vertebrates; however, the elastic and physiological properties of vertebrate muscle vary dramatically across species and tissues. Titin, and the I-band in particular, are at least partially responsible for some of these functional differences (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Freiburg <em>et al.</em> 2000</a>, <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Prado <em>et al.</em> 2005</a>, <a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Trombitás <em>et al.</em> 2000</a>) and likely contribute to variation in muscle physiology among tetrapods (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Manteca <em>et al.</em> 2017</a>). For example, cysteine residues in the I-band of titin may be responsible for mechanochemical evolution of vertebrate titin (<a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Manteca <em>et al.</em> 2017</a>). Likewise, variation in the length and structure of alternatively spliced titin isoforms can affect passive and active stiffness of muscles within a single species (<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Freiburg <em>et al.</em> 2000</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Linke <em>et al.</em> 1998</a>, <a id="xref-ref-52-2" class="xref-bibr" href="#ref-52">Prado <em>et al.</em> 2005</a>, <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Powers <em>et al.</em> 2016</a>, <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Monroy <em>et al.</em> 2017</a>). Moreover, mutations to the TTN gene have been associated with various cardiomyopathies and skeletal muscular dystrophies in humans (<a id="xref-ref-57-2" class="xref-bibr" href="#ref-57">Savarese <em>et al.</em> 2016</a>).</p><p id="p-6">Despite its central role in muscle physiology, evolution, and disease, the TTN gene is often poorly or partially annotated in non-model species. One issue is the sheer size of TTN; it is composed of more than 100 kbp with a protein length of &gt; 30,000 amino acids and the gene is more than 60 times the length of the average eukaryotic gene (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Bang <em>et al.</em> 2001</a>, <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Granzier <em>et al.</em> 2007</a>). TTN also contains 364 exons (363 coding exons and a first non-coding exon) with lengths that range from just a few base pairs to greater than 17,000 base pairs, and theoretically can produce more than one million splice variants (<a id="xref-ref-2-3" class="xref-bibr" href="#ref-2">Bang <em>et al.</em> 2001</a>, <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Guo <em>et al.</em> 2010</a>). The abundance of alternatively spliced isoforms means that numerous tissues, samples, and individuals are required for complete cDNA-based or RNA-based annotations.</p><p id="p-7">The PEVK region of TTN, an important determinant of titin and muscle elasticity, also presents a problem for most annotation tools. The PEVK region contains over 100 short, repetitive exons consisting of ∼70% proline (P), glutamate (E), valine (V), and lysine (K) residues. These features mean that PEVK exons are often missed by automated annotation tools and previous studies have relied upon manual annotation of this region (<em>e.g.</em>, <a id="xref-ref-14-3" class="xref-bibr" href="#ref-14">Freiburg <em>et al.</em> 2000</a>; <a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Granzier <em>et al.</em> 2007</a>). Nevertheless, the PEVK region is potentially an important target of selection over evolutionary time; it is evolutionarily labile, varies in length, exon structure, and amino acid content across some vertebrates, and may contribute to evolutionary adaptations in myofibril and whole muscle stiffness (<a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Witt <em>et al.</em> 1998</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Greaser <em>et al.</em> 2002</a>; <a id="xref-ref-18-3" class="xref-bibr" href="#ref-18">Granzier <em>et al.</em> 2007</a>). The PEVK region also appears to have a hierarchical structure, with greater sequence divergence in the N-terminal among vertebrates than the C-terminal (<a id="xref-ref-70-2" class="xref-bibr" href="#ref-70">Witt <em>et al.</em> 1998</a>). With the wealth of genomic data now available, more detailed analyses of the PEVK region across vertebrates are possible and key to understanding its role in muscle evolution.</p><p id="p-8">Here, we characterize the genomic structure of the PEVK region of TTN to gain insight into the structure-function relationships of I-band titin and its evolution across mammals. We first developed a custom tool, PEVK_Finder, to annotate exons within the PEVK region across diverse mammalian species. We then compared exon structure and sequence content both within one individual’s PEVK region and across species. Finally, we performed evolutionary analyses to examine the nature of selection acting on the PEVK region of TTN.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>PEVK_Finder development and optimization</h3><p id="p-9">We developed a tool called PEVK_Finder to annotate the PEVK region of TTN across vertebrates (Table S1). PEVK_Finder annotates exons according to three criteria: 1) minimum exon length (12), 2) minimum PEVK ratio per exon (0.54), and 3) sliding window length (10; determination of optimal parameter values described below). For a given species, PEVK_Finder translates the complete TTN sequence into three forward reading frames and stores the resulting amino acid sequences in memory. PEVK_Finder then utilizes a sliding window approach to identify windows with a PEVK ratio above a minimum threshold. Note that the PEVK_Finder algorithm also incorporates the amino acid alanine (A) in its PEVK ratio calculations, based on previously observed PPAK motifs in titin (<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Greaser <em>et al.</em> 2002</a>). However, all ratios reported in the text and figures of this paper will be referred to simply as <em>PEVK ratio</em>. All windows that meet the PEVK ratio requirement are stored in memory, and overlapping windows are combined into discrete sequences with unique start and end coordinates. To determine exonic boundaries, PEVK_Finder searches for paired donor and acceptor splice sites within each sequence using canonical mammalian nucleotide splicing patterns (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Burset <em>et al.</em> 2000</a>). Although these paired sites are traditionally used to define the 5′ (donor) and 3′ (acceptor) ends of intronic sequences, PEVK_Finder searches for acceptor and donor pairs of adjacent introns. For example, the acceptor site of an intron marks the 5′ end of a likely PEVK exon, while the donor site of the following intron marks the 3′ end of the exon. When no acceptor/donor pairs are found in a given sequence, PEVK_Finder discards the current sequence and moves on to the next likely PEVK exon. When multiple acceptor and/or donor sites are found in a given sequence, the sub-sequence with the minimum distance between any pair of acceptor and donor sites is considered the most likely PEVK exon. We confirmed that each non-overlapping exon was represented by a single reading frame and combined all resultant exons into a single, large exon set. The algorithmic workflow of PEVK_Finder is summarized in Figure S13. The PEVK_Finder program was written in Python using the Biopython bioinformatics tools package (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Cock <em>et al.</em>, 2009</a>).</p><p id="p-10">We optimized PEVK_Finder on the well-annotated human and mouse TTN sequences, before applying the tool to a diverse set of mammalian species (Table S1). To determine an optimal set of baseline parameters for PEVK_Finder, we subjected the untranslated DNA reference sequence of human and mouse TTN to a range of parameter settings in PEVK_Finder and compared those results to cDNA-annotated PEVK regions. The complete human and mouse TTN reference DNA were downloaded from the NCBI RefSeq database (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">O’Leary <em>et al.</em> 2016</a>). The coordinates and DNA sequences for cDNA-annotated human and mouse PEVK exons were gathered from the Ensembl genome browser (<a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">Yates <em>et al.</em> 2016</a>; Table S1). Human TTN exons 112-224 were defined as the human PEVK region, as determined by <a id="xref-ref-18-4" class="xref-bibr" href="#ref-18">Granzier <em>et al.</em> (2007)</a>. Although human TTN exon 225 is also part of the PEVK region, the second ∼half of the exon also encodes the first exon of the distal tandem Ig segment. Exon 224 is the last exclusively-PEVK exon in the PEVK region, and exon 225 was therefore excluded from all further analyses. Note that we refer to human exon numbers according to the NCBI consensus CDS database and provide NCBI exon numbers, location, sequences, and additional numbering schemes in Table S2. Mouse TTN exons 109-207 were defined as the mouse PEVK region, as determined by manual inspection of cDNA-annotated sequences.</p><p id="p-11">All possible combinations of minimum exon length (10-30), minimum PEVK ratio (0.45-0.83), and sliding window length (10-30) were used to generate ∼17,000 PEVK exon sets per species. The parameter ranges used for optimization testing were determined by examining the minimum and mean exon lengths and PEVK ratios of all human and mouse PEVK exons. To determine the lower boundary for each parameter range, a number slightly below the minimum was chosen, and a number higher than the mean was chosen to determine the upper boundary. Each resulting exon set was compared with the corresponding cDNA annotated exon set for each species using the Basic Local Alignment Search Tool (BLAST; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Altschul <em>et al.</em> 1990</a>). Only exons with 100% identity as determined by BLAST were retained for the next steps of optimization. If multiple hits per exon met these criteria, only the hit with the highest bit score was retained.</p><p id="p-12">A match score was calculated to determine how well parameter sets recover the annotated PEVK regions for the ∼17,000 parameter combinations per species. The match score is a weighted score that prioritizes exons that recapitulate annotated exons (“recovered exons”; 70%), rewards identical exons (“perfect exons”; 10%), and minimizes exons identified by PEVK_Finder that are not found in the annotations (“extraneous exons”; 20%). <em>Recovered exons</em> were calculated as the proportion of the cDNA annotated exons identified by PEVK_Finder and included exons that generated both partial and full BLAST hits; <em>perfect exons</em> were defined as the proportion of recovered exons that were 100% identical for the entire length of the annotated exons; <em>extraneous exons</em> were calculated by subtracting the number of PEVK_Finder exons with no matches in the annotated database from 100 and dividing that number by 100. When the number of matchless exons found by PEVK_Finder was &gt;100, the extraneous exons score was 0. The final match score was the sum of the three separate weighted scores and ranged from 0 to 1. The parameter space that encompassed the highest match scores was used to determine an optimal set of parameter ranges that yielded the most accurate sets of PEVK exons for a range of mammalian species. All match score scripts were written in R (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">R Core Team 2016</a>).</p></div><div id="sec-3" class="subsection"><h3>PEVK Finder validation</h3><p id="p-13">To evaluate the utility of PEVK_Finder, we compared the performance of PEVK_Finder on human and mouse TTN with a suite of popular gene annotation tools: Augustus (<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Stanke and Morgenstern 2005</a>), FGENESH (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Salamov and Solovyev 2000</a>), geneid (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Parra, Blanco and Guigó 2000</a>) and GENSCAN (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Burge and Karlin 1997</a>). We used each tool with default parameters to predict the exon-intron coordinates of PEVK exons in both the human and mouse TTN sequences. Exon coordinates generated by each tool were manually compared with cDNA-annotated PEVK exon coordinates for the corresponding species. Exons generated by any of the four tools with a partial match (&gt;= 50%) to a corresponding cDNA exon were considered a match, and a tool exon that spanned two or more cDNA exons was considered a single match. cDNA exons with no overlapping coordinates in the tool exon set were considered missing exons. Tool exons with no overlapping coordinates in the cDNA exon set were considered novel exons.</p></div><div id="sec-4" class="subsection"><h3>Phylogeny construction and PEVK region characterization across mammals</h3><p id="p-14">We downloaded 43 mammalian TTN sequences representing 16 major orders from the NCBI RefSeq database (Table S1). Our taxonomic sampling is a representative subset from <a id="xref-ref-75-2" class="xref-bibr" href="#ref-75">Zhao <em>et al.</em> (2009)</a>, which generated a phylogeny of 59 mammals that evolved to fill diverse niches including subterranean, aquatic and nocturnal niches. The genomic TTN sequences for all 43 mammals in the species list were run through PEVK_Finder and used to create novel PEVK exon sets. Forty-one of these exon sets were used for further analysis. To restrict PEVK exons to the PEVK region, terminal exons were manually determined based on the exon spacing and sequence composition patterns observed in the terminal exons of the cDNA-annotated human and mouse PEVK regions. Specifically, the first PEVK exon (human exon 112) has an amino acid sequence identical or nearly identical to “EIPPVVAPPIPLLLPTPEEKKPPPKRI” and is ∼3,000 or fewer nucleotides before human TTN exon 114, which has an ortholog in all but one species and can thus be easily recognized. The last PEVK exon has an amino acid sequence identical or nearly identical to “AKAPKEEAAKPKGPI” and is generally ∼3,000 or fewer nucleotides after the second to last exon. Exons identified by PEVK_Finder that are located beyond these landmarks may indeed be true PEVK exons, but all exon sets used for further analyses were restricted to this exon range for the sake of remaining consistent with previous studies of the PEVK region.</p><p id="p-15">Using a full time-calibrated phylogeny of extant mammalian species, we created a phylogeny of 39 of the 41 mammalian species in our study to compare PEVK exons across vertebrates and set duck-billed platypus as the outgroup (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Kumar <em>et al.</em> 2017</a>). Two bat species, <em>Myotis lucifugus</em> and <em>Myotis davidii</em>, were not included in the phylogeny because their PEVK regions recapitulated those of other closely related bat species already represented in the figure. Phylogeny importation and editing was performed in R using the Phytools package (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Revell 2012</a>; R Version 3.3.2, <a id="xref-ref-53-2" class="xref-bibr" href="#ref-53">R Core Team 2016</a>). To facilitate visual comparisons of PEVK regions, we generated exon-intron diagrams for each species with a custom R script that plotted the coordinates of PEVK exons in each exon set, color-coded by percent PEVK per exon.</p><p id="p-16">PEVK_Finder exons were compared with cDNA-based annotations or <em>in silico</em> predicted exons for the 41 mammalian exon sets. The NCBI gene prediction program Gnomon (<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Souvorov <em>et al.</em> 2010</a>) predicts the optimal coding sequence using a combination of partial alignments and <em>ab initio</em> modeling. PEVK_Finder annotations were compared with cDNA-based Gnomon predictions when cDNA was available and were compared with <em>in silico</em> Gnomon predictions when no cDNA was available. For simplicity, we refer to both sets of predictions as “Gnomon exons.” We calculated exons identified by PEVK_Finder or Gnomon, exons that were identified by both (“consensus exons”), and the total number of unique exons identified by both tools. Mean differences between Gnomon and PEVK_Finder were determined using a paired <em>t</em>-test, pairing by species.</p></div><div id="sec-5" class="subsection"><h3>Evolutionary analyses</h3><p id="p-17">To facilitate comparisons among PEVK_Finder exons, we attempted to identify “orthologous exons”. We use the term “orthologous” to distinguish exons that descend from a common DNA ancestral sequence from those that are related by duplication (<em>i.e.</em>, paralogous). Orthologous PEVK exons among species were determined using the reciprocal best BLAST method, with the exception that orthologs were determined for each exon rather than the entire TTN gene (<a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Tatusov <em>et al.</em> 1997</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bork and Koonin 1998</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Koonin 2005</a>). The nucleotide sequence of the human PEVK exon set was used as the query and compared with exon sets from the other species. Potential orthologs were called when a given human exon’s top BLASTn species hit (minimum e-value, maximum bit score among hits) returned the original query exon when performed reciprocally. Only orthologs found in the PEVK exon sets of &gt;10 species (including human) were included in further analyses. The remaining sets of orthologous pairs are referred to as “confident orthologs”. Confident orthologs were aligned by codon with the ClustalW (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Goujon <em>et al.</em> 2010</a>) algorithm as implemented in MEGA7 (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Kumar <em>et al.</em> 2016</a>), using all other default parameter settings. Codons missing &gt;15% of data among species were eliminated from the alignment. We estimated average evolutionary divergence over all sequence pairs for each confident ortholog following <a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Tamura <em>et al.</em> (2004)</a> as implemented in MEGA7 (<a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Kumar <em>et al.</em> 2016</a>). Values for each ortholog were calculated in MEGA7 with default parameters and 100 bootstrap replications.</p><p id="p-18">We also evaluated repetition and duplication within the PEVK region by comparing all PEVK exons for a single species with each other. For a given species, all possible exon pairs in the PEVK exon set were aligned, and pairwise nucleotide substitutions per exon were calculated using MEGA Proto and MEGA-CC (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Kumar <em>et al.</em> 2012</a>). The PEVK region was divided into the subregions PEVK-N and PEVK-C, defined in <em>Results: PEVK region hierarchical structure</em>. The pairwise exon alignments were divided into quadrants representing PEVK-C:PEVK-N (Quadrant I), PEVK-C:PEVK-C (Quadrant II), PEVK-N:PEVK-N (Quadrant III), and PEVK-N:PEVK-C (Quadrant IV) comparisons. For each species, we calculated mean substitutions per exon for all quadrants, and results were compared with a one-way ANOVA and a Tukey’s HSD post-hoc test. To evaluate repetition at the nucleotide level, we concatenated exonic sequences from the human PEVK-C region and generated a dot plot in the Dotlet web browser (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Junier and Pagni 2000</a>). We additionally made self-dot plots using the R Dotplot package (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Delaney 2017</a>) for all species using the full PEVK-C genomic sequence for each species, including intronic sequences. Finally, we used a BLAST search to find human LINE-1 elements in other mammals. We used the intronic sequence between human TTN exons 180 and 181 as the query and compared with all 41 mammal TTN sequences.</p><p id="p-19">The PEVK-C region was further divided into two subregions PEVK-CA and PEVK-CB for single-species analyses, also defined in <em>Results: PEVK region hierarchical structure</em>. Pairwise exon alignments were divided into ninths representing PEVK-N:PEVK-CB (Box i), PEVK-CA:PEVK-CB (Box ii), PEVK-CB:PEVK-CB (Box iii), PEVK-N:PEVK-CA (Box iv), PEVK-CA:PEVK-CA (Box v), PEVK-CB:PEVK-CA (Box vi), PEVK-N:PEVK-N (Box vii), PEVK-CA:PEVK-N (Box viii), and PEVK-CB:PEVK-N (Box ix) comparisons. Mean substitutions per exon were calculated for all species and compared using the same statistical measures used for the quadrant comparisons described above.</p><p id="p-20">We tested for positive selection acting on individual codons using site models as implemented in the Phylogenetic Analysis by Maximum Likelihood (PAML) software package (<a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">Yang 2007</a>). For each ortholog alignment, a phylogenetic tree was estimated using the M0 model, then models M0 (one ratio), M1a (neutral), and M2a (positive selection), were run. For each model, ω was set first at 0.5, then at 1 and then lastly at 3. We recorded Lnl values for each model and test statistics for two model comparisons: M0 <em>vs.</em> M1a and M1a <em>vs.</em> M2a. The PAML χ2 calculator was used to determine the p-value of each comparison. All sites under selection, as determined by Bayes Empirical Bayes analysis (<a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">Yang <em>et al.</em> 2005</a>), were recorded for estimates with posterior probabilities &gt; 0.5.</p><p id="p-21">Finally, we tested for charge shifts in codons under positive selection. For each codon, the corresponding amino acid for each species was assigned one of four values based on residue charge: hydrophobic/uncharged, polar, positively charged, or negatively charged. The ancestral charge status of each node in the corresponding mammal tree was estimated using an equal rates (ER) model. Ancestral state reconstruction was performed in R using the Phytools package (<a id="xref-ref-54-2" class="xref-bibr" href="#ref-54">Revell 2012</a>; <a id="xref-ref-53-3" class="xref-bibr" href="#ref-53">R Core Team 2016</a>).</p></div><div id="sec-6" class="subsection"><h3>Data Availability</h3><p id="p-22">The PEVK_Finder source code is available for download at <a href="https://github.com/kmuenzen/pevk_finder_public">https://github.com/kmuenzen/pevk_finder_public</a>. Tables S1 and S2 contain species names, NCBI accession numbers and TTN exon numbers for sequences used in this study. Scripts used in this study have been made available at <a href="https://github.com/kmuenzen/pevk_finder">https://github.com/kmuenzen/pevk_finder</a>. Supplemental materials are available at Figshare: <a href="https://doi.org/10.25387/g3.7544900">https://doi.org/10.25387/g3.7544900</a>.</p></div></div><div class="section results" id="sec-7"><h2 class="">Results</h2><div id="sec-8" class="subsection"><h3>PEVK_Finder parameter optimization</h3><p id="p-23">PEVK_Finder was developed to annotate the PEVK region of TTN across mammals. We determined optimal parameters (exon length, window size, and PEVK ratio) for downstream applications by comparing PEVK_Finder results with the human and mouse annotations and calculating a match score. For each human and mouse model, we tested a total of ∼17,000 parameter combinations. Optimal match scores for human were achieved when minimum exon length was 10-15 nucleotides, PEVK ratio was 0.54-0.55, and window length was 10 nucleotides (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1a</a>). Optimal match scores for mouse were achieved when minimum exon length was 12-14 nucleotides, PEVK ratio was 0.53-0.54, and window length was 10 nucleotides (Figure S1a). Default consensus parameters were therefore set as follows: window length = 10 nucleotides, PEVK ratio = 0.54, and minimum exon length = 12 nucleotides.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/4/1103/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="PEVK Finder tool optimization and evaluation of human TTN. a) Match scores of human PEVK exon sets were generated using different combinations of minimum exon length, PEVK ratio and sliding window length parameter settings. b) PEVK_Finder recovered more PEVK exons than other existing gene prediction tools (GENSCAN, Augustus, FGENESH and geneid). c) PEVK_Finder outperformed GENSCAN at recovering the exon-intron distribution of human TTN PEVK exons identified by cDNA. Vertical lines indicate exon boundaries, and the thickness of the lines is determined by the exon coordinates. Gray circles indicate exons that were missed by either PEVK Finder or GENSCAN, and black asterisks indicate automatically annotated exons that were not annotated by cDNA. In this figure, the first exon indicated by an asterisk is technically outside the pre-defined bounds of the PEVK region. The PEVK ratio scale is given in the figure." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1095756346" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;PEVK Finder tool optimization and evaluation of human TTN. a) Match scores of human PEVK exon sets were generated using different combinations of minimum exon length, PEVK ratio and sliding window length parameter settings. b) PEVK_Finder recovered more PEVK exons than other existing gene prediction tools (GENSCAN, Augustus, FGENESH and geneid). c) PEVK_Finder outperformed GENSCAN at recovering the exon-intron distribution of human TTN PEVK exons identified by cDNA. Vertical lines indicate exon boundaries, and the thickness of the lines is determined by the exon coordinates. Gray circles indicate exons that were missed by either PEVK Finder or GENSCAN, and black asterisks indicate automatically annotated exons that were not annotated by cDNA. In this figure, the first exon indicated by an asterisk is technically outside the pre-defined bounds of the PEVK region. The PEVK ratio scale is given in the figure.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/4/1103/F1.medium.gif" width="410" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/9/4/1103/F1.medium.gif" width="410" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/4/1103/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/4/1103/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492778" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-24" class="first-child">PEVK Finder tool optimization and evaluation of human TTN. a) Match scores of human PEVK exon sets were generated using different combinations of minimum exon length, PEVK ratio and sliding window length parameter settings. b) PEVK_Finder recovered more PEVK exons than other existing gene prediction tools (GENSCAN, Augustus, FGENESH and geneid). c) PEVK_Finder outperformed GENSCAN at recovering the exon-intron distribution of human TTN PEVK exons identified by cDNA. Vertical lines indicate exon boundaries, and the thickness of the lines is determined by the exon coordinates. Gray circles indicate exons that were missed by either PEVK Finder or GENSCAN, and black asterisks indicate automatically annotated exons that were not annotated by cDNA. In this figure, the first exon indicated by an asterisk is technically outside the pre-defined bounds of the PEVK region. The PEVK ratio scale is given in the figure.</p><div class="sb-div caption-clear"></div></div></div><p id="p-25">Using the optimal parameter values, PEVK_Finder identified 109 exons in the PEVK region in human TTN (113 annotated) and 93 PEVK exons in mouse TTN (99 annotated). In humans, 106 PEVK_Finder exons recovered annotated exons (94%) and 70 of these were perfect matches. Of the 36 non-perfect exon matches, 7 were overestimations of exon length and 29 were underestimations. When PEVK_Finder exon length was &gt;50% reduced relative to the annotated length, the annotated exons were often long, with low PEVK ratios. In these cases, PEVK_Finder tended to truncate the exons in regions due to low PEVK ratio (<em>e.g.</em>, human TTN exon 114). When PEVK_Finder overestimated exon length, all PEVK_Finder exons were &lt;100 nucleotides long and length differences were marginal (3-4 amino acids difference). Patterns were similar with the mouse TTN, with the exception that overestimations were more frequent (12 of 37 non-perfect exon matches), possibly due to non-canonical splice sequences (Figure S1c).</p></div><div id="sec-9" class="subsection"><h3>PEVK_Finder validation</h3><p id="p-26">To evaluate the utility of PEVK_Finder, we compared results generated from PEVK_Finder with four different gene annotation tools. Overall, PEVK_Finder outperformed all other tools when identifying PEVK exons in the TTN gene. PEVK_Finder identified 97% of human TTN exons, while no other tool identified more than 65% of human PEVK exons (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1b</a>). PEVK_Finder also missed far fewer exons than GENSCAN in both human and mouse TTN (gray circles <a id="xref-fig-1-3" class="xref-fig" href="#F1">Figures 1c</a>, S1c). PEVK_Finder was also the only tool that identified &gt;80% of both human and mouse exons in the PEVK region (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figures 1b</a>, S1b). FGENESH also identified a high proportion of mouse exons (81%), but was far less effective than PEVK_Finder at annotating human exons (62%). PEVK_Finder performed slightly better on human TTN (94%) than mouse TTN (90%). PEVK_Finder also annotated exons across a range of PEVK ratios and in regions of short, high-density exons (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figures 1c</a>, S1c); these regions were a challenge for GENSCAN. In the few instances when PEVK_Finder did fail to annotate exons, it was usually those exons with low PEVK ratios. GENSCAN also tended to lump distinct exons together, likely due to missed splicing sites. PEVK_Finder, FGENESH, and GENSCAN could be applied to both mouse and human TTN, whereas geneid and Augustus are only suitable for human data. In addition to identifying known exons, PEVK_Finder discovered novel putative PEVK exons in both the human (1; <a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1c</a>) and mouse (2; Figure S1c) TTN sequences. However, the “novel” exon identified in human TTN is technically outside the bounds of the pre-defined PEVK region (exon 112-224).</p><p id="p-27">We extended our evaluation of PEVK_Finder by evaluating its performance on other species with annotated TTN sequences. Of the 41 mammalian species tested, only five have cDNA data for TTN in NCBI (Table S3). Human TTN has the most complete annotation, with consensus cDNA from 11 isoforms. The American Pika (<em>Ochotona princeps</em>) has cDNA data from 5 isoforms, the house mouse (<em>Mus musculus</em>) and the Gairdner’s Shrewmouse (<em>Mus pahari</em>) have cDNA data from 3 isoforms each, and the Chinese Rufous Horseshoe Bat (<em>Rhinolophus sinicus</em>) has cDNA data from 1 short isoform. For the remaining 36 species, we compared PEVK_Finder results with the annotations provided by NCBI generated by the Gnomon program (<a id="xref-ref-61-2" class="xref-bibr" href="#ref-61">Souvorov <em>et al.</em> 2010</a>). Across all species, PEVK_Finder identified significantly more exons (mean: 89.4, SD: 8.33) than the Gnomon-based annotations (mean: 73.9, SD: 10.6; t = 9.37, df = 40, <em>P</em> = 1.23*10<sup>−11</sup>; Table S3). The number of exons identified by PEVK_Finder represents a significantly higher percentage of the total exons (mean: 93.6%, SD: 2.35) than Gnomon-based annotations (mean: 76.7%, SD: 9.29; <em>t</em> = 11.10, <em>P</em> = 8.71*10<sup>−14</sup>; Table S3). PEVK_Finder also identified many more putative novel PEVK exons that were not described previously by either Gnomon or cDNA (mean: 22.7; SD: 10.17).</p></div><div id="sec-10" class="subsection"><h3>PEVK region hierarchical structure</h3><p id="p-28">PEVK_Finder successfully generated exon sets for 41 of the 43 species tested. We were unable to obtain exon sets for two species (<em>Sorex araneus</em> and <em>Leptonychotes weddellii</em>) due to non-canonical splicing sequences that were not compatible with those used by PEVK_Finder to identify exon-intron boundaries. The PEVK region of the remaining 41 species exhibited structural similarities in both exon-intron spacing and PEVK content per exon (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). The total number of exons identified by PEVK_Finder ranged from 81 – 116 exons, 74 – 109 of which were within the bounds of the exons defining the start and end of the PEVK region (Table S3). Interestingly, the species with the smallest number of total PEVK exons are either aquatic diving mammals (<em>Odobenus rosmarus</em> – walrus (81), <em>Tursiops truncatus</em> – dolphin (79), <em>Neomonachus schaunslandi</em> –Hawaiian monk seal (83)) or a burrowing mammal, <em>Condylura cristata</em> (82). However, two of these species also show gaps in their PEVK regions, which may be real or may be due to assembly artifacts (see below).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/4/1103/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Phylogenetic comparison of PEVK exon structure. PEVK_Finder exon-intron plots were overlaid onto a time-calibrated phylogeny for 39 of 41 mammalian species. The dotted line indicates the approximate boundary of the PEVK-N and PEVK-C segments of the PEVK region, based on the human boundary between the two segments. PEVK ratio scale is the same as in Fig. 1. In this figure, Neomonachus schaunslandi and Tupaia chinensis are represented by their alternative names, Monachus schaunslandi and Tupaia belangeri, respectively." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1095756346" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Phylogenetic comparison of PEVK exon structure. PEVK_Finder exon-intron plots were overlaid onto a time-calibrated phylogeny for 39 of 41 mammalian species. The dotted line indicates the approximate boundary of the PEVK-N and PEVK-C segments of the PEVK region, based on the human boundary between the two segments. PEVK ratio scale is the same as in Fig. 1. In this figure, &lt;em&gt;Neomonachus schaunslandi&lt;/em&gt; and &lt;em&gt;Tupaia chinensis&lt;/em&gt; are represented by their alternative names, &lt;em&gt;Monachus schaunslandi&lt;/em&gt; and &lt;em&gt;Tupaia belangeri&lt;/em&gt;, respectively.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/4/1103/F2.medium.gif" width="321" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/9/4/1103/F2.medium.gif" width="321" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/4/1103/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/4/1103/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492699" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-29" class="first-child">Phylogenetic comparison of PEVK exon structure. PEVK_Finder exon-intron plots were overlaid onto a time-calibrated phylogeny for 39 of 41 mammalian species. The dotted line indicates the approximate boundary of the PEVK-N and PEVK-C segments of the PEVK region, based on the human boundary between the two segments. PEVK ratio scale is the same as in <a id="xref-fig-1-7" class="xref-fig" href="#F1">Fig. 1</a>. In this figure, <em>Neomonachus schaunslandi</em> and <em>Tupaia chinensis</em> are represented by their alternative names, <em>Monachus schaunslandi</em> and <em>Tupaia belangeri</em>, respectively.</p><div class="sb-div caption-clear"></div></div></div><p id="p-30">Exon structure of the upstream ∼half of the PEVK region is relatively conserved over mammalian evolution, while the downstream ∼half varies considerably. Based on this observation, we divided the PEVK region into two distinct regions: a structurally conserved region (“PEVK-N”; human exons 112– 165) and a structurally variable region (“PEVK-C”; human exons 166 – 224). Because the end of the PEVK-N region and the beginning of the PEVK-C region varies 1-5 exons between species, a visual landmark was used to manually determine the PEVK-N/C boundary for each species. The manually determined PEVK-N/C boundary is preceded by 3-4 very closely spaced, low PEVK-ratio exons, and is followed by 3-4 more low-PEVK ratio exons and a dense, high PEVK-ratio region (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>). Across mammals, intron/exon boundaries and the number of exons are more similar across the PEVK-N (mean: 50.00, SD: 2.51) <em>vs.</em> the PEVK-C region (mean: 39.46, SD: 8.12, <em>f</em> = 0.10, <em>P</em> &lt; 1.70*10<sup>−11</sup>) (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a>, Table S3). Several species (<em>e.g.</em>, chimpanzee, dolphin, walrus and alpaca) also show large deletions of PEVK exons in the PEVK-C region. The walrus TTN sequence contains a large string of &gt;9,000 ‘N’ bases, indicating that the gap in the walrus PEVK-C region is likely due to low assembly quality in this region. Alpaca TTN also contains several strings of N’s that range from 500-5,000 bases long, which are likely responsible for the large gaps. However, while several small strings of 10-150 ‘N’ bases are found in the PEVK-C region of chimpanzee and dolphin TTN, no large strings of Ns are present. The large deletions in the chimpanzee and dolphin may therefore represent true PEVK exon “deserts”. Alternatively, the gaps could also be an artifact of non-canonical splice sites or small strings of N’s interfering with splice site detection or PEVK ratios. Complete genomic sequences of TTN and/or RNA-based annotations could improve resolution of PEVK exons in these species.</p></div><div id="sec-11" class="subsection"><h3>PEVK region sequence variability within an individual</h3><p id="p-31">Although the PEVK-C region is structurally variable across mammals, its sequence content is more similar within a given individual than the PEVK-N region (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figures 2</a>, <a id="xref-fig-3-1" class="xref-fig" href="#F3">3</a>). Exons in the PEVK-C region tend to be short (mean: 75.76, SE: 0.685) with high PEVK ratios (mean: 0.84, SE: 0.003), whereas those in the PEVK-N region consist of a range of lengths (mean: 93.29, SE: 1.37: <em>t</em> = 11.44, <em>f</em> = 8.68, <em>P</em> &lt; 2.2*10<sup>−16</sup>) and lower PEVK ratios (mean: 0.79, SE: 0.003: <em>t</em> = 12.84, <em>P</em> &lt; 2.2*10<sup>−16</sup>) (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2</a>). Likewise, exon-exon identity comparisons (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3a</a>) and dot plots of nucleotide sequence (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figures 3b</a>, S14–S23) reveal large blocks of highly repetitive sequence in the PEVK-C region. When looking across all species, the mean substitutions per exon varied significantly among the four quadrants (one-way ANOVA: <em>F</em><sub>3,160</sub> = 37.62; <em>P</em> = 2 * 10<sup>−16</sup>; <a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3c</a>). <em>Post hoc</em> comparisons using the Tukey HSD test indicated that the mean number of substitutions per exon for PEVK-C:PEVK-C exon comparisons (mean: 1.01; SD: 0.18; Quadrant II) was significantly lower than comparisons for PEVK-C:PEVK-N exons (mean: 1.27; SD: 0.12; <em>P</em> &lt; 0.0001; Quadrant I), PEVK-N:PEVK-N exons (mean: 1.22; SD: 0.09; <em>P</em> &lt; 0.0001; Quadrant III), and PEVK-N:PEVK-C exons (mean: 1.27; SD: 0.12; <em>P</em> &lt; 0.0001; Quadrant IV). No other quadrant: quadrant comparisons were significantly different (<em>P</em> &gt; 0.05 in all cases). Together, these analyses suggest that exons within the PEVK-C region of an individual have more similar sequences, while exons in the PEVK-N region are more variable in sequence.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/4/1103/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Exon and nucleotide-level comparisons of PEVK-N and PEVK-C regions. a) A heat map of substitutions among PEVK exons within an individual: I) PEVK-N vs. PEVK-C, II) PEVK-C vs. PEVK-C, III) PEVK-N vs. PEVK-N, and IV) PEVK-C vs. PEVK-N. Dark pink indicates exon pairs with few substitutions, whereas light pink and green indicate exons pairs with many substitutions. In general, PEVK-C exons are highly repetitive and homogeneous, while PEVK-N exons are more variable. b) A reciprocal dot plot of the human PEVK-C nucleotide sequence that shows the repetitive nature of the PEVK-C region in humans (Dotlet, https://dotlet.vital-it.ch). c) Mean pairwise substitutions per exon across all 41 species for each quadrant from Figure 3a. Bars represent mean &#xB1; SE d) Mean P,E,V,K amino acids per exon across all 41 mammalian species. There is significantly more glutamate (E), valine (V) and lysine (K) per exon in the PEVK-N region (gray bars) and more proline (P) per exon in the PEVK-C region (black bars) across all 41 species. Bars represent mean &#xB1; SE. Asterisks denote significance at P &lt; 0.05. e) Ratio of glutamate (E) per exon in human TTN PEVK. PEVK_Finder confirms that there is relatively more glutamate (E) per exon in the PEVK-N region compared to the PEVK-C region." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1095756346" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Exon and nucleotide-level comparisons of PEVK-N and PEVK-C regions. a) A heat map of substitutions among PEVK exons within an individual: I) PEVK-N &lt;em&gt;vs.&lt;/em&gt; PEVK-C, II) PEVK-C &lt;em&gt;vs.&lt;/em&gt; PEVK-C, III) PEVK-N &lt;em&gt;vs.&lt;/em&gt; PEVK-N, and IV) PEVK-C &lt;em&gt;vs.&lt;/em&gt; PEVK-N. Dark pink indicates exon pairs with few substitutions, whereas light pink and green indicate exons pairs with many substitutions. In general, PEVK-C exons are highly repetitive and homogeneous, while PEVK-N exons are more variable. b) A reciprocal dot plot of the human PEVK-C nucleotide sequence that shows the repetitive nature of the PEVK-C region in humans (Dotlet, https://dotlet.vital-it.ch). c) Mean pairwise substitutions per exon across all 41 species for each quadrant from Figure 3a. Bars represent mean &#xB1; SE d) Mean P,E,V,K amino acids per exon across all 41 mammalian species. There is significantly more glutamate (E), valine (V) and lysine (K) per exon in the PEVK-N region (gray bars) and more proline (P) per exon in the PEVK-C region (black bars) across all 41 species. Bars represent mean &#xB1; SE. Asterisks denote significance at &lt;em&gt;P&lt;/em&gt; &lt; 0.05. e) Ratio of glutamate (E) per exon in human TTN PEVK. PEVK_Finder confirms that there is relatively more glutamate (E) per exon in the PEVK-N region compared to the PEVK-C region.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/4/1103/F3.medium.gif" width="329" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/9/4/1103/F3.medium.gif" width="329" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/4/1103/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/4/1103/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492648" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-32" class="first-child">Exon and nucleotide-level comparisons of PEVK-N and PEVK-C regions. a) A heat map of substitutions among PEVK exons within an individual: I) PEVK-N <em>vs.</em> PEVK-C, II) PEVK-C <em>vs.</em> PEVK-C, III) PEVK-N <em>vs.</em> PEVK-N, and IV) PEVK-C <em>vs.</em> PEVK-N. Dark pink indicates exon pairs with few substitutions, whereas light pink and green indicate exons pairs with many substitutions. In general, PEVK-C exons are highly repetitive and homogeneous, while PEVK-N exons are more variable. b) A reciprocal dot plot of the human PEVK-C nucleotide sequence that shows the repetitive nature of the PEVK-C region in humans (Dotlet, <a href="https://dotlet.vital-it.ch">https://dotlet.vital-it.ch</a>). c) Mean pairwise substitutions per exon across all 41 species for each quadrant from <a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3a</a>. Bars represent mean ± SE d) Mean P,E,V,K amino acids per exon across all 41 mammalian species. There is significantly more glutamate (E), valine (V) and lysine (K) per exon in the PEVK-N region (gray bars) and more proline (P) per exon in the PEVK-C region (black bars) across all 41 species. Bars represent mean ± SE. Asterisks denote significance at <em>P</em> &lt; 0.05. e) Ratio of glutamate (E) per exon in human TTN PEVK. PEVK_Finder confirms that there is relatively more glutamate (E) per exon in the PEVK-N region compared to the PEVK-C region.</p><div class="sb-div caption-clear"></div></div></div><p id="p-33">The observed sequence variability within an individual’s PEVK-C region (<a id="xref-fig-3-6" class="xref-fig" href="#F3">Figure 3a</a>, Quadrant II; self-dot plots of the PEVK-C region for each species, Figures S14–23) suggested that the PEVK-C could be further subdivided into a highly similar PEVK-CA region (exons 166-215 in human TTN) and a more variable PEVK-CB region (∼last 9 exons of the PEVK-C). The PEVK-CA/PEVK-CB boundary for each species was manually determined using a visual landmark, where the final exon in the densest part of the PEVK-C region marked the last exon of the PEVK-CA region, and the first exon of the PEVK-CB region directly followed the last PEVK-CA exon. The mean substitutions per exon varied significantly among the nine subregion comparisons (one-way ANOVA: <em>F</em><sub>8,360</sub> = 211.6; <em>P</em> = 2 * 10<sup>−16</sup>; Figure S25). <em>Post hoc</em> comparisons indicated that the number of substitutions per exon for PEVK-CA:PEVK-CA comparisons (mean: 0.64; SD: 0.13; Box v) were significantly lower than all other PEVK-CA comparisons (<em>P</em> &lt; 0.0001 in all cases; Box v <em>vs.</em> Boxes i-iv, vi-ix), as well self-comparisons including the PEVK-CB:PEVK-CB (<em>P</em> &lt; 0.0001; Box iii <em>vs.</em> Box ix). Additionally, the number of substitutions per exon for PEVK-CB:PEVK-CB exons were significantly greater than the PEVK:N-PEVK-N self-comparisons (<em>P</em> &lt; 0.0001; Box iii <em>vs.</em> Box vii). PEVK-CB:PEVK-CB exon comparisons also show similar levels of variability as between region comparisons, such as PEVK-CA and PEVK-CB exon comparisons (<em>P</em> = 0.93, Box iii <em>vs.</em> ii; <em>P =</em> 0.9, Box iii <em>vs.</em> vi). These additional analyses suggest that the PEVK-CB region is more variable in sequence than the PEVK-CA region in an individual, and that sequences with high similarity to each other are concentrated in the PEVK-CA region. Likewise, the self-dot plots (Figures S14–S23) show that each species has some highly repetitive regions in the PEVK-CA (appearance of boxes in lower left corner). The distinct patterns of sequence variability with the PEVK-C suggested that the PEVK-CA and PEVK-CB may be evolving differently. Intriguingly, cardiac N2B titin isoforms do not contain exons in the PEVK-CA, and may provide a functional basis for variable selection on the two regions (<a id="xref-ref-19-3" class="xref-bibr" href="#ref-19">Greaser <em>et al.</em> 2002</a>; <a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Guo <em>et al.</em> 2010</a>).</p><p id="p-34">In humans, the movement of long interspersed nuclear elements (LINEs) facilitated the restructuring of the PEVK-C region, resulting in a large triplication event in the PEVK-C (<a id="xref-ref-2-4" class="xref-bibr" href="#ref-2">Bang <em>et al.</em> 2001</a>). We used self-dot plots of the PEVK-C region to examine the incidence of large-scale duplications and triplications (off-axis parallel lines) across each species (Figure S24). We find evidence of large duplications in chimpanzee, orangutan, Norway rat, house mouse and star-nosed mole in the PEVK-CA region. Triplications are only apparent in koala and human (Figures S14b, S16b, S24). In koala, the internal segment of the triplication is flanked by identical sequences, similar to the human triplication event, though it is not known if the sequence is a LINE-1 element as in humans, as LINEs are species-specific. Previous work based on nucleotide divergence placed the insertion of the LINE-1 elements in human PEVK-C after the divergence of humans and other primates (<a id="xref-ref-2-5" class="xref-bibr" href="#ref-2">Bang <em>et al.</em> 2001</a>). However, here we find evidence that both orangutan and chimpanzee also have the LINE-1 element in the PEVK-C, though they only show 1 element each (2 in humans; Figures S16a–c, S24). While in humans, the LINE-1 restructuring of the PEVK-C produces a large triplication event, the region is only duplicated in orangutans and the chimpanzee shows two large repeats with a truncated third repeat (Figures S16a, S16c, S24).</p><p id="p-35">Previous work found that exons in the region corresponding to the PEVK-N were proportionally richer in glutamate (Greaser 2001; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Labeit <em>et al.</em> 2003</a>; Forbes <em>et al.</em> 2005). PEVK-Finder replicates that finding here, suggesting again that PEVK_Finder is able to reproduce previously described PEVK exons in humans (<a id="xref-fig-3-7" class="xref-fig" href="#F3">Figure 3e</a>).</p></div><div id="sec-12" class="subsection"><h3>Evolutionary analysis of the PEVK region across mammals</h3><p id="p-36">We performed a reciprocal best BLAST (<a id="xref-ref-66-2" class="xref-bibr" href="#ref-66">Tatusov <em>et al.</em> 1997</a>; <a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Bork and Koonin 1998</a>; <a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Koonin 2005</a>) to detect orthologs in the PEVK region. Overall, orthology across the PEVK region is low, with only 22 confident orthologs detected in the PEVK-N region (43% of human PEVK-N exons) and 13 in the PEVK-C region (22% of PEVK-C exons) (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figures 4</a>, S2). More orthologs are detected in the PEVK-N region than the PEVK-C region (red/orange, Figure S2), consistent with a more conserved exon structure in the PEVK-N region. Exons in the PEVK-C region display greater sequence divergence across the mammalian tree (mean: 0.11; SE: 0.01) than the PEVK-N region (mean 0.08; SE: 0.01), though this difference is not significant (<em>t</em> = 1.67, <em>P</em> = 0.11) and primarily driven by three exons in the PEVK-C region (Figure S3).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/4/1103/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Human PEVK_Finder exon-intron plot depicting orthologous exons and codons under significant positive selection. Dark gray bars (labeled with their respective exon numbers) indicate orthologous exons, gray circles indicate exons missed by PEVK Finder, and asterisks indicate codons under selection at varying levels of significance. Asterisks indicate posterior probabilities of * &gt; 0.5; ** &gt; 0.75; *** &gt; 0.95. The probability for only the most significant codon for a given exon is noted. Exons 114, 116, 122, 125, 130, 135, 136, 137, 138, 141, 142, 143, 144, 146, 152 153, and 161 are either constitutively expressed or expressed in &gt;95% of TTN transcripts in skeletal muscle according to Savarese et al. (2018). Details about codons under selection are in Figures S5&#x2013;S12. As in figure 1c, the first exon in this figure is technically outside the pre-defined bounds of the human PEVK region." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1095756346" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Human PEVK_Finder exon-intron plot depicting orthologous exons and codons under significant positive selection. Dark gray bars (labeled with their respective exon numbers) indicate orthologous exons, gray circles indicate exons missed by PEVK Finder, and asterisks indicate codons under selection at varying levels of significance. Asterisks indicate posterior probabilities of * &gt; 0.5; ** &gt; 0.75; *** &gt; 0.95. The probability for only the most significant codon for a given exon is noted. Exons 114, 116, 122, 125, 130, 135, 136, 137, 138, 141, 142, 143, 144, 146, 152 153, and 161 are either constitutively expressed or expressed in &gt;95% of TTN transcripts in skeletal muscle according to Savarese &lt;em&gt;et al.&lt;/em&gt; (2018). Details about codons under selection are in Figures S5&#x2013;S12. As in figure 1c, the first exon in this figure is technically outside the pre-defined bounds of the human PEVK region.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/4/1103/F4.medium.gif" width="394" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/9/4/1103/F4.medium.gif" width="394" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/4/1103/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/4/1103/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492713" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-37" class="first-child">Human PEVK_Finder exon-intron plot depicting orthologous exons and codons under significant positive selection. Dark gray bars (labeled with their respective exon numbers) indicate orthologous exons, gray circles indicate exons missed by PEVK Finder, and asterisks indicate codons under selection at varying levels of significance. Asterisks indicate posterior probabilities of * &gt; 0.5; ** &gt; 0.75; *** &gt; 0.95. The probability for only the most significant codon for a given exon is noted. Exons 114, 116, 122, 125, 130, 135, 136, 137, 138, 141, 142, 143, 144, 146, 152 153, and 161 are either constitutively expressed or expressed in &gt;95% of TTN transcripts in skeletal muscle according to <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Savarese <em>et al.</em> (2018)</a>. Details about codons under selection are in Figures S5–S12. As in <a id="xref-fig-1-8" class="xref-fig" href="#F1">figure 1c</a>, the first exon in this figure is technically outside the pre-defined bounds of the human PEVK region.</p><div class="sb-div caption-clear"></div></div></div><p id="p-38">The PEVK-N and PEVK-C regions also encode for significantly different amounts of the core P, E, V, and K amino acids. Across all mammal species, PEVK-N exons contain significantly more glutamate (1.7X; <em>t</em> = 33.51, <em>P</em> &lt; 6.84*10<sup>−31</sup>), lysine (1.2X; <em>t</em> = 15.76, <em>P</em> &lt; 9.32*10<sup>−19</sup>), and valine (1.2X; <em>t</em> = 13.81, <em>P</em> &lt; = 8.30*10<sup>−17</sup>) amino acids than PEVK-C exons (<a id="xref-fig-3-8" class="xref-fig" href="#F3">Figure 3d</a>). PEVK-C exons contain significantly more proline amino acids (1.1X; <em>t</em> = 13.83, <em>P</em> &lt; 7.79*10<sup>−17</sup>). Non-PEVK amino acid residues, such as isoleucine, arginine, threonine, leucine, tyrosine and serine, are also more common in PEVK-N exons (Figure S4).</p><p id="p-39">We also examined the nature of selection acting on codons in the PEVK region. Specifically, we tested for positive selection using random-sites models in PAML, which allowed ω to vary among sites but not lineages. For six sets of orthologous exons (human exons 114, 135, 137, 138, 145 and 199), models that allowed certain codons to evolve under positive selection (ω &gt; 1; M2a) were significantly better than models that restricted codons to be conserved (0 &lt; ω &lt; 1) or neutral (ω = 1; M1a; <a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>; <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). Model M1a <em>vs.</em> M0 was also significant for six exons investigated (results not shown). Five exons with sites under positive selection were in the PEVK-N region, and one was in the PEVK-C region. In total, 20 codons were under positive selection (Figures S5–S10; <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). Results were qualitatively similar for all initial ω values.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/492761/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/492761/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title"><strong>Parameter estimates and likelihood ratio tests for PEVK exons under selection</strong></span><div class="sb-div caption-clear"></div></div></div><p id="p-40">The polarity of amino acids can influence protein-binding interactions and solubility, and therefore may contribute to adaptive evolution of titin. Therefore, we used ancestral state reconstruction to examine charge transitions in TTN exons with confident orthologs. Our analyses show that codons 42 of exon 114 and 6 of exon 135 are under positive selection and have substitutions that change the charge of the codon (Figures S11–S12). Codon 42 of exon 114 features shifts from hydrophobic to polar amino acids from a minimum of 3 to ∼6 times, with shifts occurring in the branches leading to bats, beaver, cetaceans, lemur and Northern treeshrew. This codon also shifts from a hydrophobic to a positively charged amino acid in <em>Mus musculus</em>. Codon 6 of exon 135 reveals four independent transitions from positively charged to polar amino acids across mammals. Intriguingly, in cetaceans, this codon shifts from positively charged to hydrophobic. Thus, in both instances of codons with evidence for charge transitions, cetaceans are one of the few groups to experience changes in polarity and sometimes do so in unique ways (Figure S12). Together, these charge changes and low PEVK exon numbers suggest that titin in cetaceans and other aquatic diving mammals may be under different selective pressures than terrestrial mammals.</p></div></div><div class="section discussion" id="sec-13"><h2 class="">Discussion</h2><div id="sec-14" class="subsection"><h3>PEVK_Finder improves the annotation of the PEVK region across mammals</h3><p id="p-41">The PEVK region of TTN possesses many attributes that confound standard gene prediction tools; it contains numerous short, repetitive exons that vary within and across species in sequence content, exon structure, and length. However, it is this very complexity that makes the PEVK region a promising source of raw material for evolutionary adaptation of muscle (<a id="xref-ref-14-4" class="xref-bibr" href="#ref-14">Freiburg <em>et al.</em> 2000</a>; <a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Tompa 2003</a>; <a id="xref-ref-52-3" class="xref-bibr" href="#ref-52">Prado <em>et al.</em> 2005</a>). Robust annotations of the PEVK region are therefore needed to link molecular evolution of TTN with physiological performance over evolutionary time.</p><p id="p-42">While accurate gene prediction may be achieved with RNA-Seq or similar methods, relying on transcriptomic data for gene prediction is not always feasible due to time and cost constraints. For example, only 5 of the 43 species examined in this study had cDNA data supporting their TTN annotations at the time of the study. The plethora of alternatively spliced isoforms of TTN further complicates RNA-based gene predictions, as data from numerous tissues, developmental stages, and/or individuals are required to generate complete annotations (<em>e.g.</em>, <a id="xref-ref-56-2" class="xref-bibr" href="#ref-56">Savarese <em>et al.</em> 2018</a>). To this end, we developed PEVK_Finder, which combines <em>ab initio</em> and signature-based approaches to gene prediction and targets the PEVK region of TTN (<a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Wang <em>et al.</em> 2004</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Lerat 2010</a>; <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Sleator 2010</a>). Our tool depends solely on nucleotide sequence and searches for exons with motifs and patterns common to well-annotated PEVK regions (<em>i.e.</em>, human, mouse). Similar custom tools have been developed for classes of transposable elements, which are also highly repetitive and often missed by standard annotation tools (<a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Lerat 2010</a>).</p><p id="p-43">Here, we find that PEVK_Finder consistently improves the annotation of the PEVK region across mammals. One major challenge of annotating repetitive regions with short exons is that exons are commonly missed. PEVK_Finder consistently detects significantly more PEVK exons than standard annotation tools, both in model and non-model species (<a id="xref-fig-1-9" class="xref-fig" href="#F1">Figures 1b</a>, <a id="xref-fig-1-10" class="xref-fig" href="#F1">1c</a>; Table S3). Specifically, PEVK_Finder outperforms other tools in regions of short, high-density exons and across broad ranges of PEVK ratios. PEVK_Finder also identifies numerous putative PEVK exons that are not found in cDNA annotations or identified by other bioinformatics tools (<a id="xref-fig-1-11" class="xref-fig" href="#F1">Figures 1c</a>, S1c; Table S3). It is possible that these novel exons are not transcribed, but TTN has many alternatively spliced isoforms that may not be completely characterized by cDNA. For example, manual annotation of genomic TTN sequences has discovered novel TTN exons in the past (<em>e.g.</em>, <a id="xref-ref-14-5" class="xref-bibr" href="#ref-14">Freiburg <em>et al.</em> 2000</a>; <a id="xref-ref-18-5" class="xref-bibr" href="#ref-18">Granzier <em>et al.</em> 2007</a>).</p><p id="p-44">While PEVK_Finder improves available annotations in non-model species, we caution that it does not perfectly replicate the PEVK region in human and mouse TTN. PEVK_Finder occasionally misses, truncates, or joins distinct exons, albeit at a lower frequency than the other tools tested. Exons with low PEVK ratios, internal splice sites and non-canonical splice sites cause most of these errors. Indeed, we were unable to obtain exon sets for two of the 43 species we tested due to the presence of non-canonical splice sites. In the future, incorporation of hidden Markov models into the tool could improve issues with splicing and variable PEVK ratios. Given this, we argue that PEVK_Finder is most useful for determining major trends in PEVK structure and evolution in non-model species, as exemplified here. When complete, perfectly resolved annotations are required in a non-model species, PEVK_Finder should be used in conjunction with RNA-Seq or cDNA sequencing from numerous tissues, individuals, and developmental times. Likewise, pairing PEVK_Finder with another annotation tool, such as Gnomon, could also provide more comprehensive annotations (Table S3).</p></div><div id="sec-15" class="subsection"><h3>The evolution of the PEVK region</h3><p id="p-45">Our results reveal contrasting patterns of constraint and divergence across the PEVK region, suggesting two or three subregions with distinct evolutionary dynamics. The PEVK-N region shows relatively conserved length and exon structure over evolutionary time, but evidence of diversifying selection and more variable amino acid content (<a id="xref-fig-2-6" class="xref-fig" href="#F2">Figures 2</a>, <a id="xref-fig-4-3" class="xref-fig" href="#F4">4</a>). In contrast, the PEVK-C region varies dramatically in length and exon number across mammals and can be further divided into the highly similar PEVK-CA and more variable PEVK-CB (<a id="xref-fig-2-7" class="xref-fig" href="#F2">Figures 2</a>, <a id="xref-fig-3-9" class="xref-fig" href="#F3">3a</a>, <a id="xref-fig-3-10" class="xref-fig" href="#F3">3c</a>, S14–S23). Overall, the documented patterns argue for selection maintaining particular, “essential” PEVK-N and PEVK-CB exons over evolutionary time, with diversifying selection targeting specific codons in the PEVK-N. For example, 17 out of 22 PEVK-N exons with confident orthologs are either constitutively expressed or expressed in &gt;95% of TTN transcripts in skeletal muscle (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4</a>). Likewise, cardiac N2B titin contains exons in the PEVK-N and PEVK-CB, but not the PEVK-CA (<a id="xref-ref-19-4" class="xref-bibr" href="#ref-19">Greaser <em>et al.</em> 2002</a>; <a id="xref-ref-22-3" class="xref-bibr" href="#ref-22">Guo <em>et al.</em> 2010</a>). Conservation over both evolutionary time and across isoforms suggest these exons may play key roles in vertebrate muscle function. Additionally, five PEVK-N exons have codons that experienced adaptive evolution and may be implicated in diversification of muscle function over mammalian evolution. Specifically, codons in exons 114 and 135 experienced major shifts in charge over mammalian evolution and may influence titin-protein binding. Conversely, expansion and contraction of the total length of the PEVK-C region, rather than selection on any particular exon, dominates the evolution of the PEVK-C. Such length variation has the potential to contribute to functional differences, as altering the size of the titin “spring” through alternative splicing of more or fewer PEVK repeats can affect titin’s compliance, though neutral processes may also be relevant (reviewed in <a id="xref-ref-38-3" class="xref-bibr" href="#ref-38">Linke 2018</a>). Future work can focus on disentangling the effects of natural selection acting on specific codons from PEVK length variation, and how both contribute to evolutionary adaptation of TTN.</p><p id="p-46">What explains the discrepancy between the molecular patterns documented in the PEVK-N and PEVK-C regions? Intraspecific sequence comparisons of PEVK exons offer clues. Within a single species’ titin sequence, the sequence content of the PEVK-CA is more repetitive than the PEVK-N or PEVK-CB (<a id="xref-fig-3-11" class="xref-fig" href="#F3">Figures 3</a>, S5, S14-S23, S25). This suggests a mechanistic basis for expansion and contraction of the PEVK-CA over the course of mammalian evolution. Replication slippage and recombinatorial repeat expansions often occur in regions with tandem repeats, and may result in exon duplication and loss (<a id="xref-ref-67-2" class="xref-bibr" href="#ref-67">Tompa 2003</a>). Mobile LINE element insertions have also been implicated in TTN remodeling by causing PEVK exon duplication or differential splicing (<a id="xref-ref-18-6" class="xref-bibr" href="#ref-18">Granzier <em>et al.</em> 2007</a>). For example, we find that retrotransposition of LINE-1 elements facilitated tandem duplications of longer PEVK-CA regions in chimpanzee and orangutan, in addition to the previously reported human duplication (<a id="xref-ref-2-6" class="xref-bibr" href="#ref-2">Bang <em>et al.</em> 2001</a>; Figures S24, S16). Because we find that the PEVK-CA contains more repeats than the PEVK-N and PEVK-CB, we argue that any process generating exon duplication or loss occurs most frequently in the PEVK-CA.</p></div><div id="sec-16" class="subsection"><h3>Implications for titin function</h3><p id="p-47">The length and exon structure of the PEVK region vary remarkably over evolutionary time, with implications for titin functionality. The PEVK region of titin has long been known to contribute to passive stiffness of muscle (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Gautel and Goulding 1996</a>; <a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Linke <em>et al.</em> 1998</a>). Through alternative splicing, titin can be expressed as isoforms with varying lengths of the PEVK domain which correlate with the passive properties of different muscle types (<a id="xref-ref-14-6" class="xref-bibr" href="#ref-14">Freiburg <em>et al.</em> 2000</a>; <a id="xref-ref-52-4" class="xref-bibr" href="#ref-52">Prado <em>et al.</em> 2005</a>). To date, at least 4000 alternative splicing events and twelve distinct isoforms have been identified in human, mouse, and rabbit tissues (<a id="xref-ref-14-7" class="xref-bibr" href="#ref-14">Freiburg <em>et al.</em> 2000</a>; <a id="xref-ref-52-5" class="xref-bibr" href="#ref-52">Prado <em>et al.</em> 2005</a>; <a id="xref-ref-56-3" class="xref-bibr" href="#ref-56">Savarese <em>et al.</em> 2018</a>). Muscles with long PEVK segments have low passive stiffness whereas muscles with shorter segments have higher passive stiffness (<a id="xref-ref-14-8" class="xref-bibr" href="#ref-14">Freiburg <em>et al.</em> 2000</a>; <a id="xref-ref-52-6" class="xref-bibr" href="#ref-52">Prado <em>et al.</em> 2005</a>). The repetitive nature of PEVK-C exons may allow for variability in the length and stiffness of titin isoforms. Repeats within the PEVK region have been postulated to be functionally equivalent to fulfill entropic chain requirements (<a id="xref-ref-67-3" class="xref-bibr" href="#ref-67">Tompa 2003</a>). While this may be the case for repeats within the PEVK-C region, it is less clear for repeats within the PEVK-N region. Alternatively, variation in exon number across the PEVK region may be a product of neutral processes that result in exon skipping and loss, without affecting TTN function.</p><p id="p-48">While a mechanism has not yet been elucidated, several researchers have proposed that during muscle activation, titin stiffness increases in the presence of calcium possibly by titin-actin binding (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Herzog <em>et al.</em> 2012</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Nishikawa <em>et al.</em> 2012</a>; <a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Schappacher-Tilp <em>et al.</em> 2015</a>). Evidence suggests that there is a small direct effect of calcium on PEVK stiffness but that this increase cannot fully account for active muscle stiffness (<a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Tatsumi <em>et al.</em> 2001</a>; <a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Labeit <em>et al.</em> 2003</a>). Within the PEVK region, exons can be composed of strings of negatively charged glutamate residues (E-rich motifs). Results from single molecule experiments have shown that E-rich PEVK segments bind to actin filaments, which produces a viscous load that could possibly resist the sliding of the thin filament along the thick filament during muscle contraction (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Kellermayer and Granzier 1996</a>; <a id="xref-ref-35-3" class="xref-bibr" href="#ref-35">Labeit <em>et al.</em> 2003</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Nagy <em>et al.</em> 2004</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Bianco <em>et al.</em> 2007</a>). If this interaction is necessary for increasing titin stiffness in activated muscle, splicing of exons that encode for amino acids responsible for changing charge or binding affinity to actin could have dramatic effects on titin function. For example, several studies have implicated titin in the enhancement of active muscle force with stretch (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Leonard and Herzog 2010</a>; <a id="xref-ref-51-2" class="xref-bibr" href="#ref-51">Powers <em>et al.</em> 2016</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Hessel <em>et al.</em> 2017</a>). If titin stiffness failed to increase as a result of a change in amino acid charge and/or binding to actin or other proteins, muscles could show decreased force during stretch and alter an organism’s ability to move. Our data show that PEVK-N exons are conserved and confirm previous work suggesting that PEVK-N exons have a higher percentage of E-rich motifs than PEVK-C exons (Greaser 2001; <a id="xref-ref-35-4" class="xref-bibr" href="#ref-35">Labeit <em>et al.</em> 2003</a>; Forbes <em>et al.</em> 2005; <a id="xref-fig-3-12" class="xref-fig" href="#F3">Figure 3e</a>). In addition, PEVK-N exons 114 and 135 exhibit charge shifts that could affect PEVK interactions with other proteins (Figures S11, S12). However, no studies to date have shown calcium dependent PEVK-actin binding (<a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Bianco <em>et al.</em> 2007</a>; <a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Nagy <em>et al.</em> 2004</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Linke <em>et al.</em> 2002</a>). Thus, it is unlikely this interaction underlies the increase in titin stiffness during muscle activation. It remains unclear how E-rich motifs or change in charge may affect titin function. Further studies on the expression of PEVK-N exons could serve to elucidate the role of the PEVK in active and passive muscle.</p><p id="p-49">Several cardiac and skeletal muscle diseases have been associated with mutations in the TTN gene including the PEVK region (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Chauveau <em>et al.</em> 2014</a>; <a id="xref-ref-57-3" class="xref-bibr" href="#ref-57">Savarese <em>et al.</em> 2016</a>; <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Schafer <em>et al.</em> 2017</a>). Understanding how titin has evolved across mammals may provide insight about the effects that TTN mutations have on muscle physiology and disease. Our analyses have identified evolutionarily conserved PEVK exons in the PEVK-N that have previously been shown to be constitutively expressed (<a id="xref-ref-56-4" class="xref-bibr" href="#ref-56">Savarese <em>et al.</em> 2018</a>). These exons may play fundamental roles in titin function and therefore represent sites that could lead to severe deficits in function if modified. Of the 127 TTN sequence variants that have been associated with human muscular disorders, 29 are located in the I-band, 5 of which are found in the PEVK region (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Chauveau <em>et al.</em> 2014</a>). Cardiomyopathies are often associated with truncating variants in the A-band, although a few variants have been identified in the I-band as well (<a id="xref-ref-58-2" class="xref-bibr" href="#ref-58">Schafer <em>et al.</em> 2017</a>). Interestingly, to date, most of the mutations in the PEVK region that have been linked to neuromuscular disease are located in the PEVK-C region (<a id="xref-ref-57-4" class="xref-bibr" href="#ref-57">Savarese <em>et al.</em> 2016</a>). It is possible that there are more disease-causing variants in the PEVK-C compared to the PEVK-N region, or it could be that variants in the PEVK-N region have yet to be identified. It is also possible that the highly repetitive nature of the PEVK-C may facilitate errors during replication that increase the frequency of deleterious mutations. Alternatively, it has been suggested that some I-band variants could be circumvented via differential splicing (<a id="xref-ref-58-3" class="xref-bibr" href="#ref-58">Schafer <em>et al.</em> 2017</a>). Clearly, further studies on the essentiality of particular exons for muscle function and mechanisms of disease are crucial for better understanding TTN-related disorders.</p></div><div id="sec-17" class="subsection"><h3>Conclusions</h3><p id="p-50">In summary, PEVK_Finder provides a useful method for determining major trends in PEVK structure and evolution in non-model species. By characterizing the PEVK region of titin in mammals, we identify two potential pathways through which selection could shape titin function: by changes in the amino acid sequences of specific codons, and by variations in the size of the PEVK region. In the PEVK-N, exons with confident orthologs that are constitutively expressed in isoforms are strong candidates for future work testing the essentiality of particular exons that underlie titin stiffness and muscle function. Together, the construction of custom annotation tools for challenging-to-annotate genes can facilitate novel insights into the diversification and nature of selection acting on important proteins like titin.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-51">We thank Kiisa Nishikawa, Jocelyn Crawford, Silvia Leblanc, Keon Rabbani, Emma Bekele and two anonymous reviewers for their helpful comments on earlier versions of this manuscript. This work was supported by the National Science Foundation (IOS-1731917 awarded to J.A.M.) and start-up funds for F.R.F. and J.A.M.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-52">Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.7544900">https://doi.org/10.25387/g3.7544900</a>.</p></li><li class="fn" id="fn-2"><p id="p-53"><em>Communicating editor: J. Comeron</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2018-09-06"><span class="received-label">Received </span>September 6, 2018.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2019-02-05"><span class="accepted-label">Accepted </span>February 5, 2019.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2019 Muenzen <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1103.1" data-doi="10.1006/jmbi.1990.9999"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Altschul</span>  <span class="cit-name-given-names">S. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gish</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Myers</span>  <span class="cit-name-given-names">E. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lipman</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">Basic local alignment search tool.</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">215</span>: <span class="cit-fpage">403</span>–<span class="cit-lpage">410</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/S0022-2836(05)80360-2</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DAltschul%26rft.auinit1%253DS.%2BF.%26rft.volume%253D215%26rft.issue%253D3%26rft.spage%253D403%26rft.epage%253D410%26rft.atitle%253DBasic%2Blocal%2Balignment%2Bsearch%2Btool.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fjmbi.1990.9999%26rft_id%253Dinfo%253Apmid%252F2231712%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/jmbi.1990.9999&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2231712&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1103.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1990ED16700008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1103.2" data-doi="10.1161/hh2301.100981"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bang</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Centner</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fornoff</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Geach</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gotthardt</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">The complete gene sequence of titin, expression of an unusual ≈700-kDa titin isoform, and its interaction with obscurin identify a novel Z-line to I-band linking system.</span> <abbr class="cit-jnl-abbrev">Circ. Res.</abbr> <span class="cit-vol">89</span>: <span class="cit-fpage">1065</span>–<span class="cit-lpage">1072</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1161/hh2301.100981</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCirc.%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1161%252Fhh2301.100981%26rft_id%253Dinfo%253Apmid%252F11717165%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTA6ImNpcmNyZXNhaGEiO3M6NToicmVzaWQiO3M6MTA6Ijg5LzExLzEwNjUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzQvMTEwMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1103.3" data-doi="10.1529/biophysj.107.106153"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bianco</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nagy</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kengyel</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Szatmári</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mártonfalvi</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Interaction forces between F-actin and titin PEVK domain measured with optical tweezers.</span> <abbr class="cit-jnl-abbrev">Biophys. J.</abbr> <span class="cit-vol">93</span>: <span class="cit-fpage">2102</span>–<span class="cit-lpage">2109</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1529/biophysj.107.106153</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiophys.%2BJ.%26rft_id%253Dinfo%253Adoi%252F10.1529%252Fbiophysj.107.106153%26rft_id%253Dinfo%253Apmid%252F17513381%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1529/biophysj.107.106153&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17513381&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1103.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000249013900026&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1103.4" data-doi="10.1038/ng0498-313"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bork</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koonin</span>  <span class="cit-name-given-names">E. V.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Predicting functions from protein sequences—where are the bottlenecks?</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">313</span>–<span class="cit-lpage">318</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng0498-313</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DBork%26rft.auinit1%253DP.%26rft.volume%253D18%26rft.issue%253D4%26rft.spage%253D313%26rft.epage%253D318%26rft.atitle%253DPredicting%2Bfunctions%2Bfrom%2Bprotein%2Bsequences--where%2Bare%2Bthe%2Bbottlenecks%253F%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng0498-313%26rft_id%253Dinfo%253Apmid%252F9537411%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng0498-313&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9537411&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1103.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000072755500011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1103.5" data-doi="10.1006/jmbi.1997.0951"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Burge</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Karlin</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Prediction of complete gene structures in human genomic DNA.</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">268</span>: <span class="cit-fpage">78</span>–<span class="cit-lpage">94</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1006/jmbi.1997.0951</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DBurge%26rft.auinit1%253DC.%26rft.volume%253D268%26rft.issue%253D1%26rft.spage%253D78%26rft.epage%253D94%26rft.atitle%253DPrediction%2Bof%2Bcomplete%2Bgene%2Bstructures%2Bin%2Bhuman%2Bgenomic%2BDNA.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fjmbi.1997.0951%26rft_id%253Dinfo%253Apmid%252F9149143%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/jmbi.1997.0951&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9149143&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1103.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1997WV87100009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1103.6" data-doi="10.1093/nar/28.21.4364"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Burset</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seledtsov</span>  <span class="cit-name-given-names">I. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Solovyev</span>  <span class="cit-name-given-names">V. V.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Analysis of canonical and non-canonical splice sites in mammalian genomes.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">4364</span>–<span class="cit-lpage">4375</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/28.21.4364</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DBurset%26rft.auinit1%253DM.%26rft.volume%253D28%26rft.issue%253D21%26rft.spage%253D4364%26rft.epage%253D4375%26rft.atitle%253DAnalysis%2Bof%2Bcanonical%2Band%2Bnon-canonical%2Bsplice%2Bsites%2Bin%2Bmammalian%2Bgenomes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F28.21.4364%26rft_id%253Dinfo%253Apmid%252F11058137%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/28.21.4364&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11058137&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1103.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000165263900040&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1103.7" data-doi="10.1002/humu.22611"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chauveau</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rowell</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferreiro</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A Rising Titan, TTN review and mutation update.</span> <abbr class="cit-jnl-abbrev">Hum. Mutat.</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">1046</span>–<span class="cit-lpage">1059</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/humu.22611</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHum.%2BMutat.%26rft.volume%253D35%26rft.spage%253D1046%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fhumu.22611%26rft_id%253Dinfo%253Apmid%252F24980681%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/humu.22611&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24980681&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1103.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1103.8" data-doi="10.1093/molbev/msw112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chikina</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robinson</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clark</span>  <span class="cit-name-given-names">N. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Hundreds of Genes Experienced Convergent Shifts in Selective Pressure in Marine Mammals.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">2182</span>–<span class="cit-lpage">2192</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msw112</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsw112%26rft_id%253Dinfo%253Apmid%252F27329977%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msw112&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27329977&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1103.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1103.9" data-doi="10.1093/bioinformatics/btp163"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cock</span>  <span class="cit-name-given-names">P. J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Antao</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chapman</span>  <span class="cit-name-given-names">B. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cox</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Biopython: freely available Python tools for computational molecular biology and bioinformatics.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1422</span>–<span class="cit-lpage">1423</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btp163</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp163%26rft_id%253Dinfo%253Apmid%252F19304878%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp163&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19304878&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1103.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000266109500013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-web" id="cit-9.4.1103.10"><div class="cit-metadata unstructured">Delaney, N., 2017 Dotplot. GitHub Repository, <a href="https://github.com/evolvedmicrobe/dotplot">https://github.com/evolvedmicrobe/dotplot</a>.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1103.11" data-doi="10.1093/molbev/msu259"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Finseth</span>  <span class="cit-name-given-names">F. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bondra</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harrison</span>  <span class="cit-name-given-names">R. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Selective constraint dominates the evolution of genes expressed in a novel reproductive gland.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">3266</span>–<span class="cit-lpage">3281</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msu259</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsu259%26rft_id%253Dinfo%253Apmid%252F25193339%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msu259&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25193339&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1103.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1103.12" data-doi="10.1093/molbev/msv145"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Finseth</span>  <span class="cit-name-given-names">F. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dong</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Saunders</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fishman</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Duplication and adaptive evolution of a key centromeric protein in mimulus, a genus with female meiotic drive.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">2694</span>–<span class="cit-lpage">2706</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msv145</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsv145%26rft_id%253Dinfo%253Apmid%252F26104011%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msv145&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26104011&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1103.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-9.4.1103.13"><div class="cit-metadata unstructured">Forbes, J. G., A. J. Jin, K. Ma, G. Gutierrez-Cruz, W. L. Tsai, K. Wang, 2005 Titin PEVK segment: charge-driven elasticity of the open and flexible polyampholyte. J Muscle Res. Cell Motil. 26:291–301.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1103.14" data-doi="10.1161/01.RES.86.11.1114"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Freiburg</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trombitas</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hell</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cazorla</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fougerousse</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Series of exon-skipping events in the elastic spring region of titin as the structural basis for myofibrillar elastic diversity.</span> <abbr class="cit-jnl-abbrev">Circ. Res.</abbr> <span class="cit-vol">86</span>: <span class="cit-fpage">1114</span>–<span class="cit-lpage">1121</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1161/01.RES.86.11.1114</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCirculation%2BResearch%26rft.stitle%253DCirc.%2BRes.%26rft.aulast%253DFreiburg%26rft.auinit1%253DA.%26rft.volume%253D86%26rft.issue%253D11%26rft.spage%253D1114%26rft.epage%253D1121%26rft.atitle%253DSeries%2Bof%2BExon-Skipping%2BEvents%2Bin%2Bthe%2BElastic%2BSpring%2BRegion%2Bof%2BTitin%2Bas%2Bthe%2BStructural%2BBasis%2Bfor%2BMyofibrillar%2BElastic%2BDiversity%26rft_id%253Dinfo%253Adoi%252F10.1161%252F01.RES.86.11.1114%26rft_id%253Dinfo%253Apmid%252F10850961%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTA6ImNpcmNyZXNhaGEiO3M6NToicmVzaWQiO3M6MTA6Ijg2LzExLzExMTQiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzQvMTEwMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1103.15" data-doi="10.1073/pnas.0830022100"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Galindo</span>  <span class="cit-name-given-names">B. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vacquier</span>  <span class="cit-name-given-names">V. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Swanson</span>  <span class="cit-name-given-names">W. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Positive selection in the egg receptor for abalone sperm lysin.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">100</span>: <span class="cit-fpage">4639</span>–<span class="cit-lpage">4643</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.0830022100</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0830022100%26rft_id%253Dinfo%253Apmid%252F12676995%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTAwLzgvNDYzOSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzkvNC8xMTAzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1103.16" data-doi="10.1016/0014-5793(96)00338-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gautel</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goulding</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">A molecular map of titin/connectin elasticity reveals two different mechanisms acting in series.</span> <abbr class="cit-jnl-abbrev">FEBS Lett.</abbr> <span class="cit-vol">385</span>: <span class="cit-fpage">11</span>–<span class="cit-lpage">14</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/0014-5793(96)00338-9</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEBS%2BLetters%26rft.stitle%253DFEBS%2BLetters%26rft.aulast%253DGautel%26rft.auinit1%253DM.%26rft.volume%253D385%26rft.issue%253D1-2%26rft.spage%253D11%26rft.epage%253D14%26rft.atitle%253DA%2Bmolecular%2Bmap%2Bof%2Btitin%252Fconnectin%2Belasticity%2Breveals%2Btwo%2Bdifferent%2Bmechanisms%2Bacting%2Bin%2Bseries.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0014-5793%252896%252900338-9%26rft_id%253Dinfo%253Apmid%252F8641453%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0014-5793(96)00338-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8641453&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1103.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1996UJ12500004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1103.17" data-doi="10.1093/nar/gkq313"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goujon</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McWilliam</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valentin</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Squizzato</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">A new bioinformatics analysis tools framework at EMBL-EBI.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">W695</span>–<span class="cit-lpage">W699</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkq313</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkq313%26rft_id%253Dinfo%253Apmid%252F20439314%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkq313&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20439314&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1103.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000284148900113&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1103.18" data-doi="10.1152/ajpregu.00001.2007"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Granzier</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Radke</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Royal</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Irving</span>  <span class="cit-name-given-names">T. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Functional genomics of chicken, mouse, and human titin supports splice diversity as an important mechanism for regulating biomechanics of striated muscle.</span> <abbr class="cit-jnl-abbrev">Am. J. Physiol. Regul. Integr. Comp. Physiol.</abbr> <span class="cit-vol">293</span>: <span class="cit-fpage">R557</span>–<span class="cit-lpage">R567</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1152/ajpregu.00001.2007</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BJ.%2BPhysiol.%2BRegul.%2BIntegr.%2BComp.%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1152%252Fajpregu.00001.2007%26rft_id%253Dinfo%253Apmid%252F17522126%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1152/ajpregu.00001.2007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17522126&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1103.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000248726600001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1103.19" data-doi="10.1023/A:1023410523184"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Greaser</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berri</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Warren</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mozdziak</span>  <span class="cit-name-given-names">P. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Species variations in cDNA sequence and exon splicing patterns in the extensible I-band region of cardiac titin: Relation to passive tension.</span> <abbr class="cit-jnl-abbrev">J. Muscle Res. Cell Motil.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">473</span>–<span class="cit-lpage">482</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1023/A:1023410523184</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmuscle%2Bresearch%2Band%2Bcell%2Bmotility%26rft.stitle%253DJ%2BMuscle%2BRes%2BCell%2BMotil%26rft.aulast%253DGreaser%26rft.auinit1%253DM.%2BL.%26rft.volume%253D23%26rft.issue%253D5-6%26rft.spage%253D473%26rft.epage%253D482%26rft.atitle%253DSpecies%2Bvariations%2Bin%2BcDNA%2Bsequence%2Band%2Bexon%2Bsplicing%2Bpatterns%2Bin%2Bthe%2Bextensible%2BI-band%2Bregion%2Bof%2Bcardiac%2Btitin%253A%2Brelation%2Bto%2Bpassive%2Btension.%26rft_id%253Dinfo%253Adoi%252F10.1023%252FA%253A1023410523184%26rft_id%253Dinfo%253Apmid%252F12785098%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1023/A:1023410523184&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12785098&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1103.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000182302000009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-9.4.1103.20"><div class="cit-metadata unstructured">Greaser, M. L. 2001 Identification of new repeating motifs in titin. Proteins. 43:145–149.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1103.21" data-doi="10.1016/S0955-0674(99)80003-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gregorio</span>  <span class="cit-name-given-names">C. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Granzier</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorimachi</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Labeit</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Muscle assembly: a titanic achievement?</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Cell Biol.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">18</span>–<span class="cit-lpage">25</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/S0955-0674(99)80003-9</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bcell%2Bbiology%26rft.stitle%253DCurr%2BOpin%2BCell%2BBiol%26rft.aulast%253DGregorio%26rft.auinit1%253DC.%2BC.%26rft.volume%253D11%26rft.issue%253D1%26rft.spage%253D18%26rft.epage%253D25%26rft.atitle%253DMuscle%2Bassembly%253A%2Ba%2Btitanic%2Bachievement%253F%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0955-0674%252899%252980003-9%26rft_id%253Dinfo%253Apmid%252F10047523%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0955-0674(99)80003-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10047523&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1103.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000078614000002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1103.22" data-doi="10.1155/2010/180682"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bharmal</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Esbona</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Greaser</span>  <span class="cit-name-given-names">M. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Titin diversity: alternative splicing gone wild.</span> <abbr class="cit-jnl-abbrev">J. Biomed. Biotechnol.</abbr> <span class="cit-vol">2010</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">8</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiomed.%2BBiotechnol.%26rft.volume%253D2010%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1155%252F2010%252F180682%26rft_id%253Dinfo%253Apmid%252F20589071%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1155/2010/180682&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20589071&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1103.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1103.23"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Herzog</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leonard</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Joumaa</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">DuVall</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Panchangam</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The three filament model of skeletal muscle stability and force production.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biomech.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">175</span>–<span class="cit-lpage">191</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BCell.%2BBiomech.%26rft.volume%253D9%26rft.spage%253D175%26rft_id%253Dinfo%253Apmid%252F23285733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=23285733&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1103.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1103.24" data-doi="10.3389/fphys.2017.00070"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hessel</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lindstedt</span>  <span class="cit-name-given-names">S. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nishikawa</span>  <span class="cit-name-given-names">K. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Physiological mechanisms of eccentric contraction and its appications: a role for the gian titin protein.</span> <abbr class="cit-jnl-abbrev">Front. Physiol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">70</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fphys.2017.00070</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BPhysiol.%26rft.volume%253D8%26rft.spage%253D70%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffphys.2017.00070%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fphys.2017.00070&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1103.25" data-doi="10.1073/pnas.86.3.958"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hughes</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nei</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">Nucleotide substitution at major histocompatibility complex class II loci: Evidence for overdominant selection.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">86</span>: <span class="cit-fpage">958</span>–<span class="cit-lpage">962</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.86.3.958</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DHughes%26rft.auinit1%253DA.%2BL.%26rft.volume%253D86%26rft.issue%253D3%26rft.spage%253D958%26rft.epage%253D962%26rft.atitle%253DNucleotide%2Bsubstitution%2Bat%2Bmajor%2Bhistocompatibility%2Bcomplex%2Bclass%2BII%2Bloci%253A%2Bevidence%2Bfor%2Boverdominant%2Bselection.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.86.3.958%26rft_id%253Dinfo%253Apmid%252F2492668%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo4OiI4Ni8zLzk1OCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzkvNC8xMTAzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1103.26" data-doi="10.1093/bioinformatics/16.2.178"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Junier</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pagni</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Dotlet: diagonal plots in a web browser.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">178</span>–<span class="cit-lpage">179</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/16.2.178</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DJunier%26rft.auinit1%253DT.%26rft.volume%253D16%26rft.issue%253D2%26rft.spage%253D178%26rft.epage%253D179%26rft.atitle%253DDotlet%253A%2Bdiagonal%2Bplots%2Bin%2Ba%2BWeb%2Bbrowser%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252F16.2.178%26rft_id%253Dinfo%253Apmid%252F10842741%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/16.2.178&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10842741&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1103.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000087033600016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1103.27" data-doi="10.1038/nrg3734"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Karlsson</span>  <span class="cit-name-given-names">E. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kwiatkowski</span>  <span class="cit-name-given-names">D. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sabeti</span>  <span class="cit-name-given-names">P. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Natural selection and infectious disease in human populations.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">379</span>–<span class="cit-lpage">393</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg3734</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BGenet.%26rft.volume%253D15%26rft.spage%253D379%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3734%26rft_id%253Dinfo%253Apmid%252F24776769%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3734&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24776769&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1103.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1103.28" data-doi="10.1016/0014-5793(96)00055-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kellermayer</span>  <span class="cit-name-given-names">M. S. Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Granzier</span>  <span class="cit-name-given-names">H. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Calcium-dependent inhibition of in vitro thin-filament motility by native titin.</span> <abbr class="cit-jnl-abbrev">FEBS Lett.</abbr> <span class="cit-vol">380</span>: <span class="cit-fpage">281</span>–<span class="cit-lpage">286</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/0014-5793(96)00055-5</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEBS%2BLetters%26rft.stitle%253DFEBS%2BLetters%26rft.aulast%253DKellermayer%26rft.auinit1%253DM.%2BS.%26rft.volume%253D380%26rft.issue%253D3%26rft.spage%253D281%26rft.epage%253D286%26rft.atitle%253DCalcium-dependent%2Binhibition%2Bof%2Bin%2Bvitro%2Bthin-filament%2Bmotility%2Bby%2Bnative%2Btitin.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0014-5793%252896%252900055-5%26rft_id%253Dinfo%253Apmid%252F8601441%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0014-5793(96)00055-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8601441&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1103.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1996TW66500016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1103.29" data-doi="10.1146/annurev.genet.39.073003.114725"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Koonin</span>  <span class="cit-name-given-names">E. V.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Orthologs, Paralogs, and Evolutionary Genomics.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Genet.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">309</span>–<span class="cit-lpage">338</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1146/annurev.genet.39.073003.114725</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bgenetics%26rft.stitle%253DAnnu%2BRev%2BGenet%26rft.volume%253D39%26rft.spage%253D309%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.genet.39.073003.114725%26rft_id%253Dinfo%253Apmid%252F16285863%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.genet.39.073003.114725&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16285863&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1103.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000234685200015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1103.30" data-doi="10.1146/annurev.es.26.110195.002155"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kreitman</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Akashi</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Molecular Evidence for Natural Selection.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Ecol. Syst.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">403</span>–<span class="cit-lpage">422</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1146/annurev.es.26.110195.002155</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2BReview%2Bof%2BEcology%2Band%2BSystematics%26rft.stitle%253DAnnu.%2BRev.%2BEcol.%2BSyst.%26rft.aulast%253DAkashi%26rft.auinit1%253DH.%26rft.volume%253D26%26rft.issue%253D1%26rft.spage%253D403%26rft.epage%253D422%26rft.atitle%253DMolecular%2BEvidence%2Bfor%2BNatural%2BSelection%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.es.26.110195.002155%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.es.26.110195.002155&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1995TH33000019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1103.31" data-doi="10.1093/molbev/msr202"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Filipski</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Battistuzzi</span>  <span class="cit-name-given-names">F. U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kosakovsky Pond</span>  <span class="cit-name-given-names">S. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tamura</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Statistics and truth in phylogenomics.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">457</span>–<span class="cit-lpage">472</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msr202</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsr202%26rft_id%253Dinfo%253Apmid%252F21873298%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msr202&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21873298&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1103.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000299129000004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1103.32" data-doi="10.1093/molbev/msw054"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stecher</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tamura</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">MEGA7: Molecular Evolutionary Genetics Analysis Version 7.0 for Bigger Datasets.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">1870</span>–<span class="cit-lpage">1874</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msw054</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsw054%26rft_id%253Dinfo%253Apmid%252F27004904%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msw054&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27004904&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1103.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1103.33" data-doi="10.1093/molbev/msx116"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stecher</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hedges</span>  <span class="cit-name-given-names">S. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">TimeTree: A Resource for Timelines, Timetrees, and Divergence Times.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">1812</span>–<span class="cit-lpage">1819</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msx116</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsx116%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msx116&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1103.34" data-doi="10.1126/science.270.5234.293"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Labeit</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kolmerer</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Titins: giant proteins in charge of muscle ultrastructure and elasticity.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">270</span>: <span class="cit-fpage">293</span>–<span class="cit-lpage">296</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.270.5234.293</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DLabeit%26rft.auinit1%253DS.%26rft.volume%253D270%26rft.issue%253D5234%26rft.spage%253D293%26rft.epage%253D296%26rft.atitle%253DTitins%253A%2BGiant%2BProteins%2Bin%2BCharge%2Bof%2BMuscle%2BUltrastructure%2Band%2BElasticity%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.270.5234.293%26rft_id%253Dinfo%253Apmid%252F7569978%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIyNzAvNTIzNC8yOTMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzQvMTEwMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1103.35" data-doi="10.1073/pnas.2235652100"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Labeit</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Watanabe</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Witt</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fujita</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Calcium-dependent molecular spring elements in the giant protein titin.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">100</span>: <span class="cit-fpage">13716</span>–<span class="cit-lpage">13721</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.2235652100</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.2235652100%26rft_id%253Dinfo%253Apmid%252F14593205%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTAwLzIzLzEzNzE2IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS80LzExMDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1103.36" data-doi="10.1152/ajpcell.00049.2010"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Leonard</span>  <span class="cit-name-given-names">T. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Herzog</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Regulation of muscle force in the absenve of actin-myosin-based cross-bridge interaction.</span> <abbr class="cit-jnl-abbrev">Am. J. Physiol. Cell Physiol.</abbr> <span class="cit-vol">299</span>: <span class="cit-fpage">C14</span>–<span class="cit-lpage">C20</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1152/ajpcell.00049.2010</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BJ.%2BPhysiol.%2BCell%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1152%252Fajpcell.00049.2010%26rft_id%253Dinfo%253Apmid%252F20357181%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1152/ajpcell.00049.2010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20357181&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1103.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000278887800004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1103.37" data-doi="10.1038/hdy.2009.165"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lerat</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Identifying repeats and transposable elements in sequenced genomes : how to find your way through the dense forest of programs.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">104</span>: <span class="cit-fpage">520</span>–<span class="cit-lpage">533</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/hdy.2009.165</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.stitle%253DHeredity%26rft.aulast%253DLerat%26rft.auinit1%253DE.%26rft.volume%253D104%26rft.issue%253D6%26rft.spage%253D520%26rft.epage%253D533%26rft.atitle%253DIdentifying%2Brepeats%2Band%2Btransposable%2Belements%2Bin%2Bsequenced%2Bgenomes%253A%2Bhow%2Bto%2Bfind%2Byour%2Bway%2Bthrough%2Bthe%2Bdense%2Bforest%2Bof%2Bprograms.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2009.165%26rft_id%253Dinfo%253Apmid%252F19935826%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2009.165&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19935826&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1103.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000277828500002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1103.38" data-doi="10.1146/annurev-physiol-021317-121234"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Linke</span>  <span class="cit-name-given-names">W. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Titin Gene and Protein Functions in Passive and Active Muscle.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Physiol.</abbr> <span class="cit-vol">80</span>: <span class="cit-fpage">389</span>–<span class="cit-lpage">411</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1146/annurev-physiol-021317-121234</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnu.%2BRev.%2BPhysiol.%26rft.volume%253D80%26rft.spage%253D389%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-physiol-021317-121234%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev-physiol-021317-121234&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1103.39" data-doi="10.1073/pnas.95.14.8052"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Linke</span>  <span class="cit-name-given-names">W. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ivemeyer</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mundel</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stockmeier</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kolmerer</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Nature of PEVK-titin elasticity in skeletal muscle.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">95</span>: <span class="cit-fpage">8052</span>–<span class="cit-lpage">8057</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.95.14.8052</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DLinke%26rft.auinit1%253DW.%2BA.%26rft.volume%253D95%26rft.issue%253D14%26rft.spage%253D8052%26rft.epage%253D8057%26rft.atitle%253DNature%2Bof%2BPEVK-titin%2Belasticity%2Bin%2Bskeletal%2Bmuscle%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.95.14.8052%26rft_id%253Dinfo%253Apmid%252F9653138%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiOTUvMTQvODA1MiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzkvNC8xMTAzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1103.40" data-doi="10.1006/jmbi.1996.0441"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Linke</span>  <span class="cit-name-given-names">W. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ivemeyer</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Olivieri</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kolmerer</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rüegg</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Towards a molecular understanding of the elasticity of titin.</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">261</span>: <span class="cit-fpage">62</span>–<span class="cit-lpage">71</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1006/jmbi.1996.0441</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DLinke%26rft.auinit1%253DW.%2BA.%26rft.volume%253D261%26rft.issue%253D1%26rft.spage%253D62%26rft.epage%253D71%26rft.atitle%253DTowards%2Ba%2Bmolecular%2Bunderstanding%2Bof%2Bthe%2Belasticity%2Bof%2Btitin.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fjmbi.1996.0441%26rft_id%253Dinfo%253Apmid%252F8760502%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/jmbi.1996.0441&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8760502&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1103.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1996VB42800005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1103.41" data-doi="10.1006/jsbi.2002.4468"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Linke</span>  <span class="cit-name-given-names">W. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kulke</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fujita-Becker</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neagoe</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">PEVK domain of titin: An entropic spring with actin-binding properties.</span> <abbr class="cit-jnl-abbrev">J. Struct. Biol.</abbr> <span class="cit-vol">137</span>: <span class="cit-fpage">194</span>–<span class="cit-lpage">205</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1006/jsbi.2002.4468</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bstructural%2Bbiology%26rft.stitle%253DJ%2BStruct%2BBiol%26rft.aulast%253DLinke%26rft.auinit1%253DW.%2BA.%26rft.volume%253D137%26rft.issue%253D1-2%26rft.spage%253D194%26rft.epage%253D205%26rft.atitle%253DPEVK%2Bdomain%2Bof%2Btitin%253A%2Ban%2Bentropic%2Bspring%2Bwith%2Bactin-binding%2Bproperties.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fjsbi.2002.4468%26rft_id%253Dinfo%253Apmid%252F12064946%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/jsbi.2002.4468&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12064946&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1103.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000176453500021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1103.42" data-doi="10.1038/nsmb.3426"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Manteca</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schönfelder</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alonso-Caballero</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fertin</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barruetabeña</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Mechanochemical evolution of the giant muscle protein titin as inferred from resurrected proteins.</span> <abbr class="cit-jnl-abbrev">Nat. Struct. Mol. Biol.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">652</span>–<span class="cit-lpage">657</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nsmb.3426</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BStruct.%2BMol.%2BBiol.%26rft.volume%253D24%26rft.spage%253D652%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnsmb.3426%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nsmb.3426&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1103.43" data-doi="10.1242/jeb.139717"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Monroy</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Powers</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pace</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Uyeno</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nishikawa</span>  <span class="cit-name-given-names">K. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Effects of activation on the elastic properties of intact soleus muscles with a deletion in titin.</span> <abbr class="cit-jnl-abbrev">J. Exp. Biol.</abbr> <span class="cit-vol">220</span>: <span class="cit-fpage">828</span>–<span class="cit-lpage">836</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BExp.%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1242%252Fjeb.139717%26rft_id%253Dinfo%253Apmid%252F27994045%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiamV4YmlvIjtzOjU6InJlc2lkIjtzOjk6IjIyMC81LzgyOCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzkvNC8xMTAzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1103.44" data-doi="10.1242/jcs.01501"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nagy</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cacciafesta</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grama</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kengyel</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Málnási-Csizmadia</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Differential actin binding along the PEVK domain of skeletal muscle titin.</span> <abbr class="cit-jnl-abbrev">J. Cell Sci.</abbr> <span class="cit-vol">117</span>: <span class="cit-fpage">5781</span>–<span class="cit-lpage">5789</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1242/jcs.01501</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BCell%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.1242%252Fjcs.01501%26rft_id%253Dinfo%253Apmid%252F15507486%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NToiam9jZXMiO3M6NToicmVzaWQiO3M6MTE6IjExNy8yNC81NzgxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS80LzExMDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1103.45" data-doi="10.1098/rspb.2011.1304"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nishikawa</span>  <span class="cit-name-given-names">K. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Monroy</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Uyeno</span>  <span class="cit-name-given-names">T. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yeo</span>  <span class="cit-name-given-names">S. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pai</span>  <span class="cit-name-given-names">D. K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Is titin a “winding filament”? A new twist on muscle contraction.</span> <abbr class="cit-jnl-abbrev">Proc. Biol. Sci.</abbr> <span class="cit-vol">279</span>: <span class="cit-fpage">981</span>–<span class="cit-lpage">990</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1098/rspb.2011.1304</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BBiol.%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frspb.2011.1304%26rft_id%253Dinfo%253Apmid%252F21900329%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1098/rspb.2011.1304&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21900329&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1103.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1103.46" data-doi="10.1093/nar/gkv1189"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">O’Leary</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">M. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brister</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ciufo</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haddad</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Reference sequence (RefSeq) database at NCBI: current status, taxonomic expansion, and functional annotation.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">D733</span>–<span class="cit-lpage">D745</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkv1189</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkv1189%26rft_id%253Dinfo%253Apmid%252F26553804%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkv1189&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26553804&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1103.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1103.47" data-doi="10.1101/gr.10.4.511"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Parra</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blanco</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guigó</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">GeneID in Drosophila.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">511</span>–<span class="cit-lpage">515</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.10.4.511</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DParra%26rft.auinit1%253DG.%26rft.volume%253D10%26rft.issue%253D4%26rft.spage%253D511%26rft.epage%253D515%26rft.atitle%253DGeneID%2Bin%2BDrosophila%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.10.4.511%26rft_id%253Dinfo%253Apmid%252F10779490%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjEwLzQvNTExIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS80LzExMDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1103.48" data-doi="10.7554/eLife.25884"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Partha</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chauhan</span>  <span class="cit-name-given-names">B. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferreira</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robinson</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lathrop</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Subterranean mammals show convergent regression in ocular genes and enhancers, along with adaptation to tunneling.</span> <abbr class="cit-jnl-abbrev">eLife</abbr> <span class="cit-vol">6</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.7554/eLife.25884</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DeLife%26rft.volume%253D6%26rft.spage%253De25884%26rft_id%253Dinfo%253Adoi%252F10.7554%252FeLife.25884%26rft_id%253Dinfo%253Apmid%252F29035697%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.7554/eLife.25884&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=29035697&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1103.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1103.49"><div class="cit-metadata unstructured">Perutz, M. F., 1983 Species adaptation in a protein molecule. Mol. Biol. Evol. 1: 1–28.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1103.50" data-doi="10.1038/ncomms6903"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pervouchine</span>  <span class="cit-name-given-names">D. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Djebali</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Breschi</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barja</span>  <span class="cit-name-given-names">P. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Enhanced transcriptome maps from multiple mouse tissues reveal evolutionary constraint in gene expression.</span> <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">5903</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ncomms6903</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun.%26rft.volume%253D6%26rft.spage%253D5903%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms6903%26rft_id%253Dinfo%253Apmid%252F25582907%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms6903&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25582907&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1103.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1103.51" data-doi="10.1242/jeb.132027"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Powers</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nishikawa</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Joumaa</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Herzog</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Decreased force enhancement in skeletal muscle sarcomeres with a deletion in titin.</span> <abbr class="cit-jnl-abbrev">J. Exp. Biol.</abbr> <span class="cit-vol">219</span>: <span class="cit-fpage">1311</span>–<span class="cit-lpage">1316</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1242/jeb.132027</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BExp.%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1242%252Fjeb.132027%26rft_id%253Dinfo%253Apmid%252F26944495%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiamV4YmlvIjtzOjU6InJlc2lkIjtzOjEwOiIyMTkvOS8xMzExIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS80LzExMDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1103.52" data-doi="10.1085/jgp.200509364"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Prado</span>  <span class="cit-name-given-names">L. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Makarenko</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Andresen</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krüger</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Opitz</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Isoform Diversity of Giant Proteins in Relation to Passive and Active Contractile Properties of Rabbit Skeletal Muscles.</span> <abbr class="cit-jnl-abbrev">J. Gen. Physiol.</abbr> <span class="cit-vol">126</span>: <span class="cit-fpage">461</span>–<span class="cit-lpage">480</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1085/jgp.200509364</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BGen.%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1085%252Fjgp.200509364%26rft_id%253Dinfo%253Apmid%252F16230467%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamdwIjtzOjU6InJlc2lkIjtzOjk6IjEyNi81LzQ2MSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzkvNC8xMTAzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-web" id="cit-9.4.1103.53"><div class="cit-metadata unstructured">R Core Team, (2016). R: A language and environment for statistical computing. R Foundation for Statistical Computing, Vienna Austria, URL <a href="https://www.R-project.org/">https://www.R-project.org/</a></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1103.54"><div class="cit-metadata unstructured">Revell, L., 2012 phytools: An R package for phylogenetic comparative biology (and other things). Methods Ecol. Evol. 3: 217–223.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1103.55" data-doi="10.1101/gr.10.4.516"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Salamov</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Solovyev</span>  <span class="cit-name-given-names">V. V.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Ab initio gene finding in Drosophila genomic DNA.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">516</span>–<span class="cit-lpage">522</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.10.4.516</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DSalamov%26rft.auinit1%253DA.%2BA.%26rft.volume%253D10%26rft.issue%253D4%26rft.spage%253D516%26rft.epage%253D522%26rft.atitle%253DAb%2Binitio%2BGene%2BFinding%2Bin%2BDrosophila%2BGenomic%2BDNA%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.10.4.516%26rft_id%253Dinfo%253Apmid%252F10779491%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjEwLzQvNTE2IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS80LzExMDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1103.56" data-doi="10.1186/s13395-018-0156-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Savarese</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jonson</span>  <span class="cit-name-given-names">P. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huovinen</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paulin</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Auvinen</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">The complexity of titin splicing pattern in human adult skeletal muscles.</span> <abbr class="cit-jnl-abbrev">Skelet. Muscle</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">11</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s13395-018-0156-z</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSkelet.%2BMuscle%26rft.volume%253D8%26rft.spage%253D11%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13395-018-0156-z%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13395-018-0156-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1103.57"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Savarese</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sarparanta</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vihola</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Udd</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hackman</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> Increasing role of titin mutations in neuromuscular disorders. Journal of Neuoromuscular diseases 3: 293–208.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3233/JND-160158</span></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1103.58" data-doi="10.1038/ng.3719"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schafer</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Marvao</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Adami</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fiedler</span>  <span class="cit-name-given-names">L. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ng</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Titin-truncating variants affect heart function in disease cohorts and the general population.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">46</span>–<span class="cit-lpage">53</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.3719</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D49%26rft.spage%253D46%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.3719%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.3719&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1103.59" data-doi="10.1371/journal.pone.0117634"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schappacher-Tilp</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leonard</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Desch</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Herzog</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">A novel three-filament model of force generation in eccentric contraction of skeletal muscles.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">e0117634</span> (erratum: PLoS One 10: e0141188).<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0117634</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D10%26rft.spage%253De0117634%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0117634%26rft_id%253Dinfo%253Apmid%252F25816319%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0117634&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25816319&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1103.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1103.60" data-doi="10.1016/j.gene.2010.04.008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sleator</span>  <span class="cit-name-given-names">R. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">An overview of the current status of eukaryote gene prediction strategies.</span> <abbr class="cit-jnl-abbrev">Gene</abbr> <span class="cit-vol">461</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">4</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.gene.2010.04.008</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DGene%26rft.aulast%253DSleator%26rft.auinit1%253DR.%2BD.%26rft.volume%253D461%26rft.issue%253D1-2%26rft.spage%253D1%26rft.epage%253D4%26rft.atitle%253DAn%2Boverview%2Bof%2Bthe%2Bcurrent%2Bstatus%2Bof%2Beukaryote%2Bgene%2Bprediction%2Bstrategies.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.gene.2010.04.008%26rft_id%253Dinfo%253Apmid%252F20430068%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.gene.2010.04.008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20430068&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1103.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000279363900001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-book" id="cit-9.4.1103.61"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Souvorov</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kaputsin</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kiryutin</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chetverin</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tatusova</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-source">Gnomon-NCBI eukaryotic gene prediction tool</span>, <span class="cit-publ-name">National Center for Biotechnology Information</span>, <span class="cit-publ-loc">Bethesda, MD</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1103.62" data-doi="10.1093/nar/gki458"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stanke</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morgenstern</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">AUGUSTUS: A web server for gene prediction in eukaryotes that allows user-defined constraints.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">W465</span>–<span class="cit-lpage">W467</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gki458</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DStanke%26rft.auinit1%253DM.%26rft.volume%253D33%26rft.issue%253Dsuppl_2%26rft.spage%253DW465%26rft.epage%253DW467%26rft.atitle%253DAUGUSTUS%253A%2Ba%2Bweb%2Bserver%2Bfor%2Bgene%2Bprediction%2Bin%2Beukaryotes%2Bthat%2Ballows%2Buser-defined%2Bconstraints%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgki458%26rft_id%253Dinfo%253Apmid%252F15980513%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gki458&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15980513&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1103.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000230271400094&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1103.63" data-doi="10.1038/nature11180"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Steinmetz</span>  <span class="cit-name-given-names">P. R. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kraus</span>  <span class="cit-name-given-names">J. E. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Larroux</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hammel</span>  <span class="cit-name-given-names">J. U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amon-Hassenzahl</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Independent evolution of striated muscles in cnidarians and bilaterians.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">487</span>: <span class="cit-fpage">231</span>–<span class="cit-lpage">234</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature11180</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DSteinmetz%26rft.auinit1%253DP.%2BR.%26rft.volume%253D487%26rft.issue%253D7406%26rft.spage%253D231%26rft.epage%253D234%26rft.atitle%253DIndependent%2Bevolution%2Bof%2Bstriated%2Bmuscles%2Bin%2Bcnidarians%2Band%2Bbilaterians.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature11180%26rft_id%253Dinfo%253Apmid%252F22763458%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature11180&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22763458&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1103.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000306278900039&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1103.64" data-doi="10.1073/pnas.0404206101"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tamura</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nei</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Prospects for inferring very large phylogenies by using the neighbor-joining method.</span> <abbr class="cit-jnl-abbrev">Prog. Nat. Sci.</abbr> <span class="cit-vol">101</span>: <span class="cit-fpage">11030</span>–<span class="cit-lpage">11035</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.0404206101</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProg.%2BNat.%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0404206101%26rft_id%253Dinfo%253Apmid%252F15258291%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTAxLzMwLzExMDMwIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS80LzExMDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1103.65" data-doi="10.1023/A:1010349416723"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tatsumi</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maeda</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hattori</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Takahashi</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Calcium binding to an elastic portion of connectin/titin filaments.</span> <abbr class="cit-jnl-abbrev">J. Muscle Res. Cell Motil.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">149</span>–<span class="cit-lpage">162</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1023/A:1010349416723</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmuscle%2Bresearch%2Band%2Bcell%2Bmotility%26rft.stitle%253DJ%2BMuscle%2BRes%2BCell%2BMotil%26rft.aulast%253DTatsumi%26rft.auinit1%253DR.%26rft.volume%253D22%26rft.issue%253D2%26rft.spage%253D149%26rft.epage%253D162%26rft.atitle%253DCalcium%2Bbinding%2Bto%2Ban%2Belastic%2Bportion%2Bof%2Bconnectin%252Ftitin%2Bfilaments.%26rft_id%253Dinfo%253Adoi%252F10.1023%252FA%253A1010349416723%26rft_id%253Dinfo%253Apmid%252F11519738%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1023/A:1010349416723&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11519738&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1103.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000170063100005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1103.66" data-doi="10.1126/science.278.5338.631"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tatusov</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koonin</span>  <span class="cit-name-given-names">E. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lipman</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">A genomic perspective on protein families.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">278</span>: <span class="cit-fpage">631</span>–<span class="cit-lpage">637</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.278.5338.631</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DTatusov%26rft.auinit1%253DR.%2BL.%26rft.volume%253D278%26rft.issue%253D5338%26rft.spage%253D631%26rft.epage%253D637%26rft.atitle%253DA%2BGenomic%2BPerspective%2Bon%2BProtein%2BFamilies%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.278.5338.631%26rft_id%253Dinfo%253Apmid%252F9381173%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIyNzgvNTMzOC82MzEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzQvMTEwMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1103.67" data-doi="10.1002/bies.10324"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tompa</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Intrinsically unstructured proteins evolve by repeat expansion.</span> <abbr class="cit-jnl-abbrev">BioEssays</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">847</span>–<span class="cit-lpage">855</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/bies.10324</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioEssays%2B%253A%2B%2Bnews%2Band%2Breviews%2Bin%2Bmolecular%252C%2Bcellular%2Band%2Bdevelopmental%2Bbiology%26rft.stitle%253DBioessays%26rft.aulast%253DTompa%26rft.auinit1%253DP.%26rft.volume%253D25%26rft.issue%253D9%26rft.spage%253D847%26rft.epage%253D855%26rft.atitle%253DIntrinsically%2Bunstructured%2Bproteins%2Bevolve%2Bby%2Brepeat%2Bexpansion.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fbies.10324%26rft_id%253Dinfo%253Apmid%252F12938174%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/bies.10324&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12938174&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1103.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000185779600005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1103.68" data-doi="10.1016/S0006-3495(00)76555-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Trombitás</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Redkar</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Centner</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Labeit</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Extensibility of isoforms of cardiac titin: variation in contour length of molecular subsegments provides a basis for cellular passive stiffness diversity.</span> <abbr class="cit-jnl-abbrev">Biophys. J.</abbr> <span class="cit-vol">79</span>: <span class="cit-fpage">3226</span>–<span class="cit-lpage">3234</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/S0006-3495(00)76555-6</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiophysical%2BJournal%26rft.stitle%253DBiophys.%2BJ%26rft.aulast%253DTrombitas%26rft.auinit1%253DK.%26rft.volume%253D79%26rft.issue%253D6%26rft.spage%253D3226%26rft.epage%253D3234%26rft.atitle%253DExtensibility%2Bof%2BIsoforms%2Bof%2BCardiac%2BTitin%253A%2BVariation%2Bin%2BContour%2BLength%2Bof%2BMolecular%2BSubsegments%2BProvides%2Ba%2BBasis%2Bfor%2BCellular%2BPassive%2BStiffness%2BDiversity%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0006-3495%252800%252976555-6%26rft_id%253Dinfo%253Apmid%252F11106626%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0006-3495(00)76555-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11106626&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1103.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000165749000039&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1103.69" data-doi="10.1016/S1672-0229(04)02028-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">A brief review of computational gene prediction methods.</span> <abbr class="cit-jnl-abbrev">Genomics Proteomics Bioinformatics</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">216</span>–<span class="cit-lpage">221</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/S1672-0229(04)02028-5</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenomics%252C%2Bproteomics%2B%2526%2Bbioinformatics%2B%252F%2BBeijing%2BGenomics%2BInstitute%26rft.stitle%253DGenomics%2BProteomics%2BBioinformatics%26rft.aulast%253DWang%26rft.auinit1%253DZ.%26rft.volume%253D2%26rft.issue%253D4%26rft.spage%253D216%26rft.epage%253D221%26rft.atitle%253DA%2Bbrief%2Breview%2Bof%2Bcomputational%2Bgene%2Bprediction%2Bmethods.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1672-0229%252804%252902028-5%26rft_id%253Dinfo%253Apmid%252F15901250%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S1672-0229(04)02028-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15901250&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1103.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1103.70" data-doi="10.1006/jsbi.1998.3993"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Witt</span>  <span class="cit-name-given-names">C. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Olivieri</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Centner</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kolmerer</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Millevoi</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">A survey of the primary structure and the interspecies conservation of I-band titin’s elastic elements in vertebrates.</span> <abbr class="cit-jnl-abbrev">J. Struct. Biol.</abbr> <span class="cit-vol">122</span>: <span class="cit-fpage">206</span>–<span class="cit-lpage">215</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1006/jsbi.1998.3993</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bstructural%2Bbiology%26rft.stitle%253DJ%2BStruct%2BBiol%26rft.aulast%253DWitt%26rft.auinit1%253DC.%2BC.%26rft.volume%253D122%26rft.issue%253D1-2%26rft.spage%253D206%26rft.epage%253D215%26rft.atitle%253DA%2Bsurvey%2Bof%2Bthe%2Bprimary%2Bstructure%2Band%2Bthe%2Binterspecies%2Bconservation%2Bof%2BI-band%2Btitin%2527s%2Belastic%2Belements%2Bin%2Bvertebrates.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fjsbi.1998.3993%26rft_id%253Dinfo%253Apmid%252F9724622%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/jsbi.1998.3993&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9724622&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1103.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000075746900022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1103.71" data-doi="10.1093/molbev/msm088"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">Z.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">PAML 4: Phylogenetic Analysis by Maximum Likelihood.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">1586</span>–<span class="cit-lpage">1591</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msm088</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsm088%26rft_id%253Dinfo%253Apmid%252F17483113%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msm088&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17483113&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1103.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000248848400003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference  in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1103.72" data-doi="10.1093/molbev/msi097"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wong</span>  <span class="cit-name-given-names">W. S. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Bayes Empirical Bayes Inference of Amino Acid Sites Under Positive Selection.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">1107</span>–<span class="cit-lpage">1118</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msi097</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsi097%26rft_id%253Dinfo%253Apmid%252F15689528%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msi097&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15689528&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1103.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000228139400031&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference  in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1103.73" data-doi="10.1093/nar/gkv1157"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yates</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Akanni</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amode</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barrell</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Billis</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Ensembl 2016.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">D710</span>–<span class="cit-lpage">D716</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkv1157</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkv1157%26rft_id%253Dinfo%253Apmid%252F26687719%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkv1157&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26687719&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1103.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference  in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1103.74" data-doi="10.1126/science.1251385"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Larkin</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Comparative genomics reveals insights into avian genome evolution and adaptation</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">346</span>: <span class="cit-fpage">1311</span>–<span class="cit-lpage">1320</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.1251385</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DZhang%26rft.auinit1%253DG.%26rft.volume%253D346%26rft.issue%253D6215%26rft.spage%253D1311%26rft.epage%253D1320%26rft.atitle%253DComparative%2Bgenomics%2Breveals%2Binsights%2Binto%2Bavian%2Bgenome%2Bevolution%2Band%2Badaptation%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1251385%26rft_id%253Dinfo%253Apmid%252F25504712%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzNDYvNjIxNS8xMzExIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS80LzExMDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference  in text" id="ref-75">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1103.75" data-doi="10.1371/journal.pone.0008326"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ru</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Teeling</span>  <span class="cit-name-given-names">E. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Faulkes</span>  <span class="cit-name-given-names">C. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Rhodopsin molecular evolution in mammals inhabiting low light environments.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e8326</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0008326</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DZhao%26rft.auinit1%253DH.%26rft.volume%253D4%26rft.issue%253D12%26rft.spage%253De8326%26rft.epage%253De8326%26rft.atitle%253DRhodopsin%2Bmolecular%2Bevolution%2Bin%2Bmammals%2Binhabiting%2Blow%2Blight%2Benvironments.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0008326%26rft_id%253Dinfo%253Apmid%252F20016835%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0008326&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20016835&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1103.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/9/4/1103.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/4/1085" title="Identification and Functional Testing of Novel Interacting Protein Partners for the Stress Sensors Wsc1p and Mid2p of Saccharomyces cerevisiae" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/4/1117" title="Lost in Translation: On the Problem of Data Coding in Penalized Whole Genome Regression with Interactions" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/9/4"><span>Volume 9 Issue 4, April 2019</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/9/4.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/9/4.cover-source.jpg?itok=mcy5SuIK" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 9 (4)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/4/1103" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-4zWzT-_nOORvKNRYV7ibzn-z8zPFAgUHWAkRfDhW2TY" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/9/4/1103" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/9/4/1103" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/9/4/1103" class="active" data-icon-position="" data-hide-link-title="0">Evolution of the Highly Repetitive PEVK Region of Titin Across Mammals</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/492618" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-Ytc-qtf368NX6803ujIOHkxV7hmfJALyVGUpT8xaNEo" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/4/1103.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/9/4/1103" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-X-YxcZeljORQ4qPS2cht2sgoH-8MBYIf4QY-BP-BvAI" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/9/4/1103" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/4/1103.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="492618" id="node492618--2" data-pisa="ggg;9/4/1103" data-pisa-master="ggg;g3.118.200714" data-apath="/ggg/9/4/1103.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Evolution of the Highly Repetitive PEVK Region of Titin Across Mammals</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-0840-614X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Kathleen Muenzen</span>, <span class="highwire-citation-author" data-delta="1">Jenna Monroy</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-5040-9627" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Findley R. Finseth</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1103-1115;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200714 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node492618--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Kathleen Muenzen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Keck Science Department, Claremont McKenna, Pitzer, and Scripps Colleges, Claremont, CA 91711</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biomedical Informatics and Medical Education, University of Washington, Seattle, WA 98105</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kathleen%2BMuenzen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Muenzen%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKathleen%2BMuenzen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-0840-614X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Kathleen Muenzen</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jenna Monroy </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Keck Science Department, Claremont McKenna, Pitzer, and Scripps Colleges, Claremont, CA 91711</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jenna%2BMonroy%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Monroy%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJenna%2BMonroy%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:ffinseth@kecksci.claremont.edu" class="" data-icon-position="" data-hide-link-title="0">ffinseth@kecksci.claremont.edu</a>
<a href="mailto:jmonroy@kecksci.claremont.edu" class="" data-icon-position="" data-hide-link-title="0">jmonroy@kecksci.claremont.edu</a></span></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Findley R. Finseth </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Keck Science Department, Claremont McKenna, Pitzer, and Scripps Colleges, Claremont, CA 91711</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Findley%2BR.%2BFinseth%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Finseth%20FR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AFindley%2BR.%2BFinseth%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-5040-9627" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Findley R. Finseth</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:ffinseth@kecksci.claremont.edu" class="" data-icon-position="" data-hide-link-title="0">ffinseth@kecksci.claremont.edu</a>
<a href="mailto:jmonroy@kecksci.claremont.edu" class="" data-icon-position="" data-hide-link-title="0">jmonroy@kecksci.claremont.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/9/4/1103" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/9/4/1103&amp;title=Evolution%20of%20the%20Highly%20Repetitive%20PEVK%20Region%20of%20Titin%20Across%20Mammals" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/9/4/1103&amp;title=Evolution%20of%20the%20Highly%20Repetitive%20PEVK%20Region%20of%20Titin%20Across%20Mammals" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/9/4/1103&amp;title=Evolution%20of%20the%20Highly%20Repetitive%20PEVK%20Region%20of%20Titin%20Across%20Mammals" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/9/4/1103&amp;text=Evolution%20of%20the%20Highly%20Repetitive%20PEVK%20Region%20of%20Titin%20Across%20Mammals" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/9/4/1103&amp;title=Evolution%20of%20the%20Highly%20Repetitive%20PEVK%20Region%20of%20Titin%20Across%20Mammals" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/9/4/1103&amp;t=Evolution%20of%20the%20Highly%20Repetitive%20PEVK%20Region%20of%20Titin%20Across%20Mammals" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/9/4/1103&amp;title=Evolution%20of%20the%20Highly%20Repetitive%20PEVK%20Region%20of%20Titin%20Across%20Mammals" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/9/4/1103&amp;title=Evolution%20of%20the%20Highly%20Repetitive%20PEVK%20Region%20of%20Titin%20Across%20Mammals" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/492618/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="492618" id="node492618--3" data-pisa="ggg;9/4/1103" data-pisa-master="ggg;g3.118.200714" data-apath="/ggg/9/4/1103.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Evolution of the Highly Repetitive PEVK Region of Titin Across Mammals</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-0840-614X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Kathleen Muenzen</span>, <span class="highwire-citation-author" data-delta="1">Jenna Monroy</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-5040-9627" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Findley R. Finseth</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1103-1115;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200714 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node492618--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Kathleen Muenzen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Keck Science Department, Claremont McKenna, Pitzer, and Scripps Colleges, Claremont, CA 91711</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biomedical Informatics and Medical Education, University of Washington, Seattle, WA 98105</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kathleen%2BMuenzen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Muenzen%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKathleen%2BMuenzen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-0840-614X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Kathleen Muenzen</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jenna Monroy </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Keck Science Department, Claremont McKenna, Pitzer, and Scripps Colleges, Claremont, CA 91711</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jenna%2BMonroy%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Monroy%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJenna%2BMonroy%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:ffinseth@kecksci.claremont.edu" class="" data-icon-position="" data-hide-link-title="0">ffinseth@kecksci.claremont.edu</a>
<a href="mailto:jmonroy@kecksci.claremont.edu" class="" data-icon-position="" data-hide-link-title="0">jmonroy@kecksci.claremont.edu</a></span></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Findley R. Finseth </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Keck Science Department, Claremont McKenna, Pitzer, and Scripps Colleges, Claremont, CA 91711</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Findley%2BR.%2BFinseth%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Finseth%20FR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AFindley%2BR.%2BFinseth%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-5040-9627" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Findley R. Finseth</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:ffinseth@kecksci.claremont.edu" class="" data-icon-position="" data-hide-link-title="0">ffinseth@kecksci.claremont.edu</a>
<a href="mailto:jmonroy@kecksci.claremont.edu" class="" data-icon-position="" data-hide-link-title="0">jmonroy@kecksci.claremont.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1103&amp;rft.epage=1115&amp;rft.atitle=Evolution%20of%20the%20Highly%20Repetitive%20PEVK%20Region%20of%20Titin%20Across%20Mammals&amp;rft.volume=9&amp;rft.issue=4&amp;rft.date=2019-04-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Muenzen%2C+Kathleen&amp;rft.au=Monroy%2C+Jenna&amp;rft.au=Finseth%2C+Findley+R."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/492618/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/492618/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/492618/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/492618/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/492618/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/492618/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/492618/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/492618/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/492618/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/492618/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/492618/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/492618/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/9/4/1103" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-7" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-13" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/9/4/1103.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/9/4/1103.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/9/4/1103.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/9/4/1103.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
