<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/9/4/1117.full.txt" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/9/4/1117.full.pdf" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/9/4/1117/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/9/4/1117.atom" />
<meta name="HW.pisa" content="ggg;9/4/1117" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Lost in Translation: On the Problem of Data Coding in Penalized Whole Genome Regression with Interactions" />
<meta name="DC.Identifier" content="10.1534/g3.118.200961" />
<meta name="DC.Date" content="2019-04-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright Â© 2019 Martini et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Mixed models can be considered as a type of penalized regression and are everyday tools in statistical genetics. The standard mixed model for whole genome regression (WGR) is ridge regression best linear unbiased prediction (RRBLUP) which is based on an additive marker effect model. Many publications have extended the additive WGR approach by incorporating interactions between loci or between genes and environment. In this context of penalized regressions with interactions, it has been reported that translating the coding of single nucleotide polymorphisms -for instance from -1,0,1 to 0,1,2- has an impact on the prediction of genetic values and interaction effects. In this work, we identify the reason for the relevance of variable coding in the general context of penalized polynomial regression. We show that in many cases, predictions of the genetic values are not invariant to translations of the variable coding, with an exception when only the sizes of the coefficients of monomials of highest total degree are penalized. The invariance of RRBLUP can be considered as a special case of this setting, with a polynomial of total degree 1, penalizing additive effects (total degree 1) but not the fixed effect (total degree 0). The extended RRBLUP (eRRBLUP), which includes interactions, is not invariant to translations because it does not only penalize interactions (total degree 2), but also additive effects (total degree 1). This observation implies that translation-invariance can be maintained in a pair-wise epistatic WGR if only interaction effects are penalized, but not the additive effects. In this regard, approaches of pre-selecting loci may not only reduce computation time, but can also help to avoid the variable coding issue. To illustrate the practical relevance, we compare different regressions on a publicly available wheat data set. We show that for an eRRBLUP, the relevance of the marker coding for interaction effect estimates increases with the number of variables included in the model. A biological interpretation of estimated interaction effects may therefore become more difficult. Consequently, comparing reproducing kernel Hilbert space (RKHS) approaches to WGR approaches modeling effects explicitly, the supposed advantage of an increased interpretability of the latter may not be real. Our theoretical results are generally valid for penalized regressions, for instance also for the least absolute shrinkage and selection operator (LASSO). Moreover, they apply to any type of interaction modeled by products of predictor variables in a penalized regression approach or by Hadamard products of covariance matrices in a mixed model." />
<meta name="DC.Contributor" content="Johannes W. R. Martini" />
<meta name="DC.Contributor" content="Francisco Rosales" />
<meta name="DC.Contributor" content="Ngoc-Thuy Ha" />
<meta name="DC.Contributor" content="Johannes Heise" />
<meta name="DC.Contributor" content="Valentin Wimmer" />
<meta name="DC.Contributor" content="Thomas Kneib" />
<meta name="article:published_time" content="2019-04-01" />
<meta name="article:section" content="Genomic Prediction" />
<meta name="citation_title" content="Lost in Translation: On the Problem of Data Coding in Penalized Whole Genome Regression with Interactions" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Mixed models can be considered as a type of penalized regression and are everyday tools in statistical genetics. The standard mixed model for whole genome regression (WGR) is &lt;i&gt;ridge regression best linear unbiased prediction&lt;/i&gt; (RRBLUP) which is based on an additive marker effect model. Many publications have extended the additive WGR approach by incorporating interactions between loci or between genes and environment. In this context of penalized regressions with interactions, it has been reported that translating the coding of &lt;i&gt;single nucleotide polymorphisms&lt;/i&gt; -for instance from -1,0,1 to 0,1,2- has an impact on the prediction of genetic values and interaction effects. In this work, we identify the reason for the relevance of variable coding in the general context of penalized polynomial regression. We show that in many cases, predictions of the genetic values are not invariant to translations of the variable coding, with an exception when only the sizes of the coefficients of monomials of highest total degree are penalized. The invariance of RRBLUP can be considered as a special case of this setting, with a polynomial of total degree 1, penalizing additive effects (total degree 1) but not the fixed effect (total degree 0). The extended RRBLUP (eRRBLUP), which includes interactions, is not invariant to translations because it does not only penalize interactions (total degree 2), but also additive effects (total degree 1). This observation implies that translation-invariance can be maintained in a pair-wise epistatic WGR if only interaction effects are penalized, but not the additive effects. In this regard, approaches of pre-selecting loci may not only reduce computation time, but can also help to avoid the variable coding issue. To illustrate the practical relevance, we compare different regressions on a publicly available wheat data set. We show that for an eRRBLUP, the relevance of the marker coding for interaction effect estimates increases with the number of variables included in the model. A biological interpretation of estimated interaction effects may therefore become more difficult. Consequently, comparing &lt;i&gt;reproducing kernel Hilbert space&lt;/i&gt; (RKHS) approaches to WGR approaches modeling effects explicitly, the supposed advantage of an increased interpretability of the latter may not be real. Our theoretical results are generally valid for penalized regressions, for instance also for the &lt;i&gt;least absolute shrinkage and selection operator&lt;/i&gt; (LASSO). Moreover, they apply to any type of interaction modeled by products of predictor variables in a penalized regression approach or by Hadamard products of covariance matrices in a mixed model.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2019/04/01" />
<meta name="citation_mjid" content="ggg;9/4/1117" />
<meta name="citation_id" content="9/4/1117" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/9/4/1117" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/9/4/1117.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/9/4/1117.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/9/4/1117.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.118.200961" />
<meta name="citation_pmid" content="30760541" />
<meta name="citation_volume" content="9" />
<meta name="citation_issue" content="4" />
<meta name="citation_num_pages" content="13" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Genomic Prediction" />
<meta name="citation_firstpage" content="1117" />
<meta name="citation_lastpage" content="1129" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Johannes W. R. Martini" />
<meta name="citation_author_institution" content="KWS SAAT SE, Einbeck, Germany" />
<meta name="citation_author_email" content="jmartin2@gwdg.de" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-0628-6794" />
<meta name="citation_author" content="Francisco Rosales" />
<meta name="citation_author_institution" content="Universidad del PacÃ­fico, Academic Department of Finance, Lima, Peru" />
<meta name="citation_author" content="Ngoc-Thuy Ha" />
<meta name="citation_author_institution" content="Department of Animal Breeding and Genetics" />
<meta name="citation_author" content="Johannes Heise" />
<meta name="citation_author_institution" content="IT Solutions for Animal Production (vit), Verden, Germany" />
<meta name="citation_author" content="Valentin Wimmer" />
<meta name="citation_author_institution" content="KWS SAAT SE, Einbeck, Germany" />
<meta name="citation_author" content="Thomas Kneib" />
<meta name="citation_author_institution" content="Chairs of Statistics and Econometrics, University of Goettingen, Goettingen, Germany" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. M. Ãlvarez-Castro;citation_author=Ã. Carlborg;citation_title=A unified model for functional and statistical epistasis and its application in quantitative trait loci analysis.;citation_volume=176;citation_year=2007;citation_pmid=17409082;citation_doi=10.1534/genetics.106.067348" />
<meta name="citation_reference" content="citation_journal_title=Genet. Epidemiol.;citation_author=H. Aschard;citation_title=A perspective on interaction effects in genetic association studies.;citation_pages=678-688;citation_volume=40;citation_year=2016;citation_doi=10.1002/gepi.21989" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=Ã. Carlborg;citation_author=L. Jacobsson;citation_author=P. Ãhgren;citation_author=P. Siegel;citation_author=L. Andersson;citation_title=Epistasis and the release of genetic variation during long-term selection.;citation_pages=418-420;citation_volume=38;citation_year=2006;citation_issue=4;citation_pmid=16532011;citation_doi=10.1038/ng1761" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=Q. Chen;citation_author=X. Mao;citation_author=Z. Zhang;citation_author=R. Zhu;citation_author=Z. Yin;citation_title=SNP-SNP interaction analysis on soybean oil content under multi-environments.;citation_pages=e0163692;citation_volume=11;citation_year=2016;citation_doi=10.1371/journal.pone.0163692" />
<meta name="citation_reference" content="citation_author=D. Clifford;citation_author=P. McCullagh;citation_author=M. D. Clifford;citation_pages=1-3;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=H. J. Cordell;citation_title=Detecting gene-gene interactions that underlie human diseases.;citation_pages=392-404;citation_volume=10;citation_year=2009;citation_issue=6;citation_pmid=19434077;citation_doi=10.1038/nrg2579" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. Crossa;citation_author=G. de los Campos;citation_author=P. PÃ©rez;citation_author=D. Gianola;citation_author=J. BurgueÃ±o;citation_title=Prediction of genetic values of quantitative traits in plant breeding using pedigree and molecular markers.;citation_volume=186;citation_year=2010;citation_pmid=20813882;citation_doi=10.1534/genetics.110.118521" />
<meta name="citation_reference" content="citation_title=Genetics;citation_author=A. De Coninck;citation_author=B. De Baets;citation_author=D. Kourounis;citation_author=F. Verbosio;citation_author=O. Schenk;citation_title=Needles: toward large-scale genomic prediction with marker-by-environment interaction.;citation_volume=203;citation_year=2016;citation_pmid=26936924;citation_doi=10.1534/genetics.115.179887" />
<meta name="citation_reference" content="citation_journal_title=J. Anim. Sci.;citation_author=G. de los Campos;citation_author=D. Gianola;citation_author=G. J. M. Rosa;citation_title=Reproducing kernel Hilbert spaces regression: a general framework for genetic evaluation.;citation_volume=87;citation_year=2009;citation_pmid=19213705;citation_doi=10.2527/jas.2008-1259" />
<meta name="citation_reference" content="citation_title=Genetics;citation_author=G. de los Campos;citation_author=J. M. Hickey;citation_author=R. Pong-Wong;citation_author=H. D. Daetwyler;citation_author=M. P. Calus;citation_title=Whole-genome regression and prediction methods applied to plant and animal breeding.;citation_volume=193;citation_year=2013;citation_pmid=22745228;citation_doi=10.1534/genetics.112.143313" />
<meta name="citation_reference" content="de los Campos, G. and P. Perez Rodriguez, 2016âBGLR: Bayesian Generalized Linear Regression. R package version 1.0.5." />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=I. M. Ehrenreich;citation_title=Epistasis: Searching for interacting genetic variants using crosses.;citation_volume=206;citation_year=2017;citation_pmid=28592494;citation_doi=10.1534/genetics.117.203059" />
<meta name="citation_reference" content="citation_journal_title=J. Hered.;citation_author=H. Esfandyari;citation_author=M. Henryon;citation_author=P. Berg;citation_author=J. R. Thomasen;citation_author=P. Bijma;citation_title=Response to selection in finite locus models with nonadditive effects.;citation_volume=108;citation_year=2017;citation_pmid=28082328;citation_doi=10.1093/jhered/esw123" />
<meta name="citation_reference" content="citation_title=Introduction to Quantitative Genetics;citation_author=D. S. Falconer;citation_author=T. F. C. Mackay;citation_year=1996" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=N. S. Forneris;citation_author=Z. G. Vitezica;citation_author=A. Legarra;citation_author=M. PÃ©rez-Enciso;citation_title=Influence of epistasis on response to genomic selection using complete sequence data.;citation_pages=66;citation_volume=49;citation_year=2017;citation_doi=10.1186/s12711-017-0340-3" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=N. Gao;citation_author=J. W. R. Martini;citation_author=Z. Zhang;citation_author=X. Yuan;citation_author=H. Zhang;citation_title=Incorporating gene annotation into genomic prediction of complex phenotypes.;citation_volume=207;citation_year=2017;citation_pmid=28839043;citation_doi=10.1534/genetics.117.300198" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=D. Gianola;citation_title=Priors in whole-genome regression: The Bayesian alphabet returns.;citation_volume=194;citation_year=2013;citation_pmid=23636739;citation_doi=10.1534/genetics.113.151753" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=D. Gianola;citation_author=G. de los Campos;citation_author=W. G. Hill;citation_author=E. Manfredi;citation_author=R. Fernando;citation_title=Additive genetic variability and the Bayesian alphabet.;citation_volume=183;citation_year=2009;citation_pmid=19620397;citation_doi=10.1534/genetics.109.103952" />
<meta name="citation_reference" content="citation_author=D. Gianola;citation_author=G. Morota;citation_author=J. Crossa;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=D. Gianola;citation_author=J. B. C. H. M. Van Kaam;citation_title=Reproducing Kernel Hilbert Spaces Regression Methods for Genomic Assisted Prediction of Quantitative Traits;citation_pages=2289-2303;citation_volume=178;citation_year=2008;citation_issue=4;citation_pmid=18430950;citation_doi=10.1534/genetics.107.084285" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=D. Habier;citation_author=R. L. Fernando;citation_author=J. C. M. Dekkers;citation_title=The Impact of Genetic Relationship Information on Genome-Assisted Breeding Values;citation_pages=2389-2397;citation_volume=177;citation_year=2007;citation_issue=4;citation_pmid=18073436;citation_doi=10.1534/genetics.107.081190" />
<meta name="citation_reference" content="citation_journal_title=Plant Breed.;citation_author=B. J. Hayes;citation_author=N. O. I. Cogan;citation_author=L. W. Pembleton;citation_author=M. E. Goddard;citation_author=J. Wang;citation_title=Prospects for genomic selection in forage plant species.;citation_pages=133-143;citation_volume=132;citation_year=2013;citation_doi=10.1111/pbr.12037" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=B. J. Hayes;citation_author=P. M. Visscher;citation_author=M. E. Goddard;citation_title=Increased accuracy of artificial selection by using the realized relationship matrix.;citation_pages=47-60;citation_volume=91;citation_year=2009;citation_issue=1;citation_pmid=19220931;citation_doi=10.1017/S0016672308009981" />
<meta name="citation_reference" content="citation_journal_title=BMC Bioinformatics;citation_author=D. He;citation_author=L. Parida;citation_title=Does encoding matter? a novel view on the quantitative genetic trait prediction problem.;citation_pages=272;citation_volume=17;citation_year=2016;citation_doi=10.1186/s12859-016-1127-1" />
<meta name="citation_reference" content="citation_journal_title=Biometrics;citation_journal_abbrev=Biometrics;citation_author=C. R. Henderson;citation_title=Best linear unbiased estimation and prediction under a selection model.;citation_pages=423-447;citation_volume=31;citation_year=1975;citation_issue=2;citation_pmid=1174616;citation_doi=10.2307/2529430" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=C. R. Henderson;citation_title=Best Linear Unbiased Prediction of Breeding Values Not in the Model for Records;citation_pages=783-787;citation_volume=60;citation_year=1977;citation_issue=5;citation_doi=10.3168/jds.S0022-0302(77)83935-0" />
<meta name="citation_reference" content="citation_journal_title=Journal of Animal Science;citation_journal_abbrev=J ANIM SCI;citation_author=C. R. Henderson;citation_author=R. L. Quaas;citation_title=Multiple Trait Evaluation Using Relatives' Records;citation_pages=1188-1197;citation_volume=43;citation_year=1976;citation_issue=6;citation_doi=10.2527/jas1976.4361188x" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=R. Howard;citation_author=A. L. Carriquiry;citation_author=W. D. Beavis;citation_title=Parametric and nonparametric statistical methods for genomic selection of traits with additive and epistatic genetic architectures.;citation_volume=4;citation_year=2014;citation_pmid=24727289;citation_doi=10.1534/g3.114.010298" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=Y. Jiang;citation_author=J. C. Reif;citation_title=Modeling epistasis in genomic selection.;citation_volume=201;citation_year=2015;citation_pmid=26219298;citation_doi=10.1534/genetics.115.177907" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=H. P. KÃ¤rkkÃ¤inen;citation_author=Z. Li;citation_author=M. J. SillanpÃ¤Ã¤;citation_title=An efficient genome-wide multilocus epistasis search.;citation_volume=201;citation_year=2015;citation_pmid=26405029;citation_doi=10.1534/genetics.115.182444" />
<meta name="citation_reference" content="citation_journal_title=BMC Bioinformatics;citation_author=J. W. R. Martini;citation_author=N. Gao;citation_author=D. F. Cardoso;citation_author=V. Wimmer;citation_author=M. Erbe;citation_title=Genomic prediction with epistasis models: on the marker-coding-dependent performance of the extended GBLUP and properties of the categorical epistasis model (CE).;citation_pages=3;citation_volume=18;citation_year=2017;citation_doi=10.1186/s12859-016-1439-1" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=J. W. R. Martini;citation_author=V. Wimmer;citation_author=M. Erbe;citation_author=H. Simianer;citation_title=Epistasis and covariance: how gene interaction translates into genomic relationship.;citation_pages=963-976;citation_volume=129;citation_year=2016;citation_doi=10.1007/s00122-016-2675-5" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=T. H. E. Meuwissen;citation_author=B. J. Hayes;citation_author=M. E. Goddard;citation_title=Prediction of Total Genetic Value Using Genome-Wide Dense Marker Maps;citation_pages=1819-1829;citation_volume=157;citation_year=2001;citation_issue=4;citation_pmid=11290733;citation_doi=10.1093/nar/gkv1344" />
<meta name="citation_reference" content="citation_journal_title=Front. Genet.;citation_author=G. Morota;citation_author=D. Gianola;citation_title=Kernel-based whole-genome prediction of complex traits: a review.;citation_pages=363;citation_volume=5;citation_year=2014;citation_pmid=25360145;citation_doi=10.3389/fgene.2014.00363" />
<meta name="citation_reference" content="citation_title=Linear models for the prediction of animal breeding values;citation_author=R. A. Mrode;citation_year=2014;citation_doi=10.1079/9781780643915.0000" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=U. Ober;citation_author=M. Erbe;citation_author=N. Long;citation_author=E. Porcu;citation_author=M. Schlather;citation_title=Predicting genetic values: A kernel-based best linear unbiased prediction with genomic data.;citation_volume=188;citation_year=2011;citation_pmid=21515573;citation_doi=10.1534/genetics.111.128694" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=U. Ober;citation_author=W. Huang;citation_author=M. Magwire;citation_author=M. Schlather;citation_author=H. Simianer;citation_title=Accounting for genetic architecture improves sequence based genomic prediction for a Drosophila fitness trait.;citation_pages=e0126880;citation_volume=10;citation_year=2015;citation_pmid=25950439;citation_doi=10.1371/journal.pone.0126880" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=P. PÃ©rez-RodrÃ­guez;citation_author=J. Crossa;citation_author=J. Rutkoski;citation_author=J. Poland;citation_author=R. Singh;citation_title=Single-step genomic and pedigree genotype Ã environment interaction models for predicting wheat lines in international environments.;citation_pages=1-15;citation_volume=10;citation_year=2017;citation_doi=10.3835/plantgenome2016.09.0089" />
<meta name="citation_reference" content="citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=bioRxiv;citation_title=A subfunctionalization epistasis model to evaluate homeologous gene interactions in allopolyploid wheat.;citation_author=N. Santantonio;citation_author=J.-L. Jannink;citation_author=M. Sorrells;citation_pages=1-72;citation_year=2018" />
<meta name="citation_reference" content="citation_journal_title=Livest. Prod. Sci.;citation_author=L. Schaeffer;citation_title=Application of random regression models in animal breeding.;citation_pages=35-45;citation_volume=86;citation_year=2004;citation_doi=10.1016/S0301-6226(03)00151-9" />
<meta name="citation_reference" content="citation_journal_abbrev=J Anim Breed Genet;citation_author=L. Schaeffer;citation_title=Strategy for applying genome-wide selection in dairy cattle.;citation_pages=218-223;citation_volume=123;citation_year=2006;citation_issue=4;citation_pmid=16882088;citation_doi=10.1111/j.1439-0388.2006.00595.x" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=L. Shang;citation_author=Y. Wang;citation_author=S. Cai;citation_author=X. Wang;citation_author=Y. Li;citation_title=Partial dominance, overdominance, epistasis and qtl by environment interactions contribute to heterosis in two upland cotton hybrids.;citation_volume=6;citation_year=2015;citation_pmid=26715091;citation_doi=10.1534/g3.115.025809" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=I. StrandÃ©n;citation_author=O. F. Christensen;citation_title=Allele coding in genomic evaluation.;citation_pages=25-25;citation_volume=43;citation_year=2011;citation_pmid=21703021;citation_doi=10.1186/1297-9686-43-25" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=G. Su;citation_author=O. F. Christensen;citation_author=T. Ostersen;citation_author=M. Henryon;citation_author=M. S. Lund;citation_title=Estimating additive and non-additive genetic variances and predicting genetic merits using genome-wide dense single nucleotide polymorphism markers.;citation_pages=e45293-e45293;citation_volume=7;citation_year=2012;citation_issue=9;citation_pmid=23028912;citation_doi=10.1371/journal.pone.0045293" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=S. Sukumaran;citation_author=J. Crossa;citation_author=D. Jarquin;citation_author=M. Lopes;citation_author=M. P. Reynolds;citation_title=Genomic prediction with pedigree and genotype Ã environment interaction in spring wheat grown in south and west asia, north africa, and mexico.;citation_volume=7;citation_year=2017;citation_pmid=27903632;citation_doi=10.1534/g3.116.036251" />
<meta name="citation_reference" content="citation_title=J. Roy. Stat. Soc. B Met.;citation_author=R. Tibshirani;citation_title=Regression shrinkage and selection via the Lasso.;citation_pages=267-288;citation_volume=58;citation_year=1996" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=P. M. VanRaden;citation_title=Efficient Methods to Compute Genomic Predictions;citation_pages=4414-4423;citation_volume=91;citation_year=2008;citation_issue=11;citation_pmid=18946147;citation_doi=10.3168/jds.2007-0980" />
<meta name="citation_reference" content="citation_journal_title=Front. Genet.;citation_author=L. Varona;citation_author=A. Legarra;citation_author=M. A. Toro;citation_author=Z. G. Vitezica;citation_title=Non-additive effects in genomic selection.;citation_pages=78;citation_volume=9;citation_year=2018;citation_doi=10.3389/fgene.2018.00078" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=Z. G. Vitezica;citation_author=A. Legarra;citation_author=M. A. Toro;citation_author=L. Varona;citation_title=Orthogonal estimates of variances for additive, dominance, and epistatic effects in populations.;citation_volume=206;citation_year=2017;citation_pmid=28522540;citation_doi=10.1534/genetics.116.199406" />
<meta name="citation_reference" content="citation_title=Genetics;citation_author=T. Xiang;citation_author=O. F. Christensen;citation_author=Z. G. Vitezica;citation_author=A. Legarra;citation_title=Genomic model with correlation between additive and dominance effects.;citation_volume=209;citation_year=2018;citation_pmid=29743175;citation_doi=10.1534/genetics.118.301015" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Lost in Translation: On the Problem of Data Coding in Penalized Whole Genome Regression with Interactions" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4.cover-source.jpg" />
<meta name="twitter:description" content="Mixed models can be considered as a type of penalized regression and are everyday tools in statistical genetics. The standard mixed model for whole genome regression (WGR) is ridge regression best linear unbiased prediction (RRBLUP) which is based on an additive marker effect model. Many publications have extended the additive WGR approach by incorporating interactions between loci or between genes and environment. In this context of penalized regressions with interactions, it has been reported that translating the coding of single nucleotide polymorphisms -for instance from -1,0,1 to 0,1,2- has an impact on the prediction of genetic values and interaction effects. In this work, we identify the reason for the relevance of variable coding in the general context of penalized polynomial regression. We show that in many cases, predictions of the genetic values are not invariant to translations of the variable coding, with an exception when only the sizes of the coefficients of monomials of highest total degree are penalized. The invariance of RRBLUP can be considered as a special case of this setting, with a polynomial of total degree 1, penalizing additive effects (total degree 1) but not the fixed effect (total degree 0). The extended RRBLUP (eRRBLUP), which includes interactions, is not invariant to translations because it does not only penalize interactions (total degree 2), but also additive effects (total degree 1). This observation implies that translation-invariance can be maintained in a pair-wise epistatic WGR if only interaction effects are penalized, but not the additive effects. In this regard, approaches of pre-selecting loci may not only reduce computation time, but can also help to avoid the variable coding issue. To illustrate the practical relevance, we compare different regressions on a publicly available wheat data set. We show that for an eRRBLUP, the relevance of the marker coding for interaction effect estimates increases with the number of variables included in the model. A biological interpretation of estimated interaction effects may therefore become more difficult. Consequently, comparing reproducing kernel Hilbert space (RKHS) approaches to WGR approaches modeling effects explicitly, the supposed advantage of an increased interpretability of the latter may not be real. Our theoretical results are generally valid for penalized regressions, for instance also for the least absolute shrinkage and selection operator (LASSO). Moreover, they apply to any type of interaction modeled by products of predictor variables in a penalized regression approach or by Hadamard products of covariance matrices in a mixed model." />
<meta name="og-title" property="og:title" content="Lost in Translation: On the Problem of Data Coding in Penalized Whole Genome Regression with Interactions" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/9/4/1117" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Mixed models can be considered as a type of penalized regression and are everyday tools in statistical genetics. The standard mixed model for whole genome regression (WGR) is ridge regression best linear unbiased prediction (RRBLUP) which is based on an additive marker effect model. Many publications have extended the additive WGR approach by incorporating interactions between loci or between genes and environment. In this context of penalized regressions with interactions, it has been reported that translating the coding of single nucleotide polymorphisms -for instance from -1,0,1 to 0,1,2- has an impact on the prediction of genetic values and interaction effects. In this work, we identify the reason for the relevance of variable coding in the general context of penalized polynomial regression. We show that in many cases, predictions of the genetic values are not invariant to translations of the variable coding, with an exception when only the sizes of the coefficients of monomials of highest total degree are penalized. The invariance of RRBLUP can be considered as a special case of this setting, with a polynomial of total degree 1, penalizing additive effects (total degree 1) but not the fixed effect (total degree 0). The extended RRBLUP (eRRBLUP), which includes interactions, is not invariant to translations because it does not only penalize interactions (total degree 2), but also additive effects (total degree 1). This observation implies that translation-invariance can be maintained in a pair-wise epistatic WGR if only interaction effects are penalized, but not the additive effects. In this regard, approaches of pre-selecting loci may not only reduce computation time, but can also help to avoid the variable coding issue. To illustrate the practical relevance, we compare different regressions on a publicly available wheat data set. We show that for an eRRBLUP, the relevance of the marker coding for interaction effect estimates increases with the number of variables included in the model. A biological interpretation of estimated interaction effects may therefore become more difficult. Consequently, comparing reproducing kernel Hilbert space (RKHS) approaches to WGR approaches modeling effects explicitly, the supposed advantage of an increased interpretability of the latter may not be real. Our theoretical results are generally valid for penalized regressions, for instance also for the least absolute shrinkage and selection operator (LASSO). Moreover, they apply to any type of interaction modeled by products of predictor variables in a penalized regression approach or by Hadamard products of covariance matrices in a mixed model." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4.cover-source.jpg" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="canonical" href="https://www.g3journal.org/content/9/4/1117" />
    <title>Lost in Translation: On the Problem of Data Coding in Penalized Whole Genome Regression with Interactions | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings, {"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"RtGqvC4XeDsRuUuDQh0G3n67IMmD8mW5JEVl8fwPT6g","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"492621","apath":"\/ggg\/9\/4\/1117.atom","pisa":"ggg;9\/4\/1117","ac":{"\/ggg\/9\/4\/1117.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/9\/4\/1117.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;9\/4\/1117"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;9\/4\/1117"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/9\/4\/1117":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022NpoF3\\\\\\\/qLsIxitMZOTW+w00xmCDe3tgz+RIRkMlQUI6+q48dQ7V3pwHT\\\\\\\/tg\\\\\\\/y\\\\\\\/0QSuNoEKd5D6JYh46sl2pPIMI1sdE6d2pWZLiR1b1n8rIqQQlxK3kuWorT3fZQq87vtuzgH2yAUAFWvqRLvC1j4DO\\\\\\\/P11L35ggppUfYXd5Uu\\\\\\\/HEl5iNq5wQ1quu27sSDKWQ8nd0WLXsuf8vQBOUh9CX6GNaIF9tSHcssNpVN+tSzHAx88+VM665a1zaVMWsa\\\\\\\/JSu1e+HLkhuHg8QHr+yJd\\\\\\\/Yokb8ooDGBOzql+YIBz8FwbqoFfLeibTS2+brqXowpktoPzLg6pTdj0FQKc+aKLO9McHbgckG7EqjiobfPTon9VtVs4UViCQINbRl\\\\\\\/L\\\\\\\/+nOwod20au27t2JJqmxN9KsT6CtUabETirn2jojTR7uHCIiJeW5Scw\\\\\\\/nP0Gb2LR9T7zWrrOx\\\\\\\/+gXcxJoxP4wXy+lLinVXrelBPG9gHJYE0an\\\\\\\/4hJrW8o\\\\\\\/IiYg43bpR\\\\\\\/T5ZrFFq1w\\\\\\\/oC0HG2utA0YvB9RzhsKyXGqHCy8uKQczHGieqXqLjLa0V5Gp8C1z7Uuzl7rdUCmkkqPRerXdLeB9R9n34s0JneTERqas+3UTs+YSdm1SHfLpBPDGBkX1QzmrhtWhzshskrZXUySJJdhK4Kiaa17ZsEZYaP8livsRdWErKOL\\\\\\\/ihnMI7koA\\\\\\\/DXMR73ABY18YH3cCJhvEMbIOzIVuodQPZH14IRLXLpDLe+qlH4xXPNuO1tnyhaQwAysUgo5rtPJi6p8bBJz7ZtBOA5AcA1LxUB5idLjZkynbABv+\\\\\\\/fzUyz7EJYK7lac\\\\\\\/TFFBEQoiKWTB5ME5MNqdem\\\\\\\/QVCnL1HB9AypoDdzZHTsuxiODdGpYO8cUeJBZGa6K67R2i2ELGJBkj\\\\\\\/U01UKVRtW91JrswG06lYfkDNZTNbA\\\\\\\/w02RMixdJw+o\\\\\\\/C8yb1vdtSKh46HdFJzMykJPS3aylas1tS0rc38FMcUWtGnUCpnToSrjBzes49Knlj9oPfMQeWAYvpA75Ml70Czp5dQZaU5JF2\\\\\\\/bNGESGD2l3rvluGGHCvsbMfjXO2zMm18JvKeEk7RwHO9Q0T75pNMDd7OlxD35zZhWm1IgJ0dbD+IV4092kB9Ixn+D+iLRv37blsdXnSmD5oEAod3uLV+xp3DofxUUVXvicjqeUT71KZHj+MOpNt+PiJpIQ22Nz0G20FThIcHyeLV2qYaa+X7ZyAvlzkU1aDUldYXXcBensVaZu0tqzhzK3cNWohogQ15vW09ih9O9h+eGX\\\u0022,\\\u0022iv\\\u0022:\\\u0022Kn\\\\\\\/OLoI8ZEN2gp47hfRgNg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00220f5b70b1d16b9c15148cc7f172213a0992e4aee206e8d80e563420b36db1153e\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022yQVaeQQAcSoPPOzd+9ulw+ppKv1Jcu52FXpOuoqVQAIHWxcXQbK1yzXHatqGuxyhl704x3u3wBNZ9ahq38mZIdYly7y\\\\\\\/zFkRiDS+iYZpSZ3ZUktbdoBwFzpLn69ilNBLK\\\\\\\/pEhUbwe38IfyAa1eFtIgsbisfcptZRQmj7jnastVbyZySHBuQKpPUUJ\\\\\\\/xNnf8gRMcgJ4\\\\\\\/3gFPMwFK0LYu\\\\\\\/9NqTOtxaKCY3p4KKc9d3FnZp726ZWhC0GtqAy6hEKqJtMNWGdQq0FiZ1Tf4Po8E8ElPEvqycBPk6eO4kzbvu\\\\\\\/rB8cBNc32fSnkJFoCzmZAKLnHsSFE9sfP2PmAuESO9t11x1sivnhU2SCt1RtfeAoWrOxOH3Cx88jLtP\\\\\\\/2pSZ3Cu6XYItZx6c9pr2XCrLwP4Ltz0F3pceqjw3lO2e7yRRERMn5bydZof0yKZIAZOOcqsMItZqi7SFLoN78QcaBO8Uz+PvKVuoL3HXQYhPV4bBt4596ZxtUog+vhen7XhwH5qgL9KtgKw4UcTAEGSQMdaWB90ltxKO2XJ4x2lBKPA+bY2xSEfBBcWxTkAcUAm6SRa0locqH7m7m6Xmd9a7ktUVfg3iM7A2bKdRHL+BwEV0lpm2nAQMJ9soKDUOMcqsbjrbifFGg5fm\\\\\\\/lPywOk9JluISlfSIfw2DK3\\\\\\\/dZXEXTmnzL33ZDpPaZ5f5saNaTGiBNTZfjUGir+WbCnb68I6DYyLNejt64zfy8ZB203zMKA+D9xGpchdBsuVwoy3\\\\\\\/9Btpstc3Dteqeh5JGSVz4Ss4unCLOAG4FY5YjBIysWEuEFb7RCyxEBEhLT0Fh0fSD51otOBYspqJw3jHw8mrB+YGNpLUxkbX4mgje+juSJpyMf3sPyGwtPhrzS3L0AWY8Hosz\\\\\\\/vCfn1RHCcH4Ikbn5Tik5JitnOLWsX3my42eBkM1HCiNnDWUqACkfvSBgfrAhdbPzF1NjShdcpXUashp6RnQm1xwCCidKW8SqLBivHWX3YBh44wOC5t5dRGypZHWrr0pf8bIvlY4tq3StEx3nroy5fFzCUqXhd\\\\\\\/hFHqr0llbcC92faLCo0PaYwenhbsYQoB0oWTTU0E3qrCcTM2k5NbdSxCiuA9SFLOUnX83gVA1PHnQsHfpomHsMMy3I7GbnOYliK\\\\\\\/o2N9BUmF5Sxl+wMTyHmNUt0cQKV9M+E4lLziW9\\\\\\\/EGgGDSXq6MEcByUJdal1zAIF6GF\\\\\\\/a\\\\\\\/3Qw6rcIjQngijN9RahY0L\\\\\\\/McEWBKyPxtJxapK76cLCpJwhJ22G9CvO8ydPHSgBi3JvCiXRw5t\\\\\\\/+Lwomkt5cuDffdHR46qLN15R8kcFoBWBpJlfIjdqdPpJYQxoAX8rDMrtuILAp\\\\\\\/09ONp+p1TGTjmG5AApxv0oxeQNd5\\\\\\\/zH9baoBfpQ48Mwfsp6Xlkm31YS5as81ZB+FyU5vUyvx4wa8TPso73LEjV+fcK1ZcetVdtwfUK8KWsR1dSw7kvC3X55rTqEH62nrJyZ3ZaRxI2nIABjYAdL2A6NCmTGndi6tB+H4mqIANUoNroV27FAMrrrKeFiFt1aQ5phKbsXGJidGFBRJ1EesDtvfaeMF5\\\\\\\/y3N8Ol0hlR4Mk08NVBlHSV5IhEkFUJlRTUWlepcHpwYnykPnEUEam+h8cvy8MSbJppQu28vBc8Ke0a33BqbYsbFfNWiaIRqNqO1kBn97UluN7jkE2LkI25W7Pbqz6l+9RWEhkrKuBvRqDTp58n15FeXpVZYgyc1B1oLG46GeKVEPRNNcG8LyM1FayqrJPi7QuRRpNkETIISHy5JqD97YV4e8gYLMfK65hNM1fm8Tw5VXdkr2ZEEaU+GG9C7sDkGqerNxtLU1jYBS\\\\\\\/fC8HDyVccD\\\\\\\/DULua1EKMdjXhgBNJIRQ94xxiMjzhwhekzWYVRxL4RA3v7jWGLRsN\\\\\\\/fDDDbb32KaJtPPoEJtD1MTakwzDgzSRG9rMRxAhh52Zap44hmzP1x7VKNQVqyEinGHZZwn3nrWWc3RRMpNaFZdtfqxF5KDji8CIeHPUzOb7olwaIekAzmg0Y9v3NgCGlgBx60WACGBL94690oOw3gCaE9zOkh+FoXEYuTeDfMq2MksrbNWwaGVN+P7pO4\\\\\\\/RfHM\\\\\\\/0VbObQGk46SGhbOABBfd\\\\\\\/3S\\\\\\\/Eh1B6vIfn5QJxxI6MX0+2EKl8hTxbVI6Wd5GPsvjIZkCppIP+IdmPa4xG2jxo87qMDuRGQ4llJ1aHS4J32ddGVfi+7haBTirTzYGm98JSJ5KcYrUKLlm\\\\\\\/EgpFavTLUzTuEWxfmtrXgslBQdjmPFcrbTF4jUr109HnGZZqPaitXRpTH+BonogX3hCH1qfPcZ\\\\\\\/Rm7oCRaqGPbhbWpNPVnoEeIyXn1LK0P1IrdNUlhfCulQViF4BsFlBKypbcdSqMa1KJzd83SYOOIcp1gU+jDGftSvuAIv+qNK5CH0v\\\\\\\/FHFHwaKuDcQmEn52D2\\\\\\\/yMduwF+Hj9R0LTqmVp8xOud7X7rusdo2x+EbnJhPKGhV1ETUVg\\\\\\\/hiQFeHbH6q4q4CAaiDrgUvKJV0uOfdi2qxPTRfQUhThdn8DJm9kjfizLuGaXYEDcNc9TytjSQBXhdvMOwD7ZMGaxkC5P60u0+ZBn41yOQf7allsQRyuI10TeqBbxCrpuurUGLB3yMyMz3MSGplAORLRwkMC1Gm4aNxmOG4QUEOv27LY776zaZtuinfsMYM7Ka7qGeo3cywS80h9hre0Y0xm396ncYkpUtYyYex\\\\\\\/XhdoaMa5Kp\\\\\\\/qyAtHsisjEfaslm3QrE4qNlsX7BiP0h+ZtrGG\\\\\\\/3UW9Sn1Si3Vtgwy2BNV6W94MZIb11nyEKyTni+XBu\\\\\\\/BJ1kMsD+kmpuRRz72hHO0irPVrsS30FsvDacF8J2sSV\\\\\\\/QvNPGpaozxmqENefGs2LhkMzEXT0DQbX8HO1t75fdG9hfyfAMHDjv6tr+nTH35jDqhsEKqa+nrsLBjgsgwWTA1MstXzQ7j8OfwXathe\\\\\\\/XgmK6UI1gdpp7T7iO3qar+YP+lYi78uAX6yY+OPTpUtAn7llZXUO8FLVjsie8Oky\\\\\\\/EG\\\\\\\/8jLbuTDyH+GlKwSt69bYJ3FwmjN9ggIhP1aAV0kCf6mUU3eeLy3vHClk1Gi2ckhMRly5zyCojjW8iOtMq0+D6GuUDfcO9w7\\\\\\\/h1TDqHwdYRJHkpZnpYCJtwcJ03kL6BCYDxKCw6vlB8SBmT9Eu5lIUVKF0KrGwf3S07Uohb8NrFFn8+uBFulzq4dhnKmzL5Yrgkt8NFd52PuOS+IgGjKMRigBD3NSeF4HxFQ5A\\\\\\\/W6JS1StdrGXMYQsK\\\\\\\/GeX3B4rBds3jATNSQEfW6oksc7yIyfGxYEHukvi5rau3MXEmVM4OJlsykQ7PoBtB43WVKuSMwqfLBb6mKToF\\\\\\\/84Bm8pldRZITo6xCzfAS6ux24iG4SWT0eUb7NMPQwZHbcD1oKhyhH4mCDbyv3b\\\\\\\/Tc+MozB1RSwU5aAqsve\\\\\\\/KviVmSZx3ycTniHq\\\\\\\/b4HRB0gBUqRCDOYmO\\\\\\\/dUXV7gL7jWpd0IBAmLsM93MpZI6n+w\\\\\\\/F2Wi47YYbCPQTl51fLwSZWbTO6R+ToyH\\\\\\\/Q\\\\\\\/mx7GBJ7j4Eg\\\\\\\/79ADtvU23oEliaFBj8oDDy5BBts6UKvdDceKm0HX686GqivZ3kQY88V62LiTY14g713vyD0v9fpfSBdU9IWOCe9VemohGnz3nD\\\\\\\/rzPwXYTexSmIXOblBnBrt6bRzIXW6M5Yiemi35R7sBS1ZE66qCDU5tNCi7y678msR9cdSm1qVYtydOdcypdMtPfpYm2ikAIIVYVzeE7+EDIPEMvQdKQIgJ4krI7Al3xPLKdXk\\\\\\\/bvj7vS4YVJV1zMoLiaH9blk1JdgzMxLISXELGFGkhtbpvajCLY8ZpnRuQdy2qxcMPE3i7UC07Uo4AUEss1nIvybQYc44iWyftf1Tmu62k1x8uo7IMYYQxlJo4Kn\\\\\\\/IE2rxOyqC+W\\\\\\\/6\\\\\\\/SOmJLUzSQIfdJEAHaj8ypy4JDNK+pbF0KcuvET\\\\\\\/xRSRPF+cu9sk3RR9Ylu8Wf4N3+QwhbkGx+UOhi1Dnb8nyWpxt8g4yaMILyvnPIgR6t4r5bifmHvDcUIRCpKXUThmuTWH4cBbgQO+NSV7tY9mlbkpKv2wYEBYNnKn4A+n8NvwGD0qtuI2nj\\\\\\\/ZmYdhJIT7KPfR+MxJmYTaSw42\\\\\\\/8rzqoNi8Am3QCYCO0XvamS6cpCW4ofRSA+4+Ikz6h6X2QcJFy\\\\\\\/2rLxMCWFVeqto\\\\\\\/30bchMkRAfcPHsc7fud8MPMr0dnyAYLGHnHbMZyabSZYWDllYT1xczPV7P17bJqZd185r1NMmn4OT\\\\\\\/BUZBik4ifXHCFhj4D\\\\\\\/dDlYvfMPvvWLI9s8xNFfCr03xhziORVFjxku32T08pa8nFSzK+aKZ55EkMndElw5eiGv26n4ZJPzulxXCphV6uJRB2KwjScrKtB6hOXr7j+pwlPdTQowkPsUrldynGPQLHNeCK0+maZllOgsmXigwPosUWwi3YHRV9tHk50A5Xar74FiRUgs\\\\\\\/SWqHVQ\\\\\\\/B+rfP86qWX82pgdYDGWwxMk5wYaPUlr2\\\\\\\/HeGD7Z1zLcxf+Kb140+b9khT2J2NPBJhsehMy90ibV9ImKUWD6iZVsUsP1j6ApuaL4bbjeL\\\\\\\/orWaU5sLb5VLOZiNFfaTakv12vQh8dGXH5vgJyqO\\\\\\\/6GT+gkPPPGbDJMY49VgkwicEKeZ1s\\\\\\\/Mm3LMvcxMAsDrRlNU4L13LD60OL0Bty9qOzspgkW9Kkssnid5o\\\\\\\/ahGfx+ywliFgFsh2SzPwNsU\\\\\\\/AdhUlYvD5YzNjKJUI9k9KhvlCIyrVsTUkOs\\\\\\\/vG+sJCtsVP6xKoB1\\\\\\\/40S0KoNMRA66fu9lBfbtVzdv3nf4a6dDSwQk6mZLHf9fcxIU8FW2vfVim\\\\\\\/1DMdkDQNFocOjoIMK3MT4Ocu9fjrm09L6lQqSbaOnIwxv4B2gzT+qatAZofVnx0cgDRsFNNAjoNUfQOrbMa54oL5inXA339TKhbmpruM2bf2tuHgshXEjKN2xq6QcFFemsPJYT9sR\\\\\\\/lYLDuYki0TIyWp+TiKWyKdI1Y+htVTieFAcAM9N9Qa01AcxQVj\\\\\\\/havdmRZnbW4xN+Giljg7e8pEHUctiGRC8zhR\\\\\\\/xEDvx+H9tSxzEKOVCPPqIyCPk9LF5ue\\\\\\\/KU02odINFOoCwOQoB5FEfkBP3sQcA3nKCzFRYW\\\\\\\/jkU4FhOZes8hVOgDgGMiaPphE+9ooGABLM4gB8r6xmDfDg1aVPtZRtwgd4xPLVZrOpIDJ6ZTuj7lISNoNL0o4\\\\\\\/VqgcvwuKY0j2RHA6jyu5nB+uReKEI6wcaLFLs7aD5\\\\\\\/GkDMt314z078tsrsQ3kQqdwSSoeS08zk3r7u5rFxBbGpgUPHVgcxjTpnc73x1zH7\\\\\\\/ccjTsj6GxRN0ArIf8WjZjNmjHfOfulK2UWbHjEKj6sK9A02iWeuBneiozcMkyQkJplbrn31Px12dpakhZSFCh23SKGm8fv\\\\\\\/dKF5UoooO\\\\\\\/2al1ZhVWuOguwjXslWsOBzbyATb48lMW1Ss4Hf2PxmUyZcgy9npuVNs5ybzQ3LdPwDGUUqedAWH8Qf5UGO2HseyadLC2CpG3BWcZQHHlvfUo8ZvMXMjoh3GByex1BgpWrkUG71J5LlI58zgd+8oIReeDJugsepV3aUaQC7AGLH0diMF7\\\\\\\/+EuBP9X8l7yWcmsOxgqizVh05oVGv0eubQ0mhiUBN61HYA8QjlEhgemZ0CAKDjVPU55Mk3LG1UVkE5yo7bGrrsNuHf1emOGqyyjVvF8EeKeVJCtyCNZzcLzbuV0GOxG3Q5D7CPFpUAZsgxLXkfzBqff3sJ3zJWHkLcTAPQA+gpvOJ3K\\\\\\\/Sgwc6Z4ZWMj\\\\\\\/Nr27Q2wYrWPOu6XJhvfM4iF9kLRRJD7SeywpDVdFX1UpIVoy+nxfy5FJqolTl9v\\\\\\\/GimdVaJpNPaP3KM0UAqxRffnrG21ff8nhIyGLni9yGTO4wpYS6xDrjWQv9MEhieNNXWu3FloNH\\\\\\\/UBLAYOB6gYCsTYkmIBha5xmL5Cy\\\\\\\/fOj25L9\\\\\\\/KckQpLO3g4SsAHuaWov\\\\\\\/N9S0kfLTY\\\\\\\/pjSQwnjqcZUm\\\\\\\/VMElYM8gtuJRG+OqeSXQcQ0BYWeSgmJMx+W1li6EllYjPw0TYGmlYmV5\\\\\\\/jLSLXMZpt+4Dw8qS5vyQlIPyntoHZAn5GMT6dtDIMgRRSUpIU5UWZkd+4NC3Y7dRducLMaF2vHrdVdJGHxmCQzgZ6tYImYBfboNvkWyRp+FcMm2VhGr6AhqHdAIM3HALZJNkcg6Rn8UXql51Kx6wTU0Dn9TcaGgkpKs3ePKae5pk41ihqVj7M\\\\\\\/FEW\\\\\\\/kpVaD7cgjdTDK8AYVg5K2zGAn8GX5Y3MmuUWLRL3pl0KLLC6XWrP\\\\\\\/Fv8zcpdsj2+5egpaPtWuWe7cCVxhr3zfR\\\\\\\/sEsgJCioSk9k\\\\\\\/jlO6o1Xw7CLslrDez3JKGmUGNcbYqfsd9Z85pVKbYeqaG589r1gj+kMO49HXxRk0Jb5QCQB1wHXkh7jbJNnjvm3cJPZi5D0BlNhE9ITvOTVwGebI1rqu9XwtI9aU2kA6t+ozAy4ZHUjDNsW44Coz2tEGCoIzsKnMczPrDVbuVj71Tsul+LFJOWT7bM\\\\\\\/v5Ena5jiDlfmcDP044z0dtEgUgvowq6NVLIzzdXzHkiG62k5CGWLUzknvNPiHgCOP5C2sw05KgX50wJMdITiVCETnnk3zqwxboJaRIREcRLTXt9ZN+7HYoCQPV7CXoxDjVEb7J+5JxKsa1SaDkOQOiLA6jg\\\\\\\/Zra66UG6C+O8ddOuQ+zJVXtpGUyphmoBKLxznK4v5N2QLam+\\\\\\\/\\\\\\\/BI84Zu5B0b8MASmrkotqmaql\\\\\\\/uraMHt2hOWNBn9SG4fyu6KMU9RpV\\\\\\\/QRGM383ez0yNj18ZyGWUFLg+EwMou+eGb1YkN9oWu+t34M8Yp+wrjlL4ninD7ZQS835\\\\\\\/My4Bkm1qW9u4ClOjMcjuiwBkVN8+kT5tTB1OaC6syECoCpEL3RjsuNVI\\\\\\\/wX0rEGLO05TmmkzWDAAGaale2zWfbmR5EC26GiMFMH+v\\\\\\\/JZnS52AOkmuKnYcd9u3T49fKWKO+zCmrZ61x6Nu4GOcoXDEPQ5IJeYxs75hCRTUC1gFUqw+p3Am6oMEBPRGia6hvqDA0TNahH1hgP1EkM0n9djHJ1UydpISZIXDcnv4hO8IsQ+DcHfskdAbbpgvmwOLaWYQoEIscfRTlCiqebD9cmWC5gJ138qli0Cu7csLhxrrGGPPuCfCIZRlDoo7wQxfS0aQqX3Lix2xd3kOZBOAwvnvZNm1buAQOWnGmyqqTjPYYm7RZX74McdOK6krm7BtM7nqbIwa50WNn0F0+zKWeGrgoPx9vi7bwyBppTSD3K8Ns9TWLUJeBaqr8jmrtpmUi8ONXMYHxsz0u+Z\\\\\\\/yeEMRPHRHoiaKEMd3\\\\\\\/JebVP4vwUIgm1X2nmKkalQbLiq1RBGZ2PSHNGrs6pY0PQIpONFFnV3XEry4cZ0EdWH2X0rNNLxyI8sA5aHgZ8JE19GBqhZ1iKomMFV7hCdBznMixzaQpEGlZeUwADH297nhVT+noJ5gPFY65Gx0MmGMlt3dQanOGzlAXb\\\\\\\/rLNzdDJoEjj7lVmYS0NrPsXrW\\\\\\\/atEK+6N0decuZOr0R9EYmami7VTVSWNErTd1Vm1qaM2XpTqVNowcytTNLA\\\\\\\/5JOGttyVVniCaz7AUaGv5ZDSOdmvdzy45u0I8MbTzMavxiJhkMEauHEdaDg3oGOwoVNXo6qRBCnI0ybqOnRpEg6x\\\\\\\/VUAS8IZtjbWfpZcqrx4LqdLPecMd9vUNvOoKF42cJWUeCvAPeiFHhC6+TVq9jEjhP\\\\\\\/cAkDV3h6U788He9DZdi4\\\\\\\/bQk0WowJLGyg3BpazH7Tma7BL5Esniz33cdqUkMALExxCn3vR\\\\\\\/kdgU3CViAmgNvJyE+fik+P6Mu6\\\\\\\/LQ59uDfOWYMigIkQ5T91uxmrf7XJH6axcVs0841it4\\\\\\\/99MioekIQ+eaC2EyVBSjmpfFAiFtr1q5tw+8oS8BWDKbuwj3JOjcv1MNAh7+j08a5wREssT224rMBcpPsRXGX9GrYxrhp4Ag0TXRTVsSxlT5esTrf8uP+oVEkNsaNWLhEl47icRov3M\\\\\\\/Xg9SPPhrXG0oL6UsSGuzwpqnaj4lkNCfJKkxsKGJkEK2npUYjzjvV8eyVBeSMsbOKWC0Gsl+RhWcjnGN2lVFuNAEODC2W+scL66U0rkX1n1tBrx8SuW3shhRMprJL152v3F++Msj2+Xun0aNXmPnxm0Ev0ZOT4+bjkszl9tKzJIiL3GY20qinrfqGXIcbC6YI+d3m123lgz\\\\\\\/K011b0BObNwMH199EQKIDg+9+0SOypT\\\\\\\/fFku7w\\\\\\\/WeW7JP+WfBTeU79JFt3MIAzZKIF+g7Z6p\\\\\\\/xS9WAPG65UU7OvMGtv9ZjCcqKNbk5CJJnpfYWRhkkw36Uxdo16TkYiVUJRS2FiXdi1gc6erVnJJSj0XOiBr+Mc87f2JmvFvB4\\\\\\\/zKVClrKX+X8upluxxvaNVvIYU635tFx6TAXFcyMDlQBWXLvUY2Pwsfj7R5STEcW4\\\\\\\/2FL6yq2XtmKf5m9MEGnlBbYlMcwJvLCGhWZkKTynsrzYHtjanHNI0qTWhrh0eHcf1CGr3g2mj30EcI+sa2ogUotScm3qGh56QjhAdfG6lMxAPxQCdcjK4UnkH30jXgDE0yO+5dEhiave\\\\\\\/6QM0tvpOXi9nbf5X6A7g86qZYckqHCT176rAFYHQbwTRTSPYhhdlcAlLROX0E6bupKixpE4YCCUzkvDBBR3OcsGu0IkEt93OQXI5A9SyOoLs5MlIII2gLTf9bJmMqcwVjgeuY\\\\\\\/EXLZ2g9\\\\\\\/K8DMuUpV7LPuSto7aovux2sJNsV8K3Mj3uT3riwJLOtlft75FgfJA8Z+CxocGbCv8Ws\\\\\\\/434+nWLmzAoQ0WQUyjFQ+hPFlHKl+qCxxWVYbyvL3GHLOqBGlRxrQmWw5XHhoDDKgiXofm674UkJHgM56pj8bQEwx5v7iMDumDBZUA6dLWmtsiOcNCio5yWIblXhO3GXZn2jEN58fVKwBOGBbc1xo3A2Fexby95MmsvBNbMe9qDh1tqElYBblH37SlnQizYY+B5PROKWkPswKRTYZOGSt46+SRNNMhmQWvCJmdvFDJIoREOVr2C1JIEZK8rwXB0X1vMnyMBNEjGqMN4ab3M8D4LJy8LVGNduy046X7ySkFaVJh6z7IqVZ6xp522BD+POw6Zy7d4aN+t9cCTHXScLV5eo\\\\\\\/b943pUD4nrtW+4BQ1o4GcPD+mNubJBEKyhag3CxmsbnB7m3blulc0puNOrAie5XBsZZnaCjSbVDZwPPAmQ9b+aahJaujZ1jcMotxxDce4h2ESuJEK16IB7CHwE+Fi5j5BwvYZvWU0qyv0XKpYrs8g8tzo16Axu5xehUBfjX3vGB5dM1OT9fRhkIw3DCeUR+LfuW5wf2gHGUh42FdFUaXJYfWKSxTooOFYlLEJhZmUzt\\\\\\\/ZjoWjYDUdu4HHGTyM28Ys4BDSZhAxk8w80DdjaPL2\\\\\\\/CMSq2\\\\\\\/bkNRg5p1NRvcDkXBYu54oEXNYdY11+aeAsDlYmzuc6992voeTrgWKKCezyGb2WakA3l2QjPkCInVKvpT2Erj6bARNCxchWqWI1VTmRvaHDgvV+SYFRAphBR0mvsdkUN6Z5b8hVeuEWqQgNgAjtGI4kuXuie7XeznhsIPFpJoK+7INoG08lBWiYMksnmUIVKLWo9Cz0xQPScZfwUWJRx1H5fuoC0mhTjwNDab6HHrKM6M6X1D+EtxOuZ7STOPFEa6rdfv6zhDWALHMVMFsThGLHihTSlca5NiKdbY1RPQKh6UKrOBxRGp\\\\\\\/aiVWU8ODzXN6jTsTwz\\\\\\\/FlfVRHOux38IS2cFrEG3X9O1opdAAik2FbKCcROXhUPj29JS5VL\\\\\\\/lfz57tms6NfpnpUIfJ11ciLA9hVQZF+kvKJEhwwFHJsq0IcWsmET9v8G6wT5RnMAHnzehHxSSyaPrZQxLzp7U8Z5hHjCrCvU6NHDeG5yJzVu\\\\\\\/kBau3s0YeQLQFds1SELgTyBfBMhtwUyAP2W86tMSXq8MHAHqZxTOOuqerYE+yoPh3EneK+dtoFnhOuiQoCR4dHuYbWIU1WTJUT4dRHoZTc22yeAYbmxCMvuz1afm827LphSSYqEIvXTz0B58QW1v9AaE2bQ4TCIrtsJDcRBgtJpTOLwy6h0jw+qwXbfFF2EqY6xlMf5VaupPxygbu87geWmHVTcS\\\\\\\/D+0mI3i0rfzWSWLsNc5xuNgVWfiiasXljRkj\\\\\\\/Ozf\\\\\\\/XCKLXOU+V9Fwlu9XJlG6kw8m\\\\\\\/yif4LxTFzmmwKHjvD5tt\\\\\\\/HsEactvch2dtv9Y4+\\\\\\\/QtwXcIsgeUB5UdSKaioDH\\\\\\\/NxkfPxnEvBSAy+taItOXhBv94EBZCkKocj+wPOXI7POtpur0NrZnF\\\\\\\/qP3fnOVL0vCgH5ba50K7Sr63LiCAz02AqRkvUgSt4oDi0PFtCHydluabmnSxEwVOmrUqh4892Ol0vae5gYd+TPEBKH9wb5289ykCxHIW+doZnLx7J\\\\\\\/uLcgMMtoXyQlzlhB3DHUAwgswn9Pl2l5XBCKbnQqbVAOkS2rkBUh8JPlNWBq4WET9cfmO7hReQyoM50nMp79rHacDXTXR7d1bgADXCUN0MCwg28lgrCswvq4RMsaGFL0m0odMZHXlXDlb0tB0JZo5oWePCuazYK4PpAUSdpdwawBRRYo95gQ3r69OYrSCfe\\\\\\\/KHhtYpkMJ+8IkDzfJwCLH4\\\\\\\/ha9xr7nmRMEgWConLNh3DQw2sWx0BaMWnzDsNiDdZ4QzCjSIrYMNJPsd8+xPD0e0fyfFN5BCPshXg7PWvczuvz0OQYzu9r\\\\\\\/Qy74jOAcxXAAtU32NraegJenXE8Y+3GljY34OwO1WjyAzqbkiYQABFjl6t466ox6l8TG9rOeNhxUF\\\\\\\/0s5bGbfa3NOjSu0KMRQrsevV3kCLHnOWfL\\\\\\\/Szj8qFLRIMLxEpbq02L4TLDyiWQLyk2DnP0FKRX3qBOSZVHOI2blLq5Kl9y9S\\\\\\\/LSF7cXAeO3YCqlpXfPVSCwaLIhgiJq8TNyizc1I2cc1nUWX3HhDBkiP7xRyQiTB8Q3oosen2EKRF9VQkccYzUKUKTdQWTHKTfFmKwkTPLan0doIikhQga\\\\\\\/gKUZm+AQ0GkWEp2mel+WZtVgZ+zVVyQXGUrVgCaPMWdQ5h3KT0Vsfipm6PjkB8+IYTQfyb1yXmXuYDUkGrSy2OIm+3\\\\\\\/tM\\\\\\\/sUIPVD0bbuoZHxWaRIoiIukvg4Ugn5s91TlsdHEW3WgNPqdrim71JxyFByOC21O60xcfr+r7SMmyAcbsV88\\\\\\\/8KenUeX5BNG2mkPVo9My5ZoFhaOsIygzhxNpY7cKq8p0aiNs3+SKQOk5QXfQQLxTgEgq9VmYezYTYRef+N5hMaVuoQiQ43MAXe56xm+OQnPu1n5\\\\\\\/\\\\\\\/Ewyv0w591vt++JUO8KWwTloUjtf1FfKlogUXQLHapVU6u3D9hLMS6wAmHRagYuVV+KcORvwqfbI1vrR887PVYSP81sZOEVgWmgN4aOR7YkFgGyaL7QjYjs+HeBHFNstNfWZMT6vhxKuLcP3pc98o4A6hOxUx9ROkrMx\\\\\\\/NNbgei6M+b58C7FqUCV1A7AS21B8znq\\\\\\\/Yb1RJUGFflHMyv9qVPV6CKtWGzJooZSGQ6lMb4PsSkLResUKHuyeer1i6kObtEq+Xogww\\\\\\\/ysgPKmVD6DoTRoo6uT2LpIR+qwFysYRDlL0LYpRLut55ucrisHt4WLUKAP+yZnVJvFOn1tYNpoy4SLEUrI7IjP+3OJplaaJojET7y6HI9yZYPlVEFgkM5OMx3\\\\\\\/2FBlUcVGedn0rtB1ZvfQO6nisBwKbblv3fomm+CBPEliBUwMuaUfNW29ecEHmdzzOqRPUcOOYCJ00qjgLZl9Dc5YIDBBoVuk\\\\\\\/MRHcMUggnnT\\\\\\\/0EI8peo6t9ybaRl\\\\\\\/1icFYsqnTM+hxdYrecqxjQWh\\\\\\\/IXRMwkB99QQHbv8ZbiU0hEN+EBcrZyUVbRIijf36Z4TbS9eO+Vr0sN4NDYpmE7L4\\\\\\\/fVmGGjxDBmcFS0x2DvWIdJG1OmL9xEiS86\\\\\\\/JwznIv4iJ+E45QjY7qJ\\\\\\\/EPhUgegQjil\\\\\\\/vtHFTzpGaR9OpX+97WigK3ZidhFS6JbBbYOKIzBQc5XC58XZc3WlaLsFDJ+OktCqACEtiRy5TWr6Z2YIEXhkzGjZaRZrYzHA3WdPRG8WvAPeAN621ahW\\\\\\\/Xag7kgc2v0cea0llwWblegFYz0B6zB55M7Km01LNIw2C46Y49rgXgmHPnW4+OcCGZglkvpQCisYoOM+jyVe7MT7WOMOcdWiw+qQwCZGQLvV7TmmYNAUeW2bYtyo5xLG3DCXIEvQ2evEC1yx5joXzu8ZdQmdvRyppSg7RPx6rF4BepQcsvdzMygKiDByfVNRAuTohV8h6XozJfuEyoIqp7Sth5WpMuW0iWMhanYKfOcO3\\\\\\\/xscsUZD23mHBpPgl1wdERw3N7+sWXHLtR\\\\\\\/ueu8W\\\\\\\/EIxJ51swnxKoWeXz9QvAXwMd7Lg6BBI80ljZwPvPyrKXSiXwJRmnHSXN9OAQGw6TSDE97QylBhDq9iBLy+pTw0ICSqk7kukPPHZ46+rUCy1sitZPJkpcJ13+qIs\\\\\\\/uzGl11PzR0XK2+Cusu1+aHIHE25V0jmMOqEODOLAKxZLwrI5eMRaSbEWa3dhxB9Lt0vXyzsMGOGEnMgAus2DgGHjcJfFlb9C1XU3ybYfQW\\\\\\\/r5y4wMOj\\\\\\\/oy2Yse1HK+zP\\\\\\\/1q2IXMJdnXHcj2x8HkqYPdfZk\\\\\\\/IIopIFvNtAyMQoeBbjR\\\\\\\/6Ee0KHXhPq7SCa+SUR5HtgFUpMG35rlHaTrfgE+2kSbeVMXlNJEkinxVA5Ysjl6ZsALaEcFA0m6wBxZG+V4IN7eDtnoXfqNdB+v1UYyTgbAnaO24\\\\\\\/c0wcGGFFUq59zC5\\\\\\\/3ejilGG7reiT4lXnpHHQduldscsK3DUeXZMtTZe\\\\\\\/ROXPGgtfgdoMoLdQUBtMZ+NQrgCokwPAnPtT9GfMzB5k1gg0HVx1rY\\\\\\\/\\\\\\\/QOT4KJrJA3A1Yj98MxQdKnmikqPFVM7jp+txPnGwCu1WDcDTDha3DkRu\\\\\\\/ot4iMiQCZs9Afy1ODl7CWIJ9j9bR9fZfzR\\\\\\\/+hjEielTJ2ApjFV1\\\\\\\/uL4Tl9STcATXfckcgQE5p\\\\\\\/eYrP+FIfQ0MJM1\\\\\\\/zc2CI\\\\\\\/iFkoi88E7UrvGoQRnzHw54SOlftk\\\\\\\/vTg7zvGcXLrvjx7yMGFmIWPdoONISZNwm42xkmkDun3M8fE+lryH863i2iuYJvNfuDuEi7GO2qit0gjI0sup4QIEveu0gL\\\\\\\/MzDE\\\\\\\/E0W8ynb7iXyAb7hS4yNJpkM2J3Y9mUOGJaArRuT+mw7BamKOCJO5CvUrUglwhwpe+Bke4\\\\\\\/607ur0nTFU+PWW28KhbHiav+TD14ijpslaHKmpy+D4wGwawiWAK91hFOJSbs8J+rQtL1BY1bx89luCxt9fPyIl8GXq5W0z1LdR+28uEsPbgHgveriAtkc\\\\\\\/FVItiH52bDEvkZVg\\\\\\\/PJFej5jbp5bc3GDg4ILiWSvfUoOhJ+c\\\\\\\/0UlVhsiD3vRCFmM3HFL5RExLuX0Tq02In0zaklxW2HkBXmYxCcyKJXwiym76D3Jxov4L0yEHD6x7njQ6PQ\\\\\\\/TMtMT+QbFtXDV8auI7OsECwrQkb6feVwYsTcacyrG7XyhbPGHmeIc6KflMZzmI7rDFQ2CmQ8eh1hNvqtVgRxh1HCbmCBouQVEhEIHkllTlD0E3eZWLUUnYEjeAmcJ5xiUHhIqLrPbLrwfZ3Gj+l8YP9+6veG1VAmWvcmBBUMRivxvjKIFjPB01W4\\\\\\\/\\\\\\\/ikg2yGtZlsj5InNeDkdmtoAawMXQg\\\\\\\/wxKjOvlHPepdonpf6WSQA9uVLi\\\\\\\/4qB4fEXKkq0CbomDvciwlZ7xVzAbiucJbtdbCURt\\\\\\\/r+uBWjqYAdzNHJgdiGMM8w9VP2Uro7xakJiIboCwzfeVBYdNjTPH\\\\\\\/SvWsJpB7ZqehnlwBvchAcU+e5DtwqIAAlZrbtIgooJLpBAOHlGtXyajOzuIWnz9BRdya\\\\\\\/otzXrYQOUaicBpIuEhRkkptbbbnCY2+l\\\\\\\/Y4hleNU1qCVimpYbPioYq1BG\\\\\\\/MVyAy0Pa7LoY1VvCP\\\\\\\/pF0syJjUtK3dS6pEFGATMPIlKPtkN52Urey81+CoImSJgoWLxjj6pzWcLvpjn5rfeEjgLsm5aRyp+MEDeVabGNycrbOI4Gc4597iKrkYE3sE5\\\\\\\/eYC9gLaKw+iW36rK1tCFnY8IZRTpGL65HNUqPMlKKqkDHTjOUuJk\\\\\\\/JrmyrFpr0lxou6yPIQcyCrRMnKFovCZPAxcG0ny\\\\\\\/ZP6+ob\\\\\\\/il9Rpj2EPPnMtur+Cc+DxqsgT1xGIgCgc1amlYAiGvnekmvhuAixBA3oUjFGyda0QNKqTxPfp4SZN3j1C2h6Q5OzKjm0\\\\\\\/k3Ysh0MI\\\\\\\/jVheh9mq7MQznsjZEu\\\\\\\/pCVddPlniyoIAIN+xozzx2zyRpEhnhbvLwQRb7OP4wZGy0wTTDVjw0muKuFoll4NjDKTo9Cx2TwBHA67\\\\\\\/oWSRBBm9D1BaOp\\\\\\\/t2+m+eaC0M6XvybS4p5C5E\\\\\\\/sNA6zqizxSG3eJDdIMtAME2UkAmJp8YJW5rGdpgBVjU3fw5Ctv2rg6eqDICeFs3Mx+AOtc6d6t1dYi\\\\\\\/1GmI7xuvtFD0LEAJQBpIoeQuFBC8RGSqdP535UnpcSJr3Lw4T8nHdKMfbg0D\\\\\\\/DuD4Do7dUUo3paHHTH1DjnLby0mb6XuXUkb2fa5rdW7MC31hwiTorsqO3MYl7H4mGKz+laoEDcfWpek8DqUapegk8wXIUsDggWStjKwAs2RpnNo1\\\\\\\/HfZYB72OCkDp2SJOPGvUg5JZW7BZv2qhM9z90+5eCX82CZRUt7i5BDz6\\\\\\\/15t1G0WP27bwY6MD\\\\\\\/1mf5Zrf5LU\\\\\\\/lidee7wfCXu+VmUbfUxpqOoi0WMtOhNajsdEzhd1RLPikGyZcVQ8WzlYv8utc2CCip4b2Uf\\\\\\\/rVO9ujxtI0ETz0LpqeXVy\\\\\\\/YHEZGZhSqGeKWiixytUe+5ggvhMvk\\\\\\\/dS7GBkRAqwfV4Becmckfhu6v3\\\\\\\/ESGroIR\\\\\\\/Urw4OCz99WU51qqCfhD620psQA4T7GAUuVtZLpNvOnxcgi6oBK8hxC9gxQ\\\\\\\/0X7bZYJt+5d66+xNFP44fBWCYEXjVKGLHwN\\\\\\\/911mk12s+9wu8j8ZCvtIeLYYFXxfO\\\\\\\/2Poha9w5flV+J2hG+rhWilTV3eeKAQFSjeLjF3mcD\\\\\\\/9N1tFWzRuFcLaLMYbrC8NR\\\\\\\/2OpaE2QQIWmirONm3qDvuvQJhO\\\\\\\/q+P2aSypW0cXsyzuHtyc98AeIPQFGC6GRglgn1ORicorxZBbzHKUDYTGukvG9rfLXcCvhFQ6Pahk+24ZlSD3LJzSC45pk8+QClGgmMAUiqmfTNFNDVfdt34aFdRvKDG98Z7ZoY5tdHKZyWi9zovYMwUB4y63EEK5yHU9JrkJNml2B8QOejihxED9IhZGoWWsjcNeQpZM0xGBT2bB0b7ud9hkG5C1OoaK+8+JuJWfDwv85wPbkSoVWMuV5a+MUct8JwJPyls+p53uCTy7UcVPO850Qp2LnkO8\\\\\\\/HxwSb3S\\\\\\\/4wlZwYUB7D5QQB\\\\\\\/U5SE34iIlwnpPejmgoBYksGV792CU02BAUw+aPgeSlNano1Cv\\\\\\\/I6DoxEl5CMjcwv3UbAUuBu1wE8aW5TD4nuwxXTJmhSi81yOo6LfAC3ocOsRDFl7xS9krDdK22WYQ0C5ZnEN5UReNAgdt0POvcjh6hoYLgHPKN9HZd0mihUIZMOUO1Bm\\\\\\\/f\\\\\\\/\\\\\\\/kDCLCPn5on6Rgx4j2dd4lBMLoYHjEhN+2eAjv07m1dwkbi3A4CQnlTl3x5e5Ri4Sicshl51PoF4rungGQEeLqEd1hIAzcs93FuBKuWD36iVGHK9cFgqe2Z\\\\\\\/91\\\\\\\/elTLGpw5sSOIysPCChjTqsWawHXPJwIEMu5r99nJdG4iAcRIe+PLWu+Et2WB\\\\\\\/Tg9ER3535IyzHot2W0EY5A+7fqKqrxbz6CvaT4y\\\\\\\/rmrbb78r37u\\\\\\\/Ulf53Anh5tp4eCUaZ\\\\\\\/tn7QC6uqgKKkInabq6e7yPXxx2ZTjN\\\\\\\/PuviI760UfMZ+Jcoq73z7PhQ8mpKyUd+azy3eEmoZB32cFmGmX+5s9fkIKlnnXCVceuMN5XwefRiUS6rLr2TSy2+xt3HqgK2yNz13xtImsyc+9Kgw4WETgBNbAZdUcj7kltpZ7PmoIwjdDMfs3mLGVUGqrp+ze411Z+Fg5VCDQlZKErXqaHoqJfY\\\\\\\/C4KCFPrgvOIP4ixPJBAR4H5kmbmWaYDQ891\\\\\\\/CCrDDhasqmJjBWaoLaoKyxREV97gEAsuDkAicQs1kkNfDjyhZWJBRzS45s90tMWQFfs8bVkKkmpr3kosIr8ouAn1Vxw6y3htHbwS4CeTrJkWtEwmjftcXlPgVDO9F+qFDoFRxOOMiUvNhuXeFU3GjmIU9+5H8dIcbuC6+lJHJ7cit043SGIKFXSHNtpDvcwMNwU9OfZQ2jgqipwxvUFD7NGuX3Unklqp8BdrYzMUIGQQsvJvgDFchVmjSFdXkGXmsYAwg9bM\\\\\\\/5wfF5Q4e71H40uqGqXYiBeCzTkiP5sdhRmoDcUulpGScRF6oaH5u5yiMrQw+xtZca\\\\\\\/i2U\\\\\\\/YhIeRK9nGzcfbjAXtNww48eY1KuL4PomutrMeJqBphDU9XDnOL85IpdEkXMUWK1LyAqvHeOyos6XBjJfjK7gi37zzxCoJzOlPGBI2CUxGxdlvXklEleLrlfqx9qe\\\\\\\/ZJHxnktZS3H0kywwZcPSg7RC8eZeDRaXFG4JrhzehSYGveyY1aboZlnrhqGPa7ITCHlkgV7eq+hkPVE7pzuuE2PYQ3Mda7eoSvK2RW0eOPrJX33aoeSz9Dt93TgjLNtih3KV\\\\\\\/QEis2Ugu1JTKRB8E+dtVeEK01B2rSBvyobJH4MTdSqZ0LuL2vKd6ZNR9BcWRX1CAh98PZrOsFqaz8Cl93dh1ONB\\\\\\\/3EGsmhX541\\\\\\\/HkTZOJuO0l8kZq00yYdWjHg4A24efV1\\\\\\\/y\\\\\\\/VFARvaNLx4kNLnJoXQu17S0717K7b0B6D101Y+o+9hOPQdFp+YvHFKRyLTR8TfEwuf8PBCexNUjBL15rmqdla1v68efAAeCI\\\\\\\/AMwgMg\\\\\\\/Vbyq+Mh7UoiUGlI1GY3vkyEhrgBC7CzNYM9XiDugBw3\\\\\\\/Ebnhs8WhdXchrqpz0HYr\\\\\\\/B64MaWt1cc3WqBtP\\\\\\\/AqsTJdpD6HC7LVFCDAYaUaxaQkI0a1VF9wjNlXbsFSDNJFt+B0Hny0XAk9kt6I9QHO\\\\\\\/DsC0gOEnUjb7V0ZxhcDnYF2g7jLEUrS6Qu+km0yNcwI8HEq+uYhtbMSZEC46aOlmq3\\\\\\\/ZWmNI5gxezEmETftMp\\\\\\\/iJHOLSoZtQGgbozqVF0LsLyriYzyrJdc0IIEp8Jf8PAXFnfcqgT4vq5LBCZXOt4KIuSMTNUFz2zt07gGqQo+iLKWEsLi01fOq8j4T96iLNs83FYZz5c\\\\\\\/lqo3LyZMIIDPC9ngZIrp5S6dCKS26RbpE65ANC0+tzPk\\\\\\\/nKltV4uQM+B+joXMfJ2ODP1QkALiRax81LOPDTeMEjomgp8MW4KJklExdWyq71Pq265E98UILwJqqafRJ1rGpMuBJM\\\\\\\/2Jxic0E5nr0LhGKrEhkreeQgLK6t4ekdLxEUEPXdOmBxL7N1TErMW8XBthY625vieMKVGgp2I4a3+fBXvjcobM0Cv6t73NTC05mSHgE3CWA+wLnYF+IFUZch0FDKprWWmkmA5ekzto8Z6ouBzSUEXI95cLDx1yC8iXitc1+b04He2DwoXHmNGAq5ZaiQRUaziVUWj+xY99YvplMMnMSYT6tc55FvZqKNybr8XOT\\\\\\\/WFJJyZ4ARzwcbN1eZvAXzgSFQ2UXxiyv1F6p0J\\\\\\\/bpywnFfo00e\\\\\\\/qOxYBPI48LsovmAWhgjSbkpTn4k2NjRXzLpiknfFZk+Z7XzTjXL944q64f+L4fDSRWoWlKL9XlPKPqEfrNfm\\\\\\\/LytpXSK8j4aksEZ6ypX40wzC7vmrJNHtXZSfjY9VliM+2wpM3gf0l\\\\\\\/PrJrC6OvswgdwXKxRjhSr2cb0EXU\\\\\\\/u0q1QTdcYj1bvX\\\\\\\/fJ2syPPlFPEFDACOEdV8C1trTogSVVRS0SqBAAjQeb4sKHYHB577LcEQExSbuU1w4ts2IDg1KmvDyyak5fzl1Jc5Its2nlnDHEHA73WYgdeWcbuYtJAWP7IypI9HAdcgbKyx2EneX2n\\\\\\\/LyWOsxohnQOY87vZ17MEIy\\\\\\\/8CEe+CflvBWDNA4sX3Yxsd\\\\\\\/g0jPleP86JMASlJFo8FRm\\\\\\\/0fsdHhw3mOV6wzCdAkhz2yq33hcZueoUB0pb7kU9HCu2p75LGhG3lKT8U6KJaP6Kaot5ytLG04pOxPBvLK5+NkO1VnW183PXO\\\\\\\/7b880NfSUMBZ51Hs0\\\\\\\/8rAkhZPaP+aQRtcWEc2p+GLpdRub1Rcbrw\\\\\\\/dli+Cxel2EGxJsZNSwTSQ1e62JgBA2pAPZDB2jL2FX6Jp2QCHnkGM6Tx4YVUqHHx8q1+Dw3a8nIfi+YbPGs\\\\\\\/ZmmMQkjVjxS4rFyHeATnVxNv4SpdDLo8eHfDEOsoNtyJLWoWZLxNEsycgxcibeMKUF8tyTvCrQLu\\\\\\\/aZde1uYiPchlTKvKFjHXtVKhKp0GZzgoGok0frxwyq1YnxakU0EfQag6tIIWuWcbsDW5qsPS2m\\\\\\\/VnQVq7w2lNVHgUrsso9s0rJMhLPgPOF7BZoukTdEmPhNX4oMzenll9ggheV4qkVLri4BHgpBS7ReWs3+TE1b1CDaIcvrlQAgb7qQ2ekvfnvbEiDMIXF5L0WnojO5eSO6lRYimb3WusA8xLiPi0KQNaFjEccsXB08pnv0IBbE0Ipr5pF1v7L0DhoCfUa4dvRVP5wQUOCLj879RRvmvSEoBwaZSx0D7i6YPQMWeh5Pb65navFfTSy47VuFStECjxysPeEN7RpDOKlUiurzdy\\\\\\\/11ZxpLcimj6r0aao5XuQX0ftmEeE3vaxgm2a1BDVcdaRqMd2ia\\\\\\\/zNcibBXZOujQkdUTCyNRDJ\\\\\\\/aG7lZXGDqADxR+4ydN9TZ6hXRD7agtbEanRGxAkLP2y3JjX1x+2SP8PKdBLsiU7LXYKRiE9hiTj1bOYzLOL3Gzmt+93qJJZHd7yksYGHRs\\\\\\\/cddngQlIR\\\\\\\/JfGZZrWmY1EVY9UfT0tT7A8VcxgoS4YwnRRHd3OlFQsP451ejNv19\\\\\\\/JurHLaUbl0Wbe1GJ5KHLp\\\\\\\/OE1xk4OrEektXyuMaO4iXqHpTb4eCKAGSB6U4uGTQfsC\\\\\\\/lylMKafP8lIOFGN5rgTeoAUIUHlxAUz596JauqbstEpeWbanl1WM8WuYLdv0wSNf6+FYgPu0RvB0LfFzkda0RgcZT3qkTYU0xE1G\\\\\\\/drtncQr23rnBqUN\\\\\\\/nhr1TX8a2HsEQpIQ1ru5FP3HA6yoJ0FxsKE1EaW9XPKEicoUMVdwCHcsfPSgghEnFeCeLxYmVY5AQN8WDzPfMB2h2c4ZJ5ZXkoCVV1dhHAz9QnMB36+AFjFn5Rci7w0qNsyeHVnDhsxw8iTT4\\\\\\\/uW6OzuVWEzp+ecmDMKUD3T33hqGaRSmpOBnG+oZY2GwD8uXkXwE1hYV44vsxEgINWG\\\\\\\/XjQNetfGl4WTCMYVtvvZ4R9ItBzqKG+Gaz1BaGp+mhrTSLtRAMzBgJwsAQF80JpXDztxDYNx78oe+UIKcHIMBbzQ7c6VCBdDxeTDjCHB3IWTxmTKvzwD1Jt6srg6fdgTGfQiZ7a+9iQYzlDFQXu\\\\\\\/zJpeI\\\\\\\/NZnzL+uKXLOqcnoIVhm0cg0e\\\\\\\/exyFY9Y3XERV5BOPgj3o5vGEgkfItR3g8U1HUQ3tRJu5R\\\\\\\/\\\\\\\/2XFjYwZnMaVi+6vD1HHWBIPGWYe51r19Wkzsbc1\\\\\\\/Nz5Yrzr0\\\\\\\/mCRcKBl9y4powg9wa9d1UqJ0zgmJc6E0dBWi0puGbD\\\\\\\/3uI\\\\\\\/iS9P2JzhVuMq5oqaN1HmsvGKuQ\\\\\\\/FtOQO43b9MUk6ZhzMSKu+Zhqxob+\\\\\\\/IGlsMEV3h\\\\\\\/j+P47CAqRQDFk6orXJ3tRvGm9ggyNgLcdvT1rDbUedER0NBtMgsydf9czjZXCT1nRWwxsE8PCyg9vGBPjkGSgGN37e9Az8MmvczVoVxm9bomARdxcLnoh\\\\\\\/RQ2isMzsbHC5BH642m1HQZ9vjOpxw8JnZFqvb3kgTxDu6I23Zfxvta1OsG1K48Js0eP+ONEdq5woulcK9OkTdIRDocwO\\\\\\\/nQcPRpyZ4qz7w3IpkiGEbT5hz3OpBjExmDpwBgnPTMvhAWwp381r9LJXqgfL4JZzD9w+Kdq5TSU999r6WQ4X76SXrVVY0R8empmCE1n72AY+6wn6LCyvglN+rcpKLuABECBpWowWCgcK0Za78S+vp5jrq2eaMT+XYncvDiD0yKR5OkBAYLkNTxgzTvdMqg9b8pVEzBlai\\\\\\\/\\\\\\\/dCAWBiXJGur2R8bOOyDaQjfyrCZdiB4w2qmHc5yNo+2+0ZxvD4dcIfXjCjzoejZaxSkCRR6DaI2tzU5KqbbG42Mf\\\\\\\/DPEzHxytlkzTTnLNILs6s7Nb+8qyS0YFGjN\\\\\\\/d9vZTfOdNSwmTjTSaPxP6ETI9u9TM4v55kmGw6aE31H3FCYDxHaNktYgOTCQ84sM\\\\\\\/YVouXsPEoIBnxkZlUOs8IhrxmiJt1VgvmZTu6m\\\\\\\/\\\\\\\/aIyOy7VcF7DJm84VKOoyXvFjJHqBqRhypp4UslVlx9KD88k7ApYsmzhTCnaZFqpcUvhOGL79vLKLmZp+HC2GunvoqczQLnnaYd7DikCRRBmsmk8hZCU5xiB\\\\\\\/8yVhcfOb+UgcPcDjjTPPlLF4yU\\\\\\\/tTemPjEwngFlf29v9eh3DpSmFu9PdL\\\\\\\/AelYr7o6cUXEviUjzCoYSMTAD7ommAt8ZUWaP6pWNpw7n4yxl0xWPx2AK4jahtazMX17IQf35zRMDbmA9uTmYYR15DuFAg1TRAooVFupi0fNxj14XeOQNiygnSxq4ZLsrBG6\\\\\\\/epqOfJ6GbQO04aEWEo8rHgRrfczr23pV\\\\\\\/M7wUbMmGvqgBz0CfyEqPEH8GsWai4fcb2MinBqjGjSkTAP6ls3+H8y3ixCVIoK7pESvz9g1P9\\\\\\\/fdvapP+c7VcVSrOgenaLd3hT466\\\\\\\/qRn3Mz3ymDn8lM47X7YhjnI7H4BsOwl0kxcI6l06zoLUNHVxYnpzufrzehYzfHvcBc+W2psh9YKsW2c8bvb22KCZgm7aOpMAhYE+3Id5PESxv\\\\\\\/sfhZ7mznDEiFogUaA1QB\\\\\\\/q0SvcaGfkpgaMer7PWLhy+80ysYQbmkqys97oO+HBbJ+fkrL\\\\\\\/Av1LTEgyrOUXePjGr8TTes6J6qGWRJVNv8GzVT+7FgIf9+8AUFyLO9tvBBrtKgFsjIPJ41zGrwUR\\\\\\\/+Wnc4460SnbVLjHXbPTEkSJvNKFFXVMkV1f34Z7HWYBB0TwzxX3U\\\\\\\/ajJ3WgYQIKRL6dxUriBgDUhfGn5HQLXecNaaW0QdcK819IseoidJyc8MhoLMb8SU6u930kjt5En9Ed0hHZkAfwKKnN7WFFWkjAUvRVXGufTnNhaikR\\\\\\\/ffdrE5fHpmDN8bcGMbG+niak5GGsT4m0WvDbG2pxKtQ5GqS7LYYxE0z9KuMSHXpltsr4WIXLlMeR1hKXWnxDiA4skvdSIkgg01okp\\\\\\\/kQvIZw3qe\\\\\\\/B1cyNJBgFDhDqSSjFPUNxMXRt7SX1i2I72Ulec0BwyjhXO0LuY7NfFUzBle5aqehm4XQjTdI4kx9VAwCVQMwCBfqUfahSsaraVWUTTzMu0chefyC55wQ66CfxD7DeP7BT6BLAvlbh8IaFsnvW7QsKNF7kuaIN39OitoLM\\\\\\\/xLr\\\\\\\/uQtU8nmIVTZOYs2fbeCbZOa43E3rNg3onNK3Wlv\\\\\\\/jp1jDEiyYgnA3VesYc59oI\\\\\\\/SLQtIMcnDa7ZT3CVQcMu8id8rR1OuEklxGUq6KfP65PQh\\\\\\\/HmiCn7\\\\\\\/Au0r1RO7Bgcb7sMbm26iL4yWeg4276iZ\\\\\\\/3hBFDSQ1o8bUL7NzjS64WzqupTHIDMmS6li0UaR1SyTZyVBlycJGVL9BrT8XYp9SUGbRd42E\\\\\\\/EGs8EDNYZdIPsyKIlmLxt\\\\\\\/j8+eGQhRXfhCE6VrUTsooRceWTu3W5SE2jf5h3nxASWExMPjVFlsJ6DAN4UQfNbSjQivFN2nJjE58sw6FTDK5p89GpquOs9Zv1kfv90n1iLB0nWCIowzZdciOA7KW\\\\\\\/aUNoO\\\\\\\/2HdTr7KtyCZVGEPKQYt3ezgub2uDGwIYjXq1PloSq\\\\\\\/VdKuyOUn\\\\\\\/HmKNJIzkw\\\\\\\/Rok4GiPOIgiQjhvHmqSt3ZfYld\\\\\\\/xpwBsEzj3REPv3pFQB6NH48JkBP2LuIrjFoOYDym4AFzzcxS5RVfSWqVMfjJwQ6Y4iCU5S57NChpaM5zQxzQBIJRnDD4yIXW2c3rHNKsnZvmfvJW1bnINGkj5AYOZZmz2B1yKyo+Y9ldWT9lEIfYF3X3O0+KXR\\\\\\\/qtKjULn2HU5ycSPwmhG9Cb9CkOym4DqTJJLM9+nRHJajmocLqViM3MP8VK+FK0h2\\\\\\\/NWMQKR59cMlZIyUhR\\\\\\\/e6pDRW7nC7BU1wLudAjgCeedRPtQlDIGAiNRPvjuYHViB97Covg61SJVuH0AKWGdfKBIO4\\\\\\\/6KbW6zwj\\\\\\\/vpxsgQoTzepZDDLWqZtwbg\\\\\\\/gs\\\\\\\/3rMSaPKHW0FtwNUO9DeON38OtC4A75GBfODyFOVI9mOIfXOHA8TIxJU1GsJo5+KPIscLAGoEEB4hFdULa4JqeECeYSWEQVmoQMAs4LT\\\\\\\/BSQiXiNqJkcFJLbdhpcErd1EhgiKjQ3fXqXfRJnWSjf\\\\\\\/9KPsZnFLuO0XF\\\\\\\/2ly9PlM+YDEDTS3lFs6bsmBIN\\\\\\\/3wamoj9\\\\\\\/ZEoAA4sXb8GMJuaZ8nHSd+znivHm2Le+14mYvIbd0mawbGzB9oS03yBzW285j5I8IBAkwJXBNkkg3UlQPBjOSuL2GPeoLooBcJRkHsn1ZAkB15j2WX4OvwQEQgq\\\\\\\/86M6A2F0KcYLGKK8YNBScr9QmriL+dwJTWYtUl+ebfV8kbVNcyDaobliUTHTH6A1eAJ+Wm5lGt\\\\\\\/eXZJbx3gAa5e4y6UxS71Cy1JlqLesn4DKoVdSCdLCECIXXAIxAMaDINogu8OhTuV6Q07R\\\\\\\/3NOG1ao9+nwvm3a\\\\\\\/eohx6OARGLBgOw28b3Sm66J8mFlX+Ule1ZMxXlDuc7GdDnqutwBH1WjbKrws9JFwKM7Y87FCzyjCbE1tyids24ZRlmPhww7Jx8d4p6V52GDK19DRXyoSTGx6RTnSrvCvxezrG8XmW5GEGkXyIV8\\\\\\\/8ZIw3G\\\\\\\/EOxC\\\\\\\/BZWnuAWjfyHNbJFl9lJE7h4eG\\\\\\\/hvqtN8UxU4Iv25RGLNsSf3DtzC7R\\\\\\\/SO2Dpa0BMyp1Qt09NJ\\\\\\\/uEUp2vXvmU1HA4YI1GpsmMkeHYxmaWvhPcHcCoiG9uhTDWHOzoh3EpLuY\\\\\\\/FKoqWw8qv\\\\\\\/JpPlg92vROJNt0It53nfvdgbBQ8GIlMA7heM2knIRROeHP74CQPNRvrPNKZHvaBd31P0L5pwOncxgVU16yHzklyeY3argmUwD22pfjFlLnA9Y8paXlH0FuVx7JzyVRUhYXMik3dUzqywSN0+h2Zo7fg5WDGoSWJkH8HUt5Mw7XFTa2MfQlqwQhHeuZAd43ViFX1UjC0IeHoKIAsKVlszqT+pQ6IEDVH3oNX+LdqJOml7VfJUscjd0oFM3tNkdCzms+ev9Hb56wpITzZb\\\\\\\/FaSXXAv39ZylfHcyrYXL0goTsMa1iItNZVG9Df4AONZSwBSSqandCdypd1lAHtwtHekUohfBbC5pgn8Po\\\\\\\/bJneudkQivU5+ZjNnwhzLoF3PX2gxim3SHL7aq\\\\\\\/NnooU83oMmW5YRbI7bvMTPRoPjfTVBVznvqwzxbKp9CAc2lD+sXOcYiYwi0N7R1+7nAbAFn6PNG6SxlliBffG5qEMQ5PgHNiN4GiCReEtfRrKzTKbe20vfKhL8enUXWK6yfvy2XKj0wZdBH5H5CAPth8iX1PJ0fDlu9JTBpQM3XlpFOyLgEhX4I5PK1Vu70WSJHL6cxph0XZtUuIUsZNyZhC+Ldzdt81LzpDwfruwr\\\\\\\/Qj19Z49laxC9OPXpcMGNkFcnQP9J6jU2fsal6xoefcOltyBJsQOAsUlWJrVaa0xagdOuQ5vXLGIwdGh8y7s3Fb+qQhv+liB3ek3Fgum8dQZGgpilbcewszKTABBHkfvfcQNYibRPs5\\\\\\\/2Udjg9yWEVyg1yV25iJkwu4EKS0m3u+9\\\\\\\/LwvmredHfEvq6ObuHJz3IfEH95TSwNsvma7IeBqiffS7l2tCRLFrp9wpiY62xJXTpvokwkWX1GxzYQYXCpP0EJEMdiNc75K\\\\\\\/AjeUT2iWa76TyA4aEhUe5NbR3mz4M7dCENYH01CDEc\\\\\\\/RAu0FQSA0j7WIJBE3+MqDFG5JSo0TkZUlEEGB3RfuCjGUk1pdsCawx+evkiy6hx4pWyjauxxIhwh7bOgk51f7wVk5E373PjlRJb+FzZHNb3\\\\\\\/14Nu6eqr+yFRLd05lv\\\\\\\/5ospje4jeM+8GxC\\\\\\\/QWZdYyu0ImFPq+jPWWQ0kLgkun3n9jPU2pUdPBQQEFpvbfzN+h\\\\\\\/bIyNm+qz9rAo2+E2EgREksHHHaEscjFenBa+qtGJvvnhLYizB01q1+mPW0iHGs46D\\\\\\\/Eht7g2eMTWCsDORgKHkg9sAkjJK1bUPhiPxMRa76vaXP7wqStWLVxKDKUcRuUZ9\\\\\\\/kdlWyW6avSwSuq5ZIxhnJIDG+GgYImE0Pv5j62TsRPp46ApAkzei\\\\\\\/ipazCtT7j39L4KH5F+YF6oVnH\\\\\\\/vH7OeX138WCkswbDP3E+FJbrwabZa5HVJgo0A135tjWWgG6\\\\\\\/o8x39gZjSRQurP8mRviNSgdzLdAzXpgprdmY1tQgv18CXcPo3wj+7ZgO9P\\\\\\\/51ZlCX5+fnCwjhumYhX07FexOH8aCyvYl1Vcl0q3hRq2zZ\\\\\\\/n7AWkIt0mm\\\\\\\/xpRxLO19xY2QxBfssDWUcjV069vSxxeL9BPALYtPSDVWbQA04RxqLx+cT2Az3WtFJRc\\\\\\\/2bXqVj7oDvFXnuMJTAEb5R7t3jv0+rQyyDds0+rqF6ONY0l7bfmVm5qY7f+2X8o\\\\\\\/w9HTXqm6fj1G6UdJ1Oe5HNge5bhcHrTXOWRlaZO3VXjr\\\\\\\/mwa6dQUPOE0spZaWSK\\\\\\\/y7ugI4bvAmcgj5wCHCoIqLcc3kPCw0v7UkPUf+PMbVVIuswZfaIGdSmw3qI7XJb3SPBiAco5eMWFvN7K5g9CDRiwBPPfKI3nOP8gi5MsLEhFYK38jaj7bF6bcPUc1PxnMHVPGlcXzAdAf28HFmkY0byCthzlHZwrEWvq8+Cn9v1ifBXjSdSmisYCuq0uekIeLh0hvHyTycZvFVSxTlB9zLLfuK40cYRaItZe3+O1pDFsjQLA4kpRDlk5Bso7BGpwdwnpb1tLRgz8nvduFQdxzYsjoF634cJOuGKBggV6eAyFKlpuw\\\\\\\/W6iLfS6jZPBCTcFkUR9ZT38h9peFrbkALxxDRUoyiKDw\\\\\\\/vJ16z93eta27tpfQyHyPZ+rtgHOuDGI53nyA5g9q0dZyZGXLMgWsWunHe1DrGouCUbyIb5s\\\\\\\/yZ7rvGTIxBUhpKvB9vTAupnOt53dMFV6bpeTKoEUS\\\\\\\/T5x67bM+0b72L4mUuOFzZRJgwD9ySgvUVtuiVZOsmGkaRJhbc0e6jmK6XUhfjLDTFyL2yOE49SOaeDsEWF6SNnMGdyW\\\\\\\/wvE5KzjXb+H+S79I8Ph8wNYfCZTu8Pm6yeEPyHSOH7zkGE0uN61JqxzbuwKS+vj8wMPlPpDrNOHXxjNiF0FZ9N9+\\\\\\\/e89fIKcO2JFJDOFXbV8sZPriYK\\\\\\\/iq5KHUGraSrjy+a1d2LeK7LyOlARsTFT+I4IbLXBAjX7Bjc+O7I23n3gaYQ0SRE7ey2GkiYSEwnoNFVgkv4Rbr8R5uKEnVYOGsG+KtWLhYsAyTS+mcviOH9BhwRd1TKwimYmckjL1DcJW+yYqHx6nXbA3NGvOmUJUoYMzPxhhACwMqS1ppKgd+2\\\\\\\/bNzh6xh2eEmbc2t4rzoPflnTkL8LUTupXOxqSmepStfn9tkaQGHsY7pU5\\\\\\\/npPHAdxXid4wPpVATlxqw0eVtpgMGoXcpo+xFbk8nVzNpz3PnDysEi\\\\\\\/9817Wj2qpL0FXuiLk2Liq6jt3VzQeEK2wyT33zIW\\\\\\\/wRbCJLX5RRvQxy733UBWBYcBHLgtXZ7MsorU3dy8nFBt1qJF4HphGAAQqFphk68J0lm7MQKUzAQU5JkzzxA0bX6oEGuDYVTNX8s6aW0+ctvOL1fvx+aW2cKqy\\\\\\\/1eeSdl2pYF\\\\\\\/yOIT8\\\\\\\/zXfw\\\\\\\/\\\\\\\/ytzsRrzf4pXcBSZWE6Ki\\\\\\\/FTeyZCIYU+UymJXEFcMrGE7JTNmRq\\\\\\\/tULpU\\\\\\\/3MmKLRwCmwyzXSMTPnvUwuRU\\\\\\\/2Z7EAsio8rafe\\\\\\\/mFryUmtljJzX0rgrK4hiEqpValE\\\\\\\/yDXgoi07zgvTt6PCo3R\\\\\\\/jfkczbRnD5dVO669jVdUI1dfERJBq+TYEvemasoSqpfGtVhP2+ARFWX1EE19M4cguUtEbR7lRBceNwKChf7sng25RvXK5dgzMfnxAMDAGGm40YyhFaSgIdqOwJEid2Xt2U4e4zH6ee2POQ0K0hvFkNkbOYtuSO5wuTK68hzU6w7Imc8\\\\\\\/+sKLGBfsqkyZXfE0nttwweu0nzkO4wmhNuu6KLptfnAWkF5F7xwSDUZapNJ15dYq6Z51Ekn1spYc9NEmEme2niorNMNDPwPiAc9CMZlrxCOjtSvfYUxcMX7m8DTuQITEdtvvXfURJPXW7IZty4NQnCSHgJvouv1O+DY0pjEzf65FmrBr0HCcS5QvSNAxRn\\\\\\\/akRzibKsV0aiw5TeWkhFz+RoOT5ietMUG+tB8BVub33Gg4Os+ryEz48EzInFouMVVW3v7xVotrIhijvKm3RmP2e+lQsscN\\\\\\\/TDBFeAm\\\\\\\/6V5fvw\\\\\\\/tLXv03thgIgtHZ3nExjL0mR+wrreQqzjiZDQO2miOVCmOBTv0Ga6YBuXSNTNvx3GKHibe7OxOowXZOpSAVNYXnsVEDdSG62EEIZI8d3eF\\\\\\\/lVVUZFmuPYpoymmu8PIAW5Laxx8JWe0NIdhVqhuChmxZToRqItoL5xMndgYKaAydyArZoggoAK3fvYhmqtlwt8GJ3kOI\\\\\\\/sQb1GkPmpoxwB2bFXg5EwvoDlOZHEfK\\\\\\\/sPodv8pvCOVmVa2A4gD9wECjMRQ1YG4GMl9qgKM7wPzkrc5LhSOpS0Ja42JMMJXJZeuzbVvTP91Km162qA09LSUj8d1OtsfE+eniSt\\\\\\\/2jjXISFbfipk9PkfAnDlGSTYivlnWBalk\\\\\\\/sy+EetHc\\\\\\\/1J+zhBsdDYNgDrboWN10xmyPalTHxNUrCGj0erV3+AxJKnZAYO9lwp5v2hnBiinivCEU4wUuJljOQ8SBZiycbAhehnh76Fl9xvVuAmqWS8EKiHMDTM1ypQO3UsZ2KggZCdHyyIAzC3UlrpnfREXzBKtpQmYQGGhzGwSbvQgcJFwyicFHgpikJ6ncqtrqiG\\\\\\\/pmoEW6uEmvn+KIDKpDw7WpqzGWNHa4dPidlcRmFxRlfALGxVo40yffR+aXTjzS0u0errgTW+DSggQvuQ3i\\\\\\\/MrIQLFbvwDY+m1OVVoEmX6fGI4FTxyYFl7BOPZww491BEDEaaAYNhZ5fD5s5E4sNUJAs7KFE1KsWfbjCqGh3ul0UulGk5euu77rOp9NWB2J0pfOlMmNSqdjcyP1mQGhqvUySxK\\\\\\\/cuFmMZvM8JIPlDTswh1WmhnU3z5pxy+YaVsrBZdHweJvAJM8ozQnxRhXp35xaYh8rtwDxXZWERp3Vq\\\\\\\/P4isXl6x2dpVOLpbPuAW\\\\\\\/pWlo3RNVljI6Wl\\\\\\\/gQu1C6lMOFERU+DCMLs1w2qAUe4HLAWX39MBLyPb\\\\\\\/v1Ew0ed2jX5fF17vJeaj0LVkAU65KrRU5AajO2yBKuD+qmKzMX5YODQU787+l6oPo+xAKxnwezsduruKjcNtv7zSodFKhmoxJ47DhPlYETmu5yn6zUjZUrOVC2DCDCYnTIGUv5S44poIwjzozxSGFva06BjyGQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022hgGVzWj4+B6LwvADLYXosg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022475cd28431b0762c28263737c859706db0bba05ac0d8155195219ccccac7d809\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022zz4sh3t3KnBaGWgshyOzuIp\\\\\\\/Fuk4ob3byuOhOIDZQKHSFpbIz03S7luT9ti\\\\\\\/9hlLRACCa8141P5myDlvrMTJh2m6y4fy\\\\\\\/K4Y0BPKkGzxvdd82pnEFg2aCFhSQqBeOE2y2szprxv61M9Q6YTYe7Tcrqtg\\\\\\\/s1oRoo7jisdClFc7SmliQ9W6sh\\\\\\\/kxFbgMWcN07G8aCcFNFvfn\\\\\\\/jk5gDpo\\\\\\\/6rZKIVJktpCEDgDGhasjYxjpx43d8o3skdscfVchZb6AHWAuS3JiRN\\\\\\\/n89ZsKHLd7aYr99\\\\\\\/w8isfJO3FifWd3PdOowFpzo4gtUOGGC+eILTxroDs1BRZ0JedmC8rxCN\\\\\\\/ar0sAMLzj6L4GHmalpBp0wBb\\\\\\\/k\\\\\\\/6+1dfjl4T4LHMjiQHH5NzZHujimoeyedi6ZA\\\\\\\/vgi6Zd09mTlTEAyP\\\\\\\/BzyVLKUPBNtpqtPhfxRajMxIfbcz\\\\\\\/sTgOq2MeqmckEteD+SoKgZwtT+UEan32r34bOvztexoIOHV9TFDN6LwIgw8wN98pLlrvKXJt7YWbP\\\\\\\/3CUtQTa6gwHpFoab+sHVXnSdaN45i7dKjUhB73lJaXlbohQonvq25\\\\\\\/WDuwmqtMhgnrquxWyjgB1GnV2uP\\\\\\\/RMbefDMX8Dr8vJrZajndOVmIo01MT5Qg7FIlYqYlTu47ozDRGfGBk5XCD9z0nYA2Y+NBNNM6xC4fSwH6QHaIOXo+bhshcgjJMTnmhJtHPGNa4PzaYBUWYZlfh2l2le8lsifxG7nnKQPqjBZH+1rHeyEc+9bC\\\\\\\/bj\\\\\\\/qv\\\\\\\/PWEBNRuwzbIIzGVSs1rLJ99hbZj5BceUZO049BlVKsexI\\\\\\\/ahPAGdUL0PyGD\\\\\\\/xQM08RUliAW7e98\\\\\\\/ab4Ls+Dg\\\\\\\/O0c\\\\\\\/DfVpthyQUE47Ysg0UG1p55KeGfpEQv3SDSIK0IaHKpl83guyZDOemARp+9075\\\\\\\/7WliJTPWAxmpBJRu9h5pBaBx26Ns6CfacdTIF5Ydj25DrA6HMR3GEf\\\\\\\/nViDVGB6LsE\\\\\\\/MoJmbieW3\\\\\\\/6SaGsogh8PM4X\\\\\\\/V0pnCGBDERw\\\\\\\/YWu9sqM\\\\\\\/HgWJk6bbAI6GPcZIinAAK33INi6omjbH9XUDbYQeptkEgYRkK\\\\\\\/5Uq9vlyTNFF5GF5eSuHZsnt3IlEXIhj9aV\\\\\\\/2DwgoEqFvl7BqRdcEz6h4PEf\\\\\\\/282tZWF9iZb8E8HBEU4x+bTdC7P8xcc8owvJhMgZxOGKCbONdFMoR\\\\\\\/xeXmHF+5jp+quOvl9o1nXzuU5U7lU\\\\\\\/KaCkXawWolD6stndASXEIOXfGBhSaowWcigXIL6jEzrq0T7zdYIsMdW5VMN+LEKvg\\\\\\\/33hcWG9q2yd6Ej8O5hjThom\\\\\\\/CDf21OTWLXMbXnHJ1d5JJnTKuFvwE5dgrid9+isExzs9XXJ2IgDUPMFM+nW+mW2eAdvGS+M7aSQgLIGFTD3kFUC61NrfdiAhA18gXJ8ddCR07WcsE83izZNskWSWF8U2N8G1bCj5JqGxYeGHMnmYsXQwk+uUQ8SB3NRrnuofkW6XUTglh7YD2FB0Q0Jj3vwo\\\\\\\/K0kwJ45tw6O5lKoKCgW4C9vZPX+CFGEz\\\\\\\/LRfuntp+5JaAfUT4ccYUfuaXrA4LcYILOsYqvcHBl00bktKv6etczdSzQB9YjwYnEUVvZd6irOfrvXcqvQlOpjR8v5BIulZmo23279zNFVTgKRlT3sgs4amjquvJlg1JLKjqGzHrk1w220PKN4aRLePzhq88wNGQUEgYEd\\\\\\\/LgpmbMP\\\\\\\/Z2Y5ufbYRO5QHI6e4MONaWrFARojTLJBTaeuPjDwfJjI7WHauIIkwF+zQkRPUAk+ZZL57vEMXbxsKuQdVj9WYQgD\\\\\\\/lSqp7zExlCN0b2apsBu7TthoCV+01G651dpUkM19HsqKrMyMoZ9IMk\\\\\\\/nbwYKvrsQBeH2u1OeVseDidxeO5C8JEKkQ3DdwkeIVbE3J+DNBDbAwYC\\\\\\\/akdEfb4hYd9jRFsEFBPzkCCCJGRwVZybwP5N84XhZuUi\\\\\\\/2MDzAzKYD3d57\\\\\\\/P6WaS7qkPi6Mqm\\\\\\\/eNYKvHBr3RUQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022BcKoD1MmInDKuAhVuJ9p9w==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022b23cfe5903a3bcf1efe1d99f5b38fc58f2c4e1f74b4c483368d9987c2d147a2b\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-492621 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-genomic-prediction">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/4/1117" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-3UTGgSc5eapZkO77F9rXeTOGrRJwOF6sMXbBV3mSjuA" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/4/1117" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-bR7fPnH7NIyLLE1kmRJjjFChA1YAsYL1av9GAnyfoE0" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/4/1103" title="Evolution of the Highly Repetitive PEVK Region of Titin Across Mammals" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/4/1131" title="QTL Underlying Circadian Clock Parameters Under Seasonally Variable Field Settings in Arabidopsis thaliana" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="492621" id="node492621" data-pisa="ggg;9/4/1117" data-pisa-master="ggg;g3.118.200961" data-apath="/ggg/9/4/1117.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Lost in Translation: On the Problem of Data Coding in Penalized Whole Genome Regression with Interactions</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-0628-6794" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Johannes W. R. Martini</span>, <span class="highwire-citation-author" data-delta="1">Francisco Rosales</span>, <span class="highwire-citation-author" data-delta="2">Ngoc-Thuy Ha</span>, <span class="highwire-citation-author" data-delta="3">Johannes Heise</span>, <span class="highwire-citation-author" data-delta="4">Valentin Wimmer</span> and <span class="highwire-citation-author" data-delta="5">Thomas Kneib</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1117-1129;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200961 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node492621" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Johannes W. R. Martini </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>KWS SAAT SE, Einbeck, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Johannes%2BW.%2BR.%2BMartini%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Martini%20JW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJohannes%2BW.%2BR.%2BMartini%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-0628-6794" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Johannes W. R. Martini</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jmartin2@gwdg.de" class="" data-icon-position="" data-hide-link-title="0">jmartin2@gwdg.de</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Francisco Rosales </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>Universidad del PacÃ­fico, Academic Department of Finance, Lima, Peru</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Francisco%2BRosales%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rosales%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFrancisco%2BRosales%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Ngoc-Thuy Ha </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â¡</span>Department of Animal Breeding and Genetics</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ngoc-Thuy%2BHa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ha%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANgoc-Thuy%2BHa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Johannes Heise </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>Â§</span>IT Solutions for Animal Production (vit), Verden, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Johannes%2BHeise%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Heise%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJohannes%2BHeise%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Valentin Wimmer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>KWS SAAT SE, Einbeck, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Valentin%2BWimmer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wimmer%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AValentin%2BWimmer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Thomas Kneib </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Chairs of Statistics and Econometrics, University of Goettingen, Goettingen, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Thomas%2BKneib%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kneib%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AThomas%2BKneib%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/9/4/1117" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:492621" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:492621/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/4/1117.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:492621" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:492621/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/4/1117.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:492621" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:492621/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/9/4/1117.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:492621" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:492621/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">Mixed models can be considered as a type of penalized regression and are everyday tools in statistical genetics. The standard mixed model for whole genome regression (WGR) is <em>ridge regression best linear unbiased prediction</em> (RRBLUP) which is based on an additive marker effect model. Many publications have extended the additive WGR approach by incorporating interactions between loci or between genes and environment. In this context of penalized regressions with interactions, it has been reported that translating the coding of <em>single nucleotide polymorphisms</em> -for instance from -1,0,1 to 0,1,2- has an impact on the prediction of genetic values and interaction effects. In this work, we identify the reason for the relevance of variable coding in the general context of penalized polynomial regression. We show that in many cases, predictions of the genetic values are not invariant to translations of the variable coding, with an exception when only the sizes of the coefficients of monomials of highest total degree are penalized. The invariance of RRBLUP can be considered as a special case of this setting, with a polynomial of total degree 1, penalizing additive effects (total degree 1) but not the fixed effect (total degree 0). The extended RRBLUP (eRRBLUP), which includes interactions, is not invariant to translations because it does not only penalize interactions (total degree 2), but also additive effects (total degree 1). This observation implies that translation-invariance can be maintained in a pair-wise epistatic WGR if only interaction effects are penalized, but not the additive effects. In this regard, approaches of pre-selecting loci may not only reduce computation time, but can also help to avoid the variable coding issue. To illustrate the practical relevance, we compare different regressions on a publicly available wheat data set. We show that for an eRRBLUP, the relevance of the marker coding for interaction effect estimates increases with the number of variables included in the model. A biological interpretation of estimated interaction effects may therefore become more difficult. Consequently, comparing <em>reproducing kernel Hilbert space</em> (RKHS) approaches to WGR approaches modeling effects explicitly, the supposed advantage of an increased interpretability of the latter may not be real. Our theoretical results are generally valid for penalized regressions, for instance also for the <em>least absolute shrinkage and selection operator</em> (LASSO). Moreover, they apply to any type of interaction modeled by products of predictor variables in a penalized regression approach or by Hadamard products of covariance matrices in a mixed model.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/interactions" class="hw-term hw-article-keyword hw-article-keyword-interactions" rel="nofollow">Interactions</a></li><li class="kwd"><a href="/keyword/whole-genome-regression-0" class="hw-term hw-article-keyword hw-article-keyword-whole-genome-regression" rel="nofollow">whole genome regression</a></li><li class="kwd"><a href="/keyword/egblup" class="hw-term hw-article-keyword hw-article-keyword-egblup" rel="nofollow">EGBLUP</a></li><li class="kwd"><a href="/keyword/gxe" class="hw-term hw-article-keyword hw-article-keyword-gxe" rel="nofollow">GxE</a></li><li class="kwd"><a href="/keyword/hadamard-products" class="hw-term hw-article-keyword hw-article-keyword-hadamard-products" rel="nofollow">Hadamard products</a></li><li class="kwd"><a href="/keyword/genomic-selection" class="hw-term hw-article-keyword hw-article-keyword-genomic-selection" rel="nofollow">Genomic selection</a></li><li class="kwd"><a href="/keyword/genomic-prediction" class="hw-term hw-article-keyword hw-article-keyword-genomic-prediction" rel="nofollow">Genomic Prediction</a></li><li class="kwd"><a href="/keyword/genpred" class="hw-term hw-article-keyword hw-article-keyword-genpred" rel="nofollow">GenPred</a></li><li class="kwd"><a href="/keyword/shared-data-resources" class="hw-term hw-article-keyword hw-article-keyword-shared-data-resources" rel="nofollow">Shared Data Resources</a></li></ul><p id="p-4">Genomic selection based on whole genome regression (WGR) is a crucial ingredient of modern breeding programs (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Meuwissen <em>et al.</em> 2001</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Schaeffer 2006</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Habier <em>et al.</em> 2007</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Hayes <em>et al.</em> 2009</a>, <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">2013</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">de los Campos <em>et al.</em> 2013</a>). The simplest and most successful approach for modeling the genotype-phenotype relation is a linear model assigning an additive effect to each locus (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Falconer and Mackay 1996</a>). In more detail, the standard model is given by<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-1.gif"/></noscript></span><span class="disp-formula-label">(1)</span></span>where y is the <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-2.gif"/></noscript></span></span> vector of the phenotypic observations of <em>n</em> individuals and <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-3.gif"/></noscript></span></span> an <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-4.gif"/></noscript></span></span> vector with each entry equal to 1. Moreover, <em>Î¼</em> is the <em>y</em>-intercept, and M the <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-5.gif"/></noscript></span></span> matrix describing the marker states of <em>n</em> individuals at <em>p</em> loci. Dealing with <em>single nucleotide polymorphisms</em> (SNPs) and a diploid species, the entries <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-6.gif"/></noscript></span></span> can for instance be coded as 0 <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-7.gif"/></noscript></span></span>, 1 <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-8.gif"/></noscript></span></span> or <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-9.gif"/></noscript></span></span> or 2 <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-10.gif"/></noscript></span></span> counting the occurrence of the reference allele A. The most frequently used coding subtracts twice the allele frequency, centering each column of M to zero (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">VanRaden 2008</a>), and alternative approaches consider genotype frequencies (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Ãlvarez-Castro and Carlborg 2007</a>; <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Vitezica <em>et al.</em> 2017</a>). The <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-11.gif"/></noscript></span></span> vector Î² represents the allele substitution effects of the <em>p</em> loci, and Ïµ the <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-12.gif"/></noscript></span></span> error vector. For single marker regression, which may for instance be used in <em>genome-wide association studies</em> (GWAS), we could apply <em>ordinary least squares regression</em> (OLS) to estimate <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-13.gif"/></noscript></span></span>. However, in approaches of genomic selection, we model the effects of many different loci simultaneously and the number of markers <em>p</em> is usually much larger than the number of observations <em>n</em>. Different methods have been used in the last decades to deal with a large number of variables, among which <em>ridge regression best linear unbiased prediction</em> (RRBLUP) is the most popular in quantitative genetics (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Schaeffer 2004</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Mrode 2014</a>). RRBLUP penalizes the squared <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-14.gif"/></noscript></span></span> norm of Î² and has been derived with the additional model specifications of <em>Î¼</em> being a fixed unknown parameter, and <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-15.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-16.gif"/></noscript></span></span> being random. Thus, RRBLUP is not a pure ridge regression penalizing all parameters, but actually a mixed model in which the size of <em>Î¼</em> is not penalized, but only the entries of Î² are. This mixed model RRBLUP is also called <em>genomic best linear unbiased prediction</em> (GBLUP) when Eq.(1) is reformulated with <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-17.gif"/></noscript></span></span>, and thus <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-18.gif"/></noscript></span></span>. Different variants of <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-19.gif"/></noscript></span></span> are called the <em>genomic relationship matrix</em> (GRM) (<a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">VanRaden 2008</a>). In the additive effect setup, RRBLUP may be considered as the standard reference method (<a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Schaeffer 2004</a>; <a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Mrode 2014</a>), but there are many other approaches which use different assumptions on the effect distribution and other approaches to estimate additive effects (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Gianola <em>et al.</em> 2009</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Gianola 2013</a>).</p><p id="p-5">In particular, because of the immense structural contrast between the statistical additive effect model which does not include any type of interaction between the loci, and biological mechanisms in which interaction is a key concept, scientists have been interested in modeling interaction and ânon-additiveâ genomic relationship (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Howard <em>et al.</em> 2014</a>). Several manuscripts have addressed the detection of statistical interaction (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Cordell 2009</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Aschard 2016</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Chen <em>et al.</em> 2016</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Ehrenreich 2017</a>), the role of epistasis in selection response (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Carlborg <em>et al.</em> 2006</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Esfandyari <em>et al.</em> 2017</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Forneris <em>et al.</em> 2017</a>) or the predictive ability of non-additive relationship models. An important class of non-additive relationships is given by <em>reproducing kernel Hilbert space</em> models (RKHS) (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Gianola and Van Kaam 2008</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">de los Campos <em>et al.</em> 2009</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Ober <em>et al.</em> 2011</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Morota and Gianola 2014</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Gianola <em>et al.</em> 2014</a>). Moreover, a strongly followed approach simply extends Eq.(1) by explicit dominance effects or by interactions between different loci (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Jiang and Reif 2015</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Ober <em>et al.</em> 2015</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Gao <em>et al.</em> 2017</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Martini <em>et al.</em> 2016</a>; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Varona <em>et al.</em> 2018</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Su <em>et al.</em> 2012</a>; <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Xiang <em>et al.</em> 2018</a>). The latter approaches have the supposed advantage of being interpreted more easily, since we can estimate interaction effects instead of just dealing with a non-additive genomic relationship model derived from the RKHS setup.</p><p id="p-6">Adding products of predictor variables to model interactions, extends <a id="xref-disp-formula-1-1" class="xref-disp-formula" href="#disp-formula-1">Equation (1)</a> to a polynomial of total degree 2. Whereas subtracting a constant <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-20.gif"/></noscript></span></span> from the <em>i</em>-th column of M, does neither change the predictions <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-21.gif"/></noscript></span></span> of an OLS regression with interactions (provided it is well-defined) nor those of the penalized regression RRBLUP (provided the penalty factor remains fixed), the predictions of a penalized regression with interactions (<em>extended RRBLUP</em> or <em>eRRBLUP</em> or <em>eGBLUP</em>) are sensitive to a translation of the coding (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">He and Parida 2016</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Martini <em>et al.</em> 2017</a>). Moreover, also the estimates of additive effects inferred with RRBLUP or OLS in an additive model are invariant under translations of the variable coding (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">StrandÃ©n and Christensen 2011</a>), but contrary in a model including interactions, the interaction effect estimates will only be unaffected when OLS but not when eRRBLUP is used.</p><p id="p-7">In this work, we address the question why penalized regression is affected by translations of the variable coding when a polynomial model of higher total degree is used. After a theoretical summary of the different methods, we show that in many cases, translating the coding of the predictor variables has an impact on the prediction of genetic values, but that an essential translation-invariant exception is the situation of only penalizing the size of the coefficients of monomials of highest total degree. The invariance of RRBLUP can be considered as a special case of this setting, with a polynomial of total degree 1, where the size of the fixed effect (total degree 0) is not penalized, but only the additive effects (total degree 1) are. The eRRBLUP, which includes interactions, is not invariant to translations because it does not only penalize interactions (total degree 2), but also additive effects (total degree 1). In this regard, approaches of pre-selecting markers, for instance by their additive effect sizes (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">KÃ¤rkkÃ¤inen <em>et al.</em> 2015</a>), are not only computationally interesting but may also wipe away the coding problem if they allow to model the additive effects as being fixed and to penalize only interactions. Finally, we use a publicly available wheat data set to illustrate that the impact of coding on interaction effect estimates of eRRBLUP becomes stronger with increasing number of variables included in the model. This observation suggests that potential underlying biological interactions of <em>quantitative trait loci</em> (QTL) which may be in <em>linkage disequilibrium</em> (LD) with the markers, will not only have less influence on estimated interaction effects due to the direct influence of an increasing number of variables across which biological effects may be distributed when fitting the data, but the loss in biological meaning may be enhanced by the secondary effect of an increasing influence of the marker coding. Thus, the supposed advantage of a higher interpretability when modeling interaction effects explicitly in a WGR, compared to RKHS approaches defining non-additive relationships, may only be marginal.</p><p id="p-8">We start with a recapitulation of the regression methods which are relevant for the manuscript.</p><div class="section methods" id="sec-1"><h2 class="">Theory: Specification Of Regression Methods</h2><p id="p-9">If an expression includes an inverse of a matrix, we implicitly assume that the matrix is invertible for the respective statement, also if not mentioned explicitly. Analogously, some statements for OLS may implicitly assume that a unique estimate exists, which in particular restricts to cases in which the number of observations is at least the same as the number of parameters which have to be determined. Moreover, we will use âestimatedâ and âpredictedâ effects as synonym in this work since the quantities may be considered as being fixed or being random in several instances.</p><div id="sec-2" class="subsection"><h3>Additive effect regression</h3><p id="p-10">The additive effect model has already been presented in <a id="xref-disp-formula-1-2" class="xref-disp-formula" href="#disp-formula-1">Equation (1)</a>.</p><div id="sec-3" class="subsection"><h4>OLS:</h4><p id="p-11">The ordinary least squares approach determines <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-22.gif"/></noscript></span></span> by minimizing the sum of squared residuals (SSR):<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-23.gif"/></noscript></span><span class="disp-formula-label">(2)</span></span><span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-24.gif"/></noscript></span></span> denotes here the <em>i</em>-th row of M representing the genomic data of individual <em>i</em>. The solution to the minimization problem of <a id="xref-disp-formula-2-1" class="xref-disp-formula" href="#disp-formula-2">Equation (2)</a> is given by the well-known OLS estimate<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-25.gif"/></noscript></span><span class="disp-formula-label">(3)</span></span>provided that the required inverse exists, which in particular also means that <em>n</em> has to be greater than <em>p</em>.</p><p id="p-12">In problems of statistical genetics, we often deal with a high number of loci and a relatively low number of observations. In this situation of <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-26.gif"/></noscript></span></span>, the solution to <a id="xref-disp-formula-2-2" class="xref-disp-formula" href="#disp-formula-2">Equation (2)</a> is not unique but a vector subspace of which each point minimizes <a id="xref-disp-formula-2-3" class="xref-disp-formula" href="#disp-formula-2">Equation (2)</a> to zero (âoverfittingâ). Using an arbitrary value of this subspace, predictions <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-27.gif"/></noscript></span></span> for genotypes which have not been used to estimate the parameters <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-28.gif"/></noscript></span></span> usually have a low correlation with the corresponding realized phenotypes. An approach to prevent overfitting is RRBLUP.</p></div><div id="sec-4" class="subsection"><h4>Rrblup / Gblup</h4><p id="p-13">minimizes<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-29.gif"/></noscript></span><span class="disp-formula-label">(4)</span></span>for a penalty factor <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-30.gif"/></noscript></span></span>. Using an approach of maximizing a certain likelihood, the model specifications of <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-31.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-32.gif"/></noscript></span></span> determine the penalty factor as ratio of the variance components, that is <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-33.gif"/></noscript></span></span> (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Henderson 1975</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Henderson and Quaas 1976</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Henderson 1977</a>). We stress again that <a id="xref-disp-formula-4-1" class="xref-disp-formula" href="#disp-formula-4">Equation (4)</a> is not a pure ridge regression, as the name RRBLUP might suggest, but a mixed model which treats <em>Î¼</em> and Î² differently by not penalizing the size of <em>Î¼</em>. This is the version, which is most frequently used in the context of genomic prediction (often with additional fixed effects) (<a id="xref-ref-41-3" class="xref-bibr" href="#ref-41">Schaeffer 2004</a>; <a id="xref-ref-35-3" class="xref-bibr" href="#ref-35">Mrode 2014</a>).</p><p id="p-14">The corresponding solution is given by<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-34.gif"/></noscript></span><span class="disp-formula-label">(5)</span></span>where <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-35.gif"/></noscript></span></span> denotes the <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-36.gif"/></noscript></span></span> vector of zeros. The effect of the introduction of the penalization term <span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-37.gif"/></noscript></span></span> is that for the minimization of <a id="xref-disp-formula-4-2" class="xref-disp-formula" href="#disp-formula-4">Equation (4)</a>, we have a trade-off between fitting the data optimally and shrinking the squared effects to 0. The method will only âdecideâ to increase the estimate <span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-38.gif"/></noscript></span></span>, if the gain from improving the fit is greater than the penalized loss generated by the increase of <span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-39.gif"/></noscript></span></span>.</p></div></div><div id="sec-5" class="subsection"><h3>First order epistasis: Polynomials of total degree two</h3><p id="p-15">An extension of the additive model of <a id="xref-disp-formula-1-3" class="xref-disp-formula" href="#disp-formula-1">Equation (1)</a> is a first order epistasis model given by a polynomial of total degree 2 in the marker data (<a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Ober <em>et al.</em> 2015</a>; <a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Jiang and Reif 2015</a>; <a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Martini <em>et al.</em> 2016</a>)<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-40.gif"/></noscript></span><span class="disp-formula-label">(6)</span></span>Here, all variables are as previously defined and <span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-41.gif"/></noscript></span></span> the interaction effect between loci <em>j</em> and <em>k</em>. Please note that there is a variant of this model, in which also <span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-42.gif"/></noscript></span></span> is included. This interaction of a locus with itself allows to model dominance (<a id="xref-ref-32-3" class="xref-bibr" href="#ref-32">Martini <em>et al.</em> 2016</a>).</p><p id="p-16">We recapitulate some terms which are important in the context of polynomials in multiple variables. Each product of a subset of the variables <span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-43.gif"/></noscript></span></span> is called a monomial. For instance <span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-44.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-45.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-46.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-47.gif"/></noscript></span></span> are four different monomials. Since the product is commutative, <span class="inline-formula" id="inline-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-48.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-48.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-49.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-49.gif"/></noscript></span></span> are the same monomial (and their coefficients are assumed to be summed up in any polynomial which we will address later). The total degree of a monomial is the sum of the powers of the variables in the respective monomial. For instance, <span class="inline-formula" id="inline-formula-44"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-50.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-50.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-45"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-51.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-51.gif"/></noscript></span></span> are monomials of total degree 1, whereas <span class="inline-formula" id="inline-formula-46"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-52.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-52.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-47"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-53.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-53.gif"/></noscript></span></span> are monomials of total degree 2. Moreover, <span class="inline-formula" id="inline-formula-48"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-54.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-54.gif"/></noscript></span></span> is a monomial of degree 1 in each of the variables <span class="inline-formula" id="inline-formula-49"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-55.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-55.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-50"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-56.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-56.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-51"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-57.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-57.gif"/></noscript></span></span> is a monomial of degree 2 in <span class="inline-formula" id="inline-formula-52"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-58.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-58.gif"/></noscript></span></span> and of degree 0 in <span class="inline-formula" id="inline-formula-53"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-59.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-59.gif"/></noscript></span></span>. Since a polynomial model is also linear in the coefficients, the regression equations are only slightly modified.</p><div id="sec-6" class="subsection"><h4>Ols:</h4><p id="p-17"><a id="xref-disp-formula-3-1" class="xref-disp-formula" href="#disp-formula-3">Equation (3)</a> with a modified matrix M including the products of markers as additional predictor variables represents the OLS solver for model (6).</p></div><div id="sec-7" class="subsection"><h4>eRRBLUP:</h4><p id="p-18">The extended RRBLUP is based on <a id="xref-disp-formula-6-1" class="xref-disp-formula" href="#disp-formula-6">Equation (6)</a> and the assumptions of <em>Î¼</em> being fixed, <span class="inline-formula" id="inline-formula-54"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-60.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-60.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-55"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-61.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-61.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-56"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-62.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-62.gif"/></noscript></span></span>. In this case, the solution is also given by an analog of <a id="xref-disp-formula-5-1" class="xref-disp-formula" href="#disp-formula-5">Equation (5)</a>, but with two different penalty factors, <span class="inline-formula" id="inline-formula-57"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-63.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-63.gif"/></noscript></span></span> for additive effects and <span class="inline-formula" id="inline-formula-58"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-64.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-64.gif"/></noscript></span></span> for interaction effects.</p></div></div><div id="sec-8" class="subsection"><h3>Translations of the marker coding</h3><p id="p-19">In quantitative genetics, column means are often subtracted from the original 0, 1, 2 coding of M to use <span class="inline-formula" id="inline-formula-59"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-65.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-65.gif"/></noscript></span></span> with P the vector of column means of M (<a id="xref-ref-48-3" class="xref-bibr" href="#ref-48">VanRaden 2008</a>) such that<span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-66.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-66.gif"/></noscript></span></span>However, other types of translations, for instance a symmetric <span class="inline-formula" id="inline-formula-60"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-67.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-67.gif"/></noscript></span></span> coding or approaches based on genotypic frequencies (<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Ãlvarez-Castro and Carlborg 2007</a>; <a id="xref-ref-50-2" class="xref-bibr" href="#ref-50">Vitezica <em>et al.</em> 2017</a>) can be found in quantitative geneticsâ literature. Thus, the question occurs whether this has an impact on the estimates of the marker effects or on the prediction of genetic values of genotypes.</p><p id="p-20">The answer is that for the additive setup of <a id="xref-disp-formula-1-4" class="xref-disp-formula" href="#disp-formula-1">Equation (1)</a>, a shift from M to <span class="inline-formula" id="inline-formula-61"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-68.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-68.gif"/></noscript></span></span> will change <span class="inline-formula" id="inline-formula-62"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-69.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-69.gif"/></noscript></span></span> but not <span class="inline-formula" id="inline-formula-63"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-70.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-70.gif"/></noscript></span></span> and any prediction <span class="inline-formula" id="inline-formula-64"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-71.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-71.gif"/></noscript></span></span> will not be affected, neither for OLS, nor for RRBLUP (provided that Î» is not changed) (<a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">StrandÃ©n and Christensen 2011</a>; <a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Martini <em>et al.</em> 2017</a>). This invariance of the additive model does not hold for the extended RRBLUP.</p><p id="p-21">We give an example and discuss the effect of translations of the marker coding in a more general way afterward.</p><p id="p-22"><strong>Example 1</strong> (Translations of the marker coding). <em>Let the marker data of five individuals with two markers be given</em>:<span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-72.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-72.gif"/></noscript></span></span><em>Moreover</em>, <em>let us use the original matrix</em> M, <em>and the column mean centered matrix</em> <span class="inline-formula" id="inline-formula-65"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-73.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-73.gif"/></noscript></span></span>. <em>We consider the first order epistasis model</em><span class="disp-formula" id="disp-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-74.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-74.gif"/></noscript></span></span><em>and estimate the corresponding parameters with i) an OLS regression</em>, <em>ii) a mixed model regression eRRBLUP-1 with</em> <span class="inline-formula" id="inline-formula-66"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-75.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-75.gif"/></noscript></span></span>, <em>and iii) a mixed model regression eRRBLUP-2 with</em> <span class="inline-formula" id="inline-formula-67"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-76.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-76.gif"/></noscript></span></span> <em>and</em> <span class="inline-formula" id="inline-formula-68"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-77.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-77.gif"/></noscript></span></span>. <em>The difference between eRRBLUP-1 and eRRBLUP-2 is that the first method penalizes the additive effects and the interaction effect</em>, <em>whereas the latter method only penalizes the interaction effect</em>.</p><p><em>Let</em> X <em>denote the matrix</em> M <em>with an additional column of the products of the marker values of each individual</em>. <em>Analogously</em>, <span class="inline-formula" id="inline-formula-69"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-78.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-78.gif"/></noscript></span></span> <em>shall denote the matrix</em> <span class="inline-formula" id="inline-formula-70"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-79.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-79.gif"/></noscript></span></span> <em>with the additional column of the respective products</em>.<span class="disp-formula" id="disp-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-80.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-80.gif"/></noscript></span><span class="disp-formula-label">(7)</span></span><em>To estimate the effects with OLS</em>, <em>use</em> <a id="xref-disp-formula-3-2" class="xref-disp-formula" href="#disp-formula-3">Equation (3)</a> <em>with</em> X or <span class="inline-formula" id="inline-formula-71"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-81.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-81.gif"/></noscript></span></span> <em>as marker matrix. Analogously</em>, <em>use these matrices in Equation (5) for the ridge regression approaches. The difference between eRRBLUP-1 and eRRBLUP-2 manifests only in what is added to the diagonal</em>:<span class="disp-formula" id="disp-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-82.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-82.gif"/></noscript></span><span class="disp-formula-label">(8)</span></span><em>and</em><span class="disp-formula" id="disp-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-83.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-83.gif"/></noscript></span><span class="disp-formula-label">(9)</span></span><em>For the centered coding</em>, <em>substitute</em> X <em>by</em> <span class="inline-formula" id="inline-formula-72"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-84.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-84.gif"/></noscript></span></span>. <em>We summarize our observations from the results presented in</em> <a id="xref-table-wrap-1-1" class="xref-table" href="#T1"><em>Table 1</em></a> <em>as follows:</em><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/492631/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/492631/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title"><strong>Results from Example 1. âncâ denotes the use of the non-centered matrix</strong> M <strong>and âcâ indicates the use of the centered matrix</strong> <span class="inline-formula" id="inline-formula-73"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/T1/embed/mml-math-85.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/T1/embed/mml-math-85.gif"/></noscript></span></span></span><div class="sb-div caption-clear"></div></div></div></p><ul class="list-unord " id="list-1"><li id="list-item-1"><p id="p-24"><em>Comparing the centered and non-centered versions of OLS</em>, <em>the estimates for</em> Î¼, <span class="inline-formula" id="inline-formula-79"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-91.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-91.gif"/></noscript></span></span> <em>and</em> <span class="inline-formula" id="inline-formula-80"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-92.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-92.gif"/></noscript></span></span> <em>change</em>, <em>but the estimated interaction</em> <span class="inline-formula" id="inline-formula-81"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-93.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-93.gif"/></noscript></span></span> <em>as well as the prediction of</em> y <em>remains unchanged</em>.</p></li><li id="list-item-2"><p id="p-25"><em>Comparing the centered and non-centered versions of eRRBLUP-1</em>, <em>both codings give different estimates for all the parameters and these solutions produce different predictions for</em> y.</p></li><li id="list-item-3"><p id="p-26"><em>Comparing the centered and non-centered versions of eRRBLUP-2</em>, <em>both codings give different estimates for</em> Î¼, <span class="inline-formula" id="inline-formula-82"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-94.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-94.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-83"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-95.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-95.gif"/></noscript></span></span>, <em>but the same for</em> <span class="inline-formula" id="inline-formula-84"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-96.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-96.gif"/></noscript></span></span> <em>and the same predictions for</em> y.</p></li></ul><p>The different cases presented in Example 1 have a certain systematic pattern, which we discuss in the following section.</p></div></div><div class="section results" id="sec-9"><h2 class="">Theoretical Results</h2><p id="p-27">The observations made in Example 1 are explained by the following proposition which has several interesting implications. More formal proofs of the statements made can be found in the Appendix.</p><p id="p-28"><strong>Proposition 1.</strong> <em>Let</em> <span class="inline-formula" id="inline-formula-85"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-97.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-97.gif"/></noscript></span></span> <em>be the vector of marker values of individual i and let</em> <span class="inline-formula" id="inline-formula-86"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-98.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-98.gif"/></noscript></span></span> <em>be a polynomial of total degree D in the marker data. Moreover, let</em> <span class="inline-formula" id="inline-formula-87"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-99.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-99.gif"/></noscript></span></span> <em>be a translation of the marker coding and let us define a polynomial</em> <span class="inline-formula" id="inline-formula-88"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-100.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-100.gif"/></noscript></span></span> <em>in the translated variables</em> <span class="inline-formula" id="inline-formula-89"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-101.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-101.gif"/></noscript></span></span> <em>by</em> <span class="inline-formula" id="inline-formula-90"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-102.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-102.gif"/></noscript></span></span>. <em>Then for any data</em> y, <em>the sum of squared residuals (SSR) of both polynomials will be identical (each with the respective coding):</em><span class="disp-formula" id="disp-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-103.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-103.gif"/></noscript></span></span><em>Moreover</em>, <em>for any monomial m of highest total degree D</em>, <em>the corresponding coefficient</em> <span class="inline-formula" id="inline-formula-91"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-104.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-104.gif"/></noscript></span></span> <em>of</em> <span class="inline-formula" id="inline-formula-92"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-105.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-105.gif"/></noscript></span></span> <em>and</em> <span class="inline-formula" id="inline-formula-93"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-106.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-106.gif"/></noscript></span></span> of <span class="inline-formula" id="inline-formula-94"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-107.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-107.gif"/></noscript></span></span> <em>will be identical</em>:<span class="disp-formula" id="disp-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-108.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-108.gif"/></noscript></span></span>The coefficients of highest total degree <span class="inline-formula" id="inline-formula-95"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-109.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-109.gif"/></noscript></span></span> represent the additive effects in an addditive effect model, the interaction effects in a model including pair-wise interactions, the three way interactions in a polynomial model of total degree 3, and so on.</p><p id="p-29">The content of Proposition 1 can be summarized the following way: Let us assume that we have data y and a polynomial <em>f</em> which is based on marker data M. Moreover, we have the translated data <span class="inline-formula" id="inline-formula-96"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-110.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-110.gif"/></noscript></span></span>, that is an alternative coding of the predictors. We define the alternative polynomial <span class="inline-formula" id="inline-formula-97"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-111.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-111.gif"/></noscript></span></span> by the value of <em>f</em> at the corresponding point in the original coding:<span class="disp-formula" id="disp-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-112.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-112.gif"/></noscript></span><span class="disp-formula-label">(10)</span></span>The left hand equation means that we define the alternative polynomial <span class="inline-formula" id="inline-formula-98"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-113.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-113.gif"/></noscript></span></span> to be <span class="inline-formula" id="inline-formula-99"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-114.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-114.gif"/></noscript></span></span> at <span class="inline-formula" id="inline-formula-100"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-115.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-115.gif"/></noscript></span></span>. Then âby definitionâ the SSR of the fits are identical when each polynomial is used with its respective data coding. Moreover, both polynomials give âby definitionâ the same predictions <span class="inline-formula" id="inline-formula-101"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-116.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-116.gif"/></noscript></span></span> to each data point (in its respective coding). Since <em>f</em> is given, we have its coefficients (effects) and can thus use the first equality of Equation (10) to calculate the coefficients of <span class="inline-formula" id="inline-formula-102"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-117.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-117.gif"/></noscript></span></span>. The coefficients of monomials of highest total degree are the same for <em>f</em> and <span class="inline-formula" id="inline-formula-103"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-118.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-118.gif"/></noscript></span></span>. The latter statement needs a little more detailed consideration and we refer to the Appendix. We give an example.</p><p id="p-30"><strong>Example 2</strong>. <em>In the case of an interaction model based on two loci and without additional restrictions on the coefficients</em>, <em>the set of polynomials across which we screen for an optimal fitting one is</em><span class="disp-formula" id="disp-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-119.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-119.gif"/></noscript></span></span><em>Given the vector</em> <span class="inline-formula" id="inline-formula-104"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-120.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-120.gif"/></noscript></span></span>, <em>which defines the alternative coding by <span class="inline-formula" id="inline-formula-105"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-121.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-121.gif"/></noscript></span></span></em>, <em>each f can be mapped to an <span class="inline-formula" id="inline-formula-106"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-122.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-122.gif"/></noscript></span></span> of Proposition 1 by the left-hand side of Equation (10). This equation states that</em> <span class="inline-formula" id="inline-formula-107"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-123.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-123.gif"/></noscript></span></span>, <em>which is a polynomial in the variables <span class="inline-formula" id="inline-formula-108"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-124.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-124.gif"/></noscript></span></span>, is defined by the original f when we plug in the variables <span class="inline-formula" id="inline-formula-109"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-125.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-125.gif"/></noscript></span></span> and write down the expression as a function of</em> <span class="inline-formula" id="inline-formula-110"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-126.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-126.gif"/></noscript></span></span>. <em>For an example of <span class="inline-formula" id="inline-formula-111"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-127.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-127.gif"/></noscript></span></span> being <span class="inline-formula" id="inline-formula-112"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-128.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-128.gif"/></noscript></span></span></em>, and<span class="disp-formula" id="disp-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-129.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-129.gif"/></noscript></span></span><span class="inline-formula" id="inline-formula-113"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-130.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-130.gif"/></noscript></span></span> <em>would be defined by</em><span class="disp-formula" id="disp-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-131.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-131.gif"/></noscript></span></span><em>Multiplying the factors gives</em><span class="disp-formula" id="disp-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-132.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-132.gif"/></noscript></span></span><em>We have calculated <span class="inline-formula" id="inline-formula-114"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-133.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-133.gif"/></noscript></span></span> which is a function of <span class="inline-formula" id="inline-formula-115"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-134.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-134.gif"/></noscript></span></span> from the polynomial <span class="inline-formula" id="inline-formula-116"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-135.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-135.gif"/></noscript></span></span> which is a function of <span class="inline-formula" id="inline-formula-117"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-136.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-136.gif"/></noscript></span></span>. As demonstrated in Proposition 1, both polynomials share the same coefficient for their monomial of highest total degree, that is for <span class="inline-formula" id="inline-formula-118"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-137.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-137.gif"/></noscript></span></span> or <span class="inline-formula" id="inline-formula-119"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-138.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-138.gif"/></noscript></span></span>, respectively. Moreover âdue to the way <span class="inline-formula" id="inline-formula-120"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-139.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-139.gif"/></noscript></span></span> was constructed from fâ all predictions <span class="inline-formula" id="inline-formula-121"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-140.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-140.gif"/></noscript></span></span> will be identical when the respective coding is used. For instance <span class="inline-formula" id="inline-formula-122"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-141.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-141.gif"/></noscript></span></span>. In particular, this is also true for the data points which are used to estimate the coefficients, and thus the SSR is identical for both polynomials (each with the respective coding).</em></p><p id="p-31">Provided that <span class="inline-formula" id="inline-formula-123"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-142.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-142.gif"/></noscript></span></span> of Proposition 1 is a valid fit, the statements directly imply that OLS predictions for y are invariant to translations of the coding. The reasoning is the following: If any <em>f</em> has a corresponding <span class="inline-formula" id="inline-formula-124"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-143.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-143.gif"/></noscript></span></span> which has the same predictions <span class="inline-formula" id="inline-formula-125"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-144.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-144.gif"/></noscript></span></span> and the same SSR, this is also true for the OLS solution which minimizes the SSR. To make sure that each <span class="inline-formula" id="inline-formula-126"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-145.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-145.gif"/></noscript></span></span> is a valid fit, the possibility to adapt coefficients of monomials of lower total degrees is required. We cannot adapt the regression completely if certain coefficients are forced to zero by the model structure. If a coefficient is equal to zero in <em>f</em>, it may be different from zero in <span class="inline-formula" id="inline-formula-127"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-146.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-146.gif"/></noscript></span></span>. We illustrate this with an example.</p><p id="p-32"><strong><em>Example 3</em></strong> <em>(Models without certain terms of intermediate total degree). Let us consider the data</em> M <em>and</em> y <em>of Example 1 but with the assumption that marker 2 does not have an additive effect</em>, <em>which means that we force the additive effect of marker 2 to zero by the model structure. Then the effect estimates</em><span class="disp-formula" id="disp-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-147.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-147.gif"/></noscript></span></span><em>as well as the estimates</em> <span class="inline-formula" id="inline-formula-128"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-148.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-148.gif"/></noscript></span></span> <em>and</em> <span class="inline-formula" id="inline-formula-129"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-149.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-149.gif"/></noscript></span></span> <em>are different for both codings</em>.</p><p id="p-33">Example 3 illustrates a situation in which the OLS is not invariant to the change in the marker coding. The cause of this affectedness is the lack of the additive effect <span class="inline-formula" id="inline-formula-130"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-150.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-150.gif"/></noscript></span></span> in the model. Thus, a solution <em>f</em> fitting the data for the one coding may lead to an <span class="inline-formula" id="inline-formula-131"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-151.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-151.gif"/></noscript></span></span> in which the additive effect of the second marker is non-zero. Thus <span class="inline-formula" id="inline-formula-132"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-152.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-152.gif"/></noscript></span></span> is not a valid fit. A certain âcompletenessâ of the model is required to have the possibility to adapt to translations of the coding. We define this property more precisely.</p><p id="p-34"><strong>Definition 1</strong> (Completeness of a polynomial model). <em>Let</em> <span class="inline-formula" id="inline-formula-133"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-153.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-153.gif"/></noscript></span></span> <em>be the vector of the marker values of individual</em> i and let <span class="inline-formula" id="inline-formula-134"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-154.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-154.gif"/></noscript></span></span> <em>be a polynomial of total degree D in the marker data. The polynomial model f is called complete if for any monomial</em> <span class="inline-formula" id="inline-formula-135"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-155.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-155.gif"/></noscript></span></span> <em>of f</em>, <em>all monomials</em><span class="disp-formula" id="disp-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-156.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-156.gif"/></noscript></span></span><em>are included with a coefficient to be estimated.</em></p><p id="p-35">Definition 1 states that for each monomial which is included in the model, all âsmallerâ monomials have to be included as well. We illustrate this with some examples. Let us consider <a id="xref-disp-formula-6-2" class="xref-disp-formula" href="#disp-formula-6">Equation (6)</a>. Its monomials are of shape <span class="inline-formula" id="inline-formula-136"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-157.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-157.gif"/></noscript></span></span> or <span class="inline-formula" id="inline-formula-137"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-158.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-158.gif"/></noscript></span></span>. For <span class="inline-formula" id="inline-formula-138"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-159.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-159.gif"/></noscript></span></span>, Definition 1 states that <span class="inline-formula" id="inline-formula-139"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-160.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-160.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-140"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-161.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-161.gif"/></noscript></span></span> have to be included, which is obviously the case. For <span class="inline-formula" id="inline-formula-141"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-162.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-162.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-142"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-163.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-163.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-143"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-164.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-164.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-144"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-165.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-165.gif"/></noscript></span></span> have to be included, which is also true. Thus, the model is complete. Analogously, if we also include the interactions <span class="inline-formula" id="inline-formula-145"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-166.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-166.gif"/></noscript></span></span>, that is if we allow <span class="inline-formula" id="inline-formula-146"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-167.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-167.gif"/></noscript></span></span>, the model remains complete since all smaller monomials are included. Contrarily, Example 3 is based on the model<span class="disp-formula" id="disp-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-168.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-168.gif"/></noscript></span></span>Since <span class="inline-formula" id="inline-formula-147"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-169.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-169.gif"/></noscript></span></span> is included with a coefficient to be estimated, <span class="inline-formula" id="inline-formula-148"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-170.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-170.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-149"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-171.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-171.gif"/></noscript></span></span> have to be included to make the model complete. Since <span class="inline-formula" id="inline-formula-150"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-172.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-172.gif"/></noscript></span></span> is not included, the polynomial is not complete.</p><p id="p-36">Given that the model is complete and thus allowing an adaption from <em>f</em> to <span class="inline-formula" id="inline-formula-151"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-173.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-173.gif"/></noscript></span></span>, Proposition 1 has various implications. The following corollaries explain the results observed in our examples and highlight some additional properties of penalized regression methods in general. For all statements, it is assumed that penalty factors remain unchanged and that the model is complete.</p><p id="p-37"><strong>Corollary 1.</strong> <em>For a complete polynomial model of total degree D</em>, <em>the OLS estimates of the coefficients of highest total degree as well as the predictions <span class="inline-formula" id="inline-formula-152"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-174.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-174.gif"/></noscript></span></span> are invariant with respect to translations of the marker coding.</em></p><p id="p-38">Corollary 1 is a result of the OLS method being defined only by the SSR, and <em>f</em> and the corresponding <span class="inline-formula" id="inline-formula-153"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-175.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-175.gif"/></noscript></span></span> of Proposition 1 fitting the data with the same SSR and with the same prediction <span class="inline-formula" id="inline-formula-154"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-176.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-176.gif"/></noscript></span></span> when their respective coding is used. The statement of Corollary 1 has been observed in Example 1, where the OLS fits for <span class="inline-formula" id="inline-formula-155"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-177.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-177.gif"/></noscript></span></span> are identical when the coding is translated, and where the estimated coefficients <span class="inline-formula" id="inline-formula-156"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-178.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-178.gif"/></noscript></span></span> of highest total degree remain unchanged.</p><p id="p-39"><strong>Corollary 2.</strong> <em>For a complete polynomial model of total degree D</em>, <em>and a penalized regression which only penalizes the coefficients of monomials of highest total degree</em>, <em>the estimates of the coefficients of monomials of highest total degree</em>, <em>as well as the predictions <span class="inline-formula" id="inline-formula-157"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-179.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-179.gif"/></noscript></span></span> are invariant with respect to translations of the marker coding.</em></p><p id="p-40">Corollary 2 is a result of the following observation: for each <em>f</em>, its corresponding <span class="inline-formula" id="inline-formula-158"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-180.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-180.gif"/></noscript></span></span> will have the same SSR (each polynomial with its respective coding), and the same coefficients of highest total degree. Thus, it will have the same value for the target function which we aim to minimize (The target function is the analog of <a id="xref-disp-formula-4-3" class="xref-disp-formula" href="#disp-formula-4">Equation (4)</a> with the corresponding interactions and with a penalty on only the coefficients of monomials of highest total degree). Because this is true for any polynomial <em>f</em>, it is in particular true for the solution minimizing the target function. A central point of Corollary 2 is that it is valid for any penalty on the size of the estimated coefficients of highest total degree. The sufficient condition is that only these coefficients of highest total degree are penalized.</p><p id="p-41"><strong>Corollary 3.</strong> <em>RRBLUP predictions <span class="inline-formula" id="inline-formula-159"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-181.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-181.gif"/></noscript></span></span> are invariant with respect to translations of the marker coding.</em></p><p id="p-42">Corollary 2 applied to complete models of total degree 1 gives the result of Corollary 3, that is RRBLUP being invariant to translations of the marker coding. This fact has been previously proven using a marginal likelihood setup (<a id="xref-ref-44-3" class="xref-bibr" href="#ref-44">StrandÃ©n and Christensen 2011</a>), or the mixed model equations (<a id="xref-ref-31-3" class="xref-bibr" href="#ref-31">Martini <em>et al.</em> 2017</a>).</p><p id="p-43"><strong>Corollary 4.</strong> <em>An additive least absolute shrinkage and selection operator (LASSO) regression (</em><a id="xref-ref-47-1" class="xref-bibr" href="#ref-47"><em>Tibshirani 1996</em></a><em>) based on a polynomial model of total degree 1 and <span class="inline-formula" id="inline-formula-160"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-182.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-182.gif"/></noscript></span></span> penalizing the additive marker effects but not the intercept, is invariant to translations of the marker coding.</em></p><p id="p-44">Corollary 4 is a special case of Corollary 2.</p><p id="p-45">Before we illustrate the impact of marker coding on estimated effect sizes with a publicly available data set, we give a small example, highlighting cases which are not invariant to translations of the marker coding.</p><p id="p-46"><strong>Example 4</strong> (Regressions affected by marker coding).</p><ol class="list-alphalower " id="list-2"><li id="list-item-4"><p id="p-47"><em>Whereas, RRBLUP with the fixed intercept is invariant to translations (<a id="xref-ref-44-4" class="xref-bibr" href="#ref-44">StrandÃ©n and Christensen 2011</a>), pure ridge regression of an additive model of <a id="xref-disp-formula-1-5" class="xref-disp-formula" href="#disp-formula-1">Equation (1)</a> with a penalty on the size of Î¼ (ârandom interceptâ) is not invariant to translations.</em></p></li><li id="list-item-5"><p id="p-48"><em>RRBLUP without intercept is not invariant to translations of the marker coding.</em></p></li><li id="list-item-6"><p id="p-49"><em>An extended LASSO <span class="inline-formula" id="inline-formula-161"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-183.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-183.gif"/></noscript></span></span> penalizing additive effects and interactions is not invariant to translations of the coding.</em></p></li></ol><p id="p-50"><strong>Remark 1.</strong> <em>Proposition 1 stated that the coefficients of monomials of highest total degree D of f and</em> <span class="inline-formula" id="inline-formula-162"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-184.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-184.gif"/></noscript></span></span> <em>will be identical</em>. <em>This statement can even be generalized. Consider for instance the model</em><span class="disp-formula" id="disp-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-185.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-185.gif"/></noscript></span></span><em>The model is a polynomial f of total degree 2. Thus, Proposition 1 states that the coefficient of monomial <span class="inline-formula" id="inline-formula-163"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-186.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-186.gif"/></noscript></span></span> will be identical for f and <span class="inline-formula" id="inline-formula-164"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-187.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-187.gif"/></noscript></span></span>. However, since <span class="inline-formula" id="inline-formula-165"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-188.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-188.gif"/></noscript></span></span> is not included in any other monomial, its coefficient will also be identical for both polynomials. We did not generalize Proposition 1 into this direction to make the manuscript not more technical than necessary. The statement we made in Proposition 1 is sufficient to explain the observations related to genomic prediction models.</em></p></div><div class="section other1" id="sec-10"><h2 class="">Practical Implications: An Example With A Wheat Data Set</h2><p id="p-51">We illustrated by theoretical considerations and small examples that penalized polynomial regression is in many cases affected by translations of the marker coding. An important exception is the case in which only coefficients of monomials of highest total degree are penalized. To illustrate the differences in estimated effect sizes that may be expected with real data, we compare the estimated interaction effects for different codings on a publicly available wheat data set (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Crossa <em>et al.</em> 2010</a>). Moreover, we assess the impact of a changed coding on explained variance and out-of-sample predictions.</p><div id="sec-11" class="subsection"><h3>Data and method</h3><div id="sec-12" class="subsection"><h4>Data:</h4><p id="p-52">We use a well investigated wheat data set providing the state of 1279 presence/absence markers of 599 genotyped wheat lines together with records on their yield when grown in four different environments. The yield measurements are standardized to mean 0 and variance 1 (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Crossa <em>et al.</em> 2010</a>). The provided coding of the marker data are a <span class="inline-formula" id="inline-formula-166"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-189.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-189.gif"/></noscript></span></span> coding. For more details on the data see <a id="xref-ref-7-3" class="xref-bibr" href="#ref-7">Crossa <em>et al.</em> (2010)</a> or the R (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">R Core Team 2016</a>) package BGLR (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">de los Campos and Perez Rodriguez 2016</a>).</p></div><div id="sec-13" class="subsection"><h4>Codings compared:</h4><p id="p-53">We compare three different codings: The originally provided <span class="inline-formula" id="inline-formula-167"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-190.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-190.gif"/></noscript></span></span> coding, a version translated by <span class="inline-formula" id="inline-formula-168"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-191.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-191.gif"/></noscript></span></span>, that is a symmetric <span class="inline-formula" id="inline-formula-169"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-192.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-192.gif"/></noscript></span></span> coding, and a coding in which the mean of each column is subtracted (<a id="xref-ref-48-4" class="xref-bibr" href="#ref-48">VanRaden 2008</a>). We refer to these codings as the <em>original</em> coding, the <em>symmetric</em> coding and the <em>centered</em> coding.</p></div><div id="sec-14" class="subsection"><h4>Interaction effect estimates under different codings and varying number of markers:</h4><p id="p-54">The assessment of the practical impact of translations of the marker coding on the effect estimates is difficult. Since in practice, the variance components and consequently the penalty factors are estimated on the data, the translations of the marker coding may have an additional indirect effect of changing the penalty factors. Also there may be rounding effects impacting the variance component estimates when the entries of the corresponding matrix are too big, and there may be numerical issues related to the matrix inversion when solving for the effects (analogous to <a id="xref-disp-formula-5-2" class="xref-disp-formula" href="#disp-formula-5">Equation (5)</a>). A high dimension or a small determinant of the matrix to be inverted can cause numerical unprecisions which may impact the results. If possible these superposed effects should be separated from each other. For this reason, we follow an approach of reducing the number of variables and estimating the variance components only once and then fixing the penalty factors.</p><p id="p-55">For the considered data set with 1279 markers we deal with 817281 interaction effects when the full model with all pairwise interactions is used. However, if we reduce the number of markers below the number of individuals (599), we can estimate the additive effects as fixed effects and penalize only their interactions. We restrict our considerations to models including 50, 100 or 150 markers and their 1225, 4950 or 11175 interactions, respectively.</p><p id="p-56">For each environment we choose randomly 50 (100, 150) markers using the sample function of R. The eRRBLUP model which we apply afterward includes the fixed effect <em>Î¼</em>, the 50 (100, 150) additive effects and their 1225 (4950, 11175) interactions. The results reported will be based on 50 repeated random draws of the corresponding number of markers. Moreover, we estimate the variance components only for the column mean centered coding and use the corresponding penalty factors also for the estimation with other codings. This is analogous to the fact that the translational invariance of RRBLUP holds when the penalty factor remains fixed (which should be the case when <em>restricted maximum likelihood</em> is used for variance component estimation). For the estimation of the variance components, we use the regress package (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Clifford <em>et al.</em> 2014</a>). The resulting three variance components define the penalty factors. The effects are estimated by the extension of <a id="xref-disp-formula-5-3" class="xref-disp-formula" href="#disp-formula-5">Equation (5)</a> with two different penalty factors for additive effects and interactions. For each of the environments, we compare the Pearson correlation of the estimated interaction effects for the three different codings for 50 randomly drawn sets of markers.</p></div><div id="sec-15" class="subsection"><h4>Interaction effect estimates and changes in training set size:</h4><p id="p-57">The correlation of the interaction effect estimates of eRRBLUP with different codings may not only depend on the number of interactions included, but also on the number of data points provided by y, which means here the number of lines. To compare the effect of an increase in the number of markers to the effect of a reduction of lines used to estimate the interactions, we also compare the effect estimates of different codings, when the number of lines is reduced to 300 or 200. For both sizes, 50 randomly and independently drawn training and marker sets are the basis for interaction effect estimates and their Pearson correlations. The impact of the reduction of lines is only evaluated in the scenario including 50 markers and their interactions.</p></div><div id="sec-16" class="subsection"><h4>Regressions compared:</h4><p id="p-58">In a first scenario, the size of <em>Î¼</em> is not penalized, but the sizes of additive effects and of interactions are. Here, the three variance components <span class="inline-formula" id="inline-formula-170"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-193.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-193.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-171"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-194.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-194.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-172"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-195.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-195.gif"/></noscript></span></span> are estimated which define the penalty factors <span class="inline-formula" id="inline-formula-173"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-196.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-196.gif"/></noscript></span></span> for additive effects and <span class="inline-formula" id="inline-formula-174"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-197.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-197.gif"/></noscript></span></span> for the interactions. In a second setup, we only penalize the interaction effects. Here, the additive effects are modeled as being fixed (<span class="inline-formula" id="inline-formula-175"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-198.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-198.gif"/></noscript></span></span>) and thus, the two variance components <span class="inline-formula" id="inline-formula-176"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-199.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-199.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-177"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-200.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-200.gif"/></noscript></span></span> are estimated and define the penalty factor <span class="inline-formula" id="inline-formula-178"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-201.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-201.gif"/></noscript></span></span> for the interactions. This regression approach is a practical application of Corollary 2 which makes the predictions of genetic values and interaction effects independent from the choice of coding. Thus, from a theoretical perspective, it is clear that the estimates should be identical for the different codings. Yet, we use this second scenario to asses the impact of computational issues âfor instance related to the inversion of the corresponding high-dimensional matrix when solving for the interaction effectsâ on the estimates of interactions.</p></div><div id="sec-17" class="subsection"><h4>Determining (un)explained phenotypic variance:</h4><p id="p-59">To check the influence of the coding on how much phenotypic variance can be explained by the interaction effects, we estimate the variance components on the full data set with all markers and all lines. Since some variance may be attributed ârandomlyâ to additive or epistatic variance, when both corresponding covariance matrices are too similar, we focus on a model with only interaction effects and consider the variance component of the residual, that is the variance which is not explained. The residual variance is more comparable, since the different epistatic relationship matrices may be scaled differently which directly translates to the estimated variance components. We use the equation<span class="disp-formula" id="disp-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-202.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-202.gif"/></noscript></span><span class="disp-formula-label">(11)</span></span>to calculate the epistatic relationship for all 599 lines (<a id="xref-ref-32-4" class="xref-bibr" href="#ref-32">Martini <em>et al.</em> 2016</a>) and divide by the maximum of H. The regress package (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Clifford <em>et al.</em> 2014</a>) is used and the residual variance is then compared between different codings. The symbol â denotes Hadamard product, that is the entry-wise multiplication.</p></div><div id="sec-18" class="subsection"><h4>Out of sample predictions with varying number of markers and training set size:</h4><p id="p-60">To assess the impact of translations of the marker coding on out-of-sample predictions <span class="inline-formula" id="inline-formula-179"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-203.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-203.gif"/></noscript></span></span>, we randomly draw test sets of 60 out of the 599 lines to be predicted by the additive and interaction effects estimated from the training set consisting of the remaining 539 lines. We compare the Pearson correlations between the predictions <span class="inline-formula" id="inline-formula-180"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-204.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-204.gif"/></noscript></span></span> for different codings and with the known phenotypes <span class="inline-formula" id="inline-formula-181"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-205.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-205.gif"/></noscript></span></span> (âpredictive abilityâ). We do this for models including 50, 100 or 150 marker and their interactions. Moreover, we compare the predictions of a model with 50 markers, when the training set size is reduced. Since the variance of the results is increased due to sampling of markers and training and test sets, we use 200 repetitions with independently drawn marker, training and test sets.</p></div></div><div id="sec-19" class="subsection"><h3>Data Availability</h3><p id="p-61">The authors affirm that all data necessary for confirming the conclusions of the article are present within the article, figures, and tables or publicly available (<a id="xref-ref-7-4" class="xref-bibr" href="#ref-7">Crossa <em>et al.</em> 2010</a>).</p></div><div id="sec-20" class="subsection"><h3>Results</h3><div id="sec-21" class="subsection"><h4>Correlation of interaction effect estimates under different codings and varying number of variables included in the model:</h4><p id="p-62">We use a model with 50, 100 or 150 markers and their interaction effects and compare the Pearson correlations of the interaction estimates obtained with different codings. The results are summarized in <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>. The values indicate the mean correlation of 50 randomly drawn marker sets. The correlations are between 0.80 and 0.95 and thus relatively high, but not equal to 1. Moreover, for each environment, and for each comparison of two codings, the correlation of the estimates reduces when the number of markers increases. For instance, an obtained mean correlation of the interaction effect estimates using the originally provided <span class="inline-formula" id="inline-formula-182"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-206.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-206.gif"/></noscript></span></span> coding and the allele-frequency centered coding is 0.90 (environment 2; 50 randomly selected markers). For a model with 100 markers, this correlation reduces to 0.85, and to 0.80 when using 150 markers. This stepwise reduction of the correlation of the estimates when the number of markers increases, can be observed for any pairwise comparison of two codings, and any underlying data (grown in environment <span class="inline-formula" id="inline-formula-183"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-207.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-207.gif"/></noscript></span></span>).</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/492719/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/492719/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title"><strong>Pearson correlation of the estimates of the 1225 (4950, 11175) interactions when different marker codings are used. The numbers represent the mean correlation of 50 repetitions with independently, uniformly drawn markers. The standard error of the estimate was in all cases below 0.004. Colors indicate the underlying data: Environment 1, 2, 3 or 4.</strong></span><div class="sb-div caption-clear"></div></div></div><p id="p-63">Another observation across environments and varying number of markers is that the original <span class="inline-formula" id="inline-formula-184"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-208.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-208.gif"/></noscript></span></span> coding and the column mean centered coding are the most different. For instance, for environment 2 and 150 marker, the mean correlation of the interaction effect estimates is 0.80, which is smaller than the corresponding correlation for original and symmetric coding (0.88) or for symmetric and centered coding (0.88).</p><p id="p-64">For the second type of regression, in which only the size of the interactions are penalized, we receive âas stated by our theoretical resultsâ a correlation of 1 between any two codings, for each of the four underlying environmental conditions and for each number of considered markers. This circumstance also illustrates that potential numerical problems âfor instance related to the inversion of the high-dimensional matrix of Eq.(5)â do not strongly occur with the number of interactions modeled in our examples.</p></div><div id="sec-22" class="subsection"><h4>Ranks of interactions effects:</h4><p id="p-65">The previously described observations have been made on the level of correlations of interaction effect estimates. To address the question of what they mean for an individual interaction effect estimate, we consider an example of using the 150 markers with largest additive effects (<a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">KÃ¤rkkÃ¤inen <em>et al.</em> 2015</a>). We are interested in changes of interaction effect sizes and in their relative importance. We rank the estimated interactions according to their absolute values and compare these rankings for the different codings. The maximal observed rank change of an estimate is 10367 for the data of environment 1, with an interaction that is the 362<sup>th</sup> highest when the original <span class="inline-formula" id="inline-formula-185"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-209.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-209.gif"/></noscript></span></span> coding is used, but with a rank of 10729 with the centered coding. Analogously, the maximal observed rank changes are for environments 2 to 4, 10775, 10672 and 10750, respectively. All maximal rank changes are observed comparing the original and the centered coding. Recall here that there are only 11175 interactions considered. The maximal theoretically possible rank change would thus be 11174. The relative importance of these interactions changes from the top <span class="inline-formula" id="inline-formula-186"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-210.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-210.gif"/></noscript></span></span> to the <span class="inline-formula" id="inline-formula-187"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-211.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-211.gif"/></noscript></span></span> most unimportant interactions. Their effects are lost when we translate from <span class="inline-formula" id="inline-formula-188"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-212.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-212.gif"/></noscript></span></span> to the centered coding.</p></div><div id="sec-23" class="subsection"><h4>Reducing training set size:</h4><p id="p-66">The increasing influence of the choice of coding on interaction effect estimates when the number of interactions is increasing, may have a counterpart when the training set size is decreasing. We use a model including 50 randomly chosen markers and their interactions and reduce the training set size from 539 to 300 or to 200. <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a> presents the mean Pearson correlations of the interaction estimates based on 50 randomly drawn marker and training sets (of 200 or 300 lines, respectively). Here, the reduction in correlation has a similar pattern to the situation in which the number of markers in increasing. However, comparing the situation of 150 markers and 599 lines (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>) to the situation of 50 markers and 200 lines (<a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>), we see that the impact of coding on interaction effect estimates is more strongly affected by the reduction in training set size than by the increase in the number of markers. For instance, the lowest correlation in <a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a> is 0.80 for 150 markers and the data of environment 2. In <a id="xref-table-wrap-3-3" class="xref-table" href="#T3">Table 3</a>, the correlation with a test set size of 200 is 0.68. In particular, this shows that the impact does not only depend on the ratio of training set size and markers.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/492752/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/492752/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title"><strong>Pearson correlation of the estimates of the 1225 interaction effects when different marker codings are used and the number of genotypes used to estimate the effects is reduced from the 599 available lines to 300 or 200, respectively. Only the model with 50 markers and 1225 interactions is considered. The numbers represent the mean correlation of 50 repetitions with independently, uniformly drawn markers and lines. The standard error of the estimate was in all cases below 0.025. Colors indicate the underlying data: Environment 1, 2, 3 or 4.</strong></span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-24" class="subsection"><h4>(Un)explained variance:</h4><p id="p-67">To illustrate which impact the coding may have on the phenotypic variance an epistatic relationship matrix can capture, we use only the epistatic relationship to estimate a corresponding variance component and a residual variance. In a model with additive and epistatic relationship, some of the phenotypic variance may be explained by either one or the other. Thus, if the relationship matrices are not very different, small details on the phenotypic data can determine whether the variance is more attributed to the additive or to the epistatic part. If we only use the epistatic part, we can better highlight the changes in the phenotypic variance that may be explained by the epistatic relationship in the respective coding. The results are illustrated in <a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>. Whereas for the additive matrix the residual variance estimate is constant across the three different codings (not shown), the estimated residual variance varies strongly when the epistatic relationship matrix is constructed with different codings. The range of residual variance is between 0.33 and 0.42, 0.46 and 0.53, 0.35 and 0.54, and 0.38 and 0.48, for environments <span class="inline-formula" id="inline-formula-189"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-213.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-213.gif"/></noscript></span></span>, respectively. On average, the symmetric coding shows the lowest unexplained variance.</p><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/492632/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/492632/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title"><strong>Unexplained variance: Residual variance</strong> <span class="inline-formula" id="inline-formula-190"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/T4/embed/mml-math-214.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/T4/embed/mml-math-214.gif"/></noscript></span></span> <strong>estimated with epistatic relationships based on the three different codings and for the four different environments. The standard error returned by regress() was for each estimated residual variance between 0.045 and 0.064.</strong></span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-25" class="subsection"><h4>Out-of-sample predictions under varying number of variables included in the model:</h4><p id="p-68">Analogously to the experiment of increasing the number of variables in the model and comparing the interaction effect estimates, we also compare âout-of-sampleâ predictions. As previously described, eRRBLUP models based on 50, 100 or 150 markers are used to predict a test set of 60 lines by the effects estimated from the remaining 539 lines. The results for 200 randomly drawn test and marker sets are summarized in <a id="xref-table-wrap-5-1" class="xref-table" href="#T5">Table 5</a>. An (expected) observation is that the predictive ability increases with the number of markers. For instance for environment 1, the predictive ability increases from 0.44 to 0.49 for the original coding, and from 0.42 to 0.49 for the symmetric coding. This observation is consistent across all coding-environment combinations.</p><div id="T5" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/492695/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/492695/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 5</span> <span class="caption-title"><strong>Pearson correlation of predictions of a test set consisting of 60 lines when predicted by additive and interaction effect estimates based on a model including 50, 100 or 150 markers and their pairwise interaction effects. The numbers represent the means of 200 repetitions with independently, uniformly drawn marker and test sets. The training set is given by the remaining 539 lines. The standard error of the estimate was in all cases smaller than 0.01. Colors indicate the underlying data: Environment 1, 2, 3 or 4.</strong></span><div class="sb-div caption-clear"></div></div></div><p id="p-69">Moreover, contrarily to the observation we have made for the interaction effect estimates for which the correlations reduce when more markers are included into the model, the correlations of âout-of-sampleâ predictions <span class="inline-formula" id="inline-formula-191"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-215.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-215.gif"/></noscript></span></span> do not show this pattern. The results rather suggest that there is an increase in the correlation when more markers are included. For instance, with the data of environment 1, the correlation between <span class="inline-formula" id="inline-formula-192"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-216.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-216.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-193"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-217.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-217.gif"/></noscript></span></span> increases from 0.69 to 0.79 from 50 to 150 markers. This observation is not consistent across all coding-environment combinations, but an increasing correlation with the number of markers is the more prevalent case. Overall, we see that in 7 out of 12 cases, an increasing number of markers leads to an increase in correlation between different predictions. In one case â<span class="inline-formula" id="inline-formula-194"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-218.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-218.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-195"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-219.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-219.gif"/></noscript></span></span> for environment 1â no change is observed, and in four cases, the correlation of the predictions is reduced.</p></div><div id="sec-26" class="subsection"><h4>Out-of-sample predictions under varying training set size:</h4><p id="p-70">Analogously to having reduced the training set size when we have investigated the correlation of interaction effect estimates, we also perform a similar experiment for âout-of-sampleâ prediction. The results are summarized in <a id="xref-table-wrap-6-1" class="xref-table" href="#T6">Table 6</a>. An obvious and clear pattern is the reduction in predictive ability when the training set size is reduced for each coding-environment combination. Similar to the situation of increasing the number of interactions modeled, there is no clear pattern visible for the correlations of predictions obtained with different codings.</p><div id="T6" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/492686/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/492686/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 6</span> <span class="caption-title"><strong>Pearson correlation of predictions of a test set consisting of 60 lines when predicted by additive and interaction effect estimates based on a model including 50 randomly chosen markers and their pairwise interaction effects. The numbers represent the means of 200 repetitions with independently, uniformly drawn marker and test and training sets (The latter of size 539, 300 or 200). The standard error was in all cases smaller than 0.022. Colors indicate the underlying data: Environment 1, 2, 3 or 4. The case of a training set size of 539 is the case of 50 markers in</strong> <a id="xref-table-wrap-5-2" class="xref-table" href="#T5"><strong>Table 5</strong></a>.</span><div class="sb-div caption-clear"></div></div></div></div></div></div><div class="section discussion" id="sec-27"><h2 class="">Discussion</h2><div id="sec-28" class="subsection"><h3>Interpretability of effect estimates</h3><p id="p-71">The illustrated problem of the coding having an impact on the estimates of interactions in penalized regressions is essential for quantitative genetics, where Hadamard products are often used to model interaction such as epistasis or gene by environment interaction (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">PÃ©rez-RodrÃ­guez <em>et al.</em> 2017</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">De Coninck <em>et al.</em> 2016</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Shang <em>et al.</em> 2015</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Sukumaran <em>et al.</em> 2017</a>). Hadamard products of covariance matrices represent exact reformulations of certain interaction effect models (<a id="xref-ref-29-3" class="xref-bibr" href="#ref-29">Jiang and Reif 2015</a>; <a id="xref-ref-32-5" class="xref-bibr" href="#ref-32">Martini <em>et al.</em> 2016</a>).</p><p id="p-72">In particular, our observations illustrate once more that the size of interaction effect estimates obtained from WGR should be interpreted with caution because a biological meaning is not necessarily given. On the one hand the data structure (<em>e.g.</em>, population stratification), and on the other hand the coding may influence effect estimates. In models penalizing interactions and additive effects, the coding issue alone can have a drastic impact on the interaction estimates as illustrated by the consideration of the ranks of interaction effect. Some interactions were among the 4% with the highest absolute interaction effect with one coding and among the 4% with the lowest absolute effects with another coding. Moreover, as illustrated in <a id="xref-table-wrap-2-4" class="xref-table" href="#T2">Table 2</a>, the impact of variable coding on interaction effect estimates, increases with the number of modeled interactions. This observation may not be surprising, since with more interactions there are more changes when the coding is altered, and a higher number of variables also provides more flexibility to model certain effects by other interactions. However, this circumstance illustrates that the supposed advantage of an increased interpretability when modeling epistasis effects explicitly instead of âhiddenâ in an RKHS approach, may only be marginal. This statement does not doubt that the overall predictive ability for y may be increased when epistasis models are used (which has been demonstrated for instance by <a id="xref-ref-37-3" class="xref-bibr" href="#ref-37">Ober <em>et al.</em> (2015)</a>, <a id="xref-ref-29-4" class="xref-bibr" href="#ref-29">Jiang and Reif (2015)</a>, <a id="xref-ref-32-6" class="xref-bibr" href="#ref-32">Martini <em>et al.</em> (2016)</a>). In particular, interaction effects estimates have been used to select important interactions and thus improving predictive ability for different environment conditions by reducing the model to the âmore relevantâ interactions (<a id="xref-ref-32-7" class="xref-bibr" href="#ref-32">Martini <em>et al.</em> 2016</a>). However, the biological meaning of individual interaction effects obtained from an epistatic WGR is limited (at currently used training set sizes). In this regard, Corollary 2 illustrated that approaches selecting markers first and then modeling the interactions between them (<a id="xref-ref-30-3" class="xref-bibr" href="#ref-30">KÃ¤rkkÃ¤inen <em>et al.</em> 2015</a>) may provide the option to model additive effects as being fixed and to only penalize interactions effects, thus at least eliminating the coding problem and potentially facilitating the attempt to assign a biological meaning to estimated quantities. However, this raises the question of how to select the variables. Using the additive effect size as a selection criterion might for instance have the danger of missing relevant interactions, since some pairs of loci could have small additive effects but potentially large interaction effects (Mackay and Moore 2014).</p></div><div id="sec-29" class="subsection"><h3>Impact of coding on out-of-sample predictions</h3><p id="p-73">Interestingly, the observed effect of a decreasing correlation of interaction effect estimates when the number of markers is increasing cannot be observed for the correlation of out-of-sample predictions <span class="inline-formula" id="inline-formula-196"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-220.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-220.gif"/></noscript></span></span>. The different behavior of the correlations of interaction effect estimates (<a id="xref-table-wrap-2-5" class="xref-table" href="#T2">Table 2</a>) and out-of-sample predictions (<a id="xref-table-wrap-5-3" class="xref-table" href="#T5">Table 5</a>) illustrates that different WGRs may model the sum of their effects similarly, but not necessarily each summand. The estimates from a global, joint consideration of all markers will give a good description of the sum of the effects, but not necessarily of each effect. To increase the biological meaning of interaction effect estimates obtained from epistatic WGR, training set sizes will have to increase drastically to make the data more and the prior assumption less important.</p></div><div id="sec-30" class="subsection"><h3>Is there a better and a worse coding?</h3><p id="p-74">Out of the three codings compared here, the symmetric coding explains the phenotypic variation best on the considered data set (<a id="xref-table-wrap-4-2" class="xref-table" href="#T4">Table 4</a>). In the case of using only the epistatic relationship, this has also been reflected by showing the highest predictive ability when all markers were used (<a id="xref-ref-31-4" class="xref-bibr" href="#ref-31">Martini <em>et al.</em> 2017</a>). Moreover, <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Santantonio <em>et al.</em> (2018)</a> also observed a slightly improved predictive ability for the symmetric coding on a different data set. In this work, in which we modeled additive and epistatic effects, and restricted the number of markers, no clear superiority of one or the other coding was observed. The results for the case of 150 markers presented in <a id="xref-table-wrap-5-4" class="xref-table" href="#T5">Table 5</a>, might be interpreted as a sign that for a higher number of markers, the symmetric coding leads to the higher predictive ability. However, these observations do not provide a theoretical explanation for why a certain coding should be better than the other. An earlier theoretical explanation was that the symmetric coding has the advantage of being independent of the choice of the reference allele. Since other codings are affected by the choice of which allele is set as the reference, additional uncertainties may be introduced. The symmetric coding does not have this additional problem, since changing the reference allele does not have an impact on the size of the effect estimate, but only on its sign (<a id="xref-ref-31-5" class="xref-bibr" href="#ref-31">Martini <em>et al.</em> 2017</a>).</p><p id="p-75">What certainly has an impact on predictive ability, when several covariance matrices are used, is how âsimilarâ the matrices are. A good coding should make the covariance matrices more âdifferentâ to allow them to capture different components of the phenotypic variance. It is important to note here that this question may sound similar to the question of orthogonal effect estimate coding, but the level aimed at is different. Applying the coding proposed by <a id="xref-ref-50-3" class="xref-bibr" href="#ref-50">Vitezica <em>et al.</em> (2017)</a> does not necessarily make the additive G and the epistatic relationship <span class="inline-formula" id="inline-formula-197"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-221.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-221.gif"/></noscript></span></span> orthogonal to each other.</p><p id="p-76">In this regard, we also highlight that we investigated the importance of coding with respect to the method applied. In particular, the OLS approach applied to a model with interactions, but also a penalized regression only penalizing the effect sizes of monomials of highest total degree, will both provide predictions <span class="inline-formula" id="inline-formula-198"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-222.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-222.gif"/></noscript></span></span>, as well as estimated effect sizes of monomials of highest total degree, which are independent of the coding. For these methods, these estimates will not depend on whether an âorthogonalâ coding is used. We did not consider how estimates of dominance effects may relate to the estimates of additive effects or other analogous relations. Contrarily, the topic of orthogonal estimates has been discussed independently of the method applied afterward, but more in the context of how the different effects relate to each other. Thus, both discussions do not coincide.</p></div><div id="sec-31" class="subsection"><h3>Epistatic effect models and the Gaussian kernel</h3><p id="p-77">Finally, note that it has been reported that a Gaussian reproducing kernel Hilbert space regression (<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Morota and Gianola 2014</a>) can be interpreted as a limit of a penalized polynomial regression with increasing total degree (and all possible monomials) (<a id="xref-ref-29-5" class="xref-bibr" href="#ref-29">Jiang and Reif 2015</a>). Being a limit case of a method which is affected by translations of the coding, the question appears why the Gaussian kernel regression is invariant to translations of the marker coding (The invariance of the Gaussian kernel is a direct consequence of being defined on the Euclidean distance. If two genotypes are translated in the same way, their distance remains unchanged). It may be interesting from a theoretical point of view to reconsider the limit behavior of polynomial regression.</p></div><div id="sec-32" class="subsection"><h3>Summary</h3><p id="p-78">We identified the cause of the coding-dependent performance of epistasis effect models. Our results were motivated by ridge regression, but do equally hold for many other types of penalized regressions, for instance for the <span class="inline-formula" id="inline-formula-199"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-223.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-223.gif"/></noscript></span></span> penalized LASSO. The fact that the estimated effect sizes depend on the coding highlights once more that estimated interaction effect sizes should be interpreted with caution with regard to their biological, mechanistic meaning. In particular, the supposed advantage of a facilitated interpretability compared to RKHS methods may not be given when epistatic whole genome regressions are used. Moreover, the problem of coding is not only present for marker by marker interaction, but for any mixed model in which interactions are modeled by Hadamard products of covariance matrices, in particular also for gene by environment (G x E) models.</p></div><div id="sec-33" class="subsection"><h3>Outlook</h3><p id="p-79">The work on hand only addressed coding translations, but not scaling of markers. It is clear that scaling has an impact on effect estimates due to changing the penalty factors individually. How to scale markers optimally in an additive effect model is not completely understood and this investigation may also be extended to the situation of epistasis models. Moreover, the question on how to make the additive and interaction effect matrix most different should be addressed in the future.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-85">JWRM thanks KWS SAAT SE as well as the German Research Foundation (DFG) via the research training group 1644 âScaling Problems in Statisticsâ for support during his PhD thesis.</p></div><div class="section app" id="app-1"><h2 class="">Appendix: Proofs</h2><p id="p-80"><em>Proposition 1</em>. The fact that the SSR remains the same, results from the definition of the polynomials. To see that the coefficients of monomials of highest total degree D are identical, choose a monomial <span class="inline-formula" id="inline-formula-200"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-224.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-224.gif"/></noscript></span></span> of the loci <span class="inline-formula" id="inline-formula-201"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-225.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-225.gif"/></noscript></span></span> of total degree <em>D</em> of <em>f</em>. Expanding <span class="inline-formula" id="inline-formula-202"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-226.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-226.gif"/></noscript></span></span> gives the same monomial <span class="inline-formula" id="inline-formula-203"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-227.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-227.gif"/></noscript></span></span> as a summand of highest total degree, plus additional monomials of lower total degree. Thus, the coefficients of monomials of total degree <em>D</em> remain the same. â</p><p id="p-81"><em>Corollary 1</em>. Let â± be the set of polynomials in the variables <span class="inline-formula" id="inline-formula-204"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-228.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-228.gif"/></noscript></span></span> across which we look for the one minimizing the SSR with respect to our data (the OLS fit). Let <span class="inline-formula" id="inline-formula-205"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-229.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-229.gif"/></noscript></span></span> be the polynomials in the variables <span class="inline-formula" id="inline-formula-206"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-230.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-230.gif"/></noscript></span></span>. Given the vector <span class="inline-formula" id="inline-formula-207"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-231.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-231.gif"/></noscript></span></span>, which defines the alternative coding by <span class="inline-formula" id="inline-formula-208"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-232.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-232.gif"/></noscript></span></span>, the definition of <span class="inline-formula" id="inline-formula-209"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-233.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-233.gif"/></noscript></span></span> in Proposition 1 defines a map<span class="disp-formula" id="disp-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-234.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-234.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-235.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-235.gif"/></noscript></span></span>This equation states that <span class="inline-formula" id="inline-formula-210"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-236.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-236.gif"/></noscript></span></span> which is a polynomial in the variables <span class="inline-formula" id="inline-formula-211"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-237.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-237.gif"/></noscript></span></span> is defined by the original <em>f</em> when we plug in the variables <span class="inline-formula" id="inline-formula-212"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-238.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-238.gif"/></noscript></span></span> and write down the expression as a function of <span class="inline-formula" id="inline-formula-213"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-239.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-239.gif"/></noscript></span></span>. The SSR can be considered as a function<span class="disp-formula" id="disp-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-240.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-240.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-241.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-241.gif"/></noscript></span></span>Since <em>f</em> and <span class="inline-formula" id="inline-formula-214"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-242.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-242.gif"/></noscript></span></span> fit the data in the same way, we have <span class="inline-formula" id="inline-formula-215"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-243.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-243.gif"/></noscript></span></span>. Thus, for a solution <span class="inline-formula" id="inline-formula-216"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-244.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-244.gif"/></noscript></span></span> minimizing the SSR across all polynomials in coding M, <span class="inline-formula" id="inline-formula-217"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-245.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-245.gif"/></noscript></span></span> minimizes the SSR in coding <span class="inline-formula" id="inline-formula-218"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-246.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-246.gif"/></noscript></span></span>.â</p><p id="p-82"><em>Corollary 2</em>. Analogously to the proof of Corollary 1, but with a function SSR2, which is the sum of SSR and the penalty on the coefficients of monomials of highest degree, which are the same for <em>f</em> and <span class="inline-formula" id="inline-formula-219"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-247.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-247.gif"/></noscript></span></span>, which implies <span class="inline-formula" id="inline-formula-220"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-248.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1117/embed/mml-math-248.gif"/></noscript></span></span>.â</p><p id="p-83"><em>Corollary 3</em>. A special case of Corollary 2, since only the additive effects, that is only the coefficents of monomials of highest total degree are penalized.â</p><p id="p-84"><em>Corollary 4</em>. Analogous to Corollary 3.â</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-2"><p id="p-86"><em>Communicating editor: R. Wisser</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2018-12-09"><span class="received-label">Received </span>December 9, 2018.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2019-02-06"><span class="accepted-label">Accepted </span>February 6, 2019.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright Â© 2019 Martini <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.4.1117.1" data-doi="10.1534/genetics.106.067348"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ãlvarez-Castro</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carlborg</span>  <span class="cit-name-given-names">Ã.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span>â<span class="cit-article-title">A unified model for functional and statistical epistasis and its application in quantitative trait loci analysis.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">176</span>: <span class="cit-fpage">1151</span>â<span class="cit-lpage">1167</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.106.067348</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.106.067348%26rft_id%253Dinfo%253Apmid%252F17409082%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3Ni8yLzExNTEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzQvMTExNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.4.1117.2" data-doi="10.1002/gepi.21989"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Aschard</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span>â<span class="cit-article-title">A perspective on interaction effects in genetic association studies.</span> <abbr class="cit-jnl-abbrev">Genet. Epidemiol.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">678</span>â<span class="cit-lpage">688</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/gepi.21989</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BEpidemiol.%26rft.volume%253D40%26rft.spage%253D678%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fgepi.21989%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/gepi.21989&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.4.1117.3" data-doi="10.1038/ng1761"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Carlborg</span>  <span class="cit-name-given-names">Ã.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jacobsson</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ãhgren</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Siegel</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Andersson</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span>â<span class="cit-article-title">Epistasis and the release of genetic variation during long-term selection.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">418</span>â<span class="cit-lpage">420</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng1761</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DCarlborg%26rft.auinit1%253DO.%26rft.volume%253D38%26rft.issue%253D4%26rft.spage%253D418%26rft.epage%253D420%26rft.atitle%253DEpistasis%2Band%2Bthe%2Brelease%2Bof%2Bgenetic%2Bvariation%2Bduring%2Blong-term%2Bselection.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1761%26rft_id%253Dinfo%253Apmid%252F16532011%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1761&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16532011&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1117.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000236340500013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.4.1117.4" data-doi="10.1371/journal.pone.0163692"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mao</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yin</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span>â<span class="cit-article-title">SNP-SNP interaction analysis on soybean oil content under multi-environments.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">e0163692</span>. Erratum: e0169222.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0163692</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D11%26rft.spage%253De0163692%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0163692%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0163692&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">âµ</a><div class="cit ref-cit ref-book" id="cit-9.4.1117.5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Clifford</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCullagh</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clifford</span>  <span class="cit-name-given-names">M. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span>âThe regress package. R package version. <span class="cit-fpage">1</span>â<span class="cit-lpage">3</span>. <a href="https://cran.r-project.org/web/packages/regress/regress.pdf">https://cran.r-project.org/web/packages/regress/regress.pdf</a></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.4.1117.6" data-doi="10.1038/nrg2579"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cordell</span>  <span class="cit-name-given-names">H. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">Detecting gene-gene interactions that underlie human diseases.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">392</span>â<span class="cit-lpage">404</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg2579</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DCordell%26rft.auinit1%253DH.%2BJ.%26rft.volume%253D10%26rft.issue%253D6%26rft.spage%253D392%26rft.epage%253D404%26rft.atitle%253DDetecting%2Bgene-gene%2Binteractions%2Bthat%2Bunderlie%2Bhuman%2Bdiseases.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2579%26rft_id%253Dinfo%253Apmid%252F19434077%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2579&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19434077&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1117.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000266095200014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.4.1117.7" data-doi="10.1534/genetics.110.118521"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">PÃ©rez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">BurgueÃ±o</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">Prediction of genetic values of quantitative traits in plant breeding using pedigree and molecular markers.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">186</span>: <span class="cit-fpage">713</span>â<span class="cit-lpage">724</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.110.118521</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.118521%26rft_id%253Dinfo%253Apmid%252F20813882%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg2LzIvNzEzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS80LzExMTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">âµ</a><div class="cit ref-cit ref-book" id="cit-9.4.1117.8" data-doi="10.1534/genetics.115.179887"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">De Coninck</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">De Baets</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kourounis</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Verbosio</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schenk</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span>â<span class="cit-article-title">Needles: toward large-scale genomic prediction with marker-by-environment interaction.</span> <span class="cit-source">Genetics</span> <span class="cit-vol">203</span>: <span class="cit-fpage">543</span>â<span class="cit-lpage">555</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.115.179887</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.115.179887%26rft_id%253Dinfo%253Apmid%252F26936924%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjAzLzEvNTQzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS80LzExMTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.4.1117.9" data-doi="10.2527/jas.2008-1259"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosa</span>  <span class="cit-name-given-names">G. J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">Reproducing kernel Hilbert spaces regression: a general framework for genetic evaluation.</span> <abbr class="cit-jnl-abbrev">J. Anim. Sci.</abbr> <span class="cit-vol">87</span>: <span class="cit-fpage">1883</span>â<span class="cit-lpage">1887</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2527/jas.2008-1259</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAnim.%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.2527%252Fjas.2008-1259%26rft_id%253Dinfo%253Apmid%252F19213705%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2527/jas.2008-1259&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19213705&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1117.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">âµ</a><div class="cit ref-cit ref-book" id="cit-9.4.1117.10" data-doi="10.1534/genetics.112.143313"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hickey</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pong-Wong</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daetwyler</span>  <span class="cit-name-given-names">H. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Calus</span>  <span class="cit-name-given-names">M. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span>â<span class="cit-article-title">Whole-genome regression and prediction methods applied to plant and animal breeding.</span> <span class="cit-source">Genetics</span> <span class="cit-vol">193</span>: <span class="cit-fpage">327</span>â<span class="cit-lpage">345</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.112.143313</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.143313%26rft_id%253Dinfo%253Apmid%252F22745228%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkzLzIvMzI3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS80LzExMTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">âµ</a><div class="cit ref-cit ref-other" id="cit-9.4.1117.11"><div class="cit-metadata unstructured">de los Campos, G. and P. Perez Rodriguez, 2016â<em>BGLR: Bayesian Generalized Linear Regression</em>. R package version 1.0.5.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.4.1117.12" data-doi="10.1534/genetics.117.203059"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ehrenreich</span>  <span class="cit-name-given-names">I. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span>â<span class="cit-article-title">Epistasis: Searching for interacting genetic variants using crosses.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">206</span>: <span class="cit-fpage">531</span>â<span class="cit-lpage">535</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.117.203059</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.117.203059%26rft_id%253Dinfo%253Apmid%252F28592494%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjA2LzIvNTMxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS80LzExMTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.4.1117.13" data-doi="10.1093/jhered/esw123"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Esfandyari</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Henryon</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berg</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thomasen</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bijma</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span>â<span class="cit-article-title">Response to selection in finite locus models with nonadditive effects.</span> <abbr class="cit-jnl-abbrev">J. Hered.</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">318</span>â<span class="cit-lpage">327</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/jhered/esw123</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BHered.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjhered%252Fesw123%26rft_id%253Dinfo%253Apmid%252F28082328%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/jhered/esw123&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28082328&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1117.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">âµ</a><div class="cit ref-cit ref-book" id="cit-9.4.1117.14"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Falconer</span>  <span class="cit-name-given-names">D. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mackay</span>  <span class="cit-name-given-names">T. F. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span>â<span class="cit-source">Introduction to Quantitative Genetics</span>, <span class="cit-publ-name">Pearson Education</span>, <span class="cit-publ-loc">London</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.4.1117.15" data-doi="10.1186/s12711-017-0340-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Forneris</span>  <span class="cit-name-given-names">N. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vitezica</span>  <span class="cit-name-given-names">Z. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Legarra</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">PÃ©rez-Enciso</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span>â<span class="cit-article-title">Influence of epistasis on response to genomic selection using complete sequence data.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">66</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12711-017-0340-3</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D49%26rft.spage%253D66%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12711-017-0340-3%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12711-017-0340-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.4.1117.16" data-doi="10.1534/genetics.117.300198"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gao</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martini</span>  <span class="cit-name-given-names">J. W. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yuan</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span>â<span class="cit-article-title">Incorporating gene annotation into genomic prediction of complex phenotypes.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">207</span>: <span class="cit-fpage">489</span>â<span class="cit-lpage">501</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.117.300198</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.117.300198%26rft_id%253Dinfo%253Apmid%252F28839043%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjA3LzIvNDg5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS80LzExMTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.4.1117.17" data-doi="10.1534/genetics.113.151753"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span>â<span class="cit-article-title">Priors in whole-genome regression: The Bayesian alphabet returns.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">194</span>: <span class="cit-fpage">573</span>â<span class="cit-lpage">596</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.113.151753</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.151753%26rft_id%253Dinfo%253Apmid%252F23636739%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk0LzMvNTczIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS80LzExMTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.4.1117.18" data-doi="10.1534/genetics.109.103952"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hill</span>  <span class="cit-name-given-names">W. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Manfredi</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">Additive genetic variability and the Bayesian alphabet.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">183</span>: <span class="cit-fpage">347</span>â<span class="cit-lpage">363</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.109.103952</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.109.103952%26rft_id%253Dinfo%253Apmid%252F19620397%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTgzLzEvMzQ3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS80LzExMTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">âµ</a><div class="cit ref-cit ref-confproc" id="cit-9.4.1117.19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morota</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span>âGenome-enabled prediction of complex traits with kernel methods: What have we learned? Proceedings, 10th World Congress of Genetics Applied to Livestock Production.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.4.1117.20" data-doi="10.1534/genetics.107.084285"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van Kaam</span>  <span class="cit-name-given-names">J. B. C. H. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span>â<span class="cit-article-title">Reproducing kernel Hilbert spaces regression methods for genomic assisted prediction of quantitative traits.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">178</span>: <span class="cit-fpage">2289</span>â<span class="cit-lpage">2303</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.107.084285</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DGianola%26rft.auinit1%253DD.%26rft.volume%253D178%26rft.issue%253D4%26rft.spage%253D2289%26rft.epage%253D2303%26rft.atitle%253DReproducing%2BKernel%2BHilbert%2BSpaces%2BRegression%2BMethods%2Bfor%2BGenomic%2BAssisted%2BPrediction%2Bof%2BQuantitative%2BTraits%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.084285%26rft_id%253Dinfo%253Apmid%252F18430950%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3OC80LzIyODkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzQvMTExNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.4.1117.21" data-doi="10.1534/genetics.107.081190"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Habier</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dekkers</span>  <span class="cit-name-given-names">J. C. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span>â<span class="cit-article-title">The impact of genetic relationship information on genome-assisted breeding values.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">177</span>: <span class="cit-fpage">2389</span>â<span class="cit-lpage">2397</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DHabier%26rft.auinit1%253DD.%26rft.volume%253D177%26rft.issue%253D4%26rft.spage%253D2389%26rft.epage%253D2397%26rft.atitle%253DThe%2BImpact%2Bof%2BGenetic%2BRelationship%2BInformation%2Bon%2BGenome-Assisted%2BBreeding%2BValues%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.081190%26rft_id%253Dinfo%253Apmid%252F18073436%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3Ny80LzIzODkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzQvMTExNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.4.1117.22" data-doi="10.1111/pbr.12037"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cogan</span>  <span class="cit-name-given-names">N. O. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pembleton</span>  <span class="cit-name-given-names">L. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span>â<span class="cit-article-title">Prospects for genomic selection in forage plant species.</span> <abbr class="cit-jnl-abbrev">Plant Breed.</abbr> <span class="cit-vol">132</span>: <span class="cit-fpage">133</span>â<span class="cit-lpage">143</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/pbr.12037</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BBreed.%26rft.volume%253D132%26rft.spage%253D133%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fpbr.12037%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/pbr.12037&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.4.1117.23" data-doi="10.1017/S0016672308009981"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Visscher</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">Increased accuracy of artificial selection by using the realized relationship matrix.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">47</span>â<span class="cit-lpage">60</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1017/S0016672308009981</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DHayes%26rft.auinit1%253DB.%2BJ.%26rft.volume%253D91%26rft.issue%253D1%26rft.spage%253D47%26rft.epage%253D60%26rft.atitle%253DIncreased%2Baccuracy%2Bof%2Bartificial%2Bselection%2Bby%2Busing%2Bthe%2Brealized%2Brelationship%2Bmatrix.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672308009981%26rft_id%253Dinfo%253Apmid%252F19220931%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672308009981&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19220931&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1117.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000264226300006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.4.1117.24" data-doi="10.1186/s12859-016-1127-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">He</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parida</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span>â<span class="cit-article-title">Does encoding matter? a novel view on the quantitative genetic trait prediction problem.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">272</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12859-016-1127-1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BBioinformatics%26rft.volume%253D17%26rft.spage%253D272%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12859-016-1127-1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12859-016-1127-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.4.1117.25" data-doi="10.2307/2529430"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Henderson</span>  <span class="cit-name-given-names">C. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1975</span>â<span class="cit-article-title">Best linear unbiased estimation and prediction under a selection model.</span> <abbr class="cit-jnl-abbrev">Biometrics</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">423</span>â<span class="cit-lpage">447</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2307/2529430</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiometrics%26rft.stitle%253DBiometrics%26rft.aulast%253DHenderson%26rft.auinit1%253DC.%2BR.%26rft.volume%253D31%26rft.issue%253D2%26rft.spage%253D423%26rft.epage%253D447%26rft.atitle%253DBest%2Blinear%2Bunbiased%2Bestimation%2Band%2Bprediction%2Bunder%2Ba%2Bselection%2Bmodel.%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2529430%26rft_id%253Dinfo%253Apmid%252F1174616%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2529430&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1174616&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1117.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1975AF09900010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.4.1117.26" data-doi="10.3168/jds.S0022-0302(77)83935-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Henderson</span>  <span class="cit-name-given-names">C. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1977</span>â<span class="cit-article-title">Best linear unbiased prediction of breeding values not in the model for records.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">60</span>: <span class="cit-fpage">783</span>â<span class="cit-lpage">787</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3168/jds.S0022-0302(77)83935-0</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DHenderson%26rft.auinit1%253DC.%2BR.%26rft.volume%253D60%26rft.issue%253D5%26rft.spage%253D783%26rft.epage%253D787%26rft.atitle%253DBest%2BLinear%2BUnbiased%2BPrediction%2Bof%2BBreeding%2BValues%2BNot%2Bin%2Bthe%2BModel%2Bfor%2BRecords%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.S0022-0302%252877%252983935-0%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.S0022-0302(77)83935-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.4.1117.27" data-doi="10.2527/jas1976.4361188x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Henderson</span>  <span class="cit-name-given-names">C. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Quaas</span>  <span class="cit-name-given-names">R. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1976</span>â<span class="cit-article-title">Multiple trait evaluation using relativesâ records.</span> <abbr class="cit-jnl-abbrev">J. Anim. Sci.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">1188</span>â<span class="cit-lpage">1197</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2527/jas1976.4361188x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BAnimal%2BScience%26rft.stitle%253DJ%2BANIM%2BSCI%26rft.aulast%253DHenderson%26rft.auinit1%253DC.%2BR.%26rft.volume%253D43%26rft.issue%253D6%26rft.spage%253D1188%26rft.epage%253D1197%26rft.atitle%253DMultiple%2BTrait%2BEvaluation%2BUsing%2BRelatives%2527%2BRecords%26rft_id%253Dinfo%253Adoi%252F10.2527%252Fjas1976.4361188x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2527/jas1976.4361188x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1976CP41700009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.4.1117.28" data-doi="10.1534/g3.114.010298"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Howard</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carriquiry</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beavis</span>  <span class="cit-name-given-names">W. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span>â<span class="cit-article-title">Parametric and nonparametric statistical methods for genomic selection of traits with additive and epistatic genetic architectures.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">1027</span>â<span class="cit-lpage">1046</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.114.010298</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.114.010298%26rft_id%253Dinfo%253Apmid%252F24727289%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjg6IjQvNi8xMDI3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS80LzExMTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.4.1117.29" data-doi="10.1534/genetics.115.177907"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reif</span>  <span class="cit-name-given-names">J. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span>â<span class="cit-article-title">Modeling epistasis in genomic selection.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">201</span>: <span class="cit-fpage">759</span>â<span class="cit-lpage">768</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.115.177907</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.115.177907%26rft_id%253Dinfo%253Apmid%252F26219298%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjAxLzIvNzU5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS80LzExMTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.4.1117.30" data-doi="10.1534/genetics.115.182444"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">KÃ¤rkkÃ¤inen</span>  <span class="cit-name-given-names">H. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">SillanpÃ¤Ã¤</span>  <span class="cit-name-given-names">M. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span>â<span class="cit-article-title">An efficient genome-wide multilocus epistasis search.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">201</span>: <span class="cit-fpage">865</span>â<span class="cit-lpage">870</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.115.182444</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.115.182444%26rft_id%253Dinfo%253Apmid%252F26405029%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjAxLzMvODY1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS80LzExMTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.4.1117.31" data-doi="10.1186/s12859-016-1439-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Martini</span>  <span class="cit-name-given-names">J. W. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gao</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cardoso</span>  <span class="cit-name-given-names">D. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wimmer</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Erbe</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span>â<span class="cit-article-title">Genomic prediction with epistasis models: on the marker-coding-dependent performance of the extended GBLUP and properties of the categorical epistasis model (CE).</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">3</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12859-016-1439-1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BBioinformatics%26rft.volume%253D18%26rft.spage%253D3%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12859-016-1439-1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12859-016-1439-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.4.1117.32" data-doi="10.1007/s00122-016-2675-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Martini</span>  <span class="cit-name-given-names">J. W. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wimmer</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Erbe</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Simianer</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span>â<span class="cit-article-title">Epistasis and covariance: how gene interaction translates into genomic relationship.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">129</span>: <span class="cit-fpage">963</span>â<span class="cit-lpage">976</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-016-2675-5</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D129%26rft.spage%253D963%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-016-2675-5%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-016-2675-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.4.1117.33" data-doi="10.1093/nar/gkv1344"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>  <span class="cit-name-given-names">T. H. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span>â<span class="cit-article-title">Prediction of total genetic value using genome-wide dense marker maps.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">157</span>: <span class="cit-fpage">1819</span>â<span class="cit-lpage">1829</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMeuwissen%26rft.auinit1%253DT.%2BH.%2BE.%26rft.volume%253D157%26rft.issue%253D4%26rft.spage%253D1819%26rft.epage%253D1829%26rft.atitle%253DPrediction%2Bof%2BTotal%2BGenetic%2BValue%2BUsing%2BGenome-Wide%2BDense%2BMarker%2BMaps%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkv1344%26rft_id%253Dinfo%253Apmid%252F11290733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1Ny80LzE4MTkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzQvMTExNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.4.1117.34" data-doi="10.3389/fgene.2014.00363"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Morota</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span>â<span class="cit-article-title">Kernel-based whole-genome prediction of complex traits: a review.</span> <abbr class="cit-jnl-abbrev">Front. Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">363</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fgene.2014.00363</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BGenet.%26rft.volume%253D5%26rft.spage%253D363%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffgene.2014.00363%26rft_id%253Dinfo%253Apmid%252F25360145%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fgene.2014.00363&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25360145&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1117.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">âµ</a><div class="cit ref-cit ref-book" id="cit-9.4.1117.35" data-doi="10.1079/9781780643915.0000"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mrode</span>  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span>â<span class="cit-source">Linear models for the prediction of animal breeding values</span>, <span class="cit-publ-name">CABI Publishing</span>, <span class="cit-publ-loc">Wallingford</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1079/9781780643915.0000</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLinear%2Bmodels%2Bfor%2Bthe%2Bprediction%2Bof%2Banimal%2Bbreeding%2Bvalues%26rft_id%253Dinfo%253Adoi%252F10.1079%252F9781780643915.0000%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1079/9781780643915.0000&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.4.1117.36" data-doi="10.1534/genetics.111.128694"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ober</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Erbe</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Long</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Porcu</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schlather</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span>â<span class="cit-article-title">Predicting genetic values: A kernel-based best linear unbiased prediction with genomic data.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">188</span>: <span class="cit-fpage">695</span>â<span class="cit-lpage">708</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.111.128694</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.128694%26rft_id%253Dinfo%253Apmid%252F21515573%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg4LzMvNjk1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS80LzExMTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.4.1117.37" data-doi="10.1371/journal.pone.0126880"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ober</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Magwire</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schlather</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Simianer</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span>â<span class="cit-article-title">Accounting for genetic architecture improves sequence based genomic prediction for a <em>Drosophila</em> fitness trait.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">e0126880</span>. Erratum: e0132980.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0126880</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D10%26rft.spage%253De0126880%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0126880%26rft_id%253Dinfo%253Apmid%252F25950439%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0126880&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25950439&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1117.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.4.1117.38" data-doi="10.3835/plantgenome2016.09.0089"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">PÃ©rez-RodrÃ­guez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rutkoski</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Singh</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span>â<span class="cit-article-title">Single-step genomic and pedigree genotype Ã environment interaction models for predicting wheat lines in international environments.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">1</span>â<span class="cit-lpage">15</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2016.09.0089</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D10%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2016.09.0089%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2016.09.0089&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">âµ</a><div class="cit ref-cit ref-other" id="cit-9.4.1117.39"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">R Core Team</span></li></ol><cite>, <span class="cit-pub-date">2016</span> R: A language and environment for statistical computing.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.4.1117.40"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Santantonio</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span>â<span class="cit-article-title">A subfunctionalization epistasis model to evaluate homeologous gene interactions in allopolyploid wheat.</span> <abbr class="cit-jnl-abbrev">bioRxiv</abbr>: <span class="cit-fpage">1</span>â<span class="cit-lpage">72</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.4.1117.41" data-doi="10.1016/S0301-6226(03)00151-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schaeffer</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span>â<span class="cit-article-title">Application of random regression models in animal breeding.</span> <abbr class="cit-jnl-abbrev">Livest. Prod. Sci.</abbr> <span class="cit-vol">86</span>: <span class="cit-fpage">35</span>â<span class="cit-lpage">45</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/S0301-6226(03)00151-9</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLivest.%2BProd.%2BSci.%26rft.volume%253D86%26rft.spage%253D35%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0301-6226%252803%252900151-9%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0301-6226(03)00151-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.4.1117.42" data-doi="10.1111/j.1439-0388.2006.00595.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schaeffer</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span>â<span class="cit-article-title">Strategy for applying genome-wide selection in dairy cattle.</span> <abbr class="cit-jnl-abbrev">J. Anim. Breed. Genet.</abbr> <span class="cit-vol">123</span>: <span class="cit-fpage">218</span>â<span class="cit-lpage">223</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1439-0388.2006.00595.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DJ%2BAnim%2BBreed%2BGenet%26rft.aulast%253DSchaeffer%26rft.auinit1%253DL.%2BR.%26rft.volume%253D123%26rft.issue%253D4%26rft.spage%253D218%26rft.epage%253D223%26rft.atitle%253DStrategy%2Bfor%2Bapplying%2Bgenome-wide%2Bselection%2Bin%2Bdairy%2Bcattle.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1439-0388.2006.00595.x%26rft_id%253Dinfo%253Apmid%252F16882088%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1439-0388.2006.00595.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16882088&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1117.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000239006300002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.4.1117.43" data-doi="10.1534/g3.115.025809"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cai</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span>â<span class="cit-article-title">Partial dominance, overdominance, epistasis and qtl by environment interactions contribute to heterosis in two upland cotton hybrids.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">499</span>â<span class="cit-lpage">507</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.115.025809</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.115.025809%26rft_id%253Dinfo%253Apmid%252F26715091%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjc6IjYvMy80OTkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzQvMTExNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.4.1117.44" data-doi="10.1186/1297-9686-43-25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">StrandÃ©n</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Christensen</span>  <span class="cit-name-given-names">O. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span>â<span class="cit-article-title">Allele coding in genomic evaluation.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">25</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1297-9686-43-25</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DStranden%26rft.auinit1%253DI.%26rft.volume%253D43%26rft.spage%253D25%26rft.epage%253D25%26rft.atitle%253DAllele%2Bcoding%2Bin%2Bgenomic%2Bevaluation.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-43-25%26rft_id%253Dinfo%253Apmid%252F21703021%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-43-25&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21703021&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1117.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.4.1117.45" data-doi="10.1371/journal.pone.0045293"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Su</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Christensen</span>  <span class="cit-name-given-names">O. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ostersen</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Henryon</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lund</span>  <span class="cit-name-given-names">M. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">Estimating additive and non-additive genetic variances and predicting genetic merits using genome-wide dense single nucleotide polymorphism markers.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e45293</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0045293</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DSu%26rft.auinit1%253DG.%26rft.volume%253D7%26rft.issue%253D9%26rft.spage%253De45293%26rft.epage%253De45293%26rft.atitle%253DEstimating%2Badditive%2Band%2Bnon-additive%2Bgenetic%2Bvariances%2Band%2Bpredicting%2Bgenetic%2Bmerits%2Busing%2Bgenome-wide%2Bdense%2Bsingle%2Bnucleotide%2Bpolymorphism%2Bmarkers.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0045293%26rft_id%253Dinfo%253Apmid%252F23028912%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0045293&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23028912&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1117.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.4.1117.46" data-doi="10.1534/g3.116.036251"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sukumaran</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jarquin</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lopes</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reynolds</span>  <span class="cit-name-given-names">M. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span>â<span class="cit-article-title">Genomic prediction with pedigree and genotype Ã environment interaction in spring wheat grown in south and west asia, north africa, and mexico.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">481</span>â<span class="cit-lpage">495</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.116.036251</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.116.036251%26rft_id%253Dinfo%253Apmid%252F27903632%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjc6IjcvMi80ODEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzQvMTExNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">âµ</a><div class="cit ref-cit ref-book" id="cit-9.4.1117.47"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tibshirani</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span>â<span class="cit-article-title">Regression shrinkage and selection via the Lasso.</span> <span class="cit-source">J. Roy. Stat. Soc. B Met.</span> <span class="cit-vol">58</span>: <span class="cit-fpage">267</span>â<span class="cit-lpage">288</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BRoy.%2BStat.%2BSoc.%2BB%2BMet.%26rft.volume%253D58%26rft.spage%253D267%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.4.1117.48" data-doi="10.3168/jds.2007-0980"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">VanRaden</span>  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span>â<span class="cit-article-title">Efficient methods to compute genomic predictions.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">4414</span>â<span class="cit-lpage">4423</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3168/jds.2007-0980</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DVanRaden%26rft.auinit1%253DP.%2BM.%26rft.volume%253D91%26rft.issue%253D11%26rft.spage%253D4414%26rft.epage%253D4423%26rft.atitle%253DEfficient%2BMethods%2Bto%2BCompute%2BGenomic%2BPredictions%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2007-0980%26rft_id%253Dinfo%253Apmid%252F18946147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2007-0980&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18946147&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1117.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000260277200035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.4.1117.49" data-doi="10.3389/fgene.2018.00078"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Varona</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Legarra</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Toro</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vitezica</span>  <span class="cit-name-given-names">Z. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span>â<span class="cit-article-title">Non-additive effects in genomic selection.</span> <abbr class="cit-jnl-abbrev">Front. Genet.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">78</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fgene.2018.00078</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BGenet.%26rft.volume%253D9%26rft.spage%253D78%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffgene.2018.00078%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fgene.2018.00078&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.4.1117.50" data-doi="10.1534/genetics.116.199406"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vitezica</span>  <span class="cit-name-given-names">Z. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Legarra</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Toro</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Varona</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span>â<span class="cit-article-title">Orthogonal estimates of variances for additive, dominance, and epistatic effects in populations.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">206</span>: <span class="cit-fpage">1297</span>â<span class="cit-lpage">1307</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.116.199406</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.116.199406%26rft_id%253Dinfo%253Apmid%252F28522540%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjIwNi8zLzEyOTciO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzQvMTExNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">âµ</a><div class="cit ref-cit ref-book" id="cit-9.4.1117.51" data-doi="10.1534/genetics.118.301015"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xiang</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Christensen</span>  <span class="cit-name-given-names">O. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vitezica</span>  <span class="cit-name-given-names">Z. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Legarra</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span>â<span class="cit-article-title">Genomic model with correlation between additive and dominance effects.</span> <span class="cit-source">Genetics</span> <span class="cit-vol">209</span>: <span class="cit-fpage">711</span>â<span class="cit-lpage">723</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.118.301015</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.118.301015%26rft_id%253Dinfo%253Apmid%252F29743175%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjA5LzMvNzExIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS80LzExMTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/9/4/1117.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/4/1103" title="Evolution of the Highly Repetitive PEVK Region of Titin Across Mammals" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/4/1131" title="QTL Underlying Circadian Clock Parameters Under Seasonally Variable Field Settings in Arabidopsis thaliana" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/9/4"><span>Volume 9 Issue 4, April 2019</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/9/4.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/9/4.cover-source.jpg?itok=mcy5SuIK" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 9 (4)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/4/1117" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-evrRLEGz97FO84ZXbjFoOiaEVMI1KUGMOXFWG1TdMNs" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/genomic-prediction" class="" data-icon-position="" data-hide-link-title="0">Genomic Prediction</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/9/4/1117" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/9/4/1117" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/9/4/1117" class="active" data-icon-position="" data-hide-link-title="0">Lost in Translation: On the Problem of Data Coding in Penalized Whole Genome Regression with Interactions</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/492621" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-r8GP-w5Dwj1KjiIXgeO6pR3j95yQ-VMcu0DcZ9yymjY" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/4/1117.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/9/4/1117" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-29YhVD3ACqR5w5oFjWPrlLAU5TbwbC2uBvT2dePV31k" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/9/4/1117" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/4/1117.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="492621" id="node492621--2" data-pisa="ggg;9/4/1117" data-pisa-master="ggg;g3.118.200961" data-apath="/ggg/9/4/1117.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Lost in Translation: On the Problem of Data Coding in Penalized Whole Genome Regression with Interactions</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-0628-6794" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Johannes W. R. Martini</span>, <span class="highwire-citation-author" data-delta="1">Francisco Rosales</span>, <span class="highwire-citation-author" data-delta="2">Ngoc-Thuy Ha</span>, <span class="highwire-citation-author" data-delta="3">Johannes Heise</span>, <span class="highwire-citation-author" data-delta="4">Valentin Wimmer</span> and <span class="highwire-citation-author" data-delta="5">Thomas Kneib</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1117-1129;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200961 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node492621--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Johannes W. R. Martini </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>KWS SAAT SE, Einbeck, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Johannes%2BW.%2BR.%2BMartini%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Martini%20JW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJohannes%2BW.%2BR.%2BMartini%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-0628-6794" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Johannes W. R. Martini</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jmartin2@gwdg.de" class="" data-icon-position="" data-hide-link-title="0">jmartin2@gwdg.de</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Francisco Rosales </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>Universidad del PacÃ­fico, Academic Department of Finance, Lima, Peru</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Francisco%2BRosales%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rosales%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFrancisco%2BRosales%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Ngoc-Thuy Ha </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â¡</span>Department of Animal Breeding and Genetics</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ngoc-Thuy%2BHa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ha%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANgoc-Thuy%2BHa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Johannes Heise </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>Â§</span>IT Solutions for Animal Production (vit), Verden, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Johannes%2BHeise%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Heise%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJohannes%2BHeise%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Valentin Wimmer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>KWS SAAT SE, Einbeck, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Valentin%2BWimmer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wimmer%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AValentin%2BWimmer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Thomas Kneib </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Chairs of Statistics and Econometrics, University of Goettingen, Goettingen, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Thomas%2BKneib%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kneib%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AThomas%2BKneib%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/9/4/1117" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/9/4/1117&amp;title=Lost%20in%20Translation%3A%20On%20the%20Problem%20of%20Data%20Coding%20in%20Penalized%20Whole%20Genome%20Regression%20with%20Interactions" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/9/4/1117&amp;title=Lost%20in%20Translation%3A%20On%20the%20Problem%20of%20Data%20Coding%20in%20Penalized%20Whole%20Genome%20Regression%20with%20Interactions" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/9/4/1117&amp;title=Lost%20in%20Translation%3A%20On%20the%20Problem%20of%20Data%20Coding%20in%20Penalized%20Whole%20Genome%20Regression%20with%20Interactions" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/9/4/1117&amp;text=Lost%20in%20Translation%3A%20On%20the%20Problem%20of%20Data%20Coding%20in%20Penalized%20Whole%20Genome%20Regression%20with%20Interactions" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/9/4/1117&amp;title=Lost%20in%20Translation%3A%20On%20the%20Problem%20of%20Data%20Coding%20in%20Penalized%20Whole%20Genome%20Regression%20with%20Interactions" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/9/4/1117&amp;t=Lost%20in%20Translation%3A%20On%20the%20Problem%20of%20Data%20Coding%20in%20Penalized%20Whole%20Genome%20Regression%20with%20Interactions" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/9/4/1117&amp;title=Lost%20in%20Translation%3A%20On%20the%20Problem%20of%20Data%20Coding%20in%20Penalized%20Whole%20Genome%20Regression%20with%20Interactions" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/9/4/1117&amp;title=Lost%20in%20Translation%3A%20On%20the%20Problem%20of%20Data%20Coding%20in%20Penalized%20Whole%20Genome%20Regression%20with%20Interactions" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/492621/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="492621" id="node492621--3" data-pisa="ggg;9/4/1117" data-pisa-master="ggg;g3.118.200961" data-apath="/ggg/9/4/1117.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Lost in Translation: On the Problem of Data Coding in Penalized Whole Genome Regression with Interactions</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-0628-6794" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Johannes W. R. Martini</span>, <span class="highwire-citation-author" data-delta="1">Francisco Rosales</span>, <span class="highwire-citation-author" data-delta="2">Ngoc-Thuy Ha</span>, <span class="highwire-citation-author" data-delta="3">Johannes Heise</span>, <span class="highwire-citation-author" data-delta="4">Valentin Wimmer</span> and <span class="highwire-citation-author" data-delta="5">Thomas Kneib</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1117-1129;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200961 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node492621--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Johannes W. R. Martini </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>KWS SAAT SE, Einbeck, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Johannes%2BW.%2BR.%2BMartini%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Martini%20JW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJohannes%2BW.%2BR.%2BMartini%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-0628-6794" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Johannes W. R. Martini</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jmartin2@gwdg.de" class="" data-icon-position="" data-hide-link-title="0">jmartin2@gwdg.de</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Francisco Rosales </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>Universidad del PacÃ­fico, Academic Department of Finance, Lima, Peru</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Francisco%2BRosales%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rosales%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFrancisco%2BRosales%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Ngoc-Thuy Ha </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â¡</span>Department of Animal Breeding and Genetics</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ngoc-Thuy%2BHa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ha%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANgoc-Thuy%2BHa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Johannes Heise </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>Â§</span>IT Solutions for Animal Production (vit), Verden, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Johannes%2BHeise%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Heise%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJohannes%2BHeise%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Valentin Wimmer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>KWS SAAT SE, Einbeck, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Valentin%2BWimmer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wimmer%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AValentin%2BWimmer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Thomas Kneib </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Chairs of Statistics and Econometrics, University of Goettingen, Goettingen, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Thomas%2BKneib%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kneib%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AThomas%2BKneib%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1117&amp;rft.epage=1129&amp;rft.atitle=Lost%20in%20Translation%3A%20On%20the%20Problem%20of%20Data%20Coding%20in%20Penalized%20Whole%20Genome%20Regression%20with%20Interactions&amp;rft.volume=9&amp;rft.issue=4&amp;rft.date=2019-04-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Martini%2C+Johannes+W.+R.&amp;rft.au=Rosales%2C+Francisco&amp;rft.au=Ha%2C+Ngoc-Thuy&amp;rft.au=Heise%2C+Johannes&amp;rft.au=Wimmer%2C+Valentin&amp;rft.au=Kneib%2C+Thomas"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/492621/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/492621/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/492621/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/492621/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/492621/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/492621/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/492621/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/492621/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/492621/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/492621/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/492621/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/492621/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495864" id="node495864" data-pisa="ggg;10/3/1113" data-pisa-master="ggg;g3.119.400968" data-apath="/ggg/10/3/1113.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1113" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Multi-trait Genomic Prediction Model Increased the Predictive Ability for Agronomic and Malting Quality Traits in Barley (<em>Hordeum vulgare</em> L.)</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495543" id="node495543" data-pisa="ggg;10/2/769" data-pisa-master="ggg;g3.119.400759" data-apath="/ggg/10/2/769.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/2/769" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Novel Bayesian Networks for Genomic Prediction of Developmental Traits in Biomass Sorghum</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495557" id="node495557" data-pisa="ggg;10/2/539" data-pisa-master="ggg;g3.119.400866" data-apath="/ggg/10/2/539.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/2/539" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Optimized Genetic Testing for Polledness in Multiple Breeds of Cattle</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Genomic%20Prediction" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Genomic Prediction</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/9/4/1117" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Theory: Specification Of Regression Methods</a></li><li class="odd"><a href="#sec-9" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Theoretical Results</a></li><li class="even"><a href="#sec-10" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Practical Implications: An Example With A Wheat Data Set</a></li><li class="odd"><a href="#sec-27" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#app-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Appendix: Proofs</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/9/4/1117.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/9/4/1117.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/9/4/1117.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__CK8qNg7zHnCPfipcSU0qjCyIiw2pe-WKGx2V0TgizGw__wprDSLLH9CuacxJkxX2DuWt0EaauZA-f_LGjovfY3aw__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
