<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/9/4/1231" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/9/4/1231.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/9/4/1231.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/9/4/1231.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/9/4/1231/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/9/4/1231.atom" />
<meta name="HW.pisa" content="ggg;9/4/1231" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Hyperspectral Reflectance-Derived Relationship Matrices for Genomic Prediction of Grain Yield in Wheat" />
<meta name="DC.Identifier" content="10.1534/g3.118.200856" />
<meta name="DC.Date" content="2019-04-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2019 Krause et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Relation" content="https://doi.org/10.25387/g3.7653473" />
<meta name="DC.Description" content="Hyperspectral reflectance phenotyping and genomic selection are two emerging technologies that have the potential to increase plant breeding efficiency by improving prediction accuracy for grain yield. Hyperspectral cameras quantify canopy reflectance across a wide range of wavelengths that are associated with numerous biophysical and biochemical processes in plants. Genomic selection models utilize genome-wide marker or pedigree information to predict the genetic values of breeding lines. In this study, we propose a multi-kernel GBLUP approach to genomic selection that uses genomic marker-, pedigree-, and hyperspectral reflectance-derived relationship matrices to model the genetic main effects and genotype × environment ( G  ×  E ) interactions across environments within a bread wheat ( Triticum aestivum L.) breeding program. We utilized an airplane equipped with a hyperspectral camera to phenotype five differentially managed treatments of the yield trials conducted by the Bread Wheat Improvement Program of the International Maize and Wheat Improvement Center (CIMMYT) at Ciudad Obregón, México over four breeding cycles. We observed that single-kernel models using hyperspectral reflectance-derived relationship matrices performed similarly or superior to marker- and pedigree-based genomic selection models when predicting within and across environments. Multi-kernel models combining marker/pedigree information with hyperspectral reflectance phentoypes had the highest prediction accuracies; however, improvements in accuracy over marker- and pedigree-based models were marginal when correcting for days to heading. Our results demonstrate the potential of using hyperspectral imaging to predict grain yield within a multi-environment context and also support further studies on the integration of hyperspectral reflectance phenotyping into breeding programs." />
<meta name="DC.Contributor" content="Margaret R. Krause" />
<meta name="DC.Contributor" content="Lorena González-Pérez" />
<meta name="DC.Contributor" content="José Crossa" />
<meta name="DC.Contributor" content="Paulino Pérez-Rodríguez" />
<meta name="DC.Contributor" content="Osval Montesinos-López" />
<meta name="DC.Contributor" content="Ravi P. Singh" />
<meta name="DC.Contributor" content="Susanne Dreisigacker" />
<meta name="DC.Contributor" content="Jesse Poland" />
<meta name="DC.Contributor" content="Jessica Rutkoski" />
<meta name="DC.Contributor" content="Mark Sorrells" />
<meta name="DC.Contributor" content="Michael A. Gore" />
<meta name="DC.Contributor" content="Suchismita Mondal" />
<meta name="article:published_time" content="2019-04-01" />
<meta name="article:section" content="Genomic Prediction" />
<meta name="citation_title" content="Hyperspectral Reflectance-Derived Relationship Matrices for Genomic Prediction of Grain Yield in Wheat" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Hyperspectral reflectance phenotyping and genomic selection are two emerging technologies that have the potential to increase plant breeding efficiency by improving prediction accuracy for grain yield. Hyperspectral cameras quantify canopy reflectance across a wide range of wavelengths that are associated with numerous biophysical and biochemical processes in plants. Genomic selection models utilize genome-wide marker or pedigree information to predict the genetic values of breeding lines. In this study, we propose a multi-kernel GBLUP approach to genomic selection that uses genomic marker-, pedigree-, and hyperspectral reflectance-derived relationship matrices to model the genetic main effects and genotype × environment (&lt;i&gt;G&lt;/i&gt; × &lt;i&gt;E&lt;/i&gt;) interactions across environments within a bread wheat (&lt;i&gt;Triticum aestivum&lt;/i&gt; L.) breeding program. We utilized an airplane equipped with a hyperspectral camera to phenotype five differentially managed treatments of the yield trials conducted by the Bread Wheat Improvement Program of the International Maize and Wheat Improvement Center (CIMMYT) at Ciudad Obregón, México over four breeding cycles. We observed that single-kernel models using hyperspectral reflectance-derived relationship matrices performed similarly or superior to marker- and pedigree-based genomic selection models when predicting within and across environments. Multi-kernel models combining marker/pedigree information with hyperspectral reflectance phentoypes had the highest prediction accuracies; however, improvements in accuracy over marker- and pedigree-based models were marginal when correcting for days to heading. Our results demonstrate the potential of using hyperspectral imaging to predict grain yield within a multi-environment context and also support further studies on the integration of hyperspectral reflectance phenotyping into breeding programs.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2019/04/01" />
<meta name="citation_mjid" content="ggg;9/4/1231" />
<meta name="citation_id" content="9/4/1231" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/9/4/1231" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/9/4/1231.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/9/4/1231.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/9/4/1231.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.118.200856" />
<meta name="citation_pmid" content="30796086" />
<meta name="citation_volume" content="9" />
<meta name="citation_issue" content="4" />
<meta name="citation_num_pages" content="17" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Genomic Prediction" />
<meta name="citation_firstpage" content="1231" />
<meta name="citation_lastpage" content="1247" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Margaret R. Krause" />
<meta name="citation_author_institution" content="Plant Breeding and Genetics Section, School of Integrative Plant Science, Cornell University, Ithaca, New York, 14853" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-1154-337X" />
<meta name="citation_author" content="Lorena González-Pérez" />
<meta name="citation_author_institution" content="Global Wheat Program, International Maize and Wheat Improvement Center (CIMMYT), Ciudad de México, 06600, México" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-5840-0803" />
<meta name="citation_author" content="José Crossa" />
<meta name="citation_author_institution" content="Global Wheat Program, International Maize and Wheat Improvement Center (CIMMYT), Ciudad de México, 06600, México" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-9429-5855" />
<meta name="citation_author" content="Paulino Pérez-Rodríguez" />
<meta name="citation_author_institution" content="Colegio de Postgraduados, CP 56230, Montecillos, Edo. de México, Mexico," />
<meta name="citation_author" content="Osval Montesinos-López" />
<meta name="citation_author_institution" content="Facultad de Telemática, Universidad de Colima, Colima, Colima, 28040, México" />
<meta name="citation_author" content="Ravi P. Singh" />
<meta name="citation_author_institution" content="Global Wheat Program, International Maize and Wheat Improvement Center (CIMMYT), Ciudad de México, 06600, México" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-4676-5071" />
<meta name="citation_author" content="Susanne Dreisigacker" />
<meta name="citation_author_institution" content="Global Wheat Program, International Maize and Wheat Improvement Center (CIMMYT), Ciudad de México, 06600, México" />
<meta name="citation_author" content="Jesse Poland" />
<meta name="citation_author_institution" content="Department of Plant Pathology, Kansas State University, Manhattan, Kansas, 66506" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-7856-1399" />
<meta name="citation_author" content="Jessica Rutkoski" />
<meta name="citation_author_institution" content="International Rice Research Institute (IRRI), DAPO Box 7777, Metro Manila, 1301, Philippines" />
<meta name="citation_author" content="Mark Sorrells" />
<meta name="citation_author_institution" content="Plant Breeding and Genetics Section, School of Integrative Plant Science, Cornell University, Ithaca, New York, 14853" />
<meta name="citation_author" content="Michael A. Gore" />
<meta name="citation_author_institution" content="Plant Breeding and Genetics Section, School of Integrative Plant Science, Cornell University, Ithaca, New York, 14853" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-6896-8024" />
<meta name="citation_author" content="Suchismita Mondal" />
<meta name="citation_author_institution" content="Global Wheat Program, International Maize and Wheat Improvement Center (CIMMYT), Ciudad de México, 06600, México" />
<meta name="citation_author_email" content="S.Mondal@cgiar.org" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-8582-8899" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=F. M. Aguate;citation_author=S. Trachsel;citation_author=L. González-Pérez;citation_author=J. Burgueño;citation_author=J. Crossa;citation_title=Use of hyperspectral image data outperforms vegetation indices in prediction of maize yield.;citation_pages=2517-2524;citation_volume=57;citation_year=2017;citation_doi=10.2135/cropsci2017.01.0007." />
<meta name="citation_reference" content="citation_journal_title=Funct. Plant Biol.;citation_author=P. Andrade-Sanchez;citation_author=M. A. Gore;citation_author=J. T. Heun;citation_author=K. R. Thorp;citation_author=A. E. Carmo-Silva;citation_title=Development and evaluation of a field-based high-throughput phenotyping platform.;citation_pages=68-79;citation_volume=41;citation_year=2014;citation_doi=10.1071/FP13126" />
<meta name="citation_reference" content="citation_journal_title=Agronomy Journal;citation_journal_abbrev=Agron. J.;citation_author=N. Aparicio;citation_author=D. Villegas;citation_author=J. Casadesus;citation_author=J. L. Araus;citation_author=C. Royo;citation_title=Spectral Vegetation Indices as Nondestructive Tools for Determining Durum Wheat Yield;citation_pages=83-91;citation_volume=92;citation_year=2000;citation_issue=1;citation_doi=10.2134/agronj2000.92183x" />
<meta name="citation_reference" content="citation_journal_title=Trends Plant Sci.;citation_author=J. L. Araus;citation_author=J. E. Cairns;citation_title=Field high-throughput phenotyping: the new crop breeding frontier.;citation_pages=52-61;citation_volume=19;citation_year=2014;citation_pmid=24139902;citation_doi=10.1016/j.tplants.2013.09.008" />
<meta name="citation_reference" content="citation_title=Breeding for quantitative traits in plants;citation_author=R. Bernardo;citation_year=2010" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=J. Burgueño;citation_author=G. de los Campos;citation_author=K. Weigel;citation_author=J. Crossa;citation_title=Genomic prediction of breeding values when modeling genotype × environment interaction using pedigree and dense molecular markers.;citation_pages=707-719;citation_volume=52;citation_year=2012;citation_doi=10.2135/cropsci2011.06.0299" />
<meta name="citation_reference" content="citation_journal_abbrev=J Integr Plant Biol;citation_author=L. Cabrera-Bosquet;citation_author=J. Crossa;citation_author=J. von Zitzewitz;citation_author=M. D. Serret;citation_author=J. L. Araus;citation_title=High-throughput phenotyping and genomic selection: the frontiers of crop breeding converge.;citation_pages=312-320;citation_volume=54;citation_year=2012;citation_issue=5;citation_pmid=22420640;citation_doi=10.1111/j.1744-7909.2012.01116.x" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=J. Crain;citation_author=Y. Wei;citation_author=J. Barker;citation_author=S. M. Thompson;citation_author=P. D. Alderman;citation_title=Development and deployment of a portable field phenotyping platform.;citation_pages=965-975;citation_volume=56;citation_year=2016;citation_doi=10.2135/cropsci2015.05.0290" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=J. Crain;citation_author=S. Mondal;citation_author=J. E. Rutkoski;citation_author=R. Singh;citation_author=J. Poland;citation_title=Combining high-throughput phenotyping and genomic information to increase prediction and selection accuracy in wheat breeding.;citation_volume=11;citation_year=2018;citation_doi=10.3835/plantgenome2017.05.0043" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=J. Crossa;citation_author=P. Pérez;citation_author=J. Hickey;citation_author=J. Burgueño;citation_author=L. Ornella;citation_title=Genomic prediction in CIMMYT maize and wheat breeding programs.;citation_pages=48-60;citation_volume=112;citation_year=2014;citation_pmid=23572121;citation_doi=10.1038/hdy.2013.16" />
<meta name="citation_reference" content="citation_journal_title=Trends Plant Sci.;citation_author=J. Crossa;citation_author=P. Pérez-Rodríguez;citation_author=J. Cuevas;citation_author=O. Montesinos-López;citation_author=D. Jarquín;citation_title=Genomic selection in plant breeding: methods, models, and perspectives.;citation_pages=961-975;citation_volume=22;citation_year=2017;citation_doi=10.1016/j.tplants.2017.08.011" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=H. D. Daetwyler;citation_author=M. P. L. Calus;citation_author=R. Pong-Wong;citation_author=G. de los Campos;citation_author=J. M. Hickey;citation_title=Genomic prediction in animals and plants: simulation of data, validation, reporting, and benchmarking.;citation_volume=193;citation_year=2013;citation_pmid=23222650;citation_doi=10.1534/genetics.112.147983" />
<meta name="citation_reference" content="de los Campos, G., and P. Pérez-Rodríguez, 2014 Bayesian generalized linear regression. R package version 1.0.5. URL https://CRAN.R-project.org/package=BGLR." />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=R. J. Elshire;citation_author=J. C. Glaubitz;citation_author=Q. Sun;citation_author=J. A. Poland;citation_author=K. Kawamoto;citation_title=A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species.;citation_pages=e19379-e19379;citation_volume=6;citation_year=2011;citation_issue=5;citation_pmid=21573248;citation_doi=10.1371/journal.pone.0019379" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=J. B. Endelman;citation_title=Ridge regression and other kernels for genomic selection with R package rrBLUP.;citation_pages=250-255;citation_volume=4;citation_year=2011;citation_doi=10.3835/plantgenome2011.08.0024" />
<meta name="citation_reference" content="citation_journal_title=Genet.;citation_author=J. B. Endelman;citation_author=J. L. Jannink;citation_title=Shrinkage estimation of the realized relationship matrix. G3 Genes Genom.;citation_pages=1405-1413;citation_volume=2;citation_year=2012;citation_doi=10.1534/g3.112.004259" />
<meta name="citation_reference" content="citation_journal_title=Trends in plant science;citation_journal_abbrev=Trends Plant Sci;citation_author=R. T. Furbank;citation_author=M. Tester;citation_title=Phenomics--technologies to relieve the phenotyping bottleneck.;citation_pages=635-644;citation_volume=16;citation_year=2011;citation_issue=12;citation_pmid=22074787;citation_doi=10.1016/j.tplants.2011.09.005" />
<meta name="citation_reference" content="citation_title=ASReml user guide release 4.1 structural specification;citation_author=A. R. Gilmour;citation_author=B. J. Gogel;citation_author=B. R. Cullis;citation_author=S. J. Welham;citation_author=R. Thompson;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Int. J. Remote Sens.;citation_author=Y. M. Govaerts;citation_author=M. M. Verstraete;citation_author=B. Pinty;citation_author=N. Gobron;citation_title=Designing optimal spectral indices: a feasibility and proof of concept study.;citation_pages=1853-1873;citation_volume=20;citation_year=1999;citation_doi=10.1080/014311699212524" />
<meta name="citation_reference" content="citation_title=A simple model of the atmospheric radiative transfer of sunshine: algorithms and performance assessment;citation_author=C. Gueymard;citation_year=1995" />
<meta name="citation_reference" content="citation_journal_title=Energy;citation_author=C. A. Gueymard;citation_title=Interdisciplinary applications of a versatile spectral solar irradiance model: a review.;citation_pages=1551-1576;citation_volume=30;citation_year=2005;citation_doi=10.1016/j.energy.2004.04.032" />
<meta name="citation_reference" content="citation_journal_title=Plant Methods;citation_author=A. Haghighattalab;citation_author=L. González-Pérez;citation_author=S. Mondal;citation_author=D. Singh;citation_author=D. Schinstock;citation_title=Application of unmanned aerial systems for high throughput phenotyping of large wheat breeding nurseries.;citation_pages=35;citation_volume=12;citation_year=2016;citation_doi=10.1186/s13007-016-0134-6" />
<meta name="citation_reference" content="citation_journal_title=Remote Sens. Environ.;citation_author=P. M. Hansen;citation_author=J. K. Schjoerring;citation_title=REFLECTANCE MEASUREMENT OF CANOPY BIOMASS AND NITROGEN STATUS IN WHEAT CROPS USING NORMALIZED DIFFERENCE VEGETATION INDICES AND PARTIAL LEAST SQUARES REGRESSION;citation_pages=542-553;citation_volume=86;citation_year=2003;citation_doi=10.1016/S0034-4257(03)00131-7" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=N. Heslot;citation_author=J. Jannink;citation_author=M. E. Sorrells;citation_title=Using genomic prediction to characterize environments and optimize prediction accuracy in applied breeding data.;citation_pages=921-933;citation_volume=53;citation_year=2013;citation_doi=10.2135/cropsci2012.07.0420" />
<meta name="citation_reference" content="citation_journal_title=Annual review of plant biology;citation_journal_abbrev=Annu Rev Plant Biol;citation_author=P. Horton;citation_author=A. V. Ruban;citation_author=R. G. Walters;citation_title=REGULATION OF LIGHT HARVESTING IN GREEN PLANTS;citation_pages=655-684;citation_volume=47;citation_year=1996;citation_issue=1;citation_pmid=15012304;citation_doi=10.1146/annurev.arplant.47.1.655" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=D. Jarquín;citation_author=J. Crossa;citation_author=X. Lacaze;citation_author=P. Du Cheyron;citation_author=J. Daucourt;citation_title=A reaction norm model for genomic selection using high-dimensional genomic and environmental data.;citation_pages=595-607;citation_volume=127;citation_year=2014;citation_pmid=24337101;citation_doi=10.1007/s00122-013-2243-1" />
<meta name="citation_reference" content="citation_journal_title=Int. J. Remote Sens.;citation_author=M. Labus;citation_author=G. Nielsen;citation_author=R. Lawrence;citation_author=R. Engel;citation_author=D. Long;citation_title=Wheat yield estimates using multi-temporal NDVI satellite imagery.;citation_pages=4169-4180;citation_volume=23;citation_year=2002;citation_doi=10.1080/01431160110107653" />
<meta name="citation_reference" content="citation_journal_title=Sensors (Basel);citation_author=L. Li;citation_author=Q. Zhang;citation_author=D. Huang;citation_title=A review of imaging techniques for plant phenotyping.;citation_pages=20078-20111;citation_volume=14;citation_year=2014;citation_pmid=25347588;citation_doi=10.3390/s141120078" />
<meta name="citation_reference" content="citation_journal_title=Methods Enzymol.;citation_author=H. K. Lichtenthaler;citation_title=Chlorophylls and carotenoids: pigments of photosynthetic biomembranes.;citation_pages=350-382;citation_volume=148;citation_year=1987;citation_doi=10.1016/0076-6879(87)48036-1" />
<meta name="citation_reference" content="citation_journal_title=Genet.;citation_author=M. López-Cruz;citation_author=J. Crossa;citation_author=D. Bonnett;citation_author=S. Dreisigacker;citation_author=J. Poland;citation_title=Increased prediction accuracy in wheat breeding trials using markers × environment interaction genomic selection model. G3 Genes Genom.;citation_pages=569-582;citation_volume=5;citation_year=2015;citation_doi=10.1534/g3.114.016097" />
<meta name="citation_reference" content="citation_journal_title=Sensors (Basel);citation_author=A. K. Mahlein;citation_author=S. Hammersley;citation_author=E. C. Oerke;citation_author=H. W. Dehne;citation_author=H. Goldbach;citation_title=Supplemental blue LED lighting array to improve signal quality in hyperspectral imaging of plants.;citation_pages=12834-12840;citation_volume=15;citation_year=2015;citation_doi=10.3390/s150612834" />
<meta name="citation_reference" content="citation_title=International Crop Information System. Technical Development Manual, version VI;citation_title=Applications of the genealogy management system;citation_author=C. G. McLaren;citation_author=L. Ramos;citation_author=C. Lopez;citation_author=W. Eusebio;citation_pages=5.8-5.13;citation_year=2000" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=T. H. E. Meuwissen;citation_author=B. J. Hayes;citation_author=M. E. Goddard;citation_title=Prediction of Total Genetic Value Using Genome-Wide Dense Marker Maps;citation_pages=1819-1829;citation_volume=157;citation_year=2001;citation_issue=4;citation_pmid=11290733;citation_doi=10.1093/nar/gkv1344" />
<meta name="citation_reference" content="citation_journal_title=Plant Methods;citation_author=O. A. Montesinos-López;citation_author=A. Montesinos-López;citation_author=J. Crossa;citation_author=G. de los Campos;citation_author=G. Alvarado;citation_title=Predicting grain yield using canopy hyperspectral reflectance in wheat breeding data.;citation_pages=4;citation_volume=13;citation_year=2017;citation_doi=10.1186/s13007-016-0154-2" />
<meta name="citation_reference" content="citation_journal_title=Plant Methods;citation_author=A. Montesinos-López;citation_author=O. A. Montesinos-López;citation_author=J. Cuevas;citation_author=W. A. Mata-López;citation_author=J. Burgueño;citation_title=Genomic Bayesian functional regression models with interactions for predicting wheat grain yield using hyper-spectral image data.;citation_pages=62;citation_volume=13;citation_year=2017;citation_doi=10.1186/s13007-017-0212-4" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiol.;citation_author=D. Pauli;citation_author=S. C. Chapman;citation_author=R. Bart;citation_author=C. N. Topp;citation_author=C. J. Lawrence-Dill;citation_title=The quest for understanding phenotypic variation via integrated approaches in the field environment.;citation_volume=172;citation_year=2016;citation_pmid=27482076;citation_doi=10.1104/pp.16.00592" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=P. Pérez-Rodríguez;citation_author=J. Crossa;citation_author=J. Rutkoski;citation_author=J. Poland;citation_author=R. Singh;citation_title=Single-step genomic and pedigree genotype × environment interaction models for predicting wheat lines in international environments.;citation_pages=2;citation_volume=10;citation_year=2017;citation_doi=10.3835/plantgenome2016.09.0089" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=J. Poland;citation_author=J. Endelman;citation_author=J. Dawson;citation_author=J. Rutkoski;citation_author=S. Wu;citation_title=Genomic selection in wheat breeding using genotyping-by-sequencing.;citation_pages=103-113;citation_volume=5;citation_year=2012;citation_doi=10.3835/plantgenome2012.06.0006" />
<meta name="citation_reference" content="R Core Team, 2018 R: A language and environment for statistical computing. R Foundation for Statistical Computing, Vienna, Austria. URL https://www.R-project.org/." />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=C. Riedelsheimer;citation_author=A. Czedik-Eysenberg;citation_author=C. Grieder;citation_author=J. Lisec;citation_author=F. Technow;citation_title=Genomic and metabolic prediction of complex heterotic traits in hybrid maize.;citation_pages=217-220;citation_volume=44;citation_year=2012;citation_issue=2;citation_pmid=22246502;citation_doi=10.1038/ng.1033" />
<meta name="citation_reference" content="citation_journal_title=Genet.;citation_author=R. Rincent;citation_author=J. P. Charpentier;citation_author=P. Faivre-Rampant;citation_author=E. Paux;citation_author=J. Le Gouis;citation_title=Phenomic selection is a low-cost and high-throughput method based on indirect predictions: proof of concept on wheat and poplar. G3 Genes Genom.;citation_volume=8;citation_year=2018;citation_doi=10.1534/g3.118.200760" />
<meta name="citation_reference" content="citation_journal_title=Remote Sens.;citation_author=F. Rodrigues;citation_author=G. Blasch;citation_author=P. Defourny;citation_author=J. Ortiz-Monasterio;citation_author=U. Schulthess;citation_title=Multi-temporal and spectral analysis of high-resolution hyperspectral airborne imagery for precision agriculture: assessment of wheat grain yield and protein content.;citation_pages=930;citation_volume=10;citation_year=2018;citation_doi=10.3390/rs10060930" />
<meta name="citation_reference" content="citation_journal_title=Genet.;citation_author=J. E. Rutkoski;citation_author=J. A. Poland;citation_author=S. Mondal;citation_author=E. Autrique;citation_author=L. González-Pérez;citation_title=Canopy temperature and vegetation indices from high-throughput phenotyping improve accuracy of pedigree and genomic selection for grain yield in wheat. G3 Genes Genom.;citation_pages=2799-2808;citation_volume=6;citation_year=2016;citation_doi=10.1534/g3.116.032888" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=J. Sun;citation_author=J. E. Rutkoski;citation_author=J. A. Poland;citation_author=J. Crossa;citation_author=J. Jannink;citation_title=Multitrait, random regression, or simple repeatability model in high-throughput phenotyping data improve genomic prediction for wheat grain yield.;citation_pages=1;citation_volume=10;citation_year=2017;citation_doi=10.3835/plantgenome2016.11.0111" />
<meta name="citation_reference" content="citation_journal_title=Field Crops Res.;citation_author=N. Verhulst;citation_author=B. Govaerts;citation_author=V. Nelissen;citation_author=K. D. Sayre;citation_author=J. Crossa;citation_title=The effect of tillage, crop rotation and residue management on maize and wheat growth and development evaluated with an optical sensor.;citation_pages=58-67;citation_volume=120;citation_year=2011;citation_doi=10.1016/j.fcr.2010.08.012" />
<meta name="citation_reference" content="citation_journal_title=Agronomy Journal;citation_journal_abbrev=Agron. J.;citation_author=A. Viña;citation_author=A. A. Gitelson;citation_author=D. C. Rundquist;citation_author=G. Keydan;citation_author=B. Leavitt;citation_title=Monitoring Maize (Zea mays L.) Phenology with Remote Sensing;citation_pages=1139-1147;citation_volume=96;citation_year=2004;citation_issue=4;citation_doi=10.2134/agronj2004.1139" />
<meta name="citation_reference" content="citation_journal_title=Remote Sens. Environ.;citation_author=A. Viña;citation_author=A. A. Gitelson;citation_author=A. L. Nguy-Robertson;citation_author=Y. Peng;citation_title=Comparison of different vegetation indices for the remote assessments of green leaf area index of crops.;citation_pages=3468-3478;citation_volume=115;citation_year=2011;citation_doi=10.1016/j.rse.2011.08.010" />
<meta name="citation_reference" content="citation_journal_title=Field Crops Res.;citation_author=J. W. White;citation_author=P. Andrade-Sanchez;citation_author=M. A. Gore;citation_author=K. F. Bronson;citation_author=T. A. Coffelt;citation_title=Field-based phenomics for plant genetics research.;citation_pages=101-112;citation_volume=133;citation_year=2012;citation_doi=10.1016/j.fcr.2012.04.003" />
<meta name="citation_reference" content="citation_journal_title=Remote Sens. Environ.;citation_author=X. Zhang;citation_author=M. A. Friedl;citation_author=C. B. Schaaf;citation_author=A. H. Strahler;citation_author=J. C. F. Hodges;citation_title=Monitoring vegetation phenology using MODIS.;citation_pages=471-475;citation_volume=84;citation_year=2003;citation_doi=10.1016/S0034-4257(02)00135-9" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Hyperspectral Reflectance-Derived Relationship Matrices for Genomic Prediction of Grain Yield in Wheat" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4.cover-source.jpg" />
<meta name="twitter:description" content="Hyperspectral reflectance phenotyping and genomic selection are two emerging technologies that have the potential to increase plant breeding efficiency by improving prediction accuracy for grain yield. Hyperspectral cameras quantify canopy reflectance across a wide range of wavelengths that are associated with numerous biophysical and biochemical processes in plants. Genomic selection models utilize genome-wide marker or pedigree information to predict the genetic values of breeding lines. In this study, we propose a multi-kernel GBLUP approach to genomic selection that uses genomic marker-, pedigree-, and hyperspectral reflectance-derived relationship matrices to model the genetic main effects and genotype × environment ( G  ×  E ) interactions across environments within a bread wheat ( Triticum aestivum L.) breeding program. We utilized an airplane equipped with a hyperspectral camera to phenotype five differentially managed treatments of the yield trials conducted by the Bread Wheat Improvement Program of the International Maize and Wheat Improvement Center (CIMMYT) at Ciudad Obregón, México over four breeding cycles. We observed that single-kernel models using hyperspectral reflectance-derived relationship matrices performed similarly or superior to marker- and pedigree-based genomic selection models when predicting within and across environments. Multi-kernel models combining marker/pedigree information with hyperspectral reflectance phentoypes had the highest prediction accuracies; however, improvements in accuracy over marker- and pedigree-based models were marginal when correcting for days to heading. Our results demonstrate the potential of using hyperspectral imaging to predict grain yield within a multi-environment context and also support further studies on the integration of hyperspectral reflectance phenotyping into breeding programs." />
<meta name="og-title" property="og:title" content="Hyperspectral Reflectance-Derived Relationship Matrices for Genomic Prediction of Grain Yield in Wheat" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/9/4/1231" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Hyperspectral reflectance phenotyping and genomic selection are two emerging technologies that have the potential to increase plant breeding efficiency by improving prediction accuracy for grain yield. Hyperspectral cameras quantify canopy reflectance across a wide range of wavelengths that are associated with numerous biophysical and biochemical processes in plants. Genomic selection models utilize genome-wide marker or pedigree information to predict the genetic values of breeding lines. In this study, we propose a multi-kernel GBLUP approach to genomic selection that uses genomic marker-, pedigree-, and hyperspectral reflectance-derived relationship matrices to model the genetic main effects and genotype × environment ( G  ×  E ) interactions across environments within a bread wheat ( Triticum aestivum L.) breeding program. We utilized an airplane equipped with a hyperspectral camera to phenotype five differentially managed treatments of the yield trials conducted by the Bread Wheat Improvement Program of the International Maize and Wheat Improvement Center (CIMMYT) at Ciudad Obregón, México over four breeding cycles. We observed that single-kernel models using hyperspectral reflectance-derived relationship matrices performed similarly or superior to marker- and pedigree-based genomic selection models when predicting within and across environments. Multi-kernel models combining marker/pedigree information with hyperspectral reflectance phentoypes had the highest prediction accuracies; however, improvements in accuracy over marker- and pedigree-based models were marginal when correcting for days to heading. Our results demonstrate the potential of using hyperspectral imaging to predict grain yield within a multi-environment context and also support further studies on the integration of hyperspectral reflectance phenotyping into breeding programs." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4.cover-source.jpg" />
    <title>Hyperspectral Reflectance-Derived Relationship Matrices for Genomic Prediction of Grain Yield in Wheat | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"dpnWem353drf2LXEkyqrotob1qED50MNEJ3ZD7P-YYA","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"492620","apath":"\/ggg\/9\/4\/1231.atom","pisa":"ggg;9\/4\/1231","ac":{"\/ggg\/9\/4\/1231.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/9\/4\/1231.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;9\/4\/1231"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;9\/4\/1231"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/9\/4\/1231":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022knPmXl+zE83x3vxNw6hHXETWG\\\\\\\/necGhawwbLDFIFncPVWsk+Bqz\\\\\\\/4khDUdTEx\\\\\\\/aR+i4pYL6QrHFdePJmrFmXdVXbDazQuWARwBQZevYjZ2oOv9R4ymhAltzA+T6dWqy8FcoKbCNkHyZJdNLorWQp7LyadQ+aWt8D\\\\\\\/vhGq64bm1jtWfSJjSNfiagZ1cPnKyibRmbb\\\\\\\/eDh3DlWiIbRaT2vdGRtrcQjKhEI9dMrYD\\\\\\\/LgIXxa7\\\\\\\/R3PILGj+x9hkeRDreYk7v8ZrCz6PBzAzO7H05eO0aeQlZHK46NsBZ\\\\\\\/rGQqmhIRu2qPHyaJ1UnjWB6y\\\\\\\/RXhf2UHczZjEayQWIG1IRebyIAvziuzyljPxV4Gn+5jSW+5iq884mt\\\\\\\/N1o7NBIkFduBQw396tIb3\\\\\\\/rr2ki7b\\\\\\\/lueHhzBTJq4vl5twVbTFC9lNBo7\\\\\\\/QlytvrdZq\\\\\\\/1POGxenJsI063OJn0DtGnENYM4RjOYWtNVc6nH7jitKnr+H81DmmTB9vCO0xXQVGWb+0vBTFXaAVq6FciphWej9LQ2sKsYwttdNdo3x6U4QA7uKqPE+YaK7FPqoIxMgI4IfWGtfhKQI\\\\\\\/Y+Q6R0qEW7\\\\\\\/OV8WsO10mPFws7qwt67EbEfZ4FvhPWQ2onqUZRiJpvx2xiMqhn6ynNf4mC8\\\\\\\/2IpNE6WleFsznRIAKB0y9Qa0QjD2rROx7QQZ1FrLxW+z25W8OHECSvF9b1RCZ+ESl08GIzaEuatnU5hOeeYRTyzY2ULIuGsCgwucj1ebquBD1FEAgpEY5qsZs2\\\\\\\/2V5Wt9yunZOacN0vrbNOeGx1U4wb9STkI\\\\\\\/wovIieIyTZo7UEvnoYFDGQzSar\\\\\\\/t3OwvvdSEZD6zaJfs7tMDdJYB06UrDXQAjuSFA1EbSi77V0w+QAxo851hgz2PX1jDtvaBo0PK1Kvv2SI63Uc7CFm3LD0N8vobKRINW2ujQgwFDcdzxoRrk4E\\\\\\\/JZ8vtH6A9ww5A+3R\\\\\\\/u++9v8pwR9C2W+qx57k8fHXfyrnIAzEYCRLktcTnUHkK5mNq4L55FuB+cQuQnclbTE+TBx9EXecfydspyOyC+JTdAu65soG0KU5ODfISXbjY5kH5wTbpN4DajM71x3Yi4PsUsngx9ANEylV1GggVk2K6Txo4QdqDEIBa7bySoIdzRBCkulp9g3GOVH6gg2EHHd\\\\\\\/qRDsh1otHS2b0Z5Y+y7bCckMWyiWkPlXUq2jA4kDrXJKQzZyF1CrNIbl5RcPIYxUOGPs+f8BtpGJFTsHiTvNdEHqvtwfTqegEAvu\\\\\\\/biPrtxaaI9\\\u0022,\\\u0022iv\\\u0022:\\\u0022P0JXqOyb59iJVBNpKEpq4A==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002279902d22ddf3757f464838bcd1286d4ae9ade6b9b14253b3f5b5b4db6373a156\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022xtwqiUSF3fKhnwlwTJuJn1E7Z+ugbcRBo1RkQbjUbCZl\\\\\\\/4g8vP3O+\\\\\\\/0VV5XItxl2tR63FzFLWlRw9qbtqmpwMYGRVHz10neyQTr693pPu9vc\\\\\\\/WdEKtqUMrupEUjm2bI7GUo2izGoS28BITurWIRif98zixmKhw9YKCZ6DRoNKLQLXSUOnH4UkLsyFpD0qYOc\\\\\\\/g\\\\\\\/b7VY743+IcvCKh\\\\\\\/npJGKwiBGeJdE5wXYqmE5Q80O9SmKx1sBI0+MEQqjJuxpJYtCWqGPWodCrLElpMOefiAMZvz++RLr+Axn7dh9pZlxBsLfqZOGVIhrExDMzh0MqLEdBs\\\\\\\/DWMLytL0nSwBzI+1j72hodQOnHuhC\\\\\\\/zYo9TMS5Ym7kDMYHhuqIfTIYlq2Qvu\\\\\\\/W+8XopWajDTrExAIpss3yIjTHQohyaBNOje\\\\\\\/Vw5sGu5GSAnrwoTv0afEd2vlkQyJnk9leUMxYcm9ypfgRjIzo2Qpg3fZUteC6YF3SxCcUEzRsIt6toWhunjSQSkZrY2cXn5i5zrDpsaI0VJy1Faz3MCrvaA80TqmLxpdZh96gfi6CukYtbHuxsU33ecyWQsPZYrqClqAWuI+KQ1WdpANFZUEGaGZCADpV9HYzA81TRdcyUYaLDVWE20XjQTIh4JP1lBI7ynCLCCWcN\\\\\\\/XiQYICJZS3pth5AWyYtVPDIYjyYJUnOG0BGitPGPPEFp68HOiLgRFTA6l1n3O+LGpivhP36ZNccQznRi\\\\\\\/bIA7E80tZrYML416w3d6gxAWfRog4F7UsMAOkdoGIfPh6Uj1pIYfj\\\\\\\/MdBsBDOGZD7Joh3KMUXLiWDbhH2fO+VD89qYvlLnUp7a04fZk3AakYfUVnBM4qfP+eOpfAFJdiwNIdE1SE\\\\\\\/+KQkYbFcGEZnquaANZYH9nAA6hLbEq5HgS6ddkBfNloW\\\\\\\/u4GVdeLQETQAUH1T\\\\\\\/rzMaHYqS8PdVHSYDkNORVpzwstP2UZYf9uZtcdwdEMfzvitW+JDsj2RMFFZuJOT3Sy8k6YmYq8naWrWmK\\\\\\\/XOVD2Wsu0z6fHH\\\\\\\/7ux+8Q4BrZfXeye2VjZsKcMo\\\\\\\/0QVZg\\\\\\\/YvPOXsJzkK8+WUiH2ejMDWmWU1vksBDL4cQ6i\\\\\\\/s+APW7mYoEdCW2nGakZRATk+2cggevtdwpLJkKCX9oCfJprSBjj5jhfpgosPjp+pdTiiBQVETpPnRPa6mBAwlqiMN5QlVKB7khkIB6imCbScj3tPicGt4b6yNF8yXKwWLcMEgB6djdKE8NaRLNpo719nWOJOeXT2V\\\\\\\/fW2062DzSEJ\\\\\\\/k81YImVmI1vh+w81Wzb9Gp7fYDkCj1c+YG4j4uNil9DDnsueecKn4i297SabyWQDpoWaqPqxLaAPqjyHIC9TFnmzyeI854VwpdwT4qz\\\\\\\/JkEI3CakR2N6a6iXqAQruN4QfMfw\\\\\\\/D+f0REVB4wMF51CHSzEgMtGWCOF7AZlsFrFW+ut61ZXjYwrakRQFOzw08XIrR+qnPSRLDfWo78LHVIorOYIXGbhBW26aYTKs4kGS+NNRJuI6bfqXrACekrIVelweWQvFX9mc01ygBdaVJfT3Bc3g\\\\\\\/yZXg0B0zGRkRcXAAiNgvs03qE4ad30GIBVKGqJlgJnjVEY7fPvvyWbm6nozgCbndtAs5S26V0yrBwgybT0wMifQs2uaErQxzt4SdN4hILLZlRJc4ytKm4rCBUvv7T6mh9\\\\\\\/eEJs6czfWMvN1AEy4KP4OcrkdBZsXvudTu1XGflBN8bFTg6Iy5W3Ypob12GmZoW1Swo6ZrVy+Rt33WN+Kp9s6CM\\\\\\\/QJEqNe38fIZkJA5+Ypi9AmWys9RkzJ7S6RRU9dpoVsI2DgDGy9XbiUUTrQvEmJEM1wy5G4Rqt\\\\\\\/y9PAvwrb1sActCkghgfOvk3IMCozP6jQ1M4oIcQpeeKrvT\\\\\\\/CEiP5ugwjTe9sSNHWmjLX\\\\\\\/\\\\\\\/y2nwOEiYrq4VwiI2RT8MSA2w3i9gltE+Lfz+nr\\\\\\\/6i5zJ82noQiAw2647tfU2y2I3iWXtIDLRji6LrJwMslVIDEeas2sH+NSunH72ryxJGXcUIcfTgiQFvP2zBe+4AWFROusUPy86WVr4A+lj1fBknm6R4VybUKvzOfGBBBI1zsgc67pWhel\\\\\\\/e4UDmtAoYNJgarBrkcAhLN\\\\\\\/\\\\\\\/UlMQB0cXiKDYf3ed+qywWp1YXTuCgMpIaJj3nAPWsFWQGG3Rsxo4+vqgXpTqtWrRvzRDuquAZaTvzmjaYY7khYaKEY21MkUb7moP3OH43d\\\\\\\/0BpAXdYxGnySlCIc1vJK7GCeEtYY3dNFCwwmy\\\\\\\/4XTg5skOefubzFtEV6DYDgukjnsj2fyIyQtgpslny4Yql\\\\\\\/a6ELDiURj2FUDD8wTu+hzpADYIPdikzKiH2zS\\\\\\\/Sqh3xLrjGBKTnTaZgZ+XWL7msiRztkTGDGfSndjabGFiRqNNyQ+izCH+bkL9f3WTGzW6OwlD1bAGozJgIjUDyAwSDCH5EoMSuQcI7eYSRTgMzDnLJWKzxrLaCW6ghobv9DfNWbVlZ+1ffWi8HFX\\\\\\\/+l7ArA7OycRsflO82F\\\\\\\/RvgvBlP1rZ\\\\\\\/CxQAE6B2O4E1NFVAWaSrmguFq74T+tj1bKQ2B23PLKh+MGVdMR53JdoHpnN3iYpTDrEhe9NYGFZbgoN251yPBaT\\\\\\\/BZPG2xSWoO0oGOeM38NxGgMTyGkqQKEnW+4xnbmOVE6Fm0TOPlhCApGC+yK3zwKlLffNeM39Tmfw08G7khQJj2n+QNa5ejANjLSi49dj\\\\\\\/Q8dhfQ87Iev1S72q85rjq4QSKl+ee1xu7sROsrMHy7cVUGOKJP4J6CxG5cVGIgVYA1hy3p5wtoQ+g9xEO34mTQeau0HbqUHUkxwkO3GuDl60uFVj6j5eA0MZUQLUlltFJnfqCxBSngwnobRnzZCgm7qgtHDblAwNYL4LLPIobAoxWFJQuTRdaLTl2T9OUBE8k8W\\\\\\\/oWk0XS8mx7rtj66PyTrf9JsBkopCoI+fqyCtoyUqyl2Icd7VRvIocBANdTzRFxTADl2YN0u42EzicAQdzLVymltfr4mzhrAqBytxtyfl9rFzGPKzO4765KMbA2A\\\\\\\/8lGtsEzyS1L\\\\\\\/+nqbbEmdzqUZiZzF1XBrVATZPybn+rPBjkT9JGfX7CagZ4XhZtrSzoTJwm5e1AYONGWezt5iCiG+K87Xe9F68V+ut1Gp2gfAgdm3U+flmRJw7b9Tn5WD\\\\\\\/zil0m\\\\\\\/ckbCb32PMmeM6sYuuKEA9bCl04sZbHmgmTeJb91741or4sx+yNAy4HkjM94Yqtnu3oN7aASXA4YeyU\\\\\\\/\\\\\\\/z2ioggF+sUv9MSNLWK4ZDRGKlGTz2VXDg9pJXnmUmwgiNMPu+m9qVgPTyfBjJaoXZ2g+zoHQDf1zYg0vpPo1pTKE5teWLlX0ExmAddy6+tv3rTrfXgkWQFNaPMMbmzCl3Wu9MK9XCuwrPVTxocZE9YhPD1Y7zzilU\\\\\\\/BHMmOyKLoG1TgJ+fRsaQFtNOJUhy5llTs3llca+UptE\\\\\\\/MIMKhyVmPDdaPQHbhEkzQQAuuqnOgBtKi7RVJV8Jzhb37EpLrr2T5WkmY2tOCZIy7V50bO7wauRCLg0nrTMpo9QJXvelwHqgWacaeI+74+351P3irX7sRKdTSqP81\\\\\\\/rRSD\\\\\\\/1zXBd9Inku0+1wUmd4Fej0g3vfqoR0utz\\\\\\\/uKnaEW7a\\\\\\\/8ghbt0WczPwF03Nn9ExjDFJGL5ZJ3cmNxXc1OjxY4f5\\\\\\\/F+bH1mQnMlupZYTIC3B4ketlmljIs0IuYkncFRUyq4DqDG54DISGP5cnCIT50p0sZn6sekJPE92CR9cB68jupvZhMcSVgHCkka9I5hXX0XGvUgAmRUWKcEzfK+OotVAR7PdUPzZeLSDARKP8FNKYSKAs4OnrPwfcKs0khCkrL6sN0X2X5PQd+AdOwzGpOdBGOPsO0TsOembg68gxivd5ZAR05ApgYeOaR25bnjcCncLGJwDBd+vI+nEcJFmyPPtsEfGbxDGTqXNdiXbgQVao5GYXApNm4JFfH0VIqfDI+uj9z7vjZyl3RIEVvcZ06GhxtE6CiYtELyrTth3zxLZHSzUmWPerWW7cV7oCWP0kN9azvLLJYoqAJyjr63qSZcOKhxD6OyjcvAG6\\\\\\\/RKYXX\\\\\\\/ZCFwv11bXfh3LBwF6KNILH\\\\\\\/CwieaWZvMNhYbI4Qqe5UKfZjg3NwV++kuwuD0zs0lLK8fxrirV\\\\\\\/iUVa7m7rJScfHnx413I0saa+fec3mOl32agdjphO1bA2LXI0YsQo8L6nWobhfzEbt9unhQWESr\\\\\\\/1MqgZvxDa3R2mOJ7qKfq6BSibqgiwOaJSdWjpaNmfWzK4m1mYrOOnAp+hNfAY7yZHvtJWFVbMFUj6jBMYKHS2TrjhPWXfrx9wUFd1wulJe8BH31xgK6I4HsYlD5RP9jBpQchUA6QdDuOXbusasNwTPU4fWv7WaZpT9dMpfN46oTMXSiGqnnGd2\\\\\\\/XVtKI0XKARpHViXD\\\\\\\/\\\\\\\/cknToMdrsU4FG+oUDnZTEvGmgMSA1fepB7h6rBCZO2wSxPsKBMRRSHWT2dZ97kiUuNYKLko+XpGDnTyS6e6R+cFfdy6nn1Inw8IQ\\\\\\\/PXZfpZyXi\\\\\\\/nj9scqoJ+Mj5yPjzjGkBWtS1ZarnTxviCOjf+MqstVCpyr7XYAHe4Mm87yxo6IoPkkgqBB5Ash1tlOxeCdRYvnv\\\\\\\/oV\\\\\\\/aP06YM90SXFxdfbvaoLwgcNfqjY3hKtThE8VVU2HDPIx4IDJ8ekJX6obLW19zhMtMYwcXQvg8cI04\\\\\\\/VNT6lfX6X1JmtSg7kIRRE7dFfqeqvauwTEVv+dzd6Ca9C5d+Fo9O+BcVb7fC20wqrQSNZ9C9RQMRZ\\\\\\\/UccJZMy++lEVo5ukGYzd9BqFtTSuHjA4Xtm7hatgoWBYup7s3Fg7ritYQMsg4s4O1P0bkV2pr0B8slmCleDyoGNjA38ODLPqGRHAYJpxLgaDv4CTvKYV0cyVk3Pouuk\\\\\\\/jTaCiQ2QSD8F1IBTc+7JGJqeqlZ9xniymyl0ETF5Qt9x8YH350BmerjtfHjfbPD8noFVWT75qEJq1jumuPbzyNKl9aoZuyWlJATBFlw2yry1TwgcDP8WnYBUpPfgkjAX\\\\\\\/IF8pDOy+hPPqBuq9SOoldu2\\\\\\\/mBHouYBpqN0bGk0GD7\\\\\\\/ZXgfw3ij8uXWS6ZC83hFfFQMJBTnUiq0VFtPgA\\\\\\\/q5UyU7g5cZogtKgkBFw8Hu3H1iBbVKTbFChY4yFE24BVBQiss5hgtAmZ3eEhhdyKIeia2drsg+1JymIEx1uwSPjQ2D81NmhQGabF66OiZARQ1i808Qqm\\\\\\\/x9rpmC5RlW9e9+8CT8ELv1RGTpK+tK6lxsM\\\\\\\/1vxbCrlaSLuLFKNb6UvygyJ5wSSxs8zTria0sATi5\\\\\\\/ngm0zujBoh+ygSN2QVeauKiOIVnpbRwIiLRt5zHn5OTcN8bMn\\\\\\\/9R6K9cJH8tEAzPqG3ugK9qco3LZXz6SmrMS8MUZx6b1t8qBTI8thQNAvvPtDFRMvoZgyl\\\\\\\/2cG8nWvV+XRqKrhP7ylzONzG6gkXXGhPnbhW3+bc9dvYlzw1nLckhULZutpZ6u1dD2bRlN2aGuaC8DJxDvwzJ0SU49SlNVRnxLgFwAXfXRrBSSiEgFvcK1bmGYYeHHyR3WqjzuOLIUkPlgqy\\\\\\\/mD\\\\\\\/cjLsPFKwvlBgLT859\\\\\\\/y+YK4MJxcdxpaJ0B92orWPYKK51di+DoJS8qnVTB8\\\\\\\/q1CzYHy1Ydsmsl+VWhc6Xpr9G4T6g9TlKU47a33HPjxmayjE\\\\\\\/dVwzsviwcX67f14q4azYzioIgbJirkoa4IX3d4ZNX+38FEKVrHuQwBUQEt1hlErOiK4i5FYdJUpmZNdGspgfIEYYG\\\\\\\/zcZ7+ymlZIBf0bE03VuNmVjs4WskSOO1B+47nd5ZwRdmEmv7RVvKNrL6AB4wAW2J4\\\\\\\/Dld7Kntu+LXT2UuypX\\\\\\\/XVE011Tnmyv8GSK5G0aHj\\\\\\\/5UDHKeJIP4DclR4V47k5OuMihUmNCliFPqe4hs5nNhxov8Sk2uj0euF3RinCpFlzOiolJjxptTYj1FPzj5xLSbIrO3tu9ddC50jMdtnoV\\\\\\\/QX0M8gFeDfkKiaUirtuh279vp1sGP1x\\\\\\\/k3xc6Ej9lMKBpE1NfHA1dajWNAusixNAE2IwzPPUJ5KN0t1j8ycaplmuTuPS4oPN5cu931jn1pktMvY2dvp63hcNAirIG1H+8Q2OicXohC57gzJPP1M11j4MQg1+mWr\\\\\\\/z4xRnZH0Yr4lfTQcTtkHIp9BnRHGWpTr1aXTJNnWifnXf24sNdsgAi+zqdJWsbemaSdaOL8WP4pKhGTgo4H4F\\\\\\\/BVaRsdegjazssI1BfnJDbZcnArL\\\\\\\/KzRJ2dcQNF\\\\\\\/y9Fv6TJoP8gFBGUANOYjp+qQ\\\\\\\/GaQGe5z34pWGwjT\\\\\\\/oe41WC7JBkYvdD4hXH2xxuI+Yt0HIEB8563ilwEvQgNszINi9KOgab6+BPadykUZg7J7veSxkC0eSgc04Fk1cXQw9woWspGzcJYMYKh2bpO7YLe5DSn1b5HCS2Aeq2AcJuzrCTkZaDJH5vwEDmoedY3hipC3dC8+39n46cNFLjut7ppcfkTU5bc40Xo+uHC5Pa9CQtaQectHauh+UUNoAZGbZgCUNDHUuP6YWdSxxKUHAv\\\\\\\/Jd1kgrWUxvstbXxE6U4l3L3dUQDAv32szAQmZGqJqObdu9V7ryk8NxgP1NtotZZHPdEUe8LzmV8ImzDi9kmc4QRpptd0JWHjJVxQHgIFP4cHuCurn\\\\\\\/l1qkvGXx+sDUXrIcq9WdFbdyISdlokBhYOVND8J2Zb5ud7g9C8\\\\\\\/ijoFxeaDoFqGRQBBzU12QN27UZyt3Vs4bOWWAks4LQr7Ihb2mhDxfS95c80c8A\\\\\\\/jXzRLzCNfsczyc+wJWFey7krrsCXpvZWCPXPA\\\\\\\/bwcMJn+Sn72xZBxUNvnY0+jy9WFs+BrKXjWrFtMlnS094U7pFhW5d72OPgTTbyMCfA5N8bS1D+BLqIHtneBr0FRj\\\\\\\/O2jn3ihO8WRlda57EWYOc+rtYBlznQ58UnWoqmrVcTRJdUhU3Tj9AGdK7\\\\\\\/wwLp\\\\\\\/DWOO7jMRcl31qVVyutT7STLORjWv53vc+1wVLuwO652HoaaTkRiKtBskZjne4CRHwofcgSCA783Qg4ikBZhtbHzsT5epjCCqcnk4VbgAVPXHUKBwjmrakP5aIZNuFtgVaPwnvU8xY35T4Lu\\\\\\\/Za8q+vy0HvCAEk6lfxqC9FCK9d2q3\\\\\\\/gi7quf1dI46RQSFh9naXaUuY6NjiGPmOuLGZjQYr\\\\\\\/FaRrGTMzNIMutpkNOuZLC9lXNrtyQEhsdz2Kwggwwt3C9jev7gfjoEbFwjgr\\\\\\\/VoKXVB+kcEn46O9TA98aT2POQMYO65rE6cE1TUJAhV7QrxyhC0m3OUt56P7x32\\\\\\\/fzIa0BN0A6yfA7m5RrgMoewzx+3HhqxBSuFXw6fbBNi2h0n+CwQJW2VV1RyLGPiE5NQY64JTwZtYaoZq5MRQ3C\\\\\\\/Ue3L5Xwc77I8SfGOw5MhHyoUJJFY1dO90S0er0ARAGVv1t6lJCv9aGcVIx4D73D9qPAo85IaKBi5nDsUFNlZouCpAiWEM6B3u07mV\\\\\\\/0WcIsBq7RIr5E3sT9OXJYUXCEZ6Eq1cOt4R33xL2BGobfOTi0GepxWTFbGEA30Epqw9eIkGH45+oa0TKM9j345bk0mhEJ9ckO5XkBxv0q70VIFrK1jhMHdMe2GgkJteaoMIWNqw4FIZxZd6OgBFfr1zTEK5P0my9H7Pq2Q\\\\\\\/3xjdXIkUQZ7FSrkKTJXkh+tA0VknXL+ttkRcWiZ+OXqy\\\\\\\/FG6OOpS+4oiH81xx0u9PtdBl83xGz41ccL7wHE17KQAHGtDc3AxAC1yo9RtI4rpVe\\\\\\\/oQiWFoEBl2ArMr60eT0qpskLG+cBkzKRDGLLf5c\\\\\\\/oWHI0RtFULaVCAWZZ8h2phNHKW17PrAjvwGk9rx4MgsvjC+CXNrZc19O2kUgNT\\\\\\\/XvWKvu6a4tiWpLGl1s3gM6Rab27TmcBDPenyLEDrQra44RfOfshuwuq193ppsE5bfC2OeuXDdsbUYcc5pGkqY2auYqF7SVBH9fyc+N5mwA4Kjvg63pJlWIsI1LZfMXC2PZAKx9GxXpg6cQMU4Kafl5S6vhyhtESVtVsv60qXYtiVBWDJOCi3rLPtXLL7\\\\\\\/Gjsn7WLlJDatLq\\\\\\\/fQWrQvHCZ2u6++aFsa9ke+WP0wSCyU\\\\\\\/ulBiG9r2Kt28nzzRLDaqEnzehTmQoe3c\\\\\\\/DILmmhNXSbk9jK86LtDQeyBV8CmICm634JUO48Y7MQgQHD\\\\\\\/PJ0kMfQG6zUo\\\\\\\/v5kM58ihxh1LCzThldtiq8I4Uh+F+XW4d9BscDSVWEtEikqtbYXtQoyCcXRH1hU7VANl9+dQ3Ccqm5kkrfQeqqiv1qvM8VOOvlzOldEUm2c1XpojEsroBkVi2vpbDFmQq2VJ51XYu8yHB6sex8AVj4ByR\\\\\\\/+Lr8XNsQonRCJU77W88x\\\\\\\/Jbuo9qveGF3gLVQO6yXTqU+gCXP8UV7ZEjr9IpNLCvdXcdqrM6g460Ry3zvFpaIF7GwGRxaqx+BHeWaW0nxnRaRYKLu8HnCpF+wdtJ6Zsg6n5rCNyorMzJguFfirS9\\\\\\\/9bg5ukaQCC6oN2o8J5VaoLPqrWFD3X00JwT7RFqy5uPCJC1FdWpqxo0LmGbosP8k6svHLzi\\\\\\\/DPedgE5mP+jqu9nNIH2USXQ9uVe6cdjamr1yamN7wr24NGw1rxEpFNpjTMG\\\\\\\/q88QEahaUqQCLrVfQbEBNEn\\\\\\\/9lJLYkd129XMoSOzCPYaAhCqFoYn9SvevdUffKT867pjfG3MhfArilThpr9IEM40qw5EOM0GaJaFjWnhYiPukBUvw543Wzvk0gT+jWCr9isrz0ZdjLtsRu6rybI3V\\\\\\\/1OEujSjPODSquN3ewbIy0EbL+psIPMerUt0b7e52E6zzF5KTLEiQDKFaA1Xazw7KPUNNvh701fz3+zkyZ1lNojrd8Y7n99pXpyvTJXMR8P9qPKHHmQCmy6EGam5x18h6Wsp1JNInW2j9\\\\\\\/QAA+bTeXBC5djKN8N\\\\\\\/Skz2icP8mJj6jt80E6KHgQ5iRuvRMrYyjuSQS+8D1u1lrK\\\\\\\/g6o12yTJuWMJroZCtz7bIhLIfzfkZygtVBLsyLNkbvpBjpF2MXdb+7w5hemKbsseoXr7ctfBJG5l479yn5GyJ24Y\\\\\\\/BopR8Nxa7DkENl7YKzg+RGKzdT0Lkp16FDTpJOIGANEu8lWZMad1+bic57ri8S\\\\\\\/aEgcnC1JuR9mSIZpXSgM3yv+jzcVmNLxZUo4sV5s9X718UKa0MJRWxJGoMG2bM5VGR1gDlM8Xp+J8fDrDMT8D5ye5xbD3Qf2cPFglbLPBMskEsmE2QJGFmUG9HRE8MpVcAsiyOq33B4YabRsG+Zgl4qIg+k469qEeTR\\\\\\\/B3hMkHghiNZL4LlQAHp5X1Ucsjbzel24GwrIIyn8RBwm74YKMdx9fEV9uxgN40HIdenfIwV0pObWzL47pFLkpEEZKP2OuVDOKfo06v73x+1fCA7Vqrc8\\\\\\\/A1\\\\\\\/8HvrtqzvYQbmGnF3VcuNh8dUw7GdmTKTUt7voqZMu7ckgF4ZbTCqQhPhMFQYqRewoQV64Fqzow1UPL\\\\\\\/I0OE8w8wUN3FyRM++qRXFUpCax6\\\\\\\/00bUYIt+X1nG4D8XKQaxRwkJWB81YNjyZXX85r1Himffx7\\\\\\\/5yBqjHTGeoIV8eidLXdIepzliXWq6wvrPtHAI9+Qeu0UaPcqRJ+fUQR5qbTbEQFxpq6BmW+fC3J\\\\\\\/xsHD9FElCgtDzHGL+4Zbvw1he8LvsoeM0CpvFc8AqGHs9GedbeiBxHnXO4RT6R\\\\\\\/14TQqljO8isd4AQbaetUzfHO7omoBJRCf82IwgGviZjRv4irwNP980xCyK49mtkU85vjjSGOBoFNT6Ql6bF8jJuoluTdQBVU8eXcFf0qhQCKo9\\\\\\\/Nu++YfRnKucXSoVtlff\\\\\\\/v0kE10i1YB6+gwrxExXgZf\\\\\\\/pGIf9vAuVxIw33Cyqwv2roHPak\\\\\\\/DoVCv5MBMPqc1bUiLEbTavb6iOCd2KqroujfI+MJ\\\\\\\/6Eq6z9IoKVDNgSzMF6w3uTTdIOrpoo7DXi1PH3V7288JQl7j8lMciuTUWTRl8L9E6k281QbtCu0afDaeEwOOXBkYLXctOKV4XGj0CaVBHxz1ANQXB0d5R92CF1AafnOs1BfRvb4PBLP5kp60ANb6J7D\\\\\\\/3AVGGRs89uxijSbJp6Vop8z5hps0LG3rLwY3jwfQD1o0kmOPBvSPB+nc1NmkFtzIH1CG0uldroqnQ6lY+nDvmWh6ZwQ+EdskwHoh9bwAVWmnmxnDfi3JqIzPSF36ucsjhm7pE3s21x99kyBa7JJ7yX4Fh6HADlBsweHDz0jenJfAUUIpdePYo+HOLfJ5nHrTtP5QFkBOI7eKT5YeE6S4QdTZzTO9CMnVcDklOh8dZDC0xes2c1AiEhUaO7iExCr9SQOjwvsojDpb1gxnyshBfv3ciMiFV3D475FObPH++UJG5+K8DEK9W8HmVJ3ffLRBYTtMg\\\\\\\/giPDf0RBfUyPAX0vmWeHLFUyPY+yF+TnJ7LDutdh9fIJP+G84CzgS2MHYL26arO4DmeObdxXlus9BBqHhSVAlJG8\\\\\\\/Fri1ZBzQ7cuBsqptr38BeSCxUY4EZxPhcwrOfOa8HoZVevVoVf9AQMUBbg1zE8HyXAgAQ5cUKr8FcEeUs6VwbrK0DFtAuFHw8jX03XRwTOI7JeG9vGFlrdtm\\\\\\\/8jDYOdCmR8H0Whk+vZDwtUTjSRjY+Zx1rS2S9Xt+MKrJudVhob3AYevW85zNQAX396+Ih38VEW+DkdfzHi489HwDSR0f5PZL5f3jpwZkzo8Tax1cKs4UOP0OL\\\\\\\/Ybt0DngfnAlJNDnfbSHsZ0ujdehiUkPQaW1+MOR\\\\\\\/VDLoHYAqFN6hYlzKKOUEPATvK72ci9gOGp5Jk\\\\\\\/qG+wojrtCXqfZAcuzlMlqGHFhoPSa5OAubjNKVCK\\\\\\\/FSoQSFQkGcyS\\\\\\\/DOLqyJkeMXAOXytyrBRRQI8nUpKVJXg3EnRrxHWedFsXCS85ULKOZ04riU+pcp\\\\\\\/zNCcZqXGB1glVoqyTZdURlLQqeE5gAPjhVRjniLArbb5Pr7D6iYnJMaZN5ovYubhV\\\\\\\/UVK5hgAQO7y8EYrqIPgbIlHppCn2UFcwqPNuArELHOGIU2mJbba+tHLATRDpWsWNDWcenx02EsKlwZz8xmz6TnbFswWbP2ZC0fxtTVzRTUnguMoqPBNenGJG23TrdVrGwZx6Q9o2EtUG2SdW13tGMwj9Evgd5bFc8uZMGGsuy+WArmJjl7z184C9p9JwBKWSgHWUxP3zIRZQKD3ROcKOtf0xKtfNkYrUxU37mw+23r5+2L5CjMjaADoFsqS1uonSKHnJTTvWavzUkfZx57kKkImjPtslwW1dHRQUL+kru69xcMMwX8TBWbyRMjtBXYSDlziVma4KemZuPLUphrXtQrkh4oCyvSv67of13O3uwDICkZ4QDNyK2F1\\\\\\\/8ZGZZy+W6zdGWt8mIr2\\\\\\\/OxmZ7EjdDTKUiZUHUIyV3TtKaq15X500T65hDRDiKCsxpQeJj8AmLaTOXAT+yeSNtsewFwRDhiFgSKoDQLgfmcqewOWyfjK8U5e\\\\\\\/18kJf8K8mPMXvDaHCyEqv4syks2fWusxOkiVqvUgsBQvNGys65t\\\\\\\/TiuoQDfG\\\\\\\/srVim+C0HNNGk8RY4pzZJtRj+VogW2S951bVUaKXNBZUYtWW8canBR99RBjaWCg30WVpRIrqc7EqF4rrSHvJP8mMxnQPsl32mqaRC\\\\\\\/+\\\\\\\/xbUN3sgpnVExf1RRwgTNbh1BWSz1be4DSNH8gJtT+McxIqIhML1Z1WQGavjjd433Ror1jheAPv1D8VDI4poAkOisY31LOcnHugvSAbKC+XQn0Js5hprWXxOeIws4I\\\\\\\/LB+hajo9jvGqKaHnVItTwg1232sxgB0L8o6DazKj\\\\\\\/YlwwI\\\\\\\/KeBqTWvsmnO8YKtyTV6CWcuJysmhuYT+\\\\\\\/fAq27s9+vz8VbpDMs+jjVjxIiKHMgjFPPL6V7SMBIEsIhdAOY\\\\\\\/ia7AZbVkmY6l5qr2QrSh+ls85F3CFtdZECXMdRXJ2XyHBvt9dKdm4ZGTMMj1NmKsH7G9tx25ADBOFvfLewb3FQz1ej1V8EyQY3tMsoVBDbqlOLm5qn6LJleyhGj3+TjfGHghrWScXFiJIHL9bak\\\\\\\/Vx9OUomDWGbMi8GWykhbjBCBk6qvk7iaETjHhr6DDZodcXJ8\\\\\\\/oZguJZid8Ymc59nFmq9MtyzOMe2sJGvB0oycUFPO3m5c\\\\\\\/cuKLz3W7F0uGMIY\\\\\\\/z3LtxGk2OyjqrFMvFeJlwjJtfeqv2GPHqBHMYsXsct6dLlM66227zfLDfvrXyDJ3\\\\\\\/q5wwBuN+aoXpiCxQSCuY+HSIz2VKaJtV24\\\\\\\/CHQzxMqX5aGSxrU2yZeKQEHZsCDf8sIFSunVhiRlzn9au8C67egJ6oG\\\\\\\/mwhP+TwSSl9WtB0Yd\\\\\\\/fyXRP0urzuQhR2oaQU8UnS64uFqglLp8P7drxYqOB2a3MwsJMMnKfEHAMtrJpp5RoJzB10SigY6WVh4DUPbWr1dC1b3nE9VDe6vKYPoDxf6o2CoE45wR4qpp7efx9Er1nKrx3yE0jwZMlXDMPFVvnCNmWJ1R6XixE3Jczncaid23R\\\\\\\/Yrmlj1Ec7x8YOD1nK3VZgsoc9on7JRdr0cX0\\\\\\\/8AIVByIA+xbW2iHkfIcyONSglBUiSWSExIphCedDY1Fyi6yN9KrPpgNQDPlFiFG56C9EyZc6YyAVzrERRmQIyaU1crQSxwVIYHy0MODaRtSFwg4turfjPmvmEsVYC\\\\\\\/s5W4zcPNIuMdhuVt8XUTy8hnRLEAnUlZcj5Oyxpy2t8od18ykZz\\\\\\\/EgydFgtV2SwGEUjINxNQYSTUkVVUuMe\\\\\\\/i+QH+CAtm6\\\\\\\/XXCKQjCNO3w8z3aQj3jlEGHFfymLreKsmUVgry6XcsWSlVh7GG45Am616xpvhSZvuXAavH\\\\\\\/2L4Ktwt9EtdThf5eXCKP\\\\\\\/k5ArX14ykDzRsInnanJe96vuEB3kxLh+5MfgFxkfloJP3BtQLwe94fFj7vG9LaMpyDnyifYBed\\\\\\\/ysSZvagVO79jxmlVB7CLFiRNpj23ZUiW9NkyVH\\\\\\\/mDpvQh5SvTlNRLmQsSK0i3u1iCmvHBU\\\\\\\/EaNno9jG8Q\\\\\\\/KI9rKH4RzcJLV1eGBBPbGsJn\\\\\\\/zPhR2LVvlNusQfFWTrfp7shlCXTnRpG8Jb6NIU7NdrjhxxGM+\\\\\\\/6rGv95i7Ng54V\\\\\\\/V8RKWjA8yG+o7yPPqEZuYjDXMwa3rr0igiO6ZRDWYSN2L3S1RiztPi2rgxynyQE6eaZTyNCrCaNlMgM5Hr+9VuZ6kgziNotAJyzAP0OeDmzzwdc2YG\\\\\\\/DggeIByVoeEs0UE40eUjWgLI632CN37ytAK66nuLNCiAY8c\\\\\\\/m4vOwqMTvTKWly9qG1IXwGfNxDssSXv+jJ8KxuZGt+iPgwwi0AYdW4AuYyV7alwiKSTrW1bVSarbQPt2yVmYF\\\\\\\/xTSIOSKvI++K9RFA8sg\\\\\\\/wy42oIsZdO3+46cKFsh2G08gAcXnkrRMneXjQzmQD\\\\\\\/g4DjO8UVvcVU+Zw6x7rC5+oKDxqidxAg1s7K1UMJ0QWZYdP4HAyWph4MK22HMgh+IJ4rm2h6T7zvqRjsI1+a02xQfzXmfrH+qY537Huh0CdhuknciFPiOpffLPw3INAetlGoxLRo4O2CG57Iawl\\\\\\\/quRYeuAggGPgEOKDl83DdgAzNpowEbmV6XK918X7oAP0Vrb\\\\\\\/xROQDK4wNIZfUvsT9TzCBPnSmeOs\\\\\\\/NxVZ4BiycQsijYCebQyuRwS+cCeDhtLBCctcPRDN8rr+ykyZKIRw7KF9g\\\\\\\/xHqcmbE+TZzn7ojGprQYO1R9RtUkXOid0B0JOlBrkHGQtG5XiHH7xtcD8lafm+fVPftxftJuisdUIvtSlrMddPxhNPWVGRWDmKPKcz+7\\\\\\\/fLdOYk1OxjlShJ8oW2ZQfaOLTQHBaH5FKEPc0bF+kzBxtaSmH4bP1JEdiNMqPJibcgFhDRJQcIrGZ3283+jwWzj6mCg+TtIebSKpnXqK4bb\\\\\\\/VP9GbIbA0282d8wNiZ\\\\\\\/ZSI6Gn+T6ZKSbuomGQ6aBBcXH84lTotDxZwnQRAlyJllBYfY84rvHvq6pD+y0vd3IHn94q49fz+sW93ca7UDqlf9J5EBYPceRecDAx3kJNPk\\\\\\\/3cCtWaSLwQau+e2KNOkg9hKl6oZuoe44zGJobrysy8+F4LA09VhUuZ2p50kNfsggofjPjGvjQ7slYvkZkvwoMH78L500VYlTxpEI5nTdwSYvsO2K3sOu8zHnawpWNtGrr56JUvp\\\\\\\/OmG8yFyu9un8ty94snzQ6MtCsg7zbPFmMLOrhV9i92uaU3U6bQsbVqRtCyAV14Cj3D6HcoH9g4j4b9rOBtx9FiBXOIQkgCDgCbHE0oOWSRFaglAJUnwxPJ62wJirIIyjizGxH3m1MR6ml1yb+wTbLeqWuIysCo8bAe\\\\\\\/\\\\\\\/gA5cpnu4tS1dub8StXj1szOJVsB0PizUTAg9CCEsmfWPtJIfjU+dRqTf7\\\\\\\/ahog2TWbi3Brd6I8vosYu3qumFNtl1XLlAxsMNGIhA5U9neC1OMFbfwYVq4VJrD8l1y+q56fuNZLbtvNkjg2bDiVv++tEX0P\\\\\\\/L3bycmwfm7fYW1QPW14i7qqVBFypOVxyk9z06V0sLBSR4dKTkDdYpvXkYtXcr20qNk6RD7FkPpAgRtWRn7Ld27\\\\\\\/wxNn2MVRcAC2G5jNMQ6\\\\\\\/cJJdvKec4+e4BAyvScd+sdnImjjsSwUSGSkjZOegTuXGh6cwx1W4IT2fjI5abT0faMEn+0YXTZlZrfcCilZVPZXFzJPlUBqiXFzHYOrsdFoiysIiSOgyW6bade7evlCm4ZJzWIB2gLjm8gCZpA48kPPSAT7SBbGCIBCzc26uRlwhx0XUK8\\\\\\\/C4CKek9lEWrHO2cbILnZSWtmXRrSAUYRRwEIZ5wfjN2tEGMqonGLXIhvTG1R9xH4m0fE4zSW9R1\\\\\\\/wm77CYuJCAKdqFSFI\\\\\\\/wMhB58IcQgT8TFIBGziHDL3Y8qwh2IiCU6bnAoxDFHIuexBov5zbn9bXT0FuCi78bNQ8kFmqA7HfpOAmrkxSIjUg8ELpD7dPdzK8uFBoJIIEFQxsIehrxgMBGxr8TwP5fpfXl7BjLRs3J8DTlDUBv2edL0pUCId9jw+Kwa3TjvTwiJAJx2leb1BcpcaeQFqLNsTfO9n000REfI1nHmOvCRw9ajt4KgC0Vroep\\\\\\\/J2AWobGdW8kUghYmUI+BcX7eGktwS2rtpZzY7yfFzRj8d6li\\\\\\\/+Y0WopdWOx7ot8A72RuiRGuEOMGq2OUpKlsUkmRX4aNDL81fd897WnH4z2wPBBtjtwUO56cveEW1Df5RXWDJNGm70C\\\\\\\/e8TMFwpOiH9RE3Chs+MDAzDWnYsXWoaAjhMnVVxx58XZzDCHuxBYXbSekJ5Yri+B1jIczqMGkQHZMusp0IO13fyk6UKSnAdssYwAZ9fittfEpCR16XGDtoh++hqvgJhj\\\\\\\/4LbCTkemr4scoE\\\\\\\/mzkWJIM+bZGj\\\\\\\/ICGile0gEo6AuTG3EvnSwjG7arROwubcfngofXOz83yzUXVE7bQ2yQdfc\\\\\\\/W7PRXmJKVjafybT8SBgfbzyWvAoV\\\\\\\/CpjnAYrv6wc0zx5Qbq6fojRyB2w\\\\\\\/\\\\\\\/e2elqWLYNL3G0fgTatDPqfPtABW7q02wWSdzMHlHSGyjM3GYVmioxpwx3ZosOXCzF6upDHdpxY99awxRvOHLKvZTmCpnMAQlPKOX3DNW8knqYbt7XVJ1ZdqVKuQfJlKNJeb1bepQdGHjrAzSi2UnQXqcpLCpsXjy1ht1MncBY1muEsuE1vMOY7v4TIwZIqRGoT8\\\\\\\/k24Zx6nhgEacJt\\\\\\\/R83sIBkWsuk1hWADzDtG1BhPKtfN2VZ9QSRB5ZtdoKr+PQEt44oxcFRDg50rgiYnhcF1mdfazLtte7TtG8aoEs2S7UZU3p\\\\\\\/+XsrTZXFcRhP4p64NtIBJS4cyp2\\\\\\\/sSnYaclBal+MRiLDjMkODEq58ZJ2yYkuuG1YH2NReEDU5\\\\\\\/FrOnzBFLkXuXewhHfUurluoV9ouLYuBkPY2RG0z0XXv\\\\\\\/VQbiH7+hQ3iG\\\\\\\/66rcA82kjZxF7Coo+n9nbNc8MpTIC1YvWDkySgJKhZrQzwFrPADGDSxOIHg0yBAGZbmeXdNF9ompqMC8ivjE\\\\\\\/0ZTMA\\\\\\\/G1wOMgCAtXPnfOUAEXDY9EUsaX+XyGMXeaIXoanGLn6ASYCVBlobGx3LgmoXFuAYcDP8ug3KeDPofSEAKwQmj7iXqT2H1x5X8HMzjh4a8S39U2MvXFOXxQDPHJLB3ZGn9kVQALgfgvYwBEjCCU4NC15TB7PkmBsidqBTfHh6K6X2HjKkU7ir\\\\\\\/259p1m2rvuGVnEoZffeg2mzZquAg836hxyw5ezYWBkzl1jhm0DL5ruUvFcuHwvv533psafbmPB40IAzt5ot3XgCqYxwULeWj9IK7\\\\\\\/C3DCWVxYmjnaBRlVIoIV6pPDA\\\\\\\/\\\\\\\/8nNBhQ\\\\\\\/jOvHa0vnXy7hYgwenwcHPnYTnxcSrKaOsUGhi4md+RCgXO\\\\\\\/qmor8lR1yv6qzUpYRUDk2q1PwcPZJeQuiqptQ2RAkktoDPoMiNtKJr1PbEVd57VPvf+viSqqec9QCn293c8jtBPBhrQ7O7Et2PW2l4BlMtPrnyXNTXyX6zzLrFDv31\\\\\\\/t+asfrVNRAZ+N75s9VV3nbarEhRWTGIhjoBwXGHveRRTb5HB3yXXnQh8NG6TLacqhBVX8YS9WEDWlpBldPbSwv9J18A2Q43\\\\\\\/d9Jy7Y0nYygRSQ9o+i7No4IwDKUsvObSuzYI+IPdbuuEECBwJZ4Yokh+yQ6KN35xHUDQojM49U2FyS8GpvpjYpoQUCWEgCGgkN\\\\\\\/i02zaosWpwoc+SsswKWIT2Uw5nGB9KUXGRDfHx3srLxdFZMUJE61BoWB+KfQL1GVYFLBV5JeITYjzYdER0vGuudDtknAyoYpwa8zGRz8wS5ZDsNw32O8rOLb+79+1oYFq+Y59c0QB59439dtLemjQJqjgxOUBOaPE7XR\\\\\\\/DNgzzgQyQE45GUFAtrZoMYE+kiig+yH6ryzDH81cRaj9mRfXanCUxmh6DXptIE8Hbh79tlt2VJoTrhuGQCwTBCZHotOVjbmR6hwWL4cASc79fMe2yUA+if71ryks31jakty2UpTwVV\\\\\\\/OvrirrlFp1MFQ2g4Cg6I6Z\\\\\\\/hyyBk\\\\\\\/R9MotJTnqxWk1qdIOXhSOs6pwlN6HuXwentndG+MCaOhUGrCMim3l6psB40Zov8osDqWLq8wSSattjzAdyIfjV2RMKCCpTEVn9\\\\\\\/moL\\\\\\\/PmGsfFGNsRjc6HF6jD6wxlC+Y22+lfXBK4n78WH\\\\\\\/T+ggWNZJ9OXiyzWtBFkASv7gzJjQuOciRQAQmoOIotVlvYa5pnimDuDi0pMEx1tp4gopnOrUDfDcS\\\\\\\/7yMoMqfBudtJCrvTmovRI7AcxSxRBdY\\\\\\\/3pvX7+jw46gnRwH3Fywk9XKdFmbUsvQMPBTkurY1S2A7bHCZVgS\\\\\\\/u3OIeoefh5m+19j+LlrOPWu1um733Q2vCTlx+kkZjyUvfkNFpY8vKO2iJUfEYkqbGbY2IHlUJLhWgVvu+jSE1DJWQj2GuOzlne9dNdiKr+r4TGA4Xtqg8K5bMw3HpDJJ4btjKTJmGrTe0Y9bnCfJpbBca9LBHy11vV6c0eA+L2Plm10qh9IXilR3shRdMbXlQEjZl53+q2e1ZdNPozZpipqIi08bgP0P5NI8GRwcF+b6geNc2Ra52NZCBxVDJR9Biwog0yMYrhSlAM+fUBydiq6u\\\\\\\/g4DRgVA3JbYe\\\\\\\/8LmEG0RarmXZ5nr7yzha6iugJQ7RxiIENDTia8DYg1\\\\\\\/92BYckDiiAjtlfjT1LDgRNxouIlTevkOEXBhISEGInvb9kRiJJMOLv5ERswID4mkx+G8FMlpQr2iXDdmUE2qlVttlywHE4mGeIsiEbpP+\\\\\\\/6jFgNj7RL\\\\\\\/w9zM4D9kCnqv3ueBzfrnUwY4Ff2jggU3WFCTkZ6D8HA1dsVc2lDjff80Xc6KZxn1dO3vXfJ9MykPRYS42wNMNMwbWNFyc+YFelvCIymeaDKDuVfDdgaOA\\\\\\\/vmQyCDyrMmd0dvzV1pL4TWjEUgWCcN9wUjeayFqZ\\\\\\\/AORGEC\\\\\\\/1pckYsVHYmjszVpAwIxye3kjW4KhNvfL9HdJPYUrYKldoz1FwLRHk9\\\\\\\/eCy+cMw3yB8XfoeUC4z84SLeO43rYMmcshM+SPxLoueopCZ3Xu7Zhb+eYF5dqwhath7Vn\\\\\\\/2f4HqvvTDdwFa5uGe1NK8yiWHpkVRMmckbMLwEbJ+tt5drWWmGvd1zkyC8+PYJ06tH5tcioaNmzDrkGKRWX3G86UiAwZCv9umNL19A31KRpQ8Sk+lL5slxW6lkwq4oNBh9Uc\\\\\\\/HGf4NqjNn\\\\\\\/ZVkeHM6m63KtjqvDKA+XcWkVSNOO\\\\\\\/Fcuf4blOvHBjp\\\\\\\/BXndlrZmaS3zC3FnX4xjZXmlQvn\\\\\\\/ZiyQu28odaXYXhD4VO6eF1bqeyOUKP55EkshvkyogahAvJcZ4pcfXKsGxjTFrQS3kxQVmT7idZLgvo\\\\\\\/PuXEXlTeIMWZnK5lJsYDu4yg06nxtel4I2eS2IGlh3FQnb5ao0kZ\\\\\\\/+zuDu1McUf8M0tVr5Tznkgi3ozlDqDeXOBRdCTzdcPWptW4qxZBQyzH1j\\\\\\\/me4KbOiNeXAXdFVN9i4JK9zi7smfkWb43Mp5BNQmYV2trAItLdRhyPt8z6lrBH59GC4olR3iDvPfsNRMuqBUTxGp0z9rqhGD5t9t3T1lv5zaBcLyF9D0ozdD5yTb3wisIMKhvB6OrxfXWTkxyWKVHpxoI6l6N7NRKpkoTbpxSVQQBpSmjzoM5zK5a4izXrS\\\\\\\/k6OFdjyEAj6WjRX1UK\\\\\\\/oo+tPwHaRUJe2h93+dnaOEKy3Spsvnt6D\\\\\\\/Qulp\\\\\\\/ZdWeLMmX9tRtpi7GXqOTff1WZ63eIwEAeB6w\\\\\\\/01hhOfoD+3wj3Zs8PIKwzbyH0srawKZ3m9bVdlKfX6ULHe2nIDu03EPVyVvOAjYQUgB5tg9z6VlTl1WWNPVZTe4lbwePzdnqNUZva5X7EwiJlriJDCaNATjPGjFzMwMAjLbOyzMcwOnalXhd6VbsOQDKXXBM3OyiNDEwSehb9cDvKAnp7z+96GuQk8AQdxkDl+uRvaUqcRP\\\\\\\/RPhpQDjVmgEG\\\\\\\/u7A1RRE0DE27uHlqGOqrIbxg67upnLZwZR1A6DC\\\\\\\/NC41iYaB0GaVmdXsq+5A3iaLaHUjaTzO+YYoMvcer\\\\\\\/H2qWtQyb+5JusqVLYd8XuhGnwq1pDn8u6I5OX2PGPaiZlMHP5bPgmx\\\\\\\/WTD8FSlsYqPVmN7lH752mDEA3O8DmM9fBzS2GmJnD1nhesyE4lDWuea3\\\\\\\/lBE39CrPgiSAyy5ixpB0p0IQXOAsWDmXJ1H0u9mUGaxJpD2qw\\\\\\\/yqxj5EgD3Som4YGK730coioGr4ZFwG\\\\\\\/Q1uK0IbIguUbnl6Io7N6ObMCYJJjkd174jC5wh4Od6Y3oE9cNcTC2xIzgbriE\\\\\\\/P53vzVzKKkayaxh8Y3VeFEmCSpZiSgEj\\\\\\\/gwXAlFgZ2ZfLqS8r4AWe1rT91X8Lop1I8llT1\\\\\\\/Wn+sy4TOI7qi0\\\\\\\/LbHvH+yLWa4lzVaZUEYmgqAhfIAf9xs6UxK2vDYltnWyqMZYwnLh4M+leuy+LTblEKKH\\\\\\\/\\\\\\\/rfvcx1qkNRXS5sxsMwSfijcb881djdahICx4PfjL64+rqzUa1s5YeLj2adPEJJBkumTEL+FYQ1tMGSYZuLkKu8bq2+eRCLu0kPugYXEGvTcneYU0Pv\\\\\\\/24j+U9IxARQkBr93aiDqR0\\\\\\\/L9ootaCpLqwOnF+uv+LFLct4cTEC1lKntiyZPutY50cmx51KI8me3LBJrdP9Tuo0qIij\\\\\\\/UNFv0FBuG8htJRiZRQNuSSvXpQg6npqdT99fNbIFlorRuJhmLj+gaObgSF9q\\\\\\\/14ThLbeqNmHZ7zHHhqDvn0N4B1zVObrWQ0vRHfmWa2leDO5tukKIX3Qq+SCB8owf91GsnYrAulN9O+PD81l7CWILTDaxaI4ea\\\\\\\/CluAqqo6CKKF9WMhj7YWDlSnrnnR50uv80C8t34YaOjkiCvwPGiT6U9XqLX+IEu\\\\\\\/KWGmoTfBvo6KvmI6YGs5++JV8ooohVTyCm8Ycyi9hwfg7\\\\\\\/aoeOfP4lsNF1pvmeEUyiSKL39rsuYT9FGBmY6ojOwDYz+9QBziVUwwIiLB5UQmJkqiZqGeHVxvigpH\\\\\\\/uw4zSH91iAYMpgLMlvK1GLfmXvYLUjoK8uv7JFy3LflwwIZ6Ugw9EFZRQrUU8+yBlpabIBOb0mNJ7yOYQB4cP6zJNuc4iiVzk\\\\\\\/t+qw\\\\\\\/2ZLvr5o+vjGrXRIWCzJQgEJa9miIcvDnrKuWZaFlyH2g1eX5l48tKVo9QJFYYBue5LG\\\\\\\/sCYmZFGN3J2HEs0sUHZWUt+LYA70G7qY68+owctJNtxLR03Cbq8iu72Pl5eD7aVXzvlaSIsds2Hu3BK2ShzjlXSUlvNxpXpwejlSR+1TDSjcWOI10+AGARbo+EUTpX0HT\\\\\\\/7a2lueCYS5b+7FflbTvSBh40zguq3VlJFD7vru1z+IM3cZGYVbtpZAa50WPrgCxbHAb3JpCuy0bHghDZmvP09QInznFelSawzsvEe\\\\\\\/2QJcWaDyFFP3tCf\\\\\\\/52kewhEAHMDDiCd0amNC9dXnq\\\\\\\/hUtSY+iQbhuEslKHn3N+ghumMIL4VKA5VjWmaJmGlgl\\\\\\\/nEjfdP7\\\\\\\/1hY9e3For1yL\\\\\\\/3I9+\\\\\\\/HR+ITJmaCho\\\\\\\/JQXpzzSJNmNyiqHe7YyLqD2zl5yFMCX0sUAYRKxytfVCbonv9t+Lmgn\\\\\\\/cqs0szTsr5Ic\\\\\\\/WsoAEPw\\\\\\\/kwtxhEhi5cU2cLz3PBq79FZpcDcjKn4ogXWIIX1Crsy1wo5OhK+v4KODtro2FaZGrhbreD8thuom6RKnVOtopHGbGEUn2jJTG3c2VFiMdsM7t2UfBErIsTvwpxD7euj\\\\\\\/CsI2TNJwV81U4iV0gQsgHaPzv01pNuHJB269FmFvFnoFIwQoBiKdSQiDkxWd+0WkmBgLPdupau9r611GLlCQDby+qekOOysStWjdwfwyYL3hCgfgFFo413O0UC7uyiMAJELq4XovDkItmAAgDeTtmSnspdfQ3LKLSNQtIT9\\\\\\\/1W\\\\\\\/xk+dAoKWIXNhMPEMwy5gnDaIpJSc9uWwKB0qYBFO5k6sE+0PqsPx6J+ffZ+JNqeLgHOS3NVuOZELKlLBkTx593Vp1U+eXMD5Z3qJs59mGedlzI8KHMNDirFL\\\\\\\/OnWu6Uyb+d+JE8cPuRM8LIBPe078IMb7NY4F1Sp6UP+cNqYbrE8t+059d6kU+JRNpGiChfwnszik8Q7+J7iBBoN3cElqbKiNncFKOUDDZXWWxjTnKwEQ3ZobkseLiX2Msxe7T8Kp54mwWyCd9xNU8c5+tHR5NfaJSszj\\\\\\\/YPTjce4Lhxc70VIr7s1\\\\\\\/HgzG3VbDFRhK2u6ep7\\\\\\\/\\\\\\\/ygTs0nsehKFU7rVYM+dHbW2CmS+eV1Te20xZcgvwdk2pN+f5rEhdLap1W63RWIHGCsIvsmzcrB572Jx3Hlr6spGpvChS48Y8rX8xo7J\\\\\\\/gLL307Cy64JO5QcmhtcgF36blFeW1t+90uUIEFZxYNLXtQ2VSdNUrHGSSsFo4MQ+gTsHrI+lxr8AQ1OBRWxYIPEJtdZVn6EEnBtSl0o92z\\\\\\\/RCn\\\\\\\/1oiT+HBYT0RB3cfzVeyocrLvJJN2yDGO4jAqL+pk2ptGwZB77tO05s8yB07mExI64Yt9i3LlxWoinMsJeglaNwRL7v1JE6Uh71ChYEju9P9bArS96A7p6KlWW5fWApcbpBmqa8ln7G1KmETQYjB0ZAGSJFIYshXUOaU9A32g2RQkaBIhsZAW62wIKvS\\\\\\\/zlS0EQJPiN1D6I26Qeh7mnCTo4tahKNY1pkAEtNpWSXa4uzxuBC+mowa4OzPPIchFF+6ssaF1S7Uryeub8vNEeejlsYBcqI60kwQ43uSuho+fzCeCJfTGwdrdKn9H65Az4F6fUjZkDB96T8onfQY31IHifxafM7EtVpxLVwvebS\\\\\\\/i0mz\\\\\\\/9g6dc4sxROD7dIIZzvax0P2sM\\\\\\\/dddUXarsTFwF\\\\\\\/1HK0pM+8gboxcFwNGzl5nujy4XqI06MI+KYK41xrGpYjinbW7LnVu\\\\\\\/\\\\\\\/kH6q7hND9XwzSMUn9B1j6vfn25fHoes1bBrtbuE\\\\\\\/hdWIZqF8kDk+XGTch5Y7T4fxXo4gzyv2njuFi4BI6f8DNJdtZFUk0aaJZHSPKqF58ZDqUI2bbBm5EGNrM+TGlb2DVvShb3B3trkkLq605vJfbBwzQJ1bEag\\\\\\\/NBCzG0v2YxnBQVqMknxy1WZZNpU478Sj3YCIsNSlNZV4Et096zxw1dhAoUMy\\\\\\\/sA4H3xFRfr57ICM4FZgyWsSla\\\\\\\/ukOVjvkVz4q5SSv6A5cvFhgXk\\\\\\\/GEpF36gWidoNioBL\\\\\\\/MiHvWAc6hwWbyDmbKd6WENigOrB\\\\\\\/1h7pt6uBUHMF09ZAXS2wue38iZDmrM5Kp9jHwZstCjssKUjFVQpM55W3oj+1QHFEazFe8LqDSEcnwPDIiMhdqTzsf\\\\\\\/KfxD5UlemCpDoP3nuq8bt2LuaWC8l81tfTZl4VASUC1Rlr6HsfZAGXFNfml+xlRfqBqzSpL+NBuLDm\\\\\\\/7v8V4RdNFaNU9OOop6inAJpu258OMHNfKdV69I3SRkjw8BFnBdI\\\\\\\/XSc6eRynywy7FRaFrLJpTpC4vNKebUIL44wzJ8Xg8uK9NoGQTTRt7DtyFuQnC40j5xjgHrLexpmI9vjjZ8lJ\\\\\\\/xTllfRc2LeMCAL8PvBJ5HL\\\\\\\/\\\\\\\/A6AozU0xami6sXc9teinnSMqfoBiL3\\\\\\\/3ixoAr1SnYqAqSX+sJTObcLwC9fpsLmnwEYfJCELlFBRCpXH0q+LS9XjS22Xr8r3phK0CrTKyojxWEcHscC+ZCz8\\\\\\\/JXDKuM4JuAsWH\\\\\\\/EsjK8wMmf\\\\\\\/VWzbdIlcSPJDKO41iZHaRtMAb1P5ybrMyZX+CiHgVMt4AvbbCfDS+wuBWIyFkpCmlOIdyKTBLuF94ABq5jrU01EoKNNLsDzgiVKI3nuQp97xaBP7b\\\\\\\/0lsf51ddjYe3jfxUhLjI5ede2mB6EQwjpyq62ZHif8rsqo\\\\\\\/gKqVT4xDD4SL+bvVHR\\\\\\\/drniS8aETLFkel+l7BMUoyjq1OzdA43sLyunVwhH6A1K+W3OYE77V\\\\\\\/ayo3inxjuoWpx138i3hE4s3vfq7HDqsVj8Pvs1knEKZMoOVZJe+sbRq4ZKwp7KlkhTi7O76nBEoO0JHlZY\\\\\\\/kfSDPxR72xB7iC\\\\\\\/8QGS+M+B5zkMOcug+uAyHMuProLS0XLlnBmBlSMWHN2yXbCgNiovukEJt77DcFnLJo9rIPxKFhdXpi8TdcM2J2NoKdMXhLQ\\\\\\\/8NJqxMMnRSHQntnEm+9yPq5avvOYMjqKmSf0a25ydga6mzQu\\\\\\\/w0ao2cnEezLXsc4tyzZeHXUygvxIRxStBeUb5hCX+x5tBc9YngodNLgEhfKO53L3dwAdXde1LqRyBuUm\\\\\\\/Z9aSVyMF1DY1cOqJzUaFcF9MwotC0i9lgHi2CWI1+IkHjtHQHmNI96pP+i7LK3PkZIlpL0uEr0WuyvdtUnhs1KBWKm1l8qtN7Vh+D++\\\\\\\/CJOWVtl41kJ6w4gGUw3rwyV8N1ZwwI6tqUo7J9KpWLlU+ATkaJjBZax7bl0ppIjkfEjVoe8YfA1lvksqQ9d9Nj+BY5emhPGXmt1qvufKWKPrtmOU10hIWaVweIPzzT69DaFMPgDcr1YID0ixsMewPubAlM5JfgvSqAPr0kK2A2c8NK8wK5vxi6Se0J7IcYSsPOQ+RkALB1UlZTxA6cbTRHtn3KZF1y93zfg3PUuXn3Vdkd\\\\\\\/eTFyaG1TSmGNK7gYz1cKNyHDKkfCqEdq6OpiytyZ5pGDvpBnF4SIehn93YgxzH+\\\\\\\/3EyuhXp0qw+rARd8cG8WNq8YE4hKSF3unxFFrADQyHaofgYU4RJJ2xn6Mtn3YTnFMKARstUnTlbiASIS\\\\\\\/ABbB2Ipbq8AgCt23YQGgnpSjV22DFzE3GvV+wC+e60kXOk4DrLxrVwfAKfI\\\\\\\/0zNHTK+dOwllJcgQoxWrbPoROEheboUxtJ9aF8jL21M31TjjyngXcA0\\\\\\\/MObUtkQYjP1y2Cjves5s4cU2sxGqvj31saqhHTJO9dXirzx3XzNvkhN4SblA3rosdoOUX+o4\\\\\\\/Zhgtwfq\\\\\\\/K6B5QwDfVnBT8POYsTGcY6\\\\\\\/ii+GWGSuxmhbcuYj9JxobUmBR8x5y\\\\\\\/qMBwG5z6TnmDEE+7Quwxt85WYqY5rPDrVdOiNh6qpqG2IkWZ+gr434LaNoC85iJ0lAll41RqBtedPnDjTkSavDecTRKuOGSC\\\\\\\/wV+WksT1ZTqZdJGv0P63HwG4ozX99Z8sc51H3TGYETz12Q0JeeATQ0LkFCNtq1oTNi3Rt234LUQTmpGumxxD6OT8ou22AB3ebFWK2zDUdWD8Xk7yyoPQblIRoIUk1RaSOpHyOPlynKcd51AzUliHsImk4buyeY5zXYuepnIa9NMfbZ6w4dhRwhSXS7CJ5jcqOgQyp9yvs0BrmqWntxeBZ6KORFa2+ifgkKWB+TjspoDy5Jt3k29qpOuUsFCsS9Er7yvRi\\\\\\\/3hPb9lWy0\\\\\\\/EGbaZa369heGlp+6zbCBkAfFeYTRcl0sARzhOToFAlec417Bg4TKoIXaXErxjsAyYkjFijSb9WjxYvXx8tTTvF\\\\\\\/4na8Le+16ZjMLTt5\\\\\\\/RK\\\\\\\/mRjscMJUye0cKw89QzsHNVH+eNpeIyWucLVp79d5bClw6JZbnzjC4xB+e4IKOsqdqxKMlsw5D72bSEam+VSB+WPCfw2vIFPxW++6KWjp4xvTPDzDvfnybzgxDHFXBTv12qmtV+KQ7FHK0EWb2O98caOH0F2hMEzmuHV\\\\\\\/CHXo2jKLKZaJUCda1heERmj7Xtb1aMJldHwNJoo4\\\\\\\/WTRwTqBygvskRB7od\\\\\\\/TJZZypVRzOhKoU+gglDWi6IfrLwQOg7f\\\\\\\/Lc+2AlUCWXDsaZx7brZ5DDfmcwyG4Hait9DYHiT9XefUOW8zLLvE1vIxlaYUDPa0xFRcXK\\\\\\\/5s6OrmxRRphiJG91xq9dkxC1Qecqp3pg8bcf591NmP99LPM7pkOGn4S2ncyjbWl32\\\\\\\/N4y34bpcNiiujA5to4QQeguFD+9ZOUInAQqB8a7Vqx\\\\\\\/J7vhY2NAQngrMNLclg9ny+wBk0BogqbMFvtnTXaJhv609xt076RqorvQ8a2dFsmyORfb8ObD13iep+zhLXn3yDRnhOFRwB7xYTw7MysHPPoLgHLTabeL0sRT0P3oh5i1xBoz0QMX4B49HT4VNKeiTgAbVqCz07FFa69rD4i6cXSNSur+Dj9tR8pblqSPT9JY7jLEoDVPSPD1u75kN\\\\\\\/h96n+e1td8f+QS7+nWy1nNxDEXIqE6gUStP+6wRErNrphQyFSbVc10ABH1NASWeDSfKE8lbsXMr2HcG8jrYkx2EmtQn5aCrg7Q1NVwEvX0yVLMWRFT1zYe8cUQ65wU3S24DmXB0jHMufqBV4iPZ+M4QESonNK09igzH2WSndSPNWZoM8wJnajxADvwbbWCHrAPE7WeBqUUgdZQ2EDeHtKX5XTH\\\\\\\/COY39GNz0H\\\\\\\/QMDNF7Zloj2KwtzYyVvU0UakS7Q+kHRQDfsu5IvCo4mR5In7NmFOM1l8NfTUIeh51tdMK0uJHb+VmdP6+lte9QpeDnpE49rpKFUTNbbBiay68\\\\\\\/oy4c9I+LqITp3JhSbzFvtM\\\\\\\/Xa1pFnblSuAjU26G1RlqvDBd4O6rUN7DIGK6T2eUq7cHMS\\\\\\\/LIC3p9NoZ45ElrM8e6nyE2z3ig8UG\\\\\\\/CwZPhQmPWoot3T4U4CtNhP3Tt4DXCyQw5mvUia+UOBm8+5igufBvFWpvq7TkJ1cDpXOWwuG0P6kcJxOhI4FSDUHrvDe4ulX+4BSd\\\\\\\/hIIME\\\\\\\/5E79FWbyM0U50F621LfGDScrKoZabi4ph2H\\\\\\\/aYkuXuEbu1xgofGMJwjN84W61WHSmyZRFwR\\\\\\\/U9+fv6h+R8FGRbWkEVQ3ax4zu+V\\\\\\\/sZhfDCGRfBpdg6SkXsLNHcavZHbGTO1TT7v\\\\\\\/ZceRacs3fWQukoFQ0Dao4K5ufZOmev2Naa7+13Y6vMeDNEyXoA0qBcJqj+ZUfescIIKxm12bfWChJh7UVpypDnsx5XtoUGUUkE\\\\\\\/BGp9GitBKAsQ2ttVxtWVzp7aIlpqK5b56oVFzxjJhtybXC4vHRCnz3boP2fHMCEfTbqIdpMzL8pN+1lcxcj8WrUR8MwcxZAKVn3EaLG\\\\\\\/f2ZG+gjaTcOF8qE824QyyMoUWcQMBQOVDBT7lWyhaBRj7Eu7eOwNUpwliMj4+OFB6JGB7SxMUmmVNx8wvbjpc+PBZb4t\\\\\\\/LIiz9zhS5JV2FX25eEvCoWq\\\\\\\/9oXpF5xioZ520k66czQFn+ocj4Xpf5oEwa5Pi3BfWZfh57TYBkqoZ9bJo\\\\\\\/c53BJ2As7GV87xeseGxh8Xpi6VxfHLRqpWSaWnt7mm68JthuNrdUJ6owKWji4N+PR0EZ+n+DVC8h9k0OOlvYAJlq8rmkXDNEtmsxdcvQKscowN8MMfglM\\\\\\\/AYnMhcZN\\\\\\\/2NqFovAOzCVAwZXgvP0EAeXYj59htxW1KThiI8sz5R27yInGV6l2G6xnwJdwDWLYgCeFDvQgbi6nASi7t\\\\\\\/WL45cS293QhrDEZOlRZUlbg\\\\\\\/Yka27sUFp2A3Kp1WPmS9py1a1q6ULVBUUKQ7JSi3edQHzEixXmFV0\\\\\\\/U8dkUz0kfxqDTUuwuSKrWevD7u0qHCz8YROCdKU5K1xGpNO7zLYwVNXBC+MuSg5ppazyrUwvK1QaRRCCxKDtIIqZtSrn3qEBxQy4xwUVKaACkGP05WpfG6l+7HJsGcl7KAZbtoFj\\\\\\\/zMdvAuU9Lrv4qHrzZ\\\\\\\/pzRFdG\\\\\\\/K1RoDOAJE+dsWB3lVQ2\\\\\\\/dvIFlMuQFyG5sVQGpPbVMAkVXlwdEkN9gmJJLAof5rTqBmLM6xQGDjbx2zRSB6SK+r8a0BNW+8N8l7o0DklE2JTMvpnZx+fIBI4eMXDu6IF1IjgSOHQLOMkrJLCIsDWa62A98gP8tTdXOdsiTn3N8J+KvVwXQmdatfBKVZtw9JY\\\\\\\/q4M9V+FJ7PytDawCn2uccjYHjK7rlvkyXoiabd6Ps7Qck4ivUiC9zlax\\\\\\\/NfV\\\\\\\/fReJ9a2oUBRaOEtk1T2wMr4dbebgluNZHtakUJiVI+hlqYLtvrR1UiqjNQfoqpBBwVOQRjzwyOFL5gESvumic1vTO3oHmLncVqgOJ3qqMMIoD0Srj7AX3VS3RZlFEikGSzR3wj5UmFIKE\\\\\\\/ITiqN9WuHS\\\\\\\/NhtKk3N9CpUpiyIIZ4o8nhEETxwfnguXF1ErA7zfFIKFZXoB7dC1VtvVsN+jnUZHKDv7ZkHKV1Y7ssxPp8oWcD1NlWrr\\\\\\\/vB7YJrezFZ2pUPw1XBLnLAZS+673bmPFtvboVGBu2IfGXSl1F17CJNvO8WDiyV60yPVr+OV2MwmkaBsOmh0nJCsw47KRJTAlkq8t8Ke2YCezz6Gke0w7SyyZYr8axytxsda9POt5eX023gQbMKdFE8e3KkT6hs2JSKCfGXAz4xRfVjQttgKOc+HV73G7tpxWSLzYMLbD4o\\\\\\\/y481LtHxPqd1ws0MrudGxjkh9+yB7krfxrSFigDOZoYvICQvk4Gx8fajsxiFchF4uA6Fg4hX3p9NfuhGkebTJyAOQFOYy6K6scLKqzWkkoWqfNBjiLhQwOe8OWoFYKosqfh7zIlcCYJSTLMdT7BjT32YxyklCLcex0z6\\\\\\\/xRYncbYXLWQqSFwGefEF0i+Hu87EUPH+Rzn8r5Fmsxnf8jPCyCjhfeumvovu5ZBFId\\\\\\\/eXBvTMnL2SrtaukcQjikiC4yCgLefhFupJgrXaDXHsVvzrwME6Ynvpy+7g\\\\\\\/Uu5Ti4v5lhPezSQU2JkjQUFF+zjhagubYD1qkQDohZ7uj5mJuQNhmZIBTJooEGFnS1XjGpJdRJqGeLoyy65FEe2VvJqb8jpNTAC6\\\\\\\/NyK9qoPNEUMwFdJkIU84g==\\\u0022,\\\u0022iv\\\u0022:\\\u0022UybRGaidGJQDIibrwYE\\\\\\\/vQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00222f2146a6ebde6a8da85f902fe4d954340f8371b55afcec911f895288315e9f56\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022oxLtejA5eaFxRZ8XQilyMTrADzvbciuOkO\\\\\\\/qzUbQkaKppUdxREScIG4UEKZPptJhGrjcLrXJ5CwVlXVO8qTCUXKDm1a2dNNbJuUE5lqUx2OCuSbusZMhi0IkAze5\\\\\\\/3MoGh+zh7NVFk2LVN3L51FFehw33Zwt\\\\\\\/7mIya03kd+w77HGBvLM1utDD1YkhYuwn9X7gszf9zMbDb9ablV9drJ54VUIniZJ\\\\\\\/a8xf1MMlmgymETFWDTN0nVdiX1BcaS7D\\\\\\\/p2xdn4B0A9JbbT0O91\\\\\\\/13HmdSIlMau8TNdLuqdTu2ykeWmwaSy1d1jaAXgJiF+7Z5zYv\\\\\\\/uox2TmuxA\\\\\\\/VnZrRGorBWAVGpAZnvYEY68\\\\\\\/9V+v41Mn++O2pqEt8+3YFkwAq8ufzdJj4gBywaVjtHrF5TZlMC4ods1vRyVcd5lDyxb\\\\\\\/nOSuKLcjtOxHh+FuH+5ggqpUcX6xM23v8YlytqN++RhhI8h1x2JmkQGzSCT0fCemP\\\\\\\/iCguT\\\\\\\/\\\\\\\/7jfH8B6f3aPU1Ny1o0zfPg5l6ZbLBNdUjHfEs7nc9h9fsP\\\\\\\/ElvO\\\\\\\/vPkag81XvYFQJN1y4AxC4mzCtIjqIttG3iMgQnaH+j+80ilqNSkCon6g\\\\\\\/Tim6Rd0e0jRTtszzke75WmEw\\\\\\\/1Iv55ntQN9HjsXXyabsyyH0hZ033h8\\\\\\\/Gp7fyeJouN1ixFNHKSl89cXKmVK4lh7Yr4DFdXD5Kp3Kdc7PafCT6rD0BxBAZegxsDP2ozFTIArU5QnM0LqTHS7QhITKQApoac9UOliWLPdakkwE4v4TM9C5vtFLo9IO7izcMbY\\\\\\\/TLN983JFDcrcma0TLIPFEg6KHUD2HDWAGSOFVNZwITkoD927bFLJzOXQEDxKXVNFWX+mjQ\\\\\\\/q4gf4bKzLNK0\\\\\\\/FTzeDaRKtFKH0I0i9Jh2x2trmOiVV4kACZCH4oTEV8o+f89BSbP4tv+DB8TZCQSZ\\\\\\\/5sQAIhaIqWVYwvT72GnoE7\\\\\\\/hYb88lenDZGJEOxLAocMWoxPM6l4xW1C1mcdWo\\\\\\\/s7Pq5gffjfCQkPIeOIPdhWy9QSQRp8MoBqD5EGZnaIzE\\\\\\\/U7ogKkSCMASRFHApkj4Fk27PFJvhkiGRdHFyBeVAY9sXJ3mopaFA5X+X1C3nHiLVtv\\\\\\\/5Z9l8fK5DsgGCshPFCU1AjgHRT6FjL3RSJQCrTH7vN\\\\\\\/BRwAoukwYLVaAYRT986WO5fyJJK844zfDJDFYgGvaOKVw124NYMbUMTeMmZ3LOmis8IspYTF15plPmpdTa\\\\\\\/lUeTmC\\\\\\\/g7t2Aq9GRFljB8jidrnZyP\\\\\\\/wz9imzSBCW7bzKnxWB6jKa0X0WGxjb0aGnw0VdNRiWm2A4zdMoTSJS6pDQnpiEYQ3us0lfwHuFkjbKJpo1f4YQNfEc\\\\\\\/yVS6CoUdMmnR3jO\\\\\\\/nEd3sF\\\\\\\/POlkQtzzlE0aWVlEKGMwZRMLp6RxvfVfp2ddOrYQu8eZOAZGsRyfO\\\\\\\/dFM78CJsQGLTIP\\\\\\\/Zx\\\\\\\/U5szlgr2LI8TE0CFbiqgE2aQd0AFZjTsgl4R+HlVouvTBbp3\\\\\\\/2PrUfbLKjZBH1yok9ZOA9L6227q3E3XosO8TwXCTSTwr\\\\\\\/sENZ\\\\\\\/Gbtd8XlvgM1pw7jIsrujjL\\\\\\\/nL01grVpK+7rPX0hakbAYq7w6j2G8xLXXFOOrDQMjzeh2l79RtO1zUX17\\\\\\\/JKsPnRwSBlbT2cAD6lg7UEvYSJt8xxBA3pOb7ZWJASYiRgfcuDi7MIhaWeIFkyopCWE7J4lsajR05SDuMMzj6wDumBepnzE3iTN1GDUgM8kc4qI0bieMNxULXWuv9dRoc7dc\\\\\\\/WDazB+I\\\\\\\/G1p6DEmBz2g0g5lQ80J3Y+a9jTq+aFjy6rMyobNOj6RZl8xmyNDbu5tlD\\\\\\\/\\\\\\\/q1wSNJOIXrv5TrSWTZaC\\\\\\\/zdSHhyPK7KIZua9rb7bmvVxt8HsczqW3NY2uXuq9MjEmcxBjvf8j\\\\\\\/C5J2NLOFkGAOHPlmgeireYgQ+mmafMJgH51FBDNzJhETwIfG6lqn0fWWBWVdfsSEjct6e\\\\\\\/vrzN7fizCb5WKzwUEg==\\\u0022,\\\u0022iv\\\u0022:\\\u0022Pa40ZL\\\\\\\/8ClbU+Jz0H9rqYw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00229d2e9ad63c10297e27c1a11d483aa8b8761487c911f6ab6914fcd8aff8634acf\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-492620 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-genomic-prediction">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/4/1231" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-xxoXGq8xlGjrik8Qji1WjgXzcspG7QKMmyAh24i4dHg" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/4/1231" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-9DsA2neZ_KyDkCPAhprqEFPHkArr9lg0KzM17sXt-Pk" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/4/1211" title="Chromosome-Wide Evolution and Sex Determination in the Three-Sexed Nematode Auanema rhodensis" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/4/1249" title="Random Chromosome Partitioning in the Polyploid Bacterium Thermus thermophilus HB27" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="492620" id="node492620" data-pisa="ggg;9/4/1231" data-pisa-master="ggg;g3.118.200856" data-apath="/ggg/9/4/1231.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Hyperspectral Reflectance-Derived Relationship Matrices for Genomic Prediction of Grain Yield in Wheat</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-1154-337X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Margaret R. Krause</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-5840-0803" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Lorena González-Pérez</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0001-9429-5855" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>José Crossa</span>, <span class="highwire-citation-author" data-delta="3">Paulino Pérez-Rodríguez</span>, <span class="highwire-citation-author" data-delta="4">Osval Montesinos-López</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0002-4676-5071 " target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Ravi P. Singh</span>, <span class="highwire-citation-author" data-delta="6">Susanne Dreisigacker</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="7"><a href="http://orcid.org/0000-0002-7856-1399" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jesse Poland</span>, <span class="highwire-citation-author" data-delta="8">Jessica Rutkoski</span>, <span class="highwire-citation-author" data-delta="9">Mark Sorrells</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="10"><a href="http://orcid.org/0000-0001-6896-8024" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Michael A. Gore</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="11"><a href="http://orcid.org/0000-0002-8582-8899" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Suchismita Mondal</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1231-1247;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200856 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node492620" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Margaret R. Krause </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Breeding and Genetics Section, School of Integrative Plant Science, Cornell University, Ithaca, New York, 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Margaret%2BR.%2BKrause%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Krause%20MR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMargaret%2BR.%2BKrause%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-1154-337X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Margaret R. Krause</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Lorena González-Pérez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Global Wheat Program, International Maize and Wheat Improvement Center (CIMMYT), Ciudad de México, 06600, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lorena%2BGonz%C3%A1lez-P%C3%A9rez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gonz%C3%A1lez-P%C3%A9rez%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALorena%2BGonz%25C3%25A1lez-P%25C3%25A9rez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-5840-0803" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Lorena González-Pérez</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">José Crossa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Global Wheat Program, International Maize and Wheat Improvement Center (CIMMYT), Ciudad de México, 06600, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jos%C3%A9%2BCrossa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Crossa%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJos%25C3%25A9%2BCrossa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-9429-5855" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for José Crossa</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Paulino Pérez-Rodríguez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Colegio de Postgraduados, CP 56230, Montecillos, Edo. de México, Mexico,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Paulino%2BP%C3%A9rez-Rodr%C3%ADguez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=P%C3%A9rez-Rodr%C3%ADguez%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APaulino%2BP%25C3%25A9rez-Rodr%25C3%25ADguez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Osval Montesinos-López </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Facultad de Telemática, Universidad de Colima, Colima, Colima, 28040, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Osval%2BMontesinos-L%C3%B3pez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Montesinos-L%C3%B3pez%20O&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AOsval%2BMontesinos-L%25C3%25B3pez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Ravi P. Singh </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Global Wheat Program, International Maize and Wheat Improvement Center (CIMMYT), Ciudad de México, 06600, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ravi%2BP.%2BSingh%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Singh%20RP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARavi%2BP.%2BSingh%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-4676-5071 " target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Ravi P. Singh</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Susanne Dreisigacker </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Global Wheat Program, International Maize and Wheat Improvement Center (CIMMYT), Ciudad de México, 06600, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Susanne%2BDreisigacker%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dreisigacker%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASusanne%2BDreisigacker%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Jesse Poland </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Plant Pathology, Kansas State University, Manhattan, Kansas, 66506</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jesse%2BPoland%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Poland%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJesse%2BPoland%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-7856-1399" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jesse Poland</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Jessica Rutkoski </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>International Rice Research Institute (IRRI), DAPO Box 7777, Metro Manila, 1301, Philippines</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jessica%2BRutkoski%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rutkoski%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJessica%2BRutkoski%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Mark Sorrells </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Breeding and Genetics Section, School of Integrative Plant Science, Cornell University, Ithaca, New York, 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mark%2BSorrells%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sorrells%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMark%2BSorrells%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Michael A. Gore </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Breeding and Genetics Section, School of Integrative Plant Science, Cornell University, Ithaca, New York, 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BA.%2BGore%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gore%20MA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMichael%2BA.%2BGore%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-6896-8024" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Michael A. Gore</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Suchismita Mondal </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Global Wheat Program, International Maize and Wheat Improvement Center (CIMMYT), Ciudad de México, 06600, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Suchismita%2BMondal%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mondal%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASuchismita%2BMondal%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-8582-8899" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Suchismita Mondal</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:S.Mondal@cgiar.org" class="" data-icon-position="" data-hide-link-title="0">S.Mondal@cgiar.org</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/9/4/1231" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:492620" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:492620/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/4/1231.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:492620" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:492620/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/4/1231.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:492620" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:492620/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/4/1231.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:492620" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:492620/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/9/4/1231.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:492620" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:492620/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Hyperspectral reflectance phenotyping and genomic selection are two emerging technologies that have the potential to increase plant breeding efficiency by improving prediction accuracy for grain yield. Hyperspectral cameras quantify canopy reflectance across a wide range of wavelengths that are associated with numerous biophysical and biochemical processes in plants. Genomic selection models utilize genome-wide marker or pedigree information to predict the genetic values of breeding lines. In this study, we propose a multi-kernel GBLUP approach to genomic selection that uses genomic marker-, pedigree-, and hyperspectral reflectance-derived relationship matrices to model the genetic main effects and genotype × environment (<em>G</em> × <em>E</em>) interactions across environments within a bread wheat (<em>Triticum aestivum</em> L.) breeding program. We utilized an airplane equipped with a hyperspectral camera to phenotype five differentially managed treatments of the yield trials conducted by the Bread Wheat Improvement Program of the International Maize and Wheat Improvement Center (CIMMYT) at Ciudad Obregón, México over four breeding cycles. We observed that single-kernel models using hyperspectral reflectance-derived relationship matrices performed similarly or superior to marker- and pedigree-based genomic selection models when predicting within and across environments. Multi-kernel models combining marker/pedigree information with hyperspectral reflectance phentoypes had the highest prediction accuracies; however, improvements in accuracy over marker- and pedigree-based models were marginal when correcting for days to heading. Our results demonstrate the potential of using hyperspectral imaging to predict grain yield within a multi-environment context and also support further studies on the integration of hyperspectral reflectance phenotyping into breeding programs.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/hyperspectral-reflectance" class="hw-term hw-article-keyword hw-article-keyword-hyperspectral-reflectance" rel="nofollow">hyperspectral reflectance</a></li><li class="kwd"><a href="/keyword/genomic-prediction" class="hw-term hw-article-keyword hw-article-keyword-genomic-prediction" rel="nofollow">Genomic Prediction</a></li><li class="kwd"><a href="/keyword/high-throughput-phenotyping" class="hw-term hw-article-keyword hw-article-keyword-high-throughput-phenotyping" rel="nofollow">high throughput phenotyping</a></li><li class="kwd"><a href="/keyword/wheat-breeding" class="hw-term hw-article-keyword hw-article-keyword-wheat-breeding" rel="nofollow">wheat breeding</a></li><li class="kwd"><a href="/keyword/genotype-environment-interaction" class="hw-term hw-article-keyword hw-article-keyword-genotype-by-environment-interaction" rel="nofollow">genotype-by-environment interaction</a></li><li class="kwd"><a href="/keyword/gblup" class="hw-term hw-article-keyword hw-article-keyword-gblup" rel="nofollow">GBLUP</a></li><li class="kwd"><a href="/keyword/genpred" class="hw-term hw-article-keyword hw-article-keyword-genpred" rel="nofollow">GenPred</a></li><li class="kwd"><a href="/keyword/shared-data-resources" class="hw-term hw-article-keyword hw-article-keyword-shared-data-resources" rel="nofollow">Shared Data Resources</a></li></ul><p id="p-3">The aim of plant breeding programs is to develop and deliver new, high-yielding crop varieties that are adapted to a range of environmental conditions. Two major challenges of breeding for multiple environments are: (1) being able to account for the presence of genotype-by-environment interaction (<span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-1.gif"/></noscript></span></span>), and (2) the high costs of evaluating trials at multiple locations. Statistical, genomics, and phenomics tools that enable the accurate prediction and selection of candidate lines appropriate for each target environment may serve to increase the rate of genetic gain while reducing costs associated with large-scale multi-environment field trials.</p><p id="p-4">In recent years, genomic selection (GS) and high-throughput phenotyping (HTP) have emerged as potential technologies for improving breeding efficiency (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Furbank and Tester 2011</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Cabrera-Bosquet <em>et al.</em> 2012</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Crossa <em>et al.</em> 2017</a>). In GS, genome-wide marker effects are estimated for a “training set” of lines that has been phenotyped and genotyped (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Meuwissen <em>et al.</em> 2001</a>). Those estimates are then applied to selection candidates prior to phenotyping to predict trait values, which may reduce the time and cost of testing breeding lines. While GS was initially developed to predict within individual environments, recent studies have extended the genomic best linear unbiased prediction (GBLUP) model to accommodate <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-2.gif"/></noscript></span></span> interactions. <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Burgueño <em>et al.</em> (2012)</a>, <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Heslot <em>et al.</em> (2013)</a>, <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Jarquín <em>et al.</em> (2014)</a>, <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">López-Cruz <em>et al.</em> (2015)</a>, and <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Pérez-Rodríguez <em>et al.</em> (2017)</a> reported increases in prediction accuracies with extended models relative to single-environment analyses.</p><p id="p-5">HTP is based on the remote and proximal sensing of a large number of crop plants to collect relevant phenotypes while reducing labor time and cost (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">White <em>et al.</em> 2012</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Araus and Cairns 2014</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Pauli <em>et al.</em> 2016</a>). When deployed across different developmental growth stages and in multiple environments, HTP can drastically increase the phenotypic information available to breeding programs, which may help to improve selection accuracy. Although HTP traits represent indirect estimations and may not be able to provide the level of precision of direct measurement, they may be of particular use at the early generation stage in breeding programs when seed is limited. In wheat breeding, large numbers of lines are sown in small, unreplicated plots for the purpose of visual selection and seed increase prior to grain yield testing in large replicated plots. Measurements of grain yield from small plots are not meaningful, nor are they feasible to collect when thousands of lines are being screened. Therefore, acquiring accurate predictions of grain yield at the early generation stage using HTP may serve to improve selection accuracy, though an initial proof of concept at the replicated yield trial stage is necessary to effectively develop prediction approaches.</p><p id="p-6">A range of ground-based and aerial HTP platforms have been recently developed to improve the accuracy, efficiency, and scope of phenotypic data collection (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Andrade-Sanchez <em>et al.</em> 2014</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Crain <em>et al.</em> 2016</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Haghighattalab <em>et al.</em> 2016</a>). A major advantage of aerial platforms is their ability to phenotype large areas of field trials in minimal time. This enhanced efficiency increases the spatial and temporal resolution of the phenotypic data and may be critical when assessing breeding trials at multiple locations. A number of recent studies have integrated traits collected with HTP into GS to increase prediction accuracy for grain yield in wheat (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Rutkoski <em>et al.</em> 2016</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Sun <em>et al.</em> 2017</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Crain <em>et al.</em> 2018</a>).</p><p id="p-7">To date, many of the applications of aerial HTP within plant breeding programs have focused on measuring the spectral reflectance of the crop canopy. Plant cells, tissues, and pigments have wavelength-specific light absorption, reflectance, and transmittance patterns that may, for example, differentiate between healthy and stressed plants (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Li <em>et al.</em> 2014</a>). Vegetation indices (VIs) provide a convenient method to summarize spectral reflectance information into scores that may be predictive of economically important traits (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Govaerts <em>et al.</em> 1999</a>). VIs such as the normalized difference vegetation index (NDVI) have been shown to be predictive of grain yield in wheat (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Aparicio <em>et al.</em> 2000</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Labus <em>et al.</em> 2002</a>). However, since VIs are calculated from only a few wavelengths, they cannot leverage the high density of information captured by hyperspectral cameras, which record reflectance at a large number of narrowband wavelengths in the visible and near infrared regions of the light spectrum (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Viña <em>et al.</em> 2011</a>). While hyperspectral data may have a greater capacity than VIs to detect phenotypic differences between individuals, the high dimensionality of the data may complicate the interpretation. For these data to become meaningful for plant breeding, methods that can derive useful information on traits relevant to breeding are needed.</p><p id="p-8">To address the high dimensionality of hyperspectral data, <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Aguate <em>et al.</em> (2017)</a> found that integrating information from all hyperspectral wavelengths using ordinary least squares, partial least squares, and Bayesian shrinkage resulted in higher prediction accuracy than what could be achieved using individual VIs in maize. <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Montesinos-López <em>et al.</em> (2017a)</a> proposed a Bayesian functional regression analysis using hyperspectral wavelengths that likewise resulted in higher accuracies for predicting grain yield in wheat when compared to a range of VIs. <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Montesinos-López <em>et al.</em> (2017b)</a> further extended this method to incorporate genomic and pedigree information, in addition to accommodating <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-3.gif"/></noscript></span></span> by modeling hyperspectral band-by-environment (<span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-4.gif"/></noscript></span></span>) interactions. Their study found that models that included the <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-5.gif"/></noscript></span></span> term had higher prediction accuracies than those that did not, suggesting that hyperspectral reflectance may be a useful phenotype for modeling <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-6.gif"/></noscript></span></span> interactions.</p><p id="p-9">When collecting hyperspectral data within a multi-environment context, the number of predictors increases in proportion to the number of environments and phenotyping time-points observed, which may come at a computational cost depending on the type of prediction model used (<a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Montesinos-López <em>et al.</em> 2017b</a>). One possible approach that may minimize computation time would be to use the hyperspectral bands as a high dimensional predictor set, similar to the case of prediction with genomic markers in GBLUP, by creating a relationship matrix between individuals using the hyperspectral bands. This way, the number of bands could be very large without increasing the complexity of the GBLUP prediction model. Separate genomic marker/pedigree and hyperspectral reflectance kernels could be integrated to model the genetic main effects and <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-7.gif"/></noscript></span></span> interactions, respectively.</p><p id="p-10">Multi-environment field trials are often unbalanced, which can complicate their use in prediction across environments. When deploying HTP in large breeding programs, it can be difficult to ensure that all locations are phenotyped at the same stage and the same number of times throughout the season. Weather conditions, technical difficulties with the cameras or sensors, and scheduling with contracted pilots/airports may prohibit the use of aerial HTP on certain days, and lines grown at different sites may develop at faster or slower rates depending on weather and management conditions. As a result, sites may have different numbers of observed HTP time-points, as in <a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Sun <em>et al.</em> (2017)</a>. It is well documented that canopy spectral reflectance varies according to crop phenology (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Viña <em>et al.</em> 2004</a>; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Zhang <em>et al.</em> 2003</a>). One strategy for comparing HTP time-points across varying sites may be to classify them according to the predominant developmental growth stage at the time of phenotyping.</p><p id="p-11">To test these approaches, we deployed HTP on the CIMMYT Bread Wheat Improvement Program’s multi-environment yield evaluations of advanced germplasm to phenotype a range of differentially managed treatments with a hyperspectral camera at multiple time-points throughout the growing season. The main objectives of this research were to: (1) propose a multi-kernel, multi-environment GBLUP model that involves modeling genetic main effects using genomic markers or pedigrees and modeling the <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-8.gif"/></noscript></span></span> interactions using relationship matrices derived from hyperspectral reflectance data, (2) compare the prediction accuracies of models utilizing genomic marker/pedigree main effects kernels and hyperspectral <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-9.gif"/></noscript></span></span> interaction kernels separately and in combination, and (3) determine the optimal developmental growth stages for hyperspectral phenotyping based on grain yield prediction accuracy.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials And Methods</h2><div id="sec-2" class="subsection"><h3>Experimental Data</h3><p id="p-12">The dataset included a total of 3,771 bread wheat lines evaluated at the Campo Experimental Norman E. Borlaug in Ciudad Obregón, México over the course of four breeding cycles: 2013-14, 2014-15, 2015-16, and 2016-17. In each cycle, lines were sown under five differentially managed treatments: Optimal Bed, Optimal Flat, Moderate Drought, Severe Drought, and Heat. Descriptions of the managed treatments are given in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>. The 20 managed treatment-breeding cycle combinations are herein referred to as site-years. Within each of the five managed treatments, 1,092 lines were arranged into 39 trials in a α-lattice design with three replicates and six incomplete blocks per replicate. Each replicate contained two repeated checks. ‘Kachu #1’ was sown as a check in the Optimal Bed, Optimal Flat, and Moderate Drought managed treatments. ‘Baj #1’ was used as a check in the Severe Drought and Heat managed treatments. ‘Borlaug100 F2014’ was sown as the other check in all managed treatments. All lines within a breeding cycle were evaluated in all five managed treatments, while no lines other than checks overlapped between cycles. Records for some lines were removed from the analysis due to unavailability of genotypic or agronomic data, resulting in the final dataset: 588 lines in 2013-14, 1,033 lines in 2014-15, 1,063 lines in 2015-16, and 1,087 lines in 2016-17, for a total of 56,565 plots phenotyped. Each breeding cycle contained full-sib families with an average of two full-sibs per family.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/492647/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/492647/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Description of the field management conditions in each of the five managed treatments sown at the Campo Experimental Norman E. Borlaug in Ciudad Obregón, México</span><div class="sb-div caption-clear"></div></div></div><p id="p-13">Grain yield (GY) in t ha<sup>-1</sup> and lodging evaluated on an ordinal scale (0: no lodging; 5: completely lodged) were assessed in all three replicates. Heading date was recorded as the date of 50% spike emergence within the plot. Maturity date was recorded as the date of senescence of the peduncle for 50% of stems in the plot. Days to heading (DTHD) and days to maturity (DTMT) were measured as the number of days to reach heading and maturity, starting from the date of the first irrigation if sowing was on dry soil or from the sowing date if sowing was in pre-irrigated fields. DTHD and DTMT were assessed for the first replicate only due to the high heritability of those traits.</p></div><div id="sec-3" class="subsection"><h3>Hyperspectral Phenotyping</h3><p id="p-14">Hyperspectral reflectance data were collected with a hyperspectral camera (A-series, Micro-Hyperspec VNIR, Headwall Photonics, Fitchburg, MA, USA) as part of the Alava Remote Sensing Spectral Solution (ARS3, Alava Ingenieros, Madrid, Spain) mounted in a Piper PA-16 Clipper aircraft. The camera’s sensor had a 12-bit radiometric resolution, covering the light spectrum in the 380–850nm region with a 7.5nm full width at half maximum and set with an integration time of 18ms. Spectral binning resulted in 62 wavelengths between 398-847nm (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Rodrigues <em>et al.</em> 2018</a>). The flights were scheduled around noon (GMT-7) and aligned to the solar azimuth angle at a height of 300m, resulting in 30cm resolution. The aerial imagery acquisition during the growing season was spaced approximately at seven- to ten-day intervals on mostly clear days.</p><p id="p-15">Radiometric calibration of the sensor was done using coefficients derived from a calibrated uniform light source and an integrating sphere (CSTMUSS2000C Uniform Source System, LabSphere, North Sutton, NH, USA). Dark frame correction was performed for each flight dataset. Atmospheric calibration was performed using irradiance measurements acquired at the beginning and end of each flight using a Jaz spectrometer with a CC-3 Cosine Corrector (Ocean Optics Inc, FL, USA) for the 2016-17 breeding cycle. For remaining three breeding cycles, irradiance was modeled using aerosol optical depth from sun-photometer measurements (Microtops II, Solar Light Company, Glenside, PA, USA) based on the SMARTS simulation model (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Gueymard 1995</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Gueymard 2005</a>).</p><p id="p-16">Ortho-rectification and georeferencing of the imagery were performed using PARGE (ReSe Applications Schläpfer, Wil, Switzerland) based on data from the inertial navigation system (INS) attached to the camera (IG-500N model, SGB systems S. A. S., Carrières-sur-Seine, France). Hyperspectral reflectance data were extracted from the aerial images using the mean value of the pixels inside the central area of each observed plot, avoiding 0.5m from the plot border.</p><p id="p-17">Each hyperspectral phenotyping time-point within each site-year was assigned a developmental growth stage classification according to the predominant growth stage of the lines within the site-year at the time of phenotyping (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). The vegetative (VEG) stage was defined as the period between germination and 50% of plots at heading. The heading stage (HEAD) was defined as the period between 50% of plots at heading and 100% of plots at heading. The grain fill stage (GF) was defined as the period between 100% of plots at heading and 100% of plots at maturity.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/492736/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/492736/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Classification of hyperspectral phenotyping time-points as vegetative (VEG), heading (HEAD), and grain fill (GF) phenological stages</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-4" class="subsection"><h3>Genotypic Data</h3><p id="p-18">All lines were genotyped using genotyping-by-sequencing (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Elshire <em>et al.</em> 2011</a>) according to the pipeline described in <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Poland <em>et al.</em> (2012)</a>. From the initial set of 34,900 single nucleotide polymorphisms (SNPs), 8,519 SNPs remained after excluding all markers with more than 70% of missing data or minor allele frequency less than 0.05. For each marker, missing data were imputed using the sample mean of observed values (<a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Poland <em>et al.</em> 2012</a>).</p></div><div id="sec-5" class="subsection"><h3>Basic Statistical Models</h3><p id="p-19">Within each site-year, best linear unbiased estimates (BLUEs) were calculated for the agronomic traits and for each hyperspectral band at each phenotyping time-point using the following model:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-10.gif"/></noscript></span><span class="disp-formula-label">(1)</span></span>where <em>y<sub>ijkl</sub></em> is the trait value for genotype <em>i</em> within trial <em>j</em>, replicate <em>k</em>, and block <em>l</em>; <em>μ</em> is the overall mean; <em>g<sub>i</sub></em> is the fixed effect for genotype <em>i</em>; <em>t<sub>j</sub></em> is the random effect for trial <em>j</em>, which are assumed to be independently and identically distributed according to a normal distribution with mean zero and variance σ<em><sub>t</sub></em><sup>2</sup>, that is, <em>t<sub>j</sub></em> <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-11.gif"/></noscript></span></span><em>N</em>(0, σ<em><sub>t</sub></em><sup>2</sup>); <em>r<sub>k(j)</sub></em> <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-12.gif"/></noscript></span></span><em>N</em>(0, σ<em><sub>r</sub></em><sup>2</sup>) is the random effect for replicate <em>k</em> within trial <em>j</em>; <em>b<sub>l(jk)</sub></em> <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-13.gif"/></noscript></span></span> <em>N</em>(0, σ<em><sub>b</sub></em><sup>2</sup>) is the random effect for block <em>l</em> within replicate <em>k</em> and trial <em>j</em>; and ε<em><sub>ijkl</sub></em> <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-14.gif"/></noscript></span></span><em>N</em>(0, <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-15.gif"/></noscript></span></span>) is the residual effect. For DTHD and DTMT, which were recorded for only one replicate, the replicate and block effects were excluded from the model. For validation of prediction models, best linear unbiased predictions (BLUPs) for GY within each site-year were calculated by treating the effect for genotype <em>g<sub>i</sub></em> in model (1) as random with <em>g<sub>i</sub></em> <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-16.gif"/></noscript></span></span> <em>N</em>(0, σ<em><sub>g</sub></em><sup>2</sup>) and by including a covariate for lodging as a fixed effect, fit only when lodging was observed within a site-year. To correct for the influence of phenology, BLUPs for GY were calculated again using DTHD included as a fixed effect in model (1).</p><p id="p-20">BLUEs were also calculated for each hyperspectral band on a developmental growth stage basis according to the growth stage classifications detailed in <a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>. BLUEs for each hyperspectral band for the VEG, HEAD, and GF stages were estimated by fitting the following model:<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-17.gif"/></noscript></span><span class="disp-formula-label">(2)</span></span>where <em>y<sub>ijkl</sub></em> is the hyperspectral reflectance trait value for genotype <em>i</em> on time-point <em>j</em> and within trial <em>k</em>, replicate <em>l</em>, and block <em>m</em>; <em>μ</em> is the overall mean; <em>g<sub>i</sub></em> is the fixed genetic effect for genotype <em>i</em>; <em>d<sub>j</sub></em> <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-18.gif"/></noscript></span></span><em>N</em>(0, σ<em><sub>d</sub></em><sup>2</sup>) is a random effect for time-point <em>j</em>; <em>t<sub>k(j)</sub></em> <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-19.gif"/></noscript></span></span> <em>N</em>(0, σ<em><sub>t</sub></em><sup>2</sup>) is random effect for trial <em>k</em> nested within time-point <em>j</em>; <em>r<sub>l(jk)</sub></em> <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-20.gif"/></noscript></span></span><em>N</em>(0, σ<em><sub>r</sub></em><sup>2</sup>) is the random effect for replicate <em>l</em> nested within time-point <em>j</em> and trial k; <em>b<sub>m(jkl)</sub></em> ∼iid <em>N</em>(0, σ<em><sub>b</sub></em><sup>2</sup>) is the random effect for block <em>m</em> nested within time-point <em>j</em>, trial k, and rep l; and ε<em><sub>ijklm</sub></em> ∼iid <em>N</em>(0, <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-21.gif"/></noscript></span></span>) is the residual effect. BLUEs were also calculated for each hyperspectral band across all available phenotyping time-points (ALL) using model (2).</p><p id="p-21">Broad-sense heritability within site-years was calculated for GY (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bernardo 2010</a>) and each hyperspectral band as:<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-22.gif"/></noscript></span><span class="disp-formula-label">(3)</span></span>where σ<em><sub>g</sub></em><sup>2</sup> is the genetic variance, <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-23.gif"/></noscript></span></span> is the error variance, and <em>nreps</em> is the number of replicates (<em>nreps</em> = 3). σ<em><sub>g</sub></em><sup>2</sup> and <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-24.gif"/></noscript></span></span> were derived by fitting model (1) with the effect for genotype <em>g<sub>i</sub></em> treated as random with <em>g<sub>i</sub></em> <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-25.gif"/></noscript></span></span> <em>N</em>(0, σ<em><sub>g</sub></em><sup>2</sup>). For each breeding cycle, Pearson’s correlations for GY between managed treatments were calculated using the BLUEs derived from model (1). Pearson’s correlations were also calculated between GY BLUEs and the BLUEs for each hyperspectral band calculated in model (2).</p></div><div id="sec-6" class="subsection"><h3>Relationship Matrices</h3><p id="p-22">Genetic relationships between individuals were modeled using genomic markers, pedigrees, and hyperspectral reflectance phenotypes. The genomic relationship matrix (<strong><em>G</em></strong>) was calculated according to <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Endelman and Jannink (2012)</a>. The additive relationship matrix (<strong><em>A</em></strong>) was derived from pedigrees and calculated as twice the coefficient of parentage.</p><p id="p-23">Hyperspectral reflectance-based relationship matrices (<strong><em>H</em></strong>) were calculated within each site-year using the hyperspectral BLUEs calculated for 1) the individual time-points from model (1), 2) the developmental growth stages from model (2), and 3) all time-points from model (2). The matrices were calculated as <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-26.gif"/></noscript></span></span>, where <strong><em>S</em></strong> is a matrix of the centered and standardized BLUEs of the hyperspectral bands and <em>nbands</em> is the total number of hyperspectral bands observed (<em>nbands</em> = 62).</p></div><div id="sec-7" class="subsection"><h3>Prediction Models</h3><div id="sec-8" class="subsection"><h4>Genetic Main Effects:</h4><p id="p-24">To compare the utility of hyperspectral reflectance-based models to standard marker- and pedigree-based methods for genomic prediction, the following single-kernel genetic main effects model was fitted using marker- and pedigree-derived relationship matrices:<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-27.gif"/></noscript></span><span class="disp-formula-label">(4)</span></span>where <em>y<sub>ij</sub></em> is the BLUE of GY for genotype <em>j</em> in site-year <em>i</em>, <em>μ</em> is the overall mean, <em>E<sub>i</sub></em> is the fixed effect for site-year (<em>i = 1,...,I)</em>, <em>g<sub>j</sub></em> is the random effect for genotype <em>j</em> (<em>j = 1,...,J)</em>, and ε<em><sub>ij</sub></em> is the residual effect. We assume that the joint distribution of genotype effects is distributed according to a multivariate normal distribution with mean <strong>0</strong> and variance-covariance matrix σ<em><sub>g</sub></em><sup>2</sup><strong><em>K</em></strong>, that is,<span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-28.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-29.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-30.gif"/></noscript></span></span> denotes the genomic variance and <strong><em>K</em></strong> represent the genomic relationship matrix between individuals <strong><em>G</em></strong> (<strong><em>K</em></strong>=<strong><em>G</em></strong>) or the additive pedigree relationship matrix <strong><em>A</em></strong> (<strong><em>K</em></strong><em>=</em><strong><em>A</em></strong>).</p></div><div id="sec-9" class="subsection"><h4>Hyperspectral Reflectance Main Effects:</h4><p id="p-25">To mimic a situation in which genomic marker and pedigree information are not available, the following single-kernel model was fitted using the hyperspectral reflectance-derived relationship matrices only:<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-31.gif"/></noscript></span><span class="disp-formula-label">(5)</span></span>where <em>y<sub>ij</sub></em>, μ, and <em>E<sub>i</sub></em> are as defined in model (4). <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-32.gif"/></noscript></span></span> is the random effect of the hyperspectral bands for genotype <em>j</em> in site-year i with the joint distribution of the hyperspectral bands as <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-33.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-34.gif"/></noscript></span></span> is the hyperspectral band variance and <strong><em>H</em></strong> is the hyperspectral reflectance-derived relationship matrix. Unlike matrices <strong><em>G</em></strong> and <strong><em>A</em></strong>, which have only one row and column for each unique genotype evaluated, the matrix <strong><em>H</em></strong> has a row and column for each unique site-year-genotype combination where <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-35.gif"/></noscript></span></span> results in the total number of site-year-genotype combinations.</p></div><div id="sec-10" class="subsection"><h4>Genetic Main Effects + Genetic G × E:</h4><p id="p-26">As a basis for comparison to assess the advantage of <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-36.gif"/></noscript></span></span> models that integrate both marker or pedigree relationship matrices with hyperspectral reflectance-derived relationship matrices, model (4) was extended to accommodate <span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-37.gif"/></noscript></span></span> interactions using marker or pedigree information. The following multi-kernel model was fit:<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-38.gif"/></noscript></span><span class="disp-formula-label">(6)</span></span>where <em>y<sub>ij</sub></em>, μ, <em>E<sub>i</sub></em>, and <span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-39.gif"/></noscript></span></span> are as defined in model (4). The <span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-40.gif"/></noscript></span></span> term is assumed to have multivariate normal distribution <span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-41.gif"/></noscript></span></span> <em>MVN</em>(<strong>0</strong>, (<strong>Z<sub>g</sub><em>K</em>Z<sub>g</sub></strong><sup>T</sup>)∘(<strong>Z<sub>E</sub>Z<sub>E</sub></strong><sup>T</sup>)<span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-42.gif"/></noscript></span></span>) where <strong>Z<sub>g</sub></strong> and <strong>Z<sub>E</sub></strong> are incidence matrices for genotypes and site-years, <strong><em>K</em></strong> represents the genomic relationship matrix (<strong><em>K</em></strong>=<strong><em>G</em></strong>) or the additive pedigree relationship matrix (<strong><em>K</em></strong><em>=</em><strong><em>A</em></strong>), and <span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-43.gif"/></noscript></span></span> is the variance component for <span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-44.gif"/></noscript></span></span> (<a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Jarquín <em>et al.</em> 2014</a>).</p></div><div id="sec-11" class="subsection"><h4>Genetic Main Effects + Hyperspectral Reflectance G × E:</h4><p id="p-27">Finally, a multi-kernel model using marker or pedigree information to estimate the genetic main effects and hyperspectral reflectance phenotypes to model the <span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-45.gif"/></noscript></span></span> interactions was fitted:<span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-46.gif"/></noscript></span><span class="disp-formula-label">(7)</span></span>Here, <em>y<sub>ij</sub></em>, <span class="inline-formula" id="inline-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-47.gif"/></noscript></span></span> <em>E<sub>i</sub></em>, and <span class="inline-formula" id="inline-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-48.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-48.gif"/></noscript></span></span>are defined as above in model (4). <span class="inline-formula" id="inline-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-49.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-49.gif"/></noscript></span></span> is assumed to have multivariate normal distribution <span class="inline-formula" id="inline-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-50.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-50.gif"/></noscript></span></span> <em>MVN</em>(<strong>0</strong>, (<strong>Z<sub>g</sub><em>H</em>Z<sub>g</sub></strong><sup>T</sup>)∘(<strong>Z<sub>E</sub>Z<sub>E</sub></strong><sup>T</sup>)<span class="inline-formula" id="inline-formula-44"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-51.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-51.gif"/></noscript></span></span>) where <strong>Z<sub>g</sub></strong> and <strong>Z<sub>E</sub></strong> are incidence matrices for genotypes and site-years and <span class="inline-formula" id="inline-formula-45"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-52.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-52.gif"/></noscript></span></span> is the variance component for <span class="inline-formula" id="inline-formula-46"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-53.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-53.gif"/></noscript></span></span>. The (<strong>Z<sub>g</sub><em>H</em>Z<sub>g</sub></strong><sup>T</sup>)∘(<strong>Z<sub>E</sub>Z<sub>E</sub></strong><sup>T</sup>) term is obtained with the block diagonal matrix <em>BDiag</em>(<strong><em>H</em></strong><sub>11</sub>,...,<strong><em>H</em></strong><sub>II</sub>) where <strong><em>H</em></strong><sub>ii</sub> represent the hyperspectral relationship matrices for genotypes in site-year <em>i = 1,...,I</em>.</p></div></div><div id="sec-12" class="subsection"><h3>Software</h3><p id="p-28">Processing of the hyperspectral images was performed using the ARS3 hyproQ software (Álava Ingenieros, Madrid, Spain). Plot polygons for tabular data extraction were generated using ArcGIS (ESRI, Redlands, California, USA). Images were aligned manually in ArcGIS if they did not overlay the plot polygons due to INS inaccuracy.</p><p id="p-29">All models were fit using the R statistical programming language (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">R Core Team 2018</a>). Basic models (1-3) were fit with the package “ASReml-R” (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Gilmour <em>et al.</em> 2014</a>) for R, while the prediction models (4-7) were fit using the “BGLR” package (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">de los Campos and Pérez-Rodríguez 2014</a>) for R. The marker-based genetic relationship matrices were calculated using the A.mat() function within the “rrBLUP” package (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Endelman 2011</a>) for R. The coefficients of parentage for the pedigree relationship matrices were estimated using the “Browse” application within the International Crop Information System software package (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">McLaren <em>et al.</em> 2000</a>).</p></div><div id="sec-13" class="subsection"><h3>Assessing Model Prediction Accuracy</h3><p id="p-30">The above models were fit to assess model prediction accuracy for three prediction strategies representing different testing and evaluation problems relevant to plant breeding programs: 1) <strong>within site-year</strong>, 2) <strong>within breeding cycle/across managed treatments</strong>, and 3) <strong>across breeding cycles/within managed treatment</strong>.</p><p id="p-31">A train-test (TRN-TST) validation scheme (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Daetwyler <em>et al.</em> 2013</a>) of 20 random partitions was used to assess model prediction accuracy for all prediction strategies. For within site-year prediction, a random 80% of records within a given site-year were assigned to the TRN set, and the remaining 20% were used as the TST set for prediction. This reflects a scenario in which prediction of the trait values for a set of unobserved lines is performed within a managed treatment and breeding cycle of interest. Models (4) and (5) were fitted without the <em>E<sub>i</sub></em> term for site-year and models (6) and (7) were not tested for the within site-year prediction scheme because multiple site-years were not considered.</p><p id="p-32">For prediction within breeding cycle/across managed treatments, predictions were carried out across the five managed treatments within a single breeding cycle. The TRN set consisted of all records from four out of the five managed treatments within a breeding cycle plus 20% of records from the fifth managed treatment. The remaining 80% of records from the fifth managed treatment were assigned to the TST set for prediction. In this situation, lines have been previously characterized in all managed treatments except one, where only 20% of records were available.</p><p id="p-33">For prediction across breeding cycles/within managed treatment, predictions were performed across the four breeding cycles but within a single managed treatment. The TRN set contained all records from three out of the four breeding cycles for a given managed treatment plus 20% of records from the fourth breeding cycle. The remaining 80% of records from the fourth breeding cycle were assigned to the TST set for prediction. This scenario predicts performance of unobserved lines that are 1-3 breeding cycles removed from lines in the training set.</p><p id="p-34">Table S1 illustrates examples of TRN-TST partitioning for the three prediction schemes. For all prediction strategies, model prediction accuracy was assessed as the Pearson’s correlation between predictions for GY and GY BLUPs with and without correction for DTHD calculated in model (1). Reported values are the mean and standard deviation of the 20 random TRN-TST partitions implemented. The same partitioning “splits” were used to assess all models to ensure fair comparisons.</p></div><div id="sec-14" class="subsection"><h3>Data Availability</h3><p id="p-35">All phenotypic and genotypic data required to confirm the results presented in this study are available on CIMMYT Dataverse link: hdl:11529/10548109. The “GID” column denotes the unique identifiers for the genotypes. Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.7653473">https://doi.org/10.25387/g3.7653473</a>.</p></div></div><div class="section results" id="sec-15"><h2 class="">Results</h2><div id="sec-16" class="subsection"><h3>Descriptive Statistics</h3><p id="p-36">To evaluate the potential of integrating aerial hyperspectral reflectance phenotypes into GS to improve prediction accuracy for GY in wheat, we deployed an airplane equipped with a hyperspectral camera to phenotype wheat breeding trials in 20 site-years at multiple time-points throughout the growing season. Across the site-years, the Optimal Bed and Optimal Flat managed treatments had higher GY than the stressed Moderate Drought, Severe Drought, and Heat managed treatments (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). The standard deviations for GY remained relatively stable across site-years, ranging from 0.30 to 0.68 t ha<sup>-1</sup> (Table S2). Broad-sense heritability for GY was high across site-years, ranging from 0.58 to 0.94 (Table S2). Overall, correlations for GY between managed treatments were moderately positive (Figure S1). The managed treatments that received similar levels of irrigation (<em>e.g.</em>, Optimal Bed with Optimal Flat; Moderate Drought with Severe Drought) showed significant correlations (<em>p</em>-value ≤ 0.05) in all breeding cycles.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/4/1231/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Boxplot of BLUEs for wheat grain yield (t ha-1) in each of the 20 observed site-years." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1095196470" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Boxplot of BLUEs for wheat grain yield (t ha&lt;sup&gt;-1&lt;/sup&gt;) in each of the 20 observed site-years.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/4/1231/F1.medium.gif" width="440" height="201"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/9/4/1231/F1.medium.gif" width="440" height="201"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/4/1231/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/4/1231/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492625" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-37" class="first-child">Boxplot of BLUEs for wheat grain yield (t ha<sup>-1</sup>) in each of the 20 observed site-years.</p><div class="sb-div caption-clear"></div></div></div><p id="p-38">Between 3 and 11 hyperspectral phenotyping time-points were collected within each site-year (<a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>). In most site-years, at least one hyperspectral phenotyping time-point was collected during each of the three developmental growth stages. While the managed treatments within a breeding cycle were often phenotyped on the same date, the growth stage at the time of phenotyping frequently differed among the managed treatments (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). During the 2015-16 cycle, technological issues with the camera prevented early season phenotyping. As a result, no hyperspectral data were collected during the VEG stage. The Optimal Bed, Optimal Flat, and Heat managed treatments had hyperspectral phenotypes for the HEAD and GF stages only, while the Moderate Drought and Severe Drought managed treatments had hyperspectral phenotypes for the GF stage only due to early maturation of the crop under water stress.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/4/1231/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="A graphical representation of the unbalanced nature of the hyperspectral reflectance phenotypic data. Four site-years are represented: 2014-15 Optimal Flat, 2014-15 Severe Drought, 2016-17 Optimal Flat, and 2016-17 Severe Drought. The histograms represent heading dates. Each dashed line corresponds to a hyperspectral phenotyping date colored according to the predominant growth stage of the lines at the time of phenotyping." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1095196470" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;A graphical representation of the unbalanced nature of the hyperspectral reflectance phenotypic data. Four site-years are represented: 2014-15 Optimal Flat, 2014-15 Severe Drought, 2016-17 Optimal Flat, and 2016-17 Severe Drought. The histograms represent heading dates. Each dashed line corresponds to a hyperspectral phenotyping date colored according to the predominant growth stage of the lines at the time of phenotyping.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/4/1231/F2.medium.gif" width="440" height="193"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/9/4/1231/F2.medium.gif" width="440" height="193"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/4/1231/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/4/1231/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492755" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-39" class="first-child">A graphical representation of the unbalanced nature of the hyperspectral reflectance phenotypic data. Four site-years are represented: 2014-15 Optimal Flat, 2014-15 Severe Drought, 2016-17 Optimal Flat, and 2016-17 Severe Drought. The histograms represent heading dates. Each dashed line corresponds to a hyperspectral phenotyping date colored according to the predominant growth stage of the lines at the time of phenotyping.</p><div class="sb-div caption-clear"></div></div></div><p id="p-40">Broad-sense heritability estimates for the hyperspectral bands were generally between 0.5 and 0.8 for most phenotyping time-points (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). Heritabilities were relatively homogeneous across individual time-points and developmental growth stages. In all site-years, the lowest heritabilities were observed for the hyperspectral bands between 398 and 425 nm. Heritabilities in the 2013-14 Optimal Flat site-year were notably lower than in other site-years, falling between 0.00 and 0.25, though the GF time-points were more heritable with values around 0.40. Low heritabilities (&lt;0.20) were also observed for the 9 March time-point in the 2015-16 Moderate Drought site-year.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/4/1231/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Broad-sense heritabilities of the hyperspectral wavelengths for each phenotyping time-point within each site-year. Each boxplot represents the distribution of broad-sense heritability values for the 62 hyperspectral wavelengths observed. The colors correspond to the developmental growth stage of the site-year at the time of hyperspectral phenotyping." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1095196470" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Broad-sense heritabilities of the hyperspectral wavelengths for each phenotyping time-point within each site-year. Each boxplot represents the distribution of broad-sense heritability values for the 62 hyperspectral wavelengths observed. The colors correspond to the developmental growth stage of the site-year at the time of hyperspectral phenotyping.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/4/1231/F3.medium.gif" width="440" height="278"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/9/4/1231/F3.medium.gif" width="440" height="278"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/4/1231/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/4/1231/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492730" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-41" class="first-child">Broad-sense heritabilities of the hyperspectral wavelengths for each phenotyping time-point within each site-year. Each boxplot represents the distribution of broad-sense heritability values for the 62 hyperspectral wavelengths observed. The colors correspond to the developmental growth stage of the site-year at the time of hyperspectral phenotyping.</p><div class="sb-div caption-clear"></div></div></div><p id="p-42">Correlations between the individual hyperspectral bands and GY ranged from -0.68 to 0.64 across the 20 site-years and were most frequently the strongest above 500 nm (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). While there were some observable similarities in correlation patterns among time-points taken during the same growth stage within the same site-year, patterns were not uniform across site-years, breeding cycles, or managed treatments. In some site-years, there were clear differences in correlation patterns between developmental growth stages. For example, in the 2014-15 Optimal Bed site-year, correlations between GY and hyperspectral reflectance in the 398-700 nm range were around 0 during the VEG stage but became progressively more negative during the HEAD and GF stages, reaching around -0.4. For the same managed treatment during the 2016-17 breeding cycle, distinct differences in correlation patterns between the three developmental growth stages were also observed; however, the patterns do not reflect those observed in 2014-15. Despite irregularities in correlation patterns, the correlations between hyperspectral bands from the GF stages and GY were, on average, stronger by 0.10 than for hyperspectral phenotypes taken during the VEG and HEAD stages.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/4/1231/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Empirical correlations between grain yield and hyperspectral reflectance BLUEs within each site-year. Each solid line represents a phenotyping time-point and the Pearson&#x2019;s correlation between grain yield and the 62 hyperspectral wavelengths observed. Lines are colored according to the predominant developmental growth stage of the site-year at the time of hyperspectral phenotyping. Correlation values &#x2265; |0.10| are significant at a level of 0.001, as denoted by the dotted red lines." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1095196470" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Empirical correlations between grain yield and hyperspectral reflectance BLUEs within each site-year. Each solid line represents a phenotyping time-point and the Pearson&#x2019;s correlation between grain yield and the 62 hyperspectral wavelengths observed. Lines are colored according to the predominant developmental growth stage of the site-year at the time of hyperspectral phenotyping. Correlation values &#x2265; |0.10| are significant at a level of 0.001, as denoted by the dotted red lines.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/4/1231/F4.medium.gif" width="440" height="400"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/9/4/1231/F4.medium.gif" width="440" height="400"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/4/1231/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/4/1231/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492756" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-43" class="first-child">Empirical correlations between grain yield and hyperspectral reflectance BLUEs within each site-year. Each solid line represents a phenotyping time-point and the Pearson’s correlation between grain yield and the 62 hyperspectral wavelengths observed. Lines are colored according to the predominant developmental growth stage of the site-year at the time of hyperspectral phenotyping. Correlation values ≥ |0.10| are significant at a level of 0.001, as denoted by the dotted red lines.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-17" class="subsection"><h3>Model Prediction Accuracy</h3><div id="sec-18" class="subsection"><h4>Within Site-Year:</h4><p id="p-44">Five types of relationship matrices were used within single-kernel models (4) and (5) for within site-year prediction: genomic (<strong><em>G</em></strong>), pedigree (<strong><em>A</em></strong>), individual hyperspectral phenotyping time-points (<em>e.g.</em>, <strong><em>H.10Jan</em></strong>, <strong><em>H.23Mar</em></strong>, etc.), hyperspectral phenotype BLUEs calculated from multiple time-points for each developmental growth stage (<strong><em>H.VEG</em></strong>, <strong><em>H.HEAD</em></strong>, <strong><em>H.GF</em></strong>), and hyperspectral phenotype BLUEs calculated using all available time-points (<strong><em>H.ALL</em></strong>). Model (4) was used to assess accuracies using the <strong><em>G</em></strong> and <strong><em>A</em></strong> relationship matrices for prediction. Model (5) was fit to assess accuracies using the hyperspectral reflectance-derived relationship matrices.</p><p id="p-45">Prediction accuracies of models using the individual hyperspectral time-point-derived relationship matrices ranged from 0.00 to 0.75 with a mean of 0.42 (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5A</a>). These results are similar to the prediction accuracies observed for models using the <strong><em>G</em></strong> and <strong><em>A</em></strong> relationship matrices, which recorded means of 0.41 and 0.42 and ranges of 0.19-0.60 and 0.19-0.67, respectively. In 18 and 17 of the 20 site-years, at least one hyperspectral phenotyping time-point showed accuracies that were equivalent or superior to <strong><em>G</em></strong> and <strong><em>A</em></strong>, respectively. In considering the most highly predictive hyperspectral phenotyping time-point in each site-year, 11 were recorded during the GF stage, followed by 6 and 3 in the HEAD and VEG stages, respectively, although the site-years had different numbers of hyperspectral phenotyping time-points from each growth stage and some site-years had no observations recorded during the VEG and HEAD stages.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/4/1231/F5/graphic-5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Within site-year prediction accuracies, with and without correction for DTHD. Accuracy is expressed as the average Pearson&#x2019;s correlation between predictions and observed BLUPs for GY across 20 random TRN-TST partitions. Results shown according to the type of relationship matrix tested: Genomic (G), pedigree (A), individual hyperspectral time-points (e.g., H.10Jan, H.23Mar, etc.), hyperspectral BLUEs for each developmental growth stage (H.VEG, H.HEAD, H.GF), and hyperspectral BLUEs across all time-points (H.ALL). The color corresponds to the predominant developmental growth stage of the site-year at the time of phenotyping. Error bars are the standard deviation of prediction accuracy for the 20 random partitions." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1095196470" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Within site-year prediction accuracies, with and without correction for DTHD. Accuracy is expressed as the average Pearson&#x2019;s correlation between predictions and observed BLUPs for GY across 20 random TRN-TST partitions. Results shown according to the type of relationship matrix tested: Genomic (G), pedigree (A), individual hyperspectral time-points (&lt;em&gt;e.g.&lt;/em&gt;, H.10Jan, H.23Mar, etc.), hyperspectral BLUEs for each developmental growth stage (H.VEG, H.HEAD, H.GF), and hyperspectral BLUEs across all time-points (H.ALL). The color corresponds to the predominant developmental growth stage of the site-year at the time of phenotyping. Error bars are the standard deviation of prediction accuracy for the 20 random partitions.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/4/1231/F5/graphic-5.medium.gif" width="440" height="293"/><noscript><img class="highwire-fragment fragment-image" alt="" src="https://www.g3journal.org/content/ggg/9/4/1231/F5/graphic-5.medium.gif" width="440" height="293"/></noscript></span></a><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/4/1231/F5/graphic-5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/4/1231/F5/graphic-5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492780" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/4/1231/F5/graphic-6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Within site-year prediction accuracies, with and without correction for DTHD. Accuracy is expressed as the average Pearson&#x2019;s correlation between predictions and observed BLUPs for GY across 20 random TRN-TST partitions. Results shown according to the type of relationship matrix tested: Genomic (G), pedigree (A), individual hyperspectral time-points (e.g., H.10Jan, H.23Mar, etc.), hyperspectral BLUEs for each developmental growth stage (H.VEG, H.HEAD, H.GF), and hyperspectral BLUEs across all time-points (H.ALL). The color corresponds to the predominant developmental growth stage of the site-year at the time of phenotyping. Error bars are the standard deviation of prediction accuracy for the 20 random partitions." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1095196470" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Within site-year prediction accuracies, with and without correction for DTHD. Accuracy is expressed as the average Pearson&#x2019;s correlation between predictions and observed BLUPs for GY across 20 random TRN-TST partitions. Results shown according to the type of relationship matrix tested: Genomic (G), pedigree (A), individual hyperspectral time-points (&lt;em&gt;e.g.&lt;/em&gt;, H.10Jan, H.23Mar, etc.), hyperspectral BLUEs for each developmental growth stage (H.VEG, H.HEAD, H.GF), and hyperspectral BLUEs across all time-points (H.ALL). The color corresponds to the predominant developmental growth stage of the site-year at the time of phenotyping. Error bars are the standard deviation of prediction accuracy for the 20 random partitions.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/4/1231/F5/graphic-6.medium.gif" width="440" height="292"/><noscript><img class="highwire-fragment fragment-image" alt="" src="https://www.g3journal.org/content/ggg/9/4/1231/F5/graphic-6.medium.gif" width="440" height="292"/></noscript></span></a><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/4/1231/F5/graphic-6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/4/1231/F5/graphic-6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492780" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-46" class="first-child">Within site-year prediction accuracies, with and without correction for DTHD. Accuracy is expressed as the average Pearson’s correlation between predictions and observed BLUPs for GY across 20 random TRN-TST partitions. Results shown according to the type of relationship matrix tested: Genomic (G), pedigree (A), individual hyperspectral time-points (<em>e.g.</em>, H.10Jan, H.23Mar, etc.), hyperspectral BLUEs for each developmental growth stage (H.VEG, H.HEAD, H.GF), and hyperspectral BLUEs across all time-points (H.ALL). The color corresponds to the predominant developmental growth stage of the site-year at the time of phenotyping. Error bars are the standard deviation of prediction accuracy for the 20 random partitions.</p><div class="sb-div caption-clear"></div></div></div><p id="p-47">The level of prediction accuracy for the individual hyperspectral time-points was highly correlated with the strength of the relationship between hyperspectral reflectance and GY (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>). For a given time-point within a site-year, the correlation between hyperspectral reflectance and GY was taken for each of the 62 hyperspectral wavelengths. The average of the absolute values of those correlations, or their relative magnitudes, was then compared to the level of prediction accuracy for that time-point. For the 71 time-point/site-year combinations, the average magnitude of the correlation between hyperspectral reflectance and GY explained 51% of the variation in prediction accuracy, though the two characters exhibited a non-linear relationship (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6A</a>). Correlations beyond 0.25 between hyperspectral reflectance and GY provided little marginal increase in prediction accuracy. When GY was corrected for DTHD, this relationship was linear, and the correlation between reflectance and GY explained 26% of the variation in prediction accuracy (<a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure 6B</a>).</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/4/1231/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The relationship between individual hyperspectral time-point prediction accuracy and the average magnitude of the correlations between hyperspectral bands and GY. The absolute values of the correlations between hyperspectral bands and GY were calculated and then averaged across the 62 bands for each time-point within each site-year. Plotted on the x-axis, this represents the average strength of the relationship between hyperspectral reflectance and GY for each time-point within each site-year. The y-axis shows the prediction accuracy for each individual hyperspectral time-point in within site-year prediction." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1095196470" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;The relationship between individual hyperspectral time-point prediction accuracy and the average magnitude of the correlations between hyperspectral bands and GY. The absolute values of the correlations between hyperspectral bands and GY were calculated and then averaged across the 62 bands for each time-point within each site-year. Plotted on the x-axis, this represents the average strength of the relationship between hyperspectral reflectance and GY for each time-point within each site-year. The y-axis shows the prediction accuracy for each individual hyperspectral time-point in within site-year prediction.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/4/1231/F6.medium.gif" width="266" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/9/4/1231/F6.medium.gif" width="266" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/4/1231/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/4/1231/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492764" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-48" class="first-child">The relationship between individual hyperspectral time-point prediction accuracy and the average magnitude of the correlations between hyperspectral bands and GY. The absolute values of the correlations between hyperspectral bands and GY were calculated and then averaged across the 62 bands for each time-point within each site-year. Plotted on the x-axis, this represents the average strength of the relationship between hyperspectral reflectance and GY for each time-point within each site-year. The y-axis shows the prediction accuracy for each individual hyperspectral time-point in within site-year prediction.</p><div class="sb-div caption-clear"></div></div></div><p id="p-49">In considering prediction models using relationship matrices derived from the hyperspectral phenotype BLUEs for each growth stage, relationship matrix <strong><em>H.GF</em></strong> had the highest prediction accuracies in 11 of the 20 site-years, followed by <strong><em>H.HEAD</em></strong> in 8 site-years and <strong><em>H.VEG</em></strong> in 1 site-year. Combining multiple time-points in this manner did not necessarily increase accuracy. The accuracies of models using relationship matrices <strong><em>H.VEG</em></strong>, <strong><em>H.HEAD</em></strong>, and <strong><em>H.GF</em></strong> were comparable to the mean of the accuracies of models using individual time-point-derived relationship matrices. Likewise, the accuracies of models using relationship matrix <strong><em>H.ALL</em></strong>, which combined all available time-points, were on average slightly higher than the mean of the accuracies of the individual time-point models but lower than models using the most predictive time-point.</p><p id="p-50">While correcting for DTHD had negligible impacts on prediction accuracy for models using the <strong><em>G</em></strong> and <strong><em>A</em></strong> relationship matrices, the accuracies of models using the hyperspectral reflectance-derived relationship matrices decreased by 0.10 on average (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5B</a>). The greatest reductions were observed in the Severe Drought and Moderate Drought treatments, which showed average decreases in accuracy of 0.20 and 0.18, respectively. After the DTHD correction, at least one individual time-point-derived relationship matrix recorded prediction accuracies that were equivalent or superior to accuracies when using relationship matrices <strong><em>G</em></strong> and <strong><em>A</em></strong> in 13 out of the 20 site-years.</p></div><div id="sec-19" class="subsection"><h4>Within Breeding Cycle/Across Managed Treatments:</h4><p id="p-51">For prediction across managed treatments within a breeding cycle, the genomic-, pedigree-, and hyperspectral reflectance-derived relationship matrices were tested individually in single-kernel models and in combination in multi-kernel models (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>). Prediction accuracies are shown in <a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7</a>. Single-kernel models included the following: genetic main effects (<strong><em>G</em></strong> and <strong><em>A</em></strong>) from model (4) and hyperspectral reflectance main effects (<strong><em>H.VEG</em></strong>, <strong><em>H.HEAD</em></strong>, <strong><em>H.GF</em></strong>, and <strong><em>H.ALL</em></strong>) from model (5). Multi-kernel models were built by combining a main effects kernel with a<span class="inline-formula" id="inline-formula-47"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-54.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-54.gif"/></noscript></span></span> interaction kernel in models (6) and (7). In model (6), the <span class="inline-formula" id="inline-formula-48"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-55.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-55.gif"/></noscript></span></span> interaction kernel was also fit using relationship matrices <strong><em>G</em></strong> or <strong><em>A</em></strong>. These models are herein referred to as <strong><em>G + G<sub>GxE</sub></em></strong> and <strong><em>A + A<sub>GxE</sub></em></strong>. In model (7), the hyperspectral reflectance-derived relationship matrices were used to estimate the <span class="inline-formula" id="inline-formula-49"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-56.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-56.gif"/></noscript></span></span> interaction kernel. These models are referred to as <strong><em>G + H.VEG<sub>GxE</sub></em></strong>, <strong><em>A + H.HEAD<sub>GxE</sub></em></strong>, etc. according to the respective relationship matrices used to estimate the genetic main effects and <span class="inline-formula" id="inline-formula-50"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-57.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-57.gif"/></noscript></span></span> interactions. The <strong><em>H.VEG</em></strong>, <strong><em>G + H.VEG<sub>GxE</sub></em></strong>, and <strong><em>A + H.VEG<sub>GxE</sub></em></strong> models were assessed for 14 out of the 20 site-years due to the unavailability of hyperspectral data at the vegetative stage in the remaining 6 site-years. Likewise, the <strong><em>H.HEAD</em></strong>, <strong><em>G + H.HEAD<sub>GxE</sub></em></strong>, and <strong><em>A + H.HEAD<sub>GxE</sub></em></strong> models were assessed for 18 out of the 20 site-years. The remaining models using relationship matrices <strong><em>H.GF</em></strong> and <strong><em>H.ALL</em></strong> were tested for all site-years.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/492667/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/492667/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Description of the single- and multi-kernel models tested for prediction within breeding cycle/across managed treatments and across breeding cycles/within managed treatment</span><div class="sb-div caption-clear"></div></div></div><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/4/1231/F7/graphic-8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Within breeding cycle/across managed treatments prediction accuracies, with and without correction for DTHD. Accuracy is expressed as the average Pearson&#x2019;s correlation between predictions and observed BLUPs for GY across 20 random TRN-TST partitions. In each partition, the TRN set consisted of all records from four out of the five managed treatments within the breeding cycle plus 20% of records from the fifth managed treatment. Single-kernel models tested were genetic main effects only (G or A) and hyperspectral reflectance main effects (H.VEG, H.HEAD, H.GF, H.ALL). Multi-kernel models assessed were genetic main effects plus genetic GxE (G + GGxE, A + AGxE) and genetic main effects plus hyperspectral reflectance GxE (e.g., G + H.VEGGxE, A + H.ALLGxE, etc.). The color corresponds to the developmental growth stage of the site-year at the time of phenotyping. Error bars are the standard deviation of prediction accuracy for the 20 random partitions." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1095196470" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Within breeding cycle/across managed treatments prediction accuracies, with and without correction for DTHD. Accuracy is expressed as the average Pearson&#x2019;s correlation between predictions and observed BLUPs for GY across 20 random TRN-TST partitions. In each partition, the TRN set consisted of all records from four out of the five managed treatments within the breeding cycle plus 20% of records from the fifth managed treatment. Single-kernel models tested were genetic main effects only (G or A) and hyperspectral reflectance main effects (H.VEG, H.HEAD, H.GF, H.ALL). Multi-kernel models assessed were genetic main effects plus genetic GxE (G + G&lt;sub&gt;GxE&lt;/sub&gt;, A + A&lt;sub&gt;GxE&lt;/sub&gt;) and genetic main effects plus hyperspectral reflectance GxE (&lt;em&gt;e.g.&lt;/em&gt;, G + H.VEG&lt;sub&gt;GxE&lt;/sub&gt;, A + H.ALL&lt;sub&gt;GxE&lt;/sub&gt;, etc.). The color corresponds to the developmental growth stage of the site-year at the time of phenotyping. Error bars are the standard deviation of prediction accuracy for the 20 random partitions.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/4/1231/F7/graphic-8.medium.gif" width="440" height="292"/><noscript><img class="highwire-fragment fragment-image" alt="" src="https://www.g3journal.org/content/ggg/9/4/1231/F7/graphic-8.medium.gif" width="440" height="292"/></noscript></span></a><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/4/1231/F7/graphic-8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/4/1231/F7/graphic-8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492656" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/4/1231/F7/graphic-9.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Within breeding cycle/across managed treatments prediction accuracies, with and without correction for DTHD. Accuracy is expressed as the average Pearson&#x2019;s correlation between predictions and observed BLUPs for GY across 20 random TRN-TST partitions. In each partition, the TRN set consisted of all records from four out of the five managed treatments within the breeding cycle plus 20% of records from the fifth managed treatment. Single-kernel models tested were genetic main effects only (G or A) and hyperspectral reflectance main effects (H.VEG, H.HEAD, H.GF, H.ALL). Multi-kernel models assessed were genetic main effects plus genetic GxE (G + GGxE, A + AGxE) and genetic main effects plus hyperspectral reflectance GxE (e.g., G + H.VEGGxE, A + H.ALLGxE, etc.). The color corresponds to the developmental growth stage of the site-year at the time of phenotyping. Error bars are the standard deviation of prediction accuracy for the 20 random partitions." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1095196470" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Within breeding cycle/across managed treatments prediction accuracies, with and without correction for DTHD. Accuracy is expressed as the average Pearson&#x2019;s correlation between predictions and observed BLUPs for GY across 20 random TRN-TST partitions. In each partition, the TRN set consisted of all records from four out of the five managed treatments within the breeding cycle plus 20% of records from the fifth managed treatment. Single-kernel models tested were genetic main effects only (G or A) and hyperspectral reflectance main effects (H.VEG, H.HEAD, H.GF, H.ALL). Multi-kernel models assessed were genetic main effects plus genetic GxE (G + G&lt;sub&gt;GxE&lt;/sub&gt;, A + A&lt;sub&gt;GxE&lt;/sub&gt;) and genetic main effects plus hyperspectral reflectance GxE (&lt;em&gt;e.g.&lt;/em&gt;, G + H.VEG&lt;sub&gt;GxE&lt;/sub&gt;, A + H.ALL&lt;sub&gt;GxE&lt;/sub&gt;, etc.). The color corresponds to the developmental growth stage of the site-year at the time of phenotyping. Error bars are the standard deviation of prediction accuracy for the 20 random partitions.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/4/1231/F7/graphic-9.medium.gif" width="440" height="292"/><noscript><img class="highwire-fragment fragment-image" alt="" src="https://www.g3journal.org/content/ggg/9/4/1231/F7/graphic-9.medium.gif" width="440" height="292"/></noscript></span></a><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/4/1231/F7/graphic-9.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/4/1231/F7/graphic-9.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492656" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-53" class="first-child">Within breeding cycle/across managed treatments prediction accuracies, with and without correction for DTHD. Accuracy is expressed as the average Pearson’s correlation between predictions and observed BLUPs for GY across 20 random TRN-TST partitions. In each partition, the TRN set consisted of all records from four out of the five managed treatments within the breeding cycle plus 20% of records from the fifth managed treatment. Single-kernel models tested were genetic main effects only (G or A) and hyperspectral reflectance main effects (H.VEG, H.HEAD, H.GF, H.ALL). Multi-kernel models assessed were genetic main effects plus genetic GxE (G + G<sub>GxE</sub>, A + A<sub>GxE</sub>) and genetic main effects plus hyperspectral reflectance GxE (<em>e.g.</em>, G + H.VEG<sub>GxE</sub>, A + H.ALL<sub>GxE</sub>, etc.). The color corresponds to the developmental growth stage of the site-year at the time of phenotyping. Error bars are the standard deviation of prediction accuracy for the 20 random partitions.</p><div class="sb-div caption-clear"></div></div></div><p id="p-54">When considering the single-kernel hyperspectral main effect models, 2013-14 Optimal Flat had prediction accuracies close to zero. The 2013-14 Optimal Flat site-year had high levels of lodging, which may have affected the reflectance signatures of the crop canopy. The hyperspectral reflectance phenotypes for 2013-14 Optimal Flat had lower heritabilities than all other site-years. To summarize trends for the remaining 19 site-years, the results from the single-kernel hyperspectral main effects models for 2013-14 Optimal Flat were removed from the analysis. The accuracies of <strong><em>H.VEG</em></strong> averaged 0.37 across the site-years where it was tested, which was similar to the accuracies of <strong><em>G</em></strong> (0.35) and <strong><em>A</em></strong> (0.34) (<a id="xref-fig-7-2" class="xref-fig" href="#F7">Figure 7A</a>). <strong><em>H.HEAD</em></strong>, <strong><em>H.GF</em></strong>, and <strong><em>H.ALL</em></strong> showed slightly higher averages accuracies of 0.44, 0.44, and 0.46, respectively. The greatest differences in prediction accuracy between the hyperspectral reflectance-based models and <strong><em>G</em></strong> and <strong><em>A</em></strong> were observed in the 2015-16 and 2016-17 breeding cycles. In these cycles, the <strong><em>H.HEAD</em></strong>, <strong><em>H.GF</em></strong>, and <strong><em>H.ALL</em></strong> showed accuracies that were on average 0.22 greater than the <strong>G</strong> and <strong>A</strong>. When correcting all models for DTHD, mean accuracies of <strong><em>H.VEG</em></strong> (0.29), <strong><em>H.HEAD</em></strong> (0.34), <strong><em>H.GF</em></strong> (0.30), and <strong><em>H.ALL</em></strong> (0.34) models were more similar to <strong><em>G</em></strong> (0.33) and <strong><em>A</em></strong> (0.32) (<a id="xref-fig-7-3" class="xref-fig" href="#F7">Figure 7B</a>).</p><p id="p-55">Expanding the single-kernel <strong><em>G</em></strong> and <strong><em>A</em></strong> models to account for the <span class="inline-formula" id="inline-formula-51"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-58.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-58.gif"/></noscript></span></span> interactions with the <strong><em>G + G<sub>GxE</sub></em></strong> and <strong><em>A + A<sub>GxE</sub></em></strong> models improved prediction accuracies to a level of 0.43 on average, an increase of 0.08 and 0.09 over the <strong><em>G</em></strong> and <strong><em>A</em></strong> models, respectively. These improvements in accuracy were more pronounced during the 2016-17 breeding cycle, recording gains in accuracy of 0.16 and 0.19 for <strong><em>G + G<sub>GxE</sub></em></strong> and <strong><em>A + A<sub>GxE</sub></em></strong>, respectively. Likewise, slightly greater improvements were observed for the Severe Drought (gains of 0.13 and 0.16 for <strong><em>G + G<sub>GxE</sub></em></strong> and <strong><em>A + A<sub>GxE</sub></em></strong>, respectively) managed treatment. Similar trends were observed when correcting for DTHD, though accuracies were 0.04 lower on average.</p><p id="p-56">The multi-kernel models that estimated the main effects using relationship matrices <strong><em>G</em></strong> or <strong><em>A</em></strong> and the <span class="inline-formula" id="inline-formula-52"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-59.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-59.gif"/></noscript></span></span> interactions using hyperspectral reflectance-derived relationship matrices gave the highest accuracies overall. Hyperspectral reflectance-derived relationship matrices <strong><em>H.VEG</em></strong>, <strong><em>H.HEAD</em></strong>, <strong><em>H.GF</em></strong>, and <strong><em>H.ALL</em></strong> had average accuracies of 0.54, 0.56, 0.56, and 0.58, respectively, when integrated with relationship matrix <strong><em>G</em></strong> and average accuracies of 0.53, 0.55, 0.55, and 0.57, respectively, when integrated with relationship matrix <strong><em>A</em></strong>. In most site-years, no clear “best model” could be identified among the multi-kernel models within a site-year. Accuracies were similar regardless of during which growth stage the hyperspectral reflectance data were recorded or whether genomic markers or pedigrees were used to model the main effects.</p><p id="p-57">When compared to the <strong><em>G + G<sub>GxE</sub></em></strong> and <strong><em>A + A<sub>GxE</sub></em></strong> models, the use of hyperspectral reflectance to estimate the <span class="inline-formula" id="inline-formula-53"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-60.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-60.gif"/></noscript></span></span> interactions increased prediction accuracies by an average of 0.12 and 0.10, respectively, and by an average of 0.08 and 0.06, respectively, after correcting for DTHD. Compared to the corresponding single-kernel hyperspectral reflectance models, the integration of markers or pedigrees with hyperspectral reflectance in multi-kernel models increased prediction accuracies on an average by 0.11 and 0.17, respectively, and by 0.14 and 0.19, respectively when correcting for DTHD.</p></div><div id="sec-20" class="subsection"><h4>Across Breeding Cycles/Within Managed Treatment:</h4><p id="p-58">The models tested for prediction across breeding cycles/within managed treatment were the same as those tested for prediction within breeding cycle/across managed treatments (<a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>). Prediction accuracies are shown in <a id="xref-fig-8-1" class="xref-fig" href="#F8">Figure 8</a>. On average, accuracies were 0.08 lower in this prediction scheme than for prediction within a breeding cycle and across managed treatments.</p><div id="F8" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/4/1231/F8/graphic-10.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Across breeding cycles/within managed treatment prediction accuracies, with and without correction for DTHD. Accuracy is expressed as the average Pearson&#x2019;s correlation between predictions and observed BLUPs for GY across 20 random TRN-TST partitions. In each partition, the TRN set consisted of all records from three out of the four of the breeding cycles for the managed treatment plus 20% of records from the fourth breeding cycle. Single-kernel models tested were genetic main effects only (G or A) and hyperspectral reflectance main effects (H.VEG, H.HEAD, H.GF, H.ALL). Multi-kernel models assessed were genetic main effects plus genetic GxE (G + GGxE, A + AGxE) and genetic main effects plus hyperspectral reflectance GxE (e.g., G + H.VEGGxE, A + H.ALLGxE, etc.). The color corresponds to the developmental growth stage of the site-year at the time of phenotyping. Error bars are the standard deviation of prediction accuracy for the 20 random partitions." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1095196470" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Across breeding cycles/within managed treatment prediction accuracies, with and without correction for DTHD. Accuracy is expressed as the average Pearson&#x2019;s correlation between predictions and observed BLUPs for GY across 20 random TRN-TST partitions. In each partition, the TRN set consisted of all records from three out of the four of the breeding cycles for the managed treatment plus 20% of records from the fourth breeding cycle. Single-kernel models tested were genetic main effects only (G or A) and hyperspectral reflectance main effects (H.VEG, H.HEAD, H.GF, H.ALL). Multi-kernel models assessed were genetic main effects plus genetic GxE (G + G&lt;sub&gt;GxE&lt;/sub&gt;, A + A&lt;sub&gt;GxE&lt;/sub&gt;) and genetic main effects plus hyperspectral reflectance GxE (&lt;em&gt;e.g.&lt;/em&gt;, G + H.VEG&lt;sub&gt;GxE&lt;/sub&gt;, A + H.ALL&lt;sub&gt;GxE&lt;/sub&gt;, etc.). The color corresponds to the developmental growth stage of the site-year at the time of phenotyping. Error bars are the standard deviation of prediction accuracy for the 20 random partitions.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/4/1231/F8/graphic-10.medium.gif" width="440" height="294"/><noscript><img class="highwire-fragment fragment-image" alt="" src="https://www.g3journal.org/content/ggg/9/4/1231/F8/graphic-10.medium.gif" width="440" height="294"/></noscript></span></a><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/4/1231/F8/graphic-10.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/4/1231/F8/graphic-10.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492744" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/4/1231/F8/graphic-11.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Across breeding cycles/within managed treatment prediction accuracies, with and without correction for DTHD. Accuracy is expressed as the average Pearson&#x2019;s correlation between predictions and observed BLUPs for GY across 20 random TRN-TST partitions. In each partition, the TRN set consisted of all records from three out of the four of the breeding cycles for the managed treatment plus 20% of records from the fourth breeding cycle. Single-kernel models tested were genetic main effects only (G or A) and hyperspectral reflectance main effects (H.VEG, H.HEAD, H.GF, H.ALL). Multi-kernel models assessed were genetic main effects plus genetic GxE (G + GGxE, A + AGxE) and genetic main effects plus hyperspectral reflectance GxE (e.g., G + H.VEGGxE, A + H.ALLGxE, etc.). The color corresponds to the developmental growth stage of the site-year at the time of phenotyping. Error bars are the standard deviation of prediction accuracy for the 20 random partitions." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1095196470" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Across breeding cycles/within managed treatment prediction accuracies, with and without correction for DTHD. Accuracy is expressed as the average Pearson&#x2019;s correlation between predictions and observed BLUPs for GY across 20 random TRN-TST partitions. In each partition, the TRN set consisted of all records from three out of the four of the breeding cycles for the managed treatment plus 20% of records from the fourth breeding cycle. Single-kernel models tested were genetic main effects only (G or A) and hyperspectral reflectance main effects (H.VEG, H.HEAD, H.GF, H.ALL). Multi-kernel models assessed were genetic main effects plus genetic GxE (G + G&lt;sub&gt;GxE&lt;/sub&gt;, A + A&lt;sub&gt;GxE&lt;/sub&gt;) and genetic main effects plus hyperspectral reflectance GxE (&lt;em&gt;e.g.&lt;/em&gt;, G + H.VEG&lt;sub&gt;GxE&lt;/sub&gt;, A + H.ALL&lt;sub&gt;GxE&lt;/sub&gt;, etc.). The color corresponds to the developmental growth stage of the site-year at the time of phenotyping. Error bars are the standard deviation of prediction accuracy for the 20 random partitions.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/4/1231/F8/graphic-11.medium.gif" width="440" height="294"/><noscript><img class="highwire-fragment fragment-image" alt="" src="https://www.g3journal.org/content/ggg/9/4/1231/F8/graphic-11.medium.gif" width="440" height="294"/></noscript></span></a><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/4/1231/F8/graphic-11.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/4/1231/F8/graphic-11.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492744" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div></div><div class="fig-caption"><span class="fig-label">Figure 8</span> <p id="p-59" class="first-child">Across breeding cycles/within managed treatment prediction accuracies, with and without correction for DTHD. Accuracy is expressed as the average Pearson’s correlation between predictions and observed BLUPs for GY across 20 random TRN-TST partitions. In each partition, the TRN set consisted of all records from three out of the four of the breeding cycles for the managed treatment plus 20% of records from the fourth breeding cycle. Single-kernel models tested were genetic main effects only (G or A) and hyperspectral reflectance main effects (H.VEG, H.HEAD, H.GF, H.ALL). Multi-kernel models assessed were genetic main effects plus genetic GxE (G + G<sub>GxE</sub>, A + A<sub>GxE</sub>) and genetic main effects plus hyperspectral reflectance GxE (<em>e.g.</em>, G + H.VEG<sub>GxE</sub>, A + H.ALL<sub>GxE</sub>, etc.). The color corresponds to the developmental growth stage of the site-year at the time of phenotyping. Error bars are the standard deviation of prediction accuracy for the 20 random partitions.</p><div class="sb-div caption-clear"></div></div></div><p id="p-60">For the single-kernel hyperspectral main effects models, <strong><em>H.GF</em></strong> in 2013-14 Heat and all single-kernel <strong><em>H</em></strong> models for 2013-14 Optimal Flat showed prediction accuracies close to zero. The 2013-14 Heat site-year was phenotyped once during the GF late in the growing season (21 May), which was close to maturity. <a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a> shows that the correlation between hyperspectral reflectance and GY for that time-point is close to zero for each of the 62 wavelengths. To consider average trends, these were removed from the analysis. Mean accuracies of the <strong><em>H.VEG</em></strong>, <strong><em>H.HEAD</em></strong>, <strong><em>H.GF</em></strong>, and <strong><em>H.ALL</em></strong> models were 0.37, 0.45, 0.45, and 0.46, respectively (<a id="xref-fig-8-2" class="xref-fig" href="#F8">Figure 8A</a>). The accuracies of the <strong><em>G</em></strong> and <strong><em>A</em></strong> models were slightly lower, recording averages of 0.25 and 0.28, respectively. When correcting for DTHD, the mean accuracies of <strong><em>H.VEG</em></strong>, <strong><em>H.HEAD</em></strong>, <strong><em>H.GF</em></strong>, and <strong><em>H.ALL</em></strong> models were 0.28, 0.34, 0.32, and 0.34, respectively (<a id="xref-fig-8-3" class="xref-fig" href="#F8">Figure 8B</a>). While closer in performance to <strong><em>G</em></strong> and <strong><em>A</em></strong>, the single-kernel hyperspectral main effects models still conferred an advantage in some site-years.</p><p id="p-61">Expanding the <strong><em>G</em></strong> and <strong><em>A</em></strong> models to account for the <span class="inline-formula" id="inline-formula-54"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-61.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-61.gif"/></noscript></span></span> interactions marginally increased accuracies to 0.32 and 0.31 for the <strong><em>G + G<sub>GxE</sub></em></strong> and <strong><em>A + A<sub>GxE</sub></em></strong> models, respectively. As with prediction within breeding cycle/across managed treatments, the highest accuracies were observed for the multi-kernel models that estimated the genetic main effects using markers or pedigrees and the <span class="inline-formula" id="inline-formula-55"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-62.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-62.gif"/></noscript></span></span> interactions using hyperspectral reflectance. Hyperspectral reflectance matrix <strong><em>H.VEG</em></strong> had an average accuracy of 0.39 when integrated with <strong><em>G</em></strong> and 0.41 when integrated with <strong><em>A</em></strong>. <strong><em>H.HEAD</em></strong>, <strong><em>H.GF</em></strong>, and <strong><em>H.ALL</em></strong> had average accuracies of 0.46, 0.46, and 0.48, respectively, when integrated with <strong><em>G</em></strong> or <strong><em>A</em></strong>. As before, for most site-years, these multi-kernel models were observed to have similar levels of accuracy within each site-year, irrespective of which hyperspectral reflectance-derived relationship matrix was used to model the <span class="inline-formula" id="inline-formula-56"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-63.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-63.gif"/></noscript></span></span> interactions or whether markers or pedigrees were used to model the genetic main effects.</p><p id="p-62">When compared to the <strong><em>G + G<sub>GxE</sub></em></strong> and <strong><em>A + A<sub>GxE</sub></em></strong> models, estimating the <span class="inline-formula" id="inline-formula-57"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-64.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-64.gif"/></noscript></span></span> interactions using hyperspectral reflectance increased prediction accuracy by an average of 0.12 and 0.14, respectively, and by an average of 0.07 and 0.08, respectively, when correcting for DTHD. Average improvements over the corresponding single-kernel hyperspectral reflectance main effects models ranged from 0.04 to 0.06 and from 0.06 to 0.08 after correcting for DTHD.</p><p id="p-63">Mean accuracies and standard deviations of all models for the three prediction schemes can be found in Files S1, S2, and S3, respectively.</p></div></div></div><div class="section discussion" id="sec-21"><h2 class="">Discussion</h2><p id="p-64">We proposed a multi-kernel, across-environment GBLUP model that uses relationship matrices derived from genomic markers, pedigrees, and aerial hyperspectral reflectance phenotypes to estimate the genetic main effects and the <span class="inline-formula" id="inline-formula-58"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-65.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-65.gif"/></noscript></span></span> interactions in the context of a wheat breeding program. Our study found that deriving a relationship matrix from high-dimensional hyperspectral reflectance phenotypes - as if they were genomic markers - for use in GBLUP can be an effective approach for predicting GY in wheat and in many situations resulted in predictions accuracies that were equivalent or superior to the use of genomic markers or pedigrees. This is consistent with similar studies that used linear and non-linear modeling approaches including ordinary least squares, partial least squares, Bayesian shrinkage, and functional regression methods to predict grain in maize and wheat with hyperspectral reflectance data (<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Aguate <em>et al.</em> 2017</a>; <a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Montesinos-López <em>et al.</em> 2017a</a>).</p><p id="p-65">The heritabilities of the hyperspectral wavelengths and their correlations with GY were not homogeneous across time-points, growth stages, breeding cycles, or managed treatments. Wavelengths between 398 and 425 nm were observed to have the lowest heritabilities while those below 500 nm tended to have the weakest correlations with GY. These results are consistent with a similar study of canopy reflectance in wheat (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Hansen and Schjoerring 2003</a>) that found low signal-to-noise between 400 and 438 nm. According to <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Mahlein <em>et al.</em> (2015)</a>, the hyperspectral imaging systems most commonly used in agriculture have poor sensitivity to reflectance in the blue region of the light spectrum (400-500 nm). Despite this, <a id="xref-ref-34-3" class="xref-bibr" href="#ref-34">Montesinos-López <em>et al.</em> (2017a)</a> showed that removing wavelengths with low heritability did not improve prediction accuracies when using hyperspectral reflectance to predict GY in wheat. Based on this result, we used all hyperspectral wavelengths to build relationship matrices for prediction. However, the blue region of the light spectrum contains important information on the optical properties of plants, including the absorbance maxima of chlorophyll a, chlorophyll b, and carotenoids (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Lichtenthaler 1987</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Horton <em>et al.</em> 1996</a>). Further advancements in the ability of hyperspectral imaging to accurately record reflectance in the blue region may potentially improve prediction accuracies for grain yield.</p><p id="p-66">When predicting line performance within a site-year of interest, most site-years recorded at least one individual hyperspectral time-point with prediction accuracy equivalent or superior to prediction with markers or pedigrees, suggesting that hyperspectral reflectance phenotypes may be a useful alternative for generating predictions when markers and pedigree are not available. We also found that combining multiple time-points into hyperspectral BLUEs for a given growth stage generally did not improve accuracy beyond the average accuracy across the individual time-points. In practice, however, GY data would not be available to inform the identification of the most predictive time-points. Therefore, phenotyping at multiple time-points throughout the growing season to develop multi-time-point BLUEs may be an effective approach to buffer against time-points with low prediction accuracy.</p><p id="p-67">It is also possible that some information pertaining to plant growth and development is lost when integrating multiple time-points using means-based approaches. An alternative that may more fully capture temporal variations could be to model a growth curve based on reflectance data for each line in each site-year, as in <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Verhulst <em>et al.</em> (2011)</a>. A greater number of time-points than were measured in this study may be required to accurately develop growth curves. In addition, statistical methods that can integrate multiple curves from a range of hyperspectral bands may be needed. Further research in this area should be performed to compare prediction accuracies of hyperspectral reflectance-based growth curves <em>vs.</em> those achieved in this study.</p><p id="p-68">Correcting results for DTHD reduced accuracies when hyperspectral reflectance-derived relationship matrices were used, but not for marker or pedigree matrices. These results suggest that the hyperspectral reflectance measurements are also capturing physiological parameters associated with relative maturity. The greatest reductions in accuracy were observed in the Drought and Severe Drought managed treatments, where GY is typically associated with earliness. This is consistent with <a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Rutkoski <em>et al.</em> (2016)</a>, which found that correcting for DTHD in wheat GY reduced the prediction accuracies of multivariate GS models integrating NDVI and canopy temperature measurements with markers and pedigrees. The CIMMYT bread wheat breeding program maintains a high level of diversity for DTHD in germplasm development due to its wide target of geographic regions. For breeding programs with high levels of variation for DTHD, it may be advisable to perform a correction for DTHD when predicting GY using hyperspectral reflectance so as to avoid indirect selection on relative maturity.</p><p id="p-69">To test our proposed prediction approaches in a multi-environment context, we developed two prediction schemes. In the first, predictions were performed within a breeding cycle for genotypes that have been evaluated under some treatments but not others. In the second, predictions were performed across breeding cycles onto genotypes that were not previously evaluated. Overall, prediction accuracies in the second scheme were lower than for the corresponding models in the first, which is consistent with previous studies showing that predicting the performance of newly developed lines is more challenging than the prediction of lines that have been evaluated in correlated environments (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Burgueño <em>et al.</em> 2012</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Crossa <em>et al.</em> 2014</a>).</p><p id="p-70">If markers or pedigrees are not available, our results showed that predicting GY with hyperspectral reflectance-derived relationship matrices alone in single-kernel models could provide similar results in terms of accuracy. It should be noted that this study was conducted at the yield trial stage of the breeding program when families contain fewer full-sibs and the variance due to Mendelian sampling is low. As pedigrees do not account for Mendelian sampling, it is possible that prediction with hyperspectral reflectance may be more advantageous than pedigree-based GS at earlier stages of the breeding program in which families contain greater numbers of full-sibs, though further research is needed to assess the ability of hyperspectral reflectance to distinguish within-family variation.</p><p id="p-71">The optimal prediction accuracies were achieved by building combined models that used markers or pedigrees to model the genetic main effects and hyperspectral reflectance to model the <span class="inline-formula" id="inline-formula-59"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-66.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-66.gif"/></noscript></span></span> interactions. However, in prediction across breeding cycles within a managed treatment, the improvements in accuracy with the addition of markers or pedigrees were marginal when compared to the corresponding single-kernel hyperspectral reflectance models. This result was similar to <a id="xref-ref-35-3" class="xref-bibr" href="#ref-35">Montesinos-López <em>et al.</em> (2017b)</a>, which found that the addition of markers or pedigrees did not significantly improve accuracy. The modest increases observed when predicting across breeding cycles within a managed treatment may not justify the added investment of marker genotyping.</p><p id="p-72">When considering the optimal developmental growth stages during which to collect hyperspectral reflectance data, there were some differences in accuracy among growth stages when predicting with single hyperspectral phenotyping time-points within a site-year. Overall, the time-points from the HEAD and GF stages showed slightly higher accuracies than those collected during the VEG stage. However, when predicting across site-years, the advantage of phenotyping later in the season was reduced, particularly for the multi-kernel models that combined markers or pedigrees with hyperspectral reflectance-derived relationship matrices, where most often there was no considerable difference in accuracy among the different growth stages. <a id="xref-ref-34-4" class="xref-bibr" href="#ref-34">Montesinos-López <em>et al.</em> (2017a)</a> observed clear optimal time-points for hyperspectral phenotyping in a multi-environment context. However, their study represented a balanced scenario in which all lines in all environments were phenotyped an equal number of times, and individual phenotyping time-points could be used to predict across environments. While classifying time-points according to the predominant growth stage at the time of phenotyping represents a simple and efficient method for predicting across site-years, it is somewhat difficult to compare results between growth stages. The numbers of time-points observed for each growth stage were not consistent within and across site-years. In addition, the growth stage classifications used were dependent on the predominant growth stage of the site-year and did not reflect the variation in phenology within the site-year at the time of phenotyping. These challenges may warrant further investigation into multi-environment prediction when HTP datasets are unbalanced in the number of time-points observed and at which stage of crop development those observations were recorded.</p><p id="p-73">While our results suggest that hyperspectral reflectance data have the potential to add value to a breeding program by providing accurate GY predictions, the data used in this study were collected on large plot sizes that are suitable for measuring GY <em>per se</em>. While our approach may provide breeding programs with GY predictions earlier in the growing season, enabling more efficient allocation of resources at harvest, a potentially greater benefit could come from utilizing this method for smaller plot sizes where there is limited seed available to replicate and reliably assess yield. To address this, we are currently evaluating the use of aerial HTP at the early generation stage when measuring GY is not feasible.</p><p id="p-74">The integration of highly dimensional data, such as hyperspectral reflectance, into relationship matrices for use in GBLUP has several advantages. First, there is no additional development in statistical software required to implement the proposed models. Many options for fitting GBLUP such as “rrBLUP”, “GAPIT”, and “BGLR” are currently available and are being increasingly used. The models proposed here can be readily implemented in these or other existing software without requiring additional user training. In addition, with the development of improved hyperspectral sensors that register reflectance at wavelengths ranging from 400 to 2500 nm, it is likely that the number of data points available for prediction will continue to increase. By using the relationship matrix-based approach to GBLUP that we have proposed, these increases in data dimensionality will neither influence the complexity of the GBLUP model, nor will they increase the computation time of the model itself. This approach could also be useful for integrating different types of highly dimensional phenotypes for prediction, such as ionomics and metabolomics data (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Riedelsheimer <em>et al.</em> 2012</a>). <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Rincent <em>et al.</em> (2018)</a> recently showed that relationship matrices derived from near infrared spectroscopy absorbance of winter wheat grains and powdered leaf tissue between 400 and 2500 nm provided yield prediction accuracies that were superior to marker-based GBLUP approaches. Further research should be performed to assess the potential for this relationship matrix-based approach to be applied to other forms of biological data.</p><div id="sec-22" class="subsection"><h3>Conclusion</h3><p id="p-75">In this study, we have proposed a multi-kernel GBLUP model that uses genomic marker-, pedigree-, and hyperspectral reflectance-derived relationships matrices to model the genetic main effects and <span class="inline-formula" id="inline-formula-60"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-67.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1231/embed/mml-math-67.gif"/></noscript></span></span> interactions across environments within a bread wheat breeding program. We have shown that deriving relationship matrices from aerial hyperspectral reflectance phenotypes can effectively predict GY in wheat within and across managed treatments and breeding cycles. Accuracies when testing single-kernel models using hyperspectral reflectance data alone are similar to those achieved with markers or pedigrees. Our results also show that combining markers/pedigrees with hyperspectral reflectance data in multi-kernel models can increase accuracies over single-kernel approaches, but in some prediction scenarios, these increases were modest. We also suggested a method for addressing the issue of unbalanced HTP datasets involving the classification of time-points according to the predominant developmental growth stage observed at the time of phenotyping. Accuracies of multi-kernel models were roughly equivalent irrespective of the growth stage in which hyperspectral phenotyping was performed. Further research on how best to leverage multi-temporal phenotypes when the amount of data differs across site-years is needed. The methods we have proposed provide a simple and computationally efficient approach for integrating highly dimensional aerial HTP information into genomic selection and should be tested on other forms of high dimensional biological data.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-76">We are thankful to the National Science Graduate Research Fellowship (Grant No. DGE-1650441) and the U.S. Borlaug Fellowship in Global Food Security for supporting the graduate studies of Margaret R. Krause. The work presented here was supported in part by the Delivering Genetic Gain in Wheat project, supported by aid from the U.K. Government’s Department of International Development (DFID) and the Bill &amp; Melinda Gates Foundation (OPP113319). Partial support was also provided by the Agriculture and Food Research Initiative Competitive Grants 2011-68002-30029 (Triticeae-CAP) and 2017-67007-25939 (Wheat-CAP) from the USDA National Institute of Food and Agriculture. Finally, this work was also made possible through support provided by Feed the Future through the U.S. Agency for International Development, under the terms of Contract No. AID-OAA-A-13-0005. The opinions expressed herein are those of the authors and do not necessarily reflect the views of the U.S. Agency for International Development.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-2"><p id="p-77">Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.7653473">https://doi.org/10.25387/g3.7653473</a>.</p></li><li class="fn" id="fn-3"><p id="p-78"><em>Communicating editor: R. Wisser</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2018-11-22"><span class="received-label">Received </span>November 22, 2018.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2019-02-15"><span class="accepted-label">Accepted </span>February 15, 2019.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2019 Krause <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1231.1" data-doi="10.2135/cropsci2017.01.0007."><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Aguate</span>  <span class="cit-name-given-names">F. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trachsel</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">González-Pérez</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burgueño</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Use of hyperspectral image data outperforms vegetation indices in prediction of maize yield.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">57</span>: <span class="cit-fpage">2517</span>–<span class="cit-lpage">2524</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2017.01.0007</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D57%26rft.spage%253D2517%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2017.01.0007.%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2017.01.0007.&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1231.2" data-doi="10.1071/FP13126"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Andrade-Sanchez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gore</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heun</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thorp</span>  <span class="cit-name-given-names">K. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carmo-Silva</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Development and evaluation of a field-based high-throughput phenotyping platform.</span> <abbr class="cit-jnl-abbrev">Funct. Plant Biol.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">68</span>–<span class="cit-lpage">79</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1071/FP13126</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFunct.%2BPlant%2BBiol.%26rft.volume%253D41%26rft.spage%253D68%26rft_id%253Dinfo%253Adoi%252F10.1071%252FFP13126%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1071/FP13126&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1231.3" data-doi="10.2134/agronj2000.92183x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Aparicio</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Villegas</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Casadesus</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Araus</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Royo</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Spectral vegetation indices as nondestructive tools for determining durum wheat yield.</span> <abbr class="cit-jnl-abbrev">Agron. J.</abbr> <span class="cit-vol">92</span>: <span class="cit-fpage">83</span>–<span class="cit-lpage">91</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2134/agronj2000.92183x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAgronomy%2BJournal%26rft.stitle%253DAgron.%2BJ.%26rft.aulast%253DAparicio%26rft.auinit1%253DN.%26rft.volume%253D92%26rft.issue%253D1%26rft.spage%253D83%26rft.epage%253D91%26rft.atitle%253DSpectral%2BVegetation%2BIndices%2Bas%2BNondestructive%2BTools%2Bfor%2BDetermining%2BDurum%2BWheat%2BYield%26rft_id%253Dinfo%253Adoi%252F10.2134%252Fagronj2000.92183x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2134/agronj2000.92183x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000089407800014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1231.4" data-doi="10.1016/j.tplants.2013.09.008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Araus</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cairns</span>  <span class="cit-name-given-names">J. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Field high-throughput phenotyping: the new crop breeding frontier.</span> <abbr class="cit-jnl-abbrev">Trends Plant Sci.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">52</span>–<span class="cit-lpage">61</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.tplants.2013.09.008</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BPlant%2BSci.%26rft.volume%253D19%26rft.spage%253D52%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tplants.2013.09.008%26rft_id%253Dinfo%253Apmid%252F24139902%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tplants.2013.09.008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24139902&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1231.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000330013700008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-book" id="cit-9.4.1231.5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bernardo</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-source">Breeding for quantitative traits in plants</span>, <span class="cit-publ-name">Stemma Press</span>, <span class="cit-publ-loc">Woodbury, Minnesota</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1231.6" data-doi="10.2135/cropsci2011.06.0299"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Burgueño</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weigel</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genomic prediction of breeding values when modeling genotype × environment interaction using pedigree and dense molecular markers.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">707</span>–<span class="cit-lpage">719</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2011.06.0299</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D52%26rft.spage%253D707%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2011.06.0299%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2011.06.0299&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1231.7" data-doi="10.1111/j.1744-7909.2012.01116.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cabrera-Bosquet</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">von Zitzewitz</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Serret</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Araus</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">High-throughput phenotyping and genomic selection: the frontiers of crop breeding coverage.</span> <abbr class="cit-jnl-abbrev">J. Integr. Plant Biol.</abbr> <span class="cit-vol">54</span>: <span class="cit-fpage">312</span>–<span class="cit-lpage">320</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1744-7909.2012.01116.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DJ%2BIntegr%2BPlant%2BBiol%26rft.aulast%253DCabrera-Bosquet%26rft.auinit1%253DL.%26rft.volume%253D54%26rft.issue%253D5%26rft.spage%253D312%26rft.epage%253D320%26rft.atitle%253DHigh-throughput%2Bphenotyping%2Band%2Bgenomic%2Bselection%253A%2Bthe%2Bfrontiers%2Bof%2Bcrop%2Bbreeding%2Bconverge.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1744-7909.2012.01116.x%26rft_id%253Dinfo%253Apmid%252F22420640%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1744-7909.2012.01116.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22420640&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1231.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1231.8" data-doi="10.2135/cropsci2015.05.0290"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crain</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wei</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barker</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alderman</span>  <span class="cit-name-given-names">P. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Development and deployment of a portable field phenotyping platform.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">56</span>: <span class="cit-fpage">965</span>–<span class="cit-lpage">975</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2015.05.0290</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D56%26rft.spage%253D965%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2015.05.0290%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2015.05.0290&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1231.9" data-doi="10.3835/plantgenome2017.05.0043"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crain</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mondal</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rutkoski</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Singh</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Combining high-throughput phenotyping and genomic information to increase prediction and selection accuracy in wheat breeding.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">11</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2017.05.0043</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2017.05.0043%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2017.05.0043&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1231.10" data-doi="10.1038/hdy.2013.16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pérez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hickey</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burgueño</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ornella</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genomic prediction in CIMMYT maize and wheat breeding programs.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">112</span>: <span class="cit-fpage">48</span>–<span class="cit-lpage">60</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/hdy.2013.16</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D112%26rft.spage%253D48%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2013.16%26rft_id%253Dinfo%253Apmid%252F23572121%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2013.16&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23572121&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1231.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1231.11" data-doi="10.1016/j.tplants.2017.08.011"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pérez-Rodríguez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cuevas</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Montesinos-López</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jarquín</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Genomic selection in plant breeding: methods, models, and perspectives.</span> <abbr class="cit-jnl-abbrev">Trends Plant Sci.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">961</span>–<span class="cit-lpage">975</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.tplants.2017.08.011</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BPlant%2BSci.%26rft.volume%253D22%26rft.spage%253D961%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tplants.2017.08.011%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tplants.2017.08.011&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1231.12" data-doi="10.1534/genetics.112.147983"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Daetwyler</span>  <span class="cit-name-given-names">H. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Calus</span>  <span class="cit-name-given-names">M. P. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pong-Wong</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hickey</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genomic prediction in animals and plants: simulation of data, validation, reporting, and benchmarking.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">193</span>: <span class="cit-fpage">347</span>–<span class="cit-lpage">365</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.112.147983</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.147983%26rft_id%253Dinfo%253Apmid%252F23222650%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkzLzIvMzQ3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS80LzEyMzEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-web" id="cit-9.4.1231.13"><div class="cit-metadata unstructured">de los Campos, G., and P. Pérez-Rodríguez, 2014 Bayesian generalized linear regression. R package version 1.0.5. URL <a href="https://CRAN.R-project.org/package=BGLR">https://CRAN.R-project.org/package=BGLR</a>.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1231.14" data-doi="10.1371/journal.pone.0019379"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Elshire</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glaubitz</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kawamoto</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e19379</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0019379</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DElshire%26rft.auinit1%253DR.%2BJ.%26rft.volume%253D6%26rft.issue%253D5%26rft.spage%253De19379%26rft.epage%253De19379%26rft.atitle%253DA%2Brobust%252C%2Bsimple%2Bgenotyping-by-sequencing%2B%2528GBS%2529%2Bapproach%2Bfor%2Bhigh%2Bdiversity%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0019379%26rft_id%253Dinfo%253Apmid%252F21573248%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0019379&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21573248&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1231.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1231.15" data-doi="10.3835/plantgenome2011.08.0024"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Endelman</span>  <span class="cit-name-given-names">J. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Ridge regression and other kernels for genomic selection with R package rrBLUP.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">250</span>–<span class="cit-lpage">255</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2011.08.0024</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D4%26rft.spage%253D250%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2011.08.0024%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2011.08.0024&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1231.16" data-doi="10.1534/g3.112.004259"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Endelman</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Shrinkage estimation of the realized relationship matrix. G3 Genes Genom.</span> <abbr class="cit-jnl-abbrev">Genet.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">1405</span>–<span class="cit-lpage">1413</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.112.004259</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%26rft.volume%253D2%26rft.spage%253D1405%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.112.004259%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/g3.112.004259&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1231.17" data-doi="10.1016/j.tplants.2011.09.005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Furbank</span>  <span class="cit-name-given-names">R. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tester</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Phenomics – technologies to relieve the phenotyping bottleneck.</span> <abbr class="cit-jnl-abbrev">Trends Plant Sci.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">635</span>–<span class="cit-lpage">644</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.tplants.2011.09.005</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bplant%2Bscience%26rft.stitle%253DTrends%2BPlant%2BSci%26rft.aulast%253DFurbank%26rft.auinit1%253DR.%2BT.%26rft.volume%253D16%26rft.issue%253D12%26rft.spage%253D635%26rft.epage%253D644%26rft.atitle%253DPhenomics--technologies%2Bto%2Brelieve%2Bthe%2Bphenotyping%2Bbottleneck.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tplants.2011.09.005%26rft_id%253Dinfo%253Apmid%252F22074787%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tplants.2011.09.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22074787&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1231.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000298724900001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-book" id="cit-9.4.1231.18"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gilmour</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gogel</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cullis</span>  <span class="cit-name-given-names">B. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Welham</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-source">ASReml user guide release 4.1 structural specification</span>, <span class="cit-publ-name">VSN International</span>, <span class="cit-publ-loc">Hemel Hempstead, UK</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1231.19" data-doi="10.1080/014311699212524"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Govaerts</span>  <span class="cit-name-given-names">Y. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Verstraete</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pinty</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gobron</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Designing optimal spectral indices: a feasibility and proof of concept study.</span> <abbr class="cit-jnl-abbrev">Int. J. Remote Sens.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">1853</span>–<span class="cit-lpage">1873</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1080/014311699212524</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInt.%2BJ.%2BRemote%2BSens.%26rft.volume%253D20%26rft.spage%253D1853%26rft_id%253Dinfo%253Adoi%252F10.1080%252F014311699212524%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/014311699212524&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-book" id="cit-9.4.1231.20"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gueymard</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-source">A simple model of the atmospheric radiative transfer of sunshine: algorithms and performance assessment</span>, <span class="cit-publ-name">Florida Solar Energy Center</span>, <span class="cit-publ-loc">Cocoa, FL</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1231.21" data-doi="10.1016/j.energy.2004.04.032"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gueymard</span>  <span class="cit-name-given-names">C. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Interdisciplinary applications of a versatile spectral solar irradiance model: a review.</span> <abbr class="cit-jnl-abbrev">Energy</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">1551</span>–<span class="cit-lpage">1576</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.energy.2004.04.032</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEnergy%26rft.volume%253D30%26rft.spage%253D1551%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.energy.2004.04.032%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.energy.2004.04.032&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1231.22" data-doi="10.1186/s13007-016-0134-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Haghighattalab</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">González-Pérez</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mondal</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Singh</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schinstock</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Application of unmanned aerial systems for high throughput phenotyping of large wheat breeding nurseries.</span> <abbr class="cit-jnl-abbrev">Plant Methods</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">35</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s13007-016-0134-6</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BMethods%26rft.volume%253D12%26rft.spage%253D35%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13007-016-0134-6%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13007-016-0134-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1231.23" data-doi="10.1016/S0034-4257(03)00131-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hansen</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schjoerring</span>  <span class="cit-name-given-names">J. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Reflectance measurement of canopy biomass and nitrogen status in wheat crops using normalized difference vegetation indices and partial least squares regression.</span> <abbr class="cit-jnl-abbrev">Remote Sens. Environ.</abbr> <span class="cit-vol">86</span>: <span class="cit-fpage">542</span>–<span class="cit-lpage">553</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/S0034-4257(03)00131-7</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRemote%2BSens.%2BEnviron.%26rft.volume%253D86%26rft.spage%253D542%26rft.atitle%253DREFLECTANCE%2BMEASUREMENT%2BOF%2BCANOPY%2BBIOMASS%2BAND%2BNITROGEN%2BSTATUS%2BIN%2BWHEAT%2BCROPS%2BUSING%2BNORMALIZED%2BDIFFERENCE%2BVEGETATION%2BINDICES%2BAND%2BPARTIAL%2BLEAST%2BSQUARES%2BREGRESSION%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0034-4257%252803%252900131-7%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0034-4257(03)00131-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1231.24" data-doi="10.2135/cropsci2012.07.0420"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heslot</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Using genomic prediction to characterize environments and optimize prediction accuracy in applied breeding data.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">53</span>: <span class="cit-fpage">921</span>–<span class="cit-lpage">933</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2012.07.0420</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D53%26rft.spage%253D921%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2012.07.0420%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2012.07.0420&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1231.25" data-doi="10.1146/annurev.arplant.47.1.655"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Horton</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruban</span>  <span class="cit-name-given-names">A. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Walters</span>  <span class="cit-name-given-names">R. G.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Regulation of light harvesting in green plants.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Plant Physiol. Plant Mol. Biol.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">655</span>–<span class="cit-lpage">684</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1146/annurev.arplant.47.1.655</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bplant%2Bbiology%26rft.stitle%253DAnnu%2BRev%2BPlant%2BBiol%26rft.aulast%253DHorton%26rft.auinit1%253DP.%26rft.volume%253D47%26rft.issue%253D1%26rft.spage%253D655%26rft.epage%253D684%26rft.atitle%253DREGULATION%2BOF%2BLIGHT%2BHARVESTING%2BIN%2BGREEN%2BPLANTS%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.arplant.47.1.655%26rft_id%253Dinfo%253Apmid%252F15012304%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.arplant.47.1.655&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15012304&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1231.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1996UT11900025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1231.26" data-doi="10.1007/s00122-013-2243-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jarquín</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lacaze</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Du Cheyron</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daucourt</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A reaction norm model for genomic selection using high-dimensional genomic and environmental data.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">127</span>: <span class="cit-fpage">595</span>–<span class="cit-lpage">607</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-013-2243-1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D127%26rft.spage%253D595%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-013-2243-1%26rft_id%253Dinfo%253Apmid%252F24337101%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-013-2243-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24337101&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1231.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1231.27" data-doi="10.1080/01431160110107653"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Labus</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lawrence</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Engel</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Long</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Wheat yield estimates using multi-temporal NDVI satellite imagery.</span> <abbr class="cit-jnl-abbrev">Int. J. Remote Sens.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">4169</span>–<span class="cit-lpage">4180</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1080/01431160110107653</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInt.%2BJ.%2BRemote%2BSens.%26rft.volume%253D23%26rft.spage%253D4169%26rft_id%253Dinfo%253Adoi%252F10.1080%252F01431160110107653%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/01431160110107653&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000178337700004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1231.28" data-doi="10.3390/s141120078"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A review of imaging techniques for plant phenotyping.</span> <abbr class="cit-jnl-abbrev">Sensors (Basel)</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">20078</span>–<span class="cit-lpage">20111</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3390/s141120078</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSensors%2B%2528Basel%2529%26rft.volume%253D14%26rft.spage%253D20078%26rft_id%253Dinfo%253Adoi%252F10.3390%252Fs141120078%26rft_id%253Dinfo%253Apmid%252F25347588%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3390/s141120078&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25347588&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1231.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1231.29" data-doi="10.1016/0076-6879(87)48036-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lichtenthaler</span>  <span class="cit-name-given-names">H. K.</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span> <span class="cit-article-title">Chlorophylls and carotenoids: pigments of photosynthetic biomembranes.</span> <abbr class="cit-jnl-abbrev">Methods Enzymol.</abbr> <span class="cit-vol">148</span>: <span class="cit-fpage">350</span>–<span class="cit-lpage">382</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/0076-6879(87)48036-1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2BEnzymol.%26rft.volume%253D148%26rft.spage%253D350%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0076-6879%252887%252948036-1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0076-6879(87)48036-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1987M168200034&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1231.30" data-doi="10.1534/g3.114.016097"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">López-Cruz</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bonnett</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dreisigacker</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Increased prediction accuracy in wheat breeding trials using markers × environment interaction genomic selection model. G3 Genes Genom.</span> <abbr class="cit-jnl-abbrev">Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">569</span>–<span class="cit-lpage">582</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.114.016097</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%26rft.volume%253D5%26rft.spage%253D569%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.114.016097%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/g3.114.016097&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1231.31" data-doi="10.3390/s150612834"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mahlein</span>  <span class="cit-name-given-names">A. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hammersley</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oerke</span>  <span class="cit-name-given-names">E. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dehne</span>  <span class="cit-name-given-names">H. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goldbach</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Supplemental blue LED lighting array to improve signal quality in hyperspectral imaging of plants.</span> <abbr class="cit-jnl-abbrev">Sensors (Basel)</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">12834</span>–<span class="cit-lpage">12840</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3390/s150612834</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSensors%2B%2528Basel%2529%26rft.volume%253D15%26rft.spage%253D12834%26rft_id%253Dinfo%253Adoi%252F10.3390%252Fs150612834%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3390/s150612834&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-book" id="cit-9.4.1231.32"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McLaren</span>  <span class="cit-name-given-names">C. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ramos</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lopez</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eusebio</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Applications of the genealogy management system</span>, pp. <span class="cit-fpage">5.8</span>–<span class="cit-lpage">5.13</span> in <span class="cit-source"><em>International Crop Information System. Technical Development Manual</em>, <em>version VI</em></span>, edited by <span class="cit-ed"><span class="cit-name-surname">McLaren</span>  <span class="cit-name-given-names">C. G.</span></span>, <span class="cit-ed"><span class="cit-name-surname">White</span>  <span class="cit-name-given-names">J. W.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Fox</span>  <span class="cit-name-given-names">P. N.</span></span>. <span class="cit-publ-name">CIMMyT and IRRI</span>, <span class="cit-publ-loc">México</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1231.33" data-doi="10.1093/nar/gkv1344"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>  <span class="cit-name-given-names">T. H. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Prediction of total genetic value using genome-wide dense marker maps.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">157</span>: <span class="cit-fpage">1819</span>–<span class="cit-lpage">1829</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMeuwissen%26rft.auinit1%253DT.%2BH.%2BE.%26rft.volume%253D157%26rft.issue%253D4%26rft.spage%253D1819%26rft.epage%253D1829%26rft.atitle%253DPrediction%2Bof%2BTotal%2BGenetic%2BValue%2BUsing%2BGenome-Wide%2BDense%2BMarker%2BMaps%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkv1344%26rft_id%253Dinfo%253Apmid%252F11290733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1Ny80LzE4MTkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzQvMTIzMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1231.34" data-doi="10.1186/s13007-016-0154-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Montesinos-López</span>  <span class="cit-name-given-names">O. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Montesinos-López</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alvarado</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span><span class="cit-comment">a</span> <span class="cit-article-title">Predicting grain yield using canopy hyperspectral reflectance in wheat breeding data.</span> <abbr class="cit-jnl-abbrev">Plant Methods</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">4</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s13007-016-0154-2</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BMethods%26rft.volume%253D13%26rft.spage%253D4%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13007-016-0154-2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13007-016-0154-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1231.35" data-doi="10.1186/s13007-017-0212-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Montesinos-López</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Montesinos-López</span>  <span class="cit-name-given-names">O. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cuevas</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mata-López</span>  <span class="cit-name-given-names">W. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burgueño</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span><span class="cit-comment">b</span> <span class="cit-article-title">Genomic Bayesian functional regression models with interactions for predicting wheat grain yield using hyper-spectral image data.</span> <abbr class="cit-jnl-abbrev">Plant Methods</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">62</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s13007-017-0212-4</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BMethods%26rft.volume%253D13%26rft.spage%253D62%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13007-017-0212-4%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13007-017-0212-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1231.36" data-doi="10.1104/pp.16.00592"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pauli</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chapman</span>  <span class="cit-name-given-names">S. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bart</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Topp</span>  <span class="cit-name-given-names">C. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lawrence-Dill</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">The quest for understanding phenotypic variation via integrated approaches in the field environment.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">172</span>: <span class="cit-fpage">622</span>–<span class="cit-lpage">634</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.16.00592%26rft_id%253Dinfo%253Apmid%252F27482076%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czo5OiIxNzIvMi82MjIiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzQvMTIzMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1231.37" data-doi="10.3835/plantgenome2016.09.0089"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pérez-Rodríguez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rutkoski</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Singh</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Single-step genomic and pedigree genotype × environment interaction models for predicting wheat lines in international environments.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">0</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2016.09.0089</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D10%26rft.spage%253D2%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2016.09.0089%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2016.09.0089&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1231.38" data-doi="10.3835/plantgenome2012.06.0006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Endelman</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dawson</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rutkoski</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genomic selection in wheat breeding using genotyping-by-sequencing.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">103</span>–<span class="cit-lpage">113</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2012.06.0006</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D5%26rft.spage%253D103%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2012.06.0006%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2012.06.0006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-web" id="cit-9.4.1231.39"><div class="cit-metadata unstructured">R Core Team, 2018 R: A language and environment for statistical computing. R Foundation for Statistical Computing, Vienna, Austria. URL <a href="https://www.R-project.org/">https://www.R-project.org/</a>.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1231.40" data-doi="10.1038/ng.1033"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Riedelsheimer</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Czedik-Eysenberg</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grieder</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lisec</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Technow</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genomic and metabolic prediction of complex heterotic traits in hybrid maize.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">217</span>–<span class="cit-lpage">220</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.1033</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DRiedelsheimer%26rft.auinit1%253DC.%26rft.volume%253D44%26rft.issue%253D2%26rft.spage%253D217%26rft.epage%253D220%26rft.atitle%253DGenomic%2Band%2Bmetabolic%2Bprediction%2Bof%2Bcomplex%2Bheterotic%2Btraits%2Bin%2Bhybrid%2Bmaize.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.1033%26rft_id%253Dinfo%253Apmid%252F22246502%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.1033&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22246502&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1231.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1231.41" data-doi="10.1534/g3.118.200760"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rincent</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Charpentier</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Faivre-Rampant</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paux</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Le Gouis</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Phenomic selection is a low-cost and high-throughput method based on indirect predictions: proof of concept on wheat and poplar. G3 Genes Genom.</span> <abbr class="cit-jnl-abbrev">Genet.</abbr> <span class="cit-vol">8</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.118.200760</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.118.200760%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/g3.118.200760&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1231.42" data-doi="10.3390/rs10060930"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rodrigues</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blasch</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Defourny</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ortiz-Monasterio</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schulthess</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Multi-temporal and spectral analysis of high-resolution hyperspectral airborne imagery for precision agriculture: assessment of wheat grain yield and protein content.</span> <abbr class="cit-jnl-abbrev">Remote Sens.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">930</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3390/rs10060930</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRemote%2BSens.%26rft.volume%253D10%26rft.spage%253D930%26rft_id%253Dinfo%253Adoi%252F10.3390%252Frs10060930%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3390/rs10060930&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1231.43" data-doi="10.1534/g3.116.032888"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rutkoski</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mondal</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Autrique</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">González-Pérez</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Canopy temperature and vegetation indices from high-throughput phenotyping improve accuracy of pedigree and genomic selection for grain yield in wheat. G3 Genes Genom.</span> <abbr class="cit-jnl-abbrev">Genet.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">2799</span>–<span class="cit-lpage">2808</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.116.032888</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%26rft.volume%253D6%26rft.spage%253D2799%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.116.032888%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/g3.116.032888&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1231.44" data-doi="10.3835/plantgenome2016.11.0111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rutkoski</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Multitrait, random regression, or simple repeatability model in high-throughput phenotyping data improve genomic prediction for wheat grain yield.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">10</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2016.11.0111</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D18%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2016.11.0111%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2016.11.0111&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1231.45" data-doi="10.1016/j.fcr.2010.08.012"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Verhulst</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Govaerts</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nelissen</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sayre</span>  <span class="cit-name-given-names">K. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The effect of tillage, crop rotation and residue management on maize and wheat growth and development evaluated with an optical sensor.</span> <abbr class="cit-jnl-abbrev">Field Crops Res.</abbr> <span class="cit-vol">120</span>: <span class="cit-fpage">58</span>–<span class="cit-lpage">67</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.fcr.2010.08.012</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DField%2BCrops%2BRes.%26rft.volume%253D120%26rft.spage%253D58%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.fcr.2010.08.012%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.fcr.2010.08.012&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1231.46" data-doi="10.2134/agronj2004.1139"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Viña</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gitelson</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rundquist</span>  <span class="cit-name-given-names">D. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Keydan</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leavitt</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Monitoring maize (<em>Zea mays</em> L.) phenology with remote sensing.</span> <abbr class="cit-jnl-abbrev">Agron. J.</abbr> <span class="cit-vol">96</span>: <span class="cit-fpage">1139</span>–<span class="cit-lpage">1147</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2134/agronj2004.1139</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAgronomy%2BJournal%26rft.stitle%253DAgron.%2BJ.%26rft.aulast%253DVina%26rft.auinit1%253DA.%26rft.volume%253D96%26rft.issue%253D4%26rft.spage%253D1139%26rft.epage%253D1147%26rft.atitle%253DMonitoring%2BMaize%2B%2528Zea%2Bmays%2BL.%2529%2BPhenology%2Bwith%2BRemote%2BSensing%26rft_id%253Dinfo%253Adoi%252F10.2134%252Fagronj2004.1139%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2134/agronj2004.1139&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000223088900025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1231.47" data-doi="10.1016/j.rse.2011.08.010"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Viña</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gitelson</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nguy-Robertson</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peng</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Comparison of different vegetation indices for the remote assessments of green leaf area index of crops.</span> <abbr class="cit-jnl-abbrev">Remote Sens. Environ.</abbr> <span class="cit-vol">115</span>: <span class="cit-fpage">3468</span>–<span class="cit-lpage">3478</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.rse.2011.08.010</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRemote%2BSens.%2BEnviron.%26rft.volume%253D115%26rft.spage%253D3468%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.rse.2011.08.010%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.rse.2011.08.010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1231.48" data-doi="10.1016/j.fcr.2012.04.003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">White</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Andrade-Sanchez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gore</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bronson</span>  <span class="cit-name-given-names">K. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coffelt</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Field-based phenomics for plant genetics research.</span> <abbr class="cit-jnl-abbrev">Field Crops Res.</abbr> <span class="cit-vol">133</span>: <span class="cit-fpage">101</span>–<span class="cit-lpage">112</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.fcr.2012.04.003</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DField%2BCrops%2BRes.%26rft.volume%253D133%26rft.spage%253D101%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.fcr.2012.04.003%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.fcr.2012.04.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000305497300008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1231.49" data-doi="10.1016/S0034-4257(02)00135-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Friedl</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schaaf</span>  <span class="cit-name-given-names">C. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Strahler</span>  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hodges</span>  <span class="cit-name-given-names">J. C. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Monitoring vegetation phenology using MODIS.</span> <abbr class="cit-jnl-abbrev">Remote Sens. Environ.</abbr> <span class="cit-vol">84</span>: <span class="cit-fpage">471</span>–<span class="cit-lpage">475</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/S0034-4257(02)00135-9</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRemote%2BSens.%2BEnviron.%26rft.volume%253D84%26rft.spage%253D471%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0034-4257%252802%252900135-9%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0034-4257(02)00135-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000181591800011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/9/4/1231.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/4/1211" title="Chromosome-Wide Evolution and Sex Determination in the Three-Sexed Nematode Auanema rhodensis" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/4/1249" title="Random Chromosome Partitioning in the Polyploid Bacterium Thermus thermophilus HB27" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/9/4"><span>Volume 9 Issue 4, April 2019</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/9/4.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/9/4.cover-source.jpg?itok=mcy5SuIK" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 9 (4)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/4/1231" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Dry-mDBz6RWPAnvT8t2ZUhXKJIVZkYgl81CvF0pi4ZI" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/genomic-prediction" class="" data-icon-position="" data-hide-link-title="0">Genomic Prediction</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/9/4/1231" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/9/4/1231" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/9/4/1231" class="active" data-icon-position="" data-hide-link-title="0">Hyperspectral Reflectance-Derived Relationship Matrices for Genomic Prediction of Grain Yield in Wheat</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/492620" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-x0oYXgb_6xA9iyy-xueltYhHlsYpk-3NEObTvehgxZE" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/4/1231.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/9/4/1231" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-I3JG070CYqbm9GAs0YmE28odgzu14iu_zv5YpArfAVM" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/9/4/1231" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/4/1231.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="492620" id="node492620--2" data-pisa="ggg;9/4/1231" data-pisa-master="ggg;g3.118.200856" data-apath="/ggg/9/4/1231.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Hyperspectral Reflectance-Derived Relationship Matrices for Genomic Prediction of Grain Yield in Wheat</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-1154-337X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Margaret R. Krause</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-5840-0803" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Lorena González-Pérez</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0001-9429-5855" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>José Crossa</span>, <span class="highwire-citation-author" data-delta="3">Paulino Pérez-Rodríguez</span>, <span class="highwire-citation-author" data-delta="4">Osval Montesinos-López</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0002-4676-5071 " target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Ravi P. Singh</span>, <span class="highwire-citation-author" data-delta="6">Susanne Dreisigacker</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="7"><a href="http://orcid.org/0000-0002-7856-1399" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jesse Poland</span>, <span class="highwire-citation-author" data-delta="8">Jessica Rutkoski</span>, <span class="highwire-citation-author" data-delta="9">Mark Sorrells</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="10"><a href="http://orcid.org/0000-0001-6896-8024" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Michael A. Gore</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="11"><a href="http://orcid.org/0000-0002-8582-8899" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Suchismita Mondal</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1231-1247;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200856 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node492620--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Margaret R. Krause </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Breeding and Genetics Section, School of Integrative Plant Science, Cornell University, Ithaca, New York, 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Margaret%2BR.%2BKrause%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Krause%20MR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMargaret%2BR.%2BKrause%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-1154-337X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Margaret R. Krause</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Lorena González-Pérez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Global Wheat Program, International Maize and Wheat Improvement Center (CIMMYT), Ciudad de México, 06600, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lorena%2BGonz%C3%A1lez-P%C3%A9rez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gonz%C3%A1lez-P%C3%A9rez%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALorena%2BGonz%25C3%25A1lez-P%25C3%25A9rez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-5840-0803" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Lorena González-Pérez</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">José Crossa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Global Wheat Program, International Maize and Wheat Improvement Center (CIMMYT), Ciudad de México, 06600, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jos%C3%A9%2BCrossa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Crossa%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJos%25C3%25A9%2BCrossa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-9429-5855" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for José Crossa</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Paulino Pérez-Rodríguez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Colegio de Postgraduados, CP 56230, Montecillos, Edo. de México, Mexico,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Paulino%2BP%C3%A9rez-Rodr%C3%ADguez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=P%C3%A9rez-Rodr%C3%ADguez%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APaulino%2BP%25C3%25A9rez-Rodr%25C3%25ADguez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Osval Montesinos-López </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Facultad de Telemática, Universidad de Colima, Colima, Colima, 28040, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Osval%2BMontesinos-L%C3%B3pez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Montesinos-L%C3%B3pez%20O&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AOsval%2BMontesinos-L%25C3%25B3pez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Ravi P. Singh </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Global Wheat Program, International Maize and Wheat Improvement Center (CIMMYT), Ciudad de México, 06600, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ravi%2BP.%2BSingh%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Singh%20RP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARavi%2BP.%2BSingh%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-4676-5071 " target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Ravi P. Singh</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Susanne Dreisigacker </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Global Wheat Program, International Maize and Wheat Improvement Center (CIMMYT), Ciudad de México, 06600, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Susanne%2BDreisigacker%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dreisigacker%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASusanne%2BDreisigacker%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Jesse Poland </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Plant Pathology, Kansas State University, Manhattan, Kansas, 66506</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jesse%2BPoland%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Poland%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJesse%2BPoland%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-7856-1399" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jesse Poland</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Jessica Rutkoski </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>International Rice Research Institute (IRRI), DAPO Box 7777, Metro Manila, 1301, Philippines</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jessica%2BRutkoski%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rutkoski%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJessica%2BRutkoski%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Mark Sorrells </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Breeding and Genetics Section, School of Integrative Plant Science, Cornell University, Ithaca, New York, 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mark%2BSorrells%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sorrells%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMark%2BSorrells%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Michael A. Gore </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Breeding and Genetics Section, School of Integrative Plant Science, Cornell University, Ithaca, New York, 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BA.%2BGore%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gore%20MA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMichael%2BA.%2BGore%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-6896-8024" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Michael A. Gore</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Suchismita Mondal </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Global Wheat Program, International Maize and Wheat Improvement Center (CIMMYT), Ciudad de México, 06600, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Suchismita%2BMondal%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mondal%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASuchismita%2BMondal%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-8582-8899" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Suchismita Mondal</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:S.Mondal@cgiar.org" class="" data-icon-position="" data-hide-link-title="0">S.Mondal@cgiar.org</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/9/4/1231" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/9/4/1231&amp;title=Hyperspectral%20Reflectance-Derived%20Relationship%20Matrices%20for%20Genomic%20Prediction%20of%20Grain%20Yield%20in%20Wheat" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/9/4/1231&amp;title=Hyperspectral%20Reflectance-Derived%20Relationship%20Matrices%20for%20Genomic%20Prediction%20of%20Grain%20Yield%20in%20Wheat" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/9/4/1231&amp;title=Hyperspectral%20Reflectance-Derived%20Relationship%20Matrices%20for%20Genomic%20Prediction%20of%20Grain%20Yield%20in%20Wheat" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/9/4/1231&amp;text=Hyperspectral%20Reflectance-Derived%20Relationship%20Matrices%20for%20Genomic%20Prediction%20of%20Grain%20Yield%20in%20Wheat" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/9/4/1231&amp;title=Hyperspectral%20Reflectance-Derived%20Relationship%20Matrices%20for%20Genomic%20Prediction%20of%20Grain%20Yield%20in%20Wheat" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/9/4/1231&amp;t=Hyperspectral%20Reflectance-Derived%20Relationship%20Matrices%20for%20Genomic%20Prediction%20of%20Grain%20Yield%20in%20Wheat" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/9/4/1231&amp;title=Hyperspectral%20Reflectance-Derived%20Relationship%20Matrices%20for%20Genomic%20Prediction%20of%20Grain%20Yield%20in%20Wheat" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/9/4/1231&amp;title=Hyperspectral%20Reflectance-Derived%20Relationship%20Matrices%20for%20Genomic%20Prediction%20of%20Grain%20Yield%20in%20Wheat" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/492620/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="492620" id="node492620--3" data-pisa="ggg;9/4/1231" data-pisa-master="ggg;g3.118.200856" data-apath="/ggg/9/4/1231.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Hyperspectral Reflectance-Derived Relationship Matrices for Genomic Prediction of Grain Yield in Wheat</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-1154-337X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Margaret R. Krause</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-5840-0803" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Lorena González-Pérez</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0001-9429-5855" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>José Crossa</span>, <span class="highwire-citation-author" data-delta="3">Paulino Pérez-Rodríguez</span>, <span class="highwire-citation-author" data-delta="4">Osval Montesinos-López</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0002-4676-5071 " target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Ravi P. Singh</span>, <span class="highwire-citation-author" data-delta="6">Susanne Dreisigacker</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="7"><a href="http://orcid.org/0000-0002-7856-1399" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jesse Poland</span>, <span class="highwire-citation-author" data-delta="8">Jessica Rutkoski</span>, <span class="highwire-citation-author" data-delta="9">Mark Sorrells</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="10"><a href="http://orcid.org/0000-0001-6896-8024" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Michael A. Gore</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="11"><a href="http://orcid.org/0000-0002-8582-8899" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Suchismita Mondal</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1231-1247;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200856 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node492620--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Margaret R. Krause </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Breeding and Genetics Section, School of Integrative Plant Science, Cornell University, Ithaca, New York, 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Margaret%2BR.%2BKrause%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Krause%20MR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMargaret%2BR.%2BKrause%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-1154-337X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Margaret R. Krause</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Lorena González-Pérez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Global Wheat Program, International Maize and Wheat Improvement Center (CIMMYT), Ciudad de México, 06600, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lorena%2BGonz%C3%A1lez-P%C3%A9rez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gonz%C3%A1lez-P%C3%A9rez%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALorena%2BGonz%25C3%25A1lez-P%25C3%25A9rez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-5840-0803" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Lorena González-Pérez</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">José Crossa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Global Wheat Program, International Maize and Wheat Improvement Center (CIMMYT), Ciudad de México, 06600, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jos%C3%A9%2BCrossa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Crossa%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJos%25C3%25A9%2BCrossa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-9429-5855" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for José Crossa</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Paulino Pérez-Rodríguez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Colegio de Postgraduados, CP 56230, Montecillos, Edo. de México, Mexico,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Paulino%2BP%C3%A9rez-Rodr%C3%ADguez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=P%C3%A9rez-Rodr%C3%ADguez%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APaulino%2BP%25C3%25A9rez-Rodr%25C3%25ADguez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Osval Montesinos-López </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Facultad de Telemática, Universidad de Colima, Colima, Colima, 28040, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Osval%2BMontesinos-L%C3%B3pez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Montesinos-L%C3%B3pez%20O&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AOsval%2BMontesinos-L%25C3%25B3pez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Ravi P. Singh </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Global Wheat Program, International Maize and Wheat Improvement Center (CIMMYT), Ciudad de México, 06600, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ravi%2BP.%2BSingh%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Singh%20RP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARavi%2BP.%2BSingh%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-4676-5071 " target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Ravi P. Singh</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Susanne Dreisigacker </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Global Wheat Program, International Maize and Wheat Improvement Center (CIMMYT), Ciudad de México, 06600, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Susanne%2BDreisigacker%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dreisigacker%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASusanne%2BDreisigacker%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Jesse Poland </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Plant Pathology, Kansas State University, Manhattan, Kansas, 66506</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jesse%2BPoland%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Poland%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJesse%2BPoland%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-7856-1399" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jesse Poland</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Jessica Rutkoski </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>International Rice Research Institute (IRRI), DAPO Box 7777, Metro Manila, 1301, Philippines</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jessica%2BRutkoski%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rutkoski%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJessica%2BRutkoski%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Mark Sorrells </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Breeding and Genetics Section, School of Integrative Plant Science, Cornell University, Ithaca, New York, 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mark%2BSorrells%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sorrells%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMark%2BSorrells%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Michael A. Gore </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Breeding and Genetics Section, School of Integrative Plant Science, Cornell University, Ithaca, New York, 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BA.%2BGore%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gore%20MA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMichael%2BA.%2BGore%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-6896-8024" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Michael A. Gore</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Suchismita Mondal </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Global Wheat Program, International Maize and Wheat Improvement Center (CIMMYT), Ciudad de México, 06600, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Suchismita%2BMondal%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mondal%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASuchismita%2BMondal%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-8582-8899" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Suchismita Mondal</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:S.Mondal@cgiar.org" class="" data-icon-position="" data-hide-link-title="0">S.Mondal@cgiar.org</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1231&amp;rft.epage=1247&amp;rft.atitle=Hyperspectral%20Reflectance-Derived%20Relationship%20Matrices%20for%20Genomic%20Prediction%20of%20Grain%20Yield%20in%20Wheat&amp;rft.volume=9&amp;rft.issue=4&amp;rft.date=2019-04-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Krause%2C+Margaret+R.&amp;rft.au=Gonz%C3%A1lez-P%C3%A9rez%2C+Lorena&amp;rft.au=Crossa%2C+Jos%C3%A9&amp;rft.au=P%C3%A9rez-Rodr%C3%ADguez%2C+Paulino&amp;rft.au=Montesinos-L%C3%B3pez%2C+Osval&amp;rft.au=Singh%2C+Ravi+P.&amp;rft.au=Dreisigacker%2C+Susanne&amp;rft.au=Poland%2C+Jesse&amp;rft.au=Rutkoski%2C+Jessica&amp;rft.au=Sorrells%2C+Mark&amp;rft.au=Gore%2C+Michael+A.&amp;rft.au=Mondal%2C+Suchismita"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/492620/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/492620/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/492620/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/492620/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/492620/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/492620/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/492620/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/492620/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/492620/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/492620/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/492620/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/492620/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495864" id="node495864" data-pisa="ggg;10/3/1113" data-pisa-master="ggg;g3.119.400968" data-apath="/ggg/10/3/1113.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1113" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Multi-trait Genomic Prediction Model Increased the Predictive Ability for Agronomic and Malting Quality Traits in Barley (<em>Hordeum vulgare</em> L.)</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495543" id="node495543" data-pisa="ggg;10/2/769" data-pisa-master="ggg;g3.119.400759" data-apath="/ggg/10/2/769.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/2/769" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Novel Bayesian Networks for Genomic Prediction of Developmental Traits in Biomass Sorghum</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495557" id="node495557" data-pisa="ggg;10/2/539" data-pisa-master="ggg;g3.119.400866" data-apath="/ggg/10/2/539.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/2/539" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Optimized Genetic Testing for Polledness in Multiple Breeds of Cattle</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Genomic%20Prediction" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Genomic Prediction</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/9/4/1231" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials And Methods</a></li><li class="odd"><a href="#sec-15" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-21" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/9/4/1231.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/9/4/1231.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/9/4/1231.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/9/4/1231.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
