<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/9/4/1027" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/9/4/1027.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/9/4/1027.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/9/4/1027.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/9/4/1027.atom" />
<meta name="HW.pisa" content="ggg;9/4/1027" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="High-Density Linkage Map and QTLs for Growth in Snapper (Chrysophrys auratus)" />
<meta name="DC.Identifier" content="10.1534/g3.118.200905" />
<meta name="DC.Date" content="2019-04-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2019 Ashton et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Characterizing the genetic variation underlying phenotypic traits is a central objective in biological research. This research has been hampered in the past by the limited genomic resources available for most non-model species. However, recent advances in sequencing technologies and related genotyping methods are rapidly changing this. Here we report the use of genome-wide SNP data from the ecologically and commercially important marine fish species Chrysophrys auratus (snapper) to 1) construct the first linkage map for this species, 2) scan for growth QTL, and 3) search for putative candidate genes in the surrounding QTL regions. The newly constructed linkage map contained ∼11K SNP markers and is one of the densest maps to date in the fish family Sparidae. Comparisons with genome scaffolds of the recently assembled snapper genome indicated that marker placement was mostly consistent between the scaffolds and linkage map (R = 0.7), but that at fine scales (&lt; 5 cM) some precision limitations occurred. Of the 24 linkage groups, which likely reflect the 24 chromosomes of this species, three were found to contain QTL with genome-wide significance for growth-related traits. A scan of 13 candidate growth genes located the growth hormone, myogenin, and parvalbumin genes within 5.3, 9.6, and 25.0 cM of these QTL, respectively. The linkage map and QTL found in this study will advance the investigation of genome structure and aquaculture breeding efforts in this and related species." />
<meta name="DC.Contributor" content="David T. Ashton" />
<meta name="DC.Contributor" content="Peter A. Ritchie" />
<meta name="DC.Contributor" content="Maren Wellenreuther" />
<meta name="article:published_time" content="2019-04-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="High-Density Linkage Map and QTLs for Growth in Snapper (Chrysophrys auratus)" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Characterizing the genetic variation underlying phenotypic traits is a central objective in biological research. This research has been hampered in the past by the limited genomic resources available for most non-model species. However, recent advances in sequencing technologies and related genotyping methods are rapidly changing this. Here we report the use of genome-wide SNP data from the ecologically and commercially important marine fish species &lt;i&gt;Chrysophrys auratus&lt;/i&gt; (snapper) to 1) construct the first linkage map for this species, 2) scan for growth QTL, and 3) search for putative candidate genes in the surrounding QTL regions. The newly constructed linkage map contained ∼11K SNP markers and is one of the densest maps to date in the fish family Sparidae. Comparisons with genome scaffolds of the recently assembled snapper genome indicated that marker placement was mostly consistent between the scaffolds and linkage map (R = 0.7), but that at fine scales (&amp;lt; 5 cM) some precision limitations occurred. Of the 24 linkage groups, which likely reflect the 24 chromosomes of this species, three were found to contain QTL with genome-wide significance for growth-related traits. A scan of 13 candidate growth genes located the &lt;i&gt;growth hormone, myogenin, and parvalbumin&lt;/i&gt; genes within 5.3, 9.6, and 25.0 cM of these QTL, respectively. The linkage map and QTL found in this study will advance the investigation of genome structure and aquaculture breeding efforts in this and related species.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2019/04/01" />
<meta name="citation_mjid" content="ggg;9/4/1027" />
<meta name="citation_id" content="9/4/1027" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/9/4/1027" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/9/4/1027.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/9/4/1027.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/9/4/1027.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.118.200905" />
<meta name="citation_pmid" content="30804023" />
<meta name="citation_volume" content="9" />
<meta name="citation_issue" content="4" />
<meta name="citation_num_pages" content="9" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="1027" />
<meta name="citation_lastpage" content="1035" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="David T. Ashton" />
<meta name="citation_author_institution" content="The New Zealand Institute for Plant &amp; Food Research Limited, Nelson, New Zealand" />
<meta name="citation_author_email" content="david.ashton@plantandfood.co.nz" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-0996-6368" />
<meta name="citation_author" content="Peter A. Ritchie" />
<meta name="citation_author_institution" content="School of Biological Sciences, Victoria University of Wellington, Wellington, New Zealand" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-8351-7931" />
<meta name="citation_author" content="Maren Wellenreuther" />
<meta name="citation_author_institution" content="The New Zealand Institute for Plant &amp; Food Research Limited, Nelson, New Zealand" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-2764-8291" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=G. R. Abecasis;citation_author=L. R. Cardon;citation_author=W. O. Cookson;citation_title=A general test of association for quantitative traits in nuclear families.;citation_pages=279-292;citation_volume=66;citation_year=2000;citation_issue=1;citation_pmid=10631157;citation_doi=10.1086/302698" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=S. M. Aljanabi;citation_author=I. Martinez;citation_title=Universal and rapid salt-extraction of high quality genomic DNA for PCR-based techniques.;citation_volume=25;citation_year=1997;citation_pmid=9358185;citation_doi=10.1093/nar/25.22.4692" />
<meta name="citation_reference" content="citation_journal_title=Genome (Ottawa. Print);citation_journal_abbrev=Genome (Ottawa. Print);citation_author=R. Almuly;citation_author=B. Cavari;citation_author=H. Ferstman;citation_author=O. Kolodny;citation_author=B. Funkenstein;citation_title=Genomic structure and sequence of the gilthead seabream (Sparus aurata) growth hormone-encoding gene: identification of minisatellite polymorphism in intron I.;citation_pages=836-845;citation_volume=43;citation_year=2000;citation_issue=5;citation_pmid=11081974;citation_doi=10.1139/g00-051" />
<meta name="citation_reference" content="Aronesty, E., 2011 ea-utils: Command-line tools for processing biological sequencing data. https://github.com/ExpressionAnalysis/ea-utils" />
<meta name="citation_reference" content="Ashton, D. T, E. Hilario, P. Jaksons, P. A. Ritchie, M. Wellenreuther, 2014 Genetic diversity and heritability of economically important traits in captive Australasian snapper (Chrysophrys auratus). Aquaculture. 505: 190–198. https://doi.org/10.1016/j.aquaculture.2019.02.034" />
<meta name="citation_reference" content="Ashton, D. T., E. Hilario, P. Jaksons, P. A. Ritchie, and M. Wellenreuther, 2018 Genetic diversity and heritability of economically important traits in captive Australasian snapper (Chrysophrys auratus). in prep." />
<meta name="citation_reference" content="citation_journal_title=bioRxiv;citation_author=A. Barria;citation_author=K. A. Christensen;citation_author=K. Correa;citation_author=A. Jedlicki;citation_author=J. P. Lhorente;citation_title=Genome-wide association study and genomic predictions for resistance against Piscirickettsia salmonis in Coho salmon (Oncorhynchus kisutch) using ddRAD sequencing.;citation_pages=124099;citation_volume=8;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=F. Besnier;citation_author=K. A. Glover;citation_author=S. Lien;citation_author=M. Kent;citation_author=M. M. Hansen;citation_title=Identification of quantitative genetic components of fitness variation in farmed, hybrid and native salmon in the wild.;citation_pages=47-55;citation_volume=115;citation_year=2015;citation_doi=10.1038/hdy.2015.15" />
<meta name="citation_reference" content="citation_journal_title=Rice (N. Y.);citation_author=M. Bettembourg;citation_author=A. Dardou;citation_author=A. Audebert;citation_author=E. Thomas;citation_author=J. Frouin;citation_title=Genome-wide association mapping for root cone angle in rice.;citation_pages=45;citation_volume=10;citation_year=2017;citation_doi=10.1186/s12284-017-0184-z" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=S. Bhatta;citation_author=T. Iwai;citation_author=C. Miura;citation_author=M. Higuchi;citation_author=S. Shimizu-Yamaguchi;citation_title=Gonads directly regulate growth in teleosts.;citation_volume=109;citation_year=2012;citation_pmid=22733736;citation_doi=10.1073/pnas.1118704109" />
<meta name="citation_reference" content="citation_journal_title=Aquaculture;citation_author=K. Boulton;citation_author=C. Massault;citation_author=R. D. Houston;citation_author=D. J. de Koning;citation_author=C. S. Haley;citation_title=QTL affecting morphometric traits and stress response in the gilthead seabream (Sparus aurata).;citation_pages=58-66;citation_volume=319;citation_year=2011;citation_doi=10.1016/j.aquaculture.2011.06.044" />
<meta name="citation_reference" content="citation_journal_title=J. Exp. Zoolog. B Mol. Dev. Evol.;citation_author=I. Braasch;citation_author=S. M. Peterson;citation_author=T. Desvignes;citation_author=B. M. McCluskey;citation_author=P. Batzel;citation_title=A new model army: Emerging fish models to study the genomics of vertebrate Evo-Devo.;citation_volume=324;citation_year=2015;citation_doi=10.1002/jez.b.22589" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_journal_abbrev=Bioinformatics;citation_author=K. W. Broman;citation_author=H. Wu;citation_author=S. Sen;citation_author=G. A. Churchill;citation_title=R/qtl: QTL mapping in experimental crosses;citation_pages=889-890;citation_volume=19;citation_year=2003;citation_issue=7;citation_pmid=12724300;citation_doi=10.1093/bioinformatics/btg112" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=C. Castaño-Sánchez;citation_author=K. Fuji;citation_author=A. Ozaki;citation_author=O. Hasegawa;citation_author=T. Sakamoto;citation_title=A second generation genetic linkage map of Japanese flounder (Paralichthys olivaceus).;citation_pages=554-554;citation_volume=11;citation_year=2010;citation_pmid=20937088;citation_doi=10.1186/1471-2164-11-554" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=J. Catchen;citation_author=P. A. Hohenlohe;citation_author=S. Bassham;citation_author=A. Amores;citation_author=W. A. Cresko;citation_title=Stacks: an analysis tool set for population genomics.;citation_pages=3124-3140;citation_volume=22;citation_year=2013;citation_pmid=23701397;citation_doi=10.1111/mec.12354" />
<meta name="citation_reference" content="citation_journal_title=Mol. Breed.;citation_author=H. Chen;citation_author=K. Semagn;citation_author=M. Iqbal;citation_author=N. P. Moakhar;citation_author=T. Haile;citation_title=Genome-wide association mapping of genomic regions associated with phenotypic traits in Canadian western spring wheat.;citation_pages=141;citation_volume=37;citation_year=2017;citation_doi=10.1007/s11032-017-0741-6" />
<meta name="citation_reference" content="citation_journal_title=Molecular genetics and genomics :  MGG;citation_journal_abbrev=Mol Genet Genomics;citation_author=A. Cnaani;citation_author=N. Zilberman;citation_author=S. Tinman;citation_author=G. Hulata;citation_author=M. Ron;citation_title=Genome-scan analysis for quantitative trait loci in an F2 tilapia hybrid.;citation_pages=162-172;citation_volume=272;citation_year=2004;citation_issue=2;citation_pmid=15449174;citation_doi=10.1007/s00438-004-1045-1" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=D. Cros;citation_author=S. Bocs;citation_author=V. Riou;citation_author=E. Ortega-Abboud;citation_author=S. Tisné;citation_title=Genomic preselection with genotyping-by-sequencing increases performance of commercial oil palm hybrid crosses.;citation_pages=839;citation_volume=18;citation_year=2017;citation_doi=10.1186/s12864-017-4179-3" />
<meta name="citation_reference" content="citation_journal_abbrev=Curr Genomics;citation_author=J. C. Dekkers;citation_title=Application of genomics tools to animal breeding.;citation_pages=207-212;citation_volume=13;citation_year=2012;citation_issue=3;citation_pmid=23115522;citation_doi=10.2174/138920212800543057" />
<meta name="citation_reference" content="citation_journal_title=Aquaculture;citation_author=C. De-Santis;citation_author=D. R. Jerry;citation_title=Candidate growth genes in finfish — Where should we be looking?;citation_pages=22-38;citation_volume=272;citation_year=2007;citation_doi=10.1016/j.aquaculture.2007.08.036" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=R. J. Elshire;citation_author=J. C. Glaubitz;citation_author=Q. Sun;citation_author=J. A. Poland;citation_author=K. Kawamoto;citation_title=A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species.;citation_pages=e19379-e19379;citation_volume=6;citation_year=2011;citation_issue=5;citation_pmid=21573248;citation_doi=10.1371/journal.pone.0019379" />
<meta name="citation_reference" content="citation_journal_title=Evol. Appl.;citation_author=M. V. Everett;citation_author=J. E. Seeb;citation_title=Detection and mapping of QTL for temperature tolerance and body size in Chinook salmon (Oncorhynchus tshawytscha) using genotyping by sequencing.;citation_pages=480-492;citation_volume=7;citation_year=2014;citation_pmid=24822082;citation_doi=10.1111/eva.12147" />
<meta name="citation_reference" content="citation_journal_title=Front. Genet.;citation_author=J. L. Fierst;citation_title=Using linkage maps to correct and scaffold de novo genome assemblies: methods, challenges, and computational tools.;citation_pages=1-8;citation_volume=6;citation_year=2015;citation_pmid=25674101;citation_doi=10.3389/fgene.2015.00001" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_journal_abbrev=Heredity;citation_author=A. K. Greenwood;citation_author=F. C. Jones;citation_author=Y. F. Chan;citation_author=S. D. Brady;citation_author=D. M. Absher;citation_title=The genetic basis of divergent pigment patterns in juvenile threespine sticklebacks.;citation_pages=155-166;citation_volume=107;citation_year=2011;citation_issue=2;citation_pmid=21304547;citation_doi=10.1038/hdy.2011.1" />
<meta name="citation_reference" content="citation_journal_title=J. Genet.;citation_author=J. B. S. Haldane;citation_title=Sex ratio and unisexual sterility in hybrid animals.;citation_pages=101-109;citation_volume=12;citation_year=1922;citation_doi=10.1007/BF02983075" />
<meta name="citation_reference" content="citation_journal_title=Aquaculture;citation_author=S. O. Handeland;citation_author=A. K. Imsland;citation_author=S. O. Stefansson;citation_title=The effect of temperature and fish size on growth, feed intake, food conversion efficiency and stomach evacuation rate of Atlantic salmon post-smolts.;citation_pages=36-42;citation_volume=283;citation_year=2008;citation_doi=10.1016/j.aquaculture.2008.06.042" />
<meta name="citation_reference" content="citation_title=Plant Genotyping, Methods in Molecular Biology;citation_title=The restriction enzyme target approach to Genotyping by Sequencing (GBS);citation_author=E. Hilario;citation_pages=271-279;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Animal genetics;citation_journal_abbrev=Anim Genet;citation_author=R. D. Houston;citation_author=S. C. Bishop;citation_author=A. Hamilton;citation_author=D. R. Guy;citation_author=A. E. Tinch;citation_title=Detection of QTL affecting harvest traits in a commercial Atlantic salmon population.;citation_pages=753-755;citation_volume=40;citation_year=2009;citation_issue=5;citation_pmid=19397515;citation_doi=10.1111/j.1365-2052.2009.01883.x" />
<meta name="citation_reference" content="citation_journal_title=J. Anim. Sci.;citation_author=A. M. Hutson;citation_author=Z. Liu;citation_author=H. Kucuktas;citation_author=G. Umali-Maceina;citation_author=B. Su;citation_title=Quantitative trait loci map for growth and morphometric traits using a channel catfish × blue catfish interspecific hybrid system.;citation_volume=92;citation_year=2014;citation_pmid=24663183;citation_doi=10.2527/jas.2013-7191" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=S. T. Kalinowski;citation_author=M. L. Taper;citation_author=T. C. Marshall;citation_title=Revising how the computer program CERVUS accommodates genotyping error increases success in paternity assignment.;citation_pages=1099-1106;citation_volume=16;citation_year=2007;citation_issue=5;citation_pmid=17305863;citation_doi=10.1111/j.1365-294X.2007.03089.x" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=M. Kearse;citation_author=R. Moir;citation_author=A. Wilson;citation_author=S. Stones-Havas;citation_author=M. Cheung;citation_title=Geneious Basic: An integrated and extendable desktop software platform for the organization and analysis of sequence data.;citation_volume=28;citation_year=2012;citation_pmid=22543367;citation_doi=10.1093/bioinformatics/bts199" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=S. A. Knott;citation_author=J. M. Elsen;citation_author=C. S. Haley;citation_title=Methods for multiple-marker mapping of quantitative trait loci in half-sib populations.;citation_pages=71-80;citation_volume=93;citation_year=1996;citation_pmid=24162201;citation_doi=10.1007/s001220050249" />
<meta name="citation_reference" content="Ko, W., R. M. David, H. Akashi, 2003 Molecular phylogeny of the Drosophila melanogaster species subgroup. Journal of Molecular Evolution. 57: 562–573. https://doi.org/10.1007/s00239-003-2510-x" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=H. Kucuktas;citation_author=S. Wang;citation_author=P. Li;citation_author=C. He;citation_author=P. Xu;citation_title=Construction of genetic linkage maps and comparative genome analysis of Catfish using gene-associated markers.;citation_volume=181;citation_year=2009;citation_pmid=19171943;citation_doi=10.1534/genetics.108.098855" />
<meta name="citation_reference" content="citation_journal_title=Nat. Rev. Genet.;citation_author=B. Lehner;citation_title=Genotype to phenotype: lessons from model organisms for human genetics.;citation_pages=168-178;citation_volume=14;citation_year=2013;citation_pmid=23358379;citation_doi=10.1038/nrg3404" />
<meta name="citation_reference" content="citation_journal_title=DNA Res.;citation_author=Y. Li;citation_author=S. Liu;citation_author=Z. Qin;citation_author=G. Waldbieser;citation_author=R. Wang;citation_title=Construction of a high-density, high-resolution genetic map and its integration with BAC-based physical map in channel catfish.;citation_volume=22;citation_year=2015;citation_pmid=25428894;citation_doi=10.1093/dnares/dsu038" />
<meta name="citation_reference" content="citation_title=Genetics and analysis of quantitative traits;citation_author=M. Lynch;citation_author=B. Walsh;citation_year=1998" />
<meta name="citation_reference" content="citation_title=Zoology;citation_author=S. Miller;citation_author=J. Harley;citation_year=2006" />
<meta name="citation_reference" content="citation_journal_title=The Biological Bulletin;citation_journal_abbrev=Biol. Bull.;citation_author=T. H. Morgan;citation_title=NO CROSSING OVER IN THE MALE OF DROSOPHILA OF GENES IN THE SECOND AND THIRD PAIRS OF CHROMOSOMES;citation_pages=195-204;citation_volume=26;citation_year=1914;citation_issue=4;citation_doi=10.2307/1536193" />
<meta name="citation_reference" content="citation_journal_title=Fish. Sci.;citation_author=O. Murata;citation_author=T. Harada;citation_author=S. Miyashita;citation_author=K.-I. Izumi;citation_author=S. Maeda;citation_title=Selective breeding for growth in red sea bream.;citation_pages=845-849;citation_volume=62;citation_year=1996;citation_doi=10.2331/fishsci.62.845" />
<meta name="citation_reference" content="citation_journal_title=G3&amp;#58; Genes|Genomes|Genetics;citation_journal_abbrev=G3;citation_author=P. Ninwichian;citation_author=E. Peatman;citation_author=H. Liu;citation_author=H. Kucuktas;citation_author=B. Somridhivej;citation_title=Second-Generation Genetic Linkage Map of Catfish and Its Integration with the BAC-Based Physical Map;citation_pages=1233-1241;citation_volume=2;citation_year=2012;citation_issue=10;citation_pmid=23050234;citation_doi=10.1534/g3.112.003962" />
<meta name="citation_reference" content="citation_journal_title=Dev. Reprod.;citation_author=I.-S. Park;citation_author=H. W. Gil;citation_author=T. H. Lee;citation_author=Y. K. Nam;citation_author=D. S. Kim;citation_title=Comparative study of growth and gonad maturation in diploid and triploid marine medaka, Oryzias dancena.;citation_pages=305-314;citation_volume=20;citation_year=2016;citation_doi=10.12717/DR.2016.20.4.305" />
<meta name="citation_reference" content="citation_journal_title=N. Z. J. Mar. Freshw. Res.;citation_author=D. M. Parsons;citation_author=C. J. Sim-Smith;citation_author=M. Cryer;citation_author=M. P. Francis;citation_author=B. Hartill;citation_title=Snapper (Chrysophrys auratus): a review of life history and key vulnerabilities in New Zealand.;citation_pages=256-283;citation_volume=48;citation_year=2014;citation_doi=10.1080/00288330.2014.892013" />
<meta name="citation_reference" content="citation_journal_title=Journal of Animal Science;citation_journal_abbrev=J ANIM SCI;citation_author=M. F. te Pas;citation_author=A. Soumillion;citation_author=F. L. Harders;citation_author=F. J. Verburg;citation_author=T. J. van den Bosch;citation_title=Influences of myogenin genotypes on birth weight, growth rate, carcass weight, backfat thickness, and lean weight of pigs;citation_pages=2352-2356;citation_volume=77;citation_year=1999;citation_issue=9;citation_pmid=10492439;citation_doi=10.2527/1999.7792352x" />
<meta name="citation_reference" content="citation_journal_title=Sci. Rep.;citation_author=F. Pértille;citation_author=G. C. Moreira;citation_author=R. Zanella;citation_author=J. R. Nunes;citation_author=C. Boschiero;citation_title=Genome-wide association study for performance traits in chickens using genotype by sequencing approach.;citation_pages=41748;citation_volume=7;citation_year=2017;citation_doi=10.1038/srep41748" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=S. Purcell;citation_author=B. Neale;citation_author=K. Todd-Brown;citation_author=L. Thomas;citation_author=M. A. R. Ferreira;citation_title=PLINK: a tool set for whole-genome association and population-based linkage analyses.;citation_pages=559-575;citation_volume=81;citation_year=2007;citation_issue=3;citation_pmid=17701901;citation_doi=10.1086/519795" />
<meta name="citation_reference" content="citation_title=R: A language and environment for statistical computing;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol. Evol.;citation_author=P. Rastas;citation_author=F. C. F. Calboli;citation_author=B. Guo;citation_author=T. Shikano;citation_author=J. Merilä;citation_title=Construction of ultradense linkage maps with Lep-MAP2: stickleback F2 recombinant crosses as an example.;citation_volume=8;citation_year=2016;citation_pmid=26668116;citation_doi=10.1093/gbe/evv250" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=M. Roesti;citation_author=D. Moser;citation_author=D. Berner;citation_title=Recombination in the threespine stickleback genome—patterns and consequences.;citation_pages=3014-3027;citation_volume=22;citation_year=2013;citation_pmid=23601112.;citation_doi=10.1111/mec.12322" />
<meta name="citation_reference" content="citation_journal_title=Nat. Rev. Genet.;citation_author=O. Savolainen;citation_author=M. Lascoux;citation_author=J. Merilä;citation_title=Ecological genomics of local adaptation.;citation_pages=807-820;citation_volume=14;citation_year=2013;citation_pmid=24136507;citation_doi=10.1038/nrg3522" />
<meta name="citation_reference" content="citation_author=G. Seaton;citation_author=J. Hernandez-Sanchez;citation_author=J.-A. Grunchec;citation_author=I. White;citation_author=J. Allen;citation_year=2006" />
<meta name="citation_reference" content="citation_journal_title=BMC Genet.;citation_author=H. Y. Tsai;citation_author=A. Hamilton;citation_author=D. R. Guy;citation_author=A. E. Tinch;citation_author=S. C. Bishop;citation_title=The genetic architecture of growth and fillet traits in farmed Atlantic salmon (Salmo salar).;citation_pages=51;citation_volume=16;citation_year=2015;citation_doi=10.1186/s12863-015-0215-y" />
<meta name="citation_reference" content="citation_author=H. Y. Tsai;citation_author=D. Robledo;citation_author=N. R. Lowe;citation_author=M. Bekaert;citation_author=J. B. Taggart;citation_pmid=27194803;citation_doi=10.1534/g3.116.029009" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=P. M. VanRaden;citation_author=C. P. Van Tassell;citation_author=G. R. Wiggans;citation_author=T. S. Sonstegard;citation_author=R. D. Schnabel;citation_title=Invited Review: Reliability of genomic predictions for North American Holstein bulls;citation_pages=16-24;citation_volume=92;citation_year=2009;citation_issue=1;citation_pmid=19109259;citation_doi=10.3168/jds.2008-1514" />
<meta name="citation_reference" content="citation_journal_title=Front. Biol. China;citation_author=C. Wang;citation_title=Quantitative genetic estimates of growth-related traits in the common carp (Cyprinus carpio L.): A review.;citation_pages=298-304;citation_volume=4;citation_year=2009;citation_doi=10.1007/s11515-009-0031-8" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=L. Wang;citation_author=B. Bai;citation_author=P. Liu;citation_author=S. Q. Huang;citation_author=Z. Y. Wan;citation_title=Construction of high-resolution recombination maps in Asian seabass.;citation_pages=1-14;citation_volume=18;citation_year=2017;citation_doi=10.1186/s12864-017-3899-8" />
<meta name="citation_reference" content="citation_journal_title=Trends Genet.;citation_author=M. Wellenreuther;citation_author=B. Hansson;citation_title=Detecting polygenic evolution: problems, pitfalls, and promises.;citation_pages=155-164;citation_volume=32;citation_year=2016;citation_doi=10.1016/j.tig.2015.12.004" />
<meta name="citation_reference" content="citation_journal_title=Aquaculture and Fisheries;citation_author=B. Ye;citation_author=Z. Wan;citation_author=L. Wang;citation_author=H. Pang;citation_author=Y. Wen;citation_title=Heritability of growth traits in the Asian seabass (Lates calcarifer).;citation_pages=112-118;citation_volume=2;citation_year=2017;citation_doi=10.1016/j.aaf.2017.06.001" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="High-Density Linkage Map and QTLs for Growth in Snapper (Chrysophrys auratus)" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4.cover-source.jpg" />
<meta name="twitter:description" content="Characterizing the genetic variation underlying phenotypic traits is a central objective in biological research. This research has been hampered in the past by the limited genomic resources available for most non-model species. However, recent advances in sequencing technologies and related genotyping methods are rapidly changing this. Here we report the use of genome-wide SNP data from the ecologically and commercially important marine fish species Chrysophrys auratus (snapper) to 1) construct the first linkage map for this species, 2) scan for growth QTL, and 3) search for putative candidate genes in the surrounding QTL regions. The newly constructed linkage map contained ∼11K SNP markers and is one of the densest maps to date in the fish family Sparidae. Comparisons with genome scaffolds of the recently assembled snapper genome indicated that marker placement was mostly consistent between the scaffolds and linkage map (R = 0.7), but that at fine scales (&lt; 5 cM) some precision limitations occurred. Of the 24 linkage groups, which likely reflect the 24 chromosomes of this species, three were found to contain QTL with genome-wide significance for growth-related traits. A scan of 13 candidate growth genes located the growth hormone, myogenin, and parvalbumin genes within 5.3, 9.6, and 25.0 cM of these QTL, respectively. The linkage map and QTL found in this study will advance the investigation of genome structure and aquaculture breeding efforts in this and related species." />
<meta name="og-title" property="og:title" content="High-Density Linkage Map and QTLs for Growth in Snapper (Chrysophrys auratus)" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/9/4/1027" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Characterizing the genetic variation underlying phenotypic traits is a central objective in biological research. This research has been hampered in the past by the limited genomic resources available for most non-model species. However, recent advances in sequencing technologies and related genotyping methods are rapidly changing this. Here we report the use of genome-wide SNP data from the ecologically and commercially important marine fish species Chrysophrys auratus (snapper) to 1) construct the first linkage map for this species, 2) scan for growth QTL, and 3) search for putative candidate genes in the surrounding QTL regions. The newly constructed linkage map contained ∼11K SNP markers and is one of the densest maps to date in the fish family Sparidae. Comparisons with genome scaffolds of the recently assembled snapper genome indicated that marker placement was mostly consistent between the scaffolds and linkage map (R = 0.7), but that at fine scales (&lt; 5 cM) some precision limitations occurred. Of the 24 linkage groups, which likely reflect the 24 chromosomes of this species, three were found to contain QTL with genome-wide significance for growth-related traits. A scan of 13 candidate growth genes located the growth hormone, myogenin, and parvalbumin genes within 5.3, 9.6, and 25.0 cM of these QTL, respectively. The linkage map and QTL found in this study will advance the investigation of genome structure and aquaculture breeding efforts in this and related species." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4.cover-source.jpg" />
    <title>High-Density Linkage Map and QTLs for Growth in Snapper (Chrysophrys auratus) | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"1d1kAOxPzzk-3C3TkwVk5M3RgXrYP-SHpt8bTRYMrMg","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"492612","apath":"\/ggg\/9\/4\/1027.atom","pisa":"ggg;9\/4\/1027","ac":{"\/ggg\/9\/4\/1027.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/9\/4\/1027.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;9\/4\/1027"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;9\/4\/1027"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/9\/4\/1027":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022B\\\\\\\/Hp84Zjg2pIV+9LEjkpX8ZUAAFVmUm9fxbtEgLzh82\\\\\\\/MQ\\\\\\\/6IGY3Hsi9dnnaw+dqBksF8d9EpWA+0cdtyRfBcTp5vIkXfZVPi11qoo+ctbww+gJ0k+u0KgRJtX7+Ke\\\\\\\/eIzaK1SUtDL4QmKqEdb5K7lJ6b4dGJdjRN12jDyEIwPUk+zBGctGe6qVVu2ZBpgyaPHC9BCdXtiCFjhV1+\\\\\\\/BOxlub4iUPNM6Vy0grEk7+d9vVPSnod2ZFrnWJYL0xL5T46Ce3CXfQ5vMIr1UQE2dtqwcQF7i+4WjxMnsV8remGtZZOGG9UhvPmoKljdvJ4ywDpnaHlVzDYEt\\\\\\\/qcIT22pYdVEr7QGq2oYiEA8HuzYpEu1q+JzI3kj0BS58cpWRhQaJA8ry2brKW9O4j4PgMWzLVmWDKeM60\\\\\\\/risovSY+E8FUYQ2Ta2OkCijUBnR3C7UCThb7h4AHKO8maZam9n4o3ODbkTpwgm2o7wf82qnpVqJmnIAQDmpSNqzAivQMQHIyCXK1I7DNsBT86poUiNV30DiGCioxmigtCCkLhA\\\\\\\/Q7ZH49sFzKo920Ryrz4v7h7yxmXn4Wx8rfDFSd3NzO1hfxDjla3UNS6CAzlxGIWHAz45KttpIXp7Q+TvJDy8MaSxuDzueUfz1XP6wTpWX+bNiuzK6nArQercWYKpflCRUdh5EreEhltQDd87m\\\\\\\/PXjjitJGAbFdnAccULwHzGwU0WS5YUkWvd+fdMMnMoCX\\\\\\\/LjArUU4RmEWIswI+cAADJ5F9dJ8oKMLLlmjPiq35xkgN+way1tgDRJgcyjP\\\\\\\/4PViR51DCVvVidUerosPa6iYMdXplKynTa5MCLTzZy8EUloLfB2dYJJv0cENOBFGjSMunjTZcyjLFMxUtQ4vFzf6MfZRfU\\\\\\\/x2PW8sUc5xs\\\\\\\/EdUuCLVEVLKXed\\\\\\\/9KsiTILhdumSAgozqexdL63\\\\\\\/9kgT4jZPiVlNL5iUJVNNAdSb2QNJpBzRAynxY0rJrsl5xEC4XXUXHf7B+HGM2vDf7fpDf2+t5aQPppzdVgQyumepA4xicggblkVC3eRYYtzQ9ijE\\\\\\\/VX6irz6SMGiZMYbI5DILYefFk2VBcVgXuJomosbv9oQ7tkr5pY5zbZ\\\\\\\/EWH4hj2uw0vEhT9sQn3RMCIZVZCeaStiNQBNbtOkIvAAphBlu6cQhu5+6DdpyoAVbVWYHyMp+zKqFDKqsdxGFiHt6PRms50ueXjJ9WGPltqGbuZqYOrCPyM12Tm+cdATPdAf0YWT\\\\\\\/uwxf6s04sAypJliUjuwWJOkhvBDev\\\u0022,\\\u0022iv\\\u0022:\\\u0022rjAzEH3FmjeLC4g5g7Udag==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022c2c148933c7dd118169b23e312603c0196e599ec78ad892d657309e49102da9f\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022aG5w18UR+EwdIYeg63bwE\\\\\\\/x7Z\\\\\\\/KsZqEsrpzI0brJqvUTDwM1LSAwIAkwqbxX4mb0vun4OdJbBgsuZMPK7Sfde7DTi8P6HA3HQyc9yKoUxo6bE36DpnTWd32KFWrHO5lcFdR3yr9u2DIhYZh\\\\\\\/Q9AZu+mdQ288g9rIAH3gZOcttoY45Si9jwfGlUEu87CU142ldP4ivYDq8sV+V6tIWZIQxssEHAQXbDDeueeGlML+pg5SS\\\\\\\/+AMZHKaOH01v598aec\\\\\\\/ysRG3wGAlPwAqfLkO7\\\\\\\/6uH7aaUAPm\\\\\\\/u0XpApF825vKaYKZB3F0KIuS\\\\\\\/GRJD7WyQvZckLsA1s7+vIKt1uejH5jna2C8f\\\\\\\/Yb\\\\\\\/tN5ui+gY2jjH4Jl8wC45ZZbYJ7N4DfbOCkdeKAw55tpiFuQPd2yExkATXktgXOGMV7ljMgHGku+qeXymevgkKj9QJZVKLBiYB99UoqJwryCsWICtHHJQ65RbzDiPt9lcs9yRvcmznR+UUtpsGnjK0F\\\\\\\/tgPYBa9i83IgAnx+2lmAuV7JumOJ9\\\\\\\/kPBbjNzfAY4B8gDnikM3++ppWOciJyj6e3Eo24Z4n85qhEUsxmS8zLMshDku7EhVhAq+DgAKJZ2bvFW93FumNCgTZCGaEYvNhpiuGFD9bhAwa\\\\\\\/542YxH7ZBg\\\\\\\/+b71uc6BPYsCXOBhVtCbKOEcSsq5WEWhAbp\\\\\\\/TByq8lsBCEv8Ydj\\\\\\\/KfUF+2g9NLz19UTrgGkJr3GjZwwoaA\\\\\\\/ACYI2zJ1TuBfGenTxh074+YtPAO0\\\\\\\/mloPtdHf1psbaqLl\\\\\\\/sJNnERYr\\\\\\\/LA7yWjT95yx+sj+a\\\\\\\/GnbRsoYF65akYJzcdpbbTe7Ry9c0hA2cS71lwQHh2WkEyFSVyMZVLi1lzK73B\\\\\\\/slijSXeeFS8b4PbxvYWyA5CuJb4f\\\\\\\/M2Ti8ktxKdCrBXPowj56LqSGfJ8fV1m+dPfSIpATYpH80zqX25ukqbtQVbVStY5jN8hPrkjaEn4c1OZTojPZY+6QuDPF9z9iCvASpoYL1ZJTWqI24Yr\\\\\\\/bRBxOKY03cW6RxZIGkrOwRKkrstmI6pzE9MypNbn8JMcJuCG+Gvh\\\\\\\/nVRg25h24yHxBgB5G8yyoq\\\\\\\/kKFvBlfQZVQmzSobmnjEICttWamFit+\\\\\\\/monTDNoa8L7CcUIvSA8NSwRSjnNqiZv86isjqJW9wXffR0pwTAaHOZmv37Rn\\\\\\\/zouJB9WGHOQEjbVlK\\\\\\\/VZtvknL9WAUiZ9IaTZWahW5uswy+MDML5DZBe4HpGU8ztBDrL26+iExNkPrceM1nx5QlSM0ciDLVktbFyVeqKKCx3u5mCvpkRhgQ8m2wMGHue2+rLhH9zR2daXtOGjcaGFZurBtDyQ2N6I+Xq+lGaJzha+H34qSvmV9yHZoh9JEHTKZ8lTIVgRWdsPQa1OOtXi7G5W+xeX8vHoaaOtNUjv8pS5e46nGUaazX2iQtiS5ZV7qq4v3pFnqfSWiDziEkkvc5Dk6AjXDFbaD7+AhsAUN7qHuI9lIRyaHU2JiVAIeT09F22s8VG3d2LPvZssl0hQfdYOMu2WcmyqINn5CGvPY3raDb5RP\\\\\\\/GPS\\\\\\\/QTMXR3c9ZW+aSkN6PL608G4AOEJncIp47uI8LCF72l+DmKs\\\\\\\/HFhqFzxFp\\\\\\\/KUTy7fyqsfuJ0O5eJeRZznU5creRfUh4Yh+eOXyMbJ68NLYzOCFE+mmTJsxRe+x1FxmKrX4gCh+ZuGxn8aqMfAv+w1CR6YkmOlO+NrRhXR\\\\\\\/9hrjSx5RUfjA9nBaz4lq4Z6Elq0OCbfzu9DXNI1krqKCMMTD+jgC6ziuhIxgwDDLWuiBhR8x35ulHdMbZfiNHHLbGvZURFpkID5\\\\\\\/mUDXrm3kFeyIkbxs0bp5cA3y\\\\\\\/vm8DflkvijnPFOeizgDEqjt5tboLev2hAVJva8i8Ws\\\\\\\/7ZaUpyKS2BdNvZ0hrBq1QPH3qC6Rpo49sHH7tAH2KdBbHEaPQAfSCSiVOkjEnmbLaHhm1aMZY+F4jHrfbLw90kUmoeqqKttYSj8y7Gp9yBEHT6PlMhFpI27GvjuoKlzdTO\\\\\\\/rr7aCjWiEKwNT2CP1JMWYY+SKTWaXClwPRprzbrD\\\\\\\/QiMqKeIdSO4B+VojWenh+rMOd1y2IM8n6+ls3fh\\\\\\\/4BfRRuRxAf1g3EErU6\\\\\\\/HR4L1mMUM6UlKdj\\\\\\\/soweT1PLKPG0XhCTUeZORO\\\\\\\/\\\\\\\/9LtxPfLgVHDL+49sT8kBdiUMEPQGq7XSEAfltlxf5lVtI3gDiT0HSCV\\\\\\\/ePUmkkXJb+rTV\\\\\\\/gcocoeE54jBHCtjTHwq6f17pK+joc62Fg3OEyam+U0eFn2011Pi5y4FSva+YwxOKgeSXzBo3Ts3RsmCntkR+RVd5KSPw+eKTr+IBSMO2vC33SPRDJuMT8ghstI0L1gg2RlR1MjLLkktpNx9a\\\\\\\/vRKLA58clx+FU0\\\\\\\/Mg+wN78HznPRRjXCBfDDbSUjgQKy7Ksuom\\\\\\\/Qs0CRneL8kz\\\\\\\/Rtr9bn0qExqlrjvjeZtRE3ru\\\\\\\/dhJdq2ZZpQ3sVv3j6BlcGwL4dxccbjq\\\\\\\/3QLz263fRzYcPyD8mDHLkjwDUWjwMusLtix3jgBfcLrabQhjsQdyEVhoQ48cSthyg+27zzkuAtP4A1oBZ4GF0rZmrR885YDEUx7MRKx61X1Nuy0DrlZYLHkOYDhVeEw4nxSVkINZUxwFy\\\\\\\/BQ9RZnaHY\\\\\\\/IbQCJvbmqn5uUkQ+U5bqP9uVmHnc5QL+v8txcuwagJNQVDHsj1B0NMp8oF+j0KYsL77X2Il8zVIFXTX9USP82p22ULFumdoFgbuYxGNF4G3ugoQv1GFUQf2QuWGMK9D02aC5weJrrjew61Is5pxe\\\\\\\/jQThYSvivWdCRBETlWdOYqzNDRYuYSlF3EdzPJxK2iypFxrjuxgA+rSmhJXYR7CRV1uG57fSmCqhOt7Z+NwGBmSTQ1uVe6Lby3vc6Ub4ukY0JEpSejCpI1gKlxj9c95I+getM8xm6qyggtFbcG2wcr9BSLg9PYmh598xb+WgcCNp2IzEW5t0Iav0dIe7CB\\\\\\\/PvCioHlRNEgm1jgyUg1EBVoulqRDmqeCkIeh1d5dLZZBbIYzrKLz7yzPr081E6Z8aA9vQJf5kRz9wKIB3piC2Tg55FKRx91tFhRSXSD4H24M8SR1pVoSKrnUR8qSTw2ycKdcS+Xnn9HeNX3to+nuCytlfqYdoXwlZWbmcYKPYlIrfcpcvIK6LOZstcT1cVD+RM4NEny9oYwgfdE\\\\\\\/tdDx7fW8+gOmlXM3FJt5yoX\\\\\\\/WHN9OmFn6ULjiwwm8DZUnpPDRK66sJwhG07Yrkb8MUw5gnouS9mJTBhho489zMTVh\\\\\\\/aSr7dUKtLz\\\\\\\/iGuUPcAMNdJFfDXP6B6iBpwuDM7qPSh8XojL995PquhBCGKomiCz0eQ\\\\\\\/bxJipwc8ftS+nu9rlTuQwPAa2e2vAr3sPLC3N56KS5dj4vB09DTzySV5Uh1ZQt0arkiFM5Bn4ZS0BNQKgno\\\\\\\/REDrjp1dLZ6tuRbF6zsqC9CH2un2rmoK9pLfBWskGvK2Ah9koqo0pw2RPTjuI7Il9DY2FZf3FBrc81bD6N5BOKSZmuHSKWtFI2yRoMuf8ZpP4f7IyXpn8B+6eWsh9kOOn8rCu\\\\\\\/VUCizEl0Q0xwttAydMfJ+qh4WL0pheKPHzSL+6FGF4rZVKis2IQcG+lhtwFZiELbkRSWSdYkhdY5nrZBVfOYYswk7gMPKneSVDXD5Ej606E9lOSflnaf1mVQ3UlCbcfo0wgHLqEd+QalY8ZIOZVSv6mK5MgGS5Mlekt\\\\\\\/oba1XtdMpd5CRak5RACqpJyE9mkx\\\\\\\/0HnYlHWPJmARn3Yg5suZ0SUN3mWsTatyLo9+zx5valWDAF\\\\\\\/mS97rFqbrSMYrFMNt6ISY+bNSF0OXD2gZkwRchFq52urA3o1BV9wusfa5qVZI6tuhUq7GydQCUtLqM8bnd0dHKyu1Z31Jz1rGFpCDo7x35xK5p0NwgGb3NE\\\\\\\/ZpEN7Lv5G1nTqNT+R+PDjVmb45YqFuqDQagK80IgxtKlkkve3NR+qpnv91pN85FgOahy9\\\\\\\/V69Oqud8DWkDW+IfKZv85hw7U6xkKkKN0SvAIkCt16pDsvCNET8q0gFkEhaWIwqfxmgZWix+T+QzaRaiL20okzI0LHrtCPBr89ZVlPWBOhkNykI4lUkIjfAZVq5SLukwQ9yL8DTUmxxxdK8FRWmk4GMPHRkpdATgk1GGj5Hpw5k+TVozVGcUpSZSXYgDv6BHiH0Sb4TfxP0DVRraMrWMQDizEDv2taxze17onGgXOoeqwU0YEU4pY8CkdBPqxFYAthPgEurp3bjCR29XBSO\\\\\\\/4PLycNTRSFBXGGmn9lOBF14Nt7BfSfbZowx0F5aeguuqCa9yD3at7sTXkhjefJtS0SXtK0aCE8Lc7plsZt7py7bDdLm8vxaIaMuHCxtKNVUHOuafsJ9XtoFawgFDpji6trPkgPq4YD4TQKJTLpm\\\\\\\/kJ\\\\\\\/yw57Xju\\\\\\\/vcHFN9YvDDz3Q5fP1s2qeIBg7c3xfRLYdhrYsJSJNLbWguXSAbRoD9yiGZ83IWsG6gM2tdUikxTzWHUnZtSa6TLSvXX9dB6xvBTNH7WLAjmuJxTeDtYfcWlmv7apyrK5fhXrZsBFGdHNjqq54g23Bi5ajoY6wX8laRc9U1RXs1l7PiiAnFaJiLbgqCOdqKGnqXRdTcQknhJk28zcDzqexYOnh47gxG2LLKoRV2zOV82WYYxBNxONk99dQ5ZyTo2iucdwcGFcUyVI5aM\\\\\\\/23H0mubkb9dfw4UveW8idH951GLW4y7x4B1QPD7Tzwkx0jxh8DocQ2ThSdcDyPZqjv6qtXOVj9TgWePhDz5BLVLkbRo\\\\\\\/u3\\\\\\\/mXs\\\\\\\/K6z4WWpGyEaQKnGtKJjmHan7AfiKCNZLn\\\\\\\/QOaWi4ibBQoVAv\\\\\\\/iU83eXULsaVRW8LKBQCZJglTGsJVX6vBqYCOCiJzJL9emTGyF1F8+1\\\\\\\/CYB6eK80iAj+UL7LOBH9lU\\\\\\\/e766sNT44vLDstreHnHwMF5Fm92oYL1\\\\\\\/4iVzADwg0\\\\\\\/cKkB1wVQ9gPX\\\\\\\/pFBo8444jckgNQt6PJwsD0VcuXCZ0kyxBAF8seoajHn9QxxgvMFigX2YDmVXoTeOG0CEF2bfpLj4vcGy+u7pkcCaA4ltMes\\\\\\\/qHYdQqzLNtvjpT6ztCGc5gL4nM7Tc\\\\\\\/wgOewwottQKHuBWfpulkCM4wNJdAnG0ciXbDG8Vo9AzGjG7rdS\\\\\\\/5IXnCiviM8o0qB3h+LpoOUHD1wg2l\\\\\\\/e8Twhm+APmKmjchYtT3WVEphxEPo4MqJSmzQiYOFROTC6+6IfMbmzoOIkSPslsAmI6uOkyE+IBB8wuJHrjIiY1PUNv1N0yQVm0O5PLiRzQTOXRrZAw6EfcTTcC2kx9fcNJcwoe5ECvedk38aLFamYv1ogwwp2nMIiBXcSsUf5VC1kbDsSkx4XOG+CaccSX46HJsrPbKxiqXABSEVAMaA3+a3wbFSjFBWcB71ZqpR6As\\\\\\\/J6GVrsOZ4oMADxcNOUFAQ6EIk+AR2n6tAxMy73CII7peDi3hbeVLsNatbh8O52KF\\\\\\\/G\\\\\\\/GMsUKUpvZKb04FDM463Q6RtNi83YQ3wp\\\\\\\/LnKmIxuBSXCUxRBd4NBHK3t4\\\\\\\/zRfN1wI1MqoH4Hi85ByWw1FRlj\\\\\\\/IuiZDj0\\\\\\\/jQG1VW8UX9SUrKWM8huYCNzXsT74l+NJ3f4qOeawkwiS3un9eHSj6yDGODh772g8XO19ZQgg0E88wOicjD3xTHLe9nNgZAR0bVbhV4cee1DWak3Y\\\\\\\/xUEA4LhZ\\\\\\\/mONqrtj6nSthC6Wf3BxjCbfq50qD57uATBNkkMgRall3icG6s+eAPGl6dT4CWqVRsBT8BDMCrVtkwZWSOGT9\\\\\\\/avEyA\\\\\\\/mqN9Qi5HBJ9nlS4LfkicGOJhpr7GXXG+IWJrIOGD2Ehd1ut7Wn0QGd87BpNg04AwqySmt1XrMbD8hIKoiAXssYBGHI7Ps0hjL0FXysl7dkH7kjb\\\\\\\/24cYagNwR5zrNdUoXORDAJ1eyGlowSsZv5SVkI3PrfROROQv5nXVaTt7jxoyJtPud2N0dX5y4bMFSmATU2OL8puGSkEEHdcRT3irY8sRGFCmCnKmeqMUmGQO349MpE6Hcz\\\\\\\/S9wXgkmPaP8NR1oXBRPGglZhWTOz\\\\\\\/W2NADCo3Xsoicz0G6Efc9ux4OW11SdZVi5eKX1Ij2b\\\\\\\/4cChgp3u2SdzOV8SYir3A4SrAOVjHtmEncQ8Mtdx9\\\\\\\/MTR1ncoBX1m8KAA0ULCkG6Bduqz5nF7DL0GO4dq56ysR0ZFgPABn88PfwhbJya6L4STfK96gkIvcccQ26SA\\\\\\\/wwh6ng+\\\\\\\/tWIuKdvKKcGw8VQuIXA3yWpYR1UTWH4OPPMGPimlTLV\\\\\\\/D+g5Cfj\\\\\\\/DS1hknIbV3moEN3jM72EW13e8izHq9YAnYH9kfrptGEboxvsnKaE7ZAvUqHfekV\\\\\\\/UcV1XLQJS4TGqVbmtgQIU2R6IiGEMnumAQZSGRKTFyJL0RgtDI74JJR9s3dRUozcKiVFGf\\\\\\\/U2jLEfo7BAmYNXJ0h44MW8Yh4x+H0W0DXOMi36Lc3gVbwG4Chjy\\\\\\\/FeImHt3p46RNghFDew0ZT5XkibJ9RnBxSK6d6RtEZrKeRDjWX3ffLzJpGlpGEs+dnyObNx9nkNoBujf91SoV5rIgtB8o6fzX2CKIZf8DrArWo+ZbV9SaL1ihLhFyLBT4aeKNsvBF4XrLR0OyQPdgwLdsGrqXUrc86bnFaWPamtHgkW42NmhYlSFPZU38oUSLC6kgbynlUlMwmRIkPFXueU6WnybAtCCcjED2KrvyESLgNRUEfmsVDjcD9xefdq+Z5gfqlfQ2\\\\\\\/4RcPmw13IHL3\\\\\\\/UUKv3zcMXA4slhTV8kAurUWbopp\\\\\\\/gCB9A7+2q5luLNThaEdq9Qj6YPSeOTGMBj0L6bE2wXvHEtVohEQy+k3ZdBfMrUl1Lw\\\\\\\/aS1VR8pfBXFeQZAoWTmy5HQWfv0SPpWj0DJ4TLQt15OiN0JP6WcBzP6fXzXsTCaVvb2nxTnO9s8nMI+zwvjAZ\\\\\\\/MOm3LMjSl3zg7uPn8O1tSBvO2tHuD6ogFKi48fhvNuKj5EFMFAEDDgRQq9I6XOSkH16yNm+oXtG53ULtN6RHWIuzeXWnngV25nvDcCZD0MVLuC0YCzxtkXibljUAwrC8etQsh1gMfJG7ZmmtKvBwC3HXEY\\\\\\\/kcE\\\\\\\/cd+PoJ2sSx1zVp2g1lniF5pay5E2iOze+VFuWoXeuiSsdnA\\\\\\\/0Kg8v6Yxg0u5iOPMKco59L+EYObIF3rs5+CbkPto1PRILiTiMlFys4y7oUynWpm9dObt8y1GChjOl60ZSku1MaeSF8SQJMK2yijlCWvoczuk26A\\\\\\\/bKatlPrvZftQHRO9saXtOwjVnlsXsq0eJ1MrtfYtxmbG3fujDJ+tmsAyq5S8fpxblAfcRo45IpCs5LxeMBi4cKYlt2sv00TrzQqS0iYm4+4iTwRmAaoEMFkbrgTwMv2EHA4eVqXvJ2osNmgNq5roBs+9IzhIKrcZEPue5nW7YetJyMKJbwEyRk8VWOiNOBBRwGAo0dOw4ddjHcuXy\\\\\\\/vqSp\\\\\\\/1nmFClCJqmc4W\\\\\\\/CHBI4pPFlxRJRkbvMSY6JS7uFRE1rh8vzHXMsfDREek2xuiIdi0T0YvhMwRGq29WUkP\\\\\\\/\\\\\\\/FvDs3QIS02lJuc\\\\\\\/x6VFMIhwHE49cQHuUy8EXIoXp8xyQHTGuXPHQ+MHOwkF\\\\\\\/yV35gKalBGPE7SfoKrAH8OkTL\\\\\\\/O05Y8o1LU037kCu8FNl7LVHIH9EYmr800Sb49\\\\\\\/XVSmuMEvrlVIA\\\\\\\/gfj3KgB\\\\\\\/h828V5VNS\\\\\\\/ppBoC5TJR13OtYMQqkEXMUYQyScIYo8RUuRIFlOAxhIH+nL13m4AV9PxDub2nKNlvU7T5GXpByEahqN7y\\\\\\\/oGkVNKIPxWGIXvzS8nK2UZHiC3Kx\\\\\\\/2MgIxKUie+kE1n60A\\\\\\\/J39QaIBLsHBeutbMrAsFOxtUKr\\\\\\\/Jck8ljn0LHw9wN82WdjyrOkFnd\\\\\\\/xwKz62G7cfBS+\\\\\\\/UclPyI+FKM7kkz86QdveXFkKMdziWuZGjqx2lhf5Ld+i2PU9hCcAIviRMOXCmgj6ZYQIFmwR+3eIS\\\\\\\/FdDEnVTH78G0w7d55cItVy2yK3qrkIi\\\\\\\/kK5ZpozN\\\\\\\/43VGjlLcReoCYVerItSugiEBM3luxiFmPgXpoqV8TpzMhuk1T1qEDDkW8UIpCOhtNph9udTpDhDczY4yaETyaYU7IbBNY4YF2fNJI164UrFdX\\\\\\\/HnUAeua\\\\\\\/Y\\\\\\\/EDMVTQM11nSbszU7mTBF8Ynftu9A39r5R9flNNOITbmwzkl5IWZ0y\\\\\\\/yB5yUo6XE23oIuEWDiuY1WigDaQKZ4IMdP1POyidPATMDQOqr4FUPxvi+KzZpC3S9wTmjpoWvptPccZrtoFflxcUfCLvgkSY3KDh3k4bc+J93ctysoUKWAtlb6+nNvAyob1nfzGxCiHwkS2qCoB9WmjZgRxH9OToPa0bRJuLTsgJyNdfmRSCX+t8Z7St1lk7OHYpadV0gMuYfBAE3hZG4sgSZnVOHzHL6m1EiygjgQ4Q2t6a62V+1z0t\\\\\\\/14DI8Abwfdne9E69BRao+yiaivvClK2CZqqF53vhAVb6MZAf7LH+2\\\\\\\/In6j2agWjXX59tyM1dfbEL7+yk6LkXbK2+UzVSEFe2tWA3qlOmE9muqMuWljy5S7U\\\\\\\/R6ndl0hsfo09J2ehI8axj2TnVEkI5MEb2g3uDhwJSAIVPRnJZp6hVxOlP\\\\\\\/Mj7E\\\\\\\/AOeJWmmBxrWjtzzWNb75n\\\\\\\/qICYeT2GH9nxRb\\\\\\\/WUBMkX0wpuq1LFUd+EsFCRwihwUu\\\\\\\/hfZGx039h5BPTG9W7VP1ZoqY4MTF\\\\\\\/FXHK\\\\\\\/xddm6Zoc2J8ITgyFMgaaMo8iFO1yTNE\\\\\\\/uIbcmlRw0v5LgnNo8ZMwCB\\\\\\\/kX0n9Wy1IxDWA\\\\\\\/q45HcQwHFwxI09vCKTj3qQKZZWKQ9W8Vhe\\\\\\\/h3vJWurYvVlyI1sd6J+N2cLxtOCgwTieO94R29kkcFTbV1y\\\\\\\/NKq3stYA+ejlBPoD9U0NcGvIr8nWC4qQhmw1Cj8vowqI0HKoTKKkCrpWl8C3QIJSXbckxagzFhAhsOS18qFf2uG4o3A0JFEpwIU1aWi8grT+bUVI8VhNkSPIGCjLCGpeyeijVskEy3R64bh7ZGBbHDGP9XoYVS9hesYDb+\\\\\\\/J4AQsunJOP038NaOxNThkr9KPZBlLDI9UUHyfBy7PX8FxGypDsYyErOx2xoHB8PfCcklfQKW\\\\\\\/WhhRBCHkvIuIdoH4iIm3Y\\\\\\\/JIepQcMxHkVFNn+xsHE5QwcD72xV1ko5ulzN3CxbdXPbcSD\\\\\\\/65cxk3GH0+r3GqywJrhN+c886aAV+54\\\\\\\/uErq\\\\\\\/vyflAT34FyI8YctRfFicQKqQ+sMkSRZlULFzAa7G4YGiV\\\\\\\/0ccMahPaCI3ELhAXhsfDFr8A3uQFkcWOuPybh5+hgS12z6B\\\\\\\/A9stF4ALF9tgEFRugnXh+\\\\\\\/SpThjGwmiQg9QBO542yLGJXLZouwnE2aFzJHyPtS2c7Buco1rivkqtd5YEZVMoRAPA7WRm+OA6s6EeSSbyRrpu3wvAYnPIIVYr0epxG9c+YQSzyj3QjTVGKBKX97YZMj7ST2Hd8j0IXYGdLyN\\\\\\\/7UcvYcc4PSJglDvXtsmv8EXm6N+ceMusyjmFMhxUHyQJP2NGD3nZQoGzogCqY\\\\\\\/dIGnTO\\\\\\\/alRPXsKH03jyMFSB6UOFbgYPczpzK7a3tY5HR1JWtF8YHLr1AVwGrwHcoTXNgTkNT9HilSlQYLOtFyt6rdgms9MZpyTZsDaHs\\\\\\\/z2juNHywenGoTkiXZZjA0gmbbX7B8WT+aauJLIxOl1Bg1W89GmL1KRfXS\\\\\\\/eDFw\\\\\\\/E2WtWimRP26q4t76CO2DHxHlBByTJRmUjMcW2Jbnfu4PzxLhFcsDqcbFbNjWlwTCdEtzbS2zeoElsIG2imSjJxQEyohKRvvFZZO8WMAWLmks6XkA188YULErId2puUielAk7W4S6mBpFCLKZTTQMCQ+VBAeUCAYnI3QQoUmcC6nlaQS2u8SM\\\\\\\/6qipwcGO9pTv179o5bHN\\\\\\\/pfcPWnqxq9s35J\\\\\\\/DlnaxZaaUQTMVvaLpu2GMAxuS4g1ZUtVsSsJbxLAx738RMSTAWieGWt5mFfV38\\\\\\\/A1nk6kGWG7qnWum7S3pOPNMS++dktfAWRkEDFtOaXf2+K9sLAIaVEIXwelMM1U1ZMT3I5Wnj9ZYSOzbT9G3yOjGfyqXDpgCqWMP069iGphuKpwZtYjZykV6zyVMEW\\\\\\\/vivlNyxIKcUuoimZlSk7QL1OdClfEDJAX+c8mk413a\\\\\\\/1hqBBPjLB+DAyg1QXMy2LTZ\\\\\\\/E1PKDguz7KCrGp8ugi5U7gdFeoj4brp1GJQveD25hxIimSyXMzGbLLDhUNAwK34KDQ8YSOoyHP68ydAwq+QAiO11QA0yV\\\\\\\/7xYXn9gr7KP140TO9BaRNj6EG+OYaKvllDScZcyzGpW\\\\\\\/s76yv4VlSEeP2L5Qfd1TN3qzxQEfJW86kmVdx\\\\\\\/eVwGzl7bBKpNPXBS\\\\\\\/fr7j7kpxG15HOn41S3dqylDzGGVrDDq1mVrJoA1b00YzQ3cXyI28xh5H4vUB\\\\\\\/qDNwBm8UCYjts\\\\\\\/e30jbpannvBimPWuDgs+c+pNtGnIpoFMTuAXDX4\\\\\\\/R9dHeU9NPpSRiE3xoxAhfi1aWlIKVrU6ldU6gkbbkB5IqCK3TjEuyomTX2CGsm6NYgt+ub8wuW+pC9leRUee7\\\\\\\/Ti9pO+1HUlbnPHZA+rnA2CuEkn5rAjK+VjjRZOK\\\\\\\/iYRmHf4ln8XZA8rXEWQY\\\\\\\/JXxpB4MNfYbSmna9YqwE0U3bcTxlTOxwt0+DFfiBTxRuDKTqBAPmEPDzGkRIFS4iNQziklpssANykRiAtW+o99+FW1B0vNe1eQI+7cVMfn0IANjswESuq0BFQ+8fjLuYHHzjGJVzoP1wjhjGHf\\\\\\\/bR7jhv1BUxYTW3rHUw92ol+9HoD6+zdwKfg5RjKIX5Pw1qdLzAV7a3jbrCmMBbrFXL+\\\\\\\/62+qI41p1yfgIt1WlmQzK+eJf2h0u2gGw5E9fpTRnOGNr12ro9mlVlkS5\\\\\\\/8XWaJTUY+aHsGwHsQfVj0OuulPaNG\\\\\\\/HIyVY3BIdovVVOKbHK6NF+UTheaQVHdjZQd6qO2SYgrROIRkPTKkNs\\\\\\\/H6L0QixJuMEkgW4TZ1+jxcmMWkugtGFJKS\\\\\\\/TDzmRmn9AMvbrpSTIDq6WdrD0Hck0mj+uatCYu+L0ZQHwPBAO6u\\\\\\\/iCQIuYwHTb20T6uz6QEV2Fl7FRXvPMOs\\\\\\\/HFDZGTYPU9QrkBTEG6s\\\\\\\/q9ierT7REF2R88+8Y1bqkaPHx7VbRmyuFK6CUzl2CDvJKHj7r+velHauGLPnKrWCy+vbPPPPZY0NchczFPE3AjT4V0Ii93QmhJBug+9Gc\\\\\\\/7R4RGzxKgaVUCybqCv5Wdb8HPQR\\\\\\\/Ex5EszXGrlQClNw8ixjt1C9QAm\\\\\\\/MZ6qEdErZep1IW2L7o\\\\\\\/b82X6CO1ey6Z1QyHMShpQTP\\\\\\\/AiB4VOkPHIbPtzfXEpayZ0zcu+3XuGvmfGJlyKj2QIndFtHw8nn+mRRRpIPv231\\\\\\\/\\\\\\\/RMXz\\\\\\\/QmGmWmBrwV7KbcpOVLdP1N\\\\\\\/uO1hR4tvUhbYfC0RZRC84GbekYZigBiB1adPokpMwuTKrpJzO8ahWOu8ceiwwosFw0DGEqGm5J\\\\\\\/kbpScXVkQ6I1ZrKF54xAsU\\\\\\\/YTPufrXhp\\\\\\\/0QweKgFbfvl3XiBuQL7i3XiZPYv6SCNSn\\\\\\\/Crl4gdrGVBPyEkCKz6hDIojjOVCceS4Mjov0+G0LZ049owoQw3ySce9VJPNxSN\\\\\\\/1vhTYqVFRQY8hkbAfATLnbf9Wg6P5lSasx5gM9nP5okSyjh2kAYcq97noz6v0MSv8RIRCYvxUCVl\\\\\\\/Hao2dsjskE4KukwOMWxqGOKfhpAQDM4CLSu6RGX45bee+3Aik89qx7+GLxE7WVaUImabAW5BEOO6+5XkA5ki\\\\\\\/dj5NiZwGBMfoXGZJjiUW+WqO3ZnimiHdJwZQ0iqAWA8CYNckFgP5wQb9wMyfToYj1O\\\\\\\/AkAQ4p+l8OpjUgMPdmaIJ3SODEmONNJAS6lm49z7roYnlXC\\\\\\\/hNASCPVvLO9goUkI1AI9KBydJGC69yoK1StwuJYpe0ce58j6tUrofoU30G70iMQK24x8hk6hgYtRwDSu9\\\\\\\/2yrQ1qSvQwGZxZYkI4lINpAW9AVXyjuaE0671Sv8l8vADbqM\\\\\\\/9AW7inIWx8ZF8eMEAaNz\\\\\\\/bM9tB3qXNWjNsCpyDp+zs5F3OGo\\\\\\\/WY6J8DfQXjjdgclMnQFVpnuwey23uBWI6HEe72C0MYiRnZOLDkdeaoW+SY1gCXk7ILe2ajWozbe4CfJWC0rNL4qNUX0JDxojuIRtZSmIly4x+If8LpfOR1n1NDyr2QJvxQxOkIYs8truz3UrPMppGxwc6Uq1qYGv1q5rzk2TImnZznOD02cJbhDhT\\\\\\\/bPf7EE9TzEu6fYH\\\\\\\/\\\\\\\/BPDqCUUAHsR0ufNdmi\\\\\\\/dc7+eZ3lVP16zskgQJ3s3aJ5itgsQVlVKaPgGbJqezhO8qkLR8mitwHaqQ5n7wuLlScmAL\\\\\\\/YZsnwexFWHXu9dlVfo1FsMZdxWBoqCkujTIDvMka4SKuNjm90m73E4O0maaPVkfE3P3ayyobQdi8BJ6hMSzRGHGaegfmtV\\\\\\\/gDIxX96HVxy9j0cNt4I2wygeFEjr1qn7gaLgN5vbBoqPSGvx1q+7cT8JiFCEhkSZDlcSH9AEOKe7obHyifC07hd2ZX\\\\\\\/gd2W70lodtrTQoLvGrpEHvmIaNbJ63Q\\\\\\\/Xdvjy\\\\\\\/33DMyEm2LTL4iZU+Vhh7XdQEd1qPbTBb4hmmrvLv5TkbBzqMp8TUZ5nmhuTzl7NS2wRaVdsRXDGp6wTCbqkI080dX7zEqCs\\\\\\\/lKIH4Q2BFWDSuIfz+dwFQHJjCtxp4W3RqGtEC9Wk51eIBTk4KIW2YnofNbL+BF+kkqZi6gQ6qWL0WXzsVHpFkoW4bRrEkoR3aTm64BSXf9QiONFvOcBAGCDVJo3\\\\\\\/Rbwh9P6oZAV2MqIaNcVCOnbGmGKjVhua2v1H7RR\\\\\\\/QX9qEPvOJI+TdO8eBaDQUrhnDFu9q9nwSOxhxToXXnbKjZltqPD3MgXpNpQo2J3aHTUFwLZ1SKpIvm0V403bYEFI5x7hvmuCWhhXtjCmoICsOTIZ8+u0taTb73A2SYInImJjoXDLUqaoKdlhwkGtEIAF3GlHDRyBrIVVGMMDnEAp1FM221DxxDPxct21CAbJUMyUZfG+luZ7ErqJEi8SrtV6N1v6\\\\\\\/IKpTIbIbeC2fCG4tdjKL\\\\\\\/YXxnwm0L0ZfwpMaGOg6USsLmXLd2oQFPrmJSWbGPZ8k\\\\\\\/6H+lIzIYMLgg2OKA08e2IoXVEWoT02gfAODvfVv6oQ\\\\\\\/eMp7xvCmTHi5kLR5lY\\\\\\\/HBx5YxaVHvER8a5Dsxv7\\\\\\\/3cNWcjzVrhOB\\\\\\\/XuaRWe8SMJxybYVc43zJuiWewq8cWnT2qfG31IAQPC+dqcTnXRtyb\\\\\\\/vUbL7RVhpqbuFeAB9V9NUAeluw2V2x9sEpCBn123HSt0G\\\\\\\/AF87GvzCmg1IV5CHf0LvpgKWTRXpc43FWTkv8GTwcT59WNU4D1tfeHRYXKgqRYSbRDRdzzR+kh+9puKUXMl7LJtK1EmRoxxuqtJBhPizRXiBXFlHSmGvDyQzUxKDg7ONHOQ5czg8yKWttWKktnyN7\\\\\\\/0m6nV0+L22bRbEN7AWYhj9mm\\\\\\\/nQFuarqnLBZfC\\\\\\\/E8KUl99UQCdvXRP3m\\\\\\\/kCxcbd2aacfsMRnqgaFbaRBFAtma2vOp9s5b3NX2\\\\\\\/p1+2qWZO\\\\\\\/NzRzxnfRXu3M11w1fEI3AbNj7ZKVLJkrAcwSlLy6PYOkgYWmbLWMmqTTzptZWraFxA9CNpxUprex9IZZf+Di\\\\\\\/28\\\\\\\/SprOiUx2KRrejHRSOTtHclSqFXYUiRVRjxt6DakQm37Fy0I\\\\\\\/hSQupLS2ebCSG3xzHW3SaE9qb+XLdoxcBD7\\\\\\\/MK76XcNCoGEkFVdqWtqiCR5CUZ5nRIsNDBvC6CYKuVP6epAGFtbqnneJumF+i8vT8wt7DIP2UQZwPSIw77oW9tzGySi896Hc5H2hCWPrUMIkKAToOK15UxNJeN+X3WreVMyqjDeKJqB09znlfaWKuIdMwxpI5ZjXM6Q0pGU8UtxEIFRLj7pwMsE4eJB1qfoY3hGLmqqnVmLfBkeMfrdLJjawnj7RlNg6obOHG7tIEz7shUylsQX8KJBLh6sk9ndBHqBij8hxTvdaJH7EYJYqlI1i5od1+cRIhfG1qo2O7LmP5XqpN\\\\\\\/eQ0fBhzKVZDTyvnIRnGuVN6Fgc4cCaXwel8ltKXjDmdjL1fB6utmIDgLkJZM\\\\\\\/ouKGWXRiPt45UxaptY4rwfShJ6bSNVNnfU1R8gRE2+kmJyS5svAodJxJI9VmMP0tlshLaUNNV\\\\\\\/xkFYxqi7RKPCgkxE4AQkriE4JMYWHCiu78feIR+oZPQk4vep\\\\\\\/PrjDPNLtMJhm6ZGsPYLSgxsPvHFNIJ+5v6ENic78oP7uqozYnOBn+QC\\\\\\\/+CfHcajGpJ0Bjl6EiYQkt5cq15hdM14UOueeNsI2Hi\\\\\\\/S5HqzQ840mSNo3RXeMCvlHwugO8yef2ejCnvyDLr+Ogjy1Gg3NHp\\\\\\\/KBav9dO1Z6EmyAhpNaCeZ\\\\\\\/Qd1F3XGp\\\\\\\/diJHnFM1vpcRNdfmCnvA4uYNGWJ+wECQhlAt6plY9\\\\\\\/OEPtkSqTTQ+Ie5EXaBTEaSUutKH4FusOwoE+QDOzoD1Id56rwrULiyWGEowFe3o+3JhH6WBha+Q4nKwJZMfAWhfA9NQf5YGuOzOw8Sj51f+2ATAGkDj81q\\\\\\\/VL+MHHa96phCPrgey+XGexDJyElf9xyB4y9ZkUARfCxg1QNRKYydotqcfAc8MNdxKoML\\\\\\\/jnwu5OiXCSlKSzQqYrxbuS6MW6cGAPDOs5yi\\\\\\\/PrcMU5sm6I1VGpM2qpF0WR42q0xh\\\\\\\/BnDnPq1ITu52+zJktjW2r8S1Xmt506PgqQGywKzgR0vNszml93jTdwLmvZvNXN93gORzBlz+QsbNOXlGoW\\\\\\\/0iN6yYYDV8qj43VSXyHT4fHzeaehp6sbgvG2Sk77VxcjKJ0V8Jnx\\\\\\\/NOeDkkUNDcPrO+OnIQZ3Z3PHE9uq9LRuHdYmAK7Ma7nHoFMR6Lyt1XdqGjfkl8Mg3uRMuxzzxzHF1XCUbzzFdzbNSykafNZhR+9N\\\\\\\/UqUuiNUlb7Z1mDjb5w1UopVQIiKAr7NHIPaihHTJlg3FKDODKywIYZCmt0YSSGLDuH745JGUkaBd4lD0zRUXVm5Mc1AJ+h7HmhYs9XkO5WQ6XjD+9UyUkKKLZIY7V0NxKPUCw0EFyYQENuQGJo9M3\\\\\\\/jc1qKke2+RFWTEk0v8WdWa8QRvtk9KHzqZ\\\\\\\/1qkNtO2FcCZI91AMgryf3CH\\\\\\\/4oncnezIe5zg8OgiEYp75Z2X4TwQQkgD8Vuc3qXMJH4NwyvpUXwmBcJaqvlyOoZNOvkcKvYW5ygjr1dggaHQiL\\\\\\\/ZPrDefLWnh9+uB1dhKP9n9JlCt3zKSQ8zpMlYb+Yh2\\\\\\\/NtlaUJ54289G0f9Dadw8n24QrMRhpzghJ\\\\\\\/WTEAixxjZYd9z1rMvbdL0Ft6ueX7M\\\\\\\/L7iwcu8BUQFtIJPXaTtMIHtkESwf55orSOkt2ob632cEN9U547ieWAY3kzBa8hb8AwqLPUwHDNmk78WUKcu7iDG+OvMalFYCi9WeIaNPhpBZVADYtPKkej5Tjk10q0Wdkysgc\\\\\\\/4MxRSZE5InB4qBa\\\\\\\/GzSe5GnqF34H+06JNHAEFSK5UNkvgc3MZ9iccwS60yH0JcTNMMg\\\\\\\/8+R1L1o1x3W7StM1GN\\\\\\\/1KKG9vo1OaPk0aljtE927hfpx3pxKfisn6W8vKSQ1Hqubljr3lP1Xv0C9uLkm2kPJLwNb62atY3Th8zkOJiSalULiDc\\\\\\\/ItyjYZU6I3BysWModp4466yiFVZDqEVOLqeCAkLY9bgM9KBl3VFWwf4sX7Eum0Xv\\\\\\\/eeZGZ5Wfvo\\\\\\\/ZkddyWezb8maHsNoQ8Kcicp928MekY5kGy\\\\\\\/iDEjNe7WLo5A0OdquB0SzQwBlNhePNI7l7b1SatB1DtkCWbCZfeAY4aK+KM9zYmD9U4qodUGZAqDkTGLZhHI0JKo7sbYWM8TAsj63Gw7J3U18CpQZVWqjIQb8ZA3aHw\\\\\\\/LVtA+6IIJSIok9xN5cSM1+CfeRlj\\\\\\\/pLexnsKkqG8UQIrP4vcevTUATIiRs23Wdt1X1+Sl8cKB7kmk\\\\\\\/2sijdn7wCTNcn1jb8v9cUc0n9xfCozje\\\\\\\/ibGJ3dKuvdmx8GqymxnlcWJL9X\\\\\\\/tRVzyyfQ3LAGoxIi20Hfv6GoRMeLOqk9ll9USdBE5UHYQqpIorYGhyWo6QGr1vGcKItPGGcrtOy38fBuCVbc\\\\\\\/SyXfJjejdh\\\\\\\/hDCmFHOdvFXZefMAxtlW5gH2qLYLa4HjcLkVcR25LzjLHmPVFKzIDF20Y4dcpweeh0voxe8Nf8BA5BeIR\\\\\\\/iVr\\\\\\\/COu+A7DbpvXJuCxdEgeGmDiPzWkz5HfTxxBwnLsMrzEpNQ23l2RSF0vn265MgN8V+KZoznkGrLDzaUALaf6QVGBSjsiifnp9iHL\\\\\\\/rzRitLBSTEgEkUOIyehJJSueVt5Y9ToY\\\\\\\/LBBUifwp5dmzPfap3xmHj2N3l8\\\\\\\/NXCASj7jjRtkgqq2kLMLgiMKN1WQkAZSsSKtirfyDQcWHjTqCbXe+agJMpAQLOT6dMA3DbYu3w1xnLQREr6bYCpyBexI2+p5+nIXiYcutOBBr65YZTh9tY06RFnz6uE9iIUD3GZxohQb88ahhAynIfapAdxlYxg1uNyRzqLb+oh8tXiPiNbaETzoYo71ONWi7wbbej8ixcfAtR\\\\\\\/Oorfr3FVbEWwZLUF6QAyiVOsAvvQVKb1i6ozLsoCftp7Mf4tnyxs0amhPU3bnVyktfG0uh4IizWgTXOyYnCGTPl20mX6DeRRUR2hk5TGa1ngEf4V5N85ipZ64dlQuLRwEMU9AxMR\\\\\\\/FkDvHMehOfk363Tu+VjvI6DHNxuiRGVWB1bBFGjh+S4nqLPlz6RJGHsEwsqVylmAu4mvEM7zmC6ZgqC4m1NPj4F68bxOSB1IxzRDSKjK\\\\\\\/i3dLEkINfasaZWak950WyfQHr3gizuEvc5R9csv2c43Mvhgr2I0NOgeP3wpsZPUiRBxrvjRkZdqENnTunNp3FnVNzalAC6z+Hu+6bnDRwCUcJ0BXV4pCqsNw84RdwZ7hvhzjmDGs3RH+Txf\\\\\\\/DtDXeb3f2u0g7DeLRaHDTTThcDSnkrx48+jRvO+Zr7eBQVL\\\\\\\/WSMK9MDgX11oJ+JRQguoR8YqWeJk9s3uSdrulWAtcGiP1F457C76h\\\\\\\/L0Stu02QOXnKJRryjYajCR1jV09YVe4xBv7LFyZxYtUDNqljmIFkGicAcEZj6Irdjpr6mgxW6bPZncP5WrFUXjpBadH7h4eIAJ8x1122a\\\\\\\/h20erehMkvMmXFpk3FPe2V\\\\\\\/XPzw3neKm9AGIVfQ3xcA4QnoXUHdXhSLIqsrAJUeRW1iVmDpc3lrIyrcz8HW9tlU9OqLA\\\\\\\/bkPF0WsE03iVPrnFhq1uwu0ButjNEGkfOrXxvk1AdXPokYX\\\\\\\/omQZKbg8fjKmG2jjenIxSy8ZIYTqNwwcojUuvKrszxwcPGBObjrPMrIZb2Fh5TcOpTqC4h8gdGhOsWkp4BR6zu4I0\\\\\\\/+GeuDmHA3UYJqWKJ5R61cqMchLEe2KGFAiHe\\\\\\\/y56W+hCQ6rOQhnfHA2qcKjDNU\\\\\\\/IY47pn3Ro33AbS0mN7z760lu4x8FH+eDsElkoq\\\\\\\/f0p\\\\\\\/6VcwFjmoQ1h+YUefMoTE9a09+piLtj1dnuMual8w5GxLXW5+lZBV19orP8j9hdpOOV34lpirmGKjzUKwMeXRv8LOxEHa5wYskE74wwjfRwvIxPESr1wHxTfVCfpyRQ8xI2++5hsPOnVZt+MqUZsA1va\\\\\\\/SearZ68YKWdoYrubhFz4cXaUvW0fnS\\\\\\\/WufqnJ+lvfoocqE1nGeMI4+hJTLBOzwUKtrUYLytEccHgzF+xW6wTqNfvop6C7qDpoJc5QlZVotOBR70mZ2unpdhm\\\\\\\/8xdLc2kB30JBhQG8BmaRuv0irqXIqvNU4\\\\\\\/EWzJLWAdS69AEUQQTrvEqNXCmxnr81nefdJmTGHa4fuvUa6cydqMFfkFMKdknhzpZX4AZ1AG8HiA4djlCXc5vfTRv7vAzeEjB4hjL9isO5tmoxFPMtVyWn42th8in0MyNRJpQzkBxYlO4epdWtwhDYDmYbFFNoCZ1JwmUV06OzKMnSeJI6kgleJXX6tyKKF3\\\\\\\/WCFx9+Cn6y6e8WfX1Y0qnMiOgJ2Ujm0c+BlJO\\\\\\\/mj0udjpbJbW0Zm9RfuV4007+pXfrDN6ch4uXo2IuVdes36eOt6TpK+1RtXrDyYN7LhVYkgGoMgbvMYCiwRS\\\\\\\/hTUcen2jWaRVtYnkTBV31XIrDSxcUrMDOrGBDbdRLhzwFrleoCGfPUDjmAI78KHuiRnbMpmfN5Rclpv4xtFCS4bdaZgeeaTTTA1r2QIowj55TXmHsMgI4hnEMu0H2NStfWKaonS7aVkEA4OeSZGoFgguSYrljQjZieZBiImbxGXdSofDEjkeWu\\\\\\\/H6rOLGjyJCaLPv7\\\\\\\/oi32n4\\\\\\\/he54hzoQid389fAa815GV5dYIBEuLYdXXm70awH917hCInol7YC8UL41VWV2jMAJNWRztXxE6DwFLQkygtvejsQkizpHrdaih9mUKNMRnmA8YBRGcK8Siu7Q7g8hUn0hQL4uEkIMCZ4dIog3XNBfmGFMASAkEeJ+Zqpk5ddyHTMZTLPEyWcGfWt+z+T1M0JEDnvjm4c9v2cYiG5YmxVc4YQINZEV1FTepZLANhLhwRdTiJ6vuuGUaOntY0hBUjTw7Ok9Xw3zfRLrm7dh4x4yXnfsdbVGIasH79nDx1JsDsf4ftzJw\\\\\\\/l9p9BPyTpF9O7KfdstQF0blsIuTUsRvQZurcykkXtxRbapS2v1Ks\\\\\\\/esogkxhGgPtLzdYfakuZSCt+1yOjFE+RMZ280SHY9hhT\\\\\\\/aw7OPBaIYk5DiLJpnHEQ7Vvu4pk6J7CdQ0hwQqGfCQfVUctO45rPdyiqvIas9GWLj9ZiAYBGptz8VP7GZvQ3TUZL5LveN2H9ZgRPRlPmk1qa6mY2y\\\\\\\/tJniYOF2sK1exmMh9JN+OEQ8F18QJBXZo+aJTrjENQLPTTfrza\\\\\\\/pn71xVAsxsSMRNXw+qvNHrG57kP3KD3t2amX6v8jaX635G9lWTOdLiK0Hi54gMkwCXzETmrKP+oOFAZyHJYqYwwnN7dn3CkERQBjR\\\\\\\/2O+hh0S01lwTGzn8FL\\\\\\\/4ZFBohXpgwEJKCPDSpuNU9BGkJ4Pb6qsl9dzrSQSWdJxuiDFBa9YQAgZA6eAMKCDVcptn7E45q3Jax1TRXF2dfixWtZL9fycR5Xbs23i+06RETUKGUpOjRWjusu8nfHyesoZqRWtMx2rvTlzxSGA1pqtknzMGT76Z\\\\\\\/InGRTqUNEvXp53QKtpUNQH1fha72DQXAvVFESeRthQwH49Vsrtv\\\\\\\/ehRZ3B+0IqxCQ5TFkmA+GeRfgTtWTvbj\\\\\\\/QSafWwtLyFUyYSggcIhcffoaIjSA2sJs3Itjw5g1djSOeNTZxwsuovL4+jNepe0I5530ZvNioxCCsy9QY8Cha2pW\\\\\\\/Ui45t7fzrVW5LXR9UHZ0E9SkupIlibrC7NAM2GYxO9IpceG7ewsa7mryG5mm6ccqfwow4rJmFbRDrmowYPfU7vlWcaSdHEh37TTriLw8XwiWmea2xQXMPpeQdht5peZsLiaGcjrK2dBY2iAP8z0LNRVrs27jPDrUnbHHy22NhMXW6HWwL7RKqKDlVQjR28LqogXDZ9FL5XIfLk3a2\\\\\\\/uGdmw4O51VkUV2hKPA7RdV\\\\\\\/dw4yTMVr4k3Z1UtSQTYN0\\\\\\\/zCCfsj94GqSDFqLTdRg1W6Ne5LkZux\\\\\\\/cHQ3SQHuViigqh\\\\\\\/TqyddcBrx2LoiDaaXH7xPuy1ltZU5TJ3qzt4vozPosIWZjxgD9XlPFGrrvlh7CNWI1tBwxLgmeW3Ur55lJHVshc2gTovjYQWZRaGzVjNFdszDDo\\\\\\\/AQdnZLQy65XfKKoEQdMNbzRxIUAqgukRv\\\\\\\/25PIIzuNmhQnY5XDJfuMTm5g40b6h8O7aCGIZrSme47ym+wXArcbOuiyBRSD48cZiTFonmPHaUJqb78mM0qocrycGXuHkzL+GumUPXgimO3gGltZhDrL\\\\\\\/JToxjW7NgSThJLPSooMd7wOdWJJ3V5R+CKyzgMtxHM57QZjgE30+kOtEddrkMhsNmwEK34NJtYcW+DTluUxc4fwsWjJ6N++EQkQgCHmV4OfKAFGR7Ecz+nGKgifb5WAsf2F5D+FtDmgKPykAONd5w\\\\\\\/qohnvBVEyYdnaJNCFCaWQs6KAx\\\\\\\/xDxSBcs6XaKyOP4qs8QPvDhDzPatiFAa31az0Nq2llLQa3R5Q9YIgKoMi60b0gcGm6eYwipQzxshqhPAD8kdljJFX1NeDlyK4Ce6L6cnG\\\\\\\/D7U59ustyRKnWohqj3w8us83rNTUAoJ4ISUz4N5Ve43hAts0vbdnxXlUP01ond4TJ1QdnNZNCLbmWsi6UYnAryCFthtxx\\\\\\\/GW9bs5TlWCYl7L1begn0vk9Ipox2+pc9E7vbzAgTM4F4SA1A3M49X72tz26EkYpc7+Y2Nqh9nUpkSDYdr9nDHTlnZ3KOOic\\\\\\\/xay2T8OjpimG0GC8eEDirHSj+mPCxIFe4bIBqr\\\\\\\/qkLzmabee1I7wYgq1o\\\\\\\/F6LQ2eNCo+M24kTUoPmnEHKlivadllGG0TysIK8xjK0DRlayw5FLciqTYjootT1pYybEhU+\\\\\\\/OI4SwIfSNru0q9PTmGMCVg8Ie0nNmvNmrTvNebl0rCzyyiX5Foe\\\\\\\/+CqUb1vBm4o\\\\\\\/5B3XALAbOq0M53zdUtHh83rkzxFZyH+SN07gWRgPpYjCOwAGy\\\\\\\/nRJoVQ4FF+yetYnYvE44qEQBatQ07opEiS8yf30Bx0Yy4nEGS7lu7NWy8MWYjFkx3oToSqB5rQlJhuZyfJghq446nz+VMNp8sJtZMzHvZ8mxk4ho96Bat1CdPs7kF611ULIB8FVgGoytIDM0LiUfV76wEmqjpUmtx6N55SwAcSm6xgHV1iD8UV0qU\\\\\\\/AwuCLfUmdQ7NYjWMbF09Dgj7Fumq78LKZEggg6MtYNfzdtVYTV8anovitdIrzKvpIS4KBULThHEbT8JEtJXfQ2cVCZsIjJMUFreiSAyd1Szl9zOxWT1u4DKa\\\\\\\/9KtyJS4mnY++OIJjrKpJoaqwH7Dot02AQ+Ktz70dPkPoXSwhJBx3hnFz5sE1NPYE4gMvf++P41xsHv91GdzlQ+0XpH9hIlgd9H0XiVFA\\\\\\\/Dc9aaE3PO9xc+1pdJP4DRdWMmpOBISoZknySXpp\\\\\\\/rGRU3TChgiRn9YG7W\\\\\\\/f+OJIY483vnLTyAXUzYpV19VOUU\\\\\\\/\\\\\\\/sqM\\\\\\\/5qtz5PdTqDk+XUdliUTxH2SsZC9n2QTzrdak4SEy4hcIsSkBComalg3T82lUqJV41O0HNELr74inETJgk5RklFDtmdig0\\\\\\\/NWh3w2iAuvimw2uNr8GNFir+p\\\\\\\/ehHQUP\\\\\\\/TXobm69JkSZanRQFvBNw4hzaSEWeQZ5RUsFhr5rBMv1pujvbkLDYuIX4DgXg8BcxZGyPBZfvBwRcDM7uX\\\\\\\/3UBw+GGRouoNDkxZz2RfNQW+7amFSgJiGgeiPRyTh8YUM+7rW\\\\\\\/CUDdu\\\\\\\/OdPsk0IXIC4Lx1TcTZ1B9Q\\\\\\\/yN06\\\\\\\/yQZ6bQTVgYW3kDngVKfm0qTibKJaDEb5zGcDblPJUkj9X3\\\\\\\/RLJtnkQmg979\\\\\\\/fYYqK+CYCgwnx9TyfRaQ4kN4bX+sQGZlnFo2OZy\\\\\\\/XEJIohK03l8JZXVEphDdjKj18wWdgvINkZU8lfEufv478vwSg5IwSIKHDj7eljGmUlCRp2Ey6+lt5CKOgptRmo54UUwjuai2OjYeh6h6qFufJUFUYnFTgGXoDW6KCHnHlSHXXGdlA3Z3qCvMxOptbcfalKaoO2xMaAZ\\\\\\\/22HGIcHCa2ydyLw10sNnIrGRqkL2NLf\\\\\\\/V96TU0Dm1i5c8VRmy\\\\\\\/G5RUaU7XFXZGbLqhTt7GPn\\\\\\\/UDhhj+1CFDlAmGllQyQq2jbj4vgc+xlJYyjcjyHWyMEH9FnZtKMGS1tFF+bRozh31y\\\\\\\/15Pnum8gdDAGmWUVXOycyPc8yzmzt4w9YZMAzCtMoc30BLR28Y7m9qqcI21qKeRPivG4qyDd9G5BQE0NEh9Hh4X8Jm+3T6ZWkekMdpZrHcCIckGu8+Mg6EOpOgrTXWCIOIuHAelykZPtqf+UnAASKAtypKE8g37gsTKWJTUV6LnSwpwisRc\\\\\\\/p3hbC6AUiAtqHf08G8yU\\\\\\\/JWokBA9YRRNK\\\\\\\/cv5cJmmOyV8VWGncBIGRHjaEvPMgb5mb939rYU\\\\\\\/8Hgnka+lJlLWnxPm\\\\\\\/IILPqHzKSsL+06W3Ww30IaPV8CmuBEbRAFmuF2vEJV04BCf6L1M5sScrtTWUzRQX\\\\\\\/4oCypmGcD+AvVMGznp8IdzSL7pQUufaBRSLGkEKf9FF2sTPHoe9HLIzLC2UM503BqIxnB3bsXHhC815iw14sdM+hnVzyf\\\\\\\/GTyckauiTyoiqntTTplH2Xxtj1N9caQiZ\\\\\\\/Ca1wXqm7NEjVxqp6FxJDIz2YiQv8h8Nce5fYwKGtrW24FZFPstdk7wce051EBr9GAaS52MbsSdRnSuso1nPlM\\\\\\\/\\\\\\\/WAShZHzVIui0OT4N3coigFfiW5gpf+UNJfqU6iz3k82rUzE8dd82zs2p51oI1Aupl515Gk4R0IdCRa6r1U2o0K6NbeOtQ1fpccYbSP+NU821ZXa9KPl\\\\\\\/HVkeu5vGFHKukgt093\\\\\\\/zwofKP7zJVRMz9EF2YHwUmK7vY4TXnMsxCbekBI7677K+ubDFIVvfA2oZYis80fsnLGn7\\\\\\\/aaIQ7u+UDoG+Ce+XIxk1evrHVILtEAkl3Ub4HMMA51foKbEZByqtoNN0mlenwoaHX86FlvJmP55rnlD+QWJCgi5gr+2kCpbALbwW\\\\\\\/eyWRKUlDlaVSJZeNU8TzDVYHClNcWRjieAL84B8BbWR2QfyvVkM6KMt6PpMrvo3vv\\\\\\\/pa5UFXdUVef2QjZ\\\\\\\/HlW4HlnsMG9Skm57odcDRZmrMw96R9jV8+VUZZpMGyuWKKfPDlggbcRCujIn2YlxAUxe8t8VBrpYbNV4Hi4pOXDQBP1PKkCiJYgrOdS8ASGYI8zz1aPamMUQFxY5eI\\\\\\\/rUKCTq1\\\\\\\/8TY7XAr3ljZ9aLT\\\\\\\/h3JvaicHIO1ueH8yZeQGxQB716XxWSLN0P20aOCimGLxr4Ov1EvU1Y1mr0b\\\\\\\/2xuG3T8B\\\\\\\/rtw4C\\\\\\\/WJQbT2wC4w1PEWxUdt7sZYcCDB3ioMugLSH45qWE5UoAjlpnPT6yYNShHozoIU+2uonemSCgIpwK21Xu0IlX0zaQJZ52+Cx1Tl9DFW+DiPxsI1iT19Ef5s\\\\\\\/y+uPwT2Vn0eoxw2dffEXOxpF++T5i9VfvRjvqzAFoCasgBjyd82IbJPnG8RPa02Wq8Jm3G4TgAfHuJxMRZy0kziRY56KBd2SAmn5R\\\\\\\/bpYDVPYEBWuT7TWiTfOKDY18OtI0P\\\\\\\/gD\\\\\\\/543UTxnOoHGfPy7GUfaw9vKIhdggoCy5f\\\\\\\/iqbDo8AgDZjbdj0Cz\\\\\\\/Hx4ZhsFou9AuKWMxL3IsrnkD3JERxzOESUZZOXZWib3e9o0XtDnio6YLMNUncNdjgi0Edbu2TW1HJzjz\\\\\\\/ry5kPFDNdXxZwfiYePKGVU5lZYgr4Lhmz0B7rbSY6X44MHYpzcFSoWcoFCNvK1TURBgvX5SGPeJaJlMMosPcq4sQW5yklcyNvNmBOAf4Ey4ST\\\\\\\/6rs+4PwjbyySEYMApAVQVENgGIIzO8KUPAEx1+t8SR982\\\\\\\/NWhbjYicIwPH17O43v5EjWFA3qVvJrisC1y7kyqENzDJkSYh9AkRdXPIVMgn\\\\\\\/oQk9sv8TBvt2g7z6la3kUn+7FpQZGnzpVp3vjqo\\\\\\\/vSak4DYmf0bseKNEUqatofA4aY4+hRu4MAntZ21Wd+IWx3iufMi6j9nf3TqAMl2U+SBVodfxbY+6aksZQNdpGujPDcEXqN9yOFe3auRrnnT9PCk1CZGD+nE5U5UXRj9i+UqMUDT+Gds5QKrsHqLAMz5dAhIDx3ZndgJ2exI3i7tLERSLQpbYdVpsarstQffqISS4LI\\\\\\\/mgoxfWnOmAzQSbvJW8iVez0Zu2E7+sxxfrhqqcuLSJb0CD8j3ocJteUdkhdCJ4l38y3F3TokoG1K0u8tDv0SdJw4A10DZE\\\\\\\/sI+MWE07KbXN\\\\\\\/pypmLfPIV\\\\\\\/3o5QtiG\\\\\\\/YGe\\\\\\\/AfASRVT5QVEFwD+R4GYD2AY6\\\\\\\/8jX41uhoTh34hb\\\\\\\/IkXiAQp\\\\\\\/\\\\\\\/lXia2VqKSXCseoiAV9h\\\\\\\/zXx6Xk7bYs\\\\\\\/yo73VoR\\\\\\\/MdK5t9lqSBomJQSRyAe0\\\\\\\/2YNOYXjJ7lt2kNCBkJhb4su9\\\\\\\/Z4a1z7NUcR1EgsIkA1UEbXzTFAt6q8xb6XiuJAYTjufOxMkI0vc0n7b\\\\\\\/GCSQ\\\\\\\/0MXXy1ucwsXPpOV2v4DivJqVfwcv1k9z\\\\\\\/6mebclTrUb3Vf\\\\\\\/ETWP0osnlvIESqVrvlw5WAntbjDkptRMtXZ+uMURwE35m+9OV4bZ8vsx7txhFcfYzwyo4TT6uXr3oE+VZcMSAk3Q3pVQeyts33ZZfnRS36\\\\\\\/F9tmHeqxyJ2YsP6VQZqJL1lzkwK7LCU76s15VmnxvjA9Qza9qWouX\\\\\\\/GeUSNL98CqEQIEhkp9yXMzuRiAkoVa6IdilJ0nvN0vbwWzeciZoYKL7mPF+kk45IHimAmKkNJqhO9LmHFfgdIg1MrA9u4BLlAZ1p4B8B5JsGVL5qMa59dNaHc\\\\\\\/Y7nSQnvDyNw+cAV5wUm1K4zneMe7xfNdN\\\\\\\/\\\\\\\/56OvWNGw3le9TjJ184j00Ofz6on69N1fJiPdOiIPwT3mTDts4A8uFImcf8RrV\\\\\\\/kz0dMAZ3F9b5YqVK1u+XrXyREK+zOcDhRfJFZj\\\\\\\/839PviwvLzqlpOzdIAnA1aD+zKeziz8Dytufkvsi1uGnfRy7pvWb12St+17OXu0n61ItGoovp\\\\\\\/A+LXwN2XF5OKhOztmdnkvVJlKH+OWy73PRvFtUFduBCNhMEGzpf3y9IA58zqzTABw+7m6ROD9i\\\\\\\/ulqmCMQjDFio1TUOE8LCNzg6KDEItYUaPRwH9BNJ1ze5WsqAVv5rFrlTKro3n7hk2FYO\\\\\\\/QyDFQSjaCNmcoxIcanpaUneILUH9uQFEyY5Psrj+pR2JxpaEhMCmVNWqOtLChOE2eX30tEfjhywkbLAaH7dvYSdOLQx0sqxrGOpPHIPHF2F4hniWgnqShau7OfbGlXw5CiEFXS9nY3W7DbZ6hV+xPwsGBrNSlNeU262TlqMKQ+uTglGgfULT9HnxZ\\\\\\\/6\\\\\\\/oOCCL4FI3FfGQ1qvXiZ168xg6ESqZXDlNT8sBk8xrojXMmta1qWsa3AOoaFkyeH\\\\\\\/29V0963rvoWWhwZ9Sggyx\\\\\\\/SuCCu2RnAegxUiOO0sTTzite4qPqrc\\\\\\\/cAqGeTkwBXAuFCnNJZrksCGZIb3oaYZpaq9ZOax40G7pXYFIK4kGK4QkjcaNF47jbQ10LylzLKTHgijYGrn9oxOuQPjzw7Fb5f0xzsCFw88cnLWco7KU1EwbdRC3k\\\\\\\/8VeC7ff\\\\\\\/oXMloxLSR+RGWfR2xu2injgyiFD9KtqcWfvjMqaxOSvAnLggAwLdaMSWOVCLx3Yvsz9ag0wINdOcUZusWQMsyvqJfYyRRatuSeP74yJxxKhY\\\\\\\/YeO7Xslp0fBWJnjdMBEfPgGyngwktcjvPjNfQcJ26xoSj2ZRmQp2m9fdOMk+lndejzwynVMY+m\\\\\\\/gABU5vEb4iNof1l9kC3SFsZpa\\\\\\\/jAYZa+yZ3SWjEuC45ioP5Y4\\\\\\\/7+HsxJBqa8ZcVQ8fyw2eeA9gnxw+6TH6a2eS3GkxHnNMkp\\\\\\\/9GQiBiBUm0F1s8MoS\\\\\\\/q6HSQkd1ooGfI4sAYv8+htpTXwTpjP8Aojq\\\\\\\/lhpkgDJkj3jLGohWk5GhWadH4w0oRS29NBu\\\\\\\/ZiUZ\\\\\\\/PPsJ9UJN28c2f+Rtt205Lo0q0qC4Hfoc44pqOxsQN88G8eAeQIcCQnuMNyBO5eDvDfsrJr+PofLug+2A012HqviIlr+NSMuMhoFB+7TW\\\\\\\/BdftHnmW7w39dWJQj3+XruCLHX2vNLs02Ge8Zt6KpKlyiImbPl7lKyVCNG3suYUtRShZoUYvQ8Rh6HYe16yq9nuQUrfwSphktRYjH8LLc4SEP519m\\\\\\\/HDB\\\\\\\/PmuxLrAtOYK2JqyQBIONEXCWQPQQxijQw4utPeykuZdMmTwlCZn9VXfV18g84kdoDzg2UAdcXs+dKKhzfX6pcOfMVVRUPNQHGI7GbjQ0MnB9W7H63P5Uk\\\\\\\/zG4eDpB+QmdFgnT3hM3fpQ1Uz1rSo8VwZ4hDL8a4T3ZjyOhpC1JGRwjK3MsOWDIeaTpCb55FaXgZFVqvw6d0zrDNidOgPY3qHvS+aSqFZwfGCJmT5Bnm9H0CIuJ6IapaIyuziEuB4XikmvJYjpFSBWr7LmRMQzs+UpJn0B9EeWYYJAEdBsNeL8aOHaaXZkwCmW+Y37FNd4UoMo0cu5eiF02yK\\\\\\\/QzGE\\\\\\\/RyGue19lwTT0ldA1ZNcaC\\\\\\\/mFplA9qaD24sWT+lU6uqA\\\\\\\/eD\\\\\\\/KKKQKR3iyAAMlXOZiAoHNX2P6W7lTFW6z7Y7qQHhRohn5a8n\\\\\\\/JIx\\\\\\\/7wUc6Pn52tYyZwxi5BGI3KZ+Ih8F5OIi1EqdD9iTFoJA1hMuGcqPoTkl1f4YM+nKetfzc6gfrNPieihr1y9ypwnpFHeXmzxokJv18jBJCh5hvzVVxPGGdQKRaKSfWEStvECw8\\\\\\\/3bBqVUDzelcuNazfz4aBAVlxQNlaJuO8U0woSlY+eSaPRbI6hIORyC5QwYBqoiuPyPbdMHqlHjX\\\\\\\/\\\\\\\/yW25X51VIBDDIHM1BEcLoOP9AgPuOU+6i7dgJl6P8Il3KKhufzXcw3Nj6DV291vh4f+4tui+1lP1zQaqvXochGP1UfBBa9mpKD2M1to9c7tnHoxapbk4uiiaHkV4BDto1JuZ68aw4KsmCWtOS+1gExryXoO58WWeXwVnSCzVXnJysGk\\\\\\\/gzsZ2c4f6hLyNgXxTcxnem1J8uTiVNElTN69pMOZfaM2PTjqkt6Ru1MhamdxXaFe0Q5KECnVf52FZ341RfAyP+NLTTQjPjmMjrrMGpcviEE8F3zh+7IDko2za+2I4XlwiyBhS+sFUdcTyYpSVlAqpO8Nvrv\\\\\\\/Kt9ZFKiChYqkPM9taF2P+xYWkSM1qb+ZS\\\\\\\/cyzchtDX1efsCaWudS423AAcw\\\\\\\/O9qxDDPG9MCsYJwJfLIbBqhgrKw\\\\\\\/SMxzSHemRGOrpwoHS+hCTln74ATPPT+9aetRwaHhTPoy0oFDzl8vztuYcjc70VjpeTJwQyfMItm0nHEjvgnHQREGi8DIpQoCrEN2NYFf+OUt5aVi4mfhOIxNfmHRRsXO52OSrYbnuCzJu\\\\\\\/V63\\\\\\\/kzL5YxRkiz8\\\\\\\/f1qhNuogVHu0kvrVsGjsTdnOoj177wf4aRl0bodxlE+\\\\\\\/Nezd+ur\\\\\\\/sQCcBUIVJnXeHahjOopz6PZGDVTpE7cm0TdZThHVc7C\\\\\\\/rmGUNDoak1Kl21EviM2ufvk+e5oFls3cLX59uRCYeUGUbRRPyoOtq9DcGa42g3SAXjkTvqtuxtZUMDStcJ5gcJ9MQi3m04Uvhdw9PrJ1xVihICMzYxEeshcIxjN7aujxjMPgsf3k1\\\\\\\/t2ojrsmZFWffPhU8oFWNoTbMBYNOqj+8E6I9dx3k05h5WHCTlosRHonYSYSdBEaRqSUHzSdUXwoTZr0\\\\\\\/DIXugzvi8UMTc\\\\\\\/ETwg6\\\\\\\/+VyxyiYP9WVQpi6s+EVyzJ9lFf07UG2TV8NJiy7OBLELbYcFoVwejRuHSjYI5ggnye67khwdiJstGoUc+dFiMT1Tu\\\\\\\/v8VsTJJf3ipbefI10O\\\\\\\/jcOv7ZzTJlFivaxgNFa+tJNdFO6E5gdmJ1k0fFgAAZheTLcqsv0Q9kJ4t+FAucHYTxml773p6MHeIjDtLWiJe2ZOGdiRiDWz2RiJIu7frSm\\\\\\\/3E0BOnsWcyqnF8WwVkTlEopawzGHTSy0xDloOfYYRr2qm\\\\\\\/DYvAjT49LID8J6\\\\\\\/NPNxtPQGSmA4IIaI6nAnKFT9QSzHJPHNsDf\\\\\\\/wPFg==\\\u0022,\\\u0022iv\\\u0022:\\\u0022Zikyi\\\\\\\/SWuAIeKy1ERedsPw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00229387f7d1e946ca808f45f3f714937ed18cc2a4873628a7f65404133b5c0c07d5\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022JcpuZnt\\\\\\\/1Yw\\\\\\\/Tc0y7fx7AURKhD6TlFzNT6D1pifptQQ9v2p6y5l0hXEhMCHRzultCmltoipgcyyOw1Jo5Vso+LsbkcVv+aP4XRJ686gmDQRGWfOiG5u2Rj1yBKBd1y1VKOyhSSFOU4+qcQtNfcTilye2qyrBaYzLZDKwGt7ZCLfchCWCkjb16lYelcQ\\\\\\\/JmvvzXyRmd15zOOl6wL+RAcnb83aQFJcNUIyrhG1oEcNDu2R0SkGPaeSTORmUgcK4XXvM1gN9PON1VXL\\\\\\\/1EK6Q1Z6jMwpRKD6QoVUAVr6rXu1Wt5rZInTI5dg7CyOYL+iHgRiJjuKTY+mNc81nc0isnxr\\\\\\\/3dLAjph0ld8Kqt2eGAquk9uA4rLsv4rAt8GBSoAzOHIe3PSQwOCi1xyd03ZJfBsa5pL774q+yiVy0h7b4QtUyYa\\\\\\\/rdesefy\\\\\\\/tPqUwgA\\\\\\\/I9HFgy8o7ExmLIaZmkL8\\\\\\\/HR+EowX98\\\\\\\/qClCZa2L5DxxMhKlYRHa5A3nnAtspYr13lrGOYOwQKZaU7zS26Bk4dvijCvkdYzaTBo2ke+ciyuJqrMaroRmLKcQXrKObKX2fD7GXw8RB\\\\\\\/2cYrK\\\\\\\/lmfnHaQRao2V1K+U5O6E+Kf\\\\\\\/IFiEjBROHnmtit0s506kdSG7u5yp9+BEWwU1DWhCl1odADzjxGROUw7s\\\\\\\/yviEGPOAikZBKg04vi728xXJKN4llowVWxiNvcJ+pjX7IowYrNPAzp3NjTNbWDiBKrEZbtfNm\\\\\\\/LAZ9s8B9erjRLAR8hf\\\\\\\/jTgCtEAj\\\\\\\/XgElKXKaPW9R3P+cOYeCgmPGHuMY9OrDvhExd9DjlsIONS3steQNHXRgqA36C6DJ7SN71p59abnfkn80u19SOjfvuOZAX\\\\\\\/K3+ExfkREQWO\\\\\\\/2wUrBlbdsTdxvSdgo0\\\\\\\/YMQoSLOPwSET+O0RqhQ4n2kMhfBFq7gfa9tWBq+b6ta+pRvcD4q7Fo48TD\\\\\\\/y\\\\\\\/4u0ODIOck9UeZAsCT1fCJ\\\\\\\/lttcv3nO16SY9MOX\\\\\\\/vAKLhMnr2vGboD1QLcYSlwo46Alp8FzgU6tryBGfZAZYf5KxYkO3H3GelpeV7nvOPq5bzWwhCBoEZotqry08OpWdSDCx4AtfEq3okSnuShUDEj9m4xwbVNfy9vFWNL3Y5XHmwO+edcJr1viFlwZIEeXx\\\\\\\/wjIy273z3\\\\\\\/vrtcQrVcMYg7o4NjCefI4s9CpI4zyeL4gpNcHFeSr\\\\\\\/0wO8UBAguMQGo8iouufWuQF+VharGttPZbpkWe3oFL29m3uPbC8YSvLQbKQmGHMyRJ6+49phbQQQ06O4b3Le9HpdbsoVm2i5vE\\\\\\\/Ti8YhDN+duoKF++ZOWVxINDXpXEmsv+vrU+HXwXIlVeLRTu2jft+D2zitEBa1P0YCUhwpDA60W1pER28aQaGuS85642oy\\\\\\\/ZVoOB4j3CHQYO\\\\\\\/BJ6maaOhOvHhDkZ87fHYcFwR1h2xIrcO6lpikjubbXq7XgJWrO5bzPuPLeFpEp0q\\\\\\\/SGKjIxoHy8vgtffTHfKPRTqTLe4AX9frpEs6S8f+MGHfWfjCqWRiTk3EQRv4JylTwQfl0BMf3k7BdEcT5CtaCWIi\\\\\\\/9rltukUHkkEDmCh9EI4j\\\\\\\/EpD6lOyIeqbgu\\\\\\\/y\\\\\\\/mg2k0Kst3Of85YLAKZR3SewollOOZtWb2xIZTsd5kLHdbSY+6EjTidyWXmIpT69IJuLWu56eNkD2zT9j4yv2dS85Oqh7Glm8WJM+tBGAeAZqXGwG\\\\\\\/SJ2SSwiqfwXE71b3KNMUhzDzIIvezAjf5MQfOUzP9gsw\\\\\\\/jzThuwSo1fHlQFKP2uZjSa3vi8HnbrPuwVUtbgxy5TJaB9pP2\\\\\\\/bXG8KxJESKBMtoultlFpCpAJAhemx33hm+6QRYNCY0aner2OcCb\\\\\\\/qVLRa3eZYAwfltSawLs8X4zTZwq3vCK01rNRxnL0zTkldmvKumLmLWL6i6Yb6IGq3E5JZftg4A4hvs8C2vJlPOOaonp8CpQfy\\\\\\\/2iKEfQDpdoEvXzzrKfpZOZDdeV9joUl8QOz6uyZQcVVJ6Hw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022KWeTgP6mnRjalfqWDor+aA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022132cb50b702589b780d936b11d122bf6927ea20d1f63d5823761c579909743a0\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-492612 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/4/1027" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-feQ9s9Qls357Gp6BPIhBxPKEBkBbwsSzx-kueg-jGXk" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/4/1027" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-u5zocSipgxc7ptHQDsbLxdf2CkToQP1yxIUXPhO_Xro" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/4/1009" title="A Large Scale Systemic RNAi Screen in the Red Flour Beetle Tribolium castaneum Identifies Novel Genes Involved in Insect Muscle Development" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/4/1037" title="Analysis of Genetic Diversity and Population Structure in Three Forest Musk Deer Captive Populations with Different Origins" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="492612" id="node492612" data-pisa="ggg;9/4/1027" data-pisa-master="ggg;g3.118.200905" data-apath="/ggg/9/4/1027.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">High-Density Linkage Map and QTLs for Growth in Snapper (<em>Chrysophrys auratus</em>)</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-0996-6368" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>David T. Ashton</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-8351-7931" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Peter A. Ritchie</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-2764-8291" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Maren Wellenreuther</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1027-1035;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200905 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node492612" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">David T. Ashton </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The New Zealand Institute for Plant & Food Research Limited, Nelson, New Zealand</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BT.%2BAshton%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ashton%20DT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADavid%2BT.%2BAshton%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-0996-6368" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for David T. Ashton</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:david.ashton@plantandfood.co.nz" class="" data-icon-position="" data-hide-link-title="0">david.ashton@plantandfood.co.nz</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Peter A. Ritchie </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>School of Biological Sciences, Victoria University of Wellington, Wellington, New Zealand</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Peter%2BA.%2BRitchie%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ritchie%20PA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APeter%2BA.%2BRitchie%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-8351-7931" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Peter A. Ritchie</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Maren Wellenreuther </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The New Zealand Institute for Plant & Food Research Limited, Nelson, New Zealand</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Maren%2BWellenreuther%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wellenreuther%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMaren%2BWellenreuther%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-2764-8291" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Maren Wellenreuther</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/9/4/1027" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:492612" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:492612/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/4/1027.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:492612" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:492612/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/4/1027.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:492612" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:492612/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/9/4/1027.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:492612" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:492612/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Characterizing the genetic variation underlying phenotypic traits is a central objective in biological research. This research has been hampered in the past by the limited genomic resources available for most non-model species. However, recent advances in sequencing technologies and related genotyping methods are rapidly changing this. Here we report the use of genome-wide SNP data from the ecologically and commercially important marine fish species <em>Chrysophrys auratus</em> (snapper) to 1) construct the first linkage map for this species, 2) scan for growth QTL, and 3) search for putative candidate genes in the surrounding QTL regions. The newly constructed linkage map contained ∼11K SNP markers and is one of the densest maps to date in the fish family Sparidae. Comparisons with genome scaffolds of the recently assembled snapper genome indicated that marker placement was mostly consistent between the scaffolds and linkage map (R = 0.7), but that at fine scales (&lt; 5 cM) some precision limitations occurred. Of the 24 linkage groups, which likely reflect the 24 chromosomes of this species, three were found to contain QTL with genome-wide significance for growth-related traits. A scan of 13 candidate growth genes located the <em>growth hormone, myogenin, and parvalbumin</em> genes within 5.3, 9.6, and 25.0 cM of these QTL, respectively. The linkage map and QTL found in this study will advance the investigation of genome structure and aquaculture breeding efforts in this and related species.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/linkage-map" class="hw-term hw-article-keyword hw-article-keyword-linkage-map" rel="nofollow">linkage map</a></li><li class="kwd"><a href="/keyword/genome" class="hw-term hw-article-keyword hw-article-keyword-genome" rel="nofollow">genome</a></li><li class="kwd"><a href="/keyword/fish" class="hw-term hw-article-keyword hw-article-keyword-fish" rel="nofollow">fish</a></li><li class="kwd"><a href="/keyword/qtls" class="hw-term hw-article-keyword hw-article-keyword-qtls" rel="nofollow">QTLs</a></li><li class="kwd"><a href="/keyword/growth" class="hw-term hw-article-keyword hw-article-keyword-growth" rel="nofollow">growth</a></li><li class="kwd"><a href="/keyword/genotyping-sequencing" class="hw-term hw-article-keyword hw-article-keyword-genotyping-by-sequencing" rel="nofollow">genotyping-by-sequencing</a></li></ul><p id="p-3">Characterizing the genetic variation that affects phenotypic traits is a central goal in biology. Understanding this variation can inform selective breeding programs (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Dekkers 2012</a>), be used to predict disease risk in medicine (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Lehner 2013</a>), and help researchers to understand evolution in natural populations (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Savolainen <em>et al.</em> 2013</a>). While genetic research has typically been pioneered in laboratory model species, the development of affordable high-throughput genomic methods (<em>e.g.</em>, next generation sequencing) is now allowing this research to be extended to a wide range of non-model species (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Hilario 2015</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Braasch <em>et al.</em> 2015</a>).</p><p id="p-4">Locating and characterizing quantitative trait loci (QTL) is one commonly used approach to investigate how genetic variation influences a specific phenotype (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Pértille <em>et al.</em> 2017</a>; <em>e.g.</em>, <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Barria <em>et al.</em> 2017</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Chen <em>et al.</em> 2017</a>). QTL mapping methods locate molecular marker loci influencing a phenotypic trait based on a significant correlation between the allelic variation of that loci and variation of the trait (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Lynch and Walsh 1998</a>). New QTL can be informative as standalone observations, or used to identify candidate genes in the surrounding genome, which may be influencing the trait (<em>e.g.</em>, <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Bettembourg <em>et al.</em> 2017</a>). It should be noted, however, that QTL studies and associated candidate gene investigations should be seen as a first step to gain insights into the genotype-phenotype map, and that additional studies are needed to verify that candidate genes are indeed causal in affecting the trait in question. Such genotype-phenotype datasets can also be used to develop multi-marker models (based on multiple QTL), which explain variation of one or more phenotypic traits (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Cros <em>et al.</em> 2017</a>).</p><p id="p-5">Having a road-map of the genome (<em>e.g.</em>, a linkage map or high quality genome assembly) is an important prerequisite for QTL mapping, as it allows the relative positioning of different marker loci. High-quality genome assemblies are most effective because they allow genetic markers to be positioned at a base-pair level, while also providing sequence information for the surrounding area. However, most non-model species do not currently have chromosome-level genome assemblies and instead rely on linkage maps to ascertain the relative position of markers in the genome (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Braasch <em>et al.</em> 2015</a>). Although they are often less precise than high quality genome assemblies that have been built with multiple datasets, linkage maps can serve the dual purpose of bridging the resource gap before a genome is developed and providing useful information to improve the arrangement of scaffolds during the genome assembly process (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Fierst 2015</a>). Indeed, data sets that are developed for constructing a linkage map can also be used in QTL mapping if phenotypic data are available.</p><p id="p-6">Teleost fish show significant potential to benefit from more affordable and high-throughput genomic technologies for a number of reasons. Teleosts are the largest group of vertebrates, with over 26,000 species (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Miller and Harley 2006</a>), and most species have a limited geographic range. Furthermore, several species are of commercial significance because they either have large natural stocks that are harvested by the fishing industry, or they are used in selective breeding programs in aquaculture. The restricted geographic range combined with this high diversity of species that are commercially exploited in wild fisheries and aquaculture means that scientific efforts are spread thinly overall, with few to no genomic resources available for many species.</p><p id="p-7">One of the most important traits in farmed species is growth performance, as it directly affects the efficiency of production systems. Selective breeding programs have been successful to enhance growth gains both terrestrial (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">VanRaden <em>et al.</em> 2009</a>) and aquatic animals (<em>e.g.</em>, <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Murata <em>et al.</em> 1996</a>). In most animal species, growth is a complex trait that is influenced by a network of genes (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">De-Santis and Jerry 2007</a>) and multiple environmental factors, such as seasonal variation in temperature, food availability, and competition (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Handeland <em>et al.</em> 2008</a>). Moreover, growth is also commonly correlated with variation in other life-history traits, such as gonad maturation processes and reproductive timing (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Bhatta <em>et al.</em> 2012</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Park <em>et al.</em> 2016</a>). Despite the numerous factors influencing growth, most quantitative genetics studies that investigate growth report moderate to high heritabilities (<em>e.g.</em>, 0.1 - 0.5) in a wide range of taxa (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Wang 2009</a>; <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Tsai <em>et al.</em> 2015</a>; <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Ye <em>et al.</em> 2017</a>). In fish, a number of genes associated with growth have been identified (reviewed in <a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">De-Santis and Jerry 2007</a>), including <em>growth hormone</em>, insulin-like growth factors, as well as a range of myogenic growth regulators.</p><p id="p-8">Here we focus on the marine teleost <em>Chrysophrys auratus</em> (family: Sparidae), commonly referred to as the Australasian snapper (henceforth referred to as “snapper”). Snapper supports a valuable recreational and commercial inshore fishery around the northern parts of New Zealand, southern Australia, and some of the Pacific Islands (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Parsons <em>et al.</em> 2014</a>), and is a strong candidate for development into an aquaculture species in both New Zealand and Australia. Closely related sparid species are already used for several aquaculture breeding programs around the world, for example, the sister species of snapper <em>Pagrus major</em> accounts for 10% of the total value of aquaculture in Japan. As part of our long-term research program to develop snapper into an aquaculture ready species, we here seek to identify genetic variation underlying growth differences among individuals in the breeding program to aid the selection of high-quality broodstock. In particular, our specific objectives were to use genome-wide SNP data from our pedigreed population to 1) construct a high density linkage map, 2) conduct QTL mapping for three measures of growth (peduncle length, fork length, and weight), and 3) investigate the position of 13 candidate growth genes and their relative position to growth QTL.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Study population</h3><p id="p-9">A snapper breeding program was started at The New Zealand Institute for Plant &amp; Food Research Limited in 2016 and includes a population with three generations held at the Nelson Research Centre in New Zealand. Data from the two most recent generations (F<sub>1</sub> = 70 individuals, F<sub>2</sub> = 577 individuals) were investigated in this study. Uncontrolled mass spawning of the F<sub>1</sub> generation in a single tank was used to produce the offspring F<sub>2</sub> generation. This resulted in a complex pedigree, meaning that we obtained a combination of full-siblings, half-siblings, and unrelated individuals in the F<sub>2</sub> generation (Supplementary Table 1). The F<sub>2</sub> offspring were held in a single tank until they were approximately one year old and then split evenly among four tanks with comparable feeding, light, water flow, aeration, tank design. All research carried out in this study was reviewed and approved by the animal ethics committee of Victoria University of Wellington in New Zealand (Application number 2014R19).</p></div><div id="sec-3" class="subsection"><h3>Phenotyping</h3><p id="p-10">Three measures of growth were used in the current study namely fork length, peduncle length, and weight. Fork length was measured as the distance from the nose to the fork in the tail. Peduncle length was measures as the distance from the nose to the narrowest cross-section across the tail. Measurements were made when the fish were a little over one year old (between 436-487 days) and again when they were approximately three years old (1045-1131 days). Length measurements were made by collecting images of each individual and then making measurements from those images. A ruler was included in each image to provide a scale. The number of individuals measured differed between year one and year three as a result of natural mortality during the study.</p></div><div id="sec-4" class="subsection"><h3>Genotyping</h3><p id="p-11">Samples of fin tissue were collected for all fish and DNA was extracted from these samples using a modified salt extraction protocol (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Aljanabi and Martinez 1997</a>). Quantification of DNA was carried out using Hoescht 33258 fluorescent dye. Fragmentation of the extracted DNA was checked by gel electrophoresis. Samples with moderate (∼25%) amounts of fragments below 10 kbp were re-extracted and if needed fresh samples were collected.</p><p id="p-12">Only high quality genomic DNA was used for the preparation of Genotyping By Sequencing (GBS) libraries based on the protocol described by <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Elshire <em>et al.</em> (2011)</a>. For each library, one microgram of DNA was double digested with the restriction enzymes <em>Pst</em>I and <em>Msp</em>I. The adaptor ligation step was done after digestion, without allowing the DNA/adaptor mixture to dry out. The barcoded adaptors were designed by Deena Bioinformatics and bound to the <em>Pst</em>I cut sites. Adaptors were subsequently annealed according to the method of Ko <em>et al.</em> (2003). The high fidelity enzyme AccuPrime Taq DNA polymerase High Fidelity (Life Technologies) was used for amplifications. Each library was amplified separately and its quality assessed by capillary electrophoresis prior to sequencing (Fragment Analyzer, Advanced Analytical). All GBS libraries were prepared in parallel in plates. Duplicate or triplicate samples were prepared for each of the parent and grandparents and single samples for each of the offspring (except for three individuals that had poor DNA quality, for which duplicate samples were prepared). Each plate (containing 96 individual libraries) was pooled, then cleaned up, quantified and sent to the Australian Genome Research Facility (AGRF) in Melbourne, Australia, for sequencing. Each pool was sequenced on a single lane with the Illumina HiSeq 2500 platform in single end (SE) mode, with a read length of 100 bases. In total, eight pools of libraries were sequenced in eight lanes for this project.</p><p id="p-13">FastQC was used to conduct an initial check of the sequencing data quality. Sequences were then de-multiplexed and cleaned. Adapters and primers were removed and the sequencing data were cleaned using Fastq-mcf in the ea-utils package (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Aronesty 2011</a>). Genotyping was carried out on the cleaned datasets using STACKs v1.40 (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Catchen <em>et al.</em> 2013</a>). The samples were first demultiplexed from the eight sequencing libraries using the process_radtags module command. Then sequencing reads for the duplicate or triplicate samples were concatenated into a single file, after which the reads were trimmed using Fastq-mcf with a minimum sequence length of 50, and a quality threshold causing base removal of 33. Bowtie v1.0 was used to align the GBS data to the genome assembly (with the ‘ref_map.pl’ option), allowing for 3 mismatches and 10, reported alignments. The pstacks module was then run, only including data that had a minimum coverage of 8x, followed by cstacks and sstacks using the pre-set parameters for the latter two modules. The population module was then used to output the data to a Genepop file while further filtering the data by applying a minor allele frequency/MAF of 0.05 and allowing only 0.25 of missing data. These SNP filtering steps were used to minimize missing data, exclude putative sequencing errors and to have sufficient power to call heterozygotes, while keeping a substantial number of informative SNPs. After these filtering steps a total of 20,311 SNPs were retained for subsequent analyses.</p></div><div id="sec-5" class="subsection"><h3>Linkage map construction</h3><p id="p-14">The parents for each F<sub>2</sub> individual in the dataset were identified using CERVUS v3.0.7 (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Kalinowski <em>et al.</em> 2007</a>) and a subset of SNPs (n = 2174) that were present in &gt;98% of individuals. All parental pairs were selected with a 95% confidence level based on the built in permutation procedure. A linkage map was constructed based on the SNP and pedigree data in LEPMAP v2.0 (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Rastas <em>et al.</em> 2016</a>). Data from the largest 14 F<sub>2</sub> families (full and half-sibling families) were used, and included a total of 269 offspring and 14 parents, and reduced the total number of available SNPs for this analyses to 20,311 SNPs. Markers were separated into chromosomes with the SeparateChromosomes module (logarithm of odds (LOD) limit = 14, minimum markers per linkage group = 50). The marker order was then generated with the OrderMarkers module. Markers near the start and end of each linkage group (start and end 10% based on centimorgan (cM) distance) were removed if they were more than 3 cM from the next closest marker. The accuracy of the final linkage map was investigated by comparing the linkage map position (cM) with the position of markers on available genome scaffolds (base-pairs) from the genome assembly (number of scaffolds 5998). The scaffold and base-pair position for each marker in the linkage map was retrieved from the STACKS v1.40 output files. Using this information the correlation between linkage map (cM) and scaffold (base-pair) position was tested for all scaffolds that contained &gt;50 SNPs. The mean and 95% confidence interval of the correlation residuals was then calculated. Whether scaffolds were placed uniquely on one of the 24 expected linkage groups was also investigated as well as the number and total base-pairs of scaffolds able to be positioned on the linkage group. The extent of linkage disequilibrium across the linkage groups was reviewed by calculating the pairwise linkage disequilibrium results for each set of markers using PLINK v1.9 (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Purcell <em>et al.</em> 2007</a>) and then visualizing the mean value at different distances across the linkage groups in R statistical environment v3.2.3 (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">R Core Team 2013</a>). This was done for all individuals in the F<sub>2</sub> generation and separately for individuals in the largest full sibling F<sub>2</sub> family (n = 48). The sex-specific recombination rate was calculated by comparing the linkage map distance (cM) and genome scaffold distance (bp) between individual marker pairs for males and females.</p></div><div id="sec-6" class="subsection"><h3>QTL identification</h3><p id="p-15">Quantitative trait loci identification was carried out using the general model implemented in QTDT v2.6.1 (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Abecasis <em>et al.</em> 2000</a>) and the half-sibling regression method implemented in GRIDQTL v3.3.0 (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Knott <em>et al.</em> 1996</a>; <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Seaton <em>et al.</em> 2006</a>). Both methods utilize parents as controls for population stratification and can use multiple offspring per family from a complex pedigree (<a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Knott <em>et al.</em> 1996</a>; <a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Abecasis <em>et al.</em> 2000</a>). Genotype data from the F<sub>1</sub> and F<sub>2</sub> generations and phenotyping data from the F<sub>2</sub> generation were used. QTDT used 10,716 markers which had been placed on the linkage map. GRIDQTL used a subset of markers (n = 1007), which were filtered randomly to a minimum spacing of 1 cM. Before running the analysis, the genotype data were filtered for Mendelian errors by dropping loci for any individual that contained alleles not observed in either of the two parents. The phenotype measurements used for the analysis were standardized by tank and date collected to correct for temporal and tank effects. The QTL scan results from QTDT were visualized using the ggplot2 library v3.1.0 in the R statistical environment v3.2.3 (<a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">R Core Team 2013</a>). The genome-wide 95% confidence limits were calculated for QTDT using a Bonferroni correction (<em>i.e.</em>, 0.05 / 10,716 markers) and using the built in permutation procedure in GRIDQTL with 1000 permutations.</p></div><div id="sec-7" class="subsection"><h3>Candidate genes and their location</h3><p id="p-16">The position of 13 candidate growth genes for fish (<a id="xref-ref-20-3" class="xref-bibr" href="#ref-20">De-Santis and Jerry 2007</a>) and growth QTL identified in this study were compared using the genome assembly. To do this, the sequence for each candidate gene was located on the NCBIS nucleotide database from the closest related teleost species - either the DNA or mRNA sequence. DNA sequences were mapped to the genome scaffolds by selecting the largest exon for the target gene and aligning with the “Map to Reference” alignment using the “Geneious mapper” in Geneious v10.0.9 (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Kearse <em>et al.</em> 2012</a>); alignment sensitivity was set to “High Sensitivity / Medium” with default settings. For mRNA sequences the sequences were aligned with the “Map to Reference” alignment using the “RNA Seq” mapper in Geneious; alignment sensitivity was set to “High Sensitivity / Medium” with the maximum gap size increased to 1000 bp. For each alignment the percentage of matching base pairs was reported for the largest exon. The linkage group and cM position of the scaffold containing specific candidate genes was then located using the STACKs output files and the newly constructed linkage map. Candidate gene locations were then compared with the position of putative growth QTL peaks (GRIDQTL) or closest genome-wide significant marker (QTDT).</p></div><div id="sec-8" class="subsection"><h3>Data availability</h3><p id="p-17">All data used in this study including the genome assembly, GBS sequencing libraries, phenotype data and supplemental material are located in an open data repository which can be accessed via <a href="https://www.genomics-aotearoa.org.nz/data">https://www.genomics-aotearoa.org.nz/data</a>.</p></div></div><div class="section results" id="sec-9"><h2 class="">Results</h2><div id="sec-10" class="subsection"><h3>Phenotyping</h3><p id="p-18">Peduncle length, fork length, and weight were recorded when individuals where 436-487 days old (year one) and 1045-1131 days old (year three). The distribution and relative sizes of fish in year one and year three are illustrated in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>. In the first set of measurements the mean and standard deviation for fork length, peduncle length, and weight were 160.1 ± 15.0 mm, 132.1 ± 12.3 mm, and 75.5 ± 20.3 g, respectively (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). In the second set of measurements the same measures were 257.8 ± 20.2, 214.5 ± 17.0, and 361.9 ± 82.3, respectively (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). The three measures for growth were all found to be strongly positively correlated (Pearson’s R &gt; 0.93, Ashton <em>et al.</em> 2019). Strong positive correlation was also observed between year one and year three for each measure (Pearson’s R = 0.71 – 0.73; Supplementary figure 1).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/4/1027/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Peduncle length measurements at year one (n = 568) and year three (n = 314). Images on the left side of the diagram are to scale relative to each other and show the smallest and largest individual fish from a tank of F2 individuals at year one and year three, respectively." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-127240629" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Peduncle length measurements at year one (n = 568) and year three (n = 314). Images on the left side of the diagram are to scale relative to each other and show the smallest and largest individual fish from a tank of F&lt;sub&gt;2&lt;/sub&gt; individuals at year one and year three, respectively.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/4/1027/F1.medium.gif" width="425" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/9/4/1027/F1.medium.gif" width="425" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/4/1027/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/4/1027/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492791" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-19" class="first-child">Peduncle length measurements at year one (n = 568) and year three (n = 314). Images on the left side of the diagram are to scale relative to each other and show the smallest and largest individual fish from a tank of F<sub>2</sub> individuals at year one and year three, respectively.</p><div class="sb-div caption-clear"></div></div></div><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/492715/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/492715/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Peduncle length, fork length, and weight at year one and year three including number or measurements (n), mean, and standard deviation (stdev)</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-11" class="subsection"><h3>Genotyping by sequencing</h3><p id="p-20">A total of 1.6 billion reads were produced for all eight pooled GBS libraries with approximately 2, 4 or 6 million reads for each single, duplicate, or triplicate individual library respectively. Using the STACKs pipeline a total of 20,311 SNPs were found after filtering for &gt;7x coverage, present in 75% of the individuals in the population, and a minor allele frequency (MAF) of 0.05. The average coverage per SNP was 15.6x in the offspring (F<sub>1</sub>) and 23.9x in the parents (F<sub>2</sub>).</p></div><div id="sec-12" class="subsection"><h3>Pedigree structure and linkage map</h3><p id="p-21">Parents were identified for 93% of the individuals in the F<sub>1</sub> and F<sub>2</sub> generations. The remaining 7% with missing parents were mainly located in the F<sub>1</sub> generation, and were the result missing F<sub>0</sub> wild-caught individuals that were not available for sampling at the time of the study. A mixture of 127 full sibling and half sibling families were present in the F<sub>2</sub> generation.</p><p id="p-22">A total of 10,716 SNPs were positioned on the linkage map (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). The total length of the sex-averaged linkage map was 1,363.0 cM with an average marker spacing of 0.129 cM. The lengths of the male and female maps were 1401.5 cM and 1359.0 cM, respectively. The female and male recombination rates were 3.28 cM/Mb and 1.93 cM/Mb based on comparison with available scaffolds from the snapper genome assembly. Moderate correlation (R = 0.74 ± 0.20 for 1723 markers on 26 scaffolds) was found across all scaffolds with &gt;50 markers. The 95% confidence interval of the correlation residuals ranged from -5.7 to 3.2 centimorgans with a mean of -1.25; indicating that 95% of markers were placed within ∼4.5 cM of their base-pair location. Visualization of the four largest scaffolds showed a clear relationship between the ordering of markers (correlation = 0.91, 0.53, 0.78, and -0.95), but that some noise was apparent around the exact placement (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). When aligning the genome scaffolds to the linkage map, individual scaffolds were placed exclusively onto one of the 24 linkage groups. Scaffolds which included markers from the linkage map contained a total of 701 Mb or ∼95% of the total base-pairs in the snapper genome. Investigation of the degrees of linkage and linkage disequilibrium within the dataset showed a clear pattern of linkage decay over the length of the linkage groups (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). When looking at a single F<sub>2</sub> family there was a high degree of linkage. However, when looking at the whole F<sub>2</sub> generation the decay of linkage and linkage disequilibrium is much greater, with minimal linkage observed even over small distances.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/4/1027/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Visualization of the linkage map including a total of 10,716 SNPs placed along the map in 24 linkage groups. The 24 linkage groups are equivalent size and number to represent the expected 24 Chyrsophrys auratus chromosomes." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-127240629" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Visualization of the linkage map including a total of 10,716 SNPs placed along the map in 24 linkage groups. The 24 linkage groups are equivalent size and number to represent the expected 24 &lt;em&gt;Chyrsophrys auratus&lt;/em&gt; chromosomes.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/4/1027/F2.medium.gif" width="440" height="288"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/9/4/1027/F2.medium.gif" width="440" height="288"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/4/1027/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/4/1027/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492703" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-23" class="first-child">Visualization of the linkage map including a total of 10,716 SNPs placed along the map in 24 linkage groups. The 24 linkage groups are equivalent size and number to represent the expected 24 <em>Chyrsophrys auratus</em> chromosomes.</p><div class="sb-div caption-clear"></div></div></div><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/4/1027/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparison of linkage group (cM) and genome scaffold (Mb) position for loci placed on the four largest scaffolds available from the genome. The position was strongly correlated between the two approaches (scaffold_1 = 0.91, scaffold_2 = 0.53, scaffold_3 = 0.78, scaffold_4 = -0.95), but there is also noise around the precise placement on the linkage map." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-127240629" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Comparison of linkage group (cM) and genome scaffold (Mb) position for loci placed on the four largest scaffolds available from the genome. The position was strongly correlated between the two approaches (scaffold_1 = 0.91, scaffold_2 = 0.53, scaffold_3 = 0.78, scaffold_4 = -0.95), but there is also noise around the precise placement on the linkage map.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/4/1027/F3.medium.gif" width="440" height="219"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/9/4/1027/F3.medium.gif" width="440" height="219"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/4/1027/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/4/1027/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492774" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-24" class="first-child">Comparison of linkage group (cM) and genome scaffold (Mb) position for loci placed on the four largest scaffolds available from the genome. The position was strongly correlated between the two approaches (scaffold_1 = 0.91, scaffold_2 = 0.53, scaffold_3 = 0.78, scaffold_4 = -0.95), but there is also noise around the precise placement on the linkage map.</p><div class="sb-div caption-clear"></div></div></div><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/4/1027/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Linkage (R2 correlation) between each pair of markers in the dataset plotted against the distance (cM) between those markers on the linkage map. This statistic was calculated for the entire QTL mapping dataset (multiple families, families = 137, n = 539) and for the data from the single largest family in the dataset (n = 48). The results show a consistent decay of linkage across the length of the linkage maps and &gt; 5x higher linkage in the largest family than the entire dataset." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-127240629" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Linkage (R&lt;sub&gt;2&lt;/sub&gt; correlation) between each pair of markers in the dataset plotted against the distance (cM) between those markers on the linkage map. This statistic was calculated for the entire QTL mapping dataset (multiple families, families = 137, n = 539) and for the data from the single largest family in the dataset (n = 48). The results show a consistent decay of linkage across the length of the linkage maps and &gt; 5x higher linkage in the largest family than the entire dataset.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/4/1027/F4.medium.gif" width="437" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/9/4/1027/F4.medium.gif" width="437" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/4/1027/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/4/1027/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492692" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-25" class="first-child">Linkage (R<sub>2</sub> correlation) between each pair of markers in the dataset plotted against the distance (cM) between those markers on the linkage map. This statistic was calculated for the entire QTL mapping dataset (multiple families, families = 137, n = 539) and for the data from the single largest family in the dataset (n = 48). The results show a consistent decay of linkage across the length of the linkage maps and &gt; 5x higher linkage in the largest family than the entire dataset.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-13" class="subsection"><h3>QTL mapping</h3><p id="p-26">Multiple QTL were found for all three growth traits in year one (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>, <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>, and Supplementary table 2). Genome-wide significant QTL were located on linkage groups 3, 11, 16 for fork length and peduncle length in QTDT and linkage groups 3, and 16 for all growth traits in GRIDQTL. The length trait QTL on linkage groups 3, 11, and 16 from QTDT were significant at a chromosome wide level for weight. The genome-wide significant markers for QTDT in year one had moderate effect sizes ranging from a minimum R<sup>2</sup> of 0.04 to a maximum of 0.05 for markers in QTDT (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>). No genome-wide significant QTL were found for growth traits in year three.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/4/1027/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="QTL scans at year one for fork length (A and D), peduncle length (B and E), and weight (C and F) using QTDT and GridQTL. The black horizontal line on each graph indicates the 95% genome-wide significance level. Genome-wide significant QTL were found using both software on linkage groups 3 and 16. A third genome-wide significant QTL was found on linkage group 11 using QTDT, but not GridQTL. QTL were shared between all traits within each software, except for weight in QTDT which had no genome-wide significant QTL." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-127240629" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;QTL scans at year one for fork length (A and D), peduncle length (B and E), and weight (C and F) using QTDT and GridQTL. The black horizontal line on each graph indicates the 95% genome-wide significance level. Genome-wide significant QTL were found using both software on linkage groups 3 and 16. A third genome-wide significant QTL was found on linkage group 11 using QTDT, but not GridQTL. QTL were shared between all traits within each software, except for weight in QTDT which had no genome-wide significant QTL.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/4/1027/F5.medium.gif" width="440" height="279"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/9/4/1027/F5.medium.gif" width="440" height="279"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/4/1027/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/4/1027/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492763" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-27" class="first-child">QTL scans at year one for fork length (A and D), peduncle length (B and E), and weight (C and F) using QTDT and GridQTL. The black horizontal line on each graph indicates the 95% genome-wide significance level. Genome-wide significant QTL were found using both software on linkage groups 3 and 16. A third genome-wide significant QTL was found on linkage group 11 using QTDT, but not GridQTL. QTL were shared between all traits within each software, except for weight in QTDT which had no genome-wide significant QTL.</p><div class="sb-div caption-clear"></div></div></div><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/492728/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/492728/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Putative QTL markers (QTDT) that were significant for at least one trait at a genome wide significance level of 5.33. Effect size (R<sup>2</sup>) was estimated in QTDT as the difference between the R squared values of the total model and the genotype model. Loci are reported including their linkage group number (LG) and position (cM). Genome wide (**) and linkage group wide (*) significance are indicated with asterix</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-14" class="subsection"><h3>Candidate genes</h3><p id="p-28">The base-pair position on the genome scaffolds were found for all 13 candidate genes including <em>growth hormone, growth hormone receptor, growth hormone receptor type 1, growth hormone receptor type 2, insulin like growth factor 1, insulin like growth factor 2, myogenic factor 1, myogenic factor 2, myogenic regulatory factor 4, myogenic regulatory factor 6, myogenin, myostatin, and parvalbumin</em> (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>). Based on the largest exon, all genes exhibited high base-pair similarity with the target genome position (88.7 to 99.3%). Of the candidate genes investigated, <em>growth hormone, myogenin, and paravalbumin</em> were located on linkage groups containing genome-wide significant QTL. <em>Growth hormone</em> on linkage group 16 was 5.3 cM from the nearest QTL marker (QTDT) and 30.2 cM from the nearest QTL peak (GRIDQTL). <em>Myogenin</em> on linkage group 3 was 9.6 cM from the nearest QTL marker (QTDT) and 18.4 cM from the nearest QTL peak (GRIDQTL). <em>Paravalbumin</em> on linkage group 16 was 25 cM from the nearest QTL marker (QTDT) and 7.8 cM from the nearest QTL peak (GRIDQTL).</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/492676/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/492676/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Candidate gene positions on the linkage map. An asterix * indicates that the gene was located on a linkage group also containing a genome-wide significant QTL for growth</span><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-15"><h2 class="">Discussion</h2><p id="p-29">We assembled the first chromosome level linkage map for the Australian snapper <em>Chrysophrys auratus</em>. Proof checking the marker order against the snapper <em>de novo</em> genome assembly indicated that the linkage groups were of high quality. QTL mapping revealed eight markers on three linkage groups that were significantly associated with growth. Three candidate genes for growth were located on the same linkage groups as these QTL. These genomic resources will be used to inform the selective breeding program in New Zealand and will form the basis of further genomic investigation in snapper.</p><p id="p-30">Linkage maps are essential for genomic and genetic studies, and have been used extensively to derive the order and spatial position of markers (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Cnaani <em>et al.</em> 2004</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Greenwood <em>et al.</em> 2011</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Boulton <em>et al.</em> 2011</a>). Historically, most first generation linkage maps in fish have been constructed with just a handful or a few hundred markers and did not have genome sequences available to evaluate marker order (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Castaño-Sánchez <em>et al.</em> 2010</a>). However, technologies advances over the last years have facilitated an increase in the number of markers used to construct linkage maps (<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Castaño-Sánchez <em>et al.</em> 2010</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Ninwichian <em>et al.</em> 2012</a>) and some have also begun to utilize available genome data for checking both the linkage map and/or genome assembly accuracy (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Tsai <em>et al.</em> 2016</a>; <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Wang <em>et al.</em> 2017</a>). The snapper linkage map is composed of ∼11K markers and covers all 24 chromosomes of the 738 Mb genome (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>), and is with this marker number the densest to date for the family Sparidae. Even denser linkage maps are being constructed for a few other fish species, for example the linkage map for the Atlantic salmon (<em>Salmo salar</em>) includes ∼96K markers, although it should be noted that the genome size in this salmonid species is also significantly larger (2.97 Gb genome size) (<a id="xref-ref-53-2" class="xref-bibr" href="#ref-53">Tsai <em>et al.</em> 2016</a>). The average correlation between the largest snapper genome scaffolds (bp) and our linkage map (cM) was 0.74 (illustrated for the four largest scaffolds in <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>). Comparatively, the results for the recently constructed Atlantic salmon linkage map were 0.81 for the male map and 0.92 for the female map. While the correlation is reasonably high and indicates a good agreement between the map and assembly overall, it is also apparent that some variation around the exact placement of SNPs is apparent at a fine scale (&lt; 5 cM intervals). In many cases this variation is probably the result of inherent precision limitations in the dataset (sample size + number of recombination events), but some of this variation could also be the result of differential recombination patterns across the genome (as observed in <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Roesti <em>et al.</em> 2013</a>).</p><p id="p-31">Using the newly constructed linkage map and available genome scaffolds, we were able to calculate the sex-specific recombination rates for snapper, which showed that females have a higher recombination rate compared to males (female = 3.28 cM/Mb, male = 1.93 cM/Mb). This reflects observations in other fish species, with females often (but not always, <a id="xref-ref-56-2" class="xref-bibr" href="#ref-56">Wang <em>et al.</em> 2017</a>) having a higher recombination rates than males (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Kucuktas <em>et al.</em> 2009</a>; <a id="xref-ref-14-3" class="xref-bibr" href="#ref-14">Castaño-Sánchez <em>et al.</em> 2010</a>; <a id="xref-ref-53-3" class="xref-bibr" href="#ref-53">Tsai <em>et al.</em> 2016</a>). Overall, the small sex specific differences in recombination rates in this study are consistent with the ranges found in several other fish species including stickleback (<em>Gasterosteus aculeatus</em>, 3.11 cM/Mb, (<a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">Roesti <em>et al.</em> 2013</a>)), Asian seabass (<em>Lates calcarifer</em>, 2.4-2.8 cM/Mb, (<a id="xref-ref-56-3" class="xref-bibr" href="#ref-56">Wang <em>et al.</em> 2017</a>)), and channel catfish (<em>Ictalurus punctatus</em>, 2.6 cM/Mb, (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Li <em>et al.</em> 2015</a>)). Interestingly, it has long been observed that the heterogametic sex shows typically suppressed recombination (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Haldane 1922</a>). For example, in <em>Drosophila</em> spp., this reduction is so dramatic that during gametogenesis there are no chiasmata being formed and hence no recombination takes place (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Morgan 1914</a>). In mammals, species show reduced recombination frequencies in males, which are the heterogametic sex (<em>e.g.</em>, humans). It is not known how sex is being determined in snapper, but if the heterochiasmatic sex commonly has a lower recombination rate, then this may indicate that males are heterogametic in this species. Further work is needed to explore this in more detail.</p><p id="p-32">The target trait for this study was growth (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>), which was measured using peduncle length, fork length, and weight (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>). Growth is one of the primary targets for selective breeding programs because it relates directly to production output. In fish, it typically has a moderate degree of heritability, and in the current population was shown to be approximately ∼0.26 and ∼0.11 in year one and year three, respectively (Ashton <em>et al.</em>, 2019). Other factors that can affect growth include feed amounts, fish density in tanks, and tank design (size, aeration, water flow). We attempted to control for these factors in the current study by standardizing the conditions between tanks and by standardizing measures from each tank.</p><p id="p-33">Genome-wide significant QTL were found in the first year, but not in the third year, which could indicate that the genetic basis of growth in early life is lost as the fish age (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>, <a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>, Supplementary table 1). However, it seems more likely that the lack of growth QTL in year three is due to the decreased sample size from year one to year three. The lower sample size in year three was the result of natural mortality over the course of the study and decreased our number of phenotyped and genotyped fish by roughly half across two time periods. It is notable that the quantitative trait loci of genome-wide significance were highly shared among the three measures (peduncle length, fork length, and weight), which reflects that these are all measures of the same underlying trait (growth). The effect size averaged around 0.05 for individual QTL, which is similar to growth QTL observed in other species including Atlantic salmon (<em>Salmo salar</em>: 0.06 to 0.08, (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Besnier <em>et al.</em> 2015</a>)), tilapia (<em>Oreochromis niloticus</em>: 0.06 to 0.19, (<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Cnaani <em>et al.</em> 2004</a>)), chinook salmon (<em>Oncorhynchus tshawytscha</em>: 0.14 to 0.33, (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Everett and Seeb 2014</a>)), brill (<em>Scophthalmus rhombus</em>: 0.08 to 0.12, (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Houston <em>et al.</em> 2009</a>)), and catfish (<em>Ictalurus furcatus</em>: 0.01 to 0.23, (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Hutson <em>et al.</em> 2014</a>)). The identification of multiple QTL affecting growth indicates a polygenic basis for growth, something that is commonly reported for complex quantitative traits like growth (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Wellenreuther and Hansson 2016</a>).</p><p id="p-34">Determining accuracy of QTL placement is an important step in the QTL mapping process as it provides useful information about where variants responsible for an observed QTL signal (<em>e.g.</em>, candidate genes or causative alleles) are likely to be located. If high rates of linkage are present between markers, a confidence interval for the QTL region can be estimated - as seen in R/QTL (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Broman <em>et al.</em> 2003</a>). However, in the current study, pairwise correlation between markers (linkage) across the linkage map indicated very low linkage between markers over even relatively short distances (&lt; 5 cM) (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>). This is most obvious when comparing the linkage observed within the single largest family to that observed in the entire dataset (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4</a>). However, it is worth noting that on linkage groups with genome-wide significant QTL, there does appear to be a number of markers surrounding each QTL that are responding to the QTL signal. As such, it seems likely that there is some linkage between markers at a fine scale (&lt; 5 cM), but that this may be obscured by the low precision of marker placement on the linkage map. If true, the optimal way to get more precise placement of QTL regions will be improved SNP positioning using either a second improved iteration of the linkage map, the genome assembly, or a combination of both these resources. Until this is done it is likely that causative genetic variations underlying the QTL signals will be within this 5 cM scale.</p><p id="p-35">Previous studies have outlined a range of genes and molecular networks that are thought to be candidates for further investigation in teleost species (<a id="xref-ref-20-4" class="xref-bibr" href="#ref-20">De-Santis and Jerry 2007</a>). We located the position of candidate genes surrounding the detected growth QTL (<a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>), however, the large distances between the genes and the QTL peaks indicate that one needs to be cautious about a definitive link between the QTL and candidate genes. Our candidate gene search was possible using the available genome sequence data to link gene positions back to their nearest markers. Central to growth in most species is the somatotropic axis, which consists of the <em>growth hormone releasing hormone</em> (GHRH), <em>growth hormone inhibiting hormone</em> (GHIH), <em>growth hormone</em> (GH), and <em>insulin-like growth factors</em> (IGF-1 and –II) (<a id="xref-ref-20-5" class="xref-bibr" href="#ref-20">De-Santis and Jerry 2007</a>). Of these, <em>growth hormone</em> and <em>insulin-like growth factor I and II</em> were able to be mapped to the linkage map in the current study. Growth hormone was located near (within 5.3 cM) a QTL of genome-wide significance. In <em>Sparus aurata</em>, a close relative of <em>C. auratus</em>, a microsatellite repeat in the promoter region has previously been implicated for differences in growth (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Almuly <em>et al.</em> 2000</a>). This gene would be a good candidate in <em>C. auratus</em> because it is close to a QTL in the current study, and the causative microsatellite has been observed in a range of teleost species. Myogenic regulatory factors (<em>myogenin, MyoD, myf-5, and myf-6</em>) are another set of potential candidate genes (<a id="xref-ref-20-6" class="xref-bibr" href="#ref-20">De-Santis and Jerry 2007</a>). These regulatory factors have been implicated in growth in terrestrial vertebrates, but not in fish species. In this study, the <em>myogenin</em> gene was located on linkage group 3 approximately 9.6 cM from a genome-wide significant QTL. In pigs, a polymorphism in the promoter region of <em>myogenin</em> was found to account for up to 5.8% of differences in weight (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">te Pas <em>et al.</em> 1999</a>), but no research has investigated its effect in teleost species. A final candidate gene (<em>paravalbumin</em>) was located on the same linkage group as a genome wide significant QTL, but was much further away from a putative growth QTL (25 cM) than the previous two genes. A mutation in the promoter region of this gene was previously found to be involved in weight differences in the finfish species <em>Lates calcarifer</em>.</p><div id="sec-16" class="subsection"><h3>Future directions</h3><p id="p-36">While the linkage map constructed in this study can confidently place SNPs in ∼5 cM regions, further work is needed to improve the accuracy of marker placement. More accurate placement of SNPS would help with future work to fine-map and further characterize the QTL and candidate gene locations described in this study. Improved precision should be possible in the near future using the genome assembly that is being further improved by our group. Future work should also aim to detect possible sex-linked markers, to identify regions associated with sex determination, and to investigate sex-specific recombination patterns across the genome. While sex-specific information was not investigated in the current study, this is an area of particular interest in snapper and the data from this study could be used to further investigate it. In conclusion, this study provides valuable genetic and genomic resources for future evolutionary studies and aquaculture breeding programs in this and related species.</p></div></div><div class="section ack" id="ack-1"><h2 class="">ACKNOWLEDGMENTS</h2><p id="p-37">We would like to acknowledge all the PFR staff who have been involved with this work in particular Peter Jaksons, Ross Crowhurst, and Elena Hilario. This project was funded through the New Zealand Ministry of Business, Innovation, and Employment Endeavour Fund “Accelerated breeding for enhanced seafood production” (#C11X1603). </p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-38">Supplemental material available at <a href="https://www.genomics-aotearoa.org.nz/data">https://www.genomics-aotearoa.org.nz/data</a>.</p></li><li class="fn" id="fn-2"><p id="p-39"><em>Communicating editor: J. Yáñez</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2018-11-22"><span class="received-label">Received </span>November 22, 2018.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2019-01-10"><span class="accepted-label">Accepted </span>January 10, 2019.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2019 Ashton <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1027.1" data-doi="10.1086/302698"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Abecasis</span>  <span class="cit-name-given-names">G. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cardon</span>  <span class="cit-name-given-names">L. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cookson</span>  <span class="cit-name-given-names">W. O.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">A general test of association for quantitative traits in nuclear families.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">66</span>: <span class="cit-fpage">279</span>–<span class="cit-lpage">292</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1086/302698</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DAbecasis%26rft.auinit1%253DG.%2BR.%26rft.volume%253D66%26rft.issue%253D1%26rft.spage%253D279%26rft.epage%253D292%26rft.atitle%253DA%2Bgeneral%2Btest%2Bof%2Bassociation%2Bfor%2Bquantitative%2Btraits%2Bin%2Bnuclear%2Bfamilies.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F302698%26rft_id%253Dinfo%253Apmid%252F10631157%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/302698&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10631157&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1027.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000085033600028&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1027.2" data-doi="10.1093/nar/25.22.4692"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Aljanabi</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martinez</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Universal and rapid salt-extraction of high quality genomic DNA for PCR-based techniques.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">4692</span>–<span class="cit-lpage">4693</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/25.22.4692</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F25.22.4692%26rft_id%253Dinfo%253Apmid%252F9358185%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/25.22.4692&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9358185&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1027.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1997YJ40500037&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1027.3" data-doi="10.1139/g00-051"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Almuly</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cavari</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferstman</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kolodny</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Funkenstein</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Genomic structure and sequence of the gilthead seabream (<em>Sparus aurata</em>) growth hormone-encoding gene: identification of minisatellite polymorphism in intron I.</span> <abbr class="cit-jnl-abbrev">Genome</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">836</span>–<span class="cit-lpage">845</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1139/g00-051</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.stitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.aulast%253DAlmuly%26rft.auinit1%253DR.%26rft.volume%253D43%26rft.issue%253D5%26rft.spage%253D836%26rft.epage%253D845%26rft.atitle%253DGenomic%2Bstructure%2Band%2Bsequence%2Bof%2Bthe%2Bgilthead%2Bseabream%2B%2528Sparus%2Baurata%2529%2Bgrowth%2Bhormone-encoding%2Bgene%253A%2Bidentification%2Bof%2Bminisatellite%2Bpolymorphism%2Bin%2Bintron%2BI.%26rft_id%253Dinfo%253Adoi%252F10.1139%252Fg00-051%26rft_id%253Dinfo%253Apmid%252F11081974%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1139/g00-051&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11081974&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1027.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-other" id="cit-9.4.1027.4"><div class="cit-metadata unstructured">Aronesty, E., 2011 ea-utils: Command-line tools for processing biological sequencing data. <a href="https://github.com/ExpressionAnalysis/ea-utils">https://github.com/ExpressionAnalysis/ea-utils</a></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-9.4.1027.5"><div class="cit-metadata unstructured">Ashton, D. T, E. Hilario, P. Jaksons, P. A. Ritchie, M. Wellenreuther, 2014 Genetic diversity and heritability of economically important traits in captive Australasian snapper (Chrysophrys auratus). Aquaculture. 505: 190–198. <a href="https://doi.org/10.1016/j.aquaculture.2019.02.034">https://doi.org/10.1016/j.aquaculture.2019.02.034</a></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-other no-rev-xref" id="cit-9.4.1027.6"><div class="cit-metadata unstructured">Ashton, D. T., E. Hilario, P. Jaksons, P. A. Ritchie, and M. Wellenreuther, 2018 Genetic diversity and heritability of economically important traits in captive Australasian snapper (<em>Chrysophrys auratus</em>). in prep.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1027.7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Barria</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Christensen</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Correa</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jedlicki</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lhorente</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Genome-wide association study and genomic predictions for resistance against <em>Piscirickettsia salmonis</em> in Coho salmon (<em>Oncorhynchus kisutch</em>) using ddRAD sequencing.</span> <abbr class="cit-jnl-abbrev">bioRxiv</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">124099</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DbioRxiv%26rft.volume%253D8%26rft.spage%253D124099%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1027.8" data-doi="10.1038/hdy.2015.15"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Besnier</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glover</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lien</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kent</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hansen</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Identification of quantitative genetic components of fitness variation in farmed, hybrid and native salmon in the wild.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">115</span>: <span class="cit-fpage">47</span>–<span class="cit-lpage">55</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/hdy.2015.15</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D115%26rft.spage%253D47%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2015.15%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2015.15&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1027.9" data-doi="10.1186/s12284-017-0184-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bettembourg</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dardou</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Audebert</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thomas</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Frouin</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Genome-wide association mapping for root cone angle in rice.</span> <abbr class="cit-jnl-abbrev">Rice (N. Y.)</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">45</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12284-017-0184-z</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRice%2B%2528N.%2BY.%2529%26rft.volume%253D10%26rft.spage%253D45%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12284-017-0184-z%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12284-017-0184-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1027.10" data-doi="10.1073/pnas.1118704109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bhatta</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Iwai</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miura</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Higuchi</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shimizu-Yamaguchi</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Gonads directly regulate growth in teleosts.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">109</span>: <span class="cit-fpage">11408</span>–<span class="cit-lpage">11412</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1118704109</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1118704109%26rft_id%253Dinfo%253Apmid%252F22733736%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA5LzI4LzExNDA4IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS80LzEwMjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1027.11" data-doi="10.1016/j.aquaculture.2011.06.044"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Boulton</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Massault</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Houston</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Koning</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haley</span>  <span class="cit-name-given-names">C. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">QTL affecting morphometric traits and stress response in the gilthead seabream (<em>Sparus aurata</em>).</span> <abbr class="cit-jnl-abbrev">Aquaculture</abbr> <span class="cit-vol">319</span>: <span class="cit-fpage">58</span>–<span class="cit-lpage">66</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.aquaculture.2011.06.044</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAquaculture%26rft.volume%253D319%26rft.spage%253D58%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.aquaculture.2011.06.044%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.aquaculture.2011.06.044&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1027.12" data-doi="10.1002/jez.b.22589"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Braasch</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peterson</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Desvignes</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCluskey</span>  <span class="cit-name-given-names">B. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Batzel</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">A new model army: Emerging fish models to study the genomics of vertebrate Evo-Devo.</span> <abbr class="cit-jnl-abbrev">J. Exp. Zoolog. B Mol. Dev. Evol.</abbr> <span class="cit-vol">324</span>: <span class="cit-fpage">316</span>–<span class="cit-lpage">341</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/jez.b.22589</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BExp.%2BZoolog.%2BB%2BMol.%2BDev.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fjez.b.22589%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/jez.b.22589&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1027.13" data-doi="10.1093/bioinformatics/btg112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Broman</span>  <span class="cit-name-given-names">K. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sen</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Churchill</span>  <span class="cit-name-given-names">G. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">R/qtl: QTL mapping in experimental crosses.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">889</span>–<span class="cit-lpage">890</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btg112</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DBroman%26rft.auinit1%253DK.%2BW.%26rft.volume%253D19%26rft.issue%253D7%26rft.spage%253D889%26rft.epage%253D890%26rft.atitle%253DR%252Fqtl%253A%2BQTL%2Bmapping%2Bin%2Bexperimental%2Bcrosses%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtg112%26rft_id%253Dinfo%253Apmid%252F12724300%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btg112&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12724300&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1027.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000182634800014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1027.14" data-doi="10.1186/1471-2164-11-554"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Castaño-Sánchez</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fuji</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ozaki</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hasegawa</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sakamoto</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">A second generation genetic linkage map of Japanese flounder (<em>Paralichthys olivaceus</em>).</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">554</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2164-11-554</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DCastano-Sanchez%26rft.auinit1%253DC.%26rft.volume%253D11%26rft.spage%253D554%26rft.epage%253D554%26rft.atitle%253DA%2Bsecond%2Bgeneration%2Bgenetic%2Blinkage%2Bmap%2Bof%2BJapanese%2Bflounder%2B%2528Paralichthys%2Bolivaceus%2529.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-11-554%26rft_id%253Dinfo%253Apmid%252F20937088%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-11-554&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20937088&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1027.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1027.15" data-doi="10.1111/mec.12354"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Catchen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hohenlohe</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bassham</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amores</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cresko</span>  <span class="cit-name-given-names">W. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Stacks: an analysis tool set for population genomics.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">3124</span>–<span class="cit-lpage">3140</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/mec.12354</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D22%26rft.spage%253D3124%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.12354%26rft_id%253Dinfo%253Apmid%252F23701397%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.12354&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23701397&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1027.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000319675300022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1027.16" data-doi="10.1007/s11032-017-0741-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Semagn</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Iqbal</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moakhar</span>  <span class="cit-name-given-names">N. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haile</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Genome-wide association mapping of genomic regions associated with phenotypic traits in Canadian western spring wheat.</span> <abbr class="cit-jnl-abbrev">Mol. Breed.</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">141</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s11032-017-0741-6</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBreed.%26rft.volume%253D37%26rft.spage%253D141%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11032-017-0741-6%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11032-017-0741-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1027.17" data-doi="10.1007/s00438-004-1045-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cnaani</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zilberman</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tinman</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hulata</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ron</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Genome-scan analysis for quantitative trait loci in an F2 tilapia hybrid.</span> <abbr class="cit-jnl-abbrev">Mol. Genet. Genomics</abbr> <span class="cit-vol">272</span>: <span class="cit-fpage">162</span>–<span class="cit-lpage">172</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00438-004-1045-1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bgenetics%2Band%2Bgenomics%2B%253A%2B%2BMGG%26rft.stitle%253DMol%2BGenet%2BGenomics%26rft.aulast%253DCnaani%26rft.auinit1%253DA.%26rft.volume%253D272%26rft.issue%253D2%26rft.spage%253D162%26rft.epage%253D172%26rft.atitle%253DGenome-scan%2Banalysis%2Bfor%2Bquantitative%2Btrait%2Bloci%2Bin%2Ban%2BF2%2Btilapia%2Bhybrid.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00438-004-1045-1%26rft_id%253Dinfo%253Apmid%252F15449174%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00438-004-1045-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15449174&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1027.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000224527700005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1027.18" data-doi="10.1186/s12864-017-4179-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cros</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bocs</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Riou</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ortega-Abboud</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tisné</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Genomic preselection with genotyping-by-sequencing increases performance of commercial oil palm hybrid crosses.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">839</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12864-017-4179-3</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D18%26rft.spage%253D839%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-017-4179-3%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12864-017-4179-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1027.19" data-doi="10.2174/138920212800543057"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dekkers</span>  <span class="cit-name-given-names">J. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Application of genomics tools to animal breeding.</span> <abbr class="cit-jnl-abbrev">Curr. Genomics</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">207</span>–<span class="cit-lpage">212</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2174/138920212800543057</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DCurr%2BGenomics%26rft.aulast%253DDekkers%26rft.auinit1%253DJ.%2BC.%26rft.volume%253D13%26rft.issue%253D3%26rft.spage%253D207%26rft.epage%253D212%26rft.atitle%253DApplication%2Bof%2Bgenomics%2Btools%2Bto%2Banimal%2Bbreeding.%26rft_id%253Dinfo%253Adoi%252F10.2174%252F138920212800543057%26rft_id%253Dinfo%253Apmid%252F23115522%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2174/138920212800543057&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23115522&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1027.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1027.20" data-doi="10.1016/j.aquaculture.2007.08.036"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">De-Santis</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jerry</span>  <span class="cit-name-given-names">D. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Candidate growth genes in finfish — Where should we be looking?</span> <abbr class="cit-jnl-abbrev">Aquaculture</abbr> <span class="cit-vol">272</span>: <span class="cit-fpage">22</span>–<span class="cit-lpage">38</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.aquaculture.2007.08.036</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAquaculture%26rft.volume%253D272%26rft.spage%253D22%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.aquaculture.2007.08.036%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.aquaculture.2007.08.036&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1027.21" data-doi="10.1371/journal.pone.0019379"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Elshire</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glaubitz</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kawamoto</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A robust, simple Genotyping-by-Sequencing (GBS) approach for high diversity species.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e19379</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0019379</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DElshire%26rft.auinit1%253DR.%2BJ.%26rft.volume%253D6%26rft.issue%253D5%26rft.spage%253De19379%26rft.epage%253De19379%26rft.atitle%253DA%2Brobust%252C%2Bsimple%2Bgenotyping-by-sequencing%2B%2528GBS%2529%2Bapproach%2Bfor%2Bhigh%2Bdiversity%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0019379%26rft_id%253Dinfo%253Apmid%252F21573248%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0019379&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21573248&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1027.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1027.22" data-doi="10.1111/eva.12147"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Everett</span>  <span class="cit-name-given-names">M. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seeb</span>  <span class="cit-name-given-names">J. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Detection and mapping of QTL for temperature tolerance and body size in Chinook salmon (<em>Oncorhynchus tshawytscha</em>) using genotyping by sequencing.</span> <abbr class="cit-jnl-abbrev">Evol. Appl.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">480</span>–<span class="cit-lpage">492</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/eva.12147</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvol.%2BAppl.%26rft.volume%253D7%26rft.spage%253D480%26rft_id%253Dinfo%253Adoi%252F10.1111%252Feva.12147%26rft_id%253Dinfo%253Apmid%252F24822082%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/eva.12147&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24822082&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1027.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1027.23" data-doi="10.3389/fgene.2015.00001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fierst</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Using linkage maps to correct and scaffold de novo genome assemblies: methods, challenges, and computational tools.</span> <abbr class="cit-jnl-abbrev">Front. Genet.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">8</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fgene.2015.00220</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BGenet.%26rft.volume%253D6%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffgene.2015.00001%26rft_id%253Dinfo%253Apmid%252F25674101%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fgene.2015.00001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25674101&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1027.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1027.24" data-doi="10.1038/hdy.2011.1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Greenwood</span>  <span class="cit-name-given-names">A. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">F. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chan</span>  <span class="cit-name-given-names">Y. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brady</span>  <span class="cit-name-given-names">S. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Absher</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The genetic basis of divergent pigment patterns in juvenile threespine sticklebacks.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">107</span>: <span class="cit-fpage">155</span>–<span class="cit-lpage">166</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/hdy.2011.1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.stitle%253DHeredity%26rft.aulast%253DGreenwood%26rft.auinit1%253DA.%2BK.%26rft.volume%253D107%26rft.issue%253D2%26rft.spage%253D155%26rft.epage%253D166%26rft.atitle%253DThe%2Bgenetic%2Bbasis%2Bof%2Bdivergent%2Bpigment%2Bpatterns%2Bin%2Bjuvenile%2Bthreespine%2Bsticklebacks.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2011.1%26rft_id%253Dinfo%253Apmid%252F21304547%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2011.1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21304547&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1027.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000292911500007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1027.25" data-doi="10.1007/BF02983075"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Haldane</span>  <span class="cit-name-given-names">J. B. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1922</span> <span class="cit-article-title">Sex ratio and unisexual sterility in hybrid animals.</span> <abbr class="cit-jnl-abbrev">J. Genet.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">101</span>–<span class="cit-lpage">109</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/BF02983075</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BGenet.%26rft.volume%253D12%26rft.spage%253D101%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF02983075%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF02983075&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=null&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1027.26" data-doi="10.1016/j.aquaculture.2008.06.042"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Handeland</span>  <span class="cit-name-given-names">S. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Imsland</span>  <span class="cit-name-given-names">A. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stefansson</span>  <span class="cit-name-given-names">S. O.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">The effect of temperature and fish size on growth, feed intake, food conversion efficiency and stomach evacuation rate of Atlantic salmon post-smolts.</span> <abbr class="cit-jnl-abbrev">Aquaculture</abbr> <span class="cit-vol">283</span>: <span class="cit-fpage">36</span>–<span class="cit-lpage">42</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.aquaculture.2008.06.042</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAquaculture%26rft.volume%253D283%26rft.spage%253D36%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.aquaculture.2008.06.042%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.aquaculture.2008.06.042&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000260567600006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-book" id="cit-9.4.1027.27"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hilario</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">The restriction enzyme target approach to Genotyping by Sequencing (GBS)</span>, pp. <span class="cit-fpage">271</span>–<span class="cit-lpage">279</span> in <span class="cit-source"><em>Plant Genotyping</em>, <em>Methods in Molecular Biology</em></span>, <span class="cit-publ-name">Humana Press</span>, <span class="cit-publ-loc">New York, NY</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1027.28" data-doi="10.1111/j.1365-2052.2009.01883.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Houston</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bishop</span>  <span class="cit-name-given-names">S. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hamilton</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guy</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tinch</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Detection of QTL affecting harvest traits in a commercial Atlantic salmon population.</span> <abbr class="cit-jnl-abbrev">Anim. Genet.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">753</span>–<span class="cit-lpage">755</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1365-2052.2009.01883.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnimal%2Bgenetics%26rft.stitle%253DAnim%2BGenet%26rft.aulast%253DHouston%26rft.auinit1%253DR.%2BD.%26rft.volume%253D40%26rft.issue%253D5%26rft.spage%253D753%26rft.epage%253D755%26rft.atitle%253DDetection%2Bof%2BQTL%2Baffecting%2Bharvest%2Btraits%2Bin%2Ba%2Bcommercial%2BAtlantic%2Bsalmon%2Bpopulation.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2052.2009.01883.x%26rft_id%253Dinfo%253Apmid%252F19397515%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-2052.2009.01883.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19397515&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1027.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1027.29" data-doi="10.2527/jas.2013-7191"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hutson</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kucuktas</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Umali-Maceina</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Su</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Quantitative trait loci map for growth and morphometric traits using a channel catfish × blue catfish interspecific hybrid system.</span> <abbr class="cit-jnl-abbrev">J. Anim. Sci.</abbr> <span class="cit-vol">92</span>: <span class="cit-fpage">1850</span>–<span class="cit-lpage">1865</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2527/jas.2013-7191</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAnim.%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.2527%252Fjas.2013-7191%26rft_id%253Dinfo%253Apmid%252F24663183%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2527/jas.2013-7191&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24663183&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1027.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1027.30" data-doi="10.1111/j.1365-294X.2007.03089.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kalinowski</span>  <span class="cit-name-given-names">S. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taper</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marshall</span>  <span class="cit-name-given-names">T. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Revising how the computer program cervus accommodates genotyping error increases success in paternity assignment.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">1099</span>–<span class="cit-lpage">1106</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1365-294X.2007.03089.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DKalinowski%26rft.auinit1%253DS.%2BT.%26rft.volume%253D16%26rft.issue%253D5%26rft.spage%253D1099%26rft.epage%253D1106%26rft.atitle%253DRevising%2Bhow%2Bthe%2Bcomputer%2Bprogram%2BCERVUS%2Baccommodates%2Bgenotyping%2Berror%2Bincreases%2Bsuccess%2Bin%2Bpaternity%2Bassignment.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2007.03089.x%26rft_id%253Dinfo%253Apmid%252F17305863%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2007.03089.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17305863&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1027.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000244245300014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1027.31" data-doi="10.1093/bioinformatics/bts199"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kearse</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moir</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilson</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stones-Havas</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheung</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Geneious Basic: An integrated and extendable desktop software platform for the organization and analysis of sequence data.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">1647</span>–<span class="cit-lpage">1649</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/bts199</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbts199%26rft_id%253Dinfo%253Apmid%252F22543367%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bts199&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22543367&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1027.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000305419800052&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1027.32" data-doi="10.1007/s001220050249"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Knott</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Elsen</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haley</span>  <span class="cit-name-given-names">C. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Methods for multiple-marker mapping of quantitative trait loci in half-sib populations.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">93</span>: <span class="cit-fpage">71</span>–<span class="cit-lpage">80</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/BF00225729</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D93%26rft.spage%253D71%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs001220050249%26rft_id%253Dinfo%253Apmid%252F24162201%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s001220050249&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24162201&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1027.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1996VA17700011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-9.4.1027.33"><div class="cit-metadata unstructured">Ko, W., R. M. David, H. Akashi, 2003 Molecular phylogeny of the Drosophila melanogaster species subgroup. Journal of Molecular Evolution. 57: 562–573. <a href="https://doi.org/10.1007/s00239-003-2510-x">https://doi.org/10.1007/s00239-003-2510-x</a></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1027.34" data-doi="10.1534/genetics.108.098855"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kucuktas</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">He</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Construction of genetic linkage maps and comparative genome analysis of Catfish using gene-associated markers.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">181</span>: <span class="cit-fpage">1649</span>–<span class="cit-lpage">1660</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.108.098855</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.108.098855%26rft_id%253Dinfo%253Apmid%252F19171943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE4MS80LzE2NDkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzQvMTAyNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1027.35" data-doi="10.1038/nrg3404"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lehner</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genotype to phenotype: lessons from model organisms for human genetics.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">168</span>–<span class="cit-lpage">178</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg3404</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BGenet.%26rft.volume%253D14%26rft.spage%253D168%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3404%26rft_id%253Dinfo%253Apmid%252F23358379%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3404&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23358379&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1027.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1027.36" data-doi="10.1093/dnares/dsu038"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Qin</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Waldbieser</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Construction of a high-density, high-resolution genetic map and its integration with BAC-based physical map in channel catfish.</span> <abbr class="cit-jnl-abbrev">DNA Res.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">39</span>–<span class="cit-lpage">52</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/dnares/dsu038</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDNA%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fdnares%252Fdsu038%26rft_id%253Dinfo%253Apmid%252F25428894%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/dnares/dsu038&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25428894&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1027.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-book" id="cit-9.4.1027.37"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Walsh</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-source">Genetics and analysis of quantitative traits</span>, <span class="cit-publ-name">Sinauer Sunderland</span>, <span class="cit-publ-loc">MA</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-book" id="cit-9.4.1027.38"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harley</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-source">Zoology</span>, <span class="cit-publ-name">McGraw-Hill Higher Education</span>, <span class="cit-publ-loc">Philadelphia, PA</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1027.39" data-doi="10.2307/1536193"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Morgan</span>  <span class="cit-name-given-names">T. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1914</span> <span class="cit-article-title">No Crossing over in the Male of Drosophila of Genes in the Second and Third Pairs of Chromosomes.</span> <abbr class="cit-jnl-abbrev">Biol. Bull.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">195</span>–<span class="cit-lpage">204</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2307/1536193</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BBiological%2BBulletin%26rft.stitle%253DBiol.%2BBull.%26rft.aulast%253DMORGAN%26rft.auinit1%253DT.%2BH.%26rft.volume%253D26%26rft.issue%253D4%26rft.spage%253D195%26rft.epage%253D204%26rft.atitle%253DNO%2BCROSSING%2BOVER%2BIN%2BTHE%2BMALE%2BOF%2BDROSOPHILA%2BOF%2BGENES%2BIN%2BTHE%2BSECOND%2BAND%2BTHIRD%2BPAIRS%2BOF%2BCHROMOSOMES%26rft_id%253Dinfo%253Adoi%252F10.2307%252F1536193%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/1536193&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000201827800001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1027.40" data-doi="10.2331/fishsci.62.845"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Murata</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harada</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miyashita</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Izumi</span>  <span class="cit-name-given-names">K.-I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maeda</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Selective breeding for growth in red sea bream.</span> <abbr class="cit-jnl-abbrev">Fish. Sci.</abbr> <span class="cit-vol">62</span>: <span class="cit-fpage">845</span>–<span class="cit-lpage">849</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2331/fishsci.62.845</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFish.%2BSci.%26rft.volume%253D62%26rft.spage%253D845%26rft_id%253Dinfo%253Adoi%252F10.2331%252Ffishsci.62.845%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2331/fishsci.62.845&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1027.41" data-doi="10.1534/g3.112.003962"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ninwichian</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peatman</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kucuktas</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Somridhivej</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> Second-generation genetic linkage map of catfish and its integration with the BAC-based physical map. G3: (Bethesda)2: 1233–1241.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.112.003962</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2526%252358%253B%2BGenes%257CGenomes%257CGenetics%26rft.stitle%253DG3%26rft.aulast%253DNinwichian%26rft.auinit1%253DP.%26rft.volume%253D2%26rft.issue%253D10%26rft.spage%253D1233%26rft.epage%253D1241%26rft.atitle%253DSecond-Generation%2BGenetic%2BLinkage%2BMap%2Bof%2BCatfish%2Band%2BIts%2BIntegration%2Bwith%2Bthe%2BBAC-Based%2BPhysical%2BMap%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.112.003962%26rft_id%253Dinfo%253Apmid%252F23050234%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjk6IjIvMTAvMTIzMyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzkvNC8xMDI3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1027.42" data-doi="10.12717/DR.2016.20.4.305"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Park</span>  <span class="cit-name-given-names">I.-S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gil</span>  <span class="cit-name-given-names">H. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">T. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nam</span>  <span class="cit-name-given-names">Y. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">D. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Comparative study of growth and gonad maturation in diploid and triploid marine medaka, <em>Oryzias dancena</em>.</span> <abbr class="cit-jnl-abbrev">Dev. Reprod.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">305</span>–<span class="cit-lpage">314</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.12717/DR.2016.20.4.305</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDev.%2BReprod.%26rft.volume%253D20%26rft.spage%253D305%26rft_id%253Dinfo%253Adoi%252F10.12717%252FDR.2016.20.4.305%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.12717/DR.2016.20.4.305&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1027.43" data-doi="10.1080/00288330.2014.892013"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Parsons</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sim-Smith</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cryer</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Francis</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hartill</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Snapper (<em>Chrysophrys auratus</em>): a review of life history and key vulnerabilities in New Zealand.</span> <abbr class="cit-jnl-abbrev">N. Z. J. Mar. Freshw. Res.</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">256</span>–<span class="cit-lpage">283</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1080/00288330.2014.892013</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DN.%2BZ.%2BJ.%2BMar.%2BFreshw.%2BRes.%26rft.volume%253D48%26rft.spage%253D256%26rft_id%253Dinfo%253Adoi%252F10.1080%252F00288330.2014.892013%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/00288330.2014.892013&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1027.44" data-doi="10.2527/1999.7792352x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">te Pas</span>  <span class="cit-name-given-names">M. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Soumillion</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harders</span>  <span class="cit-name-given-names">F. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Verburg</span>  <span class="cit-name-given-names">F. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van den Bosch</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Influences of myogenin genotypes on birth weight, growth rate, carcass weight, backfat thickness, and lean weight of pigs.</span> <abbr class="cit-jnl-abbrev">J. Anim. Sci.</abbr> <span class="cit-vol">77</span>: <span class="cit-fpage">2352</span>–<span class="cit-lpage">2356</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2527/1999.7792352x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BAnimal%2BScience%26rft.stitle%253DJ%2BANIM%2BSCI%26rft.aulast%253Dte%2BPas%26rft.auinit1%253DM.%2BF.%26rft.volume%253D77%26rft.issue%253D9%26rft.spage%253D2352%26rft.epage%253D2356%26rft.atitle%253DInfluences%2Bof%2Bmyogenin%2Bgenotypes%2Bon%2Bbirth%2Bweight%252C%2Bgrowth%2Brate%252C%2Bcarcass%2Bweight%252C%2Bbackfat%2Bthickness%252C%2Band%2Blean%2Bweight%2Bof%2Bpigs%26rft_id%253Dinfo%253Adoi%252F10.2527%252F1999.7792352x%26rft_id%253Dinfo%253Apmid%252F10492439%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2527/1999.7792352x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10492439&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1027.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1027.45" data-doi="10.1038/srep41748"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pértille</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moreira</span>  <span class="cit-name-given-names">G. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zanella</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nunes</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boschiero</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Genome-wide association study for performance traits in chickens using genotype by sequencing approach.</span> <abbr class="cit-jnl-abbrev">Sci. Rep.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">41748</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/srep41748</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BRep.%26rft.volume%253D7%26rft.spage%253D41748%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsrep41748%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/srep41748&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1027.46" data-doi="10.1086/519795"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Purcell</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neale</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Todd-Brown</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thomas</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferreira</span>  <span class="cit-name-given-names">M. A. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">PLINK: A tool set for whole-genome association and population-based linkage analyses.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">81</span>: <span class="cit-fpage">559</span>–<span class="cit-lpage">575</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1086/519795</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DPurcell%26rft.auinit1%253DS.%26rft.volume%253D81%26rft.issue%253D3%26rft.spage%253D559%26rft.epage%253D575%26rft.atitle%253DPLINK%253A%2Ba%2Btool%2Bset%2Bfor%2Bwhole-genome%2Bassociation%2Band%2Bpopulation-based%2Blinkage%2Banalyses.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F519795%26rft_id%253Dinfo%253Apmid%252F17701901%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/519795&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17701901&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1027.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-book" id="cit-9.4.1027.47"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">R Core Team</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-source">R: A language and environment for statistical computing</span>, <span class="cit-publ-name">R Foundation for Statistical Computing</span>, <span class="cit-publ-loc">Vienna, Austria</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1027.48" data-doi="10.1093/gbe/evv250"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rastas</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Calboli</span>  <span class="cit-name-given-names">F. C. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shikano</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Merilä</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Construction of ultradense linkage maps with Lep-MAP2: stickleback F2 recombinant crosses as an example.</span> <abbr class="cit-jnl-abbrev">Genome Biol. Evol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">78</span>–<span class="cit-lpage">93</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/gbe/evv250</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevv250%26rft_id%253Dinfo%253Apmid%252F26668116%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/gbe/evv250&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26668116&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1027.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1027.49" data-doi="10.1111/mec.12322"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Roesti</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moser</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berner</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Recombination in the threespine stickleback genome—patterns and consequences.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">3014</span>–<span class="cit-lpage">3027</span>. Erratum: 3652-3652.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/mec.12322</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D22%26rft.spage%253D3014%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.12322%26rft_id%253Dinfo%253Apmid%252F23601112.%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.12322&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23601112.&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1027.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000319675300013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1027.50" data-doi="10.1038/nrg3522"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Savolainen</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lascoux</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Merilä</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Ecological genomics of local adaptation.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">807</span>–<span class="cit-lpage">820</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg3522</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BGenet.%26rft.volume%253D14%26rft.spage%253D807%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3522%26rft_id%253Dinfo%253Apmid%252F24136507%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3522&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24136507&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1027.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-confproc" id="cit-9.4.1027.51"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Seaton</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hernandez-Sanchez</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grunchec</span>  <span class="cit-name-given-names">J.-A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">White</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Allen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> GridQTL: A grid portal for QTL mapping of compute intensive datasets, in <em>Proceedings of the 8th World Congress on Genetics Applied to Livestock Production (Belo horizonte</em>, <em>Brazil)</em></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1027.52" data-doi="10.1186/s12863-015-0215-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tsai</span>  <span class="cit-name-given-names">H. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hamilton</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guy</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tinch</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bishop</span>  <span class="cit-name-given-names">S. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">The genetic architecture of growth and fillet traits in farmed Atlantic salmon (<em>Salmo salar</em>).</span> <abbr class="cit-jnl-abbrev">BMC Genet.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">51</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12863-015-0215-y</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenet.%26rft.volume%253D16%26rft.spage%253D51%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12863-015-0215-y%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12863-015-0215-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1027.53" data-doi="10.1534/g3.116.029009"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tsai</span>  <span class="cit-name-given-names">H. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robledo</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lowe</span>  <span class="cit-name-given-names">N. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bekaert</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taggart</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, 2016 Construction and annotation of a high density SNP linkage map of the Atlantic salmon (<em>Salmo salar</em>) genome. G3: (Bethesda)6: 2173–2179.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.116.029009</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.116.029009%26rft_id%253Dinfo%253Apmid%252F27194803%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjg6IjYvNy8yMTczIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS80LzEwMjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1027.54" data-doi="10.3168/jds.2008-1514"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">VanRaden</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van Tassell</span>  <span class="cit-name-given-names">C. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wiggans</span>  <span class="cit-name-given-names">G. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sonstegard</span>  <span class="cit-name-given-names">T. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schnabel</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Invited Review: Reliability of genomic predictions for North American Holstein bulls.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">92</span>: <span class="cit-fpage">16</span>–<span class="cit-lpage">24</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3168/jds.2008-1514</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DVanRaden%26rft.auinit1%253DP.%2BM.%26rft.volume%253D92%26rft.issue%253D1%26rft.spage%253D16%26rft.epage%253D24%26rft.atitle%253DInvited%2BReview%253A%2BReliability%2Bof%2Bgenomic%2Bpredictions%2Bfor%2BNorth%2BAmerican%2BHolstein%2Bbulls%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2008-1514%26rft_id%253Dinfo%253Apmid%252F19109259%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2008-1514&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19109259&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1027.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000261948600002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1027.55" data-doi="10.1007/s11515-009-0031-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Quantitative genetic estimates of growth-related traits in the common carp (<em>Cyprinus carpio L</em>.): A review.</span> <abbr class="cit-jnl-abbrev">Front. Biol. China</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">298</span>–<span class="cit-lpage">304</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s11515-009-0031-8</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BBiol.%2BChina%26rft.volume%253D4%26rft.spage%253D298%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11515-009-0031-8%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11515-009-0031-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1027.56" data-doi="10.1186/s12864-017-3899-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bai</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">S. Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wan</span>  <span class="cit-name-given-names">Z. Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Construction of high-resolution recombination maps in Asian seabass.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">14</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D18%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-017-3899-8%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12864-017-3899-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1027.57" data-doi="10.1016/j.tig.2015.12.004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wellenreuther</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hansson</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Detecting polygenic evolution: problems, pitfalls, and promises.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">155</span>–<span class="cit-lpage">164</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.tig.2015.12.004</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BGenet.%26rft.volume%253D32%26rft.spage%253D155%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2015.12.004%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tig.2015.12.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1027.58" data-doi="10.1016/j.aaf.2017.06.001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ye</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wan</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wen</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Heritability of growth traits in the Asian seabass (<em>Lates calcarifer</em>).</span> <abbr class="cit-jnl-abbrev">Aquaculture and Fisheries</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">112</span>–<span class="cit-lpage">118</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.aaf.2017.06.001</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAquaculture%2Band%2BFisheries%26rft.volume%253D2%26rft.spage%253D112%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.aaf.2017.06.001%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.aaf.2017.06.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/9/4/1027.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/4/1009" title="A Large Scale Systemic RNAi Screen in the Red Flour Beetle Tribolium castaneum Identifies Novel Genes Involved in Insect Muscle Development" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/4/1037" title="Analysis of Genetic Diversity and Population Structure in Three Forest Musk Deer Captive Populations with Different Origins" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/9/4"><span>Volume 9 Issue 4, April 2019</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/9/4.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/9/4.cover-source.jpg?itok=mcy5SuIK" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 9 (4)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/4/1027" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-i4jwYF5gGoplPNZIdHUhVXa8RegAbYcXItw7pvDPCwI" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/9/4/1027" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/9/4/1027" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/9/4/1027" class="active" data-icon-position="" data-hide-link-title="0">High-Density Linkage Map and QTLs for Growth in Snapper (Chrysophrys auratus)</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/492612" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-89-kQWZGbzqHuV7ihKr4upywDOj3D79x5kns0qqip5k" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/4/1027.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/9/4/1027" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-Y4BEmUkfzauQto1nkx-op5dEc8riBAJ4Reqnxj4w7X4" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/9/4/1027" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/4/1027.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="492612" id="node492612--2" data-pisa="ggg;9/4/1027" data-pisa-master="ggg;g3.118.200905" data-apath="/ggg/9/4/1027.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">High-Density Linkage Map and QTLs for Growth in Snapper (<em>Chrysophrys auratus</em>)</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-0996-6368" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>David T. Ashton</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-8351-7931" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Peter A. Ritchie</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-2764-8291" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Maren Wellenreuther</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1027-1035;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200905 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node492612--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">David T. Ashton </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The New Zealand Institute for Plant & Food Research Limited, Nelson, New Zealand</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BT.%2BAshton%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ashton%20DT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADavid%2BT.%2BAshton%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-0996-6368" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for David T. Ashton</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:david.ashton@plantandfood.co.nz" class="" data-icon-position="" data-hide-link-title="0">david.ashton@plantandfood.co.nz</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Peter A. Ritchie </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>School of Biological Sciences, Victoria University of Wellington, Wellington, New Zealand</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Peter%2BA.%2BRitchie%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ritchie%20PA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APeter%2BA.%2BRitchie%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-8351-7931" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Peter A. Ritchie</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Maren Wellenreuther </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The New Zealand Institute for Plant & Food Research Limited, Nelson, New Zealand</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Maren%2BWellenreuther%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wellenreuther%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMaren%2BWellenreuther%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-2764-8291" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Maren Wellenreuther</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/9/4/1027" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/9/4/1027&amp;title=High-Density%20Linkage%20Map%20and%20QTLs%20for%20Growth%20in%20Snapper%20%28Chrysophrys%20auratus%29" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/9/4/1027&amp;title=High-Density%20Linkage%20Map%20and%20QTLs%20for%20Growth%20in%20Snapper%20%28Chrysophrys%20auratus%29" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/9/4/1027&amp;title=High-Density%20Linkage%20Map%20and%20QTLs%20for%20Growth%20in%20Snapper%20%28Chrysophrys%20auratus%29" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/9/4/1027&amp;text=High-Density%20Linkage%20Map%20and%20QTLs%20for%20Growth%20in%20Snapper%20%28Chrysophrys%20auratus%29" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/9/4/1027&amp;title=High-Density%20Linkage%20Map%20and%20QTLs%20for%20Growth%20in%20Snapper%20%28Chrysophrys%20auratus%29" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/9/4/1027&amp;t=High-Density%20Linkage%20Map%20and%20QTLs%20for%20Growth%20in%20Snapper%20%28Chrysophrys%20auratus%29" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/9/4/1027&amp;title=High-Density%20Linkage%20Map%20and%20QTLs%20for%20Growth%20in%20Snapper%20%28Chrysophrys%20auratus%29" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/9/4/1027&amp;title=High-Density%20Linkage%20Map%20and%20QTLs%20for%20Growth%20in%20Snapper%20%28Chrysophrys%20auratus%29" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/492612/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="492612" id="node492612--3" data-pisa="ggg;9/4/1027" data-pisa-master="ggg;g3.118.200905" data-apath="/ggg/9/4/1027.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">High-Density Linkage Map and QTLs for Growth in Snapper (<em>Chrysophrys auratus</em>)</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-0996-6368" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>David T. Ashton</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-8351-7931" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Peter A. Ritchie</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-2764-8291" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Maren Wellenreuther</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1027-1035;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200905 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node492612--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">David T. Ashton </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The New Zealand Institute for Plant & Food Research Limited, Nelson, New Zealand</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BT.%2BAshton%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ashton%20DT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADavid%2BT.%2BAshton%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-0996-6368" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for David T. Ashton</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:david.ashton@plantandfood.co.nz" class="" data-icon-position="" data-hide-link-title="0">david.ashton@plantandfood.co.nz</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Peter A. Ritchie </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>School of Biological Sciences, Victoria University of Wellington, Wellington, New Zealand</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Peter%2BA.%2BRitchie%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ritchie%20PA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APeter%2BA.%2BRitchie%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-8351-7931" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Peter A. Ritchie</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Maren Wellenreuther </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>The New Zealand Institute for Plant & Food Research Limited, Nelson, New Zealand</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Maren%2BWellenreuther%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wellenreuther%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMaren%2BWellenreuther%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-2764-8291" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Maren Wellenreuther</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1027&amp;rft.epage=1035&amp;rft.atitle=High-Density%20Linkage%20Map%20and%20QTLs%20for%20Growth%20in%20Snapper%20%28Chrysophrys%20auratus%29&amp;rft.volume=9&amp;rft.issue=4&amp;rft.date=2019-04-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Ashton%2C+David+T.&amp;rft.au=Ritchie%2C+Peter+A.&amp;rft.au=Wellenreuther%2C+Maren"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/492612/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/492612/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/492612/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/492612/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/492612/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/492612/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/492612/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/492612/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/492612/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/492612/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/492612/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/492612/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/9/4/1027" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-9" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-15" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">ACKNOWLEDGMENTS</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/9/4/1027.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/9/4/1027.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/9/4/1027.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
