<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/9/4/1249" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/9/4/1249.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/9/4/1249.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/9/4/1249.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/9/4/1249.atom" />
<meta name="HW.pisa" content="ggg;9/4/1249" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Random Chromosome Partitioning in the Polyploid Bacterium Thermus thermophilus HB27" />
<meta name="DC.Identifier" content="10.1534/g3.119.400086" />
<meta name="DC.Date" content="2019-04-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2019 Li. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Relation" content="https://doi.org/10.25387/g3.7746878" />
<meta name="DC.Description" content="Little is known about chromosome segregation in polyploid prokaryotes. In this study, whether stringent or variable chromosome segregation occurs in polyploid thermophilic bacterium Thermus thermophilus was analyzed. A stable heterozygous strain (HL01) containing two antibiotic resistance markers at one gene locus was generated. The inheritance of the two alleles in the progeny of the heterozygous strain was then followed. During incubation without selection pressure, the fraction of heterozygous cells decreased and that of homozygous cells increased, while the relative abundance of each allele in the whole population remained constant, suggesting chromosome segregation had experienced random event. Consistently, in comparison with Bacillus subtilis in which the sister chromosomes were segregated equally, the ratios of DNA content in two daughter cells of T. thermophilus had a broader distribution and a larger standard deviation, indicating that the DNA content in the two daughter cells was not always identical. Further, the protein homologs ( i.e. , ParA and MreB) which have been suggested to be involved in bacterial chromosome partitioning did not actively participate in the chromosome segregation in T. thermophilus . Therefore, it seems that protein-based chromosome segregation machineries are less critical for the polyploid T. thermophilus , and chromosome segregation in this bacterium are not stringently controlled but tend to be variable, and random segregation can occur." />
<meta name="DC.Contributor" content="Haijuan Li" />
<meta name="article:published_time" content="2019-04-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Random Chromosome Partitioning in the Polyploid Bacterium Thermus thermophilus HB27" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Little is known about chromosome segregation in polyploid prokaryotes. In this study, whether stringent or variable chromosome segregation occurs in polyploid thermophilic bacterium &lt;i&gt;Thermus thermophilus&lt;/i&gt; was analyzed. A stable heterozygous strain (HL01) containing two antibiotic resistance markers at one gene locus was generated. The inheritance of the two alleles in the progeny of the heterozygous strain was then followed. During incubation without selection pressure, the fraction of heterozygous cells decreased and that of homozygous cells increased, while the relative abundance of each allele in the whole population remained constant, suggesting chromosome segregation had experienced random event. Consistently, in comparison with &lt;i&gt;Bacillus subtilis&lt;/i&gt; in which the sister chromosomes were segregated equally, the ratios of DNA content in two daughter cells of &lt;i&gt;T. thermophilus&lt;/i&gt; had a broader distribution and a larger standard deviation, indicating that the DNA content in the two daughter cells was not always identical. Further, the protein homologs (&lt;i&gt;i.e.&lt;/i&gt;, ParA and MreB) which have been suggested to be involved in bacterial chromosome partitioning did not actively participate in the chromosome segregation in &lt;i&gt;T. thermophilus&lt;/i&gt;. Therefore, it seems that protein-based chromosome segregation machineries are less critical for the polyploid &lt;i&gt;T. thermophilus&lt;/i&gt;, and chromosome segregation in this bacterium are not stringently controlled but tend to be variable, and random segregation can occur.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2019/04/01" />
<meta name="citation_mjid" content="ggg;9/4/1249" />
<meta name="citation_id" content="9/4/1249" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/9/4/1249" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/9/4/1249.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/9/4/1249.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/9/4/1249.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.119.400086" />
<meta name="citation_pmid" content="30792193" />
<meta name="citation_volume" content="9" />
<meta name="citation_issue" content="4" />
<meta name="citation_num_pages" content="13" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="1249" />
<meta name="citation_lastpage" content="1261" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Haijuan Li" />
<meta name="citation_author_institution" content="College of Biological and Environmental Engineering, Xi’an University, Xi’an 710065, Shaanxi, China" />
<meta name="citation_author_email" content="haijuanli@xawl.edu.cn" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-9428-6190" />
<meta name="citation_reference" content="citation_journal_title=Appl. Environ. Microbiol.;citation_author=L. Alvarez;citation_author=C. Bricio;citation_author=M. J. Gómez;citation_author=J. Berenguer;citation_title=Lateral transfer of the denitrification pathway genes among Thermus thermophilus strains.;citation_volume=77;citation_year=2011;citation_pmid=21169443;citation_doi=10.1128/AEM.02048-10" />
<meta name="citation_reference" content="citation_journal_title=Syst. Appl. Microbiol.;citation_author=A. Angelov;citation_author=H. Li;citation_author=A. Geissler;citation_author=B. Leis;citation_author=W. Liebl;citation_title=Toxicity of indoxyl derivative accumulation in bacteria and its use as a new counterselection principle.;citation_pages=585-592;citation_volume=36;citation_year=2013;citation_doi=10.1016/j.syapm.2013.06.001" />
<meta name="citation_reference" content="citation_journal_title=Systematic and Applied Microbiology;citation_journal_abbrev=Systematic and Applied Microbiology;citation_author=A. Angelov;citation_author=M. Mientus;citation_author=S. Liebl;citation_author=W. Liebl;citation_title=A two-host fosmid system for functional screening of (meta)genomic libraries from extreme thermophiles.;citation_pages=177-185;citation_volume=32;citation_year=2009;citation_issue=3;citation_pmid=19285378;citation_doi=10.1016/j.syapm.2008.01.003" />
<meta name="citation_reference" content="citation_journal_title=MBio;citation_author=L. Attaiech;citation_author=A. Minnen;citation_author=M. Kjos;citation_author=S. Gruber;citation_author=J. W. Veening;citation_title=The ParB-parS chromosome segregation system modulates competence development in Streptococcus pneumoniae.;citation_pages=e00662;citation_volume=6;citation_year=2015;citation_pmid=26126852;citation_doi=10.1128/mBio.00662-15" />
<meta name="citation_reference" content="citation_journal_title=Annu. Rev. Cell Dev. Biol.;citation_author=A. Badrinarayanan;citation_author=T. B. Le;citation_author=M. T. Laub;citation_title=Bacterial chromosome organization and segregation.;citation_pages=171-199;citation_volume=31;citation_year=2015;citation_pmid=26566111;citation_doi=10.1146/annurev-cellbio-100814-125211" />
<meta name="citation_reference" content="citation_journal_title=MBio;citation_author=J. H. Baek;citation_author=S. V. Rajagopala;citation_author=D. K. Chattoraj;citation_title=Chromosome segregation proteins of Vibrio cholerae as transcription regulators.;citation_volume=5;citation_year=2014;citation_pmid=24803519;citation_doi=10.1128/mBio.01061-14" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=A. A. Bartosik;citation_author=K. Glabski;citation_author=P. Jecz;citation_author=S. Mikulska;citation_author=A. Fogtman;citation_title=Transcriptional profiling of ParA and ParB mutants in actively dividing cells of an opportunistic human pathogen Pseudomonas aeruginosa.;citation_pages=e87276;citation_volume=9;citation_year=2014;citation_pmid=24498062;citation_doi=10.1371/journal.pone.0087276" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=S. Breuert;citation_author=T. Allers;citation_author=G. Spohn;citation_author=J. Soppa;citation_title=Regulated polyploidy in halophilic archaea.;citation_pages=e92-e92;citation_volume=1;citation_year=2006;citation_pmid=17183724;citation_doi=10.1371/journal.pone.0000092" />
<meta name="citation_reference" content="citation_journal_title=J. Biol. Chem.;citation_author=S. J. Brouns;citation_author=H. Wu;citation_author=J. Akerboom;citation_author=A. P. Turnbull;citation_author=W. M. de Vos;citation_title=Engineering a selectable marker for hyperthermophiles.;citation_volume=280;citation_year=2005;citation_pmid=15640151;citation_doi=10.1074/jbc.M413623200" />
<meta name="citation_reference" content="citation_journal_title=Biophys. J.;citation_author=J. A. Cass;citation_author=N. J. Kuwada;citation_author=B. Traxler;citation_author=P. A. Wiggins;citation_title=Escherichia coli chromosomal loci segregate from midcell with universal dynamics.;citation_pages=2597-2609;citation_volume=110;citation_year=2016;citation_doi=10.1016/j.bpj.2016.04.046" />
<meta name="citation_reference" content="citation_journal_title=International Microbiology;citation_journal_abbrev=International Microbiology;citation_author=C. E. César;citation_author=L. Álvarez;citation_author=C. Bricio;citation_author=E. van Heerden;citation_author=D. Littauer;citation_title=Unconventional lateral gene transfer in extreme thermophilic bacteria.;citation_pages=187-199;citation_volume=14;citation_year=2011;citation_issue=4;citation_pmid=22569756;citation_doi=10.2436/20.1501.01.148" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=L. Comai;citation_title=The advantages and disadvantages of being polyploid.;citation_pages=836-846;citation_volume=6;citation_year=2005;citation_issue=11;citation_pmid=16304599;citation_doi=10.1038/nrg1711" />
<meta name="citation_reference" content="citation_journal_title=Plasmid;citation_journal_abbrev=Plasmid;citation_author=M. de Grado;citation_author=P. Castan;citation_author=J. Berenguer;citation_title=A high-transformation-efficiency cloning vector for Thermus thermophilus.;citation_pages=241-245;citation_volume=42;citation_year=1999;citation_issue=3;citation_pmid=10545266;citation_doi=10.1006/plas.1999.1427" />
<meta name="citation_reference" content="citation_journal_title=J. Bacteriol.;citation_author=C. Donovan;citation_author=A. Schwaiger;citation_author=R. Krämer;citation_author=M. Bramkamp;citation_title=Subcellular localization and characterization of the ParAB system from Corynebacterium glutamicum.;citation_volume=192;citation_year=2010;citation_pmid=20435732;citation_doi=10.1128/JB.00214-10" />
<meta name="citation_reference" content="citation_journal_title=Annual review of genetics;citation_journal_abbrev=Annu Rev Genet;citation_author=G. Ebersbach;citation_author=K. Gerdes;citation_title=Plasmid segregation mechanisms.;citation_pages=453-479;citation_volume=39;citation_year=2005;citation_pmid=16285868;citation_doi=10.1146/annurev.genet.38.072902.091252" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_author=J. K. Fisher;citation_author=A. Bourniquel;citation_author=G. Witz;citation_author=B. Weiner;citation_author=M. Prentiss;citation_title=Four-dimensional imaging of E. coli nucleoid organization and dynamics in living cells.;citation_pages=882-895;citation_volume=153;citation_year=2013;citation_pmid=23623305;citation_doi=10.1016/j.cell.2013.04.006" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=M. A. Fogel;citation_author=M. K. Waldor;citation_title=A dynamic, mitotic-like mechanism for bacterial chromosome segregation;citation_pages=3269-3282;citation_volume=20;citation_year=2006;citation_issue=23;citation_pmid=17158745;citation_doi=10.1101/gad.1496506" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=D. G. Gibson;citation_author=L. Young;citation_author=R. Y. Chuang;citation_author=J. C. Venter;citation_author=C. A. Hutchison;citation_title=Enzymatic assembly of DNA molecules up to several hundred kilobases.;citation_pages=343-345;citation_volume=6;citation_year=2009;citation_issue=5;citation_pmid=19363495;citation_doi=10.1038/nmeth.1318" />
<meta name="citation_reference" content="citation_journal_title=Mol. Microbiol.;citation_author=K. Ginda;citation_author=M. Bezulska;citation_author=M. Zió-lkiewicz;citation_author=J. Dziadek;citation_author=J. Zakrzewska-Czerwińska;citation_title=ParA of Mycobacterium smegmatis co-ordinates chromosome segregation with the cell cycle and interacts with the polar growth determinant DivIVA.;citation_pages=998-1012;citation_volume=87;citation_year=2013;citation_pmid=23289458;citation_doi=10.1111/mmi.12146" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=Z. Gitai;citation_author=N. A. Dye;citation_author=A. Reisenauer;citation_author=M. Wachi;citation_author=L. Shapiro;citation_title=MreB actin-mediated segregation of a specific region of a bacterial chromosome.;citation_pages=329-341;citation_volume=120;citation_year=2005;citation_issue=3;citation_pmid=15707892;citation_doi=10.1016/j.cell.2005.01.007" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=Z. Gitai;citation_author=N. Dye;citation_author=L. Shapiro;citation_title=An actin-like gene can determine cell polarity in bacteria.;citation_volume=101;citation_year=2004;citation_pmid=15159537;citation_doi=10.1073/pnas.0402638101" />
<meta name="citation_reference" content="citation_title=Bacterial Chromatin;citation_author=P. L. Graumann;citation_title=The chromosome segregation machinery in bacteria;citation_year=2010;citation_doi=10.1007/978-90-481-3473-1_3" />
<meta name="citation_reference" content="citation_journal_title=FEMS Microbiol. Lett.;citation_author=M. Griese;citation_author=C. Lange;citation_author=J. Soppa;citation_title=Ploidy in cyanobacteria.;citation_volume=323;citation_year=2011;citation_pmid=22092711;citation_doi=10.1111/j.1574-6968.2011.02368.x" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=S. Gruber;citation_author=J. Errington;citation_title=Recruitment of condensin to replication origin regions by ParB/SpoOJ promotes chromosome segregation in B. subtilis.;citation_pages=685-696;citation_volume=137;citation_year=2009;citation_issue=4;citation_pmid=19450516;citation_doi=10.1016/j.cell.2009.02.035" />
<meta name="citation_reference" content="citation_journal_title=Sci. Rep.;citation_author=N. Guschinskaya;citation_author=R. Brunel;citation_author=M. Tourte;citation_author=G. L. Lipscomb;citation_author=M. W. Adams;citation_title=Random mutagenesis of the hyperthermophilic archaeon Pyrococcus furiosus using in vitro mariner transposition and natural transformation.;citation_pages=36711;citation_volume=6;citation_year=2016;citation_doi=10.1038/srep36711" />
<meta name="citation_reference" content="citation_journal_title=Journal of Bacteriology;citation_journal_abbrev=J. Bacteriol.;citation_author=M. T. Hansen;citation_title=Multiplicity of genome equivalents in the radiation-resistant bacterium Micrococcus radiodurans.;citation_pages=71-75;citation_volume=134;citation_year=1978;citation_issue=1;citation_pmid=649572" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=A. Henne;citation_author=H. Brüggemann;citation_author=C. Raasch;citation_author=A. Wiezer;citation_author=T. Hartsch;citation_title=The genome sequence of the extreme thermophile Thermus thermophilus.;citation_pages=547-553;citation_volume=22;citation_year=2004;citation_issue=5;citation_pmid=15064768;citation_doi=10.1038/nbt956" />
<meta name="citation_reference" content="citation_journal_title=J. Bacteriol.;citation_author=C. Hildenbrand;citation_author=T. Stock;citation_author=C. Lange;citation_author=M. Rother;citation_author=J. Soppa;citation_title=Genome copy numbers and gene conversion in methanogenic archaea.;citation_volume=193;citation_year=2011;citation_pmid=21097629;citation_doi=10.1128/JB.01016-10" />
<meta name="citation_reference" content="citation_journal_title=Molecular microbiology;citation_journal_abbrev=Mol Microbiol;citation_author=B. Hu;citation_author=G. Yang;citation_author=W. Zhao;citation_author=Y. Zhang;citation_author=J. Zhao;citation_title=MreB is important for cell shape but not for chromosome segregation of the filamentous cyanobacterium Anabaena sp. PCC 7120.;citation_pages=1640-1652;citation_volume=63;citation_year=2007;citation_issue=6;citation_pmid=17367385;citation_doi=10.1111/j.1365-2958.2007.05618.x" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=A. A. Iniesta;citation_title=ParABS system in chromosome partitioning in the bacterium Myxococcus xanthus.;citation_pages=e86897;citation_volume=9;citation_year=2014;citation_pmid=24466283;citation_doi=10.1371/journal.pone.0086897" />
<meta name="citation_reference" content="citation_author=K. Ireton;citation_year=1994" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=I. H. Jain;citation_author=V. Vijayan;citation_author=E. K. O’Shea;citation_title=Spatial ordering of chromosomes enhances the fidelity of chromosome partitioning in cyanobacteria.;citation_volume=109;citation_year=2012;citation_pmid=22869746;citation_doi=10.1073/pnas.1211144109" />
<meta name="citation_reference" content="citation_journal_title=Molecular microbiology;citation_journal_abbrev=Mol Microbiol;citation_author=D. Jakimowicz;citation_author=K. Chater;citation_author=J. Zakrzewska-Czerwínska;citation_title=The ParB protein of Streptomyces coelicolor A3(2) recognizes a cluster of parS sequences within the origin-proximal region of the linear chromosome.;citation_pages=1365-1377;citation_volume=45;citation_year=2002;citation_issue=5;citation_pmid=12207703;citation_doi=10.1046/j.1365-2958.2002.03102.x" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Microbiology;citation_journal_abbrev=Nat Rev Microbiol;citation_author=S. Jun;citation_author=A. Wright;citation_title=Entropy as the driver of chromosome segregation.;citation_pages=600-607;citation_volume=8;citation_year=2010;citation_issue=8;citation_pmid=20634810;citation_doi=10.1038/nrmicro2391" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=S. Jun;citation_author=B. Mulder;citation_title=Entropy-driven spatial organization of highly confined polymers: lessons for the bacterial chromosome.;citation_volume=103;citation_year=2006;citation_pmid=16885211;citation_doi=10.1073/pnas.0605305103" />
<meta name="citation_reference" content="citation_journal_title=The Plant journal :  for cell and molecular biology;citation_journal_abbrev=Plant J;citation_author=O. Khakhlova;citation_author=R. Bock;citation_title=Elimination of deleterious mutations in plastid genomes by gene conversion.;citation_pages=85-94;citation_volume=46;citation_year=2006;citation_issue=1;citation_pmid=16553897;citation_doi=10.1111/j.1365-313X.2006.02673.x" />
<meta name="citation_reference" content="citation_journal_title=Journal of Bacteriology;citation_journal_abbrev=J. Bacteriol.;citation_author=H. J. Kim;citation_author=M. J. Calcutt;citation_author=F. J. Schmidt;citation_author=K. F. Chater;citation_title=Partitioning of the Linear Chromosome during Sporulation of Streptomyces coelicolor A3(2) Involves an oriC-Linked parAB Locus;citation_pages=1313-1320;citation_volume=182;citation_year=2000;citation_issue=5;citation_pmid=10671452;citation_doi=10.1128/JB.182.5.1313-1320.2000" />
<meta name="citation_reference" content="citation_journal_title=Mol. Microbiol.;citation_author=T. G. Kloosterman;citation_author=R. Lenarcic;citation_author=C. R. Willis;citation_author=D. M. Roberts;citation_author=L. W. Hamoen;citation_title=Complex polar machinery required for proper chromosome segregation in vegetative and sporulating cells of Bacillus subtilis.;citation_pages=333-350;citation_volume=101;citation_year=2016;citation_doi=10.1111/mmi.13393" />
<meta name="citation_reference" content="citation_journal_title=Annual review of microbiology;citation_journal_abbrev=Annu Rev Microbiol;citation_author=R. Kolter;citation_author=D. A. Siegele;citation_author=A. Tormo;citation_title=The Stationary Phase of the Bacterial Life Cycle;citation_pages=855-874;citation_volume=47;citation_year=1993;citation_issue=1;citation_pmid=8257118;citation_doi=10.1146/annurev.mi.47.100193.004231" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=T. Kruse;citation_author=B. Blagoev;citation_author=A. Løbner-Olesen;citation_author=M. Wachi;citation_author=K. Sasaki;citation_title=Actin homolog MreB and RNA polymerase interact and are both required for chromosome segregation in Escherichia coli;citation_pages=113-124;citation_volume=20;citation_year=2006;citation_issue=1;citation_pmid=16391237;citation_doi=10.1101/gad.366606" />
<meta name="citation_reference" content="citation_journal_title=Molecular microbiology;citation_journal_abbrev=Mol Microbiol;citation_author=T. Kruse;citation_author=J. Bork-Jensen;citation_author=K. Gerdes;citation_title=The morphogenetic MreBCD proteins of Escherichia coli form an essential membrane-bound complex.;citation_pages=78-89;citation_volume=55;citation_year=2005;citation_issue=1;citation_pmid=15612918;citation_doi=10.1111/j.1365-2958.2004.04367.x" />
<meta name="citation_reference" content="citation_journal_title=The EMBO Journal;citation_journal_abbrev=EMBO Journal;citation_author=T. Kruse;citation_author=J. Møller-Jensen;citation_author=A. Løbner-Olesen;citation_author=K. Gerdes;citation_title=Dysfunctional MreB inhibits chromosome segregation in Escherichia coli;citation_pages=5283-5292;citation_volume=22;citation_year=2003;citation_issue=19;citation_pmid=14517265;citation_doi=10.1093/emboj/cdg504" />
<meta name="citation_reference" content="citation_journal_title=Biophys. J.;citation_author=T. J. Lampo;citation_author=N. J. Kuwada;citation_author=P. A. Wiggins;citation_author=A. J. Spakowitz;citation_title=Physical modeling of chromosome segregation in Escherichia coli reveals impact of force and DNA relaxation.;citation_pages=146-153;citation_volume=108;citation_year=2015;citation_pmid=25564861;citation_doi=10.1016/j.bpj.2014.10.074" />
<meta name="citation_reference" content="citation_journal_title=Molecular microbiology;citation_journal_abbrev=Mol Microbiol;citation_author=C. Lange;citation_author=K. Zerulla;citation_author=S. Breuert;citation_author=J. Soppa;citation_title=Gene conversion results in the equalization of genome copies in the polyploid haloarchaeon Haloferax volcanii.;citation_pages=666-677;citation_volume=80;citation_year=2011;citation_issue=3;citation_pmid=21338422;citation_doi=10.1111/j.1365-2958.2011.07600.x" />
<meta name="citation_reference" content="citation_journal_title=J. Bacteriol.;citation_author=K. Lasocki;citation_author=A. A. Bartosik;citation_author=J. Mierzejewska;citation_author=C. M. Thomas;citation_author=G. Jagura-Burdzy;citation_title=Deletion of the parA (soj) homologue in Pseudomonas aeruginosa causes ParB instability and affects growth rate, chromosome segregation, and motility.;citation_volume=189;citation_year=2007;citation_pmid=17545287;citation_doi=10.1128/JB.00371-07" />
<meta name="citation_reference" content="citation_journal_title=Molecular microbiology;citation_journal_abbrev=Mol Microbiol;citation_author=P. S. Lee;citation_author=A. D. Grossman;citation_title=The chromosome partitioning proteins Soj (ParA) and Spo0J (ParB) contribute to accurate chromosome partitioning, separation of replicated sister origins, and regulation of replication initiation in Bacillus subtilis.;citation_pages=853-869;citation_volume=60;citation_year=2006;citation_issue=4;citation_pmid=16677298;citation_doi=10.1111/j.1365-2958.2006.05140.x" />
<meta name="citation_reference" content="citation_journal_title=J. Biotechnol.;citation_author=B. Leis;citation_author=A. Angelov;citation_author=H. Li;citation_author=W. Liebl;citation_title=Genetic analysis of lipolytic activities in Thermus thermophilus HB27.;citation_pages=150-157;citation_volume=191;citation_year=2014;citation_doi=10.1016/j.jbiotec.2014.07.448" />
<meta name="citation_reference" content="citation_journal_title=Molecular microbiology;citation_journal_abbrev=Mol Microbiol;citation_author=P. J. Lewis;citation_author=J. Errington;citation_title=Direct evidence for active segregation of oriC regions of the Bacillus subtilis chromosome and co-localization with the SpoOJ partitioning protein.;citation_pages=945-954;citation_volume=25;citation_year=1997;citation_issue=5;citation_pmid=9364919;citation_doi=10.1111/j.1365-2958.1997.mmi530.x" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=H. Li;citation_author=A. Angelov;citation_author=V. T. Pham;citation_author=B. Leis;citation_author=W. Liebl;citation_title=Characterization of chromosomal and megaplasmid partitioning loci in Thermus thermophilus HB27.;citation_pages=317;citation_volume=16;citation_year=2015;citation_doi=10.1186/s12864-015-1523-3" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=D. C. Lin;citation_author=A. D. Grossman;citation_title=Identification and characterization of a bacterial chromosome partitioning site.;citation_pages=675-685;citation_volume=92;citation_year=1998;citation_issue=5;citation_pmid=9506522;citation_doi=10.1016/S0092-8674(00)81135-6" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=R. Madabhushi;citation_author=K. J. Marians;citation_title=Actin homolog MreB affects chromosome segregation by regulating topoisomerase IV in Escherichia coli.;citation_pages=171-180;citation_volume=33;citation_year=2009;citation_issue=2;citation_pmid=19187760;citation_doi=10.1016/j.molcel.2009.01.001" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=L. Malandrin;citation_author=H. Huber;citation_author=R. Bernander;citation_title=Nucleoid Structure and Partition in Methanococcus jannaschii: An Archaeon With Multiple Copies of the Chromosome;citation_pages=1315-1323;citation_volume=152;citation_year=1999;citation_issue=4;citation_pmid=10430562" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=J. E. Mendell;citation_author=K. D. Clements;citation_author=J. H. Choat;citation_author=E. R. Angert;citation_title=Extreme polyploidy in a large bacterium.;citation_volume=105;citation_year=2008;citation_pmid=18445653;citation_doi=10.1073/pnas.0707522105" />
<meta name="citation_reference" content="citation_journal_title=Molecular microbiology;citation_journal_abbrev=Mol Microbiol;citation_author=A. Minnen;citation_author=L. Attaiech;citation_author=M. Thon;citation_author=S. Gruber;citation_author=J. W. Veening;citation_title=SMC is recruited to oriC by ParB and promotes chromosome segregation in Streptococcus pneumoniae.;citation_pages=676-688;citation_volume=81;citation_year=2011;citation_issue=3;citation_pmid=21651626;citation_doi=10.1111/j.1365-2958.2011.07722.x" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=D. A. Mohl;citation_author=J. W. Gober;citation_title=Cell cycle-dependent polar localization of chromosome partitioning proteins in Caulobacter crescentus.;citation_pages=675-684;citation_volume=88;citation_year=1997;citation_issue=5;citation_pmid=9054507;citation_doi=10.1016/S0092-8674(00)81910-8" />
<meta name="citation_reference" content="citation_journal_title=Molecular microbiology;citation_journal_abbrev=Mol Microbiol;citation_author=D. A. Mohl;citation_author=J. Jr. Easter;citation_author=J. W. Gober;citation_title=The chromosome partitioning protein, ParB, is required for cytokinesis in Caulobacter crescentus.;citation_pages=741-755;citation_volume=42;citation_year=2001;citation_issue=3;citation_pmid=11722739;citation_doi=10.1046/j.1365-2958.2001.02643.x" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=H. Murray;citation_author=J. Errington;citation_title=Dynamic control of the DNA replication initiation protein DnaA by Soj/ParA.;citation_pages=74-84;citation_volume=135;citation_year=2008;citation_issue=1;citation_pmid=18854156;citation_doi=10.1016/j.cell.2008.07.044" />
<meta name="citation_reference" content="citation_journal_title=FEMS Microbiol. Rev.;citation_author=J. M. Navarro Llorens;citation_author=A. Tormo;citation_author=E. Martínez-García;citation_title=Stationary phase in gram-negative bacteria.;citation_volume=34;citation_year=2010;citation_pmid=20236330;citation_doi=10.1111/j.1574-6976.2010.00213.x" />
<meta name="citation_reference" content="citation_journal_title=Mutagenesis;citation_author=T. Ohta;citation_author=S. Tokishita;citation_author=R. Imazuka;citation_author=I. Mori;citation_author=J. Okamura;citation_title=beta-Glucosidase as a reporter for the gene expression studies in Thermus thermophilus and constitutive expression of DNA repair genes.;citation_volume=21;citation_year=2006;citation_pmid=16777922;citation_doi=10.1093/mutage/gel025" />
<meta name="citation_reference" content="citation_journal_title=J. Bacteriol.;citation_author=N. Ohtani;citation_author=M. Tomita;citation_author=M. Itaya;citation_title=An extreme thermophile, Thermus thermophilus, is a polyploid bacterium.;citation_volume=192;citation_year=2010;citation_pmid=20729360;citation_doi=10.1128/JB.00662-10" />
<meta name="citation_reference" content="citation_journal_title=Appl. Environ. Microbiol.;citation_author=N. Ohtani;citation_author=M. Tomita;citation_author=M. Itaya;citation_title=Curing the megaplasmid pTT27 from Thermus thermophilus HB27 and maintaining exogenous plasmids in the plasmid-free strain.;citation_volume=82;citation_year=2015;citation_pmid=26712540;citation_doi=10.1128/AEM.03603-15" />
<meta name="citation_reference" content="citation_journal_title=Mol. Microbiol.;citation_author=J. Paulsson;citation_author=M. El Karoui;citation_author=M. Lindell;citation_author=D. Hughes;citation_title=The processive kinetics of gene conversion in bacteria.;citation_pages=752-760;citation_volume=104;citation_year=2017;citation_doi=10.1111/mmi.13661" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=V. Pecoraro;citation_author=K. Zerulla;citation_author=C. Lange;citation_author=J. Soppa;citation_title=Quantification of ploidy in proteobacteria revealed the existence of monoploid, (mero-)oligoploid and polyploid species.;citation_pages=e16392-e16392;citation_volume=6;citation_year=2011;citation_issue=1;citation_pmid=21305010;citation_doi=10.1371/journal.pone.0016392" />
<meta name="citation_reference" content="citation_journal_title=Nature Cell Biology;citation_journal_abbrev=Nature Cell Biology;citation_author=J. L. Ptacin;citation_author=S. F. Lee;citation_author=E. C. Garner;citation_author=E. Toro;citation_author=M. Eckart;citation_title=A spindle-like apparatus guides bacterial chromosome segregation.;citation_pages=791-798;citation_volume=12;citation_year=2010;citation_issue=8;citation_pmid=20657594;citation_doi=10.1038/ncb2083" />
<meta name="citation_reference" content="citation_journal_title=BMC cell biology [electronic resource];citation_journal_abbrev=BMC Cell Biol;citation_author=D. Schneider;citation_author=E. Fuhrmann;citation_author=I. Scholz;citation_author=W. R. Hess;citation_author=P. L. Graumann;citation_title=Fluorescence staining of live cyanobacterial cells suggest non-stringent chromosome segregation and absence of a connection between cytoplasmic and thylakoid membranes.;citation_pages=39-39;citation_volume=8;citation_year=2007;citation_pmid=17767716;citation_doi=10.1186/1471-2121-8-39" />
<meta name="citation_reference" content="citation_journal_title=Molecular microbiology;citation_journal_abbrev=Mol Microbiol;citation_author=G. Scholefield;citation_author=R. Whiting;citation_author=J. Errington;citation_author=H. Murray;citation_title=Spo0J regulates the oligomeric state of Soj to trigger its switch from an activator to an inhibitor of DNA replication initiation.;citation_pages=1089-1100;citation_volume=79;citation_year=2011;citation_issue=4;citation_pmid=21235642;citation_doi=10.1111/j.1365-2958.2010.07507.x" />
<meta name="citation_reference" content="citation_journal_title=Molecular microbiology;citation_journal_abbrev=Mol Microbiol;citation_author=M. E. Sharpe;citation_author=J. Errington;citation_title=The Bacillus subtilis soj-spo0J locus is required for a centromere-like function involved in prespore chromosome partitioning.;citation_pages=501-509;citation_volume=21;citation_year=1996;citation_issue=3;citation_pmid=8866474;citation_doi=10.1111/j.1365-2958.1996.tb02559.x" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=C. W. Shebelut;citation_author=J. M. Guberman;citation_author=S. van Teeffelen;citation_author=A. A. Yakhnina;citation_author=Z. Gitai;citation_title=Caulobacter chromosome segregation is an ordered multistep process.;citation_volume=107;citation_year=2010;citation_pmid=20660743;citation_doi=10.1073/pnas.1005274107" />
<meta name="citation_reference" content="citation_journal_title=J. Bacteriol.;citation_author=C. W. Shebelut;citation_author=R. B. Jensen;citation_author=Z. Gitai;citation_title=Growth conditions regulate the requirements for Caulobacter chromosome segregation.;citation_volume=191;citation_year=2009;citation_pmid=19028887;citation_doi=10.1128/JB.00862-08" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=D. Slade;citation_author=A. B. Lindner;citation_author=G. Paul;citation_author=M. Radman;citation_title=Recombination and replication in DNA repair of heavily irradiated Deinococcus radiodurans.;citation_pages=1044-1055;citation_volume=136;citation_year=2009;citation_issue=6;citation_pmid=19303848;citation_doi=10.1016/j.cell.2009.01.018" />
<meta name="citation_reference" content="citation_journal_title=Biochem. Soc. Trans.;citation_author=J. Soppa;citation_title=Ploidy and gene conversion in Archaea.;citation_volume=39;citation_year=2011;citation_pmid=21265763;citation_doi=10.1042/BST0390150" />
<meta name="citation_reference" content="citation_journal_title=Biochem. Soc. Trans.;citation_author=J. Soppa;citation_title=Evolutionary advantages of polyploidy in halophilic archaea.;citation_volume=41;citation_year=2013;citation_pmid=23356308;citation_doi=10.1042/BST20120315" />
<meta name="citation_reference" content="citation_journal_title=J. Mol. Microbiol. Biotechnol.;citation_author=J. Soppa;citation_title=Polyploidy in archaea and bacteria: about desiccation resistance, giant cell size, long-term survival, enforcement by a eukaryotic host and additional aspects.;citation_pages=409-419;citation_volume=24;citation_year=2014;citation_doi=10.1159/000368855" />
<meta name="citation_reference" content="citation_journal_title=Current biology :  CB;citation_journal_abbrev=Curr Biol;citation_author=H. J. Soufo;citation_author=P. L. Graumann;citation_title=Actin-like proteins MreB and Mbl from Bacillus subtilis are required for bipolar positioning of replication origins.;citation_pages=1916-1920;citation_volume=13;citation_year=2003;citation_issue=21;citation_pmid=14588250;citation_doi=10.1016/j.cub.2003.10.024" />
<meta name="citation_reference" content="citation_journal_title=Extremophiles;citation_author=S. K. Spaans;citation_author=J. van der Oost;citation_author=S. W. Kengen;citation_title=The chromosome copy number of the hyperthermophilic archaeon Thermococcus kodakarensis KOD1.;citation_pages=741-750;citation_volume=19;citation_year=2015;citation_doi=10.1007/s00792-015-0750-5" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=N. L. Sullivan;citation_author=K. A. Marquis;citation_author=D. Z. Rudner;citation_title=Recruitment of SMC by ParB-parS organizes the origin region and promotes efficient chromosome segregation.;citation_pages=697-707;citation_volume=137;citation_year=2009;citation_issue=4;citation_pmid=19450517;citation_doi=10.1016/j.cell.2009.04.044" />
<meta name="citation_reference" content="citation_journal_title=Subcell. Biochem.;citation_author=K. Sundararajan;citation_author=E. D. Goley;citation_title=Cytoskeletal proteins in Caulobacter crescentus: spatial orchestrators of cell cycle progression, development, and cell shape.;citation_pages=103-137;citation_volume=84;citation_year=2017;citation_doi=10.1007/978-3-319-53047-5_4" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=J. A. Taylor;citation_author=G. Panis;citation_author=P. H. Viollier;citation_author=G. T. Marczynski;citation_title=A novel nucleoid-associated protein coordinates chromosome replication and chromosome partition.;citation_pages=8916-8929;citation_volume=45;citation_year=2017;citation_doi=10.1093/nar/gkx596" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=E. Toro;citation_author=S. H. Hong;citation_author=H. H. McAdams;citation_author=L. Shapiro;citation_title=Caulobacter requires a dedicated mechanism to initiate chromosome segregation.;citation_volume=105;citation_year=2008;citation_pmid=18824683;citation_doi=10.1073/pnas.0807448105" />
<meta name="citation_reference" content="citation_journal_title=Appl. Environ. Microbiol.;citation_author=L. Wang;citation_author=J. Hoffmann;citation_author=H. Watzlawick;citation_author=J. Altenbuchner;citation_title=Markerless gene deletion with cytosine deaminase in Thermus thermophilus strain HB27.;citation_volume=82;citation_year=2016;citation_pmid=26655764;citation_doi=10.1128/AEM.03524-15" />
<meta name="citation_reference" content="citation_journal_title=Curr. Opin. Microbiol.;citation_author=X. Wang;citation_author=D. Z. Rudner;citation_title=Spatial organization of bacterial chromosomes.;citation_pages=66-72;citation_volume=22;citation_year=2014;citation_pmid=25460798;citation_doi=10.1016/j.mib.2014.09.016" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=X. Wang;citation_author=H. B. Brandão;citation_author=T. B. Le;citation_author=M. T. Laub;citation_author=D. Z. Rudner;citation_title=Bacillus subtilis SMC complexes juxtapose chromosome arms as they travel from origin to terminus;citation_pages=524-527;citation_volume=355;citation_year=2017;citation_issue=6324;citation_pmid=28154080;citation_doi=10.1126/science.aai8982" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=Y. Yamaichi;citation_author=M. A. Fogel;citation_author=M. K. Waldor;citation_title=par genes and the pathology of chromosome loss in Vibrio cholerae.;citation_volume=104;citation_year=2007;citation_pmid=17197419;citation_doi=10.1073/pnas.0608341104" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=K. Zahradka;citation_author=D. Slade;citation_author=A. Bailone;citation_author=S. Sommer;citation_author=D. Averbeck;citation_title=Reassembly of shattered chromosomes in Deinococcus radiodurans.;citation_pages=569-573;citation_volume=443;citation_year=2006;citation_issue=7111;citation_pmid=17006450;citation_doi=10.1038/nature05160" />
<meta name="citation_reference" content="citation_journal_title=Microbiology;citation_author=K. Zerulla;citation_author=K. Ludt;citation_author=J. Soppa;citation_title=The ploidy level of Synechocystis sp. PCC 6803 is highly variable and is influenced by growth phase and by chemical and physical external parameters.;citation_pages=730-739;citation_volume=162;citation_year=2016;citation_doi=10.1099/mic.0.000264" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Random Chromosome Partitioning in the Polyploid Bacterium Thermus thermophilus HB27" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4.cover-source.jpg" />
<meta name="twitter:description" content="Little is known about chromosome segregation in polyploid prokaryotes. In this study, whether stringent or variable chromosome segregation occurs in polyploid thermophilic bacterium Thermus thermophilus was analyzed. A stable heterozygous strain (HL01) containing two antibiotic resistance markers at one gene locus was generated. The inheritance of the two alleles in the progeny of the heterozygous strain was then followed. During incubation without selection pressure, the fraction of heterozygous cells decreased and that of homozygous cells increased, while the relative abundance of each allele in the whole population remained constant, suggesting chromosome segregation had experienced random event. Consistently, in comparison with Bacillus subtilis in which the sister chromosomes were segregated equally, the ratios of DNA content in two daughter cells of T. thermophilus had a broader distribution and a larger standard deviation, indicating that the DNA content in the two daughter cells was not always identical. Further, the protein homologs ( i.e. , ParA and MreB) which have been suggested to be involved in bacterial chromosome partitioning did not actively participate in the chromosome segregation in T. thermophilus . Therefore, it seems that protein-based chromosome segregation machineries are less critical for the polyploid T. thermophilus , and chromosome segregation in this bacterium are not stringently controlled but tend to be variable, and random segregation can occur." />
<meta name="og-title" property="og:title" content="Random Chromosome Partitioning in the Polyploid Bacterium Thermus thermophilus HB27" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/9/4/1249" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Little is known about chromosome segregation in polyploid prokaryotes. In this study, whether stringent or variable chromosome segregation occurs in polyploid thermophilic bacterium Thermus thermophilus was analyzed. A stable heterozygous strain (HL01) containing two antibiotic resistance markers at one gene locus was generated. The inheritance of the two alleles in the progeny of the heterozygous strain was then followed. During incubation without selection pressure, the fraction of heterozygous cells decreased and that of homozygous cells increased, while the relative abundance of each allele in the whole population remained constant, suggesting chromosome segregation had experienced random event. Consistently, in comparison with Bacillus subtilis in which the sister chromosomes were segregated equally, the ratios of DNA content in two daughter cells of T. thermophilus had a broader distribution and a larger standard deviation, indicating that the DNA content in the two daughter cells was not always identical. Further, the protein homologs ( i.e. , ParA and MreB) which have been suggested to be involved in bacterial chromosome partitioning did not actively participate in the chromosome segregation in T. thermophilus . Therefore, it seems that protein-based chromosome segregation machineries are less critical for the polyploid T. thermophilus , and chromosome segregation in this bacterium are not stringently controlled but tend to be variable, and random segregation can occur." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4.cover-source.jpg" />
    <title>Random Chromosome Partitioning in the Polyploid Bacterium Thermus thermophilus HB27 | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"iDdFFEELPvIwQMRieI1JUF12jjVVOFf0Fm9Vb6wnw7A","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"492611","apath":"\/ggg\/9\/4\/1249.atom","pisa":"ggg;9\/4\/1249","ac":{"\/ggg\/9\/4\/1249.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/9\/4\/1249.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;9\/4\/1249"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;9\/4\/1249"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/9\/4\/1249":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022o5YMNguCpJxE00uEoXi+D15EjsRGOGmcFsajfVkSX5TNRFH5i3EFNBJ58my9DYJNgbtmA4+B7e8OdEKkWPBLsRhJDipokSxXpX71GVrT4LzwwvneVJBsIz74sejj1Rz\\\\\\\/J1VBOwHyCZk5CXFXaYd4aB7pR4Ll7Vnb2kblG7njXY15YI9Z0VXiokoO380TA9BHr2Y6kjhf7O23OIQz06ShGRWs4rPYtnTC10KHDOM8m4FzKB\\\\\\\/v\\\\\\\/losOy+DvCCa05rxSJAq33nz0yJTqDPOZh4Ti2V8eWL7KkHm6Qb0usTbC9grx3y+E\\\\\\\/th\\\\\\\/twEroBSr5n92TUUNXiRyt5EOzkScYKY7YyA0m39YkBKbx0n1KrEtIR2n7SXwiBjROLjqrMyulPKlF0pwhAAj\\\\\\\/a9NeJeMjBHhprWhWxnuTH40fNBlQi4Kl6KzBIcpYPN\\\\\\\/pT7dndAteoNuOrC3ESFPYDmAcMS4Ty9xTcJ8sklczx1ZPUyUIn57lW0WDDykrbc7ZYPlzDg7Ce1W6eg8C+WTS0eINeTKOREA9Mjn4EEWcmkTYLAUbouv2PYvyU23MNK+ztvCyVYIojULJl9SQYIhCrMEq0IAlhpAueK+qObB1pj5Sr2QnjK1189481hDF9lDR31bKkEAxD0AJMq6T6Ys32i8V0ztGiAotjo636vQFKIjqoEEl6YpD3tFURhIz0mvqJ2DvBSOxWPtG\\\\\\\/0flfU9mUPb5LF293p7yxwAsBQ2riH5JSrIpq583dE2CAjBi5i\\\\\\\/ri3xLnkMaYeLmhqBJg7QZQroDQ\\\\\\\/qsrlTdfreX8nUbHUClsx875LXMxUqXFrPn1iT6Y8T\\\\\\\/bIb0yxB7og1B9tPMPe+412MWHNghhmRkhjcdiJ0cr\\\\\\\/vcX3fSa0iJE5oeJM63XN7AbRZqlD\\\\\\\/eOoKAwCVra\\\\\\\/Pf8T21b9I+XpdK58YW9ZQkS1mF0o0jBzI+JCygrScEqxKkGMfX7Ahh++It6ox1XJaf3Uu9AO5dhZx6dtRVf8FkRw6Ob6UuvziGeFCiy3lb5SIzDadPlGoXPgTjg8DMI3aOMsEFlDLMAdBVvgTWi38Z4jrySM5jtmDfmTW+k4GYaYf2CYz7ReGVMMBsfZrnMrlOBtmuwXn\\\\\\\/gRwL5o5YpZUmeJ31woxY3GmcuUT+MIW2XMWJbSFKym\\\\\\\/F6iKiBT9HCNEkDIbjPpBi+8SufngnlgoDP\\\\\\\/bLQFeqOwRzBZgxbFJoIAFLl2apHZRiWKFLZxRhregCaTR+Ngz\\\\\\\/LFSCF30IqkSWaeXybM7QshD4IULpoOWXlQKFMwvhkl\\\u0022,\\\u0022iv\\\u0022:\\\u0022R\\\\\\\/xIgStIdEUnD6KDpEKrRQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022024c902c74c43598ff3e071c4fb4a82b1af75383ad8bb913577ce588049b7795\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022vz\\\\\\\/LkJvYGNMhVsqqlh3P0\\\\\\\/hbG\\\\\\\/VRH0ze3oSFyVBJzisT\\\\\\\/vVnO8i6YKMr1BObSKRKSFhtCaMElk7c1xdScUqYIkIy9r5Q69VAfMrrISbHqmezAp97x9l9GONhAd7DS5HnzMeri+n254GCYBDRVcsF5G1aR6zsiuZP2vfV5CKJGC0j5Y9wyezHXFu409jikFHfESOnKTh6P9tQSHLAd0bzUzXCh\\\\\\\/iUby5UZ7p9M0U7PWN8GvdjVHh9tKdjKgfInjFRCCOw\\\\\\\/ZdnkX\\\\\\\/CBW5jKvrTcseKcLBznqERPgOmb5KoekTx0eLK4Vxa9d8rmsGBofeNu+aEv0uRvyfehTXzG7BOQqRIQfT+3AjL7UYIU\\\\\\\/CpR+HIJHRSPkDlVUV64GJ0tMQ50\\\\\\\/h6B5bf5vj1Q8sKxSAq6YHIbwygq5G6crYMlJCxrgAuXS81ev53IKHC8EeOzqKBkCiZ7bnECgeMmeIhNzgLvMRafpJsKMorALkm4p47grYt2R7PBmI4x2yxkbDJH1Uo+meomVFrDZTUV\\\\\\\/sqq\\\\\\\/1\\\\\\\/KxuKDfAhzL4iYKasX5NALfFB1Ow0fCrGFz8fP1saH+8FyHnZo6y5j2CfF\\\\\\\/t\\\\\\\/jhjB7fdylidOMVzXTLyoHfd3reFR30gsJkwbypfJf7X4GCz55OTsZf+DPOOwyCdLGtfxHYqg7Mk0suKG776vDgw4oaKgZe5G8wxZ3XQcfzKxZcEcEro\\\\\\\/ZpXeDCOojyrxU94fNhlrtM58fSqk0AOUZwMcEWF4SzkoEFjdxTOfWRMMSvZdHMmwbdbfrZSr+SpmFv9UmNE\\\\\\\/bhPBwO\\\\\\\/\\\\\\\/fq8drbv\\\\\\\/LHjvnjFmqSB3xJeZmGixisDMxVALlpDbVi0gCq9YH9luuWdeiptkHzT4C0eBNqUg3toc1aqnRsbR9ZbZYpyTpDpfmDWevs\\\\\\\/qFN5uHEgZbc4WK409EaHv59AcCJoeZDlwxx66JaZT3FwbopL7RWpSYI+ulqb+sD7rQDCBWPYcV4yCZ59PN9ZvkkVoA+3wlVxAroROGEiH7VIKyRlOBBivIoGcgSYzdrEfkjzTg6dKhViS281Ppn5+l9YFMMDRP8\\\\\\\/Hic+hv0m2DeaKXtBqmjKhg8NoBb5BY3\\\\\\\/VqsfTe2ElxdU4LLnc7i\\\\\\\/8n2tR7LaG5Gi3aBtxOysLPP0VtnRn5zwl3K\\\\\\\/VuFHy4M22+hW\\\\\\\/qpA1B5j93EsmFLG2pI+xvj4oRuXSBopWVcO4W2dGSjLqSQm1smaPGlMf5JoB5xLrZyhf7YdUhDYDNspILZugZeS6U56OEVjyO1mhVGv9u4hVyn6u7pvCHwnD2wv27EzVCUuCvghrvI5w0lqo9CD3KwffOIieKYq6nEV3yFdnNqdys8eB+8HPbKkPKaRMPkjR5t6baEivNGtHbCNjfLAo0yqUICbPnp\\\\\\\/lrCGlGQvcKyWhbtUrKXj38R9KndPiZTgEcILDJ09Y3mdFa2bdw\\\\\\\/C6SxXhXq1FfmM9+iSRMJdHtfLp53Jz0UJVRb2Q6uA9YEF4ug4wb1GMtQUOy2vPQrvP++oA6wfL19eKAiwb3wbG+RxZQuLlOVpB34Ys75sRcOqENE2nx87PNYNjFz7M2dOUGynwigWgO297LGlnUx4hIsdt\\\\\\\/Pd8m+n25h\\\\\\\/KBkF+Wd2z+QuIz7+v7qT\\\\\\\/9NYQYHIPd00DguDdoZumnPtx9gw35RwEWE6YXJiPkJPePSCGwbyMl43uEgl9ruC2YRR6llaQYLsxcP8XgfGAwPFKP7ZfCjCPrGHHt+0S10cjCmMGjWm45m0h4a\\\\\\\/RlV+SbPu9wW\\\\\\\/X5u5O51sTT6r6mCgW1JdQy4liEmPdPM8EPA5GvIU2Nbxzuthw8lstrOHcRhOkN+bUio\\\\\\\/sYMms6PftYr10nhuFU6sbxJUKWrW3Pxlx4\\\\\\\/inTkxihFKSWCGeW9\\\\\\\/5frrv\\\\\\\/6l9lvFkB7YHF6ZwKwFTI1Ts\\\\\\\/46oQjGLPcA+mnY0ZG8xvqRr8rarQW9r0wxP9QQzaH6zxcuSJ1IvcD\\\\\\\/2yL3ds7ect981DCyA7ixlgso6KauulzOl1NuNcBTJCZiDhss+arWYHRWTkIjdmwPUu7xgYzCLxPIqzN5l7WxohLYW0E\\\\\\\/L0gfTXMUW7uc4uTkamSSh0gx1+RCVDvhAD2s3FeskH99aGUQnCswoMCz6QnG6i294saPtOVof9wP3v0ivliBv9VTmFyLdKDuNfpryNz+9NqKU3R1kltOpU9i1+Gbh7Lcpy6RPg8TePYqwoinLv68UI3PzLVUN+IJ6cNWUTIlMQaDlIXvZqK5jS1ijffOa83wQ+d+1ZO9UwR85ilprRB5Bh9OQZVmZbyjgEoKhm0XxPTQ8XP+Ebz0e1L+MMyFubA9YsH\\\\\\\/Jczy\\\\\\\/DhT\\\\\\\/eBqloZ2WdDpzxpYu4DSRgzAbtKX5btXk+coO\\\\\\\/zanlFzf03GpqbX6nCeRPNmgUUKSgWKmbsmQp2C1pRuLBZ\\\\\\\/FSGbWIQdCFWckGDlW5TgDCpIpy0ik8ABJZt0oHbPqdxiIU3P1D50VCKduTMWICrLCfqGhZLwEI3AKmVJNue\\\\\\\/ma70C7Zn+w1gRWbtzfTsF6Vlm4D\\\\\\\/a5PqSCjTDQS\\\\\\\/GpRwsTlQMmOBIk7lfKJY4QL\\\\\\\/RK5csJqVVn4eaTfZmx9DP3bxnkqdvFFevT1\\\\\\\/YKLiJQXkcBUjdUlAFrItlHyVtvmZcwNkDa6s\\\\\\\/FLs+oZieKK1fiZ9MuX\\\\\\\/SSD00QiD8oNjSpJCteHuWE2u1ePrUXRETT\\\\\\\/oD5RCvB0czAhEhR+DvO+wVMAdQLmbU67MjaNCUt\\\\\\\/So3p7luK1VEKpStPj9D2j1DFwoZcUi+z3m2+aO39iGDi1HSXekiSP7+GPwzvTujY+PeENu0xbkVU3DaUol1U4LVa+e8ji4qNCvXXFesyZVQ4sg0g0CTg8HpyEkHkJF2Ee93KeWGuimtZ\\\\\\\/Rj4ecyxIR\\\\\\\/0am3pQOOKGDw7RrZ3l6s17x9O7ElZByOFGAsi3wXjLNiW361QvTcrZOForQrZ1q342iW7dtY7OzE3\\\\\\\/4+UbE4gNsbCZ6JhSznd63H7B18TyqSO1ZlkwgAmOEf52d+VabTAYtlpaLLirUq6z2du\\\\\\\/MrJFAgPMDZYrFbKS9+3nXEHCGHjTqxQhgVCciA4zc7bZuQFvYzf0gI\\\\\\\/86SubYF8EwEE6tY5oqup1IItEoB1InxcaZd4W8KjFspEM0QyPLlhsI1hbNyOCRz3SCN3zI0EMMU+\\\\\\\/0Vxz6K2xpbFR8NQFbsAYgbKUaIhYZ48yESrrJFoCnQxD7xB2qKZYXp00YGL7XvOQKDAa6iqQZ5SDtDd0hfOkuQpA2uhjJCcF4E2e+rLD2vI0i\\\\\\\/mBGvmPixXVWHvhRz+FLDX4C0NG+fBEVeW6SsnuJiElrVZj0tg3kJDG5+gIOZD\\\\\\\/q\\\\\\\/\\\\\\\/n85gEA4GHnF8PlLD720U02fTMGfmLjxC10LNNDCBLN5KbWfyhtDuhIVOfwKzjgCUkqkCpkaoLjxlyqpTJx25zohaGEQccrYubRFr97q5tcz22O6EywtWIla4J1gKx4RHE1AarVHvhgPLZMWYvmP+rFFzfGuxKwj\\\\\\\/iuHsAfN+GOfwqxz7u4wUIttGtm07E1+JWG6IhUMEZpylwMYA3hq2LEnDKCkIoGLcGRBh+06gDfoZ4ixIH\\\\\\\/2a2vbuiKCTHf6khZN9ZIM\\\\\\\/Zj1zaIZ4nssjDqunvUaRMZT7zhWzdCaxXZjpoxakTpsseVCvba2NcgKpj\\\\\\\/z6CFjjtD0K2Mi\\\\\\\/XbdDezPdy3heQul2xDC+y2rpmYox3i\\\\\\\/q56r202hyNsToP+0heC\\\\\\\/XqIra5bYx4iOL\\\\\\\/Mj6FQ4Rgbo36WfNQQBzgZF5QZGgF17+zeXL\\\\\\\/bTlkopbJ\\\\\\\/tOMXeoX8zFklSu6OH3MqtcRjfmC8Nqat3pRk+mUwSSs7uL\\\\\\\/xaYyLitgdBdhnFhC4WPM04BzHrInZwe7Mp0O\\\\\\\/zC4284OxV+uYq51FfBr94I2Bup39AzDho+cZHXCpga68sNaVyS4epJYEHEsgJ4m92L9oOw5o5Hr6lx6p4yfSn2hbfRlTPgU0mDLhYtcxBjvgnvmmzM9hA5+uD1lQBBpEw66p+xMRQLf4n4LWvsh8KEWsZPha9VtrHMzoKfDAmLl+HxNnvtvXKQ5N0RuZgd0QwLgZ8VbwXcWw9kHchDI6fvQbpeQ3ABcD0BncazzACd8naWz44PnPzTYsa013izI6OQsvB6w2j1dEaBjCtYPE2tXEBk7odpFbl7WisF81RNsNSYRehO9SX7V8rmbi4efxS45Q5tI2ydOa6qHzq1cvGbcY7Fud4uuEpM1DPHzYSUK1\\\\\\\/9VQiTqD+f4phIXflUAagXoczUBV1pkmYODyNJ08FcxUMn86wRbZNwbS42dtkSp01I9frkQ32YnSIdooZsmCq10eDGhIG\\\\\\\/QORRKO41MqS+m80fdv+zG4h6r+rGyboWpdXowXoBaVfLjrGqDYEOlfXxkIwyfYLRnIVISB0s1BcxxkYPUcWzcLssvDiUziJhs7Fu7aa6\\\\\\\/ZofYA7\\\\\\\/g3oi19YoqYoi6OBYLBU\\\\\\\/uDj1FRtHuzXRI9\\\\\\\/k919jHzk\\\\\\\/vqkiEP8ijuDeQ8xMxe7eepmQjp9ctc5M1fq5G0vVCOmSn6lYQiY4oxEkBTXtLzKuTTNdId0mtAYVnkPuPBBHXXU3+1emB1bgn\\\\\\\/bEt1n91Ep6P4K1ZG68kY574UXfMvy8Z4Rb043GkSSUuwV46QSaDcCfeKxz1KU7rTbLKVtfRV9ZY9jE0sgf3djll\\\\\\\/fhgGn1dDvQNvj3lRM9SokhpQ7jbOg3NHlc3PUjh3yZbprg0qA90HZNNkSCu1g1BizLKKpQAwPKOWi8QmpS5MM5Vn1FPvqEuHE3ugECD+OHHI8TDXFV9FCE\\\\\\\/gDJSAq8Kj74ienCuYzRryGZdGqFT\\\\\\\/njgsfVaxy6eC00JZWBtVEmDvfNUuvYCPF363gCeIXRej02KlW788TvxchpKqLv1kZTF921uU0lyKaUWbqTM4vHZPEAskNAu58+nzFPVcoMUIfDGnFHsqQrDd86wAoM8ZHQJHd13A877tEGhQSUBwH+Jv5TKEG3J9JJ9QtlNaHies8UiqUnPQ24vJWWJQFTGWxo\\\\\\\/xy726U+Ylx8rpFwMl4x8szTQQ0\\\\\\\/Kzlw6G\\\\\\\/8wHSXtSGobEXsvrjMOET32UWUOv0syMQdV6YGVk5irN+Z0OXFRrxz7QA3e\\\\\\\/kpr0YTZi727S9CazfN7mvSJEyzAv3cPvExc+B9HT1+nT0tNY55P7g\\\\\\\/3yUW+mYfnUwQ0Ta7uCoDbgFMnyDXupU1RgJ7\\\\\\\/UNGKdf3x\\\\\\\/lCDbTCsJ8yuATgA0BcVODWt8IdwzOGoJ2lZMPph0IVqlwriK\\\\\\\/o5BdkyNAAgRobYmKNLXdAFqQluVtBeDx4xoDySGn0rnYxx0yzXOVAjgXo7+KivunwpgzpvdHLgTsxubqcXL\\\\\\\/geX0RiYUVlbUKQDQf+liM\\\\\\\/aBm\\\\\\\/XGeptSF3q6hiYog3bxRmzM0pv\\\\\\\/k1ZIfgOSheOEbY7o2vx93pKE5h5t5fYYodvIhIbhyTRsM8kx5NA5+msJHXrSwP4N1qUDIpulLTLUhaNeo6u2+4Ah+NSBZQXodtysMAXoqdtzOM3OWgVWKU+B4XJoAiNHHkT4SGQYvODoldy1GPeGHyPDeUQ7aaE772dFBsinJJMGjvjpGAC6tO\\\\\\\/daxnKavp8MSxQFvBqtedWwkRqmXeBTmztWw7dKPoRLqLjR12\\\\\\\/KtT76glwa2Q57hNmwO1eL8GYNhOcC4RTAHoJzuzCLuO8SLzdFKwND\\\\\\\/H6Ln74PdGAcgwpx5xW9nkhPPz28K41+r\\\\\\\/fppyYM\\\\\\\/BtX2gt+g9ZFtEUwA4xDyKEqneW2QUIfXMU\\\\\\\/bXfyG7iCRMbBq9CtI1qSO9uq1dip3hwGmjZYHG0IKzBjOt73m+jtTpOgySoi3XM6IkhXFnyi2aJpamESjNZc0JJkFpogkwptXdw\\\\\\\/a\\\\\\\/xA3QgfPji1LypITTUtgznf+eMj3erusw587C7D6ce3l24loOsRqpIUlYkoAzfxdig3rG9t7Txql29mFf5OPupZiBnA27ShJL6m9ZFbwsmtcV8oJ7PNB0VLdNr\\\\\\\/yzgiCLggSBvPfGfMx3d94DK\\\\\\\/NIJC\\\\\\\/E5JJm3b3MxrBif04CoFLOYgM32wKSlR0yiE3CplmkmsNtgx0LNR3zA31gxOkTOYYjjjEVDH99ASOf0Leudx232xBtsUSuJzjce7Cb0R8muYRBXsGGgrBERbVKN13eKpA1crZ0kzQCgtwoEiOe\\\\\\\/\\\\\\\/7tJlCEep46wOcehn2l4eSxrgtm+QM0vb8JXoVVAvub\\\\\\\/sI2\\\\\\\/RDz4VQHh5RHEbXWz9f0VsNOCndvf6epGkpDKwGfLajjSQXmQHFDd4b7Wn8xBghxm7C1hS2KmDsqrsGmRnStxg2n3RoDcIZHG6xHV8m1lrUcgDwR9+roO8xRGNnz3AFFE5Us\\\\\\\/9Ls5sAvSUC57pxXP\\\\\\\/y7jHI4GB13tgiFRhhrUXpCYct5saGPsvIPaHH029BSgXjyqFcfI7ZuZoKVFz\\\\\\\/esYUW+EkmaRu2gcBgTb91IRf5S3I2Mj\\\\\\\/0BmnVHqFK715FN+S2l3f6RI3o5X\\\\\\\/mX7gW2wHO5GTCdDxVDUWmcXSO5sdh5iGt60wNdr6faTsuzPZaPN+Ls2vmV5aKdD6ic8LYwChfcKmc\\\\\\\/eb47gezjNHKalwNVHj8Nm7yS3dPqrtwsqK7N2GMgNZ0q2zncVoH3V7Di8nCP+rDR6XviJg0Ln9EpPHCYIsWjV7Euj3w3FcCX2msbEo1D6gtZDiLhfRVXevS4DNQIYLhgJaN3ElSYLpYzZMK9WrZEBGdkdUbJTf0hzRGL260c2rvQ4hJUvW3aA1KaNeSZxn67loaf5TSy3y2Pj6y7ZWIB4dJt9BKkO\\\\\\\/mao63JlhVhdyB5jJu1ciYQ9+4zpAoavD5FiKOaImoirMR8vY7DqtVgi2R7zJT5KHiFkS7SZ7cwcfxDaaKbIMa\\\\\\\/ZUZsLlD9tsvgfBB6DCAheJVjx71fmiUc4YtC+pjAYrOt973Z8gm9OLMcrg0swEwn+mvofkSLh+rfxFr082FoOBkTwHPOgrPKGrwXAPZGdNpy6FdeZri+nA0SpsoRm+8YzAmC5RJSDtEU1mjHoYwChA87h\\\\\\\/S80qgcMOzBxmN0sopO4Ko9HWuGUrk+U\\\\\\\/tsUlBj9CQ5QxcZv2YdGaGN2N3uFVEBohmY8PhCxRk3TO7j9+71zIF1QXuA1lnmdkXhCnJDZ7oCsSANuMzM4FZ4FqQGunJQBqrgNXOOYi6ich446POHC8EY+eOlruOJN+mEmIyMSU1LMRCnijV+464VWDOGJppkbHNS0P5uGfWtRobPFUXOS3sfYBAB2ruq3C8fVi0Hb3ANGNoK0XkUmn2j0F+PDIQxA8XBlX0T\\\\\\\/ERxjVxj3s2T8mTfvx2FiBc8To5q5\\\\\\\/Tm5FiUl\\\\\\\/MBd0EyyGhqKiDEYYUaXX9br39wfQ\\\\\\\/utyFwbY9adar8yw2F8Kjng+yQIChM7IIfyf3Nx0sCl1DJPelVh+cxAHMsI79mme3B\\\\\\\/7KhaMPUh2xG5+FxMzk4Jz8CQ\\\\\\\/ftnyHxX8DB+CobWq2THkl6MxKF1sglpZYTPZFy72CFRRb5U3EiBO+qSNH2fD6OH7RReuV8Kx7uu4KlDxZ9+5JBriT1yHQ0UDNG6l2gmL525rgDWKdUhwVAuFhoE2RkpoAhhoNgVf9RXvpIVijZoMhn9QOVOK4gwYhkD1nOF5gwxcvDV+sAesN0Elg49WYpuMHlnILqqil8G6ILEY7xuWT+9jbSdK7f7NutD7J8VyoOhZm54kw37a\\\\\\\/0Zt9F5VHsr\\\\\\\/BkmczU68NOe2IfIU4t054q1FHdixo7imjndAab613iWNn9dG5wTwXvqQyPf2H7D6U26ne6VmllZNoEsOTAsKY8NW8WSNLJXjdtkrF8rcxCJ+PxQe8aY0\\\\\\\/ISsdhrLw787rsAALf+RAv0khuswvi8SivZr3J\\\\\\\/I\\\\\\\/RPrKYvdeVRnoW2ebkgzhAZWzOglFSGqvZ9fxYPpXCSCDY89MMLwWkPMWPDsbbtEhkrrmRKdW\\\\\\\/y9D+kuKtTUaJ2fVOOA1YV2jk0jdgMMxrLy4yHpdy5o9WlKSrn6dKaKHM+X457DCLhg42clqTHSbkthLZ8sPHXlL3EhgLfIymg7zHk3HpjMLnCgcvhPQsCLbiT2ekq46M5uq3+pZUAl0yBJsqxXfiE9C+t4Rac\\\\\\\/7hqWAnds6\\\\\\\/thoqM6chVpT9bGHQC30sHaOi678RdTXIkXJP++wcbDjY1ylN0kHYXXVqYbLRfMK3RCUGj3E1L3TgLTxT7PiXCpujxJy3TQvwj3Y2bCtpg+W4lCaqx\\\\\\\/88KDL3W5teQlVW8qmNPgIpYpft8ohSOtTQ7llSMQkwHxUXEbkl03tnE7bv0x19JRgaT6NoC5Tc4R8ekKMrdf59y7xiBSIoQQn\\\\\\\/dY3XPRkVKvXGOY2G3Yspw3TxWWGQrclGQIRQCE7y4alsTk2L0uLxHf4EccVanPCXbjTowvtbIDdZ4VnObfOAwCC8eXFYnfEDNJUUeQNdKGp6nE+tP31WaFopNeA2SU5mzsh7fh5G9\\\\\\\/87AxB2rBAdVMc74yXWmQUui978drmLVojt8DsdpgBci1ex95PJBuLx3cMSpmsCmJCKi29AfQR06GI\\\\\\\/kl4CsEmaIl1FnFDYJt+11Iu\\\\\\\/VJp7ksnSH+nPsavYuaoYh8LvswH3iTNajoiAbfH0BGkJg2QLb3pNhRGh2nn4RqzUqqEB95xoBaXF\\\\\\\/J5nGieaIKiWy2UbJ4eIotVwr96iWOlfdFL3QQkiNYI4IC2ioBQq50wo6xgp8EYsDJQhiMljlzuoMUOYRuWac3mokkcqejnM+3FuhqIdLJ\\\\\\\/uM42Z1WgsUdKIe83dbHaRObDFe632sUEozrPrKqKJqbyuTExQZDEMQ0gWdwKq2EWy+W56j3UmE8qgJ8syCcegYRNuVYiKl5AVhXuCESYnVtLy4rrPGwJKaDnoS0MaEPpWmwz6jw09YBaOW6yo2jQ5w46HtTtAcyZRsJ8rd2a+gPZoqFrI8c2tAP+HvdufRRrInGEZQ7vsV4LeTVYzqbhqv5j+V6l46ZL2jmpZSuwKINtvBqVU412Bqzzzr9og0FwUgrR9jwlePEZdHQ07MrFj5iUc2bhQw6hBoCmDT7J5orzG\\\\\\\/lNsZkTXmTxczW4wRAVLZR8jb1lbvcUNaP\\\\\\\/qu8P5bdH4YHcHNUKa9uj7dEbBxnL8+TqZ4uMWAqcYe6P3t8ADxRp0WeeeOZwkRThykOZ8\\\\\\\/BtqIIvxY7u2Fyau+mPsJIfCLLsEVF1VJ74Cjq5DP4XOIySBEwRYZPV8hWnpiJmEcEfCIGynW5jsXn22t3by0vdyaUXrhdjs\\\\\\\/9Oa9szsBSIEtyiuslb1Yu8tJC3smSWjMuFTM+ao+bFuj+z8PFQixxxTk6pN8CZ4G64khtPpr+ZMTli50+G+uBogDeiZO5ozaO99wxxBdDdTGzJxGrX4lfcRUpInuVgyD2poQSCN0dcIpLZ5WI0l4V1vgY8pt7QJ4mc8KflO+cc+Py0aaSHg1NsDyQzrZBzy5Q2UWHDNHEXyykwnFUe6gtMbynFPpIvvnO5g5+WgB4EP\\\\\\\/\\\\\\\/Pc5CPMs9buBVuvGO8Ibg00nj7LnW48EUj+s2AtPNhtHR9wAQML5WAOmFbKrJ5ZFLLDNq9OAYeCsAuZnM6Y57vyCl+63yawgbNu509jgzzbtBj\\\\\\\/PXpj0buKeUcEEk0mw+10zAFo8i1a2pNyqIHWZjmh26bzyrQMj0eeOgDnf\\\\\\\/W2dlJ0eMfIBCTJk244RSGKzoa+Ck20hvl\\\\\\\/ib0EW2ErFVBYxte96C\\\\\\\/1vAFhImQmph3IMbiTIn76F2jlO8cCXBULe84DxuoU1eCoNX\\\\\\\/LXTNpQxby4aQNCPef5ncxoSJdLu\\\\\\\/QcRGrOTw52DQNLo1VvSftAo8vgJaDHqVrUmO3OPeI1bYMnLLC+czopm\\\\\\\/mtGcVUwuadf5H54SIarEv\\\\\\\/yQI0pQgFmuvXoZe2t+mVMDL7mqxul329EfYCXLorl+vfBrhkob0oHT8fqh5YVP6CM+xvLWx9ZTwTWcCcyIwQwkw\\\\\\\/sf71pQC5XMy2wlewyx1BQigdHjyeIvPACyDRp1KqTBg1TDCjqkJWIgfOh3TEdKXM7LgYdMGoSE2uT1Kqa2oqxpPyzd38Pi1c046Qt5b6r93p5TEYawwo45pdgYuDOaUSJJi1dZGUTKNpc416tXwvD0hmJq5UOqCDFfN3V4yiLjTOUwNua5RA3J0sbJiCYGNThOGvOvwy88ys5d+ZmyrHLJzRtyOqcAsi1SVV2qiW6qtUIWcrIW7AR2yDdqaJSZI2BTGe+u49uLqU3f9fZWAj\\\\\\\/STbOOFQTYOByv65J\\\\\\\/ZJ4cpRV7mDgi2nH1kFtChv4LyaYOriXJHRQK4oHGd\\\\\\\/3vbgs5NdzXifj4vTZVY+Eqc8dbx5HqeE5usphG5uGTgspPR2flwYAJbI\\\\\\\/tYqma5N\\\\\\\/ez8q1nXcfQYtb6kkEniB0eV0Lh6usZ7dc5GcQMcabSGuMuHkWzXAw0xAcK4PAeLT7Gw6CbWJRfTFCfEBfnbHOESe3+vVohCEC8VLy3L5QDd3FdflmQVFhQkg4Ddsri0tM0ODV613ahMTwT+26DtEjCGzEVnmppuZaSmVetyv93RgZKHTYfBdcJS3PAdQQF71J0B6jv2GBSKCuH8rzWVappFDzauQuS84326U3YeNtdn5kRqnX2j\\\\\\\/pQpn4reDqyZpfdlfd7damonZ8pHQ\\\\\\\/bjG8l60UxGJk4GvGzJ93IgaRFmG8W5B4K5t4rp8DxG1bkgmIzPM4AZ3Y8TkLYLXH\\\\\\\/FtJ2q2NtIn1DrkK9JKEYcwLwYCMKxgSIzCOZOHlatTNXN1GfTki\\\\\\\/bfMMPnbv0PaV7WWQZc1+wb4ou3oHGVNEqgRIsZdbyluYob8s\\\\\\\/sAWlPC6N4cxUFVOdmZYMRJGrr5NcFAtX2ti0GFKHKRfhpuGCEGh+OWhacAdKNRxb03vMFQvOnr2NFyT\\\\\\\/kBlrd6VC1izt2jHztyfyysMFq8XR0LTaNtalCWLbPVzEv7X9umOB6xm484HswXKlC40027+uTZ0ZAGHQYGf90xff9g1Hk6kb9sKAhY+o04W5yyStxA\\\\\\\/d8Y\\\\\\\/Bbcx5SbYyD726uu3t4M4fDv2UkpRAfn\\\\\\\/fJXXgdOhUxEPkZ6y6N7DUEnztUO3AT3\\\\\\\/9BDEmf94\\\\\\\/5Umms+6UqfirFYAsT3JLUgzNM53v0InOzlNI8zdUI\\\\\\\/kAWhVCSd3y3jr6oFDJoNdwQgyjqJipiIPmt5vXFaj6gPGsuq+1xd6NRqrzQSsbM69HM+usFt2qiKERtC3\\\\\\\/VM5cAJtwHSWTJo9wuBRisMR5SJJ\\\\\\\/pUT0fuF5R16oMKIBUZadNIXtHsB7OrSXK4UPDR3XdORjRNeNzkZyAvyxEnvx7Vrvo6yC4ZnEeawcMFxt\\\\\\\/P6dyuzqXRvbaoH3XDqjSX54F\\\\\\\/VLu1SIGCJWUs4FHq8lUpnnLRGFYod7StfUg8eRlilLdN1Wi9NX7y1DxcTL2NbQ\\\\\\\/rObOOor1ozDd9YvtDWWvKcrjANTsR9jqhIY6L2aiJCpWT+Y4dI1BnuX3FPjOr\\\\\\\/cohbAS4LEAeNQaYqnC+IvX8YSAf8SidJEbZHagOt3pU+oJIN8M\\\\\\\/estLyWFmmCE5\\\\\\\/30N6NlDz+Og2XBvfKwij63Pk2\\\\\\\/p1s6B834WGfWBr8LZqR4yizU1\\\\\\\/ECL0KgouBoYov4M\\\\\\\/9FHOrUyEUzusLZ++yexyxtxFBfwLreac7EIiZY+rdH28keRt8paixoD5EizzYm+JMQqOJXiJVXtz0uVHcR00ZMj8cn+Xk8DwJ9e0jiq7nkj3hiaKYMzmd2x5TY9G\\\\\\\/qYuvtg9xUKIqUj2c1SMUTwz+DOxvcnDk6rPdCcfyFp3AWFRg01Asm9mO4tvqObNCET+OCC0B4T5UEHlbtQZWuco3Ej4pGOEnLl7dVAb0TkxyB5yCQ+X8h6hJLaam53FTC7PZL7ePzDAegjiqshR1tuM6+Y6Pdr80KdsrHfPe7P5obzcHydj48m2OTuhnrPhiIzIHkiwJbF8xUrDPtgaxL1pgFz5pkj2GABu39n524jme6V5ciGVc9PzfUyo9gs4vEkC\\\\\\\/tSiSr5rLoS13HLtF5\\\\\\\/rn4A4gwHaAG\\\\\\\/C8mR0+Xh\\\\\\\/4vvYgyNqGRdrs3Bfh\\\\\\\/wf913WmrT3s97MhEJd32cpk\\\\\\\/2fl5RYRqEvgLxnw9VzAG0+npdX\\\\\\\/2YPDVdJ2tn09E4YOBhHWl9rGwNZIbkQDzyZskQyvmo\\\\\\\/wWEg8xRSc8R6ts2khgKDFnP1fbJAbBdinq8ffEpOU9HT2MuzBVU0aZhqUdT5XCNUj+MByE1LHJuNYTEqKJxJJVWvn7B1tJ9D6Wk2tEnkq5Z+ZWUtCmLznrhHEAw+WSwa0GzXm230uY8yrPEEs2saSEHECraSVB8VCHz+g\\\\\\\/UaJxEFkBi0FJnwJ+D812qXGMUr9xu+I3j+hsSiLNsNGSSNu96RL6Uw6QyerAEn93XvrDZSQkUrG14JcggrRVwlLMJomQ+OOF6NFlG9DRm9p3jU8FuqB0jyVuLJTWT02uBCdrb1KAl5tv0esP1uUalCNEOkShU0HNyjfS2fChirRDPEF\\\\\\\/3vg3+2graO4u\\\\\\\/YtSfyox9PBKZo+WHRslQl+Yw1lDZfRf2oFTFvdLTy9lToVjwue01CvTySNVmN+LcB0at6wzw3D1tSW5X86wr5Vb43K3VwauL4QDGgZ5rWI+cAKPu+Ak1waEbIRJgNUK0gLWSU2K64vniULkRM\\\\\\\/DYj0bHxGA+bmg6GimQU20cN6iQSN9D5aGHwFzKs8rjRLNS1I28feg\\\\\\\/BxHm90m\\\\\\\/OzUyI9dB1dXdZyMq0XxzjHHumR2dKfMGLcpR2PosC\\\\\\\/N4RyMYFwdNEkXmFjulsIcrzDmk1PLOUYqlmhze+f2Hm9ZO3CK1EseUeCalPF1jX6dtVlTzfJ0L2DXmB28KcPvz1Hs40MLNytjF0sl5pGP2LwdoHIV38p\\\\\\\/U7Cr3pzKEQ2VRRB62MIYGZ++J67RbMTigApdn8v+32Noyfqb\\\\\\\/5ho8+jKittfeIlWPthk8QoHf08ke3vmwko+JtL9G0LBFdObunLJrLWQd1towIu2Wv2\\\\\\\/pGAw95T289T+KY7+Urhf3cV0zWSss0r0wXonqMOs8SxUmhwYK1T5nZvhRn31JcqGBkLHo3k\\\\\\\/MpRV28cq6O0K5sExcmfguW8M1M8tZ56Iq36RQHeLAAYk5bk058YIxJ0YHmrPTBCwNXZQL0rlK+y5n4BxdfAPiPAh9L1XEYmusZb1rGaR7IPIFl7Mi0LhYgyz8nL6H1W6KFFgsX5heWRPdrlekxvb8HR83GQNBEmMskqIYZkt5R1y3IWZgml9uCAW34RpW1c0jzoTOE+1RxRVbnRX8th7uQLE\\\\\\\/XPGEYxiylpXRyq\\\\\\\/xyIB+NcdeL6y2LbtXpZGOuViG6BIPuaE9Bzefa4n0x331pwEhZikkEc4QAdObI5q16QA3nK\\\\\\\/KM90xY0Q2hPohHqTPzDkWQhBL2YVDkh1fwyx7ztKJoXrRh6EhF9wa8wfWDXlZjJnSkWkS9YMcO4j8m4GE8wFLw1Egyh7MN8BLWvziKKvReXl75qri4FQyp1xBG0Lq8Ykk7dxFQTYmUJr5IL56EvONV\\\\\\\/PQV7DWOk3xEqAlj+StnVtCXP73Kb3HqdDisSzVoWJji756LXsiVqjedeqSLkQZCp3GRECc0Brljv4mVW1k\\\\\\\/hqBzqMc\\\\\\\/zfQe9GhNANO9JMaeWk9gGkelOygbtYTQjpmTSQSr7EhsqAyrLi1e\\\\\\\/9qX76wX4BsE5Ob3Ybi4s5RN6y39JE666rdGis8wXVrpgzcV\\\\\\\/EfkBG0f+zIWMkI5WTskkGLpbVNF19Z5tOoUIXp2gs201Ws5zYFcjBEPnR+x1Vg2KjgH0zSgFtSyxkG9hrWw2b+DmxsOAYAei8e6k+cAT8Al9rt\\\\\\\/VASv21rAcb0v0fq\\\\\\\/+W1t\\\\\\\/HHoeANU2uD+5GZHISLnW6kAKMVEPvJv3lqwK\\\\\\\/WGQJEtHnlqXhlsDDsFT1KwR82jOrX1Q8uuhAD\\\\\\\/tlQukOezmaaLenjt+xmIfkmjY312bv98gklPvaZI94Q1e\\\\\\\/ussxT\\\\\\\/K+eROxKanhkId6VhMxV8eIeiSaW0LzwRYmHGMYQxbhXgJ0a27xbGTLobCkyYiDR6zQvy2t7zfy8th6J\\\\\\\/LgaTTenYzXTiTDHff+uxgy3JG5ZO3Jgrc\\\\\\\/5HxvdVbpkH+yBdhVa+aL1z9yW0nRJrrejzi1vOAwXb7s5ETY1oRPhOUd7YUUCnRFGqtwdaHe\\\\\\\/D9l3Nl2j45eZrwgq1Be\\\\\\\/Ap2c018KhoCUG2WwunSND8KokbL3B7OfoPIcsOnU0FFPH5dMCpQWYkPX4iwzsAycNWYothvCDYfsxaey6j1D57AgPJQbmvr4I5e\\\\\\\/KRi9XPU4KB52+LS2I4zRLedBcBi3douqZa5lRONJoIhQ3xoxmj5xcTZ+lcD4S8viWbSl0rgpQnfJHJWO124n8u46kJk8EPAdebBhvRBDq+6ecP5rpKRnEGeLSazt4wv8ki++eY4MZpcQh35rOWwDvQxJnOTsuDpK\\\\\\\/pxYjI0H5he2hP5XJ81dw57dY+XyeukPKMR9yYFK6C\\\\\\\/pDz0UPSvlx4TQSsNmrQR4MO605D1FQIDyi5qwOUcWNwXCKJ\\\\\\\/s1u2HG5AHUMcttlaXMP9H6eFNyL8uF0nllaHjkaOGmKY4HIQRrhVAoazwzSQP7RaPPPBT54VH5gF+FZMQCeXrl8oS5w8SrdN\\\\\\\/2OKAdbKZI0K3A+lyI3XCs3aB+bztzfR0lM1CEWuag4wpowvwkXnRWTxYu8PRYr4qwDxcY1qtaE6wFkkcnreTY7V3sopVUep12BrZnRoYO9yg7EYXPW3\\\\\\\/WbfLl6BsBZgfqIxYlu5nFXsE1DZduoOJ6NnHyknrLmeIdnKmG9wd8Nq7SH+0MALU6zYzEklca10e4N6J0u5cL3\\\\\\\/c8ZxjqTP78Ll1ZxKhLpjgMpI2kAvoPq6fpGmVb2raHG+zZ3LGCzgZHCx94rzc5DZj3CGTJhDXlPFprhuDjl5qaC\\\\\\\/lesY\\\\\\\/dy+6\\\\\\\/TojmcsLqjka9cjIkzwFwF8ZijeAV7bnV7A8yCQ+5QEf0xvkq5WI0h3z\\\\\\\/DEv3pfAGaORf9hGxTUblNMxIYw5MhkTIqEiC1\\\\\\\/c8jTU2CLnQjm4RTMG1T+p3\\\\\\\/RtWUn9KnmEvuETL9mVgx+T8VNogz6v2Yj5lKUi8V2iFBT1UbBnDch7OYln07Zlix70Z0CGgasLI53UaDr7gHQMt+rPJ90rVh3gwuhyC\\\\\\\/XxssDEYSFRPvlnhN5MqQbmXuerD8HMR\\\\\\\/Rfebn3TworCLTxR0xMmOSmXLuMt+a1liJLNE1JSOvcGASgZfGwkQYMakktN+Sg\\\\\\\/qyLJzZLB9MmM1N9b1Ek1freoA046QJdCF++OWnC8JD2QpzTxIP50E6y\\\\\\\/\\\\\\\/UmUE6sODJy7QWONofusMT\\\\\\\/9HZ31qIczQK53jIoyqRPVM3piFif6wKrAMAMCskj8GxbLKr1sME712Hja6kLx51nsyR9zdWRXSrHxxVWTNASdlNbJawvpm9uBmTQ5IbWVStulXyQVS2v53Wg5IfYU5jiVrvyXDiM0RsFCqtyzJQJLN0GYTzeuZdjNH22h4kT9OWffwOkaSjU\\\\\\\/4spWw58Q5gJfVyVHMMUJES+gXnzHSsjcuv1kRsHKUaVnyKyM7\\\\\\\/fC0gk49Eljfnv4nFvpo9e2OgtJlPJgS6C2XGhCM02MGOdE0RBEUJLHuHJgpqGujwrv4Yku8FNRbadWnep2P3SXLr\\\\\\\/gCEtE7VvnVsQWqtf85YyouFqKHMSt2TVvBpSCY5jgJrA6ligvxEs7XSRcvKgnbwfx3us5o6xeP62HAO2oXARUk3c645K29z5IjZA0wa3Ptlla5\\\\\\\/NMadn2HQqREhrJaS54CQ30+aNFjSEXCaK4dvZ5nMOrLaUW0MS8AwG7MvLpoUpGw7EGQ4PS\\\\\\\/iH33Nh5M1h9nRWc\\\\\\\/CCfrLPwMv9L7q6qilJ7I4CXXZaXrapVzFuyuPs2qXs+DeD4Nl7sglXwd7Rja3Frr4CB1dC0Bl+Rw8yikrO96sYWnaY0+vD32om0mFqj6zuVA7tzO4OWBoQl7aQynBtGBT3A29MHYc2UBnqI8wOgaYBMcA+gny9kAs+i6dxXgT0mZb1Z4DLhb+uOE03lCmxjneG3gq6a16v1jkOxfZCjLaxO7FD9692qxNWy3+NwK5C4+klcj1SBqMWgH6PZ+TapJJN8VzzldwMNnS7zkV8oNDawFd7Di8p06iRSCW0H\\\\\\\/obt4fANtOdXNG\\\\\\\/yCIaDZ01CHIeTGSmhzp\\\\\\\/q6n+DXlIlkDfVTI\\\\\\\/77XUwHnEPTTYgaqqSozdlQ+TkuLc3lEOpNbr4a9pNtcYRB9asgDdvKTPxBxTAKZGH2VY1opcIrm6MU67Cqvp7l2IlNbkp4NQDJGk6ox3ucYJt7BArycE3bpGbyVHo\\\\\\\/4NFP1A6U8UTKDJwvJR2suAylXtj8iavVCDraZZaWDLDiHoVRMNoZqiN6EWbqcFhiBbKCh3PCrbjVhuMZlMm4ImMZdu+bEVjv20QSI0bAjgBzUvD4Z\\\\\\\/v3s8dr\\\\\\\/WDSC5eqfR03jUYCLxOt+yQZb2M8t2IqOe5A8G5UUZgQPt2SCCRj+CaMMRFrExks3m5HxhVbcuPhNzNP3fG6AlOr+mR+WaqQx9QDJ8dhl6dBiVpno4zbuBkjGLn0aUxujzm0XiJuHxUxtqJZSp7wPBsYgh0i2Vb4qaRxVmUWjgPzOO3LU0JiTRZ3tYACijbULtNaGUm53KqSOLiLQg3ZyJL0LfdNI3LpyBJjCqFqS5ZtdPRPv38qVeWOXvEL1KDzn+60QOwSUeLT684CEZo3hEfHHGxW5wlQBNnmmorY\\\\\\\/Uvayc8\\\\\\\/m8gmm8qyiLLVrxUuSN3FRKBbO\\\\\\\/SaMnv5g\\\\\\\/6QrGloSnUYhIWG97+yY7hJ3QKLZg4zYSMeDvORRjYckxLRiM5ciFewxbrPITzKiEVyz+OpqkV8ya+WAuWMwrb\\\\\\\/V8eKcYLI6Qjo59b4b6jQYTg0Z0vJuGAOrDM6pCy4+oVtDnLnF05WcP1Vy23OqqBASvgwnf6dtwEG\\\\\\\/yQC+59xropTixxpdI55EIlASx708XD9Bx1OT+My6N3kn67CNiN4t0dS9cxsqs211xWZ8XQvYS16nhVQhP67ZfZvSvtjlknO5DsFUSRL6+k8PgFSFqI6Kwk7tsht4XVh8wdVnCIKpX0tkJkCMy1UI1ShvkG5uhjUfMa70glQEKrf4+4yQlK244gET\\\\\\\/d36JgTtftWFLvCw6olfD3\\\\\\\/S1+fStvZPr80b1vL4iRUGS8+pvz\\\\\\\/tC\\\\\\\/krIVZBqL+vOF15pPGynFq0T3iq8\\\\\\\/Ts5XHyXiCOIDgghFKfwzF\\\\\\\/utXYx\\\\\\\/2GRgzrgu77CkL1I+eXaPvw4+9vRpoEucr4\\\\\\\/O8lFzXB7tnL7992Sw7xRnXOGTmaJVLPTytMqHrWtafLmtI8Lp9gYhPebUVhp3be7qZP+TYG0cJC9nZ7CaUJ5eiFbV6mM6iPaJJsGyJq2dwj8zIHpAy+kQWuvVuNrQ2vbT6lcEHLecgBLf6cFHDSzv5sbb1VzueJJfx0rMhyEt0WFWcA8\\\\\\\/a3xf3cQw2lga9H7hch6b+GeqaXTkftmmeVFfFgaCi+wIkvPWEYRMpeboTQRXDKyIsmatQLDYj4tYOQKSF6sP9Kuzsgw4n\\\\\\\/Ft+unrxJNjYkihePqNHNwPgGCZbFkW+mGJkM4VwNZ2xo5JM6T4EE\\\\\\\/1nGTJIFH9U3Bye2vmRSqbnxUWljqCBFccsBGQGEy7JOcoF+CMia2ZP1V5VtzZviKMUyXKBoyUiRGCyWgjHYiMkEl1W0yjleitrYk8hjFmpcAzPv9Yw6Qe7JT1Sjga9Cjfm+QNMTQo0lxT9pR4SdPM\\\\\\\/tF4RzdjhVQ4SKzpeqOac0DET+iuaCKiLm8ftpChWJVXRbGS4Kjhqe3zxMsD1BqrHBsGaMx0fElFc+JQrtL4jK4AO0J5g1Lr8G990a6OGNPCTxqcpFVj1ux9Rs5LBOxPBlNLjbvIy\\\\\\\/lDzu54WCtvlM\\\\\\\/HP\\\\\\\/RmgtsJ86NcVsCbu9FP28kfGcmpBol9HDxJ5yY3X7wmwyRY394L6BL1UgGpDh242jqqwv9epyk4PyrGbbznT2xJmI52PtAfK18dYKP6+k7MiQV1SDW9DSqup0ovI7sgM3HN2RADcVzyjjCiL+N3IfIgkzwqQZObT3tE9vs0o4LZRggPB3DadteWFVtdey9oclehEh6+5N8ClpJFEL6IR2oPzTGEivyTevRFWRR4HIn\\\\\\\/EdBPHo\\\\\\\/o8IxEkIveIY+MuNtUpr0RsRU9WGjTLZ3kUqyy7FhuxG9SyC7P1uELEktACX24YEJyOoj2DKNn60wzr11VgAcSpOHIq43WzTV6uALUhuMc2wcNt9CQcGxMHxiUzl0DqIP79EJ4bZj4Sz7l6R+vIXFpDhph8PlhsslTNF3MYKJqDy4sCcutDTouEHwweinlQwh5eQZyU8bJ\\\\\\\/+mf3UIcGOCPd\\\\\\\/lQdBH7kade9FB75JEKrM6hGI9ko2Z5im5EI6J1nOP+Bh8I4QELfFtDcMBN8SBNqtJw7vhaaMWzottB0GDhe54b\\\\\\\/WrFCUpBWdf9Sr96up9GHVqAQyM4eDtDR+HnrJ+y7WzAQqhnlgb8cVN50xrAoP9mEXlriKHs2k4mzH2EFaKvoXLM5Az9rFe5e+SlpDBQxTBAXbG1mp0Da3W+LtBC0+SL7VdO2ltVN3T9dnptYFGoep42Zn31q0f1fTfjdy\\\\\\\/kQkJCrt2S167H7QmbNgqeO56WDGbqehmjZ\\\\\\\/byBk5tZzs05pfK3O3hxx\\\\\\\/9Tr6rzXWV1E0HHpe0rxrR8e1WLQ1+8HYYB2V2zAkZm4r8KDyfEoDSkNCGz8BndrrouS9fYJjALXhV\\\\\\\/mMLNBJofTvqF\\\\\\\/wsofEt6tLYBSUrXoWFa4Mt+UyKOIibUMWvocVl6QKDkCjr1MDdDYdw13uFlFR3xkKJIwMxB3IyfQqVfF4J6V3v\\\\\\\/4aZ07B1gA2B2dAhAc7RLRtd\\\\\\\/LuFpP\\\\\\\/0WlX2Fc4J5iEB82rsbqDmbc3N26YchxoJ2dwbphknNDwqpP\\\\\\\/Ck89Djy+KTYT75BXgNjEyCNjvRP\\\\\\\/UgD51ypGGB\\\\\\\/FmYIj7AkFumXZzlkdnL9Q7CclqniQ7q\\\\\\\/IJwlRVJIIJ0+LXdMgjZCos8hl8SQ7l\\\\\\\/ecd+5gqDSmBV0W2AWvjihmqCjQReZ5AMGy\\\\\\\/0n2je9bymkDZ82vmNoCqNj\\\\\\\/XCHAIK3bDiwdsTWeSgKS8QFPM7cjnov42y3zWrwqlwVVew7vDyMX73TyxY\\\\\\\/JlbvT5f7QyXQ05INVGY6Xqyym3ihOzA08OcVoDCex79g34uT62uTh5UYWbMMTgSTmDTagjnn8\\\\\\\/22t+Mk4P447qC39Bl2gNlaaVtMcFOONWKb5Fk+wKUdL4YEVYja4A5GQB3Eb2b1GRuj4YV8aO+1GWz0X35fVhdQTpRoEXnQZNq24CcuW7GzzrT6LWi3sVHIwC1iymzQiPZaJLRC5NsHIc6yn0Cj1Hu8WctkQ44ZlA7N8yhgixVVVIkUe+6OzA0kYJP2y4PO3eq5ygS9zl+0V6b0fSkJgNy8aFRdSHK4Tm+Tke7yoUj68ndaFUr46lzRIzxkPNnMuekM\\\\\\\/nE\\\\\\\/4o57Xuj+SGDPK\\\\\\\/PevFlaWnq4XBW\\\\\\\/ph1jOS9zoRoUxDp6tQGwc5E37QGC7lkexSDgbPXvq9RGa9PpzDR5aIh1GcDzzYQW7QbMlOY2kgZa8DYIIjpr1Si9dk611WdEr97O\\\\\\\/FAllH0HT9DbxzeC\\\\\\\/DnxlNUm8yz1B7YvfG0g1rA3prMw3U33QfrU3zB\\\\\\\/WIXGhf8D340Sgkp1oG4nS8LMqcqSzBw3lxCHY49nzb8uAc0I+ex6HCaZioeqqhe3m8aq8lcIZXhr4WAuRFoWsgQJYCYisUsoGbxXG11NkDYRJK35inH30UA5sJ4XTomim7MU8cwpBAJE+KVwlXxd4q87Q9ZDKSMPiaiNyY5AFBl4WjkxM9HyDxQ2xVxzkusGfSKhceNCxxjuP7H948kD9YjLmp4TT+A3QEVhxpKWhAlKmTCQqzkCUxM441TP\\\\\\\/HxyBt3ufa56YY\\\\\\\/9pmoexIi8e03GeFxAd01Mz7IQwj0mjugh2ktDe8\\\\\\\/mdrYgig7VR2lAjnh2AKa+t30z+XIitzDgY5rC3xVCmX2anIlKEK+yJ+yND7oCOW4W9xtBcXTEQsOFGqpIozDalQ2B28IFDYkab4dPezfrDBXAszPXkWrbNWxKVm6pYtCT+Pk5YQ9nHvacodIIFkoMTIHtwdTM6seyLlgNvpWvzQg8ztxGrzTZoU0akKbr6OkYmDiatKwXBwLjT090AIPDe4B0Plnm5TPSieEbCuwy0o+AlkQAYdPmRC3HyYkv9aooZKzgoO2NoyVk3TbYaavTu2kqsWEw16R4Yyu319UtqBs7p7a0gXesH0ghSnkt2kvolVa7jYoOK6fMhYQsjUR85QpDpN5fNiG5z4QWJ9HcXC0JZLljDUf850xNjswVirDvstklKIRQ90enJ6ZEFR\\\\\\\/uRRutys1eUu2LVZ3CNCdpfalEAac6JXfphV8LQ3A8dJlK\\\\\\\/fgzgChbCxN2com7svZT5L5tNR08\\\\\\\/BcfRQi5rtQXiIxvyUTKGX8HWCqUxcYIt0ZHVwRbTMNDacbBgVNX4PNEbSSav\\\\\\\/wmtORoRIZL4lizc1KnnJEFfOp6j+s3dd5MLzjs\\\\\\\/Am7YqaOrh83q\\\\\\\/6V2AKN9Vu9siX4Bvuvf9b0UZArvZZ3HZvWfy74hlUF0\\\\\\\/vgU5vNVwBguk60GPyj530WVxovgA3CqpKKznqiyVdVYO54J7ptxRdwogwVheHEhSyWT5dHVGcZokHF3GA\\\\\\\/\\\\\\\/9EB4pPPwNQdoQToJdT0x1F\\\\\\\/lCV0c8mQyqLtffGghNhhOW+Vug2nc345wBxJkYu+8Jf5mmwj2T7cwTqfBgHPHQX7GK87cfDNU3\\\\\\\/4+XuAOeIS2Uma1BFJRLc3jrAfZpUB9oIPt3E69XeUew0jn9zso9XvHpq1dJzdj68v1SASbb4N4GLnxGuNuJE9QcWKPmSM5XZJNM4TNxv7lhtDN0VG4F\\\\\\\/4eYqGyWBQ8L8T37b1oeYT2A4tj2rvxaFpFAZTF0yWtuoevR3B1cR+nB5RfPXfkK71NxxdBgR0SAv\\\\\\\/D1QoSZfMlGeJnF9JyLmLP3D2bEpasZLR0CO1IsIv+hiQTe\\\\\\\/AjkTyzhhaKtVAtaWTefm7th61ykzmT9FbqmHfDGgXnXETB9vC7Q4Z+RtcRlzPZNenFUGFGWC4WzvVR0H+le1MVFHih8cotTl\\\\\\\/pq+m9jOe4jimfndJzLtJ9aOi7Dprn0N29T7pNhp7ILzH4OKUhDoo3veNJGFaT6B2Nw6rsD7pwm23iG+X0fBj0UikmZpY9MvzC6fstuTOlhfwiNwZMcDXh7XgylRkAwRMWg8NRsk5BdchVQA2MAILPQwTR81SYCdEeN43E9UFf7mYBmcH6giln3z5g42LX0h1pKjHrwGSau4c71oNZsIQmFIyETFQuwoFD9KX3DI3KI\\\\\\\/Uya7f0zR+RL3FjTC7FuNuF3feb6F0+zn7mX557Vb4DQurC+FUOhJCwNC8O2bE22boWPZ9ifQxIbjBDNseBkam\\\\\\\/27Q+C5f\\\\\\\/YZGEGHfunvmX6qX7gp1SbHSb0OJNeOBc9VhERfbQGlctXwI3KX6e9ojin4azwIlEJo\\\\\\\/ktq9IrakFXkJUkeIBRF2e1mvl3lkrA30d1ICRrq8NCfc+XW\\\\\\\/PmwgqUYNST7OPF1TYF0cnOoq5LaChL+3cWyEHfjVPmNiSeM4SvwZhONofNVFRsQB1gwEP5c57+v5US2PtSFdqDBOqK7bByV\\\\\\\/sDvenwcsOmvTzLp9nNXgUkKBJTqAu8BV7QgdXknCqxwCpZYYgYgH9QIBQPlKuR5QpoULsX62RVnO8JyOcPT37ToKE\\\\\\\/M4RUyVJq9aHX1xFJyy3KWwGOm44Z1VgMRGr+I+HszPX6iqZe8iKh\\\\\\\/baPA4h4MyTNS1qkR0GceQ8RvbA8+PmlbeWWOyUQ+yoOh9E\\\\\\\/TtiJNH1AQsKW\\\\\\\/YTP2w2lvTPgmO6wB3skYAdNBmQH1zjH84P8\\\\\\\/GAf1+avyYuG3HnyOdqlRpurjYzt6FRKzOMot35Nbv18r6GLalFeEUvhUW5fcoe2MtiqJi7uddWKxcK4JJcHRJd46I61xYMfZE3dz+bUKMrLNFBR604LrGTZkezEfO1S7OepIjUNtjckmE6wnxceq9mFfZPb35DXwQNWz6yhHCgRrM6ONcvKLU\\\\\\\/xwj3zhAlqJ+OSAUVDzezPL\\\\\\\/EwfMvYqDmLLrtelwtof\\\\\\\/\\\\\\\/2IjAhtz45DhF30WVf6gUv4gDhGHH5lUVntku6+BbQeM9bZ4XJ1PlrrPhoBIEGFofBSvbCUycUDY90+5flNKtFxSSJWiWiiRYxIkMdYK26vEVt7U+0gM\\\\\\\/6cAAUFpkqG+3kxpvtNVa3XrDUO\\\\\\\/hNdlLsp0oUyvW2rfbcFrAznJ3OpMmgfrCzcWn+GjrArFPEexPTzsgO0M52SQJYEdUIuTjhpXz2MYybvu3NFtaywdwTXgJ0ElScG18ylrY6inzLx8tz70qxESuSe22brpviF4vpB7Kn+DEVJX7yCc8H3pT1Jbm\\\\\\\/v7MbglAb+RNnI1IzahQg7Vf4ogE1CpALVi1OrCuAM17lZjKXcZAB68m+VYYRnQNNK+07JvTtHw6rrbXimoBJ57XOWgqJiBfAG\\\\\\\/a\\\\\\\/hRYgSB7xOUTx+s0mmxPLNVF6yblZ3Jhxs+3k06UaNPnSusor97087wW27B7gn3wm+rYcEP\\\\\\\/Jnfk+05cxMKUA+80UJAAjbQFc7\\\\\\\/IEnupn21wohmJWcL4H\\\\\\\/UngioldMAO4V9gniONZ\\\\\\\/DP2kVoyLZufwDlqqj\\\\\\\/MY2lMLEJB0ydqM2XX21Pn9r2X9G4o0elGq50eoVCUsysJyVbt1Ay8KvRJxFdP8CC00SX4BrpksQpbQMm5TKLIa7qVPjaXgXNhDbWrMJbMw8tKBxK+uzVGfAcexq9mAg8n+w3+fEXqzCoYfm84sa0E38Bkkq2twfqhhmnv6uWGgaT2AR01bEmf21L2pv0JOf7DpeuGuERd5Z\\\\\\\/BKKEQNgmNpTQpbpt6M6bGLQ7EMvmtmPIM8BmmgmN4XQq7B0O9YNQ1mvQT5CEHs+P2JbnaPB6e7Y5MdAPmdZh8l7VzxE\\\\\\\/++Ab5ydA1G9DTYToy8OGcpNFCom+wySsjXbnvsa4Y3Pykxr2y3uaj957jgo\\\\\\\/LpSYdnLKNn4FxniltuIYqPH9IMyHIHhbzTxsHzjB+Bc5bABC2Ve6cgvguYDemS8dN2DddPiqngVgVUNsHDkNbQ3jpg9eN4iQWGN6\\\\\\\/U\\\\\\\/ai0G4+X1XGDH1JLdN0x4cq11GN1MkMbkUCmaiGFP\\\\\\\/y\\\\\\\/M8nUYy+VbcIY5h54oewxWYqxHubXgrwEaWKxdjmtp5zh5u4YK8MeSYuCZ6M7eMPQL8WWLSqgqyVBwNiZO0mAQTzxm\\\\\\\/xnvWp56EcMC4RcY+Cj+PrimuhZL5XnF4LUhThMq2dC2PP166A8jeJLGdDkqueWx\\\\\\\/VptabxVSIFeCE\\\\\\\/pQaEiENpW2fYJQomvvSiufTifgfOwAriEz\\\\\\\/MyGfYXdDOJzcuOmgUd\\\\\\\/iBJbM4xdEqUcGi03SJ7cNaCUf1zmJus9KRC7XdUnKPJNYLbM3MIJn7xmOlMgrvhIPNY7ucARV1Pt9\\\\\\\/ZFz6zrhI+v4J2N5JIrBZHOzXJCm2Gg9QBMhksyIikZ3\\\\\\\/\\\\\\\/IljoP4Ivpn71YQztka69uFdNUEKWXjpZBqyWKi5kdYqW3hxULoOBAPT0DY6+ZYzKWPUYHx8UEz09sP4rS2Rt2T2o5i4J8OPSlaGzsfKvpiUS8hRwd3L0dk0ui1nlcCKmH2ib1ZSij7u+KugTNxArix8YMCSqbOx0DaQNsJCpc4Asz2MxMrPVNGxA5sM79So7L9xFjrdpnZL\\\\\\\/XuO2b0P3sPv6p3U9zyrTIS8HJKDQpl1mrn7T74JeyPNZifxeSx9ny56jvQFdYCym8N1\\\\\\\/39B4GTgTZR\\\\\\\/tp9oFPuccPAZlj0DlsRg6\\\\\\\/QNrZ+V3NehH7YpU2bGBmUMr\\\\\\\/eX79k2ADInMIdkTkdm7uvPI6xg\\\\\\\/ZHKNrwnlGgH2bmmd1T8SUQM1YLcEHeWqNTUPlK8MVSPAAWwV9\\\\\\\/1hkL1uZOlpGhdmPaME1ANzp50mcMScTdJCtEGgVb9\\\\\\\/XPqGCemeVsv\\\\\\\/qlJFRoO4eLBUUijz9zXPk8CHMpcs5EMyzZOTlKzuUSaluFWIrGS+bUEAO9QBZeXC3yJaUPMHVwBVlqPmYbR75pedInVwZ1TeU87Skav1TlrmB4WEnKI37+QTX1l7BrPCYeYPBZwJUBRUepgijUIFFcWoaaToGJkqyayQk7nxRY4bKS6gSp6XKcVxC\\\\\\\/EcfvOQb8epmhQUno08HpGGF8bI56XhMrM5ctcmt7cGtG1EyL2A5j+A1EHt5JU8ImlenV3KU7koXJZLuHCPi0Ye2IYv1+af3hSs24ayrtWgnQaWMkEC1wwjIZqHIfCCbsmtstUaFYj1SKFkvyoVaYLHwuxOU3s+1chdO\\\\\\\/TdMul06XJl8Vcqd8o8Jhq3AX1iAK4WmdrnH+Ku+0R2KS3E7uHrvntthK6oEQnfq3MAAUy3JL\\\\\\\/X+QJe7CHSfmmd5yrM6k2S+wHMvNf3DfLa4cMbuiGYpbhuIOvXm9DVOUKdRZI3x1qtDksF\\\\\\\/EN44TtRFI4Rso4RXJSKM9S7R\\\\\\\/tnZ0MneKtoh89k5ROOLHKx8wLXSEwn9ldA9VGai0mEKrMX5L4poB3vIeb7scNfw0PsRyOTkQSa72xKi9NZtK4t6cLq+bhx\\\\\\\/btyNur3QRTvyDfJHddKEULGawo9tUm\\\\\\\/nTrtIrJ6\\\\\\\/aoHAT25DG1Vf39Dk8eD9fyRyKnCJ6EzANGOatzyj0v0DPXk8jTwlN\\\\\\\/9J\\\\\\\/t\\\\\\\/OquC8hgX+ajK1Vl2jVBRs6WIptZL+IcoCAP8e4tGfE8i+kUhxKjhLij8KaIEBosNO68WzSVZasXEPCUhtvU1aFW4pbLybQllRbLOrEy3yLirANc96r0fml\\\\\\\/vHq3nOcEVyj0xKSemJiM54WK1nIJ+AtR\\\\\\\/CibcGFKwvIQnY3ilKcVj3LZCZ+8KXWFMl4IMl0QayKLUZ+diODMRXTIEjWZqVFEyVrWZ9XtENrg9lIjph9oTGJ5ItBFFiY9YStg5RcztsL6Z98kedDN+ElVb6lJLOzjFrf58\\\\\\\/MGlHYLp\\\\\\\/nitZ1ixw+xHlFqxt9A0SOedCZahvC+A3nq0ggEpRB4HAJDEopXsHyDqXzlFAbxqPZPetS\\\\\\\/l+BX9aqCnI8TnCl8X8VMH8N9vU50B44kYiDoe71mpDw9kFngHnI9HUFDEZKu2RbQtKZeYcFY2hLmq7BfRjwbWojBj6gHGzx5Dv+97szOCJF+IENyQpmHbg2PnZEo+fAbWDWb9TF0vcRECJrf1aUK26iQBC7NNFB0grYMAaoYglD\\\\\\\/Ip5BXxDJoGcYaSPXulhJLiGnY8T0LzvtDVPylOpYjQb1KMxxuG6oHJdKXbSL+nZiNs0X7zCx6lmvFon6jMAMZE1EhDOCHSbKcFKKEWrnjOJKkhKWQ\\\\\\\/nKpRnKgZ85TNUE8TDJnd7elTe4Mh5hDdBeGDTmYIKDItNUgoQDCznF4OIeHHBL11vj7KQmr\\\\\\\/1I5B+qhdKGyoNoGvvmHJGwVDwTwWmGkRXihm2iUgfVAElx7L9sObM0U7RzSfDvJH2Zw1Y9635euRyHzNNashuUz0rmyEyYuZSMjFLMLLNp8vg+TvkNqX3qKEoZSJ5GsTOZI5Ffd09TkZG1CRRF0RL0ZIAl2LoKYZlNdK564svCps\\\\\\\/by18NaHPKoJGsCSb3OuE\\\\\\\/MzNVNZIblLWAhTsLpTelACtTDxIbWtbS7gmjYsZgHGPolMClyTh5\\\\\\\/dPSFHxDHYm85vpV4rLlQOxitkWPYCpx3VIH3FRErl1pcafQEYhOZnaI\\\\\\\/I06U6Y7+YXu4ForIQHVSni73mFBMkKrfxQRH4Ik3pzvLeBEuY3mBwu5VTcqb4ayfOpWMEmAPoPsihdo1Zripa0R6H2pGv16J9Xs78UEchJ4K04gBJA4STzV9es\\\\\\\/z2Fnlte65xX2IOBkEGiAl61GcrcrSnlfD57817\\\\\\\/GzCsyRlWadagLu5hgNoXYQmhKgaJ44HxPZGuFurflxQ4dDz79WXKOQBPjjkxY2jxt8G+JynMvOTKF5wB4C6XXrFRyL9DG5jTyAgq+TUKLqCixP0cf8WjFA1b8bUc\\\\\\\/ZTmotS+dw0w6psi2uCN5GUBB9\\\\\\\/lBJBQXFsjaPdoPdopvBbr0lka3iJ5x6yrXCDl8RuNPZR5PBquF0mrO4txrPj1AucLYnLCUBLBeB+wcWJp6gnQy9NfEfq6aL+k+1vbJwIRx89h8V4qgjPkisbhxmQdHjp5EkqsqKo0rHe9CFl+3oYvV1REGykkkYUFAbMeKNqIbH83M4PSOGn86y6puazSYP1sWe\\\\\\\/I3+iQYDMpbE0q1pQKePwoYhC9\\\\\\\/c+FV3wuw8FgHcVgBid3oTyussSRc9ChCkLZ8o6Y\\\\\\\/BXGlGhU3wDOs\\\\\\\/htheexLLyQ5PpesxNC+SphrRPJDMmEdSBTwTd1aOy0lgt6ow90i6Xn\\\\\\\/C2PnjByBmudiQqvaLS9JGOD8eXe3Ja2vPLFf\\\\\\\/rs9xkkOlsc1vvkc1i+7kNF8ZTrEh5Wt5J\\\\\\\/YuVm8OevAp1blZs+BtUDEgjRzm8xv6KfRkF8aTtmSnuAP8LOEjtIZzdpa\\\\\\\/mFaur0UGnbbsH7wy5ZOs5tNTubbu+e7VpX511VreLJYFGWLSTiuzuCR0je1JB9dZqW\\\\\\\/F7opbz20sv48hI4ippzB+Kt7mPdmlcCDLrcHuIyevB94kZth4InpqwA6e6wV427wT9\\\\\\\/SpkcEwQSVviKOQUgJa5y7CB4tm2jl3gnLmvAoiply2qJ1ZpZSMMlDTOvU7f3zrvNlvua79NtS\\\\\\\/r8F3P\\\\\\\/ZSSPxOa1a3Aa73P9hwkzjXObbLaC0jynDWMsyKXp4147\\\\\\\/tL4SxvD9gJK+YRIWWIVc\\\\\\\/C\\\\\\\/xg9SKUtLm8CVZZ1Yx4dCG8rTQ6UdqNGOZxbAD3kZtjnnlkKha6Eneh8Q\\\\\\\/H21ORcTkbXYvE1SI4j9Qrp7BaJVui6JVZWnSsD\\\\\\\/rwIlDXql9neZtisvHgp0o\\\\\\\/lRW+1kDT2JVPTWoQNWZhpNrZAWG7yCCf7TFw9gcI21mJGpUltnsQsARzgKMJccg+bMtWuolmyLfccZdAMn4a+2BmuexqgxhFjpDer6\\\\\\\/bWVtCTtmKTrwJQmhu\\\\\\\/ZLoGh5CgphOS+129HIGa22ptZVbbzkAWXzCqeFQD97yoXbEYhXFce\\\\\\\/hRDNF1V6Glbx5TjvFNe\\\\\\\/JgNr7ZHTNYowopVWQvzMi4mCC2zWVZ88uBNLUi9q6w8FbN\\\\\\\/Tr84ER7omS9K4nDMy0Hprmcl2VKqCuKIB1asW6FwcGW9AQi65FuFGQh9e7VN\\\\\\\/3zd5FJZ2t8pOm4qLkjtTCHoHkK3OauNlkNsV9HxolV600vNmE8dIWOshOCgTmQkBaVEKRmOzgRrvGf9JVRC2eEv5s7jzLV3GUuYHFIuNC8oMwjm90\\\\\\\/30qWxatu9R0C3sI3aDTrw\\\\\\\/0Z+GgJnUOjGNliPRHjsrDdYbdpJbZz8P3TiZAZN1PGgq1z8chQLfB671A5DrxKKZNyiT5Chahgd62yE\\\\\\\/wUu3dlmoyDj8n33h33XOVNbR14JxqXJHbk\\\\\\\/NzvOU1wQhQlnvDo11Spl5lCE6SNG2CTcdCa34YqFqgFWOe97+Knl+SE5Zs+oVGqrnSjD88bYHxAUosLS2TLDPFBbqdMW3pHyyZG\\\\\\\/rmtf4q8OFr89\\\\\\\/5MYrzgMtHJQfCFvqEBkeoXtbQ5MXrZnKduLGPFv4Myz9FSEid\\\\\\\/TQeaVV2uD9qlV7Ys86v8Dp0T74mYqsRCZIaqV\\\\\\\/Xohslrp354Tl4J8nu9Yq5R3kUcXJQVZMCdrbxXCp+JRcVRkwSlPHH8t03OFpApWisYgMmtvpAOA5+0FkpMrjGyozR3d\\\\\\\/bwD1JTJqtpm1rZteU1B2Q5amDgRb7xebVAY+uMTSkg0MKLujIWLo2vMVQPmOf8BFkvqGJB05eOY6V4PBT1PKQ6iC5+lftrPAu2I3dQB9ShiaR4y6dvle9aEUsDhrUnj6FX5yGeEMfHkOXsFHfXuRLmMhZKCtliEVyIVnXNQwqXxX3b3QZcE2OxNmUqfVQOSqwKgPAEMKMOGSEjYOkrGrr3+w==\\\u0022,\\\u0022iv\\\u0022:\\\u0022yThXqr\\\\\\\/xQhotHszKGBfmNQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022ebf4731581935abfc690ac78f05b9bf90e3e02501eb3d6a72ff0d1c78821756c\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022TlPpzIrxB4F53+\\\\\\\/mtA6N+Cjo6J5q12t6KdGZcpQ6VolnuhXA6R2GHeJnylj2AFysrFVJgq06Jw2WJjuf6nLH7QY79LdQybT2FwjMycQenHZ4\\\\\\\/OEmTodGWBjcxjcmqdUAGp4NdUeJVADOIOiWtEGF57HbIEO9mkX+iKIdrcN+toH86LopzlneNVKW06jncFxB65lGfakOsOXql9rwcq+GOCkavL0GltIqr70U9icwC1zYtAJqxlEdMtZQ79xueZ3eyKPTiSoOg3uQLHAgaH9Gc\\\\\\\/fGx52mZyfSNKM7GacsR+3ARhXTQm7588BoeSoKuDGk4jH3YO0hRGVnS+0bZIrLZSpFH84N0aHXGUhox5sm0Ru8aWCikMFi5wP9qO8JC9hqQxTp0Rt3rKYhCbxXeGv9t2u+RIzZMjnkNx4ePYcMdy1hjwX6nku86SJaKSH\\\\\\\/Av21ZJt4ccGa2DhEHU0lq3kZDCcJtYlMzwM1VwW28gEhdPOw6pTTIalRHwNNxL+UkHacN0f\\\\\\\/rnQFPfvLJFRDQxQKBWEjPIHT91uLeL\\\\\\\/sKEae5HYukhYri1or6WwPwZrMKRHElP8U+MmtgEXE0P7w8FBBLFM+VURlVWHCfkkfahEgbuhu5btBiJA5nzptV3F4JB5EadfHo5J7rKvhfY4jD3D5R0uw+D6XCqld2+GUhvSDUN3nyQEfolB5aiYBGQbgdmFSNBwr6ngvaOWRgTFAjkmmthLyeJjNKs4Zi2h1pOpc8GCqaO\\\\\\\/PV9Jj\\\\\\\/UKub9NgGGr1DY8hdLy7oxQRmB4y3Iz4+9IZth3zSZLXIkGTsNcxuboyxKS++TbLsB4Ox48YHZFDaW7livtwQU34h\\\\\\\/kuAdoFQH5ZGhepmDtVrJfNXgNjC8OUm7Kv3+jWccx3m5L2y+M\\\\\\\/\\\\\\\/u0RPSMCCcT\\\\\\\/vLLuWE8cwej9VIqMYHF0Gc6hQOXAdWeDRI1DKOM8JMEjCWCLb\\\\\\\/Z+0xr1C3HwPQveEks3MWS9yYY3qPegofiFayFa62SQBCADzQuWJ5UJRIjsM3C8wKZRT2PFu8zgGzTYgc\\\\\\\/Qlz7SsXPr8Zyi7FTCVOvgr+InCa0yU2e\\\\\\\/KTX\\\\\\\/NxD1hi0E9mKXjmOwL1SWw6JmP3DwOdbTkdVhw8VRVe5hwDZ4ejoJXo531F3PRgK0i8lSqDL+yCgi8qY\\\\\\\/Ht48W5K+3ZkcjgGZ3Uf9bwAxDZNhDzW0iBxFENU1ZwbUvoBXndcmZcs7\\\\\\\/Andg7s5aykTnbgw+JhqryPbCbfOQSd8rcev7rFg7R9NUAHcxiJtYJFnm\\\\\\\/qhOMCayjnJ3LZ1ncSv8oLYaDEIXfKGj8ZoQ13X0FSsZPXm4km3jSAZ3+BSbUBiYdS\\\\\\\/dE+F1RPeu9l5O9hNRlVcbYPm5uOUuGTFMI6w6BiYzMuMJ9j8Iq0\\\\\\\/W0sS0\\\\\\\/8s\\\\\\\/NMFhS1uT81ppkuDYX\\\\\\\/L+2Ijfc2\\\\\\\/+3oljQJjm\\\\\\\/vLG3\\\\\\\/7m+CeBI6O9X5\\\\\\\/\\\\\\\/Rj9MqlbGogMq1L\\\\\\\/wMT4yxZV9eYdJRy\\\\\\\/fzr8j4wdQVdmOD7RoLQsOUp4GqbCaL5QIDldM5s+wJpWQmTvBjsL8HwdZcqg39wp+O5JHzWRGU9EZs9VZ9wb4WPsnzy5hjOXC4wWBMSErnvV7TxInYCYppieIwX\\\\\\\/oH8I+9p7gM4gbifRKPhRVNspysh3TdThcnmKU2sbijXVuI50hwlGHM3ytHcFGq2sBlFml18zTbW\\\\\\\/kvGcWyzamyyytySpaSQpnw0xiBlR1YlGTBeg5G7iM+y1e1WXtGBV2vIPP6jST1+i2hNhHgCDlyBM\\\\\\\/hWfMvAOazFXPAKMP5Xse5Z5g+Ca+yxPrxTYuMjcwDiqmKFgIzhrggj3wwSoFiB+9VrLsg4S8vb0Rm\\\\\\\/yTTX1uP2zPxETyegS7AJKUpLgbANxdVrHnYIwFzTHLy0hOJVBl2uvpmkns4IC0\\\\\\\/w\\\\\\\/tuW4fbXZRJ2zBk92V6dGhpoomR+Ic0OIsvFULEKoRYbkwzS0J9GtDFKQz\\\\\\\/+nkKkMRyoJTPlwkTfF8Oxokq0TrZpR+eQCjauNnszcSw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022eEKhOKrtXLt9klWCHvzSVg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002294838f698deff33e2ecba7141604cd8d2556708f6e4cf08d857a6613c1e06848\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-492611 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/4/1249" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-dRguyB5PCn4NouBPxckUSAfOAdUck-nQYPynG1budQY" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/4/1249" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Y-hjOlZ4JNO73kE31SbbYGCN6ig6Ke527W1VOuGvjos" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/4/1231" title="Hyperspectral Reflectance-Derived Relationship Matrices for Genomic Prediction of Grain Yield in Wheat" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/4/1263" title="Identification and Expression Analysis of Long Noncoding RNAs in Fat-Tail of Sheep Breeds" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="492611" id="node492611" data-pisa="ggg;9/4/1249" data-pisa-master="ggg;g3.119.400086" data-apath="/ggg/9/4/1249.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Random Chromosome Partitioning in the Polyploid Bacterium <em>Thermus thermophilus</em> HB27</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-9428-6190" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Haijuan Li</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1249-1261;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400086 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node492611" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Haijuan Li </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>College of Biological and Environmental Engineering, Xi’an University, Xi’an 710065, Shaanxi, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Haijuan%2BLi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Li%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AHaijuan%2BLi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-9428-6190" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Haijuan Li</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:haijuanli@xawl.edu.cn" class="" data-icon-position="" data-hide-link-title="0">haijuanli@xawl.edu.cn</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/9/4/1249" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:492611" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:492611/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/4/1249.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:492611" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:492611/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/4/1249.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:492611" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:492611/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/4/1249.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:492611" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:492611/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/9/4/1249.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:492611" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:492611/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Little is known about chromosome segregation in polyploid prokaryotes. In this study, whether stringent or variable chromosome segregation occurs in polyploid thermophilic bacterium <em>Thermus thermophilus</em> was analyzed. A stable heterozygous strain (HL01) containing two antibiotic resistance markers at one gene locus was generated. The inheritance of the two alleles in the progeny of the heterozygous strain was then followed. During incubation without selection pressure, the fraction of heterozygous cells decreased and that of homozygous cells increased, while the relative abundance of each allele in the whole population remained constant, suggesting chromosome segregation had experienced random event. Consistently, in comparison with <em>Bacillus subtilis</em> in which the sister chromosomes were segregated equally, the ratios of DNA content in two daughter cells of <em>T. thermophilus</em> had a broader distribution and a larger standard deviation, indicating that the DNA content in the two daughter cells was not always identical. Further, the protein homologs (<em>i.e.</em>, ParA and MreB) which have been suggested to be involved in bacterial chromosome partitioning did not actively participate in the chromosome segregation in <em>T. thermophilus</em>. Therefore, it seems that protein-based chromosome segregation machineries are less critical for the polyploid <em>T. thermophilus</em>, and chromosome segregation in this bacterium are not stringently controlled but tend to be variable, and random segregation can occur.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/polyploid-prokaryotes" class="hw-term hw-article-keyword hw-article-keyword-polyploid-prokaryotes" rel="nofollow">polyploid prokaryotes</a></li><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/thermus-thermophilus" class="hw-term hw-article-keyword hw-article-keyword-thermus-thermophilus" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Thermus thermophilus</em></a></li><li class="kwd"><a href="/keyword/random-chromosome-segregation" class="hw-term hw-article-keyword hw-article-keyword-random-chromosome-segregation" rel="nofollow">random chromosome segregation</a></li><li class="kwd"><a href="/keyword/para-and-mreb-homologs" class="hw-term hw-article-keyword hw-article-keyword-para-and-mreb-homologs" rel="nofollow">ParA and MreB homologs</a></li></ul><p id="p-3">An increasing number of bacterial and archaeal species are recognized as being oligo- or polyploid, <em>e.g.</em>, containing on average more than one chromosomal copy per cell. Vastly different levels of the number of genome equivalents per cell have been measured in microorganisms belonging to such distinct groups as the <em>Archaea</em>, <em>Cyanobacteria</em> and <em>Proteobacteria</em> (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Breuert <em>et al.</em> 2006</a>; <a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Schneider <em>et al.</em> 2007</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Hildenbrand <em>et al.</em> 2011</a>; <a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Pecoraro <em>et al.</em> 2011</a>). The reported genome copy numbers in these polyploid microorganisms range from two to several hundred, and there are extreme cases with several thousand copies, like in the giant endosymbiont bacterium <em>Epulopiscium</em> sp. (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Mendell <em>et al.</em> 2008</a>). It has been previously shown that a certain strain of <em>T. thermophilus</em>, a member of the deep-branching <em>Deinococcus-Thermus</em> group, which has been established as a model thermophilic organism, is also polyploid, carrying four to five genome copies per cell under slow growth condition, these genome copies are assumed to be genetically identical (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Henne <em>et al.</em> 2004</a>; <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Ohtani <em>et al.</em> 2010</a>). Almost 40 years ago, the same ploidy level has been determined for the related radiation-resistant bacterium <em>Deinococcus radiodurans</em> (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Hansen 1978</a>). Several possible evolutionary advantages of having more than one genome copy have been suggested. For example, genome redundancy is associated with higher resistance against DNA damaging agents and permits the accumulation of otherwise deleterious recessive mutations (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Comai 2005</a>; <a id="xref-ref-63-2" class="xref-bibr" href="#ref-63">Pecoraro <em>et al.</em> 2011</a>; <a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">Soppa 2013</a>, <a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">2014</a>). An important question is how polyploid microorganisms partition the multiple genome copies to daughter cells. The mechanism of genome partitioning has been extensively studied in model bacteria like <em>Escherichia coli</em> (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Lampo <em>et al.</em> 2015</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Cass <em>et al.</em> 2016</a>), <em>Bacillus subtilis</em> (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Kloosterman <em>et al.</em> 2016</a>; <a id="xref-ref-82-1" class="xref-bibr" href="#ref-82">Wang <em>et al.</em> 2017</a>) and <em>Caulobacter crescentus</em> (<a id="xref-ref-78-1" class="xref-bibr" href="#ref-78">Taylor <em>et al.</em> 2017</a>), which are monoploid (only very slowly growing <em>E. coli</em> cells are truly monoploid). In a stark contrast, very few literatures could be found describing chromosomal dynamics in bacterial cells with more than two complete chromosome copies. Basically, chromosome partitioning in polyploid bacteria could be either stringent or variable (<em>e.g.</em>, random). There is an additional layer of complexity in a dividing polyploid cell – if, for some reason, the cell carries two different alleles at the same locus (heterozygous), there would be different genetic outcomes in the progeny depending on the basic mode of chromosome partitioning. In a stringent mode, each chromosomal copy is actively passed on to the recipient cells via various partitioning mechanisms. Such a partitioning mode would lead to progeny which is genetically identical to the parents. If the chromosomes are divided randomly in the daughter cells, similar to high-copy-number plasmids, these cells can receive any combination and/or different numbers of the parental genomes. An important consequence of this mode would be that, if the parental cell was heterozygous, the daughter cells can be genetically different from the parent due to allele segregation. Such a mode has been proposed for polyploid euryarchaeota <em>Methanococcus jannaschii</em> (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Malandrin <em>et al.</em> 1999</a>), and polyploid cyanobacteria <em>Anabaena</em> sp. 7120 (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Hu <em>et al.</em> 2007</a>) and <em>Synechocystis</em> sp. (<a id="xref-ref-65-2" class="xref-bibr" href="#ref-65">Schneider <em>et al.</em> 2007</a>) based on DNA content analysis in dividing cells.</p><p id="p-4">In <em>T. thermophilus</em>, it is not known whether the chromosome copies segregate into daughter cells via a stringent or variable mode. In contrast to some polyploid cyanobacteria, the genomes of so far sequenced members of the <em>Thermales</em> order (including <em>T. thermophilus</em>) contain homologs of the actin-like MreB protein as well as presumably complete chromosomal ParAB - <em>parS</em> systems, which have been suggested to be involved in active genome partitioning in many bacterial species. For instance, in <em>E. coli</em> it has been shown that MreB is important for both chromosomal bulk DNA and origin segregation (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Kruse <em>et al.</em> 2003</a>, <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">2005</a>, <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">2006</a>; <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Madabhushi and Marians 2009</a>). Similarly, depletion of <em>mreB</em> in <em>B. subtilis</em> and <em>C. crescentus</em> leads to considerable defects in chromosome segregation, where replication origins fail to localize correctly (<a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">Soufo and Graumann 2003</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Gitai <em>et al.</em> 2004</a>, <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">2005</a>; <a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Shebelut <em>et al.</em> 2009</a>; <a id="xref-ref-77-1" class="xref-bibr" href="#ref-77">Sundararajan and Goley 2017</a>). Disruptions of <em>parAB</em> in the chromosomal ParAB - <em>parS</em> systems also cause defects in chromosome segregation in many bacteria, <em>e.g.</em>, in <em>B. subtilis</em> (Ireton <em>et al.</em> 1994; <a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Sharpe and Errington 1996</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Lewis and Errington 1997</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Lee and Grossman 2006</a>; <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Murray and Errington 2008</a>; <a id="xref-ref-76-1" class="xref-bibr" href="#ref-76">Sullivan <em>et al.</em> 2009</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Gruber and Errington 2009</a>; <a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Scholefield <em>et al.</em> 2011</a>), <em>C. crescentus</em> (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Mohl and Gober 1997</a>; <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Mohl <em>et al.</em> 2001</a>; <a id="xref-ref-79-1" class="xref-bibr" href="#ref-79">Toro <em>et al.</em> 2008</a>; <a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Ptacin <em>et al.</em> 2010</a>; <a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Shebelut <em>et al.</em> 2010</a>), <em>Streptomyces coelicolor</em> (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Kim <em>et al.</em> 2000</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Jakimowicz <em>et al.</em> 2002</a>), <em>Vibrio cholera</em> (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Fogel and Waldor 2006</a>; <a id="xref-ref-83-1" class="xref-bibr" href="#ref-83">Yamaichi <em>et al.</em> 2007</a>), <em>Pseudomonas aeruginosa</em> (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Lasocki <em>et al.</em> 2007</a>), <em>Corynebacterium glutamicum</em> (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Donovan <em>et al.</em> 2010</a>), <em>Streptococcus pneumonia</em> (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Minnen <em>et al.</em> 2011</a>) and <em>Mycobacterium smegmatis</em> (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Ginda <em>et al.</em> 2013</a>). In this manuscript, the mode of chromosome segregation in <em>T. thermophilus</em> HB27 was investigated. A heterozygous strain in which two antibiotic resistance alleles could be stably maintained in the presence of selection pressure was initially generated. The inheritance of the two alleles in the absence of selection was then followed in the daughter cells by molecular and genetic techniques. The results from these experiments suggested that in <em>T. thermophilus</em>, partitioning of the chromosomal copies would experience random event. Analysis of dividing nucleoids in <em>T. thermophilus</em> and <em>B. subtilis</em> by fluorescence microscopy further confirmed that the chromosome segregation occurred less stringently in <em>T. thermophilus</em> than in <em>B. subtilis</em>. In addition, through generating Δ<em>mreB</em>Δ<em>parA</em> double deletion mutant followed by phenotype analysis with respect to chromosome segregation, it was found that in <em>T. thermophilus</em> both the ParA and MreB homologs were not actively participated in chromosomal bulk DNA segregation. The data from these experiments strongly suggest that the protein-based chromosome segregation machineries are less important for the polyploid <em>T. thermophilus</em> cells, and the chromosome segregation in this bacterium is likely non-stringent or often irregular.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials And Methods</h2><div id="sec-2" class="subsection"><h3>Bacterial strains and growth conditions</h3><p id="p-5"><em>E. coli</em> DH5α was used as a host strain for plasmid constructions and was grown at 37° in LB medium (contains 10 g/L tryptone, 5 g/L yeast extract, 5 g/L NaCl). <em>T</em>. <em>thermophilus</em> HB27 wild type (DSM 7039) and derivative strains were grown at 70° or 60° in TB medium. TB medium contained 8 g/L trypticase peptone (BD Biosciences, Heidelberg, Germany), 4 g/L yeast extract, and 3 g/L NaCl, and had a pH of 7.5. <em>B. subtilis</em> 168 (DSM 402) was grown in LB medium at 30°. The growth media were supplemented with ampicillin (100 µg/mL for <em>E. coli</em>), kanamycin (50 µg/mL for <em>E</em>. <em>coli</em> and 20 µg/mL for <em>T</em>. <em>thermophilus</em>), bleomycin (“Bleocin”, Calbiochem, 15 µg/mL for <em>E</em>. <em>coli</em> and 3 µg/mL for <em>T</em>. <em>thermophilus</em>) or X-Gluc (5-bromo-4-chloro-3-indolyl-β-D-glucopyranoside, 100 µg/mL) when necessary.</p></div><div id="sec-3" class="subsection"><h3>Determination of chromosome copy numbers</h3><p id="p-6">The real-time quantitative PCR was essentially performed based on the method described (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Breuert <em>et al.</em> 2006</a>; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Li <em>et al.</em> 2015</a>). Specifically, two loci (TTC1610 (near <em>oriC</em>) and TTC0574 (closer to <em>terC</em>) on the chromosome were chosen as the investigation targets. Approximately 1 kbp standard fragments of the two loci were PCR amplified using <em>T. thermophilus</em> genomic DNA as a template. The PCR products were purified from agarose gels followed by photometrical measurement to determine the concentrations (Nanodrop 2000). The molecular concentrations of the standard fragments were calculated using the online software “oligo calc” (<a href="http://www.basic.northwestern.edu/biotools">www.basic.northwestern.edu/biotools</a>). A series of dilutions of the standard fragments containing defined numbers of molecules were used in the real time PCR analyses. Standard curves were then generated by plotting the C<sub>t</sub> values against the corresponding molecular concentrations in the reaction (molecules/μL). Quantified cell numbers (determined both by spectrophotometry and Neubauer counting chamber) from exponentially growing cultures were harvested and lysed by cell lysis buffer, the cell lysis efficiency was determined by cell counting. The cell lysates were then dialyzed, and aliquots from the dilution series of the cell lysates were used as templates in the real time PCR reactions. For real time PCR reactions, fragments of 139 bp (TTC1610) and 119 bp (TTC0574) were amplified, and three independent repeats were performed. Eventually, the chromosomal copy numbers per cell were calculated based on the created standard curves and the cell density.</p></div><div id="sec-4" class="subsection"><h3>Plasmid construction</h3><p id="p-7">All plasmids and strains used are listed in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a> and all primers used are listed in <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>. For construction of pUC-Δ42, the flanking regions about 1 kbp of the <em>bgl</em> gene (TTP0042, encoding β-glucosidase) were respectively PCR amplified from <em>T. thermophilus</em> HB27, using primers that permitted to generate sufficient overlaps between each other; the two flanking regions were then cloned into <em>Xba</em>I digested pUC18 vector via Gibson Assembly method (New England Biolabs) (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Gibson <em>et al.</em> 2009</a>). The plasmids pCT3FK-2 and pJ-Δ<em>pyrE</em>::<em>blm</em> were used in the generation of the stable heterozygous strain HL01. pCT3FK (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Angelov <em>et al.</em> 2009</a>) in which a kanamycin resistance marker (<em>kat</em>) was sandwiched by two flanking regions of <em>pyrE</em> gene (TTC1380) was mutagenized (Change-IT Multiple Mutation Site Directed Mutagenesis Kit, Affymetrix, USA) to introduce two <em>Zra</em>I sites in the flanking sequences, followed by restriction and re-ligation to excise 521 bp of flanking sequences, while maintain the native <em>pyrE</em> flanking regions intact, this gave pCT3FK-2. For generation of pJ-Δ<em>pyrE</em>::<em>blm</em>, a 2.3 kbp <em>pyrFE</em> region was PCR amplified and cloned in pJET1.2 giving pJ-<em>pyrFE</em>; two <em>Nde</em>I sites were introduced to pJ-<em>pyrFE</em> by the same mutagenesis tool at positions -3 and +551 relative to the ORF of the <em>pyrE</em> gene. The mutagenized pJ-<em>pyrFE</em> was then digested with <em>Nde</em>I to get rid of the <em>pyrE</em> gene, and ligated with a bleomycin resistance marker <em>blm</em> (sequence data from <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Brouns <em>et al.</em> 2005</a>). The plasmid pMK-<em>parB-sgfp</em> allowing expression of ParB-sGFP in <em>T. thermophilus</em> was constructed in a former study (<a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">Li <em>et al.</em> 2015</a>). The plasmids pUC-Δ<em>mreB</em>::<em>kat</em> and pUC-Δ<em>parA</em>::<em>blm</em> were gene deletion vectors for generation of double knockout mutant Δ<em>mreB</em>::<em>kat</em>/Δ<em>parA</em>::<em>blm</em> in <em>T. thermophilus</em> HB27. The two flanking regions (about 1 kbp) of the <em>parA</em> (TTC1605) and <em>mreB</em> (TTC1464) genes, and the <em>blm</em>/<em>kat</em> cassette were respectively amplified by PCR, using <em>T. thermophilus</em> HB27 genomic DNA and plasmids containing the <em>blm</em>/<em>kat</em> cassettes (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">de Grado <em>et al.</em> 1999</a>; <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Brouns <em>et al.</em> 2005</a>) as the templates. Gibson assembly reaction was used to assemble the three PCR fragments into <em>Xba</em>I-digested pUC18 respectively, as they contained sufficient overlaps between each other.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/492726/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/492726/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Strains and plasmids used in this study</span><div class="sb-div caption-clear"></div></div></div><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/492716/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/492716/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Primers used in this study</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-5" class="subsection"><h3><strong>Determination of the genetic outcomes of the heterozygous</strong> strain <strong>wt/</strong>Δ-bgl</h3><p id="p-10">The gene clean deletion vector pUC-Δ42 was linearized by <em>Hin</em>dIII and transformed into <em>T. thermophilus</em> HB27 based on the protocol described by <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">de Grado <em>et al.</em> (1999)</a>. The transformation reaction was appropriately diluted and plated on TB agar plates supplemented with X-Gluc. After incubation at 70° for 2 d, colonies that exhibited intermediate color were selected for genotype analysis. The correct heterozygous strain (wt/Δ-<em>bgl</em>) was then grown in TB medium for 24 h and the culture was restreaked on TB-X-Gluc plates. The phenotype of the progeny was discriminated by colony color, and the genotype was determined by PCR.</p></div><div id="sec-6" class="subsection"><h3>Generation of homozygous T. thermophilus gene deletion mutants</h3><p id="p-11">The allele exchange vectors pCT3FK-2 and pJ-Δ<em>pyrE</em>::<em>blm</em> were linearized by <em>Sca</em>I and <em>Hin</em>dIII respectively, then transformed into <em>T. thermophilus</em> HB27 in the same manner. The transformation reactions were respectively streaked on TB agar plates supplemented with kanamycin or bleomycin. Several colonies were randomly selected, and PCR (using primers flanking the <em>pyrE</em> gene) was used to sort out the colonies which only contained the mutant allele. This resulted in two homozygous strains, <em>i.e.</em>, Δ<em>pyrE</em>::<em>kat</em> and Δ<em>pyrE</em>::<em>blm</em>, which carried either the <em>kat</em> or the <em>blm</em> resistance marker at the <em>pyrE</em> gene locus. For generation of Δ<em>mreB</em>::<em>kat</em>/Δ<em>parA</em>::<em>blm</em> mutant, single Δ<em>mreB</em>::<em>kat</em> mutant was initially created. The pUC-Δ<em>mreB</em>::<em>kat</em> vector was linearized by <em>Hin</em>dIII and transformed into wild-type <em>T. thermophilus</em> cells followed by plating on agar plates containing kanamycin. Several transformants were exposed to PCR analysis to determine mutants containing homozygous Δ<em>mreB</em>::<em>kat</em> allele. Linearized pUC-Δ<em>parA</em>::<em>blm</em> was sequentially transformed into the correct Δ<em>mreB</em>::<em>kat</em> strain, transformants were selected on agar plates containing both antibiotics. The homozygous double deletion mutant Δ<em>mreB</em>::<em>kat</em>/Δ<em>parA</em>::<em>blm</em> was prescreened by PCR and further confirmed by Southern blot.</p></div><div id="sec-7" class="subsection"><h3>Construction of a stable heterozygous strain and kinetics of allele segregation</h3><p id="p-12">The heterozygous strain HL01 was obtained by transforming linearized pJ-Δ<em>pyrE</em>::<em>blm</em> to the homozygous Δ<em>pyrE</em>::<em>kat</em> strain and selecting on TB agar plates supplemented with both kanamycin and bleomycin. The heterozygous strain was initially verified by PCR as well, and further confirmed by Southern blot and DNA sequencing of the respective genomic regions. The allele segregation experiment was performed by growing cells of HL01 in 5 mL of TB medium supplemented with both antibiotics for 12 h, followed by washing the cells three times in 1 × PBS buffer and inoculation in 50 mL antibiotic-free TB medium (time point 0). Growth was continued for 120 h at 70° with agitation. Samples (2 mL) were taken at various time points as indicated in the results to determine the fraction of each phenotype by spreading the cells on TB plates (without antibiotics) and re-streaking the colonies on plates supplemented with kanamycin or bleomycin. The same colony was parallelly streaked on TB plate to determine the growth. The relative amount of each allele of the samples was estimated by Southern blot followed by band intensity measurements (Image J, NIH, USA).</p></div><div id="sec-8" class="subsection"><h3><strong>Fluorescence microscopy</strong></h3><p id="p-13">To measure the relative DNA content in two dividing cells by fluorescence microscopy, the <em>T</em>. <em>thermophilus</em> strains (HB27, HB27/ParB-sGFP, and HB27 Δ<em>mreB</em>::<em>kat</em>/Δ<em>parA</em>::<em>blm</em>) and <em>B. subtilis</em> 168 cells were grown to exponential phase. Cells from 0.5 mL cultures were washed once in 1 × PBS buffer, fixed by ethanol, and resuspended in the same volume of PBS buffer. When necessary, the DNA-specific dye DAPI (4′,6-diamidino-2-phenylindole dihydrochloride) and CFS (6-carboxyfluorescein) which stains the cell membrane were added to the cell suspension with final concentrations of 0.2 µg/mL and 10 µg/mL respectively, followed by incubating at room temperature for 30 min. The residual dyes were removed by washing the cells three times with 1 × PBS buffer. After mounting the cells on glass slides, fluorescence microscopy was carried out (Zeiss Axio Imager M1). The filter “DAPI” was used to observe DAPI-stained DNA signal, and the filter “AF488” was used to observe CFS-stained membrane signal and ParB-sGFP signal. The acquired micrographic images were analyzed with the AxioVision software (Carl Zeiss, Germany) and Image J (NIH, USA).</p></div><div id="sec-9" class="subsection"><h3>Data availability</h3><p id="p-14">The author affirm that all plasmids are available upon request, and all data necessary for confirming the results of the article are included within the article, figures and tables. Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.7746878">https://doi.org/10.25387/g3.7746878</a>.</p></div></div><div class="section results" id="sec-10"><h2 class="">Results</h2><div id="sec-11" class="subsection"><h3>Real-time <strong>PCR determination of the chromosomal copy number of T. thermophilus</strong></h3><p id="p-15">The <em>T</em>. <em>thermophilus</em> strain HB8 has been reported to contain four to five copies of chromosomes per cell under slow growth condition (<a id="xref-ref-60-2" class="xref-bibr" href="#ref-60">Ohtani <em>et al.</em> 2010</a>). For the <em>T</em>. <em>thermophilus</em> type strain HB27, the chromosomal copy numbers are not clearly reported. Using real-time PCR method, the chromosomal copy numbers of <em>T</em>. <em>thermophilus</em> HB27 during exponential growth in rich medium was determined. This real-time PCR method was developed by <a id="xref-ref-8-3" class="xref-bibr" href="#ref-8">Breuert <em>et al.</em> (2006)</a>, and has been used to determine the exact genome copy numbers in the cell populations of various archaeal and bacterial species (<a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Hildenbrand <em>et al.</em> 2011</a>; <a id="xref-ref-63-3" class="xref-bibr" href="#ref-63">Pecoraro <em>et al.</em> 2011</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Griese <em>et al.</em> 2011</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Lange <em>et al.</em> 2011</a>; <a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">Spaans <em>et al.</em> 2015</a>; <a id="xref-ref-85-1" class="xref-bibr" href="#ref-85">Zerulla <em>et al.</em> 2016</a>), emphasizing the reliablity of this method. With the aim of accuracy, two loci (TTC1610 (near <em>oriC</em>) and TTC0574 (relatively closer to <em>terC</em>) on the chromosome was chosen as the investigation targets. The copy numbers of the two loci determined by the real-time PCR method are summarized in <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>, which clearly showed that multiple sets of chromosomes existed per cell during the exponential growth phase in <em>T</em>. <em>thermophilus</em> HB27, and this is consistent with the study performed in the strain HB8 (<a id="xref-ref-60-3" class="xref-bibr" href="#ref-60">Ohtani <em>et al.</em> 2010</a>).</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/492772/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/492772/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Chromosome copy numbers in <em>T. thermophilus</em> HB27</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-12" class="subsection"><h3>Heterozygous T. thermophilus HB27 cells and their <strong>g</strong>enetic outcomes</h3><p id="p-18">Since <em>T</em>. <em>thermophilus</em> HB27 is polyploid, it is conceivable that under certain condition, heterozygous cells containing two different alleles at one gene locus would appear. In practice, this kind of heterozygous cells was previously noticed during studies of <em>T. thermophilus</em> HB27 when generating deletion mutants of non-essential genes. Take <em>bgl</em> gene for example, wild-type <em>T. thermophilus</em> HB27 cells express β-glucosidase (Bgl). It is easy to distinguish Bgl<sup>-</sup> colonies (yellow) from Bgl<sup>+</sup> colonies (blue) on agar plates supplemented with indicator substrate, <em>i.e.</em>, X-Gluc (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Ohta <em>et al.</em> 2006</a>). Interestingly, when creating Δ<em>bgl</em> mutant through direct double-crossover homologous recombination method, although apparent homozygous Δ<em>bgl</em> mutant (yellow color) could be easily detected on TB X-Gluc plates (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>), transformants which projected intermediate phenotype (the color was between yellow and blue) could also be found (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1A</a>). PCR analysis showed that these “intermediate” colonies were carrying both knockout and wild-type alleles simultaneously at the <em>bgl</em> locus (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1B</a>). However, they were genetically unstable, since after growing in liquid medium without selection followed by re-streaking on agar plates with X-Gluc, colonies showing homozygous for the knockout or wide-type <em>bgl</em> allele were observed (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1C, D</a>). This result indicated that <em>T</em>. <em>thermophilus</em> cells could form heterozygous state carrying different alleles at one locus, and without selection pressure, the alleles could segregate resulting in different genetic outcomes in the progeny.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/4/1249/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Heterozygous wt/&#x394;-bgl cells and their genetic outcomes. (A) Phenotype of homozygous and heterozygous &#x394;bgl transformants on TB-X-Gluc plate. Black arrows: yellow colonies (homozygous &#x394;bgl); white arrow: colony with intermediate color (heterozygous wt/&#x394;-bgl). (B) Genotypes of the transformants showing intermediate phenotype were determined by PCR (using primers binding the flanking regions of bgl). (C, D) The genetic outcomes of the heterozygous wt/&#x394;-bgl strain. The phenotype (C) and genotype (D) of the progeny of the heterozygous strain after growing without selection and re-streaking on TB-X-Gluc plate. wt-bgl: genotype of the blue colonies from (C), &#x394;-bgl: genotype of the yellow colonies from (C)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-769663358" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Heterozygous wt/&#x394;-&lt;em&gt;bgl&lt;/em&gt; cells and their genetic outcomes. (A) Phenotype of homozygous and heterozygous &#x394;&lt;em&gt;bgl&lt;/em&gt; transformants on TB-X-Gluc plate. Black arrows: yellow colonies (homozygous &#x394;&lt;em&gt;bgl&lt;/em&gt;); white arrow: colony with intermediate color (heterozygous wt/&#x394;-&lt;em&gt;bgl&lt;/em&gt;). (B) Genotypes of the transformants showing intermediate phenotype were determined by PCR (using primers binding the flanking regions of &lt;em&gt;bgl&lt;/em&gt;). (C, D) The genetic outcomes of the heterozygous wt/&#x394;-&lt;em&gt;bgl&lt;/em&gt; strain. The phenotype (C) and genotype (D) of the progeny of the heterozygous strain after growing without selection and re-streaking on TB-X-Gluc plate. wt-&lt;em&gt;bgl:&lt;/em&gt; genotype of the blue colonies from (C), &#x394;-&lt;em&gt;bgl&lt;/em&gt;: genotype of the yellow colonies from (C).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/4/1249/F1.medium.gif" width="440" height="400"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/9/4/1249/F1.medium.gif" width="440" height="400"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/4/1249/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/4/1249/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492707" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-19" class="first-child">Heterozygous wt/Δ-<em>bgl</em> cells and their genetic outcomes. (A) Phenotype of homozygous and heterozygous Δ<em>bgl</em> transformants on TB-X-Gluc plate. Black arrows: yellow colonies (homozygous Δ<em>bgl</em>); white arrow: colony with intermediate color (heterozygous wt/Δ-<em>bgl</em>). (B) Genotypes of the transformants showing intermediate phenotype were determined by PCR (using primers binding the flanking regions of <em>bgl</em>). (C, D) The genetic outcomes of the heterozygous wt/Δ-<em>bgl</em> strain. The phenotype (C) and genotype (D) of the progeny of the heterozygous strain after growing without selection and re-streaking on TB-X-Gluc plate. wt-<em>bgl:</em> genotype of the blue colonies from (C), Δ-<em>bgl</em>: genotype of the yellow colonies from (C).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-13" class="subsection"><h3>Construction of a stable heterozygous strain</h3><p id="p-20">In order to investigate what mechanism had drived the allele segregation phenomenon, a stable heterozygous strain was constructed by inserting two different selection markers at the same chromosomal locus (<em>pyrE</em>). To this end, initially, two homozygous strains (Δ<em>pyrE</em>::<em>kat</em> and Δ<em>pyrE</em>::<em>blm</em>) were created by replacing the <em>pyrE</em> gene with a kanamycin and a bleomycin resistant cassette, respectively (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>). Numerous examples have shown that homozygous gene deletion mutant could be obtained in <em>T. thermophilus</em> in spite of its polyploid genomic background (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Angelov <em>et al.</em> 2013</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Leis <em>et al.</em> 2014</a>; <a id="xref-ref-49-5" class="xref-bibr" href="#ref-49">Li <em>et al.</em> 2015</a>; <a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Ohtani <em>et al.</em> 2015</a>; <a id="xref-ref-80-1" class="xref-bibr" href="#ref-80">Wang <em>et al.</em> 2016</a>). In the experience, when a non-essential gene was targeted by a vector containing an antibiotic resistant marker sandwiched by two homology arms of that target region, near 90% of the resultant transformants were homozygous deletion mutant of that gene. In the generation of Δ<em>pyrE</em>::<em>kat</em> and Δ<em>pyrE</em>::<em>blm</em>, eight transformants were respectively chosen for PCR determination of the deletion mutants, and only one transformant was found to contain the wild-type <em>pyrE</em> allele (Figure S1). The possibility of obtaining homozygous gene deletion mutants with high frequency in the polyploid <em>T. thermophilus</em> genome, again suggested that after the mutant allele has been introduced into the specific genomic region, allele segregation would lead to homozygous mutant cells. Thereafter, the homozygous Δ<em>pyrE</em>::<em>kat</em> strain was transformed with linearized pJ-Δ<em>pyrE</em>::<em>blm</em> plasmid. Selection on TB agar plates supplemented with both antibiotics allowed to generate the heterozygous strain HL01. PCR, Southern blot and DNA sequencing was used to verify the genotype of the heterozygous strain, which excluded the possibility of vector integration and unequivocally showed that in HL01 the <em>pyrE</em> gene was completely replaced by both the <em>kat</em> and <em>blm</em> resistance cassettes and no wild-type <em>pyrE</em> allele was detectable (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2A, B, C</a>). The heterozygous state of the HL01strain could be stably maintained in medium supplemented with both kanamycin and bleomycin. The cell growth curves showed that there were no growth rate difference among the two homozygous strains (Δ<em>pyrE</em>::<em>kat</em> and Δ<em>pyrE</em>::<em>blm</em>) and the heterozygous strain HL01 (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2D</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/4/1249/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genotype confirmation and allele segregation kinetics of the HL01 strain. (A) Schematic drawings of the genotypes of the &#x394;pyrE::blm and &#x394;pyrE::kat strains. XbaI site was found in pyrE, blm and kat respectively. (B) PCR confirmation of the &#x394;pyrE::blm, &#x394;pyrE::kat and HL01 strains using primers F and R (the positions are shown in A). The predicted sizes of the PCR products were 1478 bp for the wild type, 2901 bp for &#x394;pyrE::kat, 1505 bp for &#x394;pyrE::blm, and both 2901 bp and 1505 bp for HL01. (C, E) Apparent genotype and allele abundance measurements of the heterozygous strain HL01 grown in liquid TB medium at 70&#xB0; in the absence of selection. (C) The changes in the relative abundance of the two alleles (kat and blm) were acquired by measuring the intensity of the bands from the Southern blot. For the Southern blot, a 450 bp biotin-labeled fragment of the pyrF gene was used as the probe for hybridization; genomic DNA was prepared from samples taken at the indicated time points and was digested with XbaI. The in silico predicted sizes are 5126 bp and 3440 bp for the &#x394;pyrE::kat and &#x394;pyrE::blm alleles, respectively. One representative of three independent analyses is shown. (E) The viable counts were determined from the antibiotic-free plates (upper panel). The changes in the fraction of each phenotype (KatRBlmR &#x2013; dark-gray bar, KatRBlmS &#x2013; white bar and KatSBlmR &#x2013; light-gray bar) in the population were followed by spreading the samples on antibiotic-free plates and restreaking 50 colonies for each time point on plates containing kanamycin or bleomycin (lower panel). Shown are mean and standard deviation from the three independent experiments. (D) The cell growth curves of the &#x394;pyrE::blm, &#x394;pyrE::kat and HL01 strains." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-769663358" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Genotype confirmation and allele segregation kinetics of the HL01 strain. (A) Schematic drawings of the genotypes of the &#x394;&lt;em&gt;pyrE&lt;/em&gt;::&lt;em&gt;blm and&lt;/em&gt; &#x394;&lt;em&gt;pyrE&lt;/em&gt;::&lt;em&gt;kat&lt;/em&gt; strains. &lt;em&gt;Xba&lt;/em&gt;I site was found in &lt;em&gt;pyrE&lt;/em&gt;, &lt;em&gt;blm&lt;/em&gt; and &lt;em&gt;kat&lt;/em&gt; respectively. (B) PCR confirmation of the &#x394;&lt;em&gt;pyrE&lt;/em&gt;::&lt;em&gt;blm&lt;/em&gt;, &#x394;&lt;em&gt;pyrE&lt;/em&gt;::&lt;em&gt;kat&lt;/em&gt; and HL01 strains using primers F and R (the positions are shown in A). The predicted sizes of the PCR products were 1478 bp for the wild type, 2901 bp for &#x394;&lt;em&gt;pyrE&lt;/em&gt;::&lt;em&gt;kat&lt;/em&gt;, 1505 bp for &#x394;&lt;em&gt;pyrE&lt;/em&gt;::&lt;em&gt;blm&lt;/em&gt;, and both 2901 bp and 1505 bp for HL01. (C, E) Apparent genotype and allele abundance measurements of the heterozygous strain HL01 grown in liquid TB medium at 70&#xB0; in the absence of selection. (C) The changes in the relative abundance of the two alleles (&lt;em&gt;kat&lt;/em&gt; and &lt;em&gt;blm&lt;/em&gt;) were acquired by measuring the intensity of the bands from the Southern blot. For the Southern blot, a 450 bp biotin-labeled fragment of the &lt;em&gt;pyrF&lt;/em&gt; gene was used as the probe for hybridization; genomic DNA was prepared from samples taken at the indicated time points and was digested with &lt;em&gt;Xba&lt;/em&gt;I. The &lt;em&gt;in silico&lt;/em&gt; predicted sizes are 5126 bp and 3440 bp for the &#x394;&lt;em&gt;pyrE&lt;/em&gt;::&lt;em&gt;kat&lt;/em&gt; and &#x394;&lt;em&gt;pyrE&lt;/em&gt;::&lt;em&gt;blm&lt;/em&gt; alleles, respectively. One representative of three independent analyses is shown. (E) The viable counts were determined from the antibiotic-free plates (upper panel). The changes in the fraction of each phenotype (Kat&lt;sup&gt;R&lt;/sup&gt;Blm&lt;sup&gt;R&lt;/sup&gt; &#x2013; dark-gray bar, Kat&lt;sup&gt;R&lt;/sup&gt;Blm&lt;sup&gt;S&lt;/sup&gt; &#x2013; white bar and Kat&lt;sup&gt;S&lt;/sup&gt;Blm&lt;sup&gt;R&lt;/sup&gt; &#x2013; light-gray bar) in the population were followed by spreading the samples on antibiotic-free plates and restreaking 50 colonies for each time point on plates containing kanamycin or bleomycin (lower panel). Shown are mean and standard deviation from the three independent experiments. (D) The cell growth curves of the &#x394;&lt;em&gt;pyrE&lt;/em&gt;::&lt;em&gt;blm&lt;/em&gt;, &#x394;&lt;em&gt;pyrE&lt;/em&gt;::&lt;em&gt;kat&lt;/em&gt; and HL01 strains.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/4/1249/F2.medium.gif" width="440" height="415"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/9/4/1249/F2.medium.gif" width="440" height="415"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/4/1249/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/4/1249/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492745" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-21" class="first-child">Genotype confirmation and allele segregation kinetics of the HL01 strain. (A) Schematic drawings of the genotypes of the Δ<em>pyrE</em>::<em>blm and</em> Δ<em>pyrE</em>::<em>kat</em> strains. <em>Xba</em>I site was found in <em>pyrE</em>, <em>blm</em> and <em>kat</em> respectively. (B) PCR confirmation of the Δ<em>pyrE</em>::<em>blm</em>, Δ<em>pyrE</em>::<em>kat</em> and HL01 strains using primers F and R (the positions are shown in A). The predicted sizes of the PCR products were 1478 bp for the wild type, 2901 bp for Δ<em>pyrE</em>::<em>kat</em>, 1505 bp for Δ<em>pyrE</em>::<em>blm</em>, and both 2901 bp and 1505 bp for HL01. (C, E) Apparent genotype and allele abundance measurements of the heterozygous strain HL01 grown in liquid TB medium at 70° in the absence of selection. (C) The changes in the relative abundance of the two alleles (<em>kat</em> and <em>blm</em>) were acquired by measuring the intensity of the bands from the Southern blot. For the Southern blot, a 450 bp biotin-labeled fragment of the <em>pyrF</em> gene was used as the probe for hybridization; genomic DNA was prepared from samples taken at the indicated time points and was digested with <em>Xba</em>I. The <em>in silico</em> predicted sizes are 5126 bp and 3440 bp for the Δ<em>pyrE</em>::<em>kat</em> and Δ<em>pyrE</em>::<em>blm</em> alleles, respectively. One representative of three independent analyses is shown. (E) The viable counts were determined from the antibiotic-free plates (upper panel). The changes in the fraction of each phenotype (Kat<sup>R</sup>Blm<sup>R</sup> – dark-gray bar, Kat<sup>R</sup>Blm<sup>S</sup> – white bar and Kat<sup>S</sup>Blm<sup>R</sup> – light-gray bar) in the population were followed by spreading the samples on antibiotic-free plates and restreaking 50 colonies for each time point on plates containing kanamycin or bleomycin (lower panel). Shown are mean and standard deviation from the three independent experiments. (D) The cell growth curves of the Δ<em>pyrE</em>::<em>blm</em>, Δ<em>pyrE</em>::<em>kat</em> and HL01 strains.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-14" class="subsection"><h3>Allele segregation kinetics in the heterozygous strain HL01 in the absence of selection</h3><p id="p-22">To detect and quantify allele segregation in the heterozygous strain, HL01 was continuously grown in the absence of selection pressure for 120 h (without dilution or regrowth from stationary phase). Samples were withdrawn at different time points and were used to determine: i) the drug resistance phenotype of 50 individual colonies per time point and ii) the relative frequency of the two alleles in the samples, using Southern blot followed by band intensity measurement. The drug resistance profile implies the genotype of the tested colony (homozygous for either the <em>kat</em> or the <em>blm</em> allele, or heterozygous), while changes in the relative abundance of each allele in the whole population could be detected by Southern blot. In this experiment, it was observed that during growth in the absence of selection pressure the fraction of heterozygous cells (as determined by the resistance phenotype) decreased and that of homozygous cells increased (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2E</a>). The Southern blot analysis showed that, despite the significant changes in the observed genotypes, the relative amount of <em>kat</em>/<em>blm</em> allele in the whole population kept almost constant throughout the experiment (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2C</a>). Apart from random chromosome partitioning, another mechanism, which is gene conversion (<a id="xref-ref-63-4" class="xref-bibr" href="#ref-63">Pecoraro <em>et al.</em> 2011</a>; <a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Lange <em>et al.</em> 2011</a>), could serve to explain the allele segregation phenomenon as well. However, in addition to changing the genotype frequency, gene conversion also leads to changes in the allele abundance in the whole population (see discussion). Since the relative amount of <em>blm</em>/<em>kat</em> allele did not change, gene conversion was ruled out. Taken together, the data from this experiment indicated that random chromosome segregation is the most likely mechanism leading to allele segregation in the heterozygous cells when they were grown in the absence of antibiotic selection pressure.</p></div><div id="sec-15" class="subsection"><h3><strong>Distribution of DNA content in two dividing daughter cells</strong></h3><p id="p-23">In bacterial <em>parAB</em> – <em>parS</em> systems, ParB can bind its <em>cis</em>-acting element <em>parS</em> (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Lin and Grossman 1998</a>). In <em>T. thermophilus</em>, it has been previously shown that the chromosomal <em>parS</em> site is located near the chromosomal <em>oriC</em>, and that ParB-sGFP fusion proteins could bind <em>parS</em> forming fluorescent foci (<a id="xref-ref-49-6" class="xref-bibr" href="#ref-49">Li <em>et al.</em> 2015</a>). The fluorescent foci numbers per cell to some extent represent the copy numbers of <em>parS</em>, <em>i.e.</em>, of the chromosomal <em>oriC</em> region. Based on these observations, and to further confirm that the chromosome partitioning mode in <em>T. thermophilus</em> is non-stingent and often variable, the DNA content in dividing daughter cells was measured based on the signal intensities of DAPI-stained nucleoids, as well as on the numbers of fluorescent foci formed by ParB-sGFP/<em>parS</em> protein-DNA complexes. The two methods have been used also in studying of random chromosome segregation pattern in cyanobacterial species (<a id="xref-ref-65-3" class="xref-bibr" href="#ref-65">Schneider <em>et al.</em> 2007</a>; <a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Hu <em>et al.</em> 2007</a>). Specifically, exponentially growing cells of <em>T. thermophilus</em> HB27 were stained with DAPI (a DNA stain) and 6-carboxyfluorescein (CFS, a membrane stain), and the relative DNA staining intensities of 100 dividing cells (cells in which the nucleoids were separated and the division septum was visible) were measured. <em>B. subtilis</em> 168 was stained and analyzed in the same way and was used as a control in which stringent chromosome partitioning has been shown. Representative images from these measurements are shown in <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>. The fluorescence signals of the two daughter nucleoids in dividing <em>T. thermophilus</em> cells (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3A</a>, top row; Figure S2) revealed unequal intensities of the nucleoids in contrast to <em>B. subtilis</em> cells, in which an almost equal distribution of the DAPI signal in the two daughter cells was observed (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3A</a>, bottom row). Further, it seems that in the <em>T. thermophilus</em> dividing cells with unequal amount of sister nucleoids, septa were also not always formed exactly at mid-cell positions (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3A</a>, top row; Figure S2). <a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3B</a> shows the relative DNA content, determined by DAPI signal intensity, in 100 pairs of daughter nucleoids in dividing cells of <em>T. thermophilus</em> and in <em>B. subtilis</em>. In the <em>B. subtilis</em> cells, the signal ratios distributed from 0.82 to one (mean 0.96, SD 0.04), while in the <em>T. thermophilus</em> cells, the relative ratios had a much broader distribution (from 0.49 to one with a mean of 0.84 and a SD of 0.15). In the polyploid <em>Synechocystis</em> sp., a similar analysis has been performed (<a id="xref-ref-65-4" class="xref-bibr" href="#ref-65">Schneider <em>et al.</em> 2007</a>), in which the relative DNA contents in the <em>B. subtilis</em> dividing cells varied from 0.76 to one, with a mean value of 0.94 and a SD of 0.05, emphasizing the reproducibility of this method. The above analysis suggested that unequal distribution of the genetic material occurred in the dividing <em>T. thermophilus</em> cells. It seems that although chromosome could segregate irregularly, the frequencies of anucleate cells were actually low, only 1.25% cells were found to be anucleate based on the fluorescent microscopic analysis. The phenomenon was also observed in cyanobacterium <em>Anabaena</em> sp. PCC 7120 which has been suggested to exert random chromosome segregation as well (<a id="xref-ref-29-3" class="xref-bibr" href="#ref-29">Hu <em>et al.</em> 2007</a>). This may be due to the fact that the daughter cells of these polyploid bacteria are likely to get at least one chromosome by chance, and subsequent chromosome replication may compensate for variance created during cell division.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/4/1249/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Relative DNA contents in dividing cells of T. thermophilus HB27, HB27/ParB-sGFP, &#x394;mreB::kat/&#x394;parA::blm, and B. subtlis 168. (A) Representative images of exponentially growing and dividing T. thermophilus and B. subtlis cells stained with 6-carboxyfluorescein (membrane) and DAPI (DNA) used in the analysis of the DNA intensity scatter. The white arrows show examples of separated nucleoids in cells where the formations of septa are visible (triangles), bars indicate 2 &#xB5;m. (B) Box plot of the distribution of DAPI intensity in 100 randomly selected dividing T. thermophilus HB27, &#x394;mreB::kat/&#x394;parA::blm and B. subtilis cells. The fluorescence intensity ratio in the two daughter cells was calculated by dividing the intensity value of the cell with lower fluorescence signal by the one with a higher one. The whiskers represent the minimum and the maximum of all the data, circles show outliers. (C) The T. thermophilus chromosomal parAB &#x2013; parS system was used to determine relative DNA contents (i.e., relative numbers of the oriC region) in dividing cells. Representative images show cells with different numbers of ParB-sGFP/parS fluorescent foci in the two daughter nucleoids (white triangles: fluorescent foci, white arrows: sister nucleoids, bar indicates 2 &#xB5;m). (D) Box plot of the distributions of relative ParB-sGFP/parS fluorescent foci numbers and relative DAPI signal intensities in 100 dividing HB27/parB-sGFP cells. The ratio was achieved by dividing the lower value by the higher one." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-769663358" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Relative DNA contents in dividing cells of &lt;em&gt;T. thermophilus&lt;/em&gt; HB27, HB27/ParB-sGFP, &#x394;&lt;em&gt;mreB&lt;/em&gt;::&lt;em&gt;kat/&lt;/em&gt;&#x394;&lt;em&gt;parA&lt;/em&gt;::&lt;em&gt;blm&lt;/em&gt;, and &lt;em&gt;B. subtlis&lt;/em&gt; 168. (A) Representative images of exponentially growing and dividing &lt;em&gt;T. thermophilus&lt;/em&gt; and &lt;em&gt;B. subtlis&lt;/em&gt; cells stained with 6-carboxyfluorescein (membrane) and DAPI (DNA) used in the analysis of the DNA intensity scatter. The white arrows show examples of separated nucleoids in cells where the formations of septa are visible (triangles), bars indicate 2 &#xB5;m. (B) Box plot of the distribution of DAPI intensity in 100 randomly selected dividing &lt;em&gt;T. thermophilus&lt;/em&gt; HB27, &#x394;&lt;em&gt;mreB&lt;/em&gt;::&lt;em&gt;kat/&lt;/em&gt;&#x394;&lt;em&gt;parA&lt;/em&gt;::&lt;em&gt;blm&lt;/em&gt; and &lt;em&gt;B. subtilis&lt;/em&gt; cells. The fluorescence intensity ratio in the two daughter cells was calculated by dividing the intensity value of the cell with lower fluorescence signal by the one with a higher one. The whiskers represent the minimum and the maximum of all the data, circles show outliers. (C) The &lt;em&gt;T. thermophilus&lt;/em&gt; chromosomal &lt;em&gt;parAB&lt;/em&gt; &#x2013; &lt;em&gt;parS&lt;/em&gt; system was used to determine relative DNA contents (&lt;em&gt;i.e.&lt;/em&gt;, relative numbers of the &lt;em&gt;oriC&lt;/em&gt; region) in dividing cells. Representative images show cells with different numbers of ParB-sGFP/&lt;em&gt;parS&lt;/em&gt; fluorescent foci in the two daughter nucleoids (white triangles: fluorescent foci, white arrows: sister nucleoids, bar indicates 2 &#xB5;m). (D) Box plot of the distributions of relative ParB-sGFP/&lt;em&gt;parS&lt;/em&gt; fluorescent foci numbers and relative DAPI signal intensities in 100 dividing HB27/parB-sGFP cells. The ratio was achieved by dividing the lower value by the higher one.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/4/1249/F3.medium.gif" width="440" height="374"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/9/4/1249/F3.medium.gif" width="440" height="374"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/4/1249/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/4/1249/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492721" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-24" class="first-child">Relative DNA contents in dividing cells of <em>T. thermophilus</em> HB27, HB27/ParB-sGFP, Δ<em>mreB</em>::<em>kat/</em>Δ<em>parA</em>::<em>blm</em>, and <em>B. subtlis</em> 168. (A) Representative images of exponentially growing and dividing <em>T. thermophilus</em> and <em>B. subtlis</em> cells stained with 6-carboxyfluorescein (membrane) and DAPI (DNA) used in the analysis of the DNA intensity scatter. The white arrows show examples of separated nucleoids in cells where the formations of septa are visible (triangles), bars indicate 2 µm. (B) Box plot of the distribution of DAPI intensity in 100 randomly selected dividing <em>T. thermophilus</em> HB27, Δ<em>mreB</em>::<em>kat/</em>Δ<em>parA</em>::<em>blm</em> and <em>B. subtilis</em> cells. The fluorescence intensity ratio in the two daughter cells was calculated by dividing the intensity value of the cell with lower fluorescence signal by the one with a higher one. The whiskers represent the minimum and the maximum of all the data, circles show outliers. (C) The <em>T. thermophilus</em> chromosomal <em>parAB</em> – <em>parS</em> system was used to determine relative DNA contents (<em>i.e.</em>, relative numbers of the <em>oriC</em> region) in dividing cells. Representative images show cells with different numbers of ParB-sGFP/<em>parS</em> fluorescent foci in the two daughter nucleoids (white triangles: fluorescent foci, white arrows: sister nucleoids, bar indicates 2 µm). (D) Box plot of the distributions of relative ParB-sGFP/<em>parS</em> fluorescent foci numbers and relative DAPI signal intensities in 100 dividing HB27/parB-sGFP cells. The ratio was achieved by dividing the lower value by the higher one.</p><div class="sb-div caption-clear"></div></div></div><p id="p-25">The above results were further verified by using the <em>T. thermophilus parAB</em> – <em>parS</em> system. For this, a C-terminal sGFP fusion of the chromosomally encoded ParB protein was constructed (<a id="xref-ref-49-7" class="xref-bibr" href="#ref-49">Li <em>et al.</em> 2015</a>), and the ParB-sGFP fusion protein was expressed from the replicative plasmid pMK18 (<a id="xref-ref-13-3" class="xref-bibr" href="#ref-13">de Grado <em>et al.</em> 1999</a>). When expressed in <em>T. thermophilus</em> HB27, well-defined fluorescent foci could be observed (<a id="xref-fig-3-6" class="xref-fig" href="#F3">Figure 3C</a>). The fluorescent foci numbers represent the copy numbers of <em>parS</em> sites, as has been shown previously (<a id="xref-ref-49-8" class="xref-bibr" href="#ref-49">Li <em>et al.</em> 2015</a>). In the example image <a id="xref-fig-3-7" class="xref-fig" href="#F3">Figure 3C</a>, five to 12 fluorescent foci could be observed in the growing cells with dividing nucleoids, indicating five to 12 copies of <em>parS</em> (and of <em>oriC</em> region) existed in one cell, again confirming the polyploidy of <em>T. thermophilus</em> (<a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>). The pairwise distribution of fluorescent foci numbers in 100 pairs of daughter cells was almost identical to the distribution of the relative DAPI signals (<a id="xref-fig-3-8" class="xref-fig" href="#F3">Figure 3D</a>), with a mean of 0.83 and a SD of 0.16. This result further confirmed that random chromosome segregation could occur in the polyploid <em>T. thermophilus</em> cells.</p></div><div id="sec-16" class="subsection"><h3>Inactivation of parA and mreB in T. thermophilus HB27</h3><p id="p-26">The Par system and actin-like protein MreB have been shown to participate in active chromosome partitioning in many bacterial species. By searching out the complete genome of <em>T. thermophilus</em> HB27 (<a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Henne <em>et al.</em> 2004</a>), only one copy of chromosomally encoded ParA and MreB homolog could be identified. And given their important role in other bacteria, their function with respect to chromosome segregation was investigated in this species. The <em>mreB</em> and <em>parA</em> genes were deleted by replacing them with the <em>kat</em> and <em>blm</em> resistance markers respectively (a schematic design see Figure S3A). Initially, homozygous Δ<em>mreB</em>::<em>kat</em> was generated (Figure S3B). The suicide vector pUC-Δ<em>parA</em>::<em>blm</em> was then transformed into the correct Δ<em>mreB</em>::<em>kat</em> strain, and selected on agar plates with both antibiotics (see materials and methods). Seven colonies were randomly selected and exposed to PCR prescreen, this was to determine transformants which contained homozygous Δ<em>parA</em>::<em>blm</em> allele in the Δ<em>mreB</em>::<em>kat</em> background. Six colonies were found to be homozygous Δ<em>mreB</em>::<em>kat</em>/Δ<em>parA</em>::<em>blm</em> mutants (Figure S3C). Southern blot analysis further confirmed that the double deletion mutant had correct insertion of <em>kat</em> and <em>blm</em> at the <em>mreB</em> and <em>parA</em> loci respectively, and no wild-type alleles were left (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). To investigate potential chromosome segregation defects in the mutant, exponentially growing cells were collected, stained by DAPI and CFS and analyzed by fluorescence microscopy. In dividing Δ<em>mreB</em>::<em>kat</em>/Δ<em>parA</em>::<em>blm</em> cells, the replicated nucleoid could segregate normally to the daughter cells (a representative image is shown in <a id="xref-fig-3-9" class="xref-fig" href="#F3">Figure 3A</a>, middle row). Furthermore, compared with the wild type, no considerable increase in the frequency of anucleate or DNA-less cells in Δ<em>mreB</em>::<em>kat</em>/Δ<em>parA</em>::<em>blm</em> was observed (1.25% in the wild type and 1.50% in the mutant, 400 cells were analyzed), suggesting that the deletion of <em>parA</em> and <em>mreB</em> in <em>T. thermophilus</em> has no effect on the chromosomal bulk nucleoid segregation. The relative DNA content in 100 pairs of dividing cells, measured by DAPI signal intensities, were plotted as well. The relative DNA content in the dividing daughter cells of Δ<em>mreB</em>::<em>kat</em>/Δ<em>parA</em>::<em>blm</em> distributed from 0.48 to one, with a mean of 0.85 and a SD of 0.13. This data are almost indistinguishable from that of the wild type strain (<a id="xref-fig-3-10" class="xref-fig" href="#F3">Figure 3B</a>). Taken together, the results suggested that in <em>T. thermophilus</em>, ParA and MreB do not actively participate in the chromosomal bulk DNA segregation process. A similar observation has been reported for the cyanobacterium <em>Anabaena</em> sp. PCC 7120, which is also polyploid (<a id="xref-ref-29-4" class="xref-bibr" href="#ref-29">Hu <em>et al.</em> 2007</a>), whereas in haploid bacteria such as <em>B. subtilis</em> and <em>C. cresentus</em>, ParA and MreB have been shown to be involved in chromosomal bulk DNA and origin segregations (Ireton <em>et al.</em> 1994; <a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">Lewis and Errington 1997</a>; <a id="xref-ref-55-2" class="xref-bibr" href="#ref-55">Mohl and Gober 1997</a>; <a id="xref-ref-74-2" class="xref-bibr" href="#ref-74">Soufo and Graumann 2003</a>; <a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Gitai <em>et al.</em> 2004</a>, <a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">2005</a>; <a id="xref-ref-69-2" class="xref-bibr" href="#ref-69">Shebelut <em>et al.</em> 2009</a>, <a id="xref-ref-68-2" class="xref-bibr" href="#ref-68">2010</a>; <a id="xref-ref-66-2" class="xref-bibr" href="#ref-66">Scholefield <em>et al.</em> 2011</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/4/1249/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Generation of the &#x394;mreB::kat/&#x394;parA::blm double deletion mutant in T. thermophilus. The parA and mreB genes were replaced by blm and kat cassette respectively, genotype confirmation of &#x394;mreB::kat/&#x394;parA::blm was performed by Southern blot, using probes binding the mreB and parA regions for hybridization respectively. The genomic DNAs were digested with BamHI for confirmation of parA deletion, and were digested with BglII for confirmation of mreB deletion. The in silico predicted sizes are respectively indicated with arrows on the left side of each image." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-769663358" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Generation of the &#x394;&lt;em&gt;mreB&lt;/em&gt;::&lt;em&gt;kat/&lt;/em&gt;&#x394;&lt;em&gt;parA&lt;/em&gt;::&lt;em&gt;blm&lt;/em&gt; double deletion mutant in &lt;em&gt;T. thermophilus&lt;/em&gt;. The &lt;em&gt;parA&lt;/em&gt; and &lt;em&gt;mreB&lt;/em&gt; genes were replaced by &lt;em&gt;blm&lt;/em&gt; and &lt;em&gt;kat&lt;/em&gt; cassette respectively, genotype confirmation of &#x394;&lt;em&gt;mreB&lt;/em&gt;::&lt;em&gt;kat/&lt;/em&gt;&#x394;&lt;em&gt;parA&lt;/em&gt;::&lt;em&gt;blm&lt;/em&gt; was performed by Southern blot, using probes binding the &lt;em&gt;mreB&lt;/em&gt; and &lt;em&gt;parA&lt;/em&gt; regions for hybridization respectively. The genomic DNAs were digested with &lt;em&gt;Bam&lt;/em&gt;HI for confirmation of &lt;em&gt;parA&lt;/em&gt; deletion, and were digested with &lt;em&gt;Bgl&lt;/em&gt;II for confirmation of &lt;em&gt;mreB&lt;/em&gt; deletion. The &lt;em&gt;in silico&lt;/em&gt; predicted sizes are respectively indicated with arrows on the left side of each image.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/4/1249/F4.medium.gif" width="440" height="192"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/9/4/1249/F4.medium.gif" width="440" height="192"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/4/1249/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/4/1249/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492786" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-27" class="first-child">Generation of the Δ<em>mreB</em>::<em>kat/</em>Δ<em>parA</em>::<em>blm</em> double deletion mutant in <em>T. thermophilus</em>. The <em>parA</em> and <em>mreB</em> genes were replaced by <em>blm</em> and <em>kat</em> cassette respectively, genotype confirmation of Δ<em>mreB</em>::<em>kat/</em>Δ<em>parA</em>::<em>blm</em> was performed by Southern blot, using probes binding the <em>mreB</em> and <em>parA</em> regions for hybridization respectively. The genomic DNAs were digested with <em>Bam</em>HI for confirmation of <em>parA</em> deletion, and were digested with <em>Bgl</em>II for confirmation of <em>mreB</em> deletion. The <em>in silico</em> predicted sizes are respectively indicated with arrows on the left side of each image.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-17"><h2 class="">Discussion</h2><div id="sec-18" class="subsection"><h3>Allele segregation in heterozygous T. thermophilus cells</h3><p id="p-28">This work showed that the <em>T. thermophilus</em> strain HB27 contains on average seven to eight chromosomal copies per cell when grown in rich medium. This number was estimated using two independent methods, <em>e.g.</em>, a real time PCR-based method and a fluorescence microscopy method, based on counting the numbers of fluorescent ParB-sGFP/<em>parS</em> foci per cell. The estimation of the chromosomal copy number is in good agreement with the one reported in another <em>T. thermophilus</em> strain, HB8 (<a id="xref-ref-60-4" class="xref-bibr" href="#ref-60">Ohtani <em>et al.</em> 2010</a>). One of several possible explanations for polyploidy is that this strategy favors the survival of bacteria living under harsh conditions. For example, due to multiple genome copies, <em>D. radiodurans</em> is able to rapidly reassemble the chromosomes destroyed by desiccation and ionizing radiation (<a id="xref-ref-84-1" class="xref-bibr" href="#ref-84">Zahradka <em>et al.</em> 2006</a>; <a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Slade <em>et al.</em> 2009</a>). <em>T</em>. <em>thermophilus</em> which grows at temperatures up to 85° may also suffer frequent DNA damage, and polyploidy may facilitate DNA repair by homologous recombination.</p><p id="p-29">An interesting phenomenon observed repeatedly in <em>T. thermophilus</em> is the allele segregation in heterozygous cells containing two different alleles in one locus (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1</a>). It was considered that this could be the result of either random chromosome segregation or gene conversion. While both processes can result in homogeneity in the progeny, their intrinsic mechanisms are different. Gene conversion is essentially a non-reciprocal exchange of information between homologous chromosomes as well as between repeated sequences on the same chromosome (<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Paulsson <em>et al.</em> 2017</a>). In principle, for a heterozygous cell containing both mutant and wild-type allele at one locus, gene conversion can conditionally occur in both directions, <em>e.g.</em>, if the mutant allele is converted against the wild-type allele, the abundance (copies) of the mutant allele will be eliminated and that of the wild-type allele will be increased, and the reverse is also true (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Khakhlova and Bock 2006</a>; <a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">Soppa 2011</a>). In polyploid archaea, allele equalization among the multiple genome copies via gene conversion has been repeatedly demonstrated (<a id="xref-ref-28-3" class="xref-bibr" href="#ref-28">Hildenbrand <em>et al.</em> 2011</a>; <a id="xref-ref-44-3" class="xref-bibr" href="#ref-44">Lange <em>et al.</em> 2011</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Guschinskaya <em>et al.</em> 2016</a>). For instance, <a id="xref-ref-44-4" class="xref-bibr" href="#ref-44">Lange <em>et al.</em> (2011)</a> showed that in <em>Haloferax volcanii</em>, the heterozygous strain <em>leuB</em>/Δ<em>leuB</em>::<em>trpA</em> would convert to homozygous <em>leuB</em> or Δ<em>leuB</em>::<em>trpA</em> strain. In the presence of tryptophan, Δ<em>leuB</em>::<em>trpA</em> allele was converted to <em>leuB</em> allele, resulting in the increase of the fraction of homozygous <em>leuB</em> strain, in the meanwhile, the abundance of the Δ<em>leuB</em>::<em>trpA</em> allele was eliminated and that of the <em>leuB</em> allele was accordingly increased in the whole population; in the presence of leucine, the conversion was occurred in the opposite direction. On the other hand, if random chromosome segregation occurred in polyploid cells carrying two different alleles at one gene locus, daughter cells tended to receive different combination and/or numbers of the replicated parental chromosomes, which would also lead to change in the genotype frequency of the progeny, however, the relative allele abundance would be always maintained. To be able to discriminate which mechanism had led to the observed allele segregation in the heterozygous <em>T. thermophilus</em> cells, a stable heterozygous strain (HL01) containing two different markers (<em>kat</em> and <em>blm</em>) at the <em>pyrE</em> locus was generated (<a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2B, C</a>). When this strain was grown in the absence of antibiotic selection, a gradual segregation of the two markers was observed (<a id="xref-fig-2-7" class="xref-fig" href="#F2">Figure 2E</a>). In this experiment the fraction of homozygous cells increased significantly, while at the same time the relative abundance of <em>kat</em> and <em>blm</em> in the population remained constant (<a id="xref-fig-2-8" class="xref-fig" href="#F2">Figure 2C, E</a>). This indicates that the allele segregation was probably triggered by chromosome random segregation but not by gene conversion (see above). Interestingly, during the allele segregation kinetics, the fraction of the heterozygous strain decreased more steady in the stationary phase than in the exponential phase (<em>e.g.</em>, the fraction was increased at the time point 6 h and 8h compared with 4 h). This may be explained by the high frequency of natural transformation in the exponential growth phase of <em>T. thermophilus</em> HB27. Although cell divisions proceeded faster in the exponential phase, DNA exchange among the cells (due to the uptake by natural transformation of free DNA released by dead cells) might also occur (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">César <em>et al.</em> 2011</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Alvarez <em>et al.</em> 2011</a>). Such an exchange would counteract allele segregation and would affect the frequency of the apparent genotypes of the two alleles. Indeed, addition of EDTA, which is known to impede DNA uptake, to the allele segregation reactions resulted in a more rapid segregation of the two alleles in the exponential phase (2 h - 8 h) (Figure S4). Another mechanism contributing to the steady allele segregation in the late stationary phase may be reductive cell division (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Kolter <em>et al.</em> 1993</a>; <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Navarro Llorens <em>et al.</em> 2010</a>). Such reductive cell division has been shown to cause decrease in the average genome copy numbers in many polyploid bacteria and archaea, as shown by the extreme example, the copy numbers of cyanobacterium <em>Synechocystis</em> PCC 6803 could decrease from 218 copies in exponential growth phase to 58 copies in stationary growth phase (<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Griese <em>et al.</em> 2011</a>). In <em>T. thermophilus</em>, the average chromosome copy number per cell was four to five under slow growth condition (<a id="xref-ref-60-5" class="xref-bibr" href="#ref-60">Ohtani <em>et al.</em> 2010</a>), while that was seven to eight in fast-growing culture, indicating that the value is also conditionally regulated. Therefore, it is conceivable that in this experiment, the decrease of average chromosome copies in the stationarily-growing cells would actually favor the daughter cells receive the same type of parental chromosome by random partitioning.</p></div><div id="sec-19" class="subsection"><h3>Chromosome random partitioning in polyploid bacterial cells</h3><p id="p-30">The complete segregation of bacterial chromosomes undergoes three separated steps: separation of the replicated origin regions, segregation of the replicated bulk chromosome, and finally resolution of the termini at the division septum (<a id="xref-ref-81-1" class="xref-bibr" href="#ref-81">Wang and Rudner 2014</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Badrinarayanan <em>et al.</em> 2015</a>). Nevertheless, the molecular mechanisms involved in the chromosome segregation in non-model bacteria are only just beginning to be elucidated. Most sequenced bacterial genomes (with the exception of <em>E. coli</em>) contain a chromosomally encoded <em>par</em> locus, which is consisted of three components: two <em>trans</em>-acting proteins (ParA and ParB) and a <em>cis</em>-acting site (<em>parS</em>). The ParAB – <em>parS</em> system is believed to drive active chromosome segregation in many bacteria, including origin and/or bulk DNA segregations. Deletion of <em>par</em> genes is lethal in some bacteria, <em>e.g.</em>, in <em>C. crescentus</em> (<a id="xref-ref-55-3" class="xref-bibr" href="#ref-55">Mohl and Gober 1997</a>; <a id="xref-ref-56-2" class="xref-bibr" href="#ref-56">Mohl <em>et al.</em> 2001</a>), <em>V. cholera</em> (<em>parAB2</em>) (<a id="xref-ref-83-2" class="xref-bibr" href="#ref-83">Yamaichi <em>et al.</em> 2007</a>) and <em>Myxococcus xanthus</em> (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Iniesta 2014</a>). In <em>C. crescentus</em>, the ParA retraction has been shown to mediate the latter part of ParB translocation, thus is crucial for the final stages of chromosome segregation (<a id="xref-ref-68-3" class="xref-bibr" href="#ref-68">Shebelut <em>et al.</em> 2010</a>). Deletion of the <em>parAB2</em> genes located on chromosome II of <em>V. cholera</em> could lead to localization and segregation defects of the chromosome II, and yield cells with only chromosome I (<a id="xref-ref-83-3" class="xref-bibr" href="#ref-83">Yamaichi <em>et al.</em> 2007</a>). In some other species, <em>par</em> mutants are viable but exhibit various severity defects of chromosome bulk DNA or origin segregation in the vegetative and/or sporulation phase of growth, <em>e.g.</em>, in <em>B. subtilis</em>, <em>S. coelicolor</em>, <em>V. cholera</em> (<em>parAB1</em> of chromosome I), <em>P. aeruginosa</em>, <em>C. glutamicum</em>, <em>S. pneumoniae</em>, and <em>M. smegmatis</em> (Ireton <em>et al.</em> 1994; Sharpe and Errington 1996; <a id="xref-ref-48-3" class="xref-bibr" href="#ref-48">Lewis and Errington 1997</a>; <a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Kim <em>et al.</em> 2000</a>; <a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Jakimowicz <em>et al.</em> 2002</a>; <a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Fogel and Waldor 2006</a>; <a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">Lasocki <em>et al.</em> 2007</a>; <a id="xref-ref-57-2" class="xref-bibr" href="#ref-57">Murray and Errington 2008</a>; <a id="xref-ref-76-2" class="xref-bibr" href="#ref-76">Sullivan <em>et al.</em> 2009</a>; <a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Gruber and Errington 2009</a>; <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Donovan <em>et al.</em> 2010</a>; <a id="xref-ref-66-3" class="xref-bibr" href="#ref-66">Scholefield <em>et al.</em> 2011</a>; <a id="xref-ref-54-2" class="xref-bibr" href="#ref-54">Minnen <em>et al.</em> 2011</a>; <a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Ginda <em>et al.</em> 2013</a>). The bacterial actin-like protein MreB has also been suggested to be involved in chromosome segregation in addition to its role in cell morphology determination (<a id="xref-ref-77-2" class="xref-bibr" href="#ref-77">Sundararajan and Goley 2017</a>). In <em>B. subtilis</em>, <a id="xref-ref-74-3" class="xref-bibr" href="#ref-74">Soufo and Graumann (2003)</a> showed that depletion of <em>mreB</em> led to the appearance of 25% anucleate cells, and depletion of the other two <em>mreB</em> homologs (<em>mbl</em> or <em>mreBH</em>) also caused a considerable increase in the fraction of anucleate cells. In <em>E. coli</em>, the initial studies truly showed that MreB is necessary for origin and bulk nucleoid segregation; when the <em>E. coli</em> cells were treated with A22 which is a new antibiotic specifically targeting MreB, chromosome segregation was inhibited, resulting in cells containing large confluent bodies of nucleoids (<a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Kruse <em>et al.</em> 2003</a>, <a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">2006</a>). In <em>C. cresentus</em>, alterations in MreB expression also cause defects in chromosome segregation (<a id="xref-ref-21-3" class="xref-bibr" href="#ref-21">Gitai <em>et al.</em> 2004</a>, <a id="xref-ref-20-3" class="xref-bibr" href="#ref-20">2005</a>). The A22 treated <em>C. cresentus</em> cells experienced growth condition-specific defects in segregation of the chromosomal origin-proximal regions (<a id="xref-ref-20-4" class="xref-bibr" href="#ref-20">Gitai <em>et al.</em> 2005</a>; <a id="xref-ref-69-3" class="xref-bibr" href="#ref-69">Shebelut <em>et al.</em> 2009</a>). In this work, the <em>T. thermophilus parA</em> and <em>mreB</em> gene were simultaneously deleted by replacement with antibiotic-resistant genes. The results showed that the <em>parA</em> and <em>mreB</em> deletion mutant had a similar chromosome partitioning pattern as the wild type (<a id="xref-fig-3-11" class="xref-fig" href="#F3">Figure 3A, B</a>), indicating no apparent chromosome segregation defects occurred in the Δ<em>mreB</em>Δ<em>parA</em> muant. Thus, both the Par and MreB systems are not involved in the chromosomal bulk DNA segregation in <em>T. thermophilus</em>. A similar result was found in the polyploid cyanobacterium <em>Anabaena</em> sp. PCC 7120, in which the lack of <em>mreB</em> did not lead to any DNA segregation defects in the two dividing daughter cells (<a id="xref-ref-29-5" class="xref-bibr" href="#ref-29">Hu <em>et al.</em> 2007</a>). Still, it is of interest what the function is of the <em>parABS</em> and <em>mreB</em> loci in <em>T. thermophilus</em> when they are not critical for cell growth and chromosomal DNA segregation. The <em>parABS</em> loci are located extremely approching <em>oriC</em> (appoximately 6 kb distance) (<a id="xref-ref-49-9" class="xref-bibr" href="#ref-49">Li <em>et al.</em> 2015</a>), and ParB could specifically bind <em>parS</em> forming nucleoprotein complexes spreading around the origin region, indicating that <em>parABS</em> are actually functional. Thus, <em>parABS</em> may contribute to other cell functions, such as serve to be transcriptional regulators. Recent studies have shown that analogous to their counterparts on the plasmids, the chromosomal Par proteins can also regulate transcriptions of other genes located around <em>parS</em> sites (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Bartosik <em>et al.</em> 2014</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Baek <em>et al.</em> 2014</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Attaiech <em>et al.</em> 2015</a>). For example, in <em>P. aeruginos</em>a, deletion of <em>parB</em> leads to global transcriptional changes affecting more than 1000 genes (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Bartosik <em>et al.</em> 2014</a>). Thus, it will prove interesting to test whether this is the situation for the <em>T. thermophilus</em> Par system in the future. Further, although MreB did not participate in the chromosome segregation, it was found to be a cell shape determinant in <em>T. thermophilus</em>. The cells of the Δ<em>mreB</em>::<em>kat</em>/Δ<em>parA</em>::<em>blm</em> double gene deletion mutant and Δ<em>mreB</em>::<em>kat</em> single gene deletion mutant have become much thicker and rounder compared with those of the wild-type strain (<a id="xref-fig-3-12" class="xref-fig" href="#F3">Figure 3A</a>, middle row). The average ratio of cell length/diameter was 4.24 ± 0.21 in Δ<em>mreB</em>::<em>kat</em>, while that in the wild type was 8.71 ± 0.86, in addition, a certain number of the mutant cells have become drastically abnormal (not shown).</p><p id="p-31">Taken together, it seems that the specific protein-based chromosome segregation machineries, <em>e.g.</em>, ParAB – <em>parS</em> system and/or MreB are critical for certain bacteria containing a single chromosome during most of the cell cycles, as they ensure that the daughter cells receive an equal number of the parental chromosome during separation of sister chromosomes. Alternatively, bacteria normally containing multiple chromosomes per cell may not require an active segregation mechanism, since the frequency of daughter cells receiving none of the parental chromosomes is actually low (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Graumann 2010</a>). This is analogous to high-copy-number plasmid systems, which typically lack an active segregation mechanism (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Ebersbach and Gerdes 2005</a>).</p><p id="p-32">An intriguing question is how the polyploid bacteria segregate their chromosome copies to the daughter cells. The allele segregation kinetics experiment in the heterozygous <em>T. thermophilus</em> strain HL01 suggested that the chromosome segregation mode in <em>T. thermophilus</em> is variable, and random segregation could occur. This suggestion was strengthened by monitoring the relative DNA content in dividing cells via DAPI-staining and by <em>in vivo</em> monitoring of ParB-sGFP/<em>parS</em> protein-DNA complexes (<a id="xref-fig-3-13" class="xref-fig" href="#F3">Figure 3</a>). Similar conclusions were drawn for the polyploid cyanobacteria <em>Anabaena</em> sp. PCC 7120 (<a id="xref-ref-29-6" class="xref-bibr" href="#ref-29">Hu <em>et al.</em> 2007</a>) and <em>Synechocystis</em> sp. (<a id="xref-ref-65-5" class="xref-bibr" href="#ref-65">Schneider <em>et al.</em> 2007</a>), and the polyploid euryarchaeota <em>M. jannaschii</em> (<a id="xref-ref-52-2" class="xref-bibr" href="#ref-52">Malandrin <em>et al.</em> 1999</a>). In these prokaryotes, it would be possible that nonprotein-based chromosome segregation mechanisms, <em>e.g.</em>, physical forces from extrusion of DNA from replication forks may help push it toward opposite poles (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Jun and Mulder 2006</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Jun and Wright 2010</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Fisher <em>et al.</em> 2013</a>; <a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Badrinarayanan <em>et al.</em> 2015</a>), thereby random chromosome segregation occur frequently. The observation that septa were not always formed exactly at the middle position of <em>T. thermophilus</em> cells could actually support the proposal. Further, it is worthy to note that the relaxation of DNA segregation control may not be a general trait for all polyploid bacteria or archaea. It has been shown that chromosomes were segregated precisely in the polyploid archaeon, <em>Halobacterium salinarum</em> (<a id="xref-ref-8-4" class="xref-bibr" href="#ref-8">Breuert <em>et al.</em> 2006</a>) as well as in the polyploid cyanobacterium, <em>Synechococcus elongates</em> (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Jain <em>et al.</em> 2012</a>).</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-33">The plasmids pCT3FK and pMK18 were kindly provided by Prof. Wolfgang Liebl and Dr. Angel Angelov from Technical University of Munich. This work was supported by the National Natural Science Foundation of China (grant number 31700059), Natural Science Foundation of Shannxi province (grant number 2018JQ3037) and Young Talent Fund of University Association for Science and Technology in Shaanxi, China (grant number 20180210).</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-5"><p id="p-34">Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.7746878">https://doi.org/10.25387/g3.7746878</a>.</p></li><li class="fn" id="fn-6"><p id="p-35"><em>Communicating editor: S. Jaspersen</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2018-12-18"><span class="received-label">Received </span>December 18, 2018.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2019-02-15"><span class="accepted-label">Accepted </span>February 15, 2019.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2019 Li</li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1249.1" data-doi="10.1128/AEM.02048-10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Alvarez</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bricio</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gómez</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berenguer</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Lateral transfer of the denitrification pathway genes among <em>Thermus thermophilus</em> strains.</span> <abbr class="cit-jnl-abbrev">Appl. Environ. Microbiol.</abbr> <span class="cit-vol">77</span>: <span class="cit-fpage">1352</span>–<span class="cit-lpage">1358</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1128/AEM.02048-10</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAppl.%2BEnviron.%2BMicrobiol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FAEM.02048-10%26rft_id%253Dinfo%253Apmid%252F21169443%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiYWVtIjtzOjU6InJlc2lkIjtzOjk6Ijc3LzQvMTM1MiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzkvNC8xMjQ5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1249.2" data-doi="10.1016/j.syapm.2013.06.001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Angelov</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Geissler</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leis</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liebl</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Toxicity of indoxyl derivative accumulation in bacteria and its use as a new counterselection principle.</span> <abbr class="cit-jnl-abbrev">Syst. Appl. Microbiol.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">585</span>–<span class="cit-lpage">592</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.syapm.2013.06.001</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSyst.%2BAppl.%2BMicrobiol.%26rft.volume%253D36%26rft.spage%253D585%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.syapm.2013.06.001%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.syapm.2013.06.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1249.3" data-doi="10.1016/j.syapm.2008.01.003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Angelov</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mientus</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liebl</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liebl</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A two-host fosmid system for functional screening of (meta)genomic libraries from extreme thermophiles.</span> <abbr class="cit-jnl-abbrev">Syst. Appl. Microbiol.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">177</span>–<span class="cit-lpage">185</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.syapm.2008.01.003</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSystematic%2Band%2BApplied%2BMicrobiology%26rft.stitle%253DSystematic%2Band%2BApplied%2BMicrobiology%26rft.aulast%253DAngelov%26rft.auinit1%253DA.%26rft.volume%253D32%26rft.issue%253D3%26rft.spage%253D177%26rft.epage%253D185%26rft.atitle%253DA%2Btwo-host%2Bfosmid%2Bsystem%2Bfor%2Bfunctional%2Bscreening%2Bof%2B%2528meta%2529genomic%2Blibraries%2Bfrom%2Bextreme%2Bthermophiles.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.syapm.2008.01.003%26rft_id%253Dinfo%253Apmid%252F19285378%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.syapm.2008.01.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19285378&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1249.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1249.4" data-doi="10.1128/mBio.00662-15"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Attaiech</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Minnen</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kjos</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gruber</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Veening</span>  <span class="cit-name-given-names">J. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">The ParB-<em>parS</em> chromosome segregation system modulates competence development in <em>Streptococcus pneumoniae</em>.</span> <abbr class="cit-jnl-abbrev">MBio</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e00662</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1128/mBio.00662-15</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMBio%26rft.volume%253D6%26rft.spage%253De00662%26rft_id%253Dinfo%253Adoi%252F10.1128%252FmBio.00662-15%26rft_id%253Dinfo%253Apmid%252F26126852%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1128/mBio.00662-15&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26126852&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1249.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1249.5" data-doi="10.1146/annurev-cellbio-100814-125211"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Badrinarayanan</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Le</span>  <span class="cit-name-given-names">T. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Laub</span>  <span class="cit-name-given-names">M. T.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Bacterial chromosome organization and segregation.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Cell Dev. Biol.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">171</span>–<span class="cit-lpage">199</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1146/annurev-cellbio-100814-125211</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnu.%2BRev.%2BCell%2BDev.%2BBiol.%26rft.volume%253D31%26rft.spage%253D171%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-cellbio-100814-125211%26rft_id%253Dinfo%253Apmid%252F26566111%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev-cellbio-100814-125211&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26566111&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1249.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1249.6" data-doi="10.1128/mBio.01061-14"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Baek</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rajagopala</span>  <span class="cit-name-given-names">S. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chattoraj</span>  <span class="cit-name-given-names">D. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Chromosome segregation proteins of <em>Vibrio cholerae</em> as transcription regulators.</span> <abbr class="cit-jnl-abbrev">MBio</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e01061-14</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1128/mBio.01061-14</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMBio%26rft_id%253Dinfo%253Adoi%252F10.1128%252FmBio.01061-14%26rft_id%253Dinfo%253Apmid%252F24803519%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoibWJpbyI7czo1OiJyZXNpZCI7czoxMzoiNS8zL2UwMTA2MS0xNCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzkvNC8xMjQ5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1249.7" data-doi="10.1371/journal.pone.0087276"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bartosik</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glabski</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jecz</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mikulska</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fogtman</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Transcriptional profiling of ParA and ParB mutants in actively dividing cells of an opportunistic human pathogen <em>Pseudomonas aeruginosa</em>.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e87276</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0087276</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De87276%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0087276%26rft_id%253Dinfo%253Apmid%252F24498062%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0087276&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24498062&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1249.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1249.8" data-doi="10.1371/journal.pone.0000092"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Breuert</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Allers</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spohn</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Soppa</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Regulated polyploidy in halophilic archaea.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">e92</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0000092</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DBreuert%26rft.auinit1%253DS.%26rft.volume%253D1%26rft.spage%253De92%26rft.epage%253De92%26rft.atitle%253DRegulated%2Bpolyploidy%2Bin%2Bhalophilic%2Barchaea.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000092%26rft_id%253Dinfo%253Apmid%252F17183724%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0000092&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17183724&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1249.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1249.9" data-doi="10.1074/jbc.M413623200"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brouns</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Akerboom</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Turnbull</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Vos</span>  <span class="cit-name-given-names">W. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Engineering a selectable marker for hyperthermophiles.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">280</span>: <span class="cit-fpage">11422</span>–<span class="cit-lpage">11431</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1074/jbc.M413623200</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M413623200%26rft_id%253Dinfo%253Apmid%252F15640151%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyODAvMTIvMTE0MjIiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzQvMTI0OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1249.10" data-doi="10.1016/j.bpj.2016.04.046"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cass</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuwada</span>  <span class="cit-name-given-names">N. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Traxler</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wiggins</span>  <span class="cit-name-given-names">P. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title"><em>Escherichia coli</em> chromosomal loci segregate from midcell with universal dynamics.</span> <abbr class="cit-jnl-abbrev">Biophys. J.</abbr> <span class="cit-vol">110</span>: <span class="cit-fpage">2597</span>–<span class="cit-lpage">2609</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.bpj.2016.04.046</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiophys.%2BJ.%26rft.volume%253D110%26rft.spage%253D2597%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.bpj.2016.04.046%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.bpj.2016.04.046&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1249.11" data-doi="10.2436/20.1501.01.148"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">César</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Álvarez</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bricio</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Heerden</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Littauer</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Unconventional lateral gene transfer in extreme thermophilic bacteria.</span> <abbr class="cit-jnl-abbrev">Int. Microbiol.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">187</span>–<span class="cit-lpage">199</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInternational%2BMicrobiology%26rft.stitle%253DInternational%2BMicrobiology%26rft.aulast%253DCesar%26rft.auinit1%253DC.%2BE.%26rft.volume%253D14%26rft.issue%253D4%26rft.spage%253D187%26rft.epage%253D199%26rft.atitle%253DUnconventional%2Blateral%2Bgene%2Btransfer%2Bin%2Bextreme%2Bthermophilic%2Bbacteria.%26rft_id%253Dinfo%253Adoi%252F10.2436%252F20.1501.01.148%26rft_id%253Dinfo%253Apmid%252F22569756%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2436/20.1501.01.148&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22569756&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1249.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1249.12" data-doi="10.1038/nrg1711"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Comai</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">The advantages and disadvantages of being polyploid.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">836</span>–<span class="cit-lpage">846</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg1711</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DComai%26rft.auinit1%253DL.%26rft.volume%253D6%26rft.issue%253D11%26rft.spage%253D836%26rft.epage%253D846%26rft.atitle%253DThe%2Badvantages%2Band%2Bdisadvantages%2Bof%2Bbeing%2Bpolyploid.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg1711%26rft_id%253Dinfo%253Apmid%252F16304599%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg1711&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16304599&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1249.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000232990900014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1249.13" data-doi="10.1006/plas.1999.1427"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de Grado</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Castan</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berenguer</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">A high-transformation-efficiency cloning vector for <em>Thermus thermophilus</em>.</span> <abbr class="cit-jnl-abbrev">Plasmid</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">241</span>–<span class="cit-lpage">245</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1006/plas.1999.1427</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlasmid%26rft.stitle%253DPlasmid%26rft.aulast%253Dde%2BGrado%26rft.auinit1%253DM.%26rft.volume%253D42%26rft.issue%253D3%26rft.spage%253D241%26rft.epage%253D245%26rft.atitle%253DA%2Bhigh-transformation-efficiency%2Bcloning%2Bvector%2Bfor%2BThermus%2Bthermophilus.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fplas.1999.1427%26rft_id%253Dinfo%253Apmid%252F10545266%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/plas.1999.1427&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10545266&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1249.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000083651500008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1249.14" data-doi="10.1128/JB.00214-10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Donovan</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schwaiger</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krämer</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bramkamp</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Subcellular localization and characterization of the ParAB system from <em>Corynebacterium glutamicum</em>.</span> <abbr class="cit-jnl-abbrev">J. Bacteriol.</abbr> <span class="cit-vol">192</span>: <span class="cit-fpage">3441</span>–<span class="cit-lpage">3451</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1128/JB.00214-10</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBacteriol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJB.00214-10%26rft_id%253Dinfo%253Apmid%252F20435732%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6MTE6IjE5Mi8xMy8zNDQxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS80LzEyNDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1249.15" data-doi="10.1146/annurev.genet.38.072902.091252"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ebersbach</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gerdes</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Plasmid segregation mechanisms.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Genet.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">453</span>–<span class="cit-lpage">479</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1146/annurev.genet.38.072902.091252</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bgenetics%26rft.stitle%253DAnnu%2BRev%2BGenet%26rft.volume%253D39%26rft.spage%253D453%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.genet.38.072902.091252%26rft_id%253Dinfo%253Apmid%252F16285868%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.genet.38.072902.091252&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16285868&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1249.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000234685200020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1249.16" data-doi="10.1016/j.cell.2013.04.006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fisher</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bourniquel</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Witz</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weiner</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Prentiss</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Four-dimensional imaging of <em>E. coli</em> nucleoid organization and dynamics in living cells.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">153</span>: <span class="cit-fpage">882</span>–<span class="cit-lpage">895</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.cell.2013.04.006</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D153%26rft.spage%253D882%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2013.04.006%26rft_id%253Dinfo%253Apmid%252F23623305%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2013.04.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23623305&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1249.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000318844000016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1249.17" data-doi="10.1101/gad.1496506"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fogel</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Waldor</span>  <span class="cit-name-given-names">M. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">A dynamic, mitotic-like mechanism for bacterial chromosome segregation.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">3269</span>–<span class="cit-lpage">3282</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gad.1496506</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DFogel%26rft.auinit1%253DM.%2BA.%26rft.volume%253D20%26rft.issue%253D23%26rft.spage%253D3269%26rft.epage%253D3282%26rft.atitle%253DA%2Bdynamic%252C%2Bmitotic-like%2Bmechanism%2Bfor%2Bbacterial%2Bchromosome%2Bsegregation%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.1496506%26rft_id%253Dinfo%253Apmid%252F17158745%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6MTA6IjIwLzIzLzMyNjkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzQvMTI0OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1249.18" data-doi="10.1038/nmeth.1318"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gibson</span>  <span class="cit-name-given-names">D. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Young</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chuang</span>  <span class="cit-name-given-names">R. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Venter</span>  <span class="cit-name-given-names">J. C.</span>  <span class="cit-name-suffix">3rd</span>.</span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hutchison</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Enzymatic assembly of DNA molecules up to several hundred kilobases.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">343</span>–<span class="cit-lpage">345</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nmeth.1318</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DGibson%26rft.auinit1%253DD.%2BG.%26rft.volume%253D6%26rft.issue%253D5%26rft.spage%253D343%26rft.epage%253D345%26rft.atitle%253DEnzymatic%2Bassembly%2Bof%2BDNA%2Bmolecules%2Bup%2Bto%2Bseveral%2Bhundred%2Bkilobases.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1318%26rft_id%253Dinfo%253Apmid%252F19363495%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1318&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19363495&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1249.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000265661600012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1249.19" data-doi="10.1111/mmi.12146"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ginda</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bezulska</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zió-lkiewicz</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dziadek</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zakrzewska-Czerwińska</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">ParA of <em>Mycobacterium smegmatis</em> co-ordinates chromosome segregation with the cell cycle and interacts with the polar growth determinant DivIVA.</span> <abbr class="cit-jnl-abbrev">Mol. Microbiol.</abbr> <span class="cit-vol">87</span>: <span class="cit-fpage">998</span>–<span class="cit-lpage">1012</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/mmi.12146</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BMicrobiol.%26rft.volume%253D87%26rft.spage%253D998%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmmi.12146%26rft_id%253Dinfo%253Apmid%252F23289458%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mmi.12146&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23289458&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1249.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1249.20" data-doi="10.1016/j.cell.2005.01.007"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gitai</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dye</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reisenauer</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wachi</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shapiro</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">MreB actin-mediated segregation of a specific region of a bacterial chromosome.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">120</span>: <span class="cit-fpage">329</span>–<span class="cit-lpage">341</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.cell.2005.01.007</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DGitai%26rft.auinit1%253DZ.%26rft.volume%253D120%26rft.issue%253D3%26rft.spage%253D329%26rft.epage%253D341%26rft.atitle%253DMreB%2Bactin-mediated%2Bsegregation%2Bof%2Ba%2Bspecific%2Bregion%2Bof%2Ba%2Bbacterial%2Bchromosome.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2005.01.007%26rft_id%253Dinfo%253Apmid%252F15707892%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2005.01.007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15707892&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1249.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000227028900009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1249.21" data-doi="10.1073/pnas.0402638101"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gitai</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dye</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shapiro</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">An actin-like gene can determine cell polarity in bacteria.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">101</span>: <span class="cit-fpage">8643</span>–<span class="cit-lpage">8648</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.0402638101</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0402638101%26rft_id%253Dinfo%253Apmid%252F15159537%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTAxLzIzLzg2NDMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzQvMTI0OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-book" id="cit-9.4.1249.22" data-doi="10.1007/978-90-481-3473-1_3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Graumann</span>  <span class="cit-name-given-names">P. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The chromosome segregation machinery in bacteria</span>, pp. <span class="cit-fpage">31</span>–<span class="cit-lpage">48</span> in <span class="cit-source">Bacterial Chromatin</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Dame</span>  <span class="cit-name-given-names">R. T.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Dorman</span>  <span class="cit-name-given-names">C. J.</span></span>. <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">Dordrecht</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/978-90-481-3473-1_3</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBacterial%2BChromatin%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-90-481-3473-1_3%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-90-481-3473-1_3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1249.23" data-doi="10.1111/j.1574-6968.2011.02368.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Griese</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lange</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Soppa</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Ploidy in cyanobacteria.</span> <abbr class="cit-jnl-abbrev">FEMS Microbiol. Lett.</abbr> <span class="cit-vol">323</span>: <span class="cit-fpage">124</span>–<span class="cit-lpage">131</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1574-6968.2011.02368.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEMS%2BMicrobiol.%2BLett.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1574-6968.2011.02368.x%26rft_id%253Dinfo%253Apmid%252F22092711%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1574-6968.2011.02368.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22092711&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1249.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000295528200004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1249.24" data-doi="10.1016/j.cell.2009.02.035"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gruber</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Errington</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Recruitment of condensin to replication origin regions by ParB/SpoOJ promotes chromosome segregation in <em>B. subtilis</em>.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">137</span>: <span class="cit-fpage">685</span>–<span class="cit-lpage">696</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.cell.2009.02.035</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DGruber%26rft.auinit1%253DS.%26rft.volume%253D137%26rft.issue%253D4%26rft.spage%253D685%26rft.epage%253D696%26rft.atitle%253DRecruitment%2Bof%2Bcondensin%2Bto%2Breplication%2Borigin%2Bregions%2Bby%2BParB%252FSpoOJ%2Bpromotes%2Bchromosome%2Bsegregation%2Bin%2BB.%2Bsubtilis.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2009.02.035%26rft_id%253Dinfo%253Apmid%252F19450516%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2009.02.035&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19450516&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1249.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000266114100017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1249.25" data-doi="10.1038/srep36711"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guschinskaya</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brunel</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tourte</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lipscomb</span>  <span class="cit-name-given-names">G. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Adams</span>  <span class="cit-name-given-names">M. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Random mutagenesis of the hyperthermophilic archaeon <em>Pyrococcus furiosus</em> using <em>in vitro</em> mariner transposition and natural transformation.</span> <abbr class="cit-jnl-abbrev">Sci. Rep.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">36711</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/srep36711</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BRep.%26rft.volume%253D6%26rft.spage%253D36711%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsrep36711%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/srep36711&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1249.26"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hansen</span>  <span class="cit-name-given-names">M. T.</span></span></li></ol><cite>, <span class="cit-pub-date">1978</span> <span class="cit-article-title">Multiplicity of genome equivalents in the radiation-resistant bacterium <em>Micrococcus radiodurans</em>.</span> <abbr class="cit-jnl-abbrev">J. Bacteriol.</abbr> <span class="cit-vol">134</span>: <span class="cit-fpage">71</span>–<span class="cit-lpage">75</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBacteriology%26rft.stitle%253DJ.%2BBacteriol.%26rft.aulast%253DHansen%26rft.auinit1%253DM%2BT%26rft.volume%253D134%26rft.issue%253D1%26rft.spage%253D71%26rft.epage%253D75%26rft.atitle%253DMultiplicity%2Bof%2Bgenome%2Bequivalents%2Bin%2Bthe%2Bradiation-resistant%2Bbacterium%2BMicrococcus%2Bradiodurans.%26rft_id%253Dinfo%253Apmid%252F649572%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6ODoiMTM0LzEvNzEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzQvMTI0OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1249.27" data-doi="10.1038/nbt956"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Henne</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brüggemann</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raasch</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wiezer</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hartsch</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">The genome sequence of the extreme thermophile <em>Thermus thermophilus</em>.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">547</span>–<span class="cit-lpage">553</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nbt956</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DHenne%26rft.auinit1%253DA.%26rft.volume%253D22%26rft.issue%253D5%26rft.spage%253D547%26rft.epage%253D553%26rft.atitle%253DThe%2Bgenome%2Bsequence%2Bof%2Bthe%2Bextreme%2Bthermophile%2BThermus%2Bthermophilus.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt956%26rft_id%253Dinfo%253Apmid%252F15064768%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt956&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15064768&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1249.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000221159700022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1249.28" data-doi="10.1128/JB.01016-10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hildenbrand</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stock</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lange</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rother</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Soppa</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genome copy numbers and gene conversion in methanogenic archaea.</span> <abbr class="cit-jnl-abbrev">J. Bacteriol.</abbr> <span class="cit-vol">193</span>: <span class="cit-fpage">734</span>–<span class="cit-lpage">743</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1128/JB.01016-10</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBacteriol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJB.01016-10%26rft_id%253Dinfo%253Apmid%252F21097629%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6OToiMTkzLzMvNzM0IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS80LzEyNDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1249.29" data-doi="10.1111/j.1365-2958.2007.05618.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">MreB is important for cell shape but not for chromosome segregation of the filamentous cyanobacterium <em>Anabaena</em> sp. PCC 7120.</span> <abbr class="cit-jnl-abbrev">Mol. Microbiol.</abbr> <span class="cit-vol">63</span>: <span class="cit-fpage">1640</span>–<span class="cit-lpage">1652</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1365-2958.2007.05618.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bmicrobiology%26rft.stitle%253DMol%2BMicrobiol%26rft.aulast%253DHu%26rft.auinit1%253DB.%26rft.volume%253D63%26rft.issue%253D6%26rft.spage%253D1640%26rft.epage%253D1652%26rft.atitle%253DMreB%2Bis%2Bimportant%2Bfor%2Bcell%2Bshape%2Bbut%2Bnot%2Bfor%2Bchromosome%2Bsegregation%2Bof%2Bthe%2Bfilamentous%2Bcyanobacterium%2BAnabaena%2Bsp.%2BPCC%2B7120.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2958.2007.05618.x%26rft_id%253Dinfo%253Apmid%252F17367385%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-2958.2007.05618.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17367385&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1249.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000244886000007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1249.30" data-doi="10.1371/journal.pone.0086897"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Iniesta</span>  <span class="cit-name-given-names">A. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">ParABS system in chromosome partitioning in the bacterium <em>Myxococcus xanthus</em>.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e86897</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0086897</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De86897%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0086897%26rft_id%253Dinfo%253Apmid%252F24466283%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0086897&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24466283&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1249.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-9.4.1249.31"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ireton</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, N. W. 4th Gunther, and A. D. Grossman, <span class="cit-pub-date">1994</span> <em>spo0J</em> is required for normal chromosome segregation as well as the initiation of sporulation in <em>Bacillus subtilis</em>. J. Bacteriol. 176: 5320–5329.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1249.32" data-doi="10.1073/pnas.1211144109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jain</span>  <span class="cit-name-given-names">I. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vijayan</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">O’Shea</span>  <span class="cit-name-given-names">E. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Spatial ordering of chromosomes enhances the fidelity of chromosome partitioning in cyanobacteria.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">109</span>: <span class="cit-fpage">13638</span>–<span class="cit-lpage">13643</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1211144109</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1211144109%26rft_id%253Dinfo%253Apmid%252F22869746%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA5LzM0LzEzNjM4IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS80LzEyNDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1249.33" data-doi="10.1046/j.1365-2958.2002.03102.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jakimowicz</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chater</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zakrzewska-Czerwínska</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">The ParB protein of <em>Streptomyces coelicolor</em> A3(2) recognizes a cluster of <em>parS</em> sequences within the origin-proximal region of the linear chromosome.</span> <abbr class="cit-jnl-abbrev">Mol. Microbiol.</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">1365</span>–<span class="cit-lpage">1377</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1046/j.1365-2958.2002.03102.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bmicrobiology%26rft.stitle%253DMol%2BMicrobiol%26rft.aulast%253DJakimowicz%26rft.auinit1%253DD.%26rft.volume%253D45%26rft.issue%253D5%26rft.spage%253D1365%26rft.epage%253D1377%26rft.atitle%253DThe%2BParB%2Bprotein%2Bof%2BStreptomyces%2Bcoelicolor%2BA3%25282%2529%2Brecognizes%2Ba%2Bcluster%2Bof%2BparS%2Bsequences%2Bwithin%2Bthe%2Borigin-proximal%2Bregion%2Bof%2Bthe%2Blinear%2Bchromosome.%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1365-2958.2002.03102.x%26rft_id%253Dinfo%253Apmid%252F12207703%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1365-2958.2002.03102.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12207703&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1249.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000177750400017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1249.34" data-doi="10.1038/nrmicro2391"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jun</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Entropy as the driver of chromosome segregation.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Microbiol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">600</span>–<span class="cit-lpage">607</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrmicro2391</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BMicrobiology%26rft.stitle%253DNat%2BRev%2BMicrobiol%26rft.aulast%253DJun%26rft.auinit1%253DS.%26rft.volume%253D8%26rft.issue%253D8%26rft.spage%253D600%26rft.epage%253D607%26rft.atitle%253DEntropy%2Bas%2Bthe%2Bdriver%2Bof%2Bchromosome%2Bsegregation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrmicro2391%26rft_id%253Dinfo%253Apmid%252F20634810%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrmicro2391&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20634810&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1249.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000279987200014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1249.35" data-doi="10.1073/pnas.0605305103"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jun</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mulder</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Entropy-driven spatial organization of highly confined polymers: lessons for the bacterial chromosome.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">103</span>: <span class="cit-fpage">12388</span>–<span class="cit-lpage">12393</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.0605305103</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0605305103%26rft_id%253Dinfo%253Apmid%252F16885211%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTAzLzMzLzEyMzg4IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS80LzEyNDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1249.36" data-doi="10.1111/j.1365-313X.2006.02673.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Khakhlova</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bock</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Elimination of deleterious mutations in plastid genomes by gene conversion.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">85</span>–<span class="cit-lpage">94</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1365-313X.2006.02673.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2Bjournal%2B%253A%2B%2Bfor%2Bcell%2Band%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BJ%26rft.aulast%253DKhakhlova%26rft.auinit1%253DO.%26rft.volume%253D46%26rft.issue%253D1%26rft.spage%253D85%26rft.epage%253D94%26rft.atitle%253DElimination%2Bof%2Bdeleterious%2Bmutations%2Bin%2Bplastid%2Bgenomes%2Bby%2Bgene%2Bconversion.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-313X.2006.02673.x%26rft_id%253Dinfo%253Apmid%252F16553897%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-313X.2006.02673.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16553897&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1249.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000236035700006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1249.37" data-doi="10.1128/JB.182.5.1313-1320.2000"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">H. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Calcutt</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schmidt</span>  <span class="cit-name-given-names">F. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chater</span>  <span class="cit-name-given-names">K. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Partitioning of the linear chromosome during sporulation of <em>Streptomyces coelicolor</em> A3(2) involves an <em>oriC</em>-linked <em>parAB</em> locus.</span> <abbr class="cit-jnl-abbrev">J. Bacteriol.</abbr> <span class="cit-vol">182</span>: <span class="cit-fpage">1313</span>–<span class="cit-lpage">1320</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1128/JB.182.5.1313-1320.2000</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBacteriology%26rft.stitle%253DJ.%2BBacteriol.%26rft.aulast%253DKim%26rft.auinit1%253DH.-J.%26rft.volume%253D182%26rft.issue%253D5%26rft.spage%253D1313%26rft.epage%253D1320%26rft.atitle%253DPartitioning%2Bof%2Bthe%2BLinear%2BChromosome%2Bduring%2BSporulation%2Bof%2BStreptomyces%2Bcoelicolor%2BA3%25282%2529%2BInvolves%2Ban%2BoriC-Linked%2BparAB%2BLocus%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJB.182.5.1313-1320.2000%26rft_id%253Dinfo%253Apmid%252F10671452%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6MTA6IjE4Mi81LzEzMTMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzQvMTI0OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1249.38" data-doi="10.1111/mmi.13393"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kloosterman</span>  <span class="cit-name-given-names">T. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lenarcic</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Willis</span>  <span class="cit-name-given-names">C. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roberts</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hamoen</span>  <span class="cit-name-given-names">L. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Complex polar machinery required for proper chromosome segregation in vegetative and sporulating cells of <em>Bacillus subtilis</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Microbiol.</abbr> <span class="cit-vol">101</span>: <span class="cit-fpage">333</span>–<span class="cit-lpage">350</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/mmi.13393</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BMicrobiol.%26rft.volume%253D101%26rft.spage%253D333%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmmi.13393%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mmi.13393&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1249.39" data-doi="10.1146/annurev.mi.47.100193.004231"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kolter</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Siegele</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tormo</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">The stationary phase of the bacterial life cycle.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Microbiol.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">855</span>–<span class="cit-lpage">874</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1146/annurev.mi.47.100193.004231</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bmicrobiology%26rft.stitle%253DAnnu%2BRev%2BMicrobiol%26rft.aulast%253DKolter%26rft.auinit1%253DR%26rft.volume%253D47%26rft.issue%253D1%26rft.spage%253D855%26rft.epage%253D874%26rft.atitle%253DThe%2BStationary%2BPhase%2Bof%2Bthe%2BBacterial%2BLife%2BCycle%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.mi.47.100193.004231%26rft_id%253Dinfo%253Apmid%252F8257118%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.mi.47.100193.004231&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8257118&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1249.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1993MA27600030&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1249.40" data-doi="10.1101/gad.366606"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kruse</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blagoev</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Løbner-Olesen</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wachi</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sasaki</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Actin homolog MreB and RNA polymerase interact and are both required for chromosome segregation in <em>Escherichia coli</em>.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">113</span>–<span class="cit-lpage">124</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gad.366606</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DKruse%26rft.auinit1%253DT.%26rft.volume%253D20%26rft.issue%253D1%26rft.spage%253D113%26rft.epage%253D124%26rft.atitle%253DActin%2Bhomolog%2BMreB%2Band%2BRNA%2Bpolymerase%2Binteract%2Band%2Bare%2Bboth%2Brequired%2Bfor%2Bchromosome%2Bsegregation%2Bin%2BEscherichia%2Bcoli%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.366606%26rft_id%253Dinfo%253Apmid%252F16391237%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6ODoiMjAvMS8xMTMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzQvMTI0OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1249.41" data-doi="10.1111/j.1365-2958.2004.04367.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kruse</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bork-Jensen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gerdes</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">The morphogenetic MreBCD proteins of <em>Escherichia coli</em> form an essential membrane-bound complex.</span> <abbr class="cit-jnl-abbrev">Mol. Microbiol.</abbr> <span class="cit-vol">55</span>: <span class="cit-fpage">78</span>–<span class="cit-lpage">89</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1365-2958.2004.04367.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bmicrobiology%26rft.stitle%253DMol%2BMicrobiol%26rft.aulast%253DKruse%26rft.auinit1%253DT.%26rft.volume%253D55%26rft.issue%253D1%26rft.spage%253D78%26rft.epage%253D89%26rft.atitle%253DThe%2Bmorphogenetic%2BMreBCD%2Bproteins%2Bof%2BEscherichia%2Bcoli%2Bform%2Ban%2Bessential%2Bmembrane-bound%2Bcomplex.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2958.2004.04367.x%26rft_id%253Dinfo%253Apmid%252F15612918%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-2958.2004.04367.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15612918&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1249.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000225949300008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1249.42" data-doi="10.1093/emboj/cdg504"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kruse</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Møller-Jensen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Løbner-Olesen</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gerdes</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Dysfunctional MreB inhibits chromosome segregation in <em>Escherichia coli</em>.</span> <abbr class="cit-jnl-abbrev">EMBO J.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">5283</span>–<span class="cit-lpage">5292</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/emboj/cdg504</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BEMBO%2BJournal%26rft.stitle%253DEMBO%2BJournal%26rft.aulast%253DKruse%26rft.auinit1%253DT.%26rft.volume%253D22%26rft.issue%253D19%26rft.spage%253D5283%26rft.epage%253D5292%26rft.atitle%253DDysfunctional%2BMreB%2Binhibits%2Bchromosome%2Bsegregation%2Bin%2BEscherichia%2Bcoli%26rft_id%253Dinfo%253Adoi%252F10.1093%252Femboj%252Fcdg504%26rft_id%253Dinfo%253Apmid%252F14517265%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZW1ib2pubCI7czo1OiJyZXNpZCI7czoxMDoiMjIvMTkvNTI4MyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzkvNC8xMjQ5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1249.43" data-doi="10.1016/j.bpj.2014.10.074"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lampo</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuwada</span>  <span class="cit-name-given-names">N. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wiggins</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spakowitz</span>  <span class="cit-name-given-names">A. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Physical modeling of chromosome segregation in <em>Escherichia coli</em> reveals impact of force and DNA relaxation.</span> <abbr class="cit-jnl-abbrev">Biophys. J.</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">146</span>–<span class="cit-lpage">153</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.bpj.2014.10.074</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiophys.%2BJ.%26rft.volume%253D108%26rft.spage%253D146%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.bpj.2014.10.074%26rft_id%253Dinfo%253Apmid%252F25564861%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.bpj.2014.10.074&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25564861&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1249.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1249.44" data-doi="10.1111/j.1365-2958.2011.07600.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lange</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zerulla</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Breuert</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Soppa</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Gene conversion results in the equalization of genome copies in the polyploid haloarchaeon <em>Haloferax volcanii</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Microbiol.</abbr> <span class="cit-vol">80</span>: <span class="cit-fpage">666</span>–<span class="cit-lpage">677</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1365-2958.2011.07600.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bmicrobiology%26rft.stitle%253DMol%2BMicrobiol%26rft.aulast%253DLange%26rft.auinit1%253DC.%26rft.volume%253D80%26rft.issue%253D3%26rft.spage%253D666%26rft.epage%253D677%26rft.atitle%253DGene%2Bconversion%2Bresults%2Bin%2Bthe%2Bequalization%2Bof%2Bgenome%2Bcopies%2Bin%2Bthe%2Bpolyploid%2Bhaloarchaeon%2BHaloferax%2Bvolcanii.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2958.2011.07600.x%26rft_id%253Dinfo%253Apmid%252F21338422%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-2958.2011.07600.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21338422&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1249.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1249.45" data-doi="10.1128/JB.00371-07"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lasocki</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bartosik</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mierzejewska</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thomas</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jagura-Burdzy</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Deletion of the <em>parA</em> (<em>soj</em>) homologue in <em>Pseudomonas aeruginosa</em> causes ParB instability and affects growth rate, chromosome segregation, and motility.</span> <abbr class="cit-jnl-abbrev">J. Bacteriol.</abbr> <span class="cit-vol">189</span>: <span class="cit-fpage">5762</span>–<span class="cit-lpage">5772</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1128/JB.00371-07</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBacteriol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJB.00371-07%26rft_id%253Dinfo%253Apmid%252F17545287%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6MTE6IjE4OS8xNS81NzYyIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS80LzEyNDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1249.46" data-doi="10.1111/j.1365-2958.2006.05140.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">P. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grossman</span>  <span class="cit-name-given-names">A. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">The chromosome partitioning proteins Soj (ParA) and Spo0J (ParB) contribute to accurate chromosome partitioning, separation of replicated sister origins, and regulation of replication initiation in <em>Bacillus subtilis</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Microbiol.</abbr> <span class="cit-vol">60</span>: <span class="cit-fpage">853</span>–<span class="cit-lpage">869</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1365-2958.2006.05140.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bmicrobiology%26rft.stitle%253DMol%2BMicrobiol%26rft.aulast%253DLee%26rft.auinit1%253DP.%2BS.%26rft.volume%253D60%26rft.issue%253D4%26rft.spage%253D853%26rft.epage%253D869%26rft.atitle%253DThe%2Bchromosome%2Bpartitioning%2Bproteins%2BSoj%2B%2528ParA%2529%2Band%2BSpo0J%2B%2528ParB%2529%2Bcontribute%2Bto%2Baccurate%2Bchromosome%2Bpartitioning%252C%2Bseparation%2Bof%2Breplicated%2Bsister%2Borigins%252C%2Band%2Bregulation%2Bof%2Breplication%2Binitiation%2Bin%2BBacillus%2Bsubtilis.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2958.2006.05140.x%26rft_id%253Dinfo%253Apmid%252F16677298%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-2958.2006.05140.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16677298&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1249.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000237297800005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1249.47" data-doi="10.1016/j.jbiotec.2014.07.448"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Leis</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Angelov</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liebl</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genetic analysis of lipolytic activities in <em>Thermus thermophilus</em> HB27.</span> <abbr class="cit-jnl-abbrev">J. Biotechnol.</abbr> <span class="cit-vol">191</span>: <span class="cit-fpage">150</span>–<span class="cit-lpage">157</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.jbiotec.2014.07.448</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiotechnol.%26rft.volume%253D191%26rft.spage%253D150%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jbiotec.2014.07.448%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jbiotec.2014.07.448&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1249.48" data-doi="10.1111/j.1365-2958.1997.mmi530.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lewis</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Errington</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Direct evidence for active segregation of <em>oriC</em> regions of the <em>Bacillus subtilis</em> chromosome and co-localization with the Spo0J partitioning protein.</span> <abbr class="cit-jnl-abbrev">Mol. Microbiol.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">945</span>–<span class="cit-lpage">954</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1365-2958.1997.mmi530.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bmicrobiology%26rft.stitle%253DMol%2BMicrobiol%26rft.aulast%253DLewis%26rft.auinit1%253DP.%2BJ.%26rft.volume%253D25%26rft.issue%253D5%26rft.spage%253D945%26rft.epage%253D954%26rft.atitle%253DDirect%2Bevidence%2Bfor%2Bactive%2Bsegregation%2Bof%2BoriC%2Bregions%2Bof%2Bthe%2BBacillus%2Bsubtilis%2Bchromosome%2Band%2Bco-localization%2Bwith%2Bthe%2BSpoOJ%2Bpartitioning%2Bprotein.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2958.1997.mmi530.x%26rft_id%253Dinfo%253Apmid%252F9364919%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-2958.1997.mmi530.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9364919&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1249.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1997XZ16400013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1249.49" data-doi="10.1186/s12864-015-1523-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Angelov</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pham</span>  <span class="cit-name-given-names">V. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leis</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liebl</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Characterization of chromosomal and megaplasmid partitioning loci in <em>Thermus thermophilus</em> HB27.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">317</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12864-015-1523-3</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D16%26rft.spage%253D317%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-015-1523-3%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12864-015-1523-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1249.50" data-doi="10.1016/S0092-8674(00)81135-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">D. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grossman</span>  <span class="cit-name-given-names">A. D.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Identification and characterization of a bacterial chromosome partitioning site.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">92</span>: <span class="cit-fpage">675</span>–<span class="cit-lpage">685</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/S0092-8674(00)81135-6</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DLin%26rft.auinit1%253DD.%2BC.%26rft.volume%253D92%26rft.issue%253D5%26rft.spage%253D675%26rft.epage%253D685%26rft.atitle%253DIdentification%2Band%2Bcharacterization%2Bof%2Ba%2Bbacterial%2Bchromosome%2Bpartitioning%2Bsite.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0092-8674%252800%252981135-6%26rft_id%253Dinfo%253Apmid%252F9506522%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0092-8674(00)81135-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9506522&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1249.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000072406000013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1249.51" data-doi="10.1016/j.molcel.2009.01.001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Madabhushi</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marians</span>  <span class="cit-name-given-names">K. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Actin homolog MreB affects chromosome segregation by regulating topoisomerase IV in <em>Escherichia coli</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">171</span>–<span class="cit-lpage">180</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.molcel.2009.01.001</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DMadabhushi%26rft.auinit1%253DR.%26rft.volume%253D33%26rft.issue%253D2%26rft.spage%253D171%26rft.epage%253D180%26rft.atitle%253DActin%2Bhomolog%2BMreB%2Baffects%2Bchromosome%2Bsegregation%2Bby%2Bregulating%2Btopoisomerase%2BIV%2Bin%2BEscherichia%2Bcoli.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2009.01.001%26rft_id%253Dinfo%253Apmid%252F19187760%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2009.01.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19187760&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1249.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000263204500008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1249.52"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Malandrin</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huber</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bernander</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Nucleoid structure and partition in <em>Methanococcus jannaschii</em>: an archaeon with multiple copies of the chromosome.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">152</span>: <span class="cit-fpage">1315</span>–<span class="cit-lpage">1323</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMalandrin%26rft.auinit1%253DL.%26rft.volume%253D152%26rft.issue%253D4%26rft.spage%253D1315%26rft.epage%253D1323%26rft.atitle%253DNucleoid%2BStructure%2Band%2BPartition%2Bin%2BMethanococcus%2Bjannaschii%253A%2BAn%2BArchaeon%2BWith%2BMultiple%2BCopies%2Bof%2Bthe%2BChromosome%26rft_id%253Dinfo%253Apmid%252F10430562%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1Mi80LzEzMTUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzQvMTI0OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1249.53" data-doi="10.1073/pnas.0707522105"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mendell</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clements</span>  <span class="cit-name-given-names">K. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Choat</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Angert</span>  <span class="cit-name-given-names">E. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Extreme polyploidy in a large bacterium.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">105</span>: <span class="cit-fpage">6730</span>–<span class="cit-lpage">6734</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.0707522105</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0707522105%26rft_id%253Dinfo%253Apmid%252F18445653%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTA1LzE4LzY3MzAiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzQvMTI0OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1249.54" data-doi="10.1111/j.1365-2958.2011.07722.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Minnen</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Attaiech</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thon</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gruber</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Veening</span>  <span class="cit-name-given-names">J. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">SMC is recruited to <em>oriC</em> by ParB and promotes chromosome segregation in <em>Streptococcus pneumoniae</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Microbiol.</abbr> <span class="cit-vol">81</span>: <span class="cit-fpage">676</span>–<span class="cit-lpage">688</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1365-2958.2011.07722.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bmicrobiology%26rft.stitle%253DMol%2BMicrobiol%26rft.aulast%253DMinnen%26rft.auinit1%253DA.%26rft.volume%253D81%26rft.issue%253D3%26rft.spage%253D676%26rft.epage%253D688%26rft.atitle%253DSMC%2Bis%2Brecruited%2Bto%2BoriC%2Bby%2BParB%2Band%2Bpromotes%2Bchromosome%2Bsegregation%2Bin%2BStreptococcus%2Bpneumoniae.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2958.2011.07722.x%26rft_id%253Dinfo%253Apmid%252F21651626%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-2958.2011.07722.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21651626&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1249.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1249.55" data-doi="10.1016/S0092-8674(00)81910-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mohl</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gober</span>  <span class="cit-name-given-names">J. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Cell cycle-dependent polar localization of chromosome partitioning proteins in <em>Caulobacter crescentus</em>.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">88</span>: <span class="cit-fpage">675</span>–<span class="cit-lpage">684</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/S0092-8674(00)81910-8</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DMohl%26rft.auinit1%253DD.%2BA.%26rft.volume%253D88%26rft.issue%253D5%26rft.spage%253D675%26rft.epage%253D684%26rft.atitle%253DCell%2Bcycle-dependent%2Bpolar%2Blocalization%2Bof%2Bchromosome%2Bpartitioning%2Bproteins%2Bin%2BCaulobacter%2Bcrescentus.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0092-8674%252800%252981910-8%26rft_id%253Dinfo%253Apmid%252F9054507%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0092-8674(00)81910-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9054507&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1249.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1997WM41300013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1249.56" data-doi="10.1046/j.1365-2958.2001.02643.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mohl</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Easter</span>  <span class="cit-name-given-names">J. Jr.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gober</span>  <span class="cit-name-given-names">J. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">The chromosome partitioning protein, ParB, is required for cytokinesis in <em>Caulobacter crescentus</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Microbiol.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">741</span>–<span class="cit-lpage">755</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1046/j.1365-2958.2001.02643.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bmicrobiology%26rft.stitle%253DMol%2BMicrobiol%26rft.aulast%253DMohl%26rft.auinit1%253DD.%2BA.%26rft.volume%253D42%26rft.issue%253D3%26rft.spage%253D741%26rft.epage%253D755%26rft.atitle%253DThe%2Bchromosome%2Bpartitioning%2Bprotein%252C%2BParB%252C%2Bis%2Brequired%2Bfor%2Bcytokinesis%2Bin%2BCaulobacter%2Bcrescentus.%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1365-2958.2001.02643.x%26rft_id%253Dinfo%253Apmid%252F11722739%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1365-2958.2001.02643.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11722739&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1249.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000172341700013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1249.57" data-doi="10.1016/j.cell.2008.07.044"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Murray</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Errington</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Dynamic control of the DNA replication initiation protein DnaA by Soj/ParA.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">135</span>: <span class="cit-fpage">74</span>–<span class="cit-lpage">84</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.cell.2008.07.044</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DMurray%26rft.auinit1%253DH.%26rft.volume%253D135%26rft.issue%253D1%26rft.spage%253D74%26rft.epage%253D84%26rft.atitle%253DDynamic%2Bcontrol%2Bof%2Bthe%2BDNA%2Breplication%2Binitiation%2Bprotein%2BDnaA%2Bby%2BSoj%252FParA.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2008.07.044%26rft_id%253Dinfo%253Apmid%252F18854156%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2008.07.044&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18854156&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1249.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000259728100010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1249.58" data-doi="10.1111/j.1574-6976.2010.00213.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Navarro Llorens</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tormo</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martínez-García</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Stationary phase in gram-negative bacteria.</span> <abbr class="cit-jnl-abbrev">FEMS Microbiol. Rev.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">476</span>–<span class="cit-lpage">495</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1574-6976.2010.00213.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEMS%2BMicrobiol.%2BRev.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1574-6976.2010.00213.x%26rft_id%253Dinfo%253Apmid%252F20236330%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1574-6976.2010.00213.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20236330&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1249.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000278341700004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1249.59" data-doi="10.1093/mutage/gel025"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ohta</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tokishita</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Imazuka</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mori</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Okamura</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">beta-Glucosidase as a reporter for the gene expression studies in <em>Thermus thermophilus</em> and constitutive expression of DNA repair genes.</span> <abbr class="cit-jnl-abbrev">Mutagenesis</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">255</span>–<span class="cit-lpage">260</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/mutage/gel025</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMutagenesis%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmutage%252Fgel025%26rft_id%253Dinfo%253Apmid%252F16777922%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/mutage/gel025&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16777922&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1249.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1249.60" data-doi="10.1128/JB.00662-10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ohtani</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tomita</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Itaya</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">An extreme thermophile, <em>Thermus thermophilus</em>, is a polyploid bacterium.</span> <abbr class="cit-jnl-abbrev">J. Bacteriol.</abbr> <span class="cit-vol">192</span>: <span class="cit-fpage">5499</span>–<span class="cit-lpage">5505</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1128/JB.00662-10</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBacteriol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJB.00662-10%26rft_id%253Dinfo%253Apmid%252F20729360%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6MTE6IjE5Mi8yMC81NDk5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS80LzEyNDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1249.61" data-doi="10.1128/AEM.03603-15"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ohtani</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tomita</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Itaya</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Curing the megaplasmid pTT27 from <em>Thermus thermophilus</em> HB27 and maintaining exogenous plasmids in the plasmid-free strain.</span> <abbr class="cit-jnl-abbrev">Appl. Environ. Microbiol.</abbr> <span class="cit-vol">82</span>: <span class="cit-fpage">1537</span>–<span class="cit-lpage">1548</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1128/AEM.03603-15</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAppl.%2BEnviron.%2BMicrobiol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FAEM.03603-15%26rft_id%253Dinfo%253Apmid%252F26712540%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiYWVtIjtzOjU6InJlc2lkIjtzOjk6IjgyLzUvMTUzNyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzkvNC8xMjQ5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1249.62" data-doi="10.1111/mmi.13661"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Paulsson</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">El Karoui</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lindell</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hughes</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">The processive kinetics of gene conversion in bacteria.</span> <abbr class="cit-jnl-abbrev">Mol. Microbiol.</abbr> <span class="cit-vol">104</span>: <span class="cit-fpage">752</span>–<span class="cit-lpage">760</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/mmi.13661</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BMicrobiol.%26rft.volume%253D104%26rft.spage%253D752%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmmi.13661%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mmi.13661&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1249.63" data-doi="10.1371/journal.pone.0016392"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pecoraro</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zerulla</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lange</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Soppa</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Quantification of ploidy in proteobacteria revealed the existence of monoploid, (mero-) oligoploid and polyploid species.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e16392</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0016392</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DPecoraro%26rft.auinit1%253DV.%26rft.volume%253D6%26rft.issue%253D1%26rft.spage%253De16392%26rft.epage%253De16392%26rft.atitle%253DQuantification%2Bof%2Bploidy%2Bin%2Bproteobacteria%2Brevealed%2Bthe%2Bexistence%2Bof%2Bmonoploid%252C%2B%2528mero-%2529oligoploid%2Band%2Bpolyploid%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0016392%26rft_id%253Dinfo%253Apmid%252F21305010%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0016392&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21305010&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1249.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1249.64" data-doi="10.1038/ncb2083"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ptacin</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">S. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garner</span>  <span class="cit-name-given-names">E. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Toro</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eckart</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">A spindle-like apparatus guides bacterial chromosome segregation.</span> <abbr class="cit-jnl-abbrev">Nat. Cell Biol.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">791</span>–<span class="cit-lpage">798</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ncb2083</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2BCell%2BBiology%26rft.stitle%253DNature%2BCell%2BBiology%26rft.aulast%253DPtacin%26rft.auinit1%253DJ.%2BL.%26rft.volume%253D12%26rft.issue%253D8%26rft.spage%253D791%26rft.epage%253D798%26rft.atitle%253DA%2Bspindle-like%2Bapparatus%2Bguides%2Bbacterial%2Bchromosome%2Bsegregation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncb2083%26rft_id%253Dinfo%253Apmid%252F20657594%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncb2083&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20657594&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1249.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000280561600011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1249.65" data-doi="10.1186/1471-2121-8-39"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schneider</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fuhrmann</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scholz</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hess</span>  <span class="cit-name-given-names">W. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Graumann</span>  <span class="cit-name-given-names">P. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Fluorescence staining of live cyanobacterial cells suggest non-stringent chromosome segregation and absence of a connection between cytoplasmic and thylakoid membranes.</span> <abbr class="cit-jnl-abbrev">BMC Cell Biol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">39</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2121-8-39</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bcell%2Bbiology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BCell%2BBiol%26rft.aulast%253DSchneider%26rft.auinit1%253DD.%26rft.volume%253D8%26rft.spage%253D39%26rft.epage%253D39%26rft.atitle%253DFluorescence%2Bstaining%2Bof%2Blive%2Bcyanobacterial%2Bcells%2Bsuggest%2Bnon-stringent%2Bchromosome%2Bsegregation%2Band%2Babsence%2Bof%2Ba%2Bconnection%2Bbetween%2Bcytoplasmic%2Band%2Bthylakoid%2Bmembranes.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2121-8-39%26rft_id%253Dinfo%253Apmid%252F17767716%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2121-8-39&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17767716&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1249.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1249.66" data-doi="10.1111/j.1365-2958.2010.07507.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Scholefield</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Whiting</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Errington</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Murray</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Spo0J regulates the oligomeric state of Soj to trigger its switch from an activator to an inhibitor of DNA replication initiation.</span> <abbr class="cit-jnl-abbrev">Mol. Microbiol.</abbr> <span class="cit-vol">79</span>: <span class="cit-fpage">1089</span>–<span class="cit-lpage">1100</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1365-2958.2010.07507.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bmicrobiology%26rft.stitle%253DMol%2BMicrobiol%26rft.aulast%253DScholefield%26rft.auinit1%253DG.%26rft.volume%253D79%26rft.issue%253D4%26rft.spage%253D1089%26rft.epage%253D1100%26rft.atitle%253DSpo0J%2Bregulates%2Bthe%2Boligomeric%2Bstate%2Bof%2BSoj%2Bto%2Btrigger%2Bits%2Bswitch%2Bfrom%2Ban%2Bactivator%2Bto%2Ban%2Binhibitor%2Bof%2BDNA%2Breplication%2Binitiation.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2958.2010.07507.x%26rft_id%253Dinfo%253Apmid%252F21235642%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-2958.2010.07507.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21235642&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1249.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000287094300019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1249.67" data-doi="10.1111/j.1365-2958.1996.tb02559.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sharpe</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Errington</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">The <em>Bacillus subtilis soj-spo0J</em> locus is required for a centromere-like function involved in prespore chromosome partitioning.</span> <abbr class="cit-jnl-abbrev">Mol. Microbiol.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">501</span>–<span class="cit-lpage">509</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1365-2958.1996.tb02559.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bmicrobiology%26rft.stitle%253DMol%2BMicrobiol%26rft.aulast%253DSharpe%26rft.auinit1%253DM.%2BE.%26rft.volume%253D21%26rft.issue%253D3%26rft.spage%253D501%26rft.epage%253D509%26rft.atitle%253DThe%2BBacillus%2Bsubtilis%2Bsoj-spo0J%2Blocus%2Bis%2Brequired%2Bfor%2Ba%2Bcentromere-like%2Bfunction%2Binvolved%2Bin%2Bprespore%2Bchromosome%2Bpartitioning.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2958.1996.tb02559.x%26rft_id%253Dinfo%253Apmid%252F8866474%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-2958.1996.tb02559.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8866474&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1249.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1996VC40400008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1249.68" data-doi="10.1073/pnas.1005274107"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shebelut</span>  <span class="cit-name-given-names">C. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guberman</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Teeffelen</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yakhnina</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gitai</span>  <span class="cit-name-given-names">Z.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Caulobacter chromosome segregation is an ordered multistep process.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">107</span>: <span class="cit-fpage">14194</span>–<span class="cit-lpage">14198</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1005274107</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1005274107%26rft_id%253Dinfo%253Apmid%252F20660743%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA3LzMyLzE0MTk0IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS80LzEyNDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1249.69" data-doi="10.1128/JB.00862-08"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shebelut</span>  <span class="cit-name-given-names">C. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jensen</span>  <span class="cit-name-given-names">R. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gitai</span>  <span class="cit-name-given-names">Z.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Growth conditions regulate the requirements for Caulobacter chromosome segregation.</span> <abbr class="cit-jnl-abbrev">J. Bacteriol.</abbr> <span class="cit-vol">191</span>: <span class="cit-fpage">1097</span>–<span class="cit-lpage">1100</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1128/JB.00862-08</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBacteriol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJB.00862-08%26rft_id%253Dinfo%253Apmid%252F19028887%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6MTA6IjE5MS8zLzEwOTciO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzQvMTI0OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1249.70" data-doi="10.1016/j.cell.2009.01.018"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Slade</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lindner</span>  <span class="cit-name-given-names">A. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paul</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Radman</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Recombination and replication in DNA repair of heavily irradiated <em>Deinococcus radiodurans</em>.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">136</span>: <span class="cit-fpage">1044</span>–<span class="cit-lpage">1055</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.cell.2009.01.018</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DSlade%26rft.auinit1%253DD.%26rft.volume%253D136%26rft.issue%253D6%26rft.spage%253D1044%26rft.epage%253D1055%26rft.atitle%253DRecombination%2Band%2Breplication%2Bin%2BDNA%2Brepair%2Bof%2Bheavily%2Birradiated%2BDeinococcus%2Bradiodurans.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2009.01.018%26rft_id%253Dinfo%253Apmid%252F19303848%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2009.01.018&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19303848&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1249.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000264403900011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1249.71" data-doi="10.1042/BST0390150"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Soppa</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Ploidy and gene conversion in Archaea.</span> <abbr class="cit-jnl-abbrev">Biochem. Soc. Trans.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">150</span>–<span class="cit-lpage">154</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1042/BST0390150</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochem.%2BSoc.%2BTrans.%26rft_id%253Dinfo%253Adoi%252F10.1042%252FBST0390150%26rft_id%253Dinfo%253Apmid%252F21265763%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoicHBiaW9zdCI7czo1OiJyZXNpZCI7czo4OiIzOS8xLzE1MCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzkvNC8xMjQ5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference  in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1249.72" data-doi="10.1042/BST20120315"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Soppa</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Evolutionary advantages of polyploidy in halophilic archaea.</span> <abbr class="cit-jnl-abbrev">Biochem. Soc. Trans.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">339</span>–<span class="cit-lpage">343</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1042/BST20120315</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochem.%2BSoc.%2BTrans.%26rft_id%253Dinfo%253Adoi%252F10.1042%252FBST20120315%26rft_id%253Dinfo%253Apmid%252F23356308%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoicHBiaW9zdCI7czo1OiJyZXNpZCI7czo4OiI0MS8xLzMzOSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzkvNC8xMjQ5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference  in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1249.73" data-doi="10.1159/000368855"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Soppa</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Polyploidy in archaea and bacteria: about desiccation resistance, giant cell size, long-term survival, enforcement by a eukaryotic host and additional aspects.</span> <abbr class="cit-jnl-abbrev">J. Mol. Microbiol. Biotechnol.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">409</span>–<span class="cit-lpage">419</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1159/000368855</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BMol.%2BMicrobiol.%2BBiotechnol.%26rft.volume%253D24%26rft.spage%253D409%26rft_id%253Dinfo%253Adoi%252F10.1159%252F000368855%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1159/000368855&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference  in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1249.74" data-doi="10.1016/j.cub.2003.10.024"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Soufo</span>  <span class="cit-name-given-names">H. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Graumann</span>  <span class="cit-name-given-names">P. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Actin-like proteins MreB and Mbl from <em>Bacillus subtilis</em> are required for bipolar positioning of replication origins.</span> <abbr class="cit-jnl-abbrev">Curr. Biol.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">1916</span>–<span class="cit-lpage">1920</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.cub.2003.10.024</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bbiology%2B%253A%2B%2BCB%26rft.stitle%253DCurr%2BBiol%26rft.aulast%253DSoufo%26rft.auinit1%253DH.%2BJ.%26rft.volume%253D13%26rft.issue%253D21%26rft.spage%253D1916%26rft.epage%253D1920%26rft.atitle%253DActin-like%2Bproteins%2BMreB%2Band%2BMbl%2Bfrom%2BBacillus%2Bsubtilis%2Bare%2Brequired%2Bfor%2Bbipolar%2Bpositioning%2Bof%2Breplication%2Borigins.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cub.2003.10.024%26rft_id%253Dinfo%253Apmid%252F14588250%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cub.2003.10.024&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14588250&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1249.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000186263900026&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference  in text" id="ref-75">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1249.75" data-doi="10.1007/s00792-015-0750-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Spaans</span>  <span class="cit-name-given-names">S. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van der Oost</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kengen</span>  <span class="cit-name-given-names">S. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">The chromosome copy number of the hyperthermophilic archaeon <em>Thermococcus kodakarensis</em> KOD1.</span> <abbr class="cit-jnl-abbrev">Extremophiles</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">741</span>–<span class="cit-lpage">750</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00792-015-0750-5</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DExtremophiles%26rft.volume%253D19%26rft.spage%253D741%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00792-015-0750-5%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00792-015-0750-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-76-1" title="View reference  in text" id="ref-76">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1249.76" data-doi="10.1016/j.cell.2009.04.044"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sullivan</span>  <span class="cit-name-given-names">N. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marquis</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rudner</span>  <span class="cit-name-given-names">D. Z.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Recruitment of SMC by ParB-<em>parS</em> organizes the origin region and promotes efficient chromosome segregation.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">137</span>: <span class="cit-fpage">697</span>–<span class="cit-lpage">707</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.cell.2009.04.044</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DSullivan%26rft.auinit1%253DN.%2BL.%26rft.volume%253D137%26rft.issue%253D4%26rft.spage%253D697%26rft.epage%253D707%26rft.atitle%253DRecruitment%2Bof%2BSMC%2Bby%2BParB-parS%2Borganizes%2Bthe%2Borigin%2Bregion%2Band%2Bpromotes%2Befficient%2Bchromosome%2Bsegregation.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2009.04.044%26rft_id%253Dinfo%253Apmid%252F19450517%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2009.04.044&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19450517&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1249.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000266114100018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-77-1" title="View reference  in text" id="ref-77">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1249.77" data-doi="10.1007/978-3-319-53047-5_4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sundararajan</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goley</span>  <span class="cit-name-given-names">E. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Cytoskeletal proteins in <em>Caulobacter crescentus</em>: spatial orchestrators of cell cycle progression, development, and cell shape.</span> <abbr class="cit-jnl-abbrev">Subcell. Biochem.</abbr> <span class="cit-vol">84</span>: <span class="cit-fpage">103</span>–<span class="cit-lpage">137</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/978-3-319-53047-5_4</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSubcell.%2BBiochem.%26rft.volume%253D84%26rft.spage%253D103%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-3-319-53047-5_4%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-3-319-53047-5_4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-78-1" title="View reference  in text" id="ref-78">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1249.78" data-doi="10.1093/nar/gkx596"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Panis</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Viollier</span>  <span class="cit-name-given-names">P. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marczynski</span>  <span class="cit-name-given-names">G. T.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">A novel nucleoid-associated protein coordinates chromosome replication and chromosome partition.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">8916</span>–<span class="cit-lpage">8929</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkx596</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft.volume%253D45%26rft.spage%253D8916%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkx596%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkx596&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-79-1" title="View reference  in text" id="ref-79">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1249.79" data-doi="10.1073/pnas.0807448105"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Toro</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hong</span>  <span class="cit-name-given-names">S. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McAdams</span>  <span class="cit-name-given-names">H. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shapiro</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Caulobacter requires a dedicated mechanism to initiate chromosome segregation.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">105</span>: <span class="cit-fpage">15435</span>–<span class="cit-lpage">15440</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.0807448105</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0807448105%26rft_id%253Dinfo%253Apmid%252F18824683%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA1LzQwLzE1NDM1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS80LzEyNDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-80-1" title="View reference  in text" id="ref-80">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1249.80" data-doi="10.1128/AEM.03524-15"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoffmann</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Watzlawick</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Altenbuchner</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Markerless gene deletion with cytosine deaminase in <em>Thermus thermophilus</em> strain HB27.</span> <abbr class="cit-jnl-abbrev">Appl. Environ. Microbiol.</abbr> <span class="cit-vol">82</span>: <span class="cit-fpage">1249</span>–<span class="cit-lpage">1255</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1128/AEM.03524-15</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAppl.%2BEnviron.%2BMicrobiol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FAEM.03524-15%26rft_id%253Dinfo%253Apmid%252F26655764%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiYWVtIjtzOjU6InJlc2lkIjtzOjk6IjgyLzQvMTI0OSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzkvNC8xMjQ5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-81-1" title="View reference  in text" id="ref-81">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1249.81" data-doi="10.1016/j.mib.2014.09.016"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rudner</span>  <span class="cit-name-given-names">D. Z.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Spatial organization of bacterial chromosomes.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Microbiol.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">66</span>–<span class="cit-lpage">72</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.mib.2014.09.016</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BOpin.%2BMicrobiol.%26rft.volume%253D22%26rft.spage%253D66%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.mib.2014.09.016%26rft_id%253Dinfo%253Apmid%252F25460798%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.mib.2014.09.016&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25460798&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1249.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-82-1" title="View reference  in text" id="ref-82">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1249.82" data-doi="10.1126/science.aai8982"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brandão</span>  <span class="cit-name-given-names">H. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Le</span>  <span class="cit-name-given-names">T. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Laub</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rudner</span>  <span class="cit-name-given-names">D. Z.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title"><em>Bacillus subtilis</em> SMC complexes juxtapose chromosome arms as they travel from origin to terminus.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">355</span>: <span class="cit-fpage">524</span>–<span class="cit-lpage">527</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.aai8982</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DWang%26rft.auinit1%253DX.%26rft.volume%253D355%26rft.issue%253D6324%26rft.spage%253D524%26rft.epage%253D527%26rft.atitle%253DBacillus%2Bsubtilis%2BSMC%2Bcomplexes%2Bjuxtapose%2Bchromosome%2Barms%2Bas%2Bthey%2Btravel%2Bfrom%2Borigin%2Bto%2Bterminus%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.aai8982%26rft_id%253Dinfo%253Apmid%252F28154080%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzNTUvNjMyNC81MjQiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzQvMTI0OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-83-1" title="View reference  in text" id="ref-83">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1249.83" data-doi="10.1073/pnas.0608341104"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yamaichi</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fogel</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Waldor</span>  <span class="cit-name-given-names">M. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title"><em>par</em> genes and the pathology of chromosome loss in <em>Vibrio cholerae</em>.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">104</span>: <span class="cit-fpage">630</span>–<span class="cit-lpage">635</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.0608341104</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0608341104%26rft_id%253Dinfo%253Apmid%252F17197419%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiIxMDQvMi82MzAiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzQvMTI0OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-84-1" title="View reference  in text" id="ref-84">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1249.84" data-doi="10.1038/nature05160"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zahradka</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Slade</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bailone</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sommer</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Averbeck</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Reassembly of shattered chromosomes in <em>Deinococcus radiodurans</em>.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">443</span>: <span class="cit-fpage">569</span>–<span class="cit-lpage">573</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DZahradka%26rft.auinit1%253DK.%26rft.volume%253D443%26rft.issue%253D7111%26rft.spage%253D569%26rft.epage%253D573%26rft.atitle%253DReassembly%2Bof%2Bshattered%2Bchromosomes%2Bin%2BDeinococcus%2Bradiodurans.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature05160%26rft_id%253Dinfo%253Apmid%252F17006450%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature05160&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17006450&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1249.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000240988200042&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-85-1" title="View reference  in text" id="ref-85">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1249.85" data-doi="10.1099/mic.0.000264"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zerulla</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ludt</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Soppa</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">The ploidy level of <em>Synechocystis</em> sp. PCC 6803 is highly variable and is influenced by growth phase and by chemical and physical external parameters.</span> <abbr class="cit-jnl-abbrev">Microbiology</abbr> <span class="cit-vol">162</span>: <span class="cit-fpage">730</span>–<span class="cit-lpage">739</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1099/mic.0.000264</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMicrobiology%26rft.volume%253D162%26rft.spage%253D730%26rft_id%253Dinfo%253Adoi%252F10.1099%252Fmic.0.000264%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1099/mic.0.000264&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/9/4/1249.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/4/1231" title="Hyperspectral Reflectance-Derived Relationship Matrices for Genomic Prediction of Grain Yield in Wheat" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/4/1263" title="Identification and Expression Analysis of Long Noncoding RNAs in Fat-Tail of Sheep Breeds" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/9/4"><span>Volume 9 Issue 4, April 2019</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/9/4.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/9/4.cover-source.jpg?itok=mcy5SuIK" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 9 (4)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/4/1249" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Dqv2cY9zI7Yr5xd1fZYTYu2jg5kwFD1ZK81x_RmmXk4" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/9/4/1249" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/9/4/1249" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/9/4/1249" class="active" data-icon-position="" data-hide-link-title="0">Random Chromosome Partitioning in the Polyploid Bacterium Thermus thermophilus HB27</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/492611" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-qVyl6CaCCjO-CqWhdzm1F79bGDWowQ94ASaaH941VDo" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/4/1249.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/9/4/1249" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-gZAMWha0oxCB-4a1xRiNaanc_1nk72tr1yHMEUp2Sv4" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/9/4/1249" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/4/1249.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="492611" id="node492611--2" data-pisa="ggg;9/4/1249" data-pisa-master="ggg;g3.119.400086" data-apath="/ggg/9/4/1249.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Random Chromosome Partitioning in the Polyploid Bacterium <em>Thermus thermophilus</em> HB27</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-9428-6190" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Haijuan Li</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1249-1261;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400086 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node492611--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Haijuan Li </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>College of Biological and Environmental Engineering, Xi’an University, Xi’an 710065, Shaanxi, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Haijuan%2BLi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Li%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AHaijuan%2BLi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-9428-6190" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Haijuan Li</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:haijuanli@xawl.edu.cn" class="" data-icon-position="" data-hide-link-title="0">haijuanli@xawl.edu.cn</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/9/4/1249" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/9/4/1249&amp;title=Random%20Chromosome%20Partitioning%20in%20the%20Polyploid%20Bacterium%20Thermus%20thermophilus%20HB27" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/9/4/1249&amp;title=Random%20Chromosome%20Partitioning%20in%20the%20Polyploid%20Bacterium%20Thermus%20thermophilus%20HB27" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/9/4/1249&amp;title=Random%20Chromosome%20Partitioning%20in%20the%20Polyploid%20Bacterium%20Thermus%20thermophilus%20HB27" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/9/4/1249&amp;text=Random%20Chromosome%20Partitioning%20in%20the%20Polyploid%20Bacterium%20Thermus%20thermophilus%20HB27" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/9/4/1249&amp;title=Random%20Chromosome%20Partitioning%20in%20the%20Polyploid%20Bacterium%20Thermus%20thermophilus%20HB27" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/9/4/1249&amp;t=Random%20Chromosome%20Partitioning%20in%20the%20Polyploid%20Bacterium%20Thermus%20thermophilus%20HB27" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/9/4/1249&amp;title=Random%20Chromosome%20Partitioning%20in%20the%20Polyploid%20Bacterium%20Thermus%20thermophilus%20HB27" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/9/4/1249&amp;title=Random%20Chromosome%20Partitioning%20in%20the%20Polyploid%20Bacterium%20Thermus%20thermophilus%20HB27" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/492611/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="492611" id="node492611--3" data-pisa="ggg;9/4/1249" data-pisa-master="ggg;g3.119.400086" data-apath="/ggg/9/4/1249.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Random Chromosome Partitioning in the Polyploid Bacterium <em>Thermus thermophilus</em> HB27</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-9428-6190" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Haijuan Li</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1249-1261;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400086 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node492611--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Haijuan Li </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>College of Biological and Environmental Engineering, Xi’an University, Xi’an 710065, Shaanxi, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Haijuan%2BLi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Li%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AHaijuan%2BLi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-9428-6190" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Haijuan Li</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:haijuanli@xawl.edu.cn" class="" data-icon-position="" data-hide-link-title="0">haijuanli@xawl.edu.cn</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1249&amp;rft.epage=1261&amp;rft.atitle=Random%20Chromosome%20Partitioning%20in%20the%20Polyploid%20Bacterium%20Thermus%20thermophilus%20HB27&amp;rft.volume=9&amp;rft.issue=4&amp;rft.date=2019-04-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Li%2C+Haijuan"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/492611/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/492611/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/492611/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/492611/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/492611/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/492611/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/492611/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/492611/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/492611/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/492611/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/492611/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/492611/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/9/4/1249" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials And Methods</a></li><li class="odd"><a href="#sec-10" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-17" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/9/4/1249.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/9/4/1249.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/9/4/1249.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/9/4/1249.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
