<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/9/4/1189" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/9/4/1189.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/9/4/1189.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/9/4/1189.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/9/4/1189/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/9/4/1189.atom" />
<meta name="HW.pisa" content="ggg;9/4/1189" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Genomic Prediction of Autotetraploids; Influence of Relationship Matrices, Allele Dosage, and Continuous Genotyping Calls in Phenotype Prediction" />
<meta name="DC.Identifier" content="10.1534/g3.119.400059" />
<meta name="DC.Date" content="2019-04-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2019 de Bem Oliveira et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Relation" content="https://doi.org/10.25387/g3.7728365" />
<meta name="DC.Description" content="Estimation of allele dosage, using genomic data, in autopolyploids is challenging and current methods often result in the misclassification of genotypes. Some progress has been made when using SNP arrays, but the major challenge is when using next generation sequencing data. Here we compare the use of read depth as continuous parameterization with ploidy parameterizations in the context of genomic selection (GS). Additionally, different sources of information to build relationship matrices were compared. A real breeding population of the autotetraploid species blueberry ( Vaccinium corybosum ), composed of 1,847 individuals was phenotyped for eight yield and fruit quality traits over two years. Continuous genotypic based models performed as well as the best models. This approach also reduces the computational time and avoids problems associated with misclassification of genotypic classes when assigning dosage in polyploid species. This approach could be very valuable for species with higher ploidy levels or for emerging crops where ploidy is not well understood. To our knowledge, this work constitutes the first study of genomic selection in blueberry. Accuracies are encouraging for application of GS for blueberry breeding. GS could reduce the time for cultivar release by three years, increasing the genetic gain per cycle by 86% on average when compared to phenotypic selection, and 32% when compared with pedigree-based selection. Finally, the genotypic and phenotypic data used in this study are made available for comparative analysis of dosage calling and genomic selection prediction models in the context of autopolyploids." />
<meta name="DC.Contributor" content="Ivone de Bem Oliveira" />
<meta name="DC.Contributor" content="Marcio F. R. Resende" />
<meta name="DC.Contributor" content="Luis Felipe V. Ferrão" />
<meta name="DC.Contributor" content="Rodrigo R. Amadeu" />
<meta name="DC.Contributor" content="Jeffrey B. Endelman" />
<meta name="DC.Contributor" content="Matias Kirst" />
<meta name="DC.Contributor" content="Alexandre S. G. Coelho" />
<meta name="DC.Contributor" content="Patricio R. Munoz" />
<meta name="article:published_time" content="2019-04-01" />
<meta name="article:section" content="Genomic Prediction" />
<meta name="citation_title" content="Genomic Prediction of Autotetraploids; Influence of Relationship Matrices, Allele Dosage, and Continuous Genotyping Calls in Phenotype Prediction" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Estimation of allele dosage, using genomic data, in autopolyploids is challenging and current methods often result in the misclassification of genotypes. Some progress has been made when using SNP arrays, but the major challenge is when using next generation sequencing data. Here we compare the use of read depth as continuous parameterization with ploidy parameterizations in the context of genomic selection (GS). Additionally, different sources of information to build relationship matrices were compared. A real breeding population of the autotetraploid species blueberry (&lt;i&gt;Vaccinium corybosum&lt;/i&gt;), composed of 1,847 individuals was phenotyped for eight yield and fruit quality traits over two years. Continuous genotypic based models performed as well as the best models. This approach also reduces the computational time and avoids problems associated with misclassification of genotypic classes when assigning dosage in polyploid species. This approach could be very valuable for species with higher ploidy levels or for emerging crops where ploidy is not well understood. To our knowledge, this work constitutes the first study of genomic selection in blueberry. Accuracies are encouraging for application of GS for blueberry breeding. GS could reduce the time for cultivar release by three years, increasing the genetic gain per cycle by 86% on average when compared to phenotypic selection, and 32% when compared with pedigree-based selection. Finally, the genotypic and phenotypic data used in this study are made available for comparative analysis of dosage calling and genomic selection prediction models in the context of autopolyploids.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2019/04/01" />
<meta name="citation_mjid" content="ggg;9/4/1189" />
<meta name="citation_id" content="9/4/1189" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/9/4/1189" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/9/4/1189.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/9/4/1189.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/9/4/1189.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.119.400059" />
<meta name="citation_pmid" content="30782769" />
<meta name="citation_volume" content="9" />
<meta name="citation_issue" content="4" />
<meta name="citation_num_pages" content="10" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Genomic Prediction" />
<meta name="citation_firstpage" content="1189" />
<meta name="citation_lastpage" content="1198" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Ivone de Bem Oliveira" />
<meta name="citation_author_institution" content="Blueberry Breeding and Genomics Lab" />
<meta name="citation_author_institution" content="Plant Genetics and Genomics Lab, Agronomy College, Federal University of Goias, GO, Brazil, 74690-900" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-3723-9747" />
<meta name="citation_author" content="Marcio F. R. Resende" />
<meta name="citation_author_institution" content="Sweet Corn Genomics and Breeding, Horticultural Sciences Department, University of Florida, Gainesville, FL 32611" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-2367-0766" />
<meta name="citation_author" content="Luis Felipe V. Ferrão" />
<meta name="citation_author_institution" content="Blueberry Breeding and Genomics Lab" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-9655-4838" />
<meta name="citation_author" content="Rodrigo R. Amadeu" />
<meta name="citation_author_institution" content="Blueberry Breeding and Genomics Lab" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-5127-4448" />
<meta name="citation_author" content="Jeffrey B. Endelman" />
<meta name="citation_author_institution" content="Department of Horticulture, University of Wisconsin, Madison, WI 53706" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-0957-4337" />
<meta name="citation_author" content="Matias Kirst" />
<meta name="citation_author_institution" content="Forest Genomics Lab, School of Forestry Resources and Conservation, University of Florida, Gainesville, FL 32610" />
<meta name="citation_author" content="Alexandre S. G. Coelho" />
<meta name="citation_author_institution" content="Plant Genetics and Genomics Lab, Agronomy College, Federal University of Goias, GO, Brazil, 74690-900" />
<meta name="citation_author" content="Patricio R. Munoz" />
<meta name="citation_author_institution" content="Blueberry Breeding and Genomics Lab" />
<meta name="citation_author_email" content="p.munoz@ufl.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-8973-9351" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=K. L. Adams;citation_author=R. Cronn;citation_author=R. Percifield;citation_author=J. F. Wendel;citation_title=Genes duplicated by polyploidy show unequal contributions to the transcriptome and organ-specific reciprocal silencing.;citation_volume=100;citation_year=2003;citation_pmid=12665616;citation_doi=10.1073/pnas.0630618100" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=R. R. Amadeu;citation_author=C. Cellon;citation_author=J. W. Olmstead;citation_author=A. A. Garcia;citation_author=M. F. Resende;citation_title=AGHmatrix: R Package to construct relationship matrices for autotetraploid and diploid species: a blueberry example.;citation_volume=9;citation_year=2016;citation_doi=10.3835/plantgenome2016.01.0009" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=P. Annicchiarico;citation_author=N. Nazzicari;citation_author=X. Li;citation_author=Y. Wei;citation_author=L. Pecetti;citation_title=Accuracy of genomic selection for alfalfa biomass yield in different reference populations.;citation_pages=1020;citation_volume=16;citation_year=2015;citation_doi=10.1186/s12864-015-2212-y" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=B. H. Ashraf;citation_author=J. Jensen;citation_author=T. Asp;citation_author=L. L. Janss;citation_title=Association studies using family pools of outcrossing crops based on allele-frequency estimates from DNA sequencing.;citation_pages=1331-1341;citation_volume=127;citation_year=2014;citation_doi=10.1007/s00122-014-2300-4" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=B. H. Ashraf;citation_author=S. Byrne;citation_author=D. Fé;citation_author=A. Czaban;citation_author=T. Asp;citation_title=Estimating genomic heritabilities at the level of family-pool samples of perennial ryegrass using genotyping-by-sequencing.;citation_pages=45-52;citation_volume=129;citation_year=2016;citation_doi=10.1007/s00122-015-2607-9" />
<meta name="citation_reference" content="citation_journal_title=Mol. Breed.;citation_author=Y. Bian;citation_author=J. Ballington;citation_author=A. Raja;citation_author=C. Brouwer;citation_author=R. Reid;citation_title=Patterns of simple sequence repeats in cultivated blueberries (Vaccinium section Cyanococcus spp.) and their use in revealing genetic diversity and population structure.;citation_pages=675-689;citation_volume=34;citation_year=2014;citation_doi=10.1007/s11032-014-0066-7" />
<meta name="citation_reference" content="citation_journal_title=Developmental biology;citation_journal_abbrev=Dev Biol;citation_author=J. A. Birchler;citation_author=U. Bhadra;citation_author=M. P. Bhadra;citation_author=D. L. Auger;citation_title=Dosage-dependent gene regulation in multicellular eukaryotes: implications for dosage compensation, aneuploid syndromes, and quantitative traits.;citation_pages=275-288;citation_volume=234;citation_year=2001;citation_issue=2;citation_pmid=11396999;citation_doi=10.1006/dbio.2001.0262" />
<meta name="citation_reference" content="citation_journal_title=Front. Plant Sci.;citation_author=P. M. Bourke;citation_author=R. E. Voorrips;citation_author=R. G. Visser;citation_author=C. Maliepaard;citation_title=Tools for genetic studies in experimental populations of polyploids.;citation_pages=513;citation_volume=9;citation_year=2018;citation_doi=10.3389/fpls.2018.00513" />
<meta name="citation_reference" content="citation_journal_title=Am. Nat.;citation_author=M. Bulmer;citation_title=The effect of selection on genetic variability.;citation_pages=201-211;citation_volume=105;citation_year=1971;citation_doi=10.1086/282718" />
<meta name="citation_reference" content="citation_journal_title=Euphytica;citation_author=C. Cellon;citation_author=R. R. Amadeu;citation_author=J. W. Olmstead;citation_author=M. R. Mattia;citation_author=L. F. V. Ferrão;citation_title=Estimation of genetic parameters and prediction of breeding values in an autotetraploid blueberry breeding population with extensive pedigree data.;citation_pages=87;citation_volume=214;citation_year=2018;citation_doi=10.1007/s10681-018-2165-8" />
<meta name="citation_reference" content="citation_journal_title=Front. Plant Sci.;citation_author=F. Cericola;citation_author=I. Lenk;citation_author=D. Fè;citation_author=S. Byrne;citation_author=C. S. Jensen;citation_title=Optimized Use of Low-Depth Genotyping-by-Sequencing for Genomic Prediction Among Multi-Parental Family Pools and Single Plants in Perennial Ryegrass (Lolium perenne L.).;citation_pages=369;citation_volume=9;citation_year=2018;citation_doi=10.3389/fpls.2018.00369" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=L. Comai;citation_title=The advantages and disadvantages of being polyploid.;citation_pages=836-846;citation_volume=6;citation_year=2005;citation_issue=11;citation_pmid=16304599;citation_doi=10.1038/nrg1711" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=H. D. Daetwyler;citation_author=M. P. L. Calus;citation_author=R. Pong-Wong;citation_author=G. de los Campos;citation_author=J. M. Hickey;citation_title=Genomic Prediction in Animals and Plants: Simulation of Data, Validation, Reporting, and Benchmarking.;citation_volume=193;citation_year=2013;citation_pmid=23222650;citation_doi=10.1534/genetics.112.147983" />
<meta name="citation_reference" content="de los Campos, G., and P. Pérez-Rodríguez, 2016 BGLR: Bayesian Generalized Linear Regression. R package version 1.0.5. https://cran.r-project.org/package=BGLR." />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=G. de los Campos;citation_author=J. M. Hickey;citation_author=R. Pong-Wong;citation_author=H. D. Daetwyler;citation_author=M. P. Calus;citation_title=Whole-genome regression and prediction methods Applied to plant and animal breeding.;citation_volume=193;citation_year=2013;citation_pmid=22745228;citation_doi=10.1534/genetics.112.143313" />
<meta name="citation_reference" content="citation_journal_title=Trends Plant Sci.;citation_author=Z. A. Desta;citation_author=R. Ortiz;citation_title=Genomic selection: genome-wide prediction in plant improvement.;citation_pages=592-601;citation_volume=19;citation_year=2014;citation_pmid=24970707;citation_doi=10.1016/j.tplants.2014.05.006" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=F. Dufresne;citation_author=M. Stift;citation_author=R. Vergilino;citation_author=B. K. Mable;citation_title=Recent progress and challenges in population genetics of polyploid organisms: an overview of current state‐of‐the‐art molecular and statistical tools.;citation_pages=40-69;citation_volume=23;citation_year=2014;citation_doi=10.1111/mec.12581" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. B. Endelman;citation_author=C. A. S. Carley;citation_author=P. C. Bethke;citation_author=J. J. Coombs;citation_author=M. E. Clough;citation_title=Genetic variance partitioning and genome-wide prediction with allele dosage information in autotetraploid potato.;citation_volume=209;citation_year=2018;citation_pmid=29514860;citation_doi=10.1534/genetics.118.300685" />
<meta name="citation_reference" content="citation_journal_title=Front. Ecol. Evol.;citation_author=L. F. V. Ferrão;citation_author=J. Benevenuto;citation_author=I. de Bem Oliveira;citation_author=C. Cellon;citation_author=J. Olmstead;citation_title=Insights into the genetic basis of blueberry fruit-related traits using diploid and polyploid models in a GWAS context.;citation_pages=107;citation_volume=6;citation_year=2018;citation_doi=10.3389/fevo.2018.00107" />
<meta name="citation_reference" content="Garrison, E., G. and Marth, 2012 Haplotype-based variant detection from short-read sequencing. arXiv preprint arXiv:1207.3907." />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=G. Gorjanc;citation_author=P. Bijma;citation_author=J. M. Hickey;citation_title=Reliability of pedigree-based and genomic evaluations in selected populations.;citation_pages=65;citation_volume=47;citation_year=2015;citation_doi=10.1186/s12711-015-0145-1" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=M. Gouy;citation_author=Y. Rousselle;citation_author=D. Bastianelli;citation_author=P. Lecomte;citation_author=L. Bonnal;citation_title=Experimental assessment of the accuracy of genomic selection in sugarcane.;citation_pages=2575-2586;citation_volume=126;citation_year=2013;citation_doi=10.1007/s00122-013-2156-z" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=F. Grandke;citation_author=P. Singh;citation_author=H. C. Heuven;citation_author=J. R. De Haan;citation_author=D. Metzler;citation_title=Advantages of continuous genotype values over genotype classes for GWAS in higher polyploids: a comparative study in hexaploid chrysanthemum.;citation_pages=672;citation_volume=17;citation_year=2016;citation_doi=10.1186/s12864-016-2926-5" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=M. Guo;citation_author=D. Davis;citation_author=J. A. Birchler;citation_title=Dosage Effects on Gene Expression in a Maize Ploidy Series;citation_pages=1349-1355;citation_volume=142;citation_year=1996;citation_issue=4;citation_pmid=8846910" />
<meta name="citation_reference" content="citation_journal_title=Front. Plant Sci.;citation_author=X. Guo;citation_author=F. Cericola;citation_author=D. Fè;citation_author=M. G. Pedersen;citation_author=I. Lenk;citation_title=Genomic Prediction in Tetraploid Ryegrass Using Allele Frequencies Based on Genotyping by Sequencing.;citation_pages=1165;citation_volume=9;citation_year=2018;citation_doi=10.3389/fpls.2018.01165" />
<meta name="citation_reference" content="citation_journal_title=Gigascience;citation_author=V. Gupta;citation_author=A. D. Estrada;citation_author=I. Blakley;citation_author=R. Reid;citation_author=K. Patel;citation_title=RNA-Seq analysis and annotation of a draft blueberry genome assembly identifies candidate genes involved in fruit ripening, biosynthesis of bioactive compounds, and stage-specific alternative splicing.;citation_pages=5;citation_volume=4;citation_year=2015;citation_pmid=25830017;citation_doi=10.1186/s13742-015-0046-9" />
<meta name="citation_reference" content="citation_journal_title=Biometrics;citation_author=C. R. Henderson;citation_title=A simple method for computing the inverse of a numerator relationship matrix used in prediction of breeding values.;citation_pages=69-83;citation_volume=32;citation_year=1976;citation_doi=10.2307/2529339" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=P. Hieter;citation_author=T. Griffiths;citation_title=GENETICS:Polyploidy--More Is More or Less;citation_pages=210-211;citation_volume=285;citation_year=1999;citation_issue=5425;citation_pmid=10428719;citation_doi=10.1126/science.285.5425.210" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=T. Jombart;citation_author=I. Ahmed;citation_title=adegenet 1.3–1: new tools for the analysis of genome-wide SNP data.;citation_volume=27;citation_year=2011;citation_pmid=21926124;citation_doi=10.1093/bioinformatics/btr521" />
<meta name="citation_reference" content="citation_title=An introduction to genetic statistics;citation_author=O. Kempthorne;citation_year=1957" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=R. J. Kerr;citation_author=L. Li;citation_author=B. Tier;citation_author=G. W. Dutkowski;citation_author=T. A. McRae;citation_title=Use of the numerator relationship matrix in genetic analysis of autopolyploid species.;citation_pages=1271-1282;citation_volume=124;citation_year=2012;citation_issue=7;citation_pmid=22311370;citation_doi=10.1007/s00122-012-1785-y" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=W. P. Lee;citation_author=M. P. Stromberg;citation_author=A. Ward;citation_author=C. Stewart;citation_author=E. P. Garrison;citation_title=MOSAIK: a hash-based algorithm for accurate next-generation sequencing short-read mapping.;citation_pages=e90581;citation_volume=9;citation_year=2014;citation_pmid=24599324;citation_doi=10.1371/journal.pone.0090581" />
<meta name="citation_reference" content="citation_journal_title=J. Am. Pomol. Soc.;citation_author=P. Lyrene;citation_title=Development of highbush blueberry cultivars adapted to Florida.;citation_pages=79;citation_volume=56;citation_year=2002" />
<meta name="citation_reference" content="citation_journal_title=Plant Breed. Rev.;citation_author=P. Lyrene;citation_title=BREEDING SOUTHERN HIGHBUSH BLUEBERRIES;citation_pages=353-414;citation_volume=30;citation_year=2008;citation_doi=10.1002/9780470380130.ch8" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=T. H. E. Meuwissen;citation_author=B. J. Hayes;citation_author=M. E. Goddard;citation_title=Prediction of Total Genetic Value Using Genome-Wide Dense Marker Maps;citation_pages=1819-1829;citation_volume=157;citation_year=2001;citation_issue=4;citation_pmid=11290733;citation_doi=10.1093/nar/gkv1344" />
<meta name="citation_reference" content="citation_title=Plant Genotyping;citation_title=Quantitative SNP genotyping of polyploids with MassARRAY and other platforms;citation_author=M. Mollinari;citation_author=O. Serang;citation_pages=215-241;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_abbrev=J Anim Breed Genet;citation_author=W. M. Muir;citation_title=Comparison of genomic and traditional BLUP-estimated breeding value accuracy and selection response under alternative trait and genomic parameters.;citation_pages=342-355;citation_volume=124;citation_year=2007;citation_issue=6;citation_pmid=18076471;citation_doi=10.1111/j.1439-0388.2007.00700.x" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=P. R. Muñoz;citation_author=M. F. Resende;citation_author=D. A. Huber;citation_author=T. Quesada;citation_author=M. D. Resende;citation_title=Genomic relationship matrix for correcting pedigree errors in breeding populations: impact on genetic parameters and genomic selection accuracy.;citation_pages=1115-1123;citation_volume=54;citation_year=2014;citation_doi=10.2135/cropsci2012.12.0673" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=P. R. Muñoz;citation_author=M. F. Resende;citation_author=S. A. Gezan;citation_author=M. D. V. Resende;citation_author=G. de Los Campos;citation_title=Unraveling additive from nonadditive effects using genomic relationship matrices.;citation_volume=198;citation_year=2014;citation_pmid=25324160;citation_doi=10.1534/genetics.114.171322" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=M. Nyine;citation_author=B. Uwimana;citation_author=N. Blavet;citation_author=E. Hřibovà;citation_author=H. Vanrespaille;citation_title=Genomic prediction in a multiploid crop: genotype by environment interaction and allele dosage effects on predictive ability in banana.;citation_volume=11;citation_year=2018;citation_doi=10.3835/plantgenome2017.10.0090" />
<meta name="citation_reference" content="citation_journal_title=Trends in genetics :  TIG;citation_journal_abbrev=Trends Genet;citation_author=T. C. Osborn;citation_author=J. C. Pires;citation_author=J. A. Birchler;citation_author=D. L. Auger;citation_author=Z. J. Chen;citation_title=Understanding mechanisms of novel gene expression in polyploids.;citation_pages=141-147;citation_volume=19;citation_year=2003;citation_issue=3;citation_pmid=12615008;citation_doi=10.1016/S0168-9525(03)00015-5" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=P. Pérez;citation_author=G. de los Campos;citation_title=Genome-wide regression and prediction with the BGLR statistical package.;citation_volume=198;citation_year=2014;citation_pmid=25009151;citation_doi=10.1534/genetics.114.164442" />
<meta name="citation_reference" content="R Core Team, 2018  R: A language and environment for statistical computing. R Foundation for Statistical Computing. Vienna. Austria. ISBN 3–900051–07–0. URL http://www.R-project.org/." />
<meta name="citation_reference" content="citation_journal_title=New Phytologist;citation_journal_abbrev=New Phytologist;citation_author=M. F. R. Resende;citation_author=P. Muñoz;citation_author=J. J. Acosta;citation_author=G. F. Peter;citation_author=J. M. Davis;citation_title=Accelerating the domestication of trees using genomic selection: accuracy of prediction models across ages and environments.;citation_pages=617-624;citation_volume=193;citation_year=2012;citation_issue=3;citation_pmid=21973055;citation_doi=10.1111/j.1469-8137.2011.03895.x" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=R. T. Resende;citation_author=M. D. V. Resende;citation_author=F. F. Silva;citation_author=C. F. Azevedo;citation_author=E. K. Takahashi;citation_title=Assessing the expected response to genomic selection of individuals and families in Eucalyptus breeding with an additive-dominant model.;citation_pages=245-255;citation_volume=119;citation_year=2017;citation_doi=10.1038/hdy.2017.37" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. F. R. Resende;citation_author=P. Muñoz;citation_author=M. D. Resende;citation_author=D. J. Garrick;citation_author=L. R. Fernando;citation_title=Accuracy of genomic selection methods in a standard data set of loblolly pine (Pinus taeda L.).;citation_volume=190;citation_year=2012;citation_pmid=22271763;citation_doi=10.1534/genetics.111.137026" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=J. Schindelin;citation_author=I. Arganda-Carreras;citation_author=V. Frise;citation_author=E. Kaynig;citation_author=M. Longair;citation_title=Fiji: an open-source platform for biological-image analysis.;citation_pages=676-682;citation_volume=9;citation_year=2012;citation_issue=7;citation_pmid=22743772;citation_doi=10.1038/nmeth.2019" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=C. A. Schmitz Carley;citation_author=J. J. Coombs;citation_author=D. S. Douches;citation_author=P. C. Bethke;citation_author=J. P. Palta;citation_title=Automated tetraploid genotype calling by hierarchical clustering.;citation_pages=717-726;citation_volume=130;citation_year=2017;citation_doi=10.1007/s00122-016-2845-5" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=O. Serang;citation_author=M. Mollinari;citation_author=A. A. F. Garcia;citation_title=Efficient exact maximum a posteriori computation for bayesian SNP genotyping in polyploids.;citation_pages=e30906-e30906;citation_volume=7;citation_year=2012;citation_issue=2;citation_pmid=22363513;citation_doi=10.1371/journal.pone.0030906" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=A. T. Slater;citation_author=G. M. Wilson;citation_author=N. O. Cogan;citation_author=J. W. Forster;citation_author=B. J. Hayes;citation_title=Improving the analysis of low heritability complex traits for enhanced genetic gain in potato.;citation_pages=809-820;citation_volume=127;citation_year=2014;citation_doi=10.1007/s00122-013-2258-7" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=A. T. Slater;citation_author=N. O. Cogan;citation_author=J. W. Forster;citation_author=B. J. Hayes;citation_author=H. D. Daetwyler;citation_title=Improving genetic gain with genomic selection in autotetraploid potato.;citation_volume=9;citation_year=2016;citation_doi=10.3835/plantgenome2016.02.0021" />
<meta name="citation_reference" content="citation_journal_title=Am. J. Bot.;citation_author=D. E. Soltis;citation_author=C. J. Visger;citation_author=D. B. Marchant;citation_author=P. S. Soltis;citation_title=Polyploidy: pitfalls and paths to a paradigm.;citation_volume=103;citation_year=2016;citation_pmid=27234228;citation_doi=10.3732/ajb.1500501" />
<meta name="citation_reference" content="citation_journal_title=Taxon;citation_author=D. E. Soltis;citation_author=P. S. Soltis;citation_author=D. W. Schemske;citation_author=J. F. Hancock;citation_author=J. N. Thompson;citation_title=Autopolyploidy in angiosperms: have we grossly underestimated the number of species?;citation_pages=13-30;citation_volume=56;citation_year=2007;citation_doi=10.2307/25065732)" />
<meta name="citation_reference" content="citation_journal_title=Trends in Ecology &amp; Evolution;citation_journal_abbrev=Trends in Ecology &amp; Evolution;citation_author=D. E. Soltis;citation_author=P. S. Soltis;citation_title=Polyploidy: recurrent formation and genome evolution.;citation_pages=348-352;citation_volume=14;citation_year=1999;citation_issue=9;citation_pmid=10441308;citation_doi=10.1016/S0169-5347(99)01638-9" />
<meta name="citation_reference" content="citation_journal_title=J. R. Stat. Soc. Series B Stat. Methodol.;citation_author=D. J. Spiegelhalter;citation_author=N. G. Best;citation_author=B. P. Carlin;citation_author=A. Van Der Linde;citation_title=Bayesian measures of model complexity and fit.;citation_pages=583-639;citation_volume=64;citation_year=2002;citation_doi=10.1111/1467-9868.00353" />
<meta name="citation_reference" content="citation_journal_title=J. Syst. Evol.;citation_author=J. P. Spoelhof;citation_author=P. S. Soltis;citation_author=D. E. Soltis;citation_title=Pure polyploidy: closing the gaps in autopolyploid research.;citation_pages=340-352;citation_volume=55;citation_year=2017;citation_doi=10.1111/jse.12253" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=E. Sverrisdóttir;citation_author=S. Byrne;citation_author=E. H. R. Sundmark;citation_author=H. O. Johnsen;citation_author=H. G. Kirk;citation_title=Genomic prediction of starch content and chipping quality in tetraploid potato using genotyping-by-sequencing.;citation_pages=2091-2108;citation_volume=130;citation_year=2017;citation_doi=10.1007/s00122-017-2944-y" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=J. G. Uitdewilligen;citation_author=A. M. A. Wolters;citation_author=B. Bjorn;citation_author=T. J. Borm;citation_author=R. G. Visser;citation_title=A next-generation sequencing method for genotyping-by-sequencing of highly heterozygous autotetraploid potato.;citation_pages=e62355;citation_volume=8;citation_year=2013;citation_pmid=23667470;citation_doi=10.1371/journal.pone.0062355" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=P. M. VanRaden;citation_title=Efficient Methods to Compute Genomic Predictions;citation_pages=4414-4423;citation_volume=91;citation_year=2008;citation_issue=11;citation_pmid=18946147;citation_doi=10.3168/jds.2007-0980" />
<meta name="citation_reference" content="citation_journal_title=BMC Proc.;citation_author=Z. Zhang;citation_author=X. Ding;citation_author=J. Liu;citation_author=D. J. de Koning;citation_author=Q. Zhang;citation_title=Genomic selection for QTL-MAS data using a trait-specific relationship matrix.;citation_pages=S15;citation_volume=5;citation_year=2011;citation_doi=10.1186/1753-6561-5-S3-S15" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Genomic Prediction of Autotetraploids; Influence of Relationship Matrices, Allele Dosage, and Continuous Genotyping Calls in Phenotype Prediction" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4.cover-source.jpg" />
<meta name="twitter:description" content="Estimation of allele dosage, using genomic data, in autopolyploids is challenging and current methods often result in the misclassification of genotypes. Some progress has been made when using SNP arrays, but the major challenge is when using next generation sequencing data. Here we compare the use of read depth as continuous parameterization with ploidy parameterizations in the context of genomic selection (GS). Additionally, different sources of information to build relationship matrices were compared. A real breeding population of the autotetraploid species blueberry ( Vaccinium corybosum ), composed of 1,847 individuals was phenotyped for eight yield and fruit quality traits over two years. Continuous genotypic based models performed as well as the best models. This approach also reduces the computational time and avoids problems associated with misclassification of genotypic classes when assigning dosage in polyploid species. This approach could be very valuable for species with higher ploidy levels or for emerging crops where ploidy is not well understood. To our knowledge, this work constitutes the first study of genomic selection in blueberry. Accuracies are encouraging for application of GS for blueberry breeding. GS could reduce the time for cultivar release by three years, increasing the genetic gain per cycle by 86% on average when compared to phenotypic selection, and 32% when compared with pedigree-based selection. Finally, the genotypic and phenotypic data used in this study are made available for comparative analysis of dosage calling and genomic selection prediction models in the context of autopolyploids." />
<meta name="og-title" property="og:title" content="Genomic Prediction of Autotetraploids; Influence of Relationship Matrices, Allele Dosage, and Continuous Genotyping Calls in Phenotype Prediction" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/9/4/1189" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Estimation of allele dosage, using genomic data, in autopolyploids is challenging and current methods often result in the misclassification of genotypes. Some progress has been made when using SNP arrays, but the major challenge is when using next generation sequencing data. Here we compare the use of read depth as continuous parameterization with ploidy parameterizations in the context of genomic selection (GS). Additionally, different sources of information to build relationship matrices were compared. A real breeding population of the autotetraploid species blueberry ( Vaccinium corybosum ), composed of 1,847 individuals was phenotyped for eight yield and fruit quality traits over two years. Continuous genotypic based models performed as well as the best models. This approach also reduces the computational time and avoids problems associated with misclassification of genotypic classes when assigning dosage in polyploid species. This approach could be very valuable for species with higher ploidy levels or for emerging crops where ploidy is not well understood. To our knowledge, this work constitutes the first study of genomic selection in blueberry. Accuracies are encouraging for application of GS for blueberry breeding. GS could reduce the time for cultivar release by three years, increasing the genetic gain per cycle by 86% on average when compared to phenotypic selection, and 32% when compared with pedigree-based selection. Finally, the genotypic and phenotypic data used in this study are made available for comparative analysis of dosage calling and genomic selection prediction models in the context of autopolyploids." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4.cover-source.jpg" />
    <title>Genomic Prediction of Autotetraploids; Influence of Relationship Matrices, Allele Dosage, and Continuous Genotyping Calls in Phenotype Prediction | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"XTtHpA4_qytPsGRudZPYwV1Du-hb4-dWCLD-fJ2W8rs","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"492601","apath":"\/ggg\/9\/4\/1189.atom","pisa":"ggg;9\/4\/1189","ac":{"\/ggg\/9\/4\/1189.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/9\/4\/1189.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;9\/4\/1189"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;9\/4\/1189"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/9\/4\/1189":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022oagtYmaqpNYrWxHGsFXiKFE49f9NXLYFk++stcjxUS\\\\\\\/JmeD4IeAAJa3uSdMJliL2aw3DktcbyB+B5TsGs3LUiwj5DvPd8D\\\\\\\/UVbg70KRd0+9bd3y49no3VYo0y0+pigL7vTwT8olNSJw3sdgcl\\\\\\\/UqhRLXVPbxWlAOKJ8luMzQiZgkHf6ZLAHC6mhnMKMj8YMVEKJ+xpNLsIvD+FbRDoqDPTqwWe6Lw+4bpytCfqSqNURUn4HxMH88Dz5TTFUjrZCdg4yXlWpEhF9ZhXwjU93uhYFGYyUGjj7YuDZUW\\\\\\\/aK8x8dpQpTj5pudYD26wKEHKJ7Na9RdxGDbZA+Hws23HJqDDzFgvBIOVMr9+VurQ1yK0m9KG3bZaqNQVq9Co1LGU4dlK6yf8k\\\\\\\/ba\\\\\\\/MPS\\\\\\\/P34WWiwpxqcyc+mHWBap4TxiZQk57fGPR5U6TfeKwxCYYpzQzYrd9rj5mQeh84ahq3JYgYS+ERox8TbGfI2YP7XkczE42fy1O\\\\\\\/TYKevpmTwRBQeNnhU+TMSrF3lrG\\\\\\\/iYLisKS\\\\\\\/1y1kLcDT4dpFjWe1az3V8BeXzZEyRNuT+gBMJ1dY\\\\\\\/GWp6huEbRhEZ1\\\\\\\/86yY7sTrjAQJexsyCt3GNIHkVWsjj\\\\\\\/woRwxiIsHnFyHPAdT7hJzaO7u6xPT1AMSYsOvsCeAeI6z6jHX0qfZiakl7EFFKzpTMowugX5zD4brHFp0A4HpWc2HWTwWx0wUt+rBjFdGnyoxlMVCFaFub6z41IAj5gc4grz81LKlQQY\\\\\\\/zKc4a6Y4FS4i6erNyv4d2bUSkU27UsZlG2iQp5VIuPZ7lB4clRZXDLtf0ZcyVQKS\\\\\\\/mjGdGvlya2y+k2WQfZg24PAcLNEW0dTYARSpIjc+Akwvd4Y6bJRdAZIWEgKjis\\\\\\\/+E3n5Tlnns8LEmYvW2j6G8ZzYj+rsbU1TORqwHY6XvlU1qijsVPJp14CmdabijvBIdrDV4\\\\\\\/nohR8quKL3M6gYM6knpRRg\\\\\\\/M+3UH4noGHIF6xGaUBaxFZ09y+eTgEvwOq30VQPJRXIQgmaCfdPN41B+TzCDWcY28jM26+v+4rNs3u6bUxy\\\\\\\/3Ck\\\\\\\/5HcgTDWGyxEOdCfiDdMWyGmGr+XoGyMRJdmsX129RWGo0+YPDXkJKNfhvrKz+lGwWpntKSf9VIX5q158Nv2BR1ce3cwPSPPQXs25ZOy9ffEpQ7k8BSswr72c\\\\\\\/j59nYn1Qi3hyQxIvW5WmxBCxK0c3tFWylmr3i3QHf+wsk1HG00rXfZ3AJ88TBUY2ZhGaLI9o9RhCzaacQXQl5OQtEq\\\u0022,\\\u0022iv\\\u0022:\\\u0022oQtN2hhja5s15jo4\\\\\\\/ws0Jg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00226342226ca5bcdb81530f62c78d80431d22b08c3b6b1c86fb9fb8172db46b2949\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022suJ0C1L3R+Bl7Nw0E36kHZ9B1xZktiUk3pkNUwMscSLLGMAqa\\\\\\\/s97cEE38ROdFYYV3vw8EXk74+FL\\\\\\\/CN\\\\\\\/1iHFskXKwwXM8qrBKt9m5S71j7De+9GtLLkupyD26xUwwWq1yPEK8qrICXa3xH+BK3rfZdxH1Vmindh\\\\\\\/tBkvsSlSD63Ah1cUykWuPoOJuioynik824qpDrdDw9ZD+PAU+IH\\\\\\\/fXfQ4elVZS8iWSZf0qbXSMeNMnHTjlxSCU7Smz3+bEty4CnmXR6bIueIfRa3Bc7NXgd3nvgQhXB4qg2n0Y10L\\\\\\\/y2e1\\\\\\\/iWCJN4TTYL0xtE408jNg6aRLiJe3\\\\\\\/10r3qD6J8I6QltT\\\\\\\/i9vnl9lGeUKHAxVj0X4dQdV82wot9U8CEs73m11Wy4n5l0o4UwTBLbhFVu1gWWZWw8lkNMbMNhJ18gUQ\\\\\\\/QEFDpltx24+aJaqN8OSkzjjxCYQSuqJ3BqkG7ka1CfHmdnowt4sOJYJpLYPoDT+ReenGB97OP4d0T8Ju4ExlVSVphhgmu7f5dbtPlRrQucKFhADQuLs5cabylPcKRt1aF1OhG9mKR9weBZB5QKrcTFiV6NcHxws1Q4JUfyy5DDZxY0C95Toyubss9Qk0I97STIt4IYyi7I7Zxw1ooDROUjdUqRZWBUQEfhddDEV5WoD\\\\\\\/e9AVK47fiFwzmvlJD+8KCjBxn7md1wNghtpqz+F3TQCsCevCgMrRP4PVtJpSRq8hy\\\\\\\/hoSiZhDKJI0PJTEfAfaoa03tEN3TT\\\\\\\/d+tU1aDvXdPco1rR73ySYz5f\\\\\\\/HF8fEBhF0O1j3bQ5p25QbNjbav8BHhum5l9i0CzGja6BSeeZm4sepEqdi8VP\\\\\\\/PwAG2QHAzk6dxLPIfFPM0Txv6Ql1oknZFi0FoiC\\\\\\\/2joIjQAjH7jEbIgF0wfpuGBc5JlbSN+zKSOhfSnP29Do6M+IDz813SfmDnjtsPkwVgA9bm21ZzW9ke07gc6f24aT8G6hx6xyMvtb3L+sgxPpXw6ht1R0\\\\\\\/7zeBZoD9ZtpQX1hXkWF4rjSLXzNyR\\\\\\\/3VaEjokH\\\\\\\/hi0GntMmmpgfVEzCUVQRc0or\\\\\\\/DMg9BEGpdwmMlJeoP\\\\\\\/ycz4bGb\\\\\\\/43mfxbeysLmqwP2zmg31OhLKtRY6lBUw62guepzB+i6\\\\\\\/L4DzXEtP6TAmEZR\\\\\\\/JPqxekaGs1wRUm+3mlh51JVKGcKILgGrv7vHHa99YsgVx3BzDb67Zo0xj8Yaw3bs6B+3CJkyXWlKTt6ILb5945CpGBGjkpjEyqJZ+vZfGQbA\\\\\\\/Q\\\\\\\/kTth6Fdf8B0DUaXxnTnGEpPv4l9CyKNODa7Zq6NTVGa8Wb2PkEvvlRLMkdp6nW1xAmXvlV0rbDSATIQMaSVGxSdRiCgrGr4cpVERm2OSRM0A1V4q\\\\\\\/Px8JjDfMKYtHHHYW1+s93SlB+yxZvT7jPrbDlJuSKAhdXLbB02JLG52HWcNiVmJn2q0G5Ok258hVWYucvlZydai4HjwnXWT4w0oqsiqGZWs6KOYheyzp1puln4xrAQ8rwKa3dN0QUJx7vHMWkymzGarVnRG1o8ZdaGWHKr\\\\\\\/281XzhTjYpEWHoy+Dd0PfWhUDR5Xyo6geKGZkbraRoC90\\\\\\\/57ZQlsBPyQKNsx4mdU2oZ2sateVBl977lCk5jSmHNGwjRnvOOUEvSE1iBiiABSIh4t7\\\\\\\/2xvh5BR8WXpO8NRzhDkkEm97Am9Rcp03JIVjW\\\\\\\/+EkVsuZBTVGfezXnsHJwfMsE5N0uyVJ\\\\\\\/rxu4p\\\\\\\/FU5N4S6vILZG\\\\\\\/+ZOx45oyoNe+AslewqGT1DQKv\\\\\\\/fNAJnv52wP2aqiSmbotpC89UBWxbQzZoH1aQZf\\\\\\\/0XySjk5y8rRYMiWM2Gt5jYDKSb6zofSCafPUpCxKxceaGoE4+pyth+cD4KmwDY7fuZlcljpXfb4S0ZNP\\\\\\\/OFRbq4rn4uK+qC\\\\\\\/79MAQpwiySeiRs3SdPwwe0Dgj1pZfKTqNdAVPL0Q3K6v9CEzABXCzEkd6x1ujRhQc51\\\\\\\/cJ\\\\\\\/PIJNh2dLAwmDSxnbBXvogmOPQGIh9sNoKf5YNSEwEHMSam+M72XHu9hfF0lC14DQuX3kyDRu+90abnNiuHTU0b3eMt\\\\\\\/51gnPC0ClMV0DQr0IaEdx9PZ8kbYQbSV+cQrNk75j0w1pCG5meGIt3gj8sVEvwFfkIBxeGLBLB2K6g8z6UKPDfkRlKOadHfzCPT2YlPAFJ8RBTj1XjYAVtuHLoCSMDRBooO0InIsI4kldkEwulpxNTsuQ0b\\\\\\\/ahdn0kt5MPvDsyGb4f3cy89DHpdr1CKXXNWujAHmpbXINj+ApSKKHgUFOlI4unfSj9eonOdzMX+LAkTDojVtV7h0nhppFYvCBdIaDPnqoXDS25P18HuhVo80+RB0uzd7TvRwW2\\\\\\\/KMCepOLPQdxmUDlWnWqH3y7EwcaWQlDCktIHBtr37v9pjbcw51KHHOQ64lMYUxO7vIS9kQwVnD86DYyY32bc4U7uPngAo7IX8k\\\\\\\/3H0KI4JKxaq2i+\\\\\\\/8fyDI6KTtObvVyLTVOjdnTBfmDAi41jrIOFGhfhiqWfgq95xvR2sdVJCczOjm7SWDwkF+upX0VkYagk4yQIeBfEIJMYNIBK\\\\\\\/sDGLSeEQ0Mb9zoKgYBtDK3gZ3pBvip8yDobZKTLok6Hsk0q7KBgWD1fr9scOPDkPOEHkZ2AnvChjhhvHnROQoZcBk9\\\\\\\/pknhWpdg1lPkDPORKx6C5zOdyOL7uFHVMzXd4nUEUn1TfJfD+b2146jcmqwqPv3pGWtERivmsT16TYssWXbPlyTQ+DoLnAQzTIy5AU8o5+0yqdUFtbgFnt870jldyIS62pxw+5tr65lmSOR1Kdn4TP9aJWPHOqM4vS+Cl0KTr8ayX0g+hLS\\\\\\\/OAPrGKaEIkYsOofx7nNZRfrQCALSPteMtZoQPev7zu6+0LfncAX731ocpXRlsi+cHEokH1LITWrrulUk6mzuv5PL\\\\\\\/jSQcbiXP9tbHW+GiGMnOycwVTkJtdvDEEd42dU9t7CMQFjN4gXPFxoReTUgsYBFUTzawziBj4hGcveJGtOHhjiHpp\\\\\\\/aYBXpookiq7Z\\\\\\\/e622bAG1gIuyfH4iI2XMviW7gb8NQ9RHE7TntBa1pBlh2NnBBfDkZfLzobHXUwQt+VymhrpVfOQs+xQ5hkGcHbXGBPhfU4C779tPZeyuZPGRu+i6MfDRbW5kdHN96D3Q\\\\\\\/PBtwjhnO5JGWGAs2W+HJEdTvgirZUHpl6AjDW9NfAc66dx37TbseAE1yz4RxxYxIpBj6JMeMFQ\\\\\\\/vvRldfQQ5QsMLqfrQ3ApD17sWEX8RCI6g0pwlMbTXdwZEaMvv\\\\\\\/Bdkv9yFp7atCbi7FjUZRE3Yj64o7cjiZsPK9AOxa1VDBgLCiNiY5tW5qSD7dFdDDOwQchtvQonHLPGSLWJrbAsGy1cODTKgd343pVY7uyHEngHitbUrNkpO7nCitgByhQIXHj53mIQC2yEhHzVOtnOxiN9WCXXLaeUoJHqwpAXNappOl0FOo4\\\\\\\/dZFnNixCTDrHOg6ltkMbqHyhN1RzLi29AQXE1H2Ctev8PfS28O12gX4KPht0cGmQpuONS+YfWZk11ilpn1\\\\\\\/gHst1CNvMiNYHej\\\\\\\/S47EBZQUFZz6Qy2uKxftlF4hgeyRnxXckxojVxZz0w\\\\\\\/OkRs2X8d5qgl9mdkV5PN7YDprg5bmRT52faCXp2FzFVu2mFBUrSFQJGE0EUMan01uaCROAoAiFKr0ineVhxllRk4PRgQbHwCi3hUAzhN5SllzvBAGKP1LJFP+5M\\\\\\\/KHKTQgFf0z6BpkMXgHl+FveR0sJhB3wA5kF4WUILg96nMB3AecGL2x6gQXKWjmnJWkIu74fUPe1wearNjZ\\\\\\\/bQKyCTJleMYjXvipS7xIGpfhdsUS3COXOND5Eg\\\\\\\/dq2kE3FQMTqcSmOwVuiQBLFdUmXWdD+kUHsvaPAvky4aUOIj178lS2m5zXj4WxLuxPQT7ee7rF7+wBatGY6Wv+KoB6V9f2n6kKAjr+U8ZS1SgC1rDBBhlDA8TkMqu3X+tc+VUGiRTM0FavQ0a+fYTr4JsqbA4NuiULStyrB8IZo9By1GJ8gRVD2nsJAUTjO6I\\\\\\\/tOjghofHTmkv\\\\\\\/0KnTmhLSKaGoYyiSNP76cj7vko7Zd6PUnXr4EzOCXwT4A+7IaBzyNM\\\\\\\/wLbN3xupjhwHlt228RwrezoP6PrBsOPBAcUDWWPliIoT7zdyZbMX7X1XT5mInHhf4lA6j9EMsHDrSX5aiMdINlc4WkrL1yDv3WVWrLmne+\\\\\\\/LvuMl5mDj2wHPyXvbLJjZHRx2\\\\\\\/bdfQuZ5Bkif\\\\\\\/paj8fVsggB8kWYGe8Bm7UYARELVti0pNfPCrjiYiGkwPqJAUGf7\\\\\\\/+8\\\\\\\/nIu7t4F5y3Z0PgDoQTDAq2ICM8uLetYtsbrQ5EVmqb2Ql0vWfZcafWyK6kyx02G\\\\\\\/BlLPGYrdNMJGkbyN9JuHS3GuSgpS2LLP8U4KDXW+Q37MvITkatUZuIGf2nfykyRrY+On8lYetJpFlLtNQp7eZOAQw4C2w9gHujViKQFe62pjP6uow4SQTCAclwMf6sX3Kb8dV7lUWnuYUq7jx66jUC6oFmX2foruOkTrzufqcwI3rQ3G+CRtyU2JFp6FpIdMzYT7QYXGyJqPo1CB+lCv9jsABil+0Pbbs92Rz41INv0T\\\\\\\/rKmC8FMVje51i8IMLH8L\\\\\\\/B70g3qoFjRkVXMstaczYH2fVCtLGc5Wo1pWVakoGBDC3QsMl464JKAuA610elG7gLCZKGpEYzE2U\\\\\\\/Jq2nGadqYNQHy8dRQEBgIG5wqluodlVU6Rc6u3zq1bumxcR3DAkTsoxRmDfjYNfl5b+6qVGJDEP8voe2MbfXIoBZ\\\\\\\/CMPZuiSIqFtwYZmSCp3NsG6zOOAaBrVuVJkdfBUnMX61lrYV7y1cPLJBZV8Hy\\\\\\\/LrEUlU2erwQOKjW9fgbWopZ0UYjNgolV3e9U8+gYX6qfR\\\\\\\/oHWFuR1Sn0hUVBmw2oINchMtGhlNfXT7Y8yCt2SxAaQzQZUxi\\\\\\\/Ppt6ittUr0he3\\\\\\\/karV3jcgfFBYPTSWyU6h6k8tCualpaq\\\\\\\/awS8M27\\\\\\\/k6uhvkMuScZ5AoN0ilSV\\\\\\\/9eGZPZYuumxteHiiIEqjXZVQivqCQojWwxG7DRJP3d8agbdU153pi3R02QVqWibeBWT9gz+L8A2rdf473Wr7SmqeiVkh5txlAv0+3wXMJDEhS\\\\\\\/VQjC54hiZlCAbab3FKrHqMyGb0KysAlX+cRHRbaQoMJwSZ8XPbZim0zf6WbmWoIeStpNkeuzzj3PFiK5kFxJhcva5kWCYFtsYeGdE0Hz7fmBAlMb++YJAARiG05N24tQIo4+EDnnNYx29JNcii0vFr00qSfERlCrJkgMC77+OU9i1zXrA4\\\\\\\/IXBnB\\\\\\\/ptmKsXEG\\\\\\\/irjKaBjjb8A2Is2KtDScx7UaoCCE8X3uNNZl\\\\\\\/7LH7shCaxwWslN59Jgwb2pt3pBnYttQyI8OkKS1i4lEsNQvwVRV+aVhq90Ogk4bG9tZMMxAVqjV1lOvuTb1mT+NtvZhK822C6af53qx7bdeS1o1uSk9sUeO2Jluamj+2sqZng4YjSbocgTe33To6BFlxCmsjY3g6iBQUlg5qO+Y+\\\\\\\/Y2mq5oA+8jhG5bOitCF3aixBwhYbKPjvvY7f8oDX+by+UHvTPeiJ15asCxoHDi45IvSUIK7PCBUwFT03UK032kf2Lun8NEz3CynaEMDxHxUMkJpw+5Jg1Xfv6fS8yU2uf8cEKCAvBbtvfVw4vialzV2vUOmoaDtHYEEhKiisU00dAZWVsjXWSvFD3FAAgNYl3Kgy5CatOFxTeQOV2sBKriszt0kuqBWtPZeabgD1UufU8DYwwG4T1mt6O\\\\\\\/3hasgI5nVNoSP2+iWbFbt+QBeKNTqeKNufyTCfXhovoKQJsYeevmsGflFvQac2fwZlhwy8u\\\\\\\/Qo5zv4pRilSIlwWTbxg8gEdvO2quNUy8RES4hLroRoyfaJSRMzdhSHGtjyiN8UZeX5IOPcNJN6f6bKjiqKqKSW+jWkWhqyEq6hUc2OgvOqxhRtD0fw8QMhhmfNU9Qzi\\\\\\\/QJro4v7aXgoxlkza\\\\\\\/WNCOvzbQ6Wt7O63glKwLP57u2Z9t1naVfHYl47Lff\\\\\\\/tZTHOXQIYtbZ4ibObgpZBD3pYhGidULWjFJgDd2uaX7zeGcuVPBlIULW4Z5xowJSmp38ok5pL\\\\\\\/CPQOALUU4FvhocQLez9y4PCGK45UzvsxmXm9g76cAGbRC3Obc7MhvjvBQ6SjblMpqoIz5xYPD27PZ6jECbuifwEQEmKQhylG1qlcI+sc7hZyK\\\\\\\/\\\\\\\/WXQjZilryQoZq8ABDQGjUFKLFUsmZ77Lxauq3fLogrRJmhKNSjRpEmeD8WWkZiJx55PP6CrpWac4WhkxXkq7MB6kz+izW567IXal\\\\\\\/BRiQz5yCFhP5U9\\\\\\\/DFZkfiUxWUfPpVqfMB9MwButGxQ2RqlLGdBj\\\\\\\/ypeUj74S2KVvx+W7Cw2bTzQ8SSay4g3M0nXkfUW0eiojAn6BjuqZ0ZJ5cRQZGXQbo\\\\\\\/\\\\\\\/1uzq\\\\\\\/gCT8mtJb2hOE\\\\\\\/2\\\\\\\/+Pm+UiiwHePAl56sGxplKgou4aYZZNMXtgtIb\\\\\\\/0JLP4gwfxdOTz33+J4doTwSNjbKXq+y\\\\\\\/K9GEQ63Do6S4DPIPXRkUIRRG4BbTLFXJVVBgeZ5Kzc1n9uUreU3Nf1v9iFmosVe15mYWtOrEl7XtXbyjIZ\\\\\\\/BGViDesVJcT3WKemBcaaU5XD3tmrdvT370IkyHcYiEP\\\\\\\/s57dBITnuOIS8Imad0QT\\\\\\\/+K7Dsebe2iAk0zyCv9pwgrzLb7Yky6XCsJWN\\\\\\\/nR426myfMz2xxnYEteLpwX2m8XfhNOMsbmcSSkxE3pXEHV9Ne8m7uN+RgwzWtshdjKz\\\\\\\/XrEmIM73bhUDvCdfZF3fSFIhQ5ej7r5E3dVyCawT4XfmVEcyErNHjgEMr8wkS+Op4Tfvtsbdpd47PIAkt7BoTkiP5gtCzxXVwdjdmiO3MvtkX3W6vm5uBxxBa6P60U+cyfguwwXB1gtjFrAerBKVRGwz9HDPN0Ij+UXWvnOcxL3H5wNGdzlcSKtTP6YHqrSjGLQL6dPUtqwZF0cf3nNvoLheBTy\\\\\\\/LJLfFpJ6yT8Gt0RJ72hMRg5uHRqvy\\\\\\\/S448ygUoPndReXE0eGv1x49jjeATLi2FjqroRBfPy4SByeiBtZiLL5IprGwhGkDStYSvGgaXmRIiLvD70Kc12pUfh6ACWDtzZ78C6NXsiwPb6r86dIPcol0yXOMZjx\\\\\\\/576R\\\\\\\/wCoEObbrCunn92tdM20xzJZOeF7XSwHO1D5d5Tq8pkYv6rvgL7eAHaacxvcXdWxyP+0icoUUPb+bDN+Rw6Qn1knwgagyME2KUiF6Am29lzLdg+JHaPnIE02hvCLgmsTKSM1uhD8f1aNCJjdXNxqzE8BHzoWTQyqvfprMzHTLCdiKf81XfNYcS\\\\\\\/yJ3El1a9U+vsSRTrxpEZt5ZvOagLS61cYuKtlAox7Sv01iAneT76VeZm+0kH87lbeDYfWBSI3Luic4MKJuD0q\\\\\\\/Lc8rt4RYxMP228zu+n1g2sPRFUk6yhMOfX4r+GSW3ghyTx2kNI\\\\\\\/mEWvnf62QerbwX6m5edsBaAhKagPZNjZiPapZdxfGQQQ4Jbnuw96ySCWjG7NfrIWphvkBbFfQHZSthvRvNOK7+ndV6Kg3Jabebie9vmvLgSCdGKg3YkaRd1Au5yuR3n0iQ0fWK9IhwGuFeKZ6DkE1r+IG2jUTWn+eSo+URdRQvotrZM6X9Xh0JOsSdK+wEuSIjrd2Adi06ybTnzc7O9qFyoJpRRASP9alMFGepPXgbceJZNHjVctmUj1a7bxKWbhrojSdeQockvVM1ZObakAyn95tlsNy+yuHl1aIFBS6VfpSHGp1Z8G7EG1ShdYGA6HE+bxxLx5QvuY3ZlVSsoE1vorva2UMO\\\\\\\/wnNOllsak0Dga0uDz4pTnMwRUhdtCdLArQf831Me9aDH6Iv1wlAqWdoQbC9RxA+lLOk0DST28ve5\\\\\\\/k0voAEicbtScoUdBicrU1jfWHXOQvlAJGG8hVO0aec6fpmvfu0ncb3HvK2AjBOpjQ7Lo80\\\\\\\/PSKhM8EPGqSMRcQRPk9wEcjbCJadNhRuLU9xv8qZQ0RR4Eh8D1+EcyBl07wmMhZf5g5JHinJ2oWDJCoTGB14sxtQRjpiS3JVLVFQSemePbUiLVa2fTzS\\\\\\\/+6Vez8+VsftlfoTKmskcfb+zd+4cr9arOwvhyBSoQESV0FLNpHrYDAVT7kPvFAa4LeebLed4OG8IL5jleETQLLS5sc1mjoCqBr9zV6Ye01gvRQEzkbygkfTahNVoJ\\\\\\\/tQOHbFAuzXWWgWJ8yg59C2cQV3h\\\\\\\/xiWhNZRdqfz01ljfFohrelN3pAkOs1mlNTcU6t+cgVZS7\\\\\\\/UW9z2AOnkyb94FYHEfOq\\\\\\\/4w2T0xJ6cQKS8ZWrBns8pV0SqRDXfCfZHzQ0bLonJ6gTMnUb\\\\\\\/1Wb2tdHUX7wSWJsN6tN4Z0oL1LF9o8\\\\\\\/+EQNcueaWuwAb3iqQvxoe+5PIptjDaNLcyA17YezEs\\\\\\\/86YxTcgvzdAPfQmWYHp9afz7TqjdR12+UiGh5ee1BC9Rpu9cpwCi8MRyFDpp6DN5kJ0OYBEHaw\\\\\\\/Ujnx4TMx3Xg6wVxFpmAEsQhpckQB27voxO2axWYGj2eHS8xY1PJn3m4HtnhEAwOKTzBwd7p1j8Sk\\\\\\\/T25Y3LyL6gitUTWu3tTvTkGqHiBlPyagc66CXXCgf\\\\\\\/bffywUl5yWkKmey2N6MjXaLOSLOgVVPrYlu7wS02macVPl29JOEDttwQfr6pPY20kOkCN3ByLYPG9\\\\\\\/63WEFF2bI6xU\\\\\\\/FOexjJ2QG0GzExYasAZiRyFVrvDxFqPqGGsTR7mBQVOUWw2RUIj8F6rWOzA3DZDK4aAUkE7SpjjTpSsuGtm+sqq8m+Ae2R6xeipVqgchD2xFyWgHIdfauWuozljULpqfTMpaqsaolaC0HzTzb5Oj5rYE3tt\\\\\\\/\\\\\\\/zmmOM2tx+05msgPeiM\\\\\\\/lHpmFhOgS35iadU6GGS5YFtvwLWxCezzWNYY6EUcaVRglKysV+lQt7fdFdy2nskTjAOksok4OI09hW\\\\\\\/2JS8AfkNFS4e5UpW4Kc6CQpk21qeeSGJemxyhP6D7Lwdha2Q+1Wbh3Vn3t5vkp6hU3SsUX45pi2RLeUs3Tlk8TzG6n1Lv1hduGZJIixvA0JfZdyoBZUw2YQBmDbK6RXor+Ik4OyB8JeMgWgTPnpJNGnFZdlH2YzyOOMAtsLl1NSP3VAPyd48\\\\\\\/52IZ9O1\\\\\\\/jNDPOX86g6uQ64wF+5c6i3TxkO+D+OQUq+egveU6UQRrBFQfMTGBJO7bgobFckAcGy7XGrGCl56m66CI\\\\\\\/ZRcUD28PBQ9eHqE92BawsEavqU7QclVk2ujDF35WbrR\\\\\\\/2RSkrzcLb5tgyIcKBR3Ff0sepLTgTZgiJPKfb+4ExrBTbCUEk0IkXoJLFSfEiWXGgqjgQ3chGqzq4PiKJXnDs0HvhhkvByjZt5F+nWx7OcCU7lBX31ONonbpolkOjVdYkRBINxSO1ObFLna1O7O479SY5rDcJeHacQOPbznjwz\\\\\\\/fYdZQbes\\\\\\\/fVXJ0xRS8kSMmzIyiiKdHJaN0\\\\\\\/R2CgH926h5qp266LvENUlPvZHmHncz1jY+PiUMrSCJZRj\\\\\\\/dm0yBoHD8SvO\\\\\\\/I74Fu\\\\\\\/C5bQGX+UrrVVL0LKNIEB1l8y+wxT+f2e8SkCDWxWNX9sRUGFNjvl1gJrVdu6UuNAjisI40wwIilFP\\\\\\\/ejliUnOFE80fln\\\\\\\/+V4T2psnqxKLTxIlxIoSgbMQ+eaNcqTMUEL5bJTObElqOhdxE\\\\\\\/RqRbV2E80pidwPFejFCC3I7bmaDGK0oM87qThBl0Nrj93jZlaldQc\\\\\\\/8S4DHjEsGknyAx\\\\\\\/sgIavJ1NZ2Y\\\\\\\/amtR0viKegqKEE5504T2uJeUsFbHCW4sellONGmtQdv1121vUCYfrz+jRv4BzCNqMV3GkmYJBzB13p3NJCHi3HLZbIu0IcGjDe+GamHQrP\\\\\\\/xHsmWuQVvJqlJ+kaAmiz\\\\\\\/BEJG9nUblvVn\\\\\\\/dySbRCDfXfFGIOBvO4GExE3IiTodNOX7vgpzFn4dNgS1mDGmGQrrq144gCFCsbIMmfe7pqHz\\\\\\\/0wgTPnVzBg+nRbB133iTfbyOLtDzBCiMPr2rzzEkRMP4XYdZKAdfgpLYuNp0F1nQtIMmMXyZmTW\\\\\\\/jlHIt3XrNPR+Ef6cVfG3x0mbyHHdEq3c3XqcdJilRH12W9Zcc6NAqdp3DG1Nht082kh\\\\\\\/C9GzcXO\\\\\\\/mziDIbH5aMNj8ky5hlcVBKvIETwxQ4hoAGPzgpBiTy3YghTzH2jp\\\\\\\/wNBYP\\\\\\\/mxTviJH05MtiQjHHRnN1Qe60SP6CxF21mseMjUZGwOgCJLF+8rlqISLm50ymjCQeA\\\\\\\/gHUHgd9FhKd89D4WAY58G4QufJhynWgVhQAgoYyNv2XrjqMIAgP+hxe7orZpfloPJkW+MXnlxGTTiY004F44yeZXgdowPB5RGZlbMR1Vy3\\\\\\\/KB5ewMxwYKdoB5sPi4sswbMvD+q0tt3xW44MWkddfPkmXQhjoUwOYa2Dk8PpNPTTTHgPmWZ4eYpX67bw999F8mGI8lUjOQ\\\\\\\/\\\\\\\/r1xAMZnf7d69FLd9eN5R4jvFTICtqgqpztrLA35SAOWw1gCyZlcdMPfzg9BIpNzQfQFIaoMikT8DihAUcO4gi6mQzA0gOjJeiq3C0iCwckEL\\\\\\\/LrCYt6+Ze0I1aizm5i7YY4l5wH5YLYP6pByxrELLxEVCqdnq8ypqrdmgR9clDqj8j6+OrTvZUnACXZF4jfyzKya\\\\\\\/KnAJby89VgoF45l9wuXPenp5fy3XrHb1UmY\\\\\\\/I98I1BtrXtECUkV1NO71okr\\\\\\\/V7tQKVUbXmtb6HdFmSaefTGjrnBbte7M03tXtOc1XJrHo0Aj3vlDqWbXIXh27n4mcAGmzc6GENplnZy\\\\\\\/rXmEvo5uTGJvRpIlecw+WRiMhttsL2cIfyh5taz1MxUZ9DkTTgnQXi3E1w6NvxcW3sx\\\\\\\/G7DthcGGsf5Z0BD2MFgQhrhi2fazAzz\\\\\\\/Al\\\\\\\/OryZJHzCyH+E9F1M53fKl9fncrHuevMCgCaSgSoSX53IaNBipwCg\\\\\\\/Rr95AAlP7HtkLrWDikjb2GVnznQgBA3+0qHf8EnmTy\\\\\\\/9dlVqGvc4piIN\\\\\\\/4sBv4e3h5nAaTihDlXqAMIpeV4KouoQMXjuAKNQsEp5\\\\\\\/h6uBAP8tcRAmqLUEIjEPjDKJ5cx8CY6sbAjhiUx8pd2mLZ\\\\\\\/P5hgOnGc2p3yn8XS8\\\\\\\/a2aN9LoU2+85Gr\\\\\\\/gaC9fkGcTbM1akgEXO+BSneSRzMn+b+LeKbSOLmZHahXJOWp9YGArdapjMofH59FIIX+kjXakWhylA\\\\\\\/Vcx5eDtOuYU8kNjW9Sa5DuB86FqQK8ONkUXdmOfOMFs5dVoFDFnd0W3ytLe8QPbMKxaWuYYY5+9MpjGfht9MnIMDjNLLFXkkAoyFeQF\\\\\\\/padIAQJOh3QolTmZC7IOJOhyriXjkibkH2CFFEY5k0uuy8XZX6xOLPtu7c7jgM83RSPTbhNGpcidoXYwVHnjoe8t7K1EupRrQUccEmeNRrB3kGGk4Nwcs\\\\\\\/S3aLupx6wE1T6DiEnGcLsUelynAJDsGGmoMikgb7G+PJ+8F28zVYjRkITBbUXVO71tdgt0pOMeX6a\\\\\\\/Oy66QrLAFiAIZsLGD7kOkO3\\\\\\\/\\\\\\\/2VOFZnkhQ9httkBdBiCmYCf\\\\\\\/sVMagXxNuh0+1joHpWVOb2HItyMkEw3Y\\\\\\\/T1bscLngP+w1xiz1hkWYIVRVulhRHO5ILdok0UC5ZkxnkuD\\\\\\\/d94KxBQcusZz9Y\\\\\\\/dqnNb3nLSuuTk0v6Nxi+NVWhDxDkqZwAzQpkCMxRmweyw0N5ojOm42h2qsPbBuDYGdZggaUpReRSS6LXOr\\\\\\\/YTkGgmXvCwlApjKEtPrBAKffAoUnbibuUIS6tSAkpwYnRiUVxAnoEJTTX3raZ1iDt6tB2njX6GNr9fP5O+9ijmomX\\\\\\\/SnrQxfIHwrrx14YXHy8HpQVybxFzTZ2pW2wlo7V1wKf6H46NoRA8h88U9Pc0rGyq7Saj+WCBPJ9VU5CNzVYYGRG8wVsp9xatgJBw0V9Z1ZGMgvgwHHr5\\\\\\\/2WRhe8WAMqJAdCGMxgUo\\\\\\\/ZmdO9xPEfItPF\\\\\\\/dOpZyOYFGOFZGK52lZr4DiaJGl1cG9tDuwbBTERJ3FAwxS8nt4xoTQkwI4UJg4qY++8fv9GLH7CZw5Npq2jWrnGEh4HBMCiLrZo2d+WAf\\\\\\\/0re9VmwTYhGisNlHM60KbFdPeDZJYyOsCNp\\\\\\\/GXuRARFFogt\\\\\\\/TRSpLGBxKyX8ZGlJMKiD8pLC6UyIkW098BlyY9pucyWSR2NOkEG0TaXycNJtLdzI680sdAtOpaw7KTkSOrg0qHuuz0HAcV+XdwRzeSsAkqXvcswJDI5K5UP5eaHHLjgU+SCUJUu\\\\\\\/adD78tTNI\\\\\\\/cVrl\\\\\\\/SI0fFBmbaqgSbwL5Hm6i\\\\\\\/Q0RcpB\\\\\\\/whBa1cDMmsX3AauIKu8Y8v\\\\\\\/FZkacSofkOw8e38okf01ZxNmFOcfvWbplqkY1HN3H+rle+rExZeiCxSqm4Tf5xOjhPN9y6C79m4WQ2NgjA2eP4u0ZZnGFIJZ8sfAYriNC\\\\\\\/1LyV\\\\\\\/lz4Vh0Hl9G+Wq0nuQiLkV5RdCT8qW180t4oLX8ireitfJ3tgRumE4S49RjM6joZfEY8QjOZEY+\\\\\\\/gB1hhgyYKs\\\\\\\/n3nx3JasbnDarG26Wwn9\\\\\\\/KtCSiRCcNu8VmBI2syRhTR1iiV183diHvTBusP7zkasVSpTVUJdjYk1kUT43Ce9XoK2\\\\\\\/QXtp2KkhFaq+mn7fQ+\\\\\\\/IkFf9pdBykgIEB3eD\\\\\\\/sON4EElF2UQdCFc5yQk2qbtF8yZa\\\\\\\/uZsG3L\\\\\\\/vlcmu7fncfI+taxtBGX9+O9aiv8QytK5he3HZIMjz8ef8DxMx5JRubLUXIwrk9wf66xlycXe0H2Bh3CcYZ6lsEEIGhpgqmGF7XmtNYD0MWP5SWZRARmHMp9dqwhDX4snoMvXuhfQjFWVNDweKMVCqRFLgN7RSwMAIBeD7DFm3gJdzd+GZs5h2SVH4q3\\\\\\\/x5zR4BY2p7FstkXCSHoNF2JrQcI0tGBaEZai1ROjx2EMZkzjroFuvdw\\\\\\\/bYkJxiLFYL+tc1Dej3yEpX3j99CmxxLuZzGp0HfM4otLp5WLq3RhuAfj+c4uQXvseZATAR5DZacTjPTDWh3TuowfLYPeJigF25DCYoc624L4tUShuYLZBE0uy6+VbuNcPGg5+7zzQoJlV5cfXO4TSy+EUdo48NzI0WTXrPEdmj+pV1EVhMGWN+FOVtKBr7W3c5LYFKtxFEvSrzlVFW\\\\\\\/IbTgNHGRfU9j12JbSsjZyYFV9GSpbip4kYwLS0tabZHBZljOlcXDaeKvQmuY2lhIm0X\\\\\\\/Vpbfsihiq7vTB4wfelcSn8mJM2tVeML8i5ephC+tNrc0tsZPMP7ausC4lbGstLzPNh1em6q6w+R+ZvAXBXNCS+TQTXo8Oozja3zqQ+YCr+HiOvsra+WGcWplVq46N76PLTNGGXjo3vy+a5sVAwjYCoWBGLbwahx2AFxfMH\\\\\\\/aKY2UXzbN\\\\\\\/inrceaDkcZK6pV3MeuB28U7T+5B+MnSK79rHgs2psr9+w9yDkDKCsLYrgbZIZs8nJmZcYHJNsQLQBcaogQ18hVehf1kjdbLYnJgJYSu\\\\\\\/N6OFUmdT38fKuYt+1J0IbY6r6NpbGR85rI6YasOR8RZEIZWxBkNAoDqDwDbZ1AqMLOJdiEvVKIdhKy8u\\\\\\\/qQqbdlrLA3nONVKj3fBKSTCM0uXHPqumaUi6\\\\\\\/fsBFeAGsLWpUUrwv0wPM\\\\\\\/bWmFReFmzkaHp3omOsckVh6vzEwwHZ2A+IlwAockw+poNJXOuMhdVZZSlepJSeBBRijInoyskMIrRKiYOz09MmwDxz+dESs8z93sZwdO6BX66fThWAcB1GIbTz3mZzG7leZdWi1\\\\\\\/DT12vLksWc\\\\\\\/v4sS68qbRnDGgfsz1zGGFGinB0gYnN8EUypT68WASDZVPr2bzR05pTGMnLdcGUVYYkQPJPQkttWrZkStOPQVfPkeAVAjwgSjATKR1ZQUd9MiiYSm2iX0CvWPjy7uW2CjtbxY6PEdKZ5lXj6lB+jWCW6e2sX9mONuTlF28MHxfLFz+30pZH4sidFQcY2hIy7JR3NGF9Z\\\\\\\/QGBuBm6UIDSgOUSnjYTofVgx2ogxU6Xy6DS4DhCTboz+xHPQ8h6WQ2b6ezkEp2A1lT8wXnWc9PMZK+aOadc9\\\\\\\/9qRgULCwdEgYpY+k+JZ2zrRSHWH\\\\\\\/zQL+RTce6bXvwsx0pSfaqHTinnHq6tqwxpDKVOjzfYvztaZeFtcsxBJJfQCJs4qU6\\\\\\\/XbRePl5RDzBSRlTiixq2lw\\\\\\\/UkWleRS1vMFt\\\\\\\/gVd+izCIY2EEWaUYHbK6kx3EL0BIlvGBh6tX2ZqjQDx5FMoKO82raYwzPMvmM2aIQjioMqZr2fG\\\\\\\/ttuPt8gYoAfEa9n5t3rwKynD2Y2G4eTDz04rBTrr\\\\\\\/bu8doBK4lbqU+Wy38svbEcLEvXtDHbsZgwwRD0ENHeUtGHonhXcYft7Rze\\\\\\\/sQeBaHzRYK5RodeI31pijA5Wze3cTL5+58Cp0U+2SVl+4qSY6rKrYiPMy7DJ3w5BkcWB+2wRDnBaPZMe4DmN0m+I0zYia4lxz7Ohrtl11CfwTSiOgjmZE1jwSXB6QRz8bCkaC+b2pfhLC\\\\\\\/8o5BKsV0f429PNyBSPcjmaluJyeOlhA0vBeR9kWKU02XN3CYFR6LFCqzfdKadpFulUShTboflXADYZEcs1kq6m\\\\\\\/0eLSRaA1txCkH3SnsNnnAmlYARIOi3QSucQ9f9ss0PQs8vaadHIfUplnyf5E7dZYha\\\\\\\/QODyiJC7dn\\\\\\\/3twUout4iBtJ7Yp3U8FMI+MdL+gxTaJHXX1NYnHcGSOezlc\\\\\\\/W56eY9nPmWEvsJ5FSD+RaHCcqhSSVt453KuqY03Fbs0O58VZPcoM3gB5q5uTy4lLHyb90GdxZ3ddNzpV71BnbL4cUyB3UBKr1yxQFEytePbaXJFApHG0tKOugWoEolMvflYlGRZ4HLvQUckoCkDsPZLLr1ImMaVh2yoAKYamBF8HPIFwjqHGJtfCTtaezYrXZMAZi9LiEbTcWqQ520bDLyM6i9kN3ifQ7n3JL8HuUUlTtV7+KiGIMmywWZZWesmllg2Kty8TSVSILOYhfG8A8oBUpZb1\\\\\\\/XccXBkLg7JUoWMy16+tivKgc43bjDhSb2ZTuXbRni1+v8JKY081wGBD5WSduZHqAleM0CwNrZEZ1N\\\\\\\/GlcArvICl9+ySHh+3xhqAIkC4lQsmF2TYnmjgN5yq8SBVO063G4TGWGOw\\\\\\\/CC8cc\\\\\\\/JgAs7bZqlA8JPmRz5ZDOMuu547eHnzHNtCaLQ1qVwjkbK9kb7+3w5MD3o40X0rZQvPIN\\\\\\\/0exsVqLMRuYyzBbHudlDJ\\\\\\\/OLV89dFpAfmHa3H9kXMZ2f9R+FB9KsUzZ+CSI8q+BKxpbcvqrsXcZw2oQvRrOV2Z6PO02sYrPKUqooAk0lwLiFy7LPVM6bdcr9OdKmUSo08LYfkwSlHzwsGXBM\\\\\\\/MV1KM8\\\\\\\/8+8G8ZlRJ3ECb1p03B4JJHEzhcovw\\\\\\\/1yrTyqoJC9EUdz1P38brm6m5MZzxzHENdqFiau+QiqD2Z4Jt62wYdmfG\\\\\\\/wbeDQVHmoyIde7Ya0QQfermu6F4Lja6jsOVzIaB1AchCzFKrvnRfIjUirYwB0wzexYFOhrOY958fr9OURMVuuI1F3s0ljViL+bEV8bS0LsVutp7Cgoi50rEz22932Ei5oCHM5\\\\\\\/wG7e8YXBjFxXf0Gx4GfmARDzZimSjG39CgovUO2pOcmIbxc2+VNc0\\\\\\\/heKXfVAApoQG18IRnO2+GQBLejBNBjdMSUr6GUAYGUVWBOaCTvSF1yeHzQKbIN0VmYnj31+bVCrZw3vUu6DScZ7UaMt9eXAIO1EOu18EfJgfZdV5CnbrYsBDf+4GFIPw9pyAlWcCXURAocFqs93vLyWjj4DvIh4K8rIuEkCzk3lvf3Il7Xtwxs6zXaMFlfSskAhMCWQujGdkVyE1ciIF3XiSBdKEMMxnT6IEr3UXQ5revjtKHLcysYiDFIrmrbJw5\\\\\\\/IHgSW3vZx0PucjlRtyICCTXgzyH+EWqEGxfIN4vOnVxm2YJck5\\\\\\\/BZ49DGdZZykGaLsOk1BPD3FLXhAKXrvTHWMp7kMMcMDjwgwJ5J\\\\\\\/jaiZ3WDhJeAn1ZTw91qUGrwOnjlBTbN7p\\\\\\\/LWJK85zjOwHTNXzxlcq+Q6roAA25yx4V9bNI9SPu3l3MvMj1nOAdfVVlsrnOxH3GDhAF3LoCWmeys7o2F9UcJAn\\\\\\\/HuYDwU7Hqoq7\\\\\\\/WFp7tUqnTDxojrR4WcQIynP2LPkCkFN25kWEOdXbkYBBbLgbXlEHF4ADKiLvR4GzVFCqSXKSQhIwJ7QS1QZL\\\\\\\/hg5wIN9tdfMtCYr8MXtnu9FJrCl7WJUmNaGh62eDnyJei\\\\\\\/H+pwQMdmNhGZ9M8KO5KjiMmI93rsSrrhtZqiKP0jOrDgZSb6mYh0ni0amEKKZmaBfGfkM14Acvrkzm8ildEQZmMMRU6bvTeQe+K2jRTyISW8ZU4wrLt9w2yofDVKVQzx\\\\\\\/9RLH2anQgLrhjnRHXG\\\\\\\/mOqo1sDCh6PPUEgCFPdrGF7avz\\\\\\\/N+5jpz1tWk\\\\\\\/HKS7U7e+Ela9c6CEe5OEyAu6YGPb+wH03RZGxXN3Jo1lpImB1wzMnkKuvSGByurFR5aTXfNM8Bs6PDWQ4rwBv98BBBrJul2n8PfV0++Vg4Bzjiuu6Dcm\\\\\\\/FdfY7PpwBFSMIXYGm0N7wzWNFcIlV7UFCehZklCiD\\\\\\\/7NW0WnpLOm9x9DdhZFw0erqPnBImSqvcLEJi5VqQuhh2sOmW196LV+SfV2Myw89RwBs6vuyttFCw0nfUL6FV29trpksLgU2l4yaeS+89MT484Hhxabnry6+nOPIQXQDNY8vCP913W\\\\\\\/Hvu2ZaQprIFSRVkpvGy783B9AVXCgcK7Kq3SLnWbBA8v6e9uFX80daUVPD6ysD0y7RyUXInrFXjRLGp+IE7S5970ZH8TN\\\\\\\/wiZjUxtvjlGacekwo0MXUIxdAJMrmynMHJIyKHskeifJSAGo0dN9R+cg9Nn+DoqWECMHvTR+0AJkg9hhwp\\\\\\\/d9AE8mGdgdBhSXKpuCoSiRLFDgmnD9ZucYdiPrjXgWz2y5IRrMyTHyNswMa6IRsm0VOFduMthc0q\\\\\\\/RCWpy64NdIc\\\\\\\/0vO9G1fyP6UvGBjOCJk13zK6P3ujBZ\\\\\\\/1N+FbqbY9dlatPqnWT+dY+WRx6gUTUi2mxiyiZCMa5adI\\\\\\\/r3obeZ034ClOpiwfQmon3hoIzKrR9QXxN2JqoExoQ+Yo+m4SDfuaAj2FjjTGUg7jOyCpwjBz7qvGpmvS\\\\\\\/vf0Z6efUfz3KXYWw6DcodcFg2K3J3GXyFvZEl0dJSJgTUCb9wQJeY6VkBeO91yODV0QlxlS80WMKSAr4784U2EykLtqCpW5d5opvmnBwg9DwNljXPJDWEx9fCrsDJLQ8PPQg6KxewCa7RJ\\\\\\\/LGkuo\\\\\\\/FYJ47jEFKBp8T4J1n6ABYm0hLmakMKuv2NP1SLvVanGcCjqVyX619zrU9MPvTenJps3trXeue92Fjjt\\\\\\\/eTsz+sW90pkwdSysofysleT4wbiaT5yO+1tGwSzWKdvDIJP21TgG3WquX499eSZ9GWAv3kOglP0rk6OMoW6nm01SqXz1Nmts2X2iLUiXfMW6Dr7hdV6D715VMBbOOQWjCSOccVUOaZX8XHT9+iDcSbOPV3MN8KT1ZDeQLhTddjjo3GBnaoB4cLZ6tA+bNUZHKANW0mATrUvmEKLqgAJJsnW+4xGC9MAIRkYBwzKesA5WEFilyOwEUudHETyPJn6AcWhGkn5pZX2s29AuVBAT632s5dge1rfUI8qQwdD2N6kfCFmwF7XER6xDBCE6\\\\\\\/KybDa+aQ0Hm9dgUQ0CpcTudA6OrdbaYgeYuIP2gDrhOJs1xlmlwrofuWYrxKjKG1W6UexdkdzAZsayWDxZ+Fb1F\\\\\\\/BpLF6\\\\\\\/wbxv3LG1NB8hE0SZ+\\\\\\\/khJX9JCTB+At5WUH82jc3dCEkP3Y7JDUQJqaxK\\\\\\\/y8Zow\\\\\\\/kAUNnZlXO0LAj7UJtnmtBgKD73HHw3bo\\\\\\\/DEvG+dk9wkMpvqjD7XaDg1qFzhHOilEnp6RWAxO+h3UBd1cIRyxtLAiufoJF\\\\\\\/Vxw7+\\\\\\\/Gbb2aSHBYa7SjkMyBF\\\\\\\/Xjo+7P\\\\\\\/4jts2yt7Dgl6N1jcwL185aFZ0OR7OcD\\\\\\\/yZUai0scMvegHpoKWCQJHbWJX7H1bzah2S0lkfLM6\\\\\\\/hrhPwj4SdFiOdLV5JcageZC+bAIhAN8YZLZUF8PMWaSCnyXr+E4IvgxBLw2nEOyjKjSWMtghkItvl76nugU3QwNu3u7fz\\\\\\\/3mtc79CInrloJ2gQNsO5UKJny++x+oRvAuEmdBfl4zFavDLexB8g2AP5IwN6FrrmUp+kijXhY8uAQLXJPQggqNg1uYfbZhhDMOu55cye6OWMntgfsBL5rjTnBLLXSQCHa94L96jt+GGTEgXgje4+z7M9ZrV4oB+zLVmxLrCtyrFBmn4gqKRGDVbDaFXQS8xQ6HyPpEYrmsGbVikescnmkeg4T2Qmi1hr+n2jBXsEj3425iqBW9\\\\\\\/L12bL8gxUYbYB3SR5Y1t7qdcSaUjDLguBKOzYqFLuY\\\\\\\/PyTN06fVIqhOy4Cey+Rl2FCjcdNDsfpTQrgfkykGw5KJ+b2auHKf\\\\\\\/avu7qhlZENKLAqPJ7cpijYv7EKoS3cXre4IhM8Gq1CleZF11Kh6xWArsktHe4tugHXWZJm13AR4gBshfknS3zqswnxekFBrSUZXKTsGCH+QuwlmmjBXWDn7ju8O64oodTvX+KarRkxD9rROrSf6XGnIH4OgxUN2pA0MuxEmQSGTfNFw+uHvCcb1MClGpvDHM301hlxLDRMCbzwVl1b\\\\\\\/7OfnqIDdabRRGVJ3ERRau0DS2hRUtBUKz59iDe1EbHEKJJ+7arAXINsKTvaoC1vjuqXp08+ITu7Ai\\\\\\\/+JgkAQ0lPKTnOBXJQR2c9xsKIYBA02bsabtQo1vPVsWLSiIcy4NvUbWbwoas2L1R9WKaFr5jvdrmjvXeSMskjB2Vn\\\\\\\/PV726vlyiV8ddgWus9v4i1A9CRsZlNZUwMCZVG2za+p8yKW86BdpJsP+L9YoeV5LudxY3Z23jq+9fPQkDz9Xc0unRPDmyeydrrnZ8STBJP9qYm4mDGlTrcbaW7k6z2JJuL9xU2mvNu5H+fQCMiYIKrqIbJBEoMLvw51AZYTKq1\\\\\\\/aiDujCvgxYuT91Jx5oAQGU2Suq14ODLcQr42VrrY+2sB97HFaL1F+LhPQ4fB3xrKeST3s1M8dSNYf+rTXvbdGrJLbO\\\\\\\/RIaWe\\\\\\\/km40RSH05Xr57gz78xZRow5rRmXKRY4Tv12Fc92VG\\\\\\\/X+6B4TQfGuJVl7Xq4CJvhrmMpSdYU7p9zIZ\\\\\\\/tf40bkLUyqZM9PaOMmhpgJhYclELUJFM5azKBQZHHRPk1yfqlMkNARl8J8Ae1IoxJJu0a8TzW6QH2GSHGmR\\\\\\\/kmCXUts9lbedyGh92ZfQO+Q77HjBtedV28MhefjZR2yf4pke7Jnj7OSdeLpXYrlMpiz3u8Ne2OgZbhGWs3Yfd1H8aPku98UNH7eGlWMynNfl6kDCfw5mFeOcMyK8t0tUIa+lZkbj4pxzz1iqbO+vUrHKt2YAIGnHkjcGiuZDsK6n63DJVA6JpSwDIeS98Dprp7TNRctCg9C0WIZePLm4kKc1kGWqWXLB4xA7t356XkYuDapGtFAWKh4H6GFjXTyhADmgK93ujiCzTRo68tT44NaoctLciNmbcWvxp2ZPMBJ0m0IhsKW2E2p+9kxfGkB0jstJg7oaklfI4RvGlDzwWBBPzbiAh6cO+h\\\\\\\/8Jb\\\\\\\/LwKhvDsbxqRCKpmOBLmH8hJ\\\\\\\/e+apZnl3UnaGEy+hOWqcDFgAIAM6UPzuZHFtXBh6roGyj7WXqMBNBeIxD\\\\\\\/yomNJyagvPdA2bbir8BT56RfgpiDkfEYDqTB3GTRFDznNo4jE\\\\\\\/vp3GA8rNDnNENLIR9uTI6uQBjcanKz2i1r6PgKxkBmqONnFiQGisNtEE64rxvK0U073BjNK3xg00tUpWnGvVVwztiBYIwkkoVCY\\\\\\\/05VRCcP4tUuABHaZtCGnfXe5vnD0pALZYCypUBWDA\\\\\\\/2DZ+TZD9pxAl+d4WEOnh24AwgTDqr0ZpdU+3oxxa2y3YEy0EnVQEz4pcWhx4ts3k7jUryXFvg9hXi4ffO1xGujS7o6Dk\\\\\\\/ROYOYzgHBpSkARYoC8RoXp8qh0FCGtCNiSThhnNNvc0NteAJ4l0bvk6lNi651md2DfyhAhmlw00Y\\\\\\\/\\\\\\\/2Ez9sQlADC1z9qUmrtCfHETFcyoFpgGJWFm\\\\\\\/eaxVH3rTyoea4GYNIg+OmtJHKRZF0CeE6SVVRiH9bv4TumVyctEpVGUj5hF29zoIDdX3sxQl9f+Ljke2G\\\\\\\/q0QRtfRFOumetVfYapyHgIx5PQBZkb0uqZBYIuKFa7vmR5pwzP3OxldHSqgPU7T1l5XoqTb0jN4dB8GhEBt06THDSk6Ej\\\\\\\/Gd8vXTXMsUtKU\\\\\\\/CDyfN5UFwlbq7EG1g8fyprvJUrfvNXQZ4+UuBOmEddfofVdyPe\\\\\\\/K\\\\\\\/w5V4g0SYsX\\\\\\\/DtNwPY40tivf6gUefRiAm+h1uxn5sAeilOLrLi1iNjTZ3PmnPic28O\\\\\\\/EPoT4QJxiPebXlOdWrjmoBT5\\\\\\\/4qshmZlwpUePOPk7A0sd01iuL4zpECu5uzMxHQ88Arfa930bcfiAxtPfOEwBGO9ycXY5eIXwFYyKoYbgH9C0R+6nWPPWwGH9mTXwxzxTNXOmkXqQN0fm8DPbALmb3eO6k0pkOZ\\\\\\\/3j2T4HdIk7DVpJpCCDi1CqGM3l5HEQJA3\\\\\\\/MyLLhMpVWFNg3nqnE0xLoiGCBvEx+xRibeGslxy6set8uluWbWQomblqToHGzOlAU4foOD7710UqXxJlnkS6LtmNoBYMbKcZ8dKr8QT907bWiUmy6N2uE3eD\\\\\\\/jdrNpOjMIF1q4K7YElfYeAYLzn6cU7QNazjXsXq90mNP3Ns\\\\\\\/FRkqiSTLCGiun6QNy6BQAfYEVd1es+3FrmS024I1wpfGLA+AjBItnuKyQ\\\\\\\/4kGs\\\\\\\/9GtjwxgWYL3f186hu\\\\\\\/r+zMhHLf1tvb8WXKliEI3GBSlmR2ikXABr6vGalvKQ4\\\\\\\/cPe0rWd7Tc3EeRF9a5cpOT6bvlJvaBvWOUWNZxAUOLLgIK6RNHzIQuXZQuq8teGspU4jAakVMvNUEIDdawYszNQthRndY1kY30yFRRjZlp9qtOKW5T91c3NH2xIJ5UKJQM5FBBW5k2xzP4BpMpbYuXAFQEqmlUfY0Jzu5URvy0HNlY3pCNqDxCSHl+vd49lI\\\\\\\/baFMVWGNLQW0YnL22EPXSWtnrQbV+SYGMEw2JBOy\\\\\\\/xsqfW\\\\\\\/xdAo6qXbqL4+1lUXvLED60DljjfKjhcKX+HCNoGI\\\\\\\/mW89sPeFX\\\\\\\/eRF6CF5Cj3aGdPysMKyrwTwR6FLAVsIPm5VAyjCwrbSfHSmEeUM0X9is72G7dxeEcDiLT1ZGgYC6Z+KMXowrW2ajIA7\\\\\\\/4R3f52fYP6URqGd4MJZDzYUNNeh2OQEFhDdOymHBdRxn7Xy+adKbM4fiKaKfu89hzJG2ZiwhsljM7qBuguA4RRnLrke+Ra2z1ZZJIsidXDYgM47pb3qos43LZI9frNMcF4pgYEa\\\\\\\/29K+bu1gOBlHeMPy+xL7YkeK3UpKiRBO\\\\\\\/1z0zQTzxtR6a+A7IMJiFMJS6s5Bqu8dLFxAL\\\\\\\/+D0v2acZxctFKJn08VwLdfeoLc1rqhECWObTwwga9WLVl\\\\\\\/HM9019Pso5sKX0kVF6ijXnK9rfpwSUX3R5h3i0EC675dm9feO5KJaMYXq5vvu+WqdJyVhj\\\\\\\/mCObtv8cnPEi6qX7x9z7YltfEAli9YvV2Qz0L4cUXGY\\\\\\\/EctXx+UCpSBVqk6ODXtKSr02lkm9MXmCIiuKpNvYiqXZ+zIUj2wXdKzs13TQ1PlABGHo7\\\\\\\/lCE4fFJGjgHA8vCLWXt4bHhv0MwbxHoHLXdXYwvQRjhaagMJ\\\\\\\/6+TUp5y6Qku8jMMj9Kd\\\\\\\/VDJxmUyIP4wWUyWPvwLtv5SoObJq1iaYCqoHPI\\\\\\\/GyakBB\\\\\\\/4eUbrPF+0ojb\\\\\\\/ScCX+1SuDjSwGLXYRdynrD6OybLAeXEBMh2nEsuFQU36D4bOZ9imLXBaRl3+LIkLiuDakZicrE3IevTgD1o8HIcBit9uL8yR42RB3Gjsywy7T89bjUK2r+92awURJiD7+CeKZvMdAFUxD6kJqQ4CeR3Tyvs64\\\\\\\/XFKd+YIPk8KvrvpatRQorwOU2DfV3v+2z4z36ZfFYlde9I97Vu7CncbKu7+c45Vwp9Ahjsaj1v9In50vys+SkIKOA8DQUiaLz5FZDSM5F1OoRLSWjFy7vFT9WfRB+DS+\\\\\\\/rqqnbKgt\\\\\\\/HfTGxcKYWe8qpknHIFZv89VK6DBp0Yqm+Yp68isv6j4xPjRlO\\\\\\\/5J2uCAlRTmTAKVuSg\\\\\\\/fCF0DbWWHVthHq\\\\\\\/aYbPlIuFOdMsrlo1IXblh5eeQCE4Je0yqCL6GOrejNu1zuWblGSYJofJCemh5SeaqxbTo90gmaigHW4IYvXQMVYEG\\\\\\\/xBTCGENKM2tuiKDij5tNITkq4O6C1LBpFBieV+Ib30jhswaukUE8\\\\\\\/KCIsJx21JJ24NHqFm51L767WJnbymlDc4QPVKRitcgNvCH\\\\\\\/uc1atb\\\\\\\/qEJ4kIdhTdh1UfeZMKdIIh2IiuK14wWZgimTOrbpYpLjs\\\\\\\/xIG\\\\\\\/z5GrLuQ5qg7hYuqnXLjlu++\\\\\\\/K1kVPBNBEuSq9KXBTDp6zm7fUy9BaFcHdiACnC1JzZwatJwkg4bslbCO6q7DhVf5nx99bNFWdUggZDpMxFUoNkYJ1TrFM2UuFVULoOnh+1KhYFPHCklHB6\\\\\\\/wm\\\\\\\/tU8QWdk1EBmuJEVUW1mq\\\\\\\/9jQG2PuvZg\\\\\\\/0ZEl3rBLKX\\\\\\\/uF0Z+BOB5Wbbj8teRK2tXL+5cXv6yihUHhdQTouHoBdHc836QntbOc+9M1UJDDHqGUd8vm5J6pXXjkOPTtCKkcyO1pgWwLE+eDK6twqGNXcekab3IqPF\\\\\\\/jtz++qZovpO\\\\\\\/JkonpKv3WasXr0RSNKorOCOIKjehgaUs8qokicD8AODRgYxcBw7wGNNRD\\\\\\\/RgDce7uAcxXFT6MHRXe003lrWddjfFurK5z1YaAINSPa6htJXUO4AGcXMWF6tVP\\\\\\\/huzYPzTV1iUDWlm6sJsrpHlpYGCCdwRszrNqZuZMfTcK9aG8aqAgbN5dL6nskk\\\\\\\/eJo\\\\\\\/XtSYyt8c\\\\\\\/Zpa+osB\\\\\\\/D87pYQQTneMlDW9iNyke5IBuZa0cXGfrwqTemPeTlAFriW1oRTULIzluyGZQTbItZQ4tSi1gF0bs415jqruf5GhhFuYRLmBly3HGW1KRg25N2MitjtBWAJxDT6SEOsR9YT10AoRKmv9emJQ\\\\\\\/f4nS7dUmF\\\\\\\/wDSoA9NwcoqTu4HwGBAeUcOw4L7myTO0DND7cJsqxp51AAHkiin65AQaE2cDYT62NFpSJtfmWjfWVYi4xFGu5leooyE2mWsslxsHbXxtfDlQRb1IvHqmGMXye3PZZwseWmbk0z6PthidsuE9Cjjt8DP\\\\\\\/KjmfLHHoUNubed1Bb4q9wsLy1QZmFfuzADLCFI9MwfIkXZ0SYaIkjUJr6Y1Z8zck0uUqnUylepZahhTRuEQy465\\\\\\\/imSYqbaTOUbhl0b1zjg0pl9hNrSwcFJp6C6mGnOp9ho+ZQ8nmTdCkHjlt\\\\\\\/OnKd96WPVClITlh+xk35i4OzDLdTlFDi+AeL0XLoPCnUPPUVco+45CVgT6mOUmVarS4R9j5S1+\\\\\\\/ivDGIbAKolDa\\\\\\\/TWGgbZ0DS1tkCq+6T4cc86a+AH1sv67x9jlnNkvCachGTq6+saviHG2AHO4YwFz+7xDK3Wq0MLfqqxyYmufZZVwxJVfV6ndGRHMNwU1kiB5yt80APgcYxIISQWEhrn1TWm9B1Aoj1+hBzzUzqlYzRD+q0JshRwdVTCtCSfF20I44BDHnUwBi9lLBjf+Hu7qAtWq2BxNfwzxLqS9rcRq60Y1Pj85\\\\\\\/ziF3CggXQa70eg1C28Nfmk047S1qSaX0dKVr+Fi+ZFlyIOxNIzPETiLuaCDlt4iCwewlhIgTRYrTAR8+TgzDlUe5fLo3rSqoVT6XmLi7nCGlyGS5lfbOMB7SSZf6gD3mMiC3bujAeV6OPpQgZcPGTbegHsTcED3Ta0eCLh+I9+F5zCMtiD5NjedZtZsDqoPmohLZs\\\\\\\/+HWuidH6AVSa51qHjSV+IwxBesnrNogwrNMgBE6E6qPc8MxpS6k+SssaX0DyOrPhPOhJqmoOAYDCOTNaV9eM70krn4rCEZ9UgVMm5uapcPIQtZiIeR\\\\\\\/SpXO4K\\\\\\\/HXBjVmjncFb\\\\\\\/eKZgjj0iFez05iwUvgJtBvxaoKbYa0aNuwUaZ6JlbINGpPS5MlBvv\\\\\\\/4kcGmG2bG32Uh6FT+OlwfyzXKvid20ZuqzsmXK57sp+JdE4bVMg9A3RfvF9m5\\\\\\\/LY\\\\\\\/dulIhaXgx\\\\\\\/C9oJ2PkOT\\\\\\\/MspXVdz+naGbDrpc+U22ulv6iVNnh7xRN6OKqYCNLXNd5y1t23GpNjuKcQApAadoPnTaEjHBDOnNd9riQVVE5A1NkvHPwOQXvzS5hiCl0ox\\\\\\\/r+AEJL+9BzdUTg7D0yskvLRDE3OJf7SJEogzyESdQ+upHtWcMAOcnsaAPcnQ0yqL82Mtsd+u5NtFhrEp+TpKT8ZOD772jCYqE113YIS8s+NJVo\\\\\\\/+tanzV4zKhpibDr3uInQPWG9QXlua4Z+gPSFeOahXrVlKGhoLZFhUvQIwHMQzF1BU8xKRW5gS1J+Bnv\\\\\\\/\\\\\\\/PwzZaHK+lPbKEmObK04TgOGBd+cuTDwm3wLPVhP7Tc0bVgiXMrdeWCevQuZ8uj+zKWmeI7jmiYXpb155NdXZYszBBlWXavLVqjxlAXV944Rz8Y\\\\\\\/Ff5jlYskRbRgVkL26\\\\\\\/tjB75pnIdbhNURQxhzBtkfpel1haZTMjiZjv2Vtr4tWFH84V9qsFV8iT56a8du8gjeApQdinr5Sog6RT7X8TZIKyuXVZX8cQRqaQGm40AxOCs\\\\\\\/hh9icpHBFc8HblMesInn0gJ6JXpd6qnU79xAHDFE392Fk3fnI7Vwd4MtuaUcvRGp5E5FPAzAxKNwj8MvWN5ZIANua9utAbjsxOYy\\\\\\\/eSvvxbOR2cNhYyJKs9Ao46xDyLeH5q8DzDF1Zy\\\\\\\/auf+RWu+fqva\\\\\\\/jmDWnDnzeeRY8vAs6T9L+maaNnXVyKG7dExjXwH9ydYLf1v50T2\\\\\\\/Y\\\\\\\/XBVy6tGvTr0usRsLCqdMziB+3VWnnwXQziPE5q2hqpuLuKY6R9o2CL2fhULus3Mwk1BQk8zgLtMtuJBi1C82X8Tavn4h+PcEgdABT4xufu8dXVMxqOLrgQZMe\\\\\\\/Nc4n371bBThn4NhWlrH\\\\\\\/1J0IdLAssNkgDYbGGgFQtz\\\\\\\/c0NVNvGG\\\\\\\/mYs3oPpZD9AHN9vn8mF72VjeUgs8Cv+8NMfaOYAo\\\\\\\/HEuuLt\\\\\\\/K\\\\\\\/tyIqI2RuFu+dk+OH\\\\\\\/FOC6ZqWVRDzy3BVd4lgBQrFjD\\\\\\\/7UgtDEQuVvIHq0CyLUZXFX96aVtEZrlYKrQ2s4L45\\\\\\\/kUbVE8z\\\\\\\/b1GdHuI4vVHW+fqiK71QsvV\\\\\\\/edqhPIOw1PPdRyP8sVXhdvpgBFlNcc+6vknM1qyG9JoWnnSsPJukNulRHoSouceVGQWyX2qzVousetcbWSZVroQSP6yqs+sJ+r7AGjfENOPeaCtpyFC2o25aN6hgA5OsL7muR0A6YdD\\\\\\\/0HZB\\\\\\\/lgVn36eI9rRkPbBy3kq8UVpAEoBfqR0CrhnbCpwkL5sekg3dfX3N134Rf\\\\\\\/uZteNWLeKvX1DKRwZznSSxTV3y1GQPtABjU5KvwNguz6JQkekYGLcBzcjKnLZ6BjggVI5eZ9JcnExI\\\\\\\/FLsEcRw23zA+R3\\\\\\\/q5SpKwXSS1rq3DYhXI06IibynNiFO8lTGiC7CePXKwq4TD7n7oI5IFyZ+Rvz0+ubzC6Ov9ZLdCXytZxIYuHI72NfgRw0pFUFIgnWqXgz6PCP7TTcn9Lb6GgZ5CsQwEaHrcJkPowG\\\\\\\/UyFmroLo1L2ppnfI+u9+dKO2x3WBTvyjSEYtylmruufsfAOpUBjX02yl+8hCLDEKthMgcR3FQOHW4Lmt8aWgPIy2z73nI+O3ggJwYZ2nhl3vhMe0y9ui2uLuXo+fsoSU2KqLuidkkxM76+LZLCQMfbHWTY7x9n8leyd+DxsBqQ+fWhWjCzMRi2MKKjOtbs4X2Ma8TZEzk7HsLVSKNVMFhZlwcapHwTXZTMUxKj9mAJSuq0UEWRGid5PxVfx13+CzeBC9wnj4lf2Wz54YutVlVJRNgq\\\\\\\/fms2E8PmzFwxrspkXj8AZxFCVnmEifgLz+p\\\\\\\/B0uaUeIYquqypWXuJHONEiP2BBGJdR9+E8BRWi9vmCUv8ND42ovv70WQxW5c5HRy3Oa8XrBFe+7uG+gH1lPq4VUVd0YTEBUgdRotFJF1QPHZfNcashGYmRhL8f1c0X41VLhXkgYKtXXIsA2ywla4JM\\\\\\\/NCEoGZBkww8Q0x5uvq\\\\\\\/4NQaAuR0+wFWh0IH9ZzLXCymheNqaHbVmC1uavA8B6nsrkAW1\\\\\\\/pdG2Wmh8GWksMc3viQ3RXsACLJxh7Eh3T0\\\\\\\/bCseg+1fxg9FR0EK2AavxLk27S5BCL0r2nKv0bIVPSKDS+BhnX69l3UtnqgcyRq3dY1GDPsdRVNSJwqCdayo8VSwjSYEd9JXYcNG7UVYsesJh6KqGit1mgvhOehwxJLjlOOy5jjfxkdrP5MyJszKRGFkFk+t1p3+nY+7DUJiwMPIpBSF9cYGV\\\\\\\/zBcmPGGYB8ThA2l4mmkdnkmX1Z\\\\\\\/dHm3guGVsd9z7Z1Jog02HlDsaXpMmkVjqQF33hwq1E\\\\\\\/KQh9kL8Fbp01w0zRP1S2oi1FnE0AFwAbkmtLguwDIouQgyDNtKYrjp8SNnBJI4vtYFCQ+vOJ\\\\\\\/u7TLEX8GUe9Ypq9o00SEp8WqF0nDhFqS5rKq07FafxDki7udWi0CabbVbJ8maqKfqRFVDMiiCzyw+Xd9PrgzriEaBQ5snIDuJ2XKdknsvUmnQlzKx5qvV3YHVHfnJgc6Zr12Vmi66pTd2Yx7NWSwNJpwXElTuBBJs824mF9Z5ziLaYdfz+6Tczz3neP84UDiAouGo3OKIycs4AWYrs1p99malAMGdMsHTGoLgRGpiDfmpgi5Q\\\\\\\/NrUGBHhNTGuHCaazLN\\\\\\\/xSyDt4CovjsmnTAT4PH76JJfldw\\\\\\\/aoEkH+AkwJd9jEGeZLzNxk2JqQiE1Hltqrx5LEoZiuSfh1\\\\\\\/ILImsXUSDOi7PhTDp9B9Y3e1hUBNxO5y+pvBJKpeQtMyArBC9Yfqr6m3WSLwRZlUunOKMRN6UlUANvRRkEpfp6upEGY\\\\\\\/SAB98qgSanbmm0XRQjD+hOkQ3Xd0YQ9BMFw9Uf5y+aas92baRTr4SKpxc+HUu9JJLpC\\\\\\\/h8ymv0xX7tsaubThPTDe2udd1lLHPo3MOODVAqBNIMKsth7enUll6T4sJ6dw2VRFvPP8jUMATT50kbBY0JI3PiK86eJM25IMAGCUcqTLcVw+aRuOm5VzxvgfoRUZTKRosfOZVzAwBvlqyWvIhPNMF77fjoHv5XFpRJ1WsG7eQBAWmVPlIDDPxM8I22lSiiAh5+1vY+ynjSnpLbtYUjiAD1eOGHtSLZwQh4pz75gykT1KfLJpSpb7alu6eI9B2IPz3HE7434\\\\\\\/sb8IB1ctqbz6YWmALaQBxJNeGE9J28ZGmmJgqAEylGDfNolDKEv6QhZj53gFvwou1pc4tfE3inDPpq4rvD4RH05mlcZ1xpMyS9LlsYaVohE3PfHz09k4D+wctQy58YpmMhC2oBRAL6I4nrMx6QQHcXeJsHtnZnLf0Ln34jW0qQcpB17c1475Kb5ddZScusoocuOaCh2uy3Haea9DyuhD5GRHcF1xgqK7XEK1+wKTUaFjKzBT6YvKHmi5BAQBgY9gEDIyaj0IV09TrwhWCVoJ+gCBIbqKxhvdpZMsEx36l70\\\\\\\/EFomnoPdMJ92CFF6vZTLbhw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022UBJEog3qKKhhnye9xcZYGQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00225c79578ca196dafabc58ce0bfbf658482d3efe65dbf3c9e9a073fe78ff929f81\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00222lArqPtx8nK7TsJxVnFWXWB2AVc0cSFsWWcmNn5ILAzxNamIjpBhBzRnFToElBcRKQAxhzcz3vlwtoa4NB1ruNPNNBeuk9dYr44rBoaTFWbH33SinNLYcnmpdo5QZpGCL8EHxV18oAafj\\\\\\\/zP7hJNIa8yKZ4zH4qp6y2oxBd56TiY02UNKRRRmUifFyQTBV6XImcYyDACF0s0qyOl0lSq3EU34fiQH6C5l8le6EDK7bNy8mu6xaEu1IKvAlXNrR4jqijb9sObKOEA8J5sUMBATtl1IgmgilpxHbJUt307i3L3+XYdO+KXqGRuBWQ5M6iioIKJQk26sqgQ+p0KVBw1\\\\\\\/AyT2jo9JvYkBdAK9h9gr0p7eyhEpcNinb\\\\\\\/Az8gxExk8BG5EjdhGoL7HUmE\\\\\\\/SOJ\\\\\\\/wzIVyxQLlDwkL8\\\\\\\/zv0HrS027Rv7RdENeuvfDL7TTKhqThmj3vbxlflpAGeSGu+kZKw4N9fxsfiu+7FRV8l3sLZvHfFFzReueKLMhNH8rYXDoxcCkETEidFTlWg7bOIEsJmLSlT0CkC6CFOffiPq5bjK9Z5Kcm\\\\\\\/eQ\\\\\\\/eKJ72w5n2WAJ\\\\\\\/fELxXxhoTtFwo7GDUXO7EgyXufuRQtJkGYt+vaHXf4hjmx4Po8j9z19LGMwVvwEBrqmLN7bxKjZKI0vVMlkaGx92XVRNDBr7f+3mNQypKKjd+ZFnj5iqxGNUVVH7G2YvIoBq7j4QVVV57+E6yOQR7x4mZtRRGdYRk8eCdkucKACkZ34dCRnRv7c1dYeT0CDKv162cLwuseKV\\\\\\\/Ag5ED2Xsb+SAQtNKy46eqPoGdfjuDyRB3GKlYtcLoS6evV21qJTK8d3jGMuGIARFm0joabZxD6e+l6lbvz2U\\\\\\\/9A69o\\\\\\\/63zDqjKwIm1zvk\\\\\\\/uqPCsPqUqHHW9EMqLVfNpbf21xrUCdDDNnyWugu4\\\\\\\/ciNXJ9vJ9xRhyUSJ1JzyTaZ3mvJtNGrS+DNBXCtMYpQIiTwqT0GS0pUXKrGWIWxrgActhR1hdfkrStNmCeZ2I3HQpnxGYY5GCojrZBkLDzhXr\\\\\\\/ny7vxoxBhiog4aKIxHAu7exxjpufg+LcNeWl7nE8ayn+o6zB3Tdv8NqfXi6Aj5HFuHVQ7H1BoqrUXqA\\\\\\\/wCQm+9FGjAq4B5rVLUnmrBJEzHEZih3BzrzBvYMY2OP0ARvGKuwkjRNt8S0g4HWnKMHNQWxw\\\\\\\/wShyWLVHUVZnNm147Z716X8NskESSge1\\\\\\\/+ENOV6m1qfezPFlDrzkk7JUicrFM3Gm7lS25oL9Syx7nmW96EhJnKTopd8t3zbzSdgfy83FkrDOF4Pm05LIE2II82K9DdvXAOk0bJOulX5szLA+2bK+SSBEnIBPFcX99yL4B6pi12jJD+LBGr5VsSMTC\\\\\\\/JoyaBc9D1qUESSubLWM5OcHIFjNQYUdXZTLkL0mOHfOI0j9YWQ8xQppm815ZD2OIUZLEMcLZsh\\\\\\\/HyS0+o0u9IPOrdEPxh0+MrctrBRE38RxvO+h3AWtCc6epdWO8iq+Vzpy05Xtyx7qVpGaQd+JEWRJL4WMzhS2mGBNg7h06sRPLfHwTiUGoesFtOVj\\\\\\\/52eZAJm9hnXY1XEwYWIRLQ9qOH4Tn18BBZ1sqZRrKLH3ho8K4ezhYc\\\\\\\/TfavF\\\\\\\/ZVXgjZ9ogy8MqbvJ4G1vd8gtbx6LfrPFrbmbPUAuyKpi8W\\\\\\\/3wVwyMFCS1SFNqWZ7EOsAalQmByinfF1qN7Ty6PojevUZ4tKdeyOvoPDmfYWtdjjSRX+0Xxscy\\\\\\\/fDNEZ9sUm1WvgA6B4JMl\\\\\\\/Hl573QEzOaaO\\\\\\\/6J2Jf64vlQyQhCUTjVTk2RKsMM19U9zZTteFAidUYAtxMBTGX0o7nKciqyZB\\\\\\\/zzNrQX9o9zF+\\\\\\\/d7x3ctZht9l+mm+9GNm4Mt1N8ggulr1O3FnK6ArlWkKpYJNPwqN\\\\\\\/cLqZ7Y60G+KCq0QfG9tQwLkm\\\\\\\/n2LnTVS8nbntmNlmRCsLRujA3jxmcuLHyqhCpkCIeROyHWIArRloTeiKOZk36ACWH\\\\\\\/8Acvw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022nrdaMFI3CLXP0JM\\\\\\\/ezfqxQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00222f2bd957acbfe80387ccc8fbc4fb7e98a17b79c55dcb34a0943a3d26fb38a70f\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-492601 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-genomic-prediction">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/4/1189" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-2aN2i_bP4QFTKXIIooKcoVNPGSkJ5tifa-Y556E9b6w" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/4/1189" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-0x_yZsTOhsv3bMA3xEbUTK8KK4_zcxMpr0FuoWDSFR8" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/4/1175" title="Mitochondrial DNA Fitness Depends on Nuclear Genetic Background in Drosophila" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/4/1199" title="A Novel Root-Knot Nematode Resistance QTL on Chromosome Vu01 in Cowpea" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="492601" id="node492601" data-pisa="ggg;9/4/1189" data-pisa-master="ggg;g3.119.400059" data-apath="/ggg/9/4/1189.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genomic Prediction of Autotetraploids; Influence of Relationship Matrices, Allele Dosage, and Continuous Genotyping Calls in Phenotype Prediction</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-3723-9747" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Ivone de Bem Oliveira</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-2367-0766" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Marcio F. R. Resende</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-9655-4838" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Luis Felipe V. Ferrão</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0001-5127-4448" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Rodrigo R. Amadeu</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0003-0957-4337" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jeffrey B. Endelman</span>, <span class="highwire-citation-author" data-delta="5">Matias Kirst</span>, <span class="highwire-citation-author" data-delta="6">Alexandre S. G. Coelho</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="7"><a href="http://orcid.org/0000-0001-8973-9351" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Patricio R. Munoz</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1189-1198;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400059 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node492601" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Ivone de Bem Oliveira </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Blueberry Breeding and Genomics Lab</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Plant Genetics and Genomics Lab, Agronomy College, Federal University of Goias, GO, Brazil, 74690-900</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ivone%2Bde%2BBem%2BOliveira%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=de%20Bem%20Oliveira%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AIvone%2Bde%2BBem%2BOliveira%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-3723-9747" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Ivone de Bem Oliveira</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Marcio F. R. Resende <span class="nlm-suffix">Jr.</span></div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Sweet Corn Genomics and Breeding, Horticultural Sciences Department, University of Florida, Gainesville, FL 32611</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marcio%2BF.%2BR.%2BResende%2B%3Cspan%2Bclass%3D%22nlm-suffix%22%3EJr.%3C/span%3E" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Resende%20MF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMarcio%2BF.%2BR.%2BResende%2BJr." rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-2367-0766" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Marcio F. R. Resende <span class="nlm-suffix">Jr.</span></a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Luis Felipe V. Ferrão </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Blueberry Breeding and Genomics Lab</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Luis%2BFelipe%2BV.%2BFerr%C3%A3o%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ferr%C3%A3o%20LF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALuis%2BFelipe%2BV.%2BFerr%25C3%25A3o%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-9655-4838" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Luis Felipe V. Ferrão</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Rodrigo R. Amadeu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Blueberry Breeding and Genomics Lab</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rodrigo%2BR.%2BAmadeu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Amadeu%20RR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARodrigo%2BR.%2BAmadeu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-5127-4448" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Rodrigo R. Amadeu</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Jeffrey B. Endelman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Horticulture, University of Wisconsin, Madison, WI 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jeffrey%2BB.%2BEndelman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Endelman%20JB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJeffrey%2BB.%2BEndelman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-0957-4337" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jeffrey B. Endelman</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Matias Kirst </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Forest Genomics Lab, School of Forestry Resources and Conservation, University of Florida, Gainesville, FL 32610</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Matias%2BKirst%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kirst%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMatias%2BKirst%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Alexandre S. G. Coelho </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Plant Genetics and Genomics Lab, Agronomy College, Federal University of Goias, GO, Brazil, 74690-900</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alexandre%2BS.%2BG.%2BCoelho%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Coelho%20AS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlexandre%2BS.%2BG.%2BCoelho%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Patricio R. Munoz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Blueberry Breeding and Genomics Lab</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Patricio%2BR.%2BMunoz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Munoz%20PR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APatricio%2BR.%2BMunoz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-8973-9351" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Patricio R. Munoz</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:p.munoz@ufl.edu" class="" data-icon-position="" data-hide-link-title="0">p.munoz@ufl.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/9/4/1189" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:492601" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:492601/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/4/1189.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:492601" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:492601/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/4/1189.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:492601" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:492601/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/4/1189.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:492601" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:492601/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/9/4/1189.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:492601" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:492601/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Estimation of allele dosage, using genomic data, in autopolyploids is challenging and current methods often result in the misclassification of genotypes. Some progress has been made when using SNP arrays, but the major challenge is when using next generation sequencing data. Here we compare the use of read depth as continuous parameterization with ploidy parameterizations in the context of genomic selection (GS). Additionally, different sources of information to build relationship matrices were compared. A real breeding population of the autotetraploid species blueberry (<em>Vaccinium corybosum</em>), composed of 1,847 individuals was phenotyped for eight yield and fruit quality traits over two years. Continuous genotypic based models performed as well as the best models. This approach also reduces the computational time and avoids problems associated with misclassification of genotypic classes when assigning dosage in polyploid species. This approach could be very valuable for species with higher ploidy levels or for emerging crops where ploidy is not well understood. To our knowledge, this work constitutes the first study of genomic selection in blueberry. Accuracies are encouraging for application of GS for blueberry breeding. GS could reduce the time for cultivar release by three years, increasing the genetic gain per cycle by 86% on average when compared to phenotypic selection, and 32% when compared with pedigree-based selection. Finally, the genotypic and phenotypic data used in this study are made available for comparative analysis of dosage calling and genomic selection prediction models in the context of autopolyploids.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/autopolyploid" class="hw-term hw-article-keyword hw-article-keyword-autopolyploid" rel="nofollow">Autopolyploid</a></li><li class="kwd"><a href="/keyword/allelic-dosage" class="hw-term hw-article-keyword hw-article-keyword-allelic-dosage" rel="nofollow">Allelic dosage</a></li><li class="kwd"><a href="/keyword/genomic-selection" class="hw-term hw-article-keyword hw-article-keyword-genomic-selection" rel="nofollow">Genomic Selection</a></li><li class="kwd"><a href="/keyword/relationship-matrices" class="hw-term hw-article-keyword hw-article-keyword-relationship-matrices" rel="nofollow">Relationship Matrices</a></li><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/vaccinium" class="hw-term hw-article-keyword hw-article-keyword-vaccinium" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Vaccinium</em></a></li><li class="kwd"><a href="/keyword/blueberry" class="hw-term hw-article-keyword hw-article-keyword-blueberry" rel="nofollow">blueberry</a></li><li class="kwd"><a href="/keyword/shared-data" class="hw-term hw-article-keyword hw-article-keyword-shared-data" rel="nofollow">Shared data</a></li><li class="kwd"><a href="/keyword/genomic-prediction" class="hw-term hw-article-keyword hw-article-keyword-genomic-prediction" rel="nofollow">Genomic Prediction</a></li><li class="kwd"><a href="/keyword/genpred" class="hw-term hw-article-keyword hw-article-keyword-genpred" rel="nofollow">GenPred</a></li><li class="kwd"><a href="/keyword/shared-data-resources" class="hw-term hw-article-keyword hw-article-keyword-shared-data-resources" rel="nofollow">Shared Data Resources</a></li></ul><p id="p-3">Polyploidy events are not an exception in plants, as about 70% of Angiosperms and 95% of Pteridophytes underwent at least one polyploidization event (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Soltis and Soltis 1999</a>). Polyploids are normally grouped into two categories, autopolyploids and allopolyploids, but intermediate forms are also possible, such as segmental allopolyploids (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Spoelhof <em>et al.</em> 2017</a>). Thresholds for polyploid classification have been controversial, but following the general taxonomic definition, autopolyploids arise from within-species whole genome duplication, and allopolyploids arise from whole genome duplication prior to or after an inter-specific hybridization event (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Soltis <em>et al.</em> 2007</a>).</p><p id="p-4">Because speciation via ploidy increase can generate new phenotypic variability, this phenomenon is considered a powerful evolutionary source (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Hieter and Griffiths 1999</a>; <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Soltis <em>et al.</em> 2016</a>). Despite the important role of polyploidization in plant evolution, its effects on inheritance of many agronomic traits and population genetics are still poorly understood when compared with diploid species (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Dufresne <em>et al.</em> 2014</a>). This especially holds true for autopolyploids. Examples of the complex nature of autopolyploid genetics are the presence of genotypes with higher allele dosage than diploids, larger number of genotypic classes, possibility of multivalent pairing, and poor knowledge of chromosome behavior during meiosis (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Slater <em>et al.</em> 2014</a>; <a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Dufresne <em>et al.</em> 2014</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Mollinari and Serang 2015</a>).</p><p id="p-5">The advent of high-throughput genotyping methods, associated with the development of genetic and statistical analysis tools, has generated significant genetic gains for diploid species (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Desta and Ortiz 2014</a>). However, the application of genomic information to polyploid crops remains a challenge (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Comai 2005</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Grandke <em>et al.</em> 2016</a>). Although the theory for the computation of average genetic effect assuming arbitrary ploidy have been published by <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Kempthorne (1957)</a>, most of the methods for analysis and interpretation of genetic data in polyploids have only recently been described (see review in <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Bourke <em>et al.</em> 2018</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Kerr <em>et al.</em> 2012</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Endelman <em>et al.</em> 2018</a>), and most of them have not yet been fully investigated for different species, especially for new breeding approaches, such as genomic selection.</p><p id="p-6">Genomic selection (GS) is a method to increase the efficiency and accelerate the selection process in breeding programs. GS is used to capture the simultaneous effects of molecular markers distributed across the genome, based in the premise that the linkage disequilibrium between causal polymorphisms and markers allow phenotype prediction based on genotypic values (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Meuwissen <em>et al.</em> 2001</a>; <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Zhang <em>et al.</em> 2011</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Daetwyler <em>et al.</em> 2013</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">de los Campos <em>et al.</em> 2013</a>). Promising results have been reported in GS studies addressing polyploids (<em>e.g.</em>, <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Gouy <em>et al.</em> 2013</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Annicchiarico <em>et al.</em> 2015</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Ashraf <em>et al.</em> 2016</a>), however simplified assumptions were mostly considered, in other words diploid genetic models were used to circumvent the complexity involved in accurately defining allelic dosage (<em>i.e.</em>, the number of copies of each allele at a given polymorphic locus). Besides the existence of methods that allow accounting for ploidy effects (<a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Kerr <em>et al.</em> 2012</a>; <a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Endelman <em>et al.</em> 2018</a>), only a few studies have inserted this factor in the analyses (<em>e.g.</em>, <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Slater <em>et al.</em> 2016</a>; <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Sverrisdóttir <em>et al.</em> 2017</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Nyine <em>et al.</em> 2018</a>). In addition, these methodologies were not yet extensively compared, a point that is addressed in this article.</p><p id="p-7">Polyploidy can affect phenotypes through allelic dosage (additive effect of multiple copies of the same alleles), or by creating more complex interactions between loci or alleles, such as dominance or epistasis (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Osborn <em>et al.</em> 2003</a>). Thus, the inclusion of allelic dosage information may improve GS results (<em>e.g.</em>, increase of accuracy) by creating a more realistic representation of the effects of each genotypic class. Although the evidence of dosage effects in the expression of important economic traits exists (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Guo <em>et al.</em> 1996</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Birchler <em>et al.</em> 2001</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Adams <em>et al.</em> 2003</a>; <a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Osborn <em>et al.</em> 2003</a>), few studies linking dosage effects to phenotype prediction have been reported in autopolyploid species (<em>e.g.</em>; <a id="xref-ref-51-2" class="xref-bibr" href="#ref-51">Slater <em>et al.</em> 2016</a>; <a id="xref-ref-57-2" class="xref-bibr" href="#ref-57">Sverrisdóttir <em>et al.</em> 2017</a>; <a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Nyine <em>et al.</em> 2018</a>; <a id="xref-ref-18-3" class="xref-bibr" href="#ref-18">Endelman <em>et al.</em> 2018</a>). Genotype classification is one of the major challenges for polyploids. Studies about genotyping calling evaluation for autopolyploids with next generation sequencing (NGS) data showed that none of the existing methods performs properly (<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Grandke <em>et al.</em> 2016</a>), unless high sequencing coverage (60-80<em>x</em>) is used (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Uitdewilligen <em>et al.</em> 2013</a>).</p><p id="p-8">Here we compare a novel approach to GS in the context of autopolyploid, using <em>Vaccinium corymbosum</em> (southern highbush blueberry, SHB) as a model. The cultivated SHB is an autotetraploid, presenting 2n = 4X = 48 chromosomes (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Lyrene 2002</a>). Inbreeding depression is strong in SHB and population improvements have been achieved by long-term recurrent phenotypic selection alongside with long testing phase and slow genetic gain per generation (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Lyrene 2008</a>). Our goal was to investigate and compare the influence of different sources of information and ploidy parameterizations used to build relationship matrices on phenotype prediction, and thus the potential of GS in blueberry breeding.</p><div class="section materials-methods" id="sec-1"><h2 class="">Material and Methods</h2><div id="sec-2" class="subsection"><h3>Population and phenotyping</h3><p id="p-9">The population used in this study encompasses one cycle of the University of Florida blueberry breeding program’s recurrent selection, comprising 1,847 SHB unique individuals. This population was originated from 124 biparental controlled crosses, from 146 parents that presented superior phenotypic performance (cultivars and advanced stage of breeding). Phenotypic data of eight yield and fruit quality-related traits were collected during two production seasons (2014 and 2015), when the plants were 2.5 and 3.5 years of age at the University of Florida Plant Science Research and Education Unit in Citra (29°24’42.01” N -82°06’36.00” W, Florida, USA). Yield (rated using a 1-5 scale), weight (g), firmness (g mm<sup>-1</sup> of compression force), scar diameter (mm), fruit diameter (mm), flower bud density (reported as buds per 20 cm of shoot), soluble solids content (<sup>o</sup>Brix), and pH were evaluated. The last three traits were phenotyped only in one year – soluble solids content and pH were phenotyped in 2014 and flower buds in 2015.</p><p id="p-10">Five berries (fully mature and presenting picking quality) were randomly sampled to compose the measurement of fruit traits for each individual. Fruit weight was measured using an analytical scale (CP2202S, Sartorious Corp., Bohemia, NY). The FirmTech II firmness tester (BioWorks Inc., Wamego, KS) was used to measure fruit diameter and firmness. The scar diameter was obtained by image analysis of the fruits using FIJI software (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Schindelin <em>et al.</em> 2012</a>). The number of flower buds was counted in the main cane upright shoot, in the top 20 cm. A digital pocket refractometer (Atago, U.S.A., Inc., Bellevue, WA) was used to obtain soluble solids measures from 300μl of berry juice. The pH was measured using a glass pH electrode (Mettler-Toldeo, Inc., Schwerzenbach, Switzerland). More details are provided by <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Amadeu <em>et al.</em> (2016)</a>, <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Cellon <em>et al.</em> (2018)</a>, and <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Ferrão <em>et al.</em> (2018)</a>.</p></div><div id="sec-3" class="subsection"><h3>Genotyping</h3><p id="p-11">Genomic DNA was extracted and genotyped using sequence capture by Rapid Genomics (Gainesville, FL). Polymorphisms were genotyped in genomic regions captured by 31,063 120-mer biotinylated probes, designed based on the 2013 blueberry draft genome sequence (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Bian <em>et al.</em> 2014</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Gupta <em>et al.</em> 2015</a>). Sequencing was performed in the Illumina HiSeq2000 platform using 100 cycle paired-end sequencing. After trimming (quality score of 20), demultiplexing, and removing barcodes, reads were aligned to the draft genome using Mosaik v.2.2.3 (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Lee <em>et al.</em> 2014</a>). Genotypes were called using FreeBayes v.1.0.1 (Garrison and Marth 2012) considering the diploid and tetraploid options. Single-nucleotide polymorphisms (SNPs) were filtered considering i) minimum sequencing depth of 40 (average depth for the population); ii) minimum SNP phred quality score (QUAL) of 10; iii) only biallelic markers; iv) maximum population missing data of 0.5; and v) minor population allele frequency of 0.05. After filtering a total of 85,973 SNP were used in the GS analysis (the average sequencing-depth per sample was 73X). Further information regarding population composition and genotyping approach were described in <a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Ferrão <em>et al.</em> (2018)</a>. The genotypes for the diploid calling were coded as <em>0</em> (<em>AA</em>), <em>1</em> (<em>AB</em>), or <em>2</em> (<em>BB</em>). For the tetraploid parameterization they were coded as <em>0</em> (<em>AAAA</em>), <em>1</em> (<em>AAAB</em>), <em>2</em> (<em>AABB</em>), <em>3</em> (<em>ABBB</em>), and <em>4</em> (<em>BBBB</em>). A third parameterization (assumption-free method) was used, which considered allele ratio <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1189/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1189/embed/mml-math-1.gif"/></noscript></span></span>, where<span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1189/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1189/embed/mml-math-2.gif"/></noscript></span></span> is the allele count (sequencing depth) of the alternative allele and <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1189/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1189/embed/mml-math-3.gif"/></noscript></span></span> is the allele count of the reference allele. No dosage calling was performed in this model (File S1); these data varied continuously between <em>0</em> and <em>1</em>.</p></div><div id="sec-4" class="subsection"><h3>Population genetics analysis</h3><p id="p-12">In order to compare the information captured by each genomic-based relationship matrix, we performed linkage disequilibrium (LD), and principal components (PC) analyses. Pearson correlation tests (r<sup>2</sup>) were performed for pairwise LD estimation among SNPs within scaffolds, considering draft reference genomes (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Bian <em>et al.</em> 2014</a>; <a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Gupta <em>et al.</em>, 2015</a>). One SNP was randomly sampled per probe interval, and a total of 22,914 SNPs were used in the analysis. LD was obtained for all marker-based scenarios: i) diploid (<em>G2</em>); ii) tetraploid (<em>G4</em>) and iii) ratio (<em>i.e.</em>, continuous genotypes; <em>Gr</em>). The LD decay over physical distance was determined as the mean distance at the LD threshold of r<sup>2</sup> = 0.2. To compare the LD among scenarios, the mean distances (Kb) and their interval confidences at r<sup>2</sup> = 0.2 were compared. The diversity captured from each relationship matrix was obtained by PC using the R package adegenet v. 1.3-1 (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Jombart and Ahmed 2011</a>).</p><p id="p-13">We also evaluated the observed heterozygosity in the population. For this, we obtained the ratio between the number of heterozygote genotypes and the total number of individuals. To estimate the heterozygosity for the continuous genotypes, empirical limits were established based on the mean and standard deviations presented for homozygotes classes of the tetraploid parameterization.</p></div><div id="sec-5" class="subsection"><h3>Models</h3><p id="p-14">One-step single-trait Bayesian linear mixed models were used to predict breeding values for each individual in the population, as follows:</p><span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1189/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1189/embed/mml-math-4.gif"/></noscript></span><span class="disp-formula-label">(1)</span></span><p id="p-15">Where <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1189/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1189/embed/mml-math-5.gif"/></noscript></span></span>is a vector of the phenotypic values of the trait being analyzed, μ is the population’s overall mean, <em>b</em> is the fixed effect of year, <em>c</em> is the random effect of <em>i</em>th column position in the field ∼ <em>N</em> (0, <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1189/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1189/embed/mml-math-6.gif"/></noscript></span></span>), <em>r</em> is the random effect of the <em>i</em>th row position in the field ∼ <em>N</em> (0, <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1189/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1189/embed/mml-math-7.gif"/></noscript></span></span>), <em>a</em> is the random effect of genotype ∼ <em>N</em> (0,<span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1189/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1189/embed/mml-math-8.gif"/></noscript></span></span>), where <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1189/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1189/embed/mml-math-9.gif"/></noscript></span></span> was replaced by the different additive relationship matrices as described in the next section. The <em>bxa</em> is the random effect of the year by genotype interaction ∼ <em>N</em> (0, <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1189/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1189/embed/mml-math-10.gif"/></noscript></span></span>), and <em>e</em> is the random residual effect ∼ <em>N</em> (0, <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1189/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1189/embed/mml-math-11.gif"/></noscript></span></span>). Row and column effects were considered nested within year only for the traits evaluated in two years. For traits measured in a single year, the same <a id="xref-disp-formula-1-1" class="xref-disp-formula" href="#disp-formula-1">equation (1)</a> was used without the year and the year by genotype interactions. The variance components for each random variable were: additive (<span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1189/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1189/embed/mml-math-12.gif"/></noscript></span></span>), column (<span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1189/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1189/embed/mml-math-13.gif"/></noscript></span></span>), row (<span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1189/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1189/embed/mml-math-14.gif"/></noscript></span></span>), year-by-genotype interaction (<span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1189/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1189/embed/mml-math-15.gif"/></noscript></span></span>), and residual (<span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1189/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1189/embed/mml-math-16.gif"/></noscript></span></span>). <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1189/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1189/embed/mml-math-17.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1189/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1189/embed/mml-math-18.gif"/></noscript></span></span> were incidence matrices for year, column, row, genotype, and year by genotype interaction, respectively. The narrow-sense heritabilities were estimated considering the ratio between the additive variance component and the total phenotypic variance (sum of all variance components).</p></div><div id="sec-6" class="subsection"><h3>Relationship matrices</h3><p id="p-16">To quantify the effect of the genetic information used to build the relationship matrices on the predictive ability (PA), we performed analyses considering different approaches to modeling the genotypic values in autotetraploid species (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>, File S1). The factors tested were: i) the source of information used to build the relationship matrix (pedigree, genomic, or no relationship information); and ii) ploidy information (diploid, tetraploid, and assumption-free method).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/492742/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/492742/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Methods and assumptions used to compare the influence of relationship matrices, ploidy and continuous genotypes in the prediction of breeding values for blueberry</span><div class="sb-div caption-clear"></div></div></div><p id="p-17">The methods chosen to obtain the relationship matrices are shown in the <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>. The R package AGHmatrix v. 0.0.3003 (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Amadeu <em>et al.</em> 2016</a>) was used to obtain all relationship matrices (description of matrices File S1). The pedigree-based relationship matrices (<em>A</em>) were built considering a diploid model (<a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Henderson 1976</a>) and autotetraploid model without double-reduction (<a id="xref-ref-31-4" class="xref-bibr" href="#ref-31">Kerr <em>et al.</em> 2012</a>). The marker-based relationship matrices (<em>G</em>) were based on the incidence matrices of markers effects (<em>X</em>) according to <a id="xref-ref-59-2" class="xref-bibr" href="#ref-59">VanRaden (2008)</a> and adapted by <a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Ashraf <em>et al.</em> (2016)</a>. Different assumptions can be made regarding the marker allele dosage in autotetraploids (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). We built the <em>X</em> matrices under three assumptions regarding the additive marker allele dosage effect: <em>i)</em> a pseudo-diploid model, where all the heterozygous genotypes were assumed as one class, corresponding to a unique effect (data coded as <em>0</em>, <em>1</em>, and <em>2</em>); <em>ii)</em> an additive autotetraploid model, where each genotype had a specific value, and cumulative additive effect was assumed (data coded as <em>0</em>, <em>1</em>, <em>2</em>, <em>3</em>, and <em>4</em>); and <em>iii)</em> an assumption-free method based on the ratio of reads count for the alternative and reference alleles (continuous parameterization, assuming values between <em>0</em> and <em>1</em>), where also a cumulative additive effect was assumed. For the construction of the relationship matrices based on marker data, the missing genotypes were substituted by the mean.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/492723/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/492723/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Theoretical genotype codes for marker-allele dosage effects considering pseudo-diploid, autotetraploid and continuous parameterizations. Adapted from <a id="xref-ref-51-3" class="xref-bibr" href="#ref-51">Slater <em>et al.</em> (2016)</a></span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-7" class="subsection"><h3>Model implementation</h3><p id="p-19">The six models described above (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>) were fitted using the R package (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">R Core Team 2018</a>) BGLR v. 1.0.5 (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">de los Campos and Pérez-Rodríguez 2016</a>). Predictions were based on 30,000 iterations of the Gibbs sampler, in which 5,000 were taken as burn-in, and a thinning of five. The number of iterations, burn-in, and thinning interval parameters were evaluated to define the final values used in the analysis (Figure S1). A single step regression approach was applied to perform all phenotypic BLUP (I matrix), pedigree-BLUP (P-BLUP), and genomic-BLUP (G-BLUP). Default hyper-parameters were used, as previously described (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Pérez and de los Campos 2014</a>).</p></div><div id="sec-8" class="subsection"><h3>Validation and model comparison</h3><p id="p-20">For each trait, models were compared based on their PA, stability (mean square errors), goodness-of-fit, and expected genetic gain. A 10-fold cross validation scheme was applied to compute model PA, for this the genotypes were assigned to ten groups, on each cross-validation step the phenotypic information for one of the groups was omitted (validation set) and predicted considering the model obtained from the remaining nine groups (training set). Because each validation group might have a different mean (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Resende <em>et al.</em> 2012b</a>), the phenotypic PA were obtained as the Pearson correlation coefficient between the empirical best linear unbiased estimation values (eBLUEs) obtained by considering all the variables in the <a id="xref-disp-formula-1-2" class="xref-disp-formula" href="#disp-formula-1">equations 1</a> as fixed (<em>i.e.</em>, Least Square means estimations; LSMeans) and the cross-validated breeding values (BV) predicted by the models for each validation fold. The goodness-of-fit for the different models was evaluated with measures of DIC (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Spiegelhalter <em>et al.</em> 2002</a>) obtained from the full data set, extracted from the object returned by BGLR. The model with the lowest value for this parameter defined the best fit for the data. For the expected genetic gain estimation we used the following formula: ΔG = (<span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1189/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1189/embed/mml-math-19.gif"/></noscript></span></span>)/<em>L</em>, where <em>PA</em> is the phenotypic predictive ability, <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1189/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4/1189/embed/mml-math-20.gif"/></noscript></span></span> is the square root of additive genetic variance in the population, <em>i</em> is the selection intensity, and <em>L</em> is the breeding cycle length. To make it comparable between methods the selection intensity (<em>i</em>) was considered constant for all methods and equal to 1.</p><p id="p-21">Average phenotypic and raw genotypic data used during the current study will become available to promote further studies on the effect of dosage calling in the context of GS modeling.</p></div><div id="sec-9" class="subsection"><h3>Data availability</h3><p id="p-22">Phenotypic datasets (eBLUES) are available from the Dyrad Digital Repository (accession number doi: 10.5061/dryad.kd4jq6h). Genotypic data and supplemental material are available at Figshare. Files include diploid, tetraploid and continuous genotypes, supplemental information 1 to 5, which includes: 1) description of the matrices used in the study; 2) model convergence figure; 3) LD distribution per parameterization; 4) principal components plots for each parameterization; 5) table of the predictive abilities, MSE, goodness-of-fit and beta for each parameterization. The authors ratify that all data necessary for confirming the conclusions of the article are present within the article, figures, and tables. Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.7728365">https://doi.org/10.25387/g3.7728365</a>.</p></div></div><div class="section results" id="sec-10"><h2 class="">Results</h2><div id="sec-11" class="subsection"><h3>Population genetics analyses</h3><p id="p-23">Linkage disequilibrium decayed below r<sup>2</sup> = 0.2 at distances of 88.3 Kb, 92.6 Kb, and 98.2 Kb for the diploid, tetraploid and continuous models, respectively (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A-C</a>). No significant difference was observed considering the confidence interval for the mean distance (Kb) at r<sup>2</sup> = 0.2 among different ploidies and continuous genotyping scenarios (Figure S2).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/4/1189/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Linkage disequilibrium decay and heterozygosity for blueberry. Linkage disequilibrium decay estimation using one marker per probe, within scaffolds for (A) diploid, (B) tetraploid and (C) continuous genotype parameterizations. Heterozygosity observed in (D) diploid, (E) tetraploid, and (F) heterozygosity empirically established for the continuous genotypes&#x2019; scenario, assuming the limits of 0.058 &#x2264; X &#x2264; 0.908." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2088183376" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Linkage disequilibrium decay and heterozygosity for blueberry. Linkage disequilibrium decay estimation using one marker per probe, within scaffolds for (A) diploid, (B) tetraploid and (C) continuous genotype parameterizations. Heterozygosity observed in (D) diploid, (E) tetraploid, and (F) heterozygosity empirically established for the continuous genotypes&#x2019; scenario, assuming the limits of 0.058 &#x2264; X &#x2264; 0.908.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/4/1189/F1.medium.gif" width="440" height="363"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/9/4/1189/F1.medium.gif" width="440" height="363"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/4/1189/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/4/1189/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492641" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-24" class="first-child">Linkage disequilibrium decay and heterozygosity for blueberry. Linkage disequilibrium decay estimation using one marker per probe, within scaffolds for (A) diploid, (B) tetraploid and (C) continuous genotype parameterizations. Heterozygosity observed in (D) diploid, (E) tetraploid, and (F) heterozygosity empirically established for the continuous genotypes’ scenario, assuming the limits of 0.058 ≤ X ≤ 0.908.</p><div class="sb-div caption-clear"></div></div></div><p id="p-25">Similarly, no major differences were found between parameterizations within methodology (<em>i.e.</em>, pedigree-based or marker-based methods) in the PC analysis (Figure S3). The first two PC components of the marker-based (<em>G</em>) matrices were consistent across all matrices, explaining approximately 20% of the variation, <em>G2</em> matrix captured 20.60% of the variation, while <em>G4</em> captured 21.71%, and <em>Gr</em> captured 23.36% (Figure S3 A-C). The PC analysis results were consistent between pedigree methodologies as well. Approximately 38% of the variation was explained (<em>i.e.</em>, 37.74% of the variability was explained for the <em>A2</em> matrix and 37.86% was explained for the <em>A4</em> matrix, Figure S3 D-E). The results obtained in the PC analysis did not justify a stratified sampling of cross-validation populations, since no evidence of sub-population structure was detected for any of the relationship matrices.</p><p id="p-26">Considering the heterozygosity observed in each scenario, genotypes assumed as homozygotes in the diploid parameterization were classified as one of the possible heterozygote classes in the tetraploid and in the assumption-free parameterizations (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1D-F</a>). As a result of this process, the tetraploid parameterization presented 37.50% more heterozygotes than the diploid parameterization. Considering the empirical thresholds established to compare the proportion of “heterozygotes” in the continuous genotypes with the ploidy parameterizations, values equal to or below 0.058 and equal to or above 0.908 were considered as “homozygotes” classes (dashed lines, <a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1F</a>). With this, 61.59% of the genotypes were considered “heterozygotes”, thus the continuous method would have presented 89.92% and 41.23% more heterozygotes than the diploid and the tetraploid parameterization, respectively. Nevertheless, some misclassification of data into classes in the diploid and tetraploid parameterization might have occurred (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A-B</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/4/1189/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Relationship between continuous values and the classes assumed in the (A) diploid and (B) tetraploid parameterizations." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2088183376" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Relationship between continuous values and the classes assumed in the (A) diploid and (B) tetraploid parameterizations.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/4/1189/F2.medium.gif" width="440" height="302"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/9/4/1189/F2.medium.gif" width="440" height="302"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/4/1189/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/4/1189/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492749" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-27" class="first-child">Relationship between continuous values and the classes assumed in the (A) diploid and (B) tetraploid parameterizations.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-12" class="subsection"><h3>Variance estimates</h3><p id="p-28">The posterior means of the genetic parameters are summarized in <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>. All the traits presented additive genetic variance significantly higher than zero. A wide range of variance was observed within a given parameter for the different methodologies, and most of the values were significantly different from each other (considering Tukey test results; <a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>, Table S1). Marker-based methodologies generated significantly smaller estimations for variance components when compared with pedigree-based estimations. Within marker-based methodologies, the assumption-free parameterization generated significantly smaller estimations. The effects of the difference in the estimation of variance components are reflected in the estimated heritabilities – smaller values were estimated for marker-based methodologies. The lowest heritability was obtained for soluble solids, flower buds, and pH. Considering all methods, narrow-sense heritability values varied between 0.152 and 0.574, for flower buds and fruit weight, respectively.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/492733/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/492733/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Genetic parameters estimated for eight yield and fruit-related traits analyzed with six linear mixed models, considering the use of ploidy information and continuous genotypes. Source of information, and dosage parameterizations for the relationship matrices indicated by the letters (<em>I</em>, <em>A</em>, or <em>G</em>), and index 2, 4, and r respectively<sup>*</sup></span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-13" class="subsection"><h3>Effect of the genetic information to build the relationship matrices</h3><p id="p-31">The incorporation of relationship information in the analysis generated better PA results than the phenotypic-BLUP model without it. Overall, we observed that higher values for the phenotypic PA were obtained when marker-based relationship matrices were used, when compared with phenotypic and pedigree BLUP (<em>I</em> and <em>A</em> matrices, respectively). However, the marker-based and pedigree-based results were not always significantly different from each other (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>, Table S1). The use of molecular data yielded phenotypic PA values ranging from 0.27 (pH) to 0.49 (fruit scar) in 2014, and from 0.15 (flower buds) to 0.51 (fruit firmness) in 2015. Lower PA values were obtained for traits with lower heritability and better results were observed for the second year of evaluation. The biggest increase in the PA values can be seen for fruit firmness – when we compared marker and pedigree results, we observed an average increase of 13.37% in 2014. Also, an increase in the PA values of 11% was observed for fruit diameter and yield in 2015 when markers were used instead of pedigree data.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/4/1189/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Phenotypic predictive abilities. Predictive abilities obtained for (A) seven traits in 2014, and (B) for six traits in 2015 considering different dosage parameterizations (indicated by the numbers 2 or 4, and r for ratio values), and different relationship matrices (indicated by the letters I, A, and G) in the prediction of breeding values of 1,847 blueberry genotypes." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2088183376" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Phenotypic predictive abilities. Predictive abilities obtained for (A) seven traits in 2014, and (B) for six traits in 2015 considering different dosage parameterizations (indicated by the numbers &lt;em&gt;2&lt;/em&gt; or &lt;em&gt;4&lt;/em&gt;, and &lt;em&gt;r&lt;/em&gt; for ratio values), and different relationship matrices (indicated by the letters &lt;em&gt;I&lt;/em&gt;, &lt;em&gt;A&lt;/em&gt;, and &lt;em&gt;G&lt;/em&gt;) in the prediction of breeding values of 1,847 blueberry genotypes.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/4/1189/F3.medium.gif" width="400" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/9/4/1189/F3.medium.gif" width="400" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/4/1189/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/4/1189/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492701" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-32" class="first-child">Phenotypic predictive abilities. Predictive abilities obtained for (A) seven traits in 2014, and (B) for six traits in 2015 considering different dosage parameterizations (indicated by the numbers <em>2</em> or <em>4</em>, and <em>r</em> for ratio values), and different relationship matrices (indicated by the letters <em>I</em>, <em>A</em>, and <em>G</em>) in the prediction of breeding values of 1,847 blueberry genotypes.</p><div class="sb-div caption-clear"></div></div></div><p id="p-33">The use of pedigree-based relationship matrices generated higher phenotypic PA values for all the traits, when compared with the assumption of unrelated individuals (<em>i.e.</em>, identity matrix). Unlike the identity matrix, the use of pedigree-based matrix assumes that there is relationship (expected values) among individuals. The phenotypic PA obtained for the pedigree methods in 2014 yielded values from 0.20 (flower bud) to 0.49 (fruit firmness). As with marker-based methods, smaller values were observed for traits with lower heritability (<em>i.e.</em>, pH, brix, and flower bud). For 2015, the PA results for the phenotypic-BLUP were 0.36, 0.38, and 0.42, for fruit weight, fruit scar, and fruit firmness, respectively. The PA values obtained for the same traits with pedigree-BLUP were 0.40, 0.45, and 0.49, respectively. No significant differences between the models’ stability were observed (Table S1).</p></div><div id="sec-14" class="subsection"><h3>Use of dosage information and continuous genotypes</h3><p id="p-34">Our results indicate that the importance of dosage in GS will vary depending on the trait being analyzed. For example, in 2014 the PA for fruit firmness, fruit scar, and fruit diameter showed modestly better phenotypic PA when the tetraploid and continuous parameterizations were applied, as opposed to the diploid parameterization (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>, Table S1). The addition of more classes for the representation of the genotypic classes added complexity to the models (Table S1), in other words bigger values of DIC were observed for <em>G4</em> and <em>Gr</em> models. Although no significant difference was observed between marker-based models, the use of relationship matrices derived from continuous genotype data (ploidy-free parameterization) performed equally well as the best models (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a>, Table S1).</p></div><div id="sec-15" class="subsection"><h3>Expected genetic gain in a perennial fruit tree, blueberry</h3><p id="p-35">The results obtained for the expected genetic gain (EGG) are summarized in <a id="xref-table-wrap-3-3" class="xref-table" href="#T3">Table 3</a>. GS offers the possibility to accelerate genetic improvement by decreasing the breeding cycle and selecting superior individuals earlier in the breeding program. Considering a breeding cycle (<em>L</em>) of 12 years (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Cellon <em>et al.</em> 2018</a>) we propose that routine genomic selection could be implemented in the second stage of the blueberry breeding program, which would allow the omission of a whole stage (stage III), and a three-year reduction for cultivar release (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/4/1189/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Proposal of GS implementation in the University of Florida blueberry breeding program. UF blueberry breeding program stages and times of selection considering the conventional process (left) compared with the proposed process implementing genomic selection (right)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2088183376" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Proposal of GS implementation in the University of Florida blueberry breeding program. UF blueberry breeding program stages and times of selection considering the conventional process (left) compared with the proposed process implementing genomic selection (right).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/4/1189/F4.medium.gif" width="440" height="195"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/9/4/1189/F4.medium.gif" width="440" height="195"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/4/1189/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/4/1189/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492649" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-36" class="first-child">Proposal of GS implementation in the University of Florida blueberry breeding program. UF blueberry breeding program stages and times of selection considering the conventional process (left) compared with the proposed process implementing genomic selection (right).</p><div class="sb-div caption-clear"></div></div></div><p id="p-37">Higher EGG was obtained for all traits when marker-based matrices (<em>i.e.</em>, genomic selection) were applied (<a id="xref-table-wrap-3-4" class="xref-table" href="#T3">Table 3</a>), which was mainly related to the reduction in cycle time. The implementation of GS in the second stage population would lead to an increase in the EGG varying from 27% (pH) to 119% (scar) when compared with the application of phenotypic BLUP. Considering the comparison of marker-based and pedigree-based models, an increase of 15% (pH) to 41% (fruit weight, fruit scar, and flower buds) in the EGG was observed (<a id="xref-table-wrap-3-5" class="xref-table" href="#T3">Table 3</a>). In addition, the use of continuous data generated EGG values that were not significantly different of the best models for all traits (<a id="xref-table-wrap-3-6" class="xref-table" href="#T3">Table 3</a>).</p></div></div><div class="section discussion" id="sec-16"><h2 class="">Discussion</h2><p id="p-38">In this study, six approaches were applied to predict breeding values for eight yield and fruit-quality traits measured in a real blueberry breeding population. Analyses were based on phenotypic, pedigree, and high-density marker data from 1,847 individuals. We compared the expected genetic gain, the stability, and the PA of models considering different sources to build the relationship matrices (only phenotype = BLUP, phenotypes + pedigree = P-BLUP, phenotypes + genomic = G-BLUP). Our results also explored models accounting for ploidy information and compared the use of genotypic data that is independent of assumptions regarding ploidy levels (continuous) to perform GS, avoiding the need for <em>a priori</em> parameterization for a given ploidy level.</p><div id="sec-17" class="subsection"><h3>Continuous data</h3><p id="p-39">Our research showed empirical evidences that the use of continuous genotypic data from NGS can be effectively applied in GS models for autotetraploid species. This method was tested and compared with marker calling methodologies at the individual level in genome wide association studies (<a id="xref-ref-23-3" class="xref-bibr" href="#ref-23">Grandke <em>et al.</em> 2016</a>). It was also tested in family pool data for GS (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Ashraf <em>et al.</em> 2014</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Cericola <em>et al.</em> 2018</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Guo <em>et al.</em> 2018</a>), as well as used at the individual level in tetraploid potato for GS by <a id="xref-ref-57-3" class="xref-bibr" href="#ref-57">Sverrisdóttir <em>et al.</em> (2017)</a>. However, to our knowledge the comparison of continuous genotypes with ploidy parameterizations for genomic selection has not yet been reported. Here we empirically compare diploid, tetraploid, and continuous data at the individual level for the application of genomic selection in an autotetraploid species.</p><p id="p-40">In polyploids, the assignment of genotypic classes based on NGS data has been a major challenge, with high risk of misclassification (<a id="xref-ref-23-4" class="xref-bibr" href="#ref-23">Grandke <em>et al.</em> 2016</a>, <a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Bourke <em>et al.</em> 2018</a>). The problem is further exacerbated as the ploidy increases – for a given level of ploidy, <em>n</em>, the expected number of genotypic classes is <em>2n+1</em>. As a consequence, the signal distribution derived from each genotypic class increasingly approximates a continuous distribution, where no clear separation is observed (<a id="xref-ref-23-5" class="xref-bibr" href="#ref-23">Grandke <em>et al.</em> 2016</a>). Despite extensive research to address these challenges (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Serang <em>et al.</em> 2012</a>), advances have been mostly limited to SNP arrays in tetraploid data (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Schmitz Carley <em>et al.</em> 2017</a>). Studies that evaluated genotype calling with NGS data obtained from polyploids show that no method works properly, and that misclassification of genotypes can significantly interfere in the results of genetic studies (<a id="xref-ref-23-6" class="xref-bibr" href="#ref-23">Grandke <em>et al.</em> 2016</a>). This misclassification can be observed in our results when are diploid, or tetraploid parameterization were used in the genomic data (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2A-B</a>), even with our high sequencing depth and with standard parameters of filtering. The use of the continuous genotyping approach provides a relevant alternative to overcome this issue that is independent of assumptions regarding ploidy level. Models that used continuous genotypic data performed as well as the best models and resulted in modestly better predictive abilities for some of the traits (<em>i.e.</em>, fruit firmness, fruit scar, and fruit diameter; <a id="xref-table-wrap-3-7" class="xref-table" href="#T3">Table 3</a>), which could indicate better prediction of future populations. The use of continuous genotypes also simplifies the analysis complexity and time, by eliminating the genotype calling and parameterization for a give ploidy, because instead, the ratio of reads assigned to each allele are used. The benefits of continuous genotyping could easily be extended to more complex polyploids (higher ploidies), where the genotype attribution is even more difficult, however higher sequencing depth would be required. Meanwhile, for more complex models, such as those that consider dominance effects, dosage calling is still necessary.</p></div><div id="sec-18" class="subsection"><h3>Relationship matrices</h3><p id="p-41">Our results also showed that including information based on the genetic merit of the individuals yielded better results when compared with the phenotypic-BLUP analysis (based on the identity matrix; <a id="xref-table-wrap-3-8" class="xref-table" href="#T3">Table 3</a>), corroborating previous studies in the literature (<em>e.g.</em>, <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Muir 2007</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Resende <em>et al.</em> 2012a</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Muñoz <em>et al.</em> 2014a</a>). In addition, the use of marker-based methodologies generated better predictions than pedigree for most of the traits. Marker-based methods allow the capture of Mendelian segregation (<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Daetwyler <em>et al.</em> 2013</a>;). This is especially important in our population, since it was composed of 117 full-sib families. In this context, pedigree-based methods have no power to distinguish variance within families. Another advantage is that marker-based methods allows the computation of genetic similarity among unidentified individuals in the pedigree, and corrections of errors in the pedigree, which can affect parameter estimation causing reduction in the genetic gain (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Muñoz <em>et al.</em> 2014b</a>).</p><p id="p-42">In our results, some non-significant differences between pedigree and marker-based methods were identified, which could be an effect of the extensive pedigree data used, as well as bias in pedigree-based estimations. Pedigree-based methods can overestimate the reliability of selection and consequently, the accuracy (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Bulmer 1971</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Gorjanc <em>et al.</em> 2015</a>). Furthermore, it also presents low efficiency to capture and estimate genetic relationships among individuals (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Resende <em>et al.</em> 2017</a>).</p><p id="p-43">It is interesting to notice that we used extensive pedigree information that dates back to 1907 for our predictions, which may not be common in other autopolyploid breeding. This extensive information can have significant implications on the estimation of relationship coefficients (<a id="xref-ref-2-3" class="xref-bibr" href="#ref-2">Amadeu <em>et al.</em> 2016</a>) and consequently, in breeding value predictions. Therefore, for breeding programs with smaller pedigree depth information, the comparison between accuracies of prediction from marker and pedigree-based methodologies could be even bigger than what was found in our study.</p></div><div id="sec-19" class="subsection"><h3>Allele dosage</h3><p id="p-44">The results obtained for both models that assumed more than three genotypic classes (<em>G4</em> and <em>Gr</em>) demonstrate the importance of considering dosage in the prediction of breeding values. However, this will depend on the trait analyzed, as previously reported by <a id="xref-ref-40-3" class="xref-bibr" href="#ref-40">Nyine <em>et al.</em> (2018)</a> and <a id="xref-ref-18-4" class="xref-bibr" href="#ref-18">Endelman <em>et al.</em> (2018)</a>. For example, modest improvement was verified in the PA for fruit firmness, fruit scar, and fruit diameter when this factor was considered in the models. The addition of classes for the representation of ploidy increased the complexity of the models (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3</a>, <a id="xref-table-wrap-3-9" class="xref-table" href="#T3">Table 3</a>, Table S1) however, these assumptions also show a more realistic representation of the nature of the species. The inclusion of nonadditive effects into the models could also improve model accuracy. <a id="xref-ref-18-5" class="xref-bibr" href="#ref-18">Endelman <em>et al.</em> (2018)</a> demonstrated that the inclusion of digenic effects, as well as accounting for ploidy information, presented a higher accuracy over diploid models when using a SNP array.</p></div><div id="sec-20" class="subsection"><h3>Genomic selection for perennial autopolyploids</h3><p id="p-45">We also demonstrate the value of applying GS in a perennial fruit tree, blueberry. One cycle of blueberry breeding takes from 12 to 15 years until the release of a new cultivar (<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Lyrene 2008</a>; <a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">Cellon <em>et al.</em> 2018</a>). By applying selection based on high-density markers at early stages of the program, the time to cultivar release could decrease by three years (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>), significantly improving the expected genetic gain per unit of time. More specifically, the use of GS would lead to an average increase of 86% in the EGG when compared with phenotypic BLUP, and an average increase of 32% over the application of pedigree-based models (<a id="xref-table-wrap-3-10" class="xref-table" href="#T3">Table 3</a>). Implementing GS as we propose here could eliminate one stage in the breeding and selection process toward cultivar development, which will reduce costs associated with field trials and phenotyping. The implementation of GS would require extra financial outlay when genotyping and accurately phenotyping the training population. However, the savings on phenotyping and field trials of future generations (selection populations) could result in a break-even financial exercise, and as a result could be a cost-effective application of GS. However, this financial analysis needs to be performed for each crop in a case-by-case basis. To promote further studies on the effect of dosage calling using NGS, as methods and software improve, we are providing genotypic and phenotypic data to use as comparison of methods in the context of GS.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Funding</h2><p id="p-46">This project was funded by the Agriculture and Food Research Initiative Grant no. 2014-67013-22418 to Patricio R. Munoz, James W. Olmstead and Jeffrey B. Endelman from the USDA National Institute of Food and Agriculture. Ivone de Bem Oliveira was funded by the CAPES (Coordenação de Aperfeiçoamento de Pessoal de Nível Superior), Grant no: 88881.131685/2016-01.</p></div><div class="section ack" id="ack-2"><h2 class="">Conflict of Interest</h2><p id="p-47">The authors declare that they have no conflict of interest.</p></div><div class="section ack" id="ack-3"><h2 class="">Acknowledgments</h2><p id="p-48">The authors thank the University of Florida blueberry breeding program technical support, especially Dr. Paul M. Lyrene, David Norden, and Werner Collante. Special thanks to James Olmstead and Catherine Cellon, who coordinate the phenotyping and genotyping of the population as part of Catherine Cellon’s MS degree.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-4"><p id="p-49">Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.7728365">https://doi.org/10.25387/g3.7728365</a>.</p></li><li class="fn" id="fn-5"><p id="p-50"><em>Communicating editor: D. J. de Koning</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2018-10-09"><span class="received-label">Received </span>October 9, 2018.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2019-02-13"><span class="accepted-label">Accepted </span>February 13, 2019.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2019 de Bem Oliveira <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1189.1" data-doi="10.1073/pnas.0630618100"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Adams</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cronn</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Percifield</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wendel</span>  <span class="cit-name-given-names">J. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Genes duplicated by polyploidy show unequal contributions to the transcriptome and organ-specific reciprocal silencing.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">100</span>: <span class="cit-fpage">4649</span>–<span class="cit-lpage">4654</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.0630618100</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0630618100%26rft_id%253Dinfo%253Apmid%252F12665616%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTAwLzgvNDY0OSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzkvNC8xMTg5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1189.2" data-doi="10.3835/plantgenome2016.01.0009"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Amadeu</span>  <span class="cit-name-given-names">R. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cellon</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Olmstead</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garcia</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Resende</span>  <span class="cit-name-given-names">M. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">AGHmatrix: R Package to construct relationship matrices for autotetraploid and diploid species: a blueberry example.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">9</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2016.01.0009</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2016.01.0009%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2016.01.0009&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1189.3" data-doi="10.1186/s12864-015-2212-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Annicchiarico</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nazzicari</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wei</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pecetti</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Accuracy of genomic selection for alfalfa biomass yield in different reference populations.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">1020</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12864-015-2212-y</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D16%26rft.spage%253D1020%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-015-2212-y%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12864-015-2212-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1189.4" data-doi="10.1007/s00122-014-2300-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ashraf</span>  <span class="cit-name-given-names">B. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jensen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Asp</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Janss</span>  <span class="cit-name-given-names">L. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Association studies using family pools of outcrossing crops based on allele-frequency estimates from DNA sequencing.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">127</span>: <span class="cit-fpage">1331</span>–<span class="cit-lpage">1341</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-014-2300-4</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D127%26rft.spage%253D1331%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-014-2300-4%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-014-2300-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1189.5" data-doi="10.1007/s00122-015-2607-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ashraf</span>  <span class="cit-name-given-names">B. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Byrne</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fé</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Czaban</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Asp</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Estimating genomic heritabilities at the level of family-pool samples of perennial ryegrass using genotyping-by-sequencing.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">129</span>: <span class="cit-fpage">45</span>–<span class="cit-lpage">52</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-015-2607-9</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D129%26rft.spage%253D45%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-015-2607-9%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-015-2607-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1189.6" data-doi="10.1007/s11032-014-0066-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bian</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ballington</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raja</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brouwer</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reid</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Patterns of simple sequence repeats in cultivated blueberries (<em>Vaccinium</em> section <em>Cyanococcus</em> spp.) and their use in revealing genetic diversity and population structure.</span> <abbr class="cit-jnl-abbrev">Mol. Breed.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">675</span>–<span class="cit-lpage">689</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s11032-014-0066-7</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBreed.%26rft.volume%253D34%26rft.spage%253D675%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11032-014-0066-7%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11032-014-0066-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1189.7" data-doi="10.1006/dbio.2001.0262"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Birchler</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bhadra</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bhadra</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Auger</span>  <span class="cit-name-given-names">D. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Dosage-dependent gene regulation in multicellular eukaryotes: implications for dosage compensation, aneuploid syndromes, and quantitative traits.</span> <abbr class="cit-jnl-abbrev">Dev. Biol.</abbr> <span class="cit-vol">234</span>: <span class="cit-fpage">275</span>–<span class="cit-lpage">288</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1006/dbio.2001.0262</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopmental%2Bbiology%26rft.stitle%253DDev%2BBiol%26rft.aulast%253DBirchler%26rft.auinit1%253DJ.%2BA.%26rft.volume%253D234%26rft.issue%253D2%26rft.spage%253D275%26rft.epage%253D288%26rft.atitle%253DDosage-dependent%2Bgene%2Bregulation%2Bin%2Bmulticellular%2Beukaryotes%253A%2Bimplications%2Bfor%2Bdosage%2Bcompensation%252C%2Baneuploid%2Bsyndromes%252C%2Band%2Bquantitative%2Btraits.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fdbio.2001.0262%26rft_id%253Dinfo%253Apmid%252F11396999%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/dbio.2001.0262&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11396999&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1189.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000169367200001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1189.8" data-doi="10.3389/fpls.2018.00513"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bourke</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Voorrips</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Visser</span>  <span class="cit-name-given-names">R. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maliepaard</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Tools for genetic studies in experimental populations of polyploids.</span> <abbr class="cit-jnl-abbrev">Front. Plant Sci.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">513</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fpls.2018.00513</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BPlant%2BSci.%26rft.volume%253D9%26rft.spage%253D513%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffpls.2018.00513%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fpls.2018.00513&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1189.9" data-doi="10.1086/282718"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bulmer</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1971</span> <span class="cit-article-title">The effect of selection on genetic variability.</span> <abbr class="cit-jnl-abbrev">Am. Nat.</abbr> <span class="cit-vol">105</span>: <span class="cit-fpage">201</span>–<span class="cit-lpage">211</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1086/282718</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BNat.%26rft.volume%253D105%26rft.spage%253D201%26rft_id%253Dinfo%253Adoi%252F10.1086%252F282718%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/282718&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1971J801400001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1189.10" data-doi="10.1007/s10681-018-2165-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cellon</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amadeu</span>  <span class="cit-name-given-names">R. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Olmstead</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mattia</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferrão</span>  <span class="cit-name-given-names">L. F. V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Estimation of genetic parameters and prediction of breeding values in an autotetraploid blueberry breeding population with extensive pedigree data.</span> <abbr class="cit-jnl-abbrev">Euphytica</abbr> <span class="cit-vol">214</span>: <span class="cit-fpage">87</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s10681-018-2165-8</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEuphytica%26rft.volume%253D214%26rft.spage%253D87%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10681-018-2165-8%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10681-018-2165-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1189.11" data-doi="10.3389/fpls.2018.00369"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cericola</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lenk</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fè</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Byrne</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jensen</span>  <span class="cit-name-given-names">C. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Optimized Use of Low-Depth Genotyping-by-Sequencing for Genomic Prediction Among Multi-Parental Family Pools and Single Plants in Perennial Ryegrass (<em>Lolium perenne</em> L.).</span> <abbr class="cit-jnl-abbrev">Front. Plant Sci.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">369</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fpls.2018.00369</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BPlant%2BSci.%26rft.volume%253D9%26rft.spage%253D369%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffpls.2018.00369%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fpls.2018.00369&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1189.12" data-doi="10.1038/nrg1711"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Comai</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">The advantages and disadvantages of being polyploid.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">836</span>–<span class="cit-lpage">846</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg1711</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DComai%26rft.auinit1%253DL.%26rft.volume%253D6%26rft.issue%253D11%26rft.spage%253D836%26rft.epage%253D846%26rft.atitle%253DThe%2Badvantages%2Band%2Bdisadvantages%2Bof%2Bbeing%2Bpolyploid.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg1711%26rft_id%253Dinfo%253Apmid%252F16304599%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg1711&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16304599&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1189.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000232990900014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1189.13" data-doi="10.1534/genetics.112.147983"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Daetwyler</span>  <span class="cit-name-given-names">H. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Calus</span>  <span class="cit-name-given-names">M. P. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pong-Wong</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hickey</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genomic Prediction in Animals and Plants: Simulation of Data, Validation, Reporting, and Benchmarking.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">193</span>: <span class="cit-fpage">347</span>–<span class="cit-lpage">365</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.112.147983</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.147983%26rft_id%253Dinfo%253Apmid%252F23222650%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkzLzIvMzQ3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS80LzExODkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-web" id="cit-9.4.1189.14"><div class="cit-metadata unstructured">de los Campos, G., and P. Pérez-Rodríguez, 2016 BGLR: Bayesian Generalized Linear Regression. R package version 1.0.5. <a href="https://cran.r-project.org/package=BGLR">https://cran.r-project.org/package=BGLR</a>.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1189.15" data-doi="10.1534/genetics.112.143313"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hickey</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pong-Wong</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daetwyler</span>  <span class="cit-name-given-names">H. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Calus</span>  <span class="cit-name-given-names">M. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Whole-genome regression and prediction methods Applied to plant and animal breeding.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">193</span>: <span class="cit-fpage">327</span>–<span class="cit-lpage">345</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.112.143313</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.143313%26rft_id%253Dinfo%253Apmid%252F22745228%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkzLzIvMzI3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS80LzExODkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1189.16" data-doi="10.1016/j.tplants.2014.05.006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Desta</span>  <span class="cit-name-given-names">Z. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ortiz</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genomic selection: genome-wide prediction in plant improvement.</span> <abbr class="cit-jnl-abbrev">Trends Plant Sci.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">592</span>–<span class="cit-lpage">601</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.tplants.2014.05.006</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BPlant%2BSci.%26rft.volume%253D19%26rft.spage%253D592%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tplants.2014.05.006%26rft_id%253Dinfo%253Apmid%252F24970707%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tplants.2014.05.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24970707&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1189.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1189.17" data-doi="10.1111/mec.12581"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dufresne</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stift</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vergilino</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mable</span>  <span class="cit-name-given-names">B. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Recent progress and challenges in population genetics of polyploid organisms: an overview of current state‐of‐the‐art molecular and statistical tools.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">40</span>–<span class="cit-lpage">69</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/mec.12581</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D23%26rft.spage%253D40%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.12581%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.12581&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000330950900007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1189.18" data-doi="10.1534/genetics.118.300685"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Endelman</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carley</span>  <span class="cit-name-given-names">C. A. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bethke</span>  <span class="cit-name-given-names">P. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coombs</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clough</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Genetic variance partitioning and genome-wide prediction with allele dosage information in autotetraploid potato.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">209</span>: <span class="cit-fpage">77</span>–<span class="cit-lpage">87</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.118.300685</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.118.300685%26rft_id%253Dinfo%253Apmid%252F29514860%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiMjA5LzEvNzciO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzQvMTE4OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1189.19" data-doi="10.3389/fevo.2018.00107"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ferrão</span>  <span class="cit-name-given-names">L. F. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Benevenuto</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Bem Oliveira</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cellon</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Olmstead</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Insights into the genetic basis of blueberry fruit-related traits using diploid and polyploid models in a GWAS context.</span> <abbr class="cit-jnl-abbrev">Front. Ecol. Evol.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">107</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fevo.2018.00107</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BEcol.%2BEvol.%26rft.volume%253D6%26rft.spage%253D107%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffevo.2018.00107%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fevo.2018.00107&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-web no-rev-xref" id="cit-9.4.1189.20"><div class="cit-metadata unstructured">Garrison, E., G. and Marth, 2012 Haplotype-based variant detection from short-read sequencing. arXiv preprint arXiv:1207.3907.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1189.21" data-doi="10.1186/s12711-015-0145-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gorjanc</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bijma</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hickey</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Reliability of pedigree-based and genomic evaluations in selected populations.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">65</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12711-015-0145-1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D47%26rft.spage%253D65%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12711-015-0145-1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12711-015-0145-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1189.22" data-doi="10.1007/s00122-013-2156-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gouy</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rousselle</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bastianelli</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lecomte</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bonnal</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Experimental assessment of the accuracy of genomic selection in sugarcane.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">126</span>: <span class="cit-fpage">2575</span>–<span class="cit-lpage">2586</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-013-2156-z</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D126%26rft.spage%253D2575%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-013-2156-z%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-013-2156-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1189.23" data-doi="10.1186/s12864-016-2926-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Grandke</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Singh</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heuven</span>  <span class="cit-name-given-names">H. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">De Haan</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Metzler</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Advantages of continuous genotype values over genotype classes for GWAS in higher polyploids: a comparative study in hexaploid chrysanthemum.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">672</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12864-016-2926-5</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D17%26rft.spage%253D672%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-016-2926-5%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12864-016-2926-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1189.24"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Birchler</span>  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Dosage effects on gene expression in a maize ploidy series.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">142</span>: <span class="cit-fpage">1349</span>–<span class="cit-lpage">1355</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DGuo%26rft.auinit1%253DM.%26rft.volume%253D142%26rft.issue%253D4%26rft.spage%253D1349%26rft.epage%253D1355%26rft.atitle%253DDosage%2BEffects%2Bon%2BGene%2BExpression%2Bin%2Ba%2BMaize%2BPloidy%2BSeries%26rft_id%253Dinfo%253Apmid%252F8846910%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE0Mi80LzEzNDkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzQvMTE4OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1189.25" data-doi="10.3389/fpls.2018.01165"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cericola</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fè</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pedersen</span>  <span class="cit-name-given-names">M. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lenk</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Genomic Prediction in Tetraploid Ryegrass Using Allele Frequencies Based on Genotyping by Sequencing.</span> <abbr class="cit-jnl-abbrev">Front. Plant Sci.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">1165</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fpls.2018.01165</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BPlant%2BSci.%26rft.volume%253D9%26rft.spage%253D1165%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffpls.2018.01165%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fpls.2018.01165&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1189.26" data-doi="10.1186/s13742-015-0046-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gupta</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Estrada</span>  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blakley</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reid</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Patel</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">RNA-Seq analysis and annotation of a draft blueberry genome assembly identifies candidate genes involved in fruit ripening, biosynthesis of bioactive compounds, and stage-specific alternative splicing.</span> <abbr class="cit-jnl-abbrev">Gigascience</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">5</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s13742-015-0046-9</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGigascience%26rft.volume%253D4%26rft.spage%253D5%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13742-015-0046-9%26rft_id%253Dinfo%253Apmid%252F25830017%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13742-015-0046-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25830017&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1189.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-2" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1189.27" data-doi="10.2307/2529339"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Henderson</span>  <span class="cit-name-given-names">C. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1976</span> <span class="cit-article-title">A simple method for computing the inverse of a numerator relationship matrix used in prediction of breeding values.</span> <abbr class="cit-jnl-abbrev">Biometrics</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">69</span>–<span class="cit-lpage">83</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2307/2529339</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiometrics%26rft.volume%253D32%26rft.spage%253D69%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2529339%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2529339&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1976BM07600004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1189.28" data-doi="10.1126/science.285.5425.210"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hieter</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Griffiths</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Polyploidy–more is more or less.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">285</span>: <span class="cit-fpage">210</span>–<span class="cit-lpage">211</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.285.5425.210</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DHieter%26rft.auinit1%253DP.%26rft.volume%253D285%26rft.issue%253D5425%26rft.spage%253D210%26rft.epage%253D211%26rft.atitle%253DGENETICS%253APolyploidy--More%2BIs%2BMore%2Bor%2BLess%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.285.5425.210%26rft_id%253Dinfo%253Apmid%252F10428719%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIyODUvNTQyNS8yMTAiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzQvMTE4OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1189.29" data-doi="10.1093/bioinformatics/btr521"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jombart</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ahmed</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">adegenet 1.3–1: new tools for the analysis of genome-wide SNP data.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">3070</span>–<span class="cit-lpage">3071</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btr521</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtr521%26rft_id%253Dinfo%253Apmid%252F21926124%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btr521&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21926124&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1189.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000296099300022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-book" id="cit-9.4.1189.30"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kempthorne</span>  <span class="cit-name-given-names">O.</span></span></li></ol><cite>, <span class="cit-pub-date">1957</span> <span class="cit-source">An introduction to genetic statistics</span>, <span class="cit-publ-name">John Wiley And Sons, Inc.</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1189.31" data-doi="10.1007/s00122-012-1785-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kerr</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tier</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dutkowski</span>  <span class="cit-name-given-names">G. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McRae</span>  <span class="cit-name-given-names">T. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Use of the numerator relationship matrix in genetic analysis of autopolyploid species.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">124</span>: <span class="cit-fpage">1271</span>–<span class="cit-lpage">1282</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-012-1785-y</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DKerr%26rft.auinit1%253DR.%2BJ.%26rft.volume%253D124%26rft.issue%253D7%26rft.spage%253D1271%26rft.epage%253D1282%26rft.atitle%253DUse%2Bof%2Bthe%2Bnumerator%2Brelationship%2Bmatrix%2Bin%2Bgenetic%2Banalysis%2Bof%2Bautopolyploid%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-012-1785-y%26rft_id%253Dinfo%253Apmid%252F22311370%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-012-1785-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22311370&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1189.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1189.32" data-doi="10.1371/journal.pone.0090581"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">W. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stromberg</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ward</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stewart</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garrison</span>  <span class="cit-name-given-names">E. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">MOSAIK: a hash-based algorithm for accurate next-generation sequencing short-read mapping.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e90581</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0090581</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De90581%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0090581%26rft_id%253Dinfo%253Apmid%252F24599324%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0090581&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24599324&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1189.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1189.33"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lyrene</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Development of highbush blueberry cultivars adapted to Florida.</span> <abbr class="cit-jnl-abbrev">J. Am. Pomol. Soc.</abbr> <span class="cit-vol">56</span>: <span class="cit-fpage">79</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAm.%2BPomol.%2BSoc.%26rft.volume%253D56%26rft.spage%253D79%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1189.34" data-doi="10.1002/9780470380130.ch8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lyrene</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Breeding southern highbush blueberries.</span> <abbr class="cit-jnl-abbrev">Plant Breed. Rev.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">353</span>–<span class="cit-lpage">414</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/9780470380130.ch8</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BBreed.%2BRev.%26rft.volume%253D30%26rft.spage%253D353%26rft.atitle%253DBREEDING%2BSOUTHERN%2BHIGHBUSH%2BBLUEBERRIES%26rft_id%253Dinfo%253Adoi%252F10.1002%252F9780470380130.ch8%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/9780470380130.ch8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1189.35" data-doi="10.1093/nar/gkv1344"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>  <span class="cit-name-given-names">T. H. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Prediction of total genetic value using genome-wide dense marker maps.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">157</span>: <span class="cit-fpage">1819</span>–<span class="cit-lpage">1829</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMeuwissen%26rft.auinit1%253DT.%2BH.%2BE.%26rft.volume%253D157%26rft.issue%253D4%26rft.spage%253D1819%26rft.epage%253D1829%26rft.atitle%253DPrediction%2Bof%2BTotal%2BGenetic%2BValue%2BUsing%2BGenome-Wide%2BDense%2BMarker%2BMaps%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkv1344%26rft_id%253Dinfo%253Apmid%252F11290733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1Ny80LzE4MTkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzQvMTE4OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-book" id="cit-9.4.1189.36"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mollinari</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Serang</span>  <span class="cit-name-given-names">O.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Quantitative SNP genotyping of polyploids with MassARRAY and other platforms</span>, pp. <span class="cit-fpage">215</span>–<span class="cit-lpage">241</span> in <span class="cit-source">Plant Genotyping</span> edited by <span class="cit-publ-name">Humana Press</span>, <span class="cit-publ-loc">New York, NY</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1189.37" data-doi="10.1111/j.1439-0388.2007.00700.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Muir</span>  <span class="cit-name-given-names">W. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Comparison of genomic and traditional BLUP‐estimated breeding value accuracy and selection response under alternative trait and genomic parameters.</span> <abbr class="cit-jnl-abbrev">J. Anim. Breed. Genet.</abbr> <span class="cit-vol">124</span>: <span class="cit-fpage">342</span>–<span class="cit-lpage">355</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1439-0388.2007.00700.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DJ%2BAnim%2BBreed%2BGenet%26rft.aulast%253DMuir%26rft.auinit1%253DW.%2BM.%26rft.volume%253D124%26rft.issue%253D6%26rft.spage%253D342%26rft.epage%253D355%26rft.atitle%253DComparison%2Bof%2Bgenomic%2Band%2Btraditional%2BBLUP-estimated%2Bbreeding%2Bvalue%2Baccuracy%2Band%2Bselection%2Bresponse%2Bunder%2Balternative%2Btrait%2Band%2Bgenomic%2Bparameters.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1439-0388.2007.00700.x%26rft_id%253Dinfo%253Apmid%252F18076471%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1439-0388.2007.00700.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18076471&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1189.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000251432800004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1189.38" data-doi="10.2135/cropsci2012.12.0673"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Muñoz</span>  <span class="cit-name-given-names">P. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Resende</span>  <span class="cit-name-given-names">M. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huber</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Quesada</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Resende</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span><span class="cit-comment">a</span> <span class="cit-article-title">Genomic relationship matrix for correcting pedigree errors in breeding populations: impact on genetic parameters and genomic selection accuracy.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">54</span>: <span class="cit-fpage">1115</span>–<span class="cit-lpage">1123</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2012.12.0673</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D54%26rft.spage%253D1115%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2012.12.0673%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2012.12.0673&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1189.39" data-doi="10.1534/genetics.114.171322"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Muñoz</span>  <span class="cit-name-given-names">P. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Resende</span>  <span class="cit-name-given-names">M. F.</span>  <span class="cit-name-suffix">Jr.</span>.</span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gezan</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Resende</span>  <span class="cit-name-given-names">M. D. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span><span class="cit-comment">b</span> <span class="cit-article-title">Unraveling additive from nonadditive effects using genomic relationship matrices.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">198</span>: <span class="cit-fpage">1759</span>–<span class="cit-lpage">1768</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.114.171322</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.171322%26rft_id%253Dinfo%253Apmid%252F25324160%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5OC80LzE3NTkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzQvMTE4OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1189.40" data-doi="10.3835/plantgenome2017.10.0090"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nyine</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Uwimana</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blavet</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hřibovà</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vanrespaille</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Genomic prediction in a multiploid crop: genotype by environment interaction and allele dosage effects on predictive ability in banana.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">11</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2017.10.0090</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2017.10.0090%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2017.10.0090&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1189.41" data-doi="10.1016/S0168-9525(03)00015-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Osborn</span>  <span class="cit-name-given-names">T. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pires</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Birchler</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Auger</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">Z. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Understanding mechanisms of novel gene expression in polyploids.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">141</span>–<span class="cit-lpage">147</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/S0168-9525(03)00015-5</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DOsborn%26rft.auinit1%253DT.%2BC.%26rft.volume%253D19%26rft.issue%253D3%26rft.spage%253D141%26rft.epage%253D147%26rft.atitle%253DUnderstanding%2Bmechanisms%2Bof%2Bnovel%2Bgene%2Bexpression%2Bin%2Bpolyploids.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0168-9525%252803%252900015-5%26rft_id%253Dinfo%253Apmid%252F12615008%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0168-9525(03)00015-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12615008&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1189.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000181584500009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1189.42" data-doi="10.1534/genetics.114.164442"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pérez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genome-wide regression and prediction with the BGLR statistical package.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">198</span>: <span class="cit-fpage">483</span>–<span class="cit-lpage">495</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.114.164442</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.164442%26rft_id%253Dinfo%253Apmid%252F25009151%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk4LzIvNDgzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS80LzExODkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-web" id="cit-9.4.1189.43"><div class="cit-metadata unstructured">R Core Team, 2018  R: A language and environment for statistical computing. R Foundation for Statistical Computing. Vienna. Austria. ISBN 3–900051–07–0. URL <a href="http://www.R-project.org/">http://www.R-project.org/</a>.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1189.44" data-doi="10.1111/j.1469-8137.2011.03895.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Resende</span>  <span class="cit-name-given-names">M. F. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Muñoz</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Acosta</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peter</span>  <span class="cit-name-given-names">G. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span><span class="cit-comment">a</span> <span class="cit-article-title">Accelerating the domestication of trees using genomic selection: accuracy of prediction models across ages and environments.</span> <abbr class="cit-jnl-abbrev">New Phytol.</abbr> <span class="cit-vol">193</span>: <span class="cit-fpage">617</span>–<span class="cit-lpage">624</span> (erratum: New Phytol. 193: 1099).<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1469-8137.2011.03895.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytologist%26rft.stitle%253DNew%2BPhytologist%26rft.aulast%253DResende%26rft.auinit1%253DM.%2BF.%26rft.volume%253D193%26rft.issue%253D3%26rft.spage%253D617%26rft.epage%253D624%26rft.atitle%253DAccelerating%2Bthe%2Bdomestication%2Bof%2Btrees%2Busing%2Bgenomic%2Bselection%253A%2Baccuracy%2Bof%2Bprediction%2Bmodels%2Bacross%2Bages%2Band%2Benvironments.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1469-8137.2011.03895.x%26rft_id%253Dinfo%253Apmid%252F21973055%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1469-8137.2011.03895.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21973055&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1189.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000298984900008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1189.45" data-doi="10.1038/hdy.2017.37"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Resende</span>  <span class="cit-name-given-names">R. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Resende</span>  <span class="cit-name-given-names">M. D. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Silva</span>  <span class="cit-name-given-names">F. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Azevedo</span>  <span class="cit-name-given-names">C. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Takahashi</span>  <span class="cit-name-given-names">E. K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Assessing the expected response to genomic selection of individuals and families in Eucalyptus breeding with an additive-dominant model.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">119</span>: <span class="cit-fpage">245</span>–<span class="cit-lpage">255</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/hdy.2017.37</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D119%26rft.spage%253D245%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2017.37%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2017.37&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1189.46" data-doi="10.1534/genetics.111.137026"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Resende</span>  <span class="cit-name-given-names">M. F. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Muñoz</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Resende</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garrick</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">L. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span><span class="cit-comment">b</span> <span class="cit-article-title">Accuracy of genomic selection methods in a standard data set of loblolly pine (<em>Pinus taeda</em> L.).</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">190</span>: <span class="cit-fpage">1503</span>–<span class="cit-lpage">1510</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.111.137026</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.137026%26rft_id%253Dinfo%253Apmid%252F22271763%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5MC80LzE1MDMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzQvMTE4OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1189.47" data-doi="10.1038/nmeth.2019"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schindelin</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arganda-Carreras</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Frise</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kaynig</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Longair</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Fiji: an open-source platform for biological-image analysis.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">676</span>–<span class="cit-lpage">682</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nmeth.2019</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DSchindelin%26rft.auinit1%253DJ.%26rft.volume%253D9%26rft.issue%253D7%26rft.spage%253D676%26rft.epage%253D682%26rft.atitle%253DFiji%253A%2Ban%2Bopen-source%2Bplatform%2Bfor%2Bbiological-image%2Banalysis.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.2019%26rft_id%253Dinfo%253Apmid%252F22743772%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.2019&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22743772&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1189.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000305942200021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1189.48" data-doi="10.1007/s00122-016-2845-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schmitz Carley</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coombs</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Douches</span>  <span class="cit-name-given-names">D. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bethke</span>  <span class="cit-name-given-names">P. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Palta</span>  <span class="cit-name-given-names">J. P.</span></span></li></ol><cite><span class="cit-etal">et al.</span>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Automated tetraploid genotype calling by hierarchical clustering.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">130</span>: <span class="cit-fpage">717</span>–<span class="cit-lpage">726</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-016-2845-5</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D130%26rft.spage%253D717%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-016-2845-5%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-016-2845-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1189.49" data-doi="10.1371/journal.pone.0030906"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Serang</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mollinari</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garcia</span>  <span class="cit-name-given-names">A. A. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Efficient exact maximum a posteriori computation for bayesian SNP genotyping in polyploids.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e30906</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0030906</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DSerang%26rft.auinit1%253DO.%26rft.volume%253D7%26rft.issue%253D2%26rft.spage%253De30906%26rft.epage%253De30906%26rft.atitle%253DEfficient%2Bexact%2Bmaximum%2Ba%2Bposteriori%2Bcomputation%2Bfor%2Bbayesian%2BSNP%2Bgenotyping%2Bin%2Bpolyploids.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0030906%26rft_id%253Dinfo%253Apmid%252F22363513%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0030906&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22363513&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1189.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1189.50" data-doi="10.1007/s00122-013-2258-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Slater</span>  <span class="cit-name-given-names">A. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilson</span>  <span class="cit-name-given-names">G. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cogan</span>  <span class="cit-name-given-names">N. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Forster</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Improving the analysis of low heritability complex traits for enhanced genetic gain in potato.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">127</span>: <span class="cit-fpage">809</span>–<span class="cit-lpage">820</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-013-2258-7</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D127%26rft.spage%253D809%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-013-2258-7%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-013-2258-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1189.51" data-doi="10.3835/plantgenome2016.02.0021"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Slater</span>  <span class="cit-name-given-names">A. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cogan</span>  <span class="cit-name-given-names">N. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Forster</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daetwyler</span>  <span class="cit-name-given-names">H. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Improving genetic gain with genomic selection in autotetraploid potato.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">9</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2016.02.0021</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2016.02.0021%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2016.02.0021&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1189.52" data-doi="10.3732/ajb.1500501"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Soltis</span>  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Visger</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marchant</span>  <span class="cit-name-given-names">D. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Soltis</span>  <span class="cit-name-given-names">P. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Polyploidy: pitfalls and paths to a paradigm.</span> <abbr class="cit-jnl-abbrev">Am. J. Bot.</abbr> <span class="cit-vol">103</span>: <span class="cit-fpage">1146</span>–<span class="cit-lpage">1166</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3732/ajb.1500501</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BJ.%2BBot.%26rft_id%253Dinfo%253Adoi%252F10.3732%252Fajb.1500501%26rft_id%253Dinfo%253Apmid%252F27234228%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiYW1qYm90IjtzOjU6InJlc2lkIjtzOjEwOiIxMDMvNy8xMTQ2IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS80LzExODkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1189.53" data-doi="10.2307/25065732)"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Soltis</span>  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Soltis</span>  <span class="cit-name-given-names">P. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schemske</span>  <span class="cit-name-given-names">D. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hancock</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">J. N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Autopolyploidy in angiosperms: have we grossly underestimated the number of species?</span> <abbr class="cit-jnl-abbrev">Taxon</abbr> <span class="cit-vol">56</span>: <span class="cit-fpage">13</span>–<span class="cit-lpage">30</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTaxon%26rft.volume%253D56%26rft.spage%253D13%26rft_id%253Dinfo%253Adoi%252F10.2307%252F25065732%2529%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/25065732)&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000244825300003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1189.54" data-doi="10.1016/S0169-5347(99)01638-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Soltis</span>  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Soltis</span>  <span class="cit-name-given-names">P. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Polyploidy: recurrent formation and genome evolution.</span> <abbr class="cit-jnl-abbrev">Trends Ecol. Evol.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">348</span>–<span class="cit-lpage">352</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/S0169-5347(99)01638-9</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2BEcology%2B%2526%2BEvolution%26rft.stitle%253DTrends%2Bin%2BEcology%2B%2526%2BEvolution%26rft.aulast%253DSoltis%26rft.auinit1%253DD.%2BE.%26rft.volume%253D14%26rft.issue%253D9%26rft.spage%253D348%26rft.epage%253D352%26rft.atitle%253DPolyploidy%253A%2Brecurrent%2Bformation%2Band%2Bgenome%2Bevolution.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0169-5347%252899%252901638-9%26rft_id%253Dinfo%253Apmid%252F10441308%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0169-5347(99)01638-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10441308&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1189.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000081781400006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1189.55" data-doi="10.1111/1467-9868.00353"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Spiegelhalter</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Best</span>  <span class="cit-name-given-names">N. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carlin</span>  <span class="cit-name-given-names">B. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van Der Linde</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Bayesian measures of model complexity and fit.</span> <abbr class="cit-jnl-abbrev">J. R. Stat. Soc. Series B Stat. Methodol.</abbr> <span class="cit-vol">64</span>: <span class="cit-fpage">583</span>–<span class="cit-lpage">639</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/1467-9868.00353</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BR.%2BStat.%2BSoc.%2BSeries%2BB%2BStat.%2BMethodol.%26rft.volume%253D64%26rft.spage%253D583%26rft_id%253Dinfo%253Adoi%252F10.1111%252F1467-9868.00353%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/1467-9868.00353&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1189.56" data-doi="10.1111/jse.12253"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Spoelhof</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Soltis</span>  <span class="cit-name-given-names">P. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Soltis</span>  <span class="cit-name-given-names">D. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Pure polyploidy: closing the gaps in autopolyploid research.</span> <abbr class="cit-jnl-abbrev">J. Syst. Evol.</abbr> <span class="cit-vol">55</span>: <span class="cit-fpage">340</span>–<span class="cit-lpage">352</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/jse.12253</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BSyst.%2BEvol.%26rft.volume%253D55%26rft.spage%253D340%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fjse.12253%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/jse.12253&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1189.57" data-doi="10.1007/s00122-017-2944-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sverrisdóttir</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Byrne</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sundmark</span>  <span class="cit-name-given-names">E. H. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnsen</span>  <span class="cit-name-given-names">H. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kirk</span>  <span class="cit-name-given-names">H. G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Genomic prediction of starch content and chipping quality in tetraploid potato using genotyping-by-sequencing.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">130</span>: <span class="cit-fpage">2091</span>–<span class="cit-lpage">2108</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-017-2944-y</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D130%26rft.spage%253D2091%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-017-2944-y%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-017-2944-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1189.58" data-doi="10.1371/journal.pone.0062355"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Uitdewilligen</span>  <span class="cit-name-given-names">J. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wolters</span>  <span class="cit-name-given-names">A. M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bjorn</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Borm</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Visser</span>  <span class="cit-name-given-names">R. G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">A next-generation sequencing method for genotyping-by-sequencing of highly heterozygous autotetraploid potato.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e62355</span> (erratum: PLoS One 10: e0141940).<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0062355</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D8%26rft.spage%253De62355%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0062355%26rft_id%253Dinfo%253Apmid%252F23667470%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0062355&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23667470&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1189.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-2" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1189.59" data-doi="10.3168/jds.2007-0980"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">VanRaden</span>  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Efficient methods to compute genomic predictions.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">4414</span>–<span class="cit-lpage">4423</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3168/jds.2007-0980</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DVanRaden%26rft.auinit1%253DP.%2BM.%26rft.volume%253D91%26rft.issue%253D11%26rft.spage%253D4414%26rft.epage%253D4423%26rft.atitle%253DEfficient%2BMethods%2Bto%2BCompute%2BGenomic%2BPredictions%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2007-0980%26rft_id%253Dinfo%253Apmid%252F18946147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2007-0980&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18946147&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1189.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000260277200035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1189.60" data-doi="10.1186/1753-6561-5-S3-S15"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ding</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Koning</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Q.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genomic selection for QTL-MAS data using a trait-specific relationship matrix.</span> <abbr class="cit-jnl-abbrev">BMC Proc.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">S15</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1753-6561-5-S3-S15</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BProc.%26rft.volume%253D5%26rft.spage%253DS15%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1753-6561-5-S3-S15%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1753-6561-5-S3-S15&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/9/4/1189.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/4/1175" title="Mitochondrial DNA Fitness Depends on Nuclear Genetic Background in Drosophila" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/4/1199" title="A Novel Root-Knot Nematode Resistance QTL on Chromosome Vu01 in Cowpea" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/9/4"><span>Volume 9 Issue 4, April 2019</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/9/4.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/9/4.cover-source.jpg?itok=mcy5SuIK" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 9 (4)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/4/1189" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-DbbpVKNK8rBsQFfWW3_b3uXcV5V7EkK2UTwkaRWnQqI" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/genomic-prediction" class="" data-icon-position="" data-hide-link-title="0">Genomic Prediction</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/9/4/1189" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/9/4/1189" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/9/4/1189" class="active" data-icon-position="" data-hide-link-title="0">Genomic Prediction of Autotetraploids; Influence of Relationship Matrices, Allele Dosage, and Continuous Genotyping Calls in Phenotype Prediction</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/492601" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-yFEi3xSA12N3a9oRxNn7NCAKpeMEeR2ffdOxLuP8S_Y" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/4/1189.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/9/4/1189" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-Hpeg4J1dDJCwufBSZpojI7YsAZmEfYOVql4Z376wX08" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/9/4/1189" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/4/1189.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="492601" id="node492601--2" data-pisa="ggg;9/4/1189" data-pisa-master="ggg;g3.119.400059" data-apath="/ggg/9/4/1189.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genomic Prediction of Autotetraploids; Influence of Relationship Matrices, Allele Dosage, and Continuous Genotyping Calls in Phenotype Prediction</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-3723-9747" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Ivone de Bem Oliveira</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-2367-0766" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Marcio F. R. Resende</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-9655-4838" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Luis Felipe V. Ferrão</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0001-5127-4448" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Rodrigo R. Amadeu</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0003-0957-4337" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jeffrey B. Endelman</span>, <span class="highwire-citation-author" data-delta="5">Matias Kirst</span>, <span class="highwire-citation-author" data-delta="6">Alexandre S. G. Coelho</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="7"><a href="http://orcid.org/0000-0001-8973-9351" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Patricio R. Munoz</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1189-1198;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400059 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node492601--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Ivone de Bem Oliveira </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Blueberry Breeding and Genomics Lab</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Plant Genetics and Genomics Lab, Agronomy College, Federal University of Goias, GO, Brazil, 74690-900</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ivone%2Bde%2BBem%2BOliveira%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=de%20Bem%20Oliveira%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AIvone%2Bde%2BBem%2BOliveira%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-3723-9747" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Ivone de Bem Oliveira</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Marcio F. R. Resende <span class="nlm-suffix">Jr.</span></div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Sweet Corn Genomics and Breeding, Horticultural Sciences Department, University of Florida, Gainesville, FL 32611</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marcio%2BF.%2BR.%2BResende%2B%3Cspan%2Bclass%3D%22nlm-suffix%22%3EJr.%3C/span%3E" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Resende%20MF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMarcio%2BF.%2BR.%2BResende%2BJr." rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-2367-0766" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Marcio F. R. Resende <span class="nlm-suffix">Jr.</span></a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Luis Felipe V. Ferrão </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Blueberry Breeding and Genomics Lab</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Luis%2BFelipe%2BV.%2BFerr%C3%A3o%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ferr%C3%A3o%20LF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALuis%2BFelipe%2BV.%2BFerr%25C3%25A3o%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-9655-4838" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Luis Felipe V. Ferrão</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Rodrigo R. Amadeu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Blueberry Breeding and Genomics Lab</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rodrigo%2BR.%2BAmadeu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Amadeu%20RR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARodrigo%2BR.%2BAmadeu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-5127-4448" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Rodrigo R. Amadeu</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Jeffrey B. Endelman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Horticulture, University of Wisconsin, Madison, WI 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jeffrey%2BB.%2BEndelman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Endelman%20JB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJeffrey%2BB.%2BEndelman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-0957-4337" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jeffrey B. Endelman</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Matias Kirst </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Forest Genomics Lab, School of Forestry Resources and Conservation, University of Florida, Gainesville, FL 32610</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Matias%2BKirst%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kirst%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMatias%2BKirst%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Alexandre S. G. Coelho </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Plant Genetics and Genomics Lab, Agronomy College, Federal University of Goias, GO, Brazil, 74690-900</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alexandre%2BS.%2BG.%2BCoelho%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Coelho%20AS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlexandre%2BS.%2BG.%2BCoelho%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Patricio R. Munoz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Blueberry Breeding and Genomics Lab</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Patricio%2BR.%2BMunoz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Munoz%20PR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APatricio%2BR.%2BMunoz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-8973-9351" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Patricio R. Munoz</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:p.munoz@ufl.edu" class="" data-icon-position="" data-hide-link-title="0">p.munoz@ufl.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/9/4/1189" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/9/4/1189&amp;title=Genomic%20Prediction%20of%20Autotetraploids%3B%20Influence%20of%20Relationship%20Matrices%2C%20Allele%20Dosage%2C%20and%20Continuous%20Genotyping%20Calls%20in%20Phenotype%20Prediction" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/9/4/1189&amp;title=Genomic%20Prediction%20of%20Autotetraploids%3B%20Influence%20of%20Relationship%20Matrices%2C%20Allele%20Dosage%2C%20and%20Continuous%20Genotyping%20Calls%20in%20Phenotype%20Prediction" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/9/4/1189&amp;title=Genomic%20Prediction%20of%20Autotetraploids%3B%20Influence%20of%20Relationship%20Matrices%2C%20Allele%20Dosage%2C%20and%20Continuous%20Genotyping%20Calls%20in%20Phenotype%20Prediction" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/9/4/1189&amp;text=Genomic%20Prediction%20of%20Autotetraploids%3B%20Influence%20of%20Relationship%20Matrices%2C%20Allele%20Dosage%2C%20and%20Continuous%20Genotyping%20Calls%20in%20Phenotype%20Prediction" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/9/4/1189&amp;title=Genomic%20Prediction%20of%20Autotetraploids%3B%20Influence%20of%20Relationship%20Matrices%2C%20Allele%20Dosage%2C%20and%20Continuous%20Genotyping%20Calls%20in%20Phenotype%20Prediction" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/9/4/1189&amp;t=Genomic%20Prediction%20of%20Autotetraploids%3B%20Influence%20of%20Relationship%20Matrices%2C%20Allele%20Dosage%2C%20and%20Continuous%20Genotyping%20Calls%20in%20Phenotype%20Prediction" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/9/4/1189&amp;title=Genomic%20Prediction%20of%20Autotetraploids%3B%20Influence%20of%20Relationship%20Matrices%2C%20Allele%20Dosage%2C%20and%20Continuous%20Genotyping%20Calls%20in%20Phenotype%20Prediction" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/9/4/1189&amp;title=Genomic%20Prediction%20of%20Autotetraploids%3B%20Influence%20of%20Relationship%20Matrices%2C%20Allele%20Dosage%2C%20and%20Continuous%20Genotyping%20Calls%20in%20Phenotype%20Prediction" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/492601/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="492601" id="node492601--3" data-pisa="ggg;9/4/1189" data-pisa-master="ggg;g3.119.400059" data-apath="/ggg/9/4/1189.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genomic Prediction of Autotetraploids; Influence of Relationship Matrices, Allele Dosage, and Continuous Genotyping Calls in Phenotype Prediction</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-3723-9747" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Ivone de Bem Oliveira</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-2367-0766" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Marcio F. R. Resende</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-9655-4838" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Luis Felipe V. Ferrão</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0001-5127-4448" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Rodrigo R. Amadeu</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0003-0957-4337" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jeffrey B. Endelman</span>, <span class="highwire-citation-author" data-delta="5">Matias Kirst</span>, <span class="highwire-citation-author" data-delta="6">Alexandre S. G. Coelho</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="7"><a href="http://orcid.org/0000-0001-8973-9351" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Patricio R. Munoz</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1189-1198;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400059 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node492601--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Ivone de Bem Oliveira </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Blueberry Breeding and Genomics Lab</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Plant Genetics and Genomics Lab, Agronomy College, Federal University of Goias, GO, Brazil, 74690-900</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ivone%2Bde%2BBem%2BOliveira%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=de%20Bem%20Oliveira%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AIvone%2Bde%2BBem%2BOliveira%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-3723-9747" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Ivone de Bem Oliveira</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Marcio F. R. Resende <span class="nlm-suffix">Jr.</span></div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Sweet Corn Genomics and Breeding, Horticultural Sciences Department, University of Florida, Gainesville, FL 32611</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marcio%2BF.%2BR.%2BResende%2B%3Cspan%2Bclass%3D%22nlm-suffix%22%3EJr.%3C/span%3E" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Resende%20MF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMarcio%2BF.%2BR.%2BResende%2BJr." rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-2367-0766" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Marcio F. R. Resende <span class="nlm-suffix">Jr.</span></a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Luis Felipe V. Ferrão </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Blueberry Breeding and Genomics Lab</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Luis%2BFelipe%2BV.%2BFerr%C3%A3o%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ferr%C3%A3o%20LF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALuis%2BFelipe%2BV.%2BFerr%25C3%25A3o%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-9655-4838" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Luis Felipe V. Ferrão</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Rodrigo R. Amadeu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Blueberry Breeding and Genomics Lab</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rodrigo%2BR.%2BAmadeu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Amadeu%20RR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARodrigo%2BR.%2BAmadeu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-5127-4448" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Rodrigo R. Amadeu</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Jeffrey B. Endelman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Horticulture, University of Wisconsin, Madison, WI 53706</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jeffrey%2BB.%2BEndelman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Endelman%20JB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJeffrey%2BB.%2BEndelman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-0957-4337" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jeffrey B. Endelman</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Matias Kirst </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Forest Genomics Lab, School of Forestry Resources and Conservation, University of Florida, Gainesville, FL 32610</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Matias%2BKirst%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kirst%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMatias%2BKirst%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Alexandre S. G. Coelho </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Plant Genetics and Genomics Lab, Agronomy College, Federal University of Goias, GO, Brazil, 74690-900</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alexandre%2BS.%2BG.%2BCoelho%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Coelho%20AS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlexandre%2BS.%2BG.%2BCoelho%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Patricio R. Munoz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Blueberry Breeding and Genomics Lab</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Patricio%2BR.%2BMunoz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Munoz%20PR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APatricio%2BR.%2BMunoz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-8973-9351" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Patricio R. Munoz</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:p.munoz@ufl.edu" class="" data-icon-position="" data-hide-link-title="0">p.munoz@ufl.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1189&amp;rft.epage=1198&amp;rft.atitle=Genomic%20Prediction%20of%20Autotetraploids%3B%20Influence%20of%20Relationship%20Matrices%2C%20Allele%20Dosage%2C%20and%20Continuous%20Genotyping%20Calls%20in%20Phenotype%20Prediction&amp;rft.volume=9&amp;rft.issue=4&amp;rft.date=2019-04-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=de+Bem+Oliveira%2C+Ivone&amp;rft.au=Resende%2C+Marcio+F.+R.&amp;rft.au=Ferr%C3%A3o%2C+Luis+Felipe+V.&amp;rft.au=Amadeu%2C+Rodrigo+R.&amp;rft.au=Endelman%2C+Jeffrey+B.&amp;rft.au=Kirst%2C+Matias&amp;rft.au=Coelho%2C+Alexandre+S.+G.&amp;rft.au=Munoz%2C+Patricio+R."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/492601/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/492601/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/492601/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/492601/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/492601/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/492601/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/492601/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/492601/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/492601/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/492601/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/492601/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/492601/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495864" id="node495864" data-pisa="ggg;10/3/1113" data-pisa-master="ggg;g3.119.400968" data-apath="/ggg/10/3/1113.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1113" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Multi-trait Genomic Prediction Model Increased the Predictive Ability for Agronomic and Malting Quality Traits in Barley (<em>Hordeum vulgare</em> L.)</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495543" id="node495543" data-pisa="ggg;10/2/769" data-pisa-master="ggg;g3.119.400759" data-apath="/ggg/10/2/769.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/2/769" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Novel Bayesian Networks for Genomic Prediction of Developmental Traits in Biomass Sorghum</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495557" id="node495557" data-pisa="ggg;10/2/539" data-pisa-master="ggg;g3.119.400866" data-apath="/ggg/10/2/539.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/2/539" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Optimized Genetic Testing for Polledness in Multiple Breeds of Cattle</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Genomic%20Prediction" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Genomic Prediction</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/9/4/1189" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Material and Methods</a></li><li class="odd"><a href="#sec-10" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-16" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Funding</a></li><li class="even"><a href="#ack-2" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Conflict of Interest</a></li><li class="odd"><a href="#ack-3" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/9/4/1189.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/9/4/1189.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/9/4/1189.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/9/4/1189.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
