<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/9/4/1153" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/9/4/1153.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/9/4/1153.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/9/4/1153.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/9/4/1153.atom" />
<meta name="HW.pisa" content="ggg;9/4/1153" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Short-Homology-Mediated CRISPR/Cas9-Based Method for Genome Editing in Fission Yeast" />
<meta name="DC.Identifier" content="10.1534/g3.118.200976" />
<meta name="DC.Date" content="2019-04-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2019 Hayashi, Tanaka. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Relation" content="https://doi.org/10.25387/g3.7685642" />
<meta name="DC.Description" content="The CRISPR/Cas9 system enables the editing of genomes of numerous organisms through the induction of the double-strand breaks (DSB) at specific chromosomal targets. We improved the CRISPR/Cas9 system to ease the direct introduction of a point mutation or a tagging sequence into the chromosome by combining it with the noncanonical homology-directed DNA repair (HDR) based genome editing in fission yeast. We constructed convenient cloning vectors, which possessed a guide RNA (gRNA) expression module, or the humanized Streptococcus pyogenes Cas9 gene that is expressed under the control of an inducible promoter to avoid the needless expression, or both a gRNA and Cas9 gene. Using this system, we attempted the short-homology-mediated genome editing and found that the HDR pathway provides high-frequency genome editing at target loci without the need of a long donor DNA. Using short oligonucleotides, we successfully introduced point mutations into two target genes at high frequency. We also precisely integrated the sequences for epitope and GFP tagging using donor DNA possessing short homology into the target loci, which enabled us to obtain cells expressing N-terminally tagged fusion proteins. This system could expedite genome editing in fission yeast, and could be applicable to other organisms." />
<meta name="DC.Contributor" content="Aki Hayashi" />
<meta name="DC.Contributor" content="Katsunori Tanaka" />
<meta name="article:published_time" content="2019-04-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Short-Homology-Mediated CRISPR/Cas9-Based Method for Genome Editing in Fission Yeast" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;The CRISPR/Cas9 system enables the editing of genomes of numerous organisms through the induction of the double-strand breaks (DSB) at specific chromosomal targets. We improved the CRISPR/Cas9 system to ease the direct introduction of a point mutation or a tagging sequence into the chromosome by combining it with the noncanonical homology-directed DNA repair (HDR) based genome editing in fission yeast. We constructed convenient cloning vectors, which possessed a guide RNA (gRNA) expression module, or the humanized &lt;i&gt;Streptococcus pyogenes Cas9&lt;/i&gt; gene that is expressed under the control of an inducible promoter to avoid the needless expression, or both a gRNA and Cas9 gene. Using this system, we attempted the short-homology-mediated genome editing and found that the HDR pathway provides high-frequency genome editing at target loci without the need of a long donor DNA. Using short oligonucleotides, we successfully introduced point mutations into two target genes at high frequency. We also precisely integrated the sequences for epitope and GFP tagging using donor DNA possessing short homology into the target loci, which enabled us to obtain cells expressing N-terminally tagged fusion proteins. This system could expedite genome editing in fission yeast, and could be applicable to other organisms.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2019/04/01" />
<meta name="citation_mjid" content="ggg;9/4/1153" />
<meta name="citation_id" content="9/4/1153" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/9/4/1153" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/9/4/1153.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/9/4/1153.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/9/4/1153.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.118.200976" />
<meta name="citation_pmid" content="30755408" />
<meta name="citation_volume" content="9" />
<meta name="citation_issue" content="4" />
<meta name="citation_num_pages" content="11" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="1153" />
<meta name="citation_lastpage" content="1163" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Aki Hayashi" />
<meta name="citation_author_institution" content="Department of Bioscience, School of Science and Technology, Kwansei Gakuin University, Sanda, Hyogo, 669-1337, Japan" />
<meta name="citation_author_email" content="akihayashi@kwansei.ac.jp" />
<meta name="citation_author_email" content="katsunori@kwansei.ac.jp" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-1953-1664" />
<meta name="citation_author" content="Katsunori Tanaka" />
<meta name="citation_author_institution" content="Department of Bioscience, School of Science and Technology, Kwansei Gakuin University, Sanda, Hyogo, 669-1337, Japan" />
<meta name="citation_author_email" content="akihayashi@kwansei.ac.jp" />
<meta name="citation_author_email" content="katsunori@kwansei.ac.jp" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-9692-0968" />
<meta name="citation_reference" content="citation_journal_title=Yeast (Chichester, England);citation_journal_abbrev=Yeast;citation_author=J. Bähler;citation_author=J. Q. Wu;citation_author=M. S. Longtine;citation_author=N. G. Shah;citation_author=A. McKenzie;citation_title=Heterologous modules for efficient and versatile PCR-based gene targeting in Schizosaccharomyces pombe.;citation_pages=943-951;citation_volume=14;citation_year=1998;citation_issue=10;citation_pmid=9717240;citation_doi=10.1002/(SICI)1097-0061(199807)14:10&lt;943::AID-YEA292&gt;3.0.CO;2-Y" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. Decottignies;citation_title=Microhomology-mediated end joining in fission yeast is repressed by pku70 and relies on genes involved in homologous recombination.;citation_volume=176;citation_year=2007;citation_pmid=17483423;citation_doi=10.1534/genetics.107.071621" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=J. E. DiCarlo;citation_author=J. E. Norville;citation_author=P. Mali;citation_author=X. Rios;citation_author=J. Aach;citation_title=Genome engineering in Saccharomyces cerevisiae using CRISPR-Cas systems.;citation_volume=41;citation_year=2013;citation_pmid=23460208;citation_doi=10.1093/nar/gkt135" />
<meta name="citation_reference" content="citation_journal_title=BioEssays :  news and reviews in molecular, cellular and developmental biology;citation_journal_abbrev=Bioessays;citation_author=J. U. Engstrom;citation_author=T. Suzuki;citation_author=E. B. Kmiec;citation_title=Regulation of targeted gene repair by intrinsic cellular processes.;citation_pages=159-168;citation_volume=31;citation_year=2009;citation_issue=2;citation_pmid=19204988;citation_doi=10.1002/bies.200800119" />
<meta name="citation_reference" content="citation_journal_title=Yeast;citation_author=R. Fernandez;citation_author=J. Berro;citation_title=Use of a fluoride channel as a new selection marker for fission yeast plasmids and application to fast genome editing with CRISPR/Cas9.;citation_pages=549-557;citation_volume=33;citation_year=2016;citation_doi=10.1002/yea.3178" />
<meta name="citation_reference" content="citation_journal_title=Gene;citation_journal_abbrev=Gene;citation_author=S. L. Forsburg;citation_author=D. A. Sherman;citation_title=General purpose tagging vectors for fission yeast.;citation_pages=191-195;citation_volume=191;citation_year=1997;citation_issue=2;citation_pmid=9218719;citation_doi=10.1016/S0378-1119(97)00058-9" />
<meta name="citation_reference" content="citation_journal_title=Trends Biotechnol.;citation_author=T. Gaj;citation_author=C. A. Gersbach;citation_author=C. F. Barbas;citation_title=ZFN, TALEN and CRISPR/Cas based methods for genome engineering.;citation_pages=397-405;citation_volume=31;citation_year=2013;citation_pmid=23664777;citation_doi=10.1016/j.tibtech.2013.04.004" />
<meta name="citation_reference" content="citation_journal_title=Genes to Cells;citation_journal_abbrev=GENES CELLS;citation_author=A. Hayashi;citation_author=D.-Q. Ding;citation_author=C. Tsutsumi;citation_author=Y. Chikashige;citation_author=H. Masuda;citation_title=Localization of gene products using a chromosomally tagged GFP-fusion library in the fission yeast Schizosaccharomyces pombe.;citation_pages=217-225;citation_volume=14;citation_year=2009;citation_issue=2;citation_pmid=19170768;citation_doi=10.1111/j.1365-2443.2008.01264.x" />
<meta name="citation_reference" content="citation_journal_title=Nature Cell Biology;citation_journal_abbrev=Nature Cell Biology;citation_author=M. T. Hayashi;citation_author=T. S. Takahashi;citation_author=T. Nakagawa;citation_author=J. Nakayama;citation_author=H. Masukata;citation_title=The heterochromatin protein Swi6/HP1 activates replication origins at the pericentromeric region and silent mating-type locus.;citation_pages=357-362;citation_volume=11;citation_year=2009;citation_issue=3;citation_pmid=19182789;citation_doi=10.1038/ncb1845" />
<meta name="citation_reference" content="citation_journal_title=Developmental genetics;citation_journal_abbrev=Dev Genet;citation_author=M.S. Hayden;citation_author=B. Byers;citation_title=Minimal extent of homology required for completion of meiotic recombination in Saccharomyces cerevisiae.;citation_pages=498-514;citation_volume=13;citation_year=1992;citation_issue=6;citation_pmid=1304427;citation_doi=10.1002/dvg.1020130611" />
<meta name="citation_reference" content="citation_journal_title=Sci. Rep.;citation_author=Y. Hisano;citation_author=T. Sakuma;citation_author=S. Nakade;citation_author=R. Ohga;citation_author=S. Ohta;citation_title=Precise in-frame integration of exogenous DNA mediated by CRISPR/Cas9 system in zebrafish.;citation_pages=8841;citation_volume=5;citation_year=2015;citation_pmid=25740433;citation_doi=10.1038/srep08841" />
<meta name="citation_reference" content="citation_journal_title=Nat. Commun.;citation_author=J. Z. Jacobs;citation_author=K. M. Ciccaglione;citation_author=V. Tournier;citation_author=M. Zaratiegui;citation_title=Implementation of the CRISPR-Cas9 system in fission yeast.;citation_pages=5344;citation_volume=5;citation_year=2014;citation_pmid=25352017;citation_doi=10.1038/ncomms6344" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=S. A. Jacobs;citation_author=S. Khorasanizadeh;citation_title=Structure of HP1 chromodomain bound to a lysine 9-methylated Histone H3 tail.;citation_volume=295;citation_year=2002;citation_pmid=11859155;citation_doi=10.1126/science.1069473" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=C. Keller;citation_author=R. Adaixo;citation_author=R. Stunnenberg;citation_author=K. J. Woolcock;citation_author=S. Hiller;citation_title=HP1(Swi6) mediates the recognition and destruction of heterochromatic RNA transcripts.;citation_pages=215-227;citation_volume=47;citation_year=2012;citation_issue=2;citation_pmid=22683269;citation_doi=10.1016/j.molcel.2012.05.009" />
<meta name="citation_reference" content="citation_journal_title=Yeast (Chichester, England);citation_journal_abbrev=Yeast;citation_author=M. Knop;citation_author=K. Siegers;citation_author=G. Pereira;citation_author=W. Zachariae;citation_author=B. Winsor;citation_title=Epitope tagging of yeast genes using a PCR-based strategy: more tags and improved practical routines.;citation_pages=963-972;citation_volume=15;citation_year=1999;citation_issue=10B;citation_pmid=10407276;citation_doi=10.1002/(SICI)1097-0061(199907)15:10B&lt;963::AID-YEA399&gt;3.0.CO;2-W" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. Kondo;citation_author=R. Ueda;citation_title=Highly improved gene targeting by germline-specific Cas9 expression in Drosophila.;citation_volume=195;citation_year=2013;citation_pmid=24002648;citation_doi=10.1534/genetics.113.156737" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=B. R. Lemos;citation_author=A. C. Kaplan;citation_author=J. E. Bae;citation_author=A. E. Ferrazzoli;citation_author=J. Kuo;citation_title=CRISPR/Cas9 cleavages in budding yeast reveal templated insertions and strand-specific insertion/deletion profiles.;citation_volume=115;citation_year=2018;citation_pmid=29440496;citation_doi=10.1073/pnas.1716855115" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=V. Martin;citation_author=L.-L. Du;citation_author=S. Rozenzhak;citation_author=P. Russell;citation_title=Protection of telomeres by a conserved Stn1-Ten1 Complex.;citation_volume=104;citation_year=2007;citation_pmid=17715303;citation_doi=10.1073/pnas.0705497104" />
<meta name="citation_reference" content="citation_journal_title=Gene;citation_journal_abbrev=Gene;citation_author=K. Maundrell;citation_title=Thiamine-repressible expression vectors pREP and pRIP for fission yeast.;citation_pages=127-130;citation_volume=123;citation_year=1993;citation_issue=1;citation_pmid=8422996;citation_doi=10.1016/0378-1119(93)90551-D" />
<meta name="citation_reference" content="citation_journal_title=Trends in genetics :  TIG;citation_journal_abbrev=Trends Genet;citation_author=M. McVey;citation_author=S. E. Lee;citation_title=MMEJ repair of double-strand breaks (director's cut): deleted sequences and alternative endings.;citation_pages=529-538;citation_volume=24;citation_year=2008;citation_issue=11;citation_pmid=18809224;citation_doi=10.1016/j.tig.2008.08.007" />
<meta name="citation_reference" content="citation_journal_title=Methods in enzymology;citation_journal_abbrev=Methods Enzymol;citation_author=S. Moreno;citation_author=A. Klar;citation_author=P. Nurse;citation_title=Molecular genetic analysis of fission yeast Schizosaccharomyces pombe.;citation_pages=795-823;citation_volume=194;citation_year=1991;citation_pmid=2005825;citation_doi=10.1016/0076-6879(91)94059-L" />
<meta name="citation_reference" content="citation_journal_title=Nat. Commun.;citation_author=S. Nakade;citation_author=T. Tsubota;citation_author=Y. Sakane;citation_author=S. Kume;citation_author=N. Sakamoto;citation_title=Microhomology-mediated end-joining-dependent integration of donor DNA in cells and animals using TALENs and CRISPR/Cas9.;citation_pages=5560;citation_volume=5;citation_year=2014;citation_pmid=25410609;citation_doi=10.1038/ncomms6560" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=J. Nakayama;citation_author=J. C. Rice;citation_author=B. D. Strahl;citation_author=C. D. Allis;citation_author=S. I. S. Grewal;citation_title=Role of Histone H3 lysine 9 methylation in epigenetic control of heterochromatin assembly.;citation_volume=292;citation_year=2001;citation_pmid=11283354;citation_doi=10.1126/science.1060118" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=R. Paix;citation_author=A. Folkmann;citation_author=D. H. Goldman;citation_author=H. Kulaga;citation_author=M. J. Grzelak;citation_title=Precision genome editing using synthesis-dependent repair of Cas9-induced DNA breaks.;citation_volume=114;citation_year=2017;citation_pmid=29183983;citation_doi=10.1073/pnas.1711979114" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=F. Port;citation_author=H.-M. Chen;citation_author=T. Lee;citation_author=S. L. Bullock;citation_title=Optimized CRISPR/Cas tools for efficient germline and somatic genome engineering in Drosophila.;citation_volume=111;citation_year=2014;citation_pmid=25002478;citation_doi=10.1073/pnas.1405500111" />
<meta name="citation_reference" content="citation_journal_title=Nat. Protoc.;citation_author=F. A. Ran;citation_author=P. D. Pd. Hsu;citation_author=J. Wright;citation_author=V. Agarwala;citation_author=D. Scott;citation_title=Genome engineering using the CRISPR-Cas9 system.;citation_pages=2281-2308;citation_volume=8;citation_year=2013;citation_pmid=24157548;citation_doi=10.1038/nprot.2013.143" />
<meta name="citation_reference" content="citation_journal_title=Chromosoma;citation_author=L. Ranjha;citation_author=S. M. Howard;citation_author=P. Cejka;citation_title=Main steps in DNA double-strand break repair: an introduction to homologous recombination and related processes.;citation_pages=187-214;citation_volume=127;citation_year=2018;citation_doi=10.1007/s00412-017-0658-1" />
<meta name="citation_reference" content="citation_journal_title=Wellcome Open Res.;citation_author=M. Rodríguez-López;citation_author=C. Cotobal;citation_author=O. Fernández-Sánchez;citation_author=N. Borbarán Bravo;citation_author=R. Oktriani;citation_title=A CRISPR/Cas9-based method and primer design tool for seamless genome editing in fission yeast;citation_pages=19;citation_volume=1;citation_year=2017;citation_doi=10.12688/wellcomeopenres.10038.2" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=J. Rubnitz;citation_author=S. Subramani;citation_title=The minimum amount of homology required for homologous recombination in mammalian cells.;citation_pages=2253-2258;citation_volume=4;citation_year=1984;citation_issue=11;citation_pmid=6096689;citation_doi=10.1128/MCB.4.11.2253" />
<meta name="citation_reference" content="citation_journal_title=Methods in enzymology;citation_journal_abbrev=Methods Enzymol;citation_author=S. A. Sabatinos;citation_author=S. L. Forsburg;citation_title=Molecular genetics of Schizosaccharomyces pombe.;citation_pages=759-795;citation_volume=470;citation_year=2010;citation_pmid=20946835;citation_doi=10.1016/S0076-6879(10)70032-X" />
<meta name="citation_reference" content="citation_journal_title=Sci. Rep.;citation_author=A. Satomura;citation_author=R. Nishioka;citation_author=H. Mori;citation_author=K. Sato;citation_author=K. Kuroda;citation_title=Precise genome-wide base editing by the CRISPR Nickase system in yeast.;citation_pages=2095;citation_volume=7;citation_year=2017;citation_doi=10.1038/s41598-017-02013-7" />
<meta name="citation_reference" content="citation_journal_title=Trends Biochem. Sci.;citation_author=A. Sfeir;citation_author=L. S. Symington;citation_title=Microhomology-mediated end joining: a back-up survival mechanism or dedicated pathway?;citation_pages=701-714;citation_volume=40;citation_year=2015;citation_pmid=26439531;citation_doi=10.1016/j.tibs.2015.08.006" />
<meta name="citation_reference" content="citation_journal_title=Nature Cell Biology;citation_journal_abbrev=Nature Cell Biology;citation_author=K. Tanaka;citation_author=P. Russell;citation_title=Mrc1 channels the DNA replication arrest signal to checkpoint kinase Cds1.;citation_pages=966-972;citation_volume=3;citation_year=2001;citation_issue=11;citation_pmid=11715017;citation_doi=10.1038/ncb1101-966" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=P. Verma;citation_author=R. A. Greenberg;citation_title=Noncanonical views of homology-directed DNA repair;citation_pages=1138-1154;citation_volume=30;citation_year=2016;citation_issue=10;citation_pmid=27222516;citation_doi=10.1101/gad.280545.116" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=B. T. Wilhelm;citation_author=S. Marguerat;citation_author=S. Watt;citation_author=F. Schubert;citation_author=V. Wood;citation_title=Dynamic repertoire of a eukaryotic transcriptome surveyed at single-nucleotide resolution.;citation_pages=1239-1243;citation_volume=453;citation_year=2008;citation_issue=7199;citation_pmid=18488015;citation_doi=10.1038/nature07002" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=Y. Xu;citation_author=M. Davenport;citation_author=T. J. Kelly;citation_title=Two-stage mechanism for activation of the DNA replication checkpoint kinase Cds1 in fission yeast;citation_pages=990-1003;citation_volume=20;citation_year=2006;citation_issue=8;citation_pmid=16618806;citation_doi=10.1101/gad.1406706" />
<meta name="citation_reference" content="citation_journal_title=Genes Genomes Genetics;citation_author=X.-R. Zhang;citation_author=J.-B. He;citation_author=Y.-Z. Wang;citation_author=L.-L. Du;citation_title=A cloning-free method for CRISPR/Cas9-mediated genome editing in fission yeast.;citation_pages=2067-2077;citation_volume=8;citation_year=2018;citation_doi=10.1534/g3.118.200164" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=A. Zhao;citation_author=A. Guo;citation_author=Z. Liu;citation_author=L. Pape;citation_title=Molecular Cloning and Analysis of Schizosaccharomyces pombe Reb1p : Sequence-Specific Recognition of Two Sites in the Far Upstream RDNA Intergenic Spacer.;citation_volume=25;citation_year=1997;citation_pmid=9016645;citation_doi=10.1093/nar/25.4.904" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Short-Homology-Mediated CRISPR/Cas9-Based Method for Genome Editing in Fission Yeast" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4.cover-source.jpg" />
<meta name="twitter:description" content="The CRISPR/Cas9 system enables the editing of genomes of numerous organisms through the induction of the double-strand breaks (DSB) at specific chromosomal targets. We improved the CRISPR/Cas9 system to ease the direct introduction of a point mutation or a tagging sequence into the chromosome by combining it with the noncanonical homology-directed DNA repair (HDR) based genome editing in fission yeast. We constructed convenient cloning vectors, which possessed a guide RNA (gRNA) expression module, or the humanized Streptococcus pyogenes Cas9 gene that is expressed under the control of an inducible promoter to avoid the needless expression, or both a gRNA and Cas9 gene. Using this system, we attempted the short-homology-mediated genome editing and found that the HDR pathway provides high-frequency genome editing at target loci without the need of a long donor DNA. Using short oligonucleotides, we successfully introduced point mutations into two target genes at high frequency. We also precisely integrated the sequences for epitope and GFP tagging using donor DNA possessing short homology into the target loci, which enabled us to obtain cells expressing N-terminally tagged fusion proteins. This system could expedite genome editing in fission yeast, and could be applicable to other organisms." />
<meta name="og-title" property="og:title" content="Short-Homology-Mediated CRISPR/Cas9-Based Method for Genome Editing in Fission Yeast" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/9/4/1153" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="The CRISPR/Cas9 system enables the editing of genomes of numerous organisms through the induction of the double-strand breaks (DSB) at specific chromosomal targets. We improved the CRISPR/Cas9 system to ease the direct introduction of a point mutation or a tagging sequence into the chromosome by combining it with the noncanonical homology-directed DNA repair (HDR) based genome editing in fission yeast. We constructed convenient cloning vectors, which possessed a guide RNA (gRNA) expression module, or the humanized Streptococcus pyogenes Cas9 gene that is expressed under the control of an inducible promoter to avoid the needless expression, or both a gRNA and Cas9 gene. Using this system, we attempted the short-homology-mediated genome editing and found that the HDR pathway provides high-frequency genome editing at target loci without the need of a long donor DNA. Using short oligonucleotides, we successfully introduced point mutations into two target genes at high frequency. We also precisely integrated the sequences for epitope and GFP tagging using donor DNA possessing short homology into the target loci, which enabled us to obtain cells expressing N-terminally tagged fusion proteins. This system could expedite genome editing in fission yeast, and could be applicable to other organisms." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/4.cover-source.jpg" />
    <title>Short-Homology-Mediated CRISPR/Cas9-Based Method for Genome Editing in Fission Yeast | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"2IzSSWKYD0v06Y_4quWLMWU6ZYmgRHUtUDYwxQRtC30","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"492598","apath":"\/ggg\/9\/4\/1153.atom","pisa":"ggg;9\/4\/1153","ac":{"\/ggg\/9\/4\/1153.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/9\/4\/1153.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;9\/4\/1153"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;9\/4\/1153"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/9\/4\/1153":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022Eu3pKdqUmeiaLuDrOlxEmc1rEJ98lvxc0iaT1HDIxMza6ROJ2yy2BeBuW0ju2EnEE2bGU\\\\\\\/ACxahUeIp4vBLSn8OxcM4NfR8oI7ihR1Nd5q4FIpkgykvi7mIj8ABieab\\\\\\\/tpVrL7sUh6EUEpI3XnEgYdY3WxhgByfMucL18so9M\\\\\\\/jz3RRB7n5+SffhTMHMkDCX4fe4nyZdV6FlRXVrAcSVCSVkhXGwO7GG\\\\\\\/r1pPUl3XXjUStAk2VNMAy6YJ\\\\\\\/adi6v5OQOKDaIsYLQcoj3726kksJ5cDNHAfbuIG8KZ2PeDoMvZPHYmj+kvBfh0IMv4V9URkRRFCKbRMuff2uX86QFiV3BoPukRONrFgGV6OD7mcqueiTIV6HdFOOtpnTIZyYE1kg+rbooLTGI06fQyNUmD3HuU9Hq5b4GHhiICLZdhia5lVKl7ptkx0TuruNYDyqEejk7CceilxTvFY0Io\\\\\\\/a0pKfb5i7U7zG8Zq0zjOPjc4ZLEtbKbAfnkqG1iFKqhzQwU3\\\\\\\/xGaDL0yvXAb6H9HGoKCNm0XiPMscHXRwQxJe2Dv\\\\\\\/n7q8ckkqEIaOD3rS7KPIDwNtASYo8LSNzuONCfMe6VuQDvQ2VvSTMT0UF34wJS0Abdm60lNS\\\\\\\/Fp7sO3y6sMrS9ZmF4IeehaSrjUiY8YM960cyAja+wgafiWNJFZCWpElacdoeTA2QjqaPmOT0yzKx7gYr7XEo1bfBLqW7kFCymUextL482JliDSB7a7RTXyt6hEA6F6FhUtkWq2Wv2eYWVBqbztGHHIxktwd2idq2S+XLPkh6\\\\\\\/tl2hPKtI6b0bc6nufY4\\\\\\\/D5x9zS9Msx6E2tMOVijB2B2kWH1bnqXv8Mjz+nt+9vJpG8\\\\\\\/cm2TZf\\\\\\\/ZA7LRP6octS+Az267PUJoXbyAMcisj8psR6\\\\\\\/QjOL7pSkDAhnpbEJkTTXo8f7fq73rLAw2FbkwzSV\\\\\\\/hFisqZJUgyIaHdst7Ajb6+b0T9mJxgvcOvp4JaTRxBQ8v6sqDchO6kOfKjwSRVW9F6FjTJ4Cae6fpzVFWvmWaks79aHWIXVOSoog0BfvsbsjB7A8ydjXqO4TMjdgqbprwVqjuY51R3m+Lb060is2Zn9R3tBXMuVplGDKQrvESvLHWXTWgnmJtjsRWFisYDFewXcGvT5u+AsmSUPnq30k5x70FVD48zMIdM8WDhi3UFnYvS+rCYqdIc4RmMCyoRZAsw73\\\\\\\/rkkI1yok2Y78151uVUUHIq7mlPIGLa92aV5tiS18mcJHn+QtdlUCNS7WjYDj26AUz7cJ7qZK\\\u0022,\\\u0022iv\\\u0022:\\\u0022SzNSgPvvpqIj8+9hpFmC5A==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002278ea242b24f341a6f67dc237ad7e21349765ac6e7a12be35a2a01229c42322d4\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022QZeFHRBjL2J8dQNcR7XZzI4u22gdenh8mWbHwnWdqr0LTS+j1+xACcNSyZ\\\\\\\/2U0hCs3FRsyh\\\\\\\/Ylc5WpHTcNlnTq1gbodAlUWCd4U088V5JBoNlGhPD9OrB+Scp+GpieWjxhQmLtFS3\\\\\\\/7M2EbU2bLdmHlJYYm4oPZC7j1RAN4HuPmhlWvu2g1vWSwGNhv7bsCCbQp3hF9lI2GLugHCeEYbLO7aEEwlj3SWZhCg7HEIsSaRTAF8G8AChCE9syzEJdbl4jp5RP0Hj20YTYRTv3JiMqEtySuwFWlDq6p3RNC8usZe2khVQyGV3rABUAs+e7yyorFcS\\\\\\\/BED4ICoI\\\\\\\/dH1ma2+8MnzaD2mtx4ZasfHaW8PSHaXpijehMKEJz3NxfZ16AJdeyLUt33cHgJbqs\\\\\\\/tJwr6x4+q5\\\\\\\/yASUVuAWDXb7sTYMZgVWdIHa0oblVKjR3X+Ogcy\\\\\\\/ShceS+X+gENAPrLAl9R1W79oa+C6oKHdHFnRrwpiPghIpKq4Roty21FgyQry9M1HxY\\\\\\\/KWiZUcm+mS4ayJz6T7EpjfGMNdxg3XpgKOgEzd6OsVRQydmVJLdHBVmEr9N\\\\\\\/td4IIu\\\\\\\/KBr3KY+6VyzZvcIvr893ad0t5iLsVXOcXRSmqU\\\\\\\/AElxBxbkNycB2qfexHMYrlsze1IadHQAYdvznAMGk1pyipo7rgWWrFOVco0B2Soxm3ggUonl4aadI03v7b4mI1sXne\\\\\\\/qAIwiAyxgudIi1fcIQKkjPlo9yde7mevv\\\\\\\/00Xzw1PChF4hBRgWqU1ia67zmrn7r+LyBqYeWJbODu79CMuw0FT9t4TltxUnpgoIEMyuwGu9\\\\\\\/3Xggdy9Aj5NoxV29HIH0FyYatv8fXlfQcJXKUK6HcdW9Hf86qGPTHzKfII+T1IhrrUMhXMDWLvIzTA+lTsuny9B6zgVdzdkBW9gXtT7wqbqoUkpNmPNNXOFZ7ZXwoBYBmuksSKm1\\\\\\\/3OBn2vU4mhkxpvrf\\\\\\\/TT1KBh8Nru\\\\\\\/x51Mt2n2o92i6pnGsjaIJgFB29gjGLDkexJWdO9HBLxlR1WCsmYrCM0BKOfcp0jfpEf+KRzVB02bwL+anFyh6mlOYsJ9eifbovhHpzmeYewYGhZcryFozzcQH8xrW\\\\\\\/t9OoMUyT79pi8368wpHScRNZcLFWMxX1ZGZhKgIfcjOisNasUy23EXaCAtJkEK\\\\\\\/RDXB0BMMUmpXoW2MAVKgfI9gLj\\\\\\\/HlM5x9B7mMp4mH7gQUnYge1\\\\\\\/jTf6XGi8+LH78LyadpVjOc5sqxBNhXj4JmMsjm+2\\\\\\\/CXfG8Hcgnj4\\\\\\\/fzze24xk8uKxYQbQJ\\\\\\\/pWDDHXuwA9L1UL47SSuS9fO462JPlXRL92avbbzHAQD3Yv8orG1awS2hU7O+TpiyjAJ+oUUBk8uMNl47THMTWbeDkvCSD6peMzdFwhTwgcBFC3MLshc9Oa70d7Jq2hRyDCQpvN6IRLsUFlSGaQ0pqvnBa86hV4kriSq7r5GHt1fTcfgDrz1p68I2putS04Gfr9mxyL1N3zsBqto9+ohMGkuHo5bLLjw7QPX50D1g0Ro1X7zr\\\\\\\/Ia3j\\\\\\\/VhNXuLsiJz7VtYMiGAtV0PAKtuN5whLzJH8oILaUJI4mwLvoimBph4qGdxe9D3l6qR5xTBwkBN8HfBKG\\\\\\\/s7F\\\\\\\/yYus6rAyBZRbvZY+mcfkNxP0xBVJEgDuvo4NqMR9pwuJtdbzaptU0y0dTW+GE6zC3mrSmphVSCo+91b1MNMlqYuPWg\\\\\\\/fR\\\\\\\/brS4CVoEybYQWtxbOk1SQwxfgli5KV0OJ50Xki1HLceoku\\\\\\\/UkfdDryx3NJeLYqWc32jQfvLXmAR\\\\\\\/ULTmgp51vLXvRxc6Prf4cZZnhVC\\\\\\\/GrBkx8rI8zocsXnfuXCSrZPKFB1PPW+dk1JIEmIGPwynwgc4h61UClvO9ifyQ5Sa+Nqqe\\\\\\\/P5prIWNglXdTIaUXWxapoMlQ+LMBYWgv+A72QVloYGlnujINLN0rRWqrWefN5aPKncKky\\\\\\\/4NhN16DDNdElVeT2cxS2uWwLk3U0LbDoickavmMCZ0L0KQs4bnjPrU98fN0kmYgpeEaZHTdEj2FvH4mwtfe1FdWhcniKTOOMlmbG2snT00+teU2k1NolZwqaf\\\\\\\/kdd5rPC8ks7iu+TVX7BScpExmHGijUuyN6NqVek86cK8m6wI\\\\\\\/PeI2eGd4JEQbiwbjBx6GXwvGzekZKC3fx9sIpANHXh+UHKAwmcWiA49TtmpbDfw4EG4q3mu5U39hZBPoL2zr7U32tlcXj9fCTI\\\\\\\/PhJUBEEp\\\\\\\/257wFirhaRfjc08mQ\\\\\\\/No26EUsUGuWK47qQU1x\\\\\\\/gbCSK+tGf97DKDc7Hv6IxVr1kIbQSvJdLYQSVPlxiy4nar20M8+O00Dj507XSuruy89CXc88W5022fd+AC6ugchnd6byxl3CCj95p7XQJDeeKogSF4tRCdPH0VbebyT8uVsWeGXF8CDjL1QPcIStMmbwJA9J524ieTipg1YbSZfDsZJT76Wi8YX4kR8XgqAI+MQutibcfves2i5vjLmlww5KdfwfIULKTHTzGVJrwcPqjN9OS0xCtGGEx2qXuoluUuz2F8hXAXEoX+EHVlrymR+I21ic3yUWm3wJUfNu\\\\\\\/aJG4hW3gC\\\\\\\/K9OCnlZgk1ZBMNynX8rV8Il2wACySsrhZi\\\\\\\/pl8AMCJJlzviLky5\\\\\\\/qRdLV3HBzdqd4PkF9rkNJVc79YXaqLdCl7w9U0SR1+FjFENEpZMiUQK7gOyd+yiCDy6Jx9m5roazQeJWQDPOngBtUoIdCRdsjx1DVAHj5Hg1iZcpGcsPRV\\\\\\\/p\\\\\\\/g0Eu8aW+AfkocUr+6Qdnv35LYz28KfHe6p\\\\\\\/gFgNuxXPFEYl\\\\\\\/haarhTxXXUpucJOICM7x+MwbL1wqN+MLZ\\\\\\\/PlS94Q9qheapAMrE\\\\\\\/EgTsqf2KibuJx50XIktLDBcSr8MAFwpqodoOfK8VKSTHkMEZjuiD8h4E8u7iKZTlx1aqrRQT4Ig31Q3PmA1zILTDkuivD\\\\\\\/8e8cfXNlum7PL2ZEsrkbDktM4K1BR913MJsKTOUdmmNd\\\\\\\/yUyQjM53mjW5qiwVEOfu3UDU4xr9xwNDk8sRYKFxEPC0lbdmH+RjdOzHavePiENxbjElHlNQ47YAXcUsyjddFhlB9UdhEQbydtnfSnRkxvIRxkFd5lGfhs845KvgwBb6luCXQfKXcXkLlVHFMe0txIAtGhkLOmioms8GZF5KBITI5zAaWCJrzWkxebqpKz+PKQn916bMhJ5vsnnzKD1suVLV71KmltB3maWjFoQtYnxa9U1UQqi97FitRlNq8XXXV910du1NmdKVBgMqTHNfgcB6harWK1Qf60w3sNtnIT7YNTKLVdCbaUH12nLxOwg5YzRSUzTHd0lrMKaP2qFzWbR9\\\\\\\/\\\\\\\/NiiEVOHZvPHRFTkDTb23jIfP0E6YapjZrutwKI4JvMIXr5pvqj03CmtwUEUsdyCde\\\\\\\/28ES2EF8+YL6cKTbQpASuja6C4MCPWeg0+L6eyag03k9umwdEEXHMB0eoVs63KBD+p2iEO2Wx6prYYL9B+G14asdxPmUJTHm3OsnkJj7apMd3\\\\\\\/I2cOezbHgtBXallv9n9rCuDoGSe+QVbT+7L2z3Z2Wc2kr6HxRGO7vfbXf5aDK3JcPFSpuACrDyKOIkFMCSHJpvJ90\\\\\\\/hg+Wo8vqMwtKWn0mltB0xEMPeQplL6Ni2Cj\\\\\\\/LA4HrIzKP1jmoefbtNzEadanVBX9A8I7rjI4SX4Zj7WYeDeWqAKbsDIzc29oCzeBvgP4rgQgkFzLvb8e0vTuIxxSiv8UDIwLak5fHu3FwhbLirWRUxyTz8OjjJPJ4hPvqfcIjH10mY1BoM3KtWEmbaoN28\\\\\\\/pTGJdgvrL2qA7fYOj8\\\\\\\/YtCWdjlIJvD0sWQ+78MKWzArEkNe3ClrDMm5f9imZ4PRUiT1nPQIuKOqorxaZfXDME43uPuyPaXgRCYMknkoRYCdztbb031nnloVkKSaUoTXoC9cxP2qo89I9yITwoYNwFp0RokDVe8F6OJ6DXjZ++9h7TyK0axNQQQ09yA9QmQSL3pjGRCJwZWzKt8TnARsokG0aEsZK3hS+Hkp7921mAgdYqqmAs5M5vkDbA\\\\\\\/BFsb9aZrQcBfJvDYZ7OzKMFQrqpbnXZQUaeH8GGYeA792SVel870u4ONxMa9EqpV+2mlslOCAEDs42l3zkBq669wxCpNTKa85q6QS30yexSdK8wfFU78JEMdcnmMp1ZSmgEuVtAOCojHuejAgZ7Vs1gPWU0D3qFxtG62bE4cSJMr1\\\\\\\/1rxHui822MPaqmw4UiHMiIdytKTfkoB3nOYP0XmdDMi0D+KNN8MSMx1eZY7d68wqOvGO+UIYj27cm2xXPL9ynDnRPRhXBSpMUPFOo0Qic2ARb6jz3dDkrkhCh1r+blB4aIJl\\\\\\\/ncD2tpe945cD2y6Yc6+4l9vPbl3qgwOBjLcbqsbgmhJgClMpgBu02td7\\\\\\\/G3Yl9kyxcfOA6oHSemFhqz5B9K06xNjZiJ5JpDy3b9BFd5M8+tG\\\\\\\/XN2DoY94NpOuDwvP9qqLwbRfhud0cqDFRyRDdPYiu3OJp90gviY+FvQeGND\\\\\\\/EKb9KNSzRThUypacqH2\\\\\\\/SkRdlLiSWdjR2UzLN6jV8ZaSY2VBqLYt6c3JrCXczMk5CQmQLiUtDcmT8mIcxb\\\\\\\/670f6+M+QV5Hq0I7ZP3MjvLApzs9EAKE6beE4JiB9NkabwhNthNcLdvBa8NkNwWddGTfpuPEloFtfk9L6+03+V0TNIbdKsr\\\\\\\/KA96qvuehkH3S9Rwoo7I98QEoDt3VKajdPl54dGhdFHwg0ffO1WFL+nHw2qCETAzVQqOtak72I9F8piSLdyUVNilSN\\\\\\\/\\\\\\\/PRvbWnXgo2ACAAhJN3hc2eWT\\\\\\\/EQHz0+5xOygHBr1r2SOVnJ\\\\\\\/oRTo2PqGqMIS4\\\\\\\/RKGw83coqPOUqzvnRc07lUnsXzRgSjqxHLetXTcJU++7TZldMFXOcN2bMZLb2Tg\\\\\\\/FQSvMpPHAtR2fA09egcsBC76OtahjK6uFowmGZiFxZTGJkRluXq5yvw8wtf2OF8ccRzcvWF2zzrJmGPBc2DJV3\\\\\\\/klAx4gl5H36LpHFWF9QcFjAEpg02rHstGi18kF7N25hbidLepzqGEZ4nworHFtvRrjORiBRNYVaBcEJdudhl1\\\\\\\/xYciKTpfwqIs8IzE6qTmSaEg0XwRArPR5DHX\\\\\\\/Dz7vdKQ+TsPk0sJkXsZQsCmzfKTD0rYQOmmCBdmMHVGqafrODbdn1zXP8pbJBLpEupNvy8uM6+4qSauDr+udTfN2uOdxVNGdCZYABSNJGk66fsb\\\\\\\/6ygmrUz7wfWScMRsRCQKCvwEKC2DtwMJRhcQxoQWA+4ub3v+ntqJwM\\\\\\\/3wDgP5VwourL8pONpPn\\\\\\\/YdIbM8tK4UoTBIA+OZ48qNO0BaJ+xebnHh1pQ455NJLtFphDPdvofq38NDFCEca\\\\\\\/wl88CUtjP\\\\\\\/RtBt5ookCEGWG6H1dcz8I+9b7F\\\\\\\/ZbOJwya5Wuk3vGlJsWEC11sB2Qndb8tsiiryTvdJn1vTDDEWEmSOtryzwSl1nyG4R7mCNrvz19Q\\\\\\\/zeRM8v8UaDZqRv\\\\\\\/6g9ZbSnbc5emzL5s1W5hBylePIzsaRdU1\\\\\\\/2dLGquSdJU4G8\\\\\\\/TcxH9ZmehcNjtaFBiyBMRX\\\\\\\/mZJzZGvFmckHYohusKL+h5PwmCU0iwUqyNxG0X1448v62a\\\\\\\/YYGI7\\\\\\\/QalVBwza4UM8pvBwuMqKcp8SN3jfnxx09uF20tuiQtgmrUSggujhTz2BwPviQgB0sD0zieMNu7Bqg2KcFKJKBJ0s4eWfXGWonbExphU\\\\\\\/Z0lL56RREwr1zh8XUdJJyNGKaTG6vBbFG+3EXNMp+k7EFpW8NEcSYJiB2H77ezaAWSzNfdD0oPD9BBokx7Fvi6+rnKqTez+7eFrpVOtjOdzyQjw\\\\\\\/\\\\\\\/IG5CCbmNmhHCeB7eTSG8noSV\\\\\\\/7LH5G3+K2CJeP9E9Qv1LOyT0U+maF0UFQJWkGIxHYWQ2R38CMTrNeHPUMDhhun6cbIHfTIc04pnKoEQiPZtM1qYrEjURE4bkcLOosmJ1EoYwxs3o0DwYvIKLBKm6nOUblWiYnOAb+itrfuAp9z7qWRcRU8h25pdCKWw6pNV\\\\\\\/EqDXejdGZGQZkk6rAV+M9G\\\\\\\/TrGfUo3DXDf+F+eRwwqlXhgAgXJDZUNHLg7AN0MH2UCN2OXpXqn94ar7PlH2jMZGx6TBOjqFMKHiTUTIWriZp3FWiyKh5vBSDZh\\\\\\\/pJsWg6fdAgMi1fOpCZAEPUuTAUxAY+Z8kfpfUwvCK29M3JSKOK9OgkqM4h8\\\\\\\/GFnbxTlEPaCnbDweIp46snoqXKW8ILdRzQkayd2KeaqRKKMhAd+6ZuGIX3LlJnxWTb4ZHQNJmEdX9zaxUbChrphvdFnA7a3acR95P+euSIFEAQxRArU54ZlyCBJ3m\\\\\\\/itdN6UZFiMJbCHLgQoMPjdavllZu85iTXacKJJcXyhHCjRUMrvZjii26bPZDjhPzmt1R6a5vzCf7MyopRkLCXOr2NuuRWVMoyfl6jtxp+zp1Dh3f4doUe1mmSGzSEpUBLUUnQEupi4LijQKWycaCss+8vgoEg93jp23hWJmQKKqDx7u8P\\\\\\\/22qxySWxeMm76bueURJFrusaeSPD4GfS57QhlkOiwEe04FVMlPA97VOFbbqmvOd+hCJbPqVbwyIcHNzCvZjKGhKZI8I4vFzJ7YQDXOZIJmTsw4PlqskePx8SPGbQ5W14+WfrZmlq+ElSzSU2DetUrOX8YPPqS3Hn5EKlGJnOtaNF6f+cjaRkZj2c1S8xlbq1f9rnjjATB3zUrwQgU89vXrZOZyFjXUZgBsoQNt9VL\\\\\\\/b+6EPkjO8cozzt\\\\\\\/t9kOBDv2whaMeNGsJwb\\\\\\\/3C09hw5hoXQlS8RrqmS7ItfCEZ8Fs2d20OMxNjB+UcCzeJXQKbjYAaHjclNseM48dPr1SWv5UgP9w2chp85OiJ+emsPsC0uQUCvaoLTx55s6MuM8Q+WQaBOPORnuY8aQW9C2hdwnLovCoYG3WtIQaFqPnw5Cwe0u\\\\\\\/GQ1oZbw3yiQpMk1LVhIy\\\\\\\/fDAiE2tRWVqiLSz\\\\\\\/oOXANpW7J0b3+0zjvhJgU9sKriqQvGV+xJuC620+Gh1VbwqCJ\\\\\\\/A14pDXJ9087Lb4PXQyeJXywnxT93mzuws9OR6YFSiqMMwQkGfSlEpdBl3RvzRWElXcKBvZfvcZkyThoxDycJ+LNdpd12bIeUcWsjZ27hJ\\\\\\\/AN8x86h\\\\\\\/+bOWCV\\\\\\\/iQFIdyueu1g\\\\\\\/cxO99bDMcLGPm9DRlf\\\\\\\/E9sCJxXc5e8z868IkPH5Z5g0y0dq3\\\\\\\/NeVX+WweXv7zXZgtbhihPsDDIwiP3ckcASAneZMi9bJDjIxpYN0Pfyay7h8AppcpGqLjHdr9IJajU7rjq7jJfFtsK0EkfM2aYvePoreFnWeJ33yzgZx4QJCYGkNBVBKSGnJTySToUBnGz0e\\\\\\\/a+fRKPWfrfYjcikXwbR5OFGq1PaIRZSG4FNj0LUwbqeeqpCEu0QcYgMbspLrn3oAKYCp6w2rDcft7K3to4O5IX1+MXs5ad0apII1ToJv5o1kaJuzP1KabGCkWRkx9bO3hwGqvcQwGVyJzM2kc\\\\\\\/qDALNvwe3E0J8+t8zeFkjC88KCDDyMt1cKjAdc\\\\\\\/eO6XvgIjRdUIdPibn5tDRJEGAhTRc77IHCcI\\\\\\\/\\\\\\\/WCF26157nfKWXLD7mRKT3Rzh4EBxV7yYkFxd1GRYXm7bWAyX+qIIME5O+39BJrMg+GSTOfLVkqBDJYq9w7rTZoMQzpJ\\\\\\\/eY1d\\\\\\\/fC9JvenOhlHZZDSd7qSfnPYYy7f7Dprtk\\\\\\\/+v6UAT7XjLgDrQ6KvNQxigz7M01JGcKSeIBShpDXLc9nXfkGtr3xt7G7UP9j958pR6usgKCDHfbHYRsRTt1SuRs2NnyFkvEB5QqbSIbveCYr1\\\\\\\/wXxwCVjCrB6WxkT6keCkwtfUaxY2Auih9qTkX42iCK7X3gPHIEJ4JYXhQzjC9y+5mV1e3+g1IO1YNu+JT43PdqPYqhcmLhgYivkl3eerF6w7XItY7jK+9GLiaq6iVgORo0lajNVtNbbPVZ4HDrbIzSX0XCQ02FDNBPVqMS7e2l7vk1C7bFHTOlHgVeBx2qX7WTwrAJZeiYKw1xARG0rL2kVrZOPvlSEGo\\\\\\\/2z\\\\\\\/s9gqKwU+X1tw\\\\\\\/kNGbKpdpFEJCbkcstlPTNpWcRB542WWnWG+4KKt0Uc7WlgH6xmOom40uaxtii25\\\\\\\/q+YgZVIqskRI5Gn7JLUEWbPGskxsh7gdX53V2DE0\\\\\\\/NSHkm8yKZUQDrn4Xupc+g9exbr\\\\\\\/oDLmpZHTsYSaa0FHAl\\\\\\\/2fkofEvUfEq53UYmltMgClOgW2JuBBXLv06uCqJ3DgNCCG8VnJfnPgAOTT\\\\\\\/cxS0owV\\\\\\\/Q0R\\\\\\\/oI75w8cWvjSUJ8sD0+VwYYAK7pcRbQyoY4OKYlPyc0Brv5nbFdCme6jbcsLC8SFEXw4965UI9HgwyGiw\\\\\\\/hDlkoSKX3DHlrjVhA\\\\\\\/vOfqAElCYZJDyUOGmAe2tWUwv010NovVx40Jydw+pvKH4TlIGTnFPah\\\\\\\/zbgdRbOy9sg1H+hCkgNwTNrFDX7AS4g0LnwG00E+OEYWrU6nmuluhV88W2FVfm7Z\\\\\\\/W6zZnXEJF1GVXi5+V\\\\\\\/n6JGuQ36xgwplToBu3FCxbNmTMwU0zxqu9GF2o\\\\\\\/xaHPQINwmtg7YD5bslCIxHzr6UFcM4hbprK0d4xxI7AySJ\\\\\\\/X9rChvJOUL7F0lk6nlH9dk2PUvM\\\\\\\/TEZSCE4H67R8+8ZEGwEAIq9jeOyiwmo196w6RRiZZ6sw3CkAsFrLINn2a1iyctA12\\\\\\\/4ZdxlI5uCbUXBGK73Z4nx0\\\\\\\/Y3mBStz4garUHiC7u+ZKS9UD4bgb5L81jGTfDyFMkJ9Sl\\\\\\\/G2ZIABoy8foZAOVG5OZSzqiZffPEoSa7ZVIQQScgDcaaBoSAXMwSQqVoEvzXtqEDjEI622rPGLUaD6ttm7u3AM9vcvVqvh3X6s6Q5zl9pYosH3tN5DzwpUWjKMnjl0v4yrDTUTZU8sdfdzW9bLjqD7RefZJJkFng+MaukvVJBAb2fIOnZuFXFM6sUlV6jpfFNj2c\\\\\\\/33KqgLLsace65H9Qu4y5HsVDE1Cv87RbPoOmHQnaQpKtUbFoxVcrvDSNBSYZJvWidvwquyGQRr5MjP1jsKYYmoA1If73iLynsnd3z30aRM9lUe3Low1sfU5t2cE84zW50ghN6yrjR1HHEl+JWV9HnhwFulpJV2eHb7\\\\\\\/rXr97B96BUmTNj9cG\\\\\\\/ApUAhcI9nKwieD7CBhgesgudKNwQ\\\\\\\/dRBm4WuNPtgrPsPucchxqcajjHSEJIOkk4mnzvSlnytzTHWJ3ttl0OzAW88Odb8kofiIfPrGUEzg+imnoXX8BYgEC1sA2581dHqiiHXvXdCHlk\\\\\\\/a9hjYqWu+vdrWQQpXYeKarA+Ip2Zhh38CMoUiXW9yGonAWPH9jxXTEMWKibVMgieQ3xtoeKC2mMxPL0DG6y1tnQrZwQWkazVxj3\\\\\\\/nnMZH64uzkpPo3Uy9\\\\\\\/luTFrOC68VKH6ho0QPZ45+fb5iwB3DTsCwKeC\\\\\\\/S7DvojA4r370AXfnMXD5Izheuwr+768iGbSbdZUok\\\\\\\/9ALvEmR04kv7p6fhN0HAAIE\\\\\\\/BLL1mmTytwO4nSO8LhJyQvYufb7c4pkecdq48h1hEC1W2cg5tJGItrn\\\\\\\/z5inMW2v2wlPm6HkiqJYH9QiZ\\\\\\\/SPyGKACE8SmojMWPocsRwS1nwjmGj4IHFiUJpc3w\\\\\\\/U3ARt8OiBZJD4b0JEW838ttt971xfZxC4eU+yWh0Zr4KX3bsxMWCaWaW+8wQpDqewoCUXzgM9GTX5Y7L5oM2b0I5xuNamdNVfeN9gI9SaUExg+DOrD9nyctp5w+Yzm1iFA4z3mWNA\\\\\\\/SiiLh9FcbTmObW077PdqyFhf+9AmgVMz+WCppWuP7eg8X3TgTNemkV\\\\\\\/llFqrMjUZlpb4XEMSKm2h1H8+N0tL+z7czWEVF0tnNdeAO0zUfpuP7pYvbKYvqm\\\\\\\/ymE\\\\\\\/h\\\\\\\/bV5JiC+wERq9sB6VvK1I4fk9smX\\\\\\\/Rt\\\\\\\/PiRabo3ayJIYY4XQnhmJsl+df8pNj1XwQ7RgdSHAIa6+zoYOiTc\\\\\\\/lHa2bN+TI4\\\\\\\/eOrblpKlMh10ECVpEq97K7TyKWWBTIUCm4hkhXOUWO3K6HW+Aby6weTMex1ZtgbaBkiWzODZ0DaTw+bRx9u0w9ytYYy5WX3PgxqTks\\\\\\\/LE9A1+mDYqfKSXjm\\\\\\\/LlLy0ryQYIaIf4Bc5OB\\\\\\\/vdCCRwF9myqdgENoekF2vCWTTYUKTxRXfPNqdTrVpCYgL+acCY7ScLU1F1UMehN8TrDweA0w+gXEQDA8SK4yr8LcTC9YPnKcTFQfRqp5olYi4WG5y5KOPDBWuvErxYOEv3pxpeCJ4cR6gj9f1sRETDonXgTfi+v+4FQt7UtMNWogRYnlXYT+d01x7+nT32DNaczjZtuUkhoWGz9WGJd2awjhPHLemGmDmnainL6bardkZwEd5a\\\\\\\/ISIm+PCz1zO+BJwT9ioC\\\\\\\/9ZepUP4LCYd7cR+h80FLEDN2Kf8TOMcRMWZJ+Yjjpt9zUn7J+UtJElIZc5okPoYZAikymZfyHg+wwbzkfZWNiayRDPbkmmZ6FKkZ7vW\\\\\\\/7k5g+SIDjyPdqqexS1sfPFZQRlE0R97jd53+WlL20hSlM9EeTAlJ2U2jVmZg\\\\\\\/3GL7n++G1iFLzsVVQlT74dtf7cWl16P277j8YxDo7zHHrP7HredfQG0jlETHNtEiSpfKT+Dl7dbPj70E0ONWezVbiZkcssH+OK\\\\\\\/1\\\\\\\/nzRKheLRhJxCT2wj+Op+MMe6ifSt3kzlEeWgdYZtXIH+AgpxkfOKeyd9tvtohHFrSzroFzQ0xYvb8FDSqM8xdBAssn+iD\\\\\\\/2HAL7r5cVPSKjtK5gq4qgi17D16GfrVPzJOcFkF9ZwUGZCvSDEChy56lzatDi7bEbdR5J4\\\\\\\/mUbh6ZOQ8KzVww9UMdho5x\\\\\\\/YIIY0fOfpaQ+x8Zc+VnE+4OPEIQkBpI6nKTBP9wNydlASl9oYSzKf7TVqOxyN3HpK++0MjF34+7ttiZghIZeIp6fIO4JriNMLRk4sm8ClC+cFNTvMbbP3AiS1w6KiSR2k3V4jVboe05SnEI9HqoqDcw1KkqHnxFrPj6uFtRmSiXbj8WIlwp9ZL\\\\\\\/UElMd0R0\\\\\\\/lyxyT4zQHqkdXe4L5DlDZxqqSwBrao6ZpocbCtEsLVXNox3gkWMn6UbGexPIO3GIpFL\\\\\\\/DMLDO+BeaBeGYdc7OC6RS5N6umJ6NbwE97OrVWNK3YiDtqY9BpoELxpfw289dTwPNxETBRqWY96WhdRAya7EHeedcLTC\\\\\\\/Jbc\\\\\\\/heUwEE94pjGbREjQMPflTAA1Z10bNKd7u4hg\\\\\\\/FhgvEa9NZ1DexyjcS5j6\\\\\\\/jb2SBSDFOKSnXj0l5KxZ\\\\\\\/4aDud2Re+L+urdaasjNL9BciH7TBvelspuyqZwANOA9IDFahRbgPwQMV\\\\\\\/JVYCGvNTt9DGeO4j2j48mxH4FVEtBZE0z5GKnYbaAurzkZsboI8Zar7lWHjvT1zKxginSUdDOa+DiQpQ4m3UsmH7f++8koiyWgLh+cujoqM2OoweeJF1QkLcxwAl8iEPPkwGpLmIvypoQjKe2bTmzcJpDEAV50gZNsZTTN5L+C32Gg+TPRQ3nIcUOXcZm9lYW1iL8wzhtNVAROW5AhbZh91NKU13ahT\\\\\\\/ZaQ\\\\\\\/Z8QxiB3FvRWlHYV7oW0bbBOyqOZphkuxacqpzg6h4fi+cdWHclkH6EpeYYFQWwm5v49IBWmu5ORbk6Cgw4Nyk\\\\\\\/oMDecK+dRt4UpOSHMb0qglhlK4NrrLT7YGgCPfvJpKIubL\\\\\\\/2mQtARWvw3w9LgfVI\\\\\\\/isLsDBC515YYjrPSYoGHDkb15+10qQ78EbJtkt3Mo857Gf\\\\\\\/B1CG3SKZG84H1\\\\\\\/UioeeW9yDDzM4ag1IDuaHuQdXkznBMGqoVk0f8IWvTPmYIVytaQzZ+lg2FjEyt0fz6yWXxEOAG\\\\\\\/j0zAd2pTooT4H3lp06EailbSuIJ8CJS\\\\\\\/AYMUNiKDFXTT\\\\\\\/8mw3oyHIO18WmALUxT8gd\\\\\\\/Be9HU5WXsVq3WVWqzqjF15NFKtgHxYvL3K+103kgRPVZn02f3L5IbnS3CUFQwGhoIpCtq\\\\\\\/b10LmFtaQYTuABxQRn56MzhrEEwS+mkVhh5jxI1e1THdA+CdG5jcZgPstqufB0ENFPdN6rCMotFpOFe7TYbBelx6ur\\\\\\\/6vrWyrZ7OtMJ8k3Ndb72ujX9ilq7wnOWd78VoU6Tz2qoOuWm3sTPu5VgT+6\\\\\\\/v4oGmmCTqbRuugHniP1KEyFL5JfQxEW3RyQUkB7m5bKMGXR2g8wys3S6Un5YJxwRw6sHIbu3c8PB2lP9BiOIwhWL6+DY5cczwfmRFx9piUFfAmgT7ABgPpDb52a0pwvoiYGleywitcXo7XDdVsmX01AxwbgZ+KTJPUGfoZOjb66Ywr21U7xq5JyxKp\\\\\\\/hZVFhi3vJA7KvRfFEu8rP25uMCRa\\\\\\\/l4dTE78EIpbBYGqWhK2z2KnJ\\\\\\\/r3O3gYINVnp7ysh\\\\\\\/5YUlBGsTsvTNof8VLAHbPo+81r\\\\\\\/tI1g2hUFQejYIr2mOTPxl0MZVNlYImEhg9tZ+B0lcN6TD6i3MCl4ATuU\\\\\\\/D4td5KMZYtNaolOnDlF4iK71Jo2P4uBqKWREM8ui7EMcPXl9PS3haLpKGMePz9LMOX7wsS\\\\\\\/7nZrOUQ9X8+lwr3KJ8zuAdcJWNQt1XVPKnslpTkbMS53tncYefT\\\\\\\/25Ot5qZgwjexf\\\\\\\/0MLzJQeWGc490NdQyYPGHHKfLQsRQSsOCotbkewnRyBYmUUj\\\\\\\/O\\\\\\\/jpBOl8Z+QtG2n+OOPoaAF\\\\\\\/i6unchxPMzG6QIFrBFfTLLN2GDBkxglLesHyeMQ11LbONLCVmToHlGK1VSgtXigDapPS6zaw2tz\\\\\\\/ruTGKTMd4ijMQXO2xb5t5K+NjxvV+1J2\\\\\\\/5bUsJ3IIDimiMcsXhUkqkJ7ssNprD9PswImrk5ROcZ2s\\\\\\\/3D0Bf10L8bvPhgB\\\\\\\/0Kbv98Av9kmvxOw3k47u3C1kQAtcv8zaOdlIyV9fPK3+bkZdZ0H1ioopDwmDnyX7KDGnkF9bdk\\\\\\\/t718usOLo8qoUYMGefw+aR9Lg7Mu5ht1BNJCFoWc\\\\\\\/GBnjdxjkNlz33Ju3Ma6Nhcw9gHmhlPDiKol3DKEp45K2d47rsw1Slip5bjKKONpZF3CDQSDqvw03C\\\\\\\/IP2vCyAVecYqW\\\\\\\/qycH91Mfjs+ts441fXmX2FzJHcMu4Nf8BfPwFcigyWJk7wSkUrRTJh6YArcTZJJYAKCwjfmdlI\\\\\\\/JMbeyRvKy1X+7SmEPTLQCwPTOy2C5bxupfjK1veNNoh67tiueHWu8abzG4g49Eu4ll9++uTgbR\\\\\\\/UXOcnmjrugC2ak3vO2P1LXayCx1gzh7nx+SB5cZJTUBV9ry2oMErKBCe0x8OqlLxPKXMrBrk2QBwdhYfimuisSV0yzXTfLwOqCpy2nvr76QjHadS9QLDj\\\\\\\/NqFGIaEzxC9Je+7qnob8JNMJuRq9jqO9H1M4xXkfiLsGA3qlOqP2qg3tc781sE94hvPUFLleO12o1HkyYssN7y7cyqiESIz5dsonNz5VAmaZrheoYC6lmtzrwzKYAOxWOF5p\\\\\\\/d+jdh7htAVe4OK0wkVNCcpd1347wSrqkwk3NP1+pILF83mi2jgbCHyfEm7GLMXx6z8JqofLE6jL8IocSj4faAhlGBH96d8I4TQlSydwpYnSbmBqF6+Agt93kH6rq0Lf9eQGUu3t4i4NDkpblGQ+ZHmE20R8arg5nz7GahRa\\\\\\\/CqZ7As3sKj2Zmrhvcv3JKfIZu1SrmpJwqhxKmR+AwdOtng+TVk\\\\\\\/B8iovuawt42ILkBJKQV4t2s8IdZJqLseEh4\\\\\\\/3iUSoGOq3G31HSLIAGyTAvYoLRO+QA93tJWrcjpzhRz\\\\\\\/Lclf3TesXUPzXal2o8E3M1vyP0whqnMXD6aqiuOvvA87cIV4poKYFmODXYHgcyrsphS\\\\\\\/WcLxhUpqsQqXTMXzKLj45OK6dTyMCPjghg5OEvYAnbTJ6fGdbVtWqV2AVfUFItmmpbAuIo\\\\\\\/2pUtLz4cX+KSfDVBOXI\\\\\\\/GRWgW4sjSpbUVu4He0cPkdFY9iC5oAeNOcgaJlHg\\\\\\\/6eNAT41R2FNFRUeWpy18oTb35T1SCQ4oXQpcep2zHzgS4L3eLoSX8XPeMzhQHy+OtqE5kcCS7YA3lOZ1vU30Dhzi5YjKb3QtnGjP10aPI1T6szI0lhQwRr3dQZYjNwWuPZ+2mD6hDJiFb3uPoTpYrdpXw3U7YG8\\\\\\\/UPg42NThM5JUydG9PZLL\\\\\\\/lFcNQtvWfXR0po0JXSV6rixPxdk8LSB9qMbogpAShqffD6ubNMMC+C0RHbMLDX0pe8NtpbWDnz0CR1+WPWivrRDp8McfqO52Wot74m6FNDDax+Unk2WuIB+PDEZo+cn4usRxbQoVB0dUKMjVJj3RgP51jFk\\\\\\\/MSbIwiiXug8v0geg0L1w\\\\\\\/UxVaHW6uriQwWMvMrwrd+A0dHk4CaJRukDGMVCMKA4yt1TJXgKTQPn0eaGd+E6ScYHJbBdzkLSAJlfBNHE\\\\\\\/o1dSNQlMnb6DWQtiKIhGkF9lPeWRbAOm5wXBdWRYOP0seyjPw2uO\\\\\\\/kgk9DYCE9VPxLV9tdDSri\\\\\\\/9sxYwv5tIDlNXaWw\\\\\\\/7uKT8g3AINVzJq6cu2J8neBWX+moKtwcpoqVUl+WQ50X74qqXmhLNkJALGyIR1JsyKXx81ft0C+K1Sn0hYWlhHhaTnC0DnUCBLFjCVOvjzKy7qaOS0mgAM60SQkmFS3UbuftliIGfIVNv4uPRAjkPFs6zMII+knYMA7fJ\\\\\\\/eaMRBMnRYWPcHDOcAAeWWsDswkbR+Opceg0\\\\\\\/Vrswtpd3a3C5sbiQkyRoTKdRG1GvEtV\\\\\\\/LTsgdKyH\\\\\\\/DB0LRKssBLxHBALgm2Ebex47IehIN6u1\\\\\\\/MBTbFIBjU4L+8IXDoUwVU0LHnIU02jq6Quf3yL5HsYKAQ6cmnOyr28cwTJdccbIzAQOUV6rrXz+zQ3LwF0zBMSx3uHQYGN88n21jkeyzl5V+ZSW2zWiQiaR\\\\\\\/QjocOpl8GOWTvntRVlMLq9DlJgTPktJq4kdE3nfshwtPR6E8CM0aVEH+FJFgLMdEfsnw8ZPgEGAe8EP9E9L0pHcLkxSbskS\\\\\\\/ziY0j173TeiNu+sqopwYjBxalflV74SSX0r76ZwsOrZxjptPuixkkmz09hQIrkxY9uhVAb5p6Ayq\\\\\\\/z1ZNzNNN6LkuxZRKyOJ5RHJ4Ve+rUnFERzAASvdjnkMmqO+PGKQV+Yi3NPpiAHu\\\\\\\/SabOuKw0Jmp1xOff8knf8bc5UGgwbjfTFMb0Czyl8FuxbMhhuAivF2fJ0jk9qApJrmvI2T+HhTxfVEubsYveeHX79eCsNFVAA8gHXliDJALcW8EbmdSp2VMrSKV1coEZuBsCwvRSU3hwXYMsHDNlG3RFzGOMA936mlUVA8skHPzv\\\\\\\/NW4\\\\\\\/espUzhqEubyoIl6SrIYZmWpZWL3e3ZqRFi8YgSKtXJ\\\\\\\/oqXQHxNYLvXi2ryzl6qO6WU4B0kx2pNNyHnamv1CUtpel8lQi+OD4xRkWifPL3W5gxHczHgon4gbWOvwovNgufUW9CYgiBEOeSCMR+zWI6iZxJqCmr3IhVW7us5zyRfUxKUG9e4U3cA0APP0ohoY9xSpkY6JVFGd1owuZt5TjnX8\\\\\\\/ZfPUh1R4Yk6IuLJuJq6rHKHU24Z6goJr6LpAyn6KEd1rkOp269J64Ndw63Z3e7z9Df\\\\\\\/ZY6SuzxO\\\\\\\/pMC9XX2x7ee9vy6eHCuz8Fd9u3Kr3TkssQAyTLBZq2N8v4DON\\\\\\\/qf2\\\\\\\/PIUuP1xch0pyKeHeXNFq7uhRoAgnpcEQknWVEY+OGkYvojneCVfRxG88JCjXIHJtjgukgtzoOYQo2NCkbrgTAsPyQr1mp9qxXDG5G5A4\\\\\\\/MbmyPh9jf+blVq7qeyEUtCUgQvoMRCykOlsm+PfZsClihedV895pS7lTH0rBaWYfN5u2s+nFJIVUDES0nSYyG0yyAe2yabD6d6j1aKhLXAxmtXnFQAIKD0ZZfOmTaRm9fiKqQxLLx6NnjEW6fuRpS1bCVkgBp\\\\\\\/gb4fv3Dmyv3RLUn7IXffnLHZikLEX9pPUwm4E0SyL4a2RA9mkuuMdMX7\\\\\\\/ijaCYl5kYNlJgexXh9dak2vwxsqpv0oLR1catj20afYO9b1zFWmFBvXB0SEKctXHAkWZWaeUTMhGIXRdXlQsOGz8uEXPLil31HYlIHINdig+tlybGS7vBRkUOCgN4G+UIKnPdgS4i0RvFURHJzuzpscJo3bMvWBrivvl6OQuOhMSqHIwvVGuWEZb3co8SNG\\\\\\\/OwiTUelGXuC+R0Lx18G90XHxlZdSHWago8uX435th7rgNAoP\\\\\\\/Wzts8A+49W6Y8hJoaVWN3ls4FGu2x9T4D7x33\\\\\\\/GLuyXKrAjFvjvFSvx1GCLdmTva36HOPX5VdDfs9nKPvin6hh7EzkeLcvQBXc8tYyHxkRC5lrCMxdBf2kZrvvNhOtaY7Jt8RSaTJdGGRspucuTEaWzzqrlPStDZMYjLRGX8Nf5TuDBnK\\\\\\\/n9KL6UIRJM4PdKCH1oqxsIamT\\\\\\\/99lw1AEV8\\\\\\\/33rw9fCxDoWPR6EqLeYqNt8BwSxrMZwuJ3UF6g+GW+ah6IGslMFwY3uQVc06zcU\\\\\\\/JnT9fajoAhiEkviPhrNZAzaVB8EfOkFIsjT0hMbGIJYH+MF7vfjbUClOi0ntPIqfSb\\\\\\\/xUR8S4KE1I8lkI2kH9\\\\\\\/LiGE9ktIiLrqhA1FTaO35kqnmSiEwlliDkHrt2AbJ9OZKL6f4YVszeKkbB9kX1vWVn9QC9efJqJ7vOAQR324Zdsf5q9tfwJe8lTUmXtddKuE5a5x9zdWxNl39MukAxNiYSR05KpyC0jCnsKbqE156mJ0i0goGlWL5wZsEeXPjHj2ofW5dTYQgOtoPbt9\\\\\\\/GUOvveCe2kFowN+WUQkQ9AThPAsY9oum1I\\\\\\\/vZyQhoUCy3qhOiLbLylSGsOczzEWdbZqvQ\\\\\\\/2hsJSdrJR4N2QgKQHIJyVUM3KV14G+xMsAMfD9o930bTrmMMky8Bsjpr9VRyBrWLk7FhtB2rAfb3nmR2ig0qt3\\\\\\\/WmZJ\\\\\\\/p5tBuJNCfkmZt0XkL8lhlrsrAYZHyAjuAkijjc+3Ev522GF4rxrBgIJGYc+E8GtqKZgSicjCBl3kesZ1EQYh7c9jx40m\\\\\\\/V6OUdvxMNxwwh+I8FAZY7jeqKAqMXRZbB\\\\\\\/gJ7aBc+TvXzU0cA3nim3JhIvEw9MzCYuYESv\\\\\\\/QzODP8w16\\\\\\\/VRm6FLigt8w9VQeALxMbAzb\\\\\\\/pDTII1JhshOLWNrcsES7QbH97WF\\\\\\\/STu\\\\\\\/oQ\\\\\\\/HYA8l2r5+yLD9nGellw\\\\\\\/\\\\\\\/jZqS9AeIQdYyDvSKuY28L1bQ\\\\\\\/fEYJZupZX9BPuXwb8FIBwN4t0q5uZgs9p3QKRq0Z6ogVpwgdc0sPGUtVqSkVX8a1ctMuSjWpKUTHuhcGSRVX89d1LOQ98k8TfotgvsT3a4rcJwXReTlaY3QmSURpa3UFkzkKnHqv1WjerwTRoYoYyGH\\\\\\\/BWtywevJBPNTm0HAV9a2VEpHiXUlMri9j2LzJf+BaP6v2vtDAW8bN64ftNuyYzHMwke225QMzJdFS3wFKuyDr6Ti9zBzKKZ5Ccex9PhHQXJrUXbGGZmajsrFy+p7rWPcyNWdZGdSyPS3ArWKBv8WaJghn7DwCVx9FjnAezn9LbRUnAjJR7ZfQ6LmY2ADoPvtXZ3bZn+yrm3cTJ8fAJVOU6GDKgbe3grucCGEJi9n7qkHiaRMOEDj\\\\\\\/HZx0HNKOepo2AC2KIsgjSMTS\\\\\\\/d181NyW08umrAkyou2bftd22woU29h\\\\\\\/KfU1YTolPzwBZss5D\\\\\\\/9XxE4QhOrdsYkHASQrntVIv6ZDJWR8YxOxYsBZHMgqArTOuUnX8+VnkKmXbDOKdz6h+OYlMWYCeYYdOpiLOgGmOdbfVI8\\\\\\\/AIGBehq14OY\\\\\\\/K8qrJUDfxo4A18FiA1VzbaOg\\\\\\\/\\\\\\\/Giai3gOyDje+6N2Z5FUgYw9TuGbC+WieiZkw2PIz17qPErRCNancklVCcNNUd\\\\\\\/WvgEYdoh3gTHzsHN0JlYD6DZTbxdLh8TAm0IN9yCSv4koKMhb5c7f+pC\\\\\\\/no3Igsdd\\\\\\\/cL4o7tJUKXpug49ewF7ut9COCcIo7m5YW+1fW6SpZR0cVKs9Pl1YOdftejJqCF7YrrFYMe3fEv1QS9pzgkkxSFblY4bJ34tpPhmMIM4JJQ\\\\\\\/61DWKWvfBlVfhW2s+oDP8k0RW5EU7VAaaBGs7Dy6hfDInWs6S68S0woUDXXFIkAvVHUfrKbPag+\\\\\\\/DhDyYFqpNxDPYN9PLtKQ20fnlxzMDKe\\\\\\\/pl4T5RuGaIreheFQpQ6\\\\\\\/PR7S338ZUWZk4o\\\\\\\/JnHNMV2ctYI7YX1+EMtW27NtBqlRZ\\\\\\\/VjsJpoRxTg0GLQlUvVCsAk3SojAqhHEmnU94alG2zPZWKz4CQIyaQ+Q3gshWGwXik3+feBx\\\\\\\/LAZscDzYssQRNESnqHr642HwxM9URwtiviptzNjk\\\\\\\/Nj4IWlfnu8MVscE9BZDuEXNZptfxSHftI2e+CZrwQR5KgSCef18vJXnj6waxLv3YjHtrcQeJTmoq6pQu1\\\\\\\/vfKRGoS1k1hbkCZW\\\\\\\/OmX6WTxz7ppHLWWi3PN+leikLOWSd7ALbQBGhXKSL5cRtAjyGOq2GJxg5KJJO5mRSp9uiJ62xaW7hi8cQOjNl8vnRCIrDcWaa2+Idm2BeSi\\\\\\\/Sch0UPJ7m4JEUbSCsxD26YE1fF5KjDUJwIBGT3CINYmGx8rFh4iGMz9C+KjQPkGgbqLXHdLXoBvqGWoAaFmR9hyrD0dzALpHN7kjxmfj7fTFmGzIGS8sOXZ5KcsNogY\\\\\\\/3SIjMPivPx84XQFawDEaHWzRCBKN8GOHp3Sardz9DH\\\\\\\/+37+MWT5Bv8lZuGc6ta\\\\\\\/2Um9WflNrxAnxowiQeNeS2KrqvJvyt0j70oEg0\\\\\\\/y8mbMK\\\\\\\/Jvb1fXSfTqBBJBHGCj0RD0TqFAQFaT\\\\\\\/MYBknZYlEonNYAIg9eKDF9Zy2KbHRdpFvJn6m7qqlpLXyprx6Voocs2FvQAu7\\\\\\\/\\\\\\\/XnTjLWcbhnja3kzoxQmwBpws3CD9VPf9VA4mSaxGrs99QZV0qBnsl0Yms9iUvYMKUWOr622N89NtELhmkMNOGM22OtEVL6fbhTcQhpGJnzffTR1PPW8OLvHbmvPAg5lSaL9y\\\\\\\/Y0OFxqs6uv7Q7l+Cya1p3IG3OWaOVxU0Rns0tjPL3mcFgz\\\\\\\/Mc2yMRVFOY4dsEhUWM8JQtxKT9U\\\\\\\/Daw2\\\\\\\/E2A3ojv4sqn1yhcpR04q8AAp\\\\\\\/El1uBEBU9vffRJY9YhOXaOBivqCKg\\\\\\\/kjkLBGx+jvt4Yv0oYrVMtl1MfuKSskXfYhkNQz4CZVh\\\\\\\/2O4bb0PU+VAKmlvoXUVkzbuP33Rj8gRpsMvsa4EP+nRfLNo7nuxndxFIr3rbdoFELPYdblx+IDEMsrQ8rlNqydtN01IeOkiRTEOMBkPYYYAx3ThaGcQls4KJdAd5nJLFC+pUuQBjOfRE5B7YzK84FEA0c6WTm1bTGyqcS6fZuEms1dFD7Sg5IVSwO0Y1YM8b0ulVMfAQW1eweMpbxJ0WC0dV4at02gvfbHWj1TFOAKtAbJ058UmFW5zXz49yZzE59QF4c85Ir5Lf7IjD+qEXYQv+BFBpOnC1XF1la6tC4TBoh+vau3caPZiYFUcF3Vq6yv8mRigeYa0B7uQuACknH6BYanS6ruC03fjxT2Z7HddD8LGcCATB+\\\\\\\/Db\\\\\\\/Kym+is1k0joq+N0uURbRLm2YeOKGr1NaCPLOeT6+eyHxRhgcCevRGxwoyLICqs8hCyboMUCPdZY4mv6vJUHUE0djv9CkPTvCvAo0DK5FKZ4pLJHjG5oeyvBHiRMjyr5UoTqv\\\\\\\/wl9GtcT4Fw3mN0bwsuhhR78AgxGhPU8uYVvHB2sFbCWVrklOkMBJR9Zm24VvG8T2a2yQpzuKCVsxokVNrizu68YnHZP+fDlEa28sSa6jxEruNu5eRdWTakEQeER44cAsbrC6PX09go7sdUNKotqJsDMQIEukLB6WJTr+me+mTg4g2xktmwnzKozW3PQzl914BuSf9KI8in4PvGHr9KqOx4WdmR8vOFIVccmyj+EPK9JMoDQlXSGgsNJFOllz\\\\\\\/ShLQEcVcFC9hf2lQTPQEFh5HNKhaaGmscSeAr81TOiSGzDAGNgVnaOKzESg+Gq81Zs4qskA+d0W+bxGGwvp9K7LDwgQa38tng2hS+CDPwiNC9bWFjk0J8YxvBfo6PqamdhGMzxv3vcWeoYZ8J1hpK9IePwslGXDKaEBiq0tw2TCPNpyWxxPpsCiIV3JqOKAeqYKYTXF1LAS08E+adK5T+avxeRGvUo369lWYyF\\\\\\\/ik5FJ\\\\\\\/87afnsRbx6NhKbqop+WPIWhAuClCwlmW6hk6tQfR8w6ZBtiBdUYPzq+SpMZ\\\\\\\/gbM+uGH1IjfBiHQPcbd2sS+v+YDoET3CGFhbJGD1L0XBMLteG9xXzpK+aMLA1xXSOjk19H\\\\\\\/sSq9Mkhg+2DoQRg1kFIr27C6swBUby3qgUVxzKno1GgytvQFvVBebd0b+xgKacShGD2P\\\\\\\/MeekKPUhmdJJbRLNRGu2cmLc+3PQUOkZz162Sk2\\\\\\\/FtPOAJloNuEVV\\\\\\\/w3tc9yOpzZzRjB3QtHr+pJ9nSKLAuiyHu2NLQ70EuIJZua5QV1atEJAaojO\\\\\\\/NOtdA1LbdcdgrwNfouy0Tcw10irSKcoDr5hMLF3h98WlNeGkFV7F45ptvX0E0z8c5C0nEK6pebTMIwbW3ouHVGu\\\\\\\/rrWHq7ECIUqOn+dIwwuwhtPXN37YiudBe+gjBR95pThmRBV5+Xgq1ouOXZQSlyvauFG5U5e2xKUse937DmVsktdH3sN3WS\\\\\\\/oBPfAypEVqrLi6fbkly285Irsvhk5MO\\\\\\\/oXJeB+e01Kh+FHM0\\\\\\\/CGasKh4dNJStD8DDBvp28lHcbVh5wWcWcgkpeECjDc4BRz6SkANtbEPtUAFQOWE0+z4uYWD4TwXcNhz\\\\\\\/iuphH7\\\\\\\/5FvMr2idNpfa\\\\\\\/VFKt708VCV3cFN8W5mxgDGrXcqRzCpqWJOqVXbrpaV1B4gEUcOKxOAXorjZq1wtv2o+m7ApOOrbIR3ms\\\\\\\/6VXLLMubuTYsJn4AXglHImJu2wHjHc0KO22c7+DoQjjh+DbQsuhlaTqvnf0NxKEDgcnzgXE\\\\\\\/x0GRumhASs6YHbXjTMCYZqcKhWxNEvxRFjCPPAdV6xP\\\\\\\/Vvuyrs2zlTnLfoJuIWAiixAvxJjsAgom7JOdYG2EyYBSUBdk7BYzMdd\\\\\\\/2W3cKKht4nWHPc6KZezGbg5cdgYuHoWmSncgZl7hB3cFaaRcgA9gLCQh8lMm1VRR6p8VQBNcYlki\\\\\\\/KS3CWNsdByApzfrpYGOCmUJuTl\\\\\\\/RXuH2mgT2y8oEvgD4q2aBrLYi7gc6ehxcX\\\\\\\/pRWmsONKTuQIq9nBiVcl0AAH2fmiqF7ffckNx9qi9Jw5bBX3G+xtRJenO0f2ccmg0zjh1Gv0p4BLbztQ5UD75BOdRX8nN4fpyhfEACWZEU9iSTX52QWme\\\\\\\/DjPZ7Wq5bsdZ7ifj\\\\\\\/SJ24OZ2yQjDWCoIn77s9KL\\\\\\\/Nize6yE\\\\\\\/Zgwwutqt+lOy4kWTyMbDB3kvH+RDEDwgfhP9siCRygbrWw5ruqAj6I4NHJYh2nXl6OnrqUx1bUOAt+CHjmfk2OqiGZCqKdYCuOEwNw2spTVaotJTuaExi8LKWTX+nascwkEBQTCND61vZRNqgoT3S6iUmoRC2eBmnsmBS2jE33MzZ3yvh9PHgc0vEXboFlnW7ylfIgQiyw\\\\\\\/x0JeKAyShVoCOHSm145BUDAvWb4fjSBhjJ9qEAUT5ayZIp6ugbB5SVXB26WQiIZ52tAGV3KtERY3GAzOD11Y1DwXYJu2LiQaxN7cVx6XSmbqYfx97qg9woXa+ezAlTgGgvEPBS\\\\\\\/2t83RAU4mFO9RZOJo7szl7Do5NN62mc9Vvleaf3uYfieFrBVn\\\\\\\/a7HGrtlBCgmUBHOyIbcMTAy7iZnzutaDUR0\\\\\\\/CBx6E3jSaC25w1O+zbSGjYjT8D5N61yxriT\\\\\\\/F0qNFiBVVRsmgBDvb29sjTeeZby9xcpigms3lEhHEHDB44r93GNI4EvjLEY3+8MHLu89uDnN2SRmXd80pGsu3qjdDSiUC5sZ9iANZHussPz0i3Vo4VOBbwiyyCi59wU1YC69eJ1oQwhX25lOkMyaC+oYfU0fuGiAZB2+2TXKZBxghilnZqHBHf4tOCY3BK\\\\\\\/\\\\\\\/CLj2R3t1yZMRjAALNXdwSXRaLqeWT++fFtmZLztx1dWsZmh8MPSKXCEnH9fN3+dlu\\\\\\\/QVfwdu2ekhXkjr7KdDb9URZTszr5RMc0AfkcdEc+zaqNJd5KiCvMxTu21OyIf\\\\\\\/sMt+aJcf4WQTVH2kxsend67syFxTUJcGlBOR\\\\\\\/b8JkOVAHBqN2HfF\\\\\\\/C7WruL9e2QEJaKZH3HVIlWoOODk6W+YnIuV2r4Awn5Ygh21eZnJit0Eh2w9LqV2OS1hNnAtUgiRYH8+lApOMJbdwQso57yzyA6blScONYm4xyEEYviJ0vt39ygYHdoHNbwI1TIxOQ890eEN7qRw\\\\\\\/t7w9Em2GvoeQxZifHpZ7azCpmPvaf93tejeSkUmv8foCJhxwOLxZPw+fVmfj868bjDwntV\\\\\\\/YF86aPjKUjH\\\\\\\/a+kUXo2aAPixwmYVqN1gkZqddyotJHgxw7ej6jbq3EtIu0CA6vQgFtvLgY5cMdluT7IYGknO8uHg2rQiseL6cXS7Zi7qWBP+YhTUCvbjsjDDOIaGlwKnBZdcBSu9iKZV709E8BIS0q+wKE07m1kmJk2l7OPEGlKva+3Sd8sSRxe62NmqtqXRcBeobwVJcfSNJ1fMDpaJZR3p7Mljl\\\\\\\/PkyGGwu6PKUvSKfvY8b3izvdfJt8Ts6j1E1hIeLLx82BwKgBCEq1yt1HETgr6dlkjYbTTQBNYg+Il+rdoLT0sNzzTzYuntVeoPCDeXnig4u+Q9uYKzso6RmRrbefwg5HIw05cS0ufSuvQvRgg3pOWERmmfUvaDADfd35KTjjrcxKKZ0ZADttlvmODhkA7GvSAN6iOc87TtTRF34s6Ineyo9GEZMpaOI1mJYbixvV8Gc4bJyexF9gKrFDPZ4aD84F2IWYZSMyNOqK7oTBWlfKD+8KtKZXrYVZrC9NtorG51BZ0bed4flISAP8gqQNjKAg1xgQDsThedUtcWxAXf1JS3U30KvwU8GtWeUBZJTTeCdkrGZuaP\\\\\\\/U6GyvNfLyuLTEzaY2NaZIe6+DU5hDKYcDG8XFq8k1qSoTKKfGtdkG6ekmjVMPoceL9jM2mIJHNqqaGdSvlC\\\\\\\/wwS73ezx2Zu9x1mCpvlTUoenEiEqHtGgWzk4knbOc0gg4YSDXtHT7Xb+DW6ZEzTpxVPtjGW+gAgFs3fUtrVq+bxbpM0sGUCrxdqeUeEn+o+OCLjMvTD4gN8iXhs4cdX34AFLV5ft6gPCpAlbKdt9P1tPFYdxxuzUT4rYu2Hd1TXvpE01vPU+Q3htsN4rCOZ4uMHf9cQmFAUgXVfNsSWQDi0d0QUPCh9bTF9UPFk3xReNdQ+EPdsdpm6g3evHrpIc\\\\\\\/yqzA+Yv9BEejIIR1mly\\\\\\\/EOrljqmfvHvfKQTkQ8A4Igxr65w\\\\\\\/eERn8zdG0ZsLzmzk+Thr2cL\\\\\\\/cRM42BiW3jXFZUcEeMZx5wlOR2yBotymdLYsH74fcM4\\\\\\\/53SnV2gnuf4lLYuJkEJ5NApWRtMPA7SCF+ZMcvG0eN3ZdTVBm8387ojRQkAFSAiV1jfLJICb6UcMCIoT9d5cdySInwSm2tCogP3pDgtKw6Xl5Wg5e2NyX7zKJH\\\\\\\/zpKZZ3wh5yvEDixPetgq7aRG2vII3V3KSlOYPS2dbvjUGsetgr+LZlbQ7MHy+XiO\\\\\\\/waTmXjIU+6eDwFFRNiwbkS+MqsWQMBhgNO8lBIashKzYckBkdgrj8ZvHTDUS7C372591bsAdk33XT1Ijv4BeEbxkB11nivAfdfAJNK7DsEyEki22\\\\\\\/A4S1ewIN8jNTdV7qnDo3A56uIXZ3\\\\\\\/CINWSGsw2p+mnyTpIFmb1WtY2CefDXVa6rawf\\\\\\\/GQoHCqWR0bnPC4+N2NVL79cXzIqVMxTrDU0P2ZWZQl4Mu5WypVwtMgrpHwY04jLSd1UPi4BN3zbsIl0ygqEiN+bK87p+Rk00k2QB9xyadZ5K9dlv\\\\\\\/cf5cLtp2yB+oWEkJ9ATQtOCZlrv2sf5+Q4E6hX21Q9zX75rPQCML31+ZVdDkiAwSy6h3hg\\\\\\\/cWBTb4nEEf\\\\\\\/T001dmzsELFPE66PnKc+cwdxKZFJ8hx\\\\\\\/l7x4KxHnPKMqpgCY2B8y5RlQ6FhTHcH7wrHIFpFYbFIsTqggBSwkA4wFZ9TwrZTQq2TbdjHRaTgMM4KPdWj98XuWxNFtANSk0EdNBdg3ig1\\\\\\\/LxgO1mWZkFX\\\\\\\/7fBz8dQYlyLNLuHgHn7a2TndiVqcTawhGxSEIPN5H00QaVVB+j+Gf86xV3HiXGR0DvrynPTLQh42c0Dg8AYBpCOxWDDknI8Jd2jDMpb4wng3KUsGJBV3ewaccdSsx49RFAhPErG0B4YemggxBkmG26t7mCSy5trDSyAKQN6ikoc51\\\\\\\/b5+lBsl\\\\\\\/M\\\\\\\/L5duiCGgLgJaiCx4YS1EMUFOcDM93gWZbx5blNkf5Q0g0wyUhH2fOaWhDWQF7HMg722phFTg+6AulHEGOuUU+xu3q+vpRv0adU1JNU\\\\\\\/hdJ4UFMWXtZ5wiNlB5GFaSY8AUvABvhoLzmEGZ+R\\\\\\\/SICTRwxeyLAdaAeBRwwLCv8I5D76x\\\\\\\/PbDbHmSIXIxvAdmX3xuMEtj5Q0s\\\\\\\/iioo9GhHvLBzGLSvbIvMV9MWhvWFzXxNTQNdB9kNFapd9JSktOoRqBC\\\\\\\/LDTiek5bvzehw5bKTB95omuqoIofHcywZTKjbPQpuBoZE6rH2xGF4ibA1GuNytx+iXcZYAxDYQyTS\\\\\\\/Dmc7HOszG\\\\\\\/NtXFu7+bxay3HbpM2sGfPiGItVD55PNgGXU\\\\\\\/0lQTFrGevkU1ntmtrJPPjbRMC1VdwP1e0nn5757GCQG7JWMpPWDDNOMvM8R2gQiCKGzQkF3LHrNGWUgRo0dJ9JBs1B6uI0egT5dV0BtAGG7IWph+gEvbL71wXOIo1z7cXRTc7mQf41EjzQ4AyC2gqb2mLPyqTkMY8dyxKJGx4CzNhXjPhR9OhPB\\\\\\\/SKSad24a25TkKLgcn59ytwiQW7N8QyesxhqDDMmmEcKD9UtBopdiS+sThhWugUd6IZp7CTkXo3DXFQvDdHGI+JrZx7y4MsAYC70MeULQpsWicZNMeiDqDppfaZebWDYudpX5XQZCFlrVNMTnpf2\\\\\\\/bxFz7ocojCmvrOzGrGNd45LaADAgNH37kdtHso+2EmYB3aRMBAljg780umKXUO0FqLwfS84mxRDRRc+HANHemr3s4vdPYFnLLBjab\\\\\\\/xSe4munwbR\\\\\\\/bdiarHLKX0dSS6KxdOwNro\\\\\\\/UsstPzXV25eZd19CYwH53ZqWsqZ0A7e9xlQ1fBirvJeZWCZOkJIZt9hNtqVJX6w9342ivBj0SOXgy\\\\\\\/Q\\\\\\\/SxU8+NOJ+lg3BTJUoAwC4UFqGxROdUSne67XRiudrrxj\\\\\\\/QHsKKGCAlnEFZpeYxG5B0SO8KZsUWtOLqd1ncV3nCG9pfA3SgHpJDNf9OCaiOgYew7QjKjp03uyFegOzy4EdJYj6T\\\\\\\/989PNOnxnX3rsVZBE5565sX1xDBi+gJBtx9MqVIJHqeWkwDhTdhRwupSGe5OYIY+P5hAupJuNXeSgE1mtQ4JhLnDJaueDGr4Sho7EluhXOLiurVR\\\\\\\/rH+GWaIlbD2XxpWAOXE3KTYUSgEa\\\\\\\/xOLNTDq+cqKoyftmn\\\\\\\/21SajTQ5vHVxfL0hPNNdQQjvNj8bUGJuNd8Pcz+4Y6FOdCRr+01Kel2WCo6BmkHImibHbfrHm3psRMbIwlOYyTewouXwZC547CsR5WjAbcG53IQxd2OvNbGPhW4Fo0QXVZyMjPoQteLQQWuNu7sKX7ko\\\\\\\/3d7TYUAyx4jbIpuOQcixeGOFeWxSRqgSJRE12f870dD+U2L7KV78zMLyPWyi45RVVGVDt7M5VD\\\\\\\/RKfA2pAXznm0NGAPBXqhhocXJEdAVu19cOus3Esm0fhHSgLwTcrIuuapiQL6v6GBm2K5P6pxaSpLX0u0excDODsmp8VPJphHsfuI4sKtndAWXYu62vT6PWBADl1PjI\\\\\\\/pwZWYAXvFOecEnBPyizc5FrnU3ozTEdXaMuCFAy2r8ggUTpRk2j6U0jYvK6wpvu\\\\\\\/IyygGlBzxqRC1RSratM0rVCDv0Fwht2IYZP\\\\\\\/X8c8yXuZNciBE5TJf\\\\\\\/s3ZrWYmDB62jQptIABOEQtvGJRF6Mmu77CKg0Z6SZI7FpM72qPrKtgb3UNnzepxyq7kWhYUz4vQPBdhOANKZ3tmUIwlhn8GNWVYKpkbKkXL12LG8Kppj+IdSo6ypayD3agpMEMWN1h3Lizk76ngj+UOIKKrl1m1iDGJXVmB0\\\\\\\/SQPwryd99zz6wXd\\\\\\\/OQANjYSbb+bMPGkQxxWZfIhvquMIFPkWLiGbPLD1J5vdz05H5DTnRADoCDMh\\\\\\\/ZaOv\\\\\\\/qK5evJkbnwRKnL4USYPfBPageJpzih5Bq4j\\\\\\\/aMmuORLIhbA73tESNjkJwk9mUfJab00dw9bVrcemSKNTnrvxtyTLipTXSoA5wgevhI9R3OHomMYmr\\\\\\\/qrY0mfjDn+0vcsBud8r6zXZTr5uBHKNMbspxWtDYCk\\\\\\\/geosSmWiJXhKO5xIn40kAkl3UFyAuSjZmCA9b25gIDvyGAVeTpRsKF4Y8NwAj7KK8YxFfsHky7tn15GmcqCypB3RaoVqYXgXCAEyAbXaQYwEtk2ym32Kpi\\\\\\\/23KzglziNh\\\\\\\/rE7iuSlS+SO8Aa8YYbTJOQIwiDa57EvVm0\\\\\\\/r3IjCgmMqLNRDKaZ2U2XonqdLic1Fjp0wb6IIuniOaGWad+hK7Lfr\\\\\\\/aH2eEDpN+oUpfL4+lR6Y33yCJu7GqAplE8eKtR8J1nO4m+92yjxBrGXJEUAsQ1oerqfBhI97\\\\\\\/bzWmzlqWL43G098ZeawM6RXu5LOtth0ikOThX4qSHCnRF8IZrMUkeRyscbyESlIa79WulJA0ytdZGa4JhI2JRO4vkm0iCd3Q7FTPqAkimg9GODNkwxrmlgEjuQND5Q8YmUckOrZYkGSafTNBQ3Mk1kUqmDJuFpAgC3\\\\\\\/KoIOW8\\\\\\\/ztkNYJbqm6OVOngr51Gft01CSr6bFCm1tFzVuX7zLhiiWRtFRtlW6ELhiZqs2BAStbldTuFrINkDjSnNQZMZVvdKSLS7pfbn1yVGki\\\\\\\/5L2rDYraK7gNA2KXQSJppZq5fF6UROg9DZ73FVOBzM3Tz8HrkpldCXHBXvUMGTAe4nRsdzgwnkUDVc1H\\\\\\\/2l5W3HhaqHL+2d57\\\\\\\/C3WXjEy5eO13mNDOX\\\\\\\/DwZszi1wQc0MXylDG85eGY9N\\\\\\\/JEvx2VE264h568lLvYc2sflyLU8i2U1S9wjtcAU5N\\\\\\\/AYokNLFvkF3khdG6xM2YdrKUoz0vPiRnbl3x0sAtED7ZMIpy8VbCbeKUpCnaHF87hcxWsjiJihJjs2FvIXxogWvrRGpdMnvOkCIzHV3WKI6axhq4koiuma+NhVkFhdPIBKbV+Uehva6Eg1OUnT3Pqkh94YZx82itpXRIxJCGn2lYX3lCVHNADuc9UDj9FbzFRdfnDH0Oto\\\\\\\/nVfgTkLMj9+FjKDSqDwqCCV6TSEfVSNqi+F+JpGNrj8+llMyLQIvy2A2U1522kFSIpaw9qCYw3zERHFSeVybqSY8iD9n0wQ5e7Odwe5r3HSGiSDu2XvmhfVcYFqmNNoIhHYIvgEhg3hYKdYs3wKahykFpvMCRp0QMDA2KY9MYoMJW91xzNGwEbU3uPugjkrh7+9hUKhEXD4Moz3WEagPHypU2YDzSxf22XAsEhuEpj5+LzC1Igk95MQlOQw32wzDG+LW+8INr9OJjTCpDTcqQ3GJS4fQa5H4Is0UyD5CJfj2m0xJpPyr0CeLXTXGJExgFdFLhsjRIznC\\\\\\\/X58MGKLCGigi3FskoZDvkhWmoGV4SBfuHr8egpOq8ZpqFkJPQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022OMpz7er7iUwp5T8BvQ\\\\\\\/Imw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002217fa1563541266e24c4112875395cecce32567cccdd5e974d9f805569a8ec6af\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022WRH0V2g1H8azgINQYaiC+jENxMC1Q5HCGUz6CCT0OxImwaWr15Fbri6XNT8fKXIWQpBfbqhVUJ5fQcn49SuZ\\\\\\\/bOsVVUzXyaRbxOu8zkdcOB7XZzEwVEsZJJttVX6NAQQwzRYDM5Vsm8QkIVYl+K9bWgaaqaXsbxV2Gi88yV9hUsL0xMQzHfwiDygghx412XSb+21b2TLHt2dtlCPk2Y7vBMpFUIxoPeXv9k\\\\\\\/JtCB+OxXw1fyZ76L5oOc+P0B3SUo9y1gzBfkxyXeJccDTzqKTQdT+Z4lwd8kFm6+G4vMnBdr3iPVPqh7r+CSUcwRJzgBHP6CTUxRz2RjgAASHgltEudZaX4aMLFSVTpGz1leX+J+x\\\\\\\/kz9BA6eSN1RpvXpaCktLCZSrVjF2mluBKgTAAkdUqBrk177B6u\\\\\\\/ITOGcblOcrF8ctKzhUudso1xzMEZi5emDb+W9ab6Z2Ntz54GF8fQMS7eSQQd\\\\\\\/qqEHZ+ULsJzZB4MRtWYmqH1trHhpcOMrRmi5h7G\\\\\\\/lXK4eBmEU04KuAR5M42QjnFUYLbEFNxTrf2EUIRxNehddSD+0p8O0FRlNaK2UV3fKrYU4ubkdlpQTI1A0NYlDxmGcjPvAEGfY3D7PV5Aw0ov7ncvFFdPtIv2n3wQObWfYrcc415j3EawZMVhhxbXdGv1avlAISgJg3RTCdmtT5drjEG\\\\\\\/hEWL4PWjTCUllfavC8F1SvADfNbjj+rAIad7M0n9wCSy8ipGEkNcydPQD\\\\\\\/m5gaTcdeR0sVn7IavIwVN9LBwRRgotIh3L8ioDvuKPIyn+TA24M9kP6xhpWXpQEkjd6E1MEpBIUzvfv9hdXUWFhLMNzOBdk9C3RZvta84lOxMNyhtk+YJU2EV+4iJgsihs+g8xTJUtPoQ2V+DZR7+TJcOBlcHCGZzTYzOOcNxRYJn42zHafTIH34iOlje7Hi2z1MQDCUIF4r0VjnHArNN1i1kN8GNLznEL2RsO49F4SJXN1ClUJaKnw+vKVUrY6eq9srPa+CpAn5A+X0R5Biip388OdDi292cXf5DkQr2UCZMUF5mtRI8Og+cTt8a+8pIEisdMxJPrsd\\\\\\\/V7PaXCa6avmUoWVKnrDolbzxYMtyBxBqA\\\\\\\/EiwvLDSwmNW8UWEajJx2UKsnLykrXitRCA6lSGamqGvWeyrCdKFIE8ItBcJi6tNAiEjQnbRUmM9mK+THQBloARbQr78QP0N2Kc8SackHQW2sLboV+gnLhrFdGVn9RtIsg+MfIzNlISJbxzeIiQnNqLdTh89EhWG9vfocWYzLNy89mPdWt\\\\\\\/dwNxUKRk1xrNVLq+TEjtoIzUL5Yk9Y2UzKIFFP7H7mXkZHJDGp3NhLv3zGgXfXUqtMX15v70Bh+RBx\\\\\\\/WIRbbPv+SMc\\\\\\\/BHhClAMdqojweKFUfY\\\\\\\/pRfpvNC+iSRzdX7yLCxwiCAUUQqb9zlhkw3tLHQdaXH\\\\\\\/uiBwo4VSlLtorGmEG2YqHVR06s0BbFFYwb71ijK2Bk0fPPhRYzZqhbdSwrQ0sjdWjsTGxpOvPvh9sLk9RoerM8tBpbVq5DTimGJyYMWiUzHwkB\\\\\\\/yOBUI9na00LUnBECnkzUfQmaCQwGqwSwtmd8Kwh2nWOvPPzhGWwjtoyH65LynxS0c3XRFpmstt9hcJoq0SyakixMDtMZF3V5DuXVZEcl2ADwUOR0lkbf8EcTIAEl940b2nSWA6vWgxaiitfUCPdesR+whSvmHKLiCjUI5IbDWAQJClGLqp7QYjeKxktH\\\\\\\/VDQJIJWcLwrpHmPzFqy4auuHC+KDpVIzYWUaWvJ9XvHNfm0gWYRh6u3KOUME9Vd16TztyoIaLrAUpASf96FEmNslX1sjm5gLD1yUsYgGXc0rdu+hlQ9jEpyr3c8J0I4Vf9r4fbPcJP44CxJh1ygFzZWexNGd5PgcoZiJ6PqxhVPAZmoKBLsjdUxifT\\\\\\\/zB\\\\\\\/0EGGGWX2oGRfUIXPGTTgvt4bEteEt8TId53LMLoe0SbSEwN0tjIiMEryg\\\\\\\/CPYf7yQ==\\\u0022,\\\u0022iv\\\u0022:\\\u002260YD5+DBOEjFePH+L+8wzQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002204894f08251b776e36a3bbc6dea3307843fbb991492e1b6fd91ed66a2f07a086\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-492598 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/4/1153" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-6hi00M9IEPli8G2-ajx2PSE55aRXQcX46FV7hSCvKtM" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/4/1153" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-XMRRy1xttDzyWHEQ_p5GdGl8BXWb3B1oUxubnKzxQDw" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/4/1141" title="RNAi Screen in Tribolium Reveals Involvement of F-BAR Proteins in Myoblast Fusion and Visceral Muscle Morphogenesis in Insects" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/4/1165" title="Bidirectional Selection for Body Weight on Standing Genetic Variation in a Chicken Model" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="492598" id="node492598" data-pisa="ggg;9/4/1153" data-pisa-master="ggg;g3.118.200976" data-apath="/ggg/9/4/1153.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Short-Homology-Mediated CRISPR/Cas9-Based Method for Genome Editing in Fission Yeast</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-1953-1664" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Aki Hayashi</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-9692-0968" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Katsunori Tanaka</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1153-1163;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200976 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node492598" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Aki Hayashi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Bioscience, School of Science and Technology, Kwansei Gakuin University, Sanda, Hyogo, 669-1337, Japan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Aki%2BHayashi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hayashi%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAki%2BHayashi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-1953-1664" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Aki Hayashi</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:akihayashi@kwansei.ac.jp" class="" data-icon-position="" data-hide-link-title="0">akihayashi@kwansei.ac.jp</a>
<a href="mailto:katsunori@kwansei.ac.jp" class="" data-icon-position="" data-hide-link-title="0">katsunori@kwansei.ac.jp</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Katsunori Tanaka </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Bioscience, School of Science and Technology, Kwansei Gakuin University, Sanda, Hyogo, 669-1337, Japan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Katsunori%2BTanaka%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tanaka%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKatsunori%2BTanaka%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-9692-0968" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Katsunori Tanaka</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:akihayashi@kwansei.ac.jp" class="" data-icon-position="" data-hide-link-title="0">akihayashi@kwansei.ac.jp</a>
<a href="mailto:katsunori@kwansei.ac.jp" class="" data-icon-position="" data-hide-link-title="0">katsunori@kwansei.ac.jp</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/9/4/1153" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:492598" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:492598/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/4/1153.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:492598" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:492598/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/4/1153.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:492598" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:492598/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/4/1153.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:492598" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:492598/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/9/4/1153.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:492598" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:492598/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">The CRISPR/Cas9 system enables the editing of genomes of numerous organisms through the induction of the double-strand breaks (DSB) at specific chromosomal targets. We improved the CRISPR/Cas9 system to ease the direct introduction of a point mutation or a tagging sequence into the chromosome by combining it with the noncanonical homology-directed DNA repair (HDR) based genome editing in fission yeast. We constructed convenient cloning vectors, which possessed a guide RNA (gRNA) expression module, or the humanized <em>Streptococcus pyogenes Cas9</em> gene that is expressed under the control of an inducible promoter to avoid the needless expression, or both a gRNA and Cas9 gene. Using this system, we attempted the short-homology-mediated genome editing and found that the HDR pathway provides high-frequency genome editing at target loci without the need of a long donor DNA. Using short oligonucleotides, we successfully introduced point mutations into two target genes at high frequency. We also precisely integrated the sequences for epitope and GFP tagging using donor DNA possessing short homology into the target loci, which enabled us to obtain cells expressing N-terminally tagged fusion proteins. This system could expedite genome editing in fission yeast, and could be applicable to other organisms.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/crisprcas9" class="hw-term hw-article-keyword hw-article-keyword-crispr-cas9" rel="nofollow">CRISPR/Cas9</a></li><li class="kwd"><a href="/keyword/short-homology-arm" class="hw-term hw-article-keyword hw-article-keyword-short-homology-arm" rel="nofollow">short-homology arm</a></li><li class="kwd"><a href="/keyword/point-mutation" class="hw-term hw-article-keyword hw-article-keyword-point-mutation" rel="nofollow">point mutation</a></li><li class="kwd"><a href="/keyword/knock" class="hw-term hw-article-keyword hw-article-keyword-knock-in" rel="nofollow">knock-in</a></li><li class="kwd"><a href="/keyword/fission-yeast" class="hw-term hw-article-keyword hw-article-keyword-fission-yeast" rel="nofollow">fission yeast</a></li></ul><p id="p-3">The CRISPR/Cas9 system provides an outstanding prospect of manipulating genomes via DNA double strand break (DSB) induction and repair in numerous organisms from yeast to human (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">DiCarlo <em>et al.</em> 2013</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Ran <em>et al.</em> 2013</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Kondo and Ueda 2013</a>). A guide RNA (gRNA)-Cas9 complex with high endonuclease activity is used to introduce the DSBs at designated points of a genomic region. The induced DSB is repaired by one of several repair pathways, mainly by non-homologous end joining (NHEJ), but also by homologous recombination (HR) and noncanonical HDR including microhomology-mediated end-joining (MMEJ), single-stranded annealing (SSA), and synthesis-dependent strand annealing (SDSA)(<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Verma and Greenberg 2016</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Ranjha <em>et al.</em> 2018</a>). NHEJ often introduces unwilling insertions or deletions (indels), whereas the DSB is precisely repaired using HR. For genome editing, HR-mediated repair requires long homologous sequences at both ends of the DSB. In the budding yeast, <em>Saccharomyces cerevisiae</em>, 25–60 bp of the homologous sequences are adequate for chromosomal integration (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Hayden and Byers 1992</a>). In mammals, more than 200 bp of the homologous sequences are required (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Rubnitz and Subramani 1984</a>). Therefore, preparation of long DNA fragments as donors for integration is necessary to perform precise genome modifications using the HR pathway. However, it is time-consuming or costly to prepare long donor DNA. To introduce point mutation or chromosomally tag in the gene of interest by the HR pathway, a series of synthesized oligonucleotides and sequential PCR steps are required to prepare the donor DNA.</p><p id="p-4">The noncanonical HDR pathway is an attractive pathway for genome editing using the CRISPR/Cas9 system since it uses short homologous sequences to repair the DSB. MMEJ is an alternative NHEJ pathway and it seems to be a type of backup repair pathway related to NHEJ and HR (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Sfeir and Symington 2015</a>). The NHEJ pathway requires the repair proteins including the Ku70–Ku80 heterodimer, XRCC4, DNA-dependent protein kinase catalytic subunit (DNA-PKcs), and DNA ligase IV. The MMEJ pathway, however, is independent on the Ku70–Ku80 heterodimer. In contrast to the HR pathway, the MMEJ pathway is an error-prone repair mechanism that requires only 5–25 bp of microhomology at the ends of the DSBs (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">McVey and Lee 2008</a>). SSA pathway is also error-prone repair mechanism that operates in the absence of 3′ single-stranded tail invasion into a homologous DNA and requires ERCC and Rad52. SDSA can also be error-prone, DNA synthesis occurs during the extension of the D-loop intermediate at the DSB, and requires Rad51 for strand invasion. It has been reported MMEJ-mediated knock-in using TALEN and the CRISPR/Cas9 systems in worm, frog, and human cells with donor DNAs containing short homology arms (6–40 bp) (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Nakade <em>et al.</em> 2014</a>). Recent paper has reported that precise in-frame integration was succeeded using 10-40 bp of short homology arms of donor vectors in zebrafish and implied that MMEJ and/or SSA would be involved in this editing because of the length of the homology arms (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Hisano <em>et al.</em> 2015</a>). For the SDSA-mediated genome editing, single-stranded oligonucleotides with 35-bp homology arms could function as efficient donors in mouse embryos and human cells (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Paix <em>et al.</em>, 2017</a>). It suggests that the HDR pathway can be adopted for effective genome editing in numerous organisms.</p><p id="p-5">The fission yeast, <em>Schizosaccharomyces pombe</em>, is an extensively studied model organism since it shares similar chromosome structures and chromatin modifications with higher eukaryotes like human and mouse. By exploiting the relative ease of editing its genome, the fission yeast has offered many insights into critical cellular processes. Fission yeast is also a suitable model organisms for optimizing the conditions of genome editing mediated by the noncanonical HDR. Its merits, including rapid growth, single colony formation, and uniform genetic background, facilitate the determination of the optimal lengths of homologous sequences required for precise genome editing. It has been reported that MMEJ takes place at S/G2 phase and requires Exo1, Rad52, and Pol4 but not Ku proteins, suggesting that the MMEJ pathway in fission yeast is related with the HR pathway and the SSA pathway (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Decottignies 2007</a>).</p><p id="p-6">The CRISPR/Cas9 genome editing system has been applied to fission yeast (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Jacobs <em>et al.</em> 2014</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Rodríguez-López <em>et al.</em> 2017</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Fernandez and Berro 2016</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Zhang <em>et al.</em> 2018</a>). However, using the prevalent CRISPR/Cas9 system in fission yeast remains laborious particularly because of the complicated gRNA expression vector construction. The gRNA expression vector has inefficient restriction sites for inserting the gRNA target sequence, resulting in the difficulty of preparing the gRNA expression vector. To avoid this cloning difficulty, it has been recently reported the cloning-free system that assembled gRNA-Cas9 expression vector by <em>in vivo</em> gap repair process (<a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Zhang <em>et al.</em> 2018</a>). Overexpression of Cas9 protein by a constitutive promoter inhibits the cell growth (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Jacobs <em>et al.</em> 2014</a>). In addition, preparing long donor DNA is labor-intensive. The use of synthetic oligonucleotides as donor DNA for knock-in has an advantage to save the labor for donor DNA preparation, and it has been reported that the synthetic oligonucleotides could introduce the genome editing using 45 bp of homology arms in fission yeast (<a id="xref-ref-37-3" class="xref-bibr" href="#ref-37">Zhang <em>et al.</em> 2018</a>). However, it takes a cost to prepare the long oligonucleotides.</p><p id="p-7">To implement the short-homology-mediated genome editing, we improved the CRISPR/Cas9 system by generating more convenient vectors to express gRNA and the Cas9 protein in fission yeast. Using this system, we optimized the length of homologous sequences required for the precise genome editing. We found that 25 bp of the homologous sequences at both ends are sufficient for the introduction of point mutation and epitope tags into the genome using short-homology-mediated genome editing. We also found that the double-stranded donor DNA could introduce the modification using short-homology-mediated insertion at a high frequency. In addition, a one-step PCR product with short-homology at both ends could introduce a sequence at a target gene locus to generate the strain expressing N-terminally tagged fusion proteins. This protocol facilitates high-efficiency manipulation of the genome with less effort and allows for the investigation of genes in fission yeast and potentially in other organisms.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials AND Methods</h2><div id="sec-2" class="subsection"><h3>Strains and media</h3><p id="p-8">The fission yeast strain PR109 (<em>h</em><sup>-</sup>, <em>leu1-32 ura4-D18</em>) was used as the original strain for all transformations. All fission yeast media are described in <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Moreno <em>et al.</em> (1991)</a>. YE (low adenine medium) plate contains 0.5% of yeast extract (Oxoid), 3% of glucose and 2% of agar. YES plate is a YE plate supplemented with 225 mg/L of amino acids and bases (leucine, histidine, adenine, uracil, and lysine (5S)). Edinburgh Minimal Medium + 5S (EMM5S) contains 11.77 g/L of EMM – Glucose (Sunrise Science Products), 2% of glucose, and the supplements described above. Strains used in this study are listed in Table S1.</p></div><div id="sec-3" class="subsection"><h3>Constructs</h3><p id="p-9">Plasmids and primers used for vector construction are listed in Tables S2 and S3. To construct a single gRNA expression vector, a 978 bp DNA fragment containing the partial <em>rrk1<sup>+</sup></em> promoter and a 966 bp DNA fragment of <em>rrk1<sup>+</sup></em> terminator were amplified from fission yeast genomic DNA with primers KT1910-KT1903 and KT1907-KT1913, respectively. A DNA fragment containing <em>rrk1<sup>+</sup></em> leader, gRNA scaffold, and the hammerhead ribozyme was amplified from pMZ283 (Addgene ID 52224) with primers KT1904-KT1906. The three fragments were mixed and amplified with primers KT1910-KT1913 to obtain 2,435 bp of concatenated DNA fragment. The fragment was cloned into <em>Pst</em>I-<em>Sac</em>I sites of pREP1 (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Maundrell 1993</a>) using the In-Fusion HD Cloning Kit (TaKaRa bio, catalog number 639649). The <em>Bbs</em>I site of <em>LEU2</em> marker gene was disrupted by site-directed mutagenesis using primers KT1928-KT1929 with PfuUltra High Fidelity DNA polymerase (Agilent, catalog number 600385). To exchange <em>Csp</em>CI sites with <em>Bbs</em>I sites as gRNA target sequence cloning sites, inverse PCR was performed using primers KT1932-KT1933 with KOD-Plus-Ver.2 (TOYOBO life science, catalog number KOD-201) to replace a 36 bp sequence with a 47 bp sequence including two <em>Bbs</em>I sites (pAH233). Humanized <em>Streptococcus pyogenes</em> (<em>Sp)</em>Cas9 was amplified from p<em>Sp</em>Cas9(BB)-2A-Puro (Addgene ID 62988) with primers KT1911-KT1912 and was cloned into the <em>Xho</em>I-<em>Bgl</em>II sites of pSLF273 (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Forsburg and Sherman 1997</a>) to construct a Cas9 expression vector that is inducible in the absence of thiamine (pAH235). The <em>adh1</em> promoter driven Cas9 expression plasmid (pAH261) was generated by exchanging the <em>nmt41</em> promoter to the <em>adh1</em> promoter. The <em>adh1</em> promoter was amplified from the genomic DNA with primers KT2284-KT2285 and was cloned into the PCR fragment amplified from the pAH235 with primers KT2286-KT2287 using In-Fusion HD Cloning Kit. To generate the <em>LEU2</em> marker vector expressing both the gRNA and Cas9 protein, the <em>nmt41p</em>-Cas9 fragment was amplified from pAH235 with primers KT2051-KT2052 and was cloned into the <em>Nco</em>I site of pAH233 using the In-Fusion HD Cloning Kit (pAH237). Site-directed mutagenesis was carried out to change <em>nmt41</em> promoter sequence of pAH237 to <em>nmt1</em> promoter using primers KT2087 and KT2088 (pAH236) and PrimeSTAR HS DNA polymerase (TaKaRa bio, catalog number R010A). To generate <em>ura4<sup>+</sup></em> marker vector expressing both the gRNA and Cas9 protein, the <em>Bbs</em>I site of <em>ura4<sup>+</sup></em> gene in pSLF173 and pSLF273 vector was disrupted with primers KT1975-KT1976 using PfuUltra High Fidelity DNA polymerase. The Cas9 DNA fragment amplified with primers KT1911-KT1912 was cloned into the <em>Xho</em>I-<em>Bgl</em>II sites of the plasmids. The <em>rrk1<sup>+</sup></em> promoter-gRNA scaffold was amplified from pAH233 with primers KT1973-KT1974 and was cloned into the <em>Mlu</em>I site of the Cas9 expression vector to construct pAH242 (<em>nmt1p</em>-Cas9 + gRNA) and pAH243 (<em>nmt41p</em>-Cas9 + gRNA).</p></div><div id="sec-4" class="subsection"><h3>Construction of the gRNA expression vector</h3><p id="p-10">For the gRNA vector digestion by <em>Bbs</em>I or <em>Bpi</em>I (NEB, catalog number R3539S or ThermoFisher, catalog number ER1011), the gRNA vectors (pAH233, pAH237 and pAH243) amplified in <em>E. coli</em> were heated at 65° for 30 min and subjected to digestion by <em>Bbs</em>I at 37° for 1 h. After the digestion was validated through agarose gel electrophoresis, the enzyme was inactivated by the incubation for 20 min at 65°. The digested vectors containing the Cas9 gene were aliquoted in small volumes and stored at –80°. The 20 bp of gRNA target sequences were determined by finding the PAM nearest to the mutation site. The number of off-target sites was searched using CRISPR direct (<a href="https://crispr.dbcls.jp">https://crispr.dbcls.jp</a>). To clone the gRNA target sequence into the gRNA scaffold, the guide oligonucleotides were synthesized as follows for ligation into the pair of <em>Bbs</em>I sites:</p><p id="p-11">5′-caccNNNNNNNNNNNNNNNNNNN-3′</p><p id="p-12">3′-NNNNNNNNNNNNNNNNNNNcaaa-5′</p><p id="p-13">For annealing oligonucleotides, equal volumes of two oligonucleotides were mixed in a microtube (each 5 μL of 100 μM oligonucleotides and 5 μL of 10 × annealing buffer (100 mM Tris [pH 7.5], 500 mM NaCl, and 10 mM EDTA) in 50 μL reaction volume, incubated at 95° for 5 min and cooled down to 25° for 30 min in a thermal cycler. The ligation reaction was carried out at 16° for 30 min using ∼20 ng of digested gRNA vector, 1/100 diluted annealed oligonucleotides (0.1 pmol), and 0.5 volume of the Ligation high Ver.2 (TOYOBO life science, catalog number LGK-201), followed by transformation of <em>E. coli DH5α</em>.</p></div><div id="sec-5" class="subsection"><h3>Design of the donor DNA for introduction of point mutation and knock-in using the CRISPR/Cas9 system</h3><p id="p-14">Donor oligonucleotides for mutagenesis were designed on the basis of the cleavage site position digested by the gRNA/Cas9 complex. The mutation site is located at ±4 bp from the cleavage site. Both ends of 15, 20, and 25 bp sequences that contained mutation sites from the cleavage site were synthesized for carrying out the introduction of point mutations in the genome (KT1023-KT1024: 15 bp <em>swi6-W104A</em> donors, KT1959-KT1960: 20 bp <em>swi6-W104A</em> donors, KT2089-KT2090: 25 bp <em>swi6-W104A</em> donors, KT2168-KT2169: 25 bp <em>mrc1-S604A</em> donors). A 575 bp of PCR fragment containing <em>swi6-W104A</em> was amplified using the primers KT1957-KT1958 and the template pKT2150, a plasmid containing the <em>swi6-W104A</em> allele generated by site-directed mutagenesis. Over 100 ng of PCR product was used to make the <em>swi6-W104A</em> mutant using the HR pathway. For 2 × Flag insertion at the 5′ end of the <em>stn1<sup>+</sup></em> gene, KT2133 and KT2197 oligonucleotides were used as donors. For GFP knock-in at the 5′ end of the <em>reb1<sup>+</sup></em> gene, a 763 bp of the <em>reb1</em>-GFP fragment was amplified using the primers KT2127-KT2179 and the template pFA6a-GFP(S65T)-kanMX6 (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Bähler <em>et al.</em> 1998</a>).</p></div><div id="sec-6" class="subsection"><h3>Transformation and genome editing</h3><p id="p-15">Transformations were performed using <em>S. pombe</em> Direct Transformation Kit (Wako pure Chemical, catalog number 296-64303) with some modifications, based on the lithium acetate/PEG method (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Sabatinos and Forsburg 2010</a>). For genome editing by two plasmids, sequential transformation was carried out. The host cells were transformed with pAH235 and plated on EMM5S–uracil-containing thiamine plates (5 μM thiamine), which represses Cas9 expression. The transformants were cultured with liquid EMM5S–uracil (without thiamine) medium at 30° for 1 day to express the Cas9 protein. The overnight culture was then diluted in EMM5S–uracil medium, and the diluted cells were cultured until the numbers of exponentially grown cells reached approximately 1 × 10<sup>7</sup> cells/mL at 30°. Mutagenesis and knock-in experiments were carried out by co-transformation. 1 × 10<sup>7</sup> cells (10 μL) transformed with pAH235 were sequentially transformed using <em>S. pombe</em> Direct Transformation Kit with 0.5–1 μg of the gRNA plasmid, 2 μL of heat denatured 2 mg/mL salmon testes DNA, the oligonucleotides for genome editing (1 nmol for single-stranded oligonucleotides, 900 pmol of mixed single stranded oligonucleotides, and 900 pmol of double-stranded oligonucleotides), or ∼125 ng of <em>swi6-W104A</em> PCR product (see Design of the donor DNA) for the HR pathway. For GFP knock-in at the 5′ end of the <em>reb1</em> gene, ∼3 μg of PCR product was used for transformation. The cells spread on an EMM5S–leucine–uracil plate were incubated at 32° for 6–7 days. The small transformants were streaked on YES plates and incubated at 32° for 2 days to obtain single colonies (<a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Rodríguez-López <em>et al.</em> 2017</a>). To confirm the loss of plasmids from the cells, a single colony was streaked on YES, EMM5S–leucine and EMM5S–uracil plates. The cells that did not grow on each EMM selective medium plates were selected and stocked. For <em>swi6-W104A</em> mutagenesis, the colonies were streaked on YES plates containing of 10 μg/mL of thiabendazol (TBZ) to evaluate the TBZ sensitivity. Using the cells exhibiting TBZ sensitivity, colony PCR was performed to amplify the region of <em>swi6</em> gene with primers KT1957-KT1958 and the PCR products were subjected to sequencing. For <em>mrc1-S604A</em> mutagenesis and knock-in experiments, all procedures were performed similar to in <em>swi6</em> mutagenesis experiment, except for the confirmation of the mutant phenotype and the insertion confirmation by microscopic observation and electrophoresis. Since <em>mrc1</em> mutants exhibit hydroxyurea (HU) sensitivity (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Tanaka and Russell 2001</a>), the transformants were streaked on YES plates containing 5 mM HU to identify the <em>mrc1</em>-deficient cells introduced by the short-homology-mediated genome editing. HU-sensitive cells were used as templates for colony PCR, and PCR products amplified with primers KT27-KT90 were subjected to sequencing. For knock-in at the 5′ end of the <em>stn1<sup>+</sup> gene</em>, colony PCR products amplified from the region near the first ATG with primers KT2049-KT2050 were separated by 2% agarose gel electrophoresis to check the Flag insertion. PCR products were sequenced to confirm the in-frame 2 × Flag insertion. For GFP knock-in at the 5′ end of the <em>reb1<sup>+</sup></em> gene, transformants were observed by fluorescent microscopy to see the GFP signals. Colony PCR products amplified from the region near the first ATG with primers KT2125-2126 were separated by 0.8% agarose gel electrophoresis to check the insertion and subjected to sequencing.</p><p id="p-16">For genome editing with a single vector expressing both a gRNA and the Cas9 protein, the cells cultured in EMM5S without thiamine medium were subjected to transformation. Transformation was carried out as described above. To obtain a homogenous mutant population, the transformants were streaked on EMM-selective medium plates (without thiamine) and incubated at 32° for 3 days to select the cells carrying the vector. The cells were streaked on YES plates at 32° for 2 days to get single colonies. To confirm the loss of vector, single colonies were streaked on YES and EMM-selective medium plates, followed by colony PCR and sequencing as described above.</p></div><div id="sec-7" class="subsection"><h3>Immunoblot analysis</h3><p id="p-17">The protein extracts were prepared using alkaline-TCA method as previously described (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Knop <em>et al.</em> 1999</a>). 1 × 10<sup>7</sup> cells growing in early log phase were harvested and used to prepare the cell extracts followed by immunoblot analysis. The 3 × Flag tagged Cas9 protein and the 2 × Flag tagged Stn1 were detected using monoclonal anti-Flag M2 antibody conjugated with HRP (Sigma, A8592). Anti-Cdc2 (Santa Cruz, SASC53) was used as a loading control.</p></div><div id="sec-8" class="subsection"><h3>Microscopy</h3><p id="p-18">The preparation of living cells expressing GFP fused proteins was carried out as described previously (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Hayashi <em>et al.</em> 2009a</a>). The cells were cultured in EMM5S liquid medium at 25°, and mounted on a glass slide. The cover glass was sealed with silicon grease (Dow corning, HVG-50) to observe the living cell by fluorescence microscopy. A BX51 microscope and DP71 CCD camera (Olympus) were used for fluorescence image acquisition and analysis.</p></div><div id="sec-9" class="subsection"><h3>Data availability</h3><p id="p-19">The plasmids generated in this study (pAH233, pAH235, pAH237, and pAH243) are available from Addgene (ID number 121436, 121437, 121438, and 121439) and National BioResource Project-Yeast (ID number FYP4232, FYP4233, FYP4234, and FYP4235, <a href="http://yeast.nig.ac.jp/yeast/top.xhtml">http://yeast.nig.ac.jp/yeast/top.xhtml</a>). Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.7685642">https://doi.org/10.25387/g3.7685642</a>.</p></div></div><div class="section results" id="sec-10"><h2 class="">Results</h2><div id="sec-11" class="subsection"><h3>The improved CRISPR/Cas9 system efficiently introduces mutations in fission yeast</h3><p id="p-20">To simplify genome editing using the CRISPR/Cas9 system in fission yeast, we modified the gRNA expression vector described in <a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">Jacobs <em>et al.</em> (2014)</a> (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A–D</a>). The vector pMZ283, which has the <em>rrk1</em> promoter, a gRNA scaffold, and a hammerhead ribozyme, is effective for expressing gRNA in fission yeast. However, the <em>Csp</em>CI cloning sites in the gRNA scaffold are digested inefficiently (<a id="xref-ref-28-3" class="xref-bibr" href="#ref-28">Rodríguez-López <em>et al.</em> 2017</a>), resulting in the difficulty of cloning gRNA target sequences. To solve this problem, we generated a vector that has <em>Bbs</em>I cloning sites for the gRNA target sequence (pAH233, <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1A</a>). <em>Bbs</em>I sites have often been used to clone gRNA target sequences in other organisms, and two <em>Bbs</em>I sites avoid self-ligation owing to different cleavage sequences (<a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Ran <em>et al.</em> 2013</a>; <a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Kondo and Ueda 2013</a>). The pAH233 vector has two <em>Bbs</em>I sites in the gRNA scaffold instead of <em>Csp</em>CI sites, which are sandwiched between 1.3 kb of the <em>rrk1</em> promoter/leader sequence and 1.0 kb of the <em>rrk1</em> terminator (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Wilhelm <em>et al.</em> 2008</a>). The annealed gRNA target oligonucleotides can be efficiently cloned at the <em>Bbs</em>I sites without phosphorylation.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/4/1153/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Construction of the gRNA and gRNA-Cas9 expression vectors. A. The gRNA expression vector (pAH233, LEU2 marker). Red rectangle: gRNA expression cassette. Green rectangle: hammerhead ribozyme sequence. B. The nmt41 promoter-driven Cas9 expression vector (pAH235, ura4+ marker). C. The gRNA-Cas9 expression vector (pAH237, LEU2 marker). D. The gRNA-Cas9 expression vector (pAH243, ura4+ marker). Asterisks indicate silent mutations introduced by site-directed mutagenesis." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-567224537" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Construction of the gRNA and gRNA-Cas9 expression vectors. A. The gRNA expression vector (pAH233, &lt;em&gt;LEU2&lt;/em&gt; marker). Red rectangle: gRNA expression cassette. Green rectangle: hammerhead ribozyme sequence. B. The &lt;em&gt;nmt41&lt;/em&gt; promoter-driven Cas9 expression vector (pAH235, &lt;em&gt;ura4&lt;sup&gt;+&lt;/sup&gt;&lt;/em&gt; marker). C. The gRNA-Cas9 expression vector (pAH237, &lt;em&gt;LEU2&lt;/em&gt; marker). D. The gRNA-Cas9 expression vector (pAH243, &lt;em&gt;ura4&lt;sup&gt;+&lt;/sup&gt;&lt;/em&gt; marker). Asterisks indicate silent mutations introduced by site-directed mutagenesis.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/4/1153/F1.medium.gif" width="440" height="406"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/9/4/1153/F1.medium.gif" width="440" height="406"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/4/1153/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/4/1153/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492634" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-21" class="first-child">Construction of the gRNA and gRNA-Cas9 expression vectors. A. The gRNA expression vector (pAH233, <em>LEU2</em> marker). Red rectangle: gRNA expression cassette. Green rectangle: hammerhead ribozyme sequence. B. The <em>nmt41</em> promoter-driven Cas9 expression vector (pAH235, <em>ura4<sup>+</sup></em> marker). C. The gRNA-Cas9 expression vector (pAH237, <em>LEU2</em> marker). D. The gRNA-Cas9 expression vector (pAH243, <em>ura4<sup>+</sup></em> marker). Asterisks indicate silent mutations introduced by site-directed mutagenesis.</p><div class="sb-div caption-clear"></div></div></div><p id="p-22">We also generated an inducible Cas9 expression vector. Because the expression of the Cas9 protein impairs the cell growth in fission yeast and the proper development of <em>Drosophila</em> (<a id="xref-ref-12-4" class="xref-bibr" href="#ref-12">Jacobs <em>et al.</em> 2014</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Port <em>et al.</em> 2014</a>), it is critical to regulate the amount of Cas9 protein in the cells. The humanized <em>Sp</em>Cas9 protein fused to the 3 × Flag tag and two NLSs (<a id="xref-ref-26-3" class="xref-bibr" href="#ref-26">Ran <em>et al.</em> 2013</a>) was cloned into pSLF273 (<em>nmt41</em> promoter, <em>ura4<sup>+</sup></em> marker) vector to be expressed under the control of the <em>nmt41</em> promoter (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1B</a>, pAH235). We further generated a vector expressing both of a gRNA and the Cas9 protein to modify the genome through a single transformation using one marker gene, either <em>LEU2</em> or <em>ura4<sup>+</sup></em> (pAH237: <em>LEU2</em> marker, pAH243: <em>ura4<sup>+</sup></em> marker, <a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1C</a> and <a id="xref-fig-1-5" class="xref-fig" href="#F1">1D</a>). To ensure Cas9 expression, we generated the vectors that expressed the Cas9 using the <em>nmt41</em> promoter on each marker gene’s vector. Immunoblot analysis was performed using anti-Flag antibodies to validate the expression of the Cas9 protein in the cells carrying pAH233 in the presence or absence of thiamine (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>). The result showed that the expression of the Cas9 protein by the <em>nmt41</em> promoter was induced in the absence of thiamine, whereas its expression was repressed in the presence of thiamine (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2A</a>). We also analyzed the expression of the Cas9 protein from pAH237 and pAH243 in the absence of thiamine and observe that the Cas9 protein was stably expressed by the <em>nmt41</em> promoter (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2B</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/4/1153/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Immunoblot of the Cas9 protein expressed by the nmt41 promoter. A. The Cas9 protein was inducibly expressed by the nmt41 promoter. Two independent transformants carrying pAH235 were cultured in EMM &#x2013; uracil + thiamine or EMM&#x2013;uracil&#x2013;thiamine medium and were prepared as protein samples for SDS-PAGE (three lanes on the left; +5 &#x3BC;M of thiamine (repressed condition), &#x2013; thiamine condition (three lanes on the right). The extract of the whole cell carrying empty vector (pSLF273) was loaded as a negative control (&#x2013;). Cas9 fused 3 &#xD7; Flag protein was detected by &#x3B1;-Flag antibody. Cdc2 (&#x3B1;-PSTAIR) was used as a loading control. B. Detection of Cas9 proteins expressed by the nmt41 promoter from the Cas9 and gRNA-Cas9 expression vectors. Two independent transformants carrying the indicated vectors were cultured in EMM-selective medium (without thiamine) to prepare the whole cell extracts followed by immunoblot analysis." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-567224537" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Immunoblot of the Cas9 protein expressed by the &lt;em&gt;nmt41&lt;/em&gt; promoter. A. The Cas9 protein was inducibly expressed by the &lt;em&gt;nmt41&lt;/em&gt; promoter. Two independent transformants carrying pAH235 were cultured in EMM &#x2013; uracil + thiamine or EMM&#x2013;uracil&#x2013;thiamine medium and were prepared as protein samples for SDS-PAGE (three lanes on the left; +5 &#x3BC;M of thiamine (repressed condition), &#x2013; thiamine condition (three lanes on the right). The extract of the whole cell carrying empty vector (pSLF273) was loaded as a negative control (&#x2013;). Cas9 fused 3 &#xD7; Flag protein was detected by &#x3B1;-Flag antibody. Cdc2 (&#x3B1;-PSTAIR) was used as a loading control. B. Detection of Cas9 proteins expressed by the &lt;em&gt;nmt41&lt;/em&gt; promoter from the Cas9 and gRNA-Cas9 expression vectors. Two independent transformants carrying the indicated vectors were cultured in EMM-selective medium (without thiamine) to prepare the whole cell extracts followed by immunoblot analysis.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/4/1153/F2.medium.gif" width="440" height="102"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/9/4/1153/F2.medium.gif" width="440" height="102"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/4/1153/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/4/1153/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492787" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-23" class="first-child">Immunoblot of the Cas9 protein expressed by the <em>nmt41</em> promoter. A. The Cas9 protein was inducibly expressed by the <em>nmt41</em> promoter. Two independent transformants carrying pAH235 were cultured in EMM – uracil + thiamine or EMM–uracil–thiamine medium and were prepared as protein samples for SDS-PAGE (three lanes on the left; +5 μM of thiamine (repressed condition), – thiamine condition (three lanes on the right). The extract of the whole cell carrying empty vector (pSLF273) was loaded as a negative control (–). Cas9 fused 3 × Flag protein was detected by α-Flag antibody. Cdc2 (α-PSTAIR) was used as a loading control. B. Detection of Cas9 proteins expressed by the <em>nmt41</em> promoter from the Cas9 and gRNA-Cas9 expression vectors. Two independent transformants carrying the indicated vectors were cultured in EMM-selective medium (without thiamine) to prepare the whole cell extracts followed by immunoblot analysis.</p><div class="sb-div caption-clear"></div></div></div><p id="p-24">To examine the efficiency of the improved CRISPR/Cas9 system, we edit the <em>ade6<sup>+</sup></em> gene. The <em>ade6</em> gRNA target sequence was designed near <em>ade6-M210</em> mutation site to evaluate the mutation efficiency based on the frequency of the appearance of red color colonies (<a id="xref-ref-12-5" class="xref-bibr" href="#ref-12">Jacobs <em>et al.</em> 2014</a>). At the beginning, we tested a combination of <em>ade6</em>-gRNA vector (<em>ade6</em>-gRNA-LEU2, pAH244) and inducible Cas9 vector (<em>nmt41p</em>-Cas9-<em>ura4<sup>+</sup></em>, pAH235). We sequentially transformed the wild type of <em>ade6<sup>+</sup></em> strain with the inducible Cas9 vector and then transformed it using the <em>ade6</em>-gRNA expression vector. Red colony color of the <em>ade6</em> mutant is because of the accumulation of red-colored intermediates in the adenine synthesis pathway. Obtained transformants were streaked on YE plates, a low adenine medium, to count the number of white and red/pink-colored colonies in order to confirm the introduction of an <em>ade6</em> mutation by the gRNA/Cas9 complex. The results revealed that 89% of transformants expressed <em>ade6</em>-gRNA and the Cas9 protein showed red/pink color, whereas all of the cells carrying the control gRNA vector and the Cas9 vector showed white color (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a> and Figure S1A). We sequenced the <em>ade6</em> gene in eight red-colored transformants and found 1–3 bp indels in the <em>ade6</em> gene (Table S4), which indicates that the CRISPR/Cas9 system efficiently induced an <em>ade6</em> mutation comparable with that of a previously reported system (<a id="xref-ref-12-6" class="xref-bibr" href="#ref-12">Jacobs <em>et al.</em> 2014</a>). We also transformed wild-type cells expressing the Cas9 protein with the <em>rrk1</em> promoter-<em>Csp</em>CI-<em>ade6</em> gRNA cassette that has previously been used (pMZ284, derived from pMZ283, Addgene ID 52225) and cloned them into a <em>LEU2</em> marker vector (pAH234; pRE-pMZ284) to validate our modified system. Both of plasmids could introduce the <em>ade6</em> mutagenesis at high efficiency (pAH244: 89%, pRE-pMZ284: 97%, <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>), however, the transformants carrying pAH235 and pRE-pMZ284 showed slow growth (Figure S2). Because both of the transformant cells carrying <em>ade6</em> gRNA plasmids showed any inhibition for cell growth (Figure S2D), it might be cause by the difference of the stability of Cas9/<em>ade6</em> gRNA complex in the cells. It is comparable with the efficiency of <em>ade6</em> mutagenesis (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/492650/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/492650/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Mutation frequency of <em>ade6<sup>+</sup></em> mutagenesis (two plasmids)</span><div class="sb-div caption-clear"></div></div></div><p id="p-25">Subsequently, we transformed the cells with the single vector expressing both the <em>ade6</em>-gRNA and the Cas9 protein. The wild-type cells were cultured in EMM5S without thiamine before the transformation. The cells were transformed with either the <em>LEU2</em>- or <em>ura4</em>-marked single vector and plated on EMM selective medium plates for 6–7 days. Transformants were streaked on YE plates to observe the colors of their colonies. The results revealed that the transformation with the <em>LEU2</em>-marked vector introduced the mutation at a high frequency (83%, <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>), whereas the transformation with the <em>ura4</em>-marked vector introduced the mutation at low frequency (13%, <a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>). The transformants were then streaked on YE or EMM selective medium plates to determine whether they lost the vector. Approximately 50% of transformants lost the vectors. The ratios of vector retention were following: <em>nmt41p-LEU2</em>: 11/33 (33%), <em>nmt41p-ura4</em><sup>+</sup>: 12/24 (50%).</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/492637/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/492637/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Mutation frequency of <em>ade6<sup>+</sup></em> mutagenesis by single plasmid</span><div class="sb-div caption-clear"></div></div></div><p id="p-26">We observed that transformants carrying one of the single vectors often exhibited a mosaic pattern of red and white cells (Figure S1B). To obtain homogenous cells, the transformants were streaked on EMM selective medium plates and incubated for 3 days to select the cells carrying the vectors. The cells carrying the vectors were then streaked on YE medium plates to obtain single colonies. The results revealed that all colonies exhibited the homogenous color (red color, Figure S1C). Comparing incubation on EMM-selective medium plate to incubation with no selective step, the frequency of the <em>ade6</em> mutagenesis was increased (<em>nmt41p-LEU2</em>: 7/8 [88%], <em>nmt41p-ura4</em><sup>+</sup>: 5/8 [63%]). The result indicates that the cells that lose the gRNA/Cas9 vector promptly fail to express sufficient Cas9 protein to achieve genome editing. Therefore, it appears that extending the duration of Cas9 expression promotes donor-less mutation induction.</p></div><div id="sec-12" class="subsection"><h3>Introducing point mutations using a combination of the short-homology-mediated repair pathway and the CRISPR/Cas9 system</h3><p id="p-27">The MMEJ-mediated knock-in has been reported in both TALEN and the CRISPR/Cas9 programmed systems in varied organisms, including frog and silkworm as well as in mammalian tissue culture (<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Nakade <em>et al.</em> 2014</a>). It is also reported that short-homology-mediated knock-in could introduce the precise editing in zebrafish (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Hisano <em>et al.</em> 2015</a>), suggesting that the short-homology-mediated repair pathway could be a useful pathway for genome editing in fission yeast. To validate the possibility and optimize the length of homology sequences for genome editing, we attempted to introduce point mutations to appropriate regions of chromosome region through co-transformation of variable length of oligonucleotides and the gRNA-Cas9 vectors. We first designed a gRNA to target the <em>swi6</em><sup>+</sup> gene, which encodes an ortholog of heterochromatin protein 1. Swi6 binds to H3K9me via its chromodomain to form the heterochromatin (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Nakayama <em>et al.</em> 2001</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Jacobs and Khorasanizadeh 2002</a>). Swi6-W104 is located in the chromodomain (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>), and Swi6-W104A mutant protein fails to interact with H3K9me, resulting in a silencing defect at <em>mat</em> locus (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Hayashi <em>et al.</em> 2009b</a>). We transformed cells carrying the Cas9 expression vector with the <em>swi6</em> gRNA expression vector. Since <em>swi6Δ</em> mutant shows the sensitivity to TBZ (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Keller <em>et al.</em> 2012</a>), which is a microtubule-depolymerizing drug, the transformants were streaked on both a YES plate and a YES plate containing 15 μg/mL of TBZ to test whether the <em>swi6</em> gRNA/Cas9 complex could introduce the mutation in the <em>swi6</em> gene (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3E</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/4/1153/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="swi6-W104A chromodomain editing by the short-homology-mediated genome editing. A. Diagram of Swi6 protein feature. CD: chromodomain, CSD: chromoshadow domain that is required for dimer formation. B. Sequences of the swi6 gene and the gRNA target DNA for introducing the swi6-W104A mutation. The PAM sequence is framed with a gray square, and the gRNA target sequence is indicated by a black arrow. The cleavage site is indicated by a black arrowhead. The gray letters (TG/AC) in the sequence indicate the nucleotides that will be substituted to GC/CG by editing (change the codon of tryptophan at 104th to alanine). C. Design of the donor oligonucleotides for short-homology-mediated mutagenesis. The mutation site (x) is located at +2 bp position from the cleavage site (closed triangle, at &#x2013;3 bp position from PAM) digested by gRNA/Cas9 complex. The oligonucleotides are designed to contain the bidirectional 25 bp sequences from the cleavage site with the base substitution mutation (x) as indicated. D. The introduced frequency of the swi6-W104A mutation by the different lengths of dsOligos. The Cas9 expressing cells were co-transformed with both of the oligonucleotides and swi6 gRNA #2 vector that possesses the target sequence near swi6-W104. TBZ-sensitive clones were analyzed to determine the mutation site by sequencing. As a control donor for the HR pathway, 575 bp length PCR product containing swi6-W104A mutation was co-transformed with the swi6 gRNA #2 vector and also the swi6 gRNA #5 vector that was designed to digest at +51 bp position from swi6-W104. E. The swi6-W104A mutant exhibits TBZ sensitivity. Fivefold-diluted cultures of the indicated strains were plated onto YES non-selective medium (N/S) and YES containing 15 &#x3BC;g/mL of TBZ and incubated for 2&#x2013;3 days at 32 &#xB0;C. F. The swi6-W104A mutagenesis by ssOligoFw, ssOligoRv, and mixed ssOligos. The number indicates the ratio of the number of the swi6-W104A mutants/total analyzed TBZ-sensitive clones." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-567224537" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;em&gt;swi6-W104A&lt;/em&gt; chromodomain editing by the short-homology-mediated genome editing. A. Diagram of Swi6 protein feature. CD: chromodomain, CSD: chromoshadow domain that is required for dimer formation. B. Sequences of the &lt;em&gt;swi6&lt;/em&gt; gene and the gRNA target DNA for introducing the &lt;em&gt;swi6-W104A&lt;/em&gt; mutation. The PAM sequence is framed with a gray square, and the gRNA target sequence is indicated by a black arrow. The cleavage site is indicated by a black arrowhead. The gray letters (TG/AC) in the sequence indicate the nucleotides that will be substituted to GC/CG by editing (change the codon of tryptophan at 104&lt;sup&gt;th&lt;/sup&gt; to alanine). C. Design of the donor oligonucleotides for short-homology-mediated mutagenesis. The mutation site (x) is located at +2 bp position from the cleavage site (closed triangle, at &#x2013;3 bp position from PAM) digested by gRNA/Cas9 complex. The oligonucleotides are designed to contain the bidirectional 25 bp sequences from the cleavage site with the base substitution mutation (x) as indicated. D. The introduced frequency of the &lt;em&gt;swi6-W104A&lt;/em&gt; mutation by the different lengths of dsOligos. The Cas9 expressing cells were co-transformed with both of the oligonucleotides and &lt;em&gt;swi6&lt;/em&gt; gRNA #2 vector that possesses the target sequence near &lt;em&gt;swi6-W104&lt;/em&gt;. TBZ-sensitive clones were analyzed to determine the mutation site by sequencing. As a control donor for the HR pathway, 575 bp length PCR product containing &lt;em&gt;swi6-W104A&lt;/em&gt; mutation was co-transformed with the &lt;em&gt;swi6&lt;/em&gt; gRNA #2 vector and also the &lt;em&gt;swi6&lt;/em&gt; gRNA #5 vector that was designed to digest at +51 bp position from &lt;em&gt;swi6-W104&lt;/em&gt;. E. The &lt;em&gt;swi6-W104A&lt;/em&gt; mutant exhibits TBZ sensitivity. Fivefold-diluted cultures of the indicated strains were plated onto YES non-selective medium (N/S) and YES containing 15 &#x3BC;g/mL of TBZ and incubated for 2&#x2013;3 days at 32 &#xB0;C. F. The &lt;em&gt;swi6-W104A&lt;/em&gt; mutagenesis by ssOligoFw, ssOligoRv, and mixed ssOligos. The number indicates the ratio of the number of the &lt;em&gt;swi6-W104A&lt;/em&gt; mutants/total analyzed TBZ-sensitive clones.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/4/1153/F3.medium.gif" width="440" height="209"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/9/4/1153/F3.medium.gif" width="440" height="209"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/4/1153/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/4/1153/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492682" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-28" class="first-child"><em>swi6-W104A</em> chromodomain editing by the short-homology-mediated genome editing. A. Diagram of Swi6 protein feature. CD: chromodomain, CSD: chromoshadow domain that is required for dimer formation. B. Sequences of the <em>swi6</em> gene and the gRNA target DNA for introducing the <em>swi6-W104A</em> mutation. The PAM sequence is framed with a gray square, and the gRNA target sequence is indicated by a black arrow. The cleavage site is indicated by a black arrowhead. The gray letters (TG/AC) in the sequence indicate the nucleotides that will be substituted to GC/CG by editing (change the codon of tryptophan at 104<sup>th</sup> to alanine). C. Design of the donor oligonucleotides for short-homology-mediated mutagenesis. The mutation site (x) is located at +2 bp position from the cleavage site (closed triangle, at –3 bp position from PAM) digested by gRNA/Cas9 complex. The oligonucleotides are designed to contain the bidirectional 25 bp sequences from the cleavage site with the base substitution mutation (x) as indicated. D. The introduced frequency of the <em>swi6-W104A</em> mutation by the different lengths of dsOligos. The Cas9 expressing cells were co-transformed with both of the oligonucleotides and <em>swi6</em> gRNA #2 vector that possesses the target sequence near <em>swi6-W104</em>. TBZ-sensitive clones were analyzed to determine the mutation site by sequencing. As a control donor for the HR pathway, 575 bp length PCR product containing <em>swi6-W104A</em> mutation was co-transformed with the <em>swi6</em> gRNA #2 vector and also the <em>swi6</em> gRNA #5 vector that was designed to digest at +51 bp position from <em>swi6-W104</em>. E. The <em>swi6-W104A</em> mutant exhibits TBZ sensitivity. Fivefold-diluted cultures of the indicated strains were plated onto YES non-selective medium (N/S) and YES containing 15 μg/mL of TBZ and incubated for 2–3 days at 32 °C. F. The <em>swi6-W104A</em> mutagenesis by ssOligoFw, ssOligoRv, and mixed ssOligos. The number indicates the ratio of the number of the <em>swi6-W104A</em> mutants/total analyzed TBZ-sensitive clones.</p><div class="sb-div caption-clear"></div></div></div><p id="p-29">According to the results, 54% of transformants exhibited TBZ sensitivity (14/26 transformants), indicating that the <em>swi6</em> gRNA/Cas9 complex had appropriately targeted endonuclease activity. To optimize the short-homology length for mutagenesis, we transformed the cells with both of <em>swi6</em> gRNA expression vectors and oligonucleotides containing different homologous length sequences near the <em>swi6-W104A</em> allele (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3B</a> and <a id="xref-fig-3-4" class="xref-fig" href="#F3">3C</a>). We designed the donor oligonucleotides that had ∼15 bp, ∼20 bp, or ∼25 bp of sequences on either side of the cleavage site (represented as cleavage site 0 in <a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3C</a>, –3 bp from protospacer adjacent motif (PAM)). We co-transformed the Cas9 expressing cells with both of the annealed donor oligonucleotides and the <em>swi6</em>-gRNA vectors to introduce the <em>swi6-W104A</em> mutation. As a control, a 575 bp PCR fragment containing <em>swi6W-104A</em> allele was used as a donor for the mutagenesis using the HR pathway.</p><p id="p-30">TBZ-sensitive colonies were used in direct colony PCR to amplify the <em>swi6-W104</em> region followed by sequencing to determine the mutation site. The results showed that 15 bp homologous oligonucleotides introduced a <em>swi6-W104A</em> mutation at 10% frequency (1/10, <a id="xref-fig-3-6" class="xref-fig" href="#F3">Figure 3D</a> and Table S4). However, most of the mutations introduced by the 15 bp homologous oligonucleotides were indel mutations, and some of mutants had rearrangements near the cleavage site (Table S4). Notably, 20 bp and 25 bp homologous oligonucleotides introduced <em>swi6-W104A</em> mutations at 70% (7/10) and 100% (8/8) of the frequency, respectively (<a id="xref-fig-3-7" class="xref-fig" href="#F3">Figure 3D</a>). As expected, PCR fragments containing <em>swi6-W104A</em> mutation could introduce an appropriate mutation into the chromosome at 100% (16/16) frequency. Therefore, even using very short oligonucleotides, the short-homology-mediated genome editing can introduce mutations at high frequency, which is comparable with the HR pathway. We also designed another <em>swi6</em> gRNA vector (+51 bp far from <em>swi6-W104</em> position, #5 in <a id="xref-fig-3-8" class="xref-fig" href="#F3">Figure 3D</a>) and tested the frequency of introduction of <em>swi6-W104A</em> mutation by the HR pathway using same PCR products (<a id="xref-fig-3-9" class="xref-fig" href="#F3">Figure 3D</a>). However, the mutation frequency was not high when the position of the DSB is relatively distant from the mutation site. Only 1 out of 14 transformants had the <em>swi6-W104A</em> mutation (7%). The result highlights the significance of the positions of the DSBs digested by the gRNA/Cas9 complex in the introduction of high-frequency mutations.</p><p id="p-31">Single-stranded oligonucleotides have also been reported to be effective in introducing mutations in other organisms (<a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">DiCarlo <em>et al.</em> 2013</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Engstrom <em>et al.</em> 2009</a>). We subsequently investigated the frequency of the mutagenesis by transformation of single-stranded oligonucleotides in fission yeast. We tested the ssOligoFw (sense strand), ssOligoRv (antisense strand), the mixed complementary ssOligos (ssOligoFw and ssOligoRv), and the annealed oligo (double-stranded Oligo (dsOligo)), which have 20 bp or 25 bp homologous sequences on either side of the cleavage site. The sequencing analysis revealed that both of ssOligoFw and ssOligoRv could introduce the <em>swi6-W104A</em> mutation (<a id="xref-fig-3-10" class="xref-fig" href="#F3">Figure 3F</a> and Table S4). The ssOligoFw introduced the mutation at higher frequency than the ssOligoRv, implying a difference in repair pathways occurred within the sense and antisense strands. Remarkably, the mixed oligos and dsOligo could highly introduce the <em>swi6-W104A</em> mutation at 90% and 100% frequency (<a id="xref-fig-3-11" class="xref-fig" href="#F3">Figure 3F</a> and Table S4).</p><p id="p-32">To investigate the efficiency of the short-homology-mediated knock-in at other loci of the chromosomes, <em>mrc1<sup>+</sup></em> gene was mutagenized by oligonucleotides using a similar procedure. The <em>mrc1<sup>+</sup></em> gene encodes mediator of replication checkpoint protein 1, which is required for Rad3-dependent activation of checkpoint kinase Cds1 in response to replication fork arrest (<a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Tanaka and Russell 2001</a>) (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A</a>). Mrc1-S604 is located in SQ repeats, which are potential substrates of Rad3/Tel1 kinase, and the <em>mrc1-S604A</em> mutant exhibited HU sensitivity due to reduced Cds1-T11 phosphorylation, which is required for the interaction with Mrc1 (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Xu <em>et al.</em> 2006</a>). We generated an <em>mrc1</em> gRNA expression vector and transformed cells carrying the Cas9 vector with the gRNA expression vector and the oligonucleotides possessing 25 bp of homologous sequences adjacent to <em>mrc1-S604</em> including the mutation (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4B</a>). The transformants were streaked on YES plates containing 5 mM HU to estimate the frequency of the introduced mutation in <em>mrc1</em> gene (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4D</a>). The frequency of the transformants exhibiting HU sensitivity was very high (96% of transformants (23/24) exhibited HU sensitivity). Sequence analysis was performed using colony PCR products to determine the mutation sites of the HU-sensitive transformants. The results showed that the frequency was as high as that for the <em>swi6</em> mutagenesis: 60% for ssOligoFw, 20% for ssOligoRv, 90% for mixed ssOligos, and 100% for dsOligo (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4C</a> and Table S4). We also introduced the <em>mrc1-S604A</em> mutation by transformation with <em>mrc1</em> gRNA-Cas9 single plasmids (pAH262 (<em>LEU2</em> marker), pAH263 (<em>ura4<sup>+</sup></em> marker)) and dsOligo, and obtained the high frequency of the knock-in (70% (7/10, <em>LEU2</em>), 67% (8/12, <em>ura4<sup>+</sup></em>), <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Tables 3</a> and S4). There is not significant difference between the knock-in efficiency that introduced by both plasmids. These results suggest the effectiveness of the short-homology-mediated genome editing method in fission yeast.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/4/1153/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Introduction of the mrc1-S604A mutation by short-homology-mediated genome editing. A. Diagram of Mrc1 protein feature. DB: DNA binding domain, SQ/TQ: potential phosphorylation sites by Rad3/Tel1 kinase, phosphodegron: degradation domain by ubiquitin-proteasome. B. Sequences of the mrc1 gene and the target gRNA sequence for introduction of the mrc1-S604A mutation. The PAM sequence is framed with a gray square, and the gRNA target sequence is indicated by a black arrow. The cleavage site is indicated by a black arrowhead. The gray letters (T/A) in the sequence indicate the nucleotides that will be substituted to G/C by editing (change the codon of serine at 604th to alanine). C. The introduced frequency of the mrc1-S604A mutation by ssOligoFw, ssOligoRv, mixed ssOligos (mixed), and annealed ssOligos (ds). The number indicates the ratio of the mrc1-S604A mutants/total analyzed HU-sensitive clones. D. The mrc1-S604A mutant exhibits HU sensitivity. Fivefold-diluted cultures of indicated strains were plated onto YES non-selective medium (N/S) and YES containing 5 mM of HU and incubated for 2&#x2013;3 days at 32 &#xB0;C." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-567224537" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Introduction of the &lt;em&gt;mrc1-S604A&lt;/em&gt; mutation by short-homology-mediated genome editing. A. Diagram of Mrc1 protein feature. DB: DNA binding domain, SQ/TQ: potential phosphorylation sites by Rad3/Tel1 kinase, phosphodegron: degradation domain by ubiquitin-proteasome. B. Sequences of the &lt;em&gt;mrc1&lt;/em&gt; gene and the target gRNA sequence for introduction of the &lt;em&gt;mrc1-S604A&lt;/em&gt; mutation. The PAM sequence is framed with a gray square, and the gRNA target sequence is indicated by a black arrow. The cleavage site is indicated by a black arrowhead. The gray letters (T/A) in the sequence indicate the nucleotides that will be substituted to G/C by editing (change the codon of serine at 604&lt;sup&gt;th&lt;/sup&gt; to alanine). C. The introduced frequency of the &lt;em&gt;mrc1-S604A&lt;/em&gt; mutation by ssOligoFw, ssOligoRv, mixed ssOligos (mixed), and annealed ssOligos (ds). The number indicates the ratio of the &lt;em&gt;mrc1-S604A&lt;/em&gt; mutants/total analyzed HU-sensitive clones. D. The &lt;em&gt;mrc1-S604A&lt;/em&gt; mutant exhibits HU sensitivity. Fivefold-diluted cultures of indicated strains were plated onto YES non-selective medium (N/S) and YES containing 5 mM of HU and incubated for 2&#x2013;3 days at 32 &#xB0;C.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/4/1153/F4.medium.gif" width="404" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/9/4/1153/F4.medium.gif" width="404" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/4/1153/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/4/1153/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492671" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-33" class="first-child">Introduction of the <em>mrc1-S604A</em> mutation by short-homology-mediated genome editing. A. Diagram of Mrc1 protein feature. DB: DNA binding domain, SQ/TQ: potential phosphorylation sites by Rad3/Tel1 kinase, phosphodegron: degradation domain by ubiquitin-proteasome. B. Sequences of the <em>mrc1</em> gene and the target gRNA sequence for introduction of the <em>mrc1-S604A</em> mutation. The PAM sequence is framed with a gray square, and the gRNA target sequence is indicated by a black arrow. The cleavage site is indicated by a black arrowhead. The gray letters (T/A) in the sequence indicate the nucleotides that will be substituted to G/C by editing (change the codon of serine at 604<sup>th</sup> to alanine). C. The introduced frequency of the <em>mrc1-S604A</em> mutation by ssOligoFw, ssOligoRv, mixed ssOligos (mixed), and annealed ssOligos (ds). The number indicates the ratio of the <em>mrc1-S604A</em> mutants/total analyzed HU-sensitive clones. D. The <em>mrc1-S604A</em> mutant exhibits HU sensitivity. Fivefold-diluted cultures of indicated strains were plated onto YES non-selective medium (N/S) and YES containing 5 mM of HU and incubated for 2–3 days at 32 °C.</p><div class="sb-div caption-clear"></div></div></div><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/492681/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/492681/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Knock-in frequency of <em>mrc1-S604A</em> by single plasmid</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-13" class="subsection"><h3>The short-homology-mediated knock-in at the 5′ end of the gene combined the CRISPR/Cas9 system</h3><p id="p-34">In organisms such as silkworm and frog, the MMEJ-mediated integration of fluorescent marker genes with TALEN and CRISPR/Cas9 system has been successful (<a id="xref-ref-22-3" class="xref-bibr" href="#ref-22">Nakade <em>et al.</em> 2014</a>). Therefore, we attempted to generate knock-in yeast cells using the short-homology-mediated CRISPR/Cas9 system. Stn1 protein is required for telomere protection and maintenance (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Martin <em>et al.</em> 2007</a>). We performed knock-in at the 5′ end of the <em>stn1<sup>+</sup></em> gene by transformation with oligonucleotides encoding a 2 × Flag epitope tag sequence sandwiched between bidirectional 25 bp homologous sequences from the first ATG of the <em>stn1<sup>+</sup></em> gene (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5A</a>). The gRNA target sequence was selected from the sequence adjacent to the first ATG of the <em>stn1<sup>+</sup></em> gene, and the activity of the <em>stn1</em> gRNA/Cas9 complex was tested by transformation without the oligonucleotides. Sequence analysis revealed that 75% of transformants had 1–3 bp deletion (6/8, Table S4), suggesting that the <em>stn1</em>-gRNA/Cas9 complex has high activity that is adequate for the introduction of the mutations in the <em>stn1<sup>+</sup></em> gene. Cells carrying the Cas9 expressing vector were co-transformed with both the <em>stn1</em> gRNA vector and donor oligonucleotides: ssOligoFw, ssOligoRv, mixed ssOligos, and annealed dsOligo. Electrophoresis analysis of colony PCR products shown in <a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5B</a> revealed that mixed ssOligos and annealed dsOligo effectively introduced the insertion (88% and 96% frequency, respectively, <a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5C</a>). In contrast, ssOligoFw and ssOligoRv introduced the insertion but at low frequency (21% and 4%, respectively, <a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5C</a>). Sequence analysis showed that the frequency of in-frame knock-in by transformation with mixed ssOligos and annealed dsOligo were high (82% and 89%, respectively, <a id="xref-fig-5-5" class="xref-fig" href="#F5">Figure 5C</a>). Some of the knock-in strains contained the in-frame 3 × Flag insertion when the cells were transformed with the mixed ssOligos and dsOligo. Although the insertion frequency was low, ssOligoFw also introduced in-frame knock-in at high fidelity (80% frequency, <a id="xref-fig-5-6" class="xref-fig" href="#F5">Figure 5C</a>). We did not obtain the knock-in strain by the transformation with ssOligoRv (<a id="xref-fig-5-7" class="xref-fig" href="#F5">Figure 5C</a>). We also found that the frequency of in-frame Flag tag insertion without indel was high (89%) in the transformants obtained by transformation with dsOligo (<a id="xref-fig-5-8" class="xref-fig" href="#F5">Figure 5C</a> and Table S4). The results suggest that the transformation with dsOligo would lead to precise knock-in without indel when using the short-homology-mediated CRISPR/Cas9 system. We performed immunoblot analysis to confirm the expression of 2 × Flag-Stn1 protein. Flag epitope tag fused Stn1 protein was detected with anti-Flag antibody (<a id="xref-fig-5-9" class="xref-fig" href="#F5">Figure 5D</a>), indicating that Flag epitope sequence was efficiently and precisely introduced at the 5′ end of the <em>stn1<sup>+</sup></em> gene.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/4/1153/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Short-homology-mediated gene knock-in with CRISPR/Cas9 system. A. 2 &#xD7; Flag knock-in at N terminal Stn1. Diagram of 2 &#xD7; Flag tagged Stn1 protein and the design of oligonucleotides for the 5&#x2032; end of the stn1+ gene knock-in. The cleavage site is located at &#x2013;3 bp position from first ATG. ATG + 2 &#xD7; Flag repeat sequence is integrated by replacing 3 bp upstream sequence from first ATG. B. Electrophoresis of colony PCR products amplified the region at the 5&#x2032; end of the stn1+. Transformed cells with annealed oligos were used as templates. The 290 bp length of PCR product was used as a control (the bottom of black arrow, wt). The fragment containing 2 &#xD7; Flag and 388 bp length of PCR product was detected (a black arrow at the top). White arrowhead indicates 3 &#xD7; Flag inserted fragment. Asterisk indicates the fragment contained a part of the carrier DNA (salmon testes) inserted at the 5&#x2032; end of the stn1+. C. The frequency of 2 &#xD7; Flag tagged stn1+ by single stranded oligonucleotides (ssFw, ssRv) and mixed ssFw and ssRv, and annealed ssFw and ssRv oligonucleotides (ds). Twenty-four independent transformants were analyzed to detect the insertion by electrophoresis. The ratio of rearrangement was indicated in the left column in the table. The denominator described at the column of in-frame without indel indicates the numbers of cells that were analyzed by sequencing. D. Immunoblot analysis of the 2 &#xD7; Flag tagged Stn1 protein. Two independent 2 &#xD7; Flag tagged strains were cultured to detect the expression of Stn1 protein with anti-Flag antibody. No tagged strain was loaded as a negative control (&#x2013;). E. Diagram of GFP tagged Reb1 protein and the design of PCR primers for tagging. Fw primer contains 5&#x2032;-UTR of reb1+ and the N-terminal GFP sequence. Rv primer contains the N terminal of reb1+ gene and C terminal GFP sequence without stop codon. The cleavage site is located at &#x2013;2 bp from first ATG. F. Electrophoresis of colony PCR products at the 5&#x2032; end of reb1+. The 563 bp length of PCR product is used as a control (the bottom of black arrowhead, wt). A black arrow indicates the fragment containing GFP gene (1,326 bp). Asterisks indicate the fragments containing the rearranged sequence at the point of the 5&#x2032; end of reb1+ gene. G. The frequency of GFP fused Reb1 expressing cells obtained by genome editing. Heat-denatured PCR product was used for transformation as a single-stranded DNA donor. H. The cellular localization of GFP fused Reb1 protein. Scale bar indicates 2.5 &#x3BC;m." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-567224537" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Short-homology-mediated gene knock-in with CRISPR/Cas9 system. A. 2 &#xD7; Flag knock-in at N terminal Stn1. Diagram of 2 &#xD7; Flag tagged Stn1 protein and the design of oligonucleotides for the 5&#x2032; end of the &lt;em&gt;stn1&lt;sup&gt;+&lt;/sup&gt;&lt;/em&gt; gene knock-in. The cleavage site is located at &#x2013;3 bp position from first ATG. ATG + 2 &#xD7; Flag repeat sequence is integrated by replacing 3 bp upstream sequence from first ATG. B. Electrophoresis of colony PCR products amplified the region at the 5&#x2032; end of the &lt;em&gt;stn1&lt;sup&gt;+&lt;/sup&gt;&lt;/em&gt;. Transformed cells with annealed oligos were used as templates. The 290 bp length of PCR product was used as a control (the bottom of black arrow, wt). The fragment containing 2 &#xD7; Flag and 388 bp length of PCR product was detected (a black arrow at the top). White arrowhead indicates 3 &#xD7; Flag inserted fragment. Asterisk indicates the fragment contained a part of the carrier DNA (salmon testes) inserted at the 5&#x2032; end of the &lt;em&gt;stn1&lt;/em&gt;&lt;sup&gt;+&lt;/sup&gt;. C. The frequency of 2 &#xD7; Flag tagged &lt;em&gt;stn1&lt;sup&gt;+&lt;/sup&gt;&lt;/em&gt; by single stranded oligonucleotides (ssFw, ssRv) and mixed ssFw and ssRv, and annealed ssFw and ssRv oligonucleotides (ds). Twenty-four independent transformants were analyzed to detect the insertion by electrophoresis. The ratio of rearrangement was indicated in the left column in the table. The denominator described at the column of in-frame without indel indicates the numbers of cells that were analyzed by sequencing. D. Immunoblot analysis of the 2 &#xD7; Flag tagged Stn1 protein. Two independent 2 &#xD7; Flag tagged strains were cultured to detect the expression of Stn1 protein with anti-Flag antibody. No tagged strain was loaded as a negative control (&#x2013;). E. Diagram of GFP tagged Reb1 protein and the design of PCR primers for tagging. Fw primer contains 5&#x2032;-UTR of &lt;em&gt;reb1&lt;sup&gt;+&lt;/sup&gt;&lt;/em&gt; and the N-terminal GFP sequence. Rv primer contains the N terminal of &lt;em&gt;reb1&lt;sup&gt;+&lt;/sup&gt;&lt;/em&gt; gene and C terminal GFP sequence without stop codon. The cleavage site is located at &#x2013;2 bp from first ATG. F. Electrophoresis of colony PCR products at the 5&#x2032; end of &lt;em&gt;reb1&lt;sup&gt;+&lt;/sup&gt;&lt;/em&gt;. The 563 bp length of PCR product is used as a control (the bottom of black arrowhead, wt). A black arrow indicates the fragment containing GFP gene (1,326 bp). Asterisks indicate the fragments containing the rearranged sequence at the point of the 5&#x2032; end of &lt;em&gt;reb1&lt;sup&gt;+&lt;/sup&gt;&lt;/em&gt; gene. G. The frequency of GFP fused Reb1 expressing cells obtained by genome editing. Heat-denatured PCR product was used for transformation as a single-stranded DNA donor. H. The cellular localization of GFP fused Reb1 protein. Scale bar indicates 2.5 &#x3BC;m.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/4/1153/F5.medium.gif" width="440" height="420"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/9/4/1153/F5.medium.gif" width="440" height="420"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/4/1153/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/4/1153/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/492711" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-35" class="first-child">Short-homology-mediated gene knock-in with CRISPR/Cas9 system. A. 2 × Flag knock-in at N terminal Stn1. Diagram of 2 × Flag tagged Stn1 protein and the design of oligonucleotides for the 5′ end of the <em>stn1<sup>+</sup></em> gene knock-in. The cleavage site is located at –3 bp position from first ATG. ATG + 2 × Flag repeat sequence is integrated by replacing 3 bp upstream sequence from first ATG. B. Electrophoresis of colony PCR products amplified the region at the 5′ end of the <em>stn1<sup>+</sup></em>. Transformed cells with annealed oligos were used as templates. The 290 bp length of PCR product was used as a control (the bottom of black arrow, wt). The fragment containing 2 × Flag and 388 bp length of PCR product was detected (a black arrow at the top). White arrowhead indicates 3 × Flag inserted fragment. Asterisk indicates the fragment contained a part of the carrier DNA (salmon testes) inserted at the 5′ end of the <em>stn1</em><sup>+</sup>. C. The frequency of 2 × Flag tagged <em>stn1<sup>+</sup></em> by single stranded oligonucleotides (ssFw, ssRv) and mixed ssFw and ssRv, and annealed ssFw and ssRv oligonucleotides (ds). Twenty-four independent transformants were analyzed to detect the insertion by electrophoresis. The ratio of rearrangement was indicated in the left column in the table. The denominator described at the column of in-frame without indel indicates the numbers of cells that were analyzed by sequencing. D. Immunoblot analysis of the 2 × Flag tagged Stn1 protein. Two independent 2 × Flag tagged strains were cultured to detect the expression of Stn1 protein with anti-Flag antibody. No tagged strain was loaded as a negative control (–). E. Diagram of GFP tagged Reb1 protein and the design of PCR primers for tagging. Fw primer contains 5′-UTR of <em>reb1<sup>+</sup></em> and the N-terminal GFP sequence. Rv primer contains the N terminal of <em>reb1<sup>+</sup></em> gene and C terminal GFP sequence without stop codon. The cleavage site is located at –2 bp from first ATG. F. Electrophoresis of colony PCR products at the 5′ end of <em>reb1<sup>+</sup></em>. The 563 bp length of PCR product is used as a control (the bottom of black arrowhead, wt). A black arrow indicates the fragment containing GFP gene (1,326 bp). Asterisks indicate the fragments containing the rearranged sequence at the point of the 5′ end of <em>reb1<sup>+</sup></em> gene. G. The frequency of GFP fused Reb1 expressing cells obtained by genome editing. Heat-denatured PCR product was used for transformation as a single-stranded DNA donor. H. The cellular localization of GFP fused Reb1 protein. Scale bar indicates 2.5 μm.</p><div class="sb-div caption-clear"></div></div></div><p id="p-36">We further attempted to perform GFP gene knock-in at the 5′ end of the <em>reb1</em><sup>+</sup> gene using short-homology-mediated knock-in. Reb1 is an rDNA-binding protein that is required for the termination of the transcription by RNA polymerase I (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Zhao <em>et al.</em> 1997</a>) (<a id="xref-fig-5-10" class="xref-fig" href="#F5">Figure 5E</a>). Reb1 fused to GFP at the C terminus is located in the nucleus and the nucleolus (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Hayashi <em>et al.</em> 2009a</a>). To generate a knock-in strain expressing GFP-fused Reb1 protein, we prepared a PCR fragment with GFP sandwiched between 24 bp upstream of the first ATG of the <em>reb1</em><sup>+</sup> gene and 24 bp downstream of the first ATG of the <em>reb1<sup>+</sup></em> gene (<a id="xref-fig-5-11" class="xref-fig" href="#F5">Figure 5E</a>). Endonuclease activity of the <em>reb1</em> gRNA/Cas9 complex was evaluated on the basis of the frequency of the mutations introduced in the <em>reb1<sup>+</sup></em> gene. We performed a sequence analysis of colony PCR products amplified near the first ATG of the <em>reb1<sup>+</sup></em> gene and found that transformants had mutations in the <em>reb1</em> gene at a 75% frequency (6/8, Table S4). The cells carrying the Cas9 vector were co-transformed with both of the <em>reb1</em> gRNA vector and PCR donor fragment, and transformants were analyzed as described above. The result showed that 7 out of 48 transformants had GFP gene insertion at a frequency of 15% (<a id="xref-fig-5-12" class="xref-fig" href="#F5">Figure 5F</a> and <a id="xref-fig-5-13" class="xref-fig" href="#F5">5G</a>). The heat-denatured PCR fragment (single-stranded PCR product) could induce GFP insertion, but the frequency was lower than that of the double-stranded PCR fragment (2/24, 8%, <a id="xref-fig-5-14" class="xref-fig" href="#F5">Figure 5G</a>). Microscopic observation revealed that N-terminal GFP fused Reb1 was localized in the nucleus and the nucleolus (<a id="xref-fig-5-15" class="xref-fig" href="#F5">Figure 5H</a>), as previously reported (<a id="xref-ref-8-3" class="xref-bibr" href="#ref-8">Hayashi <em>et al.</em> 2009a</a>). The frequency of GFP insertion was lower than that of the Flag knock-in with synthesized oligonucleotides, suggesting that the length of donor DNA used in the integration influenced the insertion efficiency.</p></div></div><div class="section discussion" id="sec-14"><h2 class="">Discussion</h2><p id="p-37">Here, we describe the implementation of the CRISPR/Cas9 system in fission yeast to analyze the gene function with less-artificial genome modification. We improved the gRNA expression vector for ease of target sequence cloning. The regulation of the Cas9 expression levels using inducible promoter could introduce genome editing within a desired time and reduce a risk of the introduction of other mutations into the chromosomes by repressing Cas9 expression. It has been reported that Cas9 protein overexpression is toxic to cell growth (<a id="xref-ref-12-7" class="xref-bibr" href="#ref-12">Jacobs <em>et al.</em> 2014</a>). We also generated the vectors that expressed the Cas9 protein using the <em>nmt1</em> promoter, which could highly induce the transcripts more than the <em>nmt41</em> promoter could. However, the vectors introduced genome modification at low efficiency (15/23 (65%): <em>nmt1p-LEU2</em>, 3/34 (9%): <em>nmt1p-ura4<sup>+</sup></em>, red colony/total transformants) and the retention of vectors in transformants was low (4/7 transformants (57%): <em>nmt1p-LEU2</em>, 0/8 transformants (0%): <em>nmt1p-ura4<sup>+</sup></em>). Immunoblot analysis results demonstrated that the Cas9 protein expressed by the <em>nmt1</em> promoter was less stable and degraded in some cases (Figure S3). The <em>ura4</em>-marked vector particularly seemed highly eliminated from the transformants. Cells carrying the <em>nmt1</em> promoter vector would survive following the removal of the vector or halting of growth on selective medium. Some of the cells could be obtained by the integration of <em>ura4</em><sup>+</sup> gene at any loci of the chromosomes to prevent the expression of the Cas9 protein. It would cause the low frequency of genome editing with the vector possessing the <em>nmt1</em> promoter. We also found that the cells expressing by <em>adh1</em> promoter showed the toxicity for cell growth and some cell lost the Cas9 expression but showed fast growth (Figures S4A and S4B). We demonstrated here that the Cas9 protein expressed by the <em>nmt41</em> promoter is enough to introduce the mutation and the regulation of the duration of the gRNA/Cas9 complex expression is critical for the introduction of genome editing.</p><p id="p-38">The short-homology-mediated genome editing enabled us to prepare genome-modified cells with less time-consuming steps. There are established protocols in yeast for generating the gene deletion strains and the chromosomally tagged strains with the antibiotic-resistant markers using common vectors like pFA6a and its derivatives (<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Bähler <em>et al.</em> 1998</a>). However, the insertion of the extra artificial sequences in some instances affects the authentic function of the gene of interest. Epitope tagging at the 3′ end of the essential genes sometimes fails because the tagged protein may lose or exhibit decreased activity. For non-essential genes, tagged alleles may also demonstrate weaker activity compared with wild-type proteins. Such effects may be due to lower levels of expression or lower mRNA stability when the fused gene uses non-self UTR sequences derived from the tagging vector. The CRISPR/Cas9 system enables the modification of the genome without introducing marker genes facilitated by the high efficiency of DSB formation at the desired chromosomal locus. In the present report, we combined the CRISPR/Cas9 system and the short-homology mediated repair pathway to perform genome editing with short oligonucleotides. In numerous organisms, point mutations and chromosomal tagging of genes are often generated via the HR pathway with a long DNA donors. Preparation of a donor DNA with long (200–500 bp) regions of homology requires serial PCR steps and, therefore, is thus time-consuming. Using long (∼100 bp) oligonucleotides for PCR can save the time required for preparation of a donor DNA. However, the cost of synthesizing long oligonucleotides is high, and the shorter homologous sequence lengths result in lower targeting efficiency. The short-homology-mediated genome editing method reported here could save time and costs by using short synthesized oligonucleotides as a donor DNA. We demonstrated that the noncanonical HDR pathway is highly effective for the introduction of point mutations into two genes using the CRISPR/Cas9 system with ∼50 bp of short oligonucleotides.</p><p id="p-39">As a donor DNA, double-stranded DNA seems improve the efficiency of the short-homology-mediated genome editing. The annealed <em>swi6-W104A</em> oligonucleotides possessing 25 bp homologous sequences demonstrated higher frequencies during the introduction of the <em>swi6-W104A</em> mutation (100%, <a id="xref-fig-3-12" class="xref-fig" href="#F3">Figure 3D</a>) than the ssOligos possessing 25 bp homologous sequences (ssOligoFw: 71%, ssOligoRv: 50%, <a id="xref-fig-3-13" class="xref-fig" href="#F3">Figure 3F</a>). The experiments that introduced the <em>mrc1-S604A</em> mutation showed that the dsOligo possessing 25 bp of homologous sequences edited the genome at a high frequency (100%), in contrast, ssOligos introduced the <em>mrc1-S604A</em> mutation at a 60% (ssOligoFw) and 20% (ssOligoRv) frequency (<a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4C</a>). For 2 × Flag knock-in at the 5′ end of the <em>stn1</em><sup>+</sup> gene, the dsOligo possessing 25 bp of homologous sequences at both ends was precisely inserted at a high frequency (89%, <a id="xref-fig-5-16" class="xref-fig" href="#F5">Figure 5C</a>), however, both ssOligos introduced an in-frame insertion at low frequency (ssOligoFw: 21%, ssOligoRv: 0%, <a id="xref-fig-5-17" class="xref-fig" href="#F5">Figure 5C</a>). These results suggest that the short-homology-mediated genome editing in fission yeast is related to the HR pathway, which takes place at G2 phase.</p><p id="p-40">The frequency of GFP tagging, however, is lower (15%, <a id="xref-fig-5-18" class="xref-fig" href="#F5">Figure 5G</a>) than the introducing frequency of the 2 × Flag knock-in at the 5′ end of the <em>stn1</em> tagging. The length of GFP gene is about 700 bp, however, which is much longer than the homologous sequences required for annealing to the target genomic loci (24 bp each). Compared with GFP fragment, the 2 × Flag the length of the 2 × Flag repeats is 54 bp, and the homologous sequences used for annealing were bidirectional 25 bp each. It highlights the significance of the efficiency that the both homology arms of oligonucleotide associate with target region of the chromosome. Although the molecular mechanism of the short-homology-mediated repair pathway requires further elucidation, the combination of the short-homology-directed repair and the CRISPR/Cas9 system in fission yeast could be carried out using the MMEJ, HR and the SSA pathways (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Decottignies 2007</a>).</p><p id="p-41">In addition to the association efficiency, the amount of a donor DNA transformed into a cell may influence the efficiency of knock-in at the target locus of the chromosome. GFP-<em>reb1</em> PCR product is 763 bp, and we did not obtain any insertion when we used 100–200 μg of PCR product for transformation. As for the HR introduced <em>swi6-W104A</em> mutation, 65–130 μg of PCR product (575 bp) was adequate for obtaining the mutant (<em>swi6</em> gRNA #2 vector, <a id="xref-fig-3-14" class="xref-fig" href="#F3">Figure 3D</a>). The observation could also be explained by the high association efficiency of donor DNA since <em>swi6-W104A</em> PCR product has over 250 bp homologous sequence at both ends.</p><p id="p-42">We also observed that the ssOligos could induce the genome editing, and the forward oligonucleotides exhibited higher frequency of precise genome editing than the reverse oligonucleotides (<a id="xref-fig-3-15" class="xref-fig" href="#F3">Figures 3F</a> and <a id="xref-fig-4-6" class="xref-fig" href="#F4">4C</a>). Notably, the mixed ssOligos could induce the genome editing at high frequencies comparable with dsOligo, although the efficiency of indel production was slightly higher (<a id="xref-fig-3-16" class="xref-fig" href="#F3">Figures 3F</a> and <a id="xref-fig-4-7" class="xref-fig" href="#F4">4C</a> and Table S4). The differences in the insertion and indel frequencies occasioned by the opposite directions of ssOligos could be caused by the functions of different repair/replication proteins at the homologous region due to the different orientations of Cas9 and gRNA binding (<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Engstrom <em>et al.</em> 2009</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Lemos <em>et al.</em> 2018</a>). We cannot exclude the possibility that the orientation of the transcripts could affect the difference of the editing frequency. It remains unclear how variably the repair/replication proteins act at short-homologous sequences near cleavage sites. As for the repetitive sequence insertion, we obtained 3 × Flag tagged <em>stn1</em> strain by transformation with both of mixed ssOligo and dsOligo (<a id="xref-fig-5-19" class="xref-fig" href="#F5">Figure 5B</a> and Table S4). The DNA replication machinery might generate an extra copy of Flag repeat sequence after the knock-in occurs in the chromosome.</p><p id="p-43">In this paper, we demonstrate that the short-homology-mediated genome editing combined with the CRISPR/Cas9 system is a powerful tool for generating point mutations and knock-in strains without a selective marker gene. It may also be possible to improve the current system using the Cas9 nickase to induce single-strand breaks, which may reduce off-target effects and introduce the genome modification at a position distant from the cleavage site (<a id="xref-ref-26-4" class="xref-bibr" href="#ref-26">Ran <em>et al.</em> 2013</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Satomura <em>et al.</em> 2017</a>). The TALEN-based genome editing system has a potential choice (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Gaj <em>et al.</em> 2013</a>) and a combination of TALEN- and short-homology-mediated genome editing could be a promising approach for modifying desired loci in any organism. We envision that this method would be a generally useful tool for genome editing and would facilitate the precise analysis of biological and molecular functions/mechanisms of genes in numerous organisms ranging from plants, animals, to humans in future studies.</p></div><div class="section ack" id="ack-1"><h2 class="">ACKNOWLEDGMENTS</h2><p id="p-44">We thank Drs. Nick Rhind, Jun-ichi Nakayama and Da-Qiao Ding for critical reading of the manuscript. We also thank Dr. Sinya Takahata for advice of the CRISPR/Cas9 system. pMZ283 and pMZ284 were gifts from Dr. Mikel Zaratiegui. pSpCas9(BB)-2A-Puro (PX459) V2.0 was a gift from Feng Zhang. A.H. was supported by a Grant for Women researcher in Kwansei Gakuin University. T.K. was supported by JSPS KAKENHI Grant Number JP16K15094 and JP15H04630.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-45">Supplemental material available at Figshare: <a href="https://doi.org/10.25387/g3.7685642">https://doi.org/10.25387/g3.7685642</a>.</p></li><li class="fn" id="fn-2"><p id="p-46"><em>Communicating editor: N. Rhind</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2018-12-15"><span class="received-label">Received </span>December 15, 2018.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2019-02-08"><span class="accepted-label">Accepted </span>February 8, 2019.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2019 Hayashi, Tanaka</li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1153.1" data-doi="10.1002/(SICI)1097-0061(199807)14:10&lt;943::AID-YEA292&gt;3.0.CO;2-Y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bähler</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">J. Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Longtine</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shah</span>  <span class="cit-name-given-names">N. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McKenzie</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Heterologous modules for efficient and versatile PCR-based gene targeting in <em>Schizosaccharomyces pombe</em>.</span> <abbr class="cit-jnl-abbrev">Yeast</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">943</span>–<span class="cit-lpage">951</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/(SICI)1097-0061(199807)14:10&lt;943::AID-YEA292&gt;3.0.CO;2-Y</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DYeast%2B%2528Chichester%252C%2BEngland%2529%26rft.stitle%253DYeast%26rft.aulast%253DB%25C3%25A4hler%26rft.auinit1%253DJ.%26rft.volume%253D14%26rft.issue%253D10%26rft.spage%253D943%26rft.epage%253D951%26rft.atitle%253DHeterologous%2Bmodules%2Bfor%2Befficient%2Band%2Bversatile%2BPCR-based%2Bgene%2Btargeting%2Bin%2BSchizosaccharomyces%2Bpombe.%26rft_id%253Dinfo%253Adoi%252F10.1002%252F%2528SICI%25291097-0061%2528199807%252914%253A10%253C943%253A%253AAID-YEA292%253E3.0.CO%253B2-Y%26rft_id%253Dinfo%253Apmid%252F9717240%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/(SICI)1097-0061(199807)14:10&lt;943::AID-YEA292&gt;3.0.CO;2-Y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9717240&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1153.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000075005100007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1153.2" data-doi="10.1534/genetics.107.071621"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Decottignies</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Microhomology-mediated end joining in fission yeast is repressed by pku70 and relies on genes involved in homologous recombination.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">176</span>: <span class="cit-fpage">1403</span>–<span class="cit-lpage">1415</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.107.071621</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.071621%26rft_id%253Dinfo%253Apmid%252F17483423%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3Ni8zLzE0MDMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzQvMTE1My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1153.3" data-doi="10.1093/nar/gkt135"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">DiCarlo</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Norville</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mali</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rios</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aach</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genome engineering in <em>Saccharomyces cerevisiae</em> using CRISPR-Cas systems.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">4336</span>–<span class="cit-lpage">4343</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkt135</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkt135%26rft_id%253Dinfo%253Apmid%252F23460208%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkt135&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23460208&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1153.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000318167900046&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1153.4" data-doi="10.1002/bies.200800119"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Engstrom</span>  <span class="cit-name-given-names">J. U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Suzuki</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kmiec</span>  <span class="cit-name-given-names">E. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Regulation of targeted gene repair by intrinsic cellular processes.</span> <abbr class="cit-jnl-abbrev">BioEssays</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">159</span>–<span class="cit-lpage">168</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/bies.200800119</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioEssays%2B%253A%2B%2Bnews%2Band%2Breviews%2Bin%2Bmolecular%252C%2Bcellular%2Band%2Bdevelopmental%2Bbiology%26rft.stitle%253DBioessays%26rft.aulast%253DEngstrom%26rft.auinit1%253DJ.%2BU.%26rft.volume%253D31%26rft.issue%253D2%26rft.spage%253D159%26rft.epage%253D168%26rft.atitle%253DRegulation%2Bof%2Btargeted%2Bgene%2Brepair%2Bby%2Bintrinsic%2Bcellular%2Bprocesses.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fbies.200800119%26rft_id%253Dinfo%253Apmid%252F19204988%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/bies.200800119&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19204988&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1153.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000263863600006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1153.5" data-doi="10.1002/yea.3178"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fernandez</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berro</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Use of a fluoride channel as a new selection marker for fission yeast plasmids and application to fast genome editing with CRISPR/Cas9.</span> <abbr class="cit-jnl-abbrev">Yeast</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">549</span>–<span class="cit-lpage">557</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/yea.3178</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DYeast%26rft.volume%253D33%26rft.spage%253D549%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fyea.3178%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/yea.3178&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1153.6" data-doi="10.1016/S0378-1119(97)00058-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Forsburg</span>  <span class="cit-name-given-names">S. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sherman</span>  <span class="cit-name-given-names">D. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">General purpose tagging vectors for fission yeast.</span> <abbr class="cit-jnl-abbrev">Gene</abbr> <span class="cit-vol">191</span>: <span class="cit-fpage">191</span>–<span class="cit-lpage">195</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/S0378-1119(97)00058-9</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGene%26rft.stitle%253DGene%26rft.aulast%253DForsburg%26rft.auinit1%253DS.%2BL.%26rft.volume%253D191%26rft.issue%253D2%26rft.spage%253D191%26rft.epage%253D195%26rft.atitle%253DGeneral%2Bpurpose%2Btagging%2Bvectors%2Bfor%2Bfission%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0378-1119%252897%252900058-9%26rft_id%253Dinfo%253Apmid%252F9218719%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0378-1119(97)00058-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9218719&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1153.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1997XG69400010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1153.7" data-doi="10.1016/j.tibtech.2013.04.004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gaj</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gersbach</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barbas</span>  <span class="cit-name-given-names">C. F.</span>  <span class="cit-name-suffix">III</span>.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">ZFN, TALEN and CRISPR/Cas based methods for genome engineering.</span> <abbr class="cit-jnl-abbrev">Trends Biotechnol.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">397</span>–<span class="cit-lpage">405</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.tibtech.2013.04.004</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BBiotechnol.%26rft.volume%253D31%26rft.spage%253D397%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tibtech.2013.04.004%26rft_id%253Dinfo%253Apmid%252F23664777%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tibtech.2013.04.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23664777&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1153.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000321175200004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1153.8" data-doi="10.1111/j.1365-2443.2008.01264.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hayashi</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ding</span>  <span class="cit-name-given-names">D.-Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tsutsumi</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chikashige</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Masuda</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span><span class="cit-comment">a</span> <span class="cit-article-title">Localization of gene products using a chromosomally tagged GFP-fusion library in the fission yeast <em>Schizosaccharomyces pombe</em>.</span> <abbr class="cit-jnl-abbrev">Genes Cells</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">217</span>–<span class="cit-lpage">225</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1365-2443.2008.01264.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2Bto%2BCells%26rft.stitle%253DGENES%2BCELLS%26rft.aulast%253DHayashi%26rft.auinit1%253DA.%26rft.volume%253D14%26rft.issue%253D2%26rft.spage%253D217%26rft.epage%253D225%26rft.atitle%253DLocalization%2Bof%2Bgene%2Bproducts%2Busing%2Ba%2Bchromosomally%2Btagged%2BGFP-fusion%2Blibrary%2Bin%2Bthe%2Bfission%2Byeast%2BSchizosaccharomyces%2Bpombe.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2443.2008.01264.x%26rft_id%253Dinfo%253Apmid%252F19170768%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-2443.2008.01264.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19170768&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1153.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1153.9" data-doi="10.1038/ncb1845"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hayashi</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Takahashi</span>  <span class="cit-name-given-names">T. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nakagawa</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nakayama</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Masukata</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span><span class="cit-comment">b</span> <span class="cit-article-title">The heterochromatin protein Swi6/HP1 activates replication origins at the pericentromeric region and silent mating-type locus.</span> <abbr class="cit-jnl-abbrev">Nat. Cell Biol.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">357</span>–<span class="cit-lpage">362</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ncb1845</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2BCell%2BBiology%26rft.stitle%253DNature%2BCell%2BBiology%26rft.aulast%253DHayashi%26rft.auinit1%253DM.%2BT.%26rft.volume%253D11%26rft.issue%253D3%26rft.spage%253D357%26rft.epage%253D362%26rft.atitle%253DThe%2Bheterochromatin%2Bprotein%2BSwi6%252FHP1%2Bactivates%2Breplication%2Borigins%2Bat%2Bthe%2Bpericentromeric%2Bregion%2Band%2Bsilent%2Bmating-type%2Blocus.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncb1845%26rft_id%253Dinfo%253Apmid%252F19182789%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncb1845&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19182789&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1153.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000263777900021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1153.10" data-doi="10.1002/dvg.1020130611"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hayden</span>  <span class="cit-name-given-names">M.S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Byers</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> Minimal extent of homology required for completion of meiotic recombination in <em>Saccharomyces cerevisiae. genesis</em>. 13: 498–514.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/dvg.1020130611</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopmental%2Bgenetics%26rft.stitle%253DDev%2BGenet%26rft.aulast%253DHayden%26rft.auinit1%253DM.%2BS.%26rft.volume%253D13%26rft.issue%253D6%26rft.spage%253D498%26rft.epage%253D514%26rft.atitle%253DMinimal%2Bextent%2Bof%2Bhomology%2Brequired%2Bfor%2Bcompletion%2Bof%2Bmeiotic%2Brecombination%2Bin%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fdvg.1020130611%26rft_id%253Dinfo%253Apmid%252F1304427%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/dvg.1020130611&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1304427&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1153.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1992KY12700010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1153.11" data-doi="10.1038/srep08841"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hisano</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sakuma</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nakade</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ohga</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ohta</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Precise in-frame integration of exogenous DNA mediated by CRISPR/Cas9 system in zebrafish.</span> <abbr class="cit-jnl-abbrev">Sci. Rep.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">8841</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/srep08841</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BRep.%26rft.volume%253D5%26rft.spage%253D8841%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsrep08841%26rft_id%253Dinfo%253Apmid%252F25740433%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/srep08841&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25740433&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1153.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1153.12" data-doi="10.1038/ncomms6344"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jacobs</span>  <span class="cit-name-given-names">J. Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ciccaglione</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tournier</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zaratiegui</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Implementation of the CRISPR-Cas9 system in fission yeast.</span> <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">5344</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ncomms6344</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun.%26rft.volume%253D5%26rft.spage%253D5344%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms6344%26rft_id%253Dinfo%253Apmid%252F25352017%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms6344&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25352017&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1153.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1153.13" data-doi="10.1126/science.1069473"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jacobs</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Khorasanizadeh</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Structure of HP1 chromodomain bound to a lysine 9-methylated Histone H3 tail.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">295</span>: <span class="cit-fpage">2080</span>–<span class="cit-lpage">2083</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.1069473</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1069473%26rft_id%253Dinfo%253Apmid%252F11859155%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyOTUvNTU2Mi8yMDgwIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS80LzExNTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1153.14" data-doi="10.1016/j.molcel.2012.05.009"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Keller</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Adaixo</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stunnenberg</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Woolcock</span>  <span class="cit-name-given-names">K. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hiller</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">HP1<sup>Swi6</sup> mediates the recognition and destruction of heterochromatic RNA transcripts.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">215</span>–<span class="cit-lpage">227</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.molcel.2012.05.009</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DKeller%26rft.auinit1%253DC.%26rft.volume%253D47%26rft.issue%253D2%26rft.spage%253D215%26rft.epage%253D227%26rft.atitle%253DHP1%2528Swi6%2529%2Bmediates%2Bthe%2Brecognition%2Band%2Bdestruction%2Bof%2Bheterochromatic%2BRNA%2Btranscripts.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2012.05.009%26rft_id%253Dinfo%253Apmid%252F22683269%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2012.05.009&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22683269&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1153.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000307084000009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1153.15" data-doi="10.1002/(SICI)1097-0061(199907)15:10B&lt;963::AID-YEA399&gt;3.0.CO;2-W"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Knop</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Siegers</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pereira</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zachariae</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Winsor</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Epitope tagging of yeast genes using a PCR-based strategy : more tags and improved practical routines.</span> <abbr class="cit-jnl-abbrev">Yeast</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">963</span>–<span class="cit-lpage">972</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/(SICI)1097-0061(199907)15:10B&lt;963::AID-YEA399&gt;3.0.CO;2-W</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DYeast%2B%2528Chichester%252C%2BEngland%2529%26rft.stitle%253DYeast%26rft.aulast%253DKnop%26rft.auinit1%253DM.%26rft.volume%253D15%26rft.issue%253D10B%26rft.spage%253D963%26rft.epage%253D972%26rft.atitle%253DEpitope%2Btagging%2Bof%2Byeast%2Bgenes%2Busing%2Ba%2BPCR-based%2Bstrategy%253A%2Bmore%2Btags%2Band%2Bimproved%2Bpractical%2Broutines.%26rft_id%253Dinfo%253Adoi%252F10.1002%252F%2528SICI%25291097-0061%2528199907%252915%253A10B%253C963%253A%253AAID-YEA399%253E3.0.CO%253B2-W%26rft_id%253Dinfo%253Apmid%252F10407276%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/(SICI)1097-0061(199907)15:10B&lt;963::AID-YEA399&gt;3.0.CO;2-W&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10407276&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1153.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000081644400007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1153.16" data-doi="10.1534/genetics.113.156737"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kondo</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ueda</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Highly improved gene targeting by germline-specific Cas9 expression in <em>Drosophila</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">195</span>: <span class="cit-fpage">715</span>–<span class="cit-lpage">721</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.113.156737</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.156737%26rft_id%253Dinfo%253Apmid%252F24002648%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk1LzMvNzE1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS80LzExNTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1153.17" data-doi="10.1073/pnas.1716855115"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lemos</span>  <span class="cit-name-given-names">B. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kaplan</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bae</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferrazzoli</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuo</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">CRISPR/Cas9 cleavages in budding yeast reveal templated insertions and strand-specific insertion/deletion profiles.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">115</span>: <span class="cit-fpage">E2040</span>–<span class="cit-lpage">E2047</span>. Erratum: E5431-E5433.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1716855115</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1716855115%26rft_id%253Dinfo%253Apmid%252F29440496%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTE1LzkvRTIwNDAiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzQvMTE1My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1153.18" data-doi="10.1073/pnas.0705497104"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Du</span>  <span class="cit-name-given-names">L.-L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rozenzhak</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Russell</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Protection of telomeres by a conserved Stn1-Ten1 Complex.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">104</span>: <span class="cit-fpage">14038</span>–<span class="cit-lpage">14043</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.0705497104</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0705497104%26rft_id%253Dinfo%253Apmid%252F17715303%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA0LzM1LzE0MDM4IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS80LzExNTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1153.19" data-doi="10.1016/0378-1119(93)90551-D"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Maundrell</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Thiamine-repressible expression vectors pREP and pRIP for fission yeast.</span> <abbr class="cit-jnl-abbrev">Gene</abbr> <span class="cit-vol">123</span>: <span class="cit-fpage">127</span>–<span class="cit-lpage">130</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/0378-1119(93)90551-D</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGene%26rft.stitle%253DGene%26rft.aulast%253DMaundrell%26rft.auinit1%253DK.%26rft.volume%253D123%26rft.issue%253D1%26rft.spage%253D127%26rft.epage%253D130%26rft.atitle%253DThiamine-repressible%2Bexpression%2Bvectors%2BpREP%2Band%2BpRIP%2Bfor%2Bfission%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0378-1119%252893%252990551-D%26rft_id%253Dinfo%253Apmid%252F8422996%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0378-1119(93)90551-D&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8422996&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1153.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1993KJ55300021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1153.20" data-doi="10.1016/j.tig.2008.08.007"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McVey</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">S. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">MMEJ repair of double-strand breaks (director’s cut): deleted sequences and alternative endings.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">529</span>–<span class="cit-lpage">538</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.tig.2008.08.007</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DMcVey%26rft.auinit1%253DM.%26rft.volume%253D24%26rft.issue%253D11%26rft.spage%253D529%26rft.epage%253D538%26rft.atitle%253DMMEJ%2Brepair%2Bof%2Bdouble-strand%2Bbreaks%2B%2528director%2527s%2Bcut%2529%253A%2Bdeleted%2Bsequences%2Band%2Balternative%2Bendings.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2008.08.007%26rft_id%253Dinfo%253Apmid%252F18809224%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tig.2008.08.007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18809224&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1153.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000261533700002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1153.21" data-doi="10.1016/0076-6879(91)94059-L"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Moreno</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klar</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nurse</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">Molecular genetic analysis of fission yeast <em>Schizosaccharomyces pombe</em>.</span> <abbr class="cit-jnl-abbrev">Methods Enzymol.</abbr> <span class="cit-vol">194</span>: <span class="cit-fpage">795</span>–<span class="cit-lpage">823</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/0076-6879(91)94059-L</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2Bin%2Benzymology%26rft.stitle%253DMethods%2BEnzymol%26rft.aulast%253DMoreno%26rft.auinit1%253DS.%26rft.volume%253D194%26rft.spage%253D795%26rft.epage%253D823%26rft.atitle%253DMolecular%2Bgenetic%2Banalysis%2Bof%2Bfission%2Byeast%2BSchizosaccharomyces%2Bpombe.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0076-6879%252891%252994059-L%26rft_id%253Dinfo%253Apmid%252F2005825%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0076-6879(91)94059-L&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2005825&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1153.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1991FN84200056&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1153.22" data-doi="10.1038/ncomms6560"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nakade</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tsubota</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sakane</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kume</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sakamoto</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Microhomology-mediated end-joining-dependent integration of donor DNA in cells and animals using TALENs and CRISPR/Cas9.</span> <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">5560</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ncomms6560</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun.%26rft.volume%253D5%26rft.spage%253D5560%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms6560%26rft_id%253Dinfo%253Apmid%252F25410609%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms6560&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25410609&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1153.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1153.23" data-doi="10.1126/science.1060118"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nakayama</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rice</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Strahl</span>  <span class="cit-name-given-names">B. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Allis</span>  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grewal</span>  <span class="cit-name-given-names">S. I. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Role of Histone H3 lysine 9 methylation in epigenetic control of heterochromatin assembly.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">292</span>: <span class="cit-fpage">110</span>–<span class="cit-lpage">113</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.1060118</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1060118%26rft_id%253Dinfo%253Apmid%252F11283354%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIyOTIvNTUxNC8xMTAiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzQvMTE1My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1153.24" data-doi="10.1073/pnas.1711979114"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Paix</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Folkmann</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goldman</span>  <span class="cit-name-given-names">D. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kulaga</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grzelak</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Precision genome editing using synthesis-dependent repair of Cas9-induced DNA breaks.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">114</span>: <span class="cit-fpage">E10745</span>–<span class="cit-lpage">E10754</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1711979114</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1711979114%26rft_id%253Dinfo%253Apmid%252F29183983%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMzoiMTE0LzUwL0UxMDc0NSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzkvNC8xMTUzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1153.25" data-doi="10.1073/pnas.1405500111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Port</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">H.-M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bullock</span>  <span class="cit-name-given-names">S. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Optimized CRISPR/Cas tools for efficient germline and somatic genome engineering in <em>Drosophila</em>.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">111</span>: <span class="cit-fpage">E2967</span>–<span class="cit-lpage">E2976</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1405500111</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1405500111%26rft_id%253Dinfo%253Apmid%252F25002478%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTExLzI5L0UyOTY3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS80LzExNTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1153.26" data-doi="10.1038/nprot.2013.143"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ran</span>  <span class="cit-name-given-names">F. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pd. Hsu</span>  <span class="cit-name-given-names">P. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Agarwala</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scott</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genome engineering using the CRISPR-Cas9 system.</span> <abbr class="cit-jnl-abbrev">Nat. Protoc.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">2281</span>–<span class="cit-lpage">2308</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nprot.2013.143</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BProtoc.%26rft.volume%253D8%26rft.spage%253D2281%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnprot.2013.143%26rft_id%253Dinfo%253Apmid%252F24157548%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nprot.2013.143&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24157548&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1153.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1153.27" data-doi="10.1007/s00412-017-0658-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ranjha</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Howard</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cejka</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Main steps in DNA double-strand break repair: an introduction to homologous recombination and related processes.</span> <abbr class="cit-jnl-abbrev">Chromosoma</abbr> <span class="cit-vol">127</span>: <span class="cit-fpage">187</span>–<span class="cit-lpage">214</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00412-017-0658-1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DChromosoma%26rft.volume%253D127%26rft.spage%253D187%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00412-017-0658-1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00412-017-0658-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1153.28" data-doi="10.12688/wellcomeopenres.10038.2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rodríguez-López</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cotobal</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernández-Sánchez</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Borbarán Bravo</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oktriani</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">A CRISPR/Cas9-based method and primer design tool for seamless genome editing in fission yeast</span>. <span class="cit-comment">[version 3; referees: 2 approved]</span> <abbr class="cit-jnl-abbrev">Wellcome Open Res.</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">19</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.12688/wellcomeopenres.10038.2</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DWellcome%2BOpen%2BRes.%26rft.volume%253D1%26rft.spage%253D19%26rft_id%253Dinfo%253Adoi%252F10.12688%252Fwellcomeopenres.10038.2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.12688/wellcomeopenres.10038.2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1153.29" data-doi="10.1128/MCB.4.11.2253"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rubnitz</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Subramani</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1984</span> <span class="cit-article-title">The minimum amount of homology required for homologous recombination in mammalian cells.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">2253</span>–<span class="cit-lpage">2258</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1128/MCB.4.11.2253</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DRubnitz%26rft.auinit1%253DJ%26rft.volume%253D4%26rft.issue%253D11%26rft.spage%253D2253%26rft.epage%253D2258%26rft.atitle%253DThe%2Bminimum%2Bamount%2Bof%2Bhomology%2Brequired%2Bfor%2Bhomologous%2Brecombination%2Bin%2Bmammalian%2Bcells.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.4.11.2253%26rft_id%253Dinfo%253Apmid%252F6096689%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjk6IjQvMTEvMjI1MyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzkvNC8xMTUzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1153.30" data-doi="10.1016/S0076-6879(10)70032-X"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sabatinos</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Forsburg</span>  <span class="cit-name-given-names">S. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Molecular genetics of <em>Schizosaccharomyces pombe</em>.</span> <abbr class="cit-jnl-abbrev">Methods Enzymol.</abbr> <span class="cit-vol">470</span>: <span class="cit-fpage">759</span>–<span class="cit-lpage">795</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/S0076-6879(10)70032-X</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2Bin%2Benzymology%26rft.stitle%253DMethods%2BEnzymol%26rft.aulast%253DSabatinos%26rft.auinit1%253DS.%2BA.%26rft.volume%253D470%26rft.spage%253D759%26rft.epage%253D795%26rft.atitle%253DMolecular%2Bgenetics%2Bof%2BSchizosaccharomyces%2Bpombe.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0076-6879%252810%252970032-X%26rft_id%253Dinfo%253Apmid%252F20946835%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0076-6879(10)70032-X&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20946835&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1153.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000275827900032&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1153.31" data-doi="10.1038/s41598-017-02013-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Satomura</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nishioka</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mori</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sato</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuroda</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Precise genome-wide base editing by the CRISPR Nickase system in yeast.</span> <abbr class="cit-jnl-abbrev">Sci. Rep.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">2095</span>. Erratum: 12354.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/s41598-017-02013-7</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BRep.%26rft.volume%253D7%26rft.spage%253D2095%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41598-017-02013-7%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41598-017-02013-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1153.32" data-doi="10.1016/j.tibs.2015.08.006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sfeir</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Symington</span>  <span class="cit-name-given-names">L. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Microhomology-mediated end joining: a back-up survival mechanism or dedicated pathway?</span> <abbr class="cit-jnl-abbrev">Trends Biochem. Sci.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">701</span>–<span class="cit-lpage">714</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.tibs.2015.08.006</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BBiochem.%2BSci.%26rft.volume%253D40%26rft.spage%253D701%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tibs.2015.08.006%26rft_id%253Dinfo%253Apmid%252F26439531%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tibs.2015.08.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26439531&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1153.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1153.33" data-doi="10.1038/ncb1101-966"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tanaka</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Russell</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Mrc1 channels the DNA replication arrest signal to checkpoint kinase Cds1.</span> <abbr class="cit-jnl-abbrev">Nat. Cell Biol.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">966</span>–<span class="cit-lpage">972</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ncb1101-966</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2BCell%2BBiology%26rft.stitle%253DNature%2BCell%2BBiology%26rft.aulast%253DTanaka%26rft.auinit1%253DK.%26rft.volume%253D3%26rft.issue%253D11%26rft.spage%253D966%26rft.epage%253D972%26rft.atitle%253DMrc1%2Bchannels%2Bthe%2BDNA%2Breplication%2Barrest%2Bsignal%2Bto%2Bcheckpoint%2Bkinase%2BCds1.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncb1101-966%26rft_id%253Dinfo%253Apmid%252F11715017%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncb1101-966&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11715017&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1153.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000172070300011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1153.34" data-doi="10.1101/gad.280545.116"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Verma</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Greenberg</span>  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Noncanonical views of homology-directed DNA repair.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">1138</span>–<span class="cit-lpage">1154</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gad.280545.116</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DVerma%26rft.auinit1%253DP.%26rft.volume%253D30%26rft.issue%253D10%26rft.spage%253D1138%26rft.epage%253D1154%26rft.atitle%253DNoncanonical%2Bviews%2Bof%2Bhomology-directed%2BDNA%2Brepair%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.280545.116%26rft_id%253Dinfo%253Apmid%252F27222516%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6MTA6IjMwLzEwLzExMzgiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzQvMTE1My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1153.35" data-doi="10.1038/nature07002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wilhelm</span>  <span class="cit-name-given-names">B. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marguerat</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Watt</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schubert</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wood</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Dynamic repertoire of a eukaryotic transcriptome surveyed at single-nucleotide resolution.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">453</span>: <span class="cit-fpage">1239</span>–<span class="cit-lpage">1243</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature07002</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DWilhelm%26rft.auinit1%253DB.%2BT.%26rft.volume%253D453%26rft.issue%253D7199%26rft.spage%253D1239%26rft.epage%253D1243%26rft.atitle%253DDynamic%2Brepertoire%2Bof%2Ba%2Beukaryotic%2Btranscriptome%2Bsurveyed%2Bat%2Bsingle-nucleotide%2Bresolution.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature07002%26rft_id%253Dinfo%253Apmid%252F18488015%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature07002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18488015&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1153.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000257075800043&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1153.36" data-doi="10.1101/gad.1406706"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davenport</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kelly</span>  <span class="cit-name-given-names">T. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Two-stage mechanism for activation of the DNA replication checkpoint kinase Cds1 in fission yeast.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">990</span>–<span class="cit-lpage">1003</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gad.1406706</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DXu%26rft.auinit1%253DY.-j.%26rft.volume%253D20%26rft.issue%253D8%26rft.spage%253D990%26rft.epage%253D1003%26rft.atitle%253DTwo-stage%2Bmechanism%2Bfor%2Bactivation%2Bof%2Bthe%2BDNA%2Breplication%2Bcheckpoint%2Bkinase%2BCds1%2Bin%2Bfission%2Byeast%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.1406706%26rft_id%253Dinfo%253Apmid%252F16618806%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6ODoiMjAvOC85OTAiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzQvMTE1My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1153.37" data-doi="10.1534/g3.118.200164"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">X.-R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">He</span>  <span class="cit-name-given-names">J.-B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Y.-Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Du</span>  <span class="cit-name-given-names">L.-L.</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">A cloning-free method for CRISPR/Cas9-mediated genome editing in fission yeast.</span> <abbr class="cit-jnl-abbrev">Genes Genomes Genetics</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">2067</span>–<span class="cit-lpage">2077</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.118.200164</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2BGenomes%2BGenetics%26rft.volume%253D8%26rft.spage%253D2067%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.118.200164%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/g3.118.200164&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-9.4.1153.38" data-doi="10.1093/nar/25.4.904"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pape</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Molecular Cloning and Analysis of <em>Schizosaccharomyces pombe</em> Reb1p : Sequence-Specific Recognition of Two Sites in the Far Upstream RDNA Intergenic Spacer.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">904</span>–<span class="cit-lpage">910</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/25.4.904</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F25.4.904%26rft_id%253Dinfo%253Apmid%252F9016645%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/25.4.904&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9016645&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F4%2F1153.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1997WJ77000032&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/9/4/1153.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/4/1141" title="RNAi Screen in Tribolium Reveals Involvement of F-BAR Proteins in Myoblast Fusion and Visceral Muscle Morphogenesis in Insects" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/4/1165" title="Bidirectional Selection for Body Weight on Standing Genetic Variation in a Chicken Model" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/9/4"><span>Volume 9 Issue 4, April 2019</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/9/4.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/9/4.cover-source.jpg?itok=mcy5SuIK" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 9 (4)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/4/1153" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-VsCHG8lO0eZ6ViKgIqEaARlNcI4dCV1H4m8bRMTd_GY" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/9/4/1153" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/9/4/1153" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/9/4/1153" class="active" data-icon-position="" data-hide-link-title="0">Short-Homology-Mediated CRISPR/Cas9-Based Method for Genome Editing in Fission Yeast</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/492598" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-aVFQ-IEjMvOy6K_BfRi4FUz1PlBQoSenQ5CI17txQSs" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/4/1153.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/9/4/1153" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-uAEOUdVE_HxwFu8qVWciHZlpaK06ROYsACgdmayEFzU" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/9/4/1153" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/4/1153.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="492598" id="node492598--2" data-pisa="ggg;9/4/1153" data-pisa-master="ggg;g3.118.200976" data-apath="/ggg/9/4/1153.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Short-Homology-Mediated CRISPR/Cas9-Based Method for Genome Editing in Fission Yeast</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-1953-1664" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Aki Hayashi</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-9692-0968" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Katsunori Tanaka</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1153-1163;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200976 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node492598--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Aki Hayashi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Bioscience, School of Science and Technology, Kwansei Gakuin University, Sanda, Hyogo, 669-1337, Japan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Aki%2BHayashi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hayashi%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAki%2BHayashi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-1953-1664" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Aki Hayashi</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:akihayashi@kwansei.ac.jp" class="" data-icon-position="" data-hide-link-title="0">akihayashi@kwansei.ac.jp</a>
<a href="mailto:katsunori@kwansei.ac.jp" class="" data-icon-position="" data-hide-link-title="0">katsunori@kwansei.ac.jp</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Katsunori Tanaka </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Bioscience, School of Science and Technology, Kwansei Gakuin University, Sanda, Hyogo, 669-1337, Japan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Katsunori%2BTanaka%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tanaka%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKatsunori%2BTanaka%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-9692-0968" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Katsunori Tanaka</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:akihayashi@kwansei.ac.jp" class="" data-icon-position="" data-hide-link-title="0">akihayashi@kwansei.ac.jp</a>
<a href="mailto:katsunori@kwansei.ac.jp" class="" data-icon-position="" data-hide-link-title="0">katsunori@kwansei.ac.jp</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/9/4/1153" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/9/4/1153&amp;title=Short-Homology-Mediated%20CRISPR/Cas9-Based%20Method%20for%20Genome%20Editing%20in%20Fission%20Yeast" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/9/4/1153&amp;title=Short-Homology-Mediated%20CRISPR/Cas9-Based%20Method%20for%20Genome%20Editing%20in%20Fission%20Yeast" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/9/4/1153&amp;title=Short-Homology-Mediated%20CRISPR/Cas9-Based%20Method%20for%20Genome%20Editing%20in%20Fission%20Yeast" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/9/4/1153&amp;text=Short-Homology-Mediated%20CRISPR/Cas9-Based%20Method%20for%20Genome%20Editing%20in%20Fission%20Yeast" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/9/4/1153&amp;title=Short-Homology-Mediated%20CRISPR/Cas9-Based%20Method%20for%20Genome%20Editing%20in%20Fission%20Yeast" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/9/4/1153&amp;t=Short-Homology-Mediated%20CRISPR/Cas9-Based%20Method%20for%20Genome%20Editing%20in%20Fission%20Yeast" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/9/4/1153&amp;title=Short-Homology-Mediated%20CRISPR/Cas9-Based%20Method%20for%20Genome%20Editing%20in%20Fission%20Yeast" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/9/4/1153&amp;title=Short-Homology-Mediated%20CRISPR/Cas9-Based%20Method%20for%20Genome%20Editing%20in%20Fission%20Yeast" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/492598/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="492598" id="node492598--3" data-pisa="ggg;9/4/1153" data-pisa-master="ggg;g3.118.200976" data-apath="/ggg/9/4/1153.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Short-Homology-Mediated CRISPR/Cas9-Based Method for Genome Editing in Fission Yeast</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0003-1953-1664" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Aki Hayashi</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-9692-0968" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Katsunori Tanaka</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">April 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 4 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1153-1163;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200976 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node492598--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Aki Hayashi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Bioscience, School of Science and Technology, Kwansei Gakuin University, Sanda, Hyogo, 669-1337, Japan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Aki%2BHayashi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hayashi%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAki%2BHayashi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-1953-1664" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Aki Hayashi</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:akihayashi@kwansei.ac.jp" class="" data-icon-position="" data-hide-link-title="0">akihayashi@kwansei.ac.jp</a>
<a href="mailto:katsunori@kwansei.ac.jp" class="" data-icon-position="" data-hide-link-title="0">katsunori@kwansei.ac.jp</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Katsunori Tanaka </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Bioscience, School of Science and Technology, Kwansei Gakuin University, Sanda, Hyogo, 669-1337, Japan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Katsunori%2BTanaka%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tanaka%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKatsunori%2BTanaka%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-9692-0968" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Katsunori Tanaka</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:akihayashi@kwansei.ac.jp" class="" data-icon-position="" data-hide-link-title="0">akihayashi@kwansei.ac.jp</a>
<a href="mailto:katsunori@kwansei.ac.jp" class="" data-icon-position="" data-hide-link-title="0">katsunori@kwansei.ac.jp</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1153&amp;rft.epage=1163&amp;rft.atitle=Short-Homology-Mediated%20CRISPR/Cas9-Based%20Method%20for%20Genome%20Editing%20in%20Fission%20Yeast&amp;rft.volume=9&amp;rft.issue=4&amp;rft.date=2019-04-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Hayashi%2C+Aki&amp;rft.au=Tanaka%2C+Katsunori"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/492598/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/492598/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/492598/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/492598/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/492598/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/492598/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/492598/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/492598/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/492598/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/492598/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/492598/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/492598/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/9/4/1153" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials AND Methods</a></li><li class="odd"><a href="#sec-10" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-14" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">ACKNOWLEDGMENTS</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/9/4/1153.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/9/4/1153.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/9/4/1153.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/9/4/1153.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
